> id: https://www.nature.com/articles/s41588-025-02292-1

> link: https://www.nature.com/articles/s41588-025-02292-1

> title: Temporal genomic dynamics shape clinical trajectory in multiple myeloma

# Temporal genomic dynamics shape clinical trajectory in multiple myeloma
_Published on Wed, 20 Aug 2025 00:00:00 GMT_

Abstract
--------

Multiple myeloma evolution is characterized by the accumulation of genomic drivers over time. To unravel this timeline and its impact on clinical outcomes, we analyzed 421 whole-genome sequences from 382 patients. Using clock-like mutational signatures, we estimated a time lag of two to four decades between the initiation of events and diagnosis. We demonstrate that odd-numbered chromosome trisomies in patients with hyperdiploidy can be acquired simultaneously with other chromosomal gains (for example, 1q gain). We show that hyperdiploidy is acquired after immunoglobulin heavy chain translocation when both events co-occur. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra copy), but fared worse than those with late 1q gain. This finding underscores that the 1q gain prognostic impact depends more on the timing of acquisition than on the number of copies gained. Overall, this study contributes to a better understanding of the life history of myeloma and may have prognostic implications.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 12 print issues and online access

$209.00 per year

only $17.42 per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02292-1?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Multiple myeloma mutational signatures landscape.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_Fig1_HTML.png)

**Fig. 2: Molecular time and temporal relationship between large chromosomal gains in multiple myeloma.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_Fig2_HTML.png)

**Fig. 3: Multiple myeloma molecular times.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_Fig3_HTML.png)

**Fig. 4: Contribution of mutational signatures in duplicated and nonduplicated mutations.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_Fig4_HTML.png)

**Fig. 5: Timing the initiation of multiple myeloma.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_Fig5_HTML.png)

**Fig. 6: Timing of canonical IGH and _MYC_ translocations.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_Fig6_HTML.png)

**Fig. 7: Timing and patterns of 1q gain and amplification.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_Fig7_HTML.png)

Data availability
-----------------

GMMG-HD6 has been uploaded on EGA—[EGAD50000000681](https://ega-archive.org/datasets/EGAD50000000681), [EGAD50000000682](https://ega-archive.org/datasets/EGAD50000000682) and [EGAD50000000683](https://ega-archive.org/datasets/EGAD50000000683). MSKCC WGS data have been uploaded to EGA—[EGAD00001011132](https://ega-archive.org/datasets/EGAD00001011132). RRMM WGS data have been uploaded to EGA—[EGAS00001006538](https://ega-archive.org/studies/EGAS00001006538), [EGAS00001004363](https://ega-archive.org/studies/EGAS00001004363), [EGAS00001004805](https://ega-archive.org/studies/EGAS00001004805) and [EGAS00001005973](https://ega-archive.org/studies/EGAS00001005973). [Source data](https://www.nature.com/articles/s41588-025-02292-1#Sec27) are provided with this paper.

Code availability
-----------------

The code used for the timing analysis is provided in Supplementary Data [2](https://www.nature.com/articles/s41588-025-02292-1#MOESM5) and on GitHub at [https://github.com/bachisiozic/Timing-the-first-multi-gain-events-in-Multiple-Myeloma/tree/main](https://github.com/bachisiozic/Timing-the-first-multi-gain-events-in-Multiple-Myeloma/tree/main).

References
----------

1.  Maura, F. et al. Moving from cancer burden to cancer genomics for smoldering myeloma: a review. _JAMA Oncol._ **6**, 425–432 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31830214)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7430066)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Moving%20from%20cancer%20burden%20to%20cancer%20genomics%20for%20smoldering%20myeloma%3A%20a%20review&journal=JAMA%20Oncol.&volume=6&pages=425-432&publication_year=2020&author=Maura%2CF) 
    
2.  Morgan, G. J., Walker, B. A. & Davies, F. E. The genetic architecture of multiple myeloma. _Nat. Rev. Cancer_ **12**, 335–348 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xltl2rurg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22495321)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genetic%20architecture%20of%20multiple%20myeloma&journal=Nat.%20Rev.%20Cancer&volume=12&pages=335-348&publication_year=2012&author=Morgan%2CGJ&author=Walker%2CBA&author=Davies%2CFE) 
    
3.  Manier, S. et al. Genomic complexity of multiple myeloma and its clinical implications. _Nat. Rev. Clin. Oncol._ **14**, 100–113 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhtl2mt7bI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27531699)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20complexity%20of%20multiple%20myeloma%20and%20its%20clinical%20implications&journal=Nat.%20Rev.%20Clin.%20Oncol.&volume=14&pages=100-113&publication_year=2017&author=Manier%2CS) 
    
4.  Rajkumar, S. V., Landgren, O. & Mateos, M. V. Smoldering multiple myeloma. _Blood_ **125**, 3069–3075 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXpsFKjtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25838344)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432003)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Smoldering%20multiple%20myeloma&journal=Blood&volume=125&pages=3069-3075&publication_year=2015&author=Rajkumar%2CSV&author=Landgren%2CO&author=Mateos%2CMV) 
    
5.  Kyle, R. A. et al. Long-term follow-up of monoclonal gammopathy of undetermined significance. _N. Engl. J. Med._ **378**, 241–249 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFKmtb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29342381)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5852672)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Long-term%20follow-up%20of%20monoclonal%20gammopathy%20of%20undetermined%20significance&journal=N.%20Engl.%20J.%20Med.&volume=378&pages=241-249&publication_year=2018&author=Kyle%2CRA) 
    
6.  Kyle, R. A. et al. Clinical course and prognosis of smoldering (asymptomatic) multiple myeloma. _N. Engl. J. Med._ **356**, 2582–2590 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXmslKjsrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17582068)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20course%20and%20prognosis%20of%20smoldering%20%28asymptomatic%29%20multiple%20myeloma&journal=N.%20Engl.%20J.%20Med.&volume=356&pages=2582-2590&publication_year=2007&author=Kyle%2CRA) 
    
7.  Barwick, B. G., Gupta, V. A., Vertino, P. M. & Boise, L. H. Cell of origin and genetic alterations in the pathogenesis of multiple myeloma. _Front. Immunol._ **10**, 1121 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsV2ls7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31231360)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6558388)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell%20of%20origin%20and%20genetic%20alterations%20in%20the%20pathogenesis%20of%20multiple%20myeloma&journal=Front.%20Immunol.&volume=10&publication_year=2019&author=Barwick%2CBG&author=Gupta%2CVA&author=Vertino%2CPM&author=Boise%2CLH) 
    
8.  Bergsagel, P. L. et al. Cyclin D dysregulation: an early and unifying pathogenic event in multiple myeloma. _Blood_ **106**, 296–303 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlvVWjtrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15755896)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1895118)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cyclin%20D%20dysregulation%3A%20an%20early%20and%20unifying%20pathogenic%20event%20in%20multiple%20myeloma&journal=Blood&volume=106&pages=296-303&publication_year=2005&author=Bergsagel%2CPL) 
    
9.  Chesi, M. & Bergsagel, P. L. Molecular pathogenesis of multiple myeloma: basic and clinical updates. _Int. J. Hematol._ **97**, 313–323 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXksFenu7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23456262)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3962846)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20pathogenesis%20of%20multiple%20myeloma%3A%20basic%20and%20clinical%20updates&journal=Int.%20J.%20Hematol.&volume=97&pages=313-323&publication_year=2013&author=Chesi%2CM&author=Bergsagel%2CPL) 
    
10.  Chesi, M. et al. Dysregulation of cyclin D1 by translocation into an IgH γ switch region in two multiple myeloma cell lines. _Blood_ **88**, 674–681 (1996).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XktFWisLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8695815)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dysregulation%20of%20cyclin%20D1%20by%20translocation%20into%20an%20IgH%20%CE%B3%20switch%20region%20in%20two%20multiple%20myeloma%20cell%20lines&journal=Blood&volume=88&pages=674-681&publication_year=1996&author=Chesi%2CM) 
    
11.  Chesi, M. et al. Frequent translocation t(4;14)(p16.3;q32.3) in multiple myeloma is associated with increased expression and activating mutations of fibroblast growth factor receptor 3. _Nat. Genet._ **16**, 260–264 (1997).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXktFGjsrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9207791)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3901950)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Frequent%20translocation%20t%284%3B14%29%28p16.3%3Bq32.3%29%20in%20multiple%20myeloma%20is%20associated%20with%20increased%20expression%20and%20activating%20mutations%20of%20fibroblast%20growth%20factor%20receptor%203&journal=Nat.%20Genet.&volume=16&pages=260-264&publication_year=1997&author=Chesi%2CM) 
    
12.  Chojnacka, M., Diamond, B., Landgren, O. & Maura, F. Defining genomic events involved in the evolutionary trajectories of myeloma and its precursor conditions. _Semin. Oncol._ **49**, 11–18 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1egt77O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35168813)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9149131)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defining%20genomic%20events%20involved%20in%20the%20evolutionary%20trajectories%20of%20myeloma%20and%20its%20precursor%20conditions&journal=Semin.%20Oncol.&volume=49&pages=11-18&publication_year=2022&author=Chojnacka%2CM&author=Diamond%2CB&author=Landgren%2CO&author=Maura%2CF) 
    
13.  Cirrincione, A. et al. Revealing novel mechanisms underlying inactivation of tumor suppressor genes on duplicated chromosomes in multiple myeloma. _Blood_ **142**, 874–874 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revealing%20novel%20mechanisms%20underlying%20inactivation%20of%20tumor%20suppressor%20genes%20on%20duplicated%20chromosomes%20in%20multiple%20myeloma&journal=Blood&volume=142&pages=874-874&publication_year=2023&author=Cirrincione%2CA) 
    
14.  Fonseca, R. et al. The recurrent IgH translocations are highly associated with nonhyperdiploid variant multiple myeloma. _Blood_ **102**, 2562–2567 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXnslykur8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12805059)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20recurrent%20IgH%20translocations%20are%20highly%20associated%20with%20nonhyperdiploid%20variant%20multiple%20myeloma&journal=Blood&volume=102&pages=2562-2567&publication_year=2003&author=Fonseca%2CR) 
    
15.  Rustad, E. H. et al. Revealing the impact of structural variants in multiple myeloma. _Blood Cancer Discov._ **1**, 258–273 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvV2ktrbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33392515)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7774871)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Revealing%20the%20impact%20of%20structural%20variants%20in%20multiple%20myeloma&journal=Blood%20Cancer%20Discov.&volume=1&pages=258-273&publication_year=2020&author=Rustad%2CEH) 
    
16.  Walker, B. A. et al. Mutational spectrum, copy number changes, and outcome: results of a sequencing study of patients with newly diagnosed myeloma. _J. Clin. Oncol._ **33**, 3911–3920 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XmvVynsbY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26282654)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20spectrum%2C%20copy%20number%20changes%2C%20and%20outcome%3A%20results%20of%20a%20sequencing%20study%20of%20patients%20with%20newly%20diagnosed%20myeloma&journal=J.%20Clin.%20Oncol.&volume=33&pages=3911-3920&publication_year=2015&author=Walker%2CBA) 
    
17.  Walker, B. A. et al. A high-risk, double-hit, group of newly diagnosed myeloma identified by genomic analysis. _Leukemia_ **33**, 159–170 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29967379)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6326953)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20high-risk%2C%20double-hit%2C%20group%20of%20newly%20diagnosed%20myeloma%20identified%20by%20genomic%20analysis&journal=Leukemia&volume=33&pages=159-170&publication_year=2018&author=Walker%2CBA) 
    
18.  Bergsagel, P. L. et al. Promiscuous translocations into immunoglobulin heavy chain switch regions in multiple myeloma. _Proc. Natl Acad. Sci. USA_ **93**, 13931–13936 (1996).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28Xnt1GltL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8943038)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC19472)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Promiscuous%20translocations%20into%20immunoglobulin%20heavy%20chain%20switch%20regions%20in%20multiple%20myeloma&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=93&pages=13931-13936&publication_year=1996&author=Bergsagel%2CPL) 
    
19.  Bolli, N. et al. Genomic patterns of progression in smoldering multiple myeloma. _Nat. Commun._ **9**, 3363 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30135448)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6105687)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20patterns%20of%20progression%20in%20smoldering%20multiple%20myeloma&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Bolli%2CN) 
    
20.  Boyle, E. M. et al. The molecular make up of smoldering myeloma highlights the evolutionary pathways leading to multiple myeloma. _Nat. Commun._ **12**, 293 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVelsb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33436579)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7804406)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20make%20up%20of%20smoldering%20myeloma%20highlights%20the%20evolutionary%20pathways%20leading%20to%20multiple%20myeloma&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Boyle%2CEM) 
    
21.  Maura, F. et al. Genomic landscape and chronological reconstruction of driver events in multiple myeloma. _Nat. Commun._ **10**, 3835 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31444325)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707220)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20landscape%20and%20chronological%20reconstruction%20of%20driver%20events%20in%20multiple%20myeloma&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Maura%2CF) 
    
22.  Misund, K. et al. MYC dysregulation in the progression of multiple myeloma. _Leukemia_ **34**, 322–326 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31439946)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MYC%20dysregulation%20in%20the%20progression%20of%20multiple%20myeloma&journal=Leukemia&volume=34&pages=322-326&publication_year=2020&author=Misund%2CK) 
    
23.  Smadja, N. V. et al. Further cytogenetic characterization of multiple myeloma confirms that 14q32 translocations are a very rare event in hyperdiploid cases. _Genes Chromosomes Cancer_ **38**, 234–239 (2003).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14506697)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Further%20cytogenetic%20characterization%20of%20multiple%20myeloma%20confirms%20that%2014q32%20translocations%20are%20a%20very%20rare%20event%20in%20hyperdiploid%20cases&journal=Genes%20Chromosomes%20Cancer&volume=38&pages=234-239&publication_year=2003&author=Smadja%2CNV) 
    
24.  Bolli, F. et al. Genomic patterns of progression in smoldering multiple myeloma. _Nat. Commun._ **9**, 3363 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30135448)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6105687)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20patterns%20of%20progression%20in%20smoldering%20multiple%20myeloma&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Bolli%2CF) 
    
25.  Barwick, B. G. et al. Multiple myeloma immunoglobulin lambda translocations portend poor prognosis. _Nat. Commun._ **10**, 1911 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31015454)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6478743)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiple%20myeloma%20immunoglobulin%20lambda%20translocations%20portend%20poor%20prognosis&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Barwick%2CBG) 
    
26.  Rustad, E. H. et al. Timing the initiation of multiple myeloma. _Nat. Commun._ **11**, 1917 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnslSktbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32317634)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7174344)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Timing%20the%20initiation%20of%20multiple%20myeloma&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Rustad%2CEH) 
    
27.  Bustoros, M. et al. Genomic profiling of smoldering multiple myeloma identifies patients at a high risk of disease progression. _J. Clin. Oncol._ **38**, 2380–2389 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFylsr3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32442065)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7367550)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20profiling%20of%20smoldering%20multiple%20myeloma%20identifies%20patients%20at%20a%20high%20risk%20of%20disease%20progression&journal=J.%20Clin.%20Oncol.&volume=38&pages=2380-2389&publication_year=2020&author=Bustoros%2CM) 
    
28.  Oben, B. et al. Whole-genome sequencing reveals progressive versus stable myeloma precursor conditions as two distinct entities. _Nat. Commun._ **12**, 1861 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXns1aku74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33767199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7994386)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20sequencing%20reveals%20progressive%20versus%20stable%20myeloma%20precursor%20conditions%20as%20two%20distinct%20entities&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Oben%2CB) 
    
29.  Walker, B. A. et al. Identification of novel mutational drivers reveals oncogene dependencies in multiple myeloma. _Blood_ **132**, 587–597 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVWlt7zK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29884741)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6097138)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20novel%20mutational%20drivers%20reveals%20oncogene%20dependencies%20in%20multiple%20myeloma&journal=Blood&volume=132&pages=587-597&publication_year=2018&author=Walker%2CBA) 
    
30.  D’Agostino, M. et al. Second Revision of the International Staging System (R2-ISS) for overall survival in multiple myeloma: a European Myeloma Network (EMN) report within the HARMONY project. _J. Clin. Oncol._ **40**, 3406–3418 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35605179)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Second%20Revision%20of%20the%20International%20Staging%20System%20%28R2-ISS%29%20for%20overall%20survival%20in%20multiple%20myeloma%3A%20a%20European%20Myeloma%20Network%20%28EMN%29%20report%20within%20the%20HARMONY%20project&journal=J.%20Clin.%20Oncol.&volume=40&pages=3406-3418&publication_year=2022&author=D%E2%80%99Agostino%2CM) 
    
31.  Weinhold, N. et al. Chromosome 1q21 abnormalities refine outcome prediction in patients with multiple myeloma: a meta-analysis of 2,596 trial patients. _Haematologica_ **106**, 2754–2758 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34092058)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8485656)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosome%201q21%20abnormalities%20refine%20outcome%20prediction%20in%20patients%20with%20multiple%20myeloma%3A%20a%20meta-analysis%20of%202%2C596%20trial%20patients&journal=Haematologica&volume=106&pages=2754-2758&publication_year=2021&author=Weinhold%2CN) 
    
32.  Walker, B. A. et al. A high-risk, Double-Hit, group of newly diagnosed myeloma identified by genomic analysis. _Leukemia_ **33**, 159–170 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1yksLnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29967379)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20high-risk%2C%20Double-Hit%2C%20group%20of%20newly%20diagnosed%20myeloma%20identified%20by%20genomic%20analysis&journal=Leukemia&volume=33&pages=159-170&publication_year=2019&author=Walker%2CBA) 
    
33.  Maura, F. et al. Genomic classification and individualized prognosis in multiple myeloma. _J. Clin. Oncol._ **42**, 1229–1240 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXnslWqt7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38194610)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20classification%20and%20individualized%20prognosis%20in%20multiple%20myeloma&journal=J.%20Clin.%20Oncol.&volume=42&pages=1229-1240&publication_year=2024&author=Maura%2CF) 
    
34.  Kaiser, M. F. et al. Co-occurrence of cytogenetic abnormalities and high-risk disease in newly diagnosed and relapsed/refractory multiple myeloma. _J. Clin. Oncol._ [https://doi.org/10.1200/jco-24-01253](https://doi.org/10.1200/jco-24-01253) (2025).
    
35.  Weinhold, N. et al. Clonal selection and double-hit events involving tumor suppressor genes underlie relapse in myeloma. _Blood_ **128**, 1735–1744 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlSitrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27516441)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5043128)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20selection%20and%20double-hit%20events%20involving%20tumor%20suppressor%20genes%20underlie%20relapse%20in%20myeloma&journal=Blood&volume=128&pages=1735-1744&publication_year=2016&author=Weinhold%2CN) 
    
36.  Lannes, R. et al. In multiple myeloma, high-risk secondary genetic events observed at relapse are present from diagnosis in tiny, undetectable subclonal populations. _J. Clin. Oncol._ **41**, 1695–1702 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXltlGht74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36343306)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In%20multiple%20myeloma%2C%20high-risk%20secondary%20genetic%20events%20observed%20at%20relapse%20are%20present%20from%20diagnosis%20in%20tiny%2C%20undetectable%20subclonal%20populations&journal=J.%20Clin.%20Oncol.&volume=41&pages=1695-1702&publication_year=2023&author=Lannes%2CR) 
    
37.  Gerstung, M. et al. The evolutionary history of 2,658 cancers. _Nature_ **578**, 122–128 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32025013)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054212)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20evolutionary%20history%20of%202%2C658%20cancers&journal=Nature&volume=578&pages=122-128&publication_year=2020&author=Gerstung%2CM) 
    
38.  Mitchell, T. J. et al. Timing the landmark events in the evolution of clear cell renal cell cancer: TRACERx renal. _Cell_ **173**, 611–623 e617 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXnsFClu78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29656891)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5927631)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Timing%20the%20landmark%20events%20in%20the%20evolution%20of%20clear%20cell%20renal%20cell%20cancer%3A%20TRACERx%20renal&journal=Cell&volume=173&pages=611-623%20e617&publication_year=2018&author=Mitchell%2CTJ) 
    
39.  Goldschmidt, H. et al. Elotuzumab in combination with lenalidomide, bortezomib, dexamethasone and autologous transplantation for newly-diagnosed multiple myeloma: results from the randomized phase III GMMG-HD6 trial. _Blood_ **138**, 486 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elotuzumab%20in%20combination%20with%20lenalidomide%2C%20bortezomib%2C%20dexamethasone%20and%20autologous%20transplantation%20for%20newly-diagnosed%20multiple%20myeloma%3A%20results%20from%20the%20randomized%20phase%20III%20GMMG-HD6%20trial&journal=Blood&volume=138&publication_year=2021&author=Goldschmidt%2CH) 
    
40.  Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. _Nature_ **578**, 94–101 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32025018)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054213)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&journal=Nature&volume=578&pages=94-101&publication_year=2020&author=Alexandrov%2CLB) 
    
41.  Maura, F. et al. A practical guide for mutational signature analysis in hematological malignancies. _Nat. Commun._ **10**, 2969 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31278357)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611883)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20practical%20guide%20for%20mutational%20signature%20analysis%20in%20hematological%20malignancies&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Maura%2CF) 
    
42.  Degasperi, A. et al. Substitution mutational signatures in whole-genome-sequenced cancers in the UK population. _Science_ **376**, abl9283 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Substitution%20mutational%20signatures%20in%20whole-genome-sequenced%20cancers%20in%20the%20UK%20population&journal=Science&volume=376&publication_year=2022&author=Degasperi%2CA) 
    
43.  Walker, B. A. et al. APOBEC family mutational signatures are associated with poor prognosis translocations in multiple myeloma. _Nat. Commun._ **6**, 6997 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2itLnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25904160)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=APOBEC%20family%20mutational%20signatures%20are%20associated%20with%20poor%20prognosis%20translocations%20in%20multiple%20myeloma&journal=Nat.%20Commun.&volume=6&publication_year=2015&author=Walker%2CBA) 
    
44.  Maura, F. et al. Biological and prognostic impact of APOBEC-induced mutations in the spectrum of plasma cell dyscrasias and multiple myeloma cell lines. _Leukemia_ **32**, 1044–1048 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXltlGktw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29209044)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5886048)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Biological%20and%20prognostic%20impact%20of%20APOBEC-induced%20mutations%20in%20the%20spectrum%20of%20plasma%20cell%20dyscrasias%20and%20multiple%20myeloma%20cell%20lines&journal=Leukemia&volume=32&pages=1044-1048&publication_year=2018&author=Maura%2CF) 
    
45.  Samur, M. K. et al. Genome-wide somatic alterations in multiple myeloma reveal a superior outcome group. _J. Clin. Oncol._ **38**, 3107–3118 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVyqurrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32687451)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7499613)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20somatic%20alterations%20in%20multiple%20myeloma%20reveal%20a%20superior%20outcome%20group&journal=J.%20Clin.%20Oncol.&volume=38&pages=3107-3118&publication_year=2020&author=Samur%2CMK) 
    
46.  Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. _Nature_ **593**, 405–410 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVSrur7F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33911282)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutation%20landscapes%20at%20single-molecule%20resolution&journal=Nature&volume=593&pages=405-410&publication_year=2021&author=Abascal%2CF) 
    
47.  Alexandrov, L. B. et al. Clock-like mutational processes in human somatic cells. _Nat. Genet._ **47**, 1402–1407 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVSgt7bN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26551669)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4783858)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clock-like%20mutational%20processes%20in%20human%20somatic%20cells&journal=Nat.%20Genet.&volume=47&pages=1402-1407&publication_year=2015&author=Alexandrov%2CLB) 
    
48.  Cagan, A. et al. Somatic mutation rates scale with lifespan across mammals. _Nature_ **604**, 517–524 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvV2iur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35418684)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9021023)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutation%20rates%20scale%20with%20lifespan%20across%20mammals&journal=Nature&volume=604&pages=517-524&publication_year=2022&author=Cagan%2CA) 
    
49.  Moore, L. et al. The mutational landscape of human somatic and germline cells. _Nature_ **597**, 381–386 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVygtLrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34433962)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20landscape%20of%20human%20somatic%20and%20germline%20cells&journal=Nature&volume=597&pages=381-386&publication_year=2021&author=Moore%2CL) 
    
50.  Luquette, L. J. et al. Single-cell genome sequencing of human neurons identifies somatic point mutation and indel enrichment in regulatory elements. _Nat. Genet._ **54**, 1564–1571 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisV2nsrvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36163278)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9833626)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20genome%20sequencing%20of%20human%20neurons%20identifies%20somatic%20point%20mutation%20and%20indel%20enrichment%20in%20regulatory%20elements&journal=Nat.%20Genet.&volume=54&pages=1564-1571&publication_year=2022&author=Luquette%2CLJ) 
    
51.  Machado, H. E. et al. Diverse mutational landscapes in human lymphocytes. _Nature_ **608**, 724–732 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFSjtbvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35948631)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9402440)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Diverse%20mutational%20landscapes%20in%20human%20lymphocytes&journal=Nature&volume=608&pages=724-732&publication_year=2022&author=Machado%2CHE) 
    
52.  Maura, F. et al. The mutagenic impact of melphalan in multiple myeloma. _Leukemia_ **35**, 2145–2150 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlKnu73E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34012133)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutagenic%20impact%20of%20melphalan%20in%20multiple%20myeloma&journal=Leukemia&volume=35&pages=2145-2150&publication_year=2021&author=Maura%2CF) 
    
53.  D’Agostino, M. et al. Early relapse risk in patients with newly diagnosed multiple myeloma characterized by next-generation sequencing. _Clin. Cancer Res._ **26**, 4832–4841 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32616499)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Early%20relapse%20risk%20in%20patients%20with%20newly%20diagnosed%20multiple%20myeloma%20characterized%20by%20next-generation%20sequencing&journal=Clin.%20Cancer%20Res.&volume=26&pages=4832-4841&publication_year=2020&author=D%E2%80%99Agostino%2CM) 
    
54.  Misund, K. et al. Clonal evolution after treatment pressure in multiple myeloma: heterogenous genomic aberrations and transcriptomic convergence. _Leukemia_ **36**, 1887–1897 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFOgtrzL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35643867)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9252918)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20evolution%20after%20treatment%20pressure%20in%20multiple%20myeloma%3A%20heterogenous%20genomic%20aberrations%20and%20transcriptomic%20convergence&journal=Leukemia&volume=36&pages=1887-1897&publication_year=2022&author=Misund%2CK) 
    
55.  Maura, F., Rustad, E. H., Boyle, E. M. & Morgan, G. J. Reconstructing the evolutionary history of multiple myeloma. _Best Pract. Res. Clin. Haematol._ **33**, 101145 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32139011)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7389821)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reconstructing%20the%20evolutionary%20history%20of%20multiple%20myeloma&journal=Best%20Pract.%20Res.%20Clin.%20Haematol.&volume=33&publication_year=2020&author=Maura%2CF&author=Rustad%2CEH&author=Boyle%2CEM&author=Morgan%2CGJ) 
    
56.  Rustad, E. H. et al. mmsig: a fitting approach to accurately identify somatic mutational signatures in hematological malignancies. _Commun. Biol._ **4**, 424 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlehurbN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33782531)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8007623)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=mmsig%3A%20a%20fitting%20approach%20to%20accurately%20identify%20somatic%20mutational%20signatures%20in%20hematological%20malignancies&journal=Commun.%20Biol.&volume=4&publication_year=2021&author=Rustad%2CEH) 
    
57.  Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. _Cell_ **184**, 2239–2254 e2239 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXoslGkur8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33831375)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8054914)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterizing%20genetic%20intra-tumor%20heterogeneity%20across%202%2C658%20human%20cancer%20genomes&journal=Cell&volume=184&pages=2239-2254%20e2239&publication_year=2021&author=Dentro%2CSC) 
    
58.  Murray, D. et al. Detection and prevalence of monoclonal gammopathy of undetermined significance: a study utilizing mass spectrometry-based monoclonal immunoglobulin rapid accurate mass measurement. _Blood Cancer J._ **9**, 102 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31836698)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6910906)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Detection%20and%20prevalence%20of%20monoclonal%20gammopathy%20of%20undetermined%20significance%3A%20a%20study%20utilizing%20mass%20spectrometry-based%20monoclonal%20immunoglobulin%20rapid%20accurate%20mass%20measurement&journal=Blood%20Cancer%20J.&volume=9&publication_year=2019&author=Murray%2CD) 
    
59.  Thorsteinsdottir, S. et al. Prevalence of smoldering multiple myeloma based on nationwide screening. _Nat. Med._ **29**, 467–472 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXislWltrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36747117)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11099842)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prevalence%20of%20smoldering%20multiple%20myeloma%20based%20on%20nationwide%20screening&journal=Nat.%20Med.&volume=29&pages=467-472&publication_year=2023&author=Thorsteinsdottir%2CS) 
    
60.  Cirrincione, A. M. et al. The biological and clinical impact of deletions before and after large chromosomal gains in multiple myeloma. _Blood_ [https://doi.org/10.1182/blood.2024024299](https://doi.org/10.1182/blood.2024024299) (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38728430)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20biological%20and%20clinical%20impact%20of%20deletions%20before%20and%20after%20large%20chromosomal%20gains%20in%20multiple%20myeloma&journal=Blood&doi=10.1182%2Fblood.2024024299&publication_year=2024&author=Cirrincione%2CAM) 
    
61.  Poos, A. M. et al. Resolving therapy resistance mechanisms in multiple myeloma by multiomics subclone analysis. _Blood_ **142**, 1633–1646 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitlGhsrfO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37390336)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10733835)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20therapy%20resistance%20mechanisms%20in%20multiple%20myeloma%20by%20multiomics%20subclone%20analysis&journal=Blood&volume=142&pages=1633-1646&publication_year=2023&author=Poos%2CAM) 
    
62.  Diamond, B. et al. Tracking the evolution of therapy-related myeloid neoplasms using chemotherapy signatures. _Blood_ **141**, 2359–2371 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXpvFart7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36626250)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10273163)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tracking%20the%20evolution%20of%20therapy-related%20myeloid%20neoplasms%20using%20chemotherapy%20signatures&journal=Blood&volume=141&pages=2359-2371&publication_year=2023&author=Diamond%2CB) 
    
63.  Maura, F. et al. Initial whole-genome sequencing of plasma cell neoplasms in first responders and recovery workers exposed to the World Trade Center attack of September 11, 2001. _Clin. Cancer Res._ **27**, 2111–2118 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33504553)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8026670)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Initial%20whole-genome%20sequencing%20of%20plasma%20cell%20neoplasms%20in%20first%20responders%20and%20recovery%20workers%20exposed%20to%20the%20World%20Trade%20Center%20attack%20of%20September%2011%2C%202001&journal=Clin.%20Cancer%20Res.&volume=27&pages=2111-2118&publication_year=2021&author=Maura%2CF) 
    
64.  Landgren, O. et al. Monoclonal gammopathy of undetermined significance (MGUS) consistently precedes multiple myeloma: a prospective study. _Blood_ **113**, 5412–5417 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXntVertLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19179464)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2689042)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Monoclonal%20gammopathy%20of%20undetermined%20significance%20%28MGUS%29%20consistently%20precedes%20multiple%20myeloma%3A%20a%20prospective%20study&journal=Blood&volume=113&pages=5412-5417&publication_year=2009&author=Landgren%2CO) 
    
65.  Diamond, B. T. et al. Tracking the evolution of therapy-related myeloid neoplasms using chemotherapy signatures. _Blood_ [https://doi.org/10.1182/blood.2022018244](https://doi.org/10.1182/blood.2022018244) (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37490651)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10934279)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tracking%20the%20evolution%20of%20therapy-related%20myeloid%20neoplasms%20using%20chemotherapy%20signatures&journal=Blood&doi=10.1182%2Fblood.2022018244&publication_year=2023&author=Diamond%2CBT) 
    
66.  Fonseca, R. et al. Prognostic value of chromosome 1q21 gain by fluorescent in situ hybridization and increase CKS1B expression in myeloma. _Leukemia_ **20**, 2034–2040 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFWnsLrO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17024118)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prognostic%20value%20of%20chromosome%201q21%20gain%20by%20fluorescent%20in%20situ%20hybridization%20and%20increase%20CKS1B%20expression%20in%20myeloma&journal=Leukemia&volume=20&pages=2034-2040&publication_year=2006&author=Fonseca%2CR) 
    
67.  Schmidt, T. M., Fonseca, R. & Usmani, S. Z. Chromosome 1q21 abnormalities in multiple myeloma. _Blood Cancer J._ **11**, 83 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33927196)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8085148)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosome%201q21%20abnormalities%20in%20multiple%20myeloma&journal=Blood%20Cancer%20J.&volume=11&publication_year=2021&author=Schmidt%2CTM&author=Fonseca%2CR&author=Usmani%2CSZ) 
    
68.  Shah, V. et al. Prediction of outcome in newly diagnosed myeloma: a meta-analysis of the molecular profiles of 1905 trial patients. _Leukemia_ **32**, 102–110 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVymtr%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28584253)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prediction%20of%20outcome%20in%20newly%20diagnosed%20myeloma%3A%20a%20meta-analysis%20of%20the%20molecular%20profiles%20of%201905%20trial%20patients&journal=Leukemia&volume=32&pages=102-110&publication_year=2018&author=Shah%2CV) 
    
69.  Croft, J. et al. Copy number evolution and its relationship with patient outcome—an analysis of 178 matched presentation-relapse tumor pairs from the Myeloma XI trial. _Leukemia_ **35**, 2043–2053 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFent7%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33262523)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Copy%20number%20evolution%20and%20its%20relationship%20with%20patient%20outcome%E2%80%94an%20analysis%20of%20178%20matched%20presentation-relapse%20tumor%20pairs%20from%20the%20Myeloma%20XI%20trial&journal=Leukemia&volume=35&pages=2043-2053&publication_year=2021&author=Croft%2CJ) 
    
70.  Schmidt, T. M. et al. Gain of chromosome 1q is associated with early progression in multiple myeloma patients treated with lenalidomide, bortezomib, and dexamethasone. _Blood Cancer J._ **9**, 94 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31767829)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6877577)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gain%20of%20chromosome%201q%20is%20associated%20with%20early%20progression%20in%20multiple%20myeloma%20patients%20treated%20with%20lenalidomide%2C%20bortezomib%2C%20and%20dexamethasone&journal=Blood%20Cancer%20J.&volume=9&publication_year=2019&author=Schmidt%2CTM) 
    
71.  Abdallah, N. et al. Clinical characteristics and treatment outcomes of newly diagnosed multiple myeloma with chromosome 1q abnormalities. _Blood Adv._ **4**, 3509–3519 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32750129)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7422105)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20characteristics%20and%20treatment%20outcomes%20of%20newly%20diagnosed%20multiple%20myeloma%20with%20chromosome%201q%20abnormalities&journal=Blood%20Adv.&volume=4&pages=3509-3519&publication_year=2020&author=Abdallah%2CN) 
    
72.  Perrot, A. et al. Development and validation of a cytogenetic prognostic index predicting survival in multiple myeloma. _J. Clin. Oncol._ **37**, 1657–1665 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFWrsr%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31091136)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6804890)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20and%20validation%20of%20a%20cytogenetic%20prognostic%20index%20predicting%20survival%20in%20multiple%20myeloma&journal=J.%20Clin.%20Oncol.&volume=37&pages=1657-1665&publication_year=2019&author=Perrot%2CA) 
    
73.  Korde, N. et al. Treatment with carfilzomib-lenalidomide-dexamethasone with lenalidomide extension in patients with smoldering or newly diagnosed multiple myeloma. _JAMA Oncol._ **1**, 746–754 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26181891)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6662597)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Treatment%20with%20carfilzomib-lenalidomide-dexamethasone%20with%20lenalidomide%20extension%20in%20patients%20with%20smoldering%20or%20newly%20diagnosed%20multiple%20myeloma&journal=JAMA%20Oncol.&volume=1&pages=746-754&publication_year=2015&author=Korde%2CN) 
    
74.  Landgren, O. et al. Safety and effectiveness of weekly carfilzomib, lenalidomide, dexamethasone, and daratumumab combination therapy for patients with newly diagnosed multiple myeloma: the MANHATTAN nonrandomized Clinical trial. _JAMA Oncol._ **7**, 862–868 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33856405)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8050789)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Safety%20and%20effectiveness%20of%20weekly%20carfilzomib%2C%20lenalidomide%2C%20dexamethasone%2C%20and%20daratumumab%20combination%20therapy%20for%20patients%20with%20newly%20diagnosed%20multiple%20myeloma%3A%20the%20MANHATTAN%20nonrandomized%20Clinical%20trial&journal=JAMA%20Oncol.&volume=7&pages=862-868&publication_year=2021&author=Landgren%2CO) 
    
75.  Maura, F. et al. Genomic and immune signatures predict clinical outcome in newly diagnosed multiple myeloma treated with immunotherapy regimens. _Nat. Cancer_ [https://doi.org/10.1038/s43018-023-00657-1](https://doi.org/10.1038/s43018-023-00657-1) (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37945755)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12065606)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20and%20immune%20signatures%20predict%20clinical%20outcome%20in%20newly%20diagnosed%20multiple%20myeloma%20treated%20with%20immunotherapy%20regimens&journal=Nat.%20Cancer&doi=10.1038%2Fs43018-023-00657-1&publication_year=2023&author=Maura%2CF) 
    
76.  Landau, H. J. et al. Accelerated single cell seeding in relapsed multiple myeloma. _Nat. Commun._ **11**, 3617 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVakt7%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32680998)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7368016)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accelerated%20single%20cell%20seeding%20in%20relapsed%20multiple%20myeloma&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Landau%2CHJ) 
    
77.  Reisinger, E. et al. OTP: an automatized system for managing and processing NGS data. _J. Biotechnol._ **261**, 53–62 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlGrsb%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28803971)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OTP%3A%20an%20automatized%20system%20for%20managing%20and%20processing%20NGS%20data&journal=J.%20Biotechnol.&volume=261&pages=53-62&publication_year=2017&author=Reisinger%2CE) 
    
78.  ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium Pan-cancer analysis of whole genomes. _Nature_ **578**, 82–93 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-cancer%20analysis%20of%20whole%20genomes&journal=Nature&volume=578&pages=82-93&publication_year=2020) 
    
79.  Danecek, P. et al. Twelve years of SAMtools and BCFtools. _Gigascience_ **10**, giab008 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33590861)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7931819)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&journal=Gigascience&volume=10&publication_year=2021&author=Danecek%2CP) 
    
80.  Ashby, C. et al. Structural variants shape the genomic landscape and clinical outcome of multiple myeloma. _Blood Cancer J._ **12**, 85 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35637217)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9151656)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20variants%20shape%20the%20genomic%20landscape%20and%20clinical%20outcome%20of%20multiple%20myeloma&journal=Blood%20Cancer%20J.&volume=12&publication_year=2022&author=Ashby%2CC) 
    
81.  Degasperi, A. et al. A practical framework and online tool for mutational signature analyses show intertissue variation and driver dependencies. _Nat. Cancer_ **1**, 249–263 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVOmt70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32118208)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7048622)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20practical%20framework%20and%20online%20tool%20for%20mutational%20signature%20analyses%20show%20intertissue%20variation%20and%20driver%20dependencies&journal=Nat.%20Cancer&volume=1&pages=249-263&publication_year=2020&author=Degasperi%2CA) 
    
82.  Maura, F. et al. Molecular evolution of classic Hodgkin lymphoma revealed through whole-genome sequencing of Hodgkin and Reed Sternberg cells. _Blood Cancer Discov._ **4**, 208–227 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlCrsb%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36723991)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10150291)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20evolution%20of%20classic%20Hodgkin%20lymphoma%20revealed%20through%20whole-genome%20sequencing%20of%20Hodgkin%20and%20Reed%20Sternberg%20cells&journal=Blood%20Cancer%20Discov.&volume=4&pages=208-227&publication_year=2023&author=Maura%2CF) 
    
83.  Gagnon, M. F. et al. Superior detection rate of plasma cell FISH using FACS-FISH. _Am. J. Clin. Pathol._ **161**, 60–70 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXktlSns70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37658775)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Superior%20detection%20rate%20of%20plasma%20cell%20FISH%20using%20FACS-FISH&journal=Am.%20J.%20Clin.%20Pathol.&volume=161&pages=60-70&publication_year=2024&author=Gagnon%2CMF) 
    
84.  Peterson, J. F. et al. Whole genome mate-pair sequencing of plasma cell neoplasm as a novel diagnostic strategy: a case of unrecognized t(2;11) structural variation. _Clin. Lymphoma Myeloma Leuk._ **19**, 598–602 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31320253)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole%20genome%20mate-pair%20sequencing%20of%20plasma%20cell%20neoplasm%20as%20a%20novel%20diagnostic%20strategy%3A%20a%20case%20of%20unrecognized%20t%282%3B11%29%20structural%20variation&journal=Clin.%20Lymphoma%20Myeloma%20Leuk.&volume=19&pages=598-602&publication_year=2019&author=Peterson%2CJF) 
    
85.  Sidana, S. et al. Rapid assessment of hyperdiploidy in plasma cell disorders using a novel multi-parametric flow cytometry method. _Am. J. Hematol._ **94**, 424–430 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXksVCkurY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30592078)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8083940)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20assessment%20of%20hyperdiploidy%20in%20plasma%20cell%20disorders%20using%20a%20novel%20multi-parametric%20flow%20cytometry%20method&journal=Am.%20J.%20Hematol.&volume=94&pages=424-430&publication_year=2019&author=Sidana%2CS) 
    
86.  Neben, K. et al. Combining information regarding chromosomal aberrations t(4;14) and del(17p13) with the International Staging System classification allows stratification of myeloma patients undergoing autologous stem cell transplantation. _Haematologica_ **95**, 1150–1157 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlWjsLrE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20220069)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2895040)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combining%20information%20regarding%20chromosomal%20aberrations%20t%284%3B14%29%20and%20del%2817p13%29%20with%20the%20International%20Staging%20System%20classification%20allows%20stratification%20of%20myeloma%20patients%20undergoing%20autologous%20stem%20cell%20transplantation&journal=Haematologica&volume=95&pages=1150-1157&publication_year=2010&author=Neben%2CK) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02292-1?format=refman&flavour=references)

Acknowledgements
----------------

This work was supported by the Myeloma Solutions Fund, Paula and Rodger Riney Multiple Myeloma Research Program Fund, the Tow Foundation, Sylvester Comprehensive Cancer Center National Cancer Institute (NCI) Core Grant (P30 CA 240139), Memorial Sloan Kettering Cancer Center NCI Core Grant (P30 CA 008748) and NYU NCI Core Grant (P30CA016087). F.M. was supported by the Leukemia and Lymphoma Society (LLS), the International Myeloma Society (IMS), the Department of Defence and the National Institutes of Health (NIH)–NCI. G.J.M. received grant support through a Translational Research Program award from the LLS (6020-20). K.H.M. has received funding from the Multiple Myeloma Research Foundation (MMRF), the American Society of Hematology and the IMS. A.M.P. is funded by the Medical Data Scientist Program of Heidelberg University, Faculty of Medicine. N.W. is funded by the Advanced Medical Scientist Program of Heidelberg University, Faculty of Medicine. The Heidelberg Team thanks the Sample Processing Lab, the High-Throughput Sequencing Unit of the Genomics & Proteomics Core Facility and the Omics IT and Data Management Core Facility of the German Cancer Research Center (DKFZ), the DKFZ–Heidelberg Center for Personalized Oncology (DKFZ–HIPO) office, the Biobank Multiple Myeloma UKHD and the Myeloma Registry for excellent services. Support and funding of the project provided by the Dietmar Hopp Foundation and the NCT Heidelberg Molecular Precision Oncology Program (project K08K) is gratefully acknowledged. Data storage service provided by SDS@hd is supported by the Ministry of Science, Research and the Arts Baden-Württemberg (MWK) and the German Research Foundation (DFG) through grants INST 35/1314-1 FUGG and INST 35/1503-1 FUGG. L.B.B. is supported by NIH–NCI (R37CA272883).

Author information
------------------

Author notes

1.  These authors contributed equally: Francesco Maura, Marcella Kaddoura, Alexandra M. Poos, Marc S. Raab, Niels Weinhold.
    

### Authors and Affiliations

1.  Myeloma Institute, Sylvester Comprehensive Cancer Center, University of Miami, Miami, FL, USA
    
    Francesco Maura, Marcella Kaddoura, Bachisio Ziccheddu, Anthony Cirrincione, Monika Chojnacka, Benjamin Diamond, Marios Papadimitriou & Ola Landgren
    
2.  Myeloma Service, Department of Medicine, Memorial Sloan Kettering Cancer Center, New York City, NY, USA
    
    Francesco Maura, Kylee Maclachlan, Alexander M. Lesokhin, Neha Korde & Saad Usmani
    
3.  Heidelberg Myeloma Center, Department of Medicine V, University Hospital Heidelberg, Medical Faculty, Heidelberg University, Heidelberg, Germany
    
    Alexandra M. Poos, Marc-Andrea Bärtsch, Lukas John, Philipp Reichert, Stefanie Huhn, Elias K. Mai, Marc S. Raab & Niels Weinhold
    
4.  Clinical Cooperation Unit Molecular Hematology/Oncology, Department of Internal Medicine V, Heidelberg University Hospital, Heidelberg University and German Cancer Research Center (DKFZ), Heidelberg, Germany
    
    Alexandra M. Poos, Marc-Andrea Bärtsch, Lukas John, Marc S. Raab & Niels Weinhold
    
5.  Division of Hematopathology, Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN, USA
    
    Linda B. Baughn
    
6.  Myeloma Research Program, NYU Langone, Perlmutter Cancer Center, New York City, NY, USA
    
    Patrick Blaney, Dylan Gagler, Faith Davies & Gareth J. Morgan
    
7.  Cytogenetics Laboratory, Department of Pathology, Memorial Sloan Kettering Cancer Center, New York City, NY, USA
    
    Yanming Zhang
    
8.  Hematopathology Service, Department of Pathology, Memorial Sloan Kettering Cancer Center, New York City, NY, USA
    
    Ahmet Dogan
    
9.  Internal Medicine V, Hematology, Oncology and Rheumatology, GMMG Study Group, Heidelberg University Hospital and the National Center for Tumor Diseases (NCT), Heidelberg, Germany
    
    Hartmut Goldschmidt
    
10.  Department of Hematology, Oncology and Clinical Immunology, University Hospital Duesseldorf, Duesseldorf, Germany
    
    Roland Fenk
    
11.  Department of Oncology, Hematology, and Blood and Marrow Transplant, University Medical Center Hamburg—Eppendorf, Hamburg, Germany
    
    Katja C. Weisel
    
12.  Division of Hematology, Department of Internal Medicine, Mayo Clinic, Rochester, MN, USA
    
    S. Vincent Rajkumar & Shaji Kumar
    

Authors

1.  Francesco Maura
    
    [View author publications](https://www.nature.com/search?author=Francesco%20Maura)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Francesco%20Maura) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Francesco%20Maura%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Marcella Kaddoura
    
    [View author publications](https://www.nature.com/search?author=Marcella%20Kaddoura)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marcella%20Kaddoura) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marcella%20Kaddoura%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Alexandra M. Poos
    
    [View author publications](https://www.nature.com/search?author=Alexandra%20M.%20Poos)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alexandra%20M.%20Poos) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alexandra%20M.%20Poos%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Linda B. Baughn
    
    [View author publications](https://www.nature.com/search?author=Linda%20B.%20Baughn)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Linda%20B.%20Baughn) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Linda%20B.%20Baughn%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Bachisio Ziccheddu
    
    [View author publications](https://www.nature.com/search?author=Bachisio%20Ziccheddu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Bachisio%20Ziccheddu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Bachisio%20Ziccheddu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Marc-Andrea Bärtsch
    
    [View author publications](https://www.nature.com/search?author=Marc-Andrea%20B%C3%A4rtsch)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marc-Andrea%20B%C3%A4rtsch) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marc-Andrea%20B%C3%A4rtsch%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Anthony Cirrincione
    
    [View author publications](https://www.nature.com/search?author=Anthony%20Cirrincione)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anthony%20Cirrincione) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anthony%20Cirrincione%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Kylee Maclachlan
    
    [View author publications](https://www.nature.com/search?author=Kylee%20Maclachlan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kylee%20Maclachlan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kylee%20Maclachlan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Monika Chojnacka
    
    [View author publications](https://www.nature.com/search?author=Monika%20Chojnacka)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Monika%20Chojnacka) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Monika%20Chojnacka%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Benjamin Diamond
    
    [View author publications](https://www.nature.com/search?author=Benjamin%20Diamond)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Benjamin%20Diamond) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Benjamin%20Diamond%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Marios Papadimitriou
    
    [View author publications](https://www.nature.com/search?author=Marios%20Papadimitriou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marios%20Papadimitriou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marios%20Papadimitriou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Patrick Blaney
    
    [View author publications](https://www.nature.com/search?author=Patrick%20Blaney)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Patrick%20Blaney) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Patrick%20Blaney%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Lukas John
    
    [View author publications](https://www.nature.com/search?author=Lukas%20John)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lukas%20John) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lukas%20John%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Philipp Reichert
    
    [View author publications](https://www.nature.com/search?author=Philipp%20Reichert)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Philipp%20Reichert) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Philipp%20Reichert%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Stefanie Huhn
    
    [View author publications](https://www.nature.com/search?author=Stefanie%20Huhn)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Stefanie%20Huhn) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Stefanie%20Huhn%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Dylan Gagler
    
    [View author publications](https://www.nature.com/search?author=Dylan%20Gagler)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Dylan%20Gagler) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Dylan%20Gagler%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Yanming Zhang
    
    [View author publications](https://www.nature.com/search?author=Yanming%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yanming%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yanming%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Ahmet Dogan
    
    [View author publications](https://www.nature.com/search?author=Ahmet%20Dogan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ahmet%20Dogan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ahmet%20Dogan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Alexander M. Lesokhin
    
    [View author publications](https://www.nature.com/search?author=Alexander%20M.%20Lesokhin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alexander%20M.%20Lesokhin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alexander%20M.%20Lesokhin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Faith Davies
    
    [View author publications](https://www.nature.com/search?author=Faith%20Davies)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Faith%20Davies) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Faith%20Davies%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Hartmut Goldschmidt
    
    [View author publications](https://www.nature.com/search?author=Hartmut%20Goldschmidt)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hartmut%20Goldschmidt) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hartmut%20Goldschmidt%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Roland Fenk
    
    [View author publications](https://www.nature.com/search?author=Roland%20Fenk)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Roland%20Fenk) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Roland%20Fenk%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Katja C. Weisel
    
    [View author publications](https://www.nature.com/search?author=Katja%20C.%20Weisel)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Katja%20C.%20Weisel) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Katja%20C.%20Weisel%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Elias K. Mai
    
    [View author publications](https://www.nature.com/search?author=Elias%20K.%20Mai)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Elias%20K.%20Mai) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Elias%20K.%20Mai%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Neha Korde
    
    [View author publications](https://www.nature.com/search?author=Neha%20Korde)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Neha%20Korde) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Neha%20Korde%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Gareth J. Morgan
    
    [View author publications](https://www.nature.com/search?author=Gareth%20J.%20Morgan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gareth%20J.%20Morgan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gareth%20J.%20Morgan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  S. Vincent Rajkumar
    
    [View author publications](https://www.nature.com/search?author=S.%20Vincent%20Rajkumar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=S.%20Vincent%20Rajkumar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22S.%20Vincent%20Rajkumar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Shaji Kumar
    
    [View author publications](https://www.nature.com/search?author=Shaji%20Kumar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shaji%20Kumar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shaji%20Kumar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Saad Usmani
    
    [View author publications](https://www.nature.com/search?author=Saad%20Usmani)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Saad%20Usmani) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Saad%20Usmani%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  Ola Landgren
    
    [View author publications](https://www.nature.com/search?author=Ola%20Landgren)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ola%20Landgren) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ola%20Landgren%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  Marc S. Raab
    
    [View author publications](https://www.nature.com/search?author=Marc%20S.%20Raab)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marc%20S.%20Raab) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marc%20S.%20Raab%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
32.  Niels Weinhold
    
    [View author publications](https://www.nature.com/search?author=Niels%20Weinhold)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Niels%20Weinhold) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Niels%20Weinhold%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

F.M., M.S.R. and N.W. designed and supervised the study. F.M., M.S.R., N.W., M.K., A.M.P., K.M., M.C., B.D., P.B., L.J., P.R., S.H., M.A.B., D.G., Y.Z., F.D., G.M. and A.D. collected data. F.M., M.S.R., N.W., M.K. and A.M.P. generated the data and wrote the manuscript. F.M., M.S.R., N.W., M.K., A.M.P., B.Z., A.C. and M.P. analyzed the data. N.K., S.U., O.L., E.K.M., H.G., K.C.W., A.L. and R.F. supervised the clinical trials and collected clinical data. V.R., S.K. and L.B. collected and analyzed the FISH data from Mayo Clinic.

### Corresponding authors

Correspondence to [Francesco Maura](mailto:mauraf@mskcc.org) or [Niels Weinhold](mailto:niels.weinhold@med.uni-heidelberg.de).

Ethics declarations
-------------------

### Competing interests

O.L. has received research funding from NIH, NCI, U.S. Food and Drug Administration, MMRF, International Myeloma Foundation, LLS, Myeloma Solutions Fund, Paula and Rodger Riney Multiple Myeloma Research Program Fund, the Tow Foundation, Perelman Family Foundation, Rising Tide Foundation, Amgen, Celgene, Janssen, Takeda, Glenmark, Seattle Genetics, Karyopharm; Honoraria/ad boards: Adaptive, Amgen, Binding Site, Bristol Myers Squibb (BMS), Celgene, Cellectis, Glenmark, Janssen, Juno, Pfizer; and serves on Independent Data Monitoring Committees for clinical trials lead by Takeda, Merck, Janssen, Theradex. G.J.M. has received funding from NIH, NCI, MMRF, LLS, Perelman Family Foundation, Amgen, Celgene, Janssen and Takeda; has received honoraria or advisory board fees from Adaptive, Amgen, BMS, Celgene and Janssen; and serves on Independent Data Monitoring Committees for clinical trials led by Takeda, Karyopharm and Sanofi. E.K.M. reports consulting or advisory role, honoraria, research funding, travel accommodation and expenses from BMS (Celgene), GlaxoSmithKline (GSK), Janssen-Cilag, Oncopeptides, Pfizer, Sanofi, Stemline and Takeda. K.C.W. reports research grant from AbbVie, Amgen, BMS/Celgene, Janssen, GSK and Sanofi; and has received honoraria and consulting fees from AbbVie, Amgen, Adaptive Biotech, AstraZeneca, BMS/Celgene, BeiGene, Janssen, GSK, Karyopharm, Novartis, Oncopeptides, Pfizer, Roche Pharma, Sanofi, Stemline and Takeda. R.F. reports consulting or advisory role, honoraria, travel accommodation and expenses from Amgen, BMS (Celgene), GSK, Janssen-Cilag, Sanofi, Stemline and Takeda. K.H.S. has served on an advisory board for AbbVie, Amgen, BMS, GSK and Janssen; received honoraria for Adaptive Biotechnologies Corporation, Amgen, BMS, GSK, Janssen and Sanofi Genzyme; and has received research funding from AbbVie and Karyopharm Therapeutics. H.G. has received grants and/or provision of investigational medicinal product from BMS/Celgene, Dietmar-Hopp-Foundation, Janssen and Sanofi; research support from Amgen, BMS, Celgene, GlycoMimetics, GSK, Heidelberg Pharma, Hoffmann-La Roche, Janssen, Millenium Pfizer, Sanofi and Novartis; advisory board fees from BMS, GSK, Janssen and Sanofi; honoraria from Amgen, BMS, GSK, Janssen, Oncopeptides, Sanofi and Pfizer; and support for attending meetings and/or travel form Amgen, BMS, GSK, Janssen, Oncopeptides, Sanofi and Pfizer. N.K. has received research funding from Janssen and AbbVie, and has served on an advisory board for Janssen. K.H.M. has received funding from the Multiple Myeloma Research Foundation, the American Society of Hematology and the IMS. L.B.B. served as a consultant for Genentech. F.M. has received consulting fees from Medidata and Sanofi. B.D. reports consulting via an independent data review committee for Janssen. All other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02292-1#MOESM2) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Study cohort genomic landscape.](https://www.nature.com/articles/s41588-025-02292-1/figures/8)

The 12 newly diagnosed multiple myeloma (NDMM) genomic groups (ref. [33](https://www.nature.com/articles/s41588-025-02292-1#ref-CR33 "Maura, F. et al. Genomic classification and individualized prognosis in multiple myeloma. J. Clin. Oncol. 42, 1229–1240 (2024).")) and their distribution across the MSKCC and German cohorts included in this study. All defining features of the 12 groups are depicted. Heatmap colors: gray = wild type; red = monoallelic involvement; brown = biallelic involvement. Immunoglobulin heavy chain (IGH) canonical translocations are reported in brown. Presence of APOBEC and hyper-APOBEC is reported in red and brown, respectively. HY: hyperdiploid.

### [Extended Data Fig. 2 Impact of APOBEC on multiple myeloma clinical outcomes.](https://www.nature.com/articles/s41588-025-02292-1/figures/9)

**a**, Comparison of mutational burden between the MSKCC and German cohorts included in this study. The P value was calculated using the Wilcoxon test. SBS: single base substitution. **b**,**c**, Impact of Hyper-APOBEC presence on progression-free survival (PFS) and overall survival (OS) in newly diagnosed multiple myeloma patients.

### [Extended Data Fig. 3 Clock-like signatures in B and myeloma cells.](https://www.nature.com/articles/s41588-025-02292-1/figures/10)

**a**, Correlation between SBS1 and SBS5 mutational burden (clock-like signatures) and patients’ age at sample collection. Samples that did not fit into the linear model reported in Fig. [1b](https://www.nature.com/articles/s41588-025-02292-1#Fig1) are plotted with transparency (residual > 1,900). Three samples with >8,000 SBS1 and SBS5 mutations were excluded for graphical purpose (MM119, MM165, and RRMM48). **b**, Comparison of SBS1 and SBS5 clock mutation rates between multiple myeloma and B-cell lymphoma. **c**, Comparison of SBS1 and SBS5 clock mutation rate between multiple myeloma (red line) and normal memory B cells (black line). Given the single cell colony expansion in normal memory B-cell data (ref. [51](https://www.nature.com/articles/s41588-025-02292-1#ref-CR51 "Machado, H. E. et al. Diverse mutational landscapes in human lymphocytes. Nature 608, 724–732 (2022).")), only clonal variants were considered for multiple myeloma. SBS: single base substitution; NDMM: newly diagnosed multiple myeloma; RRMM: relapsed refractory multiple myeloma; Mel+ and Mel−: melphalan-exposed or not-exposed RRMM patients, respectively.

### [Extended Data Fig. 4 SBS mutational signature contribution in relapsed refractory multiple myeloma (RRMM).](https://www.nature.com/articles/s41588-025-02292-1/figures/11)

This figure illustrates the contribution of SBS mutational signatures in RRMM, distinguishing between clonal and subclonal variants. _P_ values were estimated using the Wilcoxon test.

### [Extended Data Fig. 5 APOBEC mutagenesis across time.](https://www.nature.com/articles/s41588-025-02292-1/figures/12)

**a**, Comparison of APOBEC contribution in clonal variants between hyper-APOBEC and non-hyper-APOBEC. **b**–**g**, SBS mutational signature contribution among clonal and subclonal variants in samples with hyper-APOBEC.

### [Extended Data Fig. 6 Temporal relationship of distinct chromosomal aberrations.](https://www.nature.com/articles/s41588-025-02292-1/figures/13)

**a**) Examples of molecular time estimates of HY patients with 1q gain. In the top case, the 1q was acquired at the same time as the HY multi-gain event. The middle case shows a 1q gain acquired at the same time as the HY multi-gain event, but later compared to the first case. The last case (bottom) has a 1q gain acquired independently after the HY multi-gain event. **b**–**d**) The molecular time estimates of chromosomal gains shared, selected or lost over time in patients with WGS data available at two different time points. **c**) Boxplot of estimated molecular time for gains of non-HY chromosomes (1, 2, 4, 6, 8, 10, 12, 13, 14, 16, 17, 18, 20, and 22) based on the absence or presence of canonical IGH translocations. **c**) Boxplot of estimated molecular time for gains of odd-numbered chromosomes typically seen in hyperdiploid myeloma (3, 5, 7, 9, 11, 15, 19, and 21) based on the presence (left box) or absence (right box) of canonical IGH translocations. **d**) Boxplot of estimated molecular time for LOH events in HY patients, divided into odd-numbered HY chromosomes (left box), and non-HY chromosomes (right box). Comparisons were performed using the Wilcoxon test.

### [Extended Data Fig. 7 Temporal patterns of chromosomal gains across the 12 genomic subgroups of multiple myeloma (ref.](https://www.nature.com/articles/s41588-025-02292-1/figures/14) [33](https://www.nature.com/articles/s41588-025-02292-1#ref-CR33 "Maura, F. et al. Genomic classification and individualized prognosis in multiple myeloma. J. Clin. Oncol. 42, 1229–1240 (2024).")).

**a**, Estimated molecular time of all chromosomal duplications across the 12 genomic subgroups. **b**, Estimated molecular timeof odd-numbered hyperdiploid chromosomal duplications across the 12 genomic subgroups, considering only patients with a hyperdiploid profile. **c**, Estimated molecular time of 1q gains across the 12 genomic subgroups. Adjusted _P_ values for comparisons in **a**–**c** are reported in Supplementary Table [8](https://www.nature.com/articles/s41588-025-02292-1#MOESM3).

### [Extended Data Fig. 8 Timing of focal deletions and gains at the MYC locus.](https://www.nature.com/articles/s41588-025-02292-1/figures/15)

**a**,**b**, A schematic illustration summarizing the workflow for distinguishing between pregain (**a**) and postgain (**b**) deletions, as well as preduplications (**c**) and postduplications (**d**). **e**,**f**, Two possible temporal scenarios in which an MYC translocation occurs either after (**e**) or before (**f**) large chromosomal gains.

### [Extended Data Fig. 9 Impact of timing of acquisition.](https://www.nature.com/articles/s41588-025-02292-1/figures/16)

**a**,**b**, Kaplan–Meier plots for progression-free survival (PFS; **a**) and overall survival (OS; **b**) for the following three groups: early 1q gain (molecular time ≤ 0.85), late 1q gain (molecular time > 0.85) and 1q amp. Compared to Fig. [6c,d](https://www.nature.com/articles/s41588-025-02292-1#Fig6), we show here the confidence intervals of the Kaplan–Meier curves.

### [Extended Data Fig. 10 Timing copy number gains with chemotherapy mutational signatures.](https://www.nature.com/articles/s41588-025-02292-1/figures/17)

Cartoon illustrating how the melphalan mutagenic mutational signature SBS99 can be used to estimate the timing of large chromosomal gains. If 1q gain was pre-existing, SBS99 would only be detected among the nonduplicated mutations on 1q. In contrast, if 1q gain was acquired after HDM–ASCT, SBS99 mutations on one allele would be duplicated and SBS99 would be detectable among the duplicated variants. The figure is created with [BioRender.com](http://biorender.com/).

Supplementary information
-------------------------

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM1_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM2_ESM.pdf)

### [Supplementary Tables 1–14](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM3_ESM.xlsx)

Supplementary Tables 1–14.

### [Supplementary Data 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM4_ESM.pdf)

Timing multiple myeloma evolution.

### [Supplementary Data 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM5_ESM.pdf)

Timing the first multi-gain events in multiple myeloma.

### [Supplementary Data 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM6_ESM.pdf)

Phylogenetic tree of multiple myeloma patients with WGS available at two different time points.

### [Supplementary Data 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM7_ESM.pdf)

Timing multiple myeloma evolution.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM8_ESM.txt)

Data used to generate Fig. 1a–c.

### [Source Data Figs. 2 and 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM9_ESM.xlsx)

Data used to generate Figs 2b–j and 3.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM10_ESM.txt)

Data used to generate Fig. 4.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM11_ESM.txt)

Data used to generate Fig. 5.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM12_ESM.txt)

Data used to generate Fig. 6e.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02292-1/MediaObjects/41588_2025_2292_MOESM13_ESM.txt)

Data used to generate Fig. 7.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Temporal%20genomic%20dynamics%20shape%20clinical%20trajectory%20in%20multiple%20myeloma&author=Francesco%20Maura%20et%20al&contentID=10.1038%2Fs41588-025-02292-1&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-20&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02292-1)

### Cite this article

Maura, F., Kaddoura, M., Poos, A.M. _et al._ Temporal genomic dynamics shape clinical trajectory in multiple myeloma. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02292-1

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02292-1?format=refman&flavour=citation)

-   Received: 29 January 2024
    
-   Accepted: 09 July 2025
    
-   Published: 20 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02292-1
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
