> id: https://www.nature.com/articles/s41588-025-02285-0

> link: https://www.nature.com/articles/s41588-025-02285-0

> title: Analysis of individual patient pathway coordination in a cross-species single-cell kidney atlas

# Analysis of individual patient pathway coordination in a cross-species single-cell kidney atlas
_Published on Thu, 07 Aug 2025 00:00:00 GMT_

Abstract
--------

The use of single-cell RNA sequencing in clinical and translational research is limited by the challenge of identifying cell-type-specific, targetable molecular changes in individual patients and cross-species differences. Here we created an integrated single-cell kidney atlas including over 1 million cells from 140 samples, defining more than 70 conserved cell states in human and rodent models. We developed CellSpectra, a computational tool that quantifies changes in gene expression coordination across cellular functions, which we applied to kidney and lung cancer data. This tool powers our patient-level single-cell functional profiling report, which highlights cell-type-specific changes in the coordination of pathway gene expression in individuals. Our cross-species atlas facilitates the selection of a rodent model that closely reflects the cellular and pathway-level signatures observed in patient samples, advancing the application of single-cell methodologies in clinical precision medicine. Finally, using experimental models, we demonstrate how our informatics approach can be applied for the potential selection of suitable therapeutics.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

27,99 € / 30 days

cancel any time

[Learn more](https://shop.nature.com/products/plus/?region=ROW)

Subscribe to this journal

Receive 12 print issues and online access

251,40 € per year

only 20,95 € per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02285-0?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Species-integrated single-cell kidney atlas (SISKA 1.0).**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_Fig1_HTML.png)

**Fig. 2: Exploration of single-cell atlases by estimating gene expression coordination of cellular functions across samples.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_Fig2_HTML.png)

**Fig. 3: Functional similarities and differences of rodent and human kidney cells in disease states.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_Fig3_HTML.png)

**Fig. 4: Gene expression changes of cellular functions at the sample level across species and disease states.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_Fig4_HTML.png)

**Fig. 5: Validation of targetable pathways in mouse kidney disease models.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_Fig5_HTML.png)

**Fig. 6: Single-cell functional profiling report (FPR) for patient samples.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_Fig6_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41597-023-02209-9/MediaObjects/41597_2023_2209_Fig1_HTML.png)

### [Meta-analysis of single-cell and single-nucleus transcriptomics reveals kidney cell type consensus signatures](https://www.nature.com/articles/s41597-023-02209-9?fromPaywallRec=true)

Article Open access 06 June 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41581-022-00553-4/MediaObjects/41581_2022_553_Fig1_HTML.png)

### [Mapping the human kidney using single-cell genomics](https://www.nature.com/articles/s41581-022-00553-4?fromPaywallRec=true)

Article 17 March 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs12276-020-0421-1/MediaObjects/12276_2020_421_Fig1_HTML.png)

### [Human Cell Atlas and cell-type authentication for regenerative medicine](https://www.nature.com/articles/s12276-020-0421-1?fromPaywallRec=true)

Article Open access 15 September 2020

Data availability
-----------------

Previously published data can be accessed via the Kidney Precision Medicine Project website [https://www.kpmp.org](https://www.kpmp.org/) for KPMP human snRNA-seq, and Gene Expression Omnibus accession codes [GSE211785](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211785) (Susztak lab human snRNA-seq), [GSE209821](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE209821) (ZSF1 rat snRNA-seq), [GSE183839](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183839) (deoxycorticosterone acetate rat snRNA-seq), [GSE184652](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE184652) (diabetic kidney disease mouse snRNA-seq) and [GSE139107](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139107) (mouse AKI time course snRNA-seq). Unpublished raw and processed snRNA-seq mouse data have been deposited in Gene Expression Omnibus with the accession code [GSE291551](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291551). Additional published input data used for the extended human atlas were accessed via CELLxGENE (diabetic kidney disease, ref. [55](https://www.nature.com/articles/s41588-025-02285-0#ref-CR55 "Wilson, P. C. et al. Multimodal single cell sequencing implicates chromatin accessibility and genetic background in diabetic kidney disease progression. Nat. Commun. 13, 5253 (2022).")), [GSE185948](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE185948) (ADPKD) and the KPMP website (KPMP unpublished). The lung cancer core atlas was downloaded from CELLxGENE. Unpublished human snRNA-seq data from biopsies have been deposited in GEO under the accession code [GSE291551](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291551). All five single-cell atlases (SISKA, human extension, human-, mouse- and rat-only) have been deposited in Zenodo[97](https://www.nature.com/articles/s41588-025-02285-0#ref-CR97 "Klötzer, K. A. et al. A cross-species single-cell kidney atlas: data repository. Zenodo 
https://doi.org/10.5281/zenodo.15007208
(2025)."). A user-friendly web tool for gene expression browsing of the multispecies atlas (SISKA) and _R_2 distributions of rodent models compared to humans is provided on our website ([www.susztaklab.com](http://www.susztaklab.com/)). Gene set databases were downloaded from the MSigDB website (C2 database), GO\_Biological\_Process\_2023 downloaded from the enrichR website, KEGG database (accessed via the R package KEGGREST) and the STRING website (9606.clusters.proteins.v12.0). The cross-species GO database was adapted from figshare[94](https://www.nature.com/articles/s41588-025-02285-0#ref-CR94 "Gillis, J. Protocol data (Python version). figshare 
https://doi.org/10.6084/m9.figshare.13034171.v1
(2020)."). Visium spatial transcriptomics datasets were accessed via [GSE211785](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211785) (human), the DNA Data Bank of Japan ([DRA012650](https://ddbj.nig.ac.jp/public/ddbj_database/dra/fastq/DRA012/DRA012650/)) and Genomic Expression Archive (E-GEAD-450) for rat data[36](https://www.nature.com/articles/s41588-025-02285-0#ref-CR36 "Onoda, N. et al. Spatial and single-cell transcriptome analysis reveals changes in gene expression in response to drug perturbation in rat kidney. DNA Res. 29, dsac007 (2022)."), and downloaded from [https://www.10xgenomics.com/datasets/adult-mouse-kidney-ffpe-1-standard-1-3-0](https://www.10xgenomics.com/datasets/adult-mouse-kidney-ffpe-1-standard-1-3-0) (mouse). [Source data](https://www.nature.com/articles/s41588-025-02285-0#Sec37) are provided with this paper.

Code availability
-----------------

Customized codes to reproduce the integration pipeline, figures and downstream analysis were deposited on a GitHub repository ([www.github.com/susztaklab/SISKA](http://www.github.com/susztaklab/SISKA); ref. [98](https://www.nature.com/articles/s41588-025-02285-0#ref-CR98 "KonstantinKltz. susztaklab/SISKA: pre-release V0.1. Zenodo 
https://doi.org/10.5281/zenodo.15109635
(2025).")). A detailed CellSpectra repository containing the R package and documentation is available via [https://github.com/kloetzerka/CellSpectra](https://github.com/kloetzerka/CellSpectra) (ref. [99](https://www.nature.com/articles/s41588-025-02285-0#ref-CR99 "KonstantinKltz. kloetzerka/CellSpectra: CellSpectra pre-release. Zenodo 
https://doi.org/10.5281/zenodo.15112987
(2025).")).

References
----------

1.  GBD Chronic Kidney Disease Collaboration Global, regional, and national burden of chronic kidney disease, 1990–2017: a systematic analysis for the Global Burden of Disease Study 2017. _Lancet_ **395**, 709–733 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%2C%20regional%2C%20and%20national%20burden%20of%20chronic%20kidney%20disease%2C%201990%E2%80%932017%3A%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202017&journal=Lancet&volume=395&pages=709-733&publication_year=2020) 
    
2.  Kidney disease: a global health priority. _Nat. Rev. Nephrol_. **20**, 421–423 (2024).
    
3.  Kalantar-Zadeh, K., Jafar, T. H., Nitsch, D., Neuen, B. L. & Perkovic, V. Chronic kidney disease. _Lancet_ **398**, 786–802 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34175022)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVent7vK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chronic%20kidney%20disease&journal=Lancet&volume=398&pages=786-802&publication_year=2021&author=Kalantar-Zadeh%2CK&author=Jafar%2CTH&author=Nitsch%2CD&author=Neuen%2CBL&author=Perkovic%2CV) 
    
4.  Muntner, P. Longitudinal measurements of renal function. _Semin. Nephrol._ **29**, 650–657 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20006797)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Longitudinal%20measurements%20of%20renal%20function&journal=Semin.%20Nephrol.&volume=29&pages=650-657&publication_year=2009&author=Muntner%2CP) 
    
5.  Zuk, A. & Bonventre, J. V. Acute kidney injury. _Annu. Rev. Med._ **67**, 293–307 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26768243)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4845743)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xps1Wgug%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Acute%20kidney%20injury&journal=Annu.%20Rev.%20Med.&volume=67&pages=293-307&publication_year=2016&author=Zuk%2CA&author=Bonventre%2CJV) 
    
6.  Sethi, S. et al. Mayo clinic/renal pathology society consensus report on pathologic classification, diagnosis, and reporting of GN. _J. Am. Soc. Nephrol._ **27**, 1278–1287 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26567243)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mayo%20clinic%2Frenal%20pathology%20society%20consensus%20report%20on%20pathologic%20classification%2C%20diagnosis%2C%20and%20reporting%20of%20GN&journal=J.%20Am.%20Soc.%20Nephrol.&volume=27&pages=1278-1287&publication_year=2016&author=Sethi%2CS) 
    
7.  Schreibing, F. & Kramann, R. Mapping the human kidney using single-cell genomics. _Nat. Rev. Nephrol._ **18**, 347–360 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35301441)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20the%20human%20kidney%20using%20single-cell%20genomics&journal=Nat.%20Rev.%20Nephrol.&volume=18&pages=347-360&publication_year=2022&author=Schreibing%2CF&author=Kramann%2CR) 
    
8.  Lake, B. B. et al. An atlas of healthy and injured cell states and niches in the human kidney. _Nature_ **619**, 585–594 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37468583)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10356613)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFSmtLnI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20atlas%20of%20healthy%20and%20injured%20cell%20states%20and%20niches%20in%20the%20human%20kidney&journal=Nature&volume=619&pages=585-594&publication_year=2023&author=Lake%2CBB) 
    
9.  Sikkema, L. et al. An integrated cell atlas of the lung in health and disease. _Nat. Med._ **29**, 1563–1577 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37291214)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10287567)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtF2itbrE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20cell%20atlas%20of%20the%20lung%20in%20health%20and%20disease&journal=Nat.%20Med.&volume=29&pages=1563-1577&publication_year=2023&author=Sikkema%2CL) 
    
10.  Rood, J. E., Maartens, A., Hupalowska, A., Teichmann, S. A. & Regev, A. Impact of the human cell atlas on medicine. _Nat. Med._ **28**, 2486–2496 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36482102)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVyltbbO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Impact%20of%20the%20human%20cell%20atlas%20on%20medicine&journal=Nat.%20Med.&volume=28&pages=2486-2496&publication_year=2022&author=Rood%2CJE&author=Maartens%2CA&author=Hupalowska%2CA&author=Teichmann%2CSA&author=Regev%2CA) 
    
11.  Robinson, N. B. et al. The current state of animal models in research: a review. _Int J. Surg._ **72**, 9–13 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31627013)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20current%20state%20of%20animal%20models%20in%20research%3A%20a%20review&journal=Int%20J.%20Surg.&volume=72&pages=9-13&publication_year=2019&author=Robinson%2CNB) 
    
12.  Zhou, J. et al. Unified mouse and human kidney single-cell expression atlas reveal commonalities and differences in disease states. _J. Am. Soc. Nephrol._ **34**, 1843–1862 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37639336)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10631616)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unified%20mouse%20and%20human%20kidney%20single-cell%20expression%20atlas%20reveal%20commonalities%20and%20differences%20in%20disease%20states&journal=J.%20Am.%20Soc.%20Nephrol.&volume=34&pages=1843-1862&publication_year=2023&author=Zhou%2CJ) 
    
13.  Luecken, M. D. & Theis, F. J. Current best practices in single-cell RNA-seq analysis: a tutorial. _Mol. Syst. Biol._ **15**, e8746 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31217225)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6582955)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Current%20best%20practices%20in%20single-cell%20RNA-seq%20analysis%3A%20a%20tutorial&journal=Mol.%20Syst.%20Biol.&volume=15&publication_year=2019&author=Luecken%2CMD&author=Theis%2CFJ) 
    
14.  Heumos, L. et al. Best practices for single-cell analysis across modalities. _Nat. Rev. Genet._ **24**, 550–572 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37002403)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmsl2nsbg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Best%20practices%20for%20single-cell%20analysis%20across%20modalities&journal=Nat.%20Rev.%20Genet.&volume=24&pages=550-572&publication_year=2023&author=Heumos%2CL) 
    
15.  Squair, J. W. et al. Confronting false discoveries in single-cell differential expression. _Nat. Commun._ **12**, 5692 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34584091)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8479118)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFamu7vI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Confronting%20false%20discoveries%20in%20single-cell%20differential%20expression&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Squair%2CJW) 
    
16.  Zhao, K. & Rhee, S. Y. Interpreting omics data with pathway enrichment analysis. _Trends Genet._ **39**, 308–319 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36750393)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVCmt74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interpreting%20omics%20data%20with%20pathway%20enrichment%20analysis&journal=Trends%20Genet.&volume=39&pages=308-319&publication_year=2023&author=Zhao%2CK&author=Rhee%2CSY) 
    
17.  Abedini, A. et al. Single-cell multi-omic and spatial profiling of human kidneys implicates the fibrotic microenvironment in kidney disease progression. _Nat. Genet._ **56**, 1712–1724 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39048792)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11592391)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhs1ajsbfL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20multi-omic%20and%20spatial%20profiling%20of%20human%20kidneys%20implicates%20the%20fibrotic%20microenvironment%20in%20kidney%20disease%20progression&journal=Nat.%20Genet.&volume=56&pages=1712-1724&publication_year=2024&author=Abedini%2CA) 
    
18.  Kirita, Y., Wu, H., Uchimura, K., Wilson, P. C. & Humphreys, B. D. Cell profiling of mouse acute kidney injury reveals conserved cellular responses to injury. _Proc. Natl Acad. Sci. USA_ **117**, 15874–15883 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32571916)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7355049)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFSrtL7N)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell%20profiling%20of%20mouse%20acute%20kidney%20injury%20reveals%20conserved%20cellular%20responses%20to%20injury&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=15874-15883&publication_year=2020&author=Kirita%2CY&author=Wu%2CH&author=Uchimura%2CK&author=Wilson%2CPC&author=Humphreys%2CBD) 
    
19.  Wu, H. et al. Mapping the single-cell transcriptomic response of murine diabetic kidney disease to therapies. _Cell Metab._ **34**, 1064–1078 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35709763)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9262852)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFansLnM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20the%20single-cell%20transcriptomic%20response%20of%20murine%20diabetic%20kidney%20disease%20to%20therapies&journal=Cell%20Metab.&volume=34&pages=1064-1078&publication_year=2022&author=Wu%2CH) 
    
20.  Abedini, A. et al. Single-cell transcriptomics and chromatin accessibility profiling elucidate the kidney protective mechanism of mineralocorticoid receptor antagonists. _J. Clin. Invest._ **134**, e157165 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37906287)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10760974)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhs1OmtLbI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20transcriptomics%20and%20chromatin%20accessibility%20profiling%20elucidate%20the%20kidney%20protective%20mechanism%20of%20mineralocorticoid%20receptor%20antagonists&journal=J.%20Clin.%20Invest.&volume=134&publication_year=2024&author=Abedini%2CA) 
    
21.  Balzer, M. S. et al. Treatment effects of soluble guanylate cyclase modulation on diabetic kidney disease at single-cell resolution. _Cell Rep. Med._ **4**, 100992 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37023747)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10140477)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Treatment%20effects%20of%20soluble%20guanylate%20cyclase%20modulation%20on%20diabetic%20kidney%20disease%20at%20single-cell%20resolution&journal=Cell%20Rep.%20Med.&volume=4&publication_year=2023&author=Balzer%2CMS) 
    
22.  Lopez, R., Regier, J., Cole, M. B., Jordan, M. I. & Yosef, N. Deep generative modeling for single-cell transcriptomics. _Nat. Methods_ **15**, 1053–1058 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30504886)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6289068)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2hs77J)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20generative%20modeling%20for%20single-cell%20transcriptomics&journal=Nat.%20Methods&volume=15&pages=1053-1058&publication_year=2018&author=Lopez%2CR&author=Regier%2CJ&author=Cole%2CMB&author=Jordan%2CMI&author=Yosef%2CN) 
    
23.  Fischer, S. & Gillis, J. How many markers are needed to robustly determine a cell’s type? _iScience_ **24**, 103292 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34765918)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8571500)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XmtVWrtro%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20many%20markers%20are%20needed%20to%20robustly%20determine%20a%20cell%E2%80%99s%20type%3F&journal=iScience&volume=24&publication_year=2021&author=Fischer%2CS&author=Gillis%2CJ) 
    
24.  Chen, Y. et al. Structural basis of ALDH1A2 inhibition by irreversible and reversible small molecule inhibitors. _ACS Chem. Biol._ **13**, 582–590 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29240402)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6089219)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFOgt7bK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20ALDH1A2%20inhibition%20by%20irreversible%20and%20reversible%20small%20molecule%20inhibitors&journal=ACS%20Chem.%20Biol.&volume=13&pages=582-590&publication_year=2018&author=Chen%2CY) 
    
25.  Chen, A., Liu, Y., Lu, Y., Lee, K. & He, J. C. Disparate roles of retinoid acid signaling molecules in kidney disease. _Am. J. Physiol. Ren. Physiol._ **320**, F683–F692 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlCksL3P)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disparate%20roles%20of%20retinoid%20acid%20signaling%20molecules%20in%20kidney%20disease&journal=Am.%20J.%20Physiol.%20Ren.%20Physiol.&volume=320&pages=F683-F692&publication_year=2021&author=Chen%2CA&author=Liu%2CY&author=Lu%2CY&author=Lee%2CK&author=He%2CJC) 
    
26.  Uhlen, M. et al. Towards a knowledge-based Human Protein Atlas. _Nat. Biotechnol._ **28**, 1248–1250 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21139605)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFals7nO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Towards%20a%20knowledge-based%20Human%20Protein%20Atlas&journal=Nat.%20Biotechnol.&volume=28&pages=1248-1250&publication_year=2010&author=Uhlen%2CM) 
    
27.  Safizadeh Shabestari, S. A. et al. Overlapping pathogenic de novo CNVs in neurodevelopmental disorders and congenital anomalies impacting constraint genes regulating early development. _Hum. Genet._ **142**, 1201–1213 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36383254)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivV2ltLnO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overlapping%20pathogenic%20de%20novo%20CNVs%20in%20neurodevelopmental%20disorders%20and%20congenital%20anomalies%20impacting%20constraint%20genes%20regulating%20early%20development&journal=Hum.%20Genet.&volume=142&pages=1201-1213&publication_year=2023&author=Safizadeh%20Shabestari%2CSA) 
    
28.  Shi, W., Le, W., Tang, Q., Shi, S. & Shi, J. Regulon analysis identifies protective FXR and CREB5 in proximal tubules in early diabetic kidney disease. _BMC Nephrol._ **24**, 180 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37337149)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10278299)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlGgtLrI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulon%20analysis%20identifies%20protective%20FXR%20and%20CREB5%20in%20proximal%20tubules%20in%20early%20diabetic%20kidney%20disease&journal=BMC%20Nephrol.&volume=24&publication_year=2023&author=Shi%2CW&author=Le%2CW&author=Tang%2CQ&author=Shi%2CS&author=Shi%2CJ) 
    
29.  Crow, M., Paul, A., Ballouz, S., Huang, Z. J. & Gillis, J. Characterizing the replicability of cell types defined by single cell RNA-sequencing data using MetaNeighbor. _Nat. Commun._ **9**, 884 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29491377)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5830442)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterizing%20the%20replicability%20of%20cell%20types%20defined%20by%20single%20cell%20RNA-sequencing%20data%20using%20MetaNeighbor&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Crow%2CM&author=Paul%2CA&author=Ballouz%2CS&author=Huang%2CZJ&author=Gillis%2CJ) 
    
30.  Kiselev, V. Y., Andrews, T. S. & Hemberg, M. Challenges in unsupervised clustering of single-cell RNA-seq data. _Nat. Rev. Genet._ **20**, 273–282 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30617341)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFSit7s%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Challenges%20in%20unsupervised%20clustering%20of%20single-cell%20RNA-seq%20data&journal=Nat.%20Rev.%20Genet.&volume=20&pages=273-282&publication_year=2019&author=Kiselev%2CVY&author=Andrews%2CTS&author=Hemberg%2CM) 
    
31.  Grabski, I. N., Street, K. & Irizarry, R. A. Significance analysis for clustering with single-cell RNA-sequencing data. _Nat. Methods_ **20**, 1196–1202 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37429993)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11282907)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVGrt7vF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Significance%20analysis%20for%20clustering%20with%20single-cell%20RNA-sequencing%20data&journal=Nat.%20Methods&volume=20&pages=1196-1202&publication_year=2023&author=Grabski%2CIN&author=Street%2CK&author=Irizarry%2CRA) 
    
32.  Hill, R. Z. et al. Renal mechanotransduction is an essential regulator of renin. Preprint at _bioRxiv_ [https://doi.org/10.1101/2023.11.04.565646](https://doi.org/10.1101/2023.11.04.565646) (2023).
    
33.  Tefft, J. B. et al. Notch1 and Notch3 coordinate for pericyte-induced stabilization of vasculature. _Am. J. Physiol. Cell Physiol._ **322**, C185–C196 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34878922)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFGktr%2FJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Notch1%20and%20Notch3%20coordinate%20for%20pericyte-induced%20stabilization%20of%20vasculature&journal=Am.%20J.%20Physiol.%20Cell%20Physiol.&volume=322&pages=C185-C196&publication_year=2022&author=Tefft%2CJB) 
    
34.  Kuppe, C. et al. Decoding myofibroblast origins in human kidney fibrosis. _Nature_ **589**, 281–286 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33176333)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsFSqsw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Decoding%20myofibroblast%20origins%20in%20human%20kidney%20fibrosis&journal=Nature&volume=589&pages=281-286&publication_year=2021&author=Kuppe%2CC) 
    
35.  Ashburner, M. et al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium _Nat. Genet._ **25**, 25–29 (2000).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10802651)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037419)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXjtFSlsbc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium&journal=Nat.%20Genet.&volume=25&pages=25-29&publication_year=2000&author=Ashburner%2CM) 
    
36.  Onoda, N. et al. Spatial and single-cell transcriptome analysis reveals changes in gene expression in response to drug perturbation in rat kidney. _DNA Res._ **29**, dsac007 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35325072)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9014450)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatial%20and%20single-cell%20transcriptome%20analysis%20reveals%20changes%20in%20gene%20expression%20in%20response%20to%20drug%20perturbation%20in%20rat%20kidney&journal=DNA%20Res.&volume=29&publication_year=2022&author=Onoda%2CN) 
    
37.  Su, H., Lei, C. T. & Zhang, C. Interleukin-6 signaling pathway and its role in kidney disease: an update. _Front. Immunol._ **8**, 405 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28484449)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5399081)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interleukin-6%20signaling%20pathway%20and%20its%20role%20in%20kidney%20disease%3A%20an%20update&journal=Front.%20Immunol.&volume=8&publication_year=2017&author=Su%2CH&author=Lei%2CCT&author=Zhang%2CC) 
    
38.  Meier, M., Menne, J. & Haller, H. Targeting the protein kinase C family in the diabetic kidney: lessons from analysis of mutant mice. _Diabetologia_ **52**, 765–775 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19238353)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXktVOrs7g%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20the%20protein%20kinase%20C%20family%20in%20the%20diabetic%20kidney%3A%20lessons%20from%20analysis%20of%20mutant%20mice&journal=Diabetologia&volume=52&pages=765-775&publication_year=2009&author=Meier%2CM&author=Menne%2CJ&author=Haller%2CH) 
    
39.  Hewitson, T. D. & Smith, E. R. A metabolic reprogramming of glycolysis and glutamine metabolism is a requisite for renal fibrogenesis—why and how? _Front. Physiol._ **12**, 645857 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33815149)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8010236)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20metabolic%20reprogramming%20of%20glycolysis%20and%20glutamine%20metabolism%20is%20a%20requisite%20for%20renal%20fibrogenesis%E2%80%94why%20and%20how%3F&journal=Front.%20Physiol.&volume=12&publication_year=2021&author=Hewitson%2CTD&author=Smith%2CER) 
    
40.  Kanehisa, M., Furumichi, M., Sato, Y., Kawashima, M. & Ishiguro-Watanabe, M. KEGG for taxonomy-based analysis of pathways and genomes. _Nucleic Acids Res._ **51**, D587–D592 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36300620)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1entLzL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KEGG%20for%20taxonomy-based%20analysis%20of%20pathways%20and%20genomes&journal=Nucleic%20Acids%20Res.&volume=51&pages=D587-D592&publication_year=2023&author=Kanehisa%2CM&author=Furumichi%2CM&author=Sato%2CY&author=Kawashima%2CM&author=Ishiguro-Watanabe%2CM) 
    
41.  Bonventre, J. V. & Yang, L. Cellular pathophysiology of ischemic acute kidney injury. _J. Clin. Investig._ **121**, 4210–4221 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22045571)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3204829)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsVCksL%2FI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cellular%20pathophysiology%20of%20ischemic%20acute%20kidney%20injury&journal=J.%20Clin.%20Investig.&volume=121&pages=4210-4221&publication_year=2011&author=Bonventre%2CJV&author=Yang%2CL) 
    
42.  Yan, L. J. Folic acid-induced animal model of kidney disease. _Anim. Model Exp. Med._ **4**, 329–342 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFChtL8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Folic%20acid-induced%20animal%20model%20of%20kidney%20disease&journal=Anim.%20Model%20Exp.%20Med.&volume=4&pages=329-342&publication_year=2021&author=Yan%2CLJ) 
    
43.  Mootha, V. K. et al. PGC-1α-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. _Nat. Genet._ **34**, 267–273 (2003).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12808457)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXkvFSrs7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PGC-1%CE%B1-responsive%20genes%20involved%20in%20oxidative%20phosphorylation%20are%20coordinately%20downregulated%20in%20human%20diabetes&journal=Nat.%20Genet.&volume=34&pages=267-273&publication_year=2003&author=Mootha%2CVK) 
    
44.  Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. _Proc. Natl Acad. Sci. USA_ **102**, 15545–15550 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16199517)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=102&pages=15545-15550&publication_year=2005&author=Subramanian%2CA) 
    
45.  Cao, H. et al. Tuberous sclerosis 1 (Tsc1) mediated mTORC1 activation promotes glycolysis in tubular epithelial cells in kidney fibrosis. _Kidney Int._ **98**, 686–698 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32739207)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFehsr7O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tuberous%20sclerosis%201%20%28Tsc1%29%20mediated%20mTORC1%20activation%20promotes%20glycolysis%20in%20tubular%20epithelial%20cells%20in%20kidney%20fibrosis&journal=Kidney%20Int.&volume=98&pages=686-698&publication_year=2020&author=Cao%2CH) 
    
46.  Salcher, S. et al. High-resolution single-cell atlas reveals diversity and plasticity of tissue-resident neutrophils in non-small cell lung cancer. _Cancer Cell_ **40**, 1503–1520 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36368318)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9767679)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivVGjurvE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20single-cell%20atlas%20reveals%20diversity%20and%20plasticity%20of%20tissue-resident%20neutrophils%20in%20non-small%20cell%20lung%20cancer&journal=Cancer%20Cell&volume=40&pages=1503-1520&publication_year=2022&author=Salcher%2CS) 
    
47.  Lau, S. C. M., Pan, Y., Velcheti, V. & Wong, K. K. Squamous cell lung cancer: current landscape and future therapeutic options. _Cancer Cell_ **40**, 1279–1293 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36270277)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1Oqu7zF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Squamous%20cell%20lung%20cancer%3A%20current%20landscape%20and%20future%20therapeutic%20options&journal=Cancer%20Cell&volume=40&pages=1279-1293&publication_year=2022&author=Lau%2CSCM&author=Pan%2CY&author=Velcheti%2CV&author=Wong%2CKK) 
    
48.  Mullen, N. J. & Singh, P. K. Nucleotide metabolism: a pan-cancer metabolic dependency. _Nat. Rev. Cancer_ **23**, 275–294 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36973407)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10041518)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmt1ajsbw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nucleotide%20metabolism%3A%20a%20pan-cancer%20metabolic%20dependency&journal=Nat.%20Rev.%20Cancer&volume=23&pages=275-294&publication_year=2023&author=Mullen%2CNJ&author=Singh%2CPK) 
    
49.  Friedlaender, A., Drilon, A., Banna, G. L., Peters, S. & Addeo, A. The METeoric rise of MET in lung cancer. _Cancer_ **126**, 4826–4837 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32888330)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFehsL7E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20METeoric%20rise%20of%20MET%20in%20lung%20cancer&journal=Cancer&volume=126&pages=4826-4837&publication_year=2020&author=Friedlaender%2CA&author=Drilon%2CA&author=Banna%2CGL&author=Peters%2CS&author=Addeo%2CA) 
    
50.  Herbst, R. S., Morgensztern, D. & Boshoff, C. The biology and management of non-small cell lung cancer. _Nature_ **553**, 446–454 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29364287)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs12itbo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20biology%20and%20management%20of%20non-small%20cell%20lung%20cancer&journal=Nature&volume=553&pages=446-454&publication_year=2018&author=Herbst%2CRS&author=Morgensztern%2CD&author=Boshoff%2CC) 
    
51.  Bansal, A. & Simon, M. C. Glutathione metabolism in cancer progression and treatment resistance. _J. Cell Biol._ **217**, 2291–2298 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29915025)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6028537)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFGntb3N)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Glutathione%20metabolism%20in%20cancer%20progression%20and%20treatment%20resistance&journal=J.%20Cell%20Biol.&volume=217&pages=2291-2298&publication_year=2018&author=Bansal%2CA&author=Simon%2CMC) 
    
52.  Hayes, J. D. & Ashford, M. L. Nrf2 orchestrates fuel partitioning for cell proliferation. _Cell Metab._ **16**, 139–141 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22883227)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFKit77J)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nrf2%20orchestrates%20fuel%20partitioning%20for%20cell%20proliferation&journal=Cell%20Metab.&volume=16&pages=139-141&publication_year=2012&author=Hayes%2CJD&author=Ashford%2CML) 
    
53.  Leitner, B. P. et al. Multimodal analysis suggests differential immuno-metabolic crosstalk in lung squamous cell carcinoma and adenocarcinoma. _npj Precis. Oncol._ **6**, 8 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35087143)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8795406)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtVWkurnE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multimodal%20analysis%20suggests%20differential%20immuno-metabolic%20crosstalk%20in%20lung%20squamous%20cell%20carcinoma%20and%20adenocarcinoma&journal=npj%20Precis.%20Oncol.&volume=6&publication_year=2022&author=Leitner%2CBP) 
    
54.  Muto, Y. et al. Defining cellular complexity in human autosomal dominant polycystic kidney disease by multimodal single cell analysis. _Nat. Commun._ **13**, 6497 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36310237)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9618568)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivVWru73O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defining%20cellular%20complexity%20in%20human%20autosomal%20dominant%20polycystic%20kidney%20disease%20by%20multimodal%20single%20cell%20analysis&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Muto%2CY) 
    
55.  Wilson, P. C. et al. Multimodal single cell sequencing implicates chromatin accessibility and genetic background in diabetic kidney disease progression. _Nat. Commun._ **13**, 5253 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36068241)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9448792)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlGmu7rN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multimodal%20single%20cell%20sequencing%20implicates%20chromatin%20accessibility%20and%20genetic%20background%20in%20diabetic%20kidney%20disease%20progression&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Wilson%2CPC) 
    
56.  Liu, H. et al. Kidney multiome-based genetic scorecard reveals convergent coding and regulatory variants. _Science_ **387**, eadp4753 (2025).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39913582)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12013656)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXjs1aqtLw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Kidney%20multiome-based%20genetic%20scorecard%20reveals%20convergent%20coding%20and%20regulatory%20variants&journal=Science&volume=387&publication_year=2025&author=Liu%2CH) 
    
57.  Xu, C. et al. Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models. _Mol. Syst. Biol._ **17**, e9620 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33491336)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7829634)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Probabilistic%20harmonization%20and%20annotation%20of%20single-cell%20transcriptomics%20data%20with%20deep%20generative%20models&journal=Mol.%20Syst.%20Biol.&volume=17&publication_year=2021&author=Xu%2CC) 
    
58.  Lai, K. N. et al. IgA nephropathy. _Nat. Rev. Dis. Prim._ **2**, 16001 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27189177)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IgA%20nephropathy&journal=Nat.%20Rev.%20Dis.%20Prim.&volume=2&publication_year=2016&author=Lai%2CKN) 
    
59.  Yang, X. et al. Genome-wide linkage and regional association study of blood pressure response to the cold pressor test in Han Chinese: the genetic epidemiology network of salt sensitivity study. _Circ. Cardiovasc. Genet._ **7**, 521–528 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25028485)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264620)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtlOiurjO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20linkage%20and%20regional%20association%20study%20of%20blood%20pressure%20response%20to%20the%20cold%20pressor%20test%20in%20Han%20Chinese%3A%20the%20genetic%20epidemiology%20network%20of%20salt%20sensitivity%20study&journal=Circ.%20Cardiovasc.%20Genet.&volume=7&pages=521-528&publication_year=2014&author=Yang%2CX) 
    
60.  Hanukoglu, I. & Hanukoglu, A. Epithelial sodium channel (ENaC) family: phylogeny, structure-function, tissue distribution, and associated inherited diseases. _Gene_ **579**, 95–132 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26772908)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4756657)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xlt1ChtQ%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epithelial%20sodium%20channel%20%28ENaC%29%20family%3A%20phylogeny%2C%20structure-function%2C%20tissue%20distribution%2C%20and%20associated%20inherited%20diseases&journal=Gene&volume=579&pages=95-132&publication_year=2016&author=Hanukoglu%2CI&author=Hanukoglu%2CA) 
    
61.  Crowley, S. D. & Coffman, T. M. The inextricable role of the kidney in hypertension. _J. Clin. Invest._ **124**, 2341–2347 (2014).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4092877)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXpvFaitbY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20inextricable%20role%20of%20the%20kidney%20in%20hypertension&journal=J.%20Clin.%20Invest.&volume=124&pages=2341-2347&publication_year=2014&author=Crowley%2CSD&author=Coffman%2CTM) 
    
62.  Russo, C. J. et al. Association of NEDD4L ubiquitin ligase with essential hypertension. _Hypertension_ **46**, 488–491 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16103266)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXptVWkurk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20of%20NEDD4L%20ubiquitin%20ligase%20with%20essential%20hypertension&journal=Hypertension&volume=46&pages=488-491&publication_year=2005&author=Russo%2CCJ) 
    
63.  Persu, A. & Devuyst, O. Transepithelial chloride secretion and cystogenesis in autosomal dominant polycystic kidney disease. _Nephrol. Dial. Transplant._ **15**, 747–750 (2000).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10831619)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD3c3ptleiuw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transepithelial%20chloride%20secretion%20and%20cystogenesis%20in%20autosomal%20dominant%20polycystic%20kidney%20disease&journal=Nephrol.%20Dial.%20Transplant.&volume=15&pages=747-750&publication_year=2000&author=Persu%2CA&author=Devuyst%2CO) 
    
64.  Müller, R. U. et al. An update on the use of tolvaptan for autosomal dominant polycystic kidney disease: consensus statement on behalf of the ERA Working Group on Inherited Kidney Disorders, the European Rare Kidney Disease Reference Network and Polycystic Kidney Disease International. _Nephrol. Dial. Transplant._ **37**, 825–839 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35134221)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20update%20on%20the%20use%20of%20tolvaptan%20for%20autosomal%20dominant%20polycystic%20kidney%20disease%3A%20consensus%20statement%20on%20behalf%20of%20the%20ERA%20Working%20Group%20on%20Inherited%20Kidney%20Disorders%2C%20the%20European%20Rare%20Kidney%20Disease%20Reference%20Network%20and%20Polycystic%20Kidney%20Disease%20International&journal=Nephrol.%20Dial.%20Transplant.&volume=37&pages=825-839&publication_year=2022&author=M%C3%BCller%2CRU) 
    
65.  Jardine, M. J., Liyanage, T., Buxton, E. & Perkovic, V. mTOR inhibition in autosomal-dominant polycystic kidney disease (ADPKD): the question remains open. _Nephrol. Dial. Transplant._ **28**, 242–244 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23222536)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=mTOR%20inhibition%20in%20autosomal-dominant%20polycystic%20kidney%20disease%20%28ADPKD%29%3A%20the%20question%20remains%20open&journal=Nephrol.%20Dial.%20Transplant.&volume=28&pages=242-244&publication_year=2012&author=Jardine%2CMJ&author=Liyanage%2CT&author=Buxton%2CE&author=Perkovic%2CV) 
    
66.  Li, X. et al. A tumor necrosis factor-α-mediated pathway promoting autosomal dominant polycystic kidney disease. _Nat. Med._ **14**, 863–868 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18552856)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3359869)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlKltL0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20tumor%20necrosis%20factor-%CE%B1-mediated%20pathway%20promoting%20autosomal%20dominant%20polycystic%20kidney%20disease&journal=Nat.%20Med.&volume=14&pages=863-868&publication_year=2008&author=Li%2CX) 
    
67.  Song, Y., Miao, Z., Brazma, A. & Papatheodorou, I. Benchmarking strategies for cross-species integration of single-cell RNA sequencing data. _Nat. Commun._ **14**, 6495 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37838716)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10576752)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFejtrrO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Benchmarking%20strategies%20for%20cross-species%20integration%20of%20single-cell%20RNA%20sequencing%20data&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Song%2CY&author=Miao%2CZ&author=Brazma%2CA&author=Papatheodorou%2CI) 
    
68.  Rosen, Y. Towards universal cell embeddings: integrating single-cell RNA-seq datasets across species with SATURN. _Nat. Methods_ **21**, 1492–1500 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38366243)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11310084)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXjvVaksb0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Towards%20universal%20cell%20embeddings%3A%20integrating%20single-cell%20RNA-seq%20datasets%20across%20species%20with%20SATURN&journal=Nat.%20Methods&volume=21&pages=1492-1500&publication_year=2024&author=Rosen%2CY) 
    
69.  Swamy, V. S., Fufa, T. D., Hufnagel, R. B. & McGaughey, D. M. Building the mega single-cell transcriptome ocular meta-atlas. _GigaScience_ **10**, giab061 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34651173)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8514335)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Building%20the%20mega%20single-cell%20transcriptome%20ocular%20meta-atlas&journal=GigaScience&volume=10&publication_year=2021&author=Swamy%2CVS&author=Fufa%2CTD&author=Hufnagel%2CRB&author=McGaughey%2CDM) 
    
70.  Bakken, T. E. et al. Comparative cellular analysis of motor cortex in human, marmoset and mouse. _Nature_ **598**, 111–119 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34616062)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8494640)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1WqsbjM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparative%20cellular%20analysis%20of%20motor%20cortex%20in%20human%2C%20marmoset%20and%20mouse&journal=Nature&volume=598&pages=111-119&publication_year=2021&author=Bakken%2CTE) 
    
71.  Li, H. et al. Cross-species single-cell transcriptomic analysis reveals divergence of cell composition and functions in mammalian ileum epithelium. _Cell Regen._ **11**, 19 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35511361)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9072607)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1agtLnJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-species%20single-cell%20transcriptomic%20analysis%20reveals%20divergence%20of%20cell%20composition%20and%20functions%20in%20mammalian%20ileum%20epithelium&journal=Cell%20Regen.&volume=11&publication_year=2022&author=Li%2CH) 
    
72.  Chen, D. et al. Single cell atlas for 11 non-model mammals, reptiles and birds. _Nat. Commun._ **12**, 7083 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34873160)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8648889)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1Oru77M)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single%20cell%20atlas%20for%2011%20non-model%20mammals%2C%20reptiles%20and%20birds&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Chen%2CD) 
    
73.  Song, Y., Hu, Y., Dow, J., Perrimon, N. & Papatheodorou, I. ScGOclust: leveraging gene ontology to find functionally analogous cell types between distant species. _Bioinformatics_ **41**, i571–i579 (2025).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=40662793)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12261464)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhs1SnsLbI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ScGOclust%3A%20leveraging%20gene%20ontology%20to%20find%20functionally%20analogous%20cell%20types%20between%20distant%20species&journal=Bioinformatics&volume=41&pages=i571-i579&publication_year=2025&author=Song%2CY&author=Hu%2CY&author=Dow%2CJ&author=Perrimon%2CN&author=Papatheodorou%2CI) 
    
74.  Andreatta, M. & Carmona, S. J. UCell: robust and scalable single-cell gene signature scoring. _Comput. Struct. Biotechnol. J._ **19**, 3796–3798 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34285779)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8271111)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisF2gsr%2FF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCell%3A%20robust%20and%20scalable%20single-cell%20gene%20signature%20scoring&journal=Comput.%20Struct.%20Biotechnol.%20J.&volume=19&pages=3796-3798&publication_year=2021&author=Andreatta%2CM&author=Carmona%2CSJ) 
    
75.  Noureen, N., Ye, Z., Chen, Y., Wang, X. & Zheng, S. Signature-scoring methods developed for bulk samples are not adequate for cancer single-cell RNA sequencing data. _eLife_ **11**, e71994 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35212622)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8916770)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvF2hu7jI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Signature-scoring%20methods%20developed%20for%20bulk%20samples%20are%20not%20adequate%20for%20cancer%20single-cell%20RNA%20sequencing%20data&journal=eLife&volume=11&publication_year=2022&author=Noureen%2CN&author=Ye%2CZ&author=Chen%2CY&author=Wang%2CX&author=Zheng%2CS) 
    
76.  Tomfohr, J., Lu, J. & Kepler, T. B. Pathway level analysis of gene expression using singular value decomposition. _BMC Bioinformatics_ **6**, 225 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16156896)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1261155)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathway%20level%20analysis%20of%20gene%20expression%20using%20singular%20value%20decomposition&journal=BMC%20Bioinformatics&volume=6&publication_year=2005&author=Tomfohr%2CJ&author=Lu%2CJ&author=Kepler%2CTB) 
    
77.  Ozerov, I. V. et al. In silico Pathway Activation Network Decomposition Analysis (iPANDA) as a method for biomarker development. _Nat. Commun._ **7**, 13427 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27848968)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5116087)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVygu77M)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In%20silico%20Pathway%20Activation%20Network%20Decomposition%20Analysis%20%28iPANDA%29%20as%20a%20method%20for%20biomarker%20development&journal=Nat.%20Commun.&volume=7&publication_year=2016&author=Ozerov%2CIV) 
    
78.  Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. _Nature_ **462**, 108–112 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19847166)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2783335)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht12gtrvL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&journal=Nature&volume=462&pages=108-112&publication_year=2009&author=Barbie%2CDA) 
    
79.  Levy, O. et al. Age-related loss of gene-to-gene transcriptional coordination among single cells. _Nat. Metab._ **2**, 1305–1315 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33139959)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlCmtb3E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Age-related%20loss%20of%20gene-to-gene%20transcriptional%20coordination%20among%20single%20cells&journal=Nat.%20Metab.&volume=2&pages=1305-1315&publication_year=2020&author=Levy%2CO) 
    
80.  Yu, T. & Bai, Y. Capturing changes in gene expression dynamics by gene set differential coordination analysis. _Genomics_ **98**, 469–477 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21971296)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsFWqsLrM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Capturing%20changes%20in%20gene%20expression%20dynamics%20by%20gene%20set%20differential%20coordination%20analysis&journal=Genomics&volume=98&pages=469-477&publication_year=2011&author=Yu%2CT&author=Bai%2CY) 
    
81.  Leote, A. C., Lopes, F. & Beyer, A. Loss of coordination between basic cellular processes in human aging. _Nat. Aging_ **4**, 1432–1445 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39227753)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11485205)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhvFalsbfK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20coordination%20between%20basic%20cellular%20processes%20in%20human%20aging&journal=Nat.%20Aging&volume=4&pages=1432-1445&publication_year=2024&author=Leote%2CAC&author=Lopes%2CF&author=Beyer%2CA) 
    
82.  Fischer, S., Crow, M., Harris, B. D. & Gillis, J. Scaling up reproducible research for single-cell transcriptomics using MetaNeighbor. _Nat. Protoc._ **16**, 4031–4067 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234317)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8826496)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSnsbzM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scaling%20up%20reproducible%20research%20for%20single-cell%20transcriptomics%20using%20MetaNeighbor&journal=Nat.%20Protoc.&volume=16&pages=4031-4067&publication_year=2021&author=Fischer%2CS&author=Crow%2CM&author=Harris%2CBD&author=Gillis%2CJ) 
    
83.  Maan, H. et al. Characterizing the impacts of dataset imbalance on single-cell data integration. _Nat. Biotechnol._ **42**, 1899–1908 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38429430)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXkvFGltLk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterizing%20the%20impacts%20of%20dataset%20imbalance%20on%20single-cell%20data%20integration&journal=Nat.%20Biotechnol.&volume=42&pages=1899-1908&publication_year=2024&author=Maan%2CH) 
    
84.  Beckerman, P. et al. Transgenic expression of human APOL1 risk variants in podocytes induces kidney disease in mice. _Nat. Med._ **23**, 429–438 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28218918)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5603285)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjtVeqtro%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transgenic%20expression%20of%20human%20APOL1%20risk%20variants%20in%20podocytes%20induces%20kidney%20disease%20in%20mice&journal=Nat.%20Med.&volume=23&pages=429-438&publication_year=2017&author=Beckerman%2CP) 
    
85.  McGinnis, C. S., Murrow, L. M. & Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. _Cell Syst._ **8**, 329–337 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30954475)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6853612)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DoubletFinder%3A%20doublet%20detection%20in%20single-cell%20RNA%20sequencing%20data%20using%20artificial%20nearest%20neighbors&journal=Cell%20Syst.&volume=8&pages=329-337&publication_year=2019&author=McGinnis%2CCS&author=Murrow%2CLM&author=Gartner%2CZJ) 
    
86.  Young, M. D. & Behjati, S. SoupX removes ambient RNA contamination from droplet-based single-cell RNA sequencing data. _GigaScience_ **9**, giaa151 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33367645)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7763177)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SoupX%20removes%20ambient%20RNA%20contamination%20from%20droplet-based%20single-cell%20RNA%20sequencing%20data&journal=GigaScience&volume=9&publication_year=2020&author=Young%2CMD&author=Behjati%2CS) 
    
87.  Martin, F. J. et al. Ensembl 2023. _Nucleic Acids Res._ **51**, D933–D941 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36318249)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1GhsLrL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ensembl%202023&journal=Nucleic%20Acids%20Res.&volume=51&pages=D933-D941&publication_year=2023&author=Martin%2CFJ) 
    
88.  Cakir, B. et al. Comparison of visualization tools for single-cell RNAseq data. _NAR Genom. Bioinform._ **2**, lqaa052 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32766548)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7391988)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparison%20of%20visualization%20tools%20for%20single-cell%20RNAseq%20data&journal=NAR%20Genom.%20Bioinform.&volume=2&publication_year=2020&author=Cakir%2CB) 
    
89.  Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. _Genome Biol._ **19**, 15 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29409532)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5802054)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SCANPY%3A%20large-scale%20single-cell%20gene%20expression%20data%20analysis&journal=Genome%20Biol.&volume=19&publication_year=2018&author=Wolf%2CFA&author=Angerer%2CP&author=Theis%2CFJ) 
    
90.  Traag, V. A., Waltman, L. & van Eck, N. J. From Louvain to Leiden: guaranteeing well-connected communities. _Sci. Rep._ **9**, 5233 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30914743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6435756)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB3cbosVGisQ%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=From%20Louvain%20to%20Leiden%3A%20guaranteeing%20well-connected%20communities&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Traag%2CVA&author=Waltman%2CL&author=Eck%2CNJ) 
    
91.  Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. _Nucleic Acids Res._ **44**, W90–W97 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27141961)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987924)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enrichr%3A%20a%20comprehensive%20gene%20set%20enrichment%20analysis%20web%20server%202016%20update&journal=Nucleic%20Acids%20Res.&volume=44&pages=W90-W97&publication_year=2016&author=Kuleshov%2CMV) 
    
92.  Szklarczyk, D. et al. The STRING database in 2023: protein–protein association networks and functional enrichment analyses for any sequenced genome of interest. _Nucleic Acids Res._ **51**, D638–D646 (2022).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9825434)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20STRING%20database%20in%202023%3A%20protein%E2%80%93protein%20association%20networks%20and%20functional%20enrichment%20analyses%20for%20any%20sequenced%20genome%20of%20interest&journal=Nucleic%20Acids%20Res.&volume=51&pages=D638-D646&publication_year=2022&author=Szklarczyk%2CD) 
    
93.  Korotkevich, G. et al. Fast gene set enrichment analysis. Preprint at _bioRxiv_ [https://doi.org/10.1101/060012](https://doi.org/10.1101/060012) (2021).
    
94.  Gillis, J. Protocol data (Python version). _figshare_ [https://doi.org/10.6084/m9.figshare.13034171.v1](https://doi.org/10.6084/m9.figshare.13034171.v1) (2020).
    
95.  Kleshchevnikov, V. et al. Cell2location maps fine-grained cell types in spatial transcriptomics. _Nat. Biotechnol._ **40**, 661–671 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35027729)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1KqsL8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell2location%20maps%20fine-grained%20cell%20types%20in%20spatial%20transcriptomics&journal=Nat.%20Biotechnol.&volume=40&pages=661-671&publication_year=2022&author=Kleshchevnikov%2CV) 
    
96.  Megill, C. et al. cellxgene: a performant, scalable exploration platform for high dimensional sparse matrices. Preprint at _bioRxiv_ [https://doi.org/10.1101/2021.04.05.438318](https://doi.org/10.1101/2021.04.05.438318) (2021).
    
97.  Klötzer, K. A. et al. A cross-species single-cell kidney atlas: data repository. _Zenodo_ [https://doi.org/10.5281/zenodo.15007208](https://doi.org/10.5281/zenodo.15007208) (2025).
    
98.  KonstantinKltz. susztaklab/SISKA: pre-release V0.1. _Zenodo_ [https://doi.org/10.5281/zenodo.15109635](https://doi.org/10.5281/zenodo.15109635) (2025).
    
99.  KonstantinKltz. kloetzerka/CellSpectra: CellSpectra pre-release. _Zenodo_ [https://doi.org/10.5281/zenodo.15112987](https://doi.org/10.5281/zenodo.15112987) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02285-0?format=refman&flavour=references)

Acknowledgements
----------------

Work in the Susztak lab is supported by the NIH (2R01DK076077-15, 5R01DK087635-15, 5P50DK114786-07, 5R01DK105821-08 and 5R01DK132630-02). We thank the Translation Genetics in Renal Medicine (TIGER) grant from the University of Pennsylvania. N.R.Z. received grants from the NIH (1R56AG081351) and the National Science Foundation DMS/NIGMS (2245575). K.A.K. was supported by the MOLMED PhD program of the Medical University of Graz, a Marietta-Blau Grant (Österreichischer Austauschdienst, OeAD GmbH), and an Austrian Marshall Plan Foundation scholarship. M.S.B. was supported by grants from the Dr Werner Jackstädt Foundation, the German Research Foundation (Deutsche Forschungsgemeinschaft (DFG), BA 6205/2-1), Else Kröner–Fresenius Foundation and the Berlin Institute of Health at Charité—Universitätsmedizin Berlin Clinician Scientist Program. B.D. was supported by grants from the German Research Foundation (Deutsche Forschungsgemeinschaft (DFG), DU 2449/1-1 and an ASN Ben Lipps Fellowship. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Figures [1](https://www.nature.com/articles/s41588-025-02285-0#Fig1)–[6](https://www.nature.com/articles/s41588-025-02285-0#Fig6) and Extended Data Figs. [1](https://www.nature.com/articles/s41588-025-02285-0#Fig7), [2](https://www.nature.com/articles/s41588-025-02285-0#Fig8), [4](https://www.nature.com/articles/s41588-025-02285-0#Fig10), [6](https://www.nature.com/articles/s41588-025-02285-0#Fig12) and [10](https://www.nature.com/articles/s41588-025-02285-0#Fig16) were created with [BioRender.com](http://biorender.com/).

Author information
------------------

### Authors and Affiliations

1.  Renal, Electrolyte, and Hypertension Division, Department of Medicine, University of Pennsylvania, Perelman School of Medicine, Philadelphia, PA, USA
    
    Konstantin A. Klötzer, Amin Abedini, Shen Li, Michael S. Balzer, Xiujie Liang, Jonathan Levinsohn, Eunji Ha, Bernhard Dumoulin, Jonathan J. Hogan, Ghazal Quinn, Roy D. Bloom & Katalin Susztak
    
2.  Institute for Diabetes, Obesity, and Metabolism, University of Pennsylvania, Perelman School of Medicine, Philadelphia, PA, USA
    
    Konstantin A. Klötzer, Amin Abedini, Shen Li, Michael S. Balzer, Xiujie Liang, Jonathan Levinsohn, Eunji Ha, Bernhard Dumoulin, Ghazal Quinn, Roy D. Bloom & Katalin Susztak
    
3.  Penn/CHOP Kidney Innovation Center, Philadelphia, PA, USA
    
    Konstantin A. Klötzer, Amin Abedini, Shen Li, Michael S. Balzer, Xiujie Liang, Jonathan Levinsohn, Eunji Ha, Bernhard Dumoulin, Ghazal Quinn, Roy D. Bloom & Katalin Susztak
    
4.  Department of Genetics, University of Pennsylvania, Perelman School of Medicine, Philadelphia, PA, USA
    
    Konstantin A. Klötzer, Amin Abedini, Shen Li, Michael S. Balzer, Xiujie Liang, Jonathan Levinsohn, Eunji Ha, Bernhard Dumoulin & Katalin Susztak
    
5.  Division of Nephrology, Department of Internal Medicine, Medical University of Graz, Graz, Austria
    
    Konstantin A. Klötzer, Max Schuller & Kathrin Eller
    
6.  Department of Nephrology and Medical Intensive Care, Charité—Universitätsmedizin Berlin, Berlin, Germany
    
    Michael S. Balzer
    
7.  Cabaletta Bio, Philadelphia, PA, USA
    
    Jonathan J. Hogan
    
8.  Montefiore Medical Center, Albert Einstein College of Medicine, New York City, NY, USA
    
    Balazs Halmos
    
9.  Department of Statistics and Data Science, University of Pennsylvania, Wharton School, Philadelphia, PA, USA
    
    Nancy R. Zhang
    

Authors

1.  Konstantin A. Klötzer
    
    [View author publications](https://www.nature.com/search?author=Konstantin%20A.%20Kl%C3%B6tzer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Konstantin%20A.%20Kl%C3%B6tzer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Konstantin%20A.%20Kl%C3%B6tzer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Amin Abedini
    
    [View author publications](https://www.nature.com/search?author=Amin%20Abedini)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Amin%20Abedini) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Amin%20Abedini%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Shen Li
    
    [View author publications](https://www.nature.com/search?author=Shen%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shen%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shen%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Michael S. Balzer
    
    [View author publications](https://www.nature.com/search?author=Michael%20S.%20Balzer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Michael%20S.%20Balzer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Michael%20S.%20Balzer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Xiujie Liang
    
    [View author publications](https://www.nature.com/search?author=Xiujie%20Liang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiujie%20Liang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiujie%20Liang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Jonathan Levinsohn
    
    [View author publications](https://www.nature.com/search?author=Jonathan%20Levinsohn)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jonathan%20Levinsohn) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jonathan%20Levinsohn%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Eunji Ha
    
    [View author publications](https://www.nature.com/search?author=Eunji%20Ha)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eunji%20Ha) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eunji%20Ha%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Bernhard Dumoulin
    
    [View author publications](https://www.nature.com/search?author=Bernhard%20Dumoulin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Bernhard%20Dumoulin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Bernhard%20Dumoulin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Jonathan J. Hogan
    
    [View author publications](https://www.nature.com/search?author=Jonathan%20J.%20Hogan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jonathan%20J.%20Hogan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jonathan%20J.%20Hogan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Ghazal Quinn
    
    [View author publications](https://www.nature.com/search?author=Ghazal%20Quinn)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ghazal%20Quinn) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ghazal%20Quinn%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Roy D. Bloom
    
    [View author publications](https://www.nature.com/search?author=Roy%20D.%20Bloom)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Roy%20D.%20Bloom) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Roy%20D.%20Bloom%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Max Schuller
    
    [View author publications](https://www.nature.com/search?author=Max%20Schuller)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Max%20Schuller) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Max%20Schuller%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Kathrin Eller
    
    [View author publications](https://www.nature.com/search?author=Kathrin%20Eller)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kathrin%20Eller) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kathrin%20Eller%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Balazs Halmos
    
    [View author publications](https://www.nature.com/search?author=Balazs%20Halmos)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Balazs%20Halmos) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Balazs%20Halmos%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Nancy R. Zhang
    
    [View author publications](https://www.nature.com/search?author=Nancy%20R.%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nancy%20R.%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nancy%20R.%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Katalin Susztak
    
    [View author publications](https://www.nature.com/search?author=Katalin%20Susztak)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Katalin%20Susztak) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Katalin%20Susztak%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

K.S. conceptualized the study. K.A.K., A.A., M.S.B., J.L., E.H., B.D., M.S. and N.R.Z. performed computational analysis. A.A., S.L. and X.L. performed experiments. J.J.H., G.Q., R.D.B. and K.S. were responsible for the clinical collection of human samples from an observational cohort. M.S., K.E. and B.H. offered experimental and analytical suggestions. K.A.K., N.R.Z. and K.S. conceptualized CellSpectra. K.A.K. and N.R.Z. developed the R package. K.S. and N.R.Z. jointly supervised this work. K.A.K., N.R.Z. and K.S. wrote the original draft of the paper. All authors contributed to and approved the final version of the paper.

### Corresponding authors

Correspondence to [Nancy R. Zhang](mailto:nzh@wharton.upenn.edu) or [Katalin Susztak](mailto:ksusztak@pennmedicine.upenn.edu).

Ethics declarations
-------------------

### Competing interests

The Susztak lab is supported by Gilead, Novo Nordisk, Novartis, GSK, BIPI, Regeneron, Genentech, ONO and Calico. K.S. is on the scientific advisory boards of Pfizer and Otsuka. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Daniel Gale, Rafael Kramann and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02285-0#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Data summary of the multispecies kidney atlas (SISKA1.0) after QC.](https://www.nature.com/articles/s41588-025-02285-0/figures/7)

The table displays sample and cell numbers included in our unified kidney atlas (SISKA1.0) from control and diseased samples across species. The parenthesis indicates the number of individual samples. Respective disease states and rodent models are summarized in the ‘diseases and models’ column. Chronic kidney disease (CKD), acute kidney injury (AKI), ischemia-reperfusion injury (IRI) mouse model, diabetic kidney disease (DKD), folic acid mouse model (FA), Zucker fatty and spontaneously hypertensive (ZSF1) rat model of diabetes, deoxycorticosterone acetate (DOCA) rat model of hypertension, and hypertension-associated kidney disease (H-CKD). The figure is created with [BioRender.com](http://biorender.com/).

### [Extended Data Fig. 2 Relative frequencies of species across cell types in SISKA1.0.](https://www.nature.com/articles/s41588-025-02285-0/figures/8)

(**a**) UMAP of the integrated multispecies atlas (SISKA1.0) colored by species—human (blue), mouse (orange), and rat (green). (**b**) UMAP of the integrated multispecies atlas colored based on the input datasets. (**c**) UMAP of the integrated multispecies atlas, individually colored by species—human (blue), mouse (orange), and rat (green). (**d**) Bar graphs of the cell type compositions of each cluster (x-axis) across species (rat, human, mouse from top to bottom) for the 21 major annotated cell types. The y-axis is the percent of cells within each cluster (each normalized to 100%). (**e**) Bar graphs of the cell type compositions (x-axis) across input datasets for the 21 major annotated cell types. The y-axis is the percent of cells within each cluster (each normalized to 100%). The color legend corresponds to datasets in **b**. The figure is created with [BioRender.com](http://biorender.com/).

### [Extended Data Fig. 3 MetaNeighbor-based cell type similarity analysis across species and datasets.](https://www.nature.com/articles/s41588-025-02285-0/figures/9)

The x-axis and y-axis plots the cell types across seven processed datasets in an unsupervised dendrogram. The color indicates the mean area under the receiver operator characteristic curve (AUROC) of cell types. Red indicates a higher AUROC, while blue indicates a lower AUROC.

### [Extended Data Fig. 4 Differences in the proximal tubule composition of human and rodent datasets.](https://www.nature.com/articles/s41588-025-02285-0/figures/10)

(**a**) Cell type fractions of the healthy human (n = 20), mouse (n = 10), and rat (n = 8) kidney data. Asterisks indicate significant differences. PT subtypes are highlighted in red boxes. Two-sided Student t-test between human and rodent percentages per sample. \*p < 0.05, \*\*p < 0.01, \*\*\*p < 0.001, \*\*\*\*p < 0.0001. (**b**) Dot plot of PT subsegment marker genes within the human, mouse, and rat PT clusters. Dot colors indicate the scaled mean expression per cell type, the size of the dot indicates the percentage of cells expressing the respective gene. (**c**) Feature plots of healthy human, mice, and rat cells within SISKA 1.0. Arrows point towards the PTS3 segment. (**d**) Cell2location cell type abundance analysis on Visium spatial transcriptomics data of human, mouse, and rat (yellow indicates a higher estimated abundance). Plots are shown for the estimated PTS1, PTS2, and PTS3 abundance. The figure is created with [BioRender.com](http://biorender.com/).

[Source data](https://www.nature.com/articles/s41588-025-02285-0#MOESM8)

### [Extended Data Fig. 5 Conserved cell types and states in our integrated kidney atlas (SISKA 1.0).](https://www.nature.com/articles/s41588-025-02285-0/figures/11)

Fine resolution (126) clusters of SISKA1.0 ordered in a hierarchical dendrogram. The cell types with multispecies conserved gene expression are labeled in green, while those without are labeled in black. Columns visualize the composition of each cluster based on additional metadata information: Disease state (blue disease, pink healthy), species (mouse, rat, and human), sex (human only, male orange, female purple), sample identity, KPMP annotation (KPMP only), and global annotation. The right UMAP is colored by cell type labels. Principal cells of the collecting duct (CD\_PC), connective tubule (CNT), distal convoluted tubule (DCT and DCT2), intercalated cells (ICA and ICB), podocytes (Podo), parietal epithelial cells (PEC), PT (PTS1, PTS2, PTS3), injured PT (injPT), proliferating tubule (prolif\_Tubule), macula densa (MD), thick ascending limb (TAL), descending and ascending thin limb (DTL\_ATL), immune cells, endothelial cells (EC), stromal cells, injured TAL, and thick ascending and distal injured tubule (injTAL and injDCT\_CNT).

### [Extended Data Fig. 6 Hallmark features of frequent coordination changes across species.](https://www.nature.com/articles/s41588-025-02285-0/figures/12)

(**a**) Top significant cellular functions identified in diabetic kidney disease (DKD) and hypertensive chronic kidney disease (H-CKD) rat models. The x-axis shows −log10(p\_adj) values for top-ranked functions, with each bar color-coded by cell type. Functions in the DKD model were predominantly associated with podocytes, while those in the H-CKD model were linked to TAL cells, including ‘ligand-gated cation channel activity’ and inflammation-related gene sets. (**b**) Frequently significant cellular functions (p\_adj < 0.05 in at least 10% of samples across any species) were defined as conserved hallmark features. A pie chart displays the cell type distribution of these 595 hallmark features, with TAL and PT cells accounting for the majority. (**c**) Bar plot illustrating the dyscoordination prevalence of hallmark functions (percentage of significant samples) for human (blue), mouse (orange), and rat (green) samples. Functions such as ‘positive regulation of signaling receptor activity’ and ‘collagen catabolic process’ were frequently dyscoordinated across species. (**d**) R2 values and significance of coordination changes for selected hallmark features. Scatter plots show individual sample R2 values (y-axis) grouped by condition (x-axis) for TAL (‘positive regulation of signaling receptor activity’) and PT (‘collagen catabolic process’). Blue dots represent non-significant samples (p\_adj > 0.05), while red dots indicate significant coordination changes (p\_adj < 0.05). Separate panels are shown for human, mouse, and rat samples, emphasizing cross-species differences in hallmark functions. The figure is created with [BioRender.com](http://biorender.com/).

### [Extended Data Fig. 7 CellSpectra analysis of folic acid (FA) mouse models reveals consistent cell type involvement and pathway enrichment.](https://www.nature.com/articles/s41588-025-02285-0/figures/13)

(**a**) Radar plots display the percentage of significant gene sets per cell type for individual FA mice using the cross-species atlas. The radar plots highlight analysis results for different databases (KEGG, Gene Ontology) and indicate cell types such as PT, TAL, and Podo on the axes. (**b**) Similar radar plots are shown for mouse-specific data (mouse-only references), emphasizing cell-type-specific coordination changes comparable to the cross-species analysis. (**c**) Heatmaps compare GSEA-based (top) and CellSpectra-based (bottom) analyses for the KEGG pathways ‘mTOR signaling’ and ‘TNF signaling’. Enrichment scores (NES) were set to 0 and R2 values to 1 for non-significant samples (p\_adj > 0.05). Columns represent FA samples (FA1–FA4), while rows correspond to pathways. Color scales reflect NES (top) and R2 (bottom) values. These analyses demonstrate consistency in cell type involvement and pathway enrichment, highlighting the effectiveness of CellSpectra in identifying key pathways in FA-induced kidney injury.

### [Extended Data Fig. 8 CellSpectra analysis of a lung cancer single-cell atlas.](https://www.nature.com/articles/s41588-025-02285-0/figures/14)

(**a**) NSCLC single-cell atlas: adenocarcinoma (LUAD), squamous cell carcinoma (LUSC), and non-tumor tissue were analyzed. (**b**) The percentage of pathways (y-axis) showing significant coordination changes in epithelial cells and cancer cells (x-axis) in a total of 45 cancer samples (n = 43 for epithelial cells). Non-tumor tissue epithelial cells were used as the reference. (**c**) Heatmap of gene coordination changes (R2 values) in MSigDB C2 pathways in individual cases (cancer cells compared to a non-tumor epithelial reference). Red indicates lower R2 values (dyscoordination). Samples with p\_adj > 0.05 or those below the cell/gene number threshold are left white. LUSC, n = 6. LUAD, n = 30. (**d**) Percentage of pathways (y-axis) showing significant coordination changes in LUAD and LUSC samples (x-axis). Cancer cells were compared to a non-tumor epithelial reference. LUSC, n = 6. LUAD, n = 30. (**e**) Density plots of gene expression in selected genes from ‘pyrimidine metabolism’ (UMPS, TK1) and ‘glutathione conjugation’ (GSTM3, GGT5) in two cancer samples. Blue = ‘on-the-fly’ reference distribution; red arrow = query sample. Y-axis = density estimate; x-axis = gene expression. Lower R2 and higher NES in red. (**f**) Heatmap of gene coordination changes (R2) in LUAD stages II–IV (n = 15) and LUSC (n = 6) compared to LUAD stage I reference. Red = lower R2 or higher NES. GSEA results are shown on the right. Samples with p\_adj > 0.05 or insufficient data are white. (**g**) Density plot of gene expression distributions for FOS and NFE2L2 (‘oxidative stress response’) in a LUSC sample (red arrow) and the LUAD I reference distribution (blue line). Density (y-axis) and the normalized sample gene expression (x-axis) are shown. (**h**) Boxplots of normalized sample FOS and NFE2L2 gene expression (y-axis) in all LUSC samples (red, n = 6) and the pooled LUAD I reference (blue, 288 randomly permuted values equally derived from n = 16 biological replicates). All boxplots show the median (centerline), quartiles (box limits), and whiskers, which extend to the minimum and maximum values within 1.5× IQR. Data points outside this range are plotted as outliers.

[Source data](https://www.nature.com/articles/s41588-025-02285-0#MOESM9)

### [Extended Data Fig. 9 Sample number and disease summary of the extended human kidney atlas.](https://www.nature.com/articles/s41588-025-02285-0/figures/15)

Table summarizing the input data of the extended human atlas. Number of samples for five different diseases or groups (controls, CKD, AKI, CKD-ADPKD, and Penn Medicine biopsy samples) and the respective datasets. Subgroups are shown for CKD and Penn Medicine biopsies. The right column indicates the number of female and male patients per disease/condition.

### [Extended Data Fig. 10 CellSpectra functional profiling of ADPKD samples.](https://www.nature.com/articles/s41588-025-02285-0/figures/16)

(**a**) Single-cell functional profiling report of a patient with the diagnosis of autosomal dominant polycystic kidney disease (ADPKD). The radar plot indicates the total number of significant pathways (p\_adj < 0.05) per cell type (circles). Cell types below the cell number threshold are highlighted in red. Top significant gene sets based on the −log10(p-value) are highlighted for selected cell types (right). Color of the dot indicates the severity of coordination changes based on the R2 values (scaled for each gene set across all diseased samples). Size of the dot indicates the frequency of significant coordination changes for this specific cellular function among all diseased samples above the cell number cut-off. (**b**) Diseased-reference analysis overview. Eight ADPKD samples were compared to a CKD reference. Highly repetitive (prevalence > 75%; total of 185 features) cellular functions were enriched in PT cells, visualized in a pie chart (bottom). (**c**) Two cellular functions identified in the CKD-reference analysis are shown in the healthy-reference setting. Dots show individual samples, with R2 indicated on the y-axis. The x-axis groups individual samples into groups (CKD and ADPKD, PT: n = 68 and n = 7; DCT\_CNT\_CD: n = 72 and n = 8). The y-axis represents the R2 value, while red dots indicate p\_adj < 0.05. Bars display the mean R2 per group. (**d**) Boxplots visualizing the normalized sample gene expression of underlying genes (y-axis) for the non-PKD reference (orange) and the PKD query samples (red, n = 8). Reference expression was pooled from the ‘on-the-fly’ references of all query samples (2,232 perturbated values equally derived from n = 93 patients). All boxplots show the median (centerline), quartiles (box limits), and whiskers, which extend to the minimum and maximum values within 1.5× IQR. Data points outside this range are plotted as outliers. The figure is created with [BioRender.com](http://biorender.com/).

[Source data](https://www.nature.com/articles/s41588-025-02285-0#MOESM10)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM1_ESM.pdf)

Supplementary Methods and Figs. 1–21.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM4_ESM.xlsx)

Supplementary Tables 1–10.

Source data
-----------

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM5_ESM.xlsx)

Cross-species CellSpectra _P_ values (_P_adj) related to Fig. 4b,c,f.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM6_ESM.xlsx)

CellSpectra and GSEA _P_ values (_P_adj) related to Fig. 5c.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM7_ESM.xlsx)

CellSpectra _P_ values (_P_adj) related to Fig. 6d,e.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM8_ESM.xlsx)

_P_ values of cell fraction differences between human and rodents

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM9_ESM.xlsx)

CellSpectra and GSEA _P_ values (_P_adj) related to the cancer analysis related to Extended Data Fig. 8c,d,f.

### [Source Data Extended Data Fig.10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02285-0/MediaObjects/41588_2025_2285_MOESM10_ESM.xlsx)

CellSpectra _P_ values (_P_adj) related to Extended Data Fig. 10c.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Analysis%20of%20individual%20patient%20pathway%20coordination%20in%20a%20cross-species%20single-cell%20kidney%20atlas&author=Konstantin%20A.%20Kl%C3%B6tzer%20et%20al&contentID=10.1038%2Fs41588-025-02285-0&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-07&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02285-0)

### Cite this article

Klötzer, K.A., Abedini, A., Li, S. _et al._ Analysis of individual patient pathway coordination in a cross-species single-cell kidney atlas. _Nat Genet_ **57**, 1922–1934 (2025). https://doi.org/10.1038/s41588-025-02285-0

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02285-0?format=refman&flavour=citation)

-   Received: 20 July 2024
    
-   Accepted: 25 June 2025
    
-   Published: 07 August 2025
    
-   Issue Date: August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02285-0
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
