> id: https://www.nature.com/articles/s41588-025-02281-4

> link: https://www.nature.com/articles/s41588-025-02281-4

> title: Natural variation in SBRR1 shows high potential for sheath blight resistance breeding in rice

# Natural variation in SBRR1 shows high potential for sheath blight resistance breeding in rice
_Published on Wed, 30 Jul 2025 00:00:00 GMT_

Abstract
--------

Sheath blight (ShB), caused by necrotrophic fungus _Rhizoctonia solani_, is one of the most serious rice diseases worldwide. To the best of our knowledge, no genes with high potential for rice ShB resistance breeding have been previously characterized. Here we identify a _ShB resistance receptor-like kinase 1_ (_SBRR1_) gene via a genome-wide association study. The _SBRR1-R_ elite allele, containing a 256-bp insertion in its promoter, is preferentially present in _indica_ varieties in geographical regions with highly favorable conditions for ShB development. Introduction of _SBRR1-R_ into a commercial _japonica_ rice variety significantly reduces yield loss under severe ShB disease pressure. Transcription factor bHLH57 specifically binds to the 256-bp sequence and accounts for highly induced expression and stronger resistance of _SBRR1-R_. Localization of SBRR1 on plasma membrane, aided by SBRR1-interaction-protein 1, and phosphorylation of SBRR1 are required for SBRR1 to rapidly upregulate downstream chitinase genes for resistance. These findings offer mechanistical insights into ShB resistance hidden in natural rice varieties.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-025-85918-y/MediaObjects/41598_2025_85918_Fig1_HTML.png)

### [Quantitative proteomic analysis deciphers mechanisms of sheath blight resistance in novel rice landrace against _Rhizoctonia solani_](https://www.nature.com/articles/s41598-025-85918-y?fromPaywallRec=false)

Article Open access 02 August 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-00759-9/MediaObjects/41598_2021_759_Fig1_HTML.png)

### [Accumulating candidate genes for broad-spectrum resistance to rice blast in a drought-tolerant rice cultivar](https://www.nature.com/articles/s41598-021-00759-9?fromPaywallRec=false)

Article Open access 02 November 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-024-05769-7/MediaObjects/42003_2024_5769_Fig1_HTML.png)

### [Superior haplotypes of key drought-responsive genes reveal opportunities for the development of climate-resilient rice varieties](https://www.nature.com/articles/s42003-024-05769-7?fromPaywallRec=false)

Article Open access 12 January 2024

Main
----

The soil-borne necrotrophic fungal pathogen _Rhizoctonia solani_ (_R. solani_) invades a broad range of plants, including staple food crops, such as rice, maize and potato, posing a great threat to global food security[1](https://www.nature.com/articles/s41588-025-02281-4#ref-CR1 "Zhen, A. et al. The evolution and pathogenic mechanisms of the rice sheath blight pathogen. Nat. Commun. 4, 1424 (2013)."),[2](https://www.nature.com/articles/s41588-025-02281-4#ref-CR2 "Okubara, P. A., Dickman, M. B. & Blechl, A. E. Molecular and genetic aspects of controlling the soilborne necrotrophic pathogens Rhizoctonia and Pythium. Plant Sci. 228, 61–70 (2014)."). In rice, it mainly attacks leaf sheath, causing the sheath blight (ShB) disease. ShB is one of the most destructive diseases of rice worldwide and causes an average yield loss of ~10% to 30% each year—even up to 50% under favorable conditions[3](https://www.nature.com/articles/s41588-025-02281-4#ref-CR3 "Marchetti, M. A. & Bollich, C. N. Quantification of the relationship between sheath blight severity and yield loss in rice. Plant Dis. 75, 773–775 (1991)."),[4](https://www.nature.com/articles/s41588-025-02281-4#ref-CR4 "Lee, F. N. & Rush, M. C. Rice sheath blight: a major rice disease. Plant Dis. 67, 829–832 (1983)."),[5](https://www.nature.com/articles/s41588-025-02281-4#ref-CR5 "Senapati, M. et al. Rhizoctonia solani Kühn pathophysiology: status and prospects of sheath blight disease management in rice. Front. Plant Sci. 3, 88116 (2022)."). With the widespread cultivation of semi-dwarf and nitrogen-insensitive varieties, rice ShB has increasingly become serious in most rice production regions[5](https://www.nature.com/articles/s41588-025-02281-4#ref-CR5 "Senapati, M. et al. Rhizoctonia solani Kühn pathophysiology: status and prospects of sheath blight disease management in rice. Front. Plant Sci. 3, 88116 (2022)."),[6](https://www.nature.com/articles/s41588-025-02281-4#ref-CR6 "Singh, P., Mazumdar, P., Harikrishna, J. A. & Babu, S. Sheath blight of rice: a review and identification of priorities for future research. Planta 250, 1387–1407 (2019)."). Although no completely resistant rice germplasms have been described, a natural variation with clear ShB resistance among rice accessions has been reported[7](https://www.nature.com/articles/s41588-025-02281-4#ref-CR7 "Chen, Z. et al. Identification of new resistance loci against sheath blight disease in rice through genome-wide association study. Rice Sci. 26, 21–31 (2019)."),[8](https://www.nature.com/articles/s41588-025-02281-4#ref-CR8 "Zhang, F. et al. Genome-wide association analysis of the genetic basis for sheath blight resistance in rice. Rice 12, 93 (2019)."). A few varieties, like Tetep and YSBR1, have been repeatedly identified as having relatively high ShB resistance[9](https://www.nature.com/articles/s41588-025-02281-4#ref-CR9 "Molla, K. A. et al. Understanding sheath blight resistance in rice: the road behind and the road ahead. Plant Biotechnol. J. 18, 895–915 (2020)."),[10](https://www.nature.com/articles/s41588-025-02281-4#ref-CR10 "Cao, W. et al. Suppressing chlorophyll degradation by silencing OsNYC3 improves rice resistance to Rhizoctonia solani, the causal agent of sheath blight. Plant Biotechnol. J. 20, 335–349 (2022)."). Also, _indica_ rice was generally found more resistant than _japonica_ rice[11](https://www.nature.com/articles/s41588-025-02281-4#ref-CR11 "Jia, L. et al. Searching for germplasm resistant to sheath blight from the USDA rice core collection. Crop Sci. 51, 1507–1517 (2011)."). However, resistance mechanisms underlying these varieties remain poorly understood.

Rice ShB resistance is controlled by multiple quantitative trait loci (QTLs) or genes, and more than 60 QTLs for ShB resistance have been detected so far[5](https://www.nature.com/articles/s41588-025-02281-4#ref-CR5 "Senapati, M. et al. Rhizoctonia solani Kühn pathophysiology: status and prospects of sheath blight disease management in rice. Front. Plant Sci. 3, 88116 (2022)."),[12](https://www.nature.com/articles/s41588-025-02281-4#ref-CR12 "Wang, Y. et al. Fine mapping and candidate gene analysis of qSB12YSB, a gene conferring major quantitative resistance to rice sheath blight. Theor. Appl. Genet. 136, 246 (2023)."). However, although some of them present breeding potentials, to the best of our knowledge, no ShB resistance QTLs have been isolated by traditional map-based cloning strategy because of the difficulty in precisely phenotyping ShB resistance in a large segregating population[5](https://www.nature.com/articles/s41588-025-02281-4#ref-CR5 "Senapati, M. et al. Rhizoctonia solani Kühn pathophysiology: status and prospects of sheath blight disease management in rice. Front. Plant Sci. 3, 88116 (2022)."),[9](https://www.nature.com/articles/s41588-025-02281-4#ref-CR9 "Molla, K. A. et al. Understanding sheath blight resistance in rice: the road behind and the road ahead. Plant Biotechnol. J. 18, 895–915 (2020)."),[12](https://www.nature.com/articles/s41588-025-02281-4#ref-CR12 "Wang, Y. et al. Fine mapping and candidate gene analysis of qSB12YSB, a gene conferring major quantitative resistance to rice sheath blight. Theor. Appl. Genet. 136, 246 (2023)."),[13](https://www.nature.com/articles/s41588-025-02281-4#ref-CR13 "Zuo, S. et al. Fine mapping of qSB-11LE, the QTL that confers partial resistance to rice sheath blight. Theor. Appl. Genet. 126, 1257–1272 (2013)."),[14](https://www.nature.com/articles/s41588-025-02281-4#ref-CR14 "Zuo, S. et al. Fine-mapping of qSB-9TQ, a gene conferring major quantitative resistance to rice sheath blight. Mol. Breed. 34, 2191–2203 (2014)."),[15](https://www.nature.com/articles/s41588-025-02281-4#ref-CR15 "Eizenga, G. C., Jia, M. H., Pinson, S. R., Gasore, E. R. & Prasad, B. Exploring sheath blight quantitative trait loci in a Lemont/O. meridionalis advanced backcross population. Mol. Breed. 35, 1–19 (2015)."). Comparatively, the genome-wide association study (GWAS) provides advantages in mining genes, especially for complex quantitative traits in crops[16](https://www.nature.com/articles/s41588-025-02281-4#ref-CR16 "Huang, X. et al. Genome-wide association studies of 14 agronomic traits in rice landraces. Nat. Genet. 42, 961–967 (2010)."),[17](https://www.nature.com/articles/s41588-025-02281-4#ref-CR17 "Chen, R. et al. Rice functional genomics: decades’ efforts and roads ahead. Sci. China Life Sci. 65, 33–92 (2022)."), and has been used to identify several genes/alleles with breeding potential in rice[17](https://www.nature.com/articles/s41588-025-02281-4#ref-CR17 "Chen, R. et al. Rice functional genomics: decades’ efforts and roads ahead. Sci. China Life Sci. 65, 33–92 (2022)."),[18](https://www.nature.com/articles/s41588-025-02281-4#ref-CR18 "Li, W. et al. A natural allele of a transcription factor in rice confers broad-spectrum blast resistance. Cell 170, 114–126 (2017)."),[19](https://www.nature.com/articles/s41588-025-02281-4#ref-CR19 "Liu, Y. et al. Genomic basis of geographical adaptation to soil nitrogen in rice. Nature 590, 600–605 (2021)."). Regarding ShB resistance, because the varieties in GWAS are all pure lines, their phenotypes can be evaluated using multiple plants to increase reliability. By using 259 diverse rice varieties, Wang et al.[20](https://www.nature.com/articles/s41588-025-02281-4#ref-CR20 "Wang, A. et al. Identification of rice (Oryza sativa L.) genes involved in sheath blight resistance via a genome-wide association study. Plant Biotechnol. J. 19, 1553–1566 (2021).") characterized two ShB resistance genes, _OsRSR1_ and _OsRLCK5_, which improve resistance by regulating reactive oxygen species homeostasis. A stable ShB resistance QTL was identified by GWAS and considered to regulate ShB resistance by suppressing reactive oxygen species accumulation[21](https://www.nature.com/articles/s41588-025-02281-4#ref-CR21 "Oreiro, E. G. et al. Genome-wide associations and transcriptional profiling reveal ROS regulation as one underlying mechanism of sheath blight resistance in rice. Mol. Plant Microbe. 33, 212–222 (2020)."). Using GWAS, Li et al.[22](https://www.nature.com/articles/s41588-025-02281-4#ref-CR22 "Li, N. et al. Natural variation in ZmFBL41 confers banded leaf and sheath blight resistance in maize. Nat. Genet. 51, 1540–1548 (2019).") cloned the _ZmFBL41__B73_ gene that confers resistance to maize ShB and confirmed that its resistance mechanism against ShB via increasing lignin contents in cell walls is conserved in rice. Recently, Gao et al.[23](https://www.nature.com/articles/s41588-025-02281-4#ref-CR23 "Gao, M. et al. Ca2+ sensor-mediated ROS scavenging suppresses rice immunity and is exploited by a fungal effector. Cell 184, 5391–5404 (2021).") found that disruption of the Ca2+\-sensor gene _ROD1_ confers resistance to several rice diseases, including ShB, but leads to reduced grain yield; notably, they further identified a natural variation, _ROD1(SNP1__A__)_, that enhances resistance without yield loss. However, ShB resistance genes isolated from natural varieties remain very limited.

Here, we identify an elite allele of the _ShB resistance RLK1_ (_SBRR1_) gene from natural varieties through GWAS. We then show great potential for the _SBRR1-R_ allele in rice breeding and decipher its critical upstream regulator and downstream defense signals.

Results
-------

### GWAS identifies _SBRR1_ tightly associated with ShB resistance

To investigate resistance to _R. solani_ in rice, we evaluated ShB resistance of 178 commercial rice cultivars from various regions of China, Japan and South Korea by artificial inoculation in the field (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). These cultivars showed disease scores ranging from 4.15 to 7.47, with most of them (78.09%) showing susceptibility and only 1.69% exhibiting moderate resistance (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02281-4#Fig9)). We conducted a genotyping-by-sequencing analysis and identified a total of 109,444 high-quality single-nucleotide polymorphisms (SNPs) covering the whole rice genome. Structural analysis suggested that these varieties could be divided into two subpopulations, corresponding to _indica_ and _japonica_ subspecies (Extended Data Fig. [1b–e](https://www.nature.com/articles/s41588-025-02281-4#Fig9)). In order to systematically survey ShB resistance genes, we then performed a GWAS for ShB resistance using the general linear model (GLM) approach controlling population structure and identified 48 SNPs that exceeded the suggestive significance threshold of association (_P_ < 3.2 × 10−5; Fig. [1a](https://www.nature.com/articles/s41588-025-02281-4#Fig1) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). These significant SNPs correspond to seven regions located on chromosomes 1, 2, 4, 7, 8 and 11 (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)).

**Fig. 1: Identification of natural variations in _SBRR1_ that are significantly associated with ShB resistance in rice using GWAS.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/1)

**a**, Manhattan plot of the GWAS. The horizontal line depicts the Bonferroni-adjusted significance threshold (_P_ = 3.2 × 10−5). The dotted box indicates the location of the two most significant SNPs. **b**, _SBRR1_ (_ShB resistance RLK 1_) -based association mapping and pairwise LD (linkage disequilibrium) analysis. Triangles denote Indels (insertions/deletions) and dots represent SNPs. The lead SNPs/Indels are highlighted in red. The SNPs/Indels showing strong LD with the lead SNPs/Indels are connected to the pairwise LD diagram with solid lines and highlighted with asterisks. **c**, ShB resistance levels of _SBRR1-S_ and _SBRR1-R_ haplotypes. _n_ denotes the number of genotypes belonging to each haplotype group. The disease score distribution of each haplotype group is displayed as a bar plot. **d**, Comparison of _SBRR1_ expression between _SBRR1-R_ and _SBRR1-S_ varieties (_n_ = 20). _SBRR1_ expression levels were determined for 20 _SBRR1-S_ and 20 _SBRR1-R_ varieties with and without _R. solani_ infection. HPI, hours post inoculation. **e**, Transient expression assay for promoter activity in rice protoplasts. Left panel: schematic representation of different promoter versions for the reporter construct. Right panel: luciferase reporter assay (_n_ = 6). p_SBRR1-S_ and p_SBRR1-R_ indicate _SBRR1_ promoter from ZD88 (Zhendao 88) and XWX7 (Xiangwanxian 7) varieties, respectively. Data in **c**–**e** are presented as means ± standard deviation (s.d.). Statistical significances in **c** and **d** were determined by a two-sided Student’s _t_\-test. Different lowercase letters in **e** indicate a significant difference (_P_ < 0.05) based on one-way analysis of variance (ANOVA) with Duncan’s multiple range test.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM6)

The two most significant (_P_ = 1.3×10−8) SNPs, [S94780](https://www.ncbi.nlm.nih.gov/nuccore/S94780) and [S94782](https://www.ncbi.nlm.nih.gov/nuccore/S94782), and three other significant SNPs ([S94831](https://www.ncbi.nlm.nih.gov/nuccore/S94831), [S94832](https://www.ncbi.nlm.nih.gov/nuccore/S94832) and [S94833](https://www.ncbi.nlm.nih.gov/nuccore/S94833)) are localized on chromosome 11 in a linkage disequilibrium (LD) block (191 kb) (Fig. [1a](https://www.nature.com/articles/s41588-025-02281-4#Fig1), Supplementary Fig. [1a,b](https://www.nature.com/articles/s41588-025-02281-4#MOESM1), Supplementary Table [1](https://www.nature.com/articles/s41588-025-02281-4#MOESM4) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). In this block, _LOC\_Os11g10290_, encoding a G-type LecRLK (lectin receptor-like kinase) protein, was identified as the most likely candidate and named _ShB Resistance RLK1_ (_SBRR1_) hereafter (Supplementary Fig. [1c,d](https://www.nature.com/articles/s41588-025-02281-4#MOESM1) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). Further, we resequenced the _SBRR1_ alleles (chromosome 11: 5586341–5592553 bp), including a 3,341-bp promoter region, a 96-bp 5′ untranslated region (UTR), a 2,463-bp coding region, a 143-bp 3′ UTR and a 150-bp downstream sequence from 20 relatively susceptible (disease score >6.5) and 20 relatively resistant (disease score <5.5) varieties then performed a gene-based association mapping (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). We identified one variant (SNP1653 or [S94780](https://www.ncbi.nlm.nih.gov/nuccore/S94780) mentioned above) in the coding region and three variants (SNP-2697 or [S94782](https://www.ncbi.nlm.nih.gov/nuccore/S94782) mentioned above, indel (insertion/deletion) −1,171 (a 131-bp insertion) and indel-946 (a 256-bp insertion)) in the promoter region that were significantly associated with ShB resistance (Fig. [1b](https://www.nature.com/articles/s41588-025-02281-4#Fig1)). Based on the four significantly associated variants, two haplotypes of _SBRR1_ were further identified, and one of them (haplotype ‘A—131 bp—256 bp—T’) showed significantly lower ShB disease scores than the other haplotype (‘G— - — - — G’), which were hereafter named _SBRR1-R_ and _SBRR1-S_, respectively (Fig. [1c](https://www.nature.com/articles/s41588-025-02281-4#Fig1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)).

As the three promoter variants may affect expression whereas SNP1653 located in the coding region does not cause an amino acid change, we measured _SBRR1_ RNA expression levels for the 20 _SBRR1-S_ and 20 _SBRR1-R_ varieties before and after _R. solani_ inoculation. Expression of _SBRR1-R_ was significantly higher than that of _SBRR1-S_ before inoculation, and _SBRR1-R_ was more strongly induced than _SBRR1-S_, reaching a level 2.8-fold as high as that of _SBRR1-S_ 12 h post-inoculation (HPI) (Fig. [1d](https://www.nature.com/articles/s41588-025-02281-4#Fig1) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). We further confirmed the differences between _SBRR1-S_ and _SBRR1-R_ promoters by conducting a transient expression assay in rice protoplasts using a luciferase reporter gene (_LUC_) driven by _SBRR1-S_ promoter (from Zhendao 88, ZD88), _SBRR1-R_ promoter (from Xiangwanxian 7, XWX7) and three _SBRR1-R_ variant promoters containing deletions in the two indel regions of the _SBRR1-R_ promoter. The _SBRR1-R_ promoter more effectively promoted _LUC_ expression than the _SBRR1-S_ promoter by nearly twofold (Fig. [1e](https://www.nature.com/articles/s41588-025-02281-4#Fig1)). Furthermore, the higher _LUC_ expression level from the _SBRR1-R_ promoter was abolished by deletion of the 256-bp fragment (indel-946), but not indel-1171, indicating the importance of the 256-bp insertion in the higher expression of _SBRR1-R_. Together, these results suggest that the _SBRR1-R_ allele may be an elite allele conferring ShB resistance.

### SBRR1 kinase phosphorylation is required for ShB resistance

To validate the involvement of _SBRR1_ in rice ShB resistance, we evaluated the phenotypes of a T-DNA insertion mutant _sbrr1_, _SBRR1_ knockout lines (_sbrr1_\-_ko1/2/3_) and overexpression lines (_SBRR1_\-OE1/2/3), and we confirmed that _SBRR1_ positively regulates ShB resistance (Fig. [2a,b](https://www.nature.com/articles/s41588-025-02281-4#Fig2), Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02281-4#Fig10), Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)–[4](https://www.nature.com/articles/s41588-025-02281-4#MOESM1) and Supplementary Note [2](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). As an annotated kinase, we also confirmed that SBRR1 kinase activity is required for its resistance (Fig. [2c,d](https://www.nature.com/articles/s41588-025-02281-4#Fig2) and Supplementary Note [3](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). In a Phos-tag SDS-PAGE (sodium dodecyl sulfate polyacrylamide gel electrophoresis) assay using _SBRR1_\-OE plants, we found a clear bandshift for SBRR1 protein representing phosphorylated SBRR1 at 12 HPI with _R. solani_; this bandshift reached the highest level (approximately fourfold) at 18 HPI compared to 0 HPI in _SBRR1_\-OE plants but was not detected in wild type (WT), indicating that phosphorylation of SBRR1 is induced by pathogen attack (Fig. [2e](https://www.nature.com/articles/s41588-025-02281-4#Fig2) and Extended Data Fig. [3a–c](https://www.nature.com/articles/s41588-025-02281-4#Fig11)).

**Fig. 2: Transgenic verification of _SBRR1_ function in ShB resistance and requirement of kinase activity for SBRR1 function.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/2)

**a**, ShB disease severity of _SBRR1_ knockout lines (_sbrr1-ko1/2/3_, T2 generation) 14 days after _R. solani_ inoculation in a greenhouse. **b**, ShB resistance of _SBRR1_ overexpression lines (_SBRR1_\-OE1/2/3, T3 generation). **c**, Immunoblotting assay using a pSer/Thr antibody to detect autophosphorylation of His-SBRR1-ICD (ICD, intracellular domain) and His-SBRR1-ICDK553E. Ponceau S stains His-SBRR1-ICD and His-SBRR1-ICDK553E proteins. P.S., Ponceau S staining. **d**, Complementation by WT (Com, T3 generation) and kinase-dead (KiD, T3 generation) constructs for the susceptible phenotype of the _sbrr1_ mutant (_n_ = 18). **e**, Comparison of phosphorylation levels of SBRR1 protein in _SBRR1_\-OE1 with or without _R. solani_ infection. Total protein was fractionated in SDS-PAGE (sodium dodecyl sulfate polyacrylamide gel electrophoresis) containing Phos-tag or normal SDS-PAGE. ACTIN was used as a loading reference. **f**, LC-MS/MS (liquid chromatography-tandem mass spectrometry)-detected intensity ratios of phosphorylated (P-peptide) to un-phosphorylated peptide (UP-peptide) at S678/T682/T683/Y733 sites of SBRR1 protein in _SBRR1_\-OE1 line with or without _R. solani_ infection (_n_ = 3). **g**, Immunoblot assay using a pSer/Thr antibody to detect autophosphorylation of His-SBRR1-ICDTT682/683AA. P.S., Ponceau S staining. **h**, ShB resistance of _SBRR1__TT682/683AA_\-OE lines (T2 generation) at 7 days after inoculation. Data in **a**, **b**, **d**, **f** and **h** are presented as means ± s.d. Statistical significances in **a** and **b** were determined by a two-sided Student’s _t_\-test. Different lowercase letters in d and h indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range test. Phosphatase in **c**, **e** and **g** indicates calf intestinal alkaline phosphatase. Scale bar (**a**, **b** and **d**), 5 cm. The in vitro and in vivo phosphorylation assays were done independently three times.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM7)

We next used liquid chromatography-tandem mass spectrometry (LC-MS/MS) to detected amino acid residues phosphorylated upon _R. solani_ infection. We found phosphorylation of SBRR1 residues S678, T682, T683, and Y733 in _SBRR1_\-OE plants upon inoculation with _R. solani_; importantly, no phosphorylation of these residues was detected without _R. solani_ infection. In particular, the phosphorylation levels of T682 and T683 were most notable, showing ratios (counts of phosphorylated peptide/unphosphorylated peptide detected in MS) of 1.8 and 2.8, respectively (Fig. [2f](https://www.nature.com/articles/s41588-025-02281-4#Fig2) and Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). We noticed that both T682 and T683 are located in the kinase activation loop of SBRR1, and in general, phosphorylation of one to three residues in the activation loop is required for kinase activity[24](https://www.nature.com/articles/s41588-025-02281-4#ref-CR24 "Taylor, S. S. et al. Structural framework for the protein kinase family. Annu. Rev. Cell Biol. 8, 429–462 (1992)."),[25](https://www.nature.com/articles/s41588-025-02281-4#ref-CR25 "Lemmon, M. A. & Schlessinger, J. Cell signaling by receptor tyrosine kinases. Cell 141, 1117–1134 (2010)."). To evaluate the effects of T682 and T683 phosphorylation to SBRR1 kinase activity, we mutated both residues from threonine to alanine (TT682/683AA) and found that the His-SBRR1-ICDTT682/683AA (ICD, intracellular domain) protein completely abolished SBRR1 autophosphorylation (Fig. [2g](https://www.nature.com/articles/s41588-025-02281-4#Fig2) and Extended Data Fig. [3d](https://www.nature.com/articles/s41588-025-02281-4#Fig11)), suggesting that the phosphorylation of TT682/683 is essential for SBRR1 kinase activity. To extend these findings _in vivo_, we generated transgenic rice plants overexpressing _SBRR1__TT682/683AA_ under control of the 35S promoter (Extended Data Fig. [3e](https://www.nature.com/articles/s41588-025-02281-4#Fig11)). We found that _SBRR1__TT68/2683AA_\-OE plants did not improve ShB resistance compared with WT, showing significantly lower resistance than _SBRR1_\-OE plants (Fig. [2h](https://www.nature.com/articles/s41588-025-02281-4#Fig2)). Taken together, these data confirm that the kinase activity of SBRR1 and phosphorylation of TT682/683 are required for SBRR1-mediated ShB resistance.

### _SBRR1-R_ is an elite allele mainly present in _indica_ rice

To confirm the variation in the promoter region causes the difference in resistance between _SBRR1-R_ and _SBRR1-S_, we transformed the _sbrr1_ mutant with elite promoter (EPro) construct p_SBRR1_XWX7:_SBRR1_DJ (_SBRR1-R_ promoter from XWX7 driving _SBRR1-S_ coding region from Dongjin (DJ)). The three EPro lines showed significantly higher inducible _SBRR1_ transcript levels, reaching levels approximately threefold as high as that of WT DJ after _R. solani_ inoculation, indicating that the _SBRR1-R_ promoter from XWX7 indeed more strongly responded to _R. solani_ infection than the _SBRR1-S_ promoter (Fig. [3a](https://www.nature.com/articles/s41588-025-02281-4#Fig3)). When assessed for their resistance to _R. solani_, we found that the three EPro lines all showed lesion lengths (~15.22–15.70 cm) significantly shorter than WT (18.76 cm) and the _sbrr1_ mutant (23.84 cm) (Fig. [3b](https://www.nature.com/articles/s41588-025-02281-4#Fig3)). The resistance phenotype in field test is consistent with that in greenhouse and no visible changes in major agronomic traits were found in the EPro transgenic lines compared with WT (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02281-4#Fig12)). Together, these data confirm that the natural variation in the promoter region accounts for stronger resistance of the _SBRR1-R_ elite allele.

**Fig. 3: ShB resistance function, geographic distribution and molecular evolution of elite _SBRR1-R._**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/3)

**a**, Expression levels of _SBRR1_ in EPro lines (p_SBRR1_XWX7:_SBRR1_DJ, EPro1/2/3, T3 generation) (_n_ = 3). **b**, ShB resistance of EPro lines. Data in **a** and **b** are presented as means ± s. d. Statistical significance in **a** was determined by a two-sided Student’s _t_\-test. Different lowercase letters in **b** indicate a significant difference (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range test. Scale bar, 5 cm. **c**, Geographic distributions of _SBBR1-R_ and _SBRR1-S_ among 2,478 cultivated rice varieties. Orange and green circles indicate _SBRR1-R_ and _SBRR1-S_, respectively. **d**, The relative ratio of nucleotide diversity of cultivated rice to common wild rice in the whole rice chromosome 11. The dotted red line indicates the _SBRR1_ locus. The ratios of nucleotide diversity of _japonica_/_O. rufipogon_ and _indica_/_O. rufipogon_ are shown. The ratio in _SBRR1_ is higher than in flanking regions for _japonica_/_O. rufipogon_, whereas the ratio in _SBRR1_ is notably lower than in flanking regions for _indica_/_O. rufipogon_. **e**, The Tajima’s _D_ analysis in the whole chromosome 11 of cultivated rice. Purple line and dots indicate the _SBRR1_ locus. The estimated Tajima’s _D_ values for the _SBRR1_ locus are substantially positive in _indica_ but near zero in _japonica_. **f**, Haplotype network of _SBRR1_ generated using the coding and upstream 1.5-kb sequences of _SBRR1_ from cultivated rice varieties and common wild rice. Black spots on the lines indicate mutational steps between two haplotypes.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM9)

After a comprehensive survey, we found differential presence of _SBRR1-R_ in _indica_ and _japonica_ rice varieties (Supplementary Figs. [6](https://www.nature.com/articles/s41588-025-02281-4#MOESM1) and [7a,b](https://www.nature.com/articles/s41588-025-02281-4#MOESM1), Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02281-4#MOESM4) and [6](https://www.nature.com/articles/s41588-025-02281-4#MOESM4) and Supplementary Note [4](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). Geographic distribution analysis showed that the _SBRR1-R_ allele is mainly present in Southeast Asia, East Asia, South Asia and Africa, all of which have favorable temperature and humidity conditions for ShB disease development during rice growing periods (Fig. [3c](https://www.nature.com/articles/s41588-025-02281-4#Fig3)).

To further probe the molecular evolution of _SBRR1-R_, we analyzed the relative ratio of nucleotide diversity in _indica_ to wild rice and observed that it is notably lower in the _SBRR1_ locus than in flanking regions (Fig. [3d](https://www.nature.com/articles/s41588-025-02281-4#Fig3) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)) and that the estimated Tajima’s _D_ values for the _SBRR1_ locus are substantially positive in _indica_ (Fig. [3e](https://www.nature.com/articles/s41588-025-02281-4#Fig3)). These results suggest that _SBRR1_ might have experienced a balancing selection during _indica_ rice domestication. By contrast, the locus of _SBRR1_ followed a neutral evolution during _japonica_ rice domestication (Fig. [3d,e](https://www.nature.com/articles/s41588-025-02281-4#Fig3) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). Notably, we found that the _SBRR1_ allele was diverged in the common wild rice (Supplementary Table [9](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). Up to 71.43% of _O. rufipogon_ I (Or-I, the progenitor of _indica_) varieties contained the _SBRR1-R_ allele, whereas all sampled _O. rufipogon_ III (Or-III, the progenitor of _japonica_) contained the _SBRR1-S_ allele. Both phylogenetic data and haplotype network analysis results using 37 _SBRR1_ haplotypes revealed that the _indica_ varieties containing _SBRR1-R_ and the _japonica_ varieties containing _SBRR1-S_ had the closest relationship with Or-I and Or-III types of wild rice, respectively (Fig. [3f](https://www.nature.com/articles/s41588-025-02281-4#Fig3) and Supplementary Fig. [7c](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)), suggesting that _SBRR1-R_ and _SBRR1-S_ alleles are derived from Or-I and Or-III, respectively. Together, we conclude that the _SBRR1-R_ allele originated from wild rice Or-I and then was inherited by most _indica_ varieties located in the regions with more favorable conditions for _R. solani_ infection.

### _SBRR1-R_ allele shows great potential for breeding

Because the _SBRR1-R_ allele is almost absent in _japonica_ rice cultivars, which are increasingly spreading toward south China due to the large demand of _japonica_ rice, we tested the breeding potential of _SBRR1-R_ in _japonica_ rice against ShB. Based on the sequence variations in the promoter region between _SBRR1-R_ and _SBRR1-S_, we developed a diagnostic marker for _SBRR1-R_ (Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02281-4#Fig13)). Via marker assisted selection, we transferred _SBRR1-R_ from YSBR1, a rice variety containing the _SBRR1-R_ allele showing relatively high resistance to ShB[10](https://www.nature.com/articles/s41588-025-02281-4#ref-CR10 "Cao, W. et al. Suppressing chlorophyll degradation by silencing OsNYC3 improves rice resistance to Rhizoctonia solani, the causal agent of sheath blight. Plant Biotechnol. J. 20, 335–349 (2022)."),[26](https://www.nature.com/articles/s41588-025-02281-4#ref-CR26 "Feng, Z. et al. iTRAQ-based quantitative proteomics analysis of defense responses triggered by the pathogen Rhizoctonia solani infection in rice. J. Integr. Agr. 21, 139–152 (2022)."), into two widely planted _japonica_ cultivars, Taigeng 394 (TG394) and Xudao 3 (XD3) separately (Extended Data Figs. [2a,b](https://www.nature.com/articles/s41588-025-02281-4#Fig10) and [5a,b](https://www.nature.com/articles/s41588-025-02281-4#Fig13)). At the BC5F5 generation, we obtained near-isogenic lines TG394-_SBRR1__R_ and XD3-_SBRR1__R_ carrying _SBRR1-R_ in the TG394 and XD3 backgrounds, respectively (Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02281-4#Fig13)). TG394-_SBRR1__R_ showed significantly higher ShB resistance than WT TG394, with a 24.4% reduction in lesion length (19.3 versus 14.6 cm) in greenhouse assays (Fig. [4a](https://www.nature.com/articles/s41588-025-02281-4#Fig4)). Similarly, XD3-_SBRR1__R_ also displayed significantly enhanced ShB resistance compared with XD3, with a 22.0% reduction in lesion length and no differences in main agronomic traits (Extended Data Fig. [5c,d](https://www.nature.com/articles/s41588-025-02281-4#Fig13)).

**Fig. 4: _SBRR1-R_ significantly reduces grain yield losses caused by ShB in field test.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/4)

**a**, ShB lesions of TG394-_SBRR1__R_ (TG394, Taigeng 394) and TG394 measured 14 days after _R. solani_ inoculation in greenhouse (_n_ = 27). **b**, ShB phenotypes of TG394-_SBRR1__R_ and TG394 under severe disease conditions in the field. **c**–**l**, Comparison of disease score (**c**, _n_ = 60), grain yield (**d**, _n_ = 5), seed setting rate (**e**, _n_ = 11), 1,000-grain weight (**f**, _n_ = 11), panicle number (**g**, _n_ = 11), grain number per panicle (**h**, _n_ = 11), plant height (**i**, _n_ = 10), growth period (**j**, _n_ = 10), chalkiness rate (**k**, _n_ = 10) and amylose content (**l**, _n_ = 10) of TG394-_SBRR1__R_ and TG394 under slight and severe disease conditions in the field. Data in **a** and **c**–**l** are presented as means ± s.d. Statistical significance in a was determined by a two-sided Student’s _t_\-test. Different lowercase letters in **c**–**l** indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range test. N.S. indicates no significant differences. Scale bar, 5 cm in **a**, 5 cm in **b** and 1 cm in **k**.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM10)

Furthermore, we conducted a field test for TG394 and TG394-_SBRR1__R_ under two ShB disease conditions: slight and severe ShB diseases (Fig. [4b–l](https://www.nature.com/articles/s41588-025-02281-4#Fig4)). We found that under the severe ShB disease condition, TG394-_SBRR1__R_ showed an average ShB disease score of 6.13, significantly lower than TG394 (7.22), demonstrating a considerable resistance effect for _SBRR1-R_ in the field trial (Fig. [4b,c](https://www.nature.com/articles/s41588-025-02281-4#Fig4)). In the severe ShB condition, grain yield and grain quality of TG394 and TG394-_SBRR1__R_ lines were both significantly reduced, mainly attributed to decreased seed setting rate, 1,000-grain weight, increased chalkiness rate and amylose content (Fig. [4d–i](https://www.nature.com/articles/s41588-025-02281-4#Fig4)). However, the TG394-_SBRR1__R_ line had significantly lower yield loss than TG394 upon severe ShB disease occurrence: TG394 yield was reduced from 1,404.0 g/1.32 m2 to 1,028.5 g/1.32 m2 (26.75% drop) and TG394-_SBRR1__R_ yield from 1,354.3 g/1.32 m2 to 1,126.6 g/1.32 m2 (16.82% drop). Comparatively, the _SBRR1-R_ allele rescued 9.54% (98.1 g per 1.32 m2) of the 26.75% grain yield loss of TG394 due to reduced ShB disease severity (Fig. [4d](https://www.nature.com/articles/s41588-025-02281-4#Fig4)). Further analysis showed that the rescued yield losses were mainly attributed to two yield components: seed setting rate (6.75%) and 1,000-grain weight (2.00%) (Fig. [4e,f](https://www.nature.com/articles/s41588-025-02281-4#Fig4)). TG394-_SBRR1__R_ also displayed significantly lower loss in grain quality than TG394, including lower chalkiness rate (~3% reduction) and amylose content (~0.8% reduction), suggesting that _SBRR1-R_ also contributed to retaining grain quality under severe ShB disease conditions (Fig. [4k,l](https://www.nature.com/articles/s41588-025-02281-4#Fig4)). Under slight ShB disease conditions, TG394-_SBRR1__R_ showed no significant differences from TG394 in disease scores (Fig. [4c](https://www.nature.com/articles/s41588-025-02281-4#Fig4)), major agronomical traits, grain yield associated traits and grain quality, indicating that _SBRR1-R_ has no adverse impact on rice development (Fig. [4d–l](https://www.nature.com/articles/s41588-025-02281-4#Fig4) and Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02281-4#Fig13)). Meanwhile, we also conducted a similar field trial using _SBRR1_\-OE1 and WT plants and, in agreement, found that overexpressing _SBRR1_ rescued 13.2% of yield losses under severe disease conditions (Extended Data Fig. [6](https://www.nature.com/articles/s41588-025-02281-4#Fig14)). Together, these results suggest that _SBRR1-R_ carries great potential for rice breeding against ShB.

### bHLH57 differentiates _SBRR1-R_ and _SBRR1-S_ expression

To investigate how the 256-bp insertion in the _SBRR1-R_ promoter affects its higher ShB resistance (Fig. [1e](https://www.nature.com/articles/s41588-025-02281-4#Fig1)), we performed a yeast one-hybrid (Y1H) screening assay using this insertion segment and identified transcription factor bHLH57 (basic helix-loop helix 57, LOC\_Os07g35870) (Fig. [5a](https://www.nature.com/articles/s41588-025-02281-4#Fig5)). By dividing the 256-bp region into two fragments for the Y1H assay, we further narrowed down the binding site of bHLH57 to a 121 bp region (pSBRR1-M2) that contains a ‘CACCGG’ motif putatively serving as a _cis_\-motif for bHLH57 binding. Moreover, mutating ‘CACCGG’ to ‘AAACGG’ (pSBRR1-mutated-M2) abolished the interaction with bHLH57 (Fig. [5a](https://www.nature.com/articles/s41588-025-02281-4#Fig5)). Notably, this motif is only present in the 256-bp insertion sequence of _SBRR1-R_ but not in the _SBRR1-S_ promoter or its coding genomic region. Furthermore, in electrophoretic mobility shift assays (EMSAs), we detected a slower migrating band for the 21-bp native probe carrying the CACCGG motif when incubated with a GST-bHLH57 (GST, glutathione S-transferase) fusion protein. Unlabeled WT competitors, but not mutated competitors, at 20-fold greatly reduced the binding to the probe, indicating specific binding to this sequence (Fig. [5b](https://www.nature.com/articles/s41588-025-02281-4#Fig5)). To further confirm that bHLH57 binds to the _SBRR1-R_ promoter in vivo, we carried out a chromatin-immunoprecipitation-quantitative PCR (ChIP-qPCR) assay using TG394-_SBRR1__R_/_bHLH57_\-OE plants (Supplementary Fig. [8](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). The results showed that bHLH57 specifically enriched the P2 fragment carrying the CACCGG motif of the _SBRR1-R_ promoter (Fig. [5c](https://www.nature.com/articles/s41588-025-02281-4#Fig5)). We next tested the effect of bHLH57 on _SBRR1-R_ expression by conducting dual-luciferase assays in _N. benthamiana_. We found that addition of bHLH57 protein notably increased (by threefold) luciferase activities driven by the _SBRR1-R_ promoter, but not by the _SBRR1-S_ promoter (Fig. [5d](https://www.nature.com/articles/s41588-025-02281-4#Fig5)), indicating that bHLH57 is able to activate _SBRR1-R_ expression but not _SBRR1-S_ expression. In addition, we found that bHLH57 protein was localized in the nucleus and mainly expressed in leaf sheaths and blades, which is well correlated with _R. solani_ infection sites in rice (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02281-4#Fig15)). Together, these data indicate that bHLH57 activates _SBRR1-R_ expression by directly binding to its 256-bp insertion sequence.

**Fig. 5: bHLH57 accounts for the highly induced expression of _SBRR1-R_ by specifically binding to its promoter.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/5)

**a**, Y1H assay to identify potential bHLH57 (basic helix-loop helix 57) binding site. M1 and M2 represent two fragments of the 256-bp insertion. The CACCGG motif is found only in the M2 fragment. **b**, EMSA to confirm the direct binding of bHLH57 to CACCGG motif. GST-EV, glutathione S-transferase-empty vector. **c**, ChIP-qPCR assay for the enrichment of the P2 region of the _SBRR1-R_ promoter by transcription factor bHLH57 (_n_ = 3). Enriched values are normalized to the input. An unrelated _OsActin1_ intron region (ACTIN) was used as a negative control. IgG-immunoprecipitated DNA was used as a control. **d**, Dual-luciferase reporter assay to study the effects of bHLH57 on _SBRR1-R_ gene expression (_n_ = 6). 35S, cauliflower mosaic virus 35S promoter. GFP, green fluorescent protein. REN, Renilla luciferase. LUC, firefly luciferase. **e**, RNA expression levels of the _SBRR1_ gene in TG394, T394-_SBRR1__R_ and their _bhlh57-ko_ plants (_n_ = 3). HPI, hours post inoculation. **f**, ShB lesions of _bHLH57_ knockout lines (T1 generation) in TG394 and T394-_SBRR1__R_ backgrounds scored 14 days after _R. solani_ inoculation in greenhouse (_n_ = 24). **g**, ShB lesions of _bHLH57_ overexpression lines (T1 generation) in TG394 and T394-_SBRR1__R_ backgrounds scored 14 days after _R. solani_ inoculation in greenhouse (_n_ = 21). Data in **c**–**g** are presented as means ± s.d. Statistical significances in **d** and **c** were determined by a two-sided Student’s _t_\-test. Different lowercase letters in **e**–**g** indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range test. Scale bar, 5 cm in **f** and **g**. The Y1H, dual-luciferase reporter and EMSAs were done independently three times.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM11)

To determine the contribution of _bHLH57_ to _SBRR1-R-_mediated ShB resistance, we used CRISPR/Cas9 to generate _bHLH57_ knockout lines in TG394 cultivar (carrying the _SBRR1-S_ allele) and its near-isogenic line TG394-_SBRR1__R_ (with the _SBRR1-R_ allele) background separately (Supplementary Fig. [9](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). We found that the RNA level of _SBRR1_ in TG394-_SBRR1__R_ plants was about seven times of that in TG394 at 12 HPI, whereas the level in TG394-_SBRR1__R_/_bhlh57-ko_ plants was comparable with those in TG394 and TG394/_bhlh57-ko_ plants (Fig. [5e](https://www.nature.com/articles/s41588-025-02281-4#Fig5)), indicating that the strong induction pattern of _SBRR1_ expression in _SBRR1-R_ varieties depends on bHLH57. In agreement, we found that all TG394/_bhlh57-ko_ and TG394-_SBRR1__R_/_bhlh57-ko_ lines displayed lower ShB resistance than their corresponding controls (Fig. [5f](https://www.nature.com/articles/s41588-025-02281-4#Fig5)), indicating that _bHLH57_ positively regulates ShB resistance. Importantly, the fact that TG394-_SBRR1__R_ plants showed stronger ShB resistance than TG394 plants, whereas TG394-_SBRR1__R_/_bhlh57-ko_ plants displayed similar susceptible levels as TG394/_bhlh57-ko_ plants, clearly showed that _SBRR1-R_\-mediated ShB resistance requires _bHLH57_ (Fig. [5f](https://www.nature.com/articles/s41588-025-02281-4#Fig5)). We also generated _bHLH57_ overexpression lines in TG394 and TG394-_SBRR1__R_ backgrounds (Supplementary Fig. [8](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)) and found that TG394/_bHLH57_\-OE lines displayed enhanced ShB resistance in comparison with TG394, which is consistent with a previous report showing that _bHLH57_ was involved in ShB resistance[27](https://www.nature.com/articles/s41588-025-02281-4#ref-CR27 "Liu, J. et al. OsbHLH057 targets the AATCA cis‑element to regulate disease resistance and drought tolerance in rice. Plant Cell Rep. 41, 1285–1299 (2022)."). In addition, we found that TG394-_SBRR1__R_/_bHLH57_\-OE plants showed stronger resistance than both TG394-_SBRR1__R_ and TG394/_bHLH57_\-OE plants (Fig. [5g](https://www.nature.com/articles/s41588-025-02281-4#Fig5)), further confirming that _bHLH57_ and _SBRR1-R_ depend on each other to further enhance ShB resistance. Collectively, these data indicate that bHLH57 is a positively master regulator in the expression of elite _SBRR1-R_ in response to _R. solani_ infection.

### SBRR1 localization aided by SIP1 is crucial for its function

We identified a SBRR1-interacting protein (named SIP1, LOC\_Os03g63480) via yeast two-hybrid (Y2H) and confirmed that SIP1 is required for SBRR1-mediated ShB resistance (Fig. [6](https://www.nature.com/articles/s41588-025-02281-4#Fig6), Supplementary Figs. [10](https://www.nature.com/articles/s41588-025-02281-4#MOESM1) and [11](https://www.nature.com/articles/s41588-025-02281-4#MOESM1) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). To probe how SIP1 affects SBRR1-mediated resistance, we generated SBRR1 and SIP1 proteins truncated at different regions and assessed their interaction in Y2H system. We found that the plasminogen-apple-nematode (PAN) domain in the extracellular juxtamembrane region of SBRR1 and the aspartate-proline (DP)-repeat domain of SIP1 were required for their interaction (Extended Data Fig. [8](https://www.nature.com/articles/s41588-025-02281-4#Fig16)). Because DP-repeat domain has been reported to affect protein transport from endoplasmic reticulum (ER) to plasma membrane (PM)[28](https://www.nature.com/articles/s41588-025-02281-4#ref-CR28 "Chen, L. et al. The Hop/Sti1-Hsp90 Chaperone complex facilitates the maturation and transport of a PAMP receptor in rice innate immunity. Cell Host Microbe 7, 185–196 (2010)."), we hypothesized that the subcellular localization of SBRR1 might be influenced by SIP1. To test this hypothesis, we analyzed the subcellular localization of SBRR1 and SIP1 in both WT rice protoplasts and _N. benthamiana_ leaf epidermal cells. We found that the green fluorescence of the SBRR1-GFP (GFP, green fluorescent protein) protein was present on PM, merging well with the red signals of the PM marker, and that SIP1-GFP was partially co-localized with the ER marker and the PM marker (Fig. [7a](https://www.nature.com/articles/s41588-025-02281-4#Fig7), Supplementary Fig. [12a](https://www.nature.com/articles/s41588-025-02281-4#MOESM1) and Extended Data Fig. [9](https://www.nature.com/articles/s41588-025-02281-4#Fig17)). We then observed the localization pattern of SBRR1-GFP in the _sip1-ko1_ mutant and found that most SBRR1-GFP signals (~81%) were localized to ER, with only ~19% localized on PM, indicating that the _sip1-ko1_ mutant severely affected the PM localization of SBRR1 (Fig. [7b,c](https://www.nature.com/articles/s41588-025-02281-4#Fig7) and Supplementary Fig. [12b](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). We further generated an SBRR1 mutant (SBRR1ΔPAN) that lacks the PAN domain and found that almost all SBRR1ΔPAN\-GFP signals (~95%) were co-localized with the ER marker and only ~5% co-localized with the PM marker in WT rice protoplasts (Fig. [7d,e](https://www.nature.com/articles/s41588-025-02281-4#Fig7) and Supplementary Fig. [12c](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)), suggesting that deletion of the PAN domain caused retention of SBRR1 in ER.

**Fig. 6: SIP1 physically interacts with SBRR1 and is required for SBRR1-mediated ShB resistance.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/6)

**a**–**c**, Y2H assay (**a**), in vitro GST (glutathione S-transferase) pull-down assay (**b**), and in vivo co-immunoprecipitation (Co-IP) assay using _SBRR1_\-OE1 transgenic plants (**c**) for SBRR1 and SIP1 (SBRR1-interacting protein 1) interaction. ECD, extracellular domain. ICD, intracellular domain. BD, binding domain; AD, activity domain. The schematic diagram on the right side in **a** shows the different domains of SBRR1 and SIP1 proteins. Lower panel in **b** shows the protein abundance stained by CBB (Coomassie brilliant blue) in SDS-PAGE gel. FLAG, transgenic plants expressing a FLAG tag alone, and anti-PIP1;1 (plasma membrane intrinsic protein 1;1) were used as negative controls. The GST pull-down assay and Co-IP assay were performed three times independently with similar results. **d**, ShB lesions of SIP1 knockout and overexpression lines (T3 generation) scored 14 days after _R. solani_ inoculation in greenhouse. **e**, ShB lesions of _sbrr1-ko1/sip1-ko1_, _sbrr1-ko1_/_SIP1_\-OE1 and _SBRR1_\-OE1/_sip1-ko1_ lines 14 days after _R. solani_ inoculation in greenhouse. Data in **d** and **e** are presented as means ± s.d. Different lowercase letters in **d** and **e** indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range test. Scale bar, 5 cm in **d** and **e**. The Y2H, pull-down and Co-IP assays were done independently three times.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM13)

**Fig. 7: SIP1 is required for transport of SBRR1 to the plasma membrane.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/7)

**a**, Subcellular localization of SBRR1 and SIP1 in WT protoplasts. **b**,**c**, Subcellular localization of SBRR1-GFP in WT and _sip1-ko1_ protoplasts, and its partition in PM (plasma membrane) and ER (endoplasmic reticulum). **d**,**e**, Subcellular localization of SBRR1ΔPAN\-GFP (PAN, plasminogen-apple-nematode) in WT protoplasts and its partition in PM and ER. The images in **a**, **b** and **d** were taken under a laser scanning confocal microscope 12 h after transfection. Scale bar (**a**, **b** and **d**), 10 μm. The ratio of protoplasts with only PM-localized GFP signals to the total protoplasts with GFP signals was measured with 100 fluorescent cells in **c** and **e**. The protoplasts without clear PM-localized GFP signals show ER-localized GFP signals. Data in **c** and **e** are presented as means ± s.d (_n_ = 4). Different lowercase letters in c indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range test. Statistical significance in e was determined by a two-sided Student’s _t_\-test. **f**, SBRR1 protein distribution in two-phase partitioning. S, soluble fraction; M, microsomal fraction; U, upper phase of partitioning; L, lower phase of partitioning. The blot was probed with the antibody indicated to the right of each panel. PIP1;1 (plasma membrane intrinsic protein 1;1), plasma membrane marker; HSP82 (heat shock protein 82), cytoplasmic marker. The subcellular localization and two-phase partitioning assays were done independently three times.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM15)

In addition, we performed aqueous two-phase partitioning experiments using _SBRR1_\-OE1 and _SBRR1_\-OE1/_sip1-ko1_ lines and found that SBRR1 protein was all detected in the PM-enriched fraction (U, upper phase of partitioning) in _SBRR1_\-OE1 plants; whereas in _SBRR1_\-OE1/_sip1-ko1_ plants, less than one third of SBRR1 were detected in the PM-enriched fraction and more than two thirds of SBRR1 were found in the endomembrane-enriched fraction (L, lower phase of partitioning) (Fig. [7f](https://www.nature.com/articles/s41588-025-02281-4#Fig7); Supplementary note [6](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). This further confirmed that _sip1-ko_ indeed influenced the PM localization of SBRR1. The loss of proper SBRR1 localization on PM due to lack of SIP1 may be the reason that _SBRR1_\-OE1/_sip1-ko1_ showed a ShB resistance level close to WT level (Fig. [6e](https://www.nature.com/articles/s41588-025-02281-4#Fig6)). Together, these data demonstrate that SIP1 facilitates SBRR1 accumulation on PM where SBRR1 functions.

### _SBRR1_ confers ShB resistance mainly via chitinase genes

To further probe the resistance mechanism of _SBRR1_ to ShB, we performed an RNA-seq to compare transcript profiles of both _sbrr1-ko1_ and _sip1-ko1_ (most SBRR1 protein was retained in ER) plants with those of WT plants before and after inoculation with _R. solani_. We identified 1,137 and 760 differentially expressed genes (DEGs) responding specifically to _R. solani_ for _sbrr1-ko1_ and _sip1-ko1_, respectively (Extended Data Fig. [10a](https://www.nature.com/articles/s41588-025-02281-4#Fig18); Supplementary Tables [9](https://www.nature.com/articles/s41588-025-02281-4#MOESM4),[10](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). Gene ontology (GO) analysis showed that these DEGs belong to many pathways. Interestingly, we noted that multiple _chitinase_ gene-related pathways (chitin catabolic process, chitin binding and chitinase activity) and antioxidative pathways (oxidation-reduction process, oxidoreductase activity) were affected in both _sbrr1-ko1_ and _sip1-ko1_ plants (Extended Data Fig. [10a](https://www.nature.com/articles/s41588-025-02281-4#Fig18) and Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02281-4#MOESM4) and [12](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). It has been widely validated that chitinases contribute importantly to ShB resistance by degrading chitin, the most abundant polymer of fungal cell walls[9](https://www.nature.com/articles/s41588-025-02281-4#ref-CR9 "Molla, K. A. et al. Understanding sheath blight resistance in rice: the road behind and the road ahead. Plant Biotechnol. J. 18, 895–915 (2020)."). Therefore, we compared the expression levels of all 19 _chitinase_ (_Chit_) genes between _sbrr1-ko1_ and WT and between _sip1-ko1_ and WT at different timepoints post inoculation. Compared with WT plants, we found that the induced expression levels of _Chit3_ (_Loc\_Os04g41680_), _Chit4_ (_Loc\_Os03g30470_), _Chit7_ (_LOC\_Os06g51050_), _Chit12_ (_Loc\_Os08g41100_) and _Chit17_ (_Loc\_Os05g33130_) in _sbrr1-ko1_ plants, and _Chit3_, _Chit4_, and _Chit17_ in _sip1-ko1_ plants were significantly lower, and among them _Chit3_ and _Chit4_ were changed most significantly. These data suggest that chitinase genes act downstream of _SBRR1_ in response to _R. solani_ (Fig. [8a](https://www.nature.com/articles/s41588-025-02281-4#Fig8); Supplementary Fig. [13](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)).

**Fig. 8: _SBRR1_ confers ShB resistance mainly by promoting expression of chitinase genes and its working model.**

[![figure 8](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_Fig8_HTML.png)](https://www.nature.com/articles/s41588-025-02281-4/figures/8)

**a**, _Chitinase_ (_Chit_) genes showing the most significant changes in RNA expression levels between WT and _sbrr1-ko1_ or _sip1-ko1_ line at different hours post _R. solani_ inoculation (HPI) (_n_ = 3). **b**, Chitinase activity in _SBRR1_ and _SIP1_ knockout and overexpression lines at different HPI (_n_ = 3). **c**, ShB lesions of WT, _SBRR1_\-OE1 and _SBRR1_\-OE1/_chit3chit4-ko_ lines (T2 generation) 14 days after _R. solani_ inoculation in greenhouse (_n_ = 24). **d**, ShB lesions of WT, _Chit3_\-OE, _sbrr1-ko1_, _sip1-ko1_, _sbrr1-ko1_/_Chit3_\-OE (T2 generation) and _sip1-ko1/Chit3_\-OE (T2 generation) lines 14 days after _R. solani_ inoculation in greenhouse. **e**, A model for SBRR1-mediated ShB resistance. In _SBRR1-R_ varieties, transcription factor bHLH57 specifically binds to the ‘CACCGG’ motif in the 256-bp insertion of _SBRR1-R_ promoter, elevating _SBRR1_ expression, reflected by mRNA. In cytoplasm, SIP1 protein, which likely acts as a molecular chaperone, interacts with the extracellular domain of SBRR1 to facilitate effective accumulation of SBRR1 protein on plasma membrane to function. As a kinase protein, SBRR1 phosphorylation is increased upon _R. solani_ attack and required for ShB resistance. Lastly, quickly elevated expression of _Chit_ genes accounts for the majority of SBRR1-mediated downstream defense against _R. solani_. In _SBRR1-S_ varieties, due to lack of the 256-bp insertion in promoter, bHLH57 cannot effectively activate _SBRR1_ expression, and lower _SBRR1_ mRNA lead to lower accumulation of chitinases and susceptibility. Data in **a**–**d** are presented as means ± s.d. Different lowercase letters in **b**–**d** indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range test. N.S. indicates no significant differences. Scale bar, 5 cm in **c** and **d**.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM17)

Further, we measured chitinase activities in the knockout and overexpression lines of _SBRR1_ and _SIP1_ at different time points after inoculation. We found that chitinase activities were all significantly reduced (up to ~50% reduction) in _sbrr1-ko_ and _sip1-ko_ lines, but significantly increased in overexpression plants compared with WT plants, at 24 and 48 HPI (Fig. [8b](https://www.nature.com/articles/s41588-025-02281-4#Fig8)). Since the _SBRR1-R_ allele shows more rapidly induced _SBRR1_ expression levels than _SBRR1-S_, we also measured the chitinase activities and found that TG394-_SBRR1__R_ carried ~40% higher chitinase activity than TG394 at 48 HPI, but showed no differences in chitinase activity than TG394 without ShB inoculation (Extended Data Fig. [10b](https://www.nature.com/articles/s41588-025-02281-4#Fig18)).

To confirm the contribution of chitinases to SBRR1-mediated resistance, we simultaneously knocked out _Chit3_ and _Chit4_, which showed the most significant changes in their transcript levels, in _SBRR1_\-OE1 background. We found that the knockout plants (_SBRR1_\-OE1/_chit3_\-_ko_/_chit4-ko_) were significantly more susceptible to ShB than _SBRR1_\-OE1 plants (~16.4 cm versuss 14.5 cm lesion length) but still moderately more resistant than WT plants (18.7 cm), indicating that the two _chitinase_ genes are required for full function of _SBRR1_ overexpression (Fig. [8c](https://www.nature.com/articles/s41588-025-02281-4#Fig8) and Supplementary Fig. [14](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). We also overexpressed _Chit3_ in _sbrr1-ko1_, _sip1-ko1_ and WT backgrounds and found that _sbrr1_\-_ko1_/_Chit3_\-OE and _sip1_\-_ko1_/_Chit3_\-OE plants clearly exhibited stronger resistance than _sbrr1-ko1_, _sip1-ko1_ and even WT plants but lower resistance than _Chit3_\-OE plants in WT background (Fig. [8d](https://www.nature.com/articles/s41588-025-02281-4#Fig8) and Supplementary Fig. [15](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)), suggesting that _Chit3_\-OE can restore ShB resistance in _sbrr1-ko1_ and _sip1-ko1_ plants. Together, these results demonstrate that chitinase genes play a critical role in _SBRR1_\-mediated downstream defense response to _R. solani_ infection (Fig. [8e](https://www.nature.com/articles/s41588-025-02281-4#Fig8)).

Discussion
----------

Currently, available genes with breeding values for ShB resistance are very limited. Here, we identified an elite natural allele, _SBRR1-R_, which shows a significant effect on ShB resistance. We found that _SBRR1-R_ is most likely inherited from _O. rufipogon_ I and is dominantly distributed in _indica_ rice varieties from the regions with more favorable temperature and humidity conditions for _R. solani_ infection, implying the importance of this allele in combating the pathogen (Fig. [3](https://www.nature.com/articles/s41588-025-02281-4#Fig3) and Supplementary Fig. [7](https://www.nature.com/articles/s41588-025-02281-4#MOESM1)). By using a functional marker specific to the 256-bp insertion, we successfully introduced _SBRR1-R_ into two commercial _japonica_ rice varieties and found it significantly increased ShB resistance in both backgrounds. Importantly, under severe ShB conditions, this elite allele was found to rescue up to 9.54% of the total 26.75% yield loss. Certainly, this conclusion still needs more large-scale field tests in more rice varieties with different genetic backgrounds, but it suggests a great potential for _SBRR1-R_ in developing ShB resistant varieties (Fig. [4](https://www.nature.com/articles/s41588-025-02281-4#Fig4) and Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02281-4#Fig13)).

In summary, we propose a working model for SBRR1-mediated ShB resistance (Fig. [8e](https://www.nature.com/articles/s41588-025-02281-4#Fig8)). In _SBRR1-R_ varieties, transcription factor bHLH57 acts as a positively master regulator, directly binding to the 256-bp insertion sequence of _SBRR1-R_ promoter to effectively elevate _SBRR1-R_ expression upon _R. solani_ infection. In cytoplasm, SIP1 likely functions as a molecular chaperone to facilitate SBRR1 protein for effective accumulation on PM, where SBRR1 protein goes through _R. solani_\-induced phosphorylation events to activate SBRR1 function. With regard to the downstream mechanism leading to ShB resistance, the rapid activation of _Chit3_ and _Chit4_ genes accounts for the majority of SBRR1-mediated resistance. But activation of other genes, such as those involved in antioxidative pathways, may also contribute to the full SBRR1-mediated ShB resistance. However, how SBRR1 is phosphorylated and then transduces the signal into the nucleus to activate the expression of _chitinase_ genes and other genes remains to be further elucidated. We hypothesize that SBRR1 relays the intracellular signaling directly or indirectly to downstream transcription factor(s) that in turn regulate the expression of _chitinase_ and other defense genes to counteract _R. solani_ infection. In _SBRR1-S_ varieties, _SBRR1_ is expressed at lower levels due to the lack of bHLH57-promoted expression, leading to lower levels of chitinases and other relevant proteins and to ShB susceptibility. Together, our findings not only provide an elite allele for rice ShB resistance breeding but also discover a molecular module mediated by _SBRR1-R_ against necrotrophic pathogen _R.solani_ in natural rice varieties.

Methods
-------

### Plant materials

The 178 rice cultivars used for GWAS were selected from different regions of China (148), South Korea (15) and Japan (15) (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). The _sbrr1_ mutant (3D-50196L) that carries a T-DNA insertion located 19 bp downstream of the ATG start codon of _SBRR1_ was isolated from a collection of T-DNA insertion lines (Extended Data Fig. [2c,d](https://www.nature.com/articles/s41588-025-02281-4#Fig10); [http://orygenesdb.cirad.fr/](http://orygenesdb.cirad.fr/))[29](https://www.nature.com/articles/s41588-025-02281-4#ref-CR29 "Jeon, J. S. et al. T-DNA insertional mutagenesis for functional genomics in rice. Plant J. 22, 561–570 (2000)."). WT _SBRR1_ was from DJ, a _japonica_ variety. The YSBR1 variety carrying high ShB resistance, derived from the progeny of a _japonica/indica_ hybrid developed by pedigree breeding[10](https://www.nature.com/articles/s41588-025-02281-4#ref-CR10 "Cao, W. et al. Suppressing chlorophyll degradation by silencing OsNYC3 improves rice resistance to Rhizoctonia solani, the causal agent of sheath blight. Plant Biotechnol. J. 20, 335–349 (2022)."),[26](https://www.nature.com/articles/s41588-025-02281-4#ref-CR26 "Feng, Z. et al. iTRAQ-based quantitative proteomics analysis of defense responses triggered by the pathogen Rhizoctonia solani infection in rice. J. Integr. Agr. 21, 139–152 (2022)."), was the donor of the _SBRR1-R_ allele. Two temperate _japonica_ cultivars Taigeng 394 (TG394) and Xudao 3 (XD3) carrying the _SBRR1-S_ allele were used as recipients of _SBRR1-R_.

### Evaluation of ShB resistance

_R. solani_ isolate _RH-9_ with strong pathogenicity was used to inoculate rice[10](https://www.nature.com/articles/s41588-025-02281-4#ref-CR10 "Cao, W. et al. Suppressing chlorophyll degradation by silencing OsNYC3 improves rice resistance to Rhizoctonia solani, the causal agent of sheath blight. Plant Biotechnol. J. 20, 335–349 (2022)."). The isolate was first grown on potato dextrose agar medium at 28 °C for 3 days. The fungal blocks (0.7 cm in diameter) were then transferred to potato dextrose broth medium containing wood veneers with a thickness of 0.8 mm and a length of 1.0 cm and grown at 28 °C until the hyphae twined around the wood veneers. The wood veneers colonized with hyphae were used as inoculum.

For inoculation in the field, the inoculum was inserted into the third sheath from the top of a plant at the late tillering stage; three main tillers in each plant were inoculated. Disease index was recorded 30 days after heading with more than 30 sheaths each genotype using a 0–9 disease scoring system[13](https://www.nature.com/articles/s41588-025-02281-4#ref-CR13 "Zuo, S. et al. Fine mapping of qSB-11LE, the QTL that confers partial resistance to rice sheath blight. Theor. Appl. Genet. 126, 1257–1272 (2013)."). For inoculation in greenhouse, plants were grown in plots in a natural environment until the early booting stage, then transferred to a greenhouse with a relative humidity of 75%-85% under long-day conditions (14 h light at 30 °C/10 h darkness at 24 °C) for 2 days before inoculation. Three main tillers in each plant were inoculated using the same method as field inoculation. Lesion lengths were measured 7 or 14 days after inoculation.

### GWAS for rice ShB resistance

GWAS was performed with 178 rice varieties using 109,444 high-quality SNPs with a minor allele frequency of ≥2% and a missing rate of <25% (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02281-4#MOESM4)). Two kinds of univariate GWAS models, CMLM and BLINK, were used to evaluate ShB resistance-SNP associations using the Genomic Association and Prediction Integrated Tool (GAPIT)[30](https://www.nature.com/articles/s41588-025-02281-4#ref-CR30 "Lipka, A. E. et al. GAPIT: genome association and prediction integrated tool. Bioinformatics 28, 2397–2399 (2012)."). The genome-wide significant threshold of GWAS (_P_ = 3.2 × 10−5) was determined by the minimum Bayes factor calculated using the following formula: minimum Bayes factor = −e∗_P_∗ln(_P_)[31](https://www.nature.com/articles/s41588-025-02281-4#ref-CR31 "Goodman, S. N. Of p-values and Bayes: a modest proposal. Epidemiology 12, 295–297 (2001)."). The Manhattan and QQ plots for GWAS were generated using the R package CMplot[32](https://www.nature.com/articles/s41588-025-02281-4#ref-CR32 "Yin, L. et al. rMVP: a memory-efficient, visualization-enhanced, and parallel-accelerated tool for genome-wide association study. Genom. Proteom. Bioinf. 19, 619–628 (2021)."). LD blocks were defined with the Solid Spine method, and the LD heatmap was constructed using the R package “LDBlockShow”[33](https://www.nature.com/articles/s41588-025-02281-4#ref-CR33 "Dong, S. et al. LDBlockShow: a fast and convenient tool for visualizing linkage disequilibrium and haplotype blocks based on variant call format files. Brief. Bioinform. 22, bbaa227 (2021).").

### Luciferase reporter assays

The 3,123 bp promoter regions of _SBRR1_ were amplified from ZD88 (a _SBRR1-S_ variety) and XWX7 (a _SBRR1-R_ variety) individually. These two fragments and different deleted versions derived from were cloned into the pGreenII 0800-LUC vector; _Renilla luciferase_ was used as an internal control[34](https://www.nature.com/articles/s41588-025-02281-4#ref-CR34 "Hellens, R. P. et al. Transient expression vectors for functional genomics, quantification of promoter activity and RNA silencing in plants. Plant Methods 1, 13 (2005)."). These two vectors were used to transfect rice protoplasts via PEG-mediated transformation[35](https://www.nature.com/articles/s41588-025-02281-4#ref-CR35 "Chern, M. et al. A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1mediated transcriptional activation. Plant Methods 8, 6 (2012)."). Luciferase activity was measured using the Dual-Luciferase reporter assay system (Promega, E1910). Ratios of LUC to Ren activity were calculated to determine relative promoter activity. Six biological replicates were included. The primers used are listed in Supplementary Table [13](https://www.nature.com/articles/s41588-025-02281-4#MOESM4).

### Nucleotide diversity and evolutionary analyses

From the rice XWX7 reference genome, a DNA sequence containing the 256 bp Indel-946 marker and 2,000 bp upstream and 2,000 bp downstream regions (a total of 4,256 bp) was extracted as the reference sequence, named Sequence1. The second-generation sequencing data of all 430 varieties with Illumina sequencing depth >20× from the 3 K rice population were aligned to Sequence1 using the Bowtie2 software. The results were analyzed using Samtools, and a Perl script was used to count the number of reads successfully covering the 256 bp endpoints in the sequencing data of the 430 varieties. Based on this comprehensive analysis, whether these varieties contain the indel-946 sequence was determined. Simultaneously, the Bowtie2 sequence alignment, Samtools, and Perl script analysis were used to determine the SNP type of the SNPG/A site for the 430 varieties. By comparing the indel-946 marker type and the SNPG/A marker type of the 430 varieties, it was determined whether these two markers are closely linked.

The genomic sequences of 26 wild rice and 2,478 cultivated rice accessions were obtained from OryzaGenome ([http://viewer.shigen.info/oryzagenome/](http://viewer.shigen.info/oryzagenome/))[36](https://www.nature.com/articles/s41588-025-02281-4#ref-CR36 "Ohyanagi, H., Obayashi, T. & Yano, K. Plant and Cell Physiology’s 2016 Online Database Issue. Plant Cell Physiol. 57, 1–3 (2016).") and the Rice Functional Genomics and Breeding Database ([http://www.rmbreeding.cn/Snp3k](http://www.rmbreeding.cn/Snp3k))[37](https://www.nature.com/articles/s41588-025-02281-4#ref-CR37 "Sun, C. et al. RPAN: rice pan-genome browser for ~3000 rice genomes. Nucl. Acids Res. 45, 597–605 (2017)."), respectively. The distributions of _SBRR1-R_ and _SBRR1-S_ in the two rice subspecies were compared using the chi-square test. The geographical information of cultivated varieties was obtained from the Rice Functional Genomics and Breeding Database and marked on map using _Cartopy_ package v0.20.0 (ref. [38](https://www.nature.com/articles/s41588-025-02281-4#ref-CR38 "Elson, P. et al. SciTools/cartopy: v0.22.0. Zenodo (2023).")) in Python v3.6.0 to observe geographic distribution of _SBRR1-R_ and _SBRR1-S_. The map layer of precipitation from 1970 to 2000 ad was obtained from WorldClim[39](https://www.nature.com/articles/s41588-025-02281-4#ref-CR39 "Fick, S. E. & Hijmans, R. J. WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas. Int. J. Climatol. 37, 4302–4315 (2017)."). The average _F__ST_ values in each 400-kb window were estimated at chromosome 11 between _indica_ and _japonica_ subspecies using VCFtools v0.1.16 (ref. [40](https://www.nature.com/articles/s41588-025-02281-4#ref-CR40 "Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011).")). The nucleotide diversity (_π_) and Neutral test (Tajima’s _D_) of each population were calculated in 400-kb window using VCFtools v0.1.16 (ref. [40](https://www.nature.com/articles/s41588-025-02281-4#ref-CR40 "Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011).")). Phylogenetic trees and haplotype networks were constructed for all sites with minor allele frequency ≥0.01 of the coding sequences of _SBRR1_ and their upstream 1.5 kb. The phylogenetic tree for _SBRR1_ was inferred using the UPGMA method in MEGA v11 (ref. [41](https://www.nature.com/articles/s41588-025-02281-4#ref-CR41 "Koichiro, T., Glen, S. & Sudhir, K. MEGA11: Molecular evolutionary genetics analysis version 11. Mol. Biol. Evol. 38, 3022–3027 (2021).")). The haplotype network was calculated using _pegas_ package v1.2 (ref. [42](https://www.nature.com/articles/s41588-025-02281-4#ref-CR42 "Paradis, E. pegas: an R package for population genetics with an integrated-modular approach. Bioinformatics 26, 419–420 (2010).")) in R v4.1.2 software. The haplotypes that contained more than 3 rice accessions were displayed using plotting module matplotlib v3.6.0 (ref. [43](https://www.nature.com/articles/s41588-025-02281-4#ref-CR43 "Hunter, J. D. Matplotlib: a 2D graphics environment. Comput. Sci. Eng. 9, 90–95 (2007).")) in Python v3.6.0.

### Field trial for rice grain yield and quality analysis

Rice lines TG394, DJ, TG394-_SBRR1__R_ and _SBRR1_\-OE1 were included in field trial for yield evaluation. TG394-_SBRR1__R_, carrying the _SBRR1-R_ elite allele, is a near-isogenic line of TG394. _SBRR1_\-OE1 is a transgenic overexpression line in DJ background. Two ShB disease conditions/treatments (slight disease and severe disease) were tested. The slight disease condition was conducted by spraying thifluzamide (a fungicide against _R. solani_) at tillering, jointing and booting stages to suppress ShB disease occurrence. The severe disease condition was performed via artificial inoculation with _R. solani_ on six tillers per plant to ensure severe disease development. The inoculation procedure was the same as described above. For TG394 and TG394-_SBRR1__R_ rice lines, each treatment consisted of six replications/plots, and each plot was 2.37 m long and 1.38 m wide. For DJ and _SBRR1_\-OE1, each treatment consisted of five replications/plots with the same plot size as above. Each plot comprised 10 rows with 12 plants per row (11.55 cm × 23.76 cm rowing space). To reduce marginal effects, only the six inside rows with middle eight plants per row (48 plants per plot covering around 1.32 m2) were disease-rated and collected for grain yield and quality measurement. Grain yield per plot, seed setting rate, 1,000-grain weight, panicle number, grain number per panicle, plant height, growth period, chalkiness rate and amylose content were measured for all plot samples.

### In vitro phosphorylation assay

To compare the autophosphorylation activities of SBRR1-ICD (ICD, intracellular domain), SBRR1-ICDK553E, SBRR1-ICDS678A and SBRR1-ICDTT682/683AA, their coding sequences were individually ligated into the pET28a vector to generate corresponding plasmids. Fusion proteins were expressed in _Escherichia coli_ strain _BL21_ and purified using Ni-Sepharose beads (GE Healthcare, GE17-5268-01). Phosphorylation assays were performed by adding beads containing purified fusion proteins in 60 μl kinase buffer (10 mM ATP, 20 mM MgCl2, 20 mM MnCl2, 40 mM HEPES (pH 7.5), 2 mM DTT, phosphatase inhibitor cocktail and proteinase inhibitor cocktail) and incubating for 90 min at 30 °C. The samples were subsequently separated on SDS-PAGE gels and transferred to nitrocellulose membranes, followed by immunoblot analysis with a pSer/Thr antibody (ECM Biosciences, PP2551; 1:1,000 in bovine serum albumin). Horseradish peroxidase (HRP)-conjugated anti-rabbit IgG (EASYBIO, BE0101; 1:3,000 in bovine serum albumin) was used as the secondary antibody. Primer sequences used are listed in Supplementary Table [13](https://www.nature.com/articles/s41588-025-02281-4#MOESM4).

### Mycelia-containing cellulose acetate membrane inoculation

To collect large amounts of inoculated rice tissues with similar disease progression, a new inoculation method, named mycelia-containing cellulose acetate membrane inoculation, was developed. Briefly, a fungal block (0.7 cm in diameter) was transferred to a cellulose acetate membrane placed on potato dextrose broth medium and grown at 28 °C until _R. solani_ hyphae spread throughout the entire membrane. The cellulose acetate membrane colonized with hyphae was used to tightly cover 14-day-old _SBRR1_\-OE1 seedlings lying flat on a tray. The roots of the seedlings were wrapped with damp gauze to keep them moist. A blank cellulose acetate membrane without _R. solani_ served as a control. Finally, the entire setup was placed in an incubator with 80%-95% relative humidity under long-day conditions (14 h light at 30 °C/10 h darkness at 28 °C). After inoculation, most plant tissues that were close to the cellulose acetate membrane showed similar disease progression and symptoms, which greatly facilitated the collection of large amounts of inoculated rice tissues at similar disease degree.

### In vivo phosphorylation assay

To analyze the phosphorylation status of SBRR1 in vivo, protein samples were extracted from 14-day-old _SBRR1_\-OE1 seedlings at different time points after _R. solani_ inoculation and separated in SDS-PAGE gels with or without 50 μM Phos-tag acrylamide (NARD Institute, AAL-107) and 100 μM MnCl2. The samples were then transferred to NC membranes, followed by immunoblot analysis with an anti-FLAG monoclonal antibody (Sigma-Aldrich, F3165; 1:5,000 in PBS). HRP-conjugated anti-mouse IgG (EASYBIO, BE0102; 1:3,000 in PBS) were used as the secondary antibodies. The ImageJ 1.53c ([https://imagej.net/ij/ij/download.html](https://imagej.net/ij/ij/download.html)) was used to evaluate the gray value of western blotting.

### LC-MS/MS analysis for phosphorylation sites

For identification of SBRR1 phosphorylation sites by LC-MS/MS, anti-FLAG agarose (Sigma-Aldrich, A4596) was used to immunoprecipitate SBRR1-FLAG protein from _SBRR1_\-OE1 plants at 18 HPI with _R. solani_ or mock (inoculation with water). Denatured proteins were separated in 10% SDS-PAGE and followed by trypsin digestion overnight before LC-MS/MS analysis (Beijing Biotech-Pack Scientific). LC-MS/MS analysis was performed using a Q-Exactive mass spectrometer (Thermo Fisher Scientific) with a nanoflow HPLC instrument (Thermo Fisher Scientific, EASY-nLC1200 system). Phosphorylated peptides were identified and quantified based on the extracted ion chromatogram peak area using MaxQuant software (1.6.2.10). The intensity ratios at phosphorylation sites represent the normalized intensity of the phosphorylated peptide divided by the normalized intensity of the same but un-phosphorylated peptide. Three biological repeats were performed.

### One-hybrid assays in yeast

The full-length cDNA sequence of transcription factor gene _bHLH57_ was amplified and fused in frame with the GAL4 activation domain in pGADT7-Rec2 (Clontech), forming pGADT7-bHLH57. Then, the fusion construct was used with the reporter plasmid (containing _SBRR1-R_ promoter fused to HIS2) to co-transform Y187 yeast cells (Clontech). Sequences of the primers are listed in Supplementary Table [13](https://www.nature.com/articles/s41588-025-02281-4#MOESM4). DNA-protein interaction was determined by the growth of the transformants on the nutrient-deficient medium with 50 mM 3-amino-1,2,4-triazole (3-AT), following the manufacturer’s manual (Clontech).

### EMSA

For double-stranded DNA probes, equal amounts of forward and reverse strands, whose 5′-ends were labeled with biotin, were mixed, boiled then slowly cooled to 25 °C. Different amounts of unlabeled probes (20-, 100- and 200-fold over labeled probes) were incubated with biotin-labeled probes at 28 °C in binding buffer for 20 min using an EMSA kit (Thermo Fisher Scientific, 20148 Light Shift Chemiluminescent EMSA). The samples were separated by nondenaturing electrophoresis using 6% native acrylamide gels in 0.5 × Tris-borate-EDTA (TBE) buffer under 110 V for 1.5 h at 4 °C. Then DNA probes were transferred to a nylon membrane (Beyotime, FFN10) in 0.5 × TBE by running at 380 mA for 45 min at 4 °C. The biotin-labeled double-stranded DNA on nylon membrane was detected by chemiluminescent assays according to the manufacturer’s instructions (Thermo Fisher Scientific, 89880 Chemiluminescent Nucleic acid detection module kit).

### ChIP-qPCR

Three-week-old seedlings of TG394-_SBRR1__R_/_bHLH57_\-OE1 transgenic rice were fully ground into a powder with liquid nitrogen and suspended in buffer to perform ChIP assay. After protein-DNA cross-linking with formaldehyde, nuclei were isolated and chromatin DNA was broken into 200- to 1,000-bp fragments by sonication. The DNA-protein complexes were immunoprecipitated with an anti-FLAG monoclonal antibody (Sigma-Aldrich, F3165; 1:200 in PBS) and enriched using immunomagnetic beads. The collected complexes were de-crosslinked to release the DNA fragments which were used for subsequent qPCR analysis following purification. FLAG monoclonal antibody conjugated to magnetic beads was used to immunoprecipitate protein-DNA complex. Precipitated DNA fragments were analyzed by qPCR to evaluate the enrichment. _OsACTIN_ was used as a negative control. Primer sequences for this experiment are provided in Supplementary Table [13](https://www.nature.com/articles/s41588-025-02281-4#MOESM4).

### Protein interaction analysis

The Y2H assay was conducted to screen for interacting proteins of SBRR1 and analyze their interaction regions. The different truncated deletions of _SBRR1_ and _SIP1_ were cloned into pGBKT7 and pGADT7 vectors, respectively. Yeast transformation and screening process were performed according to the manufacturer’s instructions (Clontech).

For the GST pull-down assay, the _SBRR1-ECD_ (ECD, extracellular domain) and full-length _SIP1_ coding sequence fragments were inserted into pET28a and pGEX4T-2 vectors, respectively. His- and GST-tagged recombinant proteins were expressed in _BL21_ cells and purified using Ni-Sepharose beads (GE Healthcare, GE17-5268-01) and glutathione beads (GE Healthcare, GE17-0756-04), respectively. GST-SIP1- or GST-coupled beads were used to capture His-SBRR1-ECD. Proteins were separated on SDS-PAGE gels and immunoblotted with an antibody against GST (ABclonal, AE027; 1:3,000 in PBS) or His (ABclonal, AE028; 1:3,000 in PBS).

For Co-IP assay, total proteins were extracted from leaves of 14-day-old _SBRR1_\-OE1 (Ubi-SBRR1-3FLAG) and Ubi-3FLAG (negative control) transgenic rice plants using protein extraction buffer (50 mM HEPES (pH 7.5), 150 mM KCl, 1 mM DTT, 0.4% Triton-X 100 and proteinase inhibitor cocktail). Anti-FLAG agarose (Sigma-Aldrich, A4596) was used to immunoprecipitate the SBRR1 complex, and the complex was detected with FLAG monoclonal antibody (Sigma-Aldrich, F3165; 1:5,000 in PBS) or SIP1 (Beijing Protein Innovation; 1:500 in PBS). HRP-conjugated anti-mouse IgG (EASYBIO, BE0102; 1:3,000 in PBS) were used as the secondary antibodies. Primers used above are listed in Supplementary Table [13](https://www.nature.com/articles/s41588-025-02281-4#MOESM4).

### Subcellular localization

Full-length _SBRR1_ cDNA, mutant _SBRR1__ΔPAN_ and _SIP1_ cDNA were individually cloned into the PAN580-GFP vector to fuse with GFP at the N-terminus of GFP. The SBRR1-GFP, SIP1-GFP and SBRR1ΔPAN\-GFP constructs were separately used to co-transform rice protoplasts with PM marker (PIP2;1-mCherry)[44](https://www.nature.com/articles/s41588-025-02281-4#ref-CR44 "Lee, H. K. et al. Drought stress-induced Rma1H1, a RING membrane-anchor E3 ubiquitin ligase homolog, regulates aquaporin levels via ubiquitination in transgenic Arabidopsis plants. Plant Cell 21, 622–641 (2009).") or ER marker (mCherry ER-Rkcd3-959)[45](https://www.nature.com/articles/s41588-025-02281-4#ref-CR45 "Nelson, B. K., Cai, X. & Nebenfuhr, A. A multicolored set of in vivo organelle markers for co-localization studies in Arabidopsis and other plants. Plant J. 51, 1126–1136 (2007)."). Meanwhile, GFP was also fused to the C-terminus of SBRR1 and SIP1 in the pCAMBIA1305.1 vector, then introduced into _N. benthamiana_ leaves. Fluorescence images were observed using a laser scanning confocal microscope (ZEISS LSM 700). All primer sequences used are listed in Supplementary Table [13](https://www.nature.com/articles/s41588-025-02281-4#MOESM4).

### RNA-seq and functional classification of DEGs

RNA-seq was used to compare transcriptomic profiles between WT and _sbrr1-ko1_ or _sip1-ko1_ before and after inoculation with _R. solani_. The screening criteria for DEGs were fold change = 2 and _P_ < 0.05. Gene ontology analysis was performed using DAVID Resources 6.7 ([http://david.abcc.ncifcrf.gov/](http://david.abcc.ncifcrf.gov/))[46](https://www.nature.com/articles/s41588-025-02281-4#ref-CR46 "Wei, H., Sherman, B. & Lempicki, R. Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. Nucl. Acids Res. 37, 1–13 (2009).").

### Determination of chitinase activity

Chitinase activities of leaf sheath samples collected at different times after _R. solani_ inoculation were measured by a colorimetric method[47](https://www.nature.com/articles/s41588-025-02281-4#ref-CR47 "Boller, T., Gehri, A., Mauch, F. & Vogeli, U. Chitinase in bean leaves: induction by ethylene, purification, properties, and possible function. Planta 157, 22–31 (1983)."). Briefly, to 0.25 g rice sample was added 2 ml of 0.05 M acetic acid buffer (pH 5.0), and the sample was centrifuged at 16,099_g_ at 4 °C for 15 min. Each reaction mixture of 1.2 ml contained 0.4 ml crude enzyme extract, 0.4 ml of 0.05 M acetic acid buffer (pH 5.0) and 0.4 ml of 1% chitin solution. After incubation at 37 °C for 1 h, the reaction mixture was immediately centrifuged at 2,795_g_ for 30 min at 4 °C. Then 0.2 ml saturated borax solution were added to 0.4 ml supernatant, and the mixture was boiled for 7 min. After cooling, 2 ml glacial acetic acid and 1 ml of 1% _P_\-dimethylaminobenzaldehyde solution were added. After incubation at 37 °C for 15 min, the absorbance value at 585 nm was measured. One enzyme activity unit is defined as the amount of 1 mg _N_\-acetylglucosamine produced from chitin decomposition per gram of tissue per hour.

### Inclusion and ethics

Data of this study exclusively come from rice, without involving any animal experiments. Planting of transgenic materials and artificial inoculation are subject to strict regulation. All experimental data are included in the Data availability section.

### Statistics and reproducibility

All quantitative data are presented as mean ± s.d. from at least three independent experiments. Sample sizes (_n_) and _P_ values are shown in each figure and figure legend. IBM SPSS Statistics (v.23) was used for statistical analysis. Statistical differences between two groups were analyzed by two-sided Student’s _t_\-test. Statistical significances between more than two groups were analyzed by ANOVA with Duncan’s multiple range test. Results with a _P_ value < 0.05 were considered statistically significant. Luciferase reporter assays, Y1H assays, phosphorylation assays, EMSAs, Y2H assays, Co-IP, GST pull-down, subcellular localization, confocal microscopy and two-phase partitioning were done independently at least three times with similar results. No statistical method was used to predetermine the sample size, but our sample sizes are similar to those reported in previous publications[23](https://www.nature.com/articles/s41588-025-02281-4#ref-CR23 "Gao, M. et al. Ca2+ sensor-mediated ROS scavenging suppresses rice immunity and is exploited by a fungal effector. Cell 184, 5391–5404 (2021)."),[48](https://www.nature.com/articles/s41588-025-02281-4#ref-CR48 "Zhu, M. et al. The ZmCPK39–ZmDi19–ZmPR10 immune module regulates quantitative resistance to multiple foliar diseases in maize. Nat. Genet. 56, 2815–2826 (2024)."). No data were excluded from the analyses. Data distribution was assumed to be normal, but this was not formally tested.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02281-4#MOESM2) linked to this article.

Data availability
-----------------

The data supporting the findings of this study are available in the article and its [Supplementary Information](https://www.nature.com/articles/s41588-025-02281-4#MOESM1) files. The data of 109,444 high-quality SNPs covering the whole rice genome of 178 commercial rice cultivars used in this study and GWAS statistics are publicly available from figshare repository ([https://doi.org/10.6084/m9.figshare.25265239.v2](https://doi.org/10.6084/m9.figshare.25265239.v2)). Genomic DNA sequences for _SBRR1_ from the cultivars XWX7, DJ, YSBR1 and ZD88 can be found in the National Center for Biotechnology Information (NCBI) GenBank under accessions [PV423243](https://www.ncbi.nlm.nih.gov/nuccore/PV423243), [PV423242](https://www.ncbi.nlm.nih.gov/nuccore/PV423242), [PV423244](https://www.ncbi.nlm.nih.gov/nuccore/PV423244) and [PV423245](https://www.ncbi.nlm.nih.gov/nuccore/PV423245), respectively. The phosphorylation MS data has been submitted to figshare repository ([https://doi.org/10.6084/m9.figshare.27939462.v1](https://doi.org/10.6084/m9.figshare.27939462.v1)). The RNA-seq data have been deposited in the NCBI Sequence Read Archive under accession code [PRJNA953858](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA953858). [Source data](https://www.nature.com/articles/s41588-025-02281-4#Sec35) are provided with this paper.

Code availability
-----------------

All software used in this study is publicly available, as described in [Methods](https://www.nature.com/articles/s41588-025-02281-4#Sec11) and [Reporting Summary](https://www.nature.com/articles/s41588-025-02281-4#Sec31).

References
----------

1.  Zhen, A. et al. The evolution and pathogenic mechanisms of the rice sheath blight pathogen. _Nat. Commun._ **4**, 1424 (2013).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20evolution%20and%20pathogenic%20mechanisms%20of%20the%20rice%20sheath%20blight%20pathogen&journal=Nat.%20Commun.&volume=4&publication_year=2013&author=Zhen%2CA) 
    
2.  Okubara, P. A., Dickman, M. B. & Blechl, A. E. Molecular and genetic aspects of controlling the soilborne necrotrophic pathogens _Rhizoctonia_ and _Pythium_. _Plant Sci._ **228**, 61–70 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXjsVWlsr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25438786)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20and%20genetic%20aspects%20of%20controlling%20the%20soilborne%20necrotrophic%20pathogens%20Rhizoctonia%20and%20Pythium&journal=Plant%20Sci.&volume=228&pages=61-70&publication_year=2014&author=Okubara%2CPA&author=Dickman%2CMB&author=Blechl%2CAE) 
    
3.  Marchetti, M. A. & Bollich, C. N. Quantification of the relationship between sheath blight severity and yield loss in rice. _Plant Dis._ **75**, 773–775 (1991).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantification%20of%20the%20relationship%20between%20sheath%20blight%20severity%20and%20yield%20loss%20in%20rice&journal=Plant%20Dis.&volume=75&pages=773-775&publication_year=1991&author=Marchetti%2CMA&author=Bollich%2CCN) 
    
4.  Lee, F. N. & Rush, M. C. Rice sheath blight: a major rice disease. _Plant Dis._ **67**, 829–832 (1983).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rice%20sheath%20blight%3A%20a%20major%20rice%20disease&journal=Plant%20Dis.&volume=67&pages=829-832&publication_year=1983&author=Lee%2CFN&author=Rush%2CMC) 
    
5.  Senapati, M. et al. _Rhizoctonia solani_ Kühn pathophysiology: status and prospects of sheath blight disease management in rice. _Front. Plant Sci._ **3**, 88116 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rhizoctonia%20solani%20K%C3%BChn%20pathophysiology%3A%20status%20and%20prospects%20of%20sheath%20blight%20disease%20management%20in%20rice&journal=Front.%20Plant%20Sci.&volume=3&publication_year=2022&author=Senapati%2CM) 
    
6.  Singh, P., Mazumdar, P., Harikrishna, J. A. & Babu, S. Sheath blight of rice: a review and identification of priorities for future research. _Planta_ **250**, 1387–1407 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVChu7%2FE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31346804)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sheath%20blight%20of%20rice%3A%20a%20review%20and%20identification%20of%20priorities%20for%20future%20research&journal=Planta&volume=250&pages=1387-1407&publication_year=2019&author=Singh%2CP&author=Mazumdar%2CP&author=Harikrishna%2CJA&author=Babu%2CS) 
    
7.  Chen, Z. et al. Identification of new resistance loci against sheath blight disease in rice through genome-wide association study. _Rice Sci._ **26**, 21–31 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20new%20resistance%20loci%20against%20sheath%20blight%20disease%20in%20rice%20through%20genome-wide%20association%20study&journal=Rice%20Sci.&volume=26&pages=21-31&publication_year=2019&author=Chen%2CZ) 
    
8.  Zhang, F. et al. Genome-wide association analysis of the genetic basis for sheath blight resistance in rice. _Rice_ **12**, 93 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31853678)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6920286)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20analysis%20of%20the%20genetic%20basis%20for%20sheath%20blight%20resistance%20in%20rice&journal=Rice&volume=12&publication_year=2019&author=Zhang%2CF) 
    
9.  Molla, K. A. et al. Understanding sheath blight resistance in rice: the road behind and the road ahead. _Plant Biotechnol. J._ **18**, 895–915 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31811745)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7061877)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%20sheath%20blight%20resistance%20in%20rice%3A%20the%20road%20behind%20and%20the%20road%20ahead&journal=Plant%20Biotechnol.%20J.&volume=18&pages=895-915&publication_year=2020&author=Molla%2CKA) 
    
10.  Cao, W. et al. Suppressing chlorophyll degradation by silencing _OsNYC3_ improves rice resistance to _Rhizoctonia solani_, the causal agent of sheath blight. _Plant Biotechnol. J._ **20**, 335–349 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1OrsLjE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34582620)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Suppressing%20chlorophyll%20degradation%20by%20silencing%20OsNYC3%20improves%20rice%20resistance%20to%20Rhizoctonia%20solani%2C%20the%20causal%20agent%20of%20sheath%20blight&journal=Plant%20Biotechnol.%20J.&volume=20&pages=335-349&publication_year=2022&author=Cao%2CW) 
    
11.  Jia, L. et al. Searching for germplasm resistant to sheath blight from the USDA rice core collection. _Crop Sci._ **51**, 1507–1517 (2011).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Searching%20for%20germplasm%20resistant%20to%20sheath%20blight%20from%20the%20USDA%20rice%20core%20collection&journal=Crop%20Sci.&volume=51&pages=1507-1517&publication_year=2011&author=Jia%2CL) 
    
12.  Wang, Y. et al. Fine mapping and candidate gene analysis of _qSB12__YSB_, a gene conferring major quantitative resistance to rice sheath blight. _Theor. Appl. Genet._ **136**, 246 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFWqsL%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37973669)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fine%20mapping%20and%20candidate%20gene%20analysis%20of%20qSB12YSB%2C%20a%20gene%20conferring%20major%20quantitative%20resistance%20to%20rice%20sheath%20blight&journal=Theor.%20Appl.%20Genet.&volume=136&publication_year=2023&author=Wang%2CY) 
    
13.  Zuo, S. et al. Fine mapping of _qSB-11__LE_, the QTL that confers partial resistance to rice sheath blight. _Theor. Appl. Genet._ **126**, 1257–1272 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23423653)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fine%20mapping%20of%20qSB-11LE%2C%20the%20QTL%20that%20confers%20partial%20resistance%20to%20rice%20sheath%20blight&journal=Theor.%20Appl.%20Genet.&volume=126&pages=1257-1272&publication_year=2013&author=Zuo%2CS) 
    
14.  Zuo, S. et al. Fine-mapping of _qSB-9__TQ_, a gene conferring major quantitative resistance to rice sheath blight. _Mol. Breed._ **34**, 2191–2203 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslGku73O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fine-mapping%20of%20qSB-9TQ%2C%20a%20gene%20conferring%20major%20quantitative%20resistance%20to%20rice%20sheath%20blight&journal=Mol.%20Breed.&volume=34&pages=2191-2203&publication_year=2014&author=Zuo%2CS) 
    
15.  Eizenga, G. C., Jia, M. H., Pinson, S. R., Gasore, E. R. & Prasad, B. Exploring sheath blight quantitative trait loci in a Lemont/_O. meridionalis_ advanced backcross population. _Mol. Breed._ **35**, 1–19 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXpsVyrsLc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Exploring%20sheath%20blight%20quantitative%20trait%20loci%20in%20a%20Lemont%2FO.%20meridionalis%20advanced%20backcross%20population&journal=Mol.%20Breed.&volume=35&pages=1-19&publication_year=2015&author=Eizenga%2CGC&author=Jia%2CMH&author=Pinson%2CSR&author=Gasore%2CER&author=Prasad%2CB) 
    
16.  Huang, X. et al. Genome-wide association studies of 14 agronomic traits in rice landraces. _Nat. Genet._ **42**, 961–967 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlahu7rE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20972439)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&journal=Nat.%20Genet.&volume=42&pages=961-967&publication_year=2010&author=Huang%2CX) 
    
17.  Chen, R. et al. Rice functional genomics: decades’ efforts and roads ahead. _Sci. China Life Sci._ **65**, 33–92 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34881420)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rice%20functional%20genomics%3A%20decades%E2%80%99%20efforts%20and%20roads%20ahead&journal=Sci.%20China%20Life%20Sci.&volume=65&pages=33-92&publication_year=2022&author=Chen%2CR) 
    
18.  Li, W. et al. A natural allele of a transcription factor in rice confers broad-spectrum blast resistance. _Cell_ **170**, 114–126 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFSqsLfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28666113)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20natural%20allele%20of%20a%20transcription%20factor%20in%20rice%20confers%20broad-spectrum%20blast%20resistance&journal=Cell&volume=170&pages=114-126&publication_year=2017&author=Li%2CW) 
    
19.  Liu, Y. et al. Genomic basis of geographical adaptation to soil nitrogen in rice. _Nature_ **590**, 600–605 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsFWktg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33408412)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20basis%20of%20geographical%20adaptation%20to%20soil%20nitrogen%20in%20rice&journal=Nature&volume=590&pages=600-605&publication_year=2021&author=Liu%2CY) 
    
20.  Wang, A. et al. Identification of rice (_Oryza sativa_ L.) genes involved in sheath blight resistance via a genome-wide association study. _Plant Biotechnol. J._ **19**, 1553–1566 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1Orsr%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33600077)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8384605)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20rice%20%28Oryza%20sativa%20L.%29%20genes%20involved%20in%20sheath%20blight%20resistance%20via%20a%20genome-wide%20association%20study&journal=Plant%20Biotechnol.%20J.&volume=19&pages=1553-1566&publication_year=2021&author=Wang%2CA) 
    
21.  Oreiro, E. G. et al. Genome-wide associations and transcriptional profiling reveal ROS regulation as one underlying mechanism of sheath blight resistance in rice. _Mol. Plant Microbe._ **33**, 212–222 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlGltLg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20associations%20and%20transcriptional%20profiling%20reveal%20ROS%20regulation%20as%20one%20underlying%20mechanism%20of%20sheath%20blight%20resistance%20in%20rice&journal=Mol.%20Plant%20Microbe.&volume=33&pages=212-222&publication_year=2020&author=Oreiro%2CEG) 
    
22.  Li, N. et al. Natural variation in _ZmFBL41_ confers banded leaf and sheath blight resistance in maize. _Nat. Genet._ **51**, 1540–1548 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVOqu7%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31570888)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Natural%20variation%20in%20ZmFBL41%20confers%20banded%20leaf%20and%20sheath%20blight%20resistance%20in%20maize&journal=Nat.%20Genet.&volume=51&pages=1540-1548&publication_year=2019&author=Li%2CN) 
    
23.  Gao, M. et al. Ca2+ sensor-mediated ROS scavenging suppresses rice immunity and is exploited by a fungal effector. _Cell_ **184**, 5391–5404 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFGhtr3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34597584)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ca2%2B%20sensor-mediated%20ROS%20scavenging%20suppresses%20rice%20immunity%20and%20is%20exploited%20by%20a%20fungal%20effector&journal=Cell&volume=184&pages=5391-5404&publication_year=2021&author=Gao%2CM) 
    
24.  Taylor, S. S. et al. Structural framework for the protein kinase family. _Annu. Rev. Cell Biol._ **8**, 429–462 (1992).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXhvVakuw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1335745)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20framework%20for%20the%20protein%20kinase%20family&journal=Annu.%20Rev.%20Cell%20Biol.&volume=8&pages=429-462&publication_year=1992&author=Taylor%2CSS) 
    
25.  Lemmon, M. A. & Schlessinger, J. Cell signaling by receptor tyrosine kinases. _Cell_ **141**, 1117–1134 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXovFartb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20602996)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914105)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell%20signaling%20by%20receptor%20tyrosine%20kinases&journal=Cell&volume=141&pages=1117-1134&publication_year=2010&author=Lemmon%2CMA&author=Schlessinger%2CJ) 
    
26.  Feng, Z. et al. iTRAQ-based quantitative proteomics analysis of defense responses triggered by the pathogen _Rhizoctonia solani_ infection in rice. _J. Integr. Agr._ **21**, 139–152 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtVChsb7I)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=iTRAQ-based%20quantitative%20proteomics%20analysis%20of%20defense%20responses%20triggered%20by%20the%20pathogen%20Rhizoctonia%20solani%20infection%20in%20rice&journal=J.%20Integr.%20Agr.&volume=21&pages=139-152&publication_year=2022&author=Feng%2CZ) 
    
27.  Liu, J. et al. OsbHLH057 targets the AATCA cis‑element to regulate disease resistance and drought tolerance in rice. _Plant Cell Rep._ **41**, 1285–1299 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XntVWms7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35278106)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OsbHLH057%20targets%20the%20AATCA%20cis%E2%80%91element%20to%20regulate%20disease%20resistance%20and%20drought%20tolerance%20in%20rice&journal=Plant%20Cell%20Rep.&volume=41&pages=1285-1299&publication_year=2022&author=Liu%2CJ) 
    
28.  Chen, L. et al. The Hop/Sti1-Hsp90 Chaperone complex facilitates the maturation and transport of a PAMP receptor in rice innate immunity. _Cell Host Microbe_ **7**, 185–196 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlsVWqurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20227662)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Hop%2FSti1-Hsp90%20Chaperone%20complex%20facilitates%20the%20maturation%20and%20transport%20of%20a%20PAMP%20receptor%20in%20rice%20innate%20immunity&journal=Cell%20Host%20Microbe&volume=7&pages=185-196&publication_year=2010&author=Chen%2CL) 
    
29.  Jeon, J. S. et al. T-DNA insertional mutagenesis for functional genomics in rice. _Plant J._ **22**, 561–570 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXltFKksbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10886776)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=T-DNA%20insertional%20mutagenesis%20for%20functional%20genomics%20in%20rice&journal=Plant%20J.&volume=22&pages=561-570&publication_year=2000&author=Jeon%2CJS) 
    
30.  Lipka, A. E. et al. GAPIT: genome association and prediction integrated tool. _Bioinformatics_ **28**, 2397–2399 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlantbrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22796960)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GAPIT%3A%20genome%20association%20and%20prediction%20integrated%20tool&journal=Bioinformatics&volume=28&pages=2397-2399&publication_year=2012&author=Lipka%2CAE) 
    
31.  Goodman, S. N. Of p-values and Bayes: a modest proposal. _Epidemiology_ **12**, 295–297 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD3MvpsVOhtQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11337600)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Of%20p-values%20and%20Bayes%3A%20a%20modest%20proposal&journal=Epidemiology&volume=12&pages=295-297&publication_year=2001&author=Goodman%2CSN) 
    
32.  Yin, L. et al. rMVP: a memory-efficient, visualization-enhanced, and parallel-accelerated tool for genome-wide association study. _Genom. Proteom. Bioinf._ **19**, 619–628 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=rMVP%3A%20a%20memory-efficient%2C%20visualization-enhanced%2C%20and%20parallel-accelerated%20tool%20for%20genome-wide%20association%20study&journal=Genom.%20Proteom.%20Bioinf.&volume=19&pages=619-628&publication_year=2021&author=Yin%2CL) 
    
33.  Dong, S. et al. LDBlockShow: a fast and convenient tool for visualizing linkage disequilibrium and haplotype blocks based on variant call format files. _Brief. Bioinform._ **22**, bbaa227 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33126247)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LDBlockShow%3A%20a%20fast%20and%20convenient%20tool%20for%20visualizing%20linkage%20disequilibrium%20and%20haplotype%20blocks%20based%20on%20variant%20call%20format%20files&journal=Brief.%20Bioinform.&volume=22&publication_year=2021&author=Dong%2CS) 
    
34.  Hellens, R. P. et al. Transient expression vectors for functional genomics, quantification of promoter activity and RNA silencing in plants. _Plant Methods_ **1**, 13 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16359558)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1334188)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transient%20expression%20vectors%20for%20functional%20genomics%2C%20quantification%20of%20promoter%20activity%20and%20RNA%20silencing%20in%20plants&journal=Plant%20Methods&volume=1&publication_year=2005&author=Hellens%2CRP) 
    
35.  Chern, M. et al. A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1mediated transcriptional activation. _Plant Methods_ **8**, 6 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmsF2qurY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22353606)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3297495)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20rice%20transient%20assay%20system%20identifies%20a%20novel%20domain%20in%20NRR%20required%20for%20interaction%20with%20NH1%2FOsNPR1%20and%20inhibition%20of%20NH1mediated%20transcriptional%20activation&journal=Plant%20Methods&volume=8&publication_year=2012&author=Chern%2CM) 
    
36.  Ohyanagi, H., Obayashi, T. & Yano, K. _Plant and Cell Physiology_’s 2016 Online Database Issue. _Plant Cell Physiol._ **57**, 1–3 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs12is7vK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26801748)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plant%20and%20Cell%20Physiology%E2%80%99s%202016%20Online%20Database%20Issue&journal=Plant%20Cell%20Physiol.&volume=57&pages=1-3&publication_year=2016&author=Ohyanagi%2CH&author=Obayashi%2CT&author=Yano%2CK) 
    
37.  Sun, C. et al. RPAN: rice pan-genome browser for ~3000 rice genomes. _Nucl. Acids Res._ **45**, 597–605 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXivFais7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27940610)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RPAN%3A%20rice%20pan-genome%20browser%20for%20~3000%20rice%20genomes&journal=Nucl.%20Acids%20Res.&volume=45&pages=597-605&publication_year=2017&author=Sun%2CC) 
    
38.  Elson, P. et al. SciTools/cartopy: v0.22.0. Zenodo (2023).
    
39.  Fick, S. E. & Hijmans, R. J. WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas. _Int. J. Climatol._ **37**, 4302–4315 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WorldClim%202%3A%20new%201-km%20spatial%20resolution%20climate%20surfaces%20for%20global%20land%20areas&journal=Int.%20J.%20Climatol.&volume=37&pages=4302-4315&publication_year=2017&author=Fick%2CSE&author=Hijmans%2CRJ) 
    
40.  Danecek, P. et al. The variant call format and VCFtools. _Bioinformatics_ **27**, 2156–2158 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXptFCqt7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21653522)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137218)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20variant%20call%20format%20and%20VCFtools&journal=Bioinformatics&volume=27&pages=2156-2158&publication_year=2011&author=Danecek%2CP) 
    
41.  Koichiro, T., Glen, S. & Sudhir, K. MEGA11: Molecular evolutionary genetics analysis version 11. _Mol. Biol. Evol._ **38**, 3022–3027 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MEGA11%3A%20Molecular%20evolutionary%20genetics%20analysis%20version%2011&journal=Mol.%20Biol.%20Evol.&volume=38&pages=3022-3027&publication_year=2021&author=Koichiro%2CT&author=Glen%2CS&author=Sudhir%2CK) 
    
42.  Paradis, E. pegas: an R package for population genetics with an integrated-modular approach. _Bioinformatics_ **26**, 419–420 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhs1Onsrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20080509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=pegas%3A%20an%20R%20package%20for%20population%20genetics%20with%20an%20integrated-modular%20approach&journal=Bioinformatics&volume=26&pages=419-420&publication_year=2010&author=Paradis%2CE) 
    
43.  Hunter, J. D. Matplotlib: a 2D graphics environment. _Comput. Sci. Eng._ **9**, 90–95 (2007).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Matplotlib%3A%20a%202D%20graphics%20environment&journal=Comput.%20Sci.%20Eng.&volume=9&pages=90-95&publication_year=2007&author=Hunter%2CJD) 
    
44.  Lee, H. K. et al. Drought stress-induced _Rma1H1_, a RING membrane-anchor E3 ubiquitin ligase homolog, regulates aquaporin levels via ubiquitination in transgenic Arabidopsis plants. _Plant Cell_ **21**, 622–641 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXksVKrt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19234086)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2660634)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Drought%20stress-induced%20Rma1H1%2C%20a%20RING%20membrane-anchor%20E3%20ubiquitin%20ligase%20homolog%2C%20regulates%20aquaporin%20levels%20via%20ubiquitination%20in%20transgenic%20Arabidopsis%20plants&journal=Plant%20Cell&volume=21&pages=622-641&publication_year=2009&author=Lee%2CHK) 
    
45.  Nelson, B. K., Cai, X. & Nebenfuhr, A. A multicolored set of in vivo organelle markers for co-localization studies in _Arabidopsis_ and other plants. _Plant J._ **51**, 1126–1136 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFKnur%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17666025)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20multicolored%20set%20of%20in%20vivo%20organelle%20markers%20for%20co-localization%20studies%20in%20Arabidopsis%20and%20other%20plants&journal=Plant%20J.&volume=51&pages=1126-1136&publication_year=2007&author=Nelson%2CBK&author=Cai%2CX&author=Nebenfuhr%2CA) 
    
46.  Wei, H., Sherman, B. & Lempicki, R. Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. _Nucl. Acids Res._ **37**, 1–13 (2009).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bioinformatics%20enrichment%20tools%3A%20paths%20toward%20the%20comprehensive%20functional%20analysis%20of%20large%20gene%20lists&journal=Nucl.%20Acids%20Res.&volume=37&pages=1-13&publication_year=2009&author=Wei%2CH&author=Sherman%2CB&author=Lempicki%2CR) 
    
47.  Boller, T., Gehri, A., Mauch, F. & Vogeli, U. Chitinase in bean leaves: induction by ethylene, purification, properties, and possible function. _Planta_ **157**, 22–31 (1983).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3sXht1WrtLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24263941)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chitinase%20in%20bean%20leaves%3A%20induction%20by%20ethylene%2C%20purification%2C%20properties%2C%20and%20possible%20function&journal=Planta&volume=157&pages=22-31&publication_year=1983&author=Boller%2CT&author=Gehri%2CA&author=Mauch%2CF&author=Vogeli%2CU) 
    
48.  Zhu, M. et al. The ZmCPK39–ZmDi19–ZmPR10 immune module regulates quantitative resistance to multiple foliar diseases in maize. _Nat. Genet._ **56**, 2815–2826 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXisVartrnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39496881)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11631770)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20ZmCPK39%E2%80%93ZmDi19%E2%80%93ZmPR10%20immune%20module%20regulates%20quantitative%20resistance%20to%20multiple%20foliar%20diseases%20in%20maize&journal=Nat.%20Genet.&volume=56&pages=2815-2826&publication_year=2024&author=Zhu%2CM) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02281-4?format=refman&flavour=references)

Acknowledgments
---------------

We thank M. Chern for extensive editing. This study was supported by the project of National Key R&D Program of China (2024YFD1200600 to S.M.Z.), Zhongshan Biological Breeding Laboratory (ZSBBL-KY2023-01 to S.M.Z.), Seed Industry Revitalization Project of Jiangsu Province (JBGS\[2021\]001 to S.M.Z.), the National Natural Science Foundation of China (31701057 to Z.M.F., U24A20405 to H.X.K. and 32401889 to P.G.), Pinduoduo-China Agricultural University Research Fund (PC2023B02001 to J.L.), the Natural Science Foundation of Jiangsu Province (BK20240497 to P.G.), Open Project of Jiangsu Key Laboratory for Eco-Agricultural Biotechnology around Hongze Lake (HZHLAB2202 to Z.M.F.) and a Project Funded by Priority Academic Program Development of Jiangsu Higher Education Institutions (PAPD to S.M.Z.).

Author information
------------------

Author notes

1.  These authors contributed equally: Zhiming Feng, Peng Gao, Guangda Wang, Houxiang Kang.
    

### Authors and Affiliations

1.  Jiangsu Key Laboratory of Crop Genomics and Molecular Breeding/Zhongshan Biological Breeding Laboratory/Key Laboratory of Plant Functional Genomics of the Ministry of Education, Agricultural College of Yangzhou University, Yangzhou, China
    
    Zhiming Feng, Peng Gao, Guangda Wang, Jianhua Zhao, Wenya Xie, Rujia Chen, Ran Ju, Zhaogen Wei, Huimin Zhang, Jinqiao Zhang, Yafang Zhang, Keming Hu, Quanyi Sun, Yuntao Zhu, Zongxiang Chen, Zefeng Yang & Shimin Zuo
    
2.  Jiangsu Co-Innovation Center for Modern Production Technology of Grain Crops / Jiangsu Key Laboratory of Crop Genetics and Physiology, Yangzhou University, Yangzhou, China
    
    Zhiming Feng, Yafang Zhang, Keming Hu, Xijun Chen, Zongxiang Chen, Zefeng Yang & Shimin Zuo
    
3.  Lixiahe Agricultural Research Institute of Jiangsu Province, Yangzhou, China
    
    Peng Gao
    
4.  State Key Laboratory for Biology of Plant Diseases and Insect Pests / Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing, China
    
    Houxiang Kang & Xuli Wang
    
5.  State Key Laboratory of Crop Gene Resources and Breeding, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, China
    
    Yuntao Zhu & Yehui Xiong
    
6.  Jiangsu Key Laboratory for Eco-Agricultural Biotechnology around Hongze Lake, Huaiyin Normal University, Huai’an, Jiangsu, China
    
    Xi Liu
    
7.  State Key Laboratory of Agrobiotechnology and MOA Key Laboratory for Monitoring and Green Management of Crop Pests, China Agricultural University, Beijing, China
    
    Chao Yang & Jun Liu
    
8.  Department of Plant Pathology, Ohio State University, Columbus, OH, USA
    
    Guo-Liang Wang
    
9.  Ministry of Education Key Laboratory of Molecular and Cellular Biology, Hebei Collaboration Innovation Center for Cell Signaling and Environmental Adaptation, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Sciences, Hebei Normal University, Shijiazhuang, China
    
    Sheng-Wei Zhang
    
10.  Joint International Research Laboratory of Agriculture and Agri-Product Safety, the Ministry of Education of China, Institutes of Agricultural Science and Technology Development, Yangzhou University, Yangzhou, China
    
    Shimin Zuo
    

Authors

1.  Zhiming Feng
    
    [View author publications](https://www.nature.com/search?author=Zhiming%20Feng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zhiming%20Feng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zhiming%20Feng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Peng Gao
    
    [View author publications](https://www.nature.com/search?author=Peng%20Gao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peng%20Gao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peng%20Gao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Guangda Wang
    
    [View author publications](https://www.nature.com/search?author=Guangda%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Guangda%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Guangda%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Houxiang Kang
    
    [View author publications](https://www.nature.com/search?author=Houxiang%20Kang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Houxiang%20Kang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Houxiang%20Kang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Jianhua Zhao
    
    [View author publications](https://www.nature.com/search?author=Jianhua%20Zhao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jianhua%20Zhao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jianhua%20Zhao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Wenya Xie
    
    [View author publications](https://www.nature.com/search?author=Wenya%20Xie)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenya%20Xie) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenya%20Xie%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Rujia Chen
    
    [View author publications](https://www.nature.com/search?author=Rujia%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rujia%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rujia%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Ran Ju
    
    [View author publications](https://www.nature.com/search?author=Ran%20Ju)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ran%20Ju) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ran%20Ju%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Xuli Wang
    
    [View author publications](https://www.nature.com/search?author=Xuli%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xuli%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xuli%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Zhaogen Wei
    
    [View author publications](https://www.nature.com/search?author=Zhaogen%20Wei)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zhaogen%20Wei) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zhaogen%20Wei%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Huimin Zhang
    
    [View author publications](https://www.nature.com/search?author=Huimin%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Huimin%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Huimin%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Jinqiao Zhang
    
    [View author publications](https://www.nature.com/search?author=Jinqiao%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jinqiao%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jinqiao%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Yafang Zhang
    
    [View author publications](https://www.nature.com/search?author=Yafang%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yafang%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yafang%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Keming Hu
    
    [View author publications](https://www.nature.com/search?author=Keming%20Hu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Keming%20Hu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Keming%20Hu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Quanyi Sun
    
    [View author publications](https://www.nature.com/search?author=Quanyi%20Sun)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Quanyi%20Sun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Quanyi%20Sun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Yuntao Zhu
    
    [View author publications](https://www.nature.com/search?author=Yuntao%20Zhu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuntao%20Zhu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuntao%20Zhu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Yehui Xiong
    
    [View author publications](https://www.nature.com/search?author=Yehui%20Xiong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yehui%20Xiong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yehui%20Xiong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Xi Liu
    
    [View author publications](https://www.nature.com/search?author=Xi%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xi%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xi%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Xijun Chen
    
    [View author publications](https://www.nature.com/search?author=Xijun%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xijun%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xijun%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Zongxiang Chen
    
    [View author publications](https://www.nature.com/search?author=Zongxiang%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zongxiang%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zongxiang%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Chao Yang
    
    [View author publications](https://www.nature.com/search?author=Chao%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chao%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chao%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Jun Liu
    
    [View author publications](https://www.nature.com/search?author=Jun%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jun%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jun%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Guo-Liang Wang
    
    [View author publications](https://www.nature.com/search?author=Guo-Liang%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Guo-Liang%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Guo-Liang%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Sheng-Wei Zhang
    
    [View author publications](https://www.nature.com/search?author=Sheng-Wei%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sheng-Wei%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sheng-Wei%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Zefeng Yang
    
    [View author publications](https://www.nature.com/search?author=Zefeng%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zefeng%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zefeng%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Shimin Zuo
    
    [View author publications](https://www.nature.com/search?author=Shimin%20Zuo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shimin%20Zuo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shimin%20Zuo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

S.M.Z., Z.F.Y. and Z.M.F. conceived the project and designed the experiments. S.M.Z. and Z.F.Y. supervised this project. Z.M.F., P.G., G.D.W., H.X.K., J.H.Z., W.Y.X., R.J.C., R.J., X.L.W., Z.G.W., H.M.Z., J.Q.Z., Y.F.Z., K.M.H., Q.Y.S., Y.T. Z., Y.H.X., X.L., X.J.C., Z.X.C. and C.Y. performed the experiments. Z.M.F., P.G., G.D.W., H.X.K., J.H.Z. and R.J.C. participated in the result analysis. S.-W.Z., J.L. and G.-L.W. participated in project discussions and manuscript revision. Z.M.F., P.G., G.D.W., H.X.K. and S.M.Z. wrote the original draft of the manuscript. All authors reviewed and edited the final draft.

### Corresponding authors

Correspondence to [Zefeng Yang](mailto:zfyang@yzu.edu.cn) or [Shimin Zuo](mailto:smzuo@yzu.edu.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Gitta Coaker, Zhiyong Liu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02281-4#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Distribution of disease scores and genetic structure of the 178 tested rice cultivars.](https://www.nature.com/articles/s41588-025-02281-4/figures/9)

**a**, Distribution of disease scores in different rice varieties inoculated with _R. solani_ in the field. HR, highly resistant; R, resistant; MR, moderately resistant; MS, moderately susceptible; S, susceptible; HS, highly susceptible. **b**, Optimal _K_ values calculated by ADMIXTURE. The x-axis shows different values of K. The y-axis indicates cross-validation (CV) error. **c**, Visualization of population structure when K is 2, 3, or 4. **d**, Principal components analysis (PCA) scree plot. The x-axis shows the principal components. The numbers on the dots represent the percentage of the explained variance per principal component. The y-axis shows the eigen value. **e**, PCA scatterplot. PC1 and PC2 refer to the first and second principal components (PC), respectively. Triangles and circles represent _indica_ and _japonica_ varieties, respectively.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM18)

### [Extended Data Fig. 2 Expression of _SBRR1_ in four _SBRR-S_ or _SBRR1-R_ varieties and ShB resistance of T-DNA insertion mutant _sbrr1._](https://www.nature.com/articles/s41588-025-02281-4/figures/10)

**a**, Differences in the promoter region of _SBRR1_ among Dongjin, TG394, XD3 (Xudao 3) and YSBR1. ATG represents the start codon. **b**, Comparison of _SBRR1_ expression among the four varieties before and after _R. solani_ inoculation (n = 3). **c**, T-DNA insertion site in _sbrr1_. P1, P2, and P3 indicate primers used for verification of the insertion. RB (right border) and LB (left border) indicate the right and left borders of T-DNA, respectively. **d**, PCR verification of the insertion site. **e**, Comparison of ShB severity between _sbrr1_ and WT in the field. Data in b and e are presented as means ± s.d. Statistical significances in b and e were determined by a two-sided Student’s _t_ test. Scale bar, 10 cm.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM19)

### [Extended Data Fig. 3 Phosphorylation levels of SBRR1 at different time points post inoculation with _R. solani_ and SBRR1TT682/683AA protein levels in _SBRR1__TT682/683AA_\-OE lines.](https://www.nature.com/articles/s41588-025-02281-4/figures/11)

**a**, Comparison of phosphorylation levels of SBRR1 protein in _SBRR1_\-OE1 plants at different time points post _R. solani_ inoculation. Total protein was separated in SDS-PAGE containing Phos-tag or normal SDS-PAGE. **b**, Immunoblotting assay using a pSer/Thr antibody to detect phosphorylation levels of SBRR1 protein in _SBRR1_\-OE1 line with or without _R. solani_ infection. **c**, Comparison of phosphorylation levels of SBRR1 protein in _SBRR1_\-OE1 and WT plants with or without _R. solani_ infection. Total protein was separated in SDS-PAGE containing Phos-tag acrylamide or normal SDS-PAGE. ACTIN was used as a loading reference. **d**, Immunoblotting assay using a pSer/Thr antibody to detect autophosphorylation of His-SBRR1-ICDS678A. **e**, SBRR1TT682/683AA protein levels in _SBRR1__TT682/683AA_\-OE lines. Phosphatase in b and d indicates calf intestinal alkaline phosphatase. ACTIN in a, b, c and e was used as a loading reference. The _in vitro_ and _in vivo_ phosphorylation assays were done independently three times.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM21)

### [Extended Data Fig. 4 ShB disease scores (a) and main agronomic traits (b) of _SBRR1_ elite promoter (EPro) lines in field tests.](https://www.nature.com/articles/s41588-025-02281-4/figures/12)

Data are presented as means ± s.d (Disease score, n = 72; plant height, panicle number and grain number per panicle, n = 10; 1000-grain weight, n = 3). Different lowercase letters in a indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range tests. N.S. indicates no significant differences. Scale bar, 10 cm.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM22)

### [Extended Data Fig. 5 MAS scheme for transferring _SBRR1-R_, and ShB lesions and main agronomic traits of XD3 and XD3-_SBRR1__R_.](https://www.nature.com/articles/s41588-025-02281-4/figures/13)

**a**, Indel marker Indel-SBRR1-R specific to the _SBRR1-R_ allele. **b**, MAS (marker assisted selection) scheme for introducing _SBRR1-R_ from YSBR1 into temperate _japonica_ cultivars TG394 and XD3. TG394-_SBRR1__R_ and XD3-_SBRR1__R_ represent near isogenic lines harboring _SBRR1-R_ in the background of TG394 and XD3, respectively. **c**, Comparison of ShB lesion length of XD3 and XD3-_SBRR1__R_ measured 14 days after _R. solani_ inoculation in greenhouse (n = 27). Scale bar, 5 cm. **d**, Comparison of plant height (n = 10), whole growth period (n = 10), panicle number (n = 10), grain number per panicle (n = 10), seed setting rate (n = 10) and 1000-grain weight (n = 3) of XD3 and XD3-_SBRR1__R_. Data in c and d are presented as means ± s.d. Statistical significances in c and d were determined by a two-sided Student’s _t_ test.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM23)

### [Extended Data Fig. 6 Overexpression of _SBRR1_ significantly reduces grain yield losses caused by ShB under severe disease condition.](https://www.nature.com/articles/s41588-025-02281-4/figures/14)

**a**, ShB phenotypes of _SBRR1_\-OE1 and WT under severe disease conditions in the field. Scale bar, 10 cm. **b**\-**i**, Comparison of disease score (**b**, n = 60), grain yield (**c**, n = 5), seed setting rate (**d**, n = 11), 1000-grain weight (**e**, n = 11), panicle number (**f**, n = 11), grain number per panicle (**g**, n = 11), plant height (**h**, n = 10), growth period (**i**, n = 10) of _SBRR1_\-OE1 and WT under slight and severe disease conditions in the field. Data in b-i are presented as means ± s.d. Different lowercase letters in b-e indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range tests. N.S. indicates no significant differences.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM25)

### [Extended Data Fig. 7 Subcellular localization and expression pattern of _bHLH57._](https://www.nature.com/articles/s41588-025-02281-4/figures/15)

**a**, Subcellular localization of bHLH57 in WT rice protoplasts. The images were taken under a laser scanning confocal microscope 12 hours after transfection. Scale bar, 10 μm. **b**, RNA levels of _bHLH57_ in different tissues detected by reverse transcription-quantitative PCR (RT-qPCR). Data are presented as means ± s.d (n = 3). The subcellular localization assays were done independently three times.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM26)

### [Extended Data Fig. 8 The PAN domain of SBRR1 and the DP-repeat domain of SIP1 are required for their interaction in the Y2H assay.](https://www.nature.com/articles/s41588-025-02281-4/figures/16)

**a**, Western blot detection for the expression of Myc (c-Myc epitope tag)/HA (hemagglutinin epitope tag) fusion proteins in yeast. These proteins (labeled on each of the figures) were used in Fig. [6a](https://www.nature.com/articles/s41588-025-02281-4#Fig6). By using Myc/HA antibodies, each of the Myc/HA fusion proteins was detected essentially as a single major band representing the full-length GFP fusion protein in rice protoplasts, consistent with the predicted protein size. **b**, Y2H assays for the interaction of truncated SBRR1 and SIP1 proteins. **c**, Western blot analysis for confirming the expression of Myc/HA fusion proteins in yeast. These proteins (labeled on each of the figures) were used in Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02281-4#Fig16). By using Myc/HA antibodies, each of the Myc/HA fusion proteins was detected essentially as a single major band representing the full-length GFP fusion protein in rice protoplasts, consistent with the predicted protein size. **d**, _In vitro_ GST pull-down assay for the interaction between the PAN domain of SBRR1 and DP-repeat domain (DP, aspartate-proline) of SIP1. Lower panel shows the protein abundance stained by CBB (coomassie brilliant blue) in SDS-PAGE gel. The Y2H and pull-down assays were done independently three times.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM27)

### [Extended Data Fig. 9 Subcellular localization of SBRR1-GFP and SIP1-GFP proteins in _N. benthamiana_ leaf epidermal cells.](https://www.nature.com/articles/s41588-025-02281-4/figures/17)

Plasmolysis was achieved by incubating cells in 5% NaCl. The red arrows indicate positions of shrinking plasma membranes. Scale bar, 50 μm. The subcellular localization assays were done independently three times.

### [Extended Data Fig. 10 Go analysis of RNA-seq transcript profiles between _sbrr1-ko1_ or _sip1-ko1_ and WT, and Chitinase activities in TG394 and TG394-_SBRR1__R_.](https://www.nature.com/articles/s41588-025-02281-4/figures/18)

**a**, Comparisons of RNA-seq transcript profiles between _sbrr1-ko1_ and WT and between _sip1-ko1_ and WT without or with _R. solani_ inoculation (12 HPI). **b**, Chitinase activities in TG394 and TG394-_SBRR1__R_ without (0 h) or with _R. solani_ inoculation (48 HPI). Data are presented as means ± s.d (n = 3). Different lowercase letters indicate significant differences (_P_ < 0.05) based on one-way ANOVA with Duncan’s multiple range tests.

[Source data](https://www.nature.com/articles/s41588-025-02281-4#MOESM28)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM1_ESM.pdf)

Supplementary Notes 1–6, Supplementary Figs. 1-–5 and Unprocessed western blots for Supplementary Figs. 2, 8, 11 and 12.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM3_ESM.pdf)

### [Supplementary Tables 1–13](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM4_ESM.xlsx)

Supplementary Table 1.The 178 varieties used in assay for ShB resistance. Supplementary Table 2. SNPs significantly associated with ShB resistance. Supplementary Table 3. _SBRR1_ sequence variation in 20 relatively susceptible (S) and 20 relatively resistant (R) varieties. Supplementary Table 4. RNA expression levels of _SBRR1_ in _SBRR1-S_ and _SBRR1-R_ lines post inoculation with _R. solani_. Supplementary Table 5. Verification of the tight linkage between Indel-946 and SNPG/A. Supplementary Table 6. Information for accessions and _SBRR1_ alleles in _Oryza sativa_ from the 3 K rice genome resource. Supplementary Table 7. The statistical test for the relative ratio of nucleotide diversity of cultivated rice to common wild rice between the _SBRR1_ locus and flanking genomic regions. Supplementary Table 8. Information for accessions and _SBRR1_ alleles in _Oryza rufipogon_ from OryzaGenome. Supplementary Table 9. Genes differentially expressed between _sbrr1-ko1_ and WT that are specifically responsive to _R. solani_ infection. Supplementary Table 10. Genes differentially expressed between _sip1-ko1_ and WT that are specifically responsive to _R. solani_ infection. Supplementary Table 11. Gene Ontology (GO) enrichment analysis of the identified differentially expressed genes between _sbrr1-ko1_ and WT. Supplementary Table 12. Gene Ontology (GO) enrichment analysis of the identified differentially expressed genes between _sip1-ko1_ and WT. Supplementary Table 13. Primers used in this study.

### [Supplementary Data 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM5_ESM.xlsx)

Statistical source data for Supplementary Figs. 1–4 and 8, 10, 11, 13, 15.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM7_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM8_ESM.pdf)

Unprocessed western blots and gels.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM9_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM12_ESM.pdf)

Unprocessed western blots.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM13_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM14_ESM.pdf)

Unprocessed western blots and gels.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM15_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM16_ESM.pdf)

Unprocessed western blots.

### [Source Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM17_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM18_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM19_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM20_ESM.pdf)

Unprocessed gels.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM21_ESM.pdf)

Unprocessed western blots and gels.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM22_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM23_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM24_ESM.pdf)

Unprocessed gels.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM25_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM26_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM27_ESM.pdf)

Unprocessed western blots and gels.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02281-4/MediaObjects/41588_2025_2281_MOESM28_ESM.xlsx)

Statistical source data.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Natural%20variation%20in%20SBRR1%20shows%20high%20potential%20for%20sheath%20blight%20resistance%20breeding%20in%20rice&author=Zhiming%20Feng%20et%20al&contentID=10.1038%2Fs41588-025-02281-4&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-30&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY-NC-ND)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02281-4)

### Cite this article

Feng, Z., Gao, P., Wang, G. _et al._ Natural variation in _SBRR1_ shows high potential for sheath blight resistance breeding in rice. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02281-4

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02281-4?format=refman&flavour=citation)

-   Received: 17 May 2024
    
-   Accepted: 23 June 2025
    
-   Published: 30 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02281-4
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02281-4.pdf)
