> id: https://www.nature.com/articles/s41588-025-02305-z

> link: https://www.nature.com/articles/s41588-025-02305-z

> title: Precise modulation of BRG1 levels reveals features of mSWISNF dosage sensitivity

# Precise modulation of BRG1 levels reveals features of mSWISNF dosage sensitivity
_Published on Fri, 22 Aug 2025 00:00:00 GMT_

Abstract
--------

Mammalian switch/sucrose nonfermentable (mSWI/SNF) complex regulates chromatin accessibility and frequently shows alterations due to mutation in cancer and neurological diseases. Inadequate expression of mSWI/SNF in heterozygous mice can lead to developmental defects, indicating dosage-sensitive effects of mSWI/SNF. However, how its dosage affects function has remained unclear. Using a targeted protein degradation system, we investigated its dosage-sensitive effects by precisely controlling protein levels of BRG1, the ATPase subunit of the mSWI/SNF complex. We found that binding of BRG1 to chromatin exhibited a linear response to the BRG1 protein level. Although chromatin accessibility at most promoters and insulators was largely unaffected by BRG1 depletion, 44% of enhancers, including 84% of defined superenhancers, showed reduced accessibility. Notably, half of the BRG1-regulated enhancers, particularly superenhancers, exhibited a buffered response to BRG1 loss. Consistently, transcription exhibited a predominantly buffered response to changes in BRG1 levels. Collectively, our findings demonstrate a genomic feature-specific response to BRG1 dosage, shedding light on the dosage-sensitive effects of mSWI/SNF complex defects in cancer and other diseases.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41389-022-00406-6/MediaObjects/41389_2022_406_Fig1_HTML.png)

### [The SWI/SNF ATPase BRG1 facilitates multiple pro-tumorigenic gene expression programs in _SMARCB1_\-deficient cancer cells](https://www.nature.com/articles/s41389-022-00406-6?fromPaywallRec=false)

Article Open access 01 June 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-48911-z/MediaObjects/41467_2024_48911_Fig1_HTML.png)

### [Acute depletion of BRG1 reveals its primary function as an activator of transcription](https://www.nature.com/articles/s41467-024-48911-z?fromPaywallRec=false)

Article Open access 29 May 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-021-00867-2/MediaObjects/41588_2021_867_Fig1_HTML.png)

### [The SWI/SNF chromatin remodeling complex helps resolve R-loop-mediated transcription–replication conflicts](https://www.nature.com/articles/s41588-021-00867-2?fromPaywallRec=false)

Article 13 May 2021

Main
----

Packaging of DNA into chromatin restricts the access of transcription factors (TFs) to their cognate binding sites[1](https://www.nature.com/articles/s41588-025-02305-z#ref-CR1 "Clapier, C. R., Iwasa, J., Cairns, B. R. & Peterson, C. L. Mechanisms of action and regulation of ATP-dependent chromatin-remodelling complexes. Nat. Rev. Mol. Cell Biol. 18, 407–422 (2017)."),[2](https://www.nature.com/articles/s41588-025-02305-z#ref-CR2 "Kim, S. & Wysocka, J. Deciphering the multi-scale, quantitative cis-regulatory code. Mol. Cell 83, 373–392 (2023)."),[3](https://www.nature.com/articles/s41588-025-02305-z#ref-CR3 "Bell, O., Tiwari, V. K., Thomä, N. H. & Schübeler, D. Determinants and dynamics of genome accessibility. Nat. Rev. Genet. 12, 554–564 (2011)."). ATP-dependent chromatin remodelers facilitate TF binding by sliding, disassembling or ejecting nucleosomes, thereby generating open chromatin at regulatory elements such as promoters, enhancers and insulators[1](https://www.nature.com/articles/s41588-025-02305-z#ref-CR1 "Clapier, C. R., Iwasa, J., Cairns, B. R. & Peterson, C. L. Mechanisms of action and regulation of ATP-dependent chromatin-remodelling complexes. Nat. Rev. Mol. Cell Biol. 18, 407–422 (2017)."),[2](https://www.nature.com/articles/s41588-025-02305-z#ref-CR2 "Kim, S. & Wysocka, J. Deciphering the multi-scale, quantitative cis-regulatory code. Mol. Cell 83, 373–392 (2023)."),[3](https://www.nature.com/articles/s41588-025-02305-z#ref-CR3 "Bell, O., Tiwari, V. K., Thomä, N. H. & Schübeler, D. Determinants and dynamics of genome accessibility. Nat. Rev. Genet. 12, 554–564 (2011)."),[4](https://www.nature.com/articles/s41588-025-02305-z#ref-CR4 "Li, B., Carey, M. & Workman, J. L. The role of chromatin during transcription. Cell 128, 707–719 (2007)."). In mammals, chromatin remodelers are categorized into four major families based on their ATPase subunits: SWI/SNF (also known as BRG1/BRM-associated factor (BAF)); imitation switch (ISWI); chromodomain helicase DNA-binding (CHD); and INO80 (ref. [1](https://www.nature.com/articles/s41588-025-02305-z#ref-CR1 "Clapier, C. R., Iwasa, J., Cairns, B. R. & Peterson, C. L. Mechanisms of action and regulation of ATP-dependent chromatin-remodelling complexes. Nat. Rev. Mol. Cell Biol. 18, 407–422 (2017).")). Although these remodelers can co-occupy genomic regions, each chromatin remodeler also selectively remodels particular genomic loci and affects the binding of specific TFs[5](https://www.nature.com/articles/s41588-025-02305-z#ref-CR5 "Morris, S. A. et al. Overlapping chromatin remodeling systems collaborate genome-wide at dynamic chromatin transitions. Nat. Struct. Mol. Biol. 21, 73 (2014)."),[6](https://www.nature.com/articles/s41588-025-02305-z#ref-CR6 "De Dieuleveult, M. et al. Genome-wide nucleosome specificity and function of chromatin remodellers in ES cells. Nature 530, 113–116 (2016)."),[7](https://www.nature.com/articles/s41588-025-02305-z#ref-CR7 "Barisic, D., Stadler, M. B., Iurlaro, M. & Schübeler, D. Mammalian ISWI and SWI/SNF selectively mediate binding of distinct transcription factors. Nature 569, 136–140 (2019)."). The regulatory elements, which are detected by transposases and nucleases, are generally considered to be stable once opened[8](https://www.nature.com/articles/s41588-025-02305-z#ref-CR8 "Stewart-Morgan, K. R., Petryk, N. & Groth, A. Chromatin replication and epigenetic cell memory. Nat. Cell Biol. 22, 361–371 (2020)."),[9](https://www.nature.com/articles/s41588-025-02305-z#ref-CR9 "Liu, Y. et al. Widespread mitotic bookmarking by histone marks and transcription factors in pluripotent stem cells. Cell Rep. 19, 1283–1293 (2017)."). However, recent studies using chemical inhibitors or acute degradation have shown that chromatin remodelers are continuously required to maintain chromatin accessibility[10](https://www.nature.com/articles/s41588-025-02305-z#ref-CR10 "Bomber, M. L. et al. Human SMARCA5 is continuously required to maintain nucleosome spacing. Mol. Cell 83, 507–522.e6 (2023)."),[11](https://www.nature.com/articles/s41588-025-02305-z#ref-CR11 "Schick, S. et al. Acute BAF perturbation causes immediate changes in chromatin accessibility. Nat. Genet. 53, 269–278 (2021)."),[12](https://www.nature.com/articles/s41588-025-02305-z#ref-CR12 "Iurlaro, M. et al. Mammalian SWI/SNF continuously restores local accessibility to chromatin. Nat. Genet. 53, 279–287 (2021)."), revealing the dynamic nature of nucleosome positioning. These approaches overcome the limitations of conventional genetic depletion methods, in which remodelers persist for days and are subject to multiple layers of feedback regulation. As a result, they enable more direct investigation of the primary roles of chromatin remodelers[10](https://www.nature.com/articles/s41588-025-02305-z#ref-CR10 "Bomber, M. L. et al. Human SMARCA5 is continuously required to maintain nucleosome spacing. Mol. Cell 83, 507–522.e6 (2023)."),[11](https://www.nature.com/articles/s41588-025-02305-z#ref-CR11 "Schick, S. et al. Acute BAF perturbation causes immediate changes in chromatin accessibility. Nat. Genet. 53, 269–278 (2021)."),[12](https://www.nature.com/articles/s41588-025-02305-z#ref-CR12 "Iurlaro, M. et al. Mammalian SWI/SNF continuously restores local accessibility to chromatin. Nat. Genet. 53, 279–287 (2021)."),[13](https://www.nature.com/articles/s41588-025-02305-z#ref-CR13 "Brahma, S. & Henikoff, S. The BAF chromatin remodeler synergizes with RNA polymerase II and transcription factors to evict nucleosomes. Nat. Genet. 56, 100–111 (2024)."),[14](https://www.nature.com/articles/s41588-025-02305-z#ref-CR14 "Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. Nat. Struct. Mol. Biol. 28, 501–511 (2021)."),[15](https://www.nature.com/articles/s41588-025-02305-z#ref-CR15 "Ren, G. et al. Acute depletion of BRG1 reveals its primary function as an activator of transcription. Nat. Commun. 15, 4561 (2024)."),[16](https://www.nature.com/articles/s41588-025-02305-z#ref-CR16 "Martin, B. J. E. et al. Global identification of SWI/SNF targets reveals compensation by EP400. Cell 186, 5290–5307.e26 (2023)."),[17](https://www.nature.com/articles/s41588-025-02305-z#ref-CR17 "Blümli, S. et al. Acute depletion of the ARID1A subunit of SWI/SNF complexes reveals distinct pathways for activation and repression of transcription. Cell Rep. 37, 109943 (2021).").

The mammalian SWI/SNF (mSWI/SNF) complex is a multisubunit assembly derived from 29 genes[18](https://www.nature.com/articles/s41588-025-02305-z#ref-CR18 "Mashtalir, N. et al. Modular organization and assembly of SWI/SNF family chromatin remodeling complexes. Cell 175, 1272–1288.e20 (2018)."). It exists in three distinct forms, canonical BAF (cBAF), polybromo-associated BAF (PBAF) and noncanonical BAF (ncBAF), which share two mutually exclusive ATPase subunits, BRG1 (encoded by _SMARCA4_) and BRM (_SMARCA2_). Copy number variations of remodelers, including mSWI/SNF, are poorly tolerated in humans[19](https://www.nature.com/articles/s41588-025-02305-z#ref-CR19 "Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)."),[20](https://www.nature.com/articles/s41588-025-02305-z#ref-CR20 "Gourisankar, S., Krokhotin, A., Wenderski, W. & Crabtree, G. R. Context-specific functions of chromatin remodellers in development and disease. Nat. Rev. Genet. 25, 340–361 (2023)."). Moreover, mutations in genes encoding mSWI/SNF subunits occur in more than 20% of cancers[21](https://www.nature.com/articles/s41588-025-02305-z#ref-CR21 "Kadoch, C. et al. Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy. Nat. Genet. 45, 592–601 (2013)."),[22](https://www.nature.com/articles/s41588-025-02305-z#ref-CR22 "Mittal, P. & Roberts, C. W. M. The SWI/SNF complex in cancer — biology, biomarkers and therapy. Nat. Rev. Clin. Oncol. 17, 435–448 (2020)."),[23](https://www.nature.com/articles/s41588-025-02305-z#ref-CR23 "Reddy, D., Bhattacharya, S. & Workman, J. L. (mis)-Targeting of SWI/SNF complex(es) in cancer. Cancer Metastasis Rev. 42, 455–470 (2023).") and often destabilize the complex[24](https://www.nature.com/articles/s41588-025-02305-z#ref-CR24 "Wang, X. et al. SMARCB1-mediated SWI/SNF complex function is essential for enhancer regulation. Nat. Genet. 49, 289–295 (2016)."),[25](https://www.nature.com/articles/s41588-025-02305-z#ref-CR25 "Nakayama, R. T. et al. SMARCB1 is required for widespread BAF complex–mediated activation of enhancers and bivalent promoters. Nat. Genet. 49, 1613–1623 (2017)."). Notably, heterozygous mutations have also been linked to neurodevelopmental syndromes such as Coffin–Siris and Nicolaides–Baraitser[26](https://www.nature.com/articles/s41588-025-02305-z#ref-CR26 "Valencia, A. M. et al. Landscape of mSWI/SNF chromatin remodeling complex perturbations in neurodevelopmental disorders. Nat. Genet. 55, 1400 (2023)."),[27](https://www.nature.com/articles/s41588-025-02305-z#ref-CR27 "Bögershausen, N. & Wollnik, B. Mutational landscapes and phenotypic spectrum of SWI/SNF-related intellectual disability disorders. Front Mol. Neurosci. 11, 400386 (2018)."),[28](https://www.nature.com/articles/s41588-025-02305-z#ref-CR28 "Tsurusaki, Y. et al. Mutations affecting components of the SWI/SNF complex cause Coffin-Siris syndrome. Nat. Genet. 44, 376–378 (2012)."), indicating sensitivity to the dosage of mSWI/SNF[20](https://www.nature.com/articles/s41588-025-02305-z#ref-CR20 "Gourisankar, S., Krokhotin, A., Wenderski, W. & Crabtree, G. R. Context-specific functions of chromatin remodellers in development and disease. Nat. Rev. Genet. 25, 340–361 (2023)."),[29](https://www.nature.com/articles/s41588-025-02305-z#ref-CR29 "Kadoch, C. & Crabtree, G. R. Mammalian SWI/SNF chromatin remodeling complexes and cancer: mechanistic insights gained from human genomics. Sci. Adv. 1, e1500447 (2015)."),[30](https://www.nature.com/articles/s41588-025-02305-z#ref-CR30 "Chang, C. Y. et al. Increased ACTL6A occupancy within mSWI/SNF chromatin remodelers drives human squamous cell carcinoma. Mol. Cell 81, 4964–4978.e8 (2021)."),[31](https://www.nature.com/articles/s41588-025-02305-z#ref-CR31 "Wenderski, W. et al. Loss of the neural-specific BAF subunit ACTL6B relieves repression of early response genes and causes recessive autism. Proc. Natl Acad. Sci. USA 117, 10055–10066 (2020)."),[32](https://www.nature.com/articles/s41588-025-02305-z#ref-CR32 "Ronan, J. L., Wu, W. & Crabtree, G. R. From neural development to cognition: unexpected roles for chromatin. Nat. Rev. Genet. 14, 347–359 (2013)."). Consistently, _Brg1_\-heterozygous mice exhibit exencephaly, cardiac developmental defects and increased susceptibility to mammary tumors[33](https://www.nature.com/articles/s41588-025-02305-z#ref-CR33 "Bultman, S. et al. A Brg1 null mutation in the mouse reveals functional differences among mammalian SWI/SNF complexes. Mol. Cell 6, 1287–1295 (2000)."),[34](https://www.nature.com/articles/s41588-025-02305-z#ref-CR34 "Glaros, S., Cirrincione, G. M., Palanca, A., Metzger, D. & Reisman, D. Targeted knockout of BRG1 potentiates lung cancer development. Cancer Res. 68, 3689–3696 (2008)."),[35](https://www.nature.com/articles/s41588-025-02305-z#ref-CR35 "Takeuchi, J. K. et al. Chromatin remodelling complex dosage modulates transcription factor function in heart development. Nat. Commun. 2, 187 (2011)."),[36](https://www.nature.com/articles/s41588-025-02305-z#ref-CR36 "Bultman, S. J. et al. Characterization of mammary tumors from Brg1 heterozygous mice. Oncogene 27, 460–468 (2008)."). Similar dosage-sensitive effects have also been observed in _Caenorhabditis elegans_, with functional differences in cell migration and growth depending on SWI/SNF dosage[37](https://www.nature.com/articles/s41588-025-02305-z#ref-CR37 "van der Vaart, A., Godfrey, M. & Portegijs, V. Dose-dependent functions of SWI/SNF BAF in permitting and inhibiting cell proliferation in vivo. Sci. Adv. 6, eaay3823 (2020)."),[38](https://www.nature.com/articles/s41588-025-02305-z#ref-CR38 "Smith, J. J. et al. The SWI/SNF chromatin remodeling assemblies BAF and PBAF differentially regulate cell cycle exit and cellular invasion in vivo. PLoS Genet. 18, e1009981 (2021)."). Collectively, these results suggest a conserved dosage-sensitive function of mSWI/SNF. However, the molecular basis of this function remains poorly understood.

In this study, we took advantage of the dTAG system, which enables precise control of protein dosage[39](https://www.nature.com/articles/s41588-025-02305-z#ref-CR39 "Nabet, B. et al. The dTAG system for immediate and target-specific protein degradation. Nat. Chem. Biol. 14, 431–441 (2018)."),[40](https://www.nature.com/articles/s41588-025-02305-z#ref-CR40 "Naqvi, S. et al. Precise modulation of transcription factor levels identifies features underlying dosage sensitivity. Nat. Genet. 55, 841–851 (2023)."), to investigate BRG1 dosage sensitivity in mouse embryonic stem (mES) cells. By modulating BRG1 levels, we found that BRG1 binding was linearly sensitive to its dosage, regardless of TFs or histone modifications, suggesting that the genome-wide distribution of mSWI/SNF is independent of these factors. Chromatin accessibility at half of BRG1-bound enhancers and at small subsets of promoters and insulators depended on BRG1. At BRG1-bound but BRG1-independent promoters and insulator, NFYA and CTCF may contribute to maintenance of accessibility. Among BRG1-dependent enhancers, approximately half exhibited a buffered response to BRG1 dosage. Notably, superenhancers (SEs) exhibited a buffered pattern, whereas weak enhancers were more sensitive to reduction in BRG1 levels. Although BRG1 degradation had a limited impact on transcription, downregulated genes were predominantly buffered genes. We further confirmed the conservation of BRG1-dosage-sensitive accessibility regulation and its role in _MYC_ expression in human lung epithelial cells. Overall, our results reveal the genomic features of BRG1 dosage sensitivity, providing insight into its dosage-sensitive defects in cancers and other diseases.

Results
-------

### BRG1 binding exhibits a linear response to its dosage

In mES cells, _Brg1_ encodes the major catalytic subunit of mSWI/SNF, as _Brm_ is only weakly expressed[12](https://www.nature.com/articles/s41588-025-02305-z#ref-CR12 "Iurlaro, M. et al. Mammalian SWI/SNF continuously restores local accessibility to chromatin. Nat. Genet. 53, 279–287 (2021)."),[41](https://www.nature.com/articles/s41588-025-02305-z#ref-CR41 "Ho, L. et al. An embryonic stem cell chromatin remodeling complex, esBAF, is essential for embryonic stem cell self-renewal and pluripotency. Proc. Natl Acad. Sci. USA 106, 5181–5186 (2009).") (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02305-z#Fig8)). To manipulate BRG1 levels, we used the dTAG system[39](https://www.nature.com/articles/s41588-025-02305-z#ref-CR39 "Nabet, B. et al. The dTAG system for immediate and target-specific protein degradation. Nat. Chem. Biol. 14, 431–441 (2018)."),[40](https://www.nature.com/articles/s41588-025-02305-z#ref-CR40 "Naqvi, S. et al. Precise modulation of transcription factor levels identifies features underlying dosage sensitivity. Nat. Genet. 55, 841–851 (2023)."),[42](https://www.nature.com/articles/s41588-025-02305-z#ref-CR42 "Zhou, C., Wang, M., Zhang, C. & Zhang, Y. The transcription factor GABPA is a master regulator of naive pluripotency. Nat. Cell Biol. 27, 48–58 (2025).") to perform targeted degradation of the BRG1 protein by introducing an _FKBP12–F36V_ degron into the _Brg1_ locus, resulting in _FKBP12–F36V_ _Brg1_\-knock-in cells. In the presence of heterobifunctional molecule dTAG13, the degron directs the fusion protein to the proteasome for degradation[39](https://www.nature.com/articles/s41588-025-02305-z#ref-CR39 "Nabet, B. et al. The dTAG system for immediate and target-specific protein degradation. Nat. Chem. Biol. 14, 431–441 (2018).") (Extended Data Fig. [1b,c](https://www.nature.com/articles/s41588-025-02305-z#Fig8)), a process that is completed within 60 min (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02305-z#Fig8)). Addition of the degron tag did not affect the ability of BRG1 to form complexes with other mSWI/SNF components, including ARID1A (cBAF), PBRM1 (PBAF) and BRD9 (ncBAF), as evidenced by our finding that the immunoprecipitation efficiency of _Brg_1-knock-in cells was comparable to that of the untagged wild-type cells (Extended Data Fig. [1e](https://www.nature.com/articles/s41588-025-02305-z#Fig8)). As a previous study found that mSWI/SNF inhibition for 24 h mirrored the effects of genetic manipulation[12](https://www.nature.com/articles/s41588-025-02305-z#ref-CR12 "Iurlaro, M. et al. Mammalian SWI/SNF continuously restores local accessibility to chromatin. Nat. Genet. 53, 279–287 (2021)."), we treated cells with dTAG13 for 24 h to modulate BRG1 levels in a dose-dependent manner (Fig. [1a,b](https://www.nature.com/articles/s41588-025-02305-z#Fig1)). Immunostaining confirmed a population-wide decrease in BRG1 levels (Extended Data Fig. [1f](https://www.nature.com/articles/s41588-025-02305-z#Fig8)). These results demonstrated that we had successfully established a system that allowed precise control of BRG1 dosage for study of its dosage-sensitive function.

**Fig. 1: Chromatin binding of BRG1 exhibits a linear response to its dosage.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02305-z/figures/1)

**a**, Western blot analysis of BRG1 in _Brg1_\-knock-in (KI) and wild-type (WT) mES cells after treatment with the indicated concentrations of dTAG13 for 24 h. Data are representative of six independent experiments. **b**, Quantification of relative BRG1 abundance following dTAG13 treatment. BRG1 expression was normalized to that of tubulin and then to that of DMSO-treated (0 nM) _Brg1_\-KI mES cells within each biological replicate. Data represent the mean ± s.e.m. from six independent experiments. **c**, Heatmap showing profiles of protein and histone modifications (CUT&RUN and ChIP–seq) for regions associated with different genomic elements (bottom). The top panel shows aggregate coverage plots with mean enrichment at promoters (red), enhancers (cyan) and insulators (orange). **d**, Representative genome browser views of BRG1 binding at a promoter, enhancer and insulator. **e**, Numbers of BRG1 binding peaks identified at different dTAG13 concentrations. **f**, PCA of BRG1 CUT&RUN data across varying dTAG13 concentrations. **g**, Aggregate coverage plot showing mean BRG1 enrichment at different dTAG13 concentrations. **h**, Aggregate coverage plot of BRG1 enrichment at promoters, enhancers and insulators across different dTAG13 concentrations. **i**, Diagram illustrating models of linear (black) and buffered (green) dosage-sensitive responses. **j**, BRG1 signals at representative linear (red) and sensitive (blue) regions (left). The relative signal was calculated by normalizing with 0 nM signals. Genome browser view showing BRG1 binding at regions with linear and buffered responses (right). The genome locations of the browser view are indicated at the bottom of the panels. The linear and buffered models are illustrated in the diagram. **k**, Histogram of delta values for all 20,326 BRG1-binding sites. The red line indicates delta = 0. Enh., enhancer; Ins., insulator; Pro., promoter.

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM4)

To this end, we first investigated the genome-wide binding profile of BRG1 by performing a cleavage under targets and release using nuclease (CUT&RUN) assay using _Brg1_\-knock-in cells (Extended Data Fig. [1g](https://www.nature.com/articles/s41588-025-02305-z#Fig8)). BRG1 binding was enriched at promoters (23.9%, marked by histone H3 trimethylated at Lys4 (H3K4me3)), enhancers (65.0%, marked by H3 monomethylated at Lys4 and H3 acetylated at Lys27 (H3K27ac)) and insulators (11.1%, marked by CTCF) (Fig. [1c,d](https://www.nature.com/articles/s41588-025-02305-z#Fig1)), consistent with its known roles at both proximal and distal regulatory elements[6](https://www.nature.com/articles/s41588-025-02305-z#ref-CR6 "De Dieuleveult, M. et al. Genome-wide nucleosome specificity and function of chromatin remodellers in ES cells. Nature 530, 113–116 (2016)."),[16](https://www.nature.com/articles/s41588-025-02305-z#ref-CR16 "Martin, B. J. E. et al. Global identification of SWI/SNF targets reveals compensation by EP400. Cell 186, 5290–5307.e26 (2023)."),[43](https://www.nature.com/articles/s41588-025-02305-z#ref-CR43 "Gatchalian, J. et al. A non-canonical BRD9-containing BAF chromatin remodeling complex regulates naive pluripotency in mouse embryonic stem cells. Nat. Commun. 9, 5139 (2018)."). The presence of BRG1 at insulators was consistent with its reported interaction with CTCF and ncBAF complex localization patterns[43](https://www.nature.com/articles/s41588-025-02305-z#ref-CR43 "Gatchalian, J. et al. A non-canonical BRD9-containing BAF chromatin remodeling complex regulates naive pluripotency in mouse embryonic stem cells. Nat. Commun. 9, 5139 (2018)."),[44](https://www.nature.com/articles/s41588-025-02305-z#ref-CR44 "Marino, M. M. et al. Interactome mapping defines BRG1, a component of the SWI/SNF chromatin remodeling complex, as a new partner of the transcriptional regulator CTCF. J. Biol. Chem. 294, 861–873 (2019)."),[45](https://www.nature.com/articles/s41588-025-02305-z#ref-CR45 "Michel, B. C. et al. A non-canonical SWI/SNF complex is a synthetic lethal target in cancers driven by BAF complex perturbation. Nat. Cell Biol. 20, 1410–1420 (2018)."). These results confirm that the tagged BRG1 mirrors the endogenous binding profile and primarily targets promoters, enhancers and insulators.

The nature of the mechanisms underlying the target selectivity of remodelers remains a fundamental question. TFs and histone modifications have been reported to recruit mSWI/SNF, likely through physical interactions[46](https://www.nature.com/articles/s41588-025-02305-z#ref-CR46 "Minderjahn, J. et al. Mechanisms governing the pioneering and redistribution capabilities of the non-classical pioneer PU.1. Nat. Commun. 11, 402 (2020)."),[47](https://www.nature.com/articles/s41588-025-02305-z#ref-CR47 "Krishnakumar, R. et al. FOXD3 regulates pluripotent stem cell potential by simultaneously initiating and repressing enhancer activity. Cell Stem Cell 18, 104–117 (2016)."),[48](https://www.nature.com/articles/s41588-025-02305-z#ref-CR48 "King, H. W. & Klose, R. J. The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells. eLife 6, e22631 (2017)."),[49](https://www.nature.com/articles/s41588-025-02305-z#ref-CR49 "Wanior, M., Krämer, A., Knapp, S. & Joerger, A. C. Exploiting vulnerabilities of SWI/SNF chromatin remodelling complexes for cancer therapy. Oncogene 40, 3637–3654 (2021)."),[50](https://www.nature.com/articles/s41588-025-02305-z#ref-CR50 "Kim, B. W. et al. Neuronal activity-induced BRG1 phosphorylation regulates enhancer activation. Cell Rep. 36, 109357 (2021)."). Given that TFs exhibit nonlinear binding profiles and preferentially occupy high-affinity targets at low concentrations[40](https://www.nature.com/articles/s41588-025-02305-z#ref-CR40 "Naqvi, S. et al. Precise modulation of transcription factor levels identifies features underlying dosage sensitivity. Nat. Genet. 55, 841–851 (2023)."),[51](https://www.nature.com/articles/s41588-025-02305-z#ref-CR51 "Giorgetti, L. et al. Noncooperative interactions between transcription factors and clustered DNA binding sites enable graded transcriptional responses to environmental inputs. Mol. Cell 37, 418–428 (2010)."),[52](https://www.nature.com/articles/s41588-025-02305-z#ref-CR52 "Neikes, H. K. et al. Quantification of absolute transcription factor binding affinities in the native chromatin context using BANC-seq. Nat. Biotechnol. 41, 1801–1809 (2023)."),[53](https://www.nature.com/articles/s41588-025-02305-z#ref-CR53 "Belikov, S., Berg, O. G. & Wrange, Ö. Quantification of transcription factor-DNA binding affinity in a living cell. Nucleic Acids Res. 44, 3045 (2016)."), we hypothesized that the binding profile of BRG1 would follow that of TFs, assuming that its recruitment was determined by physical interactions with TFs. In this case, high-affinity and low-affinity binding sites would exhibit differential dosage sensitivity. To test this hypothesis, we profiled BRG1 binding across five dTAG13 concentrations, corresponding to different BRG1 levels (Fig. [1a,b](https://www.nature.com/articles/s41588-025-02305-z#Fig1)). Numbers of BRG1-binding sites were reduced by almost half following 0.3 nM dTAG13 treatment (78% of protein remaining), and further depletion nearly abolished detectable peaks (10 nM dTAG13, 12% of protein remaining) (Fig. [1b,e](https://www.nature.com/articles/s41588-025-02305-z#Fig1)). Principal component analysis (PCA) confirmed a global shift in BRG1 binding as dTAG13 concentrations increased (Fig. [1f](https://www.nature.com/articles/s41588-025-02305-z#Fig1)). Overall, BRG1-binding signals decreased in a gradual, dose-dependent manner (Fig. [1g](https://www.nature.com/articles/s41588-025-02305-z#Fig1)).

Next, we investigated whether BRG1 binding at different types of regulatory element (promoters, enhancers and insulators) would exhibit distinct dosage sensitivity. Notably, all three classes showed similar gradual reductions in binding (Fig. [1h](https://www.nature.com/articles/s41588-025-02305-z#Fig1)). To further analyze the kinetics of BRG1 binding, we generated two hypothetical models: a linear response model (with a proportional relationship between BRG1 dosage and binding signals); and a buffered response model (showing limited changes in binding with a subtle decrease in BRG1 levels) (Fig. [1i](https://www.nature.com/articles/s41588-025-02305-z#Fig1)). We then calculated the delta values between the distances of observed signals and those of the two models. A positive delta value indicated proximity to the linear model, whereas a negative delta value indicated closeness to the buffered model (Fig. [1j](https://www.nature.com/articles/s41588-025-02305-z#Fig1) and [Methods](https://www.nature.com/articles/s41588-025-02305-z#Sec11)). Of all BRG1-binding peaks, 92.2% followed the linear response model (Fig. [1k](https://www.nature.com/articles/s41588-025-02305-z#Fig1)). As previous studies have implicated OCT4 and H3K27ac in BRG1 recruitment[48](https://www.nature.com/articles/s41588-025-02305-z#ref-CR48 "King, H. W. & Klose, R. J. The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells. eLife 6, e22631 (2017)."),[49](https://www.nature.com/articles/s41588-025-02305-z#ref-CR49 "Wanior, M., Krämer, A., Knapp, S. & Joerger, A. C. Exploiting vulnerabilities of SWI/SNF chromatin remodelling complexes for cancer therapy. Oncogene 40, 3637–3654 (2021)."),[50](https://www.nature.com/articles/s41588-025-02305-z#ref-CR50 "Kim, B. W. et al. Neuronal activity-induced BRG1 phosphorylation regulates enhancer activation. Cell Rep. 36, 109357 (2021)."), we assessed BRG1 binding at co-occupied sites. Both sites cobound by OCT4 and BRG1 and those co-occupied by H3K27ac and BRG1 also exhibited linear responses to BRG1 dosage (Extended Data Fig. [1h](https://www.nature.com/articles/s41588-025-02305-z#Fig8)), suggesting that BRG1 binding at these loci does not heavily rely on recruitment by TFs or histone modifications.

### Variable BRG1 dependency in accessibility among regulatory elements

Given that the major function of chromatin remodeler is to regulate chromatin accessibility, we next assessed the effects of BRG1 depletion on chromatin accessibility using transposase-accessible chromatin with sequencing (ATAC-seq) at different BRG1 levels, focusing on BRG1-binding sites. Numbers of differentially accessible regions (DARs) increased progressively with decreasing BRG1 levels (Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02305-z#Fig9)), with most DARs showing reduction in or loss of accessibility (Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02305-z#Fig9)). Notably, whereas chromatin accessibility at enhancers declined gradually in response to BRG1 depletion, it remained largely unchanged at promoters and insulators (Fig. [2a,b](https://www.nature.com/articles/s41588-025-02305-z#Fig2)). Specifically, 44.2% of BRG1-bound enhancers exhibited decreased accessibility, compared to only 16.1% of promoters and 17.8% of insulators (Fig. [2c,d](https://www.nature.com/articles/s41588-025-02305-z#Fig2)). Of all DARs, 83.2% were located at enhancers, whereas only 11.1% and 5.7% were found at promoters and insulators, respectively (Fig. [2e](https://www.nature.com/articles/s41588-025-02305-z#Fig2)). A previous _Brg1_\-knockout study showed a similar trend, with enhancers affected more than promoters and insulators[48](https://www.nature.com/articles/s41588-025-02305-z#ref-CR48 "King, H. W. & Klose, R. J. The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells. eLife 6, e22631 (2017).") (Extended Data Fig. [2c](https://www.nature.com/articles/s41588-025-02305-z#Fig9)). Collectively, these results indicate that BRG1 has a more important role in maintaining the accessibility of enhancers than that of promoters and insulators.

**Fig. 2: Chromatin accessibilities at different regulatory elements exhibit varying dependencies on BRG1.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02305-z/figures/2)

**a**, Heatmap showing BRG1 binding (red) and ATAC-seq (blue) for regions falling into genomic elements at the indicated dTAG13 concentration. **b**, Line plots depicting relative BRG1 binding (red) and ATAC-seq (blue) at promoters, enhancers and insulators. **c**, Pie chart depicting the percentages of significant changes in ATAC-seq signals at BRG1 target sites in promoters, enhancers and insulators. **d**, Genome browser view showing BRG1 binding and ATAC-seq signals across regions containing promoters, enhancers and insulators. **e**, Pie charts depicting the relative distributions of significant changes in ATAC-seq signals at promoters, enhancers and insulators.

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM5)

### Compensatory mechanisms maintain accessibility in the absence of BRG1

Next, we explored the factors that might determine BRG1 dependency. To this end, we first compared BRG1-binding signals and found that their levels alone did not distinguish BRG1-dependent and BRG1-independent regions (Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02305-z#Fig10)). We next performed motif enrichment analyses to explore whether TFs contributed to the maintenance of chromatin accessibility in the absence of BRG1. The NFY and SP1 binding motifs were significantly enriched at independent promoters, whereas CTCF and CTCFL motifs were enriched at independent insulators (Fig. [3a](https://www.nature.com/articles/s41588-025-02305-z#Fig3)). By contrast, no specific motifs were uniquely enriched at independent enhancers; for instance, the OCT4 motif was identified in both dependent and independent enhancers (Fig. [3a](https://www.nature.com/articles/s41588-025-02305-z#Fig3)). Reanalysis of public chromatin immunoprecipitation followed by high-throughput sequencing (ChIP–seq) datasets revealed that NFYA and CTCF preferentially bound to BRG1-independent promoters and insulators, respectively (Fig. [3b](https://www.nature.com/articles/s41588-025-02305-z#Fig3) and Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02305-z#Fig10)). Conversely, OCT4 was more enriched at dependent enhancers (Extended Data Fig. [3b,c](https://www.nature.com/articles/s41588-025-02305-z#Fig10)). Chromatin accessibility at OCT4-binding sites was decreased following BRG1 depletion, consistent with previous results demonstrating that BRG1 is necessary for maintenance of the accessibility of OCT4-binding sites in mES cells[48](https://www.nature.com/articles/s41588-025-02305-z#ref-CR48 "King, H. W. & Klose, R. J. The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells. eLife 6, e22631 (2017).") (Extended Data Fig. [3d](https://www.nature.com/articles/s41588-025-02305-z#Fig10)).

**Fig. 3: BRG1 and SNF2H cooperatively regulate insulator elements.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02305-z/figures/3)

**a**, TF motif enrichment at BRG1-independent (Indep.) and BRG1-dependent (Dep.) ATAC-seq peaks in promoters, enhancers and insulators. The color scale represents the −log10(_P_ value), and the dot size corresponds to the effect size measured as an odds ratio. Two-sided Fisher’s exact test was performed. **b**, Aggregate coverage plot showing mean NFYA and CTCF ChIP–seq signals at BRG1-dependent and BRG1-independent regulatory elements. **c**, Aggregate coverage plot showing mean ATAC-seq signals at NFYA and CTCF-bound regions. **d**, Schematic of SNF2H and BRG1 degradation system (left). SNF2H is fused with FKBP12–F36V (_Snf2h_\-KI) to enable degradation by dTAG13. BRG1, BRM and PBRM1 are degraded by AU-15330. Western blot analysis of BRG1 and SNF2H in _Snf2h_\-KI mES cells after treatment with 1 μM AU-15330 and/or 100 nM dTAG13 for 24 h. Data are representative of three independent experiments. **e**, Aggregate coverage plot of mean ATAC-seq peaks at BRG1-dependent and BRG1-independent insulators in mES cells after 24 h of AU-15330 and/or dTAG13 treatment. **f**, Representative genome browser view showing regions regulated by BRG1 and SNF2H and those coregulated by BRG1 and SNF2H (BRG1/SNF2H).

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM4)

The distribution of the Polycomb repressive complex (PRC) is directed by mSWI/SNF[14](https://www.nature.com/articles/s41588-025-02305-z#ref-CR14 "Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. Nat. Struct. Mol. Biol. 28, 501–511 (2021)."). As well as enrichment of PRC1 and PRC2 components (RING1B and SUZ12, respectively) at promoters, we found increased binding of these components at BRG1-dependent promoters following BRG1 loss (Extended Data Fig. [3e](https://www.nature.com/articles/s41588-025-02305-z#Fig10)). Although Polycomb proteins themselves do not directly affect chromatin accessibility[54](https://www.nature.com/articles/s41588-025-02305-z#ref-CR54 "King, H. W., Fursova, N. A., Blackledge, N. P. & Klose, R. J. Polycomb repressive complex 1 shapes the nucleosome landscape but not accessibility at target genes. Genome Res. 28, 1494–1507 (2018)."),[55](https://www.nature.com/articles/s41588-025-02305-z#ref-CR55 "Hodges, H. C. et al. Dominant-negative SMARCA4 mutants alter the accessibility landscape of tissue-unrestricted enhancers. Nat. Struct. Mol. Biol. 25, 61–72 (2018)."), these results indicate that BRG1 depletion triggers broader chromatin reorganization at BRG1-dependent regulatory elements.

On the other hand, chromatin accessibility at NFYA-binding and CTCF-binding sites remained unchanged after BRG1 depletion (Fig. [3c](https://www.nature.com/articles/s41588-025-02305-z#Fig3)), suggesting that these factors contribute to the maintenance of chromatin accessibility at BRG1-independent regions. Consistently, reanalysis of _Nfya-_knockdown datasets showed that chromatin accessibility was specifically decreased at NFYA-bound BRG1-independent promoters[56](https://www.nature.com/articles/s41588-025-02305-z#ref-CR56 "Oldfield, A. J. et al. NF-Y controls fidelity of transcription initiation at gene promoters through maintenance of the nucleosome-depleted region. Nat. Commun. 10, 3072 (2019).") (Extended Data Fig. [3f](https://www.nature.com/articles/s41588-025-02305-z#Fig10)). These results, together with the observation that NFYA was mainly enriched at the BRG1-independent promoters, suggest that NFYA plays a critical part in maintaining open chromatin states at BRG1-independent promoters.

CTCF can be recruited by SNF2H, the ATPase subunit of the ISWI complex[7](https://www.nature.com/articles/s41588-025-02305-z#ref-CR7 "Barisic, D., Stadler, M. B., Iurlaro, M. & Schübeler, D. Mammalian ISWI and SWI/SNF selectively mediate binding of distinct transcription factors. Nature 569, 136–140 (2019)."),[57](https://www.nature.com/articles/s41588-025-02305-z#ref-CR57 "Dluhosova, M. et al. Epigenetic control of SPI1 gene by CTCF and ISWI ATPase SMARCA5. PLoS ONE 9, e87448 (2014)."),[58](https://www.nature.com/articles/s41588-025-02305-z#ref-CR58 "Wiechens, N. et al. The chromatin remodelling enzymes SNF2H and SNF2L position nucleosomes adjacent to CTCF and other transcription factors. PLoS Genet. 12, e1005940 (2016)."),[59](https://www.nature.com/articles/s41588-025-02305-z#ref-CR59 "Iurlaro, M. et al. Systematic assessment of ISWI subunits shows that NURF creates local accessibility for CTCF. Nat. Genet. 56, 1203–1212 (2024)."). As SNF2H and BRG1 have overlapping binding profiles[5](https://www.nature.com/articles/s41588-025-02305-z#ref-CR5 "Morris, S. A. et al. Overlapping chromatin remodeling systems collaborate genome-wide at dynamic chromatin transitions. Nat. Struct. Mol. Biol. 21, 73 (2014)."), we hypothesized that SNF2H might compensate for BRG1 at insulators. To test this hypothesis, we established _Snf2h_\-dTAG mES cells, in which SNF2H could be degraded by dTAG13 and BRG1 could be degraded by AU-15330, a proteolysis-targeting chimera (PROTAC) that targets both BRG1 and BRM[60](https://www.nature.com/articles/s41588-025-02305-z#ref-CR60 "Xiao, L. et al. Targeting SWI/SNF ATPases in enhancer-addicted prostate cancer. Nature 601, 434–439 (2022).") (Fig. [3d](https://www.nature.com/articles/s41588-025-02305-z#Fig3)). Cells were treated for 24 h, consistent with our previous experiments. As expected, AU-15330 alone had a minor effect on BRG1-independent insulators. By contrast, SNF2H depletion alone led to reduced accessibility at both BRG1-dependent and BRG1-independent insulators, with a greater effect observed at BRG1-independent insulators (Fig. [3e,f](https://www.nature.com/articles/s41588-025-02305-z#Fig3)). Simultaneous depletion of SNF2H and BRG1 further decreased accessibility at BRG1-dependent insulators but not at independent insulators (Fig. [3e,f](https://www.nature.com/articles/s41588-025-02305-z#Fig3) and Extended Data Fig. [3g](https://www.nature.com/articles/s41588-025-02305-z#Fig10)). BRG1-dependent insulators showed slightly higher enrichment of TFs in motif and ChIP–seq analyses, suggesting that BRG1-dependent insulators were occupied by more regulatory factors than BRG1-independent insulators (Extended Data Fig. [3b,h](https://www.nature.com/articles/s41588-025-02305-z#Fig10)). Overall, these results indicate that the SNF2H–CTCF axis has a dominant role in regulation of accessibility at BRG1-independent insulators, whereas both SNF2H and BRG1 contribute to the accessibility of BRG1-dependent insulators.

### Enhancer activity defines dosage sensitivity to BRG1

Next, we characterized the dosage response properties of BRG1-dependent regulatory elements with respect to chromatin accessibility. To investigate the kinetics of chromatin accessibility following BRG1 depletion, we applied the linear and buffered response models described above (Fig. [1i](https://www.nature.com/articles/s41588-025-02305-z#Fig1) and [Methods](https://www.nature.com/articles/s41588-025-02305-z#Sec11)). Overall, BRG1-dependent regulatory elements displayed an approximately equal distribution between linear and buffered responses (50.7% linear versus 49.3% buffered) (Fig. [4a](https://www.nature.com/articles/s41588-025-02305-z#Fig4), first panel). Whereas enhancers showed slight preference for a linear response (Fig. [4a](https://www.nature.com/articles/s41588-025-02305-z#Fig4), third panel), both promoters and insulators exhibited a clear preference for a buffered response (Fig. [4a](https://www.nature.com/articles/s41588-025-02305-z#Fig4), second and fourth panels). This indicates that lower levels of BRG1 are enough to maintain accessibility at promoters and insulators, but higher levels are required for maintaining enhancer accessibility.

**Fig. 4: BRG1 dosage sensitivity of enhancer accessibility is associated with enhancer activity.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02305-z/figures/4)

**a**, Histograms of delta values for BRG1-dependent regulatory elements, including all elements, promoters, enhancers and insulators. The red line indicates delta = 0. **b**, Heatmap showing BRG1-dependent enhancer regulatory elements clustered into five groups based on delta values, ranging from buffered to linear response regions. **c**, Representative genome browser views showing linear and buffered responses to BRG1 dosage. Red dots represent CUT&RUN signals under different dTAG13 concentrations, and blue dots represent ATAC-seq signals (top) under different dTAG13 concentrations. Genome browser view showing BRG1 binding at regions with linear and buffered responses (bottom). The linear and buffered models are illustrated in the diagram. **d**, Enrichment analysis of different element types per group, with enrichment calculated against all consensus regions. Dot size represents −log10(_P_ value), and the color scale corresponds to the effect size measured as an odds ratio. Two-sided Fisher’s exact test was used to compare enrichment to background regions. Obs\_Exp represents observed divided by expected frequencies. **e**, Heatmap showing relative enrichment of chromatin marks and TF binding across different groups of ATAC-seq peaks. **f**, Number of SEs in each group. **g**, Genome browser view showing ATAC-seq signals at BRG1-bound _Capns1_ SE at different dTAG13 concentrations. The linear (black) and buffered (green) models are illustrated in the diagram.

As the major function of BRG1 is to modulate enhancer accessibility, we further explored the dosage sensitivity of BRG1-dependent enhancers by dividing them into five groups based on their sensitivity: group 1 (G1) represented the most buffered enhancers and group 5 (G5) the most linearly sensitive enhancers (Fig. [4b](https://www.nature.com/articles/s41588-025-02305-z#Fig4)). When BRG1 levels were depleted to half their original value, G1 enhancers retained 80% of their original accessibility, whereas G5 enhancers retained less than 40% (Extended Data Fig. [4a](https://www.nature.com/articles/s41588-025-02305-z#Fig11)). The BRG1 binding kinetics showed similar dosage sensitivity in the different groups (Fig. [4b](https://www.nature.com/articles/s41588-025-02305-z#Fig4) and Extended Data Fig. [4b](https://www.nature.com/articles/s41588-025-02305-z#Fig11)). As shown in representative regions, whereas BRG1 binding predominantly followed a linear model (Fig. [4c](https://www.nature.com/articles/s41588-025-02305-z#Fig4), red dots), enhancer accessibility exhibited different linear and buffered responses (Figs. [1k](https://www.nature.com/articles/s41588-025-02305-z#Fig1) and [4c](https://www.nature.com/articles/s41588-025-02305-z#Fig4), blue dots), indicating that regulation of chromatin accessibility is not totally dependent on BRG1 binding.

Next, we examined the regulatory landscape in each BRG1-dependent enhancer group using ChromHMM. We found that dosage sensitivity was correlated with enhancer activity; buffered-response enhancers (G1) were more enriched for strong enhancers compared to linear-response enhancers (G5) (Fig. [4d](https://www.nature.com/articles/s41588-025-02305-z#Fig4)). The temporal dynamics of accessibility after mSWI/SNF inhibition showed no substantial differences among groups[16](https://www.nature.com/articles/s41588-025-02305-z#ref-CR16 "Martin, B. J. E. et al. Global identification of SWI/SNF targets reveals compensation by EP400. Cell 186, 5290–5307.e26 (2023).") (Extended Data Fig. [4c](https://www.nature.com/articles/s41588-025-02305-z#Fig11)), indicating that dosage sensitivity cannot be explained by differences in the speed of the response to mSWI/SNF inhibition. We then analyzed the enrichment of histone modifications and transcriptional regulators across the five groups. Although TFs including NANOG were found in all groups, G1 enhancers exhibited higher levels of H3K27ac and acetyltransferase p300 (Fig. [4e](https://www.nature.com/articles/s41588-025-02305-z#Fig4) and Extended Data Fig. [4d](https://www.nature.com/articles/s41588-025-02305-z#Fig11)), both of which are markers of active enhancers[61](https://www.nature.com/articles/s41588-025-02305-z#ref-CR61 "Creyghton, M. P. et al. Histone H3K27ac separates active from poised enhancers and predicts developmental state. Proc. Natl Acad. Sci. USA 107, 21931–21936 (2010)."). These features suggest that G1 has features of SEs, which are characterized by broader chromatin domains and higher levels of H3K27ac and MED1 (ref. [62](https://www.nature.com/articles/s41588-025-02305-z#ref-CR62 "Whyte, W. A. et al. Master transcription factors and Mediator establish super-enhancers at key cell identity genes. Cell 153, 307 (2013).")). Consistently, MED1 was specifically enriched at G1 compared to other groups (Fig. [4e](https://www.nature.com/articles/s41588-025-02305-z#Fig4)). Among the 231 SEs defined in mES cells[62](https://www.nature.com/articles/s41588-025-02305-z#ref-CR62 "Whyte, W. A. et al. Master transcription factors and Mediator establish super-enhancers at key cell identity genes. Cell 153, 307 (2013)."),[63](https://www.nature.com/articles/s41588-025-02305-z#ref-CR63 "Hnisz, D. et al. Transcriptional super-enhancers connected to cell identity and disease. Cell 155, 934 (2013)."), 97 (42%) were classified as G1 (Fig. [4f,g](https://www.nature.com/articles/s41588-025-02305-z#Fig4)). Furthermore, 195 (84.4%) of SEs were among the BRG1-dependent enhancers (G1 to G5), highlighting the critical role of BRG1 in regulation of SE accessibility. Collectively, these results show that the dosage sensitivity of enhancers is more closely correlated with enhancer activity than with specific TFs. SEs, in particular, tend to exhibit buffered responses to BRG1 dosage.

### ChromBPNet reveals sequence basis for BRG1 dosage sensitivity

To gain deeper insights into the sequence features underlying dosage sensitivity, we trained ChromBPNet on ATAC-seq data from control (0 nM dTAG13) and BRG1-depleted (100 nM dTAG13) conditions to predict chromatin accessibility[64](https://www.nature.com/articles/s41588-025-02305-z#ref-CR64 "Pampari, A. et al. ChromBPNet: bias factorized, base-resolution deep learning models of chromatin accessibility reveal cis-regulatory sequence syntax, transcription factor footprints and regulatory variants. Preprint at bioRviv 
https://doi.org/10.1101/2024.12.25.630221
(2025).") (Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02305-z#Fig12)). The model achieved high accuracy, with a Pearson’s correlation of _r_ = 0.84 across BRG1-bound peaks (Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02305-z#Fig12)).

Next, we applied the TF-MODISCO motif discovery algorithm to identify motifs with high contribution scores[64](https://www.nature.com/articles/s41588-025-02305-z#ref-CR64 "Pampari, A. et al. ChromBPNet: bias factorized, base-resolution deep learning models of chromatin accessibility reveal cis-regulatory sequence syntax, transcription factor footprints and regulatory variants. Preprint at bioRviv 
https://doi.org/10.1101/2024.12.25.630221
(2025)."),[65](https://www.nature.com/articles/s41588-025-02305-z#ref-CR65 "Avsec, Ž. et al. Effective gene expression prediction from sequence by integrating long-range interactions. Nat. Methods 18, 1196–1203 (2021)."). Motifs of SP1 and MAZ were enriched in buffered enhancers, whereas OCT4, SOX2 and KLF4 were enriched in linear enhancers (Extended Data Fig. [5c](https://www.nature.com/articles/s41588-025-02305-z#Fig12)). Marginal footprinting analysis confirmed preferential binding of OCT4 and SOX2 in linear enhancers (Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02305-z#Fig12)). Notably, buffered enhancers exhibited significantly higher GC content than linear enhancers (Extended Data Fig. [5e](https://www.nature.com/articles/s41588-025-02305-z#Fig12)), suggesting that sequence composition may shape enhancer properties and influence dosage sensitivity.

We then hypothesized that motif abundance within enhancers would contribute to dosage sensitivity. We quantified the numbers of high-score sites[66](https://www.nature.com/articles/s41588-025-02305-z#ref-CR66 "Pratt, H. E. et al. Using a comprehensive atlas and predictive models to reveal the complexity and evolution of brain-active regulatory elements. Sci. Adv. 10, eadj4452 (2024).") as putative TF binding sites. G1 enhancers exhibited the highest number of high-score sites. BRG1 depletion led to a reduction in the number of high-score sites; however, G1 enhancers retained more sites than G5 enhancers (Extended Data Fig. [5f,g](https://www.nature.com/articles/s41588-025-02305-z#Fig12)). These results suggest that motif composition and abundance within enhancers are critical for their response to BRG1 dosage.

### Dosage-sensitive transcriptional regulation by BRG1

As chromatin accessibility changes can affect transcription, we next examined how varying BRG1 levels influenced gene expression. Transcriptome analysis revealed a gradual increase in numbers of differentially expressed genes (DEGs) with decreasing BRG1 levels (Fig. [5a](https://www.nature.com/articles/s41588-025-02305-z#Fig5) and Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02305-z#Fig13)). Despite this trend, the overall number of DEGs was relatively low, with 589 (5.2%) upregulated and 1,016 (9.0%) downregulated genes among the 11,303 expressed genes (fragments per kilobase of transcript per million mapped reads (FPKM) > 1). Genes with BRG1-dependent promoters were downregulated with decreased BRG1 dosage, whereas genes with BRG1-independent promoters showed no major change (Extended Data Fig. [6b](https://www.nature.com/articles/s41588-025-02305-z#Fig13)). Enhancer-proximal genes followed a similar pattern, although with milder effects; only BRG1-dependent enhancer target genes showed modest but significant reductions in expression (Extended Data Fig. [6c](https://www.nature.com/articles/s41588-025-02305-z#Fig13)). Moreover, genes associated with linearly sensitive enhancers (G5) exhibited greater effects than those linked to buffered enhancers (G1) (Extended Data Fig. [6d](https://www.nature.com/articles/s41588-025-02305-z#Fig13)). These results indicate that promoter regulation exerts a stronger transcriptional impact than enhancer regulation.

**Fig. 5: Transcription of the majority of genes exhibits buffered response to BRG1 dosage.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02305-z/figures/5)

**a**, Numbers of DEGs at different dTAG13 concentrations (fold change >1.5). **b**, Histogram of delta values for all 1,016 DEGs. The red line indicates delta = 0. **c**, Heatmap showing BRG1-dependent genes, clustered into two groups based on delta values: negative or buffered (Expr-B) and positive or linear (Expr-L). **d**, Representative genes whose expression exhibited linear or buffered responses to BRG1 depletion. Green dots represent mean expression levels. Diagrams illustrating the linear (black) and buffered (red) models are shown. **e**, Relative gene expression of pluripotency factors at different BRG1 levels. Expression levels are normalized to the expression at 0 nM dTAG13 (DMSO), and the red dashed line represents the mean level at 0 nM. Data are from two independent experiments. **f**, _Myc_ expression at different BRG1 levels. Data are from two independent experiments. **g**, Box plot showing expression changes of MYC-target genes under different concentrations of dTAG13. Boxes indicate the 25th to 75th percentiles, the median is shown as a horizontal black line, and whiskers extend up to 1.5 times the interquartile range from the quartiles. Expression levels are normalized to the expression at 0 nM (DMSO). Data represent two independent experiments, and _P_ values were calculated with Wilcoxon’s _t_\-test (two-sided, versus 0 nM).

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM5)

To further dissect transcriptional responses, we modeled all downregulated genes into buffered or linear response categories (Fig. [5b,c](https://www.nature.com/articles/s41588-025-02305-z#Fig5)). Notably, the majority of the downregulated genes (84.4%) exhibited buffered responses, with limited numbers of genes (15.6%) showing linear responses to BRG1 dosage (Fig. [5b](https://www.nature.com/articles/s41588-025-02305-z#Fig5)). Buffered response genes retained more than 75% expression at half BRG1 levels, whereas expression of linear response genes dropped below 50% (Fig. [5d](https://www.nature.com/articles/s41588-025-02305-z#Fig5) and Extended Data Fig. [6e](https://www.nature.com/articles/s41588-025-02305-z#Fig13)). Downregulated genes were enriched for terms related to proliferation and the MAPK pathway, implying compromised self-renewal programs in mES cells (Extended Data Fig. [6f](https://www.nature.com/articles/s41588-025-02305-z#Fig13)). Whereas core pluripotency genes including _Oct4_ (also known as _Pou5f1_) remained unchanged, we observed downregulation of _Tbx3_ and _Tcl1_, targets of BRG1 and LIF-STAT3 (refs. [67](https://www.nature.com/articles/s41588-025-02305-z#ref-CR67 "Zhang, W. et al. The BAF and PRC2 complex subunits Dpf2 and Eed antagonistically converge on Tbx3 to control ESC differentiation. Cell Stem Cell 24, 138–152.e8 (2019)."),[68](https://www.nature.com/articles/s41588-025-02305-z#ref-CR68 "Ivanova, N. et al. Dissecting self-renewal in stem cells with RNA interference. Nature 442, 533–538 (2006)."),[69](https://www.nature.com/articles/s41588-025-02305-z#ref-CR69 "Ho, L. et al. esBAF facilitates pluripotency by conditioning the genome for LIF/STAT3 signalling and by regulating polycomb function. Nat. Cell Biol. 13, 903–913 (2011)."); Fig. [5e](https://www.nature.com/articles/s41588-025-02305-z#Fig5)), linking BRG1 dosage to maintenance of mES cell identity.

To assess the phenotypic consequences, we cultured mES cells long-term in varying concentrations of dTAG13. Proliferation defects became noticeable at 10 nM dTAG13, with severe impairments at 100 nM[41](https://www.nature.com/articles/s41588-025-02305-z#ref-CR41 "Ho, L. et al. An embryonic stem cell chromatin remodeling complex, esBAF, is essential for embryonic stem cell self-renewal and pluripotency. Proc. Natl Acad. Sci. USA 106, 5181–5186 (2009).") (Extended Data Fig. [7a](https://www.nature.com/articles/s41588-025-02305-z#Fig14)). Alkaline phosphatase staining, a hallmark of undifferentiated states, revealed decreased naive mES cell populations at 1 nM, with higher concentrations resulting in loss of pluripotency (Extended Data Fig. [7b](https://www.nature.com/articles/s41588-025-02305-z#Fig14)). These results indicate that maintenance of pluripotency is more sensitive to BRG1 dosage than proliferation. We also observed a BRG1-dosage-sensitive inverse correlation in expression of oncogene _Myc_ and Polycomb-target _Hox_ genes (Fig. [5f](https://www.nature.com/articles/s41588-025-02305-z#Fig5) and Extended Data Fig. [7c](https://www.nature.com/articles/s41588-025-02305-z#Fig14)), consistent with their dysregulation upon BRG1 perturbation[14](https://www.nature.com/articles/s41588-025-02305-z#ref-CR14 "Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. Nat. Struct. Mol. Biol. 28, 501–511 (2021)."),[55](https://www.nature.com/articles/s41588-025-02305-z#ref-CR55 "Hodges, H. C. et al. Dominant-negative SMARCA4 mutants alter the accessibility landscape of tissue-unrestricted enhancers. Nat. Struct. Mol. Biol. 25, 61–72 (2018)."),[70](https://www.nature.com/articles/s41588-025-02305-z#ref-CR70 "Romero, O. A. et al. The tumour suppressor and chromatin-remodelling factor BRG1 antagonizes Myc activity and promotes cell differentiation in human cancer. EMBO Mol. Med. 4, 603–616 (2012)."). Notably, _Myc_ levels were significantly increased at dTAG13 concentrations of 1 nM or higher, accompanied by activation of MYC-target genes (Fig. [5g](https://www.nature.com/articles/s41588-025-02305-z#Fig5)), reinforcing _Myc_ as a dosage-sensitive target of BRG1.

### BRG1 dosage-sensitive regulation is conserved in human cells

To determine whether the BRG1 dosage-sensitive features observed in mES cells were conserved in other cell types, we investigated BRG1 dosage sensitivity in the BEAS-2B cell line, a normal human bronchial epithelial cell line widely used in lung carcinogenesis studies[71](https://www.nature.com/articles/s41588-025-02305-z#ref-CR71 "Zhong, M. et al. Malignant transformation of human bronchial epithelial cells induced by arsenic through STAT3/miR-301a/SMAD4 loop. Sci. Rep. 8, 13291 (2018)."),[72](https://www.nature.com/articles/s41588-025-02305-z#ref-CR72 "Park, Yhee, Kim, D., Dai, J. & Zhang, Z. Human bronchial epithelial BEAS-2B cells, an appropriate in vitro model to study heavy metals induced carcinogenesis. Toxicol. Appl. Pharmacol. 287, 240–245 (2015)."),[73](https://www.nature.com/articles/s41588-025-02305-z#ref-CR73 "Liu, Y. et al. Exosomes of A549 cells induced migration, invasion, and EMT of BEAS-2B cells related to let-7c-5p and miR-181b-5p. Front. Endocrinol. 13, 926769 (2022)."). Approximately 35% of lung cancers, including non-small-cell lung cancers, harbor mutations in genes encoding mSWI/SNF subunits[22](https://www.nature.com/articles/s41588-025-02305-z#ref-CR22 "Mittal, P. & Roberts, C. W. M. The SWI/SNF complex in cancer — biology, biomarkers and therapy. Nat. Rev. Clin. Oncol. 17, 435–448 (2020)."),[74](https://www.nature.com/articles/s41588-025-02305-z#ref-CR74 "Peinado, P. et al. Comprehensive analysis of SWI/SNF inactivation in lung adenocarcinoma cell models. Cancers 12, 3712 (2020)."). Conditional knockout of _Brg1_ and/or _Brm_ in mouse lungs leads to cancer development and progression[75](https://www.nature.com/articles/s41588-025-02305-z#ref-CR75 "Marquez-Vilendrer, S. B., Rai, S. K., Gramling, S. J., Lu, L. & Reisman, D. N. Loss of the SWI/SNF ATPase subunits BRM and BRG1 drives lung cancer development. Oncoscience 3, 322–336 (2016)."), suggesting that dosage of mSWI/SNF components is critical for lung cell homeostasis. Unlike mES cells, BEAS-2B cells express both BRG1 and BRM (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02305-z#Fig15)), which exhibit cooperative and antagonistic functions[76](https://www.nature.com/articles/s41588-025-02305-z#ref-CR76 "Raab, J. R., Runge, J. S., Spear, C. C. & Magnuson, T. Co-regulation of transcription by BRG1 and BRM, two mutually exclusive SWI/SNF ATPase subunits. Epigenetics Chromatin 10, 62 (2017)."). To simplify the BRG1 dosage analysis, we generated _BRM-_knockout BESA-2B cells (Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02305-z#Fig15)). Knockout of _BRM_ did not alter cell morphology (Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02305-z#Fig15)), BRG1 binding (Extended Data Fig. [8d](https://www.nature.com/articles/s41588-025-02305-z#Fig15)) or chromatin accessibility (Extended Data Fig. [9c](https://www.nature.com/articles/s41588-025-02305-z#Fig16); comparison of ATAC-seq results for wild-type versus _BRM_\-knockout, 0 nM), indicating that the remaining mSWI/SNF complexes containing only BRG1 could maintain cell and chromatin states similar to those of wild-type cells.

We depleted BRG1 using AU-15330, a PROTAC targeting BRG1 and BRM[60](https://www.nature.com/articles/s41588-025-02305-z#ref-CR60 "Xiao, L. et al. Targeting SWI/SNF ATPases in enhancer-addicted prostate cancer. Nature 601, 434–439 (2022).") that enables rapid BRG1 degradation within 2 h (Extended Data Fig. [8e](https://www.nature.com/articles/s41588-025-02305-z#Fig15)), and observed dosage-dependent degradation after the treatment (Fig. [6a](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). Population-wide BRG1 depletion was confirmed by immunostaining (Extended Data Fig. [8f,g](https://www.nature.com/articles/s41588-025-02305-z#Fig15)). Consistent with a previous report[60](https://www.nature.com/articles/s41588-025-02305-z#ref-CR60 "Xiao, L. et al. Targeting SWI/SNF ATPases in enhancer-addicted prostate cancer. Nature 601, 434–439 (2022)."), we observed degradation of PBAF-specific subunit PBRM1 (Extended Data Fig. [8h](https://www.nature.com/articles/s41588-025-02305-z#Fig15)). These results indicate that AU-15330 allows precise control of BRG1 levels, similar to the dTAG system.

**Fig. 6: Conserved dosage sensitivity in human lung epithelial cells.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02305-z/figures/6)

**a**, Western blot analysis of BRG1 in _BRM_\-knockout (KO) BEAS-2B cells after treatment with the indicated concentrations of AU-15330 for 24 h (top). Data are presented as the mean ± s.e.m. from four independent experiments. Quantification of relative BRG1 abundance following AU-15330 treatment (bottom). BRG1 expression relative to that of tubulin was normalized to DMSO-treated (0 nM) _BRM_\-KO BEAS-2B cells. **b**, PCA of ATAC-seq data across different AU-15330 concentrations. Cells were treated with AU-15330 for 24 h. **c**, Pie chart depicting relative percentages of significantly changed ATAC-seq signals at promoters, enhancers and insulators. **d**, Pie charts depicting percentages of significantly changed ATAC-seq signals at BRG1 target sites in promoters, enhancers and insulators. **e**, Heatmap showing profiles of protein and histone modifications (CUT&RUN and ChIP–seq[89](https://www.nature.com/articles/s41588-025-02305-z#ref-CR89 "Jose, C. C. et al. Epigenetic dysregulation by nickel through repressive chromatin domain disruption. Proc. Natl Acad. Sci. USA 111, 14631–14636 (2014).")) (red) and ATAC-seq (blue) for regions falling into genomic elements at the indicated dTAG13 concentration (bottom). The top panel shows aggregate coverage plots with mean enrichment at promoters (red), enhancers (cyan) and insulators (orange). **f**, Histogram of delta values for BRG1-dependent regulatory elements (promoters + enhancers + insulators). **g**, Histograms of delta values for BRG1-dependent promoters, enhancers and insulators. The red line indicates delta = 0. **h**, Chromatin accessibility changes for each of the five groups of BRG1-dependent enhancers in BEAS-2B cells. Lines show the mean of each group, shaded areas indicate the s.e.m. **i**, Heatmap showing relative enrichment of chromatin marks and MED1 binding across different groups of ATAC-seq peaks. **j**, Number of SEs in each group. **k**, _MYC_ expression relative to that of _GAPDH_ at different BRG1 levels. The level was further normalized by 0 nM AU-15330 conditions. Data represent three independent replicates, each with three technical replicates. Bars indicate the average.

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM4)

We then investigated changes in chromatin accessibility in response to varying BRG1 levels. As in mES cells, numbers of DARs progressively increased with BRG1 degradation levels (Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02305-z#Fig16)), with most DARs showing reduced accessibility (Extended Data Fig. [9b](https://www.nature.com/articles/s41588-025-02305-z#Fig16)). PCA revealed that wild-type and _BRM_\-knockout cells were clustered together under both control (0 nM AU-15330) and depletion (1,000 nM AU-15330) conditions, with a global shift correlated with AU-15330 concentration (Fig. [6b](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). BRG1 depletion predominantly affected enhancer accessibility (Fig. [6c](https://www.nature.com/articles/s41588-025-02305-z#Fig6)), with more than 90% of BRG1-bound enhancers showing reduced accessibility (Fig. [6d,e](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). By contrast, only about half of the BRG1-bound promoters and insulators were affected (Fig. [6d,e](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). These differences among regulatory elements were consistent with our observations in mES cells. However, the numbers and proportions of affected regions were higher across all regulatory elements in BEAS-2B cells (Figs. [2c](https://www.nature.com/articles/s41588-025-02305-z#Fig2) and [6d](https://www.nature.com/articles/s41588-025-02305-z#Fig6)), suggesting a greater dependence on mSWI/SNF for chromatin accessibility in this cell type.

To understand the role of BRG1 in chromatin regulation, we compared BRG1-dependent and BRG1-independent regulatory elements (Extended Data Fig. [9c,d](https://www.nature.com/articles/s41588-025-02305-z#Fig16)). BRG1-independent promoters exhibited greater H3K4me3 enrichment, a hallmark of strong promoters (Extended Data Fig. [9c](https://www.nature.com/articles/s41588-025-02305-z#Fig16)). Motif enrichment analysis revealed that both BRG1-dependent promoters and enhancers, as well as BRG1-independent enhancers, were enriched for AP-1 components (FOS and FOSL1) and AP-1-associated factor ATF3 motifs, consistent with their cooperative functions[77](https://www.nature.com/articles/s41588-025-02305-z#ref-CR77 "Xu, Y. Z., Thuraisingam, T., Marino, R. & Radzioch, D. Recruitment of SWI/SNF complex is required for transcriptional activation of the SLC11A1 gene during macrophage differentiation of HL-60 cells. J. Biol. Chem. 286, 12839–12849 (2011)."),[78](https://www.nature.com/articles/s41588-025-02305-z#ref-CR78 "Orlando, K. A. et al. Re-expression of SMARCA4/BRG1 in small cell carcinoma of ovary, hypercalcemic type (SCCOHT) promotes an epithelial-like gene signature through an AP-1-dependent mechanism. eLife 9, 1–57 (2020).") (Extended Data Fig. [9e](https://www.nature.com/articles/s41588-025-02305-z#Fig16)). Notably, BRG1-independent promoters were enriched for NFY and SP1 motifs, as observed in mES cells (Fig. [3a](https://www.nature.com/articles/s41588-025-02305-z#Fig3)), suggesting a conserved regulatory mechanism across different cell types.

To explore the dosage sensitivity of these changes, we divided regions with decreased chromatin accessibility into buffered and linear response categories (Fig. [6f](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). Promoters and insulators exhibited mostly buffered responses, whereas enhancers showed both linear and buffered responses (Fig. [6g](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). We further classified enhancers based on their dosage sensitivity (G1–G5 groups) (Extended Data Fig. [9f](https://www.nature.com/articles/s41588-025-02305-z#Fig16)). Buffered enhancers retained more ATAC-seq signal than linear enhancers at moderate BRG1 levels (Fig. [6h](https://www.nature.com/articles/s41588-025-02305-z#Fig6) and Extended Data Fig. [9g](https://www.nature.com/articles/s41588-025-02305-z#Fig16)). G1 buffered enhancers displayed strong H3K27ac enrichment, whereas G5 linear enhancers had weaker enrichment (Fig. [6i](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). Of the 276 SEs identified based on MED1 signals, 209 (75.7%) were affected by BRG1 depletion. Notably, SEs were preferentially enriched in the G1 group (Fig. [6j](https://www.nature.com/articles/s41588-025-02305-z#Fig6)), consistent with our findings in mES cells.

_MYC_ has a critical role in lung cancer[79](https://www.nature.com/articles/s41588-025-02305-z#ref-CR79 "Wallbillich, N. J. & Lu, H. Role of c-Myc in lung cancer: progress, challenges, and prospects. Chin. Med. J. Pulm. Crit. Care Med. 1, 129–138 (2023)."), and we found its expression to be highly sensitive to BRG1 dosage: it was upregulated 1.9-fold at 38% BRG1 (Fig. [6k](https://www.nature.com/articles/s41588-025-02305-z#Fig6)). By contrast, the expression of oncogene _KRAS_ was unaffected (Extended Data Fig. [9h](https://www.nature.com/articles/s41588-025-02305-z#Fig16)). Near-complete depletion of mSWI/SNF led to growth defects, whereas partial depletion had a limited impact (Extended Data Fig. [9i](https://www.nature.com/articles/s41588-025-02305-z#Fig16)).

Collectively, our results reveal conserved mSWI/SNF dosage sensitivity in regulation of chromatin accessibility, cellular homeostasis and _MYC_ expression. Modest reductions in BRG1 levels led to upregulation of _MYC_ without compromising proliferation, whereas further depletion impaired proliferation. These findings support the proposition that mSWI/SNF functions as a tumor suppressor while remaining essential for cancer cell growth[21](https://www.nature.com/articles/s41588-025-02305-z#ref-CR21 "Kadoch, C. et al. Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy. Nat. Genet. 45, 592–601 (2013)."),[22](https://www.nature.com/articles/s41588-025-02305-z#ref-CR22 "Mittal, P. & Roberts, C. W. M. The SWI/SNF complex in cancer — biology, biomarkers and therapy. Nat. Rev. Clin. Oncol. 17, 435–448 (2020)."),[80](https://www.nature.com/articles/s41588-025-02305-z#ref-CR80 "Navickas, S. M., Giles, K. A., Brettingham-Moore, K. H. & Taberlay, P. C. The role of chromatin remodeler SMARCA4/BRG1 in brain cancers: a potential therapeutic target. Oncogene 42, 2363–2373 (2023).") and further suggest that its dosage-sensitive regulatory effects contribute to cancer progression, particularly via control of _MYC_ oncogene activation.

Discussion
----------

By precisely controlling protein levels of BRG1, the essential ATPase subunit of the mSWI/SNF complex, we investigated the dosage-sensitive effects of BRG1 on chromatin binding, chromatin accessibility and transcription. We found that BRG1 binding was linearly sensitive to its dosage, whereas chromatin accessibility of BRG1 targets exhibited either linear or buffered sensitivity depending on the type of regulatory element, and the effects on transcription were largely buffered (Fig. [7](https://www.nature.com/articles/s41588-025-02305-z#Fig7)). Promoters and insulators typically showed buffered responses, whereas enhancers displayed both buffered and linear sensitivities, depending on their activity level (Fig. [7](https://www.nature.com/articles/s41588-025-02305-z#Fig7)). These findings highlight the complexity of dosage-sensitive gene regulation and underscore the critical role of BRG1 in maintaining precise control of chromatin architecture and transcription.

**Fig. 7: Model illustrating differential responses of genomic regions to BRG1 dosage.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02305-z/figures/7)

Model illustrating the dosage sensitivity of BRG1 binding, chromatin accessibility and transcriptional responses. Chromatin accessibility response is further categorized by type of regulatory element and enhancer activity, reflecting different patterns of dosage sensitivity across different genomic regions. Red, blue and yellow peaks represent BRG1-bound promoters, enhancers and insulators, respectively. Each row indicates a different BRG1 dosage level, with the bottom row representing the BRG1-depleted condition. Regions labeled ‘Linear’ exhibited changes in signals that were correlated proportionally with the BRG1 level, whereas those labeled ‘Buffered’ showed limited changes at certain levels of BRG1 depletion. ‘_Myc_+associate’ denotes _Myc_ and MYC-target gene expression. N/A, not applicable.

The response of enhancers to BRG1 dosage showed variable sensitivity, with strong enhancers showing buffered responses. This observation was consistent with those of previous studies of the temporal dynamics following mSWI/SNF inhibition, in which strong enhancers responded slowly and weak enhancers responded rapidly[11](https://www.nature.com/articles/s41588-025-02305-z#ref-CR11 "Schick, S. et al. Acute BAF perturbation causes immediate changes in chromatin accessibility. Nat. Genet. 53, 269–278 (2021)."),[12](https://www.nature.com/articles/s41588-025-02305-z#ref-CR12 "Iurlaro, M. et al. Mammalian SWI/SNF continuously restores local accessibility to chromatin. Nat. Genet. 53, 279–287 (2021)."). Despite these similarities, our data indicate that the dosage sensitivity and temporal dynamics are not entirely the same. One possible explanation for this is that the temporal dynamics are influenced by the chromatin state at the time of inhibitor treatment, whereas dosage sensitivity reflects the degree of enhancer dependency on mSWI/SNF and contributions of other regulatory factors. A crowded chromatin environment may hinder rapid changes, whereas more accessible regions may close gradually via spontaneous nucleosome sliding, which is assumed to take place over minutes to hours[81](https://www.nature.com/articles/s41588-025-02305-z#ref-CR81 "Nagae, F., Brandani, G. B., Takada, S. & Terakawa, T. The lane-switch mechanism for nucleosome repositioning by DNA translocase. Nucleic Acids Res. 49, 9066–9076 (2021)."). Furthermore, weak enhancers may depend more on BRG1 to maintain their accessibility. In buffered regions, the effects of BRG1 may be mediated through histone modifications or other regulatory mechanisms, such that only low levels of BRG1 are required.

We also found distinct patterns of BRG1 dependency among regulatory elements. Approximately one-third of the BRG1-bound regulatory elements (7,258 of 20,326) exhibited BRG1-dependent accessibility. The enrichment of NFYA and CTCF at BRG1-independent promoters and insulators, respectively, suggests that other factors may compensate for mSWI/SNF depletion. For instance, SNF2H appeared to function predominantly at BRG1-bound insulators, although some sites required both SNF2H and BRG1. Similarly, EP400/TIP60 has been reported to compensate for mSWI/SNF loss at promoters[16](https://www.nature.com/articles/s41588-025-02305-z#ref-CR16 "Martin, B. J. E. et al. Global identification of SWI/SNF targets reveals compensation by EP400. Cell 186, 5290–5307.e26 (2023)."), suggesting that cross-talk between mSWI/SNF and other remodelers might be specific to certain types of genomic element. This compensatory role of other remodelers might contribute to the BRG1 dosage sensitivity of accessibility regulation, as both promoters and insulators were enriched for buffered response to BRG1 dosage. In addition, we observed accumulation of Polycomb proteins at BRG1-dependent promoters following BRG1 loss, and expression of _Hox_ genes, which are known to be Polycomb targets, was negatively correlated with BRG1 levels. These results suggest that Polycomb proteins may be redistributed in response to BRG1 dosage, as suggested by a study of acute BRG1 depletion[14](https://www.nature.com/articles/s41588-025-02305-z#ref-CR14 "Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. Nat. Struct. Mol. Biol. 28, 501–511 (2021)."). A recent study also found that mSWI/SNF-dependent promoters were enriched for TATA-box elements and flanked by mSWI/SNF-bound enhancers, whereas GC-rich housekeeping genes were less affected[82](https://www.nature.com/articles/s41588-025-02305-z#ref-CR82 "Basurto-Cayuela, L. et al. SWI/SNF-dependent genes are defined by their chromatin landscape. Cell Rep. 43, 113855 (2024)."). These findings raise the question of whether differences in dependency arise from compensatory mechanisms or distinct regulatory processes governing chromatin accessibility.

Our results show that the BRG1 dosage sensitivity of enhancer accessibility is linked to enhancer activity. This differential dosage sensitivity response to intermediate BRG1 levels, with decreased accessibility for typical enhancers but maintenance of accessibility for SEs, is intriguing as SEs are often associated with oncogene regulation in cancers[63](https://www.nature.com/articles/s41588-025-02305-z#ref-CR63 "Hnisz, D. et al. Transcriptional super-enhancers connected to cell identity and disease. Cell 155, 934 (2013)."). A similar pattern, with typical enhancers but not SEs being affected, has been observed in _SMARCB1_\-defectient rhabdoid tumors, in which mSWI/SNF stability is compromised[24](https://www.nature.com/articles/s41588-025-02305-z#ref-CR24 "Wang, X. et al. SMARCB1-mediated SWI/SNF complex function is essential for enhancer regulation. Nat. Genet. 49, 289–295 (2016)."),[25](https://www.nature.com/articles/s41588-025-02305-z#ref-CR25 "Nakayama, R. T. et al. SMARCB1 is required for widespread BAF complex–mediated activation of enhancers and bivalent promoters. Nat. Genet. 49, 1613–1623 (2017)."). In these contexts, residual mSWI/SNF activity is critical for expression of oncogenes, including MYC-target genes[83](https://www.nature.com/articles/s41588-025-02305-z#ref-CR83 "Moe, K. C. et al. The SWI/SNF ATPase BRG1 facilitates multiple pro-tumorigenic gene expression programs in SMARCB1-deficient cancer cells. Oncogenesis 11, 30 (2022)."). This suggests that selective enhancer dysregulation in response to altered mSWI/SNF dosage may be a general mechanism contributing to oncogenesis. Notably, the role of BRG1 in _MYC_ regulation appears to be context-dependent, promoting _MYC_ expression in leukemia and prostate cancer[16](https://www.nature.com/articles/s41588-025-02305-z#ref-CR16 "Martin, B. J. E. et al. Global identification of SWI/SNF targets reveals compensation by EP400. Cell 186, 5290–5307.e26 (2023)."),[60](https://www.nature.com/articles/s41588-025-02305-z#ref-CR60 "Xiao, L. et al. Targeting SWI/SNF ATPases in enhancer-addicted prostate cancer. Nature 601, 434–439 (2022)."),[84](https://www.nature.com/articles/s41588-025-02305-z#ref-CR84 "Hohmann, A. F. et al. Sensitivity and engineered resistance of myeloid leukemia cells to BRD9 inhibition. Nat. Chem. Biol. 12, 672–679 (2016)."),[85](https://www.nature.com/articles/s41588-025-02305-z#ref-CR85 "Bossen, C. et al. The chromatin remodeler Brg1 activates enhancer repertoires to establish B cell identity and modulate cell growth. Nat. Immunol. 16, 775–784 (2015)."),[86](https://www.nature.com/articles/s41588-025-02305-z#ref-CR86 "Shi, J. et al. Role of SWI/SNF in acute leukemia maintenance and enhancer-mediated Myc regulation. Genes Dev. 27, 2648–2662 (2013).") but repressing it in cardiomyocytes and lung cancer cells[70](https://www.nature.com/articles/s41588-025-02305-z#ref-CR70 "Romero, O. A. et al. The tumour suppressor and chromatin-remodelling factor BRG1 antagonizes Myc activity and promotes cell differentiation in human cancer. EMBO Mol. Med. 4, 603–616 (2012)."),[87](https://www.nature.com/articles/s41588-025-02305-z#ref-CR87 "Orvis, T. et al. BRG1/SMARCA4 inactivation promotes non-small cell lung cancer aggressiveness by altering chromatin organization. Cancer Res. 74, 6486–6498 (2014)."),[88](https://www.nature.com/articles/s41588-025-02305-z#ref-CR88 "Willis, M. S. et al. BRG1 and BRM function antagonistically with c-MYC in adult cardiomyocytes to regulate conduction and contractility. J. Mol. Cell. Cardiol. 105, 99–109 (2017)."). Our finding that _MYC_ is sensitive to a decrease in BRG1 dosage is consistent with observations of BRG1 disruption or rescue in lung cancer cell lines[70](https://www.nature.com/articles/s41588-025-02305-z#ref-CR70 "Romero, O. A. et al. The tumour suppressor and chromatin-remodelling factor BRG1 antagonizes Myc activity and promotes cell differentiation in human cancer. EMBO Mol. Med. 4, 603–616 (2012)."),[74](https://www.nature.com/articles/s41588-025-02305-z#ref-CR74 "Peinado, P. et al. Comprehensive analysis of SWI/SNF inactivation in lung adenocarcinoma cell models. Cancers 12, 3712 (2020)."). These studies suggest that BRG1 dosage sensitivity may have a key role in lung cancer biology. In addition, dosage-sensitive regulation of mSWI/SNF is complicated by the presence of redundant ATPase subunits, BRG1 and BRM. In both mES cells and _BRM_\-knockout BEAS-2B cells, BRG1 alone supported chromatin accessibility, and its depletion alone was sufficient to reduce accessibility of SEs. By contrast, both BRG1 and BRM need to be depleted to impair SEs in HAP1 cells[11](https://www.nature.com/articles/s41588-025-02305-z#ref-CR11 "Schick, S. et al. Acute BAF perturbation causes immediate changes in chromatin accessibility. Nat. Genet. 53, 269–278 (2021)."), indicating functional redundancy. Thus, the BRG1 dosage sensitivity observed in this study may reflect a broader principle of mSWI/SNF dosage-sensitive regulation across diverse cell types.

In conclusion, our study reveals molecular mechanisms underlying BRG1 dosage sensitivity with different effects on chromatin binding, accessibility and transcription. These findings deepen our understanding of the function of mSWI/SNF in stem cell biology and emphasize the importance of precise chromatin regulation in maintenance of cellular identity. The insights presented here represent a foundation for future exploration of how alterations in mSWI/SNF dosage affect accessibility and transcription in development, cancer and other disease contexts.

Methods
-------

This study complied with all relevant ethical regulations and was approved by the Harvard Committee on Microbiological Safety, which serves as the Institutional Biosafety Committee for Harvard Medical School, under ID 22-079-A03. All animal experiments were conducted in accordance with the National Institutes of Health Guide for the Care and Use of Laboratory Animals and approved by the Institutional Animal Care and Use Committee of Boston Children’s Hospital and Harvard Medical School under protocol number IS00000270-9.

### Cell culture, establishment of cell lines and chemical treatment

The mES cells were derived from blastocysts of C57BL/6J mice (Jax, 000664) and cultured on 0.1% gelatin-coated plates under 2i/LIF conditions. Cells were maintained in DMEM (Gibco, 11960069) supplemented with 15% fetal bovine serum (FBS, Sigma-Aldrich, F6178), 2 mM GlutaMAX (Gibco, 35050061), 1 mM sodium pyruvate (Gibco, 11360), 1× MEM nonessential amino acids (Gibco, 11140050), 0.084 mM 2-mercaptoethanol (Gibco, 21985023), 100 U ml−1 penicillin–streptomycin (Gibco, 15140122), 1000 IU ml−1 LIF (Millipore, ESG1107), 0.5 μM PD0325901 (Tocris, 4192) and 3 μM CHIR99021 (Tocris, 4423). BEAS-2B cells were purchased from the American Type Culture Collection (ATCC, CRL-3588). Cells were maintained in RPMI1640 (Gibco, 11875093) supplemented with 10% FBS and 100 U ml−1 penicillin–streptomycin. _Drosophila_ S2 cells, purchased from Gibco (R69007), were grown in Schneider’s _Drosophila_ medium (Thermo Fisher Scientific, catalog no. 21720024) supplemented with 10% heat-inactivated FBS, 0.1 % Plutonic F-68 (Gibco, 24040032) and 1% penicillin–streptomycin at 28 °C.

To generate the _Brg1_\-dTAG and _Snf2h_\-dTAG cell lines, mES cells were cotransfected with targeting plasmids and pX330.puro (Addgene, catalog no. 110403) using Lipofectamine 2000 (Invitrogen, 11668030). After transfection, cells were selected with puromycin (Gibco, A1113803). Colonies derived from single cells were picked and genotyped for successful integration, followed by validation and analysis. Sequences of single guide RNAs are listed in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02305-z#MOESM3).

To generate the _BRM_\-knockout cell line, BEAS-2B cells were cotransfected with pL-CRISPR.EFS.GFP (Addgene, catalog no. 57818) and pL-CRISPR.EFS.tRFP (Addgene, catalog no. 57819) using Lipofectamine 2000. Two days after transfection, GFP and RFP double-positive cells were sorted to 96-well plates with a single cell per well. Knockout was confirmed by genotyping, followed by western blot analysis. Genotyping primers are listed in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02305-z#MOESM3).

For chemical treatment, mES cells and BEAS-2B were seeded in 12-well plates the day before treatment. Cells were treated with dTAG13 (100 pM to 100 nM) or AU-15330 (10 nM to 1,000 nM) by complete replacement of the culture medium. After the specified treatment period, cells were harvested using TrypLE (Gibco, 12605010) or Trypsin-EDTA (Gibco, 25200072) and resuspended in fresh culture medium for downstream analysis.

### Western blot

Cells were resuspended at a final concentration of 1 × 104 cells μl−1 in 1× LDS buffer (Invitrogen, NP0007). Protein samples were prepared by sonication using a Bioruptor 300 (Diagnode), followed by heat denaturation. Proteins were separated by 3–8% SDS–PAGE, then transferred onto nitrocellulose or polyvinylidene fluoride (PVDF) membranes (Invitrogen, IB23002 or IB24002) using the iBlot2 system (Invitrogen). Membranes were blocked in 3% bovine serum albumin (Jackson ImmunoResearch, 001-000-162) prepared in Tris-buffered saline (Cell Signaling Technology, 12498S) with 0.1% Tween-20 (TBS-T) for 1 h at room temperature. After blocking, membranes were incubated overnight at 4 °C with primary antibodies. The next day, membranes were washed four times with TBS-T, followed by incubation with secondary antibodies conjugated to fluorescent-dye or HRP at room temperature. Then, membranes were washed four times with TBS-T, and those incubated with HRP antibodies were developed using SuperSignal West Pico PLUS Chemiluminescent Substrate (Thermo, 34580) before being imaged on an iBright 1500 system. The sources of all antibodies are listed in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02305-z#MOESM3).

### Immunofluorescence

Cells were seeded on a collagen-coated eight-well culture slide (Corning, 354630) and treated the following day with dTAG13 or AU-15330 by complete replacement of the culture medium. Twenty-four hours after treatment, cells were fixed in 4% paraformaldehyde (PFA) for 20 min and permeabilized with PBS containing 0.5 % Triton X-100. Cells were incubated overnight at 4 °C with primary antibodies (listed in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02305-z#MOESM3)), followed by incubation with secondary antibody for 2 h at room temperature. They were then mounted using Vectashield (Vector Laboratories, H-1200-10) and imaged with a confocal microscope (Zeiss, LSM800). The relative signal level was calculated by normalizing to the mean intensity of the dimethyl sulfoxide (DMSO)-treated control.

### CUT&RUN, ATAC-seq and RNA-seq library preparation and sequencing

CUT&RUN was performed as previously described[90](https://www.nature.com/articles/s41588-025-02305-z#ref-CR90 "Chen, Z., Djekidel, M. N. & Zhang, Y. Distinct dynamics and functions of H2AK119ub1 and H3K27me3 in mouse preimplantation embryos. Nat. Genet. 53, 551–563 (2021)."),[91](https://www.nature.com/articles/s41588-025-02305-z#ref-CR91 "Wang, M., Chen, Z. & Zhang, Y. CBP/p300 and HDAC activities regulate H3K27 acetylation dynamics and zygotic genome activation in mouse preimplantation embryos. EMBO J. 41, e112012 (2022)."). In brief, 50,000 mES cells were incubated with activated Concanavalin A Magnetic Beads (Polysciences, 86057-3). Samples were then incubated overnight with the antibody, followed by incubation with pA-MNase (home-made). DNA was fragmented by incubation with CaCl2 and subsequently purified using a phenol-chloroform extraction. Sequencing libraries were prepared using a NEBNext Ultra II DNA Library Preparation Kit for Illumina (New England Biolabs, E7645S).

ATAC-seq was performed as previously described[42](https://www.nature.com/articles/s41588-025-02305-z#ref-CR42 "Zhou, C., Wang, M., Zhang, C. & Zhang, Y. The transcription factor GABPA is a master regulator of naive pluripotency. Nat. Cell Biol. 27, 48–58 (2025)."),[92](https://www.nature.com/articles/s41588-025-02305-z#ref-CR92 "Zhang, C., Wang, M., Li, Y. & Zhang, Y. Profiling and functional characterization of maternal mRNA translation during mouse maternal-to-zygotic transition. Sci. Adv. 8, eabj3967 (2022)."). In brief, 10,000 to 20,000 mES cells were treated with adapter-loaded Tn5 transposase (home-made) at 37 °C. The reaction was stopped by adding a buffer containing SDS, and proteins were digested with Proteinase K at 55 °C. Tween-20 was added to quench the SDS. Sequencing libraries were prepared using NEBNext High-Fidelity 2× PCR Master Mix (New England Biolabs, M0541S). RNA-seq libraries were prepared using the SMART-Seq Stranded Kit (Takara Bio, 63444) using 5,000 mES cells. All libraries were sequenced on the NextSeq 550 system (Illumina) using paired-end 75-bp reads.

### Co-immunoprecipitation

Antibody beads were prepared using a Dynabeads Co-Immunoprecipitation Kit (Thermo, 14321D). In brief, 0.4 mg of beads were coupled with 1.4 μg of antibody overnight, followed by washing with HB, LB and SB buffers. Suspensions of nuclei were prepared using a modified protocol[43](https://www.nature.com/articles/s41588-025-02305-z#ref-CR43 "Gatchalian, J. et al. A non-canonical BRD9-containing BAF chromatin remodeling complex regulates naive pluripotency in mouse embryonic stem cells. Nat. Commun. 9, 5139 (2018)."). About 2.0–3.5 × 107 cells were used for each experiment. Cells were swelled in buffer A (10 mM HEPES, pH 7.9, 1.5 mM MgCl2 and 10 mM KCl) supplemented with 1 mM DTT and Complete Protease Inhibitor Cocktail (Roche, 11836170001), then lysed by homogenization using a 25-gauge needle with 5–8 strokes. Nuclei were pelleted by centrifugation at 900_g_ for 5 min and resuspended in buffer C (20 mM HEPES, pH 7.9, 20% glycerol, 420 mM NaCl, 1.5 mM MgCl2 and 0.2 mM EDTA) supplemented with 1 mM DTT and protease inhibitor cocktail. After a 30-min incubation on ice, the sample was cleared by centrifugation at 21,100_g_ for 10 min. The supernatant was collected and flash-frozen in liquid nitrogen for further use.

Before immunoprecipitation, nuclear lysates were diluted with two-thirds of the original volume of 20 mM HEPES, pH 7.9, and 0.3% NP-40 to reduce the NaCl concentration. The diluted nuclear lysates were incubated with the antibody–bead solution for 3 h at 4 °C. The samples were then washed with wash buffer (50 mM Tris, pH 8.0, 150 mM NaCl, 1 mM EDTA, 10% glycerol and 0.5% Triton X-100). Proteins were eluted in 1× LDS buffer by boiling and subsequently analyzed by western blotting.

### Cell proliferation assay

Cells were seeded in 96-well plates at 300 cells per well for 6 days, 1,500 cells per well for 3 days and 5,000 cells per well for 1 day of treatment. After the specified treatment period, DNA was stained with a CyQUANT NF Cell Proliferation Assay kit (Invitrogen, C35007) and detected with a CLARIOstar Plus plate reader (BMG LABTECH). Each sample was measured in triplicate, and the mean fluorescence intensity was used for calculations. Relative signal levels were normalized to the DMSO-treated control.

### Alkaline phosphatase staining

Cells were seeded in 12-well plates at 10,000 cells per well and then treated with dTAG13 at the specified concentrations for 6 days. At the endpoint of the treatment period, cells were washed gently with phosphate-buffered saline, stained using a StemAb Alkaline Phosphatase Staining Kit II (Amsbio, AMS.00-0055) and imaged with a BZ-X810 microscope (KEYENCE). Colonies were scored manually.

### Quantitative RT–PCR analysis

Total RNA was isolated from cells using TRIzol reagent (Invitrogen). cDNA was synthesized with SuperScript IV, and RT–PCR was performed using Fast SYBR Green Master Mix (Applied Biosystems) on a QuantStudio 6 Pro (Applied Biosystems). Relative mRNA expression levels were calculated using a standard curve, with _GAPDH_ as an internal control. Primer sequences are listed in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02305-z#MOESM3).

### Data analysis

#### CUT&RUN data analysis

Raw reads with sequencing adaptors or low-quality ends reads were trimmed using trim\_galore (v.0.6.7) with ‘–illumina -q 20–length 30–paired -j 2’. Then, the clean reads were aligned to the mouse mm10 genome via bowtie2 (v.2.5.1) with default parameters. The bam files were sorted with samtools (v.1.6). Duplicates were marked and removed with Picard (v.2.26.4-0); then, MACS2 (v.2.2.7.1) was used to call significant peaks with parameters ‘-f BAMPE–keep-dup all -g mm -q 0.05–broad–max-gap 200’. To obtain highly conserved BRG1 binding peaks, the peaks detected in both biological repeats were defined as binding sites. ChIPseeker (annotatePeak function) was used to annotate BRG1 binding peaks. Peaks located near transcription start sites (±1,000 base pairs) were defined as promoter binding. Peaks overlapping with CTCF-binding sites (CTCF ChIP–seq data were downloaded from GEO: [GSE30206](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30206)) were classified as insulator binding, and the remaining peaks were classified as enhancer binding. For Integrative Genomics Viewer (IGV) visualization of the signals, the read coverage was normalized by reads per kilobase per million (RPKM) using bamCoverage (v.3.5.5).

#### ATAC-seq data analysis

The same method as for CUT&RUN data analysis was used to remove sequencing adaptors or low-quality reads, and the clean reads were aligned to mm10 via bowtiew2 with parameter ‘-q -X 2000’. Then, Picard was used to remove duplicates, and MACS was used to call significant peaks with parameters ‘-f BAMPE–keep-dup all -g mm -B -q 0.05’.

We first tried to use _Drosophila_ spike-in DNA to construct the ATAC-seq library. The sequencing reads were aligned to _Drosophila_ genome dm6, and unmapped reads were further mapped to the mouse genome. The spike-in read numbers varied significantly from sample to sample; this did not reflect the actual biology but may have been due to inconsistent amounts of spike-in DNA added across different samples. Therefore, spike-in DNA could not be applied for normalization in our data. The total peak numbers showed limited differences among all samples (peak numbers: 0 nM 43,212; 0.3 nM 51,119; 1 nM 40,673; 3 nM 40,164; 10 nM 35,207; 100 nM 37,441). Most ATAC-seq peaks did not change after BRG11 depletion, especially the strong peaks. There were no global ATAC-seq signal changes. The strong peaks could be used for calculation of size factors. The upper-quartile normalization method from the calcNormFactors function in edgeR (v.3.32.1) was used to calculate 95% quantiles of the counts for each library, after removing genes that were zero in all libraries.

For differentially accessible peak detection, the ATAC-seq read counts for each BRG1-binding site were calculated using multiBamCov (v.2.30.0). The read count matrix was then loaded into edgeR. After normalization and estimation of dispersions, a generalized linear model was fitted, and likelihood ratio tests were applied (glmFit and glmLRT functions). Peaks with a false discovery rate of less than 0.05 were defined as differentially accessible peaks.

For motif enrichment analysis, the findMotifsGenome.pl function in HOMER2 ([http://homer.ucsd.edu/homer/motif/](http://homer.ucsd.edu/homer/motif/)) was applied to bed files. The ‘-mask‘ parameter was applied to use the repeat-masked sequence. Briefly, HOMER identifies overrepresented motifs within a set of peaks. Then, it selects random background regions as a background set. Finally, it screens its library of reliable motifs against the target and background sequences to determine enrichment. Enrichment _P_ values for all motifs in all peaks were then corrected for multiple testing by calculating the Benjamini-adjusted _P_ value. Motifs with an adjusted _P_ value less than 0.05 were considered to be significantly enriched.

#### RNA-seq data analysis

The same method as for CUT&RUN and ATAC-seq data analyses was used to remove sequencing adaptors or low-quality reads, and the clean reads were aligned to the mouse genome using HISAT2 (v.2.2.1). To quantify gene expression, featureCounts (v.2.0.1) was used to count the sequencing reads within each gene. Then, edgeR (v.3.32.1) was employed for normalization and differential expression analysis. Genes with reads per kilobase per million less than 1 were defined as having low expression and excluded from the differential expression analysis. Then, DEGs were identified using the likelihood ratio test (glmFit and glmLRT functions in edgeR). Genes with false discovery rate less than 0.05 and absolute value of fold change greater than 1.5 were defined as DEGs. enrichR (v.3.4) was used for gene ontology enrichment analysis of DEGs.

#### Modeling buffered and linear response peaks

For each BRG1-binding peak, read counts from all dosage-treated samples were normalized to the control sample. Then, _z_\-scores were calculated as follows: _z_ = (_x_ − _μ_)/σ, where _x_ is the raw signal, _μ_ is the mean across all samples and _σ_ is the standard deviation of all samples. Peaks with maximum _z_\-score in any sample were excluded from modeling. Two models—a buffered model and a linear model—were then fitted to assess changes in CUT&RUN, ATAC-seq and transcriptome data relative to BRG1 protein levels as measured by western blotting. The linear model was

${\\mathrm{RC}} \\sim 1-{\\mathrm{BRG}},$

and the buffered model was

${\\mathrm{RC}} \\sim 1-{{\\mathrm{BRG}}}^{3},$

where RC is a vector of normalized read counts among all samples, and BRG is the BRG1 protein level of all samples. The residual sum of squares, RSS, was calculated for each peak in each model as follows:

${\\rm{RSS}}=\\mathop{\\sum }\\limits\_{k=i}^{n}\\left(\\;{y}\_{i}-\\bar{y}\\right),$

and a delta value of residuals was computed to quantify the distance between the two models, enabling a comparison of their fit to the data:

${\\varDelta}={{\\mathrm{RSS}}}\_{{\\mathrm{sensitive}}}-{{\\mathrm{RSS}}}\_{{\\mathrm{buffered}}}.$

Here a positive delta value indicates that the peak is closer to the linear model than the buffered model, whereas a negative delta value indicates that the peak is closer to the buffered model.

#### SE analysis

A total of 231 SEs of mouse ES cells were obtained from R. Young’s laboratory[62](https://www.nature.com/articles/s41588-025-02305-z#ref-CR62 "Whyte, W. A. et al. Master transcription factors and Mediator establish super-enhancers at key cell identity genes. Cell 153, 307 (2013)."). The original genomic locations of these enhancers were based on mm9 genome assembly. We converted the genome coordinates from mm9 to mm10 using the LiftOver tool from USCS utilities. The overlaps of SEs with other genomic features were measured using bedtools (v.2.30.0). The SEs of BEAS-2B cells were defined as previously described[62](https://www.nature.com/articles/s41588-025-02305-z#ref-CR62 "Whyte, W. A. et al. Master transcription factors and Mediator establish super-enhancers at key cell identity genes. Cell 153, 307 (2013).").

#### Chromatin states enrichment analysis

A chromatin states map for mES cells generated using ChIP–seq data for E14 cells from the ENCODE project was downloaded from [https://github.com/guifengwei/ChromHMM\_mESC\_mm10](https://github.com/guifengwei/ChromHMM_mESC_mm10). ChromHMM was used to identify chromatin states according to combinations of histone modifications. A 12-state model was trained, containing mostly promoters, enhancers, repressed Polycomb sites, transcription elongation regions, CTCF-bound sites and intergenic regions. To calculate chromatin state enrichments of given genomic regions, we used Fisher’s exact test to compare enrichment to background regions. The background regions were generated by shuffling regions using bedtools (shuffleBed).

#### Training of ChromBPNet models

We used ChromBPNet to predict the chromatin accessibilities of the BRG1-binding sites. ChromBPNet was downloaded from GitHub ([https://github.com/kundajelab/chrombpnet](https://github.com/kundajelab/chrombpnet)), and we followed the tutorial provided to train models for ATAC-seq of untreated (0 nM) and treated (100 nM) mES cells. We first generated background regions for training a bias model to capture the Tn5 enzyme effect. We then used this model to regress out the effects of the enzyme from our ATAC-seq profiles. Next, bias-factorized ChromBPNet models were trained to predict the base-pair resolution chromatin accessibility profiles. Finally, we generated base-pair resolution importance scores for all BRG1-binding sites as described in the ChromBPNet tutorial. We further used the TF-MODISCO ([https://github.com/kundajelab/tfmodisco](https://github.com/kundajelab/tfmodisco)) motif discovery algorithm to identify sequence motifs enriched in different enhancers. We then used a previously described method[66](https://www.nature.com/articles/s41588-025-02305-z#ref-CR66 "Pratt, H. E. et al. Using a comprehensive atlas and predictive models to reveal the complexity and evolution of brain-active regulatory elements. Sci. Adv. 10, eadj4452 (2024).") to build a hidden Markov model to identify sequences with a significant positive or negative influence on chromatin accessibility within the BRG1-binding sites.

#### Statistics and reproducibility

No statistical methods were used to predetermine sample sizes, but our sample sizes are similar to those reported in previous publications[11](https://www.nature.com/articles/s41588-025-02305-z#ref-CR11 "Schick, S. et al. Acute BAF perturbation causes immediate changes in chromatin accessibility. Nat. Genet. 53, 269–278 (2021)."),[12](https://www.nature.com/articles/s41588-025-02305-z#ref-CR12 "Iurlaro, M. et al. Mammalian SWI/SNF continuously restores local accessibility to chromatin. Nat. Genet. 53, 279–287 (2021)."),[42](https://www.nature.com/articles/s41588-025-02305-z#ref-CR42 "Zhou, C., Wang, M., Zhang, C. & Zhang, Y. The transcription factor GABPA is a master regulator of naive pluripotency. Nat. Cell Biol. 27, 48–58 (2025)."). The experiments were not randomized. Drosophila DNA for ATAC-seq and spike-in yeast DNA for CUT&RUN were excluded from analyses owing to substantial variation in spike-in read counts across samples, possibly caused by inconsistent spike-in DNA amounts. Three data points were excluded from the cell proliferation assay because their values exceeded the detectable range or were suspected to result from uneven cell seeding. Data distribution was assumed to be normal, but this was not formally tested. Data collection and analysis were not performed blind to the conditions of the experiments.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02305-z#MOESM1) linked to this article.

Data availability
-----------------

The sequencing data reported in this study are available via the Gene Expression Omnibus (GEO) with accession numbers [GSE274469](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE274469) and [GSE294015](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE294015). All public datasets used in this study are listed in Supplementary Table [4](https://www.nature.com/articles/s41588-025-02305-z#MOESM3), including [GSE64825](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE64825) (ref. [6](https://www.nature.com/articles/s41588-025-02305-z#ref-CR6 "De Dieuleveult, M. et al. Genome-wide nucleosome specificity and function of chromatin remodellers in ES cells. Nature 530, 113–116 (2016).")), [GSE198517](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198517) (ref. [16](https://www.nature.com/articles/s41588-025-02305-z#ref-CR16 "Martin, B. J. E. et al. Global identification of SWI/SNF targets reveals compensation by EP400. Cell 186, 5290–5307.e26 (2023).")), [GSE87819](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87819) (ref. [48](https://www.nature.com/articles/s41588-025-02305-z#ref-CR48 "King, H. W. & Klose, R. J. The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells. eLife 6, e22631 (2017).")), [GSE115110](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115110) (ref. [56](https://www.nature.com/articles/s41588-025-02305-z#ref-CR56 "Oldfield, A. J. et al. NF-Y controls fidelity of transcription initiation at gene promoters through maintenance of the nucleosome-depleted region. Nat. Commun. 10, 3072 (2019).")), [GSE44286](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44286) (ref. [62](https://www.nature.com/articles/s41588-025-02305-z#ref-CR62 "Whyte, W. A. et al. Master transcription factors and Mediator establish super-enhancers at key cell identity genes. Cell 153, 307 (2013).")), [GSE56098](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56098) (ref. [93](https://www.nature.com/articles/s41588-025-02305-z#ref-CR93 "Buecker, C. et al. Reorganization of enhancer patterns in transition from naive to primed pluripotency. Cell Stem Cell 14, 838–853 (2014).")), [GSE155062](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155062) (ref. [94](https://www.nature.com/articles/s41588-025-02305-z#ref-CR94 "Bleckwehl, T. et al. Enhancer-associated H3K4 methylation safeguards in vitro germline competence. Nat. Commun. 12, 5771 (2021).")), [GSE30203](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30203) (ref. [95](https://www.nature.com/articles/s41588-025-02305-z#ref-CR95 "Stadler, M. B. et al. DNA-binding factors shape the mouse methylome at distal regulatory regions. Nature 480, 490–495 (2011).")), [GSE56839](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56839) (ref. [96](https://www.nature.com/articles/s41588-025-02305-z#ref-CR96 "Oldfield, A. J. et al. Histone-fold domain protein NF-Y promotes chromatin accessibility for cell type-specific master transcription factors. Mol. Cell. 55, 708–722 (2014).")), [GSE11431](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE11431) (ref. [97](https://www.nature.com/articles/s41588-025-02305-z#ref-CR97 "Chen, X. et al. Integration of external signaling pathways with the core transcriptional network in embryonic stem cells. Cell 133, 1106–1117 (2008).")) and [GSE56051](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56051) (ref. [89](https://www.nature.com/articles/s41588-025-02305-z#ref-CR89 "Jose, C. C. et al. Epigenetic dysregulation by nickel through repressive chromatin domain disruption. Proc. Natl Acad. Sci. USA 111, 14631–14636 (2014).")). ChromBPNet models are available via Figshare at [https://doi.org/10.6084/m9.figshare.28705430](https://doi.org/10.6084/m9.figshare.28705430) (ref. [98](https://www.nature.com/articles/s41588-025-02305-z#ref-CR98 "Zhang, C. ChromBPNet models. Figshare 
https://doi.org/10.6084/m9.figshare.28705430
(2025).")). [Source data](https://www.nature.com/articles/s41588-025-02305-z#Sec33) are provided with this paper.

Code availability
-----------------

The code for analysis of the sequencing data is available via GitHub at [https://github.com/YiZhang-lab/Brg1\_dTag](https://github.com/YiZhang-lab/Brg1_dTag) and Zenodo at [https://doi.org/10.5281/zenodo.15951452](https://doi.org/10.5281/zenodo.15951452) (ref. [99](https://www.nature.com/articles/s41588-025-02305-z#ref-CR99 "Zhang, C. Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. Zenodo 
https://doi.org/10.5281/zenodo.15951452
(2025).")).

References
----------

1.  Clapier, C. R., Iwasa, J., Cairns, B. R. & Peterson, C. L. Mechanisms of action and regulation of ATP-dependent chromatin-remodelling complexes. _Nat. Rev. Mol. Cell Biol._ **18**, 407–422 (2017).
    
2.  Kim, S. & Wysocka, J. Deciphering the multi-scale, quantitative _cis_\-regulatory code. _Mol. Cell_ **83**, 373–392 (2023).
    
3.  Bell, O., Tiwari, V. K., Thomä, N. H. & Schübeler, D. Determinants and dynamics of genome accessibility. _Nat. Rev. Genet._ **12**, 554–564 (2011).
    
4.  Li, B., Carey, M. & Workman, J. L. The role of chromatin during transcription. _Cell_ **128**, 707–719 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXis12ju7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17320508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20chromatin%20during%20transcription&journal=Cell&volume=128&pages=707-719&publication_year=2007&author=Li%2CB&author=Carey%2CM&author=Workman%2CJL) 
    
5.  Morris, S. A. et al. Overlapping chromatin remodeling systems collaborate genome-wide at dynamic chromatin transitions. _Nat. Struct. Mol. Biol._ **21**, 73 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvV2hsbzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24317492)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overlapping%20chromatin%20remodeling%20systems%20collaborate%20genome-wide%20at%20dynamic%20chromatin%20transitions&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=21&publication_year=2014&author=Morris%2CSA) 
    
6.  De Dieuleveult, M. et al. Genome-wide nucleosome specificity and function of chromatin remodellers in ES cells. _Nature_ **530**, 113–116 (2016).
    
7.  Barisic, D., Stadler, M. B., Iurlaro, M. & Schübeler, D. Mammalian ISWI and SWI/SNF selectively mediate binding of distinct transcription factors. _Nature_ **569**, 136–140 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVSmt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30996347)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6522387)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mammalian%20ISWI%20and%20SWI%2FSNF%20selectively%20mediate%20binding%20of%20distinct%20transcription%20factors&journal=Nature&volume=569&pages=136-140&publication_year=2019&author=Barisic%2CD&author=Stadler%2CMB&author=Iurlaro%2CM&author=Sch%C3%BCbeler%2CD) 
    
8.  Stewart-Morgan, K. R., Petryk, N. & Groth, A. Chromatin replication and epigenetic cell memory. _Nat. Cell Biol_. **22**, 361–371 (2020).
    
9.  Liu, Y. et al. Widespread mitotic bookmarking by histone marks and transcription factors in pluripotent stem cells. _Cell Rep._ **19**, 1283–1293 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXnvF2jsro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28514649)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5495017)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Widespread%20mitotic%20bookmarking%20by%20histone%20marks%20and%20transcription%20factors%20in%20pluripotent%20stem%20cells&journal=Cell%20Rep.&volume=19&pages=1283-1293&publication_year=2017&author=Liu%2CY) 
    
10.  Bomber, M. L. et al. Human SMARCA5 is continuously required to maintain nucleosome spacing. _Mol. Cell_ **83**, 507–522.e6 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXnvVamtg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36630954)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9974918)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20SMARCA5%20is%20continuously%20required%20to%20maintain%20nucleosome%20spacing&journal=Mol.%20Cell&volume=83&pages=507-522.e6&publication_year=2023&author=Bomber%2CML) 
    
11.  Schick, S. et al. Acute BAF perturbation causes immediate changes in chromatin accessibility. _Nat. Genet._ **53**, 269–278 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsFGqtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33558760)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7614082)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Acute%20BAF%20perturbation%20causes%20immediate%20changes%20in%20chromatin%20accessibility&journal=Nat.%20Genet.&volume=53&pages=269-278&publication_year=2021&author=Schick%2CS) 
    
12.  Iurlaro, M. et al. Mammalian SWI/SNF continuously restores local accessibility to chromatin. _Nat. Genet._ **53**, 279–287 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsFGqtLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33558757)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mammalian%20SWI%2FSNF%20continuously%20restores%20local%20accessibility%20to%20chromatin&journal=Nat.%20Genet.&volume=53&pages=279-287&publication_year=2021&author=Iurlaro%2CM) 
    
13.  Brahma, S. & Henikoff, S. The BAF chromatin remodeler synergizes with RNA polymerase II and transcription factors to evict nucleosomes. _Nat. Genet._ **56**, 100–111 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFagsrvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38049663)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20BAF%20chromatin%20remodeler%20synergizes%20with%20RNA%20polymerase%20II%20and%20transcription%20factors%20to%20evict%20nucleosomes&journal=Nat.%20Genet.&volume=56&pages=100-111&publication_year=2024&author=Brahma%2CS&author=Henikoff%2CS) 
    
14.  Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. _Nat. Struct. Mol. Biol._ **28**, 501–511 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSqtbfP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34117481)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8504423)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=mSWI%2FSNF%20promotes%20Polycomb%20repression%20both%20directly%20and%20through%20genome-wide%20redistribution&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=28&pages=501-511&publication_year=2021&author=Weber%2CCM) 
    
15.  Ren, G. et al. Acute depletion of BRG1 reveals its primary function as an activator of transcription. _Nat. Commun._ **15**, 4561 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1CktbrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38811575)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11137027)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Acute%20depletion%20of%20BRG1%20reveals%20its%20primary%20function%20as%20an%20activator%20of%20transcription&journal=Nat.%20Commun.&volume=15&publication_year=2024&author=Ren%2CG) 
    
16.  Martin, B. J. E. et al. Global identification of SWI/SNF targets reveals compensation by EP400. _Cell_ **186**, 5290–5307.e26 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitlSnsLzK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37922899)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11307202)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20identification%20of%20SWI%2FSNF%20targets%20reveals%20compensation%20by%20EP400&journal=Cell&volume=186&pages=5290-5307.e26&publication_year=2023&author=Martin%2CBJE) 
    
17.  Blümli, S. et al. Acute depletion of the ARID1A subunit of SWI/SNF complexes reveals distinct pathways for activation and repression of transcription. _Cell Rep._ **37**, 109943 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34731603)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8578704)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Acute%20depletion%20of%20the%20ARID1A%20subunit%20of%20SWI%2FSNF%20complexes%20reveals%20distinct%20pathways%20for%20activation%20and%20repression%20of%20transcription&journal=Cell%20Rep.&volume=37&publication_year=2021&author=Bl%C3%BCmli%2CS) 
    
18.  Mashtalir, N. et al. Modular organization and assembly of SWI/SNF family chromatin remodeling complexes. _Cell_ **175**, 1272–1288.e20 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFKjtbnL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30343899)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6791824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Modular%20organization%20and%20assembly%20of%20SWI%2FSNF%20family%20chromatin%20remodeling%20complexes&journal=Cell&volume=175&pages=1272-1288.e20&publication_year=2018&author=Mashtalir%2CN) 
    
19.  Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. _Nature_ **581**, 434–443 (2020).
    
20.  Gourisankar, S., Krokhotin, A., Wenderski, W. & Crabtree, G. R. Context-specific functions of chromatin remodellers in development and disease. _Nat. Rev. Genet_. **25**, 340–361 (2023).
    
21.  Kadoch, C. et al. Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy. _Nat. Genet._ **45**, 592–601 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXntVart7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23644491)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667980)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteomic%20and%20bioinformatic%20analysis%20of%20mammalian%20SWI%2FSNF%20complexes%20identifies%20extensive%20roles%20in%20human%20malignancy&journal=Nat.%20Genet.&volume=45&pages=592-601&publication_year=2013&author=Kadoch%2CC) 
    
22.  Mittal, P. & Roberts, C. W. M. The SWI/SNF complex in cancer — biology, biomarkers and therapy. _Nat. Rev. Clin. Oncol._ **17**, 435–448 (2020).
    
23.  Reddy, D., Bhattacharya, S. & Workman, J. L. (mis)-Targeting of SWI/SNF complex(es) in cancer. _Cancer Metastasis Rev._ **42**, 455–470 (2023).
    
24.  Wang, X. et al. SMARCB1-mediated SWI/SNF complex function is essential for enhancer regulation. _Nat. Genet_. **49**, 289–295 (2016).
    
25.  Nakayama, R. T. et al. SMARCB1 is required for widespread BAF complex–mediated activation of enhancers and bivalent promoters. _Nat. Genet_. **49**, 1613–1623 (2017).
    
26.  Valencia, A. M. et al. Landscape of mSWI/SNF chromatin remodeling complex perturbations in neurodevelopmental disorders. _Nat. Genet._ **55**, 1400 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFOiur7O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37500730)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10412456)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Landscape%20of%20mSWI%2FSNF%20chromatin%20remodeling%20complex%20perturbations%20in%20neurodevelopmental%20disorders&journal=Nat.%20Genet.&volume=55&publication_year=2023&author=Valencia%2CAM) 
    
27.  Bögershausen, N. & Wollnik, B. Mutational landscapes and phenotypic spectrum of SWI/SNF-related intellectual disability disorders. _Front Mol. Neurosci._ **11**, 400386 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20landscapes%20and%20phenotypic%20spectrum%20of%20SWI%2FSNF-related%20intellectual%20disability%20disorders&journal=Front%20Mol.%20Neurosci.&volume=11&publication_year=2018&author=B%C3%B6gershausen%2CN&author=Wollnik%2CB) 
    
28.  Tsurusaki, Y. et al. Mutations affecting components of the SWI/SNF complex cause Coffin-Siris syndrome. _Nat. Genet_. **44**, 376–378 (2012).
    
29.  Kadoch, C. & Crabtree, G. R. Mammalian SWI/SNF chromatin remodeling complexes and cancer: mechanistic insights gained from human genomics. _Sci. Adv._ **1**, e1500447 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26601204)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4640607)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mammalian%20SWI%2FSNF%20chromatin%20remodeling%20complexes%20and%20cancer%3A%20mechanistic%20insights%20gained%20from%20human%20genomics&journal=Sci.%20Adv.&volume=1&publication_year=2015&author=Kadoch%2CC&author=Crabtree%2CGR) 
    
30.  Chang, C. Y. et al. Increased ACTL6A occupancy within mSWI/SNF chromatin remodelers drives human squamous cell carcinoma. _Mol. Cell_ **81**, 4964–4978.e8 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlams7fM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34687603)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8761479)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Increased%20ACTL6A%20occupancy%20within%20mSWI%2FSNF%20chromatin%20remodelers%20drives%20human%20squamous%20cell%20carcinoma&journal=Mol.%20Cell&volume=81&pages=4964-4978.e8&publication_year=2021&author=Chang%2CCY) 
    
31.  Wenderski, W. et al. Loss of the neural-specific BAF subunit ACTL6B relieves repression of early response genes and causes recessive autism. _Proc. Natl Acad. Sci. USA_ **117**, 10055–10066 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtV2ltbrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32312822)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7211998)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20the%20neural-specific%20BAF%20subunit%20ACTL6B%20relieves%20repression%20of%20early%20response%20genes%20and%20causes%20recessive%20autism&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=10055-10066&publication_year=2020&author=Wenderski%2CW) 
    
32.  Ronan, J. L., Wu, W. & Crabtree, G. R. From neural development to cognition: unexpected roles for chromatin. _Nat. Rev. Genet._ **14**, 347–359 (2013).
    
33.  Bultman, S. et al. A _Brg1_ null mutation in the mouse reveals functional differences among mammalian SWI/SNF complexes. _Mol. Cell_ **6**, 1287–1295 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXlvFahsw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11163203)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20Brg1%20null%20mutation%20in%20the%20mouse%20reveals%20functional%20differences%20among%20mammalian%20SWI%2FSNF%20complexes&journal=Mol.%20Cell&volume=6&pages=1287-1295&publication_year=2000&author=Bultman%2CS) 
    
34.  Glaros, S., Cirrincione, G. M., Palanca, A., Metzger, D. & Reisman, D. Targeted knockout of BRG1 potentiates lung cancer development. _Cancer Res._ **68**, 3689–3696 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlvVOitLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18483251)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeted%20knockout%20of%20BRG1%20potentiates%20lung%20cancer%20development&journal=Cancer%20Res.&volume=68&pages=3689-3696&publication_year=2008&author=Glaros%2CS&author=Cirrincione%2CGM&author=Palanca%2CA&author=Metzger%2CD&author=Reisman%2CD) 
    
35.  Takeuchi, J. K. et al. Chromatin remodelling complex dosage modulates transcription factor function in heart development. _Nat. Commun._ **2**, 187 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21304516)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20remodelling%20complex%20dosage%20modulates%20transcription%20factor%20function%20in%20heart%20development&journal=Nat.%20Commun.&volume=2&publication_year=2011&author=Takeuchi%2CJK) 
    
36.  Bultman, S. J. et al. Characterization of mammary tumors from _Brg1_ heterozygous mice. _Oncogene_ **27**, 460–468 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXnt1equg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17637742)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterization%20of%20mammary%20tumors%20from%20Brg1%20heterozygous%20mice&journal=Oncogene&volume=27&pages=460-468&publication_year=2008&author=Bultman%2CSJ) 
    
37.  van der Vaart, A., Godfrey, M. & Portegijs, V. Dose-dependent functions of SWI/SNF BAF in permitting and inhibiting cell proliferation in vivo. _Sci. Adv._ **6**, eaay3823 (2020).
    
38.  Smith, J. J. et al. The SWI/SNF chromatin remodeling assemblies BAF and PBAF differentially regulate cell cycle exit and cellular invasion in vivo. _PLoS Genet._ **18**, e1009981 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20SWI%2FSNF%20chromatin%20remodeling%20assemblies%20BAF%20and%20PBAF%20differentially%20regulate%20cell%20cycle%20exit%20and%20cellular%20invasion%20in%20vivo&journal=PLoS%20Genet.&volume=18&publication_year=2021&author=Smith%2CJJ) 
    
39.  Nabet, B. et al. The dTAG system for immediate and target-specific protein degradation. _Nat. Chem. Biol_. **14**, 431–441 (2018).
    
40.  Naqvi, S. et al. Precise modulation of transcription factor levels identifies features underlying dosage sensitivity. _Nat. Genet_. **55**, 841–851 (2023).
    
41.  Ho, L. et al. An embryonic stem cell chromatin remodeling complex, esBAF, is essential for embryonic stem cell self-renewal and pluripotency. _Proc. Natl Acad. Sci. USA_ **106**, 5181–5186 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXksVert70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19279220)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2654396)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20embryonic%20stem%20cell%20chromatin%20remodeling%20complex%2C%20esBAF%2C%20is%20essential%20for%20embryonic%20stem%20cell%20self-renewal%20and%20pluripotency&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=106&pages=5181-5186&publication_year=2009&author=Ho%2CL) 
    
42.  Zhou, C., Wang, M., Zhang, C. & Zhang, Y. The transcription factor GABPA is a master regulator of naive pluripotency. _Nat. Cell Biol._ **27**, 48–58 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXksVKgsg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39747581)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11735382)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20transcription%20factor%20GABPA%20is%20a%20master%20regulator%20of%20naive%20pluripotency&journal=Nat.%20Cell%20Biol.&volume=27&pages=48-58&publication_year=2025&author=Zhou%2CC&author=Wang%2CM&author=Zhang%2CC&author=Zhang%2CY) 
    
43.  Gatchalian, J. et al. A non-canonical BRD9-containing BAF chromatin remodeling complex regulates naive pluripotency in mouse embryonic stem cells. _Nat. Commun_. **9**, 5139 (2018).
    
44.  Marino, M. M. et al. Interactome mapping defines BRG1, a component of the SWI/SNF chromatin remodeling complex, as a new partner of the transcriptional regulator CTCF. _J. Biol. Chem._ **294**, 861–873 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVymsL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30459231)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactome%20mapping%20defines%20BRG1%2C%20a%20component%20of%20the%20SWI%2FSNF%20chromatin%20remodeling%20complex%2C%20as%20a%20new%20partner%20of%20the%20transcriptional%20regulator%20CTCF&journal=J.%20Biol.%20Chem.&volume=294&pages=861-873&publication_year=2019&author=Marino%2CMM) 
    
45.  Michel, B. C. et al. A non-canonical SWI/SNF complex is a synthetic lethal target in cancers driven by BAF complex perturbation. _Nat. Cell Biol_. **20**, 1410–1420 (2018).
    
46.  Minderjahn, J. et al. Mechanisms governing the pioneering and redistribution capabilities of the non-classical pioneer PU.1. _Nat. Commun_. **11**, 402 (2020).
    
47.  Krishnakumar, R. et al. FOXD3 regulates pluripotent stem cell potential by simultaneously initiating and repressing enhancer activity. _Cell Stem Cell_ **18**, 104–117 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVehsb%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26748757)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4775235)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXD3%20regulates%20pluripotent%20stem%20cell%20potential%20by%20simultaneously%20initiating%20and%20repressing%20enhancer%20activity&journal=Cell%20Stem%20Cell&volume=18&pages=104-117&publication_year=2016&author=Krishnakumar%2CR) 
    
48.  King, H. W. & Klose, R. J. The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells. _eLife_ **6**, e22631 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28287392)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5400504)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20pioneer%20factor%20OCT4%20requires%20the%20chromatin%20remodeller%20BRG1%20to%20support%20gene%20regulatory%20element%20function%20in%20mouse%20embryonic%20stem%20cells&journal=eLife&volume=6&publication_year=2017&author=King%2CHW&author=Klose%2CRJ) 
    
49.  Wanior, M., Krämer, A., Knapp, S. & Joerger, A. C. Exploiting vulnerabilities of SWI/SNF chromatin remodelling complexes for cancer therapy. _Oncogene_ **40**, 3637–3654 (2021).
    
50.  Kim, B. W. et al. Neuronal activity-induced BRG1 phosphorylation regulates enhancer activation. _Cell Rep._ **36**, 109357 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFGgurnP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34260936)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8315893)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neuronal%20activity-induced%20BRG1%20phosphorylation%20regulates%20enhancer%20activation&journal=Cell%20Rep.&volume=36&publication_year=2021&author=Kim%2CBW) 
    
51.  Giorgetti, L. et al. Noncooperative interactions between transcription factors and clustered DNA binding sites enable graded transcriptional responses to environmental inputs. _Mol. Cell_ **37**, 418–428 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlt1Kkur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20159560)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Noncooperative%20interactions%20between%20transcription%20factors%20and%20clustered%20DNA%20binding%20sites%20enable%20graded%20transcriptional%20responses%20to%20environmental%20inputs&journal=Mol.%20Cell&volume=37&pages=418-428&publication_year=2010&author=Giorgetti%2CL) 
    
52.  Neikes, H. K. et al. Quantification of absolute transcription factor binding affinities in the native chromatin context using BANC-seq. _Nat. Biotechnol_. **41**, 1801–1809 (2023).
    
53.  Belikov, S., Berg, O. G. & Wrange, Ö. Quantification of transcription factor-DNA binding affinity in a living cell. _Nucleic Acids Res._ **44**, 3045 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlOjtbvK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26657626)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantification%20of%20transcription%20factor-DNA%20binding%20affinity%20in%20a%20living%20cell&journal=Nucleic%20Acids%20Res.&volume=44&publication_year=2016&author=Belikov%2CS&author=Berg%2COG&author=Wrange%2C%C3%96) 
    
54.  King, H. W., Fursova, N. A., Blackledge, N. P. & Klose, R. J. Polycomb repressive complex 1 shapes the nucleosome landscape but not accessibility at target genes. _Genome Res._ **28**, 1494–1507 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlWlu73K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30154222)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6169895)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polycomb%20repressive%20complex%201%20shapes%20the%20nucleosome%20landscape%20but%20not%20accessibility%20at%20target%20genes&journal=Genome%20Res.&volume=28&pages=1494-1507&publication_year=2018&author=King%2CHW&author=Fursova%2CNA&author=Blackledge%2CNP&author=Klose%2CRJ) 
    
55.  Hodges, H. C. et al. Dominant-negative SMARCA4 mutants alter the accessibility landscape of tissue-unrestricted enhancers. _Nat. Struct. Mol. Biol._ **25**, 61–72 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlCqu7vO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29323272)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dominant-negative%20SMARCA4%20mutants%20alter%20the%20accessibility%20landscape%20of%20tissue-unrestricted%20enhancers&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=25&pages=61-72&publication_year=2018&author=Hodges%2CHC) 
    
56.  Oldfield, A. J. et al. NF-Y controls fidelity of transcription initiation at gene promoters through maintenance of the nucleosome-depleted region. _Nat. Commun._ **10**, 3072 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31296853)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6624317)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NF-Y%20controls%20fidelity%20of%20transcription%20initiation%20at%20gene%20promoters%20through%20maintenance%20of%20the%20nucleosome-depleted%20region&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Oldfield%2CAJ) 
    
57.  Dluhosova, M. et al. Epigenetic control of SPI1 gene by CTCF and ISWI ATPase SMARCA5. _PLoS ONE_ **9**, e87448 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24498324)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911986)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenetic%20control%20of%20SPI1%20gene%20by%20CTCF%20and%20ISWI%20ATPase%20SMARCA5&journal=PLoS%20ONE&volume=9&publication_year=2014&author=Dluhosova%2CM) 
    
58.  Wiechens, N. et al. The chromatin remodelling enzymes SNF2H and SNF2L position nucleosomes adjacent to CTCF and other transcription factors. _PLoS Genet._ **12**, e1005940 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27019336)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4809547)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20chromatin%20remodelling%20enzymes%20SNF2H%20and%20SNF2L%20position%20nucleosomes%20adjacent%20to%20CTCF%20and%20other%20transcription%20factors&journal=PLoS%20Genet.&volume=12&publication_year=2016&author=Wiechens%2CN) 
    
59.  Iurlaro, M. et al. Systematic assessment of ISWI subunits shows that NURF creates local accessibility for CTCF. _Nat. Genet._ **56**, 1203–1212 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1yhs7%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38816647)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11176080)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20assessment%20of%20ISWI%20subunits%20shows%20that%20NURF%20creates%20local%20accessibility%20for%20CTCF&journal=Nat.%20Genet.&volume=56&pages=1203-1212&publication_year=2024&author=Iurlaro%2CM) 
    
60.  Xiao, L. et al. Targeting SWI/SNF ATPases in enhancer-addicted prostate cancer. _Nature_ **601**, 434–439 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXivValtr7L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34937944)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20SWI%2FSNF%20ATPases%20in%20enhancer-addicted%20prostate%20cancer&journal=Nature&volume=601&pages=434-439&publication_year=2022&author=Xiao%2CL) 
    
61.  Creyghton, M. P. et al. Histone H3K27ac separates active from poised enhancers and predicts developmental state. _Proc. Natl Acad. Sci. USA_ **107**, 21931–21936 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhs1Wjs7bP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21106759)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003124)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Histone%20H3K27ac%20separates%20active%20from%20poised%20enhancers%20and%20predicts%20developmental%20state&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=107&pages=21931-21936&publication_year=2010&author=Creyghton%2CMP) 
    
62.  Whyte, W. A. et al. Master transcription factors and Mediator establish super-enhancers at key cell identity genes. _Cell_ **153**, 307 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXlvVCrtbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23582322)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3653129)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Master%20transcription%20factors%20and%20Mediator%20establish%20super-enhancers%20at%20key%20cell%20identity%20genes&journal=Cell&volume=153&publication_year=2013&author=Whyte%2CWA) 
    
63.  Hnisz, D. et al. Transcriptional super-enhancers connected to cell identity and disease. _Cell_ **155**, 934 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1SrtLbL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24119843)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptional%20super-enhancers%20connected%20to%20cell%20identity%20and%20disease&journal=Cell&volume=155&publication_year=2013&author=Hnisz%2CD) 
    
64.  Pampari, A. et al. ChromBPNet: bias factorized, base-resolution deep learning models of chromatin accessibility reveal cis-regulatory sequence syntax, transcription factor footprints and regulatory variants. Preprint at _bioRviv_ [https://doi.org/10.1101/2024.12.25.630221](https://doi.org/10.1101/2024.12.25.630221) (2025).
    
65.  Avsec, Ž. et al. Effective gene expression prediction from sequence by integrating long-range interactions. _Nat. Methods_ **18**, 1196–1203 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFOltLbP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34608324)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8490152)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Effective%20gene%20expression%20prediction%20from%20sequence%20by%20integrating%20long-range%20interactions&journal=Nat.%20Methods&volume=18&pages=1196-1203&publication_year=2021&author=Avsec%2C%C5%BD) 
    
66.  Pratt, H. E. et al. Using a comprehensive atlas and predictive models to reveal the complexity and evolution of brain-active regulatory elements. _Sci. Adv._ **10**, eadj4452 (2024).
    
67.  Zhang, W. et al. The BAF and PRC2 complex subunits Dpf2 and Eed antagonistically converge on Tbx3 to control ESC differentiation. _Cell Stem Cell_ **24**, 138–152.e8 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXktFKgsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30609396)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6486830)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20BAF%20and%20PRC2%20complex%20subunits%20Dpf2%20and%20Eed%20antagonistically%20converge%20on%20Tbx3%20to%20control%20ESC%20differentiation&journal=Cell%20Stem%20Cell&volume=24&pages=138-152.e8&publication_year=2019&author=Zhang%2CW) 
    
68.  Ivanova, N. et al. Dissecting self-renewal in stem cells with RNA interference. _Nature_ **442**, 533–538 (2006).
    
69.  Ho, L. et al. esBAF facilitates pluripotency by conditioning the genome for LIF/STAT3 signalling and by regulating polycomb function. _Nat. Cell Biol_. **13**, 903–913 (2011).
    
70.  Romero, O. A. et al. The tumour suppressor and chromatin-remodelling factor BRG1 antagonizes Myc activity and promotes cell differentiation in human cancer. _EMBO Mol. Med._ **4**, 603–616 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XpsFKls70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22407764)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3407948)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20tumour%20suppressor%20and%20chromatin-remodelling%20factor%20BRG1%20antagonizes%20Myc%20activity%20and%20promotes%20cell%20differentiation%20in%20human%20cancer&journal=EMBO%20Mol.%20Med.&volume=4&pages=603-616&publication_year=2012&author=Romero%2COA) 
    
71.  Zhong, M. et al. Malignant transformation of human bronchial epithelial cells induced by arsenic through STAT3/miR-301a/SMAD4 loop. _Sci. Rep._ **8**, 13291 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30185897)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6125593)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Malignant%20transformation%20of%20human%20bronchial%20epithelial%20cells%20induced%20by%20arsenic%20through%20STAT3%2FmiR-301a%2FSMAD4%20loop&journal=Sci.%20Rep.&volume=8&publication_year=2018&author=Zhong%2CM) 
    
72.  Park, Yhee, Kim, D., Dai, J. & Zhang, Z. Human bronchial epithelial BEAS-2B cells, an appropriate in vitro model to study heavy metals induced carcinogenesis. _Toxicol. Appl. Pharmacol._ **287**, 240–245 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtV2itL%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26091798)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4549192)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20bronchial%20epithelial%20BEAS-2B%20cells%2C%20an%20appropriate%20in%20vitro%20model%20to%20study%20heavy%20metals%20induced%20carcinogenesis&journal=Toxicol.%20Appl.%20Pharmacol.&volume=287&pages=240-245&publication_year=2015&author=Park%2CYhee&author=Kim%2CD&author=Dai%2CJ&author=Zhang%2CZ) 
    
73.  Liu, Y. et al. Exosomes of A549 cells induced migration, invasion, and EMT of BEAS-2B cells related to let-7c-5p and miR-181b-5p. _Front. Endocrinol._ **13**, 926769 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Exosomes%20of%20A549%20cells%20induced%20migration%2C%20invasion%2C%20and%20EMT%20of%20BEAS-2B%20cells%20related%20to%20let-7c-5p%20and%20miR-181b-5p&journal=Front.%20Endocrinol.&volume=13&publication_year=2022&author=Liu%2CY) 
    
74.  Peinado, P. et al. Comprehensive analysis of SWI/SNF inactivation in lung adenocarcinoma cell models. _Cancers_ **12**, 3712 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnvFygu70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33321963)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7763689)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20analysis%20of%20SWI%2FSNF%20inactivation%20in%20lung%20adenocarcinoma%20cell%20models&journal=Cancers&volume=12&publication_year=2020&author=Peinado%2CP) 
    
75.  Marquez-Vilendrer, S. B., Rai, S. K., Gramling, S. J., Lu, L. & Reisman, D. N. Loss of the SWI/SNF ATPase subunits BRM and BRG1 drives lung cancer development. _Oncoscience_ **3**, 322–336 (2016).
    
76.  Raab, J. R., Runge, J. S., Spear, C. C. & Magnuson, T. Co-regulation of transcription by BRG1 and BRM, two mutually exclusive SWI/SNF ATPase subunits. _Epigenetics Chromatin_ **10**, 62 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29273066)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5740901)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Co-regulation%20of%20transcription%20by%20BRG1%20and%20BRM%2C%20two%20mutually%20exclusive%20SWI%2FSNF%20ATPase%20subunits&journal=Epigenetics%20Chromatin&volume=10&publication_year=2017&author=Raab%2CJR&author=Runge%2CJS&author=Spear%2CCC&author=Magnuson%2CT) 
    
77.  Xu, Y. Z., Thuraisingam, T., Marino, R. & Radzioch, D. Recruitment of SWI/SNF complex is required for transcriptional activation of the _SLC11A1_ gene during macrophage differentiation of HL-60 cells. _J. Biol. Chem._ **286**, 12839–12849 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXksVKms7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21300803)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3075631)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Recruitment%20of%20SWI%2FSNF%20complex%20is%20required%20for%20transcriptional%20activation%20of%20the%20SLC11A1%20gene%20during%20macrophage%20differentiation%20of%20HL-60%20cells&journal=J.%20Biol.%20Chem.&volume=286&pages=12839-12849&publication_year=2011&author=Xu%2CYZ&author=Thuraisingam%2CT&author=Marino%2CR&author=Radzioch%2CD) 
    
78.  Orlando, K. A. et al. Re-expression of SMARCA4/BRG1 in small cell carcinoma of ovary, hypercalcemic type (SCCOHT) promotes an epithelial-like gene signature through an AP-1-dependent mechanism. _eLife_ **9**, 1–57 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Re-expression%20of%20SMARCA4%2FBRG1%20in%20small%20cell%20carcinoma%20of%20ovary%2C%20hypercalcemic%20type%20%28SCCOHT%29%20promotes%20an%20epithelial-like%20gene%20signature%20through%20an%20AP-1-dependent%20mechanism&journal=eLife&volume=9&pages=1-57&publication_year=2020&author=Orlando%2CKA) 
    
79.  Wallbillich, N. J. & Lu, H. Role of c-Myc in lung cancer: progress, challenges, and prospects. _Chin. Med. J. Pulm. Crit. Care Med._ **1**, 129–138 (2023).
    
80.  Navickas, S. M., Giles, K. A., Brettingham-Moore, K. H. & Taberlay, P. C. The role of chromatin remodeler SMARCA4/BRG1 in brain cancers: a potential therapeutic target. _Oncogene_ **42**, 2363–2373 (2023).
    
81.  Nagae, F., Brandani, G. B., Takada, S. & Terakawa, T. The lane-switch mechanism for nucleosome repositioning by DNA translocase. _Nucleic Acids Res._ **49**, 9066–9076 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVCqtrfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34365508)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8450081)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20lane-switch%20mechanism%20for%20nucleosome%20repositioning%20by%20DNA%20translocase&journal=Nucleic%20Acids%20Res.&volume=49&pages=9066-9076&publication_year=2021&author=Nagae%2CF&author=Brandani%2CGB&author=Takada%2CS&author=Terakawa%2CT) 
    
82.  Basurto-Cayuela, L. et al. SWI/SNF-dependent genes are defined by their chromatin landscape. _Cell Rep._ **43**, 113855 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXkvVSru7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38427563)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SWI%2FSNF-dependent%20genes%20are%20defined%20by%20their%20chromatin%20landscape&journal=Cell%20Rep.&volume=43&publication_year=2024&author=Basurto-Cayuela%2CL) 
    
83.  Moe, K. C. et al. The SWI/SNF ATPase BRG1 facilitates multiple pro-tumorigenic gene expression programs in _SMARCB1_\-deficient cancer cells. _Oncogenesis_ **11**, 30 (2022).
    
84.  Hohmann, A. F. et al. Sensitivity and engineered resistance of myeloid leukemia cells to BRD9 inhibition. _Nat. Chem. Biol._ **12**, 672–679 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKntLfN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27376689)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990482)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sensitivity%20and%20engineered%20resistance%20of%20myeloid%20leukemia%20cells%20to%20BRD9%20inhibition&journal=Nat.%20Chem.%20Biol.&volume=12&pages=672-679&publication_year=2016&author=Hohmann%2CAF) 
    
85.  Bossen, C. et al. The chromatin remodeler Brg1 activates enhancer repertoires to establish B cell identity and modulate cell growth. _Nat. Immunol._ **16**, 775–784 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFSktrbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25985234)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4474778)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20chromatin%20remodeler%20Brg1%20activates%20enhancer%20repertoires%20to%20establish%20B%20cell%20identity%20and%20modulate%20cell%20growth&journal=Nat.%20Immunol.&volume=16&pages=775-784&publication_year=2015&author=Bossen%2CC) 
    
86.  Shi, J. et al. Role of SWI/SNF in acute leukemia maintenance and enhancer-mediated Myc regulation. _Genes Dev._ **27**, 2648–2662 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXltFCgsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24285714)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3877755)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20SWI%2FSNF%20in%20acute%20leukemia%20maintenance%20and%20enhancer-mediated%20Myc%20regulation&journal=Genes%20Dev.&volume=27&pages=2648-2662&publication_year=2013&author=Shi%2CJ) 
    
87.  Orvis, T. et al. BRG1/SMARCA4 inactivation promotes non-small cell lung cancer aggressiveness by altering chromatin organization. _Cancer Res._ **74**, 6486–6498 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFWksrfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25115300)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4233181)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BRG1%2FSMARCA4%20inactivation%20promotes%20non-small%20cell%20lung%20cancer%20aggressiveness%20by%20altering%20chromatin%20organization&journal=Cancer%20Res.&volume=74&pages=6486-6498&publication_year=2014&author=Orvis%2CT) 
    
88.  Willis, M. S. et al. BRG1 and BRM function antagonistically with c-MYC in adult cardiomyocytes to regulate conduction and contractility. _J. Mol. Cell. Cardiol._ **105**, 99–109 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXksFSiu7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28232072)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5415084)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BRG1%20and%20BRM%20function%20antagonistically%20with%20c-MYC%20in%20adult%20cardiomyocytes%20to%20regulate%20conduction%20and%20contractility&journal=J.%20Mol.%20Cell.%20Cardiol.&volume=105&pages=99-109&publication_year=2017&author=Willis%2CMS) 
    
89.  Jose, C. C. et al. Epigenetic dysregulation by nickel through repressive chromatin domain disruption. _Proc. Natl Acad. Sci. USA_ **111**, 14631–14636 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFyhtbbK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25246589)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4210008)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenetic%20dysregulation%20by%20nickel%20through%20repressive%20chromatin%20domain%20disruption&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=111&pages=14631-14636&publication_year=2014&author=Jose%2CCC) 
    
90.  Chen, Z., Djekidel, M. N. & Zhang, Y. Distinct dynamics and functions of H2AK119ub1 and H3K27me3 in mouse preimplantation embryos. _Nat. Genet._ **53**, 551–563 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXotVamtr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33821005)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8092361)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20dynamics%20and%20functions%20of%20H2AK119ub1%20and%20H3K27me3%20in%20mouse%20preimplantation%20embryos&journal=Nat.%20Genet.&volume=53&pages=551-563&publication_year=2021&author=Chen%2CZ&author=Djekidel%2CMN&author=Zhang%2CY) 
    
91.  Wang, M., Chen, Z. & Zhang, Y. CBP/p300 and HDAC activities regulate H3K27 acetylation dynamics and zygotic genome activation in mouse preimplantation embryos. _EMBO J._ **41**, e112012 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1ahsrzF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36215692)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9670200)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CBP%2Fp300%20and%20HDAC%20activities%20regulate%20H3K27%20acetylation%20dynamics%20and%20zygotic%20genome%20activation%20in%20mouse%20preimplantation%20embryos&journal=EMBO%20J.&volume=41&publication_year=2022&author=Wang%2CM&author=Chen%2CZ&author=Zhang%2CY) 
    
92.  Zhang, C., Wang, M., Li, Y. & Zhang, Y. Profiling and functional characterization of maternal mRNA translation during mouse maternal-to-zygotic transition. _Sci. Adv_. **8**, eabj3967 (2022).
    
93.  Buecker, C. et al. Reorganization of enhancer patterns in transition from naive to primed pluripotency. _Cell Stem Cell_ **14**, 838–853 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXpvFSktL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24905168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4491504)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reorganization%20of%20enhancer%20patterns%20in%20transition%20from%20naive%20to%20primed%20pluripotency&journal=Cell%20Stem%20Cell&volume=14&pages=838-853&publication_year=2014&author=Buecker%2CC) 
    
94.  Bleckwehl, T. et al. Enhancer-associated H3K4 methylation safeguards in vitro germline competence. _Nat. Commun_. **12**, 5771 (2021).
    
95.  Stadler, M. B. et al. DNA-binding factors shape the mouse methylome at distal regulatory regions. _Nature_ **480**, 490–495 (2011).
    
96.  Oldfield, A. J. et al. Histone-fold domain protein NF-Y promotes chromatin accessibility for cell type-specific master transcription factors. _Mol. Cell._ **55**, 708–722 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtlKmtbzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25132174)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4157648)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Histone-fold%20domain%20protein%20NF-Y%20promotes%20chromatin%20accessibility%20for%20cell%20type-specific%20master%20transcription%20factors&journal=Mol.%20Cell.&volume=55&pages=708-722&publication_year=2014&author=Oldfield%2CAJ) 
    
97.  Chen, X. et al. Integration of external signaling pathways with the core transcriptional network in embryonic stem cells. _Cell_ **133**, 1106–1117 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXnsF2gurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18555785)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integration%20of%20external%20signaling%20pathways%20with%20the%20core%20transcriptional%20network%20in%20embryonic%20stem%20cells&journal=Cell&volume=133&pages=1106-1117&publication_year=2008&author=Chen%2CX) 
    
98.  Zhang, C. ChromBPNet models. _Figshare_ [https://doi.org/10.6084/m9.figshare.28705430](https://doi.org/10.6084/m9.figshare.28705430) (2025).
    
99.  Zhang, C. Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. _Zenodo_ [https://doi.org/10.5281/zenodo.15951452](https://doi.org/10.5281/zenodo.15951452) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02305-z?format=refman&flavour=references)

Acknowledgements
----------------

We thank J. Yao for help with cell sorting and C. Zhou, Q. Yang and J. Ren for helpful discussions and thoughtful comments on the manuscript. This project was supported by the NIH (R01HD092465) and the Howard Hughes Medical Institute (HHMI). Y.H. is supported by a JSPS Overseas Research Fellowship from the Japan Society for the Promotion of Science (JSPS). Y.Z. is an investigator of the HHMI. This article is subject to HHMI’s Open Access to Publications policy. HHMI laboratory heads have previously granted a nonexclusive CC BY 4.0 license to the public and a sublicensable license to HHMI in their research articles. Pursuant to those licenses, the author-accepted manuscript of this article can be made freely available under a CC BY 4.0 license immediately upon publication.

Author information
------------------

Author notes

1.  Chao Zhang
    
    Present address: State Key Laboratory of Genetic Evolution & Animal Models, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, China
    
2.  These authors contributed equally: Yota Hagihara, Chao Zhang.
    

### Authors and Affiliations

1.  Howard Hughes Medical Institute, Boston Children’s Hospital, Boston, MA, USA
    
    Yota Hagihara, Chao Zhang & Yi Zhang
    
2.  Program in Cellular and Molecular Medicine, Boston Children’s Hospital, Boston, MA, USA
    
    Yota Hagihara, Chao Zhang & Yi Zhang
    
3.  Division of Hematology/Oncology, Department of Pediatrics, Boston Children’s Hospital, Boston, MA, USA
    
    Yota Hagihara, Chao Zhang & Yi Zhang
    
4.  Department of Genetics, Harvard Medical School, Boston, MA, USA
    
    Yi Zhang
    
5.  Harvard Stem Cell Institute, Boston, MA, USA
    
    Yi Zhang
    

Authors

1.  Yota Hagihara
    
    [View author publications](https://www.nature.com/search?author=Yota%20Hagihara)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yota%20Hagihara) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yota%20Hagihara%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Chao Zhang
    
    [View author publications](https://www.nature.com/search?author=Chao%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chao%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chao%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Yi Zhang
    
    [View author publications](https://www.nature.com/search?author=Yi%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yi%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yi%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Y.Z. supervised the project. Y.H. designed and performed experiments, and C.Z. performed bioinformatic analysis. Y.H., C.Z. and Y.Z. wrote the manuscript.

### Corresponding author

Correspondence to [Yi Zhang](mailto:yzhang@genetics.med.harvard.edu).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02305-z#MOESM2) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 dTAG knock-in does not perturb BRG1 complex formation.](https://www.nature.com/articles/s41588-025-02305-z/figures/8)

**a**, mRNA levels of _Brg1_ (_Smarca4_) and _Brm_ (_Smarca2_) in mESCs. Data represent two independent experiments. **b**, Schematic of dTAG mediated protein degradation system. FKBP12-F36V fused protein undergoes degradation in the presence of dTAG13. **c**, Schematic illustration of FKBP12-F36V-HA knock-in at the 3’ end of _Brg1_. **d**, Western blot analysis of Brg1-KI mESCs after treatment for different times with 100 nM dTAG13 or DMSO. Data are representative of two independent experiments. **e**, BRG1 immunoprecipitation in WT or Brg1-KI mESCs, followed by western blot analysis. Data are representative of three independent experiments. **f**, Immunostaining analysis of BRG1 in the Brg1-KI mESCs after treatment with the indicated concentrations of dTAG13 for 24 hours. Representative images (left), and relative BRG1 intensity in each cell (right). The GFP intensity of individual cell with 0 nM dTAG13 treatment was set as 1. Data includes two independent replicates. **g**, Correlations of the replicates of CUT&RUN at 0 nM. Rp: Pearson correlation. **h**, Histograms showing delta values of BRG1 binding sites co-occupied with OCT4 (left) or H3K27ac (right). The red line indicates delta = 0.

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM4)

### [Extended Data Fig. 2 The number of DARs increases progressively as BRG1 degradation increases.](https://www.nature.com/articles/s41588-025-02305-z/figures/9)

**a**, Comparison of chromatin accessibility (ATAC-seq) changes at BRG1-targeted regulatory elements following 24 hours of dTAG13 treatment at 0 nM (DMSO, x-axis) and 0.3 to 100 nM (y-axis). **b**, The numbers of differentially accessible regions (DARs) identified at varying dTAG13 concentrations. **c**, Aggregate coverage plot showing mean ATAC-seq peaks of Brg1flox/flox (red) and Brg1-KO (green) at promoters, enhancers and insulators[48](https://www.nature.com/articles/s41588-025-02305-z#ref-CR48 "King, H. W. & Klose, R. J. The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells. eLife 6, e22631 (2017).").

### [Extended Data Fig. 3 Chromatin signature of BRG1-dependent and -independent regulatory elements.](https://www.nature.com/articles/s41588-025-02305-z/figures/10)

**a**, Heatmap showing BRG1-binding (Blue) and ATAC-seq (Red) for BRG1-independent and -dependent regulatory elements. Indep, BRG1-independent regions. Dep, BRG1-dependent regions. **b**, Heatmap showing TFs and histone modifications for BRG1-independent and -dependent regulatory elements. **c**, Aggregate coverage plot showing mean OCT4 peaks at BRG1-dependent and -independent regulatory elements. **d**, Aggregate coverage plot showing mean ATAC-seq signal at OCT4-bound regions. **e**, Aggregate coverage plot showing mean RING1B (top) and SUZ12 (bottom) signals of 0 hour (red) and 8 hours (cyan) at BRG1-dependent and -independent promoters[14](https://www.nature.com/articles/s41588-025-02305-z#ref-CR14 "Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. Nat. Struct. Mol. Biol. 28, 501–511 (2021)."). BRG1 was depleted by auxin-degron system for 8 hours[14](https://www.nature.com/articles/s41588-025-02305-z#ref-CR14 "Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. Nat. Struct. Mol. Biol. 28, 501–511 (2021)."). **f**, Aggregate coverage plot showing mean ATAC-seq signal at NFYA-bound (left) and -unbound (right) regions at BRG1-independent promoters[56](https://www.nature.com/articles/s41588-025-02305-z#ref-CR56 "Oldfield, A. J. et al. NF-Y controls fidelity of transcription initiation at gene promoters through maintenance of the nucleosome-depleted region. Nat. Commun. 10, 3072 (2019)."). **g**, Heatmap showing ATAC-seq signals for BRG1-independent and BRG1-dependent insulators following treatment with 1 μM AU-15330 and/or 100 nM dTAG13 for 24 hours. **h**, Transcription factor motif enrichment at BRG1-independent (Indep) and BRG1-dependent (Dep) ATAC-seq peaks in insulators. The color scale represents −log10(P value), and dot size corresponds to the effect size measured as odds ratio (OR). A two-sided Fisher’s exact test was performed.

### [Extended Data Fig. 4 Characteristic of BRG1 dosage-sensitive enhancers.](https://www.nature.com/articles/s41588-025-02305-z/figures/11)

**a**, Chromatin accessibility changes for each of the five groups of BRG1-dependent enhancers at different BRG1 levels. Line showed mean of each group, shed area indicates SEM **b**, Histogram of BRG1 binding delta value for each group. Red line indicates delta = 0. **c**, Aggregate coverage plot showing mean ATAC-seq signal at each enhancer regions after BRM014, BRG1/BRM ATPase inhibitor, treatment for the indicated time on the right. **d**, Bottom: Heatmap showing TFs and histone modifications of the BRG1-dependent regions in each of the groups. Top: aggregate coverage plots showing mean enrichment of the indicated TFs and histone modifications for each group.

### [Extended Data Fig. 5 Machine learning model predicts features of buffered and linearly sensitive enhancers.](https://www.nature.com/articles/s41588-025-02305-z/figures/12)

**a**, Illustration of ChromBPNet predict chromatin accessibilities at _Psmd7_ gene locus. Observed track: signals detected by ATAC-seq experiments; Predicted track: ChromBPNet predicted signals; Contribution track: the genome sequence scaled according to ChromBPNet contribute scores. **b**, Scatter plot showing the correlation between predicted and observed signals. **c**, Motifs identified by ChromBPNet models in G1 enhancers (left), and G5 enhancers (right). **d**, Marginal footprints of TF motifs of different groups using predicted profiles from mESC ATAC-seq ChromBPNet models. **e**, Cumulative percentage of GC content in different groups. _P_ values were calculated with Wilcoxon’s _t_\-test (two-sided) by comparing each group with G1: _p_ = 2.5 × 10−4 (G2); _p_ = 4.2 × 10−8 (G3), _p_ = 1.3 × 10−10 (G4), _p_ = 1.1 × 10−14 (G5). \*\*\* _p_ < 0.001 **f**, Histograms showing the number of ChromBPNet predicted high score sites from the 0 nM mESCs model or 100 nM dTAG13 mESCs model. **g**, Representative genome browser view of high-score site at buffered or linear enhancers.

### [Extended Data Fig. 6 BRG1 dosage response in transcriptome.](https://www.nature.com/articles/s41588-025-02305-z/figures/13)

**a**, Comparison of transcription (RNA-seq) changes after 24 hours of dTAG13 treatment at concentrations of 0 nM (DMSO, x-axis) and 0.3 to 100 nM (y-axis). The numbers of differentially expressed genes under each condition are shown. **b**, Boxplot showing gene expression associated with BRG1-dependent promoter (1,128 genes) (top) and -independent promoter (6,826 genes) (bottom). Boxes are defined by the 25th to 75th percentiles, the median is shown as a horizontal black line, and whiskers extend up to 1.5 times the interquartile range (IQR) from the quartiles. Dash line indicates the mean signal at 0 nM condition. Data represent two independent experiments. _P_ values were calculated with Wilcoxon’s _t_\-test (two-sided). **c**, Boxplot showing gene expression associated with BRG1-dependent enhancers (top) and -independent enhancers (bottom). Genes are classified by the distance of TSS-enhancer <20 kb (left) and >20 kb (right), each including 4,738 (dep, <20 kb), 7,340 (indep, <20 kb), 3,190 (dep, >20 kb) and 3,740 (indep, >20 kb) genes. Boxes are defined by the 25th to 75th percentiles, the median is shown as a horizontal black line, and whiskers extend up to 1.5 times the IQR from the quartiles. Dashed line indicates the mean signal at 0 nM condition. Data represent two independent experiments. _P_ values were calculated with Wilcoxon’s _t_\-test (two-sided). **d**, Boxplot showing gene expression associated with each BRG1-dependent enhancer groups. Boxes are defined by the 25th to 75th percentiles, the median is shown as a horizontal black line, and whiskers extend up to 1.5 times the IQR from the quartiles. Dashed line indicates the mean signal at 0 nM condition. Each group include 1,037 (G1), 1,036 (G2), 1,037 (G3), 1,036 (G4) and 1,037 (G5) genes. Data represent two independent experiments. **e**, Line plot showing gene expression for each group. Line showed mean of each group, shed area indicates SEM **f**, Gene Ontology (GO) terms enriched for all the down-regulated genes after 100 nM dTAG13 treatment. A two-sided Fisher’s exact test was performed.

### [Extended Data Fig. 7 BRG1 dosage sensitivity in cellular homeostasis.](https://www.nature.com/articles/s41588-025-02305-z/figures/14)

**a**, Cell proliferation assay under dTAG13 treatment. Cells are counted and passaged every 3-days. Data are presented as mean ± SEM from five independent experiments. _P_ values were calculated with Student’s _t_\-test (two-sided). **b**, Representative images of alkaline phosphatase (AP) staining upon dTAG13 treatment at different concentrations for 6-days (left), and quantification of the AP staining (right). Colonies were classified into three groups based on AP signals and morphology. Red, orange and gray color represent undifferentiated (++), intermediate (+) and differentiated (-), respectively. Data are presented as mean ± SD from three independent experiments. **c**, Relative gene expression of Polycomb-targets at different BRG1 levels. Expression levels were normalized to the expression at 0 nM dTAG13 (DMSO). Data represent two independent experiments.

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM5)

### [Extended Data Fig. 8 Knockout of BRM does not perturb cell homeostasis in BEAS-2B cells.](https://www.nature.com/articles/s41588-025-02305-z/figures/15)

**a**, Western blot analysis of mSWI/SNF components in WT BEAS-2B cells after treatment with 0 nM AU-15330 (DMSO) or 1,000 nM AU-15330 for 24 hours. Data are representative of four independent experiments. **b**, Schematic illustration of BRM-KO strategy (top). Red bar indicates sgRNA-target and green arrows indicate primer pair for genotyping. Knockout of _BRM_ was confirmed by PCR genotyping (left) and Western blot (right). Data are representative of two independent experiments. **c**, Representative bright field image of WT (left) and _BRM_\-KO (right) BEAS-2B cells. Data are representative of two independent experiments. **d**, Heatmap showing profiles of BRG1 (red) and BRM (blue) in WT and _BRM_\-KO BEAS-2B cells. The top panels show aggregate coverage plots with mean enrichment. **e**, Western blot analysis of BRG1 and PBRM1 in _BRM_\-KO BEAS-2B cells treated with 1,000 nM AU-15330 for the indicated time. Data are representative of two independent experiments. **f**, Immunostaining of BRG1 in _BRM_\-KO BEAS-2B cells after treatment with the indicated concentrations of AU-15330 for 24 hours. **g**, Relative BRG1 intensity measured from the immunostaining shown in (f). Each dot represents a cell. **h**, Western blot analysis of PBRM1 in _BRM_\-KO BEAS-2B cells after treatment with the indicated concentrations of AU-15330 for 24 hours (top). Quantification of relative BRG1 abundance following AU-15330 treatment (bottom). BRG1 expression relative to Tubulin was normalized to DMSO-treated (0 nM) _BRM_\-KO BEAS-2B cells. Data are presented as mean ± SEM from four independent experiments.

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM4)

### [Extended Data Fig. 9 BRG1 dosage sensitivity and effects can also be observed in the BEAS-2B lung epithelial cells.](https://www.nature.com/articles/s41588-025-02305-z/figures/16)

**a**, Comparison of ATAC-seq changes at BRG1-targeted regions following 24 hours of AU-15330 treatment at 0 nM (DMSO, x-axis), and 10–1,000 nM (y-axis). **b**, The numbers of differentially accessible regions (DARs) identified at different AU-15330 concentrations. **c**, Heatmap showing BRG1-binding (Blue) and ATAC-seq (Red) for BRG1-independent and -dependent regulatory elements. Indep, BRG1-independent regions. Dep, BRG1-dependent regions. **d**, Genome browser view showing BRG1 binding and ATAC-seq signals across regions containing promoters, enhancers, and insulators at a representative locus. **e**, Transcription factor motif enrichment at BRG1-independent (Indep) and BRG1-dependent (Dep) ATAC-seq peaks in promoters, enhancers, and insulators. The color scale represents −log10(P value), and dot size corresponds to the effect size measured as odds ratio (OR). A two-sided Fisher’s exact test was performed. **f**, Heatmap showing BRG1-dependent enhancer regulatory elements, clustered into five groups based on delta values, ranging from buffered to linear response regions. **g**, Representative genome browser views showing linear and buffered response to BRG1 dosage (right). Red dots represent linearly sensitive regions, and blue dots represent buffered response regions (left). The linear and buffered models are illustrated in the diagram. **h**, _KRAS_ expression relative to _GAPDH_ at different BRG1 levels. The level is further normalized by 0 nM AU-15330 conditions. Data represent three independent experiments, each with three technical replicates. **i**, Relative DNA counts after 1, 3, and 6 days of AU-15330 treatment at the indicated concentrations. The signal was normalized to the 0 nM condition for samples at the indicated days. Data represent three independent experiments, each with three technical replicates. _P_ values were calculated with Student’s _t_\-test (two-sided), comparing samples of the same genotype and treatment time.

[Source data](https://www.nature.com/articles/s41588-025-02305-z#MOESM5)

Supplementary information
-------------------------

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_MOESM1_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_MOESM2_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_MOESM3_ESM.xlsx)

Supplementary Table 1. List of single guide RNAs used in this study. Supplementary Table 2. List of primer sequences used in this study for PCR and RT–qPCR. Supplementary Table 3. List of antibodies and related information used in this study. Supplementary Table 4. List of public datasets used in the study and their GSE accession codes.

Source data
-----------

### [Source Data 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_MOESM4_ESM.pdf)

Uncropped western blot and gel images for Figs. 1, 3 and 6 and Extended Data Figs. 1 and 8.

### [Source Data 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02305-z/MediaObjects/41588_2025_2305_MOESM5_ESM.xlsx)

Statistical source data for Figs. 1, 2, 5 and 6 and Extended Data Figs. 1 and 7–9.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Precise%20modulation%20of%20BRG1%20levels%20reveals%20features%20of%20mSWI%2FSNF%20dosage%20sensitivity&author=Yota%20Hagihara%20et%20al&contentID=10.1038%2Fs41588-025-02305-z&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-22&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02305-z)

### Cite this article

Hagihara, Y., Zhang, C. & Zhang, Y. Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02305-z

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02305-z?format=refman&flavour=citation)

-   Received: 10 October 2024
    
-   Accepted: 21 July 2025
    
-   Published: 22 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02305-z
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02305-z.pdf)
