> id: https://www.nature.com/articles/s41588-025-02247-6

> link: https://www.nature.com/articles/s41588-025-02247-6

> title: Single-cell Micro-C profiles 3D genome structures at high resolution and characterizes multi-enhancer hubs

# Single-cell Micro-C profiles 3D genome structures at high resolution and characterizes multi-enhancer hubs
_Published on Wed, 02 Jul 2025 00:00:00 GMT_

Abstract
--------

In animal genomes, regulatory DNA elements called enhancers govern precise spatiotemporal gene expression patterns in specific cell types. However, the spatial organization of enhancers within the nucleus to regulate target genes remains poorly understood. Here we report single-cell Micro-C (scMicro-C), a micrococcal nuclease-based three-dimensional (3D) genome mapping technique with an improved spatial resolution of 5 kb, and identified a specialized 3D enhancer structure termed ‘promoter–enhancer stripes (PESs)’, connecting a gene’s promoter to multiple enhancers. PES are formed by cohesin-mediated loop extrusion, which potentially brings multiple enhancers to the promoter. Further, we observed the prevalence of multi-enhancer hubs on genes with PES within single-cell 3D genome structures, wherein multiple enhancers form a spatial cluster in association with the gene promoter. Through its improved resolution, scMicro-C elucidates how enhancers are spatially coordinated to control genes.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-023-01391-1/MediaObjects/41588_2023_1391_Fig1_HTML.png)

### [Region Capture Micro-C reveals coalescence of enhancers and promoters into nested microcompartments](https://www.nature.com/articles/s41588-023-01391-1?fromPaywallRec=false)

Article 08 May 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41580-024-00710-6/MediaObjects/41580_2024_710_Fig1_HTML.png)

### [Enhancer selectivity in space and time: from enhancer–promoter interactions to promoter activation](https://www.nature.com/articles/s41580-024-00710-6?fromPaywallRec=false)

Article 27 February 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-025-57920-5/MediaObjects/41467_2025_57920_Fig1_HTML.png)

### [Programmable mRNA therapeutics for controlled epigenomic modulation of single and multiplexed gene expression in diverse diseases](https://www.nature.com/articles/s41467-025-57920-5?fromPaywallRec=false)

Article Open access 13 March 2025

Main
----

The human genome is hierarchically folded into a myriad of three-dimensional (3D) structures[1](https://www.nature.com/articles/s41588-025-02247-6#ref-CR1 "Rowley, M. J. & Corces, V. G. Organizational principles of 3D genome architecture. Nat. Rev. Genet. 19, 789–800 (2018)."),[2](https://www.nature.com/articles/s41588-025-02247-6#ref-CR2 "Bonev, B. & Cavalli, G. Organization and function of the 3D genome. Nat. Rev. Genet. 17, 772 (2016)."),[3](https://www.nature.com/articles/s41588-025-02247-6#ref-CR3 "Misteli, T. The self-organizing genome: principles of genome architecture and function. Cell 183, 28–45 (2020)."),[4](https://www.nature.com/articles/s41588-025-02247-6#ref-CR4 "Oudelaar, A. M. & Higgs, D. R. The relationship between genome structure and function. Nat. Rev. Genet. 22, 154–168 (2021)."), composed of DNA loops at different genomic scales. This intricate organization facilitates interactions between regulatory elements, thereby exerting a profound influence on cellular identity and function. Enhancers are _cis_\-regulatory elements (CREs) bound by a specific combination of transcription factors (TFs), which activate the expression of target genes by TF-recruited co-activators[5](https://www.nature.com/articles/s41588-025-02247-6#ref-CR5 "Cramer, P. Organization and regulation of gene transcription. Nature 573, 45–54 (2019)."). Notably, enhancers can be positioned at considerable genomic distances—several hundreds of kilobases to even megabases—away from their target genes[6](https://www.nature.com/articles/s41588-025-02247-6#ref-CR6 "Lettice, L. A. et al. A long-range Shh enhancer regulates expression in the developing limb and fin and is associated with preaxial polydactyly. Hum. Mol. Genet. 12, 1725–1735 (2003)."). Evidence indicates that many enhancers function through the formation of chromatin loops, bringing the enhancer and promoter into spatial proximity[7](https://www.nature.com/articles/s41588-025-02247-6#ref-CR7 "Blackwood, E. M. & Kadonaga, J. T. Going the distance: a current view of enhancer action. Science 281, 60–63 (1998)."),[8](https://www.nature.com/articles/s41588-025-02247-6#ref-CR8 "Bulger, M. & Groudine, M. Looping versus linking: toward a model for long-distance gene activation. Genes Dev. 13, 2465–2477 (1999)."). However, the investigation of the structure of enhancer–promoter (E–P) loops at the single-cell level has been impeded by limitations in spatial resolution.

There are more than 1 million CREs in the mammalian genome, among which tens of thousands are active within specific cell types. This diversity adds complexity to the regulatory relationship between enhancers and their target genes: multiple enhancers may collectively regulate a single gene, whereas a single enhancer may regulate one gene or influence the expression of multiple genes. Recent studies have reported the existence of multi-enhancer hubs, where multiple enhancers form a spatial cluster to coordinate gene regulation[9](https://www.nature.com/articles/s41588-025-02247-6#ref-CR9 "Tsai, A., Alves, M. R. & Crocker, J. Multi-enhancer transcriptional hubs confer phenotypic robustness. eLife 8, e45325 (2019)."),[10](https://www.nature.com/articles/s41588-025-02247-6#ref-CR10 "Chandra, A. et al. Quantitative control of Ets1 dosage by a multi-enhancer hub promotes Th1 cell differentiation and protects from allergic inflammation. Immunity 56, 1451–1467 (2023)."),[11](https://www.nature.com/articles/s41588-025-02247-6#ref-CR11 "Uyehara, C. M. & Apostolou, E. 3D enhancer-promoter interactions and multi-connected hubs: organizational principles and functional roles. Cell Rep. 42, 112068 (2023)."),[12](https://www.nature.com/articles/s41588-025-02247-6#ref-CR12 "Allahyar, A. et al. Enhancer hubs and loop collisions identified from single-allele topologies. Nat. Genet. 50, 1151–1160 (2018)."),[13](https://www.nature.com/articles/s41588-025-02247-6#ref-CR13 "Madsen, J. G. S. et al. Highly interconnected enhancer communities control lineage-determining genes in human mesenchymal stem cells. Nat. Genet. 52, 1227–1238 (2020)."),[14](https://www.nature.com/articles/s41588-025-02247-6#ref-CR14 "Li, W. et al. scNanoHi-C: a single-cell long-read concatemer sequencing method to reveal high-order chromatin structures within individual cells. Nat. Methods 20, 1493–1505 (2023)."),[15](https://www.nature.com/articles/s41588-025-02247-6#ref-CR15 "Deshpande, A. S. et al. Identifying synergistic high-order 3D chromatin conformations from genome-scale nanopore concatemer sequencing. Nat. Biotechnol. 40, 1488–1499 (2022)."), including, in some cases, enhancers from different chromosomes[16](https://www.nature.com/articles/s41588-025-02247-6#ref-CR16 "Markenscoff-Papadimitriou, E. et al. Enhancer interaction networks as a means for singular olfactory receptor expression. Cell 159, 543–557 (2014)."). Despite the observation of multi-way interactions among enhancers using long-read 3C derivatives[14](https://www.nature.com/articles/s41588-025-02247-6#ref-CR14 "Li, W. et al. scNanoHi-C: a single-cell long-read concatemer sequencing method to reveal high-order chromatin structures within individual cells. Nat. Methods 20, 1493–1505 (2023)."),[15](https://www.nature.com/articles/s41588-025-02247-6#ref-CR15 "Deshpande, A. S. et al. Identifying synergistic high-order 3D chromatin conformations from genome-scale nanopore concatemer sequencing. Nat. Biotechnol. 40, 1488–1499 (2022).") and SPRITE[17](https://www.nature.com/articles/s41588-025-02247-6#ref-CR17 "Quinodoz, S. A. et al. Higher-order inter-chromosomal hubs shape 3D genome organization in the nucleus. Cell 174, 744–757 (2018).") techniques, the prevalence and 3D organization of multi-enhancer hub structures within individual cells remain poorly understood.

Loop extrusion has a central role in shaping 3D genome organization. During this process, the structural maintenance of chromosome (SMC) complexes, condensin and cohesin, bind to chromatin and reel flanking DNA into growing loops until the complexes run into roadblocks such as convergently oriented CCCTC-binding factor (CTCF)[18](https://www.nature.com/articles/s41588-025-02247-6#ref-CR18 "Alipour, E. & Marko, J. F. Self-organization of domain structures by DNA-loop-extruding enzymes. Nucleic Acids Res. 40, 11202–11212 (2012)."),[19](https://www.nature.com/articles/s41588-025-02247-6#ref-CR19 "Sanborn, A. L. et al. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. Proc. Natl Acad. Sci. USA 112, E6456 (2015)."),[20](https://www.nature.com/articles/s41588-025-02247-6#ref-CR20 "Fudenberg, G. et al. Formation of chromosomal domains by loop extrusion. Cell Rep. 15, 2038–2049 (2016)."),[21](https://www.nature.com/articles/s41588-025-02247-6#ref-CR21 "Nasmyth, K. Disseminating the genome: joining, resolving, and separating sister chromatids during mitosis and meiosis. Annu. Rev. Genet. 35, 673–745 (2001)."),[22](https://www.nature.com/articles/s41588-025-02247-6#ref-CR22 "Davidson, I. F. et al. DNA loop extrusion by human cohesin. Science 366, 1338–1345 (2019)."),[23](https://www.nature.com/articles/s41588-025-02247-6#ref-CR23 "Kim, Y., Shi, Z., Zhang, H., Finkelstein Ilya, J. & Yu, H. Human cohesin compacts DNA by loop extrusion. Science 366, 1345–1349 (2019)."),[24](https://www.nature.com/articles/s41588-025-02247-6#ref-CR24 "Davidson, I. F. & Peters, J.-M. Genome folding through loop extrusion by SMC complexes. Nat. Rev. Mol. Cell Biol. 22, 445–464 (2021)."). Loop extrusion is also suggested to have a role in facilitating the establishment of E–P interactions[25](https://www.nature.com/articles/s41588-025-02247-6#ref-CR25 "Karpinska, M. A. & Oudelaar, A. M. The role of loop extrusion in enhancer-mediated gene activation. Curr. Opin. Genet. Dev. 79, 102022 (2023)."),[26](https://www.nature.com/articles/s41588-025-02247-6#ref-CR26 "Calderon, L. et al. Cohesin-dependence of neuronal gene expression relates to chromatin loop length. eLife 11, e76539 (2022)."),[27](https://www.nature.com/articles/s41588-025-02247-6#ref-CR27 "Kane, L. et al. Cohesin is required for long-range enhancer action at the Shh locus. Nat. Struct. Mol. Biol. 29, 891–897 (2022)."). Loop extrusion may offer an efficient mechanism for promoters or CREs to scan adjacent chromatin in search of their targets[12](https://www.nature.com/articles/s41588-025-02247-6#ref-CR12 "Allahyar, A. et al. Enhancer hubs and loop collisions identified from single-allele topologies. Nat. Genet. 50, 1151–1160 (2018)."),[28](https://www.nature.com/articles/s41588-025-02247-6#ref-CR28 "Zhang, X. et al. Fundamental roles of chromatin loop extrusion in antibody class switching. Nature 575, 385–389 (2019)."),[29](https://www.nature.com/articles/s41588-025-02247-6#ref-CR29 "Zhang, Y. et al. The fundamental role of chromatin loop extrusion in physiological V(D)J recombination. Nature 573, 600–604 (2019)."),[30](https://www.nature.com/articles/s41588-025-02247-6#ref-CR30 "Kiefer, L. et al. WAPL functions as a rheostat of protocadherin isoform diversity that controls neural wiring. Science 380, eadf8440 (2023)."),[31](https://www.nature.com/articles/s41588-025-02247-6#ref-CR31 "Chang, L.-H. et al. Multi-feature clustering of CTCF binding creates robustness for loop extrusion blocking and topologically associating domain boundaries. Nat. Commun. 14, 5615 (2023)."), although this role for extrusion has not been extensively characterized.

Chromosome conformation capture (3C or Hi-C) assays have advanced the understanding of 3D genome structures by generating genome-wide contact maps. These maps are created using restriction enzymes to cut specific sequences, allowing spatially proximal DNA fragments to ligate before conducting whole-genome sequencing[32](https://www.nature.com/articles/s41588-025-02247-6#ref-CR32 "Dekker, J., Rippe, K., Dekker, M. & Kleckner, N. Capturing chromosome conformation. Science 295, 1306–1311 (2002)."),[33](https://www.nature.com/articles/s41588-025-02247-6#ref-CR33 "Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science 326, 289–293 (2009)."). Bulk Hi-C has relatively low resolution due to restriction enzymes having limited cutting sites in the genome. Recently, Micro-C has been developed using micrococcal nuclease (MNase), which cuts genomic DNA between two nucleosomes regardless of specific sequence, creating nucleosome-sized fragments for proximity ligation[34](https://www.nature.com/articles/s41588-025-02247-6#ref-CR34 "Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Mol. Cell 78, 539–553 e8 (2020)."),[35](https://www.nature.com/articles/s41588-025-02247-6#ref-CR35 "Krietenstein, N. et al. Ultrastructural details of mammalian chromosome architecture. Mol. Cell 78, 554–565 e7 (2020)."),[36](https://www.nature.com/articles/s41588-025-02247-6#ref-CR36 "Hsieh, T. H. et al. Mapping nucleosome resolution chromosome folding in yeast by Micro-C. Cell 162, 108–119 (2015)."), thereby achieving much higher resolution. However, bulk Hi-C and Micro-C measurements can only measure population-averaged contacts and are unable to distinguish the variability of chromosome folding among individual cells. To solve this problem, we previously developed Dip-C to determine the 3D genome structure of a single human cell at 20-kb resolution by distinguishing the paternal and maternal alleles based on their SNPs[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."). In this work, we further improved the precision of single-cell 3D genome determination by developing single-cell Micro-C (scMicro-C).

Results
-------

### Development of scMicro-C

The previously reported bulk Micro-C chemistry, although offering nucleosome-level contact maps, cannot be applied to single cells because of substantial loss of DNA caused by overdigestion and low ligation efficiency[34](https://www.nature.com/articles/s41588-025-02247-6#ref-CR34 "Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Mol. Cell 78, 539–553 e8 (2020)."),[35](https://www.nature.com/articles/s41588-025-02247-6#ref-CR35 "Krietenstein, N. et al. Ultrastructural details of mammalian chromosome architecture. Mol. Cell 78, 554–565 e7 (2020)."). To achieve scMicro-C, we implemented three key improvements (Fig. [1a](https://www.nature.com/articles/s41588-025-02247-6#Fig1); [Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)). First, we employed an ionic detergent (sodium dodecyl sulfate (SDS)) to solubilize chromatin and improve ligation efficiency. Second, we systematically titrated the degree of MNase digestion to assess the effect on chromatin structure detection. Third, we omitted biotin enrichment and adopted a transposon-based whole-genome amplification (WGA) technique called multiplex end-tagging amplification (META)[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018).") to enhance the detection of chromatin contacts.

**Fig. 1: Development of scMicro-C.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02247-6/figures/1)

**a**, Schematic of the scMicro-C procedure. **b**, Chromatin length distribution to compare the ligation efficiency between SDS treatment and the original Micro-C protocol. Data generated by the capillary electrogram. **c**, Violin plot to compare the contact number between SDS treatment and original protocol (SDS, _n_ = 22; original, _n_ = 33). The box middle line represents the median value, the box limits the 25% and 75% quantiles, and the whiskers show the minimum and maximum. **d**, The chromatin length distribution depicts the degree of digestion resulting from MNase titration. **e**,**f**, Comparison among scMicro-C with different MNase titration and Dip-C, showing the percentage of reads containing contacts (**e**); the horizontal line and the box represent the median and quartiles, respectively (bottom). The whiskers indicate minima and maxima (**e**) and a downsample plot depicting the relationship between the number of reads and the number of unique contacts (**f**). The line indicates the median value, and the shadow indicates the 95% confidence interval. (**f**). 200U, _n_ = 96; 600U, _n_ = 96; 800U, _n_ = 96; 1,000U, _n_ = 96 and Dip-C, _n_ = 17. **g**, Pile-up results of chromatin loops (bulk Micro-C detected loop set, _n_ = 45,174) for four MNase titration groups. **h**, Normalized nucleosome occupancy around CTCF binding sites (left) and active TSS (right) of five scMicro-C datasets. **i**, Two-dimensional histograms to show the correlation of A/B compartment values at 100-kb resolution (left) and insulation scores at 10-kb resolution (right) between ensemble scMicro-C and bulk Micro-C. **j**, Contact maps of ensemble scMicro-C (top left) and bulk Micro-C (bottom right) at 500 kb, 50 kb, 5 kb and 1 kb from left to right to show the compartments, TADs and chromatin loops, respectively. TADs, topologically associating domains.

[Source data](https://www.nature.com/articles/s41588-025-02247-6#MOESM5)

Inspired by the Hi-C procedure[38](https://www.nature.com/articles/s41588-025-02247-6#ref-CR38 "Belton, J.-M. et al. Hi-C: a comprehensive technique to capture the conformation of genomes. Methods 58, 268–276 (2012)."), we introduced SDS treatment after MNase fragmentation in the Micro-C procedure. This step improved the chromatin accessibility for end-repair enzymes, thereby boosting ligation efficiency compared to the original bulk Micro-C procedure[34](https://www.nature.com/articles/s41588-025-02247-6#ref-CR34 "Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Mol. Cell 78, 539–553 e8 (2020)."),[35](https://www.nature.com/articles/s41588-025-02247-6#ref-CR35 "Krietenstein, N. et al. Ultrastructural details of mammalian chromosome architecture. Mol. Cell 78, 554–565 e7 (2020)."), independent of chromatin digestion levels (Fig. [1b](https://www.nature.com/articles/s41588-025-02247-6#Fig1) and Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02247-6#Fig5)). To test whether enhanced ligation efficiency improves the detection of chromatin contacts in individual cells, we isolated and processed single cells and performed WGA and sequencing. The data show that SDS treatment increases detected chromatin contacts by 8.1 times at similar sequencing depths (Fig. [1c](https://www.nature.com/articles/s41588-025-02247-6#Fig1), Extended Data Fig. [1b–d](https://www.nature.com/articles/s41588-025-02247-6#Fig5) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02247-6#MOESM4)). This may be attributable to the larger size of end repair enzymes (T4 polynucleotide kinase = 132 kDa and Klenow fragment = 68.2 kDa) compared to MNase (16.9 kDa), which poses challenges in accessing chromatin without SDS.

After establishing a protocol with high ligation efficiency, we next aimed to investigate how MNase digestion levels impact chromatin structure detection in scMicro-C experiments. Using four distinct MNase concentrations (200U, 600U, 800U and 1,000U containing approximately 4 million nuclei in a 100-μl reaction volume), we tested varying digestion degrees from low to high (Fig. [1d](https://www.nature.com/articles/s41588-025-02247-6#Fig1)). Notably, even the highest concentration (1,000U) did not reach the level of digestion of standard bulk Micro-C. We note that the high degree of digestion in bulk Micro-C statistically diminished ligation efficiency (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02247-6#Fig5)), as previously observed in the Micro-Capture-C (MCC) procedure[39](https://www.nature.com/articles/s41588-025-02247-6#ref-CR39 "Hua, P. et al. Defining genome architecture at base-pair resolution. Nature 595, 125–129 (2021)."). Subsequently, we employed fluorescence-activated nuclei sorting to sort individual nuclei into a 96-well plate for each MNase concentration. Sequencing analysis revealed median counts of chromatin contacts per cell—671.6k (200U), 817.1k (600U), 863.2k (800U) and 874.4k (1,000U; Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02247-6#Fig6)). We observed slightly increased inter-chromosomal contacts at higher levels of MNase digestion (Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02247-6#Fig6)).

With increasing MNase concentration, the single-cell profiles exhibited a higher percentage of reads containing contacts (referred to as contact ratio) and a greater number of unique contacts at the same sequencing depth (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02247-6#Fig1) and Extended Data Fig. [2c](https://www.nature.com/articles/s41588-025-02247-6#Fig6)). We note that the contact ratio is compromised when compared to bulk Micro-C employing biotin pulldown, but is notably higher than our previously developed scHi-C method, Dip-C[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."), particularly at higher MNase concentrations (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02247-6#Fig1)).

Notably, we found that chromatin structures across various scales remain largely consistent among different MNase concentrations (Fig. [1g](https://www.nature.com/articles/s41588-025-02247-6#Fig1) and Extended Data Fig. [2d–g](https://www.nature.com/articles/s41588-025-02247-6#Fig6)). Like bulk Micro-C, scMicro-C preserves nucleosome occupancy and TF footprinting profiles[39](https://www.nature.com/articles/s41588-025-02247-6#ref-CR39 "Hua, P. et al. Defining genome architecture at base-pair resolution. Nature 595, 125–129 (2021)."), with nucleosome occupancy profiles becoming slightly sharper at higher MNase concentrations (Fig. [1h](https://www.nature.com/articles/s41588-025-02247-6#Fig1) and Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02247-6#Fig7)). These features are not offered by restriction enzyme-based scHi-C (Fig. [1h](https://www.nature.com/articles/s41588-025-02247-6#Fig1) and Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02247-6#Fig7)). Based on our observations, we determined that the optimal choice for scMicro-C experiments is a digestion level of 800U.

Following the optimal scMicro-C procedure, we applied scMicro-C to GM12878 cells. In total, we profiled over 800 GM12878 cells (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02247-6#MOESM4)) and kept 724 high-quality cells after quality control ([Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)), achieving a median of 835k unique chromatin contacts per cell (s.d. = 467k, min = 228k and max = 3.5 m). To assess the fidelity of scMicro-C in capturing multiscale 3D genome organization, we aggregated all single-cell profiles (referred to as ensemble scMicro-C), yielding a contact map containing 750 million contacts.

To validate our modified Micro-C methodology, we generated two bulk Micro-C datasets for GM12878 cells using distinct protocols, as detailed in the [Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10) (modified bulk Micro-C). One dataset adhered to our modified Micro-C protocol, denoted as dataset 1, while the other dataset was produced using the original Micro-C procedure[34](https://www.nature.com/articles/s41588-025-02247-6#ref-CR34 "Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Mol. Cell 78, 539–553 e8 (2020)."), designated as dataset 2. The comparison between the two datasets confirms that the modified protocol does not compromise data quality (Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)). Together, the two datasets yielded a total of 4.4 billion valid contacts. Upon comparison with published GM12878 Hi-C data at the highest available sequencing depth (4.9 billion contacts)[40](https://www.nature.com/articles/s41588-025-02247-6#ref-CR40 "Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014)."), our bulk Micro-C data exhibited a higher signal-to-noise ratio, as evidenced by a greater number of chromatin loops (HICCUPS—20882 versus 9738) and chromatin stripes (Stripenn—3414 versus 2722), and stronger chromatin loop and stripe signals (Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)).

Comparison between bulk Micro-C and ensemble scMicro-C reveals a high level of agreement across scales, encompassing A/B compartments, topologically associating domains and fine-scale chromatin loops (Fig. [1i,j](https://www.nature.com/articles/s41588-025-02247-6#Fig1)). Notably, ensemble scMicro-C delineates chromatin features at resolutions as fine as 1 kb (Extended Data Fig. [4c](https://www.nature.com/articles/s41588-025-02247-6#Fig8)). Quantitatively, our bulk Micro-C achieves 1-kb resolution, and ensemble scMicro-C 5-kb resolution (Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02247-6#MOESM1); [Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)), with resolution defined as previously described[40](https://www.nature.com/articles/s41588-025-02247-6#ref-CR40 "Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014)."). Additionally, akin to Micro-C, when compared to bulk Hi-C and scHi-C methodologies, scMicro-C exhibits a superior signal-to-noise ratio in the detection of chromatin loops and stripes (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02247-6#Fig8)).

### scMicro-C resolves 3D genomes at kilobase resolution

Contact map-based analysis is inherently limited to two dimensions and is unable to accurately determine the position of genomic loci in 3D space. To address this limitation, several algorithms have been devised for reconstructing the 3D genome structure of both haploid and diploid cells from single-cell contact maps[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."),[41](https://www.nature.com/articles/s41588-025-02247-6#ref-CR41 "Nagano, T. et al. Single-cell Hi-C reveals cell-to-cell variability in chromosome structure. Nature 502, 59–64 (2013)."),[42](https://www.nature.com/articles/s41588-025-02247-6#ref-CR42 "Stevens, T. J. et al. 3D structures of individual mammalian genomes studied by single-cell Hi-C. Nature 544, 59–64 (2017)."), providing insights into the radial organization of genomic loci[43](https://www.nature.com/articles/s41588-025-02247-6#ref-CR43 "Tan, L. et al. Changes in genome architecture and transcriptional dynamics progress independently of sensory experience during post-natal brain development. Cell 184, 741–758.e17 (2021)."), chromosome compaction[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018).") and multiway interactions. Notably, our previous restriction enzyme-based Dip-C method enabled the reconstruction of 3D genome structures of diploid cells at 20-kb resolution[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."). A resolution of 20 kb signifies that each particle of the 3D genome structure reconstruction represents a genomic locus of 20 kb. The definition of resolution is data-driven, reflecting minimal variability across multiple independent reconstructions ([Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)). This concept aligns with DNA FISH-based microscopy methods. Despite this advancement, the 20-kb resolution of the current scHi-C technique remains insufficient for investigating the intricate folding patterns of finer-scale structures like E–P loops and chromatin stripes.

Applying our previously developed Dip-C algorithm to scMicro-C data, we reconstructed 3D genome structures of individual GM12878 cells ([Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)). Among the 724 cells analyzed, 358 cells (49.4%), 335 cells (46.1%) and 281 cells (38.8%) resolve whole-cell 3D genome structures of 20-kb, 10-kb and 5-kb resolution, respectively. Only cells demonstrating low uncertainty at each specific resolution were considered for downstream structure-based analysis ([Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)). Quality control confirmed that cells displayed high consistency across different replicates and resolutions (Extended Data Fig. [5a,b](https://www.nature.com/articles/s41588-025-02247-6#Fig9)). The attainment of 3D genome structures at a 5-kb ‘particle size’ enables the exploration of the finer-scale folding intricacies within chromatin structures (Fig. [2a](https://www.nature.com/articles/s41588-025-02247-6#Fig2)).

**Fig. 2: Kilobase 3D genome structures of scMicro-C.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02247-6/figures/2)

**a**, A haplotype-imputed contact map of a representative GM12878 cell (top), the corresponding reconstructed 3D genome structure at 5-kb resolution (middle) and a selected region (chr4: 184–185 Mb) are shown at 20-kb, 10-kb and 5-kb resolution (bottom). **b**, Heatmaps show the mean 3D distance matrices measured from single-cell 3D genome structures (top right) and the bulk contact frequency maps (bottom left) for the corresponding regions at 20-kb, 10-kb and 5-kb resolution, respectively. **c**, Scatter plots show the correlation between mean 3D distance measured from single-cell 3D genome structures and contact probability measured from bulk Micro-C at 20-kb, 10-kb and 5-kb resolution, respectively, of the same region at **b**. **d**, Contact map of a nested chromatin loop region (chr1: 224.4–225 Mb), CTCF and RAD21 ChIP–seq tracks, and cartoon schematics of three loop anchors were shown at the top. **e**, Venn diagram to show the overlap of the occurrence of three nested chromatin loops. A chromatin loop was defined as being formed when the 3D distance between two loop anchors was less than 3.5 particle radii (~240 nm). For this region, 428 single-cell 3D structures were available. The co-occurrence between loop1 and loop2, _P_ = 2.3 × 10−9; co-occurrence between loop1 and loop3, _P_ = 2.43 × 10−5; co-occurrence between loop1 and loop3, _P_ = 1.02 × 10−12. A hypergeometric test (one-sided) was used. **f**, Representative single-cell 3D chromatin structures (top) and corresponding mean 3D distance matrices (bottom) for forming loop1, loop2, loop3 and co-occurrence of three loops.

[Source data](https://www.nature.com/articles/s41588-025-02247-6#MOESM6)

To assess the fidelity of the reconstructed single-cell 3D genome structures, we measured the pairwise 3D distance matrices, which exhibit a high degree of anticorrelation with bulk Micro-C contact frequency across various resolutions (Pearson’s _r_ values = −0.931 at 20 kb, −0.922 at 10 kb and −0.907 at 5 kb; Fig. [2b,c](https://www.nature.com/articles/s41588-025-02247-6#Fig2)). Consistent results were also observed in other genomic regions (Extended Data Fig. [5c](https://www.nature.com/articles/s41588-025-02247-6#Fig9)). These findings confirmed that the single-cell kilobase-resolution 3D structures resolved by scMicro-C are both reliable and rich in information.

Reanalysis of deeply sequenced Dip-C data revealed the presence of limited 5-kb resolution cells. Despite the limited number of Dip-C cells, we compared the reconstructed 3D genome structures with those obtained from scMicro-C and observed a high level of consistency between the two methods (Extended Data Fig. [5d,e](https://www.nature.com/articles/s41588-025-02247-6#Fig9)). However, the low number of 5-kb resolution Dip-C cells precludes quantitative benchmarking. Nonetheless, based on the visual comparison of contact maps, we speculate that scMicro-C-derived 3D genome structures offer superior analysis of fine-scale chromatin structure compared to restriction enzyme-based Dip-C.

Using hundreds of 5-kb single-cell 3D genome structures, we investigated fine-scale chromatin folding, particularly multiway interactions. Although the majority of contacts identified through scMicro-C were pairwise, multiple pairwise interactions within a 5-kb bin implied simultaneous multi-loci proximity in 3D genome structures. Focusing on the nested CTCF-anchored chromatin loop (Fig. [2d](https://www.nature.com/articles/s41588-025-02247-6#Fig2)), characterized by the formation of pairwise chromatin loops between loop anchors, we assessed whether these formed independently or concurrently (Supplementary Fig. [4a](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)). Analysis of the overlaps in single-cell 3D structures that form these loops revealed a statistically significant tendency for nested chromatin loops to form simultaneously (Fig. [2e](https://www.nature.com/articles/s41588-025-02247-6#Fig2)). Our scMicro-C 3D structures directly visualized the dynamic loop formation events (Fig. [2f](https://www.nature.com/articles/s41588-025-02247-6#Fig2) and Supplementary Fig. [4b](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)), with similar results observed in other regions featuring nested chromatin loops (Supplementary Fig. [4c–h](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)). Nested chromatin loop formation was also observed in the SPRITE and MC-4C techniques[12](https://www.nature.com/articles/s41588-025-02247-6#ref-CR12 "Allahyar, A. et al. Enhancer hubs and loop collisions identified from single-allele topologies. Nat. Genet. 50, 1151–1160 (2018)."),[17](https://www.nature.com/articles/s41588-025-02247-6#ref-CR17 "Quinodoz, S. A. et al. Higher-order inter-chromosomal hubs shape 3D genome organization in the nucleus. Cell 174, 744–757 (2018)."), which detect multi-way high-order interactions. We speculate that the concurrence of nested CTCF loops is driven by the synergistic stabilization effect of spatially clustered CTCFs, which protects cohesin against WAPL-mediated release[44](https://www.nature.com/articles/s41588-025-02247-6#ref-CR44 "Li, Y. et al. The structural basis for cohesin–CTCF-anchored loops. Nature 578, 472–476 (2020).").

### Characterization of promoter–enhancer stripe (PES)

We next focused on analyzing E–P interactions. We observed that a large number of genes displayed a distinct PES chromatin structure, characterized by a line-like structure on the chromatin contact map. These stripes are anchored at the transcription start site (TSS) and extend in the direction of transcription, indicating frequent and sustained promoter–gene body interactions (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02247-6#Fig10)).

A similar stripe structure has been reported in previous studies, including at the _SOX9_ gene (referred to as a promoter-associated stripe), in the Hi-C pile-up analysis of _Stag2_\-knockout-downregulated genes (referred to as promoter-anchored stripes) and in the pile-up results at the active promoter region of bulk Micro-C data (referred to as gene stripes)[34](https://www.nature.com/articles/s41588-025-02247-6#ref-CR34 "Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Mol. Cell 78, 539–553 e8 (2020)."),[45](https://www.nature.com/articles/s41588-025-02247-6#ref-CR45 "Cheng, N. et al. STAG2 promotes the myelination transcriptional program in oligodendrocytes. eLife 11, e77848 (2022)."),[46](https://www.nature.com/articles/s41588-025-02247-6#ref-CR46 "Chen, L.-F. et al. Structural elements promote architectural stripe formation and facilitate ultra-long-range gene regulation at a human disease locus. Mol. Cell 83, 1446–1461.e6 (2023)."), albeit with a limited number of genes and without extensive characterization. Here we named genes exhibiting this PES configuration as ‘PES genes’, which included key TFs essential for B-cell function, such as early B cell factor 1 (EBF1) and interferon regulatory factor 2 (IRF2). In contrast to PES, previous studies focused primarily on ‘architectural stripes’, which are commonly located at topologically associating domain boundaries and formed between two strong convergently oriented CTCF binding sites[47](https://www.nature.com/articles/s41588-025-02247-6#ref-CR47 "Vian, L. et al. The energetics and physiological impact of cohesin extrusion. Cell 173, 1165–1178 (2018)."). These architectural stripes are typically longer and stronger than PES (Supplementary Fig. [1f](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)) and are not always associated with genes. However, it remains possible that certain PES overlap with or could themselves be considered architectural stripes.

Upon examination, we observed the presence of multiple H3K27ac-marked enhancers distributed along the PES (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02247-6#Fig10)). Noticeably, these stripes frequently exhibited focal interactions at the enhancer sites (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02247-6#Fig10)). Motivated by this observation, we performed de novo detection of PES genes in GM12878 using our bulk Micro-C dataset. We identified 819 genes with PES of 4,077 active long genes (≥50 kb) in GM12878, and pile-up results confirmed the existence of the PES (Fig. [3a](https://www.nature.com/articles/s41588-025-02247-6#Fig3) and Extended Data Fig. [6b,c](https://www.nature.com/articles/s41588-025-02247-6#Fig10)). Notably, genes with PES exhibit significantly higher expression levels than genes without (_P_ = 9.88 × 10−9; Fig. [3b](https://www.nature.com/articles/s41588-025-02247-6#Fig3)).

**Fig. 3: Characterization of PESs.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02247-6/figures/3)

**a**, Pile-up results of GM12878 PES genes (left, _n_ = 819), length-distribution-matched non-PES genes (left center, _n_ = 819) and the difference between stripy and nonstripy genes at 1-kb resolution (right center). Right, pile-up results, rescaled between the TSS and the TTS. **b**, Cumulative curves of expression level for PES and non-PES genes of GM12878. A two-sided Mann–Whitney _U_ test was used. **c**, Metagene plots of mean H3K27ac signal of stripy and nonstripy genes of GM12878. **d**, Metagene plots of mean RAD21 signal of PES and non-PES genes of GM12878. **e**, Pile-up results of PES genes (_n_ = 853) for untreated, _RAD21_\-depleted and _WAPL_\-depleted Micro-C data from mESC at 1-kb resolution. Signal value along the TSS is shown on the far right. Data are from ref. [49](https://www.nature.com/articles/s41588-025-02247-6#ref-CR49 "Hsieh, T.-H. S. et al. Enhancer–promoter interactions and transcription are largely maintained upon acute loss of CTCF, cohesin, WAPL or YY1. Nat. Genet. 54, 1919–1932 (2022)."). **f**, Cartoon model depicting cohesin-mediated loop extrusion facilitating the scanning of downstream enhancers by the promoter.

Metagene analysis revealed stronger enhancer signals (as indicated by the H3K27ac signal) in stripy and nonstripy gene bodies (Fig. [3c](https://www.nature.com/articles/s41588-025-02247-6#Fig3)). These findings were reinforced by activity-by-contact model predictions[48](https://www.nature.com/articles/s41588-025-02247-6#ref-CR48 "Fulco, C. P. et al. Activity-by-contact model of enhancer–promoter regulation from thousands of CRISPR perturbations. Nat. Genet. 51, 1664–1669 (2019)."), which identified multiple stripy gene-associated enhancers distributed along PES (Extended Data Fig. [7a–c,i](https://www.nature.com/articles/s41588-025-02247-6#Fig11)). Promoter–enhancer interaction pile-ups revealed localized intensity peaks at enhancer loci (Extended Data Fig. [6d](https://www.nature.com/articles/s41588-025-02247-6#Fig10)), and robust interactions among multiple enhancers regulating the same stripy gene suggested the formation of enhancer hubs (Extended Data Fig. [6e](https://www.nature.com/articles/s41588-025-02247-6#Fig10)). Collectively, these findings suggest that the potential function of the PES is to facilitate interactions between the promoter and multiple downstream enhancers.

We then focused on elucidating the mechanism of PES formation. We noted statistically stronger cohesin binding at stripy gene promoters compared to nonstripy genes, as indicated by cohesin subunits RAD21, SMC1A and SMC3 (Fig. [3d](https://www.nature.com/articles/s41588-025-02247-6#Fig3) and Extended Data Fig. [7f,i](https://www.nature.com/articles/s41588-025-02247-6#Fig11)), indicating cohesin’s involvement in PES establishment. To test this hypothesis, we examined the effects of cohesin depletion using a Micro-C dataset in mouse embryonic stem cells (mESC)[49](https://www.nature.com/articles/s41588-025-02247-6#ref-CR49 "Hsieh, T.-H. S. et al. Enhancer–promoter interactions and transcription are largely maintained upon acute loss of CTCF, cohesin, WAPL or YY1. Nat. Genet. 54, 1919–1932 (2022).") and observed statistically reduced PES formation after cohesin depletion. While cohesin unloading factor—WAPL depletion extended PES downstream (Fig. [3e](https://www.nature.com/articles/s41588-025-02247-6#Fig3) and Extended Data Fig. [7g,h](https://www.nature.com/articles/s41588-025-02247-6#Fig11)). These experiments demonstrate that PES formation is dependent on cohesin-mediated loop extrusion.

We next investigated whether E–P and enhancer–enhancer (E–E) interactions in genes with PES were influenced by the depletion of cohesin. Our re-analysis of this cohesin depletion Micro-C data[49](https://www.nature.com/articles/s41588-025-02247-6#ref-CR49 "Hsieh, T.-H. S. et al. Enhancer–promoter interactions and transcription are largely maintained upon acute loss of CTCF, cohesin, WAPL or YY1. Nat. Genet. 54, 1919–1932 (2022).") in the context of PES confirms that both E–P and E–E interactions were statistically attenuated following cohesin depletion, with a more pronounced reduction in stripy compared to nonstripy genes (Extended Data Fig. [7j,k](https://www.nature.com/articles/s41588-025-02247-6#Fig11)). Specifically, 21% of E–P interactions in genes with PES were identified as E–P loops, compared to 10.5% in genes without PES. Among these E–P loops, 48.5% in PES genes were statistically weakened, whereas only 35.5% of E–P loops in genes without PES exhibited such weakening (Extended Data Fig. [7l](https://www.nature.com/articles/s41588-025-02247-6#Fig11)).

Given the data, we hypothesize that cohesin-mediated loop extrusion facilitates promoter scanning of downstream genomic loci, thereby fostering interactions with multiple enhancers (Fig. [3f](https://www.nature.com/articles/s41588-025-02247-6#Fig3)). However, further mechanistic studies are required to test this hypothesis. Intriguingly, the presence of nested tiny stripes derived from enhancers in certain stripy genes indicates that the scanning may also be initiated from enhancers (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02247-6#Fig10)), a possibility that is not represented in the current model schematic.

### scMicro-C visualizes dynamic E–P interactions in PES genes

Upon gaining an understanding of the function and formation mechanism of PES, we aimed to characterize the E–P interactions of genes with PES. To do this, we used scMicro-C data to visualize the 3D structure of a specific PES gene called _EBF1_. This gene is located on chromosome 5, spans 404 kb, and has an expression level of 7.59 FPKM. EBF1 is a key regulator of B cell lineage development[50](https://www.nature.com/articles/s41588-025-02247-6#ref-CR50 "Rothenberg, E. V. Transcriptional control of early T and B cell developmental choices. Annu. Rev. Immunol. 32, 283–321 (2014).").

The _EBF1_ locus exhibits a distinct PES on the bulk Micro-C and ensemble scMicro-C contact map, connecting its promoter to seven activity-by-contact model-predicted _EBF1_ enhancers (named E1–E7; Fig. [4a](https://www.nature.com/articles/s41588-025-02247-6#Fig4)). Notably, multiple nested weak stripes within the _EBF1_ PES connecting multiple enhancers were clearly visible (Fig. [4a](https://www.nature.com/articles/s41588-025-02247-6#Fig4)).

**Fig. 4: Multi-enhancer hubs captured by single-cell high-resolution 3D genome structures.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02247-6/figures/4)

**a**, Contact map of ensemble scMicro-C (top left) and bulk Micro-C (bottom right) at the _EBF1_ locus, accompanied by ChIP-seq signals of CTCF, H3K27ac, POLR2A and RAD21, and ABC model-predicted enhancers. **b**, Sorted scMicro-C contact profiles (_n_ = 723) of the _EBF1_ gene with PES; the 5-kb bins covering the _EBF1_ TSS were used for this analysis. **c**, Schematic representation of the promoter and enhancers of the _EBF1_ gene selected for single-cell 3D genome structure analysis. **d**, Line plot showing the percentage of single-cell 3D chromatin structures with the proximity of corresponding E–P pairs. Six regions without stripes were used as controls. The dots indicate the median value, and the whiskers of control regions indicate the standard deviation. **e**, Top row, schematic representation of the E–P loops in each cell (columns). Middle row, representative single-cell 3D structure of the E–P loops. Bottom row, mean 3D distance matrices of single-cell structures, with the depicted loop indicated by a yellow circle. **f**, Bar plot of the percentage of cells (_y_ axis) that have the indicated number of enhancer interactions with the promoter (_x_ axis) at the _EBF1_ (blue) or control locus (red). For the _EBF1_ locus, the number of cells is indicated at the top of each bar. For control regions, the bar indicates the median value, and the error bar indicates the s.d. **g**, Representative single-cell 3D genome structures forming a multi-enhancer hub (red dashed circles) at _EBF1_. **h**, Mean 3D distance matrices measured from single-cell structures with the indicated number of enhancer loci interacting with _EBF1_ promoter from left to right, respectively. **i**, Boxplot of the radius of gyration (_y_\-axis) of single-cell 3D genome structures with the indicated number of enhancer loci (_x_ axis—_n_ = 114, _n_ = 66, _n_ = 51, _n_ = 44, _n_ = 28, _n_ = 8 for each group) interacting with the _EBF1_ promoter. Pairwise statistical tests showed statistical significance (_P_ < 0.05, two-sided Wilcoxon rank sum exact test) for all pairs, except for 1–2 and 3–4. The six-enhancer group was excluded due to an insufficient number of single-cell structures. The centerline indicates the median, the box the 25% and 75% quantiles, and the whiskers the maximum and minimum values. ABC, activity-by-contact.

[Source data](https://www.nature.com/articles/s41588-025-02247-6#MOESM7)

To explore the dynamics of PES formation, we sorted single-cell contact maps based on interactions with the TSS region, observing a gradual transition of contacts from the TSS to downstream regions (Fig. [4b](https://www.nature.com/articles/s41588-025-02247-6#Fig4)). This series of snapshots captures the dynamic contact between the TSS and downstream loci, highlighting the dynamic nature of the observed ‘stripes’ across individual cells. In addition to the main transition line, sporadic contacts were also noted, indicating multiple pairwise interactions between downstream loci and the TSS in individual cells (Fig. [4b](https://www.nature.com/articles/s41588-025-02247-6#Fig4)).

Three-dimensional distance analysis of six enhancer loci (E1–E7, E3/E4 merged at 5-kb resolution; Fig. [4c](https://www.nature.com/articles/s41588-025-02247-6#Fig4)) showed looping events between _EBF1_ promoter and downstream enhancers (E2–E7) occurring in approximately 30% of single-cell structures (Fig. [4d](https://www.nature.com/articles/s41588-025-02247-6#Fig4)), with a looping event defined as a 3D distance between the TSS (P) and another locus (E1–E7) less than 3.5 particle radii (~240 nm, see [Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)). This frequency contrasts with nonstripy controls, where looping frequency declined with genomic distance (Fig. [4d](https://www.nature.com/articles/s41588-025-02247-6#Fig4) and Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)), indicating that genes exhibiting PES have a higher frequency of E–P interactions.

We visualized the dynamic looping process of PES using single-cell 3D genome structures, directly capturing the dynamic E–P looping events between the promoter and multiple downstream enhancers (Fig. [4e](https://www.nature.com/articles/s41588-025-02247-6#Fig4)). Our observations at the _EBF1_ locus were corroborated by similar findings in the analysis of the _IRF2_ gene (chr4: 88.9 kb, 47.7 FPKM), which, like _EBF1_ exhibited a prominent PES structure and contained multiple enhancers within the stripe region (Extended Data Fig. [8a–d](https://www.nature.com/articles/s41588-025-02247-6#Fig12)). These findings collectively suggest that E–P interactions are highly dynamic and heterogeneous among individual cells. It is important to note that while our observations provide insights into the dynamics of looping events based on snapshots of single-cell 3D genome structures, live-imaging techniques are needed for direct visualization of these dynamic chromatin looping events[51](https://www.nature.com/articles/s41588-025-02247-6#ref-CR51 "Gabriele, M. et al. Dynamics of CTCF- and cohesin-mediated chromatin looping revealed by live-cell imaging. Science 376, 496–501 (2022)."),[52](https://www.nature.com/articles/s41588-025-02247-6#ref-CR52 "Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. Nat. Genet. 54, 1907–1918 (2022).").

### Multi-enhancer hubs observed in single-cell 3D genomes

How multiple enhancers are spatially organized to regulate gene expression is not extensively studied. Previous studies using long-read 3C derivatives revealed cooperative 3D interactions between regulatory elements[14](https://www.nature.com/articles/s41588-025-02247-6#ref-CR14 "Li, W. et al. scNanoHi-C: a single-cell long-read concatemer sequencing method to reveal high-order chromatin structures within individual cells. Nat. Methods 20, 1493–1505 (2023)."),[15](https://www.nature.com/articles/s41588-025-02247-6#ref-CR15 "Deshpande, A. S. et al. Identifying synergistic high-order 3D chromatin conformations from genome-scale nanopore concatemer sequencing. Nat. Biotechnol. 40, 1488–1499 (2022)."), such as simultaneous interactions between multiple enhancers and promoters at the α/β-globin gene locus[12](https://www.nature.com/articles/s41588-025-02247-6#ref-CR12 "Allahyar, A. et al. Enhancer hubs and loop collisions identified from single-allele topologies. Nat. Genet. 50, 1151–1160 (2018)."),[53](https://www.nature.com/articles/s41588-025-02247-6#ref-CR53 "Oudelaar, A. M. et al. Single-allele chromatin interactions identify regulatory hubs in dynamic compartmentalized domains. Nat. Genet. 50, 1744–1751 (2018).") and interchromosomal multi-enhancer hubs regulating olfactory receptors (ORs)[54](https://www.nature.com/articles/s41588-025-02247-6#ref-CR54 "Monahan, K., Horta, A. & Lomvardas, S. LHX2- and LDB1-mediated trans interactions regulate olfactory receptor choice. Nature 565, 448–453 (2019).").

We aimed to understand how the multiple enhancers of stripy genes are arranged within individual 3D structures. Upon analyzing single-cell 3D structures at the _EBF1_ locus, we observed a statistically significant number of structures in which more than one enhancer simultaneously interacts with the _EBF1_ promoter (Fig. [4f](https://www.nature.com/articles/s41588-025-02247-6#Fig4)). We refer to this structure as a ‘multi-enhancer hub’. Three-dimensional visualization of these multi-enhancer hubs in individual cells revealed a spatial cluster of enhancers associating with the _EBF1_ promoter (Fig. [4g](https://www.nature.com/articles/s41588-025-02247-6#Fig4)). The simultaneous interaction between the _EBF1_ promoter and multiple enhancers was also supported by findings from the scNanoHi-C study, which uses third-generation long-read sequencing to capture high-order multiway interactions[14](https://www.nature.com/articles/s41588-025-02247-6#ref-CR14 "Li, W. et al. scNanoHi-C: a single-cell long-read concatemer sequencing method to reveal high-order chromatin structures within individual cells. Nat. Methods 20, 1493–1505 (2023).").

Further examination of the 3D distance matrix of single-cell structures harboring between one and six enhancers revealed that a higher number of enhancers in the hub led to a more extensive interconnected enhancer network (Fig. [4h](https://www.nature.com/articles/s41588-025-02247-6#Fig4)). Additionally, we observed that with an increase in the number of enhancers, chromatin structure became more compacted, as indicated by a decrease in the radius of gyration (Fig. [4i](https://www.nature.com/articles/s41588-025-02247-6#Fig4)). These results highlight the cooperative interactions between multiple enhancers in association with the promoter, suggesting a distinctive mechanism of enhancer regulation in stripy genes.

To ascertain whether the presence of multi-enhancer hubs is a common feature among stripy genes or specific to the _EBF1_ gene, we investigated several other stripy genes, including _IRF2_ and _PIEZO2_. Analysis revealed the existence of multi-enhancer hub structures in these genes as well (Extended Data Fig. [8f](https://www.nature.com/articles/s41588-025-02247-6#Fig12) and Supplementary Fig. [6a–e](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)). This prevalence of multi-enhancer hubs was signified by the extensive E–E interactions observed in stripy genes (Extended Data Fig. [6c](https://www.nature.com/articles/s41588-025-02247-6#Fig10)), confirming the assembly of multi-enhancer hubs. Additionally, the conclusions regarding interenhancer connectivity and chromatin compaction were confirmed within the _IRF2_ gene locus (Extended Data Fig. [8g,h](https://www.nature.com/articles/s41588-025-02247-6#Fig12)).

In conclusion, single-cell kilobase-resolution 3D genome structures generated through scMicro-C technology allow for the investigation of E–P interactions at the single-cell level. This approach has unveiled the dynamic and heterogeneous nature of E–P interactions within individual cells, leading to the identification of multi-enhancer hub structures in stripy genes. These findings offer valuable insights into the distinct regulatory mechanisms governing stripy genes through the coordination of multiple enhancers.

### Polymer simulations of PES genes

To dissect the mechanisms driving multi-enhancer interaction networks in genes with PES, we conducted polymer simulations using the open2c ‘polychrom’ framework. We simulated the _EBF1_ locus with the following three key tunable parameters: cohesin-mediated loop extrusion (cohesin density, extrusion speed), cohesin capture probability of enhancers, and molecular affinities between promoters and enhancers (Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02247-6#Fig13); [Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10)).

In our simulation settings, we positioned two strong boundary elements at the TSS and TTS of the _EBF1_ gene, reflecting CTCF binding at these sites (Extended Data Fig. [9a,b](https://www.nature.com/articles/s41588-025-02247-6#Fig13)). Consistent with previous studies, cohesin blocking was sufficient to establish the stripe pattern[46](https://www.nature.com/articles/s41588-025-02247-6#ref-CR46 "Chen, L.-F. et al. Structural elements promote architectural stripe formation and facilitate ultra-long-range gene regulation at a human disease locus. Mol. Cell 83, 1446–1461.e6 (2023)."),[47](https://www.nature.com/articles/s41588-025-02247-6#ref-CR47 "Vian, L. et al. The energetics and physiological impact of cohesin extrusion. Cell 173, 1165–1178 (2018)."). However, in the absence of E–P/E–E molecular affinities and enhancer-mediated cohesin capture, neither apparent E–P/E–E loops nor enhancer stripes formed (Extended Data Fig. [9c](https://www.nature.com/articles/s41588-025-02247-6#Fig13), left). Introducing strong molecular affinities among enhancers and the promoter generates E–P/E–E interactions but fails to replicate stripes unless weak cohesin capture by enhancers was included (Extended Data Fig. [9c–e](https://www.nature.com/articles/s41588-025-02247-6#Fig13)). These patterns completely disappeared upon cohesin depletion. Strong E–P and E–E affinities combined with cohesin capture by enhancers best match experimental observations (Extended Data Fig. [9d,e](https://www.nature.com/articles/s41588-025-02247-6#Fig13), right).

Because the above observations were derived from population-ensemble contact maps, we next focused on single chromatin fiber traces to verify the existence of multi-enhancer hub structures in these simulations. We found that only simulations with strong affinities among promoters and enhancers and cohesin capture by enhancers produced a prominent number of multi-enhancer hub structures (Extended Data Fig. [10a,b](https://www.nature.com/articles/s41588-025-02247-6#Fig14)). The multi-enhancer hub structures were almost completely depleted in the absence of cohesin (Extended Data Fig. [10a,b](https://www.nature.com/articles/s41588-025-02247-6#Fig14)). Additionally, the distance matrices from simulated structures closely resemble observations from scMicro-C 3D genome structures (Extended Data Fig. [10c,d](https://www.nature.com/articles/s41588-025-02247-6#Fig14)).

In summary, computational modeling of 3D chromatin folding provides a complementary approach for exploring possible regulatory mechanisms underlying stripe gene folding, offering insights that are challenging to obtain experimentally. The simulations also confirm the physical feasibility of interconnected multi-enhancer networks at the single-chromatin fiber level.

Discussion
----------

In this study, we improved the spatial resolution of single-cell 3D genome mapping by developing scMicro-C, allowing us to study the intricate folding of fine-scale chromatin structures such as chromatin loops and stripes. We argue that the multiple pairwise interactions overlapping a given genomic segment represent high-order multi-way interactions in 3D genome structures. This proposition is supported by the identification of cooperative interactions among nested chromatin loop anchors and multiple enhancers associated with stripy genes. Our dataset advances our knowledge of genome structure and function, serving as a valuable resource for examining the folding characteristics of uncharacterized chromatin structures, including chromatin jets[55](https://www.nature.com/articles/s41588-025-02247-6#ref-CR55 "Guo, Y. et al. Chromatin jets define the properties of cohesin-driven in vivo loop extrusion. Mol. Cell 82, 3769–3780.e5 (2022).") and fountains[56](https://www.nature.com/articles/s41588-025-02247-6#ref-CR56 "Isiaka, B. N. et al. Cohesin forms fountains at active enhancers in C. elegans. Preprint at bioRxiv 
https://doi.org/10.1101/2023.07.14.549011
(2023).").

Recent studies reported that TFs, coactivators (like Mediator and BRD4) and RNA Pol II assemble into condensates at active DNA regulatory elements and mediate E–P interactions[57](https://www.nature.com/articles/s41588-025-02247-6#ref-CR57 "Cho, W.-K. et al. Mediator and RNA polymerase II clusters associate in transcription-dependent condensates. Science 361, 412–415 (2018)."),[58](https://www.nature.com/articles/s41588-025-02247-6#ref-CR58 "Boija, A. et al. Transcription factors activate genes through the phase-separation capacity of their activation domains. Cell 175, 1842–1855 (2018)."),[59](https://www.nature.com/articles/s41588-025-02247-6#ref-CR59 "Chong, S. et al. Imaging dynamic and selective low-complexity domain interactions that control gene transcription. Science 361, eaar2555 (2018)."),[60](https://www.nature.com/articles/s41588-025-02247-6#ref-CR60 "Sabari, B. R. et al. Coactivator condensation at super-enhancers links phase separation and gene control. Science 361, eaar3958 (2018)."). These transcriptional factors contain intrinsically disordered regions, which are believed to mediate their interactions through multivalent binding. The potential mechanism underlying the formation of multi-enhancer hubs observed in our scMicro-C structures may involve condensates mediated by enhancers and promoter-bound TFs and coactivators. Furthermore, our findings demonstrate that PES, E–P and E–E interactions of PES genes are dependent on cohesin. Thus, cohesin-mediated loop extrusion likely aids the scanning of promoters through multiple downstream enhancers, promoting the establishment of molecular affinities between regulatory elements, leading to the formation of multi-enhancer hubs (Extended Data Fig. [10e](https://www.nature.com/articles/s41588-025-02247-6#Fig14)). The collaboration between cohesin and molecular affinities among transcriptional proteins in establishing E–P interactions has been discussed previously[4](https://www.nature.com/articles/s41588-025-02247-6#ref-CR4 "Oudelaar, A. M. & Higgs, D. R. The relationship between genome structure and function. Nat. Rev. Genet. 22, 154–168 (2021).") but requires further experimental validation.

Multi-enhancer hub structures have been identified in olfactory sensory neurons to ensure singular OR expression[54](https://www.nature.com/articles/s41588-025-02247-6#ref-CR54 "Monahan, K., Horta, A. & Lomvardas, S. LHX2- and LDB1-mediated trans interactions regulate olfactory receptor choice. Nature 565, 448–453 (2019)."),[61](https://www.nature.com/articles/s41588-025-02247-6#ref-CR61 "Monahan, K. et al. Cooperative interactions enable singular olfactory receptor expression in mouse olfactory neurons. eLife 6, e28620 (2017)."). Knockout experiments have proven that the inter-chromosomal multi-enhancer hub is mediated by adapter protein LIM domain binding 1 (LDB1)[54](https://www.nature.com/articles/s41588-025-02247-6#ref-CR54 "Monahan, K., Horta, A. & Lomvardas, S. LHX2- and LDB1-mediated trans interactions regulate olfactory receptor choice. Nature 565, 448–453 (2019)."). LDB1 interacts with OR enhancer-bound TF LHX2 (a LIM domain protein) and possesses a self-association domain that facilitates long-range chromatin interactions through oligomerization[62](https://www.nature.com/articles/s41588-025-02247-6#ref-CR62 "Matthews, J. M. & Visvader, J. E. LIM‐domain‐binding protein 1: a multifunctional cofactor that interacts with diverse proteins. EMBO Rep. 4, 1132–1137 (2003)."). Additionally, LDB1 has been reported to have an important role in mediating long-range chromatin interactions, such as the interactions between the locus control region and the β-globin locus[63](https://www.nature.com/articles/s41588-025-02247-6#ref-CR63 "Song, S.-H., Hou, C. & Dean, A. A positive role for NLI/Ldb1 in long-range β-globin locus control region function. Mol. Cell 28, 810–822 (2007)."). Given that there are hundreds of TFs containing LIM domains in the human genome[64](https://www.nature.com/articles/s41588-025-02247-6#ref-CR64 "Kadrmas, J. L. & Beckerle, M. C. The LIM domain: from the cytoskeleton to the nucleus. Nat. Rev. Mol. Cell Biol. 5, 920–931 (2004)."), LDB1 may act as a universal mediator of E–P interactions. Recent degron experiments have demonstrated the role of LDB1 in mediating interactions between CREs[65](https://www.nature.com/articles/s41588-025-02247-6#ref-CR65 "Aboreden, N. G. et al. LDB1 establishes multi-enhancer networks to regulate gene expression. Mol. Cell 85, 376–393 (2025)."). Therefore, the multi-enhancer hub structure in stripy genes may be mediated by LDB1 (Extended Data Fig. [10f](https://www.nature.com/articles/s41588-025-02247-6#Fig14)).

Recently, a live-imaging study showed that a dynamic assembly of condensates involving super-enhancers and the promoter controls _Sox2_ transcriptional bursting[66](https://www.nature.com/articles/s41588-025-02247-6#ref-CR66 "Du, M. et al. Direct observation of a condensate effect on super-enhancer controlled gene bursting. Cell 187, 331–344 (2024)."). Given the dynamic nature of multi-enhancer hub assembly across individual cells, we speculate that the function of multi-enhancer hubs is to modulate transcriptional bursting of stripy genes. Verification of the existence of these multi-enhancer hubs will require orthogonal techniques. In the future, using the multiplexed DNA fluorescence in situ hybridization (FISH) technique to simultaneously label multiple enhancers and promoter loci may allow direct observation of cooperative interactions. Additionally, incorporating nascent RNA FISH and immunofluorescence targeting transcription-related factors will provide further insights into the functional roles and formation mechanisms of multi-enhancer hubs.

Methods
-------

### Cell culture

GM12878 lymphoblastoid cells, karyotype normal diploid cells, were grown in RPMI1640 medium (Gibco; Thermo Fisher Scientific, 11875093) supplemented with 15% FBS (Gibco, 10091148) and 1% Pen/Strep (Gibco, 15140122), maintained at 37 °C with 5% CO2 at recommended density. Upon collection, cells were spun down and washed once with ice-cold PBS.

### Modified bulk Micro-C

The Bulk Micro-C protocol was adopted from the published Micro-C protocol for mammalian cells[34](https://www.nature.com/articles/s41588-025-02247-6#ref-CR34 "Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Mol. Cell 78, 539–553 e8 (2020)."),[35](https://www.nature.com/articles/s41588-025-02247-6#ref-CR35 "Krietenstein, N. et al. Ultrastructural details of mammalian chromosome architecture. Mol. Cell 78, 554–565 e7 (2020).") with several modifications. Briefly, collected cells were crosslinked with freshly made 1% PFA (EMS, 15714) followed by 3 mM DSG (Thermo Fisher Scientific, 20593), then cells were lysed and titrated to test the appropriate MNase concentration (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02247-6#Fig5)). Before end repair, we added 50 μl 0.3% SDS (Sigma-Aldrich, 71736-100M) and incubated at 62 °C for 10 min, then quenched with 50 μl 3% Triton X-100 (Sigma-Aldrich, 93443) and incubated at 37 °C for 15 min. Then nuclei were end-repaired and biotin labeled, followed by in situ ligation with T4 ligase (NEB, B0202L). Before DNA extraction, nuclei were incubated with exonuclease III (NEB, M0206) to remove unligated biotinylated DNA ends. Extracted DNA, spanning all lengths, was sonicated to an average size of 350 bp for library preparation. Then, performed biotin pull-down and adapter ligation for sequencing.

We generated two datasets. The first followed a modified bulk Micro-C procedure, using all ligated fragments and sonication to ~350 bp for library preparation, termed dataset 1. Another exactly followed original bulk Micro-C protocol[34](https://www.nature.com/articles/s41588-025-02247-6#ref-CR34 "Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. Mol. Cell 78, 539–553 e8 (2020).") without SDS treatment and only selected the ligated di-nucleosome fragments for library preparation, termed dataset 2.

### scMicro-C protocol

Our protocol was modified from mammalian cell Micro-C protocols with three key modifications. First, we optimized MNase digestion levels to produce longer DNA pieces and reduce DNA loss to ensure 40–50% mononucleosomes and 20–25% di-nucleosomes (average 500 bp) instead of 80–90% mononucleosomes and 10–20% dinucleosomes (average 200 bp) as originally Micro-C suggested, because over-digestion produces too short fragments, which hampers transposon-based WGA procedures. Second, we added an ionic detergent (SDS) to solubilize chromatin between the MNase digestion and end repair step, which both preserves Micro-C characteristic nucleosome-resolution chromatin interactions (as suggested by our bulk data) and dramatically increases ligation efficiency (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02247-6#Fig5)), further increasing the length of the final product from ~450 bp (without SDS) to ~1,300 bp, which is critical for single-cell WGA. Third, we used our state-of-the-art WGA method, META, to further increase contact detection efficiency in single cells. We omitted all biotin-related steps to maximize the number of contacts detected per cell, as described in the Dip-C procedure[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."). For a comprehensive step-by-step guide to the scMicro-C protocol, please refer to protocols.io ([https://doi.org/10.17504/protocols.io.kqdg39wbzg25/v1](https://doi.org/10.17504/protocols.io.kqdg39wbzg25/v1)).

#### Cell crosslinking

Cells were fixed with 1% PFA (EMS, 15714) at room temperature for 10 min with rotation. PFA was quenched by the addition of 2 M Tris–HCl, pH 7.5, to a final concentration of 0.75 M and incubated at room temperature for 5 min. Then wash twice with ice-cold 1× PBS supplemented with 1× BSA (centrifugation 3,000_g_ for 5 min). After crosslinking, some pellets appear and then disappear after the sample is washed twice. Then cells were further fixed in 3 mM DSG in PBS and incubated at room temperature for 45 min. DSG was quenched by the addition of 2 M Tris–HCl, pH 7.5, to a final concentration of 0.75 M and incubated at room temperature for 5 min. After fixation, cells were washed twice with ice-cold 1× PBS supplemented with 1× BSA (centrifugation 3,000_g_ for 5 min). The pellets were stored at −80 °C.

#### MNase digestion

In total, 1 million cells were permeabilized with 100 μl Micro-C buffer1 (50 mM NaCl, 10 mM Tris–HCl pH 7.5, 5 mM MgCl2, 1 mM CaCl2, 0.2% IGEPAL CA630, 1× protease inhibitor cocktail) and incubated on ice for 20 min. The cell pellet was resuspended in 100 μl Micro-C buffer 1, and titrated to find the appropriate amounts of MNase (NEB, M0247S) to digest chromatin to 40–50% mononucleosomes and 20–25% dinucleosomes (Supplementary Fig. [2b](https://www.nature.com/articles/s41588-025-02247-6#MOESM1)), incubated at 37 °C for 10 min. Furthermore, 0.5 M ethylene glycol-bis(β-aminoethyl ether)-N,N,N′,N′-tetraacetic acid was added to a final concentration of 4 mM to stop the reaction, omitting the heat inactivation step.

### MNase titration

To test the effect of MNase digestion efficiency on chromatin structure, we performed a series of MNase titrations (200U, 600U, 800U and 1,000U) to produce digestion degrees from low to high.

#### End repair

To compare the ligation efficiency between SDS treatment and the original protocol, for the same MNase-digested sample, we split the sample into two aliquots, one treated with SDS and the other directly subjected to end repair.

#### SDS treatment procedure

Cell pellets were resuspended in 50 μl 0.5% SDS and incubated at 62 °C for 10 min. A total of 170 μl of 1.5% Triton X-100 was added, followed by incubation at 37 °C for 15 min. The pellet was washed once with 100 μl Micro-C buffer 2 (50 mM NaCl, 10 mM Tris–HCl pH 7.5, 10 mM MgCl2). End digestion and repair was executed with the following two steps: first, resuspend the cell pellet in 45 μl end repair buffer 1 (1× NEBuffer 2.1, 2 mM adenosine triphosphate, 5 mM dithiothreitol, 2.5 μl 10 U μl−1 T4 polynucleotide kinase) and incubate at 37 °C for 15 min with gentle vortex to add 5′ phosphate and remove 3′ phosphoryl groups; next, add 5 μl 5 U μl−1 Klenow fragment (NEB, M0210S) and incubate at 37 °C for 15 min with gentle vortex to remove 3′ overhangs. Blunt end repair was performed by the addition of 25 μl of end repair buffer 2 (200 μM dNTPs each, 1× T4 ligase buffer, 100 μg ml−1 BSA), incubated at room temperature for 45 min. Wash once with 1 ml Micro-C buffer 3 (50 mM Tris–HCl pH 7.5, 10 mM MgCl2).

#### Proximity ligation

Ligation was performed by the addition of 250 μl ligation mix (1× T4 ligase buffer, 100 μg ml−1 BSA, 20 U μl−1 T4 ligase buffer) and rotating at room temperature for 2.5 h. Then the cell pellet was stained with DAPI/7-AAD and only G1-phase cells were sorted for downstream amplification.

#### Plate-based single-cell amplification

Nuclei were sorted with BD Aria SORP under single-cell mode (FACSDiva v9.0 software) to 96-well PCR plates containing 2 μl lysis buffer (10 mM Tris pH 8.0, 20 mM NaCl, 1 mM ethylenediaminetetraacetic acid, 0.1% Triton X-100, 500 nM Carrier ssDNA, 1.5 mg ml−1 QIAGEN protease), then nuclei were lysed by incubating at 50 °C for 1 h, 65 °C for 1 h and 70 °C for 15 min. After lysis, nuclei could be stored at −80 °C for several months. Lysed nuclei were first transposed by the addition of a 6 μl transposition mix (leading to a final concentration of 10 mM TAPS pH 8.5, 5 mM MgCl2, 8% polyethylene glycol 8000 and 0.3 nM META transposome dimer) and incubated at 55 °C for 10 min. META transposome was assembled as previously described[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."). Transposition was stopped by the addition of a 2 μl stop mix (250 mM NaCl, 37.5 mM ethylenediaminetetraacetic acid, 2 mg ml−1 QIAGEN protease) and incubation at 50 °C for 30 min and 70 °C for 15 min. The barcoding strategy is the same as previously described[67](https://www.nature.com/articles/s41588-025-02247-6#ref-CR67 "Wu, H. et al. Highly sensitive single-cell chromatin accessibility assay and transcriptome coassay with METATAC. Proc. Natl Acad. Sci. USA 119, e2206450119 (2022)."). Then the transposed DNA was amplified by the addition of 15 μl preamplification mix (12.5 μl 2× Q5 master mix, 0.8 μl 50 μM META16 primer mix, 0.5 μl 100 mM MgCl2, 1.2 μl H2O) and incubated at 72 °C for 5 min, 98 °C for 30 s, 12 cycles of (98 °C for 10 s, 62 °C for 30 s, 72 °C for 2 min) and 65 °C for 5 min. Next, add 0.8 μl of 50 μM indexed META16-ADP1 primer and 0.8 μl of 50 μM META16-ADP2 primer to generate a 12 × 8 cell barcode combination for each 96-well plate, and incubate at 98 °C for 30 s, three cycles of ‘98 °C for 10 s, 62 °C for 30 s, 72 °C for 2 min’ at 65 °C for 5 min. After cell barcoding, a whole plate was pooled together for purification with ZYMO DCC5.

#### Library preparation

A total of 120 ng (10 μl) of the purified amplicon was used for each plate for library preparation. Add 40 μl PCR mix (25 μl 2× Q5 Master Mix, 5 μl NEBNext index primer i5 (E7600S), 5 μl NEBNext index primer i7 (E7600S) and 0.05 μl 100 mM MgCl2) and incubate at 98 °C for 30 s, two cycles of ‘98 °C for 10 s, 68 °C for 30 s and 72 °C for 2 min’ at 72 °C for 5 min. Then purified with 0.8× solid-phase reversible immobilization beads to remove <300 bp fragments.

### Analysis of bulk Micro-C data

#### Generation of contact maps

Bulk Micro-C datasets were processed using the distiller pipeline ([https://github.com/open2c/distiller-nf](https://github.com/open2c/distiller-nf), v0.3.3). In brief, raw FASTQ files were mapped to the human reference genome assembly GRCh38 using the BWA-MEM (v0.7.17). Pairs were extracted from the mapped reads using the pairtools package ([https://github.com/open2c/pairtools](https://github.com/open2c/pairtools), v0.3.0). PCR duplicates were then filtered out, and only pairs with MAPQ > 20 were kept. Contact matrices in the.mcool and.hic format were generated and balanced using the cooler ([https://github.com/open2c/cooler](https://github.com/open2c/cooler), v.0.8.11) and Juicer package ([https://github.com/aidenlab/juicer/](https://github.com/aidenlab/juicer/), v1.19.02).

#### Evaluation of data resolution

Notably, the ‘resolution’ has distinct definitions in different contexts throughout the manuscript. The ‘resolution’ commonly denotes the bin size of a Hi-C or Micro-C contact matrix. For bulk or pseudobulk datasets, we adopted the standard criterion[40](https://www.nature.com/articles/s41588-025-02247-6#ref-CR40 "Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014)."), which defines the resolution as the bin size at which ≥80% of loci have at least 1,000 contacts, consistent with other bulk studies. Owing to the limited DNA content in individual cells, this criterion is too stringent and is not applicable to single-cell studies. As such, for single-cell data, we defined the resolution as the bin size at which ≥80% of loci have at least one contact. This definition has been shown to be reliable and robust in a previous single-cell Hi-C study[68](https://www.nature.com/articles/s41588-025-02247-6#ref-CR68 "Ulianov, S. V. et al. Order and stochasticity in the folding of individual Drosophila genomes. Nat. Commun. 12, 41 (2021).").

Moreover, the single-cell contact matrix allows for the reconstruction of 3D genome structures using a simulated annealing protocol. Contact noise and sparsity contribute to deviations between 3D genome structures generated from the same contact matrix across multiple rounds. Here, the resolution refers to the locus size used to reconstruct single-cell 3D genome structures and is determined through structure comparisons, which is fundamentally distinct from the definitions applied to contact matrices as discussed before.

#### Contact scaling curves

We used the cooltools expected-_cis_ and logbin-expected functions ([https://github.com/open2c/cooltools](https://github.com/open2c/cooltools), v0.5.2) to calculate the normalized contact probability as a function of genomic separation within chromosome arms and the scaling derivatives of the curve on contact matrices at 1-kb resolution. The contact scaling curves using unbinned short-range contacts (contact distance < 10 kb) were calculated separately for each contact orientation (IN-OUT, IN-IN and OUT-OUT) using the cooltools compute\_scaling function.

#### A/B compartments

We used the cooltools eigs-cis function to calculate A/B compartments at 500-kb and 100-kb resolution.

#### Insulation scores

The cooltools insulation function was used to calculate the insulation scores at 10-kb resolution (with a window of 100 kb). Genomic loci with boundary strength > 0.2 were considered insulation boundaries and used for downstream analyses. We used the bedtools intersect function to compare two lists of insulation boundaries, and a 10-kb offset on each side was tolerated when performing this intersection. Average profiles of insulation scores around boundaries were calculated using the deepTools computeMatrix function ([https://github.com/deeptools/deepTools](https://github.com/deeptools/deepTools), v3.5.2).

#### Chromatin loops

The chromatin loops were identified using the Juicer HiCCUPS algorithm with default parameters and the chromosight ([https://github.com/koszullab/chromosight](https://github.com/koszullab/chromosight), v1.6.2) package at 10-kb and 5-kb resolution. Chromatin loops between techniques and conditions were overlapped using the bedtools pairtopair function with ‘-slop 20000’. Pileup analyses of chromatin loops were performed using the coolpuppy package ([https://github.com/open2c/coolpuppy](https://github.com/open2c/coolpuppy), v1.0.0).

#### Chromatin stripes

We called chromatin stripes using two different algorithms, Stripenn ([https://github.com/ysora/stripenn](https://github.com/ysora/stripenn), v1.1.65.7) and StripeCaller ([https://github.com/XiaoTaoWang/StripeCaller](https://github.com/XiaoTaoWang/StripeCaller), v0.1.0), at 5-kb resolution. For StripeCaller, the chromatin stripes were extended to the main diagonal of the contact matrix, and redundant calls anchored at the same locus were merged into a single stripe with maximum length. The comparisons of chromatin stripes between different techniques and conditions were then performed based on the stripe anchor and orientation.

#### PES

After obtaining chromatin stripes, we used the Stripenn algorithm to identify promoter stripes at 10-kb and 5-kb resolution. At each resolution, we only focused on genes that are longer than ten bins. First, we performed intersections between promoters and chromatin stripe anchors and considered PES as those that anchored at the promoter. This was done by using the bedtools intersect function. Because many PES exhibited only moderate signals and Stripenn used a stringent filtering criterion, some clearly visible PES were missed. Next, we identified PES directly from the contact matrix using the Stripenn score function, which quantifies stripe signal strength, or stripiness, of any stripe-like region. For each gene, we used the promoter as the anchor point and the entire gene body as the stripe interval to calculate the stripiness. We note that the uncertainty in anchor localization has a large impact on the stripiness calculation and must be taken into account. To do so, we also took into account the two bins adjacent to the bin containing the promoter. The three bins provided a total of six possible anchors, among which three anchors were one bin wide, two anchors were two bins wide and one anchor was three bins wide, resulting in six possible regions. As recommended by the Stripenn tutorial, we considered genes with a positive stripiness to have a PES. If the gene had a positive stripiness in any of the six regions, we retained it. Here _P-_value filtering was not applied. Ultimately, we combined the PES obtained by intersections and quantifications and further combined PESs at different resolutions to generate a set of PES.

Pileup analyses for PES were performed using coolpuppy with ‘--flip-negative-strand –rescale –local’.

#### Nucleosome occupancy

The nucleosome occupancy signals were extracted from the mapped reads. Briefly, PCR duplicates were filtered out from raw SAM files using the samblaster package with ‘-ignoreUnmated-r’. The DANPOS2 package ([https://sites.google.com/site/danposdoc/](https://sites.google.com/site/danposdoc/), v2.2.2) was then used to calculate the nucleosome occupancy signals with ‘dpos-a 5 --count 1000000’. The average signals of nucleosome occupancy around genomic elements of interest were then calculated with computeMatrix and normalized by setting their mean to 1.

### Analysis of scMicro-C data

#### Generation of contact maps

Single-cell contact maps were generated from raw sequencing data as we previously described[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."), using the hickit ([https://github.com/lh3/hickit](https://github.com/lh3/hickit)) and dip-c packages ([https://github.com/tanlongzhi/dip-c](https://github.com/tanlongzhi/dip-c)). Briefly, raw sequencing data was mapped with ‘bwa mem’ and then proceeded with hickit to generate a contact pairs file with default parameters. Then the contact pairs were transformed into dip-c format for further analysis with dip-c.

#### Cell exclusion

Cells with less than 200k contacts or with >45% inter-chromosomal contacts were excluded.

#### Generation of 3D genomes

The generated contact pairs were imputed to generate a haplotype-resolved contact map using hickit. Genomic regions with chromosome abnormalities were excluded from the contact map and removed from the 3D genomes afterwards. Single-cell 3D structures were generated as we previously described[37](https://www.nature.com/articles/s41588-025-02247-6#ref-CR37 "Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."), with the hickit package (with parameters ‘-M’ and ‘Sr1m -c1 -r10m -c2 -b4m -b1m -b200k -D5 -b50k -D5 -b20k -D5 -b10k -D5 -b5k’). We generated five replicate structures for each cell, using different random seeds (1–5). Repetitive regions were also removed from the 3D structure with ‘dip-c clean3’. Similar to our previous studies, each 20 kb particle represents a radius of ~100 nm (~85 nm for each 10 kb particle and ~68 nm for each 5 kb particle).

#### Three-dimensional genome structure alignment and quality control

For each cell, the median and root mean square (r.m.s.) and r.m.s. deviation (r.m.s.d.; across all particles) were calculated with ‘dip-c align’ at 20 kb, 10 kb and 5 kb resolution. Structures with median r.m.s.d. ≤ 2 particle radii were considered as low uncertainty and kept for downstream single-cell 3D genome structure analysis.

#### Three-dimensional genome structure visualization

For whole cell structure visualization, the dip-c cleaned 3dg file was converted to an mmCIF file with ‘dip-c color -n hg38.chr.txt’ to color by chromosome, ‘dip-c color -d3’ to visualize chromosomal intermingling, ‘dip-c color -c hg38.cpg. ${resolution/1000}k.txt’ to visualize A/B compartment (CpG frequency).

To visualize a region of interest, the region was extracted from the whole cell by ‘dip-c reg3’ to generate 3dg file containing that region, then transformed to an mmCIF file with ‘dip-c color -l hg38.chr.len’. The loop anchor file was generated by ‘dip-c pos -l ’leg file’’ to extract loop anchor 3dg positions from the whole cell structure, then transformed to mmCIF files by ‘paste ‘anchor name file’ ‘anchor color file’ ‘anchor 3dg file’ | python /dip-c/scripts/name\_color\_x\_y\_z\_to\_cif.py’.

The mmCIF file was visualized in PyMol ([https://pymol.org/2/](https://pymol.org/2/)).

#### Distance matrix analysis

For a region of interest, we performed two filtering criteria to select available single-cell 3D structures. First, we require the whole-cell 3D genome structure with median r.m.s.d. ≤ 2. Second, we require the 3D genome structure of that region with r.m.s.d. ≤ 1.5.

For single-cell distance matrix analysis, we extracted the region of interest from the whole cell 3D genome file with ‘dip-c reg3’ and then measured the distance matrix with ‘dip-c rg -d’.

#### Polymer simulations of loop extrusion

We performed polymer simulations of loop extrusion with the ‘open2c/polychrome’ package[69](https://www.nature.com/articles/s41588-025-02247-6#ref-CR69 "Imakaev, M., Goloborodko, A. & hbbrandao. mirnylab/polychrom: v0.1.0 (v0.1.0). Zenodo 
https://doi.org/10.5281/zenodo.3579473
(2019).") and the adapted version developed by the Boettiger laboratory[46](https://www.nature.com/articles/s41588-025-02247-6#ref-CR46 "Chen, L.-F. et al. Structural elements promote architectural stripe formation and facilitate ultra-long-range gene regulation at a human disease locus. Mol. Cell 83, 1446–1461.e6 (2023)."),[70](https://www.nature.com/articles/s41588-025-02247-6#ref-CR70 "Imakaev, M., Goloborodko, A., Boettiger, A., Chowdhury, N. & Brandão, H. B. BoettigerLab/polychrom: polymer simulations (v0.0.0). Zenodo 
https://doi.org/10.5281/zenodo.7698987
(2023)."). The chromatin polymer chain consists of 1,200 monomers, with each monomer representing 500 bp. To match the chromatin structure of the _EBF1_ locus, we positioned the promoter and TTS at monomer 1,000 and 190, respectively, and positioned 7 enhancers on the gene body at monomer 440, 560, 600, 710, 720, 820 and 900. Loop extrusion factors (LEFs, cohesin) were primarily loaded at the promoter with a probability of 90% (with parameters ‘LFE lifetime = 200, LEF separation = 200, trajectory length = 5,000’). For simulations without loop extrusion, the number of LEFs was set to 0 as previously described[71](https://www.nature.com/articles/s41588-025-02247-6#ref-CR71 "Hung, T.-C., Kingsley, D. M. & Boettiger, A. N. Boundary stacking interactions enable cross-TAD enhancer–promoter communication during limb development. Nat. Genet. 56, 306–314 (2024)."). We positioned a bidirectional extrusion-blocking element adjacent to the promoter at monomer 1,001 with a capture probability of 99%, which produces the prominent stripe pattern. To investigate the effect of loop extrusion on E–E and E–P interactions, we allowed enhancers to weakly block extrusion from the left with a capture probability of 30% or 60%. The stripe emanating from the TTS was simulated by adding the extrusion-blocking property to the TTS monomer with the same capture probability as enhancers. In all simulations, a release rate of 0.1% was used for all extrusion-blocking elements. To study the effect of E–P adhesion, we added a self-affinity of 0.1 _KT_ to all monomers and set the affinity between enhancers and promoters as 0.5 or 0.8 _KT_, respectively, where _K_ is the Boltzmann constant and _T_ is the temperature. The energy functions and the related parameters were unchanged from the previous work mentioned above. Ensemble contact maps were then generated from polymer traces using the polychrom ‘monomerResolutionContactMapSubchains’ function with a threshold of 5 monomer radii. For the analysis of individual polymer traces, we extracted coordinates using the polychrom ‘polymerutils.fetch\_block’ function and then calculated distance matrices. The PDB file for each trace was generated using the polychrom ‘polymerutils.save’ function and visualized in PyMol.

### Statistics and reproducibility

A total of 810 cells from five batches were generated in this study, and 52 cells were excluded due to poor quality (‘Cell exclusion’). For bulk Micro-C, two replicates were generated. Randomization and blinding were not applicable to this study. Specific statistics used for each analysis are referred to in the [Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10). All aspects of benchmarking with Dip-C are described in detail in the [Methods](https://www.nature.com/articles/s41588-025-02247-6#Sec10).

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02247-6#MOESM2) linked to this article.

Data availability
-----------------

Raw sequencing data generated during this study have been deposited at the Sequence Read Archive with accession number PRJNA788160. The processed data generated in this study have been uploaded to the Gene Expression Omnibus under accession number Superseries GSE281150. The subseries includes GSE225201 for bulk Micro-C, GSE278191 for scMicro-C 1000U, GSE279479 for scMicro-C 600U, GSE279583 for scMicro-C 800U, GSE279732 for scMicro-C 200U, GSE280482 for SDS test, GSE192759 for scMicro-C pool. [Source data](https://www.nature.com/articles/s41588-025-02247-6#Sec44) are provided with this paper.

Code availability
-----------------

The code used in this study is publicly available at GitHub ([https://github.com/tanlongzhi/dip-c](https://github.com/tanlongzhi/dip-c), [https://github.com/lh3/hickit](https://github.com/lh3/hickit) and [https://github.com/zhang-jiankun/Single-cell-Micro-C](https://github.com/zhang-jiankun/Single-cell-Micro-C))[72](https://www.nature.com/articles/s41588-025-02247-6#ref-CR72 "Tan, L. tanlongzhi/dip-c. GitHub 
https://github.com/tanlongzhi/dip-c
(2018)."),[73](https://www.nature.com/articles/s41588-025-02247-6#ref-CR73 "Li, H. & Tan, L. lh3/hickit. GitHub 
https://github.com/lh3/hickit
(2018)."),[74](https://www.nature.com/articles/s41588-025-02247-6#ref-CR74 "Zhang, J. zhang-jiankun/single-cell-Micro-C. GitHub 
https://github.com/zhang-jiankun/Single-cell-Micro-C
(2025).").

References
----------

1.  Rowley, M. J. & Corces, V. G. Organizational principles of 3D genome architecture. _Nat. Rev. Genet._ **19**, 789–800 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVCit7rO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30367165)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Organizational%20principles%20of%203D%20genome%20architecture&journal=Nat.%20Rev.%20Genet.&volume=19&pages=789-800&publication_year=2018&author=Rowley%2CMJ&author=Corces%2CVG) 
    
2.  Bonev, B. & Cavalli, G. Organization and function of the 3D genome. _Nat. Rev. Genet._ **17**, 772 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhslOqtr%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28704353)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Organization%20and%20function%20of%20the%203D%20genome&journal=Nat.%20Rev.%20Genet.&volume=17&publication_year=2016&author=Bonev%2CB&author=Cavalli%2CG) 
    
3.  Misteli, T. The self-organizing genome: principles of genome architecture and function. _Cell_ **183**, 28–45 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFGqsL%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32976797)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7541718)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20self-organizing%20genome%3A%20principles%20of%20genome%20architecture%20and%20function&journal=Cell&volume=183&pages=28-45&publication_year=2020&author=Misteli%2CT) 
    
4.  Oudelaar, A. M. & Higgs, D. R. The relationship between genome structure and function. _Nat. Rev. Genet._ **22**, 154–168 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVeqsr3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33235358)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20relationship%20between%20genome%20structure%20and%20function&journal=Nat.%20Rev.%20Genet.&volume=22&pages=154-168&publication_year=2021&author=Oudelaar%2CAM&author=Higgs%2CDR) 
    
5.  Cramer, P. Organization and regulation of gene transcription. _Nature_ **573**, 45–54 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Oht7nO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31462772)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Organization%20and%20regulation%20of%20gene%20transcription&journal=Nature&volume=573&pages=45-54&publication_year=2019&author=Cramer%2CP) 
    
6.  Lettice, L. A. et al. A long-range Shh enhancer regulates expression in the developing limb and fin and is associated with preaxial polydactyly. _Hum. Mol. Genet._ **12**, 1725–1735 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXltFygsrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12837695)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20long-range%20Shh%20enhancer%20regulates%20expression%20in%20the%20developing%20limb%20and%20fin%20and%20is%20associated%20with%20preaxial%20polydactyly&journal=Hum.%20Mol.%20Genet.&volume=12&pages=1725-1735&publication_year=2003&author=Lettice%2CLA) 
    
7.  Blackwood, E. M. & Kadonaga, J. T. Going the distance: a current view of enhancer action. _Science_ **281**, 60–63 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXksVOhu7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9679020)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Going%20the%20distance%3A%20a%20current%20view%20of%20enhancer%20action&journal=Science&volume=281&pages=60-63&publication_year=1998&author=Blackwood%2CEM&author=Kadonaga%2CJT) 
    
8.  Bulger, M. & Groudine, M. Looping versus linking: toward a model for long-distance gene activation. _Genes Dev._ **13**, 2465–2477 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmvFOjurs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10521391)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Looping%20versus%20linking%3A%20toward%20a%20model%20for%20long-distance%20gene%20activation&journal=Genes%20Dev.&volume=13&pages=2465-2477&publication_year=1999&author=Bulger%2CM&author=Groudine%2CM) 
    
9.  Tsai, A., Alves, M. R. & Crocker, J. Multi-enhancer transcriptional hubs confer phenotypic robustness. _eLife_ **8**, e45325 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31294690)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6650246)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-enhancer%20transcriptional%20hubs%20confer%20phenotypic%20robustness&journal=eLife&volume=8&publication_year=2019&author=Tsai%2CA&author=Alves%2CMR&author=Crocker%2CJ) 
    
10.  Chandra, A. et al. Quantitative control of Ets1 dosage by a multi-enhancer hub promotes Th1 cell differentiation and protects from allergic inflammation. _Immunity_ **56**, 1451–1467 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtFamu7zK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37263273)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10979463)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantitative%20control%20of%20Ets1%20dosage%20by%20a%20multi-enhancer%20hub%20promotes%20Th1%20cell%20differentiation%20and%20protects%20from%20allergic%20inflammation&journal=Immunity&volume=56&pages=1451-1467&publication_year=2023&author=Chandra%2CA) 
    
11.  Uyehara, C. M. & Apostolou, E. 3D enhancer-promoter interactions and multi-connected hubs: organizational principles and functional roles. _Cell Rep._ **42**, 112068 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXnslCksLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37059094)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10556201)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=3D%20enhancer-promoter%20interactions%20and%20multi-connected%20hubs%3A%20organizational%20principles%20and%20functional%20roles&journal=Cell%20Rep.&volume=42&publication_year=2023&author=Uyehara%2CCM&author=Apostolou%2CE) 
    
12.  Allahyar, A. et al. Enhancer hubs and loop collisions identified from single-allele topologies. _Nat. Genet._ **50**, 1151–1160 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSqsr7N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29988121)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhancer%20hubs%20and%20loop%20collisions%20identified%20from%20single-allele%20topologies&journal=Nat.%20Genet.&volume=50&pages=1151-1160&publication_year=2018&author=Allahyar%2CA) 
    
13.  Madsen, J. G. S. et al. Highly interconnected enhancer communities control lineage-determining genes in human mesenchymal stem cells. _Nat. Genet._ **52**, 1227–1238 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2jtLbK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33020665)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Highly%20interconnected%20enhancer%20communities%20control%20lineage-determining%20genes%20in%20human%20mesenchymal%20stem%20cells&journal=Nat.%20Genet.&volume=52&pages=1227-1238&publication_year=2020&author=Madsen%2CJGS) 
    
14.  Li, W. et al. scNanoHi-C: a single-cell long-read concatemer sequencing method to reveal high-order chromatin structures within individual cells. _Nat. Methods_ **20**, 1493–1505 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslyhu7jK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37640936)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=scNanoHi-C%3A%20a%20single-cell%20long-read%20concatemer%20sequencing%20method%20to%20reveal%20high-order%20chromatin%20structures%20within%20individual%20cells&journal=Nat.%20Methods&volume=20&pages=1493-1505&publication_year=2023&author=Li%2CW) 
    
15.  Deshpande, A. S. et al. Identifying synergistic high-order 3D chromatin conformations from genome-scale nanopore concatemer sequencing. _Nat. Biotechnol._ **40**, 1488–1499 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVWjsLnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35637420)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identifying%20synergistic%20high-order%203D%20chromatin%20conformations%20from%20genome-scale%20nanopore%20concatemer%20sequencing&journal=Nat.%20Biotechnol.&volume=40&pages=1488-1499&publication_year=2022&author=Deshpande%2CAS) 
    
16.  Markenscoff-Papadimitriou, E. et al. Enhancer interaction networks as a means for singular olfactory receptor expression. _Cell_ **159**, 543–557 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVanur%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25417106)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4243057)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhancer%20interaction%20networks%20as%20a%20means%20for%20singular%20olfactory%20receptor%20expression&journal=Cell&volume=159&pages=543-557&publication_year=2014&author=Markenscoff-Papadimitriou%2CE) 
    
17.  Quinodoz, S. A. et al. Higher-order inter-chromosomal hubs shape 3D genome organization in the nucleus. _Cell_ **174**, 744–757 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltrrJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29887377)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6548320)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Higher-order%20inter-chromosomal%20hubs%20shape%203D%20genome%20organization%20in%20the%20nucleus&journal=Cell&volume=174&pages=744-757&publication_year=2018&author=Quinodoz%2CSA) 
    
18.  Alipour, E. & Marko, J. F. Self-organization of domain structures by DNA-loop-extruding enzymes. _Nucleic Acids Res._ **40**, 11202–11212 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCiur%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23074191)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526278)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Self-organization%20of%20domain%20structures%20by%20DNA-loop-extruding%20enzymes&journal=Nucleic%20Acids%20Res.&volume=40&pages=11202-11212&publication_year=2012&author=Alipour%2CE&author=Marko%2CJF) 
    
19.  Sanborn, A. L. et al. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. _Proc. Natl Acad. Sci. USA_ **112**, E6456 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslWks7%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26499245)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4664323)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20extrusion%20explains%20key%20features%20of%20loop%20and%20domain%20formation%20in%20wild-type%20and%20engineered%20genomes&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=112&publication_year=2015&author=Sanborn%2CAL) 
    
20.  Fudenberg, G. et al. Formation of chromosomal domains by loop extrusion. _Cell Rep._ **15**, 2038–2049 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XotlentLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27210764)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4889513)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Formation%20of%20chromosomal%20domains%20by%20loop%20extrusion&journal=Cell%20Rep.&volume=15&pages=2038-2049&publication_year=2016&author=Fudenberg%2CG) 
    
21.  Nasmyth, K. Disseminating the genome: joining, resolving, and separating sister chromatids during mitosis and meiosis. _Annu. Rev. Genet._ **35**, 673–745 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XlsVOr)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11700297)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disseminating%20the%20genome%3A%20joining%2C%20resolving%2C%20and%20separating%20sister%20chromatids%20during%20mitosis%20and%20meiosis&journal=Annu.%20Rev.%20Genet.&volume=35&pages=673-745&publication_year=2001&author=Nasmyth%2CK) 
    
22.  Davidson, I. F. et al. DNA loop extrusion by human cohesin. _Science_ **366**, 1338–1345 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSnsLrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31753851)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20loop%20extrusion%20by%20human%20cohesin&journal=Science&volume=366&pages=1338-1345&publication_year=2019&author=Davidson%2CIF) 
    
23.  Kim, Y., Shi, Z., Zhang, H., Finkelstein Ilya, J. & Yu, H. Human cohesin compacts DNA by loop extrusion. _Science_ **366**, 1345–1349 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSnsLzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31780627)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7387118)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20cohesin%20compacts%20DNA%20by%20loop%20extrusion&journal=Science&volume=366&pages=1345-1349&publication_year=2019&author=Kim%2CY&author=Shi%2CZ&author=Zhang%2CH&author=Finkelstein%20Ilya%2CJ&author=Yu%2CH) 
    
24.  Davidson, I. F. & Peters, J.-M. Genome folding through loop extrusion by SMC complexes. _Nat. Rev. Mol. Cell Biol._ **22**, 445–464 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsFKjs78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33767413)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20folding%20through%20loop%20extrusion%20by%20SMC%20complexes&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=22&pages=445-464&publication_year=2021&author=Davidson%2CIF&author=Peters%2CJ-M) 
    
25.  Karpinska, M. A. & Oudelaar, A. M. The role of loop extrusion in enhancer-mediated gene activation. _Curr. Opin. Genet. Dev._ **79**, 102022 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjsVOns7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36842325)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20loop%20extrusion%20in%20enhancer-mediated%20gene%20activation&journal=Curr.%20Opin.%20Genet.%20Dev.&volume=79&publication_year=2023&author=Karpinska%2CMA&author=Oudelaar%2CAM) 
    
26.  Calderon, L. et al. Cohesin-dependence of neuronal gene expression relates to chromatin loop length. _eLife_ **11**, e76539 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFKmsLrE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35471149)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9106336)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cohesin-dependence%20of%20neuronal%20gene%20expression%20relates%20to%20chromatin%20loop%20length&journal=eLife&volume=11&publication_year=2022&author=Calderon%2CL) 
    
27.  Kane, L. et al. Cohesin is required for long-range enhancer action at the Shh locus. _Nat. Struct. Mol. Biol._ **29**, 891–897 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlOltb%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36097291)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7613721)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cohesin%20is%20required%20for%20long-range%20enhancer%20action%20at%20the%20Shh%20locus&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=29&pages=891-897&publication_year=2022&author=Kane%2CL) 
    
28.  Zhang, X. et al. Fundamental roles of chromatin loop extrusion in antibody class switching. _Nature_ **575**, 385–389 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2hsbzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31666703)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6856444)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fundamental%20roles%20of%20chromatin%20loop%20extrusion%20in%20antibody%20class%20switching&journal=Nature&volume=575&pages=385-389&publication_year=2019&author=Zhang%2CX) 
    
29.  Zhang, Y. et al. The fundamental role of chromatin loop extrusion in physiological V(D)J recombination. _Nature_ **573**, 600–604 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslKhsrzI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31511698)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6867615)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20fundamental%20role%20of%20chromatin%20loop%20extrusion%20in%20physiological%20V%28D%29J%20recombination&journal=Nature&volume=573&pages=600-604&publication_year=2019&author=Zhang%2CY) 
    
30.  Kiefer, L. et al. WAPL functions as a rheostat of protocadherin isoform diversity that controls neural wiring. _Science_ **380**, eadf8440 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlSisL3J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37347873)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WAPL%20functions%20as%20a%20rheostat%20of%20protocadherin%20isoform%20diversity%20that%20controls%20neural%20wiring&journal=Science&volume=380&publication_year=2023&author=Kiefer%2CL) 
    
31.  Chang, L.-H. et al. Multi-feature clustering of CTCF binding creates robustness for loop extrusion blocking and topologically associating domain boundaries. _Nat. Commun._ **14**, 5615 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvFWgtbzP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37699887)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10497529)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-feature%20clustering%20of%20CTCF%20binding%20creates%20robustness%20for%20loop%20extrusion%20blocking%20and%20topologically%20associating%20domain%20boundaries&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Chang%2CL-H) 
    
32.  Dekker, J., Rippe, K., Dekker, M. & Kleckner, N. Capturing chromosome conformation. _Science_ **295**, 1306–1311 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XhsVGhsbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11847345)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Capturing%20chromosome%20conformation&journal=Science&volume=295&pages=1306-1311&publication_year=2002&author=Dekker%2CJ&author=Rippe%2CK&author=Dekker%2CM&author=Kleckner%2CN) 
    
33.  Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. _Science_ **326**, 289–293 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1agsbjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19815776)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2858594)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&journal=Science&volume=326&pages=289-293&publication_year=2009&author=Lieberman-Aiden%2CE) 
    
34.  Hsieh, T. S. et al. Resolving the 3D landscape of transcription-linked mammalian chromatin folding. _Mol. Cell_ **78**, 539–553 e8 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslKhtr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32213323)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7703524)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20the%203D%20landscape%20of%20transcription-linked%20mammalian%20chromatin%20folding&journal=Mol.%20Cell&volume=78&pages=539-553%20e8&publication_year=2020&author=Hsieh%2CTS) 
    
35.  Krietenstein, N. et al. Ultrastructural details of mammalian chromosome architecture. _Mol. Cell_ **78**, 554–565 e7 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslKhtrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32213324)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7222625)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ultrastructural%20details%20of%20mammalian%20chromosome%20architecture&journal=Mol.%20Cell&volume=78&pages=554-565%20e7&publication_year=2020&author=Krietenstein%2CN) 
    
36.  Hsieh, T. H. et al. Mapping nucleosome resolution chromosome folding in yeast by Micro-C. _Cell_ **162**, 108–119 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtV2itbrN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26119342)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4509605)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20nucleosome%20resolution%20chromosome%20folding%20in%20yeast%20by%20Micro-C&journal=Cell&volume=162&pages=108-119&publication_year=2015&author=Hsieh%2CTH) 
    
37.  Tan, L., Xing, D., Chang, C. H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. _Science_ **361**, 924–928 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2mt7fK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30166492)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6360088)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Three-dimensional%20genome%20structures%20of%20single%20diploid%20human%20cells&journal=Science&volume=361&pages=924-928&publication_year=2018&author=Tan%2CL&author=Xing%2CD&author=Chang%2CCH&author=Li%2CH&author=Xie%2CXS) 
    
38.  Belton, J.-M. et al. Hi-C: a comprehensive technique to capture the conformation of genomes. _Methods_ **58**, 268–276 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVyksbjO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22652625)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hi-C%3A%20a%20comprehensive%20technique%20to%20capture%20the%20conformation%20of%20genomes&journal=Methods&volume=58&pages=268-276&publication_year=2012&author=Belton%2CJ-M) 
    
39.  Hua, P. et al. Defining genome architecture at base-pair resolution. _Nature_ **595**, 125–129 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1yhsL3N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34108683)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defining%20genome%20architecture%20at%20base-pair%20resolution&journal=Nature&volume=595&pages=125-129&publication_year=2021&author=Hua%2CP) 
    
40.  Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. _Cell_ **159**, 1665–1680 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtrrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25497547)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%203D%20map%20of%20the%20human%20genome%20at%20kilobase%20resolution%20reveals%20principles%20of%20chromatin%20looping&journal=Cell&volume=159&pages=1665-1680&publication_year=2014&author=Rao%2CSS) 
    
41.  Nagano, T. et al. Single-cell Hi-C reveals cell-to-cell variability in chromosome structure. _Nature_ **502**, 59–64 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFaksbvN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24067610)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20Hi-C%20reveals%20cell-to-cell%20variability%20in%20chromosome%20structure&journal=Nature&volume=502&pages=59-64&publication_year=2013&author=Nagano%2CT) 
    
42.  Stevens, T. J. et al. 3D structures of individual mammalian genomes studied by single-cell Hi-C. _Nature_ **544**, 59–64 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVWntb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28289288)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5385134)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=3D%20structures%20of%20individual%20mammalian%20genomes%20studied%20by%20single-cell%20Hi-C&journal=Nature&volume=544&pages=59-64&publication_year=2017&author=Stevens%2CTJ) 
    
43.  Tan, L. et al. Changes in genome architecture and transcriptional dynamics progress independently of sensory experience during post-natal brain development. _Cell_ **184**, 741–758.e17 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aks7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33484631)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Changes%20in%20genome%20architecture%20and%20transcriptional%20dynamics%20progress%20independently%20of%20sensory%20experience%20during%20post-natal%20brain%20development&journal=Cell&volume=184&pages=741-758.e17&publication_year=2021&author=Tan%2CL) 
    
44.  Li, Y. et al. The structural basis for cohesin–CTCF-anchored loops. _Nature_ **578**, 472–476 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXislCnsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31905366)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035113)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20structural%20basis%20for%20cohesin%E2%80%93CTCF-anchored%20loops&journal=Nature&volume=578&pages=472-476&publication_year=2020&author=Li%2CY) 
    
45.  Cheng, N. et al. STAG2 promotes the myelination transcriptional program in oligodendrocytes. _eLife_ **11**, e77848 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXlvFymtr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35959892)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9439679)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=STAG2%20promotes%20the%20myelination%20transcriptional%20program%20in%20oligodendrocytes&journal=eLife&volume=11&publication_year=2022&author=Cheng%2CN) 
    
46.  Chen, L.-F. et al. Structural elements promote architectural stripe formation and facilitate ultra-long-range gene regulation at a human disease locus. _Mol. Cell_ **83**, 1446–1461.e6 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmsVCgtbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36996812)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20elements%20promote%20architectural%20stripe%20formation%20and%20facilitate%20ultra-long-range%20gene%20regulation%20at%20a%20human%20disease%20locus&journal=Mol.%20Cell&volume=83&pages=1446-1461.e6&publication_year=2023&author=Chen%2CL-F) 
    
47.  Vian, L. et al. The energetics and physiological impact of cohesin extrusion. _Cell_ **173**, 1165–1178 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXosFWhsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29706548)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6065110)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20energetics%20and%20physiological%20impact%20of%20cohesin%20extrusion&journal=Cell&volume=173&pages=1165-1178&publication_year=2018&author=Vian%2CL) 
    
48.  Fulco, C. P. et al. Activity-by-contact model of enhancer–promoter regulation from thousands of CRISPR perturbations. _Nat. Genet._ **51**, 1664–1669 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1Ohs77M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31784727)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6886585)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Activity-by-contact%20model%20of%20enhancer%E2%80%93promoter%20regulation%20from%20thousands%20of%20CRISPR%20perturbations&journal=Nat.%20Genet.&volume=51&pages=1664-1669&publication_year=2019&author=Fulco%2CCP) 
    
49.  Hsieh, T.-H. S. et al. Enhancer–promoter interactions and transcription are largely maintained upon acute loss of CTCF, cohesin, WAPL or YY1. _Nat. Genet._ **54**, 1919–1932 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVCjtb3I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36471071)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9729117)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhancer%E2%80%93promoter%20interactions%20and%20transcription%20are%20largely%20maintained%20upon%20acute%20loss%20of%20CTCF%2C%20cohesin%2C%20WAPL%20or%20YY1&journal=Nat.%20Genet.&volume=54&pages=1919-1932&publication_year=2022&author=Hsieh%2CT-HS) 
    
50.  Rothenberg, E. V. Transcriptional control of early T and B cell developmental choices. _Annu. Rev. Immunol._ **32**, 283–321 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXovVemt7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24471430)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3994230)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptional%20control%20of%20early%20T%20and%20B%20cell%20developmental%20choices&journal=Annu.%20Rev.%20Immunol.&volume=32&pages=283-321&publication_year=2014&author=Rothenberg%2CEV) 
    
51.  Gabriele, M. et al. Dynamics of CTCF- and cohesin-mediated chromatin looping revealed by live-cell imaging. _Science_ **376**, 496–501 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtF2nu77I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35420890)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9069445)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20CTCF-%20and%20cohesin-mediated%20chromatin%20looping%20revealed%20by%20live-cell%20imaging&journal=Science&volume=376&pages=496-501&publication_year=2022&author=Gabriele%2CM) 
    
52.  Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. _Nat. Genet._ **54**, 1907–1918 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVCjtb3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36471076)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9729113)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cohesin%20and%20CTCF%20control%20the%20dynamics%20of%20chromosome%20folding&journal=Nat.%20Genet.&volume=54&pages=1907-1918&publication_year=2022&author=Mach%2CP) 
    
53.  Oudelaar, A. M. et al. Single-allele chromatin interactions identify regulatory hubs in dynamic compartmentalized domains. _Nat. Genet._ **50**, 1744–1751 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVCitL%2FE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30374068)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6265079)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-allele%20chromatin%20interactions%20identify%20regulatory%20hubs%20in%20dynamic%20compartmentalized%20domains&journal=Nat.%20Genet.&volume=50&pages=1744-1751&publication_year=2018&author=Oudelaar%2CAM) 
    
54.  Monahan, K., Horta, A. & Lomvardas, S. LHX2- and LDB1-mediated trans interactions regulate olfactory receptor choice. _Nature_ **565**, 448–453 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFShsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30626972)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6436840)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LHX2-%20and%20LDB1-mediated%20trans%20interactions%20regulate%20olfactory%20receptor%20choice&journal=Nature&volume=565&pages=448-453&publication_year=2019&author=Monahan%2CK&author=Horta%2CA&author=Lomvardas%2CS) 
    
55.  Guo, Y. et al. Chromatin jets define the properties of cohesin-driven in vivo loop extrusion. _Mol. Cell_ **82**, 3769–3780.e5 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFejtr%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36182691)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20jets%20define%20the%20properties%20of%20cohesin-driven%20in%20vivo%20loop%20extrusion&journal=Mol.%20Cell&volume=82&pages=3769-3780.e5&publication_year=2022&author=Guo%2CY) 
    
56.  Isiaka, B. N. et al. Cohesin forms fountains at active enhancers in C. elegans. Preprint at _bioRxiv_ [https://doi.org/10.1101/2023.07.14.549011](https://doi.org/10.1101/2023.07.14.549011) (2023).
    
57.  Cho, W.-K. et al. Mediator and RNA polymerase II clusters associate in transcription-dependent condensates. _Science_ **361**, 412–415 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlyitrrJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29930094)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6543815)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mediator%20and%20RNA%20polymerase%20II%20clusters%20associate%20in%20transcription-dependent%20condensates&journal=Science&volume=361&pages=412-415&publication_year=2018&author=Cho%2CW-K) 
    
58.  Boija, A. et al. Transcription factors activate genes through the phase-separation capacity of their activation domains. _Cell_ **175**, 1842–1855 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1Wlt7%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30449618)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20factors%20activate%20genes%20through%20the%20phase-separation%20capacity%20of%20their%20activation%20domains&journal=Cell&volume=175&pages=1842-1855&publication_year=2018&author=Boija%2CA) 
    
59.  Chong, S. et al. Imaging dynamic and selective low-complexity domain interactions that control gene transcription. _Science_ **361**, eaar2555 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29930090)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6961784)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Imaging%20dynamic%20and%20selective%20low-complexity%20domain%20interactions%20that%20control%20gene%20transcription&journal=Science&volume=361&publication_year=2018&author=Chong%2CS) 
    
60.  Sabari, B. R. et al. Coactivator condensation at super-enhancers links phase separation and gene control. _Science_ **361**, eaar3958 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29930091)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6092193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coactivator%20condensation%20at%20super-enhancers%20links%20phase%20separation%20and%20gene%20control&journal=Science&volume=361&publication_year=2018&author=Sabari%2CBR) 
    
61.  Monahan, K. et al. Cooperative interactions enable singular olfactory receptor expression in mouse olfactory neurons. _eLife_ **6**, e28620 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28933695)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5608512)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cooperative%20interactions%20enable%20singular%20olfactory%20receptor%20expression%20in%20mouse%20olfactory%20neurons&journal=eLife&volume=6&publication_year=2017&author=Monahan%2CK) 
    
62.  Matthews, J. M. & Visvader, J. E. LIM‐domain‐binding protein 1: a multifunctional cofactor that interacts with diverse proteins. _EMBO Rep._ **4**, 1132–1137 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXptlGksb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14647207)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1326422)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LIM%E2%80%90domain%E2%80%90binding%20protein%201%3A%20a%20multifunctional%20cofactor%20that%20interacts%20with%20diverse%20proteins&journal=EMBO%20Rep.&volume=4&pages=1132-1137&publication_year=2003&author=Matthews%2CJM&author=Visvader%2CJE) 
    
63.  Song, S.-H., Hou, C. & Dean, A. A positive role for NLI/Ldb1 in long-range β-globin locus control region function. _Mol. Cell_ **28**, 810–822 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXktVWhsg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18082606)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2195932)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20positive%20role%20for%20NLI%2FLdb1%20in%20long-range%20%CE%B2-globin%20locus%20control%20region%20function&journal=Mol.%20Cell&volume=28&pages=810-822&publication_year=2007&author=Song%2CS-H&author=Hou%2CC&author=Dean%2CA) 
    
64.  Kadrmas, J. L. & Beckerle, M. C. The LIM domain: from the cytoskeleton to the nucleus. _Nat. Rev. Mol. Cell Biol._ **5**, 920–931 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXptFemsLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15520811)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20LIM%20domain%3A%20from%20the%20cytoskeleton%20to%20the%20nucleus&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=5&pages=920-931&publication_year=2004&author=Kadrmas%2CJL&author=Beckerle%2CMC) 
    
65.  Aboreden, N. G. et al. LDB1 establishes multi-enhancer networks to regulate gene expression. _Mol. Cell_ **85**, 376–393 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXivVCgtL7N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39721581)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LDB1%20establishes%20multi-enhancer%20networks%20to%20regulate%20gene%20expression&journal=Mol.%20Cell&volume=85&pages=376-393&publication_year=2025&author=Aboreden%2CNG) 
    
66.  Du, M. et al. Direct observation of a condensate effect on super-enhancer controlled gene bursting. _Cell_ **187**, 331–344 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXmvVehsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38194964)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Direct%20observation%20of%20a%20condensate%20effect%20on%20super-enhancer%20controlled%20gene%20bursting&journal=Cell&volume=187&pages=331-344&publication_year=2024&author=Du%2CM) 
    
67.  Wu, H. et al. Highly sensitive single-cell chromatin accessibility assay and transcriptome coassay with METATAC. _Proc. Natl Acad. Sci. USA_ **119**, e2206450119 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xislaqt7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36161934)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9546615)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Highly%20sensitive%20single-cell%20chromatin%20accessibility%20assay%20and%20transcriptome%20coassay%20with%20METATAC&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=119&publication_year=2022&author=Wu%2CH) 
    
68.  Ulianov, S. V. et al. Order and stochasticity in the folding of individual _Drosophila_ genomes. _Nat. Commun._ **12**, 41 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsVKrtA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33397980)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7782554)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Order%20and%20stochasticity%20in%20the%20folding%20of%20individual%20Drosophila%20genomes&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Ulianov%2CSV) 
    
69.  Imakaev, M., Goloborodko, A. & hbbrandao. mirnylab/polychrom: v0.1.0 (v0.1.0). _Zenodo_ [https://doi.org/10.5281/zenodo.3579473](https://doi.org/10.5281/zenodo.3579473) (2019).
    
70.  Imakaev, M., Goloborodko, A., Boettiger, A., Chowdhury, N. & Brandão, H. B. BoettigerLab/polychrom: polymer simulations (v0.0.0). _Zenodo_ [https://doi.org/10.5281/zenodo.7698987](https://doi.org/10.5281/zenodo.7698987) (2023).
    
71.  Hung, T.-C., Kingsley, D. M. & Boettiger, A. N. Boundary stacking interactions enable cross-TAD enhancer–promoter communication during limb development. _Nat. Genet._ **56**, 306–314 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhvFantr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38238628)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Boundary%20stacking%20interactions%20enable%20cross-TAD%20enhancer%E2%80%93promoter%20communication%20during%20limb%20development&journal=Nat.%20Genet.&volume=56&pages=306-314&publication_year=2024&author=Hung%2CT-C&author=Kingsley%2CDM&author=Boettiger%2CAN) 
    
72.  Tan, L. tanlongzhi/dip-c. _GitHub_ [https://github.com/tanlongzhi/dip-c](https://github.com/tanlongzhi/dip-c) (2018).
    
73.  Li, H. & Tan, L. lh3/hickit. _GitHub_ [https://github.com/lh3/hickit](https://github.com/lh3/hickit) (2018).
    
74.  Zhang, J. zhang-jiankun/single-cell-Micro-C. _GitHub_ [https://github.com/zhang-jiankun/Single-cell-Micro-C](https://github.com/zhang-jiankun/Single-cell-Micro-C) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02247-6?format=refman&flavour=references)

Acknowledgements
----------------

We thank the staff at the Peking University High-throughput Sequencing Center (HTSC) for help with next-generation sequencing and flow cytometry; Beijing Berry Genomics for help with next-generation sequencing; and we thank X. Ji (School of Life Sciences) and X. Zhang (Biomedical Pioneering Innovation Center) at Peking University for interactions. This work was financially supported by Changping Laboratory and the Noncommunicable Chronic Diseases-National Science and Technology Major Project (2023ZD0520000).

Author information
------------------

Author notes

1.  Honggui Wu
    
    Present address: Broad Institute of MIT and Harvard, Cambridge, MA, USA
    
2.  These authors contributed equally: Honggui Wu, Jiankun Zhang, Longzhi Tan.
    

### Authors and Affiliations

1.  Biomedical Pioneering Innovation Center (BIOPIC) and School of Life Sciences, Peking University, Beijing, China
    
    Honggui Wu, Jiankun Zhang & Xiaoliang Sunney Xie
    
2.  Changping Laboratory, Beijing, China
    
    Honggui Wu, Jiankun Zhang & Xiaoliang Sunney Xie
    
3.  Department of Neurobiology, Stanford University, Stanford, CA, USA
    
    Longzhi Tan
    

Authors

1.  Honggui Wu
    
    [View author publications](https://www.nature.com/search?author=Honggui%20Wu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Honggui%20Wu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Honggui%20Wu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Jiankun Zhang
    
    [View author publications](https://www.nature.com/search?author=Jiankun%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jiankun%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jiankun%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Longzhi Tan
    
    [View author publications](https://www.nature.com/search?author=Longzhi%20Tan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Longzhi%20Tan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Longzhi%20Tan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Xiaoliang Sunney Xie
    
    [View author publications](https://www.nature.com/search?author=Xiaoliang%20Sunney%20Xie)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiaoliang%20Sunney%20Xie) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiaoliang%20Sunney%20Xie%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

H.W., L.T. and X.S.X. designed the experiments. H.W. performed the experiments. J.Z. and H.W. analyzed the data. H.W., J.Z., L.T. and X.S.X. wrote the manuscript.

### Corresponding author

Correspondence to [Xiaoliang Sunney Xie](mailto:sunneyxie@biopic.pku.edu.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Argyris Papantonis and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02247-6#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 SDS treatment improves Micro-C ligation efficiency.](https://www.nature.com/articles/s41588-025-02247-6/figures/5)

**a**, Chromatin length distribution for different batches of MNase digestion. Results were obtained from the Fragment Analyzer. **b**, Violin plot compares the composition of contacts between SDS treatment and no SDS group. **c**, Contact maps of SDS treatment of single cells and ensemble. **d**, The same as **c** for no SDS group.

[Source data](https://www.nature.com/articles/s41588-025-02247-6#MOESM8)

### [Extended Data Fig. 2 Comparison of MNase titration.](https://www.nature.com/articles/s41588-025-02247-6/figures/6)

**a**, Violin plot compares the number of contacts for MNase titration groups (200U: n = 90, 600U: n = 92, 800U: n = 92, 1000U: n = 91). The box middle line represents the median value, box limits the 25% and 75% quantiles, and the whiskers show the minimum and maximum. **b**, Violin plot compares the composition of contacts between MNase titration group. The box middle line represents the median value, box limits the 25% and 75% quantiles, and the whiskers show the minimum and maximum. **c**, Scatter plot depicts the relationship between number of reads and number of unique contacts. **d**, Contact probability on genomic separation. **e**, Left: saddle plot shows the compartment strength for four MNase titration groups. Right: Pearson correlation of the first eigenvalues at 100 kb resolution. **f**, Insulation scores at insulation boundary. **g**, Ensemble contact maps of four MNase titration groups at 500 kb resolution for chromosome 1.

[Source data](https://www.nature.com/articles/s41588-025-02247-6#MOESM9)

### [Extended Data Fig. 3 scMicro-C retains protein occupancy information.](https://www.nature.com/articles/s41588-025-02247-6/figures/7)

**a**, Heatmap shows the normalized nucleosome occupancy around CTCF motif (left) and active TSS (right). Each row represents a single cell, scMicro-C and Dip-C data were shown. **b**, Line plot shows the normalized nucleosome occupancy around CTCF (top) and active TSS (bottom), two bulk Micro-C replicates and ensemble scMicro-C are shown. **c**, Genomic track shows the nucleosome occupancy around LSM1 gene locus. **d**, Line plot shows transcription factor footprinting, calculated by detected ligation junction around corresponding TF motif.

### [Extended Data Fig. 4 Ensemble scMicro-C faithfully recapitulates multi-scale 3D genome organization.](https://www.nature.com/articles/s41588-025-02247-6/figures/8)

**a**, Left: comparison of chromatin loops between ensemble scMicro-C and bulk Micro-C. Venn diagram depicting overlap of loops between bulk Micro-C and ensemble scMicro-C using chromosight and chromatin loop APA showing bulk Micro-C (top) and ensemble scMicro-C (bottom) around shared and unique loops. The same as left, compare ensemble scMicro-C and bulk in situ Hi-C. **b**, Comparison of detected chromatin stripes between scMicro-C and bulk Micro-C or bulk Hi-C. **c**, Contact maps of bulk Hi-C, bulk Micro-C and ensemble scMicro-C at selected genomic regions of 1 kb or 2 kb resolution. **d**, Saddle plots show the A/B compartmentalization strength. **e**, Heatmap shows the Pearson correlation of A/B compartment between bulk and scMicro-C datasets. **f**, Insulation scores of bulk Hi-C, bulk Micro-C and scMicro-C. **g**, Chromatin length distribution depicts the difference between MNase digestion and restriction enzymes. **h**, Scatter plot demonstrates the relationship between scMicro-C (n = 744) and Dip-C (n = 17). **i**, Violin plot compares the composition of chromatin contacts between scMicro-C (n = 744) and Dip-C (n = 17), accompanied by original bulk Micro-C and modified bulk Micro-C, only ≥ 1 kb contacts were counted. **j**, Chromatin loops pile-up plot for Dip-C (left), scMicro-C (middle) and their difference.

### [Extended Data Fig. 5 Validation of kilobase single-cell 3D genome structure.](https://www.nature.com/articles/s41588-025-02247-6/figures/9)

**a**, 3D genome structures of a representative GM12878 cell (cell 25), 5 independent reconstructed structures are shown. **b**, 3D genome structures at different resolutions of a representative GM12878 cell (cell 25), chromosome territory, chromosome intermingling, A/B compartments and selected chromosomes are shown. **c**, Top: mean scMicro-C 3D distance matrices (top right) and bulk Micro-C contact matrices (bottom left) of selected genomic regions at 20 kb (chr2: 154–160 Mb), 10 kb (chr2: 156–159 Mb) and 5 kb (chr2: 157–158.5 Mb). Bottom: 2D histograms show the correlation between contact frequency and mean 3D distance of the same regions and resolutions. **d**, Mean 3D distance matrix of Dip-C (bottom left,n = 23) and scMicro-C (top right, n = 287) at 5 kb resolution at nested CTCF loops region shown in Fig. [2d](https://www.nature.com/articles/s41588-025-02247-6#Fig2) (chr1: 224.4–225 Mb). **e**, Left: mean 3D distance of Dip-C (bottom left,n = 30) and scMicro-C (top right, n = 412) at 5 kb resolution at chromatin region shown in Fig. [2b](https://www.nature.com/articles/s41588-025-02247-6#Fig2) (Chr5: 184-185.5 Mb). Right: Scatter plot showing the correlation of 3D distance matrix between Dip-C and scMicro-C at region shown in left.

[Source data](https://www.nature.com/articles/s41588-025-02247-6#MOESM10)

### [Extended Data Fig. 6 PES genes show focal interactions at multiple downstream enhancer loci.](https://www.nature.com/articles/s41588-025-02247-6/figures/10)

**a**, Representative contact maps depicting ‘PES genes’ along with corresponding tracks for CTCF, RAD21, H3K27ac, and PLOR2A ChIP–seq data displayed above the contact map. The quantification of stripe interaction intensity was aligned at the top, with the red arrow indicating the peak of focal interaction at the enhancer sites. **b**, Gene length distribution histogram comparing all expressed genes ≥50 kb, PES genes, and length-matched non-PES genes in GM12878 cells. **c**, Left: pile-up analysis comparing PES genes and non-PES genes centered at the TSS without rescaling of GM12878; right: the difference between stripy and non-stripy and quantification at the TSS.

### [Extended Data Fig. 7 ‘Promoter–enhancer stripe’ links the promoter of PES genes to multiple downstream enhancers.](https://www.nature.com/articles/s41588-025-02247-6/figures/11)

**a**, Density plots illustrating the number of enhancers predicted by the ABC model for PES genes versus non-PES genes of GM12878, two-sided Mann–Whitney U test was used. **b**, Density plots displaying the ABC scores of predicted enhancers for PES genes compared to non-PES genes of GM12878, with statistical analysis conducted using the two-sided Mann–Whitney U test. **c**, Heatmap visualizing the distribution of ABC model-predicted enhancers associated with PES genes of GM12878. Among PES genes, 66.2% have more downstream enhancers than upstream, 27.1% have more upstream enhancers than downstream, and 6.7% have equal downstream and upstream enhancers. **d**, Left: APA results between the transcription start sites (TSS) and downstream enhancer loci for PES genes (n = 3,116 E–P pairs); Right: quantification of the pile-up intensity, with the red arrow highlighting the peak at the enhancer site. **e**, APA result between pairwise enhancers of the same stripy gene (n = 16,614 E–E pairs). All enhancers (marked by H3K27ac) were used. **f**, Metagene plots showing the signal of PLOR2A and the cohesin subunit SMC3 at PES genes versus non-PES genes in GM12878 cells. **g**, Similar analyses as in **a** performed for mESC. **h**, Pile-up results comparing non-PES genes (n = 600) in untreated, RAD21-depleted, and WAPL-depleted bulk Micro-C data from mESC. **i**, Metagene plots illustrating the distribution of H3K27ac, POLR2A, SMC1A, and RAD21 at ‘PES genes’ versus non-PES genes in mESC. **j**, APAs between the TSS and multiple gene body enhancers of PES genes (top, n = 2,260) and non-PES genes (bottom, n = 1,109), plotted for untreated, RAD21-depleted and WAPL-depleted mESC Micro-C. **k**, APAs between pairwise gene body enhancers of PES genes (top, n = 9,510) and non-PES genes (bottom, n = 3063), plotted for untreated, RAD21-depleted and WAPL-depleted mESC Micro-C. **l**. Top: scatter plots of loop scores plotted for all E–P interactions between gene body enhancers and promoter of stripy (left) and non-PES genes (right) in the untreated and Rad21-depleted cells. Detected E–P loops were labeled as orange cross, non-loop E-P interactions were labeled as blue dot. The E–P interactions were grouped as strengthened, unchanged and weakened. Bottom: pile-up of loop and non-loop E–P interactions of PES genes for weakened (left) and unchanged (right) interaction upon cohesin depletion.

### [Extended Data Fig. 8 Single-cell 3D genome structure analysis of _IRF2_ gene locus.](https://www.nature.com/articles/s41588-025-02247-6/figures/12)

**a**, Contact map (top left is ensemble scMicro-C, bottom right is bulk Micro-C) at _IRF2_ gene locus along with CTCF, H3K27ac, POLR2A and RAD21 ChIP–seq tracks shown above, the ABC model-predicted IRF2 enhancers were indicated. **b**, Sorted single-cell contact profiles of _IRF2_ gene stripe. **c**, Cartoon depicting the genomic loci selected for single-cell 3D genome analysis. **d**, Line plot illustrating the percentage of single-cell 3D chromatin structures with the proximity of corresponding E–P pairs. Six non-stripe regions were taken as control. The whiskers of control regions represent standard deviation. **e**, Top: representative structures forming corresponding E–P loops. Bottom: mean 3D distance matrices of single-cell 3D genome structures forming corresponding E–P loops. **f**, Left: histogram summarizing single-cell 3D genome structures forming multi-enhancer hubs; Right: representative single-cell 3D structures forming multi-enhancer hubs, the hub was indicated with red circle. **g**, Mean 3D distance matrices of single-cell 3D genome structures forming multi-enhancer hubs with indicated number of enhancers. **h**, Boxplot illustrating the radius of gyration of individual single-cell 3D structures forming multi-enhancer hubs containing 1 to 6 enhancers. Pairwise statistical tests were conducted, revealing statistical significance (_P_ < 0.05) in all comparisons except for the group4-group5 comparison. The analysis used a two-sided Wilcoxon rank sum exact test. Within each boxplot, the centerline represents the median value, while the box limits indicate the 25th and 75th percentiles. The whiskers extend to the maximum and minimum values observed in the data distribution.

### [Extended Data Fig. 9 Polymer simulations of the gene stripe and enhancer interaction network at EBF1 locus.](https://www.nature.com/articles/s41588-025-02247-6/figures/13)

**a**, Schematic representation of the EBF1 locus used in polymer simulations. The TSS and TSS region are modeled as bi-directional cohesin barriers, while enhancers are modeled as uni-directional cohesin barriers with tunable strength. Arc shows the E–P and E–E interactions with adjustable affinities. Detailed simulation settings are provided in the Methods section. **b**, Chromatin contact map of the EBF1 locus from GM12878 bulk Micro-C at 2 kb resolution. **c**–**e**, Simulated contact matrices under different enhancer blocking strength: no blocking (**c**), 30% blocking probability (**d**) and 60% blocking probability (**e**). For each blocking condition, E–P and E–E affinities are varied: 0.1 KT (left), 0.5 KT (middle), and 0.8 KT (right). Each panel shows results with cohesin extrusion (top), without cohesin extrusion (middle), and the difference between the two conditions (bottom).

### [Extended Data Fig. 10 Polymer simulations recapitulate multi-enhancer hub structures.](https://www.nature.com/articles/s41588-025-02247-6/figures/14)

**a**, Top: line plots summarize the number of simulated structures forming multi-enhancer hub under a stringent threshold of 5 monomer radii with different simulation parameters. Botton: line plots illustrate the number of simulated structures forming E–P loops between EBF1 promoter and enhancers E1–E7, respectively. **b**, The same analysis as in **a**, but with a less stringent threshold of 7.5 monomer radii. **c**, Top: representative simulated 3D structures. Bottom: mean distance matrix of simulated structures forming E–P loops with enhancers E1 to E7, arranged from left to right. **d**, Top: representative simulated 3D structures. Bottom: mean distance matrix of simulated structures with 1 to 6 interacting enhancers. Results of **c** and **d** are based on parameter: cohesin blocking probability of enhancer is 60%, E–P and E–E affinities set to 0.8 KT. **e**, Cohesin-medaited loops extrusion brings multiple downstream enhancers and promoter into contact and is further stabilized by molecular affinities among transcriptional proteins bound at these elements. **f**, Adapter protein LDB1-mediated multi-enhancer hub interactions. LDB1 binds to LIM domain-containing TFs, then undergoes oligomerization through self-association domain.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM1_ESM.pdf)

Supplementary Figs. 1–7 and Supplementary Note (protocols).

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM3_ESM.pdf)

### [Supplementary Table 1.](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM4_ESM.xlsx)

scMicro-C metadata file

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM5_ESM.xlsx)

Statistical source data for Fig. 1c,e,f.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM7_ESM.xlsx)

Statistical source data for Fig. 4d,f,i.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM8_ESM.xlsx)

Statistical source data for Extended Data Fig. 1b.

### [Source Data Extended Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM9_ESM.xlsx)

Statistical source data for Extended Data Fig. 2a–c.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 12](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02247-6/MediaObjects/41588_2025_2247_MOESM11_ESM.xlsx)

Statistical source data for Extended Data Fig. 12g.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Single-cell%20Micro-C%20profiles%203D%20genome%20structures%20at%20high%20resolution%20and%20characterizes%20multi-enhancer%20hubs&author=Honggui%20Wu%20et%20al&contentID=10.1038%2Fs41588-025-02247-6&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-02&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02247-6)

### Cite this article

Wu, H., Zhang, J., Tan, L. _et al._ Single-cell Micro-C profiles 3D genome structures at high resolution and characterizes multi-enhancer hubs. _Nat Genet_ **57**, 1777–1786 (2025). https://doi.org/10.1038/s41588-025-02247-6

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02247-6?format=refman&flavour=citation)

-   Received: 13 September 2023
    
-   Accepted: 29 May 2025
    
-   Published: 02 July 2025
    
-   Issue Date: July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02247-6
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02247-6.pdf)
