> id: https://www.nature.com/articles/s41588-025-02266-3

> link: https://www.nature.com/articles/s41588-025-02266-3

> title: Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution

# Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution
_Published on Mon, 28 Jul 2025 00:00:00 GMT_

Abstract
--------

Current molecular quantitative trait locus catalogs are mostly at bulk resolution and centered on Europeans. Here, we constructed an immune cell atlas with single-cell transcriptomics of >1.5 million peripheral blood mononuclear cells, host genetics, plasma proteomics and gut metagenomics from 235 Japanese persons, including patients with coronavirus disease 2019 (COVID-19) and healthy individuals. We mapped germline genetic effects on gene expression within immune cell types and across cell states. We elucidated cell type- and context-specific human leukocyte antigen (HLA) and genome-wide associations with T and B cell receptor repertoires. Colocalization using dynamic genetic regulation provided better understanding of genome-wide association signals. Differential gene and protein expression analyses depicted cell type- and context-specific effects of polygenic risks. Various somatic mutations including mosaic chromosomal alterations, loss of Y chromosome and mitochondrial DNA (mtDNA) heteroplasmy were projected into single-cell resolution. We identified immune features specific to somatically mutated cells. Overall, immune cells are dynamically regulated in a cell state-dependent manner characterized with multiomic profiles.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-023-01421-y/MediaObjects/41588_2023_1421_Fig1_HTML.png)

### [Mapping interindividual dynamics of innate immune response at single-cell resolution](https://www.nature.com/articles/s41588-023-01421-y?fromPaywallRec=false)

Article Open access 12 June 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-021-27716-4/MediaObjects/41467_2021_27716_Fig1_HTML.png)

### [Single-cell multi-omics reveals dyssynchrony of the innate and adaptive immune system in progressive COVID-19](https://www.nature.com/articles/s41467-021-27716-4?fromPaywallRec=false)

Article Open access 21 January 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png)

### [Single-cell multi-omics analysis of the immune response in COVID-19](https://www.nature.com/articles/s41591-021-01329-2?fromPaywallRec=false)

Article Open access 20 April 2021

Main
----

Human omic technologies project biological mechanisms and disease pathophysiology into multi-layered matrix information with diverse resolutions. Integrative omics analysis anchored by germline genetic variants leveraged molecular quantitative trait locus (mQTL) catalogs[1](https://www.nature.com/articles/s41588-025-02266-3#ref-CR1 "Aguet, F. et al. Molecular quantitative trait loci. Nat. Rev. Methods Primers 3, 5 (2023)."). Multi-layered mQTL catalogs synergistically answered functional annotation of the variants and filled the path from large-scale human disease genetics (that is, genome-wide association studies; GWASs) to outcome clinical phenotypes[2](https://www.nature.com/articles/s41588-025-02266-3#ref-CR2 "Aguet, F. et al. Genetic effects on gene expression across human tissues. Nature 550, 204–213 (2017)."),[3](https://www.nature.com/articles/s41588-025-02266-3#ref-CR3 "Strober, B. J. et al. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science 18, 1318–1330 (2020)."),[4](https://www.nature.com/articles/s41588-025-02266-3#ref-CR4 "Sun, B. B. et al. Plasma proteomic associations with genetics and health in the UK Biobank. Nature 622, 329–338 (2023)."),[5](https://www.nature.com/articles/s41588-025-02266-3#ref-CR5 "Qin, Y. et al. Combined effects of host genetics and diet on human gut microbiota and incident disease in a single population cohort. Nat. Genet. 54, 134–142 (2022)."),[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022)."),[7](https://www.nature.com/articles/s41588-025-02266-3#ref-CR7 "Perez, R. K. et al. Single-cell RNA-seq reveals cell type-specific molecular and genetic associations to lupus. Science 376, eabf1970 (2022)."),[8](https://www.nature.com/articles/s41588-025-02266-3#ref-CR8 "Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. Nat. Genet. 54, 817–826 (2022)."),[9](https://www.nature.com/articles/s41588-025-02266-3#ref-CR9 "Nathan, A. et al. Single-cell eQTL models reveal dynamic T cell state dependence of disease loci. Nature 606, 120–128 (2022)."),[10](https://www.nature.com/articles/s41588-025-02266-3#ref-CR10 "Kumasaka, N. et al. Mapping interindividual dynamics of innate immune response at single-cell resolution. Nat. Genet. 55, 1066–1075 (2023)."). Such efforts initially started with bulk RNA expression profiles[2](https://www.nature.com/articles/s41588-025-02266-3#ref-CR2 "Aguet, F. et al. Genetic effects on gene expression across human tissues. Nature 550, 204–213 (2017)."),[3](https://www.nature.com/articles/s41588-025-02266-3#ref-CR3 "Strober, B. J. et al. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science 18, 1318–1330 (2020).") and have expanded to include highly diverse layers such as proteomics[4](https://www.nature.com/articles/s41588-025-02266-3#ref-CR4 "Sun, B. B. et al. Plasma proteomic associations with genetics and health in the UK Biobank. Nature 622, 329–338 (2023).") and metagenomics[5](https://www.nature.com/articles/s41588-025-02266-3#ref-CR5 "Qin, Y. et al. Combined effects of host genetics and diet on human gut microbiota and incident disease in a single population cohort. Nat. Genet. 54, 134–142 (2022)."). Of these, recent technological advances in single-cell RNA sequencing (scRNA-seq) have successfully elucidated cell state heterogeneity in a variety of tissues and environments[11](https://www.nature.com/articles/s41588-025-02266-3#ref-CR11 "Zheng, L. et al. Pan-cancer single-cell landscape of tumor-infiltrating T cells. Science 374, abe6474 (2021)."),[12](https://www.nature.com/articles/s41588-025-02266-3#ref-CR12 "Yoshida, M. et al. Local and systemic responses to SARS-CoV-2 infection in children and adults. Nature 602, 321–327 (2022)."). Genetic association mapping with scRNA-seq profiling could capture continuous genetic effects along cellular states across discrete cell types and provide more granular insights into the molecular mechanisms of human trait-associated genetic variants[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022)."),[7](https://www.nature.com/articles/s41588-025-02266-3#ref-CR7 "Perez, R. K. et al. Single-cell RNA-seq reveals cell type-specific molecular and genetic associations to lupus. Science 376, eabf1970 (2022)."),[8](https://www.nature.com/articles/s41588-025-02266-3#ref-CR8 "Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. Nat. Genet. 54, 817–826 (2022)."),[9](https://www.nature.com/articles/s41588-025-02266-3#ref-CR9 "Nathan, A. et al. Single-cell eQTL models reveal dynamic T cell state dependence of disease loci. Nature 606, 120–128 (2022)."),[10](https://www.nature.com/articles/s41588-025-02266-3#ref-CR10 "Kumasaka, N. et al. Mapping interindividual dynamics of innate immune response at single-cell resolution. Nat. Genet. 55, 1066–1075 (2023)."). Nevertheless, current single-cell expression QTL (sc-eQTL) resources are mostly centered on European ancestry[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022)."),[8](https://www.nature.com/articles/s41588-025-02266-3#ref-CR8 "Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. Nat. Genet. 54, 817–826 (2022)."),[13](https://www.nature.com/articles/s41588-025-02266-3#ref-CR13 "Oelen, R. et al. Single-cell RNA-sequencing of peripheral blood mononuclear cells reveals widespread, context-specific gene expression regulation upon pathogenic exposure. Nat. Commun. 13, 3267 (2022)."), which rationalizes the need to construct multi-layered omics with single-cell resolution in non-European ancestries.

As a common but overlooked human mQTL layer, we additionally propose a value of using somatic genetic variations, which have been studied mainly in the field of cancer. Clonally expanded blood cells with somatic mutations or clonal hematopoiesis (CH) are common in apparently healthy individuals[14](https://www.nature.com/articles/s41588-025-02266-3#ref-CR14 "Loh, P., Genovese, G. & McCarroll, S. A. Monogenic and polygenic inheritance become instruments for clonal selection. Nature 584, 136–141 (2020)."),[15](https://www.nature.com/articles/s41588-025-02266-3#ref-CR15 "Thompson, D. J. et al. Genetic predisposition to mosaic Y chromosome loss in blood. Nature 575, 652–657 (2019).") and increase the risk not only for hematological malignancies[16](https://www.nature.com/articles/s41588-025-02266-3#ref-CR16 "Laurie, C. C. et al. Detectable clonal mosaicism from birth to old age and its relationship to cancer. Nat. Genet. 44, 642–650 (2012)."),[17](https://www.nature.com/articles/s41588-025-02266-3#ref-CR17 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[18](https://www.nature.com/articles/s41588-025-02266-3#ref-CR18 "Saiki, R. et al. Combined landscape of single-nucleotide variants and copy number alterations in clonal hematopoiesis. Nat. Med. 27, 1239–1249 (2021).") but for a variety of benign disorders, including cardiovascular disease[19](https://www.nature.com/articles/s41588-025-02266-3#ref-CR19 "Jaiswal, S. et al. Clonal hematopoiesis and risk of atherosclerotic cardiovascular disease. N. Engl. J. Med. 377, 111–121 (2017)."),[20](https://www.nature.com/articles/s41588-025-02266-3#ref-CR20 "Sano, S. et al. Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality. Science 377, 292–297 (2022).") and infectious diseases[21](https://www.nature.com/articles/s41588-025-02266-3#ref-CR21 "Zekavat, S. M. et al. Hematopoietic mosaic chromosomal alterations increase the risk for diverse types of infection. Nat. Med. 27, 1012–1024 (2021)."),[22](https://www.nature.com/articles/s41588-025-02266-3#ref-CR22 "Bruhn-Olszewska, B. et al. Loss of Y in leukocytes as a risk factor for critical COVID-19 in men. Genome Med. 14, 139 (2022)."). However, the biological mechanisms underlying the associations between CH and benign disorders remain unknown. Refining somatic mutation spectra with finer resolution, namely, single cells, should especially contribute to deciphering biological mechanisms and driver-or-passenger discussions[22](https://www.nature.com/articles/s41588-025-02266-3#ref-CR22 "Bruhn-Olszewska, B. et al. Loss of Y in leukocytes as a risk factor for critical COVID-19 in men. Genome Med. 14, 139 (2022)."),[23](https://www.nature.com/articles/s41588-025-02266-3#ref-CR23 "Nam, A. S. et al. Single-cell multi-omics of human clonal hematopoiesis reveals that DNMT3A R882 mutations perturb early progenitor states through selective hypomethylation. Nat. Genet. 54, 1514–1526 (2022)."),[24](https://www.nature.com/articles/s41588-025-02266-3#ref-CR24 "Gao, T. et al. A pan-tissue survey of mosaic chromosomal alterations in 948 individuals. Nat. Genet. 55, 1901–1911 (2023)."),[25](https://www.nature.com/articles/s41588-025-02266-3#ref-CR25 "Abdel-Hafiz, H. A. et al. Y chromosome loss in cancer drives growth by evasion of adaptive immunity. Nature 619, 624–631 (2023)."),[26](https://www.nature.com/articles/s41588-025-02266-3#ref-CR26 "Lareau, C. A. et al. Single-cell multi-omics of mitochondrial DNA disorders reveals dynamics of purifying selection across human immune cells. Nat. Genet. 55, 1198–1209 (2023)."), and there is definitely a growing interest in using multi-layer omics to elucidate the mechanisms of phenotypic effects of somatic events.

To interpret cell state-dependent biological phenomena by deconvoluting immune features from multi-layer human omics into single-cell resolution, here we constructed a multiomic immune cell atlas, the Osaka Atlas of Immune Cells (OASIS), from 235 Japanese persons including patients with COVID-19 and healthy individuals. The 5′ single-cell transcriptomics data profiling over 1,500,000 peripheral blood mononuclear cells (PBMCs), which are characterized by a large number of cells per sample and the inclusion of cells under in vivo immunological stimulation by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, resulted in covering a variety of cell states. OASIS links these single-cell transcriptomics data with host genetics, plasma proteomics and metagenomics data.

We mapped the germline genetic effects on gene expression within cell types and across cell states. We also investigated HLA and genome-wide associations with variable–diversity–joining (VDJ) gene usage according to clinical status and cell types. To interpret GWAS results, we then conducted colocalization analysis of GWAS signals and our eQTLs and differential gene and protein expression analysis with polygenic risk scores (PRSs) across clinical status and cell types. In addition, by leveraging whole-genome sequencing (WGS) and SNP genotyping, we captured various somatic mutations such as mosaic chromosomal alterations (mCAs), loss of the Y chromosome (LOY) and heteroplasmy in mtDNA (mt-heteroplasmy) at single-cell resolution. We evaluated the immune features of somatically mutated cells, which were characterized as cell type-specific clonal expansions. We assessed the reactivity of clonally expanded mCAs in a patient with COVID-19 against major SARS-CoV-2 antigens using B cell receptors (BCRs). We also evaluated the effect of the gut microbiome on the differential abundance of peripheral immune cells based on a cell–cell similarity structure.

These integrative analyses at single-cell resolution demonstrated that immune cells were dynamically regulated in a cell state-dependent manner defined by the features of multi-layer omics. Our non-European, multi-layered and diversified immune single-cell atlas will contribute to equity in global diversity of human genomics and will be a valuable resource to understand complex human traits.

Results
-------

### The OASIS cohort with multi-layered single-cell omics

The OASIS cohort consists of 88 patients with COVID-19 and 147 healthy individuals of Japanese ancestry with multiomics data of immune cells (_n_ = 235; Fig. [1a](https://www.nature.com/articles/s41588-025-02266-3#Fig1), Supplementary Fig. [1a,b](https://www.nature.com/articles/s41588-025-02266-3#MOESM1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). We performed scRNA-seq and single-cell VDJ sequencing (scVDJ-seq) on 2,059,141 PBMCs using the 10x Genomics Chromium platform and obtained 1,506,953 high-quality cells ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15) and Supplementary Fig. [1c](https://www.nature.com/articles/s41588-025-02266-3#MOESM1)). We manually annotated cells based on RNA expression of known marker genes[27](https://www.nature.com/articles/s41588-025-02266-3#ref-CR27 "Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587 (2021)."),[28](https://www.nature.com/articles/s41588-025-02266-3#ref-CR28 "Mulder, K. et al. Cross-tissue single-cell landscape of human monocytes and macrophages in health and disease. Immunity 54, 1883–1900 (2021)."),[29](https://www.nature.com/articles/s41588-025-02266-3#ref-CR29 "Turner, J. S. et al. Human germinal centres engage memory and naive B cells after influenza vaccination. Nature 586, 127–132 (2020)."),[30](https://www.nature.com/articles/s41588-025-02266-3#ref-CR30 "Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 184, 1895–1913 (2021)."),[31](https://www.nature.com/articles/s41588-025-02266-3#ref-CR31 "Krämer, B. et al. Early IFN-α signatures and persistent dysfunction are distinguishing features of NK cells in severe COVID-19. Immunity 54, 2650–2669 (2021)."). We first defined seven major cell types (L1) according to the Azimuth L1 annotation (Fig. [1b](https://www.nature.com/articles/s41588-025-02266-3#Fig1))[27](https://www.nature.com/articles/s41588-025-02266-3#ref-CR27 "Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587 (2021)."). Next, we further identified 28 cell states (L2) and annotated ten cell types (LOneK1K) for comparison with OneK1K[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022).") (Fig. [1b](https://www.nature.com/articles/s41588-025-02266-3#Fig1), Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02266-3#Fig9) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). We verified our manual fine annotation using scPred[32](https://www.nature.com/articles/s41588-025-02266-3#ref-CR32 "Alquicira-Hernandez, J., Sathe, A., Ji, H. P., Nguyen, Q. & Powell, J. E. ScPred: accurate supervised method for cell-type classification from single-cell RNA-seq data. Genome Biol. 20, 264 (2019).") with the Azimuth L2 annotation as a ref. [27](https://www.nature.com/articles/s41588-025-02266-3#ref-CR27 "Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587 (2021)."), which showed high concordance (Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02266-3#Fig9)).

**Fig. 1: Overview and scRNA-seq data of OASIS.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/1)

**a**, Overview of the study design. SLE, systemic lupus erythematosus. **b**, UMAP embedding of scRNA-seq data for 1,506,953 cells. Seven major cell types and 28 fine cell types were defined by RNA expression of marker genes (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02266-3#Fig9)). Bact, activated B cell; Bmem, memory B cell; BN1, type 1 naive B cell; BN2, type 2 naive B cell; BIN, intermediate B cell; CD4CTL, CD4+ cytotoxic T celle; CD4TCM, CD4+ central memory T cell; CD4TEM, CD4+ effector memory T cell; CD8CTL, CD8+ cytotoxic T cell; CD8N, naive CD8+ T cell; CD8TCM, CD8+ central memory T cell; CD8TEM, CD8+ effector memory T cell; cDC, conventional dendritic cell; cMono, classical monocyte; cMonoIL1B, IL1B cMono; cMonoS100A, S100A cMono; intMono, intermediate monocyte; MAIT, mucosal-associated invariant T cell; Mono, monocyte; ncMono, non-classical monocyte; NK, natural killer cell; NKcyto, cytokine NK cell; pDC, plasmacytoid dendritic cell; Pro\_T, proliferating T cell; Treg, regulatory T cell. Panel a created with [BioRender.com](https://www.biorender.com/).

We generated WGS data for eQTL analysis and detecting mt-heteroplasmy and SNP array data for detecting mCAs from all samples ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). We measured the expression of 2,925 plasma proteins with the Olink assay for 227 samples. We also performed whole-genome shotgun sequencing analysis of 131 fecal DNA samples from healthy individuals and obtained phylogenetic relative abundances.

### Immune cell type-specific single-cell _cis_\-eQTL mapping

To evaluate the genetic regulation of gene expression in immune cells, we first performed single-cell _cis_\-eQTL analysis with WGS data using the pseudobulk approach. We tested for association between expressed genes and genetic variants located within 1 Mb of the transcription start site in each of seven major cell types (L1) and 28 fine cell types (L2). We also mapped conditionally independent _cis_\-eQTLs.

In total, we identified 23,443 and 34,297 eQTLs in L1 and L2, of which 19,641 and 30,802 were primary _cis_\-eQTLs (Fig. [2a](https://www.nature.com/articles/s41588-025-02266-3#Fig2) and Supplementary Table [3](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). We detected 488–4,901 (median = 3,176) genes with significant _cis_\-eQTL effects (eGenes) in L1 and 93–4,062 (median = 862) eGenes in L2, and the number of eGenes varied widely across cell types. We observed a strong association between the number of eGenes and that of cells per sample of the corresponding cell types as previously reported[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022)."),[8](https://www.nature.com/articles/s41588-025-02266-3#ref-CR8 "Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. Nat. Genet. 54, 817–826 (2022).") (Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02266-3#Fig10)). Primary significant eQTL effect sizes were negatively correlated with the number of cells per sample of the corresponding cell types (Fig. [2b](https://www.nature.com/articles/s41588-025-02266-3#Fig2)). Surprisingly, the numbers of primary eQTLs detected per cell type in this cohort were equal to or greater than those of OneK1K[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022)."), which has about four times as many samples as our cohort (Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02266-3#Fig10) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). Therefore, we extensively evaluated the relationship between sample size and cell counts per sample in _cis_\-eQTL discovery by downsampling both at L1. The number of discovered eQTLs showed a linear decrease when downsampling the rate of cell counts per sample as well as sample sizes, and a similar relationship was observed in OneK1K (Extended Data Fig. [2c,d](https://www.nature.com/articles/s41588-025-02266-3#Fig10)). These observations imply that the statistical power to detect eQTLs is highly dependent on the number of cells profiled.

**Fig. 2: Germline genetic variants and the gut metagenome affect transcriptional and phenotypic profiles of immune cells.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/2)

**a**, Number of significant independent eQTLs in each cell type of L1 and L2 levels. The bars are colored by cluster resolution (L1 or L2 level) and shaded by independence of eQTL mapping. DC, dendritic cell. **b**, Scatterplot depicting the correlation between median number of cells per sample and median effect size of significant eQTLs in each cell type. The _x_ axis is on a log10 scale. **c**, Number of significant eGenes shared among cell types in the L1 level. eGenes identified in only one cell type are colored pink, and those in multiple cell types are connected by lines and colored gray. **d**, Heatmap showing a pairwise comparison of eQTL effect size. Only significant eQTLs in one cell type (reference) that could be evaluated in other cell types (target) were analyzed. **e**, Heatmap depicting the enrichment of primary significant eQTLs of L2 cell types in promoter or enhancer regions of eight representative immune cells from the Roadmap project. **f**, Box plot showing the enrichment of primary eQTLs in promoters or enhancers of the corresponding cell types (_n_ = 28) from the Roadmap project, according to effect sizes ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). **g**, Graphical representation of neighborhoods identified by Milo in healthy individuals (_n_ = 131). Nodes are neighborhoods, colored by log2 (fold change (FC)) by the relative abundance of _R. gnavus_ adjusted for age, sex and sequencing groups. Sizes correspond to the number of cells in a neighborhood. Graph edges depict the number of cells shared between adjacent neighborhoods. Nhood, neighborhood. **h**, Beeswarm plot and box plot showing the distribution of adjusted log2 (FC) in neighborhoods (_n_ = 43,089) by the relative abundance of _R. gnavus_ among L2 cell types. Colors are represented in the same way as in **g**. Boxes denote the interquartile range (IQR), medians are shown as horizontal bars, and whiskers extend to 1.5 times the IQR in **f**,**h**. N, naive; TCM, central memory T cell; TEM, effector memory T cell; CTL, cytotoxic T cell; N1, type 1 naive; N2, type 2 naive; IN, intermediate; Cyto, cytokine; Act, activated; corr, correlation.

Next, we evaluated how much eGenes were shared across cell types. We observed that 3,422 of 8,047 eGenes were cell type specific, whereas 1,214 eGenes were shared by more than five cell types in L1 (Fig. [2c](https://www.nature.com/articles/s41588-025-02266-3#Fig2) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). In L2, 2,613 of 7,386 eGenes were significant in only one cell type. We also compared eQTL effects among cell types and observed a high level of eQTL sharing especially within the same lineages (T and natural killer, B and myeloid cells; Fig. [2d](https://www.nature.com/articles/s41588-025-02266-3#Fig2) and Supplementary Table [6](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). We compared the effects of significant eQTLs to the bulk eQTL dataset from Japanese individuals[33](https://www.nature.com/articles/s41588-025-02266-3#ref-CR33 "Ota, M. et al. Dynamic landscape of immune cell-specific gene regulation in immune-mediated diseases. Cell 184, 3006–3021 (2021).") and found high concordance (Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02266-3#MOESM1) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)).

We compared _cis_\-eQTLs from our cohort with those from OneK1K[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022).") consisting of Europeans using the multivariate adaptive shrinkage method[34](https://www.nature.com/articles/s41588-025-02266-3#ref-CR34 "Urbut, S. M., Wang, G., Carbonetto, P. & Stephens, M. Flexible statistical methods for estimating and testing effects in genomic studies with multiple conditions. Nat. Genet. 51, 187–195 (2019).") ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). The significant eQTLs in OneK1K were more likely to be replicated in the corresponding cell types of OASIS than those in OASIS (Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02266-3#Fig11)), which was more pronounced in cell types, with a higher number of eGenes in OASIS than in OneK1K. In naive CD4+ T (CD4N) cells, eQTLs that were not replicated in the other cohort showed larger differences in minor allele frequency (MAF) between East Asian and European populations than replicated eQTLs (Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02266-3#Fig11)). The direction of significant eQTL effects was almost the same in the other cohort across all cell types (Extended Data Fig. [3c](https://www.nature.com/articles/s41588-025-02266-3#Fig11)). The median proportions of shared eQTLs by magnitude in the other cohort for ten cell type pairs were 34.3% for OASIS eQTLs and 69.3% for OneK1K eQTLs when the factor was set to 0.5 (Extended Data Fig. [3c](https://www.nature.com/articles/s41588-025-02266-3#Fig11) and Supplementary Table [8](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). Similar to the replication, eQTLs in CD4N cells that were not shared between the two cohorts tended to show larger differences in MAF between East Asian and European populations than shared eQTLs (Extended Data Fig. [3d](https://www.nature.com/articles/s41588-025-02266-3#Fig11)). These results suggest the importance of constructing sc-eQTL resources from different populations with larger sample sizes.

We annotated significant eQTLs using chromatin state predictions from eight immune cell types of the Roadmap Epigenomics project[35](https://www.nature.com/articles/s41588-025-02266-3#ref-CR35 "Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317–329 (2015)."). Primary eQTLs were enriched in both promoter and enhancer regions of Roadmap immune cells. However, enrichment of eQTLs in promoter regions was shared among cell types, whereas that in enhancer regions was more cell type specific (Fig. [2e](https://www.nature.com/articles/s41588-025-02266-3#Fig2) and Supplementary Table [9](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). When primary eQTLs were stratified by effect sizes, promoter enrichment was more predominant at larger effect sizes of eQTLs, but enhancer enrichment was similar regardless of effect sizes (Fig. [2f](https://www.nature.com/articles/s41588-025-02266-3#Fig2)). Considering that more than 60% of causal variants in autoimmune diseases map to immune cell enhancers[36](https://www.nature.com/articles/s41588-025-02266-3#ref-CR36 "Farh, K. K.-H. et al. Genetic and epigenetic fine mapping of causal autoimmune disease variants. Nature 518, 337–343 (2015)."), this result motivates us to identify more eQTLs in enhancer regions that do not necessarily have large effect sizes.

### Gut bacterial abundance is associated with immune cell abundance

To reveal the sets of immune cells that were differentially abundant with bacterial abundance at finer resolution, we performed differential abundance analysis using single-cell neighborhoods (Milo)[37](https://www.nature.com/articles/s41588-025-02266-3#ref-CR37 "Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. Nat. Biotechnol. 40, 245–253 (2022)."). In this analysis, we focused on the following three species, all of which had significant associations with more than two cell types in the proportional analysis (Supplementary Fig. [5a](https://www.nature.com/articles/s41588-025-02266-3#MOESM1)) and have been reported to be involved in human diseases: _Ruminococcus gnavus_[38](https://www.nature.com/articles/s41588-025-02266-3#ref-CR38 "Franzosa, E. A. et al. Gut microbiome structure and metabolic activity in inflammatory bowel disease. Nat. Microbiol. 4, 293–305 (2019)."), _Prevotella copri_[39](https://www.nature.com/articles/s41588-025-02266-3#ref-CR39 "Nii, T. et al. Genomic repertoires linked with pathogenic potency of arthritogenic Prevotella copri isolated from the gut of patients with rheumatoid arthritis. Ann. Rheum. Dis. 82, 621–629 (2023).") and _Bacteroides vulgatus_[40](https://www.nature.com/articles/s41588-025-02266-3#ref-CR40 "Mills, R. H. et al. Multi-omics analyses of the ulcerative colitis gut microbiome link Bacteroides vulgatus proteases with disease severity. Nat. Microbiol. 7, 262–276 (2022)."). We identified 43,089 neighborhoods among PBMCs, none of which showed significant differential abundance with _R. gnavus_ abundance (Fig. [2g](https://www.nature.com/articles/s41588-025-02266-3#Fig2)). However, the increase in _R. gnavus_ was accompanied by an increase of rare cell types (that is, CD4+ cytotoxic T cells, activated B cells and plasmablasts (PBs)), along with an increase in cell types of the myeloid cluster (Fig. [2h](https://www.nature.com/articles/s41588-025-02266-3#Fig2)). Similarly, differential abundance analysis revealed that the increase in _P. copri_ was associated with an increase in PBs and plasmacytoid dendritic cells and a decrease in CD4+ cytotoxic T cells (Supplementary Fig. [5b,c](https://www.nature.com/articles/s41588-025-02266-3#MOESM1)). Such differential abundances were consistent with results of the proportional analysis, demonstrating robustness. These findings highlight that integrative omics analyses anchored by single-cell data allow us to detect biological phenomena that could not be captured by conventional bulk analyses.

### HLA and genome-wide associations with immune repertoires

We performed scVDJ-seq to construct a catalog of T cell receptor (TCR) and BCR repertoires, including usage of each gene across different cell types (Supplementary Fig. [6](https://www.nature.com/articles/s41588-025-02266-3#MOESM1)). We explored the relationship between HLA and the TCR repertoire, characterized by specific cell types. We assessed associations between HLA amino acid variants and TCR V gene usage, including TRAV and TRBV. We identified significant associations between TRAV genes and HLA amino acid variants both in HLA class I and II genes. Specifically, TRAV genes were associated with HLA class I and II gene variants exclusively in CD8+ T and CD4N cells, respectively (_P_ < 1.1 × 10−5; Fig. [3a,b](https://www.nature.com/articles/s41588-025-02266-3#Fig3)). These results align with the well-known pattern of HLA restriction on TCRs according to T cell subpopulations[41](https://www.nature.com/articles/s41588-025-02266-3#ref-CR41 "La Gruta, N. L., Gras, S., Daley, S. R., Thomas, P. G. & Rossjohn, J. Understanding the drivers of MHC restriction of T cell receptors. Nat. Rev. Immunol. 18, 467–478 (2018)."),[42](https://www.nature.com/articles/s41588-025-02266-3#ref-CR42 "Nagafuchi, Y. et al. Control of naive and effector CD4 T cell receptor repertoires by rheumatoid-arthritis-risk HLA alleles. J. Autoimmun. 133, 102907 (2022)."). Furthermore, in comparisons among CD4+ T cell subpopulations, the associations between TRAV genes and HLA class II gene variants appeared stronger in CD4N cells than in entire (that is, CD4+ T) and central memory CD4+ T cells (Fig. [3b](https://www.nature.com/articles/s41588-025-02266-3#Fig3)). This might reflect the stronger HLA restriction on the TCR repertoire formed at the central level (that is, in the thymus) than that formed at the peripheral level[43](https://www.nature.com/articles/s41588-025-02266-3#ref-CR43 "Ishigaki, K. et al. HLA autoimmune risk alleles restrict the hypervariable region of T cell receptors. Nat. Genet. 54, 393–402 (2022).").

**Fig. 3: HLA and genome-wide association analysis with TCR and BCR repertoires.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/3)

**a**, Regional plots for HLA association with TRAV gene usage in CD4+ T (left), CD4N (center) and CD8+ T (right) cells. In each plot, −log10 (_P_ values) for the association tests between amino acid variants of each HLA gene and all tested TRAV gene usages are shown along with the horizontal axis representing amino acid positions. The dashed red horizontal line represents the study-wide _P-_value significance threshold. **b**,**c**, Quantile–quantile plots for the association (**b**) and interaction (**c**) tests between HLA amino acid variants and TRAV gene usage in HLA class I (left) and II (right) genes in different cell types. Vertical and horizontal axes indicate the observed and expected –log10 (_P_ values) for the tests. **d**,**e**, Heatmaps show the maximum values of −log10 (_P_ values) for the association tests for repertoire features (horizontal axes) within individual loci (vertical axes) for TCR (**d**) and BCR (**e**) repertoires. The position range for each locus is determined by the gene body for individual genes and the uppermost and lowermost gene bodies for multiple genes or gene clusters. Only loci that exhibited significant associations with a feature in at least one cell type are displayed. Uncorrected _P_ values from two-sided tests are shown in **a**–**e**. CV, coefficient of variation; freq, frequency; len, length.

We explored the impact of HLA variation on the TCR repertoire in response to COVID-19. We observed significant interactions, particularly those for HLA class I gene variants and TRAV gene usage in CD8+ T cells (_P_ < 2.1 × 10−6; Fig. [3c](https://www.nature.com/articles/s41588-025-02266-3#Fig3)). This may reflect the generation of different TCR repertoire in SARS-CoV-2 infection according to HLA class I variation in CD8+ T cells.

Additionally, we investigated the association of genome-wide variants with the TCR repertoire, focusing on integrative features representing repertoire variation ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). Furthermore, we extended this analysis to TCR D and J genes, as the genetic association with these genes has been underexplored. We identified a significant association between class II HLA loci with principal component (PC)2 for TRAV gene usage in CD4+ T cells as well as PC1 for TRAV gene usage in CD4N cells (_P_ < 2.0 × 10−9; Fig. [3d](https://www.nature.com/articles/s41588-025-02266-3#Fig3), Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02266-3#Fig12) and Supplementary Table [11](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)).

Lastly, we conducted the same repertoire feature-wide association analysis for BCRs. We detected significant associations between PCs for several immunoglobulin gene usages and variants located within these genes (_P_ < 1.3 × 10−9; Fig. [3e](https://www.nature.com/articles/s41588-025-02266-3#Fig3), Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02266-3#Fig12) and Supplementary Table [12](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). Furthermore, somatic hypermutation (SHM)-related features for K and L chains were significantly associated with variants in the IGKV and IGLV genes (Fig. [3e](https://www.nature.com/articles/s41588-025-02266-3#Fig3) and Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02266-3#Fig12)). The stronger associations in naive B cells may be explained by the observation that SHMs in memory cells are more likely to be shaped by exposures, which can weaken the genetic effects.

### Dynamic eQTL effects across two immune-related gene modules

To evaluate the dynamic genetic regulation of gene expression along continuous cell states, we investigated the dynamic effects of eQTLs in the myeloid cluster across the two gene modules related to immunity and COVID-19 severity[12](https://www.nature.com/articles/s41588-025-02266-3#ref-CR12 "Yoshida, M. et al. Local and systemic responses to SARS-CoV-2 infection in children and adults. Nature 602, 321–327 (2022)."),[30](https://www.nature.com/articles/s41588-025-02266-3#ref-CR30 "Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 184, 1895–1913 (2021)."),[44](https://www.nature.com/articles/s41588-025-02266-3#ref-CR44 "Edahiro, R. et al. Single-cell analyses and host genetics highlight the role of innate immune cells in COVID-19 severity. Nat. Genet. 55, 753–767 (2023).") (Fig. [4a](https://www.nature.com/articles/s41588-025-02266-3#Fig4)). Module 1 was highly correlated with uniform manifold approximation and projection (UMAP)1 and module 2 with UMAP2, representing different cell states. To model dynamic eQTLs, the gene modules were divided into ten bins, and the average expression profiles per individual in each bin were reconstructed for each module (Fig. [4a](https://www.nature.com/articles/s41588-025-02266-3#Fig4) and the [Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). We evaluated dynamic eQTLs by testing the interaction between genotype and quantile rank using both linear and quadratic models for the two modules, respectively[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022)."),[8](https://www.nature.com/articles/s41588-025-02266-3#ref-CR8 "Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. Nat. Genet. 54, 817–826 (2022)."). We identified 530 and 568 genes with dynamic eQTL effects (deGenes) across modules 1 and 2 from robust candidates, respectively ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). Of these deGenes, 352 (66.4%) and 393 (69.1%) showed dynamic eQTL effects in both linear and quadratic models, and 134 (25.3%) and 117 (20.6%) did only in the quadratic model for modules 1 and 2 (Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02266-3#Fig13)). Moreover, more than half of deGenes (324 in module 1 and 362 in module 2) were module specific (Fig. [4b](https://www.nature.com/articles/s41588-025-02266-3#Fig4)), and the enriched pathways of module-specific deGenes were related to innate immunity in module 1 and antigen presentation in module 2 (Fig. [4c](https://www.nature.com/articles/s41588-025-02266-3#Fig4)). We confirmed that the model was well calibrated by 1,000 permutations (Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02266-3#Fig13) and the [Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)).

**Fig. 4: Dynamic eQTL analysis in the myeloid cluster.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/4)

**a**, Overview of dynamic eQTL analysis. We calculated the two module scores using a gene set termed ‘HALLMARK\_INFLAMMATORY\_RESPONSE’ and ‘GOBP\_RESPONSE\_TO\_ INTERFERON\_GAMMA’ ([GO:0034341](http://amigo.geneontology.org/amigo/term/GO:0034341)), respectively (left). Cells were split into ten windows of equal cell numbers according to each module score (middle). The figure design is based on a previous report[8](https://www.nature.com/articles/s41588-025-02266-3#ref-CR8 "Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. Nat. Genet. 54, 817–826 (2022)."). A linear-and-quadratic mixed model was applied to test for an interaction between genotypes and module scores by the pseudobulk approach. The single-cell negative binomial mixed-effect (NBME) model was used to identify cell state-dependent regulatory effects (right). Module 1, M1; module 2, M2. **b**, Number of eGenes with a significant genotype–module interaction (that is, dynamic eGenes) in a linear or quadratic mixed model for the two modules. **c**, Top ten biological processes by gene ontology (GO) enriched in dynamic eGenes are shown for each test. MHC, major histocompatibility antigen; _P_adj, adjusted _P_ value. **d**, Heatmap depicting the enrichment of dynamic eQTLs in promoter or enhancer regions of eight representative immune cells from the Roadmap project for each combination of module and analysis model. **e**, Forest plots showing ORs of the overlap of dynamic eQTLs (_n_ = 396–510; Supplementary Table [15](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)) with functional regions in monocytes from Roadmap compared to _cis_\-eQTLs in monocytes at L1 (_n_ = 3,175) for each combination of functional region and analysis model in two modules. Dots represent ORs, and bars represent 95% confidence intervals (CIs). **f**, Top four biological processes by GO enriched in dynamic eGenes, where dynamic eQTLs, but not cis-eQTLs, are located in functional regions of monocytes from Roadmap, are shown for each module-function combination (four tests in total). **g**, UMAPs represent the cell state-dependent eQTL strength (_β_total) for each cell calculated as a sum of the effect sizes of genotype and genotype (G) × harmonized PCs (hPCs). Labeled _P_ values are derived from NBME and pseudobulk analysis. Heatmap showing _P_ values for each genotype–hPC interaction in the full model of NBME analysis. Two-sided _P_ values are uncorrected in **e**,**g**. Dot color indicates statistical significance of the enrichment (adjusted _P_ values via the Benjamini–Hochberg method), and dot size represents the gene ratio assigned to each term in **c**,**f**.

We annotated dynamic eQTLs using the Roadmap data[35](https://www.nature.com/articles/s41588-025-02266-3#ref-CR35 "Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317–329 (2015).") as we did for _cis_\-eQTLs. Dynamic eQTLs showed monocyte-specific enrichment in both promoters and enhancers (Fig. [4d](https://www.nature.com/articles/s41588-025-02266-3#Fig4) and Supplementary Table [16](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)) but distinctly from that of _cis_\-eQTLs (Fig. [2e](https://www.nature.com/articles/s41588-025-02266-3#Fig2)). Next, we compared enrichment of dynamic eQTLs and _cis_\-eQTLs in monocytes (L1) in promoters and enhancers of monocytes. In both modules, dynamic eQTLs were more enriched for enhancers and less for promoters than _cis_\-eQTLs (Fig. [4e](https://www.nature.com/articles/s41588-025-02266-3#Fig4) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). Similar results were observed in the comparison of dynamic eQTLs and _cis_\-eQTLs in subclusters (L2) of the myeloid cluster (for example, plasmacytoid dendritic cells; Extended Data Fig. [5c](https://www.nature.com/articles/s41588-025-02266-3#Fig13)). We investigated deGenes for which dynamic eQTLs were located in functional regions, but _cis_\-eQTLs of any cell type of the myeloid cluster were not ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). Pathway enrichment analysis of such deGenes across modules and functional categories indicated that pathways related to Toll-like receptor were enriched in module 1 and those related to antigen presentation were enriched in module 2, and their enrichment pattern varied between promoters and enhancers within each module (Fig. [4f](https://www.nature.com/articles/s41588-025-02266-3#Fig4)).

### Modeling dynamic eQTLs at single-cell resolution

We performed dynamic eQTL analysis at single-cell resolution (Fig. [4a](https://www.nature.com/articles/s41588-025-02266-3#Fig4) and the [Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). As an example, we observed the strong cell state-dependent eQTL effect of [rs11080327](https://www.ncbi.nlm.nih.gov/snp/?term=rs11080327) for _SLFN5_ specifically at specific states within classical monocytes (cMono; Fig. [4g](https://www.nature.com/articles/s41588-025-02266-3#Fig4)), capturing the gene regulatory mechanism with higher resolution than previously reported[7](https://www.nature.com/articles/s41588-025-02266-3#ref-CR7 "Perez, R. K. et al. Single-cell RNA-seq reveals cell type-specific molecular and genetic associations to lupus. Science 376, eabf1970 (2022)."). Among 15 hPCs, hPC14, which represented a cell state related to the type 1 interferon pathway, showed the most significant cell state interaction (Fig. [4g](https://www.nature.com/articles/s41588-025-02266-3#Fig4) and Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02266-3#Fig13)), consistent with results in the dynamic eQTL analysis with the pseudobulk approach. Similarly, eQTLs for _NFKBIZ_ and _IFITM2_ ([rs9818678](https://www.ncbi.nlm.nih.gov/snp/?term=rs9818678), [rs741738](https://www.ncbi.nlm.nih.gov/snp/?term=rs741738)) exhibited significant cell state dependence, but their cell state interactions were different (Fig. [4g](https://www.nature.com/articles/s41588-025-02266-3#Fig4)). Expanding eQTL mapping into single-cell resolution should yield a more granular picture of dynamic genetic regulation.

### Colocalization of GWAS variants and sc-eQTLs

To better understand the genetic regulatory mechanisms of GWAS loci, we assessed colocalization of GWAS signals from 13 complex traits in an East Asian population and our eQTL signals mapped per cell types in L1 and L2. We discovered GWAS–eQTL colocalization events (PP.H4 (posterior probability of shared causal variant) > 0.8) at 121 GWAS loci and prioritized 179 candidate trait-associated genes (Fig. [5a](https://www.nature.com/articles/s41588-025-02266-3#Fig5) and Supplementary Table [19](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). About half of these GWAS loci (55 of 121) showed colocalization within only one major cell type (Fig. [5a](https://www.nature.com/articles/s41588-025-02266-3#Fig5)), and most prioritized trait-associated genes were trait specific (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02266-3#Fig14)). Cell types that exhibited colocalization were prominently specific to the traits (Fig. [5a](https://www.nature.com/articles/s41588-025-02266-3#Fig5) and Extended Data Fig. [6b](https://www.nature.com/articles/s41588-025-02266-3#Fig14)).

**Fig. 5: Interpretation of GWAS results using OASIS data.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/5)

**a**, Number of GWAS loci significantly colocalized with an eQTL for each cell type–trait combination (left). Cell types of the major cell type (L1) are colored red. Number of GWAS loci colocalized with an eQTL and how many times the colocalization was shared among major cell types for each trait (right). GWAS abbreviations: CD, Crohn’s disease; Hosp-COV, hospitalized COVID-19; HT, hyperthyroidism; IBD, inflammatory bowel diseases; LYM, lymphocyte count; MON, monocyte count; RA, rheumatoid arthritis; T1D, type 1 diabetes; UC, ulcerative colitis. **b**, Dynamic eQTLs and _cis_\-eQTLs per cell type of [rs2841281](https://www.ncbi.nlm.nih.gov/snp/?term=rs2841281) for _PLD4_. Dot size represents PP.H4 of eQTL–GWAS colocalizations. **c**, Regional association plots of systemic lupus erythematosus GWAS, _cis_\-eQTL of monocytes (L1) and dynamic eQTLs in bin 6 of module 1 for the _PLD4_ locus. Chr, chromosome. **d**, UMAP represents cell state-dependent eQTL strength (_β_total) for each cell calculated as a sum of the effect sizes of genotype and genotype × hPCs. Labeled _P_ values are derived from NBME analysis. **e**, Dynamic eQTLs and _cis_\-eQTLs per cell type of [rs2836884](https://www.ncbi.nlm.nih.gov/snp/?term=rs2836884) for _ETS2_. **f**, Regional association plots of ulcerative colitis GWAS, _cis_\-eQTL of monocytes (L1) and dynamic eQTL in bin 6 of module 1 for the _ETS2_ locus. **g**, UMAP represents cell state-dependent eQTL strength (_β_total) for each cell. Uncorrected _P_ values from two-sided tests are shown in **b**–**g**. **h**, The effect of hospitalized COVID-19 PRS on transcriptomic and proteomic levels separately in patients with COVID-19 and healthy individuals. One-sided _P_ values were calculated with 1,000 permutations for the number of differentially expressed genes (DEGs) and differentially expressed proteins (DEPs).

We next jointly colocalized dynamic eQTLs with GWAS loci. As an example, [rs2841281](https://www.ncbi.nlm.nih.gov/snp/?term=rs2841281), the lead SNP for systemic lupus erythematosus GWAS in the _PLD4_ locus, had dynamic eQTL effect only with module 1 (Fig. [5b](https://www.nature.com/articles/s41588-025-02266-3#Fig5)). While this variant moderately colocalized with some _cis_\-eQTLs per cell type, it colocalized more strongly with dynamic eQTLs in two bins of module 1 (posterior probability of shared causal variant (PP.H4) = 0.95 for bin 6, 0.92 for bin 7; Fig. [5b,c](https://www.nature.com/articles/s41588-025-02266-3#Fig5)). In addition, single-cell-resolution eQTL modeling revealed the strongest cell state-dependent eQTL effects in the boundary region between IL1B classical monocytes (cMonoIL1B) and S100A classical monocytes (Fig. [5d](https://www.nature.com/articles/s41588-025-02266-3#Fig5)). As another example, [rs2836884](https://www.ncbi.nlm.nih.gov/snp/?term=rs2836884), the lead SNP for ulcerative colitis GWAS in the _ETS2_ locus, had a more dynamic eQTL effect with module 1 than module 2 (Fig. [5e](https://www.nature.com/articles/s41588-025-02266-3#Fig5)). This variant showed nearly perfect colocalization with dynamic eQTLs in bin 6 of module 1 (PP.H4 = 0.99; Fig. [5e,f](https://www.nature.com/articles/s41588-025-02266-3#Fig5)) and had heterogeneous cell state-dependent eQTL effects even within the cell types exhibiting significant colocalization (Fig. [5g](https://www.nature.com/articles/s41588-025-02266-3#Fig5)). These observations suggest the importance of considering dynamic eQTLs to interpret GWAS signals.

### Context- and cell type-specific effects of polygenic risks

Beyond single variants, we exhibited dynamic gene regulation in a genome-wide manner. We further studied how PRSs affect transcriptomic and proteomic profiles across clinical status and/or cell types. We constructed PRSs of hospitalized patients with COVID-19 using PRS-CSx[45](https://www.nature.com/articles/s41588-025-02266-3#ref-CR45 "Ruan, Y. et al. Improving polygenic prediction in ancestrally diverse populations. Nat. Genet. 54, 573–580 (2022)."), combining the summary statistics of hospitalized COVID-19 in Europeans[46](https://www.nature.com/articles/s41588-025-02266-3#ref-CR46 "The COVID-19 Host Genetics Initiative. A second update on mapping the human genetic architecture of COVID-19. Nature 621, E7–E26 (2023).") and those in Japanese individuals[47](https://www.nature.com/articles/s41588-025-02266-3#ref-CR47 "Namkoong, H. et al. DOCK2 is involved in the host genetics and biology of severe COVID-19. Nature 609, 754–760 (2022)."),[48](https://www.nature.com/articles/s41588-025-02266-3#ref-CR48 "Wang, Q. S. et al. The whole blood transcriptional regulation landscape in 465 COVID-19 infected samples from Japan COVID-19 Task Force. Nat. Commun. 13, 4830 (2022).") as the training data ([Methods](https://www.nature.com/articles/s41588-025-02266-3#Sec15)). The phenotypic variance (Nagelkerke’s _R__2_) explained by our PRS in OASIS was 4.1%, confirming that the PRS had reasonable accuracy. We divided this PRS into four quantiles and performed differential gene and protein expression analysis with quantiles of PRSs across clinical status and cell types. We identified differentially expressed genes (DEGs) in monocytes (_n_ = 21) and CD8+ T cells (_n_ = 2) from COVID-19 scRNA-seq and differentially expressed proteins (DEPs) (_n_ = 184) from the COVID-19 proteome (_P_ < 0.045 by 1,000 permutations), whereas no DEGs or DEPs were identified in healthy individuals (Fig. [5h](https://www.nature.com/articles/s41588-025-02266-3#Fig5) and Extended Data Fig. [7a,b](https://www.nature.com/articles/s41588-025-02266-3#Fig15)). We confirmed that DEGs and DEPs specific to patients did not directly reflect the DEGs and DEPs between patients and controls (Extended Data Fig. [7c,d](https://www.nature.com/articles/s41588-025-02266-3#Fig15)). We also investigated the effect of PRSs derived from other trait GWASs and found that their effects were context and cell type specific (Extended Data Fig. [7e](https://www.nature.com/articles/s41588-025-02266-3#Fig15)). These findings indicate that PRSs, like single germline variants, might affect transcriptional and proteomic profiles in a context-specific and/or cell type-specific manner.

### Single-cell profiling of mCAs

To expand our mQTL catalogs to somatic genetic variations, we conducted single-cell deconvolution of a variety of somatic events detected with genomics data (Fig. [6a](https://www.nature.com/articles/s41588-025-02266-3#Fig6)). Using SNP array data[14](https://www.nature.com/articles/s41588-025-02266-3#ref-CR14 "Loh, P., Genovese, G. & McCarroll, S. A. Monogenic and polygenic inheritance become instruments for clonal selection. Nature 584, 136–141 (2020)."),[49](https://www.nature.com/articles/s41588-025-02266-3#ref-CR49 "Loh, P. R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."), we detected eight copy number alterations (CNAs) and six copy-neutral losses of heterozygosity (CN-LOHs) in seven and six samples, respectively (Supplementary Table [20](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). By using the individuals’ mCAs as prior information and applying Numbat[50](https://www.nature.com/articles/s41588-025-02266-3#ref-CR50 "Gao, T. et al. Haplotype-aware analysis of somatic copy number variations from single-cell transcriptomes. Nat. Biotechnol. 41, 417–426 (2023).") to the scRNA-seq data, we could distinguish the mutant cells from their wild-type counterparts in each sample, except for one patient with COVID-19 containing a shorter CNA (CH12; Supplementary Table [20](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). Because the detection of clone cells with CN-LOH depends on germline SNP alleles embedded in the scRNA-seq raw reads, an increase of such scRNA-seq-based SNP information improves the sensitivity. We thus performed deep and long sequencing by scaling target depths from 20,000 to 100,000 reads per cell and extending a read 2 length from 90 to 270 bp. Consequently, we profiled 1.8-fold SNPs and identified 2.1-fold more mutated cells with CN-LOH than with normal sequencing conditions (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02266-3#Fig16)). While cells with CN-LOH were moderately enriched in other T cells, strong cell type specificity of cells with CNA was observed in two patients with COVID-19 (CH01 and CH05; Fig. [6b](https://www.nature.com/articles/s41588-025-02266-3#Fig6), Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02266-3#Fig16) and Supplementary Table [21](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). These two patients also showed relatively larger fractions of mutant cells (median of 12.3% for three CNAs in CH01 and CH05 versus 3.06% for the other four CNAs; Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02266-3#Fig16)).

**Fig. 6: Single-cell deconvolution of mCAs.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/6)

**a**, Schematic overview of single-cell deconvolution of CH including mCAs and LOY by integrating SNP array and scRNA-seq data. **b**, Heatmaps showing in-sample ORs of each cell type containing cells with CNAs (left) and CN-LOHs (right). **c**, UMAP embedding of CH01 scRNA-seq data colored by three clones. **d**, Top ten enriched biological pathways of upregulated DEGs in monocytes of CH01 with 1p loss. Dot color indicates statistical significance of the enrichment (adjusted _P_ values via the Benjamini–Hochberg method), and dot size represents the gene count assigned to each term. **e**, UMAP embedding of CH05 scRNA-seq data colored by two clones. **f**, Network plots showing the similarity of complementarity-determining region 3 (CDR3) amino acid sequences in BCR heavy and light chains of CH05 colored by clone (left) and isotype (right). Clonotype clusters with clonal size >1 are selected. **g**, Reactivity of antibodies against SARS-CoV-2 antigens (Ag) in enzyme-linked immunosorbent assays. Dots denote mean, and error bars show s.d. measured in triplicate. S309, anti-SARS-CoV-2 S immunoglobulin G (IgG)1; CH05, recombinant antibody derived from the CH05 BCR clonotype with 17q gain; nCoV396, anti-SARS-CoV-2 N IgG1; 23B12, anti-_Candida albicans_ IgG1; OD450, optical density at 450 nm. Panel **a** created with [BioRender.com](https://www.biorender.com/).

In CH01, two clones with different CNAs were enriched in monocytes (1p loss, odds ratio (OR) = 6.6 for monocytes; 15q gain, OR = 7.2 for monocytes; Fig. [6b,c](https://www.nature.com/articles/s41588-025-02266-3#Fig6)). To characterize these clones, we assessed DEGs between mutant and normal cells in monocytes (Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02266-3#Fig16)). Most of the downregulated genes in monocytes with 1p loss and the upregulated genes in monocytes with 15q gain were located within the altered chromosomal regions themselves (that is, _cis_), highlighting accurate detection of mutant cells. But some showed _trans_\-chromosomal DEGs including upregulation of _TNFAIP3_, one of the elements of the COVID-19-specific immune response[51](https://www.nature.com/articles/s41588-025-02266-3#ref-CR51 "Ahern, D. J. et al. A blood atlas of COVID-19 defines hallmarks of disease severity and specificity. Cell 185, 916–938 (2022)."), in monocytes with 1p loss (Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02266-3#Fig16)). We further evaluated the enriched pathways of DEGs between mutant and normal clones and found that immune-related pathways, such as positive regulation of cytokine production, were significant in mutant clones with 1p loss (Fig. [6d](https://www.nature.com/articles/s41588-025-02266-3#Fig6) and Extended Data Fig. [8d,e](https://www.nature.com/articles/s41588-025-02266-3#Fig16)).

Mutant cells with 17q gain were strongly enriched in B cells in CH05 (OR = 350; Fig. [6b,e](https://www.nature.com/articles/s41588-025-02266-3#Fig6)). Pathway enrichment analysis of DEGs between mutant and normal B cells showed upregulation of immune-related pathways and downregulation of response to steroid hormones in mutant cells (Extended Data Fig. [8c,f,g](https://www.nature.com/articles/s41588-025-02266-3#Fig16)). This suggests that these mutant cells may reduce the effectiveness of systemic corticosteroid therapy and result in a worse prognosis. Furthermore, our integrated analysis of scVDJ-seq data identified a considerably large BCR clonotype with 17q gain in CH05 (clone size = 650; Fig. [6f](https://www.nature.com/articles/s41588-025-02266-3#Fig6)). This BCR clonotype was the largest clonotype among all the samples (median of maximum clone size in each sample = 5) and was mainly composed of naive B cells, which differed from most expanded clonotypes composed of class-switched PBs (Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02266-3#Fig17)). Using recombinant antibodies derived from this expanded BCR clonotype, we assessed their reactivity against SARS-CoV-2 major antigens. Of interest, the BCR clonotype did not react with any tested antigens (Fig. [6g](https://www.nature.com/articles/s41588-025-02266-3#Fig6) and Extended Data Fig. [9b](https://www.nature.com/articles/s41588-025-02266-3#Fig17)), not supporting clonal expansion through normal antibody response to SARS-CoV-2 infection. These clonally expanded mutant B cells may have reduced diversity of the BCR repertoire in SARS-CoV-2 infection, which could negatively impact the antibody response of this patient. Our deconvolution of mCAs at single-cell resolution has the potential to elucidate the impacts of somatic mutations on immune cell functions and the development of severe infections.

### Monocyte-specific accumulation of LOY

Next, we evaluated the biological effects of LOY on the immune system. Using scRNA-seq data from male samples (_n_ = 149), we defined cells with LOY as cells without expression from the male-specific region of the Y chromosome[15](https://www.nature.com/articles/s41588-025-02266-3#ref-CR15 "Thompson, D. J. et al. Genetic predisposition to mosaic Y chromosome loss in blood. Nature 575, 652–657 (2019)."),[22](https://www.nature.com/articles/s41588-025-02266-3#ref-CR22 "Bruhn-Olszewska, B. et al. Loss of Y in leukocytes as a risk factor for critical COVID-19 in men. Genome Med. 14, 139 (2022)."), which enabled quantitative estimation of the LOY status of each male. Older men showed larger fractions of cells with LOY. Six male samples with LOY detected using SNP array data (that is, genotype-based binary LOY estimation) contained relatively more cells with LOY (median of 14.2% versus 3.3%; Fig. [7a](https://www.nature.com/articles/s41588-025-02266-3#Fig7)). The single-cell-based approach was thus more sensitive to detect LOY and provide biological insights. For single-cell-based LOY, all male samples were classified as LOY high (LOY cells > 5%; _n_ = 36) or LOY low (LOY cells < 5%; _n_ = 113). Using this classification, we performed a regression analysis to answer controversial discussions on whether LOY is associated with COVID-19 risk[22](https://www.nature.com/articles/s41588-025-02266-3#ref-CR22 "Bruhn-Olszewska, B. et al. Loss of Y in leukocytes as a risk factor for critical COVID-19 in men. Genome Med. 14, 139 (2022)."). While genotype-based LOY did not show an independent impact, single-cell-based LOY was significantly associated with the risk of hospitalized COVID-19 (OR = 6.6, 95% confidence interval = 1.4–30.2 after age adjustment; Fig. [7b](https://www.nature.com/articles/s41588-025-02266-3#Fig7)).

**Fig. 7: Biological effects of mosaic LOY.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/7)

**a**, Fractions of cells with LOY in each male sample sorted by age (_x_ axis). Yellow triangles denote samples with LOY detected using SNP array data. **b**, Forest plot of associations between LOY and COVID-19 risk. ORs of hospitalized COVID-19 were calculated using univariate logistic regression and multivariate logistic regression to account for the effect of age. Dots represent ORs, and bars represent 95% confidence intervals. We used uncorrected _P_ values from two-sided tests. **c**, UMAP embeddings of PBMCs from patients with COVID-19 (left) and healthy individuals (right) colored according to the presence of LOY. **d**, Heatmaps showing ORs across samples of each cell type containing cells with LOY in patients with COVID-19 and healthy individuals. **e**, Enriched biological pathways of upregulated DEGs in cells with LOY compared to normal cells with cMonoIL1B cells of patients with COVID-19 and healthy individuals. Dot color indicates statistical significance of the enrichment (adjusted _P_ values via the Benjamini–Hochberg method), and dot size represents the gene count assigned to each term. **f**, Beeswarm plot and box plot describing the distribution of adjusted log2 (fold change (FC)) values between LOY-high and LOY-low samples from patients with COVID-19 in neighborhoods (_n_ = 12,141) from L2 cell types. Nodes are neighborhoods, colored by their log2 (FC) adjusted by age and sequencing group. Boxes denote the IQR, medians are shown as vertical bars, and whiskers extend to 1.5 times the IQR.

Cells with LOY were enriched in monocytes across patients with COVID-19 (OR = 4.0) and healthy individuals (OR = 4.5; Fig. [7c,d](https://www.nature.com/articles/s41588-025-02266-3#Fig7) and Supplementary Table [22](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). Upregulated DEGs of cMonoIL1B cells with LOY were significantly enriched in T cell-related pathways (Fig. [7e](https://www.nature.com/articles/s41588-025-02266-3#Fig7) and Extended Data Fig. [10a](https://www.nature.com/articles/s41588-025-02266-3#Fig18)). Next, we performed a differential abundance analysis between LOY-high (_n_ = 32) and LOY-low (_n_ = 33) samples from patients with COVID-19 using Milo[37](https://www.nature.com/articles/s41588-025-02266-3#ref-CR37 "Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. Nat. Biotechnol. 40, 245–253 (2022)."). We found an increase in monocytes and dendritic cells and a decrease in naive T cells for LOY-high samples (Fig. [7f](https://www.nature.com/articles/s41588-025-02266-3#Fig7)), and a similar trend was observed in healthy individuals (Extended Data Fig. [10b](https://www.nature.com/articles/s41588-025-02266-3#Fig18)). Comparison of cell proportions between the two groups suggested that a decrease in naive T cells was prominent in CD4+ T cells (_P_ = 0.013) and the proportion of regulatory T cells was significantly higher in LOY-high patients with COVID-19 (_P_ = 7.4 × 10−4; Extended Data Fig. [10c](https://www.nature.com/articles/s41588-025-02266-3#Fig18)). In aggregate, these data suggest that monocytes with LOY might affect the immune response via compositional changes in T cells.

### Context-specific enrichment of mitochondrial heteroplasmy

We also investigated the landscape of mt-heteroplasmy in immune cells by integrating our genomics and scRNA-seq data (Fig. [8a](https://www.nature.com/articles/s41588-025-02266-3#Fig8)). The Genome Analysis Toolkit (GATK)-based pipeline[52](https://www.nature.com/articles/s41588-025-02266-3#ref-CR52 "Laricchia, K. M. et al. Mitochondrial DNA variation across 56,434 individuals in gnomAD. Genome Res. 32, 569–582 (2022).") detected 36 (40.9%) patients with COVID-19 and 63 (42.9%) healthy individuals with mt-heteroplasmy (variant allele frequency (VAF) > 0.1) using WGS data (Fig. [8b](https://www.nature.com/articles/s41588-025-02266-3#Fig8)). Referring to allelic information of the predefined WGS-based heteroplasmy embedded in the scRNA-seq raw reads, we could successfully perform clonal assignment of each cell for seven patients with COVID-19 and four healthy individuals. The proportion of cells with mt-heteroplasmy in scRNA-seq reads was strongly associated with VAFs of mt-heteroplasmy in WGS data except for m.813A>G (Fig. [8c](https://www.nature.com/articles/s41588-025-02266-3#Fig8)). We evaluated cell type specificity of heteroplasmic cells for these samples, except for one healthy sample in which almost all cells had mt-heteroplasmy. We found that the cells with mt-heteroplasmy were enriched in monocytes and dendritic cells specifically for patients with COVID-19 (Fig. [8d](https://www.nature.com/articles/s41588-025-02266-3#Fig8)). Of note, the heteroplasmy of m.813A>G, with a large discrepancy between the heteroplasmic cell fraction and VAF, showed stronger cell type specificity (OR = 13.3 for monocytes, OR = 6.8 for dendritic cells; Supplementary Table [23](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). Given its location in 12S ribosomal RNA, the mutation hotspot for aminoglycoside ototoxicity that is implicated in human disease[53](https://www.nature.com/articles/s41588-025-02266-3#ref-CR53 "Zou, T. et al. Impacts of impaired mitochondrial dynamics in hearing loss: potential therapeutic targets. Front. Neurosci. 16, 998507 (2022)."), this heteroplasmy might affect cell proliferation or viability in a cell type-specific manner, causing abnormal mitochondrial function. This observation highlighted the advantage of our multi-layered omics focusing on both the mutation itself (that is, genomics) and mutated cells (that is, scRNA-seq). Although we did not detect any DEGs to distinguish heteroplasmic cells from normal ones (false discovery rate (FDR) < 0.05), COVID-19-specific enrichment suggested that mt-heteroplasmy may underlie the biological mechanisms of COVID-19.

**Fig. 8: The landscape of mitochondrial heteroplasmy.**

[![figure 8](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_Fig8_HTML.png)](https://www.nature.com/articles/s41588-025-02266-3/figures/8)

**a**, Schematic overview of single-cell deconvolution of mt-heteroplasmy by integrating WGS and scRNA-seq data. **b**, Fractions of samples with mt-heteroplasmy detected using WGS data by mtDNA region. Mitochondrial gene positions are indicated in the Circos plot. rRNA, ribosomal RNA. **c**, Correlations between the fractions of cells with heteroplasmy in scRNA-seq data and VAFs of heteroplasmy in WGS data. **d**, Heatmap showing in-sample ORs of each cell type containing cells with mt-heteroplasmy. Panel **a** created with [BioRender.com](https://www.biorender.com/).

Discussion
----------

Here, we constructed a single-cell data atlas of >1.5 million PBMCs with multi-layer omics in Japanese individuals. Our integrative analyses of single-cell data revealed that immune cells were dynamically regulated in a cell state-dependent manner defined by multiomics profiles, leading to a better understanding of the pathogenesis of COVID-19 and autoimmune diseases.

We comprehensively mapped germline genetic effects on gene expression in immune cells and observed that eQTLs were dynamically regulated across continuous cell states. We also elucidated HLA and genome-wide regulation of TCR and BCR repertoires in a cell type- and context-specific manner. Thus, including cohorts of diseased and healthy individuals contributes to including cells with a variety of biological conditions. The epigenetic properties of dynamic eQTLs were distinct from those of _cis_\-eQTLs, and colocalization analysis highlighted the value of considering genetic regulation of gene expression across continuous cell states to understand GWAS signals. In addition, as in the example of [rs11080327](https://www.ncbi.nlm.nih.gov/snp/?term=rs11080327) for _SLFN5_, single-cell resolution eQTL modeling allowed us to interpret genetic regulatory mechanisms at finer resolution than bulk and pseudobulk anlaysis[7](https://www.nature.com/articles/s41588-025-02266-3#ref-CR7 "Perez, R. K. et al. Single-cell RNA-seq reveals cell type-specific molecular and genetic associations to lupus. Science 376, eabf1970 (2022)."),[33](https://www.nature.com/articles/s41588-025-02266-3#ref-CR33 "Ota, M. et al. Dynamic landscape of immune cell-specific gene regulation in immune-mediated diseases. Cell 184, 3006–3021 (2021)."). Moreover, single-cell technology finely resolved associations between human disease-related gut bacteria and peripheral immune cell abundance.

We deconvoluted a variety of somatic events at single-cell resolution and showed their heterogeneous functional impacts on immune cells, which increased our insights into the underlying mechanisms of COVID-19 severity. As in previous studies of a single gene mutation in hematopoiesis[23](https://www.nature.com/articles/s41588-025-02266-3#ref-CR23 "Nam, A. S. et al. Single-cell multi-omics of human clonal hematopoiesis reveals that DNMT3A R882 mutations perturb early progenitor states through selective hypomethylation. Nat. Genet. 54, 1514–1526 (2022).") or mCAs in the normal adrenal gland[24](https://www.nature.com/articles/s41588-025-02266-3#ref-CR24 "Gao, T. et al. A pan-tissue survey of mosaic chromosomal alterations in 948 individuals. Nat. Genet. 55, 1901–1911 (2023)."), capturing the mutational status of individual cells has enabled us to compare mutant and normal cells within individuals, which is easier when the number of cells per sample is large. Investigating apparently healthy samples or tissues showing one or a few somatic mutations can directly reveal the phenotypic effects of the mutation, and that is not the case for cancer cells with a considerably higher burden of mutations. Of note, some observations of mCAs and mt-heteroplasmy in this study may be a result of the immunologically strong stimulus (that is, SARS-CoV-2 infection), implying the need for a study design that takes context into account.

In conclusion, we demonstrate the importance of interpreting biological phenomena by integrating features from multi-layer omics at single-cell resolution. This resource in an East Asian population will contribute to equity in global diversity of human genomics and help researchers better understand complex human traits.

Methods
-------

### Ethics and specimen collection of OASIS samples

Peripheral blood samples were obtained from patients with COVID-19 (_n_ = 88) and healthy controls (_n_ = 147) at Osaka University Hospital. Patients with COVID-19 were further categorized into moderate (_n_ = 9) and severe (_n_ = 79) groups according to disease severity based on the highest score on the World Health Organization Ordinal Scale for Clinical Improvement ever present[54](https://www.nature.com/articles/s41588-025-02266-3#ref-CR54 "World Health Organization. COVID-19 Therapeutic Trial Synopsis (World Health Organization, 2020)."). Detailed clinical data are summarized in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02266-3#MOESM3). Some of the participants (_n_COVID-19 = 73, _n_control = 75) are described elsewhere[44](https://www.nature.com/articles/s41588-025-02266-3#ref-CR44 "Edahiro, R. et al. Single-cell analyses and host genetics highlight the role of innate immune cells in COVID-19 severity. Nat. Genet. 55, 753–767 (2023)."),[47](https://www.nature.com/articles/s41588-025-02266-3#ref-CR47 "Namkoong, H. et al. DOCK2 is involved in the host genetics and biology of severe COVID-19. Nature 609, 754–760 (2022)."). There was one pair of blood relatives among the healthy participants; therefore one of them was excluded from the eQTL analysis. One patient with COVID-19 had a karyotype abnormality and was excluded from the X chromosome eQTL analysis and differential gene and protein expression analyses. This study strictly follows the principles of the Declaration of Helsinki, with written informed consent obtained from all participants before sample collection according to regular principles. Ethical approvals were obtained from the Ethics Committee of Osaka University (approval no. 734). There was no compensation for participants.

### Sequencing, alignment, quantification and quality control of scRNA-seq

Single-cell suspensions were processed through the 10x Genomics Chromium Controller. Droplet libraries were processed using Cell Ranger 5.0.0 (10x Genomics). Sequencing reads were aligned with STAR (version 2.7.2a)[55](https://www.nature.com/articles/s41588-025-02266-3#ref-CR55 "Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013).") using the GRCh38 human reference genome. Please see the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1) for details.

### Integration and manual annotation of scRNA-seq

Details are described in the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### Whole-genome sequencing data processing

DNA samples isolated from whole blood were sequenced at Macrogen Japan. DNA quantity was measured by PicoGreen, and DNA degradation was assessed by gel electrophoresis. All libraries were constructed using the TruSeq DNA PCR-Free Library Preparation Kit according to the manufacturer’s protocols. Libraries were sequenced on the HiSeq X or NovaSeq 6000 system (Illumina), producing paired-end reads 2 × 150 bp in length at an average depth of 19.4×. Sequenced reads were aligned against the Genome Reference Consortium human genome build 38 using BWA-MEM with the ALT-aware mode (version 0.7.17). For more details, please see the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### SNP array genotyping

Genomic DNA was genotyped with the use of the Infinium Asian Screening Array (Illumina). This genotyping array was built using an East Asian reference panel including whole-genome sequences, designed for effectively capturing genetic variation in East Asian populations. To increase the sample size used for subsequent statistical haplotype phasing, the accuracy of which determines the sensitivity for mCA detection using MoChA (18 May 2022)[14](https://www.nature.com/articles/s41588-025-02266-3#ref-CR14 "Loh, P., Genovese, G. & McCarroll, S. A. Monogenic and polygenic inheritance become instruments for clonal selection. Nature 584, 136–141 (2020)."),[49](https://www.nature.com/articles/s41588-025-02266-3#ref-CR49 "Loh, P. R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."), we merged the publicly available genotyping data of 54,405 Japanese individuals generated by the BioBank Japan Project using the same genotyping array. For more details, please see the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### Mapping of single-cell _cis_\-eQTLs with the pseudobulk approach

First, we performed single-cell-level normalization using scran (version 1.18.5)[56](https://www.nature.com/articles/s41588-025-02266-3#ref-CR56 "Lun, A. T. L., Bach, K. & Marioni, J. C. Pooling across cells to normalize single-cell RNA sequencing data with many zero counts. Genome Biol. 17, 75 (2016).") separately for each major cell type (L1). We only kept genes with nonzero expression (unique molecular identifier (UMI) count > 0) in more than 1% of cells in each major cell type. In the analysis of subclusters (L2) within each major cell type, the same genes were retained. Gene expression per sample per cell type (that is, pseudobulk count) was calculated as the mean of log2\-transformed normalized expression across cells and then normalized across samples using an inverse normal transform in each cell type. Samples with more than ten cells in a cell type were considered in the analysis of the corresponding cell type (Supplementary Fig. [2b](https://www.nature.com/articles/s41588-025-02266-3#MOESM1)).

To identify _cis_\-eQTLs, we used tensorQTL (version 1.0.7)[57](https://www.nature.com/articles/s41588-025-02266-3#ref-CR57 "Taylor-Weiner, A. et al. Scaling computational genomics to millions of individuals with GPUs. Genome Biol. 20, 228 (2019).") to run a linear regression for each SNP–gene pair. We restricted our search to variants within 1 Mb of the transcription start site of each gene and with MAF > 0.05. The top 15 gene expression PCs, the top two genetic PCs, age, sex, 10x chemistry (version 1 or 2) and clinical status (COVID-19 or healthy) were used as covariates for eQTL analysis. To correct for the number of association tests performed per gene, we used a _cis_ permutation pass per gene with 1,000 permutations. Finally, to correct for the number of genes tested and identify significant eGenes, we performed a genome-wide _q_\-value correction for the top associated SNP–gene pair, setting a q-value threshold of 0.05. We subsequently mapped conditionally independent _cis_\-QTLs using the stepwise regression procedure with tensorQTL. Details on downsampling sample sizes and cell counts per sample in eQTL mapping are described in the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### Generation of genotypes and single-cell eQTL mapping in OneK1K

Details are described in the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### Comparison with _cis_\-eQTLs of OneK1K

We assessed eQTL overlaps and sharing between our cohort and OneK1K[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022).") using the multivariate adaptive shrinkage method[34](https://www.nature.com/articles/s41588-025-02266-3#ref-CR34 "Urbut, S. M., Wang, G., Carbonetto, P. & Stephens, M. Flexible statistical methods for estimating and testing effects in genomic studies with multiple conditions. Nat. Genet. 51, 187–195 (2019)."). For more details, please see the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### Epigenetic marker enrichment analysis for _cis_\-eQTLs

To reveal the functional characterization of _cis_\-eQTLs, we evaluated the enrichment of significant eQTLs to epigenome marks (that is, Roadmap annotation)[35](https://www.nature.com/articles/s41588-025-02266-3#ref-CR35 "Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317–329 (2015)."). The chromatin state data were obtained from the Roadmap Epigenomics project. We used 18-state models. We considered TssA, TssFlnk, TssFlnkU and TssFlnkD as promoter regions and EnhA1, EnhA2, EnhG1, EnhG2 and EnhWk as enhancer regions. For analysis of enhancer or promoter enrichment, we used eight immune cell subsets from peripheral blood in Roadmap project data (that is, primary monocytes (Roadmap ID E029), primary B cells (E032), primary helper T memory cells (E037), primary naive helper T cells (E038), primary regulatory T cells (E044), primary natural killer cells (E046), primary CD8+ naive T cells (E047) and primary CD8+ T memory cells (E048)). To assess the enrichment of primary _cis_\-eQTLs in epigenome marks, we compared the overlap with epigenome marks between primary significant _cis_\-eQTLs of each cell type and all the variants used in the analysis for that cell type. For more details, please see the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### Metagenomics analysis

Phenol–chloroform DNA extraction and subsequent metagenome shotgun sequencing were newly performed (dataset 3) or performed in previous studies (datasets 1 and 2)[58](https://www.nature.com/articles/s41588-025-02266-3#ref-CR58 "Tomofuji, Y. et al. Metagenome-wide association study revealed disease-specific landscape of the gut microbiome of systemic lupus erythematosus in Japanese. Ann. Rheum. Dis. 80, 1575–1583 (2021)."),[59](https://www.nature.com/articles/s41588-025-02266-3#ref-CR59 "Tomofuji, Y. et al. Prokaryotic and viral genomes recovered from 787 Japanese gut metagenomes revealed microbial features linked to diets, populations, and diseases. Cell Genom. 2, 100219 (2022)."). Details on metagenomics analysis are described in the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### TCR and BCR repertoire analysis

Droplet-based sequencing data for TCR sequences and BCR sequences were aligned and quantified using 5.0.0 (10x Genomics) against the GRCh38 human VDJ reference genome. Filtered annotated contigs for TCR sequences and BCR sequences were analyzed using Scirpy (version 0.10.0)[60](https://www.nature.com/articles/s41588-025-02266-3#ref-CR60 "Sturm, G. et al. Scirpy: a Scanpy extension for analyzing single-cell T-cell receptor-sequencing data. Bioinformatics 36, 4817–4818 (2020)."). For more details, please see the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1).

### HLA and genome-wide association analysis with TCR and BCR repertoires

For HLA association analysis, we performed HLA imputation for genotyped SNPs in the MHC region with the HLA reference panel of the Japanese population (_n_ = 1,118) constructed in a previous study[61](https://www.nature.com/articles/s41588-025-02266-3#ref-CR61 "Hirata, J. et al. Genetic and phenotypic landscape of the major histocompatibilty complex region in the Japanese population. Nat. Genet. 51, 470–480 (2019).") using DEEP\*HLA[62](https://www.nature.com/articles/s41588-025-02266-3#ref-CR62 "Naito, T. et al. A deep learning method for HLA imputation and trans-ethnic MHC fine-mapping of type 1 diabetes. Nat. Commun. 12, 1639 (2021)."),[63](https://www.nature.com/articles/s41588-025-02266-3#ref-CR63 "Naito, T. & Okada, Y. HLA imputation and its application to genetic and molecular fine-mapping of the MHC region in autoimmune diseases. Semin. Immunopathol. 44, 15–28 (2022)."). We targeted the amino acid variants of class I and II HLA genes (class I, _HLA-A_, _HLA-C_ and _HLA-B_; class II, _HLA-DRB1_, _HLA-DQA1_, _HLA-DQB1_, _HLA-DPA1_ and _HLA-DPB1_). Variants imputed with _an R_2 imputation quality score in tenfold cross-validation >0.7 and MAF > 0.05 were used for the analyses.

We explored the association between HLA variants and TRAV and TRBV gene usage in each cell type. We only used productive clones for which all the TCR genes were identified. We calculated the usage frequency of each TCR V gene in each cell type by summing up clones that were derived from that gene. We summed up clones regardless of the clone size, unless otherwise specified, to sensitively capture the central restriction between HLA and TCR[43](https://www.nature.com/articles/s41588-025-02266-3#ref-CR43 "Ishigaki, K. et al. HLA autoimmune risk alleles restrict the hypervariable region of T cell receptors. Nat. Genet. 54, 393–402 (2022)."). We only targeted TCR V genes that were observed in more than 80% of samples in cell types of interest.

We investigated the association between HLA variants and TCR V gene usage based on the Wald test for a negative binomial regression model using DESeq2 (version 1.30.1)[64](https://www.nature.com/articles/s41588-025-02266-3#ref-CR64 "Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014).") for each cell type. We included the same covariates as those used in the _cis_\-eQTL analysis, excluding PCs for gene expression, in the model. For the HLA association analysis, we empirically calculated the study-wide significance threshold using a permutation procedure[65](https://www.nature.com/articles/s41588-025-02266-3#ref-CR65 "Kanai, M., Tanaka, T. & Okada, Y. Empirical estimation of genome-wide significance thresholds based on the 1000 Genomes Project data set. J. Hum. Genet. 61, 861–866 (2016).") and obtained study-wide significance thresholds of _P_ = 1.1 × 10−5 and 1.2 × 10−5 for TRAV and TRBV genes, respectively. We also investigated the change in TCR V genes according to each variant under the condition of COVID-19 by adding an interaction term between COVID-19 status and each variant into the model. In this interaction analysis, we used the usage of TCR V genes weighted by their clone sizes to consider the effect of clonal expansion. We performed these analyses only for entire (that is, CD4+ T at L1), naive (CD4N) and CD4TCM and entire CD8+ (that is, CD8+ T at L1) T cells due to the sparsity of TCR V genes (Supplementary Fig. [8](https://www.nature.com/articles/s41588-025-02266-3#MOESM1)). Please see the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02266-3#MOESM1) for details on the significance threshold.

Additionally, we performed genome-wide association analysis for integrative features representing repertoire variation for TCRs and BCRs. Integrative features included the top five PCs for V(D)J gene usage and mean and coefficient of variation for the length of CDR3 sequences and the frequency of SHM. SHM was calculated using the R package SHazaM (version 1.1.0)[66](https://www.nature.com/articles/s41588-025-02266-3#ref-CR66 "Gupta, N. T. et al. Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data. Bioinformatics 31, 3356–3358 (2015)."). As potential biases in measuring SHM due to the complex structure of germline variants in VDJ genes could not be excluded, GWAS results of SHM should be interpreted carefully. For TRBD genes, we used _TRBD1_ gene usage instead of PCs because they only have _TRBD1_ and _TRBD2_. For BCR J genes, we used the top two PCs, as the number of these genes is fewer than six. The analysis was conducted based on a liner regression model for each feature normalized by rank-based inverse normal transformation with the same covariates using PLINK (version 2.00). For this analysis, we used _P_ = 2.0 × 10−9 and 1.3 × 10−9 for the significance thresholds of the TCR and BCR repertoires, respectively, on the basis of Bonferroni correction on the number of features for a genome-wide significance threshold of _P_ = 5.0 × 10−8. Due to sparsity, this analysis was applied only to CD4+ T, CD4N and CD4TCM and CD8+ T cells for TCRs and entire, type 1 naive, type 2 naive and memory B cells for BCRs.

### Dynamic eQTL analysis with the pseudobulk approach

To identify dynamic genetic regulation of gene expression along continuous cell states, we investigated dynamic eQTLs in the myeloid cluster (that is, monocytes and dendritic cells) across the two gene modules ‘HALLMARK\_INFLAMMATORY\_RESPONSE’ and ‘GOBP\_RESPONSE\_TO\_ INTERFERON\_GAMMA’. We downloaded the two gene sets from MSigDB. Module scores were evaluated at the single-cell level using the AddModuleScore() function implemented in Seurat with default parameters. We divided the gene modules into ten windows containing roughly equal numbers of cells and averaged the expression of each gene per sample within each window separately for the two gene modules (Fig. [4a](https://www.nature.com/articles/s41588-025-02266-3#Fig4)). Pseudobulk matrices per sample within each window with more than ten cells were retained in the analysis. Next, the averaged expression was normalized using inverse normal transformation, and gene expression PCs were calculated. To account for the higher correlation in expression values derived from the same individual at multiple gene module windows, we applied (1) linear and (2) quadratic mixed models, with individuals modeled as random intercepts as previously described[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022)."),[8](https://www.nature.com/articles/s41588-025-02266-3#ref-CR8 "Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. Nat. Genet. 54, 817–826 (2022)."). We used these models to test for a significant interaction between genotypes and module categories as follows:

$\\begin{array}{l}{\\rm{Exp}}={\\beta }\_{0}+{\\beta }\_{\\rm{g}}\\times {\\rm{genotype}}+{{\\beta }\_{\\rm{a}}\\times {\\rm{age}}+\\beta }\_{{\\rm{sex}}}\\times {\\rm{sex}}+{\\beta }\_{\\rm{v}}\\times {\\rm{version}}\\\\\\qquad\\;\\;+{\\beta }\_{{\\rm{status}}}\\times {\\rm{status}}+\\mathop{\\sum }\\limits\_{i=1}^{2}{\\beta }\_{\\rm{g{{PC}}}\_{\\it{i}}}\\times {\\rm{g{{PC}}}}\_{\\it{i}}+\\mathop{\\sum }\\limits\_{j=1}^{15}{\\beta }\_{\\rm{e{{PC}}}\_{\\it{j}}}\\times {\\rm{e{{PC}}}}\_{\\it{j}}\\\\\\qquad\\;\\;+{\\beta }\_{\\rm{m}}\\times {\\rm{module}}+{\\beta }\_{\\rm{g}\\times \\rm{m}}\\times {\\rm{genotype}}\\times {\\rm{module}}+\\left(1|{\\rm{sample}}\\right)\\end{array}$

(1)

$\\begin{array}{l}{\\rm{Exp}}={\\beta }\_{0}+{\\beta }\_{\\rm{g}}\\times {\\rm{genotype}}+{\\beta }\_{\\rm{a}}\\times {\\rm{age}}+{\\beta }\_{{\\rm{sex}}}\\times {\\rm{sex}}\\\\\\qquad\\;\\;+{\\beta }\_{\\rm{v}}\\times {\\rm{version}}+{\\beta }\_{{\\rm{status}}}\\times {\\rm{status}}\\\\\\qquad\\;\\; +\\mathop{\\sum }\\limits\_{i=1}^{2}{\\beta }\_{\\rm{{{gPC}}}\_{\\it{i}}}\\times {\\rm{gPC}}\_{i}+\\mathop{\\sum }\\limits\_{j=1}^{15}{\\beta }\_{\\rm{{{ePC}}}\_{\\it{j}}}\\times {\\rm{ePC}}\_{j}+{\\beta }\_{\\rm{m}}\\times {\\rm{module}}\\\\\\qquad\\;\\;+{\\beta }\_{{\\rm{m}}^{2}}\\times {{\\rm{module}}}^{2}\\\\\\qquad\\;\\;+{\\beta }\_{\\rm{g}\\times m}\\times {\\rm{genotype}}\\times {\\rm{module}}\\\\\\qquad\\;\\;+{\\beta }\_{\\rm{g}\\times {m}^{2}}\\times {\\rm{genotype}}\\\\\\qquad\\;\\;\\times {{\\rm{module}}}^{2}+\\left(1|{\\rm{sample}}\\right).\\end{array}$

(2)

gPC represents the genotype PC and ePC represents the gene expression PC. In both cases, the null model was computed using the same parameters while excluding the interaction terms of the genotype × module category and the genotype × module category2. _P_ values were calculated by comparing each model to its respective null model using a likelihood ratio test. All models were implemented in R using the lmer() function. To minimize multiple-testing burden and focus on more robust candidates, we only applied this approach to SNP–gene pairs identified as significant lead eQTL variants by tensorQTL in at least one module window separately for the two gene modules. Next, we determined the top SNP per gene for each model and module. Finally, to correct for the number of genes tested and identify significant dynamic eGenes (deGenes), the FDR was calculated via the Benjamini–Hochberg method for the top associated SNP–gene pairs, setting an FDR threshold of 0.05. We also performed the same analysis across ‘the type 1 ISG signature score (ISG score)’ (ref. [7](https://www.nature.com/articles/s41588-025-02266-3#ref-CR7 "Perez, R. K. et al. Single-cell RNA-seq reveals cell type-specific molecular and genetic associations to lupus. Science 376, eabf1970 (2022).")).

To find enriched pathways of module-shared and module-specific deGenes, we used the compareCluster function (fun = ‘enrichGO’, pvalueCutoff = 0.05, pAdjustMethod = ‘BH’, OrgDb = ‘org.Hs.eg.db’, ont = ‘BP’) of clusterProfiler (version 3.14.3)[67](https://www.nature.com/articles/s41588-025-02266-3#ref-CR67 "Yu, G., Wang, L.-G., Han, Y. & He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Omics 16, 284–287 (2012).").

### Epigenetic marker enrichment analysis for dynamic eQTLs

We evaluated the enrichment of significant dynamic eQTLs to epigenome marks[35](https://www.nature.com/articles/s41588-025-02266-3#ref-CR35 "Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317–329 (2015).") as we did for _cis_\-eQTLs. We compared the enrichment of significant dynamic eQTLs and _cis_\-eQTLs of the myeloid cluster (L1 and L2) in promoters and enhancers of monocytes in Roadmap separately for each model and module. For dynamic eQTLs, we only considered the top associated SNPs per gene analyzed in dynamic eQTLs for each model and module, and, for _cis_\-eQTLs, we considered the top associated SNPs per gene in each cell type.

We selected the genes with significant dynamic eQTLs located in promoters or enhancers of monocytes in Roadmap for each module and also selected the genes with significant _cis_\-eQTLs located in promoters and enhancers of monocytes among any cell type of the myeloid cluster (L1 and L2). Next, we identified genes with eQTLs located in functional regions specifically for dynamic eQTLs. To evaluate the enriched pathways of these genes for each module and functional region, we used the compareCluster function (fun = ‘enrichGO’, pvalueCutoff = 0.2, pAdjustMethod = ‘BH’, OrgDb = ‘org.Hs.eg.db’, ont = ‘BP’) of clusterProfiler (version 3.14.3)[67](https://www.nature.com/articles/s41588-025-02266-3#ref-CR67 "Yu, G., Wang, L.-G., Han, Y. & He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Omics 16, 284–287 (2012).").

### Dynamic eQTL analysis at single-cell resolution

We performed eQTL mapping at single-cell resolution using the NBME model[68](https://www.nature.com/articles/s41588-025-02266-3#ref-CR68 "Kang, J. B. et al. Mapping the dynamic genetic regulatory architecture of HLA genes at single-cell resolution. Nat. Genet. 55, 2255–2268 (2023)."). To represent the continuous cell state of each cell, we used hPCs calculated with Harmony (version 0.1)[69](https://www.nature.com/articles/s41588-025-02266-3#ref-CR69 "Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Nat. Methods 16, 1289–1296 (2019).") from the top 30 original PCs in the myeloid cluster. The negative binomial model was fitted with the following formula using the glmer.nb() function in the lme4 R library (version 1.1.32):

$\\begin{array}{l}{\\rm{Exp}}\\left({\\rm{UMI}}\\right)={\\beta }\_{0}+{\\beta }\_{\\rm{g}}\\times {\\rm{genotype}}+{\\beta }\_{\\rm{a}}\\\\\\qquad\\qquad\\quad\\;\\;\\times{\\rm{age}}+{\\beta }\_{{\\rm{sex}}}\\times {\\rm{sex}}+{\\beta }\_{\\rm{v}}\\times {\\rm{version}}+{\\beta }\_{{\\rm{status}}}\\times {\\rm{status}}\\\\\\qquad\\qquad\\quad\\;\\;+{\\beta }\_{{\\rm{UMI}}}\\times \\log {\\rm{UMI}}+{\\beta }\_{{\\rm{MT}}}\\times {\\rm{MT}}\\\\\\qquad\\qquad\\quad\\;\\;+\\mathop{\\sum }\\limits\_{i=1}^{2}{\\beta }\_{\\rm{{{gPC}}}\_{\\it{i}}}\\times {\\rm{gPC}}\_{i}+\\mathop{\\sum }\\limits\_{j=1}^{15}{\\beta }\_{\\rm{{{ePC}}}\_{\\it{j}}}\\times {\\rm{ePC}}\_{j}+\\mathop{\\sum }\\limits\_{k=1}^{15}{\\beta }\_{\\rm{{{hPC}}}\_{\\it{k}}}\\times {\\rm{hPC}}\_{k}\\\\\\qquad\\qquad\\quad\\;\\;+\\mathop{\\sum }\\limits\_{k=1}^{15}{\\beta }\_{\\rm{g}\\times {\\rm{hPC}}\_{k}}\\times {\\rm{genotype}}\\times {\\rm{hPC}}\_{k}+\\left(1|{\\rm{sample}}\\right).\\end{array}$

(3)

MT represents percent mitochondrial genes, and ePC represents the gene expression PC of the raw data. To assess whether the eQTL is cell state dependent, we compared the full model (equation ([3](https://www.nature.com/articles/s41588-025-02266-3#Equ3))) to the null model without interaction terms using the likelihood ratio test. To calculate eQTL strength across cell states, we combined the genotype main effect (_β_g) with the interaction effects of each hPC, weighted by each cell’s position along each hPC (equation ([4](https://www.nature.com/articles/s41588-025-02266-3#Equ4))).

${\\beta }\_{{\\rm{total}}}={\\beta }\_{\\rm{g}}+\\mathop{\\sum }\\limits\_{k=1}^{15}{\\beta }\_{\\rm{g}\\times h{{PC}}\_{\\it{k}}}\\times {\\rm{h{{PC}}}}\_{k}.$

(4)

To assess which hPCs had a strong interaction effect with genotype, we calculated _P_ values of interaction terms in the full model. To understand which cell state each PC represents, we investigated a loading of each gene along each PC in the SCT assay using the Loadings() function implemented in Seurat and determined the genes that showed strong correlation (>0.1 absolute value) for each PC. We performed pathway enrichment analysis of PC-associated genes using the compareCluster function (fun = ‘enrichGO,’ pvalueCutoff = 0.05, pAdjustMethod = ‘BH’, OrgDb = ‘org.Hs.eg.db’, ont = ‘BP’) of clusterProfiler (version 3.14.3)[67](https://www.nature.com/articles/s41588-025-02266-3#ref-CR67 "Yu, G., Wang, L.-G., Han, Y. & He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Omics 16, 284–287 (2012).").

### Colocalization analysis

We download GWAS summary statistics in an East Asian population of 13 complex immune traits[70](https://www.nature.com/articles/s41588-025-02266-3#ref-CR70 "Ishigaki, K. et al. Multi-ancestry genome-wide association analyses identify novel genetic mechanisms in rheumatoid arthritis. Nat. Genet. 54, 1640–1651 (2022)."),[71](https://www.nature.com/articles/s41588-025-02266-3#ref-CR71 "Wang, Y. F. et al. Identification of 38 novel loci for systemic lupus erythematosus and genetic heterogeneity between ancestral groups. Nat. Commun. 12, 772 (2021)."),[72](https://www.nature.com/articles/s41588-025-02266-3#ref-CR72 "Liu, Z. et al. Genetic architecture of the inflammatory bowel diseases across East Asian and European ancestries. Nat. Genet. 55, 796–806 (2023)."),[73](https://www.nature.com/articles/s41588-025-02266-3#ref-CR73 "Sakaue, S. et al. A cross-population atlas of genetic associations for 220 human phenotypes. Nat. Genet. 53, 1415–1424 (2021)."),[74](https://www.nature.com/articles/s41588-025-02266-3#ref-CR74 "Shirai, Y. et al. Multi-trait and cross-population genome-wide association studies across autoimmune and allergic diseases identify shared and distinct genetic component. Ann. Rheum. Dis. 81, 1301–1312 (2022).") except that of hospitalized COVID-19, which was obtained through collaboration with the Japan COVID-19 Task Force (JCTF)[47](https://www.nature.com/articles/s41588-025-02266-3#ref-CR47 "Namkoong, H. et al. DOCK2 is involved in the host genetics and biology of severe COVID-19. Nature 609, 754–760 (2022)."),[48](https://www.nature.com/articles/s41588-025-02266-3#ref-CR48 "Wang, Q. S. et al. The whole blood transcriptional regulation landscape in 465 COVID-19 infected samples from Japan COVID-19 Task Force. Nat. Commun. 13, 4830 (2022).") (Supplementary Table [18](https://www.nature.com/articles/s41588-025-02266-3#MOESM3)). GWAS summary statistics in hg37 were converted to hg38 ones using LiftOver. For evaluation of eQTL signal colocalization with GWAS signals, we applied coloc (version 5.2.3)[75](https://www.nature.com/articles/s41588-025-02266-3#ref-CR75 "Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. PLoS Genet. 10, e1004383 (2014)."). We tested for 500-kb windows centered on significant GWAS lead variants (_P_ value < 5 × 10−8) outside the MHC region and considered PP.H4 > 0.8 as significant colocalization. If the GWAS lead variants were listed in the paper, we used them, and, if not, we manually defined them by clumping using PLINK version 1.9 software. In a heatmap (Extended Data Fig. [6b](https://www.nature.com/articles/s41588-025-02266-3#Fig14)), the gene with the highest number of PP.H4 > 0.8 or, in case of a tie, the gene with the higher PP.H4 was presented.

### Plasma protein expression measurements

We measured the expression of 2,925 plasma proteins using the Olink Explore 3072 platform for 227 samples (83 patients with COVID-19 and 144 healthy individuals) through collaboration with the JCTF[47](https://www.nature.com/articles/s41588-025-02266-3#ref-CR47 "Namkoong, H. et al. DOCK2 is involved in the host genetics and biology of severe COVID-19. Nature 609, 754–760 (2022)."),[48](https://www.nature.com/articles/s41588-025-02266-3#ref-CR48 "Wang, Q. S. et al. The whole blood transcriptional regulation landscape in 465 COVID-19 infected samples from Japan COVID-19 Task Force. Nat. Commun. 13, 4830 (2022)."). The Olink Explore 3072 platform quantifies the expression of each protein in a normalized scale (normalized protein expression). Because the protein measurements were separated into three batches for logistical reasons, we bridge-normalized the normalized protein expression values using the OlinkAnalyze R package (version 3.4.1), using 16 intersecting samples as bridging samples. We excluded proteins with measurements below the detection limit in more than 10% of samples. As a result, 2,852 proteins were included in the analysis.

### Polygenic risk score

We constructed multipopulation PRSs of hospitalized COVID-19 using PRS-CSx[45](https://www.nature.com/articles/s41588-025-02266-3#ref-CR45 "Ruan, Y. et al. Improving polygenic prediction in ancestrally diverse populations. Nat. Genet. 54, 573–580 (2022)."), combining the summary statistics of hospitalized COVID-19 (B2, _n_case = 32,519) in Europeans from COVID-19 HGI (round 7)[46](https://www.nature.com/articles/s41588-025-02266-3#ref-CR46 "The COVID-19 Host Genetics Initiative. A second update on mapping the human genetic architecture of COVID-19. Nature 621, E7–E26 (2023).") and those of hospitalized COVID-19 (_n_case = 2,948) in Japanese individuals from the JCTF[47](https://www.nature.com/articles/s41588-025-02266-3#ref-CR47 "Namkoong, H. et al. DOCK2 is involved in the host genetics and biology of severe COVID-19. Nature 609, 754–760 (2022)."),[48](https://www.nature.com/articles/s41588-025-02266-3#ref-CR48 "Wang, Q. S. et al. The whole blood transcriptional regulation landscape in 465 COVID-19 infected samples from Japan COVID-19 Task Force. Nat. Commun. 13, 4830 (2022).") as the training data. We excluded the MHC region and set the _φ_ value to 10−4. Next, we calculated PRSs for OASIS samples and evaluated PRS performance with Nagelkerke’s _R_2.

To construct multipopulation PRSs of systemic lupus erythematosus[71](https://www.nature.com/articles/s41588-025-02266-3#ref-CR71 "Wang, Y. F. et al. Identification of 38 novel loci for systemic lupus erythematosus and genetic heterogeneity between ancestral groups. Nat. Commun. 12, 772 (2021)."),[76](https://www.nature.com/articles/s41588-025-02266-3#ref-CR76 "Bentham, J. et al. Genetic association analyses implicate aberrant regulation of innate and adaptive immunity genes in the pathogenesis of systemic lupus erythematosus. Nat. Genet. 47, 1457–1464 (2015)."), monocyte count[73](https://www.nature.com/articles/s41588-025-02266-3#ref-CR73 "Sakaue, S. et al. A cross-population atlas of genetic associations for 220 human phenotypes. Nat. Genet. 53, 1415–1424 (2021)."),[77](https://www.nature.com/articles/s41588-025-02266-3#ref-CR77 "Vuckovic, D. et al. The polygenic and monogenic basis of blood traits and diseases. Cell 182, 1214–1231 (2020).") and lymphocyte count[73](https://www.nature.com/articles/s41588-025-02266-3#ref-CR73 "Sakaue, S. et al. A cross-population atlas of genetic associations for 220 human phenotypes. Nat. Genet. 53, 1415–1424 (2021)."),[77](https://www.nature.com/articles/s41588-025-02266-3#ref-CR77 "Vuckovic, D. et al. The polygenic and monogenic basis of blood traits and diseases. Cell 182, 1214–1231 (2020)."), we performed the same procedure as described for COVID-19. For rheumatoid arthritis[70](https://www.nature.com/articles/s41588-025-02266-3#ref-CR70 "Ishigaki, K. et al. Multi-ancestry genome-wide association analyses identify novel genetic mechanisms in rheumatoid arthritis. Nat. Genet. 54, 1640–1651 (2022)."), inflammatory bowel diseases[72](https://www.nature.com/articles/s41588-025-02266-3#ref-CR72 "Liu, Z. et al. Genetic architecture of the inflammatory bowel diseases across East Asian and European ancestries. Nat. Genet. 55, 796–806 (2023)."), Crohn’s disease[72](https://www.nature.com/articles/s41588-025-02266-3#ref-CR72 "Liu, Z. et al. Genetic architecture of the inflammatory bowel diseases across East Asian and European ancestries. Nat. Genet. 55, 796–806 (2023).") and ulcerative colitis[72](https://www.nature.com/articles/s41588-025-02266-3#ref-CR72 "Liu, Z. et al. Genetic architecture of the inflammatory bowel diseases across East Asian and European ancestries. Nat. Genet. 55, 796–806 (2023)."), we used the publicly available multipopulation PRSs. We could not evaluate PRS performance in the OASIS cohort except for the COVID-19 PRS.

### Differential gene and protein expression analysis with polygenic risk score

We divided each PRS into four quantiles in the OASIS cohort and treated them as ordinal variables 1 (lowest risk) to 4 (highest risk). DEG analysis among four quantiles of PRS was performed using the pseudobulk approach separately for patients with COVID-19 and healthy individuals. Pseudobulk matrices were created by aggregating gene counts for each cell type (L1) in each sample. Genes were considered for the analysis if they were expressed in more than 10% of cells per cell type. This analysis was performed using edgeR (version 3.32.0)[78](https://www.nature.com/articles/s41588-025-02266-3#ref-CR78 "Robinson, M. D., McCarthy, D. J. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139–140 (2010)."). We included age, sex, 10x chemistry and severity (severity was included only in patients with COVID-19) in the model as covariates. DEP analysis among four quantiles of PRSs was performed using a linear regression model with age, sex, batches and severity (severity was included only in patients with COVID-19) as covariates. Statistically significant DEGs and DEPs were defined with FDR < 0.1.

We permuted four quantiles of PRSs across samples 1,000 times in each cell type of scRNA-seq and proteomics data and assessed how often the observed number of DEGs and DEPs would be detected.

To investigate whether DEGs and DEPs between patients with COVID-19 and healthy individuals reflected those of PRSs specific to patients with COVID-19, DEG and DEP analysis between cases and controls were performed with the same method described above except that statistically significant DEGs were defined with FDR < 0.05 and absolute fold change > 2.

### Detection of somatic events using genomics data

For the detection of mCAs and LOY from SNP array data, we used the MoChA pipeline (18 May 2022)[14](https://www.nature.com/articles/s41588-025-02266-3#ref-CR14 "Loh, P., Genovese, G. & McCarroll, S. A. Monogenic and polygenic inheritance become instruments for clonal selection. Nature 584, 136–141 (2020)."),[49](https://www.nature.com/articles/s41588-025-02266-3#ref-CR49 "Loh, P. R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."). In brief, IDAT genotype intensity data were converted to GTC genotype files and then transformed into VCF files with log2 (_R_ ratio) and B allele frequency values to estimate total and relative allelic intensities, respectively. Using the phase information and log2 (_R_ ratio) and B allele frequency values, MoChA was used to detect mCAs and LOY. From the resulting candidates, calls flagged as germline copy number polymorphisms and calls that were likely germline duplications were removed. For the LOY candidates, XXY and XXX samples were filtered out. We additionally removed unclassifiable calls and calls with lower cell fraction (<0.01).

We used a previously reported GATK-based pipeline[52](https://www.nature.com/articles/s41588-025-02266-3#ref-CR52 "Laricchia, K. M. et al. Mitochondrial DNA variation across 56,434 individuals in gnomAD. Genome Res. 32, 569–582 (2022).") to detect mt-heteroplasmy from WGS data. To call homoplasmic and heteroplasmic variants in mtDNA (the circular genome including 16,569 bp), we used a mitochondrial mode implemented in GATK Mutect2. Variants with VAF > 0.9 were defined as homoplasmic, and we analyzed mt-heteroplasmy satisfying VAF > 0.1.

### Single-cell deconvolution of somatic events

We applied Numbat (version 1.3.0)[50](https://www.nature.com/articles/s41588-025-02266-3#ref-CR50 "Gao, T. et al. Haplotype-aware analysis of somatic copy number variations from single-cell transcriptomes. Nat. Biotechnol. 41, 417–426 (2023).") to scRNA-seq data to detect clones with mCAs found in SNP array data. Numbat integrated haplotype information with allele and expression signals from scRNA-seq data to enhance detection of mCAs including CN-LOHs. Expression references of the COVID-19 and healthy samples were generated from our scRNA-seq data. We specified the profiles of the genotype-based mCAs and ran Numbat with parameters init\_k = 10 and max\_entropy = 1.0.

Using scRNA-seq data from all male samples, we defined cells with LOY as cells without expression from the male-specific region of the Y chromosome[22](https://www.nature.com/articles/s41588-025-02266-3#ref-CR22 "Bruhn-Olszewska, B. et al. Loss of Y in leukocytes as a risk factor for critical COVID-19 in men. Genome Med. 14, 139 (2022)."). As single-cell-based LOY, we then classified the male samples as LOY high (cells with LOY > 5%) or LOY low (cells with LOY < 5%). The association between LOY and COVID-19 risk was evaluated using univariate logistic regression and multivariate logistic regression adjusted for age.

For mt-heteroplasmy detected in WGS data, we piled up raw reads from the scRNA-seq BAM files and generated SNP-by-cell matrices using cellSNP-lite (version 1.2.2)[79](https://www.nature.com/articles/s41588-025-02266-3#ref-CR79 "Huang, X. & Huang, Y. Cellsnp-lite: an efficient tool for genotyping single cells. Bioinformatics 37, 4569–4571 (2021)."). Next, we performed clonal assignment of each cell using the BinomMixtureVB function (binomial mixture model) implemented in vireoSNP (version 0.5.8)[80](https://www.nature.com/articles/s41588-025-02266-3#ref-CR80 "Huang, Y., McCarthy, D. J. & Stegle, O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. Genome Biol. 20, 273 (2019)."). We included samples in which more than 80% of cells showed posterior clonal assignment probability > 0.8.

### DEG analysis between somatically mutated and normal cells

DEG analysis for CNAs was conducted for mutated and normal cells from the same samples in CH01 and CH05. We used MAST (version 1.20.0)[81](https://www.nature.com/articles/s41588-025-02266-3#ref-CR81 "Finak, G. et al. MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. Genome Biol. 16, 278 (2015).") implemented in Seurat, and DEGs were considered significant if they satisfied FDR (adjusted _P_ values via the Benjamini–Hochberg method) < 0.05 and log2 (fold change) > 0.25 (in CH01) or 0.5 (in CH05). For significant DEGs, pathway enrichment analysis was performed using the enrichGO function of clusterProfiler[67](https://www.nature.com/articles/s41588-025-02266-3#ref-CR67 "Yu, G., Wang, L.-G., Han, Y. & He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Omics 16, 284–287 (2012).") with parameters OrgDb = ‘org.Hs.eg.db’, ont = ‘BP’, pvalueCutoff = 0.05 and pAdjustMethod = ‘BH’. In the CH05 analysis, immunoglobulin genes were excluded from the pathway analysis.

DEG analysis between cells with LOY and normal cells was performed using a pseudobulk approach. Pseudobulk matrices were created by aggregating gene counts for each cell type within cells with LOY or normal cells in each sample. Genes were included if they were expressed in more than 10% of cells. We included samples with more than nine cells with LOY and more than nine normal cells in the cell type. This analysis was performed using a linear mixed model implemented in the lme4 R library (version 1.1.32) with fixed effects for age and 10x chemistry and random effects for sample. DEGs were considered significant if they satisfied FDR (adjusted _P_ values via the Benjamini–Hochberg method) < 0.01 and log2 (fold change) > 0.25. Pathway analysis was performed using the same procedure as for mCAs.

### Differential abundance analysis of LOY

We used Milo (version 1.2.0)[37](https://www.nature.com/articles/s41588-025-02266-3#ref-CR37 "Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. Nat. Biotechnol. 40, 245–253 (2022).") to test for the differential abundance of cells within defined neighborhoods between LOY-high and LOY-low samples. This analysis was performed in the same manner as the metagenomic analysis. Comparisons of cell proportions between LOY-high and LOY-low samples were carried out using two-sided Wilcoxon rank-sum test.

### Statistics and reproducibility

No statistical methods were used to predetermine sample sizes. Data distribution was assumed to be normal, but this was not formally tested. No data were excluded from the analyses. We did not use any study design that required randomization or blinding.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02266-3#MOESM2) linked to this article.

Data availability
-----------------

Raw sequencing data of scRNA-seq and the protein expression matrix are available at the Japanese Genotype–phenotype Archive (JGA) with accession codes [JGAS000783](https://ddbj.nig.ac.jp/search/entry/jga-study/JGAS000783) and [JGAD000925](https://ddbj.nig.ac.jp/search/entry/jga-study/JGAD000925). A part of the raw scRNA-seq data (_n_COVID-19 = 73, _n_control = 75)[44](https://www.nature.com/articles/s41588-025-02266-3#ref-CR44 "Edahiro, R. et al. Single-cell analyses and host genetics highlight the role of innate immune cells in COVID-19 severity. Nat. Genet. 55, 753–767 (2023)."),[47](https://www.nature.com/articles/s41588-025-02266-3#ref-CR47 "Namkoong, H. et al. DOCK2 is involved in the host genetics and biology of severe COVID-19. Nature 609, 754–760 (2022).") has already been deposited and is available under controlled access at JGA with accession codes [JGAS000593](https://ddbj.nig.ac.jp/search/entry/jga-dataset/JGAS000593), [JGAS000543](https://ddbj.nig.ac.jp/search/entry/jga-dataset/JGAS000543), [JGAD000662](https://ddbj.nig.ac.jp/search/entry/jga-dataset/JGAD000662) and [JGAD000722](https://ddbj.nig.ac.jp/search/entry/jga-dataset/JGAD000722). All raw scRNA-seq data can also be accessed by applying at the NBDC with the accession code [hum0197](https://humandbs.biosciencedbc.jp/en/hum0197-latest). Participant genotype data are available at the European Genome–Phenome Archive with the accession code [EGAS00001008016](https://ega-archive.org/studies/EGAS00001008016). sc-eQTL summary statistics are available at the NBDC under accession ID [hum0197](https://humandbs.biosciencedbc.jp/en/hum0197-latest) and are also available in an interactive browser at [https://japan-omics.jp/](https://japan-omics.jp/). OneK1K genotype data were obtained from the Gene Expression Omnibus ([GSE196830](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE196830)). Processed OneK1K scRNA-seq data (h5ad file) were obtained from the Human Cell Atlas ([https://explore.data.humancellatlas.org/projects/f2078d5f-2e7d-4844-8552-f7c41a231e52](https://explore.data.humancellatlas.org/projects/f2078d5f-2e7d-4844-8552-f7c41a231e52)). OneK1K eQTL summary statistics were obtained from [https://onek1k.org/](https://onek1k.org/).

Code availability
-----------------

The codes used in this study are shared on GitHub ([https://github.com/REdahiro/OASIS\_project](https://github.com/REdahiro/OASIS_project) and [https://github.com/tatsuhikonaito/OASIS\_HLATCR](https://github.com/tatsuhikonaito/OASIS_HLATCR)) and have been deposited at Zenodo ([https://doi.org/10.5281/zenodo.15877644](https://doi.org/10.5281/zenodo.15877644) (ref. [82](https://www.nature.com/articles/s41588-025-02266-3#ref-CR82 "Edahiro, R. REdahiro/OASIS_project: v1.1. Zenodo 
https://doi.org/10.5281/zenodo.15877644
(2025).")) and [https://doi.org/10.5281/zenodo.14991132](https://doi.org/10.5281/zenodo.14991132) (ref. [83](https://www.nature.com/articles/s41588-025-02266-3#ref-CR83 "Naito, T. tatsuhikonaito/OASIS_HLATCR: v1.0.0. Zenodo 
https://doi.org/10.5281/zenodo.14991132
(2025)."))).

References
----------

1.  Aguet, F. et al. Molecular quantitative trait loci. _Nat. Rev. Methods Primers_ **3**, 5 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20quantitative%20trait%20loci&journal=Nat.%20Rev.%20Methods%20Primers&volume=3&publication_year=2023&author=Aguet%2CF) 
    
2.  Aguet, F. et al. Genetic effects on gene expression across human tissues. _Nature_ **550**, 204–213 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20effects%20on%20gene%20expression%20across%20human%20tissues&journal=Nature&volume=550&pages=204-213&publication_year=2017&author=Aguet%2CF) 
    
3.  Strober, B. J. et al. The GTEx Consortium atlas of genetic regulatory effects across human tissues. _Science_ **18**, 1318–1330 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20GTEx%20Consortium%20atlas%20of%20genetic%20regulatory%20effects%20across%20human%20tissues&journal=Science&volume=18&pages=1318-1330&publication_year=2020&author=Strober%2CBJ) 
    
4.  Sun, B. B. et al. Plasma proteomic associations with genetics and health in the UK Biobank. _Nature_ **622**, 329–338 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVGqsbbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37794186)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10567551)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plasma%20proteomic%20associations%20with%20genetics%20and%20health%20in%20the%20UK%20Biobank&journal=Nature&volume=622&pages=329-338&publication_year=2023&author=Sun%2CBB) 
    
5.  Qin, Y. et al. Combined effects of host genetics and diet on human gut microbiota and incident disease in a single population cohort. _Nat. Genet._ **54**, 134–142 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XislKnsLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35115689)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9883041)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combined%20effects%20of%20host%20genetics%20and%20diet%20on%20human%20gut%20microbiota%20and%20incident%20disease%20in%20a%20single%20population%20cohort&journal=Nat.%20Genet.&volume=54&pages=134-142&publication_year=2022&author=Qin%2CY) 
    
6.  Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. _Science_ **376**, eabf3041 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVynt7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35389779)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20eQTL%20mapping%20identifies%20cell%20type-specific%20genetic%20control%20of%20autoimmune%20disease&journal=Science&volume=376&publication_year=2022&author=Yazar%2CS) 
    
7.  Perez, R. K. et al. Single-cell RNA-seq reveals cell type-specific molecular and genetic associations to lupus. _Science_ **376**, eabf1970 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVynt7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35389781)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9297655)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20RNA-seq%20reveals%20cell%20type-specific%20molecular%20and%20genetic%20associations%20to%20lupus&journal=Science&volume=376&publication_year=2022&author=Perez%2CRK) 
    
8.  Soskic, B. et al. Immune disease risk variants regulate gene expression dynamics during CD4+ T cell activation. _Nat. Genet._ **54**, 817–826 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVSqu73J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35618845)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9197762)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Immune%20disease%20risk%20variants%20regulate%20gene%20expression%20dynamics%20during%20CD4%2B%20T%20cell%20activation&journal=Nat.%20Genet.&volume=54&pages=817-826&publication_year=2022&author=Soskic%2CB) 
    
9.  Nathan, A. et al. Single-cell eQTL models reveal dynamic T cell state dependence of disease loci. _Nature_ **606**, 120–128 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1Krsb7I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35545678)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9842455)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20eQTL%20models%20reveal%20dynamic%20T%20cell%20state%20dependence%20of%20disease%20loci&journal=Nature&volume=606&pages=120-128&publication_year=2022&author=Nathan%2CA) 
    
10.  Kumasaka, N. et al. Mapping interindividual dynamics of innate immune response at single-cell resolution. _Nat. Genet._ **55**, 1066–1075 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1WlsL7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37308670)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10260404)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20interindividual%20dynamics%20of%20innate%20immune%20response%20at%20single-cell%20resolution&journal=Nat.%20Genet.&volume=55&pages=1066-1075&publication_year=2023&author=Kumasaka%2CN) 
    
11.  Zheng, L. et al. Pan-cancer single-cell landscape of tumor-infiltrating T cells. _Science_ **374**, abe6474 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34914499)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-cancer%20single-cell%20landscape%20of%20tumor-infiltrating%20T%20cells&journal=Science&volume=374&publication_year=2021&author=Zheng%2CL) 
    
12.  Yoshida, M. et al. Local and systemic responses to SARS-CoV-2 infection in children and adults. _Nature_ **602**, 321–327 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xisleltrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34937051)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Local%20and%20systemic%20responses%20to%20SARS-CoV-2%20infection%20in%20children%20and%20adults&journal=Nature&volume=602&pages=321-327&publication_year=2022&author=Yoshida%2CM) 
    
13.  Oelen, R. et al. Single-cell RNA-sequencing of peripheral blood mononuclear cells reveals widespread, context-specific gene expression regulation upon pathogenic exposure. _Nat. Commun._ **13**, 3267 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsF2msLvK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35672358)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9174272)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20RNA-sequencing%20of%20peripheral%20blood%20mononuclear%20cells%20reveals%20widespread%2C%20context-specific%20gene%20expression%20regulation%20upon%20pathogenic%20exposure&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Oelen%2CR) 
    
14.  Loh, P., Genovese, G. & McCarroll, S. A. Monogenic and polygenic inheritance become instruments for clonal selection. _Nature_ **584**, 136–141 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Cmu7nJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32581363)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7415571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Monogenic%20and%20polygenic%20inheritance%20become%20instruments%20for%20clonal%20selection&journal=Nature&volume=584&pages=136-141&publication_year=2020&author=Loh%2CP&author=Genovese%2CG&author=McCarroll%2CSA) 
    
15.  Thompson, D. J. et al. Genetic predisposition to mosaic Y chromosome loss in blood. _Nature_ **575**, 652–657 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitF2qt7bO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31748747)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6887549)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20predisposition%20to%20mosaic%20Y%20chromosome%20loss%20in%20blood&journal=Nature&volume=575&pages=652-657&publication_year=2019&author=Thompson%2CDJ) 
    
16.  Laurie, C. C. et al. Detectable clonal mosaicism from birth to old age and its relationship to cancer. _Nat. Genet._ **44**, 642–650 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmsFOrt7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22561516)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3366033)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Detectable%20clonal%20mosaicism%20from%20birth%20to%20old%20age%20and%20its%20relationship%20to%20cancer&journal=Nat.%20Genet.&volume=44&pages=642-650&publication_year=2012&author=Laurie%2CCC) 
    
17.  Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. _N. Engl. J. Med._ **371**, 2477–2487 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25426838)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290021)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20and%20blood-cancer%20risk%20inferred%20from%20blood%20DNA%20sequence&journal=N.%20Engl.%20J.%20Med.&volume=371&pages=2477-2487&publication_year=2014&author=Genovese%2CG) 
    
18.  Saiki, R. et al. Combined landscape of single-nucleotide variants and copy number alterations in clonal hematopoiesis. _Nat. Med._ **27**, 1239–1249 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSktL3E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34239136)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combined%20landscape%20of%20single-nucleotide%20variants%20and%20copy%20number%20alterations%20in%20clonal%20hematopoiesis&journal=Nat.%20Med.&volume=27&pages=1239-1249&publication_year=2021&author=Saiki%2CR) 
    
19.  Jaiswal, S. et al. Clonal hematopoiesis and risk of atherosclerotic cardiovascular disease. _N. Engl. J. Med._ **377**, 111–121 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28636844)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6717509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20and%20risk%20of%20atherosclerotic%20cardiovascular%20disease&journal=N.%20Engl.%20J.%20Med.&volume=377&pages=111-121&publication_year=2017&author=Jaiswal%2CS) 
    
20.  Sano, S. et al. Hematopoietic loss of Y chromosome leads to cardiac fibrosis and heart failure mortality. _Science_ **377**, 292–297 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVWrtL3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35857592)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9437978)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hematopoietic%20loss%20of%20Y%20chromosome%20leads%20to%20cardiac%20fibrosis%20and%20heart%20failure%20mortality&journal=Science&volume=377&pages=292-297&publication_year=2022&author=Sano%2CS) 
    
21.  Zekavat, S. M. et al. Hematopoietic mosaic chromosomal alterations increase the risk for diverse types of infection. _Nat. Med._ **27**, 1012–1024 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1Kgt7jJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34099924)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8245201)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hematopoietic%20mosaic%20chromosomal%20alterations%20increase%20the%20risk%20for%20diverse%20types%20of%20infection&journal=Nat.%20Med.&volume=27&pages=1012-1024&publication_year=2021&author=Zekavat%2CSM) 
    
22.  Bruhn-Olszewska, B. et al. Loss of Y in leukocytes as a risk factor for critical COVID-19 in men. _Genome Med._ **14**, 139 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFSltLnO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36514076)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9747543)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20Y%20in%20leukocytes%20as%20a%20risk%20factor%20for%20critical%20COVID-19%20in%20men&journal=Genome%20Med.&volume=14&publication_year=2022&author=Bruhn-Olszewska%2CB) 
    
23.  Nam, A. S. et al. Single-cell multi-omics of human clonal hematopoiesis reveals that _DNMT3A_ R882 mutations perturb early progenitor states through selective hypomethylation. _Nat. Genet._ **54**, 1514–1526 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisVKns7fO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36138229)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10068894)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20multi-omics%20of%20human%20clonal%20hematopoiesis%20reveals%20that%20DNMT3A%20R882%20mutations%20perturb%20early%20progenitor%20states%20through%20selective%20hypomethylation&journal=Nat.%20Genet.&volume=54&pages=1514-1526&publication_year=2022&author=Nam%2CAS) 
    
24.  Gao, T. et al. A pan-tissue survey of mosaic chromosomal alterations in 948 individuals. _Nat. Genet._ **55**, 1901–1911 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXit1KgsrbK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37904053)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10838621)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20pan-tissue%20survey%20of%20mosaic%20chromosomal%20alterations%20in%20948%20individuals&journal=Nat.%20Genet.&volume=55&pages=1901-1911&publication_year=2023&author=Gao%2CT) 
    
25.  Abdel-Hafiz, H. A. et al. Y chromosome loss in cancer drives growth by evasion of adaptive immunity. _Nature_ **619**, 624–631 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1OntLjK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37344596)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10975863)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Y%20chromosome%20loss%20in%20cancer%20drives%20growth%20by%20evasion%20of%20adaptive%20immunity&journal=Nature&volume=619&pages=624-631&publication_year=2023&author=Abdel-Hafiz%2CHA) 
    
26.  Lareau, C. A. et al. Single-cell multi-omics of mitochondrial DNA disorders reveals dynamics of purifying selection across human immune cells. _Nat. Genet._ **55**, 1198–1209 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlKjsr3E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37386249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10548551)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20multi-omics%20of%20mitochondrial%20DNA%20disorders%20reveals%20dynamics%20of%20purifying%20selection%20across%20human%20immune%20cells&journal=Nat.%20Genet.&volume=55&pages=1198-1209&publication_year=2023&author=Lareau%2CCA) 
    
27.  Hao, Y. et al. Integrated analysis of multimodal single-cell data. _Cell_ **184**, 3573–3587 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtrvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34062119)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238499)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&journal=Cell&volume=184&pages=3573-3587&publication_year=2021&author=Hao%2CY) 
    
28.  Mulder, K. et al. Cross-tissue single-cell landscape of human monocytes and macrophages in health and disease. _Immunity_ **54**, 1883–1900 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1yiu7zK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34331874)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-tissue%20single-cell%20landscape%20of%20human%20monocytes%20and%20macrophages%20in%20health%20and%20disease&journal=Immunity&volume=54&pages=1883-1900&publication_year=2021&author=Mulder%2CK) 
    
29.  Turner, J. S. et al. Human germinal centres engage memory and naive B cells after influenza vaccination. _Nature_ **586**, 127–132 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVClur3E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32866963)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7566073)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20germinal%20centres%20engage%20memory%20and%20naive%20B%20cells%20after%20influenza%20vaccination&journal=Nature&volume=586&pages=127-132&publication_year=2020&author=Turner%2CJS) 
    
30.  Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. _Cell_ **184**, 1895–1913 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlsFSgtrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33657410)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7857060)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=COVID-19%20immune%20features%20revealed%20by%20a%20large-scale%20single-cell%20transcriptome%20atlas&journal=Cell&volume=184&pages=1895-1913&publication_year=2021&author=Ren%2CX) 
    
31.  Krämer, B. et al. Early IFN-α signatures and persistent dysfunction are distinguishing features of NK cells in severe COVID-19. _Immunity_ **54**, 2650–2669 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34592166)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8416549)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Early%20IFN-%CE%B1%20signatures%20and%20persistent%20dysfunction%20are%20distinguishing%20features%20of%20NK%20cells%20in%20severe%20COVID-19&journal=Immunity&volume=54&pages=2650-2669&publication_year=2021&author=Kr%C3%A4mer%2CB) 
    
32.  Alquicira-Hernandez, J., Sathe, A., Ji, H. P., Nguyen, Q. & Powell, J. E. ScPred: accurate supervised method for cell-type classification from single-cell RNA-seq data. _Genome Biol._ **20**, 264 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlyntLvM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31829268)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6907144)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ScPred%3A%20accurate%20supervised%20method%20for%20cell-type%20classification%20from%20single-cell%20RNA-seq%20data&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Alquicira-Hernandez%2CJ&author=Sathe%2CA&author=Ji%2CHP&author=Nguyen%2CQ&author=Powell%2CJE) 
    
33.  Ota, M. et al. Dynamic landscape of immune cell-specific gene regulation in immune-mediated diseases. _Cell_ **184**, 3006–3021 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVSqtrvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33930287)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamic%20landscape%20of%20immune%20cell-specific%20gene%20regulation%20in%20immune-mediated%20diseases&journal=Cell&volume=184&pages=3006-3021&publication_year=2021&author=Ota%2CM) 
    
34.  Urbut, S. M., Wang, G., Carbonetto, P. & Stephens, M. Flexible statistical methods for estimating and testing effects in genomic studies with multiple conditions. _Nat. Genet._ **51**, 187–195 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlWnu77K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30478440)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flexible%20statistical%20methods%20for%20estimating%20and%20testing%20effects%20in%20genomic%20studies%20with%20multiple%20conditions&journal=Nat.%20Genet.&volume=51&pages=187-195&publication_year=2019&author=Urbut%2CSM&author=Wang%2CG&author=Carbonetto%2CP&author=Stephens%2CM) 
    
35.  Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. _Nature_ **518**, 317–329 (2015).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4530010)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&journal=Nature&volume=518&pages=317-329&publication_year=2015&author=Roadmap%20Epigenomics%20Consortium%2C) 
    
36.  Farh, K. K.-H. et al. Genetic and epigenetic fine mapping of causal autoimmune disease variants. _Nature_ **518**, 337–343 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVemtrbP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25363779)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20and%20epigenetic%20fine%20mapping%20of%20causal%20autoimmune%20disease%20variants&journal=Nature&volume=518&pages=337-343&publication_year=2015&author=Farh%2CKK-H) 
    
37.  Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using _k_\-nearest neighbor graphs. _Nat. Biotechnol._ **40**, 245–253 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFGru7vN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34594043)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Differential%20abundance%20testing%20on%20single-cell%20data%20using%20k-nearest%20neighbor%20graphs&journal=Nat.%20Biotechnol.&volume=40&pages=245-253&publication_year=2022&author=Dann%2CE&author=Henderson%2CNC&author=Teichmann%2CSA&author=Morgan%2CMD&author=Marioni%2CJC) 
    
38.  Franzosa, E. A. et al. Gut microbiome structure and metabolic activity in inflammatory bowel disease. _Nat. Microbiol._ **4**, 293–305 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhtbnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30531976)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gut%20microbiome%20structure%20and%20metabolic%20activity%20in%20inflammatory%20bowel%20disease&journal=Nat.%20Microbiol.&volume=4&pages=293-305&publication_year=2019&author=Franzosa%2CEA) 
    
39.  Nii, T. et al. Genomic repertoires linked with pathogenic potency of arthritogenic _Prevotella copri_ isolated from the gut of patients with rheumatoid arthritis. _Ann. Rheum. Dis._ **82**, 621–629 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtVKlt77J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36627170)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20repertoires%20linked%20with%20pathogenic%20potency%20of%20arthritogenic%20Prevotella%20copri%20isolated%20from%20the%20gut%20of%20patients%20with%20rheumatoid%20arthritis&journal=Ann.%20Rheum.%20Dis.&volume=82&pages=621-629&publication_year=2023&author=Nii%2CT) 
    
40.  Mills, R. H. et al. Multi-omics analyses of the ulcerative colitis gut microbiome link _Bacteroides vulgatus_ proteases with disease severity. _Nat. Microbiol._ **7**, 262–276 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlGhsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35087228)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8852248)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-omics%20analyses%20of%20the%20ulcerative%20colitis%20gut%20microbiome%20link%20Bacteroides%20vulgatus%20proteases%20with%20disease%20severity&journal=Nat.%20Microbiol.&volume=7&pages=262-276&publication_year=2022&author=Mills%2CRH) 
    
41.  La Gruta, N. L., Gras, S., Daley, S. R., Thomas, P. G. & Rossjohn, J. Understanding the drivers of MHC restriction of T cell receptors. _Nat. Rev. Immunol._ **18**, 467–478 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29636542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%20the%20drivers%20of%20MHC%20restriction%20of%20T%20cell%20receptors&journal=Nat.%20Rev.%20Immunol.&volume=18&pages=467-478&publication_year=2018&author=Gruta%2CNL&author=Gras%2CS&author=Daley%2CSR&author=Thomas%2CPG&author=Rossjohn%2CJ) 
    
42.  Nagafuchi, Y. et al. Control of naive and effector CD4 T cell receptor repertoires by rheumatoid-arthritis-risk HLA alleles. _J. Autoimmun._ **133**, 102907 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFOlsLvL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36126366)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Control%20of%20naive%20and%20effector%20CD4%20T%20cell%20receptor%20repertoires%20by%20rheumatoid-arthritis-risk%20HLA%20alleles&journal=J.%20Autoimmun.&volume=133&publication_year=2022&author=Nagafuchi%2CY) 
    
43.  Ishigaki, K. et al. HLA autoimmune risk alleles restrict the hypervariable region of T cell receptors. _Nat. Genet._ **54**, 393–402 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XotVGit7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35332318)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9010379)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HLA%20autoimmune%20risk%20alleles%20restrict%20the%20hypervariable%20region%20of%20T%20cell%20receptors&journal=Nat.%20Genet.&volume=54&pages=393-402&publication_year=2022&author=Ishigaki%2CK) 
    
44.  Edahiro, R. et al. Single-cell analyses and host genetics highlight the role of innate immune cells in COVID-19 severity. _Nat. Genet._ **55**, 753–767 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXosVKru7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37095364)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10181941)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20analyses%20and%20host%20genetics%20highlight%20the%20role%20of%20innate%20immune%20cells%20in%20COVID-19%20severity&journal=Nat.%20Genet.&volume=55&pages=753-767&publication_year=2023&author=Edahiro%2CR) 
    
45.  Ruan, Y. et al. Improving polygenic prediction in ancestrally diverse populations. _Nat. Genet._ **54**, 573–580 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1agt7fK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35513724)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9117455)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improving%20polygenic%20prediction%20in%20ancestrally%20diverse%20populations&journal=Nat.%20Genet.&volume=54&pages=573-580&publication_year=2022&author=Ruan%2CY) 
    
46.  The COVID-19 Host Genetics Initiative. A second update on mapping the human genetic architecture of COVID-19. _Nature_ **621**, E7–E26 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVGrtbbN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20second%20update%20on%20mapping%20the%20human%20genetic%20architecture%20of%20COVID-19&journal=Nature&volume=621&pages=E7-E26&publication_year=2023) 
    
47.  Namkoong, H. et al. DOCK2 is involved in the host genetics and biology of severe COVID-19. _Nature_ **609**, 754–760 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xitl2nurrO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35940203)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9492544)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DOCK2%20is%20involved%20in%20the%20host%20genetics%20and%20biology%20of%20severe%20COVID-19&journal=Nature&volume=609&pages=754-760&publication_year=2022&author=Namkoong%2CH) 
    
48.  Wang, Q. S. et al. The whole blood transcriptional regulation landscape in 465 COVID-19 infected samples from Japan COVID-19 Task Force. _Nat. Commun._ **13**, 4830 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1SksbrO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35995775)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9395416)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20whole%20blood%20transcriptional%20regulation%20landscape%20in%20465%20COVID-19%20infected%20samples%20from%20Japan%20COVID-19%20Task%20Force&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Wang%2CQS) 
    
49.  Loh, P. R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. _Nature_ **559**, 350–355 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjsbbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29995854)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6054542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20into%20clonal%20haematopoiesis%20from%208%2C342%20mosaic%20chromosomal%20alterations&journal=Nature&volume=559&pages=350-355&publication_year=2018&author=Loh%2CPR) 
    
50.  Gao, T. et al. Haplotype-aware analysis of somatic copy number variations from single-cell transcriptomes. _Nat. Biotechnol._ **41**, 417–426 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisV2nsrrJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36163550)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haplotype-aware%20analysis%20of%20somatic%20copy%20number%20variations%20from%20single-cell%20transcriptomes&journal=Nat.%20Biotechnol.&volume=41&pages=417-426&publication_year=2023&author=Gao%2CT) 
    
51.  Ahern, D. J. et al. A blood atlas of COVID-19 defines hallmarks of disease severity and specificity. _Cell_ **185**, 916–938 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20blood%20atlas%20of%20COVID-19%20defines%20hallmarks%20of%20disease%20severity%20and%20specificity&journal=Cell&volume=185&pages=916-938&publication_year=2022&author=Ahern%2CDJ) 
    
52.  Laricchia, K. M. et al. Mitochondrial DNA variation across 56,434 individuals in gnomAD. _Genome Res._ **32**, 569–582 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXit1CmsLbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35074858)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8896463)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mitochondrial%20DNA%20variation%20across%2056%2C434%20individuals%20in%20gnomAD&journal=Genome%20Res.&volume=32&pages=569-582&publication_year=2022&author=Laricchia%2CKM) 
    
53.  Zou, T. et al. Impacts of impaired mitochondrial dynamics in hearing loss: potential therapeutic targets. _Front. Neurosci._ **16**, 998507 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36278017)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9579438)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Impacts%20of%20impaired%20mitochondrial%20dynamics%20in%20hearing%20loss%3A%20potential%20therapeutic%20targets&journal=Front.%20Neurosci.&volume=16&publication_year=2022&author=Zou%2CT) 
    
54.  World Health Organization. _COVID-19 Therapeutic Trial Synopsis_ (World Health Organization, 2020).
    
55.  Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. _Bioinformatics_ **29**, 15–21 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23104886)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&journal=Bioinformatics&volume=29&pages=15-21&publication_year=2013&author=Dobin%2CA) 
    
56.  Lun, A. T. L., Bach, K. & Marioni, J. C. Pooling across cells to normalize single-cell RNA sequencing data with many zero counts. _Genome Biol._ **17**, 75 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27122128)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pooling%20across%20cells%20to%20normalize%20single-cell%20RNA%20sequencing%20data%20with%20many%20zero%20counts&journal=Genome%20Biol.&volume=17&publication_year=2016&author=Lun%2CATL&author=Bach%2CK&author=Marioni%2CJC) 
    
57.  Taylor-Weiner, A. et al. Scaling computational genomics to millions of individuals with GPUs. _Genome Biol._ **20**, 228 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31675989)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823959)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scaling%20computational%20genomics%20to%20millions%20of%20individuals%20with%20GPUs&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Taylor-Weiner%2CA) 
    
58.  Tomofuji, Y. et al. Metagenome-wide association study revealed disease-specific landscape of the gut microbiome of systemic lupus erythematosus in Japanese. _Ann. Rheum. Dis._ **80**, 1575–1583 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisleisLjE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34426398)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Metagenome-wide%20association%20study%20revealed%20disease-specific%20landscape%20of%20the%20gut%20microbiome%20of%20systemic%20lupus%20erythematosus%20in%20Japanese&journal=Ann.%20Rheum.%20Dis.&volume=80&pages=1575-1583&publication_year=2021&author=Tomofuji%2CY) 
    
59.  Tomofuji, Y. et al. Prokaryotic and viral genomes recovered from 787 Japanese gut metagenomes revealed microbial features linked to diets, populations, and diseases. _Cell Genom._ **2**, 100219 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVWhtbbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36778050)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9903723)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prokaryotic%20and%20viral%20genomes%20recovered%20from%20787%20Japanese%20gut%20metagenomes%20revealed%20microbial%20features%20linked%20to%20diets%2C%20populations%2C%20and%20diseases&journal=Cell%20Genom.&volume=2&publication_year=2022&author=Tomofuji%2CY) 
    
60.  Sturm, G. et al. Scirpy: a Scanpy extension for analyzing single-cell T-cell receptor-sequencing data. _Bioinformatics_ **36**, 4817–4818 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsVWqsbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32614448)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7751015)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scirpy%3A%20a%20Scanpy%20extension%20for%20analyzing%20single-cell%20T-cell%20receptor-sequencing%20data&journal=Bioinformatics&volume=36&pages=4817-4818&publication_year=2020&author=Sturm%2CG) 
    
61.  Hirata, J. et al. Genetic and phenotypic landscape of the major histocompatibilty complex region in the Japanese population. _Nat. Genet._ **51**, 470–480 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtFKjsLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30692682)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20and%20phenotypic%20landscape%20of%20the%20major%20histocompatibilty%20complex%20region%20in%20the%20Japanese%20population&journal=Nat.%20Genet.&volume=51&pages=470-480&publication_year=2019&author=Hirata%2CJ) 
    
62.  Naito, T. et al. A deep learning method for HLA imputation and trans-ethnic MHC fine-mapping of type 1 diabetes. _Nat. Commun._ **12**, 1639 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmslKju7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33712626)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7955122)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20deep%20learning%20method%20for%20HLA%20imputation%20and%20trans-ethnic%20MHC%20fine-mapping%20of%20type%201%20diabetes&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Naito%2CT) 
    
63.  Naito, T. & Okada, Y. HLA imputation and its application to genetic and molecular fine-mapping of the MHC region in autoimmune diseases. _Semin. Immunopathol._ **44**, 15–28 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xmtlalt7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34786601)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HLA%20imputation%20and%20its%20application%20to%20genetic%20and%20molecular%20fine-mapping%20of%20the%20MHC%20region%20in%20autoimmune%20diseases&journal=Semin.%20Immunopathol.&volume=44&pages=15-28&publication_year=2022&author=Naito%2CT&author=Okada%2CY) 
    
64.  Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._ **15**, 550 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25516281)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Love%2CMI&author=Huber%2CW&author=Anders%2CS) 
    
65.  Kanai, M., Tanaka, T. & Okada, Y. Empirical estimation of genome-wide significance thresholds based on the 1000 Genomes Project data set. _J. Hum. Genet._ **61**, 861–866 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhslKgtbzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27305981)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5090169)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Empirical%20estimation%20of%20genome-wide%20significance%20thresholds%20based%20on%20the%201000%20Genomes%20Project%20data%20set&journal=J.%20Hum.%20Genet.&volume=61&pages=861-866&publication_year=2016&author=Kanai%2CM&author=Tanaka%2CT&author=Okada%2CY) 
    
66.  Gupta, N. T. et al. Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data. _Bioinformatics_ **31**, 3356–3358 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1Git7%2FE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26069265)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4793929)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Change-O%3A%20a%20toolkit%20for%20analyzing%20large-scale%20B%20cell%20immunoglobulin%20repertoire%20sequencing%20data&journal=Bioinformatics&volume=31&pages=3356-3358&publication_year=2015&author=Gupta%2CNT) 
    
67.  Yu, G., Wang, L.-G., Han, Y. & He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. _Omics_ **16**, 284–287 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmsFarsLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22455463)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3339379)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=clusterProfiler%3A%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters&journal=Omics&volume=16&pages=284-287&publication_year=2012&author=Yu%2CG&author=Wang%2CL-G&author=Han%2CY&author=He%2CQ-Y) 
    
68.  Kang, J. B. et al. Mapping the dynamic genetic regulatory architecture of HLA genes at single-cell resolution. _Nat. Genet._ **55**, 2255–2268 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFSmu7rL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38036787)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10787945)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20the%20dynamic%20genetic%20regulatory%20architecture%20of%20HLA%20genes%20at%20single-cell%20resolution&journal=Nat.%20Genet.&volume=55&pages=2255-2268&publication_year=2023&author=Kang%2CJB) 
    
69.  Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. _Nat. Methods_ **16**, 1289–1296 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOqsr7N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31740819)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884693)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%2C%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&journal=Nat.%20Methods&volume=16&pages=1289-1296&publication_year=2019&author=Korsunsky%2CI) 
    
70.  Ishigaki, K. et al. Multi-ancestry genome-wide association analyses identify novel genetic mechanisms in rheumatoid arthritis. _Nat. Genet._ **54**, 1640–1651 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivVWqt7nJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36333501)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10165422)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-ancestry%20genome-wide%20association%20analyses%20identify%20novel%20genetic%20mechanisms%20in%20rheumatoid%20arthritis&journal=Nat.%20Genet.&volume=54&pages=1640-1651&publication_year=2022&author=Ishigaki%2CK) 
    
71.  Wang, Y. F. et al. Identification of 38 novel loci for systemic lupus erythematosus and genetic heterogeneity between ancestral groups. _Nat. Commun._ **12**, 772 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsVGrur0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33536424)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7858632)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%2038%20novel%20loci%20for%20systemic%20lupus%20erythematosus%20and%20genetic%20heterogeneity%20between%20ancestral%20groups&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Wang%2CYF) 
    
72.  Liu, Z. et al. Genetic architecture of the inflammatory bowel diseases across East Asian and European ancestries. _Nat. Genet._ **55**, 796–806 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXps1KntrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37156999)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10290755)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20architecture%20of%20the%20inflammatory%20bowel%20diseases%20across%20East%20Asian%20and%20European%20ancestries&journal=Nat.%20Genet.&volume=55&pages=796-806&publication_year=2023&author=Liu%2CZ) 
    
73.  Sakaue, S. et al. A cross-population atlas of genetic associations for 220 human phenotypes. _Nat. Genet._ **53**, 1415–1424 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFGru7fO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34594039)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12208603)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cross-population%20atlas%20of%20genetic%20associations%20for%20220%20human%20phenotypes&journal=Nat.%20Genet.&volume=53&pages=1415-1424&publication_year=2021&author=Sakaue%2CS) 
    
74.  Shirai, Y. et al. Multi-trait and cross-population genome-wide association studies across autoimmune and allergic diseases identify shared and distinct genetic component. _Ann. Rheum. Dis._ **81**, 1301–1312 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1yntLfN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35753705)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-trait%20and%20cross-population%20genome-wide%20association%20studies%20across%20autoimmune%20and%20allergic%20diseases%20identify%20shared%20and%20distinct%20genetic%20component&journal=Ann.%20Rheum.%20Dis.&volume=81&pages=1301-1312&publication_year=2022&author=Shirai%2CY) 
    
75.  Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. _PLoS Genet._ **10**, e1004383 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24830394)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4022491)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bayesian%20test%20for%20colocalisation%20between%20pairs%20of%20genetic%20association%20studies%20using%20summary%20statistics&journal=PLoS%20Genet.&volume=10&publication_year=2014&author=Giambartolomei%2CC) 
    
76.  Bentham, J. et al. Genetic association analyses implicate aberrant regulation of innate and adaptive immunity genes in the pathogenesis of systemic lupus erythematosus. _Nat. Genet._ **47**, 1457–1464 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslant7nF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26502338)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4668589)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20association%20analyses%20implicate%20aberrant%20regulation%20of%20innate%20and%20adaptive%20immunity%20genes%20in%20the%20pathogenesis%20of%20systemic%20lupus%20erythematosus&journal=Nat.%20Genet.&volume=47&pages=1457-1464&publication_year=2015&author=Bentham%2CJ) 
    
77.  Vuckovic, D. et al. The polygenic and monogenic basis of blood traits and diseases. _Cell_ **182**, 1214–1231 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsl2itbbP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32888494)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7482360)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20polygenic%20and%20monogenic%20basis%20of%20blood%20traits%20and%20diseases&journal=Cell&volume=182&pages=1214-1231&publication_year=2020&author=Vuckovic%2CD) 
    
78.  Robinson, M. D., McCarthy, D. J. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. _Bioinformatics_ **26**, 139–140 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1WlurvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19910308)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&journal=Bioinformatics&volume=26&pages=139-140&publication_year=2010&author=Robinson%2CMD&author=McCarthy%2CDJ&author=Smyth%2CGK) 
    
79.  Huang, X. & Huang, Y. Cellsnp-lite: an efficient tool for genotyping single cells. _Bioinformatics_ **37**, 4569–4571 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFKjsrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33963851)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cellsnp-lite%3A%20an%20efficient%20tool%20for%20genotyping%20single%20cells&journal=Bioinformatics&volume=37&pages=4569-4571&publication_year=2021&author=Huang%2CX&author=Huang%2CY) 
    
80.  Huang, Y., McCarthy, D. J. & Stegle, O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. _Genome Biol._ **20**, 273 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31836005)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6909514)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Vireo%3A%20Bayesian%20demultiplexing%20of%20pooled%20single-cell%20RNA-seq%20data%20without%20genotype%20reference&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Huang%2CY&author=McCarthy%2CDJ&author=Stegle%2CO) 
    
81.  Finak, G. et al. MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. _Genome Biol._ **16**, 278 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26653891)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4676162)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAST%3A%20a%20flexible%20statistical%20framework%20for%20assessing%20transcriptional%20changes%20and%20characterizing%20heterogeneity%20in%20single-cell%20RNA%20sequencing%20data&journal=Genome%20Biol.&volume=16&publication_year=2015&author=Finak%2CG) 
    
82.  Edahiro, R. REdahiro/OASIS\_project: v1.1. _Zenodo_ [https://doi.org/10.5281/zenodo.15877644](https://doi.org/10.5281/zenodo.15877644) (2025).
    
83.  Naito, T. tatsuhikonaito/OASIS\_HLATCR: v1.0.0. _Zenodo_ [https://doi.org/10.5281/zenodo.14991132](https://doi.org/10.5281/zenodo.14991132) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02266-3?format=refman&flavour=references)

Acknowledgements
----------------

We sincerely thank all participants involved in this study. We thank and acknowledge T. Shiroyama, K. Miyake, Y. Suga and Y. Naito for contributing to clinical practice. We are grateful to Y. Yamaguchi, T. Murakami, Y. Noda, T. Niitsu, Y. Adachi, T. Enomoto, S. Amiya, R. Hara, M. Yamamoto, T. Kuge, K. Matsumoto, M. Yoneda, Y. Yamamoto, Y. Yoshimine, S. Minoda, T. Hirayama, K. Funakoshi, Y. Okita, S. Kawada, D. Nakatsubo, T. Tada, M. Okamoto and H. Shimagami for contributing to clinical practice and sample collection. We thank all members of the JCTF for their support and sharing GWAS summary statistics. This study was supported by AMED (JP23km0405217, JP23ek0109594, JP23ek0410113, JP23kk0305022, JP223fa627001, JP223fa627002, JP223fa627010, JP233fa627011, JP23zf0127008, JP23tm0524002, JP24wm0625504, JP24gm1810011 to Y. Okada and 24ek0210207h0001 to R.E.), JST Moonshot R&D (JPMJMS2021, JPMJMS2024 to Y. Okada), JSPS KAKENHI (25H01057, 24K02697 to Y. Okada), AMED–CREST (22gm1810003h0001 to A. Kumanogoh and JP24gm1810012 to R.E.), a Japan Foundation for Applied Enzymology grant, the Takeda Science Foundation, Ono Pharmaceutical Foundation for Oncology, Immunology, and Neurology, the Bioinformatics Initiative of Osaka University Graduate School of Medicine, the Institute for Open and Transdisciplinary Research Initiatives, CiDER and CAMaD, Osaka University and the RIKEN TRIP initiative (AGIS). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.

Author information
------------------

Author notes

1.  These authors contributed equally: Ryuya Edahiro, Go Sato.
    
2.  These authors jointly supervised this work: Atsushi Kumanogoh, Yukinori Okada.
    

### Authors and Affiliations

1.  Department of Statistical Genetics, Osaka University Graduate School of Medicine, Suita, Japan
    
    Ryuya Edahiro, Go Sato, Tatsuhiko Naito, Yuya Shirai, Kyuto Sonehara, Yoshihiko Tomofuji, Kenichi Yamamoto, Shinichi Namba, Noah Sasa, Qingbo S. Wang, Yugo Takahashi, Toshihiro Kishikawa, Ken Suzuki, Qingbo S. Wang, Ryuya Edahiro, Yuya Shirai, Tatsuhiko Naito, Noah Sasa, Shinichi Namba, Yukinori Okada & Yukinori Okada
    
2.  Department of Respiratory Medicine and Clinical Immunology, Osaka University Graduate School of Medicine, Suita, Japan
    
    Ryuya Edahiro, Yuya Shirai, Ryuya Edahiro, Yuya Shirai, Takayuki Shiroyama, Yoshimi Noda, Takayuki Niitsu, Yuichi Adachi, Takatoshi Enomoto, Saori Amiya, Reina Hara, Yuta Yamaguchi, Teruaki Murakami, Tomoki Kuge, Kinnosuke Matsumoto, Yuji Yamamoto, Makoto Yamamoto, Midori Yoneda, Atsushi Kumanogoh, Takayoshi Morita, Yasuhiro Kato, Haruhiko Hirata, Yoshito Takeda & Atsushi Kumanogoh
    
3.  Laboratory for Systems Genetics, RIKEN Center for Integrative Medical Sciences, Yokohama, Japan
    
    Ryuya Edahiro, Go Sato, Tatsuhiko Naito, Kyuto Sonehara, Yoshihiko Tomofuji, Shinichi Namba, Noah Sasa, Yukinori Okada & Yukinori Okada
    
4.  Department of Gastroenterological Surgery, Osaka University Graduate School of Medicine, Suita, Japan
    
    Go Sato, Mamoru Uemura, Yuichiro Doki & Hidetoshi Eguchi
    
5.  Laboratory of Statistical Immunology, Immunology Frontier Research Center (WPI-IFReC), Osaka University, Suita, Japan
    
    Yuya Shirai, Qingbo S. Wang, Yukinori Okada & Yukinori Okada
    
6.  Department of Pathology and Tumor Biology, Kyoto University, Kyoto, Japan
    
    Ryunosuke Saiki, Yasuhito Nannya, Seishi Ogawa & Seishi Ogawa
    
7.  Department of Genome Informatics, Graduate School of Medicine, the University of Tokyo, Tokyo, Japan
    
    Kyuto Sonehara, Yoshihiko Tomofuji, Shinichi Namba, Genta Nagao, Qingbo S. Wang, Yukinori Okada & Yukinori Okada
    
8.  Department of Pediatrics, Osaka University Graduate School of Medicine, Suita, Japan
    
    Kenichi Yamamoto
    
9.  Division of Health Sciences, Osaka University Graduate School of Medicine, Suita, Japan
    
    Kenichi Yamamoto
    
10.  Department of Otorhinolaryngology—Head and Neck Surgery, Osaka University Graduate School of Medicine, Suita, Japan
    
    Noah Sasa, Toshihiro Kishikawa & Noah Sasa
    
11.  Division of Pulmonary Medicine, Department of Medicine, Keio University School of Medicine, Tokyo, Japan
    
    Genta Nagao, Hiromu Tanaka, Shuhei Azekawa, Ho Lee, Hiroki Kabata, Katsunori Masaki, Hirofumi Kamata, Shinnosuke Ikemura, Shotaro Chubachi, Satoshi Okamori, Hideki Terai, Atsuho Morita, Takanori Asakura, Genta Nagao, Makoto Ishii & Koichi Fukunaga
    
12.  M&D Data Science Center, Tokyo Medical and Dental University, Tokyo, Japan
    
    Takanori Hasegawa, Kunihiko Takahashi, Tatsuhiko Anzai, Satoshi Ito & Satoru Miyano
    
13.  Department of Head and Neck Surgery, Aichi Cancer Center Hospital, Nagoya, Japan
    
    Toshihiro Kishikawa
    
14.  Department of Diabetes and Metabolic Diseases, Graduate School of Medicine, the University of Tokyo, Tokyo, Japan
    
    Ken Suzuki
    
15.  Laboratory of Human Immunology (Single Cell Genomics), WPI-IFReC, Osaka University, Suita, Japan
    
    Yu-Chen Liu, Daisuke Motooka, Ayako Takuwa & Daisuke Okuzaki
    
16.  Genome Information Research Center, Research Institute for Microbial Diseases, Osaka University, Suita, Japan
    
    Daisuke Motooka & Daisuke Okuzaki
    
17.  Integrated Frontier Research for Medical Science Division, Institute for Open and Transdisciplinary Research Initiatives, Osaka University, Suita, Japan
    
    Daisuke Motooka, Atsushi Kumanogoh, Daisuke Okuzaki & Atsushi Kumanogoh
    
18.  Department of Infectious Diseases, Keio University School of Medicine, Tokyo, Japan
    
    Sho Uchida, Shunsuke Uno, Tomoyasu Nishimura, Naoki Hasegawa & Ho Namkoong
    
19.  Health Science Research and Development Center (HeRD), Tokyo Medical and Dental University, Tokyo, Japan
    
    Ryuji Koike
    
20.  Institute of Science Tokyo, Tokyo, Japan
    
    Akinori Kimura
    
21.  Division of Health Medical Intelligence, Human Genome Center, the Institute of Medical Science, the University of Tokyo, Tokyo, Japan
    
    Takayoshi Hyugaji, Eigo Shimizu, Kotoe Katayama & Seiya Imoto
    
22.  Division of Gastroenterology and Hepatology, Department of Medicine, Keio University School of Medicine, Tokyo, Japan
    
    Kosaku Nanki, Takanori Kanai & Takanori Kanai
    
23.  Department of Immunopathology, WPI-IFReC, Osaka University, Suita, Japan
    
    Yuta Yamaguchi, Teruaki Murakami, Atsushi Kumanogoh, Takayoshi Morita, Yasuhiro Kato & Atsushi Kumanogoh
    
24.  Center for Infectious Disease Education and Research (CiDER), Osaka University, Suita, Japan
    
    Atsushi Kumanogoh, Daisuke Okuzaki & Atsushi Kumanogoh
    
25.  Japan Agency for Medical Research and Development–Core Research for Evolutional Science and Technology (AMED–CREST), Osaka University, Suita, Japan
    
    Atsushi Kumanogoh, Daisuke Okuzaki & Atsushi Kumanogoh
    
26.  Laboratory of Systems Immunology, Immunology Frontier Research Center, Osaka University, Suita, Japan
    
    Shuhei Sakakibara
    
27.  Graduate School of Medical Safety Management, Jikei University of Health Care Sciences, Osaka, Japan
    
    Shuhei Sakakibara
    
28.  Institute for the Advanced Study of Human Biology (WPI-ASHBi), Kyoto University, Kyoto, Japan
    
    Seishi Ogawa & Seishi Ogawa
    
29.  Center for Advanced Modalities and DDS (CAMaD), Osaka University, Suita, Japan
    
    Atsushi Kumanogoh
    
30.  Premium Research Institute for Human Metaverse Medicine (WPI-PRIMe), Osaka University, Suita, Japan
    
    Yukinori Okada
    
31.  Department of Biomedical Informatics, Harvard Medical School, Boston, MA, USA
    
    Masahiro Kanai
    
32.  Department of Respiratory Medicine, Juntendo University Faculty of Medicine and Graduate School of Medicine, Tokyo, Japan
    
    Kazuhisa Takahashi, Norihiro Harada, Haruhi Takagi, Masako Ichikawa, Ai Nakamura, Sonoko Harada & Yuuki Sandhu
    
33.  Department of General Medicine, Juntendo University Faculty of Medicine and Graduate School of Medicine, Tokyo, Japan
    
    Toshio Naito
    
34.  Department of Emergency and Disaster Medicine, Juntendo University Faculty of Medicine and Graduate School of Medicine, Tokyo, Japan
    
    Makoto Hiki
    
35.  Department of Cardiovascular Biology and Medicine, Juntendo University Faculty of Medicine and Graduate School of Medicine, Tokyo, Japan
    
    Makoto Hiki
    
36.  Department of Internal Medicine and Rheumatology, Juntendo University Faculty of Medicine and Graduate School of Medicine, Tokyo, Japan
    
    Yasushi Matsushita
    
37.  Atopy (Allergy) Research Center, Juntendo University Graduate School of Medicine, Tokyo, Japan
    
    Sonoko Harada
    
38.  Department of Emergency and Critical Care Medicine, Keio University School of Medicine, Tokyo, Japan
    
    Junichi Sasaki
    
39.  Department of Anesthesiology, Keio University School of Medicine, Tokyo, Japan
    
    Hiroshi Morisaki
    
40.  Department of Laboratory Medicine, Keio University School of Medicine, Tokyo, Japan
    
    Yoshifumi Uwamino
    
41.  Keio University Health Center, Tokyo, Japan
    
    Tomoyasu Nishimura
    
42.  Department of Respiratory Medicine, Saitama Cardiovascular and Respiratory Center, Kumagaya, Japan
    
    Takashi Ishiguro, Taisuke Isono, Shun Shibata, Yuma Matsui, Chiaki Hosoda, Kenji Takano, Takashi Nishida, Yoichi Kobayashi, Yotaro Takaku & Noboru Takayanagi
    
43.  Internal Medicine, Japan Community Health Care Organization, Saitama Medical Center, Saitama, Japan
    
    Soichiro Ueda, Ai Tada, Masayoshi Miyawaki, Masaomi Yamamoto, Eriko Yoshida, Reina Hayashi, Tomoki Nagasaka, Sawako Arai, Yutaro Kaneko & Kana Sasaki
    
44.  Department of Respiratory Medicine, Tokyo Women’s Medical University, Tokyo, Japan
    
    Etsuko Tagaya & Ken Arimura
    
45.  Department of General Medicine, Tokyo Women’s Medical University, Tokyo, Japan
    
    Masatoshi Kawana
    
46.  Clinical Research Center, Tokyo Medical and Dental University Hospital of Medicine, Tokyo, Japan
    
    Akifumi Endo
    
47.  Department of Medical Informatics, Tokyo Medical and Dental University Hospital of Medicine, Tokyo, Japan
    
    Yuji Uchimura
    
48.  Respiratory Medicine, Tokyo Medical and Dental University, Tokyo, Japan
    
    Yasunari Miyazaki, Takayuki Honda & Tomoya Tateishi
    
49.  Clinical Laboratory, Tokyo Medical and Dental University Hospital of Medicine, Tokyo, Japan
    
    Shuji Tohda, Naoya Ichimura, Kazunari Sonobe, Chihiro Tani Sassa & Jun Nakajima
    
50.  Department of Internal Medicine, Kawasaki Municipal Ida Hospital, Kawasaki, Japan
    
    Yasushi Nakano, Yukiko Nakajima, Ryusuke Anan, Ryosuke Arai, Yuko Kurihara, Yuko Harada, Kazumi Nishio & Tetsuya Ueda
    
51.  Department of Respiratory Medicine, Osaka Saiseikai Nakatsu Hospital, Osaka, Japan
    
    Masanori Azuma, Ryuichi Saito, Toshikatsu Sado, Yoshimune Miyazaki, Ryuichi Sato, Yuki Haruta, Tadao Nagasaki & Yoshinori Hasegawa
    
52.  Department of Infection Control, Osaka Saiseikai Nakatsu Hospital, Osaka, Japan
    
    Yoshinori Yasui
    
53.  Department of Infectious Diseases, Tosei General Hospital, Seto, Japan
    
    Yoshikazu Mutoh
    
54.  Department of Respiratory, Allergic Diseases Internal Medicine, Tosei General Hospital, Seto, Japan
    
    Tomoki Kimura, Tomonori Sato, Reoto Takei, Satoshi Hagimoto, Yoichiro Noguchi, Yasuhiko Yamano, Hajime Sasano & Sho Ota
    
55.  Department of Emergency and Critical Care Medicine, Kansai Medical University General Medical Center, Moriguchi, Japan
    
    Yasushi Nakamori, Kazuhisa Yoshiya, Fukuki Saito, Tomoyuki Yoshihara, Daiki Wada, Hiromu Iwamura, Syuji Kanayama & Shuhei Maruyama
    
56.  Japan Anti-Tuberculosis Association, Fukujuji Hospital, Kiyose, Japan
    
    Takashi Yoshiyama, Ken Ohta, Hiroyuki Kokuto, Hideo Ogata, Yoshiaki Tanaka, Kenichi Arakawa, Masafumi Shimoda & Takeshi Osawa
    
57.  Department of Pulmonary Medicine, Saitama City Hospital, Saitama, Japan
    
    Hiroki Tateno, Isano Hase, Shuichi Yoshida & Shoji Suzuki
    
58.  Department of Infectious Diseases, Saitama City Hospital, Saitama, Japan
    
    Miki Kawada
    
59.  Department of General Thoracic Surgery, Saitama City Hospital, Saitama, Japan
    
    Hirohisa Horinouchi
    
60.  Department of Pulmonary Medicine, Eiju General Hospital, Tokyo, Japan
    
    Fumitake Saito & Junichi Ochi
    
61.  Division of Infection Control, Eiju General Hospital, Tokyo, Japan
    
    Keiko Mitamura
    
62.  Department of Hematology, Eiju General Hospital, Tokyo, Japan
    
    Masao Hagihara & Tomoyuki Uchida
    
63.  Saiseikai Utsunomiya Hospital, Utsunomiya, Japan
    
    Rie Baba, Daisuke Arai, Takayuki Ogura, Hidenori Takahashi, Shigehiro Hagiwara, Shunichiro Konishi & Ichiro Nakachi
    
64.  Department of Respiratory Medicine, Tohoku University Graduate School of Medicine, Sendai, Japan
    
    Koji Murakami, Mitsuhiro Yamada, Hisatoshi Sugiura, Hirohito Sano, Shuichiro Matsumoto, Nozomu Kimura & Yoshinao Ono
    
65.  Department of Infectious Diseases, Tohoku University Graduate School of Medicine, Sendai, Japan
    
    Hiroaki Baba
    
66.  Department of Respiratory Medicine, Kitasato University Kitasato Institute Hospital, Tokyo, Japan
    
    Yusuke Suzuki, Sohei Nakayama & Keita Masuzawa
    
67.  Division of Infection Control and Prevention, Osaka University Hospital, Suita, Japan
    
    Kazunori Tomono
    
68.  Department of Biomedical Ethics and Public Policy, Osaka University Graduate School of Medicine, Suita, Japan
    
    Kazuto Kato
    
69.  Tachikawa Hospital, Tachikawa, Japan
    
    Hidefumi Koh, Tadashi Manabe, Yohei Funatsu, Fumimaro Ito, Takahiro Fukui, Keisuke Shinozuka, Sumiko Kohashi & Masatoshi Miyazaki
    
70.  Department of Emergency and Critical Care Medicine, Tokyo Women’s Medical University Medical Center East, Tokyo, Japan
    
    Tomohisa Shoko, Mitsuaki Kojima & Tomohiro Adachi
    
71.  Department of Medicine, Tokyo Women’s Medical University Medical Center East, Tokyo, Japan
    
    Motonao Ishikawa
    
72.  Department of Pediatrics, Tokyo Women’s Medical University Medical Center East, Tokyo, Japan
    
    Kenichiro Takahashi
    
73.  Internal Medicine, Sano Kosei General Hospital, Sano, Japan
    
    Takashi Inoue, Toshiyuki Hirano, Keigo Kobayashi & Hatsuyo Takaoka
    
74.  Japan Community Health Care Organization, Kanazawa Hospital, Kanazawa, Japan
    
    Kazuyoshi Watanabe
    
75.  Department of Respiratory Medicine, Saiseikai Yokohamashi Nanbu Hospital, Yokohama, Japan
    
    Naoki Miyazawa, Yasuhiro Kimura, Reiko Sado & Hideyasu Sugimoto
    
76.  Department of Clinical Laboratory, Saiseikai Yokohamashi Nanbu Hospital, Yokohama, Japan
    
    Akane Kamiya
    
77.  Internal Medicine, Internal Medicine Center, Showa University Koto Toyosu Hospital, Tokyo, Japan
    
    Naota Kuwahara, Akiko Fujiwara, Tomohiro Matsunaga, Yoko Sato & Takenori Okada
    
78.  Department of Respiratory Medicine, Japan Organization of Occupational Health and Safety, Kanto Rosai Hospital, Kawasaki, Japan
    
    Yoshihiro Hirai, Hidetoshi Kawashima & Atsuya Narita
    
79.  Department of General Internal Medicine, Japan Organization of Occupational Health and Safety, Kanto Rosai Hospital, Kawasaki, Japan
    
    Kazuki Niwa & Yoshiyuki Sekikawa
    
80.  Ishikawa Prefectural Central Hospital, Kanazawa, Japan
    
    Koichi Nishi, Masaru Nishitsuji, Mayuko Tani, Junya Suzuki & Hiroki Nakatsumi
    
81.  Kanagawa Cardiovascular and Respiratory Center, Yokohama, Japan
    
    Takashi Ogura, Hideya Kitamura, Eri Hagiwara, Kota Murohashi & Hiroko Okabayashi
    
82.  Department of Respiratory Medicine, National Hospital Organization Tokyo Medical Center, Tokyo, Japan
    
    Takao Mochimaru, Shigenari Nukaga, Ryosuke Satomi & Yoshitaka Oyamada
    
83.  Department of Allergy, National Hospital Organization Tokyo Medical Center, Tokyo, Japan
    
    Takao Mochimaru & Yoshitaka Oyamada
    
84.  Department of General Internal Medicine and Infectious Diseases, National Hospital Organization Tokyo Medical Center, Tokyo, Japan
    
    Nobuaki Mori
    
85.  Department of Respiratory Medicine, Toyohashi Municipal Hospital, Toyohashi, Japan
    
    Tomoya Baba, Yasutaka Fukui, Mitsuru Odate, Shuko Mashimo & Yasushi Makino
    
86.  Keiyu Hospital, Yokohama, Japan
    
    Kazuma Yagi, Mizuha Hashiguchi, Junko Kagyo & Tetsuya Shiomi
    
87.  Department of Respiratory Medicine, KKR Sapporo Medical Center, Sapporo, Japan
    
    Satoshi Fuke & Hiroshi Saito
    
88.  Division of General Internal Medicine, Department of Internal Medicine, St. Marianna University School of Medicine, Kawasaki, Japan
    
    Tomoya Tsuchida
    
89.  Department of Emergency and Critical Care Medicine, St. Marianna University School of Medicine, Kawasaki, Japan
    
    Shigeki Fujitani, Mumon Takita, Daiki Morikawa & Toru Yoshida
    
90.  Japanese Red Cross Medical Center, Tokyo, Japan
    
    Takehiro Izumo, Minoru Inomata, Naoyuki Kuse, Nobuyasu Awano & Mari Tone
    
91.  Matsumoto City Hospital, Matsumoto, Japan
    
    Akihiro Ito
    
92.  Department of Emergency and Critical Care Medicine, Faculty of Medicine, Fukuoka University, Fukuoka, Japan
    
    Yoshihiko Nakamura, Kota Hoshino, Junichi Maruyama & Hiroyasu Ishikura
    
93.  Department of Infection Control, Fukuoka University Hospital, Fukuoka, Japan
    
    Tohru Takata
    
94.  Department of Rheumatology, National Hospital Organization Hokkaido Medical Center, Sapporo, Japan
    
    Toshio Odani
    
95.  Department of Respiratory Medicine, National Hospital Organization Hokkaido Medical Center, Sapporo, Japan
    
    Masaru Amishima & Takeshi Hattori
    
96.  Department of Emergency and Critical Care Medicine, National Hospital Organization Hokkaido Medical Center, Sapporo, Japan
    
    Yasuo Shichinohe
    
97.  National Hospital Organization Kanazawa Medical Center, Kanazawa, Japan
    
    Takashi Kagaya, Toshiyuki Kita, Kazuhide Ohta, Satoru Sakagami & Kiyoshi Koshida
    
98.  Division of Respiratory Medicine, Department of Internal Medicine, Nihon University School of Medicine, Tokyo, Japan
    
    Kentaro Hayashi, Tetsuo Shimizu, Yutaka Kozu, Hisato Hiranuma & Yasuhiro Gon
    
99.  Musashino Red Cross Hospital, Musashino, Japan
    
    Namiki Izumi, Kaoru Nagata, Ken Ueda, Reiko Taki & Satoko Hanada
    
100.  Division of Respiratory Medicine, Social Welfare Organization Saiseikai Imperial Gift Foundation, Inc., Saiseikai Kumamoto Hospital, Kumamoto, Japan
    
    Kodai Kawamura, Kazuya Ichikado, Kenta Nishiyama, Hiroyuki Muranaka & Kazunori Nakamura
    
101.  Department of Respiratory Medicine, Nagoya University Graduate School of Medicine, Nagoya, Japan
    
    Naozumi Hashimoto, Keiko Wakahara, Sakamoto Koji, Norihito Omote & Akira Ando
    
102.  Department of Internal Medicine, Fukuoka Tokushukai Hospital, Kasuga, Japan
    
    Nobuhiro Kodama, Yasunari Kaneyama & Shunsuke Maeda
    
103.  Respiratory Medicine, Fukuoka Tokushukai Hospital, Kasuga, Japan
    
    Takashige Kuraki & Takemasa Matsumoto
    
104.  Department of Endocrinology, Hematology and Gerontology, Chiba University Graduate School of Medicine, Chiba, Japan
    
    Koutaro Yokote
    
105.  Department of Emergency and Critical Care Medicine, Chiba University Graduate School of Medicine, Chiba, Japan
    
    Taka-Aki Nakada, Ryuzo Abe, Taku Oshima & Tadanaga Shimada
    
106.  National Hospital Organization Kumamoto Medical Center, Kumamoto, Japan
    
    Masahiro Harada, Takeshi Takahashi, Hiroshi Ono, Toshihiro Sakurai & Takayuki Shibusawa
    
107.  Division of Infectious Diseases and Respiratory Medicine, Department of Internal Medicine, National Defense Medical College, Tokorozawa, Japan
    
    Yoshifumi Kimizuka, Akihiko Kawana, Tomoya Sano, Chie Watanabe & Ryohei Suematsu
    
108.  Sapporo City General Hospital, Sapporo, Japan
    
    Hisako Sageshima
    
109.  Department of Internal Medicine, Tokyo Saiseikai Central Hospital, Tokyo, Japan
    
    Ayumi Yoshifuji & Kazuto Ito
    
110.  Department of Pulmonary Medicine, Tokyo Saiseikai Central Hospital, Tokyo, Japan
    
    Saeko Takahashi, Kota Ishioka & Morio Nakamura
    
111.  Department of Respiratory Medicine, Fujisawa City Hospital, Fujisawa, Japan
    
    Makoto Masuda, Aya Wakabayashi, Hiroki Watanabe, Suguru Ueda & Masanori Nishikawa
    
112.  Uji-Tokushukai Medical Center, Uji, Japan
    
    Yusuke Chihara, Mayumi Takeuchi, Keisuke Onoi, Jun Shinozuka & Atsushi Sueyoshi
    
113.  Department of Infectious Disease and Clinical Research Institute, National Hospital Organization Kyushu Medical Center, Fukuoka, Japan
    
    Yoji Nagasaki
    
114.  Department of Respirology, National Hospital Organization Kyushu Medical Center, Fukuoka, Japan
    
    Masaki Okamoto & Yoshihisa Tokunaga
    
115.  Division of Respirology, Rheumatology and Neurology, Department of Internal Medicine, Kurume University School of Medicine, Kurume, Japan
    
    Masaki Okamoto & Yoshihisa Tokunaga
    
116.  Department of Infectious Disease, National Hospital Organization Kyushu Medical Center, Fukuoka, Japan
    
    Sayoko Ishihara & Masatoshi Shimo
    
117.  Ome Municipal General Hospital, Ome, Japan
    
    Yu Kusaka, Takehiko Ohba, Susumu Isogai, Aki Ogawa & Takuya Inoue
    
118.  Research Institute for Diseases of the Chest, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Japan
    
    Satoru Fukuyama, Keiko Kan-o & Koichiro Matsumoto
    
119.  Department of Medicine and Biosystemic Science, Kyushu University Graduate School of Medical Sciences, Fukuoka, Japan
    
    Yoshihiro Eriguchi & Akiko Yonekawa
    
120.  Daini Osaka Police Hospital, Osaka, Japan
    
    Kensuke Kanaoka, Shoichi Ihara & Kiyoshi Komuta
    
121.  Department of Emergency and Critical Care Medicine, Faculty of Medicine, University of Tsukuba, Tsukuba, Japan
    
    Yoshiaki Inoue
    
122.  Department of Hematology, Faculty of Medicine, University of Tsukuba, Tsukuba, Japan
    
    Shigeru Chiba
    
123.  Department of Nephrology, Faculty of Medicine, University of Tsukuba, Tsukuba, Japan
    
    Kunihiro Yamagata & Hirayasu Kai
    
124.  Department of Cardiovascular Surgery, Faculty of Medicine, University of Tsukuba, Tsukuba, Japan
    
    Yuji Hiramatsu
    
125.  Division of Pulmonary Medicine, Department of Medicine, Tokai University School of Medicine, Isehara, Japan
    
    Koichiro Asano, Tsuyoshi Oguma & Yoko Ito
    
126.  Department of Anesthesiology and Intensive Care Medicine, Kyoto Prefectural University of Medicine, Kyoto, Japan
    
    Satoru Hashimoto & Masaki Yamasaki
    
127.  Department of Infection Control and Laboratory Medicine, Kyoto Prefectural University of Medicine, Kyoto, Japan
    
    Yu Kasamatsu
    
128.  Department of Respiratory Internal Medicine, St. Marianna University School of Medicine, Yokohama City Seibu Hospital, Yokohama, Japan
    
    Yuko Komase, Naoya Hida, Takahiro Tsuburai & Baku Oyama
    
129.  Kinshukai Hanwa the Second Hospital, Osaka, Japan
    
    Minoru Takada & Hidenori Kanda
    
130.  Emergency and Disaster Medicine, Gifu University School of Medicine Graduate School of Medicine, Gifu, Japan
    
    Yuichiro Kitagawa, Tetsuya Fukuta, Takahito Miyake, Shozo Yoshida & Shinji Ogura
    
131.  Department of Respiratory Medicine, Tokyo Medical University Hospital, Tokyo, Japan
    
    Shinji Abe, Yuta Kono, Yuki Togashi, Hiroyuki Takoi & Ryota Kikuchi
    
132.  JA Toride medical hospital, Toride, Japan
    
    Shinichi Ogawa, Tomouki Ogata & Shoichiro Ishihara
    
133.  Okayama Rosai Hospital, Okayama, Japan
    
    Arihiko Kanehiro, Shinji Ozaki, Yasuko Fuchimoto, Sae Wada & Nobukazu Fujimoto
    
134.  Himeji St. Mary’s Hospital, Himeji, Japan
    
    Arihiko Kanehiro
    
135.  Emergency and Critical Care, Niigata University, Niigata, Japan
    
    Kei Nishiyama
    
136.  Emergency and Critical Care Center, National Hospital Organization Kyoto Medical Center, Kyoto, Japan
    
    Mariko Terashima, Satoru Beppu & Kosuke Yoshida
    
137.  National Hospital Organization Tokyo Hospital Hospital, Kiyose, Japan
    
    Osamu Narumoto, Hideaki Nagai & Nobuharu Ooshima
    
138.  Fujioka General Hospital, Fujioka, Japan
    
    Mitsuru Motegi
    
139.  Department of General Medicine, School of Medicine, International University of Health and Welfare Shioya Hospital, Ohtawara, Japan
    
    Akira Umeda & Yoshiyuki Ohira
    
140.  Department of Pharmacology, School of Pharmacy, International University of Health and Welfare Shioya Hospital, Ohtawara, Japan
    
    Kazuya Miyagawa
    
141.  Department of Respiratory Medicine, International University of Health and Welfare Shioya Hospital, Ohtawara, Japan
    
    Hisato Shimada
    
142.  Department of Clinical Laboratory, International University of Health and Welfare Shioya Hospital, Ohtawara, Japan
    
    Mayu Endo
    
143.  Department of Cardiology, Pulmonology and Nephrology, Yamagata University Faculty of Medicine, Yamagata, Japan
    
    Masafumi Watanabe, Sumito Inoue, Akira Igarashi & Masamichi Sato
    
144.  Division of Respiratory Medicine and Allergology, Department of Medicine, School of Medicine, Showa University, Tokyo, Japan
    
    Hironori Sagara, Akihiko Tanaka, Shin Ohta & Tomoyuki Kimura
    
145.  Department of Pulmonary Medicine, Fukushima Medical University, Fukushima, Japan
    
    Yoko Shibata, Yoshinori Tanino, Takefumi Nikaido, Hiroyuki Minemura & Yuki Sato
    
146.  Kansai Electric Power Hospital, Osaka, Japan
    
    Yuichiro Yamada, Takuya Hashino & Masato Shinoki
    
147.  Division of Infectious Diseases, Kumamoto City Hospital, Kumamoto, Japan
    
    Hajime Iwagoe
    
148.  Department of Respiratory Medicine, Kumamoto City Hospital, Kumamoto, Japan
    
    Hiroshi Takahashi, Kazuhiko Fujii & Hiroto Kishi
    
149.  Department of Emergency and Critical Care Medicine, Tokyo Metropolitan Police Hospital, Tokyo, Japan
    
    Masayuki Kanai, Tomonori Imamura & Tatsuya Yamashita
    
150.  Department of Respiratory Medicine, Gunma University Graduate School of Medicine, Maebashi, Japan
    
    Masakiyo Yatomi & Toshitaka Maeno
    
151.  National Hospital Organization Saitama Hospital, Wako, Japan
    
    Shinichi Hayashi, Mai Takahashi, Mizuki Kuramochi, Isamu Kamimaki & Yoshiteru Tominaga
    
152.  Tokyo Medical University Ibaraki Medical Center, Inashiki, Japan
    
    Tomoo Ishii
    
153.  Department of Internal Medicine, Kiryu Kosei General Hospital, Kiryu, Japan
    
    Mitsuyoshi Utsugi & Akihiro Ono
    
154.  Department of Pulmonary Medicine and Oncology, Graduate School of Medicine, Nippon Medical School, Tokyo, Japan
    
    Toru Tanaka, Takeru Kashiwada, Kazue Fujita, Yoshinobu Saito & Masahiro Seike
    
155.  Division of Respiratory Medicine, Tsukuba Kinen General Hospital, Tsukuba, Japan
    
    Hiroko Watanabe
    
156.  Division of Respiratory Medicine, Department of Internal Medicine, Toho University Ohashi Medical Center, Tokyo, Japan
    
    Hiroto Matsuse, Norio Kodaka, Chihiro Nakano, Takeshi Oshio & Takatomo Hirouchi
    
157.  Division of Anesthesiology, Department of Surgery Related, Kobe University Graduate School of Medicine, Kobe, Japan
    
    Shohei Makino & Moritoki Egi
    
158.  Genome Medical Science Project (Toyama), National Center for Global Health and Medicine, Tokyo, Japan
    
    Yosuke Omae & Katsushi Tokunaga
    
159.  Department of Biomolecular Engineering, Graduate School of Tokyo Institute of Technology, Tokyo, Japan
    
    Takafumi Ueno
    
160.  Laboratory of Veterinary Infectious Disease, School of Veterinary Medicine, Kitasato University, Aomori, Japan
    
    Tomomi Takano
    
161.  Laboratory of Viral Infection, Department of Infection Control and Immunology, Ōmura Satoshi Memorial Institute and Graduate School of Infection Control Sciences, Kitasato University, Tokyo, Japan
    
    Kazuhiko Katayama
    
162.  Department of Insured Medical Care Management, Tokyo Medical and Dental University Hospital of Medicine, Tokyo, Japan
    
    Masumi Ai
    
163.  Department of Organoid Medicine, Keio University School of Medicine, Tokyo, Japan
    
    Toshiro Sato
    
164.  Department of Surgery, Keio University School of Medicine, Tokyo, Japan
    
    Yuko Kitagawa
    
165.  Department of Medicine, Center for Hematology and Regenerative Medicine, Karolinska Institute, Stockholm, Sweden
    
    Seishi Ogawa
    
166.  AMED–CREST, Tokyo, Japan
    
    Takanori Kanai
    

Authors

1.  Ryuya Edahiro
    
    [View author publications](https://www.nature.com/search?author=Ryuya%20Edahiro)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ryuya%20Edahiro) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ryuya%20Edahiro%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Go Sato
    
    [View author publications](https://www.nature.com/search?author=Go%20Sato)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Go%20Sato) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Go%20Sato%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Tatsuhiko Naito
    
    [View author publications](https://www.nature.com/search?author=Tatsuhiko%20Naito)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tatsuhiko%20Naito) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tatsuhiko%20Naito%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Yuya Shirai
    
    [View author publications](https://www.nature.com/search?author=Yuya%20Shirai)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuya%20Shirai) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuya%20Shirai%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Ryunosuke Saiki
    
    [View author publications](https://www.nature.com/search?author=Ryunosuke%20Saiki)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ryunosuke%20Saiki) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ryunosuke%20Saiki%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Kyuto Sonehara
    
    [View author publications](https://www.nature.com/search?author=Kyuto%20Sonehara)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kyuto%20Sonehara) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kyuto%20Sonehara%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Yoshihiko Tomofuji
    
    [View author publications](https://www.nature.com/search?author=Yoshihiko%20Tomofuji)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yoshihiko%20Tomofuji) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yoshihiko%20Tomofuji%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Kenichi Yamamoto
    
    [View author publications](https://www.nature.com/search?author=Kenichi%20Yamamoto)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kenichi%20Yamamoto) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kenichi%20Yamamoto%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Shinichi Namba
    
    [View author publications](https://www.nature.com/search?author=Shinichi%20Namba)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shinichi%20Namba) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shinichi%20Namba%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Noah Sasa
    
    [View author publications](https://www.nature.com/search?author=Noah%20Sasa)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Noah%20Sasa) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Noah%20Sasa%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Genta Nagao
    
    [View author publications](https://www.nature.com/search?author=Genta%20Nagao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Genta%20Nagao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Genta%20Nagao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Qingbo S. Wang
    
    [View author publications](https://www.nature.com/search?author=Qingbo%20S.%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qingbo%20S.%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qingbo%20S.%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Yugo Takahashi
    
    [View author publications](https://www.nature.com/search?author=Yugo%20Takahashi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yugo%20Takahashi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yugo%20Takahashi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Takanori Hasegawa
    
    [View author publications](https://www.nature.com/search?author=Takanori%20Hasegawa)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Takanori%20Hasegawa) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Takanori%20Hasegawa%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Toshihiro Kishikawa
    
    [View author publications](https://www.nature.com/search?author=Toshihiro%20Kishikawa)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Toshihiro%20Kishikawa) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Toshihiro%20Kishikawa%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Ken Suzuki
    
    [View author publications](https://www.nature.com/search?author=Ken%20Suzuki)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ken%20Suzuki) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ken%20Suzuki%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Yu-Chen Liu
    
    [View author publications](https://www.nature.com/search?author=Yu-Chen%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yu-Chen%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yu-Chen%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Daisuke Motooka
    
    [View author publications](https://www.nature.com/search?author=Daisuke%20Motooka)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daisuke%20Motooka) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daisuke%20Motooka%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Ayako Takuwa
    
    [View author publications](https://www.nature.com/search?author=Ayako%20Takuwa)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ayako%20Takuwa) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ayako%20Takuwa%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Hiromu Tanaka
    
    [View author publications](https://www.nature.com/search?author=Hiromu%20Tanaka)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hiromu%20Tanaka) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hiromu%20Tanaka%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Shuhei Azekawa
    
    [View author publications](https://www.nature.com/search?author=Shuhei%20Azekawa)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shuhei%20Azekawa) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shuhei%20Azekawa%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Ho Namkoong
    
    [View author publications](https://www.nature.com/search?author=Ho%20Namkoong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ho%20Namkoong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ho%20Namkoong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Ryuji Koike
    
    [View author publications](https://www.nature.com/search?author=Ryuji%20Koike)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ryuji%20Koike) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ryuji%20Koike%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Akinori Kimura
    
    [View author publications](https://www.nature.com/search?author=Akinori%20Kimura)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Akinori%20Kimura) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Akinori%20Kimura%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Seiya Imoto
    
    [View author publications](https://www.nature.com/search?author=Seiya%20Imoto)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Seiya%20Imoto) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Seiya%20Imoto%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Satoru Miyano
    
    [View author publications](https://www.nature.com/search?author=Satoru%20Miyano)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Satoru%20Miyano) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Satoru%20Miyano%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  Takanori Kanai
    
    [View author publications](https://www.nature.com/search?author=Takanori%20Kanai)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Takanori%20Kanai) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Takanori%20Kanai%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Koichi Fukunaga
    
    [View author publications](https://www.nature.com/search?author=Koichi%20Fukunaga)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Koichi%20Fukunaga) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Koichi%20Fukunaga%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Mamoru Uemura
    
    [View author publications](https://www.nature.com/search?author=Mamoru%20Uemura)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mamoru%20Uemura) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mamoru%20Uemura%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  Takayoshi Morita
    
    [View author publications](https://www.nature.com/search?author=Takayoshi%20Morita)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Takayoshi%20Morita) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Takayoshi%20Morita%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  Yasuhiro Kato
    
    [View author publications](https://www.nature.com/search?author=Yasuhiro%20Kato)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yasuhiro%20Kato) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yasuhiro%20Kato%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
32.  Haruhiko Hirata
    
    [View author publications](https://www.nature.com/search?author=Haruhiko%20Hirata)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Haruhiko%20Hirata) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Haruhiko%20Hirata%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
33.  Yoshito Takeda
    
    [View author publications](https://www.nature.com/search?author=Yoshito%20Takeda)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yoshito%20Takeda) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yoshito%20Takeda%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
34.  Yuichiro Doki
    
    [View author publications](https://www.nature.com/search?author=Yuichiro%20Doki)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuichiro%20Doki) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuichiro%20Doki%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
35.  Hidetoshi Eguchi
    
    [View author publications](https://www.nature.com/search?author=Hidetoshi%20Eguchi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hidetoshi%20Eguchi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hidetoshi%20Eguchi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
36.  Daisuke Okuzaki
    
    [View author publications](https://www.nature.com/search?author=Daisuke%20Okuzaki)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daisuke%20Okuzaki) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daisuke%20Okuzaki%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
37.  Shuhei Sakakibara
    
    [View author publications](https://www.nature.com/search?author=Shuhei%20Sakakibara)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shuhei%20Sakakibara) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shuhei%20Sakakibara%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
38.  Seishi Ogawa
    
    [View author publications](https://www.nature.com/search?author=Seishi%20Ogawa)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Seishi%20Ogawa) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Seishi%20Ogawa%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
39.  Atsushi Kumanogoh
    
    [View author publications](https://www.nature.com/search?author=Atsushi%20Kumanogoh)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Atsushi%20Kumanogoh) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Atsushi%20Kumanogoh%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
40.  Yukinori Okada
    
    [View author publications](https://www.nature.com/search?author=Yukinori%20Okada)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yukinori%20Okada) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yukinori%20Okada%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Consortia

### Japan COVID-19 Task Force

-   Qingbo S. Wang
-   , Ryuya Edahiro
-   , Ho Namkoong
-   , Takanori Hasegawa
-   , Yuya Shirai
-   , Kyuto Sonehara
-   , Hiromu Tanaka
-   , Ho Lee
-   , Ryunosuke Saiki
-   , Takayoshi Hyugaji
-   , Eigo Shimizu
-   , Kotoe Katayama
-   , Masahiro Kanai
-   , Tatsuhiko Naito
-   , Noah Sasa
-   , Kenichi Yamamoto
-   , Yasuhiro Kato
-   , Takayoshi Morita
-   , Kazuhisa Takahashi
-   , Norihiro Harada
-   , Toshio Naito
-   , Makoto Hiki
-   , Yasushi Matsushita
-   , Haruhi Takagi
-   , Masako Ichikawa
-   , Ai Nakamura
-   , Sonoko Harada
-   , Yuuki Sandhu
-   , Hiroki Kabata
-   , Katsunori Masaki
-   , Hirofumi Kamata
-   , Shinnosuke Ikemura
-   , Shotaro Chubachi
-   , Satoshi Okamori
-   , Hideki Terai
-   , Atsuho Morita
-   , Takanori Asakura
-   , Junichi Sasaki
-   , Hiroshi Morisaki
-   , Yoshifumi Uwamino
-   , Kosaku Nanki
-   , Sho Uchida
-   , Shunsuke Uno
-   , Tomoyasu Nishimura
-   , Takashi Ishiguro
-   , Taisuke Isono
-   , Shun Shibata
-   , Yuma Matsui
-   , Chiaki Hosoda
-   , Kenji Takano
-   , Takashi Nishida
-   , Yoichi Kobayashi
-   , Yotaro Takaku
-   , Noboru Takayanagi
-   , Soichiro Ueda
-   , Ai Tada
-   , Masayoshi Miyawaki
-   , Masaomi Yamamoto
-   , Eriko Yoshida
-   , Reina Hayashi
-   , Tomoki Nagasaka
-   , Sawako Arai
-   , Yutaro Kaneko
-   , Kana Sasaki
-   , Etsuko Tagaya
-   , Masatoshi Kawana
-   , Ken Arimura
-   , Kunihiko Takahashi
-   , Tatsuhiko Anzai
-   , Satoshi Ito
-   , Akifumi Endo
-   , Yuji Uchimura
-   , Yasunari Miyazaki
-   , Takayuki Honda
-   , Tomoya Tateishi
-   , Shuji Tohda
-   , Naoya Ichimura
-   , Kazunari Sonobe
-   , Chihiro Tani Sassa
-   , Jun Nakajima
-   , Yasushi Nakano
-   , Yukiko Nakajima
-   , Ryusuke Anan
-   , Ryosuke Arai
-   , Yuko Kurihara
-   , Yuko Harada
-   , Kazumi Nishio
-   , Tetsuya Ueda
-   , Masanori Azuma
-   , Ryuichi Saito
-   , Toshikatsu Sado
-   , Yoshimune Miyazaki
-   , Ryuichi Sato
-   , Yuki Haruta
-   , Tadao Nagasaki
-   , Yoshinori Yasui
-   , Yoshinori Hasegawa
-   , Yoshikazu Mutoh
-   , Tomoki Kimura
-   , Tomonori Sato
-   , Reoto Takei
-   , Satoshi Hagimoto
-   , Yoichiro Noguchi
-   , Yasuhiko Yamano
-   , Hajime Sasano
-   , Sho Ota
-   , Yasushi Nakamori
-   , Kazuhisa Yoshiya
-   , Fukuki Saito
-   , Tomoyuki Yoshihara
-   , Daiki Wada
-   , Hiromu Iwamura
-   , Syuji Kanayama
-   , Shuhei Maruyama
-   , Takashi Yoshiyama
-   , Ken Ohta
-   , Hiroyuki Kokuto
-   , Hideo Ogata
-   , Yoshiaki Tanaka
-   , Kenichi Arakawa
-   , Masafumi Shimoda
-   , Takeshi Osawa
-   , Hiroki Tateno
-   , Isano Hase
-   , Shuichi Yoshida
-   , Shoji Suzuki
-   , Miki Kawada
-   , Hirohisa Horinouchi
-   , Fumitake Saito
-   , Keiko Mitamura
-   , Masao Hagihara
-   , Junichi Ochi
-   , Tomoyuki Uchida
-   , Rie Baba
-   , Daisuke Arai
-   , Takayuki Ogura
-   , Hidenori Takahashi
-   , Shigehiro Hagiwara
-   , Genta Nagao
-   , Shunichiro Konishi
-   , Ichiro Nakachi
-   , Koji Murakami
-   , Mitsuhiro Yamada
-   , Hisatoshi Sugiura
-   , Hirohito Sano
-   , Shuichiro Matsumoto
-   , Nozomu Kimura
-   , Yoshinao Ono
-   , Hiroaki Baba
-   , Yusuke Suzuki
-   , Sohei Nakayama
-   , Keita Masuzawa
-   , Shinichi Namba
-   , Takayuki Shiroyama
-   , Yoshimi Noda
-   , Takayuki Niitsu
-   , Yuichi Adachi
-   , Takatoshi Enomoto
-   , Saori Amiya
-   , Reina Hara
-   , Yuta Yamaguchi
-   , Teruaki Murakami
-   , Tomoki Kuge
-   , Kinnosuke Matsumoto
-   , Yuji Yamamoto
-   , Makoto Yamamoto
-   , Midori Yoneda
-   , Kazunori Tomono
-   , Kazuto Kato
-   , Haruhiko Hirata
-   , Yoshito Takeda
-   , Hidefumi Koh
-   , Tadashi Manabe
-   , Yohei Funatsu
-   , Fumimaro Ito
-   , Takahiro Fukui
-   , Keisuke Shinozuka
-   , Sumiko Kohashi
-   , Masatoshi Miyazaki
-   , Tomohisa Shoko
-   , Mitsuaki Kojima
-   , Tomohiro Adachi
-   , Motonao Ishikawa
-   , Kenichiro Takahashi
-   , Takashi Inoue
-   , Toshiyuki Hirano
-   , Keigo Kobayashi
-   , Hatsuyo Takaoka
-   , Kazuyoshi Watanabe
-   , Naoki Miyazawa
-   , Yasuhiro Kimura
-   , Reiko Sado
-   , Hideyasu Sugimoto
-   , Akane Kamiya
-   , Naota Kuwahara
-   , Akiko Fujiwara
-   , Tomohiro Matsunaga
-   , Yoko Sato
-   , Takenori Okada
-   , Yoshihiro Hirai
-   , Hidetoshi Kawashima
-   , Atsuya Narita
-   , Kazuki Niwa
-   , Yoshiyuki Sekikawa
-   , Koichi Nishi
-   , Masaru Nishitsuji
-   , Mayuko Tani
-   , Junya Suzuki
-   , Hiroki Nakatsumi
-   , Takashi Ogura
-   , Hideya Kitamura
-   , Eri Hagiwara
-   , Kota Murohashi
-   , Hiroko Okabayashi
-   , Takao Mochimaru
-   , Shigenari Nukaga
-   , Ryosuke Satomi
-   , Yoshitaka Oyamada
-   , Nobuaki Mori
-   , Tomoya Baba
-   , Yasutaka Fukui
-   , Mitsuru Odate
-   , Shuko Mashimo
-   , Yasushi Makino
-   , Kazuma Yagi
-   , Mizuha Hashiguchi
-   , Junko Kagyo
-   , Tetsuya Shiomi
-   , Satoshi Fuke
-   , Hiroshi Saito
-   , Tomoya Tsuchida
-   , Shigeki Fujitani
-   , Mumon Takita
-   , Daiki Morikawa
-   , Toru Yoshida
-   , Takehiro Izumo
-   , Minoru Inomata
-   , Naoyuki Kuse
-   , Nobuyasu Awano
-   , Mari Tone
-   , Akihiro Ito
-   , Yoshihiko Nakamura
-   , Kota Hoshino
-   , Junichi Maruyama
-   , Hiroyasu Ishikura
-   , Tohru Takata
-   , Toshio Odani
-   , Masaru Amishima
-   , Takeshi Hattori
-   , Yasuo Shichinohe
-   , Takashi Kagaya
-   , Toshiyuki Kita
-   , Kazuhide Ohta
-   , Satoru Sakagami
-   , Kiyoshi Koshida
-   , Kentaro Hayashi
-   , Tetsuo Shimizu
-   , Yutaka Kozu
-   , Hisato Hiranuma
-   , Yasuhiro Gon
-   , Namiki Izumi
-   , Kaoru Nagata
-   , Ken Ueda
-   , Reiko Taki
-   , Satoko Hanada
-   , Kodai Kawamura
-   , Kazuya Ichikado
-   , Kenta Nishiyama
-   , Hiroyuki Muranaka
-   , Kazunori Nakamura
-   , Naozumi Hashimoto
-   , Keiko Wakahara
-   , Sakamoto Koji
-   , Norihito Omote
-   , Akira Ando
-   , Nobuhiro Kodama
-   , Yasunari Kaneyama
-   , Shunsuke Maeda
-   , Takashige Kuraki
-   , Takemasa Matsumoto
-   , Koutaro Yokote
-   , Taka-Aki Nakada
-   , Ryuzo Abe
-   , Taku Oshima
-   , Tadanaga Shimada
-   , Masahiro Harada
-   , Takeshi Takahashi
-   , Hiroshi Ono
-   , Toshihiro Sakurai
-   , Takayuki Shibusawa
-   , Yoshifumi Kimizuka
-   , Akihiko Kawana
-   , Tomoya Sano
-   , Chie Watanabe
-   , Ryohei Suematsu
-   , Hisako Sageshima
-   , Ayumi Yoshifuji
-   , Kazuto Ito
-   , Saeko Takahashi
-   , Kota Ishioka
-   , Morio Nakamura
-   , Makoto Masuda
-   , Aya Wakabayashi
-   , Hiroki Watanabe
-   , Suguru Ueda
-   , Masanori Nishikawa
-   , Yusuke Chihara
-   , Mayumi Takeuchi
-   , Keisuke Onoi
-   , Jun Shinozuka
-   , Atsushi Sueyoshi
-   , Yoji Nagasaki
-   , Masaki Okamoto
-   , Sayoko Ishihara
-   , Masatoshi Shimo
-   , Yoshihisa Tokunaga
-   , Yu Kusaka
-   , Takehiko Ohba
-   , Susumu Isogai
-   , Aki Ogawa
-   , Takuya Inoue
-   , Satoru Fukuyama
-   , Yoshihiro Eriguchi
-   , Akiko Yonekawa
-   , Keiko Kan-o
-   , Koichiro Matsumoto
-   , Kensuke Kanaoka
-   , Shoichi Ihara
-   , Kiyoshi Komuta
-   , Yoshiaki Inoue
-   , Shigeru Chiba
-   , Kunihiro Yamagata
-   , Yuji Hiramatsu
-   , Hirayasu Kai
-   , Koichiro Asano
-   , Tsuyoshi Oguma
-   , Yoko Ito
-   , Satoru Hashimoto
-   , Masaki Yamasaki
-   , Yu Kasamatsu
-   , Yuko Komase
-   , Naoya Hida
-   , Takahiro Tsuburai
-   , Baku Oyama
-   , Minoru Takada
-   , Hidenori Kanda
-   , Yuichiro Kitagawa
-   , Tetsuya Fukuta
-   , Takahito Miyake
-   , Shozo Yoshida
-   , Shinji Ogura
-   , Shinji Abe
-   , Yuta Kono
-   , Yuki Togashi
-   , Hiroyuki Takoi
-   , Ryota Kikuchi
-   , Shinichi Ogawa
-   , Tomouki Ogata
-   , Shoichiro Ishihara
-   , Arihiko Kanehiro
-   , Shinji Ozaki
-   , Yasuko Fuchimoto
-   , Sae Wada
-   , Nobukazu Fujimoto
-   , Kei Nishiyama
-   , Mariko Terashima
-   , Satoru Beppu
-   , Kosuke Yoshida
-   , Osamu Narumoto
-   , Hideaki Nagai
-   , Nobuharu Ooshima
-   , Mitsuru Motegi
-   , Akira Umeda
-   , Kazuya Miyagawa
-   , Hisato Shimada
-   , Mayu Endo
-   , Yoshiyuki Ohira
-   , Masafumi Watanabe
-   , Sumito Inoue
-   , Akira Igarashi
-   , Masamichi Sato
-   , Hironori Sagara
-   , Akihiko Tanaka
-   , Shin Ohta
-   , Tomoyuki Kimura
-   , Yoko Shibata
-   , Yoshinori Tanino
-   , Takefumi Nikaido
-   , Hiroyuki Minemura
-   , Yuki Sato
-   , Yuichiro Yamada
-   , Takuya Hashino
-   , Masato Shinoki
-   , Hajime Iwagoe
-   , Hiroshi Takahashi
-   , Kazuhiko Fujii
-   , Hiroto Kishi
-   , Masayuki Kanai
-   , Tomonori Imamura
-   , Tatsuya Yamashita
-   , Masakiyo Yatomi
-   , Toshitaka Maeno
-   , Shinichi Hayashi
-   , Mai Takahashi
-   , Mizuki Kuramochi
-   , Isamu Kamimaki
-   , Yoshiteru Tominaga
-   , Tomoo Ishii
-   , Mitsuyoshi Utsugi
-   , Akihiro Ono
-   , Toru Tanaka
-   , Takeru Kashiwada
-   , Kazue Fujita
-   , Yoshinobu Saito
-   , Masahiro Seike
-   , Hiroko Watanabe
-   , Hiroto Matsuse
-   , Norio Kodaka
-   , Chihiro Nakano
-   , Takeshi Oshio
-   , Takatomo Hirouchi
-   , Shohei Makino
-   , Moritoki Egi
-   , Yosuke Omae
-   , Yasuhito Nannya
-   , Takafumi Ueno
-   , Tomomi Takano
-   , Kazuhiko Katayama
-   , Masumi Ai
-   , Atsushi Kumanogoh
-   , Toshiro Sato
-   , Naoki Hasegawa
-   , Katsushi Tokunaga
-   , Makoto Ishii
-   , Ryuji Koike
-   , Yuko Kitagawa
-   , Akinori Kimura
-   , Seiya Imoto
-   , Satoru Miyano
-   , Seishi Ogawa
-   , Takanori Kanai
-   , Koichi Fukunaga
-    & Yukinori Okada

### Contributions

R.E., G.S., A.K. and Y. Okada conceived and designed the study; R.E., Y. Shirai, Y. Tomofuji, T. Kishikawa, K. Suzuki, Y.-C.L., D. Motooka, A. Takuwa, D.O. and S. Sakakibara performed the experiments; R.E., G.S., Y. Shirai, R. Saiki, T. Naito, K.S., Y. Tomofuji, K. Yamamoto, S. Namba, N.S., G.N. (Japan COVID-19 Task Force), S.S. and Y. Okada conducted data analysis; R.E., G.S., T. Naito, K.S., Y. Tomofuji, S. Sakakibara, A.K. and Y. Okada wrote the manuscript; H. Hirata and Y. Takeda took care of patients and provided clinical information; Q.S.W., Y. Takahashi, T. Hasegawa, H. Tanaka, S. Azekawa, H. Namkoong, R. Koike, A. Kimura, S. Imoto, S. Miyano, T. Kanai, K. Fukunaga, M. Uemura, T. Morita, Y. Kato, H.E. and S. Ogawa provided intellectual input throughout the study, provided comments and helped edit the manuscript. All authors read and approved the final manuscript.

### Corresponding authors

Correspondence to [Ryuya Edahiro](mailto:r.edahiro@imed3.med.osaka-u.ac.jp), [Atsushi Kumanogoh](mailto:kumanogo@imed3.med.osaka-u.ac.jp) or [Yukinori Okada](mailto:yuki-okada@m.u-tokyo.ac.jp).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Expression of cell type marker genes and reference-based annotation.](https://www.nature.com/articles/s41588-025-02266-3/figures/9)

(**a**) Dotplot showing the RNA expression of marker genes of L2 cell types. (**b**) Comparison of our manual PBMC annotation (L2 cell types) vs an automated annotation performed by scPred[32](https://www.nature.com/articles/s41588-025-02266-3#ref-CR32 "Alquicira-Hernandez, J., Sathe, A., Ji, H. P., Nguyen, Q. & Powell, J. E. ScPred: accurate supervised method for cell-type classification from single-cell RNA-seq data. Genome Biol. 20, 264 (2019).") with Azimuth[27](https://www.nature.com/articles/s41588-025-02266-3#ref-CR27 "Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587 (2021).") (L2) as a reference.

### [Extended Data Fig. 2 The effect of sample sizes and cell counts per sample on eQTL discovery in OASIS and OneK1K.](https://www.nature.com/articles/s41588-025-02266-3/figures/10)

(**a**) Scatter plot depicting the correlation between median number of cells per sample and number of detected significant eGenes in each cell type of OASIS. Both x-axis and y-axis are on log10 scale. (**b**) Co-plots of the number of eGenes between OASIS and OneK1K[6](https://www.nature.com/articles/s41588-025-02266-3#ref-CR6 "Yazar, S. et al. Single-cell eQTL mapping identifies cell type-specific genetic control of autoimmune disease. Science 376, eabf3041 (2022).") in corresponding cell types. (**c,d**) The number of eGenes by down-sampling for combinations of sample sizes and cell counts per sample in OASIS (**c**) and OneK1K (**d**) at L1 level.

### [Extended Data Fig. 3 Comparison of cis-eQTLs between OASIS and OneK1K in the corresponding 10 cell type pairs.](https://www.nature.com/articles/s41588-025-02266-3/figures/11)

(**a**) The replicated ratio of eQTLs between the two cohorts in the corresponding 10 cell type pairs. (**b**) Distribution of difference of minor allele frequency (MAF) between East Asian (EAS) and European (EUR) of cis-eQTLs according to with/without cross-population eQTL replication for each cohort in naive CD4+ T cells. (**c**) Pairwise sharing by magnitude of eQTLs between the two cohorts in the corresponding 10 cell type pairs according to four thresholds of factor. (**d**) Distribution of difference of MAF between EAS and EUR of cis-eQTLs according to with/without cross-population eQTL sharing for each cohort in naive CD4+ T cells. MAF from both ancestries was calculated from high-coverage 1000 Genome Project phase 3 in (**b**) and (**d**).

### [Extended Data Fig. 4 Genome-wide association analysis for TCR/BCR repertoire features.](https://www.nature.com/articles/s41588-025-02266-3/figures/12)

Manhattan plots for the representative results for genome-wide association analysis for TCR/BCR repertoire features, including PC2 for TRAV gene usage (**a**), PC1 for TRBV gene usage (**b**), TRBD1/2 usage (**c**), and PC1 for TRBJ gene usage (**d**) as TCR features in CD4+ T cells and PC2 for IGHV gene usage (**e**), PC1 for IGHD gene usage (**f**), PC1 for IGKV PC1 (**g**), and PC1 for IGLJ gene usage (**h**) in entire B cells and mean of somatic hypermutation (SHM) in IGK (**i**) and CV of SHM in IGL (**J**) in naive1 B cells as BCR features. The red horizontal line indicates the study-wide significance threshold (_P_ = 2.0 × 10−9 and 1.6 × 10−9 for TCR and BCR repertoire features, respectively). Uncorrected _P_ values from the GWAS analysis are shown in (**a**\-**j**).

### [Extended Data Fig. 5 Properties of dynamic eQTLs and pathways representing cell-state of each PC.](https://www.nature.com/articles/s41588-025-02266-3/figures/13)

(**a**) Number of genes with a significant or non-significant genotype-module interaction in a linear or quadratic mixed model for the two gene modules. (**b**) Distribution of the number of significant dynamic eGenes in 1,000 times permuted data for each combination of modules and analysis models. Pink dotted line represents the number of dynamic eGenes observed. (**c**) Forest plots showing odds ratios of overlap of dynamic eQTLs with functional regions in Monocytes from Roadmap project[35](https://www.nature.com/articles/s41588-025-02266-3#ref-CR35 "Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317–329 (2015).") compared to cis-eQTLs in L2 cell types of myeloid cluster for each combination of functional region and analysis model. Dots represent the odds ratios and bars represent the 95% confidence intervals. Two-sided _P_ values are uncorrected. (**d**) The top 3 enriched pathways of the genes which were correlated with each PCs ( > 0.1 absolute value). Dot color indicates the statistical significance of the enrichment (adjusted _P_ values via the Benjamini–Hochberg method), and dot size represents gene ratio annotated to each term.

### [Extended Data Fig. 6 Colocalizations of GWASs in East Asian population and cis-eQTLs in OASIS.](https://www.nature.com/articles/s41588-025-02266-3/figures/14)

(**a**) Number of colocalizing eGenes shared among different autoimmune and blood-related traits. Bar plots indicating the number of genes in each set. eGenes colocalizing in only one trait are colored by pink, and those in multiple traits are connected by lines and colored by gray. (**b**) Colocalizations of GWAS variants in East Asian population and cis-eQTLs in OASIS for 6 traits with more than 10 colocalizations. Heatmap depicting PP.H4 from coloc[75](https://www.nature.com/articles/s41588-025-02266-3#ref-CR75 "Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. PLoS Genet. 10, e1004383 (2014)."). Cell types of major cell type (L1) are colored by red. GWAS abbreviations: CD, Crohn’s disease; Hosp-COV, hospitalized COVID-19; HT, hyperthyroidism; IBD, inflammatory bowel diseases; LYM, lymphocyte count; MON, monocyte count; RA, rheumatoid arthritis; SLE, systemic lupus erythematosus; T1D, type 1 diabetes; UC, ulcerative colitis.

### [Extended Data Fig. 7 The effects of polygenic risk score on transcriptome and proteome.](https://www.nature.com/articles/s41588-025-02266-3/figures/15)

(**a**) The differential gene expression analysis with 4 quantiles of hospitalized COVID-19 polygenic risk score (PRS) in monocytes separately for COVID-19 patients and healthy subjects. Differentially expressed genes (DEGs; FDR < 0.1) are colored in light blue and labeled by gene symbols if _P_ values < 1×10−4. (**b**) The differential protein expression analysis with 4 quantiles of hospitalized COVID-19 PRS in plasma proteome separately for COVID-19 patients and healthy subjects. Differentially expressed proteins (DEPs; FDR < 0.1) are colored in light blue. (**c**) The differential gene expression analysis with 4 quantiles of hospitalized COVID-19 PRS in monocytes for all samples. DEGs between patients and healthy controls (FDR < 0.05 and absolute fold change (FC) > 2) are colored in red. (**d**) The differential protein expression analysis with 4 quantiles of hospitalized COVID-19 PRS in plasma proteome for all samples. Top 250 significant DEPs between patients and healthy controls (FDR < 4.0×10−22) are colored in red. Uncorrected _P_ values are shown in (**a**\-**d**). (**e**) The effect of PRS for 7 autoimmune and blood-related traits on transcriptomics and proteomics levels separately in patients with COVID-19 and healthy subjects. One-sided _P_ values were calculated with 1,000 permutations for number of DEGs and DEPs. GWAS abbreviations: CD, Crohn’s disease; IBD, inflammatory bowel diseases; LYM, lymphocyte count; MON, monocyte count; RA, rheumatoid arthritis; SLE, systemic lupus erythematosus; UC, ulcerative colitis.

### [Extended Data Fig. 8 Characterization of the cells carrying mCAs.](https://www.nature.com/articles/s41588-025-02266-3/figures/16)

(**a**) Fractions of the mutant cells carrying each copy number alterations (CNA) event (left) and those of the mutant cells carrying each copy-neutral loss of heterozygosity (CN-LOH) event colored by setting of sequencing (right). (**b**) Heatmaps showing the in-sample odds ratios of each subcluster (L2) containing the CNA cells in CH01 (top) and CH05 (bottom). (**c**) Differential gene expression (DEG) analysis between the somatically mutant and normal cells for 1p\_Loss monocytes (left), 15q\_Gain monocytes (middle), and 17q\_Gain B cells (right). DEGs are colored in light blue (downregulated) or pink (upregulated), and DEGs on the corresponding chromosomal regions are colored in navy or red. DEGs were significant if they satisfied FDR (adjusted _P_ values via the Benjamini-Hochberg method) < 0.05 and log2 fold change > 0.25 (in CH01) or 0.5 (in CH05). (**d**\-**g**) Top ten enriched biological pathways of the downregulated DEGs in 1p\_Loss monocytes (**d**), the downregulated DEGs in 15q\_Gain monocytes (**e**), the downregulated DEGs in 17q\_Gain B cells (**f**), and the upregulated DEGs in 17q\_Gain B cells (**g**). Dot color indicates the statistical significance of the enrichment (adjusted _P_ values via the Benjamini-Hochberg method), and dot size represents gene count annotated to each term.

### [Extended Data Fig. 9 BCR clonotype of the somatically mutant cells in CH05.](https://www.nature.com/articles/s41588-025-02266-3/figures/17)

(**a**) Network plots showing the similarity of CDR3 amino acid sequence in BCR heavy and light chain from all the samples (_n_ = 235), colored by sample (left), cell type (middle), and isotype (right). The somatically mutated CH05 clonotype was surrounded by a plink dotted line. Clonotype clusters with clonal size ≥ 10 are selected. (**b**) Reactivity of antibodies against SARS-CoV-2 antigens in ELISA. The dots denote the mean and error bars do standard deviation measured in triplicate. S309, anti-SARS-CoV-2 S IgG1; CH05, recombinant antibody derived from the CH05 BCR clone; nCoV396, anti-SARS-CoV-2 N IgG1; 23B12, anti-_Candida albicans_ IgG1.

### [Extended Data Fig. 10 Characterization of the LOY cells.](https://www.nature.com/articles/s41588-025-02266-3/figures/18)

(**a**) Differential expression gene (DEG) analysis between the loss of the Y chromosome (LOY) and normal cMonoIL1B. DEGs were significant if they satisfied FDR (adjusted _P_ values via the Benjamini-Hochberg method) < 0.01 and beta > 0.25 and colored in light blue (downregulated) or pink (upregulated). (**b**) Beeswarm plot and box plot describing the distribution of adjusted log2 fold change between the LOY high and LOY low healthy controls in neighborhoods from 28 cell types (L2). Nodes are neighborhoods, colored by their log2 fold change adjusted by age and sequencing group. (**c**) Box plots showing the proportions of each cell type in the LOY high and LOY low COVID-19 patients. Wilcoxon two-sided uncorrected _P_ values are shown if they satisfied < 0.05. The boxes denote the interquartile range (IQR), the median is shown as the vertical bars, and the whiskers extend to 1.5 times the IQR in (**b**,**c**).

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_MOESM1_ESM.pdf)

Supplementary Notes, Methods and Figs. 1–10.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_MOESM2_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02266-3/MediaObjects/41588_2025_2266_MOESM3_ESM.xlsx)

All supplementary tables are summarized in a Microsoft Excel file.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Deciphering%20state-dependent%20immune%20features%20from%20multi-layer%20omics%20data%20at%20single-cell%20resolution&author=Ryuya%20Edahiro%20Qingbo%20S.%20Wang%20et%20al&contentID=10.1038%2Fs41588-025-02266-3&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-28&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY-NC-ND)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02266-3)

### Cite this article

Edahiro, R., Sato, G., Naito, T. _et al._ Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02266-3

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02266-3?format=refman&flavour=citation)

-   Received: 18 May 2024
    
-   Accepted: 16 June 2025
    
-   Published: 28 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02266-3
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02266-3.pdf)
