> id: https://www.nature.com/articles/s41588-025-02288-x

> link: https://www.nature.com/articles/s41588-025-02288-x

> title: Noncoding rare variant associations with blood traits in 166740 UK Biobank genomes

# Noncoding rare variant associations with blood traits in 166740 UK Biobank genomes
_Published on Wed, 06 Aug 2025 00:00:00 GMT_

Abstract
--------

Large biobanks with whole-genome sequencing (WGS) now enable the association of noncoding rare variants with complex human traits. Given that >98% of the genome is available for exploration, the selection of noncoding variants remains a critical yet unresolved challenge in these analyses. Here we leverage knowledge of blood gene regulation and deleteriousness scores to select noncoding variants pertinent for association with blood-related traits. Integrating WGS and 42 blood cell count and biomarker measurements for 166,740 UK Biobank samples, we perform variant collapsing tests, identifying hundreds of gene–trait associations involving noncoding variants. However, we demonstrate that most of these noncoding rare variant associations (1) reproduce associations known from previous studies and (2) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting noncoding rare variant association results.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 12 print issues and online access

$209.00 per year

only $17.42 per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02288-x?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Schematic representation of the study approach.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_Fig1_HTML.png)

**Fig. 2: Manhattan plot for gene-based rare variant associations across 42 traits and multiple annotations.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_Fig2_HTML.png)

**Fig. 3: Known gene–trait associations.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_Fig3_HTML.png)

**Fig. 4: Rare variant association type and conditioning analysis.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_Fig4_HTML.png)

**Fig. 5: Replication of discovered gene–trait associations.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_Fig5_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-021-03855-y/MediaObjects/41586_2021_3855_Fig1_HTML.png)

### [Rare variant contribution to human disease in 281,104 UK Biobank exomes](https://www.nature.com/articles/s41586-021-03855-y?fromPaywallRec=true)

Article Open access 10 August 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01868-7/MediaObjects/41588_2024_1868_Fig1_HTML.png)

### [A Genomics England haplotype reference panel and imputation of UK Biobank](https://www.nature.com/articles/s41588-024-01868-7?fromPaywallRec=true)

Article Open access 12 August 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-022-04965-x/MediaObjects/41586_2022_4965_Fig1_HTML.png)

### [The sequences of 150,119 genomes in the UK Biobank](https://www.nature.com/articles/s41586-022-04965-x?fromPaywallRec=true)

Article Open access 20 July 2022

Data availability
-----------------

The lists of gene–trait associations identified here are available as Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02288-x#MOESM4)–[12](https://www.nature.com/articles/s41588-025-02288-x#MOESM4). These lists and other source data are available via Zenodo at [https://doi.org/10.5281/zenodo.15546894](https://doi.org/10.5281/zenodo.15546894) (ref. [68](https://www.nature.com/articles/s41588-025-02288-x#ref-CR68 "Ribeiro, D. diogomribeiro/noncoding_rarevariant: v2. Zenodo 
https://doi.org/10.5281/zenodo.15546894
(2025).")). The WGS data and individual phenotypes used for association testing can be accessed via the UKBB Research Analysis Platform: [https://ukbiobank.dnanexus.com/landing](https://ukbiobank.dnanexus.com/landing). This platform is open to researchers who are listed as collaborators on UKBB-approved access applications. Public regulatory annotation datasets used include: the ABC models, available at [https://www.engreitzlab.org/resources](https://www.engreitzlab.org/resources); promoter capture Hi-C data, obtained from ref. [28](https://www.nature.com/articles/s41588-025-02288-x#ref-CR28 "Javierre, B. M. et al. Lineage-specific genome architecture links enhancers and non-coding disease variants to target gene promoters. Cell 167, 1369–1384 (2016)."), accessible at [https://osf.io/u8tzp](https://osf.io/u8tzp/); and CRDs obtained from the original publications[29](https://www.nature.com/articles/s41588-025-02288-x#ref-CR29 "Delaneau, O. et al. Chromatin three-dimensional interactions mediate genetic effects on gene expression. Science 364, eaat8266 (2019)."),[30](https://www.nature.com/articles/s41588-025-02288-x#ref-CR30 "Avalos, D. et al. Genetic variation in cis-regulatory domains suggests cell type-specific regulatory mechanisms in immunity. Commun. Biol. 6, 335 (2023)."). CADD scores of deleteriousness are available at [https://cadd.gs.washington.edu](https://cadd.gs.washington.edu/). Known gene–trait associations used are available in the GWAS catalog ([https://www.ebi.ac.uk/gwas](https://www.ebi.ac.uk/gwas)) and Genebass databases ([https://app.genebass.org](https://app.genebass.org/)). [Source data](https://www.nature.com/articles/s41588-025-02288-x#Sec26) are provided with this paper.

Code availability
-----------------

Code to reproduce analysis and plots are avilable via GitHub at [https://github.com/diogomribeiro/noncoding\_rarevariant](https://github.com/diogomribeiro/noncoding_rarevariant) and via Zenodo at [https://doi.org/10.5281/zenodo.15546894](https://doi.org/10.5281/zenodo.15546894).

References
----------

1.  Abdellaoui, A., Yengo, L., Verweij, K. J. H. & Visscher, P. M. 15 years of GWAS discovery: realizing the promise. _Am. J. Hum. Genet._ **110**, 179–194 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXovVKiuw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36634672)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9943775)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=15%20years%20of%20GWAS%20discovery%3A%20realizing%20the%20promise&journal=Am.%20J.%20Hum.%20Genet.&volume=110&pages=179-194&publication_year=2023&author=Abdellaoui%2CA&author=Yengo%2CL&author=Verweij%2CKJH&author=Visscher%2CPM) 
    
2.  Rubinacci, S., Delaneau, O. & Marchini, J. Genotype imputation using the positional Burrows Wheeler transform. _PLoS Genet._ **16**, e1009049 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVyqtLbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33196638)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7704051)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genotype%20imputation%20using%20the%20positional%20Burrows%20Wheeler%20transform&journal=PLoS%20Genet.&volume=16&publication_year=2020&author=Rubinacci%2CS&author=Delaneau%2CO&author=Marchini%2CJ) 
    
3.  Li, Y., Willer, C., Sanna, S. & Abecasis, G. Genotype imputation. _Annu. Rev. Genom. Hum. Genet._ **10**, 387–406 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht12qsrjP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genotype%20imputation&journal=Annu.%20Rev.%20Genom.%20Hum.%20Genet.&volume=10&pages=387-406&publication_year=2009&author=Li%2CY&author=Willer%2CC&author=Sanna%2CS&author=Abecasis%2CG) 
    
4.  Wu, M. C. et al. Rare-variant association testing for sequencing data with the sequence kernel association test. _Am. J. Hum. Genet._ **89**, 82–93 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXovF2mtLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21737059)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3135811)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rare-variant%20association%20testing%20for%20sequencing%20data%20with%20the%20sequence%20kernel%20association%20test&journal=Am.%20J.%20Hum.%20Genet.&volume=89&pages=82-93&publication_year=2011&author=Wu%2CMC) 
    
5.  Bocher, O. & Génin, E. Rare variant association testing in the non-coding genome. _Hum. Genet._ **139**, 1345–1362 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32500240)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rare%20variant%20association%20testing%20in%20the%20non-coding%20genome&journal=Hum.%20Genet.&volume=139&pages=1345-1362&publication_year=2020&author=Bocher%2CO&author=G%C3%A9nin%2CE) 
    
6.  Wainschtein, P. et al. Assessing the contribution of rare variants to complex trait heritability from whole-genome sequence data. _Nat. Genet._ **54**, 263–273 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XmtlCitb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35256806)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9119698)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Assessing%20the%20contribution%20of%20rare%20variants%20to%20complex%20trait%20heritability%20from%20whole-genome%20sequence%20data&journal=Nat.%20Genet.&volume=54&pages=263-273&publication_year=2022&author=Wainschtein%2CP) 
    
7.  Zuk, O. et al. Searching for missing heritability: designing rare variant association studies. _Proc. Natl Acad. Sci. USA_ **111**, E455–E464 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhs1Sntbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24443550)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3910587)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Searching%20for%20missing%20heritability%3A%20designing%20rare%20variant%20association%20studies&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=111&pages=E455-E464&publication_year=2014&author=Zuk%2CO) 
    
8.  Wang, Q. et al. Rare variant contribution to human disease in 281,104 UK Biobank exomes. _Nature_ **597**, 527–532 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFyntr7J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34375979)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8458098)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rare%20variant%20contribution%20to%20human%20disease%20in%20281%2C104%20UK%20Biobank%20exomes&journal=Nature&volume=597&pages=527-532&publication_year=2021&author=Wang%2CQ) 
    
9.  Momozawa, Y. & Mizukami, K. Unique roles of rare variants in the genetics of complex diseases in humans. _J. Hum. Genet._ **66**, 11–23 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32948841)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unique%20roles%20of%20rare%20variants%20in%20the%20genetics%20of%20complex%20diseases%20in%20humans&journal=J.%20Hum.%20Genet.&volume=66&pages=11-23&publication_year=2021&author=Momozawa%2CY&author=Mizukami%2CK) 
    
10.  Backman, J. D. et al. Exome sequencing and analysis of 454,787 UK Biobank participants. _Nature_ **599**, 628–634 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFWks73I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34662886)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8596853)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Exome%20sequencing%20and%20analysis%20of%20454%2C787%20UK%20Biobank%20participants&journal=Nature&volume=599&pages=628-634&publication_year=2021&author=Backman%2CJD) 
    
11.  Hall, S. S. Genetics: a gene of rare effect. _Nature_ **496**, 152–155 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXlslyrtbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23579660)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetics%3A%20a%20gene%20of%20rare%20effect&journal=Nature&volume=496&pages=152-155&publication_year=2013&author=Hall%2CSS) 
    
12.  Sabatine, M. S. et al. Evolocumab and clinical outcomes in patients with cardiovascular disease. _N. Engl. J. Med._ **376**, 1713–1722 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlOns7nM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28304224)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolocumab%20and%20clinical%20outcomes%20in%20patients%20with%20cardiovascular%20disease&journal=N.%20Engl.%20J.%20Med.&volume=376&pages=1713-1722&publication_year=2017&author=Sabatine%2CMS) 
    
13.  Karczewski, K. J. et al. Systematic single-variant and gene-based association testing of thousands of phenotypes in 394,841 UK Biobank exomes. _Cell Genom._ **2**, 100168 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFGhsLbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36778668)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9903662)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20single-variant%20and%20gene-based%20association%20testing%20of%20thousands%20of%20phenotypes%20in%20394%2C841%20UK%20Biobank%20exomes&journal=Cell%20Genom.&volume=2&publication_year=2022&author=Karczewski%2CKJ) 
    
14.  Hindorff, L. A. et al. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. _Proc. Natl Acad. Sci. USA_ **106**, 9362–9367 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cgur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19474294)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687147)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Potential%20etiologic%20and%20functional%20implications%20of%20genome-wide%20association%20loci%20for%20human%20diseases%20and%20traits&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=106&pages=9362-9367&publication_year=2009&author=Hindorff%2CLA) 
    
15.  Maurano, M. T. et al. Systematic localization of common disease-associated variation in regulatory DNA. _Science_ **337**, 1190–1195 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1ylsLfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22955828)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3771521)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20localization%20of%20common%20disease-associated%20variation%20in%20regulatory%20DNA&journal=Science&volume=337&pages=1190-1195&publication_year=2012&author=Maurano%2CMT) 
    
16.  Hawkes, G. et al. Whole genome sequencing analysis identifies rare, large-effect noncoding variants and regulatory regions associated with circulating protein levels. _Nat. Genet._ **57**, 626–634 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXmt1Oju7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39994471)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11906349)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole%20genome%20sequencing%20analysis%20identifies%20rare%2C%20large-effect%20noncoding%20variants%20and%20regulatory%20regions%20associated%20with%20circulating%20protein%20levels&journal=Nat.%20Genet.&volume=57&pages=626-634&publication_year=2025&author=Hawkes%2CG) 
    
17.  Hawkes, G. et al. Whole-genome sequencing in 333,100 individuals reveals rare non-coding single variant and aggregate associations with height. _Nat. Commun._ **15**, 8549 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitFGmsLfP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39362880)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11450065)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20sequencing%20in%20333%2C100%20individuals%20reveals%20rare%20non-coding%20single%20variant%20and%20aggregate%20associations%20with%20height&journal=Nat.%20Commun.&volume=15&publication_year=2024&author=Hawkes%2CG) 
    
18.  Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. _Nature_ **562**, 203–209 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2qsbvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30305743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&journal=Nature&volume=562&pages=203-209&publication_year=2018&author=Bycroft%2CC) 
    
19.  Halldorsson, B. V. et al. The sequences of 150,119 genomes in the UK Biobank. _Nature_ **607**, 732–740 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVOiu7rK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35859178)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9329122)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20sequences%20of%20150%2C119%20genomes%20in%20the%20UK%20Biobank&journal=Nature&volume=607&pages=732-740&publication_year=2022&author=Halldorsson%2CBV) 
    
20.  Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. _Nature_ **590**, 290–299 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjslKnsrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33568819)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875770)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sequencing%20of%2053%2C831%20diverse%20genomes%20from%20the%20NHLBI%20TOPMed%20Program&journal=Nature&volume=590&pages=290-299&publication_year=2021&author=Taliun%2CD) 
    
21.  All of Us Research Program Investigators et al. The ‘All of Us’ research program. _N. Engl. J. Med._ **381**, 668–676 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20%E2%80%98All%20of%20Us%E2%80%99%20research%20program&journal=N.%20Engl.%20J.%20Med.&volume=381&pages=668-676&publication_year=2019) 
    
22.  Andersson, R. & Sandelin, A. Determinants of enhancer and promoter activities of regulatory elements. _Nat. Rev. Gene._ **21**, 71–87 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFGmtr7L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Determinants%20of%20enhancer%20and%20promoter%20activities%20of%20regulatory%20elements&journal=Nat.%20Rev.%20Gene.&volume=21&pages=71-87&publication_year=2020&author=Andersson%2CR&author=Sandelin%2CA) 
    
23.  Claringbould, A. & Zaugg, J. B. Enhancers in disease: molecular basis and emerging treatment strategies. _Trends Mol. Med._ **27**, 1060–1073 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVOlsrnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34420874)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhancers%20in%20disease%3A%20molecular%20basis%20and%20emerging%20treatment%20strategies&journal=Trends%20Mol.%20Med.&volume=27&pages=1060-1073&publication_year=2021&author=Claringbould%2CA&author=Zaugg%2CJB) 
    
24.  Ribeiro, D. M. et al. The molecular basis, genetic control and pleiotropic effects of local gene co-expression. _Nat. Commun._ **12**, 4842 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVSlsLbL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34376650)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8355184)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20basis%2C%20genetic%20control%20and%20pleiotropic%20effects%20of%20local%20gene%20co-expression&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Ribeiro%2CDM) 
    
25.  Hoellinger, T. et al. Enhancer/gene relationships: need for more reliable genome-wide reference sets. _Front. Bioinform._ **3**, 1092853 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36909938)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9999192)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhancer%2Fgene%20relationships%3A%20need%20for%20more%20reliable%20genome-wide%20reference%20sets&journal=Front.%20Bioinform.&volume=3&publication_year=2023&author=Hoellinger%2CT) 
    
26.  Sonawane, A. R. et al. Understanding tissue-specific gene regulation. _Cell Rep._ **21**, 1077–1088 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslansbrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29069589)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5828531)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%20tissue-specific%20gene%20regulation&journal=Cell%20Rep.&volume=21&pages=1077-1088&publication_year=2017&author=Sonawane%2CAR) 
    
27.  Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. _Science_ **326**, 289–293 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1agsbjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19815776)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2858594)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&journal=Science&volume=326&pages=289-293&publication_year=2009&author=Lieberman-Aiden%2CE) 
    
28.  Javierre, B. M. et al. Lineage-specific genome architecture links enhancers and non-coding disease variants to target gene promoters. _Cell_ **167**, 1369–1384 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvFWgu7rM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27863249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5123897)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lineage-specific%20genome%20architecture%20links%20enhancers%20and%20non-coding%20disease%20variants%20to%20target%20gene%20promoters&journal=Cell&volume=167&pages=1369-1384&publication_year=2016&author=Javierre%2CBM) 
    
29.  Delaneau, O. et al. Chromatin three-dimensional interactions mediate genetic effects on gene expression. _Science_ **364**, eaat8266 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslyitL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31048460)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20three-dimensional%20interactions%20mediate%20genetic%20effects%20on%20gene%20expression&journal=Science&volume=364&publication_year=2019&author=Delaneau%2CO) 
    
30.  Avalos, D. et al. Genetic variation in _cis_\-regulatory domains suggests cell type-specific regulatory mechanisms in immunity. _Commun. Biol._ **6**, 335 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmsVeht7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36977773)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10050075)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20variation%20in%20cis-regulatory%20domains%20suggests%20cell%20type-specific%20regulatory%20mechanisms%20in%20immunity&journal=Commun.%20Biol.&volume=6&publication_year=2023&author=Avalos%2CD) 
    
31.  Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. _Nature_ **518**, 317–330 (2015).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4530010)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&journal=Nature&volume=518&pages=317-330&publication_year=2015) 
    
32.  Boix, C. A., James, B. T., Park, Y. P., Meuleman, W. & Kellis, M. Regulatory genomic circuitry of human disease loci by integrative epigenomics. _Nature_ **590**, 300–307 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjtF2qs74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33536621)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875769)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulatory%20genomic%20circuitry%20of%20human%20disease%20loci%20by%20integrative%20epigenomics&journal=Nature&volume=590&pages=300-307&publication_year=2021&author=Boix%2CCA&author=James%2CBT&author=Park%2CYP&author=Meuleman%2CW&author=Kellis%2CM) 
    
33.  Fulco, C. P. et al. Activity-by-contact model of enhancer-promoter regulation from thousands of CRISPR perturbations. _Nat. Genet._ **51**, 1664–1669 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1Ohs77M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31784727)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6886585)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Activity-by-contact%20model%20of%20enhancer-promoter%20regulation%20from%20thousands%20of%20CRISPR%20perturbations&journal=Nat.%20Genet.&volume=51&pages=1664-1669&publication_year=2019&author=Fulco%2CCP) 
    
34.  Nasser, J. et al. Genome-wide enhancer maps link risk variants to disease genes. _Nature_ **593**, 238–243 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXosVCmurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33828297)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9153265)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20enhancer%20maps%20link%20risk%20variants%20to%20disease%20genes&journal=Nature&volume=593&pages=238-243&publication_year=2021&author=Nasser%2CJ) 
    
35.  Gazal, S. et al. Combining SNP-to-gene linking strategies to identify disease genes and assess disease omnigenicity. _Nat. Genet._ **54**, 827–836 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFSgt77J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35668300)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9894581)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combining%20SNP-to-gene%20linking%20strategies%20to%20identify%20disease%20genes%20and%20assess%20disease%20omnigenicity&journal=Nat.%20Genet.&volume=54&pages=827-836&publication_year=2022&author=Gazal%2CS) 
    
36.  Dey, K. K. et al. SNP-to-gene linking strategies reveal contributions of enhancer-related and candidate master-regulator genes to autoimmune disease. _Cell Genom._ **2**, 100145 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVOmurfF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35873673)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9306342)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SNP-to-gene%20linking%20strategies%20reveal%20contributions%20of%20enhancer-related%20and%20candidate%20master-regulator%20genes%20to%20autoimmune%20disease&journal=Cell%20Genom.&volume=2&publication_year=2022&author=Dey%2CKK) 
    
37.  Bocher, O. et al. Testing for association with rare variants in the coding and non-coding genome: RAVA-FIRST, a new approach based on CADD deleteriousness score. _PLoS Genet._ **18**, e1009923 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFWgtL%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36112662)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9518893)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Testing%20for%20association%20with%20rare%20variants%20in%20the%20coding%20and%20non-coding%20genome%3A%20RAVA-FIRST%2C%20a%20new%20approach%20based%20on%20CADD%20deleteriousness%20score&journal=PLoS%20Genet.&volume=18&publication_year=2022&author=Bocher%2CO) 
    
38.  Rentzsch, P., Witten, D., Cooper, G. M., Shendure, J. & Kircher, M. CADD: predicting the deleteriousness of variants throughout the human genome. _Nucleic Acids Res._ **47**, D886–D894 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1CgtL%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30371827)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CADD%3A%20predicting%20the%20deleteriousness%20of%20variants%20throughout%20the%20human%20genome&journal=Nucleic%20Acids%20Res.&volume=47&pages=D886-D894&publication_year=2019&author=Rentzsch%2CP&author=Witten%2CD&author=Cooper%2CGM&author=Shendure%2CJ&author=Kircher%2CM) 
    
39.  Mbatchou, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. _Nat. Genet._ **53**, 1097–1103 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFehsbrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34017140)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computationally%20efficient%20whole-genome%20regression%20for%20quantitative%20and%20binary%20traits&journal=Nat.%20Genet.&volume=53&pages=1097-1103&publication_year=2021&author=Mbatchou%2CJ) 
    
40.  Ribeiro, D. M., Ziyani, C. & Delaneau, O. Shared regulation and functional relevance of local gene co-expression revealed by single cell analysis. _Commun. Biol._ **5**, 876 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Shared%20regulation%20and%20functional%20relevance%20of%20local%20gene%20co-expression%20revealed%20by%20single%20cell%20analysis&journal=Commun.%20Biol.&volume=5&publication_year=2022&author=Ribeiro%2CDM&author=Ziyani%2CC&author=Delaneau%2CO) 
    
41.  Hambleton, S. et al. _IRF8_ mutations and human dendritic-cell immunodeficiency. _N. Engl. J. Med._ **365**, 127–138 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXptFyju7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21524210)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3136554)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IRF8%20mutations%20and%20human%20dendritic-cell%20immunodeficiency&journal=N.%20Engl.%20J.%20Med.&volume=365&pages=127-138&publication_year=2011&author=Hambleton%2CS) 
    
42.  Tamura, T., Kurotaki, D. & Koizumi, S.-I. Regulation of myelopoiesis by the transcription factor IRF8. _Int. J. Hematol._ **101**, 342–351 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXktVKis7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25749660)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulation%20of%20myelopoiesis%20by%20the%20transcription%20factor%20IRF8&journal=Int.%20J.%20Hematol.&volume=101&pages=342-351&publication_year=2015&author=Tamura%2CT&author=Kurotaki%2CD&author=Koizumi%2CS-I) 
    
43.  Karczewski, K. J. et al. Pan-UK Biobank GWAS improves discovery, analysis of genetic architecture, and resolution into ancestry-enriched effects. Preprint at _bioRxiv_ [https://doi.org/10.1101/2024.03.13.24303864](https://doi.org/10.1101/2024.03.13.24303864) (2024).
    
44.  Yang, J. et al. Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. _Nat. Genet._ **44**, 369–375 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XktVaksr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22426310)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3593158)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conditional%20and%20joint%20multiple-SNP%20analysis%20of%20GWAS%20summary%20statistics%20identifies%20additional%20variants%20influencing%20complex%20traits&journal=Nat.%20Genet.&volume=44&pages=369-375&publication_year=2012&author=Yang%2CJ) 
    
45.  Vuckovic, D. et al. The polygenic and monogenic basis of blood traits and diseases. _Cell_ **182**, 1214–1231 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsl2itbbP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32888494)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7482360)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20polygenic%20and%20monogenic%20basis%20of%20blood%20traits%20and%20diseases&journal=Cell&volume=182&pages=1214-1231&publication_year=2020&author=Vuckovic%2CD) 
    
46.  Mostafavi, H., Spence, J. P., Naqvi, S. & Pritchard, J. K. Systematic differences in discovery of genetic effects on gene expression and complex traits. _Nat. Genet._ **55**, 1866–1875 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFOqsL%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37857933)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12270542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20differences%20in%20discovery%20of%20genetic%20effects%20on%20gene%20expression%20and%20complex%20traits&journal=Nat.%20Genet.&volume=55&pages=1866-1875&publication_year=2023&author=Mostafavi%2CH&author=Spence%2CJP&author=Naqvi%2CS&author=Pritchard%2CJK) 
    
47.  Connally, N. J. et al. The missing link between genetic association and regulatory function. _eLife_ **11**, e74970 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisVKkt7zL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36515579)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9842386)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20missing%20link%20between%20genetic%20association%20and%20regulatory%20function&journal=eLife&volume=11&publication_year=2022&author=Connally%2CNJ) 
    
48.  Li, Z. et al. Dynamic scan procedure for detecting rare-variant association regions in whole-genome sequencing studies. _Am. J. Hum. Genet._ **104**, 802–814 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXnt1Clt7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30982610)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6507043)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamic%20scan%20procedure%20for%20detecting%20rare-variant%20association%20regions%20in%20whole-genome%20sequencing%20studies&journal=Am.%20J.%20Hum.%20Genet.&volume=104&pages=802-814&publication_year=2019&author=Li%2CZ) 
    
49.  Yang, Y. et al. eSCAN: scan regulatory regions for aggregate association testing using whole-genome sequencing data. _Brief. Bioinform._ **23**, bbab497 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34882196)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eSCAN%3A%20scan%20regulatory%20regions%20for%20aggregate%20association%20testing%20using%20whole-genome%20sequencing%20data&journal=Brief.%20Bioinform.&volume=23&publication_year=2022&author=Yang%2CY) 
    
50.  Li, Z. et al. A framework for detecting noncoding rare-variant associations of large-scale whole-genome sequencing studies. _Nat. Methods_ **19**, 1599–1611 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XislCjtL3K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36303018)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10008172)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20framework%20for%20detecting%20noncoding%20rare-variant%20associations%20of%20large-scale%20whole-genome%20sequencing%20studies&journal=Nat.%20Methods&volume=19&pages=1599-1611&publication_year=2022&author=Li%2CZ) 
    
51.  Li, X. et al. Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies. _Nat. Genet._ **55**, 154–164 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFymurjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36564505)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Powerful%2C%20scalable%20and%20resource-efficient%20meta-analysis%20of%20rare%20variant%20associations%20in%20large%20whole%20genome%20sequencing%20studies&journal=Nat.%20Genet.&volume=55&pages=154-164&publication_year=2023&author=Li%2CX) 
    
52.  Hu, Y. et al. Whole-genome sequencing association analysis of quantitative red blood cell phenotypes: the NHLBI TOPMed program. _Am. J. Hum. Genet._ **108**, 874–893 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsFGku7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33887194)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8206199)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20sequencing%20association%20analysis%20of%20quantitative%20red%20blood%20cell%20phenotypes%3A%20the%20NHLBI%20TOPMed%20program&journal=Am.%20J.%20Hum.%20Genet.&volume=108&pages=874-893&publication_year=2021&author=Hu%2CY) 
    
53.  Gaynor, S. M. et al. Yield of genetic association signals from genomes, exomes, and imputation in the UK Biobank. _Nat. Genet._ **56**, 2345–2351 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitVKhtrnE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39322778)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11549045)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yield%20of%20genetic%20association%20signals%20from%20genomes%2C%20exomes%2C%20and%20imputation%20in%20the%20UK%20Biobank&journal=Nat.%20Genet.&volume=56&pages=2345-2351&publication_year=2024&author=Gaynor%2CSM) 
    
54.  Yengo, L. et al. A saturated map of common genetic variants associated with human height. _Nature_ **610**, 704–712 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1WlsbnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36224396)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9605867)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20saturated%20map%20of%20common%20genetic%20variants%20associated%20with%20human%20height&journal=Nature&volume=610&pages=704-712&publication_year=2022&author=Yengo%2CL) 
    
55.  Graham, S. E. et al. The power of genetic diversity in genome-wide association studies of lipids. _Nature_ **600**, 675–679 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWkt73M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34887591)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8730582)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20power%20of%20genetic%20diversity%20in%20genome-wide%20association%20studies%20of%20lipids&journal=Nature&volume=600&pages=675-679&publication_year=2021&author=Graham%2CSE) 
    
56.  Dickson, S. P., Wang, K., Krantz, I., Hakonarson, H. & Goldstein, D. B. Rare variants create synthetic genome-wide associations. _PLoS Biol._ **8**, e1000294 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20126254)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2811148)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rare%20variants%20create%20synthetic%20genome-wide%20associations&journal=PLoS%20Biol.&volume=8&publication_year=2010&author=Dickson%2CSP&author=Wang%2CK&author=Krantz%2CI&author=Hakonarson%2CH&author=Goldstein%2CDB) 
    
57.  Anderson, C. A., Soranzo, N., Zeggini, E. & Barrett, J. C. Synthetic associations are unlikely to account for many common disease genome-wide association signals. _PLoS Biol._ **9**, e1000580 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Oiu7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21267062)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3022527)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthetic%20associations%20are%20unlikely%20to%20account%20for%20many%20common%20disease%20genome-wide%20association%20signals&journal=PLoS%20Biol.&volume=9&publication_year=2011&author=Anderson%2CCA&author=Soranzo%2CN&author=Zeggini%2CE&author=Barrett%2CJC) 
    
58.  Wray, N. R., Purcell, S. M. & Visscher, P. M. Synthetic associations created by rare variants do not explain most GWAS results. _PLoS Biol._ **9**, e1000579 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Oiu7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21267061)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3022526)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthetic%20associations%20created%20by%20rare%20variants%20do%20not%20explain%20most%20GWAS%20results&journal=PLoS%20Biol.&volume=9&publication_year=2011&author=Wray%2CNR&author=Purcell%2CSM&author=Visscher%2CPM) 
    
59.  Hofmeister, R. J., Ribeiro, D. M., Rubinacci, S. & Delaneau, O. Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank. _Nat. Genet._ **55**, 1243–1249 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlKjsrfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37386248)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10335929)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20rare%20variant%20phasing%20of%20whole-genome%20and%20whole-exome%20sequencing%20data%20in%20the%20UK%20Biobank&journal=Nat.%20Genet.&volume=55&pages=1243-1249&publication_year=2023&author=Hofmeister%2CRJ&author=Ribeiro%2CDM&author=Rubinacci%2CS&author=Delaneau%2CO) 
    
60.  Dutta, D. et al. Meta-MultiSKAT: multiple phenotype meta-analysis for region-based association test. _Genet. Epidemiol._ **43**, 800–814 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31433078)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7006736)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Meta-MultiSKAT%3A%20multiple%20phenotype%20meta-analysis%20for%20region-based%20association%20test&journal=Genet.%20Epidemiol.&volume=43&pages=800-814&publication_year=2019&author=Dutta%2CD) 
    
61.  Smedley, D. et al. A whole-genome analysis framework for effective identification of pathogenic regulatory variants in Mendelian disease. _Am. J. Hum. Genet._ **99**, 595–606 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVWmtLrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27569544)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5011059)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20whole-genome%20analysis%20framework%20for%20effective%20identification%20of%20pathogenic%20regulatory%20variants%20in%20Mendelian%20disease&journal=Am.%20J.%20Hum.%20Genet.&volume=99&pages=595-606&publication_year=2016&author=Smedley%2CD) 
    
62.  Davydov, E. V. et al. Identifying a high fraction of the human genome to be under selective constraint using GERP++. _PLoS Comput. Biol._ **6**, e1001025 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21152010)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996323)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identifying%20a%20high%20fraction%20of%20the%20human%20genome%20to%20be%20under%20selective%20constraint%20using%20GERP%2B%2B&journal=PLoS%20Comput.%20Biol.&volume=6&publication_year=2010&author=Davydov%2CEV) 
    
63.  Ionita-Laza, I., McCallum, K., Xu, B. & Buxbaum, J. D. A spectral approach integrating functional genomic annotations for coding and noncoding variants. _Nat. Genet._ **48**, 214–220 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvFKnsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26727659)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4731313)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20spectral%20approach%20integrating%20functional%20genomic%20annotations%20for%20coding%20and%20noncoding%20variants&journal=Nat.%20Genet.&volume=48&pages=214-220&publication_year=2016&author=Ionita-Laza%2CI&author=McCallum%2CK&author=Xu%2CB&author=Buxbaum%2CJD) 
    
64.  Cheng, J. et al. Accurate proteome-wide missense variant effect prediction with AlphaMissense. _Science_ **381**, eadg7492 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvFyjtbfE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37733863)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20proteome-wide%20missense%20variant%20effect%20prediction%20with%20AlphaMissense&journal=Science&volume=381&publication_year=2023&author=Cheng%2CJ) 
    
65.  Lee, B. T. et al. The UCSC Genome Browser database: 2022 update. _Nucleic Acids Res._ **50**, D1115–D1122 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1Gqsr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34718705)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20UCSC%20Genome%20Browser%20database%3A%202022%20update&journal=Nucleic%20Acids%20Res.&volume=50&pages=D1115-D1122&publication_year=2022&author=Lee%2CBT) 
    
66.  Raudvere, U. et al. g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). _Nucleic Acids Res._ **47**, W191–W198 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktVyit70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31066453)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6602461)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=g%3AProfiler%3A%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20conversions%20of%20gene%20lists%20%282019%20update%29&journal=Nucleic%20Acids%20Res.&volume=47&pages=W191-W198&publication_year=2019&author=Raudvere%2CU) 
    
67.  Sollis, E. et al. The NHGRI-EBI GWAS catalog: knowledgebase and deposition resource. _Nucleic Acids Res._ **51**, D977–D985 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1GhsLzK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36350656)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20NHGRI-EBI%20GWAS%20catalog%3A%20knowledgebase%20and%20deposition%20resource&journal=Nucleic%20Acids%20Res.&volume=51&pages=D977-D985&publication_year=2023&author=Sollis%2CE) 
    
68.  Ribeiro, D. diogomribeiro/noncoding\_rarevariant: v2. _Zenodo_ [https://doi.org/10.5281/zenodo.15546894](https://doi.org/10.5281/zenodo.15546894) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02288-x?format=refman&flavour=references)

Acknowledgements
----------------

This work was funded by a Swiss National Science Foundation project grant (no. PP00P3\_176977) to O.D. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. We thank Z. Kutalik for fruitful discussions and support.

Author information
------------------

### Authors and Affiliations

1.  Department of Computational Biology, University of Lausanne, Lausanne, Switzerland
    
    Diogo M. Ribeiro & Robin J. Hofmeister
    
2.  Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland
    
    Simone Rubinacci
    
3.  Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA
    
    Simone Rubinacci
    
4.  Regeneron Genetics Center, Tarrytown, NY, USA
    
    Olivier Delaneau
    

Authors

1.  Diogo M. Ribeiro
    
    [View author publications](https://www.nature.com/search?author=Diogo%20M.%20Ribeiro)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Diogo%20M.%20Ribeiro) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Diogo%20M.%20Ribeiro%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Robin J. Hofmeister
    
    [View author publications](https://www.nature.com/search?author=Robin%20J.%20Hofmeister)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Robin%20J.%20Hofmeister) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Robin%20J.%20Hofmeister%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Simone Rubinacci
    
    [View author publications](https://www.nature.com/search?author=Simone%20Rubinacci)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Simone%20Rubinacci) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Simone%20Rubinacci%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Olivier Delaneau
    
    [View author publications](https://www.nature.com/search?author=Olivier%20Delaneau)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Olivier%20Delaneau) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Olivier%20Delaneau%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

D.M.R. and R.J.H. performed the experiments. D.M.R. analyzed the data and wrote the manuscript with input from O.D. D.M.R., S.R. and O.D. conceived, designed and managed the study.

### Corresponding authors

Correspondence to [Diogo M. Ribeiro](mailto:diogo.am.ribeiro@gmail.com) or [Olivier Delaneau](mailto:olivier.delaneau@regeneron.com).

Ethics declarations
-------------------

### Competing interests

O.D. is a current employee of Regeneron Genetics Center, which is a subsidiary of Regeneron Pharmaceuticals. D.M.R. currently works as a contractor for F. Hoffmann-La Roche AG. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Andrew Johnson, Jacob Ulirsch and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02288-x#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM1_ESM.pdf)

Supplementary Figs. 1–20.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM4_ESM.xlsx)

Supplementary Tables 1–12.

Source data
-----------

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM5_ESM.xlsx)

Statistical source data for Fig. 2.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM6_ESM.xlsx)

Statistical source data for Fig. 3.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM7_ESM.xlsx)

Statistical source data for Fig. 4.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02288-x/MediaObjects/41588_2025_2288_MOESM8_ESM.xlsx)

Statistical source data for Fig. 5.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Noncoding%20rare%20variant%20associations%20with%20blood%20traits%20in%20166%2C740%20UK%20Biobank%20genomes&author=Diogo%20M.%20Ribeiro%20et%20al&contentID=10.1038%2Fs41588-025-02288-x&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-06&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02288-x)

### Cite this article

Ribeiro, D.M., Hofmeister, R.J., Rubinacci, S. _et al._ Noncoding rare variant associations with blood traits in 166,740 UK Biobank genomes. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02288-x

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02288-x?format=refman&flavour=citation)

-   Received: 05 December 2023
    
-   Accepted: 03 July 2025
    
-   Published: 06 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02288-x
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
