> id: https://www.nature.com/articles/s41588-025-02259-2

> link: https://www.nature.com/articles/s41588-025-02259-2

> title: Genomics-driven discovery of superior alleles and genes for yellow rust resistance in wheat

# Genomics-driven discovery of superior alleles and genes for yellow rust resistance in wheat
_Published on Tue, 22 Jul 2025 00:00:00 GMT_

Abstract
--------

Yellow rust (YR), caused by _Puccinia striiformis_ f. sp. _tritici_, poses a significant threat to wheat production worldwide. Breeding resistant cultivars is crucial for managing this disease. However, our understanding of the genetic mechanisms underlying YR resistance remains fragmented. To address this, we conducted a comprehensive analysis with variome data from 2,191 wheat accessions worldwide and over 47,000 YR response records across several environments and pathogen races. Through genome-wide association studies, we established a landscape of 431 YR resistance loci, providing a rich resource for resistance gene deployment. Furthermore, we cloned genes corresponding to three resistance loci, namely _Yr5x_ (effective against several _P._ _striiformis_ f. sp. _tritici_ races), _Yr6/Pm5_ (conferred resistance to two pathogen species) and _YrKB_ (_TaEDR2-B_; conferring broad-spectrum rust resistance without yield penalty). These findings offer valuable insights into the genetic basis of YR resistance in wheat and lay the foundation for engineering wheat with durable disease resistance.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 12 print issues and online access

$209.00 per year

only $17.42 per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02259-2?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Global geography of wheat YR and wheat accessions, population structure and diversity of YR response.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_Fig1_HTML.png)

**Fig. 2: A landscape for tracking YR resistance genes and plentiful effective alleles.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_Fig2_HTML.png)

**Fig. 3: Isolation of _Yr5_ alleles.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_Fig3_HTML.png)

**Fig. 4: Identification and functional validation of _Yr6_/_Pm5._**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_Fig4_HTML.png)

**Fig. 5: Identification and functional validation of _TaEDR2-B._**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_Fig5_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-00785-7/MediaObjects/41598_2021_785_Fig1_HTML.png)

### [Identification of bread wheat genotypes with superior grain yield and agronomic traits through evaluation under rust epiphytotic conditions in Kenya](https://www.nature.com/articles/s41598-021-00785-7?fromPaywallRec=true)

Article Open access 01 November 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-023-39993-2/MediaObjects/41467_2023_39993_Fig1_HTML.png)

### [Sequencing trait-associated mutations to clone wheat rust-resistance gene _YrNAM_](https://www.nature.com/articles/s41467-023-39993-2?fromPaywallRec=true)

Article Open access 19 July 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-024-82772-2/MediaObjects/41598_2024_82772_Fig1_HTML.png)

### [Evaluation of resistance and molecular detection of resistance genes to wheat stripe rust of 82 wheat cultivars in Xinjiang, China](https://www.nature.com/articles/s41598-024-82772-2?fromPaywallRec=true)

Article Open access 28 December 2024

Data availability
-----------------

The raw sequence data for WGS and RNA-seq reported in this paper have been deposited in National Center for Biotechnology Information (NCBI project number: [PRJNA1255391](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA1255391) and [PRJNA1256661](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA1256661)) that are publicly accessible at [https://www.ncbi.nlm.nih.gov/](https://www.ncbi.nlm.nih.gov/). These data have also been deposited in the Genome Sequence Archive[101](https://www.nature.com/articles/s41588-025-02259-2#ref-CR101 "CNCB-NGDC Members and Partners. Database resources of the National Genomics Data Center, China National Center for Bioinformation in 2022. Nucleic Acids Res. 50, 27–38 (2022).") in National Genomics Data Center[102](https://www.nature.com/articles/s41588-025-02259-2#ref-CR102 "Chen, T. et al. The genome sequence archive family: toward explosive data growth and diverse data types. Genomics Proteomics Bioinformatics 19, 578–583 (2021)."), China National Center for Bioinformation/Beijing Institute of Genomics, Chinese Academy of Sciences (GSA: [CRA004322](http://bigd.big.ac.cn/gsa/browse/CRA004322), [CRA014998](http://bigd.big.ac.cn/gsa/browse/CRA014998), [CRA005878](http://bigd.big.ac.cn/gsa/browse/CRA005878) and [CRA021484](http://bigd.big.ac.cn/gsa/browse/CRA021484)) that are publicly accessible at [https://ngdc.cncb.ac.cn/gsa](https://ngdc.cncb.ac.cn/gsa). The genotype data of the variant call format (VCF) are available in NGDC under accession number [GVM000830](http://bigd.big.ac.cn/gsa/browse/GVM000830). The genotype data are also available via Zenodo at [https://doi.org/10.5281/zenodo.15608141](https://doi.org/10.5281/zenodo.15608141) (ref. [103](https://www.nature.com/articles/s41588-025-02259-2#ref-CR103 "Wu, J. Genomics-guided landscape unlocks superior alleles and genes for yellow rust resistance in wheat. Zenodo 
https://doi.org/10.5281/zenodo.15608140
(2025).")). To facilitate easier access and accelerate stripe rust resistance breeding, we have developed a dedicated database ([https://wheat.dftianyi.com](https://wheat.dftianyi.com/)). All cloned genes PQ112656 (_Yr5x_, original name _Yr5c_), PQ112657 (_Yr6_) and PQ112658 (_TaEDR2-B_) were available in NCBI GenBank. [Source data](https://www.nature.com/articles/s41588-025-02259-2#Sec31) are provided with this paper.

Code availability
-----------------

All software used in this study is publicly available on the Internet as described in the [Methods](https://www.nature.com/articles/s41588-025-02259-2#Sec10) and Reporting Summary.

References
----------

1.  _FAOSTAT Crops and Livestock Products_ (Food and Agriculture Organization of the United Nations (FAO), accessed 12 January 2024); [https://www.fao.org/faostat/en/#data/QC](https://www.fao.org/faostat/en/#data/QC)
    
2.  Savary, S. et al. The global burden of pathogens and pests on major food crops. _Nat. Ecol. Evol._ **3**, 430–439 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30718852)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20global%20burden%20of%20pathogens%20and%20pests%20on%20major%20food%20crops&journal=Nat.%20Ecol.%20Evol.&volume=3&pages=430-439&publication_year=2019&author=Savary%2CS) 
    
3.  Hafeez, A. N. et al. Creation and judicious application of a wheat resistance gene atlas. _Mol. Plant_ **14**, 1053–1070 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtF2msrjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33991673)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Creation%20and%20judicious%20application%20of%20a%20wheat%20resistance%20gene%20atlas&journal=Mol.%20Plant&volume=14&pages=1053-1070&publication_year=2021&author=Hafeez%2CAN) 
    
4.  Hovmøller, M. S., Walter, S. & Justesen, A. F. Escalating threat of wheat rusts. _Science_ **329**, 369 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20651122)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Escalating%20threat%20of%20wheat%20rusts&journal=Science&volume=329&publication_year=2010&author=Hovm%C3%B8ller%2CMS&author=Walter%2CS&author=Justesen%2CAF) 
    
5.  Hovmøller, M. S., Thach, T. & Justesen, A. F. Global dispersal and diversity of rust fungi in the context of plant health. _Curr. Opin. Microbiol._ **71**, 102243 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36462410)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20dispersal%20and%20diversity%20of%20rust%20fungi%20in%20the%20context%20of%20plant%20health&journal=Curr.%20Opin.%20Microbiol.&volume=71&publication_year=2023&author=Hovm%C3%B8ller%2CMS&author=Thach%2CT&author=Justesen%2CAF) 
    
6.  Wellings, C. R. Global status of stripe rust: a review of historical and current threats. _Euphytica_ **179**, 129–141 (2011).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20status%20of%20stripe%20rust%3A%20a%20review%20of%20historical%20and%20current%20threats&journal=Euphytica&volume=179&pages=129-141&publication_year=2011&author=Wellings%2CCR) 
    
7.  Chen, X. Pathogens which threaten food security: _Puccinia striiformis_, the wheat stripe rust pathogen. _Food Sec._ **12**, 239–251 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathogens%20which%20threaten%20food%20security%3A%20Puccinia%20striiformis%2C%20the%20wheat%20stripe%20rust%20pathogen&journal=Food%20Sec.&volume=12&pages=239-251&publication_year=2020&author=Chen%2CX) 
    
8.  Beddow, J. M. et al. Research investment implications of shifts in the global geography of wheat stripe rust. _Nat. Plants_ **1**, 15132 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27251389)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Research%20investment%20implications%20of%20shifts%20in%20the%20global%20geography%20of%20wheat%20stripe%20rust&journal=Nat.%20Plants&volume=1&publication_year=2015&author=Beddow%2CJM) 
    
9.  Bouvet, L. et al. The evolving battle between yellow rust and wheat: implications for global food security. _Theor. Appl. Genet._ **135**, 741–753 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34821981)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20evolving%20battle%20between%20yellow%20rust%20and%20wheat%3A%20implications%20for%20global%20food%20security&journal=Theor.%20Appl.%20Genet.&volume=135&pages=741-753&publication_year=2022&author=Bouvet%2CL) 
    
10.  McIntosh, R., Mu, J., Han, D. & Kang, Z. Wheat stripe rust resistance gene _Yr24/Yr26_: a retrospective review. _Crop J._ **6**, 321–329 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Wheat%20stripe%20rust%20resistance%20gene%20Yr24%2FYr26%3A%20a%20retrospective%20review&journal=Crop%20J.&volume=6&pages=321-329&publication_year=2018&author=McIntosh%2CR&author=Mu%2CJ&author=Han%2CD&author=Kang%2CZ) 
    
11.  McIntosh, R. A. Catalogue of gene symbols for wheat: 2024 supplement. _GrainGenes_ [https://graingenes.org/GG3/content/wheat-gene-catalogue-2024-released-covering-all-wgc-curations-until-2024](https://wheat.pw.usda.gov/GG3/WGC) (2024).
    
12.  Sharma, D. et al. A single NLR gene confers resistance to leaf and stripe rust in wheat. _Nat. Commun._ **15**, 1925 (2024).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single%20NLR%20gene%20confers%20resistance%20to%20leaf%20and%20stripe%20rust%20in%20wheat&journal=Nat.%20Commun.&volume=15&publication_year=2024&author=Sharma%2CD) 
    
13.  Chen, R., Gajendiran, K. & Wulff, B. B. H. R we there yet? Advances in cloning resistance genes for engineering immunity in crop plants. _Curr. Opin. Plant Biol._ **77**, 102489 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXis1ShsLjJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38128298)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=R%20we%20there%20yet%3F%20Advances%20in%20cloning%20resistance%20genes%20for%20engineering%20immunity%20in%20crop%20plants&journal=Curr.%20Opin.%20Plant%20Biol.&volume=77&publication_year=2024&author=Chen%2CR&author=Gajendiran%2CK&author=Wulff%2CBBH) 
    
14.  Dracatos, P. M., Lu, J., Sánchez Martín, J. & Wulff, B. B. H. Resistance that stacks up: engineering rust and mildew disease control in the cereal crops wheat and barley. _Plant Biotechnol. J._ **21**, 1938–1951 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1Smsb7F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37494504)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10502761)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resistance%20that%20stacks%20up%3A%20engineering%20rust%20and%20mildew%20disease%20control%20in%20the%20cereal%20crops%20wheat%20and%20barley&journal=Plant%20Biotechnol.%20J.&volume=21&pages=1938-1951&publication_year=2023&author=Dracatos%2CPM&author=Lu%2CJ&author=S%C3%A1nchez%20Mart%C3%ADn%2CJ&author=Wulff%2CBBH) 
    
15.  Keller, B., Wicker, T. & Krattinger, S. G. Advances in wheat and pathogen genomics: implications for disease control. _Annu. Rev. Phytopathol._ **56**, 67–87 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2ltr%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30149791)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Advances%20in%20wheat%20and%20pathogen%20genomics%3A%20implications%20for%20disease%20control&journal=Annu.%20Rev.%20Phytopathol.&volume=56&pages=67-87&publication_year=2018&author=Keller%2CB&author=Wicker%2CT&author=Krattinger%2CSG) 
    
16.  Jaganathan, D., Bohra, A., Thudi, M. & Varshney, R. K. Fine mapping and gene cloning in the post-NGS era: advances and prospects. _Theor. Appl. Genet._ **133**, 1791–1810 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlyjsrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32040676)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7214393)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fine%20mapping%20and%20gene%20cloning%20in%20the%20post-NGS%20era%3A%20advances%20and%20prospects&journal=Theor.%20Appl.%20Genet.&volume=133&pages=1791-1810&publication_year=2020&author=Jaganathan%2CD&author=Bohra%2CA&author=Thudi%2CM&author=Varshney%2CRK) 
    
17.  McGuire, A. L. et al. The road ahead in genetics and genomics. _Nat. Rev. Genet._ **21**, 581–596 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1KhtL3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32839576)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7444682)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20road%20ahead%20in%20genetics%20and%20genomics&journal=Nat.%20Rev.%20Genet.&volume=21&pages=581-596&publication_year=2020&author=McGuire%2CAL) 
    
18.  Ali, S. et al. Origin, migration routes and worldwide population genetic structure of the wheat yellow rust pathogen _Puccinia striif_ormis f. sp. _tritici_. _PLoS Pathog._ **10**, e1003903 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24465211)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900651)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Origin%2C%20migration%20routes%20and%20worldwide%20population%20genetic%20structure%20of%20the%20wheat%20yellow%20rust%20pathogen%20Puccinia%20striiformis%20f.%20sp.%20tritici&journal=PLoS%20Pathog.&volume=10&publication_year=2014&author=Ali%2CS) 
    
19.  Li, Z. & Zeng, S. _Wheat Rusts in China_ (in Chinese) (Chinese Agricultural Press, 2002).
    
20.  Stubbs, R. in _Cereal Rusts_ Vol. II (eds Roelfs A. P. & Bushnell W. R.) 61–101 (Academic Press, 1985).
    
21.  Han, D. et al. Identification and evaluation of resistance to stripe rust in 1980 wheat landraces and abroad germplasm. _Sci. Agric. Sin._ **45**, 5013–5023 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXmsF2rtb0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20and%20evaluation%20of%20resistance%20to%20stripe%20rust%20in%201980%20wheat%20landraces%20and%20abroad%20germplasm&journal=Sci.%20Agric.%20Sin.&volume=45&pages=5013-5023&publication_year=2012&author=Han%2CD) 
    
22.  Mu, J. et al. Identification of sources of resistance in geographically diverse wheat accessions to stripe rust pathogen in China. _Crop Prot._ **122**, 1–8 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20sources%20of%20resistance%20in%20geographically%20diverse%20wheat%20accessions%20to%20stripe%20rust%20pathogen%20in%20China&journal=Crop%20Prot.&volume=122&pages=1-8&publication_year=2019&author=Mu%2CJ) 
    
23.  Wang, M. et al. Variation in _TaSPL6-D_ confers salinity tolerance in bread wheat by activating _TaHKT1;5-D_ while preserving yield-related traits. _Nat. Genet._ **56**, 1257–1269 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtFKgtrrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38802564)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Variation%20in%20TaSPL6-D%20confers%20salinity%20tolerance%20in%20bread%20wheat%20by%20activating%20TaHKT1%3B5-D%20while%20preserving%20yield-related%20traits&journal=Nat.%20Genet.&volume=56&pages=1257-1269&publication_year=2024&author=Wang%2CM) 
    
24.  Wu, J. et al. Association analysis identifies new loci for resistance to Chinese _Yr26_\-virulent races of the stripe rust pathogen in a diverse panel of wheat germplasm. _Plant Dis._ **104**, 1751–1762 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFyktb3J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32293995)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20analysis%20identifies%20new%20loci%20for%20resistance%20to%20Chinese%20Yr26-virulent%20races%20of%20the%20stripe%20rust%20pathogen%20in%20a%20diverse%20panel%20of%20wheat%20germplasm&journal=Plant%20Dis.&volume=104&pages=1751-1762&publication_year=2020&author=Wu%2CJ) 
    
25.  Wu, J. et al. A large‐scale genomic association analysis identifies the candidate causal genes conferring stripe rust resistance under multiple field environments. _Plant Biotechnol. J._ **19**, 177–191 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32677132)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20large%E2%80%90scale%20genomic%20association%20analysis%20identifies%20the%20candidate%20causal%20genes%20conferring%20stripe%20rust%20resistance%20under%20multiple%20field%20environments&journal=Plant%20Biotechnol.%20J.&volume=19&pages=177-191&publication_year=2021&author=Wu%2CJ) 
    
26.  Zeng, Q. et al. Stripe rust resistance and genes in Chinese wheat cultivars and breeding lines. _Euphytica_ **196**, 271–284 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVyis7bO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stripe%20rust%20resistance%20and%20genes%20in%20Chinese%20wheat%20cultivars%20and%20breeding%20lines&journal=Euphytica&volume=196&pages=271-284&publication_year=2014&author=Zeng%2CQ) 
    
27.  Zhang, G. et al. Evaluation of the potential risk of the emerging _Yr5_\-virulent races of _Puccinia striiformis_ f. sp. _tritici_ to 165 Chinese wheat cultivars. _Plant Dis._ **106**, 1867–1874 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVahsL3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35021876)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluation%20of%20the%20potential%20risk%20of%20the%20emerging%20Yr5-virulent%20races%20of%20Puccinia%20striiformis%20f.%20sp.%20tritici%20to%20165%20Chinese%20wheat%20cultivars&journal=Plant%20Dis.&volume=106&pages=1867-1874&publication_year=2022&author=Zhang%2CG) 
    
28.  Schulthess, A. W. et al. Genomics-informed prebreeding unlocks the diversity in genebanks for wheat improvement. _Nat. Genet._ **54**, 1544–1552 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFKgsr%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36195758)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomics-informed%20prebreeding%20unlocks%20the%20diversity%20in%20genebanks%20for%20wheat%20improvement&journal=Nat.%20Genet.&volume=54&pages=1544-1552&publication_year=2022&author=Schulthess%2CAW) 
    
29.  Zhu, T. et al. Optical maps refine the bread wheat _Triticum aestivum_ cv. Chinese Spring genome assembly. _Plant J._ **107**, 303–314 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFCls7jN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33893684)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8360199)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optical%20maps%20refine%20the%20bread%20wheat%20Triticum%20aestivum%20cv.%20Chinese%20Spring%20genome%20assembly&journal=Plant%20J.&volume=107&pages=303-314&publication_year=2021&author=Zhu%2CT) 
    
30.  Balfourier, F. et al. Worldwide phylogeography and history of wheat genetic diversity. _Sci. Adv._ **5**, eaav0536 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31149630)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6541461)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Worldwide%20phylogeography%20and%20history%20of%20wheat%20genetic%20diversity&journal=Sci.%20Adv.&volume=5&publication_year=2019&author=Balfourier%2CF) 
    
31.  Zhao, X. et al. Population genomics unravels the Holocene history of bread wheat and its relatives. _Nat. Plants_ **9**, 403–419 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36928772)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Population%20genomics%20unravels%20the%20Holocene%20history%20of%20bread%20wheat%20and%20its%20relatives&journal=Nat.%20Plants&volume=9&pages=403-419&publication_year=2023&author=Zhao%2CX) 
    
32.  Brar, G. S., Fetch, T., McCallum, B. D., Hucl, P. J. & Kutcher, H. R. Virulence dynamics and breeding for resistance to stripe, stem, and leaf rust in Canada since 2000. _Plant Dis._ **103**, 2981–2995 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXptlWks7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31634033)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Virulence%20dynamics%20and%20breeding%20for%20resistance%20to%20stripe%2C%20stem%2C%20and%20leaf%20rust%20in%20Canada%20since%202000&journal=Plant%20Dis.&volume=103&pages=2981-2995&publication_year=2019&author=Brar%2CGS&author=Fetch%2CT&author=McCallum%2CBD&author=Hucl%2CPJ&author=Kutcher%2CHR) 
    
33.  Hovmøller, M. S. et al. Replacement of the European wheat yellow rust population by new races from the centre of diversity in the near‐Himalayan region. _Plant Pathol._ **65**, 402–411 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Replacement%20of%20the%20European%20wheat%20yellow%20rust%20population%20by%20new%20races%20from%20the%20centre%20of%20diversity%20in%20the%20near%E2%80%90Himalayan%20region&journal=Plant%20Pathol.&volume=65&pages=402-411&publication_year=2016&author=Hovm%C3%B8ller%2CMS) 
    
34.  Liu, T., Wan, A., Liu, D. & Chen, X. Changes of races and virulence genes in _Puccinia striiformis_ f. sp. _tritici_, the wheat stripe rust pathogen, in the United States from 1968 to 2009. _Plant Dis._ **101**, 1522–1532 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXot1CmsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30678601)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Changes%20of%20races%20and%20virulence%20genes%20in%20Puccinia%20striiformis%20f.%20sp.%20tritici%2C%20the%20wheat%20stripe%20rust%20pathogen%2C%20in%20the%20United%20States%20from%201968%20to%202009&journal=Plant%20Dis.&volume=101&pages=1522-1532&publication_year=2017&author=Liu%2CT&author=Wan%2CA&author=Liu%2CD&author=Chen%2CX) 
    
35.  Milus, E. A., Kristensen, K. & Hovmøller, M. S. Evidence for increased aggressiveness in a recent widespread strain of _Puccinia striiformis_ f. sp. _tritici_ causing stripe rust of wheat. _Phytopathology_ **99**, 89–94 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19055439)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evidence%20for%20increased%20aggressiveness%20in%20a%20recent%20widespread%20strain%20of%20Puccinia%20striiformis%20f.%20sp.%20tritici%20causing%20stripe%20rust%20of%20wheat&journal=Phytopathology&volume=99&pages=89-94&publication_year=2009&author=Milus%2CEA&author=Kristensen%2CK&author=Hovm%C3%B8ller%2CMS) 
    
36.  Wellings, C. R. & McIntosh, R. A. _Puccinia striiformis_ f. sp. _tritici_ in Australasia: Pathogenic changes during the first 10 years. _Plant Pathol._ **39**, 316–325 (1990).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Puccinia%20striiformis%20f.%20sp.%20tritici%20in%20Australasia%3A%20Pathogenic%20changes%20during%20the%20first%2010%20years&journal=Plant%20Pathol.&volume=39&pages=316-325&publication_year=1990&author=Wellings%2CCR&author=McIntosh%2CRA) 
    
37.  Chen, X. & Kang, Z. _Stripe Rust_ (Springer, 2017).
    
38.  Ali, S. et al. Yellow rust epidemics worldwide were caused by pathogen races from divergent genetic lineages. _Front. Plant Sci._ **8**, 1057 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28676811)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5477562)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yellow%20rust%20epidemics%20worldwide%20were%20caused%20by%20pathogen%20races%20from%20divergent%20genetic%20lineages&journal=Front.%20Plant%20Sci.&volume=8&publication_year=2017&author=Ali%2CS) 
    
39.  Fradgley, N. et al. A large-scale pedigree resource of wheat reveals evidence for adaptation and selection by breeders. _PLoS Biol._ **17**, e3000071 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30818353)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6413959)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20large-scale%20pedigree%20resource%20of%20wheat%20reveals%20evidence%20for%20adaptation%20and%20selection%20by%20breeders&journal=PLoS%20Biol.&volume=17&publication_year=2019&author=Fradgley%2CN) 
    
40.  Zhuang, Q. _Chinese Wheat Improvement and Pedigree Analysis_ (China Agriculture Press, 2003).
    
41.  Marchal, C. et al. BED-domain-containing immune receptors confer diverse resistance spectra to yellow rust. _Nat. Plants_ **4**, 662–668 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2nsrnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30150615)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BED-domain-containing%20immune%20receptors%20confer%20diverse%20resistance%20spectra%20to%20yellow%20rust&journal=Nat.%20Plants&volume=4&pages=662-668&publication_year=2018&author=Marchal%2CC) 
    
42.  Gardiner, L. J. et al. Mapping‐by‐sequencing in complex polyploid genomes using genic sequence capture: a case study to map yellow rust resistance in hexaploid wheat. _Plant J._ **87**, 403–419 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFOgtrbN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27144898)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5026171)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%E2%80%90by%E2%80%90sequencing%20in%20complex%20polyploid%20genomes%20using%20genic%20sequence%20capture%3A%20a%20case%20study%20to%20map%20yellow%20rust%20resistance%20in%20hexaploid%20wheat&journal=Plant%20J.&volume=87&pages=403-419&publication_year=2016&author=Gardiner%2CLJ) 
    
43.  Jia, J. et al. Genome resources for the elite bread wheat cultivar Aikang 58 and mining of elite homeologous haplotypes for accelerating wheat improvement. _Mol. Plant_ **16**, 1893–1910 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisVamsLrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37897037)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20resources%20for%20the%20elite%20bread%20wheat%20cultivar%20Aikang%2058%20and%20mining%20of%20elite%20homeologous%20haplotypes%20for%20accelerating%20wheat%20improvement&journal=Mol.%20Plant&volume=16&pages=1893-1910&publication_year=2023&author=Jia%2CJ) 
    
44.  Xie, J. et al. A rare single nucleotide variant in _Pm5e_ confers powdery mildew resistance in common wheat. _New Phytol._ **228**, 1011–1026 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVCrsLvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32569398)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20rare%20single%20nucleotide%20variant%20in%20Pm5e%20confers%20powdery%20mildew%20resistance%20in%20common%20wheat&journal=New%20Phytol.&volume=228&pages=1011-1026&publication_year=2020&author=Xie%2CJ) 
    
45.  Pal, N. et al. Meta-QTLs for multiple disease resistance involving three rusts in common wheat (_Triticum aestivum_ L.). _Theor. Appl. Genet._ **135**, 2385–2405 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvV2jt7vL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35699741)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Meta-QTLs%20for%20multiple%20disease%20resistance%20involving%20three%20rusts%20in%20common%20wheat%20%28Triticum%20aestivum%20L.%29&journal=Theor.%20Appl.%20Genet.&volume=135&pages=2385-2405&publication_year=2022&author=Pal%2CN) 
    
46.  Huang, S. et al. Utilization of the genomewide wheat 55K SNP array for genetic analysis of stripe rust resistance in common wheat line P9936. _Phytopathology_ **109**, 819–827 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsV2gt7zF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30644331)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Utilization%20of%20the%20genomewide%20wheat%2055K%20SNP%20array%20for%20genetic%20analysis%20of%20stripe%20rust%20resistance%20in%20common%20wheat%20line%20P9936&journal=Phytopathology&volume=109&pages=819-827&publication_year=2019&author=Huang%2CS) 
    
47.  Mu, J. et al. Genetic architecture of wheat stripe rust resistance revealed by combining QTL mapping using SNP-based genetic maps and bulked segregant analysis. _Theor. Appl. Genet._ **132**, 443–455 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlShsb%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30446795)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20architecture%20of%20wheat%20stripe%20rust%20resistance%20revealed%20by%20combining%20QTL%20mapping%20using%20SNP-based%20genetic%20maps%20and%20bulked%20segregant%20analysis&journal=Theor.%20Appl.%20Genet.&volume=132&pages=443-455&publication_year=2019&author=Mu%2CJ) 
    
48.  Rosewarne, G. M. et al. Analysis of leaf and stripe rust severities reveals pathotype changes and multiple minor QTLs associated with resistance in an Avocet × Pastor wheat population. _Theor. Appl. Genet._ **124**, 1283–1294 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xls1SitrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22274764)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20leaf%20and%20stripe%20rust%20severities%20reveals%20pathotype%20changes%20and%20multiple%20minor%20QTLs%20associated%20with%20resistance%20in%20an%20Avocet%20%C3%97%20Pastor%20wheat%20population&journal=Theor.%20Appl.%20Genet.&volume=124&pages=1283-1294&publication_year=2012&author=Rosewarne%2CGM) 
    
49.  Fu, D. et al. A kinase-START gene confers temperature-dependent resistance to wheat stripe rust. _Science_ **323**, 1357–1360 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXisFemtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19228999)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4737487)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20kinase-START%20gene%20confers%20temperature-dependent%20resistance%20to%20wheat%20stripe%20rust&journal=Science&volume=323&pages=1357-1360&publication_year=2009&author=Fu%2CD) 
    
50.  Xiong, H. et al. A large‐scale whole‐exome sequencing mutant resource for functional genomics in wheat. _Plant Biotechnol. J._ **21**, 2047–2056 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVegsrfE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37401008)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10502753)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20large%E2%80%90scale%20whole%E2%80%90exome%20sequencing%20mutant%20resource%20for%20functional%20genomics%20in%20wheat&journal=Plant%20Biotechnol.%20J.&volume=21&pages=2047-2056&publication_year=2023&author=Xiong%2CH) 
    
51.  Marchal, C. et al. Comparative genomics and functional studies of wheat BED-NLR loci. _Genes_ **11**, 1406 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFWmtLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33256067)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7761493)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparative%20genomics%20and%20functional%20studies%20of%20wheat%20BED-NLR%20loci&journal=Genes&volume=11&publication_year=2020&author=Marchal%2CC) 
    
52.  Bourras, S. et al. The _AvrPm3-Pm3_ effector-NLR interactions control both race-specific resistance and host-specificity of cereal mildews on wheat. _Nat. Commun._ **10**, 2292 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31123263)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6533294)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20AvrPm3-Pm3%20effector-NLR%20interactions%20control%20both%20race-specific%20resistance%20and%20host-specificity%20of%20cereal%20mildews%20on%20wheat&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Bourras%2CS) 
    
53.  Zhang, J. et al. Single amino acid change alters specificity of the multi-allelic wheat stem rust resistance locus _SR9_. _Nat. Commun._ **14**, 7354 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisVekurjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37963867)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10645757)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single%20amino%20acid%20change%20alters%20specificity%20of%20the%20multi-allelic%20wheat%20stem%20rust%20resistance%20locus%20SR9&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Zhang%2CJ) 
    
54.  Gill, B. K. et al. Function and evolution of allelic variations of _Sr13_ conferring resistance to stem rust in tetraploid wheat (_Triticum turgidum_ L.). _Plant J._ **106**, 1674–1691 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtF2ltb3I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33825238)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8362117)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Function%20and%20evolution%20of%20allelic%20variations%20of%20Sr13%20conferring%20resistance%20to%20stem%20rust%20in%20tetraploid%20wheat%20%28Triticum%20turgidum%20L.%29&journal=Plant%20J.&volume=106&pages=1674-1691&publication_year=2021&author=Gill%2CBK) 
    
55.  Naz, A. A. et al. _Lr21_ diversity unveils footprints of wheat evolution and its new role in broad‐spectrum leaf rust resistance. _Plant Cell Environ._ **44**, 3445–3458 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1yntbnN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34212402)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lr21%20diversity%20unveils%20footprints%20of%20wheat%20evolution%20and%20its%20new%20role%20in%20broad%E2%80%90spectrum%20leaf%20rust%20resistance&journal=Plant%20Cell%20Environ.&volume=44&pages=3445-3458&publication_year=2021&author=Naz%2CAA) 
    
56.  Athiyannan, N. et al. Long-read genome sequencing of bread wheat facilitates disease resistance gene cloning. _Nat. Genet._ **54**, 227–231 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XntFCjtbY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35288708)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8920886)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Long-read%20genome%20sequencing%20of%20bread%20wheat%20facilitates%20disease%20resistance%20gene%20cloning&journal=Nat.%20Genet.&volume=54&pages=227-231&publication_year=2022&author=Athiyannan%2CN) 
    
57.  Bettgenhaeuser, J. et al. The barley immune receptor _Mla_ recognizes multiple pathogens and contributes to host range dynamics. _Nat. Commun._ **12**, 6915 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFyht7vM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34824299)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8617247)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20barley%20immune%20receptor%20Mla%20recognizes%20multiple%20pathogens%20and%20contributes%20to%20host%20range%20dynamics&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Bettgenhaeuser%2CJ) 
    
58.  O’Hara, T. et al. The wheat powdery mildew resistance gene _Pm4_ also confers resistance to wheat blast. _Nat. Plants_ **10**, 984–993 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38898165)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11208137)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20wheat%20powdery%20mildew%20resistance%20gene%20Pm4%20also%20confers%20resistance%20to%20wheat%20blast&journal=Nat.%20Plants&volume=10&pages=984-993&publication_year=2024&author=O%E2%80%99Hara%2CT) 
    
59.  Asuke, S. et al. Evolution of wheat blast resistance gene _Rmg8_ accompanied by differentiation of variants recognizing the powdery mildew fungus. _Nat. Plants_ **10**, 971–983 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsVansbfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38898164)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolution%20of%20wheat%20blast%20resistance%20gene%20Rmg8%20accompanied%20by%20differentiation%20of%20variants%20recognizing%20the%20powdery%20mildew%20fungus&journal=Nat.%20Plants&volume=10&pages=971-983&publication_year=2024&author=Asuke%2CS) 
    
60.  Arora, S. et al. A wheat kinase and immune receptor form host-specificity barriers against the blast fungus. _Nat. Plants_ **9**, 385–392 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjs1agsL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36797350)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10027608)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20wheat%20kinase%20and%20immune%20receptor%20form%20host-specificity%20barriers%20against%20the%20blast%20fungus&journal=Nat.%20Plants&volume=9&pages=385-392&publication_year=2023&author=Arora%2CS) 
    
61.  Lu, P. et al. A rare gain of function mutation in a wheat tandem kinase confers resistance to powdery mildew. _Nat. Commun._ **11**, 680 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjs1OitLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32015344)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6997164)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20rare%20gain%20of%20function%20mutation%20in%20a%20wheat%20tandem%20kinase%20confers%20resistance%20to%20powdery%20mildew&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Lu%2CP) 
    
62.  Bentham, A. R. et al. Allelic compatibility in plant immune receptors facilitates engineering of new effector recognition specificities. _Plant Cell_ **35**, 3809–3827 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhtVyksbjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37486356)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10533329)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Allelic%20compatibility%20in%20plant%20immune%20receptors%20facilitates%20engineering%20of%20new%20effector%20recognition%20specificities&journal=Plant%20Cell&volume=35&pages=3809-3827&publication_year=2023&author=Bentham%2CAR) 
    
63.  McDonald, B. A. & Stukenbrock, E. H. Rapid emergence of pathogens in agro-ecosystems: global threats to agricultural sustainability and food security. _Philos. Trans. R. Soc. Lond. B Biol. Sci._ **371**, 20160026 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28080995)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5095548)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20emergence%20of%20pathogens%20in%20agro-ecosystems%3A%20global%20threats%20to%20agricultural%20sustainability%20and%20food%20security&journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&volume=371&publication_year=2016&author=McDonald%2CBA&author=Stukenbrock%2CEH) 
    
64.  Brown, J. K. M. Yield penalties of disease resistance in crops. _Curr. Opin. Plant Biol._ **5**, 339–344 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xls1ejtLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12179968)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yield%20penalties%20of%20disease%20resistance%20in%20crops&journal=Curr.%20Opin.%20Plant%20Biol.&volume=5&pages=339-344&publication_year=2002&author=Brown%2CJKM) 
    
65.  Zhang, H., Liu, Y., Zhang, X., Ji, W. & Kang, Z. A necessary considering factor for breeding: growth-defense tradeoff in plants. _Stress Biol._ **3**, 6 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37676557)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10441926)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20necessary%20considering%20factor%20for%20breeding%3A%20growth-defense%20tradeoff%20in%20plants&journal=Stress%20Biol.&volume=3&publication_year=2023&author=Zhang%2CH&author=Liu%2CY&author=Zhang%2CX&author=Ji%2CW&author=Kang%2CZ) 
    
66.  Oliva, R. et al. Broad-spectrum resistance to bacterial blight in rice using genome editing. _Nat. Biotechnol._ **37**, 1344–1350 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVCksL3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31659337)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6831514)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Broad-spectrum%20resistance%20to%20bacterial%20blight%20in%20rice%20using%20genome%20editing&journal=Nat.%20Biotechnol.&volume=37&pages=1344-1350&publication_year=2019&author=Oliva%2CR) 
    
67.  Wang, Y. et al. Simultaneous editing of three homoeoalleles in hexaploid bread wheat confers heritable resistance to powdery mildew. _Nat. Biotechnol._ **32**, 947–951 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFygs7jJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25038773)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Simultaneous%20editing%20of%20three%20homoeoalleles%20in%20hexaploid%20bread%20wheat%20confers%20heritable%20resistance%20to%20powdery%20mildew&journal=Nat.%20Biotechnol.&volume=32&pages=947-951&publication_year=2014&author=Wang%2CY) 
    
68.  He, Z., Webster, S. & He, S. Y. Growth–defense trade-offs in plants. _Curr. Biol._ **32**, 634–639 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Growth%E2%80%93defense%20trade-offs%20in%20plants&journal=Curr.%20Biol.&volume=32&pages=634-639&publication_year=2022&author=He%2CZ&author=Webster%2CS&author=He%2CSY) 
    
69.  Schrick, K. et al. Shared functions of plant and mammalian StAR-related lipid transfer (START) domains in modulating transcription factor activity. _BMC Biol._ **12**, 70 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25159688)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4169639)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Shared%20functions%20of%20plant%20and%20mammalian%20StAR-related%20lipid%20transfer%20%28START%29%20domains%20in%20modulating%20transcription%20factor%20activity&journal=BMC%20Biol.&volume=12&publication_year=2014&author=Schrick%2CK) 
    
70.  Cheng, H. et al. Frequent intra- and inter-species introgression shapes the landscape of genetic variation in bread wheat. _Genome Biol._ **20**, 136 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31300020)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6624984)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Frequent%20intra-%20and%20inter-species%20introgression%20shapes%20the%20landscape%20of%20genetic%20variation%20in%20bread%20wheat&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Cheng%2CH) 
    
71.  Hao, C. et al. Resequencing of 145 landmark cultivars reveals asymmetric sub-genome selection and strong founder genotype effects on wheat breeding in China. _Mol. Plant_ **13**, 1733–1751 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFSnsrzP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32896642)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resequencing%20of%20145%20landmark%20cultivars%20reveals%20asymmetric%20sub-genome%20selection%20and%20strong%20founder%20genotype%20effects%20on%20wheat%20breeding%20in%20China&journal=Mol.%20Plant&volume=13&pages=1733-1751&publication_year=2020&author=Hao%2CC) 
    
72.  Zhou, Y. et al. _Triticum_ population sequencing provides insights into wheat adaptation. _Nat. Genet._ **52**, 1412–1422 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFKlu7zJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33106631)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Triticum%20population%20sequencing%20provides%20insights%20into%20wheat%20adaptation&journal=Nat.%20Genet.&volume=52&pages=1412-1422&publication_year=2020&author=Zhou%2CY) 
    
73.  Guo, W. et al. Origin and adaptation to high altitude of Tibetan semi-wild wheat. _Nat. Commun._ **11**, 5085 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVehsbrO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33033250)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7545183)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Origin%20and%20adaptation%20to%20high%20altitude%20of%20Tibetan%20semi-wild%20wheat&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Guo%2CW) 
    
74.  Niu, J. et al. Whole-genome sequencing of diverse wheat accessions uncovers genetic changes during modern breeding in China and the United States. _Plant Cell_ **35**, 4199–4216 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhtVyktr%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37647532)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10689146)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20sequencing%20of%20diverse%20wheat%20accessions%20uncovers%20genetic%20changes%20during%20modern%20breeding%20in%20China%20and%20the%20United%20States&journal=Plant%20Cell&volume=35&pages=4199-4216&publication_year=2023&author=Niu%2CJ) 
    
75.  Sharma-Poudyal, D. et al. Virulence characterization of international collections of the wheat stripe rust pathogen, _Puccinia striiformis_ f. sp. _tritici_. _Plant Dis._ **97**, 379–386 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB3cjptlCisA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30722363)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Virulence%20characterization%20of%20international%20collections%20of%20the%20wheat%20stripe%20rust%20pathogen%2C%20Puccinia%20striiformis%20f.%20sp.%20tritici&journal=Plant%20Dis.&volume=97&pages=379-386&publication_year=2013&author=Sharma-Poudyal%2CD) 
    
76.  Line, R. & Qayoum, A. Virulence, aggressiveness, evolution, and distribution of races of _Puccinia striiformis_ (the cause of stripe rust of wheat) in North America 1968–1987. _US Department of Agriculture Technical Bulletin_ **1788**, 74 (1992).
    
77.  Li, H. et al. Cloning of the wheat leaf rust resistance gene _Lr47_ introgressed from _Aegilops speltoides_. _Nat. Commun._ **14**, 6072 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVCkt7zK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37770474)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10539295)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cloning%20of%20the%20wheat%20leaf%20rust%20resistance%20gene%20Lr47%20introgressed%20from%20Aegilops%20speltoides&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Li%2CH) 
    
78.  Peterson, R., Campbell, A. & Hannah, A. A diagrammatic scale for estimating rust intensity on leaves and stems of cereals. _Can. J. Res._ **26**, 496–500 (1948).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20diagrammatic%20scale%20for%20estimating%20rust%20intensity%20on%20leaves%20and%20stems%20of%20cereals&journal=Can.%20J.%20Res.&volume=26&pages=496-500&publication_year=1948&author=Peterson%2CR&author=Campbell%2CA&author=Hannah%2CA) 
    
79.  Bates, D., Maechler, M., Bolker, B. & Walker, S. lme4: linear mixed effects models using Eigen and S4. R version 1.1-4 [https://cran.r-project.org/web/packages/lme4/index.html](https://cran.r-project.org/web/packages/lme4/index.html) (2014).
    
80.  Li, H. & Durbin, R. Fast and accurate long-read alignment with Burrows-Wheeler transform. _Bioinformatics_ **26**, 589–595 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20080505)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2828108)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&journal=Bioinformatics&volume=26&pages=589-595&publication_year=2010&author=Li%2CH&author=Durbin%2CR) 
    
81.  Li, H. et al. The sequence alignment/map format and SAMtools. _Bioinformatics_ **25**, 2078–2079 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19505943)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&journal=Bioinformatics&volume=25&pages=2078-2079&publication_year=2009&author=Li%2CH) 
    
82.  McKenna, A. et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. _Genome Res._ **20**, 1297–1303 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFeru7jM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20644199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genome%20analysis%20toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&journal=Genome%20Res.&volume=20&pages=1297-1303&publication_year=2010&author=McKenna%2CA) 
    
83.  Cingolani, P. et al. Using _Drosophila melanogaster_ as a model for genotoxic chemical mutational studies with a new program, SnpSift. _Front. Genet._ **3**, 35 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22435069)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3304048)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Using%20Drosophila%20melanogaster%20as%20a%20model%20for%20genotoxic%20chemical%20mutational%20studies%20with%20a%20new%20program%2C%20SnpSift&journal=Front.%20Genet.&volume=3&publication_year=2012&author=Cingolani%2CP) 
    
84.  Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff. _Fly_ **6**, 80–92 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff&journal=Fly&volume=6&pages=80-92&publication_year=2014&author=Cingolani%2CP) 
    
85.  Danecek, P. et al. The variant call format and VCFtools. _Bioinformatics_ **27**, 2156–2158 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXptFCqt7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21653522)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137218)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20variant%20call%20format%20and%20VCFtools&journal=Bioinformatics&volume=27&pages=2156-2158&publication_year=2011&author=Danecek%2CP) 
    
86.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
87.  Price, M. N., Dehal, P. S. & Arkin, A. P. FastTree 2–approximately maximum-likelihood trees for large alignments. _PLoS ONE_ **5**, e9490 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20224823)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835736)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FastTree%202%E2%80%93approximately%20maximum-likelihood%20trees%20for%20large%20alignments&journal=PLoS%20ONE&volume=5&publication_year=2010&author=Price%2CMN&author=Dehal%2CPS&author=Arkin%2CAP) 
    
88.  Alexander, D. H., Novembre, J. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. _Genome Res._ **19**, 1655–1664 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFCjsLvL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19648217)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752134)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&journal=Genome%20Res.&volume=19&pages=1655-1664&publication_year=2009&author=Alexander%2CDH&author=Novembre%2CJ&author=Lange%2CK) 
    
89.  Cheng, S. et al. Harnessing landrace diversity empowers wheat breeding. _Nature_ **632**, 823–831 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhs1OhtL7O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38885696)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11338829)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Harnessing%20landrace%20diversity%20empowers%20wheat%20breeding&journal=Nature&volume=632&pages=823-831&publication_year=2024&author=Cheng%2CS) 
    
90.  Yin, L. et al. rMVP: a memory-efficient, visualization-enhanced, and parallel-accelerated tool for genome-wide association study. _Genom. Proteom. Bioinf_ **19**, 619–628 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=rMVP%3A%20a%20memory-efficient%2C%20visualization-enhanced%2C%20and%20parallel-accelerated%20tool%20for%20genome-wide%20association%20study&journal=Genom.%20Proteom.%20Bioinf&volume=19&pages=619-628&publication_year=2021&author=Yin%2CL) 
    
91.  Wu, J. et al. Comparative genome-wide mapping versus extreme pool-genotyping and development of diagnostic SNP markers linked to QTL for adult plant resistance to stripe rust in common wheat. _Theor. Appl. Genet._ **131**, 1777–1792 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1Wqsr7O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29909527)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparative%20genome-wide%20mapping%20versus%20extreme%20pool-genotyping%20and%20development%20of%20diagnostic%20SNP%20markers%20linked%20to%20QTL%20for%20adult%20plant%20resistance%20to%20stripe%20rust%20in%20common%20wheat&journal=Theor.%20Appl.%20Genet.&volume=131&pages=1777-1792&publication_year=2018&author=Wu%2CJ) 
    
92.  Meng, L., Li, H., Zhang, L. & Wang, J. QTL IciMapping: Integrated software for genetic linkage map construction and quantitative trait locus mapping in biparental populations. _Crop J._ **3**, 269–283 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=QTL%20IciMapping%3A%20Integrated%20software%20for%20genetic%20linkage%20map%20construction%20and%20quantitative%20trait%20locus%20mapping%20in%20biparental%20populations&journal=Crop%20J.&volume=3&pages=269-283&publication_year=2015&author=Meng%2CL&author=Li%2CH&author=Zhang%2CL&author=Wang%2CJ) 
    
93.  Van Ooijen, J. _JoinMap4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations_ (Kyazma BV, 2006).
    
94.  Kosambi, D. D. The estimation of map distances from recombination values. _Ann. Eugen._ **12**, 172–175 (1943).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20estimation%20of%20map%20distances%20from%20recombination%20values&journal=Ann.%20Eugen.&volume=12&pages=172-175&publication_year=1943&author=Kosambi%2CDD) 
    
95.  Voorrips, R. E. MapChart: software for the graphical presentation of linkage maps and QTLs. _J. Hered._ **93**, 77–78 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XktlOntrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12011185)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&journal=J.%20Hered.&volume=93&pages=77-78&publication_year=2002&author=Voorrips%2CRE) 
    
96.  Liu, S. et al. Enhanced stripe rust resistance obtained by combining _Yr30_ with a widely dispersed, consistent QTL on chromosome arm 4BL. _Theor. Appl. Genet._ **135**, 351–365 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34665265)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhanced%20stripe%20rust%20resistance%20obtained%20by%20combining%20Yr30%20with%20a%20widely%20dispersed%2C%20consistent%20QTL%20on%20chromosome%20arm%204BL&journal=Theor.%20Appl.%20Genet.&volume=135&pages=351-365&publication_year=2022&author=Liu%2CS) 
    
97.  Wang, N. et al. Inactivation of a wheat protein kinase gene confers broad-spectrum resistance to rust fungi. _Cell_ **185**, 2961–2974 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVWmtrnN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35839760)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inactivation%20of%20a%20wheat%20protein%20kinase%20gene%20confers%20broad-spectrum%20resistance%20to%20rust%20fungi&journal=Cell&volume=185&pages=2961-2974&publication_year=2022&author=Wang%2CN) 
    
98.  Xiang, M. et al. _Yr29_ combined with _QYr.nwafu-4BL.3_ confers durable resistance to stripe rust in wheat cultivar Jing 411. _Theor. Appl. Genet._ **137**, 252 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitlSitbbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39425797)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yr29%20combined%20with%20QYr.nwafu-4BL.3%20confers%20durable%20resistance%20to%20stripe%20rust%20in%20wheat%20cultivar%20Jing%20411&journal=Theor.%20Appl.%20Genet.&volume=137&publication_year=2024&author=Xiang%2CM) 
    
99.  Ma, S. et al. WheatOmics: a platform combining multiple omics data to accelerate functional genomics studies in wheat. _Mol. Plant_ **14**, 1965–1968 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVOksrrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34715393)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WheatOmics%3A%20a%20platform%20combining%20multiple%20omics%20data%20to%20accelerate%20functional%20genomics%20studies%20in%20wheat&journal=Mol.%20Plant&volume=14&pages=1965-1968&publication_year=2021&author=Ma%2CS) 
    
100.  Ayliffe, M. et al. Nonhost resistance of rice to rust pathogens. _Mol. Plant Microbe Interact._ **24**, 1143–1155 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Wjt7rL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21899436)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nonhost%20resistance%20of%20rice%20to%20rust%20pathogens&journal=Mol.%20Plant%20Microbe%20Interact.&volume=24&pages=1143-1155&publication_year=2011&author=Ayliffe%2CM) 
    
101.  CNCB-NGDC Members and Partners. Database resources of the National Genomics Data Center, China National Center for Bioinformation in 2022. _Nucleic Acids Res._ **50**, 27–38 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Database%20resources%20of%20the%20National%20Genomics%20Data%20Center%2C%20China%20National%20Center%20for%20Bioinformation%20in%202022&journal=Nucleic%20Acids%20Res.&volume=50&pages=27-38&publication_year=2022) 
    
102.  Chen, T. et al. The genome sequence archive family: toward explosive data growth and diverse data types. _Genomics Proteomics Bioinformatics_ **19**, 578–583 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34400360)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9039563)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genome%20sequence%20archive%20family%3A%20toward%20explosive%20data%20growth%20and%20diverse%20data%20types&journal=Genomics%20Proteomics%20Bioinformatics&volume=19&pages=578-583&publication_year=2021&author=Chen%2CT) 
    
103.  Wu, J. Genomics-guided landscape unlocks superior alleles and genes for yellow rust resistance in wheat. _Zenodo_ [https://doi.org/10.5281/zenodo.15608140](https://doi.org/10.5281/zenodo.15608140) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02259-2?format=refman&flavour=references)

Acknowledgements
----------------

We take this opportunity to acknowledge Prof. Changfa Wang for his contributions to our group’s resistance breeding. We thank all members of the plant immunity research team in the State Key Laboratory of Crop Stress Resistance and High-Efficiency Production at Northwest A&F University for helpful comments; M. Wang, G. Li, W. Guo, G. Li and C. Jiang for their helpful suggestions and discussions; R. A. McIntosh for language editing and proofreading of the draft manuscript; J. Guo, J. Guo, X. Bai and X. Huang for assistance with genetic transformation; X. Huang, Q. Zhang and X. Zhou for RT–qPCR assays and AQP genotyping; H. Zhao and F. Yuan for confocal experimental assistance; H. Wang for construction of a genomic DNA library; J. Tong for sharing most of the YR resistance QTL information; J. Du, Y. Peng, L. Zhang, W. Zhang, Z. Xu, Y. Yang, B. Cheng and Q. Yao for help in phenotyping; the High-Performance Computing platform of Northwest A&F University for providing computing resources. This work was supported by the National Natural Science Foundation of China (grant nos. 32421004, 31961143019, 32272088, 32201745, 32372138, 32372562, 32302377 and 32472103), the National Key R&D Program of China (grant nos. 2021YFD1401000, 2021YFD1200600 and 2023YFD1200400), Key R&D Program of Qinghai Province (grant no. 2022-NK-125) and Natural Science Basic Research Plan in Shaanxi Province of China (grant nos. 2019JCW-18 and 2020JCW-16).

Author information
------------------

Author notes

1.  These authors contributed equally: Jianhui Wu, Shengwei Ma, Jianqing Niu, Weihang Sun, Haitao Dong, Shusong Zheng.
    

### Authors and Affiliations

1.  State Key Laboratory of Crop Stress Resistance and High-Efficiency Production, College of Agronomy, Northwest A&F University, Yangling, P. R. China
    
    Jianhui Wu, Weihang Sun, Haitao Dong, Jiwen Zhao, Shengjie Liu, Rui Yu, Ying Li, Jinyu Han, Yimin Wang, Tiantian Chen, Chuanliang Zhang, Wenjing Zhang, Bingyou Ding, Lu Chang, Wenbo Xue, Weijun Zheng, Chunlian Li & Dejun Han
    
2.  Yazhouwan National Laboratory, Sanya, P. R. China
    
    Shengwei Ma, Jianqing Niu & Hong-Qing Ling
    
3.  State Key Laboratory of Plant Cell and Chromosome Engineering, Institute of Genetics and Developmental Biology, Innovation Academy for Seed Design, Chinese Academy of Sciences, Beijing, P. R. China
    
    Shengwei Ma, Jianqing Niu, Shusong Zheng & Hong-Qing Ling
    
4.  Hainan Seed industry Laboratory, Sanya, P. R. China
    
    Shengwei Ma, Jianqing Niu & Hong-Qing Ling
    
5.  State Key Laboratory of Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&F University, Yangling, P. R. China
    
    Zhensheng Kang & Qingdong Zeng
    
6.  Yangling Seed Industry Innovation Center, Yangling, P. R. China
    
    Zhensheng Kang
    
7.  College of Advanced Agricultural Sciences, University of Chinese Academy of Sciences, Beijing, P. R. China
    
    Hong-Qing Ling
    

Authors

1.  Jianhui Wu
    
    [View author publications](https://www.nature.com/search?author=Jianhui%20Wu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jianhui%20Wu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jianhui%20Wu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Shengwei Ma
    
    [View author publications](https://www.nature.com/search?author=Shengwei%20Ma)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shengwei%20Ma) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shengwei%20Ma%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Jianqing Niu
    
    [View author publications](https://www.nature.com/search?author=Jianqing%20Niu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jianqing%20Niu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jianqing%20Niu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Weihang Sun
    
    [View author publications](https://www.nature.com/search?author=Weihang%20Sun)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Weihang%20Sun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Weihang%20Sun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Haitao Dong
    
    [View author publications](https://www.nature.com/search?author=Haitao%20Dong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Haitao%20Dong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Haitao%20Dong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Shusong Zheng
    
    [View author publications](https://www.nature.com/search?author=Shusong%20Zheng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shusong%20Zheng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shusong%20Zheng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Jiwen Zhao
    
    [View author publications](https://www.nature.com/search?author=Jiwen%20Zhao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jiwen%20Zhao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jiwen%20Zhao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Shengjie Liu
    
    [View author publications](https://www.nature.com/search?author=Shengjie%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shengjie%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shengjie%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Rui Yu
    
    [View author publications](https://www.nature.com/search?author=Rui%20Yu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rui%20Yu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rui%20Yu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Ying Li
    
    [View author publications](https://www.nature.com/search?author=Ying%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ying%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ying%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Jinyu Han
    
    [View author publications](https://www.nature.com/search?author=Jinyu%20Han)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jinyu%20Han) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jinyu%20Han%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Yimin Wang
    
    [View author publications](https://www.nature.com/search?author=Yimin%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yimin%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yimin%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Tiantian Chen
    
    [View author publications](https://www.nature.com/search?author=Tiantian%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tiantian%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tiantian%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Chuanliang Zhang
    
    [View author publications](https://www.nature.com/search?author=Chuanliang%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chuanliang%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chuanliang%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Wenjing Zhang
    
    [View author publications](https://www.nature.com/search?author=Wenjing%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenjing%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenjing%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Bingyou Ding
    
    [View author publications](https://www.nature.com/search?author=Bingyou%20Ding)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Bingyou%20Ding) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Bingyou%20Ding%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Lu Chang
    
    [View author publications](https://www.nature.com/search?author=Lu%20Chang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lu%20Chang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lu%20Chang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Wenbo Xue
    
    [View author publications](https://www.nature.com/search?author=Wenbo%20Xue)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenbo%20Xue) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenbo%20Xue%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Weijun Zheng
    
    [View author publications](https://www.nature.com/search?author=Weijun%20Zheng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Weijun%20Zheng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Weijun%20Zheng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Chunlian Li
    
    [View author publications](https://www.nature.com/search?author=Chunlian%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chunlian%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chunlian%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Dejun Han
    
    [View author publications](https://www.nature.com/search?author=Dejun%20Han)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Dejun%20Han) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Dejun%20Han%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Zhensheng Kang
    
    [View author publications](https://www.nature.com/search?author=Zhensheng%20Kang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zhensheng%20Kang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zhensheng%20Kang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Qingdong Zeng
    
    [View author publications](https://www.nature.com/search?author=Qingdong%20Zeng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qingdong%20Zeng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qingdong%20Zeng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Hong-Qing Ling
    
    [View author publications](https://www.nature.com/search?author=Hong-Qing%20Ling)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hong-Qing%20Ling) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hong-Qing%20Ling%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

J.W. designed the experiments, managed the project, performed most of the experiments, analyzed the data and wrote the manuscript. D.H., Q.Z., Z.K. and H.-Q.L. conceived and supervised the project and revised the manuscript. S.M. performed most of the genotype data analysis and GWAS results and wrote the manuscript. J.N. analyzed the data and wrote the manuscript. W.S., W. Zhang, Y.L., L.C., Y.W. and J.H. performed most of the experiments related to gene cloning and functional validation. H.D., J.Z., C.Z., T.C. and B.D. participated in data analysis and revised the manuscript. S.Z., R.Y., S.L., W.X., W. Zheng and C.L. contributed to the field trials and conducted some phenotypic assays. All authors read and approved the final manuscript.

### Corresponding authors

Correspondence to [Jianhui Wu](mailto:wujh@nwafu.edu.cn), [Dejun Han](mailto:handj@nwafu.edu.cn), [Zhensheng Kang](mailto:kangzs@nwafu.edu.cn), [Qingdong Zeng](mailto:zengqd@nwafu.edu.cn) or [Hong-Qing Ling](mailto:hqling@genetics.ac.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Marco Maccaferri and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02259-2#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Schematic of experimental design of this study.](https://www.nature.com/articles/s41588-025-02259-2/figures/6)

The entire set of 2,191 wheat accessions was split into two subsets. One subset was allocated for GWAS and also served as the basis for identifying YR resistance candidate genes. The other subset was reserved for validation purposes. Additionally, the complete wheat accession panel was utilized to validate the prevalence of beneficial phenotypes.

### [Extended Data Fig. 2 Phenotyping and analysis of phenotypic data.](https://www.nature.com/articles/s41588-025-02259-2/figures/7)

**a**, Box plot distributions of seedling infection types (IT) for 12 _Pst_ races. Solid horizontal lines show median values. Half violin plots show seedling‐stage resistance density distribution in the wheat panel tested with 12 _Pst_ races. The raw data points overlying the box plots display the frequency of ITs of wheat accessions tested with each _Pst_ race. Accessions were placed in three groups; Resistant (R, IT 0 to 3), intermediate (I, IT 4 to 6), and susceptible (S, IT 7 to 9); frequencies are shown in parentheses. **b**, Regression plot, density distribution curve, and Pearson correlation coefficients (_P_ = 0.01) among different survey indicators or environments. IT and disease severity (DS) are indicated in green and orange, respectively. **c**, Estimates of variance components of DS across multiple environments. Colors in the stacked bar diagram represent different variance components; that is, G, Genetic; E, Environmental; GE: Genotype by Environment. For Fig ‘c’ and ‘d’: YL, Yangling; TS, Tianshui; JY, Jiangyou; CQ, Chongqing; GY, Guiyang. Years 19, 20, 21, and 22 correspond to 2019, 2020, 2021, and 2022, respectively.

### [Extended Data Fig. 3 Functional validation of _Yr5x_ using VIGS, transgenic complementation and expression pattern analysis.](https://www.nature.com/articles/s41588-025-02259-2/figures/8)

**a**, Relative expression of _Yr5x_ in silenced plants assessed by RT‐qPCR using _TaActin_ as the internal control. Results are means ± SD from three biological replicates and three technical replicates were made for each leaf sample. **b**, Structure of _pUbi:Yr5x_ used for transgenic assay. The _pUbi:Yr5x_ construct contained the full length of _Yr5x_ sequence. LB, left border; RB, right border. **c**, Reactions of 9 T1 transgenic lines. Positive T1 transgenic lines (+) of _Yr5x_ showed resistance to the _Pst_ race TSA‐V5. Fielder was used as susceptible control. PCR results as positive (+) or negative (−) for DNA amplification of the _Yr5x_ gene. Scale bars, 1 cm. **d**, The relative expression levels of _Yr5x_ in transgenic wheat lines. Leaves of two‐leaf seedlings were collected and _TaActin_ was used as an endogenous control. Expression values are means ± SD from three biological replicates and three technical replicates. There was no expression in non‐ transgenic plants. Error bars are standard errors of the mean. **e**, Expression pattern of _Yr5x_ in Xinong 3517 after inoculation with _Pst_ race TSA‐V5. Leaves were collected at 0, 2, 4, 5, 7, 10 and 14 dpi. Transcript levels were examined by RT‒qPCR with _TaActin_ as an endogenous control. Values and error bars of transcript levels represent means and standard errors of three biological replicates, respectively. For Fig. ‘a’, ‘d’, ‘e’, statistical significance was determined by a two‐sided _t_‐tests. Source data are provided as a Source Data file.

[Source data](https://www.nature.com/articles/s41588-025-02259-2#MOESM10)

### [Extended Data Fig. 4 Characterization of _Yr6._](https://www.nature.com/articles/s41588-025-02259-2/figures/9)

**a**, BSMV‐mediated silencing of _Yr6_ reduced resistance to yellow rust. Mild chlorotic mosaic symptoms were observed on leaves inoculated with BSMV:_Yr6_‐as1/2 at 9 dpi. Wheat leaves inoculated with BSMV:_TaPDS_ and FES buffer were used as positive controls and leaves inoculated with BSMV:γ were used as negative controls. Images show leaves of _Yr6_‐silenced plants 14 dpi with avirulent _Pst_ race V26/GS. Scale bar, 1 cm. **b**, Relative expression of _Yr6_ in silenced plants assessed by RT‐ qPCR using _TaActin_ as the internal control. Results are means ± SD from three biological replicates and three technical replicates. **c**, Expression analysis of _Yr6_. Expression patterns of _Yr6_ in seedling leaves of wheat cv. Aikang 58 (AK58) at different days post infection. _Yr6_ expression was induced by infection and reached its peak on the seventh day. Normalized values of _Yr6_ expression relative to _TaActin_ are shown as means ± SD from three biological replicates. **d**, Subcellular localization of YR6. Confocal microscope image of a _N. benthamiana_ leaf showing green fluorescence signals of YR6 protein overlapping red fluorescence signals of mCherry protein in the cytoplasm. Scale bar, 15 μm. For Fig. ‘a’, ‘b’, statistical significance was determined by a two‐sided _t_‐tests. Source data are provided as a Source Data file.

[Source data](https://www.nature.com/articles/s41588-025-02259-2#MOESM11)

### [Extended Data Fig. 5 Haplotype analysis of _YR6._](https://www.nature.com/articles/s41588-025-02259-2/figures/10)

**a**, Haplotype network of _YR6_ in cultivars (green) and landraces (orange). Diameters of the colored circles are proportional to the log2 number of accessions with the indicated haplotype. Numbers on the branches indicate number of base differences. **b**, Boxplots of IT scores with _Pst_ race V26/GS for _YR6_ haplotypes. The haplotype conferring resistance is colored green. **c**, Boxplots of IT scores for _Bgt_ race Bgt21‐2 for different haplotypes of _YR6_. The haplotype conferring resistance is colored green. For Fig. ‘b’, and ‘c’, the significance of differences (Supplementary Table [16](https://www.nature.com/articles/s41588-025-02259-2#MOESM4)) between groups was assessed by two‐sided _t_‐tests.

### [Extended Data Fig. 6 Transgenic validation of _TaEDR2‐B._](https://www.nature.com/articles/s41588-025-02259-2/figures/11)

**a**, Structure of _pUbi:TaEDR2‐B_ used for transgenesis. The _pUbi:TaEDR2‐B_ construct containing the full length CDS of _TaEDR2‐B_. LB, left border; RB, right border. **b**, Yellow rust symptoms on Zhengmai 7698 (susceptible control), Flanders (resistant control) and 11 T1 transgenic lines in seedlings (upper panel) and adult plants (lower panel). PCR results as positive (+) or negative (−) for DNA amplification (upper) and cDNA amplification (RNA expression, lower) of _TaEDR2‐B_. Positive T1 transgenic lines (+) of _TaEDR2‐B_ along with Flanders showing high susceptibility in seedings but moderate resistance to mixed _Pst_ races in adult plants. Scale bars, 1 cm.

### [Extended Data Fig. 7 Characterization of _TaEDR2‐B._](https://www.nature.com/articles/s41588-025-02259-2/figures/12)

**a**, Expression analysis of _TaEDR2‐B_. Expression patterns of _TaEDR2‐B_ in flag leaves of cultivar Flanders at different times post inoculation (dpi). _TaEDR2‐B_ expression was induced by _Pst_ infection and reached a peak on the fifth day. Normalized values of _TaEDR2‐B_ expression relative to _TaActin_ are shown as means ± SD from three biological replicates. Statistical significance was determined using unpaired two‐tailed _t‐_tests. Source data are provided as a Source Data file. **b**, Subcellular localization of TaEDR2‐B. Confocal microscope images of _N. benthamiana_ leaves showing green fluorescent TaEDR2‐B protein is diffusely located in the cytoplasm and partially overlapping the red fluorescence signal of mCherry protein. Scale bars, 15 μm.

[Source data](https://www.nature.com/articles/s41588-025-02259-2#MOESM12)

### [Extended Data Fig. 8 Evaluation of agronomic traits of _TaEDR2‐B_OE lines in the field.](https://www.nature.com/articles/s41588-025-02259-2/figures/13)

**a**, Comparison of agronomic traits in ZM7698(WT) and _TaEDR2‐B_OE lines: plant height, spike length, spikelet number per spike, kernel number per spike, days to heading, and days to anthesis. **b**,**c**, Representative spikes and seeds of ZM7698 and _TaEDR2‐B_OE lines. **d**, Thousand‐grain weights of ZM7698(WT) and _TaEDR2‐B_OE lines grown in rust‐free and disease nurseries. Scale bars: in (m), 2 cm; in (n), 5 mm; n, number of plants. For Fig. ‘a’, ‘d’, statistical significance was determined by a two‐sided _t_‐tests. Source data are provided as a Source Data file.

[Source data](https://www.nature.com/articles/s41588-025-02259-2#MOESM13)

### [Extended Data Fig. 9 Haplotype analysis of _TaEDR2‐B._](https://www.nature.com/articles/s41588-025-02259-2/figures/14)

**a**, Haplotype analysis of _TaEDR2‐B_ showing DNA polymorphisms in the gene region. Nucleotides identical to the Chinese Spring reference are shown in orange and others are in green. H8 (red) is the haplotype conferring resistance. **b**, Haplotype network of _TaEDR2‐B_. Diameter of the colored circles is proportional to the log2 number of accessions with indicated haplotype. Numbers on the branches indicate the number of base differences. Landraces and cultivars are indicated by orange and green, respectively. **c**, Boxplots of yellow rust (YR) severities for different _TaEDR2‐B_ haplotypes. **d**, Boxplots of leaf rust (LR) severities for different _TaEDR2‐B_ haplotypes, haplotypes conferring resistance are shown in green. For Fig. ‘c’, ‘d’, the significance of differences was assessed by two‐sided _t_‐tests.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM1_ESM.pdf)

Supplementary Methods and Figs. 1–9.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM4_ESM.xlsx)

Table 1 Passport information of all accessions used in this study. Table 2 Information for 349 accessions sequenced in this study. Table 3 Summary of genetic diversity in chromosomes and resistance (R) genes of 2,191 wheat accessions. Table 4 Numbers of variants with different effects. Table 5 Experimental design for 12 field experiments to test YR or leaf rust (LR) responses of 1,629 naturally or artificially inoculated accessions. Table 6 General statistics, estimated variance components and heritability yellow rust infection type (IT) and disease severity (DS) in multiple environments. Table 7 Summary of significant SNP‐trait associations and resistance allele of frequencies. Table 8 Wheat YR resistance QTL or genes from QTL mapping studies using linkage analysis in biparental or multiparent populations and GWAS. Table 9 Candidate genes within the identified QTL based on IWGSC RefSeq v.2.1. Table 10 Fine mapping of _Yr5b_ and _Yr5x_. Table 11 Candidate genes within the QTL containing _YR5_ in IWGSC RefSeq v.2.1 genome. Table 12 Genes in the list showing DNA variations among Avocet S, Baomai 6 and Xinong 3517 as identified by genome resequencing. Table 13 Map‐based cloning of _Yr6_. Table 14 Candidate genes within the QTL containing _YR6_ in IWGSC RefSeq v.2.1 and Aikang 58 (AK58) genomes, respectively. Table 15 Genes in the list showing DNA variations between AvS and Aikang 58 as identified by genome resequencing. Table 16 Polymorphic residues encoded by _Yr6/Pm5_ alleles present in various wheat cultivars. Table 17 Genetic nonsynonymous variations of _YR6_ among the 1,449 accessions and their associations with YR and powdery mildew (PM) responses. Table 18 Fine mapping of _YrKB_. Table 19 Candidate genes within the regions of _YrKB_ in IWGSC RefSeq v.2.1 genome. Table 20 Genes in the list showing DNA variations between AvS and Flanders as identified by genome resequencing. Table 21 Genetic variations of _YrKB_ among 2,183 accessions and their associations with YR and leaf rust (LR) responses. Table 22 _TaSTART_ homologous groups in _Triticum aestivum_, _Oryza sativa_, _Zea mays_ and _Arabidopsis thaliana_. Table 23 The information of Jing 411 mutants. Table 24 Primers used in this study. Table 25 Phenotypic and genotypic data for 576 wheat accessions used to validate molecular markers for _Yr5x_, _Yr6_ and _TaEDR2‐B_. Table 26 Detailed information of LD blocks shown in Supplementary Fig. 2.

### [Supplementary Data](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM5_ESM.pdf)

Gene sequence.

### [Supplementary Data](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM6_ESM.pdf)

Gene sequence.

### [Supplementary Data](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM7_ESM.pdf)

Gene sequence.

Source data
-----------

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM8_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM9_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM12_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02259-2/MediaObjects/41588_2025_2259_MOESM13_ESM.xlsx)

Statistical source data.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Genomics-driven%20discovery%20of%20superior%20alleles%20and%20genes%20for%20yellow%20rust%20resistance%20in%20wheat&author=Jianhui%20Wu%20et%20al&contentID=10.1038%2Fs41588-025-02259-2&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-07-22&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02259-2)

### Cite this article

Wu, J., Ma, S., Niu, J. _et al._ Genomics-driven discovery of superior alleles and genes for yellow rust resistance in wheat. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02259-2

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02259-2?format=refman&flavour=citation)

-   Received: 12 April 2024
    
-   Accepted: 06 June 2025
    
-   Published: 22 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02259-2
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
