> id: https://www.nature.com/articles/s41588-025-02263-6

> link: https://www.nature.com/articles/s41588-025-02263-6

> title: G-quadruplexes are promoter elements controlling nucleosome exclusion and RNA polymerase II pausing

# G-quadruplexes are promoter elements controlling nucleosome exclusion and RNA polymerase II pausing
_Published on Tue, 22 Jul 2025 00:00:00 GMT_

Abstract
--------

Despite their central role in transcription, it has been difficult to define universal sequences associated with eukaryotic promoters. Within the chromatin context, recruitment of transcriptional machinery requires promoter opening, but how DNA elements contribute to this process is unclear. Here we show that G-quadruplex (G4) secondary DNA structures are highly enriched at mammalian promoters. G4s are located at the deepest point of nucleosome exclusion at promoters and correlate with maximum promoter activity. We found that experimental G4s exclude nucleosomes in vivo and in vitro while favouring strong positioning. At model promoters, impairing G4s affected both transcriptional activity and chromatin opening. G4 destabilization also resulted in an inactive promoter state and affected the transition to effective RNA production. Finally, G4 stabilization resulted in global reduction of proximal promoter pausing. Altogether, our data introduce G4s as bona fide promoter elements allowing nucleosome exclusion and facilitating pause–release by RNA polymerase II.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 12 print issues and online access

$209.00 per year

only $17.42 per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02263-6?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: pG4s are highly enriched at promoters and correlate with maximum mammalian promoter activity.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_Fig1_HTML.png)

**Fig. 2: G4s promote nucleosome exclusion at active and inactive promoters in vivo and in vitro.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_Fig2_HTML.png)

**Fig. 3: pG4s organize nucleosomes at IGRs.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_Fig3_HTML.png)

**Fig. 4: pG4s and experimental G4s hallmark nucleosome exclusion at CGIs.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_Fig4_HTML.png)

**Fig. 5: G4 mutations impair promoter activity in single cells.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_Fig5_HTML.png)

**Fig. 6: G4 mutations at _Eef1a1_ model promoters decrease transcription and increase nucleosome density at NDRs while increasing promoter OFF times in single cells.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_Fig6_HTML.png)

**Fig. 7: Stabilization of G4 by ligand results in global pause–release by Pol II.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_Fig7_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-023-01437-4/MediaObjects/41588_2023_1437_Fig1_HTML.png)

### [G4access identifies G-quadruplexes and their associations with open chromatin and imprinting control regions](https://www.nature.com/articles/s41588-023-01437-4?fromPaywallRec=true)

Article 03 July 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-023-05015-6/MediaObjects/42003_2023_5015_Fig1_HTML.png)

### [Integrative genomic analyses of promoter G-quadruplexes reveal their selective constraint and association with gene activation](https://www.nature.com/articles/s42003-023-05015-6?fromPaywallRec=true)

Article Open access 10 June 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-025-04791-x/MediaObjects/41598_2025_4791_Fig1_HTML.png)

### [The effect of prolonged G-quadruplex stabilization on the functions of human cells](https://www.nature.com/articles/s41598-025-04791-x?fromPaywallRec=true)

Article Open access 04 June 2025

Data availability
-----------------

All datasets used in this study including published and original experiments are described in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02263-6#MOESM4). Original datasets produced in this study correspond to MNase-seq, ChIP–seq, DNase-seq, TT-seq, chrRNA-seq and G4access, which are recorded in the GEO database under [GSE159276](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159276). G4 mapping and predictions are available under [GSE187007](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE187007). [Source data](https://www.nature.com/articles/s41588-025-02263-6#Sec54) are provided with this paper.

Code availability
-----------------

Software and codes are all publicly available and have been previously described[8](https://www.nature.com/articles/s41588-025-02263-6#ref-CR8 "Fenouil, R. et al. CpG islands and GC content dictate nucleosome depletion in a transcription-independent manner at mammalian promoters. Genome Res. 22, 2399–2408 (2012)."),[70](https://www.nature.com/articles/s41588-025-02263-6#ref-CR70 "Fenouil, R. et al. Pasha: a versatile R package for piling chromatin HTS data. Bioinformatics 32, 2528–2530 (2016)."),[73](https://www.nature.com/articles/s41588-025-02263-6#ref-CR73 "Descostes, N. et al. Tyrosine phosphorylation of RNA polymerase II CTD is associated with antisense promoter transcription and active enhancers in mammalian cells. eLife 3, e02105 (2014)."). Codes used for this study have been deposited in the linked Zenodo repository: [https://zenodo.org/record/15124924](https://zenodo.org/record/15124924) (ref. [81](https://www.nature.com/articles/s41588-025-02263-6#ref-CR81 "Esnault, C. & Andrau, J. C. Scripts and codes for G-quadruplexes are promoter elements controlling nucleosome exclusion and RNA polymerase II pausing. Zenodo 
https://doi.org/10.5281/zenodo.15124924
(2025).")).

References
----------

1.  Jacob, F., Ullman, A. & Monod, J. The promotor, a genetic element necessary to the expression of an operon. _C. R. Hebd. Seances Acad. Sci._ **258**, 3125–3128 (1964).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaF2cXktFeisLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14143651)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20promotor%2C%20a%20genetic%20element%20necessary%20to%20the%20expression%20of%20an%20operon&journal=C.%20R.%20Hebd.%20Seances%20Acad.%20Sci.&volume=258&pages=3125-3128&publication_year=1964&author=Jacob%2CF&author=Ullman%2CA&author=Monod%2CJ) 
    
2.  Pribnow, D. Nucleotide sequence of an RNA polymerase binding site at an early T7 promoter. _Proc. Natl Acad. Sci. USA_ **72**, 784–788 (1975).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE2MXhs1Kjurk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1093168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC432404)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nucleotide%20sequence%20of%20an%20RNA%20polymerase%20binding%20site%20at%20an%20early%20T7%20promoter&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=72&pages=784-788&publication_year=1975&author=Pribnow%2CD) 
    
3.  Rosenberg, M. & Court, D. Regulatory sequences involved in the promotion and termination of RNA transcription. _Annu. Rev. Genet._ **13**, 319–353 (1979).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3cXnt1eitg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=94251)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulatory%20sequences%20involved%20in%20the%20promotion%20and%20termination%20of%20RNA%20transcription&journal=Annu.%20Rev.%20Genet.&volume=13&pages=319-353&publication_year=1979&author=Rosenberg%2CM&author=Court%2CD) 
    
4.  Butler, J. E. & Kadonaga, J. T. The RNA polymerase II core promoter: a key component in the regulation of gene expression. _Genes Dev._ **16**, 2583–2592 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XotFelu74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12381658)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20RNA%20polymerase%20II%20core%20promoter%3A%20a%20key%20component%20in%20the%20regulation%20of%20gene%20expression&journal=Genes%20Dev.&volume=16&pages=2583-2592&publication_year=2002&author=Butler%2CJE&author=Kadonaga%2CJT) 
    
5.  Jiang, C. & Pugh, B. F. Nucleosome positioning and gene regulation: advances through genomics. _Nat. Rev. Genet._ **10**, 161–172 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhvFGlu7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19204718)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4860946)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nucleosome%20positioning%20and%20gene%20regulation%3A%20advances%20through%20genomics&journal=Nat.%20Rev.%20Genet.&volume=10&pages=161-172&publication_year=2009&author=Jiang%2CC&author=Pugh%2CBF) 
    
6.  Radman-Livaja, M. & Rando, O. J. Nucleosome positioning: how is it established, and why does it matter? _Dev. Biol._ **339**, 258–266 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXis1ajtbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19527704)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nucleosome%20positioning%3A%20how%20is%20it%20established%2C%20and%20why%20does%20it%20matter%3F&journal=Dev.%20Biol.&volume=339&pages=258-266&publication_year=2010&author=Radman-Livaja%2CM&author=Rando%2COJ) 
    
7.  Deaton, A. M. & Bird, A. CpG islands and the regulation of transcription. _Genes Dev._ **25**, 1010–1022 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmslOgsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21576262)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3093116)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CpG%20islands%20and%20the%20regulation%20of%20transcription&journal=Genes%20Dev.&volume=25&pages=1010-1022&publication_year=2011&author=Deaton%2CAM&author=Bird%2CA) 
    
8.  Fenouil, R. et al. CpG islands and GC content dictate nucleosome depletion in a transcription-independent manner at mammalian promoters. _Genome Res._ **22**, 2399–2408 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVWgsbfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23100115)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3514669)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CpG%20islands%20and%20GC%20content%20dictate%20nucleosome%20depletion%20in%20a%20transcription-independent%20manner%20at%20mammalian%20promoters&journal=Genome%20Res.&volume=22&pages=2399-2408&publication_year=2012&author=Fenouil%2CR) 
    
9.  Bansal, M., Kumar, A. & Yella, V. R. Role of DNA sequence based structural features of promoters in transcription initiation and gene expression. _Curr. Opin. Struct. Biol._ **25**, 77–85 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXovFCju7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24503515)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20DNA%20sequence%20based%20structural%20features%20of%20promoters%20in%20transcription%20initiation%20and%20gene%20expression&journal=Curr.%20Opin.%20Struct.%20Biol.&volume=25&pages=77-85&publication_year=2014&author=Bansal%2CM&author=Kumar%2CA&author=Yella%2CVR) 
    
10.  Agarwal, T., Roy, S., Kumar, S., Chakraborty, T. K. & Maiti, S. In the sense of transcription regulation by G-quadruplexes: asymmetric effects in sense and antisense strands. _Biochemistry_ **53**, 3711–3718 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXotlKqtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24850370)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In%20the%20sense%20of%20transcription%20regulation%20by%20G-quadruplexes%3A%20asymmetric%20effects%20in%20sense%20and%20antisense%20strands&journal=Biochemistry&volume=53&pages=3711-3718&publication_year=2014&author=Agarwal%2CT&author=Roy%2CS&author=Kumar%2CS&author=Chakraborty%2CTK&author=Maiti%2CS) 
    
11.  Bochman, M. L., Paeschke, K. & Zakian, V. A. DNA secondary structures: stability and function of G-quadruplex structures. _Nat. Rev. Genet._ **13**, 770–780 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVCisb%2FE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23032257)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3725559)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20secondary%20structures%3A%20stability%20and%20function%20of%20G-quadruplex%20structures&journal=Nat.%20Rev.%20Genet.&volume=13&pages=770-780&publication_year=2012&author=Bochman%2CML&author=Paeschke%2CK&author=Zakian%2CVA) 
    
12.  David, A. P. et al. G-quadruplexes as novel _cis_\-elements controlling transcription during embryonic development. _Nucleic Acids Res._ **44**, 4163–4173 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhslCqurvJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26773060)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4872077)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=G-quadruplexes%20as%20novel%20cis-elements%20controlling%20transcription%20during%20embryonic%20development&journal=Nucleic%20Acids%20Res.&volume=44&pages=4163-4173&publication_year=2016&author=David%2CAP) 
    
13.  Smestad, J. A. & Maher, L. J. 3rd Relationships between putative G-quadruplex-forming sequences, RecQ helicases, and transcription. _BMC Med. Genet._ **16**, 91 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26449372)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4599794)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Relationships%20between%20putative%20G-quadruplex-forming%20sequences%2C%20RecQ%20helicases%2C%20and%20transcription&journal=BMC%20Med.%20Genet.&volume=16&publication_year=2015&author=Smestad%2CJA&author=Maher%2CLJ) 
    
14.  Hansel-Hertsch, R. et al. G-quadruplex structures mark human regulatory chromatin. _Nat. Genet._ **48**, 1267–1272 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFSgurvP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27618450)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=G-quadruplex%20structures%20mark%20human%20regulatory%20chromatin&journal=Nat.%20Genet.&volume=48&pages=1267-1272&publication_year=2016&author=Hansel-Hertsch%2CR) 
    
15.  Xia, Y. et al. Transmission of dynamic supercoiling in linear and multi-way branched DNAs and its regulation revealed by a fluorescent G-quadruplex torsion sensor. _Nucleic Acids Res._ **46**, 7418–7424 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosFanuro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29982790)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6101514)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transmission%20of%20dynamic%20supercoiling%20in%20linear%20and%20multi-way%20branched%20DNAs%20and%20its%20regulation%20revealed%20by%20a%20fluorescent%20G-quadruplex%20torsion%20sensor&journal=Nucleic%20Acids%20Res.&volume=46&pages=7418-7424&publication_year=2018&author=Xia%2CY) 
    
16.  Esain-Garcia, I. et al. G-quadruplex DNA structure is a positive regulator of _MYC_ transcription. _Proc. Natl Acad. Sci. USA_ **121**, e2320240121 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXlvFOgsb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38315865)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10873556)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=G-quadruplex%20DNA%20structure%20is%20a%20positive%20regulator%20of%20MYC%20transcription&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=121&publication_year=2024&author=Esain-Garcia%2CI) 
    
17.  Buratowski, S., Hahn, S., Guarente, L. & Sharp, P. A. Five intermediate complexes in transcription initiation by RNA polymerase II. _Cell_ **56**, 549–561 (1989).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1MXhs1yhtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2917366)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Five%20intermediate%20complexes%20in%20transcription%20initiation%20by%20RNA%20polymerase%20II&journal=Cell&volume=56&pages=549-561&publication_year=1989&author=Buratowski%2CS&author=Hahn%2CS&author=Guarente%2CL&author=Sharp%2CPA) 
    
18.  Haberle, V. & Stark, A. Eukaryotic core promoters and the functional basis of transcription initiation. _Nat. Rev. Mol. Cell Biol._ **19**, 621–637 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqtrjO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29946135)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6205604)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Eukaryotic%20core%20promoters%20and%20the%20functional%20basis%20of%20transcription%20initiation&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=19&pages=621-637&publication_year=2018&author=Haberle%2CV&author=Stark%2CA) 
    
19.  Vo Ngoc, L., Wang, Y. L., Kassavetis, G. A. & Kadonaga, J. T. The punctilious RNA polymerase II core promoter. _Genes Dev._ **31**, 1289–1301 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28808065)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5580651)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20punctilious%20RNA%20polymerase%20II%20core%20promoter&journal=Genes%20Dev.&volume=31&pages=1289-1301&publication_year=2017&author=Vo%20Ngoc%2CL&author=Wang%2CYL&author=Kassavetis%2CGA&author=Kadonaga%2CJT) 
    
20.  Bedrat, A., Lacroix, L. & Mergny, J. L. Re-evaluation of G-quadruplex propensity with G4Hunter. _Nucleic Acids Res._ **44**, 1746–1759 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26792894)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4770238)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Re-evaluation%20of%20G-quadruplex%20propensity%20with%20G4Hunter&journal=Nucleic%20Acids%20Res.&volume=44&pages=1746-1759&publication_year=2016&author=Bedrat%2CA&author=Lacroix%2CL&author=Mergny%2CJL) 
    
21.  Bagchi, D. N. & Iyer, V. R. The determinants of directionality in transcriptional initiation. _Trends Genet._ **32**, 322–333 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XltVKqu7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27066865)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4874897)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20determinants%20of%20directionality%20in%20transcriptional%20initiation&journal=Trends%20Genet.&volume=32&pages=322-333&publication_year=2016&author=Bagchi%2CDN&author=Iyer%2CVR) 
    
22.  Core, L. J., Waterfall, J. J. & Lis, J. T. Nascent RNA sequencing reveals widespread pausing and divergent initiation at human promoters. _Science_ **322**, 1845–1848 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsFSmtrnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19056941)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2833333)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nascent%20RNA%20sequencing%20reveals%20widespread%20pausing%20and%20divergent%20initiation%20at%20human%20promoters&journal=Science&volume=322&pages=1845-1848&publication_year=2008&author=Core%2CLJ&author=Waterfall%2CJJ&author=Lis%2CJT) 
    
23.  Seila, A. C. et al. Divergent transcription from active promoters. _Science_ **322**, 1849–1851 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsFSmtrbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19056940)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2692996)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Divergent%20transcription%20from%20active%20promoters&journal=Science&volume=322&pages=1849-1851&publication_year=2008&author=Seila%2CAC) 
    
24.  Esnault, C. et al. G4access identifies G-quadruplexes and their associations with open chromatin and imprinting control regions. _Nat. Genet._ **55**, 1359–1369 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlKqu7bN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37400615)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=G4access%20identifies%20G-quadruplexes%20and%20their%20associations%20with%20open%20chromatin%20and%20imprinting%20control%20regions&journal=Nat.%20Genet.&volume=55&pages=1359-1369&publication_year=2023&author=Esnault%2CC) 
    
25.  Mao, S. Q. et al. DNA G-quadruplex structures mold the DNA methylome. _Nat. Struct. Mol. Biol._ **25**, 951–957 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVeqs7fP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30275516)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6173298)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20G-quadruplex%20structures%20mold%20the%20DNA%20methylome&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=25&pages=951-957&publication_year=2018&author=Mao%2CSQ) 
    
26.  Chambers, V. S. et al. High-throughput sequencing of DNA G-quadruplex structures in the human genome. _Nat. Biotechnol._ **33**, 877–881 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2murnP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26192317)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-throughput%20sequencing%20of%20DNA%20G-quadruplex%20structures%20in%20the%20human%20genome&journal=Nat.%20Biotechnol.&volume=33&pages=877-881&publication_year=2015&author=Chambers%2CVS) 
    
27.  Kouzine, F. et al. Global regulation of promoter melting in naive lymphocytes. _Cell_ **153**, 988–999 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXotlGgsL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23706737)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684982)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20regulation%20of%20promoter%20melting%20in%20naive%20lymphocytes&journal=Cell&volume=153&pages=988-999&publication_year=2013&author=Kouzine%2CF) 
    
28.  van Arensbergen, J. et al. Genome-wide mapping of autonomous promoter activity in human cells. _Nat. Biotechnol._ **35**, 145–153 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28024146)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20mapping%20of%20autonomous%20promoter%20activity%20in%20human%20cells&journal=Nat.%20Biotechnol.&volume=35&pages=145-153&publication_year=2017&author=Arensbergen%2CJ) 
    
29.  Pugh, B. & Venters, B. Genomic organization of human transcription initiation complexes. _PLoS ONE_ **11**, e0149339 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26866362)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4750860)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20organization%20of%20human%20transcription%20initiation%20complexes&journal=PLoS%20ONE&volume=11&publication_year=2016&author=Pugh%2CB&author=Venters%2CB) 
    
30.  Core, L. J. et al. Analysis of nascent RNA identifies a unified architecture of initiation regions at mammalian promoters and enhancers. _Nat. Genet._ **46**, 1311–1320 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFKmtbbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25383968)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4254663)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20nascent%20RNA%20identifies%20a%20unified%20architecture%20of%20initiation%20regions%20at%20mammalian%20promoters%20and%20enhancers&journal=Nat.%20Genet.&volume=46&pages=1311-1320&publication_year=2014&author=Core%2CLJ) 
    
31.  Andersson, R. & Sandelin, A. Determinants of enhancer and promoter activities of regulatory elements. _Nat. Rev. Genet._ **21**, 71–87 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFGmtr7L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31605096)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Determinants%20of%20enhancer%20and%20promoter%20activities%20of%20regulatory%20elements&journal=Nat.%20Rev.%20Genet.&volume=21&pages=71-87&publication_year=2020&author=Andersson%2CR&author=Sandelin%2CA) 
    
32.  Esnault, C. et al. Rho-actin signaling to the MRTF coactivators dominates the immediate transcriptional response to serum in fibroblasts. _Genes Dev._ **28**, 943–958 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXot12is7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24732378)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4018493)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rho-actin%20signaling%20to%20the%20MRTF%20coactivators%20dominates%20the%20immediate%20transcriptional%20response%20to%20serum%20in%20fibroblasts&journal=Genes%20Dev.&volume=28&pages=943-958&publication_year=2014&author=Esnault%2CC) 
    
33.  Valouev, A. et al. Determinants of nucleosome organization in primary human cells. _Nature_ **474**, 516–520 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmsFyqt7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21602827)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3212987)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Determinants%20of%20nucleosome%20organization%20in%20primary%20human%20cells&journal=Nature&volume=474&pages=516-520&publication_year=2011&author=Valouev%2CA) 
    
34.  Guedin, A., Gros, J., Alberti, P. & Mergny, J. L. How long is too long? Effects of loop size on G-quadruplex stability. _Nucleic Acids Res._ **38**, 7858–7868 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFamtb7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20660477)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2995061)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=How%20long%20is%20too%20long%3F%20Effects%20of%20loop%20size%20on%20G-quadruplex%20stability&journal=Nucleic%20Acids%20Res.&volume=38&pages=7858-7868&publication_year=2010&author=Guedin%2CA&author=Gros%2CJ&author=Alberti%2CP&author=Mergny%2CJL) 
    
35.  Sen, D. & Gilbert, W. Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis. _Nature_ **334**, 364–366 (1988).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1cXls1GrsLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3393228)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Formation%20of%20parallel%20four-stranded%20complexes%20by%20guanine-rich%20motifs%20in%20DNA%20and%20its%20implications%20for%20meiosis&journal=Nature&volume=334&pages=364-366&publication_year=1988&author=Sen%2CD&author=Gilbert%2CW) 
    
36.  Umeyama, T. & Ito, T. DMS-seq for in vivo genome-wide mapping of protein–DNA interactions and nucleosome centers. _Cell Rep._ **21**, 289–300 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1SiurzP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28978481)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DMS-seq%20for%20in%20vivo%20genome-wide%20mapping%20of%20protein%E2%80%93DNA%20interactions%20and%20nucleosome%20centers&journal=Cell%20Rep.&volume=21&pages=289-300&publication_year=2017&author=Umeyama%2CT&author=Ito%2CT) 
    
37.  Barozzi, I. et al. Coregulation of transcription factor binding and nucleosome occupancy through DNA features of mammalian enhancers. _Mol. Cell_ **54**, 844–857 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnslGms7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24813947)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4048654)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coregulation%20of%20transcription%20factor%20binding%20and%20nucleosome%20occupancy%20through%20DNA%20features%20of%20mammalian%20enhancers&journal=Mol.%20Cell&volume=54&pages=844-857&publication_year=2014&author=Barozzi%2CI) 
    
38.  Fu, Y., Sinha, M., Peterson, C. L. & Weng, Z. The insulator binding protein CTCF positions 20 nucleosomes around its binding sites across the human genome. _PLoS Genet._ **4**, e1000138 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18654629)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2453330)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20insulator%20binding%20protein%20CTCF%20positions%2020%20nucleosomes%20around%20its%20binding%20sites%20across%20the%20human%20genome&journal=PLoS%20Genet.&volume=4&publication_year=2008&author=Fu%2CY&author=Sinha%2CM&author=Peterson%2CCL&author=Weng%2CZ) 
    
39.  Kouzine, F. et al. Permanganate/S1 nuclease footprinting reveals non-B DNA structures with regulatory potential across a mammalian genome. _Cell Syst._ **4**, 344–356 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvFWkuro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28237796)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7432990)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Permanganate%2FS1%20nuclease%20footprinting%20reveals%20non-B%20DNA%20structures%20with%20regulatory%20potential%20across%20a%20mammalian%20genome&journal=Cell%20Syst.&volume=4&pages=344-356&publication_year=2017&author=Kouzine%2CF) 
    
40.  Shen, J. et al. Promoter G-quadruplex folding precedes transcription and is controlled by chromatin. _Genome Biol._ **22**, 143 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33962653)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8103603)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Promoter%20G-quadruplex%20folding%20precedes%20transcription%20and%20is%20controlled%20by%20chromatin&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Shen%2CJ) 
    
41.  Tantale, K. et al. A single-molecule view of transcription reveals convoys of RNA polymerases and multi-scale bursting. _Nat. Commun._ **7**, 12248 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht12nsL3E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27461529)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4974459)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single-molecule%20view%20of%20transcription%20reveals%20convoys%20of%20RNA%20polymerases%20and%20multi-scale%20bursting&journal=Nat.%20Commun.&volume=7&publication_year=2016&author=Tantale%2CK) 
    
42.  Lyu, J., Shao, R., Kwong Yung, P. Y. & Elsässer, S. J. Genome-wide mapping of G-quadruplex structures with CUT&Tag. _Nucleic Acids Res._ **50**, e13 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XnsVOmsrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34792172)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20mapping%20of%20G-quadruplex%20structures%20with%20CUT%26Tag&journal=Nucleic%20Acids%20Res.&volume=50&publication_year=2022&author=Lyu%2CJ&author=Shao%2CR&author=Kwong%20Yung%2CPY&author=Els%C3%A4sser%2CSJ) 
    
43.  Tantale, K. et al. Stochastic pausing at latent HIV-1 promoters generates transcriptional bursting. _Nat. Commun._ **12**, 4503 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsleqsrrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34301927)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8302722)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stochastic%20pausing%20at%20latent%20HIV-1%20promoters%20generates%20transcriptional%20bursting&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Tantale%2CK) 
    
44.  Rodriguez, R. et al. A novel small molecule that alters shelterin integrity and triggers a DNA-damage response at telomeres. _J. Am. Chem. Soc._ **130**, 15758–15759 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlWqurbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18975896)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2746963)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20novel%20small%20molecule%20that%20alters%20shelterin%20integrity%20and%20triggers%20a%20DNA-damage%20response%20at%20telomeres&journal=J.%20Am.%20Chem.%20Soc.&volume=130&pages=15758-15759&publication_year=2008&author=Rodriguez%2CR) 
    
45.  Olivieri, M. et al. A genetic map of the response to DNA damage in human cells. _Cell_ **182**, 481–496 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlKkt7zE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32649862)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7384976)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genetic%20map%20of%20the%20response%20to%20DNA%20damage%20in%20human%20cells&journal=Cell&volume=182&pages=481-496&publication_year=2020&author=Olivieri%2CM) 
    
46.  Jonkers, I., Kwak, H. & Lis, J. T. Genome-wide dynamics of Pol II elongation and its interplay with promoter proximal pausing, chromatin, and exons. _eLife_ **3**, e02407 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24843027)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001325)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20dynamics%20of%20Pol%20II%20elongation%20and%20its%20interplay%20with%20promoter%20proximal%20pausing%2C%20chromatin%2C%20and%20exons&journal=eLife&volume=3&publication_year=2014&author=Jonkers%2CI&author=Kwak%2CH&author=Lis%2CJT) 
    
47.  Marsico, G. et al. Whole genome experimental maps of DNA G-quadruplexes in multiple species. _Nucleic Acids Res._ **47**, 3862–3874 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFejtrjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30892612)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6486626)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole%20genome%20experimental%20maps%20of%20DNA%20G-quadruplexes%20in%20multiple%20species&journal=Nucleic%20Acids%20Res.&volume=47&pages=3862-3874&publication_year=2019&author=Marsico%2CG) 
    
48.  Kaplan, N. et al. Nucleosome sequence preferences influence in vivo nucleosome organization. _Nat. Struct. Mol. Biol._ **17**, 918–920 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXps1egs7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20683473)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2969171)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nucleosome%20sequence%20preferences%20influence%20in%20vivo%20nucleosome%20organization&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=17&pages=918-920&publication_year=2010&author=Kaplan%2CN) 
    
49.  Segal, E. & Widom, J. What controls nucleosome positions? _Trends Genet._ **25**, 335–343 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVSitb3K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19596482)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2810357)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=What%20controls%20nucleosome%20positions%3F&journal=Trends%20Genet.&volume=25&pages=335-343&publication_year=2009&author=Segal%2CE&author=Widom%2CJ) 
    
50.  Burke, T. W. & Kadonaga, J. T. _Drosophila_ TFIID binds to a conserved downstream basal promoter element that is present in many TATA-box-deficient promoters. _Genes Dev._ **10**, 711–724 (1996).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XhvVCrt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8598298)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Drosophila%20TFIID%20binds%20to%20a%20conserved%20downstream%20basal%20promoter%20element%20that%20is%20present%20in%20many%20TATA-box-deficient%20promoters&journal=Genes%20Dev.&volume=10&pages=711-724&publication_year=1996&author=Burke%2CTW&author=Kadonaga%2CJT) 
    
51.  Parry, T. J. et al. The TCT motif, a key component of an RNA polymerase II transcription system for the translational machinery. _Genes Dev._ **24**, 2013–2018 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1KjtLbN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20801935)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2939363)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20TCT%20motif%2C%20a%20key%20component%20of%20an%20RNA%20polymerase%20II%20transcription%20system%20for%20the%20translational%20machinery&journal=Genes%20Dev.&volume=24&pages=2013-2018&publication_year=2010&author=Parry%2CTJ) 
    
52.  Vermeulen, M. et al. Selective anchoring of TFIID to nucleosomes by trimethylation of histone H3 lysine 4. _Cell_ **131**, 58–69 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXht1CntLnN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17884155)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Selective%20anchoring%20of%20TFIID%20to%20nucleosomes%20by%20trimethylation%20of%20histone%20H3%20lysine%204&journal=Cell&volume=131&pages=58-69&publication_year=2007&author=Vermeulen%2CM) 
    
53.  Bhuiyan, T. & Timmers, H. T. M. Promoter recognition: putting TFIID on the spot. _Trends Cell Biol._ **29**, 752–763 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1yktbfP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31300188)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Promoter%20recognition%3A%20putting%20TFIID%20on%20the%20spot&journal=Trends%20Cell%20Biol.&volume=29&pages=752-763&publication_year=2019&author=Bhuiyan%2CT&author=Timmers%2CHTM) 
    
54.  Muller, F. & Tora, L. Chromatin and DNA sequences in defining promoters for transcription initiation. _Biochim. Biophys. Acta_ **1839**, 118–128 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFGisbjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24275614)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20and%20DNA%20sequences%20in%20defining%20promoters%20for%20transcription%20initiation&journal=Biochim.%20Biophys.%20Acta&volume=1839&pages=118-128&publication_year=2014&author=Muller%2CF&author=Tora%2CL) 
    
55.  Frontini, M. et al. NF-Y recruitment of TFIID, multiple interactions with histone fold TAFIIs. _J. Biol. Chem._ **277**, 5841–5848 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XhvFyrsbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11689552)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NF-Y%20recruitment%20of%20TFIID%2C%20multiple%20interactions%20with%20histone%20fold%20TAFIIs&journal=J.%20Biol.%20Chem.&volume=277&pages=5841-5848&publication_year=2002&author=Frontini%2CM) 
    
56.  Teng, Y. C. et al. ATRX promotes heterochromatin formation to protect cells from G-quadruplex DNA-mediated stress. _Nat. Commun._ **12**, 3887 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFKnsb3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34162889)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8222256)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ATRX%20promotes%20heterochromatin%20formation%20to%20protect%20cells%20from%20G-quadruplex%20DNA-mediated%20stress&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Teng%2CYC) 
    
57.  Li, C. et al. Ligand-induced native G-quadruplex stabilization impairs transcription initiation. _Genome Res._ **31**, 1546–1560 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXit1ajt7vL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34400476)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8415369)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ligand-induced%20native%20G-quadruplex%20stabilization%20impairs%20transcription%20initiation&journal=Genome%20Res.&volume=31&pages=1546-1560&publication_year=2021&author=Li%2CC) 
    
58.  Dao, P., Wojtowicz, D., Nelson, S., Levens, D. & Przytycka, T. M. Ups and downs of poised RNA polymerase II in B-cells. _PLoS Comput. Biol._ **12**, e1004821 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27078128)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4831825)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ups%20and%20downs%20of%20poised%20RNA%20polymerase%20II%20in%20B-cells&journal=PLoS%20Comput.%20Biol.&volume=12&publication_year=2016&author=Dao%2CP&author=Wojtowicz%2CD&author=Nelson%2CS&author=Levens%2CD&author=Przytycka%2CTM) 
    
59.  Bossaert, M. et al. Transcription-associated topoisomerase 2α (TOP2A) activity is a major effector of cytotoxicity induced by G-quadruplex ligands. _eLife_ **10**, e65184 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislantr3L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34180392)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8279764)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription-associated%20topoisomerase%202%CE%B1%20%28TOP2A%29%20activity%20is%20a%20major%20effector%20of%20cytotoxicity%20induced%20by%20G-quadruplex%20ligands&journal=eLife&volume=10&publication_year=2021&author=Bossaert%2CM) 
    
60.  Herrero-Ruiz, A. et al. Topoisomerase IIα represses transcription by enforcing promoter-proximal pausing. _Cell Rep._ **35**, 108977 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXovV2mur0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33852840)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8052185)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Topoisomerase%20II%CE%B1%20represses%20transcription%20by%20enforcing%20promoter-proximal%20pausing&journal=Cell%20Rep.&volume=35&publication_year=2021&author=Herrero-Ruiz%2CA) 
    
61.  Tikhonova, P. et al. DNA G-quadruplexes contribute to CTCF recruitment. _Int. J. Mol. Sci._ **22**, 7090 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVWisb7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34209337)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8269367)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20G-quadruplexes%20contribute%20to%20CTCF%20recruitment&journal=Int.%20J.%20Mol.%20Sci.&volume=22&publication_year=2021&author=Tikhonova%2CP) 
    
62.  Wulfridge, P. et al. G-quadruplexes associated with R-loops promote CTCF binding. _Mol. Cell_ **83**, 3064–3079 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1Ght7bN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37552993)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10529333)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=G-quadruplexes%20associated%20with%20R-loops%20promote%20CTCF%20binding&journal=Mol.%20Cell&volume=83&pages=3064-3079&publication_year=2023&author=Wulfridge%2CP) 
    
63.  Koch, F. et al. Transcription initiation platforms and GTF recruitment at tissue-specific enhancers and promoters. _Nat. Struct. Mol. Biol._ **18**, 956–963 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXovFyhsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21765417)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20initiation%20platforms%20and%20GTF%20recruitment%20at%20tissue-specific%20enhancers%20and%20promoters&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=18&pages=956-963&publication_year=2011&author=Koch%2CF) 
    
64.  John, S. et al. Genome-scale mapping of DNase I hypersensitivity. _Curr. Protoc. Mol. Biol._ **103**, 21.27.1–21.27.20 (2013).
    
65.  Nojima, T. et al. Mammalian NET-seq reveals genome-wide nascent transcription coupled to RNA processing. _Cell_ **161**, 526–540 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXnt1eqsrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25910207)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4410947)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mammalian%20NET-seq%20reveals%20genome-wide%20nascent%20transcription%20coupled%20to%20RNA%20processing&journal=Cell&volume=161&pages=526-540&publication_year=2015&author=Nojima%2CT) 
    
66.  Lidschreiber, K. et al. Transcriptionally active enhancers in human cancer cells. _Mol. Syst. Biol._ **17**, e9873 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXntVKmsbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33502116)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7838827)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptionally%20active%20enhancers%20in%20human%20cancer%20cells&journal=Mol.%20Syst.%20Biol.&volume=17&publication_year=2021&author=Lidschreiber%2CK) 
    
67.  Bailey, T. L. et al. MEME SUITE: tools for motif discovery and searching. _Nucleic Acids Res._ **37**, W202–W208 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXosFSksb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19458158)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2703892)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MEME%20SUITE%3A%20tools%20for%20motif%20discovery%20and%20searching&journal=Nucleic%20Acids%20Res.&volume=37&pages=W202-W208&publication_year=2009&author=Bailey%2CTL) 
    
68.  Bailey, T. L. DREME: motif discovery in transcription factor ChIP–seq data. _Bioinformatics_ **27**, 1653–1659 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXntFGkurg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21543442)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3106199)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DREME%3A%20motif%20discovery%20in%20transcription%20factor%20ChIP%E2%80%93seq%20data&journal=Bioinformatics&volume=27&pages=1653-1659&publication_year=2011&author=Bailey%2CTL) 
    
69.  Langmead, B. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. _Nat. Methods_ **9**, 357–359 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1Oqt7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22388286)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&journal=Nat.%20Methods&volume=9&pages=357-359&publication_year=2012&author=Langmead%2CB&author=Salzberg%2CSL) 
    
70.  Fenouil, R. et al. Pasha: a versatile R package for piling chromatin HTS data. _Bioinformatics_ **32**, 2528–2530 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVaitbjO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27153642)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pasha%3A%20a%20versatile%20R%20package%20for%20piling%20chromatin%20HTS%20data&journal=Bioinformatics&volume=32&pages=2528-2530&publication_year=2016&author=Fenouil%2CR) 
    
71.  Kim, D. et al. TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. _Genome Biol._ **14**, R36 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23618408)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053844)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&journal=Genome%20Biol.&volume=14&publication_year=2013&author=Kim%2CD) 
    
72.  Saldanha, A. J. Java Treeview—extensible visualization of microarray data. _Bioinformatics_ **20**, 3246–3248 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVSrur%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15180930)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Java%20Treeview%E2%80%94extensible%20visualization%20of%20microarray%20data&journal=Bioinformatics&volume=20&pages=3246-3248&publication_year=2004&author=Saldanha%2CAJ) 
    
73.  Descostes, N. et al. Tyrosine phosphorylation of RNA polymerase II CTD is associated with antisense promoter transcription and active enhancers in mammalian cells. _eLife_ **3**, e02105 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24842994)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4042876)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tyrosine%20phosphorylation%20of%20RNA%20polymerase%20II%20CTD%20is%20associated%20with%20antisense%20promoter%20transcription%20and%20active%20enhancers%20in%20mammalian%20cells&journal=eLife&volume=3&publication_year=2014&author=Descostes%2CN) 
    
74.  Maurano, M. T. et al. Large-scale identification of sequence variants influencing human transcription factor occupancy in vivo. _Nat. Genet._ **47**, 1393–1401 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslant7nE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26502339)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4666772)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large-scale%20identification%20of%20sequence%20variants%20influencing%20human%20transcription%20factor%20occupancy%20in%20vivo&journal=Nat.%20Genet.&volume=47&pages=1393-1401&publication_year=2015&author=Maurano%2CMT) 
    
75.  Adelman, K. & Lis, J. T. Promoter-proximal pausing of RNA polymerase II: emerging roles in metazoans. _Nat. Rev. Genet._ **13**, 720–731 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlGnsL7M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22986266)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3552498)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Promoter-proximal%20pausing%20of%20RNA%20polymerase%20II%3A%20emerging%20roles%20in%20metazoans&journal=Nat.%20Rev.%20Genet.&volume=13&pages=720-731&publication_year=2012&author=Adelman%2CK&author=Lis%2CJT) 
    
76.  Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. _Bioinformatics_ **31**, 166–169 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Sjt7vL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25260700)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HTSeq%E2%80%94a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&journal=Bioinformatics&volume=31&pages=166-169&publication_year=2015&author=Anders%2CS&author=Pyl%2CPT&author=Huber%2CW) 
    
77.  Alexander, R. D. et al. RiboSys, a high-resolution, quantitative approach to measure the in vivo kinetics of pre-mRNA splicing and 3′-end processing in _Saccharomyces cerevisiae_. _RNA_ **16**, 2570–2580 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFOmsL3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20974745)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2995417)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RiboSys%2C%20a%20high-resolution%2C%20quantitative%20approach%20to%20measure%20the%20in%20vivo%20kinetics%20of%20pre-mRNA%20splicing%20and%203%E2%80%B2-end%20processing%20in%20Saccharomyces%20cerevisiae&journal=RNA&volume=16&pages=2570-2580&publication_year=2010&author=Alexander%2CRD) 
    
78.  Boireau, S. et al. The transcriptional cycle of HIV-1 in real-time and live cells. _J. Cell Biol._ **179**, 291–304 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXht1ekurbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17954611)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2064765)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20transcriptional%20cycle%20of%20HIV-1%20in%20real-time%20and%20live%20cells&journal=J.%20Cell%20Biol.&volume=179&pages=291-304&publication_year=2007&author=Boireau%2CS) 
    
79.  Renaud de la Faverie, A., Guedin, A., Bedrat, A., Yatsunyk, L. A. & Mergny, J. L. Thioflavin T as a fluorescence light-up probe for G4 formation. _Nucleic Acids Res._ **42**, e65 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXntlyksLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24510097)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4005661)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Thioflavin%20T%20as%20a%20fluorescence%20light-up%20probe%20for%20G4%20formation&journal=Nucleic%20Acids%20Res.&volume=42&publication_year=2014&author=Renaud%20de%20la%20Faverie%2CA&author=Guedin%2CA&author=Bedrat%2CA&author=Yatsunyk%2CLA&author=Mergny%2CJL) 
    
80.  Mueller, F. et al. FISH-quant: automatic counting of transcripts in 3D FISH images. _Nat. Methods_ **10**, 277–278 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXkslWrsbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23538861)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FISH-quant%3A%20automatic%20counting%20of%20transcripts%20in%203D%20FISH%20images&journal=Nat.%20Methods&volume=10&pages=277-278&publication_year=2013&author=Mueller%2CF) 
    
81.  Esnault, C. & Andrau, J. C. Scripts and codes for G-quadruplexes are promoter elements controlling nucleosome exclusion and RNA polymerase II pausing. _Zenodo_ [https://doi.org/10.5281/zenodo.15124924](https://doi.org/10.5281/zenodo.15124924) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02263-6?format=refman&flavour=references)

Acknowledgements
----------------

This work was supported by institutional grants from the CNRS including an 80prime2021 ‘Deciph G4’ and a grant from the ‘Agence Nationale de la Recherche’ (G4access, ANR-20-CE12-0023), Fondation pour la Recherche Medicale FRM EQU202403018039 and INCA PLBIO20-225. C.E. was supported by a grant from ARC (retour postdoc); E.G.-O. and S.B. were supported by grants from EpiGenMed LabEx of Excellence and the ‘Agence Nationale de la Recherche’ (ANR-18-CE12-0019). J.-L.M. was supported by Inserm, the CNRS and Fondation de l'École Polytechnique. We are grateful to S. Spicuglia, D. Fisher, R. Feil and M. Lagha for critical reading of the paper and to L. Esnault, M. Esnault, F. Mueller and R. Topno for help with quantification of live-cell imaging data.

Author information
------------------

### Authors and Affiliations

1.  Institut de Génétique Moléculaire de Montpellier, CNRS-UMR 5535, University of Montpellier, Montpellier, France
    
    Cyril Esnault, Amal Zine El Aabidine, Marie-Cécile Robert, Talha Magat, Alexia Pigeot, Soumya Bouchouika, Encar Garcia-Oliver, Kevin Gawron, Eugénia Basyuk, Magdalena A. Karpinska, Alja Kozulic-Pirher, Edouard Bertrand & Jean-Christophe Andrau
    
2.  Institut de Génétique Humaine, CNRS-UMR9002, University of Montpellier, Montpellier, France
    
    Marie-Cécile Robert, Magdalena A. Karpinska, Alja Kozulic-Pirher & Edouard Bertrand
    
3.  Equipe labélisée Ligue Nationale Contre le Cancer, Montpellier, France
    
    Marie-Cécile Robert, Magdalena A. Karpinska & Edouard Bertrand
    
4.  Laboratoire d’Optique et Biosciences, Ecole Polytechnique, CNRS, Inserm, Institut Polytechnique de Paris, Palaiseau, France
    
    Anne Cucchiarini, Yu Luo & Jean-Louis Mergny
    
5.  Laboratoire de Microbiologie Fondamentale et Pathogénicité, CNRS-UMR 5234, Université de Bordeaux, Bordeaux, France
    
    Eugénia Basyuk
    
6.  CNRS UMR9187, INSERM U1196, Institut Curie, PSL Research University, Orsay, France
    
    Yu Luo & Daniela Verga
    
7.  LBCMCP, Centre de Biologie Intégrative, Université de Toulouse, CNRS, UPS, Toulouse, France
    
    Raphael Mourad
    
8.  Laboratory of Pathogen Host Interactions, UMR CNRS 5235, University of Montpellier, Montpellier, France
    
    Ovidiu Radulescu
    
9.  Equipe labéllisée Fondation pour la Recherche Médicale, Montpellier, France
    
    Jean-Christophe Andrau
    

Authors

1.  Cyril Esnault
    
    [View author publications](https://www.nature.com/search?author=Cyril%20Esnault)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cyril%20Esnault) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cyril%20Esnault%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Amal Zine El Aabidine
    
    [View author publications](https://www.nature.com/search?author=Amal%20Zine%20El%20Aabidine)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Amal%20Zine%20El%20Aabidine) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Amal%20Zine%20El%20Aabidine%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Marie-Cécile Robert
    
    [View author publications](https://www.nature.com/search?author=Marie-C%C3%A9cile%20Robert)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marie-C%C3%A9cile%20Robert) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marie-C%C3%A9cile%20Robert%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Anne Cucchiarini
    
    [View author publications](https://www.nature.com/search?author=Anne%20Cucchiarini)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anne%20Cucchiarini) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anne%20Cucchiarini%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Talha Magat
    
    [View author publications](https://www.nature.com/search?author=Talha%20Magat)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Talha%20Magat) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Talha%20Magat%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Alexia Pigeot
    
    [View author publications](https://www.nature.com/search?author=Alexia%20Pigeot)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alexia%20Pigeot) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alexia%20Pigeot%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Soumya Bouchouika
    
    [View author publications](https://www.nature.com/search?author=Soumya%20Bouchouika)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Soumya%20Bouchouika) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Soumya%20Bouchouika%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Encar Garcia-Oliver
    
    [View author publications](https://www.nature.com/search?author=Encar%20Garcia-Oliver)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Encar%20Garcia-Oliver) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Encar%20Garcia-Oliver%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Kevin Gawron
    
    [View author publications](https://www.nature.com/search?author=Kevin%20Gawron)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kevin%20Gawron) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kevin%20Gawron%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Eugénia Basyuk
    
    [View author publications](https://www.nature.com/search?author=Eug%C3%A9nia%20Basyuk)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eug%C3%A9nia%20Basyuk) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eug%C3%A9nia%20Basyuk%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Magdalena A. Karpinska
    
    [View author publications](https://www.nature.com/search?author=Magdalena%20A.%20Karpinska)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Magdalena%20A.%20Karpinska) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Magdalena%20A.%20Karpinska%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Alja Kozulic-Pirher
    
    [View author publications](https://www.nature.com/search?author=Alja%20Kozulic-Pirher)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alja%20Kozulic-Pirher) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alja%20Kozulic-Pirher%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Yu Luo
    
    [View author publications](https://www.nature.com/search?author=Yu%20Luo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yu%20Luo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yu%20Luo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Daniela Verga
    
    [View author publications](https://www.nature.com/search?author=Daniela%20Verga)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniela%20Verga) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniela%20Verga%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Raphael Mourad
    
    [View author publications](https://www.nature.com/search?author=Raphael%20Mourad)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Raphael%20Mourad) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Raphael%20Mourad%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Ovidiu Radulescu
    
    [View author publications](https://www.nature.com/search?author=Ovidiu%20Radulescu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ovidiu%20Radulescu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ovidiu%20Radulescu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Jean-Louis Mergny
    
    [View author publications](https://www.nature.com/search?author=Jean-Louis%20Mergny)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jean-Louis%20Mergny) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jean-Louis%20Mergny%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Edouard Bertrand
    
    [View author publications](https://www.nature.com/search?author=Edouard%20Bertrand)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Edouard%20Bertrand) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Edouard%20Bertrand%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Jean-Christophe Andrau
    
    [View author publications](https://www.nature.com/search?author=Jean-Christophe%20Andrau)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jean-Christophe%20Andrau) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jean-Christophe%20Andrau%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

J.-C.A. and C.E. designed the experiments. C.E., E.G.-O., T.M., K.G., S.B. and A.P. performed the experiments. C.E. and A.Z.E.A. analyzed the genomic data. C.E., E. Basyuk, M.A.K., M.-C.R. and E. Bertrand performed and interpreted the microscopy experiments. Y.L., A.C., D.V. and J.-L.M. characterized and interpreted the G4s in vitro. O.R. performed the mathematical modeling. A.K.-P. and R.M. contributed to the early stage of the project. J.-C.A. and C.E. conceived of the project, suggested and interpreted experiments and wrote the article. All authors reviewed the paper.

### Corresponding authors

Correspondence to [Cyril Esnault](mailto:cyril.esnault@igmm.cnrs.fr) or [Jean-Christophe Andrau](mailto:jean-christophe.andrau@igmm.cnrs.fr).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Valentin Flury, Elmar Wolf, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02263-6#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Analysis of promoter elements and motifs in 3 independent cell lines.](https://www.nature.com/articles/s41588-025-02263-6/figures/8)

**a-** Definition of major TSS (mTSS), MEME and DREME promoter motif analyses in K562 cells. CAGE datasets (FANTOM consortium) were used to define mTSSs at the nucleotide resolution (n=8346). Heatmaps show short chromatin (nascent) RNA-seq (analysed from [GSE52914](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52914)) and Pol II ChIP-seq (ENCODE) docked on the main sense mTSS and ranked by increasing distance between sense and antisense short RNAs (top left panel). Sequence motif analysis of the transcription initiating nucleotides (INR) at the sense and antisense mTSSs are shown (bottom left). On the right panels are shown TFBS analyses using MEME or DREME and frequency analyses for all sequences features including Quadparser (QP1-7) and BRE motifs. The random control column depicts a search for the motif in 8346 random genomic sequences (with 10000 permutations). See also Supplementary Table [2](https://www.nature.com/articles/s41588-025-02263-6#MOESM4) for detailed frequencies of all motifs. **b-** Definition of mTSSs and MEME and DREME motif analyses in Raji B cells. The analysis was performed as in (**a**) over 8356 human promoters and using chrRNA-seq in Raji for mTSS determination (analysed from [GSE52914](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52914)). **c-** Definition of mTSSs, MEME and DREME motif analyses in mouse primary T cells. The analysis was performed as in (**a**) over 7947 mouse promoters and using short-RNA datasets (size-selected below 50 bp, analysed from [GSE38577](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38577)) to define mTSSs at the nucleotide resolution. Short RNA-seq and Pol II ChIP-seq data sets are shown.

### [Extended Data Fig. 2 Association of the TATAbox and G4 motifs with transcription initiation.](https://www.nature.com/articles/s41588-025-02263-6/figures/9)

**a-** pG4, BRE and SP1 motifs largely overlap at active promoters. Venn diagram of active promoters containing SP1, BRE motifs or pG4s. G4Hunter is displayed at two stringencies (1.5, red circle or 2.0, dotted inner circle). The principle of the G4Hunter algorithm is to score positively Gs and G stretches, while scoring negatively Cs and C stretches within a defined window (typically 25 nt). Overlapping G4s, above a defined threshold, are concatenated. A and T nucleotides score are fixed as null. G4Hunter scores >1.5 and 2.0 correspond to likelihood of G4 formation _in vitro_ of >95 and 99%[20](https://www.nature.com/articles/s41588-025-02263-6#ref-CR20 "Bedrat, A., Lacroix, L. & Mergny, J. L. Re-evaluation of G-quadruplex propensity with G4Hunter. Nucleic Acids Res. 44, 1746–1759 (2016)."),[24](https://www.nature.com/articles/s41588-025-02263-6#ref-CR24 "Esnault, C. et al. G4access identifies G-quadruplexes and their associations with open chromatin and imprinting control regions. Nat. Genet. 55, 1359–1369 (2023)."). **b-** Promoters containing G4 predictions (G4H1.5 or 2.0) tend to harbour less other TFBS or promoter elements as compared to all promoters (All prom). This analysis was performed with the selection described in Fig. [1](https://www.nature.com/articles/s41588-025-02263-6#Fig1) for K562 cells. **c-** Promoters with TATA boxes show more focused and directional transcription. Pol II ChIP-seq profiles in K562 cells (ENCODE) at promoters of expressed genes that contain no TATA or a noncanonical (TATAW) or a canonical (TATAWAAG) TATA box. **d-** Experimental G4 signals definition. Groups 1–6 correspond to increasing level of G4access signals. Heatmaps in K562 and Raji of predicted G4 (G4Hunter 2.0) and G4 signals using G4access[24](https://www.nature.com/articles/s41588-025-02263-6#ref-CR24 "Esnault, C. et al. G4access identifies G-quadruplexes and their associations with open chromatin and imprinting control regions. Nat. Genet. 55, 1359–1369 (2023)."), G4 ChIP[14](https://www.nature.com/articles/s41588-025-02263-6#ref-CR14 "Hansel-Hertsch, R. et al. G-quadruplex structures mark human regulatory chromatin. Nat. Genet. 48, 1267–1272 (2016)."),[25](https://www.nature.com/articles/s41588-025-02263-6#ref-CR25 "Mao, S. Q. et al. DNA G-quadruplex structures mold the DNA methylome. Nat. Struct. Mol. Biol. 25, 951–957 (2018).") G4-seq[26](https://www.nature.com/articles/s41588-025-02263-6#ref-CR26 "Chambers, V. S. et al. High-throughput sequencing of DNA G-quadruplex structures in the human genome. Nat. Biotechnol. 33, 877–881 (2015)."), GC and CpG contents and in Raji predicted G4, G4access, ssDNA-seq[27](https://www.nature.com/articles/s41588-025-02263-6#ref-CR27 "Kouzine, F. et al. Global regulation of promoter melting in naive lymphocytes. Cell 153, 988–999 (2013)."), G4-seq, GC and CpG contents. See also Figs. [1e](https://www.nature.com/articles/s41588-025-02263-6#Fig1) and [2b](https://www.nature.com/articles/s41588-025-02263-6#Fig2). The heatmaps were centred on the G4 motif upstream of TSS as shown below the tracks. **e-** Experimental G4 signal metaprofiles. Metaprofiles of the heatmaps shown in d. The signals were divided in 6 groups of ascending G4 access signals. Metaprofiles were centred on the G4 motif upstream of TSS as shown below the tracks. **f-** Metaprofiles of R-loops at active promoters and docked on pG4s (G4H 2.0) upstream of experimental TSSs in K562 cells. The signals were divided in 6 groups of ascending G4 access signals **g-** Metaprofiles of ChIP-seq of Pol II, TBP and TFIIB. (Left) Pol II signals were centered on pG4s (G4H2.0) in 1444 pG4-containing promoters of active genes (−100,+20 bp) in Raji cells and in the 6 classes defined in Extended Data Fig. [2d](https://www.nature.com/articles/s41588-025-02263-6#Fig9). (Right) Metaprofiles of ChIP-seq of Pol II, TBP, TFIIB, centered on pG4s (G4H2.0) on 1291 pG4-containing promoters of active genes (−100,+20 bp) in mouse T cells ([GSE38577](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38577)). **h-** Metaprofiles of Nucleosome density, H3K4me3 and H3K27ac. Heatmaps of MNase-seq signals (Nucl. Density) and ChIP-seq signals of H3K4me3 and H3K27ac are shown on promoters containing G4 predictions in K562, Raji and mouse primary T cells. Signals are centered on pG4s (G4H2.0). **i-** Heatmaps and metaprofiles of Polycomb-deposited H3K27me3 mark, GC and CpG-content in groups 1 and 2 defined in Fig. [2c](https://www.nature.com/articles/s41588-025-02263-6#Fig2). A green arrow indicates the sense of transcription in each graph, All accession numbers are presented in the Supplementary Table [1](https://www.nature.com/articles/s41588-025-02263-6#MOESM4).

### [Extended Data Fig. 3 G4s associate with nucleosome exclusion, Pol II binding and chromatin modifiers.](https://www.nature.com/articles/s41588-025-02263-6/figures/10)

**a-** G4 forming sequences mapped by G4access associates with nucleosome eviction and high Pol II signals. G4Hunter (G4H1.2) containing regions associated to G4access peaks were selected and compared to a control group of G4Hunter (G4H1.2) regions that do not yield G4 signals. Corresponding signals of G4Hunter (GH1.2), G4access, MNase-seq, Pol II signals and GC content track are displayed. **b-** G4 forming sequences mapped by G4 ChIP associates with nucleosome eviction and high Pol II signals. G4Hunter (G4H1.2) containing regions associated to G4 ChIP peaks were selected and compared to a control group of G4Hunter (GH1.2) regions that do not yield G4 signals. Corresponding signals of G4hunter (GH1.2), G4-ChIP, MNase-seq, Pol II signals and GC content track are displayed. **c-** Cell specific G4access signals segregate promoter activity. G4access signals at promoters in Raji and K562 cells were isolated and the union of peaks were analysed in each cell type and ranked by the ratio of signals in Raji vs K562, allowing to score for 3 classes, as indicated, of Raji, common and K562 specific regions. Signals for each class are indicated below the heat maps. **d-** Cell-specific G4access yields specific transcription and chromatin patterns in Raji and K562 cells. Transcription, active chromatin, nucleosome density and accessibility were analysed for the 3 classes in the 2 cell lines. **e-** Cell-specific G4access yields specific nascent RNA patterns in Raji and K562 cells. Box plot of chrRNA-seq at genes controlled by the 3 classes of promoters as indicated. Pval indicated by \*\*\*\* represents statistical values assessed by Mann-Whitney two-sided test and <10−3. N=64; 6489; 975 and 19383. Box and whiskers with median, upper and lower quartiles and minimal and maximal ranges. **f-** Heatmap and signal plot of NDR and no NDR selections of promoters harboring G4 signal (separated by dashed line). **g-** Heatmaps of experimental and predicted G4s at both selections. **h-** Distribution of the number of moderate (G4H1.2), strong (G4H1.5) and very strong G4 predictions (G4H2.0) in the two selections. N=53 and 1634. Box and whiskers with median, upper and lower quartiles and +/- 1.5x inter quartile range. **i-** G4s, remodelers, chromatin and transcription signals at NDR and no NDR selections.

[Source data](https://www.nature.com/articles/s41588-025-02263-6#MOESM7)

### [Extended Data Fig. 4 G4, nucleosome and Pol II Patterns of promoters with or without SP1/GC boxes.](https://www.nature.com/articles/s41588-025-02263-6/figures/11)

**a-** Heatmaps ranked by increasing signals of SP1 binding and centred on a pG4 (ChIP-seq, ENCODE). All promoters of expressed genes that contain a pG4 in K562 cells are presented (n=1766). Group 1 and 2 are depleted or enriched for SP1, respectively. Metaprofiles derived from the heatmaps (groups 1 and 2) of SP1 density, GC content nucleosome densities and G4access are shown on the right. **b-** Metaprofiles of promoters not containing a canonical GC box/SP1 binding site. **c-** Metaprofiles of promoters not containing a non-canonical GC box/SP1 binding site. Green arrows indicate the sense of transcription in each graph.

### [Extended Data Fig. 5 Chromatin and G4 dynamics following transcription inhibition, activation and chromatin opening _in vivo_ and _in vitro_ at promoters.](https://www.nature.com/articles/s41588-025-02263-6/figures/12)

**a-** Persistence of NDRs at pG4 sites following transcription inhibition by triptolide. Raji cells were treated for 2h with 1 µM. Pol II clearance and nucleosome depletion mapped by MNase-seq from active genes containing pG4s (G4H2.0) are shown ([GSE187007](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE187007) and this study). **b-** Examples of G4 (G4access) and transcription (chrRNA) dynamics following serum shock at serum-inducible genes. NIH3T3 were treated 15 min with serum following overnight starvation (see [methods](https://www.nature.com/articles/s41588-025-02263-6#Sec11)). **c-** Serum-inducible genes as determined by DESeq (Padj=0.05) are plotted as a heatmap by decreasing fold change as indicated (left). Invariant control genes heat maps are also shown (right). **d-** Metaprofiles of G4access signals following serum induction at inducible and control genes. **e-** Heatmaps and metaprofiles of nucleosome densities _in vivo_ and _in vitro_, GC and CpG-content in groups 1 and 2 defined in Fig. [2d](https://www.nature.com/articles/s41588-025-02263-6#Fig2).

### [Extended Data Fig. 6 Strong G4 predictions yield nucleosome depletion and footprint of both ss and dsDNA _in vitro._](https://www.nature.com/articles/s41588-025-02263-6/figures/13)

**a-** Metaprofiles of MNase-seq from _in vitro_ reconstituted chromatin on T cell genomic DNA ([GSE25133](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25133)) centred on pG4 at the active pG4-containing promoters (−100, +20 bp), intergenic or intragenic regions defined in K562 cells. **b-** Heatmaps ranked by increasing MNase signals showing nucleosome or G4H1.5 (docked on G4H2.0) signals. Six visually defined groups based on relative nucleosome densities are further plotted as graphs below the heatmaps. **c-** Two sets of sequences were randomly selected with (G4H>2) or without (G4H <1) G4 predictions but with identical GC content in intergenic areas of the human genome. - GC content distribution of the two selections were sampled to yield identical patterns. **d-** G4 forming potential but not GC content correlates with nucleosome exclusion. Heatmaps of the 2 selections for G4H2.0 annotation densities, DMS-seq, G4seq and _in vitro_ nucleosome densities, ranked by increasing MNase signal. DMS-seq monitors G4’s footprinting by quantifying cleavage sites in dsDNA human genomic context while G4seq monitors G4 formation _in vitro_ at genomic sequence based on nucleotide misincorporation (ssDNA). This figure shows that for equivalent GC content, most areas able to form G4s _in vitro_ do deplete nucleosome by default.

### [Extended Data Fig. 7 G4s contribute to CpG islands openness and activity.](https://www.nature.com/articles/s41588-025-02263-6/figures/14)

**a-** Selection of G4-containg and G4-depleted CpG islands. The selections were performed on the same number of sequences (2 x 1112) with similar length and GC content, with the indicated G4Hunter thresholds. For the groups to be of equal size, equivalent length and GC content the initial populations of CGIs with G4H>1.5 (21536) or G4H<1.2 (2191) were randomized to end up with 2 groups of 1112 sequences. **b-** Heatmaps as in Fig. [4a](https://www.nature.com/articles/s41588-025-02263-6#Fig4) for the selections presented in a. The DNAse data used here indicate more chromatin opening in the G4-containing group (ENCODE data for K562 cells, [GSE32970](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32970)). **c-** Average profiles of the group 1 and 2 shown in a and b, zoomed over 4 kb. For Nucleosome density, both a 20 kb and 4 kb windows are indicated to best show the differences in the 2 groups (observed essentially for the width of the NDRs).

### [Extended Data Fig. 8 Evidence for G4 formation at pG4s in model promoters _in vivo_ and _in vitro._](https://www.nature.com/articles/s41588-025-02263-6/figures/15)

**a-** G4access and CUT&Tag[42](https://www.nature.com/articles/s41588-025-02263-6#ref-CR42 "Lyu, J., Shao, R., Kwong Yung, P. Y. & Elsässer, S. J. Genome-wide mapping of G-quadruplex structures with CUT&Tag. Nucleic Acids Res. 50, e13 (2022).") at model promoters. Signals were extracted from data obtained in mouse ES cell lines at indicated model promoters. Predicted G4s scored by G4Hunter G4H2.0 are indicated. Their sequences, used for promoter assays in single cells, are shown below the tracks in the zoomed areas. Each of the G tracks (n>1) are indicated in light blue. Complete sequences of the model core promoters are indicated in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02263-6#MOESM4). **b-** G4access, G4 cut&Tag as in (**a**) and ssDNA _in vitro_ (mouse T cells) and _in vivo_ at model promoters and zoom around the gene starts. G4Hunter annotations and model G4 sequences used in FISH are shown. **c-** _In vitro_ assessment of G4 structures of model G4s using a dsDNA FRET-MC assay. (left) Cartoon representing the principle of the assay. Template oligonucleotides are incubated in G4 folding conditions prior hybridization and completion of the FRET-MC competition assay. (right) Plot of the S Factor of dsDNA samples of wt and mutants as indicated. In this assay a score <0.35 indicates a G4 is formed. Sequences of all oligonucleotides are provided in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02263-6#MOESM4). (n=2 technical replicates, means ±s.e.m).

[Source data](https://www.nature.com/articles/s41588-025-02263-6#MOESM8)

### [Extended Data Fig. 9 G4 forming potential is involved in Eef1a1 model promoter activity.](https://www.nature.com/articles/s41588-025-02263-6/figures/16)

**a-** Representative images of long movie analyses. Long movies of MS2 reporter activity in WT and mutant cell lines as indicated. (Left) maximum image projections of selected 3-D image stacks from the 8h movies. The arrows indicate transcription sites. (Right) graphs display the corresponding quantifications of the movies, with the bars representing ON (green) and OFF (red) states. **b-** Survival functions describing transcriptional bursting at the model promoters and deconvolution indicating 2 fits to describe optimal function in the WT situation and 3 fits for both TATA and G4 mutants. **c-** WT promoters require 2 main steps for transcription, and mutant promoters 3 main steps defined by 3- or 5-time constants, respectively. The values of the derived time constants (described in methods) are indicated in the right panel. **d-** Heatmaps of permissive (red) and non-permissive (blue) transcription periods for promoters of WT and mutant Eef1a1 G4mut2 in single cell live imaging (long movies, 8h, with stacks every 3 min). Each line represents an individual cell assessment. **e-** Box plots of ON and OFF period average duration in WT and G4mut2 promoters (measured in long movies). Computed P-values (two-sided Mann-Whitney test) are as follows: \*\*\*\*<1e−4. N= 204;869;169 and 857. Box and whiskers with median, upper and lower quartiles and +/− 1.5x inter quartile range.

[Source data](https://www.nature.com/articles/s41588-025-02263-6#MOESM9)

### [Extended Data Fig. 10 Time course analysis of G4s stabilization by PDS effects on nucleosome exclusion and positioning, and Pol II pausing.](https://www.nature.com/articles/s41588-025-02263-6/figures/17)

**a-** FACS analysis of γH2AX levels following PDS treatment at the indicated time (n=3 biological replicates - different cultures, time and experiments, means ±s.e.m). T24h is significantly different from t0 (Pval=0.0025) as assessed by a two-sided Student t-test. **b-** PDS influences Pol II pausing. Changes of Pol II pausing scores (see [methods](https://www.nature.com/articles/s41588-025-02263-6#Sec11)) in response to 10 µM PDS of all genes with detectable Pol II at promoters and gene bodies for 0,10, 30 and 60 min are displayed. Genes that have increased gene body signals with a P-value<0.05 (DESeq) at t=10 min are highlighted in dark blue (t=10 min), dark green (t=30 min) or dark orange (t=60 min), showing that pause release is relatively transient following treatment. **c-** Gene ontology analysis of the 602 selected pause release genes using DAVID webtool. **d-** Pause release profiles (nascent RNA) over gene bodies using chrRNA-seq and TT-seq read-outs. **e-** Other genes profiles over gene bodies. **f-** Pause release for long genes (Pol II profiles) and G4Hunter scores of the 2 classes following 10 and 60 min PDS. **g-** Pausing score’s dynamics correlates with G4 strength/stability. (top left) ten groups of pausing scores changes were defined and their trend is shown. most pause release is found on the genes with higher G4score (top right) densities of G4H annotation (G4H1.5 or 2.0, bottom panels). **h-** G4access signal and nucleosome densities dynamics at pause release genes. **i-** G4access signal and nucleosome densities dynamics at other genes.

[Source data](https://www.nature.com/articles/s41588-025-02263-6#MOESM10)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM1_ESM.pdf)

Supplementary Notes 1–4.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM3_ESM.pdf)

### [Supplementary Tables 1–5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM4_ESM.xlsx)

Supplementary Table 1. Genomic file. Table summarizing all resources of the genomic analyses presented in this study. The table depicts the cell type, the experiment type and the source of the files. Supplementary Table 2. Motif search analysis. Frequency analyses of sequence features in a 120-bp window around experimental TSSs (−100, +20). Fourteen motifs or features were analyzed (BREd, BREu, ETS, G4H1.5, G4H2.0, QP1-7, INR, NF-Y, half G4, SP1 motifs, two TATA box consensuses and two negative control motifs). Supplementary Table 3. Oligonucleotides used in this study in vitro for G4 validation assays. These sequences were used for in vitro validation of G4 structure formation using single-stranded assays (IDS, TDS, CD, FRET-MC) and double-stranded FRET-MC. Supplementary Table 4. Sequence of all mouse model promoters used in this study. These sequences were inserted upstream of the MS2 reporter and inserted in the HeLa genome using the FRT system. Supplementary Table 5. Table of qPCR oligonucleotides used in this study. All oligonucleotide pairs used for qPCR are presented; sequences, complementary genomes and genomic locations of amplicons are described.

Source data
-----------

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM5_ESM.xlsx)

Source data for Fig. 5.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM6_ESM.xlsx)

Source data for Fig. 6.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM7_ESM.xlsx)

Source data for Extended Fig. 3.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM8_ESM.xlsx)

Source data for Extended Fig. 8.

### [Source Data Extended Data Fig. 9](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM9_ESM.xlsx)

Source data for Extended Fig. 9.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02263-6/MediaObjects/41588_2025_2263_MOESM10_ESM.xlsx)

Source data for Extended Fig. 10.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=G-quadruplexes%20are%20promoter%20elements%20controlling%20nucleosome%20exclusion%20and%20RNA%20polymerase%20II%20pausing&author=Cyril%20Esnault%20et%20al&contentID=10.1038%2Fs41588-025-02263-6&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-07-22&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02263-6)

### Cite this article

Esnault, C., Zine El Aabidine, A., Robert, MC. _et al._ G-quadruplexes are promoter elements controlling nucleosome exclusion and RNA polymerase II pausing. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02263-6

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02263-6?format=refman&flavour=citation)

-   Received: 07 March 2023
    
-   Accepted: 11 June 2025
    
-   Published: 22 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02263-6
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
