> id: https://www.nature.com/articles/s41588-025-02262-7

> link: https://www.nature.com/articles/s41588-025-02262-7

> title: Improved multiancestry fine-mapping identifies cis-regulatory variants underlying molecular traits and disease risk

# Improved multiancestry fine-mapping identifies cis-regulatory variants underlying molecular traits and disease risk
_Published on Mon, 21 Jul 2025 00:00:00 GMT_

Abstract
--------

Multiancestry statistical fine-mapping of _cis_\-molecular quantitative trait loci (_cis_\-molQTL) aims to improve the precision of distinguishing causal _cis_\-molQTLs from tagging variants. Here we present the sum of shared single effects (SuShiE) model, which leverages linkage disequilibrium heterogeneity to improve fine-mapping precision, infer cross-ancestry effect size correlations and estimate ancestry-specific expression prediction weights. Through extensive simulations, we find that SuShiE consistently outperforms existing methods. We apply SuShiE to 36,907 molecular phenotypes including mRNA expression and protein levels from individuals of diverse ancestries in the TOPMed-MESA and GENOA studies. SuShiE fine-maps _cis_\-molQTLs for 18.2% more genes compared with existing methods while prioritizing fewer variants and exhibiting greater functional enrichment. While SuShiE infers highly consistent _cis_\-molQTL architectures across ancestries, it finds evidence of heterogeneity at genes with predicted loss-of-function intolerance. Lastly, using SuShiE-derived _cis_\-molQTL effect sizes, we perform transcriptome- and proteome-wide association studies on six white blood cell-related traits in the All of Us biobank and identify 25.4% more genes compared with existing methods. Overall, SuShiE provides new insights into the _cis_\-genetic architecture of molecular traits.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

27,99 € / 30 days

cancel any time

[Learn more](https://shop.nature.com/products/plus/?region=ROW)

Subscribe to this journal

Receive 12 print issues and online access

195,33 € per year

only 16,28 € per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02262-7?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: SuShiE infers credible sets with PIPs, cross-ancestry effect size correlations and ancestry-specific effect sizes by leveraging shared genetic architectures and LD heterogeneity.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_Fig1_HTML.png)

**Fig. 2: SuShiE outperforms other methods, estimates accurate effect size correlation and boosts higher power of TWAS in realistic simulations.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_Fig2_HTML.png)

**Fig. 3: SuShiE reveals _cis_\-regulatory mechanisms for mRNA expression and protein abundance.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_Fig3_HTML.png)

**Fig. 4: SuShiE identifies _cis_\-eQTL [rs2528382](https://www.ncbi.nlm.nih.gov/snp/?term=rs2528382) for _URGCP_ with functional support.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_Fig4_HTML.png)

**Fig. 5: SuShiE identifies more TWAS and PWAS genes and higher chi-square statistics compared with SuSiE and MESuSiE.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_Fig5_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01870-z/MediaObjects/41588_2024_1870_Fig1_HTML.png)

### [Fine-mapping across diverse ancestries drives the discovery of putative causal variants underlying human complex traits and diseases](https://www.nature.com/articles/s41588-024-01870-z?fromPaywallRec=true)

Article 26 August 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-023-01604-7/MediaObjects/41588_2023_1604_Fig1_HTML.png)

### [MESuSiE enables scalable and powerful multi-ancestry fine-mapping of causal variants in genome-wide association studies](https://www.nature.com/articles/s41588-023-01604-7?fromPaywallRec=true)

Article 02 January 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01896-3/MediaObjects/41588_2024_1896_Fig1_HTML.png)

### [Statistically and functionally fine-mapped blood eQTLs and pQTLs from 1,405 humans reveal distinct regulation patterns and disease relevance](https://www.nature.com/articles/s41588-024-01896-3?fromPaywallRec=true)

Article Open access 24 September 2024

Data availability
-----------------

SuShiE-derived prediction models (in both tsv format and FUSION format) for TWAS, PWAS, fine-mapping and other analyzed results across _cis_\-molQTL datasets are available via Zenodo at [https://doi.org/10.5281/zenodo.10963033](https://doi.org/10.5281/zenodo.10963033) (ref. [69](https://www.nature.com/articles/s41588-025-02262-7#ref-CR69 "Lu, Z. et al. Improved multi-ancestry fine-mapping identifies cis-regulatory variants underlying molecular traits and disease risk. Zenodo 
https://doi.org/10.5281/zenodo.10963033
(2025).")). The TOPMed-MESA data can be found and requested at dbGaP: [phs000209.v13.p3](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000209.v13.p3), [phs001416.v3.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001416.v3.p1) and [phs001416.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001416.v1.p1). The GENOA data can be found and requested at dbGaP: [phs001238.v2.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001238.v2.p1) and GEO: [GSE138914](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138914). The GEUVADIS data can be found at [https://www.internationalgenome.org/data-portal/data-collection/geuvadis](https://www.internationalgenome.org/data-portal/data-collection/geuvadis). The INTERVAL data can be found and requested at [https://ega-archive.org/datasets/EGAD00001004080](https://ega-archive.org/datasets/EGAD00001004080). The summary statistics in Chen et al. can be found at [https://doi.org/10.1016/j.cell.2020.06.045](https://doi.org/10.1016/j.cell.2020.06.045). The LDSC annotation files can be found at [https://console.cloud.google.com/storage/browser/broad-alkesgroup-public-requester-pays/](https://console.cloud.google.com/storage/browser/broad-alkesgroup-public-requester-pays/). The ENCODE cCRE v3 can be found at [https://screen.encodeproject.org/index/cversions](https://screen.encodeproject.org/index/cversions). The snATAC-seq cCRE can be found at (ref. [24](https://www.nature.com/articles/s41588-025-02262-7#ref-CR24 "Chiou, J. et al. Interpreting type 1 diabetes risk with genetics and single-cell epigenomics. Nature 594, 398–402 (2021).")). The scATAC-seq cCRE can be found at (ref. [25](https://www.nature.com/articles/s41588-025-02262-7#ref-CR25 "Satpathy, A. T. et al. Massively parallel single-cell chromatin landscapes of human immune cell development and intratumoral T cell exhaustion. Nat. Biotechnol. 37, 925–936 (2019).")). The All of Us data can be requested through [https://allofus.nih.gov](https://allofus.nih.gov/). The 1000G project data can be found at [https://www.internationalgenome.org](https://www.internationalgenome.org/). The gnomAD v4.0 dataset for pLI and LOEUF is available at [https://gnomad.broadinstitute.org/news/2023-11-gnomad-v4-0/](https://gnomad.broadinstitute.org/news/2023-11-gnomad-v4-0/). The RVIS dataset can be found at (ref. [39](https://www.nature.com/articles/s41588-025-02262-7#ref-CR39 "Petrovski, S., Wang, Q., Heinzen, E. L., Allen, A. S. & Goldstein, D. B. Genic intolerance to functional variation and the interpretation of personal genomes. PLoS Genet. 9, e1003709 (2013).")). The shet dataset can be found at (ref. [70](https://www.nature.com/articles/s41588-025-02262-7#ref-CR70 "Zeng, T., Spence, J. P., Mostafavi, H. & Pritchard, J. K.and found highly significantraint from an evolutionary model with gene features. Nat. Genet. 56, 1632–1643 (2024).")). The EDS dataset can be found at (ref. [38](https://www.nature.com/articles/s41588-025-02262-7#ref-CR38 "Wang, X. & Goldstein, D. B. Enhancer domains predict gene pathogenicity and inform gene discovery in complex disease. Am. J. Hum. Genet. 106, 215–233 (2020).")).

Code availability
-----------------

SuShiE v0.16 software is available via GitHub at [https://github.com/mancusolab/sushie](https://github.com/mancusolab/sushie). The analysis codes for simulation and real-data analysis of this Article are available via GitHub at [https://github.com/mancusolab/sushie-project-codes](https://github.com/mancusolab/sushie-project-codes) and [https://doi.org/10.5281/zenodo.10963033](https://doi.org/10.5281/zenodo.10963033) (ref. [69](https://www.nature.com/articles/s41588-025-02262-7#ref-CR69 "Lu, Z. et al. Improved multi-ancestry fine-mapping identifies cis-regulatory variants underlying molecular traits and disease risk. Zenodo 
https://doi.org/10.5281/zenodo.10963033
(2025).")). The twas\_sim software is available via GitHub at [https://github.com/mancusolab/twas\_sim](https://github.com/mancusolab/twas_sim). TOPMed RNA-seq Harmonization pipeline instructions are available via GitHub at [https://github.com/broadinstitute/gtex-pipeline/blob/master/TOPMed\_RNAseq\_pipeline.md](https://github.com/broadinstitute/gtex-pipeline/blob/master/TOPMed_RNAseq_pipeline.md). The GTEx eQTL analysis pipeline is available at [https://www.gtexportal.org/home/methods](https://www.gtexportal.org/home/methods). The PLINK2 software is available at [https://www.cog-genomics.org/plink/2.0](https://www.cog-genomics.org/plink/2.0). The BCFTOOLS v1.21 software is available at [https://samtools.github.io/bcftools/bcftools.html](https://samtools.github.io/bcftools/bcftools.html). The FUSION pipeline is available at [http://gusevlab.org/projects/fusion/](http://gusevlab.org/projects/fusion/). The LiftOver software is available at [https://genome.ucsc.edu/cgi-bin/hgLiftOver](https://genome.ucsc.edu/cgi-bin/hgLiftOver). The WashU Epigenome Browser is available at [https://epigenomegateway.wustl.edu/](https://epigenomegateway.wustl.edu/). The Plotgardener v1.8.3 software is available via GitHub at [https://github.com/PhanstielLab/plotgardener/](https://github.com/PhanstielLab/plotgardener/). The AnnoQ is available at [http://annoq.org](http://annoq.org/). The SuSiEx v1.1.2 software is available via GitHub at [https://github.com/getian107/SuSiEx](https://github.com/getian107/SuSiEx). The MESuSiE software is available via GitHub at [https://github.com/borangao/MESuSiE](https://github.com/borangao/MESuSiE). The XMAP v1.0.1 software is available via GitHub at [https://github.com/YangLabHKUST/XMAP](https://github.com/YangLabHKUST/XMAP).

References
----------

1.  Cheung, V. G. et al. Mapping determinants of human gene expression by regional and genome-wide association. _Nature_ **437**, 1365–1369 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16251966)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3005311)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtFCrurrN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20determinants%20of%20human%20gene%20expression%20by%20regional%20and%20genome-wide%20association&journal=Nature&volume=437&pages=1365-1369&publication_year=2005&author=Cheung%2CVG) 
    
2.  Aguet, F. et al. Molecular quantitative trait loci. _Nat. Rev. Methods Prim._ **3**, 4 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVyhtbk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20quantitative%20trait%20loci&journal=Nat.%20Rev.%20Methods%20Prim.&volume=3&publication_year=2023&author=Aguet%2CF) 
    
3.  Wang, G., Sarkar, A., Carbonetto, P. & Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. _J. R. Stat. Soc. Ser. B_ **82**, 1273–1300 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20simple%20new%20approach%20to%20variable%20selection%20in%20regression%2C%20with%20application%20to%20genetic%20fine%20mapping&journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B&volume=82&pages=1273-1300&publication_year=2020&author=Wang%2CG&author=Sarkar%2CA&author=Carbonetto%2CP&author=Stephens%2CM) 
    
4.  Wen, X., Luca, F. & Pique-Regi, R. Cross-population joint analysis of eQTLs: fine mapping and functional annotation. _PLoS Genet._ **11**, e1005176 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25906321)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4408026)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-population%20joint%20analysis%20of%20eQTLs%3A%20fine%20mapping%20and%20functional%20annotation&journal=PLoS%20Genet.&volume=11&publication_year=2015&author=Wen%2CX&author=Luca%2CF&author=Pique-Regi%2CR) 
    
5.  Kichaev, G. & Pasaniuc, B. Leveraging functional-annotation data in trans-ethnic fine-mapping studies. _Am. J. Hum. Genet._ **97**, 260–271 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26189819)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4573268)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2ksL%2FL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Leveraging%20functional-annotation%20data%20in%20trans-ethnic%20fine-mapping%20studies&journal=Am.%20J.%20Hum.%20Genet.&volume=97&pages=260-271&publication_year=2015&author=Kichaev%2CG&author=Pasaniuc%2CB) 
    
6.  LaPierre, N. et al. Identifying causal variants by fine mapping across multiple studies. _PLoS Genet._ **17**, e1009733 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34543273)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8491908)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Wnt7vP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identifying%20causal%20variants%20by%20fine%20mapping%20across%20multiple%20studies&journal=PLoS%20Genet.&volume=17&publication_year=2021&author=LaPierre%2CN) 
    
7.  Yuan, K. et al. Fine-mapping across diverse ancestries drives the discovery of putative causal variants underlying human complex traits and diseases. _Nat. Genet._ **56**, 1841–1850 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39187616)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11888783)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhvVeku7%2FO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fine-mapping%20across%20diverse%20ancestries%20drives%20the%20discovery%20of%20putative%20causal%20variants%20underlying%20human%20complex%20traits%20and%20diseases&journal=Nat.%20Genet.&volume=56&pages=1841-1850&publication_year=2024&author=Yuan%2CK) 
    
8.  Gao, B. & Zhou, X. MESuSiE enables scalable and powerful multi-ancestry fine-mapping of causal variants in genome-wide association studies. _Nat. Genet._ **56**, 170–179 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38168930)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11849347)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXlvVGitw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MESuSiE%20enables%20scalable%20and%20powerful%20multi-ancestry%20fine-mapping%20of%20causal%20variants%20in%20genome-wide%20association%20studies&journal=Nat.%20Genet.&volume=56&pages=170-179&publication_year=2024&author=Gao%2CB&author=Zhou%2CX) 
    
9.  Cai, M. et al. XMAP: cross-population fine-mapping by leveraging genetic diversity and accounting for confounding bias. _Nat. Commun._ **14**, 6870 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37898663)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10613261)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXit1CnsrbL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=XMAP%3A%20cross-population%20fine-mapping%20by%20leveraging%20genetic%20diversity%20and%20accounting%20for%20confounding%20bias&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Cai%2CM) 
    
10.  Gusev, A. et al. Integrative approaches for large-scale transcriptome-wide association studies. _Nat. Genet._ **48**, 245–252 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26854917)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4767558)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xit1ajtbc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20approaches%20for%20large-scale%20transcriptome-wide%20association%20studies&journal=Nat.%20Genet.&volume=48&pages=245-252&publication_year=2016&author=Gusev%2CA) 
    
11.  Gamazon, E. R. et al. A gene-based association method for mapping traits using reference transcriptome data. _Nat. Genet._ **47**, 1091–1098 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26258848)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552594)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlSnurbM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20gene-based%20association%20method%20for%20mapping%20traits%20using%20reference%20transcriptome%20data&journal=Nat.%20Genet.&volume=47&pages=1091-1098&publication_year=2015&author=Gamazon%2CER) 
    
12.  Bild, D. E. et al. Ethnic differences in coronary calcification: the Multi-Ethnic Study of Atherosclerosis (MESA). _Circulation_ **111**, 1313–1320 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15769774)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ethnic%20differences%20in%20coronary%20calcification%3A%20the%20Multi-Ethnic%20Study%20of%20Atherosclerosis%20%28MESA%29&journal=Circulation&volume=111&pages=1313-1320&publication_year=2005&author=Bild%2CDE) 
    
13.  Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. _Nature_ **590**, 290–299 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33568819)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875770)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjslKnsrY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sequencing%20of%2053%2C831%20diverse%20genomes%20from%20the%20NHLBI%20TOPMed%20Program&journal=Nature&volume=590&pages=290-299&publication_year=2021&author=Taliun%2CD) 
    
14.  Shang, L. et al. Genetic architecture of gene expression in European and African Americans: an eQTL mapping study in GENOA. _Am. J. Hum. Genet._ **106**, 496–512 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32220292)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118581)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslKks7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20architecture%20of%20gene%20expression%20in%20European%20and%20African%20Americans%3A%20an%20eQTL%20mapping%20study%20in%20GENOA&journal=Am.%20J.%20Hum.%20Genet.&volume=106&pages=496-512&publication_year=2020&author=Shang%2CL) 
    
15.  The All of Us Research Program Genomics Investigators. Genomic data in the All of Us Research Program. _Nature_ **627**, 340–346 (2024).
    
16.  Zou, Y., Carbonetto, P., Wang, G. & Stephens, M. Fine-mapping from summary data with the ‘Sum of Single Effects’ model. _PLoS Genet._ **18**, e1010299 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35853082)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9337707)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVeku7jF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fine-mapping%20from%20summary%20data%20with%20the%20%E2%80%98Sum%20of%20Single%20Effects%E2%80%99%20model&journal=PLoS%20Genet.&volume=18&publication_year=2022&author=Zou%2CY&author=Carbonetto%2CP&author=Wang%2CG&author=Stephens%2CM) 
    
17.  Tibshirani, R. Regression shrinkage and selection via the LASSO. _J. R. Stat. Soc. Ser. B_ **58**, 267–288 (1996).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regression%20shrinkage%20and%20selection%20via%20the%20LASSO&journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B&volume=58&pages=267-288&publication_year=1996&author=Tibshirani%2CR) 
    
18.  Zou, H. & Hastie, T. Regularization and variable selection via the elastic net. _J. R. Stat. Soc. Ser. B_ **67**, 301–320 (2005).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regularization%20and%20variable%20selection%20via%20the%20elastic%20net&journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B&volume=67&pages=301-320&publication_year=2005&author=Zou%2CH&author=Hastie%2CT) 
    
19.  Clark, S. A. & van der Werf, J. Genomic best linear unbiased prediction (gBLUP) for the estimation of genomic breeding values. _Methods Mol. Biol._ **1019**, 321–330 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23756897)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20best%20linear%20unbiased%20prediction%20%28gBLUP%29%20for%20the%20estimation%20of%20genomic%20breeding%20values&journal=Methods%20Mol.%20Biol.&volume=1019&pages=321-330&publication_year=2013&author=Clark%2CSA&author=Werf%2CJ) 
    
20.  GTEx Consortium. The GTEx Consortium atlas of genetic regulatory effects across human tissues. _Science_ **369**, 1318–1330 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20GTEx%20Consortium%20atlas%20of%20genetic%20regulatory%20effects%20across%20human%20tissues&journal=Science&volume=369&pages=1318-1330&publication_year=2020) 
    
21.  Võsa, U. et al. Large-scale _cis_\- and _trans_\-eQTL analyses identify thousands of genetic loci and polygenic scores that regulate blood gene expression. _Nat. Genet._ **53**, 1300–1310 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34475573)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8432599)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large-scale%20cis-%20and%20trans-eQTL%20analyses%20identify%20thousands%20of%20genetic%20loci%20and%20polygenic%20scores%20that%20regulate%20blood%20gene%20expression&journal=Nat.%20Genet.&volume=53&pages=1300-1310&publication_year=2021&author=V%C3%B5sa%2CU) 
    
22.  Mostafavi, H., Spence, J. P., Naqvi, S. & Pritchard, J. K. Systematic differences in discovery of genetic effects on gene expression and complex traits. _Nat. Genet._ **55**, 1866–1875 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37857933)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12270542)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFOqsL%2FN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20differences%20in%20discovery%20of%20genetic%20effects%20on%20gene%20expression%20and%20complex%20traits&journal=Nat.%20Genet.&volume=55&pages=1866-1875&publication_year=2023&author=Mostafavi%2CH&author=Spence%2CJP&author=Naqvi%2CS&author=Pritchard%2CJK) 
    
23.  ENCODE Project Consortium et al. Expanded encyclopaedias of DNA elements in the human and mouse genomes. _Nature_ **583**, 699–710 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Expanded%20encyclopaedias%20of%20DNA%20elements%20in%20the%20human%20and%20mouse%20genomes&journal=Nature&volume=583&pages=699-710&publication_year=2020&author=ENCODE%20Project%20Consortium%2C) 
    
24.  Chiou, J. et al. Interpreting type 1 diabetes risk with genetics and single-cell epigenomics. _Nature_ **594**, 398–402 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34012112)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10560508)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFantbvJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interpreting%20type%201%20diabetes%20risk%20with%20genetics%20and%20single-cell%20epigenomics&journal=Nature&volume=594&pages=398-402&publication_year=2021&author=Chiou%2CJ) 
    
25.  Satpathy, A. T. et al. Massively parallel single-cell chromatin landscapes of human immune cell development and intratumoral T cell exhaustion. _Nat. Biotechnol._ **37**, 925–936 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31375813)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7299161)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFWqt7fO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Massively%20parallel%20single-cell%20chromatin%20landscapes%20of%20human%20immune%20cell%20development%20and%20intratumoral%20T%20cell%20exhaustion&journal=Nat.%20Biotechnol.&volume=37&pages=925-936&publication_year=2019&author=Satpathy%2CAT) 
    
26.  Lappalainen, T. et al. Transcriptome and genome sequencing uncovers functional variation in humans. _Nature_ **501**, 506–511 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24037378)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3918453)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVertLzN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptome%20and%20genome%20sequencing%20uncovers%20functional%20variation%20in%20humans&journal=Nature&volume=501&pages=506-511&publication_year=2013&author=Lappalainen%2CT) 
    
27.  Sun, B. B. et al. Genomic atlas of the human plasma proteome. _Nature_ **558**, 73–79 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29875488)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6697541)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2lsr3E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20atlas%20of%20the%20human%20plasma%20proteome&journal=Nature&volume=558&pages=73-79&publication_year=2018&author=Sun%2CBB) 
    
28.  Cai, J. et al. URGCP promotes non-small cell lung cancer invasiveness by activating the NF-κB–MMP-9 pathway. _Oncotarget_ **6**, 36489–36504 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26429875)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4742191)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=URGCP%20promotes%20non-small%20cell%20lung%20cancer%20invasiveness%20by%20activating%20the%20NF-%CE%BAB%E2%80%93MMP-9%20pathway&journal=Oncotarget&volume=6&pages=36489-36504&publication_year=2015&author=Cai%2CJ) 
    
29.  Kachuri, L. et al. Gene expression in African Americans, Puerto Ricans and Mexican Americans reveals ancestry-specific patterns of genetic architecture. _Nat. Genet._ **55**, 952–963 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37231098)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10260401)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtFSisbvF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20expression%20in%20African%20Americans%2C%20Puerto%20Ricans%20and%20Mexican%20Americans%20reveals%20ancestry-specific%20patterns%20of%20genetic%20architecture&journal=Nat.%20Genet.&volume=55&pages=952-963&publication_year=2023&author=Kachuri%2CL) 
    
30.  Shi, H. et al. Localizing components of shared transethnic genetic architecture of complex traits from GWAS summary data. _Am. J. Hum. Genet._ **106**, 805–817 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32442408)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273527)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVejtLbO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Localizing%20components%20of%20shared%20transethnic%20genetic%20architecture%20of%20complex%20traits%20from%20GWAS%20summary%20data&journal=Am.%20J.%20Hum.%20Genet.&volume=106&pages=805-817&publication_year=2020&author=Shi%2CH) 
    
31.  Shi, H. et al. Population-specific causal disease effect sizes in functionally important regions impacted by selection. _Nat. Commun._ **12**, 1098 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33597505)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7889654)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXksFSmsrY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Population-specific%20causal%20disease%20effect%20sizes%20in%20functionally%20important%20regions%20impacted%20by%20selection&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Shi%2CH) 
    
32.  Hou, K. et al. Causal effects on complex traits are similar for common variants across segments of different continental ancestries within admixed individuals. _Nat. Genet._ **55**, 549–558 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36941441)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11120833)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXlslajuro%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Causal%20effects%20on%20complex%20traits%20are%20similar%20for%20common%20variants%20across%20segments%20of%20different%20continental%20ancestries%20within%20admixed%20individuals&journal=Nat.%20Genet.&volume=55&pages=549-558&publication_year=2023&author=Hou%2CK) 
    
33.  Saito, S. et al. Gene-specific somatic epigenetic mosaicism of FDFT1 underlies a non-hereditary localized form of porokeratosis. _Am. J. Hum. Genet._ **111**, 896–912 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38653249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11080608)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXovVSrs74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene-specific%20somatic%20epigenetic%20mosaicism%20of%20FDFT1%20underlies%20a%20non-hereditary%20localized%20form%20of%20porokeratosis&journal=Am.%20J.%20Hum.%20Genet.&volume=111&pages=896-912&publication_year=2024&author=Saito%2CS) 
    
34.  Taylor, D. J. et al. Sources of gene expression variation in a globally diverse human cohort. _Nature_ **632**, 122–130 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39020179)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11291278)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFOmtL7O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sources%20of%20gene%20expression%20variation%20in%20a%20globally%20diverse%20human%20cohort&journal=Nature&volume=632&pages=122-130&publication_year=2024&author=Taylor%2CDJ) 
    
35.  Brown, B. C., Asian Genetic Epidemiology Network Type 2 Diabetes Consortium, Ye, C. J., Price, A. L. & Zaitlen, N. Transethnic genetic-correlation estimates from summary statistics. _Am. J. Hum. Genet._ **99**, 76–88 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27321947)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5005434)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVWnsLbP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transethnic%20genetic-correlation%20estimates%20from%20summary%20statistics&journal=Am.%20J.%20Hum.%20Genet.&volume=99&pages=76-88&publication_year=2016&author=Brown%2CBC&author=Ye%2CCJ&author=Price%2CAL&author=Zaitlen%2CN) 
    
36.  Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. _Nature_ **536**, 285–291 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27535533)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018207)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlOnsbbP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060%2C706%20humans&journal=Nature&volume=536&pages=285-291&publication_year=2016&author=Lek%2CM) 
    
37.  Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. _Nature_ **581**, 434–443 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32461654)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&journal=Nature&volume=581&pages=434-443&publication_year=2020&author=Karczewski%2CKJ) 
    
38.  Wang, X. & Goldstein, D. B. Enhancer domains predict gene pathogenicity and inform gene discovery in complex disease. _Am. J. Hum. Genet._ **106**, 215–233 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32032514)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7010980)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotlSru70%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhancer%20domains%20predict%20gene%20pathogenicity%20and%20inform%20gene%20discovery%20in%20complex%20disease&journal=Am.%20J.%20Hum.%20Genet.&volume=106&pages=215-233&publication_year=2020&author=Wang%2CX&author=Goldstein%2CDB) 
    
39.  Petrovski, S., Wang, Q., Heinzen, E. L., Allen, A. S. & Goldstein, D. B. Genic intolerance to functional variation and the interpretation of personal genomes. _PLoS Genet._ **9**, e1003709 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23990802)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3749936)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVCku7rJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genic%20intolerance%20to%20functional%20variation%20and%20the%20interpretation%20of%20personal%20genomes&journal=PLoS%20Genet.&volume=9&publication_year=2013&author=Petrovski%2CS&author=Wang%2CQ&author=Heinzen%2CEL&author=Allen%2CAS&author=Goldstein%2CDB) 
    
40.  Zeng, T., Spence, J. P., Mostafavi, H. & Pritchard, J. K. Bayesian estimation of gene constraint from an evolutionary model with gene features. _Nat. Genet._ **56**, 1632–1643 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38977852)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFWhsbvN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bayesian%20estimation%20of%20gene%20constraint%20from%20an%20evolutionary%20model%20with%20gene%20features&journal=Nat.%20Genet.&volume=56&pages=1632-1643&publication_year=2024&author=Zeng%2CT&author=Spence%2CJP&author=Mostafavi%2CH&author=Pritchard%2CJK) 
    
41.  Lu, Z. et al. Multi-ancestry fine-mapping improves precision to identify causal genes in transcriptome-wide association studies. _Am. J. Hum. Genet._ **109**, 1388–1404 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35931050)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9388396)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVGns7fK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-ancestry%20fine-mapping%20improves%20precision%20to%20identify%20causal%20genes%20in%20transcriptome-wide%20association%20studies&journal=Am.%20J.%20Hum.%20Genet.&volume=109&pages=1388-1404&publication_year=2022&author=Lu%2CZ) 
    
42.  Tapia, A. L. et al. A large-scale transcriptome-wide association study (TWAS) of 10 blood cell phenotypes reveals complexities of TWAS fine-mapping. _Genet. Epidemiol._ **46**, 3–16 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34779012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20large-scale%20transcriptome-wide%20association%20study%20%28TWAS%29%20of%2010%20blood%20cell%20phenotypes%20reveals%20complexities%20of%20TWAS%20fine-mapping&journal=Genet.%20Epidemiol.&volume=46&pages=3-16&publication_year=2022&author=Tapia%2CAL) 
    
43.  Wen, J. et al. Transcriptome-wide association study of blood cell traits in African ancestry and Hispanic/Latino populations. _Genes_ **12**, 1049 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34356065)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8307403)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlWjsLbK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptome-wide%20association%20study%20of%20blood%20cell%20traits%20in%20African%20ancestry%20and%20Hispanic%2FLatino%20populations&journal=Genes&volume=12&publication_year=2021&author=Wen%2CJ) 
    
44.  Rowland, B. et al. Transcriptome-wide association study in UK Biobank Europeans identifies associations with blood cell traits. _Hum. Mol. Genet._ **31**, 2333–2347 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35138379)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9307312)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFWjsLvN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptome-wide%20association%20study%20in%20UK%20Biobank%20Europeans%20identifies%20associations%20with%20blood%20cell%20traits&journal=Hum.%20Mol.%20Genet.&volume=31&pages=2333-2347&publication_year=2022&author=Rowland%2CB) 
    
45.  Chen, M.-H. et al. Trans-ethnic and ancestry-specific blood-cell genetics in 746,667 individuals from 5 global populations. _Cell_ **182**, 1198–1213.e14 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32888493)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7480402)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsl2itbbM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trans-ethnic%20and%20ancestry-specific%20blood-cell%20genetics%20in%20746%2C667%20individuals%20from%205%20global%20populations&journal=Cell&volume=182&pages=1198-1213.e14&publication_year=2020&author=Chen%2CM-H) 
    
46.  Ding, Y. et al. Polygenic scoring accuracy varies across the genetic ancestry continuum. _Nature_ **618**, 774–781 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37198491)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10284707)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtVejsbzJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20scoring%20accuracy%20varies%20across%20the%20genetic%20ancestry%20continuum&journal=Nature&volume=618&pages=774-781&publication_year=2023&author=Ding%2CY) 
    
47.  Mester, R. et al. Impact of cross-ancestry genetic architecture on GWASs in admixed populations. _Am. J. Hum. Genet._ **110**, 927–939 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37224807)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10257009)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtVKgsb7O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Impact%20of%20cross-ancestry%20genetic%20architecture%20on%20GWASs%20in%20admixed%20populations&journal=Am.%20J.%20Hum.%20Genet.&volume=110&pages=927-939&publication_year=2023&author=Mester%2CR) 
    
48.  Zhang, J. & Stram, D. O. The role of local ancestry adjustment in association studies using admixed populations. _Genet. Epidemiol._ **38**, 502–515 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25043967)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5079159)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs12ktb3F)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20local%20ancestry%20adjustment%20in%20association%20studies%20using%20admixed%20populations&journal=Genet.%20Epidemiol.&volume=38&pages=502-515&publication_year=2014&author=Zhang%2CJ&author=Stram%2CDO) 
    
49.  Wang, L. et al. Integrating single cell expression quantitative trait loci summary statistics to understand complex trait risk genes. _Nat. Commun._ **15**, 4260 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38769300)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11519974)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitlyiu7jO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrating%20single%20cell%20expression%20quantitative%20trait%20loci%20summary%20statistics%20to%20understand%20complex%20trait%20risk%20genes&journal=Nat.%20Commun.&volume=15&publication_year=2024&author=Wang%2CL) 
    
50.  Bhattacharya, A. et al. Best practices for multi-ancestry, meta-analytic transcriptome-wide association studies: lessons from the Global Biobank Meta-analysis Initiative. _Cell Genom._ **2**, 100180 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36341024)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9631681)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1eit73L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Best%20practices%20for%20multi-ancestry%2C%20meta-analytic%20transcriptome-wide%20association%20studies%3A%20lessons%20from%20the%20Global%20Biobank%20Meta-analysis%20Initiative&journal=Cell%20Genom.&volume=2&publication_year=2022&author=Bhattacharya%2CA) 
    
51.  Wainberg, M. et al. Opportunities and challenges for transcriptome-wide association studies. _Nat. Genet._ **51**, 592–599 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30926968)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6777347)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGlt7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Opportunities%20and%20challenges%20for%20transcriptome-wide%20association%20studies&journal=Nat.%20Genet.&volume=51&pages=592-599&publication_year=2019&author=Wainberg%2CM) 
    
52.  Mancuso, N. et al. Probabilistic fine-mapping of transcriptome-wide association studies. _Nat. Genet._ **51**, 675–682 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30926970)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6619422)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGlt7g%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Probabilistic%20fine-mapping%20of%20transcriptome-wide%20association%20studies&journal=Nat.%20Genet.&volume=51&pages=675-682&publication_year=2019&author=Mancuso%2CN) 
    
53.  Blei, D. M., Kucukelbir, A. & McAuliffe, J. D. Variational inference: a review for statisticians. _J. Am. Stat. Assoc._ **112**, 859–877 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFKrtrzM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Variational%20inference%3A%20a%20review%20for%20statisticians&journal=J.%20Am.%20Stat.%20Assoc.&volume=112&pages=859-877&publication_year=2017&author=Blei%2CDM&author=Kucukelbir%2CA&author=McAuliffe%2CJD) 
    
54.  Blei, D. M., Ng, A. Y. & Jordan, M. I. Latent Dirichlet Allocation. In _Advances in Neural Information Processing Systems_ Vol. 14, 601–608 (MIT Press, 2002).
    
55.  Frankish, A. et al. GENCODE reference annotation for the human and mouse genomes. _Nucleic Acids Res._ **47**, D766–D773 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30357393)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Clu7rM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GENCODE%20reference%20annotation%20for%20the%20human%20and%20mouse%20genomes&journal=Nucleic%20Acids%20Res.&volume=47&pages=D766-D773&publication_year=2019&author=Frankish%2CA) 
    
56.  Frankish, A. et al. GENCODE 2021. _Nucleic Acids Res._ **49**, D916–D923 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33270111)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXntlejtbY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GENCODE%202021&journal=Nucleic%20Acids%20Res.&volume=49&pages=D916-D923&publication_year=2021&author=Frankish%2CA) 
    
57.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
58.  Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. _GigaScience_ **4**, 7 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25722852)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&journal=GigaScience&volume=4&publication_year=2015&author=Chang%2CCC) 
    
59.  Li, H. et al. The Sequence Alignment/Map format and SAMtools. _Bioinformatics_ **25**, 2078–2079 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19505943)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&journal=Bioinformatics&volume=25&pages=2078-2079&publication_year=2009&author=Li%2CH) 
    
60.  Danecek, P. et al. The variant call format and VCFtools. _Bioinformatics_ **27**, 2156–2158 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21653522)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137218)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXptFCqt7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20variant%20call%20format%20and%20VCFtools&journal=Bioinformatics&volume=27&pages=2156-2158&publication_year=2011&author=Danecek%2CP) 
    
61.  Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. _Nat. Genet._ **47**, 1228–1235 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26414678)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4626285)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7rF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Partitioning%20heritability%20by%20functional%20annotation%20using%20genome-wide%20association%20summary%20statistics&journal=Nat.%20Genet.&volume=47&pages=1228-1235&publication_year=2015&author=Finucane%2CHK) 
    
62.  Hujoel, M. L. A., Gazal, S., Hormozdiari, F., van de Geijn, B. & Price, A. L. Disease heritability enrichment of regulatory elements is concentrated in elements with ancient sequence age and conserved function across species. _Am. J. Hum. Genet._ **104**, 611–624 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30905396)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6451699)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvVWhsLo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disease%20heritability%20enrichment%20of%20regulatory%20elements%20is%20concentrated%20in%20elements%20with%20ancient%20sequence%20age%20and%20conserved%20function%20across%20species&journal=Am.%20J.%20Hum.%20Genet.&volume=104&pages=611-624&publication_year=2019&author=Hujoel%2CMLA&author=Gazal%2CS&author=Hormozdiari%2CF&author=Geijn%2CB&author=Price%2CAL) 
    
63.  Wen, X. Molecular QTL discovery incorporating genomic annotations using Bayesian false discovery rate control. _Ann. Appl. Stat._ **10**, 1619–1638 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20QTL%20discovery%20incorporating%20genomic%20annotations%20using%20Bayesian%20false%20discovery%20rate%20control&journal=Ann.%20Appl.%20Stat.&volume=10&pages=1619-1638&publication_year=2016&author=Wen%2CX) 
    
64.  Liu, Z. et al. Annotation Query (AnnoQ): an integrated and interactive platform for large-scale genetic variant annotation. _Nucleic Acids Res._ **50**, W57–W65 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35640593)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9252745)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXpslWitLg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Annotation%20Query%20%28AnnoQ%29%3A%20an%20integrated%20and%20interactive%20platform%20for%20large-scale%20genetic%20variant%20annotation&journal=Nucleic%20Acids%20Res.&volume=50&pages=W57-W65&publication_year=2022&author=Liu%2CZ) 
    
65.  Li, D. et al. WashU Epigenome Browser update 2022. _Nucleic Acids Res._ **50**, W774–W781 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35412637)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9252771)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXptlKjs7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WashU%20Epigenome%20Browser%20update%202022&journal=Nucleic%20Acids%20Res.&volume=50&pages=W774-W781&publication_year=2022&author=Li%2CD) 
    
66.  Kramer, N. E. et al. Plotgardener: cultivating precise multi-panel figures in R. _Bioinformatics_ **38**, 2042–2045 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35134826)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8963281)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFygs7jM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plotgardener%3A%20cultivating%20precise%20multi-panel%20figures%20in%20R&journal=Bioinformatics&volume=38&pages=2042-2045&publication_year=2022&author=Kramer%2CNE) 
    
67.  Hudson, R. R., Slatkin, M. & Maddison, W. P. Estimation of levels of gene flow from DNA sequence data. _Genetics_ **132**, 583–589 (1992).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1427045)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205159)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXhsVSku7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estimation%20of%20levels%20of%20gene%20flow%20from%20DNA%20sequence%20data&journal=Genetics&volume=132&pages=583-589&publication_year=1992&author=Hudson%2CRR&author=Slatkin%2CM&author=Maddison%2CWP) 
    
68.  Bhatia, G., Patterson, N., Sankararaman, S. & Price, A. L. Estimating and interpreting FST: the impact of rare variants. _Genome Res._ **23**, 1514–1521 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23861382)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3759727)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVOls7fN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estimating%20and%20interpreting%20FST%3A%20the%20impact%20of%20rare%20variants&journal=Genome%20Res.&volume=23&pages=1514-1521&publication_year=2013&author=Bhatia%2CG&author=Patterson%2CN&author=Sankararaman%2CS&author=Price%2CAL) 
    
69.  Lu, Z. et al. Improved multi-ancestry fine-mapping identifies _cis_\-regulatory variants underlying molecular traits and disease risk. _Zenodo_ [https://doi.org/10.5281/zenodo.10963033](https://doi.org/10.5281/zenodo.10963033) (2025).
    
70.  Zeng, T., Spence, J. P., Mostafavi, H. & Pritchard, J. K.and found highly significantraint from an evolutionary model with gene features. _Nat. Genet._ **56**, 1632–1643 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38977852)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFWhsbvN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=K.and%20found%20highly%20significantraint%20from%20an%20evolutionary%20model%20with%20gene%20features&journal=Nat.%20Genet.&volume=56&pages=1632-1643&publication_year=2024&author=Zeng%2CT&author=Spence%2CJP&author=Mostafavi%2CH&author=Pritchard%2CJ) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02262-7?format=refman&flavour=references)

Acknowledgements
----------------

We thank members of the Mancuso and Gazal laboratories for fruitful discussions regarding this Article. We also thank M. D. Edge for his thoughtful comments and suggestions. This work was funded in part by National Institutes of Health (NIH) under awards R01HG012133 (N.M.), R01CA258808 (N.M.), R01GM140287 (P.M.), R35GM142783 (N.M.), R01GM140287 (P.M.), U54HG013243 (L.W.), R35GM147789 (S.G.), K08HL159346 (J.P.), R00CA246076 (L.K.) and R01MH125252 (A.G.). MESA phenotypes (dbGaP: phs000209.v13.p3): MESA and the MESA SHARe project are conducted and supported by the National Heart, Lung, and Blood Institute (NHLBI) in collaboration with MESA investigators. Support for MESA is provided by contracts HHSN268201500003I, N01-HC-95159, N01-HC-95160, N01-HC-95161, N01-HC-95162, N01-HC-95163, N01-HC95164, N01-HC-95165, N01-HC-95166, N01-HC-95167, N01-HC-95168, N01-HC-95169, UL1-TR-001079, UL1-TR000040, UL1-TR-001420, UL1-TR-001881 and DK063491. Funding for SHARe genotyping was provided by NHLBI Contract N02-HL-64278. TOPMed-MESA WGS genotype, mRNA and protein expression data (dbGaP: phs001416.v3.p1): molecular data for the Trans-Omics in Precision Medicine (TOPMed) program was supported by the National Heart, Lung and Blood Institute (NHLBI). WGS genotype data for NHLBI TOPMed: MESA (phs001416.v3.p1) was performed at Broad Genomics (HHSN268201600034I). mRNA expression data for NHLBI TOPMed: MESA (phs001416.v3.p1) was performed at NWGC (HHSN268201600032I). SOMAscan proteomics for NHLBI TOPMed: Multi-Ethnic Study of Atherosclerosis (MESA) (phs001416.v1.p1) was performed at the Broad Institute and Beth Israel Proteomics Platform (HHSN268201600034I). Core support including centralized genomic read mapping and genotype calling, along with variant quality metrics and filtering were provided by the TOPMed Informatics Research Center (3R01HL-117626-02S1; contract HHSN268201800002I). Core support including phenotype harmonization, data management, sample-identity quality control, and general program coordination were provided by the TOPMed Data Coordinating Center (R01HL-120393; U01HL-120393; contract HHSN268201800001I). We gratefully acknowledge the studies and participants who provided biological samples and data for TOPMed. GENOA genotype (dbGaP: phs001238.v2.p1) and gene expression (GEO: GSE138914) data were supported by grants from NIH NHLBI (HL054457, HL054464, HL054481, HL119443 and HL087660). We acknowledge S. Kardia and J. Smith in preparing GENOA eQTL data. The All of Us Research Program is supported by the National Institutes of Health, Office of the Director: Regional Medical Centers: 1 OT2 OD026549; 1 OT2 OD026554; 1 OT2 OD026557; 1 OT2 OD026556; 1 OT2 OD026550; 1 OT2 OD 026552; 1 OT2 OD026553; 1 OT2 OD026548; 1 OT2 OD026551; 1 OT2 OD026555; IAA #: AOD 16037; Federally Qualified Health Centers: HHSN 263201600085U; Data and Research Center: 5 U2C OD023196; Biobank: 1 U24 OD023121; The Participant Center: U24 OD023176; Participant Technology Systems Center: 1 U24 OD023163; Communications and Engagement: 3 OT2 OD023205; 3 OT2 OD023206; and Community Partners: 1 OT2 OD025277; 3 OT2 OD025315; 1 OT2 OD025337; 1 OT2 OD025276. In addition, the All of Us Research Program would not be possible without the partnership of its participants.

Author information
------------------

### Authors and Affiliations

1.  Center for Genetic Epidemiology, Keck School of Medicine, University of Southern California, Los Angeles, CA, USA
    
    Zeyun Lu, Xinran Wang, Matthew Carr, Artem Kim, Steven Gazal & Nicholas Mancuso
    
2.  Department of Population and Public Health Sciences, Keck School of Medicine, University of Southern California, Los Angeles, CA, USA
    
    Zeyun Lu, Xinran Wang, Steven Gazal & Nicholas Mancuso
    
3.  Department of Medical Oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA, USA
    
    Zeyun Lu & Alexander Gusev
    
4.  Department of Quantitative and Computational Biology, University of Southern California, Los Angeles, CA, USA
    
    Steven Gazal & Nicholas Mancuso
    
5.  Center for Immunity and Immunotherapies, Seattle Children’s Research Institute, Seattle, WA, USA
    
    Pejman Mohammadi
    
6.  Department of Pediatrics, University of Washington School of Medicine, Seattle, WA, USA
    
    Pejman Mohammadi
    
7.  Department of Genome Sciences, University of Washington, Seattle, WA, USA
    
    Pejman Mohammadi
    
8.  Cancer Epidemiology Division, Population Sciences in the Pacific Program, University of Hawaiʻi Cancer Center, University of Hawaiʻi at Mānoa, Honolulu, HI, USA
    
    Lang Wu
    
9.  Division of Cardiology, University of California San Francisco, San Francisco, CA, USA
    
    James Pirruccello
    
10.  Department of Epidemiology and Population Health, Stanford University School of Medicine, Stanford, CA, USA
    
    Linda Kachuri
    
11.  Stanford Cancer Institute, Stanford University School of Medicine, Stanford, CA, USA
    
    Linda Kachuri
    

Authors

1.  Zeyun Lu
    
    [View author publications](https://www.nature.com/search?author=Zeyun%20Lu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zeyun%20Lu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zeyun%20Lu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Xinran Wang
    
    [View author publications](https://www.nature.com/search?author=Xinran%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xinran%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xinran%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Matthew Carr
    
    [View author publications](https://www.nature.com/search?author=Matthew%20Carr)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20Carr) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20Carr%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Artem Kim
    
    [View author publications](https://www.nature.com/search?author=Artem%20Kim)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Artem%20Kim) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Artem%20Kim%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Steven Gazal
    
    [View author publications](https://www.nature.com/search?author=Steven%20Gazal)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Steven%20Gazal) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Steven%20Gazal%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Pejman Mohammadi
    
    [View author publications](https://www.nature.com/search?author=Pejman%20Mohammadi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Pejman%20Mohammadi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Pejman%20Mohammadi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Lang Wu
    
    [View author publications](https://www.nature.com/search?author=Lang%20Wu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lang%20Wu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lang%20Wu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  James Pirruccello
    
    [View author publications](https://www.nature.com/search?author=James%20Pirruccello)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=James%20Pirruccello) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22James%20Pirruccello%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Linda Kachuri
    
    [View author publications](https://www.nature.com/search?author=Linda%20Kachuri)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Linda%20Kachuri) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Linda%20Kachuri%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Alexander Gusev
    
    [View author publications](https://www.nature.com/search?author=Alexander%20Gusev)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alexander%20Gusev) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alexander%20Gusev%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Nicholas Mancuso
    
    [View author publications](https://www.nature.com/search?author=Nicholas%20Mancuso)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nicholas%20Mancuso) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nicholas%20Mancuso%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Z.L. and N.M. developed the model and study design. Z.L. performed simulations and fine-mapping analyses. Z.L., X.W., J.P. and L.K. performed TWAS and AoU analyses. Z.L., M.C. and N.M. developed the model and inference scheme. Z.L. and A.K. prepared functional genomic annotations and performed heritability enrichment analyses. Z.L. and N.M. wrote the initial paper. Z.L., X.W., M.C., A.K., S.G., P.M., L.W., J.P., L.K., A.G. and N.M. edited the final paper.

### Corresponding authors

Correspondence to [Zeyun Lu](mailto:zeyunlu@usc.edu) or [Nicholas Mancuso](mailto:nicholas.mancuso@med.usc.edu).

Ethics declarations
-------------------

### Competing interests

L.W. provided consulting service to Pupil Bio Inc. and reviewed manuscripts for Gastroenterology Report, not related to this study, and received honorarium. S.G. received consulting fees from Eleven Therapeutics unrelated to this work. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Biao Zeng and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02262-7#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_MOESM1_ESM.pdf)

Supplementary Notes 1 and 2, and Figs. 1–39.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02262-7/MediaObjects/41588_2025_2262_MOESM4_ESM.xlsx)

Supplementary Tables 1–15.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Improved%20multiancestry%20fine-mapping%20identifies%20cis-regulatory%20variants%20underlying%20molecular%20traits%20and%20disease%20risk&author=Zeyun%20Lu%20et%20al&contentID=10.1038%2Fs41588-025-02262-7&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-07-21&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02262-7)

### Cite this article

Lu, Z., Wang, X., Carr, M. _et al._ Improved multiancestry fine-mapping identifies _cis_\-regulatory variants underlying molecular traits and disease risk. _Nat Genet_ **57**, 1881–1889 (2025). https://doi.org/10.1038/s41588-025-02262-7

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02262-7?format=refman&flavour=citation)

-   Received: 16 April 2024
    
-   Accepted: 09 June 2025
    
-   Published: 21 July 2025
    
-   Issue Date: August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02262-7
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
