> id: https://www.nature.com/articles/s41588-025-02273-4

> link: https://www.nature.com/articles/s41588-025-02273-4

> title: Haplotype-resolved gap-free genome assemblies provide insights into the divergence between Asian and European pears

# Haplotype-resolved gap-free genome assemblies provide insights into the divergence between Asian and European pears
_Published on Wed, 06 Aug 2025 00:00:00 GMT_

Abstract
--------

Pears (_Pyrus_ spp.) are self-incompatible crops with broad genetic diversity. High heterozygosity and technical limitations result in gaps within reference genomes. Our study presents the telomere-to-telomere, haplotype-resolved genomes of a representative Asian pear ‘Dangshansuli’ (_Pyrus bretschneideri_) and a European pear ‘Max Red Bartlett’ (_Pyrus communis_). Haplotype-specific genes exhibited notable differences from biallelic genes regarding transposable content, methylation patterns and expression levels. Allele-specific expression analysis suggested that the dominance effect is vital in the formation of fruit quality of pears. Population analysis of 362 accessions revealed that interspecific introgression increased pear diversity. We constructed a graph-based genome and identified structural variations associated with agronomic traits. A 286-bp insertion in the promoter region, along with differential expression of _PyACS1_, was identified between Asian and European pears, which exhibit distinct fruit-softening characteristics. Further experiments demonstrated the role of _PyACS1_ in fruit softening. Overall, this study provided insights into genetic variation and will facilitate pear improvement.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41597-025-05380-3/MediaObjects/41597_2025_5380_Fig1_HTML.png)

### [Haplotype-resolved T2T genome assembly of the pear cultivar ‘Danxiahong’](https://www.nature.com/articles/s41597-025-05380-3?fromPaywallRec=false)

Article Open access 18 June 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41597-024-04015-3/MediaObjects/41597_2024_4015_Fig1_HTML.png)

### [Telomere-to-telomere Genome Assembly of two representative Asian and European pear cultivars](https://www.nature.com/articles/s41597-024-04015-3?fromPaywallRec=false)

Article Open access 26 October 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-024-77532-1/MediaObjects/41598_2024_77532_Fig1_HTML.jpg)

### [Unlocking Spanish pear genetic diversity: strategies for construction of a national core collection](https://www.nature.com/articles/s41598-024-77532-1?fromPaywallRec=false)

Article Open access 04 November 2024

Main
----

Many vital crops display self-incompatibility (SI); examples are potato[1](https://www.nature.com/articles/s41588-025-02273-4#ref-CR1 "Zhou, Q. et al. Haplotype-resolved genome analyses of a heterozygous diploid potato. Nat. Genet. 52, 1018–1023 (2020)."), cabbage[2](https://www.nature.com/articles/s41588-025-02273-4#ref-CR2 "Murase, K. et al. Mechanism of self/nonself-discrimination in Brassica self-incompatibility. Nat. Commun. 11, 4916 (2020).") and pear[3](https://www.nature.com/articles/s41588-025-02273-4#ref-CR3 "Chen, J. et al. Phosphatidic acid counteracts S-RNase signaling in pollen by stabilizing the actin cytoskeleton. Plant Cell 30, 1023–1039 (2018)."). SI prevents inbreeding depression and maintains the genetic diversity of species, which can lead to high levels of genomic heterozygosity. The hybridization of different parents can produce offspring that show superior performance for several traits[4](https://www.nature.com/articles/s41588-025-02273-4#ref-CR4 "Huang, X. H. et al. Genomic architecture of heterosis for yield traits in rice. Nature 537, 629–633 (2016)."). However, this heterozygosity increases the complexity of the genome assembly and most assembled genomes represent a consensus consisting of haplotype mixtures that restrict investigations into allelic variations that underlie important traits at the whole-genome level. In recent years, more haplotype-resolved plant genomes have been assembled from species such as apple[5](https://www.nature.com/articles/s41588-025-02273-4#ref-CR5 "Sun, X. P. et al. Phased diploid genome assemblies and pan-genomes provide insights into the genetic history of apple domestication. Nat. Genet. 52, 1423–1432 (2020)."), tea[6](https://www.nature.com/articles/s41588-025-02273-4#ref-CR6 "Zhang, X. T. et al. Haplotype-resolved genome assembly provides insights into evolutionary history of the tea plant Camellia sinensis. Nat. Genet. 53, 1250–1259 (2021)."), lychee[7](https://www.nature.com/articles/s41588-025-02273-4#ref-CR7 "Hu, G. B. et al. Two divergent haplotypes from a highly heterozygous lychee genome suggest independent domestication events for early and late-maturing cultivars. Nat. Genet. 54, 73–83 (2022)."), potato[1](https://www.nature.com/articles/s41588-025-02273-4#ref-CR1 "Zhou, Q. et al. Haplotype-resolved genome analyses of a heterozygous diploid potato. Nat. Genet. 52, 1018–1023 (2020)."), cassava[8](https://www.nature.com/articles/s41588-025-02273-4#ref-CR8 "Hu, W. et al. Allele-defined genome reveals biallelic differentiation during cassava evolution. Mol. Plant 14, 851–854 (2021).") and sugarcane[9](https://www.nature.com/articles/s41588-025-02273-4#ref-CR9 "Zhang, Q. et al. Genomic insights into the recent chromosome reduction of autopolyploid sugarcane Saccharum spontaneum. Nat. Genet. 54, 885–896 (2022)."). These assemblies have facilitated the identification of structural variations (SVs) and allele-specific expression patterns between haplotypes and have contributed to understanding the biological characteristics of highly heterozygous species.

Pear is an SI diploid species and is one of the most economically important fruit species worldwide. Generally, pears can be divided into two main groups, Asian and European pears, which showed distinct phenotypic difference including fruit shape, sugar, acid, flavor and softening[10](https://www.nature.com/articles/s41588-025-02273-4#ref-CR10 "Wu, J. et al. Diversification and independent domestication of Asian and European pears. Genome Biol. 19, 77 (2018)."). The first Asian pear reference genome from the _Pyrus bretschneideri_ cultivar ‘Dangshansuli’ was released in 2013 (ref. [11](https://www.nature.com/articles/s41588-025-02273-4#ref-CR11 "Wu, J. et al. The genome of the pear (Pyrus bretschneideri Rehd.). Genome Res. 23, 396–408 (2013).")), followed by the European pear _Pyrus communis_ ‘Bartlett’ genomes in 2014 (ref. [12](https://www.nature.com/articles/s41588-025-02273-4#ref-CR12 "Chagne, D. et al. The draft genome dequence of European pear (Pyrus communis L. ‘Bartlett’). PLoS ONE 9, e92644 (2014).")) and 2019 (ref. [13](https://www.nature.com/articles/s41588-025-02273-4#ref-CR13 "Linsmith, G. et al. Pseudo-chromosome-length genome assembly of a double haploid ‘Bartlett’ pear (Pyrus communis L.). Gigascience 8, giz138 (2019).")). In 2019, a phased diploid Asian pear genome was assembled using single-pollen cell sequencing technology[14](https://www.nature.com/articles/s41588-025-02273-4#ref-CR14 "Shi, D. Q. et al. Single-pollen-cell sequencing for gamete-based phased diploid genome assembly in plants. Genome Res. 29, 1889–1899 (2019)."). Due to sequencing and assembly technology limitations, these earlier pear reference genomes were incomplete and lacked a large amount of genomic information. Recently, the nearly complete telomere-to-telomere (T2T) genomes of ‘Dangshansuli’ and ‘Conference’ (_P. communis_) were released using a mosaic assembly strategy[15](https://www.nature.com/articles/s41588-025-02273-4#ref-CR15 "Qi, Y. et al. Telomere-to-telomere genome assembly of two representative Asian and European pear cultivars. Sci. Data 11, 1170 (2024)."). However, the mosaic genome assembly restricted analyses to variations between two haplotypes. In addition, population structure and domestication in pear have been extensively discussed recently[10](https://www.nature.com/articles/s41588-025-02273-4#ref-CR10 "Wu, J. et al. Diversification and independent domestication of Asian and European pears. Genome Biol. 19, 77 (2018)."),[16](https://www.nature.com/articles/s41588-025-02273-4#ref-CR16 "Chen, X. et al. Genome-wide genetic diversity and IBD analysis reveals historic dissemination routes of pear in China. Tree Genet Genomes 18, 1 (2022)."). Nevertheless, the complex evolutionary history and uncertain phylogeny, especially the pattern of reticulate evolution with close wild relatives, remain to be examined. In addition, the high-quality T2T genomes provide the opportunity to construct a graph-based genome, which is a powerful tool for SV genotyping using next-generation sequencing (NGS) data[17](https://www.nature.com/articles/s41588-025-02273-4#ref-CR17 "Zhou, Y. et al. Graph pangenome captures missing heritability and empowers tomato breeding. Nature 606, 527–534 (2022).").

In this study, we constructed haplotype-resolved, T2T genome assemblies for the cultivars ‘Dangshansuli’ (DS) and ‘Max Red Bartlett’ (MRB; red bud mutation of ‘Bartlett’), which are representative of Asian and European pears, respectively. The assemblies are improved with regard to genome completeness and accuracy and provide opportunities to analyze allelic and interspecific variations. Resequencing data for 362 pear accessions were collected to investigate the evolutionary and demographic history of these pear groups. We constructed a graph-based genome by combining the two high-quality genomes with the resequencing data. SVs exhibiting significant frequency variations between Asian and European pears have been identified. We further explored the association between SVs and fruit traits and between SVs and gene expression. One 286-bp insertion sequence in the promoter region of _PyACS1_ may have contributed to changes in fruit maturity and softening in European pear fruit. The results of this study advance our understanding of heterosis, evolutionary history and critical genetic variation in the pear phenotype and will provide valuable resources for pear breeding.

Results
-------

### Haplotype-resolved T2T assemblies of pears

In view of the heterozygosity of _P. bretschneideri_ DS (1.23%) and _P. communis_ MRB (1.33%), two haplotypes of the preliminary diploid genomes were assembled using PacBio HiFi, ONT (Oxford Nanopore Technologies) ultra-long, and Hi-C data with hifiasm[18](https://www.nature.com/articles/s41588-025-02273-4#ref-CR18 "Cheng, H. Y. et al. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. Nat. Methods 18, 170–175 (2021)."). Then, Verkko[19](https://www.nature.com/articles/s41588-025-02273-4#ref-CR19 "Rautiainen, M. et al. Telomere-to-telomere assembly of diploid chromosomes with Verkko. Nat. Biotechnol. 41, 1474–1482 (2023).") and HiCanu[20](https://www.nature.com/articles/s41588-025-02273-4#ref-CR20 "Nurk, S. et al. HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads. Genome Res. 30, 1291–1305 (2020).") assembly results were used to fill the gaps and telomeric regions (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02273-4#MOESM3) and [Methods](https://www.nature.com/articles/s41588-025-02273-4#Sec11)). All gap-closure regions were supported with uniform coverage of ONT and HiFi reads (Supplementary Figs. [1](https://www.nature.com/articles/s41588-025-02273-4#MOESM1), [2](https://www.nature.com/articles/s41588-025-02273-4#MOESM1) and [3](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Finally, each haplotype was assembled into 17 contigs corresponding to the 17 pear chromosomes with genome sizes ranging from 497.50 Mb to 505.55 Mb and contig N50 sizes ≥28.96 Mb (Table [1](https://www.nature.com/articles/s41588-025-02273-4#Tab1)). The sequence 18.89 Mb to 24.13 Mb was not assembled in released _P. bretschneideri_ and _P. communis_ genomes (Supplementary Fig. [4a](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). The contig N50 size, mapping rate, Benchmarking Universal Single-copy Orthologs (BUSCO) and Long Terminal Repeat (LTR) Assembly Index supported the high quality of the assemblies (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). We identified 34 telomeres on the terminal ends of the 17 chromosomes of the haplotypes in DS and 33 in MRB, using a telomere repeat sequence (TTTAGGG) as the query (Fig. [1a,b](https://www.nature.com/articles/s41588-025-02273-4#Fig1)). In summary, we successfully assembled T2T haplotype-resolved genomes for DS and MRB.

**Table 1 Summary of genome assembly and annotation of DS and MRB**

**Fig. 1: DS and MRB genome assemblies.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02273-4/figures/1)

**a**,**b**, Genomic features across haplotype A and B of DS (**a**) and MRB (**b**): (I) telomeres and centromeres; (II) TE density (number of TEs per 200 kb); (III) gene density (number of genes per 200 kb); (IV) SNP density (number of SNPs per 200 kb); and (V) GC content per 200 kb. ‘A’ means HapA and ‘B’ HapB. **c**, _S-RNase_ and _SFB_ (S-locus-related _F-box_ gene) distribution in HapA and HapB of DS and MRB. The top graphs show gene number and TE percentage (per 50-kb window size and 20-kb step size) and the bottom graphs show the distribution of _S-RNase_; _SFB_ and other genes and the genotype of _S-RNase_ are marked. **d**, Orthologous gene pairs at an S-locus region across four haplotypes. The red line depicts orthologous _F-box_ gene pairs, the yellow line the _S-RNase_ gene pair and the green line other gene pairs.

The DS and MRB haplotypes contained 262.86–272.23 Mb of transposable elements (TEs) (Table [1](https://www.nature.com/articles/s41588-025-02273-4#Tab1)) and LTRs comprised the most abundant category of TEs (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)). A total of 42,675–43,602 protein-coding genes were predicted in the haplotypes of DS and MRB (Table [1](https://www.nature.com/articles/s41588-025-02273-4#Tab1)), with the BUSCO completeness ranging from 97.90% to 98.60% (Supplementary Fig. [7](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). The high-quality assembly and annotation results provided an opportunity to investigate the complex regions, including centromeres and S-locus alleles (Fig. [1c,d](https://www.nature.com/articles/s41588-025-02273-4#Fig1) and Supplementary Note [2](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). In addition, 577 (563) genes were identified in newly assembled regions of haplotype A (HapA) (haplotype B (HapB)) in DS and 697 (804) were in HapA (HapB) of MRB. These genes were predominantly (77.33–84.06%) involved in the regions with high TE content and high divergence between the two haplotypes (Supplementary Fig. [4b](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Within the newly assembled regions, we identified cell division cycle protein 48, a key regulator of plant development and stress response[21](https://www.nature.com/articles/s41588-025-02273-4#ref-CR21 "Ines, D. et al. CDC48 in plants and its emerging function in plant immunity. Trends Plant Sci. 29, 786–798 (2024)."), and UDP-glucose flavonoid 3-_O_\-glucosyltransferase 7, which is critical for sugar decomposition in strawberries[22](https://www.nature.com/articles/s41588-025-02273-4#ref-CR22 "Griesser, M. et al. Multi-substrate flavonol O-glucosyltransferases from strawberry (Fragaria x ananassa) achene and receptacle. J. Exp. Bot. 59, 2611–2625 (2008).").

### Identifying haplotype-specific and ASE genes

A long hybrid history has resulted in extensive genetic variations (Supplementary Note [3](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)) and alleles between the two haplotypes in the pear. Based on the presence of ortholog genes in corresponding haplotypes, 33,710–34,036 allelic genes without the same coding sequence (CDS) were defined as biallelic genes (Fig. [2a,b](https://www.nature.com/articles/s41588-025-02273-4#Fig2)). Of these genes, 6,691–8,087 contained partners with the same CDS in corresponding haplotypes (alleles with the same CDS) (Supplementary Table [11](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)). We identified 1,703–1,994 genes in a haplotype with no allelic gene (haplotype-specific genes). Haplotype-specific genes had higher abundances of TEs compared with biallelic genes in the 5-kb upstream and downstream regions and gene body regions (Fig. [2c](https://www.nature.com/articles/s41588-025-02273-4#Fig2)). The mean expression levels of haplotype-specific genes were lower than those of biallelic genes (or alleles with the same CDS) in different fruit samples of DS and MRB (Supplementary Fig. [14](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Our analysis revealed that haplotype-specific genes exhibited higher DNA methylation levels in both the gene body and the regions spanning ±5 kb from the gene in leaf and mature fruit tissues of DS and MRB (Supplementary Fig. [15](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). This suggested stronger repression by methylation in these areas. These findings were in line with previous research, which generally regarded DNA methylation as a regulatory factor for gene expression[23](https://www.nature.com/articles/s41588-025-02273-4#ref-CR23 "Weber, M. et al. Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome. Nat. Genet. 39, 457–466 (2007)."),[24](https://www.nature.com/articles/s41588-025-02273-4#ref-CR24 "Stroud, H. et al. Comprehensive analysis of silencing mutants reveals complex regulation of the Arabidopsis methylome. Cell 152, 352–364 (2013)."). The haplotype-specific genes were enriched in ‘Plant–pathogen interaction’ and ‘Environmental adaptation’ in the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways (Supplementary Fig. [16](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)) and the ‘Response to biotic stimulus’ gene ontology (GO) terms (Supplementary Fig. [17](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)).

**Fig. 2: Allelic gene variation and expression pattern in DS and MRB.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02273-4/figures/2)

**a**, The layout for allele gene classification. The alleles were divided into the haplotype-specific genes (no paired genes in the corresponding haplotype), the alleles with the same CDS and biallelic genes. The star with the dotted line represents this gene not existing. **b**, Proportion of biallelic, allele with the same CDS and haplotype-specific genes in each haplotype. **c**, The TE distribution across gene body, 5-kb upstream and 5-kb downstream regions of biallelic, allele with the same CDS and haplotype-specific genes in HapA and HapB of DS and MRB. **d**, Number of ASE genes in eight fruit stages (including five developmental and three postharvest stages) of DS and MRB. **e**, The ASE gene type alteration between DS and MRB. HapDom indicates haplotype dominance gene pair. Sub denotes sub- or neofunctionalized pair and other ASE genes (Nodiff). NoASE denotes that the gene pair did not have ASE. **f**, The density of _K_s and _K_a/_K_s values of ASE genes. **g**, The SNP density in gene features of HapA in DS (_nHapDom_ = 4,498, _nSub_ = 219, _nNodiff_ = 3,794). The center line is the median, the box limits the first and third quartiles and the whiskers 1.5× the interquartile range (IQR). The terms _nHapDom_, _nSub_ and _nNodiff_ refer to the counts of different ASE genes. The _y_ axis represents SNP numbers every 1,000 bp. The two-sided Student’s _t_\-test was used for determining the significance. **h**, Expression pattern of the two alleles of _SWEET2_ and _C3H_ in fruit samples of DS and MRB. TES, transcription end site; TSS, transcription start site.

We further investigated allele-specific expression (ASE) during the eight fruit stages of DS and MRB: 19.64% (8,511 of 43,330) and 26.21% (11,213 of 42,784) of genes showed ASE in at least one sample of DS and MRB (Fig. [2d](https://www.nature.com/articles/s41588-025-02273-4#Fig2)). These ASE genes were further classified into three categories ([Methods](https://www.nature.com/articles/s41588-025-02273-4#Sec11)): 4,498 (4,953) haplotype dominance (HapDom, genes with expression biased toward one allele), 219 (527) Sub (sub- or neofunctionalized pairs, alleles with switched expression in different samples) and 3,794 (5,733) other ASE genes (Nodiff) in DS (MRB). Only 44.52% of ASE genes in DS also exhibited an ASE pattern in MRB (Fig. [2e](https://www.nature.com/articles/s41588-025-02273-4#Fig2)). Of ASEs in DS, 22.12% maintained a consistent category with ASEs in MRB and 22.40% ASEs in DS showed an inconsistent category with ASEs in MRB. Enrichment results showed that these inconsistent ASEs between DS and MRB were enriched in biosynthesis and metabolism pathways and consistent ASEs were enriched in response to stress and different stimulus GO terms (Supplementary Fig. [18](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)).

We annotated 380 (in DS) and 450 (in MRB) ASE genes, including sugar-related fructokinases, α-glucosidases and sugar transporters, acid-related malate dehydrogenases, flavonoid-related flavanol synthases, lignin related _p_\-coumarate 3-hydroxylase (C3H), 4-coumarate: coenzyme A ligase (4CL) and cinnamoyl-CoA reductase, which might contribute to important fruit agronomic traits at the allele level (Supplementary Fig. [20a](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Among the ASE genes related to fruit traits, we observed that 95.26% (362 of 380) of the genes in DS and 88.89% (400 of 450) of the genes in MRB exhibited a haplotype dominance expression pattern, indicating that one allele had a greater influence on fruit quality and development than its counterpart. For example, SWEET2 (sugars will eventually be exported transporters 2) and CPK28 (calcium-dependent protein kinase 28) transport and increase sugar content[25](https://www.nature.com/articles/s41588-025-02273-4#ref-CR25 "Li, J. et al. Natural variations in the PbCPK28 promoter regulate sugar content through interaction with PbTST4 and PbVHA-A1 in pear. Plant J. 114, 124–141 (2023)."),[26](https://www.nature.com/articles/s41588-025-02273-4#ref-CR26 "Lama, K. et al. Extreme sugar accumulation in late fig ripening is accompanied by global changes in sugar metabolism and transporter gene expression. Physiol. Plant 174, e13648 (2022).") and showed a dominant expression pattern in DS and MRB (Fig. [2h](https://www.nature.com/articles/s41588-025-02273-4#Fig2) and Supplementary Fig. [20b](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). _Pbre1\_15G769200_ and _Pbre1\_13G264800_, which encode C3H and 4CL protein, respectively, are major structural genes involved in lignin biosynthesis[27](https://www.nature.com/articles/s41588-025-02273-4#ref-CR27 "Xue, C. et al. PbrMYB169 positively regulates lignification of stone cells in pear fruit. J. Exp. Bot. 70, 1801–1814 (2019)."),[28](https://www.nature.com/articles/s41588-025-02273-4#ref-CR28 "Barros, J. et al. 4-Coumarate 3-hydroxylase in the lignin biosynthesis pathway is a cytosolic ascorbate peroxidase. Nat. Commun. 10, 1994 (2019)."). These genes exhibited higher expression levels during the early fruit developmental stages (S1–S2) compared with their counterparts in DS HapB (Fig. [2h](https://www.nature.com/articles/s41588-025-02273-4#Fig2) and Supplementary Fig. [20b](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)).

### Genomic evolution between Asian and European pears

To explore the evolutionary history of Asian and European pears, the protein-coding genes of DS, MRB and ten other species were used in comparative analyses (Fig. [3a](https://www.nature.com/articles/s41588-025-02273-4#Fig3)). Phylogenetic analysis indicated that DS and MRB diverged approximately 3.60 million years ago (Ma). Compared with the common ancestor, 767 and 740 gene families were expanded in DS and MRB. The expanded gene families in DS were enriched in ‘Biosynthesis of plant secondary metabolites’ and ‘Plant–pathogen interaction’ pathways and ‘Response to stress’ GO terms (Supplementary Fig. [21a](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)), which are important for plant development and adaptation[29](https://www.nature.com/articles/s41588-025-02273-4#ref-CR29 "Yang, C. Q. et al. Transcriptional regulation of plant secondary metabolism. J. Integr. Plant Biol. 54, 703–712 (2012)."),[30](https://www.nature.com/articles/s41588-025-02273-4#ref-CR30 "Dodds, P. N. & Rathjen, J. P. Plant immunity: towards an integrated view of plant-pathogen interactions. Nat. Rev. Genet. 11, 539–548 (2010)."). The ‘Fructose and mannose metabolism’ and ‘Carbohydrate metabolism’ pathways and ‘Secondary metabolic process’ GO terms were featured in the expanded gene families in MRB.

**Fig. 3: Divergence and SVs between DS and MRB.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02273-4/figures/3)

**a**, Phylogenetic, molecular clock dating and gene family analysis of the DS and MRB genome with ten other species. The red oval indicates the fossil calibration point. The gain and loss of total gene families in DS, MRB and the other ten species are highlighted in green and red, respectively. **b**, Ideogram with synteny regions, SVs and inversion distribution in 200-kb bins. Synteny regions, SV and inversion distribution are plotted to the side of each chromosome. Centromeric region is plotted in chromosome representations. **c**, The proportion of SVs overlapping with TEs or different genomic regions. In the gene body, upstream 2-kb and downstream 2-kb regions of the gene body are considered genic regions and other regions are considered intergenic regions. **d**, Large inversions between two haplotypes in DS or MRB. The yellow regions on the collinear blocks represent the inversions. Hi-C heatmaps support the inversion. The dashed box represents the inversion and the arrows show the high interaction signal at the reverse diagonal. **e**, Pan-SV and core-SV count with Asian and European pear accessions. Accessions were sampled as 1,000 random combinations of each given number of accessions. **f**, The distribution and frequency density of SV genotype between Asian and European pears. The circos plot shows the distribution of the frequency variance between Asian and European pears: (I) the chromosomes of HapA of DS; and (II–V) frequency variance of SVs that overlapped with the upstream 2-kb (II), downstream 2-kb (III), exon (IV) and intron (V) regions. The density graph shows the distribution of SV frequency with reference genotype (HapA of DS as reference).

The extensive evolutionary divergence between Asian and European pears implies their marked genomic variations (Fig. [3b](https://www.nature.com/articles/s41588-025-02273-4#Fig3)). In this study, four haplotypes were used to construct a graph-based genome for SV identification and 103,069 SVs were successfully identified and genotyped. Of the SVs, 57.42% were in the gene body and gene-regulatory regions (±2 kb) and 61.27% overlapped with TEs (Fig. [3c](https://www.nature.com/articles/s41588-025-02273-4#Fig3)). In addition, 12 large inversions (>100 kb) were identified, spanning 5.79 Mb, and all were supported by ONT reads or Hi-C maps (Fig. [3d](https://www.nature.com/articles/s41588-025-02273-4#Fig3) and Supplementary Fig. [23](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Notably, a 0.58-Mb inversion on chromosome 17 (chr. 17) was present between DS HapA and HapB at the S-locus regions (Fig. [3d](https://www.nature.com/articles/s41588-025-02273-4#Fig3)). This inversion may suppress recombination and maintain the linkage of _SFB_ and _S-RNase_ genes for SI[31](https://www.nature.com/articles/s41588-025-02273-4#ref-CR31 "Zhu, S. et al. The snapdragon genomes reveal the evolutionary dynamics of the S-locus supergene. Mol. Biol. Evol. 40, msad080 (2023)."),[32](https://www.nature.com/articles/s41588-025-02273-4#ref-CR32 "Yang, Q. et al. Heterochromatic and genetic features are consistent with recombination suppression of the self-incompatibility locus in Antirrhinum. Plant J. 51, 140–151 (2007).").

To reveal the SVs between Asian and European pears, NGS data from 362 pear accessions were collected (Supplementary Table [15](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)) and mapped to the graph-based genome for SV genotyping. The number of genotyped SVs reached a plateau as more Asian and European samples were added (Fig. [3e](https://www.nature.com/articles/s41588-025-02273-4#Fig3)). A total of 83,369 SVs were retained with minor allele frequency >0.05 and missing rate <0.2. The reference (DS HapA as reference) presence frequency of 76.82% SVs was >0.5 in the Asian pear group (Fig. [3f](https://www.nature.com/articles/s41588-025-02273-4#Fig3)), and the frequencies of 42.73% SVs were >0.5 in the European pear group. There were 35,481 SVs that showed different presence frequency (false discovery rate (FDR) < 0.001 and fold-change >2) between the Asian and European pear groups.

### Evolutionary history and genetic introgression

We further explored the introgression events, demographic history and domestication process. The SNP-based and SV-based phylogenetic tree showed that a division of the 362 pear accessions into 2 distinct clades, with clade I comprising most Asian pears and clade II consisting of European pears (Fig. [4a](https://www.nature.com/articles/s41588-025-02273-4#Fig4) and Supplementary Fig. [27a](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Clade I was further subdivided into four subclades—(1) _P. bretschneideri_ and _Pyrus pyrifolia_; (2) Asian wild pears; (3) wild and cultivated _Pyrus ussuriensis_; and (4) _Pyrus sinkiangensis_ and interspecific hybrids—and clade II was subdivided into two subclades—(5) _P. communis_ and (6) European wild pears. High fixation statistics (_F_ST) and principal component analysis results supported a long divergence history between Asian and European pears (Fig. [4b](https://www.nature.com/articles/s41588-025-02273-4#Fig4) and Supplementary Fig. [27b,c](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Cultivated _P. ussuriensis_ (4.80 × 10−3) had higher nucleotide diversity than wild _P. ussuriensis_ (3.26 × 10−3). The structure analysis revealed the historical hybridization of the cultivated pears (Fig. [4a](https://www.nature.com/articles/s41588-025-02273-4#Fig4) and Supplementary Fig. [27d](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)) and cultivated _P. ussuriensis_ showed two genetic backgrounds, originating from wild _P. ussuriensis_ and cultivated _P. pyrifolia_ (or _P. bretschneideri_). The Treemix analysis detected gene flow from _P. bretschneideri_ to _P. ussuriensis_ (Supplementary Fig. [28](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)), suggesting that an introgression event may have played a role in _P. ussuriensis_ domestication.

**Fig. 4: Phylogenetic relationship, demographic history and selective sweeps of different pear groups.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02273-4/figures/4)

**a**, Phylogenetic tree and population structure of 362 pear accessions, inferred using SNPs at fourfold degenerate sites. _K_ is the the number of ancestry kinship. **b**, Mean nucleotide diversity in each group and population differentiation (_F_ST) between groups. The line width and color of the edges represent _F_ST values between the two groups. The circle size and value represent the nucleotide diversity of each pear group. The label of circle colors is the same with a phylogenetic tree plot. **c**, Historical effective population size _N_e for Asian wild accessions (except wild _P. ussuriensis_) with a generation time of 7.5 years. Orange, blue and yellow shadows represent three bottlenecks. **d**,**e**, Genome-wide screening of selective sweeps in AW versus AC pears (including _P. pyrifolia_ and _P. bretschneideri_) (**d**) and EW versus EC pears (_P. communis_) (**e**). The red dotted lines represent the threshold of the top 5% _F_ST and 5% ratio of _π_wild to _π_cultivar, respectively.

Inferences about the demographic history revealed three effective declines in population size (_N_e) in Asian (except wild _P. ussuriensis_) and EW pears (Fig. [4c](https://www.nature.com/articles/s41588-025-02273-4#Fig4) and Supplementary Fig. [29a](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). The first decline event occurred at 2.5–1.7 Ma during the Gelasian epoch (2.59–1.81 Ma)[33](https://www.nature.com/articles/s41588-025-02273-4#ref-CR33 "Cohen, K. M. et al. The ICS International Chronostratigraphic Chart. Episodes 36, 199–204 (2013)."), characterized by a dramatic temperature decrease. The second _N_e decline event occurred 1.0–0.6 Ma with another period of temperature decrease[34](https://www.nature.com/articles/s41588-025-02273-4#ref-CR34 "Guo, J. F. et al. Genomic clines across the species boundary between a hybrid pine and its progenitor in the eastern Tibetan Plateau. Plant Commun. 4, 100574 (2023).") and the last decline occurred 250 thousand years ago (ka) to 160 ka. It was followed by a quick demographic expansion from 150 ka to 10 ka, consistent with increasing temperatures[35](https://www.nature.com/articles/s41588-025-02273-4#ref-CR35 "Petit, J. R. et al. Climate and atmospheric history of the past 420,000 years from the Vostok ice core, Antarctica. Nature 399, 429–436 (1999)."). Only two _N_e decline events were observed in wild _P. ussuriensis_ (Supplementary Fig. [29b](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)), with the second event occurring from 380 ka. to 180 ka A slight rebound was observed thereafter, resulting in the lower _N_e of wild _P. ussuriensis_ compared with Asian wild (AW) and European wild (EW) pears.

### Selection signatures for fruit quality of pear

The cultivated pears, domesticated from wild progenitors, exhibit various phenotypical changes, including sugar, acid, stone cell and fruit size. In this study, we identified 21.31 Mb of selective sweep regions for AW versus Asian cultivated (AC, including _P. pyrifolia_ and _P. bretschneideri_) pears and 14.59 Mb of selective sweep regions for EW versus European cultivated (EC; _P. communis_) pears (Fig. [4d,e](https://www.nature.com/articles/s41588-025-02273-4#Fig4)); 123 selected genes related to fruit size, plant development, stone cell content, acids and sugars were identified in the selective sweeps (Supplementary Table [18](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)). Notably, a robust selective sweep region was identified at chr. 16 for AW versus AC pears, and fruit acid-, size- and sugar-related genes were identified, including _Ma1_ (_Pbre1\_16G081600_), _fw2.2_ (_Pbre1\_16G040000_ and _Pbre1\_16G040200_) and _SWEET15_ (_Pbre1\_16G220200_) (Fig. [4d](https://www.nature.com/articles/s41588-025-02273-4#Fig4)). The orthologs of _Ma1_ and _fw2.2_ also appeared in a selective sweep in cultivated apples[36](https://www.nature.com/articles/s41588-025-02273-4#ref-CR36 "Liao, L. et al. Unraveling a genetic roadmap for improved taste in the domesticated apple. Mol. Plant 14, 1454–1471 (2021)."), suggesting that apple and pear may experience convergent selection for fruit acid and size.

Stone cell content (SCC) is a crucial trait in pear domestication. SCC forms during the early stages of fruit development and is associated with lignin and cellulose formation[37](https://www.nature.com/articles/s41588-025-02273-4#ref-CR37 "Xu, S. et al. Auxin inhibits lignin and cellulose biosynthesis in stone cells of pear fruit via the PbrARF13-PbrNSC-PbrMYB132 transcriptional regulatory cascade. Plant Biotechnol. J. 21, 1408–1425 (2023)."). In Asian pear domestication, 11 structural genes identified in selective sweeps of AW versus AC pears were related to lignin and cellulose formation (Supplementary Table [18](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)). _PbrbZIP48_ (_Pbre1\_16G094200_), which regulates the expression of stone cell-related genes[38](https://www.nature.com/articles/s41588-025-02273-4#ref-CR38 "Gong, X. et al. Multi-omics analyses reveal stone cell distribution pattern in pear fruit. Plant J. 113, 626–642 (2023)."), was identified in the selective sweeps of AC pears (Fig. [4d](https://www.nature.com/articles/s41588-025-02273-4#Fig4)). _PbrmiR397a_, identified in the selective sweep, can silence the expression of LAC genes, thereby reducing lignin and stone cell contents in pear flesh[39](https://www.nature.com/articles/s41588-025-02273-4#ref-CR39 "Xue, C. et al. PbrmiR397a regulates lignification during stone cell development in pear fruit. Plant Biotechnol. J. 17, 103–117 (2019)."). These results suggested that structural genes, transcription factors and microRNAs (miRNAs) related to stone cell formation may have been selected in parallel during Asian pear domestication to reduce the SCC in fruit.

Fruit size is widely considered a domestication trait in domesticated crops[40](https://www.nature.com/articles/s41588-025-02273-4#ref-CR40 "Li, Y. et al. Genomic analyses of an extensive collection of wild and cultivated accessions provide new insights into peach breeding history. Genome Biol. 20, 36 (2019)."),[41](https://www.nature.com/articles/s41588-025-02273-4#ref-CR41 "Azzi, L. et al. Fruit growth-related genes in tomato. J. Exp. Bot. 66, 1075–1086 (2015).") and it is known to be regulated by two key processes: cell division and cell expansion[41](https://www.nature.com/articles/s41588-025-02273-4#ref-CR41 "Azzi, L. et al. Fruit growth-related genes in tomato. J. Exp. Bot. 66, 1075–1086 (2015)."). Cell division and cell expansion genes were identified, suggesting that they play a role in these processes and may affect fruit size (Supplementary Table [18](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)). _Pbre1\_13G194600_ encodes CDKG2 (cyclin-dependent kinase G2), a protein that positively regulates cell growth in _Arabidopsis_ spp.[42](https://www.nature.com/articles/s41588-025-02273-4#ref-CR42 "Jiang, S. et al. The UBP14-CDKB1;1-CDKG2 cascade controls endore-duplication and cell growth in Arabidopsis. Plant Cell 34, 1308–1325 (2022)."). _Pbre1\_08G029000_ encodes ARF5 (auxin response factor 5) and the homology of _ARF5_ has been proved to increase fruit size in tomatoes[43](https://www.nature.com/articles/s41588-025-02273-4#ref-CR43 "Liu, S. et al. Tomato AUXIN RESPONSE FACTOR 5 regulates fruit set and development via the mediation of auxin and gibberellin signaling. Sci. Rep. 8, 2971 (2018)."), suggesting its potential to increase fruit size in pears. In addition, five missense SNPs with different genotype frequencies between EW and EC pears were identified (Supplementary Fig. [30a](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)).

### SVs associated with agronomic traits

Previous studies have suggested that SVs are causative variants for agronomic traits[29](https://www.nature.com/articles/s41588-025-02273-4#ref-CR29 "Yang, C. Q. et al. Transcriptional regulation of plant secondary metabolism. J. Integr. Plant Biol. 54, 703–712 (2012)."). The high-quality 83,369-SV set provided us a chance to reveal the trait-associated SVs for pear. We performed an SV-based genome-wide association study (GWAS) for fruit transverse diameter (FTD), soluble solid content (SSC), SCC (Supplementary Note [11](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)) and single fruit weight (SFW) (Supplementary Note [11](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)), which are important traits for pear fruit quality and production. We identified a 66-bp deletion at chr. 2 that was significantly associated with FTD (Fig. [5a](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). This deletion overlapped with the selective sweep of AC pears (Fig. [5b](https://www.nature.com/articles/s41588-025-02273-4#Fig5)) and its occurrence in _P. pyrifolia_ (Ppy) and _P. bretschneideri_ (Pbr) was lower than in the AW group (Fig. [5c](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). In addition, pear accessions without deletion had significantly higher FTD compared with those with deletion (Fig. [5d](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). A candidate gene, _PyTPR_, encoding a tetratricopeptide repeat-like superfamily protein, was identified downstream of this deletion (Fig. [5e](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). This gene is involved in plant development, elongation of cotton fibers[44](https://www.nature.com/articles/s41588-025-02273-4#ref-CR44 "Fang, D. D. et al. An EMS-induced mutation in a tetratricopeptide repeat-like superfamily protein gene (Ghir_A12G008870) on chromosome A12 is responsible for the liy short fiber phenotype in cotton. Theor. Appl. Genet. 133, 271–282 (2020).") and _Arabidopsis_ roots[45](https://www.nature.com/articles/s41588-025-02273-4#ref-CR45 "Zhang, M. et al. A tetratricopeptide repeat domain-containing protein SSR1 located in mitochondria is involved in root development and auxin polar transport in Arabidopsis. Plant J. 83, 582–599 (2015)."). Accessions without deletion have significantly higher expression levels of _PyTPR_ compared with accessions with deletion (mean number of fragments per kilobase of exon per million mapped fragments (FPKM): 497.33 versus 81.85) (Fig. [5f](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). These results suggested that _PyTPR_ was under selection during Asian pear domestication and may have a potential role in pear FTD and fruit development.

**Fig. 5: SV GWAS of FTD and SSC.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02273-4/figures/5)

**a**, Manhattan plot displaying the SV GWAS outcomes of the FTD. _P_ values were computed from a two-sided _F_ test. The red dotted lines indicate the significance thresholds (1/_n_, where _n_ denotes the SV number) for SV-based GWASs. **b**, Distribution of nucleotide diversity (_π_) of AW and AC pears (including _P. pyrifolia_ and _P. bretschneideri_) and _F_ST between AW and AC pears in a 20-kb interval. The shaded region means the selective sweep of AW versus AC pears. **c**, The deletion appearance frequency in AW pears and Ppy and Pbr groups. **d**, Comparison of FTD between samples with (_n_ = 32) and those without (_n_ = 150) deletion. **e**, The significantly associated SV downstream of _PyTPR_. **f**, _PyTPR_ expression pattern in samples with (_n_ = 6) and without (_n_ = 22) deletion. **g**, Manhattan plot displaying the SV GWAS outcomes of the SSC. **h**, Comparison of SSC between samples with (_n_ = 73) and those without (_n_ = 199) deletion. **i**, The significantly associated SV downstream of _PybZIP6_. **j**, _PybZIP6_ expression pattern in DS and MRB at five fruit developmental (S1–S5) and three postharvest (S6–S8) stages. **k**, The deletion appearance frequencies in AW, Ppy, Pbr, EW and Pco groups. In **d**, **f** and **h**, _P_ values were computed from two-sided Student’s _t_\-tests. In **d**, **f** and **h**, each box represents the center line (median), the box limits (first and third quartiles) and the whiskers (1.5× the IQR).

Similarly, we identified a 1,016-bp deletion associated with SSC (Fig. [5g](https://www.nature.com/articles/s41588-025-02273-4#Fig5)) and accessions with this deletion showed significantly higher SSC than those without it (Fig. [5h](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). _Pbre1\_15G533000_, which encodes a bZIP transcription factor (PybZIP6), was identified (Fig. [5i](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). It has been reported that bZIP transcription factors are involved in SSC accumulation in pears[46](https://www.nature.com/articles/s41588-025-02273-4#ref-CR46 "Wang, H. et al. A pear S1-bZIP transcription factor PpbZIP44 modulates carbohydrate metabolism, amino acid, and flavonoid accumulation in fruits. Hortic. Res. 10, uhad140 (2023)."),[47](https://www.nature.com/articles/s41588-025-02273-4#ref-CR47 "Jia, L. et al. PbrbZIP15 promotes sugar accumulation in pear via activating the transcription of the glucose isomerase gene PbrXylA1. Plant J. 117, 1392–1412 (2024)."),[48](https://www.nature.com/articles/s41588-025-02273-4#ref-CR48 "Zhang, H., Tao, X., Fan, X., Zhang, S. & Qin, G. PpybZIP43 contributes to sucrose synthesis in pear fruits by activating PpySPS3 expression and interacts with PpySTOP1. Physiol. Plant 174, e13732 (2022)."). PybZIP6 was highly expressed during the late fruit development stages (S3–S5) in DS, MRB and four other representative pear accessions, suggesting its potential in SSC accumulation at late fruit developmental stages (Fig. [5j](https://www.nature.com/articles/s41588-025-02273-4#Fig5) and Supplementary Fig. [32](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Compared with the EW group, the deletion occurrence frequency was higher in the _P. communis_ (Pco) group (Fig. [5k](https://www.nature.com/articles/s41588-025-02273-4#Fig5)). Overall, our SV-based GWAS provided new markers that will be helpful to improve the fruit quality of pear cultivars.

### The role of _PyACS1_ in fruit maturity and softening

Nearby SVs can alter gene expression patterns due to the interruptions in regulatory or gene regions[49](https://www.nature.com/articles/s41588-025-02273-4#ref-CR49 "Shang, L. et al. A super pan-genomic landscape of rice. Cell Res. 32, 878–896 (2022)."),[50](https://www.nature.com/articles/s41588-025-02273-4#ref-CR50 "Chen, S. et al. Gene mining and genomics-assisted breeding empowered by the pangenome of tea plant Camellia sinensis. Nat. Plants 9, 1986–1999 (2023)."). RNA-seq data from 110 samples (consisting of 62 Asian pears and 38 European pears) were generated to identify the SV effect on gene expression (Supplementary Table [19](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)). A total of 4,040 SVs in the upstream 2-kb region and 568 SVs at the gene exon region were significantly correlated with gene expression (Fig. [6a](https://www.nature.com/articles/s41588-025-02273-4#Fig6)); 70.79% (2,860 of 4,040) SVs and 65.49% (372 of 568) SVs exhibited divergent occurrence (FDR < 0.001 and fold-change >2) between the Asian and the European groups, suggesting the potential of SVs in regulating intergroup gene expression. These genes were enriched in metabolism and biosynthesis process, including ‘fructose and mannose metabolism’ and ‘citrate cycle’ (Supplementary Figs. [34](https://www.nature.com/articles/s41588-025-02273-4#MOESM1) and [35](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). A total of 40 SVs and 12 SVs were significantly correlated with 36 sugar-related and 10 acid-related genes, including previously reported, fruit sugar-related genes like _PpySTOP1_, _PpySPS8_ and _PbVHA-A_[25](https://www.nature.com/articles/s41588-025-02273-4#ref-CR25 "Li, J. et al. Natural variations in the PbCPK28 promoter regulate sugar content through interaction with PbTST4 and PbVHA-A1 in pear. Plant J. 114, 124–141 (2023)."),[48](https://www.nature.com/articles/s41588-025-02273-4#ref-CR48 "Zhang, H., Tao, X., Fan, X., Zhang, S. & Qin, G. PpybZIP43 contributes to sucrose synthesis in pear fruits by activating PpySPS3 expression and interacts with PpySTOP1. Physiol. Plant 174, e13732 (2022).") (Supplementary Table [20](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)).

**Fig. 6: SVs associated with gene expression and functional impact on fruit ripening and softening of European pears.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02273-4/figures/6)

**a**, Manhattan plot displaying the SVs at exon or promoter regions significantly associated with gene expression. Sugar- and acid-related genes were marked in red and ethylene biosynthesis genes were marked in purple. **b**, The expression of _PyACS1_ in Asian (_n_ = 62), European (_n_ = 38) and interspecific hybrid (_n_ = 10) pears. The ‘_n_’ means the number of accessions (center line as median, box limits as first and third quartiles and whiskers as 1.5× the IQR). **c**, _PyACS1_ expression pattern of DS and MRB at eight fruit stages. **d**, Phenotype of WT tomato fruit and tomato transformants overexpressing _PyACS1_ (OE-_PyACS1_). **e**–**g**, Breaker stage (_n_WT = 10, _n_OE-PyACS1 = 10), ethylene production (_n_WT = 8, _n_OE_\-PyACS1_ = 12) and firmness (_n_WT = 12, _n_OE_\-PyACS1_ = 12) in the WT and OE-_PyACS1_ tomato fruit (center line as median, box limits as first and third quartiles and whiskers as 1.5× the IQR). The _n_WT and _n_OE_\-PyACS1_ refer to the numbers of fruit samples used for WT and OE-_PyACS1_. **h**, A 286-bp insertion sequence identified at the promoter region of _PyACS1_ in MRB. **i**, PCR-based screen showing the absence (right) or presence (left) of the insertion sequence in the upstream of _PyACS1_ in European (I–V represent ‘Max Red Bartlett’, ‘Red comice’, ‘Spalding’, ‘Charneus’ and ‘Anjou’, respectively) and Asian (VI–X represent ‘Dangshansuli’, ‘Cuiguan’, ‘Cuiyu’, ‘Yunhong No. 1’ and ‘Yali’, respectively) pears. **j**, The insertion appearance frequency in Asian, European and interspecific hybrid pears. **k**, TE assays showing significant differences in promoter activities for _PyACS1_proMRB and _PyACS1_proDS. The data are presented as mean ± s.e.m. **l**, Relative activity levels of luciferase of three truncated promoters (830 bp, 1,221 bp and 2,000 bp) of _PyACS1_proMRB. The data are presented as mean ± s.e.m. **m**, EMSA showing that PybHLH94 binds directly to the G-box and E-box within the insertion sequence at the promoter region of _PyACS1_. Hot-P and Mut-P represent biotin-labeled WT and mutation probes, respectively; competitor-P represents the unlabeled WT sequence. In **b**, **e**–**g** and **k**–**l**, _P_ values were computed from two-sided Student’s _t_\-tests. The numbers of biologically independent samples used in **k**–**l** is 3. The full scans of all the gels are included in [Source data](https://www.nature.com/articles/s41588-025-02273-4#Sec33).

[Source data](https://www.nature.com/articles/s41588-025-02273-4#MOESM4)

Compared with Asian pears, European pears are usually classified as climacteric fruit and ethylene production is responsible for the initiation and progression of fruit ripening and softening[51](https://www.nature.com/articles/s41588-025-02273-4#ref-CR51 "Hiwasa, K. et al. Ethylene is required for both the initiation and progression of softening in pear (Pyrus communis L.) fruit. J. Exp. Bot. 54, 771–779 (2003)."),[52](https://www.nature.com/articles/s41588-025-02273-4#ref-CR52 "Paul, V., Pandey, R. & Srivastava, G. C. The fading distinctions between classical patterns of ripening in climacteric and non-climacteric fruit and the ubiquity of ethylene—an overview. J. Food Sci. Technol. 49, 1–21 (2012)."). Five ethylene biosynthesis-related genes with SVs were identified (Supplementary Table [20](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)) and one gene (_Pbre1\_15G535800_) encoding 1-aminocyclopropane-1-carboxylic acid synthase (_PyACS1_) functions as the rate-limiting enzyme in ethylene synthesis[53](https://www.nature.com/articles/s41588-025-02273-4#ref-CR53 "Song, Q. et al. Diurnal down-regulation of ethylene biosynthesis mediates biomass heterosis. Proc. Natl Acad. Sci. USA 115, 5606–5611 (2018)."). Transcriptome analysis showed that _PyACS1_ was significantly more highly expressed in European pears than in Asian pears (Fig. [6b](https://www.nature.com/articles/s41588-025-02273-4#Fig6)). The interspecific hybrids have higher expression levels than both Asian and European pears, indicating that overdominance expression pattern may be observed in these hybrid accessions[54](https://www.nature.com/articles/s41588-025-02273-4#ref-CR54 "Liu, C. et al. The OsWRKY72-OsAAT30/OsGSTU26 module mediates reactive oxygen species scavenging to drive heterosis for salt tolerance in hybrid rice. J. Integr. Plant Biol. 66, 709–730 (2024)."). The _PyACS1_ gene was only highly expressed during the late development fruit stages (S4 and S5) and three postharvest (S6–S8) stages in MRB and the FPKM value of _PyACS1_ was <0.1 at all eight stages (S1–S8) of DS (Fig. [6c](https://www.nature.com/articles/s41588-025-02273-4#Fig6)). We further explored the functions of _PyACS1_ during fruit maturation and postharvest stages (S5–S8). Overexpressing _PyACS1_ (OE-_PyACS1_) tomato lines showed accelerated ripening (Fig. [6d, e](https://www.nature.com/articles/s41588-025-02273-4#Fig6)), higher ethylene (Fig. [6f](https://www.nature.com/articles/s41588-025-02273-4#Fig6)) and lower firmness (Fig. [6g](https://www.nature.com/articles/s41588-025-02273-4#Fig6)) compared with wild-type (WT) fruit, suggesting that _PyACS1_ promotes fruit ripening and softening.

Furthermore, a 286-bp insertion sequence was identified in the promoter region of _PyACS1_, which was present in all European pears and interspecific hybrid accessions, but not in Asian pears (Fig. [6h–j](https://www.nature.com/articles/s41588-025-02273-4#Fig6)). A PCR assay further confirmed that the 286-bp insertion sequence was present in European pears and absent in Asian pears (Fig. [6i](https://www.nature.com/articles/s41588-025-02273-4#Fig6) and Supplementary Fig. [36](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Sequence analysis of insertion showed that one E-box (CATTTG) and one G-box (CACGTT) _cis_\-element were included (Supplementary Fig. [37a](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)), which are the potential binding sites of basic helix–loop–helix transcription factors[55](https://www.nature.com/articles/s41588-025-02273-4#ref-CR55 "Dombrecht, B. et al. MYC2 differentially modulates diverse jasmonate-dependent functions in Arabidopsis. Plant Cell 19, 2225–2245 (2007)."),[56](https://www.nature.com/articles/s41588-025-02273-4#ref-CR56 "Hu, D. G. et al. The regulatory module MdPUB29-MdbHLH3 connects ethylene biosynthesis with fruit quality in apple. N. Phytol. 221, 1966–1982 (2019)."). Thus, we conducted weighted co-expression network analysis and one basic helix–loop–helix transcription factor (PybHLH94) was identified in the _PyACS1_ gene co-expression module (Supplementary Fig. [37b](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Dual-luciferase (LUC) transcription assay in tobacco protoplasts revealed that PybHLH94 can significantly activate _PyACS1_proMRB (with a 286-bp insertion), but did not activate _PyACS1_proDS (without a 286-bp insertion), suggesting insertion-dependent transcriptional regulation (Fig. [6k](https://www.nature.com/articles/s41588-025-02273-4#Fig6)). Promoter truncation results showed that activation occurred within the −1,221-bp to −830-bp region, which contained the 286-bp insertion sequence (−1,141 bp to −855 bp) (Fig. [6l](https://www.nature.com/articles/s41588-025-02273-4#Fig6) and Supplementary Fig. [37c](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). This suggests that PybHLH94 can specifically activate the promoter of _PyACS1_ by binding to the 286-bp insertion sequence. The electrophoretic mobility shift assay (EMSA) result showed that _PybHLH94_ (His) binds directly to the G-box and E-box in the 286-bp insertion sequence (Fig. [6m](https://www.nature.com/articles/s41588-025-02273-4#Fig6)).

Discussion
----------

In this study, we generated T2T haplotype-resolved genome assemblies for DS and MRB, allowing us to investigate all genomic regions and the allelic variations. Based on the haplotype assemblies, haplotype-specific genes and biallelic genes were identified. TEs and methylation have been proven to regulate gene expression[57](https://www.nature.com/articles/s41588-025-02273-4#ref-CR57 "He, L. et al. DNA methylation-free Arabidopsis reveals crucial roles of DNA methylation in regulating gene expression and development. Nat. Commun. 13, 1335 (2022)."),[58](https://www.nature.com/articles/s41588-025-02273-4#ref-CR58 "Zhang, J. et al. Autotetraploid rice methylome analysis reveals methylation variation of transposable elements and their effects on gene expression. Proc. Natl Acad. Sci. USA 112, E7022–E7029 (2015).") and variation of methylation and TE between haplotype-specific genes and biallelic genes may be related to gene expression alteration in DS and MRB. HapDom ASE has an allele with biased expression in different fruit developmental stages of the pear, suggesting the dominance effect on heterosis. Furthermore, a sub-ASE gene exhibits one allele-biased expression in several samples but shifts to another allele in other samples, suggesting the overdominance effect[59](https://www.nature.com/articles/s41588-025-02273-4#ref-CR59 "Shao, L. et al. Patterns of genome-wide allele-specific expression in hybrid rice and the implications on the genetic basis of heterosis. Proc. Natl Acad. Sci. USA 116, 5653–5658 (2019)."). As with apple and kiwifruit[5](https://www.nature.com/articles/s41588-025-02273-4#ref-CR5 "Sun, X. P. et al. Phased diploid genome assemblies and pan-genomes provide insights into the genetic history of apple domestication. Nat. Genet. 52, 1423–1432 (2020)."),[60](https://www.nature.com/articles/s41588-025-02273-4#ref-CR60 "Han, X. et al. Two haplotype-resolved, gap-free genome assemblies for Actinidia latifolia and Actinidia chinensis shed light on the regulatory mechanisms of vitamin C and sucrose metabolism in kiwifruit. Mol. Plant 16, 452–470 (2023)."), we observed that HapDom genes were more than subgenes, suggesting the major role of the dominance effect during fruit development. Our results showed that less than half of ASE genes were shared between DS and MRB and 22.12% of ASEs kept consistent categories, suggesting that ASE patterns were not conserved between different accessions. Studies showed that ASEs were related to the genetic and epigenetic variations[8](https://www.nature.com/articles/s41588-025-02273-4#ref-CR8 "Hu, W. et al. Allele-defined genome reveals biallelic differentiation during cassava evolution. Mol. Plant 14, 851–854 (2021)."),[60](https://www.nature.com/articles/s41588-025-02273-4#ref-CR60 "Han, X. et al. Two haplotype-resolved, gap-free genome assemblies for Actinidia latifolia and Actinidia chinensis shed light on the regulatory mechanisms of vitamin C and sucrose metabolism in kiwifruit. Mol. Plant 16, 452–470 (2023)."). Considering the long divergence history, variation accumulation between DS and MRB may have the potential for ASE alteration and further affect the adaptive capacity and plant development[59](https://www.nature.com/articles/s41588-025-02273-4#ref-CR59 "Shao, L. et al. Patterns of genome-wide allele-specific expression in hybrid rice and the implications on the genetic basis of heterosis. Proc. Natl Acad. Sci. USA 116, 5653–5658 (2019)."),[61](https://www.nature.com/articles/s41588-025-02273-4#ref-CR61 "Hu, H. et al. Allele-specific expression reveals multiple paths to highland adaptation in maize. Mol. Biol. Evol. 39, msac239 (2022).").

Previous studies showed that cultivated _P. ussuriensis_ was directly derived from wild _P. ussuriensis_[10](https://www.nature.com/articles/s41588-025-02273-4#ref-CR10 "Wu, J. et al. Diversification and independent domestication of Asian and European pears. Genome Biol. 19, 77 (2018)."). However, we found that cultivated _P. bretschneideri_ (and _P. pyrifolia_) played an important role in _P. ussuriensis_ domestication. Cultivated _P. ussuriensis_ showed higher nucleotide diversity than wild _P. ussuriensis_, which was likely increased by interspecific introgression[6](https://www.nature.com/articles/s41588-025-02273-4#ref-CR6 "Zhang, X. T. et al. Haplotype-resolved genome assembly provides insights into evolutionary history of the tea plant Camellia sinensis. Nat. Genet. 53, 1250–1259 (2021)."). Wild _P. ussuriensis_ has high cold tolerance and was mainly distributed in the frigid environment of China[10](https://www.nature.com/articles/s41588-025-02273-4#ref-CR10 "Wu, J. et al. Diversification and independent domestication of Asian and European pears. Genome Biol. 19, 77 (2018)."). Through introgression from cultivated _P. bretschneideri_, cultivated _P. ussuriensis_ may promote the improvement of fruit quality and preserve adaptability of wild _P. ussuriensis_ to a cold environment at the same time. These findings provide a new perspective on the domestication history of pears and offer a new strategy for breeding cold-resistant pear varieties.

Fruit sweetness, acidity and SCC are well-known domestication traits that are still the main targets in pear-breeding programs. We found many previously unreported trait-associated genes in selective sweeps based on the gold standard reference genome. _PbrbZIP48_ and _PbrmiR397a_ were also identified in the selective sweeps of Asian pears and the products (a transcription factor and miRNA, respectively) regulate lignin biosynthesis-related genes and affect the SCC in fruit flesh[39](https://www.nature.com/articles/s41588-025-02273-4#ref-CR39 "Xue, C. et al. PbrmiR397a regulates lignification during stone cell development in pear fruit. Plant Biotechnol. J. 17, 103–117 (2019)."). These findings demonstrate that transcription factors, miRNAs and protein-coding genes may cause pear domestication-associated changes. _MiR397_ also appears in the _Indica_ rice domestication process[62](https://www.nature.com/articles/s41588-025-02273-4#ref-CR62 "Swetha, C. et al. Major domestication-related phenotypes in indica rice are due to loss of miRNA-mediated laccase silencing. Plant Cell 30, 2649–2662 (2018)."), which suggests that _miR397_ was widely selected during plant domestication due to its conserved function in crop improvement. In addition, we also found that _Ma1_ and _fw2.2_ were selected during the domestication of apples and pears. _Ma1_ has been identified as a factor that can affect the fruit acidity of cultivated apples and a mutation (G to A) causes the _Ma1_ protein truncation and is responsible for the low fruit acidity in apple[5](https://www.nature.com/articles/s41588-025-02273-4#ref-CR5 "Sun, X. P. et al. Phased diploid genome assemblies and pan-genomes provide insights into the genetic history of apple domestication. Nat. Genet. 52, 1423–1432 (2020)."). Still, this mutation did not appear in pears, implying that other mutations may have been selected during pear domestication. These selected or co-selected genes provide helpful genetic resources for understanding the domestication process and facilitating pear breeding.

Based on the two high-quality genome assemblies for DS and MRB, we constructed a graph-based genome and used it for SV genotyping with next-generation sequencing (NGS) data and identified additional signatures using the SV-GWAS approach, including several potential candidate genes for agronomically important traits. Ethylene production is the response for the initiation and progression of fruit ripening and softening in European pears[51](https://www.nature.com/articles/s41588-025-02273-4#ref-CR51 "Hiwasa, K. et al. Ethylene is required for both the initiation and progression of softening in pear (Pyrus communis L.) fruit. J. Exp. Bot. 54, 771–779 (2003)."),[52](https://www.nature.com/articles/s41588-025-02273-4#ref-CR52 "Paul, V., Pandey, R. & Srivastava, G. C. The fading distinctions between classical patterns of ripening in climacteric and non-climacteric fruit and the ubiquity of ethylene—an overview. J. Food Sci. Technol. 49, 1–21 (2012)."),[63](https://www.nature.com/articles/s41588-025-02273-4#ref-CR63 "Nham, N. T. et al. A transcriptome approach towards understanding the development of ripening capacity in ‘Bartlett’ pears (Pyrus communis L.). BMC Genom. 16, 762 (2015)."). In this study, we identified that _PyACS1_ has a high expression level in European pears and provided evidence of its role in fruit ripening and softening. We found one 286-bp insertion sequence from the promoter of the _PyACS1_ in European pears and this insertion sequence contains two new _cis_\-elements that strengthen transactivation by _PybHLH94_. Asian and European pears experienced long divergence history and independent domestication events[10](https://www.nature.com/articles/s41588-025-02273-4#ref-CR10 "Wu, J. et al. Diversification and independent domestication of Asian and European pears. Genome Biol. 19, 77 (2018).") and their hybridization may form new germplasms. Overdominance gene expression patterns have been documented in hybrids of potato[64](https://www.nature.com/articles/s41588-025-02273-4#ref-CR64 "Li, D. et al. The multi-omics basis of potato heterosis. J. Integr. Plant Biol. 64, 671–687 (2022)."), rice[65](https://www.nature.com/articles/s41588-025-02273-4#ref-CR65 "Ma, X. et al. An enhanced network of energy metabolism, lysine acetylation, and growth-promoting protein accumulation is associated with heterosis in elite hybrid rice. Plant Commun. 4, 100560 (2023)."),[66](https://www.nature.com/articles/s41588-025-02273-4#ref-CR66 "Zhang, J. et al. Extensive sequence divergence between the reference genomes of two elite indica rice varieties Zhenshan 97 and Minghui 63. Proc. Natl Acad. Sci. USA 113, E5163–E5171 (2016)."),[67](https://www.nature.com/articles/s41588-025-02273-4#ref-CR67 "Shen, G. et al. The regulatory network mediated by circadian clock genes is related to heterosis in rice. J. Integr. Plant Biol. 57, 300–312 (2015).") and _Arabidopsis_ spp.[68](https://www.nature.com/articles/s41588-025-02273-4#ref-CR68 "Yang, M. et al. Genomic architecture of biomass heterosis in Arabidopsis. Proc. Natl Acad. Sci. USA 114, 8101–8106 (2017)."), which have elevated gene expression levels compared with their parental lines. Similarly, _PyACS1_ exhibits higher expression levels in interspecific hybrids compared with Asian and European pears, suggesting that, aside from the 286-bp insertion sequence, multiple factors in these hybrids may contribute to the overdominance expression pattern. Furthermore, accession-specific mutations may also influence gene expression by changing _cis_\-elements like ABRE, STRE and G-box (Supplementary Fig. [38](https://www.nature.com/articles/s41588-025-02273-4#MOESM1))[69](https://www.nature.com/articles/s41588-025-02273-4#ref-CR69 "Yang, Q. et al. ABA-responsive ABRE-BINDING FACTOR3 activates DAM3 expression to promote bud dormancy in Asian pear. Plant Cell Environ. 43, 1360–1375 (2020)."),[70](https://www.nature.com/articles/s41588-025-02273-4#ref-CR70 "Fu, J. et al. Genome-wide identification of Petunia HSF genes and potential function of PhHSF19 in benzenoid/phenylpropanoid biosynthesis. Int. J. Mol. Sci. 23, 2974 (2022)."),[71](https://www.nature.com/articles/s41588-025-02273-4#ref-CR71 "Zhang, N. et al. Changes to the core and flanking sequences of G-box elements lead to increases and decreases in gene expression in both native and synthetic soybean promoters. Plant Biotechnol. J. 17, 724–735 (2019)."). However, the direct evidence of a regulatory effect of the 286-bp insertion in _PyACS1_ should be obtained in future research, once current technical limitations, such as the lack of a stable transgenic or genome-editing system in pears, has been overcome. Overall, our results showed that a graph-based genome may be a powerful tool for accurate population-level SV genotyping and genomic studies, as well as the forward application of genes and SVs in pear genome-guided breeding.

In conclusion, we generated two T2T haplotype-resolved pear genomes in this study and provided insights into genome evolution, haplotype imbalance and interspecies variation in _Pyrus_ spp. Combining the high-quality genomic and population resequencing data allowed us to show the extensive genetic variance between Asian and European pears and identify trait-associated SVs. These genomic resources will provide a basis for further molecular biology research and will accelerate the breeding process in commercial pears.

Methods
-------

### Samples and genome sequencing

The pear cultivars ‘Dangshansuli’ (DS) and ‘Max Red Bartlett’ (MRB) are two critical commercial cultivars grown in China and Europe, respectively. Therefore, these two cultivars were chosen for genome sequencing. We generated HiFi, Hi-C and ONT ultra-long 100-kb reads for DS and MRB. To assist in gene prediction, RNA-seq data were generated from young stems, mature leaves, young leaves, seeds, pollen, radicles and fruits at different stages with: S1: 35 d after flowering blooming (d.a.f.b.); S2: 63 d.a.f.b.; S3: 91 d.a.f.b.; S4: 120 d.a.f.b.; S5: 150 d.a.f.b. (for MRB, 135 d.a.f.b.); and S6–S8: 2, 4 and 8 d after harvest. For population analysis, 362 pear accessions were used in this study, including 137 newly collected accessions and 225 previously released accessions[10](https://www.nature.com/articles/s41588-025-02273-4#ref-CR10 "Wu, J. et al. Diversification and independent domestication of Asian and European pears. Genome Biol. 19, 77 (2018)."),[72](https://www.nature.com/articles/s41588-025-02273-4#ref-CR72 "Zhang, M. Y. et al. Genome-wide association studies provide insights into the genetic determination of fruit traits of pear. Nat. Commun. 12, 1141 (2021)."). The fruit pericarp was collected from 137 pear accessions and NGS data were generated. This study provides a detailed description of the pear samples used for genome assembly and short-read sequencing (Supplementary Method [1](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)).

### T2T genome assembly

The two haplotype assemblies of DS and MRB were performed using hifiasm (v.0.19.0)[18](https://www.nature.com/articles/s41588-025-02273-4#ref-CR18 "Cheng, H. Y. et al. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. Nat. Methods 18, 170–175 (2021).") and Verkko software (v.1.4.1)[19](https://www.nature.com/articles/s41588-025-02273-4#ref-CR19 "Rautiainen, M. et al. Telomere-to-telomere assembly of diploid chromosomes with Verkko. Nat. Biotechnol. 41, 1474–1482 (2023).") with PacBio HiFi, ONT ultra-long and Hi-C data, and HiCanu[20](https://www.nature.com/articles/s41588-025-02273-4#ref-CR20 "Nurk, S. et al. HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads. Genome Res. 30, 1291–1305 (2020).") with PacBio HiFi data. The Verkko assembly sequences were aligned to hifiasm assembly sequences for gap filling. If the assembly sequence can bridge the gap in the hifiasm result, the sequence will be merged to the genome sequence. HiCanu assembly sequences were used for telomeric region filling. ALLHIC (v.0.9.8) was used to anchor the contigs onto chromosomes with Hi-C data[73](https://www.nature.com/articles/s41588-025-02273-4#ref-CR73 "Zhang, X. T. et al. Assembly of allele-aware, chromosomal-scale autopolyploid genomes based on Hi-C data. Nat. Plants 5, 833–845 (2019).") and the resulting assembly was manually corrected. To check the quality of gap filling, PacBio HiFi and ONT reads were mapped to each haplotype using Minimap2 (v.2.23)[74](https://www.nature.com/articles/s41588-025-02273-4#ref-CR74 "Li, H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics 34, 3094–3100 (2018)."). To evaluate the completeness and quality of the assembly, NGS reads were mapped to each haplotype using Burrows–Wheeler alignment (BWA, v.0.7.17)[75](https://www.nature.com/articles/s41588-025-02273-4#ref-CR75 "Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)."). SAMtools (v.1.14)[76](https://www.nature.com/articles/s41588-025-02273-4#ref-CR76 "Li, H. et al. The sequence alignment/map format and SAMtools. Bioinformatics 25, 2078–2079 (2009).") was then used to calculate the genome coverage and mapping rate. The LTR Assembly Index value was calculated using LTR\_retriever[77](https://www.nature.com/articles/s41588-025-02273-4#ref-CR77 "Ou, S. J. et al. Assessing genome assembly quality using the LTR Assembly Index (LAI). Nucleic Acids Res. 46, e126 (2018)."). Genome completeness was evaluated by BUSCO (v.5.2.2) using the embryophyta\_odb10 database[78](https://www.nature.com/articles/s41588-025-02273-4#ref-CR78 "Manni, M. et al. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. Mol. Biol. Evol. 38, 4647–4654 (2021)."). The accuracy of the genome is evaluated by calculating quasi-variance using Merqury (v.1.3)[79](https://www.nature.com/articles/s41588-025-02273-4#ref-CR79 "Rhie, A. et al. Merqury: reference-free quality, completeness, and phasing assessment for genome assemblies. Genome Biol. 21, 245 (2020)."). For haplotype switch error ratio estimation, we used the calc\_switchErr pipeline (v.1.0)[6](https://www.nature.com/articles/s41588-025-02273-4#ref-CR6 "Zhang, X. T. et al. Haplotype-resolved genome assembly provides insights into evolutionary history of the tea plant Camellia sinensis. Nat. Genet. 53, 1250–1259 (2021).") (Supplementary Fig. [39](https://www.nature.com/articles/s41588-025-02273-4#MOESM1) and Supplementary Method [2](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Hi-C data were mapped to each haplotype of DS and MRB using HiC-Pro (v.2.7.0)[80](https://www.nature.com/articles/s41588-025-02273-4#ref-CR80 "Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. Genome Biol. 16, 259 (2015)."). The output matrix file was visualized using HiCPlotter (v.0.8.1)[81](https://www.nature.com/articles/s41588-025-02273-4#ref-CR81 "Akdemir, K. C. & Chin, L. HiCPlotter integrates genomic data with interaction matrices. Genome Biol. 16, 198 (2015).").

### Repeated sequences and gene annotation

TEs in the two haplotypes of DS and MRB were annotated by combing prediction from the start and homology searches. First, RepeatScout (v.1.0.6)[82](https://www.nature.com/articles/s41588-025-02273-4#ref-CR82 "Price, A. L. et al. De novo identification of repeat families in large genomes. Bioinformatics 21, I351–I358 (2005)."), Piler (v.1.0)[83](https://www.nature.com/articles/s41588-025-02273-4#ref-CR83 "Edgar, R. C. & Myers, E. W. PILER: identification and classification of genomic repeats. Bioinformatics 21, I152–I158 (2005)."), LTR\_FINDER (v.1.07)[84](https://www.nature.com/articles/s41588-025-02273-4#ref-CR84 "Xu, Z. & Wang, H. LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons. Nucleic Acids Res. 35, W265–W268 (2007).") and RepeatModeler (v.2.0.4)[85](https://www.nature.com/articles/s41588-025-02273-4#ref-CR85 "Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. Proc. Natl Acad. Sci. USA 117, 9451–9457 (2020).") were used to build the de novo repeat sequence library and RepeatMasker (v.4.1.2)[86](https://www.nature.com/articles/s41588-025-02273-4#ref-CR86 "Tarailo-Graovac, M. & Chen, N. Using RepeatMasker to identify repetitive elements in genomic sequences. Curr Protoc Bioinformatics 25, 4.10.1–4.10.14 (2009).") was used to annotate the repeat regions based on this library. Tandem Repeats Finder (v.4.09)[87](https://www.nature.com/articles/s41588-025-02273-4#ref-CR87 "Benson, G. Tandem repeats finder: a program to analyze DNA sequences. Nucleic Acids Res. 27, 573–580 (1999).") was used to find the tandem repeats. Finally, RepeatMasker was used to retrieve the divergent TEs.

The protein-coding gene structure was predicted using a combination method involving prediction from the start, RNA-seq data prediction and homology-based prediction. The protein sequences from five species (_Arabidopsis thaliana_, _P. communis_, _Malus domestica_, _Prunus persica_ and _Pyrus betulifolia_) were downloaded, and TBLASTN (v.2.2.26)[88](https://www.nature.com/articles/s41588-025-02273-4#ref-CR88 "Altschul, S. F. et al. Basic local alignment search tool. J. Mol. Biol. 215, 403–410 (1990).") searches and GeneWise (v.2.4.1)[89](https://www.nature.com/articles/s41588-025-02273-4#ref-CR89 "Birney, E. et al. GeneWise and genomewise. Genome Res. 14, 988–995 (2004).") were used to predict the gene structure using the corresponding genomic regions of each BLAST hit. Augustus (v.2.5.5)[90](https://www.nature.com/articles/s41588-025-02273-4#ref-CR90 "Urnov, F. D. et al. Highly efficient endogenous human gene correction using designed zinc-finger nucleases. Nature 435, 646–651 (2005)."), GlimmerHMM (v.3.0.1)[91](https://www.nature.com/articles/s41588-025-02273-4#ref-CR91 "Majoros, W. H. et al. TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders. Bioinformatics 20, 2878–2879 (2004).") and Genscan (v.1.0)[92](https://www.nature.com/articles/s41588-025-02273-4#ref-CR92 "Burge, C. & Karlin, S. Prediction of complete gene structures in human genomic DNA. J. Mol. Biol. 268, 78–94 (1997).") were used for gene predictions from the start. RNA-seq data were aligned to the haplotype sequences using HISAT2 (v.2.0.4)[93](https://www.nature.com/articles/s41588-025-02273-4#ref-CR93 "Kim, D. et al. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat. Biotechnol. 37, 907–915 (2019).") and the output was used for genome-guided transcript assembly using Cufflinks (v.2.1.1)[94](https://www.nature.com/articles/s41588-025-02273-4#ref-CR94 "Trapnell, C. et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat. Biotechnol. 28, 511–515 (2010)."). De novo transcriptome read assemblies were generated using Trinity (v.2.1.1)[95](https://www.nature.com/articles/s41588-025-02273-4#ref-CR95 "Haas, B. J. et al. De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. Nat. Protoc. 8, 1494–1512 (2013)."). Evidence Modeler (v.2.0.0) was used to integrate these prediction results to predict the gene model and PASA (v.2.5.3) added to predict the UTR region[96](https://www.nature.com/articles/s41588-025-02273-4#ref-CR96 "Haas, B. J. et al. Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments. Genome Biol. 9, R7 (2008)."). The functions of protein-coding genes were identified by eggNOG-mapper (v.2.1.10)[97](https://www.nature.com/articles/s41588-025-02273-4#ref-CR97 "Cantalapiedra, C. P. et al. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. Mol. Biol. Evol. 38, 5825–5829 (2021).").

### Collinearity analysis, _K_ a and _K_ s calculations

Paralogous and orthologous gene pairs were identified in syntenic blocks using JCVI (v.1.4.18)[98](https://www.nature.com/articles/s41588-025-02273-4#ref-CR98 "Tang, H. et al. JCVI: A versatile toolkit for comparative genomics analysis. Imeta 12, e211 (2024).") with the parameter ‘--cscore=0.99’. The protein sequences of each gene pair were aligned using MAFFT (v.7.49)[99](https://www.nature.com/articles/s41588-025-02273-4#ref-CR99 "Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol. Biol. Evol. 30, 772–780 (2013).") and then converted into CDS alignments based on the CDS using ParaAT (v.2.0)[100](https://www.nature.com/articles/s41588-025-02273-4#ref-CR100 "Zhang, Z. et al. ParaAT: a parallel tool for constructing multiple protein-coding DNA alignments. Biochem. Biophys. Res. Commun. 419, 779–781 (2012)."). The number of nonsynonymous substitutions per synonymous site (_K_a), synonymous substitutions per synonymous site (_K_s) and the _K_a\-to-_K_s ratios were calculated for these pairs using the NGS method implemented in PAML (v.4.9b)[101](https://www.nature.com/articles/s41588-025-02273-4#ref-CR101 "Yang, Z. PAML 4: phylogenetic analysis by maximum likelihood. Mol. Biol. Evol. 24, 1586–1591 (2007).").

### Gene family analysis and phylogenetic reconstruction

The protein sequences from 12 species, including _P. bretschneideri_ (DS), _P. communis_ (MRB), _Oryza sativa_, _A. thaliana_, _Solanum lycopersicum_, _Prunus avium_, _P. persica_, _Prunus mume_, _M. domestica_, _Fragaria vesca_, _Rosa chinensis_ and _Citrus sinensis_ were collected ([http://www.phytozome.net](http://www.phytozome.net/)). OrthoMCL (v.6.18)[102](https://www.nature.com/articles/s41588-025-02273-4#ref-CR102 "Li, L. et al. OrthoMCL: identification of ortholog groups for eukaryotic genomes. Genome Res. 13, 2178–2189 (2003).") was used to identify the gene families and MUSCLE (v.5.2)[103](https://www.nature.com/articles/s41588-025-02273-4#ref-CR103 "Edgar, R. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32, 1792–1797 (2004).") to perform multiple sequence alignments. A maximum-likelihood tree was constructed using RAxML (v.1.1.0)[104](https://www.nature.com/articles/s41588-025-02273-4#ref-CR104 "Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313 (2014).") with the PROTGAMMAAUTO model. The MCMCTree program in PAML[101](https://www.nature.com/articles/s41588-025-02273-4#ref-CR101 "Yang, Z. PAML 4: phylogenetic analysis by maximum likelihood. Mol. Biol. Evol. 24, 1586–1591 (2007).") was applied to estimate divergence times with five calibration values chosen from the Time Tree website ([http://www.timetree.org](http://www.timetree.org/)). Expansions and contractions of orthologous gene families in the 12 species were determined using CAFE (v.2.2)[105](https://www.nature.com/articles/s41588-025-02273-4#ref-CR105 "De Bie, T. et al. CAFE: a computational tool for the study of gene family evolution. Bioinformatics 22, 1269–1271 (2006).").

### SNP and indel calling and SV analysis

MUMmer4 (v.4.0.0)[106](https://www.nature.com/articles/s41588-025-02273-4#ref-CR106 "Marcais, G. et al. MUMmer4: a fast and versatile genome alignment system. PLoS Comput. Biol. 14, e1005944 (2018).") was used to compare the haplotypes and a delta-filter was further used to filter the raw alignment results with the parameters -m -i 90 -l 100. The filtered alignment results were used to detect the SNP and insertion and deletion (indel) (≤50 bp) variations using the SyRI (v.1.6.3) pipeline with default parameters[107](https://www.nature.com/articles/s41588-025-02273-4#ref-CR107 "Goel, M. et al. SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies. Genome Biol. 20, 277 (2019)."). Minigraph (v.0.20-r559)[108](https://www.nature.com/articles/s41588-025-02273-4#ref-CR108 "Li, H. et al. The design and construction of reference pangenome graphs with minigraph. Genome Biol. 21, 265 (2020).") was used to integrate the haplotypes (HapA and HapB of DS or MRB and HapA of DS and MRB) into graphs with the option -xggs. The gfatools (v.0.4-r214)[108](https://www.nature.com/articles/s41588-025-02273-4#ref-CR108 "Li, H. et al. The design and construction of reference pangenome graphs with minigraph. Genome Biol. 21, 265 (2020).") was used to derive the SVs from the graph with the bubble-popping algorithm. The position information, reference and nonreference paths and path length of each bubble were extracted. The SVs were further divided into indels based on the lengths of the reference and nonreference paths. The variation was considered a deletion if the reference path was longer than the nonreference path. Each haplotype sequence was mapped to the graph-based genome for SV genotyping in DS and MRB using Minigraph (v.0.20-r559)[108](https://www.nature.com/articles/s41588-025-02273-4#ref-CR108 "Li, H. et al. The design and construction of reference pangenome graphs with minigraph. Genome Biol. 21, 265 (2020)."). The inversions were identified using the SyRI pipeline with the default parameters[107](https://www.nature.com/articles/s41588-025-02273-4#ref-CR107 "Goel, M. et al. SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies. Genome Biol. 20, 277 (2019)."). For SV validation, 100 SVs with lengths ranging from 51 bp to 11.24 kb and 20 SVs with lengths ranging from 22.40 kb to 181.19 kb were randomly selected. The ONT reads were aligned to the HapA of DS using Minimap2 (v.2.23)[74](https://www.nature.com/articles/s41588-025-02273-4#ref-CR74 "Li, H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics 34, 3094–3100 (2018).") to check the SVs. The alignment results were manually checked using the Integrative Genomics Viewer (v.2.15.2)[109](https://www.nature.com/articles/s41588-025-02273-4#ref-CR109 "Thorvaldsdottir, H. et al. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief. Bioinform. 14, 178–192 (2013).").

### Analysis of allele-specific gene expression

The allelic gene pairs between haplotypes were identified referring to a previous method used in sugarcane[110](https://www.nature.com/articles/s41588-025-02273-4#ref-CR110 "Zhang, J. et al. Allele-defined genome of the autopolyploid sugarcane Saccharum spontaneum L. Nat. Genet. 50, 1565–1573 (2018).") and tea[6](https://www.nature.com/articles/s41588-025-02273-4#ref-CR6 "Zhang, X. T. et al. Haplotype-resolved genome assembly provides insights into evolutionary history of the tea plant Camellia sinensis. Nat. Genet. 53, 1250–1259 (2021)."). The allele identification was based on two strategies: (1) synteny-based and (2) coordinate-based approaches. First, JCVI was used to identify the synteny region with the parameter ‘--cscore=0.99’ and all paired genes were considered allele A and B (biallelic gene). Then, gene models that were not present in syntenic blocks were mapped to genes in the corresponding haplotype using BLAST (v.2.2.26)[88](https://www.nature.com/articles/s41588-025-02273-4#ref-CR88 "Altschul, S. F. et al. Basic local alignment search tool. J. Mol. Biol. 215, 403–410 (1990).") and GMAP (v.2021-08-25)[111](https://www.nature.com/articles/s41588-025-02273-4#ref-CR111 "Wu, T. D. & Watanabe, C. K. GMAP: a genomic mapping and alignment program for mRNA and EST sequences. Bioinformatics 21, 1859–1875 (2005)."). Potential alleles were considered if two genes had >50% overlap on coordinates, >80% sequence identity and the alignment length was >80% of gene sequences. Gene models with the same CDSs (allele with same CDS) were considered as a single allele. The allele with tandem and proximal duplication genes was identified with the Tandem module in JCVI using ‘--tandem\_Nmax 10 --percent\_overlap 80’. The remaining unpaired genes were considered to be haplotype-specific genes, which were mapped to the corresponding haplotype using GMAP (v.2021-08-25)[111](https://www.nature.com/articles/s41588-025-02273-4#ref-CR111 "Wu, T. D. & Watanabe, C. K. GMAP: a genomic mapping and alignment program for mRNA and EST sequences. Bioinformatics 21, 1859–1875 (2005)."). Based on the mapping result, haplotype-specific genes were further classified into three categories, including: genes with start codon loss or stop codon gain; genes truly lost; and genes with high divergence (gene with alignment length <80% gene length or gene identity <80%). If biallelic genes were in one-to-one synteny blocks, they were used for further allele-specific gene expression analysis.

RNA-seq data of DS and MRB fruit at eight stages (S1–S8) were used for allele-specific gene expression analysis. RNA-seq reads were trimmed using Trimmomatic (v.0.39)[112](https://www.nature.com/articles/s41588-025-02273-4#ref-CR112 "Bolger, A. M. et al. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014).") and mapped against allele-aware annotated gene models using STAR (v.2.7.9)[113](https://www.nature.com/articles/s41588-025-02273-4#ref-CR113 "Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013).") software with parameters ‘--alignIntronMax 20000 --alignMatesGapMax 25000 --outFilterMultimapNmax 1’. StringTie (v.2.1.7)[114](https://www.nature.com/articles/s41588-025-02273-4#ref-CR114 "Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat. Biotechnol. 33, 290–295 (2015).") software was used for FPKM estimation based on unique mapping reads. ASE was determined using DEseq2 (ref. [115](https://www.nature.com/articles/s41588-025-02273-4#ref-CR115 "Love, M. I. et al. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014).")) with more than a twofold change and adjusted _P_ < 0.05. Based on a previous study[116](https://www.nature.com/articles/s41588-025-02273-4#ref-CR116 "Lan, X. & Pritchard, J. K. Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals. Science 352, 1009–1013 (2016)."), ASE genes were further divided into three categories: (1) haplotype dominance gene pair (Hap), in which one haplotype gene was more highly expressed than its partner allele in at least one-third of the tissues and was not significantly different in other tissues; (2) Sub (sub- or neofunctionalized pairs), in which each of the two allelic genes showed a significantly high expression level in at least one tissue; and (3) other ASE genes that were classified as no-difference pairs (NoDiff). Methylation analysis followed a previous study[117](https://www.nature.com/articles/s41588-025-02273-4#ref-CR117 "Song, B. et al. Increased DNA methylation contributes to the early ripening of pear fruits during domestication and improvement. Genome Biol. 25, 87 (2024).") in which Bismark (v.2.2.5)[118](https://www.nature.com/articles/s41588-025-02273-4#ref-CR118 "Krueger, F. & Andrews, S. R. Bismark: a flexible aligner and methylation caller for bisulfite-Seq applications. Bioinformatics 27, 1571–1572 (2011).") was used to analyze the whole-genome bisulfite sequencing data of leaf and fruit of DS and MRB and deepTools (v.3.5.1)[119](https://www.nature.com/articles/s41588-025-02273-4#ref-CR119 "Ramirez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 44, W160–W165 (2016).") to evaluate the methylation level of the upstream 5-kb gene body and the downstream 5-kb regions.

### Graph-based genome construction and SV genotyping

To construct the graph-based genome, we used the two haplotypes of DS and MRB for graph generation using Minigraph (v.0.20-r559)[108](https://www.nature.com/articles/s41588-025-02273-4#ref-CR108 "Li, H. et al. The design and construction of reference pangenome graphs with minigraph. Genome Biol. 21, 265 (2020).") and DS HapA was the base linear genome (Supplementary Fig. [40](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). The output Graphical Fragment Assembly file was used to construct a graph-based genome with the vg toolkit (v.1.46.0)[120](https://www.nature.com/articles/s41588-025-02273-4#ref-CR120 "Hickey, G. et al. Genotyping structural variants in pangenome graphs using the vg toolkit. Genome Biol. 21, 35 (2020)."). We further evaluated the performance of SV genotyping using NGS data with a graph-based genome as reference (Supplementary Method [4](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). To compare the differences in SV genotype frequency between the Asian and European pears, short sequencing reads from the 362 pear accessions were mapped to the graph-based genome using the above graph-based SV genotyping pipeline and SVs with missing data rate <0.2 and minor allele frequency >0.05 were retained. The SV genotype frequencies of the Asian and European pear populations were calculated using VCFtools (v.0.1.16)[121](https://www.nature.com/articles/s41588-025-02273-4#ref-CR121 "Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011)."). SVs were counted twice for homozygous alleles and once for heterozygous alleles.

### SNP calling

For capturing more SNP variations, we constructed a linear pangenome sequence with four haplotype sequences using the psvcp (v.1.01) pipeline[122](https://www.nature.com/articles/s41588-025-02273-4#ref-CR122 "Wang, J. et al. A pangenome analysis pipeline provides insights into functional gene identification in rice. Genome Biol. 24, 19 (2023).") and 47.82-Mb nonreference sequences were added to the reference genome (DS HapA). Raw sequencing reads were trimmed using Trimmomatic (v.0.39)[112](https://www.nature.com/articles/s41588-025-02273-4#ref-CR112 "Bolger, A. M. et al. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014).") and the clean reads were then aligned to the reference genome sequence using BWA (v.0.7.17)[75](https://www.nature.com/articles/s41588-025-02273-4#ref-CR75 "Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)."). The alignment result was sorted and duplicate reads were marked using Picard software ([https://broadinstitute.github.io/picard](https://broadinstitute.github.io/picard)). The Genome Analysis Toolkit (v.4.2.3)[123](https://www.nature.com/articles/s41588-025-02273-4#ref-CR123 "McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303 (2010).") was used for SNP calling with the HaplotypeCaller method and the SNP set was further filtered using VCFtools (v.0.1.16)[121](https://www.nature.com/articles/s41588-025-02273-4#ref-CR121 "Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011).") software with the parameters ‘-maf 0.05 -max-missing 0.9’. Finally, SNPs with minor allele frequency >0.05 and missing data rate <0.1 were retained for further analysis.

### Phylogenetic analysis and population structure study

A total of 273,985 SNPs at fourfold degenerate sites (4D SNPs) were selected for phylogenetic and population structure analyses. A maximum-likelihood tree was constructed using IQ-TREE (v.2.2.0)[124](https://www.nature.com/articles/s41588-025-02273-4#ref-CR124 "Minh, B. Q. et al. IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. Mol. Biol. Evol. 37, 1530–1534 (2020)."), with 1,000 bootstrap replicates, and the best substitution model ‘PMB + F + R10’ was selected with the ModelFinder function. Admixture (v.1.3.0)[125](https://www.nature.com/articles/s41588-025-02273-4#ref-CR125 "Alexander, D. H. et al. Fast model-based estimation of ancestry in unrelated individuals. Genome Res. 19, 1655–1664 (2009).") was used for population structure analysis and principal component analysis was performed with PLINK (v.1.90)[126](https://www.nature.com/articles/s41588-025-02273-4#ref-CR126 "Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007)."). The genome-wide selective sweeps were identified between AW and AC pears (including _P. pyrifolia_ and _P. bretschneideri_) and EW and EC pears (_P. communis_). Using VCFtools (v.0.1.16)[121](https://www.nature.com/articles/s41588-025-02273-4#ref-CR121 "Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011)."), we calculated pairwise _F_ST (wild versus cultivated populations) and nucleotide diversity (_π_) in 20-kb sliding windows (10-kb step). Regions ranking in the top 5% for both _F_ST and _π_ were classified as selective sweeps.

### Ancestral state estimation and inference of demographic history

To obtain the ancestral states, genomic resequencing data of the outgroup species hawthorn, apple and loquat were aligned against the HapA assembly of DS using BWA (v.0.7.17)[75](https://www.nature.com/articles/s41588-025-02273-4#ref-CR75 "Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009).") and ancestral alleles were called using ANGSD (v.0.940)[127](https://www.nature.com/articles/s41588-025-02273-4#ref-CR127 "Korneliussen, T. S. et al. ANGSD: analysis of next generation sequencing data. BMC Bioinf. 15, 356 (2014).") with the parameters ‘-doFasta 2 -doCounts 1’. To infer the demographic history, we used the stairway plot method[128](https://www.nature.com/articles/s41588-025-02273-4#ref-CR128 "Liu, X. & Fu, Y. X. Exploring population size changes using SNP frequency spectra. Nat. Genet. 47, 555–559 (2015)."),[129](https://www.nature.com/articles/s41588-025-02273-4#ref-CR129 "Liu, X. & Fu, Y. X. Stairway Plot 2: demographic history inference with folded SNP frequency spectra. Genome Biol. 21, 280 (2020).") and bam files for each pear accession were filtered and then used for calculating the site-frequency spectrum using ANGSD[127](https://www.nature.com/articles/s41588-025-02273-4#ref-CR127 "Korneliussen, T. S. et al. ANGSD: analysis of next generation sequencing data. BMC Bioinf. 15, 356 (2014)."), with the ‘-doSaf 1 -minMapQ 1 -minQ 20’ option. A stairway plot was performed with 200 bootstraps, 3.9 × 10−9 substitutions per site per year and 7.5 years per generation.

### SV GWAS

We performed GWASs for SSC, FTD, SCC and SFW in 362 accessions using 83,369 high-quality SV markers with GAPIT (v.3.4)[130](https://www.nature.com/articles/s41588-025-02273-4#ref-CR130 "Wang, J. & Zhang, Z. GAPIT version 3: boosting power and accuracy for genomic association and prediction. Genom. Proteom. Bioinform. 19, 629–640 (2021).") and the first three PCs and kinship matrix as covariates. The genome-wide significance thresholds (1.20 × 10−5) were determined using a uniform threshold of 1/_n_, where _n_ is the adequate number of SVs. RNA-seq data from four pear accessions at five fruit development stages (S1–S5) were downloaded for evaluating _PybZIP6_ gene expression[131](https://www.nature.com/articles/s41588-025-02273-4#ref-CR131 "Zhang, M. Y. et al. Distinct transcriptome profiles reveal gene expression patterns during fruit development and maturation in five main cultivated species of pear (Pyrus L.). Sci. Rep. 6, 28130 (2016).") and RNA-seq data of samples with SCC were downloaded for evaluating _PyC3H18_ gene expression[132](https://www.nature.com/articles/s41588-025-02273-4#ref-CR132 "Wang, R. Z. et al. A systems genetics approach reveals PbrNSC as a regulator of lignin and cellulose biosynthesis in stone cells of pear fruit. Genome Biol. 22, 313 (2021)."). STAR (v.2.7.9)[113](https://www.nature.com/articles/s41588-025-02273-4#ref-CR113 "Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013).") and StringTie (v.2.1.7)[114](https://www.nature.com/articles/s41588-025-02273-4#ref-CR114 "Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat. Biotechnol. 33, 290–295 (2015).") were used to align the reads and quantify the gene expression with the same parameters mentioned earlier (DS HapA as reference).

### SV association with gene expression

A total of 110 samples (consisting of 62 Asian pears and 38 European pears) (Supplementary Table [19](https://www.nature.com/articles/s41588-025-02273-4#MOESM3)) were collected and RNA-seq data were generated to identify the SV effect on gene expression. The FPKM values were calculated using STAR (v.2.7.9)[113](https://www.nature.com/articles/s41588-025-02273-4#ref-CR113 "Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013).") and StringTie (v.2.1.7)[114](https://www.nature.com/articles/s41588-025-02273-4#ref-CR114 "Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat. Biotechnol. 33, 290–295 (2015).") with the same parameters as above (DS HapA as reference). Then, 21,581 genes with a mean FPKM > 1 were retained for further analysis. The PanMarker program[50](https://www.nature.com/articles/s41588-025-02273-4#ref-CR50 "Chen, S. et al. Gene mining and genomics-assisted breeding empowered by the pangenome of tea plant Camellia sinensis. Nat. Plants 9, 1986–1999 (2023).") was used to identify SVs significantly associated with gene expression. SVs that overlapped with CDSs or the upstream 2-kb region were selected. All samples were classified into three groups, with groups 0, 1 and 2 representing the reference (Ref), heterozygous (Het) and alternative (Alt) genotypes, respectively. The Grubbs’ test was used to remove outliers (_P_ < 0.05) and Levene’s test was applied to assess the equality of variances for a variable calculated for each of the two groups (_P_ < 0.05). In addition, the two-sided Student’s _t_\-test was used to determine whether there was a significant difference between the means of groups 0 and 2 (_P_ < 0.05) and the locus was retained if the nonoutlier values in one of the two groups was strictly larger than that in another. We also compared the difference between group 1 and group 0 (or group 2) to evaluate the effect of heterozygous SV on the difference in gene expression.

### RNA extraction, gene cloning and RT–qPCR assays

Total RNA was isolated using the Plant Total RNA Isolation Kit Plus (Foregene) following the manufacturer’s instructions. First-strand complementary DNA was synthesized from 2 μg of total RNA using the TransScript One-Step gDNA Removal and cDNA Synthesis SuperMix (TransGen), according to the manufacturer’s protocol. All oligonucleotide primers used are listed in Supplementary Table [21](https://www.nature.com/articles/s41588-025-02273-4#MOESM3). Amplification for gene cloning was performed as follows: an initial denaturation step at 95 °C for 3 min, followed by 35 cycles of 95 °C for 15 s, 58 °C for 15 s and 72 °C for 90 s, with a final elongation step at 72 °C for 5 min.

### Dual-luciferase reporter assay

The 2-kb promoter regions (5ʹ-upstream of the ATG start codon) of _PyACS1_ from DS and MRB were cloned and inserted into the pGreen II 0800-LUC binary vector for use as reporter genes. The _PybHLH94_ CDS was amplified by PCR using a pair of gene-specific primers and inserted into the pCAMBIA1300-35S-EGFP vector by homologous recombination. The recombinant plasmids constructed in the pCAMBIA1300-35S-EGFP vector and pGreen II 0800-LUC vector were transformed into the GV3101 strain of _A. tumefaciens_. _A. tumefaciens_ GV3101 cells containing these vectors or the control vector were resuspended in the infiltration solution containing 10 mmol of 2-(_N_\-morpholino) ethanesulfonic acid hydrate, 10 mmol of MgCl2 and 0.2 μmol of acetosyringone at an optical density at 600 nm = 1.0, then cultured at 25 °C with shaking at 70 rpm for 4 h before infiltration. The _Agrobacterium_ cultures containing the pGreen II 0800-LUC and the pCAMBIA1300-35S-EGFP recombinant plasmids were mixed in a 1:9 ratio and mixtures were infiltrated into 4-week-old _Nicotiana benthamiana_ leaves for transient gene expression assays. The infiltrated tobacco plants were placed in the dark for 12 h and then grown under normal conditions for 48 h, after which the firefly luciferase (LUC) and _Renilla_ luciferase (REN) activities were assayed using dual-luciferase assay reagents (Promega). Determination of fluorescence values was performed using a microplate reader (SpectraMax Paradigm).

### Electromobility shift assay analysis

The CDSs of _PybHLH94_ were cloned into the pET-32a+ vector to generate His fusion protein constructs (PybHLH94–His). Purification of the PybHLH94–His fusion proteins was performed using HIS Sefinose resin (Sangon Biotech). Mutated probes and unlabeled probes were used for competition in the EMSA experiments. Gene promoter sequences containing putative _cis_\-element-binding sites were labeled at the 5ʹ-end using a biotin DNA-labeling kit (Beyotime Biotechnology), as directed by the manufacturer. EMSA was performed using a chemiluminescent EMSA kit (Beyotime Biotechnology).

### Construction of transgenic tomato lines

For the construction of stable transgenic lines, the full-length CDS of _PyACS1_ from MRB was inserted into the pBI101 vector by homologous recombination. The recombinant plasmids were transformed into _A. tumefaciens_ strain GV3101 using the freeze–thaw method. Transgenic tomato plants ‘Micro-Tom’ were obtained using an improvement of the method used in ref. [133](https://www.nature.com/articles/s41588-025-02273-4#ref-CR133 "Lin, T. B. Resveratrol production of tomato transformed with a vitis stilbene synthase gene. Life Sci. Res 7, 262–266 (2003).") (Supplementary Method [8](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)). Fruit firmness was quantitatively evaluated using a Brookfield CT3 Texture Analyzer following standardized protocols. Ethylene production dynamics were analyzed using a closed chamber methodology (Supplementary Method [9](https://www.nature.com/articles/s41588-025-02273-4#MOESM1)).

### Statistical analysis

The statistical tests used are described throughout the article and in the figure legends. The _χ_2 test was performed using the chisq.test function in R. The two-sided Student’s _t_\-test was calculated using Student’s _t_\-test in R. Specifically, the _χ_2 test that is FDR corrected was used for comparison of SV occurrence frequency between Asian and European pears.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02273-4#MOESM2) linked to this article.

Data availability
-----------------

Raw genome reads and RNA-seq data of DS and MRB can be downloaded from the National Center for Biotechnology Information (NCBI) BioProject (accession no. [PRJNA1237550](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1237550)). Raw WGS data of 137 pear accessions and RNA-seq data of 110 pear accessions generated in this study can be downloaded from the NCBI BioProject (accession nos. [PRJNA989672](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA989672) and [PRJNA1237442](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1237442)). Raw WGS data of previously released pear accessions were downloaded from the NCBI BioProject (accession nos. [PRJNA563813](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA563813) and [PRJNA381668](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA381668)). The genome sequences were archived in the NCBI under accession nos. [PRJNA1254919](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1254919), [PRJNA1254920](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1254920), [PRJNA1254911](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1254911) and [PRJNA1254912](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1254912). The files for genome assembly and annotation in this study are also available in the Pear Genomics Database ([http://pyrusgdb.sdau.edu.cn/](http://pyrusgdb.sdau.edu.cn/)). All data are publicly available. [Source data](https://www.nature.com/articles/s41588-025-02273-4#Sec33) are provided with this paper.

Code availability
-----------------

Customized code and scripts supporting this work are available via GitHub at [https://github.com/bioinformatic1996512/DS-MRB-genome](https://github.com/bioinformatic1996512/DS-MRB-genome) and Zenodo at [https://doi.org/10.5281/zenodo.15113930](https://doi.org/10.5281/zenodo.15113930) (ref. [134](https://www.nature.com/articles/s41588-025-02273-4#ref-CR134 "Sun, M. Bioinformatic1996512/DS-MRB-genome: code for DS and MRB genome (DSMRBv1.0). Zenodo 
https://doi.org/10.5281/zenodo.15113931
(2025).")).

References
----------

1.  Zhou, Q. et al. Haplotype-resolved genome analyses of a heterozygous diploid potato. _Nat. Genet._ **52**, 1018–1023 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFegurvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32989320)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7527274)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haplotype-resolved%20genome%20analyses%20of%20a%20heterozygous%20diploid%20potato&journal=Nat.%20Genet.&volume=52&pages=1018-1023&publication_year=2020&author=Zhou%2CQ) 
    
2.  Murase, K. et al. Mechanism of self/nonself-discrimination in _Brassica_ self-incompatibility. _Nat. Commun._ **11**, 4916 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFyqt7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33004803)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530648)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mechanism%20of%20self%2Fnonself-discrimination%20in%20Brassica%20self-incompatibility&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Murase%2CK) 
    
3.  Chen, J. et al. Phosphatidic acid counteracts S-RNase signaling in pollen by stabilizing the actin cytoskeleton. _Plant Cell_ **30**, 1023–1039 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFOjtbzN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29716992)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002197)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phosphatidic%20acid%20counteracts%20S-RNase%20signaling%20in%20pollen%20by%20stabilizing%20the%20actin%20cytoskeleton&journal=Plant%20Cell&volume=30&pages=1023-1039&publication_year=2018&author=Chen%2CJ) 
    
4.  Huang, X. H. et al. Genomic architecture of heterosis for yield traits in rice. _Nature_ **537**, 629–633 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVyhsb3K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27602511)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20architecture%20of%20heterosis%20for%20yield%20traits%20in%20rice&journal=Nature&volume=537&pages=629-633&publication_year=2016&author=Huang%2CXH) 
    
5.  Sun, X. P. et al. Phased diploid genome assemblies and pan-genomes provide insights into the genetic history of apple domestication. _Nat. Genet._ **52**, 1423–1432 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1GjsbrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33139952)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7728601)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phased%20diploid%20genome%20assemblies%20and%20pan-genomes%20provide%20insights%20into%20the%20genetic%20history%20of%20apple%20domestication&journal=Nat.%20Genet.&volume=52&pages=1423-1432&publication_year=2020&author=Sun%2CXP) 
    
6.  Zhang, X. T. et al. Haplotype-resolved genome assembly provides insights into evolutionary history of the tea plant _Camellia sinensis_. _Nat. Genet._ **53**, 1250–1259 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFOntLnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34267370)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8346365)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haplotype-resolved%20genome%20assembly%20provides%20insights%20into%20evolutionary%20history%20of%20the%20tea%20plant%20Camellia%20sinensis&journal=Nat.%20Genet.&volume=53&pages=1250-1259&publication_year=2021&author=Zhang%2CXT) 
    
7.  Hu, G. B. et al. Two divergent haplotypes from a highly heterozygous lychee genome suggest independent domestication events for early and late-maturing cultivars. _Nat. Genet._ **54**, 73–83 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xjtl2mug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34980919)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8755541)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Two%20divergent%20haplotypes%20from%20a%20highly%20heterozygous%20lychee%20genome%20suggest%20independent%20domestication%20events%20for%20early%20and%20late-maturing%20cultivars&journal=Nat.%20Genet.&volume=54&pages=73-83&publication_year=2022&author=Hu%2CGB) 
    
8.  Hu, W. et al. Allele-defined genome reveals biallelic differentiation during cassava evolution. _Mol. Plant_ **14**, 851–854 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVGitLrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33866024)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Allele-defined%20genome%20reveals%20biallelic%20differentiation%20during%20cassava%20evolution&journal=Mol.%20Plant&volume=14&pages=851-854&publication_year=2021&author=Hu%2CW) 
    
9.  Zhang, Q. et al. Genomic insights into the recent chromosome reduction of autopolyploid sugarcane _Saccharum spontaneum_. _Nat. Genet._ **54**, 885–896 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVGgsbzK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35654976)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20insights%20into%20the%20recent%20chromosome%20reduction%20of%20autopolyploid%20sugarcane%20Saccharum%20spontaneum&journal=Nat.%20Genet.&volume=54&pages=885-896&publication_year=2022&author=Zhang%2CQ) 
    
10.  Wu, J. et al. Diversification and independent domestication of Asian and European pears. _Genome Biol._ **19**, 77 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29890997)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5996476)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Diversification%20and%20independent%20domestication%20of%20Asian%20and%20European%20pears&journal=Genome%20Biol.&volume=19&publication_year=2018&author=Wu%2CJ) 
    
11.  Wu, J. et al. The genome of the pear (_Pyrus bretschneideri_ Rehd.). _Genome Res._ **23**, 396–408 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXitVemtLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23149293)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3561880)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genome%20of%20the%20pear%20%28Pyrus%20bretschneideri%20Rehd.%29&journal=Genome%20Res.&volume=23&pages=396-408&publication_year=2013&author=Wu%2CJ) 
    
12.  Chagne, D. et al. The draft genome dequence of European pear (_Pyrus communis_ L. ‘Bartlett’). _PLoS ONE_ **9**, e92644 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24699266)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3974708)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20draft%20genome%20dequence%20of%20European%20pear%20%28Pyrus%20communis%20L.%20%E2%80%98Bartlett%E2%80%99%29&journal=PLoS%20ONE&volume=9&publication_year=2014&author=Chagne%2CD) 
    
13.  Linsmith, G. et al. Pseudo-chromosome-length genome assembly of a double haploid ‘Bartlett’ pear (_Pyrus communis_ L.). _Gigascience_ **8**, giz138 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31816089)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6901071)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pseudo-chromosome-length%20genome%20assembly%20of%20a%20double%20haploid%20%E2%80%98Bartlett%E2%80%99%20pear%20%28Pyrus%20communis%20L.%29&journal=Gigascience&volume=8&publication_year=2019&author=Linsmith%2CG) 
    
14.  Shi, D. Q. et al. Single-pollen-cell sequencing for gamete-based phased diploid genome assembly in plants. _Genome Res._ **29**, 1889–1899 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVyqsr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31649061)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6836740)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-pollen-cell%20sequencing%20for%20gamete-based%20phased%20diploid%20genome%20assembly%20in%20plants&journal=Genome%20Res.&volume=29&pages=1889-1899&publication_year=2019&author=Shi%2CDQ) 
    
15.  Qi, Y. et al. Telomere-to-telomere genome assembly of two representative Asian and European pear cultivars. _Sci. Data_ **11**, 1170 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitlelu7zN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39461942)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11513147)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomere-to-telomere%20genome%20assembly%20of%20two%20representative%20Asian%20and%20European%20pear%20cultivars&journal=Sci.%20Data&volume=11&publication_year=2024&author=Qi%2CY) 
    
16.  Chen, X. et al. Genome-wide genetic diversity and IBD analysis reveals historic dissemination routes of pear in China. _Tree Genet Genomes_ **18**, 1 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20genetic%20diversity%20and%20IBD%20analysis%20reveals%20historic%20dissemination%20routes%20of%20pear%20in%20China&journal=Tree%20Genet%20Genomes&volume=18&publication_year=2022&author=Chen%2CX) 
    
17.  Zhou, Y. et al. Graph pangenome captures missing heritability and empowers tomato breeding. _Nature_ **606**, 527–534 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFWhu7bO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35676474)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9200638)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Graph%20pangenome%20captures%20missing%20heritability%20and%20empowers%20tomato%20breeding&journal=Nature&volume=606&pages=527-534&publication_year=2022&author=Zhou%2CY) 
    
18.  Cheng, H. Y. et al. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. _Nat. Methods_ **18**, 170–175 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1OntL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33526886)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7961889)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haplotype-resolved%20de%20novo%20assembly%20using%20phased%20assembly%20graphs%20with%20hifiasm&journal=Nat.%20Methods&volume=18&pages=170-175&publication_year=2021&author=Cheng%2CHY) 
    
19.  Rautiainen, M. et al. Telomere-to-telomere assembly of diploid chromosomes with Verkko. _Nat. Biotechnol._ **41**, 1474–1482 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjsVKmtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36797493)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10427740)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomere-to-telomere%20assembly%20of%20diploid%20chromosomes%20with%20Verkko&journal=Nat.%20Biotechnol.&volume=41&pages=1474-1482&publication_year=2023&author=Rautiainen%2CM) 
    
20.  Nurk, S. et al. HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads. _Genome Res._ **30**, 1291–1305 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXltl2isw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32801147)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7545148)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HiCanu%3A%20accurate%20assembly%20of%20segmental%20duplications%2C%20satellites%2C%20and%20allelic%20variants%20from%20high-fidelity%20long%20reads&journal=Genome%20Res.&volume=30&pages=1291-1305&publication_year=2020&author=Nurk%2CS) 
    
21.  Ines, D. et al. CDC48 in plants and its emerging function in plant immunity. _Trends Plant Sci._ **29**, 786–798 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXisVSjsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38218650)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CDC48%20in%20plants%20and%20its%20emerging%20function%20in%20plant%20immunity&journal=Trends%20Plant%20Sci.&volume=29&pages=786-798&publication_year=2024&author=Ines%2CD) 
    
22.  Griesser, M. et al. Multi-substrate flavonol _O_\-glucosyltransferases from strawberry (_Fragaria_ x _ananassa_) achene and receptacle. _J. Exp. Bot._ **59**, 2611–2625 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXot1Witrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18487633)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2486459)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-substrate%20flavonol%20O-glucosyltransferases%20from%20strawberry%20%28Fragaria%20x%20ananassa%29%20achene%20and%20receptacle&journal=J.%20Exp.%20Bot.&volume=59&pages=2611-2625&publication_year=2008&author=Griesser%2CM) 
    
23.  Weber, M. et al. Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome. _Nat. Genet._ **39**, 457–466 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjsV2hsrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17334365)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distribution%2C%20silencing%20potential%20and%20evolutionary%20impact%20of%20promoter%20DNA%20methylation%20in%20the%20human%20genome&journal=Nat.%20Genet.&volume=39&pages=457-466&publication_year=2007&author=Weber%2CM) 
    
24.  Stroud, H. et al. Comprehensive analysis of silencing mutants reveals complex regulation of the _Arabidopsis_ methylome. _Cell_ **152**, 352–364 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXmvVKhtw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23313553)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3597350)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20analysis%20of%20silencing%20mutants%20reveals%20complex%20regulation%20of%20the%20Arabidopsis%20methylome&journal=Cell&volume=152&pages=352-364&publication_year=2013&author=Stroud%2CH) 
    
25.  Li, J. et al. Natural variations in the PbCPK28 promoter regulate sugar content through interaction with PbTST4 and PbVHA-A1 in pear. _Plant J._ **114**, 124–141 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXkvVSitrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36710644)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Natural%20variations%20in%20the%20PbCPK28%20promoter%20regulate%20sugar%20content%20through%20interaction%20with%20PbTST4%20and%20PbVHA-A1%20in%20pear&journal=Plant%20J.&volume=114&pages=124-141&publication_year=2023&author=Li%2CJ) 
    
26.  Lama, K. et al. Extreme sugar accumulation in late fig ripening is accompanied by global changes in sugar metabolism and transporter gene expression. _Physiol. Plant_ **174**, e13648 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XkvFWns7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35150009)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9305157)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extreme%20sugar%20accumulation%20in%20late%20fig%20ripening%20is%20accompanied%20by%20global%20changes%20in%20sugar%20metabolism%20and%20transporter%20gene%20expression&journal=Physiol.%20Plant&volume=174&publication_year=2022&author=Lama%2CK) 
    
27.  Xue, C. et al. PbrMYB169 positively regulates lignification of stone cells in pear fruit. _J. Exp. Bot._ **70**, 1801–1814 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVGmtr%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30715420)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PbrMYB169%20positively%20regulates%20lignification%20of%20stone%20cells%20in%20pear%20fruit&journal=J.%20Exp.%20Bot.&volume=70&pages=1801-1814&publication_year=2019&author=Xue%2CC) 
    
28.  Barros, J. et al. 4-Coumarate 3-hydroxylase in the lignin biosynthesis pathway is a cytosolic ascorbate peroxidase. _Nat. Commun._ **10**, 1994 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31040279)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6491607)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=4-Coumarate%203-hydroxylase%20in%20the%20lignin%20biosynthesis%20pathway%20is%20a%20cytosolic%20ascorbate%20peroxidase&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Barros%2CJ) 
    
29.  Yang, C. Q. et al. Transcriptional regulation of plant secondary metabolism. _J. Integr. Plant Biol._ **54**, 703–712 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXosVKnug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22947222)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptional%20regulation%20of%20plant%20secondary%20metabolism&journal=J.%20Integr.%20Plant%20Biol.&volume=54&pages=703-712&publication_year=2012&author=Yang%2CCQ) 
    
30.  Dodds, P. N. & Rathjen, J. P. Plant immunity: towards an integrated view of plant-pathogen interactions. _Nat. Rev. Genet._ **11**, 539–548 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXovFOhtrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20585331)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plant%20immunity%3A%20towards%20an%20integrated%20view%20of%20plant-pathogen%20interactions&journal=Nat.%20Rev.%20Genet.&volume=11&pages=539-548&publication_year=2010&author=Dodds%2CPN&author=Rathjen%2CJP) 
    
31.  Zhu, S. et al. The snapdragon genomes reveal the evolutionary dynamics of the S-locus supergene. _Mol. Biol. Evol._ **40**, msad080 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFylurzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37014787)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10118302)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20snapdragon%20genomes%20reveal%20the%20evolutionary%20dynamics%20of%20the%20S-locus%20supergene&journal=Mol.%20Biol.%20Evol.&volume=40&publication_year=2023&author=Zhu%2CS) 
    
32.  Yang, Q. et al. Heterochromatic and genetic features are consistent with recombination suppression of the self-incompatibility locus in _Antirrhinum_. _Plant J._ **51**, 140–151 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXotVyru7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17521415)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Heterochromatic%20and%20genetic%20features%20are%20consistent%20with%20recombination%20suppression%20of%20the%20self-incompatibility%20locus%20in%20Antirrhinum&journal=Plant%20J.&volume=51&pages=140-151&publication_year=2007&author=Yang%2CQ) 
    
33.  Cohen, K. M. et al. The ICS International Chronostratigraphic Chart. _Episodes_ **36**, 199–204 (2013).
    
34.  Guo, J. F. et al. Genomic clines across the species boundary between a hybrid pine and its progenitor in the eastern Tibetan Plateau. _Plant Commun._ **4**, 100574 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXotlChtrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36906801)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10363505)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20clines%20across%20the%20species%20boundary%20between%20a%20hybrid%20pine%20and%20its%20progenitor%20in%20the%20eastern%20Tibetan%20Plateau&journal=Plant%20Commun.&volume=4&publication_year=2023&author=Guo%2CJF) 
    
35.  Petit, J. R. et al. Climate and atmospheric history of the past 420,000 years from the Vostok ice core, Antarctica. _Nature_ **399**, 429–436 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXjvVyqu78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Climate%20and%20atmospheric%20history%20of%20the%20past%20420%2C000%20years%20from%20the%20Vostok%20ice%20core%2C%20Antarctica&journal=Nature&volume=399&pages=429-436&publication_year=1999&author=Petit%2CJR) 
    
36.  Liao, L. et al. Unraveling a genetic roadmap for improved taste in the domesticated apple. _Mol. Plant_ **14**, 1454–1471 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlyiurrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34022440)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unraveling%20a%20genetic%20roadmap%20for%20improved%20taste%20in%20the%20domesticated%20apple&journal=Mol.%20Plant&volume=14&pages=1454-1471&publication_year=2021&author=Liao%2CL) 
    
37.  Xu, S. et al. Auxin inhibits lignin and cellulose biosynthesis in stone cells of pear fruit via the PbrARF13-PbrNSC-PbrMYB132 transcriptional regulatory cascade. _Plant Biotechnol. J._ **21**, 1408–1425 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXnvVaku7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37031416)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10281602)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Auxin%20inhibits%20lignin%20and%20cellulose%20biosynthesis%20in%20stone%20cells%20of%20pear%20fruit%20via%20the%20PbrARF13-PbrNSC-PbrMYB132%20transcriptional%20regulatory%20cascade&journal=Plant%20Biotechnol.%20J.&volume=21&pages=1408-1425&publication_year=2023&author=Xu%2CS) 
    
38.  Gong, X. et al. Multi-omics analyses reveal stone cell distribution pattern in pear fruit. _Plant J._ **113**, 626–642 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1Wqurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36546867)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-omics%20analyses%20reveal%20stone%20cell%20distribution%20pattern%20in%20pear%20fruit&journal=Plant%20J.&volume=113&pages=626-642&publication_year=2023&author=Gong%2CX) 
    
39.  Xue, C. et al. PbrmiR397a regulates lignification during stone cell development in pear fruit. _Plant Biotechnol. J._ **17**, 103–117 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXotVWqsg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29754465)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PbrmiR397a%20regulates%20lignification%20during%20stone%20cell%20development%20in%20pear%20fruit&journal=Plant%20Biotechnol.%20J.&volume=17&pages=103-117&publication_year=2019&author=Xue%2CC) 
    
40.  Li, Y. et al. Genomic analyses of an extensive collection of wild and cultivated accessions provide new insights into peach breeding history. _Genome Biol._ **20**, 36 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30791928)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6383288)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20analyses%20of%20an%20extensive%20collection%20of%20wild%20and%20cultivated%20accessions%20provide%20new%20insights%20into%20peach%20breeding%20history&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Li%2CY) 
    
41.  Azzi, L. et al. Fruit growth-related genes in tomato. _J. Exp. Bot._ **66**, 1075–1086 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitVWhsLnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25573859)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fruit%20growth-related%20genes%20in%20tomato&journal=J.%20Exp.%20Bot.&volume=66&pages=1075-1086&publication_year=2015&author=Azzi%2CL) 
    
42.  Jiang, S. et al. The UBP14-CDKB1;1-CDKG2 cascade controls endore-duplication and cell growth in _Arabidopsis_. _Plant Cell_ **34**, 1308–1325 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34999895)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8972217)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20UBP14-CDKB1%3B1-CDKG2%20cascade%20controls%20endore-duplication%20and%20cell%20growth%20in%20Arabidopsis&journal=Plant%20Cell&volume=34&pages=1308-1325&publication_year=2022&author=Jiang%2CS) 
    
43.  Liu, S. et al. Tomato AUXIN RESPONSE FACTOR 5 regulates fruit set and development via the mediation of auxin and gibberellin signaling. _Sci. Rep._ **8**, 2971 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29445121)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5813154)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tomato%20AUXIN%20RESPONSE%20FACTOR%205%20regulates%20fruit%20set%20and%20development%20via%20the%20mediation%20of%20auxin%20and%20gibberellin%20signaling&journal=Sci.%20Rep.&volume=8&publication_year=2018&author=Liu%2CS) 
    
44.  Fang, D. D. et al. An EMS-induced mutation in a tetratricopeptide repeat-like superfamily protein gene (Ghir\_A12G008870) on chromosome A12 is responsible for the liy short fiber phenotype in cotton. _Theor. Appl. Genet._ **133**, 271–282 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFynu7nN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31624873)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20EMS-induced%20mutation%20in%20a%20tetratricopeptide%20repeat-like%20superfamily%20protein%20gene%20%28Ghir_A12G008870%29%20on%20chromosome%20A12%20is%20responsible%20for%20the%20liy%20short%20fiber%20phenotype%20in%20cotton&journal=Theor.%20Appl.%20Genet.&volume=133&pages=271-282&publication_year=2020&author=Fang%2CDD) 
    
45.  Zhang, M. et al. A tetratricopeptide repeat domain-containing protein SSR1 located in mitochondria is involved in root development and auxin polar transport in _Arabidopsis_. _Plant J._ **83**, 582–599 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFWmur7I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26072661)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20tetratricopeptide%20repeat%20domain-containing%20protein%20SSR1%20located%20in%20mitochondria%20is%20involved%20in%20root%20development%20and%20auxin%20polar%20transport%20in%20Arabidopsis&journal=Plant%20J.&volume=83&pages=582-599&publication_year=2015&author=Zhang%2CM) 
    
46.  Wang, H. et al. A pear S1-bZIP transcription factor PpbZIP44 modulates carbohydrate metabolism, amino acid, and flavonoid accumulation in fruits. _Hortic. Res._ **10**, uhad140 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsV2rtbnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37575657)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10421730)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20pear%20S1-bZIP%20transcription%20factor%20PpbZIP44%20modulates%20carbohydrate%20metabolism%2C%20amino%20acid%2C%20and%20flavonoid%20accumulation%20in%20fruits&journal=Hortic.%20Res.&volume=10&publication_year=2023&author=Wang%2CH) 
    
47.  Jia, L. et al. PbrbZIP15 promotes sugar accumulation in pear via activating the transcription of the glucose isomerase gene PbrXylA1. _Plant J._ **117**, 1392–1412 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFahsr3J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38044792)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PbrbZIP15%20promotes%20sugar%20accumulation%20in%20pear%20via%20activating%20the%20transcription%20of%20the%20glucose%20isomerase%20gene%20PbrXylA1&journal=Plant%20J.&volume=117&pages=1392-1412&publication_year=2024&author=Jia%2CL) 
    
48.  Zhang, H., Tao, X., Fan, X., Zhang, S. & Qin, G. PpybZIP43 contributes to sucrose synthesis in pear fruits by activating PpySPS3 expression and interacts with PpySTOP1. _Physiol. Plant_ **174**, e13732 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhs1ajtLfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35689502)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PpybZIP43%20contributes%20to%20sucrose%20synthesis%20in%20pear%20fruits%20by%20activating%20PpySPS3%20expression%20and%20interacts%20with%20PpySTOP1&journal=Physiol.%20Plant&volume=174&publication_year=2022&author=Zhang%2CH&author=Tao%2CX&author=Fan%2CX&author=Zhang%2CS&author=Qin%2CG) 
    
49.  Shang, L. et al. A super pan-genomic landscape of rice. _Cell Res._ **32**, 878–896 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1yit7rL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35821092)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9525306)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20super%20pan-genomic%20landscape%20of%20rice&journal=Cell%20Res.&volume=32&pages=878-896&publication_year=2022&author=Shang%2CL) 
    
50.  Chen, S. et al. Gene mining and genomics-assisted breeding empowered by the pangenome of tea plant _Camellia sinensis_. _Nat. Plants_ **9**, 1986–1999 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFGit77I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38012346)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20mining%20and%20genomics-assisted%20breeding%20empowered%20by%20the%20pangenome%20of%20tea%20plant%20Camellia%20sinensis&journal=Nat.%20Plants&volume=9&pages=1986-1999&publication_year=2023&author=Chen%2CS) 
    
51.  Hiwasa, K. et al. Ethylene is required for both the initiation and progression of softening in pear (_Pyrus communis_ L.) fruit. _J. Exp. Bot._ **54**, 771–779 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXit1aru7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12554720)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ethylene%20is%20required%20for%20both%20the%20initiation%20and%20progression%20of%20softening%20in%20pear%20%28Pyrus%20communis%20L.%29%20fruit&journal=J.%20Exp.%20Bot.&volume=54&pages=771-779&publication_year=2003&author=Hiwasa%2CK) 
    
52.  Paul, V., Pandey, R. & Srivastava, G. C. The fading distinctions between classical patterns of ripening in climacteric and non-climacteric fruit and the ubiquity of ethylene—an overview. _J. Food Sci. Technol._ **49**, 1–21 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsF2rurjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23572821)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20fading%20distinctions%20between%20classical%20patterns%20of%20ripening%20in%20climacteric%20and%20non-climacteric%20fruit%20and%20the%20ubiquity%20of%20ethylene%E2%80%94an%20overview&journal=J.%20Food%20Sci.%20Technol.&volume=49&pages=1-21&publication_year=2012&author=Paul%2CV&author=Pandey%2CR&author=Srivastava%2CGC) 
    
53.  Song, Q. et al. Diurnal down-regulation of ethylene biosynthesis mediates biomass heterosis. _Proc. Natl Acad. Sci. USA_ **115**, 5606–5611 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVOltLfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29735680)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6003462)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Diurnal%20down-regulation%20of%20ethylene%20biosynthesis%20mediates%20biomass%20heterosis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=115&pages=5606-5611&publication_year=2018&author=Song%2CQ) 
    
54.  Liu, C. et al. The OsWRKY72-OsAAT30/OsGSTU26 module mediates reactive oxygen species scavenging to drive heterosis for salt tolerance in hybrid rice. _J. Integr. Plant Biol._ **66**, 709–730 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXmvVyrt74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38483018)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20OsWRKY72-OsAAT30%2FOsGSTU26%20module%20mediates%20reactive%20oxygen%20species%20scavenging%20to%20drive%20heterosis%20for%20salt%20tolerance%20in%20hybrid%20rice&journal=J.%20Integr.%20Plant%20Biol.&volume=66&pages=709-730&publication_year=2024&author=Liu%2CC) 
    
55.  Dombrecht, B. et al. MYC2 differentially modulates diverse jasmonate-dependent functions in _Arabidopsis_. _Plant Cell_ **19**, 2225–2245 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtValurzO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17616737)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1955694)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MYC2%20differentially%20modulates%20diverse%20jasmonate-dependent%20functions%20in%20Arabidopsis&journal=Plant%20Cell&volume=19&pages=2225-2245&publication_year=2007&author=Dombrecht%2CB) 
    
56.  Hu, D. G. et al. The regulatory module MdPUB29-MdbHLH3 connects ethylene biosynthesis with fruit quality in apple. _N. Phytol._ **221**, 1966–1982 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisFChtrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20regulatory%20module%20MdPUB29-MdbHLH3%20connects%20ethylene%20biosynthesis%20with%20fruit%20quality%20in%20apple&journal=N.%20Phytol.&volume=221&pages=1966-1982&publication_year=2019&author=Hu%2CDG) 
    
57.  He, L. et al. DNA methylation-free _Arabidopsis_ reveals crucial roles of DNA methylation in regulating gene expression and development. _Nat. Commun._ **13**, 1335 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XntVamtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35288562)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8921224)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20methylation-free%20Arabidopsis%20reveals%20crucial%20roles%20of%20DNA%20methylation%20in%20regulating%20gene%20expression%20and%20development&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=He%2CL) 
    
58.  Zhang, J. et al. Autotetraploid rice methylome analysis reveals methylation variation of transposable elements and their effects on gene expression. _Proc. Natl Acad. Sci. USA_ **112**, E7022–E7029 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFWqtbfN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26621743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687577)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autotetraploid%20rice%20methylome%20analysis%20reveals%20methylation%20variation%20of%20transposable%20elements%20and%20their%20effects%20on%20gene%20expression&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=112&pages=E7022-E7029&publication_year=2015&author=Zhang%2CJ) 
    
59.  Shao, L. et al. Patterns of genome-wide allele-specific expression in hybrid rice and the implications on the genetic basis of heterosis. _Proc. Natl Acad. Sci. USA_ **116**, 5653–5658 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlsFeksrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30833384)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6431163)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Patterns%20of%20genome-wide%20allele-specific%20expression%20in%20hybrid%20rice%20and%20the%20implications%20on%20the%20genetic%20basis%20of%20heterosis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=116&pages=5653-5658&publication_year=2019&author=Shao%2CL) 
    
60.  Han, X. et al. Two haplotype-resolved, gap-free genome assemblies for _Actinidia latifolia_ and _Actinidia chinensis_ shed light on the regulatory mechanisms of vitamin C and sucrose metabolism in kiwifruit. _Mol. Plant_ **16**, 452–470 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVymsbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36588343)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Two%20haplotype-resolved%2C%20gap-free%20genome%20assemblies%20for%20Actinidia%20latifolia%20and%20Actinidia%20chinensis%20shed%20light%20on%20the%20regulatory%20mechanisms%20of%20vitamin%20C%20and%20sucrose%20metabolism%20in%20kiwifruit&journal=Mol.%20Plant&volume=16&pages=452-470&publication_year=2023&author=Han%2CX) 
    
61.  Hu, H. et al. Allele-specific expression reveals multiple paths to highland adaptation in maize. _Mol. Biol. Evol._ **39**, msac239 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXptlyqsbY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36327321)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9692238)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Allele-specific%20expression%20reveals%20multiple%20paths%20to%20highland%20adaptation%20in%20maize&journal=Mol.%20Biol.%20Evol.&volume=39&publication_year=2022&author=Hu%2CH) 
    
62.  Swetha, C. et al. Major domestication-related phenotypes in indica rice are due to loss of miRNA-mediated laccase silencing. _Plant Cell_ **30**, 2649–2662 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmvFGnsLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30341147)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6305975)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Major%20domestication-related%20phenotypes%20in%20indica%20rice%20are%20due%20to%20loss%20of%20miRNA-mediated%20laccase%20silencing&journal=Plant%20Cell&volume=30&pages=2649-2662&publication_year=2018&author=Swetha%2CC) 
    
63.  Nham, N. T. et al. A transcriptome approach towards understanding the development of ripening capacity in ‘Bartlett’ pears (_Pyrus communis_ L.). _BMC Genom._ **16**, 762 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20transcriptome%20approach%20towards%20understanding%20the%20development%20of%20ripening%20capacity%20in%20%E2%80%98Bartlett%E2%80%99%20pears%20%28Pyrus%20communis%20L.%29&journal=BMC%20Genom.&volume=16&publication_year=2015&author=Nham%2CNT) 
    
64.  Li, D. et al. The multi-omics basis of potato heterosis. _J. Integr. Plant Biol._ **64**, 671–687 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhtl2lurbK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34963038)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20multi-omics%20basis%20of%20potato%20heterosis&journal=J.%20Integr.%20Plant%20Biol.&volume=64&pages=671-687&publication_year=2022&author=Li%2CD) 
    
65.  Ma, X. et al. An enhanced network of energy metabolism, lysine acetylation, and growth-promoting protein accumulation is associated with heterosis in elite hybrid rice. _Plant Commun._ **4**, 100560 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXlslWitr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36774536)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10363507)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20enhanced%20network%20of%20energy%20metabolism%2C%20lysine%20acetylation%2C%20and%20growth-promoting%20protein%20accumulation%20is%20associated%20with%20heterosis%20in%20elite%20hybrid%20rice&journal=Plant%20Commun.&volume=4&publication_year=2023&author=Ma%2CX) 
    
66.  Zhang, J. et al. Extensive sequence divergence between the reference genomes of two elite indica rice varieties Zhenshan 97 and Minghui 63. _Proc. Natl Acad. Sci. USA_ **113**, E5163–E5171 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlGmt77M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27535938)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5024649)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extensive%20sequence%20divergence%20between%20the%20reference%20genomes%20of%20two%20elite%20indica%20rice%20varieties%20Zhenshan%2097%20and%20Minghui%2063&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=113&pages=E5163-E5171&publication_year=2016&author=Zhang%2CJ) 
    
67.  Shen, G. et al. The regulatory network mediated by circadian clock genes is related to heterosis in rice. _J. Integr. Plant Biol._ **57**, 300–312 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvVOnur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25040350)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20regulatory%20network%20mediated%20by%20circadian%20clock%20genes%20is%20related%20to%20heterosis%20in%20rice&journal=J.%20Integr.%20Plant%20Biol.&volume=57&pages=300-312&publication_year=2015&author=Shen%2CG) 
    
68.  Yang, M. et al. Genomic architecture of biomass heterosis in _Arabidopsis_. _Proc. Natl Acad. Sci. USA_ **114**, 8101–8106 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFCgsr3J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28696287)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5544317)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20architecture%20of%20biomass%20heterosis%20in%20Arabidopsis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=114&pages=8101-8106&publication_year=2017&author=Yang%2CM) 
    
69.  Yang, Q. et al. ABA-responsive ABRE-BINDING FACTOR3 activates DAM3 expression to promote bud dormancy in Asian pear. _Plant Cell Environ._ **43**, 1360–1375 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVWntbfM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32092154)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ABA-responsive%20ABRE-BINDING%20FACTOR3%20activates%20DAM3%20expression%20to%20promote%20bud%20dormancy%20in%20Asian%20pear&journal=Plant%20Cell%20Environ.&volume=43&pages=1360-1375&publication_year=2020&author=Yang%2CQ) 
    
70.  Fu, J. et al. Genome-wide identification of _Petunia_ HSF genes and potential function of PhHSF19 in benzenoid/phenylpropanoid biosynthesis. _Int. J. Mol. Sci._ **23**, 2974 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XoslWrurg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35328393)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8951162)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20identification%20of%20Petunia%20HSF%20genes%20and%20potential%20function%20of%20PhHSF19%20in%20benzenoid%2Fphenylpropanoid%20biosynthesis&journal=Int.%20J.%20Mol.%20Sci.&volume=23&publication_year=2022&author=Fu%2CJ) 
    
71.  Zhang, N. et al. Changes to the core and flanking sequences of G-box elements lead to increases and decreases in gene expression in both native and synthetic soybean promoters. _Plant Biotechnol. J._ **17**, 724–735 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXkvFWju78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30191675)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Changes%20to%20the%20core%20and%20flanking%20sequences%20of%20G-box%20elements%20lead%20to%20increases%20and%20decreases%20in%20gene%20expression%20in%20both%20native%20and%20synthetic%20soybean%20promoters&journal=Plant%20Biotechnol.%20J.&volume=17&pages=724-735&publication_year=2019&author=Zhang%2CN) 
    
72.  Zhang, M. Y. et al. Genome-wide association studies provide insights into the genetic determination of fruit traits of pear. _Nat. Commun._ **12**, 1141 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20studies%20provide%20insights%20into%20the%20genetic%20determination%20of%20fruit%20traits%20of%20pear&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Zhang%2CMY) 
    
73.  Zhang, X. T. et al. Assembly of allele-aware, chromosomal-scale autopolyploid genomes based on Hi-C data. _Nat. Plants_ **5**, 833–845 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFCmsr3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31383970)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Assembly%20of%20allele-aware%2C%20chromosomal-scale%20autopolyploid%20genomes%20based%20on%20Hi-C%20data&journal=Nat.%20Plants&volume=5&pages=833-845&publication_year=2019&author=Zhang%2CXT) 
    
74.  Li, H. Minimap2: pairwise alignment for nucleotide sequences. _Bioinformatics_ **34**, 3094–3100 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVamu73J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29750242)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6137996)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Minimap2%3A%20pairwise%20alignment%20for%20nucleotide%20sequences&journal=Bioinformatics&volume=34&pages=3094-3100&publication_year=2018&author=Li%2CH) 
    
75.  Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. _Bioinformatics_ **25**, 1754–1760 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19451168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&journal=Bioinformatics&volume=25&pages=1754-1760&publication_year=2009&author=Li%2CH&author=Durbin%2CR) 
    
76.  Li, H. et al. The sequence alignment/map format and SAMtools. _Bioinformatics_ **25**, 2078–2079 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19505943)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&journal=Bioinformatics&volume=25&pages=2078-2079&publication_year=2009&author=Li%2CH) 
    
77.  Ou, S. J. et al. Assessing genome assembly quality using the LTR Assembly Index (LAI). _Nucleic Acids Res._ **46**, e126 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30107434)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6265445)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Assessing%20genome%20assembly%20quality%20using%20the%20LTR%20Assembly%20Index%20%28LAI%29&journal=Nucleic%20Acids%20Res.&volume=46&publication_year=2018&author=Ou%2CSJ) 
    
78.  Manni, M. et al. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. _Mol. Biol. Evol._ **38**, 4647–4654 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhslaktbzF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34320186)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8476166)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BUSCO%20update%3A%20novel%20and%20streamlined%20workflows%20along%20with%20broader%20and%20deeper%20phylogenetic%20coverage%20for%20scoring%20of%20eukaryotic%2C%20prokaryotic%2C%20and%20viral%20genomes&journal=Mol.%20Biol.%20Evol.&volume=38&pages=4647-4654&publication_year=2021&author=Manni%2CM) 
    
79.  Rhie, A. et al. Merqury: reference-free quality, completeness, and phasing assessment for genome assemblies. _Genome Biol._ **21**, 245 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVKmu7rF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32928274)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7488777)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Merqury%3A%20reference-free%20quality%2C%20completeness%2C%20and%20phasing%20assessment%20for%20genome%20assemblies&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Rhie%2CA) 
    
80.  Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. _Genome Biol._ **16**, 259 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26619908)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4665391)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HiC-Pro%3A%20an%20optimized%20and%20flexible%20pipeline%20for%20Hi-C%20data%20processing&journal=Genome%20Biol.&volume=16&publication_year=2015&author=Servant%2CN) 
    
81.  Akdemir, K. C. & Chin, L. HiCPlotter integrates genomic data with interaction matrices. _Genome Biol._ **16**, 198 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26392354)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4576377)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HiCPlotter%20integrates%20genomic%20data%20with%20interaction%20matrices&journal=Genome%20Biol.&volume=16&publication_year=2015&author=Akdemir%2CKC&author=Chin%2CL) 
    
82.  Price, A. L. et al. De novo identification of repeat families in large genomes. _Bioinformatics_ **21**, I351–I358 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlslyrsrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15961478)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&journal=Bioinformatics&volume=21&pages=I351-I358&publication_year=2005&author=Price%2CAL) 
    
83.  Edgar, R. C. & Myers, E. W. PILER: identification and classification of genomic repeats. _Bioinformatics_ **21**, I152–I158 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlslyrsL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15961452)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PILER%3A%20identification%20and%20classification%20of%20genomic%20repeats&journal=Bioinformatics&volume=21&pages=I152-I158&publication_year=2005&author=Edgar%2CRC&author=Myers%2CEW) 
    
84.  Xu, Z. & Wang, H. LTR\_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons. _Nucleic Acids Res._ **35**, W265–W268 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17485477)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933203)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTR_FINDER%3A%20an%20efficient%20tool%20for%20the%20prediction%20of%20full-length%20LTR%20retrotransposons&journal=Nucleic%20Acids%20Res.&volume=35&pages=W265-W268&publication_year=2007&author=Xu%2CZ&author=Wang%2CH) 
    
85.  Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. _Proc. Natl Acad. Sci. USA_ **117**, 9451–9457 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnvFeqt74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32300014)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196820)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=9451-9457&publication_year=2020&author=Flynn%2CJM) 
    
86.  Tarailo-Graovac, M. & Chen, N. Using RepeatMasker to identify repetitive elements in genomic sequences. _Curr Protoc Bioinformatics_ **25**, 4.10.1–4.10.14 (2009).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Using%20RepeatMasker%20to%20identify%20repetitive%20elements%20in%20genomic%20sequences&journal=Curr%20Protoc%20Bioinformatics&volume=25&pages=4.10.1-4.10.14&publication_year=2009&author=Tarailo-Graovac%2CM&author=Chen%2CN) 
    
87.  Benson, G. Tandem repeats finder: a program to analyze DNA sequences. _Nucleic Acids Res._ **27**, 573–580 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXhtVKmtrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9862982)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC148217)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&journal=Nucleic%20Acids%20Res.&volume=27&pages=573-580&publication_year=1999&author=Benson%2CG) 
    
88.  Altschul, S. F. et al. Basic local alignment search tool. _J. Mol. Biol._ **215**, 403–410 (1990).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3MXitVGmsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2231712)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Basic%20local%20alignment%20search%20tool&journal=J.%20Mol.%20Biol.&volume=215&pages=403-410&publication_year=1990&author=Altschul%2CSF) 
    
89.  Birney, E. et al. GeneWise and genomewise. _Genome Res._ **14**, 988–995 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjvFyku7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15123596)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC479130)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GeneWise%20and%20genomewise&journal=Genome%20Res.&volume=14&pages=988-995&publication_year=2004&author=Birney%2CE) 
    
90.  Urnov, F. D. et al. Highly efficient endogenous human gene correction using designed zinc-finger nucleases. _Nature_ **435**, 646–651 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXks1OgtLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15806097)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Highly%20efficient%20endogenous%20human%20gene%20correction%20using%20designed%20zinc-finger%20nucleases&journal=Nature&volume=435&pages=646-651&publication_year=2005&author=Urnov%2CFD) 
    
91.  Majoros, W. H. et al. TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders. _Bioinformatics_ **20**, 2878–2879 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVSru77E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15145805)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TigrScan%20and%20GlimmerHMM%3A%20two%20open%20source%20ab%20initio%20eukaryotic%20gene-finders&journal=Bioinformatics&volume=20&pages=2878-2879&publication_year=2004&author=Majoros%2CWH) 
    
92.  Burge, C. & Karlin, S. Prediction of complete gene structures in human genomic DNA. _J. Mol. Biol._ **268**, 78–94 (1997).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXjtlSqtL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9149143)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prediction%20of%20complete%20gene%20structures%20in%20human%20genomic%20DNA&journal=J.%20Mol.%20Biol.&volume=268&pages=78-94&publication_year=1997&author=Burge%2CC&author=Karlin%2CS) 
    
93.  Kim, D. et al. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. _Nat. Biotechnol._ **37**, 907–915 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFWqtL7O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31375807)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7605509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype&journal=Nat.%20Biotechnol.&volume=37&pages=907-915&publication_year=2019&author=Kim%2CD) 
    
94.  Trapnell, C. et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. _Nat. Biotechnol._ **28**, 511–515 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlsVyitLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20436464)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3146043)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&journal=Nat.%20Biotechnol.&volume=28&pages=511-515&publication_year=2010&author=Trapnell%2CC) 
    
95.  Haas, B. J. et al. De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. _Nat. Protoc._ **8**, 1494–1512 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVyrsLvF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23845962)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis&journal=Nat.%20Protoc.&volume=8&pages=1494-1512&publication_year=2013&author=Haas%2CBJ) 
    
96.  Haas, B. J. et al. Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments. _Genome Biol._ **9**, R7 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18190707)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2395244)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20program%20to%20assemble%20spliced%20alignments&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Haas%2CBJ) 
    
97.  Cantalapiedra, C. P. et al. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. _Mol. Biol. Evol._ **38**, 5825–5829 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhslyrt78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34597405)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8662613)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG-mapper%20v2%3A%20functional%20annotation%2C%20orthology%20assignments%2C%20and%20domain%20prediction%20at%20the%20metagenomic%20scale&journal=Mol.%20Biol.%20Evol.&volume=38&pages=5825-5829&publication_year=2021&author=Cantalapiedra%2CCP) 
    
98.  Tang, H. et al. JCVI: A versatile toolkit for comparative genomics analysis. _Imeta_ **12**, e211 (2024).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=JCVI%3A%20A%20versatile%20toolkit%20for%20comparative%20genomics%20analysis&journal=Imeta&volume=12&publication_year=2024&author=Tang%2CH) 
    
99.  Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. _Mol. Biol. Evol._ **30**, 772–780 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXksFWisLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23329690)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603318)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&journal=Mol.%20Biol.%20Evol.&volume=30&pages=772-780&publication_year=2013&author=Katoh%2CK&author=Standley%2CDM) 
    
100.  Zhang, Z. et al. ParaAT: a parallel tool for constructing multiple protein-coding DNA alignments. _Biochem. Biophys. Res. Commun._ **419**, 779–781 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjs1yjtL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22390928)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ParaAT%3A%20a%20parallel%20tool%20for%20constructing%20multiple%20protein-coding%20DNA%20alignments&journal=Biochem.%20Biophys.%20Res.%20Commun.&volume=419&pages=779-781&publication_year=2012&author=Zhang%2CZ) 
    
101.  Yang, Z. PAML 4: phylogenetic analysis by maximum likelihood. _Mol. Biol. Evol._ **24**, 1586–1591 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXpsVGrs7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17483113)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&journal=Mol.%20Biol.%20Evol.&volume=24&pages=1586-1591&publication_year=2007&author=Yang%2CZ) 
    
102.  Li, L. et al. OrthoMCL: identification of ortholog groups for eukaryotic genomes. _Genome Res._ **13**, 2178–2189 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXnslKhtLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12952885)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC403725)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&journal=Genome%20Res.&volume=13&pages=2178-2189&publication_year=2003&author=Li%2CL) 
    
103.  Edgar, R. MUSCLE: multiple sequence alignment with high accuracy and high throughput. _Nucleic Acids Res_ **32**, 1792–1797 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXisF2ks7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15034147)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&journal=Nucleic%20Acids%20Res&volume=32&pages=1792-1797&publication_year=2004&author=Edgar%2CR) 
    
104.  Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. _Bioinformatics_ **30**, 1312–1313 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmvFCjsbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24451623)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998144)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&journal=Bioinformatics&volume=30&pages=1312-1313&publication_year=2014&author=Stamatakis%2CA) 
    
105.  De Bie, T. et al. CAFE: a computational tool for the study of gene family evolution. _Bioinformatics_ **22**, 1269–1271 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16543274)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CAFE%3A%20a%20computational%20tool%20for%20the%20study%20of%20gene%20family%20evolution&journal=Bioinformatics&volume=22&pages=1269-1271&publication_year=2006&author=Bie%2CT) 
    
106.  Marcais, G. et al. MUMmer4: a fast and versatile genome alignment system. _PLoS Comput. Biol._ **14**, e1005944 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29373581)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5802927)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MUMmer4%3A%20a%20fast%20and%20versatile%20genome%20alignment%20system&journal=PLoS%20Comput.%20Biol.&volume=14&publication_year=2018&author=Marcais%2CG) 
    
107.  Goel, M. et al. SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies. _Genome Biol._ **20**, 277 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31842948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6913012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SyRI%3A%20finding%20genomic%20rearrangements%20and%20local%20sequence%20differences%20from%20whole-genome%20assemblies&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Goel%2CM) 
    
108.  Li, H. et al. The design and construction of reference pangenome graphs with minigraph. _Genome Biol._ **21**, 265 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33066802)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7568353)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20design%20and%20construction%20of%20reference%20pangenome%20graphs%20with%20minigraph&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Li%2CH) 
    
109.  Thorvaldsdottir, H. et al. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. _Brief. Bioinform._ **14**, 178–192 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXkvFKrur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22517427)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20Genomics%20Viewer%20%28IGV%29%3A%20high-performance%20genomics%20data%20visualization%20and%20exploration&journal=Brief.%20Bioinform.&volume=14&pages=178-192&publication_year=2013&author=Thorvaldsdottir%2CH) 
    
110.  Zhang, J. et al. Allele-defined genome of the autopolyploid sugarcane _Saccharum spontaneum_ L. _Nat. Genet._ **50**, 1565–1573 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVCitL3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30297971)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Allele-defined%20genome%20of%20the%20autopolyploid%20sugarcane%20Saccharum%20spontaneum%20L&journal=Nat.%20Genet.&volume=50&pages=1565-1573&publication_year=2018&author=Zhang%2CJ) 
    
111.  Wu, T. D. & Watanabe, C. K. GMAP: a genomic mapping and alignment program for mRNA and EST sequences. _Bioinformatics_ **21**, 1859–1875 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXjsl2ntLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15728110)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&journal=Bioinformatics&volume=21&pages=1859-1875&publication_year=2005&author=Wu%2CTD&author=Watanabe%2CCK) 
    
112.  Bolger, A. M. et al. Trimmomatic: a flexible trimmer for Illumina sequence data. _Bioinformatics_ **30**, 2114–2120 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24695404)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&journal=Bioinformatics&volume=30&pages=2114-2120&publication_year=2014&author=Bolger%2CAM) 
    
113.  Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. _Bioinformatics_ **29**, 15–21 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23104886)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&journal=Bioinformatics&volume=29&pages=15-21&publication_year=2013&author=Dobin%2CA) 
    
114.  Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. _Nat. Biotechnol._ **33**, 290–295 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXivFais70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25690850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643835)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&journal=Nat.%20Biotechnol.&volume=33&pages=290-295&publication_year=2015&author=Pertea%2CM) 
    
115.  Love, M. I. et al. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._ **15**, 550 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25516281)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Love%2CMI) 
    
116.  Lan, X. & Pritchard, J. K. Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals. _Science_ **352**, 1009–1013 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XotVSktLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27199432)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5182070)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coregulation%20of%20tandem%20duplicate%20genes%20slows%20evolution%20of%20subfunctionalization%20in%20mammals&journal=Science&volume=352&pages=1009-1013&publication_year=2016&author=Lan%2CX&author=Pritchard%2CJK) 
    
117.  Song, B. et al. Increased DNA methylation contributes to the early ripening of pear fruits during domestication and improvement. _Genome Biol._ **25**, 87 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXotVWntbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38581061)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10996114)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Increased%20DNA%20methylation%20contributes%20to%20the%20early%20ripening%20of%20pear%20fruits%20during%20domestication%20and%20improvement&journal=Genome%20Biol.&volume=25&publication_year=2024&author=Song%2CB) 
    
118.  Krueger, F. & Andrews, S. R. Bismark: a flexible aligner and methylation caller for bisulfite-Seq applications. _Bioinformatics_ **27**, 1571–1572 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvVWqurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21493656)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102221)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bismark%3A%20a%20flexible%20aligner%20and%20methylation%20caller%20for%20bisulfite-Seq%20applications&journal=Bioinformatics&volume=27&pages=1571-1572&publication_year=2011&author=Krueger%2CF&author=Andrews%2CSR) 
    
119.  Ramirez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. _Nucleic Acids Res._ **44**, W160–W165 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27079975)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987876)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=deepTools2%3A%20a%20next%20generation%20web%20server%20for%20deep-sequencing%20data%20analysis&journal=Nucleic%20Acids%20Res.&volume=44&pages=W160-W165&publication_year=2016&author=Ramirez%2CF) 
    
120.  Hickey, G. et al. Genotyping structural variants in pangenome graphs using the vg toolkit. _Genome Biol._ **21**, 35 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32051000)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7017486)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genotyping%20structural%20variants%20in%20pangenome%20graphs%20using%20the%20vg%20toolkit&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Hickey%2CG) 
    
121.  Danecek, P. et al. The variant call format and VCFtools. _Bioinformatics_ **27**, 2156–2158 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXptFCqt7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21653522)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137218)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20variant%20call%20format%20and%20VCFtools&journal=Bioinformatics&volume=27&pages=2156-2158&publication_year=2011&author=Danecek%2CP) 
    
122.  Wang, J. et al. A pangenome analysis pipeline provides insights into functional gene identification in rice. _Genome Biol._ **24**, 19 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36703158)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9878884)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20pangenome%20analysis%20pipeline%20provides%20insights%20into%20functional%20gene%20identification%20in%20rice&journal=Genome%20Biol.&volume=24&publication_year=2023&author=Wang%2CJ) 
    
123.  McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. _Genome Res._ **20**, 1297–1303 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFeru7jM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20644199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&journal=Genome%20Res.&volume=20&pages=1297-1303&publication_year=2010&author=McKenna%2CA) 
    
124.  Minh, B. Q. et al. IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. _Mol. Biol. Evol._ **37**, 1530–1534 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1egsLbL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32011700)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7182206)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IQ-TREE%202%3A%20new%20models%20and%20efficient%20methods%20for%20phylogenetic%20inference%20in%20the%20genomic%20era&journal=Mol.%20Biol.%20Evol.&volume=37&pages=1530-1534&publication_year=2020&author=Minh%2CBQ) 
    
125.  Alexander, D. H. et al. Fast model-based estimation of ancestry in unrelated individuals. _Genome Res._ **19**, 1655–1664 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFCjsLvL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19648217)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752134)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&journal=Genome%20Res.&volume=19&pages=1655-1664&publication_year=2009&author=Alexander%2CDH) 
    
126.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet_. **81**, 559–575 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
127.  Korneliussen, T. S. et al. ANGSD: analysis of next generation sequencing data. _BMC Bioinf._ **15**, 356 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ANGSD%3A%20analysis%20of%20next%20generation%20sequencing%20data&journal=BMC%20Bioinf.&volume=15&publication_year=2014&author=Korneliussen%2CTS) 
    
128.  Liu, X. & Fu, Y. X. Exploring population size changes using SNP frequency spectra. _Nat. Genet._ **47**, 555–559 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlvFajurY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25848749)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4414822)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Exploring%20population%20size%20changes%20using%20SNP%20frequency%20spectra&journal=Nat.%20Genet.&volume=47&pages=555-559&publication_year=2015&author=Liu%2CX&author=Fu%2CYX) 
    
129.  Liu, X. & Fu, Y. X. Stairway Plot 2: demographic history inference with folded SNP frequency spectra. _Genome Biol._ **21**, 280 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33203475)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7670622)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stairway%20Plot%202%3A%20demographic%20history%20inference%20with%20folded%20SNP%20frequency%20spectra&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Liu%2CX&author=Fu%2CYX) 
    
130.  Wang, J. & Zhang, Z. GAPIT version 3: boosting power and accuracy for genomic association and prediction. _Genom. Proteom. Bioinform._ **19**, 629–640 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GAPIT%20version%203%3A%20boosting%20power%20and%20accuracy%20for%20genomic%20association%20and%20prediction&journal=Genom.%20Proteom.%20Bioinform.&volume=19&pages=629-640&publication_year=2021&author=Wang%2CJ&author=Zhang%2CZ) 
    
131.  Zhang, M. Y. et al. Distinct transcriptome profiles reveal gene expression patterns during fruit development and maturation in five main cultivated species of pear (Pyrus L.). _Sci. Rep._ **6**, 28130 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVahsL%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27305967)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4910100)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20transcriptome%20profiles%20reveal%20gene%20expression%20patterns%20during%20fruit%20development%20and%20maturation%20in%20five%20main%20cultivated%20species%20of%20pear%20%28Pyrus%20L.%29&journal=Sci.%20Rep.&volume=6&publication_year=2016&author=Zhang%2CMY) 
    
132.  Wang, R. Z. et al. A systems genetics approach reveals PbrNSC as a regulator of lignin and cellulose biosynthesis in stone cells of pear fruit. _Genome Biol._ **22**, 313 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XksFaqur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34776004)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8590786)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20systems%20genetics%20approach%20reveals%20PbrNSC%20as%20a%20regulator%20of%20lignin%20and%20cellulose%20biosynthesis%20in%20stone%20cells%20of%20pear%20fruit&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Wang%2CRZ) 
    
133.  Lin, T. B. Resveratrol production of tomato transformed with a vitis stilbene synthase gene. _Life Sci. Res_ **7**, 262–266 (2003).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resveratrol%20production%20of%20tomato%20transformed%20with%20a%20vitis%20stilbene%20synthase%20gene&journal=Life%20Sci.%20Res&volume=7&pages=262-266&publication_year=2003&author=Lin%2CTB) 
    
134.  Sun, M. Bioinformatic1996512/DS-MRB-genome: code for DS and MRB genome (DSMRBv1.0). _Zenodo_ [https://doi.org/10.5281/zenodo.15113931](https://doi.org/10.5281/zenodo.15113931) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02273-4?format=refman&flavour=references)

Acknowledgements
----------------

This study was supported by the National Science Foundation of China (grant nos. 32230097 to J.W., 32441068 to J.L. and 32172531 to C.X.); the Earmarked Fund for China Agriculture Research System (grant no. CARS-28), the Earmarked Fund for Jiangsu Agricultural Industry Technology System (grant no. JATS\[2023\]412) and the Project of Zhongshan Biological Breeding Laboratory (grant no. ZSBBL-KY2023-08) (all to J.W.); and the Natural Science Foundation of Jiangsu Province (grant no. BK20242065 to J.L.). We thank the Bioinformatics Center of Nanjing Agricultural University for use of the high-performance computing platforms in support of this project.

Author information
------------------

Author notes

1.  These authors contributed equally: Manyi Sun, Beibei Cao, Kui Li, Jiaming Li, Jun Liu, Cheng Xue.
    

### Authors and Affiliations

1.  College of Horticulture, State Key Laboratory of Crop Genetics & Germplasm Enhancement and Utilization, Nanjing Agricultural University, Nanjing, China
    
    Manyi Sun, Beibei Cao, Jiaming Li, Shaozhuo Xu, Runze Wang, Yueyuan Liu, Chenjie Yao & Jun Wu
    
2.  Zhongshan Biological Breeding Laboratory, Nanjing, China
    
    Manyi Sun, Beibei Cao, Jiaming Li, Shaozhuo Xu, Yueyuan Liu, Chenjie Yao & Jun Wu
    
3.  National Key Laboratory of Wheat Improvement, Peking University Institute of Advanced Agricultural Sciences, Shandong Laboratory of Advanced Agricultural Sciences in Weifang, Weifang, China
    
    Kui Li & Hang He
    
4.  Institute of Forestry and Pomology, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China
    
    Jun Liu
    
5.  State Key Laboratory of Crop Biology, College of Horticulture Science and Engineering, Shandong Agricultural University, Tai’an, China
    
    Cheng Xue, Kaidi Gu & Mingyue Zhang
    
6.  Yantai Academy of Agricultural Sciences, Yantai, China
    
    Yuanjun Li, Qingyu Li & Meina Qu
    
7.  School of Horticulture, Anhui Agricultural University, Hefei, China
    
    Runze Wang
    

Authors

1.  Manyi Sun
    
    [View author publications](https://www.nature.com/search?author=Manyi%20Sun)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Manyi%20Sun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Manyi%20Sun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Beibei Cao
    
    [View author publications](https://www.nature.com/search?author=Beibei%20Cao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Beibei%20Cao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Beibei%20Cao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Kui Li
    
    [View author publications](https://www.nature.com/search?author=Kui%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kui%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kui%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Jiaming Li
    
    [View author publications](https://www.nature.com/search?author=Jiaming%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jiaming%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jiaming%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Jun Liu
    
    [View author publications](https://www.nature.com/search?author=Jun%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jun%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jun%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Cheng Xue
    
    [View author publications](https://www.nature.com/search?author=Cheng%20Xue)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cheng%20Xue) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cheng%20Xue%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Kaidi Gu
    
    [View author publications](https://www.nature.com/search?author=Kaidi%20Gu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kaidi%20Gu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kaidi%20Gu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Shaozhuo Xu
    
    [View author publications](https://www.nature.com/search?author=Shaozhuo%20Xu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shaozhuo%20Xu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shaozhuo%20Xu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Yuanjun Li
    
    [View author publications](https://www.nature.com/search?author=Yuanjun%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuanjun%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuanjun%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Qingyu Li
    
    [View author publications](https://www.nature.com/search?author=Qingyu%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qingyu%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qingyu%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Meina Qu
    
    [View author publications](https://www.nature.com/search?author=Meina%20Qu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Meina%20Qu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Meina%20Qu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Mingyue Zhang
    
    [View author publications](https://www.nature.com/search?author=Mingyue%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mingyue%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mingyue%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Runze Wang
    
    [View author publications](https://www.nature.com/search?author=Runze%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Runze%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Runze%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Yueyuan Liu
    
    [View author publications](https://www.nature.com/search?author=Yueyuan%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yueyuan%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yueyuan%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Chenjie Yao
    
    [View author publications](https://www.nature.com/search?author=Chenjie%20Yao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chenjie%20Yao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chenjie%20Yao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Hang He
    
    [View author publications](https://www.nature.com/search?author=Hang%20He)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hang%20He) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hang%20He%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Jun Wu
    
    [View author publications](https://www.nature.com/search?author=Jun%20Wu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jun%20Wu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jun%20Wu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

J.W. designed this project and coordinated research activities. M.S. and K.L. assembled and evaluated the genomes. K.L. performed gene annotation. M.S. analyzed allelic imbalance genome comparison and population resequencing data and contributed to graph-based genome construction and application. J. Liu, J. Li, C.X., Q.L. and M.Q. collected and provided plant materials. Y. Liu and C.Y. performed the RNA-seq analysis. B.C., J. Li and K.G. contributed to PCR transgenic experiments. C.X. and S.X. performed the EMSA and transgenic experiments. M.S., J.W., K.L., H.H., J. Li, C.X., Y. Li, M.Z. and R.W. interpreted data and contributed to the writing of the manuscript.

### Corresponding author

Correspondence to [Jun Wu](mailto:wujun@njau.edu.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Xingtan Zhang, Zhonghua Zhang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_MOESM1_ESM.pdf)

Supplementary Notes 1–11, Methods 1–9 and Figs. 1–40.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_MOESM2_ESM.pdf)

### [Supplementary Table](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_MOESM3_ESM.xlsx)

Supplementary Tables 1–21.

Source data
-----------

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02273-4/MediaObjects/41588_2025_2273_MOESM4_ESM.pdf)

Unprocessed gels for source data for Fig. 6i,m.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Haplotype-resolved%2C%20gap-free%20genome%20assemblies%20provide%20insights%20into%20the%20divergence%20between%20Asian%20and%20European%20pears&author=Manyi%20Sun%20et%20al&contentID=10.1038%2Fs41588-025-02273-4&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-06&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY-NC-ND)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02273-4)

### Cite this article

Sun, M., Cao, B., Li, K. _et al._ Haplotype-resolved, gap-free genome assemblies provide insights into the divergence between Asian and European pears. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02273-4

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02273-4?format=refman&flavour=citation)

-   Received: 20 June 2023
    
-   Accepted: 19 June 2025
    
-   Published: 06 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02273-4
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02273-4.pdf)
