> id: https://www.nature.com/articles/s41588-025-02248-5

> link: https://www.nature.com/articles/s41588-025-02248-5

> title: Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes

# Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes
_Published on Wed, 09 Jul 2025 00:00:00 GMT_

Abstract
--------

Transposable elements (TEs) are scattered across mammalian genomes. Silencing of TEs prevents harmful effects caused by either global activation leading to genome instability or insertional mutations disturbing gene transcription. However, whether the activation of a TE can cause disease without directly affecting gene expression is largely unknown. Here we show that a TE insertion can adopt nearby regulatory activity, resulting in the production of cell-type-specific viral-like particles (VLPs) that affect embryo formation. Failure to silence an LTR retrotransposon inserted upstream of the _Fgf8_ gene results in their co-expression during mouse development. VLP assembly in the _Fgf8_\-expressing cells of the developing limb triggers apoptotic cell death, resulting in a limb malformation resembling human ectrodactyly. The phenotype can be rescued by mutating the retrotransposon coding sequence, thus preventing its full endogenous retroviral cycle. Our findings illustrate that TE insertions can be incorporated into the local genomic regulatory landscape and that VLP production in post-implantation embryos can cause developmental defects.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41594-023-00960-6/MediaObjects/41594_2023_960_Fig1_HTML.png)

### [Regulation of human trophoblast gene expression by endogenous retroviruses](https://www.nature.com/articles/s41594-023-00960-6?fromPaywallRec=false)

Article Open access 03 April 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig1_HTML.png)

### [A human embryonic limb cell atlas resolved in space and time](https://www.nature.com/articles/s41586-023-06806-x?fromPaywallRec=false)

Article Open access 06 December 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-024-68608-z/MediaObjects/41598_2024_68608_Fig1_HTML.png)

### [Splicing is dynamically regulated during limb development](https://www.nature.com/articles/s41598-024-68608-z?fromPaywallRec=false)

Article Open access 27 August 2024

Main
----

About half of the mammalian genome consists of TEs[1](https://www.nature.com/articles/s41588-025-02248-5#ref-CR1 "Wells, J. N. & Feschotte, C. A field guide to eukaryotic transposable elements. Annu. Rev. Genet. 54, 539–561 (2020)."). These genetic elements can mobilize in the genome, causing insertional mutations and genomic instability[2](https://www.nature.com/articles/s41588-025-02248-5#ref-CR2 "Payer, L. M. & Burns, K. H. Transposable elements in human genetic disease. Nat. Rev. Genet. 20, 760–772 (2019)."). To prevent such harmful effects, TEs are transcriptionally silenced by DNA methylation and other epigenetic marks in most somatic cells[3](https://www.nature.com/articles/s41588-025-02248-5#ref-CR3 "Deniz, Ö., Frost, J. M. & Branco, M. R. Regulation of transposable elements by DNA modifications. Nat. Rev. Genet. 20, 417–431 (2019)."). TE expression can be triggered by remodeling of the epigenome occurring in physiological and pathological contexts. During mammalian pre-implantation development, activation of several TE families was shown to participate in establishing pluripotency and embryonic genome activation required for development[4](https://www.nature.com/articles/s41588-025-02248-5#ref-CR4 "Fadloun, A. et al. Chromatin signatures and retrotransposon profiling in mouse embryos reveal regulation of LINE-1 by RNA. Nat. Struct. Mol. Biol. 20, 332–338 (2013)."),[5](https://www.nature.com/articles/s41588-025-02248-5#ref-CR5 "Jachowicz, J. W. et al. LINE-1 activation after fertilization regulates global chromatin accessibility in the early mouse embryo. Nat. Genet. 49, 1502–1510 (2017)."),[6](https://www.nature.com/articles/s41588-025-02248-5#ref-CR6 "Sakashita, A. et al. Transcription of MERVL retrotransposons is required for preimplantation embryo development. Nat. Genet. 55, 484–495 (2023)."). In cancer, neurological disorders or aging, aberrant TE activation may have a role in disease progression[7](https://www.nature.com/articles/s41588-025-02248-5#ref-CR7 "Tam, O. H., Ostrow, L. W. & Gale Hammell, M. Diseases of the nERVous system: retrotransposon activity in neurodegenerative disease. Mob. DNA 10, 32 (2019)."),[8](https://www.nature.com/articles/s41588-025-02248-5#ref-CR8 "Gorbunova, V. et al. The role of retrotransposable elements in ageing and age-associated diseases. Nature 596, 43–53 (2021)."),[9](https://www.nature.com/articles/s41588-025-02248-5#ref-CR9 "Liu, X. et al. Resurrection of endogenous retroviruses during aging reinforces senescence. Cell 186, 287–304.e26 (2023)."),[10](https://www.nature.com/articles/s41588-025-02248-5#ref-CR10 "Shah, N. M. et al. Pan-cancer analysis identifies tumor-specific antigens derived from transposable elements. Nat. Genet. 55, 631–639 (2023)."). Yet very little is known about how TE activation affects post-implantation development. Although this critical period of cellular differentiation and organ formation is not subject to global epigenetic changes, failure to establish epigenetic marks at specific TE sequences could occur.

Precise transcription during organogenesis is influenced by promoter–enhancer communication and the three-dimensional (3D) organization of the genome[11](https://www.nature.com/articles/s41588-025-02248-5#ref-CR11 "Andrey, G. & Mundlos, S. The three-dimensional genome: regulating gene expression during pluripotency and development. Development 144, 3646–3658 (2017)."). TEs can contain transcription factor binding sites, act as alternative promoters or enhancers and modify 3D chromatin architecture[12](https://www.nature.com/articles/s41588-025-02248-5#ref-CR12 "Fueyo, R., Judd, J., Feschotte, C. & Wysocka, J. Roles of transposable elements in the regulation of mammalian transcription. Nat. Rev. Mol. Cell Biol. 23, 481–497 (2022)."). TE insertion in a gene can also affect normal transcription processes (that is, splicing, polyadenylation and disruption of an exon or a regulatory element)[13](https://www.nature.com/articles/s41588-025-02248-5#ref-CR13 "Gagnier, L., Belancio, V. P. & Mager, D. L. Mouse germ line mutations due to retrotransposon insertions. Mob. DNA 10, 15 (2019)."). However, how regulatory landscapes influence TE transcription is unknown. Here, we show that an unmethylated LTR retrotransposon insertion does not impair transcriptional activation but responds to the regulatory elements surrounding its insertion, according to the organization of the genome into topologically associating domains (TADs). Based on this result, we postulated that enhancer adoption by LTR elements could be a more common phenomenon, and we validated this concept at three developmental loci.

We focused on an endogenous retrovirus (ERV) element from the MusD (type D-related mouse provirus-like) family, probably originating from an infectious simian type D retrovirus[14](https://www.nature.com/articles/s41588-025-02248-5#ref-CR14 "Mager, D. L. & Freeman, J. D. Novel mouse type D endogenous proviruses and ETn elements share long terminal repeat and internal sequences. J. Virol. 74, 7221–7229 (2000)."). MusD are evolutionarily young elements with about 100 full-length copies in the mouse genome and are polymorphic between mouse strains[15](https://www.nature.com/articles/s41588-025-02248-5#ref-CR15 "Ribet, D., Dewannieux, M. & Heidmann, T. An active murine transposon family pair: retrotransposition of ‘master’ MusD copies and ETn trans-mobilization. Genome Res. 14, 2261–2267 (2004)."),[16](https://www.nature.com/articles/s41588-025-02248-5#ref-CR16 "Zhang, Y., Maksakova, I.A., Gagnier, L., van de Lagemaat, L. N. & Mager, D. L. Genome-wide assessments reveal extremely high levels of polymorphism of two active families of mouse endogenous retroviral elements. PLoS Genet. 4, e1000007 (2008)."),[17](https://www.nature.com/articles/s41588-025-02248-5#ref-CR17 "Nellåker, C. et al. The genomic landscape shaped by selection on transposable elements across 18 mouse strains. Genome Biol. 13, R45 (2012)."). As with many TEs, their expression is high in the inner cell mass of stage E3.5 mouse embryos, but MusD elements are also active in post-implantation embryos from E7.5 to E13.5 in specific tissues[18](https://www.nature.com/articles/s41588-025-02248-5#ref-CR18 "Brûlet, P., Condamine, H. & Jacob, F. Spatial distribution of transcripts of the long repeated ETn sequence during early mouse embryogenesis. Proc. Natl Acad. Sci. USA 82, 2054–2058 (1985)."),[19](https://www.nature.com/articles/s41588-025-02248-5#ref-CR19 "Loebel, D. A. F., Tsoi, B., Wong, N., O’Rourke, M. P. & Tam, P. P. L. Restricted expression of ETn-related sequences during post-implantation mouse development. Gene Expr. Patterns 4, 467–471 (2004)."), making them a good model to study the role of TEs during organogenesis. Using the example of the mouse dactylaplasia limb malformation, we show that enhancer adoption leads to the expression of a MusD element in the apical ectodermal ridge (AER) of the developing limb. This element assembles into VLPs, which cause the limb phenotype by triggering premature cell death of AER cells.

Results
-------

### Unmethylated MusD inserted at the _Fgf8_ locus causes dactylaplasia

Dactylaplasia is a limb malformation caused by a spontaneous allele, named _Dac1J_, on mouse chromosome 19 (refs. [20](https://www.nature.com/articles/s41588-025-02248-5#ref-CR20 "Chai, C. K. Dactylaplasia in mice. J. Hered. 72, 234–237 (1981)."),[21](https://www.nature.com/articles/s41588-025-02248-5#ref-CR21 "Johnson, K. R., Lane, P. W., Ward-Bailey, P. & Davisson, M. T. Mapping the mouse dactylaplasia mutation, Dac, and a gene that controls its expression, mdac. Genomics 29, 457–464 (1995).")). The _Dac1J_ allele was identified as a MusD LTR retrotransposon, inserted into the intergenic region between the _Fbwx4_ and _Fgf8_ genes[22](https://www.nature.com/articles/s41588-025-02248-5#ref-CR22 "Kano, H., Kurahashi, H. & Toda, T. Genetically regulated epigenetic transcriptional activation of retrotransposon insertion confers mouse dactylaplasia phenotype. Proc. Natl Acad. Sci. USA 104, 19034–19039 (2007).") (Fig. [1a,b](https://www.nature.com/articles/s41588-025-02248-5#Fig1)). The 7.4 kb-long MusD-_Dac1J_ sequence contains a retroviral sequence with _gag_, _pro_ and _pol_ genes flanked by identical 5′ and 3′ LTR[22](https://www.nature.com/articles/s41588-025-02248-5#ref-CR22 "Kano, H., Kurahashi, H. & Toda, T. Genetically regulated epigenetic transcriptional activation of retrotransposon insertion confers mouse dactylaplasia phenotype. Proc. Natl Acad. Sci. USA 104, 19034–19039 (2007).") but, like other MusD elements, lacks an _env_ gene[14](https://www.nature.com/articles/s41588-025-02248-5#ref-CR14 "Mager, D. L. & Freeman, J. D. Novel mouse type D endogenous proviruses and ETn elements share long terminal repeat and internal sequences. J. Virol. 74, 7221–7229 (2000).") (Fig. [1a](https://www.nature.com/articles/s41588-025-02248-5#Fig1)). Dactylaplasia mutants show a severe ectrodactyly-type limb phenotype characterized by missing central digits in both forepaws and hindpaws and abnormal nail-like structures (Extended Data Fig. [1a,b](https://www.nature.com/articles/s41588-025-02248-5#Fig7)). The phenotype is fully penetrant in homozygotes (Extended Data Fig. [1c](https://www.nature.com/articles/s41588-025-02248-5#Fig7)), whereas heterozygote mutants show a variable phenotype with 76.2% (32 out of 42) and 85.7% (36 out of 42) affected forelimbs and hindlimbs, respectively (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02248-5#Fig7)). Moreover, the phenotype is dependent on the presence of an epigenetic modifier (_Mdac_) that is polymorphic between mouse strains and has been refined to a region on mouse chromosome 13 (ref. [22](https://www.nature.com/articles/s41588-025-02248-5#ref-CR22 "Kano, H., Kurahashi, H. & Toda, T. Genetically regulated epigenetic transcriptional activation of retrotransposon insertion confers mouse dactylaplasia phenotype. Proc. Natl Acad. Sci. USA 104, 19034–19039 (2007).")). Accordingly, the MusD-_Dac1J_ insertion leads to dactylaplasia in 129sv mice but not in C57BL/6 mice (Fig. [1b,c](https://www.nature.com/articles/s41588-025-02248-5#Fig1)), correlating with differential CpG DNA methylation on the 5′LTR promoter (Fig. [1d](https://www.nature.com/articles/s41588-025-02248-5#Fig1)). The _Dac1J_\-5′LTR methylation is constitutive among different germ layers of the embryo and in extra-embryonic tissues (Extended Data Fig. [1e](https://www.nature.com/articles/s41588-025-02248-5#Fig7)) but does not affect methylation upstream and downstream the insertion (Fig. [1d](https://www.nature.com/articles/s41588-025-02248-5#Fig1)). For simplicity, ‘_Dac1J_’ and ‘_Dac1J_\-BL6’ will refer to mutant animals with the _Dac1J_ insertion in the 129sv (exhibiting a phenotype) and the C57BL/6 (no phenotype) backgrounds, respectively.

**Fig. 1: Dactylaplasia limb malformation is caused by an unmethylated MusD insertion at the _Fgf8_ locus.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02248-5/figures/1)

**a**, Scheme of the 7,486 bp MusD-_Dac1J_ inserted at the _Fgf8_ locus (mm10, chr19:45,100,000–45,900,000). The full-length element contains retroviral (_gag_, _pro_, _pol_) and non-retroviral sequences flanked by 5′LTR and 3′LTR. RepeatMasker names are indicated in green. **b**, The mouse _Fbxw4–Fgf8_ locus is shown in wild-type mice (top) or with the intergenic MusD-_Dac1J_ insertion in _Dac1J_/_Dac1J_ C57BL/6 (middle) or _Dac1J_/_Dac1J_ 129s2/Sv (bottom). The number and localization of the CpG measured in **d** are indicated in red. **c**, Skeletal analysis of the forelimbs and hindlimbs of E18.5 wild-type (top), _Dac1J_/_Dac1J_ C57BL/6 (middle) or 129s2/Sv (bottom) mice, stained with alcian blue (cartilage) and alizarin red (bone). Scale bars, 1 mm. **d**, DNA methylation status of five CpGs upstream of the insertion, 19 CpGs in the 5′LTR (promoter) of _Dac1J_ and eight CpGs downstream of the insertion in wild-type (C57BL/6 and 129s2/Sv, top), _Dac1J_/_Dac1J_ C57BL/6 (middle) or 129s2/Sv (bottom) mice measured by bisulfite cloning and sequencing from E11.5 limbs. White circles, unmethylated CpGs; black circles, methylated CpGs. **e**, Schematic representation of the CRISPR–Cas9 _Dac1J_ deletion in the _Dac1J_/_Dac1J_ (129s2/Sv) line. **f**, Skeletal analysis of E18.5 _Dac1J_ Δ/Δ (129s2/Sv) forelimbs and hindlimbs showing complete rescue. Scale bars, 1 mm. _n_ = 11 out of 11 E18.5 showed a similar phenotype. **g**, CpG DNA methylation status in _Dac1J_ Δ/Δ (129s2/Sv) E11.5 limbs as in **d**.

To functionally prove that the dactylaplasia was caused by the MusD-_Dac1J_ insertion at the _Fgf8_ locus, we used CRISPR–Cas9 to delete it from the _Dac1J_/_Dac1J_ line (Fig. [1e](https://www.nature.com/articles/s41588-025-02248-5#Fig1)). E18.5 embryos with a deletion of MusD-_Dac1J (_Δ_Dac1J_/Δ_Dac1J)_ show a complete rescue of the phenotype (Fig. [1f](https://www.nature.com/articles/s41588-025-02248-5#Fig1)), confirming the causative nature of the LTR insertion. The deletion of the _Dac1J_ did not affect the local CpG DNA methylation (Fig. [1g](https://www.nature.com/articles/s41588-025-02248-5#Fig1)), and epigenetic profiling using publicly available data[23](https://www.nature.com/articles/s41588-025-02248-5#ref-CR23 "Andrey, G. et al. Characterization of hundreds of regulatory landscapes in developing limbs reveals two regimes of chromatin folding. Genome Res. 27, 223–233 (2017)."),[24](https://www.nature.com/articles/s41588-025-02248-5#ref-CR24 "Kraft, K. et al. Serial genomic inversions induce tissue-specific architectural stripes, gene misexpression and congenital malformations. Nat. Cell Biol. 21, 305–310 (2019)."),[25](https://www.nature.com/articles/s41588-025-02248-5#ref-CR25 "Ringel, A. R. et al. Repression and 3D-restructuring resolves regulatory conflicts in evolutionarily rearranged genomes. Cell 185, 3689–3704.e21 (2022).") suggests that neither the _Dac1J_ insertion nor our deletion overlaps with an open chromatin region (Extended Data Fig. [1f](https://www.nature.com/articles/s41588-025-02248-5#Fig7)).

### _Fgf8_\-expressing cells are affected in _Dac1J_ embryos

The locus contains six genes: _Btrc_, _Poll_, _Dpcd_ and _Fbxw4_ are ubiquitously and lowly expressed during limb development, whereas _Lbx1_ and _Fgf8_ are specifically expressed in the muscle progenitors and AER of the developing limbs, respectively. To elucidate whether the MusD insertion affects local gene activation, we performed bulk RNA sequencing (RNA-seq) from forelimbs at early E11.5, a stage in which the activation of all genes at the locus should be completed. Except for _Fgf8_, local gene expression is not affected at that stage (Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02248-5#Fig8)). _Fgf8_ expression in AER of the developing limb ranges from E9.5 to E13.5, where it controls growth and patterning[26](https://www.nature.com/articles/s41588-025-02248-5#ref-CR26 "Crossley, P. H. & Martin, G. R. The mouse Fgf8 gene encodes a family of polypeptides and is expressed in regions that direct outgrowth and patterning in the developing embryo. Development 121, 439–451 (1995)."). Whole-mount in situ hybridization confirmed _Fgf8_ expression is affected at E11.5 in _Dac1J_ embryos, but showed no difference with wild-type at E10.5 (Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02248-5#Fig8)).

To decipher how transcription is affected over time, we performed single-cell RNA-seq (scRNA-seq) from wild-type and _Dac1J_/_Dac1J_ limb buds at E9.5, E10.5 and E11.5 (Fig. [2a](https://www.nature.com/articles/s41588-025-02248-5#Fig2)). We annotated seven cell types (Fig. [2b](https://www.nature.com/articles/s41588-025-02248-5#Fig2), Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02248-5#Fig9) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02248-5#MOESM4)) that were present at similar proportions between wild-type and _Dac1J_/_Dac1J_ at E9.5 and E10.5 (Fig. [2c](https://www.nature.com/articles/s41588-025-02248-5#Fig2) and Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02248-5#Fig9)). However, at E11.5, a 82.6% loss of AER and dorso-ventral ectoderm cells was detected (Fig. [2c](https://www.nature.com/articles/s41588-025-02248-5#Fig2) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02248-5#MOESM1)). We detected _Fgf8_ expression specifically in the AER and dorso-ventral ectoderm cells and show that it was not affected at E9.5 and only mildly affected at E10.5 in _Dac1J_/_Dac1J_ limbs compared to wild-type (Fig. [2d](https://www.nature.com/articles/s41588-025-02248-5#Fig2) and Extended Data Fig. [3c](https://www.nature.com/articles/s41588-025-02248-5#Fig9)). At E11.5, the very few AER cells in the _Dac1J_/_Dac1J_ expressed _Fgf8_ at a low level (Extended Data Figs. [2b](https://www.nature.com/articles/s41588-025-02248-5#Fig8), [3c](https://www.nature.com/articles/s41588-025-02248-5#Fig9) and [4a](https://www.nature.com/articles/s41588-025-02248-5#Fig10)). To address whether the observed _Fgf8_ expression in _Dac1J_/_Dac1J_ mice was directly caused by transcriptional changes or by the observed depletion of AER cells, we examined the expression of other AER genes. These genes show the same expression pattern as _Fgf8_ with a proper activation at E9.5, a mild or no decrease at E10.5 and a decrease at E11.5 when the AER cells are lost (Extended Data Fig. [4b](https://www.nature.com/articles/s41588-025-02248-5#Fig10)). Overall, our results indicate that AER cells rather than gene activation are affected by the MusD-_Dac1J_ insertion.

**Fig. 2: MusD-_Dac1J_ is expressed in _Fgf8_\-expressing cells before their disappearance.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02248-5/figures/2)

**a**, Schematic representation of the embryos used for limb bud scRNA-seq. **b**, Uniform manifold approximation and projection (UMAP) showing seven cell clusters identified through scRNA-seq of E9.5, E10.5 and E11.5 stage mouse forelimbs from wild-type and _Dac1J/Dac1J_ embryos. **c**, Proportion of cell types within the forelimb population in E9.5, E10.5 and E11.5 wild-type and _Dac1J/Dac1J_ embryos. The seven colors correspond to the seven cell types as represented in **b**. Gray shading represents a zoomed-in view without mesenchyme cells. It shows a strong decrease of AER and dorso-ventral cells in Dac1J/Dac1J at E11.5 compared to wild-type, as represented with black outlines. Zoomed-in view without mesenchyme cells shows a strong decrease of AER and dorso-ventral (DV) cells in _Dac1J/Dac1J_ embryos at E11.5 compared to wild-type. **d**,**e**, Dot plots showing expression and percentage of expressing cells for _Fgf8_ (**d**) and MusD-_Dac1J_ (**e**) in the seven forelimb cell cluster at E9.5 and E10.5 in wild-type (dark gray) and _Dac1J/Dac1J_ (green). Unique molecular identifier (UMI) expression in the AER and dorso-ventral ectoderm cell type is also represented as a feature plot next to the respective dot plot for each genotype and stage. **f**, Wild-type (left) and _Dac1J/Dac1J_ (right) AER and dorso-ventral cells colored depending on their status of expressing _Fgf8_ and _Dac1J_. Percent of cells expressing either MusD-_Dac1J_, MusD-_Dac1J+Fgf8_, _Fgf8_ or none of them is indicated.

### Co-expression of MusD-_Dac1J_ and _Fgf8_ in the developing limb

We confirmed that the activation and cell-type-specific expression of the other five genes at the locus was not affected in _Dac1J_/_Dac1J_ compared to wild-type (Extended Data Figs. [3c](https://www.nature.com/articles/s41588-025-02248-5#Fig9) and [4c](https://www.nature.com/articles/s41588-025-02248-5#Fig10) and Supplementary Note [2](https://www.nature.com/articles/s41588-025-02248-5#MOESM1)). However, when scRNA-seq reads were mapped to a transcriptome containing the _Dac1J_ sequence, we observed MusD-_Dac1J_ expression specifically in _Dac1J_/_Dac1J_ AER cells (Fig. [2e](https://www.nature.com/articles/s41588-025-02248-5#Fig2)). MusD-_Dac1J_ transcripts were detected at E9.5 and E10.5 but not at E11.5 when AER cells vanished (Fig. [2e](https://www.nature.com/articles/s41588-025-02248-5#Fig2) and Fig. Extended Data Fig. [4a](https://www.nature.com/articles/s41588-025-02248-5#Fig10)). Remarkably, nearly all AER cells expressing MusD-_Dac1J_ co-expressed _Fgf8_ (Fig. [2f](https://www.nature.com/articles/s41588-025-02248-5#Fig2)), showing MusD-_Dac1J_ transcription in the same pattern as _Fgf8_. These results indicate that MusD-_Dac1J_ is co-expressed with _Fgf8_ in the AER of the developing limb at E9.5 and E10.5, followed by a disappearance of most of the AER cells at E11.5.

### Active MusD-_Dac1J_ leads to an ectopic chromatin loop

We next asked whether MusD-_Dac1J_ co-expression with _Fgf8_ is linked to changes at the 3D genome architecture level. The locus consists of two TADs[27](https://www.nature.com/articles/s41588-025-02248-5#ref-CR27 "Cova, G. et al. Combinatorial effects on gene expression at the Lbx1/Fgf8 locus resolve split-hand/foot malformation type 3. Nat. Commun. 14, 1475 (2023).") (Fig. [3a](https://www.nature.com/articles/s41588-025-02248-5#Fig3)). TADs are self-interacting blocks containing genes with their regulatory elements and are insulated from each other by boundaries defined by the binding of cohesin and CTCF, or by active transcription[28](https://www.nature.com/articles/s41588-025-02248-5#ref-CR28 "Nora, E. P. et al. Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature 485, 381–385 (2012)."),[29](https://www.nature.com/articles/s41588-025-02248-5#ref-CR29 "Dixon, J. R. et al. Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature 485, 376–380 (2012)."),[30](https://www.nature.com/articles/s41588-025-02248-5#ref-CR30 "Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014)."). MusD-_Dac1J_ is located in the 175 kb TAD containing _Fbxw4_ and _Fgf8_ as well as previously characterized _Fgf8_ enhancers (blue bars in Fig. [3a](https://www.nature.com/articles/s41588-025-02248-5#Fig3)) driving its expression in the AER, the hindbrain–midbrain boundary, the branchial arches and the tailbud[27](https://www.nature.com/articles/s41588-025-02248-5#ref-CR27 "Cova, G. et al. Combinatorial effects on gene expression at the Lbx1/Fgf8 locus resolve split-hand/foot malformation type 3. Nat. Commun. 14, 1475 (2023)."),[31](https://www.nature.com/articles/s41588-025-02248-5#ref-CR31 "Marinić, M., Aktas, T., Ruf, S. & Spitz, F. An integrated holo-enhancer unit defines tissue and gene specificity of the Fgf8 regulatory landscape. Dev. Cell 24, 530–542 (2013)."). Recent data suggest that 3D genome organization can be shaped by retrotransposon-derived CTCF sites or their active transcription[32](https://www.nature.com/articles/s41588-025-02248-5#ref-CR32 "Zhang, Y. et al. Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells. Nat. Genet. 51, 1380–1388 (2019)."),[33](https://www.nature.com/articles/s41588-025-02248-5#ref-CR33 "Lawson, H. A., Liang, Y. & Wang, T. Transposable elements in mammalian chromatin organization. Nat. Rev. Genet. 24, 712–723 (2023)."). We subjected the MusD-_Dac1J_ sequence to CTCF motif scanning and identified several binding sites (Fig. [3b](https://www.nature.com/articles/s41588-025-02248-5#Fig3) and Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02248-5#Fig11)). To investigate whether MusD-_Dac1J_ affects the 3D chromatin architecture at the locus, we performed capture-Hi-C (cHi-C) and CTCF chromatin immunoprecipitation with sequencing (ChIP–seq) from _Dac1J_, _Dac1J_\-BL6 and wild-type embryonic limbs. In the _Dac1J_ mutants, the TAD structure was maintained, but an ectopic chromatin loop was formed at the MusD-_Dac1J_ insertion site (Fig. [3c](https://www.nature.com/articles/s41588-025-02248-5#Fig3)). Subtracting the cHi-C signal between mutants and wild-type showed gain of contact where the ectopic loop forms (Fig. [3d](https://www.nature.com/articles/s41588-025-02248-5#Fig3), red) and increased insulation within the _Fgf8_ TAD in _Dac1J_ mutants (Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02248-5#Fig11), blue). The _Dac1J_\-BL6 embryos did not show any change in chromatin conformation (Extended Data Fig. [5c](https://www.nature.com/articles/s41588-025-02248-5#Fig11)). ChIP–seq from embryonic limbs suggests that binding of CTCF has a minor role in the formation of the ectopic chromatin loop, as weak ectopic binding in _Dac1J_ mutants was observed strictly at E10.5 (Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02248-5#Fig11)). We next asked whether the ectopic chromatin loop formed by hypomethylated MusD-_Dac1J_ exist in a tissue in which the region is inactive. cHi-C from _Dac1J_ embryonic hearts, in which MusD-_Dac1J_ 5′LTR is hypomethylated (Extended Data Fig. [5e](https://www.nature.com/articles/s41588-025-02248-5#Fig11)), did not detect any ectopic chromatin loop (Fig. [3e,f](https://www.nature.com/articles/s41588-025-02248-5#Fig3)).

**Fig. 3: MusD-_Dac1J_ insertion leads to an ectopic chromatin loop.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02248-5/figures/3)

**a**, cHi-C of the _Lbx1/Fgf8_ locus (mm10, chr19:45,100,000–45,900,000) from wild-type E11.5 mouse limb buds. Data are shown as merged signals of _n_ = 3 biological replicates and _n_ = 1 technical replicate. The dashed lines indicate the _Lbx1_ and _Fgf8_ TADs. Published _Fgf8_ enhancers are indicated in light blue and dark blue (AER enhancers). Color bar shows the KR-normalized contact probability (white: low; black: high). **b**, Schematic representation of the MusD-_DacJ_ and the 5′LTR-LacZ with the number of CTCF sites with _P_ < 10−4 (calculated using FIMO, based on a dynamic programming algorithm to convert log-odds scores into _P_ values) indicated as orange (sense) and red (antisense) triangles. KI, knock-in. **c**–**h**, cHi-C of the _Lbx1/Fgf8_ locus (mm10, chr19:45,100,000–45,900,000) from _Dac1J/Dac1J_ E11.5 mouse limb buds (**c**,**d**), _Dac1J/Dac1J_ E11.5 mouse embryonic hearts (**e**,**f**) and _Dac1J_\-5LTR-LacZ E11.5 limb buds (**g**,**h**). Data show the cHi-C as merged signals of _n_ = 2 biological replicates and _n_ = 2 technical replicates (**c**,**d**), _n_ = 2 biological replicates (**e**,**f**) and _n_ = 3 biological replicates (**g**,**h**). Subtraction maps between mutants and wild-type (**d**, **f**, **h**) show gain (red) and loss (blue) of interaction in the mutant compared to the wild-type. The position of the MusD-_Dac1J_ insertion is indicated in green. Scale bars in **a**, **c**, **e** and **g** show the KR-normalized contact probability (white: low; black: high). Scales bars in **d**, **f** and **h** show the _z_\-scaled differential contact probability (red: gain; blue: loss). **i**, 4C-seq with a viewpoint (VP) on _Dac1J_ showing _Dac1J/Dac1J_ (129s2/Sv) and _Dac1J/Dac1J_ (C57BL/6). Green arrows indicate a difference in 4C enrichments. In vivo-confirmed _Fgf8_ enhancers are indicated in light blue and dark blue (specific AER enhancers), and the MusD-_Dac1J_ insertion point is indicated in green. **j**, Representation of the _Dac1J_\-5LTR-LacZ and β-globin-LacZ knock-ins at the _Fgf8_ locus and their β-galactosidase staining on E11.5 embryos and forelimbs, zoomed in. Scale bars, 500 µm; _n_ = 20 out of 20 and 23 out of 23 embryos showed similar staining.

Finally, we tested whether the MusD-_Dac1J_ 5′LTR promoter (containing one CTCF site; Fig. [3b](https://www.nature.com/articles/s41588-025-02248-5#Fig3)) would be sufficient to drive the ectopic chromatin loop. We generated a knock-in at the _Fgf8_ locus with the MusD-_Dac1J_ 5′LTR but replaced the retroviral sequence with a LacZ reporter gene (5LTR-LacZ-KI; Fig. [3b](https://www.nature.com/articles/s41588-025-02248-5#Fig3)). When performing cHi-C from this mutant’s embryonic limbs, we observed a similar ectopic loop as in the _Dac1J_ embryos (Fig. [3g,h](https://www.nature.com/articles/s41588-025-02248-5#Fig3) and Supplementary Note [3](https://www.nature.com/articles/s41588-025-02248-5#MOESM1)) with an unmethylated 5′LTR promoter (Extended Data Fig. [5e](https://www.nature.com/articles/s41588-025-02248-5#Fig11)). This shows that the 5′LTR promoter is sufficient to recapitulate the main 3D conformation changes mediated by MusD-_Dac1J_, even when not bound by CTCF (Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02248-5#Fig11)). However, this ectopic contact without the full-length MusD-_Dac1J_ retroviral sequence did not lead to any phenotype at E18.5 (Extended Data Fig. [5f](https://www.nature.com/articles/s41588-025-02248-5#Fig11)). Overall, our results show that the active MusD-_Dac1J_ promoter creates an ectopic chromatin loop that does not impact local gene activation or cause a phenotype.

### MusD-_Dac1J_ adopts the regulatory activity of the _Fgf8_ TAD

Our data suggest that the ectopic chromatin loop in _Dac1J_ embryos is the consequence rather than the cause of active MusD-_Dac1J_ transcription. Therefore, we wondered what drives MusD-_Dac1J_ co-expression with _Fgf8_. We performed 4C-sequencing (4C-seq) with the MusD-_Dac1J_ sequence as a viewpoint and detected more interaction within the region containing _Fgf8_ enhancers in the _Dac1J_ mutant than in the _Dac1J_\-BL6 control, suggesting MusD-_Dac1J_ 5′LTR compatibility with _Fgf8_ enhancers (Fig. [3i](https://www.nature.com/articles/s41588-025-02248-5#Fig3)). We took advantage of the 5LTR-LacZ-KI and generated a similar knock-in in which the LacZ reporter gene is driven by a β-globin minimal promoter instead of the 5′LTR (Fig. [3j](https://www.nature.com/articles/s41588-025-02248-5#Fig3)). β-galactosidase staining in E11.5 embryos with either construct inserted in the _Fgf8_ TAD both faithfully recapitulated the native _Fgf8_ expression (Fig. [3j](https://www.nature.com/articles/s41588-025-02248-5#Fig3)). This indicates that the MusD-_Dac1J_ 5′LTR promoter acts as a sensor, adopting the regulatory information within the _Fgf8_ TAD.

### Assembly of VLPs in _Dac1J_ AER cells

Having shown that MusD-_Dac1J_ uses the _Fgf8_ regulatory landscape for its transcription, we asked how this is linked to the lack of AER at E11.5. _Dac1J_ is a full-length MusD element with intact open reading frames for _gag_, _pro_ and _pol_ (Fig. [1a](https://www.nature.com/articles/s41588-025-02248-5#Fig1)), almost identical to three autonomous MusD elements[15](https://www.nature.com/articles/s41588-025-02248-5#ref-CR15 "Ribet, D., Dewannieux, M. & Heidmann, T. An active murine transposon family pair: retrotransposition of ‘master’ MusD copies and ETn trans-mobilization. Genome Res. 14, 2261–2267 (2004).") (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02248-5#Fig12) and Supplementary Note [4](https://www.nature.com/articles/s41588-025-02248-5#MOESM1)). This prompted us to examine whether MusD-_Dac1J_ produces retroviral proteins using a well-characterized Gag-MusD polyclonal antibody[15](https://www.nature.com/articles/s41588-025-02248-5#ref-CR15 "Ribet, D., Dewannieux, M. & Heidmann, T. An active murine transposon family pair: retrotransposition of ‘master’ MusD copies and ETn trans-mobilization. Genome Res. 14, 2261–2267 (2004).") for whole-embryo immunofluorescence assay. We detected Gag-MusD in all analyzed _Dac1J_/_Dac1J_ embryos but no staining in the _Dac1J_\-BL6 controls (Fig. [4a](https://www.nature.com/articles/s41588-025-02248-5#Fig4) and Extended Data Fig. [6b–d](https://www.nature.com/articles/s41588-025-02248-5#Fig12)). Cytoplasmic Gag-MusD was observed in tissues expressing _Fgf8_, such as the midbrain–hindbrain boundary, the branchial arches and the AER of both forelimbs and hindlimbs at E10.5, but not in the tailbud (Fig. [4a](https://www.nature.com/articles/s41588-025-02248-5#Fig4) and Extended Data Fig. [6c](https://www.nature.com/articles/s41588-025-02248-5#Fig12)). Gag-MusD persisted in the AER of _Dac1J_ E11.0 embryos (Extended Data Fig. [6d](https://www.nature.com/articles/s41588-025-02248-5#Fig12)). We thus analyzed the presence of VLPs in the AER cells. Remarkably, transmission electron microscopy (TEM) revealed the presence of cytoplasmic electron-dense particles of 70–90 nm, the reported size for MusD VLPs[34](https://www.nature.com/articles/s41588-025-02248-5#ref-CR34 "Ribet, D., Harper, F., Dewannieux, M., Pierron, G. & Heidmann, T. Murine MusD retrotransposon: structure and molecular evolution of an ‘intracellularized’ retrovirus. J. Virol. 81, 1888–1898 (2007).") (Fig. [4b](https://www.nature.com/articles/s41588-025-02248-5#Fig4)). VLPs were detected both in clusters and as single particles nearby the nuclear membrane and in the cisternae of the endoplasmic reticulum (Fig. [4b](https://www.nature.com/articles/s41588-025-02248-5#Fig4)). The presence of MusD-derived particles in the AER of _Dac1J_ embryos was supported by immuno-gold TEM staining, which detected VLPs at the nuclear membrane labeled by Gag-MusD antibodies in several AER cells (Fig. [4c](https://www.nature.com/articles/s41588-025-02248-5#Fig4) and Extended Data Fig. [6e–g](https://www.nature.com/articles/s41588-025-02248-5#Fig12)). MusD-_Dac1J_ VLPs were strictly intracellular and did not bud, as expected from an ERV lacking the _env_ gene[34](https://www.nature.com/articles/s41588-025-02248-5#ref-CR34 "Ribet, D., Harper, F., Dewannieux, M., Pierron, G. & Heidmann, T. Murine MusD retrotransposon: structure and molecular evolution of an ‘intracellularized’ retrovirus. J. Virol. 81, 1888–1898 (2007)."). Staining from wild-type embryos showed no signal (Extended Data Fig. [6h](https://www.nature.com/articles/s41588-025-02248-5#Fig12)). This demonstrates the presence of retroviral proteins and particles in the AER cells of the developing _Dac1J_ mutant embryos.

**Fig. 4: Presence of MusD retroviral-like particles is associated with AER cell death.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02248-5/figures/4)

**a**, Anti-Gag-MusD whole-mount immunofluorescence on E10.5 _Dac1J_/_Dac1J_ embryos showing embryo (left), forelimb (middle) and AER (right). _n_ = 5 out of 5 biological replicates were confirmed. FL, forelimb; HL, hindlimb; BA, branchial arches; MHB, midbrain–hindbrain boundary. **b**, TEM analysis on E11.0 _Dac1J_/_Dac1J_ AER cells. Three different cells are shown. Green arrowheads and dotted lines indicate single and aggregated VLPs, respectively. _n_ = 5 biological and _n_ = 2 technical replicates. C, cytoplasm; N, nucleus; ER, endoplasmic reticulum. **c**, TEM analysis after immuno-gold labeling with anti-Gag-MusD on E11.0 _Dac1J_/_Dac1J_ AER cell shows cytoplasmic aggregates of GAG. _n_ = 2 biological replicates and _n_ = 6 technical replicates were confirmed. **d**, TEM analysis on E11.0 wild-type and _Dac1J_/_Dac1J_ forelimbs. Zoomed-in view (right) shows the structure of an AER cell. Red asterisk indicates an apoptotic body. **e**, Anti-cleaved-Caspase-3 whole-mount immunofluorescence on E11.0 wild-type and _Dac1J_/_Dac1J_ forelimb and AER. _n_ = 4 out of 4 (wild-type) and _n_ = 6 out of 6 (_Dac1J_/_Dac1J_) biological replicates were confirmed. **f**,**g**, Anti-γH2AX whole-mount immunofluorescence on wild-type and _Dac1J_/_Dac1J_ E10.5 forelimbs (**f**) and E11.0 forelimbs and AER (**g**). _n_ = 3 out of 3 biological replicates were confirmed. **h**, Schematic representation of wild-type and _Dac1J_/_Dac1J_ limbs at E10.5, E11.0, E11.5 and E18.5. Although the AER of the limb bud is not morphologically affected in the _Dac1J_/_Dac1J_ at E10.5, VLPs are expressed, and we detected increased DNA damage. Half a developmental day later, more DNA damage and apoptosis occur. By E11.5, most of the AER cells have been eliminated by cell death. Consequently, at E18.5, the autopod of the _Dac1J_/_Dac1J_ embryos is severely affected by a lack of metacarpal structures, but the stylopod and zeugopod have developed similarly to the wild-type embryos. Styl., Stylopod; Zeug., Zeugopod; Aut., Autopod.

### MusD VLPs in the AER are associated with cell death

To decipher how the presence of MusD VLPs during limb development is linked to the disappearance of AER cells and the phenotype, we examined TEM images of wild-type and _Dac1J_ developing limbs at E11.0. Numerous _Dac1J_ AER cells contain up to four (or occasionally more) apoptotic bodies and phagocytes (Fig. [4d](https://www.nature.com/articles/s41588-025-02248-5#Fig4) and Extended Data Fig. [6i,j](https://www.nature.com/articles/s41588-025-02248-5#Fig12)). Apoptotic cell death was confirmed by immuno-staining with a cleaved-Caspase-3 antibody, which shows Caspase-3 in its active form in AER cells of both forelimbs and hindlimbs and lower signal in a few cells of the branchial arches but no staining in wild-type embryos (Fig. [4e](https://www.nature.com/articles/s41588-025-02248-5#Fig4) and Extended Data Fig. [7a](https://www.nature.com/articles/s41588-025-02248-5#Fig13)). We did not detect cleaved-Caspase-3 at E10.5, but γH2AX staining revealed elevated levels of DNA damage in the AER compared to wild-type (Fig. [4f](https://www.nature.com/articles/s41588-025-02248-5#Fig4)). This was even more pronounced at E11.0, concomitant to the activation of Caspase-3 (Fig. [4g](https://www.nature.com/articles/s41588-025-02248-5#Fig4) and Extended Data Fig. [7b](https://www.nature.com/articles/s41588-025-02248-5#Fig13)). Fluorescence in situ hybridization staining of the developing limb bud using an _Fgf8_ mRNA probe supported proper AER morphology at E10.0 and E10.5 in _Dac1J_ embryos (Extended Data Fig. [7c,d](https://www.nature.com/articles/s41588-025-02248-5#Fig13)) despite few cells being possibly affected by the increased DNA damage (Fig. [4f](https://www.nature.com/articles/s41588-025-02248-5#Fig4)). Typical apoptosis characteristics were observed in AER cells, such as blebbing and nuclear fragmentation, beginning at E11.0 and exacerbated at E11.5 (Extended Data Fig. [7e,f](https://www.nature.com/articles/s41588-025-02248-5#Fig13)). According to our transcriptomic data, _Fgf8_ expression globally decreased after E10.5 owing to AER cells undergoing DNA damage and apoptosis (Extended Data Fig. [7c–f](https://www.nature.com/articles/s41588-025-02248-5#Fig13)). Early E11.5 _Dac1J_ forelimbs showed a significant difference for apoptosis-related and limb patterning genes (Extended Data Fig. [8a,b](https://www.nature.com/articles/s41588-025-02248-5#Fig14)). However, we did not detect activation of the interferon pathway (Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02248-5#Fig14) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02248-5#MOESM4)), suggesting that _env_\-less MusD VLPs are not infectious. Overall, our results show that the presence of VLPs in the AER is toxic, leading to DNA damage and non-physiological apoptotic cell death during development (Fig. [4h](https://www.nature.com/articles/s41588-025-02248-5#Fig4)). This reinforces our finding that dactylaplasia is caused by a loss of AER cells between E10.5 and E11.5 rather than the misregulation of _Fgf8_.

### Lack of MusD-_Dac1J_ reverse transcription partially rescues dactylaplasia

As an autonomous ERV element, MusD undergo mRNA splicing and translation into the group-specific antigen (Gag) that assembles into a capsid polyprotein, the protease (Pro) for maturation of the gene products and the polymerase (Pol) encoding for reverse transcriptase, RNase H and integrase (Fig. [5a](https://www.nature.com/articles/s41588-025-02248-5#Fig5)). The MusD enzymatic machinery is also used by the related non-autonomous ETn elements for their retrotransposition[14](https://www.nature.com/articles/s41588-025-02248-5#ref-CR14 "Mager, D. L. & Freeman, J. D. Novel mouse type D endogenous proviruses and ETn elements share long terminal repeat and internal sequences. J. Virol. 74, 7221–7229 (2000).") (Fig. [5a](https://www.nature.com/articles/s41588-025-02248-5#Fig5)). Having shown that MusD-_Dac1J_ is transcribed, translated and produces VLPs in the AER, we wondered which step of the endogenous retroviral cycle causes the phenotype. We reasoned that the MusD VLPs-mediated AER cell death is directed by either the retroviral RNA, the capsid-containing Gag polyprotein, the replication of the ETn elements or the reverse transcription of its mRNA. To decipher between these mechanisms, we engineered five mutants carrying modification in the MusD-_Dac1J_ sequence (Fig. [5b](https://www.nature.com/articles/s41588-025-02248-5#Fig5)). Two mutants have similar consequences on the _pol_ gene: (1) ‘_Dac1J_\-ΔPol’, generated from the _Dac1J_/_Dac1J_ line, contains a deletion of the 3′ part of MusD-_Dac1J_ resulting in a shorter Pol protein (Fig. [5b](https://www.nature.com/articles/s41588-025-02248-5#Fig5) and Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02248-5#Fig15)); and (2) ‘_Dac1J-_mut-pol’ is a knock-in of the MusD-_Dac1J_ with an out-of-frame 493 bp deletion in the _pol_ gene, introducing a 2 bp frameshift resulting in a premature STOP codon as previously generated[15](https://www.nature.com/articles/s41588-025-02248-5#ref-CR15 "Ribet, D., Dewannieux, M. & Heidmann, T. An active murine transposon family pair: retrotransposition of ‘master’ MusD copies and ETn trans-mobilization. Genome Res. 14, 2261–2267 (2004).") (Fig. [5b](https://www.nature.com/articles/s41588-025-02248-5#Fig5)). We verified that the Gag protein was correctly translated in this knock-in, showing that it recapitulates _Dac1J-_mut-pol transcription and translation as in the original _Dac1J_ line (Fig. [5c](https://www.nature.com/articles/s41588-025-02248-5#Fig5) and Extended Data Fig. [9b](https://www.nature.com/articles/s41588-025-02248-5#Fig15)). The lack of a functional Pol protein leads to a major rescue of the dactylaplasia phenotype with a milder, not fully penetrant limb phenotype (Fig. [5d](https://www.nature.com/articles/s41588-025-02248-5#Fig5) and Extended Data Fig. [9c](https://www.nature.com/articles/s41588-025-02248-5#Fig15)). A total of 29.6% (32 out of 108) and 15.8% (24 out of 152) of the observed limbs were affected in _Dac1J_\-Δ/ΔPol and _Dac1J-_mut-pol KI/KI, respectively. None of these animals showed the classical one-digit dactylaplasia phenotype (Fig. [5d](https://www.nature.com/articles/s41588-025-02248-5#Fig5)).

**Fig. 5: Knock-ins of MusD-_Dac1J_ carrying mutation rescue dactylaplasia phenotypes.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02248-5/figures/5)

**a**, Schematic representation of the MusD endogenous retroviral cycle in a cell. MusD DNA insertion is transcribed into mRNA in the nucleus. MusD RNA is then translated into the Gag (producing the capsid and matrix), Pro (the protease) and Pol (producing the reverse transcriptase, RNase H and integrase) in the cytoplasm and assembled into VLPs. The assembled VLPs contain reverse transcriptase, integrase and RNase H and are formed by the Gag capsid, which was cleaved by the protease. The VLP can replicate the non-autonomous ETn retrotransposons but also undergo tRNA-primed reverse transcription to reintegrate in the genome through a cDNA intermediate (as indicated with the dashed arrows). **b**, Representation of the _Dac1J_ insertion along with the five mutated versions. Red and orange triangles represent CTCF sites. Purple band shows the location of the Lys-tRNA primer binding site, as wild-type and mutated (\*). Red hexagon represents a Stop codon. **c**, Anti-GAG-MusD whole-mount immunofluorescence on E10.5 knock-in mutants: mut-pol KI/KI, mut-PBS KI/KI, mut-gag KI/KI and mut-gag(CA) KI/KI forelimb. At least _n_ = 3 biological replicates were confirmed. **d**, Histogram of the percentage of animals showing the six possible digit phenotype situations in the different mutants. _n_ represents the number of limbs (forelimbs and hindlimbs) analyzed.

As LTR elements, MusD and ETn have tRNA primer-binding sites (PBS) to initiate reverse transcription and copy their RNA into DNA to reinsert themselves in the genome[35](https://www.nature.com/articles/s41588-025-02248-5#ref-CR35 "Schorn, A. J. & Martienssen, R. Tie-break: host and retrotransposons play tRNA. Trends Cell Biol. 28, 793–806 (2018).") (Fig. [5a](https://www.nature.com/articles/s41588-025-02248-5#Fig5)). The _Dac1J pol_ mutants prevented the replication of both MusD-_Dac1J_ itself and non-autonomous ETns. To decipher whether replication of ETn elements has a role in the phenotype, we engineered a knock-in of the full-length MusD-_Dac1J_ with a disrupted PBS (Fig. [5b](https://www.nature.com/articles/s41588-025-02248-5#Fig5) and Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02248-5#Fig15)). A coding-competent MusD element with a scrambled PBS is no longer able to prime its reverse transcription but can still replicate ETns[36](https://www.nature.com/articles/s41588-025-02248-5#ref-CR36 "Schorn, A. J., Gutbrod, M. J., LeBlanc, C. & Martienssen, R. LTR-retrotransposon control by tRNA-derived small RNAs. Cell 170, 61–71.e11 (2017).") as it correctly expresses retroviral proteins (Fig. [5c](https://www.nature.com/articles/s41588-025-02248-5#Fig5) and Extended Data Fig. [9b](https://www.nature.com/articles/s41588-025-02248-5#Fig15)). _Dac1J-_mut-PBS mutants also showed a partial rescue of the dactylaplasia phenotype with 24% (21 out of 87) of observed limbs affected (Fig. [5d](https://www.nature.com/articles/s41588-025-02248-5#Fig5) and Extended Data Fig. [9c](https://www.nature.com/articles/s41588-025-02248-5#Fig15)). The partial rescue of the MusD-_Dac1J_ element with disrupted _pol_ and PBS suggest that reverse transcription of the MusD-_Dac1J_ RNA is a major cause of dactylaplasia. Yet the presence of Gag or retroviral RNA can drive limb phenotypes to a lesser extent.

### Lack of MusD-_Dac1J_ Gag capsid fully rescues dactylaplasia

To decipher whether Gag or the retroviral RNA is responsible for these limb phenotypes, we generated two other knock-ins affecting the _gag_ gene (Fig. [5b](https://www.nature.com/articles/s41588-025-02248-5#Fig5)). Both the _gag_ complete deletion (_Dac1J-_mut-gag) and partial deletion, leaving the matrix intact but deleting the capsid (CA) protein, (_Dac1J-_mut-gag(CA)) (Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02248-5#Fig15)) showed a full rescue of the phenotype with none of the observed limbs affected (Fig. [5d](https://www.nature.com/articles/s41588-025-02248-5#Fig5)). We validated that the MusD-Gag polyclonal antibody recognizes a truncated (_Dac1J-_mut-gag(CA) mutant) but not an absent (_Dac1J-_mut-gag mutant) GAG (Fig. [5c](https://www.nature.com/articles/s41588-025-02248-5#Fig5) and Extended Data Fig. [9b](https://www.nature.com/articles/s41588-025-02248-5#Fig15)).

For all four knock-ins, none of the observed heterozygote animals show a phenotype. We also validated that the variable phenotypes do not depend on DNA methylation variability, as all animals show hypomethylated 5′LTR (Extended Data Fig. [9d,e](https://www.nature.com/articles/s41588-025-02248-5#Fig15)). These observations suggest that the MusD-_Dac1J_ Gag capsid is essential to drive a limb phenotype, but in itself is not sufficient to generate a phenotype as severe as the full-length element.

### MusD-_Dac1J_ adopts the expression surrounding its insertion

Activation of the MusD-_Dac1J_ in the AER during development is driven by adoption of the _Fgf8_ enhancers. To decipher whether this was specific to the _Fgf8_ regulatory domain or a more general mechanism, we inserted the 5′LTR-LacZ construct into the _Lbx1_, _Shh_ and _Sox9_ TADs (Fig. [6a](https://www.nature.com/articles/s41588-025-02248-5#Fig6) and Extended Data Fig. [10a,b](https://www.nature.com/articles/s41588-025-02248-5#Fig16)). When inserted in the _Lbx1_ TAD (Fig. [6a](https://www.nature.com/articles/s41588-025-02248-5#Fig6)), adjacent to but separated from the _Fgf8_ domain by a TAD boundary (Fig. [3a](https://www.nature.com/articles/s41588-025-02248-5#Fig3)), LacZ expression recapitulated the _Lbx1_ expression in muscle progenitors of the developing limbs[37](https://www.nature.com/articles/s41588-025-02248-5#ref-CR37 "Brohmann, H., Jagla, K. & Birchmeier, C. The role of Lbx1 in migration of muscle precursor cells. Development 127, 437–445 (2000).") (Fig. [6b,c](https://www.nature.com/articles/s41588-025-02248-5#Fig6)). Similarly, 5′LTR-LacZ insertion in the _Shh_ and the _Sox9_ TADs sensed the surrounding expression in the developing notochord and the limb chondrocytes, respectively (Extended Data Fig. [10c–f](https://www.nature.com/articles/s41588-025-02248-5#Fig16) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02248-5#MOESM1)). Altogether, this reveals yet undescribed properties of an unmethylated 5′LTR, adopting the regulatory activity present in the TAD where it has been inserted.

**Fig. 6: MusD-_Dac1J_ adopts the muscle progenitor expression when inserted in _Lbx1_ TAD.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02248-5/figures/6)

**a**, Representation of the three knock-in lines engineered in the _Lbx1_ TAD. Purple band shows the location of the Lys-tRNA primer binding site, as wild-type and mutated (\*). **b**, In situ hybridization for _Lbx1_ in E10.5 wild-type showing whole embryos and forelimb. Scale bars, 500 µm. At least _n_ = 3 biological replicates were confirmed. **c**, β-galactosidase staining on E10.5 _Dac1J_\-5LTR-LacZ (_Lbx1_ knock-in) showing whole embryos and forelimb. Scale bars, 500 µm. _n_ = 10 out of 10 embryos show similar staining. **d**, Anti-Gag-MusD whole-mount immunofluorescence on E11.5 _Dac1J_ KI/KI (_Lbx1_ knock-in) showing whole embryo (left) and forelimb (right). At least _n_ = 3 biological replicates were confirmed.

To investigate whether the production of the Gag-MusD polyprotein is specific to _Fgf8_\-expressing cells or could also exist in different limb cells, we engineered knock-ins of MusD-_Dac1J_ in its full-length and scrambled PBS forms in the _Lbx1_ TAD (Fig. [6a](https://www.nature.com/articles/s41588-025-02248-5#Fig6)). Gag-MusD staining was observed in the _Lbx1_\-expressing cells of the developing limb (Fig. [6d](https://www.nature.com/articles/s41588-025-02248-5#Fig6) and Extended Data Fig. [10g](https://www.nature.com/articles/s41588-025-02248-5#Fig16)). This suggests that MusD VLPs can assemble in any embryonic cells when adopting the regulatory information from a developmental gene. The AER and branchial arches are embryonic epithelium, but _Lbx1_\-expressing cells are from the muscle lineage. Gag-MusD expression in _Lbx1_\-expressing cells was maintained at E11.5 (Fig. [6d](https://www.nature.com/articles/s41588-025-02248-5#Fig6)), whereas at this stage, most of the AER cells are dead in the _Dac1J_/_Dac1J_ mutants (Fig. [2c](https://www.nature.com/articles/s41588-025-02248-5#Fig2)). When assessing cell death at E12.5, we did not observe muscle progenitor cells undergoing apoptosis (Extended Data Fig. [10h](https://www.nature.com/articles/s41588-025-02248-5#Fig16)), and embryos at E18.5 did not show any striking morphological phenotype. This suggests that cells from a muscle lineage tolerated VLP production during development. These results indicate that MusD VLPs production during development affects embryonic cells differently and highlight that mouse post-implantation development can proceed with the presence of VLPs.

Discussion
----------

In this study, we uncover that an ERV element escaping epigenetic silencing adopts the regulatory activity of nearby developmental genes, resulting in time-specific and cell-type-specific expression of retroviral products during organogenesis. In the dactylaplasia mutant, VLP assembly in the AER of the developing limb affects cell survival at a critical time preceding digit formation, leading to a lack of digits in newborn mice. The MusD-_Dac1J_ insertion is one of the few reported mutagenic LTR insertions not located in an intron of a gene[13](https://www.nature.com/articles/s41588-025-02248-5#ref-CR13 "Gagnier, L., Belancio, V. P. & Mager, D. L. Mouse germ line mutations due to retrotransposon insertions. Mob. DNA 10, 15 (2019)."). Independent of _Dac1J_, a second MusD insertion (_Dac2J_) was reported in the intron of the _Fbxw4_ gene, also causing dactylaplasia[38](https://www.nature.com/articles/s41588-025-02248-5#ref-CR38 "Sidow, A. et al. A novel member of the F-box/WD40 gene family, encoding dactylin, is disrupted in the mouse dactylaplasia mutant. Nat. Genet. 23, 104–107 (1999)."). Given that _Dac2J_ affects _Fbxw4_ transcription and both insertions are nearby _Fgf8_, dactylaplasia was believed to be primarily caused by disruption of gene transcription[38](https://www.nature.com/articles/s41588-025-02248-5#ref-CR38 "Sidow, A. et al. A novel member of the F-box/WD40 gene family, encoding dactylin, is disrupted in the mouse dactylaplasia mutant. Nat. Genet. 23, 104–107 (1999)."),[39](https://www.nature.com/articles/s41588-025-02248-5#ref-CR39 "Seto, M. L., Nunes, M. E., MacArthur, C. A. & Cunningham, M. L. Pathogenesis of ectrodactyly in the Dactylaplasia mouse: aberrant cell death of the apical ectodermal ridge. Teratology 56, 262–270 (1997)."),[40](https://www.nature.com/articles/s41588-025-02248-5#ref-CR40 "Crackower, M. A., Motoyama, J. & Tsui, L. C. Defect in the maintenance of the apical ectodermal ridge in the Dactylaplasia mouse. Dev. Biol. 201, 78–89 (1998)."),[41](https://www.nature.com/articles/s41588-025-02248-5#ref-CR41 "Friedli, M. et al. Characterization of mouse Dactylaplasia mutations: a model for human ectrodactyly SHFM3. Mamm. Genome 19, 272–278 (2008).").

Here, we show that the MusD-_Dac1J_ insertion does not directly affect gene transcription. Instead, it adopts _Fgf8_ regulatory elements as defined by the TAD boundaries. Similar results were obtained at other loci, indicating that adoption of local regulatory information by a TE element is likely to be a more common mechanism. _Fgf8_ conditional inactivation in the limb leads to a milder phenotype than in the dactylaplasia mutants[42](https://www.nature.com/articles/s41588-025-02248-5#ref-CR42 "Lewandoski, M., Sun, X. & Martin, G. R. Fgf8 signalling from the AER is essential for normal limb development. Nat. Genet. 26, 460–463 (2000)."),[43](https://www.nature.com/articles/s41588-025-02248-5#ref-CR43 "Moon, A. M. & Capecchi, M. R. Fgf8 is required for outgrowth and patterning of the limbs. Nat. Genet. 26, 455–459 (2000)."), which more closely resembles the double _Fgf8_; _Fgf4_ inactivation[44](https://www.nature.com/articles/s41588-025-02248-5#ref-CR44 "Mariani, F. V., Ahn, C. P. & Martin, G. R. Genetic evidence that FGFs have an instructive role in limb proximal–distal patterning. Nature 453, 401–405 (2008)."). The dactylaplasia phenotype is more comparable to the removal of AER at stage 25 in chick embryos, corresponding to mouse E11.5, the stage at which AER cells are lost in _Dac1J_/_Dac1J_ embryos[45](https://www.nature.com/articles/s41588-025-02248-5#ref-CR45 "Summerbell, D. A quantitative analysis of the effect of excision of the AER from the chick limb bud. J. Embryol. Exp. Morphol. 32, 651–660 (1974)."). These observations support our data showing that the morphological changes are caused by loss of AER cells at a critical developmental stage rather than changes in gene expression.

VLPs from LTR retrotransposons have been observed in pathological contexts when global epigenetic changes occur, resulting in the derepression of many elements from several families; for example, in the developing brain[46](https://www.nature.com/articles/s41588-025-02248-5#ref-CR46 "Jönsson, M. E. et al. Activation of endogenous retroviruses during brain development causes an inflammatory response. EMBO J. 40, e106423 (2021)."), tumor cells[47](https://www.nature.com/articles/s41588-025-02248-5#ref-CR47 "Bieda, K., Hoffman, A. & Boller, K. Phenotypic heterogeneity of human endogenous retrovirus particles produced by teratocarcinoma cell lines. J. Gen. Virol. 82, 591–596 (2001).") or age-related senescence[9](https://www.nature.com/articles/s41588-025-02248-5#ref-CR9 "Liu, X. et al. Resurrection of endogenous retroviruses during aging reinforces senescence. Cell 186, 287–304.e26 (2023)."). It is unclear whether this results from the activation of specific elements or the widespread activation of thousands of them. Here, we report the production of VLPs during organ formation in the mouse embryo as a result of epigenetic de-silencing of a single MusD element. The _Dac1J_ element shows high sequence similarity with the identified MusD ‘master’ copies that are competent and autonomous for retrotransposition[15](https://www.nature.com/articles/s41588-025-02248-5#ref-CR15 "Ribet, D., Dewannieux, M. & Heidmann, T. An active murine transposon family pair: retrotransposition of ‘master’ MusD copies and ETn trans-mobilization. Genome Res. 14, 2261–2267 (2004).") (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02248-5#Fig12)), suggesting that de novo integration of the MusD-_Dac1J_ element could occur, leading to genomic instability. However, detecting new insertion would be very challenging: MusD-_Dac1J_\-expressing cells represent around 4% of all embryonic limb cells (Fig. [2c](https://www.nature.com/articles/s41588-025-02248-5#Fig2)), and they die rapidly after VLP assembly.

The dactylaplasia phenotype is variable and not fully penetrant in mice carrying a heterozygous insertion of MusD-_Dac1J_, but all homozygotes display a severe phenotype, indicating a dose-dependent effect of the VLPs. By contrast, _Dac1J_ mutants carrying mutations in the _pol_ gene or the PBS exhibited a mild and low-penetrant phenotype. This remaining phenotype could be caused by the presence of the Gag capsid, as mutations in the _gag_ gene show a complete rescue of the phenotype. However, we cannot fully exclude that _pol_ genes from other active MusD elements contribute to the phenotype. Moreover, it is possible that inefficient reverse transcription taking place outside of the capsid also participates in the complete rescue of _the Dac1J_ Gag mutant. Overall, this suggests that the main trigger of the apoptotic signaling is the Gag capsid, but it also demonstrates that Gag alone cannot drive complete dactylaplasia phenotypes.

Our findings raise the question of whether VLP-mediated cell death could happen in embryonic cells other than those of the AER. As a result of MusD_\-Dac1J_ expression, a few cells in the branchial arches of the _Dac1J_ embryos undergo cell death. This does not seem to be sufficient to lead to any apparent morphological phenotype, probably because proliferating surrounding tissue can replace the few dying cells. We also showed that MusD-Gag is produced in _Lbx1_\-expressing cells of the developing limb when MusD is inserted in the _Lbx1_ TAD. Cell death was not detected in those cells, which could be explained by cell lineage specificity, genomic loci or dose-dependent effects. MusD-derived VLPs could also be beneficial for the embryo, as shown in humans, whereby blastocyst development proceeds with the presence of HERVK particles[48](https://www.nature.com/articles/s41588-025-02248-5#ref-CR48 "Grow, E. J. et al. Intrinsic retroviral reactivation in human preimplantation embryos and pluripotent cells. Nature 522, 221–246 (2015)."). There are a growing number of studies showing Gag-derived proteins co-option[49](https://www.nature.com/articles/s41588-025-02248-5#ref-CR49 "Pastuzyn, E. D. et al. The neuronal gene arc encodes a repurposed retrotransposon gag protein that mediates intercellular RNA transfer. Cell 172, 275–288.e18 (2018)."),[50](https://www.nature.com/articles/s41588-025-02248-5#ref-CR50 "Segel, M. et al. Mammalian retrovirus-like protein PEG10 packages its own mRNA and can be pseudotyped for mRNA delivery. Science 373, 882–889 (2021)."),[51](https://www.nature.com/articles/s41588-025-02248-5#ref-CR51 "Xu, J. et al. PNMA2 forms immunogenic non-enveloped virus-like capsids associated with paraneoplastic neurological syndrome. Cell 187, 831–845.e19 (2024)."). Whether or not VLP production could also be co-opted to benefit mouse development is yet to be studied, but recent work presents evidence that Gag proteins encoded by ERVs are essential for zebrafish and chicken embryonic development[52](https://www.nature.com/articles/s41588-025-02248-5#ref-CR52 "Chang, N.-C. et al. Gag proteins encoded by endogenous retroviruses are required for zebrafish development. Proc. Natl Acad. Sci. USA 122, e2411446122 (2025).").

Identifying the causative variants in patients with congenital malformations remains challenging. Our data suggest that variable DNA methylation states at a single ERV can result in malformations induced by cell-type-specific and time-specific expression of the ERV in a pattern of the host gene. Unlike most human ERVs, HERVK (HML-2) retained copies with intact open reading frames for retroviral proteins and was shown to assemble into VLPs in the human early embryo[48](https://www.nature.com/articles/s41588-025-02248-5#ref-CR48 "Grow, E. J. et al. Intrinsic retroviral reactivation in human preimplantation embryos and pluripotent cells. Nature 522, 221–246 (2015)."). Long-interspersed nuclear elements are active in humans, and their transcription was shown to be associated with immunogenic effects[53](https://www.nature.com/articles/s41588-025-02248-5#ref-CR53 "De Cecco, M. et al. L1 drives IFN in senescent cells and promotes age-associated inflammation. Nature 566, 73–78 (2019)."). Aberrant activation of HERVK or long-interspersed nuclear elements in developmental cells might potentially be involved in developmental disorders that so far remain poorly understood.

Methods
-------

The experiments were not randomized. The investigators were not blinded to allocation during experiments and outcome assessment. All animal procedures were conducted as approved by the local authorities (LAGeSo Berlin) under license numbers G0243/18 and G0176/19.

### Mice

For embryo isolation, mice were killed by cervical dislocation, and the uteri were dissected in PBS. All animal experiments followed all relevant guidelines and regulations. Mice were housed in a centrally controlled environment with a 12 h light, 12 h dark cycle, temperature of 20–22.2 °C and humidity of 30–50%. Routine bedding, food and water changes were performed. The _Dac1J_ line (SM;NZB background) was obtained from the Jackson Laboratory (strain no. 002759) and backcrossed for at least ten generations with 129s2/Sv and C57BL/6 wild-type mice to obtain the _Dac1J_\-129sv and _Dac1J_\-BL/6 lines.

### Mouse embryonic stem cell targeting

Culture and genome editing of mouse embryonic stem (ES) cells were performed as previously described[27](https://www.nature.com/articles/s41588-025-02248-5#ref-CR27 "Cova, G. et al. Combinatorial effects on gene expression at the Lbx1/Fgf8 locus resolve split-hand/foot malformation type 3. Nat. Commun. 14, 1475 (2023)."),[54](https://www.nature.com/articles/s41588-025-02248-5#ref-CR54 "Kraft, K. et al. Deletions, inversions, duplications: engineering of structural variants using CRISPR/Cas in mice. Cell Rep. 10, 833–839 (2015)."). G4 (XY, 129sv × C57BL/6 F1 hybrid, wild-type) and _Dac1J_\-129s2/Sv (derived in-house from mouse blastocysts) cells were used. In brief, mouse ES cells were seeded on a monolayer of CD1 feeder cells, and transfection was performed using FuGENE technology (Promega, cat. no. E5911). A total of 8 μg of each CRISPR construct containing the single guide RNA (sgRNA) of interest delivered through the pX459 vector and 4 μg of the DNA construct for homology-directed repair was used. After 24 h, cells were split onto DR4 puromycin-resistant feeders, and selection with puromycin was carried out for 2 days. Resistant-growing clones were then picked and grown into 96-well plates on CD1 feeders. All clones were genotyped by PCR (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02248-5#MOESM4)). Positive clones were expanded and their genotyping was further confirmed by Sanger sequencing of PCR products. They were used for further experiments only if the successful modification could be verified. A list of sgRNAs (designed using the Benchling CRISPR guides tool) used for CRISPR–Cas9 genome editing is given in Supplementary Table [4](https://www.nature.com/articles/s41588-025-02248-5#MOESM4). One of the sgRNAs is located in the repetitive MusD pol region; Supplementary Table [5](https://www.nature.com/articles/s41588-025-02248-5#MOESM4) shows all off-targets for this sgRNA. All knock-in DNA vectors used for transfection were cloned using Gibson cloning to assemble the knock-in DNA and the homology arms targeting the _Fgf8_, _Lbx1_, _Shh_ or _Sox9_ loci. The mm10 coordinates of the homology arms used in the knock-in constructs can be found in Supplementary Table [6](https://www.nature.com/articles/s41588-025-02248-5#MOESM4). The β-globin-LacZ construct contains the minimal β-globin promoter (57 bp) upstream of the LacZ gene (3,452 bp). The _Dac1J_ mut-PBS sequence (7,496 bp) was synthesized and subsequently cloned into a pUC vector by Genewiz. _Dac1J_ mut-pol, _Dac1J_ mut-gag and _Dac1J_ mut-gag(CA) were generated by Gibson cloning. The 5LTR-LacZ constructs contain the MusD-_Dac1J_ 5′LTR (318 bp) and 5′UTR (331 bp, containing the same insertion in the Lys-tRNA PBS as in _Dac1J_\-mutPBS) and the LacZ gene (3,452 bp).

### Generation of mutant mice

Mice were generated from genome-edited mouse ES cells by diploid or tetraploid aggregation[55](https://www.nature.com/articles/s41588-025-02248-5#ref-CR55 "Artus, J. & Hadjantonakis, A.-K. Generation of chimeras by aggregation of embryonic stem cells with diploid or tetraploid mouse embryos. Methods Mol. Biol. 693, 37–56 (2011)."). Confirmed CRISPR–Cas9 mutant cell lines after expansion were seeded on CD1 feeders and grown for 2 days before aggregation. Female mice of the CD1 strain were used as foster mothers. The data collection was performed according to the stage of each sample, and investigators were not blind to genotype, given that mouse breeding and analysis required knowledge about the genotype at hand. For Δ_Dac1J_, Δ_Dac1J_\-Pol (both from _Dac1J_\-129sv mouse ES cells), β-globin-LacZ KI at _Fgf8_, 5LTR-LacZ KI at _Fgf8_, 5LTR-LacZ KI at _Lbx1_, 5LTR-LacZ KI at _Shh_, 5LTR-LacZ KI at _Sox9_, _Dac1J_ mut-gag, _Dac1J_ mut-gag(CA) and _Dac1J_\-KI at _Lbx1_ (all from G4 mouse ES cells), only embryos from tetraploid aggregation were generated for analysis at different embryonic stages. For _Dac1J_\-mutPBS KI at _Fgf8_, _Dac1J_\-mutPBS KI at _Lbx1_ and _Dac1J_\-mut-pol KI at _Fgf8_ (all from G4 mouse ES cells), adult mice were generated by diploid aggregation. Founder animals for each mouse line were backcrossed with 129sv wild-type animals to establish line stock. The selection of animals for analysis and breeding was random. Both males and females were used for analysis. For all MusD-_Dac1J_ knock-in mice, the CpG DNA methylation status of the knocked-in 5LTR was assessed by bisulfite sequencing. Mice were used for the study only if the knocked-in 5LTR was hypomethylated. If the 5LTR of the _Dac1J_ was methylated at more than 20%, animals were excluded from the study. The rationale for this exclusion is that a knock-in with hypermethylated _Dac1J_\-5LTR will silence the transcript, preventing a conclusion on the effect of the mutated retroviral product.

### Skeletal preparation

E18.5 fetuses were killed and stored at −20 °C. On the first day of the staining protocol, E18.5 fetuses were kept in H2O for 1–2 h at room temperature (20–25 °C) and heat-shocked at 65 °C for 1 min. The skin, abdominal and thoracic viscera were gently removed using forceps. The fetuses were then fixed in 100% ethanol for at least 24 h. Alcian blue staining solution (150 mg alcian blue 8GX in 80% ethanol and 20% acetic acid) was used to stain the cartilage. After 24 h, fetuses were rinsed and post-fixed in 100% ethanol overnight, followed by 24 h incubation in 0.2% KOH in H2O for initial clearing. The next day, fetuses were incubated in alizarin red (50 mg l−1 alizarin red S in 0.2% KOH) to stain the bones for 24 h. Rinsing and clearing were then carried out for 1–3 days using 0.2% KOH. The stained embryos were dissected in 25% glycerol, imaged using a Zeiss SteREO Discovery V12 microscope and Leica DFC420 digital camera and subsequently stored in 80% glycerol.

### β-galactosidase staining

Embryos were dissected in ice-cold PBS and fixed in 4% paraformaldehyde (PFA) in PBS for 20 min (E11.5 or younger) or 30 min (E12.5) and then washed several times in PBS. Embryos were subsequently washed two times for 15 min in wash solution (2 mM MgCl2, 0.02% NP-40 and 0.01% C24H39NaO4 in PBS) and finally incubated at 37 °C in X-gal solution (165 mg ml−1 K4Fe(CN)63H20, 210 mg ml−1 K3Fe(CN)6 and 40 mg ml−1 X-gal diluted 1:50 in wash solution). Staining was assessed regularly, and the reaction was stopped when LacZ staining was clearly apparent (between 1 h and 12 h of staining). Samples were then washed several times with PBS and kept at 4 °C in PBS with a few drops of 4% PFA. Embryos were imaged using a Zeiss SteREO Discovery V12 microscope and a Leica DFC420 digital camera.

### DNA methylation analysis

Genomic DNA isolated from embryonic tissue was obtained following overnight lysis at 50 °C (100 mM Tris pH 8.0, 5 mM EDTA, 200 mM NaCl, 0.2% SDS and proteinase K). DNA was recovered by standard phenol–chloroform–isoamyl alcohol extraction and resuspended in water. Bisulfite conversion was performed on 1 µg of DNA using the EpiTect Bisulfite kit (Qiagen). Bisulfite-treated DNA was PCR-amplified (nested PCR), then either cloned and sequenced or analyzed by pyrosequencing. For cloning sequencing, at least eight clones were Sanger-sequenced and analyzed with BiQ Analyzer software[56](https://www.nature.com/articles/s41588-025-02248-5#ref-CR56 "Bock, C. et al. BiQ Analyzer: visualization and quality control for DNA methylation data from bisulfite sequencing. Bioinformatics 21, 4067–4068 (2005).") to analyze all 19 CpGs within the _Dac1J_ 5′LTR region. Pyrosequencing, targeting CpG1–8, was performed on the PyroMark Q24 according to the manufacturer’s instructions, and results were analyzed with the associated software. All bisulfite primers are listed in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02248-5#MOESM4).

### RNA-seq

Early E11.5 forelimb buds were microdissected from wild-type and mutant embryos in cold PBS and immediately snap-frozen for storage at −80 °C. Total RNA was extracted using the RNeasy Mini Kit according to the manufacturer’s instructions. Samples were poly-A enriched, prepared into libraries using the Kapa HyperPrep Kit and sequenced on a NovaSeq 6000 with 75 bp or 100 bp paired-end reads. RNA-seq experiments were performed in triplicate. For processing, see [Supplementary Methods](https://www.nature.com/articles/s41588-025-02248-5#MOESM1).

### scRNA-seq

scRNA-seq experiments were performed in single replicates (except for the E9.5 _Dac1J_/_Dac1J_ samples for which we had two replicates) as previously described[27](https://www.nature.com/articles/s41588-025-02248-5#ref-CR27 "Cova, G. et al. Combinatorial effects on gene expression at the Lbx1/Fgf8 locus resolve split-hand/foot malformation type 3. Nat. Commun. 14, 1475 (2023)."). In brief, E9.5, E10.5 and E11.5 limb buds of wild-type and _Dac1J_/_Dac1J_ embryos were microdissected in ice-cold PBS. A single-cell suspension was obtained by incubating the tissue for 10 min at 37 °C in 200 μl Gibco trypsin-EDTA 0.05% (Thermo Fisher Scientific, cat. no. 25300054) supplemented with 20 μl 5% BSA. Trypsinization was then stopped by adding 400 μl of 5% BSA. Cells were then resuspended by pipetting, filtered using a 0.40 μm filter, washed once with 0.04% BSA, centrifuged (5 min at 150_g_) and resuspended in 0.04% BSA. The cell count was determined using an automated cell counter (Bio-Rad), and cells were subjected to scRNA-seq (10× Genomics, Chromium Single Cell 3′ v2), aiming for a target cell recovery of up to 10,000 sequenced cells per sequencing library. Single-cell libraries were generated according to the 10× Genomics instructions. Libraries were sequenced with a minimum of 230 million 75 bp paired-end reads according to standard protocols. For processing, see [Supplementary Methods](https://www.nature.com/articles/s41588-025-02248-5#MOESM1).

### cHi-C

The cHi-C protocol was performed as previously described[27](https://www.nature.com/articles/s41588-025-02248-5#ref-CR27 "Cova, G. et al. Combinatorial effects on gene expression at the Lbx1/Fgf8 locus resolve split-hand/foot malformation type 3. Nat. Commun. 14, 1475 (2023)."). In brief, E11.5 mouse limb buds were prepared in 1× PBS and dissociated with trypsin treatment. A total of 2.5–5 × 106 nuclei were used for crosslinking, then snap-frozen and stored at −80 °C. Snap-frozen pellets were digested with DpnII, ligated and de-crosslinked. The final library was checked on agarose gel and subsequently used for cHi-C preparation. Libraries were sheared using a Covaris sonicator (duty cycle, 10%; intensity, 5; cycles per burst, 200; time, six cycles of 60 s each; set mode, frequency sweeping; temperature, 4–7 °C). Adaptors were added to the sheared DNA and amplified according to Agilent instructions for Illumina sequencing. The library was hybridized to the custom-designed SureSelect beads and indexed for sequencing following Agilent instructions. SureSelect enrichment probes were designed over the genomic interval chr19:44,365,510–46,325,510 (mm10) using the Agilent SureDesign online tool ([https://earray.chem.agilent.com/suredesign](https://earray.chem.agilent.com/suredesign)). The coordinates for generating cHi-C matrices were: chr19:44,370,000–46,330,000 (_Dac1J_\_mm10 custom genome) and chr19:44,370,000–46,320,000 (5′LTRLacZ\_mm10 custom genome). NovaSeq 6000 Illumina technology was used according to the standard protocols, with around 400 million 75 bp or 100 bp (NovaSeq 6000) paired-end reads per sample. For processing, see [Supplementary Methods](https://www.nature.com/articles/s41588-025-02248-5#MOESM1).

### 4C-seq

Mouse limb micro-dissection, cell-dissociation, crosslinking and nuclei extraction were performed as described above for cHi-C. The 4C library preparation was performed as previously described[57](https://www.nature.com/articles/s41588-025-02248-5#ref-CR57 "Franke, M. et al. Duplications disrupt chromatin architecture and rewire GPR101-enhancer communication in X-linked acrogigantism. Am. J. Hum. Genet. 109, 553–570 (2022)."), with modifications described below. A total of two to five million cells were used as starting material for all 4C-seq libraries. Experiments for the MusD-_Dac1J_ viewpoint in _Dac1J_/_Dac1J_ (C57BL/6) and _Dac1J_/_Dac1J_ (129sv/S2) samples (_n_ = 3) were performed as singletons. All 4C-seq primer sequences are listed in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02248-5#MOESM4). DpnII and BfaI were used as primary and secondary restriction enzymes, respectively. Parallel inverse PCR reactions were performed to amplify from a total of 1.6 µg template per 4C library and viewpoint. Final 4C-seq libraries were indexed using TruSeq index primers for Illumina (NEB, E7335S) and NEBNext High-Fidelity 2× PCR Master Mix (NEB). Samples were sequenced with Illumina Hi-Seq technology according to standard protocols. For processing, see [Supplementary Methods](https://www.nature.com/articles/s41588-025-02248-5#MOESM1).

### ChIP–seq

#### Tissue collection and cell fixation

Embryonic limbs (32 for E10.5 and 15–20 for E11.5) were microdissected in ice-cold 1× PBS and pooled. Limbs were washed once with PBS solution and homogenized in 2 ml trypsin solution for 10 min at 37 °C. Trypsin was stopped by adding 5 ml of a 10% FCS–PBS solution. Then, samples were filtered through a 40 μm cell strainer and complemented with 10% FCS–PBS solution. Formaldehyde (37%, diluted to a final 1%) was used to fix the samples for 10 min at room temperature; 1.425 M glycine was used to quench the fixation. The formaldehyde solution was removed by centrifugation (400_g_, 8 min), cells were washed in 10% FCS–PBS solution and centrifuged; the pellet was snap-frozen in liquid N2 and stored at −80 °C.

#### Chromatin shearing and ChIP–seq

Chromatin immunoprecipitation was performed using the iDeal ChIP–seq Kit for Transcription Factors (Diagenode, C01010055) according to the manufacturer’s instructions. In brief, fixed limbs were lysed and chromatin was sonicated in a size range of 200–500 bp using a Bioruptor Plus Sonication device (40 cycles, 30 s on, 30 s off, at high power setting) in the provided buffers. A total of 20 μg of sheared chromatin was used for CTCF immunoprecipitation with 1 μg of antibody (Diagenode, C15410210). Libraries were prepared using the Kapa HyperPrep Kit (Roche, 07962347001) and sequenced on a NovaSeq2 (E10.5) or Aviti (E11.5) sequencers. For processing, see [Supplementary Methods](https://www.nature.com/articles/s41588-025-02248-5#MOESM1).

### Whole-mount immunofluorescence

Embryos were collected in ice-cold PBS and fixed in 4% PFA for 1 h at 4 °C, washed three times in PBS and stored at 4 °C in PBS + 0.03% Na Azide until staining was performed. On the first day of the protocol, embryos were washed once with PBS for 5 min, permeabilized three times 20 min in 0.5% Triton-X–PBS (PBST) and blocked in 5% horse serum–PBST (blocking solution) overnight at 4 °C. Primary antibody incubation was performed in the blocking solution for 72 h at 4 °C under gentle rotation. Embryos were then washed three times in blocking solution and three times in PBST and incubated overnight in blocking solution at 4 °C with gentle rotation. The next day, the secondary antibody, diluted in blocking solution, was added, and embryos were incubated for 48 h at 4 °C with gentle rotation. Afterward, embryos were again washed three times in blocking solution and three times in PBST and incubated overnight in DAPI diluted in PBS (1:5,000; Sigma-Aldrich, D9542) at 4 °C with gentle rotation. On the last day, embryos were washed three times for 10 min each in PBS and post-fixed in 4% PFA at room temperature for 20 min. Before clearing, embryos were washed three times with 0.02 M phosphate buffer (0.025 M NaH2PO4 and 0.075 M Na2HPO4, pH 7.4). The clearing was performed by incubation in RIMS (13% Histodenz (Sigma-Aldrich D2158) in 0.02 M phosphate buffer) at 4 °C for at least 1 day. Whole-mount embryos were then imaged with a Zeiss LSM880 confocal laser-scanning microscope in Fast with Airyscan mode. The rabbit MusD-Gag antiserum was a gift from T. Heidemann’s laboratory, as previously generated[34](https://www.nature.com/articles/s41588-025-02248-5#ref-CR34 "Ribet, D., Harper, F., Dewannieux, M., Pierron, G. & Heidmann, T. Murine MusD retrotransposon: structure and molecular evolution of an ‘intracellularized’ retrovirus. J. Virol. 81, 1888–1898 (2007)."). Anti-MusD-Gag (1:1,000) was used with goat anti-rabbit Alexa-fluorophore 488 (1:1,000; Invitrogen, A11008). Anti-cleaved-Caspase-3 (rabbit polyclonal, Cell Signaling Technology, 9661, Asp175, 1:400) and anti-γH2AX (rabbit monoclonal, Cell Signaling, mAb 9718) were used with a donkey anti-rabbit Alexa-fluorophore 568 (1:1,000; Invitrogen, A110042).

### TEM

E11.0 embryos were collected in ice-cold PBS and fixed in 2% PFA and 2.5% glutaraldehyde in PBS for 8 h with rotation at 4 °C and then kept in PBS at 4 °C overnight. After washing three times in PBS, limbs were dissected and post-fixed in 0.5 % (w/v) osmium tetroxide (Science Services) in PBS for 1 h at room temperature and then washed four times in PBS for 20 min. Samples were incubated for 30 min in 100 mM HEPES buffer pH 7.4 containing 0.1% (w/v) tannic acid (Science Services), washed three times for 10 min in distilled water, contrasted in 2% (w/v) uranyl acetate (Sigma-Aldrich, Merck) for 1.5 h at room temperature and washed once in distilled water, followed by dehydration through a series of increasing ethanol concentrations (30% for 5 min, 50% for 10 min, 70%, 90% and 96% for 15 min each and finally three times for 10 min in 100% ethanol, respectively). Samples were then incubated for 5 min in a 1:1 mixture of propylene oxide (Sigma-Aldrich, Merck) and absolute ethanol, followed by 5 min in a 1:1 mixture of propylene oxide (Fluka, Merck) and SPURR (Low Viscosity Spurr Kit, Ted Pella) and finally infiltrated with 100 % SPURR at 4 °C overnight. The SPURR mixture was renewed twice the following day. For polymerization, limbs were transferred into flat embedding molds, surrounded with fresh resin and incubated for 3 days at 60 °C. Then, 70 nm sections were cut using a Leica UC7 ultramicrotome equipped with a 3 mm diamond knife (Diatome) and placed on 3.05 mm Formvar Carbon Coated TEM Copper Slot Grids (Plano). Sections were post-contrasted using Uranyless EM-Stain and ready-to-use 3% Lead Citrate (Science Services). To visualize retrovirus-like particles in the apical ectodermal ridge, sections were imaged automatically using Leginon[58](https://www.nature.com/articles/s41588-025-02248-5#ref-CR58 "Suloway, C. et al. Automated molecular microscopy: the new Leginon system. J. Struct. Biol. 151, 41–60 (2005).") on a Tecnai Spirit transmission electron microscope (FEI) operated at 120 kV, equipped with a 4k × 4k F416 CMOS camera (TVIPS). For full-limb pictures, acquired images (up to 657 micrographs per section through one limb bud) were then stitched to a single montage using the TrakEM2 plugin implemented in Fiji[59](https://www.nature.com/articles/s41588-025-02248-5#ref-CR59 "Cardona, A. et al. TrakEM2 software for neural circuit reconstruction. PLoS ONE 7, e38011 (2012)."),[60](https://www.nature.com/articles/s41588-025-02248-5#ref-CR60 "Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012).").

### Immuno-TEM

Embryos at E11.0 were collected in ice-cold PBS and fixed in 2% PFA and 0.2% glutaraldehyde in PBS for 1 h at 4 °C and washed three times in PBS. Limbs were dissected and stained for 30 min in 100 mM HEPES buffer pH 7.4 containing 0.1 % (w/v) tannic acid (Science Services), washed three times for 10 min in PBS and dehydrated in a series of increasing ethanol concentrations (30% for 5 min, 50% for 5 min, 70%, 90% and 96% for 10 min each and finally three times 10 min in 100% ethanol, respectively). Samples were then incubated for 30 min in a 1:1 mixture of 100% ethanol and London Resin Gold (Plano) and finally infiltrated with 100% LR Gold at 4 °C overnight. The following day, a fresh mixture of LR Gold was prepared by adding the accelerator Benzil (Plano) at a concentration of 0.2%. Specimens were incubated for two times for 2 h at room temperature and kept overnight at 4 °C, remaining in this mixture. Before polymerization, a fresh LR Gold solution containing 0.2% benzil was renewed for 4 h on the limbs 1 day later. Limbs were then transferred into flat embedding molds, surrounded with an LR Gold–benzil mixture and polymerized under a 100 W Black-Ray UV Lamp (Plano) for 2 days at 5 °C. Then, 80 nm sections were cut using a Leica UC7 ultramicrotome equipped with a 3 mm diamond knife (Diatome) and placed on 3.05 mm Formvar Carbon Coated TEM Copper Slot Grids (Plano). Grids were protected against dehydration and nonspecific binding by applying the grid on a drop of Aurion blocking solution for goat gold conjugates (Aurion) for 30 min at room temperature. Slices were then incubated with 10 µl of a 1:500 dilution of the rabbit anti-Gag-MusD primary antibody in Aurion blocking solution overnight at 4 °C in a humid chamber. The following day, grids were washed four times in buffer containing 20 mM TRIS and 0.9% NaCl. As secondary antibodies, goat anti-rabbit 10 nm gold conjugates (British BioCell) were used at 0.08 µg ml−1 and incubated for 2 h at room temperature. The immune reaction was stopped by washing four times with TRIS–NaCl buffer. After a short dip into double-distilled water, grids were dried and kept for subsequent examination on a Tecnai Spirit transmission electron microscope (FEI). To visualize the localization of gold conjugates, limb bud sections were imaged at ×6,500 nominal magnification, applying a defocus of −2 µm with a pixel size of 1.7 nm.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02248-5#MOESM2) linked to this article.

Data availability
-----------------

The data generated in this study can be downloaded in raw and processed forms from the National Center for Biotechnology Information Gene Expression Omnibus (GEO) database and are available under accession code [GSE246755](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246755) (containing SubSeries [GSE246750](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246750), [GSE246751](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246751), [GSE246752](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246752), [GSE246753](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246753) and [GSE246754](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246754)). Previously published data used in this study are accessible under [GSE185774](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE185774), [GSE116794](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116794) and [GSE84795](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84795). Data were mapped to the _Mus musculus_ mm10 genome.

Code availability
-----------------

Newly generated code for the cHi-C data processing is available on GitHub ([https://github.com/mikstapes/JGlaser-etal\_Dac1J/](https://github.com/mikstapes/JGlaser-etal_Dac1J/)) and Zenodo ([https://doi.org/10.5281/zenodo.15528362](https://doi.org/10.5281/zenodo.15528362))[61](https://www.nature.com/articles/s41588-025-02248-5#ref-CR61 "Mikie Phan. mikstapes/JGlaser-etal_Dac1J: first release (v1.0.0). Zenodo 
https://doi.org/10.5281/zenodo.15528362
(2025)."). All other analyses were performed using previously published or developed tools, as indicated in the [Methods](https://www.nature.com/articles/s41588-025-02248-5#Sec14) section.

References
----------

1.  Wells, J. N. & Feschotte, C. A field guide to eukaryotic transposable elements. _Annu. Rev. Genet._ **54**, 539–561 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVOmur3N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32955944)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8293684)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20field%20guide%20to%20eukaryotic%20transposable%20elements&journal=Annu.%20Rev.%20Genet.&volume=54&pages=539-561&publication_year=2020&author=Wells%2CJN&author=Feschotte%2CC) 
    
2.  Payer, L. M. & Burns, K. H. Transposable elements in human genetic disease. _Nat. Rev. Genet._ **20**, 760–772 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslKhsrzN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31515540)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transposable%20elements%20in%20human%20genetic%20disease&journal=Nat.%20Rev.%20Genet.&volume=20&pages=760-772&publication_year=2019&author=Payer%2CLM&author=Burns%2CKH) 
    
3.  Deniz, Ö., Frost, J. M. & Branco, M. R. Regulation of transposable elements by DNA modifications. _Nat. Rev. Genet._ **20**, 417–431 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmslKhurk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30867571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulation%20of%20transposable%20elements%20by%20DNA%20modifications&journal=Nat.%20Rev.%20Genet.&volume=20&pages=417-431&publication_year=2019&author=Deniz%2C%C3%96&author=Frost%2CJM&author=Branco%2CMR) 
    
4.  Fadloun, A. et al. Chromatin signatures and retrotransposon profiling in mouse embryos reveal regulation of LINE-1 by RNA. _Nat. Struct. Mol. Biol._ **20**, 332–338 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsV2hsrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23353788)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20signatures%20and%20retrotransposon%20profiling%20in%20mouse%20embryos%20reveal%20regulation%20of%20LINE-1%20by%20RNA&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=20&pages=332-338&publication_year=2013&author=Fadloun%2CA) 
    
5.  Jachowicz, J. W. et al. LINE-1 activation after fertilization regulates global chromatin accessibility in the early mouse embryo. _Nat. Genet._ **49**, 1502–1510 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtl2gtL%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28846101)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LINE-1%20activation%20after%20fertilization%20regulates%20global%20chromatin%20accessibility%20in%20the%20early%20mouse%20embryo&journal=Nat.%20Genet.&volume=49&pages=1502-1510&publication_year=2017&author=Jachowicz%2CJW) 
    
6.  Sakashita, A. et al. Transcription of MERVL retrotransposons is required for preimplantation embryo development. _Nat. Genet._ **55**, 484–495 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXkt1yntL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36864102)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10011141)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20of%20MERVL%20retrotransposons%20is%20required%20for%20preimplantation%20embryo%20development&journal=Nat.%20Genet.&volume=55&pages=484-495&publication_year=2023&author=Sakashita%2CA) 
    
7.  Tam, O. H., Ostrow, L. W. & Gale Hammell, M. Diseases of the nERVous system: retrotransposon activity in neurodegenerative disease. _Mob. DNA_ **10**, 32 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31372185)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6659213)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Diseases%20of%20the%20nERVous%20system%3A%20retrotransposon%20activity%20in%20neurodegenerative%20disease&journal=Mob.%20DNA&volume=10&publication_year=2019&author=Tam%2COH&author=Ostrow%2CLW&author=Gale%20Hammell%2CM) 
    
8.  Gorbunova, V. et al. The role of retrotransposable elements in ageing and age-associated diseases. _Nature_ **596**, 43–53 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslWmsrbP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34349292)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8600649)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20retrotransposable%20elements%20in%20ageing%20and%20age-associated%20diseases&journal=Nature&volume=596&pages=43-53&publication_year=2021&author=Gorbunova%2CV) 
    
9.  Liu, X. et al. Resurrection of endogenous retroviruses during aging reinforces senescence. _Cell_ **186**, 287–304.e26 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXlvFersQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36610399)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resurrection%20of%20endogenous%20retroviruses%20during%20aging%20reinforces%20senescence&journal=Cell&volume=186&pages=287-304.e26&publication_year=2023&author=Liu%2CX) 
    
10.  Shah, N. M. et al. Pan-cancer analysis identifies tumor-specific antigens derived from transposable elements. _Nat. Genet._ **55**, 631–639 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmt1Wqurs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36973455)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-cancer%20analysis%20identifies%20tumor-specific%20antigens%20derived%20from%20transposable%20elements&journal=Nat.%20Genet.&volume=55&pages=631-639&publication_year=2023&author=Shah%2CNM) 
    
11.  Andrey, G. & Mundlos, S. The three-dimensional genome: regulating gene expression during pluripotency and development. _Development_ **144**, 3646–3658 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXpvVGisbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29042476)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20genome%3A%20regulating%20gene%20expression%20during%20pluripotency%20and%20development&journal=Development&volume=144&pages=3646-3658&publication_year=2017&author=Andrey%2CG&author=Mundlos%2CS) 
    
12.  Fueyo, R., Judd, J., Feschotte, C. & Wysocka, J. Roles of transposable elements in the regulation of mammalian transcription. _Nat. Rev. Mol. Cell Biol._ **23**, 481–497 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XlsVGitbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35228718)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10470143)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Roles%20of%20transposable%20elements%20in%20the%20regulation%20of%20mammalian%20transcription&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=23&pages=481-497&publication_year=2022&author=Fueyo%2CR&author=Judd%2CJ&author=Feschotte%2CC&author=Wysocka%2CJ) 
    
13.  Gagnier, L., Belancio, V. P. & Mager, D. L. Mouse germ line mutations due to retrotransposon insertions. _Mob. DNA_ **10**, 15 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31011371)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6466679)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mouse%20germ%20line%20mutations%20due%20to%20retrotransposon%20insertions&journal=Mob.%20DNA&volume=10&publication_year=2019&author=Gagnier%2CL&author=Belancio%2CVP&author=Mager%2CDL) 
    
14.  Mager, D. L. & Freeman, J. D. Novel mouse type D endogenous proviruses and ETn elements share long terminal repeat and internal sequences. _J. Virol._ **74**, 7221–7229 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlt1Gku7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10906176)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC112243)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Novel%20mouse%20type%20D%20endogenous%20proviruses%20and%20ETn%20elements%20share%20long%20terminal%20repeat%20and%20internal%20sequences&journal=J.%20Virol.&volume=74&pages=7221-7229&publication_year=2000&author=Mager%2CDL&author=Freeman%2CJD) 
    
15.  Ribet, D., Dewannieux, M. & Heidmann, T. An active murine transposon family pair: retrotransposition of ‘master’ MusD copies and ETn trans-mobilization. _Genome Res._ **14**, 2261–2267 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXpvVCitbY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15479948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC525684)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20active%20murine%20transposon%20family%20pair%3A%20retrotransposition%20of%20%E2%80%98master%E2%80%99%20MusD%20copies%20and%20ETn%20trans-mobilization&journal=Genome%20Res.&volume=14&pages=2261-2267&publication_year=2004&author=Ribet%2CD&author=Dewannieux%2CM&author=Heidmann%2CT) 
    
16.  Zhang, Y., Maksakova, I.A., Gagnier, L., van de Lagemaat, L. N. & Mager, D. L. Genome-wide assessments reveal extremely high levels of polymorphism of two active families of mouse endogenous retroviral elements. _PLoS Genet._ **4**, e1000007 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18454193)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2265474)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20assessments%20reveal%20extremely%20high%20levels%20of%20polymorphism%20of%20two%20active%20families%20of%20mouse%20endogenous%20retroviral%20elements&journal=PLoS%20Genet.&volume=4&publication_year=2008&author=Zhang%2CY&author=Maksakova%2CIA&author=Gagnier%2CL&author=van%20de%20Lagemaat%2CLN&author=Mager%2CDL) 
    
17.  Nellåker, C. et al. The genomic landscape shaped by selection on transposable elements across 18 mouse strains. _Genome Biol._ **13**, R45 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22703977)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3446317)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genomic%20landscape%20shaped%20by%20selection%20on%20transposable%20elements%20across%2018%20mouse%20strains&journal=Genome%20Biol.&volume=13&publication_year=2012&author=Nell%C3%A5ker%2CC) 
    
18.  Brûlet, P., Condamine, H. & Jacob, F. Spatial distribution of transcripts of the long repeated ETn sequence during early mouse embryogenesis. _Proc. Natl Acad. Sci. USA_ **82**, 2054–2058 (1985).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2580305)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC397490)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatial%20distribution%20of%20transcripts%20of%20the%20long%20repeated%20ETn%20sequence%20during%20early%20mouse%20embryogenesis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=82&pages=2054-2058&publication_year=1985&author=Br%C3%BBlet%2CP&author=Condamine%2CH&author=Jacob%2CF) 
    
19.  Loebel, D. A. F., Tsoi, B., Wong, N., O’Rourke, M. P. & Tam, P. P. L. Restricted expression of ETn-related sequences during post-implantation mouse development. _Gene Expr. Patterns_ **4**, 467–471 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXks12lu7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15183314)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Restricted%20expression%20of%20ETn-related%20sequences%20during%20post-implantation%20mouse%20development&journal=Gene%20Expr.%20Patterns&volume=4&pages=467-471&publication_year=2004&author=Loebel%2CDAF&author=Tsoi%2CB&author=Wong%2CN&author=O%E2%80%99Rourke%2CMP&author=Tam%2CPPL) 
    
20.  Chai, C. K. Dactylaplasia in mice. _J. Hered._ **72**, 234–237 (1981).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL38%2FjtFOjtw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7288137)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dactylaplasia%20in%20mice&journal=J.%20Hered.&volume=72&pages=234-237&publication_year=1981&author=Chai%2CCK) 
    
21.  Johnson, K. R., Lane, P. W., Ward-Bailey, P. & Davisson, M. T. Mapping the mouse dactylaplasia mutation, _Dac_, and a gene that controls its expression, _mdac_. _Genomics_ **29**, 457–464 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXovVSlsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8666395)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20the%20mouse%20dactylaplasia%20mutation%2C%20Dac%2C%20and%20a%20gene%20that%20controls%20its%20expression%2C%20mdac&journal=Genomics&volume=29&pages=457-464&publication_year=1995&author=Johnson%2CKR&author=Lane%2CPW&author=Ward-Bailey%2CP&author=Davisson%2CMT) 
    
22.  Kano, H., Kurahashi, H. & Toda, T. Genetically regulated epigenetic transcriptional activation of retrotransposon insertion confers mouse dactylaplasia phenotype. _Proc. Natl Acad. Sci. USA_ **104**, 19034–19039 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhsVSru7bL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17984064)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2141903)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetically%20regulated%20epigenetic%20transcriptional%20activation%20of%20retrotransposon%20insertion%20confers%20mouse%20dactylaplasia%20phenotype&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=104&pages=19034-19039&publication_year=2007&author=Kano%2CH&author=Kurahashi%2CH&author=Toda%2CT) 
    
23.  Andrey, G. et al. Characterization of hundreds of regulatory landscapes in developing limbs reveals two regimes of chromatin folding. _Genome Res._ **27**, 223–233 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXpsF2it7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27923844)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5287228)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterization%20of%20hundreds%20of%20regulatory%20landscapes%20in%20developing%20limbs%20reveals%20two%20regimes%20of%20chromatin%20folding&journal=Genome%20Res.&volume=27&pages=223-233&publication_year=2017&author=Andrey%2CG) 
    
24.  Kraft, K. et al. Serial genomic inversions induce tissue-specific architectural stripes, gene misexpression and congenital malformations. _Nat. Cell Biol._ **21**, 305–310 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmsVymt7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30742094)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Serial%20genomic%20inversions%20induce%20tissue-specific%20architectural%20stripes%2C%20gene%20misexpression%20and%20congenital%20malformations&journal=Nat.%20Cell%20Biol.&volume=21&pages=305-310&publication_year=2019&author=Kraft%2CK) 
    
25.  Ringel, A. R. et al. Repression and 3D-restructuring resolves regulatory conflicts in evolutionarily rearranged genomes. _Cell_ **185**, 3689–3704.e21 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFejs7bJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36179666)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9567273)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Repression%20and%203D-restructuring%20resolves%20regulatory%20conflicts%20in%20evolutionarily%20rearranged%20genomes&journal=Cell&volume=185&pages=3689-3704.e21&publication_year=2022&author=Ringel%2CAR) 
    
26.  Crossley, P. H. & Martin, G. R. The mouse _Fgf8_ gene encodes a family of polypeptides and is expressed in regions that direct outgrowth and patterning in the developing embryo. _Development_ **121**, 439–451 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXjsl2nt7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7768185)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mouse%20Fgf8%20gene%20encodes%20a%20family%20of%20polypeptides%20and%20is%20expressed%20in%20regions%20that%20direct%20outgrowth%20and%20patterning%20in%20the%20developing%20embryo&journal=Development&volume=121&pages=439-451&publication_year=1995&author=Crossley%2CPH&author=Martin%2CGR) 
    
27.  Cova, G. et al. Combinatorial effects on gene expression at the _Lbx1_/_Fgf8_ locus resolve split-hand/foot malformation type 3. _Nat. Commun._ **14**, 1475 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXls1yjs7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36928426)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10020157)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combinatorial%20effects%20on%20gene%20expression%20at%20the%20Lbx1%2FFgf8%20locus%20resolve%20split-hand%2Ffoot%20malformation%20type%203&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Cova%2CG) 
    
28.  Nora, E. P. et al. Spatial partitioning of the regulatory landscape of the X-inactivation centre. _Nature_ **485**, 381–385 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XlslSisrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22495304)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3555144)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatial%20partitioning%20of%20the%20regulatory%20landscape%20of%20the%20X-inactivation%20centre&journal=Nature&volume=485&pages=381-385&publication_year=2012&author=Nora%2CEP) 
    
29.  Dixon, J. R. et al. Topological domains in mammalian genomes identified by analysis of chromatin interactions. _Nature_ **485**, 376–380 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XlslShurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22495300)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3356448)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Topological%20domains%20in%20mammalian%20genomes%20identified%20by%20analysis%20of%20chromatin%20interactions&journal=Nature&volume=485&pages=376-380&publication_year=2012&author=Dixon%2CJR) 
    
30.  Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. _Cell_ **159**, 1665–1680 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtrrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25497547)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%203D%20map%20of%20the%20human%20genome%20at%20kilobase%20resolution%20reveals%20principles%20of%20chromatin%20looping&journal=Cell&volume=159&pages=1665-1680&publication_year=2014&author=Rao%2CSSP) 
    
31.  Marinić, M., Aktas, T., Ruf, S. & Spitz, F. An integrated holo-enhancer unit defines tissue and gene specificity of the _Fgf8_ regulatory landscape. _Dev. Cell_ **24**, 530–542 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23453598)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20holo-enhancer%20unit%20defines%20tissue%20and%20gene%20specificity%20of%20the%20Fgf8%20regulatory%20landscape&journal=Dev.%20Cell&volume=24&pages=530-542&publication_year=2013&author=Marini%C4%87%2CM&author=Aktas%2CT&author=Ruf%2CS&author=Spitz%2CF) 
    
32.  Zhang, Y. et al. Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells. _Nat. Genet._ **51**, 1380–1388 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1WjurjN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31427791)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6722002)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptionally%20active%20HERV-H%20retrotransposons%20demarcate%20topologically%20associating%20domains%20in%20human%20pluripotent%20stem%20cells&journal=Nat.%20Genet.&volume=51&pages=1380-1388&publication_year=2019&author=Zhang%2CY) 
    
33.  Lawson, H. A., Liang, Y. & Wang, T. Transposable elements in mammalian chromatin organization. _Nat. Rev. Genet._ **24**, 712–723 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtFOksr7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37286742)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transposable%20elements%20in%20mammalian%20chromatin%20organization&journal=Nat.%20Rev.%20Genet.&volume=24&pages=712-723&publication_year=2023&author=Lawson%2CHA&author=Liang%2CY&author=Wang%2CT) 
    
34.  Ribet, D., Harper, F., Dewannieux, M., Pierron, G. & Heidmann, T. Murine MusD retrotransposon: structure and molecular evolution of an ‘intracellularized’ retrovirus. _J. Virol._ **81**, 1888–1898 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhslKjsbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17151128)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Murine%20MusD%20retrotransposon%3A%20structure%20and%20molecular%20evolution%20of%20an%20%E2%80%98intracellularized%E2%80%99%20retrovirus&journal=J.%20Virol.&volume=81&pages=1888-1898&publication_year=2007&author=Ribet%2CD&author=Harper%2CF&author=Dewannieux%2CM&author=Pierron%2CG&author=Heidmann%2CT) 
    
35.  Schorn, A. J. & Martienssen, R. Tie-break: host and retrotransposons play tRNA. _Trends Cell Biol._ **28**, 793–806 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVyqsbzI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29934075)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6520983)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tie-break%3A%20host%20and%20retrotransposons%20play%20tRNA&journal=Trends%20Cell%20Biol.&volume=28&pages=793-806&publication_year=2018&author=Schorn%2CAJ&author=Martienssen%2CR) 
    
36.  Schorn, A. J., Gutbrod, M. J., LeBlanc, C. & Martienssen, R. LTR-retrotransposon control by tRNA-derived small RNAs. _Cell_ **170**, 61–71.e11 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFSqsb7P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28666125)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5551035)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTR-retrotransposon%20control%20by%20tRNA-derived%20small%20RNAs&journal=Cell&volume=170&pages=61-71.e11&publication_year=2017&author=Schorn%2CAJ&author=Gutbrod%2CMJ&author=LeBlanc%2CC&author=Martienssen%2CR) 
    
37.  Brohmann, H., Jagla, K. & Birchmeier, C. The role of _Lbx1_ in migration of muscle precursor cells. _Development_ **127**, 437–445 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXhsVOrurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10603359)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20Lbx1%20in%20migration%20of%20muscle%20precursor%20cells&journal=Development&volume=127&pages=437-445&publication_year=2000&author=Brohmann%2CH&author=Jagla%2CK&author=Birchmeier%2CC) 
    
38.  Sidow, A. et al. A novel member of the F-box/WD40 gene family, encoding dactylin, is disrupted in the mouse dactylaplasia mutant. _Nat. Genet._ **23**, 104–107 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXlvFWht70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10471509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20novel%20member%20of%20the%20F-box%2FWD40%20gene%20family%2C%20encoding%20dactylin%2C%20is%20disrupted%20in%20the%20mouse%20dactylaplasia%20mutant&journal=Nat.%20Genet.&volume=23&pages=104-107&publication_year=1999&author=Sidow%2CA) 
    
39.  Seto, M. L., Nunes, M. E., MacArthur, C. A. & Cunningham, M. L. Pathogenesis of ectrodactyly in the _Dactylaplasia_ mouse: aberrant cell death of the apical ectodermal ridge. _Teratology_ **56**, 262–270 (1997).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXnsl2rsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9408977)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathogenesis%20of%20ectrodactyly%20in%20the%20Dactylaplasia%20mouse%3A%20aberrant%20cell%20death%20of%20the%20apical%20ectodermal%20ridge&journal=Teratology&volume=56&pages=262-270&publication_year=1997&author=Seto%2CML&author=Nunes%2CME&author=MacArthur%2CCA&author=Cunningham%2CML) 
    
40.  Crackower, M. A., Motoyama, J. & Tsui, L. C. Defect in the maintenance of the apical ectodermal ridge in the _Dactylaplasia_ mouse. _Dev. Biol._ **201**, 78–89 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXmtVyhs7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9733575)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defect%20in%20the%20maintenance%20of%20the%20apical%20ectodermal%20ridge%20in%20the%20Dactylaplasia%20mouse&journal=Dev.%20Biol.&volume=201&pages=78-89&publication_year=1998&author=Crackower%2CMA&author=Motoyama%2CJ&author=Tsui%2CLC) 
    
41.  Friedli, M. et al. Characterization of mouse _Dactylaplasia_ mutations: a model for human ectrodactyly SHFM3. _Mamm. Genome_ **19**, 272–278 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18392654)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterization%20of%20mouse%20Dactylaplasia%20mutations%3A%20a%20model%20for%20human%20ectrodactyly%20SHFM3&journal=Mamm.%20Genome&volume=19&pages=272-278&publication_year=2008&author=Friedli%2CM) 
    
42.  Lewandoski, M., Sun, X. & Martin, G. R. Fgf8 signalling from the AER is essential for normal limb development. _Nat. Genet._ **26**, 460–463 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXptVWgtrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11101846)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fgf8%20signalling%20from%20the%20AER%20is%20essential%20for%20normal%20limb%20development&journal=Nat.%20Genet.&volume=26&pages=460-463&publication_year=2000&author=Lewandoski%2CM&author=Sun%2CX&author=Martin%2CGR) 
    
43.  Moon, A. M. & Capecchi, M. R. Fgf8 is required for outgrowth and patterning of the limbs. _Nat. Genet._ **26**, 455–459 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXptVWgtrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11101845)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2001274)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fgf8%20is%20required%20for%20outgrowth%20and%20patterning%20of%20the%20limbs&journal=Nat.%20Genet.&volume=26&pages=455-459&publication_year=2000&author=Moon%2CAM&author=Capecchi%2CMR) 
    
44.  Mariani, F. V., Ahn, C. P. & Martin, G. R. Genetic evidence that FGFs have an instructive role in limb proximal–distal patterning. _Nature_ **453**, 401–405 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlvV2qtL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18449196)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2631409)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20evidence%20that%20FGFs%20have%20an%20instructive%20role%20in%20limb%20proximal%E2%80%93distal%20patterning&journal=Nature&volume=453&pages=401-405&publication_year=2008&author=Mariani%2CFV&author=Ahn%2CCP&author=Martin%2CGR) 
    
45.  Summerbell, D. A quantitative analysis of the effect of excision of the AER from the chick limb bud. _J. Embryol. Exp. Morphol._ **32**, 651–660 (1974).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaE2M7ptVyjtA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=4463222)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20quantitative%20analysis%20of%20the%20effect%20of%20excision%20of%20the%20AER%20from%20the%20chick%20limb%20bud&journal=J.%20Embryol.%20Exp.%20Morphol.&volume=32&pages=651-660&publication_year=1974&author=Summerbell%2CD) 
    
46.  Jönsson, M. E. et al. Activation of endogenous retroviruses during brain development causes an inflammatory response. _EMBO J._ **40**, e106423 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33644903)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8090857)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Activation%20of%20endogenous%20retroviruses%20during%20brain%20development%20causes%20an%20inflammatory%20response&journal=EMBO%20J.&volume=40&publication_year=2021&author=J%C3%B6nsson%2CME) 
    
47.  Bieda, K., Hoffman, A. & Boller, K. Phenotypic heterogeneity of human endogenous retrovirus particles produced by teratocarcinoma cell lines. _J. Gen. Virol._ **82**, 591–596 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhs1Sjsbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11172100)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phenotypic%20heterogeneity%20of%20human%20endogenous%20retrovirus%20particles%20produced%20by%20teratocarcinoma%20cell%20lines&journal=J.%20Gen.%20Virol.&volume=82&pages=591-596&publication_year=2001&author=Bieda%2CK&author=Hoffman%2CA&author=Boller%2CK) 
    
48.  Grow, E. J. et al. Intrinsic retroviral reactivation in human preimplantation embryos and pluripotent cells. _Nature_ **522**, 221–246 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXnsVWrtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25896322)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4503379)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intrinsic%20retroviral%20reactivation%20in%20human%20preimplantation%20embryos%20and%20pluripotent%20cells&journal=Nature&volume=522&pages=221-246&publication_year=2015&author=Grow%2CEJ) 
    
49.  Pastuzyn, E. D. et al. The neuronal gene arc encodes a repurposed retrotransposon gag protein that mediates intercellular RNA transfer. _Cell_ **172**, 275–288.e18 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVegtLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29328916)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5884693)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20neuronal%20gene%20arc%20encodes%20a%20repurposed%20retrotransposon%20gag%20protein%20that%20mediates%20intercellular%20RNA%20transfer&journal=Cell&volume=172&pages=275-288.e18&publication_year=2018&author=Pastuzyn%2CED) 
    
50.  Segel, M. et al. Mammalian retrovirus-like protein PEG10 packages its own mRNA and can be pseudotyped for mRNA delivery. _Science_ **373**, 882–889 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVCku7zK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34413232)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8431961)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mammalian%20retrovirus-like%20protein%20PEG10%20packages%20its%20own%20mRNA%20and%20can%20be%20pseudotyped%20for%20mRNA%20delivery&journal=Science&volume=373&pages=882-889&publication_year=2021&author=Segel%2CM) 
    
51.  Xu, J. et al. PNMA2 forms immunogenic non-enveloped virus-like capsids associated with paraneoplastic neurological syndrome. _Cell_ **187**, 831–845.e19 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXis1Sku7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38301645)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10922747)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PNMA2%20forms%20immunogenic%20non-enveloped%20virus-like%20capsids%20associated%20with%20paraneoplastic%20neurological%20syndrome&journal=Cell&volume=187&pages=831-845.e19&publication_year=2024&author=Xu%2CJ) 
    
52.  Chang, N.-C. et al. Gag proteins encoded by endogenous retroviruses are required for zebrafish development. _Proc. Natl Acad. Sci. USA_ **122**, e2411446122 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhtF2qurrJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=40294259)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gag%20proteins%20encoded%20by%20endogenous%20retroviruses%20are%20required%20for%20zebrafish%20development&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=122&publication_year=2025&author=Chang%2CN-C) 
    
53.  De Cecco, M. et al. L1 drives IFN in senescent cells and promotes age-associated inflammation. _Nature_ **566**, 73–78 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30728521)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6519963)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=L1%20drives%20IFN%20in%20senescent%20cells%20and%20promotes%20age-associated%20inflammation&journal=Nature&volume=566&pages=73-78&publication_year=2019&author=Cecco%2CM) 
    
54.  Kraft, K. et al. Deletions, inversions, duplications: engineering of structural variants using CRISPR/Cas in mice. _Cell Rep._ **10**, 833–839 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXisVOltrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25660031)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deletions%2C%20inversions%2C%20duplications%3A%20engineering%20of%20structural%20variants%20using%20CRISPR%2FCas%20in%20mice&journal=Cell%20Rep.&volume=10&pages=833-839&publication_year=2015&author=Kraft%2CK) 
    
55.  Artus, J. & Hadjantonakis, A.-K. Generation of chimeras by aggregation of embryonic stem cells with diploid or tetraploid mouse embryos. _Methods Mol. Biol._ **693**, 37–56 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhs1Wmu7nM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21080273)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Generation%20of%20chimeras%20by%20aggregation%20of%20embryonic%20stem%20cells%20with%20diploid%20or%20tetraploid%20mouse%20embryos&journal=Methods%20Mol.%20Biol.&volume=693&pages=37-56&publication_year=2011&author=Artus%2CJ&author=Hadjantonakis%2CA-K) 
    
56.  Bock, C. et al. BiQ Analyzer: visualization and quality control for DNA methylation data from bisulfite sequencing. _Bioinformatics_ **21**, 4067–4068 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtFCjt7bO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16141249)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BiQ%20Analyzer%3A%20visualization%20and%20quality%20control%20for%20DNA%20methylation%20data%20from%20bisulfite%20sequencing&journal=Bioinformatics&volume=21&pages=4067-4068&publication_year=2005&author=Bock%2CC) 
    
57.  Franke, M. et al. Duplications disrupt chromatin architecture and rewire GPR101-enhancer communication in X-linked acrogigantism. _Am. J. Hum. Genet._ **109**, 553–570 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XkslKhs7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35202564)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9069129)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Duplications%20disrupt%20chromatin%20architecture%20and%20rewire%20GPR101-enhancer%20communication%20in%20X-linked%20acrogigantism&journal=Am.%20J.%20Hum.%20Genet.&volume=109&pages=553-570&publication_year=2022&author=Franke%2CM) 
    
58.  Suloway, C. et al. Automated molecular microscopy: the new Leginon system. _J. Struct. Biol._ **151**, 41–60 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlsFCltbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15890530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automated%20molecular%20microscopy%3A%20the%20new%20Leginon%20system&journal=J.%20Struct.%20Biol.&volume=151&pages=41-60&publication_year=2005&author=Suloway%2CC) 
    
59.  Cardona, A. et al. TrakEM2 software for neural circuit reconstruction. _PLoS ONE_ **7**, e38011 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XptlSntL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22723842)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3378562)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TrakEM2%20software%20for%20neural%20circuit%20reconstruction&journal=PLoS%20ONE&volume=7&publication_year=2012&author=Cardona%2CA) 
    
60.  Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. _Nat. Methods_ **9**, 676–682 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKnurbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22743772)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fiji%3A%20an%20open-source%20platform%20for%20biological-image%20analysis&journal=Nat.%20Methods&volume=9&pages=676-682&publication_year=2012&author=Schindelin%2CJ) 
    
61.  Mikie Phan. mikstapes/JGlaser-etal\_Dac1J: first release (v1.0.0). _Zenodo_ [https://doi.org/10.5281/zenodo.15528362](https://doi.org/10.5281/zenodo.15528362) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02248-5?format=refman&flavour=references)

Acknowledgements
----------------

This work was supported by a grant from the Deutsche Forschungsgemeinschaft (DFG) MU 880/16-1 to S.M. J.G. was supported by the HFSP postdoctoral fellowship (LT000465/2019-L). G.A. is supported by Swiss National Science Foundation grants PP00P3\_176802 and PP00P3\_210995-6. We thank the laboratory of T. Heidmann for generously giving us the antiserum rabbit MusD-Gag. We thank K. Macura, J. Fielder, C. Franke, N. Michaelis, U. Fisher and A. Stiege for technical support, as well as T. Aktas for scientific support. We thank the Schulz laboratory and I. Dunkel for the use of their pyrosequencer. We thank T. Aktas and W. Schwarzer for their thesis dissertation work. We would like to thank all the members of the Mundlos laboratory for their continuous support and stimulation. Finally, we thank D. Ibrahim, G. Cavalheiro, N. Benetti, K. Chudzik, P. Kurbel, A. Monaco and E. van Leen for their critical reading of the paper.

Funding
-------

Open access funding provided by Max Planck Society.

Author information
------------------

Author notes

1.  Giulia Cova
    
    Present address: Department of Pathology, New York University School of Medicine, Langone Health Medical Center, New York, NY, USA
    
2.  Martin Franke & Christina Paliou
    
    Present address: Centro Andaluz de Biología del Desarrollo (CABD), Consejo Superior de Investigaciones Científicas/Universidad Pablo de Olavide, Seville, Spain
    
3.  Guillaume Andrey
    
    Present address: Department of Genetic Medicine and Development, Faculty of Medicine, University of Geneva, Geneva, Switzerland
    
4.  Guillaume Andrey
    
    Present address: Institute of Genetics and Genomics in Geneva (iGE3), University of Geneva, Geneva, Switzerland
    

### Authors and Affiliations

1.  RG Development & Disease, Max Planck Institute for Molecular Genetics, Berlin, Germany
    
    Juliane Glaser, Giulia Cova, Cesar A. Prada-Medina, Mai H. Q. Phan, Robert Schöpflin, Yasmin Aktas, Martin Franke, Guillaume Andrey, Natalia Bartzoka, Christina Paliou & Stefan Mundlos
    
2.  Microscopy and Cryo-electron Microscopy Service Group, Max Planck Institute for Molecular Genetics, Berlin, Germany
    
    Beatrix Fauler & Thorsten Mielke
    
3.  Department of Computational Molecular Biology, Max Planck Institute for Molecular Genetics, Berlin, Germany
    
    Virginie Stanislas, Robert Schöpflin & Verena Laupert
    
4.  Institute for Medical and Human Genetics, Charité-Universitätsmedizin Berlin, Berlin, Germany
    
    Natalia Bartzoka, Wing-Lee Chan & Stefan Mundlos
    
5.  Department of Developmental Genetics, Max Planck Institute for Molecular Genetics, Berlin, Germany
    
    Lars Wittler
    
6.  Berlin–Brandenburg Center for Regenerative Therapies, Charité-Universitätsmedizin Berlin, Berlin, Germany
    
    Stefan Mundlos
    

Authors

1.  Juliane Glaser
    
    [View author publications](https://www.nature.com/search?author=Juliane%20Glaser)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Juliane%20Glaser) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Juliane%20Glaser%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Giulia Cova
    
    [View author publications](https://www.nature.com/search?author=Giulia%20Cova)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Giulia%20Cova) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Giulia%20Cova%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Beatrix Fauler
    
    [View author publications](https://www.nature.com/search?author=Beatrix%20Fauler)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Beatrix%20Fauler) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Beatrix%20Fauler%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Cesar A. Prada-Medina
    
    [View author publications](https://www.nature.com/search?author=Cesar%20A.%20Prada-Medina)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cesar%20A.%20Prada-Medina) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cesar%20A.%20Prada-Medina%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Virginie Stanislas
    
    [View author publications](https://www.nature.com/search?author=Virginie%20Stanislas)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Virginie%20Stanislas) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Virginie%20Stanislas%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Mai H. Q. Phan
    
    [View author publications](https://www.nature.com/search?author=Mai%20H.%20Q.%20Phan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mai%20H.%20Q.%20Phan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mai%20H.%20Q.%20Phan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Robert Schöpflin
    
    [View author publications](https://www.nature.com/search?author=Robert%20Sch%C3%B6pflin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Robert%20Sch%C3%B6pflin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Robert%20Sch%C3%B6pflin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Yasmin Aktas
    
    [View author publications](https://www.nature.com/search?author=Yasmin%20Aktas)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yasmin%20Aktas) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yasmin%20Aktas%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Martin Franke
    
    [View author publications](https://www.nature.com/search?author=Martin%20Franke)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Martin%20Franke) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Martin%20Franke%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Guillaume Andrey
    
    [View author publications](https://www.nature.com/search?author=Guillaume%20Andrey)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Guillaume%20Andrey) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Guillaume%20Andrey%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Natalia Bartzoka
    
    [View author publications](https://www.nature.com/search?author=Natalia%20Bartzoka)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Natalia%20Bartzoka) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Natalia%20Bartzoka%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Christina Paliou
    
    [View author publications](https://www.nature.com/search?author=Christina%20Paliou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Christina%20Paliou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Christina%20Paliou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Verena Laupert
    
    [View author publications](https://www.nature.com/search?author=Verena%20Laupert)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Verena%20Laupert) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Verena%20Laupert%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Wing-Lee Chan
    
    [View author publications](https://www.nature.com/search?author=Wing-Lee%20Chan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wing-Lee%20Chan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wing-Lee%20Chan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Lars Wittler
    
    [View author publications](https://www.nature.com/search?author=Lars%20Wittler)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lars%20Wittler) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lars%20Wittler%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Thorsten Mielke
    
    [View author publications](https://www.nature.com/search?author=Thorsten%20Mielke)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Thorsten%20Mielke) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Thorsten%20Mielke%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Stefan Mundlos
    
    [View author publications](https://www.nature.com/search?author=Stefan%20Mundlos)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Stefan%20Mundlos) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Stefan%20Mundlos%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

J.G. and S.M. conceived the project. J.G. designed the experiments and generated transgenic mouse models with the help of G.C. and Y.A. L.W. performed morula aggregation. J.G. and G.C. performed the WISH experiments and the skeletal preparations. W.-L.C. performed the micro-CT. J.G. performed and analyzed the bisulfite-cloning sequencing, pyrosequencing and bulk RNA-seq. J.G. and M.F. performed the cHi-C and M.H.Q.P. and R.S. carried out the processing. ChIP–seq was performed by C.P., N.B. and J.G. and analyzed by J.G. Preparation of the scRNA-seq samples was performed by J.G., G.C., G.A. and M.F. and processed by C.A.P.-M. and V.S. 4C-seq was performed by M.F. and analyzed by V.L. J.G. performed the whole-mount immunofluorescence and HCR. B.F. and T.M. performed the TEM and immuno-EM experiments. J.G. and S.M. wrote the paper with input from G.A., G.C. and M.F.; the paper was approved by all authors.

### Corresponding authors

Correspondence to [Juliane Glaser](mailto:glaserj@ie-freiburg.mpg.de) or [Stefan Mundlos](mailto:mundlos@molgen.mpg.de).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Andrew Modzelewski and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02248-5#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Dactylaplasia phenotype and epigenetic polymorphism.](https://www.nature.com/articles/s41588-025-02248-5/figures/7)

**a**, Dorsal, ventral and micro-computed tomography (μCT) views of wild-type, _Dac1J_/WT (129s2/Sv) and _Dac1J/Dac1J_ (129s2/Sv) fore- and hind-paws from 7-month-old adults illustrating the phenotype in heterozygotes and homozygotes respectively. _n_ = 2 biological replicates were analysed. White arrows indicate observed nail-like structures. **b**, ventral high-magnification view (top) and micro-computed tomography (μCT) (middle and bottom) of _Dac1J/Dac1J_ (129s2/Sv) hind-paw illustrating a typical nail-like-structure (black arrow) which does not contain any bone. **c**, **d**, Ventral views of the various phenotypes observed in _Dac1J/Dac1J_ (129s2/Sv) (**c**) and _Dac1J_/WT (129s2/Sv) (**d**) fore- and hindpaws. _n_ = x/x paws with a similar phenotype. For **a-d**, scale bars 1 mm. **e**, DNA methylation status of 19 CpGs from the 5’LTR (promoter) of the MusD-_Dac1J_ insertion at the _Fgf8_ locus in a C57BL/6 background (left) or 129s2/Sv background (right) measured by bisulfite cloning and sequencing from E11.5 brain, and visceral yolk sac. Constitutive low and high levels of DNA methylation of the MusD-_Dac1J_ 5’LTR are observed in the 129s2/Sv and a C57BL/6 background respectively. White circles, unmethylated CpGs; black circles, methylated CpGs. **f**, Bulk RNA sequencing and epigenetic profiling (ATAC-seq, CTCF, H3K27ac and H3K4me ChIP-seq) from E11.5 wild-type forelimbs show that neither the MusD-_Dac1J_ insertion nor its deletion seem to affect regulatory elements.

### [Extended Data Fig. 2 Local gene expression changes in _Dac1J/Dac1J_ limb buds.](https://www.nature.com/articles/s41588-025-02248-5/figures/8)

**a**, Rpkm expression of the 6 genes ate the _Lbx1-Fgf8_ locus from bulk RNA-sequencing from early E11.5 forelimbs wild type (grey), _Dac1J/Dac1J_ (129s2/Sv) (light green), and _Dac1J/Dac1J_ (C57BL/6) (dark green). Data are shown as mean ± s.e.m. of _n_ = 3 biological replicates. **b**, In situ hybridization for _Fgf8_ at E10.5 (left) and in E11.5 (right) wild-type and _Dac1J/Dac1J_ showing whole embryos and forelimbs. Dotted lines draw the shape of the forelimb. AER, apical ectoderm ridge. Scale bars 500um, at least _n_ = 3 embryos were analysed per genotype.

### [Extended Data Fig. 3 Gene expression changes in _Dac1J/Dac1J_ AER.](https://www.nature.com/articles/s41588-025-02248-5/figures/9)

**a**, Violin plot from E9.5-E11.5 single-cell data showing expression of two marker genes per cluster. **b**, Histogram showing cell number in each cell cluster from wild-type and _Dac1J/Dac1J_ at E9.5, E10.5, and E11.5. **c**, Violin plot depicting the expression of _Fgf8_ (left) and _Fbxw4_ (right) between wild-type (grey) and mutant (green) in each cluster, at the three tested embryonic stages. Each dot represents a cell.

### [Extended Data Fig. 4 Single-cell expression of AER genes and genes at the _Lbx1-Fgf8_ locus.](https://www.nature.com/articles/s41588-025-02248-5/figures/10)

**a**–**c**, Dot plots showing expression and percentage of expressing cells for _Fgf8_ and MusD-_Dac1J_ at E11.5 (**a**), 6 AER genes (_Dlx1, Dlx5_, _Dlx6, Msx2, Wnt6_, and _Fgf4_) at E9.5, E10.5 and E11.5 (**b**), and the 5 genes at the _Lbx1-Fgf8_ locus at E9.5, E10.5 and E11.5 (**c**). Data are shown in the 7 forelimb cell cluster in wild-type (grey) and _Dac1J/Dac1J_ (green).

### [Extended Data Fig. 5 3D conformation changes at the _Fgf8_ locus.](https://www.nature.com/articles/s41588-025-02248-5/figures/11)

**a**, Sequences of the 6 CTCFs binding sites detected in the MusD-_Dac1J_ as identified with the FIMO (Find Individual Motif Occurrences) suite. The program uses a dynamic programming algorithm to convert log-odds scores into _p_\-values, assuming a zero-order background model. _p_\-value < 10−4 is considered as significant and marked in bold. FIMO score and _p_\-value are indicated for each binding site. Orange and red triangles represent sense and antisense CTCF sites respectively. **b**, Zoom-in from the Capture-Hi-C subtraction map between wild-type and _Dac1J/Dac1J_ (129s2/Sv) in Fig. [3d](https://www.nature.com/articles/s41588-025-02248-5#Fig3) showing the _Fgf8_ TAD. **c**, Capture-Hi-C of the _Lbx1/Fgf8_ locus (_mm10, chr19: 45,100,000-45,900,000_) from _Dac1J-Bl6_ E11.5 mouse limbs buds. Data show the c-HiC as merged signals of _n_ = 2 biological and 2 technical replicates. Subtraction maps between mutants and wild-type show gain (red) and loss (blue) of interaction in the mutant compared to the wild-type. **d**, CTCF ChIP-sequencing from forelimbs at E10.5 and E11.5 showing tracks at the _Fgf8_ locus. **e**, CpG DNA methylation status of 19 CpGs in the MusD-_Dac1J_ 5’LTR from E11.5 _Dac1J/Dac1J_ (129s2/Sv) hearts (left) and E11.5 5’LTR-LacZ KI/WT limbs (right). **f**, Skeletal analysis of E18.5 5LTR-LacZ KI +/- forelimbs stained with alcian blue (cartilage) and alizarin red (bone). Scale bars 1 mm, _n_ = 8/8 show a similar phenotype.

### [Extended Data Fig. 6 Gag-MusD expression and VLPs in the _Dac1J_ mutant embryos.](https://www.nature.com/articles/s41588-025-02248-5/figures/12)

**a**, Table showing the percentage of amino-acid identity between MusD-_Dac1J_ and the three MusD (MusD-1, 2, and 6) identified as autonomous for retro-transposition in _Ribet_ et al. _2004_. Identity, similarity, and gaps of amino-acids are indicated for both GAG and POL proteins. **b**, anti-GAG-MusD whole-mount immuno-fluorescence on E10.5 _Dac1J/Dac1J_ (C57BL/6) embryos showing whole embryo with no staining. Scale bars 1 mm. _n_ = 5/5 biological replicates were confirmed. **c**, **d**, anti-GAG-MusD whole-mount immuno-fluorescence on E10.5 (**c**) and E11.0 (**d**) _Dac1J/Dac1J_ (129s2/Sv) embryos showing branchial arches (BA), midbrain-hindbrain boundary (MHB), tailbud (TB), whole embryo and limbs (FL, forelimbs; HL hindlimbs). Scale bar 100 nm. _n_ = 5/5 (E10.5) and _n_ = 4/4 (E11.0) biological replicates were confirmed. **e**–**g**, TEM analysis after immuno-gold labeling with anti-GAG-MusD antibody on E11.0 _Dac1J/Dac1J_ AER cells shows cytoplasmic aggregates of GAG. Scale bar 100 nm. _n_ = 2 biological replicates and _n_ = 6 technical replicates were confirmed. C, cytoplasm; N, nucleus. Green circle represents one VLP with Gag capsid. Red Asterix indicates an apoptotic body. **h**, TEM analysis after immuno-gold labeling with anti-GAG-MusD antibody on E11.0 wild-type control AER cell shows no staining. _n_ = 2 biological replicates were confirmed. **i**, **j**, TEM analysis on E11.0 _Dac1J/Dac1J_ hindlimbs (**i**) and zoom-in view on 4 forelimb AER cells showing apoptotic bodies and phagocytes (**j**). Scale bars 50um and 1um.

### [Extended Data Fig. 7 Apoptotic cell death in the _Dac1J_ embryos.](https://www.nature.com/articles/s41588-025-02248-5/figures/13)

**a**, **b**, anti-cleaved-Caspase3 (**a**) and anti-gamma-H2AX (**b**) whole-mount immuno-fluorescence on E11.0 wild-type and _Dac1J/Dac1J_ showing branchial arches area. Scale bars 100um. At least _n_ = 3 biological replicates were confirmed. **c**–**f**, HCR in situ hybridization with an _Fgf8_ mRNA probe on showing embryonic forelimbs at E10.0 (30 somites), E10.5 (35 somites), E11.0 (40 somites) and E11.5 (46 somites) in wild-type (**c**) and _Dac1J/Dac1J_ (**d**) embryos. Scale bars 100um and 50um. **e**, **f**, DAPI staining showing the AER area of forelimbs in wild-type (**e**) and _Dac1J/Dac1J_ (**f**) embryos at E11.0 (40 somites) and E11.5 (46 somites). _n_ = 2 biological replicates were confirmed for each genotype and stage. Scale bars 50um.

### [Extended Data Fig. 8 Gene expression changes in response to VLPs production in forelimbs.](https://www.nature.com/articles/s41588-025-02248-5/figures/14)

**a**–**c**, Heat-map showing log2 (rpkm+1) of selected apoptosis (**a**), developing limb (**b**), and interferon-stimulated (**c**) genes. Genes showing significant (_p_\-value < 0.05) expression changes in _Dac1J/Dac1J_ compared to wild-type are indicated with an asterisk. _P_\-value was calculated using the DEseq2 package with a Wald test.

### [Extended Data Fig. 9 Knock-ins of MusD-_Dac1J_ carrying mutations.](https://www.nature.com/articles/s41588-025-02248-5/figures/15)

**a**, Detail of the 3554 bp deletion of the Pol gene and 3’ sequence (top), the 10 bp insertion of random DNA in the _Dac1J_\-mutPBS knock-in (middle), and the two Gag mutations (bottom). In the Δ/ΔPol-_Dac1J_ AER cells, Gag and Pro assemble into a VLP lacking reverse transcriptase. This incomplete VLP can neither replicate ETns, nor undergo tRNA primed reverse transcriptase of its RNA. In the _Dac1J_\-mutPBS knock-in, the 10 bp insertion in the Lys tRNA primer binding site (PBS) leads to a scrambled PBS. In the _Dac1J_\-mutPBS AER cells, Gag, Pro, and Pol assemble into VLP. The intact Pol allows possible replication of ETn elements, but the MusD VLPs are unable to undergo tRNA-primed reverse transcription. In the mut-gag knock-in, the entire _gag_ gene is deleted, but a complete pro and pol protein can be assembled. In the mut-gag(CA) knock-in, the capsid and the zinc-finger domain are deleted so that the matrix (MA) protein of the gag polyprotein can be produced. **b**, anti-GAG-MusD whole-mount immuno-fluorescence on E11.5 mut-pol KI/KI (left), mut-PBS KI/KI (middle), and mut-gag(CA) KI/KI (right) whole embryos. Scale bars 1 mm. At least _n_ = 3 biological replicates were confirmed. FL, forelimb; HL, hindlimb; BA, branchial arches; MHB, mid-hindbrain boundary. **c**, Representation of E18.5 skeletal analysis and adult limbs illustrating the six different observed phenotypes indicated in Fig. [5d](https://www.nature.com/articles/s41588-025-02248-5#Fig5). **d**, CpG DNA methylation at the 5’LTR promoter of the _Dac1J_ insertion measured by pyrosequencing in _Dac1J_ mut-pol KI, mut-PBS KI, mut-gag KI and mut-gag(CA) KI E18.5 embryos. Each dot represents the average CpG DNA methylation measured over 7 CpGs in one individual with a phenotype (red dot) or without a phenotype (black dot). Overall, no difference in 5’LTR DNA methylation is observed between the animals with or without a phenotype. **e**, Schematic representation of the 5’LTR CpGs content and the CpGs measured by pyrosequencing or bisulfite cloning sequencing.

### [Extended Data Fig. 10 MusD-_Dac1J_ adopts the expression of genes surrounding its insertion.](https://www.nature.com/articles/s41588-025-02248-5/figures/16)

**a**, **b** Schematic representation of the _Dac1J_ -5LTR-LacZ knock-in at the _Shh_ (**a**) and _Sox9_ (**b**) locus. **c**, whole mount _in situ_ hybridization of _Shh_ from E10.5 wild-type embryo. Scale bar 500um. _n_ = 3 biological replicates were confirmed. **d**, beta-galactosidase staining on E10.0 _Dac1J_\-5LTR-LacZ (_Shh_ knock-in) showing whole embryos. Scale bars 500um, _n_ = 3/4 embryos show similar staining. **e**, whole mount _in situ_ hybridization of _Sox9_ from E12.5 wild-type embryo. Scale bar 500um. _n_ = 3 biological replicates were confirmed. **f**, beta-galactosidase staining on E12.5 _Dac1J_\-5LTR-LacZ (_Sox9_ knock-in) showing whole embryos. Scale bars 500um, _n_ = 2/6 embryos show similar staining. **g**, anti-GAG-MusD whole-mount immuno-fluorescence on E11.5 _Dac1J_\-mutPBS KI/KI (_Lbx1_ knock-in) showing forelimb staining. Scale bars 100um. At least _n_ = 3 embryos were tested. **h**, Lysotracker staining of an E12.5 _Dac1J_ KI/KI (_Lbx1_ knock-in) forelimb. Expected AER staining for that stage is observed but no muscle cell progenitor staining is detected. _n_ = 3 embryos were analysed. Scale bars 100 um.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_MOESM1_ESM.pdf)

Supplementary notes and references, Supplementary Table legends, Supplementary methods and references.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02248-5/MediaObjects/41588_2025_2248_MOESM4_ESM.xlsx)

Supplementary Tables 1–7.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Enhancer%20adoption%20by%20an%20LTR%20retrotransposon%20generates%20viral-like%20particles%2C%20causing%20developmental%20limb%20phenotypes&author=Juliane%20Glaser%20et%20al&contentID=10.1038%2Fs41588-025-02248-5&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-09&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02248-5)

### Cite this article

Glaser, J., Cova, G., Fauler, B. _et al._ Enhancer adoption by an LTR retrotransposon generates viral-like particles, causing developmental limb phenotypes. _Nat Genet_ **57**, 1766–1776 (2025). https://doi.org/10.1038/s41588-025-02248-5

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02248-5?format=refman&flavour=citation)

-   Received: 12 May 2024
    
-   Accepted: 27 May 2025
    
-   Published: 09 July 2025
    
-   Issue Date: July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02248-5
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02248-5.pdf)
