> id: https://www.nature.com/articles/s41588-025-02258-3

> link: https://www.nature.com/articles/s41588-025-02258-3

> title: Genome assembly of two allotetraploid cotton germplasms reveals mechanisms of somatic embryogenesis and enables precise genome editing

# Genome assembly of two allotetraploid cotton germplasms reveals mechanisms of somatic embryogenesis and enables precise genome editing
_Published on Tue, 22 Jul 2025 00:00:00 GMT_

Abstract
--------

Somatic embryogenesis is crucial for plant genetic engineering, yet the underlying mechanisms in cotton remain poorly understood. Here we present a telomere-to-telomere assembly of Jin668 and a high-quality assembly of YZ1, two highly regenerative allotetraploid cotton germplasms. The completion of the Jin668 genome enables characterization of ~30.1 Mb of centromeric regions invaded by centromeric retrotransposon of maize and _Tekay_ retrotransposons, an ~8.1 Mb 5S rDNA array containing 25,190 copies and a ~75.1 Mb major 45S rDNA array with 8,131 copies. Comparative analyses of regenerative and recalcitrant genotypes reveal dynamic transcriptional patterns and chromatin accessibility during the initial regeneration process. A hierarchical gene regulatory network identifies _AGL15_ as a contributor to regeneration. Additionally, we demonstrate that genetic variation affects sgRNA target sites, while the Jin668 genome assembly reduces the risk of off-target effects in CRISPR-based genome editing. Together, the complete Jin668 genome reveals the complexity of genomic regions and cotton regeneration, and improves the precision of genome editing.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 12 print issues and online access

$209.00 per year

only $17.42 per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02258-3?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: A comprehensive overview of the genetic transformation system of cotton and the complete genome features of Jin668 and YZ1.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_Fig1_HTML.png)

**Fig. 2: Completeness and accuracy validation of both chromosome arms and centromeres in Jin668 genome.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_Fig2_HTML.png)

**Fig. 3: Centromeric characteristics of Jin668.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_Fig3_HTML.png)

**Fig. 4: In-depth comparative analysis of the Jin668 versus TM-1, YZ1 and ZM24.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_Fig4_HTML.png)

**Fig. 5: Overview of chromatin accessibility and transcriptome dynamics in Jin668 and TM-1 during SE.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_Fig5_HTML.png)

**Fig. 6: Functional verification of regeneration-related genes.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_Fig6_HTML.png)

**Fig. 7: Cotton genetic variation substantially impacts the efficacy of CRISPR-based genome editing.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_Fig7_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-025-02130-4/MediaObjects/41588_2025_2130_Fig1_HTML.png)

### [A telomere-to-telomere genome assembly of cotton provides insights into centromere evolution and short-season adaptation](https://www.nature.com/articles/s41588-025-02130-4?fromPaywallRec=true)

Article 17 March 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-55309-4/MediaObjects/41467_2024_55309_Fig1_HTML.png)

### [Epigenomic and 3D genomic mapping reveals developmental dynamics and subgenomic asymmetry of transcriptional regulatory architecture in allotetraploid cotton](https://www.nature.com/articles/s41467-024-55309-4?fromPaywallRec=true)

Article Open access 27 December 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01877-6/MediaObjects/41588_2024_1877_Fig1_HTML.png)

### [A telomere-to-telomere cotton genome assembly reveals centromere evolution and a Mutator transposon-linked module regulating embryo development](https://www.nature.com/articles/s41588-024-01877-6?fromPaywallRec=true)

Article 15 August 2024

Data availability
-----------------

The T2T-Jin668 and YZ1 genome assemblies and annotation data are available at NCBI ([PRJNA874817](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA874817) and [PRJNA960814](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA960814)) and T2TCotton-Hub ([http://jinlab.hzau.edu.cn/T2TCottonHub/](http://jinlab.hzau.edu.cn/T2TCottonHub/) or [http://cotton.hzau.edu.cn/T2TCottonHub/](http://cotton.hzau.edu.cn/T2TCottonHub/)). The raw sequencing data used for de novo whole-genome assembly of Jin668 and YZ1 are available in NCBI under BioProjects [PRJNA874817](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA874817) and [PRJNA960814](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA960814), respectively. RNA-seq of Jin668 and YZ1 are available in NCBI under BioProjects [PRJNA874819](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA874819) and [PRJNA960820](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA960820), respectively. ATAC–seq data of Jin668 is available in NCBI under BioProjects [PRJNA960832](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA960832). ATAC–seq and RNA-seq data for TM-1 during SE are available in NCBI under BioProjects [PRJNA960828](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA960828) and [PRJNA960825](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA960825), respectively. ATAC–seq and RNA-seq data for YZ1 during SE are available in NCBI under BioProjects [PRJNA1059614](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1059614) and [PRJNA1059613](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1059613), respectively. ATAC–seq and RNA-seq data for ZB1092 during SE are available in NCBI under BioProjects [PRJNA1059611](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1059611) and [PRJNA1059609](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1059609), respectively. The ChIP–seq data for Jin668 and YZ1 are uploaded to BioProjects [PRJNA1079680](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1079680) and [PRJNA1079682](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1079682), respectively. Seeds of Jin668 and YZ1 used in this study can be obtained from the corresponding author upon request. The reference genome assembly and annotation files of TM-1 (v3.1) used in this study were downloaded from [https://phytozome-next.jgi.doe.gov/](https://phytozome-next.jgi.doe.gov/) and are also accessible from T2TCottonHub. Additionally, all available SE-related motifs were obtained from Plant Transcription Factor Database (v5.0; [https://planttfdb.gao-lab.org/](https://planttfdb.gao-lab.org/)). Further details on data accessibility are outlined in the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02258-3#MOESM1) and [Methods](https://www.nature.com/articles/s41588-025-02258-3#Sec11). [Source data](https://www.nature.com/articles/s41588-025-02258-3#Sec34) are provided with this paper.

Code availability
-----------------

All original codes used in this article are available via Zenodo at [https://doi.org/10.5281/zenodo.15035095](https://doi.org/10.5281/zenodo.15035095) (ref. [96](https://www.nature.com/articles/s41588-025-02258-3#ref-CR96 "Xu, Z. Scripts used in ‘Genome assembly of two allotetraploid cotton germplasms reveals mechanisms of somatic embryogenesis and enables precise genome editing’. Zenodo 
https://doi.org/10.5281/zenodo.15035095
(2025).")) and GitHub ([https://github.com/tiramisutes/T2T-Cotton-Genomes](https://github.com/tiramisutes/T2T-Cotton-Genomes)).

References
----------

1.  Bhatia, S., Sharma, K., Dahiya, R. & Bera, T. (eds). _Modern Applications of Plant Biotechnology in Pharmaceutical Sciences_, pp. 209–230 (Academic Press, 2015).
    
2.  Zheng, Q. & Perry, S. E. Alterations in the transcriptome of Soybean in response to enhanced somatic embryogenesis promoted by orthologs of AGAMOUS-like15 and AGAMOUS-like18. _Plant Physiol._ **164**, 1365–1377 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXkslCltrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24481137)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3938626)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Alterations%20in%20the%20transcriptome%20of%20Soybean%20in%20response%20to%20enhanced%20somatic%20embryogenesis%20promoted%20by%20orthologs%20of%20AGAMOUS-like15%20and%20AGAMOUS-like18&journal=Plant%20Physiol.&volume=164&pages=1365-1377&publication_year=2014&author=Zheng%2CQ&author=Perry%2CSE) 
    
3.  Horstman, A., Bemer, M. & Boutilier, K. A transcriptional view on somatic embryogenesis. _Regeneration (Oxf.)_ **4**, 201–216 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29299323)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20transcriptional%20view%20on%20somatic%20embryogenesis&journal=Regeneration%20%28Oxf.%29&volume=4&pages=201-216&publication_year=2017&author=Horstman%2CA&author=Bemer%2CM&author=Boutilier%2CK) 
    
4.  Wang, K. et al. The gene _TaWOX5_ overcomes genotype dependency in wheat genetic transformation. _Nat. Plants_ **8**, 110–117 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35027699)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20gene%20TaWOX5%20overcomes%20genotype%20dependency%20in%20wheat%20genetic%20transformation&journal=Nat.%20Plants&volume=8&pages=110-117&publication_year=2022&author=Wang%2CK) 
    
5.  Chen, Z., Debernardi, J. M., Dubcovsky, J. & Gallavotti, A. Recent advances in crop transformation technologies. _Nat. Plants_ **8**, 1343–1351 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFeis7nN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36522447)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Recent%20advances%20in%20crop%20transformation%20technologies&journal=Nat.%20Plants&volume=8&pages=1343-1351&publication_year=2022&author=Chen%2CZ&author=Debernardi%2CJM&author=Dubcovsky%2CJ&author=Gallavotti%2CA) 
    
6.  Li, J. et al. Multi-omics analyses reveal epigenomics basis for cotton somatic embryogenesis through successive regeneration acclimation process. _Plant Biotechnol. J._ **17**, 435–450 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFCgtrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29999579)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-omics%20analyses%20reveal%20epigenomics%20basis%20for%20cotton%20somatic%20embryogenesis%20through%20successive%20regeneration%20acclimation%20process&journal=Plant%20Biotechnol.%20J.&volume=17&pages=435-450&publication_year=2019&author=Li%2CJ) 
    
7.  Iwase, A. et al. WIND1-based acquisition of regeneration competency in _Arabidopsis_ and rapeseed. _J. Plant Res._ **128**, 389–397 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsV2iurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25810222)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WIND1-based%20acquisition%20of%20regeneration%20competency%20in%20Arabidopsis%20and%20rapeseed&journal=J.%20Plant%20Res.&volume=128&pages=389-397&publication_year=2015&author=Iwase%2CA) 
    
8.  Lowe, K. et al. Morphogenic regulators _Baby boom_ and _Wuschel_ improve monocot transformation. _Plant Cell_ **28**, 1998 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitV2hs7%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27600536)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059793)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Morphogenic%20regulators%20Baby%20boom%20and%20Wuschel%20improve%20monocot%20transformation&journal=Plant%20Cell&volume=28&publication_year=2016&author=Lowe%2CK) 
    
9.  Debernardi, J. M. et al. A GRF–GIF chimeric protein improves the regeneration efficiency of transgenic plants. _Nat. Biotechnol._ **38**, 1274–1279 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVCrtbfK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33046875)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7642171)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20GRF%E2%80%93GIF%20chimeric%20protein%20improves%20the%20regeneration%20efficiency%20of%20transgenic%20plants&journal=Nat.%20Biotechnol.&volume=38&pages=1274-1279&publication_year=2020&author=Debernardi%2CJM) 
    
10.  Wang, M. et al. Reference genome sequences of two cultivated allotetraploid cottons, _Gossypium hirsutum_ and _Gossypium barbadense_. _Nat. Genet._ **51**, 224–229 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30510239)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reference%20genome%20sequences%20of%20two%20cultivated%20allotetraploid%20cottons%2C%20Gossypium%20hirsutum%20and%20Gossypium%20barbadense&journal=Nat.%20Genet.&volume=51&pages=224-229&publication_year=2019&author=Wang%2CM) 
    
11.  Yang, Z. et al. Extensive intraspecific gene order and gene structural variations in upland cotton cultivars. _Nat. Commun._ **10**, 2989 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31278252)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611876)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extensive%20intraspecific%20gene%20order%20and%20gene%20structural%20variations%20in%20upland%20cotton%20cultivars&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Yang%2CZ) 
    
12.  Conover, J. L. & Wendel, J. F. Deleterious mutations accumulate faster in allopolyploid than diploid cotton (_Gossypium_) and unequally between subgenomes. _Mol. Biol. Evol._ **39**, msac024 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVait7zP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35099532)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8841602)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deleterious%20mutations%20accumulate%20faster%20in%20allopolyploid%20than%20diploid%20cotton%20%28Gossypium%29%20and%20unequally%20between%20subgenomes&journal=Mol.%20Biol.%20Evol.&volume=39&publication_year=2022&author=Conover%2CJL&author=Wendel%2CJF) 
    
13.  Sun, C. et al. Precise integration of large DNA sequences in plant genomes using PrimeRoot editors. _Nat. Biotechnol._ **42**, 316–327 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXosVKru70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37095350)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Precise%20integration%20of%20large%20DNA%20sequences%20in%20plant%20genomes%20using%20PrimeRoot%20editors&journal=Nat.%20Biotechnol.&volume=42&pages=316-327&publication_year=2024&author=Sun%2CC) 
    
14.  Chen, K., Wang, Y., Zhang, R., Zhang, H. & Gao, C. CRISPR/Cas genome editing and precision plant breeding in agriculture. _Annu. Rev. Plant Biol._ **70**, 667–697 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXktFegsr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30835493)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CRISPR%2FCas%20genome%20editing%20and%20precision%20plant%20breeding%20in%20agriculture&journal=Annu.%20Rev.%20Plant%20Biol.&volume=70&pages=667-697&publication_year=2019&author=Chen%2CK&author=Wang%2CY&author=Zhang%2CR&author=Zhang%2CH&author=Gao%2CC) 
    
15.  Wang, G. et al. Precise fine-turning of _GhTFL1_ by base editing tools defines ideal cotton plant architecture. _Genome Biol._ **25**, 59 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38409014)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10895741)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Precise%20fine-turning%20of%20GhTFL1%20by%20base%20editing%20tools%20defines%20ideal%20cotton%20plant%20architecture&journal=Genome%20Biol.&volume=25&publication_year=2024&author=Wang%2CG) 
    
16.  Jiang, T., Zhang, X.-O., Weng, Z. & Xue, W. Deletion and replacement of long genomic sequences using prime editing. _Nat. Biotechnol._ **40**, 227–234 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1GnsrjK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34650270)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deletion%20and%20replacement%20of%20long%20genomic%20sequences%20using%20prime%20editing&journal=Nat.%20Biotechnol.&volume=40&pages=227-234&publication_year=2022&author=Jiang%2CT&author=Zhang%2CX-O&author=Weng%2CZ&author=Xue%2CW) 
    
17.  Fernie, A. R. & Yan, J. _De novo_ domestication: an alternative route toward new crops for the future. _Mol. Plant_ **12**, 615–631 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVOqtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30999078)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20domestication%3A%20an%20alternative%20route%20toward%20new%20crops%20for%20the%20future&journal=Mol.%20Plant&volume=12&pages=615-631&publication_year=2019&author=Fernie%2CAR&author=Yan%2CJ) 
    
18.  Shoemaker, R., Couche, L. & Galbraith, D. Characterization of somatic embryogenesis and plant regeneration in cotton (_Gossypium hirsutum_ L.). _Plant Cell Rep._ **5**, 178–181 (1986).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC2c7otlChsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24248126)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterization%20of%20somatic%20embryogenesis%20and%20plant%20regeneration%20in%20cotton%20%28Gossypium%20hirsutum%20L.%29&journal=Plant%20Cell%20Rep.&volume=5&pages=178-181&publication_year=1986&author=Shoemaker%2CR&author=Couche%2CL&author=Galbraith%2CD) 
    
19.  Jin, S. et al. Identification of a novel elite genotype for in vitro culture and genetic transformation of cotton. _Biol. Plant._ **50**, 519–524 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhtlaku77I)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20a%20novel%20elite%20genotype%20for%20in%20vitro%20culture%20and%20genetic%20transformation%20of%20cotton&journal=Biol.%20Plant.&volume=50&pages=519-524&publication_year=2006&author=Jin%2CS) 
    
20.  Wang, L. et al. The _GhmiR157a_–_GhSPL10_ regulatory module controls initial cellular dedifferentiation and callus proliferation in cotton by modulating ethylene-mediated flavonoid biosynthesis. _J. Exp. Bot._ **69**, 1081–1093 (2017).
    
21.  Xu, J. _GhL1L1_ affects cell fate specification by regulating _GhPIN1_\-mediated auxin distribution. _Plant Biotechnol. J._ **17**, 63–74 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXotVajsg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29754405)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GhL1L1%20affects%20cell%20fate%20specification%20by%20regulating%20GhPIN1-mediated%20auxin%20distribution&journal=Plant%20Biotechnol.%20J.&volume=17&pages=63-74&publication_year=2019&author=Xu%2CJ) 
    
22.  Deng, J. et al. GhTCE1–GhTCEE1 dimers regulate transcriptional reprogramming during wound-induced callus formation in cotton. _Plant Cell_ **34**, 4554–4568 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35972347)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9614502)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GhTCE1%E2%80%93GhTCEE1%20dimers%20regulate%20transcriptional%20reprogramming%20during%20wound-induced%20callus%20formation%20in%20cotton&journal=Plant%20Cell&volume=34&pages=4554-4568&publication_year=2022&author=Deng%2CJ) 
    
23.  Yuan, J. et al. GhRCD1 regulates cotton somatic embryogenesis by modulating the GhMYC3–GhMYB44–GhLBD18 transcriptional cascade. _New Phytol._ **240**, 207–223 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVKksrjF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37434324)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GhRCD1%20regulates%20cotton%20somatic%20embryogenesis%20by%20modulating%20the%20GhMYC3%E2%80%93GhMYB44%E2%80%93GhLBD18%20transcriptional%20cascade&journal=New%20Phytol.&volume=240&pages=207-223&publication_year=2023&author=Yuan%2CJ) 
    
24.  Guo, H. et al. Somatic embryogenesis critical initiation stage-specific mCHH hypomethylation reveals epigenetic basis underlying embryogenic redifferentiation in cotton. _Plant Biotechnol. J._ **18**, 1648–1650 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31925881)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7336376)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20embryogenesis%20critical%20initiation%20stage-specific%20mCHH%20hypomethylation%20reveals%20epigenetic%20basis%20underlying%20embryogenic%20redifferentiation%20in%20cotton&journal=Plant%20Biotechnol.%20J.&volume=18&pages=1648-1650&publication_year=2020&author=Guo%2CH) 
    
25.  Ge, X. et al. Efficient genotype-independent cotton genetic transformation and genome editing. _J. Integr. Plant Biol._ **65**, 907–917 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtVGgu7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36478145)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20genotype-independent%20cotton%20genetic%20transformation%20and%20genome%20editing&journal=J.%20Integr.%20Plant%20Biol.&volume=65&pages=907-917&publication_year=2023&author=Ge%2CX) 
    
26.  Liu, Y. et al. Cloning and preliminary verification of telomere-associated sequences in upland cotton. _Comp. Cytogenet._ **14**, 183–195 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32308926)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7154043)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cloning%20and%20preliminary%20verification%20of%20telomere-associated%20sequences%20in%20upland%20cotton&journal=Comp.%20Cytogenet.&volume=14&pages=183-195&publication_year=2020&author=Liu%2CY) 
    
27.  Wang, P. & Wang, F. A proposed metric set for evaluation of genome assembly quality. _Trends Genet._ **39**, 175–186 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivFSrtLrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36402623)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20proposed%20metric%20set%20for%20evaluation%20of%20genome%20assembly%20quality&journal=Trends%20Genet.&volume=39&pages=175-186&publication_year=2023&author=Wang%2CP&author=Wang%2CF) 
    
28.  Nurk, S. et al. The complete sequence of a human genome. _Science_ **376**, 44–53 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVyns7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35357919)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9186530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20complete%20sequence%20of%20a%20human%20genome&journal=Science&volume=376&pages=44-53&publication_year=2022&author=Nurk%2CS) 
    
29.  Vollger, M. R. et al. Long-read sequence and assembly of segmental duplications. _Nat. Methods_ **16**, 88–94 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisFCitLvK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30559433)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Long-read%20sequence%20and%20assembly%20of%20segmental%20duplications&journal=Nat.%20Methods&volume=16&pages=88-94&publication_year=2019&author=Vollger%2CMR) 
    
30.  Luo, S. et al. The cotton centromere contains a Ty3-Gypsy-like LTR retroelement. _PLoS ONE_ **7**, e35261 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmsFKjtL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22536361)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3334964)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20cotton%20centromere%20contains%20a%20Ty3-Gypsy-like%20LTR%20retroelement&journal=PLoS%20ONE&volume=7&publication_year=2012&author=Luo%2CS) 
    
31.  Gorinšek, B., Gubenšek, F. & Kordiš, D. A. Evolutionary genomics of chromoviruses in eukaryotes. _Mol. Biol. Evol._ **21**, 781–798 (2004).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14739248)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolutionary%20genomics%20of%20chromoviruses%20in%20eukaryotes&journal=Mol.%20Biol.%20Evol.&volume=21&pages=781-798&publication_year=2004&author=Gorin%C5%A1ek%2CB&author=Guben%C5%A1ek%2CF&author=Kordi%C5%A1%2CDA) 
    
32.  Naish, M. et al. The genetic and epigenetic landscape of the _Arabidopsis_ centromeres. _Science_ **374**, eabi7489 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34762468)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10164409)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genetic%20and%20epigenetic%20landscape%20of%20the%20Arabidopsis%20centromeres&journal=Science&volume=374&publication_year=2021&author=Naish%2CM) 
    
33.  Schmitz, R. J., Grotewold, E. & Stam, M. _Cis_\-regulatory sequences in plants: their importance, discovery, and future challenges. _Plant Cell_ **34**, 718–741 (2021).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8824567)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cis-regulatory%20sequences%20in%20plants%3A%20their%20importance%2C%20discovery%2C%20and%20future%20challenges&journal=Plant%20Cell&volume=34&pages=718-741&publication_year=2021&author=Schmitz%2CRJ&author=Grotewold%2CE&author=Stam%2CM) 
    
34.  Zhu, X. et al. Single-cell resolution analysis reveals the preparation for reprogramming the fate of stem cell niche in cotton lateral meristem. _Genome Biol._ **24**, 194 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVCjsb3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37626404)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10463415)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20resolution%20analysis%20reveals%20the%20preparation%20for%20reprogramming%20the%20fate%20of%20stem%20cell%20niche%20in%20cotton%20lateral%20meristem&journal=Genome%20Biol.&volume=24&publication_year=2023&author=Zhu%2CX) 
    
35.  Braybrook, S. A. & Harada, J. J. _LECs_ go crazy in embryo development. _Trends Plant Sci._ **13**, 624–630 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVCmsLjP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19010711)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LECs%20go%20crazy%20in%20embryo%20development&journal=Trends%20Plant%20Sci.&volume=13&pages=624-630&publication_year=2008&author=Braybrook%2CSA&author=Harada%2CJJ) 
    
36.  Ji, J. et al. _WOX4_ promotes procambial development. _Plant Physiol._ **152**, 1346–1356 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20044450)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832261)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WOX4%20promotes%20procambial%20development&journal=Plant%20Physiol.&volume=152&pages=1346-1356&publication_year=2009&author=Ji%2CJ) 
    
37.  Wang, F. et al. Chromatin accessibility dynamics and a hierarchical transcriptional regulatory network structure for plant somatic embryogenesis. _Dev. Cell_ **54**, 742–757 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFGit7%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32755547)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20accessibility%20dynamics%20and%20a%20hierarchical%20transcriptional%20regulatory%20network%20structure%20for%20plant%20somatic%20embryogenesis&journal=Dev.%20Cell&volume=54&pages=742-757&publication_year=2020&author=Wang%2CF) 
    
38.  Izhaki, A. & Bowman, J. L. KANADI and Class III HD-Zip gene families regulate embryo patterning and modulate auxin flow during embryogenesis in _Arabidopsi_s. _Plant Cell_ **19**, 495–508 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXktFGns7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17307928)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1867338)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KANADI%20and%20Class%20III%20HD-Zip%20gene%20families%20regulate%20embryo%20patterning%20and%20modulate%20auxin%20flow%20during%20embryogenesis%20in%20Arabidopsis&journal=Plant%20Cell&volume=19&pages=495-508&publication_year=2007&author=Izhaki%2CA&author=Bowman%2CJL) 
    
39.  Wang, G. et al. Development of an efficient and precise adenine base editor (ABE) with expanded target range in allotetraploid cotton (_Gossypium hirsutum_). _BMC Biol._ **20**, 45 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35164736)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8845244)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20of%20an%20efficient%20and%20precise%20adenine%20base%20editor%20%28ABE%29%20with%20expanded%20target%20range%20in%20allotetraploid%20cotton%20%28Gossypium%20hirsutum%29&journal=BMC%20Biol.&volume=20&publication_year=2022&author=Wang%2CG) 
    
40.  Li, C. et al. Targeted, random mutagenesis of plant genes with dual cytosine and adenine base editors. _Nat. Biotechnol._ **38**, 875–882 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFGltQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31932727)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeted%2C%20random%20mutagenesis%20of%20plant%20genes%20with%20dual%20cytosine%20and%20adenine%20base%20editors&journal=Nat.%20Biotechnol.&volume=38&pages=875-882&publication_year=2020&author=Li%2CC) 
    
41.  Xue, C. et al. Tuning plant phenotypes by precise, graded downregulation of gene expression. _Nat. Biotechnol._ **41**, 1758–1764 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXltFGhtLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36894598)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tuning%20plant%20phenotypes%20by%20precise%2C%20graded%20downregulation%20of%20gene%20expression&journal=Nat.%20Biotechnol.&volume=41&pages=1758-1764&publication_year=2023&author=Xue%2CC) 
    
42.  Xu, M., Du, Q., Tian, C., Wang, Y. & Jiao, Y. Stochastic gene expression drives mesophyll protoplast regeneration. _Sci. Adv._ **7**, eabg8466 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitVKkurbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34380624)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8357238)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stochastic%20gene%20expression%20drives%20mesophyll%20protoplast%20regeneration&journal=Sci.%20Adv.&volume=7&publication_year=2021&author=Xu%2CM&author=Du%2CQ&author=Tian%2CC&author=Wang%2CY&author=Jiao%2CY) 
    
43.  Zhang, L. et al. A high-quality apple genome assembly reveals the association of a retrotransposon and red fruit colour. _Nat. Commun._ **10**, 1494 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30940818)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6445120)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20high-quality%20apple%20genome%20assembly%20reveals%20the%20association%20of%20a%20retrotransposon%20and%20red%20fruit%20colour&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Zhang%2CL) 
    
44.  Qin, L. et al. High-efficient and precise base editing of C·G to T·A in the allotetraploid cotton (_Gossypium hirsutum_) genome using a modified CRISPR/Cas9 system. _Plant Biotechnol. J._ **18**, 45–56 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXntV2mtA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31116473)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-efficient%20and%20precise%20base%20editing%20of%20C%C2%B7G%20to%20T%C2%B7A%20in%20the%20allotetraploid%20cotton%20%28Gossypium%20hirsutum%29%20genome%20using%20a%20modified%20CRISPR%2FCas9%20system&journal=Plant%20Biotechnol.%20J.&volume=18&pages=45-56&publication_year=2020&author=Qin%2CL) 
    
45.  Jin, S. et al. Cytosine, but not adenine, base editors induce genome-wide off-target mutations in rice. _Science_ **364**, 292–295 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXns1CgtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30819931)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cytosine%2C%20but%20not%20adenine%2C%20base%20editors%20induce%20genome-wide%20off-target%20mutations%20in%20rice&journal=Science&volume=364&pages=292-295&publication_year=2019&author=Jin%2CS) 
    
46.  Hirano, H. et al. Structure and engineering of _Francisella novicida_ Cas9. _Cell_ **164**, 950–961 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XisFOhtrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26875867)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4899972)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20and%20engineering%20of%20Francisella%20novicida%20Cas9&journal=Cell&volume=164&pages=950-961&publication_year=2016&author=Hirano%2CH) 
    
47.  Huang, G. et al. Genome sequence of _Gossypium herbaceum_ and genome updates of _Gossypium arboreum_ and _Gossypium hirsutum_ provide insights into cotton A-genome evolution. _Nat. Genet._ **52**, 516–524 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXmvFylu74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32284579)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7203013)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20sequence%20of%20Gossypium%20herbaceum%20and%20genome%20updates%20of%20Gossypium%20arboreum%20and%20Gossypium%20hirsutum%20provide%20insights%20into%20cotton%20A-genome%20evolution&journal=Nat.%20Genet.&volume=52&pages=516-524&publication_year=2020&author=Huang%2CG) 
    
48.  Chen, Z. J. et al. Genomic diversifications of five Gossypium allopolyploid species and their impact on cotton improvement. _Nat. Genet._ **52**, 525–533 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsFSnur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32313247)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7203012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20diversifications%20of%20five%20Gossypium%20allopolyploid%20species%20and%20their%20impact%20on%20cotton%20improvement&journal=Nat.%20Genet.&volume=52&pages=525-533&publication_year=2020&author=Chen%2CZJ) 
    
49.  Sreedasyam, A. et al. Genome resources for three modern cotton lines guide future breeding efforts. _Nat. Plants_ **10**, 1039–1051 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38816498)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11208153)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20resources%20for%20three%20modern%20cotton%20lines%20guide%20future%20breeding%20efforts&journal=Nat.%20Plants&volume=10&pages=1039-1051&publication_year=2024&author=Sreedasyam%2CA) 
    
50.  Han, J. et al. Rapid proliferation and nucleolar organizer targeting centromeric retrotransposons in cotton. _Plant J._ **88**, 992–1005 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlKqsr3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27539015)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20proliferation%20and%20nucleolar%20organizer%20targeting%20centromeric%20retrotransposons%20in%20cotton&journal=Plant%20J.&volume=88&pages=992-1005&publication_year=2016&author=Han%2CJ) 
    
51.  Wick, R. R., Judd, L. M. & Holt, K. E. Performance of neural network basecalling tools for Oxford nanopore sequencing. _Genome Biol._ **20**, 129 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31234903)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6591954)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Performance%20of%20neural%20network%20basecalling%20tools%20for%20Oxford%20nanopore%20sequencing&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Wick%2CRR&author=Judd%2CLM&author=Holt%2CKE) 
    
52.  Hu, J. et al. NextDenovo: an efficient error correction and accurate assembly tool for noisy long reads. _Genome Biol._ **25**, 107 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38671502)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11046930)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NextDenovo%3A%20an%20efficient%20error%20correction%20and%20accurate%20assembly%20tool%20for%20noisy%20long%20reads&journal=Genome%20Biol.&volume=25&publication_year=2024&author=Hu%2CJ) 
    
53.  Hu, J., Fan, J., Sun, Z. & Liu, S. NextPolish: a fast and efficient genome polishing tool for long-read assembly. _Bioinformatics_ **36**, 2253–2255 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NextPolish%3A%20a%20fast%20and%20efficient%20genome%20polishing%20tool%20for%20long-read%20assembly&journal=Bioinformatics&volume=36&pages=2253-2255&publication_year=2019&author=Hu%2CJ&author=Fan%2CJ&author=Sun%2CZ&author=Liu%2CS) 
    
54.  Cheng, H., Concepcion, G. T., Feng, X., Zhang, H. & Li, H. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. _Nat. Methods_ **18**, 170–175 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1OntL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33526886)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7961889)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haplotype-resolved%20de%20novo%20assembly%20using%20phased%20assembly%20graphs%20with%20hifiasm&journal=Nat.%20Methods&volume=18&pages=170-175&publication_year=2021&author=Cheng%2CH&author=Concepcion%2CGT&author=Feng%2CX&author=Zhang%2CH&author=Li%2CH) 
    
55.  Kirov, I., Gilyok, M., Knyazev, A. & Fesenko, I. Pilot satellitome analysis of the model plant, _Physcomitrella patens_, revealed a transcribed and high-copy IGS related tandem repeat. _Comp. Cytogenet._ **12**, 493–513 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30588288)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6302065)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pilot%20satellitome%20analysis%20of%20the%20model%20plant%2C%20Physcomitrella%20patens%2C%20revealed%20a%20transcribed%20and%20high-copy%20IGS%20related%20tandem%20repeat&journal=Comp.%20Cytogenet.&volume=12&pages=493-513&publication_year=2018&author=Kirov%2CI&author=Gilyok%2CM&author=Knyazev%2CA&author=Fesenko%2CI) 
    
56.  Stovner, E. B. & Sætrom, P. epic2 efficiently finds diffuse domains in ChIP–seq data. _Bioinformatics_ **35**, 4392–4393 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFSnurbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30923821)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=epic2%20efficiently%20finds%20diffuse%20domains%20in%20ChIP%E2%80%93seq%20data&journal=Bioinformatics&volume=35&pages=4392-4393&publication_year=2019&author=Stovner%2CEB&author=S%C3%A6trom%2CP) 
    
57.  Marçais, G. & Kingsford, C. A fast, lock-free approach for efficient parallel counting of occurrences of _k_\-mers. _Bioinformatics_ **27**, 764 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21217122)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3051319)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&journal=Bioinformatics&volume=27&publication_year=2011&author=Mar%C3%A7ais%2CG&author=Kingsford%2CC) 
    
58.  Liu, J. et al. Gapless assembly of maize chromosomes using long-read technologies. _Genome Biol._ **21**, 121 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpvFCmu7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32434565)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7238635)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gapless%20assembly%20of%20maize%20chromosomes%20using%20long-read%20technologies&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Liu%2CJ) 
    
59.  Ramírez, F., Dündar, F., Diehl, S., Grüning, B. A. & Manke, T. deepTools: a flexible platform for exploring deep-sequencing data. _Nucleic Acids Res._ **42**, W187–W191 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24799436)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4086134)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=deepTools%3A%20a%20flexible%20platform%20for%20exploring%20deep-sequencing%20data&journal=Nucleic%20Acids%20Res.&volume=42&pages=W187-W191&publication_year=2014&author=Ram%C3%ADrez%2CF&author=D%C3%BCndar%2CF&author=Diehl%2CS&author=Gr%C3%BCning%2CBA&author=Manke%2CT) 
    
60.  Quinlan, A. R. BEDTools: the Swiss‐army tool for genome feature analysis. _Curr. Protoc. Bioinformatics_ **47**, 11.12. 1–11.12. 34 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25199790)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BEDTools%3A%20the%20Swiss%E2%80%90army%20tool%20for%20genome%20feature%20analysis&journal=Curr.%20Protoc.%20Bioinformatics&volume=47&pages=11.12.%201-11.12.%2034&publication_year=2014&author=Quinlan%2CAR) 
    
61.  Mikheenko, A., Bzikadze, A. V., Gurevich, A., Miga, K. H. & Pevzner, P. A. TandemTools: mapping long reads and assessing/improving assembly quality in extra-long tandem repeats. _Bioinformatics_ **36**, i75–i83 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1ymt73F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32657355)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7355294)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TandemTools%3A%20mapping%20long%20reads%20and%20assessing%2Fimproving%20assembly%20quality%20in%20extra-long%20tandem%20repeats&journal=Bioinformatics&volume=36&pages=i75-i83&publication_year=2020&author=Mikheenko%2CA&author=Bzikadze%2CAV&author=Gurevich%2CA&author=Miga%2CKH&author=Pevzner%2CPA) 
    
62.  Ou, S. et al. Benchmarking transposable element annotation methods for creation of a streamlined, comprehensive pipeline. _Genome Biol._ **20**, 275–292 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSntb3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31843001)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6913007)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Benchmarking%20transposable%20element%20annotation%20methods%20for%20creation%20of%20a%20streamlined%2C%20comprehensive%20pipeline&journal=Genome%20Biol.&volume=20&pages=275-292&publication_year=2019&author=Ou%2CS) 
    
63.  Novák, P., Neumann, P., Pech, J., Steinhaisl, J. & Macas, J. RepeatExplorer: a Galaxy-based web server for genome-wide characterization of eukaryotic repetitive elements from next-generation sequence reads. _Bioinformatics_ **29**, 792–793 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23376349)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RepeatExplorer%3A%20a%20Galaxy-based%20web%20server%20for%20genome-wide%20characterization%20of%20eukaryotic%20repetitive%20elements%20from%20next-generation%20sequence%20reads&journal=Bioinformatics&volume=29&pages=792-793&publication_year=2013&author=Nov%C3%A1k%2CP&author=Neumann%2CP&author=Pech%2CJ&author=Steinhaisl%2CJ&author=Macas%2CJ) 
    
64.  Vollger, M. R., Kerpedjiev, P., Phillippy, A. M. & Eichler, E. E. StainedGlass: interactive visualization of massive tandem repeat structures with identity heatmaps. _Bioinformatics_ **38**, 2049–2051 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFygs7%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35020798)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8963321)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=StainedGlass%3A%20interactive%20visualization%20of%20massive%20tandem%20repeat%20structures%20with%20identity%20heatmaps&journal=Bioinformatics&volume=38&pages=2049-2051&publication_year=2022&author=Vollger%2CMR&author=Kerpedjiev%2CP&author=Phillippy%2CAM&author=Eichler%2CEE) 
    
65.  Peng, R. et al. Evolutionary divergence of duplicated genomes in newly described allotetraploid cottons. _Proc. Natl Acad. Sci. USA_ **119**, e2208496119 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFKltbfK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36122204)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9522333)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolutionary%20divergence%20of%20duplicated%20genomes%20in%20newly%20described%20allotetraploid%20cottons&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=119&publication_year=2022&author=Peng%2CR) 
    
66.  Gel, B. & Serra, E. karyoploteR: an R/Bioconductor package to plot customizable genomes displaying arbitrary data. _Bioinformatics_ **33**, 3088–3090 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlWksL3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28575171)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5870550)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=karyoploteR%3A%20an%20R%2FBioconductor%20package%20to%20plot%20customizable%20genomes%20displaying%20arbitrary%20data&journal=Bioinformatics&volume=33&pages=3088-3090&publication_year=2017&author=Gel%2CB&author=Serra%2CE) 
    
67.  Hu, L. et al. The chromosome-scale reference genome of black pepper provides insight into piperine biosynthesis. _Nat. Commun._ **10**, 4702 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31619678)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6795880)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20chromosome-scale%20reference%20genome%20of%20black%20pepper%20provides%20insight%20into%20piperine%20biosynthesis&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Hu%2CL) 
    
68.  Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. _Proc. Natl Acad. Sci. USA_ **117**, 9451–9457 (2020).
    
69.  Bao, W., Kojima, K. K. & Kohany, O. Repbase Update, a database of repetitive elements in eukaryotic genomes. _Mob. DNA_ **6**, 11 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26045719)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4455052)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Repbase%20Update%2C%20a%20database%20of%20repetitive%20elements%20in%20eukaryotic%20genomes&journal=Mob.%20DNA&volume=6&publication_year=2015&author=Bao%2CW&author=Kojima%2CKK&author=Kohany%2CO) 
    
70.  Paterson, A. H. et al. Repeated polyploidization of _Gossypium_ genomes and the evolution of spinnable cotton fibres. _Nature_ **492**, 423–427 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVKns7jL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23257886)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Repeated%20polyploidization%20of%20Gossypium%20genomes%20and%20the%20evolution%20of%20spinnable%20cotton%20fibres&journal=Nature&volume=492&pages=423-427&publication_year=2012&author=Paterson%2CAH) 
    
71.  Li, H. Minimap2: pairwise alignment for nucleotide sequences. _Bioinformatics_ **34**, 3094–3100 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVamu73J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29750242)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6137996)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Minimap2%3A%20pairwise%20alignment%20for%20nucleotide%20sequences&journal=Bioinformatics&volume=34&pages=3094-3100&publication_year=2018&author=Li%2CH) 
    
72.  Goel, M., Sun, H., Jiao, W.-B. & Schneeberger, K. SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies. _Genome Biol._ **20**, 277 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31842948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6913012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SyRI%3A%20finding%20genomic%20rearrangements%20and%20local%20sequence%20differences%20from%20whole-genome%20assemblies&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Goel%2CM&author=Sun%2CH&author=Jiao%2CW-B&author=Schneeberger%2CK) 
    
73.  Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of _Drosophila melanogaster_ strain _w_1118; _iso_\-2; _iso_\-3. _Fly (Austin)_ **6**, 80–92 (2012).
    
74.  Giordano, F., Stammnitz, M. R., Murchison, E. P. & Ning, Z. scanPAV: a pipeline for extracting presence–absence variations in genome pairs. _Bioinformatics_ **34**, 3022–3024 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXptVGqsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29608694)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6129304)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=scanPAV%3A%20a%20pipeline%20for%20extracting%20presence%E2%80%93absence%20variations%20in%20genome%20pairs&journal=Bioinformatics&volume=34&pages=3022-3024&publication_year=2018&author=Giordano%2CF&author=Stammnitz%2CMR&author=Murchison%2CEP&author=Ning%2CZ) 
    
75.  Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at [https://arxiv.org/abs/1303.3997](https://arxiv.org/abs/1303.3997) (2013).
    
76.  Li, H. et al. The sequence alignment/map format and SAMtools. _Bioinformatics_ **25**, 2078–2079 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19505943)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&journal=Bioinformatics&volume=25&pages=2078-2079&publication_year=2009&author=Li%2CH) 
    
77.  McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. _Genome Res._ **20**, 1297–1303 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFeru7jM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20644199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&journal=Genome%20Res.&volume=20&pages=1297-1303&publication_year=2010&author=McKenna%2CA) 
    
78.  Birney, E., Clamp, M. & Durbin, R. GeneWise and genomewise. _Genome Res._ **14**, 988–995 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjvFyku7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15123596)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC479130)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GeneWise%20and%20genomewise&journal=Genome%20Res.&volume=14&pages=988-995&publication_year=2004&author=Birney%2CE&author=Clamp%2CM&author=Durbin%2CR) 
    
79.  Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. _Bioinformatics_ **30**, 2114–2120 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24695404)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&journal=Bioinformatics&volume=30&pages=2114-2120&publication_year=2014&author=Bolger%2CAM&author=Lohse%2CM&author=Usadel%2CB) 
    
80.  Kim, D., Langmead, B. & Salzberg, S. L. HISAT: a fast spliced aligner with low memory requirements. _Nat. Methods_ **12**, 357–360 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvFOnsL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25751142)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655817)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HISAT%3A%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&journal=Nat.%20Methods&volume=12&pages=357-360&publication_year=2015&author=Kim%2CD&author=Langmead%2CB&author=Salzberg%2CSL) 
    
81.  Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. _Nat. Biotechnol._ **33**, 290–295 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXivFais70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25690850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643835)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&journal=Nat.%20Biotechnol.&volume=33&pages=290-295&publication_year=2015&author=Pertea%2CM) 
    
82.  Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._ **15**, 550 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25516281)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Love%2CMI&author=Huber%2CW&author=Anders%2CS) 
    
83.  Langfelder, P. & Horvath, S. WGCNA: an R package for weighted correlation network analysis. _BMC Bioinformatics_ **9**, 559 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19114008)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2631488)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WGCNA%3A%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis&journal=BMC%20Bioinformatics&volume=9&publication_year=2008&author=Langfelder%2CP&author=Horvath%2CS) 
    
84.  Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. _Genome Res._ **13**, 2498–2504 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXovFWrtr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14597658)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC403769)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cytoscape%3A%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&journal=Genome%20Res.&volume=13&pages=2498-2504&publication_year=2003&author=Shannon%2CP) 
    
85.  Bu, D. et al. KOBAS-i: intelligent prioritization and exploratory visualization of biological functions for gene enrichment analysis. _Nucleic Acids Res._ **49**, W317–W325 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFWgtbnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34086934)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KOBAS-i%3A%20intelligent%20prioritization%20and%20exploratory%20visualization%20of%20biological%20functions%20for%20gene%20enrichment%20analysis&journal=Nucleic%20Acids%20Res.&volume=49&pages=W317-W325&publication_year=2021&author=Bu%2CD) 
    
86.  Grandi, F. C., Modi, H., Kampman, L. & Corces, M. R. Chromatin accessibility profiling by ATAC–seq. _Nat. Protoc._ **17**, 1518–1552 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1entrzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35478247)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9189070)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20accessibility%20profiling%20by%20ATAC%E2%80%93seq&journal=Nat.%20Protoc.&volume=17&pages=1518-1552&publication_year=2022&author=Grandi%2CFC&author=Modi%2CH&author=Kampman%2CL&author=Corces%2CMR) 
    
87.  Chen, S., Zhou, Y., Chen, Y. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. _Bioinformatics_ **34**, i884–i890 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30423086)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6129281)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=fastp%3A%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&journal=Bioinformatics&volume=34&pages=i884-i890&publication_year=2018&author=Chen%2CS&author=Zhou%2CY&author=Chen%2CY&author=Gu%2CJ) 
    
88.  Faust, G. G. & Hall, I. M. SAMBLASTER: fast duplicate marking and structural variant read extraction. _Bioinformatics_ **30**, 2503–2505 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsleiurvK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24812344)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147885)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SAMBLASTER%3A%20fast%20duplicate%20marking%20and%20structural%20variant%20read%20extraction&journal=Bioinformatics&volume=30&pages=2503-2505&publication_year=2014&author=Faust%2CGG&author=Hall%2CIM) 
    
89.  Zhang, Y. et al. Model-based analysis of ChIP–seq (MACS). _Genome Biol._ **9**, R137 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18798982)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Model-based%20analysis%20of%20ChIP%E2%80%93seq%20%28MACS%29&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Zhang%2CY) 
    
90.  Li, Q., Brown, J. B., Huang, H. & Bickel, P. J. Measuring reproducibility of high-throughput experiments. Preprint at [https://arxiv.org/abs/1110.4705](https://arxiv.org/abs/1110.4705) (2011).
    
91.  Heinz, S. et al. Simple combinations of lineage-determining transcription factors prime _cis_\-regulatory elements required for macrophage and B cell identities. _Mol. Cell_ **38**, 576–589 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXns1SlsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20513432)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898526)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Simple%20combinations%20of%20lineage-determining%20transcription%20factors%20prime%20cis-regulatory%20elements%20required%20for%20macrophage%20and%20B%20cell%20identities&journal=Mol.%20Cell&volume=38&pages=576-589&publication_year=2010&author=Heinz%2CS) 
    
92.  Machlab, D. et al. monaLisa: an R/Bioconductor package for identifying regulatory motifs. _Bioinformatics_ **38**, 2624–2625 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35199152)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9048699)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=monaLisa%3A%20an%20R%2FBioconductor%20package%20for%20identifying%20regulatory%20motifs&journal=Bioinformatics&volume=38&pages=2624-2625&publication_year=2022&author=Machlab%2CD) 
    
93.  Castro-Mondragon, J. A. et al. JASPAR 2022: the 9th release of the open-access database of transcription factor binding profiles. _Nucleic Acids Res._ **50**, D165–D173 (2021).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8728201)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=JASPAR%202022%3A%20the%209th%20release%20of%20the%20open-access%20database%20of%20transcription%20factor%20binding%20profiles&journal=Nucleic%20Acids%20Res.&volume=50&pages=D165-D173&publication_year=2021&author=Castro-Mondragon%2CJA) 
    
94.  Tan, G. & Lenhard, B. TFBSTools: an R/bioconductor package for transcription factor binding site analysis. _Bioinformatics_ **32**, 1555–1556 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhslWku73L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26794315)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4866524)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TFBSTools%3A%20an%20R%2Fbioconductor%20package%20for%20transcription%20factor%20binding%20site%20analysis&journal=Bioinformatics&volume=32&pages=1555-1556&publication_year=2016&author=Tan%2CG&author=Lenhard%2CB) 
    
95.  Concordet, J.-P. & Haeussler, M. CRISPOR: intuitive guide selection for CRISPR/Cas9 genome editing experiments and screens. _Nucleic Acids Res._ **46**, W242–W245 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyqs7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29762716)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030908)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CRISPOR%3A%20intuitive%20guide%20selection%20for%20CRISPR%2FCas9%20genome%20editing%20experiments%20and%20screens&journal=Nucleic%20Acids%20Res.&volume=46&pages=W242-W245&publication_year=2018&author=Concordet%2CJ-P&author=Haeussler%2CM) 
    
96.  Xu, Z. Scripts used in ‘Genome assembly of two allotetraploid cotton germplasms reveals mechanisms of somatic embryogenesis and enables precise genome editing’. _Zenodo_ [https://doi.org/10.5281/zenodo.15035095](https://doi.org/10.5281/zenodo.15035095) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02258-3?format=refman&flavour=references)

Acknowledgements
----------------

The study was supported by grants from the National Science Fund for Distinguished Young Scholars (32325039 to S.J.) and the Young Scientists Fund under the National Natural Science Foundation of China (32201856 to Z.X.). The computations in this paper were run on the bioinformatics computing platform of the National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University.

Author information
------------------

Author notes

1.  These authors contributed equally: Zhongping Xu, Guanying Wang.
    

### Authors and Affiliations

1.  National Key Laboratory of Crop Genetic Improvement, Hubei Hongshan Laboratory, Huazhong Agricultural University, Wuhan, China
    
    Zhongping Xu, Guanying Wang, Xiangqian Zhu, Ruipeng Wang, Longfu Zhu, Lili Tu, Maojun Wang, Xianlong Zhang & Shuangxia Jin
    
2.  Research Base, Anyang Institute of Technology, State Key Laboratory of Cotton Biology, Anyang, China
    
    Yuling Liu & Renhai Peng
    
3.  Department of Biosciences, Durham University, Durham, UK
    
    Keith Lindsey
    

Authors

1.  Zhongping Xu
    
    [View author publications](https://www.nature.com/search?author=Zhongping%20Xu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zhongping%20Xu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zhongping%20Xu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Guanying Wang
    
    [View author publications](https://www.nature.com/search?author=Guanying%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Guanying%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Guanying%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Xiangqian Zhu
    
    [View author publications](https://www.nature.com/search?author=Xiangqian%20Zhu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiangqian%20Zhu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiangqian%20Zhu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Ruipeng Wang
    
    [View author publications](https://www.nature.com/search?author=Ruipeng%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ruipeng%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ruipeng%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Longfu Zhu
    
    [View author publications](https://www.nature.com/search?author=Longfu%20Zhu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Longfu%20Zhu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Longfu%20Zhu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Lili Tu
    
    [View author publications](https://www.nature.com/search?author=Lili%20Tu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lili%20Tu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lili%20Tu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Yuling Liu
    
    [View author publications](https://www.nature.com/search?author=Yuling%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuling%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuling%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Renhai Peng
    
    [View author publications](https://www.nature.com/search?author=Renhai%20Peng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Renhai%20Peng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Renhai%20Peng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Keith Lindsey
    
    [View author publications](https://www.nature.com/search?author=Keith%20Lindsey)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Keith%20Lindsey) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Keith%20Lindsey%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Maojun Wang
    
    [View author publications](https://www.nature.com/search?author=Maojun%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maojun%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maojun%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Xianlong Zhang
    
    [View author publications](https://www.nature.com/search?author=Xianlong%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xianlong%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xianlong%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Shuangxia Jin
    
    [View author publications](https://www.nature.com/search?author=Shuangxia%20Jin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shuangxia%20Jin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shuangxia%20Jin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

S.J. and X. Zhang designed and supervised the research. Z.X. and G.W. collected materials for genome and transcriptome sequencing. Z.X. and G.W. collected materials for ATAC–seq. Z.X. and R.W. performed bioinformatics analysis. Y.L. and R.P. performed the FISH experiment. G.W. and X.Z. performed the gene editing experiments. M.W., L.T. and L.Z. contributed to the project discussion. Z.X. and G.W. wrote the manuscript with input from all other authors. S.J., K.L., X. Zhang and M.W. edited the manuscript. All authors have read and approved the manuscript.

### Corresponding authors

Correspondence to [Maojun Wang](mailto:mjwang@mail.hzau.edu.cn), [Xianlong Zhang](mailto:xlzhang@mail.hzau.edu.cn) or [Shuangxia Jin](mailto:jsx@mail.hzau.edu.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Amanda Hulse-Kemp, Jinsheng Lai and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Genomic characterization of Jin668 and YZ1.](https://www.nature.com/articles/s41588-025-02258-3/figures/8)

The k-mer (31-mer) frequency distribution was estimated by GenomeScope2.0 analysis of Jin668 (**a**) and YZ1 (**b**) genome. **c**, The distribution of telomeres, centromeres, and gaps regions in the genomes of Jin668 and YZ1.

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM7)

### [Extended Data Fig. 2 Completeness and accuracy validation of chromosome arms and centromeres in the Jin668 genome.](https://www.nature.com/articles/s41588-025-02258-3/figures/9)

**a**, Assessment of the Jin668 genome assembly using LTR Assembly Index (LAI). The x-axis lists chromosomes and y-axis represents LAI values. The red dashed line represents the mean value. **b**,**c**, Hi-C map of the Jin668 genome showing genome- and chromosome-wide all-by-all interactions. **d**, Bionano _de novo_ assembly contigs were mapped to the Jin668 reference assembly. **e**, The distribution of QV values calculated from Merqury and Merfin on different chromosomes. **f**, The distribution of different types of unique k-mers along the centromeric regions in chromosomes of Jin668. Each bar shows the number of different types of k-mers in a bin of length 20 Kb. The blue bars represent single-clump k-mers, which suggest a good base-level quality. While the orange (multiple-clumps) and green (no-clumps) bars suggest a low base-level quality in the region. The x-axis unit is megabases (Mb), and the y-axis represents values multiplied by 10³. **g**, NucFreq plot of centromeric regions in chromosomes of Jin668. HiFi coverage depth (black) along with secondary allele frequency (red) for all centromeres and surrounding regions. The x-axis represents chromosome positions in megabases (Mb), and the y-axis shows HiFi depth, ranging from 0 to 100.

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM8)

### [Extended Data Fig. 3 Segmental duplication (SD) content of the Jin668 genome.](https://www.nature.com/articles/s41588-025-02258-3/figures/10)

**a**,**b**, The intrachromosome SDs with a length greater or less than 5 kb. **c**, Comparison of SD length and identity in different regions of the genome. The identity (top) and length (bottom) of SDs across commonly delineated regions of the genome (colors).

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM9)

### [Extended Data Fig. 4 The percentage of AT/GC base composition, TEs, 5mC DNA methylation, histone modification and 3D genome architecture within the centromeres.](https://www.nature.com/articles/s41588-025-02258-3/figures/11)

Quantification of genomic features plotted along chromosome arms that were proportionally scaled between telomeres (TEL) and centromere midpoints (CEN).

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM10)

### [Extended Data Fig. 5 A comparative genome analysis of the Jin668 versus TM-1, YZ1, and ZM24.](https://www.nature.com/articles/s41588-025-02258-3/figures/12)

**a**, Genome-wide syntenic relationships among At and Dt subgenomes in four Upland cotton accessions relative to the A-genome-like Ga (A2 genome) and D-genome-like Gr (D5 genome). **b**, The distribution of variation density between Jin668 versus TM-1, YZ1, and ZM24 genomes. **c**, The distribution of SE-related genes and gene density, as well as genomic variations from Fig.1, on the chromosomes of the Jin668 genome. The circos plot from outside to inside shows: chromosome, SE-related genes, gene density in Jin668, SNPs, InDels, PAVs, inversions and translocation density between Jin668 with TM-1. **d**, The correlation between SNPs, InDels, PAVs, inversions and translocations with SE-related genes. The significance was tested using the overlapPermTest function in regioneR ([https://www.bioconductor.org/packages/devel/bioc/html/regioneR.html](https://www.bioconductor.org/packages/devel/bioc/html/regioneR.html)), which calls permTest with the appropriate parameters to perform the permutation test. SE, somatic embryogenesis.

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM11)

### [Extended Data Fig. 6 Sample preparation, sequencing, and quality evaluation of ATAC library.](https://www.nature.com/articles/s41588-025-02258-3/figures/13)

**a**, Schematic outline of genome-wide ATAC–seq and RNA-seq assays and time points of sample collection with two independent biological replicates, as well as the roadmap of an integration analysis. HAI, hours after inoculation. **b**, The principal component plots of ATAC–seq data sequenced from Jin668, YZ1, TM-1 and ZB1092, respectively. Color code is shown. Each dot represents one sample. **c**, The genome-wide distribution of ATAC–seq peaks for Jin668, YZ1, TM-1 and ZB1092. Window size: TSS ± 3.0 Kb. **d**, The distribution of SPOT values for all samples from Jin668, YZ1, TM-1 and ZB1092, respectively. **e**, The distribution of FRiP values for all samples from Jin668, YZ1, TM-1 and ZB1092, respectively. **f**, ATAC–seq profiling in Jin668, YZ1, TM-1 and ZB1092. Left, the number statistics of chromatin accessibility peaks on each chromosome. Middle, the bar chart shows the total number of peaks detected at the corresponding induction time point. Right, the pie chart illustrates the percentage distribution of peaks across various genomic regions.

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM12)

### [Extended Data Fig. 7 Identification of SE-related genes.](https://www.nature.com/articles/s41588-025-02258-3/figures/14)

Identification of SE-related genes based on genome information, chromatin accessibility and gene expression change trend at different time points of SE stage, and other external data (published literature and data generated in our laboratory earlier, etc.).

### [Extended Data Fig. 8 _AGL_ promotes cotton regeneration.](https://www.nature.com/articles/s41588-025-02258-3/figures/15)

**a**, Transcription and epigenetic tracks for _AGL_. Transcription data shown as mean ± s.d. of 3 biological replicates. **b**, The CPR of CRISPR edited, overexpression and control lines at 60 days post-induction. Data are represented as mean ± s.d. Differences between groups were evaluated by two-sided Student’s _t_\-test, \*_p_ < 0.05, \*\*_p_ < 0.01, \*\*\*_p_ < 0.001. n = 3 independent biological replicates. The P7N was used as control. **c**, The paraffin sections of CRISPR edited, overexpression and control lines for Jin668 and TM-1 at different days post-induction. Scale bars represent 100 μm.

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM13)

### [Extended Data Fig. 9 Genetic transformation experiments demonstrate that the Jin668 genome enhances the accuracy of sgRNA design for CRISPR gene editing.](https://www.nature.com/articles/s41588-025-02258-3/figures/16)

**a**, The sequence homology of a putative sgRNA (present in Jin668 but absent in TM-1) between Jin668 and TM-1. **b**,**c**, The mutational landscape of the genomic regions corresponding to targets sgRNA within the callus of Jin668 (**b**) and TM-1 (**c**) after edited using CRISPR/Cas9. The percentage signifies the ratio of reads corresponding to this mutation type to the total read count.

### [Extended Data Fig. 10 Gene expression dynamics across 15 tissues in Jin668 and YZ1.](https://www.nature.com/articles/s41588-025-02258-3/figures/17)

**a**, The profile of genes expressed in all 15 tissues of Jin668 (left) and YZ1 (right). **b**, Statistics on the number of genes co-expressed in multiple tissues of Jin668 (left) and YZ1 (right). **c**, Statistics on the number of genes specifically expressed in each tissue of Jin668 (left) and YZ1 (right). Each tissue is represented by the number corresponding to that shown in the graphic illustration in **a**.

[Source data](https://www.nature.com/articles/s41588-025-02258-3#MOESM14)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM1_ESM.pdf)

Supplementary Figs. 1–31, Supplementary Notes 1–13 and Supplementary Methods.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM2_ESM.pdf)

### [Supplementary Tables 1–35](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM3_ESM.xlsx)

Supplementary Tables 1–35.

Source data
-----------

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM4_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM5_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM7_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM8_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM9_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM12_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM13_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02258-3/MediaObjects/41588_2025_2258_MOESM14_ESM.xlsx)

Statistical source data.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Genome%20assembly%20of%20two%20allotetraploid%20cotton%20germplasms%20reveals%20mechanisms%20of%20somatic%20embryogenesis%20and%20enables%20precise%20genome%20editing&author=Zhongping%20Xu%20et%20al&contentID=10.1038%2Fs41588-025-02258-3&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-07-22&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02258-3)

### Cite this article

Xu, Z., Wang, G., Zhu, X. _et al._ Genome assembly of two allotetraploid cotton germplasms reveals mechanisms of somatic embryogenesis and enables precise genome editing. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02258-3

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02258-3?format=refman&flavour=citation)

-   Received: 21 August 2023
    
-   Accepted: 05 June 2025
    
-   Published: 22 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02258-3
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
