> id: https://www.nature.com/articles/s41588-025-02268-1

> link: https://www.nature.com/articles/s41588-025-02268-1

> title: Massively parallel immunopeptidome by DNA sequencing provides insights into cancer antigen presentation

# Massively parallel immunopeptidome by DNA sequencing provides insights into cancer antigen presentation
_Published on Mon, 28 Jul 2025 00:00:00 GMT_

Abstract
--------

Human leukocyte antigens (HLAs) are encoded by the most polymorphic genes in the human genome. HLA class I alleles control antigen presentation for T cell recognition, which is pivotal for autoimmunity, infectious diseases and cancer. Current knowledge of HLA-bound peptides is limited, skewed and falls short of population-wide HLA binding profiles for high-value targets. Here we present ESCAPE-seq (enhanced single-chain antigen presentation sequencing), a massively parallel platform for comprehensive screening of class I HLA–peptide combinations for antigen presentation via deep DNA sequencing. ESCAPE-seq demonstrates programmability, high throughput, sensitivity and nominated viral and cancer epitopes. We simultaneously assessed over 75,000 peptide–HLA combinations, revealing broadly presented epitopes from oncogenic driver mutations and fusions across diverse _HLA-A_, _HLA-B_ and _HLA-C_ alleles that cover 90% of the human population. We further identified epitopes that are differentially presented, comparing oncogenic hotspot mutations versus wild type. ESCAPE-seq enables one-shot population-wide antigen presentation discovery, offering insights into HLA specificity and immune recognition of genomic mutations.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-023-05496-5/MediaObjects/42003_2023_5496_Fig1_HTML.png)

### [HLA allele-calling using multi-ancestry whole-exome sequencing from the UK Biobank identifies 129 novel associations in 11 autoimmune diseases](https://www.nature.com/articles/s42003-023-05496-5?fromPaywallRec=false)

Article Open access 03 November 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-02106-4/MediaObjects/41598_2021_2106_Fig1_HTML.png)

### [_HLA-G_ genetic diversity and evolutive aspects in worldwide populations](https://www.nature.com/articles/s41598-021-02106-4?fromPaywallRec=false)

Article Open access 29 November 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs43018-021-00226-4/MediaObjects/43018_2021_226_Fig1_HTML.png)

### [Negative trade-off between neoantigen repertoire breadth and the specificity of HLA-I molecules shapes antitumor immunity](https://www.nature.com/articles/s43018-021-00226-4?fromPaywallRec=false)

Article 08 July 2021

Main
----

The major histocompatibility complex (MHC) is the most polymorphic region of the human genome[1](https://www.nature.com/articles/s41588-025-02268-1#ref-CR1 "Shiina, T., Hosomichi, K., Inoko, H. & Kulski, J. K. The HLA genomic loci map: expression, interaction, diversity and disease. J. Hum. Genet. 54, 15–39 (2009)."). MHC underlies the central event in adaptive immunity termed antigen presentation[2](https://www.nature.com/articles/s41588-025-02268-1#ref-CR2 "Pamer, E. & Cresswell, P. Mechanisms of MHC class I–restricted antigen processing. Annu. Rev. Immunol. 16, 323–358 (1998)."). Every cell in the body presents bits of its proteome as short peptides in the cleft of MHC proteins on the cell surface, which allows T cells to determine whether a cell is endogenous (‘self’) or foreign, has been infected or became cancerous. In humans, MHC class I (MHC-1) is known as HLA class I (HLA-I), encoded by _HLA-A_, _HLA-B_ and _HLA-C_ genes, which present antigen epitopes to CD8+ T cells to eradicate pathogen-infected cells and cancer cells[2](https://www.nature.com/articles/s41588-025-02268-1#ref-CR2 "Pamer, E. & Cresswell, P. Mechanisms of MHC class I–restricted antigen processing. Annu. Rev. Immunol. 16, 323–358 (1998)."). HLA loci are the most prominent signal in any genome-wide association study of autoimmune and infectious diseases[3](https://www.nature.com/articles/s41588-025-02268-1#ref-CR3 "Gutierrez-Arcelus, M., Rich, S. S. & Raychaudhuri, S. Autoimmune diseases – connecting risk alleles with molecular traits of the immune system. Nat. Rev. Genet. 17, 160–174 (2016)."). Acquisition of HLA alleles has been used to track ancient human population migration and the selection pressures in different continents[4](https://www.nature.com/articles/s41588-025-02268-1#ref-CR4 "Parham, P. & Ohta, T. Population biology of antigen presentation by MHC class I molecules. Science 272, 67–74 (1996)."). A decades-long goal of the entire field has been to achieve a population-wide understanding of the functional HLA binding preferences across diverse HLAs[4](https://www.nature.com/articles/s41588-025-02268-1#ref-CR4 "Parham, P. & Ohta, T. Population biology of antigen presentation by MHC class I molecules. Science 272, 67–74 (1996).").

In HLA-I antigen presentation, peptide fragments are sampled from the cellular proteome through proteolysis, transported to the endoplasmic reticulum (ER) via the TAP complex, trimmed into 8–10 amino acids when necessary and loaded onto HLA-I molecules upon high-affinity binding. The stable peptide–HLA (pHLA) complex is then transported to the cell surface for subsequent T cell recognition[5](https://www.nature.com/articles/s41588-025-02268-1#ref-CR5 "Neefjes, J., Jongsma, M. L. M., Paul, P. & Bakke, O. Towards a systems understanding of MHC class I and MHC class II antigen presentation. Nat. Rev. Immunol. 11, 823–836 (2011)."). The set of peptides presented in HLA-I is termed the immunopeptidome. Mass spectrometry (MS) is instrumental to determine the immunopeptidome and to define fundamental rules of antigen presentation, with recent advances in HLA monoallelic MS approaches enabling systematic characterization of the allele-specific peptide repertoire. However, current immunopeptidome data remain highly skewed[6](https://www.nature.com/articles/s41588-025-02268-1#ref-CR6 "Klatt, M. G. et al. Solving an MHC allele–specific bias in the reported immunopeptidome. JCI Insight 5, e141264 (2020)."), as MS-based approaches still face major challenges including requirement for large sample input, difficulty in deconvoluting multi-allele HLAs and limited sensitivity in detecting low-abundance but clinically relevant peptides such as those from pathogens and mutated oncoproteins[7](https://www.nature.com/articles/s41588-025-02268-1#ref-CR7 "Bassani-Sternberg, M. & Gfeller, D. Unsupervised HLA peptidome deconvolution improves ligand prediction accuracy and predicts cooperative effects in peptide-HLA interactions. J. Immunol. 197, 2492–2499 (2016)."),[8](https://www.nature.com/articles/s41588-025-02268-1#ref-CR8 "Pearson, H. et al. MHC class I-associated peptides derive from selective regions of the human genome. J. Clin. Invest. 126, 4690–4701 (2016)."). Computational prediction algorithms such as NetMHC offer rapid and scalable in silico antigen identification but show decreased reliability for HLA alleles with insufficient training datasets, especially for _HLA-C_ alleles with lower surface expression compared with _HLA-A_ or _HLA-__B_ alleles[9](https://www.nature.com/articles/s41588-025-02268-1#ref-CR9 "McCutcheon, J. A., Gumperz, J., Smith, K. D., Lutz, C. T. & Parham, P. Low HLA-C expression at cell surfaces correlates with increased turnover of heavy chain mRNA. J. Exp. Med. 181, 2085–2095 (1995)."),[10](https://www.nature.com/articles/s41588-025-02268-1#ref-CR10 "Jurtz, V. et al. NetMHCpan-4.0: improved peptide-MHC class I interaction predictions integrating eluted ligand and peptide binding affinity data. J. Immunol. 199, 3360–3368 (2017).").

Existing HLA-I immunopeptidome data are highly biased toward certain well-studied alleles such as _HLA-A2_, while many HLA alleles associated with disease but more frequent in non-European ancestry groups have scant biochemical characterization (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)). Such biases also lead to substantial inequities in clinical trial eligibility[11](https://www.nature.com/articles/s41588-025-02268-1#ref-CR11 "Olivier, T., Haslam, A., Tuia, J. & Prasad, V. Eligibility for human leukocyte antigen-based therapeutics by race and ethnicity. JAMA Netw. Open 6, e2338612 (2023)."). In particular, targeting HLA-C may enable population coverage with a smaller number of alleles, but the paucity of antigen binding data has limited this strategy. Other methods, such as in vitro peptide binding and T cell stimulation assays, have limited throughput due to the requirement and cost of individual peptide synthesis[12](https://www.nature.com/articles/s41588-025-02268-1#ref-CR12 "Schmittel, A., Keilholz, U. & Scheibenbogen, C. Evaluation of the interferon-γ ELISPOT-assay for quantification of peptide specific T lymphocytes from peripheral blood. J. Immunol. Methods 210, 167–174 (1997)."),[13](https://www.nature.com/articles/s41588-025-02268-1#ref-CR13 "Darwish, M. et al. High-throughput identification of conditional MHCI ligands and scaled-up production of conditional MHCI complexes. Protein Sci. 30, 1169–1183 (2021)."),[14](https://www.nature.com/articles/s41588-025-02268-1#ref-CR14 "Haj, A. K. et al. High-throughput identification of MHC class I binding peptides using an ultradense peptide array. J. Immunol. 204, 1689–1696 (2020)."). Considerable efforts have been invested over time in enhancing throughput, as evidenced by the recent advancements in methodologies such as EpiScan and time-resolved fluorescence resonance energy transfer (TR-FRET) with parallel reading, but scalability remains limited[15](https://www.nature.com/articles/s41588-025-02268-1#ref-CR15 "Gurung, H. R. et al. Systematic discovery of neoepitope-HLA pairs for neoantigens shared among patients and tumor types. Nat. Biotechnol. 42, 1107–1117 (2024)."),[16](https://www.nature.com/articles/s41588-025-02268-1#ref-CR16 "Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. Nat. Biotechnol. 41, 980–992 (2023)."). For instance, EpiScan does not work for _HLA-C_ alleles[12](https://www.nature.com/articles/s41588-025-02268-1#ref-CR12 "Schmittel, A., Keilholz, U. & Scheibenbogen, C. Evaluation of the interferon-γ ELISPOT-assay for quantification of peptide specific T lymphocytes from peripheral blood. J. Immunol. Methods 210, 167–174 (1997)."), which precludes comprehensive and unbiased HLA profiling.

Here we developed ESCAPE-seq, a high-throughput and combinatorial platform, enabling the simultaneous assessment of around 75,000 pHLA combinations (1,500 peptides across 50 HLA alleles) in a single screen. By applying ESCAPE-seq to SARS-CoV-2 viral variants, recurrent tumor driver mutations and aberrant tumor fusion variants, we successfully identified viral antigens and tumor neoantigens that are missed from computational prediction. Importantly, ESCAPE-seq revealed public shared tumor neoantigens presented across multiple HLA alleles, providing a comprehensive catalog of tumor neoantigens with potential therapeutic implications across diverse human populations.

Results
-------

### Single-chain pMHC trafficking to the cell surface depends on specific peptide–HLA binding

The peptide–MHC (pMHC) single-chain trimer (SCT) has been extensively utilized to explore pMHC–T cell receptor (TCR) interactions. The pMHC consists of an 8–10-amino-acid antigen peptide, β2\-microglobulin (B2M) and an MHC allele arranged in tandem and linked by flexible glycine- and serine-rich linkers[17](https://www.nature.com/articles/s41588-025-02268-1#ref-CR17 "Greten, T. F. et al. Peptide–β2-microglobulin–MHC fusion molecules bind antigen-specific T cells and can be used for multivalent MHC–Ig complexes. J. Immunol. Methods 271, 125–135 (2002)."),[18](https://www.nature.com/articles/s41588-025-02268-1#ref-CR18 "Yu, Y. Y. L., Netuschil, N., Lybarger, L., Connolly, J. M. & Hansen, T. H. Cutting Edge: Single-chain trimers of MHC class I molecules form stable structures that potently stimulate antigen-specific T cells and B cells. J. Immunol. 168, 3145–3149 (2002)."). Typically, a high-affinity peptide preceded by a signal peptide traffics the trimeric complex to the cell surface, endowing high surface expression in HLA-knockout cells (Extended Data Fig. [1a,b](https://www.nature.com/articles/s41588-025-02268-1#Fig7)).

Using HLA-A2 SCT, we observed that substituting the high-affinity peptide with one lacking HLA-A2 binding affinity (no affinity) led to a striking reduction of cell-surface staining. Neither B2M nor HLA-A2 antibodies showed positive staining of the no affinity peptide construct (Fig. [1a,b](https://www.nature.com/articles/s41588-025-02268-1#Fig1) and Extended Data Fig. [1a,b](https://www.nature.com/articles/s41588-025-02268-1#Fig7)). To exclude misfolding, we fused enhanced green fluorescent protein (eGFP) directly to the cytoplasmic tail of the HLA-A2 allele in both high-affinity peptide–SCT and no affinity peptide–SCT. We observed robust membrane localization only with the high-affinity pp65 peptide, whereas the nonbinding peptide displayed diffusive intracellular eGFP distribution (Fig. [1c](https://www.nature.com/articles/s41588-025-02268-1#Fig1)), indicating that stable HLA–peptide binding is required for efficient surface trafficking. Structural integrity was confirmed via two approaches: HLA-A2:NY-ESO-1 SCT-expressing cells activated cognate TCR-expressing Jurkat cells via NFAT reporter induction (Extended Data Fig. [1c](https://www.nature.com/articles/s41588-025-02268-1#Fig7)), and H2-Kb:SIINFEKL antibody showed concordant staining with total surface H2-Kb (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02268-1#Fig7)). Together, these results showed that only properly folded SCTs with high-affinity peptides can traffic to and maintain structural integrity at the cell surface.

**Fig. 1: SCTs of pMHC class I differentiate presentable peptides from those with no or low affinities.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02268-1/figures/1)

**a**, Schematic of assay, where SCTs with high-affinity peptides were displayed on cell surface when transfected, while SCTs with no peptides binding to MHC alleles failed to escape to cell surface. KO, knockout. **b**, Flow cytometry measurement of surface pHLA-A\*0201 trimers with high-affinity (red) and low-affinity peptides (blue) in SCTs. Antibodies against either B2M or HLA-A2 alleles were used to stain the cells. Blank cells are shown in gray as background signals. **c**, Imaging of pHLA SCT fused with eGFP directly at the cytoplasmic tail of HLA-A2. Repeated experiments show consistent results. Representative images are shown. Scale bar, 40 μm. **d**, Representative histograms of cell-surface SCTs for human HLA-A\*0101, B\*0702 and C\*0401, and mouse H2-Kb allele. Two examples of high-affinity peptides (red) and two of negative peptides (blue) were shown per allele. The gray line shows the background from blank cells. **e**, Plot of gMFI versus IEDB measured binding affinity for SCTs containing Y84C mutation (red) or of wild-type (WT) allele (blue). _n_ = 2–4. **f**, Histogram of B2M staining for cell-surface pHLA-A2 presentation. Three conditions were measured, respectively, B2M fused with HLA-A2 only, co-expression of B2M–HLA-A2 with a high-affinity peptide pp65 preceded with a signal peptide and, last, a normal SCT for pp65 peptide. Right: the bar graph showing the quantification of signals. _n_ = 4; \*\*\*_P_ < 0.001 by two-sided _t_\-test. Error bars denote s.e.m.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

To assess the dependence of SCT surface localization on peptide binding across multiple MHC alleles, we generated SCTs with representative alleles from HLA-A, HLA-B and HLA-C groups (A\*0101, B\*0702 and C\*0401). Leveraging the Immune Epitope Database (IEDB), we systematically fused peptides with high or no binding affinity into SCTs for each HLA-I allele (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)). Consistently, trimers with nonbinding peptides exhibited markedly reduced surface expression across examined HLA alleles (Fig. [1d](https://www.nature.com/articles/s41588-025-02268-1#Fig1)), including mouse MHC H2-Kb allele (Fig. [1d](https://www.nature.com/articles/s41588-025-02268-1#Fig1)). The geometric mean of fluorescence intensity (gMFI) for high-affinity peptides was around 1,000-fold higher than for nonbinding peptides, offering a broad dynamic range (Extended Data Fig. [1e](https://www.nature.com/articles/s41588-025-02268-1#Fig7)). We observed a loose sigmoidal relationship between gMFI and peptide HLA-A2 affinities, plateauing at higher affinities (Fig. [1e](https://www.nature.com/articles/s41588-025-02268-1#Fig1) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)). The Y84C mutation conserved across _HLA-A_, _HLA-B_ and _HLA-C_ alleles and G2C in linker generate disulfide bridges, enhancing pHLA stability and surface display[19](https://www.nature.com/articles/s41588-025-02268-1#ref-CR19 "Chour, W. et al. Large libraries of single-chain trimer peptide-MHCs enable antigen-specific CD8+ T cell discovery and analysis. Commun. Biol. 6, 1–13 (2023)."),[20](https://www.nature.com/articles/s41588-025-02268-1#ref-CR20 "Finton, K. A. K. et al. Effects of HLA single chain trimer design on peptide presentation and stability. Front. Immunol. 14, 1170462 (2023).") (Extended Data Fig. [1f](https://www.nature.com/articles/s41588-025-02268-1#Fig7)). Comparing surface display with or without cysteine mutation revealed noisier staining without mutation (Fig. [1e](https://www.nature.com/articles/s41588-025-02268-1#Fig1) and [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)), suggesting the cysteine mutation enhances sensitivity and robustness for detecting HLA-restricted peptides[21](https://www.nature.com/articles/s41588-025-02268-1#ref-CR21 "Truscott, S. M. et al. Disulfide bond engineering to trap peptides in the MHC class I binding groove. J. Immunol. 178, 6280–6289 (2007).").

HLA proteins are highly unstable without bound peptides, with the peptide-loading complex (PLC) stabilizing MHC before suitable peptide loading[22](https://www.nature.com/articles/s41588-025-02268-1#ref-CR22 "Blum, J. S., Wearsch, P. A. & Cresswell, P. Pathways of antigen processing. Annu. Rev. Immunol. 31, 443–473 (2013)."),[23](https://www.nature.com/articles/s41588-025-02268-1#ref-CR23 "Pishesha, N., Harmand, T. J. & Ploegh, H. L. A guide to antigen processing and presentation. Nat. Rev. Immunol. 22, 751–764 (2022)."). We hypothesized that SCTs with nonpresentable peptides are destabilized, impeding ER exit and membrane trafficking. _TAP1/2_ knockout minimally affected SCT presentation (Extended Data Fig. [1g](https://www.nature.com/articles/s41588-025-02268-1#Fig7)). Truncated SCTs without fusing peptides showed cytoplasmic diffusion, whereas HLA-A2 alone localized to the cell surface in HLA-knockout cells (Fig. [1f](https://www.nature.com/articles/s41588-025-02268-1#Fig1) and Extended Data Fig. [1h,i](https://www.nature.com/articles/s41588-025-02268-1#Fig7)), supporting that SCT antigen presentation bypasses PLC. When truncated SCTs were co-expressed with a presentable HLA peptide in the ER, surface presentation was partially restored (Fig. [1f](https://www.nature.com/articles/s41588-025-02268-1#Fig1) and Extended Data Fig. [1i](https://www.nature.com/articles/s41588-025-02268-1#Fig7)), albeit at least tenfold less efficiently than for the SCT. To a certain extent, this mirrors a previous method that used the co-expression of separate HLA and peptide transgenes to score antigen presentation[16](https://www.nature.com/articles/s41588-025-02268-1#ref-CR16 "Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. Nat. Biotechnol. 41, 980–992 (2023)."). These results suggest that cell-surface pHLA expression serves as a proxy for peptide binding strength, with SCTs offering higher dynamic range and sensitivity.

### ESCAPE-seq leverages HLA escape to cell surface to quantify peptide–HLA presentation

We next exploited the system’s potential for high-throughput HLA peptide screening (Fig. [2a](https://www.nature.com/articles/s41588-025-02268-1#Fig2)). Starting with known-affinity peptides from IEDB, we selected thousands of peptides with different affinities for four representative HLA alleles (A\*0101, A\*0201, B\*0702 and C\*0401), cloning them into pooled lentiviral vectors per HLA allele and introducing them into cells at a low multiplicity of infection ([Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). Cells were sorted into four intensity-based bins in log-scale reflecting their programmed SCT’s ability to escape to the cell surface (Fig. [2b](https://www.nature.com/articles/s41588-025-02268-1#Fig2)). We assigned ESCAPE-seq scores (_E_\-scores) to each peptide ([Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)), with higher _E_\-score representing higher cell-surface staining and presentation. Biological replicates demonstrated high correlation (Extended Data Fig. [2a,b](https://www.nature.com/articles/s41588-025-02268-1#Fig8)) for all alleles studied. We compared _E_\-score correlation with IEDB affinity using HLA-A\*02 SCT with or without Y84C mutation. ESCAPE-seq with Y84C mutation showed better correlation (Pearson correlation 0.77 versus 0.49 without the mutation) (Fig. [2c](https://www.nature.com/articles/s41588-025-02268-1#Fig2) and Extended Data Fig. [2c](https://www.nature.com/articles/s41588-025-02268-1#Fig8)). Therefore, we used the Y84C mutation in all subsequent ESCAPE-seq experiments (Extended Data Fig. [1f](https://www.nature.com/articles/s41588-025-02268-1#Fig7)).

**Fig. 2: ESCAPE-seq demonstrates good performance and benchmarks across all HLA-I subtypes.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02268-1/figures/2)

**a**, Schematic of ESCAPE-seq. A pool of peptides across a wide range of affinities was selected from IEDB, synthesized and cloned to build the SCT library. Cells were transduced with the pooled virus and were sorted and sequenced. **b**, Representative histogram of cell-surface HLA-A\*02 trimer expression on cells transduced with pooled virus. The whole cell population was sorted into four evenly divided bins on log-scale as indicated on the histogram. Gray line marks the signal from cells without pHLA transduced. **c**, Scatter plot of IEDB affinity versus _E_\-score calculated for each peptide. A line at 500 nM of affinity and a cutoff line for _E_\-score were drawn as indicated. **d**, Plots of recall rate (predicted positive/all positive) versus IEDB-defined affinity bins. ESCAPE-seq performance was compared with NetMHC4 in either binding affinity (ba) or eluted ligand (el) mode. _n_ = 2 biological replicates. **e**, Plot of ROC curve of HLA-A2 allele for evaluating ESCAPE-seq in comparison with NetMHC. The AUCs were noted. **f**, PRC plot for HLA-A2 allele in comparison with NetMHC. **g**, Bar plots of AUC-ROC for all four HLA-I alleles to compare the performance of ESCAPE-seq with NetMHC. **h**, Bar plots of AUC-PRC for all four HLA-I alleles to compare the performance of ESCAPE-seq with NetMHC. _n_ = 4. Error bars denote s.e.m. FSC-W, forward scatter width; HP, high-affinity peptide; MOI, multiplicity of infection; NGS, next-generation sequencing; NP, peptide with no binding affinity.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

By binning IEDB affinity of defined presentable peptides, we measured recall and precision rates (Fig. [2d](https://www.nature.com/articles/s41588-025-02268-1#Fig2) and [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). Comparison with NetMHC4, a widely used pMHC prediction tool trained on IEDB[24](https://www.nature.com/articles/s41588-025-02268-1#ref-CR24 "Reynisson, B., Alvarez, B., Paul, S., Peters, B. & Nielsen, M. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Nucleic Acids Res. 48, W449–W454 (2020)."), revealed the reliable performance of ESCAPE-seq. ESCAPE-seq yielded >90% recall for high-affinity peptides, aligning closely with NetMHC4 predictions (Fig. [2d](https://www.nature.com/articles/s41588-025-02268-1#Fig2)). The receiver operating characteristic (ROC) curve indicated an area under the curve (AUC) of 0.919 when evaluating ESCAPE-seq against IEDB measured affinity, comparable to NetMHC4 in both binding affinity (NetMHC4-BA) mode and eluted ligand modes (NetMHC-EL)(Fig. [2e](https://www.nature.com/articles/s41588-025-02268-1#Fig2)). The precision–recall curve (PRC), which offers more robust metrics when positive rates are low (typical for antigen presentation datasets where most peptides will not bind the HLAs), showed comparable performance with ESCAPE-seq, obtaining an AUC of 0.91 (Fig. [2f](https://www.nature.com/articles/s41588-025-02268-1#Fig2)).

Systematic comparisons across HLA-A\*0101, B\*0702 and C\*0401 showed ESCAPE-seq achieving similar performance to NetMHC4 (Extended Data Fig. [2d–k](https://www.nature.com/articles/s41588-025-02268-1#Fig8)), evident in both correlation coefficient (Extended Data Fig. [2l,m](https://www.nature.com/articles/s41588-025-02268-1#Fig8)) and AUC-ROC (Fig. [2g](https://www.nature.com/articles/s41588-025-02268-1#Fig2)). PRC analysis mirrored ROC findings (Fig. [2h](https://www.nature.com/articles/s41588-025-02268-1#Fig2)). Overall, our results indicate ESCAPE-seq yields comparable metrics to NetMHC on the model’s training data.

### ESCAPE-seq reveals presented epitopes from SARS-CoV-2 variants

Utilizing ESCAPE-seq, we screened for presentable peptides in SARS-CoV-2 spike and nucleocapsid proteins. All peptides were standardized to nine amino acids, comprising approximately 1,500 peptides tiling across spike and nucleocapsid proteins, plus 900 viral mutation peptides from 17 SARS-CoV-2 variant strains including alpha, beta, delta and omicron (Fig. [3a](https://www.nature.com/articles/s41588-025-02268-1#Fig3), [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10) and Supplementary Table [3](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)).

**Fig. 3: ESCAPE-seq reveals presented peptides in SARS-CoV-2 spike and nucleocapsid proteins, and strain variants.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02268-1/figures/3)

**a**, Schematic of peptide pool generation by tiling across full spike and nucleocapsid proteins, and the strain variant mutations. **b**, Scatter plot of NetMHC prediction versus _E_\-score for each peptide. Lines of cutoff values were drawn at 2% rank for NetMHC\_El and 3.2 for _E_\-score. Here, the _E_\-scores are normalized as described in the [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10). The percentage ranking score from NetMHC shows how a peptide’s predicted binding affinity compares to a large set of random natural peptides, with typical cutoffs of 0.5% and 2% as strong and weak binders, correspondingly. **c**, Consensus motif for HLA-A\*02 and HLA-B\*07 from ESCAPE-seq positive SARS peptides. **d**, Plots of _E_\-score for each peptide with three HLA alleles across the spike protein (_x_ axis, schematic drawing of spike protein at the bottom). Black circles highlight the reported presentable peptides from literature. **e**, Plots of _E_\-score for peptides spanning nucleocapsid protein. Three HLA alleles were drawn in aligned position for each peptide. Black circles highlight the reported presentable peptides from literature. **f**, Hierarchical heatmap showing the cluster and shared peptides from SARS spike protein across HLA alleles of interest. **g**, As in **f** but for nucleocapsid protein. **h**, Neoantigen plot of SARS-CoV-2 spike peptides presented by HLA-A\*02 allele, where the _x_ axis is the _E_\-score for each mutational peptide and the _y_ axis is the _E_\-score of its corresponding wild-type peptide. Three quadrants containing positive antigen peptides were highlighted. **i**, As in **h** but for HLA-B\*07 allele. **j**, Bar graph of percentage of peptides in three quadrants for all three HLA-I alleles. NTD, N-terminal domain; RBD, receptor binding domain; S2, S2 subunit of spike protein.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

ESCAPE-seq screens across three common HLA-I alleles (A\*0101, A\*0201, B\*0702) showed high replication consistency (Extended Data Fig. [3a,b](https://www.nature.com/articles/s41588-025-02268-1#Fig9),). Comparing _E_\-scores against NetMHC predictions, we observed 90% of peptides were nonpresentable by both methods whereas 2% of peptides were discordant (Fig. [3b](https://www.nature.com/articles/s41588-025-02268-1#Fig3) and Extended Data Fig. [3c–e](https://www.nature.com/articles/s41588-025-02268-1#Fig9)). ESCAPE-positive peptides’ motif patterns closely aligned with known HLA-I allele patterns (Fig. [3c](https://www.nature.com/articles/s41588-025-02268-1#Fig3)), and most literature-reported SARS-CoV-2 peptides appeared positive by ESCAPE-seq[25](https://www.nature.com/articles/s41588-025-02268-1#ref-CR25 "As, S. et al. SARS-CoV-2 epitopes are recognized by a public and diverse repertoire of human T cell receptors. Immunity 53, 1245–1257 (2020)."),[26](https://www.nature.com/articles/s41588-025-02268-1#ref-CR26 "Sekine, T. et al. Robust T cell immunity in convalescent individuals with asymptomatic or mild COVID-19. Cell 183, 158–168 (2020).") (Fig. [3d,e](https://www.nature.com/articles/s41588-025-02268-1#Fig3)). Presentable peptides were dispersed across viral proteins without forming clusters. Cross-allele clustering indicated few peptides were presented by two HLA alleles, and rarely by all three (Fig. [3f,g](https://www.nature.com/articles/s41588-025-02268-1#Fig3)). Generally, 5–10% of peptides are presentable across three alleles depending on _E_\-score cutoff (Extended Data Fig. [3f](https://www.nature.com/articles/s41588-025-02268-1#Fig9)). Interestingly, ROC-AUC between ESCAPE-seq and NetMHC was 0.95 for IEDB training peptides but decreased significantly for SARS-CoV-2 peptides (Extended Data Fig. [3g,h](https://www.nature.com/articles/s41588-025-02268-1#Fig9)). Since ESCAPE-seq performance is input-independent, this decrease likely reflected reduced NetMHC prediction power on new peptides, suggesting NetMHC performs less effectively on novel peptides compared with familiar training data—an intuitively expected outcome.

We investigated peptide presentation changes across mutations in spike and nucleocapsid proteins from SARS-CoV-2 variant strains. Notably, numerous nonpresentable wild-type peptides became presentable after mutations, and vice versa (Fig. [3h,i](https://www.nature.com/articles/s41588-025-02268-1#Fig3) and Extended Data Fig. [3i,j](https://www.nature.com/articles/s41588-025-02268-1#Fig9)). Point mutations particularly altered presentation status, with notably fewer peptides retaining their presentation status after a point mutation (Fig. [3j](https://www.nature.com/articles/s41588-025-02268-1#Fig3) and Extended Data Fig. [3k](https://www.nature.com/articles/s41588-025-02268-1#Fig9)).

### Combinatorial ESCAPE-seq achieves simultaneous profiling of peptide presentation across diverse HLA alleles

We next integrated ESCAPE-seq with a combinatorial HLA indexing strategy to screen peptide pools across multiple HLA alleles simultaneously (Fig. [4a](https://www.nature.com/articles/s41588-025-02268-1#Fig4) and [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). We integrated a barcode system with synonymous mutations in B2M gene to enable HLA allele identification via sequencing (Fig. [4a](https://www.nature.com/articles/s41588-025-02268-1#Fig4)). This multiplexed screening of peptide pools with HLA allele pools can simultaneously screen over 10,000 pHLA SCT variants, with the SCT library transduced and sorted into four bins based on cell-surface B2M expression (Fig. [2b](https://www.nature.com/articles/s41588-025-02268-1#Fig2)).

**Fig. 4: Combinatorial ESCAPE-seq achieves simultaneous profiling of peptide presentation across diverse HLA alleles in one screen.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02268-1/figures/4)

**a**, Schematic representation of generating, barcoding and sequencing for combinatorial ESCAPE-seq approach. As an example, the barcodes representing _n_ HLA alleles were introduced via synonymous mutations at the 3′ end of _B2__M_ gene (12 amino acids). Libraries of paired peptides and HLA barcodes were then amplified by three rounds of PCR for the Illumina-based sequencing platform (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)). sp, signal peptide. **b**, Schematics of combinatorial ESCAPE-seq using existing MS elution peptides. The 986 most common peptides from the MS dataset, along with 30 HLA alleles, were selected[27](https://www.nature.com/articles/s41588-025-02268-1#ref-CR27 "Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. Nat. Biotechnol. 38, 199–209 (2020)."). **c**, The percentage of epitopes identified by MS that show a positive _E_\-score by ESCAPE-seq. **d**, AUC-ROC metrics comparing ESCAPE-seq and MS results across 30 HLA alleles. Peptides were assigned a value of 1 if detected in MS and 0 if not. **e**, Schematics of combinatorial ESCAPE-seq used to screen cancer epitopes. A pool of 1,500 oncogene peptides, tiling across the 92 top cancer mutations and 31 oncogenic fusions, was randomly combined with 50 human HLA alleles. All peptides were 9 amino acids in length, and a pool of 100 known pHLA pairs were spiked into the experiment. This generated approximately 75,000 peptide–HLA pairs in one screening. **f**, Plot of _E_\-score versus IEDB binding affinity for the spike-in pHLA results. The spike-in pool included 17 known antigens and >80 positive and negative peptides selected from IEDB. **g**, Plot of _E_\-score versus NetHMC prediction for the known antigen peptides in the combinatorial pool. NetMHC4 results in both binding affinity mode (ba, top) and eluted ligand model (el, bottom) are shown. The grayed box highlights peptides for which NetMHC failed to make a prediction.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

For large-scale validation, we selected the top 986 MS-captured peptides across 30 HLA alleles from existing monoallelic HLA MS data[27](https://www.nature.com/articles/s41588-025-02268-1#ref-CR27 "Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. Nat. Biotechnol. 38, 199–209 (2020)."). We performed combinatorial ESCAPE-seq screening, examining over 29,000 pHLA interactions (Fig. [4b](https://www.nature.com/articles/s41588-025-02268-1#Fig4)). Quality control analysis showed high consistency between replicates, and MS-validated peptides exhibited substantially higher _E_\-scores (Extended Data Fig. [4a–c](https://www.nature.com/articles/s41588-025-02268-1#Fig10)).

To assess the ability of ESCAPE-seq to identify MS-validated peptides, we calculated the percentage of MS-detected peptides that were also identified by ESCAPE-seq for each HLA allele (capture percentage, Fig. [4c](https://www.nature.com/articles/s41588-025-02268-1#Fig4)). Remarkably, ESCAPE-seq captured over 80% of MS-detected peptides across most HLA alleles, with some reaching 90%, demonstrating that a single ESCAPE-seq experiment can recapitulate the majority of findings from multiple MS experiments across 30 different HLA alleles. ROC analysis revealed strong performance, with AUC-ROC values between 0.8 and 1.0 for most HLA alleles, although slightly lower (~0.7) for HLA-B56:01, B52:01 and C\*07:01 (Fig. [4d](https://www.nature.com/articles/s41588-025-02268-1#Fig4) and Extended Data Fig. [4d](https://www.nature.com/articles/s41588-025-02268-1#Fig10)). In addition, ESCAPE-seq outperformed NetMHC4.1 predictions for multiple HLA alleles (Extended Data Fig. [4e](https://www.nature.com/articles/s41588-025-02268-1#Fig10)).

Next, we performed ESCAPE-seq analysis with peptides ranging from 8 to 12 amino acids, examining 10 gene loci (3 mutations, 3 wild-type counterpart genes, 3 fusions, 1 deletion) across 25 HLA alleles, designing ~45 peptides per target that tiled across mutation or junction sites (10,000 total interactions) (Extended Data Fig. [4f–h](https://www.nature.com/articles/s41588-025-02268-1#Fig10)). The 10-mers showed the highest presentation frequency, including the well-established KRAS(G12D) 10-mer epitope (Extended Data Fig. [4i,j](https://www.nature.com/articles/s41588-025-02268-1#Fig10)). An _E_\-score heatmap of mutation position versus peptide length shows that addition of one amino acid often maintains peptide presentation ability (Extended Data Fig. [4j](https://www.nature.com/articles/s41588-025-02268-1#Fig10)).

### Combinatorial ESCAPE-seq enables population-wide discovery of cancer neoantigens

To systematically identify cancer neoantigens across human populations by combinatorial ESCAPE-seq, we assembled 92 prevalent oncogenic mutations from cancer driver genes and the 31 most frequent oncogenic fusions from the COSMIC database[28](https://www.nature.com/articles/s41588-025-02268-1#ref-CR28 "Tate, J. G. et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947 (2019).") ([Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). This curated pool encompassed point mutations such as KRAS(G12D), BRAF(V600E) and TP53(R175H), and fusion junctions of oncogenic translocations such as BCR-ABL1 (Supplementary Table [4](https://www.nature.com/articles/s41588-025-02268-1#MOESM6)). We then generated a peptide pool tiling over these mutations and presented each peptide across the 50 most common _HLA-A_, _HLA-B_ and _HLA-C_ alleles (Fig. [4e](https://www.nature.com/articles/s41588-025-02268-1#Fig4)), covering >90% of the global population[27](https://www.nature.com/articles/s41588-025-02268-1#ref-CR27 "Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. Nat. Biotechnol. 38, 199–209 (2020).").

This multiplexed screening generated over 75,000 pHLA SCT variants in a single experiment, with 100 IEDB-derived peptides spiked in as controls (Fig. [4e](https://www.nature.com/articles/s41588-025-02268-1#Fig4) and [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). The library showed high reproducibility (correlation coefficient 0.89) between replicates (Extended Data Fig. [5a,b](https://www.nature.com/articles/s41588-025-02268-1#Fig11)). We optimized the protocol to minimize peptide–barcode mis-pairing from recombination events during library amplification, achieving <10% recombination-induced nonspecific reads ([Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10) and Extended Data Fig. [5c,d](https://www.nature.com/articles/s41588-025-02268-1#Fig11)).

_E_\-score histograms per HLA allele showed bimodal distributions with dominant negative peaks, and normalized _E_\-scores were calculated per peptide–HLA pair ([Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10) and Extended Data Fig. [5e](https://www.nature.com/articles/s41588-025-02268-1#Fig11)). Spike-in peptides correlated well with IEDB affinity/elution data, confirming robust single-allele performance in combinatorial settings (Fig. [4f](https://www.nature.com/articles/s41588-025-02268-1#Fig4) and Supplementary Table [6](https://www.nature.com/articles/s41588-025-02268-1#MOESM7)).

Importantly, ESCAPE-seq uncovered previously reported tumor neoantigens paired with correct HLA alleles that were falsely predicted as nonbinders by NetMHC (Fig. [4g](https://www.nature.com/articles/s41588-025-02268-1#Fig4) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02268-1#MOESM8)). This list is enriched for tumor neoantigens presented on _HLA-C_ alleles such as KRAS(G12V) (GAVGVGKSA)/C\*0304, KRAS(G12C) (GACGVGKSA)/C\*0304, KRAS(G12D) (GADGVGKSA)/C\*0802 and EGFR(T790M) (LTSTVQLIM)/C\*0701 (refs. [15](https://www.nature.com/articles/s41588-025-02268-1#ref-CR15 "Gurung, H. R. et al. Systematic discovery of neoepitope-HLA pairs for neoantigens shared among patients and tumor types. Nat. Biotechnol. 42, 1107–1117 (2024)."),[29](https://www.nature.com/articles/s41588-025-02268-1#ref-CR29 "Tran, E. et al. T-cell transfer therapy targeting mutant KRAS in cancer. N. Engl. J. Med. 375, 2255–2262 (2016)."),[30](https://www.nature.com/articles/s41588-025-02268-1#ref-CR30 "Choi, J. et al. Systematic discovery and validation of T cell targets directed against oncogenic KRAS mutations. Cell Rep. Methods 1, 100084 (2021)."),[31](https://www.nature.com/articles/s41588-025-02268-1#ref-CR31 "Smyth, R. P. et al. Reducing chimera formation during PCR amplification to ensure accurate genotyping. Gene 469, 45–51 (2010)."),[32](https://www.nature.com/articles/s41588-025-02268-1#ref-CR32 "Jetzt, A. E. et al. High rate of recombination throughout the human immunodeficiency virus type 1 genome. J. Virol. 74, 1234–1240 (2000).")), highlighting the sensitivity of ESCAPE-seq to identify HLA-C-presented tumor neoantigens that are missed by computational prediction (Fig. [4g](https://www.nature.com/articles/s41588-025-02268-1#Fig4)). Notably, the reported HLA-C\*0304-restricted KRAS(G12V) (GAVGVGKSA) antigen can be presented by additional HLA-I alleles covering diverse human populations, including B\*5401 (East Asia), B\*5601 (Australia, Oceania) and C\*0602 (North Africa)[33](https://www.nature.com/articles/s41588-025-02268-1#ref-CR33 "Gonzalez-Galarza, F. F. et al. Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools. Nucleic Acids Res. 48, D783–D788 (2020).") (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02268-1#MOESM8)).

The landscape of neoantigen presentation across oncogenes and HLA alleles (Fig. [5a](https://www.nature.com/articles/s41588-025-02268-1#Fig5)) provided several lessons. _HLA-B_ alleles displayed substantial variance in peptide presentation (1–20% presentation rates) while _HLA-C_ alleles showed higher peptide presentation (Extended Data Fig. [5f](https://www.nature.com/articles/s41588-025-02268-1#Fig11)). Most peptides were presented by only one or two HLA alleles (Fig. [5b](https://www.nature.com/articles/s41588-025-02268-1#Fig5)), consistent with divergent allele-specific binding motifs[27](https://www.nature.com/articles/s41588-025-02268-1#ref-CR27 "Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. Nat. Biotechnol. 38, 199–209 (2020)."),[34](https://www.nature.com/articles/s41588-025-02268-1#ref-CR34 "Gfeller, D. & Bassani-Sternberg, M. Predicting antigen presentation—what could we learn from a million peptides? Front. Immunol. 9, 1716 (2018)."),[35](https://www.nature.com/articles/s41588-025-02268-1#ref-CR35 "Karnaukhov, V. et al. HLA variants have different preferences to present proteins with specific molecular functions which are complemented in frequent haplotypes. Front. Immunol. 13, 1067463 (2022)."),[36](https://www.nature.com/articles/s41588-025-02268-1#ref-CR36 "Sidney, J., Peters, B., Frahm, N., Brander, C. & Sette, A. HLA class I supertypes: a revised and updated classification. BMC Immunol. 9, 1 (2008)."). However, a small number of neoantigen peptides were presented by ≥20 HLA alleles (that is, ‘public neoantigens’) (Fig. [5b](https://www.nature.com/articles/s41588-025-02268-1#Fig5)); such antigen peptides hold great interest for potential utilization in immunotherapies. Using HLA-C\*0304 as an example, the majority of these ‘public antigens’ showed intermediate and high _E_\-scores. Some of those with the highest _E_\-scores could be readily detected and validated in independent MS experiments (Fig. [5c](https://www.nature.com/articles/s41588-025-02268-1#Fig5), Extended Data Fig. [5g](https://www.nature.com/articles/s41588-025-02268-1#Fig11) and [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)).

**Fig. 5: Combinatorial ESCAPE-seq enables population-wide antigen presentation discovery of cancer neoantigens derived from driver oncogenes.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02268-1/figures/5)

**a**, Heatmap of the _E_\-score of 1,500 oncogene peptides (_y_ axis) across 50 HLAs (_x_ axis), showing moderate clustering and absence of commonly presented peptides. **b**, Count of peptides that were commonly presented by multiple alleles (_x_ axis). Inset, magnified view of peptide count in the shaded region. **c**, Scatter plot of _E_\-score versus HLA allele count for individual peptides highlighted in the gray region in **b** that are presented by multiple HLA alleles (_x_ axis, _n_ > 12). Green dots represent peptides included in the MS experiments with monoallelic C\*0304 cells, while red dots indicate peptides detected in MS. **d**, Aggregated analysis combining all peptides presented across the same mutation. For each point mutation or fusion breakpoint, nine candidate peptides tiling the mutation were analyzed. The color map indicates the number of presented peptides out of nine. **e**, Heatmap of number of peptides presented for each oncogenic point mutation (_y_ axis) across _HLA-A_, _HLA-B_ and _HLA-C_ alleles (_x_ axis). **f**, Heatmap similar to **e** for peptides derived from oncogenic fusions. **g**, Number of HLA alleles capable of presenting peptides containing oncogenic fusion mutation (_y_ axis) plotted against mutations (_x_ axis), ordered by HLA allele count. **h**, As in **g** but for oncogenic point mutations.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

### ESCAPE-seq nominates high-priority cancer neoantigens from driver oncoproteins

We next investigated antigen presentation characteristics by aggregating results across HLA alleles for each oncogenic mutation and identified hot spots of presented peptides across HLAs (Fig. [5d,e](https://www.nature.com/articles/s41588-025-02268-1#Fig5)). Defining presentable neoantigens as those with at least one presentable peptide, several HLA alleles presented over 50% of the 92 samples tested, with _HLA-C_ alleles showing particularly high coverage (60–80%) (Extended Data Fig. [5f–h](https://www.nature.com/articles/s41588-025-02268-1#Fig11)). We observed that mutations displayed varying HLA compatibility: ‘public’ driver neoantigens such as EGFR(T790M) and MED12(G44V) were presented by >60% of HLA alleles, while ‘private’ neoantigens such as KRAS(G12C) and TP53(R284W) were presented by only 2% (Fig. [5h](https://www.nature.com/articles/s41588-025-02268-1#Fig5)).

For oncogenic fusion proteins including those prevalent in pediatric sarcomas[37](https://www.nature.com/articles/s41588-025-02268-1#ref-CR37 "Latysheva, N. S. & Babu, M. M. Discovering and understanding oncogenic gene fusions through data intensive computational approaches. Nucleic Acids Res. 44, 4487–4503 (2016)."),[38](https://www.nature.com/articles/s41588-025-02268-1#ref-CR38 "Liu, Y. et al. Etiology of oncogenic fusions in 5,190 childhood cancers and its clinical and therapeutic implication. Nat. Commun. 14, 1739 (2023)."), presentation patterns paralleled those of point mutations (Extended Data Fig. [5i,j](https://www.nature.com/articles/s41588-025-02268-1#Fig11)). We discovered a diverse range of antigen presentation compatibility for fusion oncoproteins (Fig. [5g](https://www.nature.com/articles/s41588-025-02268-1#Fig5)). For example, EWSR1–FLI fusion peptides are presented by almost 30 HLA alleles (approximately 60% of HLAs tested, a ‘public neoantigen’), whereas PAX7–FOXO fusion was presented by only one allele. Of note, all presented fusion breakpoint peptides represent potential neoantigens as they are de novo peptides consisting of two gene fragments not observed in the wild-type genome.

Considering that typical diploid human cells harbor two alleles of each HLA-A, HLA-B and HLA-C, we extrapolated the distribution of presented mutation coverage within the human population through repeated sampling ([Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). Remarkably, >90% of prevalent oncogenic mutations could be presented by at least one allele, with fusion mutations showing slightly lower but still substantial coverage (>80%) (Extended Data Fig. [5k](https://www.nature.com/articles/s41588-025-02268-1#Fig11)).

Finally, we systematically compared oncogenic (Mut) versus wild-type (WT) peptide presentation by including both in the same assay. Analysis revealed four distinct presentation patterns: (1) WT−Mut− (neither presented), (2) WT+Mut+ (both presented), (3) WT+ only (mutation enables immune escape), and (4) Mut+ only (mutation confers novel HLA binding) (Fig. [6a](https://www.nature.com/articles/s41588-025-02268-1#Fig6)). It is known that a single amino acid change may not change the TCR binding or specificity[39](https://www.nature.com/articles/s41588-025-02268-1#ref-CR39 "Zhang, H. et al. Identification of NY-ESO-1157-165 specific murine T cell receptors with distinct recognition pattern for tumor immunotherapy. Front. Immunol. 12, 644520 (2021)."),[40](https://www.nature.com/articles/s41588-025-02268-1#ref-CR40 "Wooldridge, L. et al. A single autoimmune T cell receptor recognizes more than a million different peptides. J. Biol. Chem. 287, 1168–1177 (2012)."), and thus differential antigen presentation is one of the key determinant therapeutic indexes of immunotherapies targeting cancer neoantigens. With HLA-A\*0201 as an example, we found that approximately half of the peptides containing a point mutation and their corresponding wild-type peptides are coordinately presented (Fig. [6b](https://www.nature.com/articles/s41588-025-02268-1#Fig6)). Across all HLA alleles, the prevalence of this shared population where both mutant and wild-type are equally presented is dominant (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02268-1#Fig12)), aligning with the widely accepted notion that only a few amino acids on presented peptides serve as anchor points dictating binding[41](https://www.nature.com/articles/s41588-025-02268-1#ref-CR41 "Birnbaum, M. E. et al. Deconstructing the peptide-MHC specificity of T cell recognition. Cell 157, 1073–1087 (2014)."). Notably, a substantial number of mutant peptides are well presented but the corresponding wild-type peptides are not (Fig. [6b,c](https://www.nature.com/articles/s41588-025-02268-1#Fig6) and Extended Data Fig. [6a,b](https://www.nature.com/articles/s41588-025-02268-1#Fig12), Mut+). We reasoned that these Mut+ peptides constitute promising immunogenic neoantigen candidates, since immune tolerance of these antigens will not be established due to the lack of presentation of wild-type peptides. Indeed, these Mut+ peptides contain previously reported immunogenic tumor neoantigens including KRAS(G12D) (GADGVGKSA)/C\*0802 (ref. [29](https://www.nature.com/articles/s41588-025-02268-1#ref-CR29 "Tran, E. et al. T-cell transfer therapy targeting mutant KRAS in cancer. N. Engl. J. Med. 375, 2255–2262 (2016).")), FLT3(D835Y) (YIMSDSNYV)/A\*0201 (ref. [42](https://www.nature.com/articles/s41588-025-02268-1#ref-CR42 "Giannakopoulou, E. et al. A T cell receptor targeting a recurrent driver mutation in FLT3 mediates elimination of primary human acute myeloid leukemia in vivo. Nat. Cancer 4, 1474–1490 (2023).")) and EGFR(T790M) (IMQLMPFGC)/A\*0201 (ref. [43](https://www.nature.com/articles/s41588-025-02268-1#ref-CR43 "Yamada, T. et al. EGFR T790M mutation as a possible target for immunotherapy; identification of HLA-A*0201-restricted T cell epitopes derived from the EGFR T790M mutation. PLoS ONE 8, e78389 (2013).")) (Fig. [6b,c](https://www.nature.com/articles/s41588-025-02268-1#Fig6)). Thus, ESCAPE-seq permits nomination of potential immunogenic tumor neoantigens to prioritize candidates for antigen-directed immunotherapy.

**Fig. 6: ESCAPE-seq nominates high-priority cancer neoantigens.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02268-1/figures/6)

**a**, Schematic of a scatter quadrant plot displaying _E_\-scores for peptides with mutations (Mut) versus their corresponding wild-type peptides. Dashed lines indicate the _E_\-score thresholds, dividing peptides into four groups. The Mut+ only region, where mutant peptides are presented but their corresponding wild-type peptides are not, is highlighted in green. **b**, Example scatter plot of all point mutations as described in **a** for the HLA-A0201 allele. The reported neoantigen FLT3(D835Y) is highlighted along with the Mut+ only region. **c**, Scatter plots of _E_\-scores for mutant peptides versus their corresponding wild-type peptides for three mutations: EGFR(T790M) (left), KRAS(G12D) (middle) and FLT3(D835Y) (right). Red dots indicate known neoantigen peptides reported in the literature. **d**, Schematic of the T cell stimulation assay used to assess the immunogenicity of specific peptides ([Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). T cells isolated from healthy donors were activated and stimulated with peptides, followed by intracellular staining for IFNγ and TNF for flow cytometric analysis. **e**, _E_\-scores for three EGFR(T790M) mutant peptides (p17, p22 and p23; Supplementary Table [8](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)) and their corresponding wild-type peptides presented by HLA-A2301, plotted and compared. **f**, Flow cytometric analysis of dual intracellular IFNγ and TNF staining in donor-derived T cells stimulated with either DMSO (control) or peptide p17 (an EGFR(T790M) mutant peptide). **g**, Bar plots showing the percentage of IFNγ and TNF double-positive T cells stimulated with different peptides or conditions. CEFT, a positive control of 27 peptides (JPT). Error bars represent s.e.m.; _n_ = 4 or as plotted individually. \*\*\*_P_ < 0.001. **h**, Representative flow cytometry plots of T cells, as in **f**, stimulated with DMSO (control) or fusion peptide p11. **i**, Bar plots showing the percentage of IFNγ and TNF double-positive T cells stimulated with various fusion peptides or conditions. Error bars represent s.e.m.; _n_ = 2 biological replicates as plotted individually; 2–4 technical replicates each. \*\*_P_ < 0.01. All statistics used unpaired two-sided _t_\-test. APC, antigen-presenting cell.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

To validate the immunogenicity of Mut+ only peptides, we performed an in vitro peptide-induced T cell stimulation assay using HLA-A\*23:01+ healthy donor peripheral blood mononuclear cells (PBMCs). We compared one Mut+ peptide (p17: LTSTVQLIM_)_ against two WT+Mut+ peptides (p22: QLIMQLMPF and p23: LIMQLMPFG) from EGFR(T790M) (Fig. [6e](https://www.nature.com/articles/s41588-025-02268-1#Fig6) and Extended Data Fig. [6d](https://www.nature.com/articles/s41588-025-02268-1#Fig12)). Flow cytometry analysis revealed robust IFNγ+TNF+ polyfunctional T cell responses exclusively to the Mut+ only peptide (Fig. [6d–g](https://www.nature.com/articles/s41588-025-02268-1#Fig6) and [Methods](https://www.nature.com/articles/s41588-025-02268-1#Sec10)). Extended analysis of 13 fusion-derived epitopes identified by ESCAPE-seq using HLA-A\*03:01+/A\*24:02+ healthy donor PBMCs revealed that the NPM1-ALK fusion-derived epitope was immunogenic, showing a significant increase in polyfunctional T cells after peptide stimulation (Fig. [6h,i](https://www.nature.com/articles/s41588-025-02268-1#Fig6) and Extended Data Fig. [6e](https://www.nature.com/articles/s41588-025-02268-1#Fig12)). Together, these results establish ESCAPE-seq as a powerful tool for identifying potentially immunogenic neoantigens, particularly Mut+ only peptides that bypass immune tolerance due to the lack of presentation of wild-type counterparts.

Discussion
----------

The HLA region comprises the most polymorphic segments within the human genome, with over 26,000 identified alleles within the current IMGT/HLA collection[44](https://www.nature.com/articles/s41588-025-02268-1#ref-CR44 "Robinson, J. et al. The IPD and IMGT/HLA database: allele variant databases. Nucleic Acids Res. 43, D423–D431 (2015)."). Developing methods to efficiently screen antigen-presenting peptides across numerous HLA alleles could significantly accelerate our understanding of pMHC–TCR biology. This includes the discovery of tumor neoantigens, the search for pathogenic antigens and the mapping of novel TCRs, essential to advancing therapeutic interventions. ESCAPE-seq has enabled population-wide antigen presentation analysis for HLA-I alleles, potentially joining genome-wide association studies as a general approach to understand human variation and disease.

MS-based HLA-I immunopeptidomes allow mapping of HLA-I eluted peptides and have been a mainstay of T cell antigen discovery. This approach is dominated by self peptides from the cellular proteome, limiting the sensitivity to detect clinically important peptides from pathogens and oncogenic proteins[45](https://www.nature.com/articles/s41588-025-02268-1#ref-CR45 "Hickman, H. D. et al. Toward a definition of self: proteomic evaluation of the class I peptide repertoire. J. Immunol. 172, 2944–2952 (2004)."). Further challenges for HLA-I antigen identification using MS include high cost, specialized equipment, large quantities of sample material, ambiguous assignment of peptides to specific HLA alleles and potential peptide bias due to cysteine oxidation in MS sample preparation[7](https://www.nature.com/articles/s41588-025-02268-1#ref-CR7 "Bassani-Sternberg, M. & Gfeller, D. Unsupervised HLA peptidome deconvolution improves ligand prediction accuracy and predicts cooperative effects in peptide-HLA interactions. J. Immunol. 197, 2492–2499 (2016)."),[8](https://www.nature.com/articles/s41588-025-02268-1#ref-CR8 "Pearson, H. et al. MHC class I-associated peptides derive from selective regions of the human genome. J. Clin. Invest. 126, 4690–4701 (2016)."),[13](https://www.nature.com/articles/s41588-025-02268-1#ref-CR13 "Darwish, M. et al. High-throughput identification of conditional MHCI ligands and scaled-up production of conditional MHCI complexes. Protein Sci. 30, 1169–1183 (2021)."),[46](https://www.nature.com/articles/s41588-025-02268-1#ref-CR46 "Sachs, A. et al. Impact of cysteine residues on MHC binding predictions and recognition by tumor-reactive T cells. J. Immunol. 205, 539–549 (2020)."). To address these limitations, DNA sequencing-based HLA immunopeptidome discovery methods have emerged, leveraging large-scale, cost-effective DNA oligonucleotide synthesis for antigen screening[16](https://www.nature.com/articles/s41588-025-02268-1#ref-CR16 "Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. Nat. Biotechnol. 41, 980–992 (2023)."). ESCAPE-seq and related methods[16](https://www.nature.com/articles/s41588-025-02268-1#ref-CR16 "Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. Nat. Biotechnol. 41, 980–992 (2023).") leverage the cellular ER machinery for quality control of proper and stable pHLA folding as a prerequisite for cell-surface trafficking. Our single-chain design offers several advantages compared with previous methods that separately express HLA and peptide transgenes: (1) increased sensitivity by customizable linkers and cysteine mutations; (2) improved throughput and dynamic range through direct component fusion (Fig. [1f](https://www.nature.com/articles/s41588-025-02268-1#Fig1)); and (3) simplified implementation without requiring individual HLA allele cell line generation. ESCAPE-seq allowed high-throughput profiling of _HLA-C_ alleles that previously could not be studied[16](https://www.nature.com/articles/s41588-025-02268-1#ref-CR16 "Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. Nat. Biotechnol. 41, 980–992 (2023)."). Moreover, ESCAPE-seq facilitates combinatorial screening across multiple HLA alleles, setting it apart as an innovative and versatile tool. Unlike MS-based approaches, ESCAPE-seq is not limited by the set of antigen proteins or HLA alleles present in the sample. ESCAPE-seq demonstrated comparable performance to NetMHC on training datasets while identifying true positive neoantigens missed by computational prediction, particularly for _HLA-C_ alleles with limited training data. This superior sensitivity in identifying HLA-C-presented antigens reflects insufficient computational training datasets, potentially due to lower surface expression compared with _HLA-A_ or _HLA-B_ alleles[9](https://www.nature.com/articles/s41588-025-02268-1#ref-CR9 "McCutcheon, J. A., Gumperz, J., Smith, K. D., Lutz, C. T. & Parham, P. Low HLA-C expression at cell surfaces correlates with increased turnover of heavy chain mRNA. J. Exp. Med. 181, 2085–2095 (1995)."). It was previously shown that _HLA-C_ alleles can form larger clusters than the HLA-B alleles, which may compensate for their low surface expression to trigger T cell activation[47](https://www.nature.com/articles/s41588-025-02268-1#ref-CR47 "Kennedy, P. R., Barthen, C., Williamson, D. J. & Davis, D. M. HLA-B and HLA-C differ in their nanoscale organization at cell surfaces. Front. Immunol. 10, 61 (2019)."). Thus, ESCAPE-seq could improve prediction tools by providing extensive _HLA-C_ training datasets. Our pilot investigation of SARS-CoV-2 spike and nucleocapsid proteins revealed rapid alteration of antigen presentation through variant mutations, indicating a potential immune escape mechanism. Systematic application of methods such as ESCAPE-seq may better inform the immune evasion dynamics of seasonal variants and help identify conserved and broadly recognized epitopes to facilitate vaccine design.

For cancer immunotherapy, the ideal cancer antigen is uniquely present in cancer, exists in each of the cancer cells and cannot be readily lost by the cancer to evade immunity. Driver oncogenic mutations fulfill many of these criteria, and tracking or directing immune response against driver mutations, such as KRAS(G12D), has garnered high interest[48](https://www.nature.com/articles/s41588-025-02268-1#ref-CR48 "Pearlman, A. H. et al. Targeting public neoantigens for cancer immunotherapy. Nat. Cancer 2, 487–497 (2021)."). Oncogene-derived epitopes are a prime example of high-value HLA ligands that may be missed by current immunopeptidome methods, which require processing many irrelevant but abundant epitopes from self proteins to visualize rare disease-relevant epitopes from mutated proteins. Current knowledge and existing clinical trials are biased toward HLA-A2 alleles, limiting therapeutic accessibility. ESCAPE-seq enables systematic antigen presentation profiling to nominate neoantigen epitopes that are broadly presented by diverse HLAs and are preferentially presented over cognate wild-type sequences. We demonstrated this potential through comprehensive validation of EGFR(T790M) epitope LTSTVQLIM across multiple HLA alleles using multiple orthogonal approaches. Recent HLA monoallelic MS confirmed its presentation by HLA-C\*07:01 in cells expressing the T790M minigene[49](https://www.nature.com/articles/s41588-025-02268-1#ref-CR49 "Gurung, H. R. et al. Systematic discovery of neoepitope–HLA pairs for neoantigens shared among patients and tumor types. Nat. Biotechnol. 42, 1107–1117 (2024)."), while a neoantigen vaccine clinical trial demonstrated presentation by HLA-C\*15:02 with specific T cell responses detected via pMHC tetramers[50](https://www.nature.com/articles/s41588-025-02268-1#ref-CR50 "Li, F. et al. Neoantigen vaccination induces clinical and immunologic responses in non-small cell lung cancer patients harboring EGFR mutations. J. Immunother. Cancer 9, e002531 (2021)."). Using engineered monoallelic HLA-C03:04 cell lines and HLA-eluted peptide MS, we further confirmed direct presentation by this additional allele. Moreover, our peptide stimulation assays with HLA-A\*23:01+ healthy donor PBMCs revealed significant T cell responses to this epitope, characterized by IFNγ+/TNF+ polyfunctional T cells. These comprehensive validations establish the potential of ESCAPE-seq to identify therapeutically valuable targets across diverse populations.

These insights may prove fruitful for designing the next generation of cancer vaccines or engineered T cell therapies. Nonetheless, the complexity of the native antigen presentation process—encompassing peptide digestion, PLC peptide transfer and variations across cell types—may contribute to differences observed between MS findings and DNA sequencing-based screenings[7](https://www.nature.com/articles/s41588-025-02268-1#ref-CR7 "Bassani-Sternberg, M. & Gfeller, D. Unsupervised HLA peptidome deconvolution improves ligand prediction accuracy and predicts cooperative effects in peptide-HLA interactions. J. Immunol. 197, 2492–2499 (2016)."),[8](https://www.nature.com/articles/s41588-025-02268-1#ref-CR8 "Pearson, H. et al. MHC class I-associated peptides derive from selective regions of the human genome. J. Clin. Invest. 126, 4690–4701 (2016)."). Furthermore, mutations in presented peptides may not necessarily affect TCR recognition, adding another layer of complexity. Therefore, future research efforts could be directed toward developing high-throughput assays for identifying immunogenic antigens and antigen-specific TCRs to advance antigen-directed cancer immunotherapies[51](https://www.nature.com/articles/s41588-025-02268-1#ref-CR51 "Yu, B. et al. Engineered cell entry links receptor biology with single-cell genomics. Cell 185, 4904–4920.e22 (2022)."),[52](https://www.nature.com/articles/s41588-025-02268-1#ref-CR52 "Kula, T. et al. T-Scan: a genome-wide method for the systematic discovery of T cell epitopes. Cell 178, 1016–1028 (2019)."). Beyond HLA-I, we predict ESCAPE-seq could be applied to HLA-II antigen screening by leveraging cellular quality control mechanisms for stable peptide–MHC-II complex selection. Such expansion to HLA-II would enable comprehensive profiling of both CD8+ and CD4+ T cell responses in cancer and infectious diseases.

In summary, ESCAPE-seq is a promising tool for discovery of HLA-presented antigens. ESCAPE-seq can complement NetMHC for initial screening and the immunopeptidome for more targeted screening. Its potential is significant in accelerating the discovery of potential immunogenic antigen peptides for diverse T cell pools, spanning both pathogenic and oncogenic neoantigens. Moreover, its capacity for combinatorial pooling across multiple HLAs in varied settings underscores its promise as an efficient and high-throughput platform for mapping the landscape of antigen presentations across the diversity of the human population.

Methods
-------

All research complies with Stanford University’s ethical regulations and safety regulations, and was reviewed and approved by both Stanford University ethics panel and biosafety panel.

### DNA synthesis and plasmid construction

All plasmids were made with Gibson assembly (NEB) unless specified otherwise. In brief, based on a lentiviral vector of SCTs pHLA-A0201 and pHLA-A0101 (ref. [51](https://www.nature.com/articles/s41588-025-02268-1#ref-CR51 "Yu, B. et al. Engineered cell entry links receptor biology with single-cell genomics. Cell 185, 4904–4920.e22 (2022).")), P2A-puromysin-2A-eGFP was inserted after the cytoplasmic tail of HLA alleles. Then, oligonucleotides encoding different peptides were inserted into the SCT between a signal peptide and B2M gene with a flexible linker. To build the SCT of other MHC alleles, we made the Y84C mutation first (Supplementary Fig. [1d](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)), then inserted it to replace A\*02 or A\*01 above. For example, the HLA-A\*02 was digested out and replaced with B\*0702 (Addgene, cat. no. 135509), C\*0401 and H2-Kb alleles (synthesized by TwistBio).

Oligonucleotide sequences encoding human HLA alleles were obtained from IEDB (HLA-A0101, HLA-A0201 and HLA-B0702). For each allele, we first made a cloning lentiviral vector with Esp3i sites in the place of peptide in the single-chain format[51](https://www.nature.com/articles/s41588-025-02268-1#ref-CR51 "Yu, B. et al. Engineered cell entry links receptor biology with single-cell genomics. Cell 185, 4904–4920.e22 (2022)."), followed by a P2A sequence, eGFP, T2A and puromycin-resistance gene. Afterward, various peptides obtained from literature or the IEDB database (Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02268-1#MOESM1) and [2](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)) were inserted in place. For HLA–eGFP direct fusion, the P2A sequence between HLA and eGFP above was replaced with a flexible GS-rich linker. Various point mutants and deletion mutants were generated similarly using the above vector as source. To express peptides alone in the ER, a cytomegalovirus (CMV) promoter and the signal peptide from the human growth hormone gene were utilized. The peptide expression was driven in the ER using a lentiviral vector that included a blasticidin-resistance gene. All pooled oligonucleotides encoding peptides were ordered from TwistBio. For benchmarking against the IEDB database, we retrieved the IEDB database and extracted all peptides for four common HLA alleles, A\*0101, A\*0201, B\*0702 and C\*0401. Then we randomly selected peptides across all affinities for each allele and combined to make a pool of >2,000 peptides in total. Next, the oligonucleotide pool encoding the peptides flanked by overhang sequences on both sides was inserted into the cloning vector as described above and was electroporated into competent cells (Enduro electrocompetent cells, Biosearch Technologies) using a Bio-Rad MicroPulser. For screening on SARS-CoV-2, DNA sequences encoding spike and nucleocapsid protein from strain Wuhan-Hu-1 were used. A tiling pool was made to cover the full sequencing with a 3-base-pair (bp) shift in-frame between neighbor tiles. All strain variant peptides were included in the pool based on the strain variant data (the GFF file of December 2021 from UniProt).

### Combinatorial pool generation

Fifty HLA-I alleles were selected based on their high frequencies in diverse world populations ([https://www.allelefrequencies.net/hla.asp](https://www.allelefrequencies.net/hla.asp)) or their association with disease. Overall, 14 HLA-A, 29 HLA-B and 7 HLA-C were obtained accordingly, which are A\*01:01, A\*02:01, A\*02:05, A\*02:12, A\*03:01, A\*11:01, A\*23:01, A\*24:02, A\*30:01, A\*31:01, A\*31:08, A\*34:01, A\*33:03, A\*68:01, B\*07:02, B\*08:01, B\*08:02, B\*13:01, B\*15:01, B\*15:02, B\*15:03, B\*18:01, B\*27:01, B\*27:05, B\*27:02, B\*35:01, B\*35:02, B\*35:08, B\*39:06, B\*40:01, B\*40:06, B\*40:10, B\*41:01, B\*44:0, B\*46:01, B\*48:03, B\*50:01, B\*51:01, B\*52:01, B\*54:01, B\*56:01, B\*57:01, B\*58:01, C\*03:04, C\*04:01, C\*06:02, C\*07:01, C\*07:02, C\*08:02, C\*12:03. Sequences encoding human HLA alleles were obtained from IEDB. Some restriction enzyme sites if present in HLA were mutated without changing amino acid and Y84C mutation introduced (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02268-1#Fig7)). Then part of the B2M sequence, a unique barcode per HLA allele and a long linker containing the 10x TSO sequence were added to the HLA sequence and synthesized by TwistBio. An oligonucleotide pool encoding over 1,500 peptides (cancer driver gene peptides and oncogenic fusion gene peptides) was collected as described below. Then flanking sequences for Gibson assembly were added to either side of each oligonucleotide and synthesized by TwistBio. The combinatorial paired peptide–HLA plasmid pool was generated by two steps. First, the 50 HLA alleles gene fragments were amplified by PCR with very low cycle number to yield about ~50 ng, as we noticed that PCR with high cycle number produces chimeric HLA alleles due to high homology among the alleles. Alternatively, HLA allele fragments can be inserted into lentiviral vector via digestion and ligation without the PCR step, and then inserted with the peptide pool with the Gibson reaction. This avoids the PCR step that potentially generates chimeric fragments. In this study, we first inserted each HLA-I allele individually to generate 50 cloning vectors. Then we pooled the plasmids to insert peptide pools as described below. This strategy offers the flexibility of customized selection of HLA alleles for different peptide pools of interest in the future.

After that, peptide oligonucleotide pools were amplified for six cycles with PCR. The cloning plasmid pool with 50 HLAs inserted (as described above) was digested with Esp3i and the peptide pool was inserted into it via Gibson to generate the final combinatorial pHLA pooled plasmids. We aimed to get over 50,000 colonies for the first Gibson assembly of 50 HLAs, while the last pHLA pool should give colonies of >100× (1,000× ideally; 7.5–75 million colonies) in terms of library complexity.

We also generated spike-in pools to mix during the experiments. To examine the chimeric reads and recombination of pHLA reads, we generated two spike-in pools separately, with each consisting of 25 HLA alleles with their corresponding peptide pools (Supplementary Table [6](https://www.nature.com/articles/s41588-025-02268-1#MOESM7)).

### Cell line generation and culture

HEK293T cells (ATCC) were cultured in DMEM supplemented with 10% FBS and 1% Penstrep. HLA knockout using Cas9 RNP was done as described previously. Single-guide RNAs were synthesized by Synthego, including ACUGCUACUUCUCGCCGACU (Human TAP1), CUGGUGGGGUACGGGGCUGC (human TAP2), CGGCUACUACAACCAGAGCG (HLA\_1), AGAUCACACUGACCUGGCAG (HLA-2), AGGUCAGUGUGAUCUCCGCA (HLA-3). For dual HLA and TAP knockout cells, HLA-knockout HEK293T cells were used to electroporate cas9 RNP mixed with TAP1/TAP2 single guide RNAs. The cells were cultured for 5 days. Then the cells were stained with PE-B2M (BioLegend), sorted into single cells and seeded into 96-well plates. The clonal cell wells were picked and expanded. For each well, its TAP1/2 were verified with genomic PCR and Sanger sequencing.

### Lentivirus production and titration

Lentiviruses were made as described previously[51](https://www.nature.com/articles/s41588-025-02268-1#ref-CR51 "Yu, B. et al. Engineered cell entry links receptor biology with single-cell genomics. Cell 185, 4904–4920.e22 (2022)."). In brief, per six-well plate, HEK293T cells were transfected with a viral expression vector (2 µg), pMD2.G (VSV-G WT) (1 µg) and psPax2 (2 µg) with Lipofectamine 3000. The medium was changed once the next day, and viral supernatant was collected twice at 48 h and 72 h, respectively. The virus was concentrated with 4× Lenti-X according to manufacturer’s protocol, and stored as 20× concentrated at −80 °C. For pooled pHLA virus, we used either 6-cm or 10-cm dishes to make larger quantities of virus, with proportional scaling of DNA and reagent amount. The virus was first titrated with HEK293T cells at 25% confluence, and percentage of infection was measured by a flow cytometer (Attune, Lifetechnology).

### Transfection, cell assay and flow cytometry

First, 100,000 cells were seeded onto 24-well plates and cultured overnight. The next day, 0.75 µl of DNA with 1.5 µl of Lipofectamine 3000 (Thermo Fisher) were transfected into cells. The cells were collected 1 or 2 days after transfection and incubated in full DMEM medium for 30 min at 37 °C to recover. Afterward, the cells were pelleted and stained with 2 µl of antibodies (PE anti-B2M from BioLegend, PE-Cy7 anti human HLA-A2 from BioLegend) for 30 min on ice. Cells were washed once before being examined with a flow cytometer (Attune, Lifetechnology).

### Imaging

Cells were transfected with HLA–eGFP fusion construct as described above. The images were taken with a Zeiss LSM780 confocal microscope the next day.

### Pooled antigen presentation screening

The cells were infected with virus at a multiplicity of infection at ~0.15. Puromycin was added to cells after 2 days of infection at 2 µg ml−1 final. The cells were collected after 4–6 days of drug selection. After incubating in fresh DMEM medium for 30 min at 37 °C, the cells were stained with anti-B2M-PE antibody (cat. no. 316306, clone 2M2, from BioLegend) at 2 µl (0.4 µg) per 1 million cells on ice for 30 min with intermittent mixing. Once washed, the cells were resuspended and sorted into four fractions based on PE-B2M intensity by a BD Aria cell sorter. The four bins are evenly distributed on log-scale based on PE-B2M intensity between background peak and signal’s maxima. For example, typically we make the center of the negative peak (background peak) at 100, then the low fraction bin centered around 400, the medium bin at 1,600 and the high bin at around 6,400 or higher. We typically require two biological replicates per screen, and start cells about >1,500 times the peptide pool’s complexity for each replicate.

### DNA extraction, library generation and sequencing

Sorted cells were spun down and genomic DNA was extracted with a Zymo column (Zymo QuickDNA) according to the manufacturer’s protocol, or with lysis and precipitation. In brief, the cells were first resuspended in lysis buffer (20 mM Tris, 5 mM EDTA and 50 mM NaCl, 0.1% SDS), then RNase A and proteinase K (20 mg ml−1 stock solution) were added at 5 µl per 100 µl of solution. The samples were incubated at 37 °C for 30 min then 50 °C overnight. The next day, the aqueous phase containing the DNA was obtained using phenol:chloroform:isoamyl alcohol (Invitrogen) and Maxtract High Density from Qiagen following the manufacturer’s protocol. DNA was precipitated with 70% isopropanol following standard protocol.

The library that encodes HLA peptides and barcodes was generated through three rounds of PCR (primers in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)). First, we enriched the pHLA fragments from genomic DNA by 15 cycles of PCR (98 °C for 3 min, then 15× of 20 s at 98 °C 20 s, 20 s at 58 °C and 60 s at 72 °C) with 0.8 µM primers of amp\_GH\_Fw and B2M-bc\_rev. After cleanup, 5 µl of elution was used for the second round of PCR with 0.8 µM nested primer containing Illumina adapters P7\_GH\_HLA\_fw and P5\_BC-B2M\_rev (98 °C for 1 min, then 6× of 20 s at 98 °C 20 s, 20 s at 59 °C and 60 s at 72 °C). The above primers were designed in a way compatible with the dual index used in Illumina sequencing platforms. The final libraries were obtained by a third round of index PCR with Illumina Truseq-based index primers (98 °C for 1 min, then 6× of 20 s at 98 °C 20 s, 20 s at 63 °C and 60 s at 72 °C). Due to low complexity of the libraries, typically 25% PhiX Control v3 libraries (Illumina) were spiked in when sequencing by NextSeq or HiSeq.

### Database retrieval and collection

We created three distinct sets of peptides for the evaluation. We selected 2,178 peptides with a broad range of measured affinity values from the IEDB for alleles HLA-A0\*1:01, HLA-A\*02:01, HLA-B\*07:02, HLA-C\*04:01. For HLA-C, due to the scarcity of strong binding measurements, we opted for a smaller selection of 150 peptides due to the lack of existing measured affinity values. The _E_\-scores of these peptides were measured in four separate experiments, one for each allele.

The sequence of SARS-CoV-2 Wuhan-1 strain used was a wild type The GFF file containing different strain mutations was obtained from UniProt (accession: [P0DTC2](https://www.uniprot.org/uniprot/P0DTC2), as of 2021), from which the lists of mutations per strain were extracted to build the peptide pools. In this study, for simplicity, we considered only the point mutations and in-frame deletions/mutations in spike and nucleocapsid proteins of SARS-CoV-2 (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02268-1#MOESM1)), which gave >2,600 9-mer antigens for alleles HLA-A\*01:01, HLA-A\*02:01, HLA-B\*07:02. We measured their _E_\-scores concurrently with the IEDB experiments.

Top cancer driver gene mutations were collected from COSMIC database. In brief, we first manually ranked oncogenes in the database based on occurrence. Then the top prevalent point mutations per gene were collected to get a candidate list. DNA fragments encoding peptides across each mutation were extracted from gene sequences from the NCBI website.

For fusion mutations, we similarly ranked COSMIC fusion genes based on mutation numbers from the COSMIC database, and picked the top ~30 fusion variants (for example, >5–10% or occurrence >50). Per variant, we retrieved the genomic coordinates for the breakpoint first. Then, we obtained fusion genes’ DNA and protein sequences based on genomic coordinates using FusionGDB2 ([https://compbio.uth.edu/FusionGDB2/index.html](https://compbio.uth.edu/FusionGDB2/index.html)). Next, we used the UCSC genome browser to find the DNA sequences for these two genes (version hg19), and aligned DNA sequences to find the breakpoint site on the coding sequence. The breakpoint per fusion was further confirmed by UniProt ([https://www.uniprot.org/uniprotkb](https://www.uniprot.org/uniprotkb)), where protein sequences for the two genes were aligned to fusion protein sequences obtained above to retrieve peptides across breakpoints. Per mutation/fusion point, a pool of the peptides was obtained by tiling 9–11-amino-acid short peptides across the mutation or breakage point. Altogether, we tested 1,500 top oncogenes from the COSMIC database across 50 diverse alleles from HLA-A, HLA-B and HLA-C.

### Data analysis

Custom Python scripts were built to read the fastq files using Biopython functions and count the paired reads. Here we took only the antigen peptides or HLA barcodes with exact match of their flanking sequence of 6–10 bp. Then, a count table containing read counts for each peptide–HLA pair per sorted bin was built.

### _E_\-score calculation and clustering

The transformation of four binned numbers into an _E_\-score commences with a normalization of all the reads within each bin through dividing the counts in each bin by the average. This standard procedure in genomic studies was performed to account for any discrepancies that may arise due to variations in sequencing read depth. Differentiating from the conventional RNA sequencing read normalization, we omitted log normalization in our process. Our rationale stems from the presumption that the distributions of our counts are unlikely to conform to a log-normal distribution, and thus log normalization may not provide an accurate representation of our data. Subsequently, we normalized the total number of sequencing reads measured per trimer (pHLA pair). This step is crucial in adjusting for uneven distribution of antigen–HLA reads that were originated from various steps in the ESCAPE-seq experiments, including pooled plasmid cloning, virus production and transduction, and library construction. With the normalization complete, each allele–peptide trimer now possesses normalized counts, indicating the number of cells detected in each respective bin. To synthesize these data into a singular _E_\-score, we used the formula:\\({{E{\\rm{-score}}}}={\\rm{count}}{\\rm{s}}\_{{\\rm{bg}}}\\times {w}\_{{\\rm{bg}}}\\)\\(+{\\rm{count}}{\\rm{s}}\_{{\\rm{low}}}\\times {w}\_{{\\rm{low}}}+{\\rm{count}}{\\rm{s}}\_{{\\rm{med}}}\\)\\(\\times {w}\_{{\\rm{med}}}+{\\rm{count}}{\\rm{s}}\_{{\\rm{high}}}\\times {w}\_{{\\rm{high}}}\\) where weight _w_ were asigned as \\({w}\_{{\\rm{bg}}}=0,{w}\_{{\\rm{low}}}=2,\\)\\({w}\_{{\\rm{med}}}=4,{w}\_{{\\rm{high}}}=8\\) for background (bg), low, medium (med) and high fractions, where we put weight at log-scale that matched with the binning scale during cell sorting, while assigning background bin as 0.

For combinatorial screening with multiple HLA alleles, we further normalized _E_\-score for each allele to align their distributions in a more comparable manner. This normalization per allele is instrumental for the alignment of our data to a specific reference. This is to mitigate the influence of varying allele efficacies in presentation and/or their intrinsic stability to escape from ER. To facilitate this, we calculated the mode of lower peak (Extended Data Fig. [5e](https://www.nature.com/articles/s41588-025-02268-1#Fig11)) for each allele (presumably from all negative peptides). Following this, we adjusted every _E_\-score for that specific allele such that these modes converge. This technique of alignment ensured that the _E_\-score was not skewed due to the intrinsic variations between different alleles, thus making the comparison between different alleles more accurate and meaningful.

Heatmaps were generated using the ComplexHeatmap R package. Only peptides with at least one HLA _E_\-score above threshold were included. For the heatmap in Fig. [4](https://www.nature.com/articles/s41588-025-02268-1#Fig4), the score matrix was clipped to values between 2 and 5 to enhance visualization.

### Evaluation metrics

NetMHC predictions for individual peptides in binding affinity mode (\_ba) and eluted ligand mode (\_el) were obtained through the NetMHC-4.1 web interface (peptide mode in [https://services.healthtech.dtu.dk/services/NetMHCpan-4.1/](https://services.healthtech.dtu.dk/services/NetMHCpan-4.1/)). Predictions for pooled peptides were obtained through the command line version of NetMHC-4.1 (downloaded from [https://downloads.iedb.org/tools/](https://downloads.iedb.org/tools/)).

When contrasting the measured IC50 values, _E_\-scores and NetMHCpan predictions, we employed both regression metrics (Pearson’s _r_ and Spearman’s _r_) and classification metrics (ROC-AUC and PR-AUC), calculated separately for each allele.

The regression analyses were performed after transforming IC50 values into a log-transformed version, following the format used in training predictive models: 1 − log(binding affinity)/log(50,000), as detailed in the source. This transformation resulted in affinity values in the range 0 to 1, with an IC50 greater than 0.426 corresponding to an IC50 less than 500 nM. Concurrently, for classification analyses involving ROC-AUC and PR-AUC, a thresholding of the labels was implemented. For IC50\-based labels, including those from IEDB measurements and NetMHCpan predictions, binders were defined as those with IC50 values below 500 nM. Specifically, Pearson’s and Spearman’s correlations were calculated using the modules spearmanr and pearsonr from scipy.stats. ROC and PRC and their AUCs were calculated using functions in the Python package sklearn.metrics. To get the error bars, we visualized the 95% confidence intervals of the metrics across all the alleles. For labels derived from ESCAPE-seq measurements in this paper, binders were classified as entities with an _E_\-score greater than 3.2 for single HLA allele screening. For combinatorial ESCAPE-seq, we raised the cutoff to 3.8 due to higher observed noise, presumably from multiple HLA alleles. In general, a cutoff between 3.5 and 4 is reasonable.

Furthermore, for each metric calculated, we established a 95% confidence interval. This was achieved through the utilization of a nonparametric bootstrap method, entailing the generation of 1,000 samples from the original dataset with replacement, thereby offering an estimation of the uncertainty inherent in our metric estimates.

### Evaluating predictions of _E_\-scores by NetMHCpan models

To ascertain the degree of alignment between current state-of-the-art model predictions and _E_\-scores, we calculated metrics using NetMHCpan-4.1 predictions as an estimation for _E_\-scores, employing a threshold of _E_\-score > 3.8 for the classification metrics. NetMHCpan-4.1 includes two model variants: a binding affinity model trained to predict IC50 values and an eluted ligand model trained to predict ligands identified on the cell surface via MS. Although the eluted ligand model is generally the standard, we observed a superior correlation with _E_\-scores in the binding affinity model, which we therefore adopted for comparative metrics.

All metrics indicated a high degree of congruence between model predictions and _E_\-scores when utilizing the IEDB data, which formed the training basis for these models. However, when the models were applied to the SARS and oncogene datasets, the metrics were less encouraging. Besides ROC-AUC, which is largely impervious to class imbalances characteristic of the datasets we analyzed, there was a pronounced discrepancy in performance between the IEDB metrics and those derived from the SARS and oncogene datasets.

Previous studies have demonstrated a correlation between the number of peptides in the training data for a given allele and the performance of the model on other peptides of the same allele. However, in our analysis, the number of training examples for a given allele failed to account for most of the variance observed in the model’s inaccurate prediction of stability in the oncogene and SARS datasets. This was despite the model demonstrating competent performance when evaluated on the same alleles using peptides from the training set. While a slight dip in performance is predictable when evaluating peptides absent from the training data, a 30–75% performance reduction across most metrics implies an increased challenge posed by these new datasets. The smaller proportion of positive peptides per allele in these datasets could contribute to this difficulty. However, even when resampling the data to match the percentage of positives in the IEDB dataset, a significant gap in prediction ability persisted. This observed drop-off in performance underpins the necessity of the ESCAPE-seq methodology to augment current computational prediction methods and address the identified performance gaps.

### Exploring allele similarity as defined by peptide preferences

To evaluate the similarity among HLA alleles, we may consider three different parameters: sequence similarity, structural similarity or functional similarity (the similarity in their binder peptides). Although crystal structure information is not available for all HLA alleles, sequence data are available, and with the introduction of our new oncogene dataset, we can compare the binding preferences of 50 distinct alleles.

The sequences used to compare alleles are composed of 34 amino acids from each allele, often denoted as MHC pseudo-sequences. These sequences encapsulate all the amino acids within 5 Å of the peptide binding cleft that vary between alleles. We calculate sequence similarity between sequences _s_1 and _s_2, using the equation:

$\\begin{array}{l}{\\rm{SequenceSim}}({s}\_{1},{s}\_{2})\\\\=1-\\displaystyle\\frac{\\mathop{\\sum }\\nolimits\_{a{a}\_{i}=1}^{34}{\\rm{AAsim}}({s}\_{1}\[i\],{s}\_{2}\[i\])}{\\mathop{\\sum }\\nolimits\_{a{a}\_{i}=1}^{34}{\\rm{AAsim}}({s}\_{1}\[i\],{s}\_{1}\[i\])\\times \\mathop{\\sum }\\nolimits\_{a{a}\_{i}=1}^{34}{\\rm{AAsim}}({s}\_{2}\[i\],{s}\_{2}\[i\])}\\end{array}$

where \\({\\rm{AASim}}\\left({a}\_{1},{a}\_{2}\\right)\\) represents the similarity between amino acids \\({a}\_{1}\\) and \\({a}\_{2}\\) as per the BLOSUM62 matrix, across 34 amino acids (aa1–34).

To quantify the similarity among alleles based on peptide preferences, we consider the list of _E_\-scores for all 1,500 peptides in the oncogene pool and calculate the cosine distance between the _E_\-scores for all peptides between each allele pair. These cosine distances are used for clustering the alleles and for visualizing alleles in two dimensions. Specifically, the Uniform Manifold Approximation and Projection (UMAP) was constructed with Python umap package (with parameters n\_components = 2, random\_state = 42, n\_neighbors = 5, min\_dist = 0.01, metric = ‘cosine’).

### T cell immunogenicity and T cell reporter assays

Antigen-specific T cell immunogenicity was evaluated by a previously published protocol[53](https://www.nature.com/articles/s41588-025-02268-1#ref-CR53 "Cimen Bozkus, C., Blazquez, A. B., Enokida, T. & Bhardwaj, N. A T-cell-based immunogenicity protocol for evaluating human antigen-specific responses. STAR Protoc. 2, 100758 (2021)."). In brief, HLA-typed PBMCs from four healthy donors were used. Cryopreserved PBMCs were quickly thawed in a 37 °C water bath and transferred into RPMI medium (Thermo Fisher) containing DNase I (Sigma-Aldrich) at a final concentration of 2 U ml−1, spun down and resuspended in X-VIVO 15 medium (Lonza) supplemented with cytokines promoting dendritic cell differentiation, GM-CSF (Peprotech, 1,000 IU ml−1), IL-4 (R&D Systems, 500 IU ml−1) and Flt3L (Peprotech, 50 ng ml−1). Cells were seeded at 105 cells per well in U-bottomed 96-well plates and cultured for 24 h before being stimulated with control reagents or pooled test peptides (custom peptide synthesis, JPT Peptide Technologies), where each peptide was at a final concentration of 1 μM, together with adjuvants promoting dendritic cell maturation, LPS (Invivogen, 0.1 mg ml−1), R848 (Invivogen, 10 mM) and IL-1β (R&D Systems 10 ng ml−1), in X-VIVO 15 medium. Starting 24 h after stimulation, cells were fed every 2–3 d with cytokines supporting T cell expansion, IL-2 (R&D Systems, 10 IU ml−1), IL-7 (Peprotech, 10 ng ml−1) and IL-15 (Peprotech, 10 ng ml−1), in complete RPMI medium (GIBCO) containing 10% human serum (R10). After 10 d of culture, cells were collected, pooled within groups, washed, resuspended in R10 and seeded at 2 × 105 cells per well in U-bottomed 96-well plates. Expanded T cells were then re-stimulated with control reagents or 1 μM of test peptides, either pooled or individual, together with 0.5 mg ml−1 of costimulatory antibodies, anti-CD28 (CD28.2, cat. no. 55726, BD Biosciences) and anti-CD49d (clone: 9F10, cat. no. 555502, BD Biosciences), and protein transport inhibitors BD GolgiStop, containing monensin, and BD GolgiPlug, containing brefeldin A, at the manufacturer’s recommended concentrations. After 8 h of incubation at 37 °C, cells were processed for intracellular staining for flow cytometry using BD Cytofix/Cytoperm reagents according to the manufacturer’s protocol. The following antibodies were used: for surface staining CD3 (clone: SK7, cat. no. 344804, BioLegend), CD4 (clone RPA-T4, BV785) and CD8a (clone: RPA-T8, cat. no. 301049, BioLegend), and for intracellular staining IFN-g (clone: B27, cat. no. 506507, PE) and TNF-a (clone: Mab11, cat. no. 502930, PE/Cy7). All antibodies were purchased from BioLegend. LIVE/DEAD Fixable Aqua Dead Cell Stain Kit (Thermo Fisher) was used for live and dead cell discrimination. Data were acquired using the Invitrogen Attune NxT flow cytometer and FlowJo v.10 was used for analysis. DMSO (Sigma-Aldrich) was used at an equal volume to the test peptides and served as the vehicle/negative control. Significance was evaluated by _t_\-test comparing DMSO versus peptide-specific cytokine formation by CD8+ T cells. For the T cell reporter assay in Extended Data Fig. [1c](https://www.nature.com/articles/s41588-025-02268-1#Fig7), we transfected HLA/TAP-knockout HEK293T cells with SCTs containing either HLA-A2:SLLMWITQC (NY-ESO-1 epitope) or HLA-A2:NLVPMVATV (CMV epitope). After 48 h, SCT-transfected cells were cocultured with Jurkat NFAT-GFP reporter cells stably expressing NY-ESO-1-specific 1G4 TCR. TCR activation was measured by quantifying GFP+ Jurkat cells using a flow cytometer (BD FACSCanto) at coculture time points ranging from 5 min to 24 h.

### Mass spectrometry

MS experiments based on HLA-C\*0304 allele were done as previously published[16](https://www.nature.com/articles/s41588-025-02268-1#ref-CR16 "Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. Nat. Biotechnol. 41, 980–992 (2023)."). We first transduced wild-type HLA-C\*0304 allele with lentivirus with puromycin resistance into HEK293T cells with dual HLA and TAP knockouts as described above. Afterward, a pool of peptides with a range of low and high _E_\-score with HLA-C\*0304 from ESCAPE-seq were transduced to the cells with lentivirus carrying the blasticidin-resistance gene. Cells were selected with puromycin and blasticidin and expanded for 2 weeks. Approximately 600 million cells were collected for MS experiments. MS experiments were outsourced to MS works (MI). Peptides (100%) were desalted using solid-phase extraction (SPE) with a Waters μHLB C18 plate. Peptides were loaded directly and eluted using 30:70 acetonitrile:water (0.1% TFA). Eluted peptides were lyophilized and reconstituted in 0.1% TFA. Peptides (50%) were analyzed in analytical duplicate by nano liquid chromatography tandem MS using a Waters NanoAcquity system interfaced to a Thermo Fisher Fusion Lumos mass spectrometer. Peptides were loaded on a trapping column and eluted over a 75-μm analytical column at 350 nl min−1; both columns were packed with Luna C18 resin (Phenomenex). A 2-h gradient was employed. The mass spectrometer was operated using a custom data-dependent method, with MS performed in the Orbitrap at 60,000 full-width at half-maximum (FWHM) resolution and sequential tandem MS performed using high-resolution CID and EThcD in the Orbitrap at 15,000 FWHM resolution. All MS data were acquired from _m_/_z_ 300–1,600. A 3-s cycle time was employed for all steps.

### Statistics and reproducibility

We usually included two biological replicates for pooled screening and each pooled screen contained negative and positive controls. We required two replicates that were highly correlated (for example, Pearson’s correlation > 0.9) for further downstream analysis. For individual experiments, we used 3–4 replicates unless samples were limited (for example, HLA-typed donor PBMCs) or measurements were highly repeatable (in which case, we selected just two samples to report in the manuscript).

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02268-1#MOESM2) linked to this article.

Data availability
-----------------

All sequencing files have been deposited to the SRA (accession [PRJNA1268025](http://www.ncbi.nlm.nih.gov/sra?term=PRJNA1268025)). MS data were included as [Supplementary Information](https://www.nature.com/articles/s41588-025-02268-1#Sec31). Other data and materials are available upon request. [Source data](https://www.nature.com/articles/s41588-025-02268-1#Sec32) are provided with this paper.

Code availability
-----------------

Customized Python scripts were included as [Supplementary Information](https://www.nature.com/articles/s41588-025-02268-1#Sec31).

References
----------

1.  Shiina, T., Hosomichi, K., Inoko, H. & Kulski, J. K. The HLA genomic loci map: expression, interaction, diversity and disease. _J. Hum. Genet._ **54**, 15–39 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXjsVyksb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19158813)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20HLA%20genomic%20loci%20map%3A%20expression%2C%20interaction%2C%20diversity%20and%20disease&journal=J.%20Hum.%20Genet.&volume=54&pages=15-39&publication_year=2009&author=Shiina%2CT&author=Hosomichi%2CK&author=Inoko%2CH&author=Kulski%2CJK) 
    
2.  Pamer, E. & Cresswell, P. Mechanisms of MHC class I–restricted antigen processing. _Annu. Rev. Immunol._ **16**, 323–358 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXislyntrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9597133)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mechanisms%20of%20MHC%20class%20I%E2%80%93restricted%20antigen%20processing&journal=Annu.%20Rev.%20Immunol.&volume=16&pages=323-358&publication_year=1998&author=Pamer%2CE&author=Cresswell%2CP) 
    
3.  Gutierrez-Arcelus, M., Rich, S. S. & Raychaudhuri, S. Autoimmune diseases – connecting risk alleles with molecular traits of the immune system. _Nat. Rev. Genet._ **17**, 160–174 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26907721)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896831)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autoimmune%20diseases%20%E2%80%93%20connecting%20risk%20alleles%20with%20molecular%20traits%20of%20the%20immune%20system&journal=Nat.%20Rev.%20Genet.&volume=17&pages=160-174&publication_year=2016&author=Gutierrez-Arcelus%2CM&author=Rich%2CSS&author=Raychaudhuri%2CS) 
    
4.  Parham, P. & Ohta, T. Population biology of antigen presentation by MHC class I molecules. _Science_ **272**, 67–74 (1996).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XitVKkt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8600539)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Population%20biology%20of%20antigen%20presentation%20by%20MHC%20class%20I%20molecules&journal=Science&volume=272&pages=67-74&publication_year=1996&author=Parham%2CP&author=Ohta%2CT) 
    
5.  Neefjes, J., Jongsma, M. L. M., Paul, P. & Bakke, O. Towards a systems understanding of MHC class I and MHC class II antigen presentation. _Nat. Rev. Immunol._ **11**, 823–836 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsVCntLnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22076556)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Towards%20a%20systems%20understanding%20of%20MHC%20class%20I%20and%20MHC%20class%20II%20antigen%20presentation&journal=Nat.%20Rev.%20Immunol.&volume=11&pages=823-836&publication_year=2011&author=Neefjes%2CJ&author=Jongsma%2CMLM&author=Paul%2CP&author=Bakke%2CO) 
    
6.  Klatt, M. G. et al. Solving an MHC allele–specific bias in the reported immunopeptidome. _JCI Insight_ **5**, e141264 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32897882)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7566711)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Solving%20an%20MHC%20allele%E2%80%93specific%20bias%20in%20the%20reported%20immunopeptidome&journal=JCI%20Insight&volume=5&publication_year=2020&author=Klatt%2CMG) 
    
7.  Bassani-Sternberg, M. & Gfeller, D. Unsupervised HLA peptidome deconvolution improves ligand prediction accuracy and predicts cooperative effects in peptide-HLA interactions. _J. Immunol._ **197**, 2492–2499 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVCqtr%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27511729)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unsupervised%20HLA%20peptidome%20deconvolution%20improves%20ligand%20prediction%20accuracy%20and%20predicts%20cooperative%20effects%20in%20peptide-HLA%20interactions&journal=J.%20Immunol.&volume=197&pages=2492-2499&publication_year=2016&author=Bassani-Sternberg%2CM&author=Gfeller%2CD) 
    
8.  Pearson, H. et al. MHC class I-associated peptides derive from selective regions of the human genome. _J. Clin. Invest._ **126**, 4690–4701 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27841757)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5127664)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MHC%20class%20I-associated%20peptides%20derive%20from%20selective%20regions%20of%20the%20human%20genome&journal=J.%20Clin.%20Invest.&volume=126&pages=4690-4701&publication_year=2016&author=Pearson%2CH) 
    
9.  McCutcheon, J. A., Gumperz, J., Smith, K. D., Lutz, C. T. & Parham, P. Low HLA-C expression at cell surfaces correlates with increased turnover of heavy chain mRNA. _J. Exp. Med._ **181**, 2085–2095 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXlvFSjtLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7760000)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Low%20HLA-C%20expression%20at%20cell%20surfaces%20correlates%20with%20increased%20turnover%20of%20heavy%20chain%20mRNA&journal=J.%20Exp.%20Med.&volume=181&pages=2085-2095&publication_year=1995&author=McCutcheon%2CJA&author=Gumperz%2CJ&author=Smith%2CKD&author=Lutz%2CCT&author=Parham%2CP) 
    
10.  Jurtz, V. et al. NetMHCpan-4.0: improved peptide-MHC class I interaction predictions integrating eluted ligand and peptide binding affinity data. _J. Immunol._ **199**, 3360–3368 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs12mt7nI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28978689)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NetMHCpan-4.0%3A%20improved%20peptide-MHC%20class%20I%20interaction%20predictions%20integrating%20eluted%20ligand%20and%20peptide%20binding%20affinity%20data&journal=J.%20Immunol.&volume=199&pages=3360-3368&publication_year=2017&author=Jurtz%2CV) 
    
11.  Olivier, T., Haslam, A., Tuia, J. & Prasad, V. Eligibility for human leukocyte antigen-based therapeutics by race and ethnicity. _JAMA Netw. Open_ **6**, e2338612 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37883087)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10603498)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Eligibility%20for%20human%20leukocyte%20antigen-based%20therapeutics%20by%20race%20and%20ethnicity&journal=JAMA%20Netw.%20Open&volume=6&publication_year=2023&author=Olivier%2CT&author=Haslam%2CA&author=Tuia%2CJ&author=Prasad%2CV) 
    
12.  Schmittel, A., Keilholz, U. & Scheibenbogen, C. Evaluation of the interferon-γ ELISPOT-assay for quantification of peptide specific T lymphocytes from peripheral blood. _J. Immunol. Methods_ **210**, 167–174 (1997).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXht1Wnur8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9520299)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluation%20of%20the%20interferon-%CE%B3%20ELISPOT-assay%20for%20quantification%20of%20peptide%20specific%20T%20lymphocytes%20from%20peripheral%20blood&journal=J.%20Immunol.%20Methods&volume=210&pages=167-174&publication_year=1997&author=Schmittel%2CA&author=Keilholz%2CU&author=Scheibenbogen%2CC) 
    
13.  Darwish, M. et al. High-throughput identification of conditional MHCI ligands and scaled-up production of conditional MHCI complexes. _Protein Sci._ **30**, 1169–1183 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsVSqsbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33840137)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8138534)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-throughput%20identification%20of%20conditional%20MHCI%20ligands%20and%20scaled-up%20production%20of%20conditional%20MHCI%20complexes&journal=Protein%20Sci.&volume=30&pages=1169-1183&publication_year=2021&author=Darwish%2CM) 
    
14.  Haj, A. K. et al. High-throughput identification of MHC class I binding peptides using an ultradense peptide array. _J. Immunol._ **204**, 1689–1696 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXptFSgtbY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32060132)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-throughput%20identification%20of%20MHC%20class%20I%20binding%20peptides%20using%20an%20ultradense%20peptide%20array&journal=J.%20Immunol.&volume=204&pages=1689-1696&publication_year=2020&author=Haj%2CAK) 
    
15.  Gurung, H. R. et al. Systematic discovery of neoepitope-HLA pairs for neoantigens shared among patients and tumor types. _Nat. Biotechnol._ **42**, 1107–1117 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFOqsL%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37857725)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20discovery%20of%20neoepitope-HLA%20pairs%20for%20neoantigens%20shared%20among%20patients%20and%20tumor%20types&journal=Nat.%20Biotechnol.&volume=42&pages=1107-1117&publication_year=2024&author=Gurung%2CHR) 
    
16.  Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. _Nat. Biotechnol._ **41**, 980–992 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitleqtw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36593401)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10314971)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-throughput%2C%20targeted%20MHC%20class%20I%20immunopeptidomics%20using%20a%20functional%20genetics%20screening%20platform&journal=Nat.%20Biotechnol.&volume=41&pages=980-992&publication_year=2023&author=Bruno%2CPM) 
    
17.  Greten, T. F. et al. Peptide–β2-microglobulin–MHC fusion molecules bind antigen-specific T cells and can be used for multivalent MHC–Ig complexes. _J. Immunol. Methods_ **271**, 125–135 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xoslansbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12445736)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Peptide%E2%80%93%CE%B22-microglobulin%E2%80%93MHC%20fusion%20molecules%20bind%20antigen-specific%20T%20cells%20and%20can%20be%20used%20for%20multivalent%20MHC%E2%80%93Ig%20complexes&journal=J.%20Immunol.%20Methods&volume=271&pages=125-135&publication_year=2002&author=Greten%2CTF) 
    
18.  Yu, Y. Y. L., Netuschil, N., Lybarger, L., Connolly, J. M. & Hansen, T. H. Cutting Edge: Single-chain trimers of MHC class I molecules form stable structures that potently stimulate antigen-specific T cells and B cells. _J. Immunol._ **168**, 3145–3149 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xis1ajs7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11907065)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cutting%20Edge%3A%20Single-chain%20trimers%20of%20MHC%20class%20I%20molecules%20form%20stable%20structures%20that%20potently%20stimulate%20antigen-specific%20T%20cells%20and%20B%20cells&journal=J.%20Immunol.&volume=168&pages=3145-3149&publication_year=2002&author=Yu%2CYYL&author=Netuschil%2CN&author=Lybarger%2CL&author=Connolly%2CJM&author=Hansen%2CTH) 
    
19.  Chour, W. et al. Large libraries of single-chain trimer peptide-MHCs enable antigen-specific CD8+ T cell discovery and analysis. _Commun. Biol._ **6**, 1–13 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large%20libraries%20of%20single-chain%20trimer%20peptide-MHCs%20enable%20antigen-specific%20CD8%2B%20T%20cell%20discovery%20and%20analysis&journal=Commun.%20Biol.&volume=6&pages=1-13&publication_year=2023&author=Chour%2CW) 
    
20.  Finton, K. A. K. et al. Effects of HLA single chain trimer design on peptide presentation and stability. _Front. Immunol._ **14**, 1170462 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1eks7rJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37207206)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10189100)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Effects%20of%20HLA%20single%20chain%20trimer%20design%20on%20peptide%20presentation%20and%20stability&journal=Front.%20Immunol.&volume=14&publication_year=2023&author=Finton%2CKAK) 
    
21.  Truscott, S. M. et al. Disulfide bond engineering to trap peptides in the MHC class I binding groove. _J. Immunol._ **178**, 6280–6289 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXksl2nuro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17475856)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disulfide%20bond%20engineering%20to%20trap%20peptides%20in%20the%20MHC%20class%20I%20binding%20groove&journal=J.%20Immunol.&volume=178&pages=6280-6289&publication_year=2007&author=Truscott%2CSM) 
    
22.  Blum, J. S., Wearsch, P. A. & Cresswell, P. Pathways of antigen processing. _Annu. Rev. Immunol._ **31**, 443–473 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXnsFClsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23298205)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4026165)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathways%20of%20antigen%20processing&journal=Annu.%20Rev.%20Immunol.&volume=31&pages=443-473&publication_year=2013&author=Blum%2CJS&author=Wearsch%2CPA&author=Cresswell%2CP) 
    
23.  Pishesha, N., Harmand, T. J. & Ploegh, H. L. A guide to antigen processing and presentation. _Nat. Rev. Immunol._ **22**, 751–764 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvV2iurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35418563)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20guide%20to%20antigen%20processing%20and%20presentation&journal=Nat.%20Rev.%20Immunol.&volume=22&pages=751-764&publication_year=2022&author=Pishesha%2CN&author=Harmand%2CTJ&author=Ploegh%2CHL) 
    
24.  Reynisson, B., Alvarez, B., Paul, S., Peters, B. & Nielsen, M. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. _Nucleic Acids Res._ **48**, W449–W454 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1entr7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32406916)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7319546)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NetMHCpan-4.1%20and%20NetMHCIIpan-4.0%3A%20improved%20predictions%20of%20MHC%20antigen%20presentation%20by%20concurrent%20motif%20deconvolution%20and%20integration%20of%20MS%20MHC%20eluted%20ligand%20data&journal=Nucleic%20Acids%20Res.&volume=48&pages=W449-W454&publication_year=2020&author=Reynisson%2CB&author=Alvarez%2CB&author=Paul%2CS&author=Peters%2CB&author=Nielsen%2CM) 
    
25.  As, S. et al. SARS-CoV-2 epitopes are recognized by a public and diverse repertoire of human T cell receptors. _Immunity_ **53**, 1245–1257 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SARS-CoV-2%20epitopes%20are%20recognized%20by%20a%20public%20and%20diverse%20repertoire%20of%20human%20T%20cell%20receptors&journal=Immunity&volume=53&pages=1245-1257&publication_year=2020&author=As%2CS) 
    
26.  Sekine, T. et al. Robust T cell immunity in convalescent individuals with asymptomatic or mild COVID-19. _Cell_ **183**, 158–168 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFKjs7jE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32979941)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7427556)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Robust%20T%20cell%20immunity%20in%20convalescent%20individuals%20with%20asymptomatic%20or%20mild%20COVID-19&journal=Cell&volume=183&pages=158-168&publication_year=2020&author=Sekine%2CT) 
    
27.  Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. _Nat. Biotechnol._ **38**, 199–209 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktbvP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31844290)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20large%20peptidome%20dataset%20improves%20HLA%20class%20I%20epitope%20prediction%20across%20most%20of%20the%20human%20population&journal=Nat.%20Biotechnol.&volume=38&pages=199-209&publication_year=2020&author=Sarkizova%2CS) 
    
28.  Tate, J. G. et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. _Nucleic Acids Res._ **47**, D941–D947 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Cgt7bM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30371878)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=COSMIC%3A%20the%20Catalogue%20Of%20Somatic%20Mutations%20In%20Cancer&journal=Nucleic%20Acids%20Res.&volume=47&pages=D941-D947&publication_year=2019&author=Tate%2CJG) 
    
29.  Tran, E. et al. T-cell transfer therapy targeting mutant KRAS in cancer. _N. Engl. J. Med._ **375**, 2255–2262 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitFaqt73O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27959684)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5178827)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=T-cell%20transfer%20therapy%20targeting%20mutant%20KRAS%20in%20cancer&journal=N.%20Engl.%20J.%20Med.&volume=375&pages=2255-2262&publication_year=2016&author=Tran%2CE) 
    
30.  Choi, J. et al. Systematic discovery and validation of T cell targets directed against oncogenic KRAS mutations. _Cell Rep. Methods_ **1**, 100084 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVWgs77E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35474673)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9017224)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20discovery%20and%20validation%20of%20T%20cell%20targets%20directed%20against%20oncogenic%20KRAS%20mutations&journal=Cell%20Rep.%20Methods&volume=1&publication_year=2021&author=Choi%2CJ) 
    
31.  Smyth, R. P. et al. Reducing chimera formation during PCR amplification to ensure accurate genotyping. _Gene_ **469**, 45–51 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht12nsrvP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20833233)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reducing%20chimera%20formation%20during%20PCR%20amplification%20to%20ensure%20accurate%20genotyping&journal=Gene&volume=469&pages=45-51&publication_year=2010&author=Smyth%2CRP) 
    
32.  Jetzt, A. E. et al. High rate of recombination throughout the human immunodeficiency virus type 1 genome. _J. Virol._ **74**, 1234–1240 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXmtFyiuw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10627533)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC111457)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High%20rate%20of%20recombination%20throughout%20the%20human%20immunodeficiency%20virus%20type%201%20genome&journal=J.%20Virol.&volume=74&pages=1234-1240&publication_year=2000&author=Jetzt%2CAE) 
    
33.  Gonzalez-Galarza, F. F. et al. Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools. _Nucleic Acids Res._ **48**, D783–D788 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Glt7%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31722398)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Allele%20frequency%20net%20database%20%28AFND%29%202020%20update%3A%20gold-standard%20data%20classification%2C%20open%20access%20genotype%20data%20and%20new%20query%20tools&journal=Nucleic%20Acids%20Res.&volume=48&pages=D783-D788&publication_year=2020&author=Gonzalez-Galarza%2CFF) 
    
34.  Gfeller, D. & Bassani-Sternberg, M. Predicting antigen presentation—what could we learn from a million peptides? _Front. Immunol._ **9**, 1716 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30090105)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6068240)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Predicting%20antigen%20presentation%E2%80%94what%20could%20we%20learn%20from%20a%20million%20peptides%3F&journal=Front.%20Immunol.&volume=9&publication_year=2018&author=Gfeller%2CD&author=Bassani-Sternberg%2CM) 
    
35.  Karnaukhov, V. et al. HLA variants have different preferences to present proteins with specific molecular functions which are complemented in frequent haplotypes. _Front. Immunol._ **13**, 1067463 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXpsFWhsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36605212)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9808399)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HLA%20variants%20have%20different%20preferences%20to%20present%20proteins%20with%20specific%20molecular%20functions%20which%20are%20complemented%20in%20frequent%20haplotypes&journal=Front.%20Immunol.&volume=13&publication_year=2022&author=Karnaukhov%2CV) 
    
36.  Sidney, J., Peters, B., Frahm, N., Brander, C. & Sette, A. HLA class I supertypes: a revised and updated classification. _BMC Immunol._ **9**, 1 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18211710)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2245908)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HLA%20class%20I%20supertypes%3A%20a%20revised%20and%20updated%20classification&journal=BMC%20Immunol.&volume=9&publication_year=2008&author=Sidney%2CJ&author=Peters%2CB&author=Frahm%2CN&author=Brander%2CC&author=Sette%2CA) 
    
37.  Latysheva, N. S. & Babu, M. M. Discovering and understanding oncogenic gene fusions through data intensive computational approaches. _Nucleic Acids Res._ **44**, 4487–4503 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFWrtrfN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27105842)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4889949)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovering%20and%20understanding%20oncogenic%20gene%20fusions%20through%20data%20intensive%20computational%20approaches&journal=Nucleic%20Acids%20Res.&volume=44&pages=4487-4503&publication_year=2016&author=Latysheva%2CNS&author=Babu%2CMM) 
    
38.  Liu, Y. et al. Etiology of oncogenic fusions in 5,190 childhood cancers and its clinical and therapeutic implication. _Nat. Commun._ **14**, 1739 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXnt1Clu7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37019972)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10076316)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Etiology%20of%20oncogenic%20fusions%20in%205%2C190%20childhood%20cancers%20and%20its%20clinical%20and%20therapeutic%20implication&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Liu%2CY) 
    
39.  Zhang, H. et al. Identification of NY-ESO-1157-165 specific murine T cell receptors with distinct recognition pattern for tumor immunotherapy. _Front. Immunol._ **12**, 644520 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXpvVOqtrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33833762)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8021954)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20NY-ESO-1157-165%20specific%20murine%20T%20cell%20receptors%20with%20distinct%20recognition%20pattern%20for%20tumor%20immunotherapy&journal=Front.%20Immunol.&volume=12&publication_year=2021&author=Zhang%2CH) 
    
40.  Wooldridge, L. et al. A single autoimmune T cell receptor recognizes more than a million different peptides. _J. Biol. Chem._ **287**, 1168–1177 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjslynuw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22102287)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single%20autoimmune%20T%20cell%20receptor%20recognizes%20more%20than%20a%20million%20different%20peptides&journal=J.%20Biol.%20Chem.&volume=287&pages=1168-1177&publication_year=2012&author=Wooldridge%2CL) 
    
41.  Birnbaum, M. E. et al. Deconstructing the peptide-MHC specificity of T cell recognition. _Cell_ **157**, 1073–1087 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXoslynsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24855945)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4071348)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deconstructing%20the%20peptide-MHC%20specificity%20of%20T%20cell%20recognition&journal=Cell&volume=157&pages=1073-1087&publication_year=2014&author=Birnbaum%2CME) 
    
42.  Giannakopoulou, E. et al. A T cell receptor targeting a recurrent driver mutation in FLT3 mediates elimination of primary human acute myeloid leukemia in vivo. _Nat. Cancer_ **4**, 1474–1490 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVeqsrnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37783807)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10597840)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20T%20cell%20receptor%20targeting%20a%20recurrent%20driver%20mutation%20in%20FLT3%20mediates%20elimination%20of%20primary%20human%20acute%20myeloid%20leukemia%20in%20vivo&journal=Nat.%20Cancer&volume=4&pages=1474-1490&publication_year=2023&author=Giannakopoulou%2CE) 
    
43.  Yamada, T. et al. EGFR T790M mutation as a possible target for immunotherapy; identification of HLA-A\*0201-restricted T cell epitopes derived from the EGFR T790M mutation. _PLoS ONE_ **8**, e78389 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslGisbvM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24223798)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818324)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=EGFR%20T790M%20mutation%20as%20a%20possible%20target%20for%20immunotherapy%3B%20identification%20of%20HLA-A%2A0201-restricted%20T%20cell%20epitopes%20derived%20from%20the%20EGFR%20T790M%20mutation&journal=PLoS%20ONE&volume=8&publication_year=2013&author=Yamada%2CT) 
    
44.  Robinson, J. et al. The IPD and IMGT/HLA database: allele variant databases. _Nucleic Acids Res._ **43**, D423–D431 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVymtrbF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25414341)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20IPD%20and%20IMGT%2FHLA%20database%3A%20allele%20variant%20databases&journal=Nucleic%20Acids%20Res.&volume=43&pages=D423-D431&publication_year=2015&author=Robinson%2CJ) 
    
45.  Hickman, H. D. et al. Toward a definition of self: proteomic evaluation of the class I peptide repertoire. _J. Immunol._ **172**, 2944–2952 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhsVCrsbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14978097)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Toward%20a%20definition%20of%20self%3A%20proteomic%20evaluation%20of%20the%20class%20I%20peptide%20repertoire&journal=J.%20Immunol.&volume=172&pages=2944-2952&publication_year=2004&author=Hickman%2CHD) 
    
46.  Sachs, A. et al. Impact of cysteine residues on MHC binding predictions and recognition by tumor-reactive T cells. _J. Immunol._ **205**, 539–549 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVGhtbjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32571843)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Impact%20of%20cysteine%20residues%20on%20MHC%20binding%20predictions%20and%20recognition%20by%20tumor-reactive%20T%20cells&journal=J.%20Immunol.&volume=205&pages=539-549&publication_year=2020&author=Sachs%2CA) 
    
47.  Kennedy, P. R., Barthen, C., Williamson, D. J. & Davis, D. M. HLA-B and HLA-C differ in their nanoscale organization at cell surfaces. _Front. Immunol._ **10**, 61 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht12gsr7M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30761133)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6362897)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HLA-B%20and%20HLA-C%20differ%20in%20their%20nanoscale%20organization%20at%20cell%20surfaces&journal=Front.%20Immunol.&volume=10&publication_year=2019&author=Kennedy%2CPR&author=Barthen%2CC&author=Williamson%2CDJ&author=Davis%2CDM) 
    
48.  Pearlman, A. H. et al. Targeting public neoantigens for cancer immunotherapy. _Nat. Cancer_ **2**, 487–497 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsFeit7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34676374)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8525885)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20public%20neoantigens%20for%20cancer%20immunotherapy&journal=Nat.%20Cancer&volume=2&pages=487-497&publication_year=2021&author=Pearlman%2CAH) 
    
49.  Gurung, H. R. et al. Systematic discovery of neoepitope–HLA pairs for neoantigens shared among patients and tumor types. _Nat. Biotechnol._ **42**, 1107–1117 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFOqsL%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37857725)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20discovery%20of%20neoepitope%E2%80%93HLA%20pairs%20for%20neoantigens%20shared%20among%20patients%20and%20tumor%20types&journal=Nat.%20Biotechnol.&volume=42&pages=1107-1117&publication_year=2024&author=Gurung%2CHR) 
    
50.  Li, F. et al. Neoantigen vaccination induces clinical and immunologic responses in non-small cell lung cancer patients harboring EGFR mutations. _J. Immunother. Cancer_ **9**, e002531 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34244308)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8268925)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neoantigen%20vaccination%20induces%20clinical%20and%20immunologic%20responses%20in%20non-small%20cell%20lung%20cancer%20patients%20harboring%20EGFR%20mutations&journal=J.%20Immunother.%20Cancer&volume=9&publication_year=2021&author=Li%2CF) 
    
51.  Yu, B. et al. Engineered cell entry links receptor biology with single-cell genomics. _Cell_ **185**, 4904–4920.e22 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtV2ktrjP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36516854)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Engineered%20cell%20entry%20links%20receptor%20biology%20with%20single-cell%20genomics&journal=Cell&volume=185&pages=4904-4920.e22&publication_year=2022&author=Yu%2CB) 
    
52.  Kula, T. et al. T-Scan: a genome-wide method for the systematic discovery of T cell epitopes. _Cell_ **178**, 1016–1028 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFKktrvF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31398327)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6939866)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=T-Scan%3A%20a%20genome-wide%20method%20for%20the%20systematic%20discovery%20of%20T%20cell%20epitopes&journal=Cell&volume=178&pages=1016-1028&publication_year=2019&author=Kula%2CT) 
    
53.  Cimen Bozkus, C., Blazquez, A. B., Enokida, T. & Bhardwaj, N. A T-cell-based immunogenicity protocol for evaluating human antigen-specific responses. _STAR Protoc._ **2**, 100758 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVKhtLzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34458873)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8377590)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20T-cell-based%20immunogenicity%20protocol%20for%20evaluating%20human%20antigen-specific%20responses&journal=STAR%20Protoc.&volume=2&publication_year=2021&author=Cimen%20Bozkus%2CC&author=Blazquez%2CAB&author=Enokida%2CT&author=Bhardwaj%2CN) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02268-1?format=refman&flavour=references)

Acknowledgements
----------------

We thank members of the Chang and Yu laboratory and R. Schreiber (Washington University) for discussion. This study was supported by National Institutes of Health grant no. U54CA260517 (H.Y.C.), the Parker Institute for Cancer Immunotherapy (B.Y., H.Y.C.), the V Foundation (B.Y.),the Donald E. & Delia B. Baxter Foundation (B.Y.), Margaret E. Early Medical Research Trust (B.Y.) and Robert E. and May R. Wright Foundation (B.Y.). J.B. was supported by an HHMI Hanna Gray Fellowship. A.T.S. is supported by a Lloyd J. Old STAR award from the Cancer Research Institute, a Career Award for Medical Scientists from the Burroughs Wellcome Fund, a Pew-Stewart Scholars for Cancer Research Award and the CRISPR Cures for Cancer Initiative Award. H.Y.C. was an Investigator of the Howard Hughes Medical Institute.

Author information
------------------

Author notes

1.  Howard Y. Chang
    
    Present address: Amgen Research, South San Francisco, CA, USA
    

### Authors and Affiliations

1.  Department of Dermatology, Stanford University, Stanford, CA, USA
    
    Quanming Shi, Elana P. Simon, Julia A. Belk, Yaowen Zhang, Ashley Kwong, Yonglu Che & Howard Y. Chang
    
2.  Department of Genetics, Stanford University, Stanford, CA, USA
    
    Quanming Shi, Elana P. Simon, Julia A. Belk, Yaowen Zhang, Ashley Kwong, Yonglu Che & Howard Y. Chang
    
3.  Department of Biomedical Data Science, Stanford University School of Medicine, Stanford, CA, USA
    
    Elana P. Simon
    
4.  Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA
    
    Cansu Cimen Bozkus, Anna Kaminska, Leandra Velazquez, Mansi Saxena & Nina Bhardwaj
    
5.  Department of Molecular Microbiology and Immunology, Keck School of Medicine, University of Southern California, Los Angeles, CA, USA
    
    Zilin Zhang, Shuo Wang, Nuoya Yang & Bingfei Yu
    
6.  Department of Pathology, Stanford University School of Medicine, Stanford, CA, USA
    
    Robert R. Stickels, Theodore L. Roth & Ansuman T. Satpathy
    
7.  Ragon Institute of Mass General, MIT and Harvard, Cambridge, MA, USA
    
    Charles R. Crain, Laura Schmidt-Hong & Gaurav D. Gaiha
    
8.  Program in Immunology, Harvard Medical School, Boston, MA, USA
    
    Laura Schmidt-Hong
    
9.  Department of Pathology and Immunology, Washington University, St. Louis, MO, USA
    
    Cheryl F. Lichti
    
10.  Bursky Center for Human Immunology and Immunotherapy, Washington University, St. Louis, MO, USA
    
    Cheryl F. Lichti
    
11.  Division of Gastroenterology, Massachusetts General Hospital, Boston, MA, USA
    
    Gaurav D. Gaiha
    
12.  Program in Health Sciences and Technology, Harvard Medical School, Massachusetts Institute of Technology, Cambridge, MA, USA
    
    Gaurav D. Gaiha
    
13.  Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA
    
    Bingfei Yu
    
14.  Alfred E. Mann Department of Biomedical Engineering, University of Southern California, Los Angeles, CA, USA
    
    Bingfei Yu
    

Authors

1.  Quanming Shi
    
    [View author publications](https://www.nature.com/search?author=Quanming%20Shi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Quanming%20Shi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Quanming%20Shi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Elana P. Simon
    
    [View author publications](https://www.nature.com/search?author=Elana%20P.%20Simon)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Elana%20P.%20Simon) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Elana%20P.%20Simon%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Cansu Cimen Bozkus
    
    [View author publications](https://www.nature.com/search?author=Cansu%20Cimen%20Bozkus)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cansu%20Cimen%20Bozkus) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cansu%20Cimen%20Bozkus%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Anna Kaminska
    
    [View author publications](https://www.nature.com/search?author=Anna%20Kaminska)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anna%20Kaminska) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anna%20Kaminska%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Leandra Velazquez
    
    [View author publications](https://www.nature.com/search?author=Leandra%20Velazquez)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Leandra%20Velazquez) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Leandra%20Velazquez%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Mansi Saxena
    
    [View author publications](https://www.nature.com/search?author=Mansi%20Saxena)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mansi%20Saxena) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mansi%20Saxena%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Zilin Zhang
    
    [View author publications](https://www.nature.com/search?author=Zilin%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zilin%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zilin%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Julia A. Belk
    
    [View author publications](https://www.nature.com/search?author=Julia%20A.%20Belk)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Julia%20A.%20Belk) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Julia%20A.%20Belk%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Shuo Wang
    
    [View author publications](https://www.nature.com/search?author=Shuo%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shuo%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shuo%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Nuoya Yang
    
    [View author publications](https://www.nature.com/search?author=Nuoya%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nuoya%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nuoya%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Yaowen Zhang
    
    [View author publications](https://www.nature.com/search?author=Yaowen%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yaowen%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yaowen%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Ashley Kwong
    
    [View author publications](https://www.nature.com/search?author=Ashley%20Kwong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ashley%20Kwong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ashley%20Kwong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Yonglu Che
    
    [View author publications](https://www.nature.com/search?author=Yonglu%20Che)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yonglu%20Che) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yonglu%20Che%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Robert R. Stickels
    
    [View author publications](https://www.nature.com/search?author=Robert%20R.%20Stickels)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Robert%20R.%20Stickels) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Robert%20R.%20Stickels%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Charles R. Crain
    
    [View author publications](https://www.nature.com/search?author=Charles%20R.%20Crain)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Charles%20R.%20Crain) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Charles%20R.%20Crain%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Laura Schmidt-Hong
    
    [View author publications](https://www.nature.com/search?author=Laura%20Schmidt-Hong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Laura%20Schmidt-Hong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Laura%20Schmidt-Hong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Cheryl F. Lichti
    
    [View author publications](https://www.nature.com/search?author=Cheryl%20F.%20Lichti)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cheryl%20F.%20Lichti) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cheryl%20F.%20Lichti%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Gaurav D. Gaiha
    
    [View author publications](https://www.nature.com/search?author=Gaurav%20D.%20Gaiha)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gaurav%20D.%20Gaiha) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gaurav%20D.%20Gaiha%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Theodore L. Roth
    
    [View author publications](https://www.nature.com/search?author=Theodore%20L.%20Roth)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Theodore%20L.%20Roth) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Theodore%20L.%20Roth%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Nina Bhardwaj
    
    [View author publications](https://www.nature.com/search?author=Nina%20Bhardwaj)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nina%20Bhardwaj) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nina%20Bhardwaj%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Ansuman T. Satpathy
    
    [View author publications](https://www.nature.com/search?author=Ansuman%20T.%20Satpathy)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ansuman%20T.%20Satpathy) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ansuman%20T.%20Satpathy%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Bingfei Yu
    
    [View author publications](https://www.nature.com/search?author=Bingfei%20Yu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Bingfei%20Yu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Bingfei%20Yu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Howard Y. Chang
    
    [View author publications](https://www.nature.com/search?author=Howard%20Y.%20Chang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Howard%20Y.%20Chang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Howard%20Y.%20Chang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Q.S., B.Y. and H.Y.C. conceived the idea. Q.S. and B.Y. designed the experiments. Q.S. performed the screening with Z.Z., S.W. and N.Y. Y.Z. and A. Kwong assisted in experiments. Q.S., B.Y. and E.P.S. analyzed the data and J.A.B. assisted with analysis. C.C.B., A. Kaminska, L.V., M.S. and N.B. contributed to the PBMC epitope validation experiment and analysis. C.R.C., L.S.-H. and G.D.G. helped with epitope discovery and prioritization. C.F.L. advised on MS experiments. A.T.S., Y.C., R.R.S. and T.L.R. helped with data interpretation for the project. Q.S., B.Y. and H.Y.C. wrote the manuscript with input from co-authors. H.Y.C. and B.Y. supervised the project.

### Corresponding authors

Correspondence to [Bingfei Yu](mailto:bingfeiy@usc.edu) or [Howard Y. Chang](mailto:howchang@stanford.edu).

Ethics declarations
-------------------

### Competing interests

Stanford University has filed a patent based on this work in which Q.S., B.Y. and H.Y.C. are named as inventors. H.Y.C. is a co-founder of Accent Therapeutics, Boundless Bio, Cartography Biosciences and Orbital Therapeutics, and an advisor to 10x Genomics, Arsenal Biosciences, Chroma Medicine and Exai Bio until 15 December 2024. H.Y.C. is an employee and stockholder of Amgen as of 16 December 2024. T.L.R. is a co-founder and former Chief Scientific Officer of Arsenal Bioscience, and an advisor to NewLimit. A.T.S. is a co-founder of Immunai, Cartography Biosciences, Santa Ana Bio and Prox Biosciences, is an advisor to 10x Genomics and Wing Venture Capital, and receives research funding from Astellas Pharma. N.B. is an extramural PICI member. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Chunhong Zheng and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02268-1#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 ESCAPE provides consistent readout across HLA-A, B and C alleles.](https://www.nature.com/articles/s41588-025-02268-1/figures/7)

**a**, Histogram of β2m staining in wild-type (WT) HEK293T cells, HEK293T cells with HLA-A, -B, and -C knocked out using CRISPR-Cas9 RNP (HLA-KO), and HLA-KO HEK293T cells expressing NYESO pHLA-A0201 single-chain trimer (SCT). Bottom: isotype antibody staining of HLA-KO HEK293T cells expressing pHLA-A0201. **b**, Bar plots quantifying the geometric mean fluorescence intensity (gMFI) of β2m staining in HEK293T cells expressing SCTs of HLA-A0201 fused to high-affinity (HP) peptides versus peptides with no binding affinity (NP) in Fig.[1b](https://www.nature.com/articles/s41588-025-02268-1#Fig1). Two peptides were tested for both HP and NP groups. **c**, T-cell stimulation assay showing that NYESO pHLA-A0201-expressing cells induce eGFP expression in T cells co-expressing a cognate 1G4 TCR and an NFAT reporter. In contrast, pp65 pHLA-A0201-expressing cells do not stimulate T cells. **_d_**, Direct staining of OVA-pH-2Kb-expressing cells using anti-SIINFEKL pH-2Kb and anti-H-2Kb antibodies. **_e_**, gMFI of cells expressing SCTs fused with HP and NP peptides for HLA-A0101, HLA-A_0201, HLA-B_0702, and HLA-C\*0401. **f**, Motif analysis of amino acids surrounding the conserved Y84 residue across all HLA-A, -B, and -C alleles. The arrow indicates the conserved Y84 residue, which is mutated to Y84C in SCT constructs. **g**, Effect of TAP1/2 knock-out. gMFI quantification of HP and NP with A\*02, B\*07 and C\*04 alleles in both HLA KO cells and HLA/TAP dual KO cells. **h**, Fluorescent image of direct eGFP fusion with B2M-A\*02 allele (top) and A\*02 allele (bottom). **i**, Compare the effect of TAP KO in Fig. [1f](https://www.nature.com/articles/s41588-025-02268-1#Fig1). In either HLA KO cells or HLA/TAP dual KO cells, the gMFI of B2M-A2 fusion with or without co-expression of pp65 peptide were compared. \*\*\* p < 0.001; \*\*\*\* p < 0.0001. Number of replicates n are depicted on bar plots. Error bars represent standard error of mean (SEM).

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

### [Extended Data Fig. 2 Comparison of ESCAPE-seq and NetMHC prediction using IEDB training data.](https://www.nature.com/articles/s41588-025-02268-1/figures/8)

**a**, Representative correlation plot of read count between replicates for ESCAPE screen. HLA-A\*02 was used here an example. 4 fractions (BG/background, low, medium and high) were highlighted in different colors. Pearson correlation coefficients were shown. **b**, Correlation of E-score between replicate of ESCAPE-seq using HLA alleles including A\*02, A\*01, B\*07 and C\*04. Pearson correlation coefficients of each were shown in the plots. **c**, Scatter plot of IEDB affinities vs. ESCAPE-seq E-score for wild-type HLA-A\*0201 allele. **d**, Scatter plot of IEDB affinities vs. ESCAPE-seq E-score for HLA-A\*0101 allele without Y84C mutation. **e**, ROC curves of ESCAPE, NetMHC\_el and NetMHC\_ba as comparison using IEDB affinity as ground truth. **f**, Plot of recall rate of ESCAPE-seq vs. IEDB affinity for HLA-A\*0101 allele. NetMHC\_ba and NetMHC\_el prediction results were shown as a comparison. **g-i**, Similar plots as (**d-f**) for HLA-B\*0702 allele. **j-k**, Similar plots as (**c,e**) for HLA-C\*0401 allele. **l**, Comparison of Pearson correlation coefficient for ESCAPE-seq, NetMHC\_ba and NetMHC\_el predictions across 4 HLA alleles examined here. **m**, Similar to (k) for Spearson correlation coefficient. N = 6, error bars represents SEM.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

### [Extended Data Fig. 3 Comparison of ESCAPE-seq and NetMHC prediction on peptides from SARS-CoV-2 virus.](https://www.nature.com/articles/s41588-025-02268-1/figures/9)

**a**, Representative correlation plot of read count between replicates for HLA-A\*0201. Pearson correlation coefficients were shown. **b**, Correlation of E-score between biological replicates for SARS antigen screening for HLA-A\*0201, HLA-A\*0101 and HLA-B\*0702 alleles. Pearson correlation coefficients were shown in each plot. **c**, Plot of NetMHC\_el rank and NetMHC\_ba vs. E-score for HLA-A\*0201. **d**, The same plot as (c) for HLA-A\*0101. **e**, The same plots as (c) for HLA-B\*0702. **f**, Plots of count and percentage of presentable peptides (y-axis) vs E-score cutoffs. Error bars are SEM from n = 2 biological replicates. 3 HLA alleles are compared. Replicate n = 2. **g**, Bar plot of ROC-AUC of NetMHC with ESCAPE E-scores of peptides from IEDB training database (IEDB) and from SARS-CoV2 proteins (SARS-CoV2) for A\*01, A\*02 and B\*07 3 alleles in x-axis. **h**, The same plot as (g) for PRC-AUC metrics across the 2 peptide pools. \*\*\*\* notes p-value < 0.0001; n = 6. **i**, Scatter plots of E-score of antigen peptides containing variant mutation (Mut, x-axis) vs E-score of its corresponding wild-type peptide (WT, y-axis) for spike-HLA-A\*0101 pairs. **j**, The same plots for protein N pHLA pairs for all 3 alleles. **k**, Quantification of percentage of presented peptides in each group in S3j.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

### [Extended Data Fig. 4 Characterization of peptide length and performance of ESCAPE-seq using MS immunopeptidome data.](https://www.nature.com/articles/s41588-025-02268-1/figures/10)

**a**, Representative correlation plot of read counts between replicates for ESCAPE-seq performed on a pool of peptides from the MS dataset. Pearson correlation coefficients are shown in the plot. **b**, Scatter plot of calculated E-scores between replicates from (**a**). **c**, E-score distributions for peptides that were not detected (ND) or detected by MS. Examples are shown for the HLA-A\*_03:01 and HLA-A\*_02:01 alleles. **d**, Representative ROC curves for 4 alleles (A\*_02:01, B\*_07:02, C\*_03:04, and C\*_04:01) out of a total of 30 alleles in the pool. **e**, Across all 30 alleles, comparison of E-scores with NetMHC predictions in both EL (eluted ligand) and BA (binding affinity) modes. Arrows highlight alleles where E-scores show better AUC-ROC metrics. **f**, Schematic of ESCAPE-seq screening for peptides of 8–12 amino acids in length across 25 HLA alleles. **g**, Scatter plots of read counts between biological replicates, and **h**, scatter plots of calculated E-scores between biological replicates. Pearson correlation coefficients are shown in each plot. **i**, Bar plot showing the number of presented peptides identified by ESCAPE-seq at different peptide lengths. **j**, E-score heatmap for peptides generated from the same gene locus with varying lengths per allele. Examples include peptides derived from EGFR T790M presented by HLA-B\*_52:01 and peptides tiling across_ KRAS G12D _presented by HLA-A\*_68:01. The heatmap illustrates changes in E-scores when an epitope is extended by one amino acid at either the N- or C-terminus. In some cases, peptide presentability remains consistent when an additional amino acid is added to the N-terminus (left and right panels) or the C-terminus (right panel).

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

### [Extended Data Fig. 5 Combinatorial ESCAPE-seq benchmark and MS validation of selected cancer neoantigens.](https://www.nature.com/articles/s41588-025-02268-1/figures/11)

**a**, Representative histogram of B2M staining of HEK-KO cells transduced with the combinatorial virus pool. 4 fractions (termed Background, Low, Medium and High) were sorted based on staining signals on log-scale. **b**, The correlation of reads (**left**) and E-score (**right**) of combinatorial ESCAPE-seq. **c**, Heatmap showing the normalized read count for spike-in peptide HLA pairs. Here the input pool only consisted of spike-in peptide paired with HLA group-a. All reads from paired group-b were recombination events from various steps involved in the screen, such as template switch in virus pool, chimeric reads from PCR amplification and library generation, or during sequencing. **d**, The violin plot showed the quantification of read counts in (**c**). In each violin plot, the top and lower bars represent the mean and median. inset: zoom in plot. **e**,Normalized distribution of E-score for each HLA-I allele, colored in HLA-A, B, C subtypes. **f**, Percentage of presented peptides varies across different HLA alleles. **g**, Intensity traces show the detection of peptides (YIMSDSNYV, LTSTVQLIM and SSYGQQSSL) with Mass spectroscopy (MS). **h**, Plot of the percentage of driver mutations from (Fig. [5e](https://www.nature.com/articles/s41588-025-02268-1#Fig5)) that were presented by each HLAs in x-axis. **i**, Presented peptide percentage for oncogenic fusion peptides. **j**, Plot of presented fusion rate from (Fig. [5f](https://www.nature.com/articles/s41588-025-02268-1#Fig5)) across HLA-I. **k**, Histogram of percentage of driver mutations (red line) or gene fusion mutations (blue line) can be presented by diploid human cells, where we repeatedly sampled 2 alleles from HLA-A, B, C alleles 1000 times to calculate the mutation coverage to generate the distribution.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

### [Extended Data Fig. 6 Characterization of WT vs Mut antigen presentation across HLA alleles and validation with T-cell stimulation assay.](https://www.nature.com/articles/s41588-025-02268-1/figures/12)

**a**, Bar plot showing the percentage of presented peptides in the WT + , Mut + , and WT+Mut+ (both) groups, as depicted in Fig. [6a-b](https://www.nature.com/articles/s41588-025-02268-1#Fig6), across all HLA alleles examined in this study. The arrow highlights the bar for HLA-A\*23:01. **b**, Bar plot showing the percentage of presented peptides in the WT + , Mut + , and WT+Mut+ (both) groups, as depicted in Fig. [6c](https://www.nature.com/articles/s41588-025-02268-1#Fig6), across different oncogene mutations, ranked by the percentage of peptides in the Mut+ group. **c**, Gating strategy for the T-cell stimulation assay. Cells were gated sequentially by FSC/SSC, single cells, live cells, and CD8 + T cells. Double-positive cells for IFN-γ and TNF were quantified and used in the bar plots shown in Fig. [6g](https://www.nature.com/articles/s41588-025-02268-1#Fig6) and Fig. [6i](https://www.nature.com/articles/s41588-025-02268-1#Fig6). **d**, Scatter plot of E-scores for mutant peptides (x-axis) versus the E-scores of their corresponding wild-type (WT) peptides (y-axis) for the HLA-A\*2301 allele. Dashed lines indicate E-score thresholds that categorize peptides into four groups. Peptides from the EGFR T790M mutation (p17, p22, and p23) are labeled. **e**, Flow cytometry plot corresponding to Fig. [6h](https://www.nature.com/articles/s41588-025-02268-1#Fig6), showing stimulation by a peptide pool.

[Source data](https://www.nature.com/articles/s41588-025-02268-1#MOESM10)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM1_ESM.pdf)

Supplementary Tables 1–3, 5 and 8, Notes 1 and 2 and Fig. 1.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM3_ESM.pdf)

### [Supplementary Data 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM4_ESM.pdf)

Methods and QC.

### [Supplementary Data 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM5_ESM.xlsx)

Results peaks from MS.

### [Supplementary Table 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM6_ESM.xls)

List of oncogenic point mutations and gene fusion junctions, related to Figs. 4 and 5.

### [Supplementary Table 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM7_ESM.xlsx)

List of spike-in peptides for Figs. 4 and 5.

### [Supplementary Table 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM8_ESM.xlsx)

List of known antigens in oncogene pools for Figs. 4 and 5.

### [Supplementary Code](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM9_ESM.zip)

Python script to read fastq files and output a table of _E_\-score, with annotation in the code file.

Source data
-----------

### [Source Data](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02268-1/MediaObjects/41588_2025_2268_MOESM10_ESM.xls)

Contains data sheet for main Figs. 1, 2 and 6 and Extended Data Figs. 1–3. Tables of A0201IEDB, A0101IEDB and B0702IEDB are _E_\-scores for IEDB peptides, used in Fig. 2 and Extended Data Fig. 2. Tables of A0201SAR2, A0101SAR2 and B0702SAR2 contain ESCAPE-seq reads and scores/statistics of SAR2 peptides, used to generate Fig. 3 and Extended Data Fig. 3. Tables of oncogene and onco\_wt contain all the combinatorial ESCAPE-seq results of the oncogenic peptides pool, where onco\_wt contains the _E_\-score for corresponding wild-type peptides. They were used for Figs. 4–6 and Extended Data Figs. 4–6.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Massively%20parallel%20immunopeptidome%20by%20DNA%20sequencing%20provides%20insights%20into%20cancer%20antigen%20presentation&author=Quanming%20Shi%20et%20al&contentID=10.1038%2Fs41588-025-02268-1&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-28&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02268-1)

### Cite this article

Shi, Q., Simon, E.P., Cimen Bozkus, C. _et al._ Massively parallel immunopeptidome by DNA sequencing provides insights into cancer antigen presentation. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02268-1

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02268-1?format=refman&flavour=citation)

-   Received: 09 February 2024
    
-   Accepted: 16 June 2025
    
-   Published: 28 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02268-1
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02268-1.pdf)
