> id: https://www.nature.com/articles/s41588-025-02250-x

> link: https://www.nature.com/articles/s41588-025-02250-x

> title: Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy

# Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy
_Published on Tue, 15 Jul 2025 00:00:00 GMT_

Abstract
--------

With age, clonal expansions occur pervasively across normal tissues yet only in rare instances lead to cancer, despite being driven by well-established cancer drivers. Characterization of the factors that influence clonal progression is needed to inform interventional approaches. Germline genetic variation influences cancer risk and shapes tumor mutational profile, but its influence on the mutational landscape of normal tissues is not well known. Here we studied the impact of germline genetic variation on clonal hematopoiesis (CH) in 731,835 individuals. We identified 22 new CH-predisposition genes, most of which predispose to CH driven by specific mutational events. CH-predisposition genes contribute to unique somatic landscapes, reflecting the influence of germline genetic backdrop on gene-specific CH fitness. Correspondingly, somatic–germline interactions influence the risk of CH progression to hematologic malignancies. These results demonstrate that germline genetic variation influences somatic evolution in the blood, findings that likely extend to other tissues.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41576-021-00356-6/MediaObjects/41576_2021_356_Fig1_HTML.png)

### [Germline risk of clonal haematopoiesis](https://www.nature.com/articles/s41576-021-00356-6?fromPaywallRec=false)

Article 13 May 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-022-01121-z/MediaObjects/41588_2022_1121_Fig1_HTML.png)

### [Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis](https://www.nature.com/articles/s41588-022-01121-z?fromPaywallRec=false)

Article Open access 14 July 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01755-1/MediaObjects/41588_2024_1755_Fig1_HTML.png)

### [Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis](https://www.nature.com/articles/s41588-024-01755-1?fromPaywallRec=false)

Article Open access 14 May 2024

Main
----

Cancer is a genetic disease stemming from a combination of inherited and acquired mutations. Much of our understanding of the influence of germline genetic factors on carcinogenesis comes from studies of tumor genomes. Tumors from germline mutation carriers may show characteristic genomic patterns described as mutational signatures, which reflect unique processes of mutagenesis[1](https://www.nature.com/articles/s41588-025-02250-x#ref-CR1 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)."),[2](https://www.nature.com/articles/s41588-025-02250-x#ref-CR2 "Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. Nature 534, 47–54 (2016)."),[3](https://www.nature.com/articles/s41588-025-02250-x#ref-CR3 "Campbell, P. J. et al. Pan-cancer analysis of whole genomes. Nature 578, 82–93 (2020)."),[4](https://www.nature.com/articles/s41588-025-02250-x#ref-CR4 "Middlebrooks, C. D. et al. Association of germline variants in the APOBEC3 region with cancer risk and enrichment with APOBEC-signature mutations in tumors. Nat. Genet. 48, 1330–1338 (2016)."),[5](https://www.nature.com/articles/s41588-025-02250-x#ref-CR5 "Nik-Zainal, S. et al. Association of a germline copy number polymorphism of APOBEC3A and APOBEC3B with burden of putative APOBEC-dependent mutations in breast cancer. Nat. Genet. 46, 487–491 (2014)."). Beyond mutagenesis, germline genetic variation shapes tissue-specific mutational fitness, with clones bearing a selective advantage attaining dominance at the expense of others[6](https://www.nature.com/articles/s41588-025-02250-x#ref-CR6 "Yu, C.-C. et al. Mutant allele specific imbalance in oncogenes with copy number alterations: Occurrence, mechanisms, and potential clinical implications. Cancer Lett. 384, 86–93 (2017)."),[7](https://www.nature.com/articles/s41588-025-02250-x#ref-CR7 "Chatrath, A., Ratan, A. & Dutta, A. Germline variants that affect tumor progression. Trends Genet. 37, 433–443 (2021)."). It is increasingly clear that these processes of mutation acquisition and selection are not limited to tumors but pervasive across normal tissues. Deep sequencing of normal tissues has shown that expansion of clones bearing acquired mutations in well-established cancer drivers is pervasive with age[8](https://www.nature.com/articles/s41588-025-02250-x#ref-CR8 "Martincorena, I. et al. Universal patterns of selection in cancer and somatic tissues. Cell 171, 1029–1041.e21 (2017)."),[9](https://www.nature.com/articles/s41588-025-02250-x#ref-CR9 "Salk, J. J. et al. Ultra-sensitive TP53 sequencing for cancer detection reveals progressive clonal selection in normal tissue over a century of human lifespan. Cell Rep. 28, 132–144.e3 (2019)."),[10](https://www.nature.com/articles/s41588-025-02250-x#ref-CR10 "Martincorena, I. et al. Somatic mutant clones colonize the human esophagus with age. Science 
https://doi.org/10.1126/science.aau3879
(2018)."),[11](https://www.nature.com/articles/s41588-025-02250-x#ref-CR11 "Blokzijl, F. et al. Tissue-specific mutation accumulation in human adult stem cells during life. Nature 538, 260–264 (2016)."),[12](https://www.nature.com/articles/s41588-025-02250-x#ref-CR12 "Brunner, S. F. et al. Somatic mutations and clonal dynamics in healthy and cirrhotic human liver. Nature 574, 538–542 (2019)."),[13](https://www.nature.com/articles/s41588-025-02250-x#ref-CR13 "Moore, L. et al. The mutational landscape of normal human endometrial epithelium. Nature 580, 640–646 (2020)."),[14](https://www.nature.com/articles/s41588-025-02250-x#ref-CR14 "Martincorena, I. et al. Tumor evolution. High burden and pervasive positive selection of somatic mutations in normal human skin. Science 348, 880–886 (2015)."),[15](https://www.nature.com/articles/s41588-025-02250-x#ref-CR15 "Yoshida, K. et al. Tobacco smoking and somatic mutations in human bronchial epithelium. Nature 578, 266–272 (2020)."). Interestingly, these mutant clones progress to cancer in only a small minority of individuals. Improved characterization of the factors promoting cancer transformation is critical to inform prevention strategies and develop new therapeutic approaches.

Our understanding of the natural history and impact of somatic mutations on health is understood best within the hematopoietic system, because it is the only tissue in which sampling (in this case of the peripheral blood) reflects the pooled collective output of all stem cells. Mutant hematopoietic stem and progenitor cells (HSPCs) show varying fitness advantages, largely dictated by gene-specific effects[16](https://www.nature.com/articles/s41588-025-02250-x#ref-CR16 "Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)."),[17](https://www.nature.com/articles/s41588-025-02250-x#ref-CR17 "Robertson, N. A. et al. Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects. Nat. Med. 28, 1439–1446 (2022)."). This, in turn, translates into risk of progression to hematologic malignancy, with clones showing the highest fitness generally conferring the highest risk of transformation[17](https://www.nature.com/articles/s41588-025-02250-x#ref-CR17 "Robertson, N. A. et al. Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects. Nat. Med. 28, 1439–1446 (2022)."). However, variation exists among mutation-specific effects between individuals. which may be explained by both inherited and environmental factors. There has recently emerged a preliminary understanding of how germline factors influence clonal hematopoiesis (CH). Among individuals with rare Mendelian cancer predisposition syndromes, carriers often show distinct CH mutational profiles reflecting strong selective pressures[18](https://www.nature.com/articles/s41588-025-02250-x#ref-CR18 "Kennedy, A. L. et al. Distinct genetic pathways define pre-malignant versus compensatory clonal hematopoiesis in Shwachman–Diamond syndrome. Nat. Commun. 12, 1334 (2021)."),[19](https://www.nature.com/articles/s41588-025-02250-x#ref-CR19 "Churpek, J. E. et al. Genomic analysis of germ line and somatic variants in familial myelodysplasia/acute myeloid leukemia. Blood 126, 2484–2490 (2015)."),[20](https://www.nature.com/articles/s41588-025-02250-x#ref-CR20 "Brown, A. L. et al. RUNX1-mutated families show phenotype heterogeneity and a somatic mutation profile unique to germline predisposed AML. Blood Adv. 4, 1131–1144 (2020)."),[21](https://www.nature.com/articles/s41588-025-02250-x#ref-CR21 "Liggett, L. A. et al. Precocious clonal hematopoiesis in Down syndrome is accompanied by immune dysregulation. Blood Adv. 5, 1791–1796 (2021)."). However, the extent to which germline genetic variation might influence CH fitness and progression to hematologic malignancy in the general population has not been systematically studied.

Here among 731,835 individuals across 6 diverse cohorts, we sought to characterize the relationship between germline genetic variation on CH mutational landscape and how germline–somatic interactions influence the risk of CH progression to hematologic malignancy.

### UKBB germline and CH mutational landscape

In 428,530 UK Biobank (UKBB) participants with whole-exome sequencing (WES) data (Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)–[4](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)), we queried 236 cancer predisposition genes (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)) for inherited mutations, henceforth called pathogenic or likely pathogenic germline variants (PGVs), using the American College of Medical Genetics and Genomics (ACMG) criteria[22](https://www.nature.com/articles/s41588-025-02250-x#ref-CR22 "Richards, S. et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet. Med. 17, 405–424 (2015)."). We refer to individuals with PGVs as germline carriers throughout. We classified genes according to their inheritance mode and evidence of previous association with hematologic malignancy. Overall, 8% of UKBB participants harbored a PGV in a gene with a dominant inheritance mode (germline dominant) and 10% in a gene with a recessive inheritance mode (germline recessive). The vast majority of germline carriers were heterozygous (99.9%), with only 87 individuals found to be homozygous or to carry two different PGVs in the same gene (potential compound heterozygous carriers). Similar to previous studies in western European populations[23](https://www.nature.com/articles/s41588-025-02250-x#ref-CR23 "Hu, C. et al. A population-based study of genes previously implicated in breast cancer. N. Engl. J. Med. 384, 440–451 (2021)."), _CHEK2_ (0.9%) was the most commonly mutated gene with a dominant inheritance mode, followed by _ATM_ (0.5%) and _BRCA2_ (0.4%) (Fig. [1a](https://www.nature.com/articles/s41588-025-02250-x#Fig1) and Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) and [6](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). The vast majority harbored one PGV with only 1,327 (0.3%) participants having PGVs in multiple genes with a dominant inheritance mode. As expected, PGV carriers were more likely to have a history of cancer and to be diagnosed with cancer at a younger age (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)).

**Fig. 1: Germline and CH mutational landscape of the UKBB.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02250-x/figures/1)

**a**, Distribution of pathogenic germline variants by mutation type for the top-10 most-mutated dominant and recessive germline genes. Genes were classified as to whether they have been linked to any cancer in the heterozygous state (dominant) or whether they have been linked to cancer only when biallelic (recessive). **b**, Prevalence of CH-heme and mCA-auto by age among people stratified by germline carrier status. CH-heme stands for CH in genes with known relevance to hematologic malignancy and mCA-auto for autosomal mosaic chromosomal alterations. Data are presented as the CH prevalence fitted using polynomial regression of degree 2 (center line) ± 95% CI for the fitted line (error bands). ORs with 95% CIs were calculated using a multivariable logistic regression model comparing the odds of having CH between people with dominant (_n_ = 33,106) or recessive (_n_ = 43,981) germline variants in reference and those without a germline variant (_n_ = 354,774) after adjustment for age at blood draw, the first three genetic PCs and exome sequencing batch. **c**, Prevalence of CH-heme in specific genes and mCA-auto types by germline carrier status.mCAs are labeled by chromosome arm and alteration type: gain (+), loss (–), or copy-neutral loss of heterozygosity (=). Multivariable logistic regression adjusted for the above covariates was performed to test for differences in the prevalence of specific CH mutations between people with (_n_ = 73,756) and those without (_n_ = 354,774) germline variants. \*_P_ < 0.05, \*\*_P_ < 0.01, \*\*\*_P_ < 0.001. The two-sided _P_ value is not corrected for multiple testing (see Supplementary Table [10](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for exact _P_ values).

To identify CH, we re-analyzed blood WES data using the consensus of two somatic variant callers (Mutect2 and VarDict). A series of post-variant calling filtering steps were used to remove germline variants and artifacts and detect CH in cancer driver genes, with a minimum variant allele fraction (VAF) of 2% ([Methods](https://www.nature.com/articles/s41588-025-02250-x#Sec7) and Supplementary Tables [7](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) and [8](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Within The Cancer Genome Atlas (TCGA), we used matched blood and tumor genomic sequencing to test the accuracy of our approach in discriminating CH from rare germline genetic variants. Applying the same strategy to detect CH in TCGA, we confirmed that >99% of our CH calls were correctly assigned. Overall, 6.2% of individuals had CH in a hematologic malignancy driver gene (CH-heme) and 0.7% in a solid tumor driver gene (CH-solid). As expected, the frequency of CH increased with age (Fig. [1b](https://www.nature.com/articles/s41588-025-02250-x#Fig1), Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02250-x#Fig6) and Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)). CH-heme but not CH-solid was more prevalent among germline carriers. Germline-dominant carriers had a stronger association with CH-heme compared with germline-recessive carriers (Fig. [1b](https://www.nature.com/articles/s41588-025-02250-x#Fig1), Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02250-x#Fig6) and Supplementary Table [9](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). The maximum VAF (_P_ < 0.001) and number of CH-heme mutations (_P_ = 0.002) were also slightly higher among germline-dominant carriers (Extended Data Fig. [1c](https://www.nature.com/articles/s41588-025-02250-x#Fig6)).

We interrogated the SNP array data using a well-established copy number mutation caller, MoChA[24](https://www.nature.com/articles/s41588-025-02250-x#ref-CR24 "Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."), for the presence of mosaic copy number events (mCAs). The prevalence of mCAs in autosomal chromosomes (mCA-auto), loss of X chromosome (LOX) and loss of Y chromosome (LOY) were 3.1%, 3.2% and 8.6%, respectively (Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02250-x#Fig6)). Germline carriers had a higher risk of mCA-auto and LOY but not LOX, driven by genes with a dominant inheritance mode (Fig. [1b](https://www.nature.com/articles/s41588-025-02250-x#Fig1), Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02250-x#Fig6) and Supplementary Table [9](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Copy neutral loss of heterozygosity (CNLOH) was the most common event observed with the association between germline carriers and mCA-auto driven by CNLOH (Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02250-x#Fig6)). Among the top-10 most commonly mutated CH-heme genes, six were slightly enriched among germline carriers, with only _DNMT3A_ and _ASXL1_ being statistically significant (Fig. [1c](https://www.nature.com/articles/s41588-025-02250-x#Fig1) and Supplementary Table [10](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Similarly, an increased frequency of mCA-auto was observed across multiple genomic regions but was only statistically significant for 1p CNLOH, 11q CNLOH and loss of 13q and 15q CNLOH (Fig. [1c](https://www.nature.com/articles/s41588-025-02250-x#Fig1) and Supplementary Table [10](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). In summary, we observed a high frequency of individuals harboring a PGV in a cancer susceptibility gene in the UKBB and that germline carriers had a higher frequency of CH, specifically CH driven by hematologic driver genes and CNLOH events in autosomal chromosomes, suggesting germline selection for specific somatic events.

### Germline predisposition to CH

Given the association between PGVs in cancer predisposition genes and CH, we next sought to identify specific genes that conferred a higher risk of CH. We focused subsequent analyses on CH-heme and mCA-auto events because these were most strongly associated with germline carrier status. Using multivariable logistic regression adjusted for age at blood draw, the first three genetic principal components (PCs) and exome sequencing batch, we identified 14 genes associated with CH (false discovery rate (FDR)-corrected _P_ value: _q_ < 0.05; Fig. [2a](https://www.nature.com/articles/s41588-025-02250-x#Fig2) and Supplementary Table [11](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). These included genes implicated in DNA damage repair (DDR) or sensing (_CHEK2_, _ATM_, _TP53_ and _NBN_), telomere maintenance (_POT1_, _TINF2_ and _CTC1_), RAS signaling (_PTPN11_ and _SOS1_) and the JAK–STAT pathway (_MPL_). Also included were _ETV6_ and _RUNX1_, genes encoding transcription factors, _SAMD9L_, encoding a tumor suppressor, and _ABCB11_, which encodes a bile salt exporter pump in the liver. Most are known or hypothesized hematologic cancer predisposition genes. _ABCB11_ has not been previously linked to hematologic cancer. Although biallelic _NBN_ mutations have been associated with hematologic cancer[25](https://www.nature.com/articles/s41588-025-02250-x#ref-CR25 "Escherich, C. S. et al. Germ line genetic NBN variation and predisposition to B-cell acute lymphoblastic leukemia in children. Blood 143, 2270–2283 (2024)."), heterozygous _NBN_ carriers have not been linked to subtypes of hematologic malignancy, although there is an association with overall cancer predisposition[26](https://www.nature.com/articles/s41588-025-02250-x#ref-CR26 "Belhadj, S. et al. NBN pathogenic germline variants are associated with pan-cancer susceptibility and in vitro DNA damage response defects. Clin. Cancer Res. 29, 422–431 (2023)."). We tested for an association between these genes and CH in five validation cohorts: All of Us, Mass General Brigham Biobank (MGBB), TCGA, Memorial Sloan Kettering–Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT) and Center for Common Disease Genomics (CCDG), which included a total of 24,803 CH carriers among 303,305 individuals. In total, eight were significantly associated with CH in the replication cohorts (_RUNX1_ (_P_ = 1.5 × 10−2 for CH-heme), _MPL_ (_P_ = 1.2 × 10−24 for mCA-auto), _TP53_ (_P_ = 2.0 × 10−6 for CH-heme), _ATM_ (_P_ = 2.0 × 10−2 for CH-heme; _P_ = 8.9 × 10−12 for mCA-auto), _NBN_ (_P_ = 4.7 × 10−2 for mCA-auto), _CHEK2_ (_P_ = 2.9 × 10−7 for CH-heme), _ETV6_ (_P_ = 1.1 × 10−3 for CH-heme) and _PTPN11_ (_P_ = 4.7 × 10−2 for CH-heme)) (Fig. [2a](https://www.nature.com/articles/s41588-025-02250-x#Fig2), Supplementary Table [11](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) and Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)). All were directionally consistent besides _SOS1_ where only a small number of germline carriers were observed (_n_ = 15) and none had CH. Out of the eight genes that were significantly associated with CH in the replication cohort, two have not been previously associated with CH (_NBN_ and _PTPN11_). Among CH-positive individuals, some but not all CH susceptibility germline carriers showed slightly higher CH VAF and mutational burden compared with individuals without germline pathogenic variants (Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02250-x#Fig7)).

**Fig. 2: Germline predisposition to CH.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02250-x/figures/2)

**a**, Within the UKBB, we identified 14 cancer predisposition genes that were associated with CH-heme (red) or mCA-auto (blue). Data are presented as OR (dot) ± 95% CI (whiskers). Black diamonds indicate ORs and 95% CIs from a fixed-effects meta-analysis in our replication cohorts, which include the All of Us (_n_ = 192,003), MGBB (_n_ = 49,941), the Washington University CCDG (_n_ = 37,184), TCGA (_n_ = 7,161) and MSK-IMPACT (_n_ = 17,016) cohorts. **b**, Heatmap showing the log(OR) within the UKBB between CH-heme in specific genes and germline genes that were statistically significantly (FDR-corrected _P_ < 0.05) associated with higher risk of overall CH-heme. **c**, Heatmap showing the log(OR) between specific mCA-auto types and germline genes that significantly increased overall mCA-auto. The color scale is the same for **b** and **c**. Pair-wise associations that were statistically significant (_P_ < 0.05, two sided with no correction for multiple testing) are shown in the black box in our replication cohort (solid line) and those that were directionally consistent (dashed line). For all analyses, the OR for CH was calculated using multivariable, Firth’s bias-reduced, logistic regression comparing germline carriers with individuals without germline variants, adjusted for age at blood draw, the first three genetic PCs and exome sequencing batch. \*_q_ (FDR-corrected _P_) < 0.05, \*\*_q_ < 0.01, \*\*\*_q_ < 0.001.

Tumors from germline mutation carriers can show unique mutational signatures. To this end, we investigated whether the trinucleotide context of CH mutations differed between germline carriers and noncarriers. Even among mismatch repair germline carriers with tumors that are known to show distinct signature patterns[27](https://www.nature.com/articles/s41588-025-02250-x#ref-CR27 "Georgeson, P. et al. Evaluating the utility of tumour mutational signatures for identifying hereditary colorectal cancer and polyposis syndrome carriers. Gut 70, 2138–2149 (2021)."), the SBS1 ‘clock-like’ mutational signature predominated for CH mutations (Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02250-x#Fig7) and Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)). The SBS1 signature is characterized by a predominance of C>T substitutions, in particular when cytosine is followed by guanine (CpG). Among both noncarriers and germline carriers, the proportion of CpG substitutions was not significantly different even after adjusting for age (_P_ = 0.50). This suggests that, similar to CH in individuals without PGVs, CH in middle-aged germline variant carriers is driven largely by age-related mutational processes.

Among the 14 germline CH-predisposition genes that we identified in the UKBB, we observed marked heterogeneity in the strength of associations with acquired mutations in specific genes (Fig. [2b](https://www.nature.com/articles/s41588-025-02250-x#Fig2)). Some of these reflect known patterns of acquired mutational selection in PGV carriers (for example, PGVs in _RUNX1_ and acquired _SRSF2_) or patterns of co-mutational selection in hematologic cancers (for example, _TP53_ with 7p- in AML). However, most have not been reported (Supplementary Table [12](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Among 24 CH-germline gene-specific associations where we observed ≥1 co-occurring events in our validation cohorts, 18 were significant (_P_ < 0.05) and an additional 4 were directionally consistent. We tested the association between CH functional classes and the eight CH-predisposition genes that were replicated in our validation cohorts. We observed that _CHEK2_ germline variation was positively associated with CH in genes involved in DNA methylation (odds ratio (OR) = 1.71, 95% confidence interval (CI) = 1.51–1.92, _P_ < 0.001) but negatively associated with DDR genes (OR = 0.32, 95% CI = 0.13–0.81, _P_ = 0.016). We also observed heterogeneity in the strength of the associations between germline variants and specific acquired chromosomal alterations (Fig. [2c](https://www.nature.com/articles/s41588-025-02250-x#Fig2)). Among the strongest associations that we observed, many were between germline variation in cancer-predisposing genes and acquired CNLOH or deletion in overlapping regions (for example, _ATM_ and 11q CNLOH, _MPL_ and 1p CNLOH, and _NBN_ and 8q CNLOH). Among 24 mCA-germline gene-specific associations with ≥1 co-occurring events, 16 were significant (_P_ < 0.05) and the other 8 were directionally consistent in our validation cohort (Supplementary Table [12](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)).

We hypothesized that heterogeneity between the strength of the association across germline variants and CH in specific genes and/or genetic regions likely reflected differences in CH fitness. In the absence of longitudinal data, we used a recently published method[16](https://www.nature.com/articles/s41588-025-02250-x#ref-CR16 "Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020).") to quantify the fitness of specific CH mutations based on their VAF distribution. We focused on the association between germline _CHEK2_ and _DNMT3A_ CH, on the association with the most co-occurring events. Comparison of the VAF distribution of CH in _DNMT3A_ (overall and R882) between _CHEK2_ PGV carriers and noncarriers suggested a substantial increase in mutation rate and a modest increase in fitness of _DNMT3A_ CH among _CHEK2_ germline carriers (Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)). However, within this framework there are multiple scenarios where an increased fitness effect can mimic an increase in mutation rate, including a relative increase in fitness for smaller VAF clones, which decreases for higher VAF events as clone-specific factors predominate.

Based on our findings of heterogeneity between germline–CH associations by CH gene and genetic regions, we explored whether we might identify additional genes that predisposed to CH in specific genes or genetic regions, but not CH globally. We detected an additional 262 associations with _q_ < 0.05, involving 84 genes (Supplementary Table [13](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Out of 55 associations with ≥1 co-occurring events, 30 were significantly associated (_P_ < 0.05) and an additional 24 were directionally consistent in our validation cohorts. The 30 germline–CH associations that we replicated in the validation cohort consisted of 23 genes (_ATR_, _BUB1B_, _CBL_, _DOCK8_, _ERCC1_, _ERCC2_, _ERCC3_, _ERCC4_, _ERCC6L2_, _FANCI_, _FH_, _KIT_, _LIG4_, _LZTR1_, _MRE11_, _MUTYH_, _NTHL1_, _PRDM9_, _RAD51D_, _RTEL1_, _SH2B3_, _SPRED1_ and _TGFBR1_). Most genes (_n_ = 20) have not been previously linked to CH except for _FH_, _MRE11_ and _SH2B3_. Out of 23, over half (14 genes) have been previously associated with hematologic cancer predisposition. However, for most (_n_ = 10) disease manifestation has been noted only when present in the biallelic state. In contrast, the associations with CH that we observed here were with heterozygous germline carriers. This included _DOCK8_, a regulator of lymphocyte differentiation, members of Fanconi’s anemia pathway (_FANCI_), the base-excision repair pathway (_ERCC1_, _ERCC2_, _ERCC3_ and _ERCC5_), spindle checkpoint regulation (_BUB1B_), double-strand break repair (_MRE11_ and _LIG4_) and centromere maintenance (_ERRCC6L6_).

There is little knowledge about the mutational profile of hematologic cancers among biallelic mutations in these genes. However, individuals with biallelic mutations in _ERCC6L6_ (ref. [28](https://www.nature.com/articles/s41588-025-02250-x#ref-CR28 "Baccelli, F. et al. ERCC6L2-related disease: a novel entity of bone marrow failure disorder with high risk of clonal evolution. Ann. Hematol. 102, 699–705 (2023).")) and telomere biology disorders[29](https://www.nature.com/articles/s41588-025-02250-x#ref-CR29 "Ferrer, A., Mangaonkar, A. A. & Patnaik, M. M. Clonal hematopoiesis and myeloid neoplasms in the context of telomere biology disorders. Curr. Hematol. Malig. Rep. 17, 61–68 (2022).") are known to be prone to acute myeloid leukemia (AML) driven by mutations in the DDR pathway, including _TP53_ and _PPM1D_. Similarly, we observed a higher frequency of CH in _PPM1D_ (OR = 3.41, 95% CI = 1.57–7.40, _P_ = 0.002) among heterozygote carriers of _ERCC6L6_. Within the context of telomere disorders, CH in the DDR pathway, particularly _PPM1D_, is common and thought to be compensatory, resulting in suppression of apoptosis triggered by telomere dysregulation[30](https://www.nature.com/articles/s41588-025-02250-x#ref-CR30 "Lasho, T. & Patnaik, M. M. Adaptive and maladaptive clonal hematopoiesis in telomere biology disorders. Curr. Hematol. Malig. Rep. 19, 35–44 (2024)."). Although heterozygote carriers of these autosomal-recessive hematologic cancer predisposition genes may not show strong increased risks for cancer, similar cellular stressors may be shared between heterozygote and biallelic carriers, reflected by shared patterns of CH. Taken together, our findings suggest that germline predisposition to CH is largely characterized by gene-specific associations likely reflecting somatic–germline interactions influencing HSPC fitness.

### Germline predisposition to hematologic malignancies

As CH is a precursor of hematologic malignancy, we hypothesized that CH-predisposition genes would also increase the risk of hematologic malignancy. During up to 15 years of follow-up, 5,248 UKBB participants developed hematologic malignancy, including 1,303 cases of myeloid malignancies and 3,963 with lymphoid malignancies. We tested for an association between germline carriers and risk of hematologic malignancy for the 98 CH-predisposition genes that we identified in the UKBB, including 14 genes associated with CH overall and 94 genes associated only with CH in specific genes or genetic regions. Overall, we found 16 germline genes associated with an increased risk of hematologic malignancy, most of which (_n_ = 8) were associated with CH overall (Fig. [3](https://www.nature.com/articles/s41588-025-02250-x#Fig3) and Supplementary Table [14](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Among these 16 genes, most are known hematologic malignancy predisposition genes including _CBL_[31](https://www.nature.com/articles/s41588-025-02250-x#ref-CR31 "Saito, Y. et al. Casitas B-cell lymphoma mutation in childhood T-cell acute lymphoblastic leukemia. Leuk. Res. 36, 1009–1015 (2012).") and _POT1_ (refs. [32](https://www.nature.com/articles/s41588-025-02250-x#ref-CR32 "DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. N. Engl. J. Med. 388, 2422–2433 (2023)."),[33](https://www.nature.com/articles/s41588-025-02250-x#ref-CR33 "Lim, T. L. et al. Germline POT1 variants can predispose to myeloid and lymphoid neoplasms. Leukemia 36, 283–287 (2022).")) with lymphoid malignancies, and _ETV6_ (ref. [34](https://www.nature.com/articles/s41588-025-02250-x#ref-CR34 "Di Paola, J. & Porter, C. C. ETV6-related thrombocytopenia and leukemia predisposition. Blood 134, 663–667 (2019).")), _RUNX1_ (ref. [20](https://www.nature.com/articles/s41588-025-02250-x#ref-CR20 "Brown, A. L. et al. RUNX1-mutated families show phenotype heterogeneity and a somatic mutation profile unique to germline predisposed AML. Blood Adv. 4, 1131–1144 (2020).")), _TP53_ (ref. [35](https://www.nature.com/articles/s41588-025-02250-x#ref-CR35 "Wong, T. N. et al. Role of TP53 mutations in the origin and evolution of therapy-related acute myeloid leukaemia. Nature 518, 552–555 (2015).")), _DDX41_ (ref. [36](https://www.nature.com/articles/s41588-025-02250-x#ref-CR36 "Li, P. et al. The genetic landscape of germline DDX41 variants predisposing to myeloid neoplasms. Blood 140, 716–755 (2022).")), _SOS1_ (ref. [37](https://www.nature.com/articles/s41588-025-02250-x#ref-CR37 "Molteni, E. et al. Prevalence and clinical expression of germ line predisposition to myeloid neoplasms in adults with marrow hypocellularity. Blood 142, 643–657 (2023).")), _CBL_[38](https://www.nature.com/articles/s41588-025-02250-x#ref-CR38 "Niemeyer, C. M. et al. Germline CBL mutations cause developmental abnormalities and predispose to juvenile myelomonocytic leukemia. Nat. Genet. 42, 794–800 (2010)."), _PTPN11_ (ref. [39](https://www.nature.com/articles/s41588-025-02250-x#ref-CR39 "Alfayez, M. et al. The clinical impact of PTPN11 mutations in adults with acute myeloid leukemia. Leukemia 35, 691–700 (2021).")) and _RTEL1_ (ref. [40](https://www.nature.com/articles/s41588-025-02250-x#ref-CR40 "Marsh, J. C. W. et al. Heterozygous RTEL1 variants in bone marrow failure and myeloid neoplasms. Blood Adv. 2, 36–48 (2018).")) with myeloid malignancies. Overall, among UKBB participants, 5% (_n_ = 20,943) were germline carriers for a hematologic cancer predisposition gene with a dominant inheritance mode. It is interesting that we identified several genes in which heterozygous germline variants have not previously linked to hematologic malignancy. This included _XRCC2_ (hazard ratio (HR) = 4.2, 95% CI = 1.4–13.2, _P_ = 0.012) and _SLX4_ (HR = 2.8, 95% CI = 1.2–6.7, _P_ = 0.022) associated with increased risk of myeloid malignancies and _MLH1_ (HR = 2.1, 95% CI = 1.05–4.2, _P_ = 0.037) and _NTHL1_ (HR = 1.5, 95% CI = 1.1–2.2, _P_ = 0.023) associated with increased risk of lymphoid malignancies. These four genes have been linked to cancer, including hematologic malignancy when present as homozygous or compound heterozygous state previously[41](https://www.nature.com/articles/s41588-025-02250-x#ref-CR41 "Park, J.-Y. et al. Complementation of hypersensitivity to DNA interstrand crosslinking agents demonstrates that XRCC2 is a Fanconi anaemia gene. J. Med. Genet. 53, 672–680 (2016)."),[42](https://www.nature.com/articles/s41588-025-02250-x#ref-CR42 "Kim, Y. et al. Mutations of the SLX4 gene in Fanconi anemia. Nat. Genet. 43, 142–146 (2011)."),[43](https://www.nature.com/articles/s41588-025-02250-x#ref-CR43 "Porter, C. C. Germ line mutations associated with leukemias. Hematology 2016, 302–308 (2016)."),[44](https://www.nature.com/articles/s41588-025-02250-x#ref-CR44 "Grolleman, J. E. et al. Mutational signature analysis reveals NTHL1 deficiency to cause a multi-tumor phenotype. Cancer Cell 35, 256–266.e5 (2019)."), but not in heterozygous PGV carriers. We also identified increased risk of myeloid malignancies among people with heterozygous PGVs in _POLE_ (HR = 2.5, 95% CI = 1.2–5.3, _P_ = 0.02) where biallelic mutations result in immunodeficiency[45](https://www.nature.com/articles/s41588-025-02250-x#ref-CR45 "Palles, C. et al. Germline mutations affecting the proofreading domains of POLE and POLD1 predispose to colorectal adenomas and carcinomas. Nat. Genet. 45, 136–144 (2013).").

**Fig. 3: Association between CH-predisposition genes and hematologic malignancy in the UKBB.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02250-x/figures/3)

Germline CH-predisposition genes are shown that were also associated with the risk of hematologic malignancy (HM). Data are presented as HR ± 95% CI for myeloid (_n_ = 1,303) or lymphoid (_n_ = 3,963) malignancies that were calculated using Cox’s regression adjusted for age at blood draw, the first three genetic PCs and exome sequencing batch. \*_P_ < 0.05, \*\*_P_ < 0.01, \*\*\*_P_ < 0.001. The two-sided _P_ value has no correction for multiple testing. See Supplementary Table [14](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for exact _P_ values.

We further refined the spectrum of hematologic malignancy associated with _CHEK2_ and _ATM_. Although PGVs in _ATM_ have been previously linked to lymphoid malignancies[46](https://www.nature.com/articles/s41588-025-02250-x#ref-CR46 "Lampson, B. L. et al. Rare germline ATM variants influence the development of chronic lymphocytic leukemia. J. Clin. Oncol. 41, 1116–1128 (2023)."), we also observed an association with the development of myeloid malignancies (HR = 2.0, 95% CI = 1.1–3.5, _P_ = 0.018). Several small (<400 individuals) studies have linked _CHEK2_ PGVs to myeloid malignancies[47](https://www.nature.com/articles/s41588-025-02250-x#ref-CR47 "Boonen, R. A. C. M., Vreeswijk, M. P. G. & van Attikum, H. CHEK2 variants: linking functional impact to cancer risk. Trends Cancer 8, 759–770 (2022)."),[48](https://www.nature.com/articles/s41588-025-02250-x#ref-CR48 "Freiman, L. et al. Germline CHEK2 mutations in patients with myeloid neoplasms. Leukemia 38, 908–911 (2024)."). We show robust evidence in a large population of 3,978 PGV carriers that _CHEK2_ is linked to risk of both lymphoid (HR = 2.1, 95% CI = 1.7–2.6, _P_ = 1.9 × 10−10) and myeloid neoplasm (HR = 3.3, 95% CI = 2.4–4.6, _P_ = 1.1 × 10−13). Both _ATM_ and _CHEK2_ were associated with a wide range of hematologic cancer subtypes, including both primary and secondary (occurring after a solid tumor diagnosis) hematologic cancers (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02250-x#Fig8)). Mutation-specific effects for cancer predisposition genes on solid tumor risk have been observed but are not well characterized for hematologic cancers. For example, loss of function (LOF) mutations in _CHEK2_ may confer higher risks of cancer compared with missense mutations[47](https://www.nature.com/articles/s41588-025-02250-x#ref-CR47 "Boonen, R. A. C. M., Vreeswijk, M. P. G. & van Attikum, H. CHEK2 variants: linking functional impact to cancer risk. Trends Cancer 8, 759–770 (2022)."). We compared the strength of the association between CH and germline LOF versus missense variants in _CHEK2_ and _ATM_ (Extended Data Fig. [4a](https://www.nature.com/articles/s41588-025-02250-x#Fig9) and Supplementary Table [15](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). LOF germline variants in _CHEK2_ were associated with higher risk of CH (driven largely by the del1100C European founder mutation) compared with missense mutations. In contrast, the frequency of CH was similar for missense and LOF germline _ATM_ variants. The risk of hematologic cancer appeared similar for misense and LOF variants in both _ATM_ and _CHEK2_, but would require larger numbers of hematologic cancers for a more refined estimation of potential heterogeneity (Extended Data Fig. [4b](https://www.nature.com/articles/s41588-025-02250-x#Fig9) and Supplementary Table [16](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)).

### Germline–CH interactions influence hematologic malignancy risk

Given that germline variation predisposes to both CH and hematologic malignancy, we sought to characterize the interaction between PGVs and CH on hematologic malignancy risk. First, among germline carriers, we compared the risk of hematologic malignancy between individuals with and those without CH. We observed a multiplicative interaction between germline predisposition and CH on hematologic malignancy risk (_P_ = 0.014). CH carriers with pathogenic germline variants have a higher risk of developing hematologic malignancy (HR = 1.3, 95% CI = 1.1–1.5, _P_ = 2.4 × 10−5) compared with CH carriers without PGVs. We next investigated this across individual hematologic malignancy predisposition genes. In the presence of CH, germline carriers generally showed marked increased risks of hematologic malignancy, with significantly lower risks in the absence of CH (Fig. [4a](https://www.nature.com/articles/s41588-025-02250-x#Fig4) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). This pattern was observed for both myeloid and lymphoid malignancies. An exception to this pattern was _DDX41_, which showed similar risks in the presence and absence of CH. Progression to myeloid neoplasms among _DDX41_ PGV carriers is characterized by the acquisition of subclonal, second somatic mutations in _DDX41_. Perhaps as a result of the low depth of the sequencing data, we did not observe secondary somatic _DDX41_ events among _DDX41_ PGV carriers. Taken together, these data suggest that CH is a strong risk stratification tool for hematologic malignancy risk among germline carriers.

**Fig. 4: Germline–CH interactions stratify the risk of hematologic malignancy.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02250-x/figures/4)

**a**, HRs (center dot) and 95% CIs for myeloid or lymphoid malignancy among people with pathogenic variants in germline genes that predispose to both CH and hematologic malignancy (HM) stratified by the presence of any CH (including any CH-heme and mCA-auto). Differences between the risk of hematologic cancer across CH-positive and CH-negative germline carriers were calculated using Firth’s bias-reduced logistic regression limited to germline variant carriers. \*_P_ < 0.1, \*\*_P_ < 0.01, \*\*\*_P_ < 0.001. The two-sided _P_ value has no correction for multiple testing (see Supplementary Table [17](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for exact _P_ values). **b**, Predicted distribution of 25-year absolute risk of myeloid malignancies among UKBB individuals aged 50–74 years with _CHEK2_ (_n_ = 3,012), _ATM_ (_n_ = 1,592) or no pathogenic germline variants (_n_ = 269,050). Analyses in both **a** and **b** were performed using Cox’s regression adjusted for age at blood draw, first three genetic PCs and exome sequencing batch. **c**, Comparison of distribution of 25-year absolute risk of myeloid malignancy among people at the top percentiles of risk across people with _CHEK2_ (_n_ = 30)_, ATM_ (_n_ = 14) or no germline variant (_n_ = 2,690). The center line represents the median, the box limits the upper and lower quartiles and the whiskers 1.5× the interquartile range (IQR).

To evaluate the extent to which CH and germline profiles, together with clinical phenotypes, can identify individuals at a clinically meaningful risk of myeloid malignancy, we estimated the 25-year absolute risk of myeloid malignancy among individuals in the UKBB. First, we compared the number of individuals needed to screen to identify at least one individual at a moderate (≥5%) absolute risk of myeloid malignancy using CH alone or CH plus germline mutation status. Using CH plus clinical factors, we estimated that it would require 432 individuals whereas incorporation of germline mutation profile would reduce this slightly by ~10% to 392 individuals. Next, we estimated the absolute risk among those with different germline backgrounds. A substantially higher fraction of germline _CHEK2_ carriers (2%) and _ATM_ carriers (1%) showed a 25-year absolute risk >5% compared with noncarriers (0.2%; Fig. [4b](https://www.nature.com/articles/s41588-025-02250-x#Fig4)). To identify at least one individual at 5% absolute risk of myeloid malignancy, it would require CH screening of 454 noncarriers but only 48 _CHEK2_ carriers and 76 _ATM_ carriers. For people at the highest risk (top 0.5%), the median 25-year absolute risk of myeloid neoplasm was 46% for _CHEK2_ and 30% for _ATM_ carriers compared with only 4% among noncarriers (Fig. [4c](https://www.nature.com/articles/s41588-025-02250-x#Fig4)). Thus, screening for CH among germline carriers can more efficiently identify individuals at higher absolute risk for myeloid malignancy compared with a population-level screening agnostic of inherited predisposition.

If heterogeneity in the strength of associations between germline predisposition genes and specific CH mutations reflects differential fitness effects, we reasoned that this should influence the gene-specific risk of CH progression to hematologic malignancy. To test this hypothesis, we classified CH among germline carriers into two categories: CH genes that showed a moderate or strong association with PGVs in a specific gene (referred to as germline-selected CH) and CH in genes showing weak or no association with the germline gene (referred to as nonselected CH) (Fig. [5a](https://www.nature.com/articles/s41588-025-02250-x#Fig5)). Among germline carriers with CH, those with germline-selected CH showed a markedly higher risk of both myeloid and lymphoid malignancy compared with those with nonselected CH (Fig. [5b](https://www.nature.com/articles/s41588-025-02250-x#Fig5) and Supplementary Table [18](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Overall, the risks of progression were 2.7-fold and 13.1-fold higher, respectively, for germline-selected CH compared with germline-nonselected CH. We further investigated whether this pattern was consistent across germline carriers or limited to specific germline genes. Among germline carriers with at least two hematologic cancer cases in both germline-selected and nonselected CH, we observed consistent effects across genes (Fig. [5c,d](https://www.nature.com/articles/s41588-025-02250-x#Fig5) and Supplementary Table [18](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Finally, we sought to understand whether the risk of progression for specific CH genes varied by germline genetic backdrop. We tested for differences in the risk of CH progression to myeloid malignancy for _DNMT3A_, the most commonly mutated CH gene, among _CHEK2_ germline carriers, the most commonly mutated germline gene. In accordance with our finding of a higher fitness advantage of _DNMT3A_ CH among _CHEK2_ germline carriers, the risk of _DNMT3A_ progression to myeloid malignancies was higher among _CHEK2_ germline carriers compared with noncarriers (HR = 2.8, 95% CI = 1.01–7.5, _P_ = 0.047; Fig. [5e](https://www.nature.com/articles/s41588-025-02250-x#Fig5)).

**Fig. 5: Risk of CH progression to hematologic cancer varies by germline background.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02250-x/figures/5)

**a**, Graphic illustration describing our analysis studying the impact of germline-selected CH on risk of hematologic cancer. We defined germline-selected CH in a given germline carrier as the presence of a CH mutation showing evidence of enrichment in that specific germline gene. **b**–**d**, Risks for myeloid or lymphoid malignancy among individuals with germline-selected CH (red) compared with those with germline-nonselected CH (blue) calculated using Cox’s regression adjusted for age at blood draw, the first three genetic PCs and exome sequencing batch. Data are presented as HRs ± 95% CIs. **b**, HRs among all germline carriers. **c**,**d**, HRs for myeloid (**c**) and lymphoid (**d**) malignancies among specific germline gene carriers. The number of samples is as follows: germline carriers (_n_ = 73,781), _CHEK2_ (_n_ = 3,337), _ATM_ (_n_ = 1,736) and _NTHL1_ (_n_ = 1,608). \*_P_ < 0.05, \*\*_P_ < 0.01, \*\*\*_P_ < 0.001 (see Supplementary Table [18](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for exact _P_ values). **e**, Kaplan–Meier plot for 10-year, myeloid malignancy-free survival probability among people with _DNMT3A_ CH mutation stratified by _CHEK2_ germline carrier status. The _P_ value was derived from Cox’s regression limited to _DNMT3A_ CH carriers, testing for a difference in the HR for developing myeloid malignancies between _CHEK2_ germline carriers and noncarriers. All _P_ values are two-sided with no correction for multiple testing. Icons in **a** created with [BioRender.com](http://biorender.com/).

Discussion
----------

Here we performed a systematic assessment of the impact of PGVs on CH. We identified several genes not previously linked to CH predisposition across diverse racial groups (_NBN_, _PTPN11_, _ATR_, _BUB1B_, _CBL_, _DOCK8_, _ERCC1_, _ERCC2_, _ERCC3_, _ERCC4_, _ERCC6L2_, _FANCI_, _KIT, LIG4_, _LZTR1_, _MUTYH_, _NTHL1_, _PRDM9_, _RAD51D_, _RTEL1_, _SPRED1_ and _TGFBR1_). Most of these conferred an increased risk of specific somatic events rather than CH overall, thus highlighting that germline predisposition to CH varies by somatic alteration. In addition, we identified five new candidate hematologic cancer predisposition genes: _XRCC2_, _SLX4_, _MLH1_, _NTHL1_ and _POLE_. Given that our replication cohorts had either short or no follow-up for hematologic cancer development, these genes require validation in future work.

We showed that germline genetic variation influences both the mutational landscape of CH and the risk of progression to hematologic malignancy. Germline genetic variation is known to influence both mutagenesis and clonal fitness[1](https://www.nature.com/articles/s41588-025-02250-x#ref-CR1 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)."),[2](https://www.nature.com/articles/s41588-025-02250-x#ref-CR2 "Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. Nature 534, 47–54 (2016)."),[3](https://www.nature.com/articles/s41588-025-02250-x#ref-CR3 "Campbell, P. J. et al. Pan-cancer analysis of whole genomes. Nature 578, 82–93 (2020)."),[4](https://www.nature.com/articles/s41588-025-02250-x#ref-CR4 "Middlebrooks, C. D. et al. Association of germline variants in the APOBEC3 region with cancer risk and enrichment with APOBEC-signature mutations in tumors. Nat. Genet. 48, 1330–1338 (2016)."),[5](https://www.nature.com/articles/s41588-025-02250-x#ref-CR5 "Nik-Zainal, S. et al. Association of a germline copy number polymorphism of APOBEC3A and APOBEC3B with burden of putative APOBEC-dependent mutations in breast cancer. Nat. Genet. 46, 487–491 (2014)."),[6](https://www.nature.com/articles/s41588-025-02250-x#ref-CR6 "Yu, C.-C. et al. Mutant allele specific imbalance in oncogenes with copy number alterations: Occurrence, mechanisms, and potential clinical implications. Cancer Lett. 384, 86–93 (2017)."),[7](https://www.nature.com/articles/s41588-025-02250-x#ref-CR7 "Chatrath, A., Ratan, A. & Dutta, A. Germline variants that affect tumor progression. Trends Genet. 37, 433–443 (2021)."). Although both processes may explain our observed associations between germline variation and CH, our results suggested a larger role for selection. First, we observed that germline variation was associated with specific CH drivers, yet the aging-related mutational signature of CH mutation was predominant across carriers and noncarriers. Second, CH drivers that were enriched among germline carriers showed a markedly higher risk of progression to hematologic malignancy compared with CH drivers observed at a similar frequency among germline carriers and noncarriers. Taken together, these observations suggested that CH arises with normal aging with germline genetic backdrop influencing CH fitness and cancer risk. Among children and young adults with germline hematologic predisposition, CH has been shown to compensate for germline defects, leading to restoration of normal hematopoiesis in some cases but malignant transformation in others[18](https://www.nature.com/articles/s41588-025-02250-x#ref-CR18 "Kennedy, A. L. et al. Distinct genetic pathways define pre-malignant versus compensatory clonal hematopoiesis in Shwachman–Diamond syndrome. Nat. Commun. 12, 1334 (2021)."). Longitudinal studies of germline carriers will be important to further elucidate CH dynamics and the secondary events demarcating progression of CH. Previous studies of common germline genetic drivers of CH have also identified gene-specific CH-predisposition loci[24](https://www.nature.com/articles/s41588-025-02250-x#ref-CR24 "Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."),[49](https://www.nature.com/articles/s41588-025-02250-x#ref-CR49 "Bick, A. G. et al. Inherited causes of clonal haematopoiesis in 97,691 whole genomes. Nature 586, 763–768 (2020)."),[50](https://www.nature.com/articles/s41588-025-02250-x#ref-CR50 "Kessler, M. D. et al. Common and rare variant associations with clonal haematopoiesis phenotypes. Nature 612, 301–309 (2022)."),[51](https://www.nature.com/articles/s41588-025-02250-x#ref-CR51 "Liu, J., Osman, A. E. G., Bolton, K. & Godley, L. A. Germline predisposition to clonal hematopoiesis. Leuk. Res. 132, 107344 (2023)."). Most notably, a common inherited polymorphism in the _TCL1A_ promoter has been associated with an increased risk of _DNMT3A_ CH, but a decreased risk for other genes including _TET2_ (ref. [52](https://www.nature.com/articles/s41588-025-02250-x#ref-CR52 "Weinstock, J. S. et al. Aberrant activation of TCL1A promotes stem cell expansion in clonal haematopoiesis. Nature 616, 755–763 (2023).")). Subsequent functional characterization suggested that this was mediated by differential impact of _TCL1A_ activation on the CH expansion rate. Our results suggested that germline–somatic interactions could occur more broadly across CH-predisposition genes than previously shown. Further elucidation of germline predisposition to CH will enable characterization of the extent to which germline genetic variation dictates gene-specific CH rates.

Recent studies characterizing genomic[53](https://www.nature.com/articles/s41588-025-02250-x#ref-CR53 "Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. Nature 559, 400–404 (2018)."),[54](https://www.nature.com/articles/s41588-025-02250-x#ref-CR54 "Desai, P. et al. Somatic mutations precede acute myeloid leukemia years before diagnosis. Nat. Med. 24, 1015–1023 (2018)."),[55](https://www.nature.com/articles/s41588-025-02250-x#ref-CR55 "Bolton, K. L. et al. Cancer therapy shapes the fitness landscape of clonal hematopoiesis. Nat. Genet. 52, 1219–1226 (2020)."), proteomic[56](https://www.nature.com/articles/s41588-025-02250-x#ref-CR56 "Tran, D. et al. Plasma proteomic signature predicts myeloid neoplasm risk. Clin. Cancer Res. 
https://doi.org/10.1158/1078-0432.CCR-23-3468
(2024).") and clinical predictors[53](https://www.nature.com/articles/s41588-025-02250-x#ref-CR53 "Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. Nature 559, 400–404 (2018).") of hematologic cancer have enabled the development of disease-specific prognostic tools[57](https://www.nature.com/articles/s41588-025-02250-x#ref-CR57 "Weeks, L. D. et al. Prediction of risk for myeloid malignancy in clonal hematopoiesis. NEJM Evid. 
https://doi.org/10.1056/EVIDoa2200310
(2023)."),[58](https://www.nature.com/articles/s41588-025-02250-x#ref-CR58 "Gu, M. et al. Multiparameter prediction of myeloid neoplasia risk. Nat. Genet. 55, 1523–1530 (2023)."). Predictive models to identify high-risk individuals are critical for both clinical management and guidance of interventional trials. Our findings suggest that joint characterization of germline genetic variation and CH represents an opportunity to further improve CH risk stratification. Overall, 5% of individuals in our study were found to harbor a pathogenic germline variant in a hematologic cancer predisposition gene, with 22% of germline carriers also harboring CH, the latter certainly being underestimated as a result of the low sequencing depth of our cohorts. Within individuals with germline predisposition to hematologic cancer, CH identified a subgroup of individuals at high absolute risk of progression. As there were no established interventions for CH, routine screening of individuals with moderate risk of hematologic cancer, for example, of _CHEK2_ or _ATM_ carriers, would not be currently warranted. However, our findings highlighted the relevance of germline carriers as a target population for intervention and early interception studies.

Inherited genetic backdrop may influence somatic evolution in normal tissues beyond the hematopoietic system. With the increasing scope of sequencing efforts, broader characterization of germline–somatic interactions across tissue types will be possible. Here we provided a first approximation of the broad spectrum of germline–somatic interactions of biological and clinical relevance to cancer development.

Methods
-------

### Study populations

We included 731,835 individuals with normal blood DNA sequencing data available from the UKBB (_n_ = 428,530), All of Us (_n_ = 192,003), MGBB (_n_ = 49,941), CCDG (_n_ = 37,184), TCGA (_n_ = 7,161) and MSK-IMPACT (_n_ = 17,016). The UKBB study was approved by the North West Multi-centre Research Ethics Committee and other studies were approved by their respective institutional review boards (IRBs) (see Supplementary Table [2](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for protocol numbers). Written informed consent was obtained from all participants. We excluded individuals who were diagnosed with hematologic malignancy before or within 90 d of blood draw. We also randomly excluded one individual from each related pair (kinship score >0.1875; Supplementary Note [1](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)). Descriptions of the individual cohort characteristics and sequencing methods are briefly described below (see Supplementary Table [2](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) and Supplementary Notes [1](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)–[3](https://www.nature.com/articles/s41588-025-02250-x#MOESM1) for additional information).

#### UKBB

The UKBB[59](https://www.nature.com/articles/s41588-025-02250-x#ref-CR59 "Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015).") is a longitudinal population-based cohort of 502,368 middle-aged individuals enrolled between 2006 and 2010 in the United Kingdom. At the initial visit, participants aged 38–74 years answered questions about sociodemographic, lifestyle and health-related factors and provided biological samples including blood, urine and saliva for biological measurements and genotyping. Follow-up data on health outcomes were available through linkage to electronic health records (EHRs), the cancer registry and the health registry. We included 428,530 individuals with SNP array and WES data available. WES of blood DNA at an average depth of 55× was performed on an Illumina NovaSeq 6000 platform using a 75-bp paired-end protocol by the Regeneron Genetics Center[60](https://www.nature.com/articles/s41588-025-02250-x#ref-CR60 "Van Hout, C. V. et al. Exome sequencing and characterization of 49,960 individuals in the UK Biobank. Nature 586, 749–756 (2020)."). The first 50,000 participants were genotyped using the Affymetrix UK BiLEVE Axiom array and the remaining participants with the UKBB Axiom array.

#### All of Us

All of Us[61](https://www.nature.com/articles/s41588-025-02250-x#ref-CR61 "Bick, A. G. et al. Genomic data in the All of Us research program. Nature 627, 340–346 (2024).") is an ongoing longitudinal cohort study initiated in 2018 that aims to enroll at least 1 million individuals, including those from under-representative communities or racial or ethnic minorities across the United States of America. Briefly, participants complete baseline surveys, physical measurements, biospecimen donation and consent for data usage at enrollment. As of 15 February 2023, longitudinal health data were available for >287,000 participants through linkage to EHRs harmonized using the Observational Medical Outcomes Partnership Common Data Model. PCR-free DNA libraries were constructed using an Illumina Kapa HyperPrep kit and sequenced on the Illumina NovaSeq 6000 platform using 150-bp paired-end protocol to a mean depth of 30×. Genotyping was performed using Illumina Infinium Global Diversity Array.

#### MGBB

The MGBB[62](https://www.nature.com/articles/s41588-025-02250-x#ref-CR62 "Uddin, M. M. et al. Germline genomic and phenomic landscape of clonal hematopoiesis in 323,112 individuals. Preprint at MedRxiv 
https://doi.org/10.1101/2022.07.29.22278015
(2022).") is an ongoing hospital-based biobank enrolling patients at Massachusetts General Hospital, Brigham and Women’s Hospital and affiliated hospitals. At enrollment, participants provide consent for sharing EHR data, blood samples and answers to surveys covering lifestyle, environmental and family history information. The blood DNA exome was captured using a TWIST Human Core Exome kit and sequencing was performed using Illumina NovaSeq 6000 instruments to a mean coverage of 71×, using a 150-bp paired-end protocol. The Illumina Global Screening Array was used for genotyping.

#### CCDG

The CCDG is a collaborative genomic sequencing program aiming to identify genetic causes of common diseases. We included 37,184 multi-ancestry cases and controls for cardiovascular disease from 26 cohorts (except for PAGE and T1DGC) sequenced at the McDonnell Genome Institute at Washington University. Whole-genome sequencing (WGS) of the blood was performed using the Illumina platform (HiSeq X10, HiSeq 2000, HiSeq 2500, MiSeq 2 or NovaSeq 6000) to a targeted depth of 20–30× using a 150-bp paired-end protocol.

#### TCGA

TCGA[63](https://www.nature.com/articles/s41588-025-02250-x#ref-CR63 "Huang, K. et al. Pathogenic germline variants in 10,389 adult cancers. Cell 173, 355–370.e14 (2018).") collected baseline clinical data and tumor or matched normal samples spanning 33 cancer types from 11,428 patients with treatment-naive cancer to discover molecular changes at the DNA, RNA, protein and epigenetic levels in human tumors. WES was performed on an Illumina platform to an average depth of 100×. Genotyping was performed using Affymetrix Genome-wide SNP 6.0 array. WES sequencing reads (removing samples from whole-genome amplification) and genotyping array data were downloaded from the National Cancer Institute (NCI) Genomic Data Common (GDC) portal.

#### MSK-IMPACT

The present study included 17,016 patients with non-hematologic cancers at MSK Cancer Center (MSKCC), who underwent paired tumor and blood sequencing using the MSK-IMPACT panel as part of an institutional prospective tumor-sequencing protocol and consented to germline genetic testing before 1 July 2020 (ref. [64](https://www.nature.com/articles/s41588-025-02250-x#ref-CR64 "Cheng, D. T. et al. Comprehensive detection of germline variants by MSK-IMPACT, a clinical diagnostic platform for solid tumor molecular oncology and concurrent cancer predisposition testing. BMC Med. Genet. 10, 33 (2017).")) (and whose CH calls were available as described[65](https://www.nature.com/articles/s41588-025-02250-x#ref-CR65 "Bolton, K. L. et al. Clonal hematopoiesis is associated with risk of severe Covid-19. Nat. Commun. 12, 5975 (2021).")). MSK-IMPACT is a next-generation sequencing assay that uses hybridization capture to target all protein-coding exons from the canonical transcript of 468 cancer-associated genes[66](https://www.nature.com/articles/s41588-025-02250-x#ref-CR66 "Cheng, D. T. et al. Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT): a hybridization capture-based next-generation sequencing clinical assay for solid tumor molecular oncology. J. Mol. Diagn. 17, 251–264 (2015)."). Sequencing was performed on an Illumina HiSeq 2500 with 100-bp paired-end reads, achieving an average depth of 497×.

### Germline variant calling

We profiled exome or genome sequencing data for pathogenic germline variants in 236 known cancer predisposition genes (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)). Within the UKBB, germline single nucleotide variants (SNVs) and small insertions and deletions (indels) were called using DeepVariant followed by joint genotyping using GLnexus. DRAGEN pipeline was used for All of Us[61](https://www.nature.com/articles/s41588-025-02250-x#ref-CR61 "Bick, A. G. et al. Genomic data in the All of Us research program. Nature 627, 340–346 (2024).") and GATK Hyplotypecaller for CCDG, MGBB[62](https://www.nature.com/articles/s41588-025-02250-x#ref-CR62 "Uddin, M. M. et al. Germline genomic and phenomic landscape of clonal hematopoiesis in 323,112 individuals. Preprint at MedRxiv 
https://doi.org/10.1101/2022.07.29.22278015
(2022)."), TCGA[63](https://www.nature.com/articles/s41588-025-02250-x#ref-CR63 "Huang, K. et al. Pathogenic germline variants in 10,389 adult cancers. Cell 173, 355–370.e14 (2018).") and MSK-IMPACT[67](https://www.nature.com/articles/s41588-025-02250-x#ref-CR67 "Franch-Expósito, S. et al. Associations between cancer predisposition mutations and clonal hematopoiesis in patients with solid tumors. JCO Precis. Oncol. 
https://doi.org/10.1200/PO.23.00070
(2023).") (see Supplementary Note [2](https://www.nature.com/articles/s41588-025-02250-x#MOESM1) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for detailed version and workflow). We removed variants and samples with a high missing genotyping rate (≥10%). We further removed variants that: (1) violated the Hardy–Weinberg equilibrium (_P_ < 10−15), (2) had low coverage (depth of coverage (DP) < 7 for SNV or DP < 10 for indel) or (3) had a low VAF (<0.2)[68](https://www.nature.com/articles/s41588-025-02250-x#ref-CR68 "Szustakowski, J. D. et al. Advancing human genetics research and drug discovery through exome sequencing of the UK Biobank. Nat. Genet. 53, 942–948 (2021)."). After quality control, we further restricted variants to those with possible deleterious consequences including only: (1) protein-coding variants, (2) rare variants with population allele frequency <0.005 in the UKBB or gnomAD, (3) variants noted as pathogenic or likely pathogenic in ClinVar, (4) LOF variants in tumor-suppressor genes and (5) missense variants predicted to be pathogenic computationally (REVEL score >0.5 or CADD score >20). Retained variants were then classified as per the ACMG guidelines[22](https://www.nature.com/articles/s41588-025-02250-x#ref-CR22 "Richards, S. et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet. Med. 17, 405–424 (2015)."). In all downstream analyses we included pathogenic or likely pathogenic variants as per ACMG and high-risk missense variants of uncertain significance (see Supplementary Note [2](https://www.nature.com/articles/s41588-025-02250-x#MOESM1) for detailed scoring scheme).

### CH variant calling

Variant calling of WES or WGS data from UKBB, TCGA and CCDG was performed using Mutect2 v.4.2.1.0 (ref. [69](https://www.nature.com/articles/s41588-025-02250-x#ref-CR69 "Cibulskis, K. et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. Nat. Biotechnol. 31, 213–219 (2013).")) and VarDictJava v.1.6.0 (ref. [70](https://www.nature.com/articles/s41588-025-02250-x#ref-CR70 "Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. Nucleic Acids Res. 44, e108 (2016).")). We included coding variants in 184 genes with known relevance to myeloid and/or lymphoid malignancy (CH-heme) (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)) and 755 genes with known relevance to solid tumors (CH-solid)[56](https://www.nature.com/articles/s41588-025-02250-x#ref-CR56 "Tran, D. et al. Plasma proteomic signature predicts myeloid neoplasm risk. Clin. Cancer Res. 
https://doi.org/10.1158/1078-0432.CCR-23-3468
(2024)."). Briefly, variants that were passed by both callers and with VAF ≥ 2% supported by three or more reads, with at least one from both forward and reverse reads, were retained. We then excluded variants that were: (1) present in a panel of sequencing data from the 50 youngest individuals (<41 years) without known CH hotspot mutations from the UKBB, (2) recurrent in >1% of individuals but not reported in previous CH studies, (3) present with overall population frequency >5 × 10−4 in the gnomAD[71](https://www.nature.com/articles/s41588-025-02250-x#ref-CR71 "Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. Nature 625, 92–100 (2024).") or (4) present at a high VAF of ≥35%, unless it was a clear cancer somatic hotspot according to criteria as previously described[55](https://www.nature.com/articles/s41588-025-02250-x#ref-CR55 "Bolton, K. L. et al. Cancer therapy shapes the fitness landscape of clonal hematopoiesis. Nat. Genet. 52, 1219–1226 (2020)."). Further detail about CH variant calling and filtering is described in Supplementary Note [3](https://www.nature.com/articles/s41588-025-02250-x#MOESM1). Variant calling of WGS from All of Us and MGBB was performed using GATK4 Mutect2 and CH mutations in 74 myeloid malignancy driver genes were detected as previously described[72](https://www.nature.com/articles/s41588-025-02250-x#ref-CR72 "Vlasschaert, C. et al. A practical approach to curate clonal hematopoiesis of indeterminate potential in human genetic data sets. Blood 141, 2214–2223 (2023)."),[73](https://www.nature.com/articles/s41588-025-02250-x#ref-CR73 "Bick, A. & Regan, J. CHIP-Detection-Mutect2. Terra.bio 
https://app.terra.bio/#workspaces/terra-outreach/CHIP-Detection-Mutect2
(2016)."). Mosaic chromosomal alterations (mCAs) were detected and filtered from genotyping intensity data using MoChA[24](https://www.nature.com/articles/s41588-025-02250-x#ref-CR24 "Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018).") in all cohorts.

### Statistical analysis

To test for an association between germline carrier status and CH, we used multivariable Firth’s bias-reduced logistic regression (R package logistf, v.1.26.0), adjusted for age at blood draw, the first three genetic PCs and WES batch. The same set of covariates was adjusted for in all logistic, linear and Cox’s regression models. All _P_ values were two sided. Multiple hypothesis testing correction using the FDR method[74](https://www.nature.com/articles/s41588-025-02250-x#ref-CR74 "Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B 57, 289–300 (1995).") was performed to control for inflation of type I error. For the 14 germline genes that were found to be significantly associated with overall CH status (CH-heme and mCA-auto), we reported uncorrected _P_ values for analyses testing their association with specific CH genes or regions because the goal was to define the relative strength of associations with different CH subtypes. For significant associations between germline mutation status and CH, we performed additional sensitivity analyses further adjusting for smoking status, sex and prior history of cancer and obtained similar results (Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)). To ensure that our results were not driven by individuals with undiagnosed hematologic cancer or hematologic cancer predisposition syndromes, including monoclonal B cell lymphocytosis and clonal cytopenia of undetermined significance, we performed sensitivity analyses excluding individuals with abnormal blood counts and those who developed hematologic cancer within 5 years of blood sampling and found similar results (Supplementary Fig. [6](https://www.nature.com/articles/s41588-025-02250-x#MOESM1)). Linear regression (R package stats, v.4.1.1) was used to test for an association between PGV genes and CH characteristics (including maximum VAF and number of mutations) among CH-positive individuals. We attempted to replicate statistically significant associations (FDR-corrected _P_ < 0.05) between germline carrier status and CH using the same approach within All of Us, MGBB, CCDG, TCGA and MSK-IMPACT. We performed a meta-analysis (R package metafor, v.4.4-0) across replication cohorts using a fixed-effects model.

For genes we identified as predisposing to CH, we used Cox’s proportional hazards regression (R package survival v.3.5-7) to test for an association between germline carrier status and risk of myeloid or lymphoid malignancy. The time scale started at cohort enrollment and participants were followed until hematologic malignancy, death from other causes, loss to follow-up or end of cancer registry follow-up (September 2023), whichever came first. To test for an interaction between germline carrier status and CH on the risk of hematologic malignancy development, we used Cox’s regression to estimate the risk of myeloid or lymphoid malignancy among people with pathogenic variants in germline genes that predispose to both CH and hematologic malignancy, stratified by the presence of any CH. Differences in the OR for hematologic malignancy between CH-positive and CH-negative germline carriers were calculated using Firth’s bias-reduced logistic regression limited to germline carriers.

We defined germline-selected CH in a given germline carrier as the presence of a CH mutation showing evidence of enrichment in that specific germline gene (OR ≥ 1.5). We compared the risk for myeloid or lymphoid malignancy among individuals with germline-selected CH with those with germline-nonselected CH using Cox’s regression. The difference in the risk of _DNMT3A_ CH progression to myeloid malignancies between _CHEK2_ germline carriers and noncarriers was derived from Cox’s regression limited to _DNMT3A_ CH carriers.

We modeled the 25-year absolute risk of myeloid malignancy among UKBB individuals aged 50–74 years with the iCARE R package (v.1.20.0)[75](https://www.nature.com/articles/s41588-025-02250-x#ref-CR75 "Maas, P. et al. Breast cancer risk from modifiable and nonmodifiable risk factors among white women in the United States. JAMA Oncol. 2, 1295–1302 (2016).") by combining: (1) the multivariable log(HRs) for clinical risk factors for myeloid malignancies, including number of CH-heme mutations, the maximum VAF of CH-heme (≥2%), number of mCA-auto events, specific CH-heme and mCA-auto events associated with hematologic malignancy, age at blood draw, the first three genetic PCs, WES batch and peripheral blood count indices (neutrophil count, lymphocyte count, monocyte count, eosinophil count, basophil count, platelet count, red cell distribution width, mean corpuscular volume and hemoglobin concentration); (2) the distribution of the above risk factors in UKBB individuals aged 50–74 years with no PGV, _CHEK2_ PGV or _ATM_ PGV; (3) age-specific myeloid malignancy rates for individuals aged 50–74 years as reported in SEER; and (4) competing hazards for mortality in individuals in the United Kingdom aged 50–74 years as reported in Cancer Research UK. We adjusted for the difference in the age-specific incidence and mortality rates for PGV populations by accounting for the relative risk conferred by the PGV.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02250-x#MOESM2) linked to this article.

Data availability
-----------------

Summary statistics for PGV, CH calls and associations are available in Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02250-x#MOESM4)–[20](https://www.nature.com/articles/s41588-025-02250-x#MOESM4). Individual-level CH calls have been returned to UKBB (return no. 6423) and are available to approved researchers. Individual-level phenotypic, sequencing reads and genotyping array data are available as follows: UKBB data are already available to approved researchers at [https://www.ukbiobank.ac.uk/enable-your-research](https://www.ukbiobank.ac.uk/enable-your-research). TCGA data are available at the database of Genotypes and Phenotypes (dbGaP) (accession no. [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178)) and can be downloaded by approved researchers from the NCI GDC portal. The germline SNVs or indels were downloaded by following the instructions at [https://gdc.cancer.gov/about-data/publications/PanCanAtlas-Germline-AWG](https://gdc.cancer.gov/about-data/publications/PanCanAtlas-Germline-AWG). The present study used data from the All of Us Research Program’s Controlled Tier Dataset v.7, available to authorized users on the Researcher Workbench ([https://www.researchallofus.org/data-tools/workbench](https://www.researchallofus.org/data-tools/workbench/)). The CH mutation calls and germline variant calls necessary to replicate the findings in the article from MSK-IMPACT are available via GitHub at [https://github.com/kbolton-lab/germlineCH\_NG](https://github.com/kbolton-lab/germlineCH_NG) (ref. [76](https://www.nature.com/articles/s41588-025-02250-x#ref-CR76 "Liu, J. et al. germlineCH_NG: code for statistical testing and figure/table generation. Zenodo 
https://doi.org/10.5281/zenodo.15200120
(2025).")). Federally funded genotyping and exome sequencing data for 13,500 participants from the MGBB are available in dbGaP (accession no. [phs002018](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002018)). Additional genotyping and exome sequencing data for the remaining ~36,500 participants from MGBB were generated via third-party collaborations, and accessed and secondarily analyzed via Massachusetts General Hospital (MGH) IRB protocol (no. 2018P001236) for the present study. Access to these datasets requires application through the MGBB research portal ([https://biobank.massgeneralbrigham.org](https://biobank.massgeneralbrigham.org/)) and compliance with relevant institutional data use agreements. Processing times vary depending on data type but typically range from 6 weeks to 8 weeks after application approval. External collaborations are feasible with MGBB investigators and appropriate IRB protocols. CCDG data are available at the dbGaP: Washington University Coronary Artery Disease Study (accession no. [phs001227](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001227)); NHLBI TOPMed—Penn Medicine BioBank Early Onset Atrial Fibrillation Study (accession no. [phs001601](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001601)); NHLBI TOPMed—Vanderbilt University BioVU Atrial Fibrillation Genetics Study (accession no. [phs001624](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001624)); NHLBI TOPMed—the BioMe Biobank at Mount Sinai (accession no. [phs001644](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001644)); CCDG—Cardiovascular: Emory Cohort (accession no. [phs001880](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001880)); CCDG—Cardiovascular: eMERGE—Northwestern Cohort (accession no. [phs001913](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001913)); CCDG—Cardiovascular: Cleveland Clinic Cohort (accession no. [phs001871](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs00187)); National Heart, Blood, and Lung Institute (NHLBI) TOPMed—Intermountain INSPIRE Registry (accession no. [phs001545](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001545)); NHLBI TOPMed—Malmö Preventive Project (accession no. [phs001544](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001544)); NHLBI TOPMed—MGH Atrial Fibrillation Study (accession no. [phs001062](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001062)); NHLBI TOPMed—The GENetics in Atrial Fibrillation Study (accession no. [phs001547](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001547)); NHLBI TOPMed—the Johns Hopkins University School of Medicine Atrial Fibrillation Genetics Study (accession no. [phs001598](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001598)); NHLBI TOPMed—the Vanderbilt AF Ablation Registry (accession no. [phs000997](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000997)); CCDG—CVD: VIRGO—Variation in Recovery: Role of Gender on Outcomes of Young Acute Myocardial Infarction (AMI) Patients (accession no. [phs001259](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001259)); CCDG—Cardiovascular: University of Pennsylvania Cohort (accession no. [phs001502](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001502)); CCDG—Cardiovascular: Atherosclerosis Thrombosis and Vascular Biology (accession no. [phs001592](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001592)); CCDG—Cardiovascular: the Bangladesh Risk of Acute Vascular Events Study (accession no. [phs001398](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001398)); and NHLBI TOPMed—Atherosclerosis Risk in Communities (accession no. [phs001211](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001211)).

Code availability
-----------------

The codes for statistical testing and for figure and table generation are publicly available via GitHub at [https://github.com/kbolton-lab/germlineCH\_NG](https://github.com/kbolton-lab/germlineCH_NG) and via Zenodo at [https://doi.org/10.5281/zenodo.15200120](https://doi.org/10.5281/zenodo.15200120) (ref. [76](https://www.nature.com/articles/s41588-025-02250-x#ref-CR76 "Liu, J. et al. germlineCH_NG: code for statistical testing and figure/table generation. Zenodo 
https://doi.org/10.5281/zenodo.15200120
(2025).")).

References
----------

1.  Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. _Nature_ **578**, 94–101 (2020).
    
    [Article](https://doi.org/10.1038%2Fs41586-020-1943-3)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32025018)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054213)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&journal=Nature&doi=10.1038%2Fs41586-020-1943-3&volume=578&pages=94-101&publication_year=2020&author=Alexandrov%2CLB) 
    
2.  Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. _Nature_ **534**, 47–54 (2016).
    
    [Article](https://doi.org/10.1038%2Fnature17676)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XntVCksrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27135926)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4910866)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Landscape%20of%20somatic%20mutations%20in%20560%20breast%20cancer%20whole-genome%20sequences&journal=Nature&doi=10.1038%2Fnature17676&volume=534&pages=47-54&publication_year=2016&author=Nik-Zainal%2CS) 
    
3.  Campbell, P. J. et al. Pan-cancer analysis of whole genomes. _Nature_ **578**, 82–93 (2020).
    
    [Article](https://doi.org/10.1038%2Fs41586-020-1969-6)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-cancer%20analysis%20of%20whole%20genomes&journal=Nature&doi=10.1038%2Fs41586-020-1969-6&volume=578&pages=82-93&publication_year=2020&author=Campbell%2CPJ) 
    
4.  Middlebrooks, C. D. et al. Association of germline variants in the APOBEC3 region with cancer risk and enrichment with APOBEC-signature mutations in tumors. _Nat. Genet._ **48**, 1330–1338 (2016).
    
    [Article](https://doi.org/10.1038%2Fng.3670)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFenu7nO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27643540)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6583788)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20of%20germline%20variants%20in%20the%20APOBEC3%20region%20with%20cancer%20risk%20and%20enrichment%20with%20APOBEC-signature%20mutations%20in%20tumors&journal=Nat.%20Genet.&doi=10.1038%2Fng.3670&volume=48&pages=1330-1338&publication_year=2016&author=Middlebrooks%2CCD) 
    
5.  Nik-Zainal, S. et al. Association of a germline copy number polymorphism of APOBEC3A and APOBEC3B with burden of putative APOBEC-dependent mutations in breast cancer. _Nat. Genet._ **46**, 487–491 (2014).
    
    [Article](https://doi.org/10.1038%2Fng.2955)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmtlWrtbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24728294)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4137149)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20of%20a%20germline%20copy%20number%20polymorphism%20of%20APOBEC3A%20and%20APOBEC3B%20with%20burden%20of%20putative%20APOBEC-dependent%20mutations%20in%20breast%20cancer&journal=Nat.%20Genet.&doi=10.1038%2Fng.2955&volume=46&pages=487-491&publication_year=2014&author=Nik-Zainal%2CS) 
    
6.  Yu, C.-C. et al. Mutant allele specific imbalance in oncogenes with copy number alterations: Occurrence, mechanisms, and potential clinical implications. _Cancer Lett._ **384**, 86–93 (2017).
    
    [Article](https://doi.org/10.1016%2Fj.canlet.2016.10.013)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1yisbnP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27725226)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutant%20allele%20specific%20imbalance%20in%20oncogenes%20with%20copy%20number%20alterations%3A%20Occurrence%2C%20mechanisms%2C%20and%20potential%20clinical%20implications&journal=Cancer%20Lett.&doi=10.1016%2Fj.canlet.2016.10.013&volume=384&pages=86-93&publication_year=2017&author=Yu%2CC-C) 
    
7.  Chatrath, A., Ratan, A. & Dutta, A. Germline variants that affect tumor progression. _Trends Genet._ **37**, 433–443 (2021).
    
    [Article](https://doi.org/10.1016%2Fj.tig.2020.10.005)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1Cqu7nE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33203571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germline%20variants%20that%20affect%20tumor%20progression&journal=Trends%20Genet.&doi=10.1016%2Fj.tig.2020.10.005&volume=37&pages=433-443&publication_year=2021&author=Chatrath%2CA&author=Ratan%2CA&author=Dutta%2CA) 
    
8.  Martincorena, I. et al. Universal patterns of selection in cancer and somatic tissues. _Cell_ **171**, 1029–1041.e21 (2017).
    
    [Article](https://doi.org/10.1016%2Fj.cell.2017.09.042)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs12lsLzO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29056346)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5720395)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Universal%20patterns%20of%20selection%20in%20cancer%20and%20somatic%20tissues&journal=Cell&doi=10.1016%2Fj.cell.2017.09.042&volume=171&pages=1029-1041.e21&publication_year=2017&author=Martincorena%2CI) 
    
9.  Salk, J. J. et al. Ultra-sensitive TP53 sequencing for cancer detection reveals progressive clonal selection in normal tissue over a century of human lifespan. _Cell Rep._ **28**, 132–144.e3 (2019).
    
    [Article](https://doi.org/10.1016%2Fj.celrep.2019.05.109)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWns7zF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31269435)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6639023)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ultra-sensitive%20TP53%20sequencing%20for%20cancer%20detection%20reveals%20progressive%20clonal%20selection%20in%20normal%20tissue%20over%20a%20century%20of%20human%20lifespan&journal=Cell%20Rep.&doi=10.1016%2Fj.celrep.2019.05.109&volume=28&pages=132-144.e3&publication_year=2019&author=Salk%2CJJ) 
    
10.  Martincorena, I. et al. Somatic mutant clones colonize the human esophagus with age. _Science_ [https://doi.org/10.1126/science.aau3879](https://doi.org/10.1126/science.aau3879) (2018).
    
    [Article](https://doi.org/10.1126%2Fscience.aau3879)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30337457)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6298579)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutant%20clones%20colonize%20the%20human%20esophagus%20with%20age&journal=Science&doi=10.1126%2Fscience.aau3879&publication_year=2018&author=Martincorena%2CI) 
    
11.  Blokzijl, F. et al. Tissue-specific mutation accumulation in human adult stem cells during life. _Nature_ **538**, 260–264 (2016).
    
    [Article](https://doi.org/10.1038%2Fnature19768)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1SltLbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27698416)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536223)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tissue-specific%20mutation%20accumulation%20in%20human%20adult%20stem%20cells%20during%20life&journal=Nature&doi=10.1038%2Fnature19768&volume=538&pages=260-264&publication_year=2016&author=Blokzijl%2CF) 
    
12.  Brunner, S. F. et al. Somatic mutations and clonal dynamics in healthy and cirrhotic human liver. _Nature_ **574**, 538–542 (2019).
    
    [Article](https://doi.org/10.1038%2Fs41586-019-1670-9)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31645727)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6837891)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutations%20and%20clonal%20dynamics%20in%20healthy%20and%20cirrhotic%20human%20liver&journal=Nature&doi=10.1038%2Fs41586-019-1670-9&volume=574&pages=538-542&publication_year=2019&author=Brunner%2CSF) 
    
13.  Moore, L. et al. The mutational landscape of normal human endometrial epithelium. _Nature_ **580**, 640–646 (2020).
    
    [Article](https://doi.org/10.1038%2Fs41586-020-2214-z)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVCjsb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32350471)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20landscape%20of%20normal%20human%20endometrial%20epithelium&journal=Nature&doi=10.1038%2Fs41586-020-2214-z&volume=580&pages=640-646&publication_year=2020&author=Moore%2CL) 
    
14.  Martincorena, I. et al. Tumor evolution. High burden and pervasive positive selection of somatic mutations in normal human skin. _Science_ **348**, 880–886 (2015).
    
    [Article](https://doi.org/10.1126%2Fscience.aaa6806)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXosFaktbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25999502)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471149)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tumor%20evolution.%20High%20burden%20and%20pervasive%20positive%20selection%20of%20somatic%20mutations%20in%20normal%20human%20skin&journal=Science&doi=10.1126%2Fscience.aaa6806&volume=348&pages=880-886&publication_year=2015&author=Martincorena%2CI) 
    
15.  Yoshida, K. et al. Tobacco smoking and somatic mutations in human bronchial epithelium. _Nature_ **578**, 266–272 (2020).
    
    [Article](https://doi.org/10.1038%2Fs41586-020-1961-1)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1Sksbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31996850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7021511)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tobacco%20smoking%20and%20somatic%20mutations%20in%20human%20bronchial%20epithelium&journal=Nature&doi=10.1038%2Fs41586-020-1961-1&volume=578&pages=266-272&publication_year=2020&author=Yoshida%2CK) 
    
16.  Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. _Science_ **367**, 1449–1454 (2020).
    
    [Article](https://doi.org/10.1126%2Fscience.aay9333)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslymtb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32217721)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20evolutionary%20dynamics%20and%20fitness%20landscape%20of%20clonal%20hematopoiesis&journal=Science&doi=10.1126%2Fscience.aay9333&volume=367&pages=1449-1454&publication_year=2020&author=Watson%2CCJ) 
    
17.  Robertson, N. A. et al. Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects. _Nat. Med._ **28**, 1439–1446 (2022).
    
    [Article](https://doi.org/10.1038%2Fs41591-022-01883-3)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhslSlu7fK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35788175)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9307482)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Longitudinal%20dynamics%20of%20clonal%20hematopoiesis%20identifies%20gene-specific%20fitness%20effects&journal=Nat.%20Med.&doi=10.1038%2Fs41591-022-01883-3&volume=28&pages=1439-1446&publication_year=2022&author=Robertson%2CNA) 
    
18.  Kennedy, A. L. et al. Distinct genetic pathways define pre-malignant versus compensatory clonal hematopoiesis in Shwachman–Diamond syndrome. _Nat. Commun._ **12**, 1334 (2021).
    
    [Article](https://doi.org/10.1038%2Fs41467-021-21588-4)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXltlyktbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33637765)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7910481)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20genetic%20pathways%20define%20pre-malignant%20versus%20compensatory%20clonal%20hematopoiesis%20in%20Shwachman%E2%80%93Diamond%20syndrome&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-21588-4&volume=12&publication_year=2021&author=Kennedy%2CAL) 
    
19.  Churpek, J. E. et al. Genomic analysis of germ line and somatic variants in familial myelodysplasia/acute myeloid leukemia. _Blood_ **126**, 2484–2490 (2015).
    
    [Article](https://doi.org/10.1182%2Fblood-2015-04-641100)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xjt1KgsLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26492932)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4661171)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20analysis%20of%20germ%20line%20and%20somatic%20variants%20in%20familial%20myelodysplasia%2Facute%20myeloid%20leukemia&journal=Blood&doi=10.1182%2Fblood-2015-04-641100&volume=126&pages=2484-2490&publication_year=2015&author=Churpek%2CJE) 
    
20.  Brown, A. L. et al. RUNX1-mutated families show phenotype heterogeneity and a somatic mutation profile unique to germline predisposed AML. _Blood Adv._ **4**, 1131–1144 (2020).
    
    [Article](https://doi.org/10.1182%2Fbloodadvances.2019000901)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis12ktrnK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32208489)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094007)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RUNX1-mutated%20families%20show%20phenotype%20heterogeneity%20and%20a%20somatic%20mutation%20profile%20unique%20to%20germline%20predisposed%20AML&journal=Blood%20Adv.&doi=10.1182%2Fbloodadvances.2019000901&volume=4&pages=1131-1144&publication_year=2020&author=Brown%2CAL) 
    
21.  Liggett, L. A. et al. Precocious clonal hematopoiesis in Down syndrome is accompanied by immune dysregulation. _Blood Adv._ **5**, 1791–1796 (2021).
    
    [Article](https://doi.org/10.1182%2Fbloodadvances.2020003858)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33787858)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8045490)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Precocious%20clonal%20hematopoiesis%20in%20Down%20syndrome%20is%20accompanied%20by%20immune%20dysregulation&journal=Blood%20Adv.&doi=10.1182%2Fbloodadvances.2020003858&volume=5&pages=1791-1796&publication_year=2021&author=Liggett%2CLA) 
    
22.  Richards, S. et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. _Genet. Med._ **17**, 405–424 (2015).
    
    [Article](https://doi.org/10.1038%2Fgim.2015.30)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25741868)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4544753)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Standards%20and%20guidelines%20for%20the%20interpretation%20of%20sequence%20variants%3A%20a%20joint%20consensus%20recommendation%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20and%20the%20Association%20for%20Molecular%20Pathology&journal=Genet.%20Med.&doi=10.1038%2Fgim.2015.30&volume=17&pages=405-424&publication_year=2015&author=Richards%2CS) 
    
23.  Hu, C. et al. A population-based study of genes previously implicated in breast cancer. _N. Engl. J. Med._ **384**, 440–451 (2021).
    
    [Article](https://doi.org/10.1056%2FNEJMoa2005936)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33471974)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8127622)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20population-based%20study%20of%20genes%20previously%20implicated%20in%20breast%20cancer&journal=N.%20Engl.%20J.%20Med.&doi=10.1056%2FNEJMoa2005936&volume=384&pages=440-451&publication_year=2021&author=Hu%2CC) 
    
24.  Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. _Nature_ **559**, 350–355 (2018).
    
    [Article](https://doi.org/10.1038%2Fs41586-018-0321-x)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjsbbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29995854)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6054542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20into%20clonal%20haematopoiesis%20from%208%2C342%20mosaic%20chromosomal%20alterations&journal=Nature&doi=10.1038%2Fs41586-018-0321-x&volume=559&pages=350-355&publication_year=2018&author=Loh%2CP-R) 
    
25.  Escherich, C. S. et al. Germ line genetic NBN variation and predisposition to B-cell acute lymphoblastic leukemia in children. _Blood_ **143**, 2270–2283 (2024).
    
    [Article](https://doi.org/10.1182%2Fblood.2023023336)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1SisbfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38446568)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11443573)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germ%20line%20genetic%20NBN%20variation%20and%20predisposition%20to%20B-cell%20acute%20lymphoblastic%20leukemia%20in%20children&journal=Blood&doi=10.1182%2Fblood.2023023336&volume=143&pages=2270-2283&publication_year=2024&author=Escherich%2CCS) 
    
26.  Belhadj, S. et al. NBN pathogenic germline variants are associated with pan-cancer susceptibility and in vitro DNA damage response defects. _Clin. Cancer Res._ **29**, 422–431 (2023).
    
    [Article](https://doi.org/10.1158%2F1078-0432.CCR-22-1703)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjvFOqsb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36346689)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NBN%20pathogenic%20germline%20variants%20are%20associated%20with%20pan-cancer%20susceptibility%20and%20in%20vitro%20DNA%20damage%20response%20defects&journal=Clin.%20Cancer%20Res.&doi=10.1158%2F1078-0432.CCR-22-1703&volume=29&pages=422-431&publication_year=2023&author=Belhadj%2CS) 
    
27.  Georgeson, P. et al. Evaluating the utility of tumour mutational signatures for identifying hereditary colorectal cancer and polyposis syndrome carriers. _Gut_ **70**, 2138–2149 (2021).
    
    [Article](https://doi.org/10.1136%2Fgutjnl-2019-320462)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1GisrvP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33414168)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluating%20the%20utility%20of%20tumour%20mutational%20signatures%20for%20identifying%20hereditary%20colorectal%20cancer%20and%20polyposis%20syndrome%20carriers&journal=Gut&doi=10.1136%2Fgutjnl-2019-320462&volume=70&pages=2138-2149&publication_year=2021&author=Georgeson%2CP) 
    
28.  Baccelli, F. et al. ERCC6L2-related disease: a novel entity of bone marrow failure disorder with high risk of clonal evolution. _Ann. Hematol._ **102**, 699–705 (2023).
    
    [Article](https://link.springer.com/doi/10.1007/s00277-023-05128-2)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjsVGksLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36790458)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9998559)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ERCC6L2-related%20disease%3A%20a%20novel%20entity%20of%20bone%20marrow%20failure%20disorder%20with%20high%20risk%20of%20clonal%20evolution&journal=Ann.%20Hematol.&doi=10.1007%2Fs00277-023-05128-2&volume=102&pages=699-705&publication_year=2023&author=Baccelli%2CF) 
    
29.  Ferrer, A., Mangaonkar, A. A. & Patnaik, M. M. Clonal hematopoiesis and myeloid neoplasms in the context of telomere biology disorders. _Curr. Hematol. Malig. Rep._ **17**, 61–68 (2022).
    
    [Article](https://link.springer.com/doi/10.1007/s11899-022-00662-8)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35524933)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9077347)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20and%20myeloid%20neoplasms%20in%20the%20context%20of%20telomere%20biology%20disorders&journal=Curr.%20Hematol.%20Malig.%20Rep.&doi=10.1007%2Fs11899-022-00662-8&volume=17&pages=61-68&publication_year=2022&author=Ferrer%2CA&author=Mangaonkar%2CAA&author=Patnaik%2CMM) 
    
30.  Lasho, T. & Patnaik, M. M. Adaptive and maladaptive clonal hematopoiesis in telomere biology disorders. _Curr. Hematol. Malig. Rep._ **19**, 35–44 (2024).
    
    [Article](https://link.springer.com/doi/10.1007/s11899-023-00719-2)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38095828)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adaptive%20and%20maladaptive%20clonal%20hematopoiesis%20in%20telomere%20biology%20disorders&journal=Curr.%20Hematol.%20Malig.%20Rep.&doi=10.1007%2Fs11899-023-00719-2&volume=19&pages=35-44&publication_year=2024&author=Lasho%2CT&author=Patnaik%2CMM) 
    
31.  Saito, Y. et al. Casitas B-cell lymphoma mutation in childhood T-cell acute lymphoblastic leukemia. _Leuk. Res._ **36**, 1009–1015 (2012).
    
    [Article](https://doi.org/10.1016%2Fj.leukres.2012.04.018)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XntFaiu7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22591685)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3693942)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Casitas%20B-cell%20lymphoma%20mutation%20in%20childhood%20T-cell%20acute%20lymphoblastic%20leukemia&journal=Leuk.%20Res.&doi=10.1016%2Fj.leukres.2012.04.018&volume=36&pages=1009-1015&publication_year=2012&author=Saito%2CY) 
    
32.  DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. _N. Engl. J. Med._ **388**, 2422–2433 (2023).
    
    [Article](https://doi.org/10.1056%2FNEJMoa2300503)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1Kgtr7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37140166)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10501156)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Familial%20clonal%20hematopoiesis%20in%20a%20long%20telomere%20syndrome&journal=N.%20Engl.%20J.%20Med.&doi=10.1056%2FNEJMoa2300503&volume=388&pages=2422-2433&publication_year=2023&author=DeBoy%2CEA) 
    
33.  Lim, T. L. et al. Germline POT1 variants can predispose to myeloid and lymphoid neoplasms. _Leukemia_ **36**, 283–287 (2022).
    
    [Article](https://doi.org/10.1038%2Fs41375-021-01335-w)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitVGgsbvK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34193977)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germline%20POT1%20variants%20can%20predispose%20to%20myeloid%20and%20lymphoid%20neoplasms&journal=Leukemia&doi=10.1038%2Fs41375-021-01335-w&volume=36&pages=283-287&publication_year=2022&author=Lim%2CTL) 
    
34.  Di Paola, J. & Porter, C. C. ETV6-related thrombocytopenia and leukemia predisposition. _Blood_ **134**, 663–667 (2019).
    
    [Article](https://doi.org/10.1182%2Fblood.2019852418)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVCgsrjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31248877)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6706811)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETV6-related%20thrombocytopenia%20and%20leukemia%20predisposition&journal=Blood&doi=10.1182%2Fblood.2019852418&volume=134&pages=663-667&publication_year=2019&author=Paola%2CJ&author=Porter%2CCC) 
    
35.  Wong, T. N. et al. Role of TP53 mutations in the origin and evolution of therapy-related acute myeloid leukaemia. _Nature_ **518**, 552–555 (2015).
    
    [Article](https://doi.org/10.1038%2Fnature13968)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFams77O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25487151)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20TP53%20mutations%20in%20the%20origin%20and%20evolution%20of%20therapy-related%20acute%20myeloid%20leukaemia&journal=Nature&doi=10.1038%2Fnature13968&volume=518&pages=552-555&publication_year=2015&author=Wong%2CTN) 
    
36.  Li, P. et al. The genetic landscape of germline DDX41 variants predisposing to myeloid neoplasms. _Blood_ **140**, 716–755 (2022).
    
    [Article](https://doi.org/10.1182%2Fblood.2021015135)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitF2isbjN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35671390)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9389629)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genetic%20landscape%20of%20germline%20DDX41%20variants%20predisposing%20to%20myeloid%20neoplasms&journal=Blood&doi=10.1182%2Fblood.2021015135&volume=140&pages=716-755&publication_year=2022&author=Li%2CP) 
    
37.  Molteni, E. et al. Prevalence and clinical expression of germ line predisposition to myeloid neoplasms in adults with marrow hypocellularity. _Blood_ **142**, 643–657 (2023).
    
    [Article](https://doi.org/10.1182%2Fblood.2022019304)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslWnsL7N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37216690)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10644067)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prevalence%20and%20clinical%20expression%20of%20germ%20line%20predisposition%20to%20myeloid%20neoplasms%20in%20adults%20with%20marrow%20hypocellularity&journal=Blood&doi=10.1182%2Fblood.2022019304&volume=142&pages=643-657&publication_year=2023&author=Molteni%2CE) 
    
38.  Niemeyer, C. M. et al. Germline CBL mutations cause developmental abnormalities and predispose to juvenile myelomonocytic leukemia. _Nat. Genet._ **42**, 794–800 (2010).
    
    [Article](https://doi.org/10.1038%2Fng.641)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXpvVWgsL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20694012)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4297285)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germline%20CBL%20mutations%20cause%20developmental%20abnormalities%20and%20predispose%20to%20juvenile%20myelomonocytic%20leukemia&journal=Nat.%20Genet.&doi=10.1038%2Fng.641&volume=42&pages=794-800&publication_year=2010&author=Niemeyer%2CCM) 
    
39.  Alfayez, M. et al. The clinical impact of PTPN11 mutations in adults with acute myeloid leukemia. _Leukemia_ **35**, 691–700 (2021).
    
    [Article](https://doi.org/10.1038%2Fs41375-020-0920-z)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Wls77M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32561839)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20clinical%20impact%20of%20PTPN11%20mutations%20in%20adults%20with%20acute%20myeloid%20leukemia&journal=Leukemia&doi=10.1038%2Fs41375-020-0920-z&volume=35&pages=691-700&publication_year=2021&author=Alfayez%2CM) 
    
40.  Marsh, J. C. W. et al. Heterozygous RTEL1 variants in bone marrow failure and myeloid neoplasms. _Blood Adv._ **2**, 36–48 (2018).
    
    [Article](https://doi.org/10.1182%2Fbloodadvances.2017008110)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlsVCkt7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29344583)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5761623)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Heterozygous%20RTEL1%20variants%20in%20bone%20marrow%20failure%20and%20myeloid%20neoplasms&journal=Blood%20Adv.&doi=10.1182%2Fbloodadvances.2017008110&volume=2&pages=36-48&publication_year=2018&author=Marsh%2CJCW) 
    
41.  Park, J.-Y. et al. Complementation of hypersensitivity to DNA interstrand crosslinking agents demonstrates that XRCC2 is a Fanconi anaemia gene. _J. Med. Genet._ **53**, 672–680 (2016).
    
    [Article](https://doi.org/10.1136%2Fjmedgenet-2016-103847)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXmtVyqu7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27208205)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complementation%20of%20hypersensitivity%20to%20DNA%20interstrand%20crosslinking%20agents%20demonstrates%20that%20XRCC2%20is%20a%20Fanconi%20anaemia%20gene&journal=J.%20Med.%20Genet.&doi=10.1136%2Fjmedgenet-2016-103847&volume=53&pages=672-680&publication_year=2016&author=Park%2CJ-Y) 
    
42.  Kim, Y. et al. Mutations of the SLX4 gene in Fanconi anemia. _Nat. Genet._ **43**, 142–146 (2011).
    
    [Article](https://doi.org/10.1038%2Fng.750)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmsFCntw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21240275)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3345287)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutations%20of%20the%20SLX4%20gene%20in%20Fanconi%20anemia&journal=Nat.%20Genet.&doi=10.1038%2Fng.750&volume=43&pages=142-146&publication_year=2011&author=Kim%2CY) 
    
43.  Porter, C. C. Germ line mutations associated with leukemias. _Hematology_ **2016**, 302–308 (2016).
    
    [Article](https://doi.org/10.1182%2Fasheducation-2016.1.302)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27913495)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6142470)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germ%20line%20mutations%20associated%20with%20leukemias&journal=Hematology&doi=10.1182%2Fasheducation-2016.1.302&volume=2016&pages=302-308&publication_year=2016&author=Porter%2CCC) 
    
44.  Grolleman, J. E. et al. Mutational signature analysis reveals NTHL1 deficiency to cause a multi-tumor phenotype. _Cancer Cell_ **35**, 256–266.e5 (2019).
    
    [Article](https://doi.org/10.1016%2Fj.ccell.2018.12.011)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXivFSmsrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30753826)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20signature%20analysis%20reveals%20NTHL1%20deficiency%20to%20cause%20a%20multi-tumor%20phenotype&journal=Cancer%20Cell&doi=10.1016%2Fj.ccell.2018.12.011&volume=35&pages=256-266.e5&publication_year=2019&author=Grolleman%2CJE) 
    
45.  Palles, C. et al. Germline mutations affecting the proofreading domains of POLE and POLD1 predispose to colorectal adenomas and carcinomas. _Nat. Genet._ **45**, 136–144 (2013).
    
    [Article](https://doi.org/10.1038%2Fng.2503)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVOqsrvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23263490)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germline%20mutations%20affecting%20the%20proofreading%20domains%20of%20POLE%20and%20POLD1%20predispose%20to%20colorectal%20adenomas%20and%20carcinomas&journal=Nat.%20Genet.&doi=10.1038%2Fng.2503&volume=45&pages=136-144&publication_year=2013&author=Palles%2CC) 
    
46.  Lampson, B. L. et al. Rare germline ATM variants influence the development of chronic lymphocytic leukemia. _J. Clin. Oncol._ **41**, 1116–1128 (2023).
    
    [Article](https://doi.org/10.1200%2FJCO.22.00269)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtVWjs70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36315919)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rare%20germline%20ATM%20variants%20influence%20the%20development%20of%20chronic%20lymphocytic%20leukemia&journal=J.%20Clin.%20Oncol.&doi=10.1200%2FJCO.22.00269&volume=41&pages=1116-1128&publication_year=2023&author=Lampson%2CBL) 
    
47.  Boonen, R. A. C. M., Vreeswijk, M. P. G. & van Attikum, H. CHEK2 variants: linking functional impact to cancer risk. _Trends Cancer_ **8**, 759–770 (2022).
    
    [Article](https://doi.org/10.1016%2Fj.trecan.2022.04.009)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVygu7bL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35643632)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CHEK2%20variants%3A%20linking%20functional%20impact%20to%20cancer%20risk&journal=Trends%20Cancer&doi=10.1016%2Fj.trecan.2022.04.009&volume=8&pages=759-770&publication_year=2022&author=Boonen%2CRACM&author=Vreeswijk%2CMPG&author=Attikum%2CH) 
    
48.  Freiman, L. et al. Germline CHEK2 mutations in patients with myeloid neoplasms. _Leukemia_ **38**, 908–911 (2024).
    
    [Article](https://doi.org/10.1038%2Fs41375-024-02179-w)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXktF2ntr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38378842)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germline%20CHEK2%20mutations%20in%20patients%20with%20myeloid%20neoplasms&journal=Leukemia&doi=10.1038%2Fs41375-024-02179-w&volume=38&pages=908-911&publication_year=2024&author=Freiman%2CL) 
    
49.  Bick, A. G. et al. Inherited causes of clonal haematopoiesis in 97,691 whole genomes. _Nature_ **586**, 763–768 (2020).
    
    [Article](https://doi.org/10.1038%2Fs41586-020-2819-2)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVyiu7nK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33057201)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7944936)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inherited%20causes%20of%20clonal%20haematopoiesis%20in%2097%2C691%20whole%20genomes&journal=Nature&doi=10.1038%2Fs41586-020-2819-2&volume=586&pages=763-768&publication_year=2020&author=Bick%2CAG) 
    
50.  Kessler, M. D. et al. Common and rare variant associations with clonal haematopoiesis phenotypes. _Nature_ **612**, 301–309 (2022).
    
    [Article](https://doi.org/10.1038%2Fs41586-022-05448-9)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVWhtLzN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36450978)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9713173)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Common%20and%20rare%20variant%20associations%20with%20clonal%20haematopoiesis%20phenotypes&journal=Nature&doi=10.1038%2Fs41586-022-05448-9&volume=612&pages=301-309&publication_year=2022&author=Kessler%2CMD) 
    
51.  Liu, J., Osman, A. E. G., Bolton, K. & Godley, L. A. Germline predisposition to clonal hematopoiesis. _Leuk. Res._ **132**, 107344 (2023).
    
    [Article](https://doi.org/10.1016%2Fj.leukres.2023.107344)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVGjtr7M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37421681)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Germline%20predisposition%20to%20clonal%20hematopoiesis&journal=Leuk.%20Res.&doi=10.1016%2Fj.leukres.2023.107344&volume=132&publication_year=2023&author=Liu%2CJ&author=Osman%2CAEG&author=Bolton%2CK&author=Godley%2CLA) 
    
52.  Weinstock, J. S. et al. Aberrant activation of TCL1A promotes stem cell expansion in clonal haematopoiesis. _Nature_ **616**, 755–763 (2023).
    
    [Article](https://doi.org/10.1038%2Fs41586-023-05806-1)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXns1yqs7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37046083)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10360040)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aberrant%20activation%20of%20TCL1A%20promotes%20stem%20cell%20expansion%20in%20clonal%20haematopoiesis&journal=Nature&doi=10.1038%2Fs41586-023-05806-1&volume=616&pages=755-763&publication_year=2023&author=Weinstock%2CJS) 
    
53.  Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. _Nature_ **559**, 400–404 (2018).
    
    [Article](https://doi.org/10.1038%2Fs41586-018-0317-6)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjs7rI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29988082)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6485381)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prediction%20of%20acute%20myeloid%20leukaemia%20risk%20in%20healthy%20individuals&journal=Nature&doi=10.1038%2Fs41586-018-0317-6&volume=559&pages=400-404&publication_year=2018&author=Abelson%2CS) 
    
54.  Desai, P. et al. Somatic mutations precede acute myeloid leukemia years before diagnosis. _Nat. Med._ **24**, 1015–1023 (2018).
    
    [Article](https://doi.org/10.1038%2Fs41591-018-0081-z)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjsLrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29988143)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6849383)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutations%20precede%20acute%20myeloid%20leukemia%20years%20before%20diagnosis&journal=Nat.%20Med.&doi=10.1038%2Fs41591-018-0081-z&volume=24&pages=1015-1023&publication_year=2018&author=Desai%2CP) 
    
55.  Bolton, K. L. et al. Cancer therapy shapes the fitness landscape of clonal hematopoiesis. _Nat. Genet._ **52**, 1219–1226 (2020).
    
    [Article](https://doi.org/10.1038%2Fs41588-020-00710-0)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFKlu73E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33106634)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7891089)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cancer%20therapy%20shapes%20the%20fitness%20landscape%20of%20clonal%20hematopoiesis&journal=Nat.%20Genet.&doi=10.1038%2Fs41588-020-00710-0&volume=52&pages=1219-1226&publication_year=2020&author=Bolton%2CKL) 
    
56.  Tran, D. et al. Plasma proteomic signature predicts myeloid neoplasm risk. _Clin. Cancer Res._ [https://doi.org/10.1158/1078-0432.CCR-23-3468](https://doi.org/10.1158/1078-0432.CCR-23-3468) (2024).
    
57.  Weeks, L. D. et al. Prediction of risk for myeloid malignancy in clonal hematopoiesis. _NEJM Evid._ [https://doi.org/10.1056/EVIDoa2200310](https://doi.org/10.1056/EVIDoa2200310) (2023).
    
    [Article](https://doi.org/10.1056%2FEVIDoa2200310)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37483562)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10361696)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prediction%20of%20risk%20for%20myeloid%20malignancy%20in%20clonal%20hematopoiesis&journal=NEJM%20Evid.&doi=10.1056%2FEVIDoa2200310&publication_year=2023&author=Weeks%2CLD) 
    
58.  Gu, M. et al. Multiparameter prediction of myeloid neoplasia risk. _Nat. Genet._ **55**, 1523–1530 (2023).
    
    [Article](https://doi.org/10.1038%2Fs41588-023-01472-1)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslKnt7%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37620601)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10484784)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiparameter%20prediction%20of%20myeloid%20neoplasia%20risk&journal=Nat.%20Genet.&doi=10.1038%2Fs41588-023-01472-1&volume=55&pages=1523-1530&publication_year=2023&author=Gu%2CM) 
    
59.  Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. _PLoS Med._ **12**, e1001779 (2015).
    
    [Article](https://doi.org/10.1371%2Fjournal.pmed.1001779)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25826379)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380465)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UK%20Biobank%3A%20an%20open%20access%20resource%20for%20identifying%20the%20causes%20of%20a%20wide%20range%20of%20complex%20diseases%20of%20middle%20and%20old%20age&journal=PLoS%20Med.&doi=10.1371%2Fjournal.pmed.1001779&volume=12&publication_year=2015&author=Sudlow%2CC) 
    
60.  Van Hout, C. V. et al. Exome sequencing and characterization of 49,960 individuals in the UK Biobank. _Nature_ **586**, 749–756 (2020).
    
    [Article](https://doi.org/10.1038%2Fs41586-020-2853-0)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33087929)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7759458)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Exome%20sequencing%20and%20characterization%20of%2049%2C960%20individuals%20in%20the%20UK%20Biobank&journal=Nature&doi=10.1038%2Fs41586-020-2853-0&volume=586&pages=749-756&publication_year=2020&author=Hout%2CCV) 
    
61.  Bick, A. G. et al. Genomic data in the All of Us research program. _Nature_ **627**, 340–346 (2024).
    
    [Article](https://doi.org/10.1038%2Fs41586-023-06957-x)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20data%20in%20the%20All%20of%20Us%20research%20program&journal=Nature&doi=10.1038%2Fs41586-023-06957-x&volume=627&pages=340-346&publication_year=2024&author=Bick%2CAG) 
    
62.  Uddin, M. M. et al. Germline genomic and phenomic landscape of clonal hematopoiesis in 323,112 individuals. Preprint at _MedRxiv_ [https://doi.org/10.1101/2022.07.29.22278015](https://doi.org/10.1101/2022.07.29.22278015) (2022).
    
63.  Huang, K. et al. Pathogenic germline variants in 10,389 adult cancers. _Cell_ **173**, 355–370.e14 (2018).
    
    [Article](https://doi.org/10.1016%2Fj.cell.2018.03.039)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFahsrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29625052)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5949147)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathogenic%20germline%20variants%20in%2010%2C389%20adult%20cancers&journal=Cell&doi=10.1016%2Fj.cell.2018.03.039&volume=173&pages=355-370.e14&publication_year=2018&author=Huang%2CK) 
    
64.  Cheng, D. T. et al. Comprehensive detection of germline variants by MSK-IMPACT, a clinical diagnostic platform for solid tumor molecular oncology and concurrent cancer predisposition testing. _BMC Med. Genet._ **10**, 33 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20detection%20of%20germline%20variants%20by%20MSK-IMPACT%2C%20a%20clinical%20diagnostic%20platform%20for%20solid%20tumor%20molecular%20oncology%20and%20concurrent%20cancer%20predisposition%20testing&journal=BMC%20Med.%20Genet.&volume=10&publication_year=2017&author=Cheng%2CDT) 
    
65.  Bolton, K. L. et al. Clonal hematopoiesis is associated with risk of severe Covid-19. _Nat. Commun._ **12**, 5975 (2021).
    
    [Article](https://doi.org/10.1038%2Fs41467-021-26138-6)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1ektLfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34645798)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8514469)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20is%20associated%20with%20risk%20of%20severe%20Covid-19&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-26138-6&volume=12&publication_year=2021&author=Bolton%2CKL) 
    
66.  Cheng, D. T. et al. Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT): a hybridization capture-based next-generation sequencing clinical assay for solid tumor molecular oncology. _J. Mol. Diagn._ **17**, 251–264 (2015).
    
    [Article](https://doi.org/10.1016%2Fj.jmoldx.2014.12.006)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlt1aitLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25801821)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5808190)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Memorial%20Sloan%20Kettering-Integrated%20Mutation%20Profiling%20of%20Actionable%20Cancer%20Targets%20%28MSK-IMPACT%29%3A%20a%20hybridization%20capture-based%20next-generation%20sequencing%20clinical%20assay%20for%20solid%20tumor%20molecular%20oncology&journal=J.%20Mol.%20Diagn.&doi=10.1016%2Fj.jmoldx.2014.12.006&volume=17&pages=251-264&publication_year=2015&author=Cheng%2CDT) 
    
67.  Franch-Expósito, S. et al. Associations between cancer predisposition mutations and clonal hematopoiesis in patients with solid tumors. _JCO Precis. Oncol._ [https://doi.org/10.1200/PO.23.00070](https://doi.org/10.1200/PO.23.00070) (2023).
    
68.  Szustakowski, J. D. et al. Advancing human genetics research and drug discovery through exome sequencing of the UK Biobank. _Nat. Genet._ **53**, 942–948 (2021).
    
    [Article](https://doi.org/10.1038%2Fs41588-021-00885-0)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVenurnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34183854)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Advancing%20human%20genetics%20research%20and%20drug%20discovery%20through%20exome%20sequencing%20of%20the%20UK%20Biobank&journal=Nat.%20Genet.&doi=10.1038%2Fs41588-021-00885-0&volume=53&pages=942-948&publication_year=2021&author=Szustakowski%2CJD) 
    
69.  Cibulskis, K. et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. _Nat. Biotechnol._ **31**, 213–219 (2013).
    
    [Article](https://doi.org/10.1038%2Fnbt.2514)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXjsFCntbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23396013)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3833702)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sensitive%20detection%20of%20somatic%20point%20mutations%20in%20impure%20and%20heterogeneous%20cancer%20samples&journal=Nat.%20Biotechnol.&doi=10.1038%2Fnbt.2514&volume=31&pages=213-219&publication_year=2013&author=Cibulskis%2CK) 
    
70.  Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. _Nucleic Acids Res._ **44**, e108 (2016).
    
    [Article](https://doi.org/10.1093%2Fnar%2Fgkw227)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27060149)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914105)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=VarDict%3A%20a%20novel%20and%20versatile%20variant%20caller%20for%20next-generation%20sequencing%20in%20cancer%20research&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgkw227&volume=44&publication_year=2016&author=Lai%2CZ) 
    
71.  Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. _Nature_ **625**, 92–100 (2024).
    
    [Article](https://doi.org/10.1038%2Fs41586-023-06045-0)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFGgsrjF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38057664)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genomic%20mutational%20constraint%20map%20using%20variation%20in%2076%2C156%20human%20genomes&journal=Nature&doi=10.1038%2Fs41586-023-06045-0&volume=625&pages=92-100&publication_year=2024&author=Chen%2CS) 
    
72.  Vlasschaert, C. et al. A practical approach to curate clonal hematopoiesis of indeterminate potential in human genetic data sets. _Blood_ **141**, 2214–2223 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXpt1emtr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36652671)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10273159)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20practical%20approach%20to%20curate%20clonal%20hematopoiesis%20of%20indeterminate%20potential%20in%20human%20genetic%20data%20sets&journal=Blood&volume=141&pages=2214-2223&publication_year=2023&author=Vlasschaert%2CC) 
    
73.  Bick, A. & Regan, J. CHIP-Detection-Mutect2. _Terra.bio_ [https://app.terra.bio/#workspaces/terra-outreach/CHIP-Detection-Mutect2](https://app.terra.bio/#workspaces/terra-outreach/CHIP-Detection-Mutect2) (2016).
    
74.  Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. _J. R. Stat. Soc. Ser. B_ **57**, 289–300 (1995).
    
75.  Maas, P. et al. Breast cancer risk from modifiable and nonmodifiable risk factors among white women in the United States. _JAMA Oncol._ **2**, 1295–1302 (2016).
    
    [Article](https://doi.org/10.1001%2Fjamaoncol.2016.1025)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27228256)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5719876)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Breast%20cancer%20risk%20from%20modifiable%20and%20nonmodifiable%20risk%20factors%20among%20white%20women%20in%20the%20United%20States&journal=JAMA%20Oncol.&doi=10.1001%2Fjamaoncol.2016.1025&volume=2&pages=1295-1302&publication_year=2016&author=Maas%2CP) 
    
76.  Liu, J. et al. germlineCH\_NG: code for statistical testing and figure/table generation. _Zenodo_ [https://doi.org/10.5281/zenodo.15200120](https://doi.org/10.5281/zenodo.15200120) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02250-x?format=refman&flavour=references)

Acknowledgements
----------------

This work was supported by the MDS Foundation (to K.L.B.), the Children’s Discovery Institute (to K.L.B.) and the National Institutes of Health (NIH; grant nos. R01HL148050 and R01HL168894 to P.N.). K.H.S. and K.L.B. were supported by the Prostate Cancer Foundation Challenge Award. M.J.W. was supported by the Edward P. Evans Foundation. A.N. was supported by funds from the SciLifeLab & Wallenberg Data Driven Life Science Program (grant no. KAW 2020.0239), the Swedish Cancer Foundation (grant nos. 22.0577JIA and 22.2362Pj) and the Swedish Research Council (grant no. 2023-03131). A.G.B. is supported by the NIH (grant nos. DP5 OD029586, R01AG088657 and R01 AG083736), a Burroughs Wellcome Fund Career Award for Medical Scientists, an E.P. Evans Foundation grant, a Pew Charitable Trusts and Alexander and Margaret Stewart Trust Pew-Stewart Scholar for Cancer Research award and a Hevolution/AFAR New Investigator Award in Aging Biology and Geroscience Research. This research has been conducted using the UKBB Resource under application no. 55288 and uses data provided by patients and collected by the National Health Service as part of their care and support. We gratefully acknowledge All of Us participants for their contributions, without whom this research would not have been possible. We also thank the NIH’s All of Us Research Program for making available the participant data examined in this study.

Author information
------------------

Author notes

1.  These authors contributed equally: Daniel C. Link, Zsofia K. Stadler, Alexander G. Bick, Pradeep Natarajan, Kelly L. Bolton.
    

### Authors and Affiliations

1.  Cancer Biology Graduate Program, Washington University School of Medicine, St. Louis, MO, USA
    
    Jie Liu
    
2.  Division of Oncology, Department of Medicine, Washington University School of Medicine, St. Louis, MO, USA
    
    Jie Liu, Duc Tran, Brian J. Wiley, Irenaeus C. C. Chan, Indraniel Das, Li Ding, Matthew J. Walter, Daniel C. Link & Kelly L. Bolton
    
3.  Program in Medical and Population Genetics and Cardiovascular Disease Initiative, Broad Institute of MIT and Harvard, Cambridge, MA, USA
    
    Liying Xue, Md Mesbah Uddin, Abhishek Niroula, Gabriel Griffin, Benjamin L. Ebert & Pradeep Natarajan
    
4.  Department of Medicine, Queen’s University, Kingston, Ontario, Canada
    
    Caitlyn Vlasschaert
    
5.  Early Cancer Institute, University of Cambridge, Cambridge, UK
    
    Caroline J. Watson, Hamish A. J. MacGregor & Jamie R. Blundell
    
6.  Centre for Cancer Genetic Epidemiology, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK
    
    Hamish A. J. MacGregor
    
7.  Division of Public Health Sciences, Department of Surgery, Washington University School of Medicine, St. Louis, MO, USA
    
    Xiaoyu Zong & Yin Cao
    
8.  Department of Medical Biochemistry and Cell Biology, Institute of Biomedicine, University of Gothenburg, Gothenburg, Sweden
    
    Abhishek Niroula
    
9.  SciLifeLab, University of Gothenburg, Gothenburg, Sweden
    
    Abhishek Niroula
    
10.  Division of Genetic Medicine, Department of Medicine, Vanderbilt University, Nashville, TN, USA
    
    Taralynn Mack, Yash Pershad, Brian Sharber & Alexander G. Bick
    
11.  Memorial Sloan-Kettering Cancer Center, New York, NY, USA
    
    Michael Berger, Ahmet Zehir, Ryan Ptashkin, Ross L. Levine, Elli Papaemmanuil, Vijai Joseph, Teng Gao, Yelena Kemel, Diana Mandelker, Semanti Mukherjee, Kenneth Offit & Zsofia K. Stadler
    
12.  Department of Epidemiologic Methods and Etiologic Research, Leibniz Institute for Prevention Research and Epidemiology—BIPS and Faculty of Human and Health Sciences, University of Bremen, Bremen, Germany
    
    Konrad H. Stopsack
    
13.  Department of Computational Biomedicine, Cedars-Sinai Medical Center, Los Angeles, CA, USA
    
    Paul D. P. Pharoah
    
14.  Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, MO, USA
    
    Yin Cao
    
15.  Division of Gastroenterology, Department of Medicine, Washington University School of Medicine, St. Louis, MO, USA
    
    Yin Cao
    
16.  Department of Biostatistics, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA
    
    Nilanjan Chatterjee
    
17.  Division of Hematology/Oncology, Department of Medicine, and the Robert H. Lurie Comprehensive Cancer Center, Northwestern University, Chicago, IL, USA
    
    Lucy A. Godley
    

Authors

1.  Jie Liu
    
    [View author publications](https://www.nature.com/search?author=Jie%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jie%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jie%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Duc Tran
    
    [View author publications](https://www.nature.com/search?author=Duc%20Tran)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Duc%20Tran) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Duc%20Tran%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Liying Xue
    
    [View author publications](https://www.nature.com/search?author=Liying%20Xue)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Liying%20Xue) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Liying%20Xue%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Brian J. Wiley
    
    [View author publications](https://www.nature.com/search?author=Brian%20J.%20Wiley)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Brian%20J.%20Wiley) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Brian%20J.%20Wiley%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Caitlyn Vlasschaert
    
    [View author publications](https://www.nature.com/search?author=Caitlyn%20Vlasschaert)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Caitlyn%20Vlasschaert) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Caitlyn%20Vlasschaert%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Caroline J. Watson
    
    [View author publications](https://www.nature.com/search?author=Caroline%20J.%20Watson)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Caroline%20J.%20Watson) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Caroline%20J.%20Watson%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Hamish A. J. MacGregor
    
    [View author publications](https://www.nature.com/search?author=Hamish%20A.%20J.%20MacGregor)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hamish%20A.%20J.%20MacGregor) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hamish%20A.%20J.%20MacGregor%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Xiaoyu Zong
    
    [View author publications](https://www.nature.com/search?author=Xiaoyu%20Zong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiaoyu%20Zong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiaoyu%20Zong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Irenaeus C. C. Chan
    
    [View author publications](https://www.nature.com/search?author=Irenaeus%20C.%20C.%20Chan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Irenaeus%20C.%20C.%20Chan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Irenaeus%20C.%20C.%20Chan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Indraniel Das
    
    [View author publications](https://www.nature.com/search?author=Indraniel%20Das)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Indraniel%20Das) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Indraniel%20Das%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Md Mesbah Uddin
    
    [View author publications](https://www.nature.com/search?author=Md%20Mesbah%20Uddin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Md%20Mesbah%20Uddin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Md%20Mesbah%20Uddin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Abhishek Niroula
    
    [View author publications](https://www.nature.com/search?author=Abhishek%20Niroula)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Abhishek%20Niroula) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Abhishek%20Niroula%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Gabriel Griffin
    
    [View author publications](https://www.nature.com/search?author=Gabriel%20Griffin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gabriel%20Griffin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gabriel%20Griffin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Benjamin L. Ebert
    
    [View author publications](https://www.nature.com/search?author=Benjamin%20L.%20Ebert)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Benjamin%20L.%20Ebert) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Benjamin%20L.%20Ebert%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Taralynn Mack
    
    [View author publications](https://www.nature.com/search?author=Taralynn%20Mack)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Taralynn%20Mack) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Taralynn%20Mack%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Yash Pershad
    
    [View author publications](https://www.nature.com/search?author=Yash%20Pershad)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yash%20Pershad) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yash%20Pershad%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Brian Sharber
    
    [View author publications](https://www.nature.com/search?author=Brian%20Sharber)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Brian%20Sharber) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Brian%20Sharber%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Michael Berger
    
    [View author publications](https://www.nature.com/search?author=Michael%20Berger)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Michael%20Berger) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Michael%20Berger%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Ahmet Zehir
    
    [View author publications](https://www.nature.com/search?author=Ahmet%20Zehir)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ahmet%20Zehir) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ahmet%20Zehir%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Ryan Ptashkin
    
    [View author publications](https://www.nature.com/search?author=Ryan%20Ptashkin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ryan%20Ptashkin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ryan%20Ptashkin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Ross L. Levine
    
    [View author publications](https://www.nature.com/search?author=Ross%20L.%20Levine)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ross%20L.%20Levine) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ross%20L.%20Levine%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Elli Papaemmanuil
    
    [View author publications](https://www.nature.com/search?author=Elli%20Papaemmanuil)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Elli%20Papaemmanuil) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Elli%20Papaemmanuil%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Vijai Joseph
    
    [View author publications](https://www.nature.com/search?author=Vijai%20Joseph)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Vijai%20Joseph) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Vijai%20Joseph%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Teng Gao
    
    [View author publications](https://www.nature.com/search?author=Teng%20Gao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Teng%20Gao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Teng%20Gao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Yelena Kemel
    
    [View author publications](https://www.nature.com/search?author=Yelena%20Kemel)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yelena%20Kemel) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yelena%20Kemel%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Diana Mandelker
    
    [View author publications](https://www.nature.com/search?author=Diana%20Mandelker)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Diana%20Mandelker) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Diana%20Mandelker%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  Konrad H. Stopsack
    
    [View author publications](https://www.nature.com/search?author=Konrad%20H.%20Stopsack)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Konrad%20H.%20Stopsack) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Konrad%20H.%20Stopsack%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Paul D. P. Pharoah
    
    [View author publications](https://www.nature.com/search?author=Paul%20D.%20P.%20Pharoah)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Paul%20D.%20P.%20Pharoah) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Paul%20D.%20P.%20Pharoah%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Semanti Mukherjee
    
    [View author publications](https://www.nature.com/search?author=Semanti%20Mukherjee)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Semanti%20Mukherjee) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Semanti%20Mukherjee%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  Li Ding
    
    [View author publications](https://www.nature.com/search?author=Li%20Ding)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Li%20Ding) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Li%20Ding%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  Yin Cao
    
    [View author publications](https://www.nature.com/search?author=Yin%20Cao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yin%20Cao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yin%20Cao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
32.  Matthew J. Walter
    
    [View author publications](https://www.nature.com/search?author=Matthew%20J.%20Walter)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20J.%20Walter) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20J.%20Walter%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
33.  Jamie R. Blundell
    
    [View author publications](https://www.nature.com/search?author=Jamie%20R.%20Blundell)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jamie%20R.%20Blundell) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jamie%20R.%20Blundell%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
34.  Nilanjan Chatterjee
    
    [View author publications](https://www.nature.com/search?author=Nilanjan%20Chatterjee)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nilanjan%20Chatterjee) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nilanjan%20Chatterjee%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
35.  Kenneth Offit
    
    [View author publications](https://www.nature.com/search?author=Kenneth%20Offit)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kenneth%20Offit) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kenneth%20Offit%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
36.  Lucy A. Godley
    
    [View author publications](https://www.nature.com/search?author=Lucy%20A.%20Godley)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lucy%20A.%20Godley) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lucy%20A.%20Godley%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
37.  Daniel C. Link
    
    [View author publications](https://www.nature.com/search?author=Daniel%20C.%20Link)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20C.%20Link) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20C.%20Link%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
38.  Zsofia K. Stadler
    
    [View author publications](https://www.nature.com/search?author=Zsofia%20K.%20Stadler)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zsofia%20K.%20Stadler) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zsofia%20K.%20Stadler%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
39.  Alexander G. Bick
    
    [View author publications](https://www.nature.com/search?author=Alexander%20G.%20Bick)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alexander%20G.%20Bick) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alexander%20G.%20Bick%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
40.  Pradeep Natarajan
    
    [View author publications](https://www.nature.com/search?author=Pradeep%20Natarajan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Pradeep%20Natarajan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Pradeep%20Natarajan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
41.  Kelly L. Bolton
    
    [View author publications](https://www.nature.com/search?author=Kelly%20L.%20Bolton)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kelly%20L.%20Bolton) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kelly%20L.%20Bolton%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

J.L. and K.L.B. conceived of and designed the study, curated pathogenic germline variants and drafted the paper. J.L. and X.Z. curated phenotypic data. J.L., D.T., B.J.W. and I.C.C.C. called variants and performed postprocessing of sequencing data. J.L., D.T., L.X., C.J.W. and H.A.J.M. performed the statistical analysis and/or participated in data interpretation. D.C.L., Z.K.S., A.G.B., P.N. and K.L.B. jointly supervised the study. J.L., D.T., L.X., B.J.W., C.V., C.J.W., H.A.J.M., X.Z., I.C.C.C., I.D., M.M.U., A.N., G.G., B.L.E., T.M., Y.P., B.S., M.B., A.Z., R.P., R.L.L., E.P., V.J., T.G., Y.K., D.M., K.H.S., P.D.P.P., S.M., L.D., Y.C., M.J.W., J.R.B., N.C., K.O., L.A.G., D.C.L., Z.K.S., A.G.B., P.N. and K.L.B. contributed to the writing of the paper and approved it for submission.

### Corresponding author

Correspondence to [Kelly L. Bolton](mailto:bolton@wustl.edu).

Ethics declarations
-------------------

### Competing interests

K.L.B. reports research grants and personal fees from Servier. E.P. is the co-founder of and holds a fiduciary role in Isabl Inc. P.N. reports research grants from Allelica, Amgen, Apple, Boston Scientific, Genentech or Roche and Novartis; personal fees from Allelica, Apple, AstraZeneca, Blackstone Life Sciences, Bristol Myers Squibb, Creative Education Concepts, CRISPR Therapeutics, Eli Lilly & Co, Esperion Therapeutics, Foresite Capital, Foresite Labs, Genentech or Roche, GV, HeartFlow, Magnet Biomedicine, Merck, Novartis, Novo Nordisk, TenSixteen Bio and Tourmaline Bio; equity in Bolt, Candela, Mercury, MyOme, Parameter Health, Preciseli and TenSixteen Bio; and spousal employment at Vertex Pharmaceuticals, all unrelated to the present work. R.L.L. is on the Supervisory Board of QIAGEN (compensation or equity), a co-founder of or board member at Ajax (equity) and a scientific advisor to Mission Bio, Kurome, Anovia, Bakx, Syndax, Scorpion, Zentalis, Auron, Prelude and C4 Therapeutics; for each of these entities he receives equity or compensation. He has received research support from the Cure Breast Cancer Foundation, Calico, Zentalis and Ajax, and has consulted for Jubilant, Goldman Sachs, Incyte, AstraZeneca and Janssen. A.G.B. reports personal fees and equity in TenSixteen Bio, all unrelated to this work. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Kirsten Grønbæk and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02250-x#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Distribution of CH by germline carrier status in the UK Biobank.](https://www.nature.com/articles/s41588-025-02250-x/figures/6)

**a**. Prevalence by age for CH in gene with known relevance to both myeloid and lymphoid malignancies, only myeloid malignancies, only lymphoid malignancies or solid tumors (left), and mCA in autosomal chromosomes, loss of X chromosome, or loss of Y chromosome (right). Data are presented as the CH prevalence fitted using polynomial regression of degree 2 (center line) ± 95% confidence interval (CI) for the fitted line (error bands). **b**. Shown in red and blue are the odds ratios (center dot) and 95% CI (error bars) for CH categories were calculated using multivariable Firth’s bias-reduced logistic regression among people with dominant (n = 33,106) or recessive (n = 43,981) germline variant compared to those without germline variant (n = 354,774) after adjustment for age at blood draw, the first three genetic principal components, and exome sequencing batch. \*_Q_ value (FDR-corrected _P_ value) < 0.05, \*\*_Q_ < 0.01, \*\*\*_Q_ < 0.001. See Supplementary Table [9](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for exact _Q_ values. **c**. Comparison of the proportion of people with 1, 2 or 3 + CH and the distribution of maximum variant allele frequency between people with dominant germline variant (n = 33,106) and people with no germline variant (n = 354,774). _P_ values for the associations between the number of CH and having dominant germline variant were derived from logistic regression and _P_ values for the association between the maximum VAF of CH and having dominant germline variant were derived from linear regression, with adjustment for the above covariates. Center line represents median. Box limits represent upper and lower quartiles. Whiskers represent 1.5x interquartile range. Cell fractions of mCA-auto were not included due to inaccuracy. \*_P_ < 0.05, \*\*_P_ < 0.01, \*\*\*_P_ < 0.001. Two-sided _P_ values not corrected for multiple hypothesis testing.

### [Extended Data Fig. 2 CH characteristics by germline status in the UK Biobank.](https://www.nature.com/articles/s41588-025-02250-x/figures/7)

**a**. Heatmap showing the beta coefficients for associations between the number and maximum logVAF of CH mutations and germline genes that significantly increased overall CH-heme or mCA-auto. Shown are the coefficients from linear regression adjusted for age at blood draw, the first three genetic principal components, and exome sequencing batch. \*_P_ value < 0.05. Two-sided _P_ value is not corrected for multiple hypothesis testing. **b**. Comparison of single base substitutions composition (top) and trinucleotide context (middle and bottom) for CH by germline status. Shown are genes/pathways with at least 10 SNVs. NER: Nucleotide Excision Repair, BER: Base excision repair, MMR: mismatch repair.

### [Extended Data Fig. 3 Association between germline predisposition genes and subtypes of hematologic malignancies in the UK Biobank.](https://www.nature.com/articles/s41588-025-02250-x/figures/8)

Heatmap showing the log hazard ratio for hematologic malignancy subtypes among germline pathogenic variants carriers of genes that predispose to both CH and hematologic malignancies compared to non-carriers. ‘Secondary’ refers to hematologic cancers occurring after a solid tumor diagnosis. Hazard ratios were obtained from Cox regression adjusted for age at blood draw, the first three genetic principal components, and exome sequencing batch. \*_P_ < 0.05. Two-sided _P_ value is not corrected for multiple hypothesis testing. AML, acute myeloid leukemia; MPN, myeloproliferative neoplasms; MDS, myelodysplastic syndromes; CML, chronic myelogenous leukemia; NHL, non-Hodgkin’s lymphoma; HL, Hodgkin’s lymphoma; CLL, chronic lymphocytic leukemia; MM, multiple myeloma.

### [Extended Data Fig. 4 Association of _CHEK2_ and _ATM_ germline carrier status with CH and hematologic malignancies by germline variant type in the UK Biobank.](https://www.nature.com/articles/s41588-025-02250-x/figures/9)

**a**. Shown in red and blue are the odds ratios (center dot) and 95% confidence intervals (error bars) for CH-heme and mCA-auto among people with germline _CHEK2_ loss of function variants, _CHEK2_ missense variants, _ATM_ loss of function variants or _ATM_ missense variants compared to non-carriers. The odds ratios were calculated using multivariable Firth’s bias-reduced logistic regression adjusted for age at blood draw, the first three genetic principal components, and exome sequencing batch. **b**. Shown in red and blue are the hazard ratios (center dot) and 95% confidence intervals (error bars) for myeloid and lymphoid malignancies by _CHEK2_ or _ATM_ germline variant type. The hazard ratios were calculated using Cox regression adjusted for the same covariates as above. The number of individuals for above analyses are as follows: germline _CHEK2_ loss of function variant carriers (n = 2,670), _CHEK2_ missense variant carriers (n = 1,318), _ATM_ loss of function variant carriers (n = 1,564) and _ATM_ missense variant carriers (n = 535). \*_P_ < 0.05, \*\*_P_ < 0.01, \*\*\*_P_ < 0.001. Two-sided _P_ value is not corrected for multiple hypothesis testing. See Supplementary Tables [15](https://www.nature.com/articles/s41588-025-02250-x#MOESM4), [16](https://www.nature.com/articles/s41588-025-02250-x#MOESM4) for exact _P_ values.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_MOESM1_ESM.pdf)

Supplementary Notes 1–5 and Figs. 1–6.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02250-x/MediaObjects/41588_2025_2250_MOESM4_ESM.xlsx)

Supplementary Tables 1–20 show descriptive and analytics results.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Germline%20genetic%20variation%20impacts%20clonal%20hematopoiesis%20landscape%20and%20progression%20to%20malignancy&author=Jie%20Liu%20et%20al&contentID=10.1038%2Fs41588-025-02250-x&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-15&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY-NC-ND)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02250-x)

### Cite this article

Liu, J., Tran, D., Xue, L. _et al._ Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02250-x

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02250-x?format=refman&flavour=citation)

-   Received: 28 August 2024
    
-   Accepted: 29 May 2025
    
-   Published: 15 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02250-x
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02250-x.pdf)
