> id: https://www.nature.com/articles/s41588-025-02221-2

> link: https://www.nature.com/articles/s41588-025-02221-2

> title: Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers

# Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers
_Published on Fri, 04 Jul 2025 00:00:00 GMT_

Abstract
--------

Synthetic lethal interactions (SLIs) based on genomic alterations in cancer have been therapeutically explored. We investigated the SLI space as a function of differential RNA expression in cancer and normal tissue. Computational analyses of functional genomic and gene expression resources uncovered a cancer-specific SLI between the paralogs cytidine diphosphate diacylglycerol synthase 1 (_CDS1_) and _CDS2_. The essentiality of _CDS2_ for cell survival is observed for mesenchymal-like cancers, which have low or absent _CDS1_ expression and account for roughly half of all cancers. Mechanistically, the _CDS1–2_ SLI is accompanied by disruption of lipid homeostasis, including accumulation of cholesterol esters and triglycerides, and apoptosis. Genome-wide CRISPR–Cas9 knockout screens in _CDS1_\-negative cancer cells identify no common escape mechanism of death caused by _CDS2_ ablation, indicating the robustness of the SLI. Synthetic lethality is driven by CDS2 dosage and depends on catalytic activity. Thus, CDS2 may serve as a pharmacologically tractable target in mesenchymal-like cancers.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_Fig1_HTML.png)

### [The synthetic lethal interaction between _CDS1_ and _CDS2_ is a vulnerability in uveal melanoma and across multiple tumor types](https://www.nature.com/articles/s41588-025-02222-1?fromPaywallRec=false)

Article Open access 04 July 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-90047-3/MediaObjects/41598_2021_90047_Fig1_HTML.png)

### [From multi-omics integration towards novel genomic interaction networks to identify key cancer cell line characteristics](https://www.nature.com/articles/s41598-021-90047-3?fromPaywallRec=false)

Article Open access 18 May 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-020-71224-2/MediaObjects/41598_2020_71224_Fig1_HTML.png)

### [A highly efficient non-viral process for programming mesenchymal stem cells for gene directed enzyme prodrug cancer therapy](https://www.nature.com/articles/s41598-020-71224-2?fromPaywallRec=false)

Article Open access 31 August 2020

Main
----

SLIs can provide a therapeutic index in cancer[1](https://www.nature.com/articles/s41588-025-02221-2#ref-CR1 "Hartwell, L. H., Szankasi, P., Roberts, C. J., Murray, A. W. & Friend, S. H. Integrating genetic approaches into the discovery of anticancer drugs. Science 278, 1064–1068 (1997)."), as exemplified by the dependency on poly(ADP-ribose) polymerase (PARP) of _BRCA_\-deficient hereditary breast and ovarian cancer. Whereas healthy cells retain one functional germline _BRCA_ copy, somatic loss of the second copy results in cancer. As _BRCA_ deficiency results in loss of homology-based DNA repair[2](https://www.nature.com/articles/s41588-025-02221-2#ref-CR2 "Venkitaraman, A. R. Cancer susceptibility and the functions of BRCA1 and BRCA2. Cell 108, 171–182 (2002)."), it causes a strong dependency on PARP for genomic stability and, hence, survival. This SLI has been clinically exploited with specific PARP inhibitors[3](https://www.nature.com/articles/s41588-025-02221-2#ref-CR3 "Bryant, H. E. et al. Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase. Nature 434, 913–917 (2005)."),[4](https://www.nature.com/articles/s41588-025-02221-2#ref-CR4 "Farmer, H. et al. Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy. Nature 434, 917–921 (2005)."),[5](https://www.nature.com/articles/s41588-025-02221-2#ref-CR5 "Fong, P. C. et al. Inhibition of poly(ADP-ribose) polymerase in tumors from mutation carriers. N. Engl. J. Med. 361, 123–134 (2009)."). This clinical success, and other examples, have spurred efforts to identify additional cancer-associated SLIs[6](https://www.nature.com/articles/s41588-025-02221-2#ref-CR6 "Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. Nature 568, 551–556 (2019)."),[7](https://www.nature.com/articles/s41588-025-02221-2#ref-CR7 "Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. iScience 13, 488–497 (2019)."),[8](https://www.nature.com/articles/s41588-025-02221-2#ref-CR8 "Frezza, C. et al. Haem oxygenase is synthetically lethal with the tumour suppressor fumarate hydratase. Nature 477, 225–228 (2011)."),[9](https://www.nature.com/articles/s41588-025-02221-2#ref-CR9 "Feng, X. et al. Genome-wide CRISPR screens using isogenic cells reveal vulnerabilities conferred by loss of tumor suppressors. Sci. Adv. 8, eabm6638 (2022)."),[10](https://www.nature.com/articles/s41588-025-02221-2#ref-CR10 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[11](https://www.nature.com/articles/s41588-025-02221-2#ref-CR11 "Oike, T. et al. A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1. Cancer Res. 73, 5508–5518 (2013)."),[12](https://www.nature.com/articles/s41588-025-02221-2#ref-CR12 "Hoffman, G. R. et al. Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers. Proc. Natl Acad. Sci. USA 111, 3128–3133 (2014)."),[13](https://www.nature.com/articles/s41588-025-02221-2#ref-CR13 "Neggers, J. E. et al. Synthetic lethal interaction between the ESCRT paralog enzymes VPS4A and VPS4B in cancers harboring loss of chromosome 18q or 16q. Cell Rep. 33, 108475 (2020)."),[14](https://www.nature.com/articles/s41588-025-02221-2#ref-CR14 "Jerby-Arnon, L. et al. Predicting cancer-specific vulnerability via data-driven detection of synthetic lethality. Cell 158, 1199–1209 (2014)."),[15](https://www.nature.com/articles/s41588-025-02221-2#ref-CR15 "Muller, F. L. et al. Passenger deletions generate therapeutic vulnerabilities in cancer. Nature 488, 337–342 (2012)."),[16](https://www.nature.com/articles/s41588-025-02221-2#ref-CR16 "Zhao, D. et al. Synthetic essentiality of chromatin remodelling factor CHD1 in PTEN-deficient cancer. Nature 542, 484–488 (2017)."),[17](https://www.nature.com/articles/s41588-025-02221-2#ref-CR17 "Lee, R. et al. Synthetic essentiality of tryptophan 2,3-dioxygenase 2 in APC-mutated colorectal cancer. Cancer Discov. 12, 1702–1717 (2022)."),[18](https://www.nature.com/articles/s41588-025-02221-2#ref-CR18 "Mavrakis, K. J. et al. Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5. Science 351, 1208–1213 (2016)."),[19](https://www.nature.com/articles/s41588-025-02221-2#ref-CR19 "Kryukov, G. V. et al. MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells. Science 351, 1214–1218 (2016)."),[20](https://www.nature.com/articles/s41588-025-02221-2#ref-CR20 "Tong, A. H. Y. et al. Systematic genetic analysis with ordered arrays of yeast deletion mutants. Science 294, 2364–2368 (2001)."),[21](https://www.nature.com/articles/s41588-025-02221-2#ref-CR21 "Pan, X. et al. A robust toolkit for functional profiling of the yeast genome. Mol. Cell 16, 487–496 (2004)."),[22](https://www.nature.com/articles/s41588-025-02221-2#ref-CR22 "Costanzo, M. et al. The genetic landscape of a cell. Science 327, 425–431 (2010)."),[23](https://www.nature.com/articles/s41588-025-02221-2#ref-CR23 "Shalem, O. et al. Genome-scale CRISPR–Cas9 knockout screening in human cells. Science 343, 84–87 (2014)."),[24](https://www.nature.com/articles/s41588-025-02221-2#ref-CR24 "Wang, T., Wei, J. J., Sabatini, D. M. & Lander, E. S. Genetic screens in human cells using the CRISPR–Cas9 system. Science 343, 80–84 (2014)."),[25](https://www.nature.com/articles/s41588-025-02221-2#ref-CR25 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 565 (2021)."),[26](https://www.nature.com/articles/s41588-025-02221-2#ref-CR26 "van Delft, M. F. et al. The BH3 mimetic ABT-737 targets selective Bcl-2 proteins and efficiently induces apoptosis via Bak/Bax if Mcl-1 is neutralized. Cancer Cell 10, 389–399 (2006)."),[27](https://www.nature.com/articles/s41588-025-02221-2#ref-CR27 "Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. Genome Biol. 21, 262 (2020)."),[28](https://www.nature.com/articles/s41588-025-02221-2#ref-CR28 "Tsherniak, A. et al. Defining a cancer dependency map. Cell 170, 564–576.e16 (2017)."),[29](https://www.nature.com/articles/s41588-025-02221-2#ref-CR29 "Lee, J. S. et al. Synthetic lethality-mediated precision oncology via the tumor transcriptome. Cell 184, 2487–2502.e13 (2021)."),[30](https://www.nature.com/articles/s41588-025-02221-2#ref-CR30 "Lee, J. S. et al. Harnessing synthetic lethality to predict the response to cancer treatment. Nat. Commun. 9, 2546 (2018)."),[31](https://www.nature.com/articles/s41588-025-02221-2#ref-CR31 "Helming, K. C. et al. ARID1B is a specific vulnerability in ARID1A-mutant cancers. Nat. Med. 20, 251–254 (2014)."),[32](https://www.nature.com/articles/s41588-025-02221-2#ref-CR32 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."). Similar to _BRCA_–_PARP_, data on genomic alterations specific to cancer are commonly used to find SLIs[2](https://www.nature.com/articles/s41588-025-02221-2#ref-CR2 "Venkitaraman, A. R. Cancer susceptibility and the functions of BRCA1 and BRCA2. Cell 108, 171–182 (2002)."),[3](https://www.nature.com/articles/s41588-025-02221-2#ref-CR3 "Bryant, H. E. et al. Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase. Nature 434, 913–917 (2005)."),[4](https://www.nature.com/articles/s41588-025-02221-2#ref-CR4 "Farmer, H. et al. Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy. Nature 434, 917–921 (2005)."),[6](https://www.nature.com/articles/s41588-025-02221-2#ref-CR6 "Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. Nature 568, 551–556 (2019)."),[7](https://www.nature.com/articles/s41588-025-02221-2#ref-CR7 "Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. iScience 13, 488–497 (2019)."),[8](https://www.nature.com/articles/s41588-025-02221-2#ref-CR8 "Frezza, C. et al. Haem oxygenase is synthetically lethal with the tumour suppressor fumarate hydratase. Nature 477, 225–228 (2011)."),[9](https://www.nature.com/articles/s41588-025-02221-2#ref-CR9 "Feng, X. et al. Genome-wide CRISPR screens using isogenic cells reveal vulnerabilities conferred by loss of tumor suppressors. Sci. Adv. 8, eabm6638 (2022)."),[10](https://www.nature.com/articles/s41588-025-02221-2#ref-CR10 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[11](https://www.nature.com/articles/s41588-025-02221-2#ref-CR11 "Oike, T. et al. A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1. Cancer Res. 73, 5508–5518 (2013)."),[12](https://www.nature.com/articles/s41588-025-02221-2#ref-CR12 "Hoffman, G. R. et al. Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers. Proc. Natl Acad. Sci. USA 111, 3128–3133 (2014)."),[13](https://www.nature.com/articles/s41588-025-02221-2#ref-CR13 "Neggers, J. E. et al. Synthetic lethal interaction between the ESCRT paralog enzymes VPS4A and VPS4B in cancers harboring loss of chromosome 18q or 16q. Cell Rep. 33, 108475 (2020)."),[14](https://www.nature.com/articles/s41588-025-02221-2#ref-CR14 "Jerby-Arnon, L. et al. Predicting cancer-specific vulnerability via data-driven detection of synthetic lethality. Cell 158, 1199–1209 (2014)."),[15](https://www.nature.com/articles/s41588-025-02221-2#ref-CR15 "Muller, F. L. et al. Passenger deletions generate therapeutic vulnerabilities in cancer. Nature 488, 337–342 (2012)."),[16](https://www.nature.com/articles/s41588-025-02221-2#ref-CR16 "Zhao, D. et al. Synthetic essentiality of chromatin remodelling factor CHD1 in PTEN-deficient cancer. Nature 542, 484–488 (2017)."),[17](https://www.nature.com/articles/s41588-025-02221-2#ref-CR17 "Lee, R. et al. Synthetic essentiality of tryptophan 2,3-dioxygenase 2 in APC-mutated colorectal cancer. Cancer Discov. 12, 1702–1717 (2022)."),[18](https://www.nature.com/articles/s41588-025-02221-2#ref-CR18 "Mavrakis, K. J. et al. Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5. Science 351, 1208–1213 (2016)."),[19](https://www.nature.com/articles/s41588-025-02221-2#ref-CR19 "Kryukov, G. V. et al. MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells. Science 351, 1214–1218 (2016)."),[31](https://www.nature.com/articles/s41588-025-02221-2#ref-CR31 "Helming, K. C. et al. ARID1B is a specific vulnerability in ARID1A-mutant cancers. Nat. Med. 20, 251–254 (2014)."), such as the dependency on _WRN_ in microsatellite unstable cancers[6](https://www.nature.com/articles/s41588-025-02221-2#ref-CR6 "Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. Nature 568, 551–556 (2019)."),[7](https://www.nature.com/articles/s41588-025-02221-2#ref-CR7 "Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. iScience 13, 488–497 (2019).").

In addition to genomic alterations, cancers also have distinct gene expression profiles relative to healthy tissue. Publicly available datasets provide extensive transcriptional information as well as genetic dependencies in cancer. For example, the cancer Dependency Map (DepMap) is a comprehensive research initiative providing matched CRISPR screening and gene expression data for 913 cell lines across 30 cancer lineages (2021Q2)[33](https://www.nature.com/articles/s41588-025-02221-2#ref-CR33 "Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. Nat. Genet. 49, 1779–1784 (2017)."),[34](https://www.nature.com/articles/s41588-025-02221-2#ref-CR34 "DepMap, B. DepMap 21Q2 Public. figshare 
https://doi.org/10.6084/m9.figshare.14541774.v2
(2021)."). In addition, the Cancer Genome Atlas (TCGA) provides genome-wide gene expression data for 9,264 tumor and 741 healthy samples[35](https://www.nature.com/articles/s41588-025-02221-2#ref-CR35 "Weinstein, J. N. et al. The cancer genome atlas pan-cancer analysis project. Nat. Genet. 45, 1113–1120 (2013)."). Together, these datasets provide 250 million datapoints on gene expression and genetic dependencies in cancer and can be mined to predict SLIs. For example, gene expression data have been used to retrospectively predict patient responses to targeted therapy using computational synthetic lethality analyses[29](https://www.nature.com/articles/s41588-025-02221-2#ref-CR29 "Lee, J. S. et al. Synthetic lethality-mediated precision oncology via the tumor transcriptome. Cell 184, 2487–2502.e13 (2021).").

Here, we performed unbiased computational analyses to find cancer-specific SLIs, taking advantage of these public functional genomic and gene expression resources, particularly the Genotype-Tissue Expression (GTEx) project, which provides genome-wide gene expression data for 17,382 healthy tissue samples[36](https://www.nature.com/articles/s41588-025-02221-2#ref-CR36 "Lonsdale, J. et al. The Genotype-Tissue Expression (GTEx) project. Nat. Genet. 45, 580–585 (2013)."). We scored all ~340 million gene pairs for synthetic lethality and cancer specificity. We focused on the strong SLI that we uncovered between _CDS1_ and _CDS2_, investigating cancer specificity, robustness of the synthetic lethal relationship in vitro and in mice, potential escape mechanisms, mechanism of cell death and lipidomic and proteomic analysis.

Results
-------

### _CDS1_–_2_ form a common cancer-associated SLI pair

To query for SLIs based on differential gene expression between cancer and normal tissue, we set up a bioinformatic pipeline using public datasets on gene dependencies and RNA expression (Fig. [1a](https://www.nature.com/articles/s41588-025-02221-2#Fig1)). This pipeline identifies anchor–target gene pairs, in which the anchor gene shows a relatively reduced expression level in cancer compared with normal tissue and in which disruption of the target gene results in cancer lethality (Fig. [1b](https://www.nature.com/articles/s41588-025-02221-2#Fig1)).

**Fig. 1: _CDS1_–_2_ form a common cancer-associated SLI pair.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02221-2/figures/1)

**a**, Diagram depicting synthetic lethality based on lack of expression in cancer. **b**, Schematic depicting the computational analysis to identify cancer-specific SLIs. **c**, The correlation (Pearson’s _r_) of RNA, proteomic, mutation or methylation anchor gene data and gene dependency data in cancer cell lines for previously functionally validated synthetic lethal pairs. The rank of the known target gene partners was plotted for each data type. The significance cut-off was the top 5% enrichment (rank-based). _n_ (cancer cell lines) = 913 (RNA), 299 (protein), 986 (damaging mutation) and 513 (methylation). **d**, Top predicted synthetic lethal pairs, named by their target gene and anchor gene, respectively. The _y_ axis presents the correlation (Pearson’s _r_) between the anchor gene expression and target gene dependency (DepMap). The _x_ axis presents the average reduction in anchor expression in patient tumor samples compared with patient healthy tissue samples (TCGA). Bonferroni-corrected _P_ values from Pearson correlation tests were used to determine significance for synthetic lethality (_P_ < 0.05). For the top synthetic lethal interactions, the significance of cancer specificity was assessed in Extended Data Fig. [1d,e](https://www.nature.com/articles/s41588-025-02221-2#Fig8). _n_ = 913 cancer cell lines, 10,005 patient samples (9,264 tumor and 741 healthy). **e**, For each cancer lineage, the percentage of Δ_CDS2_ lethal cancer cell lines plotted against the percentage of low _CDS1_\-expressing cancer cell lines (DepMap). _n_ = 906 cancer cell lines. **f**, Probability of survival for patients with the highest and lowest quartiles of _CDS1_ expression in patient tumor samples (TCGA). The hazard ratio for the _CDS1_\-low quartile was 1.90 \[1.715–2.106\] compared with the _CDS1_\-high quartile. The log-rank Mantel–Cox test was used. _n_ = 10,163 patients. **g**, Calibration gene-normalized RNA expression in healthy tissue samples (GTEx), cancer cell lines (DepMap), Δ_CDS2_ lethal cancer cell lines (DepMap) and Δ_CDS2_ nonlethal cancer cell lines (DepMap), presented in violin plots for the genes _B2M_, _E2F1_, _CDKN2B_ and _CDS1_. The three dotted or dashed lines in each violin plot represent the quartile cut-offs. For significant differences, the percentage change in average TPM was indicated, with one overall average for controls. The Kruskal–Wallis test followed by Dunn’s test was used. _n_ = 17,382 healthy tissue samples and 913 cancer cell lines (657 Δ_CDS2_ nonlethal and 256 Δ_CDS2_ lethal). **h**, Calibration gene-normalized CDS1 protein levels in matched healthy lung and lung tumor samples[47](https://www.nature.com/articles/s41588-025-02221-2#ref-CR47 "Satpathy, S. et al. A proteogenomic portrait of lung squamous cell carcinoma. Cell 184, 4348–4371.e40 (2021)."). The three dotted or dashed lines in each violin plot represent the quartile cut-offs. The Mann–Whitney _U_ test was used. _n_ = 110 tumor samples and 102 adjacent healthy lung samples. In **f**, **g** and **h**: \*\*\*_P_ < 0.001. NS, not significant.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM4)

The DepMap includes additional types of data that we incorporated in our pipeline to score SLIs, including promoter methylation data, protein expression data, damaging mutation data and cancer-type data[33](https://www.nature.com/articles/s41588-025-02221-2#ref-CR33 "Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. Nat. Genet. 49, 1779–1784 (2017)."),[34](https://www.nature.com/articles/s41588-025-02221-2#ref-CR34 "DepMap, B. DepMap 21Q2 Public. figshare 
https://doi.org/10.6084/m9.figshare.14541774.v2
(2021)."),[37](https://www.nature.com/articles/s41588-025-02221-2#ref-CR37 "Nusinow, D. P. et al. Quantitative proteomics of the Cancer Cell Line Encyclopedia. Cell 180, 387–402.e16 (2020)."),[38](https://www.nature.com/articles/s41588-025-02221-2#ref-CR38 "Ghandi, M. et al. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019)."). The pipeline uses combined gene dependency data from project SCORE (Sanger Institute) and project Achilles (Broad Institute). RNA expression data provided sufficient power to detect previously established SLIs, including _BRCA1_–_PARP1_ and _WRN_–_MLH1_ (Fig. [1c](https://www.nature.com/articles/s41588-025-02221-2#Fig1) and Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02221-2#Fig8))[3](https://www.nature.com/articles/s41588-025-02221-2#ref-CR3 "Bryant, H. E. et al. Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase. Nature 434, 913–917 (2005)."),[4](https://www.nature.com/articles/s41588-025-02221-2#ref-CR4 "Farmer, H. et al. Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy. Nature 434, 917–921 (2005)."),[6](https://www.nature.com/articles/s41588-025-02221-2#ref-CR6 "Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. Nature 568, 551–556 (2019)."),[7](https://www.nature.com/articles/s41588-025-02221-2#ref-CR7 "Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. iScience 13, 488–497 (2019)."),[10](https://www.nature.com/articles/s41588-025-02221-2#ref-CR10 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[11](https://www.nature.com/articles/s41588-025-02221-2#ref-CR11 "Oike, T. et al. A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1. Cancer Res. 73, 5508–5518 (2013)."),[12](https://www.nature.com/articles/s41588-025-02221-2#ref-CR12 "Hoffman, G. R. et al. Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers. Proc. Natl Acad. Sci. USA 111, 3128–3133 (2014)."),[13](https://www.nature.com/articles/s41588-025-02221-2#ref-CR13 "Neggers, J. E. et al. Synthetic lethal interaction between the ESCRT paralog enzymes VPS4A and VPS4B in cancers harboring loss of chromosome 18q or 16q. Cell Rep. 33, 108475 (2020)."),[25](https://www.nature.com/articles/s41588-025-02221-2#ref-CR25 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 565 (2021)."),[26](https://www.nature.com/articles/s41588-025-02221-2#ref-CR26 "van Delft, M. F. et al. The BH3 mimetic ABT-737 targets selective Bcl-2 proteins and efficiently induces apoptosis via Bak/Bax if Mcl-1 is neutralized. Cancer Cell 10, 389–399 (2006)."),[39](https://www.nature.com/articles/s41588-025-02221-2#ref-CR39 "DeWeirdt, P. C. et al. Genetic screens in isogenic mammalian cell lines without single cell cloning. Nat. Commun. 11, 752 (2020)."), with _WRN_–_MLH1_ serving as an example of the broader SLI between _WRN_ and the microsatellite instability caused by _MLH1_ deficiency. Although some SLIs may be best detected in single cancer types, we observed that analyzing all cancer types together improved resolution for identifying established interactions (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02221-2#Fig8)). Therefore, we determined the SLI scores for all gene pairs with RNA expression data for all cancer types simultaneously.

The _CDS1_–_CDS2_ and _NAA10_–_NAA11_ gene pairs received high SLI scores (Fig. [1d](https://www.nature.com/articles/s41588-025-02221-2#Fig1), Extended Data Fig. [1b–e](https://www.nature.com/articles/s41588-025-02221-2#Fig8) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). However, the _NAA10_–_NAA11_ SLI failed to show cancer specificity and was therefore not pursued. In contrast, the _CDS1_–_CDS2_ SLI demonstrated significant cancer specificity. The high SLI score for _CDS1_–_CDS2_ is consistent with other analyses and screens scoring _CDS1_–_CDS2_ as a candidate synthetic lethal pair, but to our knowledge this has not yet been pursued[10](https://www.nature.com/articles/s41588-025-02221-2#ref-CR10 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[25](https://www.nature.com/articles/s41588-025-02221-2#ref-CR25 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 565 (2021)."),[32](https://www.nature.com/articles/s41588-025-02221-2#ref-CR32 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."),[40](https://www.nature.com/articles/s41588-025-02221-2#ref-CR40 "Lin, H. K. et al. Functional buffering via cell-specific gene expression promotes tissue homeostasis and cancer robustness. Sci. Rep. 12, 2974 (2022)."). CDS1 and CDS2 are enzymes that are conserved in plants and yeast. They serve to convert phosphatidic acid (PA) into cytidine diphosphate diacylglycerol (CDP-DAG), conceivably representing the bottleneck in phosphatidylinositol (PI) synthesis[41](https://www.nature.com/articles/s41588-025-02221-2#ref-CR41 "Halford, S., Dulai, K. S., Daw, S. C., Fitzgibbon, J. & Hunt, D. M. Isolation and chromosomal localization of two human CDP-diacylglycerol synthase (CDS) genes. Genomics 54, 140–144 (1998)."),[42](https://www.nature.com/articles/s41588-025-02221-2#ref-CR42 "Volta, M. et al. Identification and characterization of CDS2, a mammalian homolog of the Drosophila CDP-diacylglycerol synthase gene. Genomics 55, 68–77 (1999)."),[43](https://www.nature.com/articles/s41588-025-02221-2#ref-CR43 "D’Souza, K., Kim, Y. J., Balla, T. & Epand, R. M. Distinct properties of the two isoforms of CDP-diacylglycerol synthase. Biochemistry 53, 7358–7367 (2014)."). PI constitutes an essential component of cellular membranes and is also used as a critical kinase substrate regulating cell proliferation and survival[44](https://www.nature.com/articles/s41588-025-02221-2#ref-CR44 "Goncalves, M. D., Hopkins, B. D. & Cantley, L. C. Phosphatidylinositol 3-kinase, growth disorders, and cancer. N. Engl. J. Med. 379, 2052–2062 (2018)."),[45](https://www.nature.com/articles/s41588-025-02221-2#ref-CR45 "Cantley, L. C. The phosphoinositide 3-kinase pathway. Science 296, 1655–1657 (2002)."). Aberrations in PI signaling components act as common cancer drivers and are clinically targeted with phosphatidylinositol 3-kinase (PI3K) inhibitors[46](https://www.nature.com/articles/s41588-025-02221-2#ref-CR46 "Vanhaesebroeck, B., Perry, M. W. D., Brown, J. R., André, F. & Okkenhaug, K. PI3K inhibitors are finally coming of age. Nat. Rev. Drug Discov. 20, 741–769 (2021).").

Our analysis confirmed _FAM50A_–_B_, _DDX3X_–_DDX3Y_ and _EIF1AX_–_EIF1AY_ as cancer-specific SLIs, in agreement with previous reports (Extended Data Fig. [1e](https://www.nature.com/articles/s41588-025-02221-2#Fig8))[10](https://www.nature.com/articles/s41588-025-02221-2#ref-CR10 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[25](https://www.nature.com/articles/s41588-025-02221-2#ref-CR25 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 565 (2021)."). For several other SLIs, cancer specificity was previously inferred from genomic data[6](https://www.nature.com/articles/s41588-025-02221-2#ref-CR6 "Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. Nature 568, 551–556 (2019)."),[7](https://www.nature.com/articles/s41588-025-02221-2#ref-CR7 "Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. iScience 13, 488–497 (2019)."),[10](https://www.nature.com/articles/s41588-025-02221-2#ref-CR10 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[11](https://www.nature.com/articles/s41588-025-02221-2#ref-CR11 "Oike, T. et al. A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1. Cancer Res. 73, 5508–5518 (2013)."),[13](https://www.nature.com/articles/s41588-025-02221-2#ref-CR13 "Neggers, J. E. et al. Synthetic lethal interaction between the ESCRT paralog enzymes VPS4A and VPS4B in cancers harboring loss of chromosome 18q or 16q. Cell Rep. 33, 108475 (2020)."), but they did not score in the present study as such, for example, _WRN_–_MLH1_. This is conceivably the result of the relatively low frequency of genomic alterations in _MLH1_ coupled to our focus on transcriptomic rather than genomic data.

In contrast, we noted that _CDS2_ loss was associated with lethality in 19 out of 21 cancer types (Fig. [1e](https://www.nature.com/articles/s41588-025-02221-2#Fig1)). Several of these concerned common cancers, including lung, blood, brain and skin cancers (Extended Data Fig. [1f](https://www.nature.com/articles/s41588-025-02221-2#Fig8)). From here onwards, lethality as a function of _CDS2_ loss will be referred to as Δ_CDS2_ lethality. Patients with _CDS1_\-low cancers showed significantly worse survival compared with patients with _CDS1_\-high cancers (Fig. [1f](https://www.nature.com/articles/s41588-025-02221-2#Fig1)). For these reasons we have focused here on the _CDS1_–_CDS2_ SLI.

To validate and quantify cancer-specific loss of _CDS1_ expression, we compared pan-cancer DepMap expression data and pan-tissue GTEx expression data (Fig. [1g](https://www.nature.com/articles/s41588-025-02221-2#Fig1)). The data were calibrated on housekeeping genes, while control analyses on reference genes were performed to determine reliability. In line with the TCGA data (Fig. [1d](https://www.nature.com/articles/s41588-025-02221-2#Fig1)), we observed common absence, or low expression, of _CDS1_ in cancer cell lines compared with healthy tissue (64% average reduction, _P_ < 0.0001). As expected, given the SLI, low or no _CDS1_ expression strongly correlated with Δ_CDS2_ lethality in cancer cell lines (90% average reduction, _P_ < 0.0001). We also observed that lung, blood, brain and skin cancers have reduced _CDS1_ levels compared with their lineages of origin (Extended Data Fig. [1g](https://www.nature.com/articles/s41588-025-02221-2#Fig8)), suggesting that these cancers may suppress transcription of _CDS1_ during cancer development. For lung cancer, differential CDS1 levels were confirmed using a cohort comprising patient-matched proteomics data from healthy and tumor tissue (Fig. [1h](https://www.nature.com/articles/s41588-025-02221-2#Fig1))[47](https://www.nature.com/articles/s41588-025-02221-2#ref-CR47 "Satpathy, S. et al. A proteogenomic portrait of lung squamous cell carcinoma. Cell 184, 4348–4371.e40 (2021)."). Thus, our in silico analyses predict that the _CDS1–CDS2_ gene pair constitutes a human cancer-associated SLI, specifically in common _CDS1_\-low cancers.

### _CDS1_ and _CDS2_ are synthetic lethal across cancer types

For wet lab validation of the computational predictions, we used a panel of cancer cell lines, which are also in the DepMap (Fig. [2a](https://www.nature.com/articles/s41588-025-02221-2#Fig2)). Their _CDS1_ RNA levels were confirmed by quantitative (q)PCR analysis (Fig. [2b](https://www.nature.com/articles/s41588-025-02221-2#Fig2)). Low-throughput CRISPR perturbations were used to quantify the lethality inferred from genome-wide CRISPR knockout screens. By including fluorescent tracker cells serving as an internal control for each experimental condition, we quantified lethality over extended timeframes, showing both minimal variation and the effect size over time (Fig. [2c](https://www.nature.com/articles/s41588-025-02221-2#Fig2)).

**Fig. 2: _CDS1_ and _CDS2_ are synthetic lethal across cancer types.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02221-2/figures/2)

**a**, Plot depicting synthetic lethality upon CRISPR perturbation of _CDS2_ (Δ_CDS2_ lethality) and _CDS1_ RNA expression for cancer cell lines (DepMap). Cancer cell lines were categorized by _CDS1_ RNA levels and cell lines used for validation are marked. The data distribution is depicted by a Δ_CDS2_ lethality histogram (top) and a _CDS1_ expression histogram (right). _n_ = 913 cancer cell lines. **b**, Plot depicting _CDS1_ RNA expression as measured by RNA sequencing (RNA-seq; DepMap) or qPCR analysis. The correlation (Pearson’s _r_) and associated _P_ value were added. There was an average of two independent experiments with each _n_ = 4 replicates for 8 cancer cell lines. **c**, Diagram depicting the method for quantifying lethality on CRISPR perturbation using fluorescent tracker cells. The mCherry-positive percentage was quantified by flow cytometry. **d**, Graph depicting cumulative lethality 14 d after CRISPR perturbation of _CDS2_ using tracker cells (**c**). _CDS2_ sgRNAs, positive control sgRNA (_RPL19_) and negative control sgRNA (sgControl) were included. Two-way ANOVA followed up by Dunnett’s test was used. _n_ = 3 replicates for 7 cancer cell lines with 2 _CDS2_ sgRNAs; see the 8-d measurement in Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02221-2#Fig9). There is an independent experiment repeat in **e** (three cancer cell lines) and Fig. [3b](https://www.nature.com/articles/s41588-025-02221-2#Fig3) (four cancer cell lines); the latter also includes an additional _CDS1_\-high cancer cell line. **e**, For two _CDS1_\-negative cell lines (NCI-H2030 and SK-MEL-2), _CDS1_ or GFP (control) was ectopically expressed and the 14-d Δ_CDS2_ lethality was determined using tracker cells (**c**). For one _CDS1_\-high cell line _CDS1_ (sg_CDS1_ versus sgControl) was perturbed by CRISPR and the 8-d Δ_CDS2_ lethality was determined using tracker cells (**c**). Two-way ANOVA followed by Tukey’s test was used. _n_ = 3 replicates for 3 cancer cell lines. A431 and NCI-H2030/SK-MEL-2 are separate experiments. **f**, Left: diagram depicting method for quantifying synthetic lethality using ectopic _CDS1_ or GFP expression. The GFP-positive percentage was quantified by flow cytometry. Right: graph depicting quantification of the 14-d synthetic lethality in melanoma. Two-way ANOVA followed by Šidák’s test was used. _n_ = 3 replicates for 4 cancer cell lines; see the 7-d measurement in Extended Data Fig. [2d](https://www.nature.com/articles/s41588-025-02221-2#Fig9). **g**, Graph depicting snapshot of the total protein-normalized level of cleaved caspase-3 in Δ_CDS2_ or control samples of BLM and SK-MEL-2 cell lines, as determined by quantitative western blotting. Two-way ANOVA followed by Šidák’s test was used. _n_ = 3 replicates from independent lentiviral transductions for 2 cancer cell lines. **h**, Left: diagram depicting the in vivo variant of the method for quantifying synthetic lethality. Right: graph depicting quantification of the in vivo synthetic lethality in two melanoma cell-line models 17 d after subcutaneous melanoma tumor inoculation. Tumors were analyzed by flow cytometry. Two-way ANOVA followed by Šidák’s test was used. _n_ = 6 NOD-scid _Il2rγ_\-null mice per group each for 2 cancer cell lines. In **b** and **d**–**h**: \*\*\*_P_ < 0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM5)

On perturbation of _CDS2_ with one of two separate single guide (sg)RNAs, we were able to confirm Δ_CDS2_ lethality in _CDS1_\-low or _CDS1_\-negative cell lines across several cancer types (Fig. [2d](https://www.nature.com/articles/s41588-025-02221-2#Fig2) and Extended Data Fig. [2a–c](https://www.nature.com/articles/s41588-025-02221-2#Fig9)). Furthermore, as predicted, a _CDS1_\-high cell line was Δ_CDS2_ nonlethal. Compared with the DepMap data, we observed strong lethality. Furthermore, Δ_CDS2_ lethality was still evident in cell lines with low levels of _CDS1_ RNA (Fig. [2d](https://www.nature.com/articles/s41588-025-02221-2#Fig2), green).

To validate the _CDS1_ dependency of the Δ_CDS2_ lethality, we either ectopically expressed _CDS1_ in _CDS1_\-negative cancer cell lines or perturbed _CDS1_ in _CDS1_\-proficient cancer cells. When _CDS1_ was introduced into two _CDS1_\-negative cell lines, the Δ_CDS2_ lethality was largely negated. Conversely, when _CDS1_ was perturbed in a _CDS1_\-high cell line, it exhibited increased Δ_CDS2_ lethality (Fig. [2e](https://www.nature.com/articles/s41588-025-02221-2#Fig2)). To validate these observations, we admixed control green fluorescent protein (GFP)-expressing cancer cell lines with _CDS1_\-restored cell lines in an additional panel of four human melanoma models. The results confirmed strong synthetic lethality on perturbation of _CDS2_ in this panel (Fig. [2f](https://www.nature.com/articles/s41588-025-02221-2#Fig2) and Extended Data Fig. [2d](https://www.nature.com/articles/s41588-025-02221-2#Fig9)). Together, these functional experiments confirm that _CDS1_ and _CDS2_ constitute a synthetic lethal pair across a panel of cancer cell lines in vitro.

### Δ_CDS2_ lethality is associated with apoptosis

We hypothesized that Δ_CDS2_ lethality may result in apoptotic cell death in vitro. To investigate this, we collected samples from BLM and SK-MEL-2 cancer cell lines and measured cleaved caspase-3 by quantitative western blotting as a measure of apoptosis (Fig. [2g](https://www.nature.com/articles/s41588-025-02221-2#Fig2)). For quantification, the cleaved caspase-3 signal was compared with the total protein signal in the same capillary. As positive controls, we analyzed cleaved caspase-3 levels on induction of apoptosis by TPCA-1 + tumor necrosis factor (TNF) for BLM cells and staurosporine for SK-MEL-2 cells (Extended Data Fig. [2e](https://www.nature.com/articles/s41588-025-02221-2#Fig9)). Although it may be possible that, in some settings, other types of cell death may additionally occur, we observed a significant increase in cleaved caspase-3 in Δ_CDS2_ cancer cells, indicating that Δ_CDS2_ lethality is associated with tumor cell apoptosis.

### Δ_CDS2_ lethality in vivo

Next, we investigated whether the SLI between _CDS1_ and _CDS2_ observed in silico and in vitro can be recapitulated in vivo. For this purpose, we again admixed control GFP-expressing cancer cell lines with _CDS1_\-restored cell lines. Tumor cells were inoculated into immunodeficient NOD-scid _Il2rγ_\-null mice and analyzed by flow cytometry of the tumors 17 d later. We observed a striking inability of _CDS2_\-perturbed cells to contribute to tumor formation in vivo (Fig. [2h](https://www.nature.com/articles/s41588-025-02221-2#Fig2) and Extended Data Fig. [2f,g](https://www.nature.com/articles/s41588-025-02221-2#Fig9); note that the tumor growth curves are derived from cell mixes including both GFP-expressing and ectopically _CDS1_\-restored cells). Together, these results demonstrate that _CDS1_ and _CDS2_ form a robust synthetic lethal pair in cancer, in silico, in vitro and in vivo.

### No common escape mechanism for Δ_CDS2_ lethality

The computational and in vitro and in vivo functional validation data above demonstrate the broad cancer range and reproducibility of the _CDS1_–_2_ SLI, prompting us to further challenge its robustness. Specifically, the rate-limiting role of the CDS enzymes in PI synthesis led us to investigate whether any cells can rewire their signaling network such that they can escape from this SLI. To investigate this in an unbiased, genome-wide fashion, we performed CRISPR knockout rescue screens in a panel of four _CDS1_\-negative human cancer cell lines and, as a control, one _CDS1_\-high cancer cell line (Fig. [3a](https://www.nature.com/articles/s41588-025-02221-2#Fig3) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). In parallel, cells from the screens were used to track Δ_CDS2_ lethality during the screen. These analyses confirmed Δ_CDS2_ lethality in four _CDS1_\-negative cancer cell lines and extended our data on the lack of Δ_CDS2_ lethality in _CDS1_\-high cancer cell lines to an additional cancer cell line (Fig. [3b](https://www.nature.com/articles/s41588-025-02221-2#Fig3)). In addition, colony formation assays were performed to visualize the lethal effect during the screen (Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02221-2#Fig10); note that for K562 a different readout was used, because it is a suspension cell line).

**Fig. 3: No common escape mechanism for Δ_CDS2_ lethality.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02221-2/figures/3)

**a**, Diagram depicting the method used to screen for Δ_CDS2_ lethality escape mechanisms. One _CDS1_\-high cancer cell line and four _CDS1_\-negative cancer cell lines were screened. **b**, The lethality of the screen cells between 15 d and 25 d. Lethality was quantified using fluorescent tracker cells as depicted in Fig. [2c](https://www.nature.com/articles/s41588-025-02221-2#Fig2). Two-way ANOVA followed by Šídák’s test was used. _n_ = 3 replicates for 5 cancer cell lines. **c**, Graph depicting the screen hits identified in three of four or all four _CDS1_\-negative cancer cell lines. The effect size was calculated using the average Δ_CDS2_ lethality in the lethality assays with screen cells (**b**) and the fold-change enrichment of the second-best guides for each gene. Negative binomial modeling per sgRNA, followed by alpha-Robust Rank Aggregation per gene and Benjamini–Hochberg correction for the screens, was used to determine significance (_P_ < 0.05). _n_ = 1 for 4 cancer cell lines at 200× coverage with 4 sgRNAs per gene and 1,000 sgControls in the CRISPR library. **d**, Graph depicting Pearson’s _r_ between Δ_CDS2_ lethality and other features of cancer cell lines corrected for _CDS1_ expression (DepMap). The correlation between Δ_CDS2_ lethality and _CDS1_ expression was added (red line). The features included were RNA expression, damaging mutations and hotspot mutations. Bonferroni-corrected _P_ values from Pearson correlation tests were used. The gray box indicates the nonsignificant features. _n_ (cancer cell lines) = 1,021. In **b**: \*\*\*_P_ < 0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM6)

Potential escape mechanisms were determined using Δ_CDS2_ lethality quantified with tracker cells. However, the results of the screens (Fig. [3c](https://www.nature.com/articles/s41588-025-02221-2#Fig3) and Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02221-2#Fig10)) indicated no common escape mechanism for Δ_CDS2_ lethality. For example, the screen performed in SK-MEL-2 cells yielded no significant enrichment even after an additional 14 d (32 d in total; Extended Data Fig. [3c](https://www.nature.com/articles/s41588-025-02221-2#Fig10)), while the other screens yielded only some cell-line-specific rescue (Extended Data Fig. [3d–f](https://www.nature.com/articles/s41588-025-02221-2#Fig10)). Analysis of the dropout of essential genes confirmed the high quality of the screens (Extended Data Fig. [3g](https://www.nature.com/articles/s41588-025-02221-2#Fig10)). To comprehensively cover all altered gene expression and DNA mutations, we also analyzed the DepMap for potential escape mechanisms (Fig. [3d](https://www.nature.com/articles/s41588-025-02221-2#Fig3)). The results confirm that only _CDS1_ expression commonly affects Δ_CDS2_ lethality. We expanded our Δ_CDS2_ lethality validation to also cover outliers in the DepMap data (Extended Data Fig. [3h](https://www.nature.com/articles/s41588-025-02221-2#Fig10)): even for these we observed strong Δ_CDS2_ lethality in cultured cancer cells. Although we cannot fully exclude the existence of specific escape mechanisms, together these findings suggest that, at least under the conditions tested in the present study, there are no common escape mechanisms for the combined loss of _CDS1_ and _CDS2_, which is in agreement with the idea that CDS1 and CDS2 together serve as a bottleneck for PI synthesis.

### Mesenchymal-like cancers depend on _CDS2_ for PI synthesis

To understand which cancer types show reduced _CDS1_ expression and, hence, Δ_CDS2_ lethality, we characterized _CDS1_\-low cancers using publicly available data (Fig. [4a](https://www.nature.com/articles/s41588-025-02221-2#Fig4)). First, we noted that _CDS1_\-high cancer cells express high levels of the epithelial marker gene E-cadherin (_CDH1_), whereas _CDS1_\-low cancer cell lines express mesenchymal markers like _ZEB1_, _ZEB2_ and vimentin (_VIM_) (Fig. [4b](https://www.nature.com/articles/s41588-025-02221-2#Fig4) and Extended Data Fig. [4a,b](https://www.nature.com/articles/s41588-025-02221-2#Fig11))[48](https://www.nature.com/articles/s41588-025-02221-2#ref-CR48 "Yang, J. & Weinberg, R. A. Epithelial–mesenchymal transition: at the crossroads of development and tumor metastasis. Dev. Cell 14, 818–829 (2008)."),[49](https://www.nature.com/articles/s41588-025-02221-2#ref-CR49 "Mak, M. P. et al. A patient-derived, pan-cancer EMT signature identifies global molecular alterations and immune target enrichment following epithelial-to-mesenchymal transition. Clin. Cancer Res. 22, 609–620 (2016)."). Mechanistically, this is in agreement with the notion that _CDS1_ expression is suppressed in mesenchymal cancers by the transcription factor ZEB1, previously reported to bind the _CDS1_ locus and suppress its expression[50](https://www.nature.com/articles/s41588-025-02221-2#ref-CR50 "Gemmill, R. M. et al. ZEB1-responsive genes in non-small cell lung cancer. Cancer Lett. 300, 66–78 (2011)."),[51](https://www.nature.com/articles/s41588-025-02221-2#ref-CR51 "Feingold, E. A. et al. The ENCODE (ENCyclopedia of DNA Elements) project. Science 306, 636–640 (2004)."). Therefore, we measured the effect of sgRNAs for _ZEB1_ on _CDS1_ expression by qPCR and computationally scored the correlation between the expression of _ZEB1_ and _CDS1_ or _CDH1_ across cancer types in the DepMap. Both results support the notion that _ZEB1_ suppresses _CDS1_ expression in mesenchymal-like cancers (Fig. [4c,d](https://www.nature.com/articles/s41588-025-02221-2#Fig4)). Besides this major regulatory mechanism, we also observed that a rare subset of the blood lineage cancer cell lines exhibits methylation of the _CDS1_ promoter (Extended Data Fig. [4c](https://www.nature.com/articles/s41588-025-02221-2#Fig11)). _CDS1_ methylation did not correlate with _ZEB1_ expression. Overall, these findings indicate that the suppression of _CDS1_ expression in mesenchymal-like cancers by _ZEB1_ results in their strong dependency on _CDS2_ (Fig. [4e](https://www.nature.com/articles/s41588-025-02221-2#Fig4)).

**Fig. 4: Mesenchymal-like cancers depend on _CDS2_ for PI synthesis.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02221-2/figures/4)

**a**, Method for phenotyping cancer cell lines that are Δ_CDS2_ lethal and _CDS1_\-low compared to those that are Δ_CDS2_ nonlethal and _CDS1_\-high (DepMap) (illustrative). **b**, Genome-wide calculated Pearson correlations using the method depicted in **a** (DepMap). Representative genes enriched on either side are marked. Bonferroni-corrected _P_ values from Pearson correlation tests were used to determine the significance cut-off. _n_ = 913 cancer cell lines. **c**, Plot depicting _CDS1_ RNA expression on _ZEB1_ CRISPR perturbation as measured by qPCR analysis. The dashed line indicates the baseline of no change (1-fold change). Two-way ANOVA followed by Dunnett’s test was used for each cancer cell line. _n_ = 4 replicates for 2 cancer cell lines in 2 independent experiments with each 2 sg_ZEB1_ and 1 sgControl. **d**, For each cancer lineage, the percentage of _ZEB1_\-high cancer cell lines (DepMap) was plotted against the percentage of _CDS1-_high or _CDH1_\-high cancer cell lines. A linear regression, the associated Pearson correlation and associated _P_ values were added. _n_ = 906 cancer cell lines. **e**, Diagram depicting the characteristics of cancer cell lines with and without the _CDS1–CDS2_ SLI. _CDH1_ and _VIM_ are representative epithelial and mesenchymal markers, respectively. **f**, Diagram depicting the core CDS pathway. The lethality on perturbation (color) and lineage-specific expression patterns (gray boxes on the left) of each enzyme in the pathway are indicated (DepMap). Gene dependency is represented by CERES. _n_ = 913 cancer cell lines. **g**, Correlation between Δ_CDS2_ lethality and Δ_CDIPT_ lethality, Δ_PIK3CA_ lethality, PIK3CA inhibitor lethality (alpelisib) or the pan-PI3K inhibitor lethality (copanlisib) in solid cancer cell lines (DepMap). Bonferroni-corrected _P_ values from Pearson correlation tests were used. _n_ (cancer cell lines) = 865 (_CDIPT_ and _PIK3CA_), 428 (copanlisib) and 441 (alpelisib). **h**, Rescue of synthetic lethality on supplementation with PI or phosphatidylcholine (PC, control). The method to quantify synthetic lethality was introduced in Fig. [2f](https://www.nature.com/articles/s41588-025-02221-2#Fig2). PI and PC were complexed with lipid-free bovine serum albumin (BSA). Treatment was with 0.25 mM lipid for 7 d, started 7 d after transduction. Two-way ANOVA followed by Dunnett’s test was used. _n_ = 3 replicates for 2 cancer cell lines. In **c**, **d**, **g** and **h**: \*\*\*_P_ < 0.01, \*\*\*_P_ < 0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM7)

Gene set enrichment analysis (GSEA) indicated that _CDS1_\-low cancers are enriched for the Hallmark epithelial–mesenchymal transition (EMT) gene set (_P_ = 0.0001 DepMap, _P_ = 0.009 TCGA; Extended Data Fig. [4d](https://www.nature.com/articles/s41588-025-02221-2#Fig11)). Mesenchymal-transitioned cancers are common, and often highly metastatic and therapy-refractory[48](https://www.nature.com/articles/s41588-025-02221-2#ref-CR48 "Yang, J. & Weinberg, R. A. Epithelial–mesenchymal transition: at the crossroads of development and tumor metastasis. Dev. Cell 14, 818–829 (2008)."),[52](https://www.nature.com/articles/s41588-025-02221-2#ref-CR52 "Hanahan, D. & Weinberg, R. A. Hallmarks of cancer: the next generation. Cell 144, 646–674 (2011)."),[53](https://www.nature.com/articles/s41588-025-02221-2#ref-CR53 "Fischer, K. R. et al. Epithelial-to-mesenchymal transition is not required for lung metastasis but contributes to chemoresistance. Nature 527, 472–476 (2015)."),[54](https://www.nature.com/articles/s41588-025-02221-2#ref-CR54 "Mani, S. A. et al. The epithelial–mesenchymal transition generates cells with properties of stem cells. Cell 133, 704–715 (2008)."),[55](https://www.nature.com/articles/s41588-025-02221-2#ref-CR55 "Van Espen, B. et al. RNF185 control of COL3A1 expression limits prostate cancer migration and metastatic potential. Mol. Cancer Res. 21, 41–54 (2024)."). These findings are in agreement with our observations that low _CDS1_ expression is common, more frequent in cancers compared with their healthy tissue of origin and associated with worse survival (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02221-2#Fig1) and Extended Data Fig. [1g](https://www.nature.com/articles/s41588-025-02221-2#Fig8)). Furthermore, when expanding this comparison with healthy tissue to 16 cancer types covering _CDS1_ and _CDH1_, we observed that both genes are expressed at low levels in mesenchymal-like cancer cell lines (Extended Data Fig. [4e](https://www.nature.com/articles/s41588-025-02221-2#Fig11)). Together, these results suggest that suppression of _CDS1_ expression is an integral element of EMT in cancer.

Next, we zoomed in on the pathway in which CDS1 and CDS2 are involved, as has been defined by previous studies (Fig. [4f](https://www.nature.com/articles/s41588-025-02221-2#Fig4))[41](https://www.nature.com/articles/s41588-025-02221-2#ref-CR41 "Halford, S., Dulai, K. S., Daw, S. C., Fitzgibbon, J. & Hunt, D. M. Isolation and chromosomal localization of two human CDP-diacylglycerol synthase (CDS) genes. Genomics 54, 140–144 (1998)."),[42](https://www.nature.com/articles/s41588-025-02221-2#ref-CR42 "Volta, M. et al. Identification and characterization of CDS2, a mammalian homolog of the Drosophila CDP-diacylglycerol synthase gene. Genomics 55, 68–77 (1999)."),[43](https://www.nature.com/articles/s41588-025-02221-2#ref-CR43 "D’Souza, K., Kim, Y. J., Balla, T. & Epand, R. M. Distinct properties of the two isoforms of CDP-diacylglycerol synthase. Biochemistry 53, 7358–7367 (2014)."),[56](https://www.nature.com/articles/s41588-025-02221-2#ref-CR56 "Bateman, A. et al. UniProt: the Universal Protein Knowledgebase in 2023. Nucleic Acids Res. 51, D523–D531 (2023)."). We incorporated expression and dependency data from the DepMap for the enzymes involved. In line with the requirement for CDS1 or CDS2, the next enzyme in the pathway, cytidine diphosphate diacylglycerol synthase inositol-3-phosphatidyltransferase (CDIPT), is essential for survival. In addition, _CDS2_ and _CDIPT_ are significantly co-dependent in DepMap cancer cell lines, which is indicative of a similar mechanism of dependency (Fig. [4g](https://www.nature.com/articles/s41588-025-02221-2#Fig4))[28](https://www.nature.com/articles/s41588-025-02221-2#ref-CR28 "Tsherniak, A. et al. Defining a cancer dependency map. Cell 170, 564–576.e16 (2017)."),[57](https://www.nature.com/articles/s41588-025-02221-2#ref-CR57 "Wainberg, M. et al. A genome-wide atlas of co-essential modules assigns function to uncharacterized genes. Nat. Genet. 53, 638–649 (2021)."). Accordingly, when supplementing the cell culture medium with exogenous PI, we observed a significant rescue of cell death, suggesting that Δ_CDS2_ lethality is due, at least in part, to insufficient availability of PI (Fig. [4h](https://www.nature.com/articles/s41588-025-02221-2#Fig4) and Extended Data Fig. [4f](https://www.nature.com/articles/s41588-025-02221-2#Fig11)).

CDS2 acts upstream of the PI3K signaling module, regulating growth and survival. Unexpectedly, DepMap analysis revealed that Δ_CDS2_ lethality is not accompanied by lethality with either genetic loss of _PIK3CA_ or pharmacological PIK3CA inhibition (alpelisib). Instead, we observed a strong anticorrelation between Δ_CDS2_ lethality and Δ_PIK3CA_ lethality (Fig. [4g](https://www.nature.com/articles/s41588-025-02221-2#Fig4)). This anticorrelation was independent of PI3K isoforms, as judged by pan-PI3K inhibitor responses. These results demonstrate an essential role of _CDS_ for PI synthesis in mesenchymal cancers. Furthermore, they unexpectedly point to a contribution of _CDS1_ and _CDS2_ to survival signaling independent of the classic PI3K pathway.

### Cholesterol production is induced on _CDS1+CDS2_ ablation

To biochemically define the effects of _CDS2_ perturbation in _CDS1_\-negative cancer cells, we performed multiomic analyses in a panel of four _CDS1_\-negative cancer cell lines and, as a control, one _CDS1_\-high cancer cell line (Fig. [5a](https://www.nature.com/articles/s41588-025-02221-2#Fig5)). Proteomics was performed to identify deregulated cellular processes on CDS2 loss. In addition, lipidomics was performed to quantify buildup or depletion of lipids, specifically those in the CDS pathway.

**Fig. 5: Cholesterol production is induced on _CDS1+CDS2_ ablation.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02221-2/figures/5)

**a**, Diagram depicting the samples generated for quantifying proteins and lipids (Fig. [6](https://www.nature.com/articles/s41588-025-02221-2#Fig6)). One _CDS1_\-high cancer cell line and four _CDS1_\-negative cancer cell lines were analyzed. **b**, Average Δ_CDS2_ protein changes for the four _CDS1_\-negative cancer cell lines (_y_ axis) plotted against the Δ_CDS2_ protein changes in the _CDS1_\-high cancer cell line (_x_ axis). The significance was determined using the Student’s _t_\-test. _n_ = 4 replicates for 5 cancer cell lines. An independent experiment repeat using a second independent instrument is shown in Extended Data Fig. [5c](https://www.nature.com/articles/s41588-025-02221-2#Fig12) for SK-MEL-2. **c**, CDS1 protein levels plotted against CDS2 protein levels for a panel of ten cancer cell lines. _CDS1_ expression from public RNA-seq was added as in Fig. [2b](https://www.nature.com/articles/s41588-025-02221-2#Fig2) (DepMap) with additional data for BLM and SK-MEL-147 (accession no. [SRP132830](https://www.ncbi.nlm.nih.gov/sra/?term=SRP132830)). For each axis, the relative abundance to the highest cancer cell line included was plotted. Data were normalized to the median levels of calibration proteins and the mean of the experiments was used. CDS1 was detected in all MCF7 and A431 replicates, but no other replicates. _n_ = 3 and 4 replicates from 2 independent experiments.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM8)

The proteomics analysis allowed for quantification of ~7,000 proteins in each cell line (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). As expected, the levels of CDS2 protein had dropped tenfold in all five cell lines on _CDS2_ CRISPR perturbation (Fig. [5b](https://www.nature.com/articles/s41588-025-02221-2#Fig5) and Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02221-2#Fig12)). In addition, large groups of proteins were commonly and significantly down- or upregulated in _CDS1_\-low cancer cell lines, suggesting an orchestrated response (Fig. [5b](https://www.nature.com/articles/s41588-025-02221-2#Fig5) and Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02221-2#Fig12)). Gene ontology (GO) term enrichment analysis of these proteins revealed major induction of cholesterol import and production on Δ_CDS2_ lethality (Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02221-2#Fig12)), which was a robust effect across cancer cell lines and different experiments and proteomics analyses (Fig. [5b](https://www.nature.com/articles/s41588-025-02221-2#Fig5) and Extended Data Fig. [5a,c](https://www.nature.com/articles/s41588-025-02221-2#Fig12)). Overall, we observed that _CDS2_ perturbation in _CDS1_\-negative cancer cells results in upregulation of cholesterol proteins, including HMGCS1, SQLE, DHCR24, LDLR and SCARB1.

We expanded the proteomics to cover also CDS1 protein by using a different method ([Methods](https://www.nature.com/articles/s41588-025-02221-2#Sec13)). As expected, on CRISPR perturbation of _CDS1_, the level of CDS1 protein dropped by 97% (Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02221-2#Fig12)). We observed that CDS1 protein was clearly detected in all replicates in _CDS1_ RNA high cancer cell lines. However, it was not detectable in the _CDS1_ RNA-low or -negative cancer cell lines (log2(transcripts per kilobase per million mapped reads (TPM)) < 3) in a panel of 10 cancer cell lines (Fig. [5c](https://www.nature.com/articles/s41588-025-02221-2#Fig5)).

### Lipid homeostasis requires either _CDS1_ or _CDS2_ expression

Lipidomic analysis allowed for quantification of the major lipid classes in the cells, with phosphorylated PI (PIP) serving as the major signaling molecule[44](https://www.nature.com/articles/s41588-025-02221-2#ref-CR44 "Goncalves, M. D., Hopkins, B. D. & Cantley, L. C. Phosphatidylinositol 3-kinase, growth disorders, and cancer. N. Engl. J. Med. 379, 2052–2062 (2018).") (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02221-2#Fig13) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). Major changes as a function of _CDS1_ expression were robustly detected on CDS2 loss (Fig. [6a](https://www.nature.com/articles/s41588-025-02221-2#Fig6) and Extended Data Fig. [6a–c](https://www.nature.com/articles/s41588-025-02221-2#Fig13); note that the stars indicate the number of _CDS1_\-negative cancer cell lines with _P_ < 0.01). For example, cholesterol esters (CEs) and triglycerides (TAGs) were massively upregulated. Furthermore, we observed strong buildup of CDS2 substrates and depletion of the downstream product PI.

**Fig. 6: Lipid homeostasis requires either _CDS1_ or _CDS2_ expression.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02221-2/figures/6)

**a**, Five Δ_CDS2_ lipid class changes presented (full results in Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02221-2#Fig13)). The asterisks indicate the number of _CDS1_\-negative cell lines with at least _P_ < 0.01 significant lipid alterations compared with the _CDS1_\-high cell line. Two-way ANOVA followed by Šidák’s test was used. _n_ = 4 replicates for 5 cancer cell lines. **b**, Cell areas staining positive for the lipid compartment dye BODIPY in live-cell imaging. Two-way ANOVA followed by Šidák’s test was used. _n_ = 4 replicates for 2 cancer cell lines; see **d** for an independent experiment for one cancer cell line. **c**, Representative combined light and transmission electron microscopy image of an sgControl and an sg_CDS2_ SK-MEL-2 cell stained for lipids (Nile Red) and DNA (Hoechst). _n_ = 1 for 1 cancer cell line. Two independent light microscopy experiments were performed for quantification (**b** and **d**). **d**, Lipid droplet quantification on supplementation with 0.25 mM PI or PA in SK-MEL-2. PI and PA were complexed with lipid-free BSA. Treatment was started at 8 d and refreshed at 11 d. Two-way ANOVA followed by Šidák’s test was used. _n_ = 4 replicates for 1 cancer cell line. See **b** an for independent experiment repeat without lipid treatments, also covering one additional cancer cell line. **e**, Diagram depicting lipid compartments and the CDS2 pathway. The Δ_CDS2_ lipid class changes in the four _CDS1_\-negative cell lines are marked by arrows. DGKs, diacylglycerol kinases. In **a** the significance was indicated by one asterisk for each cell line with _P_ < 0.01. In **b** and **d**: \*\*\*_P_ < 0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM9)

In line with the buildup of CEs and TAGs, _CDS2_\-perturbed cells formed large lipid droplets[58](https://www.nature.com/articles/s41588-025-02221-2#ref-CR58 "Olzmann, J. A. & Carvalho, P. Dynamics and functions of lipid droplets. Nat. Rev. Mol. Cell Biol. 20, 137–155 (2019)."), which were visible by both light and electron microscopy, staining positively for the lipid dyes BODIPY (boron dipyrromethene) and Nile Red. Quantification of BODIPY-stained live cells revealed that on average 3% of the imaged _CDS2_\-knockout cells were made up of lipid (Fig. [6b](https://www.nature.com/articles/s41588-025-02221-2#Fig6)). Combined light and electron microscopy of fixed cells showed these lipid droplets in more detail (Fig. [6c](https://www.nature.com/articles/s41588-025-02221-2#Fig6)). Supplementation with the downstream CDS product PI significantly reduced the percentage area of lipid droplets in _CDS2_\-perturbed cells (Fig. [6d](https://www.nature.com/articles/s41588-025-02221-2#Fig6)). Combined with the lack of lipid droplets with the CDS substrate PA in control cells, this suggests that a shortage of PI drives the formation of most of the droplets. Together, these findings again corroborate that CDS signaling represents a biochemical bottleneck to support lipid homeostasis, demonstrating that lethality upon _CDS2_ perturbation in _CDS1_\-low cell lines is accompanied by major changes in lipid metabolism and induction of the cholesterol pathway (Fig. [6e](https://www.nature.com/articles/s41588-025-02221-2#Fig6)).

### CDS2 dosage and catalytic activity govern synthetic lethality

Given that our in silico, in vitro and in vivo results demonstrate that _CDS1_ and _CDS2_ form a robust synthetic lethal pair in mesenchymal-like cancers, we set out to investigate any favorable characteristics for pharmacological inhibition. First, we investigated the dose dependency of _CDS1–CDS2_ synthetic lethality. Using DepMap data, we observed that the relationship between _CDS1_ expression and Δ_CDS2_ lethality closely follows a dose–response-like relationship (Fig. [7a](https://www.nature.com/articles/s41588-025-02221-2#Fig7)). Of cancer cell lines, 42% have at least half the maximum synthetic lethality. This is in line with our data linking low _CDS1_ expression to mesenchymal-like cancers and the notion that roughly half of all cancers are mesenchymal like (Fig. [4a–d](https://www.nature.com/articles/s41588-025-02221-2#Fig4) and Extended Data Fig. [4a,b,d,e](https://www.nature.com/articles/s41588-025-02221-2#Fig11))[59](https://www.nature.com/articles/s41588-025-02221-2#ref-CR59 "Hanahan, D. & Weinberg, R. A. The hallmarks of cancer. Cell 100, 57–70 (2000)."),[60](https://www.nature.com/articles/s41588-025-02221-2#ref-CR60 "Christofori, G. & Semb, H. The role of the cell-adhesion molecule E-cadherin as a tumour-suppressor gene. Trends Biochem. Sci. 24, 73–76 (1999)."). We used low and high doses of _CDS2_ short hairpin (sh)RNA to generate two levels of _CDS2_ knockdown (Extended Data Fig. [7a](https://www.nature.com/articles/s41588-025-02221-2#Fig14)), which confirmed the dose-dependent lethality (Fig. [7b](https://www.nature.com/articles/s41588-025-02221-2#Fig7)). Together, these data support a dose-dependent essentiality of CDS signaling for survival in cancer cell lines.

**Fig. 7: CDS2 dosage and catalytic activity govern synthetic lethality.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02221-2/figures/7)

**a**, The relationship between _CDS1_ expression and Δ_CDS2_ lethality. Deciles for cancer cell line _CDS1_ expression were plotted against Δ_CDS2_ lethality using DepMap data, with variation in Δ_CDS2_ lethality indicated by the interquartile range. A sigmoid curve was fitted through the _CDS1_ expression deciles with _r_ = 0.998. Hill’s equation yielded a log2(TPM + 0.0625) of 1.273 as the level of _CDS1_ expression, with 50% of maximum synthetic lethality among the cancer cell line collection (dashed line). For the totals above and below this level, numbers and percentages are shown on top, together with a histogram of the cancer cell line distribution. _n_ = 913 cancer cell lines (DepMap). **b**, Graph depicting quantification of the 10-d synthetic lethality in BLM using the method for quantifying synthetic lethality with ectopic _CDS1_ or GFP expression described in Fig. [2f](https://www.nature.com/articles/s41588-025-02221-2#Fig2). Two-way ANOVA followed by Šidák’s test was used. _n_ = 3 replicates for 1 cancer cell line for 3 sh_CDS2_. **c**, Tumor growth after subcutaneous inoculation of control and _CDS2_\-perturbed BLM melanoma cells. Two-way ANOVA was used (_n_ = 9 NOD-scid _Il2rγ_\-null mice per group for 1 cancer cell line). **d**, For two _CDS1_\-negative melanoma cancer cell lines _CDS1_, _GFP_ (control), _CDS2_–GFP or _CDS2_–GFP Asp384Ala was ectopically expressed and the 14-d Δ_CDS2_ lethality was determined using tracker cells (Fig. [2c](https://www.nature.com/articles/s41588-025-02221-2#Fig2)). Two-way ANOVA followed by Šidák’s test was used. _n_ = 3 replicates for 2 cancer cell lines. The _CDS2_–GFP and _CDS2_–GFP Asp384Ala cell line variants were each generated in duplicate via independent transductions (no. 1, 2). Δ_CDS2_ lethality in _CDS1_ or _GFP_ variants was replicated in one independent experiment for SK-MEL-2 (Fig. [2e](https://www.nature.com/articles/s41588-025-02221-2#Fig2)) and two independent experiments for BLM (Fig. [2f](https://www.nature.com/articles/s41588-025-02221-2#Fig2): in vitro; Fig. [2h](https://www.nature.com/articles/s41588-025-02221-2#Fig2): in vivo). **e**, A diagram summarizing the mechanistic aspects of _CDS1–CDS2_ synthetic lethality. On the left, upregulated cholesterol proteins are listed. On the right, buildup of the substrate lipids (PA, DAG) and of the lipid droplet lipids (CEs, TAGs) and depletion of the product PI is indicated. In **b**–**d**: \*\*\*_P_ < 0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM10)

Next, we investigated whether the observed in vivo SLI between _CDS1_ and _CDS2_ can be recapitulated without admixing the cancer cells. For this purpose, tumor cells were inoculated into immunodeficient NOD-scid _Il2rγ_\-null mice. We observed an identical time to establishment of palpable tumors in mice injected with control or C_DS2-_perturbed tumor cells (Extended Data Fig. [7b](https://www.nature.com/articles/s41588-025-02221-2#Fig14)). However, robust Δ_CDS2_ lethality manifested in time, again demonstrating the SLI in vivo, now with a homogeneous population of _CDS2_\-ablated tumor cells (Fig. [7c](https://www.nature.com/articles/s41588-025-02221-2#Fig7)).

In line with the idea that CDS1 and CDS2 form a biochemical bottleneck in lipid homeostasis, we hypothesized that loss of the catalytic activity of CDS2 drives Δ_CDS2_ lethality. To study this, we designed a catalytically inactive variant of CDS2 based on TmCdsA from _Thermotoga_ _maritima_, a thermophilic bacterium, for which this was previously defined (Extended Data Fig. [7c,d](https://www.nature.com/articles/s41588-025-02221-2#Fig14))[61](https://www.nature.com/articles/s41588-025-02221-2#ref-CR61 "Liu, X., Yin, Y., Wu, J. & Liu, Z. Structure and mechanism of an intramembrane liponucleotide synthetase central for phospholipid biosynthesis. Nat. Commun. 5, 4244 (2014)."),[62](https://www.nature.com/articles/s41588-025-02221-2#ref-CR62 "Yariv, B. et al. Using evolutionary data to make sense of macromolecules with a ‘face-lifted’ ConSurf. Protein Sci. 32, e4582 (2023)."),[63](https://www.nature.com/articles/s41588-025-02221-2#ref-CR63 "Hekkelman, M. L., de Vries, I., Joosten, R. P. & Perrakis, A. AlphaFill: enriching AlphaFold models with ligands and cofactors. Nat. Methods 20, 205–213 (2022)."). As expected, add-back of either _CDS1_ or catalytically proficient _CDS2_ rescued Δ_CDS2_ lethality compared with the control (Fig. [7d](https://www.nature.com/articles/s41588-025-02221-2#Fig7)). However, Δ_CDS2_ lethality could not be rescued by add-back of catalytically inactive _CDS2_.

Discussion
----------

Genomic alterations in cancer can serve as a mechanistic basis for SLIs[3](https://www.nature.com/articles/s41588-025-02221-2#ref-CR3 "Bryant, H. E. et al. Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase. Nature 434, 913–917 (2005)."),[4](https://www.nature.com/articles/s41588-025-02221-2#ref-CR4 "Farmer, H. et al. Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy. Nature 434, 917–921 (2005)."),[6](https://www.nature.com/articles/s41588-025-02221-2#ref-CR6 "Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. Nature 568, 551–556 (2019)."),[7](https://www.nature.com/articles/s41588-025-02221-2#ref-CR7 "Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. iScience 13, 488–497 (2019)."),[8](https://www.nature.com/articles/s41588-025-02221-2#ref-CR8 "Frezza, C. et al. Haem oxygenase is synthetically lethal with the tumour suppressor fumarate hydratase. Nature 477, 225–228 (2011)."),[9](https://www.nature.com/articles/s41588-025-02221-2#ref-CR9 "Feng, X. et al. Genome-wide CRISPR screens using isogenic cells reveal vulnerabilities conferred by loss of tumor suppressors. Sci. Adv. 8, eabm6638 (2022)."),[10](https://www.nature.com/articles/s41588-025-02221-2#ref-CR10 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[11](https://www.nature.com/articles/s41588-025-02221-2#ref-CR11 "Oike, T. et al. A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1. Cancer Res. 73, 5508–5518 (2013)."),[12](https://www.nature.com/articles/s41588-025-02221-2#ref-CR12 "Hoffman, G. R. et al. Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers. Proc. Natl Acad. Sci. USA 111, 3128–3133 (2014)."),[13](https://www.nature.com/articles/s41588-025-02221-2#ref-CR13 "Neggers, J. E. et al. Synthetic lethal interaction between the ESCRT paralog enzymes VPS4A and VPS4B in cancers harboring loss of chromosome 18q or 16q. Cell Rep. 33, 108475 (2020)."),[14](https://www.nature.com/articles/s41588-025-02221-2#ref-CR14 "Jerby-Arnon, L. et al. Predicting cancer-specific vulnerability via data-driven detection of synthetic lethality. Cell 158, 1199–1209 (2014)."),[15](https://www.nature.com/articles/s41588-025-02221-2#ref-CR15 "Muller, F. L. et al. Passenger deletions generate therapeutic vulnerabilities in cancer. Nature 488, 337–342 (2012)."),[16](https://www.nature.com/articles/s41588-025-02221-2#ref-CR16 "Zhao, D. et al. Synthetic essentiality of chromatin remodelling factor CHD1 in PTEN-deficient cancer. Nature 542, 484–488 (2017)."),[17](https://www.nature.com/articles/s41588-025-02221-2#ref-CR17 "Lee, R. et al. Synthetic essentiality of tryptophan 2,3-dioxygenase 2 in APC-mutated colorectal cancer. Cancer Discov. 12, 1702–1717 (2022)."),[18](https://www.nature.com/articles/s41588-025-02221-2#ref-CR18 "Mavrakis, K. J. et al. Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5. Science 351, 1208–1213 (2016)."),[19](https://www.nature.com/articles/s41588-025-02221-2#ref-CR19 "Kryukov, G. V. et al. MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells. Science 351, 1214–1218 (2016)."),[31](https://www.nature.com/articles/s41588-025-02221-2#ref-CR31 "Helming, K. C. et al. ARID1B is a specific vulnerability in ARID1A-mutant cancers. Nat. Med. 20, 251–254 (2014)."). Cancers also show distinct gene expression patterns compared with healthy tissue, prompting us to explore the cancer SLI space based on differential gene expression in cancer and healthy tissues. Taking advantage of extensive publicly available cancer datasets, we identified by computational mining that CDS2 serves as a synthetic lethal target in mesenchymal-like cancers. Our bioinformatic pipeline revealed a potential therapeutic index because of differential _CDS1_ expression in cancer cells (low) and normal cells (high). We confirmed that _CDS1_ and _CDS2_ constitute a functional SLI across many cancer types, particularly mesenchymal-like cancers, representing roughly half of all cancers[59](https://www.nature.com/articles/s41588-025-02221-2#ref-CR59 "Hanahan, D. & Weinberg, R. A. The hallmarks of cancer. Cell 100, 57–70 (2000)."),[60](https://www.nature.com/articles/s41588-025-02221-2#ref-CR60 "Christofori, G. & Semb, H. The role of the cell-adhesion molecule E-cadherin as a tumour-suppressor gene. Trends Biochem. Sci. 24, 73–76 (1999)."), including both EMT cancers and cancers with a mesenchymal origin. The SLI is mechanistically explained by the fact that the combined loss of _CDS1_ and _CDS2_ is incompatible with cell survival, which, in combination with the low or absent expression of _CDS1_ in mesenchymal-like cancer cells, results in Δ_CDS2_ lethality (Fig. [7e](https://www.nature.com/articles/s41588-025-02221-2#Fig7)). Our data are in agreement with results by Chan et al. (reported in the companion paper to this manuscript[64](https://www.nature.com/articles/s41588-025-02221-2#ref-CR64 "Chan, P. Y. et al. The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability across multiple tumor types. Nat. Genet. 
https://doi.org/10.1038/s41588-025-02222-1
(2025).")) who reported that _CDS1_ and _CDS2_ are a synthetic lethal gene pair in uveal melanoma.

A series of CRISPR knockout rescue screens not only confirmed the synthetic lethal relationship between _CDS1_ and _CDS2_, but also that they form a bottleneck in a pathway required for PI synthesis that apparently is essential for (cancer) cell viability. We found no common escape mechanism by CRISPR screening and computational analyses. However, ectopic _CDS1_ introduction rescues Δ_CDS2_ lethality and is compatible with cancer cell survival. Taken together, these observations suggest that it may not be straightforward for cancer cells to restore _CDS1_ expression upon inactivation of one of its regulators to rescue cell death on _CDS2_ inactivation. An example is the mesenchymal transcription factor ZEB1, depletion of which induces _CDS1_, but also reduces cancer cell survival[33](https://www.nature.com/articles/s41588-025-02221-2#ref-CR33 "Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. Nat. Genet. 49, 1779–1784 (2017)."),[50](https://www.nature.com/articles/s41588-025-02221-2#ref-CR50 "Gemmill, R. M. et al. ZEB1-responsive genes in non-small cell lung cancer. Cancer Lett. 300, 66–78 (2011)."),[51](https://www.nature.com/articles/s41588-025-02221-2#ref-CR51 "Feingold, E. A. et al. The ENCODE (ENCyclopedia of DNA Elements) project. Science 306, 636–640 (2004)."). Mechanistically, we show that Δ_CDS2_ lethality is associated with cancer cell apoptosis, which can be rescued at least in part by supplementing exogenous PI. The latter is consistent with a requirement for CDS2 to produce PI. Our data also strongly suggest that, unexpectedly, PI synthesis is required for cell survival independent of classic PI3K-dependent survival signaling, because cell lines that depend on _CDS1_ or _CDS2_ for survival did not require PI3K. Our data imply that CDS2 inhibition may be explored to eliminate particularly mesenchymal-like cancers, which are common, highly metastatic and relatively resistant to therapy[48](https://www.nature.com/articles/s41588-025-02221-2#ref-CR48 "Yang, J. & Weinberg, R. A. Epithelial–mesenchymal transition: at the crossroads of development and tumor metastasis. Dev. Cell 14, 818–829 (2008)."),[52](https://www.nature.com/articles/s41588-025-02221-2#ref-CR52 "Hanahan, D. & Weinberg, R. A. Hallmarks of cancer: the next generation. Cell 144, 646–674 (2011)."),[53](https://www.nature.com/articles/s41588-025-02221-2#ref-CR53 "Fischer, K. R. et al. Epithelial-to-mesenchymal transition is not required for lung metastasis but contributes to chemoresistance. Nature 527, 472–476 (2015)."),[54](https://www.nature.com/articles/s41588-025-02221-2#ref-CR54 "Mani, S. A. et al. The epithelial–mesenchymal transition generates cells with properties of stem cells. Cell 133, 704–715 (2008)."),[55](https://www.nature.com/articles/s41588-025-02221-2#ref-CR55 "Van Espen, B. et al. RNF185 control of COL3A1 expression limits prostate cancer migration and metastatic potential. Mol. Cancer Res. 21, 41–54 (2024).").

To date, no inhibitor is available for CDS2 targeting. However, as our data demonstrate that its enzymatic activity is involved, pharmacological targeting may be possible. For example, for PI3K and diglyceride acyltransferase (DGAT), enzymes acting in the same pathway, inhibitors exist, while for PI3K inhibition, isoform-specific inhibitors are also available[46](https://www.nature.com/articles/s41588-025-02221-2#ref-CR46 "Vanhaesebroeck, B., Perry, M. W. D., Brown, J. R., André, F. & Okkenhaug, K. PI3K inhibitors are finally coming of age. Nat. Rev. Drug Discov. 20, 741–769 (2021)."). In addition, we observed that essentiality of CDS2 is dose-dependent, indicating that it may be amenable to pharmacological inhibition. As dependency data for healthy tissues are not included in the DepMap data, we cannot exclude that dose-limiting toxicities for CDS2 inhibition may occur. Proliferating fibroblasts are mesenchymal-like, but given that they are mostly involved in wound healing or are cancer-associated[65](https://www.nature.com/articles/s41588-025-02221-2#ref-CR65 "Kalluri, R. & Zeisberg, M. Fibroblasts in cancer. Nat. Rev. Cancer 6, 392–401 (2006)."),[66](https://www.nature.com/articles/s41588-025-02221-2#ref-CR66 "Gurtner, G. C., Werner, S., Barrandon, Y. & Longaker, M. T. Wound repair and regeneration. Nature 453, 314–321 (2008)."), their targeting may not cause severe toxicity or even be clinically beneficial. Furthermore, the GTEx data suggest that high expression of _CDS1_ is ubiquitous across healthy tissues while CDS2-targeting morpholinos are tolerated in mice[67](https://www.nature.com/articles/s41588-025-02221-2#ref-CR67 "Stratman, A. N. et al. Anti-angiogenic effects of VEGF stimulation on endothelium deficient in phosphoinositide recycling. Nat. Commun. 11, 1204 (2020)."). We would propose developing a CDS2-specific inhibitor to maximize the therapeutic index; this may be possible given the different acyl-chain preferences for CDS2 and CDS1 (ref. [43](https://www.nature.com/articles/s41588-025-02221-2#ref-CR43 "D’Souza, K., Kim, Y. J., Balla, T. & Epand, R. M. Distinct properties of the two isoforms of CDP-diacylglycerol synthase. Biochemistry 53, 7358–7367 (2014).")). Clearly, a CDS2-specific inhibitor would be required to determine whether, as our large-scale computational and functional data in vitro and in vivo predict, a therapeutic index for clinically exploiting the _CDS1–2_ SLI indeed exists.

Methods
-------

### Software

Flow cytometry data were collected using FACSDiva (v.8.0) and analyzed using Flowjo (v.10.6.0). For data analysis and statistics, Excel (v.16.71), R (v.4.2.2), RStudio (v.2023.03.0 + 386) or GraphPad Prism (v.9.4.1 or v.10.0.3) was used. The quantitative elements of figures were generated using GraphPad Prism or Flowjo. The visual elements of graphs and plots were designed and generated in Adobe Illustrator (v.26.2). MAGeCK v.0.5.9.5 was used for screen analysis[68](https://www.nature.com/articles/s41588-025-02221-2#ref-CR68 "Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Genome Biol. 15, 554 (2014)."). For live-cell imaging and analysis, ZEN (v.3.5), FIJI (v.2.14.0), Java (v.1.8.0) and ImageJ (v.1.54f) were used[69](https://www.nature.com/articles/s41588-025-02221-2#ref-CR69 "Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012)."). ChimeraX v.1.5rc202210210730 was used for protein visualization[70](https://www.nature.com/articles/s41588-025-02221-2#ref-CR70 "Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. Protein Sci. 32, e4792 (2023)."). ConSurf and AlphaFill were used for evaluating conservation[62](https://www.nature.com/articles/s41588-025-02221-2#ref-CR62 "Yariv, B. et al. Using evolutionary data to make sense of macromolecules with a ‘face-lifted’ ConSurf. Protein Sci. 32, e4582 (2023)."),[63](https://www.nature.com/articles/s41588-025-02221-2#ref-CR63 "Hekkelman, M. L., de Vries, I., Joosten, R. P. & Perrakis, A. AlphaFill: enriching AlphaFold models with ligands and cofactors. Nat. Methods 20, 205–213 (2022)."). DIA-NN (v.1.9 or v.1.9.1), Proteome Discoverer (Thermo Fisher Scientific, v.3.1.0.638) using Chimerys on Ardia (v.1.0.0-qf.1.) and Perseus (v.1.6.15.0) were used for proteomics analysis ([Supplementary Note](https://www.nature.com/articles/s41588-025-02221-2#MOESM1))[71](https://www.nature.com/articles/s41588-025-02221-2#ref-CR71 "Tyanova, S. et al. The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods 13, 731–740 (2016)."),[72](https://www.nature.com/articles/s41588-025-02221-2#ref-CR72 "Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high-throughput. Nat. Methods 17, 41–44 (2020)."). For gene set enrichment, GSEA (v.4.1.0) or STRING was used. For combined light and electron microscopy analysis, LAS X (v.3.7.3.23245), MAPS (v.3.17) and ICY ec-CLEM-v2 (v.ICY-82440) were used. For genome-wide screen sequencing BCLConvert (v.3.9.2-3.e17), Perl (v.5.010) and xcalibr (v.0.12031) were used. For qPCR, Quantstudio (v.1.5.2) was used.

### Nomenclature

HGNC (HUGO Gene Nomenclature Committee) gene names and UniProt protein names were used.

### Cell line culture

Cell lines were cultured in Dulbecco’s modified Eagle’s medium (DMEM; Gibco, cat. no. 41966052) 10% fetal calf serum (FCS; Sigma-Aldrich, cat. no. 3101120) or Roswell Park Memorial Institute (RPMI; Thermo Fisher Scientific, cat. no. 21875034) 10% FCS for lung and blood cancer cell lines. Cells were passaged at subculturing ratios corresponding to 80% confluence at the next passage. During experiments, medium was supplemented with penicillin and streptomycin (Invitrogen, cat. no. 15140122).

Authenticated cell lines were used by STR profiling cell lines with a kit (Promega, cat. no. B9510) or delivering samples to Eurofins (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). Public references were available for all cell lines except SK-MEL-147 and BLM. For SK-MEL-147, a reference was requested via private correspondence[73](https://www.nature.com/articles/s41588-025-02221-2#ref-CR73 "Hanniford, D. et al. Epigenetic silencing of CDR1as drives IGF2BP3-mediated melanoma invasion and metastasis. Cancer Cell 37, 55–70.e15 (2020)."). _Mycoplasma_\-negative cells were used by screening cells in culture for _Mycoplasma_ spp. monthly[74](https://www.nature.com/articles/s41588-025-02221-2#ref-CR74 "Young, L., Sung, J. & Masters, J. R. Detection of mycoplasma in cell cultures. Nat. Protoc. 5, 929–934 (2010).").

### Cancer-specific synthetic lethality prediction

Briefly, to rank potential synthetic lethal candidate pairs, Pearson’s correlation between fitness data (CERES or Chronos) and messenger RNA expression, proteomics, mutation or methylation data were calculated with 21Q2 or 23Q4 DepMap data[34](https://www.nature.com/articles/s41588-025-02221-2#ref-CR34 "DepMap, B. DepMap 21Q2 Public. figshare 
https://doi.org/10.6084/m9.figshare.14541774.v2
(2021)."),[75](https://www.nature.com/articles/s41588-025-02221-2#ref-CR75 "DepMap, B. DepMap 23Q4 Public. figshare 
https://doi.org/10.25452/figshare.plus.24667905.v2
(2023)."). We used Pearson’s _r_ because it measures the strength and direction of the linear relationship between two variables, in this case anchor gene status and target gene dependency. For methylation and mutation, the ranking is reversed, because more mutation or methylation corresponds with less gene activity (whereas more RNA or protein corresponds with more gene activity). To validate this approach, several positive control, synthetic lethal pairs were selected. Positive control analyses were run using all pairs with the anchor genes.

To identify the most promising new SLIs, the anchored approach was replaced with a genome-wide analysis encompassing 340 million gene pairs. Each pair includes the anchor gene (expression gene) and the target gene (fitness effect gene), respectively. For each pair we calculated Pearson’s correlation between the expression of the anchor gene and Δtarget gene lethality across all cancer cell lines included. From the results, the top pair for each anchor was ranked to form a hit list. All duplicate target genes were removed to avoid false positives arising from anchor gene coexpression with other anchor genes. The change in expression was calculated for each anchor gene in the synthetic lethality prediction using reanalyzed TCGA healthy and tumor RNA data[76](https://www.nature.com/articles/s41588-025-02221-2#ref-CR76 "Rahman, M. et al. Alternative preprocessing of RNA-sequencing data in the Cancer Genome Atlas leads to improved analysis results. Bioinformatics 31, 3666–3672 (2015)."). Increases in expression were plotted as a 0% reduction in Fig. [1d](https://www.nature.com/articles/s41588-025-02221-2#Fig1).

To analyze the DepMap for escape mechanisms, we used 23Q4 DepMap data and calculated the partial Pearson’s correlation between the expression of each gene and Δ_CDS2_ lethality across all cancer cell lines corrected for _CDS1_ expression. We did the same for damaging and hotspot mutation data, also correcting for _CDS1_ expression. We corrected for _CDS1_ expression to exclude false positives from _CDS1_ expression coexpressed or co-mutated features.

### Cut-offs used in analyses

For Fig. [1e,g](https://www.nature.com/articles/s41588-025-02221-2#Fig1) and Extended Data Fig. [1f,g](https://www.nature.com/articles/s41588-025-02221-2#Fig8), Δ_CDS2_ lethal cancer cell lines were selected by CERES <−0.5. For context, a CERES of −1 corresponds to the core essential, whereas a CERES of 0 corresponds to the nonessential[33](https://www.nature.com/articles/s41588-025-02221-2#ref-CR33 "Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. Nat. Genet. 49, 1779–1784 (2017)."). For Fig. [1e](https://www.nature.com/articles/s41588-025-02221-2#Fig1), low _CDS1_\-expressing cancer cell lines were selected by log2(TPM + 1) < 1.5. For Fig. [4d](https://www.nature.com/articles/s41588-025-02221-2#Fig4) and Extended Data Fig. [4b](https://www.nature.com/articles/s41588-025-02221-2#Fig11), the cut-offs for high expression were put right before the high peak on the histograms (log2(TPM + 1) of 2.5 for _CDS1_, 4.5 for _CDH1_, 2 for _ZEB1_ and 6 for _VIM_).

### Patient mortality and survival analysis

The 21Q2 DepMap data for cell lines with available RNA and dependency data were used for Δ_CDS2_ lethality. Lineages of patient mortality data and DepMap data were matched manually (Supplementary Table [6](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)).

### Comparing CDS1 expression between healthy tissue and cancer cell lines

GTEx v.8 TPM was downloaded from the portal on 1 January 2022. The 21Q2 DepMap data were downloaded from the DepMap portal. Calibration genes with available TPM in DepMap and GTEx were used for normalization[77](https://www.nature.com/articles/s41588-025-02221-2#ref-CR77 "Eisenberg, E. & Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569–574 (2013)."). Up- and downregulation controls were selected from the literature[78](https://www.nature.com/articles/s41588-025-02221-2#ref-CR78 "Li, M., Sun, Q. & Wang, X. Transcriptional landscape of human cancers. Oncotarget 8, 34534–34551 (2017)."),[79](https://www.nature.com/articles/s41588-025-02221-2#ref-CR79 "Zehir, A. et al. Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients. Nat. Med. 23, 703–713 (2017)."). Briefly, the TPM for each calibration gene was normalized to their own average expression. Then the TPM for _B2M_, _CDS1_, _CDKN2B_ and _E2F1_ was normalized to the average of the calibration gene expression. Calibrated results are similar to uncalibrated results as a result of intrinsic total RNA calibration of the GTEx and DepMap dataset TPM data. The analysis was also performed for blood, skin, lung and brain lineage data, with the original labeling from the DepMap and GTEx, also including soft-tissue atypical teratoid rhabdoid tumors, originally labeled ATRT, in the DepMap brain category. This was expanded to 16 cancer types using minor labeling changes for lineage matching between datasets.

### Healthy and tumor proteome data

Tumor and normal adjacent lung proteome data from Satpathy et al. were downloaded from entry PDC000234 and analyzed[47](https://www.nature.com/articles/s41588-025-02221-2#ref-CR47 "Satpathy, S. et al. A proteogenomic portrait of lung squamous cell carcinoma. Cell 184, 4348–4371.e40 (2021)."). Specifically, log2\-transformed ratios to internal controls were collected and sample names were mapped to their healthy or tumor tissue origin. Before plotting and statistical testing log2\-transformation was undone, resulting in ratios to internal control. The internal control was reported to be a mix of all samples. Still, calibration genes with available ratios in all samples were used for normalization[77](https://www.nature.com/articles/s41588-025-02221-2#ref-CR77 "Eisenberg, E. & Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569–574 (2013)."). Each calibration gene ratio was normalized to its own average ratio. Then, the ratio for CDS1 was normalized to the geometric mean of the normalized calibration gene ratios for each sample.

### Quantitative PCR to quantify RNA

Three housekeeping genes for calibration were selected from a previously described list of options based on stable pan-cancer cell line expression in the DepMap[77](https://www.nature.com/articles/s41588-025-02221-2#ref-CR77 "Eisenberg, E. & Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569–574 (2013)."). The qPCR was performed as previously described[80](https://www.nature.com/articles/s41588-025-02221-2#ref-CR80 "Apriamashvili, G. et al. Ubiquitin ligase STUB1 destabilizes IFNγ-receptor complex to suppress tumor IFNγ signaling. Nat. Commun. 13, 1923 (2022)."). The average of quadruplicate cycle threshold (_C_t) values was calculated for each gene. The −ΔΔ_C_t method was used for analysis: To normalize to calibration genes, the geometric mean of the three calibration gene _C_t values was subtracted from the _CDS1_ _C_t value. To normalize to A431, the _C_t value of A431 was subtracted from the other cell line _C_t values. By base-2 exponential transformation and multiplication by −1, _C_t values were transformed to fold-changes. One representative calibration gene (_VPS29_) was selected and used for Fig. [4c](https://www.nature.com/articles/s41588-025-02221-2#Fig4) and Extended Data Fig. [7a](https://www.nature.com/articles/s41588-025-02221-2#Fig14). Primer sequences are available in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02221-2#MOESM3).

### Lentiviral transduction

Cell lines were lentivirally transduced as previously described[80](https://www.nature.com/articles/s41588-025-02221-2#ref-CR80 "Apriamashvili, G. et al. Ubiquitin ligase STUB1 destabilizes IFNγ-receptor complex to suppress tumor IFNγ signaling. Nat. Commun. 13, 1923 (2022)."). Transduced cells were always kept on selection during cell culture in vitro. The complete selection was verified by the loss of the control untransduced cells on selection. Low-throughput CRISPR perturbation using pLentiCRISPR v.2 Puro or Blast and ectopic expression using pLX304 Blast (_CDS1_, _CDS2_–_GFP_, _CDS2_–_GFP Asp384Ala_ and GFP)[81](https://www.nature.com/articles/s41588-025-02221-2#ref-CR81 "Yang, X. et al. A public genome-scale lentiviral expression library of human ORFs. Nat. Methods 8, 659–661 (2011)."), pLKO.1 Puro or pCDH Puro (mCherry) was performed as previously described[82](https://www.nature.com/articles/s41588-025-02221-2#ref-CR82 "Vredevoogd, D. W. et al. Augmenting immunotherapy impact by lowering tumor TNF cytotoxicity threshold. Cell 178, 585–599.e15 (2019)."),[80](https://www.nature.com/articles/s41588-025-02221-2#ref-CR80 "Apriamashvili, G. et al. Ubiquitin ligase STUB1 destabilizes IFNγ-receptor complex to suppress tumor IFNγ signaling. Nat. Commun. 13, 1923 (2022)."). SHC002 (shControl) and TRCN00000s ending on 35859, 35861 and 35862 (sh_CDS2_ nos. 1–3) from MISSION TRC-Hs 1.5 (human) were used. The shRNA virus was concentrated 40× (Millipore, cat. no. UFC910024). For high-dose transduction, it was used undiluted (sh_CDS2_ nos. 2 and 3) or 1:10 diluted (sh_CDS2_ no. 1) with a final protamine sulfate (Sigma-Aldrich, cat. no. 1101230005) concentration of 8 µg ml−1. For low-dose transductions, it was used 1:100 diluted without protamine sulfate. For Fig. [2e](https://www.nature.com/articles/s41588-025-02221-2#Fig2) Blast resistance was introduced to mCherry-transduced cells using pLentiCRISPR v.2 Blast. In Fig. [2d](https://www.nature.com/articles/s41588-025-02221-2#Fig2), sg_CDS2_ no. 1 and no. 2 were used and for all other experiments sg_CDS2_ no. 2 was used. The sg_CDS1_ was introduced using pLentiCRISPR v.2 Blast. The sgRNA sequences are available in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02221-2#MOESM3).

### Cloning _CDS2_–GFP and _CDS2_–GFP Asp384Ala

The pLX304 Blast containing _CDS2_ was edited to generate _CDS2_–GFP and _CDS2_–GFP Asp384Ala[81](https://www.nature.com/articles/s41588-025-02221-2#ref-CR81 "Yang, X. et al. A public genome-scale lentiviral expression library of human ORFs. Nat. Methods 8, 659–661 (2011)."). Carboxy-terminal GFP was added with a flexible glycine and serine linker sequence (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). The sg_CDS2_ no. 2 target site sequence was altered with silent mutations (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). For the mutant, a nonsilent sequence change was made to encode an Asp384Ala amino acid substitution (5′–3′: GAC → GCC). Cloning involved ligation with T4 DNA ligase, restriction with NheI + SbfI (GFP addition, Asp384Ala mutation) and restriction with XmaI + SbfI (silent mutations). Gene blocks were ordered from IDT and enzymes were from New England Biolabs (NEB). _CDS2_–GFP and _CDS2_–GFP Asp384Ala plasmids are available on request.

### Determining (synthetic) lethality by tracking fluorescent cells

Briefly, after lentiviral transduction (1 d) and puromycin selection of sgRNA-transduced cells (2 d), the indicated cell mixes were prepared. For the in vivo experiments cells were selected 1 d longer. To determine lethality, sgRNA-transduced cells were mixed with non-sgRNA-transduced, mCherry-positive cells. On the day of mixing (day 0) and at the indicated later time points, samples from each cell mix were analyzed by flow cytometry to determine the ratio between negative and positive cells. The final ratios were normalized to day 0 ratios and sgControl ratios. For lethality, normalized ratios were transformed into percentages to obtain survival percentages and subtracted from 100 to obtain lethality percentages. To determine synthetic lethality ectopic _CDS1_\- or GFP-expressing cells, both transduced with the same sgRNA, were mixed on day 0. Then, lethality was calculated for the GFP-positive cells in an identical manner as for the mCherry-negative cells. For Extended Data Fig. [3h](https://www.nature.com/articles/s41588-025-02221-2#Fig10) and Fig. [7d](https://www.nature.com/articles/s41588-025-02221-2#Fig7), unmixed mCherry-positive cells were used to determine the fraction of background mCherry-negative cells among the mCherry-positive cells. This fraction was multiplied with the mCherry-positive cell number from each mixed sample and the resulting expected background was subtracted from the respective mCherry-negative cell counts of that sample.

### Flow cytometry

Flow cytometry data were collected on a BD Fortessa flow cytometer, regularly maintained in the Netherlands Cancer Institute (NKI) flow cytometry facility. One of several live or dead dyes was always included: DAPI (Sigma-Aldrich, cat. no. D9542-50MG), PI (Merck, cat. no. 537059-100MG), DRAQ7 (Thermo Fisher Scientific, cat. no. D15106) and fixable near infrared (Thermo Fisher Scientific, cat. no. L34976). BSA, 0.1%, in phosphate-buffered saline (PBS) was used as a washing and staining solution. Samples were washed 3× (including resuspension as one wash) in 0.1% BSA in PBS, once for dead cell experiments to preserve dead cells and 6× for in vivo samples to reduce postdigestion debris. The gating strategies used are presented in Extended Data Fig. [8](https://www.nature.com/articles/s41588-025-02221-2#Fig15).

### Western blotting

Quantitative western blots (Abby) with total protein were prepared and ran according to the manufacturer’s instructions using the manufacturer’s reagents. Cleaved caspase-3 antibody at 1:100 dilution (primary 9664S; CST) and undiluted anti-rabbit immunoglobulin G horeseradish peroxidase (ProteinSimple, cat. no. 042-406) were used. CRISPR-perturbed samples for cell death analysis were split on day 11 and harvested on day 12, both after transduction. Samples for western blotting by simple western blotting were collected as previously described, with the additional inclusion of the floating cell fraction[80](https://www.nature.com/articles/s41588-025-02221-2#ref-CR80 "Apriamashvili, G. et al. Ubiquitin ligase STUB1 destabilizes IFNγ-receptor complex to suppress tumor IFNγ signaling. Nat. Commun. 13, 1923 (2022)."). As a control, we measured cleaved caspase-3 on induction of apoptosis with 1 µM staurosporine (CST, cat. no. 9953S) in SK-MEL-2 or 200 ng ml−1 of TNF (Immunotools, cat. no. 11343017) + 10 µM TPCA-1 (Selleckchem, cat. no. S2824) in BLM[83](https://www.nature.com/articles/s41588-025-02221-2#ref-CR83 "Wiernicki, B. et al. Excessive phospholipid peroxidation distinguishes ferroptosis from other cell death modes including pyroptosis. Cell Death Dis. 11, 922 (2020)."),[84](https://www.nature.com/articles/s41588-025-02221-2#ref-CR84 "Nassour, J. et al. Telomere-to-mitochondria signalling by ZBP1 mediates replicative crisis. Nature 614, 767–773 (2023).").

### TIDE analysis

Genomic DNA was isolated from cell pellets using All-prep DNA/RNA kit (QIAGEN) according to the manufacturer’s instruction. A region around the sg_CDS2_ no. 2 target site was amplified using NEBNext High Fidelity 2× PCR Master Mix (NEB) according to the manufacturer’s protocol. Primer sequences are available in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02221-2#MOESM3). PCR products were purified by Minelute (QIAGEN) according to the manufacturer’s protocol. Sanger sequencing (Macrogen) of each sample was performed in triplicate and chromatograms of each sg_CDS2_ replicate were compared with each of three control sequences. Comparisons were made using the tracking of indels by decomposition (TIDE) website with settings of insertions and deletions (indels) size range 25 for NCI-H2030, LPS141 and K562 and 30 for SK-MEL-2 and MCF7 (ref. [85](https://www.nature.com/articles/s41588-025-02221-2#ref-CR85 "Brinkman, E. K., Chen, T., Amendola, M. & Van Steensel, B. Easy quantitative assessment of genome editing by sequence trace decomposition. Nucleic Acids Res. 42, e168 (2014).")). The significance cut-off for calling indels was the default 0.001. The decomposition window was adjusted for each comparison. The frameshift percentage was calculated using the percentage without indels combined with significant indel percentages for multiples of three nucleotides.

### Animal studies

All animal studies were approved by the animal ethics committee of the NKI and performed under approved NKI CCD (Centrale Commissie Dierproeven) projects, according to the ethical and procedural guidelines established by the NKI and Dutch legislation (project no. AVD3010020172464 with protocol ID 9.1.10902 for the experiment in Fig. [2h](https://www.nature.com/articles/s41588-025-02221-2#Fig2) and Extended Data Fig. [2f,g](https://www.nature.com/articles/s41588-025-02221-2#Fig9) and project no. AVD30100202316772 with protocol ID 34.1.11655 for the experiment in Fig. [7c](https://www.nature.com/articles/s41588-025-02221-2#Fig7)). Mice were housed in single-use standard cages at controlled filtered air humidity (55%), temperature (21 °C) and light cycle. All housing material, food and water were autoclaved or irradiated before use. Animal husbandry has been previously described[80](https://www.nature.com/articles/s41588-025-02221-2#ref-CR80 "Apriamashvili, G. et al. Ubiquitin ligase STUB1 destabilizes IFNγ-receptor complex to suppress tumor IFNγ signaling. Nat. Commun. 13, 1923 (2022)."). Synthetic lethality in vivo was quantified like it was in vitro (described above) with the following adaptations: Puromycin selection was not continued in vivo_._ Two million melanoma cell mixes in 1:1 PBS Matrigel prepared on day 0 were injected subcutaneously into the flank of weight-randomized 13- to 15-week-old NOD-scid _Il2rγ_\-null (Jax) mice. Mice were numbered and researchers were blinded from this step onwards. Tumor sizes were measured with calipers and the volume was calculated using length × width × width × 0.5. Tumors were harvested, digested and processed into homogeneous single-cell solutions. Then samples were stained for human B2M (BioLegend, cat. no. 316320) at 1:100 dilution to exclude mouse cells and analyzed by flow cytometry. Results were normalized to day 0 and sgControl. For each group one sample was unblinded during flow cytometry acquisition to set voltages and gates for all groups. Growth curves from one million subcutaneously injected unmixed melanoma cells were determined in the same manner as for the mixed cells.

### Genome-wide CRISPR perturbation

Screening was performed as published previously[82](https://www.nature.com/articles/s41588-025-02221-2#ref-CR82 "Vredevoogd, D. W. et al. Augmenting immunotherapy impact by lowering tumor TNF cytotoxicity threshold. Cell 178, 585–599.e15 (2019)."),[86](https://www.nature.com/articles/s41588-025-02221-2#ref-CR86 "Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. Nat. Commun. 9, 5408 (2018)."),[87](https://www.nature.com/articles/s41588-025-02221-2#ref-CR87 "Joung, J. et al. Genome-scale CRISPR–Cas9 knockout and transcriptional activation screening. Nat. Protoc. 12, 828–863 (2017)."). The one-vector Brunello library was amplified in liquid culture at 20,000× coverage and maintenance of the original sgRNA distribution was confirmed. Screen cells were transduced at a multiplicity of infection of 0.3–0.7 with polybrene (Sigma-Aldrich, cat. no. TR-1003-G) at 8 µg ml−1. Screens were performed at 200× coverage. Novaseq was used to sequence the screen samples and the amplified Brunello library. Sequencing primers with unique dual indices per sample were used (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)).

To allow dropout of essential sgRNAs, cells were cultured for 10 d after Brunello library transduction. After this, single sgControl and sg_CDS2_ lentivirus transduction cells were cultured for 18 d. The second-best sgRNA out of four guides was used to calculate the effect size. For SK-MEL-2, comparatively few cells were available as a result of lethality and no hits were detected. To ensure detection of any escape mechanism, a cryopreserved sample from day 7 was cultured until day 32 after single sgControl and sg_CDS2_ transduction, and those results were used for the presented analyses.

The colony formation assays were started with screen cells on day 8. For K562, MCF7, NCI-H2030, LPS141 and SK-MEL-2 cell lines, 125,000, 125,000, 62,500, 31,250 and 125,000 cells were seeded, respectively. The Crystal Violet staining was done when control cells had reached confluence, which was after 4, 14, 12, 7 and 12 d, respectively. A seeding density corresponding to the longest culture time for MCF7 was chosen to ensure maximum sensitivity for detecting a potential difference. For K562 suspension cells, the cells were instead counted with Trypan Blue using an automatic cell counter (BioRad, cat. no. TC20). In addition, lethality was tracked from day 8 until day 19 using tracker cells and used to calculate rescue percentages. The quality control receiver operating characteristic area under the curve (ROC-AUC) was generated as described previously[86](https://www.nature.com/articles/s41588-025-02221-2#ref-CR86 "Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. Nat. Commun. 9, 5408 (2018)."). We used ROC-AUC because it is a standard metric for evaluating CRISPR screen performance, providing a clear measure of how well true positives are distinguished from background noise (true negatives).

### Lethality and synthetic lethality rescue experiments

For the synthetic lethality rescue experiments, the method to track synthetic lethality by tracking GFP positivity was used. The indicated cell mixes were generated 7 d after transduction with sgControl or sg_CDS2_ lentivirus. After overnight attachment, treatment was started and samples were harvested to measure GFP positivity at the starting point. Cells were split and refreshed on day 3 and measured on day 7 after the start of treatment. Phospholipids were complexed with BSA. Stock solutions of 500 µM were generated by incubating phospholipid with 2.5% fatty-acid free BSA from Sigma-Aldrich (cat. no. A8806) in DMEM at 37 °C for 1 h. Liver phosphatidylcholine (PC; cat. no. 840055P-25MG) or PI (cat. no. 840042P-25MG) from Avanti Polar Lipids was used. Final GFP-negative to GFP-positive ratios were normalized to input ratios and sgControl ratios to obtain the synthetic lethal fraction. For rescue, the fold-change in the synthetic lethal fraction was calculated by dividing by the untreated synthetic lethal fractions and transformed into rescue percentages by subtraction from 1 and multiplication by 100. The resulting rescue percentages correspond to the part of the total synthetic lethality that was negated in the treated conditions. For the overnight lethality rescue, CRISPR-perturbed samples for cell death analysis were split on day 11 and harvested on day 12, both after transduction. PI was dissolved in medium and added to the cells at 2 mM.

### GSEAs

Hallmark GSEAs were performed using GSEA v.4.1.0 (Java tool) with the settings h.all.v2022.1.Hs.symbols.gmt, 25,000 permutations and No\_Collapse on Pearson correlations generated as described from DepMap or TCGA data[88](https://www.nature.com/articles/s41588-025-02221-2#ref-CR88 "Mootha, V. K. et al. PGC-1α-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Nat. Genet. 34, 267–273 (2003)."),[89](https://www.nature.com/articles/s41588-025-02221-2#ref-CR89 "Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545–15550 (2005)."). Gene set enrichment for the GO term processes was performed using the Stringdb ‘multiple proteins’ function on the list of 37 genes significant only in the 4 _CDS1_\-negative cell lines[90](https://www.nature.com/articles/s41588-025-02221-2#ref-CR90 "Szklarczyk, D. et al. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 47, D607–D613 (2019).").

### Lipidomics

Quantitative targeted lipidomics was carried out using a flow-injection assay employing lipid class separation by differential mobility spectroscopy and selective multiple reaction monitoring (MRM) per lipid species (Lipidyzer platform). A detailed description of lipid extraction, software and the quantitative nature of the approach can be found elsewhere[91](https://www.nature.com/articles/s41588-025-02221-2#ref-CR91 "Ghorasaini, M. et al. Cross-laboratory standardization of preclinical lipidomics using differential mobility spectrometry and multiple reaction monitoring. Anal. Chem. 93, 16369–16378 (2021)."),[92](https://www.nature.com/articles/s41588-025-02221-2#ref-CR92 "Ghorasaini, M. et al. Congruence and complementarity of differential mobility spectrometry and NMR spectroscopy for plasma lipidomics. Metabolites 12, 1030 (2022)."),[93](https://www.nature.com/articles/s41588-025-02221-2#ref-CR93 "Su, B. et al. A DMS shotgun lipidomics workflow application to facilitate high-throughput, comprehensive lipidomics. J. Am. Soc. Mass. Spectrom. 32, 2655–2663 (2021)."). Briefly, after the addition of >60 deuterated internal standards, lipids were extracted using methyl _tert_\-butyl ether. Organic extracts were subsequently dried under a gentle stream of nitrogen and reconstituted in running buffer. Lipids were subsequently analyzed using flow injection in MRM mode employing a Shimadzu Nexera series HPLC (high-performance liquid chromatograph) and a Sciex QTrap 5500 mass spectrometer. For further analysis, lipidomic data normalized to cell number were used to calculate the quadruplicate fold-changes for plotting and statistical analysis.

### Proteomics

A detailed sample- and data-processing protocol for generating the proteomic results is available in [Supplementary Note](https://www.nature.com/articles/s41588-025-02221-2#MOESM1). For plotting Fig. [5b](https://www.nature.com/articles/s41588-025-02221-2#Fig5) and Extended Data Fig. [5a,c](https://www.nature.com/articles/s41588-025-02221-2#Fig12), log2\-transformed protein group abundances were used that were available for each cell line in at least four out of four replicates in the sgControl or sg_CDS2_ sample group. Differentially expressed proteins were determined using a two-sided Student’s _t_\-test (thresholds: _P_ < 0.05). For plotting Fig. [5c](https://www.nature.com/articles/s41588-025-02221-2#Fig5) and Extended Data Fig. [5c](https://www.nature.com/articles/s41588-025-02221-2#Fig12), fold-changes relative to the average protein abundance per experiment were calculated for CDS1, CDS2 and calibration proteins. Then, for the CDS1 and CDS2 proteins, the fold-change to the median calibration protein levels per sample was calculated. For the CDS1 and CDS2 proteins, the fold-change to average abundance by experiment was recalculated and, respectively, replicates and experiments were averaged. With the combined data, the relative abundance of the CDS1 and CDS2 proteins to the highest included cancer cell line was calculated and plotted.

### Light microscopy

Live cells stained with BODIPY (2 µM, Thermo Fisher Scientific, cat. no. D3922) were imaged on a regularly maintained Zeiss Axio Observer Z1 microscope at the NKI imaging facility using phase contrast and fluorescence equipment. A home-built, temperature-controlled, live-cell chamber and 5% CO2 incubation system were used. Data were analyzed using custom ImageJ code. Briefly, for each condition, the low complexity area on phase contrast was excluded to calculate cell area and BODIPY fluorescence was used to calculate the lipid area. Cell lines were imaged 14 d (SK-MEL-2) or 20 d (NCI-H2030) after transduction (Fig. [6b](https://www.nature.com/articles/s41588-025-02221-2#Fig6)). For imaging, cells were seeded several days in advance in a twofold dilution series of seven steps. At the time of imaging, different conditions were imaged at a similar density using the available dilution series.

### Electron microscopy

Correlative light electron microscopy (CLEM) was performed as described before[94](https://www.nature.com/articles/s41588-025-02221-2#ref-CR94 "Grootemaat, A. E. et al. Lipid and nucleocapsid N-protein accumulation in COVID-19 patient lung and infected cells. Microbiol. Spectr. 10, e0127121 (2022)."),[95](https://www.nature.com/articles/s41588-025-02221-2#ref-CR95 "Bedard, M. et al. A terpene nucleoside from M. tuberculosis induces lysosomal lipid storage in foamy macrophages. J. Clin. Invest. 133, e161276 (2023)."). Briefly, for CLEM 150- to 300-nm cryosections of cells were placed on to an EM grid, stained with Nile Red (Sigma-Aldrich, cat. no. 72485) and Hoechst 33342 (Thermo Fisher Scientific, cat. no. H3570) and imaged with a Wide-field Fluorescence Microscope (Leica DM6, ×100 oil objective). After washing and contrasting for EM purposes, the identical grid was imaged using a transmission electron microscope (Thermo Fisher Scientific, Talos, cat. no. L120c with a Ceta 16M camera).

### Statistics and replication

Significance was tested and depicted as indicated. Two-sided tests were employed. To ensure reproducibility, we addressed experimental variation by including multiple replicates and biological variation by including multiple cell lines and we employed multiple methods whenever possible. Replicates were generated during the same period in time and independent experiments were performed during different moments in time. For admixing, experimental replicates were cultured independently from the start to the finish. The assumption of normality was kept at a minimum by using nonparametric tests for large datasets (Kruskal–Wallis test and Mann–Whitney _U_ test). Data processing was always verified by a second researcher. Complex bioinformatic analyses were redone independently.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02221-2#MOESM2) linked to this article.

Data availability
-----------------

Tables presenting a selection of plotted source data, unplotted source data and methodological data are available (Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)–[7](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). To allow large file sizes, unprocessed source data are available on repositories. Unprocessed proteomic data are available on our entry in the Proteomics IDEntifications repository, under accession no. [PXD045833](http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD045833) (ref. [96](https://www.nature.com/articles/s41588-025-02221-2#ref-CR96 "Arnoldus, T. et al. CDS2 is a synthetic lethal target in mesenchymal-like cancers (proteomic dataset). PRIDE 
www.ebi.ac.uk/pride/archive/projects/PXD045833
(2025).")). All other unprocessed source data for the present study are available via figshare at [https://doi.org/10.6084/m9.figshare.27951504](https://doi.org/10.6084/m9.figshare.27951504) (ref. [97](https://www.nature.com/articles/s41588-025-02221-2#ref-CR97 "Arnoldus, T. et al. CDS2 is a synthetic lethal target in mesenchymal-like cancers (dataset). figshare 
https://doi.org/10.6084/m9.figshare.27951504
(2025).")). Detailed instructions for downloading the specific public data files used in our software notebooks, including links to repositories when possible, are available via Zenodo at [https://doi.org/10.5281/zenodo.15194712](https://doi.org/10.5281/zenodo.15194712) (ref. [98](https://www.nature.com/articles/s41588-025-02221-2#ref-CR98 "Arnoldus, T. PeeperLab/cds2-synthetic-lethal-notebooks: v1.0.1. Zenodo 
https://doi.org/10.5281/zenodo.15194712
(2025).")). Briefly here, we used the HGNC database ([https://www.genenames.org](http://genenames.org/)); the UniProt database ([https://www.uniprot.org](https://www.uniprot.org/)); ENCODE data via Harmonizone ([https://maayanlab.cloud/harmonizone](https://maayanlab.cloud/Harmonizome/))[51](https://www.nature.com/articles/s41588-025-02221-2#ref-CR51 "Feingold, E. A. et al. The ENCODE (ENCyclopedia of DNA Elements) project. Science 306, 636–640 (2004)."); DepMap data from 21Q2 and 23Q4 on figshare and the DepMap portal ([https://depmap.org/portal](https://depmap.org/portal))[33](https://www.nature.com/articles/s41588-025-02221-2#ref-CR33 "Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. Nat. Genet. 49, 1779–1784 (2017)."),[34](https://www.nature.com/articles/s41588-025-02221-2#ref-CR34 "DepMap, B. DepMap 21Q2 Public. figshare 
https://doi.org/10.6084/m9.figshare.14541774.v2
(2021)."),[37](https://www.nature.com/articles/s41588-025-02221-2#ref-CR37 "Nusinow, D. P. et al. Quantitative proteomics of the Cancer Cell Line Encyclopedia. Cell 180, 387–402.e16 (2020)."),[38](https://www.nature.com/articles/s41588-025-02221-2#ref-CR38 "Ghandi, M. et al. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019)."),[75](https://www.nature.com/articles/s41588-025-02221-2#ref-CR75 "DepMap, B. DepMap 23Q4 Public. figshare 
https://doi.org/10.25452/figshare.plus.24667905.v2
(2023)."); reprocessed TCGA RNA-seq data from Gene Expression Omnibus, accession no. [GSE62944](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944) (ref. [76](https://www.nature.com/articles/s41588-025-02221-2#ref-CR76 "Rahman, M. et al. Alternative preprocessing of RNA-sequencing data in the Cancer Genome Atlas leads to improved analysis results. Bioinformatics 31, 3666–3672 (2015).")); original RNA-seq and clinical CDR TCGA data[35](https://www.nature.com/articles/s41588-025-02221-2#ref-CR35 "Weinstein, J. N. et al. The cancer genome atlas pan-cancer analysis project. Nat. Genet. 45, 1113–1120 (2013).") from the Genomic Data Commons website ([https://gdc.cancer.gov](https://gdc.cancer.gov/); downloaded 12 January 2023); bulk expression gene TPM data from the GTEx[36](https://www.nature.com/articles/s41588-025-02221-2#ref-CR36 "Lonsdale, J. et al. The Genotype-Tissue Expression (GTEx) project. Nat. Genet. 45, 580–585 (2013).") portal ([https://gtexportal.org/home](https://gtexportal.org/home)) (v.8); Satpathy et al. lung proteome data and tumor normal mapping from Proteomic Data Commons, accession no. [PCD000234](https://pdc.cancer.gov/pdc/study/PDC000234) (ref. [47](https://www.nature.com/articles/s41588-025-02221-2#ref-CR47 "Satpathy, S. et al. A proteogenomic portrait of lung squamous cell carcinoma. Cell 184, 4348–4371.e40 (2021).")); untreated (0 h) BLM and SK-MEL-147 cell-line RNA-seq data from refine.bio, accession no. [SRP132830](https://www.ncbi.nlm.nih.gov/sra/?term=SRP132830) (ref. [82](https://www.nature.com/articles/s41588-025-02221-2#ref-CR82 "Vredevoogd, D. W. et al. Augmenting immunotherapy impact by lowering tumor TNF cytotoxicity threshold. Cell 178, 585–599.e15 (2019).")); and 2013–2017 US SEER cancer mortality data from the SEER website ([https://seer.cancer.gov](https://seer.cancer.gov/); downloaded 18 January 2023; copy in Supplementary Table [6](https://www.nature.com/articles/s41588-025-02221-2#MOESM3)). [Source data](https://www.nature.com/articles/s41588-025-02221-2#Sec42) are provided with this paper.

Code availability
-----------------

For the computational analyses for which custom code was used, software notebooks designed to ease reproduction are available via Zenodo at [https://doi.org/10.5281/zenodo.15194712](https://doi.org/10.5281/zenodo.15194712) (ref. [98](https://www.nature.com/articles/s41588-025-02221-2#ref-CR98 "Arnoldus, T. PeeperLab/cds2-synthetic-lethal-notebooks: v1.0.1. Zenodo 
https://doi.org/10.5281/zenodo.15194712
(2025).")). Either files required to run the provided software are directly supplied or detailed instructions for downloading are supplied on our Zenodo entry.

References
----------

1.  Hartwell, L. H., Szankasi, P., Roberts, C. J., Murray, A. W. & Friend, S. H. Integrating genetic approaches into the discovery of anticancer drugs. _Science_ **278**, 1064–1068 (1997).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXnt1Wju7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9353181)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrating%20genetic%20approaches%20into%20the%20discovery%20of%20anticancer%20drugs&journal=Science&volume=278&pages=1064-1068&publication_year=1997&author=Hartwell%2CLH&author=Szankasi%2CP&author=Roberts%2CCJ&author=Murray%2CAW&author=Friend%2CSH) 
    
2.  Venkitaraman, A. R. Cancer susceptibility and the functions of _BRCA1_ and _BRCA2_. _Cell_ **108**, 171–182 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XhtV2hsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11832208)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cancer%20susceptibility%20and%20the%20functions%20of%20BRCA1%20and%20BRCA2&journal=Cell&volume=108&pages=171-182&publication_year=2002&author=Venkitaraman%2CAR) 
    
3.  Bryant, H. E. et al. Specific killing of _BRCA2_\-deficient tumours with inhibitors of poly(ADP-ribose) polymerase. _Nature_ **434**, 913–917 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXjtFOmsrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15829966)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Specific%20killing%20of%20BRCA2-deficient%20tumours%20with%20inhibitors%20of%20poly%28ADP-ribose%29%20polymerase&journal=Nature&volume=434&pages=913-917&publication_year=2005&author=Bryant%2CHE) 
    
4.  Farmer, H. et al. Targeting the DNA repair defect in _BRCA_ mutant cells as a therapeutic strategy. _Nature_ **434**, 917–921 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXjtFOmsrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15829967)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20the%20DNA%20repair%20defect%20in%20BRCA%20mutant%20cells%20as%20a%20therapeutic%20strategy&journal=Nature&volume=434&pages=917-921&publication_year=2005&author=Farmer%2CH) 
    
5.  Fong, P. C. et al. Inhibition of poly(ADP-ribose) polymerase in tumors from mutation carriers. _N. Engl. J. Med._ **361**, 123–134 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXosVKrtrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19553641)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inhibition%20of%20poly%28ADP-ribose%29%20polymerase%20in%20tumors%20from%20mutation%20carriers&journal=N.%20Engl.%20J.%20Med.&volume=361&pages=123-134&publication_year=2009&author=Fong%2CPC) 
    
6.  Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. _Nature_ **568**, 551–556 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVSgtro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30971823)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6580861)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WRN%20helicase%20is%20a%20synthetic%20lethal%20target%20in%20microsatellite%20unstable%20cancers&journal=Nature&volume=568&pages=551-556&publication_year=2019&author=Chan%2CEM) 
    
7.  Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. _iScience_ **13**, 488–497 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXls1aqs7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30898619)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6441948)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Werner%20syndrome%20helicase%20is%20required%20for%20the%20survival%20of%20cancer%20cells%20with%20microsatellite%20instability&journal=iScience&volume=13&pages=488-497&publication_year=2019&author=Kategaya%2CL&author=Perumal%2CSK&author=Hager%2CJH&author=Belmont%2CLD) 
    
8.  Frezza, C. et al. Haem oxygenase is synthetically lethal with the tumour suppressor fumarate hydratase. _Nature_ **477**, 225–228 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtFOlsLrO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21849978)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haem%20oxygenase%20is%20synthetically%20lethal%20with%20the%20tumour%20suppressor%20fumarate%20hydratase&journal=Nature&volume=477&pages=225-228&publication_year=2011&author=Frezza%2CC) 
    
9.  Feng, X. et al. Genome-wide CRISPR screens using isogenic cells reveal vulnerabilities conferred by loss of tumor suppressors. _Sci. Adv._ **8**, eabm6638 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsV2rurfM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35559673)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9106303)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20CRISPR%20screens%20using%20isogenic%20cells%20reveal%20vulnerabilities%20conferred%20by%20loss%20of%20tumor%20suppressors&journal=Sci.%20Adv.&volume=8&publication_year=2022&author=Feng%2CX) 
    
10.  Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. _Cell Rep._ **39**, 110636 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35417719)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interrogation%20of%20cancer%20gene%20dependencies%20reveals%20paralog%20interactions%20of%20autosome%20and%20sex%20chromosome-encoded%20genes&journal=Cell%20Rep.&volume=39&publication_year=2022&author=K%C3%B6ferle%2CA) 
    
11.  Oike, T. et al. A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1. _Cancer Res._ **73**, 5508–5518 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtlKkurfO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23872584)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20synthetic%20lethality-based%20strategy%20to%20treat%20cancers%20harboring%20a%20genetic%20deficiency%20in%20the%20chromatin%20remodeling%20factor%20BRG1&journal=Cancer%20Res.&volume=73&pages=5508-5518&publication_year=2013&author=Oike%2CT) 
    
12.  Hoffman, G. R. et al. Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in _BRG1_\-deficient cancers. _Proc. Natl Acad. Sci. USA_ **111**, 3128–3133 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXjtVKjurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24520176)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3939885)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20epigenetics%20approach%20identifies%20BRM%2FSMARCA2%20as%20a%20critical%20synthetic%20lethal%20target%20in%20BRG1-deficient%20cancers&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=111&pages=3128-3133&publication_year=2014&author=Hoffman%2CGR) 
    
13.  Neggers, J. E. et al. Synthetic lethal interaction between the ESCRT paralog enzymes VPS4A and VPS4B in cancers harboring loss of chromosome 18q or 16q. _Cell Rep._ **33**, 108475 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthetic%20lethal%20interaction%20between%20the%20ESCRT%20paralog%20enzymes%20VPS4A%20and%20VPS4B%20in%20cancers%20harboring%20loss%20of%20chromosome%2018q%20or%2016q&journal=Cell%20Rep.&volume=33&publication_year=2020&author=Neggers%2CJE) 
    
14.  Jerby-Arnon, L. et al. Predicting cancer-specific vulnerability via data-driven detection of synthetic lethality. _Cell_ **158**, 1199–1209 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsV2jtLvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25171417)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Predicting%20cancer-specific%20vulnerability%20via%20data-driven%20detection%20of%20synthetic%20lethality&journal=Cell&volume=158&pages=1199-1209&publication_year=2014&author=Jerby-Arnon%2CL) 
    
15.  Muller, F. L. et al. Passenger deletions generate therapeutic vulnerabilities in cancer. _Nature_ **488**, 337–342 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1Wkt7vL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22895339)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3712624)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Passenger%20deletions%20generate%20therapeutic%20vulnerabilities%20in%20cancer&journal=Nature&volume=488&pages=337-342&publication_year=2012&author=Muller%2CFL) 
    
16.  Zhao, D. et al. Synthetic essentiality of chromatin remodelling factor CHD1 in PTEN-deficient cancer. _Nature_ **542**, 484–488 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXisV2ktLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28166537)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5448706)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthetic%20essentiality%20of%20chromatin%20remodelling%20factor%20CHD1%20in%20PTEN-deficient%20cancer&journal=Nature&volume=542&pages=484-488&publication_year=2017&author=Zhao%2CD) 
    
17.  Lee, R. et al. Synthetic essentiality of tryptophan 2,3-dioxygenase 2 in APC-mutated colorectal cancer. _Cancer Discov._ **12**, 1702–1717 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1KlsrvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35537038)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9262860)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthetic%20essentiality%20of%20tryptophan%202%2C3-dioxygenase%202%20in%20APC-mutated%20colorectal%20cancer&journal=Cancer%20Discov.&volume=12&pages=1702-1717&publication_year=2022&author=Lee%2CR) 
    
18.  Mavrakis, K. J. et al. Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5. _Science_ **351**, 1208–1213 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xjs12gtr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26912361)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disordered%20methionine%20metabolism%20in%20MTAP%2FCDKN2A-deleted%20cancers%20leads%20to%20dependence%20on%20PRMT5&journal=Science&volume=351&pages=1208-1213&publication_year=2016&author=Mavrakis%2CKJ) 
    
19.  Kryukov, G. V. et al. MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells. _Science_ **351**, 1214–1218 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xjs12gtro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26912360)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4997612)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MTAP%20deletion%20confers%20enhanced%20dependency%20on%20the%20PRMT5%20arginine%20methyltransferase%20in%20cancer%20cells&journal=Science&volume=351&pages=1214-1218&publication_year=2016&author=Kryukov%2CGV) 
    
20.  Tong, A. H. Y. et al. Systematic genetic analysis with ordered arrays of yeast deletion mutants. _Science_ **294**, 2364–2368 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXptFGrurY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11743205)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20genetic%20analysis%20with%20ordered%20arrays%20of%20yeast%20deletion%20mutants&journal=Science&volume=294&pages=2364-2368&publication_year=2001&author=Tong%2CAHY) 
    
21.  Pan, X. et al. A robust toolkit for functional profiling of the yeast genome. _Mol. Cell_ **16**, 487–496 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVWhtr%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15525520)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20robust%20toolkit%20for%20functional%20profiling%20of%20the%20yeast%20genome&journal=Mol.%20Cell&volume=16&pages=487-496&publication_year=2004&author=Pan%2CX) 
    
22.  Costanzo, M. et al. The genetic landscape of a cell. _Science_ **327**, 425–431 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXnt1ahug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20093466)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600254)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genetic%20landscape%20of%20a%20cell&journal=Science&volume=327&pages=425-431&publication_year=2010&author=Costanzo%2CM) 
    
23.  Shalem, O. et al. Genome-scale CRISPR–Cas9 knockout screening in human cells. _Science_ **343**, 84–87 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXkslyj)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24336571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-scale%20CRISPR%E2%80%93Cas9%20knockout%20screening%20in%20human%20cells&journal=Science&volume=343&pages=84-87&publication_year=2014&author=Shalem%2CO) 
    
24.  Wang, T., Wei, J. J., Sabatini, D. M. & Lander, E. S. Genetic screens in human cells using the CRISPR–Cas9 system. _Science_ **343**, 80–84 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXkslOq)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24336569)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20screens%20in%20human%20cells%20using%20the%20CRISPR%E2%80%93Cas9%20system&journal=Science&volume=343&pages=80-84&publication_year=2014&author=Wang%2CT&author=Wei%2CJJ&author=Sabatini%2CDM&author=Lander%2CES) 
    
25.  Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. _Nat. Commun._ **12**, 565 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combinatorial%20CRISPR%20screen%20identifies%20fitness%20effects%20of%20gene%20paralogues&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Thompson%2CNA) 
    
26.  van Delft, M. F. et al. The BH3 mimetic ABT-737 targets selective Bcl-2 proteins and efficiently induces apoptosis via Bak/Bax if Mcl-1 is neutralized. _Cancer Cell_ **10**, 389–399 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17097561)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2953559)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20BH3%20mimetic%20ABT-737%20targets%20selective%20Bcl-2%20proteins%20and%20efficiently%20induces%20apoptosis%20via%20Bak%2FBax%20if%20Mcl-1%20is%20neutralized&journal=Cancer%20Cell&volume=10&pages=389-399&publication_year=2006&author=Delft%2CMF) 
    
27.  Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. _Genome Biol._ **21**, 262 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFOis7%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33059726)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7558751)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiplex%20enCas12a%20screens%20detect%20functional%20buffering%20among%20paralogs%20otherwise%20masked%20in%20monogenic%20Cas9%20knockout%20screens&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Dede%2CM&author=McLaughlin%2CM&author=Kim%2CE&author=Hart%2CT) 
    
28.  Tsherniak, A. et al. Defining a cancer dependency map. _Cell_ **170**, 564–576.e16 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1GisLjP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28753430)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5667678)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defining%20a%20cancer%20dependency%20map&journal=Cell&volume=170&pages=564-576.e16&publication_year=2017&author=Tsherniak%2CA) 
    
29.  Lee, J. S. et al. Synthetic lethality-mediated precision oncology via the tumor transcriptome. _Cell_ **184**, 2487–2502.e13 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXptVWhsLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33857424)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9310669)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthetic%20lethality-mediated%20precision%20oncology%20via%20the%20tumor%20transcriptome&journal=Cell&volume=184&pages=2487-2502.e13&publication_year=2021&author=Lee%2CJS) 
    
30.  Lee, J. S. et al. Harnessing synthetic lethality to predict the response to cancer treatment. _Nat. Commun._ **9**, 2546 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29959327)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6026173)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Harnessing%20synthetic%20lethality%20to%20predict%20the%20response%20to%20cancer%20treatment&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Lee%2CJS) 
    
31.  Helming, K. C. et al. ARID1B is a specific vulnerability in ARID1A-mutant cancers. _Nat. Med._ **20**, 251–254 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXivFais7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24562383)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3954704)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ARID1B%20is%20a%20specific%20vulnerability%20in%20ARID1A-mutant%20cancers&journal=Nat.%20Med.&volume=20&pages=251-254&publication_year=2014&author=Helming%2CKC) 
    
32.  Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. _Cell Rep._ **36**, 109597 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFels7%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34469736)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8534300)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20of%20synthetic%20lethal%20and%20tumor%20suppressor%20paralog%20pairs%20in%20the%20human%20genome&journal=Cell%20Rep.&volume=36&publication_year=2021&author=Parrish%2CPCR) 
    
33.  Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. _Nat. Genet._ **49**, 1779–1784 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslehtrjI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29083409)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computational%20correction%20of%20copy%20number%20effect%20improves%20specificity%20of%20CRISPR-Cas9%20essentiality%20screens%20in%20cancer%20cells&journal=Nat.%20Genet.&volume=49&pages=1779-1784&publication_year=2017&author=Meyers%2CRM) 
    
34.  DepMap, B. DepMap 21Q2 Public. _figshare_ [https://doi.org/10.6084/m9.figshare.14541774.v2](https://doi.org/10.6084/m9.figshare.14541774.v2) (2021).
    
35.  Weinstein, J. N. et al. The cancer genome atlas pan-cancer analysis project. _Nat. Genet._ **45**, 1113–1120 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24071849)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3919969)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20cancer%20genome%20atlas%20pan-cancer%20analysis%20project&journal=Nat.%20Genet.&volume=45&pages=1113-1120&publication_year=2013&author=Weinstein%2CJN) 
    
36.  Lonsdale, J. et al. The Genotype-Tissue Expression (GTEx) project. _Nat. Genet._ **45**, 580–585 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXotlGls74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Genotype-Tissue%20Expression%20%28GTEx%29%20project&journal=Nat.%20Genet.&volume=45&pages=580-585&publication_year=2013&author=Lonsdale%2CJ) 
    
37.  Nusinow, D. P. et al. Quantitative proteomics of the Cancer Cell Line Encyclopedia. _Cell_ **180**, 387–402.e16 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnvFeltLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31978347)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7339254)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantitative%20proteomics%20of%20the%20Cancer%20Cell%20Line%20Encyclopedia&journal=Cell&volume=180&pages=387-402.e16&publication_year=2020&author=Nusinow%2CDP) 
    
38.  Ghandi, M. et al. Next-generation characterization of the Cancer Cell Line Encyclopedia. _Nature_ **569**, 503–508 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXptlOhtLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31068700)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6697103)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Next-generation%20characterization%20of%20the%20Cancer%20Cell%20Line%20Encyclopedia&journal=Nature&volume=569&pages=503-508&publication_year=2019&author=Ghandi%2CM) 
    
39.  DeWeirdt, P. C. et al. Genetic screens in isogenic mammalian cell lines without single cell cloning. _Nat. Commun._ **11**, 752 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjs1OhtL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32029722)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7005275)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20screens%20in%20isogenic%20mammalian%20cell%20lines%20without%20single%20cell%20cloning&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=DeWeirdt%2CPC) 
    
40.  Lin, H. K. et al. Functional buffering via cell-specific gene expression promotes tissue homeostasis and cancer robustness. _Sci. Rep._ **12**, 2974 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XkvFSmsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35194081)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8863889)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20buffering%20via%20cell-specific%20gene%20expression%20promotes%20tissue%20homeostasis%20and%20cancer%20robustness&journal=Sci.%20Rep.&volume=12&publication_year=2022&author=Lin%2CHK) 
    
41.  Halford, S., Dulai, K. S., Daw, S. C., Fitzgibbon, J. & Hunt, D. M. Isolation and chromosomal localization of two human CDP-diacylglycerol synthase (CDS) genes. _Genomics_ **54**, 140–144 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXnslejsLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9806839)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Isolation%20and%20chromosomal%20localization%20of%20two%20human%20CDP-diacylglycerol%20synthase%20%28CDS%29%20genes&journal=Genomics&volume=54&pages=140-144&publication_year=1998&author=Halford%2CS&author=Dulai%2CKS&author=Daw%2CSC&author=Fitzgibbon%2CJ&author=Hunt%2CDM) 
    
42.  Volta, M. et al. Identification and characterization of CDS2, a mammalian homolog of the _Drosophila_ CDP-diacylglycerol synthase gene. _Genomics_ **55**, 68–77 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXoslaitA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9889000)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20and%20characterization%20of%20CDS2%2C%20a%20mammalian%20homolog%20of%20the%20Drosophila%20CDP-diacylglycerol%20synthase%20gene&journal=Genomics&volume=55&pages=68-77&publication_year=1999&author=Volta%2CM) 
    
43.  D’Souza, K., Kim, Y. J., Balla, T. & Epand, R. M. Distinct properties of the two isoforms of CDP-diacylglycerol synthase. _Biochemistry_ **53**, 7358–7367 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25375833)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20properties%20of%20the%20two%20isoforms%20of%20CDP-diacylglycerol%20synthase&journal=Biochemistry&volume=53&pages=7358-7367&publication_year=2014&author=D%E2%80%99Souza%2CK&author=Kim%2CYJ&author=Balla%2CT&author=Epand%2CRM) 
    
44.  Goncalves, M. D., Hopkins, B. D. & Cantley, L. C. Phosphatidylinositol 3-kinase, growth disorders, and cancer. _N. Engl. J. Med._ **379**, 2052–2062 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisV2ltbvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30462943)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phosphatidylinositol%203-kinase%2C%20growth%20disorders%2C%20and%20cancer&journal=N.%20Engl.%20J.%20Med.&volume=379&pages=2052-2062&publication_year=2018&author=Goncalves%2CMD&author=Hopkins%2CBD&author=Cantley%2CLC) 
    
45.  Cantley, L. C. The phosphoinositide 3-kinase pathway. _Science_ **296**, 1655–1657 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XktlChu7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12040186)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20phosphoinositide%203-kinase%20pathway&journal=Science&volume=296&pages=1655-1657&publication_year=2002&author=Cantley%2CLC) 
    
46.  Vanhaesebroeck, B., Perry, M. W. D., Brown, J. R., André, F. & Okkenhaug, K. PI3K inhibitors are finally coming of age. _Nat. Rev. Drug Discov._ **20**, 741–769 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlaitrjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34127844)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9297732)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PI3K%20inhibitors%20are%20finally%20coming%20of%20age&journal=Nat.%20Rev.%20Drug%20Discov.&volume=20&pages=741-769&publication_year=2021&author=Vanhaesebroeck%2CB&author=Perry%2CMWD&author=Brown%2CJR&author=Andr%C3%A9%2CF&author=Okkenhaug%2CK) 
    
47.  Satpathy, S. et al. A proteogenomic portrait of lung squamous cell carcinoma. _Cell_ **184**, 4348–4371.e40 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslGisL%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34358469)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8475722)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20proteogenomic%20portrait%20of%20lung%20squamous%20cell%20carcinoma&journal=Cell&volume=184&pages=4348-4371.e40&publication_year=2021&author=Satpathy%2CS) 
    
48.  Yang, J. & Weinberg, R. A. Epithelial–mesenchymal transition: at the crossroads of development and tumor metastasis. _Dev. Cell_ **14**, 818–829 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXnsVCrs7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18539112)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epithelial%E2%80%93mesenchymal%20transition%3A%20at%20the%20crossroads%20of%20development%20and%20tumor%20metastasis&journal=Dev.%20Cell&volume=14&pages=818-829&publication_year=2008&author=Yang%2CJ&author=Weinberg%2CRA) 
    
49.  Mak, M. P. et al. A patient-derived, pan-cancer EMT signature identifies global molecular alterations and immune target enrichment following epithelial-to-mesenchymal transition. _Clin. Cancer Res._ **22**, 609–620 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitFeru7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26420858)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20patient-derived%2C%20pan-cancer%20EMT%20signature%20identifies%20global%20molecular%20alterations%20and%20immune%20target%20enrichment%20following%20epithelial-to-mesenchymal%20transition&journal=Clin.%20Cancer%20Res.&volume=22&pages=609-620&publication_year=2016&author=Mak%2CMP) 
    
50.  Gemmill, R. M. et al. ZEB1-responsive genes in non-small cell lung cancer. _Cancer Lett._ **300**, 66–78 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVGns7zM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20980099)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ZEB1-responsive%20genes%20in%20non-small%20cell%20lung%20cancer&journal=Cancer%20Lett.&volume=300&pages=66-78&publication_year=2011&author=Gemmill%2CRM) 
    
51.  Feingold, E. A. et al. The ENCODE (ENCyclopedia of DNA Elements) project. _Science_ **306**, 636–640 (2004).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20ENCODE%20%28ENCyclopedia%20of%20DNA%20Elements%29%20project&journal=Science&volume=306&pages=636-640&publication_year=2004&author=Feingold%2CEA) 
    
52.  Hanahan, D. & Weinberg, R. A. Hallmarks of cancer: the next generation. _Cell_ **144**, 646–674 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXjsFeqtrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21376230)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hallmarks%20of%20cancer%3A%20the%20next%20generation&journal=Cell&volume=144&pages=646-674&publication_year=2011&author=Hanahan%2CD&author=Weinberg%2CRA) 
    
53.  Fischer, K. R. et al. Epithelial-to-mesenchymal transition is not required for lung metastasis but contributes to chemoresistance. _Nature_ **527**, 472–476 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVajs7%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26560033)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4662610)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epithelial-to-mesenchymal%20transition%20is%20not%20required%20for%20lung%20metastasis%20but%20contributes%20to%20chemoresistance&journal=Nature&volume=527&pages=472-476&publication_year=2015&author=Fischer%2CKR) 
    
54.  Mani, S. A. et al. The epithelial–mesenchymal transition generates cells with properties of stem cells. _Cell_ **133**, 704–715 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmsVOru7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18485877)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728032)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20epithelial%E2%80%93mesenchymal%20transition%20generates%20cells%20with%20properties%20of%20stem%20cells&journal=Cell&volume=133&pages=704-715&publication_year=2008&author=Mani%2CSA) 
    
55.  Van Espen, B. et al. RNF185 control of COL3A1 expression limits prostate cancer migration and metastatic potential. _Mol. Cancer Res._ **21**, 41–54 (2024).
    
56.  Bateman, A. et al. UniProt: the Universal Protein Knowledgebase in 2023. _Nucleic Acids Res._ **51**, D523–D531 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtl2itb%2FI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UniProt%3A%20the%20Universal%20Protein%20Knowledgebase%20in%202023&journal=Nucleic%20Acids%20Res.&volume=51&pages=D523-D531&publication_year=2023&author=Bateman%2CA) 
    
57.  Wainberg, M. et al. A genome-wide atlas of co-essential modules assigns function to uncharacterized genes. _Nat. Genet._ **53**, 638–649 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXovV2ktLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33859415)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8763319)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genome-wide%20atlas%20of%20co-essential%20modules%20assigns%20function%20to%20uncharacterized%20genes&journal=Nat.%20Genet.&volume=53&pages=638-649&publication_year=2021&author=Wainberg%2CM) 
    
58.  Olzmann, J. A. & Carvalho, P. Dynamics and functions of lipid droplets. _Nat. Rev. Mol. Cell Biol._ **20**, 137–155 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVGms7fE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30523332)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6746329)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20and%20functions%20of%20lipid%20droplets&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=20&pages=137-155&publication_year=2019&author=Olzmann%2CJA&author=Carvalho%2CP) 
    
59.  Hanahan, D. & Weinberg, R. A. The hallmarks of cancer. _Cell_ **100**, 57–70 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXks1CktA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10647931)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20hallmarks%20of%20cancer&journal=Cell&volume=100&pages=57-70&publication_year=2000&author=Hanahan%2CD&author=Weinberg%2CRA) 
    
60.  Christofori, G. & Semb, H. The role of the cell-adhesion molecule E-cadherin as a tumour-suppressor gene. _Trends Biochem. Sci._ **24**, 73–76 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXltFCgsr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10098402)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20the%20cell-adhesion%20molecule%20E-cadherin%20as%20a%20tumour-suppressor%20gene&journal=Trends%20Biochem.%20Sci.&volume=24&pages=73-76&publication_year=1999&author=Christofori%2CG&author=Semb%2CH) 
    
61.  Liu, X., Yin, Y., Wu, J. & Liu, Z. Structure and mechanism of an intramembrane liponucleotide synthetase central for phospholipid biosynthesis. _Nat. Commun._ **5**, 4244 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXktFentLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24968740)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structure%20and%20mechanism%20of%20an%20intramembrane%20liponucleotide%20synthetase%20central%20for%20phospholipid%20biosynthesis&journal=Nat.%20Commun.&volume=5&publication_year=2014&author=Liu%2CX&author=Yin%2CY&author=Wu%2CJ&author=Liu%2CZ) 
    
62.  Yariv, B. et al. Using evolutionary data to make sense of macromolecules with a ‘face-lifted’ ConSurf. _Protein Sci._ **32**, e4582 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXktFOiuro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36718848)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9942591)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Using%20evolutionary%20data%20to%20make%20sense%20of%20macromolecules%20with%20a%20%E2%80%98face-lifted%E2%80%99%20ConSurf&journal=Protein%20Sci.&volume=32&publication_year=2023&author=Yariv%2CB) 
    
63.  Hekkelman, M. L., de Vries, I., Joosten, R. P. & Perrakis, A. AlphaFill: enriching AlphaFold models with ligands and cofactors. _Nat. Methods_ **20**, 205–213 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36424442)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9911346)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=AlphaFill%3A%20enriching%20AlphaFold%20models%20with%20ligands%20and%20cofactors&journal=Nat.%20Methods&volume=20&pages=205-213&publication_year=2022&author=Hekkelman%2CML&author=Vries%2CI&author=Joosten%2CRP&author=Perrakis%2CA) 
    
64.  Chan, P. Y. et al. The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability across multiple tumor types. _Nat. Genet._ [https://doi.org/10.1038/s41588-025-02222-1](https://doi.org/10.1038/s41588-025-02222-1) (2025).
    
65.  Kalluri, R. & Zeisberg, M. Fibroblasts in cancer. _Nat. Rev. Cancer_ **6**, 392–401 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XjslGitLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16572188)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fibroblasts%20in%20cancer&journal=Nat.%20Rev.%20Cancer&volume=6&pages=392-401&publication_year=2006&author=Kalluri%2CR&author=Zeisberg%2CM) 
    
66.  Gurtner, G. C., Werner, S., Barrandon, Y. & Longaker, M. T. Wound repair and regeneration. _Nature_ **453**, 314–321 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlvV2qtbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18480812)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Wound%20repair%20and%20regeneration&journal=Nature&volume=453&pages=314-321&publication_year=2008&author=Gurtner%2CGC&author=Werner%2CS&author=Barrandon%2CY&author=Longaker%2CMT) 
    
67.  Stratman, A. N. et al. Anti-angiogenic effects of VEGF stimulation on endothelium deficient in phosphoinositide recycling. _Nat. Commun._ **11**, 1204 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlt1SgsLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32139674)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7058007)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Anti-angiogenic%20effects%20of%20VEGF%20stimulation%20on%20endothelium%20deficient%20in%20phosphoinositide%20recycling&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Stratman%2CAN) 
    
68.  Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. _Genome Biol._ **15**, 554 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25476604)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAGeCK%20enables%20robust%20identification%20of%20essential%20genes%20from%20genome-scale%20CRISPR%2FCas9%20knockout%20screens&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Li%2CW) 
    
69.  Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. _Nat. Methods_ **9**, 676–682 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKnurbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22743772)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fiji%3A%20an%20open-source%20platform%20for%20biological-image%20analysis&journal=Nat.%20Methods&volume=9&pages=676-682&publication_year=2012&author=Schindelin%2CJ) 
    
70.  Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. _Protein Sci._ **32**, e4792 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXit12ntLzK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37774136)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10588335)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20tools%20for%20structure%20building%20and%20analysis&journal=Protein%20Sci.&volume=32&publication_year=2023&author=Meng%2CEC) 
    
71.  Tyanova, S. et al. The Perseus computational platform for comprehensive analysis of (prote)omics data. _Nat. Methods_ **13**, 731–740 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVKntbnN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27348712)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Perseus%20computational%20platform%20for%20comprehensive%20analysis%20of%20%28prote%29omics%20data&journal=Nat.%20Methods&volume=13&pages=731-740&publication_year=2016&author=Tyanova%2CS) 
    
72.  Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high-throughput. _Nat. Methods_ **17**, 41–44 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXksFGjuw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31768060)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DIA-NN%3A%20neural%20networks%20and%20interference%20correction%20enable%20deep%20proteome%20coverage%20in%20high-throughput&journal=Nat.%20Methods&volume=17&pages=41-44&publication_year=2020&author=Demichev%2CV&author=Messner%2CCB&author=Vernardis%2CSI&author=Lilley%2CKS&author=Ralser%2CM) 
    
73.  Hanniford, D. et al. Epigenetic silencing of CDR1as drives IGF2BP3-mediated melanoma invasion and metastasis. _Cancer Cell_ **37**, 55–70.e15 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVektLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31935372)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7184928)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenetic%20silencing%20of%20CDR1as%20drives%20IGF2BP3-mediated%20melanoma%20invasion%20and%20metastasis&journal=Cancer%20Cell&volume=37&pages=55-70.e15&publication_year=2020&author=Hanniford%2CD) 
    
74.  Young, L., Sung, J. & Masters, J. R. Detection of mycoplasma in cell cultures. _Nat. Protoc._ **5**, 929–934 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlsVantb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20431538)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Detection%20of%20mycoplasma%20in%20cell%20cultures&journal=Nat.%20Protoc.&volume=5&pages=929-934&publication_year=2010&author=Young%2CL&author=Sung%2CJ&author=Masters%2CJR) 
    
75.  DepMap, B. DepMap 23Q4 Public. _figshare_ [https://doi.org/10.25452/figshare.plus.24667905.v2](https://doi.org/10.25452/figshare.plus.24667905.v2) (2023).
    
76.  Rahman, M. et al. Alternative preprocessing of RNA-sequencing data in the Cancer Genome Atlas leads to improved analysis results. _Bioinformatics_ **31**, 3666–3672 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlamsrfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26209429)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4804769)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Alternative%20preprocessing%20of%20RNA-sequencing%20data%20in%20the%20Cancer%20Genome%20Atlas%20leads%20to%20improved%20analysis%20results&journal=Bioinformatics&volume=31&pages=3666-3672&publication_year=2015&author=Rahman%2CM) 
    
77.  Eisenberg, E. & Levanon, E. Y. Human housekeeping genes, revisited. _Trends Genet._ **29**, 569–574 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVehurjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23810203)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20housekeeping%20genes%2C%20revisited&journal=Trends%20Genet.&volume=29&pages=569-574&publication_year=2013&author=Eisenberg%2CE&author=Levanon%2CEY) 
    
78.  Li, M., Sun, Q. & Wang, X. Transcriptional landscape of human cancers. _Oncotarget_ **8**, 34534–34551 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28427185)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5470989)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptional%20landscape%20of%20human%20cancers&journal=Oncotarget&volume=8&pages=34534-34551&publication_year=2017&author=Li%2CM&author=Sun%2CQ&author=Wang%2CX) 
    
79.  Zehir, A. et al. Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients. _Nat. Med._ **23**, 703–713 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXntFKitLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28481359)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5461196)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20landscape%20of%20metastatic%20cancer%20revealed%20from%20prospective%20clinical%20sequencing%20of%2010%2C000%20patients&journal=Nat.%20Med.&volume=23&pages=703-713&publication_year=2017&author=Zehir%2CA) 
    
80.  Apriamashvili, G. et al. Ubiquitin ligase STUB1 destabilizes IFNγ-receptor complex to suppress tumor IFNγ signaling. _Nat. Commun._ **13**, 1923 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xps1Cktb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35395848)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8993893)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ubiquitin%20ligase%20STUB1%20destabilizes%20IFN%CE%B3-receptor%20complex%20to%20suppress%20tumor%20IFN%CE%B3%20signaling&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Apriamashvili%2CG) 
    
81.  Yang, X. et al. A public genome-scale lentiviral expression library of human ORFs. _Nat. Methods_ **8**, 659–661 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXotV2htrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21706014)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3234135)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20public%20genome-scale%20lentiviral%20expression%20library%20of%20human%20ORFs&journal=Nat.%20Methods&volume=8&pages=659-661&publication_year=2011&author=Yang%2CX) 
    
82.  Vredevoogd, D. W. et al. Augmenting immunotherapy impact by lowering tumor TNF cytotoxicity threshold. _Cell_ **178**, 585–599.e15 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlKlu7vI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31303383)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Augmenting%20immunotherapy%20impact%20by%20lowering%20tumor%20TNF%20cytotoxicity%20threshold&journal=Cell&volume=178&pages=585-599.e15&publication_year=2019&author=Vredevoogd%2CDW) 
    
83.  Wiernicki, B. et al. Excessive phospholipid peroxidation distinguishes ferroptosis from other cell death modes including pyroptosis. _Cell Death Dis._ **11**, 922 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlCqur%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33110056)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7591475)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Excessive%20phospholipid%20peroxidation%20distinguishes%20ferroptosis%20from%20other%20cell%20death%20modes%20including%20pyroptosis&journal=Cell%20Death%20Dis.&volume=11&publication_year=2020&author=Wiernicki%2CB) 
    
84.  Nassour, J. et al. Telomere-to-mitochondria signalling by ZBP1 mediates replicative crisis. _Nature_ **614**, 767–773 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXivFWhtbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36755096)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9946831)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomere-to-mitochondria%20signalling%20by%20ZBP1%20mediates%20replicative%20crisis&journal=Nature&volume=614&pages=767-773&publication_year=2023&author=Nassour%2CJ) 
    
85.  Brinkman, E. K., Chen, T., Amendola, M. & Van Steensel, B. Easy quantitative assessment of genome editing by sequence trace decomposition. _Nucleic Acids Res._ **42**, e168 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25300484)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4267669)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Easy%20quantitative%20assessment%20of%20genome%20editing%20by%20sequence%20trace%20decomposition&journal=Nucleic%20Acids%20Res.&volume=42&publication_year=2014&author=Brinkman%2CEK&author=Chen%2CT&author=Amendola%2CM&author=Steensel%2CB) 
    
86.  Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. _Nat. Commun._ **9**, 5408 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimized%20libraries%20for%20CRISPR%E2%80%93Cas9%20genetic%20screens%20with%20multiple%20modalities&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Sanson%2CKR) 
    
87.  Joung, J. et al. Genome-scale CRISPR–Cas9 knockout and transcriptional activation screening. _Nat. Protoc._ **12**, 828–863 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXmtlaks7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28333914)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5526071)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-scale%20CRISPR%E2%80%93Cas9%20knockout%20and%20transcriptional%20activation%20screening&journal=Nat.%20Protoc.&volume=12&pages=828-863&publication_year=2017&author=Joung%2CJ) 
    
88.  Mootha, V. K. et al. PGC-1α-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. _Nat. Genet._ **34**, 267–273 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXkvFSrs7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12808457)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PGC-1%CE%B1-responsive%20genes%20involved%20in%20oxidative%20phosphorylation%20are%20coordinately%20downregulated%20in%20human%20diabetes&journal=Nat.%20Genet.&volume=34&pages=267-273&publication_year=2003&author=Mootha%2CVK) 
    
89.  Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. _Proc. Natl Acad. Sci. USA_ **102**, 15545–15550 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16199517)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=102&pages=15545-15550&publication_year=2005&author=Subramanian%2CA) 
    
90.  Szklarczyk, D. et al. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. _Nucleic Acids Res._ **47**, D607–D613 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30476243)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=STRING%20v11%3A%20protein%E2%80%93protein%20association%20networks%20with%20increased%20coverage%2C%20supporting%20functional%20discovery%20in%20genome-wide%20experimental%20datasets&journal=Nucleic%20Acids%20Res.&volume=47&pages=D607-D613&publication_year=2019&author=Szklarczyk%2CD) 
    
91.  Ghorasaini, M. et al. Cross-laboratory standardization of preclinical lipidomics using differential mobility spectrometry and multiple reaction monitoring. _Anal. Chem._ **93**, 16369–16378 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1aqtLjK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34859676)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8674878)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-laboratory%20standardization%20of%20preclinical%20lipidomics%20using%20differential%20mobility%20spectrometry%20and%20multiple%20reaction%20monitoring&journal=Anal.%20Chem.&volume=93&pages=16369-16378&publication_year=2021&author=Ghorasaini%2CM) 
    
92.  Ghorasaini, M. et al. Congruence and complementarity of differential mobility spectrometry and NMR spectroscopy for plasma lipidomics. _Metabolites_ **12**, 1030 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivV2ksbrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36355113)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9699282)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Congruence%20and%20complementarity%20of%20differential%20mobility%20spectrometry%20and%20NMR%20spectroscopy%20for%20plasma%20lipidomics&journal=Metabolites&volume=12&publication_year=2022&author=Ghorasaini%2CM) 
    
93.  Su, B. et al. A DMS shotgun lipidomics workflow application to facilitate high-throughput, comprehensive lipidomics. _J. Am. Soc. Mass. Spectrom._ **32**, 2655–2663 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Wnu7rO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34637296)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8985811)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20DMS%20shotgun%20lipidomics%20workflow%20application%20to%20facilitate%20high-throughput%2C%20comprehensive%20lipidomics&journal=J.%20Am.%20Soc.%20Mass.%20Spectrom.&volume=32&pages=2655-2663&publication_year=2021&author=Su%2CB) 
    
94.  Grootemaat, A. E. et al. Lipid and nucleocapsid N-protein accumulation in COVID-19 patient lung and infected cells. _Microbiol. Spectr._ **10**, e0127121 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35171025)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lipid%20and%20nucleocapsid%20N-protein%20accumulation%20in%20COVID-19%20patient%20lung%20and%20infected%20cells&journal=Microbiol.%20Spectr.&volume=10&publication_year=2022&author=Grootemaat%2CAE) 
    
95.  Bedard, M. et al. A terpene nucleoside from _M. tuberculosis_ induces lysosomal lipid storage in foamy macrophages. _J. Clin. Invest._ **133**, e161276 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20terpene%20nucleoside%20from%20M.%20tuberculosis%20induces%20lysosomal%20lipid%20storage%20in%20foamy%20macrophages&journal=J.%20Clin.%20Invest.&volume=133&publication_year=2023&author=Bedard%2CM) 
    
96.  Arnoldus, T. et al. CDS2 is a synthetic lethal target in mesenchymal-like cancers (proteomic dataset). _PRIDE_ [www.ebi.ac.uk/pride/archive/projects/PXD045833](https://www.ebi.ac.uk/pride/archive/projects/PXD045833) (2025).
    
97.  Arnoldus, T. et al. CDS2 is a synthetic lethal target in mesenchymal-like cancers (dataset). _figshare_ [https://doi.org/10.6084/m9.figshare.27951504](https://doi.org/10.6084/m9.figshare.27951504) (2025).
    
98.  Arnoldus, T. PeeperLab/cds2-synthetic-lethal-notebooks: v1.0.1. _Zenodo_ [https://doi.org/10.5281/zenodo.15194712](https://doi.org/10.5281/zenodo.15194712) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02221-2?format=refman&flavour=references)

Acknowledgements
----------------

We thank D. Adams and P. Y. Chan for sharing unpublished data and coordinating submission of the paper. We thank E. Hernando for sharing an SK-MEL-147 STR reference profile. We give a special thanks to A. Perrakis for advice on structure–function experiments, M. Cherevatenko and I. Darraz for experimental support, T. Jalving, M. Rodriguez Gomez, V. Kumar, J. S. Nieto and S. Schieven for support with data processing, M. Nulle for help with mouse work and G. Driessen for help with the graphic design. We are grateful to the NKI and MPI for Biology of Ageing facilities for supporting flow cytometry, imaging, cryogenic storage, mouse work and genomics. We acknowledge support for proteomics by the X-omics Initiative (project no. 184.034.019), part of the NWO National Roadmap for Large-Scale Research Infrastructures. D.S.P. is funded by the Oncode Institute, which is partly financed by the Dutch Cancer Society KWF. Research at the NKI is supported by institutional grants of the Dutch Cancer Society and the Dutch Ministry of Health, Welfare and Sport.

Author information
------------------

### Authors and Affiliations

1.  Division of Molecular Oncology and Immunology, Oncode Institute, Netherlands Cancer Institute, Amsterdam, The Netherlands
    
    Tim Arnoldus, Alex van Vliet, Adriaan F. H. de Groot, Qinglin Piao, Jing Dong, Susan E. van Hal-van Veen & Daniel S. Peeper
    
2.  Proteomics facility, Netherlands Cancer Institute, Amsterdam, The Netherlands
    
    Onno B. Bleijerveld & Maarten Altelaar
    
3.  Center for Proteomics and Metabolomics, Leiden University Medical Center, Leiden, The Netherlands
    
    Niek Blomberg & Martin Giera
    
4.  Cluster of Excellence Cellular Stress Responses in Aging-associated Diseases, University of Cologne, Faculty of Medicine and University Hospital Cologne, Institute for Metabolomics in Ageing, Cologne, Germany
    
    Désirée Schatton & Christian Frezza
    
5.  BioImaging Facility, Netherlands Cancer Institute, Amsterdam, The Netherlands
    
    Rolf Harkes
    
6.  Electron Microscopy Centre Amsterdam, Amsterdam University Medical Center, Amsterdam, The Netherlands
    
    Anita E. Grootemaat & Nicole N. van der Wel
    
7.  Mouse Clinic for Cancer and Aging Preclinical Intervention facility, Netherlands Cancer Institute, Amsterdam, The Netherlands
    
    Natalie Proost & Marieke van de Ven
    
8.  Department of Cell and Chemical Biology, Oncode Institute, Leiden University Medical Center, Leiden, The Netherlands
    
    Birol Cabukusta
    
9.  Cluster of Excellence Cellular Stress Responses in Aging-associated Diseases, University of Cologne, Faculty of Mathematics and Natural Sciences, Institute of Genetics, Cologne, Germany
    
    Christian Frezza
    
10.  Biomolecular Mass Spectrometry and Proteomics, Utrecht University, Utrecht, The Netherlands
    
    Maarten Altelaar
    

Authors

1.  Tim Arnoldus
    
    [View author publications](https://www.nature.com/search?author=Tim%20Arnoldus)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tim%20Arnoldus) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tim%20Arnoldus%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Alex van Vliet
    
    [View author publications](https://www.nature.com/search?author=Alex%20van%20Vliet)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alex%20van%20Vliet) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alex%20van%20Vliet%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Onno B. Bleijerveld
    
    [View author publications](https://www.nature.com/search?author=Onno%20B.%20Bleijerveld)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Onno%20B.%20Bleijerveld) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Onno%20B.%20Bleijerveld%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Adriaan F. H. de Groot
    
    [View author publications](https://www.nature.com/search?author=Adriaan%20F.%20H.%20de%20Groot)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Adriaan%20F.%20H.%20de%20Groot) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Adriaan%20F.%20H.%20de%20Groot%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Qinglin Piao
    
    [View author publications](https://www.nature.com/search?author=Qinglin%20Piao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qinglin%20Piao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qinglin%20Piao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Niek Blomberg
    
    [View author publications](https://www.nature.com/search?author=Niek%20Blomberg)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Niek%20Blomberg) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Niek%20Blomberg%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Désirée Schatton
    
    [View author publications](https://www.nature.com/search?author=D%C3%A9sir%C3%A9e%20Schatton)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=D%C3%A9sir%C3%A9e%20Schatton) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22D%C3%A9sir%C3%A9e%20Schatton%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Jing Dong
    
    [View author publications](https://www.nature.com/search?author=Jing%20Dong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jing%20Dong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jing%20Dong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Susan E. van Hal-van Veen
    
    [View author publications](https://www.nature.com/search?author=Susan%20E.%20van%20Hal-van%20Veen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Susan%20E.%20van%20Hal-van%20Veen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Susan%20E.%20van%20Hal-van%20Veen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Rolf Harkes
    
    [View author publications](https://www.nature.com/search?author=Rolf%20Harkes)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rolf%20Harkes) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rolf%20Harkes%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Anita E. Grootemaat
    
    [View author publications](https://www.nature.com/search?author=Anita%20E.%20Grootemaat)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anita%20E.%20Grootemaat) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anita%20E.%20Grootemaat%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Natalie Proost
    
    [View author publications](https://www.nature.com/search?author=Natalie%20Proost)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Natalie%20Proost) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Natalie%20Proost%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Birol Cabukusta
    
    [View author publications](https://www.nature.com/search?author=Birol%20Cabukusta)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Birol%20Cabukusta) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Birol%20Cabukusta%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Christian Frezza
    
    [View author publications](https://www.nature.com/search?author=Christian%20Frezza)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Christian%20Frezza) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Christian%20Frezza%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Marieke van de Ven
    
    [View author publications](https://www.nature.com/search?author=Marieke%20van%20de%20Ven)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marieke%20van%20de%20Ven) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marieke%20van%20de%20Ven%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Nicole N. van der Wel
    
    [View author publications](https://www.nature.com/search?author=Nicole%20N.%20van%20der%20Wel)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nicole%20N.%20van%20der%20Wel) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nicole%20N.%20van%20der%20Wel%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Martin Giera
    
    [View author publications](https://www.nature.com/search?author=Martin%20Giera)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Martin%20Giera) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Martin%20Giera%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Maarten Altelaar
    
    [View author publications](https://www.nature.com/search?author=Maarten%20Altelaar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maarten%20Altelaar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maarten%20Altelaar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Daniel S. Peeper
    
    [View author publications](https://www.nature.com/search?author=Daniel%20S.%20Peeper)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20S.%20Peeper) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20S.%20Peeper%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

T.A. designed the study, performed the analyses, performed the experiments and wrote the paper. A.v.V. performed the analyses. A.F.H.d.G., Q.P., J.D., D.S., B.C. and C.F. performed various experiments. O.B.B. and M.A. measured the proteome levels. N.B. and M.G. measured the lipidome levels. S.E.v.H., N.P. and M.v.d.V. performed the mouse work. R.H. performed the BODIPY analysis. A.E.G. and N.N.v.d.W. performed the electron microscopy. D.S.P. designed the study, supervised the study, wrote the paper and obtained the funding.

### Corresponding author

Correspondence to [Daniel S. Peeper](mailto:d.peeper@nki.nl).

Ethics declarations
-------------------

### Competing interests

T.A. and D.S.P. are named inventors on a patent application related to this work (no. WO2024240955A1). D.S.P. is co-founder, shareholder and advisor of Flindr Therapeutics, which is unrelated to the present study. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewers for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 _CDS1-2_ form a common cancer-associated SLI pair.](https://www.nature.com/articles/s41588-025-02221-2/figures/8)

**a**, Enrichment of previously functionally validated synthetic lethal interactions (_FAM50A-FAM50B_, _RPP25L-RPP25_, _EIF1AX-EIF1AY_, _DDX3X-DDX3Y_, _DNAJC19-DNAJC15_, _TTC7A-TTC7B, MCL1-BCL2L1_) for different sample sizes. The top 100 Pearson correlations between RNA and dependency data were plotted using all DepMap cancer cell lines or cancer type subsets. _n_ (cancer cell lines) = 913 (all), 122 (lung), 70 (CNS), 63 (skin), 40 (breast). **b**, Top: plot depicting Δ_CDS2_ lethality and _CDS1_ RNA expression for cancer cell lines (DepMap). Bottom: plot depicting Δ_FAM50A_ lethality and _FAM50B_ RNA expression for cancer cell lines (DepMap). _n_ = 913 cancer cell lines. **c**, Replication of the analysis and method of plotting presented in Fig. [1d](https://www.nature.com/articles/s41588-025-02221-2#Fig1) which used 21Q2 CERES data using 23Q4 Chronos data (DepMap). _n_ = 1,014 cancer cell lines, 10,005 patient samples (9,264 tumor). **d**, The average fold change in patient tumor samples compared to patient healthy samples (TCGA) for the anchor genes of the top ten predicted synthetic lethal pairs in Fig. [1d](https://www.nature.com/articles/s41588-025-02221-2#Fig1). _n_ = 10,005 patient samples (9,264 tumor). **e**, The RNA expression in patient tumor and healthy samples for downregulated anchor genes in panel **d** (TCGA). Dotted lines represent the quartile cut-offs. Kruskal-Wallis test followed by Dunn’s test was used. _n_ = 10,005 patient samples (9,264 tumor). **f**, For each cancer type the relative cancer mortality (SEER) was plotted against the frequency of Δ_CDS2_ lethal cell lines (DepMap). The fraction of Δ_CDS2_ lethal cell lines within each type is categorically depicted. _n_ (cancer cell lines) = 906 (40 breast, 122 lung, 97 blood, 92 brain, 64 skin, 45 soft tissue). **g**, For the top lineages from panel **f**, the calibration gene-normalized _CDS1_ expression in healthy tissue samples (GTEx), cancer cell lines (DepMap), Δ_CDS2_ lethal cancer cell lines and Δ_CDS2_ nonlethal cancer cell lines is presented. Dotted lines represent the quartile cut-offs. Kruskal-Wallis test followed by Dunn’s test was used. _n_ (healthy) = 578 (lung), 2,642 (brain), 929 (blood), 1,809 (skin). _n_ (cancer) = 122 (lung, 35 Δ_CDS2_ lethal), 75 (brain, 25 Δ_CDS2_ lethal), 48 (blood, 35 Δ_CDS2_ lethal), 63 (skin, 24 Δ_CDS2_ lethal). In **e** and **g**: \*\*\*_P_<0.001. ns, not significant.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM11)

### [Extended Data Fig. 2 _CDS1_ and _CDS2_ are synthetic lethal across cancer types.](https://www.nature.com/articles/s41588-025-02221-2/figures/9)

**a**, As an example, representative histograms for the flow cytometry data used to calculate lethality for NCI-H2030 in Fig. [2d](https://www.nature.com/articles/s41588-025-02221-2#Fig2) are presented. **b**, The 8-d results for the experiment in Fig. [2d](https://www.nature.com/articles/s41588-025-02221-2#Fig2). Two-way ANOVA followed up by Dunnett’s test was used. _n_ = 3 replicates for 7 cancer cell lines with 2 _CDS2_ sgRNAs. **c**, The percentage of cancer cells with frameshift upon sg_CDS2_ compared to sgControl determined using TIDE. _n_ = 3 replicates for 5 cancer cell lines. **d**, The 7-d results for the experiment in Fig. [2f](https://www.nature.com/articles/s41588-025-02221-2#Fig2). Two-way ANOVA followed by Šidák’s test was used. _n_ = 3 replicates for 4 cancer cell lines. **e**, Graph depicting a snapshot of the total protein normalized level of cleaved caspase-3 in apoptotic or control samples of BLM and SK-MEL-2 cell lines, as determined by quantitative western blotting (Abby). Apoptosis was induced by TPCA-1 + TNF for BLM cells and staurosporine for SK-MEL-2 cells. Two-way ANOVA followed by Šidák’s test was used. _n_ = 3 replicates for 2 cancer cell lines. **f**, Representative histograms for the flow cytometry data used to calculate synthetic lethality in Fig. [2h](https://www.nature.com/articles/s41588-025-02221-2#Fig2). **g**, Growth curves for the experiment in Fig. [2h](https://www.nature.com/articles/s41588-025-02221-2#Fig2). Note that the tumor growth curves are derived from cell mixes including both GFP-expressing and ectopically _CDS1_\-restored cells. Two-way ANOVA was used. Mean is presented by dark lines. _n_ = 6 NOD-scid _Il2rγ_\-null mice per group for 2 cancer cell lines. In **b**\-**e** and **g**: \*_P_<0.05, \*\*_P_<0.01, \*\*\*_P_<0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM12)

### [Extended Data Fig. 3 No common escape mechanism for Δ_CDS2_ lethality.](https://www.nature.com/articles/s41588-025-02221-2/figures/10)

**a**, Top: Colony formation assays with adherent cell lines used for the screen (crystal violets). Bottom: Cell survival assay with the suspension screen cell line (cell counts). _n_ = 1 for 5 cancer cell lines, an independent experiment was performed for quantification (Fig. [3b](https://www.nature.com/articles/s41588-025-02221-2#Fig3)). **b**, The average Δ_CDS2_ changes for knockouts of each other gene were plotted for the four _CDS1_\-negative cancer cell lines against the _CDS1_\-high cell line. The dotted line corresponds to the Δ_CDS2_ change required to compensate for the average lethality quantified in Fig. [3b](https://www.nature.com/articles/s41588-025-02221-2#Fig3). _n_ = 1 for 5 cancer cell lines at 200x coverage with 4 sgRNA per gene and 1,000 sgControls in the CRISPR library. **c-f**, Lethality rescue plotted against the Benjamini-Hochberg corrected _P_ values from MAGeCK for each gene in the separate screens. Each of these panels presents the results for a different cancer cell line. The rescue was calculated using average lethality quantified in Fig. [3b](https://www.nature.com/articles/s41588-025-02221-2#Fig3). The two genes significant in three cell lines (_LCMT1_ & _EP300_) and the two significant genes in two cell lines with the strongest rescue (_PPP6C_ & _PTPN11_) were marked. Note that outliers above 100% were not plotted. _n_ = 1 for 4 cancer cell lines at 200x coverage with 4 sgRNA per gene and 1,000 sgControls in the CRISPR library. **g**, As a quality control, each control arm was compared with the original library to detect essential genes (true positives) over non-essential genes (true negatives). The resulting area under the receiver operator curve (ROC-AUC) values for each screen are presented. A ROC-AUC above 90% of maximum is considered outstanding for CRISPR screens[86](https://www.nature.com/articles/s41588-025-02221-2#ref-CR86 "Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. Nat. Commun. 9, 5408 (2018)."). _n_ = 1 for 5 cancer cell lines at 200x coverage with 4 sgRNA per gene and 1,000 sgControls in the CRISPR library. **h**, Left: Plot depicting Δ_CDS2_ lethality and _CDS1_ RNA expression for cancer cell lines (DepMap). Cancer cell lines used for validation are marked. Right: The 14-d lethality of the marked cancer cell lines. Lethality was quantified using fluorescent tracker cells as depicted in Fig. [2c](https://www.nature.com/articles/s41588-025-02221-2#Fig2). Two-way ANOVA followed by Šidák’s test was used. _n_ = 1,014 cancer cell lines (DepMap, left) or 4 replicates for 4 cancer cell lines (right). In **h**: _\*\*\*__P_<0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM13)

### [Extended Data Fig. 4 Mesenchymal-like cancers depend on _CDS2_ for PI synthesis.](https://www.nature.com/articles/s41588-025-02221-2/figures/11)

**a**, _CDH1_ and _VIM_ expression in Δ_CDS2_ lethal, Δ_CDS2_ nonlethal, _CDS1_\-low (bottom quartile) and _CDS1_\-high (top quartile) cancer cell lines (DepMap). Mann-Whitney _U_ test was used. _n_ = 913 cancer cell lines (657 nonlethal, 256 lethal, 345 _CDS1_ low, 352 _CDS1_ high). **b**, The percentage of _CDS1_\-high cancer cell lines (DepMap) was plotted against the percentage of _VIM-_high or _CDH1_\-high cancer cell lines by type. A linear regression and Pearson correlation test were added. _n_ = 906 cancer cell lines. **c**, _CDS1_ promoter methylation in all cancer cell lines and the blood subtype (DepMap). Histograms show the distribution of _CDS1_ promoter methylation (top). Mann-Whitney _U_ test was used. _n_ = 792 cancer cell lines (72 blood). **d**, The gene set enrichment analysis (GSEA) results for Hallmark epithelial to mesenchymal transition. Results are from a GSEA on genome-wide _CDS1_ coexpression (Pearson’s r) using DepMap and TCGA data. The Benjamini-Hochberg corrected GSEA _P_ values were used. _n_ = 913 cancer cell lines, 11,069 patient samples. **e**, For 16 cancer types the calibration gene-normalized RNA expression in healthy tissue samples (GTEx) and cancer cell lines (DepMap) was compared. The cut-off for low expression of _CDS1_ or _CDH1_ expression was the bottom-tertile boundary for expression in healthy tissue samples of the same type (right). _CDH1_ and _VIM_ expression were plotted as a percentage of the highest expressing cancer type included (left). Kruskal-Wallis test followed by Dunn’s test was used by type. The Pearson correlation test was used across types. _n_ (healthy/cancer) = 2,642/70 (brain), 929/48 (blood), 653/8 (thyroid), 89/25 (kidney), 1,809/63 (skin), 226/22 (liver), 180/53 (ovarian), 578/122 (lung), 142/32 (uterus), 19/13 (cervix), 359/29 (stomach), 328/42 (pancreatic), 459/40 (breast), 1,445/28 (esophagus), 966/50 (colorectal), 245/5 (prostate). **f**, Left: the method for quantifying the dead cell fraction using the DNA-intercalating dye DRAQ7. Right: rescue of Δ_CDS2_ cell death upon overnight supplementation with 2 mM phosphatidylinositol (PI) started 11 d after transduction of BLM cells. One-way ANOVA followed by Šidák’s test was used. _n_ = 3 replicates for 1 cancer cell line. In **a**\-**f**: _\*__P_<0.05, \*\*_P_<0.01, \*\*\*_P_<0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM14)

### [Extended Data Fig. 5 Cholesterol production is induced upon _CDS1+CDS2_ ablation.](https://www.nature.com/articles/s41588-025-02221-2/figures/12)

**a**, Volcano plots presenting the Δ_CDS2_\-induced protein changes in five cancer cell lines. In addition to CDS2, the top and bottom five proteins significant in at least three out of four _CDS1_\-negative cancer cell lines were marked. _n_ = 4 replicates for 5 cancer cell lines. **b**, The top ten significantly enriched GO terms among Δ_CDS2_\-changed proteins in the four _CDS1_\-negative cancer cell lines. Benjamini-Hochberg corrected _P_ values from Stringdb were used. _n_ = 4 replicates for 4 cancer cell lines. **c**, Graph depicting a second independent experiment for SK-MEL-2 using a different instrument (_y_ axis, Astral). Data is plotted against the experiment presented in Fig. [5b](https://www.nature.com/articles/s41588-025-02221-2#Fig5) (_x_ axis). All plotted changes were significant in both experiments. Only the 13 cholesterol-related proteins from Fig. [5b](https://www.nature.com/articles/s41588-025-02221-2#Fig5) and CDS2 were plotted. Significance was determined by Student’s t-test. _n_ = 4 replicates for 1 cancer cell line in two independent experiments. **d**, For one _CDS1_\-high cell line _CDS1_ (sg_CDS1_, versus sgControl) was perturbed by CRISPR and the change in CDS1 protein level was plotted. Significance was determined by Student’s t-test. _n_ = 4 replicates for 1 cancer cell line. In **b** and **d**: _\*__P_<0.05, \*\*_P_<0.01, \*\*\*_P_<0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM15)

### [Extended Data Fig. 6 Lipid homeostasis requires either _CDS1_ or _CDS2_ expression.](https://www.nature.com/articles/s41588-025-02221-2/figures/13)

**a**, Δ_CDS2_ lipid changes, presented in alphabetic order. Only lipid classes detected in all samples were plotted. Stars indicate number of _CDS1_\-negative cancer cell lines with at least two-star significant lipid alterations compared to the _CDS1_\-high cancer cell line. For abbreviations see bottom of figure description. Two-way ANOVA followed up by Šidák’s test was used. _n_ = 4 replicates for 5 cancer cell lines. **b**, CDS2 pathway with Δ_CDS2_\-changed lipid classes in _CDS1_\-negative cancer cell lines marked with arrows. For abbreviations see bottom of figure description. Included enzymes are annotated on UniProt. **c**, Graphs depicting sg_CDS2_ lipid changes in a second independent experiment for SK-MEL-2 (_y_ axis). Key lipids from Fig. [6a](https://www.nature.com/articles/s41588-025-02221-2#Fig6) were plotted. Data is plotted against the Fig. [6a](https://www.nature.com/articles/s41588-025-02221-2#Fig6) data (_x_ axis). All plotted changes were significant. Two-way ANOVA followed up by Šidák’s test was used. _n_ = 4 replicates for 1 cancer cell line in 2 independent experiments. In **a**: Significance was indicated by one \* for each cell line with _P_< 0.01. Lipids: CE = cholesterol esters, Cerd18:0 = ceramide d18:0, Cerd18:1 = ceramide d18:1, DAG = diacylglycerol, FFA = free fatty acids, HexCER = hexosylceramides, LPC = lysophosphatidylcholine, LPE = lysophosphatidylethanolamine, PA = phosphatidic acid, PC = phosphatidylcholine, PE = phosphatidylethanolamine, PG = phosphatidylglycerol, PI = phosphatidylinositol, PS = phosphatidylserine, SM = sphingomyelin, TAG = triglycerides. Enzymes: PEMT = phosphatidylethanolamine n-methyltransferase, CEPT1 = choline/ethanolamine phosphotransferase 1, DGATs = diacylglycerol o-acyltransferases, DGKs = diacylglycerol kinases, CDS = cytidine diphosphate diacylglycerol synthase, TAM41 = mitochondrial phosphatidate cytidylyltransferase, CDIPT = cytidine diphosphate diacylglycerol synthase inositol-3-phosphatidyltransferase, PGS1 = phosphatidylglycerophosphate synthase 1, PTPMT1 = protein tyrosine phosphatase mitochondrial 1.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM16)

### [Extended Data Fig. 7 CDS2 dosage and catalytic activity govern synthetic lethality.](https://www.nature.com/articles/s41588-025-02221-2/figures/14)

**a**, Graph depicting RNA quantification by qPCR of 10-d _CDS2_ knockdown in ectopic _CDS1_\-restored BLM cells from the experiment depicted in Fig. [7b](https://www.nature.com/articles/s41588-025-02221-2#Fig7). Two-way ANOVA followed up by Šidák’s test was used. _n_ = 3 replicates for 1 cancer cell line for 3 sh_CDS2_. **b**, Plot depicting time until palpable tumors were formed for the experiment in Fig. [7c](https://www.nature.com/articles/s41588-025-02221-2#Fig7). Student’s t-test was used. _n_ = 9 NOD-scid _Il2rγ_\-null mice per group. **c**, Alignment of a part of the human CDS2 amino acid sequence with bacterial TmCdsA showing they both contain an aspartic acid (red box). Added on top is the conservation across 150 species of each of the presented amino acids as determined by ConSurf[62](https://www.nature.com/articles/s41588-025-02221-2#ref-CR62 "Yariv, B. et al. Using evolutionary data to make sense of macromolecules with a ‘face-lifted’ ConSurf. Protein Sci. 32, e4582 (2023)."). **d**, The computationally predicted (Alphafill) ion binding to the aspartic acid highlighted in panel **c** for bacterial TmCdsA and human CDS2[63](https://www.nature.com/articles/s41588-025-02221-2#ref-CR63 "Hekkelman, M. L., de Vries, I., Joosten, R. P. & Perrakis, A. AlphaFill: enriching AlphaFold models with ligands and cofactors. Nat. Methods 20, 205–213 (2022)."). The bacterial structure was validated in a previous report. In **a**: _\*\*\*__P_<0.001.

[Source data](https://www.nature.com/articles/s41588-025-02221-2#MOESM17)

### [Extended Data Fig. 8 Gating strategies.](https://www.nature.com/articles/s41588-025-02221-2/figures/15)

**a**, The gating strategy for in vitro GFP or mCherry mixing experiments. **b**, The gating strategy for the in vivo mixing experiment. **c**, The gating strategy for dead cell quantification.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM1_ESM.pdf)

Supplementary Note A detailed sample- and data-processing protocol for generating the proteomic results.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM2_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM3_ESM.xlsx)

Supplementary Table 1 Synthetic lethality scores and cancer-specificity scores for the top synthetic lethal gene pairs. Supplementary Table 2 The CRISPR screen data. Supplementary Table 3 The proteomic data. Supplementary Table 4 The lipidomic data. Supplementary Table 5 STR profiles generated from the cell lines used in the present study. Supplementary Table 6 Methodological details for SEER lineage matching. Supplementary Table 7 Primer, sgRNA and other sequences used in the present study.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM4_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM5_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM6_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM7_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM8_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM9_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM10_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM11_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Extended Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM12_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM13_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM14_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM15_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM16_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_MOESM17_ESM.xlsx)

Plotted data and statistics: unprocessed data and code are available on repositories.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Cytidine%20diphosphate%20diacylglycerol%20synthase%202%20is%20a%20synthetic%20lethal%20target%20in%20mesenchymal-like%20cancers&author=Tim%20Arnoldus%20et%20al&contentID=10.1038%2Fs41588-025-02221-2&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-04&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02221-2)

### Cite this article

Arnoldus, T., van Vliet, A., Bleijerveld, O.B. _et al._ Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers. _Nat Genet_ **57**, 1659–1671 (2025). https://doi.org/10.1038/s41588-025-02221-2

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02221-2?format=refman&flavour=citation)

-   Received: 17 November 2023
    
-   Accepted: 09 May 2025
    
-   Published: 04 July 2025
    
-   Issue Date: July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02221-2
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02221-2.pdf)
