> id: https://www.nature.com/articles/s41588-025-02267-2

> link: https://www.nature.com/articles/s41588-025-02267-2

> title: Large-scale genome-wide analyses of stuttering

# Large-scale genome-wide analyses of stuttering
_Published on Mon, 28 Jul 2025 00:00:00 GMT_

Abstract
--------

Developmental stuttering is a highly heritable, common speech condition characterized by prolongations, blocks and repetitions of speech. Although stuttering is highly heritable and enriched within families, the genetic architecture is largely understudied. We reasoned that there are both shared and distinct genetic variants impacting stuttering risk within sex and ancestry groups. To test this idea, we performed eight primary genome-wide association analyses of self-reported stuttering that were stratified by sex and ancestry, as well as secondary meta-analyses of more than one million individuals (99,776 cases and 1,023,243 controls), identifying 57 unique loci. We validated the genetic risk of self-reported stuttering in two independent datasets. We further show genetic similarity of stuttering with autism, depression and impaired musical rhythm across sexes, with follow-up analyses highlighting potentially causal relationships among these traits. Our findings provide well-powered insights into genetic factors underlying stuttering.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-46128-8/MediaObjects/41467_2024_46128_Fig1_HTML.png)

### [Structural models of genome-wide covariance identify multiple common dimensions in autism](https://www.nature.com/articles/s41467-024-46128-8?fromPaywallRec=false)

Article Open access 27 February 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-022-26845-0/MediaObjects/41598_2022_26845_Fig1_HTML.png)

### [Genome-wide association study of school grades identifies genetic overlap between language ability, psychopathology and creativity](https://www.nature.com/articles/s41598-022-26845-0?fromPaywallRec=false)

Article Open access 09 January 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41380-024-02409-8/MediaObjects/41380_2024_2409_Fig1_HTML.png)

### [Genetic architecture of childhood speech disorder: a review](https://www.nature.com/articles/s41380-024-02409-8?fromPaywallRec=false)

Article Open access 16 February 2024

Main
----

Developmental stuttering is the most common fluency disorder, with more than 400 million people affected worldwide and a lifetime prevalence of 5–8% among global populations[1](https://www.nature.com/articles/s41588-025-02267-2#ref-CR1 "Yairi, E. & Ambrose, N. Epidemiology of stuttering: 21st century advances. J. Fluen. Disord. 38, 66–87 (2013)."). Stuttering is characterized, in part, by syllable and word repetitions, sound prolongations and involuntary breaks between words, called blocks, that disrupt the forward movement of speech[2](https://www.nature.com/articles/s41588-025-02267-2#ref-CR2 "Chang, S. E. et al. Stuttering: our current knowledge, research opportunities, and ways to address critical gaps. Neurobiol. Lang. (Camb.) 6, nol_a_00162 (2025)."). The onset of developmental stuttering typically occurs during childhood between ages 2 and 5, and an estimated 80% of children who stutter will spontaneously recover, with or without the aid of speech therapy[3](https://www.nature.com/articles/s41588-025-02267-2#ref-CR3 "Andrews, G. & Harris, M. The Syndrome of Stuttering. Clinics in Developmental Medicine No. 17 (William Heinemann Medical Books, 1964)."),[4](https://www.nature.com/articles/s41588-025-02267-2#ref-CR4 "Månsson, H. Childhood stuttering. J. Fluen. Disord. 25, 47–57 (2000)."),[5](https://www.nature.com/articles/s41588-025-02267-2#ref-CR5 "Yairi, E. & Ambrose, N. A longitudinal study of stuttering in children: a preliminary report. J. Speech Hear. Res. 35, 755–760 (1992)."). Although many individuals who stutter seek therapies, including speech interventions, behavior modification, cognitive interventions and technology-based feedback interventions to manage various aspects of stuttering[6](https://www.nature.com/articles/s41588-025-02267-2#ref-CR6 "Baxter, S. et al. The state of the art in non‐pharmacological interventions for developmental stuttering. Part 1: a systematic review of effectiveness. Int. J. Lang. Commun. Disord. 50, 676–718 (2015)."), outside of spontaneous recovery, stuttering does not have a known cure.

For those who experience persistent stuttering into adolescence and adulthood, the impact can be profound and life-long. People who stutter often describe negative perceptions of identity and self-worth, and reduced overall quality of life[7](https://www.nature.com/articles/s41588-025-02267-2#ref-CR7 "Daniels, D. E. & Gabel, R. M. The impact of stuttering on identity construction. Top. Lang. Disord. 24, 200–215 (2004)."). Young people who stutter experience increased bullying and decreased classroom participation, and they report a more negative educational experience; stuttering is associated with depression and suicidal ideation in this population[7](https://www.nature.com/articles/s41588-025-02267-2#ref-CR7 "Daniels, D. E. & Gabel, R. M. The impact of stuttering on identity construction. Top. Lang. Disord. 24, 200–215 (2004)."),[8](https://www.nature.com/articles/s41588-025-02267-2#ref-CR8 "Daniels, D. E., Gabel, R. M. & Hughes, S. Recounting the K–12 school experiences of adults who stutter: a qualitative analysis. J. Fluen. Disord. 37, 71–82 (2012)."),[9](https://www.nature.com/articles/s41588-025-02267-2#ref-CR9 "McAllister, J., Collier, J. & Shepstone, L. The impact of adolescent stuttering on educational and employment outcomes: evidence from a birth cohort study. J. Fluen. Disord. 37, 106–121 (2012)."). For adults, stuttering can negatively impact employability, perceived job performance, socioeconomic status and mental and social wellbeing[7](https://www.nature.com/articles/s41588-025-02267-2#ref-CR7 "Daniels, D. E. & Gabel, R. M. The impact of stuttering on identity construction. Top. Lang. Disord. 24, 200–215 (2004)."),[8](https://www.nature.com/articles/s41588-025-02267-2#ref-CR8 "Daniels, D. E., Gabel, R. M. & Hughes, S. Recounting the K–12 school experiences of adults who stutter: a qualitative analysis. J. Fluen. Disord. 37, 71–82 (2012)."),[10](https://www.nature.com/articles/s41588-025-02267-2#ref-CR10 "Klein, J. F. & Hood, S. B. The impact of stuttering on employment opportunities and job performance. J. Fluen. Disord. 29, 255–273 (2004)."),[11](https://www.nature.com/articles/s41588-025-02267-2#ref-CR11 "Craig, A., Blumgart, E. & Tran, Y. The impact of stuttering on the quality of life in adults who stutter. J. Fluen. Disord. 34, 61–71 (2009).").

Developmental stuttering is also sexually dimorphic. At stuttering onset, the male-to-female ratio is approximately even (between 1:1 and 2:1)[4](https://www.nature.com/articles/s41588-025-02267-2#ref-CR4 "Månsson, H. Childhood stuttering. J. Fluen. Disord. 25, 47–57 (2000)."), but stuttering is substantially more common in males than females in adolescent and adult populations (approximately 4:1)[1](https://www.nature.com/articles/s41588-025-02267-2#ref-CR1 "Yairi, E. & Ambrose, N. Epidemiology of stuttering: 21st century advances. J. Fluen. Disord. 38, 66–87 (2013)."),[12](https://www.nature.com/articles/s41588-025-02267-2#ref-CR12 "Craig, A., Hancock, K., Tran, Y., Craig, M. & Peters, K. Epidemiology of stuttering in the community across the entire life span. J. Speech Lang. Hear. Res. 45, 1097–1105 (2002).") owing to differences in the rate of recovery by sex[13](https://www.nature.com/articles/s41588-025-02267-2#ref-CR13 "Singer, C. M., Hessling, A., Kelly, E. M., Singer, L. & Jones, R. M. Clinical characteristics associated with stuttering persistence: a meta-analysis. J. Speech Lang. Hear. Res. 63, 2995–3018 (2020)."),[14](https://www.nature.com/articles/s41588-025-02267-2#ref-CR14 "Singer, C. M., Otieno, S., Chang, S.-E. & Jones, R. M. Predicting persistent developmental stuttering using a cumulative risk approach. J. Speech Lang. Hear. Res. 65, 70–95 (2022)."). However, the mechanisms leading to differences in prevalence and the rate of recovery by sex are not known.

Studies of stuttering within families, twins and population isolates provide overwhelming evidence for a strong genetic influence on stuttering risk, with heritability estimates ranging from 0.42 to 0.84 (refs. [15](https://www.nature.com/articles/s41588-025-02267-2#ref-CR15 "Shugart, Y. Y. et al. Results of a genome-wide linkage scan for stuttering. Am. J. Med. Genet. 124A, 133–135 (2004)."),[16](https://www.nature.com/articles/s41588-025-02267-2#ref-CR16 "Riaz, N. et al. Genomewide significant linkage to stuttering on chromosome 12. Am. J. Hum. Genet. 76, 647–651 (2005)."),[17](https://www.nature.com/articles/s41588-025-02267-2#ref-CR17 "Suresh, R. et al. New complexities in the genetics of stuttering: significant sex-specific linkage signals. Am. J. Hum. Genet. 78, 554–563 (2006)."),[18](https://www.nature.com/articles/s41588-025-02267-2#ref-CR18 "Wittke-Thompson, J. K. et al. Genetic studies of stuttering in a founder population. J. Fluen. Disord. 32, 33–50 (2007)."),[19](https://www.nature.com/articles/s41588-025-02267-2#ref-CR19 "Kang, C. et al. Mutations in the lysosomal enzyme-targeting pathway and persistent stuttering. N. Engl. J. Med. 362, 677–685 (2010)."),[20](https://www.nature.com/articles/s41588-025-02267-2#ref-CR20 "Lan, J. et al. Association between dopaminergic genes (SLC6A3 and DRD2) and stuttering among Han Chinese. J. Hum. Genet. 54, 457–460 (2009)."),[21](https://www.nature.com/articles/s41588-025-02267-2#ref-CR21 "Domingues, C. E. F. et al. A genetic linkage study in Brazil identifies a new locus for persistent developmental stuttering on chromosome 10. Genet. Mol. Res. 13, 2094–2101 (2014)."),[22](https://www.nature.com/articles/s41588-025-02267-2#ref-CR22 "Mohammadi, H. et al. Sex steroid hormones and sex hormone binding globulin levels, CYP17 MSP AI (−34 T:C) and CYP19 codon 39 (Trp:Arg) variants in children with developmental stuttering. Brain Lang. 175, 47–56 (2017)."),[23](https://www.nature.com/articles/s41588-025-02267-2#ref-CR23 "Raza, M. H., Amjad, R., Riazuddin, S. & Drayna, D. Studies in a consanguineous family reveal a novel locus for stuttering on chromosome 16q. Hum. Genet. 131, 311–313 (2012)."),[24](https://www.nature.com/articles/s41588-025-02267-2#ref-CR24 "Raza, M. H. et al. Association between rare variants in AP4E1, a component of intracellular trafficking, and persistent stuttering. Am. J. Hum. Genet. 97, 715–725 (2015)."),[25](https://www.nature.com/articles/s41588-025-02267-2#ref-CR25 "van Beijsterveldt, C. E. M., Felsenfeld, S. & Boomsma, D. I. Bivariate genetic analyses of stuttering and nonfluency in a large sample of 5-year-old twins. J. Speech Lang. Hear. Res. 53, 609–619 (2010)."),[26](https://www.nature.com/articles/s41588-025-02267-2#ref-CR26 "Fagnani, C., Fibiger, S., Skytthe, A. & Hjelmborg, J. V. B. Heritability and environmental effects for self-reported periods with stuttering: a twin study from Denmark. Logoped. Phoniatr. Vocol. 36, 114–120 (2011).")). To date, family studies have identified six candidate causal stuttering genes: _GNPTAB_, _GNPTG_ and _NAGPA_[19](https://www.nature.com/articles/s41588-025-02267-2#ref-CR19 "Kang, C. et al. Mutations in the lysosomal enzyme-targeting pathway and persistent stuttering. N. Engl. J. Med. 362, 677–685 (2010)."),[27](https://www.nature.com/articles/s41588-025-02267-2#ref-CR27 "Kazemi, N., Estiar, M. A., Fazilaty, H. & Sakhinia, E. Variants in GNPTAB, GNPTG and NAGPA genes are associated with stutterers. Gene 647, 93–100 (2018)."); _DRD2_ (ref. [20](https://www.nature.com/articles/s41588-025-02267-2#ref-CR20 "Lan, J. et al. Association between dopaminergic genes (SLC6A3 and DRD2) and stuttering among Han Chinese. J. Hum. Genet. 54, 457–460 (2009).")); _AP4E1_ (ref. [24](https://www.nature.com/articles/s41588-025-02267-2#ref-CR24 "Raza, M. H. et al. Association between rare variants in AP4E1, a component of intracellular trafficking, and persistent stuttering. Am. J. Hum. Genet. 97, 715–725 (2015).")); and _CYP17A1_ (ref. [22](https://www.nature.com/articles/s41588-025-02267-2#ref-CR22 "Mohammadi, H. et al. Sex steroid hormones and sex hormone binding globulin levels, CYP17 MSP AI (−34 T:C) and CYP19 codon 39 (Trp:Arg) variants in children with developmental stuttering. Brain Lang. 175, 47–56 (2017).")); however, these findings have not replicated in other families and explain little of the genetic heritability of stuttering in populations[28](https://www.nature.com/articles/s41588-025-02267-2#ref-CR28 "Kang, C. et al. Evaluation of the association between polymorphisms at the DRD2 locus and stuttering. J. Hum. Genet. 56, 472–473 (2011)."),[29](https://www.nature.com/articles/s41588-025-02267-2#ref-CR29 "Frigerio Domingues, C. E. et al. Are variants in sex hormone metabolizing genes associated with stuttering? Brain Lang. 191, 28–30 (2019)."). Two prior studies identified two genome-wide significant loci that confer stuttering risk at the population level[30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022)."),[31](https://www.nature.com/articles/s41588-025-02267-2#ref-CR31 "Shaw, D. M. et al. Phenome risk classification enables phenotypic imputation and gene discovery in developmental stuttering. Am. J. Hum. Genet. 108, 2271–2283 (2021)."). These previous investigations, leveraging both family data and global outbred populations, demonstrated that stuttering genetic risk factors are complex and involve both rare, familial and common variation. However, larger sample sizes are needed to elucidate ancestry-specific genetic risk factors for this common complex trait, especially to examine sexual dimorphism. Furthermore, models that leverage genetic risk markers (that is, polygenic risk scores (PRS), genetic correlation analysis and causal inference models) may illuminate the broader clinical impact of the genetic risk of stuttering.

Here, to identify both shared and distinct signals impacting stuttering risk, we report the results of eight primary ancestry-specific and sex-specific genome-wide association studies (GWAS) of stuttering and secondary meta-analyses in samples totaling more than 1.1 million individuals (99,776 cases). These analyses are well-powered to detect stuttering risk alleles with modest effect size and explore shared and distinct genetic effects across genetic ancestry-specific and sex-specific groups through cross-strata look-ups and meta-analyses.

Overall, this study reveals the complex genetic architecture of stuttering, identifying 24 signals in the primary ancestry-specific and sex-specific analyses and 63 signals in secondary meta-analyses for self-reported stuttering, mapping to 57 distinct loci. These loci met a two-tiered multiple test correction threshold. First, we applied a false discovery rate threshold of 5% across all eight primary analyses, and second, a traditional _P_ value threshold of 5 × 10−8. We validated the observed genetic effects in two independent datasets, including an international clinically ascertained stuttering cohort called the International Stuttering Project (ISP)[30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022).") and a cohort of self-reported stuttering in the National Longitudinal Study of Adolescent to Adult Health (Add Health)[32](https://www.nature.com/articles/s41588-025-02267-2#ref-CR32 "Harris, K. M. et al. Cohort profile: the National Longitudinal Study of Adolescent to Adult Health (Add Health). Int. J. Epidemiol. 48, 1415–1415k (2019)."). We then leveraged our results to explore genetic correlations between stuttering and its comorbidities. Together, these advances inform our understanding of the molecular etiology of stuttering and lay groundwork for the future of precision care in developmental speech disorders.

Results
-------

### Study overview

We performed eight primary non-overlapping GWAS of self-reported stuttering that were stratified by sex and genetic ancestry in samples from 23andMe, Inc. Stuttering status was based on 23andMe participant responses to the survey question ‘Have you ever had a stammer or stutter?’ The decision to stratify by sex and genetic ancestry group was driven by several factors (see the [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1) for a detailed description of how genetic ancestry and sex groups were defined). In addition to these primary analyses, we report secondary results of meta-analyses across sex by ancestry, meta-analyses across ancestry by sex and a meta-analysis of all eight primary GWAS (see Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02267-2#Fig5) for an overview of analyses).

### GWAS design

In aggregate, the datasets included 99,776 participants responding ‘yes’ to the question ‘Have you ever had a stammer or stutter?’ (cases, 48,217 males; 51,559 females) and 1,023,243 participants responding ‘no’ (controls, 392,414 males; 630,829 females; Table [1](https://www.nature.com/articles/s41588-025-02267-2#Tab1)). Overall, the study cohort is 60.76% female. Male stuttering prevalence in this dataset is 12.29% and female stuttering prevalence is 8.17%. Distribution by age group is shown in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM3), with age defined as the current age at the time the analysis was conducted. We divided our sample broadly into four continental ancestry groups: African ancestry (AFR), East Asian ancestry (EAS), European ancestry (EUR) and Latino/Admixed American ancestry (AMR), which were defined through genetic analysis[33](https://www.nature.com/articles/s41588-025-02267-2#ref-CR33 "Durand, E. Y., Do, C. B., Mountain, J. L. & Macpherson, J. M. Ancestry composition: a novel, efficient pipeline for ancestry deconvolution. Preprint at 
https://doi.org/10.1101/010512
(2014).") (see Supplementary Table [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) for sample sizes by ancestries and [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1) for description of genetic ancestry determination).

**Table 1 Demographics of study participants**

### Genetic correlations across ancestry-specific and sex-specific GWAS

In the primary analyses of each ancestry-specific and sex-specific GWAS (eight total), we considered autosomal and X chromosome variants that were successfully imputed across all platforms and reached our quality control thresholds (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). We estimated the genetic correlation between the EUR male and EUR female analyses and EAS male and EAS female analyses using linkage disequilibrium score regression (LDSC)[34](https://www.nature.com/articles/s41588-025-02267-2#ref-CR34 "Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat. Genet. 47, 291–295 (2015)."),[35](https://www.nature.com/articles/s41588-025-02267-2#ref-CR35 "Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Nat. Genet. 47, 1236–1241 (2015).") (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Statistically significant genetic correlation between sexes was only observed for the EUR population (_r_g = 0.8952, _P_ < 10−50). Despite this finding, we observed no overlap in genome-wide significant hits between the EUR male and EUR female analyses (Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2)). Comparisons of results across ancestry through concordance analysis can be found in the [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1).

**Table 2 Sentinel loci from primary ancestry- and sex-specific GWAS**

### Ancestry- and sex-specific GWAS

From the eight genetic ancestry-specific and sex-specific GWAS, we identified 24 loci (Supplementary Figs. [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[24](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)) associated with stuttering that surpassed a false discovery rate threshold of 5% across all eight primary analyses as well as a conventional genome-wide significance threshold of _P_ < 5 × 10−8. The identified loci included nine loci from the EUR female study, ten loci from the EUR male study, three loci from the AFR male study and one locus each from the AMR female and AMR male studies (Fig. [1](https://www.nature.com/articles/s41588-025-02267-2#Fig1), Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2) and Extended Data Fig. [2a–h](https://www.nature.com/articles/s41588-025-02267-2#Fig6)). No loci reached genome-wide significance in the AFR female GWAS, the EAS female GWAS or the EAS male GWAS. Sentinel hits from the EUR male GWAS implicated _VRK2_, _CAMTA1_, _MYO16_, _MMAB_, _CTNND2_, _SEMA6D_, _IRS2_ and _COL14A1_ as the most likely impacted functional genes (Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2) and Supplementary Figs. [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[10](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). Sentinel hits from the EUR female GWAS implicated _SLC39A8_, _DCC_, _SRPK2_, _NMUR2_, _TSHZ2_ and _MITF_ as the most likely impacted functional genes (Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2) and Supplementary Figs. [11](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[19](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). Sentinel hits from the AFR male GWAS implicated _PTPRQ_, _SHISA2_ and _CYTH4_ as likely functional genes; AMR female GWAS implicated _SORCS1_; and AMR male GWAS implicated _RGCC_ (Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2) and Supplementary Figs. [20](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[24](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). Across all eight ancestry-specific and sex-specific GWAS, no loci on the X chromosome surpassed genome-wide significance. We also present findings from secondary sex-combined meta-analyses, ancestry-combined meta-analyses and a meta-analysis of all sex and ancestry groups (see Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02267-2#Fig5) for study overview and Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02267-2#Fig7), [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15) and [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1) for heterogeneity of effect size estimates in the eight primary GWAS).

**Fig. 1: Miami plot of EUR female and EUR male GWAS.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02267-2/figures/1)

The EUR female association study (top panel) included 570,071 total samples (40,137 self-reported stuttering cases) and 29,449,463 autosomal variants. Nine loci reached genome-wide significance (dotted line, _P_ < 5.00 × 10−8) through logistic regression (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). The EUR male association study (bottom panel) included 374,279 total samples (38,257 self-reported stuttering cases) and 29,409,446 autosomal variants. Ten loci reached genome-wide significance (dotted line, _P_ < 5.00 × 10−8) through logistic regression (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). The _x_ axis represents chromosome base pair coordinates in human genome build 37, and the _y_ axis represents observed −log10(_P_) for each analysis. Annotated genes for each GWAS are the predicted functional gene for each locus (when available) according to the Open Targets Genetics V2G pipeline (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)).

### Sex-combined, ancestry-specific meta-analyses

To better understand ancestry-specific effects across sexes, we performed sex-combined, ancestry-specific meta-analyses in METAL[36](https://www.nature.com/articles/s41588-025-02267-2#ref-CR36 "Willer, C. J., Li, Y. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics 26, 2190–2191 (2010)."). We identified 36 loci associated with stuttering at the conventional significance threshold of _P_ < 5 × 10−8 (Supplementary Figs. [25](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[60](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). Specifically, 28 loci were associated with stuttering in the sex-combined EUR analysis (Table [3](https://www.nature.com/articles/s41588-025-02267-2#Tab3), Extended Data Fig. [4a](https://www.nature.com/articles/s41588-025-02267-2#Fig8) and Supplementary Figs. [25](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[52](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)), seven loci were associated with stuttering in the sex-combined AMR analysis (Table [3](https://www.nature.com/articles/s41588-025-02267-2#Tab3), Extended Data Fig. [4b](https://www.nature.com/articles/s41588-025-02267-2#Fig8) and Supplementary Figs. [53](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[59](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)) and one locus was associated with stuttering in the sex-combined EAS analysis (Table [3](https://www.nature.com/articles/s41588-025-02267-2#Tab3), Extended Data Fig. [4c](https://www.nature.com/articles/s41588-025-02267-2#Fig8) and Supplementary Fig. [60](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). No loci reached the conventional significance threshold in the sex-combined AFR analysis (Extended Data Fig. [4d](https://www.nature.com/articles/s41588-025-02267-2#Fig8)).

**Table 3 Sentinel loci from secondary sex-combined, ancestry-specific meta-analysis**

Sentinel hits from the sex-combined EUR meta-analysis implicated _SLC39A8_, _VRK2_, _CAMTA1_, _LRP1B_, _PTBP2_, _GRM5_, _CTNND2_, _CREB3L4_, _SEMA6D_, _KCTD10_, _UBAP2_, _LYSMD4_, _MAP2K6_, _ARMC2_, _SOX5_, _ZNF567_, _KCNH8_, _ADGRL3_, _TMEM71_, _SGCD_, _MET_ and _SRPK2_ as the most likely impacted functional genes (Table [3](https://www.nature.com/articles/s41588-025-02267-2#Tab3)). Sentinel hits from the sex-combined AMR meta-analysis implicated _LRRC69_, _PACRGL_, _CLRN2_ and _C16orf95_ as the most likely impacted functional genes (Table [3](https://www.nature.com/articles/s41588-025-02267-2#Tab3)). Sentinel hits from the sex-combined EAS meta-analysis implicated _ADCY5_ as a likely functional gene (Table [3](https://www.nature.com/articles/s41588-025-02267-2#Tab3)).

### Ancestry-combined, sex-specific meta-analyses

We also aggregated association summary statistics by sex across ancestry groups by multi-ancestry meta-regression, using MR-MEGA[37](https://www.nature.com/articles/s41588-025-02267-2#ref-CR37 "Mägi, R. et al. Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Hum. Mol. Genet. 26, 3639–3650 (2017)."), and identified five loci in our female-specific meta-analysis (Table [4](https://www.nature.com/articles/s41588-025-02267-2#Tab4), Extended Data Fig. [5a,b](https://www.nature.com/articles/s41588-025-02267-2#Fig9) and Supplementary Figs. [61](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[65](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)) and three loci in our male-specific meta-analysis (Table [4](https://www.nature.com/articles/s41588-025-02267-2#Tab4), Extended Data Fig. [5a,c](https://www.nature.com/articles/s41588-025-02267-2#Fig9) and Supplementary Figs. [66](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[68](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)).

**Table 4 Sentinel loci from secondary ancestry-combined, sex-specific meta-analyses and ancestry-combined and sex-combined meta-analysis**

Sentinel hits from the female-specific meta-analysis implicated _SRPK2_, _ZFP64_, _CBLN4_ and _DCC_ as the most likely impacted functional genes (Table [4](https://www.nature.com/articles/s41588-025-02267-2#Tab4)). Sentinel hits from the male-specific meta-analysis implicated _CAMTA1_, _VRK2_ and _KCTD10_ as the most likely impacted functional genes (Table [4](https://www.nature.com/articles/s41588-025-02267-2#Tab4)).

### Ancestry-combined and sex-combined meta-analysis

Last, we performed an ancestry-combined and sex-combined meta-analysis, using all ancestry-specific and sex-specific preliminary GWAS analyses, through multi-ancestry meta-regression, again using MR-MEGA[37](https://www.nature.com/articles/s41588-025-02267-2#ref-CR37 "Mägi, R. et al. Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Hum. Mol. Genet. 26, 3639–3650 (2017)."). This analysis identified 19 loci reaching the conventional significance threshold (Table [4](https://www.nature.com/articles/s41588-025-02267-2#Tab4), Extended Data Fig. [6](https://www.nature.com/articles/s41588-025-02267-2#Fig10) and Supplementary Figs. [69](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)–[87](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). Sentinel hits from the ancestry-combined and sex-combined meta-analysis were mapped to _SLC39A8_, _CAMTA1_, _CBLN4_, _VRK2_, _CREB3L4_, _KCTD10_, _CTSC_, _SOX5_ and _SCN8A_ as the most likely impacted functional genes (Table [4](https://www.nature.com/articles/s41588-025-02267-2#Tab4)).

From these sentinel hits, we report 57 distinct loci (defined as non-overlapping within ±1 Mb of the sentinel single-nucleotide polymorphism (SNP)) associated with self-reported stuttering across the primary ancestry-specific and sex-specific GWAS and the secondary sex-combined, ancestry-specific meta-analyses, ancestry-combined, sex-specific meta-analyses, and ancestry-combined, sex-combined meta-analysis. However, sex-specific participation bias can result in spurious autosomal associations[38](https://www.nature.com/articles/s41588-025-02267-2#ref-CR38 "Pirastu, N. et al. Genetic analyses identify widespread sex-differential participation bias. Nat. Genet. 53, 663–671 (2021)."). Hence, we compared the 57 stuttering-associated loci to 158 autosomal loci spuriously associated with sex in prior work[38](https://www.nature.com/articles/s41588-025-02267-2#ref-CR38 "Pirastu, N. et al. Genetic analyses identify widespread sex-differential participation bias. Nat. Genet. 53, 663–671 (2021)."). We found six of the 57 stuttering-associated loci were located within the SNP windows for loci previously associated with sex-specific participation bias (Table [4](https://www.nature.com/articles/s41588-025-02267-2#Tab4)). For consistency of association between non-overlapping samples within this study as well as with previously identified stuttering loci in the literature, please see the [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1).

### Genetic heritability

We calculated SNP-based liability-scale heritability for our male and female EUR studies using LDSC[34](https://www.nature.com/articles/s41588-025-02267-2#ref-CR34 "Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat. Genet. 47, 291–295 (2015)."),[35](https://www.nature.com/articles/s41588-025-02267-2#ref-CR35 "Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Nat. Genet. 47, 1236–1241 (2015)."). As with the genetic correlation analyses, for AFR and AMR, LDSC and other common approaches are biased in the presence of admixture. For EAS, the sample size prevented reliable heritability estimates[39](https://www.nature.com/articles/s41588-025-02267-2#ref-CR39 "The Neale Lab. Insights from estimates of SNP-heritability for >2,000 traits and disorders in UK Biobank. 
http://www.nealelab.is/blog/2017/9/20/insights-from-estimates-of-snp-heritability-for-2000-traits-and-disorders-in-uk-biobank
(2017)."). Hence, heritability analyses were not performed for the AFR, AMR and EAS ancestries. Liability-scaled heritability was estimated to be 9.11% (SE = 0.0054) for EUR females and 9.62% (SE = 0.0052) for EUR males, assuming a stuttering population prevalence of 10%. LDSC intercept and observed scale heritability estimates are reported in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

Partitioned SNP-based heritability of stuttering by broad functional annotation[35](https://www.nature.com/articles/s41588-025-02267-2#ref-CR35 "Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Nat. Genet. 47, 1236–1241 (2015).") showed significant enrichments of conserved regions, as well as a chromatin mark for enhancers, H3K4me1, in EUR male, EUR female and sex-combined EUR stuttering results (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02267-2#Fig11) and Supplementary Tables [8](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[10](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). EUR male and sex-combined EUR stuttering results were enriched for weak enhancers, repressed marks and a marker for active chromatin, H3K9ac (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02267-2#Fig11) and Supplementary Tables [8](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[10](https://www.nature.com/articles/s41588-025-02267-2#MOESM3); _P_ < 9.6 × 10−4). Sex-combined EUR stuttering results were enriched for fetal and adult DNase hypersensitive sites, introns and marks of active chromatin sites H3K27ac and H3K4me3 (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02267-2#Fig11) and Supplementary Table [8](https://www.nature.com/articles/s41588-025-02267-2#MOESM3); _P_ < 9.60 × 10−4).

Furthermore, we used LDSC to explore whether genes expressed in specific cell or tissue types are enriched for stuttering-associated variants[40](https://www.nature.com/articles/s41588-025-02267-2#ref-CR40 "Finucane, H. K. et al. Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types. Nat. Genet. 50, 621–629 (2018)."). For brain cell types, we found that our EUR female and sex-combined EUR stuttering results were enriched for neurons (Extended Data Fig. [8](https://www.nature.com/articles/s41588-025-02267-2#Fig12) and Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) and [12](https://www.nature.com/articles/s41588-025-02267-2#MOESM3); _P_ < 0.017). We then tested for enrichment of effects of stuttering-associated variants in brain tissues previously associated with stuttering in imaging studies (full results shown in Extended Data Fig. [9](https://www.nature.com/articles/s41588-025-02267-2#Fig13), Supplementary Tables [12](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) and [13](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) and [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1))[41](https://www.nature.com/articles/s41588-025-02267-2#ref-CR41 "Lu, C. et al. The neural substrates for atypical planning and execution of word production in stuttering. Exp. Neurol. 221, 146–156 (2010)."),[42](https://www.nature.com/articles/s41588-025-02267-2#ref-CR42 "Chang, S.-E., Garnett, E. O., Etchell, A. & Chow, H. M. Functional and neuroanatomical bases of developmental stuttering: current insights. Neuroscientist 25, 566–582 (2019)."),[43](https://www.nature.com/articles/s41588-025-02267-2#ref-CR43 "Etchell, A. C., Civier, O., Ballard, K. J. & Sowman, P. F. A systematic literature review of neuroimaging research on developmental stuttering between 1995 and 2016. J. Fluen. Disord. 55, 6–45 (2018)."),[44](https://www.nature.com/articles/s41588-025-02267-2#ref-CR44 "Liu, J. et al. A functional imaging study of self-regulatory capacities in persons who stutter. PLoS ONE 9, e89891 (2014)."),[45](https://www.nature.com/articles/s41588-025-02267-2#ref-CR45 "Neef, N. E. et al. Altered morphology of the nucleus accumbens in persistent developmental stuttering. J. Fluen. Disord. 55, 84–93 (2018)."),[46](https://www.nature.com/articles/s41588-025-02267-2#ref-CR46 "Toyomura, A., Fujii, T. & Kuriki, S. Effect of an 8-week practice of externally triggered speech on basal ganglia activity of stuttering and fluent speakers. NeuroImage 109, 458–468 (2015)."),[47](https://www.nature.com/articles/s41588-025-02267-2#ref-CR47 "Chang, S. E. & Zhu, D. C. Neural network connectivity differences in children who stutter. Brain 136, 3709–3726 (2013)."),[48](https://www.nature.com/articles/s41588-025-02267-2#ref-CR48 "Chang, S.-E., Horwitz, B., Ostuni, J., Reynolds, R. & Ludlow, C. L. Evidence of left inferior frontal–premotor structural and functional connectivity deficits in adults who stutter. Cereb. Cortex 21, 2507–2518 (2011)."), and enrichment was further investigated by examining tissue-specific annotations for active chromatin and enhancers (specifically, known histone marks H3K27ac, H3K9ac, H3K4me1, H3K4me3 and H3K36me3) (see Extended Data Fig. [10](https://www.nature.com/articles/s41588-025-02267-2#Fig14), Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[13](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) and [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1) for full results).

### Genetic correlation

To explore whether the genetic architecture of stuttering is shared with previously identified comorbidities, we performed genetic correlation analyses with traits associated with stuttering in the literature (Supplementary Tables [14](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) and [15](https://www.nature.com/articles/s41588-025-02267-2#MOESM3))[49](https://www.nature.com/articles/s41588-025-02267-2#ref-CR49 "Choo, A. L., Smith, S. A. & Li, H. Associations between stuttering, comorbid conditions and executive function in children: a population-based study. BMC Psychol. 8, 113 (2020)."),[50](https://www.nature.com/articles/s41588-025-02267-2#ref-CR50 "Wieland, E. A., McAuley, J. D., Dilley, L. C. & Chang, S.-E. Evidence for a rhythm perception deficit in children who stutter. Brain Lang. 144, 26–34 (2015)."),[51](https://www.nature.com/articles/s41588-025-02267-2#ref-CR51 "Garnett, E. O. et al. Auditory rhythm discrimination in adults who stutter: an fMRI study. Brain Lang. 236, 105219 (2023)."),[52](https://www.nature.com/articles/s41588-025-02267-2#ref-CR52 "Ladányi, E., Persici, V., Fiveash, A., Tillmann, B. & Gordon, R. L. Is atypical rhythm a risk factor for developmental speech and language disorders? Wiley Interdiscip. Rev. Cogn. Sci. 11, e1528 (2020)."),[53](https://www.nature.com/articles/s41588-025-02267-2#ref-CR53 "Pruett, D. G. et al. Identifying developmental stuttering and associated comorbidities in electronic health records and creating a phenome risk classifier. J. Fluen. Disord. 68, 105847 (2021)."),[54](https://www.nature.com/articles/s41588-025-02267-2#ref-CR54 "Arenas, R. M., Walker, E. A. & Oleson, J. J. Developmental stuttering in children who are hard of hearing. Lang. Speech Hear. Serv. Sch. 48, 234–248 (2017)."),[55](https://www.nature.com/articles/s41588-025-02267-2#ref-CR55 "Strom, M. A. & Silverberg, J. I. Eczema is associated with childhood speech disorder: a retrospective analysis from the National Survey of Children’s Health and the National Health Interview Survey. J. Pediatrics 168, 185–192.e4 (2016)."),[56](https://www.nature.com/articles/s41588-025-02267-2#ref-CR56 "Ajdacic-Gross, V. et al. Subtypes of stuttering determined by latent class analysis in two Swiss epidemiological surveys. PLoS ONE 13, e0198450 (2018)."),[57](https://www.nature.com/articles/s41588-025-02267-2#ref-CR57 "Briley, P. M. & Merlo, S. Presence of allergies and their impact on sleep in children who stutter. Perspect. ASHA Spec. Interest Groups 5, 1454–1466 (2020)."),[58](https://www.nature.com/articles/s41588-025-02267-2#ref-CR58 "Jacobs, M. M., Merlo, S. & Briley, P. M. Sleep duration, insomnia, and stuttering: the relationship in adolescents and young adults. J. Commun. Disord. 91, 106106 (2021)."),[59](https://www.nature.com/articles/s41588-025-02267-2#ref-CR59 "Merlo, S. & Briley, P. M. Sleep problems in children who stutter: evidence from population data. J. Commun. Disord. 82, 105935 (2019)."),[60](https://www.nature.com/articles/s41588-025-02267-2#ref-CR60 "Mohammadi, H. et al. Sleep problems, social anxiety and stuttering severity in adults who do and adults who do not stutter. J. Clin. Med. 12, 161 (2023)."),[61](https://www.nature.com/articles/s41588-025-02267-2#ref-CR61 "Briley, P. M., Merlo, S. & Ellis, C. Sex differences in childhood stuttering and coexisting developmental disorders. J. Dev. Phys. Disabil. 34, 505–527 (2022)."),[62](https://www.nature.com/articles/s41588-025-02267-2#ref-CR62 "Lebrun, Y. Stuttering and epilepsy. J. Neurolinguist. 6, 433–444 (1991)."),[63](https://www.nature.com/articles/s41588-025-02267-2#ref-CR63 "Tomisato, S., Oishi, N., Asano, K., Watanabe, Y. & Ogawa, K. Developmental disability and psychiatric conditions in 39 patients with stuttering. Jpn. J. Logoped. Phoniatr. 57, 7–11 (2016)."),[64](https://www.nature.com/articles/s41588-025-02267-2#ref-CR64 "Tichenor, S. E., Palasik, S. & Yaruss, J. S. Understanding the broader impact of stuttering: suicidal ideation. Am. J. Speech Lang. Pathol. 32, 2087–2110 (2023)."),[65](https://www.nature.com/articles/s41588-025-02267-2#ref-CR65 "Briley, P. M., Gerlach, H. & Jacobs, M. M. Relationships between stuttering, depression, and suicidal ideation in young adults: accounting for gender differences. J. Fluen. Disord. 67, 105820 (2021)."),[66](https://www.nature.com/articles/s41588-025-02267-2#ref-CR66 "Bernard, R., Hofslundsengen, H. & Frazier Norbury, C. Anxiety and depression symptoms in children and adolescents who stutter: a systematic review and meta-analysis. J. Speech Lang. Hear. Res. 65, 624–644 (2022)."),[67](https://www.nature.com/articles/s41588-025-02267-2#ref-CR67 "Iverach, L. & Rapee, R. M. Social anxiety disorder and stuttering: current status and future directions. J. Fluen. Disord. 40, 69–82 (2014)."),[68](https://www.nature.com/articles/s41588-025-02267-2#ref-CR68 "Tichenor, S. E. & Yaruss, J. S. Stuttering as defined by adults who stutter. J. Speech Lang. Hear. Res. 62, 4356–4369 (2019)."),[69](https://www.nature.com/articles/s41588-025-02267-2#ref-CR69 "Tichenor, S. E., Johnson, C. A. & Yaruss, J. S. A preliminary investigation of attention-deficit/hyperactivity disorder characteristics in adults who stutter. J. Speech Lang. Hear. Res. 64, 839–853 (2021)."),[70](https://www.nature.com/articles/s41588-025-02267-2#ref-CR70 "Alm, P. A. Stuttering in relation to anxiety, temperament, and personality: review and analysis with focus on causality. J. Fluen. Disord. 40, 5–21 (2014)."),[71](https://www.nature.com/articles/s41588-025-02267-2#ref-CR71 "Pirinen, V. et al. A comprehensive analysis of speech disfluencies in autistic young adults and control young adults: group differences in typical, stuttering-like, and atypical disfluencies. J. Speech Lang. Hear. Res. 66, 832–848 (2023)."),[72](https://www.nature.com/articles/s41588-025-02267-2#ref-CR72 "Plexico, L. W., Cleary, J. E., McAlpine, A. & Plumb, A. M. Disfluency characteristics observed in young children with autism spectrum disorders: a preliminary report. Perspect. Fluen. Fluen. Disord. 20, 42–50 (2010)."),[73](https://www.nature.com/articles/s41588-025-02267-2#ref-CR73 "Scaler Scott, K., Tetnowski, J. A., Flaitz, J. R. & Yaruss, J. S. Preliminary study of disfluency in school‐aged children with autism. Int. J. Lang. Commun. Disord. 49, 75–89 (2014)."),[74](https://www.nature.com/articles/s41588-025-02267-2#ref-CR74 "Tetnowski, J. A. & Donaher, J. Stuttering and autism spectrum disorders: assessment and treatment. Semin. Speech Lang. 43, 117–129 (2022)."),[75](https://www.nature.com/articles/s41588-025-02267-2#ref-CR75 "Iverach, L. et al. Mood and substance use disorders among adults seeking speech treatment for stuttering. J. Speech Lang. Hear. Res. 53, 1178–1190 (2010)."),[76](https://www.nature.com/articles/s41588-025-02267-2#ref-CR76 "Heelan, M., McAllister, J. & Skinner, J. Stuttering, alcohol consumption and smoking. J. Fluen. Disord. 48, 27–34 (2016)."). For traits with sex-stratified summary statistics, we observed significant positive genetic correlations, after Bonferroni correction, within both our EUR male and EUR female studies for depression (EUR male, _P_ = 6.82 × 10−5; EUR female, _P_ = 4.53 × 10−8; Fig. [2a](https://www.nature.com/articles/s41588-025-02267-2#Fig2) and Supplementary Table [16](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). We also observed a number of significant positive genetic correlations in EUR females only: for hearing loss (_P_ = 6.50 × 10−6), asthma (_P_ = 2.91 × 10−8), daytime sleepiness (_P_ = 6.96 × 10−8), attention deficit hyperactivity disorder (ADHD) (_P_ = 4.66 × 10−6) and body mass index (BMI) (_P_ = 4.08 × 10−17) as well as a significant negative genetic correlation for frequency of alcohol consumption (_P_ = 5.0 × 10−4) and walking pace (_P_ = 8.81 × 10−11; Fig. [2a](https://www.nature.com/articles/s41588-025-02267-2#Fig2) and Supplementary Table [16](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). No traits were significantly genetically correlated with stuttering in EUR males exclusively.

**Fig. 2: Forest plot showing genetic correlations for stuttering and previously reported comorbid traits.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02267-2/figures/2)

**a**, Ancestry-specific and sex-specific genetic correlations performed for each indicated trait with self-reported stuttering in EUR males and EUR females. Each trait is color-coded according to descriptive category (behavioral, circadian rhythm, immune, metabolic, motor, neurological, speech and language). **b**, Ancestry-specific and sex-specific genetic correlation estimates (EUR male and EUR female) and sex-combined, ancestry-specific (sex-combined EUR) genetic correlation estimates for beat synchronization and autism. Male-specific correlations are designated by triangles, female-specific correlations are designated by circles and sex-combined correlations are designated by squares. Data points represent the correlation coefficients; error bars, SE. Asterisks denote significant genetic correlations by LDSC. See Supplementary Table [15](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) for full information on traits used for analyses.

For traits without sex-stratified summary statistics, we observed significant positive genetic correlations, after Bonferroni correction, with autism (EUR male, _P_ = 1.11 × 10−6; EUR female, _P_ = 2.05 × 10−11; sex-combined EUR, _P_ = 4.51 × 10−13) and significant negative genetic correlations with beat synchronization (EUR male, _P_ = 4.00 × 10−4; EUR female, _P_ = 3.33 × 10−7; sex-combined EUR, _P_ = 1.13 × 10−7; Fig. [2b](https://www.nature.com/articles/s41588-025-02267-2#Fig2) and Supplementary Table [16](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)).

### Mendelian randomization

We performed debiased inverse-variance weighted Mendelian randomization analyses, which handle balanced horizontal pleiotropy and allow for weak instruments[77](https://www.nature.com/articles/s41588-025-02267-2#ref-CR77 "Ye, T., Shao, J. & Kang, H. Debiased inverse-variance weighted estimator in two-sample summary-data Mendelian randomization. Ann. Stat. 49, 2079–2100 (2021)."),[78](https://www.nature.com/articles/s41588-025-02267-2#ref-CR78 "Sanderson, E. et al. Mendelian randomization. Nat. Rev. Methods Prim. 2, 6 (2022)."), to assess the causal relationships between stuttering and previously reported co-occurring traits with significant genetic correlations. Analyses were performed for 11 sex-specific traits and two sex-combined traits genetically correlated with stuttering (after Bonferroni correction) to estimate causal effects between these traits and the self-reported stuttering phenotype captured in our EUR male GWAS, EUR female GWAS or sex-combined EUR meta-analysis. We estimated a significant causal effect of slower walking pace, autism and impaired rhythm on EUR female stuttering (Fig. [3a](https://www.nature.com/articles/s41588-025-02267-2#Fig3) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Additionally, we found evidence of bi-directional causal relationships between both higher BMI and depression and EUR female stuttering (Fig. [3a,b](https://www.nature.com/articles/s41588-025-02267-2#Fig3) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). We also observed a bi-directional relationship between impaired beat synchronization and stuttering in the EUR male and sex-combined EUR analyses (Fig. [3a,b](https://www.nature.com/articles/s41588-025-02267-2#Fig3) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Furthermore, we observed a bi-directional relationship between autism and stuttering in the sex-combined EUR analysis (Fig. [3b](https://www.nature.com/articles/s41588-025-02267-2#Fig3) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Lastly, in EUR females, we observed significant causal effect estimates of female stuttering on ADHD risk (Fig. [3b](https://www.nature.com/articles/s41588-025-02267-2#Fig3) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Results of other Mendelian randomization methods can be found in Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

**Fig. 3: Forest plot showing results of Mendelian randomization analysis for stuttering and previously reported comorbid traits.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02267-2/figures/3)

**a**, Debiased inverse-variance weighted Mendelian randomization analysis estimating causal inference (comorbid trait → stuttering). **b**, Debiased inverse-variance weighted Mendelian randomization analysis estimating causal inference (stuttering → comorbid trait). Data are represented as estimates; error bars, SE. Filled circles are statistically significant (debiased inverse-variance weighted estimator _P_ < 3.33 × 10−3 after Bonferroni correction for testing 15 associations that were genetically correlated with stuttering). Full information on traits used for analyses can be found in Supplementary Table [15](https://www.nature.com/articles/s41588-025-02267-2#MOESM3). Full results along with other Mendelian randomization methods can be found in Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3). ASD, autism spectrum disorder.

### PRS analyses

Stuttering PRS were derived from our EUR female and EUR male GWAS, and applied to EUR participants in two independent studies of developmental stuttering: ISP (893 EUR cases and 6,052 EUR controls)[30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022).") and Add Health (588 EUR cases and 6,621 EUR controls)[32](https://www.nature.com/articles/s41588-025-02267-2#ref-CR32 "Harris, K. M. et al. Cohort profile: the National Longitudinal Study of Adolescent to Adult Health (Add Health). Int. J. Epidemiol. 48, 1415–1415k (2019)."). Overall, male-specific PRS models out-performed female-specific PRS models (Fig. [4](https://www.nature.com/articles/s41588-025-02267-2#Fig4)). In particular, the male-specific PRS model derived from the EUR male GWAS demonstrated significant predictive value for both male and female EUR in the ISP (area under the curve (AUC), 0.6108 for male, 0.6065 for female; Fig. [4a](https://www.nature.com/articles/s41588-025-02267-2#Fig4) and Supplementary Fig. [88](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)) and Add Health (AUC, 0.5373 for male, 0.5529 for female; Fig. [4b](https://www.nature.com/articles/s41588-025-02267-2#Fig4) and Supplementary Fig. [88](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). PRS values for cases and controls within the ISP cohort and Add Health study participants can be found in Supplementary Table [18](https://www.nature.com/articles/s41588-025-02267-2#MOESM3). The lack of large genetic data resources from clinically ascertained participants matching other ancestries prevented validation of PRS derived from non-EUR analyses; however, despite low power, we present preliminary results of cross-ancestry testing of the EUR male and EUR female PRS models in AFR ISP and AFR Add Health study participants (Supplementary Fig. [89](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)).

**Fig. 4: Performance of self-reported stuttering PRS model in independent EUR stuttering datasets.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02267-2/figures/4)

PRS were developed using EUR male or EUR female GWAS results and applied to clinically validated ISP and self-report Add Health subjects, and demonstrate increased stuttering liability within stuttering cases. The model was developed and trained using the default auto-phi shrinkage parameter through PRScs. LD panels were constructed using 1000 Genomes Project phase 3 EUR reference data. **a**, AUC model performance in the ISP cohort. **b**, AUC model performance in the Add Health cohort.

Discussion
----------

Our GWAS of self-reported stuttering in males and females is the largest to date, comprising nearly 100,000 cases and over one million controls. Although some complex traits with differences in prevalence by sex or ancestry show similar genetic architectures across these groups[79](https://www.nature.com/articles/s41588-025-02267-2#ref-CR79 "Mahajan, A. et al. Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation. Nat. Genet. 54, 560–572 (2022)."),[80](https://www.nature.com/articles/s41588-025-02267-2#ref-CR80 "Hautakangas, H. et al. Genome-wide analysis of 102,084 migraine cases identifies 123 risk loci and subtype-specific risk alleles. Nat. Genet. 54, 152–160 (2022)."), we found that there are both shared and distinct genetic effects impacting stuttering risk within sex and/or ancestry groups. Our primary GWAS and secondary meta-analyses identified 57 distinct signals (mapping to 48 unique genes), none of which have been previously reported in stuttering literature. Our estimates of stuttering heritability are in line with SNP-based estimates for other common, complex traits such as insomnia[81](https://www.nature.com/articles/s41588-025-02267-2#ref-CR81 "Jansen, P. R. et al. Genome-wide analysis of insomnia in 1,331,010 individuals identifies new risk loci and functional pathways. Nat. Genet. 51, 394–403 (2019)."), type 2 diabetes[82](https://www.nature.com/articles/s41588-025-02267-2#ref-CR82 "Xue, A. et al. Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes. Nat. Commun. 9, 2941 (2018).") and beat synchronization[83](https://www.nature.com/articles/s41588-025-02267-2#ref-CR83 "Niarchou, M. et al. Genome-wide association study of musical beat synchronization demonstrates high polygenicity. Nat. Hum. Behav. 6, 1292–1309 (2022)."). Most prior studies of genetic risk factors for stuttering have explored rare variant effects in pedigrees; however, we find effects that are consistent with high polygenicity, suggesting a genetic architecture similar to other common complex traits[81](https://www.nature.com/articles/s41588-025-02267-2#ref-CR81 "Jansen, P. R. et al. Genome-wide analysis of insomnia in 1,331,010 individuals identifies new risk loci and functional pathways. Nat. Genet. 51, 394–403 (2019)."),[82](https://www.nature.com/articles/s41588-025-02267-2#ref-CR82 "Xue, A. et al. Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes. Nat. Commun. 9, 2941 (2018)."),[83](https://www.nature.com/articles/s41588-025-02267-2#ref-CR83 "Niarchou, M. et al. Genome-wide association study of musical beat synchronization demonstrates high polygenicity. Nat. Hum. Behav. 6, 1292–1309 (2022).").

To validate whether the genetic architecture captured by this analysis reflects the genetic etiology of clinically ascertained developmental stuttering, we developed PRS models from the sex-specific EUR GWAS results and applied them to the ISP[30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022).") and Add Health[32](https://www.nature.com/articles/s41588-025-02267-2#ref-CR32 "Harris, K. M. et al. Cohort profile: the National Longitudinal Study of Adolescent to Adult Health (Add Health). Int. J. Epidemiol. 48, 1415–1415k (2019).") stuttering cohorts (which are predominantly EUR samples) for validation. Within the ISP, a clinically ascertained cohort enriched with males and persistent cases of stuttering, the EUR male-derived models showed significant differences in liability scores between stuttering cases and controls in both sexes. By contrast, the EUR female model had significant predictive performance only in EUR females in the ISP. In Add Health, a self-report cohort, both the male and female PRS models significantly predicted case or control status. The difference in the predictive performance between females and males in these two external validation cohorts is notable, with several possible non-mutually exclusive explanations: (1) compared to 23andMe EUR males, the trait captured by 23andMe EUR females is less representative of developmental stuttering; (2) the genetic liability for developmental stuttering varies between males and females and is perhaps confounded by differences in genetic susceptibility to persistent versus recovered stuttering; or (3) genetic variation contributing to developmental stuttering risk may be confounded by horizontal pleiotropy modulated by sex.

The first possibility, that the EUR female stuttering phenotype is not capturing clinically diagnosed developmental stuttering as well as the EUR male stuttering phenotype, may have different possible explanations as well. Prior studies have found that sex-differential participation bias in 23andMe data can lead to spurious genetic associations with sex, particularly in complex traits associated with sex-differential participation bias[38](https://www.nature.com/articles/s41588-025-02267-2#ref-CR38 "Pirastu, N. et al. Genetic analyses identify widespread sex-differential participation bias. Nat. Genet. 53, 663–671 (2021)."). For the 57 unique stuttering-associated loci, six were within the SNP windows for loci previously identified to be spuriously associated with sex[38](https://www.nature.com/articles/s41588-025-02267-2#ref-CR38 "Pirastu, N. et al. Genetic analyses identify widespread sex-differential participation bias. Nat. Genet. 53, 663–671 (2021)."). We may also be observing sex-specific differences in adult recall accuracy. Females are more likely to recover from stuttering in early childhood[1](https://www.nature.com/articles/s41588-025-02267-2#ref-CR1 "Yairi, E. & Ambrose, N. Epidemiology of stuttering: 21st century advances. J. Fluen. Disord. 38, 66–87 (2013)."),[13](https://www.nature.com/articles/s41588-025-02267-2#ref-CR13 "Singer, C. M., Hessling, A., Kelly, E. M., Singer, L. & Jones, R. M. Clinical characteristics associated with stuttering persistence: a meta-analysis. J. Speech Lang. Hear. Res. 63, 2995–3018 (2020)."),[14](https://www.nature.com/articles/s41588-025-02267-2#ref-CR14 "Singer, C. M., Otieno, S., Chang, S.-E. & Jones, R. M. Predicting persistent developmental stuttering using a cumulative risk approach. J. Speech Lang. Hear. Res. 65, 70–95 (2022)."); therefore, the self-report stuttering phenotype may be more impacted by the accuracy of recall of stuttering in childhood in females than in males. There may be other explanations for a difference in how well the self-report phenotype serves as a proxy for clinical diagnosis by sex; for example, female participants in our study may be more likely to report subclinical stuttering than males. Future research will be needed to deconvolute genetic risk factors that are specific to sex and persistence.

To our knowledge, all variants reaching genome-wide significance in our study are novel findings for stuttering. The Open Targets Genetics ‘Variant-to-Gene’ (V2G) pipeline[84](https://www.nature.com/articles/s41588-025-02267-2#ref-CR84 "Mountjoy, E. et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. Nat. Genet. 53, 1527–1533 (2021)."),[85](https://www.nature.com/articles/s41588-025-02267-2#ref-CR85 "Ghoussaini, M. et al. Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. Nucleic Acids Res. 49, D1311–D1320 (2021).") assigned _VRK2_ as the most likely functional gene (_r_2 between these variants is 0.31 in the CEU population) for one locus spanning the two top hits in EUR male ([rs35609938](https://www.ncbi.nlm.nih.gov/snp/?term=rs35609938) and [rs1040225](https://www.ncbi.nlm.nih.gov/snp/?term=rs1040225))[86](https://www.nature.com/articles/s41588-025-02267-2#ref-CR86 "Machiela, M. J. & Chanock, S. J. LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics 31, 3555–3557 (2015)."). Specifically, [rs35609938](https://www.ncbi.nlm.nih.gov/snp/?term=rs35609938) occurs downstream of _VRK2_ and upstream of _FANCL_, and [rs1040225](https://www.ncbi.nlm.nih.gov/snp/?term=rs1040225) occurs within either an intronic or upstream region of _VRK2_, depending on isoform (Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2) and Supplementary Figs. [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM1) and [2](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). Interestingly, _FANCL_ and _VRK2_ were recently implicated in musical beat synchronization[83](https://www.nature.com/articles/s41588-025-02267-2#ref-CR83 "Niarchou, M. et al. Genome-wide association study of musical beat synchronization demonstrates high polygenicity. Nat. Hum. Behav. 6, 1292–1309 (2022)."). Rhythm perception impairments have been linked to several speech and language conditions, including stuttering[51](https://www.nature.com/articles/s41588-025-02267-2#ref-CR51 "Garnett, E. O. et al. Auditory rhythm discrimination in adults who stutter: an fMRI study. Brain Lang. 236, 105219 (2023)."),[52](https://www.nature.com/articles/s41588-025-02267-2#ref-CR52 "Ladányi, E., Persici, V., Fiveash, A., Tillmann, B. & Gordon, R. L. Is atypical rhythm a risk factor for developmental speech and language disorders? Wiley Interdiscip. Rev. Cogn. Sci. 11, e1528 (2020)."). Complex rhythm discrimination is below average in adults[51](https://www.nature.com/articles/s41588-025-02267-2#ref-CR51 "Garnett, E. O. et al. Auditory rhythm discrimination in adults who stutter: an fMRI study. Brain Lang. 236, 105219 (2023).") and children[50](https://www.nature.com/articles/s41588-025-02267-2#ref-CR50 "Wieland, E. A., McAuley, J. D., Dilley, L. C. & Chang, S.-E. Evidence for a rhythm perception deficit in children who stutter. Brain Lang. 144, 26–34 (2015).") who stutter, consistent with the Atypical Rhythm Risk Hypothesis[52](https://www.nature.com/articles/s41588-025-02267-2#ref-CR52 "Ladányi, E., Persici, V., Fiveash, A., Tillmann, B. & Gordon, R. L. Is atypical rhythm a risk factor for developmental speech and language disorders? Wiley Interdiscip. Rev. Cogn. Sci. 11, e1528 (2020)."), which posits that those with atypical rhythm may be at risk for developmental speech and language disorders. Clinically, synchronizing speech with external pacing cues, such as a metronome, can often temporarily decrease stuttering disfluencies[50](https://www.nature.com/articles/s41588-025-02267-2#ref-CR50 "Wieland, E. A., McAuley, J. D., Dilley, L. C. & Chang, S.-E. Evidence for a rhythm perception deficit in children who stutter. Brain Lang. 144, 26–34 (2015)."),[87](https://www.nature.com/articles/s41588-025-02267-2#ref-CR87 "Wingate, M. E. & Howell, P. Foundations of stuttering. J. Acoust. Soc. Am. 112, 1229–1231 (2002)."),[88](https://www.nature.com/articles/s41588-025-02267-2#ref-CR88 "Brady, J. P. Metronome-conditioned speech retraining for stuttering. Behav. Ther. 2, 129–150 (1971)."),[89](https://www.nature.com/articles/s41588-025-02267-2#ref-CR89 "Brady, J. P. Studies on the metronome effect on stuttering. Behav. Res. Ther. 7, 197–204 (1969)."). Impairments in rhythm processing may be functionally related to stuttering, and our findings offer the first genetic support for this hypothesis.

Our study also provides additional evidence of the effect for three genes that have been previously reported in family-based studies of stuttering. Although the previously reported variants were not directly genotyped and were too rare for accurate imputation or not present in the 23andMe data, we found three unique signals surpassing Bonferroni significance (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)) for the following genes: _GNPTAB_, _GNPTG_[19](https://www.nature.com/articles/s41588-025-02267-2#ref-CR19 "Kang, C. et al. Mutations in the lysosomal enzyme-targeting pathway and persistent stuttering. N. Engl. J. Med. 362, 677–685 (2010)."),[27](https://www.nature.com/articles/s41588-025-02267-2#ref-CR27 "Kazemi, N., Estiar, M. A., Fazilaty, H. & Sakhinia, E. Variants in GNPTAB, GNPTG and NAGPA genes are associated with stutterers. Gene 647, 93–100 (2018).") and _AP4E1_ (ref. [24](https://www.nature.com/articles/s41588-025-02267-2#ref-CR24 "Raza, M. H. et al. Association between rare variants in AP4E1, a component of intracellular trafficking, and persistent stuttering. Am. J. Hum. Genet. 97, 715–725 (2015).")); for _GNPTG_, the sentinel variant in our replication analysis was extremely rare (minor allele frequency of 4.22 × 10−5). _AP4E1_ interacts with previously reported gene _NAGPA_[24](https://www.nature.com/articles/s41588-025-02267-2#ref-CR24 "Raza, M. H. et al. Association between rare variants in AP4E1, a component of intracellular trafficking, and persistent stuttering. Am. J. Hum. Genet. 97, 715–725 (2015)."),[29](https://www.nature.com/articles/s41588-025-02267-2#ref-CR29 "Frigerio Domingues, C. E. et al. Are variants in sex hormone metabolizing genes associated with stuttering? Brain Lang. 191, 28–30 (2019)."), another gene that has been previously implicated in family studies; together, these results provide modest additional support for the role of these genes in stuttering.

Imaging studies have demonstrated that people who stutter exhibit differences in a variety of brain areas[42](https://www.nature.com/articles/s41588-025-02267-2#ref-CR42 "Chang, S.-E., Garnett, E. O., Etchell, A. & Chow, H. M. Functional and neuroanatomical bases of developmental stuttering: current insights. Neuroscientist 25, 566–582 (2019)."),[43](https://www.nature.com/articles/s41588-025-02267-2#ref-CR43 "Etchell, A. C., Civier, O., Ballard, K. J. & Sowman, P. F. A systematic literature review of neuroimaging research on developmental stuttering between 1995 and 2016. J. Fluen. Disord. 55, 6–45 (2018)."), including the frontal cortex[44](https://www.nature.com/articles/s41588-025-02267-2#ref-CR44 "Liu, J. et al. A functional imaging study of self-regulatory capacities in persons who stutter. PLoS ONE 9, e89891 (2014)."), cingulate cortex[41](https://www.nature.com/articles/s41588-025-02267-2#ref-CR41 "Lu, C. et al. The neural substrates for atypical planning and execution of word production in stuttering. Exp. Neurol. 221, 146–156 (2010)."),[44](https://www.nature.com/articles/s41588-025-02267-2#ref-CR44 "Liu, J. et al. A functional imaging study of self-regulatory capacities in persons who stutter. PLoS ONE 9, e89891 (2014)."), basal ganglia (caudate, substantia nigra)[45](https://www.nature.com/articles/s41588-025-02267-2#ref-CR45 "Neef, N. E. et al. Altered morphology of the nucleus accumbens in persistent developmental stuttering. J. Fluen. Disord. 55, 84–93 (2018)."),[46](https://www.nature.com/articles/s41588-025-02267-2#ref-CR46 "Toyomura, A., Fujii, T. & Kuriki, S. Effect of an 8-week practice of externally triggered speech on basal ganglia activity of stuttering and fluent speakers. NeuroImage 109, 458–468 (2015)."),[47](https://www.nature.com/articles/s41588-025-02267-2#ref-CR47 "Chang, S. E. & Zhu, D. C. Neural network connectivity differences in children who stutter. Brain 136, 3709–3726 (2013)."),[48](https://www.nature.com/articles/s41588-025-02267-2#ref-CR48 "Chang, S.-E., Horwitz, B., Ostuni, J., Reynolds, R. & Ludlow, C. L. Evidence of left inferior frontal–premotor structural and functional connectivity deficits in adults who stutter. Cereb. Cortex 21, 2507–2518 (2011)."), inferior temporal lobe[90](https://www.nature.com/articles/s41588-025-02267-2#ref-CR90 "Hosseini, R., Walsh, B., Tian, F. & Wang, S. An fNIRS-based feature learning and classification framework to distinguish hemodynamic patterns in children who stutter. IEEE Trans. Neural Syst. Rehabil. Eng. 26, 1254–1263 (2018)."),[91](https://www.nature.com/articles/s41588-025-02267-2#ref-CR91 "Hartwigsen, G., Neef, N. E., Camilleri, J. A., Margulies, D. S. & Eickhoff, S. B. Functional segregation of the right inferior frontal gyrus: evidence from coactivation-based parcellation. Cereb. Cortex 29, 1532–1546 (2019)."),[92](https://www.nature.com/articles/s41588-025-02267-2#ref-CR92 "Chesters, J., Möttönen, R. & Watkins, K.E. Neural changes after training with transcranial direct current stimulation to increase speech fluency in adults who stutter. Preprint at 
https://doi.org/10.31219/osf.io/8st3j
(2021).") and cerebellum[41](https://www.nature.com/articles/s41588-025-02267-2#ref-CR41 "Lu, C. et al. The neural substrates for atypical planning and execution of word production in stuttering. Exp. Neurol. 221, 146–156 (2010)."). Our gene module enrichment analysis (see [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)) revealed enrichments in the frontal cortex, cortex, anterior cingulate cortex, nucleus accumbens of the basal ganglia and cerebellum. These results, as well as our finding that stuttering-associated variants are involved in neurons and regulation of gene expression in the brain (see Extended Data Figs. [7](https://www.nature.com/articles/s41588-025-02267-2#Fig11)–[10](https://www.nature.com/articles/s41588-025-02267-2#Fig11) and Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[13](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)) and that more than 20 genes identified in our analyses have been previously implicated in neurological and mental disorder traits, provide additional evidence for the neurological underpinning of stuttering risk[41](https://www.nature.com/articles/s41588-025-02267-2#ref-CR41 "Lu, C. et al. The neural substrates for atypical planning and execution of word production in stuttering. Exp. Neurol. 221, 146–156 (2010)."),[42](https://www.nature.com/articles/s41588-025-02267-2#ref-CR42 "Chang, S.-E., Garnett, E. O., Etchell, A. & Chow, H. M. Functional and neuroanatomical bases of developmental stuttering: current insights. Neuroscientist 25, 566–582 (2019)."),[43](https://www.nature.com/articles/s41588-025-02267-2#ref-CR43 "Etchell, A. C., Civier, O., Ballard, K. J. & Sowman, P. F. A systematic literature review of neuroimaging research on developmental stuttering between 1995 and 2016. J. Fluen. Disord. 55, 6–45 (2018)."),[44](https://www.nature.com/articles/s41588-025-02267-2#ref-CR44 "Liu, J. et al. A functional imaging study of self-regulatory capacities in persons who stutter. PLoS ONE 9, e89891 (2014)."),[45](https://www.nature.com/articles/s41588-025-02267-2#ref-CR45 "Neef, N. E. et al. Altered morphology of the nucleus accumbens in persistent developmental stuttering. J. Fluen. Disord. 55, 84–93 (2018)."),[46](https://www.nature.com/articles/s41588-025-02267-2#ref-CR46 "Toyomura, A., Fujii, T. & Kuriki, S. Effect of an 8-week practice of externally triggered speech on basal ganglia activity of stuttering and fluent speakers. NeuroImage 109, 458–468 (2015)."),[47](https://www.nature.com/articles/s41588-025-02267-2#ref-CR47 "Chang, S. E. & Zhu, D. C. Neural network connectivity differences in children who stutter. Brain 136, 3709–3726 (2013)."),[48](https://www.nature.com/articles/s41588-025-02267-2#ref-CR48 "Chang, S.-E., Horwitz, B., Ostuni, J., Reynolds, R. & Ludlow, C. L. Evidence of left inferior frontal–premotor structural and functional connectivity deficits in adults who stutter. Cereb. Cortex 21, 2507–2518 (2011)."),[93](https://www.nature.com/articles/s41588-025-02267-2#ref-CR93 "Loucks, T., Kraft, S. J., Choo, A. L., Sharma, H. & Ambrose, N. G. Functional brain activation differences in stuttering identified with a rapid fMRI sequence. J. Fluen. Disord. 36, 302–307 (2011)."),[94](https://www.nature.com/articles/s41588-025-02267-2#ref-CR94 "Chow, H. M. et al. Linking lysosomal enzyme targeting genes and energy metabolism with altered gray matter volume in children with persistent stuttering. Neurobiol. Lang. 1, 365–380 (2020)."). Future follow-up analyses are needed to functionally validate distinct enrichment features in males and females.

Genetic correlation analysis revealed a significant correlation of increased stuttering risk in both EUR males and females with increased risk of depression. In EUR females, we also observed a significant correlation of increased stuttering risk with hearing loss, asthma, daytime sleepiness, ADHD and BMI, and a negative correlation with alcohol consumption and walking pace. The connection between stuttering and sleep is further supported by a recent large-scale multi-ancestry GWAS meta-analysis of short and long sleep duration[95](https://www.nature.com/articles/s41588-025-02267-2#ref-CR95 "Austin-Zimmerman, I. et al. Genome-wide association studies and cross-population meta-analyses investigating short and long sleep duration. Nat. Commun. 14, 6059 (2023)."), which identified five genes associated with sleep duration that overlap with our stuttering findings: _PTPB2, SRPK2_, _KCTD10_, _MMAB_ and _SLC39A8_. In addition, for traits without available sex-stratified summary statistics, genetic correlation analysis revealed increased risk of autism and impaired beat synchronization. These genetic correlations, and their respective directions, are largely consistent with previous literature identifying traits comorbid with stuttering[49](https://www.nature.com/articles/s41588-025-02267-2#ref-CR49 "Choo, A. L., Smith, S. A. & Li, H. Associations between stuttering, comorbid conditions and executive function in children: a population-based study. BMC Psychol. 8, 113 (2020)."),[50](https://www.nature.com/articles/s41588-025-02267-2#ref-CR50 "Wieland, E. A., McAuley, J. D., Dilley, L. C. & Chang, S.-E. Evidence for a rhythm perception deficit in children who stutter. Brain Lang. 144, 26–34 (2015)."),[51](https://www.nature.com/articles/s41588-025-02267-2#ref-CR51 "Garnett, E. O. et al. Auditory rhythm discrimination in adults who stutter: an fMRI study. Brain Lang. 236, 105219 (2023)."),[52](https://www.nature.com/articles/s41588-025-02267-2#ref-CR52 "Ladányi, E., Persici, V., Fiveash, A., Tillmann, B. & Gordon, R. L. Is atypical rhythm a risk factor for developmental speech and language disorders? Wiley Interdiscip. Rev. Cogn. Sci. 11, e1528 (2020)."),[53](https://www.nature.com/articles/s41588-025-02267-2#ref-CR53 "Pruett, D. G. et al. Identifying developmental stuttering and associated comorbidities in electronic health records and creating a phenome risk classifier. J. Fluen. Disord. 68, 105847 (2021)."),[54](https://www.nature.com/articles/s41588-025-02267-2#ref-CR54 "Arenas, R. M., Walker, E. A. & Oleson, J. J. Developmental stuttering in children who are hard of hearing. Lang. Speech Hear. Serv. Sch. 48, 234–248 (2017)."),[55](https://www.nature.com/articles/s41588-025-02267-2#ref-CR55 "Strom, M. A. & Silverberg, J. I. Eczema is associated with childhood speech disorder: a retrospective analysis from the National Survey of Children’s Health and the National Health Interview Survey. J. Pediatrics 168, 185–192.e4 (2016)."),[56](https://www.nature.com/articles/s41588-025-02267-2#ref-CR56 "Ajdacic-Gross, V. et al. Subtypes of stuttering determined by latent class analysis in two Swiss epidemiological surveys. PLoS ONE 13, e0198450 (2018)."),[57](https://www.nature.com/articles/s41588-025-02267-2#ref-CR57 "Briley, P. M. & Merlo, S. Presence of allergies and their impact on sleep in children who stutter. Perspect. ASHA Spec. Interest Groups 5, 1454–1466 (2020)."),[58](https://www.nature.com/articles/s41588-025-02267-2#ref-CR58 "Jacobs, M. M., Merlo, S. & Briley, P. M. Sleep duration, insomnia, and stuttering: the relationship in adolescents and young adults. J. Commun. Disord. 91, 106106 (2021)."),[59](https://www.nature.com/articles/s41588-025-02267-2#ref-CR59 "Merlo, S. & Briley, P. M. Sleep problems in children who stutter: evidence from population data. J. Commun. Disord. 82, 105935 (2019)."),[60](https://www.nature.com/articles/s41588-025-02267-2#ref-CR60 "Mohammadi, H. et al. Sleep problems, social anxiety and stuttering severity in adults who do and adults who do not stutter. J. Clin. Med. 12, 161 (2023)."),[61](https://www.nature.com/articles/s41588-025-02267-2#ref-CR61 "Briley, P. M., Merlo, S. & Ellis, C. Sex differences in childhood stuttering and coexisting developmental disorders. J. Dev. Phys. Disabil. 34, 505–527 (2022)."),[62](https://www.nature.com/articles/s41588-025-02267-2#ref-CR62 "Lebrun, Y. Stuttering and epilepsy. J. Neurolinguist. 6, 433–444 (1991)."),[63](https://www.nature.com/articles/s41588-025-02267-2#ref-CR63 "Tomisato, S., Oishi, N., Asano, K., Watanabe, Y. & Ogawa, K. Developmental disability and psychiatric conditions in 39 patients with stuttering. Jpn. J. Logoped. Phoniatr. 57, 7–11 (2016)."),[64](https://www.nature.com/articles/s41588-025-02267-2#ref-CR64 "Tichenor, S. E., Palasik, S. & Yaruss, J. S. Understanding the broader impact of stuttering: suicidal ideation. Am. J. Speech Lang. Pathol. 32, 2087–2110 (2023)."),[65](https://www.nature.com/articles/s41588-025-02267-2#ref-CR65 "Briley, P. M., Gerlach, H. & Jacobs, M. M. Relationships between stuttering, depression, and suicidal ideation in young adults: accounting for gender differences. J. Fluen. Disord. 67, 105820 (2021)."),[66](https://www.nature.com/articles/s41588-025-02267-2#ref-CR66 "Bernard, R., Hofslundsengen, H. & Frazier Norbury, C. Anxiety and depression symptoms in children and adolescents who stutter: a systematic review and meta-analysis. J. Speech Lang. Hear. Res. 65, 624–644 (2022)."),[67](https://www.nature.com/articles/s41588-025-02267-2#ref-CR67 "Iverach, L. & Rapee, R. M. Social anxiety disorder and stuttering: current status and future directions. J. Fluen. Disord. 40, 69–82 (2014)."),[68](https://www.nature.com/articles/s41588-025-02267-2#ref-CR68 "Tichenor, S. E. & Yaruss, J. S. Stuttering as defined by adults who stutter. J. Speech Lang. Hear. Res. 62, 4356–4369 (2019)."),[69](https://www.nature.com/articles/s41588-025-02267-2#ref-CR69 "Tichenor, S. E., Johnson, C. A. & Yaruss, J. S. A preliminary investigation of attention-deficit/hyperactivity disorder characteristics in adults who stutter. J. Speech Lang. Hear. Res. 64, 839–853 (2021)."),[70](https://www.nature.com/articles/s41588-025-02267-2#ref-CR70 "Alm, P. A. Stuttering in relation to anxiety, temperament, and personality: review and analysis with focus on causality. J. Fluen. Disord. 40, 5–21 (2014)."),[71](https://www.nature.com/articles/s41588-025-02267-2#ref-CR71 "Pirinen, V. et al. A comprehensive analysis of speech disfluencies in autistic young adults and control young adults: group differences in typical, stuttering-like, and atypical disfluencies. J. Speech Lang. Hear. Res. 66, 832–848 (2023)."),[72](https://www.nature.com/articles/s41588-025-02267-2#ref-CR72 "Plexico, L. W., Cleary, J. E., McAlpine, A. & Plumb, A. M. Disfluency characteristics observed in young children with autism spectrum disorders: a preliminary report. Perspect. Fluen. Fluen. Disord. 20, 42–50 (2010)."),[73](https://www.nature.com/articles/s41588-025-02267-2#ref-CR73 "Scaler Scott, K., Tetnowski, J. A., Flaitz, J. R. & Yaruss, J. S. Preliminary study of disfluency in school‐aged children with autism. Int. J. Lang. Commun. Disord. 49, 75–89 (2014)."),[74](https://www.nature.com/articles/s41588-025-02267-2#ref-CR74 "Tetnowski, J. A. & Donaher, J. Stuttering and autism spectrum disorders: assessment and treatment. Semin. Speech Lang. 43, 117–129 (2022)."),[75](https://www.nature.com/articles/s41588-025-02267-2#ref-CR75 "Iverach, L. et al. Mood and substance use disorders among adults seeking speech treatment for stuttering. J. Speech Lang. Hear. Res. 53, 1178–1190 (2010)."),[76](https://www.nature.com/articles/s41588-025-02267-2#ref-CR76 "Heelan, M., McAllister, J. & Skinner, J. Stuttering, alcohol consumption and smoking. J. Fluen. Disord. 48, 27–34 (2016)."). However, we recognize that as a volunteer-based study, there are documented biases in UK Biobank that may affect our genetic correlation and Mendelian randomization results, especially when it comes to genetic correlates of behavior, lifestyle and social outcomes[96](https://www.nature.com/articles/s41588-025-02267-2#ref-CR96 "Schoeler, T. et al. Participation bias in the UK Biobank distorts genetic associations and downstream analyses. Nat. Hum. Behav. 7, 1216–1227 (2023)."). Therefore, increasing representativeness in biobanks and increasing representation and documentation of communication disorders in particular will be paramount for future studies. We regret that many language and motor-related phenotypes that may be correlated with stuttering are underrepresented in the genetics literature, do not have publicly available GWAS summary statistics or did not meet our minimum binary trait case requirement of _n_ > 1,000 and were not tested in our genetic correlation analyses.

We also performed Mendelian randomization analyses[77](https://www.nature.com/articles/s41588-025-02267-2#ref-CR77 "Ye, T., Shao, J. & Kang, H. Debiased inverse-variance weighted estimator in two-sample summary-data Mendelian randomization. Ann. Stat. 49, 2079–2100 (2021).") to assess causal relationships between stuttering and traits that have been previously reported as co-occurring with stuttering (see Supplementary Table [14](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Overall, walking pace and autism were predicted to have a causal effect on stuttering. In addition, stuttering was predicted to have a causal effect on ADHD and autism. Furthermore, we observed significant bi-directional effects between stuttering and BMI. In general, these results bolster previous studies showing preliminary evidence for associations between stuttering and BMI[53](https://www.nature.com/articles/s41588-025-02267-2#ref-CR53 "Pruett, D. G. et al. Identifying developmental stuttering and associated comorbidities in electronic health records and creating a phenome risk classifier. J. Fluen. Disord. 68, 105847 (2021)."), as well as gross motor coordination[97](https://www.nature.com/articles/s41588-025-02267-2#ref-CR97 "Kim, K. S., Daliri, A., Flanagan, J. R. & Max, L. Dissociated development of speech and limb sensorimotor learning in stuttering: Speech auditory-motor learning is impaired in both children and adults who stutter. Neuroscience 451, 1–21 (2020)."). We also observed significant bi-directional causal effects between stuttering and depression. This result is consistent with several studies suggesting that both males and females who stutter report elevated symptoms of depression compared to their fluent counterparts[65](https://www.nature.com/articles/s41588-025-02267-2#ref-CR65 "Briley, P. M., Gerlach, H. & Jacobs, M. M. Relationships between stuttering, depression, and suicidal ideation in young adults: accounting for gender differences. J. Fluen. Disord. 67, 105820 (2021)."),[66](https://www.nature.com/articles/s41588-025-02267-2#ref-CR66 "Bernard, R., Hofslundsengen, H. & Frazier Norbury, C. Anxiety and depression symptoms in children and adolescents who stutter: a systematic review and meta-analysis. J. Speech Lang. Hear. Res. 65, 624–644 (2022)."),[98](https://www.nature.com/articles/s41588-025-02267-2#ref-CR98 "Ardila, A. et al. An epidemiologic study of stuttering. J. Commun. Disord. 27, 37–48 (1994)."). Specifically, communication difficulties caused by stuttering can result in feelings of frustration and hopelessness and, along with broader societal stigma toward stuttering, can negatively impact psychological health[61](https://www.nature.com/articles/s41588-025-02267-2#ref-CR61 "Briley, P. M., Merlo, S. & Ellis, C. Sex differences in childhood stuttering and coexisting developmental disorders. J. Dev. Phys. Disabil. 34, 505–527 (2022)."),[99](https://www.nature.com/articles/s41588-025-02267-2#ref-CR99 "Corcoran, J. A. & Stewart, M. Stories of stuttering. J. Fluen. Disord. 23, 247–264 (1998)."),[100](https://www.nature.com/articles/s41588-025-02267-2#ref-CR100 "Boyle, M. P. Enacted stigma and felt stigma experienced by adults who stutter. J. Commun. Disord. 73, 50–61 (2018)."). We also observed significant bi-directional effects between stuttering and beat synchronization, or the ability to clap to a beat. This finding is especially compelling considering the role rhythm perception may have in stuttering: as previously mentioned, rhythm discrimination is below average in adults[51](https://www.nature.com/articles/s41588-025-02267-2#ref-CR51 "Garnett, E. O. et al. Auditory rhythm discrimination in adults who stutter: an fMRI study. Brain Lang. 236, 105219 (2023).") and children[50](https://www.nature.com/articles/s41588-025-02267-2#ref-CR50 "Wieland, E. A., McAuley, J. D., Dilley, L. C. & Chang, S.-E. Evidence for a rhythm perception deficit in children who stutter. Brain Lang. 144, 26–34 (2015).") who stutter, and synchronizing speech with external pacing cues, such as a metronome[88](https://www.nature.com/articles/s41588-025-02267-2#ref-CR88 "Brady, J. P. Metronome-conditioned speech retraining for stuttering. Behav. Ther. 2, 129–150 (1971)."),[89](https://www.nature.com/articles/s41588-025-02267-2#ref-CR89 "Brady, J. P. Studies on the metronome effect on stuttering. Behav. Res. Ther. 7, 197–204 (1969)."), can temporarily decrease stuttering disfluencies. Interestingly, we observed different significant causal relationships across sexes, which may be a result of observed differences in genetic risk between sexes. The distinct causal pathways in males and females relating stuttering to genetically correlated traits are notable. However, as previously discussed, females are more likely to recover from stuttering than males[13](https://www.nature.com/articles/s41588-025-02267-2#ref-CR13 "Singer, C. M., Hessling, A., Kelly, E. M., Singer, L. & Jones, R. M. Clinical characteristics associated with stuttering persistence: a meta-analysis. J. Speech Lang. Hear. Res. 63, 2995–3018 (2020)."),[14](https://www.nature.com/articles/s41588-025-02267-2#ref-CR14 "Singer, C. M., Otieno, S., Chang, S.-E. & Jones, R. M. Predicting persistent developmental stuttering using a cumulative risk approach. J. Speech Lang. Hear. Res. 65, 70–95 (2022)."); thus, one limitation of this study is an inability to fully differentiate between effects related to sex and those related to stuttering persistence. Improved granularity of self-report with information regarding stuttering persistence will be necessary to further clarify these effects.

Overall, we leveraged 99,776 cases and over one million controls to identify 57 unique genome-wide significant loci associated with ancestry-specific and sex-specific self-reported stuttering and validated male and female PRS for self-reported stuttering in two independent stuttering datasets. This study provides insight into the genetic contributions to stuttering at the population level, demonstrating that genetic risk is complex, polygenic and dominated by low to modest genetic effects. After decades of progress examining the behavioral, neural and physiological contributions of language, articulation, speech–motor coordination, temperament and emotion to stuttering, the addition of genetic studies may help provide a mechanistic framework for integrating findings across these domains. We demonstrate shared molecular underpinnings between stuttering and other associated traits, including depression, autism and beat synchronization. An unresolved question in the field of stuttering, with lengthy historical speculation, is whether persistent stuttering and recovery from stuttering represent distinct subtypes[101](https://www.nature.com/articles/s41588-025-02267-2#ref-CR101 "Cox, N. J. & Kidd, K. K. Can recovery from stuttering be considered a genetically milder subtype of stuttering? Behav. Genet. 13, 129–139 (1983)."),[102](https://www.nature.com/articles/s41588-025-02267-2#ref-CR102 "Seider, R. A., Kidd, K. K. & Gladstien, K. L. Recovery and persistence of stuttering among relatives of stutterers. J. Speech Hear. Disord. 48, 402–409 (1983)."). Thus far, studies have yielded conflicting results with no clear answer[103](https://www.nature.com/articles/s41588-025-02267-2#ref-CR103 "Ambrose, N. G., Yairi, E., Loucks, T. M., Seery, C. H. & Throneburg, R. Relation of motor, linguistic and temperament factors in epidemiologic subtypes of persistent and recovered stuttering: initial findings. J. Fluen. Disord. 45, 12–26 (2015)."),[104](https://www.nature.com/articles/s41588-025-02267-2#ref-CR104 "Yairi, E., Ambrose, N. & Cox, N. Genetics of stuttering: a critical review. J. Speech Lang. Hear. Res. 39, 771–784 (1996)."),[105](https://www.nature.com/articles/s41588-025-02267-2#ref-CR105 "Yairi, E. Subtyping stuttering I: a review. J. Fluen. Disord. 32, 165–196 (2007)."); the analyses presented here motivate continued research into causal differences between females and males as well as between persistent and recovered stuttering. These findings represent a critical step toward the next era of research for this common, complex, costly and heritable condition.

Methods
-------

### Ethics

We have complied with all ethical guidelines. All participants provided informed consent to participate in the research. This study has been approved by Vanderbilt IRB (181575 and 180583).

### Studies

#### 23andMe, Inc

GWAS included research participants from 23andMe who self-reported stuttering status through a questionnaire. Cases included participants who answered ‘yes’ (99,776 individuals) to the question ‘Have you ever had a stammer or stutter?’ Controls (1,023,243 individuals) included participants who answered ‘no’ to this same question (Table [1](https://www.nature.com/articles/s41588-025-02267-2#Tab1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). As is common in population-based studies investigating stuttering, our study relies on self-report (see ref. [106](https://www.nature.com/articles/s41588-025-02267-2#ref-CR106 "Bloodstein, O. & Ratner, N. B. A Handbook on Stuttering 6th edn (Delmar Cengage Learning, 2008)."), in which all but two of the reviewed papers were based on retrospective questionnaire or interview-style surveys)[1](https://www.nature.com/articles/s41588-025-02267-2#ref-CR1 "Yairi, E. & Ambrose, N. Epidemiology of stuttering: 21st century advances. J. Fluen. Disord. 38, 66–87 (2013)."),[106](https://www.nature.com/articles/s41588-025-02267-2#ref-CR106 "Bloodstein, O. & Ratner, N. B. A Handbook on Stuttering 6th edn (Delmar Cengage Learning, 2008)."). All individuals included in the analyses provided informed consent and answered surveys online according to the 23andMe human subject protocol, which was reviewed and approved by Ethical & Independent Review Services, a private institutional review board ([http://www.eandireview.com](http://www.eandireview.com/)). Although developmental stuttering is by far the most common form of stuttering, the self-report phenotype in this study may also include other rarer forms of stuttering, such as acquired neurogenic stuttering. Compared to developmental stuttering, acquired stuttering is uncommon in clinical practice[107](https://www.nature.com/articles/s41588-025-02267-2#ref-CR107 "Theys, C., Van Wieringen, A., Sunaert, S., Thijs, V. & De Nil, L. F. A one year prospective study of neurogenic stuttering following stroke: incidence and co-occurring disorders. J. Commun. Disord. 44, 678–687 (2011)."). Therefore, although we expect the genetic signatures captured in these analyses to reflect effects for developmental stuttering, we use the general term ‘stuttering’ to describe the self-reported phenotype. Despite these phenotyping considerations, large-scale sample collection through surveys can dramatically increase power—a major limitation in genetic studies of stuttering thus far—and enable the discovery of robust and reproducible effects[108](https://www.nature.com/articles/s41588-025-02267-2#ref-CR108 "Wells, H. R. R. et al. GWAS identifies 44 independent associated genomic loci for self-reported adult hearing difficulty in UK Biobank. Am. J. Hum. Genet. 105, 788–802 (2019)."),[109](https://www.nature.com/articles/s41588-025-02267-2#ref-CR109 "Skelton, M. et al. Self‐reported medication use as an alternate phenotyping method for anxiety and depression in the UK Biobank. Am. J. Med. Genet. B Neuropsychiatr. Genet. 186, 389–398 (2021)."),[110](https://www.nature.com/articles/s41588-025-02267-2#ref-CR110 "Hyde, C. L. et al. Identification of 15 genetic loci associated with risk of major depression in individuals of European descent. Nat. Genet. 48, 1031–1036 (2016)."),[111](https://www.nature.com/articles/s41588-025-02267-2#ref-CR111 "Purves, K. L. et al. A major role for common genetic variation in anxiety disorders. Mol. Psychiatry 25, 3292–3303 (2020).").

#### ISP

We assessed the predictive performance of polygenic scores derived from 23andMe using participants with developmental stuttering from the ISP. Stuttering status in the ISP cohort was confirmed by speech–language pathologists with expertise in stuttering and fluency disorders (see ref. [30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022).") for a detailed description of this study and genotyping information).

#### Add Health

We also assessed the predictive performance of polygenic scores derived from 23andMe using participants who self-reported stuttering on an Add Health questionnaire. Add Health is an ongoing, longitudinal study investigating social, behavioral and biological factors that influence health and developmental outcomes from early adolescence through adulthood. Add Health collects general demographics, health survey data, in-home physical data and biological data from all participants (see ref. [32](https://www.nature.com/articles/s41588-025-02267-2#ref-CR32 "Harris, K. M. et al. Cohort profile: the National Longitudinal Study of Adolescent to Adult Health (Add Health). Int. J. Epidemiol. 48, 1415–1415k (2019).") for genotyping information). Stuttering cases were defined as participants who at any point answered ‘yes’ to the following survey question: ‘Do you have a problem with stuttering or stammering?’ All control individuals answered ‘no’ to the above question. Self-reported race/ethnicity was used to group participants.

### Statistical analysis

Eight ancestry-specific and sex-specific genome-wide association analyses were performed to determine variant association with stuttering (Table [1](https://www.nature.com/articles/s41588-025-02267-2#Tab1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Each GWAS used a logistic regression that assumed an additive model for allelic effects, where _P_ is the probability of self-reported stuttering:

$\\begin{array}{l}\\mathrm{ln}\\left(\\displaystyle\\frac{P}{1-P}\\right)={\\beta }\_{0}+{\\beta }\_{{\\mathrm{age}}}{\\mathrm{age}}+{\\beta }\_{{\\mathrm{pc}}.0}{\\mathrm{pc}}.0+{\\beta }\_{{\\mathrm{pc}}.1}{\\mathrm{pc}}.1+{\\beta }\_{{\\mathrm{pc}}.2}{\\mathrm{pc}}.2\\\\+{\\beta }\_{{\\mathrm{pc}}.3}{\\mathrm{pc}}.3+{\\beta }\_{{\\mathrm{pc}}.4}{\\mathrm{pc}}.4+{\\beta }\_{{\\mathrm{platform}}}{\\mathrm{platform}}+{\\beta }\_{{\\mathrm{genotype}}}{\\mathrm{genotype}}+{\\rm{\\varepsilon }}\\end{array}$

SNP significance was evaluated by a likelihood ratio test. Results for the X chromosome were computed similarly, in sex-stratified analyses with male genotypes coded as if they were homozygous diploid for the observed allele. Principal components (pc) for each logistic regression model were derived independently for each ancestry, using ~65,000 high-quality genotyped variants present across all five genotyping platforms. Principal components were computed on a subset of participants randomly sampled across all the genotyping platforms (137k, 102k, 1,000k and 360k participants were used for AFR, EAS, EUR and AMR, respectively). Principal components for participants not included in the analysis were obtained by projection, using the eigenvectors from the analysis and the SNP weights. Summary statistics were reported for imputed autosomal and X chromosome variants that were successfully imputed across all platforms (v2, v3, v4 and v5) and reached the following quality control thresholds: average _r_2 > 0.5, minimum _r_2 > 0.3, batch check _P_ > 1 × 10−50 and surpassed a minor allele count of 30. Loci in our primary analyses met a two-tiered multiple test correction: first, we applied a false discovery rate threshold of 5% across all autosomal and X chromosome variants; second, a traditional _P_ value threshold of 5 × 10−8 was applied. Sentinel variants were defined as the most significant variant found within a ±1 Mb window. The top 10,000 SNPs for the ancestry-specific and sex-specific GWAS can be found in Supplementary Data [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM4)–[8](https://www.nature.com/articles/s41588-025-02267-2#MOESM4). For more information on obtaining access to the full dataset, see Data availability.

For our secondary analyses, we performed ancestry-specific, sex-combined meta-analyses, using METAL with the inverse-variance weighted (STDERR command) option[36](https://www.nature.com/articles/s41588-025-02267-2#ref-CR36 "Willer, C. J., Li, Y. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics 26, 2190–2191 (2010).") ([https://genome.sph.umich.edu/wiki/METAL\_Documentation](https://genome.sph.umich.edu/wiki/METAL_Documentation)) to meta-analyze our EUR male and EUR female summary statistics, EAS male and EAS female summary statistics, AFR male and AFR female summary statistics and AMR male and AMR female summary statistics. Our sex-combined EUR summary statistics were used for partitioned heritability, genetic correlations and Mendelian randomization analyses.

We aggregated association summary statistics across ancestry-specific association studies using multi-ancestry meta-regression, as implemented in MR-MEGA[37](https://www.nature.com/articles/s41588-025-02267-2#ref-CR37 "Mägi, R. et al. Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Hum. Mol. Genet. 26, 3639–3650 (2017)."). Secondary analyses were also performed for the female-specific and the male-specific meta-analyses, and an ancestry-combined, sex-combined meta-analysis. We included three axes of genetic variation as covariates in the ancestry-combined, sex-combined meta-analysis and, given the lower number of contributing analyses in the female-specific and male-specific meta-analyses and limits on the number of possible axes of genetic variation, included one axis as a covariate in the sex-specific analyses. Resulting _P_ values were adjusted for genomic control.

### Annotation

The sentinel variant for each genome-wide significant locus was reported for each ancestry-specific and sex-specific study. Genome-wide significance[112](https://www.nature.com/articles/s41588-025-02267-2#ref-CR112 "Risch, N. & Merikangas, K. The future of genetic studies of complex human diseases. Science 273, 1516–1517 (1996).") was defined as _P_ < 5 × 10−8. Annotated gene(s) for each locus included the predicted functional gene(s) for each loci (when available) according to the Open Targets Genetics V2G pipeline, which combines evidence from molecular quantitative trait loci, chromatin interactions, in silico functional predictions from Ensembl and distance between the variant and gene canonical transcription start site[84](https://www.nature.com/articles/s41588-025-02267-2#ref-CR84 "Mountjoy, E. et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. Nat. Genet. 53, 1527–1533 (2021)."),[85](https://www.nature.com/articles/s41588-025-02267-2#ref-CR85 "Ghoussaini, M. et al. Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. Nucleic Acids Res. 49, D1311–D1320 (2021)."). Loci were defined according to independent LD blocks identified in 1000 Genomes reference data, using the matched ancestry reference. Reported sentinel variants represent the variant with the smallest _P_ value within each associated region. All reported positional coordinates (chromosome and base pair locations) refer to human genome reference build 37. We also checked for replication (consistency across sex and/or ancestry) of genome-wide significant signals in the primary and secondary GWAS analyses, with replication defined as a Bonferroni-adjusted correction for the number of variants and look-ups performed (_P_ < 1.92 × 10−4, using a Bonferroni correction; Supplementary Table [4](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)).

### Within-ancestry genetic correlation

To better understand the shared genetic relationship within ancestry groups, we performed genetic correlations within the EUR and EAS ancestries using LDSC[34](https://www.nature.com/articles/s41588-025-02267-2#ref-CR34 "Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat. Genet. 47, 291–295 (2015)."),[35](https://www.nature.com/articles/s41588-025-02267-2#ref-CR35 "Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Nat. Genet. 47, 1236–1241 (2015).") and relevant ancestry reference panels. EUR and EAS reference were generated using 1000 Genomes reference data and accessed through LDSC ([https://alkesgroup.broadinstitute.org/LDSCORE](https://alkesgroup.broadinstitute.org/LDSCORE)). Genetic correlations were not performed for the AFR and AMR ancestries, given that LDSC and other methods can produce biased results owing to admixture[113](https://www.nature.com/articles/s41588-025-02267-2#ref-CR113 "Luo, Y. et al. Estimating heritability and its enrichment in tissue-specific gene sets in admixed populations. Hum. Mol. Genet. 30, 1521–1534 (2021)."). Within-ancestry genetic correlation results can be found in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02267-2#MOESM3). Our within-ancestry genetic correlation results indicate there may be a strong relationship between EUR male and EUR female findings; however, there are differences in genome-wide significant sentinel variants (Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2)). Given that the LDSC approach and other common genetic correlation approaches rely on LD estimates, they can produce biased estimates of correlation when the data being compared have different LD patterns[113](https://www.nature.com/articles/s41588-025-02267-2#ref-CR113 "Luo, Y. et al. Estimating heritability and its enrichment in tissue-specific gene sets in admixed populations. Hum. Mol. Genet. 30, 1521–1534 (2021)."). Therefore, genetic correlation analyses were not performed within AFR and AMR analyses because genetic correlation approaches are biased in the presence of admixture, and correlations were also not tested across ancestry groups[113](https://www.nature.com/articles/s41588-025-02267-2#ref-CR113 "Luo, Y. et al. Estimating heritability and its enrichment in tissue-specific gene sets in admixed populations. Hum. Mol. Genet. 30, 1521–1534 (2021).").

### Variant effect size concordance analysis

We compared summary statistics from each ancestry-specific and sex-specific GWAS to all others to test for concordance between the summary statistics (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02267-2#MOESM3))[31](https://www.nature.com/articles/s41588-025-02267-2#ref-CR31 "Shaw, D. M. et al. Phenome risk classification enables phenotypic imputation and gene discovery in developmental stuttering. Am. J. Hum. Genet. 108, 2271–2283 (2021)."). As we were interested in investigating a broader genetic architecture, rather than including only variants with associations surpassing genome-wide significance, we examined variants with _P_ < 0.005. To establish our observed concordance rate, we determined the number of shared genetic variants with the same direction of effect divided by the total number of shared variants between the two datasets being compared. Given that datasets may have different LD patterns, we sought to estimate the expected concordance effects under the null hypothesis, preserving the LD structure observed in the data. To do so, we defined blocks of variants with the same direction of effect within 10 kb windows within each dataset. Next, we performed 25 permutations to randomly assign the direction of effect for these blocks to compute a simulated concordance rate for each permutation (defined as the number of shared genetic variants having the same direction of effect divided by the total number of shared variants between the two datasets). To establish our concordance rate under the null, we took the mean of the 25 simulated concordance rates. The rate of concordance expected under the null was used to compare the observed and expected concordance rates using a binomial _t_\-test.

### Effect size heterogeneity

To test for heterogeneity of effect estimates for our 24 sentinel variants from our eight ancestry-specific and sex-specific primary analyses, we used a Wald hypothesis test[114](https://www.nature.com/articles/s41588-025-02267-2#ref-CR114 "Clogg, C. C., Petkova, E. & Haritou, A. Statistical methods for comparing regression coefficients between models. Am. J. Sociol. 100, 1261–1293 (1995).") to examine the differential effects for each sentinel variant compared to the effects of the other seven primary GWAS. We calculated the test statistic, defined as the quotient of the difference in coefficients and the standard error of this difference. Under the null hypothesis positing equality of coefficients across subgroups, the test statistic conforms approximately to a standard normal distribution, as dictated by the principles of the Wald test. Significance was determined by using a Bonferroni-adjusted _P_ value threshold of 2.98 × 10−4, correcting for the 24 sentinel variants in our primary analysis multiplied by seven (the number of Wald test comparisons performed).

### SNP heritability and partitioned heritability

Genome-wide SNP-based heritability (_h_2) was calculated using summary statistics resulting from the EUR male and EUR female GWAS results using the LDSC software. We used LDSC to estimate liability-scaled _h_2, assuming a 10% lifetime population prevalence of stuttering based on the observed frequency of stuttering cases (Table [1](https://www.nature.com/articles/s41588-025-02267-2#Tab1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). EUR LD maps were generated using 1000 Genomes reference data and accessed through LDSC ([https://alkesgroup.broadinstitute.org/LDSCORE](https://alkesgroup.broadinstitute.org/LDSCORE)). Heritability calculations were not estimated for the AFR, AMR and EAS ancestries. For AFR and AMR, heritability estimates are often biased in the presence of admixture[113](https://www.nature.com/articles/s41588-025-02267-2#ref-CR113 "Luo, Y. et al. Estimating heritability and its enrichment in tissue-specific gene sets in admixed populations. Hum. Mol. Genet. 30, 1521–1534 (2021)."). For EAS, given the limited sample size, heritability estimates are likely to be unreliable[39](https://www.nature.com/articles/s41588-025-02267-2#ref-CR39 "The Neale Lab. Insights from estimates of SNP-heritability for >2,000 traits and disorders in UK Biobank. 
http://www.nealelab.is/blog/2017/9/20/insights-from-estimates-of-snp-heritability-for-2000-traits-and-disorders-in-uk-biobank
(2017).").

To better understand the types of variation that contribute most to stuttering, we estimated partitioned SNP heritability for our EUR male GWAS, EUR female GWAS and sex-combined EUR meta-analysis, using stratified LDSC[34](https://www.nature.com/articles/s41588-025-02267-2#ref-CR34 "Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat. Genet. 47, 291–295 (2015)."),[35](https://www.nature.com/articles/s41588-025-02267-2#ref-CR35 "Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Nat. Genet. 47, 1236–1241 (2015)."). LD scores, regression weights and allele frequencies for EUR populations were obtained from [https://alkesgroup.broadinstitute.org/LDSCORE](https://alkesgroup.broadinstitute.org/LDSCORE). We performed 80 different tests, resulting in a Bonferroni-corrected global significance threshold of _P_ < 6.25 × 10−4. Partitioning was performed for 52 baseline annotations as previously described[40](https://www.nature.com/articles/s41588-025-02267-2#ref-CR40 "Finucane, H. K. et al. Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types. Nat. Genet. 50, 621–629 (2018)."). Enrichment was considered significant for _P_ < 9.6 × 10−4, derived by Bonferroni correction for 52 gene sets.

Next, we tested for enrichment of cell-type-specific and tissue-specific heritability[40](https://www.nature.com/articles/s41588-025-02267-2#ref-CR40 "Finucane, H. K. et al. Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types. Nat. Genet. 50, 621–629 (2018).") in our EUR male GWAS, EUR female GWAS and sex-combined EUR meta-analysis, while controlling for the baseline models. Data for the brain cell types used to estimate enrichment of heritability consisted of neuron, astrocyte and oligodendrocyte data from a previous publication[115](https://www.nature.com/articles/s41588-025-02267-2#ref-CR115 "Cahoy, J. D. et al. A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. J. Neurosci. 28, 264–278 (2008)."). Enrichment tests were considered significant at _P_ < 0.017, derived by Bonferroni correction for three gene sets. Gene expression data (computed from GTEx Project data[116](https://www.nature.com/articles/s41588-025-02267-2#ref-CR116 "GTEx Consortium. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science 348, 648–660 (2015).")) used to estimate enrichment of heritability consisted of eight brain regions with empirical evidence of relation to stuttering[41](https://www.nature.com/articles/s41588-025-02267-2#ref-CR41 "Lu, C. et al. The neural substrates for atypical planning and execution of word production in stuttering. Exp. Neurol. 221, 146–156 (2010)."),[42](https://www.nature.com/articles/s41588-025-02267-2#ref-CR42 "Chang, S.-E., Garnett, E. O., Etchell, A. & Chow, H. M. Functional and neuroanatomical bases of developmental stuttering: current insights. Neuroscientist 25, 566–582 (2019)."),[43](https://www.nature.com/articles/s41588-025-02267-2#ref-CR43 "Etchell, A. C., Civier, O., Ballard, K. J. & Sowman, P. F. A systematic literature review of neuroimaging research on developmental stuttering between 1995 and 2016. J. Fluen. Disord. 55, 6–45 (2018)."),[44](https://www.nature.com/articles/s41588-025-02267-2#ref-CR44 "Liu, J. et al. A functional imaging study of self-regulatory capacities in persons who stutter. PLoS ONE 9, e89891 (2014)."),[45](https://www.nature.com/articles/s41588-025-02267-2#ref-CR45 "Neef, N. E. et al. Altered morphology of the nucleus accumbens in persistent developmental stuttering. J. Fluen. Disord. 55, 84–93 (2018)."),[46](https://www.nature.com/articles/s41588-025-02267-2#ref-CR46 "Toyomura, A., Fujii, T. & Kuriki, S. Effect of an 8-week practice of externally triggered speech on basal ganglia activity of stuttering and fluent speakers. NeuroImage 109, 458–468 (2015)."),[47](https://www.nature.com/articles/s41588-025-02267-2#ref-CR47 "Chang, S. E. & Zhu, D. C. Neural network connectivity differences in children who stutter. Brain 136, 3709–3726 (2013)."),[48](https://www.nature.com/articles/s41588-025-02267-2#ref-CR48 "Chang, S.-E., Horwitz, B., Ostuni, J., Reynolds, R. & Ludlow, C. L. Evidence of left inferior frontal–premotor structural and functional connectivity deficits in adults who stutter. Cereb. Cortex 21, 2507–2518 (2011)."). Enrichments were considered significant at _P_ < 6.25 × 10−3, derived by Bonferroni correction for eight gene sets. Lastly, 20 tissue-specific annotations of active chromatin sequences and enhancers, derived from the Roadmap Epigenomics consortium[117](https://www.nature.com/articles/s41588-025-02267-2#ref-CR117 "Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317–330 (2015).") and EN-TEx[40](https://www.nature.com/articles/s41588-025-02267-2#ref-CR40 "Finucane, H. K. et al. Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types. Nat. Genet. 50, 621–629 (2018)."),[118](https://www.nature.com/articles/s41588-025-02267-2#ref-CR118 "ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature 489, 57–74 (2012).") with epigenetic marks of monomethylation (me1), trimethylation (me3) and acetylation (ac) were used to estimate enrichment of heritability. Analyses were performed using epigenetic marks of H3K27ac, H3K9ac, H3K4me1, H3K4me3 and H3K36me3 in four brain regions previously associated with stuttering[41](https://www.nature.com/articles/s41588-025-02267-2#ref-CR41 "Lu, C. et al. The neural substrates for atypical planning and execution of word production in stuttering. Exp. Neurol. 221, 146–156 (2010)."),[43](https://www.nature.com/articles/s41588-025-02267-2#ref-CR43 "Etchell, A. C., Civier, O., Ballard, K. J. & Sowman, P. F. A systematic literature review of neuroimaging research on developmental stuttering between 1995 and 2016. J. Fluen. Disord. 55, 6–45 (2018)."),[44](https://www.nature.com/articles/s41588-025-02267-2#ref-CR44 "Liu, J. et al. A functional imaging study of self-regulatory capacities in persons who stutter. PLoS ONE 9, e89891 (2014)."),[46](https://www.nature.com/articles/s41588-025-02267-2#ref-CR46 "Toyomura, A., Fujii, T. & Kuriki, S. Effect of an 8-week practice of externally triggered speech on basal ganglia activity of stuttering and fluent speakers. NeuroImage 109, 458–468 (2015)."),[47](https://www.nature.com/articles/s41588-025-02267-2#ref-CR47 "Chang, S. E. & Zhu, D. C. Neural network connectivity differences in children who stutter. Brain 136, 3709–3726 (2013)."),[48](https://www.nature.com/articles/s41588-025-02267-2#ref-CR48 "Chang, S.-E., Horwitz, B., Ostuni, J., Reynolds, R. & Ludlow, C. L. Evidence of left inferior frontal–premotor structural and functional connectivity deficits in adults who stutter. Cereb. Cortex 21, 2507–2518 (2011)."). These active histone marks were considered significant at _P_ < 2.5 × 10−3, a Bonferroni adjustment for 20 gene sets.

### Validation of loci from the literature

Locus validation analysis was performed using methods detailed in a previous publication[30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022)."); however, the previously calculated effective number of tests was multiplied by eight, given that we checked for validation of signals across our eight ancestry-specific and sex-specific GWAS. As such, the effective number of tests used for our Bonferroni correction represented the number of independent tag SNPs in each gene, with pairwise _r_2 < 0.4 multiplied by eight. Gene validation results were Bonferroni-corrected for the effective number of tests in each gene, and we report the variant with the minimum _P_ value for each gene.

SNP-based replications looked for replication of the top hits reported in previous publications[30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022)."),[31](https://www.nature.com/articles/s41588-025-02267-2#ref-CR31 "Shaw, D. M. et al. Phenome risk classification enables phenotypic imputation and gene discovery in developmental stuttering. Am. J. Hum. Genet. 108, 2271–2283 (2021).") across all eight ancestry-specific and sex-specific studies. Replication for the 16 sentinel variants identified in ref. [30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022).") and replication results for the 11 sentinel variants identified in ref. [31](https://www.nature.com/articles/s41588-025-02267-2#ref-CR31 "Shaw, D. M. et al. Phenome risk classification enables phenotypic imputation and gene discovery in developmental stuttering. Am. J. Hum. Genet. 108, 2271–2283 (2021).") can be found in Supplementary Table [6](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

### Stuttering PRS model development

PRS models were trained using the EUR male and EUR female self-reported stuttering GWAS results in PRScs[119](https://www.nature.com/articles/s41588-025-02267-2#ref-CR119 "Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat. Commun. 10, 1776 (2019).") using a continuous shrinkage before adjusting individual SNP weights for LD and variant significance. Default auto-phi parameters were used in both the male-derived and female-derived models and were not optimized to prevent overfitting. EUR LD reference panels were constructed using the 1000 Genomes Project phase 3 EUR reference and used for EUR and AFR analyses. The male PRS model included 1,024,432 variant predictors, and the female PRS model included 1,024,431 variant predictors. Although the PRS we developed are not clinically useful, here we leveraged these scores to evaluate whether the risk of self-reported stuttering captured in the 23andMe analyses is predictive of risk in clinically validated cohorts. Each model was applied to both the ISP[30](https://www.nature.com/articles/s41588-025-02267-2#ref-CR30 "Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. HGG Adv. 3, 100073 (2022).") and Add Health samples[32](https://www.nature.com/articles/s41588-025-02267-2#ref-CR32 "Harris, K. M. et al. Cohort profile: the National Longitudinal Study of Adolescent to Adult Health (Add Health). Int. J. Epidemiol. 48, 1415–1415k (2019)."), matched according to ancestry and stratified by sex. The ISP testing set included 651 EUR male stuttering cases and 4,264 sex-matched controls, as well as 242 EUR female cases and 1,788 female controls; 48 AFR male stuttering cases and 308 sex-matched controls were included as well as 16 AFR female stuttering cases and 90 sex-matched controls. The Add Health testing set included 352 EUR male stuttering cases and 3,104 sex-matched controls; 236 EUR female cases and 3,517 female controls; 117 AFR male stuttering cases and 847 sex-matched controls were included as well as 107 AFR female stuttering cases and 1,101 sex-matched controls.

Genetic datasets were scored using PLINK (v.1.9)[120](https://www.nature.com/articles/s41588-025-02267-2#ref-CR120 "Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007)."). Genetic liability scores were _z_\-score-normalized. Liability score distributions between cases and controls were compared by Student’s two-sample _t_\-test. Predictive performance was interpreted using the AUC metric.

### Genetic correlation

The phenotypes chosen for the genetic correlation analyses were either found to be associated with stuttering within electronic health records at Vanderbilt University Medical Center[53](https://www.nature.com/articles/s41588-025-02267-2#ref-CR53 "Pruett, D. G. et al. Identifying developmental stuttering and associated comorbidities in electronic health records and creating a phenome risk classifier. J. Fluen. Disord. 68, 105847 (2021).") or previously associated with stuttering in other extant literature (see Supplementary Table [14](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) for phenotypes and references). Specifically, we compared our EUR male stuttering GWAS results to summary statistics from GWAS of EUR males and our EUR female stuttering GWAS results to summary statistics from GWAS of EUR females for 16 traits encompassing the following categories, for which sex-specific summary statistics were available: behavioral, circadian rhythm, immune, metabolic, motor, neurological and hearing traits. Sex-specific GWAS results were obtained from [http://www.nealelab.is/uk-biobank](http://www.nealelab.is/uk-biobank) (see Supplementary Table [15](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). We also explored the genetic correlation of stuttering with two traits for which sex-stratified summary statistics were not available to us: beat synchronization and autism (Supplementary Table [15](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). For beat synchronization and autism, we compared our EUR male, EUR female and sex-combined EUR summary statistics for each trait to the available sex-combined EUR GWAS summary statistics for the trait. Overall, we performed genetic correlation analyses for 18 traits previously associated with stuttering. A case sample size of >1,000 was required for binary traits. Owing to these constraints, genetic correlation analyses were only performed in the EUR-specific GWAS. All genetic correlation estimates were calculated using LDSC[34](https://www.nature.com/articles/s41588-025-02267-2#ref-CR34 "Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat. Genet. 47, 291–295 (2015)."),[35](https://www.nature.com/articles/s41588-025-02267-2#ref-CR35 "Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Nat. Genet. 47, 1236–1241 (2015)."). Results were Bonferroni-corrected for the number of tests performed (32 sex-specific analyses and six sex-combined analyses, _P_ < 1.32 × 10−3). Full results are shown in Supplementary Table [16](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

### Mendelian randomization

We performed debiased inverse-variance weighted[77](https://www.nature.com/articles/s41588-025-02267-2#ref-CR77 "Ye, T., Shao, J. & Kang, H. Debiased inverse-variance weighted estimator in two-sample summary-data Mendelian randomization. Ann. Stat. 49, 2079–2100 (2021).") Mendelian randomization analysis using the MendelianRandomization R package[121](https://www.nature.com/articles/s41588-025-02267-2#ref-CR121 "Yavorska, O. O. & Burgess, S. MendelianRandomization: an R package for performing Mendelian randomization analyses using summarized data. Int. J. Epidemiol. 46, 1734–1739 (2017)."). Debiased inverse-variance weighted Mendelian randomization can robustly handle weak instrument bias, as well as balanced horizontal pleiotropy[77](https://www.nature.com/articles/s41588-025-02267-2#ref-CR77 "Ye, T., Shao, J. & Kang, H. Debiased inverse-variance weighted estimator in two-sample summary-data Mendelian randomization. Ann. Stat. 49, 2079–2100 (2021)."),[78](https://www.nature.com/articles/s41588-025-02267-2#ref-CR78 "Sanderson, E. et al. Mendelian randomization. Nat. Rev. Methods Prim. 2, 6 (2022)."). We performed Mendelian randomization analysis for all associations that were significantly genetically correlated with stuttering (Fig. [2](https://www.nature.com/articles/s41588-025-02267-2#Fig2) and Supplementary Table [16](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Specifically, we performed a sex-specific Mendelian randomization analysis for depression, walking pace, BMI, ADHD, beat synchronization and autism, and sex-combined Mendelian randomization analyses for beat synchronization and autism. All of these traits have prior evidence of association with stuttering in the literature[49](https://www.nature.com/articles/s41588-025-02267-2#ref-CR49 "Choo, A. L., Smith, S. A. & Li, H. Associations between stuttering, comorbid conditions and executive function in children: a population-based study. BMC Psychol. 8, 113 (2020)."),[50](https://www.nature.com/articles/s41588-025-02267-2#ref-CR50 "Wieland, E. A., McAuley, J. D., Dilley, L. C. & Chang, S.-E. Evidence for a rhythm perception deficit in children who stutter. Brain Lang. 144, 26–34 (2015)."),[51](https://www.nature.com/articles/s41588-025-02267-2#ref-CR51 "Garnett, E. O. et al. Auditory rhythm discrimination in adults who stutter: an fMRI study. Brain Lang. 236, 105219 (2023)."),[52](https://www.nature.com/articles/s41588-025-02267-2#ref-CR52 "Ladányi, E., Persici, V., Fiveash, A., Tillmann, B. & Gordon, R. L. Is atypical rhythm a risk factor for developmental speech and language disorders? Wiley Interdiscip. Rev. Cogn. Sci. 11, e1528 (2020)."),[53](https://www.nature.com/articles/s41588-025-02267-2#ref-CR53 "Pruett, D. G. et al. Identifying developmental stuttering and associated comorbidities in electronic health records and creating a phenome risk classifier. J. Fluen. Disord. 68, 105847 (2021)."),[54](https://www.nature.com/articles/s41588-025-02267-2#ref-CR54 "Arenas, R. M., Walker, E. A. & Oleson, J. J. Developmental stuttering in children who are hard of hearing. Lang. Speech Hear. Serv. Sch. 48, 234–248 (2017)."),[55](https://www.nature.com/articles/s41588-025-02267-2#ref-CR55 "Strom, M. A. & Silverberg, J. I. Eczema is associated with childhood speech disorder: a retrospective analysis from the National Survey of Children’s Health and the National Health Interview Survey. J. Pediatrics 168, 185–192.e4 (2016)."),[56](https://www.nature.com/articles/s41588-025-02267-2#ref-CR56 "Ajdacic-Gross, V. et al. Subtypes of stuttering determined by latent class analysis in two Swiss epidemiological surveys. PLoS ONE 13, e0198450 (2018)."),[57](https://www.nature.com/articles/s41588-025-02267-2#ref-CR57 "Briley, P. M. & Merlo, S. Presence of allergies and their impact on sleep in children who stutter. Perspect. ASHA Spec. Interest Groups 5, 1454–1466 (2020)."),[58](https://www.nature.com/articles/s41588-025-02267-2#ref-CR58 "Jacobs, M. M., Merlo, S. & Briley, P. M. Sleep duration, insomnia, and stuttering: the relationship in adolescents and young adults. J. Commun. Disord. 91, 106106 (2021)."),[59](https://www.nature.com/articles/s41588-025-02267-2#ref-CR59 "Merlo, S. & Briley, P. M. Sleep problems in children who stutter: evidence from population data. J. Commun. Disord. 82, 105935 (2019)."),[60](https://www.nature.com/articles/s41588-025-02267-2#ref-CR60 "Mohammadi, H. et al. Sleep problems, social anxiety and stuttering severity in adults who do and adults who do not stutter. J. Clin. Med. 12, 161 (2023)."),[61](https://www.nature.com/articles/s41588-025-02267-2#ref-CR61 "Briley, P. M., Merlo, S. & Ellis, C. Sex differences in childhood stuttering and coexisting developmental disorders. J. Dev. Phys. Disabil. 34, 505–527 (2022)."),[62](https://www.nature.com/articles/s41588-025-02267-2#ref-CR62 "Lebrun, Y. Stuttering and epilepsy. J. Neurolinguist. 6, 433–444 (1991)."),[63](https://www.nature.com/articles/s41588-025-02267-2#ref-CR63 "Tomisato, S., Oishi, N., Asano, K., Watanabe, Y. & Ogawa, K. Developmental disability and psychiatric conditions in 39 patients with stuttering. Jpn. J. Logoped. Phoniatr. 57, 7–11 (2016)."),[64](https://www.nature.com/articles/s41588-025-02267-2#ref-CR64 "Tichenor, S. E., Palasik, S. & Yaruss, J. S. Understanding the broader impact of stuttering: suicidal ideation. Am. J. Speech Lang. Pathol. 32, 2087–2110 (2023)."),[65](https://www.nature.com/articles/s41588-025-02267-2#ref-CR65 "Briley, P. M., Gerlach, H. & Jacobs, M. M. Relationships between stuttering, depression, and suicidal ideation in young adults: accounting for gender differences. J. Fluen. Disord. 67, 105820 (2021)."),[66](https://www.nature.com/articles/s41588-025-02267-2#ref-CR66 "Bernard, R., Hofslundsengen, H. & Frazier Norbury, C. Anxiety and depression symptoms in children and adolescents who stutter: a systematic review and meta-analysis. J. Speech Lang. Hear. Res. 65, 624–644 (2022)."),[67](https://www.nature.com/articles/s41588-025-02267-2#ref-CR67 "Iverach, L. & Rapee, R. M. Social anxiety disorder and stuttering: current status and future directions. J. Fluen. Disord. 40, 69–82 (2014)."),[68](https://www.nature.com/articles/s41588-025-02267-2#ref-CR68 "Tichenor, S. E. & Yaruss, J. S. Stuttering as defined by adults who stutter. J. Speech Lang. Hear. Res. 62, 4356–4369 (2019)."),[69](https://www.nature.com/articles/s41588-025-02267-2#ref-CR69 "Tichenor, S. E., Johnson, C. A. & Yaruss, J. S. A preliminary investigation of attention-deficit/hyperactivity disorder characteristics in adults who stutter. J. Speech Lang. Hear. Res. 64, 839–853 (2021)."),[70](https://www.nature.com/articles/s41588-025-02267-2#ref-CR70 "Alm, P. A. Stuttering in relation to anxiety, temperament, and personality: review and analysis with focus on causality. J. Fluen. Disord. 40, 5–21 (2014)."),[71](https://www.nature.com/articles/s41588-025-02267-2#ref-CR71 "Pirinen, V. et al. A comprehensive analysis of speech disfluencies in autistic young adults and control young adults: group differences in typical, stuttering-like, and atypical disfluencies. J. Speech Lang. Hear. Res. 66, 832–848 (2023)."),[72](https://www.nature.com/articles/s41588-025-02267-2#ref-CR72 "Plexico, L. W., Cleary, J. E., McAlpine, A. & Plumb, A. M. Disfluency characteristics observed in young children with autism spectrum disorders: a preliminary report. Perspect. Fluen. Fluen. Disord. 20, 42–50 (2010)."),[73](https://www.nature.com/articles/s41588-025-02267-2#ref-CR73 "Scaler Scott, K., Tetnowski, J. A., Flaitz, J. R. & Yaruss, J. S. Preliminary study of disfluency in school‐aged children with autism. Int. J. Lang. Commun. Disord. 49, 75–89 (2014)."),[74](https://www.nature.com/articles/s41588-025-02267-2#ref-CR74 "Tetnowski, J. A. & Donaher, J. Stuttering and autism spectrum disorders: assessment and treatment. Semin. Speech Lang. 43, 117–129 (2022)."),[75](https://www.nature.com/articles/s41588-025-02267-2#ref-CR75 "Iverach, L. et al. Mood and substance use disorders among adults seeking speech treatment for stuttering. J. Speech Lang. Hear. Res. 53, 1178–1190 (2010)."),[76](https://www.nature.com/articles/s41588-025-02267-2#ref-CR76 "Heelan, M., McAllister, J. & Skinner, J. Stuttering, alcohol consumption and smoking. J. Fluen. Disord. 48, 27–34 (2016).") (see Supplementary Table [14](https://www.nature.com/articles/s41588-025-02267-2#MOESM3) for summary). All summary statistics for depression, walking pace, BMI, autism and beat synchronization were obtained from either [http://www.nealelab.is/uk-biobank](http://www.nealelab.is/uk-biobank), PGC+iPSYCH data[122](https://www.nature.com/articles/s41588-025-02267-2#ref-CR122 "Martin, J. et al. A genetic investigation of sex bias in the prevalence of attention-deficit/hyperactivity disorder. Biol. Psychiatry 83, 1044–1053 (2018).") or 23andMe[83](https://www.nature.com/articles/s41588-025-02267-2#ref-CR83 "Niarchou, M. et al. Genome-wide association study of musical beat synchronization demonstrates high polygenicity. Nat. Hum. Behav. 6, 1292–1309 (2022).")). Significant variants for each comorbid trait (_P_ < 5 × 10−6) were filtered to include only those tested in both the comorbid trait GWAS and the stuttering GWAS. Furthermore, independent instrumental SNPs were selected based on LD using the 1000 Genomes Project EUR reference data, retaining SNPs with _r_2 < 0.02 in 1,000 kb windows using PLINK[120](https://www.nature.com/articles/s41588-025-02267-2#ref-CR120 "Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007)."). Analysis details and results of other common Mendelian randomization methods are annotated in Supplementary Table [17](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

### GWAS Catalog

After filtering the GWAS Catalog[123](https://www.nature.com/articles/s41588-025-02267-2#ref-CR123 "Buniello, A. et al. The NHGRI-EBI GWAS catalog of published genome-wide association studies, targeted arrays and summary statistics 2019. Nucleic Acids Res. 47, D1005–D1012 (2019).") (release date, 2022-21-12) to contain only genome-wide significant loci (_P_ < 5.00 × 10−8), 48 unique genes from all ancestry-specific and sex-specific GWAS, sex-combined, ancestry-specific meta-analyses, ancestry-combined, sex-specific meta-analyses and ancestry-combined, sex-combined meta-analysis were included in the GWAS Catalog search. Traits associated with these genes in the GWAS Catalog were binned into 22 trait categories (Supplementary Table [19](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)). Our genome-wide significant hits and the associated GWAS Catalog findings can be found in Supplementary Table [19b](https://www.nature.com/articles/s41588-025-02267-2#Sec37). The number of unique genes per category can be seen in Supplementary Fig. [90](https://www.nature.com/articles/s41588-025-02267-2#MOESM1).

### Gene module enrichment

We performed an enrichment test for gene modules using our most likely genes for each identified signal in either the EUR male or EUR female GWAS to identify sets of highly correlated genes (gene modules) that were associated with stuttering risk (see [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1)). The top associated genes were determined for all variants with _P_ < 5.00 × 10−6 using the Open Targets Genetics V2G pipeline[84](https://www.nature.com/articles/s41588-025-02267-2#ref-CR84 "Mountjoy, E. et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. Nat. Genet. 53, 1527–1533 (2021)."),[85](https://www.nature.com/articles/s41588-025-02267-2#ref-CR85 "Ghoussaini, M. et al. Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. Nucleic Acids Res. 49, D1311–D1320 (2021)."). Gene co-expression networks, as previously defined[124](https://www.nature.com/articles/s41588-025-02267-2#ref-CR124 "Gerring, Z. F., Gamazon, E. R. & Derks, E. M. A gene co-expression network-based analysis of multiple brain tissues reveals novel genes and molecular pathways underlying major depression. PLoS Genet. 15, e1008245 (2019)."), consist of groups of functionally related genes or ‘modules’. Module enrichments were reported for any gene-tissue-specific analysis with a false discovery rate-adjusted _P_ value of <0.05 among any of the 49 available GTEx tissues. Using g:Profiler, we conducted a competitive gene pathway analysis for reported module enrichments, followed by annotation for the biological pathways identified (Supplementary Tables [20](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[24](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)).

### Colocalization

Bayesian colocalization analysis was performed between our EUR male and EUR female genome-wide significant associations and tissue-specific expression quantitative trait locus signals from GTEx V8 data[116](https://www.nature.com/articles/s41588-025-02267-2#ref-CR116 "GTEx Consortium. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science 348, 648–660 (2015).") using fast enrichment aided colocalization analysis (see [Supplementary Note](https://www.nature.com/articles/s41588-025-02267-2#MOESM1))[125](https://www.nature.com/articles/s41588-025-02267-2#ref-CR125 "Pividori, M. et al. PhenomeXcan: mapping the genome to the phenome through the transcriptome. Sci. Adv. 6, eaba2083 (2020)."),[126](https://www.nature.com/articles/s41588-025-02267-2#ref-CR126 "Wen, X., Pique-Regi, R. & Luca, F. Integrating molecular QTL data into genome-wide genetic association analysis: probabilistic assessment of enrichment and colocalization. PLoS Genet. 13, e1006646 (2017)."). We examined colocalization solely within regions where a variant was identified as a top hit (Table [2](https://www.nature.com/articles/s41588-025-02267-2#Tab2)). Evaluated regions included all sentinel variants in either the EUR male, EUR female, AFR male, AMR male or AMR female GWAS, as well as any other variant found in the same LD block. LD blocks were defined based on LD calculated in the 1000 Genomes EUR population reference data[127](https://www.nature.com/articles/s41588-025-02267-2#ref-CR127 "Berisa, T. & Pickrell, J. K. Approximately independent linkage disequilibrium blocks in human populations. Bioinformatics 32, 283–285 (2015)."). Colocalization analyses were tissue-specific and included all GTEx V8 tissues available. For significant colocalized signals, we report regional colocalization probability (that is, the probability that the association signal is in the shared region across stuttering and expression[128](https://www.nature.com/articles/s41588-025-02267-2#ref-CR128 "Hukku, A., Sampson, M. G., Luca, F., Pique-Regi, R. & Wen, X. Analyzing and reconciling colocalization and transcriptome-wide association studies from the perspective of inferential reproducibility. Am. J. Hum. Genet. 109, 825–837 (2022).")) of >0.05 (Supplementary Table [25](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)).

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02267-2#MOESM2) linked to this article.

Data availability
-----------------

Ancestry-specific and sex-specific summary statistics of self-reported stuttering will be made available through the 23andMe website to qualified researchers under agreement with 23andMe that protects the privacy of the 23andMe participants. Interested investigators should visit the 23andMe Publication Dataset Access Program at [https://research.23andme.com/dataset-access](https://research.23andme.com/dataset-access). The top 10,000 SNPs for all primary analyses are provided in Supplementary Data [1](https://www.nature.com/articles/s41588-025-02267-2#MOESM4)–[8](https://www.nature.com/articles/s41588-025-02267-2#MOESM4).

Code availability
-----------------

The code for the concordance analysis can be found on GitHub ([https://github.com/belowlab/Concordance-Analysis](https://github.com/belowlab/Concordance-Analysis)) and Zenodo ([https://doi.org/10.5281/zenodo.14884575](https://doi.org/10.5281/zenodo.14884575))[129](https://www.nature.com/articles/s41588-025-02267-2#ref-CR129 "Scartozzi, A. Concordance-analysis, version 1.0. Zenodo 
https://doi.org/10.5281/zenodo.14884575
(2025).").

References
----------

1.  Yairi, E. & Ambrose, N. Epidemiology of stuttering: 21st century advances. _J. Fluen. Disord._ **38**, 66–87 (2013).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epidemiology%20of%20stuttering%3A%2021st%20century%20advances&journal=J.%20Fluen.%20Disord.&volume=38&pages=66-87&publication_year=2013&author=Yairi%2CE&author=Ambrose%2CN) 
    
2.  Chang, S. E. et al. Stuttering: our current knowledge, research opportunities, and ways to address critical gaps. _Neurobiol. Lang. (Camb.)_ **6**, nol\_a\_00162 (2025).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=40201450)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stuttering%3A%20our%20current%20knowledge%2C%20research%20opportunities%2C%20and%20ways%20to%20address%20critical%20gaps&journal=Neurobiol.%20Lang.%20%28Camb.%29&volume=6&publication_year=2025&author=Chang%2CSE) 
    
3.  Andrews, G. & Harris, M. _The Syndrome of Stuttering._ Clinics in Developmental Medicine No. 17 (William Heinemann Medical Books, 1964).
    
4.  Månsson, H. Childhood stuttering. _J. Fluen. Disord._ **25**, 47–57 (2000).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Childhood%20stuttering&journal=J.%20Fluen.%20Disord.&volume=25&pages=47-57&publication_year=2000&author=M%C3%A5nsson%2CH) 
    
5.  Yairi, E. & Ambrose, N. A longitudinal study of stuttering in children: a preliminary report. _J. Speech Hear. Res._ **35**, 755–760 (1992).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK3s%2Fit1Gqtg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1405530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20longitudinal%20study%20of%20stuttering%20in%20children%3A%20a%20preliminary%20report&journal=J.%20Speech%20Hear.%20Res.&volume=35&pages=755-760&publication_year=1992&author=Yairi%2CE&author=Ambrose%2CN) 
    
6.  Baxter, S. et al. The state of the art in non‐pharmacological interventions for developmental stuttering. Part 1: a systematic review of effectiveness. _Int. J. Lang. Commun. Disord._ **50**, 676–718 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26123259)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4755200)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20state%20of%20the%20art%20in%20non%E2%80%90pharmacological%20interventions%20for%20developmental%20stuttering.%20Part%201%3A%20a%20systematic%20review%20of%20effectiveness&journal=Int.%20J.%20Lang.%20Commun.%20Disord.&volume=50&pages=676-718&publication_year=2015&author=Baxter%2CS) 
    
7.  Daniels, D. E. & Gabel, R. M. The impact of stuttering on identity construction. _Top. Lang. Disord._ **24**, 200–215 (2004).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20impact%20of%20stuttering%20on%20identity%20construction&journal=Top.%20Lang.%20Disord.&volume=24&pages=200-215&publication_year=2004&author=Daniels%2CDE&author=Gabel%2CRM) 
    
8.  Daniels, D. E., Gabel, R. M. & Hughes, S. Recounting the K–12 school experiences of adults who stutter: a qualitative analysis. _J. Fluen. Disord._ **37**, 71–82 (2012).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Recounting%20the%20K%E2%80%9312%20school%20experiences%20of%20adults%20who%20stutter%3A%20a%20qualitative%20analysis&journal=J.%20Fluen.%20Disord.&volume=37&pages=71-82&publication_year=2012&author=Daniels%2CDE&author=Gabel%2CRM&author=Hughes%2CS) 
    
9.  McAllister, J., Collier, J. & Shepstone, L. The impact of adolescent stuttering on educational and employment outcomes: evidence from a birth cohort study. _J. Fluen. Disord._ **37**, 106–121 (2012).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20impact%20of%20adolescent%20stuttering%20on%20educational%20and%20employment%20outcomes%3A%20evidence%20from%20a%20birth%20cohort%20study&journal=J.%20Fluen.%20Disord.&volume=37&pages=106-121&publication_year=2012&author=McAllister%2CJ&author=Collier%2CJ&author=Shepstone%2CL) 
    
10.  Klein, J. F. & Hood, S. B. The impact of stuttering on employment opportunities and job performance. _J. Fluen. Disord._ **29**, 255–273 (2004).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20impact%20of%20stuttering%20on%20employment%20opportunities%20and%20job%20performance&journal=J.%20Fluen.%20Disord.&volume=29&pages=255-273&publication_year=2004&author=Klein%2CJF&author=Hood%2CSB) 
    
11.  Craig, A., Blumgart, E. & Tran, Y. The impact of stuttering on the quality of life in adults who stutter. _J. Fluen. Disord._ **34**, 61–71 (2009).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20impact%20of%20stuttering%20on%20the%20quality%20of%20life%20in%20adults%20who%20stutter&journal=J.%20Fluen.%20Disord.&volume=34&pages=61-71&publication_year=2009&author=Craig%2CA&author=Blumgart%2CE&author=Tran%2CY) 
    
12.  Craig, A., Hancock, K., Tran, Y., Craig, M. & Peters, K. Epidemiology of stuttering in the community across the entire life span. _J. Speech Lang. Hear. Res._ **45**, 1097–1105 (2002).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12546480)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epidemiology%20of%20stuttering%20in%20the%20community%20across%20the%20entire%20life%20span&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=45&pages=1097-1105&publication_year=2002&author=Craig%2CA&author=Hancock%2CK&author=Tran%2CY&author=Craig%2CM&author=Peters%2CK) 
    
13.  Singer, C. M., Hessling, A., Kelly, E. M., Singer, L. & Jones, R. M. Clinical characteristics associated with stuttering persistence: a meta-analysis. _J. Speech Lang. Hear. Res._ **63**, 2995–3018 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32772868)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7890223)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20characteristics%20associated%20with%20stuttering%20persistence%3A%20a%20meta-analysis&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=63&pages=2995-3018&publication_year=2020&author=Singer%2CCM&author=Hessling%2CA&author=Kelly%2CEM&author=Singer%2CL&author=Jones%2CRM) 
    
14.  Singer, C. M., Otieno, S., Chang, S.-E. & Jones, R. M. Predicting persistent developmental stuttering using a cumulative risk approach. _J. Speech Lang. Hear. Res._ **65**, 70–95 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34902288)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Predicting%20persistent%20developmental%20stuttering%20using%20a%20cumulative%20risk%20approach&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=65&pages=70-95&publication_year=2022&author=Singer%2CCM&author=Otieno%2CS&author=Chang%2CS-E&author=Jones%2CRM) 
    
15.  Shugart, Y. Y. et al. Results of a genome-wide linkage scan for stuttering. _Am. J. Med. Genet._ **124A**, 133–135 (2004).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14699610)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Results%20of%20a%20genome-wide%20linkage%20scan%20for%20stuttering&journal=Am.%20J.%20Med.%20Genet.&volume=124A&pages=133-135&publication_year=2004&author=Shugart%2CYY) 
    
16.  Riaz, N. et al. Genomewide significant linkage to stuttering on chromosome 12. _Am. J. Hum. Genet._ **76**, 647–651 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXisleisLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15714404)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1199301)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomewide%20significant%20linkage%20to%20stuttering%20on%20chromosome%2012&journal=Am.%20J.%20Hum.%20Genet.&volume=76&pages=647-651&publication_year=2005&author=Riaz%2CN) 
    
17.  Suresh, R. et al. New complexities in the genetics of stuttering: significant sex-specific linkage signals. _Am. J. Hum. Genet._ **78**, 554–563 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xjt1GlsLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16532387)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1424690)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20complexities%20in%20the%20genetics%20of%20stuttering%3A%20significant%20sex-specific%20linkage%20signals&journal=Am.%20J.%20Hum.%20Genet.&volume=78&pages=554-563&publication_year=2006&author=Suresh%2CR) 
    
18.  Wittke-Thompson, J. K. et al. Genetic studies of stuttering in a founder population. _J. Fluen. Disord._ **32**, 33–50 (2007).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20studies%20of%20stuttering%20in%20a%20founder%20population&journal=J.%20Fluen.%20Disord.&volume=32&pages=33-50&publication_year=2007&author=Wittke-Thompson%2CJK) 
    
19.  Kang, C. et al. Mutations in the lysosomal enzyme-targeting pathway and persistent stuttering. _N. Engl. J. Med._ **362**, 677–685 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXisFOqsr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20147709)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2936507)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutations%20in%20the%20lysosomal%20enzyme-targeting%20pathway%20and%20persistent%20stuttering&journal=N.%20Engl.%20J.%20Med.&volume=362&pages=677-685&publication_year=2010&author=Kang%2CC) 
    
20.  Lan, J. et al. Association between dopaminergic genes (_SLC6A3_ and _DRD2_) and stuttering among Han Chinese. _J. Hum. Genet._ **54**, 457–460 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1ansrzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19590515)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20between%20dopaminergic%20genes%20%28SLC6A3%20and%20DRD2%29%20and%20stuttering%20among%20Han%20Chinese&journal=J.%20Hum.%20Genet.&volume=54&pages=457-460&publication_year=2009&author=Lan%2CJ) 
    
21.  Domingues, C. E. F. et al. A genetic linkage study in Brazil identifies a new locus for persistent developmental stuttering on chromosome 10. _Genet. Mol. Res._ **13**, 2094–2101 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVSlu77F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24737434)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genetic%20linkage%20study%20in%20Brazil%20identifies%20a%20new%20locus%20for%20persistent%20developmental%20stuttering%20on%20chromosome%2010&journal=Genet.%20Mol.%20Res.&volume=13&pages=2094-2101&publication_year=2014&author=Domingues%2CCEF) 
    
22.  Mohammadi, H. et al. Sex steroid hormones and sex hormone binding globulin levels, CYP17 MSP AI (−34 T:C) and CYP19 codon 39 (Trp:Arg) variants in children with developmental stuttering. _Brain Lang._ **175**, 47–56 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28992603)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sex%20steroid%20hormones%20and%20sex%20hormone%20binding%20globulin%20levels%2C%20CYP17%20MSP%20AI%20%28%E2%88%9234%20T%3AC%29%20and%20CYP19%20codon%2039%20%28Trp%3AArg%29%20variants%20in%20children%20with%20developmental%20stuttering&journal=Brain%20Lang.&volume=175&pages=47-56&publication_year=2017&author=Mohammadi%2CH) 
    
23.  Raza, M. H., Amjad, R., Riazuddin, S. & Drayna, D. Studies in a consanguineous family reveal a novel locus for stuttering on chromosome 16q. _Hum. Genet._ **131**, 311–313 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22205390)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Studies%20in%20a%20consanguineous%20family%20reveal%20a%20novel%20locus%20for%20stuttering%20on%20chromosome%2016q&journal=Hum.%20Genet.&volume=131&pages=311-313&publication_year=2012&author=Raza%2CMH&author=Amjad%2CR&author=Riazuddin%2CS&author=Drayna%2CD) 
    
24.  Raza, M. H. et al. Association between rare variants in AP4E1, a component of intracellular trafficking, and persistent stuttering. _Am. J. Hum. Genet._ **97**, 715–725 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsleksbfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26544806)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4667129)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20between%20rare%20variants%20in%20AP4E1%2C%20a%20component%20of%20intracellular%20trafficking%2C%20and%20persistent%20stuttering&journal=Am.%20J.%20Hum.%20Genet.&volume=97&pages=715-725&publication_year=2015&author=Raza%2CMH) 
    
25.  van Beijsterveldt, C. E. M., Felsenfeld, S. & Boomsma, D. I. Bivariate genetic analyses of stuttering and nonfluency in a large sample of 5-year-old twins. _J. Speech Lang. Hear. Res._ **53**, 609–619 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20029049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bivariate%20genetic%20analyses%20of%20stuttering%20and%20nonfluency%20in%20a%20large%20sample%20of%205-year-old%20twins&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=53&pages=609-619&publication_year=2010&author=Beijsterveldt%2CCEM&author=Felsenfeld%2CS&author=Boomsma%2CDI) 
    
26.  Fagnani, C., Fibiger, S., Skytthe, A. & Hjelmborg, J. V. B. Heritability and environmental effects for self-reported periods with stuttering: a twin study from Denmark. _Logoped. Phoniatr. Vocol._ **36**, 114–120 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21080843)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Heritability%20and%20environmental%20effects%20for%20self-reported%20periods%20with%20stuttering%3A%20a%20twin%20study%20from%20Denmark&journal=Logoped.%20Phoniatr.%20Vocol.&volume=36&pages=114-120&publication_year=2011&author=Fagnani%2CC&author=Fibiger%2CS&author=Skytthe%2CA&author=Hjelmborg%2CJVB) 
    
27.  Kazemi, N., Estiar, M. A., Fazilaty, H. & Sakhinia, E. Variants in _GNPTAB_, _GNPTG_ and _NAGPA_ genes are associated with stutterers. _Gene_ **647**, 93–100 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXlvFehug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29289611)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Variants%20in%20GNPTAB%2C%20GNPTG%20and%20NAGPA%20genes%20are%20associated%20with%20stutterers&journal=Gene&volume=647&pages=93-100&publication_year=2018&author=Kazemi%2CN&author=Estiar%2CMA&author=Fazilaty%2CH&author=Sakhinia%2CE) 
    
28.  Kang, C. et al. Evaluation of the association between polymorphisms at the _DRD2_ locus and stuttering. _J. Hum. Genet._ **56**, 472–473 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXnvFOktrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21390039)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3474249)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluation%20of%20the%20association%20between%20polymorphisms%20at%20the%20DRD2%20locus%20and%20stuttering&journal=J.%20Hum.%20Genet.&volume=56&pages=472-473&publication_year=2011&author=Kang%2CC) 
    
29.  Frigerio Domingues, C. E. et al. Are variants in sex hormone metabolizing genes associated with stuttering? _Brain Lang._ **191**, 28–30 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30798091)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6404540)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Are%20variants%20in%20sex%20hormone%20metabolizing%20genes%20associated%20with%20stuttering%3F&journal=Brain%20Lang.&volume=191&pages=28-30&publication_year=2019&author=Frigerio%20Domingues%2CCE) 
    
30.  Polikowsky, H. G. et al. Population-based genetic effects for developmental stuttering. _HGG Adv._ **3**, 100073 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35047858)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Population-based%20genetic%20effects%20for%20developmental%20stuttering&journal=HGG%20Adv.&volume=3&publication_year=2022&author=Polikowsky%2CHG) 
    
31.  Shaw, D. M. et al. Phenome risk classification enables phenotypic imputation and gene discovery in developmental stuttering. _Am. J. Hum. Genet._ **108**, 2271–2283 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1KmtL7J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34861174)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8715184)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phenome%20risk%20classification%20enables%20phenotypic%20imputation%20and%20gene%20discovery%20in%20developmental%20stuttering&journal=Am.%20J.%20Hum.%20Genet.&volume=108&pages=2271-2283&publication_year=2021&author=Shaw%2CDM) 
    
32.  Harris, K. M. et al. Cohort profile: the National Longitudinal Study of Adolescent to Adult Health (Add Health). _Int. J. Epidemiol._ **48**, 1415–1415k (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31257425)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857761)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cohort%20profile%3A%20the%20National%20Longitudinal%20Study%20of%20Adolescent%20to%20Adult%20Health%20%28Add%20Health%29&journal=Int.%20J.%20Epidemiol.&volume=48&pages=1415-1415k&publication_year=2019&author=Harris%2CKM) 
    
33.  Durand, E. Y., Do, C. B., Mountain, J. L. & Macpherson, J. M. Ancestry composition: a novel, efficient pipeline for ancestry deconvolution. Preprint at [https://doi.org/10.1101/010512](https://doi.org/10.1101/010512) (2014).
    
34.  Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. _Nat. Genet._ **47**, 291–295 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFCntb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25642630)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4495769)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LD%20Score%20regression%20distinguishes%20confounding%20from%20polygenicity%20in%20genome-wide%20association%20studies&journal=Nat.%20Genet.&volume=47&pages=291-295&publication_year=2015&author=Bulik-Sullivan%2CBK) 
    
35.  Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. _Nat. Genet._ **47**, 1236–1241 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7vM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26414676)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4797329)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20atlas%20of%20genetic%20correlations%20across%20human%20diseases%20and%20traits&journal=Nat.%20Genet.&volume=47&pages=1236-1241&publication_year=2015&author=Bulik-Sullivan%2CB) 
    
36.  Willer, C. J., Li, Y. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. _Bioinformatics_ **26**, 2190–2191 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVGru73F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20616382)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922887)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=METAL%3A%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&journal=Bioinformatics&volume=26&pages=2190-2191&publication_year=2010&author=Willer%2CCJ&author=Li%2CY&author=Abecasis%2CGR) 
    
37.  Mägi, R. et al. Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. _Hum. Mol. Genet._ **26**, 3639–3650 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28911207)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5755684)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trans-ethnic%20meta-regression%20of%20genome-wide%20association%20studies%20accounting%20for%20ancestry%20increases%20power%20for%20discovery%20and%20improves%20fine-mapping%20resolution&journal=Hum.%20Mol.%20Genet.&volume=26&pages=3639-3650&publication_year=2017&author=M%C3%A4gi%2CR) 
    
38.  Pirastu, N. et al. Genetic analyses identify widespread sex-differential participation bias. _Nat. Genet._ **53**, 663–671 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsVOgsbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33888908)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611642)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20analyses%20identify%20widespread%20sex-differential%20participation%20bias&journal=Nat.%20Genet.&volume=53&pages=663-671&publication_year=2021&author=Pirastu%2CN) 
    
39.  The Neale Lab. _Insights from estimates of SNP-heritability for >2,000 traits and disorders in UK Biobank_. [http://www.nealelab.is/blog/2017/9/20/insights-from-estimates-of-snp-heritability-for-2000-traits-and-disorders-in-uk-biobank](http://www.nealelab.is/blog/2017/9/20/insights-from-estimates-of-snp-heritability-for-2000-traits-and-disorders-in-uk-biobank) (2017).
    
40.  Finucane, H. K. et al. Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types. _Nat. Genet._ **50**, 621–629 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXnsVGrtr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29632380)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896795)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Heritability%20enrichment%20of%20specifically%20expressed%20genes%20identifies%20disease-relevant%20tissues%20and%20cell%20types&journal=Nat.%20Genet.&volume=50&pages=621-629&publication_year=2018&author=Finucane%2CHK) 
    
41.  Lu, C. et al. The neural substrates for atypical planning and execution of word production in stuttering. _Exp. Neurol._ **221**, 146–156 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19879262)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20neural%20substrates%20for%20atypical%20planning%20and%20execution%20of%20word%20production%20in%20stuttering&journal=Exp.%20Neurol.&volume=221&pages=146-156&publication_year=2010&author=Lu%2CC) 
    
42.  Chang, S.-E., Garnett, E. O., Etchell, A. & Chow, H. M. Functional and neuroanatomical bases of developmental stuttering: current insights. _Neuroscientist_ **25**, 566–582 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30264661)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20and%20neuroanatomical%20bases%20of%20developmental%20stuttering%3A%20current%20insights&journal=Neuroscientist&volume=25&pages=566-582&publication_year=2019&author=Chang%2CS-E&author=Garnett%2CEO&author=Etchell%2CA&author=Chow%2CHM) 
    
43.  Etchell, A. C., Civier, O., Ballard, K. J. & Sowman, P. F. A systematic literature review of neuroimaging research on developmental stuttering between 1995 and 2016. _J. Fluen. Disord._ **55**, 6–45 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20systematic%20literature%20review%20of%20neuroimaging%20research%20on%20developmental%20stuttering%20between%201995%20and%202016&journal=J.%20Fluen.%20Disord.&volume=55&pages=6-45&publication_year=2018&author=Etchell%2CAC&author=Civier%2CO&author=Ballard%2CKJ&author=Sowman%2CPF) 
    
44.  Liu, J. et al. A functional imaging study of self-regulatory capacities in persons who stutter. _PLoS ONE_ **9**, e89891 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24587104)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3937393)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20functional%20imaging%20study%20of%20self-regulatory%20capacities%20in%20persons%20who%20stutter&journal=PLoS%20ONE&volume=9&publication_year=2014&author=Liu%2CJ) 
    
45.  Neef, N. E. et al. Altered morphology of the nucleus accumbens in persistent developmental stuttering. _J. Fluen. Disord._ **55**, 84–93 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Altered%20morphology%20of%20the%20nucleus%20accumbens%20in%20persistent%20developmental%20stuttering&journal=J.%20Fluen.%20Disord.&volume=55&pages=84-93&publication_year=2018&author=Neef%2CNE) 
    
46.  Toyomura, A., Fujii, T. & Kuriki, S. Effect of an 8-week practice of externally triggered speech on basal ganglia activity of stuttering and fluent speakers. _NeuroImage_ **109**, 458–468 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25595501)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Effect%20of%20an%208-week%20practice%20of%20externally%20triggered%20speech%20on%20basal%20ganglia%20activity%20of%20stuttering%20and%20fluent%20speakers&journal=NeuroImage&volume=109&pages=458-468&publication_year=2015&author=Toyomura%2CA&author=Fujii%2CT&author=Kuriki%2CS) 
    
47.  Chang, S. E. & Zhu, D. C. Neural network connectivity differences in children who stutter. _Brain_ **136**, 3709–3726 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24131593)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3859219)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neural%20network%20connectivity%20differences%20in%20children%20who%20stutter&journal=Brain&volume=136&pages=3709-3726&publication_year=2013&author=Chang%2CSE&author=Zhu%2CDC) 
    
48.  Chang, S.-E., Horwitz, B., Ostuni, J., Reynolds, R. & Ludlow, C. L. Evidence of left inferior frontal–premotor structural and functional connectivity deficits in adults who stutter. _Cereb. Cortex_ **21**, 2507–2518 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21471556)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3183422)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evidence%20of%20left%20inferior%20frontal%E2%80%93premotor%20structural%20and%20functional%20connectivity%20deficits%20in%20adults%20who%20stutter&journal=Cereb.%20Cortex&volume=21&pages=2507-2518&publication_year=2011&author=Chang%2CS-E&author=Horwitz%2CB&author=Ostuni%2CJ&author=Reynolds%2CR&author=Ludlow%2CCL) 
    
49.  Choo, A. L., Smith, S. A. & Li, H. Associations between stuttering, comorbid conditions and executive function in children: a population-based study. _BMC Psychol._ **8**, 113 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33129350)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7603732)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Associations%20between%20stuttering%2C%20comorbid%20conditions%20and%20executive%20function%20in%20children%3A%20a%20population-based%20study&journal=BMC%20Psychol.&volume=8&publication_year=2020&author=Choo%2CAL&author=Smith%2CSA&author=Li%2CH) 
    
50.  Wieland, E. A., McAuley, J. D., Dilley, L. C. & Chang, S.-E. Evidence for a rhythm perception deficit in children who stutter. _Brain Lang._ **144**, 26–34 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25880903)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5382013)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evidence%20for%20a%20rhythm%20perception%20deficit%20in%20children%20who%20stutter&journal=Brain%20Lang.&volume=144&pages=26-34&publication_year=2015&author=Wieland%2CEA&author=McAuley%2CJD&author=Dilley%2CLC&author=Chang%2CS-E) 
    
51.  Garnett, E. O. et al. Auditory rhythm discrimination in adults who stutter: an fMRI study. _Brain Lang._ **236**, 105219 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36577315)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Auditory%20rhythm%20discrimination%20in%20adults%20who%20stutter%3A%20an%20fMRI%20study&journal=Brain%20Lang.&volume=236&publication_year=2023&author=Garnett%2CEO) 
    
52.  Ladányi, E., Persici, V., Fiveash, A., Tillmann, B. & Gordon, R. L. Is atypical rhythm a risk factor for developmental speech and language disorders? _Wiley Interdiscip. Rev. Cogn. Sci._ **11**, e1528 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32244259)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7415602)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Is%20atypical%20rhythm%20a%20risk%20factor%20for%20developmental%20speech%20and%20language%20disorders%3F&journal=Wiley%20Interdiscip.%20Rev.%20Cogn.%20Sci.&volume=11&publication_year=2020&author=Lad%C3%A1nyi%2CE&author=Persici%2CV&author=Fiveash%2CA&author=Tillmann%2CB&author=Gordon%2CRL) 
    
53.  Pruett, D. G. et al. Identifying developmental stuttering and associated comorbidities in electronic health records and creating a phenome risk classifier. _J. Fluen. Disord._ **68**, 105847 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identifying%20developmental%20stuttering%20and%20associated%20comorbidities%20in%20electronic%20health%20records%20and%20creating%20a%20phenome%20risk%20classifier&journal=J.%20Fluen.%20Disord.&volume=68&publication_year=2021&author=Pruett%2CDG) 
    
54.  Arenas, R. M., Walker, E. A. & Oleson, J. J. Developmental stuttering in children who are hard of hearing. _Lang. Speech Hear. Serv. Sch._ **48**, 234–248 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28915514)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5944374)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Developmental%20stuttering%20in%20children%20who%20are%20hard%20of%20hearing&journal=Lang.%20Speech%20Hear.%20Serv.%20Sch.&volume=48&pages=234-248&publication_year=2017&author=Arenas%2CRM&author=Walker%2CEA&author=Oleson%2CJJ) 
    
55.  Strom, M. A. & Silverberg, J. I. Eczema is associated with childhood speech disorder: a retrospective analysis from the National Survey of Children’s Health and the National Health Interview Survey. _J. Pediatrics_ **168**, 185–192.e4 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Eczema%20is%20associated%20with%20childhood%20speech%20disorder%3A%20a%20retrospective%20analysis%20from%20the%20National%20Survey%20of%20Children%E2%80%99s%20Health%20and%20the%20National%20Health%20Interview%20Survey&journal=J.%20Pediatrics&volume=168&pages=185-192.e4&publication_year=2016&author=Strom%2CMA&author=Silverberg%2CJI) 
    
56.  Ajdacic-Gross, V. et al. Subtypes of stuttering determined by latent class analysis in two Swiss epidemiological surveys. _PLoS ONE_ **13**, e0198450 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30086147)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6080750)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Subtypes%20of%20stuttering%20determined%20by%20latent%20class%20analysis%20in%20two%20Swiss%20epidemiological%20surveys&journal=PLoS%20ONE&volume=13&publication_year=2018&author=Ajdacic-Gross%2CV) 
    
57.  Briley, P. M. & Merlo, S. Presence of allergies and their impact on sleep in children who stutter. _Perspect. ASHA Spec. Interest Groups_ **5**, 1454–1466 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Presence%20of%20allergies%20and%20their%20impact%20on%20sleep%20in%20children%20who%20stutter&journal=Perspect.%20ASHA%20Spec.%20Interest%20Groups&volume=5&pages=1454-1466&publication_year=2020&author=Briley%2CPM&author=Merlo%2CS) 
    
58.  Jacobs, M. M., Merlo, S. & Briley, P. M. Sleep duration, insomnia, and stuttering: the relationship in adolescents and young adults. _J. Commun. Disord._ **91**, 106106 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34015644)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sleep%20duration%2C%20insomnia%2C%20and%20stuttering%3A%20the%20relationship%20in%20adolescents%20and%20young%20adults&journal=J.%20Commun.%20Disord.&volume=91&publication_year=2021&author=Jacobs%2CMM&author=Merlo%2CS&author=Briley%2CPM) 
    
59.  Merlo, S. & Briley, P. M. Sleep problems in children who stutter: evidence from population data. _J. Commun. Disord._ **82**, 105935 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31522013)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sleep%20problems%20in%20children%20who%20stutter%3A%20evidence%20from%20population%20data&journal=J.%20Commun.%20Disord.&volume=82&publication_year=2019&author=Merlo%2CS&author=Briley%2CPM) 
    
60.  Mohammadi, H. et al. Sleep problems, social anxiety and stuttering severity in adults who do and adults who do not stutter. _J. Clin. Med._ **12**, 161 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sleep%20problems%2C%20social%20anxiety%20and%20stuttering%20severity%20in%20adults%20who%20do%20and%20adults%20who%20do%20not%20stutter&journal=J.%20Clin.%20Med.&volume=12&publication_year=2023&author=Mohammadi%2CH) 
    
61.  Briley, P. M., Merlo, S. & Ellis, C. Sex differences in childhood stuttering and coexisting developmental disorders. _J. Dev. Phys. Disabil._ **34**, 505–527 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sex%20differences%20in%20childhood%20stuttering%20and%20coexisting%20developmental%20disorders&journal=J.%20Dev.%20Phys.%20Disabil.&volume=34&pages=505-527&publication_year=2022&author=Briley%2CPM&author=Merlo%2CS&author=Ellis%2CC) 
    
62.  Lebrun, Y. Stuttering and epilepsy. _J. Neurolinguist._ **6**, 433–444 (1991).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stuttering%20and%20epilepsy&journal=J.%20Neurolinguist.&volume=6&pages=433-444&publication_year=1991&author=Lebrun%2CY) 
    
63.  Tomisato, S., Oishi, N., Asano, K., Watanabe, Y. & Ogawa, K. Developmental disability and psychiatric conditions in 39 patients with stuttering. _Jpn. J. Logoped. Phoniatr._ **57**, 7–11 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Developmental%20disability%20and%20psychiatric%20conditions%20in%2039%20patients%20with%20stuttering&journal=Jpn.%20J.%20Logoped.%20Phoniatr.&volume=57&pages=7-11&publication_year=2016&author=Tomisato%2CS&author=Oishi%2CN&author=Asano%2CK&author=Watanabe%2CY&author=Ogawa%2CK) 
    
64.  Tichenor, S. E., Palasik, S. & Yaruss, J. S. Understanding the broader impact of stuttering: suicidal ideation. _Am. J. Speech Lang. Pathol._ **32**, 2087–2110 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37473446)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10561962)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%20the%20broader%20impact%20of%20stuttering%3A%20suicidal%20ideation&journal=Am.%20J.%20Speech%20Lang.%20Pathol.&volume=32&pages=2087-2110&publication_year=2023&author=Tichenor%2CSE&author=Palasik%2CS&author=Yaruss%2CJS) 
    
65.  Briley, P. M., Gerlach, H. & Jacobs, M. M. Relationships between stuttering, depression, and suicidal ideation in young adults: accounting for gender differences. _J. Fluen. Disord._ **67**, 105820 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Relationships%20between%20stuttering%2C%20depression%2C%20and%20suicidal%20ideation%20in%20young%20adults%3A%20accounting%20for%20gender%20differences&journal=J.%20Fluen.%20Disord.&volume=67&publication_year=2021&author=Briley%2CPM&author=Gerlach%2CH&author=Jacobs%2CMM) 
    
66.  Bernard, R., Hofslundsengen, H. & Frazier Norbury, C. Anxiety and depression symptoms in children and adolescents who stutter: a systematic review and meta-analysis. _J. Speech Lang. Hear. Res._ **65**, 624–644 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35084999)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Anxiety%20and%20depression%20symptoms%20in%20children%20and%20adolescents%20who%20stutter%3A%20a%20systematic%20review%20and%20meta-analysis&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=65&pages=624-644&publication_year=2022&author=Bernard%2CR&author=Hofslundsengen%2CH&author=Frazier%20Norbury%2CC) 
    
67.  Iverach, L. & Rapee, R. M. Social anxiety disorder and stuttering: current status and future directions. _J. Fluen. Disord._ **40**, 69–82 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Social%20anxiety%20disorder%20and%20stuttering%3A%20current%20status%20and%20future%20directions&journal=J.%20Fluen.%20Disord.&volume=40&pages=69-82&publication_year=2014&author=Iverach%2CL&author=Rapee%2CRM) 
    
68.  Tichenor, S. E. & Yaruss, J. S. Stuttering as defined by adults who stutter. _J. Speech Lang. Hear. Res._ **62**, 4356–4369 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31830837)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stuttering%20as%20defined%20by%20adults%20who%20stutter&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=62&pages=4356-4369&publication_year=2019&author=Tichenor%2CSE&author=Yaruss%2CJS) 
    
69.  Tichenor, S. E., Johnson, C. A. & Yaruss, J. S. A preliminary investigation of attention-deficit/hyperactivity disorder characteristics in adults who stutter. _J. Speech Lang. Hear. Res._ **64**, 839–853 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33647218)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20preliminary%20investigation%20of%20attention-deficit%2Fhyperactivity%20disorder%20characteristics%20in%20adults%20who%20stutter&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=64&pages=839-853&publication_year=2021&author=Tichenor%2CSE&author=Johnson%2CCA&author=Yaruss%2CJS) 
    
70.  Alm, P. A. Stuttering in relation to anxiety, temperament, and personality: review and analysis with focus on causality. _J. Fluen. Disord._ **40**, 5–21 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stuttering%20in%20relation%20to%20anxiety%2C%20temperament%2C%20and%20personality%3A%20review%20and%20analysis%20with%20focus%20on%20causality&journal=J.%20Fluen.%20Disord.&volume=40&pages=5-21&publication_year=2014&author=Alm%2CPA) 
    
71.  Pirinen, V. et al. A comprehensive analysis of speech disfluencies in autistic young adults and control young adults: group differences in typical, stuttering-like, and atypical disfluencies. _J. Speech Lang. Hear. Res._ **66**, 832–848 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36763844)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20comprehensive%20analysis%20of%20speech%20disfluencies%20in%20autistic%20young%20adults%20and%20control%20young%20adults%3A%20group%20differences%20in%20typical%2C%20stuttering-like%2C%20and%20atypical%20disfluencies&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=66&pages=832-848&publication_year=2023&author=Pirinen%2CV) 
    
72.  Plexico, L. W., Cleary, J. E., McAlpine, A. & Plumb, A. M. Disfluency characteristics observed in young children with autism spectrum disorders: a preliminary report. _Perspect. Fluen. Fluen. Disord._ **20**, 42–50 (2010).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disfluency%20characteristics%20observed%20in%20young%20children%20with%20autism%20spectrum%20disorders%3A%20a%20preliminary%20report&journal=Perspect.%20Fluen.%20Fluen.%20Disord.&volume=20&pages=42-50&publication_year=2010&author=Plexico%2CLW&author=Cleary%2CJE&author=McAlpine%2CA&author=Plumb%2CAM) 
    
73.  Scaler Scott, K., Tetnowski, J. A., Flaitz, J. R. & Yaruss, J. S. Preliminary study of disfluency in school‐aged children with autism. _Int. J. Lang. Commun. Disord._ **49**, 75–89 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24372887)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Preliminary%20study%20of%20disfluency%20in%20school%E2%80%90aged%20children%20with%20autism&journal=Int.%20J.%20Lang.%20Commun.%20Disord.&volume=49&pages=75-89&publication_year=2014&author=Scaler%20Scott%2CK&author=Tetnowski%2CJA&author=Flaitz%2CJR&author=Yaruss%2CJS) 
    
74.  Tetnowski, J. A. & Donaher, J. Stuttering and autism spectrum disorders: assessment and treatment. _Semin. Speech Lang._ **43**, 117–129 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35697034)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stuttering%20and%20autism%20spectrum%20disorders%3A%20assessment%20and%20treatment&journal=Semin.%20Speech%20Lang.&volume=43&pages=117-129&publication_year=2022&author=Tetnowski%2CJA&author=Donaher%2CJ) 
    
75.  Iverach, L. et al. Mood and substance use disorders among adults seeking speech treatment for stuttering. _J. Speech Lang. Hear. Res._ **53**, 1178–1190 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20643799)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mood%20and%20substance%20use%20disorders%20among%20adults%20seeking%20speech%20treatment%20for%20stuttering&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=53&pages=1178-1190&publication_year=2010&author=Iverach%2CL) 
    
76.  Heelan, M., McAllister, J. & Skinner, J. Stuttering, alcohol consumption and smoking. _J. Fluen. Disord._ **48**, 27–34 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stuttering%2C%20alcohol%20consumption%20and%20smoking&journal=J.%20Fluen.%20Disord.&volume=48&pages=27-34&publication_year=2016&author=Heelan%2CM&author=McAllister%2CJ&author=Skinner%2CJ) 
    
77.  Ye, T., Shao, J. & Kang, H. Debiased inverse-variance weighted estimator in two-sample summary-data Mendelian randomization. _Ann. Stat._ **49**, 2079–2100 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Debiased%20inverse-variance%20weighted%20estimator%20in%20two-sample%20summary-data%20Mendelian%20randomization&journal=Ann.%20Stat.&volume=49&pages=2079-2100&publication_year=2021&author=Ye%2CT&author=Shao%2CJ&author=Kang%2CH) 
    
78.  Sanderson, E. et al. Mendelian randomization. _Nat. Rev. Methods Prim._ **2**, 6 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVCqtbY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mendelian%20randomization&journal=Nat.%20Rev.%20Methods%20Prim.&volume=2&publication_year=2022&author=Sanderson%2CE) 
    
79.  Mahajan, A. et al. Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation. _Nat. Genet._ **54**, 560–572 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1ynsbvJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35551307)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9179018)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-ancestry%20genetic%20study%20of%20type%202%20diabetes%20highlights%20the%20power%20of%20diverse%20populations%20for%20discovery%20and%20translation&journal=Nat.%20Genet.&volume=54&pages=560-572&publication_year=2022&author=Mahajan%2CA) 
    
80.  Hautakangas, H. et al. Genome-wide analysis of 102,084 migraine cases identifies 123 risk loci and subtype-specific risk alleles. _Nat. Genet._ **54**, 152–160 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XislKnsLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35115687)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8837554)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20analysis%20of%20102%2C084%20migraine%20cases%20identifies%20123%20risk%20loci%20and%20subtype-specific%20risk%20alleles&journal=Nat.%20Genet.&volume=54&pages=152-160&publication_year=2022&author=Hautakangas%2CH) 
    
81.  Jansen, P. R. et al. Genome-wide analysis of insomnia in 1,331,010 individuals identifies new risk loci and functional pathways. _Nat. Genet._ **51**, 394–403 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmsVyqs78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30804565)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20analysis%20of%20insomnia%20in%201%2C331%2C010%20individuals%20identifies%20new%20risk%20loci%20and%20functional%20pathways&journal=Nat.%20Genet.&volume=51&pages=394-403&publication_year=2019&author=Jansen%2CPR) 
    
82.  Xue, A. et al. Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes. _Nat. Commun._ **9**, 2941 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30054458)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6063971)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20analyses%20identify%20143%20risk%20variants%20and%20putative%20regulatory%20mechanisms%20for%20type%202%20diabetes&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Xue%2CA) 
    
83.  Niarchou, M. et al. Genome-wide association study of musical beat synchronization demonstrates high polygenicity. _Nat. Hum. Behav._ **6**, 1292–1309 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35710621)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9489530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20of%20musical%20beat%20synchronization%20demonstrates%20high%20polygenicity&journal=Nat.%20Hum.%20Behav.&volume=6&pages=1292-1309&publication_year=2022&author=Niarchou%2CM) 
    
84.  Mountjoy, E. et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. _Nat. Genet._ **53**, 1527–1533 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlOlurbE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34711957)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7611956)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20open%20approach%20to%20systematically%20prioritize%20causal%20variants%20and%20genes%20at%20all%20published%20human%20GWAS%20trait-associated%20loci&journal=Nat.%20Genet.&volume=53&pages=1527-1533&publication_year=2021&author=Mountjoy%2CE) 
    
85.  Ghoussaini, M. et al. Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. _Nucleic Acids Res._ **49**, D1311–D1320 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFOnsLnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33045747)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Open%20Targets%20Genetics%3A%20systematic%20identification%20of%20trait-associated%20genes%20using%20large-scale%20genetics%20and%20functional%20genomics&journal=Nucleic%20Acids%20Res.&volume=49&pages=D1311-D1320&publication_year=2021&author=Ghoussaini%2CM) 
    
86.  Machiela, M. J. & Chanock, S. J. LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. _Bioinformatics_ **31**, 3555–3557 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVersbfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26139635)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4626747)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LDlink%3A%20a%20web-based%20application%20for%20exploring%20population-specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants&journal=Bioinformatics&volume=31&pages=3555-3557&publication_year=2015&author=Machiela%2CMJ&author=Chanock%2CSJ) 
    
87.  Wingate, M. E. & Howell, P. Foundations of stuttering. _J. Acoust. Soc. Am._ **112**, 1229–1231 (2002).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Foundations%20of%20stuttering&journal=J.%20Acoust.%20Soc.%20Am.&volume=112&pages=1229-1231&publication_year=2002&author=Wingate%2CME&author=Howell%2CP) 
    
88.  Brady, J. P. Metronome-conditioned speech retraining for stuttering. _Behav. Ther._ **2**, 129–150 (1971).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Metronome-conditioned%20speech%20retraining%20for%20stuttering&journal=Behav.%20Ther.&volume=2&pages=129-150&publication_year=1971&author=Brady%2CJP) 
    
89.  Brady, J. P. Studies on the metronome effect on stuttering. _Behav. Res. Ther._ **7**, 197–204 (1969).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaF1M3osVCgsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=5808691)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Studies%20on%20the%20metronome%20effect%20on%20stuttering&journal=Behav.%20Res.%20Ther.&volume=7&pages=197-204&publication_year=1969&author=Brady%2CJP) 
    
90.  Hosseini, R., Walsh, B., Tian, F. & Wang, S. An fNIRS-based feature learning and classification framework to distinguish hemodynamic patterns in children who stutter. _IEEE Trans. Neural Syst. Rehabil. Eng._ **26**, 1254–1263 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29877850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6242270)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20fNIRS-based%20feature%20learning%20and%20classification%20framework%20to%20distinguish%20hemodynamic%20patterns%20in%20children%20who%20stutter&journal=IEEE%20Trans.%20Neural%20Syst.%20Rehabil.%20Eng.&volume=26&pages=1254-1263&publication_year=2018&author=Hosseini%2CR&author=Walsh%2CB&author=Tian%2CF&author=Wang%2CS) 
    
91.  Hartwigsen, G., Neef, N. E., Camilleri, J. A., Margulies, D. S. & Eickhoff, S. B. Functional segregation of the right inferior frontal gyrus: evidence from coactivation-based parcellation. _Cereb. Cortex_ **29**, 1532–1546 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29912435)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20segregation%20of%20the%20right%20inferior%20frontal%20gyrus%3A%20evidence%20from%20coactivation-based%20parcellation&journal=Cereb.%20Cortex&volume=29&pages=1532-1546&publication_year=2019&author=Hartwigsen%2CG&author=Neef%2CNE&author=Camilleri%2CJA&author=Margulies%2CDS&author=Eickhoff%2CSB) 
    
92.  Chesters, J., Möttönen, R. & Watkins, K.E. Neural changes after training with transcranial direct current stimulation to increase speech fluency in adults who stutter. Preprint at [https://doi.org/10.31219/osf.io/8st3j](https://doi.org/10.31219/osf.io/8st3j) (2021).
    
93.  Loucks, T., Kraft, S. J., Choo, A. L., Sharma, H. & Ambrose, N. G. Functional brain activation differences in stuttering identified with a rapid fMRI sequence. _J. Fluen. Disord._ **36**, 302–307 (2011).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20brain%20activation%20differences%20in%20stuttering%20identified%20with%20a%20rapid%20fMRI%20sequence&journal=J.%20Fluen.%20Disord.&volume=36&pages=302-307&publication_year=2011&author=Loucks%2CT&author=Kraft%2CSJ&author=Choo%2CAL&author=Sharma%2CH&author=Ambrose%2CNG) 
    
94.  Chow, H. M. et al. Linking lysosomal enzyme targeting genes and energy metabolism with altered gray matter volume in children with persistent stuttering. _Neurobiol. Lang._ **1**, 365–380 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Linking%20lysosomal%20enzyme%20targeting%20genes%20and%20energy%20metabolism%20with%20altered%20gray%20matter%20volume%20in%20children%20with%20persistent%20stuttering&journal=Neurobiol.%20Lang.&volume=1&pages=365-380&publication_year=2020&author=Chow%2CHM) 
    
95.  Austin-Zimmerman, I. et al. Genome-wide association studies and cross-population meta-analyses investigating short and long sleep duration. _Nat. Commun._ **14**, 6059 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVCkt77I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37770476)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10539313)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20studies%20and%20cross-population%20meta-analyses%20investigating%20short%20and%20long%20sleep%20duration&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Austin-Zimmerman%2CI) 
    
96.  Schoeler, T. et al. Participation bias in the UK Biobank distorts genetic associations and downstream analyses. _Nat. Hum. Behav._ **7**, 1216–1227 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37106081)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10365993)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Participation%20bias%20in%20the%20UK%20Biobank%20distorts%20genetic%20associations%20and%20downstream%20analyses&journal=Nat.%20Hum.%20Behav.&volume=7&pages=1216-1227&publication_year=2023&author=Schoeler%2CT) 
    
97.  Kim, K. S., Daliri, A., Flanagan, J. R. & Max, L. Dissociated development of speech and limb sensorimotor learning in stuttering: Speech auditory-motor learning is impaired in both children and adults who stutter. _Neuroscience_ **451**, 1–21 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1Cqu73E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33091464)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dissociated%20development%20of%20speech%20and%20limb%20sensorimotor%20learning%20in%20stuttering%3A%20Speech%20auditory-motor%20learning%20is%20impaired%20in%20both%20children%20and%20adults%20who%20stutter&journal=Neuroscience&volume=451&pages=1-21&publication_year=2020&author=Kim%2CKS&author=Daliri%2CA&author=Flanagan%2CJR&author=Max%2CL) 
    
98.  Ardila, A. et al. An epidemiologic study of stuttering. _J. Commun. Disord._ **27**, 37–48 (1994).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK2c3nsV2isQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8006205)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20epidemiologic%20study%20of%20stuttering&journal=J.%20Commun.%20Disord.&volume=27&pages=37-48&publication_year=1994&author=Ardila%2CA) 
    
99.  Corcoran, J. A. & Stewart, M. Stories of stuttering. _J. Fluen. Disord._ **23**, 247–264 (1998).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stories%20of%20stuttering&journal=J.%20Fluen.%20Disord.&volume=23&pages=247-264&publication_year=1998&author=Corcoran%2CJA&author=Stewart%2CM) 
    
100.  Boyle, M. P. Enacted stigma and felt stigma experienced by adults who stutter. _J. Commun. Disord._ **73**, 50–61 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29574262)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enacted%20stigma%20and%20felt%20stigma%20experienced%20by%20adults%20who%20stutter&journal=J.%20Commun.%20Disord.&volume=73&pages=50-61&publication_year=2018&author=Boyle%2CMP) 
    
101.  Cox, N. J. & Kidd, K. K. Can recovery from stuttering be considered a genetically milder subtype of stuttering? _Behav. Genet._ **13**, 129–139 (1983).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL3s3jtVKisg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=6860250)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Can%20recovery%20from%20stuttering%20be%20considered%20a%20genetically%20milder%20subtype%20of%20stuttering%3F&journal=Behav.%20Genet.&volume=13&pages=129-139&publication_year=1983&author=Cox%2CNJ&author=Kidd%2CKK) 
    
102.  Seider, R. A., Kidd, K. K. & Gladstien, K. L. Recovery and persistence of stuttering among relatives of stutterers. _J. Speech Hear. Disord._ **48**, 402–409 (1983).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL2c%2Fms1WgsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=6645435)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Recovery%20and%20persistence%20of%20stuttering%20among%20relatives%20of%20stutterers&journal=J.%20Speech%20Hear.%20Disord.&volume=48&pages=402-409&publication_year=1983&author=Seider%2CRA&author=Kidd%2CKK&author=Gladstien%2CKL) 
    
103.  Ambrose, N. G., Yairi, E., Loucks, T. M., Seery, C. H. & Throneburg, R. Relation of motor, linguistic and temperament factors in epidemiologic subtypes of persistent and recovered stuttering: initial findings. _J. Fluen. Disord._ **45**, 12–26 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Relation%20of%20motor%2C%20linguistic%20and%20temperament%20factors%20in%20epidemiologic%20subtypes%20of%20persistent%20and%20recovered%20stuttering%3A%20initial%20findings&journal=J.%20Fluen.%20Disord.&volume=45&pages=12-26&publication_year=2015&author=Ambrose%2CNG&author=Yairi%2CE&author=Loucks%2CTM&author=Seery%2CCH&author=Throneburg%2CR) 
    
104.  Yairi, E., Ambrose, N. & Cox, N. Genetics of stuttering: a critical review. _J. Speech Lang. Hear. Res._ **39**, 771–784 (1996).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK2s%2FgtFGltw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetics%20of%20stuttering%3A%20a%20critical%20review&journal=J.%20Speech%20Lang.%20Hear.%20Res.&volume=39&pages=771-784&publication_year=1996&author=Yairi%2CE&author=Ambrose%2CN&author=Cox%2CN) 
    
105.  Yairi, E. Subtyping stuttering I: a review. _J. Fluen. Disord._ **32**, 165–196 (2007).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Subtyping%20stuttering%20I%3A%20a%20review&journal=J.%20Fluen.%20Disord.&volume=32&pages=165-196&publication_year=2007&author=Yairi%2CE) 
    
106.  Bloodstein, O. & Ratner, N. B. _A Handbook on Stuttering_ 6th edn (Delmar Cengage Learning, 2008).
    
107.  Theys, C., Van Wieringen, A., Sunaert, S., Thijs, V. & De Nil, L. F. A one year prospective study of neurogenic stuttering following stroke: incidence and co-occurring disorders. _J. Commun. Disord._ **44**, 678–687 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC3Mbos1ektQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21807377)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20one%20year%20prospective%20study%20of%20neurogenic%20stuttering%20following%20stroke%3A%20incidence%20and%20co-occurring%20disorders&journal=J.%20Commun.%20Disord.&volume=44&pages=678-687&publication_year=2011&author=Theys%2CC&author=Wieringen%2CA&author=Sunaert%2CS&author=Thijs%2CV&author=Nil%2CLF) 
    
108.  Wells, H. R. R. et al. GWAS identifies 44 independent associated genomic loci for self-reported adult hearing difficulty in UK Biobank. _Am. J. Hum. Genet._ **105**, 788–802 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVylsb3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31564434)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6817556)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GWAS%20identifies%2044%20independent%20associated%20genomic%20loci%20for%20self-reported%20adult%20hearing%20difficulty%20in%20UK%20Biobank&journal=Am.%20J.%20Hum.%20Genet.&volume=105&pages=788-802&publication_year=2019&author=Wells%2CHRR) 
    
109.  Skelton, M. et al. Self‐reported medication use as an alternate phenotyping method for anxiety and depression in the UK Biobank. _Am. J. Med. Genet. B Neuropsychiatr. Genet._ **186**, 389–398 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34658127)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Self%E2%80%90reported%20medication%20use%20as%20an%20alternate%20phenotyping%20method%20for%20anxiety%20and%20depression%20in%20the%20UK%20Biobank&journal=Am.%20J.%20Med.%20Genet.%20B%20Neuropsychiatr.%20Genet.&volume=186&pages=389-398&publication_year=2021&author=Skelton%2CM) 
    
110.  Hyde, C. L. et al. Identification of 15 genetic loci associated with risk of major depression in individuals of European descent. _Nat. Genet._ **48**, 1031–1036 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1KlsrrJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27479909)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5706769)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%2015%20genetic%20loci%20associated%20with%20risk%20of%20major%20depression%20in%20individuals%20of%20European%20descent&journal=Nat.%20Genet.&volume=48&pages=1031-1036&publication_year=2016&author=Hyde%2CCL) 
    
111.  Purves, K. L. et al. A major role for common genetic variation in anxiety disorders. _Mol. Psychiatry_ **25**, 3292–3303 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlWqtbvN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31748690)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20major%20role%20for%20common%20genetic%20variation%20in%20anxiety%20disorders&journal=Mol.%20Psychiatry&volume=25&pages=3292-3303&publication_year=2020&author=Purves%2CKL) 
    
112.  Risch, N. & Merikangas, K. The future of genetic studies of complex human diseases. _Science_ **273**, 1516–1517 (1996).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28Xls1Wqs7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8801636)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20future%20of%20genetic%20studies%20of%20complex%20human%20diseases&journal=Science&volume=273&pages=1516-1517&publication_year=1996&author=Risch%2CN&author=Merikangas%2CK) 
    
113.  Luo, Y. et al. Estimating heritability and its enrichment in tissue-specific gene sets in admixed populations. _Hum. Mol. Genet._ **30**, 1521–1534 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVCktr%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33987664)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8330913)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estimating%20heritability%20and%20its%20enrichment%20in%20tissue-specific%20gene%20sets%20in%20admixed%20populations&journal=Hum.%20Mol.%20Genet.&volume=30&pages=1521-1534&publication_year=2021&author=Luo%2CY) 
    
114.  Clogg, C. C., Petkova, E. & Haritou, A. Statistical methods for comparing regression coefficients between models. _Am. J. Sociol._ **100**, 1261–1293 (1995).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Statistical%20methods%20for%20comparing%20regression%20coefficients%20between%20models&journal=Am.%20J.%20Sociol.&volume=100&pages=1261-1293&publication_year=1995&author=Clogg%2CCC&author=Petkova%2CE&author=Haritou%2CA) 
    
115.  Cahoy, J. D. et al. A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. _J. Neurosci._ **28**, 264–278 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtV2ltw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18171944)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6671143)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20transcriptome%20database%20for%20astrocytes%2C%20neurons%2C%20and%20oligodendrocytes%3A%20a%20new%20resource%20for%20understanding%20brain%20development%20and%20function&journal=J.%20Neurosci.&volume=28&pages=264-278&publication_year=2008&author=Cahoy%2CJD) 
    
116.  GTEx Consortium. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. _Science_ **348**, 648–660 (2015).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4547484)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Genotype-Tissue%20Expression%20%28GTEx%29%20pilot%20analysis%3A%20multitissue%20gene%20regulation%20in%20humans&journal=Science&volume=348&pages=648-660&publication_year=2015) 
    
117.  Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. _Nature_ **518**, 317–330 (2015).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4530010)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&journal=Nature&volume=518&pages=317-330&publication_year=2015) 
    
118.  ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. _Nature_ **489**, 57–74 (2012).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&journal=Nature&volume=489&pages=57-74&publication_year=2012) 
    
119.  Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. _Nat. Commun._ **10**, 1776 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30992449)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467998)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20prediction%20via%20Bayesian%20regression%20and%20continuous%20shrinkage%20priors&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Ge%2CT&author=Chen%2CC-Y&author=Ni%2CY&author=Feng%2CY-CA&author=Smoller%2CJW) 
    
120.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
121.  Yavorska, O. O. & Burgess, S. MendelianRandomization: an R package for performing Mendelian randomization analyses using summarized data. _Int. J. Epidemiol._ **46**, 1734–1739 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28398548)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5510723)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MendelianRandomization%3A%20an%20R%20package%20for%20performing%20Mendelian%20randomization%20analyses%20using%20summarized%20data&journal=Int.%20J.%20Epidemiol.&volume=46&pages=1734-1739&publication_year=2017&author=Yavorska%2COO&author=Burgess%2CS) 
    
122.  Martin, J. et al. A genetic investigation of sex bias in the prevalence of attention-deficit/hyperactivity disorder. _Biol. Psychiatry_ **83**, 1044–1053 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29325848)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5992329)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genetic%20investigation%20of%20sex%20bias%20in%20the%20prevalence%20of%20attention-deficit%2Fhyperactivity%20disorder&journal=Biol.%20Psychiatry&volume=83&pages=1044-1053&publication_year=2018&author=Martin%2CJ) 
    
123.  Buniello, A. et al. The NHGRI-EBI GWAS catalog of published genome-wide association studies, targeted arrays and summary statistics 2019. _Nucleic Acids Res._ **47**, D1005–D1012 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1GqtrvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30445434)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20NHGRI-EBI%20GWAS%20catalog%20of%20published%20genome-wide%20association%20studies%2C%20targeted%20arrays%20and%20summary%20statistics%202019&journal=Nucleic%20Acids%20Res.&volume=47&pages=D1005-D1012&publication_year=2019&author=Buniello%2CA) 
    
124.  Gerring, Z. F., Gamazon, E. R. & Derks, E. M. A gene co-expression network-based analysis of multiple brain tissues reveals novel genes and molecular pathways underlying major depression. _PLoS Genet._ **15**, e1008245 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31306407)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6658115)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20gene%20co-expression%20network-based%20analysis%20of%20multiple%20brain%20tissues%20reveals%20novel%20genes%20and%20molecular%20pathways%20underlying%20major%20depression&journal=PLoS%20Genet.&volume=15&publication_year=2019&author=Gerring%2CZF&author=Gamazon%2CER&author=Derks%2CEM) 
    
125.  Pividori, M. et al. PhenomeXcan: mapping the genome to the phenome through the transcriptome. _Sci. Adv._ **6**, eaba2083 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1CmtLfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32917697)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11206444)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PhenomeXcan%3A%20mapping%20the%20genome%20to%20the%20phenome%20through%20the%20transcriptome&journal=Sci.%20Adv.&volume=6&publication_year=2020&author=Pividori%2CM) 
    
126.  Wen, X., Pique-Regi, R. & Luca, F. Integrating molecular QTL data into genome-wide genetic association analysis: probabilistic assessment of enrichment and colocalization. _PLoS Genet._ **13**, e1006646 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28278150)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5363995)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrating%20molecular%20QTL%20data%20into%20genome-wide%20genetic%20association%20analysis%3A%20probabilistic%20assessment%20of%20enrichment%20and%20colocalization&journal=PLoS%20Genet.&volume=13&publication_year=2017&author=Wen%2CX&author=Pique-Regi%2CR&author=Luca%2CF) 
    
127.  Berisa, T. & Pickrell, J. K. Approximately independent linkage disequilibrium blocks in human populations. _Bioinformatics_ **32**, 283–285 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26395773)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4731402)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Approximately%20independent%20linkage%20disequilibrium%20blocks%20in%20human%20populations&journal=Bioinformatics&volume=32&pages=283-285&publication_year=2015&author=Berisa%2CT&author=Pickrell%2CJK) 
    
128.  Hukku, A., Sampson, M. G., Luca, F., Pique-Regi, R. & Wen, X. Analyzing and reconciling colocalization and transcriptome-wide association studies from the perspective of inferential reproducibility. _Am. J. Hum. Genet._ **109**, 825–837 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1WgtrrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35523146)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9118134)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analyzing%20and%20reconciling%20colocalization%20and%20transcriptome-wide%20association%20studies%20from%20the%20perspective%20of%20inferential%20reproducibility&journal=Am.%20J.%20Hum.%20Genet.&volume=109&pages=825-837&publication_year=2022&author=Hukku%2CA&author=Sampson%2CMG&author=Luca%2CF&author=Pique-Regi%2CR&author=Wen%2CX) 
    
129.  Scartozzi, A. Concordance-analysis, version 1.0. _Zenodo_ [https://doi.org/10.5281/zenodo.14884575](https://doi.org/10.5281/zenodo.14884575) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02267-2?format=refman&flavour=references)

Acknowledgements
----------------

We thank participants from 23andMe, Add Health and the ISP. This research was supported by National Institutes of Health (NIH) grants from the National Institute on Deafness and Other Communication Disorders (NIDCD) to Vanderbilt University Medical Center and Wayne State University (1R03DC015329 and R01DC017175) and to Vanderbilt University Medical Center (5R21DC016723 and R01DC020311) supporting H.G.P., D.M.S., J.E.B., E.J.L. R.M.J. and S.K. D.G.P. received funding from the National Center for Advancing Translational Sciences of the NIH under award number TL1TR002244. A.C.S. received funding from the NIDCD under award number F31DC022482. The content is solely the responsibility of the authors and does not necessarily represent the official views of any funding agencies.

Author information
------------------

Author notes

1.  These authors contributed equally: Hannah G. Polikowsky, Alyssa C. Scartozzi, Douglas M. Shaw.
    

### Authors and Affiliations

1.  Vanderbilt Genetics Institute, Vanderbilt University Medical Center, Nashville, TN, USA
    
    Hannah G. Polikowsky, Alyssa C. Scartozzi, Douglas M. Shaw, Hung-Hsin Chen, Lauren E. Petty, Alexander S. Petty & Jennifer E. Below
    
2.  Hearing and Speech Sciences, Vanderbilt University, Nashville, TN, USA
    
    Dillon G. Pruett & Robin M. Jones
    
3.  Curtin School of Allied Health, Curtin University, Perth, Australia
    
    Emily J. Lowther, Janet M. Beilby & Kathryn Z. Viljoen
    
4.  Department of Epidemiology, University of Texas MD Anderson Cancer Center, Houston, TX, USA
    
    Shu-Hsien Cho, Yao Yu & Chad D. Huff
    
5.  Maze Therapeutics, San Francisco, CA, USA
    
    Sahar Mozaffari
    
6.  Department of Epidemiology, Gillings School of Global Public Health, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA
    
    Christy L. Avery & Heather M. Highland
    
7.  Carolina Population Center, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA
    
    Christy L. Avery & Kathleen Mullan Harris
    
8.  Department of Sociology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA
    
    Kathleen Mullan Harris
    
9.  Department of Otolaryngology, Vanderbilt University Medical Center, Nashville, TN, USA
    
    Reyna L. Gordon
    
10.  Department of Hearing and Speech, Vanderbilt University Medical Center, Nashville, TN, USA
    
    Robin M. Jones
    
11.  Communication Sciences and Disorders, Wayne State University, Detroit, MI, USA
    
    Shelly Jo Kraft
    
12.  23andMe, Inc, Sunnyvale, CA, USA
    
    Stella Aslibekyan, Adam Auton, Elizabeth Babalola, Robert K. Bell, Jessica Bielenberg, Katarzyna Bryc, Emily Bullis, Daniella Coker, Gabriel Cuellar Partida, Devika Dhamija, Sayantan Das, Sarah L. Elson, Teresa Filshtein, Kipper Fletez-Brant, Pierre Fontanillas, Will Freyman, Pooja M. Gandhi, Karl Heilbron, Barry Hicks, David A. Hinds, Ethan M. Jewett, Yunxuan Jiang, Katelyn Kukar, Keng-Han Lin, Maya Lowe, Jey McCreight, Matthew H. McIntyre, Steven J. Micheletti, Meghan E. Moreno, Joanna L. Mountain, Priyanka Nandakumar, Elizabeth S. Noblin, Jared O’Connell, Aaron A. Petrakovitz, G. David Poznik, Morgan Schumacher, Anjali J. Shastri, Janie F. Shelton, Jingchunzi Shi, Suyash Shringarpure, Vinh Tran, Joyce Y. Tung, Xin Wang, Wei Wang, Catherine H. Weldon, Peter Wilton, Alejandro Hernandez, Corinna Wong & Christophe Toukam Tchakouté
    

Authors

1.  Hannah G. Polikowsky
    
    [View author publications](https://www.nature.com/search?author=Hannah%20G.%20Polikowsky)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hannah%20G.%20Polikowsky) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hannah%20G.%20Polikowsky%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Alyssa C. Scartozzi
    
    [View author publications](https://www.nature.com/search?author=Alyssa%20C.%20Scartozzi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alyssa%20C.%20Scartozzi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alyssa%20C.%20Scartozzi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Douglas M. Shaw
    
    [View author publications](https://www.nature.com/search?author=Douglas%20M.%20Shaw)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Douglas%20M.%20Shaw) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Douglas%20M.%20Shaw%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Dillon G. Pruett
    
    [View author publications](https://www.nature.com/search?author=Dillon%20G.%20Pruett)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Dillon%20G.%20Pruett) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Dillon%20G.%20Pruett%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Hung-Hsin Chen
    
    [View author publications](https://www.nature.com/search?author=Hung-Hsin%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hung-Hsin%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hung-Hsin%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Lauren E. Petty
    
    [View author publications](https://www.nature.com/search?author=Lauren%20E.%20Petty)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lauren%20E.%20Petty) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lauren%20E.%20Petty%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Alexander S. Petty
    
    [View author publications](https://www.nature.com/search?author=Alexander%20S.%20Petty)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alexander%20S.%20Petty) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alexander%20S.%20Petty%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Emily J. Lowther
    
    [View author publications](https://www.nature.com/search?author=Emily%20J.%20Lowther)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Emily%20J.%20Lowther) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Emily%20J.%20Lowther%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Shu-Hsien Cho
    
    [View author publications](https://www.nature.com/search?author=Shu-Hsien%20Cho)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shu-Hsien%20Cho) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shu-Hsien%20Cho%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Yao Yu
    
    [View author publications](https://www.nature.com/search?author=Yao%20Yu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yao%20Yu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yao%20Yu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Sahar Mozaffari
    
    [View author publications](https://www.nature.com/search?author=Sahar%20Mozaffari)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sahar%20Mozaffari) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sahar%20Mozaffari%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Christy L. Avery
    
    [View author publications](https://www.nature.com/search?author=Christy%20L.%20Avery)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Christy%20L.%20Avery) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Christy%20L.%20Avery%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Kathleen Mullan Harris
    
    [View author publications](https://www.nature.com/search?author=Kathleen%20Mullan%20Harris)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kathleen%20Mullan%20Harris) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kathleen%20Mullan%20Harris%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Reyna L. Gordon
    
    [View author publications](https://www.nature.com/search?author=Reyna%20L.%20Gordon)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Reyna%20L.%20Gordon) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Reyna%20L.%20Gordon%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Janet M. Beilby
    
    [View author publications](https://www.nature.com/search?author=Janet%20M.%20Beilby)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Janet%20M.%20Beilby) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Janet%20M.%20Beilby%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Kathryn Z. Viljoen
    
    [View author publications](https://www.nature.com/search?author=Kathryn%20Z.%20Viljoen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kathryn%20Z.%20Viljoen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kathryn%20Z.%20Viljoen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Robin M. Jones
    
    [View author publications](https://www.nature.com/search?author=Robin%20M.%20Jones)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Robin%20M.%20Jones) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Robin%20M.%20Jones%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Chad D. Huff
    
    [View author publications](https://www.nature.com/search?author=Chad%20D.%20Huff)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chad%20D.%20Huff) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chad%20D.%20Huff%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Heather M. Highland
    
    [View author publications](https://www.nature.com/search?author=Heather%20M.%20Highland)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Heather%20M.%20Highland) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Heather%20M.%20Highland%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Shelly Jo Kraft
    
    [View author publications](https://www.nature.com/search?author=Shelly%20Jo%20Kraft)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shelly%20Jo%20Kraft) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shelly%20Jo%20Kraft%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Jennifer E. Below
    
    [View author publications](https://www.nature.com/search?author=Jennifer%20E.%20Below)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jennifer%20E.%20Below) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jennifer%20E.%20Below%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Consortia

### 23andMe Research Team

-   Stella Aslibekyan
-   , Adam Auton
-   , Elizabeth Babalola
-   , Robert K. Bell
-   , Jessica Bielenberg
-   , Katarzyna Bryc
-   , Emily Bullis
-   , Daniella Coker
-   , Gabriel Cuellar Partida
-   , Devika Dhamija
-   , Sayantan Das
-   , Sarah L. Elson
-   , Teresa Filshtein
-   , Kipper Fletez-Brant
-   , Pierre Fontanillas
-   , Will Freyman
-   , Pooja M. Gandhi
-   , Karl Heilbron
-   , Barry Hicks
-   , David A. Hinds
-   , Ethan M. Jewett
-   , Yunxuan Jiang
-   , Katelyn Kukar
-   , Keng-Han Lin
-   , Maya Lowe
-   , Jey McCreight
-   , Matthew H. McIntyre
-   , Steven J. Micheletti
-   , Meghan E. Moreno
-   , Joanna L. Mountain
-   , Priyanka Nandakumar
-   , Elizabeth S. Noblin
-   , Jared O’Connell
-   , Aaron A. Petrakovitz
-   , G. David Poznik
-   , Morgan Schumacher
-   , Anjali J. Shastri
-   , Janie F. Shelton
-   , Jingchunzi Shi
-   , Suyash Shringarpure
-   , Vinh Tran
-   , Joyce Y. Tung
-   , Xin Wang
-   , Wei Wang
-   , Catherine H. Weldon
-   , Peter Wilton
-   , Alejandro Hernandez
-   , Corinna Wong
-    & Christophe Toukam Tchakouté

### Contributions

J.E.B. oversaw the entire study. J.E.B. and S.J.K. conceived the study. H.G.P., A.C.S., D.M.S. and J.E.B. drafted the manuscript, with sections contributed by D.G.P. and H.-H.C. H.G.P., A.C.S., D.M.S., H.-C.C., L.E.P., S.-H.C. and H.M.H. performed analyses. Y.Y., C.D.H. and R.M.J. assisted in the interpretation of all analyses. A.S.P. provided computational support to all analyses. H.M.H., C.L.A. and K.M.H. managed the Add Health Study. S.J.K., J.M.B., E.J.L. and K.Z.V. managed the ISP cohort. R.L.G. contributed the beat synchronization GWAS summary statistics. S.M. and the 23andMe Research Team performed all ancestry-specific and sex-specific GWAS analyses. All authors critically reviewed the manuscript.

### Corresponding author

Correspondence to [Jennifer E. Below](mailto:jennifer.e.below@vanderbilt.edu).

Ethics declarations
-------------------

### Competing interests

S.M. and members of the 23andMe Research Team were or are employed by and hold stock or stock options in 23andMe, Inc. D.M.S. is currently employed by AstraZeneca. The remaining authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Beate St Pourcain and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Study design.](https://www.nature.com/articles/s41588-025-02267-2/figures/5)

**a**, Overview of primary and secondary GWAS analyses. **b**, Flow chart of study design and downstream analyses. Primary analyses are indicated in yellow. Secondary analyses are indicated in blue. Green boxes represent analyses that consist of using data from both the primary and secondary analyses. ‘\*’ indicate analyses performed in European ancestry only.

### [Extended Data Fig. 2 Manhattan and Q-Q plots for primary genome-wide association analyses.](https://www.nature.com/articles/s41588-025-02267-2/figures/6)

Manhattan plots for genome-wide association analyses performed via logistic regression (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). The dotted line represents genome-wide significance (_P_ < 5 × 10−8). Q-Q plot _x_\-axes represent expected −log10(_P_\-value), and the _y_\-axes represent observed −log10(_P_\-value) with 95% CI indicated by shaded region. **a**, The AFR female association study included 24,783 total samples (3,008 self-reported stuttering cases) and 29,555,214 autosomal variants. No loci reached genome-wide significance (dotted line _P_ < 5 × 10−8). λgc = 1.0165. **b**, The AFR male association study included 13,977 total samples (2,480 self-reported stuttering cases) and 29,443,778 autosomal variants. Three loci reached genome-wide significance (dotted line _P_ < 5 × 10−8). λgc = 1.0156. **c**, The AMR female association study included 74,187 total samples (7,209 self-reported stuttering cases) and 29,814,878 autosomal variants. One locus reached genome-wide significance (dotted line _P_ < 5 × 10−8). λgc = 1.0340. **d**, The AMR male association study included 45,206 total samples (6,346 self-reported stuttering cases) and 29,759,321 autosomal variants. One locus reached genome-wide significance (dotted line _P_ < 5 × 10−8). λgc = 1.0177. **e**, The EAS female association study included 13,347 total samples (1,205 self-reported stuttering cases) and 10,595,969 autosomal variants. No loci reached genome-wide significance (dotted line _P_ < 5 × 10−8). λgc = 1.0081. **f**, The EAS male association study included 7,169 total samples (1,134 self-reported stuttering cases) and 10,586,108 autosomal variants. No loci reached genome-wide significance (dotted line _P_ < 5 × 10−8). λgc = 1.0182. **g**, Q-Q plot for the EUR female GWAS; λgc = 1.0855. **h**, Q-Q plot for the EUR male GWAS; λgc = 1.0834.

### [Extended Data Fig. 3 Forest plots showing significant differences in effect size heterogeneity.](https://www.nature.com/articles/s41588-025-02267-2/figures/7)

Wald tests were performed to test for heterogeneity of effect estimates for the 24 sentinel variants that reached genome-wide significance in the eight ancestry- and sex-specific GWAS compared to the other seven primary GWAS. Plot displays the eight sentinel variants that demonstrated significant differences in effect size by ancestry and/or sex. Data points represent effect size +/− standard error. Each genetic ancestry is color coded. Female-specific effect estimates are shown using circles. Male-specific effect estimates are shown using triangles. Effect estimates that are not shown indicate that the variant was not found. The symbol ‘+’ represents the effect of the result in the Discovery GWAS. The symbol ‘\*’ denotes significant differences in effect size estimates compared to the Discovery GWAS after Bonferroni correction. Full GWAS samples sizes can be found in Table [1](https://www.nature.com/articles/s41588-025-02267-2#Tab1).

### [Extended Data Fig. 4 Manhattan and Q-Q plots for the sex-combined meta-analyses.](https://www.nature.com/articles/s41588-025-02267-2/figures/8)

Manhattan plots for sex-combined meta-analyses performed using the inverse variance weighted option in METAL (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). The dotted line represents genome-wide significance (_P_ < 5 × 10−8). Q-Q plot x-axes represent expected −log10(_P_\-value), and y-axes represent observed −log10(_P_\-value) with 95% CI indicated by shaded region. **a**, The meta-analysis includes 944,350 total samples (78,394 self-reported stuttering cases) and 29,566,899 autosomal variants observed across EUR male and EUR female GWAS; λgc = 1.0586. **b**, The meta-analysis includes 119,393 total samples (13,555 self-reported stuttering cases) and 29,948,846 autosomal variants observed across AMR male and AMR female GWAS; λgc = 1.0070. **c**, The meta-analysis includes 20,516 total samples (2,339 self-reported stuttering cases) and 10,656,266 autosomal variants observed across EAS male and EAS female GWAS; λgc = 1.0023. **d**, The meta-analysis includes 38,760 total samples (5,488 self-reported stuttering cases) and 29,705,842 autosomal variants observed across AFR male and AFR female GWAS; λgc = 1.0019.

### [Extended Data Fig. 5 Miami plot for female-specific meta-analysis and male-specific meta-analysis.](https://www.nature.com/articles/s41588-025-02267-2/figures/9)

Ancestry-combined, sex-specific meta-analyses were performed using multi-ancestry meta-regression via MR-MEGA (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). **a**, The multi-ancestry female meta-analysis (top panel in orange) included 682,388 total samples (51,559 self-reported stuttering cases) and 41,960,804 autosomal variants (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). Five loci reached genome-wide significance (dotted line _P_ < 5 × 10−8). The multi-ancestry male meta-analysis (bottom panel in blue) included 440,631 total samples (48,217 self-reported stuttering cases) and 41,937,718 autosomal variants (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)). Three loci reached genome-wide significance (dotted line _P_ < 5 × 10−8). The x-axis represents chromosome base pair coordinates in human genome build 37, and the y-axis represents observed −log10(_P_\-value) for each GWAS. Annotated genes for each GWAS includes the predicted functional gene for each loci (when available) according to Open Targets V2G pipeline. **b**, Q-Q plot for the female-specific meta-analysis. **c**, Q-Q plot for the male-specific meta-analysis. Q-Q plot _x_\-axes represent expected −log10(_P_\-value), and _y_\-axes represent observed −log10(_P_\-value), with 95% CI indicated by shaded region. Note: _P_\-values were adjusted for genomic control.

### [Extended Data Fig. 6 Manhattan and Q-Q plot for the ancestry- and sex-combined meta-analysis.](https://www.nature.com/articles/s41588-025-02267-2/figures/10)

The multi-ancestry meta-regression meta-analysis performed via MR-MEGA (see [Methods](https://www.nature.com/articles/s41588-025-02267-2#Sec15)) included 1,023,243 total samples (99,776 self-reported stuttering cases) and 42,031,853 autosomal variants observed across all eight independent sex and ancestries-specific GWAS. The dotted line represents genome-wide significance (_P_ < 5 × 10−8). Q-Q plot _x_\-axis represents expected −log10(_P_\-value), and _y_\-axis represents observed −log10(_P_\-value) with 95% CI indicated by shaded region. Note: _P_\-values were adjusted for genomic control.

### [Extended Data Fig. 7 Partitioned heritability estimates for broad functional categories.](https://www.nature.com/articles/s41588-025-02267-2/figures/11)

Partitioned heritability was performed using the 52 broad functional annotations defined by Finucane et al. Enrichment is the proportion of _h_2/proportion of SNPs. The horizontal, dotted line indicates no enrichment. Error bars indicate enrichment estimates +/− standard error. Asterisks indicate enrichment estimates are significant based on a Bonferroni _P_\-value of 9.6 × 10−4 (52 gene-sets). **a**, Broad functional categories partitioned heritability estimates in EUR female self-reported stuttering. **b**, Broad functional categories partitioned heritability estimates in EUR male self-reported stuttering. **c**, Broad functional categories partitioned heritability estimates in sex-combined EUR self-reported stuttering. Full results are presented in Supplementary Tables [8](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[10](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

### [Extended Data Fig. 8 Partitioned heritability estimates for brain cell types.](https://www.nature.com/articles/s41588-025-02267-2/figures/12)

Partitioned heritability was performed using the three cell-type annotations defined by Finucane et al. The −log10(_P_\-value) for enrichment estimates is indicated on the _y_\-axis. The horizontal dotted line indicates significant after Bonferroni correction (_P_ < 1.7 × 10−2, three tests). **a**, Brain cell type partitioned heritability estimates in EUR female self-reported stuttering. **b**, Brain cell type partitioned heritability estimates in EUR male self-reported stuttering. **c**, Brain cell type partitioned heritability estimates in sex-combined EUR self-reported stuttering. Full results are presented in Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[13](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

### [Extended Data Fig. 9 Partitioned heritability estimates for tissue-specific gene expression.](https://www.nature.com/articles/s41588-025-02267-2/figures/13)

Partitioned heritability was performed using the tissue-specific expression annotations defined by Finucane et al. The −log10(_P_\-value) for enrichment estimates is indicated on the _y_\-axis. The horizontal dotted line indicates significant after Bonferroni correction (_P_ < 6.25 × 10−3, eight tests). **a**, Tissue-specific gene expression partitioned heritability estimates in EUR female self-reported stuttering. **b**, Tissue-specific gene expression partitioned heritability estimates in EUR male self-reported stuttering. **c**, Tissue-specific gene expression partitioned heritability estimates in sex-combined EUR self-reported stuttering. Full results are presented in Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[13](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

### [Extended Data Fig. 10 Partitioned heritability estimates for tissue-specific chromatin marks.](https://www.nature.com/articles/s41588-025-02267-2/figures/14)

Partitioned heritability was performed using the tissue-specific chromatin annotations defined by Finucane et al. The −log10(_P_\-value) for enrichment estimates is indicated on the _y_\-axis. The horizontal dotted line indicates significant after Bonferroni correction (_P_ < 2.5 × 10−3, 20 tests). **a**, Tissue-specific chromatin partitioned heritability estimates in EUR female self-reported stuttering. **b**, Tissue-specific chromatin partitioned heritability estimates in EUR male self-reported stuttering. **c**, Chromatin partitioned heritability estimates in sex-combined EUR self-reported stuttering. Full results are presented in Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02267-2#MOESM3)–[13](https://www.nature.com/articles/s41588-025-02267-2#MOESM3).

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_MOESM1_ESM.pdf)

Supplementary Note and Supplementary Figs. 1–90.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_MOESM2_ESM.pdf)

### [Supplementary Table 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_MOESM3_ESM.xlsx)

Supplementary Tables 1–25.

### [Supplementary Data 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02267-2/MediaObjects/41588_2025_2267_MOESM4_ESM.xlsx)

Supplementary Data 1–8.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Large-scale%20genome-wide%20analyses%20of%20stuttering&author=Hannah%20G.%20Polikowsky%20Stella%20Aslibekyan%20et%20al&contentID=10.1038%2Fs41588-025-02267-2&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-28&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02267-2)

### Cite this article

Polikowsky, H.G., Scartozzi, A.C., Shaw, D.M. _et al._ Large-scale genome-wide analyses of stuttering. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02267-2

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02267-2?format=refman&flavour=citation)

-   Received: 10 April 2023
    
-   Accepted: 16 June 2025
    
-   Published: 28 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02267-2
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02267-2.pdf)
