> id: https://www.nature.com/articles/s41588-025-02284-1

> link: https://www.nature.com/articles/s41588-025-02284-1

> title: Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states

# Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
_Published on Wed, 06 Aug 2025 00:00:00 GMT_

Abstract
--------

Cell atlas projects have revealed that common cell types often comprise distinct, recurrent transcriptional states, but the function and regulation of these states remain poorly understood. Here, we show that systematic activation of transcription factors can recreate such states in vitro, providing tractable models for mechanistic and functional studies. Using a scalable CRISPR activation (CRISPRa) Perturb-seq platform, we activated 1,836 transcription factors in two cell types. CRISPRa induced gene expression within physiological ranges, with chromatin features predicting responsiveness. Comparisons with atlas datasets showed that transcription factor perturbations recapitulated key fibroblast states and identified their regulators, including _KLF2_ and _KLF4_ for a universal state present in many tissues, and _PLAGL1_ for a disease-associated inflammatory state. Inducing the universal state suppressed the inflammatory state, suggesting therapeutic potential. These findings position CRISPRa as a nuanced tool for perturbing differentiated cells and establish a general strategy for studying clinically relevant transcriptional states ex vivo.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

27,99 € / 30 days

cancel any time

[Learn more](https://shop.nature.com/products/plus/?region=ROW)

Subscribe to this journal

Receive 12 print issues and online access

195,33 € per year

only 16,28 € per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02284-1?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Droplet overloading for high-throughput Perturb-seq.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_Fig1_HTML.png)

**Fig. 2: Efficacy of CRISPRa against TFs measured using Perturb-seq.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_Fig2_HTML.png)

**Fig. 3: Off-target effects driven by the seed region of the protospacer.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_Fig3_HTML.png)

**Fig. 4: Cell context shapes TF activity and function.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_Fig4_HTML.png)

**Fig. 5: Linking in vitro gene programs to in vivo transcriptional states.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_Fig5_HTML.png)

**Fig. 6: Gene programs and regulators of major fibroblast states.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_Fig6_HTML.png)

**Fig. 7: In vivo drivers of the universal and inflammatory states and cross-regulation between them.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_Fig7_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41593-022-01180-9/MediaObjects/41593_2022_1180_Fig1_HTML.png)

### [CRISPRi screens in human iPSC-derived astrocytes elucidate regulators of distinct inflammatory reactive states](https://www.nature.com/articles/s41593-022-01180-9?fromPaywallRec=true)

Article 27 October 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-52490-4/MediaObjects/41467_2024_52490_Fig1_HTML.png)

### [Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements](https://www.nature.com/articles/s41467-024-52490-4?fromPaywallRec=true)

Article Open access 18 September 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-025-09686-5/MediaObjects/41598_2025_9686_Fig1_HTML.png)

### [Induction of cancer stem cells from mouse fibroblasts](https://www.nature.com/articles/s41598-025-09686-5?fromPaywallRec=true)

Article Open access 15 July 2025

Data availability
-----------------

Raw sequencing data and metadata are available on the Sequence Read Archive under accession number PRJNA1108254. Processed single-cell data, including guide phenotypes, are available on Zenodo ([https://doi.org/10.5281/zenodo.15213597](https://doi.org/10.5281/zenodo.15213597) (ref. [78](https://www.nature.com/articles/s41588-025-02284-1#ref-CR78 "Southard, K. et al. Hs27-CRISPRa-TFs cellranger outputs. Zenodo 
https://doi.org/10.5281/ZENODO.15213597
(2025).")), [https://doi.org/10.5281/zenodo.15200179](https://doi.org/10.5281/zenodo.15200179) (ref. [79](https://www.nature.com/articles/s41588-025-02284-1#ref-CR79 "Southard, K. et al. Hs27-CRISPRa-TFs. Zenodo 
https://doi.org/10.5281/ZENODO.15200179
(2025).")), [https://doi.org/10.5281/zenodo.15211972](https://doi.org/10.5281/zenodo.15211972) (ref. [80](https://www.nature.com/articles/s41588-025-02284-1#ref-CR80 "Southard, K. et al. RPE1-CRISPRa-TFs cellranger outputs. Zenodo 
https://doi.org/10.5281/ZENODO.15211972
(2025).")), [https://doi.org/10.5281/zenodo.15213619](https://doi.org/10.5281/zenodo.15213619) (ref. [81](https://www.nature.com/articles/s41588-025-02284-1#ref-CR81 "Southard, K. et al. RPE1-CRISPRa-TFs. Zenodo 
https://doi.org/10.5281/ZENODO.15213619
(2025).")), [https://doi.org/10.5281/zenodo.15215389](https://doi.org/10.5281/zenodo.15215389) (ref. [82](https://www.nature.com/articles/s41588-025-02284-1#ref-CR82 "Southard, K. et al. RPE1-E150-Benchmarking cellranger outputs. Zenodo 
https://doi.org/10.5281/ZENODO.15215389
(2025).")), [https://doi.org/10.5281/zenodo.15215414](https://doi.org/10.5281/zenodo.15215414) (ref. [83](https://www.nature.com/articles/s41588-025-02284-1#ref-CR83 "Southard, K. et al. RPE1-E150-Benchmarking. Zenodo 
https://doi.org/10.5281/ZENODO.15215414
(2025).")), [https://doi.org/10.5281/zenodo.15215154](https://doi.org/10.5281/zenodo.15215154) (ref. [84](https://www.nature.com/articles/s41588-025-02284-1#ref-CR84 "Southard, K. et al. K562 dCas9-CUT&RUN. Zenodo 
https://doi.org/10.5281/ZENODO.15215154
(2025).")) and [https://doi.org/10.5281/zenodo.15215216](https://doi.org/10.5281/zenodo.15215216) (ref. [85](https://www.nature.com/articles/s41588-025-02284-1#ref-CR85 "Southard, K. et al. RPE-1 and Hs27 CUT&RUN, ATAC-seq, and RNA-seq characterization. Zenodo 
https://doi.org/10.5281/ZENODO.15215216
(2025)."))). External datasets include the Human Protein Atlas[2](https://www.nature.com/articles/s41588-025-02284-1#ref-CR2 "Karlsson, M. et al. A single-cell type transcriptomics map of human tissues. Sci. Adv. 7, eabh2169 (2021).") single-cell data ([https://www.proteinatlas.org/download/tsv/rna\_single\_cell\_cluster.tsv.zip](https://www.proteinatlas.org/download/tsv/rna_single_cell_cluster.tsv.zip)), KRABopedia[56](https://www.nature.com/articles/s41588-025-02284-1#ref-CR56 "Tribolet-Hardy, J. et al. Genetic features and genomic targets of human KRAB-zinc finger proteins. Genome Res. 33, 1409–1423 (2023).") ([https://tronoapps.epfl.ch/web/krabopedia/ind.php](https://tronoapps.epfl.ch/web/krabopedia/ind.php)) and DNA methylation data from a previous publication[54](https://www.nature.com/articles/s41588-025-02284-1#ref-CR54 "Loyfer, N. et al. A DNA methylation atlas of normal human cell types. Nature 613, 355–364 (2023).") ([GSE186458](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE186458)). Fibroblast atlas datasets were previously published, including [https://singlecell.broadinstitute.org/single\_cell/study/SCP738/cross-tissue-single-cell-stromal-atlas-identifies-shared-pathological-fibroblast-phenotypes-in-four-chronic-inflammatory-diseases](https://singlecell.broadinstitute.org/single_cell/study/SCP738/cross-tissue-single-cell-stromal-atlas-identifies-shared-pathological-fibroblast-phenotypes-in-four-chronic-inflammatory-diseases) (ref. [40](https://www.nature.com/articles/s41588-025-02284-1#ref-CR40 "Korsunsky, I. et al. Cross-tissue, single-cell stromal atlas identifies shared pathological fibroblast phenotypes in four chronic inflammatory diseases. Med 3, 481–518.e14 (2022).")) (registration required for download), [GSE246219](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246219) ([http://pan-fib.cancer-pku.cn](http://pan-fib.cancer-pku.cn/) and [https://doi.org/10.5281/zenodo.13950609](https://doi.org/10.5281/zenodo.13950609))[41](https://www.nature.com/articles/s41588-025-02284-1#ref-CR41 "Gao, Y. et al. Cross-tissue human fibroblast atlas reveals myofibroblast subtypes with distinct roles in immune modulation. Cancer Cell 42, 1764–1783.e10 (2024)."), [GSE218392](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218392) (ref. [35](https://www.nature.com/articles/s41588-025-02284-1#ref-CR35 "Amrute, J. M. et al. Targeting immune–fibroblast cell communication in heart failure. Nature 635, 423–433 (2024).")) and [https://www.fibroxplorer.com/download](https://www.fibroxplorer.com/download)[39](https://www.nature.com/articles/s41588-025-02284-1#ref-CR39 "Buechler, M. B. et al. Cross-tissue organization of the fibroblast lineage. Nature 593, 575–579 (2021)."). Additional datasets used for the analysis of seed-based off-target effects included [https://doi.org/10.1016/j.cell.2014.09.029](https://doi.org/10.1016/j.cell.2014.09.029) (ref. [11](https://www.nature.com/articles/s41588-025-02284-1#ref-CR11 "Gilbert, L. A. et al. Genome-scale CRISPR-mediated control of gene repression and activation. Cell 159, 647–661 (2014).")) (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02284-1#MOESM4)), [https://doi.org/10.3389/fmolb.2023.1110445](https://doi.org/10.3389/fmolb.2023.1110445) (ref. [59](https://www.nature.com/articles/s41588-025-02284-1#ref-CR59 "Uthaya et al. A genome-wide CRISPR activation screen identifies SCREEM a novel SNAI1 super-enhancer demarcated by eRNAs. Front. Mol. Biosci. 10, 1110445 (2023).")) (Supplementary Table [4](https://www.nature.com/articles/s41588-025-02284-1#MOESM4)), CD44 LOW mageck\_sgrna\_summary, [https://doi.org/10.1038/s41467-018-07901-8](https://doi.org/10.1038/s41467-018-07901-8) (ref. [17](https://www.nature.com/articles/s41588-025-02284-1#ref-CR17 "Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. Nat. Commun. 9, 5416 (2018).")) (Supplementary Data [5](https://www.nature.com/articles/s41588-025-02284-1#MOESM1)), pXPR\_118 A375 Vem SetA analysis, pXPR\_109 Mel Sel SetAB analysis, [https://doi.org/10.1016/j.cell.2021.03.025](https://doi.org/10.1016/j.cell.2021.03.025) (ref. [60](https://www.nature.com/articles/s41588-025-02284-1#ref-CR60 "Nuñez, J. K. et al. Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing. Cell 184, 2503–2519.e17 (2021).")) (Supplementary Table [4](https://www.nature.com/articles/s41588-025-02284-1#MOESM4)); CRISPRoff. Microscopy data are provided as source data files. [Source data](https://www.nature.com/articles/s41588-025-02284-1#Sec37) are provided with this paper.

Code availability
-----------------

All custom analysis code generated in this study is available at [https://github.com/norman-lab-msk/TFs\_CRISPRa](https://github.com/norman-lab-msk/TFs_CRISPRa) ([https://doi.org/10.5281/zenodo.15373940](https://doi.org/10.5281/zenodo.15373940))[86](https://www.nature.com/articles/s41588-025-02284-1#ref-CR86 "K. Southard & Norman, T. norman-lab-msk/TFs_CRISPRa: CRISPRa TFs Perturb-seq v0.1. Zenodo 
https://doi.org/10.5281/ZENODO.15373940
(2025).").

References
----------

1.  Regev, A. et al. The human cell atlas. _eLife_ **6**, e27041 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29206104)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5762154)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20human%20cell%20atlas&journal=eLife&volume=6&publication_year=2017&author=Regev%2CA) 
    
2.  Karlsson, M. et al. A single-cell type transcriptomics map of human tissues. _Sci. Adv._ **7**, eabh2169 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitVKkur%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34321199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318366)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single-cell%20type%20transcriptomics%20map%20of%20human%20tissues&journal=Sci.%20Adv.&volume=7&publication_year=2021&author=Karlsson%2CM) 
    
3.  Consortium, T. T. S. et al. The tabula sapiens: a multiple-organ, single-cell transcriptomic atlas of humans. _Science_ **376**, eabl4896 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20tabula%20sapiens%3A%20a%20multiple-organ%2C%20single-cell%20transcriptomic%20atlas%20of%20humans&journal=Science&volume=376&publication_year=2022&author=Consortium%2CTTS) 
    
4.  Jain, S. et al. Advances and prospects for the Human BioMolecular Atlas Program (HuBMAP). _Nat. Cell Biol._ **25**, 1089–1100 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFSmtL7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37468756)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10681365)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Advances%20and%20prospects%20for%20the%20Human%20BioMolecular%20Atlas%20Program%20%28HuBMAP%29&journal=Nat.%20Cell%20Biol.&volume=25&pages=1089-1100&publication_year=2023&author=Jain%2CS) 
    
5.  Frenkel, M. & Raman, S. Discovering mechanisms of human genetic variation and controlling cell states at scale. _Trends Genet._ **40**, 587–600 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXovF2nu7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38658256)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11607914)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovering%20mechanisms%20of%20human%20genetic%20variation%20and%20controlling%20cell%20states%20at%20scale&journal=Trends%20Genet.&volume=40&pages=587-600&publication_year=2024&author=Frenkel%2CM&author=Raman%2CS) 
    
6.  Rafelski, S. M. & Theriot, J. A. Establishing a conceptual framework for holistic cell states and state transitions. _Cell_ **187**, 2633–2651 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtFGjs7jL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38788687)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Establishing%20a%20conceptual%20framework%20for%20holistic%20cell%20states%20and%20state%20transitions&journal=Cell&volume=187&pages=2633-2651&publication_year=2024&author=Rafelski%2CSM&author=Theriot%2CJA) 
    
7.  Zhang, M. et al. Molecularly defined and spatially resolved cell atlas of the whole mouse brain. _Nature_ **624**, 343–354 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXis1SksLfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38092912)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10719103)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecularly%20defined%20and%20spatially%20resolved%20cell%20atlas%20of%20the%20whole%20mouse%20brain&journal=Nature&volume=624&pages=343-354&publication_year=2023&author=Zhang%2CM) 
    
8.  Patel, A. S. & Yanai, I. A developmental constraint model of cancer cell states and tumor heterogeneity. _Cell_ **187**, 2907–2918 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1Grt7bM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38848676)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11256907)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20developmental%20constraint%20model%20of%20cancer%20cell%20states%20and%20tumor%20heterogeneity&journal=Cell&volume=187&pages=2907-2918&publication_year=2024&author=Patel%2CAS&author=Yanai%2CI) 
    
9.  Rood, J. E., Maartens, A., Hupalowska, A., Teichmann, S. A. & Regev, A. Impact of the Human Cell Atlas on medicine. _Nat. Med._ **28**, 2486–2496 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVyltbbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36482102)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Impact%20of%20the%20Human%20Cell%20Atlas%20on%20medicine&journal=Nat.%20Med.&volume=28&pages=2486-2496&publication_year=2022&author=Rood%2CJE&author=Maartens%2CA&author=Hupalowska%2CA&author=Teichmann%2CSA&author=Regev%2CA) 
    
10.  Joung, J. et al. A transcription factor atlas of directed differentiation. _Cell_ **186**, 209–229.e26 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXltVGjtA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36608654)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10344468)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20transcription%20factor%20atlas%20of%20directed%20differentiation&journal=Cell&volume=186&pages=209-229.e26&publication_year=2023&author=Joung%2CJ) 
    
11.  Gilbert, L. A. et al. Genome-scale CRISPR-mediated control of gene repression and activation. _Cell_ **159**, 647–661 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslelsrfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25307932)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4253859)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-scale%20CRISPR-mediated%20control%20of%20gene%20repression%20and%20activation&journal=Cell&volume=159&pages=647-661&publication_year=2014&author=Gilbert%2CLA) 
    
12.  Konermann, S. et al. Genome-scale transcriptional activation by an engineered CRISPR–Cas9 complex. _Nature_ **517**, 583–588 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVWhtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25494202)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-scale%20transcriptional%20activation%20by%20an%20engineered%20CRISPR%E2%80%93Cas9%20complex&journal=Nature&volume=517&pages=583-588&publication_year=2015&author=Konermann%2CS) 
    
13.  Liu, Y. et al. CRISPR activation screens systematically identify factors that drive neuronal fate and reprogramming. _Cell Stem Cell_ **23**, 758–771.e8 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFSqsrbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30318302)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6214761)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CRISPR%20activation%20screens%20systematically%20identify%20factors%20that%20drive%20neuronal%20fate%20and%20reprogramming&journal=Cell%20Stem%20Cell&volume=23&pages=758-771.e8&publication_year=2018&author=Liu%2CY) 
    
14.  Norman, T. M. et al. Exploring genetic interaction manifolds constructed from rich single-cell phenotypes. _Science_ **365**, 786–793 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1CjtbzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31395745)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6746554)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Exploring%20genetic%20interaction%20manifolds%20constructed%20from%20rich%20single-cell%20phenotypes&journal=Science&volume=365&pages=786-793&publication_year=2019&author=Norman%2CTM) 
    
15.  Tian, R. et al. Genome-wide CRISPRi/a screens in human neurons link lysosomal failure to ferroptosis. _Nat. Neurosci._ **24**, 1020–1034 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFGltbbK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34031600)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8254803)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20CRISPRi%2Fa%20screens%20in%20human%20neurons%20link%20lysosomal%20failure%20to%20ferroptosis&journal=Nat.%20Neurosci.&volume=24&pages=1020-1034&publication_year=2021&author=Tian%2CR) 
    
16.  Yang, J. et al. Genome-scale CRISPRa screen identifies novel factors for cellular reprogramming. _Stem Cell Rep._ **12**, 757–771 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtV2isbY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-scale%20CRISPRa%20screen%20identifies%20novel%20factors%20for%20cellular%20reprogramming&journal=Stem%20Cell%20Rep.&volume=12&pages=757-771&publication_year=2019&author=Yang%2CJ) 
    
17.  Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. _Nat. Commun._ **9**, 5416 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisFyntLjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30575746)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6303322)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimized%20libraries%20for%20CRISPR%E2%80%93Cas9%20genetic%20screens%20with%20multiple%20modalities&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Sanson%2CKR) 
    
18.  Papalexi, E. et al. Characterizing the molecular regulation of inhibitory immune checkpoints with multimodal single-cell screens. _Nat. Genet._ **53**, 322–331 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlsV2lt7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33649593)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8011839)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterizing%20the%20molecular%20regulation%20of%20inhibitory%20immune%20checkpoints%20with%20multimodal%20single-cell%20screens&journal=Nat.%20Genet.&volume=53&pages=322-331&publication_year=2021&author=Papalexi%2CE) 
    
19.  Griffith, A. L. et al. Optimization of Cas12a for multiplexed genome-scale transcriptional activation. _Cell Genomics_ **3**, 100387 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVWrurfO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37719144)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10504673)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimization%20of%20Cas12a%20for%20multiplexed%20genome-scale%20transcriptional%20activation&journal=Cell%20Genomics&volume=3&publication_year=2023&author=Griffith%2CAL) 
    
20.  Adamson, B. et al. A multiplexed single-cell CRISPR screening platform enables systematic dissection of the unfolded protein response. _Cell_ **167**, 1867–1882.e21 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitFWlsrjK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27984733)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5315571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20multiplexed%20single-cell%20CRISPR%20screening%20platform%20enables%20systematic%20dissection%20of%20the%20unfolded%20protein%20response&journal=Cell&volume=167&pages=1867-1882.e21&publication_year=2016&author=Adamson%2CB) 
    
21.  Dixit, A. et al. Perturb-seq: dissecting molecular circuits with scalable single-cell RNA profiling of pooled genetic screens. _Cell_ **167**, 1853–1866.e17 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitFWlsrvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27984732)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5181115)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Perturb-seq%3A%20dissecting%20molecular%20circuits%20with%20scalable%20single-cell%20RNA%20profiling%20of%20pooled%20genetic%20screens&journal=Cell&volume=167&pages=1853-1866.e17&publication_year=2016&author=Dixit%2CA) 
    
22.  Bock, C. et al. High-content CRISPR screening. _Nat. Rev. Methods Prim._ **2**, 8 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVCqur8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-content%20CRISPR%20screening&journal=Nat.%20Rev.%20Methods%20Prim.&volume=2&publication_year=2022&author=Bock%2CC) 
    
23.  Pacalin, N. M. et al. Bidirectional epigenetic editing reveals hierarchies in gene regulation. _Nat. Biotechnol._ **43**, 355–368 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtV2ku7%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38760566)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bidirectional%20epigenetic%20editing%20reveals%20hierarchies%20in%20gene%20regulation&journal=Nat.%20Biotechnol.&volume=43&pages=355-368&publication_year=2025&author=Pacalin%2CNM) 
    
24.  Schmidt, R. et al. CRISPR activation and interference screens decode stimulation responses in primary human T cells. _Science_ **375**, eabj4008 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xjtlyqtbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35113687)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9307090)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CRISPR%20activation%20and%20interference%20screens%20decode%20stimulation%20responses%20in%20primary%20human%20T%20cells&journal=Science&volume=375&publication_year=2022&author=Schmidt%2CR) 
    
25.  Chardon, F. M. et al. Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements. _Nat. Commun._ **15**, 8209 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitVeksb3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39294132)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11411074)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiplex%2C%20single-cell%20CRISPRa%20screening%20for%20cell%20type%20specific%20regulatory%20elements&journal=Nat.%20Commun.&volume=15&publication_year=2024&author=Chardon%2CFM) 
    
26.  Replogle, J. M. et al. Mapping information-rich genotype–phenotype landscapes with genome-scale Perturb-seq. _Cell_ **185**, 2559–2575.e28 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFWgsLrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35688146)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9380471)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20information-rich%20genotype%E2%80%93phenotype%20landscapes%20with%20genome-scale%20Perturb-seq&journal=Cell&volume=185&pages=2559-2575.e28&publication_year=2022&author=Replogle%2CJM) 
    
27.  Smith, M. H. et al. Drivers of heterogeneity in synovial fibroblasts in rheumatoid arthritis. _Nat. Immunol._ **24**, 1200–1210 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtFGmsL3N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37277655)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10307631)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Drivers%20of%20heterogeneity%20in%20synovial%20fibroblasts%20in%20rheumatoid%20arthritis&journal=Nat.%20Immunol.&volume=24&pages=1200-1210&publication_year=2023&author=Smith%2CMH) 
    
28.  Wei, K. et al. Notch signalling drives synovial fibroblast identity and arthritis pathology. _Nature_ **582**, 259–264 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVCiur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32499639)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7841716)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Notch%20signalling%20drives%20synovial%20fibroblast%20identity%20and%20arthritis%20pathology&journal=Nature&volume=582&pages=259-264&publication_year=2020&author=Wei%2CK) 
    
29.  Mizoguchi, F. et al. Functionally distinct disease-associated fibroblast subsets in rheumatoid arthritis. _Nat. Commun._ **9**, 789 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29476097)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5824882)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functionally%20distinct%20disease-associated%20fibroblast%20subsets%20in%20rheumatoid%20arthritis&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Mizoguchi%2CF) 
    
30.  Smillie, C. S. et al. Intra- and Inter-cellular rewiring of the human colon during ulcerative colitis. _Cell_ **178**, 714–730.e22 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOgsL7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31348891)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6662628)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intra-%20and%20Inter-cellular%20rewiring%20of%20the%20human%20colon%20during%20ulcerative%20colitis&journal=Cell&volume=178&pages=714-730.e22&publication_year=2019&author=Smillie%2CCS) 
    
31.  Kinchen, J. et al. Structural remodeling of the human colonic mesenchyme in inflammatory bowel disease. _Cell_ **175**, 372–386.e17 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVahsLfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30270042)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6176871)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20remodeling%20of%20the%20human%20colonic%20mesenchyme%20in%20inflammatory%20bowel%20disease&journal=Cell&volume=175&pages=372-386.e17&publication_year=2018&author=Kinchen%2CJ) 
    
32.  Cadinu, P. et al. Charting the cellular biogeography in colitis reveals fibroblast trajectories and coordinated spatial remodeling. _Cell_ **187**, 2010–2028.e30 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXns1Ohsbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38569542)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11017707)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Charting%20the%20cellular%20biogeography%20in%20colitis%20reveals%20fibroblast%20trajectories%20and%20coordinated%20spatial%20remodeling&journal=Cell&volume=187&pages=2010-2028.e30&publication_year=2024&author=Cadinu%2CP) 
    
33.  Tsukui, T., Wolters, P. J. & Sheppard, D. Alveolar fibroblast lineage orchestrates lung inflammation and fibrosis. _Nature_ **631**, 627–634 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFCqs73E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38987592)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12088911)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Alveolar%20fibroblast%20lineage%20orchestrates%20lung%20inflammation%20and%20fibrosis&journal=Nature&volume=631&pages=627-634&publication_year=2024&author=Tsukui%2CT&author=Wolters%2CPJ&author=Sheppard%2CD) 
    
34.  Tsukui, T. et al. Collagen-producing lung cell atlas identifies multiple subsets with distinct localization and relevance to fibrosis. _Nat. Commun._ **11**, 1920 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnslSktL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32317643)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7174390)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Collagen-producing%20lung%20cell%20atlas%20identifies%20multiple%20subsets%20with%20distinct%20localization%20and%20relevance%20to%20fibrosis&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Tsukui%2CT) 
    
35.  Amrute, J. M. et al. Targeting immune–fibroblast cell communication in heart failure. _Nature_ **635**, 423–433 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitlanurrO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39443792)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12334188)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20immune%E2%80%93fibroblast%20cell%20communication%20in%20heart%20failure&journal=Nature&volume=635&pages=423-433&publication_year=2024&author=Amrute%2CJM) 
    
36.  Alexanian, M. et al. Chromatin remodelling drives immune cell–fibroblast communication in heart failure. _Nature_ **635**, 434–443 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitlantL3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39443808)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11698514)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20remodelling%20drives%20immune%20cell%E2%80%93fibroblast%20communication%20in%20heart%20failure&journal=Nature&volume=635&pages=434-443&publication_year=2024&author=Alexanian%2CM) 
    
37.  Chen, X. & Song, E. Turning foes to friends: targeting cancer-associated fibroblasts. _Nat. Rev. Drug Discov._ **18**, 99–115 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlClsL7J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30470818)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Turning%20foes%20to%20friends%3A%20targeting%20cancer-associated%20fibroblasts&journal=Nat.%20Rev.%20Drug%20Discov.&volume=18&pages=99-115&publication_year=2019&author=Chen%2CX&author=Song%2CE) 
    
38.  Pradhan, R. N., Krishnamurty, A. T., Fletcher, A. L., Turley, S. J. & Müller, S. A bird’s eye view of fibroblast heterogeneity: a pan-disease, pan-cancer perspective. _Immunological Rev._ **302**, 299–320 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtl2ltrjI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20bird%E2%80%99s%20eye%20view%20of%20fibroblast%20heterogeneity%3A%20a%20pan-disease%2C%20pan-cancer%20perspective&journal=Immunological%20Rev.&volume=302&pages=299-320&publication_year=2021&author=Pradhan%2CRN&author=Krishnamurty%2CAT&author=Fletcher%2CAL&author=Turley%2CSJ&author=M%C3%BCller%2CS) 
    
39.  Buechler, M. B. et al. Cross-tissue organization of the fibroblast lineage. _Nature_ **593**, 575–579 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVOksLnO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33981032)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-tissue%20organization%20of%20the%20fibroblast%20lineage&journal=Nature&volume=593&pages=575-579&publication_year=2021&author=Buechler%2CMB) 
    
40.  Korsunsky, I. et al. Cross-tissue, single-cell stromal atlas identifies shared pathological fibroblast phenotypes in four chronic inflammatory diseases. _Med_ **3**, 481–518.e14 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvFWktrvF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35649411)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-tissue%2C%20single-cell%20stromal%20atlas%20identifies%20shared%20pathological%20fibroblast%20phenotypes%20in%20four%20chronic%20inflammatory%20diseases&journal=Med&volume=3&pages=481-518.e14&publication_year=2022&author=Korsunsky%2CI) 
    
41.  Gao, Y. et al. Cross-tissue human fibroblast atlas reveals myofibroblast subtypes with distinct roles in immune modulation. _Cancer Cell_ **42**, 1764–1783.e10 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitVegu7%2FE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39303725)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-tissue%20human%20fibroblast%20atlas%20reveals%20myofibroblast%20subtypes%20with%20distinct%20roles%20in%20immune%20modulation&journal=Cancer%20Cell&volume=42&pages=1764-1783.e10&publication_year=2024&author=Gao%2CY) 
    
42.  Croft, A. P. et al. Distinct fibroblast subsets drive inflammation and damage in arthritis. _Nature_ **570**, 246–251 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVOgt7fL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31142839)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6690841)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20fibroblast%20subsets%20drive%20inflammation%20and%20damage%20in%20arthritis&journal=Nature&volume=570&pages=246-251&publication_year=2019&author=Croft%2CAP) 
    
43.  Krishnamurty, A. T. et al. LRRC15+ myofibroblasts dictate the stromal setpoint to suppress tumour immunity. _Nature_ **611**, 148–154 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFWit7jN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36171287)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9630141)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LRRC15%2B%20myofibroblasts%20dictate%20the%20stromal%20setpoint%20to%20suppress%20tumour%20immunity&journal=Nature&volume=611&pages=148-154&publication_year=2022&author=Krishnamurty%2CAT) 
    
44.  McCartney, E. E., Chung, Y. & Buechler, M. B. Life of Pi: exploring functions of _Pi16_+ fibroblasts. _F1000Res_ **13**, 126 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFOqs7nE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38919948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11196929)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Life%20of%20Pi%3A%20exploring%20functions%20of%20Pi16%2B%20fibroblasts&journal=F1000Res&volume=13&publication_year=2024&author=McCartney%2CEE&author=Chung%2CY&author=Buechler%2CMB) 
    
45.  Melms, J. C. et al. A molecular single-cell lung atlas of lethal COVID-19. _Nature_ **595**, 114–119 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFOltbbE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33915568)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8814825)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20molecular%20single-cell%20lung%20atlas%20of%20lethal%20COVID-19&journal=Nature&volume=595&pages=114-119&publication_year=2021&author=Melms%2CJC) 
    
46.  Horlbeck, M. A. et al. Compact and highly active next-generation libraries for CRISPR-mediated gene repression and activation. _eLife_ **5**, e19760 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27661255)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5094855)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Compact%20and%20highly%20active%20next-generation%20libraries%20for%20CRISPR-mediated%20gene%20repression%20and%20activation&journal=eLife&volume=5&publication_year=2016&author=Horlbeck%2CMA) 
    
47.  Replogle, J. M. et al. Maximizing CRISPRi efficacy and accessibility with dual-sgRNA libraries and optimal effectors. _eLife_ **11**, e81856 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisVOlt77J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36576240)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9829409)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Maximizing%20CRISPRi%20efficacy%20and%20accessibility%20with%20dual-sgRNA%20libraries%20and%20optimal%20effectors&journal=eLife&volume=11&publication_year=2022&author=Replogle%2CJM) 
    
48.  Gasperini, M. et al. A genome-wide framework for mapping gene regulation via cellular genetic screens. _Cell_ **176**, 377–390.e19 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXjs1Squg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30612741)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6690346)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genome-wide%20framework%20for%20mapping%20gene%20regulation%20via%20cellular%20genetic%20screens&journal=Cell&volume=176&pages=377-390.e19&publication_year=2019&author=Gasperini%2CM) 
    
49.  Yao, D. et al. Scalable genetic screening for regulatory circuits using compressed Perturb-seq. _Nat. Biotechnol._ **42**, 1282–1295 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitF2js7vI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37872410)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20genetic%20screening%20for%20regulatory%20circuits%20using%20compressed%20Perturb-seq.&journal=Nat.%20Biotechnol.&volume=42&pages=1282-1295&publication_year=2024&author=Yao%2CD) 
    
50.  Morris, J. A. et al. Discovery of target genes and pathways at GWAS loci by pooled single-cell CRISPR screens. _Science_ **380**, eadh7699 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtVKksrbE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37141313)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10518238)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20of%20target%20genes%20and%20pathways%20at%20GWAS%20loci%20by%20pooled%20single-cell%20CRISPR%20screens&journal=Science&volume=380&publication_year=2023&author=Morris%2CJA) 
    
51.  Datlinger, P. et al. Ultra-high-throughput single-cell RNA sequencing and perturbation screening with combinatorial fluidic indexing. _Nat. Methods_ **18**, 635–642 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1Sks77M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34059827)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612019)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ultra-high-throughput%20single-cell%20RNA%20sequencing%20and%20perturbation%20screening%20with%20combinatorial%20fluidic%20indexing&journal=Nat.%20Methods&volume=18&pages=635-642&publication_year=2021&author=Datlinger%2CP) 
    
52.  Wu, B. et al. Overloading and unpacKing (OAK)—droplet-based combinatorial indexing for ultra-high throughput single-cell multiomic profiling. _Nat. Commun._ **15**, 9146 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitleqtbfK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39443484)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11499997)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overloading%20and%20unpacKing%20%28OAK%29%E2%80%94droplet-based%20combinatorial%20indexing%20for%20ultra-high%20throughput%20single-cell%20multiomic%20profiling&journal=Nat.%20Commun.&volume=15&publication_year=2024&author=Wu%2CB) 
    
53.  Wu, Q. et al. Massively parallel characterization of CRISPR activator efficacy in human induced pluripotent stem cells and neurons. _Mol. Cell_ **83**, 1125–1139.e8 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXltV2qtrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36917981)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10114495)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Massively%20parallel%20characterization%20of%20CRISPR%20activator%20efficacy%20in%20human%20induced%20pluripotent%20stem%20cells%20and%20neurons&journal=Mol.%20Cell&volume=83&pages=1125-1139.e8&publication_year=2023&author=Wu%2CQ) 
    
54.  Loyfer, N. et al. A DNA methylation atlas of normal human cell types. _Nature_ **613**, 355–364 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXktFOnug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36599988)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9811898)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20DNA%20methylation%20atlas%20of%20normal%20human%20cell%20types&journal=Nature&volume=613&pages=355-364&publication_year=2023&author=Loyfer%2CN) 
    
55.  Cazares, T. A. et al. maxATAC: genome-scale transcription-factor binding prediction from ATAC-seq with deep neural networks. _PLoS Comput. Biol._ **19**, e1010863 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtlentbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36719906)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9917285)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=maxATAC%3A%20genome-scale%20transcription-factor%20binding%20prediction%20from%20ATAC-seq%20with%20deep%20neural%20networks&journal=PLoS%20Comput.%20Biol.&volume=19&publication_year=2023&author=Cazares%2CTA) 
    
56.  Tribolet-Hardy, J. et al. Genetic features and genomic targets of human KRAB-zinc finger proteins. _Genome Res._ **33**, 1409–1423 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37730438)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10547255)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20features%20and%20genomic%20targets%20of%20human%20KRAB-zinc%20finger%20proteins&journal=Genome%20Res.&volume=33&pages=1409-1423&publication_year=2023&author=Tribolet-Hardy%2CJ) 
    
57.  O’Geen, H., Henry, I. M., Bhakta, M. S., Meckler, J. F. & Segal, D. J. A genome-wide analysis of Cas9 binding specificity using ChIP-seq and targeted sequence capture. _Nucleic Acids Res._ **43**, 3389–3404 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25712100)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4381059)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genome-wide%20analysis%20of%20Cas9%20binding%20specificity%20using%20ChIP-seq%20and%20targeted%20sequence%20capture&journal=Nucleic%20Acids%20Res.&volume=43&pages=3389-3404&publication_year=2015&author=O%E2%80%99Geen%2CH&author=Henry%2CIM&author=Bhakta%2CMS&author=Meckler%2CJF&author=Segal%2CDJ) 
    
58.  Rostain, W. et al. Cas9 off-target binding to the promoter of bacterial genes leads to silencing and toxicity. _Nucleic Acids Res._ **51**, 3485–3496 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1ylsrrN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36929199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10123097)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cas9%20off-target%20binding%20to%20the%20promoter%20of%20bacterial%20genes%20leads%20to%20silencing%20and%20toxicity&journal=Nucleic%20Acids%20Res.&volume=51&pages=3485-3496&publication_year=2023&author=Rostain%2CW) 
    
59.  Uthaya et al. A genome-wide CRISPR activation screen identifies _SCREEM_ a novel _SNAI1_ super-enhancer demarcated by eRNAs. _Front. Mol. Biosci._ **10**, 1110445 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genome-wide%20CRISPR%20activation%20screen%20identifies%20SCREEM%20a%20novel%20SNAI1%20super-enhancer%20demarcated%20by%20eRNAs&journal=Front.%20Mol.%20Biosci.&volume=10&publication_year=2023&author=Uthaya%2C) 
    
60.  Nuñez, J. K. et al. Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing. _Cell_ **184**, 2503–2519.e17 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33838111)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8376083)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20programmable%20transcriptional%20memory%20by%20CRISPR-based%20epigenome%20editing&journal=Cell&volume=184&pages=2503-2519.e17&publication_year=2021&author=Nu%C3%B1ez%2CJK) 
    
61.  Li, Y. et al. Genome-wide analyses reveal a role of Polycomb in promoting hypomethylation of DNA methylation valleys. _Genome Biol._ **19**, 18 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29422066)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5806489)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20analyses%20reveal%20a%20role%20of%20Polycomb%20in%20promoting%20hypomethylation%20of%20DNA%20methylation%20valleys&journal=Genome%20Biol.&volume=19&publication_year=2018&author=Li%2CY) 
    
62.  Cui, A. et al. Dictionary of immune responses to cytokines at single-cell resolution. _Nature_ **625**, 377–384 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFGhu7jJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38057668)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dictionary%20of%20immune%20responses%20to%20cytokines%20at%20single-cell%20resolution&journal=Nature&volume=625&pages=377-384&publication_year=2024&author=Cui%2CA) 
    
63.  Fang, F. et al. Early growth response 3 (Egr-3) is induced by transforming growth factor-β and regulates fibrogenic responses. _Am. J. Pathol._ **183**, 1197–1208 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsV2kur3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23906810)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3791870)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Early%20growth%20response%203%20%28Egr-3%29%20is%20induced%20by%20transforming%20growth%20factor-%CE%B2%20and%20regulates%20fibrogenic%20responses&journal=Am.%20J.%20Pathol.&volume=183&pages=1197-1208&publication_year=2013&author=Fang%2CF) 
    
64.  Li, A. et al. GATA6 triggers fibroblast activation and tracheal fibrosis through the Wnt/β-catenin pathway. _Cell. Signal._ **105**, 110593 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvFWgs7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36682592)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GATA6%20triggers%20fibroblast%20activation%20and%20tracheal%20fibrosis%20through%20the%20Wnt%2F%CE%B2-catenin%20pathway&journal=Cell.%20Signal.&volume=105&publication_year=2023&author=Li%2CA) 
    
65.  Stebler, S. & Raghunath, M. The Scar-in-a-Jar: In Vitro Fibrosis Model for Anti-Fibrotic Drug Testing. In _Myofibroblasts: Methods and Protocols_ (eds. Hinz, B. & Lagares, D.) 147–156 (Springer, 2021). [https://doi.org/10.1007/978-1-0716-1382-5\_11](https://doi.org/10.1007/978-1-0716-1382-5_11)
    
66.  Vázquez-García, I. et al. Ovarian cancer mutational processes drive site-specific immune evasion. _Nature_ **612**, 778–786 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36517593)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9771812)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ovarian%20cancer%20mutational%20processes%20drive%20site-specific%20immune%20evasion&journal=Nature&volume=612&pages=778-786&publication_year=2022&author=V%C3%A1zquez-Garc%C3%ADa%2CI) 
    
67.  Pollak, N. M., Hoffman, M., Goldberg, I. J. & Drosatos, K. Krüppel-like factors: crippling and uncrippling metabolic pathways. _JACC Basic Transl. Sci._ **3**, 132–156 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29876529)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5985828)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Kr%C3%BCppel-like%20factors%3A%20crippling%20and%20uncrippling%20metabolic%20pathways&journal=JACC%20Basic%20Transl.%20Sci.&volume=3&pages=132-156&publication_year=2018&author=Pollak%2CNM&author=Hoffman%2CM&author=Goldberg%2CIJ&author=Drosatos%2CK) 
    
68.  Varrault, A. et al. Identification of Plagl1/Zac1 binding sites and target genes establishes its role in the regulation of extracellular matrix genes and the imprinted gene network. _Nucleic Acids Res._ **45**, 10466–10480 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXps1Kitr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28985358)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5737700)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20Plagl1%2FZac1%20binding%20sites%20and%20target%20genes%20establishes%20its%20role%20in%20the%20regulation%20of%20extracellular%20matrix%20genes%20and%20the%20imprinted%20gene%20network&journal=Nucleic%20Acids%20Res.&volume=45&pages=10466-10480&publication_year=2017&author=Varrault%2CA) 
    
69.  Jonsson, M. K. B. et al. A transcriptomic and epigenomic comparison of fetal and adult human cardiac fibroblasts reveals novel key transcription factors in adult cardiac fibroblasts. _JACC Basic Transl. Sci._ **1**, 590–602 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30167544)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6113540)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20transcriptomic%20and%20epigenomic%20comparison%20of%20fetal%20and%20adult%20human%20cardiac%20fibroblasts%20reveals%20novel%20key%20transcription%20factors%20in%20adult%20cardiac%20fibroblasts&journal=JACC%20Basic%20Transl.%20Sci.&volume=1&pages=590-602&publication_year=2016&author=Jonsson%2CMKB) 
    
70.  Tsuda, T. et al. Zinc finger protein _Zac1_ is expressed in chondrogenic sites of the mouse. _Dev. Dyn._ **229**, 340–348 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhvVeku78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14745959)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Zinc%20finger%20protein%20Zac1%20is%20expressed%20in%20chondrogenic%20sites%20of%20the%20mouse&journal=Dev.%20Dyn.&volume=229&pages=340-348&publication_year=2004&author=Tsuda%2CT) 
    
71.  Chrysanthopoulou, A. et al. Down-regulation of KLF2 in lung fibroblasts is linked with COVID-19 immunofibrosis and restored by combined inhibition of NETs, JAK-1/2 and IL-6 signaling. _Clin. Immunol._ **247**, 109240 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVOhsr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36693535)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9862710)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Down-regulation%20of%20KLF2%20in%20lung%20fibroblasts%20is%20linked%20with%20COVID-19%20immunofibrosis%20and%20restored%20by%20combined%20inhibition%20of%20NETs%2C%20JAK-1%2F2%20and%20IL-6%20signaling&journal=Clin.%20Immunol.&volume=247&publication_year=2023&author=Chrysanthopoulou%2CA) 
    
72.  Shi, J. et al. KLF2 attenuates bleomycin-induced pulmonary fibrosis and inflammation with regulation of AP-1. _Biochem. Biophys. Res. Commun._ **495**, 20–26 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslKqs7nI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29079188)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KLF2%20attenuates%20bleomycin-induced%20pulmonary%20fibrosis%20and%20inflammation%20with%20regulation%20of%20AP-1&journal=Biochem.%20Biophys.%20Res.%20Commun.&volume=495&pages=20-26&publication_year=2018&author=Shi%2CJ) 
    
73.  Chandran, R. R. et al. Distinct roles of KLF4 in mesenchymal cell subtypes during lung fibrogenesis. _Nat. Commun._ **12**, 7179 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislars7fE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34893592)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8664937)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20roles%20of%20KLF4%20in%20mesenchymal%20cell%20subtypes%20during%20lung%20fibrogenesis&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Chandran%2CRR) 
    
74.  Penke, L. R. et al. KLF4 is a therapeutically tractable brake on fibroblast activation that promotes resolution of pulmonary fibrosis. _JCI Insight_ **7**, e160688 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35852857)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9462506)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KLF4%20is%20a%20therapeutically%20tractable%20brake%20on%20fibroblast%20activation%20that%20promotes%20resolution%20of%20pulmonary%20fibrosis&journal=JCI%20Insight&volume=7&publication_year=2022&author=Penke%2CLR) 
    
75.  Noda, S. et al. Simultaneous downregulation of KLF5 and Fli1 is a key feature underlying systemic sclerosis. _Nat. Commun._ **5**, 5797 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvFahsb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25504335)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Simultaneous%20downregulation%20of%20KLF5%20and%20Fli1%20is%20a%20key%20feature%20underlying%20systemic%20sclerosis&journal=Nat.%20Commun.&volume=5&publication_year=2014&author=Noda%2CS) 
    
76.  Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. _Nature_ **518**, 317–330 (2015).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4530010)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&journal=Nature&volume=518&pages=317-330&publication_year=2015&author=Roadmap%20Epigenomics%20Consortium%2C) 
    
77.  Wang, X. et al. Antibody-free profiling of transcription factor occupancy during early embryogenesis by FitCUT&RUN. _Genome Res._ **32**, 378–388 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXit1CmsLnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34965941)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8805719)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Antibody-free%20profiling%20of%20transcription%20factor%20occupancy%20during%20early%20embryogenesis%20by%20FitCUT%26RUN&journal=Genome%20Res.&volume=32&pages=378-388&publication_year=2022&author=Wang%2CX) 
    
78.  Southard, K. et al. Hs27-CRISPRa-TFs cellranger outputs. _Zenodo_ [https://doi.org/10.5281/ZENODO.15213597](https://doi.org/10.5281/ZENODO.15213597) (2025).
    
79.  Southard, K. et al. Hs27-CRISPRa-TFs. _Zenodo_ [https://doi.org/10.5281/ZENODO.15200179](https://doi.org/10.5281/ZENODO.15200179) (2025).
    
80.  Southard, K. et al. RPE1-CRISPRa-TFs cellranger outputs. _Zenodo_ [https://doi.org/10.5281/ZENODO.15211972](https://doi.org/10.5281/ZENODO.15211972) (2025).
    
81.  Southard, K. et al. RPE1-CRISPRa-TFs. _Zenodo_ [https://doi.org/10.5281/ZENODO.15213619](https://doi.org/10.5281/ZENODO.15213619) (2025).
    
82.  Southard, K. et al. RPE1-E150-Benchmarking cellranger outputs. _Zenodo_ [https://doi.org/10.5281/ZENODO.15215389](https://doi.org/10.5281/ZENODO.15215389) (2025).
    
83.  Southard, K. et al. RPE1-E150-Benchmarking. _Zenodo_ [https://doi.org/10.5281/ZENODO.15215414](https://doi.org/10.5281/ZENODO.15215414) (2025).
    
84.  Southard, K. et al. K562 dCas9-CUT&RUN. _Zenodo_ [https://doi.org/10.5281/ZENODO.15215154](https://doi.org/10.5281/ZENODO.15215154) (2025).
    
85.  Southard, K. et al. RPE-1 and Hs27 CUT&RUN, ATAC-seq, and RNA-seq characterization. _Zenodo_ [https://doi.org/10.5281/ZENODO.15215216](https://doi.org/10.5281/ZENODO.15215216) (2025).
    
86.  K. Southard & Norman, T. norman-lab-msk/TFs\_CRISPRa: CRISPRa TFs Perturb-seq v0.1. _Zenodo_ [https://doi.org/10.5281/ZENODO.15373940](https://doi.org/10.5281/ZENODO.15373940) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02284-1?format=refman&flavour=references)

Acknowledgements
----------------

We gratefully acknowledge all the members of the Norman Lab for their valuable thoughts and discussion on this work. We thank D. H. Al-Rawi for her insight and support, and A. Mohammad for his assistance with data transfer. We additionally thank the Shah lab (MSKCC) for facilitating analysis of the MSK-SPECTRUM fibroblasts. We acknowledge the use of the Epigenetics Research Innovation Lab, the Integrated Genomics Operation Core and the Flow Cytometry Core Facility, which is funded by the NCI Cancer Center Support Grant (CCSG, P30 CA08748), Cycle for Survival and the Marie-Josée and Henry R. Kravis Center for Molecular Oncology. This work was funded by the National Institutes of Health (NIH) Director’s New Innovator Award DP2 GM140925, Damon Runyon Cancer Research Foundation Dale Frey Award AWD-GC-259296 and NIH U01 HG012103 (to T.M.N.). R.C.A. is a Robert Black Fellow of the Damon Runyon Cancer Research Foundation (DRG-2462-22), and D.D.O. was supported by NIH T32 GM132083. This research was funded in part through the NIH/NCI Cancer Center Support Grant P30 CA008748.

Author information
------------------

Author notes

1.  These authors contributed equally: Kaden M. Southard, Rico C. Ardy.
    

### Authors and Affiliations

1.  Computational and Systems Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan & Thomas M. Norman
    
2.  Louis V. Gerstner, Jr. Graduate School of Biomedical Sciences, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Anran Tang
    
3.  Tri-Institutional Training Program in Computational Biology and Medicine, New York, NY, USA
    
    Deirdre D. O’Sullivan, Eli Metzner & Karthik Guruvayurappan
    

Authors

1.  Kaden M. Southard
    
    [View author publications](https://www.nature.com/search?author=Kaden%20M.%20Southard)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kaden%20M.%20Southard) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kaden%20M.%20Southard%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Rico C. Ardy
    
    [View author publications](https://www.nature.com/search?author=Rico%20C.%20Ardy)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rico%20C.%20Ardy) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rico%20C.%20Ardy%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Anran Tang
    
    [View author publications](https://www.nature.com/search?author=Anran%20Tang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anran%20Tang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anran%20Tang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Deirdre D. O’Sullivan
    
    [View author publications](https://www.nature.com/search?author=Deirdre%20D.%20O%E2%80%99Sullivan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Deirdre%20D.%20O%E2%80%99Sullivan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Deirdre%20D.%20O%E2%80%99Sullivan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Eli Metzner
    
    [View author publications](https://www.nature.com/search?author=Eli%20Metzner)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eli%20Metzner) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eli%20Metzner%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Karthik Guruvayurappan
    
    [View author publications](https://www.nature.com/search?author=Karthik%20Guruvayurappan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Karthik%20Guruvayurappan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Karthik%20Guruvayurappan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Thomas M. Norman
    
    [View author publications](https://www.nature.com/search?author=Thomas%20M.%20Norman)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Thomas%20M.%20Norman) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Thomas%20M.%20Norman%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

K.M.S., R.C.A. and T.M.N. conceptualized the study. K.M.S., R.C.A., A.T., E.M. and T.M.N. developed the methodology. K.M.S., A.T., K.G., E.M. and T.M.N. performed formal analysis. R.C.A. validated the work. K.M.S., R.C.A., E.M. and D.D.O. conducted the investigation. K.M.S., R.C.A., A.T. and T.M.N. wrote the original draft of the manuscript; K.M.S., R.C.A., A.T. and T.M.N. contributed to manuscript review and editing. K.M.S., R.C.A., A.T., K.G., E.M. and T.M.N. visualized the project. T.M.N. supervised the research. T.M.N. acquired funding.

### Corresponding author

Correspondence to [Thomas M. Norman](mailto:normantm@mskcc.org).

Ethics declarations
-------------------

### Competing interests

T.M.N. is an author on US Patent No. 11,214,797B2, related to Perturb-seq. The remaining authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02284-1#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Development and testing of VPH-dCas9 CRISPRa effector.](https://www.nature.com/articles/s41588-025-02284-1/figures/8)

**a**. Schematic of VPH-dCas9 CRISPRa effector constructs used in this study. BFP construct (top) was used in BJ fibroblast CRISPRa cell lines, while the mCherry construct (bottom) was used in to create Hs27 fibroblast and RPE-1 CRISRPa cell lines. **b**. Flow cytometry data demonstrating activation of cell surface markers CD45 and CD55 by the VPH-dCas9 CRISPRa system in Hs27 and RPE-1 cells. **c**. Flow cytometry data showing variable efficacy of eight predicted CRISPRa guides targeting _CD45_ in BJ and Hs27 fibroblast cells. Non-targeting control is shown in light blue. **d**. Design and quality control analysis of the transcription factor CRISPRa library. Most genes are targeted with 6 independent guides filtered for off-target binding. There is also an overrepresented library of 81 non-targeting control guides that is spiked-in separately. Right panels assess evenness of representation of different guides as measured by sequencing after cloning. Counts are normalized so that a representation of 1 corresponds to the median representation of all guides in that library.

### [Extended Data Fig. 2 Testing performance of droplet overloading model, related to Fig.](https://www.nature.com/articles/s41588-025-02284-1/figures/9) [1](https://www.nature.com/articles/s41588-025-02284-1#Fig1).

**a**. To benchmark reconstruction accuracy, we split datasets into disjoint subsets, infer regression coefficients using a linear model, and assess reconstruction with various metrics. The simplest metric computes Pearson correlation between the (stacked) regression coefficient matrices. **b**. Visual comparison of regression coefficients inferred from singlets (left) vs. an equivalent number of doublets (right), either with (top) or without (bottom) stable gene-based normalization. Stable gene normalization removes differences caused by transcript capture efficiency, producing visually indistinguishable results. Summary shown in Fig. [1h](https://www.nature.com/articles/s41588-025-02284-1#Fig1) (“Doublets” bar). **c**. Design of doublet downsampling experiment. A baseline is established by correlating regression coefficients from two disjoint sets of 16,800 singlets. Doublets (_n_ = 19,512) are then subsampled to varying depths, and correlations are computed to evaluate the relative performance of reconstruction from doublets alone. **d**. An alternative experiment design mimics real-world usage by subsampling droplet mixtures proportionally to their observed frequencies. **e**. Visual comparison of regression coefficients inferred using either all singlets (left) vs. all droplets with up to 6 cells (right), either with (top) or without (bottom) stable gene normalization. As in (**b**), normalization removes structural artifacts. **f**. Another approach for assessing performance is to compare (using Pearson correlation) the reconstructed perturbation-perturbation correlation matrices, which summarize relative perturbation effects used in downstream analyses like clustering. Experimental design as in Fig. [1h](https://www.nature.com/articles/s41588-025-02284-1#Fig1). **g**. The regression model also produces _p_\-values for differential expression. We assess similarity of inferred differentially expressed genes by stacking _p_\-value matrices and computing Spearman correlations, using the same design as Fig. [1h](https://www.nature.com/articles/s41588-025-02284-1#Fig1). **h**. Spearman correlation of _p_\-values at varying levels of downsampling using the design from (**c**). Blue and magenta denote results with and without stable gene normalization, respectively. Dashed lines show performance from baseline computed using 16,800 withheld singlets. **i**. Overall perturbation strength can be quantified by computing the number of differentially expressed genes (FDR < 5%) per perturbation. Correlating these counts across reconstructions evaluates consistency in ranking perturbation strength. Colors and baselines as in (**h**). **j**. Comparison of differentially expressed gene counts (FDR < 5%) inferred from singlets vs. all droplets with ≤6 cells. Dashed line denotes line of equality.

### [Extended Data Fig. 3 On-target activity of the TF-CRISPRa library and modeling of the epigenetic determinants of susceptibility to CRISPRa, related to Fig.](https://www.nature.com/articles/s41588-025-02284-1/figures/10) [2](https://www.nature.com/articles/s41588-025-02284-1#Fig2).

**a**. Detailed summary of guide activity in Hs27 vs. RPE-1 cells (compare with Fig. [2c](https://www.nature.com/articles/s41588-025-02284-1#Fig2)). Guides are classified based on whether the target gene is expressed in control cells (containing non-targeting guides) and whether the guide shows on-target activation in our regression procedure (Methods). This conservative approach may miss marginally active guides due to noise intrinsic to single-cell RNA sequencing. **b**. Upset plot showing overlap in guides with on-target activity between Hs27 and RPE-1. **c**. Upset plot showing overlap in activatable genes (via at least one guide) between Hs27 and RPE-1. **d**. CRISPRa exhibits diminishing returns as baseline expression increases. Genes are ranked by mean expression in control cells. Plot shows the rolling median of target gene fold-change in expression. **e**. For each transcription factor in Hs27 cells, plot compares direct CRISPRa-induced expression to the maximal indirect activation because of perturbing another factor. **f**. As in (e) for RPE-1 cells. **g**. Selection of promoters for modeling epigenetic determinants of CRISPRa responsiveness. We focused on genes unexpressed in controls but activatable by either: two out of three guides, when only three guides were available, or at least three distinct guides otherwise. Plot compares epigenetic features used for model training for genes above and below our expression threshold (red line). **h**. KRAB zinc finger (KRAB-ZNF) TFs are resistant to CRISPRa activation. Plot shows number of active guides for KRAB-ZNFs vs. other TFs. **i**. Top: Selected features at promoters that were and were not activatable in Hs27. Bottom: Features from reduced model in RPE-1. **j**. For a subset of genes, our experiment included guides targeting distinct transcription start sites that behaved differently. Plots compare selected features at example promoters in Hs27 cells. Red indicates activatable promoters. **k**. Comparison of genomic features used in logistic regression modeling to predict CRISPRa susceptibility at two _LMO2_ promoters. The CRISPRa-responsive promoter (top) and CRISPRa-refractory promoter (bottom) are shown with their associated features. Predicted YY1 binding (pred-YY1) was predicted from ATAC-seq data using MaxATAC. **l**. Predictive performance in RPE-1 cells of a reduced model trained on pred-YY1, H3K27me3, and KRAB zinc finger features from Hs27 cells.

### [Extended Data Fig. 4 Seed-driven phenotypes in other datasets, related to Fig.](https://www.nature.com/articles/s41588-025-02284-1/figures/11) [3](https://www.nature.com/articles/s41588-025-02284-1#Fig3).

**a**. Scatter plot comparing the strength of on-target activation induced by each guide to the number of differentially expressed genes. **b**. Comparison of target activation between guides with or without bad seeds. **c**. In a genome-wide CRISPRa fitness screen using an alternative SunTag CRISPRa effector[11](https://www.nature.com/articles/s41588-025-02284-1#ref-CR11 "Gilbert, L. A. et al. Genome-scale CRISPR-mediated control of gene repression and activation. Cell 159, 647–661 (2014)."), a regression model (Methods) assigning guides’ effects to on-target and seed-dependent components identifies seeds with consistent fitness effects across guides. **d**. Regression analysis as in C applied to a CRISPRa screen targeting enhancers using an alternative VPH CRISPRa effector[59](https://www.nature.com/articles/s41588-025-02284-1#ref-CR59 "Uthaya et al. A genome-wide CRISPR activation screen identifies SCREEM a novel SNAI1 super-enhancer demarcated by eRNAs. Front. Mol. Biosci. 10, 1110445 (2023)."). **e**. Regression analysis as in **c** applied to a genome-wide CRISPRa using a PH CRISPRa effector[17](https://www.nature.com/articles/s41588-025-02284-1#ref-CR17 "Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. Nat. Commun. 9, 5416 (2018)."). **f**. Regression analysis as in C applied to a genome-wide CRISPRa using an alternative VPH CRISPRa effector[17](https://www.nature.com/articles/s41588-025-02284-1#ref-CR17 "Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. Nat. Commun. 9, 5416 (2018)."). **g**. Median fitness effects in non-targeting control sgRNAs grouped by seed in a genome-wide CRISPRi screen[11](https://www.nature.com/articles/s41588-025-02284-1#ref-CR11 "Gilbert, L. A. et al. Genome-scale CRISPR-mediated control of gene repression and activation. Cell 159, 647–661 (2014)."). **h**. Identification of seed sequence-driven clusters in a large-scale CRISPRi Perturb-seq experiment in K562 cells[26](https://www.nature.com/articles/s41588-025-02284-1#ref-CR26 "Replogle, J. M. et al. Mapping information-rich genotype–phenotype landscapes with genome-scale Perturb-seq. Cell 185, 2559–2575.e28 (2022)."). In this case each construct contains two guides (guide seeds shown as: g1, g2), with one guide almost always containing an AGAGG seed. **i**. Regression analysis as in C applied to sgRNAs from a genome-wide CRISPRi screen[11](https://www.nature.com/articles/s41588-025-02284-1#ref-CR11 "Gilbert, L. A. et al. Genome-scale CRISPR-mediated control of gene repression and activation. Cell 159, 647–661 (2014)."). **j**. Median fitness effects in non-targeting control sgRNAs grouped by seed in a CRISPRoff fitness screen[60](https://www.nature.com/articles/s41588-025-02284-1#ref-CR60 "Nuñez, J. K. et al. Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing. Cell 184, 2503–2519.e17 (2021).").

### [Extended Data Fig. 5 Classification of guides and targets by phenotypic effect and transcription factor family, related to Fig.](https://www.nature.com/articles/s41588-025-02284-1/figures/12) [4](https://www.nature.com/articles/s41588-025-02284-1#Fig4).

**a**. Guide classification based on transcriptional phenotypes. We identified transcription factors causing reproducible changes in transcriptional phenotypes using the clustering-based criterion described in the text. Guides are classified as “active” if at least two guides targeting the same transcription factor cluster together in our phenotypic analysis. Additionally, guides targeting other transcription factors that fall into these active clusters are classified as “expanded active.” The upset plots compare these classifications with the on-target activation metrics established in Fig. [2](https://www.nature.com/articles/s41588-025-02284-1#Fig2). On-target activation may not be detected for some guides because our criteria are conservative and due to lowly expressed target genes being masked by limited sequencing depth. **b**. These plots replicate Fig. [4](https://www.nature.com/articles/s41588-025-02284-1#Fig4) E/F, but with coloring by transcription factor family to highlight how homology drives clustering. TFs separated by “/” indicate co-clustering.

### [Extended Data Fig. 6 Cell-type-specific differences in gene responsiveness to perturbations, related to Fig.](https://www.nature.com/articles/s41588-025-02284-1/figures/13) [4](https://www.nature.com/articles/s41588-025-02284-1#Fig4).

**a**. We identify two sets of genes that vary within our experiments. First, Highly Variable Genes (HVGs) are defined as having abnormally high expression variability relative to genes of similar expression levels, which is here termed “excess CV”. From this we identify Perturbation Variable Genes (PVGs), which show increased variability specifically in response to perturbations. In technical terms, PVGs have high excess CV in perturbed cells relative to control cells. Top left shows overlap of PVGs (top 1% of genes by this metric) between cell types. The other panels compare the fold increase in excess CV between perturbed and control cells across the two cell types, broken down into groups based on whether the gene is expressed specifically in one cell type (left and top) or detected in both (right). **b**. HVGs in Hs27 and RPE-1 cells in response to CRISPRa perturbations of transcription factors. Top left panel shows overlap of HVGs, defined as the top 1% of genes scored via this metric. The other panels compare excess CV measurements of genes across the two cell types, with genes grouped as in (**a**). Selected HVGs are labeled, with a notable enrichment of cell cycle regulators in conserved HVGs (purple), indicating that some HVGs are driven by endogenous sources of variation.

[Source data](https://www.nature.com/articles/s41588-025-02284-1#MOESM9)

### [Extended Data Fig. 7 Extraction of gene expression programs using sparse PCA, related to Fig.](https://www.nature.com/articles/s41588-025-02284-1/figures/14) [4](https://www.nature.com/articles/s41588-025-02284-1#Fig4).

**a**. Extraction of interpretable gene expression programs. We used a modified sparse PCA approach that enforces positivity of coefficients so that genes in programs rise or fall together. To identify the number of programs to extract, we fit the model over bootstrapped resamples of the dataset, used HDBSCAN to cluster the extracted programs, and then reran the algorithm on the dataset using the number of HDBSCAN clusters as the dimensionality (Methods). Figure shows the structure of sparse gene expression programs, which can overlap and vary in size. **b**. Programs (rows) are defined by positive weights on genes (columns) in Hs27 fibroblasts. All program definitions are in Supplemental Table [5](https://www.nature.com/articles/s41588-025-02284-1#MOESM4). **c**. Program 1, corresponding to MYC activity. Visualization shows highest weighted genes in the program (bottom) and top scoring negative (top) and positive (bottom) regulators. **d**. Program 47, corresponding to mTORC1-mediated activation of ATF4. **e**. Program 0, corresponding to the cell cycle. **f**. Program 21, corresponding to an oxidative stress response driven by _NFE2L2_. This response was also identified in the GO term analysis in Fig. [4h](https://www.nature.com/articles/s41588-025-02284-1#Fig4). **g**. Program 19, corresponding to antigen presentation and driven largely by _IRF1_. This response was also identified in the GO term analysis in Fig. [4h](https://www.nature.com/articles/s41588-025-02284-1#Fig4). **h**. Concordant programs identified via the sparse PCA approach in RPE-1 cells.

### [Extended Data Fig. 8 Fibroblast gene program links with _in vivo_ states and phenotypic validation, related to Figs.](https://www.nature.com/articles/s41588-025-02284-1/figures/15) [5](https://www.nature.com/articles/s41588-025-02284-1#Fig5) and [6](https://www.nature.com/articles/s41588-025-02284-1#Fig6).

**a**. Analysis of overlap between gene programs identified by sparse PCA in our Perturb-seq data (_x_\-axes) and previously defined fibroblast states from four independent atlases of fibroblast states (_y_\-axes). Significant associations (FDR < 0.1) were determined by permutation testing comparing the top 50 genes from each program with differentially expressed genes from each state. Significant links to recurring gene programs are summarized in Fig. [5a](https://www.nature.com/articles/s41588-025-02284-1#Fig5). **b**. Consensus state marker genes with state-specific expression across fibroblast atlases. Five representative markers for each state (myofibroblast, inflammatory, universal, and antigen presentation) were chosen based on their consistent differential expression across atlases and membership in the corresponding _in vitro_ gene programs. **c**. Immunofluorescence imaging showing morphological changes in Hs27 fibroblasts following CRISPRa perturbation of _HES7_ and _ZNF296_ compared to non-targeting control (NTC). Top: DAPI nuclear staining; Bottom: ACTA2 staining. Scale bars: 200 μm. **d**. Analysis of the alternative inflammatory Program 27. Heatmap displays program-defining genes (columns) and their expression changes in response to repressing (top) and activating (bottom) transcription factor perturbations (rows). Changes are quantified relative to expression in control cells. **e**. Quantification of _de novo_ gene expression induced by transcription factors. Newly induced genes were identified using the pipeline for quantifying on-target activation from Fig. [3](https://www.nature.com/articles/s41588-025-02284-1#Fig3). Left: Total number of newly induced genes per transcription factor. Right: Detailed analysis highlighting newly induced genes, including _PI16_ activation by _KLF5_. **f**. Left: Flow cytometry analysis of PI16 expression in BJ fibroblasts following cDNA overexpression of _KLF4_ and _KLF5_ compared to empty vector (EV) control. Right: flow cytometry gating strategy for the analysis of PI16 expression for both cDNA and CRISPRa PI16 validation experiments. Gating was as follows: cells in FSC-A and SSC-A (top left), single-cells (FSC-A, FSC-H, top right), GFP+ (cDNA/guide expression), histogram of PI16 staining.

[Source data](https://www.nature.com/articles/s41588-025-02284-1#MOESM10)

### [Extended Data Fig. 9 Extended analysis of universal and inflammatory state regulators across fibroblast atlases, related to Fig.](https://www.nature.com/articles/s41588-025-02284-1/figures/16) [7](https://www.nature.com/articles/s41588-025-02284-1#Fig7).

**a**. Analysis of transcription factor activity _in vitro_ versus correlation with state markers _in vivo_ across four independent fibroblast atlases, extending the analysis in Fig. [7a-c](https://www.nature.com/articles/s41588-025-02284-1#Fig7). For each atlas, scatter plots compare each transcription factor’s ability to induce universal (left) or inflammatory (right) programs in vitro (_y_\-axes) versus its expression correlation with state markers (_PI16_ or _COL3A1_, respectively) _in vivo_ (_x_\-axes). As shown in Fig. [7a](https://www.nature.com/articles/s41588-025-02284-1#Fig7), transcription factors are categorized as conditionally active (magenta), correlated but inactive (yellow), or potentially causal regulators (blue). Dashed lines indicate outlier thresholds defined as 1.5 x interquartile range. **b**. UMAP visualization showing expression pattern of _KLF2_ in comparison to _PI16_ in fibroblasts within the ovarian cancer dataset. **c**. Expression of state markers and key transcription factors across atlas-defined fibroblast states. Dot size indicates fraction of expressing cells and color intensity shows mean expression level. The ratio of _PLAGL1_ to _KLF2/4_ expression (purple/yellow scale) specifically marks inflammatory fibroblasts, which have both high _PLAGL1_ and low _KLF2/4_ levels. **d**. Collagen deposition visualized by immunofluorescence in Hs27 fibroblasts with single or combined perturbations of _EGR3_ and _KLF4_. Top: DAPI nuclear staining; Bottom: COL1 staining. Compare with Fig. [7f](https://www.nature.com/articles/s41588-025-02284-1#Fig7).

[Source data](https://www.nature.com/articles/s41588-025-02284-1#MOESM11)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM1_ESM.pdf)

Supplementary Figs. 1–3, Supplementary Notes and Methods.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM3_ESM.pdf)

### [Supplementary Tables 1–19](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM4_ESM.xlsx)

Supplementary Tables 1–19.

Source data
-----------

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM5_ESM.xlsx)

Statistical Source Data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM6_ESM.xlsx)

Statistical Source Data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM7_ESM.zip)

Unprocessed image files.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM8_ESM.zip)

Unprocessed image files.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM9_ESM.xlsx)

Statistical Source Data.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM10_ESM.zip)

Unprocessed image files.

### [Source Data Extended Data Fig. 9](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02284-1/MediaObjects/41588_2025_2284_MOESM11_ESM.zip)

Unprocessed image files.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Comprehensive%20transcription%20factor%20perturbations%20recapitulate%20fibroblast%20transcriptional%20states&author=Kaden%20M.%20Southard%20et%20al&contentID=10.1038%2Fs41588-025-02284-1&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-06&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02284-1)

### Cite this article

Southard, K.M., Ardy, R.C., Tang, A. _et al._ Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02284-1

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02284-1?format=refman&flavour=citation)

-   Received: 24 February 2025
    
-   Accepted: 27 June 2025
    
-   Published: 06 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02284-1
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
