> id: https://www.nature.com/articles/s41588-025-02294-z

> link: https://www.nature.com/articles/s41588-025-02294-z

> title: Super-pangenome analyses across 35 accessions of 23 Avena species highlight their complex evolutionary history and extensive genomic diversity

# Super-pangenome analyses across 35 accessions of 23 Avena species highlight their complex evolutionary history and extensive genomic diversity
_Published on Wed, 20 Aug 2025 00:00:00 GMT_

Abstract
--------

Common oat, belonging to the genus _Avena_ with 30 recognized species, is a nutritionally important cereal crop and high-quality forage worldwide. Here, we construct a genus-level super-pangenome of _Avena_ comprising 35 high-quality genomes from 14 cultivated oat accessions and 21 wild species. The fully resolved phylogenomic analysis unveils the origin and evolutionary scenario of _Avena_ species, and the super-pangenome analysis identifies 26.62% and 59.93% specific genes and haplotypes in wild species. We delineate the landscape of structural variations (SVs) and the transcriptome profile based 1,401 RNA-sequencing (RNA-seq) samples from diverse abiotic stress treatments in oat. We highlight the crucial role of SVs in modulating gene expression and shaping adaptation to diverse stresses. Further combining SV-based genome-wide association studies (GWASs), we characterize 13 candidate genes associated with drought resistance such as _AsARF7_, validated by transgenic oat lines. Our study provides unprecedented genomic resources to facilitate genomic, evolution and molecular breeding research in oat.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

27,99 € / 30 days

cancel any time

[Learn more](https://shop.nature.com/products/plus/?region=ROW)

Subscribe to this journal

Receive 12 print issues and online access

195,33 € per year

only 16,28 € per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02294-z?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Phylogeny, population structure and diverse phenotypes of cultivated oats and wild species in _Avena._**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_Fig1_HTML.png)

**Fig. 2: Inferred phylogeny, origin and evolution of _Avena_ species.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_Fig2_HTML.png)

**Fig. 3: Super-pangenome of _Avena_ and rich specific genetic resources in wild species.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_Fig3_HTML.png)

**Fig. 4: Characteristics of SVs among the 17 hexaploid oats.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_Fig4_HTML.png)

**Fig. 5: The impact of SVs on gene expression alteration.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_Fig5_HTML.png)

**Fig. 6: Identification and validation of key SVs and genes related to oat drought tolerance based on the SV-based GWAS.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_Fig6_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-022-04732-y/MediaObjects/41586_2022_4732_Fig1_HTML.png)

### [The mosaic oat genome gives insights into a uniquely healthy cereal crop](https://www.nature.com/articles/s41586-022-04732-y?fromPaywallRec=true)

Article Open access 18 May 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41477-024-01866-x/MediaObjects/41477_2024_1866_Fig1_HTML.png)

### [The near-complete genome assembly of hexaploid wild oat reveals its genome evolution and divergence with cultivated oats](https://www.nature.com/articles/s41477-024-01866-x?fromPaywallRec=true)

Article 03 December 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-022-01127-7/MediaObjects/41588_2022_1127_Fig1_HTML.png)

### [Reference genome assemblies reveal the origin and evolution of allohexaploid oat](https://www.nature.com/articles/s41588-022-01127-7?fromPaywallRec=true)

Article Open access 18 July 2022

Data availability
-----------------

The raw data (CRA015654 and CRA015559) and genome assemblies (GWHEROY00000000-GWHGDJH00000000.1) in this study have been deposited at the Chinese National Genomics Data Center ([https://bigd.big.ac.cn/](https://bigd.big.ac.cn/)) under the BioProject accession number [PRJCA024628](https://ngdc.cncb.ac.cn/search/all?&q=PRJCA024628). Raw sequencing data (SRR31832837–SRR31832882) and genome assemblies (JBKZJG000000000–JBKZJZ000000000 and JBKZKA000000000–JBKZKH000000000) have been deposited at the NCBI under the BioProject [PRJNA1200805](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1200805). Transcriptome sequencing data under normal growth (SRR31798633–SRR31798686 and SRR31814753–SRR31815091) have been deposited at the NCBI under the BioProject [PRJNA1201252](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1201252). Transcriptome sequencing data under different stresses (SRR31832975–SRR31833946) have been deposited at the NCBI under the BioProject [PRJNA1203203](http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1203203). Whole-genome resequencing data of 1,078 hexaploid oat accessions were obtained from the OatOmics database ([http://www.oatomics.com](http://www.oatomics.com/)). [Source data](https://www.nature.com/articles/s41588-025-02294-z#Sec46) are provided with this paper.

Code availability
-----------------

The scripts used in this study are available via GitHub ([https://github.com/HongyuZhang-HBU/oat-pangenome](https://github.com/HongyuZhang-HBU/oat-pangenome)) and Zenodo[111](https://www.nature.com/articles/s41588-025-02294-z#ref-CR111 "Zhang, H. et al. Super-pangenome analyses across 35 accessions of 23 Avena species highlight their complex evolutionary history and extensive genomic diversity. Zenodo 
https://doi.org/10.5281/zenodo.15761906
(2025).") ([https://doi.org/10.5281/zenodo.15761906](https://doi.org/10.5281/zenodo.15761906)).

References
----------

1.  Rasane, P., Jha, A., Sabikhi, L., Kumar, A. & Unnikrishnan, V. S. Nutritional advantages of oats and opportunities for its processing as value added foods — a review. _J. Food Sci. Technol._ **52**, 662–675 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25694675)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsVSns7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nutritional%20advantages%20of%20oats%20and%20opportunities%20for%20its%20processing%20as%20value%20added%20foods%20%E2%80%94%20a%20review&journal=J.%20Food%20Sci.%20Technol.&volume=52&pages=662-675&publication_year=2015&author=Rasane%2CP&author=Jha%2CA&author=Sabikhi%2CL&author=Kumar%2CA&author=Unnikrishnan%2CVS) 
    
2.  Shi, J., Tian, Z., Lai, J. & Huang, X. Plant pan-genomics and its applications. _Mol. Plant_ **16**, 168–186 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36523157)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFylu7bP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plant%20pan-genomics%20and%20its%20applications&journal=Mol.%20Plant&volume=16&pages=168-186&publication_year=2023&author=Shi%2CJ&author=Tian%2CZ&author=Lai%2CJ&author=Huang%2CX) 
    
3.  Kamal, N. et al. The mosaic oat genome gives insights into a uniquely healthy cereal crop. _Nature_ **606**, 113–119 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35585233)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9159951)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhtlajtr3I)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mosaic%20oat%20genome%20gives%20insights%20into%20a%20uniquely%20healthy%20cereal%20crop&journal=Nature&volume=606&pages=113-119&publication_year=2022&author=Kamal%2CN) 
    
4.  Peng, Y. et al. Reference genome assemblies reveal the origin and evolution of allohexaploid oat. _Nat. Genet._ **54**, 1248–1258 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35851189)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9355876)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVGmu7nJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reference%20genome%20assemblies%20reveal%20the%20origin%20and%20evolution%20of%20allohexaploid%20oat&journal=Nat.%20Genet.&volume=54&pages=1248-1258&publication_year=2022&author=Peng%2CY) 
    
5.  Liu, Q., Lin, L., Zhou, X., Peterson, P. M. & Wen, J. Unraveling the evolutionary dynamics of ancient and recent polyploidization events in _Avena_ (Poaceae). _Sci. Rep._ **7**, 41944 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28157193)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5291219)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXit1Srtb0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unraveling%20the%20evolutionary%20dynamics%20of%20ancient%20and%20recent%20polyploidization%20events%20in%20Avena%20%28Poaceae%29&journal=Sci.%20Rep.&volume=7&publication_year=2017&author=Liu%2CQ&author=Lin%2CL&author=Zhou%2CX&author=Peterson%2CPM&author=Wen%2CJ) 
    
6.  Yan, H. et al. High-density marker profiling confirms ancestral genomes of _Avena_ species and identifies D-genome chromosomes of hexaploid oat. _Theor. Appl. Genet._ **129**, 2133–2149 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27522358)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5069325)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlGmsbrF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-density%20marker%20profiling%20confirms%20ancestral%20genomes%20of%20Avena%20species%20and%20identifies%20D-genome%20chromosomes%20of%20hexaploid%20oat&journal=Theor.%20Appl.%20Genet.&volume=129&pages=2133-2149&publication_year=2016&author=Yan%2CH) 
    
7.  Loskutov, I. G., Gnutikov, A. A., Blinova, E. V. & Rodionov, A. V. The application of Vavilov’s approaches to the phylogeny and evolution of cultivated species of the genus _Avena_ L. _Vavilovskii Zhurnal Genet. Selektsii_ **27**, 921–932 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38239973)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10792501)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB1c3osFKgsA%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20application%20of%20Vavilov%E2%80%99s%20approaches%20to%20the%20phylogeny%20and%20evolution%20of%20cultivated%20species%20of%20the%20genus%20Avena%20L&journal=Vavilovskii%20Zhurnal%20Genet.%20Selektsii&volume=27&pages=921-932&publication_year=2023&author=Loskutov%2CIG&author=Gnutikov%2CAA&author=Blinova%2CEV&author=Rodionov%2CAV) 
    
8.  Fu, Y. B. Oat evolution revealed in the maternal lineages of 25 _Avena_ species. _Sci. Rep._ **8**, 4252 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29523798)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5844911)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Oat%20evolution%20revealed%20in%20the%20maternal%20lineages%20of%2025%20Avena%20species&journal=Sci.%20Rep.&volume=8&publication_year=2018&author=Fu%2CYB) 
    
9.  Liu, Q. et al. Comparative chloroplast genome analyses of _Avena_: insights into evolutionary dynamics and phylogeny. _BMC Plant Biol._ **20**, 406 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32878602)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7466839)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVSnurbO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparative%20chloroplast%20genome%20analyses%20of%20Avena%3A%20insights%20into%20evolutionary%20dynamics%20and%20phylogeny&journal=BMC%20Plant%20Biol.&volume=20&publication_year=2020&author=Liu%2CQ) 
    
10.  Morikawa, T. & Nishihara, M. Genomic and polyploid evolution in genus _Avena_ as revealed by RFLPs of repeated DNA sequences. _Genes Genet. Syst._ **84**, 199–208 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19745568)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtlyqs7nJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20and%20polyploid%20evolution%20in%20genus%20Avena%20as%20revealed%20by%20RFLPs%20of%20repeated%20DNA%20sequences&journal=Genes%20Genet.%20Syst.&volume=84&pages=199-208&publication_year=2009&author=Morikawa%2CT&author=Nishihara%2CM) 
    
11.  Yan, H. H. et al. Phylogenetic analysis of the genus _Avena_ based on chloroplast intergenic spacer _psb_A-_trn_H and single-copy nuclear gene _Acc1_. _Genome_ **57**, 267–277 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25188288)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVynurfN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phylogenetic%20analysis%20of%20the%20genus%20Avena%20based%20on%20chloroplast%20intergenic%20spacer%20psbA-trnH%20and%20single-copy%20nuclear%20gene%20Acc1&journal=Genome&volume=57&pages=267-277&publication_year=2014&author=Yan%2CHH) 
    
12.  Liu, B. B. et al. Phylogenomic conflict analyses in the apple genus _Malus_ s.l. reveal widespread hybridization and allopolyploidy driving diversification, with insights into the complex biogeographic history in the Northern Hemisphere. _J. Integr. Plant Biol_. **64**, 1020–1043 (2022).
    
13.  Meleshko, O. et al. Extensive genome-wide phylogenetic discordance is due to incomplete lineage sorting and not ongoing introgression in a rapidly radiated bryophyte genus. _Mol. Biol. Evol._ **38**, 2750–2766 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33681996)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8233498)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlCktrnK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extensive%20genome-wide%20phylogenetic%20discordance%20is%20due%20to%20incomplete%20lineage%20sorting%20and%20not%20ongoing%20introgression%20in%20a%20rapidly%20radiated%20bryophyte%20genus&journal=Mol.%20Biol.%20Evol.&volume=38&pages=2750-2766&publication_year=2021&author=Meleshko%2CO) 
    
14.  Liu, S. Y. et al. An integrative framework reveals widespread gene flow during the early radiation of oaks and relatives in Quercoideae (Fagaceae). _J. Integr. Plant Biol_. **67**, 1119–1141 (2025).
    
15.  Li, W. et al. Plant pan-genomics: recent advances, new challenges, and roads ahead. _J. Genet. Genomics_ **49**, 833–846 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35750315)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plant%20pan-genomics%3A%20recent%20advances%2C%20new%20challenges%2C%20and%20roads%20ahead&journal=J.%20Genet.%20Genomics&volume=49&pages=833-846&publication_year=2022&author=Li%2CW) 
    
16.  Tieman, D. et al. A chemical genetic roadmap to improved tomato flavor. _Science_ **355**, 391–394 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28126817)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVGrsb8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20chemical%20genetic%20roadmap%20to%20improved%20tomato%20flavor&journal=Science&volume=355&pages=391-394&publication_year=2017&author=Tieman%2CD) 
    
17.  Zhuang, Y. et al. Phylogenomics of the genus _Glycine_ sheds light on polyploid evolution and life-strategy transition. _Nat. Plants_ **8**, 233–244 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35288665)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xptlels74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phylogenomics%20of%20the%20genus%20Glycine%20sheds%20light%20on%20polyploid%20evolution%20and%20life-strategy%20transition&journal=Nat.%20Plants&volume=8&pages=233-244&publication_year=2022&author=Zhuang%2CY) 
    
18.  Yu, H. et al. A route to de novo domestication of wild allotetraploid rice. _Cell_ **184**, 1156–1170 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33539781)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsVCitLw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20route%20to%20de%20novo%20domestication%20of%20wild%20allotetraploid%20rice&journal=Cell&volume=184&pages=1156-1170&publication_year=2021&author=Yu%2CH) 
    
19.  Li, N. et al. Super-pangenome analyses highlight genomic diversity and structural variation across wild and cultivated tomato species. _Nat. Genet._ **55**, 852–860 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37024581)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10181942)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXntF2ntbw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Super-pangenome%20analyses%20highlight%20genomic%20diversity%20and%20structural%20variation%20across%20wild%20and%20cultivated%20tomato%20species&journal=Nat.%20Genet.&volume=55&pages=852-860&publication_year=2023&author=Li%2CN) 
    
20.  Tang, D. et al. Genome evolution and diversity of wild and cultivated potatoes. _Nature_ **606**, 535–541 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35676481)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9200641)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFWhu7bN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20evolution%20and%20diversity%20of%20wild%20and%20cultivated%20potatoes&journal=Nature&volume=606&pages=535-541&publication_year=2022&author=Tang%2CD) 
    
21.  Shi, T. et al. The super-pangenome of _Populus_ unveils genomic facets for its adaptation and diversification in widespread forest trees. _Mol. Plant_ **17**, 725–746 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38486452)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXnslahu7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20super-pangenome%20of%20Populus%20unveils%20genomic%20facets%20for%20its%20adaptation%20and%20diversification%20in%20widespread%20forest%20trees&journal=Mol.%20Plant&volume=17&pages=725-746&publication_year=2024&author=Shi%2CT) 
    
22.  Fominaya, A., Vega, C. & Ferrer, E. Giemsa C-banded karyotypes of _Avena_ species. _Genome_ **30**, 627–632 (1988).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Giemsa%20C-banded%20karyotypes%20of%20Avena%20species&journal=Genome&volume=30&pages=627-632&publication_year=1988&author=Fominaya%2CA&author=Vega%2CC&author=Ferrer%2CE) 
    
23.  Latta, R. G., Bekele, W. A., Wight, C. P. & Tinker, N. A. Comparative linkage mapping of diploid, tetraploid, and hexaploid _Avena_ species suggests extensive chromosome rearrangement in ancestral diploids. _Sci. Rep._ **9**, 12298 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31444367)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707241)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparative%20linkage%20mapping%20of%20diploid%2C%20tetraploid%2C%20and%20hexaploid%20Avena%20species%20suggests%20extensive%20chromosome%20rearrangement%20in%20ancestral%20diploids&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Latta%2CRG&author=Bekele%2CWA&author=Wight%2CCP&author=Tinker%2CNA) 
    
24.  Yan, H. et al. Genome size variation in the genus _Avena_. _Genome_ **59**, 209–220 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26881940)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20size%20variation%20in%20the%20genus%20Avena&journal=Genome&volume=59&pages=209-220&publication_year=2016&author=Yan%2CH) 
    
25.  Xiong, Z., Gaeta, R. T. & Pires, J. C. Homoeologous shuffling and chromosome compensation maintain genome balance in resynthesized allopolyploid _Brassica napus_. _Proc. Natl Acad. Sci. USA_ **108**, 7908–7913 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21512129)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3093481)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmsVSktbk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Homoeologous%20shuffling%20and%20chromosome%20compensation%20maintain%20genome%20balance%20in%20resynthesized%20allopolyploid%20Brassica%20napus&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=108&pages=7908-7913&publication_year=2011&author=Xiong%2CZ&author=Gaeta%2CRT&author=Pires%2CJC) 
    
26.  Chester, M. et al. Extensive chromosomal variation in a recently formed natural allopolyploid species, _Tragopogon miscellus_ (Asteraceae). _Proc. Natl Acad. Sci. USA_ **109**, 1176–1181 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22228301)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3268322)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XislWks78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extensive%20chromosomal%20variation%20in%20a%20recently%20formed%20natural%20allopolyploid%20species%2C%20Tragopogon%20miscellus%20%28Asteraceae%29&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=109&pages=1176-1181&publication_year=2012&author=Chester%2CM) 
    
27.  Peng, Y. et al. Phylogenetic relationships in the genus _Avena_ based on the nuclear _Pgk1_ gene. _PLoS ONE_ **13**, e0200047 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30408035)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6224039)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phylogenetic%20relationships%20in%20the%20genus%20Avena%20based%20on%20the%20nuclear%20Pgk1%20gene&journal=PLoS%20ONE&volume=13&publication_year=2018&author=Peng%2CY) 
    
28.  Morikawa, T., Gushiken, Y. & Tsurukawa, N. Chromosomal diversity and morphological dimorphism in Moroccan wild oat, _Avena agadiriana_. _Plant Syst. Evol._ **281**, 107–113 (2009).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosomal%20diversity%20and%20morphological%20dimorphism%20in%20Moroccan%20wild%20oat%2C%20Avena%20agadiriana&journal=Plant%20Syst.%20Evol.&volume=281&pages=107-113&publication_year=2009&author=Morikawa%2CT&author=Gushiken%2CY&author=Tsurukawa%2CN) 
    
29.  Paczos-Grzęda, E. & Bednarek, P. Comparative analysis of hexaploid _Avena_ species using REMAP and ISSR methods. _Turk. J. Bot_. **38**, 1103–1111 (2014).
    
30.  Loskutov, I. G. On evolutionary pathways of _Avena_ species. _Genet. Resour. Crop Evol._ **55**, 211–220 (2008).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=On%20evolutionary%20pathways%20of%20Avena%20species&journal=Genet.%20Resour.%20Crop%20Evol.&volume=55&pages=211-220&publication_year=2008&author=Loskutov%2CIG) 
    
31.  Badaeva, E. D., Shelukhina, O. Y., Dedkova, O. S., Loskutov, I. G. & Pukhalskyi, V. A. Comparative cytogenetic analysis of hexaploid _Avena_ L. species. _Russ. J. Genet._ **47**, 691–702 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXnsFWkurY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparative%20cytogenetic%20analysis%20of%20hexaploid%20Avena%20L.%20species&journal=Russ.%20J.%20Genet.&volume=47&pages=691-702&publication_year=2011&author=Badaeva%2CED&author=Shelukhina%2COY&author=Dedkova%2COS&author=Loskutov%2CIG&author=Pukhalskyi%2CVA) 
    
32.  Ociepa, T. The oat gene pools—review about the use of wild species in improving cultivated oat/Pule genowe owsa—przeglad informacji na temat wykorzystania dzikich gatunkow w ulepszaniu owsa uprawnego. _J. Cent. Eur. Agric._ **20**, 251–261 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20oat%20gene%20pools%E2%80%94review%20about%20the%20use%20of%20wild%20species%20in%20improving%20cultivated%20oat%2FPule%20genowe%20owsa%E2%80%94przeglad%20informacji%20na%20temat%20wykorzystania%20dzikich%20gatunkow%20w%20ulepszaniu%20owsa%20uprawnego&journal=J.%20Cent.%20Eur.%20Agric.&volume=20&pages=251-261&publication_year=2019&author=Ociepa%2CT) 
    
33.  Ladizinsky, G. & Fainstein, R. Domestication of the protein-rich tetraploid wild oats _Avena magna_ and _A. murphyi_. _Euphytica_ **26**, 221–223 (1977).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Domestication%20of%20the%20protein-rich%20tetraploid%20wild%20oats%20Avena%20magna%20and%20A.%20murphyi&journal=Euphytica&volume=26&pages=221-223&publication_year=1977&author=Ladizinsky%2CG&author=Fainstein%2CR) 
    
34.  Ladizinsky, G. Domestication via hybridization of the wild tetraploid oats _Avena magna_ and _A. murphyi_. _Theor. Appl. Genet._ **91**, 639–646 (1995).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24169892)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC2c7gvFGmtA%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Domestication%20via%20hybridization%20of%20the%20wild%20tetraploid%20oats%20Avena%20magna%20and%20A.%20murphyi&journal=Theor.%20Appl.%20Genet.&volume=91&pages=639-646&publication_year=1995&author=Ladizinsky%2CG) 
    
35.  Okoń, S., Ociepa, T., Nucia, A., Cieplak, M. & Kowalczyk, K. Is every wild species a rich source of disease resistance? _Avena fatua_ L.—potential donor of resistance to powdery mildew. _Plants_ **10**, 560 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33809685)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8002336)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Is%20every%20wild%20species%20a%20rich%20source%20of%20disease%20resistance%3F%20Avena%20fatua%20L.%E2%80%94potential%20donor%20of%20resistance%20to%20powdery%20mildew&journal=Plants&volume=10&publication_year=2021&author=Oko%C5%84%2CS&author=Ociepa%2CT&author=Nucia%2CA&author=Cieplak%2CM&author=Kowalczyk%2CK) 
    
36.  Zhai, C. et al. The isolation and characterization of _Pik_, a rice blast resistance gene which emerged after rice domestication. _New Phytol._ **189**, 321–334 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21118257)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXltlGnsw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20isolation%20and%20characterization%20of%20Pik%2C%20a%20rice%20blast%20resistance%20gene%20which%20emerged%20after%20rice%20domestication&journal=New%20Phytol.&volume=189&pages=321-334&publication_year=2011&author=Zhai%2CC) 
    
37.  Ramírez-González, R. H. et al. The transcriptional landscape of polyploid wheat. _Science_ **361**, eaar6089 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30115782)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20transcriptional%20landscape%20of%20polyploid%20wheat&journal=Science&volume=361&publication_year=2018&author=Ram%C3%ADrez-Gonz%C3%A1lez%2CRH) 
    
38.  Ma, P.-F. et al. Genome assemblies of 11 bamboo species highlight diversification induced by dynamic subgenome dominance. _Nat. Genet._ **56**, 710–720 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38491323)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11018529)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXmtFKmsrY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20assemblies%20of%2011%20bamboo%20species%20highlight%20diversification%20induced%20by%20dynamic%20subgenome%20dominance&journal=Nat.%20Genet.&volume=56&pages=710-720&publication_year=2024&author=Ma%2CP-F) 
    
39.  Qin, P. et al. Pan-genome analysis of 33 genetically diverse rice accessions reveals hidden genomic variations. _Cell_ **184**, 3542–3558 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34051138)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtr%2FM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-genome%20analysis%20of%2033%20genetically%20diverse%20rice%20accessions%20reveals%20hidden%20genomic%20variations&journal=Cell&volume=184&pages=3542-3558&publication_year=2021&author=Qin%2CP) 
    
40.  Yan, H. et al. Pangenomic analysis identifies structural variation associated with heat tolerance in pearl millet. _Nat. Genet._ **55**, 507–518 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36864101)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10011142)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXkt1yntLw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pangenomic%20analysis%20identifies%20structural%20variation%20associated%20with%20heat%20tolerance%20in%20pearl%20millet&journal=Nat.%20Genet.&volume=55&pages=507-518&publication_year=2023&author=Yan%2CH) 
    
41.  Uga, Y. et al. Control of root system architecture by _DEEPER ROOTING 1_ increases rice yield under drought conditions. _Nat. Genet._ **45**, 1097–1102 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23913002)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXht1WgsrnM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Control%20of%20root%20system%20architecture%20by%20DEEPER%20ROOTING%201%20increases%20rice%20yield%20under%20drought%20conditions&journal=Nat.%20Genet.&volume=45&pages=1097-1102&publication_year=2013&author=Uga%2CY) 
    
42.  Lo, S. F. et al. Ectopic expression of specific GA2 oxidase mutants promotes yield and stress tolerance in rice. _Plant Biotechnol. J._ **15**, 850–864 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27998028)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5466439)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVWjsrfP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ectopic%20expression%20of%20specific%20GA2%20oxidase%20mutants%20promotes%20yield%20and%20stress%20tolerance%20in%20rice&journal=Plant%20Biotechnol.%20J.&volume=15&pages=850-864&publication_year=2017&author=Lo%2CSF) 
    
43.  Wei, H., Wang, X., He, Y., Xu, H. & Wang, L. Clock component OsPRR73 positively regulates rice salt tolerance by modulating _OsHKT2;1_\-mediated sodium homeostasis. _EMBO J._ **40**, e105086 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33347628)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXktFegtg%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clock%20component%20OsPRR73%20positively%20regulates%20rice%20salt%20tolerance%20by%20modulating%20OsHKT2%3B1-mediated%20sodium%20homeostasis&journal=EMBO%20J.&volume=40&publication_year=2021&author=Wei%2CH&author=Wang%2CX&author=He%2CY&author=Xu%2CH&author=Wang%2CL) 
    
44.  Kim, J.-J. et al. Overexpression of a proton pumping gene _OVP1_ enhances salt stress tolerance, root growth and biomass yield by regulating ion balance in rice (_Oryza sativa_ L.). _Environ. Exp. Bot._ **175**, 104033 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXntFGjsrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overexpression%20of%20a%20proton%20pumping%20gene%20OVP1%20enhances%20salt%20stress%20tolerance%2C%20root%20growth%20and%20biomass%20yield%20by%20regulating%20ion%20balance%20in%20rice%20%28Oryza%20sativa%20L.%29&journal=Environ.%20Exp.%20Bot.&volume=175&publication_year=2020&author=Kim%2CJ-J) 
    
45.  An, X., Zhang, L., Luo, J., Zhao, S. & Jiao, T. Effects of oat hay content in diets on nutrient metabolism and the rumen microflora in sheep. _Animals_ **10**, 2341 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33317030)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7763615)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Effects%20of%20oat%20hay%20content%20in%20diets%20on%20nutrient%20metabolism%20and%20the%20rumen%20microflora%20in%20sheep&journal=Animals&volume=10&publication_year=2020&author=An%2CX&author=Zhang%2CL&author=Luo%2CJ&author=Zhao%2CS&author=Jiao%2CT) 
    
46.  Katsiotis, A., Hagidimitriou, M. & Heslop-Harrison, J. S. The close relationship between the A and B genomes in _Avena_ L. (Poaceae) determined by molecular cytogenetic analysis of total genomic, tandemly and dispersed repetitive DNA sequences. _Ann. Bot._ **79**, 103–109 (1997).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXhslKrsLw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20close%20relationship%20between%20the%20A%20and%20B%20genomes%20in%20Avena%20L.%20%28Poaceae%29%20determined%20by%20molecular%20cytogenetic%20analysis%20of%20total%20genomic%2C%20tandemly%20and%20dispersed%20repetitive%20DNA%20sequences&journal=Ann.%20Bot.&volume=79&pages=103-109&publication_year=1997&author=Katsiotis%2CA&author=Hagidimitriou%2CM&author=Heslop-Harrison%2CJS) 
    
47.  Zhang, C. et al. High-quality genome of a modern soybean cultivar and resequencing of 547 accessions provide insights into the role of structural variation. _Nat. Genet._ **56**, 2247–2258 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39251789)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhvFCntL%2FF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-quality%20genome%20of%20a%20modern%20soybean%20cultivar%20and%20resequencing%20of%20547%20accessions%20provide%20insights%20into%20the%20role%20of%20structural%20variation&journal=Nat.%20Genet.&volume=56&pages=2247-2258&publication_year=2024&author=Zhang%2CC) 
    
48.  Li, W. et al. Genomic insights into the divergence between hulled and hulless oats. _Cell Rep._ **44**, 116055 (2025).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=40711879)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhslSgtrjJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20insights%20into%20the%20divergence%20between%20hulled%20and%20hulless%20oats&journal=Cell%20Rep.&volume=44&publication_year=2025&author=Li%2CW) 
    
49.  He, Q. et al. The near-complete genome assembly of hexaploid wild oat reveals its genome evolution and divergence with cultivated oats. _Nat. Plants_ **10**, 2062–2078 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39627369)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXislaktr7I)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20near-complete%20genome%20assembly%20of%20hexaploid%20wild%20oat%20reveals%20its%20genome%20evolution%20and%20divergence%20with%20cultivated%20oats&journal=Nat.%20Plants&volume=10&pages=2062-2078&publication_year=2024&author=He%2CQ) 
    
50.  Li, Y. et al. Subtelomeric assembly of a multi-gene pathway for antimicrobial defense compounds in cereals. _Nat. Commun._ **12**, 2563 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33963185)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8105312)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Subtelomeric%20assembly%20of%20a%20multi-gene%20pathway%20for%20antimicrobial%20defense%20compounds%20in%20cereals&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Li%2CY) 
    
51.  Maughan, P. J. et al. Genomic insights from the first chromosome-scale assemblies of oat (_Avena_ spp.) diploid species. _BMC Biol._ **17**, 92 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31757219)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6874827)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20insights%20from%20the%20first%20chromosome-scale%20assemblies%20of%20oat%20%28Avena%20spp.%29%20diploid%20species&journal=BMC%20Biol.&volume=17&publication_year=2019&author=Maughan%2CPJ) 
    
52.  Li, H. Fast and accurate short read alignment with Burrows–Wheeler transform. _Bioinformatics_ **25**, 1754–1760 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19451168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&journal=Bioinformatics&volume=25&pages=1754-1760&publication_year=2009&author=Li%2CH) 
    
53.  Li, H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. _Bioinformatics_ **27**, 2987–2993 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21903627)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3198575)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtlGkur7L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20statistical%20framework%20for%20SNP%20calling%2C%20mutation%20discovery%2C%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data&journal=Bioinformatics&volume=27&pages=2987-2993&publication_year=2011&author=Li%2CH) 
    
54.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
55.  Alexander, D. H., Novembre, J. & Lange, K. L. Fast model-based estimation of ancestry in unrelated individuals. _Genome Res._ **19**, 1655–1664 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19648217)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752134)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFCjsLvL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&journal=Genome%20Res.&volume=19&pages=1655-1664&publication_year=2009&author=Alexander%2CDH&author=Novembre%2CJ&author=Lange%2CKL) 
    
56.  Cheng, H., Concepcion, G. T., Feng, X., Zhang, H. & Li, H. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. _Nat. Methods_ **18**, 170–175 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33526886)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7961889)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1OntL0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haplotype-resolved%20de%20novo%20assembly%20using%20phased%20assembly%20graphs%20with%20hifiasm&journal=Nat.%20Methods&volume=18&pages=170-175&publication_year=2021&author=Cheng%2CH&author=Concepcion%2CGT&author=Feng%2CX&author=Zhang%2CH&author=Li%2CH) 
    
57.  Durand, N. C. et al. Juicer provides a one-click system for analyzing loop-resolution Hi-C experiments. _Cell Syst._ **3**, 95–98 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27467249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5846465)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFKksbk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Juicer%20provides%20a%20one-click%20system%20for%20analyzing%20loop-resolution%20Hi-C%20experiments&journal=Cell%20Syst.&volume=3&pages=95-98&publication_year=2016&author=Durand%2CNC) 
    
58.  Durand, N. C. et al. Juicebox provides a visualization system for Hi-C contact maps with unlimited zoom. _Cell Syst._ **3**, 99–101 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27467250)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5596920)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFKks7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Juicebox%20provides%20a%20visualization%20system%20for%20Hi-C%20contact%20maps%20with%20unlimited%20zoom&journal=Cell%20Syst.&volume=3&pages=99-101&publication_year=2016&author=Durand%2CNC) 
    
59.  Manni, M., Berkeley, M. R., Seppey, M., Simão, F. A. & Zdobnov, E. M. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. _Mol. Biol. Evol._ **38**, 4647–4654 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34320186)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8476166)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhslaktbzF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BUSCO%20update%3A%20novel%20and%20streamlined%20workflows%20along%20with%20broader%20and%20deeper%20phylogenetic%20coverage%20for%20scoring%20of%20eukaryotic%2C%20prokaryotic%2C%20and%20viral%20genomes&journal=Mol.%20Biol.%20Evol.&volume=38&pages=4647-4654&publication_year=2021&author=Manni%2CM&author=Berkeley%2CMR&author=Seppey%2CM&author=Sim%C3%A3o%2CFA&author=Zdobnov%2CEM) 
    
60.  Ou, S. & Jiang, N. LTR\_retriever: a highly accurate and sensitive program for identification of long terminal repeat retrotransposons. _Plant Physiol._ **176**, 1410–1422 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29233850)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1CjtbzO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTR_retriever%3A%20a%20highly%20accurate%20and%20sensitive%20program%20for%20identification%20of%20long%20terminal%20repeat%20retrotransposons&journal=Plant%20Physiol.&volume=176&pages=1410-1422&publication_year=2018&author=Ou%2CS&author=Jiang%2CN) 
    
61.  Robert Hubley, A. S. _RepeatModeler Open-1.0_ [www.repeatmasker.org/RepeatModeler/](http://www.repeatmasker.org/RepeatModeler/) (2010).
    
62.  Xu, Z. & Wang, H. LTR\_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons. _Nucleic Acids Res._ **35**, W265–W268 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17485477)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933203)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTR_FINDER%3A%20an%20efficient%20tool%20for%20the%20prediction%20of%20full-length%20LTR%20retrotransposons&journal=Nucleic%20Acids%20Res.&volume=35&pages=W265-W268&publication_year=2007&author=Xu%2CZ&author=Wang%2CH) 
    
63.  Ellinghaus, D., Kurtz, S. & Willhoeft, U. LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons. _BMC Bioinformatics_ **9**, 18 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18194517)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2253517)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTRharvest%2C%20an%20efficient%20and%20flexible%20software%20for%20de%20novo%20detection%20of%20LTR%20retrotransposons&journal=BMC%20Bioinformatics&volume=9&publication_year=2008&author=Ellinghaus%2CD&author=Kurtz%2CS&author=Willhoeft%2CU) 
    
64.  Tarailo-Graovac, M. & Chen, N. Using RepeatMasker to identify repetitive elements in genomic sequences. _Curr. Protoc. Bioinformatics_ **Chapter 4**, 4.10.1–4.10.14 (2009).
    
65.  Zhang, R.-G. et al. TEsorter: an accurate and fast method to classify LTR-retrotransposons in plant genomes. _Hortic. Res._ **9**, uhac017 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35184178)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9002660)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TEsorter%3A%20an%20accurate%20and%20fast%20method%20to%20classify%20LTR-retrotransposons%20in%20plant%20genomes&journal=Hortic.%20Res.&volume=9&publication_year=2022&author=Zhang%2CR-G) 
    
66.  Haas, B. J. et al. De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. _Nat. Protoc._ **8**, 1494–1512 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23845962)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVyrsLvF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis&journal=Nat.%20Protoc.&volume=8&pages=1494-1512&publication_year=2013&author=Haas%2CBJ) 
    
67.  Haas, B. J. et al. Improving the _Arabidopsis_ genome annotation using maximal transcript alignment assemblies. _Nucleic Acids Res._ **31**, 5654–5666 (2003).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14500829)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC206470)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXns1Cntbs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&journal=Nucleic%20Acids%20Res.&volume=31&pages=5654-5666&publication_year=2003&author=Haas%2CBJ) 
    
68.  Stanke, M. & Waack, S. Gene prediction with a hidden Markov model and a new intron submodel. _Bioinformatics_ **19**, ii215–ii225 (2003).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14534192)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&journal=Bioinformatics&volume=19&pages=ii215-ii225&publication_year=2003&author=Stanke%2CM&author=Waack%2CS) 
    
69.  Gremme, G., Brendel, V., Sparks, M. E. & Kurtz, S. Engineering a software tool for gene structure prediction in higher organisms. _Inf. Softw. Technol._ **47**, 965–978 (2005).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Engineering%20a%20software%20tool%20for%20gene%20structure%20prediction%20in%20higher%20organisms&journal=Inf.%20Softw.%20Technol.&volume=47&pages=965-978&publication_year=2005&author=Gremme%2CG&author=Brendel%2CV&author=Sparks%2CME&author=Kurtz%2CS) 
    
70.  Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. _Nat. Biotechnol._ **37**, 907–915 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31375807)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7605509)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFWqtL7O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype&journal=Nat.%20Biotechnol.&volume=37&pages=907-915&publication_year=2019&author=Kim%2CD&author=Paggi%2CJM&author=Park%2CC&author=Bennett%2CC&author=Salzberg%2CSL) 
    
71.  Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. _Nat. Biotechnol._ **33**, 290–295 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25690850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643835)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXivFais70%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&journal=Nat.%20Biotechnol.&volume=33&pages=290-295&publication_year=2015&author=Pertea%2CM) 
    
72.  Haas, B. J. et al. Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments. _Genome Biol._ **9**, R7 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18190707)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2395244)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Haas%2CBJ) 
    
73.  Aramaki, T. et al. KofamKOALA: KEGG Ortholog assignment based on profile HMM and adaptive score threshold. _Bioinformatics_ **36**, 7 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KofamKOALA%3A%20KEGG%20Ortholog%20assignment%20based%20on%20profile%20HMM%20and%20adaptive%20score%20threshold&journal=Bioinformatics&volume=36&publication_year=2020&author=Aramaki%2CT) 
    
74.  Jia, K. H. et al. SubPhaser: a robust allopolyploid subgenome phasing method based on subgenome-specific _k_\-mers. _New Phytol._ **235**, 801–809 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35460274)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVeiurjM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SubPhaser%3A%20a%20robust%20allopolyploid%20subgenome%20phasing%20method%20based%20on%20subgenome-specific%20k-mers&journal=New%20Phytol.&volume=235&pages=801-809&publication_year=2022&author=Jia%2CKH) 
    
75.  Emms, D. M. & Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. _Genome Biol._ **20**, 238 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31727128)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857279)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OrthoFinder%3A%20phylogenetic%20orthology%20inference%20for%20comparative%20genomics&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Emms%2CDM&author=Kelly%2CS) 
    
76.  Edgar, R. C. MUSCLE: multiple sequence alignment with high accuracy and high throughput. _Nucleic Acids Res._ **32**, 1792–1797 (2004).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15034147)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXisF2ks7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&journal=Nucleic%20Acids%20Res.&volume=32&pages=1792-1797&publication_year=2004&author=Edgar%2CRC) 
    
77.  Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. _Bioinformatics_ **30**, 1312–1313 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24451623)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998144)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmvFCjsbc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&journal=Bioinformatics&volume=30&pages=1312-1313&publication_year=2014&author=Stamatakis%2CA) 
    
78.  Yang, Z. PAML 4: phylogenetic analysis by maximum likelihood. _Mol. Biol. Evol._ **24**, 1586–1591 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17483113)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXpsVGrs7c%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&journal=Mol.%20Biol.%20Evol.&volume=24&pages=1586-1591&publication_year=2007&author=Yang%2CZ) 
    
79.  He, Q. et al. High-quality genome of allotetraploid _Avena barbata_ provides insights into the origin and evolution of B subgenome in _Avena_. _J. Integr. Plant Biol_. **67**, 1515–1532 (2025).
    
80.  Han, M. V., Thomas, G. W., Lugo-Martinez, J. & Hahn, M. W. Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3. _Mol. Biol. Evol._ **30**, 1987–1997 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23709260)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFSrt7zF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estimating%20gene%20gain%20and%20loss%20rates%20in%20the%20presence%20of%20error%20in%20genome%20assembly%20and%20annotation%20using%20CAFE%203&journal=Mol.%20Biol.%20Evol.&volume=30&pages=1987-1997&publication_year=2013&author=Han%2CMV&author=Thomas%2CGW&author=Lugo-Martinez%2CJ&author=Hahn%2CMW) 
    
81.  Sun, P. et al. WGDI: a user-friendly toolkit for evolutionary analyses of whole-genome duplications and ancestral karyotypes. _Mol. Plant_ **15**, 1841–1851 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36307977)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivFSrsrfL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WGDI%3A%20a%20user-friendly%20toolkit%20for%20evolutionary%20analyses%20of%20whole-genome%20duplications%20and%20ancestral%20karyotypes&journal=Mol.%20Plant&volume=15&pages=1841-1851&publication_year=2022&author=Sun%2CP) 
    
82.  Zhang, R.-G. et al. Subgenome-aware analyses suggest a reticulate allopolyploidization origin in three _Papaver_ genomes. _Nat. Commun._ **14**, 2204 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37076529)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10115784)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXosVCqt78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Subgenome-aware%20analyses%20suggest%20a%20reticulate%20allopolyploidization%20origin%20in%20three%20Papaver%20genomes&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Zhang%2CR-G) 
    
83.  Doyle, J. & Egan, A. Dating the origins of polyploidy events. _New Phytol._ **186**, 73–85 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20028472)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dating%20the%20origins%20of%20polyploidy%20events&journal=New%20Phytol.&volume=186&pages=73-85&publication_year=2009&author=Doyle%2CJ&author=Egan%2CA) 
    
84.  Jin, J. J. et al. GetOrganelle: a fast and versatile toolkit for accurate de novo assembly of organelle genomes. _Genome Biol._ **21**, 241 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32912315)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7488116)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GetOrganelle%3A%20a%20fast%20and%20versatile%20toolkit%20for%20accurate%20de%20novo%20assembly%20of%20organelle%20genomes&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Jin%2CJJ) 
    
85.  Danecek, P. et al. Twelve years of SAMtools and BCFtools. _Gigascience_ **10**, giab008 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33590861)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7931819)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&journal=Gigascience&volume=10&publication_year=2021&author=Danecek%2CP) 
    
86.  Danecek, P. & McCarthy, S. BCFtools/csq: haplotype-aware variant consequences. _Bioinformatics_ **33**, 2037–2039 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28205675)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5870570)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2ntL3O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BCFtools%2Fcsq%3A%20haplotype-aware%20variant%20consequences&journal=Bioinformatics&volume=33&pages=2037-2039&publication_year=2017&author=Danecek%2CP&author=McCarthy%2CS) 
    
87.  Wang, D., Zhang, Y., Zhang, Z., Zhu, J. & Yu, J. KaKs\_Calculator 2.0: a toolkit incorporating γ-series methods and sliding window strategies. _Genomics Proteomics Bioinformatics_ **8**, 77–80 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20451164)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5054116)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXnsVOmurg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KaKs_Calculator%202.0%3A%20a%20toolkit%20incorporating%20%CE%B3-series%20methods%20and%20sliding%20window%20strategies&journal=Genomics%20Proteomics%20Bioinformatics&volume=8&pages=77-80&publication_year=2010&author=Wang%2CD&author=Zhang%2CY&author=Zhang%2CZ&author=Zhu%2CJ&author=Yu%2CJ) 
    
88.  Zhang, Y. et al. Telomere-to-telomere _Citrullus_ super-pangenome provides direction for watermelon breeding. _Nat. Genet._ **56**, 1750–1761 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38977857)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11319210)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFWhsbrF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomere-to-telomere%20Citrullus%20super-pangenome%20provides%20direction%20for%20watermelon%20breeding&journal=Nat.%20Genet.&volume=56&pages=1750-1761&publication_year=2024&author=Zhang%2CY) 
    
89.  Huang, Y. et al. Pangenome analysis provides insight into the evolution of the orange subfamily and a key gene for citric acid accumulation in citrus fruits. _Nat. Genet._ **55**, 1964–1975 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37783780)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVeqs77L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pangenome%20analysis%20provides%20insight%20into%20the%20evolution%20of%20the%20orange%20subfamily%20and%20a%20key%20gene%20for%20citric%20acid%20accumulation%20in%20citrus%20fruits&journal=Nat.%20Genet.&volume=55&pages=1964-1975&publication_year=2023&author=Huang%2CY) 
    
90.  Zhang, Z. et al. ParaAT: a parallel tool for constructing multiple protein-coding DNA alignments. _Biochem. Biophys. Res. Commun._ **419**, 779–781 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22390928)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjs1yjtL4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ParaAT%3A%20a%20parallel%20tool%20for%20constructing%20multiple%20protein-coding%20DNA%20alignments&journal=Biochem.%20Biophys.%20Res.%20Commun.&volume=419&pages=779-781&publication_year=2012&author=Zhang%2CZ) 
    
91.  Potter, S. C. et al. HMMER web server: 2018 update. _Nucleic Acids Res._ **46**, W200–W204 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29905871)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030962)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrsrc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HMMER%20web%20server%3A%202018%20update&journal=Nucleic%20Acids%20Res.&volume=46&pages=W200-W204&publication_year=2018&author=Potter%2CSC) 
    
92.  Li, P. et al. RGAugury: a pipeline for genome-wide prediction of resistance gene analogs (RGAs) in plants. _BMC Genomics_ **17**, 852 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27806688)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5093994)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitlCjt74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RGAugury%3A%20a%20pipeline%20for%20genome-wide%20prediction%20of%20resistance%20gene%20analogs%20%28RGAs%29%20in%20plants&journal=BMC%20Genomics&volume=17&publication_year=2016&author=Li%2CP) 
    
93.  Chen, S., Zhou, Y., Chen, Y. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. _Bioinformatics_ **34**, i884–i890 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30423086)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6129281)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=fastp%3A%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&journal=Bioinformatics&volume=34&pages=i884-i890&publication_year=2018&author=Chen%2CS&author=Zhou%2CY&author=Chen%2CY&author=Gu%2CJ) 
    
94.  Marçais, G. et al. MUMmer4: a fast and versatile genome alignment system. _PLoS Comput. Biol._ **14**, e1005944 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29373581)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5802927)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MUMmer4%3A%20a%20fast%20and%20versatile%20genome%20alignment%20system&journal=PLoS%20Comput.%20Biol.&volume=14&publication_year=2018&author=Mar%C3%A7ais%2CG) 
    
95.  Goel, M., Sun, H., Jiao, W. B. & Schneeberger, K. SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies. _Genome Biol._ **20**, 277 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31842948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6913012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SyRI%3A%20finding%20genomic%20rearrangements%20and%20local%20sequence%20differences%20from%20whole-genome%20assemblies&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Goel%2CM&author=Sun%2CH&author=Jiao%2CWB&author=Schneeberger%2CK) 
    
96.  Li, X. et al. Large-scale gene expression alterations introduced by structural variation drive morphotype diversification in _Brassica oleracea_. _Nat. Genet._ **56**, 517–529 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38351383)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10937405)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXjs1Slt7c%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large-scale%20gene%20expression%20alterations%20introduced%20by%20structural%20variation%20drive%20morphotype%20diversification%20in%20Brassica%20oleracea&journal=Nat.%20Genet.&volume=56&pages=517-529&publication_year=2024&author=Li%2CX) 
    
97.  Zhou, X. & Stephens, M. Genome-wide efficient mixed-model analysis for association studies. _Nat. Genet._ **44**, 821–824 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22706312)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3386377)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xos12gur0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20efficient%20mixed-model%20analysis%20for%20association%20studies&journal=Nat.%20Genet.&volume=44&pages=821-824&publication_year=2012&author=Zhou%2CX&author=Stephens%2CM) 
    
98.  Zhang, J. et al. Vacuum and co-cultivation agroinfiltration of (germinated) seeds results in tobacco rattle virus (TRV) mediated whole-plant virus-induced gene silencing (VIGS) in wheat and maize. _Front. Plant Sci._ **8**, 393 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28382049)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5360694)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Vacuum%20and%20co-cultivation%20agroinfiltration%20of%20%28germinated%29%20seeds%20results%20in%20tobacco%20rattle%20virus%20%28TRV%29%20mediated%20whole-plant%20virus-induced%20gene%20silencing%20%28VIGS%29%20in%20wheat%20and%20maize&journal=Front.%20Plant%20Sci.&volume=8&publication_year=2017&author=Zhang%2CJ) 
    
99.  Liu, N. et al. Comprehensive co-expression network reveals the fine-tuning of _AsHSFA2c_ in balancing drought tolerance and growth in oat. _Commun. Biol._ **8**, 393 (2025).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=40057657)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11890764)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXlslymtbs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20co-expression%20network%20reveals%20the%20fine-tuning%20of%20AsHSFA2c%20in%20balancing%20drought%20tolerance%20and%20growth%20in%20oat&journal=Commun.%20Biol.&volume=8&publication_year=2025&author=Liu%2CN) 
    
100.  Tian, T. et al. _Arabidopsis_ FAR-RED ELONGATED HYPOCOTYL3 integrates age and light signals to negatively regulate leaf senescence. _Plant Cell_ **32**, 1574–1588 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32152188)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7203920)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFCmurnP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Arabidopsis%20FAR-RED%20ELONGATED%20HYPOCOTYL3%20integrates%20age%20and%20light%20signals%20to%20negatively%20regulate%20leaf%20senescence&journal=Plant%20Cell&volume=32&pages=1574-1588&publication_year=2020&author=Tian%2CT) 
    
101.  Liu, N. et al. A lncRNA fine-tunes salicylic acid biosynthesis to balance plant immunity and growth. _Cell Host Microbe_ **30**, 1124–1138 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35908550)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvFyrtrbO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20lncRNA%20fine-tunes%20salicylic%20acid%20biosynthesis%20to%20balance%20plant%20immunity%20and%20growth&journal=Cell%20Host%20Microbe&volume=30&pages=1124-1138&publication_year=2022&author=Liu%2CN) 
    
102.  Zhao, X. et al. Global identification of _Arabidopsis_ lncRNAs reveals the regulation of _MAF4_ by a natural antisense RNA. _Nat. Commun._ **9**, 5056 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30498193)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6265284)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20identification%20of%20Arabidopsis%20lncRNAs%20reveals%20the%20regulation%20of%20MAF4%20by%20a%20natural%20antisense%20RNA&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Zhao%2CX) 
    
103.  Zhou, J. et al. Intronic heterochromatin prevents cryptic transcription initiation in _Arabidopsis_. _Plant J._ **101**, 1185–1197 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31647592)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlais7vI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intronic%20heterochromatin%20prevents%20cryptic%20transcription%20initiation%20in%20Arabidopsis&journal=Plant%20J.&volume=101&pages=1185-1197&publication_year=2020&author=Zhou%2CJ) 
    
104.  Li, Q. et al. DEAD-box helicases modulate dicing body formation in _Arabidopsis_. _Sci. Adv._ **7**, eabc6266 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33910901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8081359)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtl2ktL3I)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DEAD-box%20helicases%20modulate%20dicing%20body%20formation%20in%20Arabidopsis&journal=Sci.%20Adv.&volume=7&publication_year=2021&author=Li%2CQ) 
    
105.  Wang, K. et al. Author Correction: The gene _TaWOX5_ overcomes genotype dependency in wheat genetic transformation. _Nat. Plants_ **8**, 717–720 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35637359)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Author%20Correction%3A%20The%20gene%20TaWOX5%20overcomes%20genotype%20dependency%20in%20wheat%20genetic%20transformation&journal=Nat.%20Plants&volume=8&pages=717-720&publication_year=2022&author=Wang%2CK) 
    
106.  Liu, X. et al. Uncovering the transcriptional regulatory network involved in boosting wheat regeneration and transformation. _Nat. Plants_ **9**, 908–925 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37142750)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXptlChsrY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Uncovering%20the%20transcriptional%20regulatory%20network%20involved%20in%20boosting%20wheat%20regeneration%20and%20transformation&journal=Nat.%20Plants&volume=9&pages=908-925&publication_year=2023&author=Liu%2CX) 
    
107.  Yu, Y. et al. Enhancing wheat regeneration and genetic transformation through overexpression of _TaLAX1_. _Plant Commun._ **5**, 100738 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37897039)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXis1amt7fF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enhancing%20wheat%20regeneration%20and%20genetic%20transformation%20through%20overexpression%20of%20TaLAX1&journal=Plant%20Commun.&volume=5&publication_year=2024&author=Yu%2CY) 
    
108.  Ishida, Y., Tsunashima, M., Hiei, Y. & Komari, T. Wheat (_Triticum aestivum_ L.) transformation using immature embryos. _Methods Mol. Biol._ **1223**, 189–198 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25300841)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XjtVOru7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Wheat%20%28Triticum%20aestivum%20L.%29%20transformation%20using%20immature%20embryos&journal=Methods%20Mol.%20Biol.&volume=1223&pages=189-198&publication_year=2015&author=Ishida%2CY&author=Tsunashima%2CM&author=Hiei%2CY&author=Komari%2CT) 
    
109.  Said, M. et al. The _Agropyron cristatum_ karyotype, chromosome structure and cross-genome homoeology as revealed by fluorescence in situ hybridization with tandem repeats and wheat single-gene probes. _Theor. Appl. Genet._ **131**, 2213–2227 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30069594)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6154037)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVensr3L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Agropyron%20cristatum%20karyotype%2C%20chromosome%20structure%20and%20cross-genome%20homoeology%20as%20revealed%20by%20fluorescence%20in%20situ%20hybridization%20with%20tandem%20repeats%20and%20wheat%20single-gene%20probes&journal=Theor.%20Appl.%20Genet.&volume=131&pages=2213-2227&publication_year=2018&author=Said%2CM) 
    
110.  Xi, W. et al. New ND-FISH-positive oligo probes for identifying _Thinopyrum_ chromosomes in wheat backgrounds. _Int. J. Mol. Sci._ **20**, 2031 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31027154)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6515231)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpvVarsQ%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=New%20ND-FISH-positive%20oligo%20probes%20for%20identifying%20Thinopyrum%20chromosomes%20in%20wheat%20backgrounds&journal=Int.%20J.%20Mol.%20Sci.&volume=20&publication_year=2019&author=Xi%2CW) 
    
111.  Zhang, H. et al. Super-pangenome analyses across 35 accessions of 23 _Avena_ species highlight their complex evolutionary history and extensive genomic diversity. _Zenodo_ [https://doi.org/10.5281/zenodo.15761906](https://doi.org/10.5281/zenodo.15761906) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02294-z?format=refman&flavour=references)

Acknowledgements
----------------

This work was supported by the Natural Science Foundation of Hebei Province (grant nos. C2023201074 and C2021201048 to H.D.), the Young Elite Scientists Sponsorship Program by CAST (grant YESS20210080 to H.D.), the National Natural Science Foundation of China (32100500 to H.D.), open funds of the State Key Laboratory of Plant Environmental Resilience (grant no. SKLPERKF2406 to Z.G.), the Interdisciplinary Research Program of Natural Science of Hebei University (grant no. 513201422004 to H.D.) and Post-graduate’s Innovation Fund Project of Hebei University (grant no. HBU2025BS011 to H.Z.). We sincerely thank Y. Fan at Chengdu University, Y. Liu at the Dingxi Academy of Agricultural Sciences and B. Wu at the Institute of Crop Sciences, Chinese Academy of Agricultural Sciences for providing the oat accession resources. We also thank K. Wang at the Institute of Crop Sciences, Chinese Academy of Agricultural Sciences for providing the TaWOX5 plasmid.

Author information
------------------

Author notes

1.  These authors contributed equally: Hongyu Zhang, Ningkun Liu, Yaru Wang, Xinyuan Zheng, Wei Li.
    

### Authors and Affiliations

1.  College of Life Sciences, Institute of Life Science and Green Development, Hebei University, Baoding, China
    
    Hongyu Zhang, Ningkun Liu, Yaru Wang, Xinyuan Zheng, Wei Li, Ze Liu, Jianan Liu, Yu Wang, Longsheng Xing, Tao Li, Yange Yun, Qinghong Zhou, Meijia Wang, Yujie Qin, Jinjiang Yan, Zhizhong Gong, Qiang He & Huilong Du
    
2.  Hebei Basic Science Center for Biotic Interaction, Hebei University, Baoding, China
    
    Ningkun Liu, Wei Li, Qiang He & Huilong Du
    
3.  State Key Laboratory of Plant Environmental Resilience, College of Biological Sciences, China Agricultural University, Beijing, China
    
    Zhizhong Gong
    

Authors

1.  Hongyu Zhang
    
    [View author publications](https://www.nature.com/search?author=Hongyu%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hongyu%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hongyu%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Ningkun Liu
    
    [View author publications](https://www.nature.com/search?author=Ningkun%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ningkun%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ningkun%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Yaru Wang
    
    [View author publications](https://www.nature.com/search?author=Yaru%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yaru%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yaru%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Xinyuan Zheng
    
    [View author publications](https://www.nature.com/search?author=Xinyuan%20Zheng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xinyuan%20Zheng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xinyuan%20Zheng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Wei Li
    
    [View author publications](https://www.nature.com/search?author=Wei%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wei%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wei%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Ze Liu
    
    [View author publications](https://www.nature.com/search?author=Ze%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ze%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ze%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Jianan Liu
    
    [View author publications](https://www.nature.com/search?author=Jianan%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jianan%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jianan%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Yu Wang
    
    [View author publications](https://www.nature.com/search?author=Yu%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yu%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yu%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Longsheng Xing
    
    [View author publications](https://www.nature.com/search?author=Longsheng%20Xing)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Longsheng%20Xing) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Longsheng%20Xing%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Tao Li
    
    [View author publications](https://www.nature.com/search?author=Tao%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tao%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tao%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Yange Yun
    
    [View author publications](https://www.nature.com/search?author=Yange%20Yun)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yange%20Yun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yange%20Yun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Qinghong Zhou
    
    [View author publications](https://www.nature.com/search?author=Qinghong%20Zhou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qinghong%20Zhou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qinghong%20Zhou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Meijia Wang
    
    [View author publications](https://www.nature.com/search?author=Meijia%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Meijia%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Meijia%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Yujie Qin
    
    [View author publications](https://www.nature.com/search?author=Yujie%20Qin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yujie%20Qin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yujie%20Qin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Jinjiang Yan
    
    [View author publications](https://www.nature.com/search?author=Jinjiang%20Yan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jinjiang%20Yan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jinjiang%20Yan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Zhizhong Gong
    
    [View author publications](https://www.nature.com/search?author=Zhizhong%20Gong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zhizhong%20Gong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zhizhong%20Gong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Qiang He
    
    [View author publications](https://www.nature.com/search?author=Qiang%20He)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qiang%20He) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qiang%20He%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Huilong Du
    
    [View author publications](https://www.nature.com/search?author=Huilong%20Du)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Huilong%20Du) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Huilong%20Du%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

H.D. conceived and supervised the project; H.D., Q.H. and Z.G. designed the study. Yaru Wang, X.Z., Q.H., Yu Wang and H.Z. sequenced and processed the raw data. Q.H., H.Z., Yaru Wang, M.W. and X.Z. assembled and annotated the genome. H.Z., X.Z., Z.L., Q.Z. and L.X. performed phylogenetic and genome evolution analyses; Yaru Wang, Q.H. and H.Z. conducted the transcriptome analysis; H.Z., Yaru Wang, Q.H., X.Z. and J.L. conducted pangenome analyses. Q.H., Yaru Wang, T.L., W.L. and N.L. performed TE and SV analyses. W.L., J.L. and Q.H. performed the population genetics analysis. N.L., Q.H., Y.Y., T.L., Y.Q. and J.Y. conceived of and designed the experiments. H.D. composed the outline of the paper. Q.H., W.L. and H.Z. wrote the paper. H.D. revised the paper.

### Corresponding authors

Correspondence to [Zhizhong Gong](mailto:gongzz@cau.edu.cn), [Qiang He](mailto:heqiang_cqnu@163.com) or [Huilong Du](mailto:huilongdu@hbu.edu.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Mark Chapman and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02294-z#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Geographical distribution and phenotype diversity of cultivated and wild oats accessions.](https://www.nature.com/articles/s41588-025-02294-z/figures/7)

**a**, Principal component analysis (PCA) of 1,079 globally collected hexaploid oat accessions. PC1 (32.14%) clearly separates wild oat accessions (G1) and cultivated oat accessions (G2-G6); PC2 (19.45%) clearly separates G5 group of hulless oats from other groups; **b**, Geographic distribution of the 17 diverse representative accessions among 1,079 hexaploid oat accessions. The world map was generated using the WorldClim ([https://worldclim.org/](https://worldclim.org/)) whose data are freely available for academic use ([https://worldclim.org/about.html](https://worldclim.org/about.html)). The color of points corresponds to Fig. [1a](https://www.nature.com/articles/s41588-025-02294-z#Fig1). **c-e**, The highly diverse agronomic phenotypes of representative wild oat accessions, including plant architecture (**c**), spike architecture (**d**) and spikelet (**e**).

### [Extended Data Fig. 2 Circos plot of the reference genome C0191 and assembly assessment of genomes in our study.](https://www.nature.com/articles/s41588-025-02294-z/figures/8)

**a**, Circos plot showing chromosome-level features of C0191 reference genome. Tracks represent (I) chromosome length, (II) GC content, (III) gene density, (IV)TE density, (V) LTR density, (VI) DNA TE density, (VII) other TE density and (VIII) syntenic blocks. **b**, BUSCO completeness assessment for genome annotation of 35 oat accessions.

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM11)

### [Extended Data Fig. 3 Phylogenetic relationships and karyotypes evolution of _Avena_ species.](https://www.nature.com/articles/s41588-025-02294-z/figures/9)

**a**, Distribution of _Ks_ for orthologue duplicates identified between two (sub)genomes to estimate divergence time. Ad (_A. damascene_), As (_A. brevis_), D (_A. maroccana_\-D), B (_A. vaviloviana_\-B). **b**, SNP-based phylogenetic tree constructed with _A. longiglumis_ as the reference genome based on our resequencing data and published resequencing data. **c**, A specific FISH probe targeting the As subtypes can label _A. hispanica_, _A. brevis_ and _A. nuda_, but not _A. hirtula_, as observed in cells at the metaphase of mitosis. Scale bars, 5 μm. **d**, The reconstructed ancestral karyotype of the Avena genus using rice as the outgroup. Gene families with expansion and contraction in each species and node are highlighted in blue and orange, respectively. The red pentagrams represent the position of the inferred ancestor of A, B, C and D lineages. The arrows represented that the chromosome 1 and 4 in C subgenomes of tetraploids exhibited large-scale rearrangements. The red dashed boxes represent a large segment specific to the D subgenome of _A. insularis_ but absent in that of _A. maroccana_ and _A. murphyi_, then was translocated to the end of chromosome 2D during the process of polyploidization and retained in the hexaploid oats. **e**, The density of SVs in C1071, Afat, Aste, Aocc with the C0191 genome as reference. The average sequence similarity between subgenomes of C0191 and C1071, Afat, Aste, Aocc are shown in the heatmap. **f**, Proportions of contrasting gene tree topologies for 100 randomly selected single-copy genes (1,000 times) with regard to three major conflicting relationships. The values in the upper right corner of each tree show the percentage of windows that recovered the topology.

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM12)

### [Extended Data Fig. 4 The divergent genome composition between _A. agadiriana_ and AABB tetraploid species.](https://www.nature.com/articles/s41588-025-02294-z/figures/10)

**a**, Unsupervised hierarchical clustering of differential 15-mers validates that _A. vaviloviana_, _A. barbata_, _A. abyssinica_ are successfully phased into two subgenomes based on clearly distinct patterns, while the two subgenomes of _A. agadiriana_ showed higher sequence identity, making the subgenomes phasing relatively challenging and easily confused. **b**, The coalescent-based tree reconstructed from 100 randomly selected single-copy genes (1,000 times). The values in the upper right corner of each tree show the percentage of windows that recovered the topology. As (_A. hispanica_, _A. brevis_, _A. atlantica_, _A. nuda_). **c**, Intergenomic synteny among _A. agadiriana_, _A. abyssinica_, _Avaviloviana_ and _A. barbata_. Some large inversions and translocations between _A. agadiriana_ and species with AABB genome constitution were shown in blue. **d**, The average sequence similarity between the subgenomes of _A. agadiriana_, and three AABB species, as well as other subgenomes. **e**, Distribution of synonymous divergence (_Ks_) for orthologue genes identified between each subgenome of _A. agadiriana_ (Aaga-Ac’ and Aaga-As’) and other subgenomes.

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM13)

### [Extended Data Fig. 5 Refined model for the origins, polyploidizations and evolutionary trajectories of _Avena_ species.](https://www.nature.com/articles/s41588-025-02294-z/figures/11)

**a**, Phylogenetic trees of 24 oat species with _L. perenne_ and _T. aestivum_ as an outgroup based on 26 chloroplast (cp) genomes and 7,137 mitochondrial (mt) SNP data sets. **b-f**, Divergence and polyploidization time estimation based on the 2,456 single-copy orthologous groups using MCMCTree, and distribution of synonymous divergence (_Ks_) for orthologue duplicates identified between two subgenomes.

### [Extended Data Fig. 6 Genetic diversity in cultivated and wild oats.](https://www.nature.com/articles/s41588-025-02294-z/figures/12)

**a**, Venn diagram showing the numbers of common and unique gene families identified in panA, panC and panD of cultivated oats and panA, panB, panC and panD of wild oats. **b**, The distribution of CDS length in core, dispensable and private genes in cultivated and wild pangenomes. Cul-core (n = 1,460,017), Cul-dispensable (n = 394,369), Cul-private (n = 8,665), Wild-core (n = 776,159), Wild-dispensable (n = 553,189), Wild-private (n = 35,786). **c**, Proportion of genes with Pfam domains in core, dispensable, and private genes in cultivated and wild oats. Blue histograms indicate the genes with Pfam domain annotation; gray histograms indicate the gene without Pfam domain annotation. **d**, Comparison of gene expression levels in core, dispensable, and private genes in cultivated and wild oats. **e-h**, Comparison of the TE coverage with a window of 5 Mb (**e**), complete BUSCOs (n = 17) (**f**), _Ka/Ks_ (n = 10,000) (**g**), NBS-encoding gene number (n = 17) (**h**) among the three subgenomes of hexaploid oats. **i**, The number of specific gene families derived from each wild oats compared to cultivated oats. The pie chart represents the proportion of gene families unique to wild oats that are derived from different number of subgenomes. **j**, A candidate introgression region from _A. maroccana_. The sequence similarity and SNP dataset all supported that _A. maroccana_ showed close relationship with cultivated oats in this region.

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM14)

### [Extended Data Fig. 7 Verification and analysis of structural variations.](https://www.nature.com/articles/s41588-025-02294-z/figures/13)

**a**, HiFi reads were used to validate the borders of 6 deletions randomly selected in C0080 and C0298. **b**, Schematic diagrams showed the distribution of inversion between two oat genomes (x-axis: C0191). Red box represented the validated inversion events relative to C0191. **c**, Illustration of inversion identified between two oat genomes by Hi-C contact map. Chromatin interaction heatmap revealed inversion signals appearing after manual flipping. These maps supported the inversions in chromosome 7D, 3 A, 5D and 3 C. **d**, Ten insertions were randomly selected for validation by PCR amplification. The detailed information of the validation of 10 SVs were presented in Supplementary Table [17](https://www.nature.com/articles/s41588-025-02294-z#MOESM1). SV8, SV9 and SV10 came from other gels. Uncropped images are shown in Supplementary Figure 20. Different bands show the presence or absence of the SVs amplified in two oat genomes. The asterisk indicates the target band. The experiments were repeated independently three times, with similar results. **e**, Length distribution of deletions and insertions among 16 oat accessions. **f**, The haplotype of chromosome 4 A, 1 C, 2 C and 1D among C0355, C0298 and _A. sterilis_ genomes indicated the similar genotype of C0298 with _A. sterilis_. **g**, Density of SV number per 100 bp in gene bodies and 3 kb flanking regions among 16 oat accessions.

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM15)

### [Extended Data Fig. 8 The effects of SVs on the gene expression.](https://www.nature.com/articles/s41588-025-02294-z/figures/14)

**a**, Phylogenetic tree of RNA-seq samples. **b**, Principal-component analysis (PCA) for the RNA-seq samples based on the FPKM of each sample. **c**, The relative expression percentage of each subgenome based on the 1:1:1 (1:1) gene triads (pairs) in 18 oat accessions. The relative expression of different subgenomes of each accession summed to 100%. Error bar represents the SD of all RNA-seq samples of each accession. **d**, Comparison of average gene expressions between SV genes and no-SV genes among all RNA-seq samples in 16 oat accessions. **e**, The number of SV genes with suppression and promotion expression under normal growing and four different stress treatments.

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM17)

### [Extended Data Fig. 9 Gene expression analysis under different stress treatments.](https://www.nature.com/articles/s41588-025-02294-z/figures/15)

**a**, Heatmap of the expression levels of the genes under normal and different stress treatments of both leaf and root. **b**, Functional analysis of differential expression genes from four different stress treatments. **c**, RT-qPCR detection of _A105905C025039.1_ expression level in normal and drought conditions was shown in the left. Comparison of the expression level (FPKM) of _A105905C025039.1_ between the two haplotypes was shown in the right. **d-f**, RT-qPCR detection of _A105902D033372.1_, _A105904C0105377.1_ and _A105902A0179076.1_ expression level in control and different stress treatments in two tissues were shown in the left (Two-tailed Student’s _t_ test). Error bar represents the SD of three biological replicates. Comparison of the expression level (FPKM) of _A105902D033372.1_, _A105904C0105377.1_ and _A105902A0179076.1_ between the two haplotypes in two tissues under corresponding stress treatment were shown in the right (two-sided Wilcoxon rank-sum test).

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM18)

### [Extended Data Fig. 10 Functional validation of _AsARF7._](https://www.nature.com/articles/s41588-025-02294-z/figures/16)

**a**, The relationship among drought phenotype, SVs and gene expression**. b**, The expression heatmap of _AsARF7_ under ABA, drought and PEG6000 treatment. PEG600\_S represents the aerial part after PEG600 treatment. PEG600\_R represents the root part after PEG600 treatment. **c**, The expression of _AsARF7_ in different tissues including root, stem, leaf, flag-leaf, flower, fruit pod and seed. **d**, Detection of _AsARF7_ gene expression level in knockdown mutant strains. **e**, Water loss rate statistics of _AsARF7_ knockdown strains with three biological replicates. **f**, Chlorophyll content detection statistics of _AsARF7_ knockdown strains under drought treatment. **g**, Nuclear run-on assay showed that the transcription of _AsARF7_ was decreased in PEG6000 treatment plants. **h**, Genetic transformation in _A. sativa_. The up part: schematic representation of the _pUBI::AsARF7-GFP_ vector with the UBI promoter and NOS terminator; The down part: detection of GFP fluorescence signal in callus induced from mature embryos infected with the _pUBI::AsARF7-GFP_ vector (leaf). Regeneration phenotypes of mature embryos infected with the _pUBI::AsARF7-GFP_ vector. Scale bar, 1 cm (right). Error bar represents the SD of three (**c,d,g**) of five (**f**) biological replicates.

[Source data](https://www.nature.com/articles/s41588-025-02294-z#MOESM19)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM1_ESM.pdf)

Supplementary Notes 1–9 and Figs. 1–19.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM3_ESM.pdf)

### [Supplementary Tables 1–26](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM4_ESM.xlsx)

Supplementary Tables 1–26.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM5_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM7_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM8_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM9_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM12_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM13_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM14_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM15_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM16_ESM.pdf)

Unprocessed gels.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM17_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 9](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM18_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02294-z/MediaObjects/41588_2025_2294_MOESM19_ESM.xlsx)

Statistical source data.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Super-pangenome%20analyses%20across%2035%20accessions%20of%2023%20Avena%20species%20highlight%20their%20complex%20evolutionary%20history%20and%20extensive%20genomic%20diversity&author=Hongyu%20Zhang%20et%20al&contentID=10.1038%2Fs41588-025-02294-z&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-20&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02294-z)

### Cite this article

Zhang, H., Liu, N., Wang, Y. _et al._ Super-pangenome analyses across 35 accessions of 23 _Avena_ species highlight their complex evolutionary history and extensive genomic diversity. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02294-z

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02294-z?format=refman&flavour=citation)

-   Received: 31 May 2024
    
-   Accepted: 14 July 2025
    
-   Published: 20 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02294-z
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
