> id: https://www.nature.com/articles/s41588-025-02280-5

> link: https://www.nature.com/articles/s41588-025-02280-5

> title: Pangenome analysis provides insights into legume evolution and breeding

# Pangenome analysis provides insights into legume evolution and breeding
_Published on Wed, 30 Jul 2025 00:00:00 GMT_

Abstract
--------

Grain legumes hold great promise for advancing sustainable agriculture. Although the evolutionary history of legume species has been investigated, the conserved mechanisms that drive adaptive evolution and govern agronomic improvement remain elusive. Here we present high-quality genome assemblies for nine widely consumed pulses, including common bean, chickpea, pea, lentil, faba bean, pigeon pea, cowpea, mung bean and hyacinth bean. Pangenome analysis reveals the expansion of distinct gene sets in cool-season and warm-season legumes, highlighting the role of gene birth and duplication in the autoregulation of nodulation. Notably, hundreds of genes undergo convergent selection during the evolution of legumes, affecting agronomic traits such as seed weight. In addition, we demonstrate that tandem amplification of transposable elements in gene-depleted regions has a crucial role in driving genome enlargement and the formation of regulatory elements in cool-season legumes. Our results provide insights into the molecular mechanisms underlying the diversification of legumes and represent a valuable resource for facilitating legume breeding.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 12 print issues and online access

$209.00 per year

only $17.42 per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02280-5?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Pangenome analysis of legume species.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_Fig1_HTML.png)

**Fig. 2: Evolution of nodulation-related genes in legumes.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_Fig2_HTML.png)

**Fig. 3: Gene expansion and loss during legume evolution.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_Fig3_HTML.png)

**Fig. 4: Convergent selection during the evolution of legumes.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_Fig4_HTML.png)

**Fig. 5: Expansion of TEs during the evolution of legumes.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_Fig5_HTML.png)

**Fig. 6: Evolution of regulatory elements.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_Fig6_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-51032-2/MediaObjects/41467_2024_51032_Fig1_HTML.png)

### [Adaptive gene loss in the common bean pan-genome during range expansion and domestication](https://www.nature.com/articles/s41467-024-51032-2?fromPaywallRec=true)

Article Open access 07 August 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-025-02170-w/MediaObjects/41588_2025_2170_Fig1_HTML.png)

### [Pangenome analysis reveals structural variation associated with seed size and weight traits in peanut](https://www.nature.com/articles/s41588-025-02170-w?fromPaywallRec=true)

Article Open access 28 April 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-83581-7/MediaObjects/41598_2021_83581_Fig1_HTML.png)

### [Genome-wide association study identified candidate genes for seed size and seed composition improvement in _M. truncatula_](https://www.nature.com/articles/s41598-021-83581-7?fromPaywallRec=true)

Article Open access 19 February 2021

Data availability
-----------------

The raw sequencing data generated in this study are available in the National Genomics Data Center (NGDC; [https://ngdc.cncb.ac.cn/gsa/](https://ngdc.cncb.ac.cn/gsa/)) under accession PRJCA024173. Seven genome assemblies and annotations are available in the Genome Warehouse (GWH) database in NGDC ([https://ngdc.cncb.ac.cn/gwh/](https://ngdc.cncb.ac.cn/gwh/)) under accessions GWHEUBL00000000.1 (chickpea), GWHEUBM00000000.1 (cowpea), GWHEUBQ00000000.1 (hyacinth bean), GWHEUBR00000000.1 (lentil), GWHEUBS00000000.1 (mung bean), GWHEUBT00000000.1 (pea) and GWHEUBU00000000.1 (pigeon pea). The genome assemblies and annotations for common bean ([https://figshare.com/s/0fcd95bb0973ca370681](https://figshare.com/s/0fcd95bb0973ca370681)) and faba bean ([https://figshare.com/s/aa54699538c1438e515b](https://figshare.com/s/aa54699538c1438e515b)) are available in the figshare database. [Source data](https://www.nature.com/articles/s41588-025-02280-5#Sec28) are provided with this paper.

Code availability
-----------------

All data were analyzed with standard programs and open-source package/tools as detailed in [Methods](https://www.nature.com/articles/s41588-025-02280-5#Sec10).

References
----------

1.  Foyer, C. H. et al. Neglecting legumes has compromised human health and sustainable food production. _Nat. Plants_ **2**, 16112 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28221372)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neglecting%20legumes%20has%20compromised%20human%20health%20and%20sustainable%20food%20production&journal=Nat.%20Plants&volume=2&publication_year=2016&author=Foyer%2CCH) 
    
2.  Lavin, M., Herendeen, P. S. & Wojciechowski, M. F. Evolutionary rates analysis of Leguminosae implicates a rapid diversification of lineages during the tertiary. _Syst. Biol._ **54**, 575–594 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16085576)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolutionary%20rates%20analysis%20of%20Leguminosae%20implicates%20a%20rapid%20diversification%20of%20lineages%20during%20the%20tertiary&journal=Syst.%20Biol.&volume=54&pages=575-594&publication_year=2005&author=Lavin%2CM&author=Herendeen%2CPS&author=Wojciechowski%2CMF) 
    
3.  Schmutz, J. et al. Genome sequence of the palaeopolyploid soybean. _Nature_ **463**, 178–183 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20075913)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&journal=Nature&volume=463&pages=178-183&publication_year=2010&author=Schmutz%2CJ) 
    
4.  Abbo, S. et al. Reconsidering domestication of legumes versus cereals in the ancient near east. _Q. Rev. Biol._ **84**, 29–50 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19326787)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reconsidering%20domestication%20of%20legumes%20versus%20cereals%20in%20the%20ancient%20near%20east&journal=Q.%20Rev.%20Biol.&volume=84&pages=29-50&publication_year=2009&author=Abbo%2CS) 
    
5.  Alam, O. & Purugganan, M. D. Domestication and the evolution of crops: variable syndromes, complex genetic architectures, and ecological entanglements. _Plant Cell_ **36**, 1227–1241 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38243576)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11062453)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Domestication%20and%20the%20evolution%20of%20crops%3A%20variable%20syndromes%2C%20complex%20genetic%20architectures%2C%20and%20ecological%20entanglements&journal=Plant%20Cell&volume=36&pages=1227-1241&publication_year=2024&author=Alam%2CO&author=Purugganan%2CMD) 
    
6.  Schmutz, J. et al. A reference genome for common bean and genome-wide analysis of dual domestications. _Nat. Genet._ **46**, 707–713 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24908249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7048698)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20reference%20genome%20for%20common%20bean%20and%20genome-wide%20analysis%20of%20dual%20domestications&journal=Nat.%20Genet.&volume=46&pages=707-713&publication_year=2014&author=Schmutz%2CJ) 
    
7.  Wu, X. et al. Differential selection of yield and quality traits has shaped genomic signatures of cowpea domestication and improvement. _Nat. Genet._ **56**, 992–1005 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38649710)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Differential%20selection%20of%20yield%20and%20quality%20traits%20has%20shaped%20genomic%20signatures%20of%20cowpea%20domestication%20and%20improvement&journal=Nat.%20Genet.&volume=56&pages=992-1005&publication_year=2024&author=Wu%2CX) 
    
8.  Liu, C. et al. High-quality genome assembly and pan-genome studies facilitate genetic discovery in mung bean and its improvement. _Plant Commun._ **3**, 100352 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35752938)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9700124)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-quality%20genome%20assembly%20and%20pan-genome%20studies%20facilitate%20genetic%20discovery%20in%20mung%20bean%20and%20its%20improvement&journal=Plant%20Commun.&volume=3&publication_year=2022&author=Liu%2CC) 
    
9.  Njaci, I. et al. Chromosome-level genome assembly and population genomic resource to accelerate orphan crop lablab breeding. _Nat. Commun._ **14**, 1915 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37069152)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10110558)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosome-level%20genome%20assembly%20and%20population%20genomic%20resource%20to%20accelerate%20orphan%20crop%20lablab%20breeding&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Njaci%2CI) 
    
10.  Garg, V. et al. Chromosome-length genome assemblies of six legume species provide insights into genome organization, evolution, and agronomic traits for crop improvement. _J. Adv. Res._ **42**, 315–329 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36513421)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosome-length%20genome%20assemblies%20of%20six%20legume%20species%20provide%20insights%20into%20genome%20organization%2C%20evolution%2C%20and%20agronomic%20traits%20for%20crop%20improvement&journal=J.%20Adv.%20Res.&volume=42&pages=315-329&publication_year=2022&author=Garg%2CV) 
    
11.  Yang, T. et al. Improved pea reference genome and pan-genome highlight genomic features and evolutionary characteristics. _Nat. Genet._ **54**, 1553–1563 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36138232)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9534762)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improved%20pea%20reference%20genome%20and%20pan-genome%20highlight%20genomic%20features%20and%20evolutionary%20characteristics&journal=Nat.%20Genet.&volume=54&pages=1553-1563&publication_year=2022&author=Yang%2CT) 
    
12.  Jayakodi, M. et al. The giant diploid faba genome unlocks variation in a global protein crop. _Nature_ **615**, 652–659 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36890232)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10033403)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20giant%20diploid%20faba%20genome%20unlocks%20variation%20in%20a%20global%20protein%20crop&journal=Nature&volume=615&pages=652-659&publication_year=2023&author=Jayakodi%2CM) 
    
13.  Ramsay, L. et al. Genomic rearrangements have consequences for introgression breeding as revealed by genome assemblies of wild and cultivated lentil species. Preprint at _bioRxiv_ [https://doi.org/10.1101/2021.07.23.453237](https://doi.org/10.1101/2021.07.23.453237) (2021).
    
14.  Varshney, R. K. et al. Draft genome sequence of chickpea (_Cicer arietinum_) provides a resource for trait improvement. _Nat. Biotechnol._ **31**, 240–246 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23354103)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Draft%20genome%20sequence%20of%20chickpea%20%28Cicer%20arietinum%29%20provides%20a%20resource%20for%20trait%20improvement&journal=Nat.%20Biotechnol.&volume=31&pages=240-246&publication_year=2013&author=Varshney%2CRK) 
    
15.  Wang, L. et al. A telomere-to-telomere gap-free assembly of soybean genome. _Mol. Plant_ **16**, 1711–1714 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37634078)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20telomere-to-telomere%20gap-free%20assembly%20of%20soybean%20genome&journal=Mol.%20Plant&volume=16&pages=1711-1714&publication_year=2023&author=Wang%2CL) 
    
16.  Xie, M. et al. A reference-grade wild soybean genome. _Nat. Commun._ **10**, 1216 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30872580)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6418295)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20reference-grade%20wild%20soybean%20genome&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Xie%2CM) 
    
17.  Hufnagel, B. et al. High-quality genome sequence of white lupin provides insight into soil exploration and seed quality. _Nat. Commun._ **11**, 492 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31980615)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6981116)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-quality%20genome%20sequence%20of%20white%20lupin%20provides%20insight%20into%20soil%20exploration%20and%20seed%20quality&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Hufnagel%2CB) 
    
18.  Van Velzen, R., Doyle, J. J. & Geurts, R. A resurrected scenario: single gain and massive loss of nitrogen-fixing nodulation. _Trends Plant Sci._ **24**, 49–57 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30409687)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20resurrected%20scenario%3A%20single%20gain%20and%20massive%20loss%20of%20nitrogen-fixing%20nodulation&journal=Trends%20Plant%20Sci.&volume=24&pages=49-57&publication_year=2019&author=Velzen%2CR&author=Doyle%2CJJ&author=Geurts%2CR) 
    
19.  Roy, S. et al. Celebrating 20 years of genetic discoveries in legume nodulation and symbiotic nitrogen fixation. _Plant Cell_ **32**, 15–41 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31649123)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Celebrating%2020%20years%20of%20genetic%20discoveries%20in%20legume%20nodulation%20and%20symbiotic%20nitrogen%20fixation&journal=Plant%20Cell&volume=32&pages=15-41&publication_year=2020&author=Roy%2CS) 
    
20.  Markmann, K., Giczey, G. & Parniske, M. Functional adaptation of a plant receptor-kinase paved the way for the evolution of intracellular root symbioses with bacteria. _PLoS Biol._ **6**, e68 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18318603)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2270324)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20adaptation%20of%20a%20plant%20receptor-kinase%20paved%20the%20way%20for%20the%20evolution%20of%20intracellular%20root%20symbioses%20with%20bacteria&journal=PLoS%20Biol.&volume=6&publication_year=2008&author=Markmann%2CK&author=Giczey%2CG&author=Parniske%2CM) 
    
21.  Amor, B. B. et al. The NFP locus of _Medicago truncatula_ controls an early step of Nod factor signal transduction upstream of a rapid calcium flux and root hair deformation. _Plant J._ **34**, 495–506 (2003).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12753588)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20NFP%20locus%20of%20Medicago%20truncatula%20controls%20an%20early%20step%20of%20Nod%20factor%20signal%20transduction%20upstream%20of%20a%20rapid%20calcium%20flux%20and%20root%20hair%20deformation&journal=Plant%20J.&volume=34&pages=495-506&publication_year=2003&author=Amor%2CBB) 
    
22.  Smit, P. et al. Medicago LYK3, an entry receptor in rhizobial nodulation factor signaling. _Plant Physiol._ **145**, 183–191 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17586690)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1976573)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Medicago%20LYK3%2C%20an%20entry%20receptor%20in%20rhizobial%20nodulation%20factor%20signaling&journal=Plant%20Physiol.&volume=145&pages=183-191&publication_year=2007&author=Smit%2CP) 
    
23.  Catoira, R. et al. Four genes of _Medicago truncatula_ controlling components of a nod factor transduction pathway. _Plant Cell_ **12**, 1647–1666 (2000).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11006338)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC149076)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Four%20genes%20of%20Medicago%20truncatula%20controlling%20components%20of%20a%20nod%20factor%20transduction%20pathway&journal=Plant%20Cell&volume=12&pages=1647-1666&publication_year=2000&author=Catoira%2CR) 
    
24.  Horvath, B. et al. _Medicago truncatula_ IPD3 is a member of the common symbiotic signaling pathway required for rhizobial and mycorrhizal symbioses. _Mol. Plant Microbe Interact._ **24**, 1345–1358 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21692638)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Medicago%20truncatula%20IPD3%20is%20a%20member%20of%20the%20common%20symbiotic%20signaling%20pathway%20required%20for%20rhizobial%20and%20mycorrhizal%20symbioses&journal=Mol.%20Plant%20Microbe%20Interact.&volume=24&pages=1345-1358&publication_year=2011&author=Horvath%2CB) 
    
25.  Carvunis, A. R. et al. Proto-genes and de novo gene birth. _Nature_ **487**, 370–374 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22722833)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3401362)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proto-genes%20and%20de%20novo%20gene%20birth&journal=Nature&volume=487&pages=370-374&publication_year=2012&author=Carvunis%2CAR) 
    
26.  Reid, D. E., Ferguson, B. J. & Gresshoff, P. M. Inoculation- and nitrate-induced CLE peptides of soybean control NARK-dependent nodule formation. _Mol. Plant Microbe Interact._ **24**, 606–618 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21198362)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inoculation-%20and%20nitrate-induced%20CLE%20peptides%20of%20soybean%20control%20NARK-dependent%20nodule%20formation&journal=Mol.%20Plant%20Microbe%20Interact.&volume=24&pages=606-618&publication_year=2011&author=Reid%2CDE&author=Ferguson%2CBJ&author=Gresshoff%2CPM) 
    
27.  Schauser, L., Roussis, A., Stiller, J. & Stougaard, J. A plant regulator controlling development of symbiotic root nodules. _Nature_ **402**, 191–195 (1999).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10647012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20plant%20regulator%20controlling%20development%20of%20symbiotic%20root%20nodules&journal=Nature&volume=402&pages=191-195&publication_year=1999&author=Schauser%2CL&author=Roussis%2CA&author=Stiller%2CJ&author=Stougaard%2CJ) 
    
28.  Griesmann, M. et al. Phylogenomics reveals multiple losses of nitrogen-fixing root nodule symbiosis. _Science_ **361**, eaat1743 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29794220)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phylogenomics%20reveals%20multiple%20losses%20of%20nitrogen-fixing%20root%20nodule%20symbiosis&journal=Science&volume=361&publication_year=2018&author=Griesmann%2CM) 
    
29.  Ding, Y., Shi, Y. & Yang, S. Molecular regulation of plant responses to environmental temperatures. _Mol. Plant_ **13**, 544–564 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32068158)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20regulation%20of%20plant%20responses%20to%20environmental%20temperatures&journal=Mol.%20Plant&volume=13&pages=544-564&publication_year=2020&author=Ding%2CY&author=Shi%2CY&author=Yang%2CS) 
    
30.  Zhou, J. M. & Zhang, Y. Plant immunity: danger perception and signaling. _Cell_ **181**, 978–989 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32442407)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plant%20immunity%3A%20danger%20perception%20and%20signaling&journal=Cell&volume=181&pages=978-989&publication_year=2020&author=Zhou%2CJM&author=Zhang%2CY) 
    
31.  Li, Z. et al. Patterns and processes of diploidization in land plants. _Annu. Rev. Plant Biol._ **72**, 387–410 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33684297)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Patterns%20and%20processes%20of%20diploidization%20in%20land%20plants&journal=Annu.%20Rev.%20Plant%20Biol.&volume=72&pages=387-410&publication_year=2021&author=Li%2CZ) 
    
32.  Wang, L. et al. Altered chromatin architecture and gene expression during polyploidization and domestication of soybean. _Plant Cell_ **33**, 1430–1446 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33730165)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8254482)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Altered%20chromatin%20architecture%20and%20gene%20expression%20during%20polyploidization%20and%20domestication%20of%20soybean&journal=Plant%20Cell&volume=33&pages=1430-1446&publication_year=2021&author=Wang%2CL) 
    
33.  Wang, M. et al. Parallel selection on a dormancy gene during domestication of crops from multiple families. _Nat. Genet._ **50**, 1435–1441 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30250128)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Parallel%20selection%20on%20a%20dormancy%20gene%20during%20domestication%20of%20crops%20from%20multiple%20families&journal=Nat.%20Genet.&volume=50&pages=1435-1441&publication_year=2018&author=Wang%2CM) 
    
34.  Zhou, Z. et al. Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean. _Nat. Biotechnol._ **33**, 408–414 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25643055)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resequencing%20302%20wild%20and%20cultivated%20accessions%20identifies%20genes%20related%20to%20domestication%20and%20improvement%20in%20soybean&journal=Nat.%20Biotechnol.&volume=33&pages=408-414&publication_year=2015&author=Zhou%2CZ) 
    
35.  Varshney, R. K. et al. Whole-genome resequencing of 292 pigeonpea accessions identifies genomic regions associated with domestication and agronomic traits. _Nat. Genet._ **49**, 1082–1088 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28530677)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20resequencing%20of%20292%20pigeonpea%20accessions%20identifies%20genomic%20regions%20associated%20with%20domestication%20and%20agronomic%20traits&journal=Nat.%20Genet.&volume=49&pages=1082-1088&publication_year=2017&author=Varshney%2CRK) 
    
36.  Varshney, R. K. et al. A chickpea genetic variation map based on the sequencing of 3,366 genomes. _Nature_ **599**, 622–627 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34759320)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8612933)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20chickpea%20genetic%20variation%20map%20based%20on%20the%20sequencing%20of%203%2C366%20genomes&journal=Nature&volume=599&pages=622-627&publication_year=2021&author=Varshney%2CRK) 
    
37.  Li, Y. J., Yu, Y., Liu, X., Zhang, X. S. & Su, Y. H. The _Arabidopsis_ MATERNAL EFFECT EMBRYO ARREST45 protein modulates maternal auxin biosynthesis and controls seed size by inducing AINTEGUMENTA. _Plant Cell_ **33**, 1907–1926 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33730150)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8290293)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Arabidopsis%20MATERNAL%20EFFECT%20EMBRYO%20ARREST45%20protein%20modulates%20maternal%20auxin%20biosynthesis%20and%20controls%20seed%20size%20by%20inducing%20AINTEGUMENTA&journal=Plant%20Cell&volume=33&pages=1907-1926&publication_year=2021&author=Li%2CYJ&author=Yu%2CY&author=Liu%2CX&author=Zhang%2CXS&author=Su%2CYH) 
    
38.  Liu, Y. et al. SoyOmics: a deeply integrated database on soybean multi-omics. _Mol. Plant_ **16**, 794–797 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36950735)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SoyOmics%3A%20a%20deeply%20integrated%20database%20on%20soybean%20multi-omics&journal=Mol.%20Plant&volume=16&pages=794-797&publication_year=2023&author=Liu%2CY) 
    
39.  Zheng, H. et al. Construction of chromosome segment substitution lines and inheritance of seed-pod characteristics in wild soybean. _Front. Plant Sci._ **13**, 869455 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35783974)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9247457)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Construction%20of%20chromosome%20segment%20substitution%20lines%20and%20inheritance%20of%20seed-pod%20characteristics%20in%20wild%20soybean&journal=Front.%20Plant%20Sci.&volume=13&publication_year=2022&author=Zheng%2CH) 
    
40.  Wicker, T. et al. A unified classification system for eukaryotic transposable elements. _Nat. Rev. Genet._ **8**, 973–982 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17984973)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20unified%20classification%20system%20for%20eukaryotic%20transposable%20elements&journal=Nat.%20Rev.%20Genet.&volume=8&pages=973-982&publication_year=2007&author=Wicker%2CT) 
    
41.  Kreplak, J. et al. A reference genome for pea provides insight into legume genome evolution. _Nat. Genet._ **51**, 1411–1422 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31477930)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20reference%20genome%20for%20pea%20provides%20insight%20into%20legume%20genome%20evolution&journal=Nat.%20Genet.&volume=51&pages=1411-1422&publication_year=2019&author=Kreplak%2CJ) 
    
42.  International Brachypodium Initiative. Genome sequencing and analysis of the model grass _Brachypodium distachyon_. _Nature_ **463**, 763–768 (2010).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20sequencing%20and%20analysis%20of%20the%20model%20grass%20Brachypodium%20distachyon&journal=Nature&volume=463&pages=763-768&publication_year=2010) 
    
43.  Zhu, T. et al. Optical maps refine the bread wheat _Triticum aestivum_ cv. Chinese Spring genome assembly. _Plant J._ **107**, 303–314 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33893684)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8360199)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optical%20maps%20refine%20the%20bread%20wheat%20Triticum%20aestivum%20cv.%20Chinese%20Spring%20genome%20assembly&journal=Plant%20J.&volume=107&pages=303-314&publication_year=2021&author=Zhu%2CT) 
    
44.  Chuong, E. B., Elde, N. C. & Feschotte, C. Regulatory activities of transposable elements: from conflicts to benefits. _Nat. Rev. Genet._ **18**, 71–86 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27867194)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulatory%20activities%20of%20transposable%20elements%3A%20from%20conflicts%20to%20benefits&journal=Nat.%20Rev.%20Genet.&volume=18&pages=71-86&publication_year=2017&author=Chuong%2CEB&author=Elde%2CNC&author=Feschotte%2CC) 
    
45.  Noshay, J. M. et al. Assessing the regulatory potential of transposable elements using chromatin accessibility profiles of maize transposons. _Genetics_ **217**, 1–13 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33683350)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Assessing%20the%20regulatory%20potential%20of%20transposable%20elements%20using%20chromatin%20accessibility%20profiles%20of%20maize%20transposons&journal=Genetics&volume=217&pages=1-13&publication_year=2021&author=Noshay%2CJM) 
    
46.  Crisp, P. A. et al. Stable unmethylated DNA demarcates expressed genes and their _cis_\-regulatory space in plant genomes. _Proc. Natl Acad. Sci. USA_ **117**, 23991–24000 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32879011)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7519222)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stable%20unmethylated%20DNA%20demarcates%20expressed%20genes%20and%20their%20cis-regulatory%20space%20in%20plant%20genomes&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=23991-24000&publication_year=2020&author=Crisp%2CPA) 
    
47.  Chen, W. et al. Convergent selection of a WD40 protein that enhances grain yield in maize and rice. _Science_ **375**, eabg7985 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35324310)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Convergent%20selection%20of%20a%20WD40%20protein%20that%20enhances%20grain%20yield%20in%20maize%20and%20rice&journal=Science&volume=375&publication_year=2022&author=Chen%2CW) 
    
48.  McGurk, M. P. & Barbash, D. A. Double insertion of transposable elements provides a substrate for the evolution of satellite DNA. _Genome Res._ **28**, 714–725 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29588362)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5932611)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Double%20insertion%20of%20transposable%20elements%20provides%20a%20substrate%20for%20the%20evolution%20of%20satellite%20DNA&journal=Genome%20Res.&volume=28&pages=714-725&publication_year=2018&author=McGurk%2CMP&author=Barbash%2CDA) 
    
49.  Zhang, Y. et al. Evolutionary rewiring of the wheat transcriptional regulatory network by lineage-specific transposable elements. _Genome Res._ **31**, 2276–2289 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34503979)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8647832)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolutionary%20rewiring%20of%20the%20wheat%20transcriptional%20regulatory%20network%20by%20lineage-specific%20transposable%20elements&journal=Genome%20Res.&volume=31&pages=2276-2289&publication_year=2021&author=Zhang%2CY) 
    
50.  Cheng, H., Concepcion, G. T., Feng, X., Zhang, H. & Li, H. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. _Nat. Methods_ **18**, 170–175 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33526886)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7961889)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Haplotype-resolved%20de%20novo%20assembly%20using%20phased%20assembly%20graphs%20with%20hifiasm&journal=Nat.%20Methods&volume=18&pages=170-175&publication_year=2021&author=Cheng%2CH&author=Concepcion%2CGT&author=Feng%2CX&author=Zhang%2CH&author=Li%2CH) 
    
51.  Li, H. Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences. _Bioinformatics_ **32**, 2103–2110 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27153593)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4937194)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Minimap%20and%20miniasm%3A%20fast%20mapping%20and%20de%20novo%20assembly%20for%20noisy%20long%20sequences&journal=Bioinformatics&volume=32&pages=2103-2110&publication_year=2016&author=Li%2CH) 
    
52.  Alonge, M. et al. Automated assembly scaffolding using RagTag elevates a new tomato system for high-throughput genome editing. _Genome Biol._ **23**, 258 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36522651)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9753292)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automated%20assembly%20scaffolding%20using%20RagTag%20elevates%20a%20new%20tomato%20system%20for%20high-throughput%20genome%20editing&journal=Genome%20Biol.&volume=23&publication_year=2022&author=Alonge%2CM) 
    
53.  Guan, D. et al. Identifying and removing haplotypic duplication in primary genome assemblies. _Bioinformatics_ **36**, 2896–2898 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31971576)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7203741)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identifying%20and%20removing%20haplotypic%20duplication%20in%20primary%20genome%20assemblies&journal=Bioinformatics&volume=36&pages=2896-2898&publication_year=2020&author=Guan%2CD) 
    
54.  Dudchenko, O. et al. De novo assembly of the _Aedes aegypti_ genome using Hi-C yields chromosome-length scaffolds. _Science_ **356**, 92–95 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28336562)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635820)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20assembly%20of%20the%20Aedes%20aegypti%20genome%20using%20Hi-C%20yields%20chromosome-length%20scaffolds&journal=Science&volume=356&pages=92-95&publication_year=2017&author=Dudchenko%2CO) 
    
55.  Durand, N. C. et al. Juicer provides a one-click system for analyzing loop-resolution Hi-C experiments. _Cell Syst._ **3**, 95–98 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27467249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5846465)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Juicer%20provides%20a%20one-click%20system%20for%20analyzing%20loop-resolution%20Hi-C%20experiments&journal=Cell%20Syst.&volume=3&pages=95-98&publication_year=2016&author=Durand%2CNC) 
    
56.  Durand, N. C. et al. Juicebox provides a visualization system for Hi-C contact maps with unlimited zoom. _Cell Syst._ **3**, 99–101 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27467250)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5596920)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Juicebox%20provides%20a%20visualization%20system%20for%20Hi-C%20contact%20maps%20with%20unlimited%20zoom&journal=Cell%20Syst.&volume=3&pages=99-101&publication_year=2016&author=Durand%2CNC) 
    
57.  Ellinghaus, D., Kurtz, S. & Willhoeft, U. LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons. _BMC Bioinformatics_ **9**, 18 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18194517)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2253517)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTRharvest%2C%20an%20efficient%20and%20flexible%20software%20for%20de%20novo%20detection%20of%20LTR%20retrotransposons&journal=BMC%20Bioinformatics&volume=9&publication_year=2008&author=Ellinghaus%2CD&author=Kurtz%2CS&author=Willhoeft%2CU) 
    
58.  Ou, S. & Jiang, N. LTR\_FINDER\_parallel: parallelization of LTR\_FINDER enabling rapid identification of long terminal repeat retrotransposons. _Mob. DNA_ **10**, 48 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31857828)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6909508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTR_FINDER_parallel%3A%20parallelization%20of%20LTR_FINDER%20enabling%20rapid%20identification%20of%20long%20terminal%20repeat%20retrotransposons&journal=Mob.%20DNA&volume=10&publication_year=2019&author=Ou%2CS&author=Jiang%2CN) 
    
59.  Ou, S. & Jiang, N. LTR\_retriever: a highly accurate and sensitive program for identification of long terminal repeat retrotransposons. _Plant Physiol._ **176**, 1410–1422 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29233850)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LTR_retriever%3A%20a%20highly%20accurate%20and%20sensitive%20program%20for%20identification%20of%20long%20terminal%20repeat%20retrotransposons&journal=Plant%20Physiol.&volume=176&pages=1410-1422&publication_year=2018&author=Ou%2CS&author=Jiang%2CN) 
    
60.  Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. _Proc. Natl Acad. Sci. USA_ **117**, 9451–9457 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32300014)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196820)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=9451-9457&publication_year=2020&author=Flynn%2CJM) 
    
61.  Yan, H., Bombarely, A. & Li, S. DeepTE: a computational method for de novo classification of transposons with convolutional neural network. _Bioinformatics_ **36**, 4269–4275 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32415954)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DeepTE%3A%20a%20computational%20method%20for%20de%20novo%20classification%20of%20transposons%20with%20convolutional%20neural%20network&journal=Bioinformatics&volume=36&pages=4269-4275&publication_year=2020&author=Yan%2CH&author=Bombarely%2CA&author=Li%2CS) 
    
62.  Bao, W., Kojima, K. K. & Kohany, O. Repbase Update, a database of repetitive elements in eukaryotic genomes. _Mob. DNA_ **6**, 11 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26045719)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4455052)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Repbase%20Update%2C%20a%20database%20of%20repetitive%20elements%20in%20eukaryotic%20genomes&journal=Mob.%20DNA&volume=6&publication_year=2015&author=Bao%2CW&author=Kojima%2CKK&author=Kohany%2CO) 
    
63.  Kim, D., Langmead, B. & Salzberg, S. L. HISAT: a fast spliced aligner with low memory requirements. _Nat. Methods_ **12**, 357–360 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25751142)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655817)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HISAT%3A%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&journal=Nat.%20Methods&volume=12&pages=357-360&publication_year=2015&author=Kim%2CD&author=Langmead%2CB&author=Salzberg%2CSL) 
    
64.  Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. _Nat. Biotechnol._ **33**, 290–295 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25690850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643835)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&journal=Nat.%20Biotechnol.&volume=33&pages=290-295&publication_year=2015&author=Pertea%2CM) 
    
65.  Grabherr, M. G. et al. Full-length transcriptome assembly from RNA-seq data without a reference genome. _Nat. Biotechnol._ **29**, 644–652 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21572440)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3571712)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Full-length%20transcriptome%20assembly%20from%20RNA-seq%20data%20without%20a%20reference%20genome&journal=Nat.%20Biotechnol.&volume=29&pages=644-652&publication_year=2011&author=Grabherr%2CMG) 
    
66.  Haas, B. J. et al. Improving the _Arabidopsis_ genome annotation using maximal transcript alignment assemblies. _Nucleic Acids Res._ **31**, 5654–5666 (2003).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14500829)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC206470)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&journal=Nucleic%20Acids%20Res.&volume=31&pages=5654-5666&publication_year=2003&author=Haas%2CBJ) 
    
67.  Keilwagen, J., Hartung, F., Paulini, M., Twardziok, S. O. & Grau, J. Combining RNA-seq data and homology-based gene prediction for plants, animals and fungi. _BMC Bioinformatics_ **19**, 189 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29843602)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5975413)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combining%20RNA-seq%20data%20and%20homology-based%20gene%20prediction%20for%20plants%2C%20animals%20and%20fungi&journal=BMC%20Bioinformatics&volume=19&publication_year=2018&author=Keilwagen%2CJ&author=Hartung%2CF&author=Paulini%2CM&author=Twardziok%2CSO&author=Grau%2CJ) 
    
68.  Zhuang, Y. et al. Phylogenomics of the genus _Glycine_ sheds light on polyploid evolution and life-strategy transition. _Nat. Plants_ **8**, 233–244 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35288665)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phylogenomics%20of%20the%20genus%20Glycine%20sheds%20light%20on%20polyploid%20evolution%20and%20life-strategy%20transition&journal=Nat.%20Plants&volume=8&pages=233-244&publication_year=2022&author=Zhuang%2CY) 
    
69.  Lonardi, S. et al. The genome of cowpea (_Vigna unguiculata_ (L.) Walp.). _Plant J._ **98**, 767–782 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31017340)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6852540)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genome%20of%20cowpea%20%28Vigna%20unguiculata%20%28L.%29%20Walp.%29&journal=Plant%20J.&volume=98&pages=767-782&publication_year=2019&author=Lonardi%2CS) 
    
70.  Kang, Y. J. et al. Genome sequence of mungbean and insights into evolution within _Vigna_ species. _Nat. Commun._ **5**, 5443 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25384727)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20sequence%20of%20mungbean%20and%20insights%20into%20evolution%20within%20Vigna%20species&journal=Nat.%20Commun.&volume=5&publication_year=2014&author=Kang%2CYJ) 
    
71.  Varshney, R. K. et al. Draft genome sequence of pigeonpea (_Cajanus cajan_), an orphan legume crop of resource-poor farmers. _Nat. Biotechnol._ **30**, 83–89 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22057054)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Draft%20genome%20sequence%20of%20pigeonpea%20%28Cajanus%20cajan%29%2C%20an%20orphan%20legume%20crop%20of%20resource-poor%20farmers&journal=Nat.%20Biotechnol.&volume=30&pages=83-89&publication_year=2011&author=Varshney%2CRK) 
    
72.  Young, N. D. et al. The Medicago genome provides insight into the evolution of rhizobial symbioses. _Nature_ **480**, 520–524 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22089132)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272368)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Medicago%20genome%20provides%20insight%20into%20the%20evolution%20of%20rhizobial%20symbioses&journal=Nature&volume=480&pages=520-524&publication_year=2011&author=Young%2CND) 
    
73.  Ouyang, S. et al. The TIGR rice genome annotation resource: improvements and new features. _Nucleic Acids Res._ **35**, D883–7 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17145706)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20TIGR%20rice%20genome%20annotation%20resource%3A%20improvements%20and%20new%20features&journal=Nucleic%20Acids%20Res.&volume=35&pages=D883-7&publication_year=2007&author=Ouyang%2CS) 
    
74.  Lamesch, P. et al. The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools. _Nucleic Acids Res._ **40**, D1202–10 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22140109)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Arabidopsis%20Information%20Resource%20%28TAIR%29%3A%20improved%20gene%20annotation%20and%20new%20tools&journal=Nucleic%20Acids%20Res.&volume=40&pages=D1202-10&publication_year=2012&author=Lamesch%2CP) 
    
75.  Stanke, M. et al. AUGUSTUS: ab initio prediction of alternative transcripts. _Nucleic Acids Res._ **34**, W435–W439 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16845043)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1538822)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&journal=Nucleic%20Acids%20Res.&volume=34&pages=W435-W439&publication_year=2006&author=Stanke%2CM) 
    
76.  Haas, B. J. et al. Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments. _Genome Biol._ **9**, R7 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18190707)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2395244)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Haas%2CBJ) 
    
77.  Manni, M., Berkeley, M. R., Seppey, M., Simao, F. A. & Zdobnov, E. M. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. _Mol. Biol. Evol._ **38**, 4647–4654 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34320186)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8476166)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BUSCO%20update%3A%20novel%20and%20streamlined%20workflows%20along%20with%20broader%20and%20deeper%20phylogenetic%20coverage%20for%20scoring%20of%20eukaryotic%2C%20prokaryotic%2C%20and%20viral%20genomes&journal=Mol.%20Biol.%20Evol.&volume=38&pages=4647-4654&publication_year=2021&author=Manni%2CM&author=Berkeley%2CMR&author=Seppey%2CM&author=Simao%2CFA&author=Zdobnov%2CEM) 
    
78.  Jones, P. et al. InterProScan 5: genome-scale protein function classification. _Bioinformatics_ **30**, 1236–1240 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24451626)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998142)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&journal=Bioinformatics&volume=30&pages=1236-1240&publication_year=2014&author=Jones%2CP) 
    
79.  Emms, D. M. & Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. _Genome Biol._ **20**, 238 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31727128)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857279)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OrthoFinder%3A%20phylogenetic%20orthology%20inference%20for%20comparative%20genomics&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Emms%2CDM&author=Kelly%2CS) 
    
80.  Liu, Y. et al. Pan-genome of wild and cultivated soybeans. _Cell_ **182**, 162–176 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32553274)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-genome%20of%20wild%20and%20cultivated%20soybeans&journal=Cell&volume=182&pages=162-176&publication_year=2020&author=Liu%2CY) 
    
81.  Sun, P. et al. WGDI: a user-friendly toolkit for evolutionary analyses of whole-genome duplications and ancestral karyotypes. _Mol. Plant_ **15**, 1841–1851 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36307977)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WGDI%3A%20a%20user-friendly%20toolkit%20for%20evolutionary%20analyses%20of%20whole-genome%20duplications%20and%20ancestral%20karyotypes&journal=Mol.%20Plant&volume=15&pages=1841-1851&publication_year=2022&author=Sun%2CP) 
    
82.  Nguyen, L. T., Schmidt, H. A., von Haeseler, A. & Minh, B. Q. IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. _Mol. Biol. Evol._ **32**, 268–274 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25371430)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&journal=Mol.%20Biol.%20Evol.&volume=32&pages=268-274&publication_year=2015&author=Nguyen%2CLT&author=Schmidt%2CHA&author=Haeseler%2CA&author=Minh%2CBQ) 
    
83.  Yang, Z. PAML 4: phylogenetic analysis by maximum likelihood. _Mol. Biol. Evol._ **24**, 1586–1591 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17483113)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&journal=Mol.%20Biol.%20Evol.&volume=24&pages=1586-1591&publication_year=2007&author=Yang%2CZ) 
    
84.  Kumar, S. et al. TimeTree 5: an expanded resource for species divergence times. _Mol. Biol. Evol._ **39**, msac174 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35932227)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9400175)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TimeTree%205%3A%20an%20expanded%20resource%20for%20species%20divergence%20times&journal=Mol.%20Biol.%20Evol.&volume=39&publication_year=2022&author=Kumar%2CS) 
    
85.  Mendes, F. K., Vanderpool, D., Fulton, B. & Hahn, M. W. CAFE 5 models variation in evolutionary rates among gene families. _Bioinformatics_ **36**, 5516–5518 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33325502)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CAFE%205%20models%20variation%20in%20evolutionary%20rates%20among%20gene%20families&journal=Bioinformatics&volume=36&pages=5516-5518&publication_year=2021&author=Mendes%2CFK&author=Vanderpool%2CD&author=Fulton%2CB&author=Hahn%2CMW) 
    
86.  Letunic, I. & Bork, P. Interactive Tree of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. _Nucleic Acids Res._ **49**, W293–W296 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33885785)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8265157)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20of%20Life%20%28iTOL%29%20v5%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&journal=Nucleic%20Acids%20Res.&volume=49&pages=W293-W296&publication_year=2021&author=Letunic%2CI&author=Bork%2CP) 
    
87.  Niyikiza, D. et al. Interactions of gene expression, alternative splicing, and DNA methylation in determining nodule identity. _Plant J._ **103**, 1744–1766 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32491251)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactions%20of%20gene%20expression%2C%20alternative%20splicing%2C%20and%20DNA%20methylation%20in%20determining%20nodule%20identity&journal=Plant%20J.&volume=103&pages=1744-1766&publication_year=2020&author=Niyikiza%2CD) 
    
88.  Xue, Y. et al. Genome wide transcriptome analysis reveals complex regulatory mechanisms underlying phosphate homeostasis in soybean nodules. _Int. J. Mol. Sci._ **19**, 2924 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30261621)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6213598)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20wide%20transcriptome%20analysis%20reveals%20complex%20regulatory%20mechanisms%20underlying%20phosphate%20homeostasis%20in%20soybean%20nodules&journal=Int.%20J.%20Mol.%20Sci.&volume=19&publication_year=2018&author=Xue%2CY) 
    
89.  Shen, Y. et al. Global dissection of alternative splicing in paleopolyploid soybean. _Plant Cell_ **26**, 996–1008 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24681622)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001406)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20dissection%20of%20alternative%20splicing%20in%20paleopolyploid%20soybean&journal=Plant%20Cell&volume=26&pages=996-1008&publication_year=2014&author=Shen%2CY) 
    
90.  Adhikari, S., Damodaran, S. & Subramanian, S. Lateral root and nodule transcriptomes of soybean. _Data_ **4**, 64 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lateral%20root%20and%20nodule%20transcriptomes%20of%20soybean&journal=Data&volume=4&publication_year=2019&author=Adhikari%2CS&author=Damodaran%2CS&author=Subramanian%2CS) 
    
91.  Stein, J. C. et al. Genomes of 13 domesticated and wild rice relatives highlight genetic conservation, turnover and innovation across the genus _Oryza_. _Nat. Genet._ **50**, 285–296 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29358651)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomes%20of%2013%20domesticated%20and%20wild%20rice%20relatives%20highlight%20genetic%20conservation%2C%20turnover%20and%20innovation%20across%20the%20genus%20Oryza&journal=Nat.%20Genet.&volume=50&pages=285-296&publication_year=2018&author=Stein%2CJC) 
    
92.  Mount, D. W. Using the Basic Local Alignment Search Tool (BLAST). _CSH Protoc._ **2007**, pdb top17 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21357135)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Using%20the%20Basic%20Local%20Alignment%20Search%20Tool%20%28BLAST%29&journal=CSH%20Protoc.&volume=2007&publication_year=2007&author=Mount%2CDW) 
    
93.  Ngou, B. P. M., Heal, R., Wyler, M., Schmid, M. W. & Jones, J. D. G. Concerted expansion and contraction of immune receptor gene repertoires in plant genomes. _Nat. Plants_ **8**, 1146–1152 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36241733)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9579050)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Concerted%20expansion%20and%20contraction%20of%20immune%20receptor%20gene%20repertoires%20in%20plant%20genomes&journal=Nat.%20Plants&volume=8&pages=1146-1152&publication_year=2022&author=Ngou%2CBPM&author=Heal%2CR&author=Wyler%2CM&author=Schmid%2CMW&author=Jones%2CJDG) 
    
94.  Van de Weyer, A. L. et al. A species-wide inventory of NLR genes and alleles in _Arabidopsis thaliana_. _Cell_ **178**, 1260–1272 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31442410)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6709784)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20species-wide%20inventory%20of%20NLR%20genes%20and%20alleles%20in%20Arabidopsis%20thaliana&journal=Cell&volume=178&pages=1260-1272&publication_year=2019&author=Weyer%2CAL) 
    
95.  Wang, Y. et al. MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity. _Nucleic Acids Res._ **40**, e49 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22217600)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3326336)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&journal=Nucleic%20Acids%20Res.&volume=40&publication_year=2012&author=Wang%2CY) 
    
96.  Li, H. & Durbin, R. Fast and accurate long-read alignment with Burrows–Wheeler transform. _Bioinformatics_ **26**, 589–595 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20080505)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2828108)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&journal=Bioinformatics&volume=26&pages=589-595&publication_year=2010&author=Li%2CH&author=Durbin%2CR) 
    
97.  McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. _Genome Res._ **20**, 1297–1303 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20644199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&journal=Genome%20Res.&volume=20&pages=1297-1303&publication_year=2010&author=McKenna%2CA) 
    
98.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
99.  Danecek, P. et al. The variant call format and VCFtools. _Bioinformatics_ **27**, 2156–2158 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21653522)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137218)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20variant%20call%20format%20and%20VCFtools&journal=Bioinformatics&volume=27&pages=2156-2158&publication_year=2011&author=Danecek%2CP) 
    
100.  Krueger, F. & Andrews, S. R. Bismark: a flexible aligner and methylation caller for Bisulfite-seq applications. _Bioinformatics_ **27**, 1571–1572 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21493656)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102221)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bismark%3A%20a%20flexible%20aligner%20and%20methylation%20caller%20for%20Bisulfite-seq%20applications&journal=Bioinformatics&volume=27&pages=1571-1572&publication_year=2011&author=Krueger%2CF&author=Andrews%2CSR) 
    
101.  Langmead, B. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. _Nat. Methods_ **9**, 357–359 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22388286)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&journal=Nat.%20Methods&volume=9&pages=357-359&publication_year=2012&author=Langmead%2CB&author=Salzberg%2CSL) 
    
102.  Zhang, Y. et al. Model-based analysis of ChIP–seq (MACS). _Genome Biol._ **9**, R137 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18798982)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Model-based%20analysis%20of%20ChIP%E2%80%93seq%20%28MACS%29&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Zhang%2CY) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02280-5?format=refman&flavour=references)

Acknowledgements
----------------

We thank the Bioinformatics Center at Nanjing Agricultural University for data analysis and H. Chen from Jiangsu Academy of Agricultural Sciences for providing materials. This work was supported by the Biological Breeding-National Science and Technology Major Project (2023ZD04073), the National Natural Science Foundation of China (32372126 and 32300494), the China National Postdoctoral Program for Innovative Talents (BX2021132) and the China Postdoctoral Science Foundation (2022M721657).

Author information
------------------

### Authors and Affiliations

1.  State Key Laboratory of Crop Genetics & Germplasm Enhancement and Utilization, Jiangsu Collaborative Innovation Center for Modern Crop Production, Nanjing Agricultural University, Nanjing, China
    
    Longfei Wang, Xinyu Jiang, Wu Jiao, Junrong Mao, Wenxue Ye & Qingxin Song
    
2.  National Key Laboratory of Agricultural Microbiology, Hubei Hongshan Laboratory, College of Life Science and Technology, Huazhong Agricultural University, Wuhan, China
    
    Yangrong Cao
    
3.  Key Laboratory of Soybean Biology in Chinese Ministry of Education, College of Agriculture, Northeast Agricultural University, Harbin, China
    
    Qingshan Chen
    

Authors

1.  Longfei Wang
    
    [View author publications](https://www.nature.com/search?author=Longfei%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Longfei%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Longfei%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Xinyu Jiang
    
    [View author publications](https://www.nature.com/search?author=Xinyu%20Jiang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xinyu%20Jiang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xinyu%20Jiang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Wu Jiao
    
    [View author publications](https://www.nature.com/search?author=Wu%20Jiao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wu%20Jiao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wu%20Jiao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Junrong Mao
    
    [View author publications](https://www.nature.com/search?author=Junrong%20Mao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Junrong%20Mao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Junrong%20Mao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Wenxue Ye
    
    [View author publications](https://www.nature.com/search?author=Wenxue%20Ye)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenxue%20Ye) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenxue%20Ye%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Yangrong Cao
    
    [View author publications](https://www.nature.com/search?author=Yangrong%20Cao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yangrong%20Cao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yangrong%20Cao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Qingshan Chen
    
    [View author publications](https://www.nature.com/search?author=Qingshan%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qingshan%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qingshan%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Qingxin Song
    
    [View author publications](https://www.nature.com/search?author=Qingxin%20Song)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qingxin%20Song) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qingxin%20Song%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Q.S. conceived of and designed the research. L.W., J.M. and W.Y. performed the experiments. L.W., X.J., W.J. and Y.C. analyzed the data. Q.C. provided materials and reagents. L.W. and Q.S. wrote the paper. All authors read and approved the final paper.

### Corresponding author

Correspondence to [Qingxin Song](mailto:qxsong@njau.edu.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02280-5#MOESM2) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Genome-wide Hi-C contact maps of legumes.](https://www.nature.com/articles/s41588-025-02280-5/figures/7)

Hi-C interaction matrices throughout chromosomes in different legume species. The color intensity represents the frequency of contact.

### [Extended Data Fig. 2 Comparison of genomes and annotations between assemblies in this study and published assemblies.](https://www.nature.com/articles/s41588-025-02280-5/figures/8)

**a**–**h**, Collinearity of chromosomes between assemblies in this study and published assemblies. **i**, Fraction of genes with protein domains among the newly annotated genes.

### [Extended Data Fig. 3 Characters of core, softcore, shell and cloud genes.](https://www.nature.com/articles/s41588-025-02280-5/figures/9)

**a**, Percentage of core, softcore, shell and cloud genes in each species. **b**, Nonsynonymous/synonymous substitution ratios (_K_a/Ks) within core, softcore and shell genes. Different letters above each column indicate statistical significance at _P_ < 0.05. Numbers represent the sample size. Significance was tested using one-way ANOVA test and multiple comparisons were analyzed using Scheffe’s test. The upper and lower edges of the boxes represent the 75% and 25% quartiles, and the central line denotes the median and the whiskers extend to 1.5× IQR. **c**, Proportion of core, softcore, shell and cloud genes with (w) or without (w/o) annotated protein domains. **d**, Fraction of core, softcore, shell and cloud genes in heterochromatin (Hete) and euchromatin (Eu). **e**–**i**, Distributions of DNA methylation (**e**,**f**), chromatin accessibility (**g**) and histone modifications (**h**,**i**) around core, softcore, shell and cloud genes. **j**, Expression profiles of cloud genes in multiple tissues at different developmental stages. log10(FPKM + 0.001) values were used to plot the heatmap. Column 1, root; 2–3, cotyledon; 4–5, stem; 6, shoot; 7–9, leaf; 10–12, leaf bud; 13–17, flower; 18–20, pod; 21–23, pod and seed; 24–28, seed. **k**,**l**, GO analysis of core (**k**) and dispensable (**l**) genes.

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM10)

### [Extended Data Fig. 4 Evolution of annotated nodulation-related genes.](https://www.nature.com/articles/s41588-025-02280-5/figures/10)

**a**, GO analysis of highly-expressed core (left panel) and dispensable (right panel) genes in nodules. **b**, Percentage of core, softcore, shell and cloud genes in annotated nodulation-related genes. **c**, Fraction of annotated nodulation-related genes in different age ranks. **d**, The presence or absence of common symbiosis signaling components for root-nodule and arbuscular mycorrhiza symbiosis in land plants. **e**, Percentage of nodule highly expressed and cloned nodulation-related genes in early land plants. **f**, A maximum likelihood phylogenetic tree of legumes with 11 outgroups.

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM11)

### [Extended Data Fig. 5 Gene expansion and loss during legume evolution.](https://www.nature.com/articles/s41588-025-02280-5/figures/11)

**a**, Expansion and contraction of gene families in legumes. Numbers in parentheses indicate gene family counts subject to expansion (red) and contraction (blue). **b**, Copy numbers of _CBF_ genes in legumes. **c**–**f**, Percentage of core, softcore, shell and cloud genes in whole-genome duplicated (WGD), small-scale duplicated (SSD) and single-copy (singleton) genes.

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM12)

### [Extended Data Fig. 6 Selection of _YUC4_ during the evolution of legumes.](https://www.nature.com/articles/s41588-025-02280-5/figures/12)

**a**, _π_ ratios between wild species and landraces across 600 kb flanking genomic region of the _YUC4_. _π_ ratios were calculated using 20-kb sliding windows with 2-kb steps. **b**, _F_ST between wild species and landraces across 600 kb genomic regions of _YUC4_ in soybean, pigeon pea, chickpea and pea. _F_ST values were calculated using 100-kb sliding windows with 10-kb steps. **c**–**e**, Proportion of three haplotypes of _GmYUC4a_ in wild soybeans, landraces and cultivars. **f**, Relative expression levels of _GmYUC4a_ in accessions with Hap1 or Hap2 (n = 5). The _P_ value was determined by two-tailed Student’s _t_ test. **g**, Haplotype analysis of _GmYUC4a_ in a Chinese cultivar Suinong 14 (SN14), a wild soybean ZYD00006 (ZYD) and two chromosome segment substitution lines (R3 and R170).

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM13)

### [Extended Data Fig. 7 A maximum likelihood phylogenetic tree of _YUCCA_ genes in soybean, pigeon pea, chickpea, pea and _Arabidopsis_.](https://www.nature.com/articles/s41588-025-02280-5/figures/13)

Red stars indicate genes that underwent convergent selection. _GmYUC4a, GmW82.14G124100_; _PsYUC__4_, _Pissa.06G068100_; _CaYUC__4_, _Cicar.07G377300_; _CcYUC4_, _Cajca.01G068800_. _YUC1–11_ indicate orthologs of _YUCCAs_ in _Arabidopsis_.

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM14)

### [Extended Data Fig. 8 TE expansion and genome enlargement.](https://www.nature.com/articles/s41588-025-02280-5/figures/14)

**a**, Distributions of TEs throughout chromosomes. _Gm_, _Glycine max_; _Gs_, _Glycine soja_; _Pv_, _Phaseolus vulgaris_; _Vu_, _Vigna unguiculata_; _Vr_, _Vigna radiata_; _Lp_, _Lablab purpureus_; _Cc_, _Cajanus cajan_. **b**, Fraction of TEs in heterochromatic regions. **c**–**f**, Number of TEs in top 10 families (copy numbers) in _C. arietinum_ (**c**), _P. sativum_ (**d**)_, L. culinaris_ (**e**) and _V. faba_ (**f**). **g**,**h**, Correlations between F01 TE amplification and intergenic region increase from _C. arietinum_ to _P. sativum_ (**g**) or from _L. culinaris_ to _V. faba_ (**h**). Pearson correlation coefficient and _P_ value were calculated by two-sided test. **i**, Percentage of F01 TEs in Gypsy elements of _B. distachyon_ and _T. aestivum_. Colinear regions of two consecutive genes between _B. distachyon_ and _T. aestivum_ were used for analysis. **j**, Distribution of distances between adjacent F01 TEs in _T. aestivum_. **k**, Distributions of DNA methylation throughout chromosomes of cool-season legumes. **l**, CG methylation levels of F01 and other TEs in _P. sativum_ and _V. faba_. Numbers represent the sample size. **m**, The proportion of expressed F01 and other TEs (FPKM > 0) in _P. sativum_ and _V. faba_.

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM15)

### [Extended Data Fig. 9 Characterization of TE-derived OCRs.](https://www.nature.com/articles/s41588-025-02280-5/figures/15)

**a**–**c**, Percentage of cTE-, pTE- and nTE-OCRs in genes, promoters (2 kb upstream of TSS), downstream (500 bp downstream of TTS) and distal intergenic regions. **d**–**k**, DNA methylation levels of cTE-, pTE- and nTE-OCRs in _C. arietinum_ (**d**,**e**), _P. sativum_ (**f**–**h**), _L. culinaris_ (**i**–**k**) and _V. faba_ (**d**,**e**). **l**,**m**, CHG (**l**) and CHH (**m**) methylation levels of OCRs (open) within TEs and remaining parts (close). Numbers represent the sample size and the _P_ values were calculated by two-sided Wilcoxon rank-sum test in **d**–**m**. The upper and lower edges of the boxes represent the 75% and 25% quartiles, and the central line denotes the median and the whiskers extend to 1.5× IQR in **d**–**m**.

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM16)

### [Extended Data Fig. 10 Characterization of heavily methylated OCRs.](https://www.nature.com/articles/s41588-025-02280-5/figures/16)

**a**, Fraction of OCRs in unmethylated regions (UMRs) or other regions. **b**–**d**, Distributions of CG-OCRs in different kinds of TEs in _V. faba_ (**b**), _P. sativum_ (**c**) and _L. culinaris_ (**d**). Randomly selected distal regions with CG methylation were used as control. CG-OCRs were significantly enriched in DNA transposon (particularly _Mutator_) with _P_ < 2.2 × 10−16 (two-tailed Fisher’s exact test). **e**,**f**, Percentage of CG-OCRs and UMR-OCRs with transcripts in _P. sativum_ (**e**) and _L. culinaris_ (**f**). The _P_ values were determined by two-tailed Fisher’s exact test. **g**,**h**, Chromatin accessibility of CG-OCRs and UMR-OCRs in _P. sativum_ (**g**) and _L. culinaris_ (**h**). Different letters above each column indicate statistical significance at _P_ < 0.05. Numbers represent the sample size. Significance was tested using one-way ANOVA test and multiple comparisons were analyzed using Scheffe’s test. The upper and lower edges of the boxes represent the 75% and 25% quartiles, and the central line denotes the median and the whiskers extend to 1.5× IQR.

[Source data](https://www.nature.com/articles/s41588-025-02280-5#MOESM17)

Supplementary information
-------------------------

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM1_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM2_ESM.pdf)

### [Supplementary Tables 1–14](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM3_ESM.xlsx)

Supplementary Tables 1–14.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM4_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM5_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM7_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM8_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM9_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM12_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM13_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM14_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM15_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 9](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM16_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02280-5/MediaObjects/41588_2025_2280_MOESM17_ESM.xlsx)

Statistical source data.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Pangenome%20analysis%20provides%20insights%20into%20legume%20evolution%20and%20breeding&author=Longfei%20Wang%20et%20al&contentID=10.1038%2Fs41588-025-02280-5&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-07-30&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02280-5)

### Cite this article

Wang, L., Jiang, X., Jiao, W. _et al._ Pangenome analysis provides insights into legume evolution and breeding. _Nat Genet_ **57**, 2052–2061 (2025). https://doi.org/10.1038/s41588-025-02280-5

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02280-5?format=refman&flavour=citation)

-   Received: 27 July 2024
    
-   Accepted: 27 June 2025
    
-   Published: 30 July 2025
    
-   Issue Date: August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02280-5
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
