> id: https://www.nature.com/articles/s41588-025-02264-5

> link: https://www.nature.com/articles/s41588-025-02264-5

> title: Insights into Yersinia pestis evolution through rearrangement analysis of 242 complete genomes

# Insights into Yersinia pestis evolution through rearrangement analysis of 242 complete genomes
_Published on Mon, 04 Aug 2025 00:00:00 GMT_

Abstract
--------

_Yersinia pestis_, the bacterium that causes the plague, has a dynamic genome with highly conserved fragments prone to rearrangement, influencing gene function and evolution. However, understanding these patterns is limited by few complete genomes and analytical methods. We developed a dual-validation strategy to analyze 242 complete genomes of _Y. pestis_ natural isolates from diverse phylogroups. We detected 459 rearrangements, which enhanced phylogenetic resolution and resolved the third pandemic’s polytomy. Rearrangements are primarily mediated by four common insertion sequences, with IS1661 and IS100 showing the highest activity. These rearrangements are under strong positive selection, evidenced by 43 hotspots and convergent evolution in the _r__psO-pnp_ operon, whose disruptions and reconnections altered gene expressions and temperature stress responses. We also identified unique structural alterations in human avirulent phylogroups, inactivating three genes and reordering 17 intergenic regions, some affecting virulence-related genes. This study provides a fresh perspective on _Y. pestis_ evolution, revealing experimental targets and establishing a methodology for microbes with frequent rearrangements.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

27,99 € / 30 days

cancel any time

[Learn more](https://shop.nature.com/products/plus/?region=ROW)

Subscribe to this journal

Receive 12 print issues and online access

195,33 € per year

only 16,28 € per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02264-5?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Maximum-likelihood phylogenetic tree of 242 _Y. pestis_ complete genomes.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_Fig1_HTML.png)

**Fig. 2: Characterization of synteny blocks and rearrangement-related adjacencies in the _Y. pestis_ genome.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_Fig2_HTML.png)

**Fig. 3: Rearrangement events in _Y. pestis_ phylogroups.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_Fig3_HTML.png)

**Fig. 4: Analysis of key elements mediating _Y. pestis_ genomic rearrangements.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_Fig4_HTML.png)

**Fig. 5: Characterization of rearrangement hotspots.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_Fig5_HTML.png)

**Fig. 6: The _rpsO-pnp_ operon is associated with a reversion mutation within a hotspot.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_Fig6_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-022-04394-6/MediaObjects/42003_2022_4394_Fig1_HTML.png)

### [Plagued by a cryptic clock: insight and issues from the global phylogeny of _Yersinia pestis_](https://www.nature.com/articles/s42003-022-04394-6?fromPaywallRec=true)

Article Open access 19 January 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-023-05186-2/MediaObjects/42003_2023_5186_Fig1_HTML.png)

### [Genomic diversity of _Yersinia pestis_ from Yunnan Province, China, implies a potential common ancestor as the source of two plague epidemics](https://www.nature.com/articles/s42003-023-05186-2?fromPaywallRec=true)

Article Open access 15 August 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-55581-4/MediaObjects/41467_2024_55581_Fig1_HTML.png)

### [Hotspots of genetic change in _Yersinia pestis_](https://www.nature.com/articles/s41467-024-55581-4?fromPaywallRec=true)

Article Open access 04 January 2025

Data availability
-----------------

The complete genome assemblies and long-read sequencing data generated in this study have been deposited in GenBank and the Sequence Read Archive under BioProject no. [PRJNA1190511](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1190511/) and in the NMDC under BioProject no. [NMDC10018925](https://nmdc.cn/resource/genomics/project/detail/NMDC10018925). Accession numbers for each genome are listed in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02264-5#MOESM4). Publicly available complete genomes and the RNA-seq data used in this study are provided in Supplementary Tables [2](https://www.nature.com/articles/s41588-025-02264-5#MOESM4) and [10](https://www.nature.com/articles/s41588-025-02264-5#MOESM4), respectively. Insertion sequence types were identified using sequences from the ISfinder database. Functional annotations were assigned using the online eggNOG database. [Source data](https://www.nature.com/articles/s41588-025-02264-5#Sec29) are provided with this paper.

Code availability
-----------------

The code relevant to this manuscript is publicly available via GitHub at [github.com/WuYarong/YP\_Rearrangement\_Analysis](http://github.com/WuYarong/YP_Rearrangement_Analysis) and via Zenodo at [https://doi.org/10.5281/zenodo.15152926](https://doi.org/10.5281/zenodo.15152926) (ref. [54](https://www.nature.com/articles/s41588-025-02264-5#ref-CR54 "Wu, Y. WuYarong/YP_Rearrangement_Analysis: v1.0.0. Zenodo 
https://doi.org/10.5281/zenodo.15152926
(2025).")). It was implemented in Python v.3.7.10, using the packages SciPy (v.1.5.2), NumPy (v.1.18.1) and statsmodels (v.0.11.1). The insertion sequence copy number was calculated using a custom script based on the BLASTn v.2.13.0+ output format 6.

References
----------

1.  Sun, S., Ke, R., Hughes, D., Nilsson, M. & Andersson, D. I. Genome-wide detection of spontaneous chromosomal rearrangements in bacteria. _PLoS ONE_ **7**, e42639 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22880062)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3411829)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFOls7zP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20detection%20of%20spontaneous%20chromosomal%20rearrangements%20in%20bacteria&journal=PLoS%20ONE&volume=7&publication_year=2012&author=Sun%2CS&author=Ke%2CR&author=Hughes%2CD&author=Nilsson%2CM&author=Andersson%2CDI) 
    
2.  Periwal, V. & Scaria, V. Insights into structural variations and genome rearrangements in prokaryotic genomes. _Bioinformatics_ **31**, 1–9 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25189783)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXisFalsrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20into%20structural%20variations%20and%20genome%20rearrangements%20in%20prokaryotic%20genomes&journal=Bioinformatics&volume=31&pages=1-9&publication_year=2015&author=Periwal%2CV&author=Scaria%2CV) 
    
3.  Ho, S. S., Urban, A. E. & Mills, R. E. Structural variation in the sequencing era. _Nat. Rev. Genet._ **21**, 171–189 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31729472)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOisr3N)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20variation%20in%20the%20sequencing%20era&journal=Nat.%20Rev.%20Genet.&volume=21&pages=171-189&publication_year=2020&author=Ho%2CSS&author=Urban%2CAE&author=Mills%2CRE) 
    
4.  Raeside, C. et al. Large chromosomal rearrangements during a long-term evolution experiment with _Escherichia coli_. _mBio_ **5**, e01377-14 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25205090)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4173774)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large%20chromosomal%20rearrangements%20during%20a%20long-term%20evolution%20experiment%20with%20Escherichia%20coli&journal=mBio&volume=5&publication_year=2014&author=Raeside%2CC) 
    
5.  Yebra, G. et al. Radical genome remodelling accompanied the emergence of a novel host-restricted bacterial pathogen. _PLoS Pathog._ **17**, e1009606 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34015034)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8171923)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1Gksr7F)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Radical%20genome%20remodelling%20accompanied%20the%20emergence%20of%20a%20novel%20host-restricted%20bacterial%20pathogen&journal=PLoS%20Pathog.&volume=17&publication_year=2021&author=Yebra%2CG) 
    
6.  Irvine, S. et al. Genomic and transcriptomic characterization of _Pseudomonas aeruginosa_ small colony variants derived from a chronic infection model. _Microb. Genom._ **5**, e000262 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30920365)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6521587)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20and%20transcriptomic%20characterization%20of%20Pseudomonas%20aeruginosa%20small%20colony%20variants%20derived%20from%20a%20chronic%20infection%20model&journal=Microb.%20Genom.&volume=5&publication_year=2019&author=Irvine%2CS) 
    
7.  Le, V. V. H., León-Quezada, R. I., Biggs, P. J. & Rakonjac, J. A large chromosomal inversion affects antimicrobial sensitivity of _Escherichia coli_ to sodium deoxycholate. _Microbiology_ **168**, 001232 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20large%20chromosomal%20inversion%20affects%20antimicrobial%20sensitivity%20of%20Escherichia%20coli%20to%20sodium%20deoxycholate.&journal=Microbiology&volume=168&publication_year=2022&author=Le%2CVVH&author=Le%C3%B3n-Quezada%2CRI&author=Biggs%2CPJ&author=Rakonjac%2CJ) 
    
8.  Achtman, M. et al. _Yersinia pestis_, the cause of plague, is a recently emerged clone of _Yersinia pseudotuberculosis_. _Proc. Natl Acad. Sci. USA_ **96**, 14043–14048 (1999).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10570195)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC24187)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXns1Oqu7g%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yersinia%20pestis%2C%20the%20cause%20of%20plague%2C%20is%20a%20recently%20emerged%20clone%20of%20Yersinia%20pseudotuberculosis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=96&pages=14043-14048&publication_year=1999&author=Achtman%2CM) 
    
9.  Yang, R. et al. _Yersinia pestis_ and plague: some knowns and unknowns. _Zoonoses_ **3**, 5 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37602146)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10438918)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXns12iurY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yersinia%20pestis%20and%20plague%3A%20some%20knowns%20and%20unknowns.&journal=Zoonoses&volume=3&publication_year=2023&author=Yang%2CR) 
    
10.  Morelli, G. et al. _Yersinia pestis_ genome sequencing identifies patterns of global phylogenetic diversity. _Nat. Genet._ **42**, 1140–1143 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21037571)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2999892)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlGrsL3O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yersinia%20pestis%20genome%20sequencing%20identifies%20patterns%20of%20global%20phylogenetic%20diversity&journal=Nat.%20Genet.&volume=42&pages=1140-1143&publication_year=2010&author=Morelli%2CG) 
    
11.  Cui, Y. et al. Historical variations in mutation rate in an epidemic pathogen, _Yersinia pestis_. _Proc. Natl Acad. Sci. USA_ **110**, 577–582 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23271803)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOhur4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Historical%20variations%20in%20mutation%20rate%20in%20an%20epidemic%20pathogen%2C%20Yersinia%20pestis&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=110&pages=577-582&publication_year=2013&author=Cui%2CY) 
    
12.  Wu, Y. et al. Hotspots of genetic change in _Yersinia pestis_. _Nat. Commun._ **16**, 388 (2025).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39755708)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11700214)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhtFals74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hotspots%20of%20genetic%20change%20in%20Yersinia%20pestis&journal=Nat.%20Commun.&volume=16&publication_year=2025&author=Wu%2CY) 
    
13.  Wu, Y. et al. Small insertions and deletions drive genomic plasticity during adaptive evolution of _Yersinia pestis_. _Microbiol. Spectr._ **10**, e0224221 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35438532)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Small%20insertions%20and%20deletions%20drive%20genomic%20plasticity%20during%20adaptive%20evolution%20of%20Yersinia%20pestis&journal=Microbiol.%20Spectr.&volume=10&publication_year=2022&author=Wu%2CY) 
    
14.  Parkhill, J. et al. Genome sequence of _Yersinia pestis_, the causative agent of plague. _Nature_ **413**, 523–527 (2001).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11586360)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXnsFygurc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20sequence%20of%20Yersinia%20pestis%2C%20the%20causative%20agent%20of%20plague&journal=Nature&volume=413&pages=523-527&publication_year=2001&author=Parkhill%2CJ) 
    
15.  Liang, Y. et al. Genome rearrangements of completely sequenced strains of _Yersinia pestis_. _J. Clin. Microbiol._ **48**, 1619–1623 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20200297)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2863931)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXnt1Cgur0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20rearrangements%20of%20completely%20sequenced%20strains%20of%20Yersinia%20pestis&journal=J.%20Clin.%20Microbiol.&volume=48&pages=1619-1623&publication_year=2010&author=Liang%2CY) 
    
16.  Darling, A. E., Miklós, I. & Ragan, M. A. Dynamics of genome rearrangement in bacterial populations. _PLoS Genet._ **4**, e1000128 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18650965)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483231)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamics%20of%20genome%20rearrangement%20in%20bacterial%20populations&journal=PLoS%20Genet.&volume=4&publication_year=2008&author=Darling%2CAE&author=Mikl%C3%B3s%2CI&author=Ragan%2CMA) 
    
17.  Li, Y. et al. Different region analysis for genotyping _Yersinia pestis_ isolates from China. _PLoS ONE_ **3**, e2166 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18478120)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2367435)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Different%20region%20analysis%20for%20genotyping%20Yersinia%20pestis%20isolates%20from%20China&journal=PLoS%20ONE&volume=3&publication_year=2008&author=Li%2CY) 
    
18.  Anisimov, A. P., Lindler, L. E. & Pier, G. B. Intraspecific diversity of _Yersinia pestis_. _Clin. Microbiol. Rev._ **17**, 434–464 (2004).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15084509)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC387406)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXktl2isro%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intraspecific%20diversity%20of%20Yersinia%20pestis&journal=Clin.%20Microbiol.%20Rev.&volume=17&pages=434-464&publication_year=2004&author=Anisimov%2CAP&author=Lindler%2CLE&author=Pier%2CGB) 
    
19.  Li, Y. et al. The GntR-like transcriptional regulator HutC involved in motility, biofilm-forming ability, and virulence in _Vibrio parahaemolyticus_. _Micro. Pathog._ **167**, 105546 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1OmsLrL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20GntR-like%20transcriptional%20regulator%20HutC%20involved%20in%20motility%2C%20biofilm-forming%20ability%2C%20and%20virulence%20in%20Vibrio%20parahaemolyticus&journal=Micro.%20Pathog.&volume=167&publication_year=2022&author=Li%2CY) 
    
20.  Koebnik, R., Hantke, K. & Braun, V. The TonB-dependent ferrichrome receptor FcuA of _Yersinia enterocolitica_: evidence against a strict co-evolution of receptor structure and substrate specificity. _Mol. Microbiol._ **7**, 383–393 (1993).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8384682)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXit1Kku7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20TonB-dependent%20ferrichrome%20receptor%20FcuA%20of%20Yersinia%20enterocolitica%3A%20evidence%20against%20a%20strict%20co-evolution%20of%20receptor%20structure%20and%20substrate%20specificity&journal=Mol.%20Microbiol.&volume=7&pages=383-393&publication_year=1993&author=Koebnik%2CR&author=Hantke%2CK&author=Braun%2CV) 
    
21.  Islam, M. M., Kim, K., Lee, J. C. & Shin, M. LeuO, a LysR-type transcriptional regulator, is involved in biofilm formation and virulence of _Acinetobacter baumannii_. _Front. Cell. Infect. Microbiol._ **11**, 738706 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34708004)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8543017)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XmslWktr8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LeuO%2C%20a%20LysR-type%20transcriptional%20regulator%2C%20is%20involved%20in%20biofilm%20formation%20and%20virulence%20of%20Acinetobacter%20baumannii&journal=Front.%20Cell.%20Infect.%20Microbiol.&volume=11&publication_year=2021&author=Islam%2CMM&author=Kim%2CK&author=Lee%2CJC&author=Shin%2CM) 
    
22.  Jiao, W. B. & Schneeberger, K. Chromosome-level assemblies of multiple _Arabidopsis_ genomes reveal hotspots of rearrangements with altered evolutionary dynamics. _Nat. Commun._ **11**, 989 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32080174)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7033125)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlt1Shtrg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosome-level%20assemblies%20of%20multiple%20Arabidopsis%20genomes%20reveal%20hotspots%20of%20rearrangements%20with%20altered%20evolutionary%20dynamics&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Jiao%2CWB&author=Schneeberger%2CK) 
    
23.  Crow, K. D., Wagner, G. P. & SMBE Tri-National Young Investigators Proceedings of the SMBE Tri-National Young Investigators’ Workshop 2005. What is the role of genome duplication in the evolution of complexity and diversity? _Mol. Biol. Evol._ **23**, 887–892 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16368775)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XkvVeqtLY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proceedings%20of%20the%20SMBE%20Tri-National%20Young%20Investigators%E2%80%99%20Workshop%202005.%20What%20is%20the%20role%20of%20genome%20duplication%20in%20the%20evolution%20of%20complexity%20and%20diversity%3F&journal=Mol.%20Biol.%20Evol.&volume=23&pages=887-892&publication_year=2006&author=Crow%2CKD&author=Wagner%2CGP&author=SMBE%20Tri-National%20Young%20Investigators%2C) 
    
24.  Hawkey, J., Monk, J. M., Billman-Jacobe, H., Palsson, B. & Holt, K. E. Impact of insertion sequences on convergent evolution of _Shigella_ species. _PLoS Genet._ **16**, e1008931 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32644999)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7373316)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVKhsLrE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Impact%20of%20insertion%20sequences%20on%20convergent%20evolution%20of%20Shigella%20species&journal=PLoS%20Genet.&volume=16&publication_year=2020&author=Hawkey%2CJ&author=Monk%2CJM&author=Billman-Jacobe%2CH&author=Palsson%2CB&author=Holt%2CKE) 
    
25.  Tenaillon, O. et al. The molecular diversity of adaptive convergence. _Science_ **335**, 457–461 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22282810)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFajsbs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20diversity%20of%20adaptive%20convergence&journal=Science&volume=335&pages=457-461&publication_year=2012&author=Tenaillon%2CO) 
    
26.  Seferbekova, Z. et al. High rates of genome rearrangements and pathogenicity of _Shigella_ spp. _Front. Microbiol._ **12**, 628622 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33912145)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8072062)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High%20rates%20of%20genome%20rearrangements%20and%20pathogenicity%20of%20Shigella%20spp&journal=Front.%20Microbiol.&volume=12&publication_year=2021&author=Seferbekova%2CZ) 
    
27.  Zabelkin, A., Yakovleva, Y., Bochkareva, O. & Alexeev, N. PaReBrick: PArallel REarrangements and BReaks identification toolkit. _Bioinformatics_ **38**, 357–363 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34601581)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XmtlyjtLw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PaReBrick%3A%20PArallel%20REarrangements%20and%20BReaks%20identification%20toolkit&journal=Bioinformatics&volume=38&pages=357-363&publication_year=2022&author=Zabelkin%2CA&author=Yakovleva%2CY&author=Bochkareva%2CO&author=Alexeev%2CN) 
    
28.  Régnier, P. & Portier, C. Initiation, attenuation and RNase III processing of transcripts from the _Escherichia coli_ operon encoding ribosomal protein S15 and polynucleotide phosphorylase. _J. Mol. Biol._ **187**, 23–32 (1986).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3007765)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Initiation%2C%20attenuation%20and%20RNase%20III%20processing%20of%20transcripts%20from%20the%20Escherichia%20coli%20operon%20encoding%20ribosomal%20protein%20S15%20and%20polynucleotide%20phosphorylase&journal=J.%20Mol.%20Biol.&volume=187&pages=23-32&publication_year=1986&author=R%C3%A9gnier%2CP&author=Portier%2CC) 
    
29.  Clarke, D. J. & Dowds, B. C. The gene coding for polynucleotide phosphorylase in _Photorhabdus_ sp. strain K122 is induced at low temperatures. _J. Bacteriol._ **176**, 3775–3784 (1994).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8206856)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC205567)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXkslCqu7c%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20gene%20coding%20for%20polynucleotide%20phosphorylase%20in%20Photorhabdus%20sp.%20strain%20K122%20is%20induced%20at%20low%20temperatures&journal=J.%20Bacteriol.&volume=176&pages=3775-3784&publication_year=1994&author=Clarke%2CDJ&author=Dowds%2CBC) 
    
30.  Goverde, R. L., Huis in’t Veld, J. H., Kusters, J. G. & Mooi, F. R. The psychrotrophic bacterium _Yersinia enterocolitica_ requires expression of _pnp_, the gene for polynucleotide phosphorylase, for growth at low temperature (5 °C). _Mol. Microbiol._ **28**, 555–569 (1998).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9632258)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXjtF2mtrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20psychrotrophic%20bacterium%20Yersinia%20enterocolitica%20requires%20expression%20of%20pnp%2C%20the%20gene%20for%20polynucleotide%20phosphorylase%2C%20for%20growth%20at%20low%20temperature%20%285%E2%80%89%C2%B0C%29&journal=Mol.%20Microbiol.&volume=28&pages=555-569&publication_year=1998&author=Goverde%2CRL&author=Huis%20in%E2%80%99t%20Veld%2CJH&author=Kusters%2CJG&author=Mooi%2CFR) 
    
31.  Bralley, P., Gatewood, M. L. & Jones, G. H. Transcription of the _rpsO_\-_pnp_ operon of _Streptomyces coelicolor_ involves four temporally regulated, stress responsive promoters. _Gene_ **536**, 177–185 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24211388)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFOjtbzN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20of%20the%20rpsO-pnp%20operon%20of%20Streptomyces%20coelicolor%20involves%20four%20temporally%20regulated%2C%20stress%20responsive%20promoters&journal=Gene&volume=536&pages=177-185&publication_year=2014&author=Bralley%2CP&author=Gatewood%2CML&author=Jones%2CGH) 
    
32.  Zhang, Q. et al. _Yersinia pestis_ biovar _Microtus_ strain 201, an avirulent strain to humans, provides protection against bubonic plague in rhesus macaques. _Hum. Vaccin. Immunother._ **10**, 368–377 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24225642)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFCisr3J)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yersinia%20pestis%20biovar%20Microtus%20strain%20201%2C%20an%20avirulent%20strain%20to%20humans%2C%20provides%20protection%20against%20bubonic%20plague%20in%20rhesus%20macaques&journal=Hum.%20Vaccin.%20Immunother.&volume=10&pages=368-377&publication_year=2014&author=Zhang%2CQ) 
    
33.  Song, Y. et al. Complete genome sequence of _Yersinia pestis_ strain 91001, an isolate avirulent to humans. _DNA Res._ **11**, 179–197 (2004).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15368893)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmtFGqsL0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complete%20genome%20sequence%20of%20Yersinia%20pestis%20strain%2091001%2C%20an%20isolate%20avirulent%20to%20humans&journal=DNA%20Res.&volume=11&pages=179-197&publication_year=2004&author=Song%2CY) 
    
34.  Ellis, M. J., Trussler, R. S., Charles, O. & Haniford, D. B. A transposon-derived small RNA regulates gene expression in _Salmonella typhimurium_. _Nucleic Acids Res._ **45**, 5470–5486 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28335027)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5435999)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFemsrs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20transposon-derived%20small%20RNA%20regulates%20gene%20expression%20in%20Salmonella%20typhimurium&journal=Nucleic%20Acids%20Res.&volume=45&pages=5470-5486&publication_year=2017&author=Ellis%2CMJ&author=Trussler%2CRS&author=Charles%2CO&author=Haniford%2CDB) 
    
35.  Ellis, M. J., Trussler, R. S. & Haniford, D. B. A _cis_\-encoded sRNA, Hfq and mRNA secondary structure act independently to suppress IS_200_ transposition. _Nucleic Acids Res._ **43**, 6511–6527 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26044710)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4513863)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1Ols7%2FE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cis-encoded%20sRNA%2C%20Hfq%20and%20mRNA%20secondary%20structure%20act%20independently%20to%20suppress%20IS200%20transposition&journal=Nucleic%20Acids%20Res.&volume=43&pages=6511-6527&publication_year=2015&author=Ellis%2CMJ&author=Trussler%2CRS&author=Haniford%2CDB) 
    
36.  Hu, F., Lin, Y. & Tang, J. MLGO: phylogeny reconstruction and ancestral inference from gene-order data. _BMC Bioinformatics_ **15**, 354 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25376663)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4236499)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MLGO%3A%20phylogeny%20reconstruction%20and%20ancestral%20inference%20from%20gene-order%20data&journal=BMC%20Bioinformatics&volume=15&publication_year=2014&author=Hu%2CF&author=Lin%2CY&author=Tang%2CJ) 
    
37.  Drillon, G., Champeimont, R., Oteri, F., Fischer, G. & Carbone, A. Phylogenetic reconstruction based on synteny block and gene adjacencies. _Mol. Biol. Evol._ **37**, 2747–2762 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32384156)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7475045)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1egu7rF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phylogenetic%20reconstruction%20based%20on%20synteny%20block%20and%20gene%20adjacencies&journal=Mol.%20Biol.%20Evol.&volume=37&pages=2747-2762&publication_year=2020&author=Drillon%2CG&author=Champeimont%2CR&author=Oteri%2CF&author=Fischer%2CG&author=Carbone%2CA) 
    
38.  Bohnenkämper, L., Braga, M. D. V., Doerr, D. & Stoye, J. Computing the rearrangement distance of natural genomes. _J. Comput. Biol._ **28**, 410–431 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33393848)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8082732)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computing%20the%20rearrangement%20distance%20of%20natural%20genomes&journal=J.%20Comput.%20Biol.&volume=28&pages=410-431&publication_year=2021&author=Bohnenk%C3%A4mper%2CL&author=Braga%2CMDV&author=Doerr%2CD&author=Stoye%2CJ) 
    
39.  Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. _Bioinformatics_ **30**, 2114–2120 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24695404)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&journal=Bioinformatics&volume=30&pages=2114-2120&publication_year=2014&author=Bolger%2CAM&author=Lohse%2CM&author=Usadel%2CB) 
    
40.  Wick, R. R., Judd, L. M., Gorrie, C. L. & Holt, K. E. Unicycler: resolving bacterial genome assemblies from short and long sequencing reads. _PLoS Comput. Biol._ **13**, e1005595 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28594827)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5481147)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unicycler%3A%20resolving%20bacterial%20genome%20assemblies%20from%20short%20and%20long%20sequencing%20reads&journal=PLoS%20Comput.%20Biol.&volume=13&publication_year=2017&author=Wick%2CRR&author=Judd%2CLM&author=Gorrie%2CCL&author=Holt%2CKE) 
    
41.  Koren, S. et al. Canu: scalable and accurate long-read assembly via adaptive _k_\-mer weighting and repeat separation. _Genome Res._ **27**, 722–736 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28298431)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411767)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFyjsrvI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&journal=Genome%20Res.&volume=27&pages=722-736&publication_year=2017&author=Koren%2CS) 
    
42.  Kolmogorov, M., Yuan, J., Lin, Y. & Pevzner, P. A. Assembly of long, error-prone reads using repeat graphs. _Nat. Biotechnol._ **37**, 540–546 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30936562)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosV2qsrs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Assembly%20of%20long%2C%20error-prone%20reads%20using%20repeat%20graphs&journal=Nat.%20Biotechnol.&volume=37&pages=540-546&publication_year=2019&author=Kolmogorov%2CM&author=Yuan%2CJ&author=Lin%2CY&author=Pevzner%2CPA) 
    
43.  Hunt, M. et al. Circlator: automated circularization of genome assemblies using long sequencing reads. _Genome Biol._ **16**, 294 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26714481)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4699355)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Circlator%3A%20automated%20circularization%20of%20genome%20assemblies%20using%20long%20sequencing%20reads&journal=Genome%20Biol.&volume=16&publication_year=2015&author=Hunt%2CM) 
    
44.  Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. & Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. _Genome Res._ **25**, 1043–1055 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25977477)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4484387)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1SltbfE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CheckM%3A%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates%2C%20single%20cells%2C%20and%20metagenomes&journal=Genome%20Res.&volume=25&pages=1043-1055&publication_year=2015&author=Parks%2CDH&author=Imelfort%2CM&author=Skennerton%2CCT&author=Hugenholtz%2CP&author=Tyson%2CGW) 
    
45.  Kurtz, S. et al. Versatile and open software for comparing large genomes. _Genome Biol._ **5**, R12 (2004).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14759262)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC395750)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&journal=Genome%20Biol.&volume=5&publication_year=2004&author=Kurtz%2CS) 
    
46.  Benson, G. Tandem repeats finder: a program to analyze DNA sequences. _Nucleic Acids Res._ **27**, 573–580 (1999).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9862982)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC148217)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXhtVKmtrg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&journal=Nucleic%20Acids%20Res.&volume=27&pages=573-580&publication_year=1999&author=Benson%2CG) 
    
47.  Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at [https://arxiv.org/abs/1303.3997](https://arxiv.org/abs/1303.3997) (2013).
    
48.  McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. _Genome Res._ **20**, 1297–1303 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20644199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFeru7jM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&journal=Genome%20Res.&volume=20&pages=1297-1303&publication_year=2010&author=McKenna%2CA) 
    
49.  Nguyen, L.-T., Schmidt, H. A., von Haeseler, A. & Minh, B. Q. IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. _Mol. Biol. Evol._ **32**, 268–274 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25371430)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXivFGltrs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&journal=Mol.%20Biol.%20Evol.&volume=32&pages=268-274&publication_year=2015&author=Nguyen%2CL-T&author=Schmidt%2CHA&author=von%20Haeseler%2CA&author=Minh%2CBQ) 
    
50.  Hoang, D. T., Chernomor, O., von Haeseler, A., Minh, B. Q. & Vinh, L. S. UFBoot2: improving the ultrafast bootstrap approximation. _Mol. Biol. Evol._ **35**, 518–522 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29077904)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlyjs7rK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UFBoot2%3A%20improving%20the%20ultrafast%20bootstrap%20approximation&journal=Mol.%20Biol.%20Evol.&volume=35&pages=518-522&publication_year=2018&author=Hoang%2CDT&author=Chernomor%2CO&author=von%20Haeseler%2CA&author=Minh%2CBQ&author=Vinh%2CLS) 
    
51.  Letunic, I. & Bork, P. Interactive Tree of Life (iTOL) v6: recent updates to the phylogenetic tree display and annotation tool. _Nucleic Acids Res._ **52**, W78–W82 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38613393)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11223838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interactive%20Tree%20of%20Life%20%28iTOL%29%20v6%3A%20recent%20updates%20to%20the%20phylogenetic%20tree%20display%20and%20annotation%20tool&journal=Nucleic%20Acids%20Res.&volume=52&pages=W78-W82&publication_year=2024&author=Letunic%2CI&author=Bork%2CP) 
    
52.  Argimón, S. et al. Microreact: visualizing and sharing data for genomic epidemiology and phylogeography. _Microb. Genom._ **2**, e000093 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28348833)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5320705)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Microreact%3A%20visualizing%20and%20sharing%20data%20for%20genomic%20epidemiology%20and%20phylogeography&journal=Microb.%20Genom.&volume=2&publication_year=2016&author=Argim%C3%B3n%2CS) 
    
53.  Popescu, A.-A., Huber, K. T. & Paradis, E. ape 3.0: new tools for distance-based phylogenetics and evolutionary analysis in R. _Bioinformatics_ **28**, 1536–1537 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22495750)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xns1Slt70%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ape%203.0%3A%20new%20tools%20for%20distance-based%20phylogenetics%20and%20evolutionary%20analysis%20in%20R&journal=Bioinformatics&volume=28&pages=1536-1537&publication_year=2012&author=Popescu%2CA-A&author=Huber%2CKT&author=Paradis%2CE) 
    
54.  Wu, Y. WuYarong/YP\_Rearrangement\_Analysis: v1.0.0. _Zenodo_ [https://doi.org/10.5281/zenodo.15152926](https://doi.org/10.5281/zenodo.15152926) (2025).
    
55.  Siguier, P., Perochon, J., Lestrade, L., Mahillon, J. & Chandler, M. ISfinder: the reference centre for bacterial insertion sequences. _Nucleic Acids Res._ **34**, D32–D36 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16381877)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XisFOhug%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ISfinder%3A%20the%20reference%20centre%20for%20bacterial%20insertion%20sequences&journal=Nucleic%20Acids%20Res.&volume=34&pages=D32-D36&publication_year=2006&author=Siguier%2CP&author=Perochon%2CJ&author=Lestrade%2CL&author=Mahillon%2CJ&author=Chandler%2CM) 
    
56.  Minkin, I. & Medvedev, P. Scalable multiple whole-genome alignment and locally collinear block construction with SibeliaZ. _Nat. Commun._ **11**, 6327 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33303762)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7728760)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1Sht73K)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20multiple%20whole-genome%20alignment%20and%20locally%20collinear%20block%20construction%20with%20SibeliaZ&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Minkin%2CI&author=Medvedev%2CP) 
    
57.  Seemann, T. Prokka: rapid prokaryotic genome annotation. _Bioinformatics_ **30**, 2068–2069 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24642063)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFCrtLjI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prokka%3A%20rapid%20prokaryotic%20genome%20annotation&journal=Bioinformatics&volume=30&pages=2068-2069&publication_year=2014&author=Seemann%2CT) 
    
58.  Tonkin-Hill, G. et al. Producing polished prokaryotic pangenomes with the Panaroo pipeline. _Genome Biol._ **21**, 180 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32698896)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7376924)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Producing%20polished%20prokaryotic%20pangenomes%20with%20the%20Panaroo%20pipeline&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Tonkin-Hill%2CG) 
    
59.  Goel, M., Sun, H., Jiao, W.-B. & Schneeberger, K. SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies. _Genome Biol._ **20**, 277 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31842948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6913012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SyRI%3A%20finding%20genomic%20rearrangements%20and%20local%20sequence%20differences%20from%20whole-genome%20assemblies&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Goel%2CM&author=Sun%2CH&author=Jiao%2CW-B&author=Schneeberger%2CK) 
    
60.  Goel, M. & Schneeberger, K. plotsr: visualizing structural similarities and rearrangements between multiple genomes. _Bioinformatics_ **38**, 2922–2926 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35561173)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9113368)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhslalur3L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=plotsr%3A%20visualizing%20structural%20similarities%20and%20rearrangements%20between%20multiple%20genomes&journal=Bioinformatics&volume=38&pages=2922-2926&publication_year=2022&author=Goel%2CM&author=Schneeberger%2CK) 
    
61.  Sedlazeck, F. J. et al. Accurate detection of complex structural variations using single-molecule sequencing. _Nat. Methods_ **15**, 461–468 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29713083)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5990442)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFOmtLrP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20detection%20of%20complex%20structural%20variations%20using%20single-molecule%20sequencing&journal=Nat.%20Methods&volume=15&pages=461-468&publication_year=2018&author=Sedlazeck%2CFJ) 
    
62.  Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. _Bioinformatics_ **31**, 166–169 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25260700)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Sjt7vL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HTSeq%E2%80%94a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&journal=Bioinformatics&volume=31&pages=166-169&publication_year=2015&author=Anders%2CS&author=Pyl%2CPT&author=Huber%2CW) 
    
63.  Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._ **15**, 550 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25516281)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Love%2CMI&author=Huber%2CW&author=Anders%2CS) 
    
64.  Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. _Nucleic Acids Res._ **47**, D309–D314 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30418610)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=eggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&journal=Nucleic%20Acids%20Res.&volume=47&pages=D309-D314&publication_year=2019&author=Huerta-Cepas%2CJ) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02264-5?format=refman&flavour=references)

Acknowledgements
----------------

This work was supported by the National Key Research and Development Program of China (no. 2022YFC2305304 to Y.C.), the National Natural Science Foundation of China (no. 31970002 to Y.C.) and the State Key Laboratory of Pathogen and Biosecurity (no. SKLPBS2215 to Y.W.).

Author information
------------------

### Authors and Affiliations

1.  State Key Laboratory of Pathogen and Biosecurity, Academy of Military Medical Sciences, Beijing, China
    
    Yarong Wu, Kai Mu, Yan Guo, Yajun Song, Ruifu Yang & Yujun Cui
    
2.  The Center for Microbes, Development and Health, CAS Key Laboratory of Molecular Virology and Immunology, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, China
    
    Chao Yang
    

Authors

1.  Yarong Wu
    
    [View author publications](https://www.nature.com/search?author=Yarong%20Wu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yarong%20Wu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yarong%20Wu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Chao Yang
    
    [View author publications](https://www.nature.com/search?author=Chao%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chao%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chao%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Kai Mu
    
    [View author publications](https://www.nature.com/search?author=Kai%20Mu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kai%20Mu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kai%20Mu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Yan Guo
    
    [View author publications](https://www.nature.com/search?author=Yan%20Guo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yan%20Guo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yan%20Guo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Yajun Song
    
    [View author publications](https://www.nature.com/search?author=Yajun%20Song)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yajun%20Song) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yajun%20Song%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Ruifu Yang
    
    [View author publications](https://www.nature.com/search?author=Ruifu%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ruifu%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ruifu%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Yujun Cui
    
    [View author publications](https://www.nature.com/search?author=Yujun%20Cui)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yujun%20Cui) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yujun%20Cui%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Y.C. designed the study. Y.G. conducted the DNA library construction and long-read sequencing. K.M. performed the complete genome assembly. Y.W. and C.Y. analyzed the genomic rearrangements. Y.W. and Y.C. drafted the manuscript. Y.S. provided valuable advice and insights. Y.C. and R.Y. supervised the project. All authors read, revised and approved the final manuscript.

### Corresponding authors

Correspondence to [Ruifu Yang](mailto:ruifuyang@gmail.com) or [Yujun Cui](mailto:cuiyujun.new@gmail.com).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Hendrik Poinar and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02264-5#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Pipeline for identifying rearrangement events in _Y. pestis_ phylogroups.](https://www.nature.com/articles/s41588-025-02264-5/figures/7)

**a**, Synteny block construction without a reference genome using SibellaZ and maf2synteny tools. The adjacency marks synteny block neighbors, with ‘h’ for the head and ‘t’ for the tail of blocks. For instance, ‘2h-3t’ illustrates that the head of block no. 2 is adjacent to the tail of block no. 3. **b**, Pairwise rearrangement detection based on reference genomes with SyRI software. SYN denotes syntenic regions, INV for inversions, TRANS for translocations, INVTR for inverted translocations, and DUP for duplications. Our focus of genomic rearrangements was on INV, TRANS, and INVTR. **c**, Synteny block analysis combined with pairwise rearrangement identification. **d**, Determining rearrangement events in the last common ancestor (LCA) of phylogroups. **e**, Detecting rearrangement events within specific phylogroups.

### [Extended Data Fig. 2 Synteny blocks dynamics in _Y. pestis._](https://www.nature.com/articles/s41588-025-02264-5/figures/8)

**a**, Clustered heatmap of 113 accessory blocks in _Y. pestis_ strains, based on their copy number variations. Color intensity reflects copy number, with dendrogram showing clustering. Names of 242 _Y. pestis_ strains and _Y. pseudotuberculosis_ outgroup are listed on the right. **b**, Close-up on the phylogroup-specific distribution of 43 synteny blocks, a subset derived from panel **a**, using the same heatmap color legend. Blocks are arranged by variations across phylogroups in the heatmap, highlighted with purple outlines. Parallelism scores from PareBrick analysis are shown above, while blue lines below mark the merging of adjacent blocks into larger segments, with prophage and ribosomal RNA (rRNA) related blocks labeled.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 3 Principal component analysis (PCA) of phylogroup clustering in _Y. pestis_ and comparison of synteny diversity with _Y. pseudotuberculosis._](https://www.nature.com/articles/s41588-025-02264-5/figures/9)

**a**, PCA cumulative variance plot of 178 synteny block adjacencies associated with rearrangements. The first six principal components (PCs) accounted for over 60.2% of the variance. **b**, PCA cumulative variance plot of 3,185 single nucleotide polymorphisms (SNPs) and clustering of _Y. pestis_ phylogroups on PC1 and PC2. The first six PCs accounted for 69.8% of the variance. The shaded area represents the cluster of early-diverging 0.PE phylogroups, including 0.PE7, 0.PE2, 0.PE4A, 0.PE4B, and 0.PE4C. **c**, PCA cumulative variance plot of 2,023 indels and clustering of _Y. pestis_ phylogroups on PC1 and PC2. The first six PCs accounted for 58.6% of the variance. The shaded area represents the cluster of early-diverging 0.PE phylogroups, including 0.PE7, 0.PE2, 0.PE4A, 0.PE4B, and 0.PE4C. **d**, Synteny diversity comparison between 242 _Y. pestis_ and 25 _Y. pseudotuberculosis_ strains. A sliding window analysis was performed using the IP32953 chromosome as the reference. Synteny diversity in 50-kb windows (10-kb steps) is visualized in purple for _Y. pestis_ and orange for _Y. pseudotuberculosis_, with 5-kb windows (1-kb steps) in grey for both.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 4 Phylogroup LCA rearrangement events with varied reference genomes.](https://www.nature.com/articles/s41588-025-02264-5/figures/10)

**a**, Rearrangements in the LCAs of branch 0 phylogroups, using the _Y. pseudotuberculosis_ IP32953 chromosome as a reference. Synteny diversity (_π__syn_) for each phylogroup’s strains (5 kb sliding windows with a 1 kb step-size) is aligned with this reference genome. **b**, Rearrangements in the LCAs of phylogroups within branches 2-4, along with 1.ANT1 and 1.IN1 from branch 1, using the chromosome of strain 43005 (from 0.ANT3) as a reference. **c**, Rearrangements in 1.IN1-1.IN5 and 1.ORI1 LCAs, referenced against the chromosome of strain 15002 (from 1.IN2). **d**, Rearrangements in 1.ORI2 and 1.ORI3 LCAs, with the chromosome of strain El-Dorado (from 1.ORI1) as the reference. To be noted, 0.PE2, 0.PE4B, 0.PE4C, and 0.ANT3 (represented as 0.PE4-0.ANT1) have been presented in Fig. [3](https://www.nature.com/articles/s41588-025-02264-5#Fig3). In order to comprehensively depict the various populations of _Y. pestis_, they are listed here again.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 5 Comparative synteny diversity in _Y. pestis_ phylogroups.](https://www.nature.com/articles/s41588-025-02264-5/figures/11)

**a**, Synteny diversity (_π__syn_) in IP32953 reference alignments. This panel illustrates a prominent inversion in the _Y. pestis_ phylogenetic stem between 0.PE4 and 0.ANT1, along with a smaller-scale translocation, demarcated by shaded areas. **b**, Synteny diversity (_π__syn_) in 43005 reference alignments. This section highlights stepwise genomic rearrangements within branch 1, as indicated in color-coded shaded regions.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 6 Heatmap of average genomic synteny proportion between pairwise strains across _Y. pestis_ phylogroups.](https://www.nature.com/articles/s41588-025-02264-5/figures/12)

Color gradient from blue to red represents increasing genomic collinearity from low to high.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 7 Rearrangement patterns in branch 2 for the 2.ANT and 2.MED phylogroups.](https://www.nature.com/articles/s41588-025-02264-5/figures/13)

Numerical labels represent rearrangement counts, with orange points highlighting events that delineate phylogenetic splits. Given the complexity of determining synteny block order for the common ancestor of the 2.ANT and 2.MED phylogroups, our analysis was limited to their rearrangements compared with strain 43005 from the 0.ANT3 phylogroup, as marked by asterisks. The same rearrangement variations occurring across different phylogroups were independently counted within each respective population.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 8 Distribution of four common insertion sequences in various states across _Y. pestis_ chromosomes.](https://www.nature.com/articles/s41588-025-02264-5/figures/14)

Both single elements (for example, IS100 or IS1541) and combined elements (for example, ‘IS100-IS1541’, representing adjacent insertion sequences treated as a single unit) are shown (n = 242 chromosomes). Combined elements are counted separately and not included in single-element counts. Counts reflect the number of genomic fragments mapped to each insertion sequence type, irrespective of fragment length, which may result in slight differences from the copy number accumulation analyses described in the [Supplementary Notes](https://www.nature.com/articles/s41588-025-02264-5#MOESM1). In boxplots, the top, middle, and bottom lines indicate the 75th percentile, median, and 25th percentile, respectively. Whiskers extend to 1.5 times the interquartile range (IQR), and points beyond this range are shown as outliers.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 9 Comparison of two methods for identifying rearrangement hotspots and reassessment using an alternative reference.](https://www.nature.com/articles/s41588-025-02264-5/figures/15)

**a**, Comparison of the methods using strain 43005 as reference. Linear regression was performed between the -log10\-transformed corrected _p_\-values from the Poisson distribution method (y-axis) and the parallelism break scores from PareBrick (x-axis) in R, with hotspot 391t-42h (unfilled circle) excluded as an outlier. The solid line represents the fitted regression line (predicted mean response), with the shaded area indicating the 95% confidence interval (CI) of the fitted values. Adjusted _r_² and _P_ value for the slope coefficient are shown. Horizontal and vertical dashed lines mark the detection thresholds of the Poisson method (corrected _P_ = 0.05, -log10 scale) and PareBrick (score=1), respectively. Red diamonds indicate hotspots identified by the Poisson method only, while green highlights those identified by PareBrick but lacking statistical significance in the Poisson method. **b**, Profiling rearrangement hotspots using IP32953 chromosome as reference. See Fig. [5](https://www.nature.com/articles/s41588-025-02264-5#Fig5) for legend details. **c**, Comparison of the methods using strain IP32953 as reference. The figure legend is the same as in panel **a**.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

### [Extended Data Fig. 10 Evolutionary changes of the _rpsO-pnp_ operon within _Y. pestis._](https://www.nature.com/articles/s41588-025-02264-5/figures/16)

The left panel displays the phylogenetic tree of 242 _Y. pestis_ strains, with _Y. pseudotuberculosis_ IP32953 as the outgroup. The tree highlights five main branches of _Y. pestis_ in distinct colors, with strain names positioned near the tree. The phylogenetic positions of the three global pandemics are annotated. _Y. pestis_ strains from the first two, as ancient DNA without complete genomes, are not included in this study and marked by dashed arrows. The colored bar to the right of the tree represents designated phylogroups. The second bar denotes the _rpsO-pnp_ operon’s state as disrupted or connected. The third bar, normalized to an average sequencing depth of 100, indicates the number of long reads supporting the operon’s connected state. The final bar shows whether there is an IS1661 insertion between the _rpsO_ and _pnp_ genes, with ‘IS1661-free’ marking the absence of such insertion.

[Source data](https://www.nature.com/articles/s41588-025-02264-5#MOESM6)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_MOESM1_ESM.pdf)

Supplementary Notes 1–4 and Figs. 1–4.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_MOESM3_ESM.pdf)

### [Supplementary Tables 1–17](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_MOESM4_ESM.xlsx)

Supplementary Tables 1–17.

### [Supplementary Data](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_MOESM5_ESM.xlsx)

Statistical source data for Supplementary Figs. 1–4.

Source data
-----------

### [Source data for Figs. 1–6 and Extended Data Figs 2–10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02264-5/MediaObjects/41588_2025_2264_MOESM6_ESM.xlsx)

Statistical source data for Figs. 1–6 and Extended Data Figs. 2–10.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Insights%20into%20Yersinia%20pestis%20evolution%20through%20rearrangement%20analysis%20of%20242%20complete%20genomes&author=Yarong%20Wu%20et%20al&contentID=10.1038%2Fs41588-025-02264-5&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-04&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02264-5)

### Cite this article

Wu, Y., Yang, C., Mu, K. _et al._ Insights into _Yersinia pestis_ evolution through rearrangement analysis of 242 complete genomes. _Nat Genet_ **57**, 1994–2003 (2025). https://doi.org/10.1038/s41588-025-02264-5

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02264-5?format=refman&flavour=citation)

-   Received: 06 June 2024
    
-   Accepted: 11 June 2025
    
-   Published: 04 August 2025
    
-   Issue Date: August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02264-5
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
