> id: https://www.nature.com/articles/s41588-025-02297-w

> link: https://www.nature.com/articles/s41588-025-02297-w

> title: Tracing the evolution of single-cell 3D genomes in Kras-driven cancers

# Tracing the evolution of single-cell 3D genomes in Kras-driven cancers
_Published on Mon, 18 Aug 2025 00:00:00 GMT_

Abstract
--------

Although three-dimensional (3D) genome structures are altered in cancer, it remains unclear how these changes evolve and diversify during cancer progression. Leveraging genome-wide chromatin tracing to visualize 3D genome folding directly in tissues, we generated 3D genome cancer atlases of oncogenic Kras-driven mouse lung adenocarcinoma (LUAD) and pancreatic ductal adenocarcinoma. Here we define nonmonotonic, stage-specific alterations in 3D genome compaction, heterogeneity and compartmentalization as cancers progress from normal to preinvasive and ultimately to invasive tumors, discovering a potential structural bottleneck in early tumor progression. Remarkably, 3D genome architectures distinguish morphologic cancer states in single cells, despite considerable cell-to-cell heterogeneity. Analyses of genome compartmentalization changes not only showed that compartment-associated genes are more homogeneously regulated but also elucidated prognostic and dependency genes in LUAD, as well as an unexpected role for Rnf2 in 3D genome regulation. Our results highlight the power of single-cell 3D genome mapping to identify diagnostic, prognostic and therapeutic biomarkers in cancer.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-025-07647-2/MediaObjects/42003_2025_7647_Fig1_HTML.png)

### [3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression](https://www.nature.com/articles/s42003-025-07647-2?fromPaywallRec=false)

Article Open access 04 March 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-025-02188-0/MediaObjects/41588_2025_2188_Fig1_HTML.png)

### [Three-dimensional genome landscape of primary human cancers](https://www.nature.com/articles/s41588-025-02188-0?fromPaywallRec=false)

Article Open access 12 May 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-023-06054-z/MediaObjects/41586_2023_6054_Fig1_HTML.png)

### [Pan-cancer whole-genome comparison of primary and metastatic solid tumours](https://www.nature.com/articles/s41586-023-06054-z?fromPaywallRec=false)

Article Open access 10 May 2023

Main
----

Cancer cells exhibit profound alterations in nuclear size, shape and chromatin texture[1](https://www.nature.com/articles/s41588-025-02297-w#ref-CR1 "Zink, D., Fischer, A. H. & Nickerson, J. A. Nuclear structure in cancer cells. Nat. Rev. Cancer 4, 677–687 (2004).")—features that are conventionally used for diagnosis and establishing pathologic cancer grade[1](https://www.nature.com/articles/s41588-025-02297-w#ref-CR1 "Zink, D., Fischer, A. H. & Nickerson, J. A. Nuclear structure in cancer cells. Nat. Rev. Cancer 4, 677–687 (2004)."). It has become increasingly clear that the three-dimensional (3D) genome organization within the nucleus has a crucial role in both normal physiology and disease[2](https://www.nature.com/articles/s41588-025-02297-w#ref-CR2 "Krijger, P. H. L. & de Laat, W. Regulation of disease-associated gene expression in the 3D genome. Nat. Rev. Mol. Cell Biol. 17, 771–782 (2016)."),[3](https://www.nature.com/articles/s41588-025-02297-w#ref-CR3 "Zheng, H. & Xie, W. The role of 3D genome organization in development and cell differentiation. Nat. Rev. Mol. Cell Biol. 20, 535–550 (2019)."),[4](https://www.nature.com/articles/s41588-025-02297-w#ref-CR4 "Yu, M. & Ren, B. The three-dimensional organization of mammalian genomes. Annu. Rev. Cell Dev. Biol. 33, 265–289 (2017)."),[5](https://www.nature.com/articles/s41588-025-02297-w#ref-CR5 "Bickmore, W. A. & van Steensel, B. Genome architecture: domain organization of interphase chromosomes. Cell 152, 1270–1284 (2013)."),[6](https://www.nature.com/articles/s41588-025-02297-w#ref-CR6 "Van Steensel, B. & Furlong, E. E. M. The role of transcription in shaping the spatial organization of the genome. Nat. Rev. Mol. Cell Biol. 20, 327–337 (2019)."),[7](https://www.nature.com/articles/s41588-025-02297-w#ref-CR7 "Bonora, G., Plath, K. & Denholtz, M. A mechanistic link between gene regulation and genome architecture in mammalian development. Curr. Opin. Genet. Dev. 27, 92–101 (2014)."),[8](https://www.nature.com/articles/s41588-025-02297-w#ref-CR8 "Mir, M., Bickmore, W., Furlong, E. E. M. & Narlikar, G. Chromatin topology, condensates and gene regulation: shifting paradigms or just a phase? Development 146, dev182766 (2019)."),[9](https://www.nature.com/articles/s41588-025-02297-w#ref-CR9 "Misteli, T. The self-organizing genome: principles of genome architecture and function. Cell 183, 28–45 (2020)."). Here we present genome-wide, in situ single-cell cancer 3D genome atlases, generated using an imaging-based 3D genomics method that we have pioneered—chromatin tracing[10](https://www.nature.com/articles/s41588-025-02297-w#ref-CR10 "Hu, M. & Wang, S. Chromatin tracing: imaging 3D genome and nucleome. Trends Cell Biol. 31, 5–8 (2021)."),[11](https://www.nature.com/articles/s41588-025-02297-w#ref-CR11 "Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. Science 353, 598–602 (2016)."). Unlike sequencing-based 3D genomics technologies (for example, high-throughput chromosome conformation capture (Hi-C)) that relied on population averaging of cells and indirect inference of altered genome organization in averaged cancer cells[12](https://www.nature.com/articles/s41588-025-02297-w#ref-CR12 "Hnisz, D. et al. Activation of proto-oncogenes by disruption of chromosome neighborhoods. Science 351, 1454–1458 (2016)."),[13](https://www.nature.com/articles/s41588-025-02297-w#ref-CR13 "Flavahan, W. A. et al. Insulator dysfunction and oncogene activation in IDH mutant gliomas. Nature 529, 110–114 (2016)."),[14](https://www.nature.com/articles/s41588-025-02297-w#ref-CR14 "Flavahan, W. A. et al. Altered chromosomal topology drives oncogenic programs in SDH-deficient GISTs. Nature 575, 229–233 (2019)."),[15](https://www.nature.com/articles/s41588-025-02297-w#ref-CR15 "Johnstone, S. E. et al. Large-scale topological changes restrain malignant progression in colorectal cancer. Cell 182, 1474–1489 (2020)."),[16](https://www.nature.com/articles/s41588-025-02297-w#ref-CR16 "Xu, J. et al. Subtype-specific 3D genome alteration in acute myeloid leukaemia. Nature 611, 387–398 (2022)."),[17](https://www.nature.com/articles/s41588-025-02297-w#ref-CR17 "Mathur, R. et al. Glioblastoma evolution and heterogeneity from a 3D whole-tumor perspective. Cell 187, 446–463 (2024)."), our approach captures 3D genome folding directly in individual cells in the native tissue environment in vivo during cancer progression from normal to preinvasive and to invasive tumor cells. Our results also define stage-specific alterations in 3D genome architectures during lung adenocarcinoma (LUAD) and pancreatic ductal adenocarcinoma (PDAC) progression and illustrate how 3D genome evolution can nominate genes that govern prognosis, delineate cancer cell dependency and regulate the 3D genome. Our findings and methodologies offer a unique resource by which in situ single-cell 3D genome information helps to identify potential diagnostic, prognostic and therapeutic cancer biomarkers.

Results
-------

### Genome-wide chromatin tracing in mouse LUAD

To directly visualize chromatin folding in single cells within tissues, we performed genome-wide chromatin tracing targeting 473 genomic loci spanning all 19 mouse autosomes at an average genomic interval of 5 megabases (Mb) (Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02297-w#MOESM3)–[3](https://www.nature.com/articles/s41588-025-02297-w#MOESM3)). These genomic regions contained ‘classic’ oncogenes, tumor suppressors and super-enhancers. We adapted a DNA multiplexed error-robust fluorescence in situ hybridization (DNA MERFISH) 100-choose-2 combinatorial barcoding design[18](https://www.nature.com/articles/s41588-025-02297-w#ref-CR18 "Su, J.-H., Zheng, P., Kinrot, S. S., Bintu, B. & Zhuang, X. Genome-scale imaging of the 3D organization and transcriptional activity of chromatin. Cell 182, 1641–1659 (2020).") as follows: each locus was assigned a unique 100-bit binary barcode with two ‘1’ bits and 98 ‘0’ bits[18](https://www.nature.com/articles/s41588-025-02297-w#ref-CR18 "Su, J.-H., Zheng, P., Kinrot, S. S., Bintu, B. & Zhuang, X. Genome-scale imaging of the 3D organization and transcriptional activity of chromatin. Cell 182, 1641–1659 (2020)."); each ‘1’ bit physically corresponded to a unique overhang readout sequence on primary probes hybridized to the genome (Fig. [1a](https://www.nature.com/articles/s41588-025-02297-w#Fig1)). The primary probes targeting each genomic locus contained two versions of readout sequences that corresponded to the two ‘1’ bits in the barcode of the locus. After hybridizing all the primary probes, we sequentially hybridized dye-labeled readout probes to image the 100 bits, with 50 rounds of two-color imaging (Fig. [1a](https://www.nature.com/articles/s41588-025-02297-w#Fig1)). We then fitted the 3D positions of the foci imaged by fluorescence in situ hybridization (FISH), and decoded the barcodes and reconstructed chromatin traces in single cells (Fig. [1b,c](https://www.nature.com/articles/s41588-025-02297-w#Fig1)). To validate the quality of genome-wide chromatin tracing, we also measured the mean spatial distance among each pair of target genomic loci and constructed a mean interloci distance matrix (Fig. [1d](https://www.nature.com/articles/s41588-025-02297-w#Fig1)). We first performed this in lung alveolar type 2 (AT2) cells—the putative cell of origin for LUAD[19](https://www.nature.com/articles/s41588-025-02297-w#ref-CR19 "Ferone, G., Lee, M. C., Sage, J. & Berns, A. Cells of origin of lung cancers: lessons from mouse studies. Genes Dev. 34, 1017–1032 (2020)."),[20](https://www.nature.com/articles/s41588-025-02297-w#ref-CR20 "Kaiser, A. M. et al. p53 governs an AT1 differentiation programme in lung cancer suppression. Nature 619, 851–859 (2023)."),[21](https://www.nature.com/articles/s41588-025-02297-w#ref-CR21 "Han, G. et al. An atlas of epithelial cell states and plasticity in lung adenocarcinoma. Nature 627, 656–663 (2024).")—in wild-type (WT) mice. The resulting matrix featured lower intrachromosomal distances and higher interchromosomal distances, consistent with chromosome territory organization[22](https://www.nature.com/articles/s41588-025-02297-w#ref-CR22 "Cremer, T. & Cremer, C. Chromosome territories, nuclear architecture and gene regulation in mammalian cells. Nat. Rev. Genet. 2, 292–301 (2001)."). Notably, intrachromosomal interloci distances measured from WT AT2 biological replicates were highly reproducible (Extended Data Fig. [1a,b](https://www.nature.com/articles/s41588-025-02297-w#Fig7)). Within each chromosome, interloci spatial distances scaled with expected genomic distances, with longer chromosomes reaching larger spatial distances (Extended Data Fig. [1c](https://www.nature.com/articles/s41588-025-02297-w#Fig7)). On average, intrachromosomal interloci spatial distances followed a power-law function versus genomic distances, with a scaling factor of about one-tenth (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02297-w#Fig7)), in line with prior studies (Extended Data Fig. [1e,f](https://www.nature.com/articles/s41588-025-02297-w#Fig7))[23](https://www.nature.com/articles/s41588-025-02297-w#ref-CR23 "Takei, Y. et al. Single-cell nuclear architecture across cell types in the mouse brain. Science 374, 586–594 (2021)."),[24](https://www.nature.com/articles/s41588-025-02297-w#ref-CR24 "Liu, M. et al. Multiplexed imaging of nucleome architectures in single cells of mammalian tissue. Nat. Commun. 11, 2907 (2020).").

**Fig. 1: A genome-scale chromatin tracing strategy to visualize cancer 3D genomes.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02297-w/figures/1)

**a**, Schematic illustration of the experimental procedure. **b**, Raw FISH foci (left) in bit 28 in a WT AT2 cell. Zoom-in images (right) of raw FISH foci showing the decoding procedure and reconstructed genomic loci for five loci (shown as dots with five pseudocolors each appearing twice). Representative results from five biological replicates (_n_ = 4 mice) are shown in **b** and **c**. **c**, Reconstructed chromatin traces (left) superimposed with DAPI staining. The traces are 2D projections of _x_ and _y_ coordinates. The DAPI image is a maximum-intensity _z_ projection from a 10-μm z stack. The 3D positions (right) of all decoded genomic loci in a single-cell nucleus. Different pseudocolors represent different autosomes. **d**, Matrix of mean interloci distances between all genomic loci in AT2 cells. _n_ = 4,806 AT2 cells. **e**, Immunofluorescence staining (top) of cell-type markers, DAPI staining and fluorescent protein imaging of lung tissue from a WT mouse, a _K-MADM-Trp53_ mouse with adenomas and a _K-MADM-Trp53_ mouse with LUAD. Representative results from five biological replicates in WT (_n_ = 4 mice) and adenoma (_n_ = 4 mice) and six biological replicates in LUAD (_n_ = 5 mice) states are shown. Representative cells (bottom) of each state. The images are maximum-intensity _z_ projections from 10-μm z stacks. Chr, chromosome.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM4)

Having established the approach in normal tissues, we next performed chromatin tracing on the _K-MADM-Trp53_ mouse lung cancer model (Fig. [1a](https://www.nature.com/articles/s41588-025-02297-w#Fig1) and Extended Data Fig. [1g,h](https://www.nature.com/articles/s41588-025-02297-w#Fig7)) that induces sparse and sequential _Kras_ and _Trp53_ mutations in the lung epithelium to mimic the faithful genetic and histologic progression of human LUAD[25](https://www.nature.com/articles/s41588-025-02297-w#ref-CR25 "The Cancer Genome Atlas Research Network. Comprehensive molecular profiling of lung adenocarcinoma. Nature 511, 543–550 (2014)."),[26](https://www.nature.com/articles/s41588-025-02297-w#ref-CR26 "Slebos, R. J. C. et al. p53 alterations in atypical alveolar hyperplasia of the human lung. Hum. Pathol. 29, 801–808 (1998)."),[27](https://www.nature.com/articles/s41588-025-02297-w#ref-CR27 "Vogelstein, B. et al. Cancer genome landscapes. Science 339, 1546–1558 (2013)."). Specifically, we leveraged the mosaic analysis with double markers (MADM) system[28](https://www.nature.com/articles/s41588-025-02297-w#ref-CR28 "Zong, H., Espinosa, J. S., Su, H. H., Muzumdar, M. D. & Luo, L. Mosaic analysis with double markers in mice. Cell 121, 479–492 (2005).") to unambiguously link genotypes with fluorescence labeling[28](https://www.nature.com/articles/s41588-025-02297-w#ref-CR28 "Zong, H., Espinosa, J. S., Su, H. H., Muzumdar, M. D. & Luo, L. Mosaic analysis with double markers in mice. Cell 121, 479–492 (2005)."),[29](https://www.nature.com/articles/s41588-025-02297-w#ref-CR29 "Muzumdar, M. D., Luo, L. & Zong, H. Modeling sporadic loss of heterozygosity in mice by using mosaic analysis with double markers (MADM). Proc. Natl Acad. Sci. USA 104, 4495–4500 (2007)."). Subclones of green (green fluorescent protein, GFP+) _Trp53_−/−, red (tdTomato+) _Trp53__+/+_ and yellow (GFP+/tdTomato+) _Trp53__+/−_ cells can be traced in preinvasive _Kras_ mutant lung adenomas initiated by inhaled lentiviral Cre infection[30](https://www.nature.com/articles/s41588-025-02297-w#ref-CR30 "Muzumdar, M. D. et al. Clonal dynamics following p53 loss of heterozygosity in Kras-driven cancers. Nat. Commun. 7, 12685 (2016)."). Only _Trp53_−/−, and not _Trp53__+/+_ or _Trp53__+/−_ adenoma cells, progressed to advanced LUAD[30](https://www.nature.com/articles/s41588-025-02297-w#ref-CR30 "Muzumdar, M. D. et al. Clonal dynamics following p53 loss of heterozygosity in Kras-driven cancers. Nat. Commun. 7, 12685 (2016)."), enabling in vivo tracking of morphologically distinct stages from AT2 to preinvasive adenoma to LUAD. We combined multiplexed fluorescence imaging of GFP, tdTomato, surfactant protein C (SPC) and CD45 with genome-wide chromatin tracing on the same tissue section to achieve cancer state-specific single-cell 3D genome profiling (Extended Data Fig. [1g,h](https://www.nature.com/articles/s41588-025-02297-w#Fig7)). SPC marks AT2 and cancer cells, whereas the pan-immune marker CD45 allows us to exclude macrophages, which are labeled alongside cancer cells by the MADM system (presumably through lentiviral Cre infection; Fig. [1e](https://www.nature.com/articles/s41588-025-02297-w#Fig1)).

To build a complete 3D genome atlas capturing the full spectrum of tumor development, we analyzed 26,852 cells from 16 replicates across multiple lung lobes (most containing several discrete tumors) of 13 mice harboring all stages of LUAD progression which are as follows: AT2, AdenomaR (red _Trp53__+/+_), AdenomaY (yellow _Trp53__+/−_), AdenomaG (green _Trp53_−/−) and LUAD cells. The detection efficiency of each target region and chromosome trace length distribution were consistent across cell states (Extended Data Fig. [1i,j](https://www.nature.com/articles/s41588-025-02297-w#Fig7)). Overall, we detected a median of 115 loci per cell (Extended Data Fig. [1k](https://www.nature.com/articles/s41588-025-02297-w#Fig7)). A (active) and B (inactive) compartment assignments from chromatin tracing correlated with those from bulk Hi-C of LUAD tumors (Extended Data Fig. [1l,m](https://www.nature.com/articles/s41588-025-02297-w#Fig7)). Together, these metrics validated the utility of chromatin tracing to compare chromatin conformations across cell states during LUAD progression.

### A 3D genome bottleneck in lung adenoma

To study how the 3D genome evolves during LUAD progression, we first quantified intrachromosomal interloci distances to assess chromatin compaction in each cell state (AT2, AdenomaR, AdenomaY, AdenomaG and LUAD). Intrachromosomal compaction globally increased during early progression from AT2 to preinvasive adenoma cells at the population level and recovered with the transition to LUAD (Fig. [2a–c](https://www.nature.com/articles/s41588-025-02297-w#Fig2) and Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02297-w#Fig8)). Approximately 80% of locus pairs followed this nonmonotonic trend (Fig. [2d](https://www.nature.com/articles/s41588-025-02297-w#Fig2)). As an orthogonal approach, chromosome paint of mouse chromosome 19 showed consistently greater whole chromosome compaction in adenoma cells compared to AT2 and LUAD cells (Fig. [2e](https://www.nature.com/articles/s41588-025-02297-w#Fig2)). Strikingly, single-cell comparisons of chromatin traces revealed that chromatin conformations in adenoma cells were globally less heterogeneous than WT AT2 and LUAD cells (Fig. [2a,f,g](https://www.nature.com/articles/s41588-025-02297-w#Fig2) and Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02297-w#Fig8)) with more than 60% locus pairs exhibiting this trend (Fig. [2h](https://www.nature.com/articles/s41588-025-02297-w#Fig2)). Notably, diminished heterogeneity in the adenoma state was observed both within individual mice (for AT2 cells) or tumors (for adenoma and LUAD) and across mice (Fig. [2i,j](https://www.nature.com/articles/s41588-025-02297-w#Fig2)), arguing for phenotypic convergence of chromatin compaction during early lung cancer progression. Furthermore, the recovery of heterogeneity and especially compaction in LUAD cells diverged from the AT2 state for many locus pairs (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02297-w#Fig9)), suggesting that locus-specific 3D genomic changes may confer a selective advantage in LUAD progression.

**Fig. 2: Systematic changes of 3D genome compaction and heterogeneity during lung cancer progression.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02297-w/figures/2)

**a**, Mean interloci distances (top) and COV of interloci distances (bottom) of mouse chr13 in WT AT2 (_n_ = 4,806), _Trp53__+/+_ adenoma (AdenomaR, _n_ = 791), _Trp53__+/−_ adenoma (AdenomaY, _n_ = 1,603), _Trp53__−/−_ adenoma (AdenomaG, _n_ = 1,941) and _Trp53__−/−_ LUAD (_n_ = 17,711) cells. Cell numbers are identical in **a**–**j**. **b**, Distributions of decompaction (mean interloci distance) scores of each autosome (_n_ = 19 autosomes) in each cell state. _P_ values from two-sided Wilcoxon signed-rank tests versus AT2 with matched genomic loci are displayed. **c**, Mean log2(FC) of decompaction scores of each chromosome, comparing each cancer state to AT2. An asterisk indicates FDR < 0.05, two-sided Wilcoxon signed-rank test. **d**, Clustered heatmap of _z_ scores (scaled by column) of _cis_\-chromosomal mean interloci distances. Percentages of locus pairs with different decompaction change patterns are shown below. **e**, Chr19 territory size measured by whole chromosome paint. _P_ values from one-sided Wilcoxon rank-sum tests are shown. **f**, Distributions of heterogeneity (COV of interloci distance) scores of each autosome (_n_ = 19) in each cell state. _P_ values from two-sided Wilcoxon signed-rank tests versus AT2 with matched genomic loci are displayed. **g**, Mean log2(FC) of heterogeneity scores of each chromosome, comparing each cancer state to AT2. An asterisk indicates FDR < 0.05, two-sided Wilcoxon signed-rank test. **h**, Clustered heatmap of _z_ scores (scaled by column) of COV of _cis_\-chromosomal interloci distances. Percentages of locus pairs with different chromatin folding heterogeneity change patterns are shown below. **i**,**j**, Distributions of 3D genome heterogeneity scores of autosomes (_n_ = 19 autosomes) for cell pairs from the same mice (AT2, _n_ = 4 mice) or same tumors (12 adenoma tumors and 6 LUAD tumors, intramouse or intratumoral; **i**) or from different mice (intermouse or intertumoral; **j**). _P_ values from two-sided Wilcoxon signed-rank tests versus AT2 with matched autosomes are shown. In **b**, **e**, **f**, **i** and **j**, horizontal lines of each box represent the 75th percentile, median and 25th percentile. Whiskers indicate nonoutlier maximum and nonoutlier minimum, with outlier thresholds at 1.5× interquartile range beyond box limits. FC, fold change.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM5)

To further characterize 3D genome evolution during LUAD progression, we quantified several additional structural features of chromosomes across cell states, including: (1) long-range intermixing (calculated using a demixing score metric benchmarked using active and inactive X-chromosome folding conformations in human cells[11](https://www.nature.com/articles/s41588-025-02297-w#ref-CR11 "Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. Science 353, 598–602 (2016).")) (Fig. [3a](https://www.nature.com/articles/s41588-025-02297-w#Fig3)), (2) A–B compartment polarization (quantified using a polarization index metric[11](https://www.nature.com/articles/s41588-025-02297-w#ref-CR11 "Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. Science 353, 598–602 (2016).")), (3) interchromosomal distance and (4) radial localization of each genomic locus in the nucleus (Methods). _Reduced_ long-range intermixing (Fig. [3b,c](https://www.nature.com/articles/s41588-025-02297-w#Fig3)) and _increased_ A–B compartment polarization (Fig. [3d](https://www.nature.com/articles/s41588-025-02297-w#Fig3)) were seen for all adenoma cell states compared to AT2 cells. Moreover, chromosomes in adenoma states were more spatially associated (reduced interchromosomal distance; Fig. [3e,f](https://www.nature.com/articles/s41588-025-02297-w#Fig3)), with a subset (for example, chromosome 7, chromosome 13 and chromosome 19) showing consistent changes in radial distribution within the nucleus compared to AT2 cells (Fig. [3g,h](https://www.nature.com/articles/s41588-025-02297-w#Fig3)). Similar to chromosome compaction, these phenotypes largely recovered in LUAD cells (Figs. [2](https://www.nature.com/articles/s41588-025-02297-w#Fig2) and [3](https://www.nature.com/articles/s41588-025-02297-w#Fig3)). Notably, interchromosomal distances in LUAD largely surpassed those of AT2 cells (Fig. [3e,i](https://www.nature.com/articles/s41588-025-02297-w#Fig3)), leading to an expected larger overall nuclear volume in LUAD than in AT2 and adenoma cells (Fig. [3j](https://www.nature.com/articles/s41588-025-02297-w#Fig3)). Together, these data demonstrated stereotypical, nonmonotonic and stage-specific 3D genome conformations during lung cancer progression, which could represent a structural bottleneck for tumor development.

**Fig. 3: Changes of cancer state-specific 3D genome features during lung cancer progression.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02297-w/figures/3)

**a**, Demixing scores (s.d. of normalized mean interloci distances) of the active chrX (X_a_, _n_ = 95) and inactive chrX (X_i_, _n_ = 95) in human IMR90 cells show a reduction (increased intermixing) in X_i_, as previously described using other analyses[11](https://www.nature.com/articles/s41588-025-02297-w#ref-CR11 "Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. Science 353, 598–602 (2016)."). _P_ value from two-sided Levene’s test is shown. **b**, Distributions of demixing scores of each autosome (_n_ = 19) in each cell state. _n_ = 4,806, 791, 1,603, 1,941 and 17,711 for WT AT2, AdenomaR, AdenomaY, AdenomaG and LUAD cells, respectively. Cell numbers of each cell state are identical in **b**–**i**. **c**, log2(FC) of the demixing score of each autosome, comparing each cancer state to WT AT2 cell state. An asterisk indicates FDR < 0.05, two-sided Levene’s test. **d**, Distribution of polarization indices of each autosome (_n_ = 19) in each cell state. **e**, Distribution of interchromosomal distances in each cell state. **f**, log2(FC) of mean interloci distances for AdenomaR, AdenomaY and AdenomaG relative to AT2 cells. Yellow lines highlight the boundaries of chromosomes. **g**, Distributions of radial scores of chr13, chr7 and chr19 in each cell state. **h**, Mean log2(FC) of radial scores of each chromosome, comparing each cancer state to the AT2 cell state. An asterisk indicates FDR < 0.05, two-sided Wilcoxon signed-rank test. **i**, log2(FC) of mean interloci distances for LUAD relative to AT2 cells. Yellow lines highlight the boundaries of chromosomes. **j**, Nuclear convex hull volume of WT AT2, AdenomaR, AdenomaY, AdenomaG and LUAD cells. _P_ values of two-sided Wilcoxon rank-sum tests are shown. _n_ = 3,410, 157, 689, 878 and 8,834 for WT AT2, AdenomaR, AdenomaG, AdenomaY and LUAD cells, respectively (cells with at least 10 traces). In **b**, **d**, **e** and **g**, _P_ values of two-sided Wilcoxon signed-rank test comparing each cancer state to WT AT2 state with matched autosomes (**b**,**d**) or matched genomic loci (**e**,**g**) are displayed. In **b**, **d**, **e**, **g** and **j**, horizontal lines of each box represent the 75th percentile, median and 25th percentile. Whiskers indicate nonoutlier maximum and nonoutlier minimum, with outlier thresholds at 1.5× interquartile range beyond box limits.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM6)

Besides the chromatin conformation alterations in cancer evolution, we also identified several cancer state-independent 3D genomic features. For example, spatial proximity frequencies were the highest for _trans_\-chromosomal A–A compartment interactions, followed by A–B and B–B interactions (Extended Data Fig. [4a–i](https://www.nature.com/articles/s41588-025-02297-w#Fig10)). The same trend was also observed in long-range _cis_\-chromosomal compartment interactions (Extended Data Fig. [4j–l](https://www.nature.com/articles/s41588-025-02297-w#Fig10)). Furthermore, we identified a consistent negative correlation between radial scores and A–B compartment scores (Extended Data Fig. [4m](https://www.nature.com/articles/s41588-025-02297-w#Fig10)). These observations align with previously reported findings in mouse fetal livers and human lung fibroblasts[18](https://www.nature.com/articles/s41588-025-02297-w#ref-CR18 "Su, J.-H., Zheng, P., Kinrot, S. S., Bintu, B. & Zhuang, X. Genome-scale imaging of the 3D organization and transcriptional activity of chromatin. Cell 182, 1641–1659 (2020)."),[24](https://www.nature.com/articles/s41588-025-02297-w#ref-CR24 "Liu, M. et al. Multiplexed imaging of nucleome architectures in single cells of mammalian tissue. Nat. Commun. 11, 2907 (2020)."), arguing that they may represent general principles of 3D genome organization independent of cell state, cell type and even mammalian species.

To determine whether the nonmonotonic 3D genome changes are driven by genetic alterations, we performed whole-exome sequencing and analyzed single nucleotide variants (SNVs) and copy number variants (CNVs) in isolated GFP+ _Kras__G12D_/_Trp53__−/−_ adenomas and LUAD from _K-MADM-Trp53_ mice (Supplementary Table [4](https://www.nature.com/articles/s41588-025-02297-w#MOESM3) and Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02297-w#Fig11)). Few SNVs were observed in both adenoma and LUAD tumors, none of which resulted in protein-altering mutations in known cancer driver genes[31](https://www.nature.com/articles/s41588-025-02297-w#ref-CR31 "Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. Cell 173, 371–385 (2018).") (Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02297-w#Fig11)), as this has been previously described in the conventional _Kras__LSL-G12D/WT__; Trp53__flox/flox_ (KP) LUAD model[32](https://www.nature.com/articles/s41588-025-02297-w#ref-CR32 "McFadden, D. G. et al. Mutational landscape of EGFR-, MYC-, and Kras-driven genetically engineered mouse models of lung adenocarcinoma. Proc. Natl Acad. Sci. USA 113, E6409–E6417 (2016)."). Furthermore, adenoma cells did not harbor recurrent CNVs (Extended Data Fig. [5c,d](https://www.nature.com/articles/s41588-025-02297-w#Fig11)), suggesting that the observed phenotypic convergence in chromatin compaction (Fig. [2i,j](https://www.nature.com/articles/s41588-025-02297-w#Fig2)) was not triggered by additional genetic evolution beyond oncogenic Kras expression as a tumor-initiating event. In contrast, a subset of LUAD tumors exhibited gains of chromosome 6 and losses of chromosomes 4, 9 and 11 (Extended Data Fig. [5c,d](https://www.nature.com/articles/s41588-025-02297-w#Fig11)), also consistent with the KP LUAD model[32](https://www.nature.com/articles/s41588-025-02297-w#ref-CR32 "McFadden, D. G. et al. Mutational landscape of EGFR-, MYC-, and Kras-driven genetically engineered mouse models of lung adenocarcinoma. Proc. Natl Acad. Sci. USA 113, E6409–E6417 (2016)."). As an orthogonal approach, single-cell CNVs inferred from single-nucleus RNA sequencing (snRNA-seq) of adenoma and LUAD (Extended Data Fig. [5e–g](https://www.nature.com/articles/s41588-025-02297-w#Fig11)) showed increased copy number heterogeneity in LUAD in comparison to adenoma (Extended Data Fig. [5h](https://www.nature.com/articles/s41588-025-02297-w#Fig11)). These copy number alterations may partially explain the divergent recovery in heterogeneity and decompaction observed in LUAD cells relative to AT2 cells. However, the lack of systematic gains or losses across most chromosomes and tumors suggests that CNVs cannot account for the convergent global chromatin structure differences between adenoma and LUAD. Collectively, these data argue that the observed 3D genome alterations are not merely a surrogate of mutational evolution in this model.

### Conserved 3D genome reorganization in PDAC progression

To study whether the global 3D genome changes in LUAD progression are conserved in another Kras-driven cancer type, we leveraged an analogous _K-MADM-Trp53_ model of PDAC, which directs Cre recombinase activity to the pancreas using a _Pdx1-cre_ transgene and recapitulates major genetic and histologic events in human PDAC progression[30](https://www.nature.com/articles/s41588-025-02297-w#ref-CR30 "Muzumdar, M. D. et al. Clonal dynamics following p53 loss of heterozygosity in Kras-driven cancers. Nat. Commun. 7, 12685 (2016)."),[33](https://www.nature.com/articles/s41588-025-02297-w#ref-CR33 "Raphael, B. J. et al. Integrated genomic characterization of pancreatic ductal adenocarcinoma. Cancer Cell 32, 185–203 (2017)."),[34](https://www.nature.com/articles/s41588-025-02297-w#ref-CR34 "Australian Pancreatic Cancer Genome Initiative et al. Genomic analyses identify molecular subtypes of pancreatic cancer. Nature 531, 47–52 (2016)."). We performed genome-wide chromatin tracing targeting the same 473 genomic loci combined with cytokeratin 19 (a marker for normal and malignant duct cells) immunofluorescence, GFP, tdTomato and DAPI imaging in the same single cells. We analyzed 2,677 cells across PDAC progression stages, including normal duct cells, preinvasive pancreatic intraepithelial neoplasia (PanIN) cells (R, Y, G) and invasive PDAC cells (Extended Data Fig. [6a–d](https://www.nature.com/articles/s41588-025-02297-w#Fig12)). Mean interloci distances showed the expected chromosome territory features and genomic distance scaling (Extended Data Fig. [6e,f](https://www.nature.com/articles/s41588-025-02297-w#Fig12)), indicating high-quality traces. Analyses of key chromatin folding features largely recapitulated findings in LUAD progression, including (1) intrachromosomal compaction, (2) intrachromosomal heterogeneity, (3) long-range intermixing, (4) interchromosomal distance and (5) radial localization of genomic loci in the nucleus (Extended Data Fig. [6g–k](https://www.nature.com/articles/s41588-025-02297-w#Fig12)). When comparing PanIN to WT duct cells, we observed that chromosomes largely exhibited more compact folding (especially in PanIN R), decreased heterogeneity, reduced long-range intermixing, decreased interchromosomal distances and changes in radial distribution for a subset of chromosomes (for example, chromosome 1, chromosome 17 and chromosome 19). These phenotypes at least partially recovered upon progression to PDAC. Thus, stereotypical, nonmonotonic and stage-specific changes in 3D genome conformations seen in PDAC progression mirrored those in LUAD, arguing that the 3D genome changes were not limited to a single Kras-driven cancer type.

### Single-cell 3D genome distinguishes cancer cell states

We next determined whether different morphologic cancer cell states can be classified with only single-cell 3D genome data. Using subsampling analysis, we first established that as few as 100 cells per cell state are sufficient to recapitulate most 3D genome changes described above (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02297-w#Fig13)). We then leveraged a ‘Trace2State’ pipeline that performs dimensionality reduction of single-cell 3D genome conformation data to visualize clusters of different morphologic cell states. We adopted a previously developed single-cell A/B (scA/B) compartment score metric for each target genomic locus (Fig. [4a](https://www.nature.com/articles/s41588-025-02297-w#Fig4))[35](https://www.nature.com/articles/s41588-025-02297-w#ref-CR35 "Tan, L. et al. Changes in genome architecture and transcriptional dynamics progress independently of sensory experience during post-natal brain development. Cell 184, 741–758 (2021)."),[36](https://www.nature.com/articles/s41588-025-02297-w#ref-CR36 "Tan, L., Xing, D., Chang, C.-H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. Science 361, 924–928 (2018)."), which is defined as the mean A–B compartment score of all its spatially adjacent loci (<1200-nm radius) and reflects the A–B compartment identity of each locus at the single-cell level. We used these scores as high-dimensional input variables to generate a 2D display using three independent dimensionality reduction methods[37](https://www.nature.com/articles/s41588-025-02297-w#ref-CR37 "Van der Maaten, L. & Hinton, G. Visualizing data using t-SNE. J. Mach. Learn. Res. 9, 2579–2605 (2008)."),[38](https://www.nature.com/articles/s41588-025-02297-w#ref-CR38 "McInnes, L., Healy, J., Saul, N. & Großberger, L. UMAP: uniform manifold approximation and projection. J. Open Source Softw. 3, 861 (2018)."),[39](https://www.nature.com/articles/s41588-025-02297-w#ref-CR39 "Wang, Y., Huang, H., Rudin, C. & Shaposhnik, Y. Understanding how dimension reduction tools work: an empirical approach to deciphering t-SNE, UMAP, TriMAP, and PaCMAP for data visualization. J. Mach. Learn. Res. 22, 9129–9201 (2021)."). Single-cell 3D genome conformations of AT2 and LUAD cells were largely distinct, whereas those of adenoma cells were clustered between them (Fig. [4a](https://www.nature.com/articles/s41588-025-02297-w#Fig4)), consistent with morphologic progression. Supervised machine learning models trained with the scA/B profiles classified different cancer cell states (Methods), with a support vector machine model achieving 90% overall accuracy in predicting these states (Fig. [4b,c](https://www.nature.com/articles/s41588-025-02297-w#Fig4)). Similar results were obtained for cell states during PDAC progression (Fig. [4d–f](https://www.nature.com/articles/s41588-025-02297-w#Fig4)). These results support the utility of compartment-level 3D genome organization in distinguishing morphological cell states during oncogenesis at the single-cell level.

**Fig. 4: The single-cell 3D genome distinguishes and encodes cancer progression states.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02297-w/figures/4)

**a**, Cartoon illustration of scA/B score calculation (top-left). t-SNE (top-right), UMAP (bottom-left) and PaCMAP (bottom-right) plots of single-cell 3D genome conformations. _n_ = 3,410, 157, 689, 878 and 8,834 for WT AT2, AdenomaR, AdenomaG, AdenomaY and LUAD cells, respectively. Cell numbers of each cell state are identical in **a**–**c**, **g** and **h**. **b**, Confusion matrix of supervised machine learning in mouse lung cells. The number in each matrix element represents the precision in each predicted state. **c**, ROC curves of the machine learning model in mouse lung cells. The AUC values are shown. **d**, PCA plot of single-cell 3D genome conformations. _n_ = 1,103, 191 and 268 for normal duct, PanIN and PDAC cells, respectively. Cell numbers of each cell state are identical in **d**–**f**. **e**, Confusion matrix of supervised machine learning in mouse pancreas cells. The number in each matrix element represents the precision in each predicted state. **f**, ROC curves of the machine learning model in mouse pancreas cells. The AUC values are shown. **g**, PCA plot of single-cell 3D genome conformations of adenoma and LUAD cells (left). Leiden clustering separates Adenoma-like and LUAD-like clusters (right). **h**, Percentages of cells with adenoma-like or LUAD-like 3D genome conformations in **g**, in each of the AdenomaR/AdenomaY, AdenomaG and LUAD states. The adenoma-like or LUAD-like conformation state for each cell is assigned based on a Leiden clustering approach. _P_ values from two-sided Fisher’s exact test are shown. t-SNE, _t_\-distributed stochastic neighbor embedding; UMAP, uniform manifold approximation and projection; PaCMAP, pairwise controlled manifold approximation; ROC, receiver operating characteristic; AUC, area under the curve.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM7)

We hypothesized that the subset of _Trp53__−/−_ AdenomaG cells progressing to LUAD has acquired specific 3D genome features. Indeed, AdenomaG cells often exhibited chromosome-scale structural features intermediate between those of nonprogressing AdenomaR/AdenomaY cells and LUAD cells (Figs. [2](https://www.nature.com/articles/s41588-025-02297-w#Fig2) and [3](https://www.nature.com/articles/s41588-025-02297-w#Fig3)). Furthermore, 3D genome dimensionality reduction analyses showed that AdenomaR/AdenomaY cells and LUAD cells formed distinct but overlapping clusters, with AdenomaG cells dispersed within and between them (Fig. [4g](https://www.nature.com/articles/s41588-025-02297-w#Fig4)). Leiden clustering[40](https://www.nature.com/articles/s41588-025-02297-w#ref-CR40 "Traag, V. A., Waltman, L. & van Eck, N. J. From Louvain to Leiden: guaranteeing well-connected communities. Sci. Rep. 9, 5233 (2019).") segmented the low-dimensional space into adenoma-like and LUAD-like regions and revealed that more AdenomaG cells (versus other adenoma cells) adopted LUAD-like 3D conformations (Fig. [4g,h](https://www.nature.com/articles/s41588-025-02297-w#Fig4)), indicating that 3D chromatin organization changes associated with LUAD precede morphologic progression.

### 3D genomics delineates prognostic and predictive biomarkers

Because AdenomaG cells were more likely to adopt LUAD-like compartment organization than nonprogressing cells (AdenomaR/AdenomaY), we examined how compartment transitions between AdenomaG and LUAD relate to gene expression. We reasoned that compartment alterations could promote stable gene activation or repression with functional consequences, due to differences in active–repressive epigenetic signatures, RNA polymerase II (Pol II) and transcription factor densities in these compartments[41](https://www.nature.com/articles/s41588-025-02297-w#ref-CR41 "Willemin, A., Szabó, D. & Pombo, A. Epigenetic regulatory layers in the 3D nucleus. Mol. Cell 84, 415–428 (2024)."). Therefore, we postulated that genes with increased expression in genomic loci with increased scA/B scores in LUAD cells are important for LUAD progression, which we termed candidate progression drivers (CPDs). We further hypothesized that these CPDs associated with compartment changes are more homogenously expressed than genes upregulated by compartment-independent mechanisms and—similar to genes with diminished intratumoral expression heterogeneity in human lung cancer[42](https://www.nature.com/articles/s41588-025-02297-w#ref-CR42 "Martínez-Ruiz, C. et al. Genomic–transcriptomic evolution in lung cancer and metastasis. Nature 616, 543–552 (2023).")—more likely to undergo positive selection during tumor progression.

To link 3D genome alterations with gene expression during LUAD progression, we first performed bulk RNA-seq on GFP+ tumors of _K-MADM-Trp53_ mice (Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02297-w#Fig11) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02297-w#MOESM3)). Unsupervised hierarchical clustering revealed two distinct tumor groups (Extended Data Fig. [5i](https://www.nature.com/articles/s41588-025-02297-w#Fig11)) corresponding to AdenomaG and LUAD cells based on expected changes in gene expression with progression (Extended Data Fig. [5j,k](https://www.nature.com/articles/s41588-025-02297-w#Fig11))[43](https://www.nature.com/articles/s41588-025-02297-w#ref-CR43 "Marjanovic, N. D. et al. Emergence of a high-plasticity cell state during lung cancer evolution. Cancer Cell 38, 229–246 (2020)."). Genes upregulated or downregulated in LUAD (versus AdenomaG) showed increased or decreased scA/B scores, respectively (Fig. [5a](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). On the contrary, no substantial associations were identified between gene expression changes and scA/B score changes by randomly subsampling two groups of cells (each with equal size as AdenomaG or LUAD) for scA/B score changes (_P_ = 0.46, downregulated genes; _P_ = 0.66, upregulated genes). We next developed a ‘Trace2Biomarker’ pipeline to call CPDs (genes with concurrently increased scA/B score and gene expression in LUAD, Fig. [5b](https://www.nature.com/articles/s41588-025-02297-w#Fig5); Methods). To test the hypothesis that A–B compartment organization contributes to more stable gene expression regulation, we quantified gene expression heterogeneity/homogeneity in LUAD cells derived from _K-MADM-Trp53_ mice analyzed by snRNA-seq (Extended Data Fig. [5e–g](https://www.nature.com/articles/s41588-025-02297-w#Fig11)). Using a cosine similarity metric to quantify gene expression homogeneity across single cells, CPD genes showed substantially higher expression homogeneity than top upregulated genes in regions with unchanged scA/B scores (Fig. [5c](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). We validated this finding in an independent scRNA-seq dataset from the conventional KP LUAD model[43](https://www.nature.com/articles/s41588-025-02297-w#ref-CR43 "Marjanovic, N. D. et al. Emergence of a high-plasticity cell state during lung cancer evolution. Cancer Cell 38, 229–246 (2020).") (Fig. [5d](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). Strikingly, RNA MERFISH established that CPD gene expression homogeneity was maintained even in a LUAD cell line isolated from the _K-MADM-Trp53_ mouse model (SA6082inf) (Fig. [5e](https://www.nature.com/articles/s41588-025-02297-w#Fig5)), highlighting the potential functional importance of CPDs. Collectively, these observations suggest that genes associated with compartment changes during tumor evolution are more homogeneously regulated, leading to a more stable expression of potential driver genes than compartment-independent regulatory mechanisms.

**Fig. 5: Single-cell 3D genomics nominates prognostic genes and genetic dependencies.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02297-w/figures/5)

**a**, Normalized scA/B score changes of genes with decreased (_n_ = 94 genes) or increased (_n_ = 120 genes) expression from AdenomaG to LUAD. _P_ values were from one-sided one-sample _t_ tests. **b**, Heatmap of CPD gene expression comparing AdenomaG and LUAD. **c**,**d**, Gene expression homogeneity of LUAD cells in the _K-MADM-Trp53_ (**c**) and _Kras__LSL-G12D/WT__; Trp53__flox/flox_ (KP) (**d**) models comparing CPDs (expression up + scA/B up) with control genes (expression up + scA/B unchanged). _P_ values were from two-sided Wilcoxon rank-sum tests. **e**, Decoded RNA MERFISH image (top) with RNA molecules pseudo-colored by gene identities in the SA6082inf LUAD cell line. Gene expression homogeneity (bottom) across single cells comparing CPDs with control genes. _P_ values were from two-sided Wilcoxon rank-sum tests. **f**, Kaplan–Meier survival curves (top) comparing TCGA LUAD patients with gene expression profiles most or least (top versus bottom 20%) correlated with CPDs and controls (expression up + scA/B unchanged; expression up only). A total of 21 genes are included per panel. _n_ = 96 tumors per group. _P_ values were from two-sided log-rank tests. A multivariate Cox regression model (bottom) analyzing CPD and control signatures in predicting patient survival after controlling for clinical covariates across the TCGA LUAD dataset (_n_ = 478). **g**, Dependency scores (DEMETER2) of LUAD cell lines (_n_ = 57) comparing CPDs and controls (expression up + scA/B unchanged or expression up only). _n_ = 19–20 genes per group with available RNAi screen data in Cancer Dependency Map. Lower score = more dependent. _P_ values were from the Kruskal–Wallis test with Dunn’s post hoc test. **h**, Cell viability (mean ± s.d., normalized to NTC, _n_ = 3 replicates per hairpin, 3 hairpins per gene) of arrayed RNAi screen targeting CPDs in KP (top) and SA6082inf (bottom) cells. NTC (green), positive controls (red) and CPDs with substantial phenotypes (≥2 hairpins with <80% of NTC cell count, purple) are indicated. In **a**, **c**–**e** and **g**, horizontal lines of each box represent the 75th percentile, median and 25th percentile. Whiskers in **a** and **c**–**e** indicate nonoutlier maximum and nonoutlier minimum, with outlier thresholds at 1.5× interquartile range beyond box limits. Whiskers in **g** indicate maximum and minimum. NTC, nontargeting control.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM8)

To investigate the clinical relevance of CPD genes, we scored individual LUAD patient samples from The Cancer Genome Atlas (TCGA)[25](https://www.nature.com/articles/s41588-025-02297-w#ref-CR25 "The Cancer Genome Atlas Research Network. Comprehensive molecular profiling of lung adenocarcinoma. Nature 511, 543–550 (2014).") for their correlation with the CPD gene signature. Patients in the top correlated quintile (_n_ = 96) exhibited substantially worse survival than patients in the most anticorrelated quintile (_n_ = 96; Fig. [5f](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). In contrast, top differentially expressed genes in genomic regions with unchanged scA/B scores were poor predictors of survival (Fig. [5f](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). A multivariate Cox regression analysis controlling for available clinical covariates (age, sex and stage) across the entire TCGA LUAD dataset (_n_ = 478) confirmed that the CPD gene signature was independently associated with worse survival (hazard ratio (HR) = 5.12, _P_ = 0.002; Fig. [5f](https://www.nature.com/articles/s41588-025-02297-w#Fig5)), whereas the top upregulated genes in genomic regions with unchanged scA/B scores or irrespective of scA/B score were not. LUAD cell lines from the Cancer Dependency Map[44](https://www.nature.com/articles/s41588-025-02297-w#ref-CR44 "Tsherniak, A. et al. Defining a cancer dependency map. Cell 170, 564–576 (2017).") also showed greater dependency for CPD genes than for the control gene sets (Fig. [5g](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). Finally, we performed an arrayed lentiviral RNAi screen targeting 18 CPD genes with short hairpin RNAs (shRNAs, _n_ = 3 per gene; Supplementary Table [6](https://www.nature.com/articles/s41588-025-02297-w#MOESM3)) in two primary LUAD mouse cell lines (SA6082inf from the _K-MADM-Trp53_ model and 31671 from the conventional KP model[45](https://www.nature.com/articles/s41588-025-02297-w#ref-CR45 "DuPage, M., Dooley, A. L. & Jacks, T. Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase. Nat. Protoc. 4, 1064–1072 (2009).")). Knockdown of 16 of 18 (89%) CPD genes reduced viability in both cell lines (Fig. [5h](https://www.nature.com/articles/s41588-025-02297-w#Fig5))—including five targets not previously implicated in lung cancer (_Ephx2_, _Peg3_, _Apob_, _Oit1_ and _Slc6a19_; Supplementary Table [7](https://www.nature.com/articles/s41588-025-02297-w#MOESM3)). In contrast, only 6.4% random genes would affect cell proliferation in the Cancer Dependency Map[46](https://www.nature.com/articles/s41588-025-02297-w#ref-CR46 "Dempster, J. M. et al. Chronos: a cell population dynamics model of CRISPR experiments that improves inference of gene fitness effects. Genome Biol. 22, 343 (2021)."),[47](https://www.nature.com/articles/s41588-025-02297-w#ref-CR47 "Pacini, C. et al. Integrated cross-study datasets of genetic dependencies in cancer. Nat. Commun. 12, 1661 (2021)."),[48](https://www.nature.com/articles/s41588-025-02297-w#ref-CR48 "Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR–Cas9 essentiality screens in cancer cells. Nat. Genet. 49, 1779–1784 (2017)."),[49](https://www.nature.com/articles/s41588-025-02297-w#ref-CR49 "Dempster, J. M. et al. Extracting biological insights from the Project Achilles genome-scale CRISPR screens in cancer cell lines. Preprint at bioRxiv 
https://doi.org/10.1101/720243
(2019).") ([Supplementary Note](https://www.nature.com/articles/s41588-025-02297-w#MOESM1)). These results suggest that combining 3D genome organization and gene expression data enhances both prognostic (survival) and predictive (genetic dependency) power beyond gene expression alone.

### Enhancer–promoter (E–P) contacts in CPDs

To assess local chromatin structure changes of CPD genes at higher resolution during LUAD progression, we performed fine-scale chromatin tracing of the _cis_\-regulatory regions of 15 genes, including CPDs and the known oncogenes _Kras_ and _Myc_ (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02297-w#Fig14)). For each gene, we targeted 40 consecutive 5–20-kb loci (20 loci for _Foxa3_ due to short genomic length) spanning the promoter and candidate enhancers (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02297-w#Fig14)) and profiled a total of 12,661 AT2, AdenomaR, AdenomaY, AdenomaG and LUAD cells with a median of 336 loci detected per cell (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02297-w#Fig14)). The kilobase-resolution chromatin folding became increasingly more compact during progression from AT2 to adenoma and then to LUAD (Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02297-w#Fig14)), suggesting that the trend of chromatin compaction from adenoma to LUAD depends on the length scale, as previously reported for chromatin compaction in other contexts[50](https://www.nature.com/articles/s41588-025-02297-w#ref-CR50 "Cheng, Y., Liu, M., Hu, M. & Wang, S. TAD-like single-cell domain structures exist on both active and inactive X chromosomes and persist under epigenetic perturbations. Genome Biol. 22, 309 (2021)."),[51](https://www.nature.com/articles/s41588-025-02297-w#ref-CR51 "Hafner, A. et al. Loop stacking organizes genome folding from TADs to chromosomes. Mol. Cell 83, 1377–1392 (2023)."),[52](https://www.nature.com/articles/s41588-025-02297-w#ref-CR52 "Matsumoto, Y., Yasuda, H., Mita, S., Marunouchit, T. & Yamada, M. Evidence for the involvement of Hl histone phosphorylation in chromosome condensation. Nature 284, 181–183 (1980)."). To identify E–P interactions, we accounted for the confounding gene-scale compaction and analyzed normalized interloci distances between putative active enhancers and promoters ([Supplementary Note](https://www.nature.com/articles/s41588-025-02297-w#MOESM1) and Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02297-w#Fig14)). Most E–P loops for CPD genes exhibited increased interactions during progression from adenoma to LUAD, concordant with increased gene expression (Extended Data Fig. [8d,e](https://www.nature.com/articles/s41588-025-02297-w#Fig14)). _Kras_ showed increased E–P looping in LUAD (Extended Data Fig. [8e,f](https://www.nature.com/articles/s41588-025-02297-w#Fig14)), suggesting an alternative mechanism for enhancing Kras expression in the adenoma-to-LUAD transition beyond amplification[32](https://www.nature.com/articles/s41588-025-02297-w#ref-CR32 "McFadden, D. G. et al. Mutational landscape of EGFR-, MYC-, and Kras-driven genetically engineered mouse models of lung adenocarcinoma. Proc. Natl Acad. Sci. USA 113, E6409–E6417 (2016)."). Similarly, _Myc_—basal expression of which is required for _Kras_\-driven lung tumor initiation[53](https://www.nature.com/articles/s41588-025-02297-w#ref-CR53 "Sodir, N. M. et al. Reversible Myc hypomorphism identifies a key Myc-dependency in early cancer evolution. Nat. Commun. 13, 6782 (2022).") and overexpression of which is sufficient to induce lung tumorigenesis[54](https://www.nature.com/articles/s41588-025-02297-w#ref-CR54 "Tran, P. T. et al. Combined inactivation of MYC and K-Ras oncogenes reverses tumorigenesis in lung adenocarcinomas and lymphomas. PLoS ONE 3, e2125 (2008).")—showed increased looping between its promoter and two putative downstream enhancers (Extended Data Fig. [8e,f](https://www.nature.com/articles/s41588-025-02297-w#Fig14)) and increased scA/B scores (Extended Data Fig. [8g](https://www.nature.com/articles/s41588-025-02297-w#Fig14)) in the AT2-to-adenoma transition, concordant with increased expression. Collectively, these data suggest that both local looping and larger-scale compartmental alterations may enhance driver gene expression.

### Rnf2 regulates 3D genome through a noncanonical function

To identify potential regulators of 3D genome architecture in LUAD progression, we first assessed whether neighboring immune cells in the tumor microenvironment affected 3D genome organization and found little influence. Analysis of AT2/cancer cells delineated by proximity to CD45+ cells showed negligible differences in chromatin compaction, conformation heterogeneity, demixing, and A–B compartment polarization (Extended Data Fig. [9a–d](https://www.nature.com/articles/s41588-025-02297-w#Fig15)). Dimensionality reduction further revealed that AT2/cancer cells adjacent to and distant from immune cells contained similar spectra of single-cell 3D genome conformations (Extended Data Fig. [9e](https://www.nature.com/articles/s41588-025-02297-w#Fig15)). These findings suggest that changes in 3D genome organization during LUAD progression are likely independent of immune cell interactions.

To identify possible cell-autonomous upstream regulators of cancer 3D genome reorganization, we developed a ‘Trace2Regulator’ pipeline (Methods) using the Binding Analysis for Regulation of Transcription (BART) algorithm[55](https://www.nature.com/articles/s41588-025-02297-w#ref-CR55 "Ma, W. et al. BARTweb: a web server for transcriptional regulator association analysis. NAR Genom. Bioinform. 3, lqab022 (2021)."),[56](https://www.nature.com/articles/s41588-025-02297-w#ref-CR56 "Wang, Z. et al. BART: a transcription factor prediction tool with query gene sets or epigenomic profiles. Bioinformatics 34, 2867–2869 (2018).") to predict transcription factors and chromatin regulators enriched at upregulated genes in loci with increased scA/B scores in the adenoma-to-LUAD transition. Rnf2, a major component of the polycomb repressive complex 1 (PRC1) catalyzing mono-ubiquitination of lysine 119 of histone H2A (H2AK119ub)[57](https://www.nature.com/articles/s41588-025-02297-w#ref-CR57 "Wang, H. et al. Role of histone H2A ubiquitination in Polycomb silencing. Nature 431, 873–878 (2004)."), emerged as a top candidate. Canonically, PRC1 works in concert with PRC2, which deposits trimethylation of lysine 27 of histone H3 (H3K27me3) to repress transcription[58](https://www.nature.com/articles/s41588-025-02297-w#ref-CR58 "Blackledge, N. P., Rose, N. R. & Klose, R. J. Targeting polycomb systems to regulate gene expression: modifications to a complex story. Nat. Rev. Mol. Cell Biol. 16, 643–649 (2015)."). We knocked down Rnf2 (shRnf2) in KP LUAD cells (Fig. [6a](https://www.nature.com/articles/s41588-025-02297-w#Fig6)) and observed a substantial negative effect on cell viability (Fig. [6b](https://www.nature.com/articles/s41588-025-02297-w#Fig6)). scA/B score changes upon Rnf2 knockdown substantially correlated with those during LUAD progression (Fig. [6c](https://www.nature.com/articles/s41588-025-02297-w#Fig6)), whereas random shuffling of the control and Rnf2 knockdown cell identities led to no correlation (Spearman _R_ = −0.07, _P_ = 0.1 for shRnf2-1; _R_ = −0.02, _P_ = 0.7 for shRnf2-3). These data suggest that Rnf2 partially regulates 3D genome reorganization in LUAD cells.

**Fig. 6: Rnf2 partially regulates 3D genome organization changes during the adenoma-to-LUAD transition.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02297-w/figures/6)

**a**, Western blot analysis of Rnf2 protein levels following Rnf2 knockdown with three independent hairpins in the KP cell line. Control cells were constructed with an NTC shRNA sequence (shNTC). HSP90 is loading control. **b**, Cell viability (mean ± s.d., normalized to mean of shNTC) following knockdown with three Rnf2 shRNAs, _n_ = 3 replicates per hairpin. _P_ values from two-sided two-sample _t_ tests are shown. **c**, scA/B score changes from shRnf2 to shNTC versus those from AdenomaG to LUAD show a substantial positive correlation, _n_ = 473 target genomic regions. Spearman correlation coefficients and _P_ values from two-sided _t_ tests are shown. **d**, Rnf2 peak densities in A (_n_ = 209 regions) and B (_n_ = 264 regions) compartments in shNTC. The horizontal lines of each box from top to bottom represent the 75th percentile, median and 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values at least 1.5× interquartile range away from the top or bottom of the box. _P_ value of two-sided Wilcoxon rank-sum test is shown. **e**, CUT&RUN read density heatmaps of Rnf2, H3K4me3, H3K27me3, RNA polymerase II with phosphorylated S5 modification, mono-ubiquitination of lysine 119 of histone H2A (H2AK119ub) and BMI1 in shNTC KP cells. Rnf2 peak regions (−5 kb, +5 kb) of all target genomic regions are shown and are categorized as active (H3K4me3+, H3K27me3−), bivalent (H3K4me3+, H3K27me3+), repressed (H3K4me3−, H3K27me3+) or other (H3K4me3−, H3K27me3−) based on chromatin marks. **f**, scA/B score changes from shRnf2 to shNTC versus those from AdenomaG to LUAD show a stronger or similarly positive correlation using target genomic regions with only active Rnf2 peaks, _n_ = 113 target genomic regions. Spearman correlation coefficients and _P_ values from two-sided _t_ tests are shown.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM9)

Because predicted Rnf2 binding sites coincided with increased scA/B scores, we speculated that Rnf2 may preferentially bind A compartment regions. Indeed, cleavage under targets and release using nuclease (CUT&RUN) of Rnf2 revealed higher Rnf2 peak density in compartment A regions (Fig. [6d](https://www.nature.com/articles/s41588-025-02297-w#Fig6)). Although Rnf2 canonically mediates gene silencing through polycomb repression[57](https://www.nature.com/articles/s41588-025-02297-w#ref-CR57 "Wang, H. et al. Role of histone H2A ubiquitination in Polycomb silencing. Nature 431, 873–878 (2004)."), it can also noncanonically associate with epigenetically active loci[59](https://www.nature.com/articles/s41588-025-02297-w#ref-CR59 "Aranda, S., Mas, G. & Di Croce, L. Regulation of gene transcription by polycomb proteins. Sci. Adv. 1, e1500737 (2015)."),[60](https://www.nature.com/articles/s41588-025-02297-w#ref-CR60 "Rai, K. et al. Dual roles of RNF2 in melanoma progression. Cancer Discov. 5, 1314–1327 (2015)."). As both polycomb-repressed and active chromatin regions tend to localize in compartment A[61](https://www.nature.com/articles/s41588-025-02297-w#ref-CR61 "Vieux-Rochas, M., Fabre, P. J., Leleu, M., Duboule, D. & Noordermeer, D. Clustering of mammalian Hox genes with other H3K27me3 targets within an active nuclear domain. Proc. Natl Acad. Sci. USA 112, 4672–4677 (2015)."),[62](https://www.nature.com/articles/s41588-025-02297-w#ref-CR62 "Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science 326, 289–293 (2009)."), we evaluated the chromatin states of Rnf2-bound regions. Specifically, we clustered Rnf2 CUT&RUN peaks in all target genomic regions with histone H3 lysine 4 trimethylation (H3K4me3) and H3K27me3 CUT&RUN profiles. Most Rnf2 peaks (51.6%) resided in active regions (H3K4me3+, H3K27me3−; Fig. [6e](https://www.nature.com/articles/s41588-025-02297-w#Fig6)), and fewer (23.3% and 4.3%, respectively) were in repressed (H3K4me3−, H3K27me3+) and bivalent (H3K4me3+, H3K27me3+) regions (Fig. [6e](https://www.nature.com/articles/s41588-025-02297-w#Fig6)). CUT&RUN of RNA Pol II with phosphorylated S5 confirmed the active transcriptional states of the active regions (Fig. [6e](https://www.nature.com/articles/s41588-025-02297-w#Fig6)). Notably, scA/B score changes upon Rnf2 knockdown were better or similarly correlated with those in LUAD progression, if only active Rnf2-bound regions were analyzed (Fig. [6f](https://www.nature.com/articles/s41588-025-02297-w#Fig6)). These results suggest that Rnf2 in active loci may regulate 3D genome reorganization associated with LUAD progression.

We further performed H2AK119ub CUT&RUN and found that Rnf2 peaks in active regions did not colocalize with strong H2AK119ub peaks, while another core PRC1 component, BMI1, colocalized with Rnf2 in all regions (Fig. [6e](https://www.nature.com/articles/s41588-025-02297-w#Fig6)). This suggests that the canonical ubiquitin ligase function of Rnf2 may be dispensable for its role in shaping the 3D genome. Indeed, rescue experiments following Rnf2 knockdown by expressing WT Rnf2 or a catalytic-mutant Rnf2 (I53S) lacking ubiquitin ligase activity[63](https://www.nature.com/articles/s41588-025-02297-w#ref-CR63 "Ben-Saadon, R., Zaaroor, D., Ziv, T. & Ciechanover, A. The polycomb protein Ring1B generates self atypical mixed ubiquitin chains required for its in vitro histone H2A ligase activity. Mol. Cell 24, 701–711 (2006).") both showed scA/B score changes correlated with those during LUAD progression (Extended Data Fig. [10a,b](https://www.nature.com/articles/s41588-025-02297-w#Fig16)). These results demonstrate that Rnf2 partially regulates 3D genome changes during the adenoma-to-LUAD progression through a ubiquitin ligase-independent function. To test whether Rnf2 directly reorganizes the 3D genome in LUAD cells, we used a degradation tag (dTAG) system[64](https://www.nature.com/articles/s41588-025-02297-w#ref-CR64 "Nabet, B. et al. The dTAG system for immediate and target-specific protein degradation. Nat. Chem. Biol. 14, 431–441 (2018).") to induce acute Rnf2 protein loss in KP cells. Rnf2 protein was rapidly degraded after 0.5 h of dTAG ligand (dTAG-13) administration but H2AK119ub levels were unchanged (Extended Data Fig. [10c–e](https://www.nature.com/articles/s41588-025-02297-w#Fig16)). We found that the scA/B score changed substantially upon rapid Rnf2 degradation and these alterations correlated with those during LUAD progression or upon Rnf2 knockdown (Extended Data Fig. [10f](https://www.nature.com/articles/s41588-025-02297-w#Fig16)). In contrast, random shuffling of degraded and control cell identities led to no substantial correlation (Spearman _R_ = 0.04, _P_ = 0.4 for LUAD progression; _R_ = 0.02, _P_ = 0.6 for Rnf2 knockdown). These results support a direct role for Rnf2 in 3D genome reorganization independent of its ubiquitin ligase function.

Discussion
----------

In this study, we generated comprehensive 3D genome atlases in physiologically relevant mouse models of Kras-driven lung and pancreatic cancer progression and introduced a generally applicable pipeline to measure and interpret biological heterogeneity of the 3D genome in complex tissues and diseases (Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02297-w#MOESM1)). Notably, we showed that single-cell 3D genome organization can distinguish and predict morphologic cancer states with high accuracy (Fig. [4](https://www.nature.com/articles/s41588-025-02297-w#Fig4)), despite the intrinsic variability of 3D chromatin conformation even among cells of the same type[50](https://www.nature.com/articles/s41588-025-02297-w#ref-CR50 "Cheng, Y., Liu, M., Hu, M. & Wang, S. TAD-like single-cell domain structures exist on both active and inactive X chromosomes and persist under epigenetic perturbations. Genome Biol. 22, 309 (2021)."),[65](https://www.nature.com/articles/s41588-025-02297-w#ref-CR65 "Ou, H. D. et al. ChromEMT: visualizing 3D chromatin structure and compaction in interphase and mitotic cells. Science 357, eaag0025 (2017)."),[66](https://www.nature.com/articles/s41588-025-02297-w#ref-CR66 "Bintu, B. et al. Super-resolution chromatin tracing reveals domains and cooperative interactions in single cells. Science 362, eaau1783 (2018)."),[67](https://www.nature.com/articles/s41588-025-02297-w#ref-CR67 "Takei, Y. et al. Integrated spatial genomics reveals global architecture of single nuclei. Nature 590, 344–350 (2021)."),[68](https://www.nature.com/articles/s41588-025-02297-w#ref-CR68 "Payne, A. C. et al. In situ genome sequencing resolves DNA sequence and structure in intact biological samples. Science 371, eaay3446 (2021)."). We further showed that high-dimensional 3D genome data can be used to identify genes that govern prognosis and delineate cancer cell dependencies (Fig. [5](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). Finally, our results underscore the potential of 3D genome data to identify previously unappreciated 3D genome regulators, such as Rnf2, which appears to have a noncanonical activating function during lung tumor evolution (Fig. [6](https://www.nature.com/articles/s41588-025-02297-w#Fig6) and Extended Data Fig. [10](https://www.nature.com/articles/s41588-025-02297-w#Fig16)).

Recent scRNA-seq[43](https://www.nature.com/articles/s41588-025-02297-w#ref-CR43 "Marjanovic, N. D. et al. Emergence of a high-plasticity cell state during lung cancer evolution. Cancer Cell 38, 229–246 (2020).") and single-cell assay for transposase-accessible chromatin[69](https://www.nature.com/articles/s41588-025-02297-w#ref-CR69 "LaFave, L. M. et al. Epigenomic state transitions characterize tumor progression in mouse lung adenocarcinoma. Cancer Cell 38, 212–228 (2020).") studies in Kras-driven LUAD models have revealed transitional and plastic cell states with molecular alterations that can drive tumor initiation, progression and metastasis[20](https://www.nature.com/articles/s41588-025-02297-w#ref-CR20 "Kaiser, A. M. et al. p53 governs an AT1 differentiation programme in lung cancer suppression. Nature 619, 851–859 (2023)."),[43](https://www.nature.com/articles/s41588-025-02297-w#ref-CR43 "Marjanovic, N. D. et al. Emergence of a high-plasticity cell state during lung cancer evolution. Cancer Cell 38, 229–246 (2020)."),[69](https://www.nature.com/articles/s41588-025-02297-w#ref-CR69 "LaFave, L. M. et al. Epigenomic state transitions characterize tumor progression in mouse lung adenocarcinoma. Cancer Cell 38, 212–228 (2020)."). While transcriptional heterogeneity gradually increased during LUAD progression in these models, we observed a striking nonmonotonic evolution of the 3D genome (Fig. [2](https://www.nature.com/articles/s41588-025-02297-w#Fig2)), suggesting a potential 3D genome structural bottleneck. Comparison of 3D genomes across tumors and mice suggested that the increased compaction and decreased heterogeneity of chromatin conformations in adenoma cells may be due to phenotypic convergence. Although additional genetic alterations were not observed (Extended Data Fig. [5a–d](https://www.nature.com/articles/s41588-025-02297-w#Fig11)), this convergence could be due to the driver oncogene _Kras_, as similar 3D genome structural changes were observed in a separate Kras-driven PDAC model (Extended Data Fig. [6](https://www.nature.com/articles/s41588-025-02297-w#Fig12)). In support of a functional role of these 3D genome alterations during lung tumor evolution, combining 3D genome and transcriptomic data improved the identification of important genes (CPDs) that predict prognosis and dependency (Fig. [5](https://www.nature.com/articles/s41588-025-02297-w#Fig5)). These genes are more homogenously expressed across LUAD cells both in vitro and in vivo, consistent with longitudinal data from the human tracking cancer evolution through therapy (TRACERx) lung cancer cohort demonstrating that genes with decreased intratumoral heterogeneity undergo positive selection[70](https://www.nature.com/articles/s41588-025-02297-w#ref-CR70 "TRACERx Consortium. et al. A clonal expression biomarker associates with lung cancer mortality. Nat. Med. 25, 1540–1548 (2019)."). To directly map 3D genome alterations to transcriptionally and epigenetically defined cell states, future studies should combine genome-wide chromatin tracing with spatial transcriptomics[71](https://www.nature.com/articles/s41588-025-02297-w#ref-CR71 "Moffitt, J. R. et al. Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region. Science 362, eaau5324 (2018)."),[72](https://www.nature.com/articles/s41588-025-02297-w#ref-CR72 "Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S. & Zhuang, X. Spatially resolved, highly multiplexed RNA profiling in single cells. Science 348, aaa6090 (2015)."),[73](https://www.nature.com/articles/s41588-025-02297-w#ref-CR73 "Fang, R. et al. Conservation and divergence of cortical cell organization in human and mouse revealed by MERFISH. Science 377, 56–62 (2022)."),[74](https://www.nature.com/articles/s41588-025-02297-w#ref-CR74 "Allen, W. E., Blosser, T. R., Sullivan, Z. A., Dulac, C. & Zhuang, X. Molecular and spatial signatures of mouse brain aging at single-cell resolution. Cell 186, 194–208 (2023)."), epigenomics[75](https://www.nature.com/articles/s41588-025-02297-w#ref-CR75 "Lu, T., Ang, C. E. & Zhuang, X. Spatially resolved epigenomic profiling of single cells in complex tissues. Cell 186, 2275–2279 (2023).") and proteomics[67](https://www.nature.com/articles/s41588-025-02297-w#ref-CR67 "Takei, Y. et al. Integrated spatial genomics reveals global architecture of single nuclei. Nature 590, 344–350 (2021)."),[76](https://www.nature.com/articles/s41588-025-02297-w#ref-CR76 "Lin, J.-R. et al. Highly multiplexed immunofluorescence imaging of human tissues and tumors using t-CyCIF and conventional optical microscopes. eLife 7, e31657 (2018).") in the same single cancer cells in native tissues. Furthermore, as DNA FISH has been used to map structural variations in human cancer, broad application of single-cell in situ 3D genome mapping to clinical samples may hold promise for advancing cancer diagnosis, subtype classification, treatment response prediction and target discovery.

Methods
-------

### Ethics Statement

All animal studies were approved by the Institutional Animal Care and Use Committee of Yale University. The tumor sizes in this study did not exceed the maximal tumor size (1 cm3) permitted by the institutional review board.

### Mouse and cell lines

#### Mouse strains

Mice were housed in a specific-pathogen-free facility with controlled temperature, humidity and day/night cycles and maintained in a mixed background. _MADM11-GT_ (013749), _MADM11-TG_ (013751) and _Pdx1-cre_ (014647) were obtained from the Jackson Laboratory (JAX). _Kras__LSL-G12D_ (JAX, 008179) and _Trp53__KO_ (JAX, 002101) mice were a gift from T. Jacks. _Kras__LSL-G12D_/_WT__; MADM11-TG, Trp53__KO_/_MADM11-TG, Trp53__WT_ breeder mice were generated as previously described[30](https://www.nature.com/articles/s41588-025-02297-w#ref-CR30 "Muzumdar, M. D. et al. Clonal dynamics following p53 loss of heterozygosity in Kras-driven cancers. Nat. Commun. 7, 12685 (2016)."). _Kras__LSL-G12D_/_WT__; MADM11-TG, Trp53__KO_/_MADM11-TG_, _Trp53__WT_ were crossed with _MADM11-GT_ mice (with or without _Pdx1-cre_) to generate _K-MADM-Trp53_ or _Pdx1-cre; K-MADM-Trp53_ experimental mice of both sexes for lung and pancreatic cancer analyses, respectively. Genotyping primers and protocols have been previously reported[30](https://www.nature.com/articles/s41588-025-02297-w#ref-CR30 "Muzumdar, M. D. et al. Clonal dynamics following p53 loss of heterozygosity in Kras-driven cancers. Nat. Commun. 7, 12685 (2016)."). Large-scale chromatin tracing in lung included four WT mice (91-day male, 191-day female, 283-day male and 91-day female), four mice with adenoma (225-day male, 221-day male, 232-day female and 224-day male) and five mice with LUAD (232-day male, 487-day male, 494-day male, 221-day male and 551-day male). Large-scale chromatin tracing in pancreas included a WT mouse (59-day male) and a mouse with PanIN and PDAC tumors (46-day male). Fine-scale chromatin tracing in mouse lung included a WT mouse (191-day female), a mouse with adenoma (225-day male) and a mouse with LUAD (494-day male).

#### Lentivirus production and infection

pPGK-Cre lentiviral backbone was amplified and transfected into 293T cells using VSV-G (Addgene, 8454) and psPAX2 (Addgene, 12260) packaging vectors and TransIT-LT1 kit (MirusBio). After 48–72 h, virus was collected, filtered and ultracentrifuged. Lentivirus was resuspended in OptiMEM (Thermo Fisher Scientific) and administered intratracheally to _K-MADM-Trp53_ mice at 6–10 weeks of age to generate lung tumors.

#### Cell line

The KP LUAD cell line, 31671, was a gift from N. Joshi and was derived from an autochthonous _Kras__LSL-G12D_/_WT__; Trp53__flox_/_flox_ mouse administered with Adeno-Cre[77](https://www.nature.com/articles/s41588-025-02297-w#ref-CR77 "Damo, M. et al. Inducible de novo expression of neoantigens in tumor cells and mice. Nat. Biotechnol. 39, 64–73 (2021)."). The _K-MADM-Trp53_ LUAD cell line (SA6082inf) was derived from collagenase-based dissociation of a large GFP+ tumor dissected from a _K-MADM-Trp53_ mouse. _Kras_ and _Trp53_ genotypes were confirmed by PCR. Both cell lines were cultured in DMEM (Corning, 10-013-CV) containing 10% (vol/vol) FBS (Gibco, 26140-079) and 1% (vol/vol) Penicillin-Streptomycin (Thermo Fisher Scientific, 15140-122) at 37 °C with 5% CO2. Cells were passaged whenever they reached confluency. Both cell lines were seeded onto UV-sterilized coverslips (Bioptechs, 40-mm diameter; 1.5) and allowed to grow until 60–70% confluency before primary probe hybridization. All cell lines were tested for the presence of mycoplasma using PCR (ATCC, 30-1012K).

### Probe design and synthesis

Detailed probe design and synthesis information are included in the [Supplementary Note](https://www.nature.com/articles/s41588-025-02297-w#MOESM1). Template probe sequences for chromatin tracing and RNA MERFISH are provided in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02297-w#MOESM3). Codebooks for chromatin tracing and RNA MERFISH, and genome coordinates and target region features in chromatin tracing are provided in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02297-w#MOESM3). PCR primers, reverse transcription primers for probe synthesis and adaptors, readout and blocking probes for sequential readout imaging are provided in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02297-w#MOESM3).

### Mouse _K-MADM-Trp53_ lung tissue chromatin tracing experiments

#### Tissue preparation

_K-MADM-Trp53_ mice were dissected at sign of respiratory distress through CO2 asphyxiation. Lungs were perfused with ice-cold 1× PBS, incubated in 4% (vol/vol) paraformaldehyde (Electron Microscopy Sciences, 15710-S) overnight, and cryoprotected with 30% sucrose. Tissue was embedded in Tissue-Tek optimal cutting temperature compound (OCT) and frozen on dry ice before storage at −80 °C.

#### Tissue sectioning

Frozen mouse lung blocks were sectioned at a thickness of 10 μm at −20 °C on a cryostat. The following three consecutive tissue slices were sectioned at a time: one for hematoxylin and eosin (H&E) staining, one for whole-section fluorescence imaging and one for chromatin tracing. The sections were air-dried at room temperature for 1 h and then either used directly or stored at −20 °C for months.

#### H&E staining and whole-section fluorescence imaging

H&E staining was performed by Yale Pathology Tissue Services. For whole-section fluorescence imaging, the section was mounted in mounting medium with DAPI (VectorLabs, H-1800-2). Stitched fluorescence images in 353-nm, 488-nm and 592-nm illumination channels for DAPI, GFP and tdTomato, respectively, were collected with a Plan-Apochromat ×10/0.45 M27 objective on a Zeiss Axio Imager M2 microscope.

#### Co-immunofluorescence and fluorescent protein imaging of the chromatin tracing section

Frozen sections were first balanced at room temperature for 10 min, followed by hydration in Dulbecco’s phosphate-buffered saline (DPBS) for 5 min twice. Tissue sections were permeabilized with 0.5% (vol/vol) Triton X-100 in DPBS for 30 min at room temperature and washed twice in DPBS for 2 min. Tissue sections were then blocked for 30 min at room temperature in blocking buffer containing 1% (wt/vol) BSA (Sigma-Aldrich, A9647-100G), 22.52 mg ml−1 glycine (AmericanBio, AB00730), 10% (vol/vol) donkey serum (MilliporeSigma, S30–100 ml), 5% (vol/vol) goat serum (Invitrogen, 31873) and 0.1% (vol/vol) Tween-20 in DPBS. Tissue sections were incubated with rabbit anti-SPC antibody (MilliporeSigma, AB3786, 1:50) and rat anti-CD45 antibody (BioLegend, 103101, 1:100) in blocking buffer at 4 °C overnight. The samples were washed thrice in DPBS for 5 min and then incubated with DyLight 800-labeled donkey anti-rabbit secondary antibody (Invitrogen, SA5-10044, 1:1,000) and Alexa Fluor 647-labeled goat anti-rat secondary antibody (Invitrogen, A-21247, 1:1,000) in blocking buffer for 1 h at room temperature. The samples were washed thrice in DPBS for 5 min and then incubated with DAPI (Thermo Fisher Scientific, 62248) at 1:1,000 dilution in 2× saline-sodium citrate buffer (SSC) for 10 min. The samples were then mounted onto a Bioptechs FCS2 flow chamber and replenished with freshly prepared imaging buffer with an oxygen scavenging system (50 mM Tris–HCl pH 8.0, 10% wt/vol glucose, 2 mM Trolox (Sigma-Aldrich, 238813), 0.5 mg ml−1 glucose oxidase (Sigma-Aldrich, G2133), 40 μg ml−1 catalase (Sigma-Aldrich, C30)). The imaging buffer was covered with a layer of mineral oil (Sigma-Aldrich, 330779) in the reservoir tube to prevent continuous oxidation. We then selected multiple fields of view (FOVs) at predefined tumor regions based on tumor grades of the adjacent whole-section fluorescence images. Tumor grades were confirmed independently by two investigators (S.S.A. and M.D.M.). At each FOV, we sequentially took z-stack images of DAPI, GFP fluorescence, tdTomato fluorescence, anti-SPC immunofluorescence and anti-CD45 immunofluorescence with 405-nm, 488-nm, 560-nm, 647-nm and 750-nm laser illuminations. The z-stacks (7–10 μm total) had a 200-nm step size and a 0.4 s exposure time per step.

#### Primary probe hybridization

After imaging, tissue sections were de-assembled from the chamber, treated with 1 μg ml−1 proteinase K (Invitrogen, AM2546) in 2% (vol/vol) sodium dodecyl sulfate in 2× SSC at 37 °C for 10 min, and washed twice in DPBS for 2 min. Tissue sections were then treated with 0.1 M HCl for 5 min and washed twice in DPBS for 2 min. Tissue sections were digested with 0.1 mg ml−1 RNase A (Thermo Fisher Scientific, EN0531) in DPBS for 45 min at 37 °C and washed twice with 2× SSC for 2 min. Tissue sections were treated with prehybridization buffer containing 50% (vol/vol) formamide (Sigma-Aldrich, F7503) and 0.1% (vol/vol) Tween-20 in 2× SSC. Synthesized primary probes were dissolved in 25 μl hybridization buffer containing 50% (vol/vol) formamide and 20% (vol/vol) dextran sulfate (MilliporeSigma, S4030) in 2× SSC. The total probe concentration was 30–40 μM. Coverslips were immersed in hybridization buffer containing probes in 60 mm petri dishes, heat denatured in a 90 °C water bath for 3.5 min and subsequently incubated at 47 °C in a humid chamber for 36–48 h. Samples were washed with 50% (vol/vol) formamide in 2× SSC for 15 min twice followed by 2× SSC for 15 min all at room temperature. Washed samples were then incubated with 0.1-μm yellow–green beads (Invitrogen, F8803) resuspended in 2× SSC as fiducial markers for drift correction, washed with 2× SSC briefly, incubated with DAPI at 1:1,000 dilution in 2× SSC for 10 min for image registration and washed again with 2× SSC for 2 min twice.

#### Readout probe hybridization and imaging

After the primary probe hybridization, the sample was repeatedly hybridized with adaptors and readout probes, imaged and photobleached for 50 rounds. Each adaptor probe was 60 nucleotides (nt) in length, containing a 20-nt primary probe binding region and two consecutive 20-nt readout probe binding regions. Readout probes were 30-nt oligos conjugated with Alexa Fluor 647 (or Cy5) or ATTO 565 (or Cy3) fluorophores with 20-nt complementary to the readout probe binding regions of adaptors (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02297-w#MOESM3)). To perform automatic buffer exchange, we used a Bioptechs FCS2 flow chamber and a computer-controlled, custom-built fluidics system[11](https://www.nature.com/articles/s41588-025-02297-w#ref-CR11 "Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. Science 353, 598–602 (2016)."),[72](https://www.nature.com/articles/s41588-025-02297-w#ref-CR72 "Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S. & Zhuang, X. Spatially resolved, highly multiplexed RNA profiling in single cells. Science 348, aaa6090 (2015)."). First, we used DAPI images for registration to images taken before primary probe hybridization. We selected the same FOVs as those selected during co-immunofluorescence and fluorescent protein imaging. At each FOV, we took z-stack images with 488-nm and 405-nm laser illuminations for fiducial beads and DAPI, respectively. For each round of readout probe hybridization, we flowed 2 ml of readout probe hybridization buffer (20% vol/vol ethylene carbonate (Sigma-Aldrich, E26258) in 2× SSC) containing two adaptors each at 50 nM through the chamber and incubated for 15 min at room temperature. We flowed through 2 ml wash buffer (20% vol/vol ethylene carbonate in 2× SSC) for 2 min and 2 ml readout probe hybridization buffer containing two dye-labeled readout probes each at 75 nM, with 15 min incubation at room temperature. We further flowed through 2 ml wash buffer for 2 min and 2 ml imaging buffer. At each FOV, dye-labeled readout probes were imaged in 647-nm and 560-nm channels and fiducial beads were imaged in the 488-nm channel. The z-stacks (7–10 μm total) had a 200-nm step size and a 0.4 s exposure time per step. After imaging, we switched buffer to readout probe hybridization buffer containing 1 μM dye-free readout probes (blocking oligos) and photobleached with continuous simultaneous laser illuminations with 750-nm, 647-nm and 560-nm lasers for 40 s. We then flowed 5 ml 2× SSC to wash away unbound blocking oligos before the next hybridization round. A total of 50 hybridization rounds was performed.

### Arrayed RNAi screen of cell proliferation

Lentiviral infection conditions were optimized in 96-well plates for initial cell seeding number, lentiviral dosage, antibiotic concentration and assay time. KP and _K-MADM-Trp53_ LUAD cells were seeded in complete cell culture media at a density of 250 cells per well in a 96-well plate, incubated for 24 h, and infected with lentiviral supernatant of Sigma Mission shRNAs targeting CPDs obtained from Yale Cancer Center Functional Genomics Core. A total of 200 μl of media was added to each well, comprising 50 μl of lentiviral supernatant and 150 μl of complete cell culture medium with 10 μg ml−1 polybrene (EMD Millipore, TR-1003-G). Each shRNA hairpin was tested in triplicate. All lentiviral infections were performed in duplicate to rule out the influence of lentiviral dosage on cell growth—one replicate with 6 μg/ml puromycin selection and the other replicate with no puromycin selection. After 24 h of lentiviral incubation, corresponding wells were treated with or without puromycin selection for 48 h. The cells were then incubated with complete cell culture media for 4–5 days. Total viable cell count was determined with CellTiter-Glo Luminescent Cell Viability Assay (Promega, G7572) using a Promega luminescence plate reader. For data analysis, we deducted luminescence readout values of blank wells from those of test wells. We then normalized luminescence readout values of each target shRNA hairpin to those of nontargeting control shRNA sequence (shNTC). All shRNA hairpin sequences are provided in Supplementary Table [6](https://www.nature.com/articles/s41588-025-02297-w#MOESM3).

### Rnf2 targeted degradation

Two million cells of the KP LUAD cell line 31671 were nucleofected (Amaxa) with 2 μg homology-directed repair (HDR) template and 2 μM RNP (IDT Alt-R CRISPR–Cas9 sgRNA) for Rnf2-dTAG. The HDR template contained a 555-nt 5′ homology arm homologous to the 5′ end of the Rnf2 stop codon, a 30-nt linker and two HA tags, an FKBPF36V and mScarlet insert (Supplementary Table [6;](https://www.nature.com/articles/s41588-025-02297-w#MOESM3) amplified from a gift plasmid from D. Schatz at Yale University) and a 999-nt 3′ homology arm homologous to the 3′ end of the Rnf2 stop codon. After transfection, the cells were seeded into 6-well plates and cultured for 24–48 h before FACS sorting. mScarlet+ cells were sorted into 96-well plates. Single colonies were manually picked, genotyped with PCR and validated with sequencing. The spacer sequence for Rnf2 sgRNA is GACTTTATTATGCACCCACCA. The dTAG-13 ligand and negative control ligand were added to the cells at a final concentration of 500 nM. The cells were incubated at 37 °C with 5% CO2.

### Image analysis

All image analyses were performed with MATLAB R2019b.

The genome-wide chromatin tracing image analysis pipeline consists of the following steps: color correction, drift correction, nucleus segmentation, foci fitting, decoding and trace linking. Detailed methods are listed in the [Supplementary Note](https://www.nature.com/articles/s41588-025-02297-w#MOESM1).

### Data analysis and statistics

#### Decompaction

To compare levels of chromatin compaction, we defined a decompaction score as the mean interloci distance between a locus pair on a chromosome of a cell state.

#### Heterogeneity

To compare chromatin conformation heterogeneity, we defined a heterogeneity score as the coefficient of variation (COV) of interloci distances between a locus pair on a chromosome of a cell state (variation among different chromosome copies). Similarly, the heterogeneity scores for cell pairs were defined as the root mean square of interloci distance difference normalized by the mean.

#### Demixing

To compare levels of chromatin intermixing, we defined a demixing score as the s.d. of all normalized mean interloci distances (mean interloci distance normalized to the mean of all mean interloci distances on the chromosome) on a chromosome of a cell state. The normalization accounted for and excluded the influence of chromatin compaction on demixing measurements.

#### A and B compartment polarization analysis

To identify A and B compartments, we adapted a previous algorithm to determine A and B compartment scores[11](https://www.nature.com/articles/s41588-025-02297-w#ref-CR11 "Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. Science 353, 598–602 (2016)."),[24](https://www.nature.com/articles/s41588-025-02297-w#ref-CR24 "Liu, M. et al. Multiplexed imaging of nucleome architectures in single cells of mammalian tissue. Nat. Commun. 11, 2907 (2020)."). In brief, we generated a mean interloci distance matrix for each chromosome, normalized the mean distance to the expected distance calculated by power-law fitting of spatial versus genomic distances and calculated the Pearson correlation coefficient among each column pair of the normalized matrix. Finally, we applied a principal component analysis (PCA) of the Pearson correlation matrix and used coefficients of the first principal component as compartment scores. We calculated correlations between compartment scores and an averaged profile of histone H3 lysine 4 monomethylation (H3K4me1), H3K4me3, DNase I hypersensitivity site and gene densities[78](https://www.nature.com/articles/s41588-025-02297-w#ref-CR78 "Davis, C. A. et al. The encyclopedia of DNA elements (ENCODE): data portal update. Nucleic Acids Res. 46, D794–D801 (2018)."),[79](https://www.nature.com/articles/s41588-025-02297-w#ref-CR79 "The ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature 489, 57–74 (2012)."), and flipped signs of compartment scores if necessary, so that A and B compartment genomic regions had positive and negative scores, respectively. We then used a previously described polarization index metric to quantify the polarization of A and B compartments[24](https://www.nature.com/articles/s41588-025-02297-w#ref-CR24 "Liu, M. et al. Multiplexed imaging of nucleome architectures in single cells of mammalian tissue. Nat. Commun. 11, 2907 (2020)."). For chromosome 6 in Fig. [3d](https://www.nature.com/articles/s41588-025-02297-w#Fig3), we used the first 30 target genomic regions to calculate the polarization index to match target region numbers in different chromosomes. We downloaded H3K4me1 and H3K4me3 chromatin immunoprecipitation sequencing peaks and DNA DNase I hypersensitivity sites from The Encyclopedia of DNA Elements (ENCODE)—ENCFF536DWZ, ENCFF508WEP and ENCFF268DLZ. The gene density profile was downloaded from the UCSC table browser.

#### Radial score analysis

To calculate the radial score of each genomic region in each single cell, we measured the mean spatial distance among each region to the centroid of all target regions and normalized the distance to the average spatial distances from all genomic regions to the centroid.

#### The ‘Trace2State’ pipeline for single-cell chromatin conformation-based cell state visualization and classification

First, we constructed an input matrix where each row represented a single cell and each column represented the scA/B score of each genomic region[35](https://www.nature.com/articles/s41588-025-02297-w#ref-CR35 "Tan, L. et al. Changes in genome architecture and transcriptional dynamics progress independently of sensory experience during post-natal brain development. Cell 184, 741–758 (2021)."). The scA/B score was calculated as the mean A–B compartment score of all its spatially adjacent genomic regions within a 1,200-nm 3D neighborhood. Only cells with at least 10 traces were analyzed. Missing values were replaced with 0s. The matrix was used as input in a PCA, and the first 50 principal components were used for downstream visualizations. We scaled the 50 principal components with the PCA().fit-transform function in Python 3.11.12, and visualized the single-cell data with _t_\-distributed stochastic neighbor embedding, uniform manifold approximation and projection and pairwise controlled manifold approximation[37](https://www.nature.com/articles/s41588-025-02297-w#ref-CR37 "Van der Maaten, L. & Hinton, G. Visualizing data using t-SNE. J. Mach. Learn. Res. 9, 2579–2605 (2008)."),[38](https://www.nature.com/articles/s41588-025-02297-w#ref-CR38 "McInnes, L., Healy, J., Saul, N. & Großberger, L. UMAP: uniform manifold approximation and projection. J. Open Source Softw. 3, 861 (2018)."),[39](https://www.nature.com/articles/s41588-025-02297-w#ref-CR39 "Wang, Y., Huang, H., Rudin, C. & Shaposhnik, Y. Understanding how dimension reduction tools work: an empirical approach to deciphering t-SNE, UMAP, TriMAP, and PaCMAP for data visualization. J. Mach. Learn. Res. 22, 9129–9201 (2021)."). To classify different cell states with supervised machine learning, we supplied the Classification Learner App of MATLAB with the same input scA/B score matrix and applied fivefold cross-validation to prevent overfitting. All machine learning models were trained, and the model with the highest prediction accuracy (medium Gaussian support vector machine, regularization included with box constraint level = 1) was retained to plot the confusion matrix and receiver operating characteristic curves.

#### The ‘Trace2Biomarker’ pipeline for CPD gene identification

To identify CPD genes, we first identified marker genomic regions with substantially changed scA/B scores during LUAD progression. Specifically, we extracted cells with more than 325 genomic regions. We then performed rank normalization (using the tiedrank function in MATLAB) of scA/B scores in each cell. We performed Wilcoxon rank-sum tests of rank-normalized scA/B scores of each genomic region comparing AdenomaG and LUAD. Genomic regions with _P_ < 0.1 were defined as marker genomic regions. We then defined CPD genes as genes located in marker genomic regions with increased scA/B scores and higher expression (mean expression count > 10, fold change > 3, false discovery rate (FDR) < 0.05) in LUAD versus AdenomaG cells. We defined control genes as those with increased expression (mean expression count > 10, fold change > 3, FDR < 0.05) in regions with unchanged scA/B scores (_P_ ≥ 0.1). Genes with increased expression only are defined as those with increased expression (mean expression count > 10, fold change > 3, FDR < 0.05) in LUAD versus AdenomaG cells, irrespective of scA/B score changes. Equal gene numbers (top 21 upregulated by fold change) were used for each gene list above and interrogated in TCGA patient survival analyses.

#### The ‘Trace2Regulator’ pipeline for 3D genome regulator identification

To identify chromatin regulators that bind to genes in marker loci, we first identified genes with increased expression (FDR < 0.1, fold change > 1) in genomic regions with increased scA/B scores (_P_ < 0.1) from AdenomaG to LUAD cells. We then input the gene list into BART[55](https://www.nature.com/articles/s41588-025-02297-w#ref-CR55 "Ma, W. et al. BARTweb: a web server for transcriptional regulator association analysis. NAR Genom. Bioinform. 3, lqab022 (2021)."),[56](https://www.nature.com/articles/s41588-025-02297-w#ref-CR56 "Wang, Z. et al. BART: a transcription factor prediction tool with query gene sets or epigenomic profiles. Bioinformatics 34, 2867–2869 (2018).") to generate chromatin regulators predicted to bind to the input genes, using default parameters. We further knocked down candidate chromatin regulators with shRNAs in the KP LUAD cell line and performed DNA MERFISH. We further compared 3D genome alterations upon the candidate chromatin regulator knockdown with those during the AdenomaG-to-LUAD progression. Candidate regulators showing concordant 3D genome alterations were identified as 3D genome regulators.

### CUT&RUN experiment and analysis

#### CUT&RUN protocol

CUT&RUN was performed using the Epicypher CUTANA ChIC/CUT&RUN Kit (14-1048) according to the manufacturer’s specifications with the following conditions and modifications: for binding to the Concanavalin beads, 500,000 cells per sample were prepared, counted twice using a hemacytometer, and averaged. Nuclei were prepared according to the Epicypher Cut and Run Manual Appendix. Nuclei were incubated with activated beads. For _Escherichia coli_ spike-in DNA, 0.1 ng was added to each sample. The following antibodies (0.5 μg of antibody per reaction) were used: IgG control (CUTANA Kit Rabbit IgG CUT&RUN Negative Control Antibody), Rnf2 antibody (Cell Signaling Technology, 5694), H3K4me3 antibody (EpiCypher, 13-0041), H3K27me3 antibody (Cell Signaling Technology, 9733), H2AK119ub antibody (Cell Signaling Technology, 8240), BMI1 antibody (Active Motif, 39993) and RNA Pol II p-ser5 (Abcam, ab193467).

#### Library preparation

Libraries were prepared using Epicypher CUTANA CUT&RUN Library Prep Kit (14-1001) according to the manufacturer’s specifications with the following modification: SPRI-select beads were used after library preparation to perform 200–700 bp size selection to enrich for DNA fragments from CUT&RUN and remove adaptor dimers or high molecular weight DNA. Library quality was analyzed using Agilent Tapestation D1000 High Sensitivity Tapes (5067–5584). Sequencing was performed with 10 million reads per sample, 150 bp paired-end reads, on an Illumina NovaSeq 6000.

#### CUT&RUN data analysis

CUT&RUN reads were analyzed for quality control using FastQC ([https://www.bioinformatics.babraham.ac.uk/projects/fastqc/](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)) and trimmed using Trimmomatic. Reads were then aligned to mm9 and K-12 _E. coli_ genome U00096.3 by Bowtie2 2.3.4. Picard 2.27.4 was used to downsample the reads so that the read depths of _E. coli_ sequences from different samples matched each other. SAMTools 1.11 was then used to convert to BAM format, index, isolate uniquely mapped paired reads and remove duplicates. MACS2 2.2.7.1 was used to call sample narrow peaks, using IgG as an input. Read counts across genomic intervals and peak visualization were performed using deepTools 3.3 (ref. [80](https://www.nature.com/articles/s41588-025-02297-w#ref-CR80 "Ramírez, F., Dündar, F., Diehl, S., Grüning, B. A. & Manke, T. DeepTools: a flexible platform for exploring deep-sequencing data. Nucleic Acids Res. 42, 187–191 (2014).")), and .bw files were visualized using Integrative Genomics Viewer.

Other detailed methods are included in the [Supplementary Note](https://www.nature.com/articles/s41588-025-02297-w#MOESM1).

### Statistics and reproducibility

Statistical tests and sample sizes are reported in the corresponding sections. The number of replicates was determined to be the maximum number that could be collected with the available equipment and personnel during the study period. Our study analyzed AT2 cells, AT2-derived cancer cells and immune cells expressing the pan-immune marker CD45. Other cells are excluded from the analyses. Random subsampling was performed to show that as few as 100 cells per cell state were sufficient to recapitulate the 3D genome organization changes for most structural features in lung. The investigators were not blinded to allocation during experiments and outcome assessment. We visually inspected data distributions for each statistical test, but normality and equal variances were not formally tested. Wilcoxon tests were used to avoid normality assumptions.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02297-w#MOESM2) linked to this article.

Data availability
-----------------

The snRNA-seq and bulk RNA-seq data are available from Gene Expression Omnibus (GEO; [GSE275588](https://www.ncbi.xyz/geo/query/acc.cgi?acc=GSE275588), [GSE295452](https://www.ncbi.xyz/geo/query/acc.cgi?acc=GSE295452)). The CUT&RUN data are available from GEO ([GSE295857](https://www.ncbi.xyz/geo/query/acc.cgi?acc=GSE295857)). The whole-exome sequencing data are available from SRA ([PRJNA1255157](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA1255157)). The Hi-C and analyzed chromatin tracing data are deposited to the 4D Nucleome portal ([https://data.4dnucleome.org/publications/e83173f7-95e4-40ad-9748-e13b2a518fc6/#expsets-table](https://data.4dnucleome.org/publications/e83173f7-95e4-40ad-9748-e13b2a518fc6/#expsets-table)). Analyzed imaging and sequencing data are available at [https://campuspress.yale.edu/wanglab/Cancer3DGenome/](https://campuspress.yale.edu/wanglab/Cancer3DGenome/). [Source data](https://www.nature.com/articles/s41588-025-02297-w#Sec46) are provided with this paper.

Code availability
-----------------

Open-source code for imaging data collection is available at [https://github.com/ZhuangLab/storm-control](https://github.com/ZhuangLab/storm-control). All original codes generated for this study are available at [https://campuspress.yale.edu/wanglab/Cancer3DGenome/](https://campuspress.yale.edu/wanglab/Cancer3DGenome/) and on Zenodo via [https://doi.org/10.5281/zenodo.15527545](https://doi.org/10.5281/zenodo.15527545) (ref. [81](https://www.nature.com/articles/s41588-025-02297-w#ref-CR81 "Liu, M. Tracing the evolution of single-cell 3D genomes in Kras-driven cancers. Zenodo 
https://doi.org/10.5281/ZENODO.15527545
(2025).")).

References
----------

1.  Zink, D., Fischer, A. H. & Nickerson, J. A. Nuclear structure in cancer cells. _Nat. Rev. Cancer_ **4**, 677–687 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXntFCmsbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15343274)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nuclear%20structure%20in%20cancer%20cells&journal=Nat.%20Rev.%20Cancer&volume=4&pages=677-687&publication_year=2004&author=Zink%2CD&author=Fischer%2CAH&author=Nickerson%2CJA) 
    
2.  Krijger, P. H. L. & de Laat, W. Regulation of disease-associated gene expression in the 3D genome. _Nat. Rev. Mol. Cell Biol._ **17**, 771–782 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVWhsr3L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27826147)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulation%20of%20disease-associated%20gene%20expression%20in%20the%203D%20genome&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=17&pages=771-782&publication_year=2016&author=Krijger%2CPHL&author=Laat%2CW) 
    
3.  Zheng, H. & Xie, W. The role of 3D genome organization in development and cell differentiation. _Nat. Rev. Mol. Cell Biol._ **20**, 535–550 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlOgur7J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31197269)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%203D%20genome%20organization%20in%20development%20and%20cell%20differentiation&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=20&pages=535-550&publication_year=2019&author=Zheng%2CH&author=Xie%2CW) 
    
4.  Yu, M. & Ren, B. The three-dimensional organization of mammalian genomes. _Annu. Rev. Cell Dev. Biol._ **33**, 265–289 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1yltbrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28783961)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5837811)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20three-dimensional%20organization%20of%20mammalian%20genomes&journal=Annu.%20Rev.%20Cell%20Dev.%20Biol.&volume=33&pages=265-289&publication_year=2017&author=Yu%2CM&author=Ren%2CB) 
    
5.  Bickmore, W. A. & van Steensel, B. Genome architecture: domain organization of interphase chromosomes. _Cell_ **152**, 1270–1284 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXktFWqtbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23498936)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20architecture%3A%20domain%20organization%20of%20interphase%20chromosomes&journal=Cell&volume=152&pages=1270-1284&publication_year=2013&author=Bickmore%2CWA&author=Steensel%2CB) 
    
6.  Van Steensel, B. & Furlong, E. E. M. The role of transcription in shaping the spatial organization of the genome. _Nat. Rev. Mol. Cell Biol._ **20**, 327–337 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmslKgtLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30886333)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7116054)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20transcription%20in%20shaping%20the%20spatial%20organization%20of%20the%20genome&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=20&pages=327-337&publication_year=2019&author=Van%20Steensel%2CB&author=Furlong%2CEEM) 
    
7.  Bonora, G., Plath, K. & Denholtz, M. A mechanistic link between gene regulation and genome architecture in mammalian development. _Curr. Opin. Genet. Dev._ **27**, 92–101 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Oit73L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24998386)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20mechanistic%20link%20between%20gene%20regulation%20and%20genome%20architecture%20in%20mammalian%20development&journal=Curr.%20Opin.%20Genet.%20Dev.&volume=27&pages=92-101&publication_year=2014&author=Bonora%2CG&author=Plath%2CK&author=Denholtz%2CM) 
    
8.  Mir, M., Bickmore, W., Furlong, E. E. M. & Narlikar, G. Chromatin topology, condensates and gene regulation: shifting paradigms or just a phase? _Development_ **146**, dev182766 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1Gjt7vP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31554625)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6803379)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20topology%2C%20condensates%20and%20gene%20regulation%3A%20shifting%20paradigms%20or%20just%20a%20phase%3F&journal=Development&volume=146&publication_year=2019&author=Mir%2CM&author=Bickmore%2CW&author=Furlong%2CEEM&author=Narlikar%2CG) 
    
9.  Misteli, T. The self-organizing genome: principles of genome architecture and function. _Cell_ **183**, 28–45 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFGqsL%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32976797)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7541718)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20self-organizing%20genome%3A%20principles%20of%20genome%20architecture%20and%20function&journal=Cell&volume=183&pages=28-45&publication_year=2020&author=Misteli%2CT) 
    
10.  Hu, M. & Wang, S. Chromatin tracing: imaging 3D genome and nucleome. _Trends Cell Biol._ **31**, 5–8 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlentbfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33191055)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20tracing%3A%20imaging%203D%20genome%20and%20nucleome&journal=Trends%20Cell%20Biol.&volume=31&pages=5-8&publication_year=2021&author=Hu%2CM&author=Wang%2CS) 
    
11.  Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. _Science_ **353**, 598–602 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Oks7vF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27445307)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4991974)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatial%20organization%20of%20chromatin%20domains%20and%20compartments%20in%20single%20chromosomes&journal=Science&volume=353&pages=598-602&publication_year=2016&author=Wang%2CS) 
    
12.  Hnisz, D. et al. Activation of proto-oncogenes by disruption of chromosome neighborhoods. _Science_ **351**, 1454–1458 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XksFCht78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26940867)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4884612)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Activation%20of%20proto-oncogenes%20by%20disruption%20of%20chromosome%20neighborhoods&journal=Science&volume=351&pages=1454-1458&publication_year=2016&author=Hnisz%2CD) 
    
13.  Flavahan, W. A. et al. Insulator dysfunction and oncogene activation in _IDH_ mutant gliomas. _Nature_ **529**, 110–114 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitVyntbzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26700815)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insulator%20dysfunction%20and%20oncogene%20activation%20in%20IDH%20mutant%20gliomas&journal=Nature&volume=529&pages=110-114&publication_year=2016&author=Flavahan%2CWA) 
    
14.  Flavahan, W. A. et al. Altered chromosomal topology drives oncogenic programs in SDH-deficient GISTs. _Nature_ **575**, 229–233 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2hsb7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31666694)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6913936)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Altered%20chromosomal%20topology%20drives%20oncogenic%20programs%20in%20SDH-deficient%20GISTs&journal=Nature&volume=575&pages=229-233&publication_year=2019&author=Flavahan%2CWA) 
    
15.  Johnstone, S. E. et al. Large-scale topological changes restrain malignant progression in colorectal cancer. _Cell_ **182**, 1474–1489 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1yltLvK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32841603)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7575124)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large-scale%20topological%20changes%20restrain%20malignant%20progression%20in%20colorectal%20cancer&journal=Cell&volume=182&pages=1474-1489&publication_year=2020&author=Johnstone%2CSE) 
    
16.  Xu, J. et al. Subtype-specific 3D genome alteration in acute myeloid leukaemia. _Nature_ **611**, 387–398 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XislCjsLbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36289338)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10060167)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Subtype-specific%203D%20genome%20alteration%20in%20acute%20myeloid%20leukaemia&journal=Nature&volume=611&pages=387-398&publication_year=2022&author=Xu%2CJ) 
    
17.  Mathur, R. et al. Glioblastoma evolution and heterogeneity from a 3D whole-tumor perspective. _Cell_ **187**, 446–463 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhslCjur0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38242087)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10832360)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Glioblastoma%20evolution%20and%20heterogeneity%20from%20a%203D%20whole-tumor%20perspective&journal=Cell&volume=187&pages=446-463&publication_year=2024&author=Mathur%2CR) 
    
18.  Su, J.-H., Zheng, P., Kinrot, S. S., Bintu, B. & Zhuang, X. Genome-scale imaging of the 3D organization and transcriptional activity of chromatin. _Cell_ **182**, 1641–1659 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Gqsr%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32822575)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7851072)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-scale%20imaging%20of%20the%203D%20organization%20and%20transcriptional%20activity%20of%20chromatin&journal=Cell&volume=182&pages=1641-1659&publication_year=2020&author=Su%2CJ-H&author=Zheng%2CP&author=Kinrot%2CSS&author=Bintu%2CB&author=Zhuang%2CX) 
    
19.  Ferone, G., Lee, M. C., Sage, J. & Berns, A. Cells of origin of lung cancers: lessons from mouse studies. _Genes Dev._ **34**, 1017–1032 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlOlur7P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32747478)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7397855)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cells%20of%20origin%20of%20lung%20cancers%3A%20lessons%20from%20mouse%20studies&journal=Genes%20Dev.&volume=34&pages=1017-1032&publication_year=2020&author=Ferone%2CG&author=Lee%2CMC&author=Sage%2CJ&author=Berns%2CA) 
    
20.  Kaiser, A. M. et al. p53 governs an AT1 differentiation programme in lung cancer suppression. _Nature_ **619**, 851–859 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFSmtL%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37468633)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11288504)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=p53%20governs%20an%20AT1%20differentiation%20programme%20in%20lung%20cancer%20suppression&journal=Nature&volume=619&pages=851-859&publication_year=2023&author=Kaiser%2CAM) 
    
21.  Han, G. et al. An atlas of epithelial cell states and plasticity in lung adenocarcinoma. _Nature_ **627**, 656–663 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXkslClurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38418883)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10954546)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20atlas%20of%20epithelial%20cell%20states%20and%20plasticity%20in%20lung%20adenocarcinoma&journal=Nature&volume=627&pages=656-663&publication_year=2024&author=Han%2CG) 
    
22.  Cremer, T. & Cremer, C. Chromosome territories, nuclear architecture and gene regulation in mammalian cells. _Nat. Rev. Genet._ **2**, 292–301 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXisVCqurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11283701)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosome%20territories%2C%20nuclear%20architecture%20and%20gene%20regulation%20in%20mammalian%20cells&journal=Nat.%20Rev.%20Genet.&volume=2&pages=292-301&publication_year=2001&author=Cremer%2CT&author=Cremer%2CC) 
    
23.  Takei, Y. et al. Single-cell nuclear architecture across cell types in the mouse brain. _Science_ **374**, 586–594 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVarsrfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34591592)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20nuclear%20architecture%20across%20cell%20types%20in%20the%20mouse%20brain&journal=Science&volume=374&pages=586-594&publication_year=2021&author=Takei%2CY) 
    
24.  Liu, M. et al. Multiplexed imaging of nucleome architectures in single cells of mammalian tissue. _Nat. Commun._ **11**, 2907 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFCjtr%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32518300)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7283333)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiplexed%20imaging%20of%20nucleome%20architectures%20in%20single%20cells%20of%20mammalian%20tissue&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Liu%2CM) 
    
25.  The Cancer Genome Atlas Research Network. Comprehensive molecular profiling of lung adenocarcinoma. _Nature_ **511**, 543–550 (2014).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4231481)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20molecular%20profiling%20of%20lung%20adenocarcinoma&journal=Nature&volume=511&pages=543-550&publication_year=2014) 
    
26.  Slebos, R. J. C. et al. p53 alterations in atypical alveolar hyperplasia of the human lung. _Hum. Pathol._ **29**, 801–808 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXls1Kmu7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9712420)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=p53%20alterations%20in%20atypical%20alveolar%20hyperplasia%20of%20the%20human%20lung&journal=Hum.%20Pathol.&volume=29&pages=801-808&publication_year=1998&author=Slebos%2CRJC) 
    
27.  Vogelstein, B. et al. Cancer genome landscapes. _Science_ **339**, 1546–1558 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXksFOhsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23539594)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3749880)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cancer%20genome%20landscapes&journal=Science&volume=339&pages=1546-1558&publication_year=2013&author=Vogelstein%2CB) 
    
28.  Zong, H., Espinosa, J. S., Su, H. H., Muzumdar, M. D. & Luo, L. Mosaic analysis with double markers in mice. _Cell_ **121**, 479–492 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXktlShsr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15882628)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mosaic%20analysis%20with%20double%20markers%20in%20mice&journal=Cell&volume=121&pages=479-492&publication_year=2005&author=Zong%2CH&author=Espinosa%2CJS&author=Su%2CHH&author=Muzumdar%2CMD&author=Luo%2CL) 
    
29.  Muzumdar, M. D., Luo, L. & Zong, H. Modeling sporadic loss of heterozygosity in mice by using mosaic analysis with double markers (MADM). _Proc. Natl Acad. Sci. USA_ **104**, 4495–4500 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjsFKmtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17360552)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1810340)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Modeling%20sporadic%20loss%20of%20heterozygosity%20in%20mice%20by%20using%20mosaic%20analysis%20with%20double%20markers%20%28MADM%29&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=104&pages=4495-4500&publication_year=2007&author=Muzumdar%2CMD&author=Luo%2CL&author=Zong%2CH) 
    
30.  Muzumdar, M. D. et al. Clonal dynamics following _p53_ loss of heterozygosity in _Kras_\-driven cancers. _Nat. Commun._ **7**, 12685 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVOru7bK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27585860)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5025814)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20dynamics%20following%20p53%20loss%20of%20heterozygosity%20in%20Kras-driven%20cancers&journal=Nat.%20Commun.&volume=7&publication_year=2016&author=Muzumdar%2CMD) 
    
31.  Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. _Cell_ **173**, 371–385 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFaiu7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29625053)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6029450)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20characterization%20of%20cancer%20driver%20genes%20and%20mutations&journal=Cell&volume=173&pages=371-385&publication_year=2018&author=Bailey%2CMH) 
    
32.  McFadden, D. G. et al. Mutational landscape of _EGFR-_, _MYC-_, and _Kras-_driven genetically engineered mouse models of lung adenocarcinoma. _Proc. Natl Acad. Sci. USA_ **113**, E6409–E6417 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1Srt7bF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27702896)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5081629)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20landscape%20of%20EGFR-%2C%20MYC-%2C%20and%20Kras-driven%20genetically%20engineered%20mouse%20models%20of%20lung%20adenocarcinoma&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=113&pages=E6409-E6417&publication_year=2016&author=McFadden%2CDG) 
    
33.  Raphael, B. J. et al. Integrated genomic characterization of pancreatic ductal adenocarcinoma. _Cancer Cell_ **32**, 185–203 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20genomic%20characterization%20of%20pancreatic%20ductal%20adenocarcinoma&journal=Cancer%20Cell&volume=32&pages=185-203&publication_year=2017&author=Raphael%2CBJ) 
    
34.  Australian Pancreatic Cancer Genome Initiative et al. Genomic analyses identify molecular subtypes of pancreatic cancer. _Nature_ **531**, 47–52 (2016).
    
35.  Tan, L. et al. Changes in genome architecture and transcriptional dynamics progress independently of sensory experience during post-natal brain development. _Cell_ **184**, 741–758 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aks7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33484631)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Changes%20in%20genome%20architecture%20and%20transcriptional%20dynamics%20progress%20independently%20of%20sensory%20experience%20during%20post-natal%20brain%20development&journal=Cell&volume=184&pages=741-758&publication_year=2021&author=Tan%2CL) 
    
36.  Tan, L., Xing, D., Chang, C.-H., Li, H. & Xie, X. S. Three-dimensional genome structures of single diploid human cells. _Science_ **361**, 924–928 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2mt7fK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30166492)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6360088)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Three-dimensional%20genome%20structures%20of%20single%20diploid%20human%20cells&journal=Science&volume=361&pages=924-928&publication_year=2018&author=Tan%2CL&author=Xing%2CD&author=Chang%2CC-H&author=Li%2CH&author=Xie%2CXS) 
    
37.  Van der Maaten, L. & Hinton, G. Visualizing data using t-SNE. _J. Mach. Learn. Res._ **9**, 2579–2605 (2008).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20data%20using%20t-SNE&journal=J.%20Mach.%20Learn.%20Res.&volume=9&pages=2579-2605&publication_year=2008&author=Maaten%2CL&author=Hinton%2CG) 
    
38.  McInnes, L., Healy, J., Saul, N. & Großberger, L. UMAP: uniform manifold approximation and projection. _J. Open Source Softw._ **3**, 861 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UMAP%3A%20uniform%20manifold%20approximation%20and%20projection&journal=J.%20Open%20Source%20Softw.&volume=3&publication_year=2018&author=McInnes%2CL&author=Healy%2CJ&author=Saul%2CN&author=Gro%C3%9Fberger%2CL) 
    
39.  Wang, Y., Huang, H., Rudin, C. & Shaposhnik, Y. Understanding how dimension reduction tools work: an empirical approach to deciphering t-SNE, UMAP, TriMAP, and PaCMAP for data visualization. _J. Mach. Learn. Res._ **22**, 9129–9201 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%20how%20dimension%20reduction%20tools%20work%3A%20an%20empirical%20approach%20to%20deciphering%20t-SNE%2C%20UMAP%2C%20TriMAP%2C%20and%20PaCMAP%20for%20data%20visualization&journal=J.%20Mach.%20Learn.%20Res.&volume=22&pages=9129-9201&publication_year=2021&author=Wang%2CY&author=Huang%2CH&author=Rudin%2CC&author=Shaposhnik%2CY) 
    
40.  Traag, V. A., Waltman, L. & van Eck, N. J. From Louvain to Leiden: guaranteeing well-connected communities. _Sci. Rep._ **9**, 5233 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB3cbosVGisQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30914743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6435756)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=From%20Louvain%20to%20Leiden%3A%20guaranteeing%20well-connected%20communities&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Traag%2CVA&author=Waltman%2CL&author=Eck%2CNJ) 
    
41.  Willemin, A., Szabó, D. & Pombo, A. Epigenetic regulatory layers in the 3D nucleus. _Mol. Cell_ **84**, 415–428 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsleqtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38242127)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10872226)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenetic%20regulatory%20layers%20in%20the%203D%20nucleus&journal=Mol.%20Cell&volume=84&pages=415-428&publication_year=2024&author=Willemin%2CA&author=Szab%C3%B3%2CD&author=Pombo%2CA) 
    
42.  Martínez-Ruiz, C. et al. Genomic–transcriptomic evolution in lung cancer and metastasis. _Nature_ **616**, 543–552 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37046093)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10115639)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%E2%80%93transcriptomic%20evolution%20in%20lung%20cancer%20and%20metastasis&journal=Nature&volume=616&pages=543-552&publication_year=2023&author=Mart%C3%ADnez-Ruiz%2CC) 
    
43.  Marjanovic, N. D. et al. Emergence of a high-plasticity cell state during lung cancer evolution. _Cancer Cell_ **38**, 229–246 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVClu7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32707077)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7745838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Emergence%20of%20a%20high-plasticity%20cell%20state%20during%20lung%20cancer%20evolution&journal=Cancer%20Cell&volume=38&pages=229-246&publication_year=2020&author=Marjanovic%2CND) 
    
44.  Tsherniak, A. et al. Defining a cancer dependency map. _Cell_ **170**, 564–576 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1GisLjP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28753430)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5667678)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defining%20a%20cancer%20dependency%20map&journal=Cell&volume=170&pages=564-576&publication_year=2017&author=Tsherniak%2CA) 
    
45.  DuPage, M., Dooley, A. L. & Jacks, T. Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase. _Nat. Protoc._ **4**, 1064–1072 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXovFSrtLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19561589)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2757265)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conditional%20mouse%20lung%20cancer%20models%20using%20adenoviral%20or%20lentiviral%20delivery%20of%20Cre%20recombinase&journal=Nat.%20Protoc.&volume=4&pages=1064-1072&publication_year=2009&author=DuPage%2CM&author=Dooley%2CAL&author=Jacks%2CT) 
    
46.  Dempster, J. M. et al. Chronos: a cell population dynamics model of CRISPR experiments that improves inference of gene fitness effects. _Genome Biol._ **22**, 343 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34930405)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8686573)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chronos%3A%20a%20cell%20population%20dynamics%20model%20of%20CRISPR%20experiments%20that%20improves%20inference%20of%20gene%20fitness%20effects&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Dempster%2CJM) 
    
47.  Pacini, C. et al. Integrated cross-study datasets of genetic dependencies in cancer. _Nat. Commun._ **12**, 1661 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmslKis7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33712601)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7955067)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20cross-study%20datasets%20of%20genetic%20dependencies%20in%20cancer&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Pacini%2CC) 
    
48.  Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR–Cas9 essentiality screens in cancer cells. _Nat. Genet._ **49**, 1779–1784 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslehtrjI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29083409)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computational%20correction%20of%20copy%20number%20effect%20improves%20specificity%20of%20CRISPR%E2%80%93Cas9%20essentiality%20screens%20in%20cancer%20cells&journal=Nat.%20Genet.&volume=49&pages=1779-1784&publication_year=2017&author=Meyers%2CRM) 
    
49.  Dempster, J. M. et al. Extracting biological insights from the Project Achilles genome-scale CRISPR screens in cancer cell lines. Preprint at _bioRxiv_ [https://doi.org/10.1101/720243](https://doi.org/10.1101/720243) (2019).
    
50.  Cheng, Y., Liu, M., Hu, M. & Wang, S. TAD-like single-cell domain structures exist on both active and inactive X chromosomes and persist under epigenetic perturbations. _Genome Biol._ **22**, 309 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XksFaqtL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34749781)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8574027)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TAD-like%20single-cell%20domain%20structures%20exist%20on%20both%20active%20and%20inactive%20X%20chromosomes%20and%20persist%20under%20epigenetic%20perturbations&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Cheng%2CY&author=Liu%2CM&author=Hu%2CM&author=Wang%2CS) 
    
51.  Hafner, A. et al. Loop stacking organizes genome folding from TADs to chromosomes. _Mol. Cell_ **83**, 1377–1392 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXpt1emtLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37146570)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10167645)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loop%20stacking%20organizes%20genome%20folding%20from%20TADs%20to%20chromosomes&journal=Mol.%20Cell&volume=83&pages=1377-1392&publication_year=2023&author=Hafner%2CA) 
    
52.  Matsumoto, Y., Yasuda, H., Mita, S., Marunouchit, T. & Yamada, M. Evidence for the involvement of Hl histone phosphorylation in chromosome condensation. _Nature_ **284**, 181–183 (1980).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3cXkt1entrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7360244)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evidence%20for%20the%20involvement%20of%20Hl%20histone%20phosphorylation%20in%20chromosome%20condensation&journal=Nature&volume=284&pages=181-183&publication_year=1980&author=Matsumoto%2CY&author=Yasuda%2CH&author=Mita%2CS&author=Marunouchit%2CT&author=Yamada%2CM) 
    
53.  Sodir, N. M. et al. Reversible Myc hypomorphism identifies a key Myc-dependency in early cancer evolution. _Nat. Commun._ **13**, 6782 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivVOisLfK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36351945)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9646778)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reversible%20Myc%20hypomorphism%20identifies%20a%20key%20Myc-dependency%20in%20early%20cancer%20evolution&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Sodir%2CNM) 
    
54.  Tran, P. T. et al. Combined inactivation of MYC and K-Ras oncogenes reverses tumorigenesis in lung adenocarcinomas and lymphomas. _PLoS ONE_ **3**, e2125 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18461184)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2365560)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combined%20inactivation%20of%20MYC%20and%20K-Ras%20oncogenes%20reverses%20tumorigenesis%20in%20lung%20adenocarcinomas%20and%20lymphomas&journal=PLoS%20ONE&volume=3&publication_year=2008&author=Tran%2CPT) 
    
55.  Ma, W. et al. BARTweb: a web server for transcriptional regulator association analysis. _NAR Genom. Bioinform._ **3**, lqab022 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33860225)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8034776)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BARTweb%3A%20a%20web%20server%20for%20transcriptional%20regulator%20association%20analysis&journal=NAR%20Genom.%20Bioinform.&volume=3&publication_year=2021&author=Ma%2CW) 
    
56.  Wang, Z. et al. BART: a transcription factor prediction tool with query gene sets or epigenomic profiles. _Bioinformatics_ **34**, 2867–2869 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFSrsb7I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29608647)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6084568)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BART%3A%20a%20transcription%20factor%20prediction%20tool%20with%20query%20gene%20sets%20or%20epigenomic%20profiles&journal=Bioinformatics&volume=34&pages=2867-2869&publication_year=2018&author=Wang%2CZ) 
    
57.  Wang, H. et al. Role of histone H2A ubiquitination in Polycomb silencing. _Nature_ **431**, 873–878 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXotl2ktrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15386022)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20histone%20H2A%20ubiquitination%20in%20Polycomb%20silencing&journal=Nature&volume=431&pages=873-878&publication_year=2004&author=Wang%2CH) 
    
58.  Blackledge, N. P., Rose, N. R. & Klose, R. J. Targeting polycomb systems to regulate gene expression: modifications to a complex story. _Nat. Rev. Mol. Cell Biol._ **16**, 643–649 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFyqsL3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26420232)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5469428)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20polycomb%20systems%20to%20regulate%20gene%20expression%3A%20modifications%20to%20a%20complex%20story&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=16&pages=643-649&publication_year=2015&author=Blackledge%2CNP&author=Rose%2CNR&author=Klose%2CRJ) 
    
59.  Aranda, S., Mas, G. & Di Croce, L. Regulation of gene transcription by polycomb proteins. _Sci. Adv._ **1**, e1500737 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26665172)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4672759)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regulation%20of%20gene%20transcription%20by%20polycomb%20proteins&journal=Sci.%20Adv.&volume=1&publication_year=2015&author=Aranda%2CS&author=Mas%2CG&author=Croce%2CL) 
    
60.  Rai, K. et al. Dual roles of RNF2 in melanoma progression. _Cancer Discov._ **5**, 1314–1327 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvF2ltbzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26450788)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4670809)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dual%20roles%20of%20RNF2%20in%20melanoma%20progression&journal=Cancer%20Discov.&volume=5&pages=1314-1327&publication_year=2015&author=Rai%2CK) 
    
61.  Vieux-Rochas, M., Fabre, P. J., Leleu, M., Duboule, D. & Noordermeer, D. Clustering of mammalian _Hox_ genes with other H3K27me3 targets within an active nuclear domain. _Proc. Natl Acad. Sci. USA_ **112**, 4672–4677 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsVOnsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25825760)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4403207)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clustering%20of%20mammalian%20Hox%20genes%20with%20other%20H3K27me3%20targets%20within%20an%20active%20nuclear%20domain&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=112&pages=4672-4677&publication_year=2015&author=Vieux-Rochas%2CM&author=Fabre%2CPJ&author=Leleu%2CM&author=Duboule%2CD&author=Noordermeer%2CD) 
    
62.  Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. _Science_ **326**, 289–293 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1agsbjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19815776)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2858594)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&journal=Science&volume=326&pages=289-293&publication_year=2009&author=Lieberman-Aiden%2CE) 
    
63.  Ben-Saadon, R., Zaaroor, D., Ziv, T. & Ciechanover, A. The polycomb protein Ring1B generates self atypical mixed ubiquitin chains required for its in vitro histone H2A ligase activity. _Mol. Cell_ **24**, 701–711 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtlCns7zO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17157253)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20polycomb%20protein%20Ring1B%20generates%20self%20atypical%20mixed%20ubiquitin%20chains%20required%20for%20its%20in%20vitro%20histone%20H2A%20ligase%20activity&journal=Mol.%20Cell&volume=24&pages=701-711&publication_year=2006&author=Ben-Saadon%2CR&author=Zaaroor%2CD&author=Ziv%2CT&author=Ciechanover%2CA) 
    
64.  Nabet, B. et al. The dTAG system for immediate and target-specific protein degradation. _Nat. Chem. Biol._ **14**, 431–441 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXosFGmsbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29581585)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6295913)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20dTAG%20system%20for%20immediate%20and%20target-specific%20protein%20degradation&journal=Nat.%20Chem.%20Biol.&volume=14&pages=431-441&publication_year=2018&author=Nabet%2CB) 
    
65.  Ou, H. D. et al. ChromEMT: visualizing 3D chromatin structure and compaction in interphase and mitotic cells. _Science_ **357**, eaag0025 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28751582)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5646685)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ChromEMT%3A%20visualizing%203D%20chromatin%20structure%20and%20compaction%20in%20interphase%20and%20mitotic%20cells&journal=Science&volume=357&publication_year=2017&author=Ou%2CHD) 
    
66.  Bintu, B. et al. Super-resolution chromatin tracing reveals domains and cooperative interactions in single cells. _Science_ **362**, eaau1783 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30361340)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6535145)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Super-resolution%20chromatin%20tracing%20reveals%20domains%20and%20cooperative%20interactions%20in%20single%20cells&journal=Science&volume=362&publication_year=2018&author=Bintu%2CB) 
    
67.  Takei, Y. et al. Integrated spatial genomics reveals global architecture of single nuclei. _Nature_ **590**, 344–350 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlSmsro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33505024)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7878433)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20spatial%20genomics%20reveals%20global%20architecture%20of%20single%20nuclei&journal=Nature&volume=590&pages=344-350&publication_year=2021&author=Takei%2CY) 
    
68.  Payne, A. C. et al. In situ genome sequencing resolves DNA sequence and structure in intact biological samples. _Science_ **371**, eaay3446 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXltlaltLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33384301)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In%20situ%20genome%20sequencing%20resolves%20DNA%20sequence%20and%20structure%20in%20intact%20biological%20samples&journal=Science&volume=371&publication_year=2021&author=Payne%2CAC) 
    
69.  LaFave, L. M. et al. Epigenomic state transitions characterize tumor progression in mouse lung adenocarcinoma. _Cancer Cell_ **38**, 212–228 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVClu7jL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32707078)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7641015)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenomic%20state%20transitions%20characterize%20tumor%20progression%20in%20mouse%20lung%20adenocarcinoma&journal=Cancer%20Cell&volume=38&pages=212-228&publication_year=2020&author=LaFave%2CLM) 
    
70.  TRACERx Consortium. et al. A clonal expression biomarker associates with lung cancer mortality. _Nat. Med._ **25**, 1540–1548 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20clonal%20expression%20biomarker%20associates%20with%20lung%20cancer%20mortality&journal=Nat.%20Med.&volume=25&pages=1540-1548&publication_year=2019) 
    
71.  Moffitt, J. R. et al. Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region. _Science_ **362**, eaau5324 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30385464)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6482113)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%2C%20spatial%2C%20and%20functional%20single-cell%20profiling%20of%20the%20hypothalamic%20preoptic%20region&journal=Science&volume=362&publication_year=2018&author=Moffitt%2CJR) 
    
72.  Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S. & Zhuang, X. Spatially resolved, highly multiplexed RNA profiling in single cells. _Science_ **348**, aaa6090 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25858977)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4662681)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatially%20resolved%2C%20highly%20multiplexed%20RNA%20profiling%20in%20single%20cells&journal=Science&volume=348&publication_year=2015&author=Chen%2CKH&author=Boettiger%2CAN&author=Moffitt%2CJR&author=Wang%2CS&author=Zhuang%2CX) 
    
73.  Fang, R. et al. Conservation and divergence of cortical cell organization in human and mouse revealed by MERFISH. _Science_ **377**, 56–62 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhslars7%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35771910)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9262715)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conservation%20and%20divergence%20of%20cortical%20cell%20organization%20in%20human%20and%20mouse%20revealed%20by%20MERFISH&journal=Science&volume=377&pages=56-62&publication_year=2022&author=Fang%2CR) 
    
74.  Allen, W. E., Blosser, T. R., Sullivan, Z. A., Dulac, C. & Zhuang, X. Molecular and spatial signatures of mouse brain aging at single-cell resolution. _Cell_ **186**, 194–208 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xjt1WitrvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36580914)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20and%20spatial%20signatures%20of%20mouse%20brain%20aging%20at%20single-cell%20resolution&journal=Cell&volume=186&pages=194-208&publication_year=2023&author=Allen%2CWE&author=Blosser%2CTR&author=Sullivan%2CZA&author=Dulac%2CC&author=Zhuang%2CX) 
    
75.  Lu, T., Ang, C. E. & Zhuang, X. Spatially resolved epigenomic profiling of single cells in complex tissues. _Cell_ **186**, 2275–2279 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXpvFarurs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37172568)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10234104)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatially%20resolved%20epigenomic%20profiling%20of%20single%20cells%20in%20complex%20tissues&journal=Cell&volume=186&pages=2275-2279&publication_year=2023&author=Lu%2CT&author=Ang%2CCE&author=Zhuang%2CX) 
    
76.  Lin, J.-R. et al. Highly multiplexed immunofluorescence imaging of human tissues and tumors using t-CyCIF and conventional optical microscopes. _eLife_ **7**, e31657 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29993362)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6075866)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Highly%20multiplexed%20immunofluorescence%20imaging%20of%20human%20tissues%20and%20tumors%20using%20t-CyCIF%20and%20conventional%20optical%20microscopes&journal=eLife&volume=7&publication_year=2018&author=Lin%2CJ-R) 
    
77.  Damo, M. et al. Inducible de novo expression of neoantigens in tumor cells and mice. _Nat. Biotechnol._ **39**, 64–73 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVKkt7rL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32719479)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inducible%20de%20novo%20expression%20of%20neoantigens%20in%20tumor%20cells%20and%20mice&journal=Nat.%20Biotechnol.&volume=39&pages=64-73&publication_year=2021&author=Damo%2CM) 
    
78.  Davis, C. A. et al. The encyclopedia of DNA elements (ENCODE): data portal update. _Nucleic Acids Res._ **46**, D794–D801 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlGisbzF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29126249)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20encyclopedia%20of%20DNA%20elements%20%28ENCODE%29%3A%20data%20portal%20update&journal=Nucleic%20Acids%20Res.&volume=46&pages=D794-D801&publication_year=2018&author=Davis%2CCA) 
    
79.  The ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. _Nature_ **489**, 57–74 (2012).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3439153)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&journal=Nature&volume=489&pages=57-74&publication_year=2012) 
    
80.  Ramírez, F., Dündar, F., Diehl, S., Grüning, B. A. & Manke, T. DeepTools: a flexible platform for exploring deep-sequencing data. _Nucleic Acids Res._ **42**, 187–191 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DeepTools%3A%20a%20flexible%20platform%20for%20exploring%20deep-sequencing%20data&journal=Nucleic%20Acids%20Res.&volume=42&pages=187-191&publication_year=2014&author=Ram%C3%ADrez%2CF&author=D%C3%BCndar%2CF&author=Diehl%2CS&author=Gr%C3%BCning%2CBA&author=Manke%2CT) 
    
81.  Liu, M. Tracing the evolution of single-cell 3D genomes in Kras-driven cancers. _Zenodo_ [https://doi.org/10.5281/ZENODO.15527545](https://doi.org/10.5281/ZENODO.15527545) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02297-w?format=refman&flavour=references)

Acknowledgements
----------------

We thank A. Xiao (Yale), M. King (Yale), B. Bintu (UCSD), P. Zheng (MIT), R. Homer (Yale), K. Politi (Yale), R. Herbst (Yale) and members of the Wang and Muzumdar laboratories for helpful discussions; M. Lemmon (Yale) for critical reading of the paper; the Yale Cancer Center (YCC) Functional Genomics Core for assistance with RNAi screening assays; Yale Pathology Tissue Services (YPTS) for histology support; J. Ishizuka (Yale) and D. Braun (Yale) for generously offering lab space for nuclei isolation for snRNA-seq; the Yale Center for Genome Analysis (YCGA) and Keck Microarray Shared Resource at Yale University for RNA and snRNA library preparation and sequencing of wild-type lung and mouse tumors; A. Bhutkar (MIT) and S. Blatt (MIT) for sharing the ssGSEA computational pipeline for TCGA survival analyses; the ENCODE Consortium and the laboratories of B. Ren (New York Genome Center) and J. Stamatoyannopoulos (UW) for generating the H3K4me1 and H3K4me3 ChIP–seq and DNase-seq datasets in adult mouse lung; N. Joshi (Yale) for providing the KP cell line; R. Weinberg (MIT), D. Trono (EPFL) and D. Schatz (Yale) for constructs; and T. Jacks (MIT) and L. Luo (Stanford) for mouse strains. S.S.A. and C.F.R. were supported by postdoctoral fellowships through the Yale Cancer Biology Training Program (T32CA193200). C.F.R. is supported by an NIH Research Supplement to Promote Diversity in Health-Related Research (R01CA276108-02S1). T.B.J. was in part supported by NIH training grant 5T32GM007205. J.S.D.R. was supported by an NIH Predoctoral Training Grant (2T32GM007499). J.P.T. recognizes support from NIH (R01LM013385). M.D.M. acknowledges support from an NIH Director’s New Innovator Award (DP2CA248136), AACR-Genentech NextGen Grant for Transformative Cancer Research (19-20-18-MUZU), Yale SPORE in Lung Cancer Career Enhancement Program award (P50CA196530) and, in part, the Yale Comprehensive Cancer Center Support Grant (P30CA016359). S.W. was partly supported by the NIH (UH3CA268202, U01CA260701, R01HG011245, DP2GM137414 and R01HG012969), Pershing Square Sohn Cancer Research Alliance, American Federation for Aging Research and Hevolution Foundation. This work was largely supported by an NCI IMAT program award (R33CA251037) and by grant R01CA292936 to M.D.M. and S.W.

Author information
------------------

Author notes

1.  These authors contributed equally: Miao Liu, Shengyan Jin, Sherry S. Agabiti.
    

### Authors and Affiliations

1.  Department of Genetics, Yale School of Medicine, Yale University, New Haven, CT, USA
    
    Miao Liu, Shengyan Jin, Sherry S. Agabiti, Tyler B. Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Mandar Deepak Muzumdar & Siyuan Wang
    
2.  Yale Cancer Biology Institute, Yale University, West Haven, CT, USA
    
    Sherry S. Agabiti, Christian F. Ruiz & Mandar Deepak Muzumdar
    
3.  M.D.-Ph.D. Program, Yale University, New Haven, CT, USA
    
    Tyler B. Jensen, Mandar Deepak Muzumdar & Siyuan Wang
    
4.  Department of Biostatistics, Yale School of Public Health, Yale University, New Haven, CT, USA
    
    Moein Rajaei & Jeffrey P. Townsend
    
5.  Department of Chronic Disease Epidemiology, Yale School of Public Health, Yale University, New Haven, CT, USA
    
    Fang-Yong Li
    
6.  Yale Cancer Center, Yale University, New Haven, CT, USA
    
    Fang-Yong Li, Jeffrey P. Townsend, Mandar Deepak Muzumdar & Siyuan Wang
    
7.  Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT, USA
    
    Jeffrey P. Townsend
    
8.  Program in Genetics, Genomics, and Epigenetics, Yale Cancer Center, Yale University, New Haven, CT, USA
    
    Jeffrey P. Townsend & Mandar Deepak Muzumdar
    
9.  Department of Internal Medicine, Section of Medical Oncology, Yale School of Medicine, Yale University, New Haven, CT, USA
    
    Mandar Deepak Muzumdar
    
10.  Yale Combined Program in the Biological and Biomedical Sciences, Yale University, New Haven, CT, USA
    
    Mandar Deepak Muzumdar & Siyuan Wang
    
11.  Molecular Cell Biology, Genetics, and Development Program, Yale University, New Haven, CT, USA
    
    Mandar Deepak Muzumdar & Siyuan Wang
    
12.  Department of Cell Biology, Yale School of Medicine, Yale University, New Haven, CT, USA
    
    Siyuan Wang
    
13.  Biochemistry, Quantitative Biology, Biophysics, and Structural Biology Program, Yale University, New Haven, CT, USA
    
    Siyuan Wang
    
14.  Yale Center for RNA Science and Medicine, Yale School of Medicine, Yale University, New Haven, CT, USA
    
    Siyuan Wang
    
15.  Yale Liver Center, Yale School of Medicine, Yale University, New Haven, CT, USA
    
    Siyuan Wang
    

Authors

1.  Miao Liu
    
    [View author publications](https://www.nature.com/search?author=Miao%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Miao%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Miao%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Shengyan Jin
    
    [View author publications](https://www.nature.com/search?author=Shengyan%20Jin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shengyan%20Jin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shengyan%20Jin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Sherry S. Agabiti
    
    [View author publications](https://www.nature.com/search?author=Sherry%20S.%20Agabiti)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sherry%20S.%20Agabiti) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sherry%20S.%20Agabiti%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Tyler B. Jensen
    
    [View author publications](https://www.nature.com/search?author=Tyler%20B.%20Jensen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tyler%20B.%20Jensen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tyler%20B.%20Jensen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Tianqi Yang
    
    [View author publications](https://www.nature.com/search?author=Tianqi%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tianqi%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tianqi%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Jonathan S. D. Radda
    
    [View author publications](https://www.nature.com/search?author=Jonathan%20S.%20D.%20Radda)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jonathan%20S.%20D.%20Radda) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jonathan%20S.%20D.%20Radda%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Christian F. Ruiz
    
    [View author publications](https://www.nature.com/search?author=Christian%20F.%20Ruiz)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Christian%20F.%20Ruiz) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Christian%20F.%20Ruiz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Gabriel Baldissera
    
    [View author publications](https://www.nature.com/search?author=Gabriel%20Baldissera)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gabriel%20Baldissera) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gabriel%20Baldissera%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Moein Rajaei
    
    [View author publications](https://www.nature.com/search?author=Moein%20Rajaei)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Moein%20Rajaei) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Moein%20Rajaei%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Fang-Yong Li
    
    [View author publications](https://www.nature.com/search?author=Fang-Yong%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Fang-Yong%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Fang-Yong%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Jeffrey P. Townsend
    
    [View author publications](https://www.nature.com/search?author=Jeffrey%20P.%20Townsend)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jeffrey%20P.%20Townsend) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jeffrey%20P.%20Townsend%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Mandar Deepak Muzumdar
    
    [View author publications](https://www.nature.com/search?author=Mandar%20Deepak%20Muzumdar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mandar%20Deepak%20Muzumdar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mandar%20Deepak%20Muzumdar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Siyuan Wang
    
    [View author publications](https://www.nature.com/search?author=Siyuan%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Siyuan%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Siyuan%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

M.D.M. and S.W. conceived of and supervised the study. M.L., M.D.M. and S.W. designed the study with help from S.J. and S.S.A. M.L., S.J., S.S.A., T.B.J. and J.S.D.R. performed experiments. M.L., S.J., S.S.A., T.B.J., T.Y., C.F.R., G.B., M.R. and F.-Y.L. analyzed data with help from J.P.T., M.D.M. and S.W. M.L., S.J., S.S.A., T.B.J., M.D.M. and S.W. wrote the paper with input from all authors.

### Corresponding authors

Correspondence to [Mandar Deepak Muzumdar](mailto:mandar.muzumdar@yale.edu) or [Siyuan Wang](mailto:siyuan.wang@yale.edu).

Ethics declarations
-------------------

### Competing interests

S.W., M.L., M.D.M., S.J. and S.S.A. are inventors on a patent applied for by Yale University related to this work. M.D.M. received research funding from a Genentech-supported AACR grant and an honorarium from Nested Therapeutics. S.W. is a co-inventor on a patent applied for by Harvard University related to MERFISH. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewers for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Genome-scale chromatin tracing visualizes 3D genome organization in vivo.](https://www.nature.com/articles/s41588-025-02297-w/figures/7)

**a**, Pearson correlation coefficients of mean _cis_\-chromosomal interloci distances between WT datasets. **b**, Mean _cis_\-chromosomal interloci distances of WT dataset 3 versus mean _cis_\-chromosomal interloci distances of WT dataset 2. The red line is a fitted linear regression line. The black line is the _y_ = _x_ line. **c**, Mean _cis_\-chromosomal interloci spatial distance versus genomic distance for all pairs of genomic loci on each autosome in AT2 cells. Different pseudocolors represent different autosomes. _n_ = 6,039 intrachromosomal interloci pairs in **b**–**d**. _n_ = 4,806 WT AT2 cells in **c** and **d**. **d**, Power-law scaling of all 19 mouse autosomes (chr1–19) in WT mouse lung. **e**, Power-law scaling of chr19 in E14.5 mouse fetal liver. Data were re-analyzed from ref. [24](https://www.nature.com/articles/s41588-025-02297-w#ref-CR24 "Liu, M. et al. Multiplexed imaging of nucleome architectures in single cells of mammalian tissue. Nat. Commun. 11, 2907 (2020)."). **f**, Power-law scaling of all 20 mouse chromosomes (chr1–19, chrX) in the mouse brain inhibitory neurons expressing Vip. Data were re-analyzed from ref. [23](https://www.nature.com/articles/s41588-025-02297-w#ref-CR23 "Takei, Y. et al. Single-cell nuclear architecture across cell types in the mouse brain. Science 374, 586–594 (2021)."). **g**, Schematic illustration of the experimental procedure. The schematic in panel **g** was created with BioRender.com. **h**, Whole-section fluorescence images of WT mouse lung and _K-MADM-Trp53_ mouse lungs containing adenomas or LUAD. Results are representative of 5 biological replicates in WT (4 mice) and adenoma (4 mice) states, and 6 biological replicates in the LUAD (5 mice) state. **i**, Detection efficiency of each target genomic region along detected chromosome traces in each cancer state. **j**, Distribution of the trace length across individual chromosomes in cells of each cancer state. **k**, Distribution of detected loci number per cell. _n_ = 26,852 cells combining the five cell states. The horizontal lines of the box from top to bottom represent the 75th percentile, the median and the 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values at least 1.5 times interquartile range away from the top or bottom of the box. **l**, A and B compartment score profiles of example autosomes measured by Hi-C and chromatin tracing in mouse LUAD. **m**, Fraction of genomic regions with concordant A and B compartment assignment between Hi-C and chromatin tracing. For **d**–**f**, lines are fitted power-law functions, and S is the scaling factor.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM11)

### [Extended Data Fig. 2 Systematic changes of intrachromosomal compaction and chromatin conformation heterogeneity during lung cancer progression.](https://www.nature.com/articles/s41588-025-02297-w/figures/8)

**a**,**b**, Mean interloci distances (**a**) and coefficients of variation (COV) of interloci distances (**b**) of all autosomes in WT AT2 (_n_ = 4,806), _Trp53__+/+_ adenoma (AdenomaR, _n_ = 791), _Trp53__+/−_ adenoma (AdenomaY, _n_ = 1,603), _Trp53__−/−_ adenoma (AdenomaG, _n_ = 1,941) and _Trp53__−/−_ LUAD (_n_ = 17,711) cells.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM12)

### [Extended Data Fig. 3 Subtraction maps showing systematic differences of intrachromosomal compaction and chromatin conformation heterogeneity during lung cancer progression.](https://www.nature.com/articles/s41588-025-02297-w/figures/9)

**a**,**b**, Subtraction maps of mean interloci distances (**a**) and coefficients of variation (COV) of interloci distances (**b**) in all autosomes of _Trp53__+/+_ adenoma (AdenomaR, _n_ = 791), _Trp53__+/−_ adenoma (AdenomaY, _n_ = 1,603), _Trp53__−/−_ adenoma (AdenomaG, _n_ = 1,941) and _Trp53__−/−_ LUAD (_n_ = 17,711) cells minus those of WT AT2 (_n_ = 4,806) cells.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM13)

### [Extended Data Fig. 4 Cancer state-independent 3D genome features during lung cancer progression.](https://www.nature.com/articles/s41588-025-02297-w/figures/10)

**a**–**c**, Normalized _trans_\-chromosomal proximity frequency between genomic loci in AT2 (**a**), adenoma (**b**) and LUAD (**c**) cells. The proximity frequency between each pair of _trans_\-chromosomal genomic regions were normalized to the mean proximity frequency of all loci pairs of the two corresponding chromosomes. A cutoff distance of 800 nm was used for defining proximity. The genomic regions were re-ordered so that A compartment loci were grouped separately from B compartment loci. **d**–**f**, Distribution of normalized _trans_\-chromosomal proximity frequencies of pairs of A loci (A–A), pairs of B loci (B–B) and pairs of A and B loci (A–B) in AT2 (**d**), adenoma (**e**) and LUAD (**f**) cells. **g**–**i**, Normalized _trans_\-chromosomal proximity frequencies of A–A, B–B and A–B loci pairs in AT2 (**g**), adenoma (**h**) and LUAD (**i**) cells. The horizontal lines of each box from top to bottom represent the 75th percentile, the median and the 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values at least 1.5 times interquartile range away from the top or bottom of the box. _P_ values from two-sided Wilcoxon rank-sum tests are shown. **j**–**l**, The proximity frequency between each pair of _cis_\-chromosomal genomic regions as a function of their genomic distances in AT2 (**j**), adenoma (**k**) and LUAD (**l**) cells. **m**, Radial scores versus A–B compartment scores of genomic loci in the five cell states. The lines are fitted linear regression lines. Pearson correlation coefficients (_R_) and _P_ values from two-sided _t_\-tests are shown. _n_ = 4,806, 791, 1,603, 1,941 and 17,711 for WT AT2, AdenomaR, AdenomaY, AdenomaG and LUAD cells, respectively. Cell numbers are the same in all panels.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM14)

### [Extended Data Fig. 5 Exome and RNA sequencing analyses of lung tumors from _K-MADM-Trp53_ mice.](https://www.nature.com/articles/s41588-025-02297-w/figures/11)

**a**, Representative large GFP+ (green) tumor for whole-exome and RNA sequencing. Scale bar = 2.5 mm. Image is representative of 6 mice. **b**, Frequency of nonsynonymous single nucleotide variants (SNVs) per megabase (Mb) with variant allele fraction (VAF) > 5% in AdenomaG and LUAD. **c**,**d**, Heatmap (**c**) and distribution (**d**) of mean copy number of each autosome across AdenomaG and LUAD (_n_ = 6 tumors per group). Black lines (**d**) represent the median. **e**, Schematic illustration of snRNA-seq in _K-MADM-Trp53_ mice. The schematic in panel **e** was created with BioRender.com. **f**,**g**, UMAP of single-cell gene expression profiles in _K-MADM-Trp53_ lung tumors (_n_ = 6,300 cells). Different cell-type clusters were labeled with different colors (**f**) based on gene set enrichment patterns of adenoma (_n_ = 1,631) and LUAD (_n_ = 4,669)-specific genes (**g**). **h**, Distribution of coefficient of variation of mean inferred copy numbers of all genes on each autosome (_n_ = 19 autosomes) in adenoma and LUAD. _P_ value from two-sided Wilcoxon signed-rank test. Horizontal lines of each box represent the 75th percentile, median and 25th percentile. Whiskers indicate nonoutlier maximum and nonoutlier minimum, with outlier thresholds at 1.5 times interquartile range beyond box limits. **i**,**j**, Unsupervised hierarchical clustering of all expressed genes (**i**) segregates dissected green tumors into two clusters (_n_ = 6 tumors per cluster) identified as AdenomaG and LUAD based on known histologic progression markers (ref. [43](https://www.nature.com/articles/s41588-025-02297-w#ref-CR43 "Marjanovic, N. D. et al. Emergence of a high-plasticity cell state during lung cancer evolution. Cancer Cell 38, 229–246 (2020)."); **j**). Row-normalized expression counts are shown in heatmap (**i**). Horizontal lines in each violin in **j** represent the 75th percentile, median and 25th percentile. _P_ values were from two-sided Wilcoxon rank-sum tests. **k**, Gene set enrichment analysis using MSigDB Hallmarks (H1) shows top enriched gene sets for upregulated (red, log2 fold change > 2 and FDR < 0.05) and downregulated (blue, log2 fold change <−1 and FDR < 0.05) genes in LUAD over AdenomaG. _P_ values were from one-sided hypergeometric tests.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM15)

### [Extended Data Fig. 6 Genome-wide chromatin tracing of pancreatic ductal adenocarcinoma progression.](https://www.nature.com/articles/s41588-025-02297-w/figures/12)

**a**,**b**, Whole-section fluorescence images of a _K-MADM-Trp53_ pancreas with PanINs (**a**) and PDAC (**b**). **c**,**d**, Immunofluorescence staining (top) of the duct cell marker CK19 and DAPI staining in a field of view, each in **a** and **b**. Fluorescent protein imaging (bottom) of GFP, tdTomato and DAPI staining in the corresponding fields. Images are from two independent experiments. The images are maximum-intensity z-projections from 10-μm z-stacks. **e**, Matrix of mean interloci distances between all genomic loci in normal duct cells. _n_ = 1,529 cells. **f**, Mean interloci spatial distance versus genomic distance for all pairs of genomic loci on each autosome in duct cells. Different pseudocolors represent different autosomes. _n_ = 6,039 intrachromosomal interloci pairs. _n_ = 1,529 cells. **g**–**i**, Distributions of the decompaction, heterogeneity and demixing scores of each autosome (_n_ = 19 autosomes) in each pancreatic cell state. _n_ = 1,529, 123, 189, 361 and 475 for normal duct, PanIN R, PanIN Y, PanIN G and PDAC cells, respectively. Cell numbers of each cell state are identical in **g**–**k**. **j**, log2 fold change of mean interchromosomal distances, comparing each cancer state to normal duct cells. **k**, Distribution of the mean log2 fold change of radial scores of chr17, chr1 and chr19, comparing each cancer state to normal duct cells. _P_ values of two-sided Wilcoxon signed-rank tests comparing each cancer state to WT duct cells with matched genomic loci (**g**, **h**, **j** and **k**) or matched autosomes (**i**) are shown. In **g**–**k**, the horizontal lines of each box represent the 75th percentile, median and 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values away from the top or bottom of the box by more than 1.5 times interquartile range.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM16)

### [Extended Data Fig. 7 Subsampling analysis of chromatin folding changes during lung cancer progression.](https://www.nature.com/articles/s41588-025-02297-w/figures/13)

**a**–**c**, Distributions of the decompaction (mean interloci distance), heterogeneity (COV of interloci distance) and demixing scores of each autosome (_n_ = 19) in a randomly subsampled population of 100 cells per cell state. **d**, Distribution of interchromosomal distances in a randomly subsampled population of 100 cells per cell state. **e**, Distribution of the polarization indices of A and B compartments of each autosome (_n_ = 19) in a randomly subsampled population of 100 cells per cell state. **f**, Distribution of the radial scores of chr15, chr2 and chr10 in a randomly subsampled population of 100 cells per cell state. _P_ values of two-sided Wilcoxon signed-rank test (**a**–**c**, **e** and **f**) and two-sided Wilcoxon rank-sum test (**d**) are displayed. In **a**–**f**, the horizontal lines of each box from top to bottom represent the 75th percentile, median and 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values at least 1.5 times interquartile range away from the top or bottom of the box.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM17)

### [Extended Data Fig. 8 Fine-scale tracing of candidate drivers of cancer progression.](https://www.nature.com/articles/s41588-025-02297-w/figures/14)

**a**, Schematic illustration of high-resolution chromatin tracing targeting the _cis_\-regulatory regions of 15 genes, including CPD, _Kras_ and _Myc_. Distribution of detected loci number per cell is displayed. _n_ = 12,661 cells combining the five cell states. **b**, Distribution of the decompaction (mean interloci distance) scores of all target genes. _n_ = 2,328, 654, 558 and 9,121 for normal AT2, AdenomaR/Y, AdenomaG and LUAD cells, respectively. Cell numbers in each cell state are identical in **b**–**f**. _P_ values of two-sided Wilcoxon signed-rank tests are displayed. **c**, Pileup heatmap of normalized E**–**P distances centered around each E**–**P loop. E**–**P loops of CPD genes are called in LUAD cells. **d**, Distribution of the log2 fold change of the normalized distances between promoter and putative enhancers of CPD genes, comparing each cancer state to AT2 cells. _P_ value of two-sided Wilcoxon signed-rank test is displayed. **e**, Normalized interloci distance matrices of the target genomic regions surrounding the _Kras_, _Myc_ and _Foxa3_ (CPD) genes in AT2, adenoma and LUAD cell states. The green circles designate putative enhancer–promoter contacts. Putative enhancer and gene annotation tracks are aligned to the target regions. **f**, Distribution of the normalized distances between promoter and putative enhancers of the _Kras_ and _Myc_ genes. _P_ values of two-sided Wilcoxon rank-sum tests are displayed. **g**, Distribution of the normalized scA/B scores of _Myc_ gene-containing target genomic region in AT2 and adenoma cells. _P_ value of two-sided Wilcoxon rank-sum test is displayed. In **a**, **b**, **d**, **f** and **g**, the horizontal lines of each box from top to bottom represent the 75th percentile, median and 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values at least 1.5 times interquartile range away from the top or bottom of the box.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM18)

### [Extended Data Fig. 9 3D genome organization in cancer cells is largely independent of spatial proximity to immune cells.](https://www.nature.com/articles/s41588-025-02297-w/figures/15)

**a**–**c**, Mean log2 fold change of decompaction (**a**), heterogeneity (**b**) and demixing (**c**) scores of each chromosome, comparing AT2/cancer cells near (less than 10 μm) versus far (more than 10 μm) from CD45+ immune cells. The _P_ values (**a**,**b**) or FDR (**c**) with significance (<0.05) from two-sided Wilcoxon signed-rank test (**a**,**b**) or two-sided Levene’s test (**c**) are displayed. **d**, Distribution of polarization indices of A–B compartments in AT2/cancer cells near or far from immune cells. Two-sided Wilcoxon rank-sum test yielded no significant _P_ values (_P_ < 0.05). **e**, t-SNE plots of single-cell 3D chromatin conformations in AT2, adenoma and LUAD cells show no distinct clusters based on spatial proximity to immune cells. In **a**, **b** and **d**, the horizontal lines of each box from top to bottom represent the 75th percentile, median and 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values at least 1.5 times interquartile range away from the top or bottom of the box. In **a**\-**d**, _n_ = 3,806, 758, 1,563, 1,882 and 17,711 for WT AT2, AdenomaR, AdenomaY, AdenomaG and LUAD cells, respectively (excluding early AT2 and adenoma replicates without CD45 labeling). In **e**, _n_ = 2,719, 1,710 and 8,834 for WT AT2, Adenoma and LUAD cells, respectively (cells with at least 10 traces).

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM19)

### [Extended Data Fig. 10 Rnf2 regulates 3D genome organization through a ubiquitin ligase-independent activity.](https://www.nature.com/articles/s41588-025-02297-w/figures/16)

**a**, Western blot analysis of Rnf2 protein levels following rescue of shRNA knockdown (shRnf2-3) with stable transduction of Rnf2 WT, ubiquitin ligase dead Rnf2 I53S mutant or empty vector (EV) control. HSP90 is loading control. Results were repeated in three biological replicates. **b**, scA/B score changes from shRnf2-3 to shRnf2-3 + WT Rnf2 (left) and to shRnf2-3 + I53S Rnf2 (right) versus those from AdenomaG to LUAD. _n_ = 473 target genomic regions. Spearman correlation coefficients and _P_ values from two-sided _t_\-tests are shown. **c**, Western blot of Rnf2 and H2AK119ub in KP LUAD cells with Rnf2-dTAG after treatment with the dTAG-13 ligand or negative control ligand (a diastereomer of dTAG-13) at the designated times (0, 0.5 or 12 h). Results were repeated in three biological replicates. **d**, Immunofluorescence images of Rnf2 and H2AK119ub in KP LUAD cells with Rnf2-dTAG after treatment with dTAG-13 ligand or negative control at designated times. **e**, Quantification of the immunofluorescence intensities in **d**. _n_ = 461, 534, 177, 1,040, 214, 602, 251 and 1,168 cells from left to right. The horizontal lines of each box from top to bottom represent the 75th percentile, median and 25th percentile. Whiskers extend to the nonoutlier maximum and nonoutlier minimum. Outliers are defined as values at least 1.5 times interquartile range away from the top or bottom of the box. _P_ value from two-sided Wilcoxon rank-sum test is shown. **f**, scA/B score changes from Rnf2-degraded cells (dTAG-13) to Rnf2 nondegraded cells (dTAG-13 negative control) versus those from AdenomaG to LUAD and those from shRnf2 to shNTC, with _n_ = 473 target genomic regions. Spearman correlation coefficients and _P_ values from two-sided _t_\-tests are shown.

[Source data](https://www.nature.com/articles/s41588-025-02297-w#MOESM20)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM1_ESM.pdf)

Supplementary Note and Supplementary Fig. 1.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM2_ESM.pdf)

### [Supplementary Tables 1–7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM3_ESM.xlsx)

Supplementary Table 1: Template oligonucleotide probe libraries for genome-wide (Supplementary Table 1-1) and fine-scale (Supplementary Table 1-2) chromatin tracing and RNA MERFISH (Supplementary Table 1-3). Supplementary Table 2: Codebooks for all target genomic loci in genome-wide chromatin tracing (Supplementary Table 2-1) and RNA MERFISH (Supplementary Table 2-2), and coordinates of genomic features in genome-wide chromatin tracing (Supplementary Tables 2-1,2-3) and in fine-scale chromatin tracing (Supplementary Table 2-4). In ‘Supplementary Table 2-1’, the first column indicates which chromosome each target genomic region in genome-wide chromatin tracing is located in. The second and third columns indicate the start coordinate and end coordinate of the target genomic region. Each region corresponds to a topologically associating domain (TAD). The fourth column indicates the binary barcode for each genomic region. ‘Supplementary Table 2-2’ contains the gene name and binary barcode of each RNA MERFISH target gene. ‘Supplementary Table 2-3’ contains mitogen-activated protein kinase (MAPK) pathway gene, oncogene and tumor suppressor gene names and coordinates of super-enhancers (mm9) in all target TADs in genome-wide chromatin tracing. Each row corresponds to a target TAD in Supplementary Table 2-1. ‘Supplementary Table 2-4’ contains gene names and genomic coordinates of each target region (mm10) in fine-scale chromatin tracing. Supplementary Table 3: Oligo sequences for adaptors, common readout probes, blocking oligos and primers. The spreadsheet contains the following eight blocks: ‘large-scale adapter’, ‘fine-scale adapter’, ‘large- and fine-scale readout’, ‘blocking oligo’, ‘large- and fine-scale primer’, ‘RNA MERFISH adapter’, ‘RNA MERFISH readout’ and ‘RNA MERFISH primer’. ‘Large-scale adapter’ contains the sequences of all 100 adaptor oligos that can bind to the primary probes in genome-wide chromatin tracing and correspond to the 100 bits in the codebook. ‘Fine-scale adapter’ contains the sequences of loci-specific adaptor probes and gene-specific adaptor probes. ‘RNA MERFISH adapter’ contains sequences of the 16 adaptor oligos that can bind to the RNA MERFISH primary probes and correspond to the 16 bits in the codebook. ‘Large- and fine-scale readout’ and ‘RNA MERFISH readout’ contain the sequences of dye-conjugated readout probes that bind the corresponding adaptors. ‘Blocking oligo’ contains dye-free oligos that have the same sequences as the common readout oligos to block any unbound sites of the previous adaptors in genome-wide chromatin tracing, fine-scale chromatin tracing and RNA MERFISH. ‘Large- and fine-scale primer’ and ‘RNA MERFISH primer’ contain the forward and reverse primer sequences for the template probe library amplification and primary probe synthesis. Supplementary Table 4: Whole-exome sequencing analysis of lung tumors from _K-MADM-Trp53_ mice. Mutation calls of variants (variant allele fraction > 5%) for adenoma and LUAD tumors (_n_ = 6 tumors per group). This table includes mouse number (sample), variant chromosome, genomic position, reference base (REF), variant base (ALT), variant allele fraction (Tumor\_AF), number of reference read (AD\[0\]) and variant reads (AD\[1\]) in tumor and normal, gene, and protein-altering effect (missense, splice variant, stop\_gained). Supplementary Table 5: RNA-seq analysis of lung tumors from _K-MADM-Trp53_ mice. Differential expression analysis (DESeq2) of large green tumors derived from _K-MADM-Trp53_ mice and classified as AdenomaG and LUAD based on marker gene expression. For each gene, log2 normalized expression counts (DESeq2) for each sample, average expression counts across all samples, log2 fold change (LUAD versus AdenomaG), _P_ value and FDR (padj) are shown. Supplementary Table 6: shRNA sequences for lentiviral transduced stable cell lines to knock down candidate progression driver genes and sequence of the FKBPF36V and mScarlet insert. The table contains two blocks. Block 1 contains sequences for all the shRNAs of the candidate genes, positive controls and negative controls. The first column contains the gene names. The second column contains the shRNA sequences. The third column contains the hairpin numbers. Block 2 contains sequences for the FKBPF36V and mScarlet insert used for Rnf2 targeted degradation. Supplementary Table 7: List of candidate progression driver genes. References (PubMed IDs (PMID)) are listed for genes with prior literature evidence for a functional role in lung cancer pathogenesis.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM4_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM5_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM7_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM8_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM9_ESM.pdf)

Unprocessed western blots.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM12_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM13_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM14_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM15_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM16_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM17_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM18_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 9](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM19_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM20_ESM.pdf)

Unprocessed western blots.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02297-w/MediaObjects/41588_2025_2297_MOESM21_ESM.xlsx)

Statistical source data.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Tracing%20the%20evolution%20of%20single-cell%203D%20genomes%20in%20Kras-driven%20cancers&author=Miao%20Liu%20et%20al&contentID=10.1038%2Fs41588-025-02297-w&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-18&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02297-w)

### Cite this article

Liu, M., Jin, S., Agabiti, S.S. _et al._ Tracing the evolution of single-cell 3D genomes in Kras-driven cancers. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02297-w

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02297-w?format=refman&flavour=citation)

-   Received: 02 September 2024
    
-   Accepted: 16 July 2025
    
-   Published: 18 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02297-w
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02297-w.pdf)
