> id: https://www.nature.com/articles/s41588-025-02265-4

> link: https://www.nature.com/articles/s41588-025-02265-4

> title: NKX2-1 drives neuroendocrine transdifferentiation of prostate cancer via epigenetic and 3D chromatin remodeling

# NKX2-1 drives neuroendocrine transdifferentiation of prostate cancer via epigenetic and 3D chromatin remodeling
_Published on Mon, 21 Jul 2025 00:00:00 GMT_

Abstract
--------

A substantial amount of castration-resistant prostate cancer (CRPC) progresses into a neuroendocrine (NE) subtype, known as NEPC, which is associated with poor clinical outcomes. Here we report distinct three-dimensional chromatin architectures between NEPC and CRPC tumors, which were recapitulated by isogenic cell lines undergoing NE transformation (NET). Mechanistically, pioneer factors such as FOXA2 initiate binding at NE enhancers to mediate regional DNA demethylation and induce neural transcription factor (TF) _NKX2-1_ expression. NKX2-1 preferentially binds gene promoters and interacts with enhancer-bound FOXA2 through chromatin looping. NKX2-1 is highly expressed in NEPC and indispensable for NET of prostate cancer. NKX2-1/FOXA2 further recruits p300/CBP to activate NE enhancers, and pharmacological inhibition of p300/CBP effectively blunts NE gene expression and abolishes NEPC tumor growth. Taken together, our study reports a hierarchical network of TFs governed by NKX2-1 in critically regulating chromatin remodeling and driving luminal-to-NE transformation and suggests promising therapeutic approaches to mitigate NEPC.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-021-22139-7/MediaObjects/41467_2021_22139_Fig1_HTML.png)

### [Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer](https://www.nature.com/articles/s41467-021-22139-7?fromPaywallRec=false)

Article Open access 30 March 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-87441-2/MediaObjects/41598_2021_87441_Fig1_HTML.png)

### [Novel, non-invasive markers for detecting therapy induced neuroendocrine differentiation in castration-resistant prostate cancer patients](https://www.nature.com/articles/s41598-021-87441-2?fromPaywallRec=false)

Article Open access 15 April 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-021-26042-z/MediaObjects/41467_2021_26042_Fig1_HTML.png)

### [Subtype heterogeneity and epigenetic convergence in neuroendocrine prostate cancer](https://www.nature.com/articles/s41467-021-26042-z?fromPaywallRec=false)

Article Open access 01 October 2021

Main
----

Following extensive treatment with androgen receptor (AR) pathway inhibitors, advanced prostate cancer (PCa) frequently develops treatment resistance. Approximately 20% of these castration-resistant prostate cancers (CRPC) transdifferentiate to display, at least partially, neuroendocrine (NE) histological and molecular features, thus called neuroendocrine PCa (NEPC)[1](https://www.nature.com/articles/s41588-025-02265-4#ref-CR1 "Abida, W. et al. Genomic correlates of clinical outcome in advanced prostate cancer. Proc. Natl Acad. Sci. USA 116, 11428–11436 (2019)."),[2](https://www.nature.com/articles/s41588-025-02265-4#ref-CR2 "Aggarwal, R. et al. Clinical and genomic characterization of treatment-emergent small-cell neuroendocrine prostate cancer: a multi-institutional prospective study. J. Clin. Oncol. 36, 2492–2503 (2018)."),[3](https://www.nature.com/articles/s41588-025-02265-4#ref-CR3 "Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. Nat. Med. 22, 298–305 (2016)."),[4](https://www.nature.com/articles/s41588-025-02265-4#ref-CR4 "Conteduca, V. et al. Clinical features of neuroendocrine prostate cancer. Eur J. Cancer 121, 7–18 (2019)."). While genetic alterations of tumor-suppressor genes _TP53_ and _RB1_ are more frequent in NEPC[5](https://www.nature.com/articles/s41588-025-02265-4#ref-CR5 "Ku, S. Y. et al. Rb1 and Trp53 cooperate to suppress prostate cancer lineage plasticity, metastasis, and antiandrogen resistance. Science 355, 78–83 (2017)."),[6](https://www.nature.com/articles/s41588-025-02265-4#ref-CR6 "Mu, P. et al. SOX2 promotes lineage plasticity and antiandrogen resistance in TP53- and RB1-deficient prostate cancer. Science 355, 84–88 (2017)."),[7](https://www.nature.com/articles/s41588-025-02265-4#ref-CR7 "Zou, M. et al. Transdifferentiation as a mechanism of treatment resistance in a mouse model of castration-resistant prostate cancer. Cancer Discov. 7, 736–749 (2017)."),[8](https://www.nature.com/articles/s41588-025-02265-4#ref-CR8 "Zhou, Z. et al. Synergy of p53 and Rb deficiency in a conditional mouse model for metastatic prostate cancer. Cancer Res. 66, 7889–7898 (2006)."),[9](https://www.nature.com/articles/s41588-025-02265-4#ref-CR9 "Park, J. W. et al. Reprogramming normal human epithelial tissues to a common, lethal neuroendocrine cancer lineage. Science 362, 91–95 (2018)."), it largely shares genomic changes with CRPC tumors[3](https://www.nature.com/articles/s41588-025-02265-4#ref-CR3 "Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. Nat. Med. 22, 298–305 (2016)."). In contrast, substantial epigenetic differences in DNA methylation[3](https://www.nature.com/articles/s41588-025-02265-4#ref-CR3 "Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. Nat. Med. 22, 298–305 (2016)."), chromatin accessibility[10](https://www.nature.com/articles/s41588-025-02265-4#ref-CR10 "Cejas, P. et al. Subtype heterogeneity and epigenetic convergence in neuroendocrine prostate cancer. Nat. Commun. 12, 5775 (2021).") and histone modifications[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).") have been noted in NEPC and might have causative roles in promoting NE transformation (NET) of PCa. A plethora of transcription factors (TFs) and chromatin modifiers, including SOX2, MYCN, EZH2 and FOXA1, are deregulated in NEPC to suppress AR signaling and/or mediate epigenetic remodeling, yielding lineage plasticity[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021)."),[12](https://www.nature.com/articles/s41588-025-02265-4#ref-CR12 "Kim, J. et al. FOXA1 inhibits prostate cancer neuroendocrine differentiation. Oncogene 36, 4072–4080 (2017)."),[13](https://www.nature.com/articles/s41588-025-02265-4#ref-CR13 "Dardenne, E. et al. N-Myc induces an EZH2-mediated transcriptional program driving neuroendocrine prostate cancer. Cancer Cell 30, 563–577 (2016)."),[14](https://www.nature.com/articles/s41588-025-02265-4#ref-CR14 "Lee, J. K. et al. N-Myc drives neuroendocrine prostate cancer initiated from human prostate epithelial cells. Cancer Cell 29, 536–547 (2016)."),[15](https://www.nature.com/articles/s41588-025-02265-4#ref-CR15 "Bishop, J. L. et al. The master neural transcription factor BRN2 is an androgen receptor-suppressed driver of neuroendocrine differentiation in prostate cancer. Cancer Discov. 7, 54–71 (2017)."),[16](https://www.nature.com/articles/s41588-025-02265-4#ref-CR16 "Guo, H. et al. ONECUT2 is a driver of neuroendocrine prostate cancer. Nat. Commun. 10, 278 (2019)."),[17](https://www.nature.com/articles/s41588-025-02265-4#ref-CR17 "Cyrta, J. et al. Role of specialized composition of SWI/SNF complexes in prostate cancer lineage plasticity. Nat. Commun. 11, 5549 (2020)."),[18](https://www.nature.com/articles/s41588-025-02265-4#ref-CR18 "Davies, A. et al. An androgen receptor switch underlies lineage infidelity in treatment-resistant prostate cancer. Nat. Cell Biol. 23, 1023–1034 (2021)."). Of high relevance, FOXA2, a nuclear biomarker of NEPC, is a pioneer factor that promotes NET of PCa, in part by driving KIT pathway activations[19](https://www.nature.com/articles/s41588-025-02265-4#ref-CR19 "Qi, J. et al. Siah2-dependent concerted activity of HIF and FoxA2 regulates formation of neuroendocrine phenotype and neuroendocrine prostate tumors. Cancer Cell 18, 23–38 (2010)."),[20](https://www.nature.com/articles/s41588-025-02265-4#ref-CR20 "Han, M. et al. FOXA2 drives lineage plasticity and KIT pathway activation in neuroendocrine prostate cancer. Cancer Cell 40, 1306–1323 (2022).").

Recent advances in the study of three-dimensional (3D) chromatin folding have suggested that the human genome is hierarchically organized into multiple layers, including A/B compartments[21](https://www.nature.com/articles/s41588-025-02265-4#ref-CR21 "Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science 326, 289–293 (2009)."), topologically associating domains (TADs)[22](https://www.nature.com/articles/s41588-025-02265-4#ref-CR22 "Dixon, J. R. et al. Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature 485, 376–380 (2012)."), and enhancer–promoter (E–P) chromatin loops[23](https://www.nature.com/articles/s41588-025-02265-4#ref-CR23 "Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014)."). Such 3D chromatin structure, shaped by lineage-specific TFs, is lineage-specific and critical for cell fate determination[24](https://www.nature.com/articles/s41588-025-02265-4#ref-CR24 "Wang, R. et al. MyoD is a 3D genome structure organizer for muscle cell identity. Nat. Commun. 13, 205 (2022)."),[25](https://www.nature.com/articles/s41588-025-02265-4#ref-CR25 "Di Giammartino, D. C. et al. KLF4 is involved in the organization and regulation of pluripotency-associated three-dimensional enhancer networks. Nat. Cell Biol. 21, 1179–1190 (2019)."),[26](https://www.nature.com/articles/s41588-025-02265-4#ref-CR26 "Stadhouders, R. et al. Transcription factors orchestrate dynamic interplay between genome topology and gene regulation during cell reprogramming. Nat. Genet. 50, 238–249 (2018)."),[27](https://www.nature.com/articles/s41588-025-02265-4#ref-CR27 "Stadhouders, R., Filion, G. J. & Graf, T. Transcription factors and 3D genome conformation in cell-fate decisions. Nature 569, 345–354 (2019)."). A recent Hi-C sequencing of 80 clinical CRPC highlighted major interactions among 3D chromatin structure, epigenetic landscape and gene expression[28](https://www.nature.com/articles/s41588-025-02265-4#ref-CR28 "Zhao, S. G. et al. Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer. Nat. Genet. 56, 1689–1700 (2024)."). However, the 3D chromatin architecture in NEPC tumors and its difference from CRPC have not been explored. Furthermore, it remains elusive how lineage-specific pioneer factors cooperate with neuronal TFs to regulate epigenetic remodeling and chromatin reorganization to facilitate the luminal-to-NE cell type switch that underlies NET.

NKX2-1 is a neuronal TF that is detected early in the endodermal thyroid and lung and in restricted neuroblast cells of the brain[29](https://www.nature.com/articles/s41588-025-02265-4#ref-CR29 "Lazzaro, D., Price, M., de Felice, M. & Di Lauro, R. The transcription factor TTF-1 is expressed at the onset of thyroid and lung morphogenesis and in restricted regions of the foetal brain. Development 113, 1093–1104 (1991)."),[30](https://www.nature.com/articles/s41588-025-02265-4#ref-CR30 "Minoo, P. et al. TTF-1 regulates lung epithelial morphogenesis. Dev. Biol. 172, 694–698 (1995)."),[31](https://www.nature.com/articles/s41588-025-02265-4#ref-CR31 "Minoo, P., Su, G., Drum, H., Bringas, P. & Kimura, S. Defects in tracheoesophageal and lung morphogenesis in Nkx2.1(−/−) mouse embryos. Dev. Biol. 209, 60–71 (1999)."). It regulates the identity of neuronal progenitor cells, mediates interneuron specification and directs postmitotic neuron migration[32](https://www.nature.com/articles/s41588-025-02265-4#ref-CR32 "Butt, S. J. et al. The requirement of Nkx2-1 in the temporal specification of cortical interneuron subtypes. Neuron 59, 722–732 (2008)."),[33](https://www.nature.com/articles/s41588-025-02265-4#ref-CR33 "Nobrega-Pereira, S. et al. Postmitotic Nkx2-1 controls the migration of telencephalic interneurons by direct repression of guidance receptors. Neuron 59, 733–745 (2008)."). In the lung, it has been shown to recruit FOXA proteins to lung-specific loci to promote cell growth and cellular identity while repressing gastric differentiation[34](https://www.nature.com/articles/s41588-025-02265-4#ref-CR34 "Snyder, E. L. et al. Nkx2-1 represses a latent gastric differentiation program in lung adenocarcinoma. Mol. Cell 50, 185–199 (2013)."),[35](https://www.nature.com/articles/s41588-025-02265-4#ref-CR35 "Camolotto, S. A. et al. FoxA1and FoxA2 drive gastric differentiation and suppress squamous identity in NKX2-1-negative lung cancer. eLife 7, e38579 (2018)."),[36](https://www.nature.com/articles/s41588-025-02265-4#ref-CR36 "Orstad, G. et al. FoxA1 and FoxA2 control growth and cellular identity in NKX2-1-positive lung adenocarcinoma. Dev. Cell 57, 1866–1882 (2022)."). NKX2-1 is undetectable in the prostate epithelium but is highly expressed in more than 50% of NE lesions[37](https://www.nature.com/articles/s41588-025-02265-4#ref-CR37 "Wang, W. & Epstein, J. I. Small cell carcinoma of the prostate. A morphologic and immunohistochemical study of 95 cases. Am. J. Surg. Pathol. 32, 65–71 (2008)."). Ectopic overexpression (OE) of NKX2-1 has been shown to facilitate ASCL1 in redistributing FOXA1 from luminal- to NE-specific regulatory elements, resulting in NET[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021)."). This could be an adaptive event as FOXA1 is usually downregulated, rather than upregulated, in NEPC[12](https://www.nature.com/articles/s41588-025-02265-4#ref-CR12 "Kim, J. et al. FOXA1 inhibits prostate cancer neuroendocrine differentiation. Oncogene 36, 4072–4080 (2017)."). How NKX2-1, as a neuronal TF, actively drives NET and the mechanisms of its induction in NEPC tumors are yet to be characterized.

Here we showed different 3D chromatin architecture in NEPC versus CRPC tumors that were recapitulated in isogenic LNCaP cells undergoing NET following FOXA2 OE. Mechanistically, NKX2-1, predominantly binding the promoters, is induced by FOXA2 and interacts with enhancer-bound FOXA2, which further stabilizes FOXA2 binding at NE enhancers and propels regional epigenetic remodeling. Subsequently, NKX2-1 and FOXA2 induce NE enhancer activation and NE transcriptional program through recruiting p300/CBP. Accordingly, p300/CBP inhibition abolishes NEPC growth in vitro and in vivo. Taken together, our study deciphers a mechanism fundamental to cellular identity switch during NET and identifies potential therapeutic approaches to suppress NEPC.

Results
-------

### Distinct 3D chromatin architecture in NEPC versus CRPC tumors

To compare the 3D chromatin architecture of PCa cells, we performed Hi-C analyses of a panel of patient-derived xenograft (PDX) tumors with over 600 million valid paired-end reads per sample (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM4)). Unsupervised hierarchical clustering of the pairwise correlation coefficients of Hi-C samples based on their first principal component largely separated CRPC and NEPC tumors, except LuCaP147, which has a transcriptome in between[38](https://www.nature.com/articles/s41588-025-02265-4#ref-CR38 "Bolis, M. et al. Dynamic prostate cancer transcriptome analysis delineates the trajectory to disease progression. Nat. Commun. 12, 7033 (2021).") (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02265-4#Fig9)). Next, we identified chromatin loops at 10 kb resolution[39](https://www.nature.com/articles/s41588-025-02265-4#ref-CR39 "Roayaei Ardakany, A., Gezer, H. T., Lonardi, S. & Ay, F. Mustache: multi-scale detection of chromatin loops from Hi-C and Micro-C maps using scale-space representation. Genome Biol. 21, 256 (2020).") and defined those preferentially (_P_ < 0.001) detected in at least two NEPC samples in pairwise comparisons to CRPC as NEPC-enriched loops (_n_ = 3,702) and vice versa for CRPC-enriched loops (_n_ = 1,808). Concordantly, NEPC-enriched loops showed significantly (_P_ = 0.026) higher aggregate peak analysis (APA) scores in NEPC tumors than CRPC (Fig. [1a](https://www.nature.com/articles/s41588-025-02265-4#Fig1)), and vice versa for CRPC-enriched loops (_P_ = 0.022; Fig. [1b](https://www.nature.com/articles/s41588-025-02265-4#Fig1)).

**Fig. 1: Distinct 3D chromatin architecture in NEPC compared to CRPC tumors.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/1)

**a**,**b**, APA plots of NEPC-enriched (**a**) and CRPC-enriched (**b**) loops in NEPC (LuCaP93, 145.1, 145.2, and NCI-H660) and CRPC samples (LuCaP35CR, 70CR, 77CR, 147). The APA score is annotated within each plot. Scale bars indicate log2 of observed over expected enrichment. _P_ = 0.026 and _P_ = 0.022, respectively, for NEPC- and CRPC-enriched loop APA scores between NEPC and CRPC samples. _P_ values by one-sided Student’s _t_ test. **c**, Volcano plot showing genes linked to NEPC-enriched loops that are differentially expressed in NEPC versus CRPC using _P_ < 0.05 (two-sided _t_ test) and log2(FC) ≥ 1. **d**, Volcano plot of genes linked to CRPC-enriched loops and are differentially expressed in CRPC versus NEPC using _P_ < 0.05 (two-sided _t_ test) and log2(FC) ≥ 1. **e**,**f**, GO analyses of genes linked to NEPC-enriched (**e**) and CRPC-enriched (**f**) loops and are upregulated in NEPC and CRPC, respectively, as defined in **c** and **d**. **g**,**h**, Hi-C contact maps (5 kb resolution) of the _HOXB_ gene cluster (**g**) and _AR_ gene region (**h**) in CRPC (top row) and NEPC (bottom row) models. Gene boxes and 1D ChIP–seq tracks of matched models are aligned at the bottom. Blue arrows in **g** indicate NEPC-enriched loops at the _HOXB1–HOXB9_ loci, and black arrows indicate CRPC-enriched loops at the _HOXB13_ locus. The scale bars indicate ICE-normalized contact frequency. **i**,**j**. The bar graphs showing the percentage of NEPC-enriched (**i**) or CRPC-enriched (**j**) loop anchors that overlap with previously reported[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).") NE-enriched CREs (_n_ = 14,985) and Ad-CREs (_n_ = 4,338). The enriched motifs at overlapped anchors are shown on the right. _P_ values by one-sided permutation test (without multiple comparisons) to evaluate the significant differences of the indicated overlap relative to control regions of equal size.

To assess the potential targets of these NEPC/CRPC-enriched loops, we identified genes within ±3 kb of the loop anchors and examined their expression in public NEPC versus CRPC RNA-seq data[40](https://www.nature.com/articles/s41588-025-02265-4#ref-CR40 "Labrecque, M. P. et al. Molecular profiling stratifies diverse phenotypes of treatment-refractory metastatic castration-resistant prostate cancer. J. Clin. Invest. 129, 4492–4505 (2019)."),[41](https://www.nature.com/articles/s41588-025-02265-4#ref-CR41 "Nouruzi, S. et al. ASCL1 activates neuronal stem cell-like lineage programming through remodeling of the chromatin landscape in prostate cancer. Nat. Commun. 13, 2282 (2022).") (Fig. [1c,d](https://www.nature.com/articles/s41588-025-02265-4#Fig1)). Gene Ontology (GO) analyses revealed that NEPC-loop-associated genes that were also upregulated in NEPC were enriched for neuron development processes, whereas CRPC-loop genes that were upregulated in CRPC were enriched for epithelial and prostatic functions (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02265-4#Fig1)). For instance, a unique TAD at the _HOXB1–HOXB9_ cluster, encoding genes crucial for embryonic development, was present in NEPC but not CRPC tumors (Fig. [1g](https://www.nature.com/articles/s41588-025-02265-4#Fig1)). Accordingly, this region was strongly marked by H3K27me3 in CRPC, which was diminished in NEPC with a striking gain of active histone marker H3K27ac, accompanied by their concordant upregulation in NEPC (Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02265-4#Fig9)). By contrast, the adjacent luminal TF _HOXB13_ belonged to a different TAD that showed strong chromatin interactions in CRPC tumors and was diminished in NEPC, with a concordant loss of H3K27ac and gain of H3K27me3 (Fig. [1g](https://www.nature.com/articles/s41588-025-02265-4#Fig1)), in agreement with its downregulation in NEPC[42](https://www.nature.com/articles/s41588-025-02265-4#ref-CR42 "Lu, X. et al. HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer. Nat. Genet. 54, 670–683 (2022)."). A similar loss in chromatin interactions between a recently reported super-enhancer (SE)[43](https://www.nature.com/articles/s41588-025-02265-4#ref-CR43 "Takeda, D. Y. et al. A somatically acquired enhancer of the androgen receptor is a noncoding driver in advanced prostate cancer. Cell 174, 422–432 (2018).") and the _AR_ promoter was also observed (Fig. [1h](https://www.nature.com/articles/s41588-025-02265-4#Fig1) and Extended Data Fig. [1c](https://www.nature.com/articles/s41588-025-02265-4#Fig9)), whereas chromatin interactions at NE regulators, such as _ASCL1_ and _INSM1_, were enhanced in NEPC versus CRPC (Extended Data Fig. [1d–g](https://www.nature.com/articles/s41588-025-02265-4#Fig9)).

As lineage-specific TFs bind at enhancers and might orchestrate CRPC- and NEPC-specific enhancer looping to target promoters, we focused on previously defined NE-enriched candidate regulatory elements (NE-CREs) and adenocarcinoma-enriched CREs (Ad-CREs)[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021)."). Not surprisingly, significantly more NEPC- and CRPC-enriched loops overlapped with NE-CREs and Ad-CREs, respectively, than with control chromatin regions (_P_ < 0.001 by permutation test; Fig. [1i,j](https://www.nature.com/articles/s41588-025-02265-4#Fig1)). Interestingly, the DNA-binding motif of FOXA2, a biomarker[44](https://www.nature.com/articles/s41588-025-02265-4#ref-CR44 "Mirosevich, J. et al. Expression and role of Foxa proteins in prostate cancer. Prostate 66, 1013–1028 (2006)."),[45](https://www.nature.com/articles/s41588-025-02265-4#ref-CR45 "Park, J. W., Lee, J. K., Witte, O. N. & Huang, J. FOXA2 is a sensitive and specific marker for small cell neuroendocrine carcinoma of the prostate. Mod. Pathol. 30, 1262–1272 (2017).") and a driver of NEPC[19](https://www.nature.com/articles/s41588-025-02265-4#ref-CR19 "Qi, J. et al. Siah2-dependent concerted activity of HIF and FoxA2 regulates formation of neuroendocrine phenotype and neuroendocrine prostate tumors. Cancer Cell 18, 23–38 (2010)."),[20](https://www.nature.com/articles/s41588-025-02265-4#ref-CR20 "Han, M. et al. FOXA2 drives lineage plasticity and KIT pathway activation in neuroendocrine prostate cancer. Cancer Cell 40, 1306–1323 (2022)."), was significantly enriched in NEPC-enriched loops that anchored at NE-CREs. In contrast, the motifs of luminal TFs were enriched in CRPC-enriched loops overlapping with Ad-CREs (Fig. [1i,j](https://www.nature.com/articles/s41588-025-02265-4#Fig1)). Altogether, our data demonstrate striking differences in 3D chromatin architecture between NEPC and CRPC tumors and nominate FOXA2 as a major regulator of NE-specific E–P looping.

### FOXA2 drives NET of PCa, showing clinically relevant phenotypes

We infected PCa cell lines with FOXA2 OE lentivirus and observed a gradual transformation of AR-positive LNCaP and 22Rv1 cells to NEPC phenotype over the course of 28 days and of the AR-negative Du145 cells in only 7 days (Extended Data Fig. [2a,b](https://www.nature.com/articles/s41588-025-02265-4#Fig10) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). On the flip side, FOXA2 was upregulated in the previously reported NEPC model of LNCaP with _TP53_ and _RB1_ knockdown (KD)[6](https://www.nature.com/articles/s41588-025-02265-4#ref-CR6 "Mu, P. et al. SOX2 promotes lineage plasticity and antiandrogen resistance in TP53- and RB1-deficient prostate cancer. Science 355, 84–88 (2017)."), and FOXA2 knockout (KO) abolished NE marker gene expression (Extended Data Fig. [2c](https://www.nature.com/articles/s41588-025-02265-4#Fig10)). Furthermore, LNCaP with FOXA2 OE grew xenograft tumors much more rapidly than the control cells, showing mixed adenocarcinoma to small-cell carcinoma histology and strong synaptophysin (SYP) staining that closely recapitulated treatment-induced NEPC in patients (Extended Data Fig. [2d–f](https://www.nature.com/articles/s41588-025-02265-4#Fig10) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)).

RNA-sequencing (RNA-seq) analyses of time-course LNCaP+FOXA2 cells revealed many differentially expressed genes (_n_ = 12,835), indicating major transcriptional reprogramming. _K_\-means clustering revealed six clusters, including three clusters each of upregulated (C1–C3, _n_ = 6,825) and downregulated genes (C4–C6, _n_ = 6,010) that were respectively involved in neuron/embryonic development and luminal cell functions (Fig. [2a](https://www.nature.com/articles/s41588-025-02265-4#Fig2) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). Western blot (WB) confirmed gradually reduced expression of luminal TFs over time and increased expression of NE markers, such as SYP, NCAM1 and BRN2, manifesting at D21 and peaking at D28 (Fig. [2b](https://www.nature.com/articles/s41588-025-02265-4#Fig2)). Critically, D0, D2 and D7 LNCaP+FOXA2 cells showed transcriptomes similar to AR+/NE− clinical PCa samples[40](https://www.nature.com/articles/s41588-025-02265-4#ref-CR40 "Labrecque, M. P. et al. Molecular profiling stratifies diverse phenotypes of treatment-refractory metastatic castration-resistant prostate cancer. J. Clin. Invest. 129, 4492–4505 (2019)."), while D21 and D28 cells more closely clustered with AR−/NE+ (NEPC) tumors (Fig. [2c](https://www.nature.com/articles/s41588-025-02265-4#Fig2)). Notably, D14 cells fell somewhere in between, indicating an intermediate transcriptome. These data suggest that LNCaP+FOXA2 cells faithfully modeled the clinical transition of PCa from adenocarcinoma to NEPC and identified D14 as the tipping point of the transition.

**Fig. 2: 3D chromatin reorganization in isogenic PCa cells undergoing NET.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/2)

**a**, Heatmap shows six clusters of genes differentially expressed over the time course (adjusted _P_ < 0.0001). Representative genes and significantly enriched molecular concepts are shown on the right. Bulk RNA-seq was performed in LNCaP cells with FOXA2 OE at the indicated time points. Color bar—_z_ score. Adjusted _P_ values were calculated using the likelihood ratio test, followed by Benjamini–Hochberg correction. **b**, WB analyses of luminal and NE lineage markers in time-course LNCaP+FOXA2 cells. LNCaP cells were infected with control (plv) or FOXA2 virus and followed up for 28 days. Data shown are from one of three (_n_ = 3) independent experiments. **c**, Integrative analyses of ARS and NES scores in time-course LNCaP+FOXA2 samples and those of clinical PCa samples (GSE126078). ARS and NES scores were calculated by gene set variation analysis. **d**, APA plots of luminal (top row) and NE (bottom row) loops (D28 versus D0) in time-course LNCaP+FOXA2 samples and NCI-H660. The APA score is annotated within each plot. Scale bars indicate log2 of observed over expected enrichment. **e**,**f**. Hi-C contact maps (5 kb resolution) of the _AR_ gene region (**e**) and the _HOXB_ gene cluster (**f**) in time-course LNCaP+FOXA2 cells. Gene boxes and 1D ChIP–seq tracks of matched models are aligned at the bottom. Black arrows indicate luminal loops at _AR_ and _HOXB13_ loci, and blue arrows indicate NE loops at _HOXB1–HOXB9_ loci. Enhancers are highlighted in light blue, and promoters in light orange color. FOXA2, H3K27ac, H3K4me1, H3K27me3 and CTCF tracks under the D0 contact map used their corresponding ChIP–seq performed in closely related D2 cells. All others were done in conditions exactly matched to Hi-C. The scale bars indicate ICE-normalized contact frequency.

To determine whether such major transcriptional reprogramming may be propelled by 3D chromatin reorganization, we performed Hi-C of time-course LNCaP+FOXA2 cells. Not surprisingly, D0 and D14 cells harbored 3D chromatin architecture similar to CRPC tumors, whereas D28 cells were closely grouped with NEPC tumors (Extended Data Fig. [2g](https://www.nature.com/articles/s41588-025-02265-4#Fig10) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). Interestingly, despite the NE-like transcriptome, D21 cells remained in the CRPC cluster, suggesting that stable chromatin reorganization lagged transcriptional reprogramming during NET. Similar to findings in clinically relevant PDX tumors (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02265-4#Fig1)), genes associated with NE (enriched in D28) and luminal (enriched in D0) loops were, respectively, implicated in axonogenesis/developmental processes and epithelial functions (Fig. [2d](https://www.nature.com/articles/s41588-025-02265-4#Fig2), Supplementary Note [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM1) and Supplementary Fig. [1a](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). A TAD between AR enhancer and promoter was detected at D0, which was gradually eliminated over time, accompanied by a decrease of H3K4me1 and H3K27ac and gain of H3K27me3 (Fig. [2e](https://www.nature.com/articles/s41588-025-02265-4#Fig2)). Likewise, there were contrasting changes in epigenetic modifications and TADs at the _HOXB1–HOXB9_ cluster and _HOXB13_ gene over time (Fig. [2f](https://www.nature.com/articles/s41588-025-02265-4#Fig2)), recapitulating our earlier observations in PDX tumors. Therefore, our LNCaP+FOXA2 system represents an isogenic cell model of NET with high clinical relevance.

### Clonal transformation and expansion during NET of PCa

To understand the epigenetic bases of transcriptional reprogramming and 3D chromatin reorganization during NET, we performed the assay for transposase-accessible chromatin using sequencing (ATAC–seq) in time-course LNCaP+FOXA2 cells. PCA analyses of ATAC–seq peaks showed that D0/D2/D7 cells and D21/D28 cells were, respectively, clustered closer to CRPC and NEPC PDX tumors[10](https://www.nature.com/articles/s41588-025-02265-4#ref-CR10 "Cejas, P. et al. Subtype heterogeneity and epigenetic convergence in neuroendocrine prostate cancer. Nat. Commun. 12, 5775 (2021)."), whereas the D14 cells were positioned somewhere in between, indicating a transitioning stage (Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02265-4#Fig11)). _K_\-means clustering revealed two clusters, each of gradually increasing (C3–C4) and gradually decreasing (C5–C6) gene-distal ATAC–seq peaks, hereafter defined as NE and luminal enhancers, respectively (Fig. [3a,b](https://www.nature.com/articles/s41588-025-02265-4#Fig3)). Accordingly, NE and luminal enhancers, respectively, are enriched for motifs of neural/stem cell TFs and luminal TFs and associated with genes involved in developmental processes and luminal cell functions (Extended Data Fig. [3b,c](https://www.nature.com/articles/s41588-025-02265-4#Fig11) and Supplementary Note [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). There were also some constant peaks, such as C1, that predominantly bound at promoter regions and enriched for motifs of ubiquitous TFs, such as SP1, that regulate housekeeping gene expression. Altogether, these results show a remarkable switch of chromatin accessibility from luminal-to-NE enhancers from D0 to D28 of NET, in agreement with transcriptional reprogramming and 3D chromatin rewiring.

**Fig. 3: Single-cell multiome identified transitioning individual cells with intermediate transcriptome and chromatin states.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/3)

**a**, _K_\-means clustering reveals six clusters of differential ATAC–seq peaks (±2 kb) across time-course LNCaP+FOXA2 samples (adjusted _P_ < 0.0001). Heatmaps shown here were scaled across samples. Significantly enriched TF motifs and GO terms for each cluster, along with the number of peaks, are shown on the right. Color bar at the bottom indicates the scale of enrichment intensity. Adjusted _P_ values were calculated using the likelihood ratio test followed by Benjamini–Hochberg correction. **b**, Genomic distribution of the six clusters of ATAC–seq peaks in **a**. **c**–**f**, scRNA-seq UMAP visualization of D0, D14 and D21 LNCaP+FOXA2 cells (**c**), AR (**d**) and NE (**e**) signature genes, which are further quantified for the proportion of AR+/ARS+, AR+/ARS−, AR−/NES− and AR−/NES+ cells (**f**). **g**, RNA velocity analysis of D14 LNCaP+FOXA2 cells. The data are visualized as streamlines in a UMAP-based embedding. Blue dots indicate _KLK3_\-high cells, and the yellow dots indicate _KLK3_\-low cells. **h**–**j**, scATAC–seq UMAP visualization of D0, D14 and D21 LNCaP+FOXA2 cells (**h**), chromatin accessibility (**i**) and expression (**j**) of ARS/NES genes. Red and green arrows indicate transitioning D14 and D21 cells, respectively. enhs, enhancers.

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM5)

Next, we wondered whether we could capture individual transitioning cells with intermediate transcriptome and chromatin states. Using single-cell multiome (single-cell RNA-seq (scRNA-seq) and scATAC–seq), we analyzed 2,110 D0, 1,739 D14 and 2,344 D21 LNCaP+FOXA2 cells, whose transcriptome, respectively, aligned with clinical primary PCa, CRPC and NEPC, as expected[46](https://www.nature.com/articles/s41588-025-02265-4#ref-CR46 "Cheng, Q. et al. Pre-existing castration-resistant prostate cancer-like cells in primary prostate cancer promote resistance to hormonal therapy. Eur. Urol. 81, 446–455 (2022).") (Extended Data Fig. [3d,e](https://www.nature.com/articles/s41588-025-02265-4#Fig11)). Uniform manifold approximation and projection (UMAP) analyses revealed two distinct clusters that comprised primarily D0 and D21 cells that expressed high levels of luminal and NE genes, respectively (Fig. [3c–e](https://www.nature.com/articles/s41588-025-02265-4#Fig3) and Extended Data Fig. [3f](https://www.nature.com/articles/s41588-025-02265-4#Fig11)). Of note, most of the D14 and some of the D21 cells still belonged to the D0 cluster. These cells remained _AR_\-positive but had a reduced AR signature (ARS) score and started to express some NE signature (NES) genes (Supplementary Note [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)), indicating an intermediate transcriptome. Furthermore, RNA velocity analyses of D14 cells revealed a strong tendency for the _AR_+/ARS+ (_KLK3_+) cells to become _AR_+/ARS− (_KLK3_\-low; Fig. [3g](https://www.nature.com/articles/s41588-025-02265-4#Fig3)), supporting the concept of clonal transformation. UMAP analyses of matched scATAC–seq data likewise confirmed the following two clusters of cells: an NE cluster of D21 cells and a luminal cluster with mixed D0, D14 and some D21 cells (Fig. [3h](https://www.nature.com/articles/s41588-025-02265-4#Fig3)) and validated that most D14 and some D21 cells had an intermediate chromatin state that was consistent with their transitioning transcriptome (Fig. [3i,j](https://www.nature.com/articles/s41588-025-02265-4#Fig3) and Supplementary Note [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). Moreover, clonal variance analyses using scRNA-seq data verified that LNCaP, like many cancer cell lines[47](https://www.nature.com/articles/s41588-025-02265-4#ref-CR47 "Qin, J. et al. The PSA−/lo prostate cancer cell population harbors self-renewing long-term tumor-propagating cells that resist castration. Cell Stem Cell 10, 556–569 (2012)."),[48](https://www.nature.com/articles/s41588-025-02265-4#ref-CR48 "Horning, A. M. et al. Single-cell RNA-seq reveals a subpopulation of prostate cancer cells with enhanced cell-cycle-related transcription and attenuated androgen response. Cancer Res. 78, 853–864 (2018)."),[49](https://www.nature.com/articles/s41588-025-02265-4#ref-CR49 "Minussi, D. C. et al. Breast tumours maintain a reservoir of subclonal diversity during expansion. Nature 592, 302–308 (2021)."), contained multiple clones and revealed two major clones, clone 1 and clone 2, which, respectively, dominated D0 and D28 cells. Critically, both clones went through NET from D0 to D28, with clone 2 cells becoming fully transformed and predominant after D21 (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02265-4#Fig12) and Supplementary Note [3](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)), supporting a mixed model of clonal transformation and expansion underlying NET of PCa.

### NKX2-1 induction is required for FOXA2-driven NET

FOXA2 is a pioneering factor that requires the coordination of lineage-specific TF for enhancer activation[50](https://www.nature.com/articles/s41588-025-02265-4#ref-CR50 "Wang, A. et al. Epigenetic priming of enhancers predicts developmental competence of hESC-derived endodermal lineage intermediates. Cell Stem Cell 16, 386–399 (2015)."). Focusing on neural TFs that were turned on during NET, we found multiple FOXA2-binding events at regulatory elements of _NKX2-1_, as early as D2 following FOXA2 OE, followed by an increase of H3K4me1 at D14 and of H3K27ac at D21, resulting in _NKX2-1_ upregulation (Fig. [4a](https://www.nature.com/articles/s41588-025-02265-4#Fig4) and Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02265-4#Fig13)). In concordance, _FOXA2_ KD directly reduced NKX2-1 expression in various NEPC models, including LNCaP cells with stable FOXA2 OE (>28 days), termed LuNE cells hereafter (Fig. [4b](https://www.nature.com/articles/s41588-025-02265-4#Fig4) and Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02265-4#Fig13)). Moreover, Hi-C revealed increased interactions between FOXA2-bound NKX2-1 enhancers and the _NKX2-1_ promoter over LNCaP+FOXA2 time course, which were verified in NCI-H660 and NEPC PDXs (Extended Data Fig. [5c,d](https://www.nature.com/articles/s41588-025-02265-4#Fig13)), suggesting that FOXA2 binds to _NKX2-1_ enhancer to directly induce _NKX2-1_ transcription via E–P looping.

**Fig. 4: NKX2-1 is induced by FOXA2 and required for FOXA2-driven NET.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/4)

**a**, Genome browser view of _NKX2-1_ mRNA (top row), and FOXA2, H3K4me1, H3K27ac ChIP–seq signal around the _NKX2-1_ gene in time-course LNCaP+FOXA2 cells. The promoter is highlighted in light blue, and enhancers are highlighted in yellow. **b**, WB of NEPC cell lines (LuNE, NCI-H660) and organoids (LuCaP145.2) with control or _FOXA2_ KD. Data shown are from one of two (_n_ = 2) independent experiments. LuNE cells—stable LNCaP+FOXA2 cells. **c**, FOXA2 Co-IP showing its interaction with NKX2-1 protein in NEPC cell line NCI-H660. Data shown are from one of three (_n_ = 3) independent experiments. **d**, Heatmap showing AR, NKX2-1 and FOXA2 occupancy at previously reported[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).") NE-CREs (_n_ = 14,985) and Ad-CREs (_n_ = 4,338) in LNCaP or NEPC models (NCI-H660, LuCaP93, LuCap145.1, LuCaP145.2). Scale bar—enrichment intensity. **e**, Heatmap showing active (H3K27ac) enhancer (H3K4me1) and promoter (H3K4me3) marks at NKX2-1-only, FOXA2-only and shared binding sites in LuCaP145.1. Scale bar—enrichment intensity. **f**, APA plots of FOXA2- and NKX2-1-anchored E–P loops identified in D28 in time-course LNCaP+FOXA2 Hi-C samples. The APA score is annotated within each plot. Scale bars indicate log2 of observed over expected enrichment. **g**, _NKX2-1_ KD abolished FOXA2-induced NET. LNCaP cells were co-infected with FOXA2 virus along with either sg_NC_ or sg_NKX2-1_ virus. The infected cells were collected at the indicated time points for WB analyses of luminal and NE markers. The red star indicates a nonspecific band. Data shown are from one of three (_n_ = 3) independent experiments. **h**, Heatmap showing FOXA2, NKX2-1, H3K27ac and H3K4me1 ChIP–seq around (±2 kb) the six ATAC–seq peak clusters in LNCaP+FOXA2 cells, with or without _NKX2-1_ KD, at indicated time points. Peaks within each cluster were separately sorted by FOXA2 ChIP–seq intensity. Scale bar—enrichment intensity.

Next, we attempted to determine whether NKX2-1 induction is required for FOXA2-driven NET. We observed strong interactions between FOXA2 and NKX2-1 proteins and substantial co-occupancy at NE enhancers[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).") (Fig. [4c,d](https://www.nature.com/articles/s41588-025-02265-4#Fig4)). A majority of FOXA2-binding sites were co-occupied by NKX2-1 and marked by H3K27ac and H3K4me1, suggesting active NE enhancers (Fig. [4e](https://www.nature.com/articles/s41588-025-02265-4#Fig4) and Extended Data Fig. [5e,f](https://www.nature.com/articles/s41588-025-02265-4#Fig13)), whereas NKX2-1-only binding sites were enriched for promoter and marked by H3K4me3 (Fig. [4e](https://www.nature.com/articles/s41588-025-02265-4#Fig4) and Extended Data Fig. [5e–g](https://www.nature.com/articles/s41588-025-02265-4#Fig13)). Hi-C revealed a substantial increase in the intensity and number of chromatin loops anchored at FOXA2 and NKX2-1 binding sites, indicating that NKX2-1, which primarily binds at promoters, interacts with enhancer-bound FOXA2 through E–P looping to propel NE enhancer priming and activation (Fig. [4f](https://www.nature.com/articles/s41588-025-02265-4#Fig4) and Extended Data Fig. [5h](https://www.nature.com/articles/s41588-025-02265-4#Fig13)). Indeed, KD of NKX2-1 abolished FOXA2-driven switches of luminal and NE marker gene expression (Fig. [4g](https://www.nature.com/articles/s41588-025-02265-4#Fig4)). Mechanistically, FOXA2 primarily bound to luminal enhancers that were marked by H3K4me1 and H3K27ac and notably also at future NE enhancers immediately following OE, but gradually shifted from luminal-to-NE enhancers over time (Fig. [4h](https://www.nature.com/articles/s41588-025-02265-4#Fig4)). By contrast, NKX2-1 formed strong and selective binding at NE enhancers starting at D21, accompanied by an abrupt luminal-to-NE enhancer switch of H3K27ac and H3K4me1. Critically, NKX2-1 KD not only abolished such changes of the enhancer marks but also halted the shift of FOXA2 cistrome from luminal-to-NE enhancers, suggesting a feed-forward loop between FOXA2 and NKX2-1 binding (Fig. [4h](https://www.nature.com/articles/s41588-025-02265-4#Fig4)). Therefore, FOXA2 initiates NE enhancers but requires the induction of NKX2-1 to complete NET.

### NKX2-1 is upregulated in NEPC and accelerates NET of PCa

To examine the clinical relevance of our findings, we analyzed publicly available PCa datasets[3](https://www.nature.com/articles/s41588-025-02265-4#ref-CR3 "Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. Nat. Med. 22, 298–305 (2016)."),[40](https://www.nature.com/articles/s41588-025-02265-4#ref-CR40 "Labrecque, M. P. et al. Molecular profiling stratifies diverse phenotypes of treatment-refractory metastatic castration-resistant prostate cancer. J. Clin. Invest. 129, 4492–4505 (2019)."),[51](https://www.nature.com/articles/s41588-025-02265-4#ref-CR51 "Haider, M. et al. Epithelial mesenchymal-like transition occurs in a subset of cells in castration resistant prostate cancer bone metastases. Clin. Exp. Metastasis 33, 239–248 (2016).") and found _NKX2-1_ and _FOXA2_ to be significantly co-induced in NEPC tumors (Fig. [5a,b](https://www.nature.com/articles/s41588-025-02265-4#Fig5) and Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02265-4#Fig14)). Concordantly, H3K27ac was markedly increased while DNA methylation drastically reduced around the _NKX2-1_ gene in NEPC versus CRPC tumors, whereas genomic alterations of _NKX2-1_ were infrequent (Extended Data Fig. [6b–d](https://www.nature.com/articles/s41588-025-02265-4#Fig14)). Significantly, tissue microarrays (TMA) analyses validated strongly elevated NKX2-1 and FOXA2 proteins in AR−/NE+ tumors relative to other CRPC subtypes (Fig. [5c](https://www.nature.com/articles/s41588-025-02265-4#Fig5)). Moreover, while FOXA2 was also increased in some AR−/NE− tumors, strong NKX2-1 staining was restricted to AR−/NE+ tumors, indicating higher specificity. Of noteworthy, some NKX2-1-high tumors lacked FOXA2 expression, suggesting that NKX2-1 might be induced by and collaborate with other cofactors such as ASCL1 to regulate NEPC[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).") (Extended Data Fig. [7a–f](https://www.nature.com/articles/s41588-025-02265-4#Fig15) and Supplementary Note [4](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)).

**Fig. 5: NKX2-1 is highly expressed in NEPC tumors and accelerates NET of PCa.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/5)

**a**, Volcano plots showing human TFs differentially expressed in CRPC versus NEPC. _X_ axis represents log2 of FC, while the _y_ axis shows −log10 of _q_ values. Each dot represents a TF, red dots—log2(FC) ≥ 1 and adjusted _P_ < 0.05. Adjusted _P_ values by Wald test with Benjamini–Hochberg correction. **b**, _NKX2-1_ and _FOXA2_ gene expression in human PCa samples with distinct expression of AR and NE genes. _P_ values by two-sided Wilcoxon test. **c**, IHC of NKX2-1 and FOXA2 proteins in clinical CRPC TMAs. Representative IHC images (_n_ = 3) are shown in ×10 with the insets shown in ×40 (left). Scale bar = 100 µm. IHC staining intensities in AR+NE−, AR−NE−, AR+NE+ and AR−NE+ samples are quantified on the right. **d**, Concomitant NKX2-1 OE accelerated FOXA2-driven NET. LNCaP cells were infected with FOXA2, NKX2-1 or both. Cells were collected at the indicated time points and analyzed by WB (_n_ = 3). **e**, Heatmap showing FOXA2 ChIP–seq in LNCaP cells with OE of FOXA2 alone, or FOXA2 and NKX2-1 at D14 or D28. Peaks were centered around (±2 kb) the six ATAC–seq clusters and sorted by FOXA2 ChIP–seq intensity. Scale bar—enrichment intensity. **f**, WB (left, _n_ = 3) and RT–PCR (right) showing _NKX2-1_ KD decreases the expression of NE lineage markers in NCI-H660. NCI-H660 cells were transfected with control or _NKX2-1_ siRNAs and collected on day 5 after transfection for WB and RT–PCR analyses. RT–PCR data were normalized to _GAPDH_. Shown are the mean ± s.e.m. of technical replicates from one of three (_n_ = 3) independent experiments. _P_ values by two-sided _t_ test. **g**, WB (left, _n_ = 3) and RT–PCR (right) showing _NKX2-1_ KD abolishes _RB1_/TP53-KD-induced NET. LNCaP cells were co-infected with sg_RB1_ and sg_TP53_ virus along with either sg_NC_ or sg_NKX2-1_ virus. The infected cells were collected at 4 weeks after infection for analyses of NE and stem cell lineage markers. Data of RT–PCR are shown as in **f**. **h**, RT–PCR (left) and WB (right, _n_ = 3) showing NKX2-1 OE promotes NET in ENZ-resistant AR+/PSA− cell line 42D. The infected cells were collected at 4 weeks after infection for analyses of NE lineage markers. Data of RT–PCR are shown as in **f**.

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM6)

We next attempted to investigate directly whether NKX2-1 co-expression facilitates FOXA2 in driving NET. We found that concurrent OE of NKX2-1 and FOXA2 drastically accelerated NET of LNCaP cells compared to FOXA2 OE alone, whereas NKX2-1 OE alone failed to drive NET (Fig. [5d,e](https://www.nature.com/articles/s41588-025-02265-4#Fig5) and Extended Data Fig. [7g](https://www.nature.com/articles/s41588-025-02265-4#Fig15)). To further assess if NKX2-1 is a general regulator of NEPC, we performed _NKX2-1_ KD in NCI-H660 and observed an ablation of NE marker genes at both protein and mRNA levels (Fig. [5f](https://www.nature.com/articles/s41588-025-02265-4#Fig5)). Likewise, NKX2-1 was also upregulated in the NE-like LNCaP cells with _RB1_/_TP53_\-KD, and its depletion abolished NE marker expression (Fig. [5g](https://www.nature.com/articles/s41588-025-02265-4#Fig5)). On the other hand, NKX2-1 was undetectable in 42D, an AR+/PSA− cell line derived in vivo upon resistance to AR antagonist enzalutamide (ENZ)[18](https://www.nature.com/articles/s41588-025-02265-4#ref-CR18 "Davies, A. et al. An androgen receptor switch underlies lineage infidelity in treatment-resistant prostate cancer. Nat. Cell Biol. 23, 1023–1034 (2021)."), and its OE remarkably increased the expression of NE markers despite low FOXA2 in these cells (Fig. [5h](https://www.nature.com/articles/s41588-025-02265-4#Fig5)). Taken together, our data demonstrated that NKX2-1 is selectively upregulated in NEPC and accelerates NET through collaboration with FOXA2 and other cofactors.

### FOXA2 induces regional DNA demethylation around NE enhancers

We next sought to understand how FOXA2, after initial binding to the enhancers, prepares the chromatin for NET. Using the directed methylation with long-read sequencing (DiMeLo-seq) approach[52](https://www.nature.com/articles/s41588-025-02265-4#ref-CR52 "Altemose, N. et al. DiMeLo-seq: a long-read, single-molecule method for mapping protein–DNA interactions genome wide. Nat. Methods 19, 711–723 (2022)."), we observed sharp peaks of CpG methylation (mCpG) right at the _NKX2-1_ promoter in D2 cells, which were slightly decreased at D14 and completely lost at D28, when abundant N6-methyl-deoxyadenosine (mA), reflecting FOXA2 binding, was detected (Fig. [6a](https://www.nature.com/articles/s41588-025-02265-4#Fig6)). Globally, strong D28 FOXA2-binding peaks showed much higher mA levels in D28 cells, as expected, but, critically, much less DNA methylation, indicated by deep and broad DNA methylation valleys (Fig. [6b](https://www.nature.com/articles/s41588-025-02265-4#Fig6)). As controls, these peaks lacked mA but showed strong DNA methylation in D2 cells (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02265-4#Fig16)). Furthermore, at the level of single DNA molecules, there was also a mutually exclusive pattern of mA and mCpG at each time point, with D28 cells containing DNA molecules with the highest levels of mA and the lowest mCpG (Fig. [6c](https://www.nature.com/articles/s41588-025-02265-4#Fig6)), suggesting that FOXA2 binding led to regional DNA demethylation at NE enhancers over the time course.

**Fig. 6: FOXA2 induces regional DNA demethylation at NE enhancers.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/6)

**a**, IGV view of FOXA2 ChIP–seq (top), mCpG (middle) and single DNA molecules of mCpG and mA (bottom) around the _NKX2-1_ gene in D2, D14 and D28 LNCaP+FOXA2 cells. Green box—CpG islands. Yellow box—_NKX2-1_ promoter. **b**, FOXA2 DiMeLo-seq was performed in D28 LNCaP+FOXA2 cells, and mA and mCpG were called with a probability ≥0.5. Aggregate mA and mCpG curves (top) for each quartile were created with a 50-bp rolling window centered (±1 kb) at D28-specific FOXA2 peaks, which were sorted into four quartiles with q4 comprising the strongest peaks. Inset—a zoomed-in version of the mCpG curves. Heatmap (bottom) shows mA and mCpG on single DNA molecules. A and CG base density (scale bar at the bottom) across the 2 kb region of FOXA2 peaks within each quartile are shown in the 1D heatmaps. Color scales are shown as in **a**. **c**, Correlation plots of mA and mCpG counts on the same DNA molecules in D2, D14 and D28 LNCaP+FOXA2 cells. **d**, PCA analyses of methylation profiles of CRPC, NEPC PDX and D2, D14, D28 LNCaP+FOXA2 samples. DMRs identified between D2 and D28 were used for PCA analysis. **e**, IGV view of %mCpG in samples named on the left at _HOXB2_ (top row) and _AR_ promoter (bottom row) region. CpG islands are shown in green. **f**, FOXA2 DiMeLo-seq showing FOXA2 binding (mA/A) at previously reported[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).") Ad-CREs and NE-CREs in NEPC PDX (LuCaP145.1 and LuCaP145.2) and cell line (NCI-H660). **g**, Correlation plots of mA and mCpG counts on the same DNA molecules in NEPC PDX (LuCaP145.1 and LuCaP145.2) and cell line (NCI-H660). q1 comprises the weakest peaks, and q4 comprises the strongest peaks of FOXA2 based on FOXA2 ChIP–seq signals in respective samples. IGV, integrative genomics viewer; q1, quartile 1; q4, quartile 4.

To evaluate the clinical relevance of these findings, we first identified differentially methylated regions (DMRs) between D28 and D2 LNCaP+FOXA2 cells. As expected, genes with hyper-DMR in D28 were involved in epithelial cell function, whereas hypo-DMR genes were significantly enriched in embryo development and neurogenesis, and they clustered D2 and D28 cells with CRPC and NEPC tumors, respectively (Extended Data Fig. [8b,c](https://www.nature.com/articles/s41588-025-02265-4#Fig16)). To correlate DNA methylation with FOXA2 binding, we performed DiMeLo-seq in a number of PDX tumors and observed that DNA methylation was significantly lower at Ad-CREs and NE-CREs, respectively, in CRPC and NEPC PDXs, being consistent with their respective luminal and NE enhancer activation (Extended Data Fig. [8d,e](https://www.nature.com/articles/s41588-025-02265-4#Fig16)). NEPC tumors shared methylation profiles with D28 cells, which were clearly separated from D2 and D14 cells and CRPC tumors (Fig. [6d](https://www.nature.com/articles/s41588-025-02265-4#Fig6)). For instance, the _HOXB2_ gene, induced in D28 cells, was marked by dense mCpG in D2 and CRPC PDXs but became completely demethylated in D28 and NEPC PDXs, whereas an opposite DNA methylation remodeling was observed at the AR promoter (Fig. [6e](https://www.nature.com/articles/s41588-025-02265-4#Fig6)). Significantly, mA, reflecting FOXA2 binding, was substantially more enriched at NE-CREs than Ad-CREs in NEPC PDX tumors and was mutually exclusive to mCpG (Fig. [6f, g](https://www.nature.com/articles/s41588-025-02265-4#Fig6)).

### p300/CBP are critical therapeutic targets in NEPC

To investigate how NE enhancers are ultimately activated, we performed mass spectrometry (MS) and found that FOXA2 interacts with histone acetyltransferases CREBBP (CBP) and EP300 (p300), which was further confirmed by co-immunoprecipitation (Co-IP; Fig. [7a–c](https://www.nature.com/articles/s41588-025-02265-4#Fig7)). Moreover, chromatin immunoprecipitation followed by sequencing (ChIP–seq) demonstrated that p300 and consequently H3K27ac were strongly enriched at FOXA2 and NKX2-1 co-occupied NE enhancers, which were abolished by either _FOXA2_ or _NKX2-1_ KD (Fig. [7d](https://www.nature.com/articles/s41588-025-02265-4#Fig7) and Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02265-4#Fig17)), suggesting that both FOXA2 and NKX2-1 are required for p300 recruitment to NE enhancers to catalyze H3K27ac. Not surprisingly, the depletion of p300 and, to some extent, CBP greatly reduced H3K27ac at these enhancer elements. Significantly, CCS1477, an orally active and selective inhibitor of p300/CBP bromodomain[53](https://www.nature.com/articles/s41588-025-02265-4#ref-CR53 "Welti, J. et al. Targeting the p300/CBP axis in lethal prostate cancer. Cancer Discov 11, 1118–1137 (2021)."), completely eliminated H3K27ac at NE enhancers, many of which were SEs that regulate lineage-specific TFs and pro-oncogenes (Extended Data Fig. [9b–f](https://www.nature.com/articles/s41588-025-02265-4#Fig17) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). To determine whether p300/CBP are required for FOXA2/NKX2-1-mediated transcriptional program, we performed RNA-seq in LuNE cells with KD of control, _p300_, _CBP_ or both (Extended Data Fig. [9g](https://www.nature.com/articles/s41588-025-02265-4#Fig17)). Notably, most FOXA2/NKX2-1-regulated genes were similarly regulated by _CBP_ or _p300_ KD (Fig. [7e](https://www.nature.com/articles/s41588-025-02265-4#Fig7)), with their induced genes involved in neurogenesis and development (Extended Data Fig. [9h,i](https://www.nature.com/articles/s41588-025-02265-4#Fig17)). CCS1477 treatment of LuNE cells led to the repression of a large number of genes that were enriched for development, differentiation and neurogenesis and dependent on FOXA2, NKX2-1 and CBP/p300 (Fig. [7f,g](https://www.nature.com/articles/s41588-025-02265-4#Fig7) and Extended Data Fig. [9j](https://www.nature.com/articles/s41588-025-02265-4#Fig17)). Of note, LuNE cell SE-associated genes were significantly suppressed by CCS1477, which, when used in LNCaP cells, inhibited a distinct set of genes associated with luminal SEs, suggesting its ability to preferentially target lineage-specific TFs and proto-oncogenes controlled by their respective SEs[54](https://www.nature.com/articles/s41588-025-02265-4#ref-CR54 "Loven, J. et al. Selective inhibition of tumor oncogenes by disruption of super-enhancers. Cell 153, 320–334 (2013)."),[55](https://www.nature.com/articles/s41588-025-02265-4#ref-CR55 "Whyte, W. A. et al. Master transcription factors and mediator establish super-enhancers at key cell identity genes. Cell 153, 307–319 (2013).") (Fig. [7h](https://www.nature.com/articles/s41588-025-02265-4#Fig7), Supplementary Fig. [2a,b](https://www.nature.com/articles/s41588-025-02265-4#MOESM1) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)).

**Fig. 7: NKX2-1 and FOXA2 recruit p300/CBP to activate NE enhancers and induce NEPC tumor growth, which can be abolished by p300/CBP inhibition.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/7)

**a**, Volcano plot showing FOXA2-interacting proteins in LuNE cells by MS (pooled data from three Co-IP replicates). The _x_ axis represents log2(FC), and the _y_ axis represents −log10_P_ by two-sided _t_ test. Each dot represents a protein. **b**, Co-IP showing that FOXA2 and NKX2-1 interact with CBP and p300 in LuNE cells. Data shown are from one of three (_n_ = 3) independent experiments. **c**, Co-IP showing that NKX2-1 interacts with FOXA2 and p300 in NCI-H660 cells. Data are shown as in **b** (_n_ = 3). **d**, Heatmaps showing indicated ChIP–seq intensity centered (±2 kb) around the five clusters of FOXA2-binding sites identified in the time-course LNCaP+FOXA2 cells. Scale bar—enrichment intensity. **e**, Heatmap showing that genes induced by FOXA2 and NKX2-1 (FC ≥ 2 and adjusted _P_ < 0.05) are regulated by p300/CBP in LuNE cells. Data shown is the log2(FC) of sg_NKX2-1_, sg_FOXA2_ or sg_FOXA2_+sg_NKX2-1_ relative to sg_NC_, as well as of sh_p300_, sh_CBP_ or sh_p300_+sh_CBP_ to sh_Ctrl_. Adjusted _P_ values by Wald test with Benjamini–Hochberg correction. **f**, LuNE cells were treated with DMSO or CCS1477 (250 nM) for 72 h. Differentially expressed genes were identified by DESeq2 with FC ≥ 2, adjusted _P_ < 0.05. Color bar—_z_ score. Adjusted _P_ values are calculated as in **e**. **g**, GO analysis of CCS1477-repressed genes identified in **f**. Top enriched molecular concepts are shown on the _y_ axis, while the _x_ axis indicates enrichment. _P_ values by one-sided hypergeometric test. **h**, Gene set enrichment analysis (GSEA) showing enrichment of SE-associated genes in LuNE cells treated with DMSO or CCS1477. _P_ value by two-sided permutation test with Benjamini–Hochberg correction. **i**,**j**. Tumor growth curve (**i**) and weight (**j**) of LuNE xenograft tumors treated with vehicle or CCS1477 (_n_ = 6 mice per group). Data are mean ± s.e.m, and _P_ values by two-sided _t_ test. **k**,**l**. Tumor growth curve (**k**) and weight (**l**) of LuCaP145.2 xenograft tumors treated with vehicle or CCS1477 (_n_ = 6 mice per group in **k**, _n_ = 7 tumors per group in **l**). Data are mean ± s.d, and _P_ values by two-sided _t_ test. **m**, Representative IHC images (_n_ = 6 mice per group) of H3K27ac, SOX2, NKX2-1 and Ki-67 in LuCaP145.2 xenograft tumors treated with vehicle or CCS1477. Scale bar = 30 µm. Insets are shown at higher magnification (×40).

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM7)

Finally, we evaluated the efficacy of CCS1477 in inhibiting NEPC. Depletion of _NKX2-1_, _FOXA2_, _p300_ or, to a lesser extent, _CBP_ substantially inhibited LuNE cell colony formation, supporting the essentiality of their target genes (Extended Data Fig. [10a](https://www.nature.com/articles/s41588-025-02265-4#Fig18)). Consistently, CCS1477 massively inhibited colony formation of LuNE and LNCaP cells by inducing cell cycle arrest, without affecting the benign and normal prostate cell lines (Extended Data Fig. [10b–e](https://www.nature.com/articles/s41588-025-02265-4#Fig18) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02265-4#MOESM1)). Moreover, CCS1477 achieved a similarly strong growth inhibition of additional NEPC cells, LuCaP145.2 and NCI-H660 (Extended Data Fig. [10f](https://www.nature.com/articles/s41588-025-02265-4#Fig18)). To examine the efficacy of CCS1477 in targeting NEPC tumors in vivo, we performed subcutaneous injection of LuNE cells in severe combined immunodeficient (SCID) mice. We started to see apparent growth reduction after 3 weeks of treatment (Fig. [7i](https://www.nature.com/articles/s41588-025-02265-4#Fig7)). Interestingly, despite treatment ending at day 33, tumor inhibition continued, with an inhibition rate (IR) of 68.7% (_P_ = 0.003) at day 46 when the experiment was terminated. There was a remarkable difference in endpoint tumor weight (_P_ = 0.0006) and volume, while the body weights of the mice were not significantly affected (Fig. [7j](https://www.nature.com/articles/s41588-025-02265-4#Fig7) and Extended Data Fig. [10g,h](https://www.nature.com/articles/s41588-025-02265-4#Fig18)). CCS1477 drastically reduced H3K27ac and NE TFs NKX2-1 and SOX2, as expected, but it was unable to rescue luminal markers AR and PSA (Extended Data Fig. [10i,j](https://www.nature.com/articles/s41588-025-02265-4#Fig18)). Similar tumor-inhibitory effects of CCS1477 were also observed in NEPC PDX (LuCaP145.2) tumors (Fig. [7k–m](https://www.nature.com/articles/s41588-025-02265-4#Fig7) and Extended Data Fig. [10k,l](https://www.nature.com/articles/s41588-025-02265-4#Fig18)). In summary, these data support that p300/CBP has an important role in mediating NEPC SE addition and growth, which can be effectively mitigated by CCS1477.

Discussion
----------

NEPC is a subtype of CRPC that exhibits NE features and displays distinct epigenetic and transcriptional programs compared to most CRPC tumors. Consistent with this, our data showed that the 3D genome conformation was reorganized in NEPC compared to CRPC PDX tumors, despite substantial heterogeneity within each group, especially among NEPC samples. This chromatin reorganization is marked by the loss of luminal E–P loops and the formation of new E–P loops and TADs that favor the NE program, which was recapitulated by isogenic LNCaP cells undergoing NET upon FOXA2 OE. New E–P loops mediated by the interactions between enhancer-bound FOXA2 and promoter-bound NKX2-1 emerged at D14, following the induction of NKX2-1 expression, and drastically increased at D21 and D28. This may be further strengthened as the NE enhancers get activated by p300/CBP (Fig. [8](https://www.nature.com/articles/s41588-025-02265-4#Fig8)), which is known to promote the formation of subnuclear condensates that mediate long-range DNA interactions[56](https://www.nature.com/articles/s41588-025-02265-4#ref-CR56 "Ma, L. et al. Co-condensation between transcription factor and coactivator p300 modulates transcriptional bursting kinetics. Mol. Cell 81, 1682–1697 (2021)."),[57](https://www.nature.com/articles/s41588-025-02265-4#ref-CR57 "Ibrahim, Z. et al. Structural insights into p300 regulation and acetylation-dependent genome organisation. Nat. Commun. 13, 7759 (2022).").

**Fig. 8: Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 and FOXA2 drive NE transdifferentiation of PCa.**

[![figure 8](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_Fig8_HTML.png)](https://www.nature.com/articles/s41588-025-02265-4/figures/8)

As a pioneer factor, ectopic FOXA2, once overexpressed in luminal cells, is capable of binding to inaccessible NE lineage enhancers. It induces regional DNA demethylation and increases local chromatin accessibility, thereby activating the expression of NE lineage TFs, such as NKX2-1. Once induced, NKX2-1 preferentially binds to gene promoters and interacts with enhancer-bound FOXA2 through chromatin looping (intermediate stage). This interaction further strengthens FOXA2 occupancy at NE enhancers. Together, FOXA2 and NKX2-1 recruit p300/CBP, which catalyze H3K27 acetylation, further promoting 3D chromatin reorganization, activating NE enhancers and driving the expression of NE lineage-specific transcriptional programs.

Closely aligned with the 3D chromatin reorganization are the changes in the epigenetic landscape in both PDX tumors and LNCaP+FOXA2 cells undergoing NET. We found that FOXA2 OE leads to regional DNA demethylation around its binding sites, which is consistent with recent reports of FOXA2 function in endodermal lineage intermediates to catalyze regional DNA demethylation that is critical for liver and pancreatic endocrine cell specification[58](https://www.nature.com/articles/s41588-025-02265-4#ref-CR58 "Li, J. et al. TET1 dioxygenase is required for FOXA2-associated chromatin remodeling in pancreatic β-cell differentiation. Nat. Commun. 13, 3907 (2022)."),[59](https://www.nature.com/articles/s41588-025-02265-4#ref-CR59 "Reizel, Y. et al. FoxA-dependent demethylation of DNA initiates epigenetic memory of cellular identity. Dev. Cell 56, 602–612 (2021)."). We believe regional DNA demethylation at NE enhancers following FOXA2 binding is a critical step toward subsequent deposition of H3K4me1 to prime these lineage-specific enhancers for following activation by lineage-inductive TFs, as previously observed during embryonic development[50](https://www.nature.com/articles/s41588-025-02265-4#ref-CR50 "Wang, A. et al. Epigenetic priming of enhancers predicts developmental competence of hESC-derived endodermal lineage intermediates. Cell Stem Cell 16, 386–399 (2015)."),[59](https://www.nature.com/articles/s41588-025-02265-4#ref-CR59 "Reizel, Y. et al. FoxA-dependent demethylation of DNA initiates epigenetic memory of cellular identity. Dev. Cell 56, 602–612 (2021)."),[60](https://www.nature.com/articles/s41588-025-02265-4#ref-CR60 "Jozwik, K. M., Chernukhin, I., Serandour, A. A., Nagarajan, S. & Carroll, J. S. FOXA1 directs H3K4 monomethylation at enhancers via recruitment of the methyltransferase MLL3. Cell Rep. 17, 2715–2723 (2016)."),[61](https://www.nature.com/articles/s41588-025-02265-4#ref-CR61 "Donaghey, J. et al. Genetic determinants and epigenetic effects of pioneer-factor occupancy. Nat. Genet. 50, 250–258 (2018)."),[62](https://www.nature.com/articles/s41588-025-02265-4#ref-CR62 "Cernilogar, F. M. et al. Pre-marked chromatin and transcription factor co-binding shape the pioneering activity of Foxa2. Nucleic Acids Res. 47, 9069–9086 (2019)."),[63](https://www.nature.com/articles/s41588-025-02265-4#ref-CR63 "Lee, K. et al. FOXA2 is required for enhancer priming during pancreatic differentiation. Cell Rep. 28, 382–393 e387 (2019)."),[64](https://www.nature.com/articles/s41588-025-02265-4#ref-CR64 "Zaret, K. S. Pioneer transcription factors initiating gene network changes. Annu. Rev. Genet. 54, 367–385 (2020)."). Our study highlighted the important role of FOXA2 in initiating an epigenetic memory of the NE cellular identity. While how FOXA2 mediates DNA demethylation and how it leads to enhancer priming are beyond the scope of the current study, FOXA proteins have been shown to recruit TET1 for DNA demethylation[65](https://www.nature.com/articles/s41588-025-02265-4#ref-CR65 "Yang, Y. A. et al. FOXA1 potentiates lineage-specific enhancer activation through modulating TET1 expression and function. Nucleic Acids Res. 44, 8153–8164 (2016).") and MLL3 for H3K4me1 deposition at target enhancers in other cell types[60](https://www.nature.com/articles/s41588-025-02265-4#ref-CR60 "Jozwik, K. M., Chernukhin, I., Serandour, A. A., Nagarajan, S. & Carroll, J. S. FOXA1 directs H3K4 monomethylation at enhancers via recruitment of the methyltransferase MLL3. Cell Rep. 17, 2715–2723 (2016).").

We report an essential feed-forward loop, orchestrated by lineage-inductive pioneer factors and TFs, for NET of PCa. _NKX2-1_ is a direct transcriptional target of FOXA2. Once NKX2-1 is induced by FOXA2 OE, it binds to promoters and interacts with enhancer-bound FOXA2 by forming E–P loops. NKX2-1 binding further stabilizes FOXA2 recruitment to NE enhancers, facilitating subsequent NE enhancer priming and activation (Fig. [8](https://www.nature.com/articles/s41588-025-02265-4#Fig8)), which is consistent with recent findings for its role in enforcing organ cell type-specific gene expression during pancreatic, hepatic and lung development[66](https://www.nature.com/articles/s41588-025-02265-4#ref-CR66 "Geusz, R. J. et al. Sequence logic at enhancers governs a dual mechanism of endodermal organ fate induction by FOXA pioneer factors. Nat. Commun. 12, 6636 (2021)."). Although our current study deciphers this fundamental mechanism of NET using the LNCaP+FOXA2 model, the fact that FOXA2 and NKX2-1 are co-expressed in a majority of patients with NEPC might suggest its generalizability. We did observe a more specific and broader induction of NKX2-1 in NEPC tumors than FOXA2, suggesting that NKX2-1 might collaborate with and function through other pioneering factors. For instance, a previous study has noted that ectopic NKX2-1 could enhance the ability of ASCL1 to redistribute FOXA1 to NE enhancers and induce NET in LNCaP cells that are negative for FOXA2 (ref. [11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).")). Indeed, we found significantly overlapping binding sites between NKX2-1 and ASCL1 in FOXA2-low PDX (Extended Data Fig. [7e](https://www.nature.com/articles/s41588-025-02265-4#Fig15)). Therefore, NKX2-1 may cooperate with ASCL1 in FOXA2-low NEPC and with both pioneer factors in NEPC tumors expressing all three proteins.

Presently, there are no effective treatments for NEPC except platinum-based chemotherapy. Our study delineated that p300/CBP is essential for FOXA2 and NKX2-1-mediated NE gene expression and NEPC growth. We found that CCS1477, a p300/CBP inhibitor that targets AR and c-Myc in CRPC and is currently in clinical trial[53](https://www.nature.com/articles/s41588-025-02265-4#ref-CR53 "Welti, J. et al. Targeting the p300/CBP axis in lethal prostate cancer. Cancer Discov 11, 1118–1137 (2021)."), also significantly inhibited NEPC cell growth. CCS1477 inhibits lineage-specific SEs to suppress cancer type-specific TFs and proto-oncogenes that are distinct in LNCaP and LuNE cells. CCS1477 thus reduced the growth of both CRPC and NEPC, but not benign prostate cells, likely due to PCa being enhancer-addicted[67](https://www.nature.com/articles/s41588-025-02265-4#ref-CR67 "Xiao, L. et al. Targeting SWI/SNF ATPases in enhancer-addicted prostate cancer. Nature 601, 434–439 (2022)."). Consistent with our findings, a recent study reported that CCS1477 concurrently inhibits two distinct hematological malignancies—myeloid leukemia and multiple myeloma—by impairing their respective lineage-specific enhancer activities[68](https://www.nature.com/articles/s41588-025-02265-4#ref-CR68 "Nicosia, L. et al. Therapeutic targeting of EP300/CBP by bromodomain inhibition in hematologic malignancies. Cancer Cell 41, 2136–2153 (2023)."). However, CCS1477 was unable to convert NEPC tumors back to adenocarcinoma, likely due to the chromatin/epigenetic state of the terminally transformed NEPC cells being fairly stable. Therefore, our results suggest the therapeutic potential of p300/CBP inhibitors in simultaneously targeting CRPC and NEPC and indicate the necessity of early detection of NET to prevent/reverse NEPC progression.

Methods
-------

### Ethics statement

Our research complies with all relevant ethical regulations. Mouse handling and experimental procedures were approved by the Institutional Animal Care and Use Committee at Northwestern University in accordance with the US National Institutes of Health Guidelines for the Care and Use of Laboratory Animals and the Animal Welfare Act. TMAs containing metastatic CRPC specimens were obtained as part of the University of Washington Medical Center Prostate Cancer Donor Program, which is approved by the University of Washington Institutional Review Board.

### Cell lines, chemical reagents and antibodies

PCa cell lines LNCaP, 22Rv1, DU145, NCI-H660, human benign prostatic hyperplasia cell line BPH-1, human normal prostate epithelial cell line RWPE-1 and human embryonic kidney cell line HEK293T cells were obtained from the American Type Culture Collection (ATCC). LNCaP, 22Rv1, DU145, BPH-1 and HEK293T were maintained in either RPMI1640 or Dulbecco’s modified Eagle’s medium with 10% FBS and 1% penicillin and streptomycin. RWPE-1 cells were maintained in keratinocyte serum-free medium with 0.05 mg ml−1 BPE and 5 ng ml−1 epidermal growth factor. NCI-H660 cells were maintained in ATCC-formulated RPMI1640 medium (30-2001) with 0.005 mg ml−1 insulin, 0.01 mg ml−1 transferrin, 30 nM sodium selenite, 10 nM hydrocortisone, 10 nM β-estradiol and 5% FBS. The 42D cells were maintained in RPMI1640 with 5% FBS, 1% penicillin and streptomycin and 10 μM ENZ for all experiments. All the cells were authenticated within 6 months of growth, and cells under culture were frequently tested for potential mycoplasma contamination. CCS1477 (CT-CCS1477) was purchased from Chemietek. All antibodies used in this study are listed in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM4).

### Constructs and lentivirus infection

Human _FOXA2_ coding region (CDS) was first amplified using cDNA from PC-3 cells as a template. _NKX2-1_ and mouse _Foxa2_ CDS were amplified using _NKX2-1_ (Addgene, 119173) and m_Foxa2_ (Addgene, 33014) as templates, respectively, and then cloned into plenti CMV Neo DEST (705-1) (Addgene, 17392) vector by In-Fusion HD Cloning Kits (TaKaRa, 638948). _FOXA2_, _NKX2-1_, _RB1_ and _TP53_ gRNAs were cloned into lentiCRISPR v2 (Addgene, 52961) vector. The shRNAs targeting CBP and p300 were cloned into pLKO.1-TRC lentiviral vector (Addgene, 10878). si_NKX2-1_ 1 (s224731), si_NKX2-1_ 2 (s14152) and si_ASCL1_ (s1657) were purchased from Thermo Fisher Scientific. The primers and oligonucleotides used in this study are listed in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02265-4#MOESM4), and all the plasmids were verified by Sanger sequencing. For details on lentivirus generation and infection, see Supplementary Method [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM1).

### Co-IP

Nuclear fraction was used for all Co-IP experiments in this study. Nuclear proteins were isolated as previously described with some modifications[42](https://www.nature.com/articles/s41588-025-02265-4#ref-CR42 "Lu, X. et al. HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer. Nat. Genet. 54, 670–683 (2022)."),[69](https://www.nature.com/articles/s41588-025-02265-4#ref-CR69 "Xu, B. et al. Altered chromatin recruitment by FOXA1 mutations promotes androgen independence and prostate cancer progression. Cell Res. 29, 773–775 (2019)."). For detailed, step-by-step protocols, see Supplementary Method [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM1). All antibodies used in WB and Co-IP are listed in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM4).

### 3D organoid cell culture and colony formation assay

LuCaP145.2 and LuCaP93 PDX-derived organoids were generated as previously described[70](https://www.nature.com/articles/s41588-025-02265-4#ref-CR70 "Drost, J. et al. Organoid culture systems for prostate epithelial and cancer tissue. Nat. Protoc. 11, 347–358 (2016)."). For detailed, step-by-step protocols, see Supplementary Method [3](https://www.nature.com/articles/s41588-025-02265-4#MOESM1). For details on the colony formation assay, see Supplementary Method [4](https://www.nature.com/articles/s41588-025-02265-4#MOESM1).

### MS analysis

Chromatin fraction was used for the MS experiments in this study. Chromatin proteins were isolated as previously described with some modifications[42](https://www.nature.com/articles/s41588-025-02265-4#ref-CR42 "Lu, X. et al. HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer. Nat. Genet. 54, 670–683 (2022)."),[69](https://www.nature.com/articles/s41588-025-02265-4#ref-CR69 "Xu, B. et al. Altered chromatin recruitment by FOXA1 mutations promotes androgen independence and prostate cancer progression. Cell Res. 29, 773–775 (2019)."). For detailed, step-by-step protocols, see Supplementary Method [5](https://www.nature.com/articles/s41588-025-02265-4#MOESM1). MS analysis was performed using the Orbitrap Velos Pro system. SEQUEST was used for protein identification and peptide sequencing. The data in Fig. [7a](https://www.nature.com/articles/s41588-025-02265-4#Fig7) were analyzed as previously[71](https://www.nature.com/articles/s41588-025-02265-4#ref-CR71 "Aguilan, J. T., Kulej, K. & Sidoli, S. Guide for protein fold change and p-value calculation for non-experts in proteomics. Mol. Omics 16, 573–582 (2020)."). Briefly, the sum of all peptide intensities for each protein in triplicate was log2\-transformed and normalized by the average. Missing peptide intensities were imputed from a Gaussian distribution with a width of 0.3, centered at the sample distribution mean minus 2.5 times the sample s.d. The criterion for statistically significant differential interactions was determined using an unpaired two-sided _t_ test with a _P_ value < 0.01 and a fold change (FC; FOXA2 versus IgG) greater than 8.

### RNA extraction, RT–qPCR and RNA-seq

RNA extraction, RT–qPCR and RNA-seq were performed as previously with some modifications[42](https://www.nature.com/articles/s41588-025-02265-4#ref-CR42 "Lu, X. et al. HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer. Nat. Genet. 54, 670–683 (2022)."),[69](https://www.nature.com/articles/s41588-025-02265-4#ref-CR69 "Xu, B. et al. Altered chromatin recruitment by FOXA1 mutations promotes androgen independence and prostate cancer progression. Cell Res. 29, 773–775 (2019)."). For detailed, step-by-step protocols, see Supplementary Method [6](https://www.nature.com/articles/s41588-025-02265-4#MOESM1). All primers used in RT–PCR analysis are listed in Supplementary Table [3](https://www.nature.com/articles/s41588-025-02265-4#MOESM4).

### ChIP, ChIP–seq and ATAC–seq

ChIP, ChIP–seq and ATAC–seq were performed using the previously described protocol with some modifications[42](https://www.nature.com/articles/s41588-025-02265-4#ref-CR42 "Lu, X. et al. HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer. Nat. Genet. 54, 670–683 (2022)."),[69](https://www.nature.com/articles/s41588-025-02265-4#ref-CR69 "Xu, B. et al. Altered chromatin recruitment by FOXA1 mutations promotes androgen independence and prostate cancer progression. Cell Res. 29, 773–775 (2019)."). For detailed, step-by-step protocols, see Supplementary Method [7](https://www.nature.com/articles/s41588-025-02265-4#MOESM1). The information on antibodies used in ChIP is listed in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM4).

### Bulk RNA-seq analysis

Human PCa cell RNA-seq reads were mapped to the National Center for Biotechnology Information human genome GRCh38. Raw counts of genes were calculated by STAR. Fragments per kilobase of transcript per million mapped reads (FPKM) values were calculated by an in-house Perl script. The details for differential gene expression analysis were summarized here. Briefly, as Fig. [1c,d](https://www.nature.com/articles/s41588-025-02265-4#Fig1) data were derived from the public data and did not have raw count data, which is required for DESeq2, we used FPKM value for each gene and performed differential expression analyses using _t_ tests between the two groups. Figure [2a](https://www.nature.com/articles/s41588-025-02265-4#Fig2) used DESeq2 with the likelihood ratio test, considering the time-course nature of the data, whereas Figs. [5a](https://www.nature.com/articles/s41588-025-02265-4#Fig5) ([GSE126078](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126078) and ref. [3](https://www.nature.com/articles/s41588-025-02265-4#ref-CR3 "Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. Nat. Med. 22, 298–305 (2016).") datasets) and [7e,f](https://www.nature.com/articles/s41588-025-02265-4#Fig7) used DESeq2 with the default Wald test in a pairwise manner. DESeq2 has built-in multiple-test corrections for all tests. Figure [5a](https://www.nature.com/articles/s41588-025-02265-4#Fig5) ([GSE74685](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74685) (microarray)) dataset used the Bioconductor limma package. The ARS and NES genes used in Fig. [2c](https://www.nature.com/articles/s41588-025-02265-4#Fig2) were from a previous study[3](https://www.nature.com/articles/s41588-025-02265-4#ref-CR3 "Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. Nat. Med. 22, 298–305 (2016)."). The scatterplot was generated using Gene Set Variation Analysis (1.52.3)[72](https://www.nature.com/articles/s41588-025-02265-4#ref-CR72 "Hanzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinform 14, 7 (2013)."). RNA-seq data of patients with PCa were downloaded from [GSE126078](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126078) (ref. [40](https://www.nature.com/articles/s41588-025-02265-4#ref-CR40 "Labrecque, M. P. et al. Molecular profiling stratifies diverse phenotypes of treatment-refractory metastatic castration-resistant prostate cancer. J. Clin. Invest. 129, 4492–4505 (2019).")).

### ChIP–seq, SE and ATAC–seq analyses

For details on ChIP–seq, SE and ATAC–seq analyses, see Supplementary Method [8](https://www.nature.com/articles/s41588-025-02265-4#MOESM1).

### Single-cell sequencing sample preparation and analysis

A total of 5,000 D0, D14 and D21 LNCaP+FOXA2 cells were used for a single-cell Multiome ATAC + Gene Expression assay, and the Multiome libraries were prepared using 10X Genomics Chromium Next GEM Single-Cell Multiome ATAC + Gene Expression Reagent Kits (1000285) per the manufacturer’s protocol. Single-cell libraries were sequenced in a NovaSeq 6000 with an average depth of 30,000 reads per cell. Raw sequencing data were processed and aligned to hg38 using Cell Ranger ARC (2.0.2). Low-quality cells with low unique molecular identifier counts and high mitochondrial ratios were filtered out, and the sequence metrics were updated in Supplementary Table [4](https://www.nature.com/articles/s41588-025-02265-4#MOESM4). Each time point was analyzed individually at the expression and accessibility modality and then combined across time points within each modality using Seurat (4.3.0) and Signac (1.6.0) in R (4.0.3). For each time point, we performed quality control on ATAC fragments, RNA counts, nucleosome signal and transcription start site enrichment metrics, resulting in 2,110, 1,739 and 2,344 cells for D0, D14 and D21, respectively. The RNA modality was normalized using SCTransform. ATAC peaks were called using MACS2 and quantified. Counts were normalized using latent semantic indexing. These processes were performed separately for each condition.

To correct for batch effects, integration of the expression modality was performed using the Seurat V3 integration pipeline, which uses reciprocal PCA to identify anchors with 3,000 integration features. UMAP was constructed using the first 30 PCs. To visualize the gene expression, log-normalized data were used. ARS and NES were calculated using ‘AddModuleScore’. AR+ was defined as ≥0.6, which is the mean of normalized AR expression in D21. ARS+ and NES+ cells are defined as cells with a signature score ≥0. Monocle3 (0.2.3.0) was used to learn the trajectory graph and calculate the pseudotime. Integration of the chromatin accessibility was done by first finding a common peak set between all the conditions, counting features and renormalizing. Integration anchors were found using ‘rlsi’ using 2:50 dimensions. ARS and NES for the chromatin modality were generated by identifying peaks located within 5 kb upstream or within the gene body of the respective genes. Scores were obtained using Signac’s ‘AddChromatinModule’, which calculates chromVAR deviations.

Velocyto (v0.17.17) was used to generate spliced and unspliced counts for all cells. RNA velocity analysis was performed on a subset of previously identified high-quality cells. Genes included in the analysis were filtered based on detection levels, and the top 3,000 highly variable genes were selected and normalized. K-nearest neighbor (KNN) imputation was applied to 20 PCA dimensions. The ‘constant velocity’ assumption was used for velocity estimation. RNA velocities were projected onto the UMAP calculated in Seurat.

For details on the quantification of _FOXA2_ and _NKX2-1_ double-positive cells in D21, as well as the integrative analysis of LNCaP+FOXA2 scRNA-seq data with human PCa, see Supplementary Method [9](https://www.nature.com/articles/s41588-025-02265-4#MOESM1).

### Copy number variation (CNV) analysis

Quality control of scRNA-seq and scATAC–seq data was performed as described above. CNV analyses of scRNA-seq data (D0, D14 and D21) were performed by CopyKAT (1.1.0)[73](https://www.nature.com/articles/s41588-025-02265-4#ref-CR73 "Gao, R. et al. Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes. Nat. Biotechnol. 39, 599–608 (2021)."), which applies a Bayesian segmentation approach to estimate copy number profiles with 220 Kb genomic windows. Briefly, the raw count matrix is variance-stabilized and smoothed for outliers. The cell-line mode was enabled, meaning a synthetic baseline derived from data variations was used. Consensus chromosome breakpoints among single-cell clusters were detected, and final copy number profiles were calculated per cell. Raw gene expression matrices for each time point were provided to CopyKAT using default parameters with cell-line mode enabled. Cells in each time point that had CNV variations between −0.1 and 0.1 in more than 80% of the regions were excluded, resulting in 2,352, 2,138 and 4,558 cells for D0, D14 and D21, respectively. The matrices for resulting high-quality cells were then combined. Clones were identified using hierarchical clustering with the Ward D2 method and Euclidean distance. The heatmap was produced with heatmap.3 function.

### Single-cell nanopore RNA-seq (scNanoRNA-seq) sample preparation and analysis

scNanoRNA-seq was performed as previously described[74](https://www.nature.com/articles/s41588-025-02265-4#ref-CR74 "Shiau, C. K. et al. High throughput single cell long-read sequencing analyses of same-cell genotypes and phenotypes in human tumors. Nat. Commun. 14, 4124 (2023)."). Briefly, barcoded full-length cDNAs of single cells were prepared using the 10X Genomics Next GEM Single-Cell 3′ gene expression kit with elongation time extending to 3 min to enrich longer molecules as previously described[75](https://www.nature.com/articles/s41588-025-02265-4#ref-CR75 "Lebrigand, K., Magnone, V., Barbry, P. & Waldmann, R. High throughput error corrected Nanopore single cell transcriptome sequencing. Nat. Commun. 11, 4025 (2020)."). Then, the full-length cDNA libraries were prepared for nanopore long-read sequencing using the SQK-LSK114 ligation sequencing kit (Oxford Nanopore). The final sequencing was run on a PromethION flow cell (R10.4) with one sample per flow cell. Base calling was done using Guppy (v6.4.6). For details on single-nucleotide variant and germline mutation analyses in LNCaP+FOXA2 time-course cells (D0 and D28), see Supplementary Method [10](https://www.nature.com/articles/s41588-025-02265-4#MOESM1).

### Hi-C sample preparation and analysis

FOXA2-D0, D14, D21 and D28 LNCaP cells and NCI-H660 cells were fixed with 2% formaldehyde (final concentration) at room temperature for 10 min, whereas tumor tissues from LuCaP PDX were fixed with 2% formaldehyde at room temperature for 20 min, followed by adding 2.5 M glycine to a final concentration of 0.2 M to quench the reaction. Hi-C libraries were generated using the Arima-Hi-C+ kit (Arima Genomics, A510008) per the manufacturer’s protocol. The Hi-C libraries were sequenced using paired-end 2 × 150 bp reads on the Illumina NovaSeq 6000 or NovaSeq X Plus. The sequence metrics are provided in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02265-4#MOESM4).

Hi-C data were processed into.mcool maps with multiresolution using the runHiC (0.8.6) pipeline, with chromap as the aligner to hg38, and iterative correction and eigenvector decomposition (ICE)-normalization was applied. Maps were uploaded to and visualized on resgen.io, which relies on HiGlass. A/B compartments were called using cooltools (0.5.4) eigs\_cis function with GC content phasing at 100 kb resolution. The first eigenvector was extracted for each sample, and bins with nonzero values across all samples were retained. The top 25% most variable bins were identified separately for the 9 samples in Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02265-4#Fig9) and the 17 samples in Extended Data Fig. [2g](https://www.nature.com/articles/s41588-025-02265-4#Fig10). Pearson correlations between the most variable eigenvectors were then calculated pairwise.

NEPC- and CRPC-enriched loops were identified using the diffMustache module in Mustache (Fig. [1a,b](https://www.nature.com/articles/s41588-025-02265-4#Fig1)). Loops from each CRPC and NEPC PDX sample were called at 10 kb resolution with an FDR of 0.05 and a sparsity threshold (-st) of 0.8. These criteria were used to evaluate loop intensity as well as the local background to exclude artifacts. To identify NEPC-enriched loops, first, each NEPC PDX was compared to each of the four CRPC PDXs independently to identify differential loops with an FDR of 0.001. The differential loops shared in at least two of such pairwise NEPC–CRPC comparisons were then defined as gained loops for that NEPC PDX, and this process was done for all four NEPC PDXs. Finally, differential loops gained in two or more NEPC PDXs were defined as NEPC subtype-enriched loops. Analogous analysis was performed to identify CRPC subtype-enriched loops.

APAs to determine genome-wide loop intensity in Figs. [1a,b](https://www.nature.com/articles/s41588-025-02265-4#Fig1), [2d](https://www.nature.com/articles/s41588-025-02265-4#Fig2) and [4f](https://www.nature.com/articles/s41588-025-02265-4#Fig4) were made with coolpup.py (1.0.0). Loops called at 10 kb resolution were piled up and centered at a 50 kb × 50 kb matrix. The averaged loop intensity was normalized to the expected value. APA scores are the mean of the central 3 × 3 square of the matrix. Significance of the differences between NEPC-enriched loops and CRPC-enriched loops was performed using a Student’s _t_ test by comparing the APA scores of the NEPC samples to those of the CRPC samples.

Genes linked to NEPC- or CRPC-enriched loops were defined as genes within ±3 kb of loop anchors. Each gene’s expression was obtained as publicly available FPKM values, and differential expression was determined with a _t_ test between the two groups, in Fig. [1c,d](https://www.nature.com/articles/s41588-025-02265-4#Fig1). CRPC- or NEPC-enriched loops that overlap with CREs were defined as loops with either anchor having an overlap with CREs. Ad/NE-CREs were converted from hg19 to hg38 using UCSC’s liftOver tool. Statistical tests of the significance of the overlap between subtype-enriched loops and CREs were calculated using a permutation test with 1,000 randomizations where the loop anchors were sampled from random regions within the same chromosomes. Enriched motifs were identified using these overlapping CREs with HOMER. FOXA2- and NKX2-1-anchored loops were annotated based on the presence of matching ChIP–seq binding sites within the anchors. Figure [4f](https://www.nature.com/articles/s41588-025-02265-4#Fig4) shows the set of loops in D28 with FOXA2 and NKX2-1 at loop anchors, excluding any loops that overlap with those in D0.

### Reduced representation methylation sequencing using nanopore technology

Reduced representation methylation sequencing libraries were generated using the Oxford Nanopore protocol. Briefly, genomic DNA from LuCaP PDX was extracted using the Quick-DNA Miniprep Plus Kit (Zymo, D4068) and fragmented to an average size of 8 kb using g-TUBE (Covaris, 520079). DNA libraries were prepared with the Ligation Sequencing Kit (ON SQK-LSK110) following the manufacturer’s protocol. The sequencing was performed on an Oxford Nanopore GridION sequencer using R9.4.1 flow cells (FLO-MIN106D). The adaptive sampling method was used to target regions of interest, which included CpGs, CpG islands, shores, shelves and promoter regions. Sequence metrics are provided in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02265-4#MOESM4). Base calling and alignment were performed using Guppy (v6.2.1\_gpu), Minimap2 (v2.26) and Remora (v2.0.0).

### DiMeLo-seq sample preparation and analysis

DiMeLo-seq, a method that uses antibody-tethered enzymes to methylate DNA near a target protein’s binding sites in situ, was performed using the previously described protocol with some modifications[52](https://www.nature.com/articles/s41588-025-02265-4#ref-CR52 "Altemose, N. et al. DiMeLo-seq: a long-read, single-molecule method for mapping protein–DNA interactions genome wide. Nat. Methods 19, 711–723 (2022)."). For detailed, step-by-step protocols, see Supplementary Method [11](https://www.nature.com/articles/s41588-025-02265-4#MOESM1). The sequence metrics were updated in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02265-4#MOESM4).

### Mouse xenograft studies

Mice were housed in specific pathogen-free animal facilities (at 20–23 °C, with 40–60% humidity and 12-h light/12-h dark cycle). LuCaP PDX were derived from resected metastatic PCa with the informed consent of patient donors as described previously[76](https://www.nature.com/articles/s41588-025-02265-4#ref-CR76 "Nguyen, H. M. et al. LuCaP prostate cancer patient-derived xenografts reflect the molecular heterogeneity of advanced disease and serve as models for evaluating cancer therapeutics. Prostate 77, 654–671 (2017).") under a protocol approved by the University of Washington Human Subjects Division institutional review board. Non-obese diabetic (NOD)/SCID male mice at 6–7 weeks old were purchased from Charles River Laboratories. For the LuNE xenograft, 2 × 106 cells were subcutaneously injected into the right flank of the mice. Tumor volumes were measured twice per week with digital calipers, using the following formula: _V_ = _L_ × _W_2/2 (_V_, mm3; _L_, mm; _W_, mm). Once the size of tumors reached approximately 70 mm3, the mice were randomized to receive vehicle (5% DMSO:95% methylcellulose) or CCS1477 (20 mg kg−1) daily by oral gavage for 33 days. For LuCaP145.2 PDX xenograft, fresh tumors were cut into small pieces (8 mm3) using a scalpel and were subcutaneously implanted into both sides of SCID mice. The size of tumors reached approximately 150 mm3 at 1 month after implantation, and then mice were randomized to receive vehicle (5% DMSO:95% methylcellulose) or CCS1477 (20 mg kg−1) every day by oral gavage for 18 days. The tumors and other organs of mice were collected for downstream analyses at the endpoint (tumor size did not exceed the maximal tumor volume of 2,000 mm3, as specified in the approved protocol). The IR of tumor growth was calculated as (tumor volume of vehicle − tumor volume of CCS1477)/tumor volume of vehicle.

### Immunohistochemistry (IHC) and TMA analysis

For detailed step-by-step protocols on IHC, see Supplementary Method [12](https://www.nature.com/articles/s41588-025-02265-4#MOESM1). For human TMA staining, TMAs containing metastatic CRPC specimens were obtained as part of the University of Washington Medical Center Prostate Cancer Donor Program, which is approved by the University of Washington Institutional Review Board. All specimens for IHC were formalin-fixed (decalcified in formic acid for bone specimens), paraffin-embedded and examined histologically for the presence of a non-necrotic tumor. TMAs were constructed using 1-mm-diameter duplicate cores (_n_ = 126) from CRPC tissues (_n_ = 23 patients), which consisted of visceral metastases and bone metastases (_n_ = 63 sites) from patients who died within 8 h. TMAs staining of FOXA2 and NKX2-1 was conducted by the Northwestern Pathology Facility. The information of the antibodies used for IHC is listed in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02265-4#MOESM4). Images were captured with TissueFax Plus from TissueGnostics, exported to TissueFAX viewer and analyzed using Photoshop CS4 (Adobe). FOXA2 and NKX2-1 immunostaining was scored blindly by a pathologist using a score of 0–3 for intensities of negative, weak, moderate or intense and multiplied by the percentage of stained cancer cells.

### Statistics and reproducibility

No statistical methods were used to predetermine sample sizes, but our sample sizes are similar to those reported in previous publications[20](https://www.nature.com/articles/s41588-025-02265-4#ref-CR20 "Han, M. et al. FOXA2 drives lineage plasticity and KIT pathway activation in neuroendocrine prostate cancer. Cancer Cell 40, 1306–1323 (2022)."),[42](https://www.nature.com/articles/s41588-025-02265-4#ref-CR42 "Lu, X. et al. HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer. Nat. Genet. 54, 670–683 (2022)."). For each independent in vitro experiment, at least three technical replicates were used. All in vitro experiments were independently repeated at least two or three times, as specified in Fig. [2b](https://www.nature.com/articles/s41588-025-02265-4#Fig2), Fig. [4b,c,g](https://www.nature.com/articles/s41588-025-02265-4#Fig4), Fig. [5d,f–h](https://www.nature.com/articles/s41588-025-02265-4#Fig5), Fig. [7b–c](https://www.nature.com/articles/s41588-025-02265-4#Fig7), Extended Data Fig. [2b–c](https://www.nature.com/articles/s41588-025-02265-4#Fig10), Extended Data Fig. [7d,f,g](https://www.nature.com/articles/s41588-025-02265-4#Fig15), Extended Data Fig. [9g](https://www.nature.com/articles/s41588-025-02265-4#Fig17) and Extended Data Fig. [10d,j](https://www.nature.com/articles/s41588-025-02265-4#Fig18). No data were excluded from the analyses. The error bars in the figures represent the s.d. of the mean, unless stated otherwise. Two-sided Student’s _t_ tests were used to assess statistical significance in RT–qPCR experiments and cell-based functional assays. The statistical analysis methods and software version used for NGS data analysis are provided in the corresponding code and figure legends (Fig. [1c–f,i–j](https://www.nature.com/articles/s41588-025-02265-4#Fig1), Fig. [5a–b](https://www.nature.com/articles/s41588-025-02265-4#Fig5), Fig. [7e–f,h](https://www.nature.com/articles/s41588-025-02265-4#Fig7), Extended Data Fig. [6a,c](https://www.nature.com/articles/s41588-025-02265-4#Fig14), Extended Data Fig. [7b](https://www.nature.com/articles/s41588-025-02265-4#Fig15) and Extended Data Fig. [9j](https://www.nature.com/articles/s41588-025-02265-4#Fig17)). For in vivo experiments, the number of animals was determined based on the variability in tumor take rate and growth and is provided in the respective figure legends. Animals were randomly assigned to treatment groups once tumors reached the mean volume specified in Fig. [7i,k](https://www.nature.com/articles/s41588-025-02265-4#Fig7). No data were excluded from the analyses. Data distributions were assumed to be normal but were not formally tested. Statistical analyses were performed using a two-sided _t_ test.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02265-4#MOESM2) linked to this article.

Data availability
-----------------

All sequencing data (RNA-seq, ATAC–seq, ChIP–seq, Hi-C, reduced representation methylation sequencing, DiMeLo-seq and single-cell sequencing) generated for the study have been deposited in the Gene Expression Omnibus (GEO; [GSE239278](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE239278)) at [https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE239278](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE239278).

The MS proteomic data have been deposited in the ProteomeXchange Consortium via the PRIDE[77](https://www.nature.com/articles/s41588-025-02265-4#ref-CR77 "Perez-Riverol, Y. et al. The PRIDE database and related tools and resources in 2019: improving support for quantification data. Nucleic Acids Res. 47, D442–D450 (2019).") partner repository with the dataset identifiers [PXD061127](http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD061127) and [PXD061080](http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD061080). Source data are provided with this paper. The published RNA-seq, ATAC–seq and H3K27ac ChIP–seq data from LuCaP PDX, referenced in this study, are available in the GEO database under the accessions [GSE126078](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126078), [GSE156292](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156292) and [GSE161948](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161948), respectively. The WGBS and RNA-seq data from human patients, referenced in this study, are available in dbGaP (phs001648 and phs000909.v.p1) and in the GEO database under the accessions [GSE74685](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74685), [GSE126078](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126078), [GSE21034](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21034), [GSE6919](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6919) and [GSE77930](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77930). The scRNA-seq data from human patients, referenced in this study, are from ref. [46](https://www.nature.com/articles/s41588-025-02265-4#ref-CR46 "Cheng, Q. et al. Pre-existing castration-resistant prostate cancer-like cells in primary prostate cancer promote resistance to hormonal therapy. Eur. Urol. 81, 446–455 (2022)."). [Source data](https://www.nature.com/articles/s41588-025-02265-4#Sec35) are provided with this paper.

Code availability
-----------------

The code for NGS analyses performed in this paper has been uploaded to GitHub ([https://github.com/JYULAB/NKX2-1\_NEPC\_project](https://github.com/JYULAB/NKX2-1_NEPC_project)) and Zenodo ([https://doi.org/10.5281/zenodo.15610492](https://doi.org/10.5281/zenodo.15610492))[78](https://www.nature.com/articles/s41588-025-02265-4#ref-CR78 "Zhao, J. & Keo, V. Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer. Zenodo 
https://doi.org/10.5281/zenodo.15610492
(2025).").

References
----------

1.  Abida, W. et al. Genomic correlates of clinical outcome in advanced prostate cancer. _Proc. Natl Acad. Sci. USA_ **116**, 11428–11436 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtV2itLnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31061129)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6561293)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20correlates%20of%20clinical%20outcome%20in%20advanced%20prostate%20cancer&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=116&pages=11428-11436&publication_year=2019&author=Abida%2CW) 
    
2.  Aggarwal, R. et al. Clinical and genomic characterization of treatment-emergent small-cell neuroendocrine prostate cancer: a multi-institutional prospective study. _J. Clin. Oncol._ **36**, 2492–2503 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXivVSrsbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29985747)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6366813)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20and%20genomic%20characterization%20of%20treatment-emergent%20small-cell%20neuroendocrine%20prostate%20cancer%3A%20a%20multi-institutional%20prospective%20study&journal=J.%20Clin.%20Oncol.&volume=36&pages=2492-2503&publication_year=2018&author=Aggarwal%2CR) 
    
3.  Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. _Nat. Med._ **22**, 298–305 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xit1ajtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26855148)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4777652)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Divergent%20clonal%20evolution%20of%20castration-resistant%20neuroendocrine%20prostate%20cancer&journal=Nat.%20Med.&volume=22&pages=298-305&publication_year=2016&author=Beltran%2CH) 
    
4.  Conteduca, V. et al. Clinical features of neuroendocrine prostate cancer. _Eur J. Cancer_ **121**, 7–18 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsleltrnN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31525487)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6803064)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20features%20of%20neuroendocrine%20prostate%20cancer&journal=Eur%20J.%20Cancer&volume=121&pages=7-18&publication_year=2019&author=Conteduca%2CV) 
    
5.  Ku, S. Y. et al. Rb1 and Trp53 cooperate to suppress prostate cancer lineage plasticity, metastasis, and antiandrogen resistance. _Science_ **355**, 78–83 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Wjtw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28059767)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5367887)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rb1%20and%20Trp53%20cooperate%20to%20suppress%20prostate%20cancer%20lineage%20plasticity%2C%20metastasis%2C%20and%20antiandrogen%20resistance&journal=Science&volume=355&pages=78-83&publication_year=2017&author=Ku%2CSY) 
    
6.  Mu, P. et al. SOX2 promotes lineage plasticity and antiandrogen resistance in TP53- and RB1-deficient prostate cancer. _Science_ **355**, 84–88 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1WjtA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28059768)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5247742)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SOX2%20promotes%20lineage%20plasticity%20and%20antiandrogen%20resistance%20in%20TP53-%20and%20RB1-deficient%20prostate%20cancer&journal=Science&volume=355&pages=84-88&publication_year=2017&author=Mu%2CP) 
    
7.  Zou, M. et al. Transdifferentiation as a mechanism of treatment resistance in a mouse model of castration-resistant prostate cancer. _Cancer Discov._ **7**, 736–749 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFCisrbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28411207)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501744)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transdifferentiation%20as%20a%20mechanism%20of%20treatment%20resistance%20in%20a%20mouse%20model%20of%20castration-resistant%20prostate%20cancer&journal=Cancer%20Discov.&volume=7&pages=736-749&publication_year=2017&author=Zou%2CM) 
    
8.  Zhou, Z. et al. Synergy of p53 and Rb deficiency in a conditional mouse model for metastatic prostate cancer. _Cancer Res._ **66**, 7889–7898 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XotFWqsbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16912162)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synergy%20of%20p53%20and%20Rb%20deficiency%20in%20a%20conditional%20mouse%20model%20for%20metastatic%20prostate%20cancer&journal=Cancer%20Res.&volume=66&pages=7889-7898&publication_year=2006&author=Zhou%2CZ) 
    
9.  Park, J. W. et al. Reprogramming normal human epithelial tissues to a common, lethal neuroendocrine cancer lineage. _Science_ **362**, 91–95 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVehs7rP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30287662)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6414229)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reprogramming%20normal%20human%20epithelial%20tissues%20to%20a%20common%2C%20lethal%20neuroendocrine%20cancer%20lineage&journal=Science&volume=362&pages=91-95&publication_year=2018&author=Park%2CJW) 
    
10.  Cejas, P. et al. Subtype heterogeneity and epigenetic convergence in neuroendocrine prostate cancer. _Nat. Commun._ **12**, 5775 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFGqtL%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34599169)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8486778)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Subtype%20heterogeneity%20and%20epigenetic%20convergence%20in%20neuroendocrine%20prostate%20cancer&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Cejas%2CP) 
    
11.  Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. _Nat. Commun._ **12**, 1979 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnvFegu7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33785741)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8010057)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reprogramming%20of%20the%20FOXA1%20cistrome%20in%20treatment-emergent%20neuroendocrine%20prostate%20cancer&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Baca%2CSC) 
    
12.  Kim, J. et al. FOXA1 inhibits prostate cancer neuroendocrine differentiation. _Oncogene_ **36**, 4072–4080 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvFagsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28319070)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5509480)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXA1%20inhibits%20prostate%20cancer%20neuroendocrine%20differentiation&journal=Oncogene&volume=36&pages=4072-4080&publication_year=2017&author=Kim%2CJ) 
    
13.  Dardenne, E. et al. N-Myc induces an EZH2-mediated transcriptional program driving neuroendocrine prostate cancer. _Cancer Cell_ **30**, 563–577 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1GqtLnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27728805)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5540451)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=N-Myc%20induces%20an%20EZH2-mediated%20transcriptional%20program%20driving%20neuroendocrine%20prostate%20cancer&journal=Cancer%20Cell&volume=30&pages=563-577&publication_year=2016&author=Dardenne%2CE) 
    
14.  Lee, J. K. et al. N-Myc drives neuroendocrine prostate cancer initiated from human prostate epithelial cells. _Cancer Cell_ **29**, 536–547 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XltlOrsLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27050099)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4829466)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=N-Myc%20drives%20neuroendocrine%20prostate%20cancer%20initiated%20from%20human%20prostate%20epithelial%20cells&journal=Cancer%20Cell&volume=29&pages=536-547&publication_year=2016&author=Lee%2CJK) 
    
15.  Bishop, J. L. et al. The master neural transcription factor BRN2 is an androgen receptor-suppressed driver of neuroendocrine differentiation in prostate cancer. _Cancer Discov._ **7**, 54–71 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsF2msQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27784708)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20master%20neural%20transcription%20factor%20BRN2%20is%20an%20androgen%20receptor-suppressed%20driver%20of%20neuroendocrine%20differentiation%20in%20prostate%20cancer&journal=Cancer%20Discov.&volume=7&pages=54-71&publication_year=2017&author=Bishop%2CJL) 
    
16.  Guo, H. et al. ONECUT2 is a driver of neuroendocrine prostate cancer. _Nat. Commun._ **10**, 278 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30655535)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6336817)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ONECUT2%20is%20a%20driver%20of%20neuroendocrine%20prostate%20cancer&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Guo%2CH) 
    
17.  Cyrta, J. et al. Role of specialized composition of SWI/SNF complexes in prostate cancer lineage plasticity. _Nat. Commun._ **11**, 5549 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1OjtbfO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33144576)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7642293)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20specialized%20composition%20of%20SWI%2FSNF%20complexes%20in%20prostate%20cancer%20lineage%20plasticity&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Cyrta%2CJ) 
    
18.  Davies, A. et al. An androgen receptor switch underlies lineage infidelity in treatment-resistant prostate cancer. _Nat. Cell Biol._ **23**, 1023–1034 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFKksbfM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34489572)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9012003)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20androgen%20receptor%20switch%20underlies%20lineage%20infidelity%20in%20treatment-resistant%20prostate%20cancer&journal=Nat.%20Cell%20Biol.&volume=23&pages=1023-1034&publication_year=2021&author=Davies%2CA) 
    
19.  Qi, J. et al. Siah2-dependent concerted activity of HIF and FoxA2 regulates formation of neuroendocrine phenotype and neuroendocrine prostate tumors. _Cancer Cell_ **18**, 23–38 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXpslSjt7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20609350)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2919332)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Siah2-dependent%20concerted%20activity%20of%20HIF%20and%20FoxA2%20regulates%20formation%20of%20neuroendocrine%20phenotype%20and%20neuroendocrine%20prostate%20tumors&journal=Cancer%20Cell&volume=18&pages=23-38&publication_year=2010&author=Qi%2CJ) 
    
20.  Han, M. et al. FOXA2 drives lineage plasticity and KIT pathway activation in neuroendocrine prostate cancer. _Cancer Cell_ **40**, 1306–1323 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivVSjsbnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36332622)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXA2%20drives%20lineage%20plasticity%20and%20KIT%20pathway%20activation%20in%20neuroendocrine%20prostate%20cancer&journal=Cancer%20Cell&volume=40&pages=1306-1323&publication_year=2022&author=Han%2CM) 
    
21.  Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. _Science_ **326**, 289–293 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1agsbjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19815776)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2858594)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&journal=Science&volume=326&pages=289-293&publication_year=2009&author=Lieberman-Aiden%2CE) 
    
22.  Dixon, J. R. et al. Topological domains in mammalian genomes identified by analysis of chromatin interactions. _Nature_ **485**, 376–380 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XlslShurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22495300)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3356448)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Topological%20domains%20in%20mammalian%20genomes%20identified%20by%20analysis%20of%20chromatin%20interactions&journal=Nature&volume=485&pages=376-380&publication_year=2012&author=Dixon%2CJR) 
    
23.  Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. _Cell_ **159**, 1665–1680 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtrrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25497547)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%203D%20map%20of%20the%20human%20genome%20at%20kilobase%20resolution%20reveals%20principles%20of%20chromatin%20looping&journal=Cell&volume=159&pages=1665-1680&publication_year=2014&author=Rao%2CSS) 
    
24.  Wang, R. et al. MyoD is a 3D genome structure organizer for muscle cell identity. _Nat. Commun._ **13**, 205 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XoslOiuw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35017543)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8752600)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MyoD%20is%20a%203D%20genome%20structure%20organizer%20for%20muscle%20cell%20identity&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Wang%2CR) 
    
25.  Di Giammartino, D. C. et al. KLF4 is involved in the organization and regulation of pluripotency-associated three-dimensional enhancer networks. _Nat. Cell Biol._ **21**, 1179–1190 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31548608)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7339746)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=KLF4%20is%20involved%20in%20the%20organization%20and%20regulation%20of%20pluripotency-associated%20three-dimensional%20enhancer%20networks&journal=Nat.%20Cell%20Biol.&volume=21&pages=1179-1190&publication_year=2019&author=Giammartino%2CDC) 
    
26.  Stadhouders, R. et al. Transcription factors orchestrate dynamic interplay between genome topology and gene regulation during cell reprogramming. _Nat. Genet._ **50**, 238–249 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkslGitbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29335546)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5810905)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20factors%20orchestrate%20dynamic%20interplay%20between%20genome%20topology%20and%20gene%20regulation%20during%20cell%20reprogramming&journal=Nat.%20Genet.&volume=50&pages=238-249&publication_year=2018&author=Stadhouders%2CR) 
    
27.  Stadhouders, R., Filion, G. J. & Graf, T. Transcription factors and 3D genome conformation in cell-fate decisions. _Nature_ **569**, 345–354 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXpslGlu7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31092938)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20factors%20and%203D%20genome%20conformation%20in%20cell-fate%20decisions&journal=Nature&volume=569&pages=345-354&publication_year=2019&author=Stadhouders%2CR&author=Filion%2CGJ&author=Graf%2CT) 
    
28.  Zhao, S. G. et al. Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer. _Nat. Genet._ **56**, 1689–1700 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFOmt7fE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39020220)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11319208)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20analyses%20highlight%20interactions%20between%20the%20three-dimensional%20genome%20and%20DNA%2C%20RNA%20and%20epigenomic%20alterations%20in%20metastatic%20prostate%20cancer&journal=Nat.%20Genet.&volume=56&pages=1689-1700&publication_year=2024&author=Zhao%2CSG) 
    
29.  Lazzaro, D., Price, M., de Felice, M. & Di Lauro, R. The transcription factor TTF-1 is expressed at the onset of thyroid and lung morphogenesis and in restricted regions of the foetal brain. _Development_ **113**, 1093–1104 (1991).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38XhslSqs7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1811929)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20transcription%20factor%20TTF-1%20is%20expressed%20at%20the%20onset%20of%20thyroid%20and%20lung%20morphogenesis%20and%20in%20restricted%20regions%20of%20the%20foetal%20brain&journal=Development&volume=113&pages=1093-1104&publication_year=1991&author=Lazzaro%2CD&author=Price%2CM&author=Felice%2CM&author=Lauro%2CR) 
    
30.  Minoo, P. et al. TTF-1 regulates lung epithelial morphogenesis. _Dev. Biol._ **172**, 694–698 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XhtVCjuw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8612983)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TTF-1%20regulates%20lung%20epithelial%20morphogenesis&journal=Dev.%20Biol.&volume=172&pages=694-698&publication_year=1995&author=Minoo%2CP) 
    
31.  Minoo, P., Su, G., Drum, H., Bringas, P. & Kimura, S. Defects in tracheoesophageal and lung morphogenesis in _Nkx2.1_(−/−) mouse embryos. _Dev. Biol._ **209**, 60–71 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXisVaisrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10208743)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defects%20in%20tracheoesophageal%20and%20lung%20morphogenesis%20in%20Nkx2.1%28%E2%88%92%2F%E2%88%92%29%20mouse%20embryos&journal=Dev.%20Biol.&volume=209&pages=60-71&publication_year=1999&author=Minoo%2CP&author=Su%2CG&author=Drum%2CH&author=Bringas%2CP&author=Kimura%2CS) 
    
32.  Butt, S. J. et al. The requirement of _Nkx2-1_ in the temporal specification of cortical interneuron subtypes. _Neuron_ **59**, 722–732 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtFKhtbjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18786356)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2562525)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20requirement%20of%20Nkx2-1%20in%20the%20temporal%20specification%20of%20cortical%20interneuron%20subtypes&journal=Neuron&volume=59&pages=722-732&publication_year=2008&author=Butt%2CSJ) 
    
33.  Nobrega-Pereira, S. et al. Postmitotic Nkx2-1 controls the migration of telencephalic interneurons by direct repression of guidance receptors. _Neuron_ **59**, 733–745 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtFKhtbjF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18786357)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2643060)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Postmitotic%20Nkx2-1%20controls%20the%20migration%20of%20telencephalic%20interneurons%20by%20direct%20repression%20of%20guidance%20receptors&journal=Neuron&volume=59&pages=733-745&publication_year=2008&author=Nobrega-Pereira%2CS) 
    
34.  Snyder, E. L. et al. Nkx2-1 represses a latent gastric differentiation program in lung adenocarcinoma. _Mol. Cell_ **50**, 185–199 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXksFOmtrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23523371)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3721642)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nkx2-1%20represses%20a%20latent%20gastric%20differentiation%20program%20in%20lung%20adenocarcinoma&journal=Mol.%20Cell&volume=50&pages=185-199&publication_year=2013&author=Snyder%2CEL) 
    
35.  Camolotto, S. A. et al. FoxA1and FoxA2 drive gastric differentiation and suppress squamous identity in NKX2-1-negative lung cancer. _eLife_ **7**, e38579 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30475207)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6303105)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FoxA1and%20FoxA2%20drive%20gastric%20differentiation%20and%20suppress%20squamous%20identity%20in%20NKX2-1-negative%20lung%20cancer&journal=eLife&volume=7&publication_year=2018&author=Camolotto%2CSA) 
    
36.  Orstad, G. et al. FoxA1 and FoxA2 control growth and cellular identity in NKX2-1-positive lung adenocarcinoma. _Dev. Cell_ **57**, 1866–1882 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVSgt77N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35835117)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9378547)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FoxA1%20and%20FoxA2%20control%20growth%20and%20cellular%20identity%20in%20NKX2-1-positive%20lung%20adenocarcinoma&journal=Dev.%20Cell&volume=57&pages=1866-1882&publication_year=2022&author=Orstad%2CG) 
    
37.  Wang, W. & Epstein, J. I. Small cell carcinoma of the prostate. A morphologic and immunohistochemical study of 95 cases. _Am. J. Surg. Pathol._ **32**, 65–71 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18162772)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Small%20cell%20carcinoma%20of%20the%20prostate.%20A%20morphologic%20and%20immunohistochemical%20study%20of%2095%20cases&journal=Am.%20J.%20Surg.%20Pathol.&volume=32&pages=65-71&publication_year=2008&author=Wang%2CW&author=Epstein%2CJI) 
    
38.  Bolis, M. et al. Dynamic prostate cancer transcriptome analysis delineates the trajectory to disease progression. _Nat. Commun._ **12**, 7033 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1Gjt7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34857732)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8640014)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dynamic%20prostate%20cancer%20transcriptome%20analysis%20delineates%20the%20trajectory%20to%20disease%20progression&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Bolis%2CM) 
    
39.  Roayaei Ardakany, A., Gezer, H. T., Lonardi, S. & Ay, F. Mustache: multi-scale detection of chromatin loops from Hi-C and Micro-C maps using scale-space representation. _Genome Biol._ **21**, 256 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32998764)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7528378)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mustache%3A%20multi-scale%20detection%20of%20chromatin%20loops%20from%20Hi-C%20and%20Micro-C%20maps%20using%20scale-space%20representation&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Roayaei%20Ardakany%2CA&author=Gezer%2CHT&author=Lonardi%2CS&author=Ay%2CF) 
    
40.  Labrecque, M. P. et al. Molecular profiling stratifies diverse phenotypes of treatment-refractory metastatic castration-resistant prostate cancer. _J. Clin. Invest._ **129**, 4492–4505 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31361600)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6763249)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20profiling%20stratifies%20diverse%20phenotypes%20of%20treatment-refractory%20metastatic%20castration-resistant%20prostate%20cancer&journal=J.%20Clin.%20Invest.&volume=129&pages=4492-4505&publication_year=2019&author=Labrecque%2CMP) 
    
41.  Nouruzi, S. et al. ASCL1 activates neuronal stem cell-like lineage programming through remodeling of the chromatin landscape in prostate cancer. _Nat. Commun._ **13**, 2282 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFGgtb%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35477723)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9046280)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ASCL1%20activates%20neuronal%20stem%20cell-like%20lineage%20programming%20through%20remodeling%20of%20the%20chromatin%20landscape%20in%20prostate%20cancer&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Nouruzi%2CS) 
    
42.  Lu, X. et al. HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer. _Nat. Genet._ **54**, 670–683 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFWitb%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35468964)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9117466)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HOXB13%20suppresses%20de%20novo%20lipogenesis%20through%20HDAC3-mediated%20epigenetic%20reprogramming%20in%20prostate%20cancer&journal=Nat.%20Genet.&volume=54&pages=670-683&publication_year=2022&author=Lu%2CX) 
    
43.  Takeda, D. Y. et al. A somatically acquired enhancer of the androgen receptor is a noncoding driver in advanced prostate cancer. _Cell_ **174**, 422–432 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFCqtL3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29909987)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6046260)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20somatically%20acquired%20enhancer%20of%20the%20androgen%20receptor%20is%20a%20noncoding%20driver%20in%20advanced%20prostate%20cancer&journal=Cell&volume=174&pages=422-432&publication_year=2018&author=Takeda%2CDY) 
    
44.  Mirosevich, J. et al. Expression and role of Foxa proteins in prostate cancer. _Prostate_ **66**, 1013–1028 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xnt1Cnt7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16001449)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Expression%20and%20role%20of%20Foxa%20proteins%20in%20prostate%20cancer&journal=Prostate&volume=66&pages=1013-1028&publication_year=2006&author=Mirosevich%2CJ) 
    
45.  Park, J. W., Lee, J. K., Witte, O. N. & Huang, J. FOXA2 is a sensitive and specific marker for small cell neuroendocrine carcinoma of the prostate. _Mod. Pathol._ **30**, 1262–1272 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVCiu77L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28621319)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6330177)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXA2%20is%20a%20sensitive%20and%20specific%20marker%20for%20small%20cell%20neuroendocrine%20carcinoma%20of%20the%20prostate&journal=Mod.%20Pathol.&volume=30&pages=1262-1272&publication_year=2017&author=Park%2CJW&author=Lee%2CJK&author=Witte%2CON&author=Huang%2CJ) 
    
46.  Cheng, Q. et al. Pre-existing castration-resistant prostate cancer-like cells in primary prostate cancer promote resistance to hormonal therapy. _Eur. Urol._ **81**, 446–455 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtVKqt7%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35058087)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9018600)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pre-existing%20castration-resistant%20prostate%20cancer-like%20cells%20in%20primary%20prostate%20cancer%20promote%20resistance%20to%20hormonal%20therapy&journal=Eur.%20Urol.&volume=81&pages=446-455&publication_year=2022&author=Cheng%2CQ) 
    
47.  Qin, J. et al. The PSA−/lo prostate cancer cell population harbors self-renewing long-term tumor-propagating cells that resist castration. _Cell Stem Cell_ **10**, 556–569 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xms12ms70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22560078)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3348510)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20PSA%E2%88%92%2Flo%20prostate%20cancer%20cell%20population%20harbors%20self-renewing%20long-term%20tumor-propagating%20cells%20that%20resist%20castration&journal=Cell%20Stem%20Cell&volume=10&pages=556-569&publication_year=2012&author=Qin%2CJ) 
    
48.  Horning, A. M. et al. Single-cell RNA-seq reveals a subpopulation of prostate cancer cells with enhanced cell-cycle-related transcription and attenuated androgen response. _Cancer Res._ **78**, 853–864 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisl2gt7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29233929)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20RNA-seq%20reveals%20a%20subpopulation%20of%20prostate%20cancer%20cells%20with%20enhanced%20cell-cycle-related%20transcription%20and%20attenuated%20androgen%20response&journal=Cancer%20Res.&volume=78&pages=853-864&publication_year=2018&author=Horning%2CAM) 
    
49.  Minussi, D. C. et al. Breast tumours maintain a reservoir of subclonal diversity during expansion. _Nature_ **592**, 302–308 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnt1equr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33762732)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8049101)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Breast%20tumours%20maintain%20a%20reservoir%20of%20subclonal%20diversity%20during%20expansion&journal=Nature&volume=592&pages=302-308&publication_year=2021&author=Minussi%2CDC) 
    
50.  Wang, A. et al. Epigenetic priming of enhancers predicts developmental competence of hESC-derived endodermal lineage intermediates. _Cell Stem Cell_ **16**, 386–399 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsVKqt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25842977)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4478079)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenetic%20priming%20of%20enhancers%20predicts%20developmental%20competence%20of%20hESC-derived%20endodermal%20lineage%20intermediates&journal=Cell%20Stem%20Cell&volume=16&pages=386-399&publication_year=2015&author=Wang%2CA) 
    
51.  Haider, M. et al. Epithelial mesenchymal-like transition occurs in a subset of cells in castration resistant prostate cancer bone metastases. _Clin. Exp. Metastasis_ **33**, 239–248 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitVGrsbvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26667932)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epithelial%20mesenchymal-like%20transition%20occurs%20in%20a%20subset%20of%20cells%20in%20castration%20resistant%20prostate%20cancer%20bone%20metastases&journal=Clin.%20Exp.%20Metastasis&volume=33&pages=239-248&publication_year=2016&author=Haider%2CM) 
    
52.  Altemose, N. et al. DiMeLo-seq: a long-read, single-molecule method for mapping protein–DNA interactions genome wide. _Nat. Methods_ **19**, 711–723 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVWhsr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35396487)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9189060)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DiMeLo-seq%3A%20a%20long-read%2C%20single-molecule%20method%20for%20mapping%20protein%E2%80%93DNA%20interactions%20genome%20wide&journal=Nat.%20Methods&volume=19&pages=711-723&publication_year=2022&author=Altemose%2CN) 
    
53.  Welti, J. et al. Targeting the p300/CBP axis in lethal prostate cancer. _Cancer Discov_ **11**, 1118–1137 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVeksrrF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33431496)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8102310)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20the%20p300%2FCBP%20axis%20in%20lethal%20prostate%20cancer&journal=Cancer%20Discov&volume=11&pages=1118-1137&publication_year=2021&author=Welti%2CJ) 
    
54.  Loven, J. et al. Selective inhibition of tumor oncogenes by disruption of super-enhancers. _Cell_ **153**, 320–334 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXlvVCgsrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23582323)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3760967)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Selective%20inhibition%20of%20tumor%20oncogenes%20by%20disruption%20of%20super-enhancers&journal=Cell&volume=153&pages=320-334&publication_year=2013&author=Loven%2CJ) 
    
55.  Whyte, W. A. et al. Master transcription factors and mediator establish super-enhancers at key cell identity genes. _Cell_ **153**, 307–319 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXlvVCrtbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23582322)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3653129)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Master%20transcription%20factors%20and%20mediator%20establish%20super-enhancers%20at%20key%20cell%20identity%20genes&journal=Cell&volume=153&pages=307-319&publication_year=2013&author=Whyte%2CWA) 
    
56.  Ma, L. et al. Co-condensation between transcription factor and coactivator p300 modulates transcriptional bursting kinetics. _Mol. Cell_ **81**, 1682–1697 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlslShsL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33651988)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Co-condensation%20between%20transcription%20factor%20and%20coactivator%20p300%20modulates%20transcriptional%20bursting%20kinetics&journal=Mol.%20Cell&volume=81&pages=1682-1697&publication_year=2021&author=Ma%2CL) 
    
57.  Ibrahim, Z. et al. Structural insights into p300 regulation and acetylation-dependent genome organisation. _Nat. Commun._ **13**, 7759 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFartb%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36522330)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9755262)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20insights%20into%20p300%20regulation%20and%20acetylation-dependent%20genome%20organisation&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Ibrahim%2CZ) 
    
58.  Li, J. et al. TET1 dioxygenase is required for FOXA2-associated chromatin remodeling in pancreatic β-cell differentiation. _Nat. Commun._ **13**, 3907 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVChsb%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35798741)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9263144)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TET1%20dioxygenase%20is%20required%20for%20FOXA2-associated%20chromatin%20remodeling%20in%20pancreatic%20%CE%B2-cell%20differentiation&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Li%2CJ) 
    
59.  Reizel, Y. et al. FoxA-dependent demethylation of DNA initiates epigenetic memory of cellular identity. _Dev. Cell_ **56**, 602–612 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlsFSitL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33636105)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8129911)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FoxA-dependent%20demethylation%20of%20DNA%20initiates%20epigenetic%20memory%20of%20cellular%20identity&journal=Dev.%20Cell&volume=56&pages=602-612&publication_year=2021&author=Reizel%2CY) 
    
60.  Jozwik, K. M., Chernukhin, I., Serandour, A. A., Nagarajan, S. & Carroll, J. S. FOXA1 directs H3K4 monomethylation at enhancers via recruitment of the methyltransferase MLL3. _Cell Rep._ **17**, 2715–2723 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitVegu7jE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27926873)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5177601)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXA1%20directs%20H3K4%20monomethylation%20at%20enhancers%20via%20recruitment%20of%20the%20methyltransferase%20MLL3&journal=Cell%20Rep.&volume=17&pages=2715-2723&publication_year=2016&author=Jozwik%2CKM&author=Chernukhin%2CI&author=Serandour%2CAA&author=Nagarajan%2CS&author=Carroll%2CJS) 
    
61.  Donaghey, J. et al. Genetic determinants and epigenetic effects of pioneer-factor occupancy. _Nat. Genet._ **50**, 250–258 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkslGiurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29358654)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6517675)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20determinants%20and%20epigenetic%20effects%20of%20pioneer-factor%20occupancy&journal=Nat.%20Genet.&volume=50&pages=250-258&publication_year=2018&author=Donaghey%2CJ) 
    
62.  Cernilogar, F. M. et al. Pre-marked chromatin and transcription factor co-binding shape the pioneering activity of Foxa2. _Nucleic Acids Res._ **47**, 9069–9086 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVWksr7M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31350899)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6753583)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pre-marked%20chromatin%20and%20transcription%20factor%20co-binding%20shape%20the%20pioneering%20activity%20of%20Foxa2&journal=Nucleic%20Acids%20Res.&volume=47&pages=9069-9086&publication_year=2019&author=Cernilogar%2CFM) 
    
63.  Lee, K. et al. FOXA2 is required for enhancer priming during pancreatic differentiation. _Cell Rep._ **28**, 382–393 e387 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlGjtrjP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31291575)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6636862)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXA2%20is%20required%20for%20enhancer%20priming%20during%20pancreatic%20differentiation&journal=Cell%20Rep.&volume=28&pages=382-393%20e387&publication_year=2019&author=Lee%2CK) 
    
64.  Zaret, K. S. Pioneer transcription factors initiating gene network changes. _Annu. Rev. Genet._ **54**, 367–385 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslCktrrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32886547)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7900943)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pioneer%20transcription%20factors%20initiating%20gene%20network%20changes&journal=Annu.%20Rev.%20Genet.&volume=54&pages=367-385&publication_year=2020&author=Zaret%2CKS) 
    
65.  Yang, Y. A. et al. FOXA1 potentiates lineage-specific enhancer activation through modulating TET1 expression and function. _Nucleic Acids Res._ **44**, 8153–8164 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvFCmu7fN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27257062)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5041455)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXA1%20potentiates%20lineage-specific%20enhancer%20activation%20through%20modulating%20TET1%20expression%20and%20function&journal=Nucleic%20Acids%20Res.&volume=44&pages=8153-8164&publication_year=2016&author=Yang%2CYA) 
    
66.  Geusz, R. J. et al. Sequence logic at enhancers governs a dual mechanism of endodermal organ fate induction by FOXA pioneer factors. _Nat. Commun._ **12**, 6636 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFWksLvM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34789735)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8599738)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sequence%20logic%20at%20enhancers%20governs%20a%20dual%20mechanism%20of%20endodermal%20organ%20fate%20induction%20by%20FOXA%20pioneer%20factors&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Geusz%2CRJ) 
    
67.  Xiao, L. et al. Targeting SWI/SNF ATPases in enhancer-addicted prostate cancer. _Nature_ **601**, 434–439 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXivValtr7L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34937944)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20SWI%2FSNF%20ATPases%20in%20enhancer-addicted%20prostate%20cancer&journal=Nature&volume=601&pages=434-439&publication_year=2022&author=Xiao%2CL) 
    
68.  Nicosia, L. et al. Therapeutic targeting of EP300/CBP by bromodomain inhibition in hematologic malignancies. _Cancer Cell_ **41**, 2136–2153 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisVertbfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37995682)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Therapeutic%20targeting%20of%20EP300%2FCBP%20by%20bromodomain%20inhibition%20in%20hematologic%20malignancies&journal=Cancer%20Cell&volume=41&pages=2136-2153&publication_year=2023&author=Nicosia%2CL) 
    
69.  Xu, B. et al. Altered chromatin recruitment by FOXA1 mutations promotes androgen independence and prostate cancer progression. _Cell Res._ **29**, 773–775 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31324884)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6796844)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Altered%20chromatin%20recruitment%20by%20FOXA1%20mutations%20promotes%20androgen%20independence%20and%20prostate%20cancer%20progression&journal=Cell%20Res.&volume=29&pages=773-775&publication_year=2019&author=Xu%2CB) 
    
70.  Drost, J. et al. Organoid culture systems for prostate epithelial and cancer tissue. _Nat. Protoc._ **11**, 347–358 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xos1Ortg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26797458)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4793718)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Organoid%20culture%20systems%20for%20prostate%20epithelial%20and%20cancer%20tissue&journal=Nat.%20Protoc.&volume=11&pages=347-358&publication_year=2016&author=Drost%2CJ) 
    
71.  Aguilan, J. T., Kulej, K. & Sidoli, S. Guide for protein fold change and p-value calculation for non-experts in proteomics. _Mol. Omics_ **16**, 573–582 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVCqu7jN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32968743)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Guide%20for%20protein%20fold%20change%20and%20p-value%20calculation%20for%20non-experts%20in%20proteomics&journal=Mol.%20Omics&volume=16&pages=573-582&publication_year=2020&author=Aguilan%2CJT&author=Kulej%2CK&author=Sidoli%2CS) 
    
72.  Hanzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. _BMC Bioinform_ **14**, 7 (2013).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GSVA%3A%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-seq%20data&journal=BMC%20Bioinform&volume=14&publication_year=2013&author=Hanzelmann%2CS&author=Castelo%2CR&author=Guinney%2CJ) 
    
73.  Gao, R. et al. Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes. _Nat. Biotechnol._ **39**, 599–608 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFOitb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33462507)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8122019)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Delineating%20copy%20number%20and%20clonal%20substructure%20in%20human%20tumors%20from%20single-cell%20transcriptomes&journal=Nat.%20Biotechnol.&volume=39&pages=599-608&publication_year=2021&author=Gao%2CR) 
    
74.  Shiau, C. K. et al. High throughput single cell long-read sequencing analyses of same-cell genotypes and phenotypes in human tumors. _Nat. Commun._ **14**, 4124 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVKms7nK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37433798)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10336110)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High%20throughput%20single%20cell%20long-read%20sequencing%20analyses%20of%20same-cell%20genotypes%20and%20phenotypes%20in%20human%20tumors&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Shiau%2CCK) 
    
75.  Lebrigand, K., Magnone, V., Barbry, P. & Waldmann, R. High throughput error corrected Nanopore single cell transcriptome sequencing. _Nat. Commun._ **11**, 4025 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1WrtrrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32788667)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7423900)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High%20throughput%20error%20corrected%20Nanopore%20single%20cell%20transcriptome%20sequencing&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Lebrigand%2CK&author=Magnone%2CV&author=Barbry%2CP&author=Waldmann%2CR) 
    
76.  Nguyen, H. M. et al. LuCaP prostate cancer patient-derived xenografts reflect the molecular heterogeneity of advanced disease and serve as models for evaluating cancer therapeutics. _Prostate_ **77**, 654–671 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXksVSktLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28156002)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5354949)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LuCaP%20prostate%20cancer%20patient-derived%20xenografts%20reflect%20the%20molecular%20heterogeneity%20of%20advanced%20disease%20and%20serve%20as%20models%20for%20evaluating%20cancer%20therapeutics&journal=Prostate&volume=77&pages=654-671&publication_year=2017&author=Nguyen%2CHM) 
    
77.  Perez-Riverol, Y. et al. The PRIDE database and related tools and resources in 2019: improving support for quantification data. _Nucleic Acids Res._ **47**, D442–D450 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1GqtrzK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30395289)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20PRIDE%20database%20and%20related%20tools%20and%20resources%20in%202019%3A%20improving%20support%20for%20quantification%20data&journal=Nucleic%20Acids%20Res.&volume=47&pages=D442-D450&publication_year=2019&author=Perez-Riverol%2CY) 
    
78.  Zhao, J. & Keo, V. Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer. _Zenodo_ [https://doi.org/10.5281/zenodo.15610492](https://doi.org/10.5281/zenodo.15610492) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02265-4?format=refman&flavour=references)

Acknowledgements
----------------

This work was partially supported by the Northwestern University Pathology Core Facility, the Center for Advanced Microscopy/Nikon Imaging Center and Emory University’s Integrated Cellular Imaging Core and Integrated Genomics Core. Next-generation sequencing was done at the University of Chicago Genomics Facility, Northwestern Sequencing Core, Admera Health and Florida State University College of Medicine. IP–MS was done at the Taplin Mass Spectrometry Facility of Harvard Medical School. We thank members of the Yu Lab for helpful discussions. We thank E. Corey (University of Washington) for generating and maintaining the LuCaP PDX models, which were partially funded by NIH (awards P50CA97186 and PO1CA163227), and for sharing them with us. The CRPC TMA was graciously provided by C. Morrissey (Department of Urology, University of Washington School of Medicine, Seattle, WA, USA) through the Prostate Cancer Biorepository Network. We thank J. Huang (Department of Pathology, Duke University School of Medicine, Durham, NC, USA) for sharing their scRNA-seq data of clinical PCa samples. We thank A. Zoubeidi (Department of Urologic Sciences, University of British Columbia, Vancouver, BC) for sharing the ENZ-resistant AR+/PSA− cell line 42D. This work was supported by the National Institutes of Health (NIH)–National Cancer Institute (NCI) (training grant T32CA09560 to G.G.), NIH (grants R35GM142539 to R.G., R01CA257446 to J.Y., R50CA211271 to J.C.Z., U01CA113913 to M.G.S. and R01CA286147 to J.C.Z. and J.Y.), the Prostate Cancer SPORE (grant P50CA180995 to J.Y. and X.Y.), the Department of Defense (grants PC210266 to J.Y., and PC220151 and PC220158 to J.C.Z.) and the Prostate Cancer Foundation (2017CHAL2008 to J.Y. and J.C.Z.).

Author information
------------------

Author notes

1.  These authors contributed equally: Xiaodong Lu, Viriya Keo.
    

### Authors and Affiliations

1.  Department of Urology, Emory University School of Medicine, Atlanta, GA, USA
    
    Xiaodong Lu, Viriya Keo, Wanqing Xie, Galina Gritsina, Martin G. Sanda & Jindan Yu
    
2.  Division of Hematology and Oncology, Department of Medicine, Northwestern University Feinberg School of Medicine, Chicago, IL, USA
    
    Viriya Keo, Irina Cheng & Galina Gritsina
    
3.  Department of Biochemistry and Molecular Genetics, Northwestern University Feinberg School of Medicine, Chicago, IL, USA
    
    Juan Wang, Lina Lu, Cheng-Kai Shiau, Yueying He, Qiushi Jin & Ruli Gao
    
4.  Department of Human Genetics, Emory University School of Medicine, Atlanta, GA, USA
    
    Peng Jin, Victor G. Corces, Jonathan C. Zhao & Jindan Yu
    
5.  Winship Cancer Institute, Emory University School of Medicine, Atlanta, GA, USA
    
    Martin G. Sanda, Jonathan C. Zhao & Jindan Yu
    
6.  Department of Genetics, Stanford University, Stanford, CA, USA
    
    Nicolas Altemose
    

Authors

1.  Xiaodong Lu
    
    [View author publications](https://www.nature.com/search?author=Xiaodong%20Lu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiaodong%20Lu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiaodong%20Lu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Viriya Keo
    
    [View author publications](https://www.nature.com/search?author=Viriya%20Keo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Viriya%20Keo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Viriya%20Keo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Irina Cheng
    
    [View author publications](https://www.nature.com/search?author=Irina%20Cheng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Irina%20Cheng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Irina%20Cheng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Wanqing Xie
    
    [View author publications](https://www.nature.com/search?author=Wanqing%20Xie)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wanqing%20Xie) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wanqing%20Xie%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Galina Gritsina
    
    [View author publications](https://www.nature.com/search?author=Galina%20Gritsina)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Galina%20Gritsina) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Galina%20Gritsina%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Juan Wang
    
    [View author publications](https://www.nature.com/search?author=Juan%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Juan%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Juan%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Lina Lu
    
    [View author publications](https://www.nature.com/search?author=Lina%20Lu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lina%20Lu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lina%20Lu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Cheng-Kai Shiau
    
    [View author publications](https://www.nature.com/search?author=Cheng-Kai%20Shiau)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cheng-Kai%20Shiau) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cheng-Kai%20Shiau%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Yueying He
    
    [View author publications](https://www.nature.com/search?author=Yueying%20He)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yueying%20He) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yueying%20He%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Qiushi Jin
    
    [View author publications](https://www.nature.com/search?author=Qiushi%20Jin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qiushi%20Jin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qiushi%20Jin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Peng Jin
    
    [View author publications](https://www.nature.com/search?author=Peng%20Jin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peng%20Jin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peng%20Jin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Martin G. Sanda
    
    [View author publications](https://www.nature.com/search?author=Martin%20G.%20Sanda)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Martin%20G.%20Sanda) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Martin%20G.%20Sanda%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Victor G. Corces
    
    [View author publications](https://www.nature.com/search?author=Victor%20G.%20Corces)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Victor%20G.%20Corces) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Victor%20G.%20Corces%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Nicolas Altemose
    
    [View author publications](https://www.nature.com/search?author=Nicolas%20Altemose)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nicolas%20Altemose) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nicolas%20Altemose%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Ruli Gao
    
    [View author publications](https://www.nature.com/search?author=Ruli%20Gao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ruli%20Gao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ruli%20Gao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Jonathan C. Zhao
    
    [View author publications](https://www.nature.com/search?author=Jonathan%20C.%20Zhao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jonathan%20C.%20Zhao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jonathan%20C.%20Zhao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Jindan Yu
    
    [View author publications](https://www.nature.com/search?author=Jindan%20Yu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jindan%20Yu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jindan%20Yu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

J.Y., J.C.Z. and X.L. conceived the project and designed the experiments. J.C.Z., V.K., I.C., J.W., L.L., C.S., Y.H., R.G. and J.Y. conducted bioinformatic and statistical analyses. G.G. and W.X. assisted with mouse experiments. X.L. and W.X. performed H&E and IHC experiments. Q.J. performed LNCaP+FOXA2 D14 and D28 Hi-C experiments. X.L. performed the rest of the experiments. N.A. provided pA-Hia5 and consulted on DiMeLo-seq experiments. M.S., P.J. and V.C. provided valuable comments on the project. X.L. V.K., J.Z. and J.Y. wrote the paper.

### Corresponding authors

Correspondence to [Jonathan C. Zhao](mailto:changsheng.zhao@emory.edu) or [Jindan Yu](mailto:jindan.yu@emory.edu).

Ethics declarations
-------------------

### Competing interests

All authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02265-4#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Distinct 3D chromatin architecture in NEPC compared to CRPC tumors.](https://www.nature.com/articles/s41588-025-02265-4/figures/9)

**a**, Unsupervised hierarchical clustering of Hi-C samples using the top 25% most variable PC1 of Hi-C matrices. CRPC PDXs: 35CR, 70CR, 77CR, 147. NEPC PDX: 93, 145.1 and 145.2. NEPC cell line: NCI-H660. Androgen-dependent PCa cell line: LNCaP. **b**,**c**, RNA-seq data of _HOXB_ genes (**b**) and _AR_ (**c**) in the indicated models. RNA-seq data for NCI-H660 were generated in-house, and RNA-seq data for the rest models were obtained from [GSE126078](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126078). RNA-seq was performed in triplicate for 35CR and NCI-H660, and in duplicate for the other models. **d**,**e**, Hi-C contact maps (5 kb resolution) of _ASCL1_ (**d**) and _INSM1_(**e**) locus in CRPC (top row) and NEPC (bottom row) models. Gene boxes and 1D ChIP–seq tracks of matched models are aligned at the bottom. Blue arrows indicate NEPC-enriched loops. The scale bars indicate ICE-normalized contact frequency. **f**,**g**, RNA-seq data of _ASCL1_ (**f**) and _INSM1_ (**g**) in the indicated models. The data source is the same as in **b**,**c**.

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM8)

### [Extended Data Fig. 2 Remarkable 3D chromatin reorganization during NE transformation of isogenic PCa cells.](https://www.nature.com/articles/s41588-025-02265-4/figures/10)

**a**, Representative brightfield images showing the morphology of FOXA2-OE cells at days 7, 14, 21 and 28. LNCaP cells were infected with control (plv) or FOXA2 virus and followed up for 28 days. Cells were imaged every week. Scale bar: 100 μm. **b**, FOXA2 OE induced NET in 22Rv1 (left) and DU145 (right) cells. 22Rv1 or DU145 cells were infected with control (plv) or FOXA2 virus, and 22Rv1 cells were collected for WB analysis at week 4, whereas DU145 cells were collected at day 7. The data shown for 22Rv1 are from one of two independent experiments (n = 2), and those for DU145 are from one of three independent experiments (n = 3). **c**, LNCaP cells were co-infected with sg_RB1_&_TP53_ virus along with either sg_NC_ or sg_FOXA2_ virus for 4 weeks before WB. The data shown are from one of two independent experiments (n = 2). **d**, Tumor growth curve of control (plv, n = 4 mice) and FOXA2-OE (n = 6 mice) LNCaP xenograft tumors. Data are mean ± s.d., and _p_ value by two-sided _t_ test. **e**, Hematoxylin and Eosin (H&E) of control (plv) and FOXA2-OE LNCaP xenograft tumors. Representative H&E images (×4) of FOXA2-OE tumors (n = 4) are shown on the left. The circled region indicates the tumor region with small-cell carcinoma-like features, and the high magnification images of small-cell carcinoma and adenocarcinoma tumors (×40) are shown in the middle. Scale bar, 30 µm. The percentage of small-cell carcinoma and adenocarcinoma tumors in the control and FOXA2-OE groups is quantified on the right. **f**, IHC of FOXA2 and SYP proteins in control (plv) and FOXA2-OE LNCaP xenograft tumors. Representative IHC images of FOXA2+/SYP+ and FOXA2+/SYP− tumors in the FOXA2-OE group (n = 4) are shown on the left, and the quantification of them is shown on the right. Scale bar, 30 µm. Insets are shown at higher magnification. **g**, Unsupervised hierarchical clustering of time-course LNCaP+FOXA2 samples, CRPC and NEPC (in red font) based on the top 25% most variable PC1 of Hi-C matrices.

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM8)

### [Extended Data Fig. 3 Single-cell multiome analyses identified transitioning individual cells with intermediate transcriptome and chromatin states.](https://www.nature.com/articles/s41588-025-02265-4/figures/11)

**a**, PCA analyses of chromatin accessibility of LNCaP+FOXA2 cells, along with LuCaP PDX, Merkel cell carcinoma (MCC) and NEPC cell lines (NCI-H660 and EF1-CFCE) using differentially accessible peaks (n = 368,330) identified by DESeq2 (adjusted _p_ < 0.05). Adjusted _p_ values were calculated by a likelihood ratio test with Benjamini–Hochberg correction. **b**, Genome browser views of ATAC–seq signal at NE lineage genes (_POU3F2_ and _NCAM1_, top row) and luminal lineage genes (_KLK3_ and _HOXB13_, bottom row) in time-course LNCaP+FOXA2 cells. **c**, TF motifs were plotted by ranks generated from their associated differential _p_ values at ATAC–seq peaks in D2 and D28 LNCaP+FOXA2 samples. _P_ values by two-sided hypergeometric test with Benjamini–Hochberg correction. **d**, UMAP integration of D0, D14 and D21 LNCaP+FOXA2 cells with previously published clinical PCa cells. **e**, Pseudotime trajectories in PCa cells shown in **d**. Color reflects pseudotime distance (primary PCa located at time = 0). **f**, UMAP visualization of AR and AR target genes (_KLK3_, _NKX3-1_) and NE lineage genes (_SYP_, _NCAM1_, _POU3F2_) in D0, D14 and D21 LNCaP+FOXA2 scRNA-seq samples. Color bars: normalized expression.

### [Extended Data Fig. 4 Clonality analyses of single LNCaP+FOXA2 cells support a mixed model of clonal transformation and expansion during NET.](https://www.nature.com/articles/s41588-025-02265-4/figures/12)

**a**, Heatmap showing two clones delineated by single-cell CNV profiles inferred from scRNA-seq data by CopyKAT in FOXA2-driven NET model. Columns are gene windows ordered by genomic positions, while rows are individual cells that have been clustered using hierarchical clustering. Red color in the heatmap indicates amplification, and blue color indicates deletion. Color bar indicates log copy number ratios of each segment. **b**, scRNA-seq UMAP visualization of D0, D14 and D21 LNCaP+FOXA2 cells indicating clone 1 and clone 2 D0, D14 and D21 LNCaP+FOXA2 cells. **c**, Violin plots showing AR and NE signature scores of clone 1 and clone 2 in D0, D14 and D21 LNCaP+FOXA2 scRNA-seq samples. **d**, Heatmap showing two clones separated by SNVs (n = 1,093) inferred from scNanoRNA-seq data of D0 and D28 LNCaP+FOXA2 cells, clustered by _k_\-means. Each row is one SNV, and each column is a cell. Variant allele frequency (VAF) is calculated as alternate allele read counts divided by all read counts for that nucleotide such as red represents mutant and blue for WT. **e**, UMAP visualization of scNanoRNA-seq data of D0 and D28 LNCaP+FOXA2 cells and their clonality. **f**, UMAP visualization of scNanoRNA-seq data of D0 and D28 LNCaP+FOXA2 cells and their expression of AR and NE signature genes. **g**, Violin plots showing AR and NE signature scores of clone 1 and clone 2 in D0 and D28 LNCaP+FOXA2 cells based on scNanoRNA-seq data. **h**, Heatmap showing germline mutations in D0 and/or D28 LNCaP+FOXA2 cells based on scNanoRNA-seq data. Each row represents one germline mutation, and each column is a cell. WT represents SNVs with supporting reads for the wildtype allele only, MUT are those that have at least 1 supporting read for the alternative allele and NA are those with 0 read coverage. Germline mutations were separately identified for each time point, with their mutation status plotted across the other time point without pre-filtering, wherein they might have 0 (showing as NA) or insufficient read coverage to capture the mutant allele (appearing as WT). Violin plots in **c** and **g** show median values (black line), interquartile range (IQR, black box) and the range of all data points within 1.5× IQR (whiskers).

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM9)

### [Extended Data Fig. 5 NKX2-1 is induced by FOXA2 and required for FOXA2-driven NET.](https://www.nature.com/articles/s41588-025-02265-4/figures/13)

**a**, RT–PCR analysis of _NKX2-1_ mRNA in time-course LNCaP+FOXA2 cells and LuNE cells. Data were normalized to GAPDH. Shown are the mean ± s.e.m. of technical replicates from one of three (n = 3) independent experiments. _P_ values by two-sided _t_ test. **b**, Genome browser view of FOXA2 ChIP–seq signal around the _NKX2-1_ gene in NEPC cells (NCI-H660, LuCaP145.2). **c**,**d**, Hi-C contact maps (5 kb resolution) of _NKX2-1_ gene region in time-course LNCaP+FOXA2 cells (**c**), NCI-H660 and LuCaP PDX (**d**). The mRNA expression of _NKX2-1_ in the indicated samples is shown on the bottom right. Gene boxes and 1D ChIP–seq tracks of matched models are aligned at the bottom. Black arrows indicate enhancer–promoter loops at _NKX2-1_ locus. Enhancers are highlighted in light blue, and promoters in light orange color. FOXA2, NKX2-1, H3K27ac, H3K4me1, H3K27me3 and CTCF tracks under the D0 contact map used their corresponding ChIP–seq performed in closely related D2 cells. All others were done in conditions exactly matched to Hi-C. The scale bars indicate ICE-normalized contact frequency. **e**, Heatmap showing active (H3K27ac) enhancer (H3K4me1) and promoter (H3K4me3) marks at NKX2-1-only, FOXA2-only and shared binding sites in NCI-H660 cells. Scale bar: enrichment intensity. **f**, Pie chart showing the genomic distribution of FOXA2-only, NKX2-1-only and shared binding sites identified in NCI-H660 cells. **g**, TF motifs that significantly enriched at NKX2-1-only, FOXA2-only binding sites in NCI-H660 (top) and LuCaP145.1 (bottom). _P_ values by two-sided hypergeometric test with Benjamini–Hochberg correction. **h**, The number of chromatin loops anchored by FOXA2 and NKX2-1 in time-course LNCaP+FOXA2 Hi-C samples.

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM8)

### [Extended Data Fig. 6 NKX2-1 is highly expressed in NEPC tumors and critical for NET of PCa.](https://www.nature.com/articles/s41588-025-02265-4/figures/14)

**a**, Positive correlation between _NKX2-1_ and _FOXA2_ mRNA levels in PCa datasets. Shown on x and y axis are z scores of log2(MAS5.0 signal intensity + 1). The linear regression line (blue) with its 95% confidence interval (gray) is shown. _P_ value by two-sided _t_ test without multiple comparisons. **b**, IGV views of H3K27ac (top) and DNA methylation (bottom, 5mC) at _NKX2-1_ locus in LuCaP PDX. The green boxes indicate CpG islands. **c**, Boxplots showing average CpG methylation level at CpG island (CpG:314) of _NKX2-1_ in CRPC and NEPC patient samples in ref. [3](https://www.nature.com/articles/s41588-025-02265-4#ref-CR3 "Beltran, H. et al. Divergent clonal evolution of castration-resistant neuroendocrine prostate cancer. Nat. Med. 22, 298–305 (2016).") (top, CRPC, n = 18; NEPC, n = 10) and in Zhao (2020) (bottom, CRPC, n = 93; NEPC, n = 5) dataset. Shown on the y axis are average percentage of methylated CpG at CpG island: 314. Boxplots represent the median and interquartile range (IQR); whiskers indicate the smallest and largest non-outlier values. _P_ value by two-sided _t_ test without multiple comparisons. **d**, Genomic alteration of _NKX2-1_ in the published PCa dataset. The data presented here consists of combined samples from following five studies: MSK, Clin Cancer Res 2020; SU2C/PCF Dream Team, PNAS 2019; Multi-Institute, Nat Med 2016; SU2C/PCF Dream Team, Cell 2015; MCTP, Nature 2012.

### [Extended Data Fig. 7 NKX2-1 is regulated by ASCL1 in FOXA2-low NEPC.](https://www.nature.com/articles/s41588-025-02265-4/figures/15)

**a**, The RNA-seq data of _FOXA2_, _NKX2-1_, _ASCL1_ and _NEUROD1_ in LuCaP NEPC PDX. _NKX2-1_ is expressed in both FOXA2-high (49, 145.1 and 145.2) and FOXA2-low (93 and 173.1) PDX. RNA-seq data were obtained from [GSE126078](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126078). **b**, Positive correlation between _NKX2-1_ and _ASCL1_ mRNA levels in PCa dataset. Shown on x and y axes are z scores of log2(MAS5.0 signal intensity + 1). The linear regression line (blue) with its 95% confidence interval (gray) is shown. _P_ value by two-sided _t_ test without multiple comparisons. **c**, IGV showing ASCL1, active enhancer marker (H3K4me1 and H3K27ac), promoter marker (H3K4me3) at _NKX2-1_ locus in LuCaP93 PDX. ASCL1 strongly binds to promoter and enhancer regions of NKX2-1. **d**, _ASCL1_ knockdown decreased expression of NKX2-1 at both mRNA (left) and protein (right) levels in LuCaP93 organoids. RT–PCR data shown are the mean ± s.e.m. of technical replicates from one of three (n = 3) independent experiments. _P_ values by unpaired two-sided _t_ test. WB data shown are from one of two (n = 2) independent experiments. **e**, Heatmap showing NKX2-1, ASCL1 binding at NKX2-1-only, shared and ASCL1-only sites in LuCaP93 PDX. Scale bar: enrichment intensity. **f**, _NKX2-1_ knockdown reduced LuCaP93 organoids growth. The quantification of the average number of organoids per field (n = 7 fields) is shown. _P_ values by two-sided _t_ test. **g**, NKX2-1 OE alone was unable to induce NET in LNCaP cells. LNCaP cells were infected with FOXA2 or NKX2-1 virus, and cells were collected for RT–PCR (left) and WB (right) analyses at week 4. The data are shown as in **d**.

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM8)

### [Extended Data Fig. 8 FOXA2 induces regional DNA demethylation during NET of PCa.](https://www.nature.com/articles/s41588-025-02265-4/figures/16)

**a**, The mA and mCpG profiling on single DNA molecules spanning FOXA2 peak center at 4 quartiles of FOXA2 peaks as in Fig. [6b](https://www.nature.com/articles/s41588-025-02265-4#Fig6) in D2 LNCaP+FOXA2 cells. Aggregate curves (top) for each quartile were created with a 50-bp rolling window. Base density across the 2-kb region for each quartile is indicated in the one-dimensional heatmaps (bottom); the scale bars indicate the number of adenine bases and CG dinucleotides sequenced at each position relative to FOXA2 peaks center as in Fig. [6b](https://www.nature.com/articles/s41588-025-02265-4#Fig6). Color scales are shown as in Fig. [6a](https://www.nature.com/articles/s41588-025-02265-4#Fig6). **b**, GO analyses by MSigDB of the top 200 genes associated with hyper-DMRs (left) and hypo-DMRs (right) between D2 and D28 LNCaP+FOXA2 cells. Top enriched molecular concepts are shown on the y axis, while the x axis indicates enrichment significance in −log10(p value); _p_ values were calculated by the one-sided hypergeometric test. **c**, PCA analyses of CRPC/NEPC PDX and patient samples using the top 200 genes as in **b**. **d**,**e**, Aggregated mCpG intensity plots showing the DNA methylation profiles of CRPC PDX (**d**) and NEPC PDX (**e**) at previously reported NE-CREs and Ad-CREs[11](https://www.nature.com/articles/s41588-025-02265-4#ref-CR11 "Baca, S. C. et al. Reprogramming of the FOXA1 cistrome in treatment-emergent neuroendocrine prostate cancer. Nat. Commun. 12, 1979 (2021).").

### [Extended Data Fig. 9 CCS1477 targets lineage-specific enhancers active in a particular cell.](https://www.nature.com/articles/s41588-025-02265-4/figures/17)

**a**, Heatmap showing FOXA2, NKX2-1 and p300 co-occupation at NE (C3–C4) and shared enhancers (C1–C2) in LuNE cells. The luminal/NE enhancers (enhs) are identified in Fig. [3a](https://www.nature.com/articles/s41588-025-02265-4#Fig3). Color bar at the bottom indicates the scale of ChIP–seq enrichment intensity. **b**, Heatmap showing H3K27ac at luminal and NE enhancers were, respectively, inhibited by CCS1477 in LNCaP and LuNE cells. The luminal/NE enhancers (enhs) are identified as in **a**. Color bar is shown as in **a**. **c**, Super-enhancers (SEs) were identified using H3K27ac ChIP–seq in LuNE cells. Hockey-stick plot showing the normalized rank and signals of H3K27ac. Representative SE-associated lineage-specific TFs and oncogenic genes are highlighted in red. **d**, IGV showing H3K27ac enrichment at SE-associated _HOXA_ gene cluster, _HOXB_ gene cluster and _NFIB_ gene locus in LuNE cells treated with DMSO or CCS1477. The blue horizontal lines on the top of the tracks mark SEs. **e**, SEs were identified in LNCaP cells and depicted as in **c**. Representative SE-associated lineage-specific TFs and oncogenic genes are highlighted in red. **f**, IGV showing H3K27ac status at SE-associated gene loci in LNCaP cells treated with DMSO or CCS1477. The blue horizontal lines on the top of the tracks mark SEs. **g**, WB confirming _p300_ and _CBP_ KD in LuNE cells subjected to sh_CBP_ and/or shp_300_. The data shown are from one of two (n = 2) independent experiments. **h**, GO analysis of p300/CBP-induced genes in LuNE cells. GO analysis was performed by MSigDB. Top enriched molecular concepts are shown on the y axis, while the x axis indicates enrichment significance. −log10(p value) is shown, _p_ values by one-sided hypergeometric test. **i**, GO analysis of FOXA2/NKX2-1-induced genes in LuNE cells, performed as in **g**. **j**, Heatmap showing that most of CCS1477-regulated genes (FC ≥ 2, adjusted p < 0.05) were similarly regulated by p300/CBP, NKX2-1 and FOXA2. Data shown are the log2(FC) of CCS1477 relative to DMSO, and of sg_NKX2-1_, sg_FOXA2_ or sg_FOXA2_+sg_NKX2-1_ relative to sgNC, and of sh_p300_, sh_CBP_ or sh_p300_+sh_CBP_ to shCtrl. Adjusted _p_ values by Wald test with Benjamini–Hochberg correction.

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM10)

### [Extended Data Fig. 10 p300/CBP inhibition suppresses NEPC cell growth in vitro and in vivo_._](https://www.nature.com/articles/s41588-025-02265-4/figures/18)

**a**, Colony formation assay of LuNE cells with the indicated gene KD. **b**,**c**, Colony formation assay of LuNE (**b**), LNCaP (**c**) cells treated with the indicated concentration of CCS1477. **d**, WB analysis of markers for cell cycle arrest (p21) and apoptosis (cleaved PARP and cleaved caspase 3) in LNCaP and LuNE cells treated with 500 nM CCS1477 for 24 h. Cycloheximide (CHX, 100 μg ml−1, 24 h)-treated cells were used as a control for cytotoxic effects. The data shown are from one of two (n = 2) independent experiments. **e**, Colony formation assay of benign prostatic hyperplasia BPH-1 and normal prostate epithelial RWPE-1 treated with the indicated concentration of CCS1477. **f**, Growth of LuCaP145.2 organoids and NCI-H660 cells treated with increasing doses of CCS1477. **g**, Images of LuNE xenograft tumors treated with vehicle or CCS1477. **h**, Body weight of mice treated with vehicle or CCS1477 for 33 days. Data in each time point are mean ± s.d. Y axis shows the percentage of body weight change. **i**, Representative IHC images of H3K27ac, SOX2 and NKX2-1 in LuNE xenograft tumors treated with vehicle or CCS1477 (n = 5 tumors per group). Scale bar, 30 µm. Insets are shown at higher magnification (×40). **j**, WB analysis of LuNE cells treated with 500 nM CCS1477 up to 60 h. The data shown are from one of two (n = 2) independent experiments. **k**, Images of LuCaP145.2 xenograft tumors treated with vehicle or CCS1477. **l**, H&E (left) and IHC of AR and PSA proteins (right) in LuCaP77CR (as a CRPC control) and in LuCaP145.2 PDX tumors treated with vehicle or CCS1477. Representative H&E and IHC images (n = 5 tumors per group) are shown. Scale bar, 30 µm. Insets are shown at higher magnification (×40).

[Source data](https://www.nature.com/articles/s41588-025-02265-4#MOESM10)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM1_ESM.pdf)

Supplementary Notes 1–5, Supplementary Methods 1–12 and Supplementary Figs. 1 and 2.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM3_ESM.pdf)

### [Supplementary Tables 1–5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM4_ESM.xlsx)

Supplementary Tables 1–5.

Source data
-----------

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM5_ESM.xlsx)

Statistical source data for Fig. 3f.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM6_ESM.xlsx)

Statistical source data for Fig. 5c,f–h.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM7_ESM.xlsx)

Statistical source data for Fig. 7a,i–l.

### [Source Data Extended Data Figs. 1, 2, 5 and 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM8_ESM.xlsx)

Statistical source data for Extended Data Figs. 1b,c,f,g, 2d–f, 5a,d and 7a,d,f,g.

### [Source Data Figs. 2, 4, 5 and 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM9_ESM.pdf)

Unprocessed WBs for Figs. 2b, 4b,c,g, 5d,f–h and 7b,c.

### [Source Data Extended Data Figs. 2, 7, 9 and 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02265-4/MediaObjects/41588_2025_2265_MOESM10_ESM.pdf)

Unprocessed WBs for Extended Data Figs. 2b,c, 7d,g, 9g and 10d,j.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=NKX2-1%20drives%20neuroendocrine%20transdifferentiation%20of%20prostate%20cancer%20via%20epigenetic%20and%203D%20chromatin%20remodeling&author=Xiaodong%20Lu%20et%20al&contentID=10.1038%2Fs41588-025-02265-4&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-21&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02265-4)

### Cite this article

Lu, X., Keo, V., Cheng, I. _et al._ NKX2-1 drives neuroendocrine transdifferentiation of prostate cancer via epigenetic and 3D chromatin remodeling. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02265-4

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02265-4?format=refman&flavour=citation)

-   Received: 26 July 2023
    
-   Accepted: 16 June 2025
    
-   Published: 21 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02265-4
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02265-4.pdf)
