> id: https://www.nature.com/articles/s41588-025-02217-y

> link: https://www.nature.com/articles/s41588-025-02217-y

> title: Detecting and quantifying clonal selection in somatic stem cells

# Detecting and quantifying clonal selection in somatic stem cells
_Published on Thu, 03 Jul 2025 00:00:00 GMT_

Abstract
--------

As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01755-1/MediaObjects/41588_2024_1755_Fig1_HTML.png)

### [Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis](https://www.nature.com/articles/s41588-024-01755-1?fromPaywallRec=false)

Article Open access 14 May 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-54711-2/MediaObjects/41467_2024_54711_Fig1_HTML.png)

### [Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life](https://www.nature.com/articles/s41467-024-54711-2?fromPaywallRec=false)

Article Open access 26 November 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-020-2819-2/MediaObjects/41586_2020_2819_Fig1_HTML.png)

### [Inherited causes of clonal haematopoiesis in 97,691 whole genomes](https://www.nature.com/articles/s41586-020-2819-2?fromPaywallRec=false)

Article 14 October 2020

Main
----

Normal stem cells accumulate somatic variation throughout development[1](https://www.nature.com/articles/s41588-025-02217-y#ref-CR1 "Spencer Chapman, M. et al. Lineage tracing of human development through somatic mutations. Nature 595, 85–90 (2021).") and postnatal life[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."),[3](https://www.nature.com/articles/s41588-025-02217-y#ref-CR3 "Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019)."),[4](https://www.nature.com/articles/s41588-025-02217-y#ref-CR4 "Brunner, S. F. et al. Somatic mutations and clonal dynamics in healthy and cirrhotic human liver. Nature 574, 538–542 (2019)."),[5](https://www.nature.com/articles/s41588-025-02217-y#ref-CR5 "Moore, L. et al. The mutational landscape of normal human endometrial epithelium. Nature 580, 640–646 (2020)."),[6](https://www.nature.com/articles/s41588-025-02217-y#ref-CR6 "Grossmann, S. et al. Development, maturation, and maintenance of human prostate inferred from somatic mutations. Cell Stem Cell 28, 1262–1274. e5 (2021)."),[7](https://www.nature.com/articles/s41588-025-02217-y#ref-CR7 "De, S. Somatic mosaicism in healthy human tissues. Trends Genet. 27, 217–223 (2011)."),[8](https://www.nature.com/articles/s41588-025-02217-y#ref-CR8 "Vijg, J. & Dong, X. Pathogenic mechanisms of somatic mutation and genome mosaicism in aging. Cell 182, 12–23 (2020)."). Over time, stem cell clones vary in size owing to genetic drift[9](https://www.nature.com/articles/s41588-025-02217-y#ref-CR9 "Lopez-Garcia, C., Klein, A. M., Simons, B. D. & Winton, D. J. Intestinal stem cell replacement follows a pattern of neutral drift. Science 330, 822–825 (2010)."),[10](https://www.nature.com/articles/s41588-025-02217-y#ref-CR10 "Snippert, H. J. et al. Intestinal crypt homeostasis results from neutral competition between symmetrically dividing Lgr5 stem cells. Cell 143, 134–144 (2010)."), or selection[4](https://www.nature.com/articles/s41588-025-02217-y#ref-CR4 "Brunner, S. F. et al. Somatic mutations and clonal dynamics in healthy and cirrhotic human liver. Nature 574, 538–542 (2019)."),[5](https://www.nature.com/articles/s41588-025-02217-y#ref-CR5 "Moore, L. et al. The mutational landscape of normal human endometrial epithelium. Nature 580, 640–646 (2020)."),[6](https://www.nature.com/articles/s41588-025-02217-y#ref-CR6 "Grossmann, S. et al. Development, maturation, and maintenance of human prostate inferred from somatic mutations. Cell Stem Cell 28, 1262–1274. e5 (2021)."),[11](https://www.nature.com/articles/s41588-025-02217-y#ref-CR11 "Martincorena, I. et al. High burden and pervasive positive selection of somatic mutations in normal human skin. Science 348, 880–886 (2015)."),[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."),[14](https://www.nature.com/articles/s41588-025-02217-y#ref-CR14 "Poon, G. Y., Watson, C. J., Fisher, D. S. & Blundell, J. R. Synonymous mutations reveal genome-wide levels of positive selection in healthy tissues. Nat. Genet. 53, 1597–1605 (2021)."),[15](https://www.nature.com/articles/s41588-025-02217-y#ref-CR15 "Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)."),[16](https://www.nature.com/articles/s41588-025-02217-y#ref-CR16 "Huang, A.Y. et al. Somatic cancer driver mutations are enriched and associated with inflammatory states in Alzheimer’s disease microglia. Preprint at bioRxiv 
https://doi.org/10.1101/2024.01.03.574078
(2024)."),[17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022)."). Quantifying proliferation, differentiation, somatic mutation and selection dynamics in tissue stem cells is fundamental to our understanding of somatic mosaicism.

Clonal hematopoiesis (CH), the expansion of hematopoietic stem cell (HSC) clones, is a paradigm for somatic mosaicism. CH, detected with increased frequency as humans age, is associated with known driver events: either somatic variants in genes recurrently mutated in myeloid cancers (for example, _DNMT3A_, _TET2_ and _ASXL1_), or chromosomal copy number alterations[18](https://www.nature.com/articles/s41588-025-02217-y#ref-CR18 "Busque, L. et al. Recurrent somatic TET2 mutations in normal elderly individuals with clonal hematopoiesis. Nat. Genet. 44, 1179–1181 (2012)."),[19](https://www.nature.com/articles/s41588-025-02217-y#ref-CR19 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[20](https://www.nature.com/articles/s41588-025-02217-y#ref-CR20 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[21](https://www.nature.com/articles/s41588-025-02217-y#ref-CR21 "Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."),[22](https://www.nature.com/articles/s41588-025-02217-y#ref-CR22 "Xie, M. et al. Age-related mutations associated with clonal hematopoietic expansion and malignancies. Nat. Med. 20, 1472–1478 (2014)."),[23](https://www.nature.com/articles/s41588-025-02217-y#ref-CR23 "Terao, C. et al. Chromosomal alterations among age-related haematopoietic clones in Japan. Nature 584, 130–135 (2020)."),[24](https://www.nature.com/articles/s41588-025-02217-y#ref-CR24 "Saiki, R. et al. Combined landscape of single-nucleotide variants and copy number alterations in clonal hematopoiesis. Nat. Med. 27, 1239–1249 (2021)."),[25](https://www.nature.com/articles/s41588-025-02217-y#ref-CR25 "Bernstein, N. et al. Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis. Nat. Genet. 56, 1147–1155 (2024)."),[26](https://www.nature.com/articles/s41588-025-02217-y#ref-CR26 "Niroula, A. et al. Distinction of lymphoid and myeloid clonal hematopoiesis. Nat. Med. 27, 1921–1927 (2021)."). CH drivers not only provide a selective advantage to HSCs[15](https://www.nature.com/articles/s41588-025-02217-y#ref-CR15 "Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)."),[27](https://www.nature.com/articles/s41588-025-02217-y#ref-CR27 "Jakobsen, N. A. et al. Selective advantage of mutant stem cells in human clonal hematopoiesis is associated with attenuated response to inflammation and aging. Cell Stem Cell 31, 1127–1144 (2024)."), but also generate proinflammatory myeloid cells[28](https://www.nature.com/articles/s41588-025-02217-y#ref-CR28 "Encabo, H. H. et al. Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils. Cell Stem Cell 30, 781–799. e9 (2023)."),[29](https://www.nature.com/articles/s41588-025-02217-y#ref-CR29 "Wang, H. et al. Clonal hematopoiesis driven by mutated DNMT3A promotes inflammatory bone loss. Cell 187, 3690–3711 (2024)."). CH with known drivers is associated with a heightened risk of blood cancer[19](https://www.nature.com/articles/s41588-025-02217-y#ref-CR19 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[20](https://www.nature.com/articles/s41588-025-02217-y#ref-CR20 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[22](https://www.nature.com/articles/s41588-025-02217-y#ref-CR22 "Xie, M. et al. Age-related mutations associated with clonal hematopoietic expansion and malignancies. Nat. Med. 20, 1472–1478 (2014)."),[30](https://www.nature.com/articles/s41588-025-02217-y#ref-CR30 "Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. Nature 559, 400–404 (2018)."), excess cardiovascular mortality[31](https://www.nature.com/articles/s41588-025-02217-y#ref-CR31 "Jaiswal, S. et al. Clonal hematopoiesis and risk of atherosclerotic cardiovascular disease. N. Engl. J. Med. 377, 111–121 (2017)."), chronic pathology and infection[32](https://www.nature.com/articles/s41588-025-02217-y#ref-CR32 "Dharan, N. J. et al. HIV is associated with an increased risk of age-related clonal hematopoiesis among older adults. Nat. Med. 27, 1006–1011 (2021)."),[33](https://www.nature.com/articles/s41588-025-02217-y#ref-CR33 "Zekavat, S. M. et al. Hematopoietic mosaic chromosomal alterations increase the risk for diverse types of infection. Nat. Med. 27, 1012–1024 (2021)."),[34](https://www.nature.com/articles/s41588-025-02217-y#ref-CR34 "Kim, P. G. et al. Dnmt3a-mutated clonal hematopoiesis promotes osteoporosis. J. Exp. Med. 218, e20211872 (2021)."),[35](https://www.nature.com/articles/s41588-025-02217-y#ref-CR35 "Wong, W. J. et al. Clonal haematopoiesis and risk of chronic liver disease. Nature 616, 747–754 (2023)."),[36](https://www.nature.com/articles/s41588-025-02217-y#ref-CR36 "Vlasschaert, C. et al. Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury. Nat. Med. 30, 810–817 (2024)."). Mathematical analysis of somatic variant distributions predicts that CH is far more prevalent than suggested by known drivers[14](https://www.nature.com/articles/s41588-025-02217-y#ref-CR14 "Poon, G. Y., Watson, C. J., Fisher, D. S. & Blundell, J. R. Synonymous mutations reveal genome-wide levels of positive selection in healthy tissues. Nat. Genet. 53, 1597–1605 (2021)."); expanded clones without known driver events are possibly more frequent than those with known driver events[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).").

These observations raise the following questions: (1) Do CH drivers emerge by chance, or does increased turnover and/or somatic mutation rate facilitate their acquisition? (2) Do stem cell clones with known and unknown drivers differ in their frequency and clinical phenotypes? (3) When do selection events occur in life, and (4) how fast do the selected clones grow? These questions are best addressed with an affordable, scalable method to identify clonal selection and quantify stem cell dynamics.

Here, we present a method to detect clonal selection and quantify stem cell parameters without knowledge of the driver event, using a bulk primary sample from any tissue. The method applies population genetics theory for the accumulation and spread of somatic variants in development and subsequent homeostatic tissue renewal, and a Bayesian inference framework to discriminate clonal selection from neutral evolution and, moreover, quantify stem cell parameters. We validate our method with synthetic data and published experimental data and then apply it to a new cohort of human nonleukemic bone marrow (BM) samples and a published dataset of brain tissue samples.

Results
-------

### Quantifying clonal selection

We reasoned that the distribution of somatic variants in stem cells provides information on the dynamics of genetic drift and selection. To derive the expected variant allele frequency (VAF) distribution of a renewing tissue, we modeled expanding stem cell numbers during development, followed by an adult phase in which stem cell numbers are balanced (Fig. [1a](https://www.nature.com/articles/s41588-025-02217-y#Fig1) and [Methods](https://www.nature.com/articles/s41588-025-02217-y#Sec12)). We focused on somatic single-nucleotide variants (SSNVs) acquired during both phases. In early life, stem cell numbers are small and few SSNVs are acquired, which will be at high VAF (Fig. [1b](https://www.nature.com/articles/s41588-025-02217-y#Fig1); variants A and B). Later, when stem cell numbers are larger, more variants are acquired in total but at lower VAF (Fig. [1b](https://www.nature.com/articles/s41588-025-02217-y#Fig1); variants C, D and E). Neutral SSNVs drift with time; some randomly increase their VAF (Fig. [1b](https://www.nature.com/articles/s41588-025-02217-y#Fig1); variants B, C, D and F) while others decrease or are extinguished (Fig. [1b](https://www.nature.com/articles/s41588-025-02217-y#Fig1); variant E). We model neutral clonal dynamics during development and homeostasis by linking expanding (supercritical) and homeostatic (critical) birth–death processes ([Methods](https://www.nature.com/articles/s41588-025-02217-y#Sec12), ‘Population genetics model’). We find an analytical solution for the VAF distribution as a function of time that lends itself to parameter inference from experimental data.

**Fig. 1: Population genetics model of drift and selection in homeostatic tissues.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02217-y/figures/1)

**a**, Modeled processes and associated parameters in the model of drift. Stem cells either divide symmetrically with rate _λ_, or exit the stem cell compartment by differentiating (or dying), with rate _δ_. The stem cell count (_N_) increases in development (_λ_ > _δ_) until reaching steady-state numbers (_N_ss) and remains constant during adulthood (_N_ = _N_ss and _λ_ = _δ_). On average, cells acquire _μ_ neutral variants during each cell division. **b**, Schematic illustrating variant accumulation during development and subsequent homeostasis. **c**,**d**, Cumulative number of SSNVs versus VAF in development (**c**) and in adult life (**d**) (the scaling of the _x_ axis is transformed to \\(\\frac{1}{\\rm{VAF}}\\) to spread out low-frequency variants). **e**–**g**, Simulated cumulative VAF distribution of SSNVs at selected ages between 0 and 100 years for 5 × 103 stem cells (_λ_ = 5 per year, _μ_ = 10 per division) (**e**); for 5 × 104 stem cells (_λ_ = 5 per year, _μ_ = 10 per division) (**f**); and for 5 × 105 stem cells (_λ_ = 5 per year, _μ_ = 10 per division) (**g**). **h**, Model of clonal selection. A selective driver event reduces the loss rate (differentiation or death) by a factor _s_, causing selective outgrowth of the mutant clone (red); the remaining parameters are defined in **a**. The VAF of the selected clone increases exponentially with the age at measurement, _a_. **i**, As **b**, but here an acquired driver mutation (D) causes selective outgrowth of the mutant clone (red). **j**, All variants in the selected clone’s cell of origin are inherited by its progeny and hence reach a high VAF during clonal expansion, reflected in a shoulder in the cumulative VAF distribution. **k**, Simulated cumulative VAF distributions when a driver mutation is acquired at different ages, and the SSNVs are measured 45 years later, when the clone has reached a size of 32%. In the simulation, the selected clone grows by 22% per year (_s_ = 0.02, _λ_ = 10 per year, _μ_ = 1 per division, _N_ss = 25,000). **l**, Simulated cumulative VAF distributions measured at varying ages after a driver mutation was acquired at 20 years of age. As **k**, the selected clone grows by 22% per year (_s_ = 0.02, _λ_ = 10 per year, _μ_ = 1 per division, _N_ss = 25,000).

Our model suggests that during development genetic drift is negligible, resulting in an invariant linear relation between cumulative SSNV count and \\(1/\\rm{VAF}\\) (Fig. [1c](https://www.nature.com/articles/s41588-025-02217-y#Fig1))[37](https://www.nature.com/articles/s41588-025-02217-y#ref-CR37 "Bozic, I., Gerold, J. M. & Nowak, M. A. Quantifying clonal and subclonal passenger mutations in cancer evolution. PLoS Comput. Biol. 12, e1004731 (2016)."),[38](https://www.nature.com/articles/s41588-025-02217-y#ref-CR38 "Williams, M. J., Werner, B., Barnes, C. P., Graham, T. A. & Sottoriva, A. Identification of neutral tumor evolution across cancer types. Nat. Genet. 48, 238–244 (2016)."). During homeostasis, genetic drift alters the cumulative VAF histogram by increasing the number of SSNVs with low VAF (Fig. [1d](https://www.nature.com/articles/s41588-025-02217-y#Fig1)). The extent of homeostatic drift depends on the number of stem cells and their self-renewal rate. With 5 × 103 stem cells dividing five times per year, drift increases the count of variants with high VAF over time (Fig. [1e](https://www.nature.com/articles/s41588-025-02217-y#Fig1); we assume that the transition from development to homeostasis occurs around birth). For 5 × 104 stem cells and the same division rate, the variant count will increase with age at lower VAFs and remain stable at high VAFs (Fig. [1f](https://www.nature.com/articles/s41588-025-02217-y#Fig1)). Finally, with 5 × 105 stem cells, the distribution is nearly stable with age for all VAFs >1% (Fig. [1g](https://www.nature.com/articles/s41588-025-02217-y#Fig1) and Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02217-y#Fig8)). Previous work inferred that 4 × 104 to 2 × 105 HSCs divide 0.6–6 times per year[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."). With these values, our model suggests that the VAF histogram is insensitive to genetic drift during life at high (≥10%) but not low VAF. Including extensively self-renewing progenitors in the model, as observed in mice[39](https://www.nature.com/articles/s41588-025-02217-y#ref-CR39 "Busch, K. et al. Fundamental properties of unperturbed haematopoiesis from stem cells in vivo. Nature 518, 542–546 (2015)."), only marginally modifies the distribution of somatic variants with ≥1% VAF (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02217-y#MOESM1)), supporting our focus on stem cell dynamics to interpret measured VAF histograms.

Next, we modeled how clonal selection alters the VAF distribution. We consider that the leading selected clone, born at time _t_s with selective advantage _s_, expands while displacing normal stem cells (Fig. [1h,i](https://www.nature.com/articles/s41588-025-02217-y#Fig1)). With clonal selection, neutral variants may have originated: (1) in normal stem cells before the driver event, (2) in the selected clone, or (3) in nonselected stem cells after the driver event occurred. We find analytical expressions for the evolution of all three types of variants, which together yield the cumulative VAF distribution of SSNVs ([Methods](https://www.nature.com/articles/s41588-025-02217-y#Sec12)). Expansion of a selected clone increases the VAF of the founding driver mutation (mutation D in Fig. [1i](https://www.nature.com/articles/s41588-025-02217-y#Fig1)) and of all other neutral SSNVs in the founding cell, which generates a shoulder in the cumulative VAF distribution (Fig. [1j](https://www.nature.com/articles/s41588-025-02217-y#Fig1)). Thus, the shape of the distribution discriminates between selection and neutral evolution, whereas total variant count may fail to do so (compare the VAF histograms in Fig. [1k](https://www.nature.com/articles/s41588-025-02217-y#Fig1), shaped by selection, with Fig. [1e,f](https://www.nature.com/articles/s41588-025-02217-y#Fig1), shaped by drift).

The shoulder height increases with the age at which the selected clone originated, _t_s, (Fig. [1k](https://www.nature.com/articles/s41588-025-02217-y#Fig1)), because the number of neutral variants in the clone’s stem cell of origin is the higher the older the individual. As the selected clone expands, the shoulder reaches higher VAFs (Fig. [1l](https://www.nature.com/articles/s41588-025-02217-y#Fig1)). Thus, the VAF reached between clone origin and sample acquisition yields the selective advantage _s_ (Fig. [1h](https://www.nature.com/articles/s41588-025-02217-y#Fig1), lower panel). Collectively, the shoulder in the cumulative VAF distribution provides a robust signature of clonal selection agnostic of driver identity. Moreover, its height and position allow inference of the clone’s age and time-averaged selective advantage (Extended Data Fig. [1b,c](https://www.nature.com/articles/s41588-025-02217-y#Fig8)).

### Selection detected in noisy whole-genome sequencing data

Sufficiently deep whole-genome sequencing (WGS) measures VAFs with approximately binomially distributed error[40](https://www.nature.com/articles/s41588-025-02217-y#ref-CR40 "Körber, V. et al. Evolutionary trajectories of IDHWT glioblastomas reveal a common path of early tumorigenesis instigated years ahead of initial diagnosis. Cancer Cell 35, 692–704. e12 (2019)."). To understand how this error influences detection of clonal selection in bulk WGS data, we simulated WGS data with our models of neutral evolution (Fig. [1a,b](https://www.nature.com/articles/s41588-025-02217-y#Fig1)) and selection, focusing on a single selected clone (Fig. [1h,i](https://www.nature.com/articles/s41588-025-02217-y#Fig1)). We performed repeated stochastic simulations, generating 10 datasets with neutral evolution and 70 datasets with selection (2.5% ≤ VAF ≤ 37.5%). To model measurement error, we drew variant reads from binomial distributions for sequencing depths of 30×, 90× or 270× ([Methods](https://www.nature.com/articles/s41588-025-02217-y#Sec12) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)). We then used approximate Bayesian computation (ABC) to fit both the model of neutral evolution and of selection to each dataset (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02217-y#Fig8)), yielding posterior probabilities for both models to reproduce the noisy data.

To determine sensitivity and specificity in detecting selection, we used the posteriors to compute receiver operating curves (ROC) for identifying clones of different sizes. At 90× sequencing depth, we found that selected clones with VAF ≥ 5% were detected reliably (Fig. [2a](https://www.nature.com/articles/s41588-025-02217-y#Fig2); area under the curve (AUC) = 0.96 for VAF = 5% and AUC ~ 1 for VAF = 7.5%), whereas smaller clones were not (Fig. [2a](https://www.nature.com/articles/s41588-025-02217-y#Fig2); VAF = 2.5%; AUC = 0.4). Discrimination between selection and neutral evolution was optimal at a threshold of 15% for the posterior probability of the selection model (Fig. [2a](https://www.nature.com/articles/s41588-025-02217-y#Fig2), operating points correspond to the 15% threshold, achieving ~100% sensitivity and 90% specificity for clone size 5% VAF and ~100% sensitivity and specificity for clone size 7.5% VAF). Exemplary selection posteriors for simulated 90× WGS datasets show discrimination between neutral evolution and selection of clones ≥5% VAF (Fig. [2b](https://www.nature.com/articles/s41588-025-02217-y#Fig2)). Further, our ROC analysis predicts that 30× WGS will allow detection of large, selected clones >20% VAF, while deeper WGS enables detection of smaller clones with VAF > 1% at 270× (Fig. [2c](https://www.nature.com/articles/s41588-025-02217-y#Fig2)).

**Fig. 2: Benchmarking SCIFER with simulated data.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02217-y/figures/2)

**a**, ROCs quantifying the detection of clonal selection by SCIFER for different clone sizes (color-encoded). ROCs were generated by applying SCIFER to simulated data, generated with a stochastic birth–death process with (_s_ = 0.02, corresponding to a selective advantage of 2% increase in birth versus death) or without (_s_ = 0) selection of a clone initiated at 20 years of life (_λ_ = 10 per year, _μ_ = 1 per division, _N_ss = 25,000, and assuming sequencing with an average coverage of 90×). In total 63 cases, with selected clone sizes of VAF 0%, 2.5%, 5%, 7.5%, 10%, 25% and 37.5%, were generated. Models with, or without, clonal selection were fit to the data using ABC. True positives and false positives were evaluated for varying posterior probability thresholds of clonal selection. For selected clones with VAF ≥ 5%, the difference between true positives and false positives was maximal for a selection threshold of 15% (operating points, shown in red). **b**, Posterior probability for clonal selection (colored bars) and neutral evolution (gray bars) conditioned on selected clones with VAF ≥ 5% for six simulated cases with varying clone size. The dashed line marks the selection threshold at 15% conditional posterior probability. **c**, Accuracy of SCIFER to distinguish clonal selection from genetic drift in simulated WGS data. Shown are AUC computed from the ROCs shown in **a** and from ROCs obtained in analogy for simulated sequencing depths of 30× and 270×. The simulated data were generated as in **a**. For 270× sequencing depth, an additional 17 cases with selected clone sizes of 0.5% VAF and 1% VAF were used for model evaluation. **d**, Model scheme for two selected clones (red and orange) that compete with normal cells (blue). The two selected clones are born at times _t_s1 and _t_s2, and expand due to decreased loss rates (_δ_s1 and _δ_s2); the total cell count remains constant over time. **e**, Selection of two sequential clones manifests itself in two subclonal shoulders whose heights scale with the time points of driver acquisition.

Taken together, our method for selected clone inference, termed SCIFER, is suitable for detecting clonal selection in bulk WGS data.

### SCIFER identifies sequential selected clones

Next we extended SCIFER beyond a single leading selected clone to study two competing clones, born at times _t_s1 and _t_s2, and conferring a selective advantage of, respectively, _s_1 and _s_2 (Fig. [2d](https://www.nature.com/articles/s41588-025-02217-y#Fig2)). Considering branched (Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02217-y#Fig9)) and linear evolution (Extended Data Fig. [2b](https://www.nature.com/articles/s41588-025-02217-y#Fig9)), we found that, in both cases, the VAF distribution has two subclonal shoulders, generated by the two competing clones (Extended Data Fig. [2c,d](https://www.nature.com/articles/s41588-025-02217-y#Fig9)). As with a single clone, the shoulders have heights proportional to the clonal birth dates that increase in VAF according to the fitness of the selected clones (Fig. [2e](https://www.nature.com/articles/s41588-025-02217-y#Fig2)). We found that the two-clone inference model is particularly appropriate for the resolution of deeply sequenced WGS data (270×).

### Benchmarking SCIFER with human hematopoiesis phylogenies

We benchmarked SCIFER with published WGS data from human hematopoietic stem and progenitor cell (HSPC) single-cell clones[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."),[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). Previously determined HSC parameters of neutral hematopoiesis (HSC number, self-renewal rate and rate of somatic variants acquisition)[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018).") provided a quantitative reference for SCIFER. Because variant calling is a source of error in WGS, we used a stringent approach intersecting the results of two callers, Mutect2 and Strelka[41](https://www.nature.com/articles/s41588-025-02217-y#ref-CR41 "Tanner, G., Westhead, D. R., Droop, A. & Stead, L. F. Benchmarking pipelines for subclonal deconvolution of bulk tumour sequencing data. Nat. Commun. 12, 6396 (2021).") (Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02217-y#Fig10)). To examine the robustness of SCIFER with respect to variant calling, we also performed our analyses on the original variant calls obtained with Caveman[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."), which included substantially more SSNVs (Extended Data Fig. [3b,c](https://www.nature.com/articles/s41588-025-02217-y#Fig10)). Reconstructing the phylogenetic tree with our calling approach recapitulated the original work (Fig. [3a](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). Applying SCIFER to pseudo-bulk data generated from single cells (Fig. [3b](https://www.nature.com/articles/s41588-025-02217-y#Fig3) and Extended Data Fig. [3d](https://www.nature.com/articles/s41588-025-02217-y#Fig10)) detected neutral evolution, irrespective of the variant calling algorithm, in agreement with the published findings (Fig. [3c](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). The inferred HSC number (Fig. [3d](https://www.nature.com/articles/s41588-025-02217-y#Fig3)) and rate of self-renewing divisions (Fig. [3e](https://www.nature.com/articles/s41588-025-02217-y#Fig3)) were similar for the two variant callers (6 × 104 to 2 × 105 HSCs dividing 3–8 times per year using Caveman; 2 × 104 to 5 × 104 HSCs dividing 0.4–6 times per year using Mutect2 and Strelka; ranges are 80% credible intervals). Both estimates overlapped with the previously published numbers (4 × 104 to 2 × 105 HSCs dividing 0.6–6 times per year)[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."). The rate of SSNV acquisition inferred by SCIFER using Caveman caller (3–5 SSNVs per cell division; Fig. [3f](https://www.nature.com/articles/s41588-025-02217-y#Fig3)) overlapped with the original estimate (3–28 SSNVs per division). As expected, this rate was smaller (0.6–1 SSNV per cell division) with our more conservative calling approach (intersecting Mutect2 and Strelka).

**Fig. 3: Benchmarking SCIFER with published pseudo-bulk data.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02217-y/figures/3)

**a**, Reconstructed single-cell phylogenies after re-calling SSNVs and indels from single-cell WGS data[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."). **b**, Left, VAF distribution of SSNVs shown in **a**, truncated at 1%. Right, model fit to the cumulative \\(\\frac{1}{\\rm{VAF}}\\) distribution (points and error bars, measured data and their standard deviation, which, assuming Poisson-distributed measurements, is the square root of the measured data; red area, 95% posterior probabilities of the model fit computed from simulations using 100 posterior samples). **c**, Posterior probability for neutral evolution for pseudo-bulk WGS data from ref. [2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018).") (labeled Lee-Six) and three samples from ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). SCIFER was applied twice to the data from ref. [2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."), using the SSNV counts obtained with Caveman or with Mutect2 and Strelka. **d**–**f**, Inferred HSC number (**d**), division rate (**e**) and number of SSNVs per division (**f**) for the cases shown in **c** (median and 80% credible intervals for each sample, estimated from 1,000 posterior samples; gray areas, 95% confidence band for the five estimates obtained with SCIFER). Estimates from ref. [2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018).") are given for comparison. **g**, Single-cell phylogeny of published sample KX004 (ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).")). **h**, As **b**, but for the sample shown in **g** (gray area in right panel, 80% credible interval of the estimated clone size computed from 1,000 posterior samples). **i**, Posterior probabilities for selection (conditioned on clones with VAF ≥ 5%) and neutral evolution for samples introduced in **g** and Extended Data Fig. [3i](https://www.nature.com/articles/s41588-025-02217-y#Fig10). Dashed line, 15% selection threshold. **j**, Age of leading selected clones in the cases shown in **i**, estimated by SCIFER and by phylodynamic modeling in the original publications (points, median; error bars, 80% credible intervals, estimated from 1,000 posterior samples). **k**,**l**, Estimated clonal growth rates (**k**) and stem cell parameters (**l**) for the samples shown in **i** (points, median; error bars, 80% credible intervals, estimated from 1,000 posterior samples). **m**, Single-cell phylogenies of published sample KX003 (ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).")). **n**, As **h** but for the sample shown in **m**. **o**, As **i**, but for sample KX003 (ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).")). **p**, Estimated stem cell and selection parameters for the sample shown in **m**. Shown are median and 80% credible intervals, estimated from 1,000 posterior samples.

SCIFER infers HSC number, self-renewal rate and mutation rate separately (Extended Data Fig. [3e,f](https://www.nature.com/articles/s41588-025-02217-y#Fig10)), which is based on: (1) using both expansion and homeostatic phases (Supplementary Note [2](https://www.nature.com/articles/s41588-025-02217-y#MOESM1) and Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02217-y#MOESM1)), assuming a homogeneous mutation rate throughout; and (2) the use of absolute variant counts, because many rapidly dividing stem cells accumulate more variants than few rarely dividing stem cells (although the ratio of stem cell number and self-renewal rate may remain unchanged[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."),[15](https://www.nature.com/articles/s41588-025-02217-y#ref-CR15 "Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020).")). Thus, SCIFER detected neutral evolution and quantified HSC number and division rate from a single pseudo-bulk sample.

To further validate SCIFER, we generated additional pseudo-bulk data from three neutrally evolving, published human HSPC single-cell WGS datasets (AX001, KX001 and KX002)[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).") (Extended Data Fig. [3g,h](https://www.nature.com/articles/s41588-025-02217-y#Fig10)). In all cases, SCIFER detected neutral evolution (Fig. [3c](https://www.nature.com/articles/s41588-025-02217-y#Fig3)) and inferred similar HSC parameters (Fig. [3d,f](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). Interestingly, for the oldest individual (AX001, 63 years old), the inferred HSC division rate was higher than for the other individuals (29 and 38 years old) (Fig. [3e](https://www.nature.com/articles/s41588-025-02217-y#Fig3)).

Next, we asked whether SCIFER detects clonal selection in pseudo-bulk WGS data from single HSPC clones from two published samples, KX0004 (ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).")) (_DNMT3A_ mutation, ~6%–8% VAF; Fig. [3g](https://www.nature.com/articles/s41588-025-02217-y#Fig3)) and id2259 (ref. [12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022).")) (_SF3B1_ mutation, nearly 50% VAF; Extended Data Fig. [3i](https://www.nature.com/articles/s41588-025-02217-y#Fig10)). SCIFER identified the leading _DNMT3A_ (VAF 6%) and _SF3B1_ (VAF 49%) clones (Fig. [3h](https://www.nature.com/articles/s41588-025-02217-y#Fig3) and Extended Data Fig. [3i](https://www.nature.com/articles/s41588-025-02217-y#Fig10)) as selected (Fig. [3i](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). Moreover, using the two-clone model, SCIFER also detected the second largest clone in KX004 (Extended Data Fig. [3j,k](https://www.nature.com/articles/s41588-025-02217-y#Fig10)). The original studies estimated that the _SF3B1_ mutation arose 25–30 years before tissue sampling, and the _DNMT3A_ mutation 45–50 years before tissue sampling. SCIFER determined very similar clone ages (Fig. [3j](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). The estimated clonal growth rates (Fig. [3k](https://www.nature.com/articles/s41588-025-02217-y#Fig3)) were concordant with previous data suggesting stronger selection of _SF3B1_ mutant clones[42](https://www.nature.com/articles/s41588-025-02217-y#ref-CR42 "Robertson, N. A. et al. Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects. Nat. Med. 28, 1439–1446 (2022)."). Inferred stem cell parameters (Fig. [3l](https://www.nature.com/articles/s41588-025-02217-y#Fig3)) were similar to individuals without CH (Fig. [3d–f](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). Finally, we applied SCIFER to a published sample without known CH driver, KX003 (ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).")), having the largest expanded clone with ~17% VAF (Fig. [3m](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). SCIFER detected a leading selected clone at this VAF (Fig. [3n,o](https://www.nature.com/articles/s41588-025-02217-y#Fig3)). Stem cell and selection parameters were similar to the samples above with known CH driver mutations (Fig. [3p](https://www.nature.com/articles/s41588-025-02217-y#Fig3), compare with Fig. [3j–l](https://www.nature.com/articles/s41588-025-02217-y#Fig3)).

In summary, SCIFER robustly identified and quantified neutral evolution and selection in pseudo-bulk WGS data of human HSPCs, without previous knowledge of a driver event.

### SCIFER uncovers clonal complexity in human BM

Next, we generated genuine bulk WGS data from BM HSPCs of 22 humans, aged 30–89 years (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)). Based on targeted sequencing (with VAF ≥ 1% sensitivity; Supplementary Table [3](https://www.nature.com/articles/s41588-025-02217-y#MOESM4))[27](https://www.nature.com/articles/s41588-025-02217-y#ref-CR27 "Jakobsen, N. A. et al. Selective advantage of mutant stem cells in human clonal hematopoiesis is associated with attenuated response to inflammation and aging. Cell Stem Cell 31, 1127–1144 (2024)."), we selected 12 individuals with known CH drivers (6 with _DNMT3A_, 4 _TET2_, 1 _ASXL1_ and 1 with both _TET2_ and _ASXL1_ mutations), and 10 individuals without known drivers. The samples are labeled 1–22, followed by letters for the clones detected (D, T, A or U, for, respectively, mutations in _DNMT3A_, _TET2_ or _ASXL1_, or unknown drivers; N for neutral evolution). If SCIFER detected two clones, the larger clone is indicated first. All individuals had normal blood counts and blood films, without an antecedent history of blood or inflammatory disorder[27](https://www.nature.com/articles/s41588-025-02217-y#ref-CR27 "Jakobsen, N. A. et al. Selective advantage of mutant stem cells in human clonal hematopoiesis is associated with attenuated response to inflammation and aging. Cell Stem Cell 31, 1127–1144 (2024).").

We performed WGS of Lin–CD34+ HSPCs (Extended Data Fig. [4a–f](https://www.nature.com/articles/s41588-025-02217-y#Fig11)) at 90× for all cases, and for 19 cases with sufficient DNA at 270×; hair follicle DNA at 30× WGS served as a germline control. A comparison with WGS data from 14 BM MNC samples is given in Supplementary Note [3](https://www.nature.com/articles/s41588-025-02217-y#MOESM1) and Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02217-y#MOESM1). WGS data were concordant with panel sequencing. Somatic single base-pair substitution profiles indicated similar age-related variant accumulation across all individuals (Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02217-y#Fig12))[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."),[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."),[43](https://www.nature.com/articles/s41588-025-02217-y#ref-CR43 "Osorio, F. G. et al. Somatic mutations reveal lineage relationships and age-related mutagenesis in human hematopoiesis. Cell Rep. 25, 2308–2316. e4 (2018)."). We found no copy number aberrations (CNAs) in driver genes (Extended Data Fig. [6a,b](https://www.nature.com/articles/s41588-025-02217-y#Fig13) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)).

We first applied SCIFER to the 12 individuals with mutations in known CH drivers (in total 14 such mutations, with 2 samples containing two, 17-TT and 12-AT). SCIFER identified a selected clone in all samples (both at 90× and 270×), associated with a shoulder in the VAF histogram (Fig. [4a–c](https://www.nature.com/articles/s41588-025-02217-y#Fig4) and Extended Data Fig. [7a,b](https://www.nature.com/articles/s41588-025-02217-y#Fig14)). In all but one case, the clone detected by SCIFER, without knowledge of a putative driver, agreed with the VAF of the CH driver detected by panel sequencing (Fig. [4d](https://www.nature.com/articles/s41588-025-02217-y#Fig4)). Only for 21-DU, SCIFER did not detect the _DNMT3A_ clone, but instead a subclone without a known CH driver (Extended Data Fig. [7b](https://www.nature.com/articles/s41588-025-02217-y#Fig14)). The _DNMT3A_ clone originated from a founding cell with only six SNVs (Extended Data Fig. [7b](https://www.nature.com/articles/s41588-025-02217-y#Fig14)), likely during embryogenesis. This case indicates a practical detection limit for clones of very early origin. Taken together, SCIFER reliably detected selected clones with known drivers originating in postnatal life.

**Fig. 4: Clonal selection for known CH drivers.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02217-y/figures/4)

**a**, Model fit to the cumulative VAF distribution measured in CD34+ HSPCs of samples 7-T and 10-D with 270× WGS (points and error bars, measured data and their standard deviation, which, assuming Poisson-distributed measurements, is the square root of the measured data; purple area, 95% posterior probabilities of the model fit, estimated from simulations using 100 posterior samples; gray area, 80% credible interval of the clone size, estimated from 1,000 posterior samples; red points and error bars, mean and 95% confidence interval (CI) of the VAF of known CH drivers, based on binomial distributions with sample size and success probability values of 267 and 0.08, and 205 and 0.06, corresponding to read coverage and measured VAF in 7-T and 10-D, respectively). **b**, Model support for clonal selection (conditioned on clones ≥5% VAF) and neutral evolution based on 90× WGS data in 12 cases with selection and with at least one CH driver mutation in _AXSL1_, _DNMT3A_ and _TET2_. Dashed line, 15% selection threshold. **c**, As **b**, but based on 270× bulk WGS data, where available (model support for selection conditioned on clones ≥2% VAF). **d**, Estimated sizes of the selected clones (median and 80% credible intervals, computed from 1,000 posterior samples, for 270× WGS, where available, and 90× WGS else) versus measured VAF of known CH driver (mean and 95% CI according to binomial distributions with sample size taken as read coverage and success probability taken as measured VAF; for the 13 mutations, read coverage and VAFs are as follows: 249 and 0.03, 246 and 0.04, 205 and 0.06, 194 and 0.08, 267 and 0.08, 127 and 0.09, 264 and 0.14, 242 and 0.15, 275 and 0.16, 272 and 0.18, 230 and 0.20, 270 and 0.26, and 260 and 0.28). **e**, As in **a**, but for sample 12-AT (selected clones in blue and green; 95% CIs of the VAFs of mutations in _ASXL1_ and _TET2_ based on binomial distributions with sample size and success probability of 242 and 0.15 and 246 and 0.04, respectively). **f**, As in **b**, but for a second selected clone. **g**, Estimated stem cell parameters for the cases shown in **c**, showing median and 80% credible intervals for each sample, based on 1,000 posterior samples. **h**, As in **g**, but showing the ratio between stem cell number and division rate. y, years.

Applied to the 270× data, SCIFER found second clones in seven of ten samples, associated with a second shoulder in the VAF histogram (Fig. [4e,f](https://www.nature.com/articles/s41588-025-02217-y#Fig4); Extended Data Fig. [7c](https://www.nature.com/articles/s41588-025-02217-y#Fig14)). Two secondary clones were associated with mutations in _TET2_ (included in Fig. [4d](https://www.nature.com/articles/s41588-025-02217-y#Fig4)), whereas five had no known driver (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02217-y#MOESM4) and [Methods](https://www.nature.com/articles/s41588-025-02217-y#Sec12)). Thus, clones without known drivers were abundant and, in all but one case (20-UT), had lower VAF than clones with known drivers.

We next asked whether HSC dynamics varied with mutations in different driver genes. The total number of HSCs, their division rate and the rate of SSNV acquisition were in similar ranges across the three common CH driver genes (Fig. [4g](https://www.nature.com/articles/s41588-025-02217-y#Fig4)). Allowing for _TET2_ mutations to increase the murine HSPC pool[44](https://www.nature.com/articles/s41588-025-02217-y#ref-CR44 "Moran-Crusio, K. et al. Tet2 loss leads to increased hematopoietic stem cell self-renewal and myeloid transformation. Cancer Cell 20, 11–24 (2011)."),[45](https://www.nature.com/articles/s41588-025-02217-y#ref-CR45 "Quivoron, C. et al. TET2 inactivation results in pleiotropic hematopoietic abnormalities in mouse and is a recurrent event during human lymphomagenesis. Cancer Cell 20, 25–38 (2011).") did not change the inferred parameters (Extended Data Fig. [7d–f](https://www.nature.com/articles/s41588-025-02217-y#Fig14)). Moreover, parameter estimates did not depend on sequencing coverage (Fig. [4g](https://www.nature.com/articles/s41588-025-02217-y#Fig4)), or whether the leading-clone or two-clone SCIFER models were used (Extended Data Fig. [7g](https://www.nature.com/articles/s41588-025-02217-y#Fig14)). For 5-DU and 10-D, HSC numbers were inferred with 90× data to be smaller than for the other cases. For 10-D, the selected clone was at the lower end of identifiable VAFs, and deeper sequencing made its stem cell number agree with the majority of cases. The 5-DU parameter estimates, by contrast, were consistent outliers for 90× and 270× and also had a consistently smaller _N_ to _λ_ ratio (Fig. [4h](https://www.nature.com/articles/s41588-025-02217-y#Fig4)). Apart from this case, we inferred that 1.6 × 105 to 1 × 107 HSCs divided 1.4–14 times per year, acquiring 1–5 SSNVs per division. Moreover, the _N_ to _λ_ ratio, giving the timescale over which neutral evolution will cause a variant to reach fixation, ranged from 1.4 × 104 to 22 × 106 years (Fig. [4h](https://www.nature.com/articles/s41588-025-02217-y#Fig4)), confirming that large clones arose by selection, not drift. Taken together, SCIFER identifies up to two selected clones and quantifies the stem cell parameters from bulk WGS data.

### Clonal selection without known CH drivers

Next we studied the ten cases in which targeted sequencing did not detect known CH mutations, at 90× and 270× (Fig. [5a](https://www.nature.com/articles/s41588-025-02217-y#Fig5)) (30–76 years of age).

**Fig. 5: Clonal selection for unknown CH drivers.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02217-y/figures/5)

**a**, Cumulative \\(\\frac{1}{\\rm{VAF}}\\) distributions for samples without a known CH driver at 90× or 270× WGS coverage. **b**, Model support for clonal selection (conditioned on clones with VAF ≥ 5%) and neutral evolution across samples introduced in **a** at 90× WGS. Dashed line, 15% selection threshold. **c**, As in **b**, but for the leading (left) or second selected clone (right) at 270× WGS, where available (posterior probabilities conditioned on clones with VAF ≥ 2%). **d**, Model fit to the cumulative \\(\\frac{1}{\\rm{VAF}}\\) distribution for samples 1-N and 16-UU at 270× WGS (points and error bars, measured data and their standard deviation, which, assuming Poisson-distributed measurements, is the square root of the measured data; purple area, 95% posterior probabilities of the model fit computed from simulations using 100 posterior samples; blue and green areas, 80% credible intervals of the estimated sizes of the selected clones, computed from 1,000 posterior samples). **e**,**f**, Inferred ratio between HSC number and division rate (**e**) and stem cell parameters (**f**) for neutrally evolving samples. Shown are median and 80% credible intervals for each sample, computed from 1,000 posterior samples; estimates obtained from 90× and 270× WGS are shown side by side. **g**,**h**, as in **e** and **f**, but for samples with unknown drivers. **i**,**j**, Estimated number of newly acquired SSNVs per HSC division (**i**) and number of HSCs contributing to hematopoiesis (**j**) in the 4 neutrally evolving cases compared with the 12 cases with selection for a known CH driver (introduced in Fig. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4)) and the 6 cases with selection for an unknown driver (introduced in this figure; points and error bars, median and 80% credible intervals for each sample, estimated from 1,000 posterior samples for 270× WGS data, where available, and 90× WGS data else; boxplots, median and interquartile range, whiskers extend to the largest and smallest value no further than 1.5 times the interquartile range). **k**, Left, as in **j**, but showing the estimated HSC division rate. Right, estimated division rate versus age at sampling. Gray, neutral evolution; red, clonal selection.

At 90×, SCIFER found clear evidence for clonal selection in three cases, and borderline evidence in one sample (3-N; Fig. [5b](https://www.nature.com/articles/s41588-025-02217-y#Fig5) and Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02217-y#Fig15)). Probing at 270× ruled out selection in 3-N (Fig. [5c](https://www.nature.com/articles/s41588-025-02217-y#Fig5)), and identified three additional samples with selection (11-UU, 13-U, 14-U), as well as additional smaller clones in 8-UU, 11-UU and 16-UU (Fig. [5c,d](https://www.nature.com/articles/s41588-025-02217-y#Fig5) and Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02217-y#Fig15)). None of the leading and secondary clones had known drivers (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)). Thus, in six of ten cases without known CH drivers, SCIFER discovered one or more selected clones.

Although all three samples with a large SSNV count (11-UU, 13-U, 16-UU; 500–800 SSNVs) contained selected clones, several samples with one or two selected clones had SSNV counts indistinguishable from those of the four neutrally evolving samples (1-N, 3-N, 4-N, 15-N). This suggests that using bulk SSNV count alone is insufficient to detect CH.

For all neutral samples, SCIFER inferred the _N_\-to-_λ_ ratio to be ~5 × 104 to 106 years, further supporting the notion that clones do not drift to large size in normal human hematopoiesis[15](https://www.nature.com/articles/s41588-025-02217-y#ref-CR15 "Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020).") (Fig. [5e](https://www.nature.com/articles/s41588-025-02217-y#Fig5)). Moreover, we inferred 105 to 3 × 106 HSCs, an HSC division rate of 1–14 times a year and a rate of acquired SSNV per cell division of 1–4 (Fig. [5f](https://www.nature.com/articles/s41588-025-02217-y#Fig5)). These parameter estimates were insensitive to sequencing depth and consistent with our inferences from neutral phylogenetic trees (Fig. [3d–f](https://www.nature.com/articles/s41588-025-02217-y#Fig3)), and from the cases with selection of unknown drivers (Fig. [5g,h](https://www.nature.com/articles/s41588-025-02217-y#Fig5)). However, the division rate of the whole HSC population showed more variation in cases with selection (compare the middle panels in Fig. [5f](https://www.nature.com/articles/s41588-025-02217-y#Fig5) and [h](https://www.nature.com/articles/s41588-025-02217-y#Fig5)). Hence, we compared HSC parameters in all samples with selection (including the cases in Fig. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4)) versus neutrally evolving samples. The rate of SSNV acquisition and HSC numbers were overall very similar (Fig. [5i,j](https://www.nature.com/articles/s41588-025-02217-y#Fig5)). The HSC division rates were more varied, and 11 of 18 individuals with CH had a high inferred division rate (Fig. [5k](https://www.nature.com/articles/s41588-025-02217-y#Fig5)). Hence, HSC division rate appears to be more variable between individuals and, in some cases, clonal selection was associated with high division rate.

### Selection across life

Phylogenetic reconstruction has previously suggested that expanded clones originate before 40 years of age[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). SCIFER estimated that clones typically emerge several decades before BM sampling (Extended Data Fig. [9a,b](https://www.nature.com/articles/s41588-025-02217-y#Fig16)) with no significant differences between clones with and without known drivers (Fig. [6a](https://www.nature.com/articles/s41588-025-02217-y#Fig6)). Secondary (smaller) clones generally emerged later (Fig. [6b](https://www.nature.com/articles/s41588-025-02217-y#Fig6)). The time-averaged clonal growth rates showed substantial inter-individual variation, especially in cases with unknown drivers (Fig. [6c](https://www.nature.com/articles/s41588-025-02217-y#Fig6) and Extended Data Fig. [9c,d](https://www.nature.com/articles/s41588-025-02217-y#Fig16)). Combining all individuals with selection, we found that selected clones were born at approximately constant rate from early childhood to about 50 years of age (Fig. [6d](https://www.nature.com/articles/s41588-025-02217-y#Fig6); the leveling-off at 50 years likely reflects a technical limit of clone detection), suggesting that selection events in HSCs occur uniformly across life.

**Fig. 6: Selection dynamics with and without known drivers.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02217-y/figures/6)

**a**, Estimated age of the selected clone (for the 12 cases introduced in Fig. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4) and the 6 cases introduced in Fig. [5](https://www.nature.com/articles/s41588-025-02217-y#Fig5); points, median; error bars, 80% credible intervals estimated from 1,000 posterior samples; parameters were estimated with the two-clone model from 270× WGS data, where available, and from 90× WGS data else; boxplots, median and interquartile range, whiskers extend to the largest and smallest value no further than 1.5× the interquartile range). **b**, Age of the second selected clone estimated with the two-clone model (for the ten cases introduced in Figs. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4) and [5](https://www.nature.com/articles/s41588-025-02217-y#Fig5); points, median; error bars, 80% credible intervals, estimated from 1,000 posterior samples; boxplots, median and interquartile range, whiskers extend to the largest and smallest value no further than 1.5× the interquartile range). **c**, Estimated growth rates for the 28 selected clones introduced in **a** and **b** (points, median; error bars, 80% credible intervals, estimated from 1,000 posterior samples; estimates were obtained with the two-clone model from 270× WGS data, where available, and from 90× WGS data else; boxplots, median and interquartile range, whiskers extend to the largest and smallest value no further than 1.5× the interquartile range). **d**, Cumulative distribution of estimated age at CH driver acquisition (median); shaded area, lower and upper bounds of the cumulative distribution of estimated age at CH driver acquisition based on 80% credible intervals of the estimated parameters; points and error bars, median and 80% credible intervals for the per-sample estimates, estimated from 1,000 posterior samples. Data are from the 28 selected clones introduced in **a** and **b** (estimates were obtained with the two-clone model from 270× WGS data, where available, and from 90× WGS data else).

### Pervasive selection in the juvenile human brain

We applied SCIFER to bulk WGS data (average coverage 175× ± 70) for 185 human brain samples from 131 individuals aged 4–95 years[17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022)."). This cohort comprised samples from different brain regions (cortex, striatum, hippocampus) from 44 normal (neurotypical) individuals and 87 individuals with neurological disorders (autism spectrum disorder, schizophrenia, Tourette syndrome; Extended Data Fig. [10a](https://www.nature.com/articles/s41588-025-02217-y#Fig17)).

First, we applied SCIFER to two cases (LIBD82 and NC7-CX-OLI) in which brain tumor-initiating driver mutations had been identified, despite no histological indication of malignancy[17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022)."). In LIBD82, trisomy 7 and monosomy 10, characteristic of IDH-wild type glioblastoma[40](https://www.nature.com/articles/s41588-025-02217-y#ref-CR40 "Körber, V. et al. Evolutionary trajectories of IDHWT glioblastomas reveal a common path of early tumorigenesis instigated years ahead of initial diagnosis. Cancer Cell 35, 692–704. e12 (2019)."), were detected in the hippocampus, but not cortex. SCIFER detected a selected clone of a size concordant with the CNA allele frequency (Fig. [7a](https://www.nature.com/articles/s41588-025-02217-y#Fig7)). In NC7-CX-OLI, the R140Q _IDH2_ hotspot mutation associated with _IDH_\-mutant glioma[46](https://www.nature.com/articles/s41588-025-02217-y#ref-CR46 "Buccoliero, A. M. et al. Pediatric high grade glioma classification criteria and molecular features of a case series. Genes 13, 624 (2022)."),[47](https://www.nature.com/articles/s41588-025-02217-y#ref-CR47 "Kurdi, M. et al. The cancer driver genes IDH1 and IDH2 and CD204 in WHO-grade 4 astrocytoma: crosstalk between cancer metabolism and tumour associated macrophage recruitment in tumour microenvironment. Biologics 17, 15–22 (2023).") was detected in cortical oligodendrocytes (NeuN–/Sox10+) and striatal interneurons (NeuN+/CITP2−). In both samples, SCIFER inferred selected clones, consistent with the VAFs of the _IDH2_ mutation (Fig. [7b](https://www.nature.com/articles/s41588-025-02217-y#Fig7)). The striatal clone contained additional cancer-associated driver mutations (_NRAS_ G12D and _DNMT3A_ splice donor variants). Interestingly, SCIFER inferred that the striatal clone was both larger (~15% VAF) and younger (acquired at age 55 years) than the cortical clone (~7.5% VAF and acquired at age 35 years) and had a faster growth rate. Although SCIFER cannot distinguish whether all driver mutations co-occurred in the same cells, or whether different clones with indistinguishable sizes coexisted, the large clone size in striatal interneurons suggests that a subclone with additional driver mutations replaced the ancestral clone. In summary, SCIFER detected selected clones in brain samples associated with known driver mutations.

**Fig. 7: Clonal selection in the human brain.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02217-y/figures/7)

**a**, SCIFER fit to SSNVs measured in the hippocampus of LIBD82 (points and error bars, measured data and standard deviation, which, assuming Poisson-distributed measurements, is the square root of the measured data; purple area, 95% posterior probabilities of the model fit computed from simulations using 100 posterior samples; gray area, 80% credible interval of the clone size, estimated from 1,000 posterior samples; SCZ, schizophrenia). Red point, reported VAF of trisomy 7 and monosomy 10 (ref. [17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022).")). **b**, As in **a**, but for cortical oligodendrocytes and striatal interneurons of NC7 (red points, mean VAF; red lines, 95% CI based on binomial distributions with sample size and success probability of 26 and 0.15 and 29 and 0.07 (NC7-CX-OLI); 30 and 0.1, 33 and 0.06, 32 and 0.22, and 32 and 0.16 (NC7-STR-INT), corresponding to read coverage and measured VAF, respectively)[17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022)."). **c**, As in **a**, but for LIBD87 (cortex) and TS1 (striatum). **d**, Model support for clonal selection (conditioned on clones with ≥5% VAF for <150× WGS and ≥2% VAF for ≥150×) and neutral evolution across 185 brain samples from 131 individuals. Dashed line, 15% selection threshold. **e**, Left, average incidence of clonal selection versus age (summarizing ages into 10 year-bins; in total, 36 of 131 individuals with selection; line and shaded area, LOESS regression with 95% CI). Right, age of individuals with (_n_ = 36) and without selection (_n_ = 95; boxplots, median and interquartile range, whiskers, largest and smallest value no further than 1.5× the interquartile range; _P_ = 0.00002662, Wilcoxon test statistic _W_ = 2,525, two-sided Wilcoxon rank sum test). **f**, Median posterior stem cell parameter values by location (cortex, _n_ = 128; hippocampus, _n_ = 17; striatum, _n_ = 40; boxplots, median and interquartile range, whiskers, largest and smallest value no further than 1.5× the interquartile range; points, outliers; _P_ values, one-way analysis of variance with Kruskal–Wallis test). **g**, Estimated stem cell division rate versus age for 185 samples (median and 80% credible intervals computed from 1,000 posterior samples each; blue line and shaded area, LOESS regression with 95% CI). **h**, Cumulative distribution of estimated age at driver acquisition (median values of 44 samples with selection); lower and upper bounds based on 80% credible intervals (based on 1,000 posterior samples for each sample).

Across the cohort overall, SCIFER detected selected clones in 24% (44 of 185) of all samples (36 of 131 individuals; Fig. [7c,d](https://www.nature.com/articles/s41588-025-02217-y#Fig7)). To independently validate these results, we compared the sizes of selected clones with the VAFs of CNAs reported in the original publication[17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022).") (which are not used by SCIFER). Three individuals harbored duplications on chromosome 2 (AN09412; chr. 2: 96200001–102400000) or chromosome 3 (TS1, chr. 3: 113070000–113240000 and NC6, chr. 3: 60800001–61300000); their VAFs agreed with the selected clone detected by SCIFER, thus corroborating the SCIFER results. The presence of selected clones increased with age (Fig. [7e](https://www.nature.com/articles/s41588-025-02217-y#Fig7)) and was not significantly associated with clinical phenotype, sex or sample location in the brain (based on a generalized linear model taking into account age, location and phenotype; Extended Data Fig. [10b](https://www.nature.com/articles/s41588-025-02217-y#Fig17)). SCIFER inferred ~105 stem cells, in cortex, striatum or hippocampus, that divide on average two or three times per year (Fig. [7f](https://www.nature.com/articles/s41588-025-02217-y#Fig7) and Extended Data Fig. [10c](https://www.nature.com/articles/s41588-025-02217-y#Fig17)). Because human brain consists of hardly renewing neurons and glia cells that renew and have a shorter lifespan[48](https://www.nature.com/articles/s41588-025-02217-y#ref-CR48 "Ernst, A. et al. Neurogenesis in the striatum of the adult human brain. Cell 156, 1072–1083 (2014).") we performed stochastic simulations to mirror this situation. These simulations suggest that VAFs measured in unsorted brain tissue report on the average renewal rate of these two subpopulations (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02217-y#MOESM1)). Interestingly, the division rate of neural stem cells appears to decrease in the first 25 years of life, after which it remains constant (Fig. [7g](https://www.nature.com/articles/s41588-025-02217-y#Fig7)). Between two divisions, they acquired typically fewer than two SSNVs (Extended Data Fig. [10c](https://www.nature.com/articles/s41588-025-02217-y#Fig17)). SCIFER estimates that the vast majority of selected clones began to grow in the first 25 years of life (Fig. [7h](https://www.nature.com/articles/s41588-025-02217-y#Fig7)). Consequently, elderly individuals had older clones (Extended Data Fig. [10d](https://www.nature.com/articles/s41588-025-02217-y#Fig17)). Moreover, the clonal growth rate decreased with age (Extended Data Fig. [10d](https://www.nature.com/articles/s41588-025-02217-y#Fig17)). The focused age incidence of clonal selection in the brain contrasts with the constant rate in hematopoiesis.

Discussion
----------

Here, we present a population genetics approach to detect clonal selection and quantify tissue stem cell dynamics from snapshot WGS data. Akin to recent approaches to cancer evolution[37](https://www.nature.com/articles/s41588-025-02217-y#ref-CR37 "Bozic, I., Gerold, J. M. & Nowak, M. A. Quantifying clonal and subclonal passenger mutations in cancer evolution. PLoS Comput. Biol. 12, e1004731 (2016)."),[38](https://www.nature.com/articles/s41588-025-02217-y#ref-CR38 "Williams, M. J., Werner, B., Barnes, C. P., Graham, T. A. & Sottoriva, A. Identification of neutral tumor evolution across cancer types. Nat. Genet. 48, 238–244 (2016)."),[40](https://www.nature.com/articles/s41588-025-02217-y#ref-CR40 "Körber, V. et al. Evolutionary trajectories of IDHWT glioblastomas reveal a common path of early tumorigenesis instigated years ahead of initial diagnosis. Cancer Cell 35, 692–704. e12 (2019)."),[49](https://www.nature.com/articles/s41588-025-02217-y#ref-CR49 "Williams, M. J. et al. Quantification of subclonal selection in cancer from bulk sequencing data. Nat. Genet. 50, 895–903 (2018)."),[50](https://www.nature.com/articles/s41588-025-02217-y#ref-CR50 "Körber, V. et al. Neuroblastoma arises in early fetal development and its evolutionary duration predicts outcome. Nat. Genet. 55, 619–630 (2023)."),[51](https://www.nature.com/articles/s41588-025-02217-y#ref-CR51 "Gabbutt, C. et al. Fluctuating methylation clocks for cell lineage tracing at high temporal resolution in human tissues. Nat. Biotechnol. 40, 720–730 (2022)."), our theory delineates how neutral evolution versus selection shape the somatic VAF spectra. Compared with previous work in tumors[38](https://www.nature.com/articles/s41588-025-02217-y#ref-CR38 "Williams, M. J., Werner, B., Barnes, C. P., Graham, T. A. & Sottoriva, A. Identification of neutral tumor evolution across cancer types. Nat. Genet. 48, 238–244 (2016)."),[49](https://www.nature.com/articles/s41588-025-02217-y#ref-CR49 "Williams, M. J. et al. Quantification of subclonal selection in cancer from bulk sequencing data. Nat. Genet. 50, 895–903 (2018)."), SCIFER explicitly treats genetic drift and its interplay with clonal selection, which is key for nonmalignant, homeostatic tissues. SCIFER identifies selection without knowledge of the underlying driver, detects whether more than one selected clone is present, and quantifies their age and selective advantage(s).

At 270×, SCIFER has a sensitivity of detecting clones at 2% VAF in a single bulk sample at a fraction of the cost and effort of single-cell colony-derived phylogenetic reconstruction. Recently, analysis of bulk WGS of blood samples aimed to identify CH, assuming selection increases total detected somatic variants[52](https://www.nature.com/articles/s41588-025-02217-y#ref-CR52 "Weinstock, J. S. et al. Aberrant activation of TCL1A promotes stem cell expansion in clonal haematopoiesis. Nature 616, 755–763 (2023)."). Here, we found that this is not necessarily the case and hence would like to caution against using large SSNV count in a WGS sample as a sign of selection.

Agreement of HSC parameters inferred from one phylogenetic tree with neutral evolution[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018).") and SCIFER from the 22 BM samples studied here is remarkable, because SCIFER strongly relies on variants with higher VAF emerging relatively early, whereas the phylodynamic inference uses low-VAF SSNVs generated later. This may indicate that HSC dynamics are rather uniform across time. Nevertheless, it is important to note that our inferred parameters are time-averaged. Because analysis of driver VAF in serial blood samples suggests that expansion rates may change over time[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."), it would be valuable to apply SCIFER to serial samples to quantify stem cell dynamics over time.

Our parameter inference suggests variation in HSC division rate between individuals. Indeed, mean telomere length has been shown to decrease faster in individuals with CH[53](https://www.nature.com/articles/s41588-025-02217-y#ref-CR53 "Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. Sci. Adv. 8, eabl6579 (2022)."), and longer telomere length is a causative risk factor for CH[54](https://www.nature.com/articles/s41588-025-02217-y#ref-CR54 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[55](https://www.nature.com/articles/s41588-025-02217-y#ref-CR55 "DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. N. Engl. J. Med. 388, 2422–2433 (2023)."), suggesting that long telomeres allow CH HSCs to undergo enhanced cell division. It should be kept in mind that our parameter estimates rely on the assumption of a constant mutation rate in HSCs along the human lifespan. During adult life, HSC mutation rate appears constant[56](https://www.nature.com/articles/s41588-025-02217-y#ref-CR56 "Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. Nature 593, 405–410 (2021)."); whether this is true in fetal development is unclear.

Consistent with previous studies[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."),[14](https://www.nature.com/articles/s41588-025-02217-y#ref-CR14 "Poon, G. Y., Watson, C. J., Fisher, D. S. & Blundell, J. R. Synonymous mutations reveal genome-wide levels of positive selection in healthy tissues. Nat. Genet. 53, 1597–1605 (2021)."), we identified pervasive clonal selection in the absence of known drivers. However, in three cases selected clones were at the limit of detection, and were not found in the 90× data. Such clones just below the detection limit may affect the measured VAF distribution at low frequencies, and bias SCIFER to show an erroneously increased drift effect. This potential problem can be addressed by deeper sequencing, and potentially extending SCIFER to more than two clones (for which the online code provides an option).

Although the statistical power of the current study is limited, we observed several structural variants (SVs) previously described as recurrent somatic mutations[57](https://www.nature.com/articles/s41588-025-02217-y#ref-CR57 "Vattathil, S. & Scheet, P. Extensive hidden genomic mosaicism revealed in normal tissue. Am. J. Hum. Genet. 98, 571–578 (2016)."),[58](https://www.nature.com/articles/s41588-025-02217-y#ref-CR58 "Laurie, C. C. et al. Detectable clonal mosaicism from birth to old age and its relationship to cancer. Nat. Genet. 44, 642–650 (2012)."), associated with increased risk for hematological cancer[21](https://www.nature.com/articles/s41588-025-02217-y#ref-CR21 "Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."),[58](https://www.nature.com/articles/s41588-025-02217-y#ref-CR58 "Laurie, C. C. et al. Detectable clonal mosaicism from birth to old age and its relationship to cancer. Nat. Genet. 44, 642–650 (2012).") (Supplementary Tables [6](https://www.nature.com/articles/s41588-025-02217-y#MOESM4) and [7](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)). Moreover, in-depth analyses of the genome-wide SSNV profiles revealed potential candidates for driver mutations (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)), one of which had significantly higher ratios of nonsynonymous-to-synonymous SSNVs in ~200,000 UK Biobank blood samples[25](https://www.nature.com/articles/s41588-025-02217-y#ref-CR25 "Bernstein, N. et al. Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis. Nat. Genet. 56, 1147–1155 (2024)."). By applying SCIFER to large cohorts it may be possible to more accurately determine the frequency of individuals with unknown driver events, and their clinical phenotype.

Applying SCIFER to brain tissue, we found clonal selection in ~25% of the samples. The prevalence of unknown drivers was even more pronounced than in hematopoiesis. Moreover, in two individuals, we found pre-malignant selected clones with characteristic brain tumor-initiating driver mutations without histological evidence of malignancy. This raises the question of how frequent pre-malignancy in the human brain is and whether the situation is similar to blood[19](https://www.nature.com/articles/s41588-025-02217-y#ref-CR19 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[20](https://www.nature.com/articles/s41588-025-02217-y#ref-CR20 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."). Selected clones in the brain were largely born in the first 25 years of life, coinciding with the period of extensive postnatal brain development[59](https://www.nature.com/articles/s41588-025-02217-y#ref-CR59 "Lebel, C., Walker, L., Leemans, A., Phillips, L. & Beaulieu, C. Microstructural maturation of the human brain from childhood to adulthood. Neuroimage 40, 1044–1055 (2008)."). Brain cell-type heterogeneity (neurons and glia cells) precludes a more accurate quantitation of the stem cell dynamics, which will require analysis of purified cell populations. Nevertheless, stochastic simulations suggest that VAFs in tissues composed of heterogeneous cell types reflect on average cell turnover (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02217-y#MOESM1)). We envisage that SCIFER will be useful to study clonal selection in other solid organs in humans.

Methods
-------

### Ethical approval

Patient samples were collected with informed consent under the Mechanisms of Age-Related Clonal Haematopoiesis (MARCH) Study. Written informed consent was obtained from all participants in accordance with the Declaration of Helsinki. This study was approved by the Yorkshire & The Humber – Bradford Leeds Research Ethics Committee (REC Ref17:/YH/0382).

### Study samples

Participants were recruited from individuals undergoing elective total hip replacement surgery at the Nuffield Orthopaedic Centre, Oxford. Exclusion criteria were history of rheumatoid arthritis or other inflammatory arthritis, history of septic arthritis in the limb undergoing surgery, history of hematological cancer, bisphosphonate use and oral steroid use. Patient characteristics are summarized in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02217-y#MOESM4). At the time of surgery, trabecular bone fragments and BM aspirates were obtained from the femoral canal and collected in anticoagulated buffer containing acid–citrate–dextrose, heparin sodium and DNase. Samples of peripheral blood were collected in EDTA vacutainers. Hair follicle samples were collected from participants as a germline control. Peripheral blood and BM mononuclear cells (MNCs) were isolated by Ficoll density gradient centrifugation and viably frozen. Peripheral blood granulocytic cell pellets were frozen for later DNA extraction. Genomic DNA was extracted from BM MNCs, peripheral blood granulocytes and hair follicles using a DNeasy Blood & Tissue Kit (Qiagen).

### Cell sorting

Thawing media was prepared with IMDM medium (Gibco) supplemented with 20% fetal bovine serum (FBS) and 110 µg ml−1 DNase. BM samples were thawed at 37 °C in a water bath, 1 ml of warm FBS was added and the suspension was then diluted by dropwise addition of 8 ml of thawing media. The suspension was centrifuged at 400 _g_ for 10 min, cells were resuspended in flow cytometry staining medium (IMDM with 10% FBS and 10 μg ml−1 DNase), filtered through a 35-μm cell strainer and placed on ice.

Cells were stained with the following antibodies: anti-CD34-PE (1:160, BioLegend, clone 581), anti-CD3-PE/Cy7 (1:100, BioLegend, clone HIT3a), anti-CD2-PE/Cy5 (1:160, BioLegend, clone RPA-2.10), anti-CD4-PE/Cy5 (1:160, BioLegend, clone RPA-T4), anti-CD7-PE/Cy5 (1:160, BioLegend, clone CD7-6B7), anti-CD8a-PE/Cy5 (1:320, BioLegend, clone RPA-T8), anti-CD11b-PE/Cy5 (1:160, BioLegend, clone ICRF44), anti-CD14-PE/Cy5 (1:160, eBioscience, clone 61D3), anti-CD19-PE/Cy5 (1:160, BioLegend, clone HIB19), anti-CD20-PE/Cy5 (1:160, BioLegend, clone 2H7), anti-CD56-PE/Cy5 (1:80, BioLegend, clone MEM188) and anti-CD235ab-PE/Cy5 (1:320, BioLegend, clone HIR2). Following antibody incubations, cells were washed with 1 ml of flow cytometry staining buffer, centrifuged at 350 _g_ for 5 min and resuspended in flow cytometry staining buffer containing 1:10,000 Hoechst 33342 live–dead stain.

Cell sorting was performed on a BD FACSAria Fusion or Sony MA900 equipped with a 100-µm nozzle or sorting chip. Unstained, single-stained and fluorescence-minus-one controls were used to determine background staining and compensation in each channel. Doublets and dead cells were excluded. The following populations were sorted with a mean purity >95%: Lin–CD34+ HSPCs, Lin+CD3+ T cells and Lin+/–CD34–CD3– MNCs. Genomic DNA from sorted cell populations was extracted using a QIAamp DNA Micro Kit (Qiagen).

### Whole-genome sequencing

#### Library preparation and sequencing

Sequencing libraries were prepared using the Illumina DNA Prep Kit (cat. no. 20018704) and Nextera DNA CD Indexes (cat. no. 20018707) according to manufacturer’s guidelines. The Qubit HS DNA Assay Kit (Invitrogen, cat. no. Q32854) and a tape station run with the Agilent HS D5000 Assay Kit (cat. no. 5067-5593) were used for quality control. Thereafter, libraries were diluted to 10 nM, pooled and sequenced on either HiSeq X PE150 or on NovaSeq 6000 PE150.

Low-quality bases on raw sequencing reads were trimmed using Trim Galore v.0.5.0 ([https://www.bioinformatics.babraham.ac.uk/projects/trim\_galore/](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/)) with cutadapt v.2.8 (ref. [60](https://www.nature.com/articles/s41588-025-02217-y#ref-CR60 "Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J 17, 10–12 (2011).")) with the following settings: --quality 30, --illumina --length 32 --trim-n --clir\_R1 2 --clip\_R2 2 --three\_prime\_clip\_R1 2 --three\_prime\_clip\_R2 2. Trimmed read pairs were mapped to the human reference genome (build 37 with the GRCh37.75 genome annotation) using bwa mem v.0.7.12 (ref. [61](https://www.nature.com/articles/s41588-025-02217-y#ref-CR61 "Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009).")). Mapped reads were coordinate-sorted with samtools sort v.1.5 (ref. [61](https://www.nature.com/articles/s41588-025-02217-y#ref-CR61 "Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009).")), followed by marking duplicate read pairs with gatk MarkDuplicates v.4.0.9.0 (ref. [62](https://www.nature.com/articles/s41588-025-02217-y#ref-CR62 "Van der Auwera, G.A. & O’Connor, B.D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra (O’Reilly Media, 2020).")) and indexing with samtools index.

#### Detection of SSNVs and insertions–deletions

Somatic SNVs and small insertions–deletions (indels) in BM and blood samples were called with Strelka v.2.9.2 (ref. [63](https://www.nature.com/articles/s41588-025-02217-y#ref-CR63 "Kim, S. et al. Strelka2: fast and accurate calling of germline and somatic variants. Nat. Methods 15, 591–594 (2018).")) and Mutect2, GATK v.4.2.0.0 (ref. [62](https://www.nature.com/articles/s41588-025-02217-y#ref-CR62 "Van der Auwera, G.A. & O’Connor, B.D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra (O’Reilly Media, 2020).")), using the matched hair follicle samples as the germline control. Variants in repeat regions and simple repeat regions (downloaded from UCSC table browser, setting the assembly to hg19, the track to ‘RepeatMasker’ or ‘Simple Repeats’; accession date: 5 December 2018) were filtered using bedtools intersect v.2.24.0 (ref. [64](https://www.nature.com/articles/s41588-025-02217-y#ref-CR64 "Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010).")). Only variants that passed the default filters of both Strelka and Mutect2 were retained. To identify remaining germline variants, variants were looked up in dbSNP (v.150) and the population frequency of reported variants was annotated based on the Genome Aggregation Database (gnomAD v.2.1.1; [https://gnomad.broadinstitute.org/](https://gnomad.broadinstitute.org/)). Variants were then filtered to exclude potential contamination of germline variants based on the global allele frequency (AF) of gnomAD (retaining variants with AF < 0.001). In a few cases, a CH driver was identified by panel sequencing at low VAF, but not recovered by Strelka and Mutect; here, we re-examined the respective position using bcftools mpileup v.1.10.2 (ref. [65](https://www.nature.com/articles/s41588-025-02217-y#ref-CR65 "Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 10, giab008 (2021).")). All variants were annotated with ANNOVAR (v.May2018; [http://annovar.openbioinformatics.org/](http://annovar.openbioinformatics.org/))[66](https://www.nature.com/articles/s41588-025-02217-y#ref-CR66 "Wang, K., Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 38, e164–e164 (2010).") according to the human genome build v.19. For SSNVs, VAFs were recalculated directly from the BAM files using alleleCounter v.4.0.2 ([https://github.com/cancerit/alleleCount](https://github.com/cancerit/alleleCount); with default base and mapping quality thresholds) and MaC ([https://github.com/nansari-pour/MaC](https://github.com/nansari-pour/MaC)).

#### Detection of copy number variants

Genome-wide subclonal CNAs were identified with Battenberg (v.2.2.10; [https://github.com/Wedge-lab/battenberg](https://github.com/Wedge-lab/battenberg)), which has been described in detail previously[67](https://www.nature.com/articles/s41588-025-02217-y#ref-CR67 "Nik-Zainal, S. et al. The life history of 21 breast cancers. Cell 149, 994–1007 (2012).").

#### Detection of structural variants

SVs from whole-genome data were identified using Manta v.1.6.0 (ref. [68](https://www.nature.com/articles/s41588-025-02217-y#ref-CR68 "Chen, X. et al. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. Bioinformatics 32, 1220–1222 (2016).")) with default tumor–normal pair settings comparing samples with the germline controls. We kept variants that passed Manta’s default filter, had a minimal SOMATICSCORE of 30, were not classified as IMPRECISE, had at least three variants and a VAF of 5% in the blood sample, and at most two variant reads and VAF < 5% in the control sample. SVs previously described in somatic tissues[57](https://www.nature.com/articles/s41588-025-02217-y#ref-CR57 "Vattathil, S. & Scheet, P. Extensive hidden genomic mosaicism revealed in normal tissue. Am. J. Hum. Genet. 98, 571–578 (2016)."),[58](https://www.nature.com/articles/s41588-025-02217-y#ref-CR58 "Laurie, C. C. et al. Detectable clonal mosaicism from birth to old age and its relationship to cancer. Nat. Genet. 44, 642–650 (2012).") and associated with hematological cancer[21](https://www.nature.com/articles/s41588-025-02217-y#ref-CR21 "Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."),[58](https://www.nature.com/articles/s41588-025-02217-y#ref-CR58 "Laurie, C. C. et al. Detectable clonal mosaicism from birth to old age and its relationship to cancer. Nat. Genet. 44, 642–650 (2012).") were classified as putative drivers (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)).

#### Driver analysis

To comprehensively search for candidate driver mutations, we concatenated published lists of putative driver genes in CH and leukemia from Intogen[69](https://www.nature.com/articles/s41588-025-02217-y#ref-CR69 "Gonzalez-Perez, A. et al. IntOGen-mutations identifies cancer drivers across tumor types. Nat. Methods 10, 1081–1082 (2013).") (subsetting genes described in ‘ALL’, ‘AML’, ‘CLL’, ‘CML’ or ‘MM’ from release 2020.02.01, and the CH gene list from ref. [70](https://www.nature.com/articles/s41588-025-02217-y#ref-CR70 "Pich, O., Reyes-Salazar, I., Gonzalez-Perez, A. & Lopez-Bigas, N. Discovering the drivers of clonal hematopoiesis. Nat. Commun. 13, 4267 (2022).")), Cosmic[71](https://www.nature.com/articles/s41588-025-02217-y#ref-CR71 "Sondka, Z. et al. The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Nat. Rev. Cancer 18, 696–705 (2018).") v.94 (subsetting on genes with annotation in leukemia), ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."), and a curated list of CH-associated mutations compiled from five large studies[19](https://www.nature.com/articles/s41588-025-02217-y#ref-CR19 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[20](https://www.nature.com/articles/s41588-025-02217-y#ref-CR20 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[27](https://www.nature.com/articles/s41588-025-02217-y#ref-CR27 "Jakobsen, N. A. et al. Selective advantage of mutant stem cells in human clonal hematopoiesis is associated with attenuated response to inflammation and aging. Cell Stem Cell 31, 1127–1144 (2024)."),[30](https://www.nature.com/articles/s41588-025-02217-y#ref-CR30 "Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. Nature 559, 400–404 (2018)."),[72](https://www.nature.com/articles/s41588-025-02217-y#ref-CR72 "Acuna-Hidalgo, R. et al. Ultra-sensitive sequencing identifies high prevalence of clonal hematopoiesis-associated mutations throughout adult life. Am. J. Hum. Genet. 101, 50–64 (2017)."),[73](https://www.nature.com/articles/s41588-025-02217-y#ref-CR73 "Desai, P. et al. Somatic mutations precede acute myeloid leukemia years before diagnosis. Nat. Med. 24, 1015–1023 (2018).").

Mutations identified by Strelka and targeting any of these genes were looked up in ClinVar[74](https://www.nature.com/articles/s41588-025-02217-y#ref-CR74 "Landrum, M. J. et al. ClinVar: improving access to variant interpretations and supporting evidence. Nucleic Acids Res. 46, D1062–D1067 (2018).") (v.20221231) and annotated accordingly. Using the R package drawProteins[75](https://www.nature.com/articles/s41588-025-02217-y#ref-CR75 "Brennan, P. drawProteins: a Bioconductor/R package for reproducible and programmatic generation of protein schematics. F1000Research 7, 1105 (2018).") v.1.18.0 we collected information on the protein domain targeted by each variant. Moreover, we manually annotated variant information (involvement in disease, functional evidence for mutations at this site, SNP identifier (SNPID), association with genetic disorders and whether the variant targets a functional domain) using the manually curated variant information ‘homo\_sapiens\_variation.txt’ from Uniprot[76](https://www.nature.com/articles/s41588-025-02217-y#ref-CR76 "Consortium, U. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47, D506–D515 (2019).") (downloaded 4 April 2023) and variant information provided on [www.uniprot.org](http://www.uniprot.org/). Thereafter, we computed SIFT[77](https://www.nature.com/articles/s41588-025-02217-y#ref-CR77 "Ng, P. C. & Henikoff, S. SIFT: predicting amino acid changes that affect protein function. Nucleic Acids Res. 31, 3812–3814 (2003).") and Revel[78](https://www.nature.com/articles/s41588-025-02217-y#ref-CR78 "Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Am. J. Hum. Genet. 99, 877–885 (2016).") prediction scores for each substitution. We kept variants causing a ‘stopgain’, ‘stoploss’, ‘frameshift\_insertion’ or ‘frameshift\_deletion’, or with ’Conflicting interpretations of pathogenicity’, ‘Likely pathogenic’, ‘Pathogenic/Likely pathogenic’ or of ‘uncertain significance’ according to ClinVar, or annotated to a related disease (based on uniprot.org) or annotated as pathogenic (uniprot.org) or with a REVEL score ≥0.75 or targeting either _ASXL1_, or _DNMT3A_ or _TET2_.

This yielded a set of variant positions, which we merged with curated positions in known CH drivers[27](https://www.nature.com/articles/s41588-025-02217-y#ref-CR27 "Jakobsen, N. A. et al. Selective advantage of mutant stem cells in human clonal hematopoiesis is associated with attenuated response to inflammation and aging. Cell Stem Cell 31, 1127–1144 (2024)."). Based on this set, we re-called variants in all blood and control samples with bcftools mpileup[65](https://www.nature.com/articles/s41588-025-02217-y#ref-CR65 "Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 10, giab008 (2021).") v.1.10.2, using the option -p, and bcftools call, using the option -mA. Variant calls were performed in batches and merged using bcftools merge. In the final set, we kept variants targeting any of _ASXL1_, _DNMT3A_ or _TET2_ and variants that were found on at least three reads in the sample and on fewer than five reads in the controls.

### Analysis of single-cell WGS data

We tested our population genetics model on published single-cell WGS data from refs. [2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."),[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). To this end, we computed pseudo-bulk VAFs from the single-cell phylogenies as

${\\text{VAF}}=\\frac{{n}\_{\\text{variant cells}}}{2{n}\_{\\text{cells}}}$

where _n_variant cells is the number of cells harboring a variant and _n_cells is the total number of sequenced cells; the factor 2 accounts for diploidy.

#### Parameter estimation

We fit our population genetics model to the cumulative VAF distribution truncated at 0.01, as detailed below and using the prior probabilities outlined in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02217-y#MOESM4).

#### Reanalysis of SSNV and indels

To assess differences in variant calling pipelines, we re-called SSNVs and indels in the data from ref. [2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."). To this end, we intersected the results between Mutect2 (ref. [62](https://www.nature.com/articles/s41588-025-02217-y#ref-CR62 "Van der Auwera, G.A. & O’Connor, B.D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra (O’Reilly Media, 2020).")) and Strelka2 (ref. [63](https://www.nature.com/articles/s41588-025-02217-y#ref-CR63 "Kim, S. et al. Strelka2: fast and accurate calling of germline and somatic variants. Nat. Methods 15, 591–594 (2018).")), filtered remaining germline variants using gnomAD (retaining variants with AF <0.001) and recalculated VAFs directly from the BAM files using alleleCounter v.4.0.2 ([https://github.com/cancerit/alleleCount](https://github.com/cancerit/alleleCount); with default base and mapping quality thresholds) and MaC ([https://github.com/nansari-pour/MaC](https://github.com/nansari-pour/MaC)). We filtered the remaining variants as stated in ref. [2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."). In brief, we removed variants occurring in >120 of the 140 colonies, variants that fell within 10 bp of each other and variants with a coverage <6 on autosomes or <3 on sex chromosomes in more than five samples. Moreover, we retained only variants with a mean VAF > 0.3 across all samples with at least one mutant read. Finally, we excluded sites at which >10% of the samples with at least one mutant read had a VAF < 0.1.

#### Phylogenetic reconstruction

We reconstructed single-cell phylogenies based on the re-called SSNVs and indels from ref. [2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018).") by converting the mutation table into a fasta file, learning the phylogenetic tree using MPBoot v.1.1.0 (ref. [79](https://www.nature.com/articles/s41588-025-02217-y#ref-CR79 "Hoang, D. T. et al. MPBoot: fast phylogenetic maximum parsimony tree inference and bootstrap approximation. BMC Evol. Biol. 18, 11 (2018).")) and plotting the tree using custom scripts from ref. [13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).") ([https://github.com/emily-mitchell/normal\_haematopoiesis/](https://github.com/emily-mitchell/normal_haematopoiesis/)).

### Analysis of brain samples

SSNVs of 457 human brain samples were downloaded from ref. [17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022)."). Of these, we analyzed 177 samples with an average coverage >100×, as well as 8 samples from individual NC7 (NC7-CX-ASTMIG, NC7-CX-INT, NC7-CX-OLI, NC7-CX-PYR, NC7-STR-ASTMIG, NC7-STR-INT, NC7-STR-MSN and NC7-STR-OLI) that contained multiple known driver mutations but had average coverage between 32× and 40× (Supplementary Table [8](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)). Both tier1 and tier2 variant calls were used for analysis.

### Population genetics model

#### Theory

We modeled the evolution of VAFs mechanistically, accounting for accumulation, drift and selection of somatic variants in a homeostatic tissue. The model is parametrized with the rate at which HSCs divide during adulthood, _λ_, the number of SSNVs acquired between two cell divisions, _μ_, the number of HSCs during adulthood, _N_ss, as well as the time of origin of the selected clone, _t_s, and its selective advantage expressed by _r_. We bundled the model functions in an R package, SCIFER, available on [https://github.com/VerenaK90/SCIFER](https://github.com/VerenaK90/SCIFER).

### Modeling the site frequency spectrum of somatic variants generated by neutral evolution

The time-dependent site frequency spectrum _S__i_(_t_) gives the number of variants with clone size _i_, where _i_ ranges from some minimally observable clone size (owing to WGS sequencing depth) up to the total number of stem cells _N_(_t_). We derive analytical expressions for the site frequency spectra (SFS) resulting from neutral evolution and clonal selection and compare these with measured VAF histograms.

To begin with neutral evolution, we develop a stochastic model for accumulation and drift of neutral somatic variants during developmental expansion and subsequent homeostasis of the stem cell pool. The model assumes that stem cells proliferate via symmetric self-renewing divisions, with rate _λ_, and are lost by differentiation and cell death, with rate _δ_. Between two subsequent stem cell divisions, on average _μ_ new variants are introduced. These variants are inherited to daughter cells and, depending on the dynamics of the corresponding stem cell clone, may either go extinct or drift to variable frequencies. The SFS generated by these dynamics is:

${S}\_{i}\\left(t\\right)=\\mathop{\\int}\\limits\_{0}^{t}\\lambda \\mu N\\left({t}^{{\\prime} }\\right){P}\_{1,i}(t,t{\\prime} ){\\rm{d}}{t}^{{\\prime} }$

describing the generation of new variants at time _t_′, in a population of size _N_(_t_′), and their drift to clone size _i_ up to the time of measurement _t_ with probability _P_1,_i_(_t_,_t_′). To compare _S_i(_t_) with measured VAF histograms, we transform from clone size to VAF:

${\\text{VAF}}=\\frac{i}{2N}$

To describe developmental expansion of the stem cell pool followed by homeostasis, we concatenated linear birth–death processes. During development, the division rate will exceed the loss rate, _λ_exp > _δ_exp, hence defining a supercritical birth–death process. At time _t_1, the system reaches its steady-state with a constant number of active stem cells, _N_ss. The cellular dynamics are now appropriately described by a critical birth–death process with steady-state rate _λ_ss = _δ_ss.

To compute the SFS during developmental expansion, we consider the probability that a cell acquiring a new variant will expand to a clone of size _a_ in time _t_ (ref. [80](https://www.nature.com/articles/s41588-025-02217-y#ref-CR80 "Bailey, N. The Elements of Stochastic Processes (Wiley, 1964).")):

${P}\_{\\exp ,1,a}\\left(t\\right)=\\left\\{\\begin{array}{c}x(t),\\text{if}\\;{a}=0\\\\ \\left(1-x\\left(t\\right)\\right)\\left(1-y\\left(t\\right)\\right){y\\left(t\\right)}^{a-1},\\text{if}\\;a\\ge 1\\end{array}\\right.$

(1)

with

${{x}}\\left({{t}}\\right)=\\displaystyle\\frac{{\\delta }\_{\\exp }{{\\rm{e}}}^{({\\lambda }\_{\\exp }-{\\delta }\_{\\exp }){{t}}}-{\\delta }\_{\\exp }}{{\\lambda }\_{\\exp }{{\\rm{e}}}^{({\\lambda }\_{\\exp }-{\\delta }\_{\\exp }){{t}}}-{\\delta }\_{\\exp }},{{y}}\\left({{t}}\\right)=\\displaystyle\\frac{{\\lambda }\_{\\exp }{{{e}}}^{({\\lambda }\_{\\exp }-{\\delta }\_{\\exp }){{t}}}-{\\lambda }\_{\\exp }}{{\\lambda }\_{\\exp }{{{e}}}^{({\\lambda }\_{\\exp }-{\\delta }\_{\\exp }){{t}}}-{\\delta }\_{\\exp }}$

(2)

The measured VAF histograms report on the number of variants with a given frequency. To calculate variant number in the model, we note that between time _t_′ and _t_′ + d_t_′ on average _μ__λ_exp_N_(_t_′)d_t_′ variants are generated. Hence,

${\\mu {\\lambda }\_{\\exp }N\\left({t}^{{\\prime} }\\right)\\text{d}t{\\prime} \\times P}\_{\\exp ,1,a}\\left(t-{t}^{{\\prime} }\\right)$

(3)

variants introduced at time _t_′ each occur in _a_ cells at time _t_.

During tissue homeostasis, when stem cell division and loss will occur both at steady-state rate _λ_ss, drift is described by the critical birth–death process. A variant in _a_ cells at _t_1 (when the homeostatic stem cell number is reached) will drift to occur in _b_ cells within time _t_ with probability[80](https://www.nature.com/articles/s41588-025-02217-y#ref-CR80 "Bailey, N. The Elements of Stochastic Processes (Wiley, 1964).")

${P}\_{\\text{ss},a,b}(t)=\\left\\{\\begin{array}{c}{p(t)}^{a},{\\text{if}}\\,b=0\\\\ \\mathop{\\sum }\\limits\_{j=0}^{{{b}}}\\frac{j}{b}{{a}\\choose{j}}{p\\left(t\\right)}^{a-j}{\\left(1-p(t)\\right)\\;}^{j}{{b}\\choose{j}}{p(t)}^{b-j}{\\left(1-p(t)\\right)}{j\\atop},{\\text{otherwise}}\\end{array}\\right.$

(4)

with

$p\\left(t\\right)=\\frac{{\\lambda }\_{\\text{ss}}t}{1+{\\lambda }\_{\\text{ss}}t}$

(5)

Thus, at _t_ ≥ _t_1, the number of variants occurring exactly in _b_ cells is

$\\mathop{\\sum }\\limits\_{a=1}^{{N}\_{\\text{ss}}}{\\mu {\\lambda }\_{\\exp }N({t}^{{\\prime} })P}\_{\\exp ,1,a}\\left({t}\_{1}-{t}^{{\\prime} }\\right){P}\_{\\text{ss},a,b}\\left(t-{t}\_{1}\\right)\\text{d}{t}^{{\\prime} }$

(6)

for variants generated during developmental expansion (_t_′ < _t_1).

Finally, we consider variants acquired during homeostasis, which evolve according to the critical birth–death process entirely. Hence, the number of such variants occurring exactly in _b_ cells is

$\\mu {\\lambda }\_{\\text{ss}}{N}\_{\\text{ss}}\\times {P}\_{\\text{ss},1,b}\\left(t-t{\\prime} \\right)\\text{d}{t}^{{\\prime} }$

(7)

where _t_′ ≥ _t_1. Combining the contribution of both phases, we arrive at the SFS of neutral variants in a homeostatic tissue without selection:

$\\begin{array}{l}{S}\_{i}\\left(t\\right)=\\underbrace{{\\displaystyle\\int }\_{\\!0}^{{t}\_{1}}\\mathop{\\sum }\\limits\_{a=1}^{{N}\_{\\text{ss}}}{\\mu {\\lambda }\_{\\exp }N\\left({t}^{{\\prime} }\\right)P}\_{\\exp ,1,a}\\left({t}\_{1}-{t}^{{\\prime} }\\right){P}\_{\\text{ss},a,i}\\left(t-{t}\_{1}\\right)\\text{d}{t}^{{\\prime} }}\_{{\\rm{variants}}\\; {\\rm{generated}}\\; {\\rm{in}}\\; {\\rm{development}}}\\\\\\quad\\quad+\\underbrace{{\\displaystyle\\int }\_{\\!{t}\_{1}}^{t}\\mu {\\lambda }\_{\\text{ss}}{N}\_{\\text{ss}}{P}\_{\\text{ss},1,i}\\left(t-{t}^{{\\prime} }\\right)\\text{d}{t}^{{\\prime} }}\_{{\\rm{variants}}\\; {\\rm{generated}}\\; {\\rm{in}}\\; {\\rm{homeostasis}}}\\end{array}$

(8)

Equation ([8](https://www.nature.com/articles/s41588-025-02217-y#Equ8)) generalizes a result by Ohtsuki and Innan[81](https://www.nature.com/articles/s41588-025-02217-y#ref-CR81 "Ohtsuki, H. & Innan, H. Forward and backward evolutionary processes and allele frequency spectrum in a cancer cell population. Theor. Popul. Biol. 117, 43–50 (2017).") for expanding tissues.

### Modeling the site frequency spectrum of somatic variants under selection

**Modeling a single selected clone in a homeostatic tissue.** Clonal selection will modify the SFS. Consider that a positively selected mutation is acquired at time _t_s and reduces the rate of stem cell loss by the factor _r_ < 1 (the alternative for imparting selective advantage, an increase in the rate of stem cell division, yields very similar results). The cell number of mutated stem cells, _n_2(_t_), expands at the expense of the number of normal stem cells, _n_1(_t_) = _N_ss − _n_2(_t_), according to the competition model

$\\begin{array}{c}\\frac{{\\rm{d}}{n}\_{1}}{{{\\rm{d}}t}}={\\lambda }\_{\\text{ss}}{n}\_{1}\\left(t\\right)\\left(1-{\\rho }\_{{n}\_{1},{n}\_{1}}\\frac{{n}\_{1}(t)}{{N}\_{\\text{ss}}}-{\\rho }\_{{n}\_{1},{n}\_{2}}\\frac{{n}\_{2}\\left(t\\right)}{{N}\_{\\text{ss}}}\\right)=g({n}\_{1},{n}\_{2}),\\\\ \\frac{{\\rm{d}}{n}\_{2}}{{{\\rm{d}}t}}={\\lambda }\_{\\text{ss}}{n}\_{2}\\left(t\\right)\\left(1-{\\rho }\_{{n}\_{2},{n}\_{2}}\\frac{{n}\_{2}(t)}{{N}\_{\\text{ss}}}-{\\rho }\_{{n}\_{2},{n}\_{1}}\\frac{{n}\_{1}\\left(t\\right)}{{N}\_{\\text{ss}}}\\right)=h({n}\_{1},{n}\_{2})\\end{array}$

(9)

The _ρ_ parameters denote phenomenological competition coefficients between and in the mutant clone and the normal stem cells, maintaining homeostasis. We have no further growth if either normal or mutated stem cells fill the entire compartment, _g_(_N_ss, 0) = 0 = _h_(0, _N_ss) and _g_(0, _N_ss) = 0 = _h_(_N_ss, 0), implying that \\({\\rho }\_{{n}\_{1},{n}\_{1}}=1={\\rho }\_{{n}\_{2},{n}\_{2}}\\). Moreover, \\({\\rho }\_{{n}\_{2},{n}\_{1}}=r\\) and hence

$\\frac{{\\rm{d}}{n}\_{1}}{{{\\rm{d}}t}}={\\lambda }\_{\\text{ss}}{n}\_{1}\\left(t\\right)\\left(1-\\frac{{n}\_{1}\\left(t\\right)}{{N}\_{\\text{ss}}}-\\left(2-r\\right)\\frac{{n}\_{2}\\left(t\\right)}{{N}\_{\\text{ss}}}\\right)$

(10)

$\\frac{{\\rm{d}}{n}\_{2}}{{{\\rm{d}}t}}={\\lambda }\_{\\text{ss}}{n}\_{2}\\left(t\\right)\\left(1-\\frac{{n}\_{2}(t)}{{N}\_{\\text{ss}}}-r\\frac{{n}\_{1}\\left(t\\right)}{{N}\_{\\text{ss}}}\\right)$

(11)

The number of mutant stem cells _n_2(_t_) will remain much smaller than the number of normal stem cells for extended periods, and hence we approximate equation ([11](https://www.nature.com/articles/s41588-025-02217-y#Equ11)) by

$\\frac{{\\rm{d}}{n}\_{2}}{{{\\rm{d}}t}}={\\lambda }\_{\\text{ss}}{n}\_{2}\\left(t\\right)\\left(1-r\\right)$

(12)

Equations ([10](https://www.nature.com/articles/s41588-025-02217-y#Equ10)) and ([12](https://www.nature.com/articles/s41588-025-02217-y#Equ12)) will be used when computing the SFS.

With clonal selection, the SFS has three principal contributions for somatic variants originating: (1) before the positively selected mutation occurred, _S__i_,1; (2) after this mutation occurred and happening in the mutant clone, _S__i_,2; and (3) after the mutation occurred but happening in normal stem cells, _S__i_,3:

${S}\_{i}\\left(t\\right)={S}\_{i,1}\\left(t\\right)+{S}\_{i,2}\\left(t\\right)+{S}\_{i,3}\\left(t\\right)$

(13)

We specify each contribution in turn. The SFS in the mutant clone, _S__i_,2, is

${S}\_{i,2}\\left(t\\right)={\\int }\_{\\!{t}\_{\\text{s}}}^{t}{\\mu {\\lambda }\_{\\text{ss}}{n}\_{2}({t}^{{\\prime} })P}\_{\\exp ,1,i}\\left(t-{t}^{{\\prime} }|{\\lambda }\_{\\text{ss}},r{\\lambda }\_{\\text{ss}}\\right)\\text{d}{t}^{{\\prime} }$

(14)

where _P_exp,1,_i_ is given by equation ([1](https://www.nature.com/articles/s41588-025-02217-y#Equ1)).

The SFS in normal stem cells, _S__i_,1 and _S__i_,3, are shaped by the decline in the number of normal stem cells after _t_S (equation ([10](https://www.nature.com/articles/s41588-025-02217-y#Equ10))). We approximate this process with a subcritical birth–death process with division rate _λ_ss and effective loss rate _δ_eff(_t_). The loss rate is chosen such that the expectation of the decline of normal stem cells in the linear birth–death process equals the number of normal stem cells lost by the competition dynamics, _D_. Equation ([10](https://www.nature.com/articles/s41588-025-02217-y#Equ10)) implies that

$D={\\int }\_{\\!{t}\_{\\text{s}}}^{t}{\\lambda }\_{\\text{ss}}\\left(\\frac{{n}\_{1}\\left(t\\right)}{{N}\_{\\text{ss}}}+\\left(2-r\\right)\\frac{{n}\_{2}\\left(t\\right)}{{N}\_{\\text{ss}}}\\right){n}\_{1}\\left(t\\right)\\text{d}t$

(15)

Therefore, the effective loss rate _δ_eff(_t_) is defined by

${\\int }\_{\\!{t}\_{\\text{s}}}^{t}{\\delta }\_{\\text{eff}}{N}\_{\\text{ss}}\\exp \\left(\\left({\\lambda }\_{\\text{ss}}-{\\delta }\_{\\text{eff}}\\right)(t-{t}\_{\\text{s}})\\right)\\text{d}t=D$

(16)

The SFS of variants acquired in normal stem cells after _t_s, _S__i_,3, is the superposition of (_N_ss − 1) independent linear subcritical birth–death processes and is given by

${S}\_{i,3}\\left(t\\right)=(N\_{\\text{ss}}-1){\\int }\_{\\!{t}\_{\\text{s}}}^{t}{\\mu {\\lambda }\_{\\text{ss}}{\\rm{e}}^{\\left({\\lambda }\_{\\text{ss}}-{\\delta }\_{\\text{eff}}\\right)(t^{{\\prime} }-{t}\_{\\text{s}})}P}\_{\\exp ,1,i}\\left(t-{t}^{{\\prime} }|{\\lambda }\_{\\text{ss}},{\\delta }\_{\\text{eff}}\\right)\\text{d}{t}^{{\\prime} }$

(17)

Finally, we compute the SFS of variants acquired before _t_s, _S__i_,1. These variants may be inherited to the selected clone, in which case they will be present in all selected cells and, additionally, in some of the normal cells. Alternatively, they may be present in normal cells only. To distinguish the two cases, we consider a variant that was acquired before _t_s and is present in _k_ cells at _t_s. Assuming that the driver mutation is acquired in a random cell, the probability of this variant being inherited to the selected clone is \\(\\frac{k}{{N}\_{\\text{ss}}}\\), while the probability of this variant being exclusively present in normal cells is \\(\\frac{1-k}{{N}\_{\\text{ss}}}\\). In the former case, all _n_2(_t_) selected cells will harbor the variant at the time of measurement, _t_; in addition, the _k_ − 1 normal stem cells harboring the variant at time _t_S may reach a clone size between 0 and _N_ss − _n_2(_t_), according to the drift dynamics of normal stem cells. In the alternative case, the variant does not end up in the selected clone and hence solely drifts in the normal cells. Taken together, this yields

${S}\_{i,1}\\left(t\\right)=\\left\\{\\begin{array}{c}\\mathop{\\sum }\\limits\_{k=1}^{{N}\_{\\text{ss}}}{S}\_{k}\\left({t}\_{\\text{s}}\\right)\\left\[\\overbrace{\\displaystyle\\frac{k}{{N}\_{\\text{ss}}}{P}\_{\\exp ,k-1,i-{n}\_{2}\\left(t\\right)}\\left(t-{t}\_{\\text{s}}|{\\lambda }\_{\\text{ss}},{\\delta }\_{\\text{eff}}\\right)}^{\\rm{variants}\\; \\rm{drift}\\; \\rm{in}\\; \\rm{normal}\\; \\rm{cells}\\;{\\rm{and}}\\;\\rm{present}\\; \\rm{in}\\; \\rm{all}\\; \\rm{selected}\\; \\rm{cells}}\\right.\\\\\\qquad\\qquad\\;\\left.+\\overbrace{\\left(1-\\displaystyle\\frac{k}{{N}\_{\\text{ss}}}\\right){P}\_{\\exp ,k,i}\\left(t-{t}\_{\\text{s}}|{\\lambda }\_{\\text{ss}},{\\delta }\_{\\text{eff}}\\right)}^{\\rm{variants}\\; \\rm{not}\\; \\rm{present}\\; \\rm{in}\\; \\rm{selected}\\; \\rm{cells}}\\right\],\\,i\\ge {n}\_{2}\\left(t\\right),\\\\\\qquad\\qquad\\qquad\\; \\mathop{\\sum }\\limits\_{k=1}^{{N}\_{\\text{ss}}}\\left(1-\\displaystyle\\frac{k}{{N}\_{\\text{ss}}}\\right){S}\_{k}\\left({t}\_{\\text{s}}\\right){P}\_{\\exp ,k,i}\\left(t-{t}\_{\\text{s}}|{\\lambda }\_{\\text{ss}},{\\delta }\_{\\text{eff}}\\right),\\,i < {n}\_{2}(t),\\end{array}\\right.$

(18)

where _S__k_(_t_s) is the SFS at _t__s_ and _P_exp,_a_,_b_ is the clone size distribution generated by a subcritical birth–death process initiated by _a_ cells[80](https://www.nature.com/articles/s41588-025-02217-y#ref-CR80 "Bailey, N. The Elements of Stochastic Processes (Wiley, 1964)."):

${P}\_{\\exp ,a,b}(t)=\\left\\{\\begin{array}{c}{x\\left(t\\right)}^{a},\\text{if}\\,b=0,\\\\ {\\sum }\_{j=0}^{\\min \\left(a,b\\right)}{{\\;a\\;}\\choose{\\;j\\;}}{{a+b-j-1}\\choose{a-1}}{x\\left(t\\right)}^{a-j}{y\\left(t\\right)}^{b-j}{\\left(1-x\\left(t\\right)-y\\left(t\\right)\\right)}^{\\;j},\\text{if}\\,b\\ge 1\\end{array}\\right.$

(19)

Thus, we have specified the contributions to the SFS of a stem cell population containing a selected clone, _S__i_(_t_) = _S__i_,1(_t_) + _S__i_,2(_t_) + _S__i_,3(_t_).

**Modeling a single selected clone without size compensation.** We here develop a version of our model in which the selected clone expands unrestrictedly. We assume a constant number _N_ss of normal stem cells, whereas the selected clone exponentially expands. Hence, the population dynamics of normal (_n_1) and mutant (_n_2) stem cells now read

$\\begin{array}{c}{n}\_{1}\\left(t\\right)={N}\_{\\text{ss}},\\\\ {n}\_{2}\\left(t\\right)={\\rm{e}}^{{\\lambda }\_{\\text{ss}}\\left(1-r\\right)(t-{t}\_{{\\rm{s}}})}\\end{array}$

(20)

As before, the SFS comprises variants that originated (1) before the positively selected mutation occurred, _S__i_,1; (2) after this mutation occurred and happening in the mutant clone, _S__i_,2; and (iii) after the mutation occurred but happening in normal stem cells, _S__i_,3. _S__i_,2 is the same as in the competition model and is modeled by equation ([14](https://www.nature.com/articles/s41588-025-02217-y#Equ14)). _S__i_,3 is now the superposition of _N_ss independent linear critical birth–death processes and reads

${S}\_{i,3}\\left(t\\right)={N}\_{\\text{ss}}{\\int }\_{\\!{t}\_{\\text{S}}}^{t}{\\mu {\\lambda }\_{\\text{ss}}P}\_{\\text{ss},1,i}\\left(t-{t}^{{\\prime} }|{\\lambda }\_{\\text{ss}}\\right)\\text{d}{t}^{{\\prime} }$

(21)

Likewise, _S__i_,1 is now governed by genetic drift according to a critical birth–death process in the founder cell population, in addition to selection of the mutant clone. It reads

$\\begin{array}{l}{S}\_{i,1}\\left(t\\right)=\\\\\\left\\{\\begin{array}{c}\\mathop{\\sum }\\limits\_{k=1}^{{N}\_{\\text{ss}}}{S}\_{k}\\left({t}\_{\\text{s}}\\right)\\left\[\\displaystyle\\frac{k}{{N}\_{\\text{ss}}}{P}\_{\\text{ss},k-1,i-C\\left(t\\right)}\\left(t-{t}\_{\\text{s}}|{\\lambda }\_{\\text{ss}}\\right)\\right.\\\\\\left.+\\left(1-\\displaystyle\\frac{k}{{N}\_{\\text{ss}}}\\right){P}\_{\\text{ss},k,i}\\left(t-{t}\_{\\text{s}}|{\\lambda }\_{\\text{ss}}\\right)\\right\],i\\ge {n}\_{2}\\left(t\\right),\\\\\\qquad\\qquad \\mathop{\\sum }\\limits\_{k=1}^{{N}\_{\\text{ss}}}{\\left(1-\\displaystyle\\frac{k}{{N}\_{\\text{ss}}}\\right)}S\_{k}\\left({t}\_{\\text{s}}\\right){P}\_{\\text{ss},k,i}\\left(t-{t}\_{\\text{s}}|{\\lambda }\_{\\text{ss}}\\right),i < {n}\_{2}(t).\\end{array}\\right.\\end{array}$

(22)

**Modeling multiple selected clones in a homeostatic tissue.** In the following, we generalize our model to account for the selection of multiple competing clones in a homeostatic tissue. We use _τ__c_ to denote the time at which a particular clone _c_ was born, where _c_ = 1 identifies normal cells and 1 < _c_ ≤ _C_ identifies the _C_ − 1 selected clones. For each clone the variable _v__c_ reports the identity of its mother clone. We assume that all clones compete for a limited space with carrying capacity _N_ss. As with the one-clone model, we implement clonal selection as a reduction in the loss rates, while leaving division rates unchanged. Denoting the competition coefficient between clone _c_ and other clones in the tissue with \\({\\rho }\_{c\\bullet }\\), we model the expected number of cells in clone _c_, _n__c_, with a system of ordinary differential equations,

$\\frac{{\\text{d}n}\_{c}}{\\text{d}t}={\\lambda }\_{\\text{ss}}{n}\_{c}\\left(1-\\mathop{\\sum }\\limits\_{j;1\\le\\; j\\le C}{\\rho }\_{{jc}}\\frac{{n}\_{j}}{{N}\_{\\text{ss}}}\\right)$

(23)

with conditions

$\\begin{array}{c}{n}\_{c}\\left({\\tau }\_{c}\\right)=1,\\\\ {n}\_{c}\\left({t < \\tau }\_{c}\\right)=0,\\\\ {n}\_{{v}\_{c}}\\left({\\tau }\_{c}\\right)={n}\_{{v}\_{c}}\\left({\\tau }\_{c}-{\\rm{d}t}\\right)-1.\\end{array}$

(24)

The competition matrix _ρ_ is defined by the selective advantages, and, requiring Σ_j__n__j_ = _N_ss at all times, is given by

$\\rho =\\left(\\begin{array}{cccc}1 & 2-{r}\_{2} & 2-{r}\_{3} & \\ldots \\\\ {r}\_{2} & 1 & 2-{r}\_{3}/{r}\_{2} & \\ldots \\\\ {r}\_{3} & {r}\_{3}/{r}\_{2} & 1 & \\ldots \\\\ \\ldots & \\ldots & \\ldots & \\ldots \\end{array}\\right)$

where _r__c_ models a reduction of cell loss in clone _c_ at _τ__c_ and takes values 0 < _r__c_ < 1. Ultimately, we are interested in _S__i_(_t_), the expected number of variants that are found in a clone of size _i_ at the time of measurement, _t_. As in the one-clone model, _S__i_(_t_) has contributions from variants acquired in normal cells and variants acquired in the selected clones. In the following, we denote the SFS of variants acquired in a particular clone with _S__i_,_c_(_t_). Hence,

${S}\_{i}\\left(t\\right)=\\mathop{\\sum }\\limits\_{c=1}^{C}{S}\_{i,c}\\left(t\\right)$

(25)

To evaluate _S__i_,_c_(_t_), we denote with \\({\\phi }\_{c}\\) the set of subclones generated by a mutation in clone _c_, in the following termed ‘daughters’ of clone _c_, and with \\(\\varphi \\subset {\\phi }\_{c}\\) the set of all possible combinations of \\({\\phi }\_{c}\\). Finally, the set _Ψ__c_ denotes the entire progeny of clone _c_, including daughters, granddaughters and so on. Consider a variant that was acquired in clone _c_ and is present in _k_ cells at time _t_′. This variant will, on average, be inherited to the daughter clone \\(d,d\\in {\\phi }\_{c}\\), with probability

$\\kappa \\left(c,d,k,t{\\prime} \\right)=H({\\tau }\_{d}-{t}^{{\\prime} })\\frac{k}{{n}\_{c}(t{\\prime} )}$

(26)

where _H_ is the heavyside step function and _τ__d_ is the birth date of clone _d_. Considering a particular subset of daughters of clone _c_, \\({\\varphi }\_{j}\\in \\varphi\\), the probability of inheriting the variant to all daughters in this subset is, on average,

$\\begin{array}{l}{{{K}}}\\left(c,{\\varphi }\_{j},k,t{\\prime} \\right)=\\overbrace{{\\mathop{\\prod}\\nolimits\_{l\\in {\\varphi }\_{j},{\\varphi }\_{j}\\in \\varphi }}\\kappa \\left(c,l,k,{t}^{{\\prime} }\\right)}^{{\\textrm{presence\\; in\\; the\\; subset\\; of\\; daughters\\;}}{\\varphi }\_{j}\\in \\varphi }\\\\\\times \\underbrace{{\\mathop{\\prod}\\nolimits \_{l\\in \\left\\{{\\phi }\_{c}{\\rm{\\backslash }}{\\varphi }\_{j}\\right\\},{\\varphi }\_{j}\\in \\varphi }}\\left(1-\\kappa \\left(c,l,k,{t}^{{\\prime} }\\right)\\right)}\_{{\\textrm{absence\\; in\\; the\\; remaining\\; daughters\\;}}}\\end{array}$

(27)

We compute _S__i_,_c_(_t_) by considering all possible combinations of daughters that may inherit variants acquired in clone _c._ This yields

$\\begin{array}{l}{S}\_{i,c}\\left(t\\right)\\\\=\\overbrace{{\\displaystyle\\int}\_{\\!{\\tau }\_{c}}^{t}\\,{\\stackrel{\\textrm{acquisition\\; of\\; variants}}{\\mu {\\lambda }\_{\\text{ss}}{n}\_{c}(t{\\prime} )}}\\,\\underbrace{\\sum \_{{\\varphi }\_{j}\\in \\varphi }{{{\\rm K}}}\\left(c,{\\varphi }\_{j},1,t{\\prime} \\right){P}\_{c}\\left(1,{\\stackrel{\\textrm{drift\\; within\\; clone\\;}c}{i-\\sum \_{k\\in {\\varphi }\_{j}}{n}\_{k}\\left(t\\right)}},{t}^{{\\prime} },t\\right)}\_{{\\textrm{all\\; possible\\; combinations\\; of\\; daugthers}}}\\text{d}{t}^{{\\prime} }+}^{{\\textrm{variants\\; acquired\\; in\\; clone\\;}}c\\;{\\textrm{and\\; inherited\\; to\\; at\\; least\\;}}1\\;{\\textrm{daughter\\; of\\; clone\\;}}c}\\\\ \\underbrace{{\\displaystyle\\int}\_{\\!{\\tau}\_{c}}^{t}\\overbrace{\\mu {\\lambda }\_{\\text{ss}}{n}\_{c}(t{\\prime} )}^{{\\textrm{acquisition\\; of\\; variants}}}\\left(1-\\sum \_{{\\varphi }\_{j}\\in \\varphi }{{{\\rm K}}}\\left(c,{\\varphi }\_{j},1,t{\\prime} \\right)\\right){P}\_{c}(1,i,{t}^{{\\prime} },t)\\text{d}{t}^{{\\prime} }}\_{{\\textrm{variants\\; acquired\\; in\\; clone\\;}}c\\;{\\textrm{and\\; not\\; inherited\\; to\\; any\\; daughter\\; of\\; clone\\;}}c}\\end{array}$

(28)

where _P__c_(1,_i_,_t_′,_t_) is the probability that a variant acquired in clone _c_ has drifted to _i_ cells within a time span _t_ − _t_′. Specifically, _P__c_(1,_i_,_t_′,_t_) is defined by nonlinear birth–death processes, because the division and loss rates in clone _c_ change with time, subject to the dynamics in equation ([23](https://www.nature.com/articles/s41588-025-02217-y#Equ23)). In analogy to the one-clone model, we approximate this nonlinearity by modeling _S__i_,_c_(_t_) with linear birth–death processes in a step-wise fashion, considering time intervals between clonal birth dates, _τ_ turning points in _n_(_t_) (as determined by numeric evaluation of equation ([23](https://www.nature.com/articles/s41588-025-02217-y#Equ23))) and the end point as time points of interest. Specifically, we modeled the drift of somatic variants in each time interval using linear birth–death processes, which we parametrized with an effective loss rate of cells in clone _c_, \\({\\delta }\_{\\text{eff},c,{t}\_{a}\\le t < {t}\_{b}},\\) and which we evaluated for an effective time span \\({\\varDelta }\_{\\text{eff},{c,t}\_{a}\\le t < {t}\_{b}}\\), where _t__a_ and _t__b_ denote the start and end point of the interval. To define \\({\\delta }\_{\\text{eff},c,{t}\_{a}\\le t < {t}\_{b}}\\) and \\({\\varDelta }\_{c,\\text{eff},{t}\_{a}\\le t < {t}\_{b}}\\), we distinguished the case when the number of cells in clone _c_ increased (_n__c_(_t__b_) > _n__c_(_t__a_)) from the case when it decreased (_n__c_(_t__b_) < _n__c_(_t__a_)) during the time interval (_t__a_,_t__b_). If _n__c_(_t__b_) > _n__c_(_t__a_), we defined \\({\\delta }\_{\\text{eff},c,{(t}\_{a},{t}\_{b})}={r}\_{c}{\\lambda }\_{\\text{ss}}\\) and, to avoid overshooting the actual clone size, \\({\\varDelta }\_{{\\rm{eff}},c,(t\_{a},{t}\_{b})}=\\frac{\\log {n}\_{c}({t}\_{b})/{n}\_{c}({t}\_{a})}{{\\lambda }\_{\\text{ss}}(1-{r}\_{c})}\\). By contrast, if _n__c_(_t__b_) < _n__c_(_t__a_), we parametrized \\({\\delta }\_{\\text{eff},c{(,t}\_{a},{t}\_{b})}\\) such that the expected decline in a linear birth–death process equals the number of cells lost by competition dynamics. Hence, \\({\\delta }\_{{\\text{eff}},c({t}\_{a},{t}\_{b})}\\) is defined by

$\\begin{array}{l}{\\displaystyle\\int }\_{\\!{{\\rm{t}}}\_{{\\rm{a}}}}^{{{\\rm{t}}}\_{{\\rm{b}}}}{\\delta }\_{\\text{eff},c,(t\_{a},{t}\_{b})}{n}\_{{{c}}}({t}\_{a})\\exp (({{\\rm{\\lambda }}}\_{\\text{ss}}-{\\delta }\_{\\text{eff},c,(t\_{a},{t}\_{b})}){\\rm{t}}{\\prime} )\\text{d}t{\\prime} \\\\={\\displaystyle\\int }\_{\\!{{\\rm{t}}}\_{{\\rm{a}}}}^{{{\\rm{t}}}\_{{\\rm{b}}}}{{\\rm{\\lambda }}}\_{\\text{ss}}{n}\_{{{c}}}({t}^{{\\prime} })\\mathop{\\sum }\\limits\_{1\\le\\; j\\le C}{{\\rho }\_{jc}}\\frac{{n}\_{j}({t}^{{\\prime} })}{{N}\_{{\\rm{ss}}}}\\text{d}t{\\prime}\\end{array}$

(29)

where the right-hand side gives the number of death events in clone _c_ according to the competition dynamics in equation ([23](https://www.nature.com/articles/s41588-025-02217-y#Equ23)) and the left-hand side gives the number of death events according to a linear birth–death process. Moreover, if _n__c_(_t__b_) < _n__c_(_t__a_), we defined \\({\\varDelta }\_{c,{\\rm{eff}},({t}\_{a},{t}\_{b})}={t}\_{b}-{t}\_{a}\\). We then approximated _S__i_,_c_(_t_) by recursively evaluating

$\\begin{array}{l}{S}\_{i,c}\\left({t}\_{b}\\right)=\\mathop{\\sum }\\limits\_{k=1}^{{N}\_{\\text{ss}}}{S}\_{k,c}\\left({t}\_{a}\\right)\\\\\\left\[\\overbrace{\\sum \_{{\\varphi }\_{j}\\in \\varphi }{{{K}}}\\left(c,{\\varphi }\_{j},k,{t}\_{a}\\right){P}\_{\\exp ,k,i-\\sum \_{l\\in {\\varphi }\_{j}}{n}\_{l}\\left(t\\right)}\\left({\\varDelta }\_{{\\rm{eff}},c,(t\_{a}{,t}\_{b})}|{{\\lambda }\_{\\text{ss}},\\delta }\_{\\text{eff},c,(t\_{a},{t}\_{b})}\\right)}^{A}\\right.\\\\\\left.+\\overbrace{\\left(1-\\sum \_{{\\varphi }\_{j}\\in \\varphi }{{{K}}}\\left(c,{\\varphi }\_{j}k,{t}\_{a}\\right)\\right){P}\_{\\exp ,k,i}\\left({\\varDelta }\_{{\\rm{eff}},c,({t}\_{a},{t}\_{b})}|{{\\lambda }\_{\\text{ss}},\\delta }\_{\\text{eff},c,(t\_{a},{t}\_{b})}\\right)}^{B}\\right\]\\\\+\\overbrace{{n}\_{c}\\left({t}\_{a}\\right){\\int }\_{{t}\_{a}}^{{t}\_{b}}\\mu {\\lambda }\_{\\text{ss}}{\\rm{e}}^{({\\lambda }\_{\\text{ss}}-{\\delta }\_{\\text{eff},c,({t}\_{a},{t}\_{b})})({\\varDelta }\_{c,\\text{eff},({t}\_{a},{t}\_{b})})}{P}\_{\\exp ,1,i}\\left({\\varDelta }\_{{\\rm{eff}},c,(t\_{a},{t}\_{b})}|{{\\lambda }\_{\\text{ss}},\\delta }\_{\\text{eff},c,(t\_{a},{t}\_{b})}\\right)}^{C}\\end{array}$

(30)

until _t__b_ = _t_. Here, terms (A) and (B) compute the drift of variants acquired before _t__a_, of which some (A) will be inherited to subclones of clone _c_, whereas others (B) will be found exclusively in clone _c_. Finally, term (C) computes the drift of variants newly acquired in the time interval (_t__a_,_t__b_).

#### Parameter estimation

Model fits were obtained for each donor separately. In a first step, clonal selection was distinguished from genetic drift by applying the one-clone model of SCIFER to the data. For blood samples classified as selected and sequenced at high resolution (≥150× WGS), subsequent refinement was achieved by applying the two-clone model in a second step. Here, both possible topologies between two selected clones (linear and branched evolution) were tested, using prior distributions that were informed by the initial one-clone model fits (Supplementary Table [10](https://www.nature.com/articles/s41588-025-02217-y#MOESM4); note that constraining the prior distributions based on the one-clone model fits facilitates the detection of additional subclones). All model fits were performed using ABC based on sequential Monte Carlo as implemented in pyABC[82](https://www.nature.com/articles/s41588-025-02217-y#ref-CR82 "Klinger, E., Rickert, D. & Hasenauer, J. pyABC: distributed, likelihood-free inference. Bioinformatics 34, 3591–3593 (2018).") (posterior sample size of 1,000 and termination criterion _ε_min = 0.05). Briefly, ABC samples parameter sets from user-defined prior distributions (Supplementary Tables [9](https://www.nature.com/articles/s41588-025-02217-y#MOESM4) and [10](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)) and simulates the expected VAF distribution for each parameter set. The simulated VAF distributions are compared with the measured ones and the 1,000 parameter sets by minimal distance between measured and simulated data are selected. In each of the following iterations, the parameter set is expanded by adding random noise to the current parameter set, and the distance between model and data is reassessed. ABC is terminated once the distance between model and data is smaller than a defined threshold, _ε_min, yielding an estimate for the posterior distributions of the model parameters. We performed the following steps in each ABC iteration:

1.  (1)
    
    \[Experimental data\]: Determine the cumulative VAF distribution at time _t_ for the experimental data. For 90× bulk WGS data from human BM or peripheral blood samples, include variants if they are supported by at least 3 reads, absent in the germline control sample, and if the locus is covered by at least 10 and at most 300 reads. For 270× bulk WGS data from human CD34+ HSPCs, include variants if they are supported by at least 3 reads. For pseudo-bulk WGS data, include all variants. For bulk WGS data from human brain, include all tier1–tier2 variants from ref. [17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022)."). Compute the cumulative number of VAFs ≥_f_, defined as \\({M}\_{\\text{experimental},\\;f}={\\sum }\_{\\text{VAF}=f}^{1}{\\;S}\_{\\text{experimental},\\text{VAF}},\\) where _S_experimental denotes the measured SFS and _S_experimental,VAF is the number of variants with a particular VAF. We evaluated _M_experimental,_f_ for bins with width 0.01, spanning 0.05 ≤ _f_ ≤ 1 for bulk WGS data with an average coverage <150×, 0.02 ≤ _f_ ≤ 1 for bulk WGS data with an average coverage ≥150×, and 0.01 ≤ _f_ ≤ 1 for pseudo-bulk data.
    
2.  (2)
    
    \[Model\]: Sample parameters from their prior distributions (Supplementary Table [9](https://www.nature.com/articles/s41588-025-02217-y#MOESM4), one-clone model; Supplementary Table [10](https://www.nature.com/articles/s41588-025-02217-y#MOESM4), two-clone model).
    
3.  (3)
    
    \[Model\]: Simulate the expected cumulative VAF distribution, _M_sim,_f_(_t_data), where _t_data is the age of the patient and _f_ the minimal VAF (for numerical implementation of the model see Supplementary Note [4](https://www.nature.com/articles/s41588-025-02217-y#MOESM1)). The bins are as with the experimental data. Depending on the following criterion, based on the prior parameter sample, the cumulative VAF histogram is simulated with either the neutral model or the selection model: if a selected clone could grow above the detection limit with the prior parameter sample, the selection model is used; otherwise the neutral model is used. Formally, if \\(\\frac{1}{2}{\\rm{e}}^{{\\lambda }\_{{\\rm{SS}}}^{{\\rm{prior}}}(1-{r}^{{\\rm{prior}}})({t}\_{\\text{data}}-{t}\_{{\\rm{S}}}^{{\\rm{prior}}})}\\ge \\gamma {N}\_{{\\rm{SS}}}^{{\\rm{prior}}}\\) (one-clone model) or if any \\(\\frac{1}{2}{n}\_{c,c > 1}\\left({t}\_{{\\rm{data}}}|{\\lambda }\_{{\\rm{SS}}}^{{\\rm{prior}}},{{\\tau }^{{\\rm{prior}}},r}^{{\\rm{prior}}},\\phi ,\\psi {,N}\_{{\\rm{SS}}}^{{\\rm{prior}}}\\right)\\ge \\gamma {N}\_{{\\rm{SS}}}^{{\\rm{prior}}}\\) (two-clone model) then the selection model is used. The detection limit is set to _ɣ_ = 0.025 for 90× and 30× WGS data, and _ɣ_ = 0.005 for pseudo-bulk and 270× bulk WGS data. Importantly, note that the selection model can return a posterior corresponding to neutral evolution.
    
4.  (4)
    
    \[Model\]: Add the number of variants present in the founder cell of the hematopoietic system, \\({\\varDelta }\_{{\\rm{clonal}}}^{{\\rm{prior}}}\\), to _M_sim,0.5 (corresponding to mutations acquired during early development).
    
5.  (5)
    
    \[Model\]: Simulate experimental error of sequencing. To this end, generate the expected SFS for the _j_th frequency bin, _S_sim(_f__j_) = _M_sim(_f__j_) − _M_sim(_f__j_+1). For bulk WGS data, sample for each simulated variant a sequencing coverage ϑ from a Poisson distribution with mean \\(\\hat{\\vartheta }\\) corresponding to the average sequencing depth. Thereafter, sample VAFs for each variant according to \\(\\frac{B\\left(f,\\vartheta \\right)}{\\vartheta}\\), where _B_ denotes the binomial distribution and _f_ is the true VAF in the tissue. Discard variants supported by less than 3 reads. For pseudo-bulk WGS data, sample VAFs for each variant according to \\(\\frac{B\\left(2f,{n}\_{\\text{cells}}\\right)}{(2{n}\_{\\text{cells}})}\\), where \\({n}\_{\\text{cells}}\\) is the number of sequenced single-cell clones. Compute the sampled cumulative VAF distribution, _M_sim,_f_,sampled.
    
6.  (6)
    
    \[Model versus Experimental data\]: Determine the distance function for ABC
    

$d=\\mathop{\\sum }\\limits\_{f}{\\left({M}\_{\\text{sim},\\;f,\\text{sampled}}-{M}\_{\\text{experimental},\\;f}\\right)}^{2}$

#### Classification of cases as neutrally evolving or as selected

We classified pseudo-bulks and samples sequenced at <150× as selected if at least 15% of the posterior samples report a selected clone size ≥0.1. These thresholds were defined based on in silico generated test data, and correspond to the WGS depth of 90× (Fig. [2](https://www.nature.com/articles/s41588-025-02217-y#Fig2)). WGS data with ≥150× coverage allow for higher resolution and hence, we classified cases as selected if at least 15% of the posterior samples report a selected clone size ≥0.04 according to the one-clone model, and as harboring two selected clones if at least 15% of the posterior samples report a size ≥0.04 for both selected clones. Upon sample classification, we computed the 80% highest density intervals for each parameter on the parameter subsets supporting neutral evolution or clonal selection, respectively.

### Simulation of phylogenetic trees and in silico evaluation of model performance

We validated the population genetics model with simulated data, generated according to stochastic birth–death processes (see Supplementary Note [5](https://www.nature.com/articles/s41588-025-02217-y#MOESM1) for a description of the simulations and [https://github.com/VerenaK90/SCIFER](https://github.com/VerenaK90/SCIFER) for their computational implementation).

Simulations used to evaluate model performance (Fig. [2](https://www.nature.com/articles/s41588-025-02217-y#Fig2)) were run for stem cells only. Simulations of neutral evolution were parametrized with _N_ss,S = 25,000, _μ_ = 1, _λ_ss,S = 10 per year, _δ_ss,S = 10 per year and with _τ_ = 250 (that is, summarizing 1% of the reactions occurring in 25,000 stem cells). Simulations of clonal selection were parametrized with _N_ss,S = 25,000, _μ_ = 1, _λ_ss,S = 10 per year, _δ_ss,S = 10 per year, _t_s = 20 years, _s_ = 0.02 and with _τ_ = 2,500.

Simulations of neutral evolution used to assess the effect of differentiation into a single progenitor cell population on the VAF histogram were parametrized with _N_ss,S = 1,000, μ = 1, λss,S = 1 per year, δss,S = 1 per year, _N_ss,P = 2,500, λss,P = 4.6 per year, δss,S = 5 per year and _τ_ = 2,500.

Simulations of neutral evolution in a heterogenous tissue where stem cells differentiate into two progenitor cell types during development, but continue to produce only one of them during adulthood were parametrized with _N_ss,S = 1,000, μ = 1, λss,S = 1 per year, \\({\\delta }\_{\\text{ss},\\text{S} > {P}\_{1}}\\) = 1 per year, \\({\\delta }\_{\\text{ss},\\text{S} > {P}\_{2}}\\) = 0 per year, \\({N}\_{\\text{ss},{P}\_{1}}\\) = 6,000 \\({\\lambda }\_{\\text{ss},{P}\_{1}}\\) = 3 per year, \\({\\delta }\_{\\text{ss},{P}\_{1}}\\) = 3.167 per year, \\({N}\_{\\text{ss},{P}\_{2}}\\) = 4,000, \\({\\lambda }\_{\\text{ss},{P}\_{2}}\\) = 0 per year, \\({\\delta }\_{\\text{ss},{P}\_{2}}\\) = 0 per year and _τ_ = 1,000.

#### Parameter inference

To infer the dynamic stem cell parameters (_μ_, _δ_exp, _λ_ss,S, _N_ss,S, _t__s_, _r_) (Fig. [2](https://www.nature.com/articles/s41588-025-02217-y#Fig2)), we subsampled 10,000 cells from the simulated trees at time points specified in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02217-y#MOESM4) and computed the simulated VAF distribution from subsampled trees. To account for technical noise, we simulated sequencing by sampling for each variant a sequencing coverage \\(\\vartheta \\propto {Pois}(\\hat{\\vartheta })\\), where ϑ is the average sequencing coverage and thereafter sampling mutant reads according to _B_(_VAF_,ϑ), where _B_ is the binomial distribution. We simulated VAFs for average sequencing coverages of 30×, 90× and 270×, and fitted the population genetics model to the simulated bulk WGS data as described above for the real data.

#### Sensitivity and specificity of detecting selected clones

To evaluate the sensitivity and specificity of the out model, we analyzed the posterior probability of clonal selection. In accordance with the minimal clone sizes used in the inference setup (see above), we computed the probability of clonal selection as \\(P\\left({\\rm{selection}}\\right)=\\frac{{\\sum }\_{i}\\frac{{n}\_{2}({t}\_{\\text{data}}|{\\theta }\_{i})}{{N}\_{\\text{ss},i}}\\ge 0.05}{{\\sum }\_{i}1}\\) for 30× and 90× sequencing depths, and as \\(P\\left({\\rm{selection}}\\right)=\\frac{{\\sum }\_{i}\\frac{{n}\_{2}({t}\_{\\text{data}}|{\\theta }\_{i})}{{N}\_{\\text{ss},i}}\\ge 0.01}{{\\sum }\_{i}1}\\) for 270× sequencing depth, where \\({n}\_{2}({t}\_{\\text{data}}|{\\theta }\_{i})\\) is the size of the selected clone at the patient age, _t_data, given the _i_th parameter sample, θ_i_, and _N_ss,_i_ is the _i_th estimate of the stem cell number. Among all parameter sets reporting a clone size ≥0.05 (30× and 90× coverage) or ≥0.01 (270× coverage), we determined median clone size and, if they were at least 5% in size, rounded it with 5% accuracy, or else rounded them with 1% accuracy. Thereafter, we computed the sensitivity and specificity of our approach, by classifying cases as selected if _P_(selection) ≥ _β_, varying the threshold _β_ between 1% and 100%. For each of the true clone sizes (1%, 2%, 5%, 10%, 15%, 20%, 25%, 50%, 75%; Supplementary Table [1](https://www.nature.com/articles/s41588-025-02217-y#MOESM4)), we computed the number of true positives as the number of cases where the actual clone size was correctly inferred (we classified inferred clone sizes between 0.5 and 1.5 of the actual clone size as correct). Conversely, we computed the number of false positives as the number of cases in which SCIFER erroneously reported a particular clone size, albeit the actual clone size was zero.

The difference between true positives and false positives was maximal for _β_ = 15%. At this threshold, clones of size ≥5% VAF were reliably inferred (90×).

### Statistics and reproducibility

Bayesian parameter inference and statistical analyses were conducted with pyABC[82](https://www.nature.com/articles/s41588-025-02217-y#ref-CR82 "Klinger, E., Rickert, D. & Hasenauer, J. pyABC: distributed, likelihood-free inference. Bioinformatics 34, 3591–3593 (2018).") v.0.12.6, using python v.3.10.1 and R (v.4.2.0 and v.4.2.1). We used the following R packages: ape[83](https://www.nature.com/articles/s41588-025-02217-y#ref-CR83 "Paradis, E. & Schliep, K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics 35, 526–528 (2019).") v.5.6-2, phytools[84](https://www.nature.com/articles/s41588-025-02217-y#ref-CR84 "Revell, L. J. phytools: an R package for phylogenetic comparative biology (and other things). Methods Ecol. Evol. 3, 217–223 (2012).") v.1.2-0, phangorn[85](https://www.nature.com/articles/s41588-025-02217-y#ref-CR85 "Schliep, K., Potts, A. A., Morrison, D. A. & Grimm, G. W. Intertwining phylogenetic trees and networks. Methods Ecol. Evol. 8, 1212–1220 (2017).") v.2.10.0, castor[86](https://www.nature.com/articles/s41588-025-02217-y#ref-CR86 "Louca, S. & Doebeli, M. Efficient comparative phylogenetics on large trees. Bioinformatics 34, 1053–1055 (2018).") v.1.7.5, TreeTools v.1.8.0, deSolve[87](https://www.nature.com/articles/s41588-025-02217-y#ref-CR87 "Soetaert, K., Petzoldt, T. & Setzer, R. W. Solving differential equations in R: package deSolve. J. Stat. Softw. 33, 1–25 (2010).") v.1.33, openxlsx v.4.2.5, cdata v.1.2.0, ggpubr v.0.4.0, RRphylo[88](https://www.nature.com/articles/s41588-025-02217-y#ref-CR88 "Castiglione, S. et al. A new method for testing evolutionary rate variation and shifts in phenotypic evolution. Methods Ecol. Evol. 9, 974–983 (2018).") v.2.7.0, ggplot2 (ref. [89](https://www.nature.com/articles/s41588-025-02217-y#ref-CR89 "Villanueva, R.A.M. & Chen, Z.J. ggplot2: Elegant Graphics for Data Analysis (Taylor & Francis, 2019).")) v.3.4.2, cgwtools v.3.3, ggVennDiagram v.1.2.2, ggbeeswarm v.0.6.0, ggsci v.2.9, Hmisc v.4.7.1, lemon v.0.4.5, data.table v.1.14.2, RColorBrewer v.1.1.3, ggridges v.0.5.4, doParallel v.1.0.17, foreach v.1.5.2, parallel v.2.1, wesanderson v.0.3.6, bedr v.1.0.7, ggformula v.0.10.2, HDInterval v.0.2.2, reshape2 v.1.4.4 (ref. [90](https://www.nature.com/articles/s41588-025-02217-y#ref-CR90 "Wickham, H. Reshaping data with the reshape package. J. Stat. Softw. 21, 1–20 (2007).")), dplyr v.1.0.9 and scales v.1.2.1. Flow cytometry data was analyzed with FlowJo v.10.8.1.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02217-y#MOESM2) linked to this article.

Data availability
-----------------

Single-cell WGS data were part of previously published studies[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."),[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). WGS data from these studies are deposited at the European Genome-Phenome Archive ([https://www.ebi.ac.uk/ega/](https://www.ebi.ac.uk/ega/)) under accession nos. [EGAD00001004086](https://ega-archive.org/datasets/EGAD00001004086), [EGAD00001007851](https://ega-archive.org/datasets/EGAD00001007851) and [EGAD00001007684](https://ega-archive.org/datasets/EGAD00001007684). Substitution calls from these studies are deposited on Mendeley Data ([https://doi.org/10.17632/yzjw2stk7f.1](https://doi.org/10.17632/yzjw2stk7f.1); ref. [91](https://www.nature.com/articles/s41588-025-02217-y#ref-CR91 "Lee-Six, H. & Campbell, P. Population dynamics of normal human blood inferred from spontaneous somatic mutations – substitution calls. Mendeley Data V1 
https://doi.org/10.17632/yzjw2stk7f.1
(2018).")), ([https://doi.org/10.17632/np54zjkvxr.2](https://doi.org/10.17632/np54zjkvxr.2); ref. [92](https://www.nature.com/articles/s41588-025-02217-y#ref-CR92 "Campbell, P. Dataset for: Clonal dynamics of haematopoiesis across the human lifespan. Mendeley Data V2 
https://doi.org/10.17632/np54zjkvxr.2
(2022).")) and on figshare ([https://doi.org/10.6084/m9.figshare.15029118](https://doi.org/10.6084/m9.figshare.15029118); ref. [93](https://www.nature.com/articles/s41588-025-02217-y#ref-CR93 "Almeida, J.G.D., Fabre, M., Gerstung, M. & Vassiliou, G. Data. figshare 
figshare.com/articles/dataset/Data/15029118
(2021).")). WGS data generated in this study (aligned bam files) are available at the European Genome-Phenome Archive under accession no. [EGAS00001007558](https://ega-archive.org/studies/EGAS00001007558). The bam files contain all relevant meta data for back conversion into fastq files and realignment. Variant calls and model fits have been made available on Mendeley data[94](https://www.nature.com/articles/s41588-025-02217-y#ref-CR94 "Körber, V., Vyas, P. & Höfer, T. Clonal hematopoiesis quantified in human bone marrow whole genome sequencing data. Mendeley Data V1 
https://doi.org/10.17632/gkzvmg5f6z.1
(2025).") ([https://doi.org/10.17632/gkzvmg5f6z.1](https://doi.org/10.17632/gkzvmg5f6z.1)). Patient information and driver mutations are available as Supplementary Information to this manuscript. We also used the following publicly available datasets: hg19 reference genome ([https://ftp.ensembl.org/pub/grch37/release-99/fasta/homo\_sapiens/dna/Homo\_sapiens.GRCh37.dna.primary\_assembly.fa.gz](https://ftp.ensembl.org/pub/grch37/release-99/fasta/homo_sapiens/dna/Homo_sapiens.GRCh37.dna.primary_assembly.fa.gz)), gnomAD v.2.1.1. ([https://storage.googleapis.com/gcp-public-data–gnomad/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.vcf.bgz](https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.vcf.bgz)), repeat regions and simple repeat regions (downloaded from UCSC table browser, setting the assembly to hg19, the track to ‘RepeatMasker’ or ‘Simple Repeats’), annovar (version May2018; [http://annovar.openbioinformatics.org/](http://annovar.openbioinformatics.org/)), dbSNP v.150 ([https://ftp.ncbi.nlm.nih.gov/snp/organisms/human\_9606\_b150\_GRCh37p13/VCF/00-All.vcf.gz](https://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh37p13/VCF/00-All.vcf.gz)), Clinvar (version 20221231, [https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf\_GRCh37/archive\_2.0/2023/clinvar\_20221231.vcf.gz](https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh37/archive_2.0/2023/clinvar_20221231.vcf.gz)) and manually curated variants from Uniprot ([https://ftp.uniprot.org/pub/databases/uniprot/current\_release/knowledgebase/variants/homo\_sapiens\_variation.txt.gz](https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/variants/homo_sapiens_variation.txt.gz)).

Code availability
-----------------

Code to reproduce the analysis is available via GitHub at [https://github.com/VerenaK90/SCIFER](https://github.com/VerenaK90/SCIFER) (ref. [95](https://www.nature.com/articles/s41588-025-02217-y#ref-CR95 "Körber, V. VerenaK90/SCIFER. Zenodo 
https://doi.org/10.5281/zenodo.14507248
(2024).")) and [https://github.com/VerenaK90/Clonal\_hematopoiesis](https://github.com/VerenaK90/Clonal_hematopoiesis) (ref. [96](https://www.nature.com/articles/s41588-025-02217-y#ref-CR96 "Körber, V. VerenaK90/Clonal_hematopoiesis. Zenodo 
https://doi.org/10.5281/zenodo.14627371
(2025).")).

References
----------

1.  Spencer Chapman, M. et al. Lineage tracing of human development through somatic mutations. _Nature_ **595**, 85–90 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVOksLjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33981037)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lineage%20tracing%20of%20human%20development%20through%20somatic%20mutations&journal=Nature&volume=595&pages=85-90&publication_year=2021&author=Spencer%20Chapman%2CM) 
    
2.  Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. _Nature_ **561**, 473–478 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1OmsrrJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30185910)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6163040)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Population%20dynamics%20of%20normal%20human%20blood%20inferred%20from%20somatic%20mutations&journal=Nature&volume=561&pages=473-478&publication_year=2018&author=Lee-Six%2CH) 
    
3.  Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. _Nature_ **574**, 532–537 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31645730)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20landscape%20of%20somatic%20mutation%20in%20normal%20colorectal%20epithelial%20cells&journal=Nature&volume=574&pages=532-537&publication_year=2019&author=Lee-Six%2CH) 
    
4.  Brunner, S. F. et al. Somatic mutations and clonal dynamics in healthy and cirrhotic human liver. _Nature_ **574**, 538–542 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31645727)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6837891)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutations%20and%20clonal%20dynamics%20in%20healthy%20and%20cirrhotic%20human%20liver&journal=Nature&volume=574&pages=538-542&publication_year=2019&author=Brunner%2CSF) 
    
5.  Moore, L. et al. The mutational landscape of normal human endometrial epithelium. _Nature_ **580**, 640–646 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVCjsb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32350471)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20landscape%20of%20normal%20human%20endometrial%20epithelium&journal=Nature&volume=580&pages=640-646&publication_year=2020&author=Moore%2CL) 
    
6.  Grossmann, S. et al. Development, maturation, and maintenance of human prostate inferred from somatic mutations. _Cell Stem Cell_ **28**, 1262–1274. e5 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlvVWhsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33657416)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8260206)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%2C%20maturation%2C%20and%20maintenance%20of%20human%20prostate%20inferred%20from%20somatic%20mutations&journal=Cell%20Stem%20Cell&volume=28&pages=1262-1274.%20e5&publication_year=2021&author=Grossmann%2CS) 
    
7.  De, S. Somatic mosaicism in healthy human tissues. _Trends Genet._ **27**, 217–223 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvVaqur0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21496937)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mosaicism%20in%20healthy%20human%20tissues&journal=Trends%20Genet.&volume=27&pages=217-223&publication_year=2011&author=De%2CS) 
    
8.  Vijg, J. & Dong, X. Pathogenic mechanisms of somatic mutation and genome mosaicism in aging. _Cell_ **182**, 12–23 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlKktLzF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32649873)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7354350)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathogenic%20mechanisms%20of%20somatic%20mutation%20and%20genome%20mosaicism%20in%20aging&journal=Cell&volume=182&pages=12-23&publication_year=2020&author=Vijg%2CJ&author=Dong%2CX) 
    
9.  Lopez-Garcia, C., Klein, A. M., Simons, B. D. & Winton, D. J. Intestinal stem cell replacement follows a pattern of neutral drift. _Science_ **330**, 822–825 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlKht7rL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20929733)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intestinal%20stem%20cell%20replacement%20follows%20a%20pattern%20of%20neutral%20drift&journal=Science&volume=330&pages=822-825&publication_year=2010&author=Lopez-Garcia%2CC&author=Klein%2CAM&author=Simons%2CBD&author=Winton%2CDJ) 
    
10.  Snippert, H. J. et al. Intestinal crypt homeostasis results from neutral competition between symmetrically dividing Lgr5 stem cells. _Cell_ **143**, 134–144 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1Cmu7zI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20887898)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intestinal%20crypt%20homeostasis%20results%20from%20neutral%20competition%20between%20symmetrically%20dividing%20Lgr5%20stem%20cells&journal=Cell&volume=143&pages=134-144&publication_year=2010&author=Snippert%2CHJ) 
    
11.  Martincorena, I. et al. High burden and pervasive positive selection of somatic mutations in normal human skin. _Science_ **348**, 880–886 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXosFaktbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25999502)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471149)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High%20burden%20and%20pervasive%20positive%20selection%20of%20somatic%20mutations%20in%20normal%20human%20skin&journal=Science&volume=348&pages=880-886&publication_year=2015&author=Martincorena%2CI) 
    
12.  Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. _Nature_ **606**, 335–342 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVant7jN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35650444)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9177423)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20longitudinal%20dynamics%20and%20natural%20history%20of%20clonal%20haematopoiesis&journal=Nature&volume=606&pages=335-342&publication_year=2022&author=Fabre%2CMA) 
    
13.  Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. _Nature_ **606**, 343–350 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVant7vO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35650442)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9177428)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20dynamics%20of%20haematopoiesis%20across%20the%20human%20lifespan&journal=Nature&volume=606&pages=343-350&publication_year=2022&author=Mitchell%2CE) 
    
14.  Poon, G. Y., Watson, C. J., Fisher, D. S. & Blundell, J. R. Synonymous mutations reveal genome-wide levels of positive selection in healthy tissues. _Nat. Genet._ **53**, 1597–1605 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVahsbvJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34737428)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synonymous%20mutations%20reveal%20genome-wide%20levels%20of%20positive%20selection%20in%20healthy%20tissues&journal=Nat.%20Genet.&volume=53&pages=1597-1605&publication_year=2021&author=Poon%2CGY&author=Watson%2CCJ&author=Fisher%2CDS&author=Blundell%2CJR) 
    
15.  Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. _Science_ **367**, 1449–1454 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslymtb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32217721)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20evolutionary%20dynamics%20and%20fitness%20landscape%20of%20clonal%20hematopoiesis&journal=Science&volume=367&pages=1449-1454&publication_year=2020&author=Watson%2CCJ) 
    
16.  Huang, A.Y. et al. Somatic cancer driver mutations are enriched and associated with inflammatory states in Alzheimer’s disease microglia. Preprint at _bioRxiv_ [https://doi.org/10.1101/2024.01.03.574078](https://doi.org/10.1101/2024.01.03.574078) (2024).
    
17.  Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. _Science_ **377**, 511–517 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVWrurvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35901164)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9420557)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20somatic%20mutations%20in%20131%20human%20brains%20reveals%20aging-associated%20hypermutability&journal=Science&volume=377&pages=511-517&publication_year=2022&author=Bae%2CT) 
    
18.  Busque, L. et al. Recurrent somatic TET2 mutations in normal elderly individuals with clonal hematopoiesis. _Nat. Genet._ **44**, 1179–1181 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlyltrnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23001125)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3483435)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Recurrent%20somatic%20TET2%20mutations%20in%20normal%20elderly%20individuals%20with%20clonal%20hematopoiesis&journal=Nat.%20Genet.&volume=44&pages=1179-1181&publication_year=2012&author=Busque%2CL) 
    
19.  Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. _N. Engl. J. Med._ **371**, 2488–2498 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25426837)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4306669)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Age-related%20clonal%20hematopoiesis%20associated%20with%20adverse%20outcomes&journal=N.%20Engl.%20J.%20Med.&volume=371&pages=2488-2498&publication_year=2014&author=Jaiswal%2CS) 
    
20.  Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. _N. Engl. J. Med._ **371**, 2477–2487 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25426838)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290021)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20and%20blood-cancer%20risk%20inferred%20from%20blood%20DNA%20sequence&journal=N.%20Engl.%20J.%20Med.&volume=371&pages=2477-2487&publication_year=2014&author=Genovese%2CG) 
    
21.  Loh, P.-R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. _Nature_ **559**, 350–355 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjsbbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29995854)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6054542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20into%20clonal%20haematopoiesis%20from%208%2C342%20mosaic%20chromosomal%20alterations&journal=Nature&volume=559&pages=350-355&publication_year=2018&author=Loh%2CP-R) 
    
22.  Xie, M. et al. Age-related mutations associated with clonal hematopoietic expansion and malignancies. _Nat. Med._ **20**, 1472–1478 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslOisrvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25326804)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4313872)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Age-related%20mutations%20associated%20with%20clonal%20hematopoietic%20expansion%20and%20malignancies&journal=Nat.%20Med.&volume=20&pages=1472-1478&publication_year=2014&author=Xie%2CM) 
    
23.  Terao, C. et al. Chromosomal alterations among age-related haematopoietic clones in Japan. _Nature_ **584**, 130–135 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Cmu7nE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32581364)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7489641)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosomal%20alterations%20among%20age-related%20haematopoietic%20clones%20in%20Japan&journal=Nature&volume=584&pages=130-135&publication_year=2020&author=Terao%2CC) 
    
24.  Saiki, R. et al. Combined landscape of single-nucleotide variants and copy number alterations in clonal hematopoiesis. _Nat. Med._ **27**, 1239–1249 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSktL3E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34239136)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combined%20landscape%20of%20single-nucleotide%20variants%20and%20copy%20number%20alterations%20in%20clonal%20hematopoiesis&journal=Nat.%20Med.&volume=27&pages=1239-1249&publication_year=2021&author=Saiki%2CR) 
    
25.  Bernstein, N. et al. Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis. _Nat. Genet._ **56**, 1147–1155 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtVyht7jE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38744975)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11176083)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20somatic%20mutations%20in%20whole%20blood%20from%20200%2C618%20individuals%20identifies%20pervasive%20positive%20selection%20and%20novel%20drivers%20of%20clonal%20hematopoiesis.&journal=Nat.%20Genet.&volume=56&pages=1147-1155&publication_year=2024&author=Bernstein%2CN) 
    
26.  Niroula, A. et al. Distinction of lymphoid and myeloid clonal hematopoiesis. _Nat. Med._ **27**, 1921–1927 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Ohs77F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34663986)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8621497)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinction%20of%20lymphoid%20and%20myeloid%20clonal%20hematopoiesis&journal=Nat.%20Med.&volume=27&pages=1921-1927&publication_year=2021&author=Niroula%2CA) 
    
27.  Jakobsen, N. A. et al. Selective advantage of mutant stem cells in human clonal hematopoiesis is associated with attenuated response to inflammation and aging. _Cell Stem Cell_ **31**, 1127–1144 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtl2ku7bE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38917807)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11512683)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Selective%20advantage%20of%20mutant%20stem%20cells%20in%20human%20clonal%20hematopoiesis%20is%20associated%20with%20attenuated%20response%20to%20inflammation%20and%20aging.&journal=Cell%20Stem%20Cell&volume=31&pages=1127-1144&publication_year=2024&author=Jakobsen%2CNA) 
    
28.  Encabo, H. H. et al. Loss of TET2 in human hematopoietic stem cells alters the development and function of neutrophils. _Cell Stem Cell_ **30**, 781–799. e9 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20TET2%20in%20human%20hematopoietic%20stem%20cells%20alters%20the%20development%20and%20function%20of%20neutrophils&journal=Cell%20Stem%20Cell&volume=30&pages=781-799.%20e9&publication_year=2023&author=Encabo%2CHH) 
    
29.  Wang, H. et al. Clonal hematopoiesis driven by mutated DNMT3A promotes inflammatory bone loss. _Cell_ **187**, 3690–3711 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1ehsrvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38838669)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11246233)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20driven%20by%20mutated%20DNMT3A%20promotes%20inflammatory%20bone%20loss.&journal=Cell&volume=187&pages=3690-3711&publication_year=2024&author=Wang%2CH) 
    
30.  Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. _Nature_ **559**, 400–404 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjs7rI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29988082)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6485381)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prediction%20of%20acute%20myeloid%20leukaemia%20risk%20in%20healthy%20individuals&journal=Nature&volume=559&pages=400-404&publication_year=2018&author=Abelson%2CS) 
    
31.  Jaiswal, S. et al. Clonal hematopoiesis and risk of atherosclerotic cardiovascular disease. _N. Engl. J. Med._ **377**, 111–121 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28636844)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6717509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20and%20risk%20of%20atherosclerotic%20cardiovascular%20disease&journal=N.%20Engl.%20J.%20Med.&volume=377&pages=111-121&publication_year=2017&author=Jaiswal%2CS) 
    
32.  Dharan, N. J. et al. HIV is associated with an increased risk of age-related clonal hematopoiesis among older adults. _Nat. Med._ **27**, 1006–1011 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1Kgt7jI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34099923)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HIV%20is%20associated%20with%20an%20increased%20risk%20of%20age-related%20clonal%20hematopoiesis%20among%20older%20adults&journal=Nat.%20Med.&volume=27&pages=1006-1011&publication_year=2021&author=Dharan%2CNJ) 
    
33.  Zekavat, S. M. et al. Hematopoietic mosaic chromosomal alterations increase the risk for diverse types of infection. _Nat. Med._ **27**, 1012–1024 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1Kgt7jJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34099924)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8245201)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hematopoietic%20mosaic%20chromosomal%20alterations%20increase%20the%20risk%20for%20diverse%20types%20of%20infection&journal=Nat.%20Med.&volume=27&pages=1012-1024&publication_year=2021&author=Zekavat%2CSM) 
    
34.  Kim, P. G. et al. Dnmt3a-mutated clonal hematopoiesis promotes osteoporosis. _J. Exp. Med._ **218**, e20211872 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFaksb%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34698806)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8552148)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dnmt3a-mutated%20clonal%20hematopoiesis%20promotes%20osteoporosis&journal=J.%20Exp.%20Med.&volume=218&publication_year=2021&author=Kim%2CPG) 
    
35.  Wong, W. J. et al. Clonal haematopoiesis and risk of chronic liver disease. _Nature_ **616**, 747–754 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXns1yqs7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37046084)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10405350)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20haematopoiesis%20and%20risk%20of%20chronic%20liver%20disease&journal=Nature&volume=616&pages=747-754&publication_year=2023&author=Wong%2CWJ) 
    
36.  Vlasschaert, C. et al. Clonal hematopoiesis of indeterminate potential is associated with acute kidney injury. _Nat. Med._ **30**, 810–817 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXlsV2ksrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38454125)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10957477)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20of%20indeterminate%20potential%20is%20associated%20with%20acute%20kidney%20injury&journal=Nat.%20Med.&volume=30&pages=810-817&publication_year=2024&author=Vlasschaert%2CC) 
    
37.  Bozic, I., Gerold, J. M. & Nowak, M. A. Quantifying clonal and subclonal passenger mutations in cancer evolution. _PLoS Comput. Biol._ **12**, e1004731 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26828429)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4734774)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantifying%20clonal%20and%20subclonal%20passenger%20mutations%20in%20cancer%20evolution&journal=PLoS%20Comput.%20Biol.&volume=12&publication_year=2016&author=Bozic%2CI&author=Gerold%2CJM&author=Nowak%2CMA) 
    
38.  Williams, M. J., Werner, B., Barnes, C. P., Graham, T. A. & Sottoriva, A. Identification of neutral tumor evolution across cancer types. _Nat. Genet._ **48**, 238–244 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xps1Cqtg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26780609)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4934603)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20neutral%20tumor%20evolution%20across%20cancer%20types&journal=Nat.%20Genet.&volume=48&pages=238-244&publication_year=2016&author=Williams%2CMJ&author=Werner%2CB&author=Barnes%2CCP&author=Graham%2CTA&author=Sottoriva%2CA) 
    
39.  Busch, K. et al. Fundamental properties of unperturbed haematopoiesis from stem cells in vivo. _Nature_ **518**, 542–546 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjt1Smsro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25686605)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fundamental%20properties%20of%20unperturbed%20haematopoiesis%20from%20stem%20cells%20in%20vivo&journal=Nature&volume=518&pages=542-546&publication_year=2015&author=Busch%2CK) 
    
40.  Körber, V. et al. Evolutionary trajectories of IDHWT glioblastomas reveal a common path of early tumorigenesis instigated years ahead of initial diagnosis. _Cancer Cell_ **35**, 692–704. e12 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30905762)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolutionary%20trajectories%20of%20IDHWT%20glioblastomas%20reveal%20a%20common%20path%20of%20early%20tumorigenesis%20instigated%20years%20ahead%20of%20initial%20diagnosis&journal=Cancer%20Cell&volume=35&pages=692-704.%20e12&publication_year=2019&author=K%C3%B6rber%2CV) 
    
41.  Tanner, G., Westhead, D. R., Droop, A. & Stead, L. F. Benchmarking pipelines for subclonal deconvolution of bulk tumour sequencing data. _Nat. Commun._ **12**, 6396 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVams7fO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34737285)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8569188)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Benchmarking%20pipelines%20for%20subclonal%20deconvolution%20of%20bulk%20tumour%20sequencing%20data&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Tanner%2CG&author=Westhead%2CDR&author=Droop%2CA&author=Stead%2CLF) 
    
42.  Robertson, N. A. et al. Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects. _Nat. Med._ **28**, 1439–1446 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhslSlu7fK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35788175)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9307482)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Longitudinal%20dynamics%20of%20clonal%20hematopoiesis%20identifies%20gene-specific%20fitness%20effects&journal=Nat.%20Med.&volume=28&pages=1439-1446&publication_year=2022&author=Robertson%2CNA) 
    
43.  Osorio, F. G. et al. Somatic mutations reveal lineage relationships and age-related mutagenesis in human hematopoiesis. _Cell Rep._ **25**, 2308–2316. e4 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlCls7vO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30485801)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6289083)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutations%20reveal%20lineage%20relationships%20and%20age-related%20mutagenesis%20in%20human%20hematopoiesis&journal=Cell%20Rep.&volume=25&pages=2308-2316.%20e4&publication_year=2018&author=Osorio%2CFG) 
    
44.  Moran-Crusio, K. et al. Tet2 loss leads to increased hematopoietic stem cell self-renewal and myeloid transformation. _Cancer Cell_ **20**, 11–24 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslCms7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21723200)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3194039)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tet2%20loss%20leads%20to%20increased%20hematopoietic%20stem%20cell%20self-renewal%20and%20myeloid%20transformation&journal=Cancer%20Cell&volume=20&pages=11-24&publication_year=2011&author=Moran-Crusio%2CK) 
    
45.  Quivoron, C. et al. TET2 inactivation results in pleiotropic hematopoietic abnormalities in mouse and is a recurrent event during human lymphomagenesis. _Cancer Cell_ **20**, 25–38 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslCms7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21723201)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TET2%20inactivation%20results%20in%20pleiotropic%20hematopoietic%20abnormalities%20in%20mouse%20and%20is%20a%20recurrent%20event%20during%20human%20lymphomagenesis&journal=Cancer%20Cell&volume=20&pages=25-38&publication_year=2011&author=Quivoron%2CC) 
    
46.  Buccoliero, A. M. et al. Pediatric high grade glioma classification criteria and molecular features of a case series. _Genes_ **13**, 624 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFeqs7bI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35456430)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9028123)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pediatric%20high%20grade%20glioma%20classification%20criteria%20and%20molecular%20features%20of%20a%20case%20series&journal=Genes&volume=13&publication_year=2022&author=Buccoliero%2CAM) 
    
47.  Kurdi, M. et al. The cancer driver genes _IDH1_ and _IDH2_ and _CD204_ in WHO-grade 4 astrocytoma: crosstalk between cancer metabolism and tumour associated macrophage recruitment in tumour microenvironment. _Biologics_ **17**, 15–22 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtlagsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36778762)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9912343)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20cancer%20driver%20genes%20IDH1%20and%20IDH2%20and%20CD204%20in%20WHO-grade%204%20astrocytoma%3A%20crosstalk%20between%20cancer%20metabolism%20and%20tumour%20associated%20macrophage%20recruitment%20in%20tumour%20microenvironment.&journal=Biologics&volume=17&pages=15-22&publication_year=2023&author=Kurdi%2CM) 
    
48.  Ernst, A. et al. Neurogenesis in the striatum of the adult human brain. _Cell_ **156**, 1072–1083 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXivFCis74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24561062)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neurogenesis%20in%20the%20striatum%20of%20the%20adult%20human%20brain&journal=Cell&volume=156&pages=1072-1083&publication_year=2014&author=Ernst%2CA) 
    
49.  Williams, M. J. et al. Quantification of subclonal selection in cancer from bulk sequencing data. _Nat. Genet._ **50**, 895–903 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVelsbvJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29808029)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6475346)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantification%20of%20subclonal%20selection%20in%20cancer%20from%20bulk%20sequencing%20data&journal=Nat.%20Genet.&volume=50&pages=895-903&publication_year=2018&author=Williams%2CMJ) 
    
50.  Körber, V. et al. Neuroblastoma arises in early fetal development and its evolutionary duration predicts outcome. _Nat. Genet._ **55**, 619–630 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36973454)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10101850)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neuroblastoma%20arises%20in%20early%20fetal%20development%20and%20its%20evolutionary%20duration%20predicts%20outcome&journal=Nat.%20Genet.&volume=55&pages=619-630&publication_year=2023&author=K%C3%B6rber%2CV) 
    
51.  Gabbutt, C. et al. Fluctuating methylation clocks for cell lineage tracing at high temporal resolution in human tissues. _Nat. Biotechnol._ **40**, 720–730 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xjtl2mtw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34980912)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9110299)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fluctuating%20methylation%20clocks%20for%20cell%20lineage%20tracing%20at%20high%20temporal%20resolution%20in%20human%20tissues&journal=Nat.%20Biotechnol.&volume=40&pages=720-730&publication_year=2022&author=Gabbutt%2CC) 
    
52.  Weinstock, J. S. et al. Aberrant activation of TCL1A promotes stem cell expansion in clonal haematopoiesis. _Nature_ **616**, 755–763 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXns1yqs7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37046083)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10360040)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aberrant%20activation%20of%20TCL1A%20promotes%20stem%20cell%20expansion%20in%20clonal%20haematopoiesis.&journal=Nature&volume=616&pages=755-763&publication_year=2023&author=Weinstock%2CJS) 
    
53.  Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. _Sci. Adv._ **8**, eabl6579 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1GhtL3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35385311)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8986098)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mendelian%20randomization%20supports%20bidirectional%20causality%20between%20telomere%20length%20and%20clonal%20hematopoiesis%20of%20indeterminate%20potential&journal=Sci.%20Adv.&volume=8&publication_year=2022&author=Nakao%2CT) 
    
54.  Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. _Nat. Genet._ **54**, 1155–1166 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVaqsrzP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35835912)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9355874)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20analyses%20of%20200%2C453%20individuals%20yield%20new%20insights%20into%20the%20causes%20and%20consequences%20of%20clonal%20hematopoiesis&journal=Nat.%20Genet.&volume=54&pages=1155-1166&publication_year=2022&author=Kar%2CSP) 
    
55.  DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. _N. Engl. J. Med._ **388**, 2422–2433 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1Kgtr7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37140166)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10501156)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Familial%20clonal%20hematopoiesis%20in%20a%20long%20telomere%20syndrome&journal=N.%20Engl.%20J.%20Med.&volume=388&pages=2422-2433&publication_year=2023&author=DeBoy%2CEA) 
    
56.  Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. _Nature_ **593**, 405–410 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVSrur7F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33911282)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutation%20landscapes%20at%20single-molecule%20resolution&journal=Nature&volume=593&pages=405-410&publication_year=2021&author=Abascal%2CF) 
    
57.  Vattathil, S. & Scheet, P. Extensive hidden genomic mosaicism revealed in normal tissue. _Am. J. Hum. Genet._ **98**, 571–578 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XjtFOis7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26942289)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4800050)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extensive%20hidden%20genomic%20mosaicism%20revealed%20in%20normal%20tissue&journal=Am.%20J.%20Hum.%20Genet.&volume=98&pages=571-578&publication_year=2016&author=Vattathil%2CS&author=Scheet%2CP) 
    
58.  Laurie, C. C. et al. Detectable clonal mosaicism from birth to old age and its relationship to cancer. _Nat. Genet._ **44**, 642–650 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmsFOrt7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22561516)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3366033)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Detectable%20clonal%20mosaicism%20from%20birth%20to%20old%20age%20and%20its%20relationship%20to%20cancer&journal=Nat.%20Genet.&volume=44&pages=642-650&publication_year=2012&author=Laurie%2CCC) 
    
59.  Lebel, C., Walker, L., Leemans, A., Phillips, L. & Beaulieu, C. Microstructural maturation of the human brain from childhood to adulthood. _Neuroimage_ **40**, 1044–1055 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD1c7pvFOgsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18295509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Microstructural%20maturation%20of%20the%20human%20brain%20from%20childhood%20to%20adulthood&journal=Neuroimage&volume=40&pages=1044-1055&publication_year=2008&author=Lebel%2CC&author=Walker%2CL&author=Leemans%2CA&author=Phillips%2CL&author=Beaulieu%2CC) 
    
60.  Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. _EMBnet J_ **17**, 10–12 (2011).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&journal=EMBnet%20J&volume=17&pages=10-12&publication_year=2011&author=Martin%2CM) 
    
61.  Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. _Bioinformatics_ **25**, 1754–1760 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19451168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&journal=Bioinformatics&volume=25&pages=1754-1760&publication_year=2009&author=Li%2CH&author=Durbin%2CR) 
    
62.  Van der Auwera, G.A. & O’Connor, B.D. _Genomics in the Cloud: Using Docker, GATK, and WDL in Terra_ (O’Reilly Media, 2020).
    
63.  Kim, S. et al. Strelka2: fast and accurate calling of germline and somatic variants. _Nat. Methods_ **15**, 591–594 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlCmt7%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30013048)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Strelka2%3A%20fast%20and%20accurate%20calling%20of%20germline%20and%20somatic%20variants&journal=Nat.%20Methods&volume=15&pages=591-594&publication_year=2018&author=Kim%2CS) 
    
64.  Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. _Bioinformatics_ **26**, 841–842 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXivFGkurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20110278)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&journal=Bioinformatics&volume=26&pages=841-842&publication_year=2010&author=Quinlan%2CAR&author=Hall%2CIM) 
    
65.  Danecek, P. et al. Twelve years of SAMtools and BCFtools. _Gigascience_ **10**, giab008 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33590861)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7931819)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&journal=Gigascience&volume=10&publication_year=2021&author=Danecek%2CP) 
    
66.  Wang, K., Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. _Nucleic Acids Res._ **38**, e164–e164 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20601685)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2938201)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ANNOVAR%3A%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&journal=Nucleic%20Acids%20Res.&volume=38&pages=e164-e164&publication_year=2010&author=Wang%2CK&author=Li%2CM&author=Hakonarson%2CH) 
    
67.  Nik-Zainal, S. et al. The life history of 21 breast cancers. _Cell_ **149**, 994–1007 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XntlymtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22608083)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3428864)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20life%20history%20of%2021%20breast%20cancers&journal=Cell&volume=149&pages=994-1007&publication_year=2012&author=Nik-Zainal%2CS) 
    
68.  Chen, X. et al. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. _Bioinformatics_ **32**, 1220–1222 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlGqurvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26647377)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Manta%3A%20rapid%20detection%20of%20structural%20variants%20and%20indels%20for%20germline%20and%20cancer%20sequencing%20applications&journal=Bioinformatics&volume=32&pages=1220-1222&publication_year=2016&author=Chen%2CX) 
    
69.  Gonzalez-Perez, A. et al. IntOGen-mutations identifies cancer drivers across tumor types. _Nat. Methods_ **10**, 1081–1082 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVeqs7%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24037244)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758042)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IntOGen-mutations%20identifies%20cancer%20drivers%20across%20tumor%20types&journal=Nat.%20Methods&volume=10&pages=1081-1082&publication_year=2013&author=Gonzalez-Perez%2CA) 
    
70.  Pich, O., Reyes-Salazar, I., Gonzalez-Perez, A. & Lopez-Bigas, N. Discovering the drivers of clonal hematopoiesis. _Nat. Commun._ **13**, 4267 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvFaqsLzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35871184)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9308779)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovering%20the%20drivers%20of%20clonal%20hematopoiesis&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Pich%2CO&author=Reyes-Salazar%2CI&author=Gonzalez-Perez%2CA&author=Lopez-Bigas%2CN) 
    
71.  Sondka, Z. et al. The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. _Nat. Rev. Cancer_ **18**, 696–705 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVOjsLbN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30293088)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6450507)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COSMIC%20Cancer%20Gene%20Census%3A%20describing%20genetic%20dysfunction%20across%20all%20human%20cancers&journal=Nat.%20Rev.%20Cancer&volume=18&pages=696-705&publication_year=2018&author=Sondka%2CZ) 
    
72.  Acuna-Hidalgo, R. et al. Ultra-sensitive sequencing identifies high prevalence of clonal hematopoiesis-associated mutations throughout adult life. _Am. J. Hum. Genet._ **101**, 50–64 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFSmsb%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28669404)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501773)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ultra-sensitive%20sequencing%20identifies%20high%20prevalence%20of%20clonal%20hematopoiesis-associated%20mutations%20throughout%20adult%20life&journal=Am.%20J.%20Hum.%20Genet.&volume=101&pages=50-64&publication_year=2017&author=Acuna-Hidalgo%2CR) 
    
73.  Desai, P. et al. Somatic mutations precede acute myeloid leukemia years before diagnosis. _Nat. Med._ **24**, 1015–1023 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjsLrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29988143)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6849383)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutations%20precede%20acute%20myeloid%20leukemia%20years%20before%20diagnosis&journal=Nat.%20Med.&volume=24&pages=1015-1023&publication_year=2018&author=Desai%2CP) 
    
74.  Landrum, M. J. et al. ClinVar: improving access to variant interpretations and supporting evidence. _Nucleic Acids Res._ **46**, D1062–D1067 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlGisLfL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29165669)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ClinVar%3A%20improving%20access%20to%20variant%20interpretations%20and%20supporting%20evidence&journal=Nucleic%20Acids%20Res.&volume=46&pages=D1062-D1067&publication_year=2018&author=Landrum%2CMJ) 
    
75.  Brennan, P. drawProteins: a Bioconductor/R package for reproducible and programmatic generation of protein schematics. _F1000Research_ **7**, 1105 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30210791)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6107989)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=drawProteins%3A%20a%20Bioconductor%2FR%20package%20for%20reproducible%20and%20programmatic%20generation%20of%20protein%20schematics&journal=F1000Research&volume=7&publication_year=2018&author=Brennan%2CP) 
    
76.  Consortium, U. UniProt: a worldwide hub of protein knowledge. _Nucleic Acids Res._ **47**, D506–D515 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UniProt%3A%20a%20worldwide%20hub%20of%20protein%20knowledge&journal=Nucleic%20Acids%20Res.&volume=47&pages=D506-D515&publication_year=2019&author=Consortium%2CU) 
    
77.  Ng, P. C. & Henikoff, S. SIFT: predicting amino acid changes that affect protein function. _Nucleic Acids Res._ **31**, 3812–3814 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXltVWjs7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12824425)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC168916)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SIFT%3A%20predicting%20amino%20acid%20changes%20that%20affect%20protein%20function&journal=Nucleic%20Acids%20Res.&volume=31&pages=3812-3814&publication_year=2003&author=Ng%2CPC&author=Henikoff%2CS) 
    
78.  Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. _Am. J. Hum. Genet._ **99**, 877–885 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGrurnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27666373)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065685)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=REVEL%3A%20an%20ensemble%20method%20for%20predicting%20the%20pathogenicity%20of%20rare%20missense%20variants&journal=Am.%20J.%20Hum.%20Genet.&volume=99&pages=877-885&publication_year=2016&author=Ioannidis%2CNM) 
    
79.  Hoang, D. T. et al. MPBoot: fast phylogenetic maximum parsimony tree inference and bootstrap approximation. _BMC Evol. Biol._ **18**, 11 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29390973)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5796505)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MPBoot%3A%20fast%20phylogenetic%20maximum%20parsimony%20tree%20inference%20and%20bootstrap%20approximation&journal=BMC%20Evol.%20Biol.&volume=18&publication_year=2018&author=Hoang%2CDT) 
    
80.  Bailey, N. _The Elements of Stochastic Processes_ (Wiley, 1964).
    
81.  Ohtsuki, H. & Innan, H. Forward and backward evolutionary processes and allele frequency spectrum in a cancer cell population. _Theor. Popul. Biol._ **117**, 43–50 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28866007)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Forward%20and%20backward%20evolutionary%20processes%20and%20allele%20frequency%20spectrum%20in%20a%20cancer%20cell%20population&journal=Theor.%20Popul.%20Biol.&volume=117&pages=43-50&publication_year=2017&author=Ohtsuki%2CH&author=Innan%2CH) 
    
82.  Klinger, E., Rickert, D. & Hasenauer, J. pyABC: distributed, likelihood-free inference. _Bioinformatics_ **34**, 3591–3593 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXps12isbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29762723)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=pyABC%3A%20distributed%2C%20likelihood-free%20inference&journal=Bioinformatics&volume=34&pages=3591-3593&publication_year=2018&author=Klinger%2CE&author=Rickert%2CD&author=Hasenauer%2CJ) 
    
83.  Paradis, E. & Schliep, K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. _Bioinformatics_ **35**, 526–528 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVWgs7%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30016406)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ape%205.0%3A%20an%20environment%20for%20modern%20phylogenetics%20and%20evolutionary%20analyses%20in%20R&journal=Bioinformatics&volume=35&pages=526-528&publication_year=2019&author=Paradis%2CE&author=Schliep%2CK) 
    
84.  Revell, L. J. phytools: an R package for phylogenetic comparative biology (and other things). _Methods Ecol. Evol._ **3**, 217–223 (2012).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=phytools%3A%20an%20R%20package%20for%20phylogenetic%20comparative%20biology%20%28and%20other%20things%29.&journal=Methods%20Ecol.%20Evol.&volume=3&pages=217-223&publication_year=2012&author=Revell%2CLJ) 
    
85.  Schliep, K., Potts, A. A., Morrison, D. A. & Grimm, G. W. Intertwining phylogenetic trees and networks. _Methods Ecol. Evol._ **8**, 1212–1220 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intertwining%20phylogenetic%20trees%20and%20networks.&journal=Methods%20Ecol.%20Evol.&volume=8&pages=1212-1220&publication_year=2017&author=Schliep%2CK&author=Potts%2CAA&author=Morrison%2CDA&author=Grimm%2CGW) 
    
86.  Louca, S. & Doebeli, M. Efficient comparative phylogenetics on large trees. _Bioinformatics_ **34**, 1053–1055 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlemtLvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29091997)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20comparative%20phylogenetics%20on%20large%20trees&journal=Bioinformatics&volume=34&pages=1053-1055&publication_year=2018&author=Louca%2CS&author=Doebeli%2CM) 
    
87.  Soetaert, K., Petzoldt, T. & Setzer, R. W. Solving differential equations in R: package deSolve. _J. Stat. Softw._ **33**, 1–25 (2010).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Solving%20differential%20equations%20in%20R%3A%20package%20deSolve&journal=J.%20Stat.%20Softw.&volume=33&pages=1-25&publication_year=2010&author=Soetaert%2CK&author=Petzoldt%2CT&author=Setzer%2CRW) 
    
88.  Castiglione, S. et al. A new method for testing evolutionary rate variation and shifts in phenotypic evolution. _Methods Ecol. Evol._ **9**, 974–983 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20new%20method%20for%20testing%20evolutionary%20rate%20variation%20and%20shifts%20in%20phenotypic%20evolution&journal=Methods%20Ecol.%20Evol.&volume=9&pages=974-983&publication_year=2018&author=Castiglione%2CS) 
    
89.  Villanueva, R.A.M. & Chen, Z.J. _ggplot2: Elegant Graphics for Data Analysis_ (Taylor & Francis, 2019).
    
90.  Wickham, H. Reshaping data with the reshape package. _J. Stat. Softw._ **21**, 1–20 (2007).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reshaping%20data%20with%20the%20reshape%20package&journal=J.%20Stat.%20Softw.&volume=21&pages=1-20&publication_year=2007&author=Wickham%2CH) 
    
91.  Lee-Six, H. & Campbell, P. Population dynamics of normal human blood inferred from spontaneous somatic mutations – substitution calls. _Mendeley Data V1_ [https://doi.org/10.17632/yzjw2stk7f.1](https://doi.org/10.17632/yzjw2stk7f.1) (2018).
    
92.  Campbell, P. Dataset for: Clonal dynamics of haematopoiesis across the human lifespan. _Mendeley Data V2_ [https://doi.org/10.17632/np54zjkvxr.2](https://doi.org/10.17632/np54zjkvxr.2) (2022).
    
93.  Almeida, J.G.D., Fabre, M., Gerstung, M. & Vassiliou, G. Data. _figshare_ [figshare.com/articles/dataset/Data/15029118](https://figshare.com/articles/dataset/Data/15029118) (2021).
    
94.  Körber, V., Vyas, P. & Höfer, T. Clonal hematopoiesis quantified in human bone marrow whole genome sequencing data. _Mendeley Data V1_ [https://doi.org/10.17632/gkzvmg5f6z.1](https://doi.org/10.17632/gkzvmg5f6z.1) (2025).
    
95.  Körber, V. VerenaK90/SCIFER. _Zenodo_ [https://doi.org/10.5281/zenodo.14507248](https://doi.org/10.5281/zenodo.14507248) (2024).
    
96.  Körber, V. VerenaK90/Clonal\_hematopoiesis. _Zenodo_ [https://doi.org/10.5281/zenodo.14627371](https://doi.org/10.5281/zenodo.14627371) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02217-y?format=refman&flavour=references)

Acknowledgements
----------------

We thank the participants and allied healthcare professionals who supported collection of bone marrow and blood samples. V.K. acknowledges funding by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation – project no. 526169089). T.H. was supported by the Deutsche Forschungsgemeinschaft (project no. 497777320), the German Federal Ministry of Education and Research (BMBF; project no. 01KD2206B – SATURN3), the DKFZ-MOST cooperation program, Project Ca 211 and DKFZ core funding. P.V. acknowledges funding from the Medical Research Council Molecular Haematology Unit Programme Grant (MC\_UU\_00029/8), Blood Cancer UK Programme Continuity Grant 13008, NIHR Senior Fellowship and the Oxford Biomedical Research Centre Haematology Theme. N.A.J. was supported by a Medical Research Council and Leukaemia UK Clinical Research Training Fellowship (MR/R002258/1). M.M., B.U. and M.A.S. were funded by the Haematology Theme of the Oxford NIHR Biomedical Research Centre. Research by T.H. was supported in part by grant NSF PHY-2309135, the Gordon and Betty Moore Foundation grant no. 2919.02, and the Chan Zuckerberg Initiative DAF grant to the Kavli Institute for Theoretical Physics (KITP). We thank D. Cruz Hernandez, A. Groom, N. Becker, M. Günther and all members of the Höfer and Vyas groups for discussions. The authors are grateful for support by the MRC WIMM Flow Cytometry and Single Cell Facilities and the DKFZ NGS core facility.

Funding
-------

Open access funding provided by Deutsches Krebsforschungszentrum (DKFZ).

Author information
------------------

Author notes

1.  These authors contributed equally: Paresh Vyas, Thomas Höfer.
    

### Authors and Affiliations

1.  Division of Theoretical Systems Biology, German Cancer Research Center (DKFZ), Heidelberg, Germany
    
    Verena Körber, Nina Claudino, Eva Thielecke, Franziska Esau & Thomas Höfer
    
2.  MRC Molecular Haematology Unit, Oxford Biomedical Research Center, Hematology Theme, Oxford Centre for Haematology, Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, University of Oxford, Oxford, UK
    
    Verena Körber, Niels Asger Jakobsen, Naser Ansari-Pour, Rachel Moore, Marlen Metzner, Eva Thielecke, Batchimeg Usukhbayar, Mirian Angulo Salazar & Paresh Vyas
    
3.  Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, Botnar Research Centre, University of Oxford, Oxford, UK
    
    Naser Ansari-Pour, Simon Newman, Benjamin J. L. Kendrick, Adrian H. Taylor, Roger Gundle, Bridget Watkins, Kim Wheway, Debra Beazley, Stephanie G. Dakin, Antony Palmer & Andrew J. Carr
    
4.  Cambridge Genomics Laboratory, Cambridge University Hospitals, Cambridge, UK
    
    Rachel Moore
    
5.  Nuffield Orthopaedic Centre, Oxford University Hospitals NHS Foundation Trust, Oxford, UK
    
    Simon Newman, Benjamin J. L. Kendrick, Adrian H. Taylor, Rasheed Afinowi-Luitz, Roger Gundle & Antony Palmer
    

Authors

1.  Verena Körber
    
    [View author publications](https://www.nature.com/search?author=Verena%20K%C3%B6rber)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Verena%20K%C3%B6rber) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Verena%20K%C3%B6rber%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Niels Asger Jakobsen
    
    [View author publications](https://www.nature.com/search?author=Niels%20Asger%20Jakobsen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Niels%20Asger%20Jakobsen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Niels%20Asger%20Jakobsen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Naser Ansari-Pour
    
    [View author publications](https://www.nature.com/search?author=Naser%20Ansari-Pour)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Naser%20Ansari-Pour) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Naser%20Ansari-Pour%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Rachel Moore
    
    [View author publications](https://www.nature.com/search?author=Rachel%20Moore)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rachel%20Moore) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rachel%20Moore%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Nina Claudino
    
    [View author publications](https://www.nature.com/search?author=Nina%20Claudino)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nina%20Claudino) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nina%20Claudino%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Marlen Metzner
    
    [View author publications](https://www.nature.com/search?author=Marlen%20Metzner)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marlen%20Metzner) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marlen%20Metzner%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Eva Thielecke
    
    [View author publications](https://www.nature.com/search?author=Eva%20Thielecke)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eva%20Thielecke) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eva%20Thielecke%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Franziska Esau
    
    [View author publications](https://www.nature.com/search?author=Franziska%20Esau)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Franziska%20Esau) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Franziska%20Esau%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Batchimeg Usukhbayar
    
    [View author publications](https://www.nature.com/search?author=Batchimeg%20Usukhbayar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Batchimeg%20Usukhbayar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Batchimeg%20Usukhbayar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Mirian Angulo Salazar
    
    [View author publications](https://www.nature.com/search?author=Mirian%20Angulo%20Salazar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mirian%20Angulo%20Salazar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mirian%20Angulo%20Salazar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Simon Newman
    
    [View author publications](https://www.nature.com/search?author=Simon%20Newman)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Simon%20Newman) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Simon%20Newman%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Benjamin J. L. Kendrick
    
    [View author publications](https://www.nature.com/search?author=Benjamin%20J.%20L.%20Kendrick)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Benjamin%20J.%20L.%20Kendrick) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Benjamin%20J.%20L.%20Kendrick%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Adrian H. Taylor
    
    [View author publications](https://www.nature.com/search?author=Adrian%20H.%20Taylor)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Adrian%20H.%20Taylor) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Adrian%20H.%20Taylor%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Rasheed Afinowi-Luitz
    
    [View author publications](https://www.nature.com/search?author=Rasheed%20Afinowi-Luitz)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rasheed%20Afinowi-Luitz) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rasheed%20Afinowi-Luitz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Roger Gundle
    
    [View author publications](https://www.nature.com/search?author=Roger%20Gundle)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Roger%20Gundle) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Roger%20Gundle%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Bridget Watkins
    
    [View author publications](https://www.nature.com/search?author=Bridget%20Watkins)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Bridget%20Watkins) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Bridget%20Watkins%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Kim Wheway
    
    [View author publications](https://www.nature.com/search?author=Kim%20Wheway)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kim%20Wheway) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kim%20Wheway%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Debra Beazley
    
    [View author publications](https://www.nature.com/search?author=Debra%20Beazley)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Debra%20Beazley) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Debra%20Beazley%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Stephanie G. Dakin
    
    [View author publications](https://www.nature.com/search?author=Stephanie%20G.%20Dakin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Stephanie%20G.%20Dakin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Stephanie%20G.%20Dakin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Antony Palmer
    
    [View author publications](https://www.nature.com/search?author=Antony%20Palmer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Antony%20Palmer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Antony%20Palmer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Andrew J. Carr
    
    [View author publications](https://www.nature.com/search?author=Andrew%20J.%20Carr)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Andrew%20J.%20Carr) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Andrew%20J.%20Carr%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Paresh Vyas
    
    [View author publications](https://www.nature.com/search?author=Paresh%20Vyas)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Paresh%20Vyas) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Paresh%20Vyas%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Thomas Höfer
    
    [View author publications](https://www.nature.com/search?author=Thomas%20H%C3%B6fer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Thomas%20H%C3%B6fer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Thomas%20H%C3%B6fer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

V.K., N.A.J., P.V. and T.H. conceived the project. S.N., B.J.L.K., A.H.T., R.A.-L., R.G., B.W., K.W., D.B., S.G.D., A.J.C. and A.P. collected bone marrow and blood samples. N.A.J. performed flow cytometry. N.A.J., M.M., R.M., B.U. and M.A.S. performed DNA extraction. N.C. prepared DNA sequencing libraries. V.K. and N.A.-P. analyzed whole-genome sequencing data. E.T. helped to analyze data for the revision. F.E. helped with phylogenetic inference. V.K. developed the mathematical model, with input from T.H., performed parameter estimation and all computations. V.K., T.H. and P.V. wrote the manuscript with input from all coauthors. T.H. and P.V. acquired funding and administered the project.

### Corresponding authors

Correspondence to [Verena Körber](mailto:verena.korber@ndcls.ox.ac.uk), [Paresh Vyas](mailto:paresh.vyas@imm.ox.ac.uk) or [Thomas Höfer](mailto:t.hoefer@dkfz.de).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Jamie Blundell, Seishi Ogawa and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02217-y#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Quantifying selection and drift from bulk whole genome sequencing data.](https://www.nature.com/articles/s41588-025-02217-y/figures/8)

**a**, Simulated cumulative variant allele frequency (VAF) distribution of somatic single nucleotide variants (SSNVs) at selected ages between 0 and 100 years for stem cell numbers ranging between 5,000 and 1 M; _λ_\=5/year, _μ_\=10/division. **b**, Simulated cumulative VAF distribution of SSNVs when a driver is acquired at 20 years of age, and the selected clone grows by 22% per year (_s_ = 0.02, _λ_\=10/year, _μ_\=1/division, _N_ss = 25,000). Shown are the time points at which the clone sizes reached 5%, 10%, 15%, 20%, 25%, or 30% VAF for stem cell numbers ranging between 5,000 and 1 M. **c**, Simulated cumulative VAF distribution of somatic SSNVs when a driver is acquired at 20 years of age and has reached a clone size of 10% VAF (_λ_\=10/year, _μ_\=1/division, _N_ss = 25,000). Shown are the VAF distributions for selective advantages (_s_) ranging between 0.05 and 0.95, and stem cell numbers ranging between 5,000 and 1 M. **d**, Parameter estimation with approximate Bayesian computation. First, the expected variant allele frequency histogram is analytically computed. Then, sequencing noise is simulated by drawing from a binomial distribution with average 90x coverage. The modelled cumulative distribution is compared to the measured data for varying VAFs (1% step size).

### [Extended Data Fig. 2 Modeling two selected clones in stem cell homeostasis.](https://www.nature.com/articles/s41588-025-02217-y/figures/9)

**a**, Schematic illustrating variant accumulation during development and subsequent homeostasis, with two selected clones (red and orange) emerging by branched evolution during homeostasis. **b**, Schematic illustrating variant accumulation during development and subsequent homeostasis, with two selected clones (red and orange) emerging by linear evolution during homeostasis. **c**, Simulated cumulative variant allele frequency (VAF) distribution of somatic single nucleotide variants (SSNVs) when two selected clones, founded at 30 years and 50 years, evolve in parallel. The selective advantage of the first clone is fixed at 0.018, while the selective advantages of the second clone varies between 0.033 and 0.038 (color-encoded; 25,000 stem cells; _λ_\=10/year, _μ_\=1/division; arrows highlight positions of the selected clones). **d**, Simulated cumulative VAF distribution of SSNVs when two selected clones, founded at 30 years and 50 years, evolve linearly. The selective advantage of the first clone is fixed at 0.018, while the selective advantages of the second clone varies between 0.041 and 0.035 (color-encoded; 25,000 stem cells; _λ_\=10/year, _μ_\=1/division; arrows highlight positions of the selected clones).

### [Extended Data Fig. 3 Quantifying drift and selection in published data.](https://www.nature.com/articles/s41588-025-02217-y/figures/10)

**a**, Somatic variant calling pipeline (using hg19). SSNV, somatic single nucleotide variant. SCNA, somatic copy number abnormality. SSV, somatic structural variant. **b**, Number of shared and unique SSNVs in 140 whole genomes from hematopoietic stem cand progenitor cell (HSPC) clones from published data[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018).") identified by Caveman (original publication) and our pipeline (shown in **a**). **c**, Number of SSNVs of different classes identified by each variant caller. **d**, Left, variant allele frequency (VAF) distribution (truncated at 0.01 VAF) of SSNVs identified by Caveman[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."). Right, model fit to the cumulative 1/VAF distribution shown in left (points and error bars, measured data and their standard deviation, which, assuming Poisson-distributed measurements, is the square root of the measured data; red area, 95% posterior probabilities of the model fit, estimated from simulations using 100 posterior samples). **e**, Two-dimensional posterior probability distributions for the parameters estimated by SCIFER using SSNVs identified with Mutect2 and Strelka in published data[2](https://www.nature.com/articles/s41588-025-02217-y#ref-CR2 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."). Axis limits, range of prior distributions. **f**, As in **e**, but using SSNVs identified with Caveman. Note that the posterior for the fraction of cell loss, \\({\\delta }\_{\\exp }/{\\lambda }\_{\\exp }\\), is relatively broad, but has little effect on the mutation rate and other parameters (see Supplementary Note [2](https://www.nature.com/articles/s41588-025-02217-y#MOESM1)). **g**, As in **d** (left panel) but for samples AX001, KX001 and KX002 without clonal selection (SSNVs from published data[13](https://www.nature.com/articles/s41588-025-02217-y#ref-CR13 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).")). **h**, As in **d** (right panel) but for the data shown in **g**. **i**, Left, phylogenetic tree of published sample id2259[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022).") (reconstruction from the original publication). Middle, VAF distribution (truncated at 0.01 VAF) of SSNVs (taken from[12](https://www.nature.com/articles/s41588-025-02217-y#ref-CR12 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022).")). Right, as in **d**, but for id2259 (grey area, 80% credible interval of the clone size, estimated from 1,000 posterior samples). **j**, Left, as in **d** but for sample KX004 when allowing for two selected subclones (based on the phylogeny, we assumed branched evolution, c.f. Figure [3g](https://www.nature.com/articles/s41588-025-02217-y#Fig3); blue and green areas, 80% credible intervals of the size of leading and second selected clone, estimated from 1,000 posterior samples). Right, posterior probability for clonal selection obtained with the 2-clone model (conditioned on clones with VAF ≥ 2%). **k**, Median and 80% credible interval (estimated from 1,000 posterior samples) for stem cell and selection parameters for KX004, obtained with a one-clone and two-clone model.

### [Extended Data Fig. 4 FACS gating for hematopoietic stem and progenitor cells.](https://www.nature.com/articles/s41588-025-02217-y/figures/11)

**a-f**, Cells were stained with a panel of antibodies (Methods), then single and live mononuclear cells were sorted (MNCs, **a-d**). T cell depleted MNCs (MNC(–T)) were sorted from the CD3-, CD34+ cell fraction (**e** and **f**), and, hematopoietic stem and progenitor cells were sorted from the CD3-Lin-CD34+ cell fraction (**f**). Image created with FlowJo v10.8.1; values in **a**\-**f** give percentages.

### [Extended Data Fig. 5 Somatic variants in hematopoietic stem and progenitor cells.](https://www.nature.com/articles/s41588-025-02217-y/figures/12)

Genome-wide profile of trinucleotide substitution patterns (x-axis) (either from 270x whole genome sequencing (WGS) data, where available, or from 90x WGS data) across the genome in CD34+ hematopoietic stem and progenitor cells from the samples indicated.

### [Extended Data Fig. 6 Copy number variants in hematopoietic stem and progenitor cells.](https://www.nature.com/articles/s41588-025-02217-y/figures/13)

**a**, Genome-wide (chromosomes indicated and arrayed across x-axis) profiles of copy number gains (red) and losses (blue) in CD34+ hematopoietic stem and progenitor cells (HSPCs) across the 10 samples without known clonal hematopoiesis (CH) driver. **b**, Genome-wide (chromosomes indicated and arrayed across x-axis) profiles of copy number gains (red) and losses (blue) in CD34+ HSPC across the 12 samples with known CH driver.

### [Extended Data Fig. 7 Quantifying clonal selection from bulk whole genome sequencing.](https://www.nature.com/articles/s41588-025-02217-y/figures/14)

**a**, Model fits to cumulative variant allele frequency (VAF) distributions measured by 90x whole genome sequencing (WGS) of CD34+ hematopoietic stem and progenitor cells (HSPCs) from twelve individuals with known clonal hematopoiesis (CH) driver mutations (points and error bars, measured data and standard deviation, which, assuming Poisson-distributed measurements, is the square root of the measured data). Violet areas, 95% posterior probabilities of model fit, estimated from simulations using 100 posterior samples. Grey areas, 80% credible interval for the clone size, estimated from 1,000 posterior samples; red points and error bars, mean and 95% confidence interval of the VAF of known CH drivers, based on binomial distributions with sample size/success probability of 140/0.23 (5-DU), 168/0.15 (6-AU), 102/0.11(7-T), 89/0.18 (9-DU), 77/0.06 (10-D), 126/0.21(12-AT), 204/0.16 (17-TT), 230/0.13 (17-TT), 226/0.06 (17-TT), 70/0.09 (18-DU), 127/0.09 (19-D), 55/0.05 (20-UT), 156/0.39 (21-DU), 103/0.27 (22-TU), corresponding to read coverage and measured VAF, respectively. **b**, As in **a**, but showing an expanded view (VAF ≥ 0.2) for sample 21-DU (90x WGS). Selection (inferred at 7% VAF) was not associated with the _DNMT3A_ mutation (43% VAF). At most 6 variants were acquired prior to the _DNMT3A_ mutation, suggesting acquisition during early development. **c**, As in **a**, but for 270x WGS from seven individuals with known CH driver mutations. Colored rectangles, 80% credible interval for the clone size of leading and second selected clone, estimated from 1,000 posterior samples (where applicable); red points and error bars, mean and 95% confidence interval of the VAF of known CH drivers, based on binomial distributions with sample size/success probability of 270/0.26 (5-DU), 272/0.18 (6-AU), 230/0.2 (9-DU), 264/0.14 (17-TT), 275/0.16 (17-TT), 279/0.07 (17-TT), 194/0.08 (18-DU), 249/0.03 (20-UT), 260/0.28 (22-TU), corresponding to read coverage and measured VAF, respectively. **d**, Scheme for a model variant where a selected clone (red) expands without replacing normal cells (blue). **e**, As in **a**, but using the model variant introduced in **d**. **f**, Estimated model parameters obtained with SCIFER and the model variant introduced in **c** for three cases with a mutation in _TET2_ (points and error bars, median and 80% credible intervals, estimated from 1,000 posterior samples). **g**, Estimated model parameters obtained with the one-clone model and the two-clone model. Points and error bars, median and 80% credible estimates, estimated from 1,000 posterior samples; dashed line, bisectrix.

### [Extended Data Fig. 8 Quantifying neutral drift and clonal selection in cases without known driver from bulk whole genome sequencing.](https://www.nature.com/articles/s41588-025-02217-y/figures/15)

**a**, Model fits to the cumulative variant allele frequency (VAF) distributions measured by 90x whole genome sequencing (WGS) of CD34+ hematopoietic stem and progenitor cell (HSPC) samples from ten individuals (1-N, 2-U, 3-N, 4-N, 8-UU, 11-UU, 13-U, 14-U, 15-N, 16-UU) without known clonal hematopoiesis (CH) driver mutation (points and error bars, mean and standard deviation of the measured data, where standard deviations were computed based on Poisson distributions with mean corresponding to the respective somatic single nucleotide variant (SSNV) count; purple areas show the 95% posterior probabilities of the model fit, estimated from simulations using 100 posterior samples). The model fits of 1-N, 4-N and 15-N show no evidence of selection; the model fits of 3-N shows weak evidence of selection, which is, however, invalidated when probed with 270x WGS (shown in **b**). **b**, Model fits to the cumulative VAF distributions measured by 270x WGS of CD34+ HSPC samples from seven individuals (3-N, 4-N, 8-UU, 11-UU, 13-U, 14-U, 15-N, 16-UU) without known CH driver mutation (points and error bars, mean and standard deviation of the measured data, where standard deviations were computed based on Poisson distributions with mean corresponding to the respective SSNV count; purple areas show the 95% posterior probabilities of the model fit, estimated from simulations using 100 posterior samples). Colored rectangles, 80% credible interval for the estimated clone size of the leading and second selected clone inferred with SCIFER (where applicable). The model fits show no evidence of selection for 3-N, 4-N and 15-N but evidence for one selected clone in 13-U and 14-U (grey rectangles) and for two selected clones in 8-UU and 11-UU (blue and green rectangles; no 270x WGS was available for 2-U).

### [Extended Data Fig. 9 Selection dynamics of leading and subsequent clones.](https://www.nature.com/articles/s41588-025-02217-y/figures/16)

**a**, Estimated age of the leading selected clone (for the twelve cases with known clonal hematopoiesis (CH) drivers and the six cases with unknown drivers introduced in Figs. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4) and [5](https://www.nature.com/articles/s41588-025-02217-y#Fig5); points and error bars, median and 80% credible intervals, estimated from 1,000 posterior samples; parameter estimates are compared between model fits obtained from 90x and 270x whole genome sequencing (WGS) data). **b**, Estimated age of the second selected clone (for the ten cases introduced in Figs. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4) and [5](https://www.nature.com/articles/s41588-025-02217-y#Fig5); points and error bars, median and 80% credible intervals, estimated from 1,000 posterior samples). **c**, Estimated clonal growth rate of the leading selected clone (for the twelve cases with known drivers and the six cases with unknown drivers introduced in Figs. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4) and [5](https://www.nature.com/articles/s41588-025-02217-y#Fig5); points and error bars, median and 80% credible intervals, estimated from 1,000 posterior samples; parameter estimates are compared between model fits obtained from 90x and 270x WGS data). **d**, Estimated clonal growth rate of the second selected clone (for the ten cases introduced in Figs. [4](https://www.nature.com/articles/s41588-025-02217-y#Fig4) and [5](https://www.nature.com/articles/s41588-025-02217-y#Fig5); points and error bars, median and 80% credible intervals, estimated from 1,000 posterior samples).

### [Extended Data Fig. 10 Quantifying clonal selection in human brain samples.](https://www.nature.com/articles/s41588-025-02217-y/figures/17)

**a**, Cohort characteristics, showing age, sex and phenotype of the profiled individuals, as well as location of the analyzed samples. **b**, Number of individuals with evidence for clonal selection or neutral evolution stratified by phenotype (left), sex (middle), and location (right). **c**, Estimated number of stem cells (left) and number of somatic single nucleotide variants (SSNVs) per division (right) plotted against age for the 185 analyzed brain samples (points and error bars, median and 80% credible intervals, estimated from 1,000 posterior samples; blue line and shaded area, LOESS regression and 95% confidence interval). **d**, Estimated age of the selected clone (left) and clonal growth rate (right) plotted against age for the 44 analyzed brain samples with evidence for clonal selection (points and error bars, median and 80% credible intervals, estimated from 1,000 posterior samples; blue line and shaded area, LOESS regression and 95% confidence interval).

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_MOESM1_ESM.pdf)

Supplementary Figs 1–3 and Notes 1–5.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02217-y/MediaObjects/41588_2025_2217_MOESM4_ESM.xlsx)

Supplementary Table 1. Simulated data. Supplementary Table 2. Clinical characteristics. Supplementary Table 3. List of variants detected by panel sequencing of BM MNCs in the MARCH cohort (hg38 coordinates). Supplementary Table 4. List of copy number alterations (hg19 coordinates, identified with Battenberg). Supplementary Table 5. List of putative driver mutations (hg19 coordinates). Supplementary Table 6. Structural variants that potentially act as drivers of hematological malignancies. Supplementary Table 7. Structural variants (hg19 coordinates, Manta output). Supplementary Table 8. Analysis of deep whole genome sequenced brain samples from Bae et al.[17](https://www.nature.com/articles/s41588-025-02217-y#ref-CR17 "Bae, T. et al. Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability. Science 377, 511–517 (2022)."). Supplementary Table 9. Prior probabilities to model site frequency spectra measured in human bone marrow/blood samples with a one-clone model. Supplementary Table 10. Prior probabilities to model site frequency spectra measured in human bone marrow/blood samples with a two-clone model.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Detecting%20and%20quantifying%20clonal%20selection%20in%20somatic%20stem%20cells&author=Verena%20K%C3%B6rber%20et%20al&contentID=10.1038%2Fs41588-025-02217-y&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-03&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02217-y)

### Cite this article

Körber, V., Jakobsen, N.A., Ansari-Pour, N. _et al._ Detecting and quantifying clonal selection in somatic stem cells. _Nat Genet_ **57**, 1718–1729 (2025). https://doi.org/10.1038/s41588-025-02217-y

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02217-y?format=refman&flavour=citation)

-   Received: 10 October 2023
    
-   Accepted: 01 May 2025
    
-   Published: 03 July 2025
    
-   Issue Date: July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02217-y
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02217-y.pdf)
