> id: https://www.nature.com/articles/s41588-025-02298-9

> link: https://www.nature.com/articles/s41588-025-02298-9

> title: Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores

# Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores
_Published on Wed, 27 Aug 2025 00:00:00 GMT_

Abstract
--------

Electronic health record (EHR)-based phenotype risk scores (PheRS) leverage individuals’ health trajectories to estimate disease risk, similar to how polygenic scores (PGS) use genetic information. While PGS generalizability has been studied, less is known about PheRS generalizability across healthcare systems and whether PheRS are complementary to PGS. We trained elastic-net-based PheRS to predict the onset of 13 common diseases for 845,929 individuals (age = 32–70 years) from three biobank-based studies in Finland (FinnGen), the UK (UKB) and Estonia (EstB). All PheRS were statistically significantly associated with the diseases of interest and most generalized well without retraining when applied to other studies. PheRS and PGS were only moderately correlated and models including both predictors improved onset prediction compared to PGS alone for 8 of 13 diseases. Our results indicate that EHR-based risk scores can transfer well between EHRs, capture largely independent information from PGS, and provide additive benefits for disease risk prediction.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-48938-2/MediaObjects/41467_2024_48938_Fig1_HTML.png)

### [A unified framework for estimating country-specific cumulative incidence for 18 diseases stratified by polygenic risk](https://www.nature.com/articles/s41467-024-48938-2?fromPaywallRec=false)

Article Open access 12 June 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41431-025-01850-9/MediaObjects/41431_2025_1850_Fig1_HTML.png)

### [The Spanish Polygenic Score reference distribution: a resource for personalized medicine](https://www.nature.com/articles/s41431-025-01850-9?fromPaywallRec=false)

Article Open access 24 April 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-025-05838-9/MediaObjects/41598_2025_5838_Fig1_HTML.png)

### [A computational framework for defining and validating reproducible phenotyping algorithms of 313 diseases in the UK Biobank](https://www.nature.com/articles/s41598-025-05838-9?fromPaywallRec=false)

Article Open access 09 July 2025

Main
----

With the advent of large-scale genetic studies and the widespread availability of electronic health record (EHR) data, it is possible to combine these resources to more efficiently predict the risk of a wide range of diseases[1](https://www.nature.com/articles/s41588-025-02298-9#ref-CR1 "Johnson, K. B. et al. Precision medicine, AI, and the future of personalized health care. Clin. Transl. Sci. 14, 86–93 (2021)."),[2](https://www.nature.com/articles/s41588-025-02298-9#ref-CR2 "Shickel, B., Tighe, P. J., Bihorac, A. & Rashidi, P. Deep EHR: a survey of recent advances in deep learning techniques for electronic health record (EHR) analysis. IEEE J. Biomed. Health Inform. 22, 1589–1604 (2018)."),[3](https://www.nature.com/articles/s41588-025-02298-9#ref-CR3 "Tang, A. S. et al. Harnessing EHR data for health research. Nat. Med. 30, 1847–1855 (2024)."). Disease risk estimation can guide the efficient allocation of screening, preventative interventions and treatments in the early stages of diseases. Two lines of research have emerged in the past years. Some researchers have focused on machine learning approaches for EHR data[2](https://www.nature.com/articles/s41588-025-02298-9#ref-CR2 "Shickel, B., Tighe, P. J., Bihorac, A. & Rashidi, P. Deep EHR: a survey of recent advances in deep learning techniques for electronic health record (EHR) analysis. IEEE J. Biomed. Health Inform. 22, 1589–1604 (2018)."),[4](https://www.nature.com/articles/s41588-025-02298-9#ref-CR4 "Ayala Solares, J. R. et al. Deep learning for electronic health records: a comparative review of multiple deep neural architectures. J. Biomed. Inform. 101, 103337 (2020).") and showed some promising results in deriving EHR-based predictors for pancreatic cancer[5](https://www.nature.com/articles/s41588-025-02298-9#ref-CR5 "Placido, D. et al. A deep learning algorithm to predict risk of pancreatic cancer from disease trajectories. Nat. Med. 29, 1113–1122 (2023).") and cardiovascular disease[6](https://www.nature.com/articles/s41588-025-02298-9#ref-CR6 "Zhao, J. et al. Learning from longitudinal data in electronic health record and genetic data to improve cardiovascular event prediction. Sci. Rep. 9, 717 (2019)."),[7](https://www.nature.com/articles/s41588-025-02298-9#ref-CR7 "Petrazzini, B. O. et al. Coronary risk estimation based on clinical data in electronic health records. J. Am. Coll. Cardiol. 79, 1155–1166 (2022)."),[8](https://www.nature.com/articles/s41588-025-02298-9#ref-CR8 "Forrest, I. S. et al. Machine learning-based marker for coronary artery disease: derivation and validation in two longitudinal cohorts. Lancet 401, 215–225 (2023)."), among others. Many studies have focused on genetic data. Polygenic scores (PGS) use combined information from a person’s genome to estimate their genetic risk of developing a specific disease or trait. Numerous studies have examined the predictive ability of PGS across multiple diseases, and there is an extensive discussion about their clinical and public health value[9](https://www.nature.com/articles/s41588-025-02298-9#ref-CR9 "Xiang, R. et al. Recent advances in polygenic scores: translation, equitability, methods and FAIR tools. Genome Med. 16, 33 (2024).").

EHR and PGS-based prediction models have different strengths and limitations. EHRs allow access to a vast variety of data, including but not limited to disease diagnosis history, laboratory measurements, free text reports and various socioeconomic information[10](https://www.nature.com/articles/s41588-025-02298-9#ref-CR10 "Beesley, L. J. et al. The emerging landscape of health research based on biobanks linked to electronic health records: existing resources, statistical challenges, and potential opportunities. Stat. Med. 39, 773–800 (2020)."). However, EHRs are also known to be noisy[1](https://www.nature.com/articles/s41588-025-02298-9#ref-CR1 "Johnson, K. B. et al. Precision medicine, AI, and the future of personalized health care. Clin. Transl. Sci. 14, 86–93 (2021)."),[2](https://www.nature.com/articles/s41588-025-02298-9#ref-CR2 "Shickel, B., Tighe, P. J., Bihorac, A. & Rashidi, P. Deep EHR: a survey of recent advances in deep learning techniques for electronic health record (EHR) analysis. IEEE J. Biomed. Health Inform. 22, 1589–1604 (2018)."), and the models are expected to suffer from poor generalizability because of differences in data availability, as well as in clinical and recording practices across healthcare systems[2](https://www.nature.com/articles/s41588-025-02298-9#ref-CR2 "Shickel, B., Tighe, P. J., Bihorac, A. & Rashidi, P. Deep EHR: a survey of recent advances in deep learning techniques for electronic health record (EHR) analysis. IEEE J. Biomed. Health Inform. 22, 1589–1604 (2018)."),[3](https://www.nature.com/articles/s41588-025-02298-9#ref-CR3 "Tang, A. S. et al. Harnessing EHR data for health research. Nat. Med. 30, 1847–1855 (2024)."),[5](https://www.nature.com/articles/s41588-025-02298-9#ref-CR5 "Placido, D. et al. A deep learning algorithm to predict risk of pancreatic cancer from disease trajectories. Nat. Med. 29, 1113–1122 (2023)."),[10](https://www.nature.com/articles/s41588-025-02298-9#ref-CR10 "Beesley, L. J. et al. The emerging landscape of health research based on biobanks linked to electronic health records: existing resources, statistical challenges, and potential opportunities. Stat. Med. 39, 773–800 (2020)."),[11](https://www.nature.com/articles/s41588-025-02298-9#ref-CR11 "Botsis, T., Hartvigsen, G., Chen, F. & Weng, C. Secondary use of EHR: data quality issues and informatics opportunities. Summit Transl. Bioinform. 2010, 1–5 (2010)."),[12](https://www.nature.com/articles/s41588-025-02298-9#ref-CR12 "Rajkomar, A. et al. Scalable and accurate deep learning with electronic health records. NPJ Digit. Med. 1, 18 (2018)."),[13](https://www.nature.com/articles/s41588-025-02298-9#ref-CR13 "Shi, X., Li, X. & Cai, T. Spherical regression under mismatch corruption with application to automated knowledge translation. J. Am. Stat. Assoc. 116, 1953–1964 (2021)."). So far, most research has been conducted on a single EHR with limited work on validating the models in different EHR systems and countries[2](https://www.nature.com/articles/s41588-025-02298-9#ref-CR2 "Shickel, B., Tighe, P. J., Bihorac, A. & Rashidi, P. Deep EHR: a survey of recent advances in deep learning techniques for electronic health record (EHR) analysis. IEEE J. Biomed. Health Inform. 22, 1589–1604 (2018)."),[12](https://www.nature.com/articles/s41588-025-02298-9#ref-CR12 "Rajkomar, A. et al. Scalable and accurate deep learning with electronic health records. NPJ Digit. Med. 1, 18 (2018)."),[14](https://www.nature.com/articles/s41588-025-02298-9#ref-CR14 "Wornow, M. et al. The shaky foundations of large language models and foundation models for electronic health records. NPJ Digit. Med. 6, 135 (2023)."),[15](https://www.nature.com/articles/s41588-025-02298-9#ref-CR15 "Xie, F. et al. Deep learning for temporal data representation in electronic health records: a systematic review of challenges and methodologies. J. Biomed. Inform. 126, 103980 (2022)."). Recent studies, however, show promising results when validating EHR-based predictors in different healthcare systems in the US and UK. For example, an EHR-based prediction model trained in a US study (BioMe) outperformed conventional clinical guidelines in predicting coronary artery disease (CAD) susceptibility, and the results could be externally replicated in the UK Biobank (UKB)[7](https://www.nature.com/articles/s41588-025-02298-9#ref-CR7 "Petrazzini, B. O. et al. Coronary risk estimation based on clinical data in electronic health records. J. Am. Coll. Cardiol. 79, 1155–1166 (2022)."),[8](https://www.nature.com/articles/s41588-025-02298-9#ref-CR8 "Forrest, I. S. et al. Machine learning-based marker for coronary artery disease: derivation and validation in two longitudinal cohorts. Lancet 401, 215–225 (2023)."). A similar recent study successfully transferred an EHR-based model trained in the BioMe study for the prediction of autoimmune diseases to All of Us, another US-based study. Another systematic effort to train deep learning-based prediction models on the UKB EHR data for 1,568 diseases showed that when transferring these models to the All of Us study, 1,347 (85.9%) of the models improved disease onset prediction over a baseline model with age and sex[16](https://www.nature.com/articles/s41588-025-02298-9#ref-CR16 "Steinfeldt, J. et al. Medical history predicts phenome-wide disease onset and enables the rapid response to emerging health threats. Nat. Commun. 16, 585 (2025).").

The PGS are less likely to suffer from measurement errors compared to EHR-based models; however, they are known to be poorly transferable across ancestries, thus risking increasing health disparities[17](https://www.nature.com/articles/s41588-025-02298-9#ref-CR17 "Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat. Genet. 51, 584–591 (2019)."),[18](https://www.nature.com/articles/s41588-025-02298-9#ref-CR18 "Mars, N. et al. Genome-wide risk prediction of common diseases across ancestries in one million people. Cell Genom. 2, 100118 (2022)."). PGS are also not routinely measured in healthcare settings, although some healthcare systems have piloted programs to return PGS to individuals[19](https://www.nature.com/articles/s41588-025-02298-9#ref-CR19 "Sabatello, M. et al. Return of polygenic risk scores in research: stakeholders’ views on the eMERGE-IV study. HGG Adv. 5, 100281 (2024)."),[20](https://www.nature.com/articles/s41588-025-02298-9#ref-CR20 "Leitsalu, L. et al. Lessons learned during the process of reporting individual genomic results to participants of a population-based biobank. Eur. J. Hum. Genet. 31, 1048–1056 (2023)."). Further, as PGS keep improving through larger and more representative genome-wide association studies (GWAS), there is a growing interest in the integration of other predictors and risk factors to better capture the disease risk of individuals. Some recently published studies have integrated, for example, proteomics[21](https://www.nature.com/articles/s41588-025-02298-9#ref-CR21 "Ritchie, S. C. et al. Integrative analysis of the plasma proteome and polygenic risk of cardiometabolic diseases. Nat. Metab. 3, 1476–1483 (2021)."),[22](https://www.nature.com/articles/s41588-025-02298-9#ref-CR22 "Møller, P. L. et al. Combining polygenic and proteomic risk scores with clinical risk factors to improve performance for diagnosing absence of coronary artery disease in patients with de novo chest pain. Circ. Genom. Precis. Med. 16, 442–451 (2023).") or metabolomics-based risk scores[23](https://www.nature.com/articles/s41588-025-02298-9#ref-CR23 "Nightingale Health Biobank Collaborative Group Metabolomic and genomic prediction of common diseases in 700,217 participants in three national biobanks. Nat. Commun. 15, 10092 (2024)."),[24](https://www.nature.com/articles/s41588-025-02298-9#ref-CR24 "Aguilar, O. T., Chang, C., Bismuth, E. & Rivas, M. A. Integrative machine learning approaches for predicting disease risk using multi-omics data from the UK Biobank. Preprint at bioRxiv 
https://doi.org/10.1101/2024.04.16.589819
(2024).") with PGS. Compared to omics, EHR data has the advantage that it is already routinely electronically collected in many countries and does not require invasive and often relatively expensive additional measurements[3](https://www.nature.com/articles/s41588-025-02298-9#ref-CR3 "Tang, A. S. et al. Harnessing EHR data for health research. Nat. Med. 30, 1847–1855 (2024)."). Notably, there is a gap in our understanding of how PGS complements both established clinical risk factors and EHR-based risk scores. Numerous studies have investigated the additive value of PGS with clinical risk factors for a subset of diseases, including type 2 diabetes (T2D) and CAD[25](https://www.nature.com/articles/s41588-025-02298-9#ref-CR25 "Lambert, S. A., Abraham, G. & Inouye, M. Towards clinical utility of polygenic risk scores. Hum. Mol. Genet. 28, R133–R142 (2019)."),[26](https://www.nature.com/articles/s41588-025-02298-9#ref-CR26 "Mars, N. et al. Polygenic and clinical risk scores and their impact on age at onset and prediction of cardiometabolic diseases and common cancers. Nat. Med. 26, 549–557 (2020)."),[27](https://www.nature.com/articles/s41588-025-02298-9#ref-CR27 "Mohsen, F., Al-Absi, H. R. H., Yousri, N. A., El Hajj, N. & Shah, Z. A scoping review of artificial intelligence-based methods for diabetes risk prediction. NPJ Digit. Med. 6, 197 (2023)."). For EHR-based risk scores and many other diseases, the added benefit of PGS for disease onset prediction and risk stratification remains understudied.

In this study, we aimed to directly compare, within and across studies, the predictive performance and generalizability of EHR-based scores versus PGS using a longitudinal prospective design. In this context, we define generalizability as the extent to which models trained in one setting (for example, a specific study or population) maintain their predictive accuracy and associations when applied to another, previously unseen context (for example, a different study or population). We conducted this comparison across 13 common diseases and three large biobank-based studies with high-quality EHR data—UK Biobank[28](https://www.nature.com/articles/s41588-025-02298-9#ref-CR28 "Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018).") (UKB; United Kingdom), FinnGen[29](https://www.nature.com/articles/s41588-025-02298-9#ref-CR29 "Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. Nature 613, 508–518 (2023).") (Finland) and Estonian Biobank (EstB; Estonia)[30](https://www.nature.com/articles/s41588-025-02298-9#ref-CR30 "Leitsalu, L. et al. Cohort profile: Estonian Biobank of the Estonian Genome Center, University of Tartu. Int. J. Epidemiol. 44, 1137–1147 (2015)."). We created the EHR-based scores using the PheRS framework[31](https://www.nature.com/articles/s41588-025-02298-9#ref-CR31 "Bastarache, L. et al. Phenotype risk scores identify patients with unrecognized Mendelian disease patterns. Science 359, 1233–1239 (2018)."),[32](https://www.nature.com/articles/s41588-025-02298-9#ref-CR32 "Lebovitch, D. S., Johnson, J. S., Dueñas, H. R. & Huckins, L. M. Phenotype risk scores: moving beyond ‘cases’ and ‘controls’ to classify psychiatric disease in hospital-based biobanks. Preprint at medRxiv 
https://doi.org/10.1101/2021.01.25.21249615
(2021).") with PheRS derived from longitudinal diagnostic codes translated into consistent disease diagnoses using phecodes[33](https://www.nature.com/articles/s41588-025-02298-9#ref-CR33 "Wu, P. et al. Mapping ICD-10 and ICD-10-CM codes to phecodes: workflow development and initial evaluation. JMIR Med. Inform. 7, e14325 (2019).").

Results
-------

### Study overview

We included 845,929 individuals (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)) aged 32–70 years on 1 January 2011 (Fig. [1a](https://www.nature.com/articles/s41588-025-02298-9#Fig1)). These individuals belong to three biobank-based studies (FinnGen, UKB, EstB) linked with national registers or EHRs. The individuals gathered a total of 293,019 new diagnoses during an 8-year prediction period (1 January 2011 to 31 December 2018) across the following 13 common and high-burden diseases: prostate cancer, breast cancer, colorectal cancer, lung cancer, T2D, atrial fibrillation (AF), major depressive disorder (MDD), coronary heart disease (CHD), hip osteoarthritis (hip OA), knee osteoarthritis (knee OA), asthma, gout and epilepsy. We observed the highest number of events for knee OA (_n_ = 43,767) and the lowest for lung cancer (_n_ = 4,796; Fig. [1c](https://www.nature.com/articles/s41588-025-02298-9#Fig1) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)).

**Fig. 1: Overview of the study design and number of diagnoses across the three biobank-based studies.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02298-9/figures/1)

**a**, Outline of the study design. A separate study was conducted for each of the 13 diseases in the three biobank-based studies. Each study consisted of an observation and a prediction period, separated by a washout period that starts two years before the baseline date and during which no predictors were recorded. Each disease’s case and control definitions were based on diagnoses acquired in the prediction period (1 January 2011 to 31 December 2018). We removed all individuals diagnosed before our baseline (1 January 2011) and only considered adults aged 32–70 years in 2011 (see [Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10) for more details). **b**, We compared the PGS with PheRS—trained on phecodes recorded during the observation period (1 January 1999 to 31 December 2008). The PGS were based on recent publicly available GWAS summary statistics using MegaPRS. Ultimately, each individual was assigned 13 different PGS and PheRS scores describing their risk of getting a disease diagnosis during the prediction period. We trained the PheRS on 50% of individuals separately in the three studies (FinnGen, UKB and EstB). In each study, we then used the other half of the population as a test set, where we used the scores as predictors in Cox-PH. **c**, Number of new diagnoses for each disease during the prediction period (1 January 2011 to 31 December 2018) for each of the 13 diseases in the three studies (green, EstB; peach, FinnGen; and brown, UKB). The figure is created with BioRender.com. COPD, chronic obstructive pulmonary disease; OA, osteoarthritis.

[Source data](https://www.nature.com/articles/s41588-025-02298-9#MOESM5)

### Construction of PGS and PheRS

We constructed the PGS and PheRS separately for each disease (Fig. [1b](https://www.nature.com/articles/s41588-025-02298-9#Fig1)). PGS were previously derived by the INTERVENE consortium[34](https://www.nature.com/articles/s41588-025-02298-9#ref-CR34 "Jermy, B. et al. A unified framework for estimating country-specific cumulative incidence for 18 diseases stratified by polygenic risk. Nat. Commun. 15, 5007 (2024)."). PheRS were based on phecodes[35](https://www.nature.com/articles/s41588-025-02298-9#ref-CR35 "Denny, J. C. et al. PheWAS: demonstrating the feasibility of a phenome-wide scan to discover gene-disease associations. Bioinformatics 26, 1205–1210 (2010).") recorded during a 10-year observation period (1 January 1999 to 31 December 2009; Fig. [1a](https://www.nature.com/articles/s41588-025-02298-9#Fig1)), separated from the prediction period (1 January 2011 to 31 December 2018) by a 2-year washout period, during which no phecodes were recorded, meaning all the predictors were collected at minimum two years before the disease occurrence. Overall, we considered 234 phecodes with a prevalence of at least 1% in any study. However, for each disease, we excluded closely related diagnoses as predictors based on the phecodes exclusion ranges ([Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10); Supplementary Table [3](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)). For example, we did not use phecodes for secondary diabetes, T1D, or abnormal glucose tolerance as predictors of T2D.

Each PheRS model was trained separately to predict disease occurrence in the prediction period using 50% of the individuals in each study. We used elastic net models, a type of regularized regression method that combines the properties of both Ridge (L2) and Lasso (L1) regression[32](https://www.nature.com/articles/s41588-025-02298-9#ref-CR32 "Lebovitch, D. S., Johnson, J. S., Dueñas, H. R. & Huckins, L. M. Phenotype risk scores: moving beyond ‘cases’ and ‘controls’ to classify psychiatric disease in hospital-based biobanks. Preprint at medRxiv 
https://doi.org/10.1101/2021.01.25.21249615
(2021)."). The effect of age and sex was regressed out from the PheRS, and, when comparing PheRS and PGS, the first ten genetic principal components (PCs) were also regressed out from the scores to make them comparable. A more detailed description of the PheRS construction can be found in the [Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10). Disease prevalence during the prediction period (1 January 2011 to 31 December 2018; Fig. [1c](https://www.nature.com/articles/s41588-025-02298-9#Fig1)) varied substantially across the three studies. For example, we found a higher prevalence of knee OA in the EstB (12.3%, _n_ = 14,180) compared to FinnGen (4.8%, _n_ = 12,874) and the UKB (3.6%, _n_ = 16,713), while T2D diagnoses showed a lower prevalence both in the EstB (3.6%, _n_ = 104,161) and UKB (3.6%, _n_ = 16,850) compared to FinnGen (6.8%, _n_ = 18,099; Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)).

### PheRS were significantly associated with all 13 diseases

We evaluated the association between PheRS and 13 diseases independently of age and sex using Cox proportional hazard models (Cox-PH[36](https://www.nature.com/articles/s41588-025-02298-9#ref-CR36 "Cox, D. R. Regression models and life-tables. J. R. Stat. Soc. B 34, 187–202 (1972).")) on a test set in each study. All PheRS were significantly associated (_P_ < 0.05) with higher disease risk (Fig. [2a](https://www.nature.com/articles/s41588-025-02298-9#Fig2) and Supplementary Tables [4](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [5](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)) with the largest association for gout (meta-analyzed hazard ratio (HR) per 1 s.d. of PheRS = 1.59, 95% confidence interval (CI) = 1.47–1.71), T2D (HR = 1.49, 95% CI = 1.37–1.61), and lung cancer (HR = 1.46, 95% CI = 1.39–1.54). Further, adding the PheRS to a baseline model with age and sex significantly (_P_ < 0.05; one-tailed _P_ values based on the _z_ scores of the c-index differences) improved the predictive accuracy (c index) in all three studies for 7 of 13 diseases—asthma, MDD, T2D, knee OA, hip OA, gout and AF (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02298-9#Fig6) and Supplementary Tables [6](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3); additionally, Supplementary Table [8](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) shows the area under the precision–recall curve results). However, in the meta-analysis, the differences in baseline (age + sex) c indices in the different studies meant that only the improvements for MDD, gout, epilepsy, and asthma were significant (_P_ < 0.05; Fig. [2b](https://www.nature.com/articles/s41588-025-02298-9#Fig2) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)). The improvements persisted for four of these diseases (asthma, MDD, T2D and knee OA) in all of the three studies when compared to a baseline including, additionally, the highest achieved education level and the Charlson comorbidity index[37](https://www.nature.com/articles/s41588-025-02298-9#ref-CR37 "Charlson, M. E., Pompei, P., Ales, K. L. & MacKenzie, C. R. A new method of classifying prognostic comorbidity in longitudinal studies: development and validation. J. Chronic Dis. 40, 373–383 (1987)."),[38](https://www.nature.com/articles/s41588-025-02298-9#ref-CR38 "Deyo, R. A., Cherkin, D. C. & Ciol, M. A. Adapting a clinical comorbidity index for use with ICD-9-CM administrative databases. J. Clin. Epidemiol. 45, 613–619 (1992).") (CCI; Supplementary Fig. [1a](https://www.nature.com/articles/s41588-025-02298-9#MOESM1)). Overall, integrating education and CCI only led to minor improvements in the model’s discriminative ability compared to a baseline with age and sex (Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02298-9#Fig7)).

**Fig. 2: PheRS performance across studies.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02298-9/figures/2)

**a**, Association between PheRS and disease onset during the prediction period, independent of age and sex. The HR point estimates and 95% CIs in each study—FinnGen, peach; UKB, brown; EstB, green; and meta-analyzed results, red. The HRs are shown for an increase of the PheRS by 1 s.d. after regressing out age and sex. **b**, Increase in predictive accuracy when adding the PheRS to a baseline model with age and sex. The meta-analyzed c-index point estimates and 95% CIs of the baseline models (_x_ axis) compared to models with added PheRS (_y_ axis). Diseases with significant differences are labeled (one-tailed _P_ < 0.05 based on the _z_ scores of the c index increases), and those passing Bonferroni correction for multiple hypothesis testing (_P_ < 0.05/13) are marked with an asterisk. _P_ values of the significant c index increase—gout, 1.7 × 10−2; MDD, 8.6 × 10−3; asthma, 7.6 × 10−3; and epilepsy, 6.5 × 10−5. All _P_ values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3), and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

[Source data](https://www.nature.com/articles/s41588-025-02298-9#MOESM6)

We found that all PheRS were correlated, mostly positively, with the total number of phecodes an individual had recorded (Pearsons’ _r_ ranging from 0.82 (for asthma in FinnGen) to −0.66 (for breast cancer in FinnGen); Extended Data Fig. [1c](https://www.nature.com/articles/s41588-025-02298-9#Fig6)). While the two predictors were strongly correlated, we found that the PheRS improved predictions also over an extended baseline model accounting for the number of unique diagnoses for 9 of 13, 11 of 13 and 4 of 13 diseases in FinnGen, UKB and EstB, respectively (Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02298-9#Fig6)) and that the magnitude of the PheRS HRs was only slightly reduced for 1 of 13 diseases in FinnGen, 0 of 13 in the UKB and 5 of 13 in the EstB (Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02298-9#Fig8)). To further test whether this meant that the PheRS were more predictive in older individuals who have had more time to accumulate diagnoses in their EHR, we stratified the FinnGen test set to a younger group aged 32–51 years and an older group aged 52–70 years. However, unexpectedly, we found a substantially stronger association of the PheRS in the younger age group for 4 of 13 diseases, and only for breast cancer was the relative risk in the older group substantially larger than for the younger group, while no differences were observed in the remaining diseases (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02298-9#Fig9)).

### PheRS generalize well between studies

We examined PheRS generalizability by comparing externally- and internally-trained PheRS. All PheRS models for a given disease share the same phecode predictors. When a phecode was not observed in a study but had a nonzero coefficient in a model trained in an external study, the coefficient was multiplied by zero and did not affect the prediction. Externally-trained PheRS were trained on the training set of the UKB and EstB study and tested on the same test set as the FinnGen and UKB internally-trained PheRS. Externally-trained PheRS were moderately to strongly correlated with internally-trained PheRS (average Pearson’s _r_ = 0.43, range = −0.05 to 0.76; Fig. [3a](https://www.nature.com/articles/s41588-025-02298-9#Fig3) and Supplementary Table [9](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)). Not surprisingly, PheRS that were poor predictors of the disease were also poorly correlated between their internally-trained and externally-trained versions (that is, colorectal and breast cancer). Most externally-trained PheRS were substantially associated with disease risk in FinnGen (Fig. [3b](https://www.nature.com/articles/s41588-025-02298-9#Fig3), left and middle) and showed substantial improvements in c index over age and sex (Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02298-9#Fig10)). In the EstB, the HRs for hip OA, gout and knee OA were not significantly lower when trained in the UKB compared to the PheRS models trained in the EstB (Fig. [3b](https://www.nature.com/articles/s41588-025-02298-9#Fig3), right). Furthermore, for the PheRS models for hip OA transferred to FinnGen and the UKB, as well as knee OA, AF, gout and prostate cancer PheRS trained in the EstB and transferred to the UKB, c-index improvements were not significantly different from those achieved by the internally-trained PheRS (Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02298-9#Fig10)). Nonetheless, we observed that most PheRS disease associations were significantly lower with the externally-trained PheRS (Fig. [3b](https://www.nature.com/articles/s41588-025-02298-9#Fig3)).

**Fig. 3: PheRS external validation.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02298-9/figures/3)

**a**, Correlations (Persons’ _r_) between the internally- and externally-trained PheRS. Partial correlation point estimates and 95% CIs of the PheRS (_y_ axis) after regressing out the effect of age and sex, with the c-index point estimates and 95% CIs of the internally-trained and internally-tested PheRS as references (_x_ axis). All PheRS were trained on 50% of individuals. The _P_ values of the correlations are listed in Supplementary Table [10](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). **b**, Associations of the internally-trained PheRS with each disease compared to the externally-trained models. HR point estimates and 95% CIs of the internally-trained PheRS (_x_ axis) versus the externally-trained PheRS (_y_ axis). Left and middle, the models transferred to FinnGen, with EstB on the middle and UKB on the left. Right, the PheRS trained in the EstB and tested in the UKB. The HRs are shown for a 1 s.d. increase of the PheRS after regressing out age and sex. Diseases with no significant differences (two-tailed _P_ ≥ 0.05 of the _z_ scores of the HR differences) are marked with an asterisk, and those not passing Bonferroni correction for multiple hypothesis testing (_P_ ≥ 0.05/13) are labeled. All _P_ values are listed in Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

[Source data](https://www.nature.com/articles/s41588-025-02298-9#MOESM7)

### Phecode importance varies across studies

Despite good PheRS generalizability, we found marked differences in the prevalence of different phecodes between the studies (Supplementary Table [11](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)). While we found 527 different three-digit phecodes recorded in all three studies for at least five individuals, only 234 had a prevalence of at least 1% in any of the studies and only 48 were common across all studies (Fig. [4a](https://www.nature.com/articles/s41588-025-02298-9#Fig4)). These differences can be partially explained by different types of diagnostic information from the EHR available in each study. For example, the inclusion of primary care diagnoses in the EstB study leads to a higher number of phecodes, with 33% (_n_ = 77) unique to that study (Fig. [4a](https://www.nature.com/articles/s41588-025-02298-9#Fig4)). The FinnGen and UKB studies, on the other hand, only used diagnoses from secondary care. Notably, we found that training PheRS only using codes common (prevalence of at least 1%) to all three studies increased PheRS generalizability from EstB to FinnGen (Extended Data Fig. [6](https://www.nature.com/articles/s41588-025-02298-9#Fig11)).

**Fig. 4: Phecode prevalence and coefficients in each study.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02298-9/figures/4)

**a**, A Venn diagram showing the number of phecodes present in each of the three studies and shared between all combinations of the three studies. We only considered phecodes with a prevalence of ≥1% in each study. Peach color indicates FinnGen-specific codes, brown indicates UKB-specific codes and green indicates EstB-specific codes. The same color coding applies to panels **b** and **d**. **b**, Phecode prevalences for selected example codes in the three studies. The black dashed line indicates a prevalence of 1%. **c**, Median of PheRS coefficients over the 13 diseases in each study. Only coefficients used by at least 7 of 13 models in the studies are shown (see [Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10) for phecode exclusion rules in the PheRS models). Different colors and the _y_\-axis labels indicate different phecode categories. Black dashed lines correspond to a coefficient value of 0. **d**, A detailed look at all the PheRS coefficients for MDD in the three studies. Black color marks common phecodes in the MDD PheRS models across the studies, while other colors indicate biobank-specific codes (peach, FinnGen; brown, UKB; and green, EstB). PheRS coefficients were standardized to 0 mean and 1 s.d. for each model separately for easier comparison of coefficient importance across the studies. Please note that this normalization will force approximately half of the coefficients to have a negative scaled value with respect to the mean at 0. Abdom., abdominal.

[Source data](https://www.nature.com/articles/s41588-025-02298-9#MOESM8)

The set of phecodes unique to each study included important predictors for many of the diseases. To highlight one example, in each study, we found neuralgia (code 766) to be among the top 20 predictors for hip OA, CHD and MDD in the EstB. In FinnGen, schizophrenia (code 295) was an important predictor in T2D, lung cancer and epilepsy; and in the UKB, tobacco use disorder (code 318) was among the most important predictors for T2D, lung cancer, CHD and MDD. However, other predictors such as hypertension (code 401), overweight (code 278), alcohol abuse (code 317) and peripheral nerve disorders (code 351) were prevalent diagnoses in all three studies and showed a large consistent effect across diseases (Fig. [4b,c](https://www.nature.com/articles/s41588-025-02298-9#Fig4) and Supplementary Tables [12](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [13](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)).

We took a closer look at the top predictors in the individual PheRS models. Fig. [4d](https://www.nature.com/articles/s41588-025-02298-9#Fig4) shows the shared and study-specific predictors in the PheRS models for MDD.

We found that the top predictors in each PheRS captured the following three main categories: substance abuse, sleep disorders and pain-related problems. The most consistent phecode related to substance abuse in all three studies was alcohol abuse (code 317; FinnGen rank 3, UKB rank 2 and EstB rank 4), while other diagnoses, such as tobacco use disorder (code 318) were only captured in the UKB study (rank 4). The most important predictors related to pain disorders in FinnGen were intervertebral disc disorders (code 722, rank 4) and migraine (code 340, rank 5), while in the UKB it was back pain (code 760, rank 7) and in the EstB peripheral nerve disorders (code 351, rank 9) and other headache syndromes (code 229, rank 10). Nevertheless, while the list of most important predictors varied, each of the PheRS models also captured other pain-related diagnoses with lower ranks (Supplementary Table [12](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)). An additional leave-one-out analysis (LOO) performed in FinnGen (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02298-9#Fig12)) showed that removing most top predictors only minimally reduced the models’ performances, underscoring the shared contribution of correlated features to the overall performance. Extended Data Fig. [8](https://www.nature.com/articles/s41588-025-02298-9#Fig13) shows, for six additional diseases, how common and study-specific phecodes contributed to PheRS prediction.

### PGS and PheRS are orthogonal predictors

Finally, we compared the PheRS and corresponding PGS associations. Both were significantly associated with all diseases in the meta-analysis (_P_ < 0.05). However, the magnitude of the associations varied across diseases. For 4 of 13 diseases (epilepsy, MDD, knee OA and lung cancer), the PheRS showed a stronger association with the diseases than the PGS, and for 4 of 13 there were no substantial differences (Extended Data Figs. [9a-1](https://www.nature.com/articles/s41588-025-02298-9#Fig14) and [10a](https://www.nature.com/articles/s41588-025-02298-9#Fig15)). However, when looking at the top 10% of most at-risk individuals compared to the 20% at average risk (40–60% percentile), the PheRS captured the risk better for 8 of 13 diseases (T2D, gout, lung cancer, asthma, MDD, epilepsy, hip OA and knee OA; Fig. [5a](https://www.nature.com/articles/s41588-025-02298-9#Fig5)). Moreover, PheRS provided additional information on top of PGS. Adding PheRS to a model with PGS, age, sex, and the first ten PCs, significantly increased the c index for 9 of 13 diseases in FinnGen, 2 of 4 diseases in the UKB, and 6 of 13 diseases in the EstB (Fig. [5c](https://www.nature.com/articles/s41588-025-02298-9#Fig5)). Similarly, adding the PGS to a model with PheRS, age, and sex and PCs led to significant improvements for 10 of 13 in FinnGen, 3 of 4 in the UKB and 6 of 13 in the EstB (Extended Data Fig. [9c](https://www.nature.com/articles/s41588-025-02298-9#Fig14)). The number of diseases with significant improvements due to adding the PheRS was similar to that achieved when adding the PheRS to age and sex (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02298-9#Fig6); see [Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10) and [Supplementary Methods](https://www.nature.com/articles/s41588-025-02298-9#MOESM1) and [Supplementary Results](https://www.nature.com/articles/s41588-025-02298-9#MOESM1) for more details)

**Fig. 5: Comparison of PGS and PheRS.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02298-9/figures/5)

**a**, Left, associations of PGS (_x_ axis) and PheRS (_y_ axis) scores with each disease. vs., versus. The meta-analyzed HR point estimates (95% CI) for the top 10% at risk compared to those in the average 20% risk group (40–60% percentile) based on the scores after regressing out age, sex and the first ten PCs. Right, PheRS HR point estimates per 1 s.d. increase in models without PGS (_x_ axis) and ones where we adjust for the effects of PGS (_y_ axis). **b**, Correlation point estimates (Persons’ _r_, _y_ axis) and 95% CIs between the PheRS and PGS scores separately in each study (FinnGen, triangle; UKB, square; and EstB, circle), with the c-index point estimates of the PGS models as a reference after regressing out the effect of age, sex and the first ten PCs (_x_ axis). The _P_ values of the correlations are listed in Supplementary Table [9](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). **c**, C-index improvements when adding PheRS to models with PGS. C-index point estimates and 95% CIs of the models with only PGS, sex, age and PCs (_x_ axis) compared to those with added PheRS (_y_ axis). Due to sample overlap with the GWASs, PGS could only be calculated for four diseases in UKB (see [Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10) for details). Diseases with significant differences are labeled (_P_ < 0.05; two-tailed _P_ values based on the _z_ scores of the HR differences and one-tailed _P_ values for the c-index increases) and those passing Bonferroni correction of multiple hypothesis testing (_P_ < 0.05/13) are marked with an asterisk. All _P_ values are listed in Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

[Source data](https://www.nature.com/articles/s41588-025-02298-9#MOESM9)

Overall, we found that the EHR data and genetic information capture largely orthogonal information as shown by the low correlation between the two scores (average Pearsons’ _r_ = 0.02, range = 0.00–0.08; Supplementary Table [9](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)), which was not just driven by the low predictiveness of the PGS (Fig. [5b](https://www.nature.com/articles/s41588-025-02298-9#Fig5) and Extended Data Fig. [9b](https://www.nature.com/articles/s41588-025-02298-9#Fig14) shows the correlations in relation to Nagelkerke’s pseudo-_R_2). Furthermore, when adding the PGS to the models, the HRs of the PheRS did not change substantially and vice versa for the PGS HRs, when adding the PheRS (Fig. [5a](https://www.nature.com/articles/s41588-025-02298-9#Fig5), right, and Extended Data Fig. [9a-2](https://www.nature.com/articles/s41588-025-02298-9#Fig14)).

Discussion
----------

In this study, we investigated the accuracy and generalizability of EHR-based models (PheRS) in predicting the 8-year risk for 13 common diseases in three large biobank-based studies (FinnGen, EstB and UKB) compared to PGS. Our results highlight the complementarity of PheRS and PGS for a range of diseases, suggesting that combining EHR and genetic data can be an advantageous strategy for the prediction of many common diseases. Both PheRS and PGS were derived to be independent from age and sex effects, thus providing orthogonal information to these two key risk factors. Furthermore, we were able to successfully externally validate the models trained in EstB and UKB, suggesting that the PheRS models capture relevant risk factors that are not only study or healthcare system specific.

While the performance of the PheRS models varied between diseases, the PheRS for asthma, MDD, T2D, knee OA and gout, in particular, showed consistent improvement over baseline across all three studies. For asthma, knee OA and gout, the PheRS were less predictive in the UKB compared to the other cohorts. This could be reflecting the richer health information available in FinnGen with longer register coverage, and in EstB with inclusion of primary care data, both allowing more accurate phenotyping and detection of diagnoses before the baseline date. Colorectal cancer, prostate cancer, and breast cancer PheRS models performed poorly across the studies, likely due to the low case counts for these diseases in our data. PheRS outperformed a baseline of age and sex for all 13 diseases across the studies (except for breast and prostate cancers in the EstB), and substantially improved over counting the number of previous diagnoses for 9, 11 and 4 diseases in FinnGen, UKB and EstB, respectively, indicating that different existing diagnoses contribute to disease risk differently depending on the target disease predicted. We expected to see low generalizability of the PheRS between studies due to differences in clinical and disease coding practices in different countries and healthcare systems. Nonetheless, we found that the PheRS replicated well for many of the diseases; although, as expected, most PheRS trained within-study performed better. The good generalizability of PheRS was also surprising given the large variability in prevalence of phecodes we found across studies, with only 20% of them observed in all three studies. However, our results are in line with a few previous studies that show that it is possible to create predictors that are generalizable across healthcare systems[6](https://www.nature.com/articles/s41588-025-02298-9#ref-CR6 "Zhao, J. et al. Learning from longitudinal data in electronic health record and genetic data to improve cardiovascular event prediction. Sci. Rep. 9, 717 (2019)."),[7](https://www.nature.com/articles/s41588-025-02298-9#ref-CR7 "Petrazzini, B. O. et al. Coronary risk estimation based on clinical data in electronic health records. J. Am. Coll. Cardiol. 79, 1155–1166 (2022)."),[8](https://www.nature.com/articles/s41588-025-02298-9#ref-CR8 "Forrest, I. S. et al. Machine learning-based marker for coronary artery disease: derivation and validation in two longitudinal cohorts. Lancet 401, 215–225 (2023)."),[16](https://www.nature.com/articles/s41588-025-02298-9#ref-CR16 "Steinfeldt, J. et al. Medical history predicts phenome-wide disease onset and enables the rapid response to emerging health threats. Nat. Commun. 16, 585 (2025).").

Upon closer examination of the phecodes prevalent in each study and their importance in the PheRS models, we find that the PheRS models that generalize well from the UKB or EstB to FinnGen use both study-specific phecodes and phecodes shared between the studies. In some cases, such as gout, a large part of the generalizability of the models could already be explained by a few major risk factors, such as hypertension, high BMI and diabetes that are consistent in all three studies[39](https://www.nature.com/articles/s41588-025-02298-9#ref-CR39 "Singh, J. A. & Gaffo, A. Gout epidemiology and comorbidities. Semin. Arthritis Rheum. 50, S11–S16 (2020)."). In other cases, the relevance of each predictor was more intricate. For example, in UKB, one of the most important phecodes for MDD was tobacco use disorder, but this code had very low prevalence in FinnGen. Instead, we found that the phecode for alcohol abuse (code 317) was a prevalent and important predictor in all three studies. Both alcohol abuse and sleep disorders, another important and prevalent predictor in all the studies, are known complex comorbidities of MDD[40](https://www.nature.com/articles/s41588-025-02298-9#ref-CR40 "Zhao, Y. et al. The brain structure, immunometabolic and genetic mechanisms underlying the association between lifestyle and depression. Nat. Ment. Health 1, 736–750 (2023)."). We hypothesize that many of the different phecodes captured a single underlying risk factor. For example, several different pain-related diagnoses were among the top predictors for MDD, each likely capturing underlying pain problems[41](https://www.nature.com/articles/s41588-025-02298-9#ref-CR41 "Frediani, F. & Villani, V. Migraine and depression. Neurol. Sci. 28, S161–S165 (2007)."), with the top predictors differing between models. The elastic net penalty allows a nonzero coefficient for many correlating phecodes, which alleviates the issue of the same underlying medical issue being coded differently in different EHRs. This suggests that leveraging similarity between diagnostic codes is an essential aspect in creating generalizable EHR-based predictors.

We kept the PheRS approach simple to demonstrate its feasibility. More complex models could further exploit the longitudinal nature of EHR information and use other data modalities available in the EHR systems[5](https://www.nature.com/articles/s41588-025-02298-9#ref-CR5 "Placido, D. et al. A deep learning algorithm to predict risk of pancreatic cancer from disease trajectories. Nat. Med. 29, 1113–1122 (2023)."),[15](https://www.nature.com/articles/s41588-025-02298-9#ref-CR15 "Xie, F. et al. Deep learning for temporal data representation in electronic health records: a systematic review of challenges and methodologies. J. Biomed. Inform. 126, 103980 (2022)."),[42](https://www.nature.com/articles/s41588-025-02298-9#ref-CR42 "Kline, A. et al. Multimodal machine learning in precision health: a scoping review. NPJ Digit. Med. 5, 171 (2022)."). Further, by using a 2-year washout period and excluding very closely related conditions from the predictors, we remained conservative in removing comorbidities directly related to the disease. Without this buffer, the performance of the models will likely increase and be more relevant in a clinical context[6](https://www.nature.com/articles/s41588-025-02298-9#ref-CR6 "Zhao, J. et al. Learning from longitudinal data in electronic health record and genetic data to improve cardiovascular event prediction. Sci. Rep. 9, 717 (2019)."). To improve the generalizability of the models, we collapsed phecodes into the first three digits to reduce the effect of different diagnostic codes being used in different countries to describe the same underlying phenomenon[43](https://www.nature.com/articles/s41588-025-02298-9#ref-CR43 "Kiser, A. C. et al. Standard vocabularies to improve machine learning model transferability with electronic health record data: retrospective cohort study using health care–associated infection. JMIR Med. Inform. 10, e39057 (2022)."). For example, in the EstB study, phecodes hypertensive heart disease (code 401.21) and essential hypertension (code 401.1) were equally prevalent diagnoses, capturing the risk factor hypertension (code 401), while in FinnGen hypertensive heart disease (code 401.21) had a prevalence of <1%. Other approaches could include mapping diagnostic codes to OMOP-concepts, which have been shown to facilitate EHR-based models that transfer between different countries[16](https://www.nature.com/articles/s41588-025-02298-9#ref-CR16 "Steinfeldt, J. et al. Medical history predicts phenome-wide disease onset and enables the rapid response to emerging health threats. Nat. Commun. 16, 585 (2025)."). Regardless, differences in the types of data available in different cohorts remain a challenge. For example, in our study, EstB was the only study with primary care information. Such fundamental differences would be best addressed by compiling comprehensive datasets. Even if all the studies included the same sources of health data, medical codes are used differently in different countries to describe the same underlying condition. This would require approaches that can map the meaning of the codes used, rather than the actual codes.

Notably, while we did not exclude individuals based on their genetic ancestry, the UKB still consists mainly, and FinnGen and the EstB almost exclusively, of individuals of European ancestry. Thus, our study does not properly assess the important issue that individuals of different ethnicities face inequalities in healthcare access[44](https://www.nature.com/articles/s41588-025-02298-9#ref-CR44 "Fiscella, K. & Sanders, M. R. Racial and ethnic disparities in the quality of health care. Annu. Rev. Public Health 37, 375–394 (2016)."),[45](https://www.nature.com/articles/s41588-025-02298-9#ref-CR45 "Mahajan, S. et al. Trends in differences in health status and health care access and affordability by race and ethnicity in the United States, 1999–2018. JAMA 326, 637–648 (2021)."). Important open questions for future work, in addition to the generalizability of EHR-based scores for non-European genetic ancestries, include how to optimally model diagnostic codes for best generalizability as well as leveraging data from different and diverse cohorts with, for example, federated learning approaches[46](https://www.nature.com/articles/s41588-025-02298-9#ref-CR46 "Antunes, R. S., André da Costa, C., Küderle, A., Yari, I. A. & Eskofier, B. Federated learning for healthcare: systematic review and architecture proposal. ACM Trans. Intell. Syst. Technol. 13, 54:1–54:23 (2022).").

To our knowledge, the correlation between PGS and EHR-based scores has not been comprehensively studied. For CAD, details in ref. [7](https://www.nature.com/articles/s41588-025-02298-9#ref-CR7 "Petrazzini, B. O. et al. Coronary risk estimation based on clinical data in electronic health records. J. Am. Coll. Cardiol. 79, 1155–1166 (2022).") showed that the inclusion of PGS did not improve prediction compared to an EHR-based score, while details in ref. [6](https://www.nature.com/articles/s41588-025-02298-9#ref-CR6 "Zhao, J. et al. Learning from longitudinal data in electronic health record and genetic data to improve cardiovascular event prediction. Sci. Rep. 9, 717 (2019).") showed that the inclusion of genetic information substantially improved models with both EHR-based predictors and the gold standard model for CAD risk prediction (American College of Cardiology/American Heart Association Pooled Cohort Risk Equations). For 8/13 diseases studied here, we observe a substantial improvement in onset prediction when integrating PheRS on top of PGS. While for many of the cancers (colorectal cancer, prostate cancer, and breast cancer), the PGS was more informative, for diseases such as MDD, epilepsy and knee OA, the PheRS better captured the risk. Interestingly, PheRS was specifically better than PGS in capturing high-risk individuals. Individuals in the top 10% of PheRS had higher HRs than those in the top 10% of PGS for 8 of 13 diseases, probably reflecting those individuals with key comorbidities. However, while we used the same methodology for the percentile assignment for the PheRS and PGS, prior work on the PGS, as detailed in ref. [47](https://www.nature.com/articles/s41588-025-02298-9#ref-CR47 "Ding, Y. et al. Large uncertainty in individual polygenic risk score estimation impacts PRS-based risk stratification. Nat. Genet. 54, 30–39 (2022)."), has shown that there can be large uncertainties in the assignment of an individual to a risk group. Similar comprehensive robustness analyses of PheRS percentile assignment are an interesting direction for future work. Overall, we observed very low correlation between PGS and PheRS, indicating that these two data sources contain largely independent information. A few prior studies on the interaction between PGS and selected risk factors found no evidence for interaction[40](https://www.nature.com/articles/s41588-025-02298-9#ref-CR40 "Zhao, Y. et al. The brain structure, immunometabolic and genetic mechanisms underlying the association between lifestyle and depression. Nat. Ment. Health 1, 736–750 (2023)."),[48](https://www.nature.com/articles/s41588-025-02298-9#ref-CR48 "Lewis, C. M. & Vassos, E. Polygenic risk scores from research tools to clinical instruments. Genome Med. 12, 44 (2020).").

A patient’s diagnostic history has always been a key piece of information for medical professionals when considering future treatment. As we move towards translating PGS for clinical use, it is worth considering a comprehensive integration of information about an individual’s diagnosis history, which is often collected in a centralized electronic manner in many countries. This would not be a large shift from current practice, as selected comorbidities are used in many clinical risk stratification algorithms, for example, QRisk[49](https://www.nature.com/articles/s41588-025-02298-9#ref-CR49 "Hippisley-Cox, J., Coupland, C. & Brindle, P. Development and validation of QRISK3 risk prediction algorithms to estimate future risk of cardiovascular disease: prospective cohort study. BMJ 357, j2099 (2017).") for evaluating the risk of heart attack or stroke in the next 10 years, or QDiabetes for evaluating the 10-year risk of T2D[50](https://www.nature.com/articles/s41588-025-02298-9#ref-CR50 "Hippisley-Cox, J. & Coupland, C. Development and validation of QDiabetes-2018 risk prediction algorithm to estimate future risk of type 2 diabetes: cohort study. BMJ 359, j5019 (2017)."). A recent study showed that EHR-based models explicitly trained to predict the risk of five different cardiovascular events performed similarly or better than conventional risk scores (QRISK3, ASCVD and SCORE2)[16](https://www.nature.com/articles/s41588-025-02298-9#ref-CR16 "Steinfeldt, J. et al. Medical history predicts phenome-wide disease onset and enables the rapid response to emerging health threats. Nat. Commun. 16, 585 (2025)."). Similarly, details in ref. [6](https://www.nature.com/articles/s41588-025-02298-9#ref-CR6 "Zhao, J. et al. Learning from longitudinal data in electronic health record and genetic data to improve cardiovascular event prediction. Sci. Rep. 9, 717 (2019).") show that machine learning models trained on longitudinal EHR data outperformed the gold standard risk model (American College of Cardiology/American Heart Association Pooled Cohort Risk Equations) for the prediction of cardiovascular disease. These comparisons are interesting for diseases with established risk scores. However, for many of the diseases studied here, there are no established risk algorithms, making an EHR-based risk stratification approach even more relevant.

In this study, we showed that across many diseases and multiple studies with different underlying healthcare systems and EHRs, relatively simple elastic net-based risk scores that consider an individual’s previous diagnosis history can improve disease risk prediction when combined with PGS. Information already available from the EHR provides orthogonal information to PGS and could be a cost-effective approach for risk estimation.

Methods
-------

### Ethics declarations

Patients and control participants in FinnGen provided informed consent for biobank research, based on the Finnish Biobank Act. Alternatively, separate research cohorts, collected before the Finnish Biobank Act came into effect (in September 2013) and the start of FinnGen (August 2017), were collected based on study-specific consents and later transferred to the Finnish Biobanks after approval by Fimea (Finnish Medicines Agency), the National Supervisory Authority for Welfare and Health. Recruitment protocols followed the biobank protocols approved by Fimea. The Coordinating Ethics Committee of the Hospital District of Helsinki and Uusimaa (HUS) approved the FinnGen study under ethics statement HUS/990/2017. The study is also approved by the Finnish Institute for Health and Welfare (permits THL/2031/6.02.00/2017, THL/1101/5.05.00/2017, THL/341/6.02.00/2018, THL/2222/6.02.00/2018, THL/283/6.02.00/2019, THL/1721/5.05.00/2019 and THL/1524/5.05.00/2020); the Digital and Population Data Services Agency (permits VRK43431/2017-3, VRK/6909/2018-3 and VRK/4415/2019-3); the Social Insurance Institution of Finland (permits KELA 58/522/2017, KELA 131/522/2018, KELA 70/522/2019, KELA 98/522/2019, KELA 134/522/2019, KELA 138/522/2019, KELA 2/522/2020 and KELA 16/522/2020); Findata (permits THL/2364/14.02/2020, THL/4055/14.06.00/2020, THL/3433/14.06.00/2020, THL/4432/14.06/2020, THL/5189/14.06/2020, THL/5894/14.06.00/2020, THL/6619/14.06.00/2020, THL/209/14.06.00/2021, THL/688/14.06.00/2021, THL/1284/14.06.00/2021, THL/1965/14.06.00/2021, THL/5546/14.02.00/2020, THL/2658/14.06.00/2021 and THL/4235/14.06.00/2021); Statistics Finland (permits TK-53-1041-17, TK/143/07.03.00/2020 (earlier TK-53-90-20), TK/1735/07.03.00/2021 and TK/3112/07.03.00/2021) and the Finnish Registry for Kidney Diseases (permission based on the meeting minutes dated 4 July 2019). The biobank access decisions for FinnGen samples and data used in FinnGen Data Freeze 10 include approvals from the following biobanks: THL Biobank (BB2017\_55, BB2017\_111, BB2018\_19, BB\_2018\_34, BB\_2018\_67, BB2018\_71, BB2019\_7, BB2019\_8, BB2019\_26, BB2020\_1 and BB2021\_65); Finnish Red Cross Blood Service Biobank (7 December 2017); Helsinki Biobank (HUS/359/2017, HUS/248/2020, HUS/150/2022 §§12–18 and §23); Auria Biobank (AB17-5154 and amendment 1 (17 August 2020), amendments BB\_2021-0140, BB\_2021-0156 (26 August 2021, 2 February 2022), BB\_2021-0169, BB\_2021-0179, BB\_2021-0161, AB20-5926 and amendment 1 (23 April 2020) with its modification (22 September 2021)); Biobank Borealis of Northern Finland (2017\_1013, 2021\_5010, 2021\_5018, 2021\_5015, 2021\_5023, 2021\_5017 and 2022\_6001); Biobank of Eastern Finland (1186/2018 and amendments §§22/2020, 53/2021, 13/2022, 14/2022 and 15/2022); Finnish Clinical Biobank Tampere (MH0004 and amendments (21 February 2020 and 6 October 2020), §§8/2021, 9/2022, 10/2022, 12/2022, 20/2022, 21/2022, 22/2022 and 23/2022); Central Finland Biobank (1-2017); Terveystalo Biobank (STB 2018001 and amendment dated 25 August 2020); Finnish Hematological Registry and Clinical Biobank (decision dated 18 June 2021) and Arctic Biobank (P0844: ARC\_2021\_1001).

Ethics approval for the UK Biobank study was obtained from the North West Centre for Research Ethics Committee (11/NW/0382). The UK Biobank data used in this study were obtained under approved application 78537.

The activities of the EstBB are regulated by the Human Genes Research Act, which was adopted in 2000 specifically for the operations of the EstBB. Individual-level data analysis in the EstBB was carried out under ethical approval 1.1-12/624 from the Estonian Committee on Bioethics and Human Research (Estonian Ministry of Social Affairs), using data according to release application S22, document 6-7/GI/16259 from the EstBB.

### Study setup

As outlined in Fig. [1b](https://www.nature.com/articles/s41588-025-02298-9#Fig1), each study consisted of a 10-year observation (6 years for EstBB due to shorter follow-up) and an 8-year prediction period, separated by a 2-year washout period. Each disease’s case and control definitions were based on diagnoses acquired in the 8-year prediction period (from 1 January 2011 to after 1 January 2019). The International Classification of Diseases (ICD) codes used to define the cases for each disease were based on previous harmonization between FinnGen and the EstBB phenotypes by the INTERVENE consortium[34](https://www.nature.com/articles/s41588-025-02298-9#ref-CR34 "Jermy, B. et al. A unified framework for estimating country-specific cumulative incidence for 18 diseases stratified by polygenic risk. Nat. Commun. 15, 5007 (2024).") (Supplementary Table [14](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)). We consider all individuals as controls who were not cases. We only considered adults aged 32–70 in 1 January 2011 and removed all individuals diagnosed with the disease before this time. The lower limit for age of inclusion was chosen due to the inclusion of education level in some of the models and was determined based on the median age of obtaining a doctoral degree in the FinnGen dataset. Using this lower limit, most individuals included have finished their highest level of education. Furthermore, we removed all individuals with a diagnosis outside the prediction period (from 1 January 2011 to after 1 January 2019) and those lost to follow-up before the start of the prediction period. The ICD-codes used to define the cases for each disease and the number of cases and controls in each study are listed in Supplementary Tables [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [14](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

We included 845,929 individuals (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)) from three biobank-based studies—FinnGen[29](https://www.nature.com/articles/s41588-025-02298-9#ref-CR29 "Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. Nature 613, 508–518 (2023)."), UKB[28](https://www.nature.com/articles/s41588-025-02298-9#ref-CR28 "Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018).") and EstB[30](https://www.nature.com/articles/s41588-025-02298-9#ref-CR30 "Leitsalu, L. et al. Cohort profile: Estonian Biobank of the Estonian Genome Center, University of Tartu. Int. J. Epidemiol. 44, 1137–1147 (2015).") linked with national registers or EHRs. In FinnGen, we used Data Freeze 10, which includes 412,090 individuals, of whom 266,179 were aged 32–70 years in 1 January 2011. The longitudinal ICD-code diagnoses used to define the phecodes and the case and control status for each disease were based on in- and outpatient hospital register information. The UKB study included 464,076 individuals aged 40–70 years, with the ICD-code diagnoses based on inpatient information. The EstB study included 199,868 individuals, of whom 115,674 were aged 32–70 years. Here we also had primary care data as well as self-reported diagnoses available. More details on the phenotype harmonization can be found in ref. [34](https://www.nature.com/articles/s41588-025-02298-9#ref-CR34 "Jermy, B. et al. A unified framework for estimating country-specific cumulative incidence for 18 diseases stratified by polygenic risk. Nat. Commun. 15, 5007 (2024).") and the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02298-9#MOESM1).

### Predictors

#### PGS

The PGS were previously computed by the INTERVENE consortium[34](https://www.nature.com/articles/s41588-025-02298-9#ref-CR34 "Jermy, B. et al. A unified framework for estimating country-specific cumulative incidence for 18 diseases stratified by polygenic risk. Nat. Commun. 15, 5007 (2024).") and based on the recent publicly available GWAS summary statistics, with minimal overlap with our study cohorts (Supplementary Table [15](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)) using MegaPRS[51](https://www.nature.com/articles/s41588-025-02298-9#ref-CR51 "Zhang, Q., Privé, F., Vilhjálmsson, B. & Speed, D. Improved genetic prediction of complex traits from individual-level data or summary statistics. Nat. Commun. 12, 4192 (2021).") with the BLD-LDAK heritability model. For the Cox-PH models, we removed individuals from the studies that were part of the GWAS on which the PGS were based. Due to the large overlap with the UKB individuals, we only had PGS for gout, epilepsy, breast and prostate cancer available in the UKB.

#### PheRS

For the EHR-based models, we trained elastic net models[32](https://www.nature.com/articles/s41588-025-02298-9#ref-CR32 "Lebovitch, D. S., Johnson, J. S., Dueñas, H. R. & Huckins, L. M. Phenotype risk scores: moving beyond ‘cases’ and ‘controls’ to classify psychiatric disease in hospital-based biobanks. Preprint at medRxiv 
https://doi.org/10.1101/2021.01.25.21249615
(2021).") on ICD-9 and ICD-10 diagnoses mapped to phecodes. The phecode mapping was based on v1.2b1 of the phecode map[33](https://www.nature.com/articles/s41588-025-02298-9#ref-CR33 "Wu, P. et al. Mapping ICD-10 and ICD-10-CM codes to phecodes: workflow development and initial evaluation. JMIR Med. Inform. 7, e14325 (2019)."),[35](https://www.nature.com/articles/s41588-025-02298-9#ref-CR35 "Denny, J. C. et al. PheWAS: demonstrating the feasibility of a phenome-wide scan to discover gene-disease associations. Bioinformatics 26, 1205–1210 (2010).") from [https://phewascatalog.org/](https://phewascatalog.org/), with some manual additions. Since we only considered diagnoses during the observation period starting in 1999, all diagnoses were ICD-10-based in our data. To obtain the most comprehensive mapping, we removed all special characters from the ICD codes. If a match could not be found in the phecode map, we shortened the code by one digit until it could be mapped or was removed. The complete mapping used can be found in Supplementary Table [16](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). We gathered all phecodes in their three-digit parent node in the phecode ontology; for example, type 1 diabetes (250.1), T2D (250.2), and T2D with ketoacidosis (250.21) were all mapped to the same phecode diabetes mellitus (250). For each disease, we separately excluded predictors that were part of the exclusion range of the phecodes (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)), for example, for T2D, we did not use secondary diabetes (phecode 249), diabetes mellitus (250) and conditions complicating pregnancy (649) as predictors. The phecode conditions complicating pregnancy was excluded because it was the parent node of the phecode Diabetes or abnormal glucose tolerance complicating pregnancy (649.1), which is in the exclusion range of the phecode for T2D (250.2). We only considered phecodes with a prevalence of at least 1% of the study population (Supplementary Table [11](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)).

We implemented the PheRS using the LogisticRegression function from scikit-learn (version 1.3.2)[52](https://www.nature.com/articles/s41588-025-02298-9#ref-CR52 "Pedregosa, F. et al. Scikit-learn: machine learning in Python. J. Mach. Learn. Res. 12, 2825–2830 (2011)."). We included age (at the start of the prediction period 1 January 2011) and sex as predictors in the PheRS models because they are important predictors, and otherwise the models would reconstruct predictors for age and sex using combinations of the phecode diagnoses, which would make interpretation of the phecode coefficient values challenging. Nonetheless, the effect of age and sex was then regressed out when evaluating the performances of the PheRS (see below). Models were penalized with the elastic net penalty. Predictors were coded as 1/0, where 1 = ‘predictor observed during the observation window’ and 0 = ‘predictor not observed during the observation window’, for each disease separately. For training, 50% of the data was used, and this was further divided into training (85%) and hold-out test (15%) sets. Sizes of the training datasets are shown for each disease and study in Supplementary Table [22](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). L1 to L2 ratio hyperparameter of the elastic net models was optimized using grid search and fivefold cross-validation over the range 0.05–0.95 (step size = 0.05), simultaneously with inverse of the regularization strength (C) over the following possible values: 1 × 10−5, 5 × 10−5, 1 × 10−4, 5 × 10−4, 1 × 10−3, 5 × 10−3, 1 × 10−2, 5 × 10−2, 1 × 10−1, 5 × 10−1, 1. Balanced class weights were used, based on class frequencies in the training data. The LOO analysis assessing the impact of the removal of individual phecodes to PheRS performance was performed using a ridge penalty instead of elastic net. This was done to cut running time substantially, as using ridge removes the L1 to L2 ratio hyperparameter and its optimization. Otherwise, the ridge models were fitted similarly to the elastic net models. Before running the LOO analysis, we tested that switching to ridge did not generally reduce the PheRS performance in FinnGen (Extended Data Fig. [7a](https://www.nature.com/articles/s41588-025-02298-9#Fig12)).

Model fitting was done using stochastic average gradient descent. The best L1 to L2 ratio was selected based on the average precision score using 5-fold cross-validation on the training split. Missing values of predictors were imputed to the mean of the corresponding predictor in the study-specific training data, and all predictors were standardized to zero mean and unit variance on the study-specific training data before model fitting.

The PheRS models trained within the UKB or the EstB data on 50% of individuals were used to make predictions in FinnGen and UKB test sets, as is without any retraining within the studies. Standardization and imputation were performed based on the biobank-specific training data, meaning that, for example, when assessing the performance of the UKB-trained model in FinnGen, the FinnGen test set data were imputed and standardized based on the feature-specific means and s.d. from the UKB.

### Cox-PH models

Ultimately, each individual was assigned 13 different PGS and PheRS scores describing their risk of getting a disease diagnosis in the prediction period based on genetic or EHR-based information. To make the PheRS and PGS comparable, we regressed out the effect of age, sex and the first ten genetic PCs from all continuous scores using the residuals from a logistic regression with the score as outcome. When only considering PheRS performance, we regressed out only age and sex. Subsequently, we scaled all predictors to have a mean of zero and a s.d. of 1. We then used these scores in separate Cox-PH analyses, with survival time defined as the period from 2011 until diagnosis, censoring (end of follow-up), or the end of the prediction period.

Additionally, we considered the CCI ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02298-9#MOESM1))[37](https://www.nature.com/articles/s41588-025-02298-9#ref-CR37 "Charlson, M. E., Pompei, P., Ales, K. L. & MacKenzie, C. R. A new method of classifying prognostic comorbidity in longitudinal studies: development and validation. J. Chronic Dis. 40, 373–383 (1987)."),[38](https://www.nature.com/articles/s41588-025-02298-9#ref-CR38 "Deyo, R. A., Cherkin, D. C. & Ciol, M. A. Adapting a clinical comorbidity index for use with ICD-9-CM administrative databases. J. Clin. Epidemiol. 45, 613–619 (1992).")—developed to account for the individual’s overall comorbidity burden—and the individual’s highest achieved education level in 2011 as an indicator of their socioeconomic status. For the CCI, we compared the top 10% of individuals with the highest CCI to the rest. The high-risk group included individuals with a CCI ≥ 2 and a few younger ones with a CCI of 1. For the highest education level, we mapped each study’s education coding to the 2011 International Standard Classification of Education (ISCED-11; Supplementary Table [17](https://www.nature.com/articles/s41588-025-02298-9#MOESM3)) codes. We compared the risk of individuals with basic education (ISCED-11: 1–4) to those who achieved higher education levels (ISCED-11: 5–7).

### Statistics

We used the survival[53](https://www.nature.com/articles/s41588-025-02298-9#ref-CR53 "Therneau, T. M. Survival analysis. R package version 3.8-3 
cran.r-project.org/web/packages/survival/index.html
(2024).") package (version 3.2-7) in R for creating the Cox-PH models and the Hmisc[54](https://www.nature.com/articles/s41588-025-02298-9#ref-CR54 "Harrell, F. E. Jr & Dupont, C. Hmisc: Harrell Miscellaneous. R package version 5.2-0 
hbiostat.org/r/hmisc/
(2024).") package (version 5.1.0) to calculate the c indices and 95% CIs. For a Cox-PH model with binary outcomes, the predicted survival times can be shown to be equal to the survival probability, so the c index is equivalent to the area under the receiver operating characteristic curve (AUC)[55](https://www.nature.com/articles/s41588-025-02298-9#ref-CR55 "Harrell, F. E., Lee, K. L. & Mark, D. B. Multivariable prognostic models: issues in developing models, evaluating assumptions and adequacy, and measuring and reducing errors. Stat. Med. 15, 361–387 (1996)."),[56](https://www.nature.com/articles/s41588-025-02298-9#ref-CR56 "Pencina, M. J. & D’Agostino, R. B. Overall C as a measure of discrimination in survival analysis: model specific population value and confidence interval estimation. Stat. Med. 23, 2109–2123 (2004)."). The meta-analysis of the HRs and c indices was performed using the metafor[57](https://www.nature.com/articles/s41588-025-02298-9#ref-CR57 "Viechtbauer, W. metafor: meta-analysis package for R. R version 4.8-0 
https://cran.r-project.org/web/packages/metafor/index.html
(2024)."),[58](https://www.nature.com/articles/s41588-025-02298-9#ref-CR58 "Viechtbauer, W. Conducting meta-analyses in R with the metafor package. J. Stat. Softw. 36, 1–48 (2010).") package (version 4.6-0) in R with a random effects model. We used two-tailed _P_ values, calculated using the pnorm function in the stats package (version 3.6.2) in R, based on the _z_ scores of the β differences to compare the differences in HR magnitudes and one-tailed _P_ values for the statistical testing of increases in the c indices. Additionally, we used Bonferroni correction to account for multiple hypothesis testing in each study (_n_ = 13). Correlations were calculated, using the cor.test function from the stats package in R. For regressing out the covariates—age, sex and PCs—from the PheRS and PGS, we used scaled residuals from glm models with the stats package in R.

### Comparison of phecode coefficients between different PheRS models

The elastic net hyperparameters were separately optimized for each PheRS model. This means that the absolute magnitudes of the coefficients for phecodes are not comparable between different PheRS. However, the relative importances of phecodes can still be compared, that is, whether, for example, the same phecodes are among the most important predictors in two different PheRS. To make visualization of the phecode importances in different PheRS clearer, we standardized the coefficients of each PheRS separately to a mean of 0 and a s.d. of 1 for the display items. Further, in each study, we ranked the phecodes in descending order by the PheRS coefficient values and assigned them ascending ranks. Thus, a lower rank indicates a higher PheRS coefficient in the model. Both the unscaled PheRS coefficients and ranks are Supplementary Table [12](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02298-9#MOESM2) linked to this article.

Data availability
-----------------

The individual-level data in these studies are protected for data privacy, and access is regulated through the biobanks. The Finnish biobank data can be accessed through the Fingenious services ([https://site.fingenious.fi/en/](https://site.fingenious.fi/en/)) managed by FINBB. Researchers interested in EstBB can request access at [https://genomics.ut.ee/en/content/estonian-biobank#dataaccess](https://genomics.ut.ee/en/content/estonian-biobank#dataaccess) and the UKB data are available through a procedure described at [https://www.ukbiobank.ac.uk/use-our-data/apply-for-access/](https://www.ukbiobank.ac.uk/use-our-data/apply-for-access/). The GWAS data used in this study are available in the GWAS catalog database ([https://www.ebi.ac.uk/gwas/](https://www.ebi.ac.uk/gwas/)) under accession codes listed in Supplementary Table [15](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). The PGS scores generated in this study are available in the PGS Catalog ([https://www.pgscatalog.org/](https://www.pgscatalog.org/)) under publication ID PGP000618 and score IDs PGS004869–PGS004886. The mapping between ICD and phecodes used in this study is available at [https://phewascatalog.org](https://phewascatalog.org/). [Source data](https://www.nature.com/articles/s41588-025-02298-9#Sec23) are provided with this paper.

Code availability
-----------------

The code for PheRS model training is available at [https://github.com/intervene-EU-H2020/INTERVENE\_PheRS](https://github.com/intervene-EU-H2020/INTERVENE_PheRS) and [https://zenodo.org/records/15691460](https://zenodo.org/records/15691460) (ref. [59](https://www.nature.com/articles/s41588-025-02298-9#ref-CR59 "Hartonen & Detrois, K. Intervene-EU-H2020/INTERVENE_PheRS: V1.0.0. Zenodo 
https://doi.org/10.5281/zenodo.15691460
(2025).")) and for the Cox-PH models as well as the final analysis of results at [https://github.com/intervene-EU-H2020/onset\_prediction](https://github.com/intervene-EU-H2020/onset_prediction) and [https://zenodo.org/records/15681882](https://zenodo.org/records/15681882) (ref. [60](https://www.nature.com/articles/s41588-025-02298-9#ref-CR60 "Detrois, K. Intervene-EU-H2020/onset_prediction: v1.0.0. Zenodo 
https://doi.org/10.5281/zenodo.15681882
(2025).")).

References
----------

1.  Johnson, K. B. et al. Precision medicine, AI, and the future of personalized health care. _Clin. Transl. Sci._ **14**, 86–93 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32961010)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Precision%20medicine%2C%20AI%2C%20and%20the%20future%20of%20personalized%20health%20care&journal=Clin.%20Transl.%20Sci.&volume=14&pages=86-93&publication_year=2021&author=Johnson%2CKB) 
    
2.  Shickel, B., Tighe, P. J., Bihorac, A. & Rashidi, P. Deep EHR: a survey of recent advances in deep learning techniques for electronic health record (EHR) analysis. _IEEE J. Biomed. Health Inform._ **22**, 1589–1604 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29989977)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20EHR%3A%20a%20survey%20of%20recent%20advances%20in%20deep%20learning%20techniques%20for%20electronic%20health%20record%20%28EHR%29%20analysis&journal=IEEE%20J.%20Biomed.%20Health%20Inform.&volume=22&pages=1589-1604&publication_year=2018&author=Shickel%2CB&author=Tighe%2CPJ&author=Bihorac%2CA&author=Rashidi%2CP) 
    
3.  Tang, A. S. et al. Harnessing EHR data for health research. _Nat. Med._ **30**, 1847–1855 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsV2ntLvN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38965433)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Harnessing%20EHR%20data%20for%20health%20research&journal=Nat.%20Med.&volume=30&pages=1847-1855&publication_year=2024&author=Tang%2CAS) 
    
4.  Ayala Solares, J. R. et al. Deep learning for electronic health records: a comparative review of multiple deep neural architectures. _J. Biomed. Inform._ **101**, 103337 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31916973)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20learning%20for%20electronic%20health%20records%3A%20a%20comparative%20review%20of%20multiple%20deep%20neural%20architectures&journal=J.%20Biomed.%20Inform.&volume=101&publication_year=2020&author=Ayala%20Solares%2CJR) 
    
5.  Placido, D. et al. A deep learning algorithm to predict risk of pancreatic cancer from disease trajectories. _Nat. Med._ **29**, 1113–1122 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXps1Knt74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37156936)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10202814)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20deep%20learning%20algorithm%20to%20predict%20risk%20of%20pancreatic%20cancer%20from%20disease%20trajectories&journal=Nat.%20Med.&volume=29&pages=1113-1122&publication_year=2023&author=Placido%2CD) 
    
6.  Zhao, J. et al. Learning from longitudinal data in electronic health record and genetic data to improve cardiovascular event prediction. _Sci. Rep._ **9**, 717 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30679510)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6345960)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Learning%20from%20longitudinal%20data%20in%20electronic%20health%20record%20and%20genetic%20data%20to%20improve%20cardiovascular%20event%20prediction&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Zhao%2CJ) 
    
7.  Petrazzini, B. O. et al. Coronary risk estimation based on clinical data in electronic health records. _J. Am. Coll. Cardiol._ **79**, 1155–1166 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35331410)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8956801)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coronary%20risk%20estimation%20based%20on%20clinical%20data%20in%20electronic%20health%20records&journal=J.%20Am.%20Coll.%20Cardiol.&volume=79&pages=1155-1166&publication_year=2022&author=Petrazzini%2CBO) 
    
8.  Forrest, I. S. et al. Machine learning-based marker for coronary artery disease: derivation and validation in two longitudinal cohorts. _Lancet_ **401**, 215–225 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36563696)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Machine%20learning-based%20marker%20for%20coronary%20artery%20disease%3A%20derivation%20and%20validation%20in%20two%20longitudinal%20cohorts&journal=Lancet&volume=401&pages=215-225&publication_year=2023&author=Forrest%2CIS) 
    
9.  Xiang, R. et al. Recent advances in polygenic scores: translation, equitability, methods and FAIR tools. _Genome Med._ **16**, 33 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38373998)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10875792)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Recent%20advances%20in%20polygenic%20scores%3A%20translation%2C%20equitability%2C%20methods%20and%20FAIR%20tools&journal=Genome%20Med.&volume=16&publication_year=2024&author=Xiang%2CR) 
    
10.  Beesley, L. J. et al. The emerging landscape of health research based on biobanks linked to electronic health records: existing resources, statistical challenges, and potential opportunities. _Stat. Med._ **39**, 773–800 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31859414)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20emerging%20landscape%20of%20health%20research%20based%20on%20biobanks%20linked%20to%20electronic%20health%20records%3A%20existing%20resources%2C%20statistical%20challenges%2C%20and%20potential%20opportunities&journal=Stat.%20Med.&volume=39&pages=773-800&publication_year=2020&author=Beesley%2CLJ) 
    
11.  Botsis, T., Hartvigsen, G., Chen, F. & Weng, C. Secondary use of EHR: data quality issues and informatics opportunities. _Summit Transl. Bioinform._ **2010**, 1–5 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21347133)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041534)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Secondary%20use%20of%20EHR%3A%20data%20quality%20issues%20and%20informatics%20opportunities&journal=Summit%20Transl.%20Bioinform.&volume=2010&pages=1-5&publication_year=2010&author=Botsis%2CT&author=Hartvigsen%2CG&author=Chen%2CF&author=Weng%2CC) 
    
12.  Rajkomar, A. et al. Scalable and accurate deep learning with electronic health records. _NPJ Digit. Med._ **1**, 18 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31304302)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6550175)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20and%20accurate%20deep%20learning%20with%20electronic%20health%20records&journal=NPJ%20Digit.%20Med.&volume=1&publication_year=2018&author=Rajkomar%2CA) 
    
13.  Shi, X., Li, X. & Cai, T. Spherical regression under mismatch corruption with application to automated knowledge translation. _J. Am. Stat. Assoc._ **116**, 1953–1964 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpsl2js7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spherical%20regression%20under%20mismatch%20corruption%20with%20application%20to%20automated%20knowledge%20translation&journal=J.%20Am.%20Stat.%20Assoc.&volume=116&pages=1953-1964&publication_year=2021&author=Shi%2CX&author=Li%2CX&author=Cai%2CT) 
    
14.  Wornow, M. et al. The shaky foundations of large language models and foundation models for electronic health records. _NPJ Digit. Med._ **6**, 135 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37516790)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10387101)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20shaky%20foundations%20of%20large%20language%20models%20and%20foundation%20models%20for%20electronic%20health%20records&journal=NPJ%20Digit.%20Med.&volume=6&publication_year=2023&author=Wornow%2CM) 
    
15.  Xie, F. et al. Deep learning for temporal data representation in electronic health records: a systematic review of challenges and methodologies. _J. Biomed. Inform._ **126**, 103980 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34974189)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20learning%20for%20temporal%20data%20representation%20in%20electronic%20health%20records%3A%20a%20systematic%20review%20of%20challenges%20and%20methodologies&journal=J.%20Biomed.%20Inform.&volume=126&publication_year=2022&author=Xie%2CF) 
    
16.  Steinfeldt, J. et al. Medical history predicts phenome-wide disease onset and enables the rapid response to emerging health threats. _Nat. Commun._ **16**, 585 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhtFeitbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39794311)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11724087)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Medical%20history%20predicts%20phenome-wide%20disease%20onset%20and%20enables%20the%20rapid%20response%20to%20emerging%20health%20threats&journal=Nat.%20Commun.&volume=16&publication_year=2025&author=Steinfeldt%2CJ) 
    
17.  Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. _Nat. Genet._ **51**, 584–591 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGltLw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30926966)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities&journal=Nat.%20Genet.&volume=51&pages=584-591&publication_year=2019&author=Martin%2CAR) 
    
18.  Mars, N. et al. Genome-wide risk prediction of common diseases across ancestries in one million people. _Cell Genom._ **2**, 100118 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvFOjtbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35591975)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9010308)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20risk%20prediction%20of%20common%20diseases%20across%20ancestries%20in%20one%20million%20people&journal=Cell%20Genom.&volume=2&publication_year=2022&author=Mars%2CN) 
    
19.  Sabatello, M. et al. Return of polygenic risk scores in research: stakeholders’ views on the eMERGE-IV study. _HGG Adv._ **5**, 100281 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38414240)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10950748)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Return%20of%20polygenic%20risk%20scores%20in%20research%3A%20stakeholders%E2%80%99%20views%20on%20the%20eMERGE-IV%20study&journal=HGG%20Adv.&volume=5&publication_year=2024&author=Sabatello%2CM) 
    
20.  Leitsalu, L. et al. Lessons learned during the process of reporting individual genomic results to participants of a population-based biobank. _Eur. J. Hum. Genet._ **31**, 1048–1056 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1GnurbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36192438)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lessons%20learned%20during%20the%20process%20of%20reporting%20individual%20genomic%20results%20to%20participants%20of%20a%20population-based%20biobank&journal=Eur.%20J.%20Hum.%20Genet.&volume=31&pages=1048-1056&publication_year=2023&author=Leitsalu%2CL) 
    
21.  Ritchie, S. C. et al. Integrative analysis of the plasma proteome and polygenic risk of cardiometabolic diseases. _Nat. Metab._ **3**, 1476–1483 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xmtlagsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34750571)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8574944)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20of%20the%20plasma%20proteome%20and%20polygenic%20risk%20of%20cardiometabolic%20diseases&journal=Nat.%20Metab.&volume=3&pages=1476-1483&publication_year=2021&author=Ritchie%2CSC) 
    
22.  Møller, P. L. et al. Combining polygenic and proteomic risk scores with clinical risk factors to improve performance for diagnosing absence of coronary artery disease in patients with de novo chest pain. _Circ. Genom. Precis. Med._ **16**, 442–451 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37753640)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combining%20polygenic%20and%20proteomic%20risk%20scores%20with%20clinical%20risk%20factors%20to%20improve%20performance%20for%20diagnosing%20absence%20of%20coronary%20artery%20disease%20in%20patients%20with%20de%20novo%20chest%20pain&journal=Circ.%20Genom.%20Precis.%20Med.&volume=16&pages=442-451&publication_year=2023&author=M%C3%B8ller%2CPL) 
    
23.  Nightingale Health Biobank Collaborative Group Metabolomic and genomic prediction of common diseases in 700,217 participants in three national biobanks. _Nat. Commun._ **15**, 10092 (2024).
    
24.  Aguilar, O. T., Chang, C., Bismuth, E. & Rivas, M. A. Integrative machine learning approaches for predicting disease risk using multi-omics data from the UK Biobank. Preprint at _bioRxiv_ [https://doi.org/10.1101/2024.04.16.589819](https://doi.org/10.1101/2024.04.16.589819) (2024).
    
25.  Lambert, S. A., Abraham, G. & Inouye, M. Towards clinical utility of polygenic risk scores. _Hum. Mol. Genet._ **28**, R133–R142 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpvFOjtLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31363735)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Towards%20clinical%20utility%20of%20polygenic%20risk%20scores&journal=Hum.%20Mol.%20Genet.&volume=28&pages=R133-R142&publication_year=2019&author=Lambert%2CSA&author=Abraham%2CG&author=Inouye%2CM) 
    
26.  Mars, N. et al. Polygenic and clinical risk scores and their impact on age at onset and prediction of cardiometabolic diseases and common cancers. _Nat. Med._ **26**, 549–557 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXmvVCiu7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32273609)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20and%20clinical%20risk%20scores%20and%20their%20impact%20on%20age%20at%20onset%20and%20prediction%20of%20cardiometabolic%20diseases%20and%20common%20cancers&journal=Nat.%20Med.&volume=26&pages=549-557&publication_year=2020&author=Mars%2CN) 
    
27.  Mohsen, F., Al-Absi, H. R. H., Yousri, N. A., El Hajj, N. & Shah, Z. A scoping review of artificial intelligence-based methods for diabetes risk prediction. _NPJ Digit. Med._ **6**, 197 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37880301)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10600138)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20scoping%20review%20of%20artificial%20intelligence-based%20methods%20for%20diabetes%20risk%20prediction&journal=NPJ%20Digit.%20Med.&volume=6&publication_year=2023&author=Mohsen%2CF&author=Al-Absi%2CHRH&author=Yousri%2CNA&author=El%20Hajj%2CN&author=Shah%2CZ) 
    
28.  Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. _Nature_ **562**, 203–209 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2qsbvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30305743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&journal=Nature&volume=562&pages=203-209&publication_year=2018&author=Bycroft%2CC) 
    
29.  Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. _Nature_ **613**, 508–518 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslKmtrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36653562)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9849126)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FinnGen%20provides%20genetic%20insights%20from%20a%20well-phenotyped%20isolated%20population&journal=Nature&volume=613&pages=508-518&publication_year=2023&author=Kurki%2CMI) 
    
30.  Leitsalu, L. et al. Cohort profile: Estonian Biobank of the Estonian Genome Center, University of Tartu. _Int. J. Epidemiol._ **44**, 1137–1147 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24518929)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cohort%20profile%3A%20Estonian%20Biobank%20of%20the%20Estonian%20Genome%20Center%2C%20University%20of%20Tartu&journal=Int.%20J.%20Epidemiol.&volume=44&pages=1137-1147&publication_year=2015&author=Leitsalu%2CL) 
    
31.  Bastarache, L. et al. Phenotype risk scores identify patients with unrecognized Mendelian disease patterns. _Science_ **359**, 1233–1239 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkslCjtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29590070)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5959723)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phenotype%20risk%20scores%20identify%20patients%20with%20unrecognized%20Mendelian%20disease%20patterns&journal=Science&volume=359&pages=1233-1239&publication_year=2018&author=Bastarache%2CL) 
    
32.  Lebovitch, D. S., Johnson, J. S., Dueñas, H. R. & Huckins, L. M. Phenotype risk scores: moving beyond ‘cases’ and ‘controls’ to classify psychiatric disease in hospital-based biobanks. Preprint at _medRxiv_ [https://doi.org/10.1101/2021.01.25.21249615](https://doi.org/10.1101/2021.01.25.21249615) (2021).
    
33.  Wu, P. et al. Mapping ICD-10 and ICD-10-CM codes to phecodes: workflow development and initial evaluation. _JMIR Med. Inform._ **7**, e14325 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31553307)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6911227)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20ICD-10%20and%20ICD-10-CM%20codes%20to%20phecodes%3A%20workflow%20development%20and%20initial%20evaluation&journal=JMIR%20Med.%20Inform.&volume=7&publication_year=2019&author=Wu%2CP) 
    
34.  Jermy, B. et al. A unified framework for estimating country-specific cumulative incidence for 18 diseases stratified by polygenic risk. _Nat. Commun._ **15**, 5007 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtl2mtr7L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38866767)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11169548)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20unified%20framework%20for%20estimating%20country-specific%20cumulative%20incidence%20for%2018%20diseases%20stratified%20by%20polygenic%20risk&journal=Nat.%20Commun.&volume=15&publication_year=2024&author=Jermy%2CB) 
    
35.  Denny, J. C. et al. PheWAS: demonstrating the feasibility of a phenome-wide scan to discover gene-disease associations. _Bioinformatics_ **26**, 1205–1210 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlt1equ7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20335276)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2859132)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PheWAS%3A%20demonstrating%20the%20feasibility%20of%20a%20phenome-wide%20scan%20to%20discover%20gene-disease%20associations&journal=Bioinformatics&volume=26&pages=1205-1210&publication_year=2010&author=Denny%2CJC) 
    
36.  Cox, D. R. Regression models and life-tables. _J. R. Stat. Soc. B_ **34**, 187–202 (1972).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regression%20models%20and%20life-tables&journal=J.%20R.%20Stat.%20Soc.%20B&volume=34&pages=187-202&publication_year=1972&author=Cox%2CDR) 
    
37.  Charlson, M. E., Pompei, P., Ales, K. L. & MacKenzie, C. R. A new method of classifying prognostic comorbidity in longitudinal studies: development and validation. _J. Chronic Dis._ **40**, 373–383 (1987).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL2s7ms1GnsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3558716)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20new%20method%20of%20classifying%20prognostic%20comorbidity%20in%20longitudinal%20studies%3A%20development%20and%20validation&journal=J.%20Chronic%20Dis.&volume=40&pages=373-383&publication_year=1987&author=Charlson%2CME&author=Pompei%2CP&author=Ales%2CKL&author=MacKenzie%2CCR) 
    
38.  Deyo, R. A., Cherkin, D. C. & Ciol, M. A. Adapting a clinical comorbidity index for use with ICD-9-CM administrative databases. _J. Clin. Epidemiol._ **45**, 613–619 (1992).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK38zgtVeqsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1607900)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adapting%20a%20clinical%20comorbidity%20index%20for%20use%20with%20ICD-9-CM%20administrative%20databases&journal=J.%20Clin.%20Epidemiol.&volume=45&pages=613-619&publication_year=1992&author=Deyo%2CRA&author=Cherkin%2CDC&author=Ciol%2CMA) 
    
39.  Singh, J. A. & Gaffo, A. Gout epidemiology and comorbidities. _Semin. Arthritis Rheum._ **50**, S11–S16 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32620196)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gout%20epidemiology%20and%20comorbidities&journal=Semin.%20Arthritis%20Rheum.&volume=50&pages=S11-S16&publication_year=2020&author=Singh%2CJA&author=Gaffo%2CA) 
    
40.  Zhao, Y. et al. The brain structure, immunometabolic and genetic mechanisms underlying the association between lifestyle and depression. _Nat. Ment. Health_ **1**, 736–750 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20brain%20structure%2C%20immunometabolic%20and%20genetic%20mechanisms%20underlying%20the%20association%20between%20lifestyle%20and%20depression&journal=Nat.%20Ment.%20Health&volume=1&pages=736-750&publication_year=2023&author=Zhao%2CY) 
    
41.  Frediani, F. & Villani, V. Migraine and depression. _Neurol. Sci._ **28**, S161–S165 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17508165)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Migraine%20and%20depression&journal=Neurol.%20Sci.&volume=28&pages=S161-S165&publication_year=2007&author=Frediani%2CF&author=Villani%2CV) 
    
42.  Kline, A. et al. Multimodal machine learning in precision health: a scoping review. _NPJ Digit. Med._ **5**, 171 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36344814)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9640667)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multimodal%20machine%20learning%20in%20precision%20health%3A%20a%20scoping%20review&journal=NPJ%20Digit.%20Med.&volume=5&publication_year=2022&author=Kline%2CA) 
    
43.  Kiser, A. C. et al. Standard vocabularies to improve machine learning model transferability with electronic health record data: retrospective cohort study using health care–associated infection. _JMIR Med. Inform._ **10**, e39057 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36040784)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9472055)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Standard%20vocabularies%20to%20improve%20machine%20learning%20model%20transferability%20with%20electronic%20health%20record%20data%3A%20retrospective%20cohort%20study%20using%20health%20care%E2%80%93associated%20infection&journal=JMIR%20Med.%20Inform.&volume=10&publication_year=2022&author=Kiser%2CAC) 
    
44.  Fiscella, K. & Sanders, M. R. Racial and ethnic disparities in the quality of health care. _Annu. Rev. Public Health_ **37**, 375–394 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26789384)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Racial%20and%20ethnic%20disparities%20in%20the%20quality%20of%20health%20care&journal=Annu.%20Rev.%20Public%20Health&volume=37&pages=375-394&publication_year=2016&author=Fiscella%2CK&author=Sanders%2CMR) 
    
45.  Mahajan, S. et al. Trends in differences in health status and health care access and affordability by race and ethnicity in the United States, 1999–2018. _JAMA_ **326**, 637–648 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34402830)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371573)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trends%20in%20differences%20in%20health%20status%20and%20health%20care%20access%20and%20affordability%20by%20race%20and%20ethnicity%20in%20the%20United%20States%2C%201999%E2%80%932018&journal=JAMA&volume=326&pages=637-648&publication_year=2021&author=Mahajan%2CS) 
    
46.  Antunes, R. S., André da Costa, C., Küderle, A., Yari, I. A. & Eskofier, B. Federated learning for healthcare: systematic review and architecture proposal. _ACM Trans. Intell. Syst. Technol._ **13**, 54:1–54:23 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Federated%20learning%20for%20healthcare%3A%20systematic%20review%20and%20architecture%20proposal&journal=ACM%20Trans.%20Intell.%20Syst.%20Technol.&volume=13&pages=54%3A1-54%3A23&publication_year=2022&author=Antunes%2CRS&author=Andr%C3%A9%20da%20Costa%2CC&author=K%C3%BCderle%2CA&author=Yari%2CIA&author=Eskofier%2CB) 
    
47.  Ding, Y. et al. Large uncertainty in individual polygenic risk score estimation impacts PRS-based risk stratification. _Nat. Genet._ **54**, 30–39 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislynsrvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34931067)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large%20uncertainty%20in%20individual%20polygenic%20risk%20score%20estimation%20impacts%20PRS-based%20risk%20stratification&journal=Nat.%20Genet.&volume=54&pages=30-39&publication_year=2022&author=Ding%2CY) 
    
48.  Lewis, C. M. & Vassos, E. Polygenic risk scores from research tools to clinical instruments. _Genome Med._ **12**, 44 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32423490)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7236300)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20risk%20scores%20from%20research%20tools%20to%20clinical%20instruments&journal=Genome%20Med.&volume=12&publication_year=2020&author=Lewis%2CCM&author=Vassos%2CE) 
    
49.  Hippisley-Cox, J., Coupland, C. & Brindle, P. Development and validation of QRISK3 risk prediction algorithms to estimate future risk of cardiovascular disease: prospective cohort study. _BMJ_ **357**, j2099 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28536104)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5441081)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20and%20validation%20of%20QRISK3%20risk%20prediction%20algorithms%20to%20estimate%20future%20risk%20of%20cardiovascular%20disease%3A%20prospective%20cohort%20study&journal=BMJ&volume=357&publication_year=2017&author=Hippisley-Cox%2CJ&author=Coupland%2CC&author=Brindle%2CP) 
    
50.  Hippisley-Cox, J. & Coupland, C. Development and validation of QDiabetes-2018 risk prediction algorithm to estimate future risk of type 2 diabetes: cohort study. _BMJ_ **359**, j5019 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29158232)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5694979)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20and%20validation%20of%20QDiabetes-2018%20risk%20prediction%20algorithm%20to%20estimate%20future%20risk%20of%20type%202%20diabetes%3A%20cohort%20study&journal=BMJ&volume=359&publication_year=2017&author=Hippisley-Cox%2CJ&author=Coupland%2CC) 
    
51.  Zhang, Q., Privé, F., Vilhjálmsson, B. & Speed, D. Improved genetic prediction of complex traits from individual-level data or summary statistics. _Nat. Commun._ **12**, 4192 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFOnsrnK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234142)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8263809)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improved%20genetic%20prediction%20of%20complex%20traits%20from%20individual-level%20data%20or%20summary%20statistics&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Zhang%2CQ&author=Priv%C3%A9%2CF&author=Vilhj%C3%A1lmsson%2CB&author=Speed%2CD) 
    
52.  Pedregosa, F. et al. Scikit-learn: machine learning in Python. _J. Mach. Learn. Res._ **12**, 2825–2830 (2011).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scikit-learn%3A%20machine%20learning%20in%20Python&journal=J.%20Mach.%20Learn.%20Res.&volume=12&pages=2825-2830&publication_year=2011&author=Pedregosa%2CF) 
    
53.  Therneau, T. M. Survival analysis. R package version 3.8-3 [cran.r-project.org/web/packages/survival/index.html](https://cran.r-project.org/web/packages/survival/index.html) (2024).
    
54.  Harrell, F. E. Jr & Dupont, C. Hmisc: Harrell Miscellaneous. R package version 5.2-0 [hbiostat.org/r/hmisc/](https://hbiostat.org/r/hmisc/) (2024).
    
55.  Harrell, F. E., Lee, K. L. & Mark, D. B. Multivariable prognostic models: issues in developing models, evaluating assumptions and adequacy, and measuring and reducing errors. _Stat. Med._ **15**, 361–387 (1996).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8668867)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multivariable%20prognostic%20models%3A%20issues%20in%20developing%20models%2C%20evaluating%20assumptions%20and%20adequacy%2C%20and%20measuring%20and%20reducing%20errors&journal=Stat.%20Med.&volume=15&pages=361-387&publication_year=1996&author=Harrell%2CFE&author=Lee%2CKL&author=Mark%2CDB) 
    
56.  Pencina, M. J. & D’Agostino, R. B. Overall C as a measure of discrimination in survival analysis: model specific population value and confidence interval estimation. _Stat. Med._ **23**, 2109–2123 (2004).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15211606)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overall%20C%20as%20a%20measure%20of%20discrimination%20in%20survival%20analysis%3A%20model%20specific%20population%20value%20and%20confidence%20interval%20estimation&journal=Stat.%20Med.&volume=23&pages=2109-2123&publication_year=2004&author=Pencina%2CMJ&author=D%E2%80%99Agostino%2CRB) 
    
57.  Viechtbauer, W. metafor: meta-analysis package for R. R version 4.8-0 [https://cran.r-project.org/web/packages/metafor/index.html](https://cran.r-project.org/web/packages/metafor/index.html) (2024).
    
58.  Viechtbauer, W. Conducting meta-analyses in R with the metafor package. _J. Stat. Softw._ **36**, 1–48 (2010).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conducting%20meta-analyses%20in%20R%20with%20the%20metafor%20package&journal=J.%20Stat.%20Softw.&volume=36&pages=1-48&publication_year=2010&author=Viechtbauer%2CW) 
    
59.  Hartonen & Detrois, K. Intervene-EU-H2020/INTERVENE\_PheRS: V1.0.0. _Zenodo_ [https://doi.org/10.5281/zenodo.15691460](https://doi.org/10.5281/zenodo.15691460) (2025).
    
60.  Detrois, K. Intervene-EU-H2020/onset\_prediction: v1.0.0. _Zenodo_ [https://doi.org/10.5281/zenodo.15681882](https://doi.org/10.5281/zenodo.15681882) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02298-9?format=refman&flavour=references)

Acknowledgements
----------------

We would like to acknowledge the participants and investigators of the FinnGen, UK Biobank and the Estonian Biobank studies.

This study has received funding from the European Union’s Horizon 2020 research and innovation program (grant agreement 101016775). This Estonian Biobank study was funded by the European Union through the European Regional Development Fund (project 2014-2020.4.01.15-0012 GENTRANSMED). A.G. has received funding from the European Union’s Horizon 2020 research and innovation program (grant 101016775), the European Research Council under the same program (grant 945733) and the Academy of Finland (fellowship grant 323116).

We would like to acknowledge the participants and investigators of the FinnGen study listed in Supplementary Table [19](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). The FinnGen project is funded by two grants from Business Finland (HUS 4685/31/2016 and UH 4386/31/2016) and the following industry partners: AbbVie, AstraZeneca UK, Biogen MA, Bristol Myers Squibb (and Celgene Corporation & Celgene International II Sàrl), Genentech, Merck Sharp & Dohme, Pfizer, GlaxoSmithKline Intellectual Property Development, Sanofi US Services, Maze Therapeutics, Janssen Biotech, Novartis AG and Boehringer Ingelheim International GmbH. Following biobanks are acknowledged for delivering biobank samples to FinnGen: Auria Biobank ([www.auria.fi/biopankki](http://www.auria.fi/biopankki)), THL Biobank ([www.thl.fi/biobank](http://www.thl.fi/biobank)), Helsinki Biobank ([www.helsinginbiopankki.fi](http://www.helsinginbiopankki.fi/)), Biobank Borealis of Northern Finland ([https://www.ppshp.fi/Tutkimus-ja-opetus/Biopankki/Pages/Biobank-Borealis-briefly-in-English.aspx](https://www.ppshp.fi/Tutkimus-ja-opetus/Biopankki/Pages/Biobank-Borealis-briefly-in-English.aspx)), Finnish Clinical Biobank Tampere ([www.tays.fi/en-US/Research\_and\_development/Finnish\_Clinical\_Biobank\_Tampere](http://www.tays.fi/en-US/Research_and_development/Finnish_Clinical_Biobank_Tampere)), Biobank of Eastern Finland ([www.ita-suomenbiopankki.fi/en](http://www.ita-suomenbiopankki.fi/en)), Central Finland Biobank ([www.ksshp.fi/fi-FI/Potilaalle/Biopankki](http://www.ksshp.fi/fi-FI/Potilaalle/Biopankki)), Finnish Red Cross Blood Service Biobank ([www.veripalvelu.fi/verenluovutus/biopankkitoiminta](http://www.veripalvelu.fi/verenluovutus/biopankkitoiminta)), Terveystalo Biobank ([www.terveystalo.com/fi/Yritystietoa/Terveystalo-Biopankki/Biopankki/](http://www.terveystalo.com/fi/Yritystietoa/Terveystalo-Biopankki/Biopankki/)) and Arctic Biobank ([https://www.oulu.fi/en/university/faculties-and-units/faculty-medicine/northern-finland-birth-cohorts-and-arctic-biobank](https://www.oulu.fi/en/university/faculties-and-units/faculty-medicine/northern-finland-birth-cohorts-and-arctic-biobank)). All Finnish Biobanks are members of BBMRI.fi infrastructure ([https://www.bbmri-eric.eu/national-nodes/finland/](https://www.bbmri-eric.eu/national-nodes/finland/)). Finnish Biobank Cooperative—FINBB ([https://finbb.fi/](https://finbb.fi/)) is the coordinator of BBMRI-ERIC operations in Finland.

The authors would like to thank participants and scientists involved in making the UK Biobank resource available at [https://www.ukbiobank.ac.uk/](https://www.ukbiobank.ac.uk/). This research has been conducted using the UKB resource under approved application 78537.

The EstBB research team received funding from the Estonian Research Council (grant TT17 ‘Estonian Centre for Genomics’). Data analysis was carried out in part in the High-Performance Computing Center of the University of Tartu. K.L. and R.M. received funding from the Estonian Research Council grants PUT (PRG1911) and TK (TK214).

We would like to acknowledge CSC—IT Center for Science, Finland, for computational resources.

The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.

Funding
-------

Open Access funding provided by University of Helsinki (including Helsinki University Central Hospital).

Author information
------------------

Author notes

1.  These authors contributed equally: Kira E. Detrois, Tuomo Hartonen.
    

### Authors and Affiliations

1.  Institute for Molecular Medicine Finland, FIMM, HiLIFE, University of Helsinki, Helsinki, Finland
    
    Kira E. Detrois, Tuomo Hartonen, Bradley Jermy, Zhiyu Yang, Samuli Ripatti & Andrea Ganna
    
2.  Estonian Genome Centre, Institute of Genomics, University of Tartu, Tartu, Estonia
    
    Maris Teder-Laving, Kristi Läll & Reedik Mägi
    
3.  Broad Institute of MIT and Harvard, Cambridge, MA, USA
    
    Samuli Ripatti & Andrea Ganna
    
4.  Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA
    
    Samuli Ripatti & Andrea Ganna
    
5.  Department of Public Health, University of Helsinki, Helsinki, Finland
    
    Samuli Ripatti
    

Authors

1.  Kira E. Detrois
    
    [View author publications](https://www.nature.com/search?author=Kira%20E.%20Detrois)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kira%20E.%20Detrois) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kira%20E.%20Detrois%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Tuomo Hartonen
    
    [View author publications](https://www.nature.com/search?author=Tuomo%20Hartonen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tuomo%20Hartonen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tuomo%20Hartonen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Maris Teder-Laving
    
    [View author publications](https://www.nature.com/search?author=Maris%20Teder-Laving)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maris%20Teder-Laving) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maris%20Teder-Laving%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Bradley Jermy
    
    [View author publications](https://www.nature.com/search?author=Bradley%20Jermy)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Bradley%20Jermy) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Bradley%20Jermy%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Kristi Läll
    
    [View author publications](https://www.nature.com/search?author=Kristi%20L%C3%A4ll)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kristi%20L%C3%A4ll) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kristi%20L%C3%A4ll%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Zhiyu Yang
    
    [View author publications](https://www.nature.com/search?author=Zhiyu%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zhiyu%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zhiyu%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Reedik Mägi
    
    [View author publications](https://www.nature.com/search?author=Reedik%20M%C3%A4gi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Reedik%20M%C3%A4gi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Reedik%20M%C3%A4gi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Samuli Ripatti
    
    [View author publications](https://www.nature.com/search?author=Samuli%20Ripatti)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Samuli%20Ripatti) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Samuli%20Ripatti%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Andrea Ganna
    
    [View author publications](https://www.nature.com/search?author=Andrea%20Ganna)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Andrea%20Ganna) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Andrea%20Ganna%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Consortia

### Estonian Biobank research team

-   Reedik Mägi

### FinnGen

-   Samuli Ripatti
-    & Andrea Ganna

### Contributions

K.E.D., T.H. and A.G. wrote the manuscript with substantial input from all the other authors. T.H. and K.E.D. developed the PheRS code, and K.E.D. wrote the Cox model code. B.J. contributed the PGS. K.E.D. and T.H. conducted the analyses in FinnGen; T.H. and K.E.D. in the UK Biobank; and M.T.-L. and K.L. in the Estonian Biobank. K.E.D. combined the results and created the figures together with T.H. K.E.D., T.H. and Z.Y. preprocessed the FinnGen and UK Biobank data; M.T.-L. and K.L. preprocessed the Estonian Biobank data. T.H., B.J. and A.G. developed the original draft of the manuscript. A.G., S.R. and R.M. supervised the study.

### Corresponding author

Correspondence to [Andrea Ganna](mailto:andrea.ganna@helsinki.fi).

Ethics declarations
-------------------

### Competing interests

A.G. is the founder of Real World Genetics Oy. B.J. became an employee of BioMarin after his part of this work was completed. K.L. has participated as an analyst in a collaborative research project at the Institute of Genomics, University of Tartu, which was funded by Geneto OÜ. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Prediction accuracies of the PheRS.](https://www.nature.com/articles/s41588-025-02298-9/figures/6)

**a**, (1) Increase in prediction accuracy when adding the PheRS to a baseline model with age and sex in all three studies. The c-index point estimates and 95% CIs of the baseline model (x-axis), compared to a model with added PheRS (y-axis). (2) Increase in prediction accuracy when adding the PheRS to a baseline model with the additional predictor of number of unique phecodes/diagnoses. Diseases with a significant difference are labeled (one-tailed p-values < 0.05 based on the z-scores of the c-index increases) and those passing Bonferroni correction for multiple hypothesis testing (p-value < 0.05/13) are marked with \*. All p-values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). **b**, The c-index point estimates of the PheRS model and 95% CIs separately in each study (FinnGen: peach, UKB: brown, EstB: green and meta-analyzed results: red). The effect of age and sex was removed by regressing them out from the PheRS. **c**, Correlation between the PheRS and the total number of unique phecodes of each individual (FinnGen: peach, UKB: brown, EstB: green). Pearson’s r point estimates and 95% CI (x-axis) for each disease (y-axis). The p-values of the correlations are listed in Supplementary Table [10](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

### [Extended Data Fig. 2 Prediction improvements of CCI and education.](https://www.nature.com/articles/s41588-025-02298-9/figures/7)

Increase in prediction accuracy when adding: CCI (**a**), education (**b**), and both CCI and education (**c**), to a baseline model with age and sex in all three studies. All panels show the c-index point estimates and 95% CIs of the baseline model (x-axis), compared to a model with added predictors (y-axis). Diseases with significant increases in c index are labeled (p-values < 0.05; one-tailed p-values based on the z-scores of the c-index increases) and those passing Bonferroni correction for multiple hypothesis testing (p-values < 0.05/13) are marked with \*. All p-values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

### [Extended Data Fig. 3 PheRS vs. the unique number of phecodes.](https://www.nature.com/articles/s41588-025-02298-9/figures/8)

**a**, Association of the unique number of phecodes (y-axis) vs. PheRS (x-axis) with each disease. **b**, Association of the PheRS with each disease in a model without (x-axis) and with the unique number of phecodes (y-axis). Both panels show HR point estimates and 95% CIs. Diseases with significant differences are labeled (p-values < 0.05; two-tailed p-values based on the z-scores of the HR differences and one-tailed p-values for the c-index increases) and those passing Bonferroni correction for multiple hypothesis testing (p-values < 0.05/13) are marked with \*. All p-values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

### [Extended Data Fig. 4 Age-stratified results in FinnGen.](https://www.nature.com/articles/s41588-025-02298-9/figures/9)

HR point estimates and 95% CIs of the PheRS (**a**) and PGS (**b**) in younger (32-51) compared to older individuals (52–70 years). The PheRS were trained on all individuals aged 32–70 years. Diseases with significant differences are labeled (p-values < 0.05; two-tailed p-values based on the z-scores of the HR differences) and those passing Bonferroni correction for multiple hypothesis testing (p-values < 0.05/13) are marked with \*. All p-values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

### [Extended Data Fig. 5 Internally-trained vs. externally-trained PheRS models.](https://www.nature.com/articles/s41588-025-02298-9/figures/10)

The c-index point estimates and 95% CIs of the baseline models with age and sex and the models with added PheRS. The c-indices of the baseline models are shown by the dark gray boxes (the left-most marker for each disease). The results of the PheRS models trained internally in FinnGen (left and middle plot) and the UKB (right plot) are shown as boxes with the color of each disease and the externally-trained PheRS as stars. The left panel shows the results with the UKB-trained and the right panel the EstB-trained models. A single star (\*) denotes diseases where the c-index increase of the externally-trained PheRS is significantly higher than the baseline and two stars (\*\*) where this increase is not significantly different from the internally-trained PheRS. All p-values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

### [Extended Data Fig. 6 PheRS trained on all phecodes vs. only common phecodes.](https://www.nature.com/articles/s41588-025-02298-9/figures/11)

Association of PheRS trained on all phecodes in a study (x-axis) vs. PheRS trained only on phecodes common in all three studies (y-axis). Phecodes are considered common if they are recorded for at least 1% of all three study populations during the observation period. Both panels show HR point estimates and 95% CIs. Diseases with significant differences are labeled (p-values < 0.05; two-tailed p-values based on the z-scores of the HR differences and one-tailed p-values for the c-index increases) and those passing Bonferroni correction for multiple hypothesis testing (p-values < 0.05/13) are marked with \*. All p-values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). **a**, PheRS trained and tested in the same study. **b**, PheRS trained and tested in different studies. The first and second plots show the PheRS tested in FinnGen and trained in the UKB and EstB. The third plot shows the PheRS tested in the UKB and trained in the EstB.

### [Extended Data Fig. 7 Leave-one-out analyses in FinnGen.](https://www.nature.com/articles/s41588-025-02298-9/figures/12)

**a**, HR point estimates and 95% CIs for PheRS trained using ridge regression (y-axis) and elastic net (x-axis) in FinnGen and tested in FinnGen. All models also included age and sex as predictors. **b**, Leave-one-out analysis of the top 20 predictors of each PheRS model for the different diseases using ridge regression. Area under the receiver-operating curve (AUC) of each PheRS model, with the bars indicating the 95% CIs of the full model with all predictors using bootstrapping (N = 1,000).

### [Extended Data Fig. 8 PheRS coefficients.](https://www.nature.com/articles/s41588-025-02298-9/figures/13)

A detailed look at all the PheRS coefficients in the three studies for (**a**) gout, (**b**) asthma, (**c**) knee osteoarthritis, (**d**) type 2 diabetes, (**e**) coronary heart disease and (**f**) atrial fibrillation. Black color marks common phecodes in the PheRS models across the studies, while other colors indicate biobank-specific codes (peach = FinnGen, brown = UKB, green = EstB). PheRS coefficients are standardized to 0 mean and 1 standard deviation for each model separately for easier comparison of coefficient importances across the studies.

### [Extended Data Fig. 9 PheRS and PGS comparison.](https://www.nature.com/articles/s41588-025-02298-9/figures/14)

**a**, (**1**) Association of the PGS (x-axis) and PheRS (y-axis) scores with each disease. The meta-analyzed HR point estimates and 95% CIs for 1 s.d. increase on the scores after regressing out age, sex, and the first 10 PCs. (**2**) PGS HR point estimates and 95% CIs for 1 s.d. increase in a model without PheRS (x-axis) and one where we adjust for the effect of PheRS (y-axis). **b**, Correlation (Pearsons’ _r, y-axis_) point estimates and 95% CIs between the PheRS and PGS scores separately in each study (FinnGen: triangles, UKB: squares, EstB: circles), with the Nagelkerke’s pseudo-R² point estimates of the PGS (x-axis). **c**, C-index improvements when adding PGS to models with PheRS. C-index point estimates and 95% CIs of the models with only PheRS, sex, age, and PCs (x-axis) compared to those with added PGS (y-axis). Due to sample overlap with the GWASs, PGS could only be calculated for 4 diseases in UKB (see [Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10) for details). Diseases with significant differences are labeled (p-values < 0.05; two-tailed p-values based on the z-scores of the HR differences and one-tailed p-values for the c-index increases) and those passing Bonferroni correction for multiple hypothesis testing (p-values < 0.05/13) are marked with \*. All p-values are listed in Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3).

### [Extended Data Fig. 10 PGS associations.](https://www.nature.com/articles/s41588-025-02298-9/figures/15)

**a**, Association of PGS (x-axis) vs. PheRS (y-axis) with each disease in each study. HR point estimates and 95% CIs.The HRs are shown for 1 s.d. increase of the PheRS after regressing-out age, sex, and the first 10 PCs. Diseases with significant differences are labeled (p-values < 0.05; two-tailed p-values based on the z-scores of the HR differences and one-tailed p-values for the c-index increases) and those passing Bonferroni correction for multiple hypothesis testing (p-values < 0.05/13) are marked with \*. All p-values are listed in Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and [7](https://www.nature.com/articles/s41588-025-02298-9#MOESM3) and the number of cases and controls for each disease in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02298-9#MOESM3). **b**, Association between PGS and disease onset during the prediction period. The HR point estimates and 95% CIs in each study—FinnGen: peach, UKB: brown, EstB: green, and meta-analyzed result: red. The HRs are shown for an increase of the PGS by 1 s.d. after regressing-out age, sex, and the first ten PCs. Due to sample overlap with the GWASs, PGS could only be calculated for 4 diseases in UKB (see [Methods](https://www.nature.com/articles/s41588-025-02298-9#Sec10) for details).

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM1_ESM.pdf)

Supplementary methods and results.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM3_ESM.pdf)

### [Supplementary Tables 1–20](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM4_ESM.xlsx)

Supplementary Tables 1–20.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM5_ESM.xlsx)

Source data for Fig. 1c.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM6_ESM.xlsx)

Statistical source data for Fig. 2a,b.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM7_ESM.xlsx)

Statistical source data for Fig. 3a,b.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM8_ESM.xlsx)

Source data for Fig. 4b–d.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02298-9/MediaObjects/41588_2025_2298_MOESM9_ESM.xlsx)

Statistical source data for Fig. 5a–d.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Cross-biobank%20generalizability%20and%20accuracy%20of%20electronic%20health%20record-based%20predictors%20compared%20to%20polygenic%20scores&author=Kira%20E.%20Detrois%20et%20al&contentID=10.1038%2Fs41588-025-02298-9&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-27&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02298-9)

### Cite this article

Detrois, K.E., Hartonen, T., Teder-Laving, M. _et al._ Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02298-9

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02298-9?format=refman&flavour=citation)

-   Received: 16 September 2024
    
-   Accepted: 16 July 2025
    
-   Published: 27 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02298-9
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02298-9.pdf)
