> id: https://www.nature.com/articles/s41588-025-02224-z

> link: https://www.nature.com/articles/s41588-025-02224-z

> title: Decomposition of phenotypic heterogeneity in autism reveals underlying genetic programs

# Decomposition of phenotypic heterogeneity in autism reveals underlying genetic programs
_Published on Wed, 09 Jul 2025 00:00:00 GMT_

Abstract
--------

Unraveling the phenotypic and genetic complexity of autism is extremely challenging yet critical for understanding the biology, inheritance, trajectory and clinical manifestations of the many forms of the condition. Using a generative mixture modeling approach, we leverage broad phenotypic data from a large cohort with matched genetics to identify robust, clinically relevant classes of autism and their patterns of core, associated and co-occurring traits, which we further validate and replicate in an independent cohort. We demonstrate that phenotypic and clinical outcomes correspond to genetic and molecular programs of common, de novo and inherited variation and further characterize distinct pathways disrupted by the sets of mutations in each class. Remarkably, we discover that class-specific differences in the developmental timing of affected genes align with clinical outcome differences. These analyses demonstrate the phenotypic complexity of children with autism, identify genetic programs underlying their heterogeneity, and suggest specific biological dysregulation patterns and mechanistic hypotheses.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-022-01072-5/MediaObjects/41588_2022_1072_Fig1_HTML.png)

### [Genetic correlates of phenotypic heterogeneity in autism](https://www.nature.com/articles/s41588-022-01072-5?fromPaywallRec=false)

Article Open access 02 June 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41380-022-01938-4/MediaObjects/41380_2022_1938_Fig1_HTML.png)

### [Identifying rare genetic variants in 21 highly multiplex autism families: the role of diagnosis and autistic traits](https://www.nature.com/articles/s41380-022-01938-4?fromPaywallRec=false)

Article Open access 26 January 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-46128-8/MediaObjects/41467_2024_46128_Fig1_HTML.png)

### [Structural models of genome-wide covariance identify multiple common dimensions in autism](https://www.nature.com/articles/s41467-024-46128-8?fromPaywallRec=false)

Article Open access 27 February 2024

Main
----

At its core, autism spectrum disorder (ASD) is characterized by persistent deficits in social communication and interaction, alongside restricted and repetitive patterns of behavior, interests or activities[1](https://www.nature.com/articles/s41588-025-02224-z#ref-CR1 "Diagnostic and Statistical Manual of Mental Disorders 5th edn (American Psychiatric Association, 2013)."),[2](https://www.nature.com/articles/s41588-025-02224-z#ref-CR2 "Lord, C. et al. Autism spectrum disorder. Nat. Rev. Dis. Prim. 6, 5 (2020)."). Numbers of ASD diagnoses have been rising rapidly in recent years, and with the widening of diagnostic criteria, there is increasing heterogeneity within the autistic population, phenotypically and genetically[3](https://www.nature.com/articles/s41588-025-02224-z#ref-CR3 "Chiarotti, F. & Venerosi, A. Epidemiology of autism spectrum disorders: a review of worldwide prevalence estimates since 2014. Brain Sci. 10, 274 (2020)."). Autism displays a complex phenotypic structure: core features can vary substantially in severity and presentation and can coincide with extensive and unique spectra of associated phenotypes and co-occurring conditions for each individual[2](https://www.nature.com/articles/s41588-025-02224-z#ref-CR2 "Lord, C. et al. Autism spectrum disorder. Nat. Rev. Dis. Prim. 6, 5 (2020)."),[4](https://www.nature.com/articles/s41588-025-02224-z#ref-CR4 "Simonoff, E. et al. Psychiatric disorders in children with autism spectrum disorders: prevalence, comorbidity, and associated factors in a population-derived sample. J. Am. Acad. Child Adolesc. Psychiatry 47, 921–929 (2008)."). This wide array of phenotypes is matched by the broad genetic heterogeneity of individuals with autism. Despite substantial evidence for a genetic basis of the condition[5](https://www.nature.com/articles/s41588-025-02224-z#ref-CR5 "Sanders, S. J. et al. De novo mutations revealed by whole-exome sequencing are strongly associated with autism. Nature 485, 237–241 (2012)."),[6](https://www.nature.com/articles/s41588-025-02224-z#ref-CR6 "Sanders, S. J. et al. Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci. Neuron 87, 1215–1233 (2015)."),[7](https://www.nature.com/articles/s41588-025-02224-z#ref-CR7 "Sandin, S. et al. The heritability of autism spectrum disorder. JAMA 318, 1182–1184 (2017)."),[8](https://www.nature.com/articles/s41588-025-02224-z#ref-CR8 "Grove, J. et al. Identification of common genetic risk variants for autism spectrum disorder. Nat. Genet. 51, 431–444 (2019)."),[9](https://www.nature.com/articles/s41588-025-02224-z#ref-CR9 "Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)."),[10](https://www.nature.com/articles/s41588-025-02224-z#ref-CR10 "Fu, J. M. et al. Rare coding variation provides insight into the genetic architecture and phenotypic context of autism. Nat. Genet. 54, 1320–1331 (2022)."),[11](https://www.nature.com/articles/s41588-025-02224-z#ref-CR11 "Trost, B. et al. Genomic architecture of autism from comprehensive whole-genome sequence annotation. Cell 185, 4409–4427.e18 (2022)."),[12](https://www.nature.com/articles/s41588-025-02224-z#ref-CR12 "Warrier, V. et al. Genetic correlates of phenotypic heterogeneity in autism. Nat. Genet. 54, 1293–1304 (2022)."),[13](https://www.nature.com/articles/s41588-025-02224-z#ref-CR13 "Antaki, D. et al. A phenotypic spectrum of autism is attributable to the combined effects of rare variants, polygenic risk and sex. Nat. Genet. 54, 1284–1292 (2022)."),[14](https://www.nature.com/articles/s41588-025-02224-z#ref-CR14 "Hashem, S. et al. Genetics of structural and functional brain changes in autism spectrum disorder. Transl. Psychiatry 10, 229 (2020)."),[15](https://www.nature.com/articles/s41588-025-02224-z#ref-CR15 "Weiner, D. J. et al. Statistical and functional convergence of common and rare genetic influences on autism at chromosome 16p. Nat. Genet. 54, 1630–1639 (2022)."),[16](https://www.nature.com/articles/s41588-025-02224-z#ref-CR16 "Rolland, T. et al. Phenotypic effects of genetic variants associated with autism. Nat. Med. 29, 1671–1680 (2023)."),[17](https://www.nature.com/articles/s41588-025-02224-z#ref-CR17 "Courchesne, E. et al. The ASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry 24, 88–107 (2019)."),[18](https://www.nature.com/articles/s41588-025-02224-z#ref-CR18 "Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. Nature 542, 433–438 (2017).") and hundreds of ASD-associated genes having been identified[19](https://www.nature.com/articles/s41588-025-02224-z#ref-CR19 "Banerjee-Basu, S. & Packer, A. SFARI Gene: an evolving database for the autism research community. Dis. Model. Mech. 3, 133–135 (2010)."),[20](https://www.nature.com/articles/s41588-025-02224-z#ref-CR20 "Zhou, X. et al. Integrating de novo and inherited variants in 42,607 autism cases identifies mutations in new moderate-risk genes. Nat. Genet. 54, 1305–1319 (2022)."),[21](https://www.nature.com/articles/s41588-025-02224-z#ref-CR21 "Viggiano, M. et al. Genomic analysis of 116 autism families strengthens known risk genes and highlights promising candidates. NPJ Genom. Med. 9, 21 (2024)."),[22](https://www.nature.com/articles/s41588-025-02224-z#ref-CR22 "Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). Mol. Autism 4, 36 (2013)."), we lack a coherent mapping of genetic variation to phenotypes. Here, we leverage broad phenotypic data for a large autism cohort to parse both genetic and phenotypic heterogeneity and identify robust phenotypic classes of individuals and their underlying genetic programs.

Previous studies have demonstrated extensive heterogeneity in autism phenotypes, including cognitive or adaptive behavior[23](https://www.nature.com/articles/s41588-025-02224-z#ref-CR23 "Feczko, E. et al. Subtyping cognitive profiles in autism spectrum disorder using a functional random forest algorithm. Neuroimage 172, 674–688 (2018)."),[24](https://www.nature.com/articles/s41588-025-02224-z#ref-CR24 "Mandelli, V. et al. Prognostic early snapshot stratification of autism based on adaptive functioning. Nat. Ment. Health 1, 327–336 (2023)."), morphology[25](https://www.nature.com/articles/s41588-025-02224-z#ref-CR25 "Chan, A. J. S. et al. Genome-wide rare variant score associates with morphological subtypes of autism spectrum disorder. Nat. Commun. 13, 6463 (2022)."), neuroanatomical imaging profiles[26](https://www.nature.com/articles/s41588-025-02224-z#ref-CR26 "Buch, A. M. et al. Molecular and network-level mechanisms explaining individual differences in autism spectrum disorder. Nat. Neurosci. 26, 650–663 (2023)."),[27](https://www.nature.com/articles/s41588-025-02224-z#ref-CR27 "Ellegood, J. et al. Clustering autism: using neuroanatomical differences in 26 mouse models to gain insight into the heterogeneity. Mol. Psychiatry 20, 118–125 (2015).") and clinical outcomes[28](https://www.nature.com/articles/s41588-025-02224-z#ref-CR28 "Luo, Y. et al. A multidimensional precision medicine approach identifies an autism subtype characterized by dyslipidemia. Nat. Med. 26, 1375–1379 (2020)."). However, linking this heterogeneity to genetic factors has been challenging and limited to trait-centric approaches instead of considering the phenotypes of individuals holistically (that is, the combination of traits of each individual)[12](https://www.nature.com/articles/s41588-025-02224-z#ref-CR12 "Warrier, V. et al. Genetic correlates of phenotypic heterogeneity in autism. Nat. Genet. 54, 1293–1304 (2022)."),[13](https://www.nature.com/articles/s41588-025-02224-z#ref-CR13 "Antaki, D. et al. A phenotypic spectrum of autism is attributable to the combined effects of rare variants, polygenic risk and sex. Nat. Genet. 54, 1284–1292 (2022)."),[16](https://www.nature.com/articles/s41588-025-02224-z#ref-CR16 "Rolland, T. et al. Phenotypic effects of genetic variants associated with autism. Nat. Med. 29, 1671–1680 (2023)."). Trait-centric approaches marginalize co-occurring phenotypes when focusing on one trait. However, as traits are not independent, they cannot be separately associated with patterns of genetic variation. During development, traits affect each other in complex ways, compensating for or exacerbating individual phenotype measures. A person-centered approach can capture the sum of these developmental processes at later ages, offering strong clinical value for prognosis with individualized genotype–phenotype relationships. This type of approach has shown promise in applications to other complex psychiatric conditions[29](https://www.nature.com/articles/s41588-025-02224-z#ref-CR29 "Cai, N. et al. Minimal phenotyping yields genome-wide association signals of low specificity for major depression. Nat. Genet. 52, 437–447 (2020)."),[30](https://www.nature.com/articles/s41588-025-02224-z#ref-CR30 "LaBianca, S. et al. Polygenic profiles define aspects of clinical heterogeneity in attention deficit hyperactivity disorder. Nat. Genet. 56, 234–244 (2024).").

This person-centered, quantitative phenotypic analysis enables us to address the longstanding challenge of deconvolving the complexity of genetic signals in autism. We leverage a unique cohort with both broad phenotypic and genotypic data at scale (_n_ = 5,392) to parse heterogeneity consistent with clinically meaningful presentations of autism[31](https://www.nature.com/articles/s41588-025-02224-z#ref-CR31 "Lombardo, M. V., Lai, M.-C. & Baron-Cohen, S. Big data approaches to decomposing heterogeneity across the autism spectrum. Mol. Psychiatry 24, 1435–1450 (2019)."),[32](https://www.nature.com/articles/s41588-025-02224-z#ref-CR32 "Agelink van Rentergem, J. A., Deserno, M. K. & Geurts, H. M. Validation strategies for subtypes in psychiatry: a systematic review of research on autism spectrum disorder. Clin. Psychol. Rev. 87, 102033 (2021)."). Using a generative mixture modeling framework, we decompose phenotypic information to identify, validate and replicate four latent classes, allowing us to associate each of them with different genetic programs. We conduct a thorough investigation of genetic influences in the context of the phenotypic heterogeneity defining our classes, showing that patterns in common genetic variation measured by polygenic scores coincide with their phenotypic and diagnostic traits. We then analyze de novo and rare inherited variation and identify diverging genetic profiles across gene sets and pathways. Finally, we demonstrate that rare variation is associated with class-specific gene expression patterns during development that align with clinical milestones and individual presentations of autism.

Results
-------

### Identifying the structures of autism phenotypes

To best reflect the complexity of presentations across autistic individuals, we identified 239 item-level and composite phenotype features present in 5,392 individuals from the SPARK cohort[33](https://www.nature.com/articles/s41588-025-02224-z#ref-CR33 "SPARK Consortium. SPARK: a US cohort of 50,000 families to accelerate autism research. Neuron 97, 488–493 (2018)."), a nationwide effort to collect and track genetic and clinical presentations of autism. Briefly, these features represent responses on standard diagnostic questionnaires (the Social Communication Questionnaire-Lifetime (SCQ)[34](https://www.nature.com/articles/s41588-025-02224-z#ref-CR34 "Rutter, M., Bailey, A. & Lord, C. The social Communication Questionnaire: Manual (Western Psychological Services, 2003)."), Repetitive Behavior Scale-Revised (RBS-R)[35](https://www.nature.com/articles/s41588-025-02224-z#ref-CR35 "Lam, K. S. L. & Aman, M. G. The Repetitive Behavior Scale-Revised: independent validation in individuals with autism spectrum disorders. J. Autism Dev. Disord. 37, 855–866 (2007)."), Child Behavior Checklist 6–18 (CBCL)[36](https://www.nature.com/articles/s41588-025-02224-z#ref-CR36 "Offermans, J. E. et al. The development and validation of a subscale for the school-age Child Behavior CheckList to screen for autism spectrum disorder. J. Autism Dev. Disord. 53, 1034–1052 (2023).")) and a background history form focused on developmental milestones. These data were analyzed with a general finite mixture model (GFMM) to minimize statistical assumptions while accommodating heterogeneous (continuous, binary and categorical) data types ([Methods](https://www.nature.com/articles/s41588-025-02224-z#Sec9)). The model captures the underlying distributions in the data and provides an inherently person-centered approach, separating individuals into classes rather than fragmenting each individual into separate phenotypic categories (Fig. [1a](https://www.nature.com/articles/s41588-025-02224-z#Fig1)).

**Fig. 1: Overview of study design and description of identified subclasses.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02224-z/figures/1)

**a**, Study design for parsing the phenotypic heterogeneity of autism and deciphering the genetic factors contributing to individual presentations. A GFMM was trained on a matrix of probands (_n_ = 5,392 individuals) by phenotype (239 features describing item-level and composite phenotypic measure data). We describe four data-driven classes of autism that exhibit differing phenotypic presentations and trait patterns. These four subclasses were further characterized by external validations and genetic analyses. **b**, To demonstrate differences in phenotypic patterns, we assessed the propensity of each class toward seven phenotype categories. Values close to 1 indicate that the majority of phenotypes within the category were significantly and positively enriched for the phenotype domain compared to probands in other classes (indicating higher difficulties), and values close to −1 indicate significant negative enrichment or depletion for a given phenotype domain compared to probands in other classes (indicating lower difficulties). Sample sizes for all analyses shown were as follows: Broadly affected, _n_ = 554 (magenta); Social/behavioral, _n_ = 1,976 (green); Mixed ASD with DD, _n_ = 1,002 (blue); Moderate challenges, _n_ = 1,860 (orange); unaffected siblings, _n_ = 1,972. **c**, Distributions of two key developmental milestones: the age when the individual first walked and the age when they first used words (both in months) across the four classes, with nonautistic siblings as a control (_n_ = 1,972). **d**, Individual total scores from the SCQ by class, with nonautistic siblings as a control (_n_ = 1,972). Center lines in all box plots represent the median, box limits represent the 25th and 75th percentiles, whiskers extend to show 1.5× the interquartile range, and outliers are shown separately as open circles. One-sided independent _t_\-tests with adjustment for multiple comparisons (Benjamini–Hochberg correction) were used to determine the significance of enrichment for each class compared to siblings, as well as for all possible class–class comparisons (\*FDR < 0.1, \*\*FDR < 0.05, \*\*\*FDR < 0.01; NS, not significant). Stars directly above the box plot indicate comparisons to siblings, whereas stars accompanied by horizontal bars indicate class–class comparisons. Schematic in **a** created with [BioRender.com](http://biorender.com/).

We selected a GFMM with four latent classes representing four different patterns of phenotype profile by considering six standard model fit statistical measures and the overall interpretability of the model solutions. After training models with two to ten latent classes, we found that four classes presented the best balance of model fit as measured by the Bayesian information criterion (BIC), validation log likelihood and other statistical measures of fit (Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02224-z#Fig6) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). In addition, a four-class solution offered the best interpretability in terms of phenotypic separation (Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02224-z#Fig7)), as evaluated by clinical collaborators with extensive experience working with autistic individuals. We also found the four-class model to be highly stable and robust to various perturbations (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02224-z#Fig8)).

As observed clinically, classes differed not only in severity of autism symptoms but also in the degree to which co-occurring cognitive, behavioral and psychiatric concerns factored into their presentation. For clinical interpretability, we assigned each of the 239 phenotype features to one of the following seven categories defined in the literature[35](https://www.nature.com/articles/s41588-025-02224-z#ref-CR35 "Lam, K. S. L. & Aman, M. G. The Repetitive Behavior Scale-Revised: independent validation in individuals with autism spectrum disorders. J. Autism Dev. Disord. 37, 855–866 (2007)."),[37](https://www.nature.com/articles/s41588-025-02224-z#ref-CR37 "Berument, S. K., Rutter, M., Lord, C., Pickles, A. & Bailey, A. Autism screening questionnaire: diagnostic validity. Br. J. Psychiatry 175, 444–451 (1999)."),[38](https://www.nature.com/articles/s41588-025-02224-z#ref-CR38 "Grove, R. et al. Factor structure of the Social Communication Questionnaire in preschool aged autistic children. J. Child Fam. Stud. 28, 3385–3391 (2019)."),[39](https://www.nature.com/articles/s41588-025-02224-z#ref-CR39 "Achenbach, T. M. & Edelbrock, C. S. The Child Behavior Profile: II. Boys aged 12–16 and girls aged 6–11 and 12–16. J. Consult. Clin. Psychol. 47, 223–233 (1979)."): limited social communication, restricted and/or repetitive behavior, attention deficit, disruptive behavior, anxiety and/or mood symptoms, developmental delay (DD) and self-injury (Fig. [1b](https://www.nature.com/articles/s41588-025-02224-z#Fig1)). We identified one class that demonstrated high scores (greater difficulties) across core autism categories of social communication and restricted and/or repetitive behaviors compared to other autistic children, as well as disruptive behavior, attention deficit and anxiety, but no reports of developmental delays; this class was named Social/behavioral (_n_ = 1,976). A second class, Mixed ASD with DD (_n_ = 1,002), showed a more nuanced presentation, with some features enriched and some depleted among the restricted and/or repetitive behavior, social communication and self-injury categories and overall strong enrichment of developmental delays compared to both nonautistic siblings and individuals in other classes (false discovery rate (FDR) < 0.01; 0.19 < Cohen’s _d_ <0.46; Fig. [1c](https://www.nature.com/articles/s41588-025-02224-z#Fig1), Extended Data Fig. [4a](https://www.nature.com/articles/s41588-025-02224-z#Fig9) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). Individuals in the last two classes scored consistently lower (fewer difficulties) and consistently higher than other autistic children across all seven categories. These two classes were termed Moderate challenges (_n_ = 1,860) and Broadly affected (_n_ = 554). Although individuals in the Moderate challenges class scored below other autistic children across these measured categories, those in all classes still scored significantly higher than nonautistic siblings on the SCQ, the only diagnostic questionnaire with sibling responses, supporting their ASD diagnoses (Fig. [1d](https://www.nature.com/articles/s41588-025-02224-z#Fig1)). Furthermore, classes displayed significant differences across measures (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)) and significantly greater between-class variability than within-class variability (Extended Data Fig. [4b](https://www.nature.com/articles/s41588-025-02224-z#Fig9)), further supporting their phenotypic separation. Additional characteristics of the classes, including sex and age distributions, can be seen in Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02224-z#Fig10).

### Clinical attributes and replication of phenotype classes

The characteristics of the four phenotypic classes we identified were consistent with data on diagnoses of co-occurring conditions and parent reports that were external to our modeling and class identification. A medical history questionnaire, with reports on diagnoses of conditions such as attention-deficit hyperactivity disorder (ADHD), obsessive–compulsive disorder, language delays, depression and anxiety, was not included in the GFMM, but we found that enrichment patterns of these diagnoses matched the class-specific phenotypic profiles and further distinguished the classes phenotypically (Fig. [2a](https://www.nature.com/articles/s41588-025-02224-z#Fig2) and Supplementary Table [3](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). The Broadly affected class displayed significant enrichment in almost all measured co-occurring conditions, with the Social/behavioral class matching or exceeding the same diagnostic levels for ADHD, anxiety and major depression (Social/behavioral FDR < 0.01, 1.65 < fold enrichment (FE) < 2.36 compared to out-of-class probands; Fig. [2a](https://www.nature.com/articles/s41588-025-02224-z#Fig2)), reflecting enrichments in phenotypic profiles (Fig. [1b](https://www.nature.com/articles/s41588-025-02224-z#Fig1)). The Mixed ASD with DD class was highly enriched in language delay, intellectual disability and motor disorders, compared to both siblings (FDR < 0.01, 8.8 < FE < 20.0) and probands in other classes (FDR < 0.01, 1.38 < FE < 2.33), consistent with the high scores of this class in the categories of developmental delay and restricted and/or repetitive behavior, and individuals in this class showed significantly lower levels of ADHD, anxiety and depression, as expected based on their phenotypic profile. The two classes with greater developmental delays, Mixed ASD with DD and Broadly affected, also showed significantly higher reported levels of cognitive impairment (FDR < 0.01, 1.74 < FE < 3.14), lower levels of language ability (FDR < 0.01, 0.51 < FE < 0.78) and much earlier ages at diagnosis (FDR < 0.01, 0.22 < Cohen’s _d_ < 0.98) than the two classes without substantial developmental delays (Fig. [2b](https://www.nature.com/articles/s41588-025-02224-z#Fig2), Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02224-z#Fig10) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). In addition, average numbers of interventions (such as medication, counseling, physical therapy or other forms of therapy) were highest among the Broadly affected and Social/behavioral classes (Fig. [2b](https://www.nature.com/articles/s41588-025-02224-z#Fig2)). These diagnostic data represented the best available external validation, although the natural associations between behavioral diagnoses and the behavioral questionnaires on which our model was trained meant that this was not a fully orthogonal validation set. However, the consistency observed here further supported the validity of the self-reported data. Together, these analyses of medical features show that the four classes were phenotypically consistent, supporting their separation in genetic analyses.

**Fig. 2: Clinical validation and replication of subclass characteristics.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02224-z/figures/2)

**a**, Clinical validation of classes with external medical diagnoses. We computed the FE (_x_ axis) and statistical significance (FDR) for a selection of available diagnoses for each class. Open circles indicate FDR > 0.05, whereas closed circles indicate FDR < 0.05. Statistical comparisons were computed against siblings as background using one-sided binomial tests and adjusted for multiple comparisons (Benjamini–Hochberg correction). The dotted line indicates FE = 1. Sample sizes for all analyses shown were as follows: Broadly affected, _n_ = 554 (magenta); Social/behavioral, _n_ = 1,970 (green); Mixed ASD with DD, _n_ = 1,001 (blue); Moderate challenges, _n_ = 1,856 (orange); unaffected siblings (_n_ = 1,599). **b**, External validation of classes with additional parent-reported data from background history and medical history questionnaires. Displayed are: language level at enrollment, parent report with four levels reflecting language abilities (0, nonverbal; 1, single words; 2, phrases; 3, sentences), total number of interventions for probands (including options such as medication, social skills groups, speech therapy and counseling), cognitive impairment at enrollment, a binary indicator of a diagnosis of intellectual disability or cognitive impairment, and age at diagnosis in months. Box plots (center lines represent the median, box limits represent the 25th and 75th percentiles, whiskers represent 1.5× the interquartile range, open circles represent outliers) were plotted for continuous variables (one-sided independent _t_\-tests), whereas bar plots displaying means and 95% confidence intervals were plotted for binary variables (one-sided binomial tests) and categorical variables (one-sided independent _t_\-tests; distributions shown in Extended Data Fig. [5d](https://www.nature.com/articles/s41588-025-02224-z#Fig10)). \*FDR < 0.1, \*\*FDR < 0.05, \*\*\*FDR < 0.01. **c**, Replication of phenotype classes in the SSC. An independent model was trained on the SPARK dataset (_n_ = 6,393) for features matching across the two cohorts and applied to the SSC dataset for all individuals with complete data across features (_n_ = 861). Bars display Pearson correlation coefficients (_x_ axis) between SPARK and SSC category enrichment proportions across four classes.

Furthermore, the four phenotype classes were replicated well in an independent autism cohort that was deeply phenotyped by trained clinicians, the Simons Simplex Collection (SSC)[40](https://www.nature.com/articles/s41588-025-02224-z#ref-CR40 "Fischbach, G. D. & Lord, C. The Simons Simplex Collection: a resource for identification of autism genetic risk factors. Neuron 68, 192–195 (2010)."). Most phenotypic questionnaires used in the SPARK model were available for SSC, with the exception of item-level CBCL data. We combined these matched data, resulting in 108 training features present for both cohorts. To demonstrate the generalizability of our model to the SSC cohort (_n_ = 861), we applied a GFMM trained on SPARK data to the SSC test set, as well as independently training a GFMM on the SSC data. We computed the enrichment and depletion of each feature within each class across the seven phenotype categories for both cohorts, as described above for the original SPARK model. We demonstrated strong replication of the autism classes in the SSC cohort, with highly similar feature enrichment patterns across all seven categories (Fig. [2c](https://www.nature.com/articles/s41588-025-02224-z#Fig2) and Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02224-z#Fig11)). We further assessed the significance of the overall model similarity with several permutation tests (Extended Data Fig. [6b,c](https://www.nature.com/articles/s41588-025-02224-z#Fig11)), shuffling both the SSC class labels and the SPARK phenotypes before training, and never observed a higher correlation value with permuted data than the true correlation of 0.927 (_P_ < 1 × 10−4). The phenotypic classes defined here are therefore concordant with clinical data and can be replicated in an external cohort.

### Dissimilar genetic signals underlie phenotypic heterogeneity

We expected that the differences in phenotypes, co-occurring diagnoses and developmental milestones across the four autism classes would correspond to class-specific patterns in genetic signals for common variants. We computed polygenic scores (PGS) of children with European ancestry for autism and five other well-powered genome-wide association studies (GWAS) of related traits and conditions (ancestry principal component analysis in Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02224-z#Fig12); GWAS references in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)); these showed significant differences across the four classes that qualitatively matched their clinical and phenotypic characteristics (Fig. [3a](https://www.nature.com/articles/s41588-025-02224-z#Fig3)). Several PGS signals within the classes matched their diagnostic burdens, with the Broadly affected and Social/behavioral classes showing significantly higher ADHD signals relative to both nonautistic siblings and other classes (FDR < 0.01, Cohen’s _d_ > 0.22 compared to siblings; FDR < 0.06, Cohen’s _d_ > 0.13 for significant cross-class comparisons; Supplementary Table [6](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)) and significant enrichment of ADHD diagnoses (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). The Social/behavioral class also showed both the highest average PGS signal and highest diagnostic burden for major depressive disorder (FDR = 0.00327, Cohen’s _d_ = 0.204; Figs. [2a](https://www.nature.com/articles/s41588-025-02224-z#Fig2) and [3a](https://www.nature.com/articles/s41588-025-02224-z#Fig3)). In addition, the Broadly affected class, which was most enriched for intellectual disability, cognitive impairment and developmental delays, exhibited significantly lower educational attainment and IQ PGS compared with siblings and other classes (FDR < 0.1, Cohen’s _d_ > 0.17), demonstrating that co-occurring conditions were associated with common genetic variation that significantly differed among the four identified classes. Notably, none of the classes had a statistically significant signal for the autism PGS, owing to the high variance of this score across our cohort and their siblings.

**Fig. 3: Genetic analyses of genome-wide common and rare variant signals.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02224-z/figures/3)

**a**, PGS for ASD GWAS and related phenotypes and conditions. PGS were normalized by the mean of sibling scores within each condition. Sample sizes were as follows: Moderate challenges, _n_ = 822; Broadly affected, _n_ = 225, Social/behavioral, _n_ = 425; Mixed ASD with DD, _n_ = 822; unaffected siblings, _n_ = 476. **b**, Count per offspring of high-impact DNVs (left) and high-impact rare inherited variants (right) across all protein-coding genes. High-impact variants were defined as variants predicted to be either high-confidence LoF or likely pathogenic missense. Sample sizes for **b** and **c** were as follows: Moderate challenges, _n_ = 809; Broadly affected, _n_ = 145; Social/behavioral, _n_ = 640; Mixed ASD with DD, _n_ = 419; unaffected siblings, _n_ = 1,013. **c**, Analysis of evolutionarily constrained genes across autism classes and nonautistic siblings. Using the gene-centric measure of evolutionary constraint, pLI, we assigned genes with pLI ≥ 0.5 to one of two categories: pLI ≥ 0.995 (higher constraint genes) or 0.5 ≤ pLI < 0.995 (lower constraint genes). Count burdens (dnLoF) per offspring were then computed for each class. In all parts, circles indicate the mean and error bars show the 95% confidence intervals. Statistical significance was computed with one-sided independent _t_\-tests and adjusted for multiple comparisons using Benjamini–Hochberg correction.\*FDR < 0.1, \*\*FDR < 0.05, \*\*\*FDR < 0.01. Stars above the 95% confidence intervals indicate comparisons to siblings, whereas stars accompanied by horizontal bars indicate direct class comparisons. All statistically significant comparisons are shown.

In addition to dissimilar patterns of common variants, we observed significant differences in rare genetic variation between the phenotypic classes (Fig. [3b](https://www.nature.com/articles/s41588-025-02224-z#Fig3) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). We conducted de novo and inherited variant calling on the whole exomes of individuals from the cohort using HAT[41](https://www.nature.com/articles/s41588-025-02224-z#ref-CR41 "Ng, J. K. & Turner, T. N. HAT: de novo variant calling for highly accurate short-read and long-read sequencing data. Bioinformatics 40, btad775 (2024).") and further classified variants as either loss-of-function (LoF), missense or synonymous. Count burden enrichments were then computed for each variant type in the four classes and among nonautistic siblings ([Methods](https://www.nature.com/articles/s41588-025-02224-z#Sec9)). The Broadly affected class displayed the greatest enrichment for high-confidence de novo LoF (dnLoF) and de novo missense (dnMis) variants compared to both nonautistic siblings and other classes (FDR = 0.01, FE = 1.66 compared to siblings; 0.044 < FDR < 0.086, 1.28 < FE < 1.43 for all class comparisons), whereas the Social/behavioral class displayed the lowest enrichment compared to siblings (FDR = 0.086, FE = 1.17), although we found significant burden in all four classes (Fig. [3b](https://www.nature.com/articles/s41588-025-02224-z#Fig3), left; FDR: 0.01, 0.01, 0.086, 0.04). Rare inherited LoF and missense variants displayed statistically significant increases only in the Mixed ASD with DD class (FDR = 0.016, FE = 2.55 compared to siblings), with a significantly higher count per proband in this class compared to the Moderate challenges and Social/behavioral classes (FDR < 0.016, FE > 1.05; Fig. [3b](https://www.nature.com/articles/s41588-025-02224-z#Fig3), right; FDR: 0.57, 0.29, 0.97, 0.016). Our analyses differentiated the two classes with greater intellectual disability and developmental delays, showing that the Broadly affected class had more high-impact de novo variants (DNVs), whereas the Mixed ASD with DD class had a combination of high-impact de novo and rare inherited variants compared to nonautistic controls, suggesting a stronger inherited component for the children in this class.

We also observed differences across categories of LoF constraint, including potential significance that had been masked by grouping heterogeneous classes of probands together. Using the gene-level measure of probability of LoF intolerance (pLI)[42](https://www.nature.com/articles/s41588-025-02224-z#ref-CR42 "Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)."), we assigned genes into one of two categories: high-constraint genes (pLI ≥ 0.995) and intermediate-constraint genes (0.5 ≤ pLI < 0.995). When we examined the burden of dnLoF variants across the classes in high-constraint genes, we observed a pattern consistent with our findings above: the Broadly affected class displayed the greatest burden counts in high-constraint genes compared to both siblings and other classes (FDR < 0.1 for two of three classes, 1.76 < FE < 2.3), although there was a significant increase relative to siblings among all autistic classes (Fig. [3c](https://www.nature.com/articles/s41588-025-02224-z#Fig3), left; FDR: 6 × 10−4, 0.013, 0.013, 0.013; odds ratio (OR): 3.69, 6.31, 2.87, 3.25). This finding was consistent with prior work showing an excess burden of mutations in high-constraint genes among probands[9](https://www.nature.com/articles/s41588-025-02224-z#ref-CR9 "Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)."),[43](https://www.nature.com/articles/s41588-025-02224-z#ref-CR43 "Zhou, J. et al. Whole-genome deep-learning analysis identifies contribution of noncoding mutations to autism risk. Nat. Genet. 51, 973–980 (2019)."). However, previous work identified no significant increase in intermediate-constraint genes in probands. By separating heterogeneous classes, we found significant enrichment of dnLoF variation in genes of intermediate constraint in the Moderate challenges class (Fig. [3c](https://www.nature.com/articles/s41588-025-02224-z#Fig3), right, Supplementary Table [7](https://www.nature.com/articles/s41588-025-02224-z#MOESM4); FDR: 0.09, 0.26, 0.26, 0.26, OR: 2.54, 2.84, 1.75, 1.95), suggesting that perhaps less-essential genes are affected in this class of individuals. The genetic differences observed between and across classes support the importance of separation of individuals to identify the genetic architectures underlying clinical phenotypic presentations.

### Unique gene sets and pathways associate with phenotypes

We obtained further insight into specific genes and processes dysregulated by the variants across the phenotype classes by investigating the count burdens of de novo and inherited variants in ASD-relevant gene sets[9](https://www.nature.com/articles/s41588-025-02224-z#ref-CR9 "Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)."),[19](https://www.nature.com/articles/s41588-025-02224-z#ref-CR19 "Banerjee-Basu, S. & Packer, A. SFARI Gene: an evolving database for the autism research community. Dis. Model. Mech. 3, 133–135 (2010)."),[44](https://www.nature.com/articles/s41588-025-02224-z#ref-CR44 "Werling, D. M. et al. An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder. Nat. Genet. 50, 727–736 (2018)."),[45](https://www.nature.com/articles/s41588-025-02224-z#ref-CR45 "Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. Cell 146, 247–261 (2011)."),[46](https://www.nature.com/articles/s41588-025-02224-z#ref-CR46 "GTEx Consortium et al. Genetic effects on gene expression across human tissues. Nature 550, 204–213 (2017).") (Supplementary [Data](https://www.nature.com/articles/s41588-025-02224-z#MOESM5)). Although all four classes displayed enrichment of de novo variation among ASD-related gene sets compared to nonautistic siblings, there were clear differences in the levels and patterns of enrichment among classes (Fig. [4a](https://www.nature.com/articles/s41588-025-02224-z#Fig4), Extended Data Fig. [8a,b](https://www.nature.com/articles/s41588-025-02224-z#Fig13) and Supplementary Tables [8](https://www.nature.com/articles/s41588-025-02224-z#MOESM4) and [9](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). For example, all autism-specific gene sets had significantly higher dnLoF mutation burdens (compared to siblings) in the classes with greater developmental delays (average FE: 17.2, 10.9) than in the classes with lower developmental delays (average FE: 5.8, 5.0), suggesting that cognitive outcomes are associated with rare high-impact mutations in a small subset of relevant genes (Fig. [4a](https://www.nature.com/articles/s41588-025-02224-z#Fig4)). OR analysis further subdivided the classes, with ORs for dnLoF variants being greatest in the classes with greater developmental delays (Fig. [4b](https://www.nature.com/articles/s41588-025-02224-z#Fig4), left; FDR: 0.09, 0.002, 0.009, 0.001; OR: 6.3, 28.7, 11.2, 19.7), whereas the ORs for de novo synonymous variants were uniformly distributed, with no significant increases compared with siblings (Fig. [4b](https://www.nature.com/articles/s41588-025-02224-z#Fig4), left; FDR: 0.85, 0.85, 0.85, 1.0; OR: 0.7, 1.6, 1.4, 1.1; other gene sets are shown in Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02224-z#Fig13)). Furthermore, DNVs in fragile X mental retardation protein (FMRP) target genes were strongly associated with the Broadly affected class, with significant enrichment over probands in other classes (FDR = 0.04, FE = 2.2, OR = 12.8). Individuals with fragile X syndrome exhibit developmental delays and intellectual disabilities, as seen across both the Broadly affected and Mixed ASD with DD classes, but they also tend to display mood disorders such as anxiety and impulsive, hyperactive and aggressive behaviors[47](https://www.nature.com/articles/s41588-025-02224-z#ref-CR47 "Garber, K. B., Visootsak, J. & Warren, S. T. Fragile X syndrome. Eur. J. Hum. Genet. 16, 666–672 (2008)."), which we observed only in the Broadly affected class.

**Fig. 4: Functional gene set analyses reveal differing genetic profiles.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02224-z/figures/4)

**a**, Enrichment and significance of dnLoF burden in each class and gene set. We retrieved seven relevant gene sets and computed the aggregated dnLoF burden for each individual in every gene set. FE (bubble size) and FDR significance (open versus closed circles) were computed relative to nonautistic siblings using one-sided independent _t_\-tests followed by adjustment for multiple comparisons (Benjamini–Hochberg correction). An FDR cutoff of 0.05 was used to determine significance. **b**, ORs (_y_ axis) across classes for dnLoF variation in ASD risk genes (left) and FMRP target genes (right). ORs for de novo synonymous (dnSyn) variation are displayed for ASD risk genes. Statistical significance was computed with Fisher’s exact tests comparing each class to nonautistic siblings and adjusted for multiple comparisons using Benjamini–Hochberg correction. For ASD risk genes, FDR: 0.09, 0.002, 0.009, 0.001 for dnLoF; FDR: 0.85, 0.85, 0.85, 1.0 for dnSyn; for FMRP genes, FDR: 0.004, 0.0004, 0.0005, 0.002 for dnLoF. \*FDR < 0.1, \*\*FDR < 0.05; \*\*\*FDR < 0.01. **c**, Top significantly affected gene ontology (GO) biological processes and molecular functions are reported for each class against a genome-wide background of protein-coding genes. Gene sets for GO enrichment analyses include all protein-coding genes affected by high-confidence dnLoF or pathogenic missense variation present in individuals from each class. The plots display FE (_x_ axis) and log-transformed FDR (bubble size, hypergeometric test with multiple hypothesis correction). Terms were selected by FDR and sorted by FE. For the Moderate challenges, Social/behavioral and Mixed ASD with DD classes, an FDR cutoff of 0.05 was used, whereas a cutoff of 0.1 was used for the Broadly affected class. Shaded boxes represent GO biological processes, and unshaded boxes represent GO molecular functions. Sample sizes in all analyses shown were as follows: Moderate challenges, _n_ = 809; Broadly affected, _n_ = 145; Social/behavioral, _n_ = 640; Mixed ASD with DD, _n_ = 419; siblings, _n_ = 1,013.

Molecular pathways affected by the patterns of genetic variation observed across phenotypic classes also suggested different underlying biological mechanisms. Analysis of biological processes affected by high-confidence dnLoF or damaging missense variations in each class revealed little overlap in the top enriched biological processes and no overlap in top molecular functions between all four classes, suggesting that affected genes represent pathways uniquely associated with class-specific phenotypes (Fig. [4c](https://www.nature.com/articles/s41588-025-02224-z#Fig4) and Supplementary Table [10](https://www.nature.com/articles/s41588-025-02224-z#MOESM4)). In particular, compared to a gene-based background, the Social/behavioral class was highly enriched for processes of chromatin organization (FE = 3.5, FDR = 1.9 × 10−3), regulation of DNA repair (FE = 5.3, FDR = 1.0 × 10−2) and microtubule activity (FE = 34.2, FDR = 1.4 × 10−4). The Moderate challenges class displayed strong enrichment for histone modification (FE = 3.56, FDR = 1.1 × 10−4) and chromatin organization (FE = 3.5, FDR = 2.0 × 10−5). By contrast, the Mixed ASD with DD class was characterized by processes of neuronal action potential (FE = 19.0, FDR = 1.0 × 10−2) and membrane depolarization (FE = 25.0, FDR = 1.5 × 10−3), negative regulation of protein depolymerization (FE = 13.7, FDR = 3.6 × 10−3) and voltage-gated sodium channel activity (FE = 28.8, FDR = 3.5 × 10−3). Overall, our analysis directly suggests hypotheses for specific biological dysregulations underlying each autism class, providing a framework for directed examination of mechanistic insights in continuing autism research.

### Class-specific developmental gene expression patterns

We found that genes affected by variants in each ASD phenotypic class were associated with unique patterns of gene expression trajectories throughout brain development. This analysis leveraged cell-type-specific developmental gene expression trajectories of the human prefrontal cortex (Fig. [5a](https://www.nature.com/articles/s41588-025-02224-z#Fig5) and [Methods](https://www.nature.com/articles/s41588-025-02224-z#Sec9))[48](https://www.nature.com/articles/s41588-025-02224-z#ref-CR48 "Herring, C. A. et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. Cell 185, 4428–4447.e28 (2022)."). We found that the Mixed ASD with DD class was enriched for dnLoF variants that affected genes expressed in all major prefrontal cortex cell types, mostly during the fetal and neonatal stages, with declining expression later in development (these patterns were termed ‘trans down’ and ‘down’ by Herring et al.[48](https://www.nature.com/articles/s41588-025-02224-z#ref-CR48 "Herring, C. A. et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. Cell 185, 4428–4447.e28 (2022).")) (Fig. [5b](https://www.nature.com/articles/s41588-025-02224-z#Fig5) and Extended Data Fig. [9](https://www.nature.com/articles/s41588-025-02224-z#Fig14)). By contrast, the Social/behavioral class was enriched only for LoF variants in genes highly expressed postnatally compared to nonautistic siblings (termed ‘trans up’ and ‘up’ genes; Fig. [5b](https://www.nature.com/articles/s41588-025-02224-z#Fig5)) in inhibitory interneurons of the medial ganglionic eminence. Furthermore, the Mixed ASD with DD class was significantly enriched for high-impact variation in genes with the ‘down’ trend compared to the Social/behavioral class in principal excitatory neurons and inhibitory interneurons (FE > 2, FDR < 0.05; Supplementary Table [11](https://www.nature.com/articles/s41588-025-02224-z#MOESM1)); conversely, the Social/behavioral class was enriched for variation in genes with the ‘up’ trend compared to the Mixed ASD with DD class (FDR = 0.001, FE = 2.84 for principal excitatory neurons). These developmental gene expression patterns were aligned with the developmental clinical milestones of the classes: the Mixed ASD with DD class had the latest average age of developmental milestone attainment (FDR < 1.9 × 10−19, Cohen’s _d_ > 0.38 compared to the Social/behavioral class; Fig. [1c](https://www.nature.com/articles/s41588-025-02224-z#Fig1) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02224-z#MOESM1)) and the earliest average age of diagnosis (FDR = 6.97 × 10−150, Cohen’s _d_ = 0.99 compared to the Social/behavioral class; Fig. [2b](https://www.nature.com/articles/s41588-025-02224-z#Fig2) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02224-z#MOESM1)). By contrast, the Social/behavioral class, which had variants in later-expressed genes, showed less impact on early development, with later ages at diagnosis and developmental milestones almost in line with those of nonautistic siblings (Cohen’s _d_ < 0.07 compared to siblings). The Moderate challenges class displayed enrichment in mostly prenatal gene sets (‘down’ and ‘trans down’ genes), although the genes affected in this class tended to be of lower evolutionary constraint than genes affected in the Mixed ASD with DD class, which may have contributed to the differences in outcomes (Moderate challenges, median pLI = 0.75; Mixed ASD with DD, median pLI = 0.95; _P_ = 0.026). Finally, the Broadly affected class displayed significance for all trends, indicating broad dysregulation across developmental stages and cell types. Our findings thus demonstrate that there are class-specific differences in the developmental timing of genes that are dysregulated, and that these correspond to differences in clinical milestones and outcomes between the classes.

**Fig. 5: Cell-type-specific and developmental-stage-specific analysis of variant impacts.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02224-z/figures/5)

**a**, Trends from Herring et al.[48](https://www.nature.com/articles/s41588-025-02224-z#ref-CR48 "Herring, C. A. et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. Cell 185, 4428–4447.e28 (2022).") representing the gene expression trajectories of brain development genes differentially expressed across developmental stages. Gene expression trajectories follow one of four general patterns: ‘up’ (first), ‘trans up’ (second), ‘trans down’ (third), ‘down’ (fourth). Trends are measured across the six stages of development (_x_ axis): fetal, neonatal, infancy, childhood, adolescence and adulthood. **b**, Patterns of dnLoF variant enrichment across classes (_x_ axis), major cell types of the prefrontal cortex (_y_ axis) and gene expression trends (_y_ axis). For each class, we computed the FE (bubble size) and corrected _P_ values (FDR, one-sided independent _t_\-tests adjusted for multiple comparisons using Benjamini–Hochberg correction) of variant burden compared to that of nonautistic siblings. Open circles indicate FDR > 0.05 (not significant), and closed circles indicate significant enrichment (FDR ≤ 0.05). Each column is colored by the corresponding phenotypic class color, with purple representing the combined pool of all probands (_n_ = 2,013). Sample sizes in the analysis were as follows: Moderate challenges, _n_ = 809; Broadly affected, _n_ = 145; Social/behavioral, _n_ = 640; Mixed ASD with DD, _n_ = 419; siblings _n_ = 1,013. Statistics for all class–class comparisons for this analysis can be found in Supplementary Table [11](https://www.nature.com/articles/s41588-025-02224-z#MOESM1). Cell type and trend combinations with no significant enrichment in any class are not shown. MGE, medial ganglionic eminence; CGE, caudal ganglionic eminence.

Discussion
----------

Unraveling the complexity of autism is a particularly challenging yet critical task for supporting the needs of autistic individuals and understanding the biology, inheritance, trajectory and phenotypes of the many forms of the condition. Here, a person-centered approach powered the classification of more than 5,000 autistic children based on a broad set of phenotypes that interact and co-occur, revealing diverging genetic and developmental signals tied to clinical presentations.

The classes we have defined, validated and replicated reflect a robust quantitative analysis of broad phenotypic information in a large sample. The reliance on self-reported data, however, may have introduced rater effects, and intellectual disability is a known confounder in assessing social behaviors. Despite these challenges, we have shown that phenotypic presentation does not reflect a spectrum of intellectual disability, and the self-reported data were both internally consistent with medical history and reproducible in an external cohort with clinician-reported data. The clear separation of genetic signals for each class establishes a concrete set of hypotheses that can be tested with larger cohorts and more comprehensive phenotyping. It will therefore be crucial to expand both the cohort size and the quality and breadth of the phenotyping to more completely capture the full diversity of the autistic population. Coupled with genetic data, future studies incorporating digital phenotypes and longitudinal data will be better equipped to power significant associations between phenotype classes and genotypes, beyond single gene models. Additional genetic information from whole-genome sequencing offers broad potential for discovery of regulatory mechanisms contributing to phenotypic outcomes.

Although all classes had some enrichment for common and rare variants, we found differing signals that defined each autism class. The Social/behavioral class demonstrated the highest signal for common variants associated with co-occurring ADHD and depression, with enrichment for high-impact DNVs in neuronal genes predominantly expressed postnatally. By contrast, the Mixed ASD with DD class was highly enriched for high-impact rare variants, both de novo and inherited, particularly in neuronal genes primarily expressed in utero. The Moderate challenges class was significantly associated with rare variants in genes of lower evolutionary constraint. Finally, the Broadly affected class had the greatest enrichment of high-impact DNVs, specifically in highly constrained genes and FMRP target genes. Some of these trends were consistent with known associations between intellectual disability and genetic pathways, although these classes, which were constructed based on a much richer set of features, provide a more nuanced understanding of autistic individuals and the range of challenges they may face, along with differences in complex genetic architectures that may underlie their condition. By leveraging a person-centered approach, we reveal that these robust, phenotypically separable classes display characteristic patterns of genetic variation. Unlike previous approaches, our analyses directly associate genetic signals with sets of co-occurring phenotypes and further implicate specific affected pathways, brain cell types and developmental stages.

Our findings point to new directions that biologists and neuroscientists could pursue to gain insights into the underlying neurobiology of particular autism presentations and offer the potential for more precise clinical diagnosis and guidance. Future research could also examine how interventions may differ among the classes. In sum, we have demonstrated that person-centered quantitative phenotypic analysis, combined with matched genetic sequencing data at scale, is crucial for uncovering phenotypic classes and the corresponding genetic factors that contribute to the broad heterogeneity of autism.

Methods
-------

### Ethics

We received approval to access and analyze deidentified genetic and phenotypic data from the two cohorts from SFARI Base and the Princeton University IRB Committee in the Office of Research Integrity. Our research complies with all relevant ethical regulations.

### Participants

We restricted our sample to children aged 4 to 18 years from the SPARK cohort[33](https://www.nature.com/articles/s41588-025-02224-z#ref-CR33 "SPARK Consortium. SPARK: a US cohort of 50,000 families to accelerate autism research. Neuron 97, 488–493 (2018)."), including both autistic individuals and their nonautistic siblings. The SPARK Phenotype Dataset V9 battery assays core autistic traits (social, language-related and repetitive behaviors), cognitive measures and co-occurring behavioral features (anxiety, aggression and so on). Measures in SPARK with a sufficiently high overlap among participants were selected to maximize the cohort size while maintaining breadth of phenotype data. These measures included the background history (a form that asks parents to report on ages at developmental milestones), SCQ[34](https://www.nature.com/articles/s41588-025-02224-z#ref-CR34 "Rutter, M., Bailey, A. & Lord, C. The social Communication Questionnaire: Manual (Western Psychological Services, 2003)."), RBS-R[35](https://www.nature.com/articles/s41588-025-02224-z#ref-CR35 "Lam, K. S. L. & Aman, M. G. The Repetitive Behavior Scale-Revised: independent validation in individuals with autism spectrum disorders. J. Autism Dev. Disord. 37, 855–866 (2007).") and CBCL[36](https://www.nature.com/articles/s41588-025-02224-z#ref-CR36 "Offermans, J. E. et al. The development and validation of a subscale for the school-age Child Behavior CheckList to screen for autism spectrum disorder. J. Autism Dev. Disord. 53, 1034–1052 (2023).") (Supplementary [Note](https://www.nature.com/articles/s41588-025-02224-z#MOESM1)). All autistic children with data from these four phenotype measurement tools were included in the mixture modeling analysis. The same sample of individuals was measured repeatedly. To maximize our sample and feature sizes, we excluded features with less than 90% completeness across the cohort and only included individuals with complete measures across the remaining features. Our resulting cohort included 239 item-level and composite phenotype features and 5,392 individuals in SPARK alone. The cohort had a mean age of 8.56 years with a standard deviation of 3.15 years. Our sibling cohort comprised 1,972 paired siblings (related to at least one proband in the autistic cohort) with no ASD diagnosis, age less than 18 years and available whole-exome sequence data. Of the four phenotypic measures, only the background history and SCQ were available for SPARK siblings. For trio-based analyses, we only used complete trios in SPARK (_n_ = 1,992 probands, _n_ = 837 paired siblings). In both our phenotypic and genetic analyses, only individuals who passed quality control and had complete data were included.

### Mixture modeling analyses

#### Mixture model and covariates

The phenotype matrix included 5,392 individuals and 241 features (239 training features, two covariates). We used the StepMix package (v.1.2.5)[49](https://www.nature.com/articles/s41588-025-02224-z#ref-CR49 "Morin, S. et al. StepMix: a Python package for pseudo-likelihood estimation of generalized mixture models with external variables. Preprint at 
https://doi.org/10.48550/arXiv.2304.03853
(2023).") to fit a model and predict labels for each individual using a GFMM approach. This model expands the Gaussian mixture model by using different probability density functions (Gaussian, binomial or multinomial) to model continuous, binary and categorical features. We trained a one-step model with covariates (sex and age at evaluation) and hyperparameters n\_init = 200, n\_components = 4. The model computed four probabilities per individual to assign each individual to a latent class.

#### Stability and robustness analyses

To demonstrate robustness and stability, we trained 2,000 models with random initializations. We ranked models by their log likelihoods and retained the top 100 models for further analysis. To demonstrate robustness to perturbations of the sample set, we trained 100 independent models (each with 20 random initializations) with a subsampling rate of 50% of the individuals in the sample. In each analysis, we computed the proportion of individuals assigned to the same class, as well as the feature concordance across categories of phenotypes.

#### Exploratory class enumeration

The main tunable hyperparameter of the GFMM is the number of latent classes. It is recommended that this number is chosen by identifying an ‘elbow’ in statistical measures of model fit, in combination with careful consideration of practical interpretability[50](https://www.nature.com/articles/s41588-025-02224-z#ref-CR50 "Masyn, K. E. in The Oxford Handbook of Quantitative Methods, Vol. 2 (ed Little, T. D.) Ch. 25 (Oxford Univ. Press, 2013)."),[51](https://www.nature.com/articles/s41588-025-02224-z#ref-CR51 "Nylund-Gibson, K. & Choi, A. Y. Ten frequently asked questions about latent class analysis. Transl. Issues Psychol. Sci. 4, 440–461 (2018)."),[52](https://www.nature.com/articles/s41588-025-02224-z#ref-CR52 "Sinha, P., Calfee, C. S. & Delucchi, K. L. Practitioner’s guide to latent class analysis: methodological considerations and common pitfalls. Crit. Care Med. 49, e63–e79 (2021)."). We tuned this hyperparameter in a multistep process that considered various statistical measures of model fit and indicators of optimality. We evaluated models with 1–12 components and computed the validation log likelihood, Akaike information criterion, consistent Akaike information criterion, BIC, sample-size-adjusted BIC, likelihood ratio test statistics (Lo–Mendell–Rubin likelihood ratio test) and more, over 50–200 independent runs with randomly generated seeds (Supplementary [Note](https://www.nature.com/articles/s41588-025-02224-z#MOESM1)).

#### Interpretation

We consulted clinical collaborators on the interpretability of multiple candidate models and found that the four-class solution offered the best phenotypic separation and most clinically relevant classes. Less complex models offered insufficient phenotypic reduction, whereas more complex models overfitted our data, resulting in fragmented classes that were challenging to interpret owing to mixed enrichments within feature categories. Phenotypic descriptions for three-, five- and six-class model solutions can be found in Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02224-z#Fig7). The final four-class breakdown was as follows: Moderate challenges, _n_ = 1,860 (34%); Broadly affected, _n_ = 554 (10%); Social/behavioral, _n_ = 1,976 (37%); Mixed ASD with DD, _n_ = 1,002 (19%). Sex breakdown and percentages by class were as follows: Moderate challenges, 1,459 males (78%), 401 females (22%); Broadly affected, 441 males (80%), 113 females (20%); Social/behavioral, 1,475 males (75%), 501 females (25%); Mixed ASD with DD, 799 males (80%), 203 females (20%).

As outlined above, there was not one clear indicator for this parameter choice. We evaluated a rigorous combination of statistical measures, combined with interpretability, to select our final model: a four-component model that provided the best balance of model fit, complexity and interpretation.

#### Phenotype categories

To phenotypically define and interpret our model outputs, we grouped the 239 features used in training the GFMM into seven factors previously defined in the literature[35](https://www.nature.com/articles/s41588-025-02224-z#ref-CR35 "Lam, K. S. L. & Aman, M. G. The Repetitive Behavior Scale-Revised: independent validation in individuals with autism spectrum disorders. J. Autism Dev. Disord. 37, 855–866 (2007)."),[37](https://www.nature.com/articles/s41588-025-02224-z#ref-CR37 "Berument, S. K., Rutter, M., Lord, C., Pickles, A. & Bailey, A. Autism screening questionnaire: diagnostic validity. Br. J. Psychiatry 175, 444–451 (1999)."),[38](https://www.nature.com/articles/s41588-025-02224-z#ref-CR38 "Grove, R. et al. Factor structure of the Social Communication Questionnaire in preschool aged autistic children. J. Child Fam. Stud. 28, 3385–3391 (2019)."),[39](https://www.nature.com/articles/s41588-025-02224-z#ref-CR39 "Achenbach, T. M. & Edelbrock, C. S. The Child Behavior Profile: II. Boys aged 12–16 and girls aged 6–11 and 12–16. J. Consult. Clin. Psychol. 47, 223–233 (1979)."). Our categories included: limited social communication, restricted and/or repetitive behavior, attention deficit, disruptive behavior, anxiety and/or mood symptoms, developmental delay and self-injury. Each category was a composite of features assigned based on prior grouping evidence.

To more thoroughly examine how the factors defined the four phenotype classes, we performed the following analysis: first, we computed the enrichment (upper-tail significance) or depletion (lower-tail significance) of each feature in each class relative to the other three classes. For binary variables, a one-sided binomial test was used. For categorical and continuous variables, a one-sided independent _t_\-test was used. Benjamini–Hochberg multiple hypothesis correction was performed for each autism class and direction of enrichment. We then computed the proportion of features with upper-tail and lower-tail significance in each category and class. At this stage, features were excluded based on three criteria: (1) features that had no significant enrichment or depletion in any class; (2) continuous or categorical features with Cohen’s _d_ values between −0.2 and 0.2 across all classes; and (3) binary features with FE of less than 1.5 across all classes. This resulted in 220 contributory features. After computing the proportion of features enriched or depleted in each class and category, we negated the depleted proportions and summed them with the positive enriched proportions. This enabled us to compute a total proportion and direction of significant features for each category and class. This score represented the ‘affinity’ of the class towards a phenotype category relative to other probands.

#### Phenotypic replication in the SSC

We replicated the four SPARK phenotype classes in an independent cohort, the SSC[40](https://www.nature.com/articles/s41588-025-02224-z#ref-CR40 "Fischbach, G. D. & Lord, C. The Simons Simplex Collection: a resource for identification of autism genetic risk factors. Neuron 68, 192–195 (2010)."). We first extracted, processed and integrated the same phenotype measures from SSC that were used in the SPARK model: SCQ, RBS-R, CBCL 6–18 (only composite scores) and a form consisting of parent-reported timing of developmental milestone attainment. We identified 108 phenotype features that were present in both SPARK and SSC. To demonstrate the generalizability of the SPARK model to SSC, we trained a model on only the SPARK data for the 108 common features with the same hyperparameters as the model described above (_n_ = 6,393 individuals in SPARK with complete data for the 108 features). We compiled the same features for _n_ = 861 individuals in SSC who had complete data across the training features. We then applied the trained model to the SSC test data and predicted a class label for each individual from SSC. At this point, we excluded noncontributory features _post hoc_ based on only the training data and according to the criteria described above. We obtained the proportion and direction of enrichment for each class and category across both SPARK and SSC. The Pearson’s correlation coefficients between SPARK and SSC for each of the seven phenotype categories were 0.98, 0.92, 0.94, 0.92, 0.89, 0.97 and 0.98.

Next, we obtained a significance of overall model similarity with two permutation tests. First, we randomly shuffled the class labels of SSC participants for _n_ = 10,000 repetitions, obtaining a distribution of chance correlations between SPARK and SSC. We then compared the true correlation between SPARK and SSC (unshuffled labels) to the chance correlation distribution, allowing us to obtain a _P_ value for overall model similarity that accounted for chance correlations. Second, we shuffled the SPARK sample data 1,000 times, each time training an independent model on the shuffled phenotypes, and projected those permuted clusters onto the SSC to test the concordance of the permuted cluster projection. We then compared the distribution of correlations with permuted cluster projections to the correlation with the real cluster projection.

#### External phenotypic validation

Phenotype measures that were available for the majority of our cohort but were not included in model training were used for external validation of the classes and exploratory phenotypic associations of clinical and diagnostic variables. The majority of individuals in our sample cohort (_n_ = 5,381) had a complete or nearly complete basic medical screening form and self-reported (or parent-reported) registration form available. We computed the FE and _P_ values (one-sided binomial test) for each basic medical screening diagnosis within each class compared to nonautistic siblings. For the registration form, we tested for enrichment using one-sided independent _t_\-tests for continuous and categorical variables and one-sided binomial tests for binary variables. We also tested all class–class comparisons for every feature. We performed Benjamini–Hochberg correction to correct for multiple comparisons.

### Genetic analyses

#### Polygenic scores

The most recent well-powered GWAS for ASD and several ASD-related traits and co-occurring conditions were used to compute PGS for the subset of our cohort with genotyping array data and European ancestry (determined by a self-reported race of white) passing some basic quality-control metrics (_n_ = 2,294 autistic individuals, _n_ = 426 nonautistic siblings). Genotypes were determined from Infinium Global Screen Array BeadChips available from SPARK[33](https://www.nature.com/articles/s41588-025-02224-z#ref-CR33 "SPARK Consortium. SPARK: a US cohort of 50,000 families to accelerate autism research. Neuron 97, 488–493 (2018)."). Summary statistics were downloaded and uniformly processed before computing PRS with PLINK’s clumping algorithm[53](https://www.nature.com/articles/s41588-025-02224-z#ref-CR53 "Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015)."). Results were regressed by sex and the first six principal components to control for ancestry. All GWAS statistics used were based on populations of European ancestry. Adjustment for multiple comparisons was performed for each GWAS using Benjamini–Hochberg correction.

#### DNV calling and filtering

DNVs were called using the HAT[41](https://www.nature.com/articles/s41588-025-02224-z#ref-CR41 "Ng, J. K. & Turner, T. N. HAT: de novo variant calling for highly accurate short-read and long-read sequencing data. Bioinformatics 40, btad775 (2024).") software for all complete trios in our proband and sibling cohort with whole-exome sequence v.3 data (_n_ = 2,013 probands, _n_ = 1,013 paired siblings) from the variant call format files released by the SPARK Consortium. Briefly, this pipeline, which was optimized for whole-genome sequencing and whole-exome sequencing data, used variant calls from both DeepVariant (v.1.1.0)[54](https://www.nature.com/articles/s41588-025-02224-z#ref-CR54 "Poplin, R. et al. A universal SNP and small-indel variant caller using deep neural networks. Nat. Biotechnol. 36, 983–987 (2018).") and GATK HaplotypeCaller (v.4.1.2.0)[55](https://www.nature.com/articles/s41588-025-02224-z#ref-CR55 "McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303 (2010).") to identify variants that appeared in both variant call sets in the child (genotype of 0/1 or 1/1) but not in either parent (both genotypes 0/0). Variants were then filtered based on several quality metrics including read depth, genotype quality score and genomic regions. Variants in recent repeats, low complexity regions and centromeres were filtered out. We used the default values for the filtering parameters of minimum depth = 10 and minimum genotype quality score = 20. For filtering and quality control of DNVs, individuals with a high variant count (>3 s.d. above the mean across all trios) were excluded from the analysis, and nonsingleton DNVs (variants appearing in multiple SPARK families) were removed from further processing and analysis. This pipeline identified an average of 2.72 DNVs per proband (s.d. = 2.20) and 2.67 DNVs per sibling (s.d. = 1.63).

#### Rare inherited variation

The DNV pipeline was adapted to identify inherited variants using the same quality metrics. Inherited variants were identified as variants appearing in a child (genotype of 0/1 or 1/1) and in at least one parent (either genotype 0/1 or 1/1). They were then filtered in the same way as the DNVs to identify a high-confidence set of inherited variants. We identified an average of 41,725.2 inherited variants per proband (s.d. = 2,580.7) and 41,778.9 inherited variants per sibling (s.d. = 2,314.6). In all analyses, rare inherited variants were defined as inherited variants with an allele frequency <1% in gnomAD (v.4.1.0).

#### Gene sets and resources

We computed variant enrichments in seven different gene sets, including the set of all protein-coding genes from GENCODE v.29 (Supplementary [Data](https://www.nature.com/articles/s41588-025-02224-z#MOESM5)). The SFARI gene set was extracted from SFARIGene and included category 1 genes only[19](https://www.nature.com/articles/s41588-025-02224-z#ref-CR19 "Banerjee-Basu, S. & Packer, A. SFARI Gene: an evolving database for the autism research community. Dis. Model. Mech. 3, 133–135 (2010)."),[22](https://www.nature.com/articles/s41588-025-02224-z#ref-CR22 "Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). Mol. Autism 4, 36 (2013)."). The Satterstrom gene set was retrieved from ref. [9](https://www.nature.com/articles/s41588-025-02224-z#ref-CR9 "Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)."). The rest of the gene sets were retrieved from ref. [44](https://www.nature.com/articles/s41588-025-02224-z#ref-CR44 "Werling, D. M. et al. An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder. Nat. Genet. 50, 727–736 (2018).") and included genes with pLI > 0.9 from ExAC[56](https://www.nature.com/articles/s41588-025-02224-z#ref-CR56 "Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)."), predicted ASD risk genes (FDR < 0.3) from ref. [6](https://www.nature.com/articles/s41588-025-02224-z#ref-CR6 "Sanders, S. J. et al. Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci. Neuron 87, 1215–1233 (2015).") and target genes of FMRP[45](https://www.nature.com/articles/s41588-025-02224-z#ref-CR45 "Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. Cell 146, 247–261 (2011)."). Finally, brain-expressed genes were selected using the expression table from GTEx v.7 (gene median transcripts per million per tissue)[46](https://www.nature.com/articles/s41588-025-02224-z#ref-CR46 "GTEx Consortium et al. Genetic effects on gene expression across human tissues. Nature 550, 204–213 (2017)."). The genes selected were those whose expression in brain tissue was at least five times higher than the median expression across all tissues. The high-constraint and moderate-constraint gene sets were selected using pLI scores from gnomAD[42](https://www.nature.com/articles/s41588-025-02224-z#ref-CR42 "Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020).").

#### Count-based burden analyses

For analyses of de novo and inherited variation, we analyzed the distributions of variant count burden for each class of probands or siblings. After extracting de novo and inherited variants for each individual, we ran the variants through Ensembl’s Variant Effect Prediction[57](https://www.nature.com/articles/s41588-025-02224-z#ref-CR57 "McLaren, W. et al. The Ensembl Variant Effect Predictor. Genome Biol. 17, 122 (2016).") tool (release 111.0) with the –pick flag (most severe consequence) and the LOFTEE[42](https://www.nature.com/articles/s41588-025-02224-z#ref-CR42 "Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020).") (v.1.0.4) and AlphaMissense[58](https://www.nature.com/articles/s41588-025-02224-z#ref-CR58 "Cheng, J. et al. Accurate proteome-wide missense variant effect prediction with AlphaMissense. Science 381, eadg7492 (2023).") plugins. To predict LoF variants, we flagged variants with the following predicted consequences: stop gained, frameshift variant, splice acceptor variant, splice donor variant, start lost, stop lost and transcript ablation. We further subset the variants to only include those flagged as high-confidence (‘HC’) by LOFTEE. To predict high-confidence missense mutations, we flagged variants with the following predicted consequences: missense variant, inframe deletion, inframe insertion and protein-altering variant. We further subset the variants to those predicted as ‘likely pathogenic’ by AlphaMissense. Some individuals had zero exome-wide calls for DNVs (_n_ = 91 probands and _n_ = 67 siblings). We accounted for their variant counts in each count-based analysis of DNVs.

We aggregated the counts of dnLoF, dnMis, rare inherited LoF variants and rare inherited missense variants for each individual. Modeling the variant counts as a distribution for each class and siblings, we performed hypothesis testing (one-tailed independent _t_\-tests) followed by Benjamini–Hochberg correction for each variant class. Hypothesis testing was performed for all class–sibling and class–class comparisons.

#### Gene set and OR analyses

We computed the gene-set-specific variant burden counts and repeated the same hypothesis testing procedure using one-sided independent _t_\-tests followed by Benjamini–Hochberg correction. All proband variant count distributions were tested against the sibling variant count distribution, and each class was also tested relative to all out-of-class probands and other classes. ORs were computed for each phenotype class and gene set. All comparisons were made for each class against siblings using Fisher’s exact tests followed by Benjamini–Hochberg correction for each gene set and variant type separately.

#### GO term analysis

To extract the top disrupted biological processes for each class, we flagged dnLoF and dnMis variants present in all protein-coding genes across individuals in each class. To extract the top GO biological processes and molecular functions for each class, we used ShinyGO 0.80 (ref. [59](https://www.nature.com/articles/s41588-025-02224-z#ref-CR59 "Ge, S. X., Jung, D. & Yao, R. ShinyGO: a graphical gene-set enrichment tool for animals and plants. Bioinformatics 36, 2628–2629 (2020).")) (results generated on September 27, 2024), which computes FE and FDR enrichment values using a hypergeometric test, with all protein-coding genes as background. We selected the terms by FDR enrichment and sorted for top terms by FE values. For the Moderate challenges, Social/behavioral and Mixed ASD with DD classes, we used an FDR cutoff of 0.05. For the Broadly affected class, we used an FDR cutoff of 0.1 owing to the smaller class size.

#### Developmental gene expression analysis

We leveraged a single-cell human prefrontal cortex dataset collected from postmortem tissues at six different stages of development[48](https://www.nature.com/articles/s41588-025-02224-z#ref-CR48 "Herring, C. A. et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. Cell 185, 4428–4447.e28 (2022)."). We used Table S3 from ref. [48](https://www.nature.com/articles/s41588-025-02224-z#ref-CR48 "Herring, C. A. et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. Cell 185, 4428–4447.e28 (2022).") to retrieve brain development gene sets (termed devDEGs) for prefrontal cortex cell types that followed one of four general gene trend patterns defined by the authors: ‘up’ (increasing expression throughout stages, with the highest expression in late stages), ‘trans up’ (increasing expression in earlier stages, peak in middle stages, followed by a decline in later stages), ‘down’ (declining expression throughout stages, with the highest expression in the fetal stage) and ‘trans down’ (declining expression in early stages, lowest expression in middle stages and increasing expression in late stages). devDEGs were identified through a differential expression analysis of the six developmental stages (fetal, neonatal, infancy, childhood, adolescence, adulthood). Although 14 major trajectories were identified, we opted to use the four general gene trends for clarity and interpretability. We further combined clusters of cells into the four identified major cell type categories: principal excitatory neurons, inhibitory interneurons (medial ganglionic eminence), inhibitory interneurons (caudal ganglionic eminence) and glia. This was done by taking the union of devDEGs associated with each major cell-type category. We conducted a count burden analysis of dnLoF mutations in each phenotype class compared to nonautistic siblings and for every class–class comparison and computed FE values (of the means) and _P_ values (one-sided independent _t_\-tests) followed by Benjamini–Hochberg correction.

To compare the constraint of genes affected in the Moderate challenges and Mixed ASD with DD classes, we extracted the genes with high-confidence dnLoF variants in each class for the following combinations of gene expression trends and cell types: down in principal excitatory neurons, down in inhibitory interneurons (medial ganglionic eminence), down in inhibitory interneurons (caudal ganglionic eminence) and down in glia. These categories displayed enrichment for dnLoF variants across both classes. We extracted the pLI values for affected genes in each class and conducted a one-tailed Mood’s median test to determine whether the distributions of pLI scores were significantly different between the two classes.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02224-z#MOESM2) linked to this article.

Data availability
-----------------

To abide by the informed consent that individuals with autism and their family members signed when agreeing to participate in a SFARI cohort (SSC and SPARK), researchers must be approved by SFARI Base ([https://base.sfari.org](https://base.sfari.org/)).

Code availability
-----------------

Code and scripts used for the analyses presented in this manuscript are available via GitHub at [https://github.com/FunctionLab/asd-pheno-classes](https://github.com/FunctionLab/asd-pheno-classes) and via Zenodo at [https://doi.org/10.5281/zenodo.15324658](https://doi.org/10.5281/zenodo.15324658) (ref. [60](https://www.nature.com/articles/s41588-025-02224-z#ref-CR60 "Litman, A. Decomposition of phenotypic heterogeneity in autism reveals underlying genetic programs. Zenodo 
https://doi.org/10.5281/zenodo.15324658
(2025).")).

References
----------

1.  _Diagnostic and Statistical Manual of Mental Disorders_ 5th edn (American Psychiatric Association, 2013).
    
2.  Lord, C. et al. Autism spectrum disorder. _Nat. Rev. Dis. Prim._ **6**, 5 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31949163)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autism%20spectrum%20disorder&journal=Nat.%20Rev.%20Dis.%20Prim.&volume=6&publication_year=2020&author=Lord%2CC) 
    
3.  Chiarotti, F. & Venerosi, A. Epidemiology of autism spectrum disorders: a review of worldwide prevalence estimates since 2014. _Brain Sci._ **10**, 274 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32370097)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7288022)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epidemiology%20of%20autism%20spectrum%20disorders%3A%20a%20review%20of%20worldwide%20prevalence%20estimates%20since%202014&journal=Brain%20Sci.&volume=10&publication_year=2020&author=Chiarotti%2CF&author=Venerosi%2CA) 
    
4.  Simonoff, E. et al. Psychiatric disorders in children with autism spectrum disorders: prevalence, comorbidity, and associated factors in a population-derived sample. _J. Am. Acad. Child Adolesc. Psychiatry_ **47**, 921–929 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18645422)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Psychiatric%20disorders%20in%20children%20with%20autism%20spectrum%20disorders%3A%20prevalence%2C%20comorbidity%2C%20and%20associated%20factors%20in%20a%20population-derived%20sample&journal=J.%20Am.%20Acad.%20Child%20Adolesc.%20Psychiatry&volume=47&pages=921-929&publication_year=2008&author=Simonoff%2CE) 
    
5.  Sanders, S. J. et al. De novo mutations revealed by whole-exome sequencing are strongly associated with autism. _Nature_ **485**, 237–241 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XlslSisr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22495306)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667984)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20mutations%20revealed%20by%20whole-exome%20sequencing%20are%20strongly%20associated%20with%20autism&journal=Nature&volume=485&pages=237-241&publication_year=2012&author=Sanders%2CSJ) 
    
6.  Sanders, S. J. et al. Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci. _Neuron_ **87**, 1215–1233 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFGhu7%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26402605)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624267)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20into%20autism%20spectrum%20disorder%20genomic%20architecture%20and%20biology%20from%2071%20risk%20loci&journal=Neuron&volume=87&pages=1215-1233&publication_year=2015&author=Sanders%2CSJ) 
    
7.  Sandin, S. et al. The heritability of autism spectrum disorder. _JAMA_ **318**, 1182–1184 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28973605)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5818813)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20heritability%20of%20autism%20spectrum%20disorder&journal=JAMA&volume=318&pages=1182-1184&publication_year=2017&author=Sandin%2CS) 
    
8.  Grove, J. et al. Identification of common genetic risk variants for autism spectrum disorder. _Nat. Genet._ **51**, 431–444 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmsVyqs70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30804558)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6454898)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20common%20genetic%20risk%20variants%20for%20autism%20spectrum%20disorder&journal=Nat.%20Genet.&volume=51&pages=431-444&publication_year=2019&author=Grove%2CJ) 
    
9.  Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. _Cell_ **180**, 568–584.e23 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVShsr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31981491)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7250485)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large-scale%20exome%20sequencing%20study%20implicates%20both%20developmental%20and%20functional%20changes%20in%20the%20neurobiology%20of%20autism&journal=Cell&volume=180&pages=568-584.e23&publication_year=2020&author=Satterstrom%2CFK) 
    
10.  Fu, J. M. et al. Rare coding variation provides insight into the genetic architecture and phenotypic context of autism. _Nat. Genet._ **54**, 1320–1331 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFOksLzI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35982160)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9653013)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rare%20coding%20variation%20provides%20insight%20into%20the%20genetic%20architecture%20and%20phenotypic%20context%20of%20autism&journal=Nat.%20Genet.&volume=54&pages=1320-1331&publication_year=2022&author=Fu%2CJM) 
    
11.  Trost, B. et al. Genomic architecture of autism from comprehensive whole-genome sequence annotation. _Cell_ **185**, 4409–4427.e18 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivVKms7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36368308)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10726699)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20architecture%20of%20autism%20from%20comprehensive%20whole-genome%20sequence%20annotation&journal=Cell&volume=185&pages=4409-4427.e18&publication_year=2022&author=Trost%2CB) 
    
12.  Warrier, V. et al. Genetic correlates of phenotypic heterogeneity in autism. _Nat. Genet._ **54**, 1293–1304 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVGgsbzP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35654973)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9470531)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20correlates%20of%20phenotypic%20heterogeneity%20in%20autism&journal=Nat.%20Genet.&volume=54&pages=1293-1304&publication_year=2022&author=Warrier%2CV) 
    
13.  Antaki, D. et al. A phenotypic spectrum of autism is attributable to the combined effects of rare variants, polygenic risk and sex. _Nat. Genet._ **54**, 1284–1292 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVGgsbzO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35654974)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20phenotypic%20spectrum%20of%20autism%20is%20attributable%20to%20the%20combined%20effects%20of%20rare%20variants%2C%20polygenic%20risk%20and%20sex&journal=Nat.%20Genet.&volume=54&pages=1284-1292&publication_year=2022&author=Antaki%2CD) 
    
14.  Hashem, S. et al. Genetics of structural and functional brain changes in autism spectrum disorder. _Transl. Psychiatry_ **10**, 229 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32661244)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7359361)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetics%20of%20structural%20and%20functional%20brain%20changes%20in%20autism%20spectrum%20disorder&journal=Transl.%20Psychiatry&volume=10&publication_year=2020&author=Hashem%2CS) 
    
15.  Weiner, D. J. et al. Statistical and functional convergence of common and rare genetic influences on autism at chromosome 16p. _Nat. Genet._ **54**, 1630–1639 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XislSlu7bI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36280734)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9649437)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Statistical%20and%20functional%20convergence%20of%20common%20and%20rare%20genetic%20influences%20on%20autism%20at%20chromosome%2016p&journal=Nat.%20Genet.&volume=54&pages=1630-1639&publication_year=2022&author=Weiner%2CDJ) 
    
16.  Rolland, T. et al. Phenotypic effects of genetic variants associated with autism. _Nat. Med._ **29**, 1671–1680 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlCgu7zO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37365347)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10353945)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phenotypic%20effects%20of%20genetic%20variants%20associated%20with%20autism&journal=Nat.%20Med.&volume=29&pages=1671-1680&publication_year=2023&author=Rolland%2CT) 
    
17.  Courchesne, E. et al. The ASD Living Biology: from cell proliferation to clinical phenotype. _Mol. Psychiatry_ **24**, 88–107 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29934544)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20ASD%20Living%20Biology%3A%20from%20cell%20proliferation%20to%20clinical%20phenotype&journal=Mol.%20Psychiatry&volume=24&pages=88-107&publication_year=2019&author=Courchesne%2CE) 
    
18.  Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. _Nature_ **542**, 433–438 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prevalence%20and%20architecture%20of%20de%20novo%20mutations%20in%20developmental%20disorders&journal=Nature&volume=542&pages=433-438&publication_year=2017) 
    
19.  Banerjee-Basu, S. & Packer, A. SFARI Gene: an evolving database for the autism research community. _Dis. Model. Mech._ **3**, 133–135 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20212079)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SFARI%20Gene%3A%20an%20evolving%20database%20for%20the%20autism%20research%20community&journal=Dis.%20Model.%20Mech.&volume=3&pages=133-135&publication_year=2010&author=Banerjee-Basu%2CS&author=Packer%2CA) 
    
20.  Zhou, X. et al. Integrating de novo and inherited variants in 42,607 autism cases identifies mutations in new moderate-risk genes. _Nat. Genet._ **54**, 1305–1319 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFOksLzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35982159)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9470534)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrating%20de%20novo%20and%20inherited%20variants%20in%2042%2C607%20autism%20cases%20identifies%20mutations%20in%20new%20moderate-risk%20genes&journal=Nat.%20Genet.&volume=54&pages=1305-1319&publication_year=2022&author=Zhou%2CX) 
    
21.  Viggiano, M. et al. Genomic analysis of 116 autism families strengthens known risk genes and highlights promising candidates. _NPJ Genom. Med._ **9**, 21 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXnt1Sltb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38519481)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10959942)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20analysis%20of%20116%20autism%20families%20strengthens%20known%20risk%20genes%20and%20highlights%20promising%20candidates&journal=NPJ%20Genom.%20Med.&volume=9&publication_year=2024&author=Viggiano%2CM) 
    
22.  Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). _Mol. Autism_ **4**, 36 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24090431)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3851189)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SFARI%20Gene%202.0%3A%20a%20community-driven%20knowledgebase%20for%20the%20autism%20spectrum%20disorders%20%28ASDs%29&journal=Mol.%20Autism&volume=4&publication_year=2013&author=Abrahams%2CBS) 
    
23.  Feczko, E. et al. Subtyping cognitive profiles in autism spectrum disorder using a functional random forest algorithm. _Neuroimage_ **172**, 674–688 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC1MzkvVWrsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29274502)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Subtyping%20cognitive%20profiles%20in%20autism%20spectrum%20disorder%20using%20a%20functional%20random%20forest%20algorithm&journal=Neuroimage&volume=172&pages=674-688&publication_year=2018&author=Feczko%2CE) 
    
24.  Mandelli, V. et al. Prognostic early snapshot stratification of autism based on adaptive functioning. _Nat. Ment. Health_ **1**, 327–336 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prognostic%20early%20snapshot%20stratification%20of%20autism%20based%20on%20adaptive%20functioning&journal=Nat.%20Ment.%20Health&volume=1&pages=327-336&publication_year=2023&author=Mandelli%2CV) 
    
25.  Chan, A. J. S. et al. Genome-wide rare variant score associates with morphological subtypes of autism spectrum disorder. _Nat. Commun._ **13**, 6463 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xislyqt7zJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36309498)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9617891)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20rare%20variant%20score%20associates%20with%20morphological%20subtypes%20of%20autism%20spectrum%20disorder&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Chan%2CAJS) 
    
26.  Buch, A. M. et al. Molecular and network-level mechanisms explaining individual differences in autism spectrum disorder. _Nat. Neurosci._ **26**, 650–663 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXltFGht74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36894656)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11446249)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20and%20network-level%20mechanisms%20explaining%20individual%20differences%20in%20autism%20spectrum%20disorder&journal=Nat.%20Neurosci.&volume=26&pages=650-663&publication_year=2023&author=Buch%2CAM) 
    
27.  Ellegood, J. et al. Clustering autism: using neuroanatomical differences in 26 mouse models to gain insight into the heterogeneity. _Mol. Psychiatry_ **20**, 118–125 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFylsbrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25199916)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clustering%20autism%3A%20using%20neuroanatomical%20differences%20in%2026%20mouse%20models%20to%20gain%20insight%20into%20the%20heterogeneity&journal=Mol.%20Psychiatry&volume=20&pages=118-125&publication_year=2015&author=Ellegood%2CJ) 
    
28.  Luo, Y. et al. A multidimensional precision medicine approach identifies an autism subtype characterized by dyslipidemia. _Nat. Med._ **26**, 1375–1379 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKlurrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32778826)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20multidimensional%20precision%20medicine%20approach%20identifies%20an%20autism%20subtype%20characterized%20by%20dyslipidemia&journal=Nat.%20Med.&volume=26&pages=1375-1379&publication_year=2020&author=Luo%2CY) 
    
29.  Cai, N. et al. Minimal phenotyping yields genome-wide association signals of low specificity for major depression. _Nat. Genet._ **52**, 437–447 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlvFymtbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32231276)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7906795)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Minimal%20phenotyping%20yields%20genome-wide%20association%20signals%20of%20low%20specificity%20for%20major%20depression&journal=Nat.%20Genet.&volume=52&pages=437-447&publication_year=2020&author=Cai%2CN) 
    
30.  LaBianca, S. et al. Polygenic profiles define aspects of clinical heterogeneity in attention deficit hyperactivity disorder. _Nat. Genet._ **56**, 234–244 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFSmu7rJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38036780)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20profiles%20define%20aspects%20of%20clinical%20heterogeneity%20in%20attention%20deficit%20hyperactivity%20disorder&journal=Nat.%20Genet.&volume=56&pages=234-244&publication_year=2024&author=LaBianca%2CS) 
    
31.  Lombardo, M. V., Lai, M.-C. & Baron-Cohen, S. Big data approaches to decomposing heterogeneity across the autism spectrum. _Mol. Psychiatry_ **24**, 1435–1450 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30617272)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6754748)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Big%20data%20approaches%20to%20decomposing%20heterogeneity%20across%20the%20autism%20spectrum&journal=Mol.%20Psychiatry&volume=24&pages=1435-1450&publication_year=2019&author=Lombardo%2CMV&author=Lai%2CM-C&author=Baron-Cohen%2CS) 
    
32.  Agelink van Rentergem, J. A., Deserno, M. K. & Geurts, H. M. Validation strategies for subtypes in psychiatry: a systematic review of research on autism spectrum disorder. _Clin. Psychol. Rev._ **87**, 102033 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33962352)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Validation%20strategies%20for%20subtypes%20in%20psychiatry%3A%20a%20systematic%20review%20of%20research%20on%20autism%20spectrum%20disorder&journal=Clin.%20Psychol.%20Rev.&volume=87&publication_year=2021&author=Agelink%20van%20Rentergem%2CJA&author=Deserno%2CMK&author=Geurts%2CHM) 
    
33.  SPARK Consortium. SPARK: a US cohort of 50,000 families to accelerate autism research. _Neuron_ **97**, 488–493 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SPARK%3A%20a%20US%20cohort%20of%2050%2C000%20families%20to%20accelerate%20autism%20research&journal=Neuron&volume=97&pages=488-493&publication_year=2018) 
    
34.  Rutter, M., Bailey, A. & Lord, C. _The social Communication Questionnaire: Manual_ (Western Psychological Services, 2003).
    
35.  Lam, K. S. L. & Aman, M. G. The Repetitive Behavior Scale-Revised: independent validation in individuals with autism spectrum disorders. _J. Autism Dev. Disord._ **37**, 855–866 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17048092)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Repetitive%20Behavior%20Scale-Revised%3A%20independent%20validation%20in%20individuals%20with%20autism%20spectrum%20disorders&journal=J.%20Autism%20Dev.%20Disord.&volume=37&pages=855-866&publication_year=2007&author=Lam%2CKSL&author=Aman%2CMG) 
    
36.  Offermans, J. E. et al. The development and validation of a subscale for the school-age Child Behavior CheckList to screen for autism spectrum disorder. _J. Autism Dev. Disord._ **53**, 1034–1052 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35165798)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20development%20and%20validation%20of%20a%20subscale%20for%20the%20school-age%20Child%20Behavior%20CheckList%20to%20screen%20for%20autism%20spectrum%20disorder&journal=J.%20Autism%20Dev.%20Disord.&volume=53&pages=1034-1052&publication_year=2023&author=Offermans%2CJE) 
    
37.  Berument, S. K., Rutter, M., Lord, C., Pickles, A. & Bailey, A. Autism screening questionnaire: diagnostic validity. _Br. J. Psychiatry_ **175**, 444–451 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD3c3kvFOmsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10789276)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autism%20screening%20questionnaire%3A%20diagnostic%20validity&journal=Br.%20J.%20Psychiatry&volume=175&pages=444-451&publication_year=1999&author=Berument%2CSK&author=Rutter%2CM&author=Lord%2CC&author=Pickles%2CA&author=Bailey%2CA) 
    
38.  Grove, R. et al. Factor structure of the Social Communication Questionnaire in preschool aged autistic children. _J. Child Fam. Stud_. **28**, 3385–3391 (2019).
    
39.  Achenbach, T. M. & Edelbrock, C. S. The Child Behavior Profile: II. Boys aged 12–16 and girls aged 6–11 and 12–16. _J. Consult. Clin. Psychol._ **47**, 223–233 (1979).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaE1M3jvFyltA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=469068)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Child%20Behavior%20Profile%3A%20II.%20Boys%20aged%2012%E2%80%9316%20and%20girls%20aged%206%E2%80%9311%20and%2012%E2%80%9316&journal=J.%20Consult.%20Clin.%20Psychol.&volume=47&pages=223-233&publication_year=1979&author=Achenbach%2CTM&author=Edelbrock%2CCS) 
    
40.  Fischbach, G. D. & Lord, C. The Simons Simplex Collection: a resource for identification of autism genetic risk factors. _Neuron_ **68**, 192–195 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlSmtr7P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20955926)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Simons%20Simplex%20Collection%3A%20a%20resource%20for%20identification%20of%20autism%20genetic%20risk%20factors&journal=Neuron&volume=68&pages=192-195&publication_year=2010&author=Fischbach%2CGD&author=Lord%2CC) 
    
41.  Ng, J. K. & Turner, T. N. HAT: de novo variant calling for highly accurate short-read and long-read sequencing data. _Bioinformatics_ **40**, btad775 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXit1Ogu7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38175776)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10777354)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HAT%3A%20de%20novo%20variant%20calling%20for%20highly%20accurate%20short-read%20and%20long-read%20sequencing%20data&journal=Bioinformatics&volume=40&publication_year=2024&author=Ng%2CJK&author=Turner%2CTN) 
    
42.  Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. _Nature_ **581**, 434–443 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32461654)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&journal=Nature&volume=581&pages=434-443&publication_year=2020&author=Karczewski%2CKJ) 
    
43.  Zhou, J. et al. Whole-genome deep-learning analysis identifies contribution of noncoding mutations to autism risk. _Nat. Genet._ **51**, 973–980 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVChsr7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31133750)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6758908)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20deep-learning%20analysis%20identifies%20contribution%20of%20noncoding%20mutations%20to%20autism%20risk&journal=Nat.%20Genet.&volume=51&pages=973-980&publication_year=2019&author=Zhou%2CJ) 
    
44.  Werling, D. M. et al. An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder. _Nat. Genet._ **50**, 727–736 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXosVCks7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29700473)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5961723)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20analytical%20framework%20for%20whole-genome%20sequence%20association%20studies%20and%20its%20implications%20for%20autism%20spectrum%20disorder&journal=Nat.%20Genet.&volume=50&pages=727-736&publication_year=2018&author=Werling%2CDM) 
    
45.  Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. _Cell_ **146**, 247–261 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXptleksbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21784246)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3232425)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FMRP%20stalls%20ribosomal%20translocation%20on%20mRNAs%20linked%20to%20synaptic%20function%20and%20autism&journal=Cell&volume=146&pages=247-261&publication_year=2011&author=Darnell%2CJC) 
    
46.  GTEx Consortium et al. Genetic effects on gene expression across human tissues. _Nature_ **550**, 204–213 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20effects%20on%20gene%20expression%20across%20human%20tissues&journal=Nature&volume=550&pages=204-213&publication_year=2017) 
    
47.  Garber, K. B., Visootsak, J. & Warren, S. T. Fragile X syndrome. _Eur. J. Hum. Genet._ **16**, 666–672 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFGgurk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18398441)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fragile%20X%20syndrome&journal=Eur.%20J.%20Hum.%20Genet.&volume=16&pages=666-672&publication_year=2008&author=Garber%2CKB&author=Visootsak%2CJ&author=Warren%2CST) 
    
48.  Herring, C. A. et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. _Cell_ **185**, 4428–4447.e28 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XislKgt7rI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36318921)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20prefrontal%20cortex%20gene%20regulatory%20dynamics%20from%20gestation%20to%20adulthood%20at%20single-cell%20resolution&journal=Cell&volume=185&pages=4428-4447.e28&publication_year=2022&author=Herring%2CCA) 
    
49.  Morin, S. et al. StepMix: a Python package for pseudo-likelihood estimation of generalized mixture models with external variables. Preprint at [https://doi.org/10.48550/arXiv.2304.03853](https://doi.org/10.48550/arXiv.2304.03853) (2023).
    
50.  Masyn, K. E. in _The Oxford Handbook of Quantitative Methods_, Vol. 2 (ed Little, T. D.) Ch. 25 (Oxford Univ. Press, 2013).
    
51.  Nylund-Gibson, K. & Choi, A. Y. Ten frequently asked questions about latent class analysis. _Transl. Issues Psychol. Sci._ **4**, 440–461 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ten%20frequently%20asked%20questions%20about%20latent%20class%20analysis&journal=Transl.%20Issues%20Psychol.%20Sci.&volume=4&pages=440-461&publication_year=2018&author=Nylund-Gibson%2CK&author=Choi%2CAY) 
    
52.  Sinha, P., Calfee, C. S. & Delucchi, K. L. Practitioner’s guide to latent class analysis: methodological considerations and common pitfalls. _Crit. Care Med._ **49**, e63–e79 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33165028)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7746621)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Practitioner%E2%80%99s%20guide%20to%20latent%20class%20analysis%3A%20methodological%20considerations%20and%20common%20pitfalls&journal=Crit.%20Care%20Med.&volume=49&pages=e63-e79&publication_year=2021&author=Sinha%2CP&author=Calfee%2CCS&author=Delucchi%2CKL) 
    
53.  Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. _Gigascience_ **4**, 7 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25722852)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&journal=Gigascience&volume=4&publication_year=2015&author=Chang%2CCC) 
    
54.  Poplin, R. et al. A universal SNP and small-indel variant caller using deep neural networks. _Nat. Biotechnol._ **36**, 983–987 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslOhtbjE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30247488)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20universal%20SNP%20and%20small-indel%20variant%20caller%20using%20deep%20neural%20networks&journal=Nat.%20Biotechnol.&volume=36&pages=983-987&publication_year=2018&author=Poplin%2CR) 
    
55.  McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. _Genome Res._ **20**, 1297–1303 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFeru7jM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20644199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&journal=Genome%20Res.&volume=20&pages=1297-1303&publication_year=2010&author=McKenna%2CA) 
    
56.  Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. _Nature_ **536**, 285–291 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlOnsbbP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27535533)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018207)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060%2C706%20humans&journal=Nature&volume=536&pages=285-291&publication_year=2016&author=Lek%2CM) 
    
57.  McLaren, W. et al. The Ensembl Variant Effect Predictor. _Genome Biol._ **17**, 122 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27268795)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4893825)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Ensembl%20Variant%20Effect%20Predictor&journal=Genome%20Biol.&volume=17&publication_year=2016&author=McLaren%2CW) 
    
58.  Cheng, J. et al. Accurate proteome-wide missense variant effect prediction with AlphaMissense. _Science_ **381**, eadg7492 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvFyjtbfE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37733863)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20proteome-wide%20missense%20variant%20effect%20prediction%20with%20AlphaMissense&journal=Science&volume=381&publication_year=2023&author=Cheng%2CJ) 
    
59.  Ge, S. X., Jung, D. & Yao, R. ShinyGO: a graphical gene-set enrichment tool for animals and plants. _Bioinformatics_ **36**, 2628–2629 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlOlsrnP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31882993)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ShinyGO%3A%20a%20graphical%20gene-set%20enrichment%20tool%20for%20animals%20and%20plants&journal=Bioinformatics&volume=36&pages=2628-2629&publication_year=2020&author=Ge%2CSX&author=Jung%2CD&author=Yao%2CR) 
    
60.  Litman, A. Decomposition of phenotypic heterogeneity in autism reveals underlying genetic programs. _Zenodo_ [https://doi.org/10.5281/zenodo.15324658](https://doi.org/10.5281/zenodo.15324658) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02224-z?format=refman&flavour=references)

Acknowledgements
----------------

We thank all the families in SPARK, the SPARK clinical sites and SPARK staff. We acknowledge access to the SPARK phenotypic and genetic datasets on SFARI Base. Approved researchers can obtain the SPARK population dataset described in this study by applying at [https://base.sfari.org](https://base.sfari.org/). This work used computing resources supported by the Scientific Computing Core at the Flatiron Institute and was supported by funding from the NIH NIGMS no. R01GM071966 (O.G.T.), Simons Foundation grant 395506 (O.G.T.) and NIH NHGRI training grant T32HG003284 (A.L.).

Author information
------------------

Author notes

1.  These authors contributed equally: Aviya Litman, Natalie Sauerwald.
    

### Authors and Affiliations

1.  Quantitative and Computational Biology Program, Princeton University, Princeton, NJ, USA
    
    Aviya Litman
    
2.  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ, USA
    
    Aviya Litman, Chandra L. Theesfeld & Olga G. Troyanskaya
    
3.  Center for Computational Biology, Flatiron Institute, New York, NY, USA
    
    Natalie Sauerwald, Christopher Y. Park, Yun Hao & Olga G. Troyanskaya
    
4.  Simons Foundation, New York, NY, USA
    
    LeeAnne Green Snyder & Jennifer Foss-Feig
    
5.  Department of Psychiatry, Mount Sinai Icahn School of Medicine, New York, NY, USA
    
    Jennifer Foss-Feig
    
6.  Seaver Autism Center for Research and Treatment, Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA
    
    Jennifer Foss-Feig
    
7.  Cognitive and Brain Sciences Department, Ben Gurion University of the Negev, Be’er Sheva, Israel
    
    Ilan Dinstein
    
8.  Azrieli National Centre for Autism and Neurodevelopment Research, Ben Gurion University of the Negev, Be’er Sheva, Israel
    
    Ilan Dinstein
    
9.  Psychology Department, Ben Gurion University of the Negev, Be’er Sheva, Israel
    
    Ilan Dinstein
    
10.  Princeton Precision Health, Princeton, NJ, USA
    
    Chandra L. Theesfeld & Olga G. Troyanskaya
    
11.  Department of Computer Science, Princeton University, Princeton, NJ, USA
    
    Olga G. Troyanskaya
    

Authors

1.  Aviya Litman
    
    [View author publications](https://www.nature.com/search?author=Aviya%20Litman)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Aviya%20Litman) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Aviya%20Litman%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Natalie Sauerwald
    
    [View author publications](https://www.nature.com/search?author=Natalie%20Sauerwald)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Natalie%20Sauerwald) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Natalie%20Sauerwald%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  LeeAnne Green Snyder
    
    [View author publications](https://www.nature.com/search?author=LeeAnne%20Green%20Snyder)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=LeeAnne%20Green%20Snyder) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22LeeAnne%20Green%20Snyder%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Jennifer Foss-Feig
    
    [View author publications](https://www.nature.com/search?author=Jennifer%20Foss-Feig)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jennifer%20Foss-Feig) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jennifer%20Foss-Feig%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Christopher Y. Park
    
    [View author publications](https://www.nature.com/search?author=Christopher%20Y.%20Park)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Christopher%20Y.%20Park) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Christopher%20Y.%20Park%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Yun Hao
    
    [View author publications](https://www.nature.com/search?author=Yun%20Hao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yun%20Hao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yun%20Hao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Ilan Dinstein
    
    [View author publications](https://www.nature.com/search?author=Ilan%20Dinstein)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ilan%20Dinstein) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ilan%20Dinstein%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Chandra L. Theesfeld
    
    [View author publications](https://www.nature.com/search?author=Chandra%20L.%20Theesfeld)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chandra%20L.%20Theesfeld) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chandra%20L.%20Theesfeld%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Olga G. Troyanskaya
    
    [View author publications](https://www.nature.com/search?author=Olga%20G.%20Troyanskaya)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Olga%20G.%20Troyanskaya) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Olga%20G.%20Troyanskaya%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

A.L. and N.S. contributed equally to this work, conceiving the study, performing analyses, and writing the paper with the help of C.L.T. and O.G.T. L.G.S. and J.F.-F. provided clinical expertise and supervision for phenotype analyses. C.Y.P., Y.H. and I.D. proposed experiments and gave advice on genetic analyses.

### Corresponding authors

Correspondence to [Natalie Sauerwald](mailto:nsauerwald@flatironinstitute.org) or [Olga G. Troyanskaya](mailto:ogt@genomics.princeton.edu).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Andrew Schork, Varun Warrier and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02224-z#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Exploratory class enumeration analyses suggest four components for the SPARK generative mixture model.](https://www.nature.com/articles/s41588-025-02224-z/figures/6)

A crucial step in mixture modeling is tuning the number of model components through evaluation of fit metrics. Models were trained with varying numbers of components (_x_\-axis) on the phenotype data of _n_ = 5,392 probands over 50–200 iterations with randomly generated seeds. Points represent means and error bars represent the standard deviation for a variety of statistical indicators (_y_\-axis): Validation Log Likelihood (LL), Akaike Information Criterion (AIC), Bayesian Information Criterion (BIC), Sample-Size-Adjusted BIC (SABIC), Consistent AIC (CAIC), and the Likelihood Ratio Test (LRT). The evaluation of these indicators combined with interpretation of each candidate model and class resulted in the selection of a four-component model (shown as vertical dotted line in all plots).

### [Extended Data Fig. 2 Model characteristics for 3-, 5-, and 6-class models.](https://www.nature.com/articles/s41588-025-02224-z/figures/7)

In addition to a wide range of statistics, we also considered interpretability in selecting the four-class model. Top plots represent proportion and direction of affinity of each class (_y_\-axis) towards each of seven phenotype categories (_x_\-axis). Bottom plots show SCQ total score and developmental milestone scores (_y_\-axis) for proband classes (_n_ = 5,392 total) and unaffected siblings (_n_ = 1,972). Though the three-class model (_n_ = 1,868, 1,068, and 2,456) also shows separation between classes, there remains a lot of within-class variation, suggesting that additional classes would better capture the heterogeneity of the cohort. The five-class model (_n_ = 781, 1,828, 949, 462, and 1,372) is challenging to interpret, with mixed enrichment patterns and overlap between classes. The six class solution (_n_ = 929, 1,356, 1,263, 390, 690, and 764) is even less interpretable, likely overfitting to our cohort and splitting classes with substantial similarities. All models were trained on phenotype data from _n_ = 5,392 probands in the sample. Center lines in all boxplots represent the median, box limits represent the 25th and 75th percentiles, whiskers extend to show 1.5× interquartile range, and outliers are shown separately as open circles. One-sided independent _t_\-tests with adjustment for multiple comparisons (Benjamini-Hochberg correction) were used to determine significance of enrichment for each class compared to siblings (\* indicates FDR < 0.1, \*\* indicates FDR < 0.05, and \*\*\* indicates FDR < 0.01 in all figures). Stars directly above the boxplot indicate comparisons to siblings.

### [Extended Data Fig. 3 Model stability and robustness to subsampling and random initializations.](https://www.nature.com/articles/s41588-025-02224-z/figures/8)

**a**, We trained 100 independent models (each with 20 random initializations) with a subsampling rate of 50% of individuals in the sample (_n_ = 2,696). We compared both the individual class assignments and the overall phenotypic correlations between the subsampled models and the full model. We observed very strong stability across the subsampled models, with averages of 88–96% of individuals assigned to the same group as the full model, as shown in the heatmap representing the proportion of individuals assigned to the same group in the subsampled model as the full model. Additionally, the feature concordance among subsamples and across categories of phenotypes remains very stable, as shown in the bar plot quantifying correlations between the feature enrichments of the subsampled model and the full model (_x_\-axis). In all displayed plots, values and bars represent means, error intervals represent the 95% confidence interval, the color bar varies continuously from low proportion (white) to high proportion (magenta), and individual data points are overlaid on the right. **b**, We trained 2,000 independent models on the full sample (_n_ = 5,392) with random initializations, each time recording the log likelihood (LL) of the model. We ranked the trained models by their LL and retained the top 100 models for further analysis. We observe very strong stability across the top randomly initialized models, with averages of 90–97% of individuals assigned to the same group as the original model reported in the manuscript, as shown in the heatmap representing the proportion of individuals assigned to the same group in the randomly initialized models as the original model. Additionally, the feature concordance among randomly initialized models and across categories of phenotypes remains very stable, as shown in the bar plot quantifying correlations between the feature enrichments of the randomly initialized models and the original model (_x_\-axis).

### [Extended Data Fig. 4 Comparison of feature variances.](https://www.nature.com/articles/s41588-025-02224-z/figures/9)

**a**, Variability of feature enrichment and direction of enrichment computed for the sample (_n_ = 5,392). To demonstrate variability in feature significance within phenotype categories and classes, enrichments and depletions are plotted for each combination of phenotype category and class, with the _y_\-axis representing the proportion and direction of enrichment of features assigned to the phenotype category (_x_\-axis) and computed within the class. Sample sizes for analyses shown are: Broadly affected (magenta, _n_ = 554), Social/behavioral (green, _n_ = 1,976), Mixed ASD with DD (blue, _n_ = 1,002), Moderate challenges (orange, _n_ = 1,860), and unaffected siblings (_n_ = 1,972). **b**, We computed the normalized variance (_x_\-axis) of each feature across and within classes for _n_ = 5,392 probands. To obtain the distribution of within class variances, we computed the weighted average of the variances across the four classes for each feature. We plotted the distributions of feature variances, demonstrating that variances between classes significantly exceed variances within classes for _n_ = 205 features (_P_ = 9.97 × 10−176; one-sided independent _t_\-test).

### [Extended Data Fig. 5 Age and sex distribution breakdown by class.](https://www.nature.com/articles/s41588-025-02224-z/figures/10)

**a**, Breakdown of proportion of SPARK cohort individuals (total _n_ = 5,392) assigned by the generative mixture model to each latent class. Classes are represented by the corresponding class colors and numbers are rounded to the nearest percent. **b**, Proportion of males and females (_y_\-axis) for individuals (total _n_ = 5,392) assigned to each latent class. **c**, Density of age distribution (_x_\-axis) for individuals assigned to each latent class (total _n_ = 5,392). Colors correspond to class colors. **d**, Stacked bar plots showing distribution of proportions (_y_\-axis) across classes (_x_\-axis) for two variables: language level at enrollment, a parent report with four levels reflecting language abilities (0 = Nonverbal, 1 = Single words, 2 = Phrases, 3 = Sentences), and cognitive impairment at enrollment (binary True/False). Sample sizes are: Broadly affected (magenta, _n_ = 554), Social/behavioral (green, _n_ = 1,976), Mixed ASD with DD (blue, _n_ = 1,002), and Moderate challenges (orange, _n_ = 1,860). One-sided binomial tests (cognitive impairment) and one-sided independent _t_\-tests (language level) were used to test for significance between classes (data not available for unaffected sibling control). _P_\-values were adjusted for multiple comparisons using Benjamini-Hochberg correction. \* indicates FDR < 0.1, \*\* indicates FDR < 0.05, and \*\*\* indicates FDR < 0.01 in all figures.

### [Extended Data Fig. 6 Phenotypic replication of autism classes in the Simons Simplex Collection (SSC).](https://www.nature.com/articles/s41588-025-02224-z/figures/11)

**a**, Independent phenotypic replication of classes in the SSC cohort. A model was independently trained on the Simons Simplex Collection (SSC) dataset for 108 matching features and _n_ = 861 probands. The feature enrichment patterns were computed for the seven phenotype categories (_y_\-axis) and Pearson correlation coefficients with the full SPARK model (_n_ = 5,392) are shown (_x_\-axis). **b**, Permutation analysis of SSC phenotypic replication. Pearson correlation coefficients (_x_\-axis) computed between the feature enrichment patterns of latent classes in SSC and SPARK for 10,000 random permutations of class labels in the SSC. The dashed vertical red line represents the true correlation from applying the model trained on SPARK data to predict labels on individuals in SSC (Pearson’s _r_ = 0.927). No chance correlations exceeded the true correlation. Model was trained on _n_ = 6,393 probands in SPARK and applied to _n_ = 861 probands in the SSC. **c**, Pearson correlation coefficients (_x_\-axis) computed between the feature enrichment patterns of latent classes in SSC and SPARK for 1,000 permuted cluster projections onto the SSC. All models were trained independently on the shuffled phenotype labels of the SPARK dataset (shuffling was repeated for every model). The permuted phenotype models were projected onto the SSC and correlated with the true SPARK labels. The dashed vertical red line represents the true correlation from applying the model trained on SPARK data to predict labels on individuals in SSC (Pearson’s _r_ = 0.927). No chance correlations exceeded the true correlation. Models were trained on _n_ = 5,392 probands in SPARK and _n_ = 861 probands in the SSC.

### [Extended Data Fig. 7 Ancestry PCA plot for genotyped cohort.](https://www.nature.com/articles/s41588-025-02224-z/figures/12)

Top two principal components of the SNP data (_x_\-axis, _y_\-axis) representing genetic ancestry. Points are colored by self-reported race.

### [Extended Data Fig. 8 Gene set variant burden analyses.](https://www.nature.com/articles/s41588-025-02224-z/figures/13)

**a**, dnLoF burden was computed for each class against out-of-class probands in each of seven gene sets (_y_\-axis) using one-sided independent _t_\-tests (adjusted for multiple comparisons using Benjamini-Hochberg correction). Fold enrichment (bubble size), FDR significance (closed vs. open circles at indicated threshold of 0.05), and direction of enrichment (color) are shown. Sample sizes in all analyses shown: Moderate challenges _n_ = 809, Broadly affected _n_ = 145, Social/behavioral _n_ = 640, Mixed ASD with DD _n_ = 419, unaffected siblings _n_ = 1,013. All computed statistics, class comparisons, and comparisons to siblings can be found in Supplementary Table [8](https://www.nature.com/articles/s41588-025-02224-z#MOESM1). **b**, Scatter plots displaying fold enrichment (_x_\-axis) versus FDR significance (_y_\-axis) of _de novo_ missense (dnMis) burden, rare inherited LoF (inhLoF) burden, and rare inherited missense (inhMis) burden across classes and seven autism-relevant gene sets (symbol key). We computed the aggregated burdens for each individual across every gene set. _P_\-values and log2 fold change were computed relatively to non-autistic siblings using one-sided independent _t_\-tests. _P_\-values were adjusted for multiple comparisons using Benjamini-Hochberg correction to compute the log-transformed _Q_\-values (_y_\-axis). Empty shapes represent tests that did not pass multiple hypothesis testing with a threshold of 0.05. Each gene set is represented by a different symbol as indicated in the key. **c**, Odds ratios for various gene sets included in this manuscript. Odds ratios (_y_\-axis) were computed for dnLoF and dnSyn variants in each autism class against non-autistic siblings across autism-relevant gene sets (four shown here, two shown in Fig. [4b](https://www.nature.com/articles/s41588-025-02224-z#Fig4)).

### [Extended Data Fig. 9 dnLoF enrichments across PFC celltypes, developmental gene trends, and phenotype classes.](https://www.nature.com/articles/s41588-025-02224-z/figures/14)

dnLoF variant enrichment was computed across cell types and developmental gene expression trends (_y_\-axis) for each phenotype class (_x_\-axis). Open circles represent lack of significance (FDR > 0.05), whereas closed circles represent FDR ≤ 0.05 (one-sided independent _t_\-tests adjusted for multiple comparisons with Benjamini-Hochberg correction). Lack of circles represents lack of data. Colors correspond to phenotype class colors and bubble size corresponds to fold enrichment. Sample sizes are: all probands (_n_ = 2,013), Moderate challenges _n_ = 809, Broadly affected _n_ = 145, Social/behavioral _n_ = 640, Mixed ASD with DD _n_ = 419, unaffected siblings _n_ = 1,013.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_MOESM1_ESM.pdf)

Supplementary Note.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_MOESM3_ESM.pdf)

### [Supplementary Tables 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_MOESM4_ESM.xlsx)

Supplementary Tables 1–11.

### [Supplementary Data 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02224-z/MediaObjects/41588_2025_2224_MOESM5_ESM.csv)

ASD-relevant gene sets used in genetic analyses.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Decomposition%20of%20phenotypic%20heterogeneity%20in%20autism%20reveals%20underlying%20genetic%20programs&author=Aviya%20Litman%20et%20al&contentID=10.1038%2Fs41588-025-02224-z&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-09&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02224-z)

### Cite this article

Litman, A., Sauerwald, N., Green Snyder, L. _et al._ Decomposition of phenotypic heterogeneity in autism reveals underlying genetic programs. _Nat Genet_ **57**, 1611–1619 (2025). https://doi.org/10.1038/s41588-025-02224-z

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02224-z?format=refman&flavour=citation)

-   Received: 25 July 2024
    
-   Accepted: 02 May 2025
    
-   Published: 09 July 2025
    
-   Issue Date: July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02224-z
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02224-z.pdf)
