> id: https://www.nature.com/articles/s41588-025-02330-y

> link: https://www.nature.com/articles/s41588-025-02330-y

> title: Chromatin dynamics of a large-sized genome provides insights into polyphenism and X0 dosage compensation of locusts

# Chromatin dynamics of a large-sized genome provides insights into polyphenism and X0 dosage compensation of locusts
_Published on Mon, 01 Sep 2025 00:00:00 GMT_

Abstract
--------

Locusts are characterized by a large genome size, polyphenism and an X0 sex determination system. Here we generated chromosome-level genomes for both desert and migratory locusts, as well as a comprehensive chromatin map for the latter. We found that genome enlargement is associated with an increased number of enhancers in expanded intronic regions. To explore the function of expanded enhancers, we identified a distal enhancer that contributes to behavioral differences between solitary and gregarious locusts. In the X0 sex system, H4K16ac enrichment and H4K20me1 depletion maintain balanced X-linked expression in male soma. Notably, the distance-dependent H4K16ac enrichment diminishes gradually in intergenic regions, revealing a special dosage compensation mechanism in large genomes. Furthermore, the distance-dependent H4K16ac results in a lag in the evolution of dosage compensation for the X-linked genes translocated recently from autosomes. Therefore, expanded intronic and intergenic regions shape a distinct chromatin regulation landscape in large genomes.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-023-48870-3/MediaObjects/41598_2023_48870_Fig1_HTML.png)

### [Pericentromeric recombination suppression and the ‘large X effect’ in plants](https://www.nature.com/articles/s41598-023-48870-3?fromPaywallRec=false)

Article Open access 07 December 2023

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-020-20830-9/MediaObjects/41467_2020_20830_Fig1_HTML.png)

### [Large organized chromatin lysine domains help distinguish primitive from differentiated cell populations](https://www.nature.com/articles/s41467-020-20830-9?fromPaywallRec=false)

Article Open access 21 January 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-53892-0/MediaObjects/41467_2024_53892_Fig1_HTML.png)

### [Development and evolution of Drosophila chromatin landscape in a 3D genome context](https://www.nature.com/articles/s41467-024-53892-0?fromPaywallRec=false)

Article Open access 01 November 2024

Main
----

The migratory locust (_Locusta migratoria_), with a 6.9-Gb genome, is a typical representative of large-sized-genome insect species and is characterized by marked density-dependent polyphenism and an X0 sex determination system[1](https://www.nature.com/articles/s41588-025-02330-y#ref-CR1 "Wang, X. & Kang, L. Molecular mechanisms of phase change in locusts. Annu Rev. Entomol. 59, 225–244 (2014)."). Polyphenism or phenotypic plasticity, which is the ability of organisms with the same genotypes or genome to change their phenotype in response to a changing environment, has an obvious epigenetic characteristic. As a unique biology feature of locusts, polyphenism facilitates the rapid behavioral changes between solitary and gregarious phases to adapt to changes in population density[1](https://www.nature.com/articles/s41588-025-02330-y#ref-CR1 "Wang, X. & Kang, L. Molecular mechanisms of phase change in locusts. Annu Rev. Entomol. 59, 225–244 (2014)."). Behavioral changes lead to aggregation of locusts, which can ultimately result in the formation of destructive swarms. Therefore, behavioral dissimilarity between the two phases of locusts is the most important phase-related difference because it is the main cause of locust plague outbreaks. The gene _Henna_ encodes the most critical enzyme in dopamine biosynthesis, which determines behavioral changes in locusts[1](https://www.nature.com/articles/s41588-025-02330-y#ref-CR1 "Wang, X. & Kang, L. Molecular mechanisms of phase change in locusts. Annu Rev. Entomol. 59, 225–244 (2014)."),[2](https://www.nature.com/articles/s41588-025-02330-y#ref-CR2 "Ma, Z., Guo, W., Guo, X., Wang, X. & Kang, L. Modulation of behavioral phase changes of the migratory locust by the catecholamine metabolic pathway. Proc. Natl Acad. Sci. USA 108, 3882–3887 (2011)."),[3](https://www.nature.com/articles/s41588-025-02330-y#ref-CR3 "Yang, M. et al. MicroRNA-133 inhibits behavioral aggregation by controlling dopamine synthesis in locusts. PLoS Genet. 10, e1004206 (2014)."). Behavioral plasticity is notably regulated at the transcriptional level, which is influenced by chromatin regulation. Compared to species with small genomes, such as the fruit fly, the gene size of _Henna_ in the migratory locust has increased 32-fold, primarily because of the expansion in intron size. Therefore, _Henna_ represents a unique case in the migratory locust for investigating the role of chromatin in the regulation of insect polyphenism, particularly with respect to expanded intronic regions.

Sexual differentiation is associated closely with chromatin regulation[4](https://www.nature.com/articles/s41588-025-02330-y#ref-CR4 "Gagnidze, K., Weil, Z. M. & Pfaff, D. W. Histone modifications proposed to regulate sexual differentiation of brain and behavior. Bioessays 32, 932–939 (2010)."). Locusts are typical representatives of the X0 sex determination system, which is characterized by male heterogamety. Although unpaired homologous chromosomes can disrupt the equilibrium of gene expression, differences in X chromosome number between the two sexes are mitigated by dosage compensation[5](https://www.nature.com/articles/s41588-025-02330-y#ref-CR5 "Meyer, B. J. The X chromosome in C. elegans sex determination and dosage compensation. Curr. Opin. Genet. Dev. 74, 101912 (2022)."). Dosage compensation strategies involving histone modifications are variable among different sex determination models[6](https://www.nature.com/articles/s41588-025-02330-y#ref-CR6 "Vielle, A. et al. H4K20me1 contributes to downregulation of X-linked genes for C. elegans dosage compensation. PLoS Genet. 8, e1002933 (2012)."). However, the dosage compensation mechanism of an X0 sex determination system, such as that of locusts, that adheres strictly to obligate sexuality following Mendelian inheritance, remains unresolved. In principle, translocated genes between sex chromosomes and autosomes are embedded more frequently in intergenic regions in large genomes than in small genomes because of more spacious context. The spread of H4K16ac, catalyzed by male-specific lethal (MSL) complex, can promote dosage compensation to the flanking intergenic that lack MSL complex binding[7](https://www.nature.com/articles/s41588-025-02330-y#ref-CR7 "Straub, T., Grimaud, C., Gilfillan, G. D., Mitterweger, A. & Becker, P. B. The chromosomal high-affinity binding sites for the Drosophila dosage compensation complex. PLoS Genet. 4, e1000302 (2008)."). Therefore, the expansive intergenic regions in large genomes provide an ideal system for exploring the effects of dosage compensation and its associated spreading efficiency in gene translocation.

Here we generated a chromosome-level genome assembly of the migratory locust for comprehensive annotation of chromatin structure. Taking polyphenism as an example, we compared chromatin differences to explore the roles of _cis_\-regulatory elements in the behavioral changes of locusts. Furthermore, we determined allele-aware X-linked expression in XX females and investigated the role of histone modification in the loss and maintenance of balanced X-linked expression. We further generated a chromosome-level genome assembly of the desert locust and revealed the role of histone modification in X-linked gene translocation in locusts.

Results
-------

### Chromosome-level assembly of the migratory locust

We conducted genome sequencing on an adult female locust with a heterozygosity of 1.58%, using a combination of long-read sequencing, optical mapping and chromosome conformation capture based on proximity ligation (Figs. [1a,b](https://www.nature.com/articles/s41588-025-02330-y#Fig1), Supplementary Figs. [1](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and [2](https://www.nature.com/articles/s41588-025-02330-y#MOESM1), Supplementary Table [1](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). In female locusts, the third longest scaffold, identified as the X chromosome based on Hoechst staining of metaphase chromosomes, showed equal read coverage to autosomes and about twice that in male locusts (Figs. [1c,d](https://www.nature.com/articles/s41588-025-02330-y#Fig1)). The high contiguity of the LMv3.1 assembly greatly improved the completeness of the transposable elements (TEs) (Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). Fluorescent in situ hybridization (FISH) of a pericentromeric satellite DNA sequence (Fig. [1e](https://www.nature.com/articles/s41588-025-02330-y#Fig1) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)) showed fluorescent bands on one of the chromosome ends (Fig. [1f](https://www.nature.com/articles/s41588-025-02330-y#Fig1)), demonstrating the anticipated chromosome end regions. We predicted 18,127 protein-coding genes using RNA expression and homology data (Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The LMv3.1 assembly presented higher mapping rates of full-length transcripts than the LMv2.4 assembly, indicating improved coverage of RNA transcript structures (Supplementary Figs. [5](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and [6](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The improvement in genome contiguity facilitates further investigation of the epigenomic landscape in locusts.

**Fig. 1: Chromosome-level assembly of the migratory locust.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02330-y/figures/1)

**a**, Summary of sequencing datasets in this study. **b**, Heatmap showing frequency of Hi-C contacts along the migratory locust genome assembly. **c**, Identification of the X chromosome using the chromosome quotient method. The X chromosome was identified by aligning the ~30X DNA-seq data from both male and female locusts to the genome assembly. To calculate the read coverage, the chromosomes were split into 30-Mb bins. The bin numbers for chromosomes 1–11 and X are 40, 33, 27, 24, 20, 18, 16, 16, 5, 4, 3 and 27, respectively. Boxplots indicate the median (center line), the first and third quartiles (box limits) and whiskers extending to 1.5× the interquartile range. **d**, Characterization of locust chromosomes. Embryos of the migratory locust were subjected to chromosome characterization using Hoechst 33342 nucleic acid stain; _n_ = 1 independent biological replicates for demonstration purpose. **e**, The percentage distribution of satellite DNA sequences along locust chromosomes. Because centromeric satellite DNAs show a typical size greater than 100 bp, short tandem repeats with monomer lengths <100 bp were excluded from this analysis. The chromosomes were divided by partitioning into 1-Mb bins. **f**, FISH of the satellite DNA LmCentro188. The signal probe for LmCentro188 is conjugated with Alexa Fluor 488, and the chromosomes were labeled using Hoechst nucleic acid staining; _n_ = 2 independent biological replicates.

### Increased number of intronic enhancers in large-sized genes

To determine the chromatin structure, we performed cleavage under targets and tagmentation (CUT&Tag) sequencing for a panel of 13 histone modifications, assay for transposase-accessible chromatin using sequencing (ATAC-seq), transcription start site (TSS) sequencing (TSS-seq) and strand-specific RNA sequencing (RNA-seq) in brains. The hierarchical clustering of read coverage clearly reflected the noncrossreactivity of the primary antibodies, the consistency of biological replicates and the differences among sequencing methods (Fig. [2a](https://www.nature.com/articles/s41588-025-02330-y#Fig2)). We demonstrated that the abundance of histone modifications is associated with gene expression levels (Supplementary Note [2](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and Supplementary Fig. [7](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). Approximately 40% of the genomic regions were classified into the 11 ChromHMM-defined chromatin states, which are associated with distinct regulatory elements (Fig. [2b](https://www.nature.com/articles/s41588-025-02330-y#Fig2), Supplementary Note [3](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and Supplementary Figs. [8](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)–[11](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). As exemplified in Supplementary Fig. [12](https://www.nature.com/articles/s41588-025-02330-y#MOESM1), chromatin states allow for inference of gene expression. The intergenic region contributes the largest portion of chromatin states, due to its dominance in genomic composition (Supplementary Fig. [13](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The permissive and repressed chromatin states have a considerable portion of TEs, indicating that TEs contribute to host regulatory innovation and are subject to epigenetic impact (Fig. [2c](https://www.nature.com/articles/s41588-025-02330-y#Fig2), Supplementary Note [4](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and Supplementary Figs. [14](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and [15](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)).

**Fig. 2: Difference in enhancer number between large-sized and short-sized genes.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02330-y/figures/2)

**a**, Pearson correlations among sequencing assays, tissues/organs and biological replicates based on the normalized signals in 10-kb window bins. **b**, Emission patterns of the 12 chromatin states. **c**, Enrichment profiles of chromatin states overlapped with different TE superfamilies. The 16 most abundant TE superfamilies were included in this analysis. **d**, Genomic region occupancy of chromatin states in short and long introns. Short-size introns correspond to the lower 25% of data, whereas large-size introns correspond to the upper 75% of data. **e**, Correlation between intron length changes and changes in enhancer number in the brains of the migratory locust (LM), the fruit fly (DM) and the honey bee (AM). A peak was considered an active enhancer if the abundance of both H3K4me1 and H3K27ac exceeded their first quartile (Q1). Error bands represent the 95% confidence interval around the fitted smoothing curve. **f**, Correlation between gene length changes and expression changes in somatic and reproductive tissues. Transcripts per kilobase million (TPM) was used as the unit for gene expression quantification. **g**, RNA expression of genes varies with different sizes in the presence and absence of chromatin states E5 and E7. Error bars, s.d. Short-size genes correspond to the lower 25% of data, whereas large-size genes correspond to the upper 75% of data; _n_ = 4 independent biological replicates. Data are shown as mean ± s.e.m. _P_ = 0.0125 for the E5 comparison and _P_ = 0.0058 for the E7 comparison. \*_P_ < 0.05. **h**, Histone modification deposition on gene-body regions in short-size and large-size genes. Deposition abundance was determined using TPM + 1 on a log2 scale. \*False discovery rate < 0.01. Boxplots indicate the median (center line), the first and third quartiles (box limits), and whiskers extending to 1.5× the interquartile range. PTM, post-translational modification.

The intron length in locusts exceeds the values observed in other insects (Supplementary Figs. [5](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and [16](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The chromatin states E5 (enhancer) and E7 (bivalent enhancer) occupy a broader region in the large introns than in the short introns, indicating a higher number of enhancers in large introns (Fig. [2d](https://www.nature.com/articles/s41588-025-02330-y#Fig2)). The changes in intron length are correlated positively with the changes in enhancer number in the locust compared to the fruit fly and the honey bee, suggesting that the expansion of intron size is associated with an increase in enhancer number (Fig. [2e](https://www.nature.com/articles/s41588-025-02330-y#Fig2)). The changes in gene length do not have a significant impact on the gene expression changes in the migratory locust compared to the fruit fly (Fig. [2f](https://www.nature.com/articles/s41588-025-02330-y#Fig2)). Furthermore, expression values were distributed evenly among short-sized and large-sized genes in locusts (Supplementary Fig. [17](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)), demonstrating that gene structure enlargement in locusts does not significantly impact gene expression. We also found that the RNA expression of the large-sized gene associated with an intronic enhancer is significantly higher than that of the large-sized gene lacking intronic enhancers (Fig. [2g](https://www.nature.com/articles/s41588-025-02330-y#Fig2)), although it remains unclear whether these enhancers serve the role of enhancing transcription levels or adding extra control over gene expression. Compared with the short-sized genes, the large-sized genes show greater depletion of H3K4me2, H3K4me3 and H3K9ac, as well as greater deposition of H3K27me2 and H3K27me3 (Fig. [2h](https://www.nature.com/articles/s41588-025-02330-y#Fig2)), which are catalyzed primarily by polycomb repressive complexes[8](https://www.nature.com/articles/s41588-025-02330-y#ref-CR8 "Wiles, E. T. & Selker, E. U. H3K27 methylation: a promiscuous repressive chromatin mark. Curr. Opin. Genet. Dev. 43, 31–37 (2017)."). These results suggest that polycomb repressive complexes facilitate the spreading of H3K27me3, increase chromatin compaction and tether long-distance chromatin loops that promote contact between distal enhancer in the large-sized genes[9](https://www.nature.com/articles/s41588-025-02330-y#ref-CR9 "Kraft, K. et al. Polycomb-mediated genome architecture enables long-range spreading of H3K27 methylation. Proc. Natl Acad. Sci. USA 119, e2201883119 (2022)."). Taken together, the increase in enhancer numbers in large-sized genes, coupled with genome size expansion, may ensure equalized gene expression across genes of varying lengths in locusts.

### Regulation of the intronic enhancer in behavioral plasticity

Because polyphenism is clearly modulated epigenetically[10](https://www.nature.com/articles/s41588-025-02330-y#ref-CR10 "Ernst, U. R. et al. Epigenetics and locust life phase transitions. J. Exp. Biol. 218, 88–99 (2015)."), we tried to use the behavioral changes of locusts as an example of epigenetic regulation in large genomes. Principal component analysis revealed the distinguishable spatial distribution of histone modifications linked to regulatory, repressive and transcription marks between the two locust phases (Fig. [3a](https://www.nature.com/articles/s41588-025-02330-y#Fig3)). In the brain, we identified 1,069 genes that were upregulated and 485 genes that were downregulated in gregarious locusts. Gene ontology analysis suggests that the differentially expressed genes are involved in regulatory mechanisms underpinning behavioral regulation, signal transduction, synaptic plasticity and immune modulation (Supplementary Fig. [18](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The two most significantly positively correlated histone modifications—H3K4me3 and H3K27ac—at the TSS region emphasize their role in promoter regulation (Fig. [3b](https://www.nature.com/articles/s41588-025-02330-y#Fig3)). Conversely, the two most significantly negatively correlated histone modifications—H3K4me1 and H3K27me3—were situated 10 kb upstream of the TSS region, implying their involvement in a distal regulation as enhancers. Compared to the other three genomic categories, including promoters, introns and intergenic regions, H3K4me1 in the 10 kb region upstream of TSS showed the least overlap with H3K27ac (Supplementary Fig. [19](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)), indicating primed enhancers lacking transcriptional enhancement[11](https://www.nature.com/articles/s41588-025-02330-y#ref-CR11 "Heinz, S., Romanoski, C. E., Benner, C. & Glass, C. K. The selection and function of cell type-specific enhancers. Nat. Rev. Mol. Cell Biol. 16, 144–154 (2015)."). We identified the binding sites of transcription regulators associated with differential H3K4me3/H3K27ac at promoters and differential H3K4me1/H3K27me3 situated 10 kb upstream of the TSS region (Fig. [3c](https://www.nature.com/articles/s41588-025-02330-y#Fig3)). We found higher frequencies of five transcription regulators located in these two regions, including _cg_ (which establishes long-range chromatin contacts at promoters[12](https://www.nature.com/articles/s41588-025-02330-y#ref-CR12 "Vorobyeva, N. E., Krasnov, A. N., Erokhin, M., Chetverina, D. & Mazina, M. Su(Hw) interacts with Combgap to establish long-range chromatin contacts. Epigenetics Chromatin 17, 17 (2024).")), _Trl_ (a transcription activator associated with nucleosome remodeling and long-distance enhancer–promoter communication[13](https://www.nature.com/articles/s41588-025-02330-y#ref-CR13 "Chetverina, D., Erokhin, M. & Schedl, P. GAGA factor: a multifunctional pioneering chromatin protein. Cell. Mol. Life Sci. 78, 4125–4141 (2021).")), _l(3)neo38_ (an essential regulator of nervous-system development[14](https://www.nature.com/articles/s41588-025-02330-y#ref-CR14 "Reddington, J. P. et al. Lineage-resolved enhancer and promoter usage during a time course of embryogenesis. Dev. Cell 55, 648–664 (2020).")), _Sp1_ (which controls gene expression within dopaminergic neurons[15](https://www.nature.com/articles/s41588-025-02330-y#ref-CR15 "Kvetnansky, R., Sabban, E. L. & Palkovits, M. Catecholaminergic systems in stress: structural and molecular genetic approaches. Physiol. Rev. 89, 535–606 (2009).")) and _Clamp_ (a pioneer factor that increases chromatin accessibility at promoters[16](https://www.nature.com/articles/s41588-025-02330-y#ref-CR16 "Duan, J. et al. CLAMP and Zelda function together to promote Drosophila zygotic genome activation. eLife 10, e69937 (2021).")). Three of these five transcription regulators, including _Trl_, _Sp1_ and _Clamp_ were differentially expressed between the solitary and gregarious locusts. Therefore, these three transcription regulators were probably involved in regulating differentially histone modification regions in brains of solitary and gregarious locusts.

**Fig. 3: Chromatin differentiation of locust phase changes.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02330-y/figures/3)

**a**, Principal component analysis of seven histone modifications. **b**, Pearson correlation analysis between chromatin signals and the phase-related gene expression of gregarious and solitary locusts. **c**, Association of transcription factors (TFs) with differential histone modifications near promoters. Differentially expressed (DE) genes, which were positively and negatively correlated with the differential H3K4me3/H3K27ac at promoters and the differential H3K4me1/H3K27me3 situated 10 kb upstream of the promoters, respectively, were included in this analysis. TF weight is calculated based on the frequencies of nonredundant TF binding sites. Positive and negative regulation refer to the relationships between histone modification abundance and gene expression, respectively. **d**, Combinatorial chromatin signals of _Henna_. The _y_ axis was scaled separately for different sequencing methods. **e**, Quantitative 3C assays to assess the spatial interaction between EH1 and TSS1 in gregarious locusts. Control DNA represents the fragments amplified by two internal primers between two enzyme sites. Digested DNA represents random ligated genomic DNA. The ligation site represents the site where EH1 is ligated to TSS1; _n_ = 3 independent biological replicates. Data are shown as mean ± s.e.m. _P_ = 0.0131; \*_P_ < 0.05. **f**, Knockout EH1 by the CRISPR–Cas9 system in gregarious locusts. EH1+/− and EH1−/− indicate heterozygous and homozygous mutants, respectively; _n_ = 1 independent biological replicates for demonstration purpose. **g**, qPCR quantification of _Henna_ mRNA expression in wild type, heterozygous mutants and homozygous mutants. At least _n_ = 3 independent biological replicates. Data are shown as mean ± s.e.m. _P_ values for comparisons of wild type with EH1+/− and EH1−/− are 0.0004 and 0.0000, respectively. \*_P_ < 0.05. **h**, Western blot of _Henna_ protein expression; _n_ = 3 independent biological replicates. Data are shown as mean ± s.e.m.; _P_ = 0.017, \*_P_ < 0.05. **i**, Arena behavioral assays for quantifying phase-related behavior. Independent biological replicates; _n_ = 15 (EH1−/−), _n_ = 7 (EH1+/−) and _n_ = 20 (wild type). A _P_greg (probabilistic metric of gregariousness) close to 0 indicates typical gregarious behavior; _P_ = 0.0014 for EH1−/− and _P_ = 0.0414 for EH1+/−. UTR, untranslated region.

[Source data](https://www.nature.com/articles/s41588-025-02330-y#MOESM3)

Because the enlargement of genome size results in the presence of long introns[17](https://www.nature.com/articles/s41588-025-02330-y#ref-CR17 "Liu, Q., Jiang, F., Zhang, J., Li, X. & Kang, L. Transcription initiation of distant core promoters in a large-sized genome of an insect. BMC Biol. 19, 62 (2021)."), we sought to explore whether the intronic _cis_\-regulatory elements are involved in the regulation of behavioral plasticity of locusts. We used the gene _Henna_ as an example because we have shown previously that it is the most critical gene in determining behavioral plasticity in dopamine biosynthesis pathway[2](https://www.nature.com/articles/s41588-025-02330-y#ref-CR2 "Ma, Z., Guo, W., Guo, X., Wang, X. & Kang, L. Modulation of behavioral phase changes of the migratory locust by the catecholamine metabolic pathway. Proc. Natl Acad. Sci. USA 108, 3882–3887 (2011)."),[3](https://www.nature.com/articles/s41588-025-02330-y#ref-CR3 "Yang, M. et al. MicroRNA-133 inhibits behavioral aggregation by controlling dopamine synthesis in locusts. PLoS Genet. 10, e1004206 (2014)."). The dominant TSS of _Henna_ in brains, as identified by TSS-seq, was TSS1, whose chromatic pattern was correlated with differential RNA-seq expression between solitary and gregarious locusts (Fig. [3d](https://www.nature.com/articles/s41588-025-02330-y#Fig3)). In gregarious locusts, we observed upregulated deposition of ATAC, H3K4me3 and H3K27ac in the TSS1 region, as well as H3K36me in the gene-body region. Notably, we found an enhancer, EH1, which was decorated with high H3K4me1 and low H3K4me3 signals, in the 10.3-kb intronic region upstream of the first _Henna_ coding exon. The significantly higher interaction frequency in quantitative chromosome conformation capture (3C) DNA confirmed the close three-dimensional physical proximity between TSS1 and EH1 (Fig. [3e](https://www.nature.com/articles/s41588-025-02330-y#Fig3)). The EH1-knockout mutants in gregarious locusts resulted in a significant reduction of _Henna_ expression at mRNA and protein levels (Figs. [3f,g,h](https://www.nature.com/articles/s41588-025-02330-y#Fig3)). Consequently, the EH1-knockout mutants of gregarious locusts showed a significant behavioral shift toward solitary behavioral traits in behavioral arena assays (Fig. [3i](https://www.nature.com/articles/s41588-025-02330-y#Fig3)). Indeed, our previous studies have demonstrated that RNAi silencing of _Henna_ expression induces a behavioral change from solitary to gregarious phases[2](https://www.nature.com/articles/s41588-025-02330-y#ref-CR2 "Ma, Z., Guo, W., Guo, X., Wang, X. & Kang, L. Modulation of behavioral phase changes of the migratory locust by the catecholamine metabolic pathway. Proc. Natl Acad. Sci. USA 108, 3882–3887 (2011)."). Therefore, the knockout of EH1 in gregarious locusts resulted in altered behavior—a phenotype copied by silencing _Henna_ expression. Collectively, the enhancers located in long intron interact with promoters to regulate behavioral plasticity of locusts.

### Chromatin changes of X chromosome during meiotic silencing

In the brain and leg muscles, we found a strong positive diagonal correlation between male and female gene expression on the X chromosome and autosomes (Fig. [4a](https://www.nature.com/articles/s41588-025-02330-y#Fig4)), indicating a comparable expression between X chromosome and autosomes in the two sexes. Unlike autosomes, the X chromosome exhibited higher gene expression in ovaries compared to testes. So, the gene expression between the sexes is re-equilibrated in soma but not in gonads.

**Fig. 4: Global interaction and chromatin change of the X chromosome during meiotic silencing.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02330-y/figures/4)

**a**, Correlation of gene expression in brains, leg muscles and gonads. Error bands, 95% confidence interval of the linear model fit. _P_ values < 2.2 × 10−16, Pearson correlation test. **b**, Expression ratios between X-linked and autosomal genes in gonads under different minimum TPM cutoffs. Error bands, 95% confidence interval around the fitted smoothing curve. **c**, Pearson correlations of Hi-C interaction frequency matrices of chr. 1 and chr. X at 500-kb resolution. The intensity of each pixel represents the normalized frequency of interaction between a pair of genomic loci. Arrowheads indicate parallel lines along the main diagonal of the interaction map. **d**, A log–log plot of interaction frequency decay with distance in testes. We assumed independence between the X chromosome and the autosomes. Normalized Hi-C data binned at 25-kb scale were converted to interaction frequencies. The _x_ axis shows the log10 genomic distance, and the _y_ axis shows the median log10 Hi-C interaction frequency for any given genomic distance. **e**, Relative enrichments of _trans_ contacts by calculating the ratio of the number of _trans_\- over _cis_\-mapping read pairs for each chromosome. **f**, Condensation of the X chromosome in a meiotic cell of spermatocytes in testes. A series of 24 DNA probes (the single-copy X-linked gene _Pex5_) was selected randomly. Nuclei were stained with Hoechst 33342; _n_ = 2 independent biological replicates. The position of the X chromosome is indicated by arrowheads. **g**, Read abundances of H4K16ac and H4K20me3 are shown in 1-Mb bin units. _P_ = 0.0000, \*_P_ < 0.05. **h**, The distribution and abundance of H4K16ac in testicular follicles of adult locusts. Nuclei were stained with Hoechst 33342 staining; _n_ = 2 independent biological replicates. **i**, Association of Hi-C interaction frequency with gene expression and H4K16ac signals in chr. 1 and chr. X. The A and B compartments, determined by the first principal component (PC1), reflected the transcription and H4K16ac state on a linear genomic scale.

The X/A ratio in testes held near 0.5—a key indicator of meiotic sex chromosome inactivation (Fig. [4b](https://www.nature.com/articles/s41588-025-02330-y#Fig4)). The X chromosome showed increased Hi-C signals/interaction frequencies, as evidenced by the presence of parallel lines along the main diagonal in the interaction map (Fig. [4c](https://www.nature.com/articles/s41588-025-02330-y#Fig4)). The increased interaction frequencies are unique to the X chromosome and not observed in the other autosomes (Supplementary Fig. [20](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The log–log plot of interaction frequencies against distance showed that the X chromosome deviated gradually from the autosomes as distance increased (Fig. [4d](https://www.nature.com/articles/s41588-025-02330-y#Fig4)). Compared to autosomes, the slower interaction frequency decay in the X chromosome indicates the relatively higher interaction frequencies at long-range distances, from 10 Mb to 60 Mb. Consequently, the inactivated X chromosome is more compacted in higher-order chromatin structures, consistent with the observation that active chromatin regions tend to interact[18](https://www.nature.com/articles/s41588-025-02330-y#ref-CR18 "Sexton, T. et al. Three-dimensional folding and functional organization principles of the Drosophila genome. Cell 148, 458–472 (2012)."). We quantified the pairwise chromosomal interactions within (intrachromosomal _cis_ contacts) and between (interchromosomal _trans_ contacts) chromosomes to measure spatial distance between the X chromosome and autosomes. As expected, the intrachromosomal _cis_ contacts are more prevalent than the interchromosomal _trans_ contacts (Supplementary Fig. [21](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). Furthermore, the interchromosomal _trans_ contacts are less frequent, specifically on the X chromosome (Fig. [4e](https://www.nature.com/articles/s41588-025-02330-y#Fig4)), indicating a more distal spatial position of the X chromosome compared to autosomes within chromosomal territories. The DNA FISH assay of a single-copy X-linked gene (_Pex5_) reveals that the X chromosome shows greater condensation than autosomes (Fig. [4f](https://www.nature.com/articles/s41588-025-02330-y#Fig4)). These results suggests that the inactivated X chromosome is farther from the transcription-associated hubs at nuclear speckles, which are connected closely among activated autosomes[19](https://www.nature.com/articles/s41588-025-02330-y#ref-CR19 "Monahan, K., Horta, A. & Lomvardas, S. LHX2- and LDB1-mediated trans interactions regulate olfactory receptor choice. Nature 565, 448–453 (2019).").

The global reduction of H4K16ac signals of X-linked genes in testes is in accordance with the loss of expression balance between male and female locusts, indicating chromatin suppression of gene expression on the X chromosome in male locusts (Fig. [4g](https://www.nature.com/articles/s41588-025-02330-y#Fig4)). The early germ cells displayed an enrichment of H4K16ac (Fig. [4h](https://www.nature.com/articles/s41588-025-02330-y#Fig4)), whereas the spermatocytes, undergoing meiotic division, exhibited a depletion of H4K16ac, corresponding to the global reduction of H4K16ac signals in X-linked genes in the testes[20](https://www.nature.com/articles/s41588-025-02330-y#ref-CR20 "Arafat, E. A., El-Samad, L. M., Moussian, B. & Hassan, M. A. Insights into spermatogenesis in the migratory locust, Locusta migratoria (Linnaeus, 1758)(Orthoptera: Acrididae), following histological and ultrastructural features of the testis. Micron 172, 103502 (2023)."). The binary classification of A (active transcriptional state) and B (inactive transcriptional state) compartments showed that different H4K16ac distribution was correlated strongly with transcription activity (Fig. [4i](https://www.nature.com/articles/s41588-025-02330-y#Fig4) and Supplementary Fig. [22](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The number of high expressed genes in compartment A in the X chromosome (0.572 Mb−1) was lower than that in autosomes (0.915 Mb−1). These results show that the loss of expression balance of the X chromosome in testes might be correlated with interchromosomal spatial distance, compartment organization and H4K16ac depletion. Therefore, the relatively remote spatial position of the gigantic X chromosome, decorated with distinct chromatin structures, results in the formation of segregated chromosome territories compared to autosomes during meiotic silencing.

### Dosage compensation of the X chromosome in soma

In brains and leg muscles, X-linked gene expression in female and male locusts was equal to that of the autosomal genes, indicating complete dosage compensation in male locusts (Fig. [5a](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). X chromosome expression in female locusts can be achieved through biallelic expression of both alleles or by random inactivation of one allele with hyperactivation of the other in animals[21](https://www.nature.com/articles/s41588-025-02330-y#ref-CR21 "Disteche, C. M. Dosage compensation of the sex chromosomes. Annu Rev. Genet. 46, 537–560 (2012)."). The transcriptome analysis of heterozygosity in a female individual revealed biallelic expression of X-linked genes in female locusts (Fig. [5b](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). To eliminate potential influences from A-to-I RNA editing, the results, even after removing adenine sites, robustly supported the biallelic expression of X-linked genes in female soma.

**Fig. 5: Balanced expression of the X chromosome in soma.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02330-y/figures/5)

**a**, Expression ratios between X-linked and autosomal genes in brains and muscles under different minimum TPM cutoffs for gene inclusion. Error bands, 95% confidence interval around the fitted smoothing curve. **b**, Allelic X chromosome expression in a female. To ensure accurate identification of heterozygous alleles, the biallelic heterozygosity sites were determined using DNA-seq for the same individual locust that was subjected to RNA-seq. **c**, Genomic prevalence of the histone modifications H4K20me1, H4K16ac, H3K27me3 and H3K9me3. The genomic prevalence was determined by the average portion of the reads located in the genic and intergenic regions. **d**, Density distribution of the ratios (female/male) of histone modification abundances in 200-kb bins between the two sexes in brains. **e**, Enrichment profiles of histone modification of H4K20me1, H4K16ac, H3K27me3 and H3K9me3 across genic region. The theoretical estimation in males was inferred assuming no dosage changes in females. **f**, Density distribution of the ratios (female/male) of histone modification abundances between the two sexes in different genomic regions in brains of the migratory locust. The different genomic regions were classified based on the distance to the nearest genes. **g**, Immunofluorescence staining of H4K16ac in brains. Nuclei were stained with Hoechst 33342 staining; _n_ = 2 independent biological replicates. **h**, Density distribution of the ratios (female/male) of histone modification abundances between the two sexes in the fruit fly. The average chromatin immunoprecipitation sequencing signals in 2-kb bins were compared between female and male locusts. **i**, Density distribution of the ratios (female/male) of histone modification abundances between the two sexes in different genomic regions in 2–4 h embryos of the fruit fly. The different genomic regions were classified based on the distance to the nearest genes.

We then profiled the genomic distributions of H4K20me1, H4K16ac, H3K27me3 and H3K9me3 to determine whether these chromatin modifications are involved with dosage compensation in locusts. H4K16ac was more prevalent in the genic region compared to H4K20me1, H3K27me3 and H3K9me3 (Fig. [5c](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). In autosomes, the equal genic and intergenic signals between female and male locusts for these four chromatin marks corresponded to the balanced autosomal gene expression in brains (Fig. [5d](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). However, within the genic region of the X chromosome, only the density of female/male enrichment ratios of H4K16ac signals centered around 1, consistent with its role in complete dosage compensation. All four histone modifications showed similar levels between autosomal and X-linked genes across genic regions in females (Fig. [5e](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). To account for the difference in chromosome numbers between females and males, we estimated male data (the abundance of histone modification in male brains) theoretically assuming no dosage changes in females. As expected, the H4K16ac levels were comparable between the autosomal and X-linked genes in males, aligning with the well-recognized role of H4K16ac in establishing dosage compensation in heterogametic organisms[22](https://www.nature.com/articles/s41588-025-02330-y#ref-CR22 "Gelbart, M. E., Larschan, E., Peng, S., Park, P. J. & Kuroda, M. I. Drosophila MSL complex globally acetylates H4K16 on the male X chromosome for dosage compensation. Nat. Struct. Mol. Biol. 16, 825–832 (2009)."). In fact, the enrichment levels of H4K16ac on X-linked genes in males were higher than those theoretically estimated for male data. Different from the widespread distribution of H4K16ac across the entire gene body, H4K20me1 localized mainly to the 5′ end of the gene body. However, the H4K20me1 level of X-linked genes in males was more depleted than that of the theoretically estimated male data. In fruit flies, the absence of Sex-lethal (SXL) expression in males permits MSL complex assembly, leading to H4K16ac enrichment on the X chromosome for dosage compensation[23](https://www.nature.com/articles/s41588-025-02330-y#ref-CR23 "Samata, M. & Akhtar, A. Dosage compensation of the X chromosome: a complex epigenetic assignment involving chromatin regulators and long noncoding RNAs. Annu. Rev. Biochem. 87, 323–350 (2018)."). Despite the close association between H4K20me1 and MSL complex, we found no significant differences in RNA and protein expression of SXL and MSL subunits between females and males, indicating an SXL-independent initiation of dosage compensation in locusts (Supplementary Figs. [23](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and [24](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). Overall, the depletion of H4K20me1 and the enrichment of H4K16ac suggest that they are connected functionally with maintaining balanced expression of the X chromosome in the somatic organs of male locusts.

Different from H4K20me1, H3K27me3 and H3K9me3, levels of H4K16ac decreased with the increase of the distance to the nearest gene in the intergenic region on the X chromosome (Fig. [5f](https://www.nature.com/articles/s41588-025-02330-y#Fig5)), indicating a distance-dependent dosage effect of H4K16ac and partial dosage compensation. Different from the fruit fly[24](https://www.nature.com/articles/s41588-025-02330-y#ref-CR24 "Keller Valsecchi, C. I., Marois, E., Basilicata, M. F., Georgiev, P. & Akhtar, A. Life Sci. Alliance 4, e202000996 (2021)."), we did not detect a dominant immunofluorescence signal of H4K16ac across the entire male X genome (Fig. [5g](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). To further confirm the uniqueness of distance-dependent H4K16ac enrichment in the large genome of locusts, we analyzed H4K16ac profiling data from the fruit fly[25](https://www.nature.com/articles/s41588-025-02330-y#ref-CR25 "Rieder, L. E., Jordan, W. T. 3rd & Larschan, E. N. Targeting of the dosage-compensated male X-chromosome during early Drosophila development. Cell Rep. 29, 4268–4275 (2019)."). In contrast to the locust genome, we observed complete dosage compensation in both genic and intergenic regions of the compact genome of the fruit fly (Fig. [5h](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). As expected, the distance-dependent H4K16ac enrichment was not observed on the X chromosome of the fruit fly (Fig. [5i](https://www.nature.com/articles/s41588-025-02330-y#Fig5)). Therefore, the dosage effects of H4K16ac, initiated from genic region, faded gradually during subsequent spreading into intergenic regions, which constrain the spreading of dosage effects in locusts.

### Dosage compensation on translocated X-linked genes

By examining the conservation of X-linked genes across insects at order level, we found that the gene configuration of the locust X chromosome is the result of extensive interchromosomal gene exchanges through drastic genome rearrangement (Supplementary Note [6](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and Supplementary Figs. [25](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)–[27](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). To further investigate the formation of X-linked genes of the migratory locust within Orthoptera, we generated a chromosome-level genome of the desert locust (_Schistocerca gregaria_ (SG), Caelifera, Acrididae). We determined the pairwise chromosome associations of the two locusts, the pygmy grasshopper (_Zhengitettix transpicula_ (ZT), Caelifera, Tetrigidae) and the pygmy mole cricket (_Xya riparia_ (XR), Caelifera, Tridactyloidea), which belongs to a basal taxon of Caelifera. The chromosomes of the migratory locust are identical to those of the desert locust but exhibit one-to-several or one-to-zero correspondence with those of the pygmy grasshopper and the pygmy mole cricket, demonstrating that genome size expansion accompanies large-scale chromosome rearrangement during the early diversification of Caelifera species (Fig. [6a](https://www.nature.com/articles/s41588-025-02330-y#Fig6) and Supplementary Figs. [28](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)–[30](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The X-linked genes of the migratory locust have largely persisted on the X chromosome since the emergence of Caelifera (Fig. [6b](https://www.nature.com/articles/s41588-025-02330-y#Fig6)), although it cannot be concluded whether the X chromosome of the pygmy mole cricket represents the ancestral X chromosome of Caelifera or has undergone a chromosome fusion event between an autosome and a sex chromosome. To investigate the chromosomal rearrangement of X-linked gene of the migratory locust, the one-to-one orthologs among the four species were divided into the five gene categories: ancient autosome (LMaSGaXRaZTa), ancient X (LMxSGxXRxZTx), ancient AtoX (translocated from Autosomes to X chromosome before the divergence of two locust species, LMxSGxXRaZTa), ancient XtoA (LMaSGaXRxZTx) and recent AtoX (LMxSGaXRaZTa). Compared with the ancient AtoX and ancient X categories, the lower ratios of female/male in H4K20me1 signals showed insufficient depletion of H4K20me1 in the recent AtoX category. Furthermore, the recent AtoX category showed a higher ratio of female/male in H4K16ac signals than the ancient AtoX and ancient X categories (Fig. [6c](https://www.nature.com/articles/s41588-025-02330-y#Fig6)). This suggests an incomplete recovery of dosage compensation of recently translocated X-linked genes in the migratory locust.

**Fig. 6: Dosage compensation turnover on X-linked gene derived from autosomes.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02330-y/figures/6)

**a**, Pairwise dotplots showing the significance of chromosome–chromosome associations among the migratory locust, the desert locust, the pygmy grasshopper and the pygmy mole cricket. **b**, One-to-one ortholog linking among the chromosomes of four orthopteran species. Vertical lines connect orthologs across the four species. Only connections between chromosome pairs with significantly associations are shown. **c**, Ratios (female/male) of genic H4K16ac and H4K20me1 signals between the two sexes. Reads per kilobase million values were used to determine the histone modification signals. The one-to-one orthologs of the migratory locust were divided into the six gene categories: ancient autosome (LMaSGaXRaZTa, _n_ = 4,661), ancient X (LMxSGxXRxZTx, _n_ = 314), ancient AtoX (LMxSGxXRaZTa, _n_ = 27), ancient XtoA (LMaSGaXRxZTx, _n_ = 22) and recent AtoX (LMxSGaXRaZTa, _n_ = 6); n, number of one-to-one orthologs. The recent XtoA category was not included due to the absence of any ortholog. Boxplots indicate the median (center line), the first and third quartiles (box limits), and whiskers extending to 1.5× the interquartile range. The _P_ values for comparisons of recent AtoX with ancient X, ancient AtoX, ancient XtoA and ancient autosomes were (0.0151, 0.0011), (0.0403, 0.0017), (0.0023, 0.0017) and (0.0017, 0.0011) for H4K16ac and H4K20me1, respectively. \*_P_ < 0.05. **d**, d_N_/d_S_ comparison in the five gene categories. d_N_/d_S_, the ratio of nonsynonymous substitutions per nonsynonymous site (d_N_) to synonymous substitutions per synonymous site (d_S_). Boxplots indicate the median (center line), the first and third quartiles (box limits), and whiskers extending to 1.5× the interquartile range. The _P_ values for comparisons of ancient AtoX with ancient X, recent AtoX, ancient XtoA and ancient autosomes were 2 × 10−6, 0.0359, 0.0002 and 2 × 10−6, respectively; \*_P_ < 0.05. **e**, Heatmap showing abundance of histone modification signals for the recent AtoX, ancient XtoA and ancient AtoX categories.

Compared to the ancient X and the two autosomal categories, the ancient AtoX category, but not the recent AtoX category, had significantly higher d_N_/d_S_ ratios (Fig. [6d](https://www.nature.com/articles/s41588-025-02330-y#Fig6)), suggesting that adaptive evolution is more likely to have affected the ancient AtoX category. Accordingly, the ancient AtoX category displayed a broader range and higher levels of histone modification than the recent AtoX category (Fig. [6e](https://www.nature.com/articles/s41588-025-02330-y#Fig6)). This implies that X-linked translocations have the potential to accumulate sequence divergence or to maintain mutations that cannot be effectively eliminated by recombination, ultimately contributing to gene function innovation over time. In the recent AtoX category, several genes, including _Oat_ and _msps_, were reported previously to be associated with functions that harm males or benefit females[26](https://www.nature.com/articles/s41588-025-02330-y#ref-CR26 "Levillain, O., Diaz, J. J., Blanchard, O. & Dechaud, H. Testosterone down-regulates ornithine aminotransferase gene and up-regulates arginase II and ornithine decarboxylase genes for polyamines synthesis in the murine kidney. Endocrinology 146, 950–959 (2005)."),[27](https://www.nature.com/articles/s41588-025-02330-y#ref-CR27 "Lu, W., Lakonishok, M. & Gelfand, V. I. The dynamic duo of microtubule polymerase mini spindles/XMAP215 and cytoplasmic dynein is essential for maintaining Drosophila oocyte fate. Proc. Natl Acad. Sci. USA 120, e2303376120 (2023)."). Therefore, sexually antagonistic selection promotes the relocation of female-related genes from autosomes to X chromosome in locusts (Supplementary Note [7](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). Consequently, these genes can be repressed by meiotic sex chromosome inactivation, facilitating to mitigate female antagonism and to increase male fitness during spermatogenesis. Therefore, the translocated X-linked genes in the migratory locust are likely to be convergently recruited to the X chromosome and subject to relocation of dosage compensation due to the need to suppress their functional roles favoring females during spermatogenesis. Taken together, the recent AtoX category showed distinct differences in histone modification, suggesting a lag in the relocation of dosage compensation for the X-linked genes that were recently translocated autosomes in the migratory locust.

Discussion
----------

In this study, an epigenomic map, based on a high-quality genome, allowed us to accurately identify chromatin modifications of spacious noncoding regions of locusts in an unprecedented manner due to improved assembly completeness in intronic and intergenic regions. The increased number of enhancers in expanded intronic regions and the distance-dependent H4K16ac dosage compensation in expanded intergenic regions reveal distinct epigenomic characteristics in the dark matter of large-sized genomes.

We observed that the phase-related histone modification differences decorating promoters and enhancers were correlated with differential transcription between solitary and gregarious phases, underscoring the significant influences of chromatin regulation on determining alternative developmental programming in locusts[2](https://www.nature.com/articles/s41588-025-02330-y#ref-CR2 "Ma, Z., Guo, W., Guo, X., Wang, X. & Kang, L. Modulation of behavioral phase changes of the migratory locust by the catecholamine metabolic pathway. Proc. Natl Acad. Sci. USA 108, 3882–3887 (2011)."),[28](https://www.nature.com/articles/s41588-025-02330-y#ref-CR28 "Yang, P., Hou, L., Wang, X. & Kang, L. Core transcriptional signatures of phase change in the migratory locust. Protein Cell 10, 883–901 (2019)."). Indeed, the involvement of H3K4me3 and H3K27ac on promoters has also been confirmed previously in caste determination of honey bees and carpenter ants, suggesting that these histone modifications play a conserved role in the phenotypic plasticity of a wide range of insect species[29](https://www.nature.com/articles/s41588-025-02330-y#ref-CR29 "Simola, D. F. et al. A chromatin link to caste identity in the carpenter ant Camponotus floridanus. Genome Res. 23, 486–496 (2013)."),[30](https://www.nature.com/articles/s41588-025-02330-y#ref-CR30 "Wojciechowski, M. et al. Phenotypically distinct female castes in honey bees are defined by alternative chromatin states during larval development. Genome Res. 28, 1532–1542 (2018)."). We found that the _cis_\-regulatory element EH1 functions specifically as an enhancer to regulate the behavioral plasticity of locusts. We also identified an EH1-like sequence in the homologous intron of _Henna_ in the desert locust (Supplementary Fig. [31](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)), but not in the two Caeliferan insects or in the three insects with small-sized genomes. Since _Henna_ has been reported to be involved in the regulation of phenotypic plasticity in the honey bee[31](https://www.nature.com/articles/s41588-025-02330-y#ref-CR31 "Zayed, A., Naeger, N. L., Rodriguez-Zas, S. L. & Robinson, G. E. Common and novel transcriptional routes to behavioral maturation in worker and male honey bees. Genes Brain Behav. 11, 253–261 (2012)."), we further checked its epigenomic data but did not find a positionally homologous enhancer of _Henna_[32](https://www.nature.com/articles/s41588-025-02330-y#ref-CR32 "Lowe, R., Wojciechowski, M., Ellis, N. & Hurd, P. J. Chromatin accessibility-based characterisation of brain gene regulatory networks in three distinct honey bee polyphenisms. Nucleic Acids Res. 50, 11550–11562 (2022)."). These results indicate that EH1 may be present only in locust genomes and may contribute solely to the specialized regulatory function of behavioral plasticity in locusts. Therefore, these findings reveal that, accompanying genome size expansion, an increasing number of enhancers have been recruited in large-sized genes in locusts.

Different from the fruit fly, where the male X chromosome exhibits predominant immunostaining intensity of H4K16ac, we did not observe this phenomenon in male locusts[33](https://www.nature.com/articles/s41588-025-02330-y#ref-CR33 "Turner, B. M., Birley, A. J. & Lavender, J. Histone H4 isoforms acetylated at specific lysine residues define individual chromosomes and chromatin domains in Drosophila polytene nuclei. Cell 69, 375–384 (1992)."). Despite H4K16ac not exhibiting preferential enrichment on the male X chromosome in mosquitoes, a common characteristic of dosage compensation mechanisms is the elevated H4K16ac levels on dosage-compensated sex chromosomes[24](https://www.nature.com/articles/s41588-025-02330-y#ref-CR24 "Keller Valsecchi, C. I., Marois, E., Basilicata, M. F., Georgiev, P. & Akhtar, A. Life Sci. Alliance 4, e202000996 (2021)."). The intergenic regions occupy a broader area in the locust X chromosome than in the fruit fly X chromosome. TEs, which make up 71.1% of the locust X chromosome, may play a role in assembling and organizing constitutive heterochromatin in intergenic regions by providing docking sites for protein complexes associated with repressive histone modifications or through the presence of hotspots for TE insertions[34](https://www.nature.com/articles/s41588-025-02330-y#ref-CR34 "Marsano, R. M. & Dimitri, P. Constitutive heterochromatin in eukaryotic genomes: a mine of transposable elements. Cells 11, 761 (2022)."),[35](https://www.nature.com/articles/s41588-025-02330-y#ref-CR35 "Lawson, H. A., Liang, Y. & Wang, T. Transposable elements in mammalian chromatin organization. Nat. Rev. Genet. 24, 712–723 (2023)."). Therefore, we found a mosaic pattern of immunostaining intensity, similar to that on autosomes, rather than a prevailing intensity along the X chromosome. These results are also supported by the distance-based decoration of H4K16ac outside genic regions and the lack of H4K16ac enrichment in intergenic regions in locusts. Therefore, the distance-dependent H4K16ac enrichments in intergenic regions demonstrate peculiarly large genome effects on dosage compensation.

Our results ruled out the long-term conservation of X-linked genes due to extensive interchromosomal gene exchanges through drastic genome rearrangement (Supplementary Note [6](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and Supplementary Figs. [25](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)–[27](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). A recent study published another chromosome-level genome assembly of the migratory locust[36](https://www.nature.com/articles/s41588-025-02330-y#ref-CR36 "Li, X., Mank, J. E. & Ban, L. The grasshopper genome reveals long-term gene content conservation of the X Chromosome and temporal variation in X Chromosome evolution. Genome Res. 34, 997–1007 (2024)."). However, this present study presents different conclusions from the latter study regarding the conservation of X-linked genes. The discrepancy is because of the use of different criteria to define conservation levels. The numbers of X-linked genes vary greatly between the migratory locust (1,985 and 12 pairs of chromosomes) and the fruit fly (80 and four pairs of chromosomes). They identified only 35 X-linked homologs, which may provide limited evidence for concluding that the X chromosome has maintained long-term gene content conservation between these two insect species. Indeed, our results show that only ten X-linked orthologs are present on the sex chromosomes of all the four insect orders (Supplementary Figs. [26](https://www.nature.com/articles/s41588-025-02330-y#MOESM1) and [27](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). These findings suggest that, throughout insect evolution, there has been little evidence of strong selective pressure to preserve a substantial proportion of X-linked genes on the X chromosome. The incomplete recovery of H4K16ac-mediated dosage compensation and insufficient depletion of H4K20me1, as well as restricted histone modification decoration in the recent AtoX category indicate a lagging relocation of dosage compensation on recently translocated X-linked genes. This contrasts with the re-established compensation and abundant histone decoration in the ancient AtoX category, indicating that the dosage compensation effects of flanking pre-existing X-linked genes cannot spread efficiently to the newly translocated X-linked genes. The spreading of H4K16ac, catalyzed by the MSL complex, to the flanking intergenic and genic regions lacking MSL complex binding is important for dosage compensation[22](https://www.nature.com/articles/s41588-025-02330-y#ref-CR22 "Gelbart, M. E., Larschan, E., Peng, S., Park, P. J. & Kuroda, M. I. Drosophila MSL complex globally acetylates H4K16 on the male X chromosome for dosage compensation. Nat. Struct. Mol. Biol. 16, 825–832 (2009)."). Since the MSL complex loads at chromatin entry sites, mainly in introns and nearby 3-kb genic regions[7](https://www.nature.com/articles/s41588-025-02330-y#ref-CR7 "Straub, T., Grimaud, C., Gilfillan, G. D., Mitterweger, A. & Becker, P. B. The chromosomal high-affinity binding sites for the Drosophila dosage compensation complex. PLoS Genet. 4, e1000302 (2008)."), the limited efficiency of H4K16ac spread may require a greater number of chromatin entry sites in the locust genome compared to smaller genomes. Consequently, the newly translocated X-linked genes in the locust genome may not be able to quickly establish recognition by the MSL complex. Therefore, distance-dependent dosage compensation in intergenic region results in a lagging relocation of dosage compensation to X-linked genes recently translocated from autosomes in large genomes.

Methods
-------

### Ethics declarations

No ethical approval was required for the use of locusts in this study.

### Insects

The locust colonies were reared under a 14 h/10 h light/dark photo regimen at 30 °C and were fed fresh wheat seedlings and wheat bran. An adult female individual was dissected to remove intestinal tracts, which were frozen immediately in liquid nitrogen and stored at −80 °C until DNA extraction.

### Chromosome staining and estimation of chromosome size

To obtain embryos for chromosome staining analysis, locust eggs were dissected from egg pods to isolate embryos. The embryos were immersed in 0.04% colchicine for 90 min followed immediately by hypotonic treatment for 30 min, and were then immersed in 3:1 ethanol–acetic acid for 90 min. Afterward, the embryos were immersed in 80% acetic acid for 2 min and dispersed and suspended gently for 3 min. The cell suspension was aspirated into a micropipette and dropped onto a clean slide at 60 °C to dry the solution. The slide was stained with Hoechst 33342 (Thermo Fisher Scientific) for 5 min and visualized using a confocal microscopy system (Zeiss LSM 710). We measured the chromosome bivalents from one female embryonic cell at metaphase of mitosis. All the bivalents were well separated. The pixels of each chromosome were measured by the Image J software and arranged in order of decreasing value.

### Genomic library construction and DNA sequencing

The DNA samples used for genomic sequencing were obtained using a sodium dodecyl sulfate method. Initially, the tissues were ground carefully with liquid nitrogen, and the cell nucleus was cleaned subsequently using a modified homogenization buffer. Following this step, the residual material was washed with 1× PBS buffer (Invitrogen) and the cell nucleus was lysed using a total lysis buffer. Subsequently, RNA and protein were eliminated using RNase A (Qiagen) and protease K (Qiagen), respectively. Finally, the DNA was eluted using an elution buffer (Qiagen) after phenol/chloroform extraction. For the migratory locust, two libraries with insert sizes of 20 kb and 40 kb were constructed separately using SMRTbell Template Preparation 1.0 using BluePippin size-selection (Sage Science), and were subsequently sequenced using on a PacBio Sequel instrument with the Sequel Sequencing Kit v.2.1. To obtain the optical mapping data, genomic DNA was labeled using the Bionano Prep DNA Labeling Kit-DLS, and loaded into a nanochannel array on a Saphyr instrument (Bionano Genomics). For the desert locust, DNA extraction was constructed for Nanopore libraries and then sequenced using a PromethION platform. The sequencing libraries were constructed using the ligation sequencing kit (SQK-LSK109) and loaded onto FLO-PRO002 flow cells. The genomic DNA was also used to construct a paired-end library using Nextera XT DNA Library Prep Kit (Illumina). This short-read library was sequenced on an Illumina HiSeq X Ten platform.

### DNA labeling for optical mapping

Direct labeling was performed using the Direct Labeling and Staining Kit (Bionano Genomics, catalogue number 80005) according to the manufacturer’s manual. In brief, high-molecular-weight DNA was incubated at 37 °C with DLE labeling mix. Following proteinase K digestion and cleanup of the unincorporated DL-Green label, the labeled DNA was prestained, homogenized and quantified using on a Qubit Fluorometer (Thermo Fisher). The labeled sample was loaded into a nanochannel array of flow cells, imaged and digitized in a Saphyr instrument (Bionano Genomics) according to the manufacturer’s instructions.

### Genome assembly

GenomeScope v.1.0.0 was used to conduct a K-mer frequency survey of genome composition based on 21-mer frequency distribution to estimate genome size and heterozygosity. For the migratory locust, The raw PacBio reads were corrected and trimmed using Canu v.1.7[37](https://www.nature.com/articles/s41588-025-02330-y#ref-CR37 "Koren, S. et al. Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. Genome Res. 27, 722–736 (2017)."), and the contig-level assembly were performed using FALCON v.2.0.4 followed by the FALCON-Unzip v.0.4.0 module[38](https://www.nature.com/articles/s41588-025-02330-y#ref-CR38 "Chin, C. S. et al. Phased diploid genome assembly with single-molecule real-time sequencing. Nat. Methods 13, 1050–1054 (2016)."). The remaining alternative haplotypes from the initial primary contigs were removed using Purge Haplotigs v.1.0.0, and haplotype switch errors were eliminated using the FALCON-Phase v.0.1.0[39](https://www.nature.com/articles/s41588-025-02330-y#ref-CR39 "Kronenberg, Z. N. et al. Extended haplotype-phasing of long-read de novo genome assemblies using Hi-C. Nat. Commun. 12, 1935 (2021)."). All PacBio reads were aligned to the resulting assemblies using BLASR v.5.3.3 and then the consensus sequences were called using the Arrow program in SMRT link v.7.0. The PacBio-corrected assemblies were further improved by two rounds of polishing with Pilon v.1.23 using paired-end Illumina data[40](https://www.nature.com/articles/s41588-025-02330-y#ref-CR40 "Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS ONE 9, e112963 (2014)."). Data visualization, de novo assembly and hybrid scaffolding of the optical map data with a minimum size of 150 kb were performed using the Bionano Access v.1.2.1 and Bionano Solve v.3.2.1 software. Gaps in the hybrid-scaffolded assembly were closed using PBJelly from PBSuite v.15.8.24 using PacBio and Nanopore read datasets[41](https://www.nature.com/articles/s41588-025-02330-y#ref-CR41 "English, A. C. et al. Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology. PLoS ONE 7, e47768 (2012)."). We subsequently reperformed error correction procedures to polish the sequences in the gap regions. For the desert locust, de novo genome assemblies were generated by NextDenovo v.2.4.0 and NextPolish v.1.3.1 using long-read Nanopore sequencing.

### Scaffolding of a chromosome-level genome assembly

For Hi-C experiments, tissues were cross-linked with 1% fresh formaldehyde and then quenched with 0.2 M glycine for 5 min. The resulting cells were incubated with 0.1% SDS then 1% Triton X-100. Genomic DNA was digested with the _Dpn_II restriction enzyme (NEB) and incubated at 37 °C overnight. The cohesive ends were filled in with biotin-14-dATP and 5 U μl−1 Klenow fragment of DNA polymerase I (NEB), followed by a 90 min incubation at 37 °C with gentle rotation. The resulting DNA fragments were then ligated by adding a mix of T4 DNA ligase (Thermo Fisher Scientific), 10% Triton X-100 and bovine serum albumin (NEB). The cross-linking was reversed with proteinase K and the biotinylated fragments were pulled down using Dynabeads MyOne Streptavidin C1 beads (Thermo Fisher Scientific). The Hi-C library for Illumina sequencing was prepared using the NEBNext Ultra II DNA library Prep Kit for Illumina (NEB). Trimmed Hi‐C reads were aligned to the polished genome assembly using bwa v.0.7.17 and filtered for Hi‐C‐specific artifacts using the HiC‐Pro v.2.11.1 pipeline[42](https://www.nature.com/articles/s41588-025-02330-y#ref-CR42 "Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. Genome Biol. 16, 259 (2015)."),[43](https://www.nature.com/articles/s41588-025-02330-y#ref-CR43 "Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)."). The Juicer v.1.5.6 and 3D-DNA v.180922 pipelines were used to group, order and orientate scaffolds and to generate a chromosome-level genome assembly. The potential misjoins, translocations and inversions within the boundaries of each chromosome were corrected by manually introducing breakpoints at the mis-assembly sites based on visualization with Juicebox v.1.5.3[44](https://www.nature.com/articles/s41588-025-02330-y#ref-CR44 "Durand, N. C. et al. Juicebox provides a visualization system for Hi-C contact maps with unlimited zoom. Cell Syst. 3, 99–101 (2016).").

### Sex chromosome identification

Because both male and female locusts have the same complement of autosomes, the read coverage of autosomes is expected to be roughly equal in the male and female data in X0/XX sex determination. The sex chromosome was identified by using a chromosome quotient method that relies on the simple assumption that female locusts have a twofold increase of read coverage compared to male locusts[45](https://www.nature.com/articles/s41588-025-02330-y#ref-CR45 "Hall, A. B. et al. A male-determining factor in the mosquito Aedes aegypti. Science 348, 1268–1270 (2015)."). The DNA-seq reads from an individual of both a male and female locust were aligned separately to the genome assembly using the Bowtie2 v.2.4.1 program[46](https://www.nature.com/articles/s41588-025-02330-y#ref-CR46 "Langmead, B. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Nat. Methods 9, 357–359 (2012)."). The resulting bam files were filtered to remove low-quality alignments using the SAMtools v.1.9 program[47](https://www.nature.com/articles/s41588-025-02330-y#ref-CR47 "Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079 (2009)."). Contigs derived from the X chromosome were also identified in the contig-level assembly to verify the mis-scaffolding links in Hi-C scaffolding.

### Analysis of transposable elements

Transposable elements and simple repeats were predicted de novo and classified using the RepeatModeler v.2.01 package with RMBlast v.2.9.0-p2 engine[48](https://www.nature.com/articles/s41588-025-02330-y#ref-CR48 "Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. Proc. Natl Acad. Sci. USA 117, 9451–9457 (2020)."). Other prerequisite modules including RECON v.1.08, RepeatScout v.1.0.6, TRF v.4.0.9, Dfam v.2.0 and Repbase-derived libraries were installed with RepeatModeler v.2.01. A combination of the consensus sequences of RepeatModeler output and the locust transposable elements deposited in Repbase were used as input for RepeatMasker v.4.11 to determine the genomic coordinates of annotated transposable elements and generate a masked version of the locust genome assembly. SINE retrotransposons were classified into non-LTR retrotransposons based on the classification system of Repbase. The Kimura distances between genome copies and TE consensus from the RepeatModeler library were determined using RepeatLandscape after genome masking with RepeatMasker. Tandem repeats were identified using Tandem Repeats Finder v.4.09 and were processed by pyTanFinder ([https://github.com/Kirovez/pyTanFinder](https://github.com/Kirovez/pyTanFinder)).

### Cleavage under targets and tagmentation

The selection of histone modifications is introduced in Supplementary Note [8](https://www.nature.com/articles/s41588-025-02330-y#MOESM1). The commercial antibodies include H3K9ac (Abcam, catalogue number ab32129), H3K14ac (Abcam, catalogue number ab52946), H3K18ac (Abcam, catalogue number ab177870), H3K27ac (Abcam, catalogue number ab4729), H3K27me1 (Active Motif, catalogue number 61015), H3K27me2 (Abcam, catalogue number ab24684), H3K27me3 (Cell Signaling Technology, catalogue number 9733S), H3K36me3 (Active Motif, catalogue number 61022), H3K4me1 (Abcam, catalogue number ab8895), H3K4me2 (Abcam, catalogue number ab32356), H3K4me3 (Abcam, catalogue number ab8580), H3K9me3 (Abcam, catalogue number ab8898), H4K20me3 (Abcam, catalogue number ab9053), H4K16ac (Abcam, catalogue number ab109463) and H4K20me1 (Abcam, catalogue number ab9051). The locust tissues were isolated and immobilized to concanavalin A-coated beads (Novoprotein). The beads-bound cells were incubated with primary antibodies by rotating overnight. The resulting cells were incubated with secondary antibodies at room temperature for 1 h and were then washed three times with DIG-wash buffer to remove unbound antibodies. Cells were resuspended in DIG-300 buffer (300 mM NaCl) with pA/G-Tn5 adapter complex for 1 h at room temperature by rotating, followed by tagmentation at 37 °C for 1 h. Subsequent washes were also performed under stringent salt conditions (300 mM NaCl) to suppress the affinity of Tn5 for open chromatin[49](https://www.nature.com/articles/s41588-025-02330-y#ref-CR49 "Kaya-Okur, H. S., Janssens, D. H., Henikoff, J. G., Ahmad, K. & Henikoff, S. Efficient low-cost chromatin profiling with CUT&Tag. Nat. Protoc. 15, 3264–3283 (2020)."). Barcoded I5 and I7 primers were used for PCR amplification to generate the sequencing libraries. The final sequencing libraries were sequenced on an Illumina NovaSeq 6000 system. Paired-end reads were aligned to LMv.3.1 using Bowtie2 v.2.4.1. The multiBigwigSummary and plotCorrelation functions from deepTools v.2.27.1 were used to assess replicate reproducibility. The computeMatrix and plotHeatmap functions from deepTools v.2.27.1 were used to generate the heatmap. The CUT&Tag peaks were called by MACS2 v.2.2.7.1 and SEACR v.1.3 for narrow and broad peaks, respectively[50](https://www.nature.com/articles/s41588-025-02330-y#ref-CR50 "Zhang, Y. et al. Model-based analysis of ChIP–Seq (MACS). Genome Biol. 9, R137 (2008)."),[51](https://www.nature.com/articles/s41588-025-02330-y#ref-CR51 "Meers, M. P., Tenenbaum, D. & Henikoff, S. Peak calling by sparse enrichment analysis for CUT&RUN chromatin profiling. Epigenetics Chromatin 12, 42 (2019)."). The fragment size plots of the CUT&Tag experiments showed periodical peaks corresponding to the mononucleosomes, dinucleosomes and trinucleosomes (Supplementary Fig. [32](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). The signal intensities of stranded RNA-seq correlated positively with H3K36me3 (Fig. [2a](https://www.nature.com/articles/s41588-025-02330-y#Fig2)), which is consistent with their demarcation role in gene-body deposition. The negative correlation of the repressed histone modification marks (H3K27me3, H3K27me2 and H4K20me3) with gene-body transcription suggests these repressed histone modification marks are more likely to silence expression in the nongenic region. The pairwise-correlated permissive histone modification marks (H3K18ac, H3K14ac, H3K27ac and H3K9ac) exhibited a positive correlation with promoter-associated transcription (H3K4me3, H3K4me2 and TSS-seq). For the histone modification datasets, we obtained a total of 683,539 peaks with an average number of 52,579 (ranging from 13,555 for H3K4me2 to 99,091 for H3K27me2; Supplementary Fig. [33](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)).

### Transcription start sites

Total RNAs were isolated using TRIzol reagent (Invitrogen). Genomic DNA was removed using TURBO DNase (Invitrogen) and poly-A RNA was enriched twice using a Dynabeads Oligo (dT) 25 kit (Thermo Fisher Scientific). Poly-A RNA was treated with calf intestinal alkaline phosphatase at 37 °C for 1 h and then treated with tobacco acid pyrophosphatase at 37 °C for 1 h. The sequencing libraries were constructed using the NEBNext Ultra II DNA Library Prep Kit for Illumina (NEB) following the manufacturer’s instructions and sequenced on an Illumina NovaSeq 6000 system. The trimmed reads were aligned to LMv.3.1 using HISAT2 v.2.2.1 without allowing soft clipping alignments. The resulting BAM files were used in the identification of oligo-capping transcription start sites using the CAGEr v.1.24.0 package in R v.3.5.1[52](https://www.nature.com/articles/s41588-025-02330-y#ref-CR52 "Haberle, V., Forrest, A. R., Hayashizaki, Y., Carninci, P. & Lenhard, B. CAGEr: precise TSS data retrieval and high-resolution promoterome mining for integrative analyses. Nucleic Acids Res. 43, e51 (2015).").

### ATAC sequencing

Locust tissues were digested using trypsin/EDTA and resuspended in lysis buffer to permeabilize membranes. The nuclear pellets were resuspended in the tagmentation mix (Vazyme) and incubated at 37 °C for 30 min. DNA Clean Beads (Vazyme) was then added to terminate the tagmentation reaction and to purify DNA fragments. Barcoded I5 and I7 primers were used for PCR amplification to generate sequencing libraries. The resulting sequencing libraries were purified with DNA Clean Beads (Vazyme) and subjected to sequencing on an Illumina NovaSeq 6000 system. ATAC-seq data analyses were based on the ENCODE pipeline. Trimmed reads were aligned to LMv.3.1 using Bowtie2 v.2.4.1. The aligned reads +4 base pairs (bp) for positive strands and −5 bp for negative strands were shifted to adjust the 9-bp duplication created by Tn5 transposase. The ATAC-seq peaks were called on nucleosome-free reads by MACS2 v.2.2.7.1 using the narrow parameter[50](https://www.nature.com/articles/s41588-025-02330-y#ref-CR50 "Zhang, Y. et al. Model-based analysis of ChIP–Seq (MACS). Genome Biol. 9, R137 (2008)."). The consensus peaks were retained for further analyses. ATAC-seq experiments generated an enrichment of fragment size around 100 and 200 bp, indicating nucleosome-free and mononucleosome-bound fragments (Supplementary Fig. [34](https://www.nature.com/articles/s41588-025-02330-y#MOESM1)). ATAC-seq signals were enriched at TSSs that were captured by Oligo-capping sequencing.

### Annotation of chromatin states

We used ChromHMM v.1.23 to analyze the combinatorial patterns of 13 histone modifications[53](https://www.nature.com/articles/s41588-025-02330-y#ref-CR53 "Ernst, J. & Kellis, M. Chromatin-state discovery and genome annotation with ChromHMM. Nat. Protoc. 12, 2478–2492 (2017)."). The chromatin state annotation definitions are adopted from those used for the human genome[54](https://www.nature.com/articles/s41588-025-02330-y#ref-CR54 "Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317–330 (2015)."). Briefly, the promoter state is associated primarily with H3K4me3 and H3K27ac. The flanking promoter state shows high levels of H3K4me3 but lacks H3K27ac, reflecting the broader distribution of H3K4me3 compared to H3K27ac. The transcribed at gene 5′ end state is associated with H3K4me3 and H3K36me3. The two transcription states are associated specifically with H3K36me3 and H3K27me1, respectively. The enhancer state is linked with H3K27ac and H3K4me1, and the bivalent enhancer state is characterized by the presence of enhancer-associated marks (H3K18ac, H3K4ac and H3K27ac) together with repressive marks H3K27me3. The repressed polycomb state is defined by its exclusive association with H3K27me3, since H3K27me3 is catalyzed primarily by polycomb repressive complexes. The difference between the two H3K9me3-associated heterochromatin states is the presence of H4K20me3.

### Fluorescent in situ hybridization

FISH was carried out using the FISH Kit (BersinBio) with small modifications. Chromosome preparation was incubated in the digestive working solution. After three washes in distilled water, the slides were dehydrated in an ethanol series then air-dried. DNA probes and slides were denatured and hybridized overnight. After hybridization, slides were washed under low stringency conditions (2× SSC, 0.1% NP-40/2× SSC and 2× SSC).

### Immunofluorescence

Testes were fixed in 4% paraformaldehyde (PFA) at 4 °C overnight, and seminiferous tubules were then isolated. Brains were collected in PBS and then digested in collagenase/dispase solution (2 mg ml−1, Roche) for 10–15 min at 28 °C. Incubation with primary antibodies against H4K16ac (Abcam, catalogue number ab109463) was performed at 4 °C in a moist chamber overnight. The secondary antibody, Alexa Fluor 546 goat anti-rabbit IgG (Thermo Fisher, catalogue number A-11035), was diluted in immunofluorescence staining secondary antibody dilution buffer. Hoechst was used as nuclear marker.

### Genomic fragment knockout using the CRISPR–Cas9 system

We masked the repetitive region to remove segments with the potential to generate off-target (guide RNAs (gRNAs). Subsequently, we designed gRNAs based on nonrepetitive regions flanking EH1.The cutting efficiency of the gRNAs was determined using Illumina amplicon sequencing. gRNAs were transcribed using a template assembled with the GeneArt Precision gRNA Synthesis Kit (Thermo Fisher). Homology donors were produced by incorporating two homology arms located in the flanking region of EH1. The mixture of gRNAs, homology donors and TrueCut Cas9 Protein v.2 (Thermo Fisher) was microinjected into the eggs laid within 2 h. Microinjected eggs were placed in disposable sterile Petri dishes. Knockout of the 379-bp genomic fragment encompassing EH1 was verified by Sanger sequencing.

### Behavioral assays

Gregarious locusts were reared in large cages (40 cm × 40 cm × 40 cm) at a density of approximately 600 locusts per cage. Solitary locusts were cultured individually in white metal boxes (10 cm × 10 cm × 25 cm) supplied with charcoal-filtered compressed air. Development into the solitary or gregarious phase depends on rearing conditions. The behavioral assay for the third day of fourth-instar locust nymphs was conducted in a rectangular Perspex arena with opaque walls and a clear top. A locust was released into the arena and monitored for 5 min. Individual behavioral data were recorded automatically and analyzed using the EthoVision video tracking system. The three behavioral variables were as follows: total distance moved (TDM) and total duration of movement (TDMV), representing motor activity levels, and attraction index (AI), defined as the total duration in the stimulus area minus the total duration in the opposite area, representing attraction or repulsion to the stimulus group. The behavioral phase status of each locust was assessed by applying a single probabilistic metric of gregariousness, _P_greg. _P_greg = e_η_/(1 + e_η_), where _η_ = −2.11 + 0.005 × AI + 0.012 × TDM + 0.015 × TDMV. e represents Euler’s number, approximately 2.718. A _P_greg value of 1 indicates fully gregarious behavior, whereas a value of 0 indicates fully solitary behavior.

### Quantitative chromosome conformation capture assays

Cells from locust brains were collected to prepare single-cell suspensions in 1 ml PBS. The resulting cells were incubated with 2% formaldehyde for cross-linking and were quenched with a final concentration of 0.125 mM glycine. The cells were not treated with cross-linking as a digested DNA control. Cells were lysed with 500 μl 1× cold lysis buffer (10 mM Tris–HCl pH 8.0, 10 mM NaCl, 0.2% NP-40). Each sample was digested overnight with _Dpn_II enzyme. Ligation reaction was performed at 16 °C for ~22 h in 1.5-ml tubes containing 10× T4 ligase buffer, 125 μl water, 4 μl 40 U per microliter of LT4 ligase. The crosslinks were reversed with Proteinase K (Invitrogen) at 65 °C overnight. DNA were then purified using phenol–chloroform extraction and quantified by Qubit dsDNA HS Assay (Invitrogen). The primer sequences are provided in Supplementary Table [2](https://www.nature.com/articles/s41588-025-02330-y#MOESM1).

### Quantitative reverse transcription PCR

Total RNA was extracted using TRIzol reagent (Invitrogen). Quantitative reverse transcription PCR was performed using FastStart DNA Master SYBR Green I (Roche) with a LightCycler 480 instrument (Roche). Relative expression levels were calculated using the 2−ΔΔCt method.

### Statistics

For statistical analysis, a two-sided Student’s _t-_test was used to evaluate differences in (1) mRNA and protein expression of _Henna_, (2) spatial interaction between EH1 and TSS1 and (3) read abundances of H4K16ac between autosomes and the X chromosome. A two-sided Wilcoxon rank-sum exact test was used to evaluate differences in (1) histone modification deposition on gene-body regions in short-size and large-size genes, (2) female-to-male ratios of genic H4K16ac and H4K20me1 signals (with Benjamini–Hochberg adjusted _P_ values) and (3) arena behavioral assays. A two-sided Pearson’s product-moment correlation test was used to assess the statistical significance of correlations in (1) intron length ratios between the locust and the two other insects, (2) RNA expression ratios between the locust and the fruit fly and (3) gene expression between females and males. A one-sided Student’s _t_\-test was used to evaluate differences in d_N_/d_S_ values for translocated genes, with Benjamini–Hochberg adjusted _P_ values. The significance between chromosome–chromosome associations in Caeliferan species was assessed with the two-sided Fisher’s exact test against a null model of random ortholog permutation with a Bonferroni correction.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02330-y#MOESM2) linked to this article.

Data availability
-----------------

The sequencing data and genome assemblies have been deposited in the Genome Sequence Archive at the National Genomics Data Center of the China National Center for Bioinformation under the BioProject accession number [PRJCA043246](https://ngdc.cncb.ac.cn/bioproject/browse/PRJCA043246). The genome annotations are available via figshare at [https://doi.org/10.6084/m9.figshare.27889818](https://doi.org/10.6084/m9.figshare.27889818) (ref. [55](https://www.nature.com/articles/s41588-025-02330-y#ref-CR55 "Jiang, F. Chromosome-level genome assemblies of the migratory locust and the desert locust. figshare 
https://doi.org/10.6084/m9.figshare.27889818
(2025).")). The transcriptome data of the brain, testis and fat body of the fruit fly were retrieved from the NCBI database under accession number [PRJNA75285](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA75285). The H3K27ac and H3K4me1 profiling data from fruit fly and honey bee brains were retrieved from the NCBI GEO database under accession numbers [GSE210427](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE210427) and [GSE206992](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE206992), respectively. The H4K16ac profiling data from female and male fruit flies were retrieved from the NCBI database under GEO accession number [GSE133637](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE133637). [Source data](https://www.nature.com/articles/s41588-025-02330-y#Sec34) are provided with this paper.

Code availability
-----------------

The processing commands for sequencing data analysis and Figs. [1–6](https://www.nature.com/articles/s41588-025-02330-y#Fig1) are available via figshare at [https://doi.org/10.6084/m9.figshare.27889818](https://doi.org/10.6084/m9.figshare.27889818) (ref. [55](https://www.nature.com/articles/s41588-025-02330-y#ref-CR55 "Jiang, F. Chromosome-level genome assemblies of the migratory locust and the desert locust. figshare 
https://doi.org/10.6084/m9.figshare.27889818
(2025).")).

References
----------

1.  Wang, X. & Kang, L. Molecular mechanisms of phase change in locusts. _Annu Rev. Entomol._ **59**, 225–244 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24160426)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXjtlSlsbg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20mechanisms%20of%20phase%20change%20in%20locusts&journal=Annu%20Rev.%20Entomol.&volume=59&pages=225-244&publication_year=2014&author=Wang%2CX&author=Kang%2CL) 
    
2.  Ma, Z., Guo, W., Guo, X., Wang, X. & Kang, L. Modulation of behavioral phase changes of the migratory locust by the catecholamine metabolic pathway. _Proc. Natl Acad. Sci. USA_ **108**, 3882–3887 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21325054)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053982)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXjs1Wqsbc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Modulation%20of%20behavioral%20phase%20changes%20of%20the%20migratory%20locust%20by%20the%20catecholamine%20metabolic%20pathway&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=108&pages=3882-3887&publication_year=2011&author=Ma%2CZ&author=Guo%2CW&author=Guo%2CX&author=Wang%2CX&author=Kang%2CL) 
    
3.  Yang, M. et al. MicroRNA-133 inhibits behavioral aggregation by controlling dopamine synthesis in locusts. _PLoS Genet._ **10**, e1004206 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24586212)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3937255)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MicroRNA-133%20inhibits%20behavioral%20aggregation%20by%20controlling%20dopamine%20synthesis%20in%20locusts&journal=PLoS%20Genet.&volume=10&publication_year=2014&author=Yang%2CM) 
    
4.  Gagnidze, K., Weil, Z. M. & Pfaff, D. W. Histone modifications proposed to regulate sexual differentiation of brain and behavior. _Bioessays_ **32**, 932–939 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20836091)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFOns77J)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Histone%20modifications%20proposed%20to%20regulate%20sexual%20differentiation%20of%20brain%20and%20behavior&journal=Bioessays&volume=32&pages=932-939&publication_year=2010&author=Gagnidze%2CK&author=Weil%2CZM&author=Pfaff%2CDW) 
    
5.  Meyer, B. J. The X chromosome in _C. elegans_ sex determination and dosage compensation. _Curr. Opin. Genet. Dev._ **74**, 101912 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35490475)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9462529)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtVeqs73L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20X%20chromosome%20in%20C.%20elegans%20sex%20determination%20and%20dosage%20compensation&journal=Curr.%20Opin.%20Genet.%20Dev.&volume=74&publication_year=2022&author=Meyer%2CBJ) 
    
6.  Vielle, A. et al. H4K20me1 contributes to downregulation of X-linked genes for _C. elegans_ dosage compensation. _PLoS Genet._ **8**, e1002933 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23028348)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3441679)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVSjtLrI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=H4K20me1%20contributes%20to%20downregulation%20of%20X-linked%20genes%20for%20C.%20elegans%20dosage%20compensation&journal=PLoS%20Genet.&volume=8&publication_year=2012&author=Vielle%2CA) 
    
7.  Straub, T., Grimaud, C., Gilfillan, G. D., Mitterweger, A. & Becker, P. B. The chromosomal high-affinity binding sites for the _Drosophila_ dosage compensation complex. _PLoS Genet._ **4**, e1000302 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19079572)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2586088)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20chromosomal%20high-affinity%20binding%20sites%20for%20the%20Drosophila%20dosage%20compensation%20complex&journal=PLoS%20Genet.&volume=4&publication_year=2008&author=Straub%2CT&author=Grimaud%2CC&author=Gilfillan%2CGD&author=Mitterweger%2CA&author=Becker%2CPB) 
    
8.  Wiles, E. T. & Selker, E. U. H3K27 methylation: a promiscuous repressive chromatin mark. _Curr. Opin. Genet. Dev._ **43**, 31–37 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27940208)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvFSiu7%2FL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=H3K27%20methylation%3A%20a%20promiscuous%20repressive%20chromatin%20mark&journal=Curr.%20Opin.%20Genet.%20Dev.&volume=43&pages=31-37&publication_year=2017&author=Wiles%2CET&author=Selker%2CEU) 
    
9.  Kraft, K. et al. Polycomb-mediated genome architecture enables long-range spreading of H3K27 methylation. _Proc. Natl Acad. Sci. USA_ **119**, e2201883119 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35617427)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9295753)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFKrsrfI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polycomb-mediated%20genome%20architecture%20enables%20long-range%20spreading%20of%20H3K27%20methylation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=119&publication_year=2022&author=Kraft%2CK) 
    
10.  Ernst, U. R. et al. Epigenetics and locust life phase transitions. _J. Exp. Biol._ **218**, 88–99 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25568455)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenetics%20and%20locust%20life%20phase%20transitions&journal=J.%20Exp.%20Biol.&volume=218&pages=88-99&publication_year=2015&author=Ernst%2CUR) 
    
11.  Heinz, S., Romanoski, C. E., Benner, C. & Glass, C. K. The selection and function of cell type-specific enhancers. _Nat. Rev. Mol. Cell Biol._ **16**, 144–154 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25650801)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4517609)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjtFenu7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20selection%20and%20function%20of%20cell%20type-specific%20enhancers&journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&volume=16&pages=144-154&publication_year=2015&author=Heinz%2CS&author=Romanoski%2CCE&author=Benner%2CC&author=Glass%2CCK) 
    
12.  Vorobyeva, N. E., Krasnov, A. N., Erokhin, M., Chetverina, D. & Mazina, M. Su(Hw) interacts with Combgap to establish long-range chromatin contacts. _Epigenetics Chromatin_ **17**, 17 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38773468)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11106861)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1CqsbfJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Su%28Hw%29%20interacts%20with%20Combgap%20to%20establish%20long-range%20chromatin%20contacts&journal=Epigenetics%20Chromatin&volume=17&publication_year=2024&author=Vorobyeva%2CNE&author=Krasnov%2CAN&author=Erokhin%2CM&author=Chetverina%2CD&author=Mazina%2CM) 
    
13.  Chetverina, D., Erokhin, M. & Schedl, P. GAGA factor: a multifunctional pioneering chromatin protein. _Cell. Mol. Life Sci._ **78**, 4125–4141 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33528710)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8815297)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjtVWrur4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GAGA%20factor%3A%20a%20multifunctional%20pioneering%20chromatin%20protein&journal=Cell.%20Mol.%20Life%20Sci.&volume=78&pages=4125-4141&publication_year=2021&author=Chetverina%2CD&author=Erokhin%2CM&author=Schedl%2CP) 
    
14.  Reddington, J. P. et al. Lineage-resolved enhancer and promoter usage during a time course of embryogenesis. _Dev. Cell_ **55**, 648–664 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33171098)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlemsLbI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lineage-resolved%20enhancer%20and%20promoter%20usage%20during%20a%20time%20course%20of%20embryogenesis&journal=Dev.%20Cell&volume=55&pages=648-664&publication_year=2020&author=Reddington%2CJP) 
    
15.  Kvetnansky, R., Sabban, E. L. & Palkovits, M. Catecholaminergic systems in stress: structural and molecular genetic approaches. _Physiol. Rev._ **89**, 535–606 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19342614)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXkvFegu70%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Catecholaminergic%20systems%20in%20stress%3A%20structural%20and%20molecular%20genetic%20approaches&journal=Physiol.%20Rev.&volume=89&pages=535-606&publication_year=2009&author=Kvetnansky%2CR&author=Sabban%2CEL&author=Palkovits%2CM) 
    
16.  Duan, J. et al. CLAMP and Zelda function together to promote _Drosophila_ zygotic genome activation. _eLife_ **10**, e69937 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34342574)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8367384)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsV2ns7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CLAMP%20and%20Zelda%20function%20together%20to%20promote%20Drosophila%20zygotic%20genome%20activation.&journal=eLife&volume=10&publication_year=2021&author=Duan%2CJ) 
    
17.  Liu, Q., Jiang, F., Zhang, J., Li, X. & Kang, L. Transcription initiation of distant core promoters in a large-sized genome of an insect. _BMC Biol._ **19**, 62 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33785021)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8011201)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1Wrur3E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20initiation%20of%20distant%20core%20promoters%20in%20a%20large-sized%20genome%20of%20an%20insect&journal=BMC%20Biol.&volume=19&publication_year=2021&author=Liu%2CQ&author=Jiang%2CF&author=Zhang%2CJ&author=Li%2CX&author=Kang%2CL) 
    
18.  Sexton, T. et al. Three-dimensional folding and functional organization principles of the _Drosophila_ genome. _Cell_ **148**, 458–472 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22265598)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvFehsLo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Three-dimensional%20folding%20and%20functional%20organization%20principles%20of%20the%20Drosophila%20genome&journal=Cell&volume=148&pages=458-472&publication_year=2012&author=Sexton%2CT) 
    
19.  Monahan, K., Horta, A. & Lomvardas, S. LHX2- and LDB1-mediated trans interactions regulate olfactory receptor choice. _Nature_ **565**, 448–453 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30626972)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6436840)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFShsLY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LHX2-%20and%20LDB1-mediated%20trans%20interactions%20regulate%20olfactory%20receptor%20choice&journal=Nature&volume=565&pages=448-453&publication_year=2019&author=Monahan%2CK&author=Horta%2CA&author=Lomvardas%2CS) 
    
20.  Arafat, E. A., El-Samad, L. M., Moussian, B. & Hassan, M. A. Insights into spermatogenesis in the migratory locust, _Locusta migratoria_ (Linnaeus, 1758)(Orthoptera: Acrididae), following histological and ultrastructural features of the testis. _Micron_ **172**, 103502 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37422968)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVGju77F)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20into%20spermatogenesis%20in%20the%20migratory%20locust%2C%20Locusta%20migratoria%20%28Linnaeus%2C%201758%29%28Orthoptera%3A%20Acrididae%29%2C%20following%20histological%20and%20ultrastructural%20features%20of%20the%20testis.&journal=Micron&volume=172&publication_year=2023&author=Arafat%2CEA&author=El-Samad%2CLM&author=Moussian%2CB&author=Hassan%2CMA) 
    
21.  Disteche, C. M. Dosage compensation of the sex chromosomes. _Annu Rev. Genet._ **46**, 537–560 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22974302)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3767307)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVykur%2FO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dosage%20compensation%20of%20the%20sex%20chromosomes&journal=Annu%20Rev.%20Genet.&volume=46&pages=537-560&publication_year=2012&author=Disteche%2CCM) 
    
22.  Gelbart, M. E., Larschan, E., Peng, S., Park, P. J. & Kuroda, M. I. _Drosophila_ MSL complex globally acetylates H4K16 on the male X chromosome for dosage compensation. _Nat. Struct. Mol. Biol._ **16**, 825–832 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19648925)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2722042)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXpt1OgtLg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Drosophila%20MSL%20complex%20globally%20acetylates%20H4K16%20on%20the%20male%20X%20chromosome%20for%20dosage%20compensation&journal=Nat.%20Struct.%20Mol.%20Biol.&volume=16&pages=825-832&publication_year=2009&author=Gelbart%2CME&author=Larschan%2CE&author=Peng%2CS&author=Park%2CPJ&author=Kuroda%2CMI) 
    
23.  Samata, M. & Akhtar, A. Dosage compensation of the X chromosome: a complex epigenetic assignment involving chromatin regulators and long noncoding RNAs. _Annu. Rev. Biochem._ **87**, 323–350 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29668306)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXnvFShsLY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dosage%20compensation%20of%20the%20X%20chromosome%3A%20a%20complex%20epigenetic%20assignment%20involving%20chromatin%20regulators%20and%20long%20noncoding%20RNAs&journal=Annu.%20Rev.%20Biochem.&volume=87&pages=323-350&publication_year=2018&author=Samata%2CM&author=Akhtar%2CA) 
    
24.  Keller Valsecchi, C. I., Marois, E., Basilicata, M. F., Georgiev, P. & Akhtar, A. _Life Sci. Alliance_ **4**, e202000996 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34266874)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8321682)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=&journal=Life%20Sci.%20Alliance&volume=4&publication_year=2021&author=Keller%20Valsecchi%2CCI&author=Marois%2CE&author=Basilicata%2CMF&author=Georgiev%2CP&author=Akhtar%2CA) 
    
25.  Rieder, L. E., Jordan, W. T. 3rd & Larschan, E. N. Targeting of the dosage-compensated male X-chromosome during early _Drosophila_ development. _Cell Rep._ **29**, 4268–4275 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31875538)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952266)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFeks7g%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20of%20the%20dosage-compensated%20male%20X-chromosome%20during%20early%20Drosophila%20development&journal=Cell%20Rep.&volume=29&pages=4268-4275&publication_year=2019&author=Rieder%2CLE&author=Jordan%2CWT&author=Larschan%2CEN) 
    
26.  Levillain, O., Diaz, J. J., Blanchard, O. & Dechaud, H. Testosterone down-regulates ornithine aminotransferase gene and up-regulates arginase II and ornithine decarboxylase genes for polyamines synthesis in the murine kidney. _Endocrinology_ **146**, 950–959 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15539552)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXotlWmsg%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Testosterone%20down-regulates%20ornithine%20aminotransferase%20gene%20and%20up-regulates%20arginase%20II%20and%20ornithine%20decarboxylase%20genes%20for%20polyamines%20synthesis%20in%20the%20murine%20kidney&journal=Endocrinology&volume=146&pages=950-959&publication_year=2005&author=Levillain%2CO&author=Diaz%2CJJ&author=Blanchard%2CO&author=Dechaud%2CH) 
    
27.  Lu, W., Lakonishok, M. & Gelfand, V. I. The dynamic duo of microtubule polymerase mini spindles/XMAP215 and cytoplasmic dynein is essential for maintaining _Drosophila_ oocyte fate. _Proc. Natl Acad. Sci. USA_ **120**, e2303376120 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37722034)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10523470)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFWhsL3M)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20dynamic%20duo%20of%20microtubule%20polymerase%20mini%20spindles%2FXMAP215%20and%20cytoplasmic%20dynein%20is%20essential%20for%20maintaining%20Drosophila%20oocyte%20fate&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=120&publication_year=2023&author=Lu%2CW&author=Lakonishok%2CM&author=Gelfand%2CVI) 
    
28.  Yang, P., Hou, L., Wang, X. & Kang, L. Core transcriptional signatures of phase change in the migratory locust. _Protein Cell_ **10**, 883–901 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31292921)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6881432)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Core%20transcriptional%20signatures%20of%20phase%20change%20in%20the%20migratory%20locust&journal=Protein%20Cell&volume=10&pages=883-901&publication_year=2019&author=Yang%2CP&author=Hou%2CL&author=Wang%2CX&author=Kang%2CL) 
    
29.  Simola, D. F. et al. A chromatin link to caste identity in the carpenter ant _Camponotus floridanus_. _Genome Res._ **23**, 486–496 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23212948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589537)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXktlems74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20chromatin%20link%20to%20caste%20identity%20in%20the%20carpenter%20ant%20Camponotus%20floridanus&journal=Genome%20Res.&volume=23&pages=486-496&publication_year=2013&author=Simola%2CDF) 
    
30.  Wojciechowski, M. et al. Phenotypically distinct female castes in honey bees are defined by alternative chromatin states during larval development. _Genome Res._ **28**, 1532–1542 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30135090)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6169885)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlWlu7vO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phenotypically%20distinct%20female%20castes%20in%20honey%20bees%20are%20defined%20by%20alternative%20chromatin%20states%20during%20larval%20development&journal=Genome%20Res.&volume=28&pages=1532-1542&publication_year=2018&author=Wojciechowski%2CM) 
    
31.  Zayed, A., Naeger, N. L., Rodriguez-Zas, S. L. & Robinson, G. E. Common and novel transcriptional routes to behavioral maturation in worker and male honey bees. _Genes Brain Behav._ **11**, 253–261 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22050787)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XotVSmtbs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Common%20and%20novel%20transcriptional%20routes%20to%20behavioral%20maturation%20in%20worker%20and%20male%20honey%20bees&journal=Genes%20Brain%20Behav.&volume=11&pages=253-261&publication_year=2012&author=Zayed%2CA&author=Naeger%2CNL&author=Rodriguez-Zas%2CSL&author=Robinson%2CGE) 
    
32.  Lowe, R., Wojciechowski, M., Ellis, N. & Hurd, P. J. Chromatin accessibility-based characterisation of brain gene regulatory networks in three distinct honey bee polyphenisms. _Nucleic Acids Res._ **50**, 11550–11562 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36330958)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9723623)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXovVSgt7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20accessibility-based%20characterisation%20of%20brain%20gene%20regulatory%20networks%20in%20three%20distinct%20honey%20bee%20polyphenisms&journal=Nucleic%20Acids%20Res.&volume=50&pages=11550-11562&publication_year=2022&author=Lowe%2CR&author=Wojciechowski%2CM&author=Ellis%2CN&author=Hurd%2CPJ) 
    
33.  Turner, B. M., Birley, A. J. & Lavender, J. Histone H4 isoforms acetylated at specific lysine residues define individual chromosomes and chromatin domains in _Drosophila_ polytene nuclei. _Cell_ **69**, 375–384 (1992).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1568251)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38XitlansL8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Histone%20H4%20isoforms%20acetylated%20at%20specific%20lysine%20residues%20define%20individual%20chromosomes%20and%20chromatin%20domains%20in%20Drosophila%20polytene%20nuclei&journal=Cell&volume=69&pages=375-384&publication_year=1992&author=Turner%2CBM&author=Birley%2CAJ&author=Lavender%2CJ) 
    
34.  Marsano, R. M. & Dimitri, P. Constitutive heterochromatin in eukaryotic genomes: a mine of transposable elements. _Cells_ **11**, 761 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35269383)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8909793)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xpt1Kmt74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Constitutive%20heterochromatin%20in%20eukaryotic%20genomes%3A%20a%20mine%20of%20transposable%20elements.&journal=Cells&volume=11&publication_year=2022&author=Marsano%2CRM&author=Dimitri%2CP) 
    
35.  Lawson, H. A., Liang, Y. & Wang, T. Transposable elements in mammalian chromatin organization. _Nat. Rev. Genet._ **24**, 712–723 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37286742)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtFOksr7K)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transposable%20elements%20in%20mammalian%20chromatin%20organization&journal=Nat.%20Rev.%20Genet.&volume=24&pages=712-723&publication_year=2023&author=Lawson%2CHA&author=Liang%2CY&author=Wang%2CT) 
    
36.  Li, X., Mank, J. E. & Ban, L. The grasshopper genome reveals long-term gene content conservation of the X Chromosome and temporal variation in X Chromosome evolution. _Genome Res._ **34**, 997–1007 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39103228)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11368200)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20grasshopper%20genome%20reveals%20long-term%20gene%20content%20conservation%20of%20the%20X%20Chromosome%20and%20temporal%20variation%20in%20X%20Chromosome%20evolution&journal=Genome%20Res.&volume=34&pages=997-1007&publication_year=2024&author=Li%2CX&author=Mank%2CJE&author=Ban%2CL) 
    
37.  Koren, S. et al. Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. _Genome Res._ **27**, 722–736 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28298431)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411767)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFyjsrvI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&journal=Genome%20Res.&volume=27&pages=722-736&publication_year=2017&author=Koren%2CS) 
    
38.  Chin, C. S. et al. Phased diploid genome assembly with single-molecule real-time sequencing. _Nat. Methods_ **13**, 1050–1054 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27749838)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5503144)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ykur7K)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phased%20diploid%20genome%20assembly%20with%20single-molecule%20real-time%20sequencing&journal=Nat.%20Methods&volume=13&pages=1050-1054&publication_year=2016&author=Chin%2CCS) 
    
39.  Kronenberg, Z. N. et al. Extended haplotype-phasing of long-read de novo genome assemblies using Hi-C. _Nat. Commun._ **12**, 1935 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33911078)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8081726)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXpvFartLg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extended%20haplotype-phasing%20of%20long-read%20de%20novo%20genome%20assemblies%20using%20Hi-C&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Kronenberg%2CZN) 
    
40.  Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. _PLoS ONE_ **9**, e112963 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25409509)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237348)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&journal=PLoS%20ONE&volume=9&publication_year=2014&author=Walker%2CBJ) 
    
41.  English, A. C. et al. Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology. _PLoS ONE_ **7**, e47768 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23185243)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3504050)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVagsr3E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mind%20the%20gap%3A%20upgrading%20genomes%20with%20Pacific%20Biosciences%20RS%20long-read%20sequencing%20technology&journal=PLoS%20ONE&volume=7&publication_year=2012&author=English%2CAC) 
    
42.  Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. _Genome Biol._ **16**, 259 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26619908)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4665391)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HiC-Pro%3A%20an%20optimized%20and%20flexible%20pipeline%20for%20Hi-C%20data%20processing&journal=Genome%20Biol.&volume=16&publication_year=2015&author=Servant%2CN) 
    
43.  Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. _Bioinformatics_ **25**, 1754–1760 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19451168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&journal=Bioinformatics&volume=25&pages=1754-1760&publication_year=2009&author=Li%2CH&author=Durbin%2CR) 
    
44.  Durand, N. C. et al. Juicebox provides a visualization system for Hi-C contact maps with unlimited zoom. _Cell Syst._ **3**, 99–101 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27467250)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5596920)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFKks7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Juicebox%20provides%20a%20visualization%20system%20for%20Hi-C%20contact%20maps%20with%20unlimited%20zoom&journal=Cell%20Syst.&volume=3&pages=99-101&publication_year=2016&author=Durand%2CNC) 
    
45.  Hall, A. B. et al. A male-determining factor in the mosquito _Aedes aegypti_. _Science_ **348**, 1268–1270 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25999371)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5026532)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXps1Oltbw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20male-determining%20factor%20in%20the%20mosquito%20Aedes%20aegypti&journal=Science&volume=348&pages=1268-1270&publication_year=2015&author=Hall%2CAB) 
    
46.  Langmead, B. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. _Nat. Methods_ **9**, 357–359 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22388286)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1Oqt7c%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&journal=Nat.%20Methods&volume=9&pages=357-359&publication_year=2012&author=Langmead%2CB&author=Salzberg%2CSL) 
    
47.  Li, H. et al. The Sequence Alignment/Map format and SAMtools. _Bioinformatics_ **25**, 2078–2079 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19505943)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&journal=Bioinformatics&volume=25&pages=2078-2079&publication_year=2009&author=Li%2CH) 
    
48.  Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. _Proc. Natl Acad. Sci. USA_ **117**, 9451–9457 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32300014)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196820)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnvFeqt74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=9451-9457&publication_year=2020&author=Flynn%2CJM) 
    
49.  Kaya-Okur, H. S., Janssens, D. H., Henikoff, J. G., Ahmad, K. & Henikoff, S. Efficient low-cost chromatin profiling with CUT&Tag. _Nat. Protoc._ **15**, 3264–3283 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32913232)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318778)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvV2kur3O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20low-cost%20chromatin%20profiling%20with%20CUT%26Tag&journal=Nat.%20Protoc.&volume=15&pages=3264-3283&publication_year=2020&author=Kaya-Okur%2CHS&author=Janssens%2CDH&author=Henikoff%2CJG&author=Ahmad%2CK&author=Henikoff%2CS) 
    
50.  Zhang, Y. et al. Model-based analysis of ChIP–Seq (MACS). _Genome Biol._ **9**, R137 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18798982)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Model-based%20analysis%20of%20ChIP%E2%80%93Seq%20%28MACS%29&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Zhang%2CY) 
    
51.  Meers, M. P., Tenenbaum, D. & Henikoff, S. Peak calling by sparse enrichment analysis for CUT&RUN chromatin profiling. _Epigenetics Chromatin_ **12**, 42 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31300027)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6624997)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Peak%20calling%20by%20sparse%20enrichment%20analysis%20for%20CUT%26RUN%20chromatin%20profiling&journal=Epigenetics%20Chromatin&volume=12&publication_year=2019&author=Meers%2CMP&author=Tenenbaum%2CD&author=Henikoff%2CS) 
    
52.  Haberle, V., Forrest, A. R., Hayashizaki, Y., Carninci, P. & Lenhard, B. CAGEr: precise TSS data retrieval and high-resolution promoterome mining for integrative analyses. _Nucleic Acids Res._ **43**, e51 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25653163)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4417143)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CAGEr%3A%20precise%20TSS%20data%20retrieval%20and%20high-resolution%20promoterome%20mining%20for%20integrative%20analyses&journal=Nucleic%20Acids%20Res.&volume=43&publication_year=2015&author=Haberle%2CV&author=Forrest%2CAR&author=Hayashizaki%2CY&author=Carninci%2CP&author=Lenhard%2CB) 
    
53.  Ernst, J. & Kellis, M. Chromatin-state discovery and genome annotation with ChromHMM. _Nat. Protoc._ **12**, 2478–2492 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29120462)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5945550)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsl2js7rN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin-state%20discovery%20and%20genome%20annotation%20with%20ChromHMM&journal=Nat.%20Protoc.&volume=12&pages=2478-2492&publication_year=2017&author=Ernst%2CJ&author=Kellis%2CM) 
    
54.  Roadmap Epigenomics Consortium et al. Integrative analysis of 111 reference human epigenomes. _Nature_ **518**, 317–330 (2015).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4530010)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&journal=Nature&volume=518&pages=317-330&publication_year=2015&author=Roadmap%20Epigenomics%20Consortium%2C) 
    
55.  Jiang, F. Chromosome-level genome assemblies of the migratory locust and the desert locust. _figshare_ [https://doi.org/10.6084/m9.figshare.27889818](https://doi.org/10.6084/m9.figshare.27889818) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02330-y?format=refman&flavour=references)

Acknowledgements
----------------

This work was supported by the National Key R&D Program of China (2022YFD1400500 to F.J.), the International Partnership Program of Chinese Academy of Sciences (151542KYSB20200016 to F.J. and W.G.), the National Natural Science Foundation of China (32088102 to L.K., 32300397 to Q. Liu, and 32270523 to J.H.), the Young Elite Scientists Sponsorship Program by CAST (2024QNRC001 to Q. Liu) and the Initiative Scientific Research Program of Institute of Zoology, Chinese Academy of Sciences (2024IOZ0202 to F.J.). This work was also supported by the Agriculture Science and Technology Major Project (to F.J.). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.

Author information
------------------

Author notes

1.  These authors contributed equally: Qing Liu, Feng Jiang, Ran Li, Shanlin Liu, Wenjing Fang.
    

### Authors and Affiliations

1.  State Key Laboratory of Integrated Management of Pest Insects and Rodents, Institute of Zoology, Chinese Academy of Sciences, Beijing, China
    
    Qing Liu, Feng Jiang, Ran Li, Shanlin Liu, Wenjing Fang, Xiao Li, Ting Hu, Lianyun Feng, Jie Zhang, Zhikang Liu, Jing He, Wei Guo, Xianhui Wang & Le Kang
    
2.  College of Life Sciences, University of Chinese Academy of Sciences, Beijing, China
    
    Feng Jiang, Ran Li, Wenjing Fang, Ting Hu, Lianyun Feng & Qiye Li
    
3.  GrandOmics Biosciences, Wuhan, China
    
    Shanlin Liu, Zongyi Sun, Jingjing Li, Yunan Gao & Jiacheng Yi
    
4.  BGI Research, Wuhan, China
    
    Qiye Li
    
5.  College of Life Science, Hebei University, Baoding, China
    
    Xiaoxiao Wang, Liya Wei & Le Kang
    
6.  Institute of Cell and Gene Technology, Shenzhen University of Advanced Technology, Shenzhen, China
    
    Le Kang
    

Authors

1.  Qing Liu
    
    [View author publications](https://www.nature.com/search?author=Qing%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qing%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qing%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Feng Jiang
    
    [View author publications](https://www.nature.com/search?author=Feng%20Jiang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Feng%20Jiang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Feng%20Jiang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Ran Li
    
    [View author publications](https://www.nature.com/search?author=Ran%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ran%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ran%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Shanlin Liu
    
    [View author publications](https://www.nature.com/search?author=Shanlin%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shanlin%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shanlin%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Wenjing Fang
    
    [View author publications](https://www.nature.com/search?author=Wenjing%20Fang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenjing%20Fang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenjing%20Fang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Xiao Li
    
    [View author publications](https://www.nature.com/search?author=Xiao%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiao%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiao%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Ting Hu
    
    [View author publications](https://www.nature.com/search?author=Ting%20Hu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ting%20Hu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ting%20Hu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Lianyun Feng
    
    [View author publications](https://www.nature.com/search?author=Lianyun%20Feng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lianyun%20Feng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lianyun%20Feng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Jie Zhang
    
    [View author publications](https://www.nature.com/search?author=Jie%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jie%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jie%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Zhikang Liu
    
    [View author publications](https://www.nature.com/search?author=Zhikang%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zhikang%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zhikang%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Jing He
    
    [View author publications](https://www.nature.com/search?author=Jing%20He)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jing%20He) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jing%20He%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Wei Guo
    
    [View author publications](https://www.nature.com/search?author=Wei%20Guo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wei%20Guo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wei%20Guo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Xianhui Wang
    
    [View author publications](https://www.nature.com/search?author=Xianhui%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xianhui%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xianhui%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Zongyi Sun
    
    [View author publications](https://www.nature.com/search?author=Zongyi%20Sun)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zongyi%20Sun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zongyi%20Sun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Jingjing Li
    
    [View author publications](https://www.nature.com/search?author=Jingjing%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jingjing%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jingjing%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Yunan Gao
    
    [View author publications](https://www.nature.com/search?author=Yunan%20Gao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yunan%20Gao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yunan%20Gao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Jiacheng Yi
    
    [View author publications](https://www.nature.com/search?author=Jiacheng%20Yi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jiacheng%20Yi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jiacheng%20Yi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Qiye Li
    
    [View author publications](https://www.nature.com/search?author=Qiye%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qiye%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qiye%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Xiaoxiao Wang
    
    [View author publications](https://www.nature.com/search?author=Xiaoxiao%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiaoxiao%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiaoxiao%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Liya Wei
    
    [View author publications](https://www.nature.com/search?author=Liya%20Wei)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Liya%20Wei) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Liya%20Wei%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Le Kang
    
    [View author publications](https://www.nature.com/search?author=Le%20Kang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Le%20Kang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Le%20Kang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Q. Liu, F.J. and L.K. designed and supervised the study. Q. Liu, F.J. and S.L. performed data analysis. R.L., Q. Liu and W.F. performed experiments. X.L., T.H., L.F., J.Z., Z.L., J.H., W.G., Xianhui Wang, Xiaoxiao Wang and L.W. contributed to the experiments. Xianhui Wang, Z.S., J.L., Y.G., J.Y. and Q. Li contributed to the genome assembly and annotation. F.J., Q. Liu and L.K. wrote the paper with input from all authors.

### Corresponding author

Correspondence to [Le Kang](mailto:lkang@ioz.ac.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Steven Van Belleghem and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_MOESM1_ESM.pdf)

Supplementary Notes 1–8, Figs. 1–34 and Tables 1 and 2.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_MOESM2_ESM.pdf)

Source data
-----------

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02330-y/MediaObjects/41588_2025_2330_MOESM3_ESM.pdf)

Source data for Fig. 3, unprocessed western blots and gels.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Chromatin%20dynamics%20of%20a%20large-sized%20genome%20provides%20insights%20into%20polyphenism%20and%20X0%20dosage%20compensation%20of%20locusts&author=Qing%20Liu%20et%20al&contentID=10.1038%2Fs41588-025-02330-y&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-09-01&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY-NC-ND)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02330-y)

### Cite this article

Liu, Q., Jiang, F., Li, R. _et al._ Chromatin dynamics of a large-sized genome provides insights into polyphenism and X0 dosage compensation of locusts. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02330-y

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02330-y?format=refman&flavour=citation)

-   Received: 04 June 2024
    
-   Accepted: 04 August 2025
    
-   Published: 01 September 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02330-y
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02330-y.pdf)
