> id: https://www.nature.com/articles/s41588-025-02286-z

> link: https://www.nature.com/articles/s41588-025-02286-z

> title: LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes

# LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes
_Published on Mon, 11 Aug 2025 00:00:00 GMT_

Abstract
--------

Mixed-model association analysis (MMAA) is the preferred tool for performing genome-wide association studies. However, existing MMAA tools often have long runtimes and high memory requirements. Here we present LDAK-KVIK, an MMAA tool for analysis of quantitative and binary phenotypes. LDAK-KVIK is computationally efficient, requiring less than 10 CPU hours and 5 Gb memory to analyze genome-wide data for 350,000 individuals. Using simulated phenotypes, we show that LDAK-KVIK produces well-calibrated test statistics for both homogeneous and heterogeneous datasets. When applied to real phenotypes, LDAK-KVIK has the highest power among all tools considered. For example, across 40 quantitative UK Biobank phenotypes (average sample size 349,000), LDAK-KVIK finds 16% more independent, genome-wide significant loci than classical linear regression, whereas BOLT-LMM and REGENIE find 15% and 11% more, respectively. LDAK-KVIK can also be used to perform gene-based tests; across the 40 quantitative UK Biobank phenotypes, LDAK-KVIK finds 18% more significant genes than the leading existing tool. Last, LDAK-KVIK produces state-of-the-art polygenic scores.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 12 print issues and online access

$209.00 per year

only $17.42 per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02286-z?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Type 1 errors and power of MMAA tools for quantitative phenotypes.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02286-z/MediaObjects/41588_2025_2286_Fig1_HTML.png)

**Fig. 2: Exploiting the dependency of per-SNP heritability on MAF.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02286-z/MediaObjects/41588_2025_2286_Fig2_HTML.png)

**Fig. 3: Performance of MMAA tools when analyzing 40 quantitative UK Biobank phenotypes.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02286-z/MediaObjects/41588_2025_2286_Fig3_HTML.png)

**Fig. 4: Accuracy of step 1 PGS and power of LDAK-KVIK-GBAT.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02286-z/MediaObjects/41588_2025_2286_Fig4_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-021-24485-y/MediaObjects/41467_2021_24485_Fig1_HTML.png)

### [Improved genetic prediction of complex traits from individual-level data or summary statistics](https://www.nature.com/articles/s41467-021-24485-y?fromPaywallRec=true)

Article Open access 07 July 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-021-00931-x/MediaObjects/41588_2021_931_Fig1_HTML.png)

### [A cross-population atlas of genetic associations for 220 human phenotypes](https://www.nature.com/articles/s41588-021-00931-x?fromPaywallRec=true)

Article 30 September 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-021-00870-7/MediaObjects/41588_2021_870_Fig1_HTML.png)

### [Computationally efficient whole-genome regression for quantitative and binary traits](https://www.nature.com/articles/s41588-021-00870-7?fromPaywallRec=true)

Article 20 May 2021

Data availability
-----------------

Our study used data from UK Biobank, which we applied for and downloaded from [www.ukbiobank.ac.uk](http://www.ukbiobank.ac.uk/). The UK Biobank has ethics approval from the North West Multi-centre Research Ethics Committee. The summary statistics and PGS SNP weightings from our enlarged analysis of 62 quantitative phenotypes can be downloaded from [www.ldak-kvik.com/summaries](http://www.ldak-kvik.com/summaries).

Code availability
-----------------

LDAK-KVIK is part of the software package LDAK. The latest version of LDAK can be downloaded from [www.dougspeed.com](http://www.dougspeed.com/) and the version used in this work is available via Zenodo at [https://doi.org/10.5281/zenodo.15747229](https://doi.org/10.5281/zenodo.15747229) (ref. [46](https://www.nature.com/articles/s41588-025-02286-z#ref-CR46 "Hof, J. P. & Speed, D. LDAK 6.1. Zenodo 
https://doi.org/10.5281/zenodo.15747229
(2025).")). Full instructions for running LDAK-KVIK are provided at [www.ldak-kvik.com](http://www.ldak-kvik.com/).

References
----------

1.  Uffelmann, E. et al. Genome-wide association studies. _Nat. Rev. Methods Prim._ **1**, 59 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVGnsLo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20studies&journal=Nat.%20Rev.%20Methods%20Prim.&volume=1&publication_year=2021&author=Uffelmann%2CE) 
    
2.  Visscher, P. M. et al. 10 years of GWAS discovery: biology, function, and translation. _Am. J. Hum. Genet_ **101**, 5–22 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28686856)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501872)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFChu7rI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=10%20years%20of%20GWAS%20discovery%3A%20biology%2C%20function%2C%20and%20translation&journal=Am.%20J.%20Hum.%20Genet&volume=101&pages=5-22&publication_year=2017&author=Visscher%2CPM) 
    
3.  Chen, Z. et al. China Kadoorie Biobank of 0.5 million people: survey methods, baseline characteristics and long-term follow-up. _Int J. Epidemiol._ **40**, 1652–1666 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22158673)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3235021)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=China%20Kadoorie%20Biobank%20of%200.5%20million%20people%3A%20survey%20methods%2C%20baseline%20characteristics%20and%20long-term%20follow-up&journal=Int%20J.%20Epidemiol.&volume=40&pages=1652-1666&publication_year=2011&author=Chen%2CZ) 
    
4.  Keaton, J. M. et al. Genome-wide analysis in over 1 million individuals of European ancestry yields improved polygenic risk scores for blood pressure traits. _Nat. Genet._ **56**, 778–791 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38689001)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11096100)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXpsFGrt7w%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20analysis%20in%20over%201%20million%20individuals%20of%20European%20ancestry%20yields%20improved%20polygenic%20risk%20scores%20for%20blood%20pressure%20traits&journal=Nat.%20Genet.&volume=56&pages=778-791&publication_year=2024&author=Keaton%2CJM) 
    
5.  Haines, J. L. et al. Complement factor H variant increases the risk of age-related macular degeneration. _Science_ **308**, 419–421 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15761120)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXjtFOntL8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complement%20factor%20H%20variant%20increases%20the%20risk%20of%20age-related%20macular%20degeneration&journal=Science&volume=308&pages=419-421&publication_year=2005&author=Haines%2CJL) 
    
6.  Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. _Nature_ **613**, 508–518 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36653562)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9849126)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslKmtrs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FinnGen%20provides%20genetic%20insights%20from%20a%20well-phenotyped%20isolated%20population&journal=Nature&volume=613&pages=508-518&publication_year=2023&author=Kurki%2CMI) 
    
7.  Leitsalu, L. et al. Cohort profile: Estonian Biobank of the Estonian Genome Center, University of Tartu. _Int. J. Epidemiol._ **44**, 1137–1147 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24518929)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cohort%20profile%3A%20Estonian%20Biobank%20of%20the%20Estonian%20Genome%20Center%2C%20University%20of%20Tartu&journal=Int.%20J.%20Epidemiol.&volume=44&pages=1137-1147&publication_year=2014&author=Leitsalu%2CL) 
    
8.  Lippert, C. et al. FaST linear mixed models for genome-wide association studies. _Nat. Methods_ **8**, 833–835 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21892150)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtFWku7nF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FaST%20linear%20mixed%20models%20for%20genome-wide%20association%20studies&journal=Nat.%20Methods&volume=8&pages=833-835&publication_year=2011&author=Lippert%2CC) 
    
9.  Zhou, X. & Stephens, M. Efficient multivariate linear mixed model algorithms for genome-wide association studies. _Nat. Methods_ **11**, 407–409 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24531419)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4211878)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXislCltrk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20multivariate%20linear%20mixed%20model%20algorithms%20for%20genome-wide%20association%20studies&journal=Nat.%20Methods&volume=11&pages=407-409&publication_year=2014&author=Zhou%2CX&author=Stephens%2CM) 
    
10.  Yang, J., Zaitlen, N. A., Goddard, M. E., Visscher, P. M. & Price, A. L. Advantages and pitfalls in the application of mixed-model association methods. _Nat. Genet._ **46**, 100–106 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24473328)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3989144)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Advantages%20and%20pitfalls%20in%20the%20application%20of%20mixed-model%20association%20methods&journal=Nat.%20Genet.&volume=46&pages=100-106&publication_year=2014&author=Yang%2CJ&author=Zaitlen%2CNA&author=Goddard%2CME&author=Visscher%2CPM&author=Price%2CAL) 
    
11.  Svishcheva, G. R., Axenovich, T. I., Belonogova, N. M., van Duijn, C. M. & Aulchenko, Y. S. Rapid variance components–based method for whole-genome association analysis. _Nat. Genet._ **44**, 1166–1170 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22983301)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlGjs7vF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20variance%20components%E2%80%93based%20method%20for%20whole-genome%20association%20analysis&journal=Nat.%20Genet.&volume=44&pages=1166-1170&publication_year=2012&author=Svishcheva%2CGR&author=Axenovich%2CTI&author=Belonogova%2CNM&author=Duijn%2CCM&author=Aulchenko%2CYS) 
    
12.  Loh, P.-R. et al. Efficient Bayesian mixed-model analysis increases association power in large cohorts. _Nat. Genet._ **47**, 284–290 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25642633)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342297)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFCntb4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20Bayesian%20mixed-model%20analysis%20increases%20association%20power%20in%20large%20cohorts&journal=Nat.%20Genet.&volume=47&pages=284-290&publication_year=2015&author=Loh%2CP-R) 
    
13.  Mbatchou, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. _Nat. Genet._ **53**, 1097–1103 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34017140)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFehsbrI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computationally%20efficient%20whole-genome%20regression%20for%20quantitative%20and%20binary%20traits&journal=Nat.%20Genet.&volume=53&pages=1097-1103&publication_year=2021&author=Mbatchou%2CJ) 
    
14.  Zhou, W. et al. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. _Nat. Genet._ **50**, 1335–1341 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30104761)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6119127)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFSqtbfN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficiently%20controlling%20for%20case-control%20imbalance%20and%20sample%20relatedness%20in%20large-scale%20genetic%20association%20studies&journal=Nat.%20Genet.&volume=50&pages=1335-1341&publication_year=2018&author=Zhou%2CW) 
    
15.  Loh, P. Mixed-model association for biobank-scale datasets. _Nat. Genet._ **50**, 906–908 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29892013)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6309610)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFWru7nE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mixed-model%20association%20for%20biobank-scale%20datasets&journal=Nat.%20Genet.&volume=50&pages=906-908&publication_year=2018&author=Loh%2CP) 
    
16.  Campos, A. I. et al. Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores. _Nat. Genet._ **55**, 1769–1776 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37723263)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvFemu7%2FF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Boosting%20the%20power%20of%20genome-wide%20association%20studies%20within%20and%20across%20ancestries%20by%20using%20polygenic%20scores&journal=Nat.%20Genet.&volume=55&pages=1769-1776&publication_year=2023&author=Campos%2CAI) 
    
17.  Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. _Nature_ [https://doi.org/10.1038/s41586-018-0579-z](https://doi.org/10.1038/s41586-018-0579-z) (2018).
    
18.  Sudlow, C. et al. The UK Biobank resource with deep phenotyping and genomic data. _PLoS Med._ **12**, e1001779 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25826379)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380465)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&journal=PLoS%20Med.&volume=12&publication_year=2015&author=Sudlow%2CC) 
    
19.  Jiang, L. et al. A resource-efficient tool for mixed model association analysis of large-scale data. _Nat. Genet._ **51**, 1749–1755 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31768069)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1OhsrfE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20resource-efficient%20tool%20for%20mixed%20model%20association%20analysis%20of%20large-scale%20data&journal=Nat.%20Genet.&volume=51&pages=1749-1755&publication_year=2019&author=Jiang%2CL) 
    
20.  Berrandou, T., Balding, D. & Speed, D. LDAK-GBAT: fast and powerful gene-based association testing using summary statistics. _Am. J. Hum. Genet._ **110**, 23–29 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36480927)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVKktrzM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LDAK-GBAT%3A%20fast%20and%20powerful%20gene-based%20association%20testing%20using%20summary%20statistics&journal=Am.%20J.%20Hum.%20Genet.&volume=110&pages=23-29&publication_year=2023&author=Berrandou%2CT&author=Balding%2CD&author=Speed%2CD) 
    
21.  MacKay, D. J. _Information Theory, Inference and Learning Algorithms_ (Cambridge Univ. Press, 2003).
    
22.  Dey, R., Schmidt, E. M., Abecasis, G. R. & Lee, S. A fast and accurate algorithm to test for binary phenotypes and its application to PheWAS. _Am. J. Hum. Genet_ **101**, 37–49 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28602423)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501775)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXpslSrurY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20fast%20and%20accurate%20algorithm%20to%20test%20for%20binary%20phenotypes%20and%20its%20application%20to%20PheWAS&journal=Am.%20J.%20Hum.%20Genet&volume=101&pages=37-49&publication_year=2017&author=Dey%2CR&author=Schmidt%2CEM&author=Abecasis%2CGR&author=Lee%2CS) 
    
23.  Ma, Y., Bi, W. & Zhang, J.-F. Empirical saddlepoint approximation and its application to genome-wide association studies. In _2021 40th Chinese Control Conference (CCC)_ 6380–6385 (IEEE, 2021).
    
24.  Bi, W., Fritsche, L. G., Mukherjee, B., Kim, S. & Lee, S. A fast and accurate method for genome-wide time-to-event data analysis and its application to UK Biobank. _Am. J. Hum. Genet._ **107**, 222–233 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32589924)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7413891)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1ymsrrP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20fast%20and%20accurate%20method%20for%20genome-wide%20time-to-event%20data%20analysis%20and%20its%20application%20to%20UK%20Biobank&journal=Am.%20J.%20Hum.%20Genet.&volume=107&pages=222-233&publication_year=2020&author=Bi%2CW&author=Fritsche%2CLG&author=Mukherjee%2CB&author=Kim%2CS&author=Lee%2CS) 
    
25.  Speed, D., Holmes, J. & Balding, D. Evaluating and improving heritability models using summary statistics. _Nat. Genet._ **52**, 458–462 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32203469)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlsVeit7Y%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluating%20and%20improving%20heritability%20models%20using%20summary%20statistics&journal=Nat.%20Genet.&volume=52&pages=458-462&publication_year=2020&author=Speed%2CD&author=Holmes%2CJ&author=Balding%2CD) 
    
26.  Zhang, Q., Privé, F., Vilhjálmsson, B. & Speed, D. Improved genetic prediction of complex traits from individual-level data or summary statistics. _Nat. Commun._ **12**, 4192 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34234142)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8263809)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFOnsrnK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improved%20genetic%20prediction%20of%20complex%20traits%20from%20individual-level%20data%20or%20summary%20statistics&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Zhang%2CQ&author=Priv%C3%A9%2CF&author=Vilhj%C3%A1lmsson%2CB&author=Speed%2CD) 
    
27.  Speed, D. et al. Reevaluation of SNP heritability in complex human traits. _Nat. Genet._ **49**, 986–992 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28530675)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5493198)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXotVOksrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reevaluation%20of%20SNP%20heritability%20in%20complex%20human%20traits&journal=Nat.%20Genet.&volume=49&pages=986-992&publication_year=2017&author=Speed%2CD) 
    
28.  Speed, D. & Balding, D. SumHer better estimates the SNP heritability of complex traits from summary statistics. _Nat. Genet._ **51**, 277–284 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30510236)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2iu7jO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SumHer%20better%20estimates%20the%20SNP%20heritability%20of%20complex%20traits%20from%20summary%20statistics&journal=Nat.%20Genet.&volume=51&pages=277-284&publication_year=2019&author=Speed%2CD&author=Balding%2CD) 
    
29.  Zou, H. & Hastie, T. Regularization and variable selection via the elastic net. _J. R. Stat. Soc. B_ **67**, 301–320 (2005).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regularization%20and%20variable%20selection%20via%20the%20elastic%20net&journal=J.%20R.%20Stat.%20Soc.%20B&volume=67&pages=301-320&publication_year=2005&author=Zou%2CH&author=Hastie%2CT) 
    
30.  _The ICD-10 Classification of Mental and Behavioural Disorders: Clinical Descriptions and Diagnostic Guidelines_, Vol. 1 (World Health Organization, 1992).
    
31.  de Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. _PLoS Comput. Biol._ **11**, e1004219 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25885710)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4401657)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAGMA%3A%20generalized%20gene-set%20analysis%20of%20GWAS%20data&journal=PLoS%20Comput.%20Biol.&volume=11&publication_year=2015&author=Leeuw%2CCA&author=Mooij%2CJM&author=Heskes%2CT&author=Posthuma%2CD) 
    
32.  Bakshi, A. et al. Fast set-based association analysis using summary data from GWAS identifies novel gene loci for human complex traits. _Sci. Rep._ **6**, 32894 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27604177)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5015118)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsV2isrvE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20set-based%20association%20analysis%20using%20summary%20data%20from%20GWAS%20identifies%20novel%20gene%20loci%20for%20human%20complex%20traits&journal=Sci.%20Rep.&volume=6&publication_year=2016&author=Bakshi%2CA) 
    
33.  Momin, M. M. et al. A method for an unbiased estimate of cross-ancestry genetic correlation using individual-level data. _Nat. Commun._ **14**, 722 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36759513)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9911789)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtlGksb4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20method%20for%20an%20unbiased%20estimate%20of%20cross-ancestry%20genetic%20correlation%20using%20individual-level%20data&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Momin%2CMM) 
    
34.  Zeng, J. et al. Signatures of negative selection in the genetic architecture of human complex traits. _Nat. Genet._ **50**, 746–753 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29662166)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXnvVSlt70%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Signatures%20of%20negative%20selection%20in%20the%20genetic%20architecture%20of%20human%20complex%20traits&journal=Nat.%20Genet.&volume=50&pages=746-753&publication_year=2018&author=Zeng%2CJ) 
    
35.  Schoech, A. et al. Quantification of frequency-dependent genetic architectures and action of negative selection in 25 UK Biobank traits. _Nat. Commun._ **10**, 790 (2019).
    
36.  Nagai, A. et al. Overview of the BioBank Japan Project: study design and profile. _J. Epidemiol._ **27**, S2–S8 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28189464)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5350590)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overview%20of%20the%20BioBank%20Japan%20Project%3A%20study%20design%20and%20profile&journal=J.%20Epidemiol.&volume=27&pages=S2-S8&publication_year=2017&author=Nagai%2CA) 
    
37.  Pruitt, K. D., Tatusova, T. & Maglott, D. R. NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. _Nucleic Acids Res._ **35**, D61–D65 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17130148)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXivFGkuw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NCBI%20reference%20sequences%20%28RefSeq%29%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&journal=Nucleic%20Acids%20Res.&volume=35&pages=D61-D65&publication_year=2007&author=Pruitt%2CKD&author=Tatusova%2CT&author=Maglott%2CDR) 
    
38.  Pain, O. et al. Evaluation of polygenic prediction methodology within a reference-standardized framework. _PLoS Genet._ **17**, e1009021 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33945532)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8121285)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFaltLbP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluation%20of%20polygenic%20prediction%20methodology%20within%20a%20reference-standardized%20framework&journal=PLoS%20Genet.&volume=17&publication_year=2021&author=Pain%2CO) 
    
39.  Pain, O., Al-Chalabi, A. & Lewis, C. M. The GenoPred pipeline: a comprehensive and scalable pipeline for polygenic scoring. _Bioinformatics_ **40**, btae551 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39292536)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11462442)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXitFSjsrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20GenoPred%20pipeline%3A%20a%20comprehensive%20and%20scalable%20pipeline%20for%20polygenic%20scoring&journal=Bioinformatics&volume=40&publication_year=2024&author=Pain%2CO&author=Al-Chalabi%2CA&author=Lewis%2CCM) 
    
40.  Mak, T., Porsch, R., Choi, S., Zhou, X. & Sham, P. Polygenic scores via penalized regression on summary statistics. _Genet. Epidemiol._ **41**, 469–480 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28480976)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20scores%20via%20penalized%20regression%20on%20summary%20statistics&journal=Genet.%20Epidemiol.&volume=41&pages=469-480&publication_year=2017&author=Mak%2CT&author=Porsch%2CR&author=Choi%2CS&author=Zhou%2CX&author=Sham%2CP) 
    
41.  Yang, S. & Zhou, X. Accurate and scalable construction of polygenic scores in large biobank data sets. _Am. J. Hum. Genet._ **106**, 679–693 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32330416)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7212266)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotlSnt7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20and%20scalable%20construction%20of%20polygenic%20scores%20in%20large%20biobank%20data%20sets&journal=Am.%20J.%20Hum.%20Genet.&volume=106&pages=679-693&publication_year=2020&author=Yang%2CS&author=Zhou%2CX) 
    
42.  Privé, F., Arbel, J. & Vilhjálmsson, B. LDpred2: better, faster, stronger. _Bioinformatics_ **36**, 5424–5431 (2021).
    
43.  Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. _Nat. Commun._ **10**, 1776 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30992449)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467998)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20prediction%20via%20Bayesian%20regression%20and%20continuous%20shrinkage%20priors&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Ge%2CT&author=Chen%2CC-Y&author=Ni%2CY&author=Feng%2CY-CA&author=Smoller%2CJW) 
    
44.  Pazokitoroudi, A. et al. Efficient variance components analysis across millions of genomes. _Nat. Commun._ **11**, 4020 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32782262)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7419517)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsF2qtrjP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20variance%20components%20analysis%20across%20millions%20of%20genomes&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Pazokitoroudi%2CA) 
    
45.  Tseng, P. Convergence of a block coordinate descent method for nondifferentiable minimization. _J. Optim. Theory Appl._ **109**, 475–494 (2001).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Convergence%20of%20a%20block%20coordinate%20descent%20method%20for%20nondifferentiable%20minimization&journal=J.%20Optim.%20Theory%20Appl.&volume=109&pages=475-494&publication_year=2001&author=Tseng%2CP) 
    
46.  Hof, J. P. & Speed, D. LDAK 6.1. _Zenodo_ [https://doi.org/10.5281/zenodo.15747229](https://doi.org/10.5281/zenodo.15747229) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02286-z?format=refman&flavour=references)

Acknowledgements
----------------

This research was conducted using the UK Biobank Resource under application number 21432. The computing for this project was performed on the GenomeDK cluster (Aarhus University). We thank A. Halager and D. Søndergaard (Aarhus University) for programming suggestions; D. Balding (University of Melbourne) for helpful comments on the manuscript; J. Bajzik, A. Depope and M. Robinson (Institute of Science and Technology Austria) for assistance with the UK Biobank Research Analysis Platform; H. Heydari (University of Toronto) for advice on heritability estimation; and S. Jin (Karolinska Institute) for testing LDAK-KVIK. D.S. is supported by the Aarhus University Research Foundation (AUFF), by the Independent Research Fund Denmark (project no. 7025-00094B) and by a European Research Council Consolidator Grant (ID 101088901, acronym ClassifyDiseases).

Author information
------------------

### Authors and Affiliations

1.  Radboud University Medical Center, IQ Health Science Department, Nijmegen, the Netherlands
    
    Jasper P. Hof
    
2.  Aarhus University, Center for Quantitative Genetics and Genomics, Aarhus, Denmark
    
    Jasper P. Hof & Doug Speed
    

Authors

1.  Jasper P. Hof
    
    [View author publications](https://www.nature.com/search?author=Jasper%20P.%20Hof)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jasper%20P.%20Hof) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jasper%20P.%20Hof%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Doug Speed
    
    [View author publications](https://www.nature.com/search?author=Doug%20Speed)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Doug%20Speed) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Doug%20Speed%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

J.H. and D.S. jointly developed the software, performed the analysis and wrote the manuscript.

### Corresponding author

Correspondence to [Doug Speed](mailto:doug@qgg.au.dk).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Longda Jiang, Po-Ru Loh and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02286-z#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02286-z/MediaObjects/41588_2025_2286_MOESM1_ESM.pdf)

Supplementary Notes 1–10, Figs. 1–46 and Tables 1–9.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02286-z/MediaObjects/41588_2025_2286_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02286-z/MediaObjects/41588_2025_2286_MOESM3_ESM.pdf)

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=LDAK-KVIK%20performs%20fast%20and%20powerful%20mixed-model%20association%20analysis%20of%20quantitative%20and%20binary%20phenotypes&author=Jasper%20P.%20Hof%20et%20al&contentID=10.1038%2Fs41588-025-02286-z&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-11&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02286-z)

### Cite this article

Hof, J.P., Speed, D. LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02286-z

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02286-z?format=refman&flavour=citation)

-   Received: 08 July 2024
    
-   Accepted: 30 June 2025
    
-   Published: 11 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02286-z
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
