> id: https://www.nature.com/articles/s41588-025-02306-y

> link: https://www.nature.com/articles/s41588-025-02306-y

> title: Inactivation of -13-glucan synthase-like 5 confers broad-spectrum resistance to Plasmodiophora brassicae pathotypes in cruciferous plants

# Inactivation of -13-glucan synthase-like 5 confers broad-spectrum resistance to Plasmodiophora brassicae pathotypes in cruciferous plants
_Published on Mon, 01 Sep 2025 00:00:00 GMT_

Abstract
--------

Clubroot disease, caused by the obligate intracellular rhizarian protist _Plasmodiophora brassicae_, is devastating to cruciferous crops worldwide. Widespread field _P._ _brassicae_ pathotypes frequently overcome the pathotype-specific resistance of modern varieties, posing a challenge for durable control of this disease. Here a genome-wide association study of 3 years of data comprising field clubroot phenotyping of 244 genome-resequenced _Brassica napus_ accessions identified a strong association of _β-1,3-glucan synthase-like 5_ (_GSL5_) with clubroot susceptibility. _GSL5_ was evolutionarily conserved, and inactivation of _GSL5_ by genome editing in _Arabidopsis_, _B._ _napus_, _Brassica_ _rapa_ and _Brassica oleracea_ conferred broad-spectrum, high-level resistance to _P._ _brassicae_ pathotypes without yield penalties in _B._ _napus_. _GSL5_ inactivation derepressed the jasmonic acid-mediated immunity during _P._ _brassicae_ secondary infection, and this immune repression was possibly reinforced through stabilization of GSL5 by a _P._ _brassicae_ effector, facilitating clubroot susceptibility. Our study provides durable resistance resources for cruciferous clubroot disease control and insights into plant resistance against intracellular eukaryotic phytopathogens.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-93327-0/MediaObjects/41598_2021_93327_Fig1_HTML.png)

### [Clubroot resistance derived from the European _Brassica napus_ cv. ‘Tosca’ is not effective against virulent _Plasmodiophora brassicae_ isolates from Alberta, Canada](https://www.nature.com/articles/s41598-021-93327-0?fromPaywallRec=false)

Article Open access 14 July 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-85836-9/MediaObjects/41598_2021_85836_Fig1_HTML.png)

### [Identification of resistance loci against new pathotypes of _Plasmodiophora brassicae_ in _Brassica napus_ based on genome-wide association mapping](https://www.nature.com/articles/s41598-021-85836-9?fromPaywallRec=false)

Article Open access 23 March 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-024-64795-x/MediaObjects/41598_2024_64795_Fig1_HTML.png)

### [Resynthesizing _Brassica napus_ with race specific resistance genes and race non-specific QTLs to multiple races of _Plasmodiophora brassicae_](https://www.nature.com/articles/s41598-024-64795-x?fromPaywallRec=false)

Article Open access 25 June 2024

Main
----

Cruciferous crops are important both economically and nutritionally, providing products such as vegetables, edible oils and proteins. However, their production is threatened by clubroot disease, which has been rapidly spreading in recent years in more than 80 countries and has resulted in annual yield losses of 10–15% worldwide[1](https://www.nature.com/articles/s41588-025-02306-y#ref-CR1 "Javed, M. A. et al. The clubroot pathogen Plasmodiophora brassicae: a profile update. Mol. Plant Pathol. 24, 89–106 (2023)."),[2](https://www.nature.com/articles/s41588-025-02306-y#ref-CR2 "Strelkov, S. E. & Hwang, S. F. Clubroot in the Canadian canola crop: 10 years into the outbreak. Can. J. Plant. Pathol. 36, 27–36 (2014)."),[3](https://www.nature.com/articles/s41588-025-02306-y#ref-CR3 "Chai, A. L., Xie, X. W., Shi, Y. X. & Li, B. J. Research status of clubroot (Plasmodiophora brassicae) on cruciferous crops in China. Can. J. Plant Pathol. 36, 142–153 (2014)."). This disease is caused by the rhizarian protist _Plasmodiophora_ _brassicae_, which, distinct from most known phytopathogens that are extracellular, is an obligate intracellular parasite[4](https://www.nature.com/articles/s41588-025-02306-y#ref-CR4 "Aist, J. R. & Williams, P. The cytology and kinetics of cabbage root hair penetration by Plasmodiophora brassicae. Can. J. Bot. 49, 2023–2034 (1971)."),[5](https://www.nature.com/articles/s41588-025-02306-y#ref-CR5 "Ingram, D. & Tommerup, I. C. The life history of Plasmodiophora brassicae Woron. Proc. R. Soc. Lond. B 180, 103–112 (1972)."),[6](https://www.nature.com/articles/s41588-025-02306-y#ref-CR6 "Liu, L. et al. Refining the life cycle of Plasmodiophora brassicae. Phytopathology 110, 1704–1712 (2020).") and exclusively jeopardizes cruciferous species. The _P._ _brassicae_ life cycle is rather complicated and unique, and can be divided into three life stages: the in vitro resting spore phase in the soil (the life form for long-term survival), the primary infection phase in the root epidermis tissue (1–7 days post inoculation (dpi)) and the secondary infection phase in the root cortex tissue (8–24 dpi)[5](https://www.nature.com/articles/s41588-025-02306-y#ref-CR5 "Ingram, D. & Tommerup, I. C. The life history of Plasmodiophora brassicae Woron. Proc. R. Soc. Lond. B 180, 103–112 (1972)."),[6](https://www.nature.com/articles/s41588-025-02306-y#ref-CR6 "Liu, L. et al. Refining the life cycle of Plasmodiophora brassicae. Phytopathology 110, 1704–1712 (2020)."). To establish the primary infection inside root epidermal cells, the primary zoospore germinates from the resting spore in the soil, finds the host, develops the cyst on the root surface, punctures the cell wall and injects the _P._ _brassicae_ parasite into the root epidermal cell[4](https://www.nature.com/articles/s41588-025-02306-y#ref-CR4 "Aist, J. R. & Williams, P. The cytology and kinetics of cabbage root hair penetration by Plasmodiophora brassicae. Can. J. Bot. 49, 2023–2034 (1971)."). Establishment of the secondary infection inside the root cortical cells by the secondary zoospores is thought to be carried out through a similar penetration process[4](https://www.nature.com/articles/s41588-025-02306-y#ref-CR4 "Aist, J. R. & Williams, P. The cytology and kinetics of cabbage root hair penetration by Plasmodiophora brassicae. Can. J. Bot. 49, 2023–2034 (1971)."). Multiplication of the _P._ _brassicae_ parasites during secondary infection causes cortical cell hyperplasia and hypertrophy and the formation of root galls, ultimately destroying the normal root architecture and killing the plant[5](https://www.nature.com/articles/s41588-025-02306-y#ref-CR5 "Ingram, D. & Tommerup, I. C. The life history of Plasmodiophora brassicae Woron. Proc. R. Soc. Lond. B 180, 103–112 (1972)."),[6](https://www.nature.com/articles/s41588-025-02306-y#ref-CR6 "Liu, L. et al. Refining the life cycle of Plasmodiophora brassicae. Phytopathology 110, 1704–1712 (2020)."),[7](https://www.nature.com/articles/s41588-025-02306-y#ref-CR7 "Liu, L. et al. Comparing the infection biology of Plasmodiophora brassicae in clubroot susceptible and resistant hosts and non-hosts. Front. Microbiol. 11, 507036 (2020)."). As a representative of unique and intracellular phytopathogens, the underlying mechanisms of interaction between _P._ _brassicae_ and the host plant are poorly understood.

The most cost-effective and efficient approach for controlling clubroot disease is to explore resistant genes for use in breeding. The _Brassica_ germplasm has been extensively screened for clubroot resistance, and only a limited number of germplasm sources have been found to carry complete clubroot resistance. European fodder turnips (_Brassica_ _rapa_ subsp. _rapifera_) in particular are used as the major source for clubroot-resistance breeding[8](https://www.nature.com/articles/s41588-025-02306-y#ref-CR8 "Hasan, J., Megha, S. & Rahman, H. Clubroot in Brassica: recent advances in genomics, breeding, and disease management. Genome 64, 735–760 (2021)."),[9](https://www.nature.com/articles/s41588-025-02306-y#ref-CR9 "Mehraj, H. et al. Genetics of clubroot and Fusarium wilt disease resistance in Brassica vegetables: the application of marker assisted breeding for disease resistance. Plants (Basel) 9, 726 (2020)."),[10](https://www.nature.com/articles/s41588-025-02306-y#ref-CR10 "Lai, S. et al. Clubroot resistant in cruciferous crops: recent advances in genes and QTLs identification and utilization. Hortic. Res. 12, uhaf105 (2025)."). Presently, about 39 major clubroot-resistant loci have been reported in various _Brassica_ species, and most of them are dominant and pathotype-specific[10](https://www.nature.com/articles/s41588-025-02306-y#ref-CR10 "Lai, S. et al. Clubroot resistant in cruciferous crops: recent advances in genes and QTLs identification and utilization. Hortic. Res. 12, uhaf105 (2025)."). Among them, three clubroot-resistance genes, _CRa_ (or _CRA3.7.1_), _Crr1a_ and _CRA8.2.4_, have been isolated, all encoding classical TNL (toll/interleukin-1 receptor, nucleotide-binding site, leucine-rich repeat) type proteins[11](https://www.nature.com/articles/s41588-025-02306-y#ref-CR11 "Ueno, H. et al. Molecular characterization of the CRa gene conferring clubroot resistance in Brassica rapa. Plant Mol. Biol. 80, 621–629 (2012)."),[12](https://www.nature.com/articles/s41588-025-02306-y#ref-CR12 "Hatakeyama, K. et al. Identification and characterization of Crr1a, a gene for resistance to clubroot disease (Plasmodiophora brassicae Woronin) in Brassica rapa L. PLoS ONE 8, e54745 (2013)."),[13](https://www.nature.com/articles/s41588-025-02306-y#ref-CR13 "Yang, Z. et al. R gene triplication confers European fodder turnip with improved clubroot resistance. Plant Biotechnol. J. 20, 1502–1517 (2022)."), but the resistance mechanisms are unclear. Moreover, field _P._ _brassicae_ populations are highly heterogeneous and specialized into diverse pathotypes[14](https://www.nature.com/articles/s41588-025-02306-y#ref-CR14 "Williams, P. H. A system for the determination of races of Plasmodiophora brassicae that infect cabbage and rutabaga. Phytopathology 56, 624–626 (1966)."),[15](https://www.nature.com/articles/s41588-025-02306-y#ref-CR15 "Buczacki, S. et al. Study of physiologic specialization in Plasmodiophora brassicae: proposals for attempted rationalization through an international approach. Trans. Br. Mycological Soc. 65, 295–303 (1975)."),[16](https://www.nature.com/articles/s41588-025-02306-y#ref-CR16 "Zamani-Noor, N. et al. Pathotype characterization of Plasmodiophora brassicae, the cause of clubroot in central Europe and Sweden (2016–2020). Pathogens 11, 1440 (2022)."),[17](https://www.nature.com/articles/s41588-025-02306-y#ref-CR17 "Zeng, L. et al. Pathotype characterization of Plasmodiophora brassicae by European clubroot differential and Williams sets in China. Plant Dis. 108, 847–851 (2023)."),[18](https://www.nature.com/articles/s41588-025-02306-y#ref-CR18 "Hollman, K. B., Hwang, S. F., Manolii, V. P. & Strelkov, S. E. Pathotypes of Plasmodiophora brassicae collected from clubroot resistant canola (Brassica napus L.) cultivars in western Canada in 2017–2018. Can. J. Plant Pathol. 43, 622–630 (2021)."); for example, up to 26 pathotypes have been identified from 33 _P._ _brassicae_ field isolates in China[17](https://www.nature.com/articles/s41588-025-02306-y#ref-CR17 "Zeng, L. et al. Pathotype characterization of Plasmodiophora brassicae by European clubroot differential and Williams sets in China. Plant Dis. 108, 847–851 (2023)."), and loss of pathotype-specific resistance in modern varieties has been frequently reported within 3–5 years after planting in diseased fields[18](https://www.nature.com/articles/s41588-025-02306-y#ref-CR18 "Hollman, K. B., Hwang, S. F., Manolii, V. P. & Strelkov, S. E. Pathotypes of Plasmodiophora brassicae collected from clubroot resistant canola (Brassica napus L.) cultivars in western Canada in 2017–2018. Can. J. Plant Pathol. 43, 622–630 (2021)."),[19](https://www.nature.com/articles/s41588-025-02306-y#ref-CR19 "Strelkov, S. E., Hwang, S. F., Manolii, V. P., Cao, T. & Feindel, D. Emergence of new virulence phenotypes of Plasmodiophora brassicae on canola (Brassica napus) in Alberta, Canada. Eur. J. Plant Pathol. 145, 517–529 (2016)."). This has posed a great challenge for cruciferous disease-resistance breeding to efficiently and durably control clubroot disease. Broad-spectrum resistance is not easily overcome by _P._ _brassicae_ pathotypes and thus is desirable for breeding in cruciferous crops, but no single gene conferring broad-spectrum, high-level clubroot resistance has been reported in cruciferous species.

In this study, we report a conserved clubroot-susceptible gene, _GSL5_, in cruciferous species. The inactivation of _GSL5_ conferred broad-spectrum and high-level resistance to _P._ _brassicae_ pathotypes in four different cruciferous species. We further revealed that _GSL5_\-mediated susceptibility is possibly caused by its stabilization by a _P._ _brassicae_ effector and suppression of the jasmonic acid (JA) immune pathway. _GSL5_ inactivation derepresses this immune response and results in JA-dependent resistance during _P._ _brassicae_ secondary infection.

Results
-------

### Association of _GSL5_ with clubroot susceptibility

We used a genome-wide association study (GWAS) to detect loci of resistance or susceptibility to clubroot. We selected 244 _Brassica_ _napus_ accessions from our collection pool to construct a panel optimized for population structure, ensuring representative worldwide geographic distribution and diversity of the panel accessions (Fig. [1a](https://www.nature.com/articles/s41588-025-02306-y#Fig1), Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02306-y#Fig7) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)). All accessions underwent genome resequencing with at least 10× coverage of the _B._ _napus_ genome, yielding 2,797,642 high-quality single-nucleotide polymorphisms (SNPs)[20](https://www.nature.com/articles/s41588-025-02306-y#ref-CR20 "Zhang, Y. et al. Structural variation reshapes population gene expression and trait variation in 2,105 Brassica napus accessions. Nat. Genet. 56, 2538–2550 (2024)."). This panel of _B._ _napus_ accessions was evaluated for clubroot resistance or susceptibility in diseased fields in Zhijiang county, Hubei province and in Taiping county, Anhui province, from 2016 to 2018. The predominant _P._ _brassicae_ pathotype was determined to be ECD16/30/31 in Zhijiang county and ECD16/7/12 in Taiping county[17](https://www.nature.com/articles/s41588-025-02306-y#ref-CR17 "Zeng, L. et al. Pathotype characterization of Plasmodiophora brassicae by European clubroot differential and Williams sets in China. Plant Dis. 108, 847–851 (2023)."). Disease scales 0–3 were used to score the disease severity of each _B._ _napus_ accession (Fig. [1b](https://www.nature.com/articles/s41588-025-02306-y#Fig1)) and to generate a disease index (DI) to evaluate their resistance or susceptibility levels[17](https://www.nature.com/articles/s41588-025-02306-y#ref-CR17 "Zeng, L. et al. Pathotype characterization of Plasmodiophora brassicae by European clubroot differential and Williams sets in China. Plant Dis. 108, 847–851 (2023)."). Although the DI of all 244 _B._ _napus_ accessions displayed a normal distribution in each of the 3 years, most were found to be susceptible (DI ≥ 30), and accessions with a high level of resistance that would be valuable for crop breeding (DI ≤ 10) were rare (Fig. [1c](https://www.nature.com/articles/s41588-025-02306-y#Fig1)). The best linear unbiased prediction values[21](https://www.nature.com/articles/s41588-025-02306-y#ref-CR21 "Ren, D., An, L., Li, B., Qiao, L. & Liu, W. Efficient weighting methods for genomic best linear-unbiased prediction (BLUP) adapted to the genetic architectures of quantitative traits. Heredity 126, 320–334 (2021).") were generated using the DI over all 3 years for each accession. Using the above SNPs as the genotypic data and the best linear unbiased prediction values as phenotypic data, we performed a GWAS using three different models and detected five significantly associated loci, the most significant of which contained four SNPs on chromosome A09 that were repeatedly detected (Fig. [1d](https://www.nature.com/articles/s41588-025-02306-y#Fig1) and Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02306-y#Fig8)). We found that all four significant SNPs were located at the first exon of _BnaA09g01630D_ (Fig. [1e](https://www.nature.com/articles/s41588-025-02306-y#Fig1)), which was annotated as _β-1,3-glucan synthase-like_ 5 (_BnaA09.GSL5_). These four SNPs could divide most of the accessions into two haplotype groups (Hap\_1 and Hap\_2), which showed significant differences in susceptibility levels across phenotyping data from all 3 years (Fig. [1f](https://www.nature.com/articles/s41588-025-02306-y#Fig1)). Among them, SNP 885,826 resulted in an amino acid change from leucine to phenylalanine in the BnaA09.GSL5 protein, while the other three SNPs were synonymous (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)). Notably, the allotetraploid _B._ _napus_ genome encompassed another syntenic _GSL5_ (_BnaC09.GSL5_, _BnaC09g00800D_) in the C subgenome; however, there were no SNPs in its genome locus in any of the 244 accessions (Supplementary Fig. [1a](https://www.nature.com/articles/s41588-025-02306-y#MOESM1)), explaining why _BnaC09.GSL5_ was not detected by the GWAS. Importantly, transcriptomic data from 12 different _B._ _napus_ tissues[22](https://www.nature.com/articles/s41588-025-02306-y#ref-CR22 "Wang, W. et al. WeiTsing, a pericycle-expressed ion channel, safeguards the stele to confer clubroot resistance. Cell 186, 2656–2671.e18 (2023).") revealed that both _BnaA09.GSL5_ and _BnaC09.GSL5_ had the highest expression levels in the root tissues (Supplementary Fig. [1b](https://www.nature.com/articles/s41588-025-02306-y#MOESM1)), indicating an important role in the roots of _B._ _napus_. In _Arabidopsis_, _GSL5_, also known as _Powdery Mildew Resistance 4_ (_PMR4_) or _Callose synthase like 12_ (_CASL12_), was responsible for callose synthesis at the wounding or pathogen-infection sites, which strengthened the cell wall but negatively controlled powdery mildew resistance in a salicylic acid-dependent manner[23](https://www.nature.com/articles/s41588-025-02306-y#ref-CR23 "Nishimura, M. T. et al. Loss of a callose synthase results in salicylic acid-dependent disease resistance. Science 301, 969–972 (2003)."),[24](https://www.nature.com/articles/s41588-025-02306-y#ref-CR24 "Jacobs, A. K. et al. An Arabidopsis callose synthase, GSL5, is required for wound and papillary callose formation. Plant Cell 15, 2503–2513 (2003)."). These collective data indicate a role of _GSL5_ in clubroot susceptibility in _B._ _napus_ and led us to further investigate its functions.

**Fig. 1: GWAS of field clubroot phenotyping in the _B._ _napus_ accessions.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02306-y/figures/1)

**a**, Worldwide geographic distribution of 244 _B._ _napus_ accessions. The map was generated with the map data function in the ggplot2 package. **b**, Standard disease scales scored for clubroot disease severity: 0, no clubs; 1, a few small clubs present on less than one-third of the lateral roots or on the main root (a club present on the lateral root is indicated with red arrowhead); 2, moderate clubs present on one-third to two-thirds of the lateral roots or the main root; 3, severe clubs on the main root or present on more than two-thirds of the lateral roots. Scale bars, 2 cm. **c**, The disease index distribution of 244 _B._ _napus_ accessions in 2 provinces from 2016 to 2018. ZJ, Zhijiang county (Hubei province); TP, Taiping county (Anhui province). **d**, Manhattan plot of the GWAS. The dashed red line indicates the Bonferroni-adjusted significance threshold (_P_ = 1.8 × 10−8). The arrow indicates the significantly associated locus. **e**, _BnaA09.GSL5_\-based association mapping and pairwise linkage disequilibrium analysis. In the top panel, the lead SNPs are highlighted with enlarged dots and diamond, and the other SNPs are colored according to their linkage disequilibrium (_R_2 value) with the lead SNP. In the bottom panel, the SNPs showing strong linkage disequilibrium with the lead SNP are outlined with a black triangle. A genomic region from 885,240 bp to 891,443 bp on chromosome A09 was mapped and located at the genomic region of _BnaA09g01630D_ (_BnaA09.GSL5_). **f**, Haplotypes (Hap) of _BnaA09.GSL5_, defined by _B._ _napus_ natural SNPs. _n_ denotes the number of genotypes belonging to each haplotype group. Statistical significance was determined using a two-sided Student’s _t-_test. The disease index distribution of each haplotype group is displayed as a box and whisker plot in which the whiskers extend to the 10th and 90th percentiles. Center line, median; box limits, 25th and 75th percentiles.

### _GSL5_ inactivation confers broad-spectrum clubroot resistance

_GSL5_ orthologs from representative cruciferous species, including _Arabidopsis_ _thaliana_, _B._ _napus_, _B._ _rapa_, _Brassica_ _oleracea_ and _Raphanus sativus_, were collected and analyzed, and we found that they shared highly similar sequence architecture and size at both the gene and protein levels (Fig. [2a,b](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). The _GSL5_ gene from these five species contained three major exons and two introns of about 5,700 bp in length, while the GSL5 proteins were annotated with two separated putative glucan synthase subunits and up to 16 transmembrane domains of approximately 1,780 amino acids in length, sharing more than 91% sequence identity (Fig. [2b](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). In 30 genome-sequenced cruciferous species, the GSL5 proteins also shared more than 89% sequence identity (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)), indicating that GSL5 proteins are highly conserved in cruciferous species. In _Arabidopsis_, GSL5 was predominantly localized at the plasma membrane of leaf cells[25](https://www.nature.com/articles/s41588-025-02306-y#ref-CR25 "Ellinger, D. et al. Elevated early callose deposition results in complete penetration resistance to powdery mildew in Arabidopsis. Plant Physiol. 161, 1433–1444 (2013)."),[26](https://www.nature.com/articles/s41588-025-02306-y#ref-CR26 "Liu, L. et al. The plant trans-Golgi network component ECHIDNA regulates defense, cell death, and endoplasmic reticulum stress. Plant Physiol. 191, 558–574 (2023)."). To verify the function of _GSL5_ in regulating clubroot resistance or susceptibility, an _Arabidopsis_ mutant, _gsl5_ (also known as _pmr4-1_), which introduces a stop codon into the second exon of _GSL5_ (ref. [23](https://www.nature.com/articles/s41588-025-02306-y#ref-CR23 "Nishimura, M. T. et al. Loss of a callose synthase results in salicylic acid-dependent disease resistance. Science 301, 969–972 (2003).")), was used for the clubroot resistance test. The wild-type plants (Col-0) were highly susceptible to _P._ _brassicae_, characterized by wilting or death of the overground parts and hypertrophy throughout the root system at 24 dpi (Fig. [2c,d](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). By contrast, the _gsl5_ plants exhibited a high level of resistance to clubroot disease, with no visible disease symptoms on either the shoots or the roots (Fig. [2c,d](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). We then performed genome editing to knock out _GSL5_ from _Arabidopsis_ and obtained the homozygous mutant _gsl5-KO_, in which a 248 bp sequence was deleted in the first exon of _GSL5_ (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02306-y#Fig9)). A clubroot-resistance assay showed that the _gsl5-KO_ mutant conferred a high level of resistance to _P._ _brassicae_, comparable to that of the _gsl5_ mutant at 24 dpi (Fig. [2e,f](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). Complementation of the _gsl5_ mutant by introducing the _GSL5_ genomic region of _Arabidopsis_ restored clubroot susceptibility to the wild-type level, while _GSL5-_overexpressing plants, as expected, were highly susceptible (Fig. [2e,f](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). During _P._ _brassicae_ secondary infection in cortex tissue, _GSL5_ displayed increasing expression levels (Supplementary Fig. [1c](https://www.nature.com/articles/s41588-025-02306-y#MOESM1)). Therefore, _GSL5_ performs as a susceptibility gene that is required for _P._ _brassicae_ virulence in _Arabidopsis_. To verify the functions of _BnaA09.GSL5 Hap\_1_ and _BnaA09.GSL5 Hap\_2_, we cloned their genomic regions and introduced them into the _Arabidopsis_ mutant _gsl5_. We found that expression of the susceptible natural variant _BnaA09.GSL5 Hap\_1_ restored _gsl5_ clubroot susceptibility to the wild-type level at 24 dpi (Fig. [2g,h](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). By contrast, expression of the natural variant _BnaA09.GSL5 Hap\_2_, which is relatively less susceptible, restored _gsl5_ susceptibility to _P._ _brassicae_ with a significantly lower DI than that of _gsl5_ plants expressing _BnaA09.GSL5 Hap\_1_ at 24 dpi (Fig. [2g,h](https://www.nature.com/articles/s41588-025-02306-y#Fig2)). Furthermore, all transgenes of the _GSL5_ genomic regions from _B._ _napus_ C subgenome, _B._ _rapa_, _B._ _oleracea_ and _R._ _sativus_ introduced into the _Arabidopsis_ mutant _gsl5_ functionally complemented the _Arabidopsis_ _gsl5_ mutant and restored clubroot susceptibility (Fig. [2g,h](https://www.nature.com/articles/s41588-025-02306-y#Fig2)), supporting a conserved role of _GSL5_ for clubroot susceptibility in different cruciferous species. Importantly, after inoculation with 36 field _P._ _brassicae_ isolates representing 26 pathotypes epidemic to China[17](https://www.nature.com/articles/s41588-025-02306-y#ref-CR17 "Zeng, L. et al. Pathotype characterization of Plasmodiophora brassicae by European clubroot differential and Williams sets in China. Plant Dis. 108, 847–851 (2023)."), the _Arabidopsis gsl5_ mutant exhibited broad-spectrum and high-level clubroot resistance (Fig. [2i](https://www.nature.com/articles/s41588-025-02306-y#Fig2) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)). We then performed genome editing to knock out _GSL5_ from _B._ _napus_ (two elite varieties, Westar and Zhongshuang 11), _B._ _rapa_ (var. _chinensis_) and _B._ _oleracea_ (var. _capitata_) (Fig. [3a](https://www.nature.com/articles/s41588-025-02306-y#Fig3)). Homozygous _gsl5_ plants of the above three cruciferous crops were verified with PCR and sequencing (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)) and subjected to a clubroot-resistance assay. As expected, all _gsl5_ plants of _B._ _napus_, _B._ _rapa_ and _B._ _oleracea_ exhibited strong resistance to clubroot disease (Fig. [3b,c](https://www.nature.com/articles/s41588-025-02306-y#Fig3)). Moreover, the _gsl5_ plants of _B._ _napus_ var. ZS11 were highly resistant to all 17 tested field _P._ _brassicae_ isolates representing 15 pathotypes (Fig. [3d](https://www.nature.com/articles/s41588-025-02306-y#Fig3)). Owing to high self-incompatibility, only a few homozygous _gsl5_ seeds of _B._ _rapa_ and _B._ _oleracea_ were obtained and tested, with only seven _P._ _brassicae_ isolates (seven pathotypes). Indeed, they were highly resistant to all seven tested isolates (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02306-y#Fig10)). Taken together, _GSL5_ is an evolutionarily conserved clubroot-susceptible gene in cruciferous species, and inactivation of _GSL5_ by genome editing can confer broad-spectrum and high-level clubroot resistance.

**Fig. 2: Functional validation of cruciferous _GSL5_s in _Arabidopsis_.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02306-y/figures/2)

**a**, Diagram representation of _GSL5_ genes from _A._ _thaliana_, _B._ _napus_, _B._ _rapa_, _B._ _oleracea_ and _R._ _sativus_. The allotetraploid _B._ _napus_ contains two _GSL5_s in its genome. nt, nucleotide. **b**, Diagram representation of GSL5 proteins from the above five cruciferous species. aa, amino acid; TMs, transmembrane domains; FKS1, FKS1-like 1,3-beta-glucan synthase domain. **c**, Clubroot resistance test of the _Arabidopsis_ _gsl5_ (_pmr4-1_) mutant. Wild-type plants were used as the susceptible control. Representative photos were taken at 24 dpi with _P._ _brassicae_. **d**, Disease index values of wild-type and _gsl5_ mutant plants. **e**, Genetic validation of _GSL5_ for clubroot susceptibility in _Arabidopsis_. _gsl5-KO_, the _gsl5_ mutant generated by genome editing; _GSL5-OE_, _GSL5_\-overexpressing plants; _gsl5 GSL5_, the complemented plants generated by expressing the _GSL5_ genomic region in the _gsl5_ mutant. Representative root photos were taken at 24 dpi. **f**, Disease index values of wild-type, _gsl5_, _gsl5-KO_, _GSL5-OE_ and _gsl5 GSL5_ mutant plants. **g**, Genetic validation of _GSL5_s from four cruciferous crops for clubroot susceptibility in _Arabidopsis_. Representative root photos were taken at 24 dpi. Hap\_1, _BnaA09.GSL5_ Hap\_1; Hap\_2, _BnaA09.GSL5_ Hap\_2; _BnaC09.GSL5_, the _GSL5_ from the C subgenome of _B. napus_; _BraGSL5_, the _GSL5_ from _B. rapa_; _BolGSL5_, the _GSL5_ from _B. oleracea_; _RsaGSL5_, the _GSL5_ from _R. sativus_ L. **h**, Disease index values of materials in **g**. **i**, Resistance test of _Arabidopsis gsl5_ plants against 36 field _P._ _brassicae_ isolates (representing 26 pathotypes). Wild-type plants were used as the susceptible control (\*\*_P_ < 0.01). Data in bar graphs are presented as means; error bars, s.d. In **d**, **f** and **h**, statistical significance was determined by a two-sided Student’s _t_\-test with 3 biological replicates in each of 30 plants. Scale bars in **c**, **e** and **g**, 1 cm. In **i**, statistical significance was determined by the Fisher–Freeman–Halton test; sample sizes and _P_ values are listed in Supplementary Table [7](https://www.nature.com/articles/s41588-025-02306-y#MOESM4).

**Fig. 3: Functional validation of _GSL5s_ in cruciferous crops.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02306-y/figures/3)

**a**, Genome editing to knock out _GSL5_s in _B._ _napus_, _B._ _rapa_ and _B._ _oleracea_. A consensus DNA sequence shared by four _GSL5_s of the above _Brassica_ crops in the first exon (blue rectangle) was used to design two guide RNA sites in opposite directions. The edited base of homozygous _gsl5-KO_ plants is highlighted in peach, and the resultant mutations are indicated on the right. **b**, Clubroot resistance test with the edited _gsl5-KO_ plants. The parental plants were used as the susceptible controls. WT, wild type. Representative photos of roots were taken at 30 dpi. Scale bars, 4 cm. **c**, Disease index values of materials in **b**. **d**, Broad-spectrum resistance test of ZS11-_gsl5-KO_ plants against 17 _P._ _brassicae_ isolates (representing 15 pathotypes) collected from 9 provinces. ZS11 plants were used as the susceptible control. Numbers below the _x_ axis represent sample sizes. **e**, The field phenotype of mature _B._ _napus gsl5-KO_ plants in Hanchuan county, Hubei province. The parental lines, Westar and ZS11, were used as controls. Scale bars, 20 cm. **f**, Evaluation of the impact of _GSL5_ mutation on the yield in _B._ _napus_ varieties Westar and ZS11 in two provinces. **g**, Evaluation of the impact of _GSL5_ mutation on the seed oil content in _B._ _napus_ varieties Westar and ZS11 in two provinces. In **f** and **g**, data are presented as means; error bars, s.d. In **c**, **d**, **f** and **g**, statistical significance was determined by a two-sided Student’s _t-_test. In **c** and **g**, statistical significance was determined using three biological replicates and ten plants each.

[Source data](https://www.nature.com/articles/s41588-025-02306-y#MOESM5)

### _GSL5_ inactivation without yield penalties in _B._ _napus_

To clarify whether _GSL5_ inactivation impacts key agronomic traits in _B._ _napus_, we performed small-scale field trials in clubroot-free fields of Gansu province (spring rapeseed region) and Hubei province (winter rapeseed region) to assess the usefulness of _GSL5_ mutation in Westar and ZS11 with transgene-free seeds. Multiple key agronomic traits were evaluated, including seed yield, seed oil content, plant height, branch number, silique number per plant, seed number per silique and 1,000-seed weight. We found that in both tested field sites, _gsl5_ mutation did not generally impact the yield or agronomic traits in either the Westar or ZS11 background (Fig. [3e,f](https://www.nature.com/articles/s41588-025-02306-y#Fig3) and Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02306-y#Fig11)). For seed yield, Westar and Westar-_gsl5_ yielded 2.17 and 2.08 kg per 10 m2, respectively, in Gansu and 1.29 and 1.27 kg per 10 m2, respectively, in Hubei, while ZS11 and ZS11-_gsl5_ yielded 3.95 and 3.75 kg per 10 m2, respectively, in Gansu and 3.25 and 3.19 kg per 10 m2, respectively, in Hubei (Fig. [3f](https://www.nature.com/articles/s41588-025-02306-y#Fig3)). We found that _gsl5_ mutation did not affect the seed oil content in the Westar background but reduced the seed oil content in the ZS11 background by 3.51% in both tested field sites (Fig. [3g](https://www.nature.com/articles/s41588-025-02306-y#Fig3)). There were no significant differences in the other traits assessed between _gsl5_ and wild-type plants of Westar and ZS11 at both locations (Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02306-y#Fig11)). In the clubroot diseased field, the yield of ZS11-_gsl5_ plants, retaining a high level of clubroot resistance, was twofold higher than that of the susceptible ZS11 wild-type plants (Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02306-y#MOESM1)). These results demonstrate that _GSL5_ mutation does not impact the key agronomic traits of _B._ _napus_, supporting its great value for cruciferous crop breeding.

### _gsl5_ resistance is associated with the induction of a defense response

To understand the mechanism of _gsl5_ resistance against _P._ _brassicae_, we first investigated which life stage of _P._ _brassicae_ is blocked by _gsl5_ resistance by comparing the infection process of _P._ _brassicae_\-inoculated _Arabidopsis_ wild-type and _gsl5_ plants. Confocal microscopy showed that, similar to that observed in the susceptible wild-type plants, _P._ _brassicae_ completed the primary infection at 7 dpi and released mature secondary zoospores in _gsl5_ plants, leaving empty zoosporangia in the root epidermal cells (Fig. [4a](https://www.nature.com/articles/s41588-025-02306-y#Fig4)). However, at 12 dpi, when _P._ _brassicae_ had established secondary infection and developed secondary plasmodia in the cortical cells of wild-type plants, no _P._ _brassicae_\-invaded cortical cells were observed in _gsl5_ plants. At 21 dpi, such plasmodia were more abundant and larger in wild-type plants but rarely observed in _gsl5_ plants (Fig. [4b](https://www.nature.com/articles/s41588-025-02306-y#Fig4)). These results support that _gsl5_ resistance efficiently blocks the establishment of secondary infection in the cortex tissue.

**Fig. 4: Cytological and physiological dissection of _gsl5_ resistance.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02306-y/figures/4)

**a**, Confocal microscopy of the _P._ _brassicae_ zoosporangia life stage in root epidermal cells of _Arabidopsis_ at 7 dpi. Zoosporangia containing the secondary zoospores are stained green (red arrows), whereas the zoosporangia releasing the secondary zoospores are not stained green (yellow arrows). This observation was repeated three times with the same result. Scale bars, 10 μm. **b**, Histological analysis of the main roots of _P._ _brassicae_\-inoculated _Arabidopsis_ wild-type and _gsl5_ plants at 12 dpi and 21 dpi. Representative cortical cells colonized by _P._ _brassicae_ parasites in wild-type plants are indicated with black arrowheads. This analysis was repeated three times with the same results. Scale bars, 100 μm. **c**, Relative expressions of pathogenesis-related genes _PR1_, _PR2_ and _PR5_ in the roots of _P._ _brassicae_\-inoculated _Arabidopsis_ wild-type and _gsl5_ plants at 12 dpi. **d**–**f**, Quantification of the lignin content (**d**), callose content (**e**) and reactive oxygen species (ROS) content (**f**) in the roots of _Arabidopsis_ wild-type and _gsl5_ plants without or with _P._ _brassicae_ inoculation at 12 dpi. **g**, Evans Blue staining of the roots of _Arabidopsis_ wild-type and _gsl5_ plants without or with _P._ _brassicae_ inoculation at 12 dpi. This analysis was repeated two times with the same result. Scale bars, 1 cm. **h**, Quantification of cytosolic ion leakage in the root tissues at 12 dpi. In **c**–**f** and **h**, data are presented as means and were quantified with three biological replicates (_n_ = 3); error bars, s.d. Different lowercase letters indicate a significant difference (_P_ < 0.05) based on one-way ANOVA with Tukey’s test in **d**–**f**. Statistical significance was determined by a two-sided Student’s _t_\-test in **c** and **h**.

We detected some defense reaction indicators in roots of wild-type and _gsl5_ plants in response to _P._ _brassicae_ infection at 12 dpi (the early secondary infection stage determining disease symptom occurrence). We found that _P._ _brassicae_ early secondary infection induced stronger defense reactions in _gsl5_ plants than in wild-type plants, including significantly higher expression levels of the defense response genes _Pathogenesis-Related 1_ (_PR1_), _PR2_ and _PR5_ (>6.1-fold) (Fig. [4c](https://www.nature.com/articles/s41588-025-02306-y#Fig4)); a significant increase in lignin content (Fig. [4d](https://www.nature.com/articles/s41588-025-02306-y#Fig4)) and in callose (β-1,3-glucan) content (Fig. [4e](https://www.nature.com/articles/s41588-025-02306-y#Fig4)), possibly resulting from the upregulation of the remaining _GSL_ genes identified previously in _Arabidopsis_[27](https://www.nature.com/articles/s41588-025-02306-y#ref-CR27 "Ellinger, D. & Voigt, C. A. Callose biosynthesis in Arabidopsis with a focus on pathogen response: what we have learned within the last decade. Ann. Bot. 114, 1349–1358 (2014).") (Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02306-y#MOESM1)); more cell death as indicated by deeper Evans Blue staining[22](https://www.nature.com/articles/s41588-025-02306-y#ref-CR22 "Wang, W. et al. WeiTsing, a pericycle-expressed ion channel, safeguards the stele to confer clubroot resistance. Cell 186, 2656–2671.e18 (2023).") (Fig. [4g](https://www.nature.com/articles/s41588-025-02306-y#Fig4)); and a significant increase (41.2%) in cytoplasmic ion leakage (an indicator of cell death[28](https://www.nature.com/articles/s41588-025-02306-y#ref-CR28 "Mittler, R., Shulaev, V. & Lam, E. Coordinated activation of programmed cell death and defense mechanisms in transgenic tobacco plants expressing a bacterial proton pump. Plant Cell 7, 29–42 (1995)."),[29](https://www.nature.com/articles/s41588-025-02306-y#ref-CR29 "Mittler, R., Simon, L. & Lam, E. Pathogen-induced programmed cell death in tobacco. J. Cell Sci. 110, 1333–1344 (1997)."),[30](https://www.nature.com/articles/s41588-025-02306-y#ref-CR30 "Mittler, R. et al. Transgenic tobacco plants with reduced capability to detoxify reactive oxygen intermediates are hyperresponsive to pathogen infection. Proc. Natl Acad. Sci. USA 96, 14165–14170 (1999).")) (Fig. [4h](https://www.nature.com/articles/s41588-025-02306-y#Fig4)). However, reactive oxygen species accumulated less in _gsl5_ roots than in wild-type roots whether _P._ _brassicae_ was inoculated or not (Fig. [4f](https://www.nature.com/articles/s41588-025-02306-y#Fig4)). These results support that _GSL5_ inactivation derepresses the robust defense response induced by _P._ _brassicae_ early secondary infection.

### _GSL5_ inactivation derepressed JA-mediated resistance

We then crossed the _gsl5_ mutant with a set of mutants or transgenic plants with defects in specific immune pathways and generated double mutants or transgenic plants, which were confirmed by PCR and sequencing with specific primers (Fig. [5a](https://www.nature.com/articles/s41588-025-02306-y#Fig5) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)). The clubroot-resistance assay showed that introduction of an exogenous salicylic acid hydroxylase, _NahG_, or the mutation _sid2-1_, which is defective in salicylic acid-mediated immunity, failed to restore _gsl5_ susceptibility to _P._ _brassicae_, and neither did the mutations _ald1_, defective in pipecolic acid-mediated immunity, and _pad4_, defective in phytoalexin and _Resistance_ gene-mediated immunity (Fig. [5a,b](https://www.nature.com/articles/s41588-025-02306-y#Fig5)). However, we found that introduction of the mutations _jar1-1_ or _ein2-1_, which are defective in JA and ethylene (ET)-mediated immunity, completely restored _gsl5_ clubroot susceptibility to the wild-type level (Fig. [5a,b](https://www.nature.com/articles/s41588-025-02306-y#Fig5)), demonstrating a critical role of JA/ET-mediated immunity for _gsl5_ resistance. We then measured the contents of endogenous JA precursor OPDA (oxylipin 12-oxo-phytodienoic acid), JA and the JA bioactive derivative JA-Ile, as well as the ET precursor ACC (1-aminocyclopropane-1-carboxylic acid) in the roots of _gsl5_ and wild-type plants before and 12 days after _P._ _brassicae_ inoculation. The results showed that without _P._ _brassicae_ inoculation, _gsl5_ mutation did not obviously alter the biosynthesis of JA and its precursor OPDA but resulted in a JA-Ile increase compared to the wild-type plants (Fig. [5c](https://www.nature.com/articles/s41588-025-02306-y#Fig5)). By contrast, _P._ _brassicae_ early secondary infection induced a twofold, threefold and 2.3-fold increase of OPDA, JA and JA-Ile contents, respectively in _gsl5_ roots compared to wild-type roots (Fig. [5c](https://www.nature.com/articles/s41588-025-02306-y#Fig5)). Intriguingly, the ACC content in _gsl5_ roots was quantified below that of wild-type roots before and after _P._ _brassicae_ inoculation (Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02306-y#Fig12)), indicating an unassociated link between _gsl5_ clubroot resistance and ET content. Moreover, we performed an RNA sequencing assay of the root tissues of _gsl5_ and wild-type plants before, 9 days after and 15 days after _P._ _brassicae_ inoculation (Supplementary Table [6](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)) to investigate whether the genes involved in JA and ET biosynthesis, transport, modification and signaling were altered in the expression levels. We found that compared to that of wild-type plants, most genes of the JA pathway in _gsl5_ root tissue were upregulated by _P._ _brassicae_ early secondary infection but were not obviously altered without _P._ _brassicae_ inoculation (Fig. [5d](https://www.nature.com/articles/s41588-025-02306-y#Fig5)). By contrast, the expressions of ET pathway genes were differentiated largely whether _P._ _brassicae_ was inoculated or not (Extended Data Fig. [6b](https://www.nature.com/articles/s41588-025-02306-y#Fig12)). KEGG enrichment of the differentially expressed genes between _gsl5_ and wild-type plants after _P._ _brassicae_ inoculation also strongly supported the involvement of JA pathway genes in _gsl5_ resistance, as the terms ‘jasmonic acid mediated signaling pathway’ ‘cellular response to jasmonic acid stimulus’ and ‘jasmonic acid biosynthetic process’ were significantly enriched (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02306-y#Fig13)). We then used exogenous methyl jasmonate (MeJA) to treat the wild-type plants of _Arabidopsis_ and _B._ _napus_ cv. ZS11 after _P._ _brassicae_ inoculation and found that the MeJA application markedly reduced the clubroot disease severity in both _Arabidopsis_ and _B._ _napus_ plants (Fig. [5e–h](https://www.nature.com/articles/s41588-025-02306-y#Fig5)). Collectively, these results show that _GSL5_ inactivation derepresses the JA immune pathway and confers broad-spectrum clubroot resistance during _P._ _brassicae_ secondary infection.

**Fig. 5: Genetic dissection of _gsl5_ resistance.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02306-y/figures/5)

**a**, Clubroot resistance test of _gsl5_\-derived double mutants or transgenic plants defective in immune pathways. Wild-type and _gsl5_ plants were used as susceptible and resistant controls, respectively. Representative photos were taken at 24 dpi. Scale bars, 1 cm. **b**, Disease index values of materials in **a**. **c**, Measurement of JA precursor OPDA, JA and JA-Ile contents in the roots of wild-type, _gsl5_ and _jar1-1_ plants without or with _P._ _brassicae_ inoculation at 12 dpi. FW, fresh weight. **d**, Transcriptome profiling of the gene expressions involved in JA biosynthesis, transport, modification and signaling in the roots before or after _P._ _brassicae_ inoculation at 9 days and 15 days. **e**, Treatment of _Arabidopsis_ wild-type plants with exogenous MeJA significantly reduced clubroot severity. Representative root photos were taken at 24 dpi. Scale bars, 1 cm. **f**, Disease index values of _Arabidopsis_ wild-type plants treated with an increasing concentration gradient of MeJA. **g**, Treatment of _B. napus_ variety ZS11 plants with exogenous MeJA significantly reduced clubroot severity. Representative root photos were taken at 24 dpi. Scale bars, 4 cm. **h**, Disease index values of _B. napus_ variety ZS11 plants treated with an increasing concentration gradient of MeJA. Data in **f** and **h** are presented as means; error bars, s.d. The contents in **c** and transcriptomic data in **d** were determined with three biological replicates (_n_ = 3). In **c**, different lowercase letters indicate a significant difference (_P_ < 0.05) based on one-way ANOVA with Tukey’s test. In **b**, **f** and **h**, statistical significance was determined by a two-sided Student’s _t_\-test with 3 biological replicates and 30 plants each.

### Stabilization of GSL5 by PbPDIa facilitated clubroot susceptibility

To further understand the crucial role GSL5 has in host clubroot susceptibility, we constructed a yeast cDNA library with the root tissues of _Arabidopsis_ wild-type plants during _P._ _brassicae_ secondary infection and screened for effectors that interacted with GSL5. As a result, we revealed a _P._ _brassicae_ protein, PBRA\_005301, as a GSL5-interacting candidate from the yeast library. PBRA\_005301 contained a signal peptide at the amino terminus and a protein disulfide isomerase TMX3 domain at the carboxyl terminus (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02306-y#Fig14)), belonging to the protein disulfide isomerase (PDI) family and thereby named _P._ _brassicae_ protein disulfide isomerase a (PbPDIa). Compared to the primary infection in the root epidermis tissue, _PbPDIa_ displayed an increasing expression level in the secondary infection in the cortex tissue, indicating a role of _PbPDIa_ in _P._ _brassicae_ pathogenicity (Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02306-y#Fig14)). Functional validation of the signal peptide of PbPDIa confirmed its secretory capacity (Extended Data Fig. [8c,d](https://www.nature.com/articles/s41588-025-02306-y#Fig14)). We then performed yeast two-hybrid and pull-down assays and demonstrated the interaction between GSL5 and PbPDIa (Fig. [6a,b](https://www.nature.com/articles/s41588-025-02306-y#Fig6)). Confocal microscopy revealed that PbPDIa is localized in the endomembrane system, including the endoplasmic reticulum, plasma membrane and nuclear envelope (Fig. [6c](https://www.nature.com/articles/s41588-025-02306-y#Fig6)). Importantly, host-induced gene silencing of _PbPDIa_ in _Arabidopsis_ plants significantly compromised the pathogenicity of all six different _P._ _brassicae_ pathotypes and largely reduced the disease severity (Fig. [6d–f](https://www.nature.com/articles/s41588-025-02306-y#Fig6)). PDI proteins were able to catalyze the formation of a disulfide bond in proteins, contributing to protein stability[31](https://www.nature.com/articles/s41588-025-02306-y#ref-CR31 "Matsusaki, M. et al. The protein disulfide isomerase family: from proteostasis to pathogenesis. Biochim. Biophys. Acta Gen. Subj. 1864, 129338 (2020)."). Indeed, PbPDIa exhibited enzymatic activities characteristic of the PDI family and restored the enzymatic activity of inactivated RNase A to the level of normal RNase A by catalyzing the formation of disulfide bonds (Fig. [6g](https://www.nature.com/articles/s41588-025-02306-y#Fig6)). Furthermore, we found that in vitro incubation of GSL5 with PbPDIa significantly enhanced the protein stability of GSL5, probably resulting from the formation of protein disulfide catalyzed by PbPDIa (Fig. [6h](https://www.nature.com/articles/s41588-025-02306-y#Fig6)).

**Fig. 6: Dissecting the role of _GSL5_ in clubroot susceptibility.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02306-y/figures/6)

**a**, Validation of PbPDIa and GSL5 interaction by yeast two-hybrid assays. This experiment was repeated twice with the same result. **b**, Validation of PbPDIa and GSL5 interaction by pull-down assays. This experiment was repeated twice with the same result. **c**, Subcellular localization of PbPDIa in planta. Leaves of _N._ _benthamiana_ were infiltrated with _Agrobacterium_ strains containing mCherry (mock) and a PbPDIa-mCherry vector. Three independent experiments showed similar results. Scale bars, 20 μm. **d**, Host-induced gene silencing of _PbPDIa_ in _Arabidopsis_ plants compromised the virulence of _P._ _brassicae_. Representative photos of shoots and roots of each material at 21 dpi are displayed. Scale bars, 1 cm. **e**, Relative expressions of _PbPDIa_ in _Arabidopsis_ wild-type plants and transgenic _PbPDIa-RNAi #1_ and _PbPDIa-RNAi_ _#2_ plants after _P. brassicae_ inoculation at 15 days. The _ubiquitin 5_ gene (_UBQ5_) was used as the internal control in _Arabidopsis_. **f**, Disease index values of _Arabidopsis_ wild-type plants and transgenic _PbPDIa-RNAi #1_ and _PbPDIa-RNAi #2_ plants after inoculation with six pathotypes of _P. brassicae_. **g**, Validation of the PDI enzymatic activity of PbPDIa. rRNase A, RNase A with reduced enzymatic activity. **h**, Incubation of GSL5 with PbPDIa significantly increased the stability of the GSL5 protein. **i**, A working model of _GSL5_ for clubroot susceptibility or resistance in the root tissue. Whether or not _GSL5_ was inactivated did not affect _P._ _brassicae_ primary infection in the root epidermis tissue. In the secondary infection, the thick line indicates enhanced suppression, while the thin line indicates compromised suppression. RS, resting spores; PZ, primary zoospores; UPP, uninucleate primary plasmodium; BPP, binucleate primary plasmodium; MPP, multinucleate primary plasmodium; ZS, zoosporangia; EZS, empty zoosporangia; USP, uninucleate secondary plasmodium; MSP, multinucleate secondary plasmodium. In **e** and **g**, data are presented as means and quantified with three biological replicates (_n_ = 3); error bars, s.d. In **f**, data are presented as mean value, and data below the _x_ axis represent sample size. In **g**, different lowercase letters indicate a significant difference (_P_ < 0.05) based on one-way ANOVA with Tukey’s test. In **e**, statistical significance was determined by a two-sided Student’s _t_\-test.

[Source data](https://www.nature.com/articles/s41588-025-02306-y#MOESM6)

Finally, we propose a model to illustrate how _GSL5_ performs as a crucial susceptible gene for clubroot disease in cruciferous plants (Fig. [6i](https://www.nature.com/articles/s41588-025-02306-y#Fig6)). Whether or not _GSL5_ is present has no effect on _P._ _brassicae_ primary infection in the root epidermal cells; however, in the early secondary infection in the cortical cells, _P._ _brassicae_ uninucleate secondary plasmodia secretes _PbPDIa_ to stabilize the GSL5 protein and strengthen GSL5-mediated suppression of JA immunity. Thus, _P._ _brassicae_ can complete the secondary infection and cause cortical cell hypertrophy and, ultimately, gall formation. Upon _GSL5_ inactivation, JA-mediated defense responses and cell death are derepressed but not activated, requiring a trigger by _P._ _brassicae_ early secondary infection, thereby blocking _P._ _brassicae_ young uninucleate plasmodia from further growth and multiplication. This results in a broad-spectrum, high level of clubroot resistance.

Discussion
----------

Presently, high-level qualitative clubroot resistance resources in _B._ _napus_ and _B._ _oleracea_ are very rare. The loci or genes with high-level qualitative resistance in _B._ _napus_ cultivars that are currently in use have been transferred mainly from fodder _B._ _rapa_ species by interspecific hybridization[8](https://www.nature.com/articles/s41588-025-02306-y#ref-CR8 "Hasan, J., Megha, S. & Rahman, H. Clubroot in Brassica: recent advances in genomics, breeding, and disease management. Genome 64, 735–760 (2021)."),[10](https://www.nature.com/articles/s41588-025-02306-y#ref-CR10 "Lai, S. et al. Clubroot resistant in cruciferous crops: recent advances in genes and QTLs identification and utilization. Hortic. Res. 12, uhaf105 (2025)."),[13](https://www.nature.com/articles/s41588-025-02306-y#ref-CR13 "Yang, Z. et al. R gene triplication confers European fodder turnip with improved clubroot resistance. Plant Biotechnol. J. 20, 1502–1517 (2022)."), usually taking a long time for clubroot-resistance breeding. Furthermore, these loci or gene resistances are pathotype-specific and have been frequently reported owing to a loss of resistance in diseased fields 3–5 years after planting[18](https://www.nature.com/articles/s41588-025-02306-y#ref-CR18 "Hollman, K. B., Hwang, S. F., Manolii, V. P. & Strelkov, S. E. Pathotypes of Plasmodiophora brassicae collected from clubroot resistant canola (Brassica napus L.) cultivars in western Canada in 2017–2018. Can. J. Plant Pathol. 43, 622–630 (2021)."),[19](https://www.nature.com/articles/s41588-025-02306-y#ref-CR19 "Strelkov, S. E., Hwang, S. F., Manolii, V. P., Cao, T. & Feindel, D. Emergence of new virulence phenotypes of Plasmodiophora brassicae on canola (Brassica napus) in Alberta, Canada. Eur. J. Plant Pathol. 145, 517–529 (2016).") because of a rapid shift of diverse field _P._ _brassicae_ pathotypes under high selection pressure of resistant cultivars; for example, the novel pathotypes 2C, 6D, 8D, 9 A, 9B, 9 C, 11 A, 13 A and 13B present in 2017 and 2018 in Canada[18](https://www.nature.com/articles/s41588-025-02306-y#ref-CR18 "Hollman, K. B., Hwang, S. F., Manolii, V. P. & Strelkov, S. E. Pathotypes of Plasmodiophora brassicae collected from clubroot resistant canola (Brassica napus L.) cultivars in western Canada in 2017–2018. Can. J. Plant Pathol. 43, 622–630 (2021)."). This situation imposes a great challenge for clubroot-resistance breeding in cruciferous crops to durably control clubroot disease, given that no single gene conferring broad-spectrum and qualitative resistance has been reported in cruciferous crops. In this study, _GSL5_ inactivation by genome editing achieved broad-spectrum and high-level qualitative resistance, not only in rapeseed, but also in _B._ _rapa_ and _B._ _oleracea_ vegetables, thus providing a valuable gene resource and a solution to accelerate the development of disease-resistant varieties and durably control clubroot disease in these crops. Importantly, _gsl5_ resistance resulting from JA-mediated immunity was not constitutively activated but rather induced by _P._ _brassicae_ secondary infection (Figs. [4](https://www.nature.com/articles/s41588-025-02306-y#Fig4) and [5](https://www.nature.com/articles/s41588-025-02306-y#Fig5)), thereby avoiding negative impacts on key agronomic traits and seed yield of _B._ _napus_.

It has been well established that plants have evolved numerous cell-surface and intracellular immune receptors to detect invading pathogens and subsequently activate a cascade of defense responses[32](https://www.nature.com/articles/s41588-025-02306-y#ref-CR32 "Zhou, J. M. & Zhang, Y. Plant immunity: danger perception and signaling. Cell 181, 978–989 (2020)."),[33](https://www.nature.com/articles/s41588-025-02306-y#ref-CR33 "Zhang, J., Coaker, G., Zhou, J. M. & Dong, X. Plant immune mechanisms: from reductionistic to holistic points of view. Mol. Plant 13, 1358–1378 (2020)."). However, this point is challenged by the interactions between _P._ _brassicae_ and resistant hosts, given that _P._ _brassicae_ can establish and complete primary infection in the root epidermis tissue of resistant hosts, as observed in susceptible hosts in this study and previous studies[7](https://www.nature.com/articles/s41588-025-02306-y#ref-CR7 "Liu, L. et al. Comparing the infection biology of Plasmodiophora brassicae in clubroot susceptible and resistant hosts and non-hosts. Front. Microbiol. 11, 507036 (2020)."),[22](https://www.nature.com/articles/s41588-025-02306-y#ref-CR22 "Wang, W. et al. WeiTsing, a pericycle-expressed ion channel, safeguards the stele to confer clubroot resistance. Cell 186, 2656–2671.e18 (2023)."),[34](https://www.nature.com/articles/s41588-025-02306-y#ref-CR34 "Ochoa, J. C. et al. Natural variation in Arabidopsis responses to Plasmodiophora brassicae reveals an essential role for Resistance to Plasmodiophora brassicae 1 (RPB1). Plant J. 116, 1421–1440 (2023)."). Moreover, the immune responses may not occur in the root epidermis tissues of clubroot-resistant hosts. This conclusion is further supported by research in which a typical _Resistance_ gene, _Crr1a_, and a novel gene, _WeiTsing/RPB1_, both conferring a high level of clubroot resistance, were found to be exclusively expressed in the root cortex tissue and not in the root epidermis tissue[12](https://www.nature.com/articles/s41588-025-02306-y#ref-CR12 "Hatakeyama, K. et al. Identification and characterization of Crr1a, a gene for resistance to clubroot disease (Plasmodiophora brassicae Woronin) in Brassica rapa L. PLoS ONE 8, e54745 (2013)."),[22](https://www.nature.com/articles/s41588-025-02306-y#ref-CR22 "Wang, W. et al. WeiTsing, a pericycle-expressed ion channel, safeguards the stele to confer clubroot resistance. Cell 186, 2656–2671.e18 (2023)."),[34](https://www.nature.com/articles/s41588-025-02306-y#ref-CR34 "Ochoa, J. C. et al. Natural variation in Arabidopsis responses to Plasmodiophora brassicae reveals an essential role for Resistance to Plasmodiophora brassicae 1 (RPB1). Plant J. 116, 1421–1440 (2023)."). Indeed, _P._ _brassicae_ primary infection does not cause obvious cytological alterations in the root epidermal cells or any visible disease symptoms[6](https://www.nature.com/articles/s41588-025-02306-y#ref-CR6 "Liu, L. et al. Refining the life cycle of Plasmodiophora brassicae. Phytopathology 110, 1704–1712 (2020)."),[7](https://www.nature.com/articles/s41588-025-02306-y#ref-CR7 "Liu, L. et al. Comparing the infection biology of Plasmodiophora brassicae in clubroot susceptible and resistant hosts and non-hosts. Front. Microbiol. 11, 507036 (2020)."). However, the situation is different in the root cortex tissue, where _P._ _brassicae_ secondary infection causes cell enlargement and gall formation, and, expectedly, _Resistance_ genes respond in the cortex tissue to activate immune responses and disrupt _P._ _brassicae_ secondary infection. Therefore, our study provides insights into the pathogenesis and resistance mechanisms surrounding the interactions between plants and intracellular eukaryotic phytopathogens, which are unique and distinct from all known interaction systems between plants and extracellular phytopathogens.

Although immune responses are triggered in the cortical cells during _P._ _brassicae_ secondary infection, as explained above, the underlying molecular mechanisms remain poorly understood. Studies on _WeiTsing_ described pentameric cation channel-mediated Ca2+ permeability and associated immunity[22](https://www.nature.com/articles/s41588-025-02306-y#ref-CR22 "Wang, W. et al. WeiTsing, a pericycle-expressed ion channel, safeguards the stele to confer clubroot resistance. Cell 186, 2656–2671.e18 (2023)."), whereas the TNL-type proteins Crr1a, CRa and CRA8.2.4 imply a conserved and canonical resistance mechanism, although such a mechanism has not been reported[11](https://www.nature.com/articles/s41588-025-02306-y#ref-CR11 "Ueno, H. et al. Molecular characterization of the CRa gene conferring clubroot resistance in Brassica rapa. Plant Mol. Biol. 80, 621–629 (2012)."),[12](https://www.nature.com/articles/s41588-025-02306-y#ref-CR12 "Hatakeyama, K. et al. Identification and characterization of Crr1a, a gene for resistance to clubroot disease (Plasmodiophora brassicae Woronin) in Brassica rapa L. PLoS ONE 8, e54745 (2013)."),[13](https://www.nature.com/articles/s41588-025-02306-y#ref-CR13 "Yang, Z. et al. R gene triplication confers European fodder turnip with improved clubroot resistance. Plant Biotechnol. J. 20, 1502–1517 (2022)."). Currently, why or how immune responses against _P._ _brassicae_ infection are initiated is not known. In contrast to typical dominant and pathotype-specific resistance genes, most recessive broad-spectrum resistance genes are novel and unique and work through diverse resistance mechanisms; for example, inactivation of _MLO_ and _PsIPK1_ in wheat[35](https://www.nature.com/articles/s41588-025-02306-y#ref-CR35 "Li, S. et al. Genome-edited powdery mildew resistance in wheat without growth penalties. Nature 602, 455–460 (2022)."),[36](https://www.nature.com/articles/s41588-025-02306-y#ref-CR36 "Wang, N. et al. Inactivation of a wheat protein kinase gene confers broad-spectrum resistance to rust fungi. Cell 185, 2961–2974.e19 (2022)."), and _RBL1__Δ12_ and _BSR-K1_ in rice[37](https://www.nature.com/articles/s41588-025-02306-y#ref-CR37 "Sha, G. et al. Genome editing of a rice CDP-DAG synthase confers multipathogen resistance. Nature 618, 1017–1023 (2023)."),[38](https://www.nature.com/articles/s41588-025-02306-y#ref-CR38 "Zhou, X. et al. Loss of function of a rice TPR-domain RNA-binding protein confers broad-spectrum disease resistance. Proc. Natl Acad. Sci. USA 115, 3174–3179 (2018)."). The present study provides another unique case, showing that GSL5, a well-recognized β-1,3-glucan synthase involved in cell wall solidification in response to biotic and abiotic stress, acts as a suppressor of JA-mediated broad-spectrum clubroot resistance in cruciferous plants, although the underlying molecular details remain to be determined.

Methods
-------

### Plant materials and growth

The _B._ _napus_ accessions in this study were planted in Zhijiang (30° 48’ N, 111° 77’ E) and Taiping (29° 71’ N, 118° 29’ E), spanning three growing seasons (2016–2018). The edited _B._ _napus_ plants were planted in Hubei (30° 54’ N, 113° 81’ E) and Gansu (38° 43’ N and 100° 81’ E) in 2023. All _B._ _napus_ materials were maintained by self-pollination. Crop management of field experiments for agronomic trait tests followed the standard protocol of the China National Rapeseed Variety Field Test.

All _A._ _thaliana_ and _Nicotiana benthamiana_ plants were grown under controlled conditions at 21–24°C and with a 16 h photoperiod. The _Arabidopsis_ materials used in this study include _pmr4-1_ (_gsl5_)[23](https://www.nature.com/articles/s41588-025-02306-y#ref-CR23 "Nishimura, M. T. et al. Loss of a callose synthase results in salicylic acid-dependent disease resistance. Science 301, 969–972 (2003)."), _sid2-1_ (ref. [39](https://www.nature.com/articles/s41588-025-02306-y#ref-CR39 "Nawrath, C. & Métraux, J.-P. Salicylic acid induction-deficient mutants of Arabidopsis express PR-2 and PR-5 and accumulate high levels of camalexin after pathogen inoculation. Plant Cell 11, 1393–1404 (1999).")), _jar1-1_ (ref. [40](https://www.nature.com/articles/s41588-025-02306-y#ref-CR40 "Staswick, P. E., Tiryaki, I. & Rowe, M. L. Jasmonate response locus JAR1 and several related Arabidopsis genes encode enzymes of the firefly luciferase superfamily that show activity on jasmonic, salicylic, and indole-3-acetic acids in an assay for adenylation. Plant Cell 14, 1405–1415 (2002).")), _ein2-1_ (ref. [41](https://www.nature.com/articles/s41588-025-02306-y#ref-CR41 "Roman, G., Lubarsky, B., Kieber, J. J., Rothenberg, M. & Ecker, J. R. Genetic analysis of ethylene signal transduction in Arabidopsis thaliana: five novel mutant loci integrated into a stress response pathway. Genetics 139, 1393–1409 (1995).")), _ald1_ (ref. [42](https://www.nature.com/articles/s41588-025-02306-y#ref-CR42 "Song, J. T., Lu, H. & Greenberg, J. T. Divergent roles in Arabidopsis thaliana development and defense of two homologous genes, ABERRANT GROWTH AND DEATH2 and AGD2-LIKE DEFENSE RESPONSE PROTEIN1, encoding novel aminotransferases. Plant Cell 16, 353–366 (2004).")), _pad4_ (ref. [43](https://www.nature.com/articles/s41588-025-02306-y#ref-CR43 "Jirage, D. et al. Arabidopsis thaliana PAD4 encodes a lipase-like gene that is important for salicylic acid signaling. Proc. Natl Acad. Sci. USA 96, 13583–13588 (1999).")) and the transgenic line _NahG_[44](https://www.nature.com/articles/s41588-025-02306-y#ref-CR44 "Lawton, K. et al. Systemic acquired resistance in Arabidopsis requires salicylic acid but not ethylene. Mol. Plant Microbe Interact. 8, 863–870 (1995)."). The homozygosity of T-DNA insertion mutants was genotyped by PCR with both gene-specific and T-DNA border primers. The point mutation lines were genotyped by sequencing the PCR products after amplification with gene-specific primers. The resultant homozygous double mutants were used for the clubroot resistance test. The primers used in this study are listed in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02306-y#MOESM4).

### Phenotyping for field traits

All field experiments for phenotyping the field traits were carried out in a randomized block design with three replicates. Each replicate contained at least 60 plants with a space of about 30 cm between rows, and each row was 2 m in length with a space between plants of approximately 10 cm. The seeds of each accession were sown in late September (26–30), which is the beginning of the growing season for winter canola plantation areas in China. In early December (about 70 days after sowing), the roots of the susceptible control (Westar) became severely clubbed, and the growth of the plants, as well as the _P._ _brassicae_ infection, was almost terminated in the clubroot disease fields owing to the lower temperature in winter (usually below 8 °C). All tested _B._ _napus_ accessions in the clubroot disease fields were harvested to score clubroot disease severity. Disease scales 0, 1, 2 and 3 were used to score each plant of each replicate of 244 _B._ _napus_ accessions and to generate the DI to quantify the disease severity of each accession based on a previously described method[7](https://www.nature.com/articles/s41588-025-02306-y#ref-CR7 "Liu, L. et al. Comparing the infection biology of Plasmodiophora brassicae in clubroot susceptible and resistant hosts and non-hosts. Front. Microbiol. 11, 507036 (2020)."): 0, no clubs; 1, a few small clubs present on less than one-third of the lateral roots or on the main root; 2, moderate clubs present on one-third to two-thirds of the lateral roots or the main root; and 3, severe clubs on the main root or present on more than two-thirds of the lateral roots. In early May of the next year—the end of the growing season for winter canola plantation areas in China—the _B._ _napus_ plants were harvested for scoring of the key agronomic traits.

To evaluate the key agronomic traits, each replicate comprised 15 rows with 20 plants in each row. The sampling method depended on specific traits: (1) plant height (the length (cm) from the cotyledon node to the tip of a plant); (2) branch number (the number of primary branches per plant); (3) the 1,000-seed weight (in g) per plant; (4) silique number per plant (all siliques in each plant); (5) seed number per silique (the average seed number of the bottom 20 siliques from the main inflorescence of each plant); (6) yield (the seed weight of all plants within each block); and (7) seed oil content. Data from the first five key agronomic traits were determined based on three replicates, each of which comprised ten randomized plants. A Foss NIR Systems 5000 Near-Infrared Reflectance Spectroscope was used to measure the oil content of seeds collected at the maturity period[45](https://www.nature.com/articles/s41588-025-02306-y#ref-CR45 "Zhao, C. et al. Genome-wide association analysis combined with quantitative trait loci mapping and dynamic transcriptome unveil the genetic control of seed oil content in Brassica napus L. Front. Plant Sci. 13, 929197 (2022).").

### GWAS and favorable allele identification

A total of 244 _B._ _napus_ accessions were genome-resequenced with at least tenfold genome coverage, and a total of 2,797,642 filtered SNPs with a missing rate of ≤0.1 and minor allele frequency of ≥0.05 were obtained[20](https://www.nature.com/articles/s41588-025-02306-y#ref-CR20 "Zhang, Y. et al. Structural variation reshapes population gene expression and trait variation in 2,105 Brassica napus accessions. Nat. Genet. 56, 2538–2550 (2024)."). The clubroot DI of each _B._ _napus_ accession was investigated in three consecutive years (2016–2018) and used to generate the best linear unbiased prediction using the R script lme4 ([https://cran.r-project.org/web/packages/lme4](https://cran.r-project.org/web/packages/lme4)) and lsmeans[21](https://www.nature.com/articles/s41588-025-02306-y#ref-CR21 "Ren, D., An, L., Li, B., Qiao, L. & Liu, W. Efficient weighting methods for genomic best linear-unbiased prediction (BLUP) adapted to the genetic architectures of quantitative traits. Heredity 126, 320–334 (2021)."). Three models, including a general linear model, mixed linear model and fixed and random model Circulating Probability Unification (FarmCPU), were used for genome-wide association analysis. The significant _P_ value threshold of the GWAS was set to 1.8 × 10−8 (−log10_P_ = 7.74, calculated as 0.05 / total number of SNPs). Haplotype block estimation was based on the confidence interval method[46](https://www.nature.com/articles/s41588-025-02306-y#ref-CR46 "Gabriel, S. B. The structure of haplotype blocks in the human genome. Science 296, 2225–2229 (2002)."). The linkage disequilibrium of the whole genome and the significantly associated regions were analyzed using PopLDdecay and LDBlockShow software[47](https://www.nature.com/articles/s41588-025-02306-y#ref-CR47 "Zhang, C., Dong, S. S., Xu, J. Y., He, W. M. & Yang, T. L. PopLDdecay: a fast and effective tool for linkage disequilibrium decay analysis based on variant call format files. Bioinformatics 35, 1786–1788 (2018)."),[48](https://www.nature.com/articles/s41588-025-02306-y#ref-CR48 "Dong, S.-S. et al. LDBlockShow: a fast and convenient tool for visualizing linkage disequilibrium and haplotype blocks based on variant call format files. Brief. Bioinform. 22, bbaa227 (2020).").

The SNPs of the gene-related regions were extracted from the genomic variation files. A phylogenetic tree was constructed using PHYLIP (v.3.696) with the neighbor-joining method and default parameters and was visualized by FigTree (v.1.4.3)[49](https://www.nature.com/articles/s41588-025-02306-y#ref-CR49 "Retief, J. D. Phylogenetic analysis using PHYLIP. Methods Mol. Biol. 132, 243–258 (2000)."). PCA analysis was performed using PLINK (v.1.90b4.6), and the first two principal components of the PCA analysis were illustrated by the ggplot2 package in R (v.4.1)[50](https://www.nature.com/articles/s41588-025-02306-y#ref-CR50 "Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007)."). Population structure was analyzed by ADMIXTURE (v.1.3.0) with the following parameters: the number of subgroups, _K_, ranged from two to eight, and the cross-validation error was calculated for each _K_ value[51](https://www.nature.com/articles/s41588-025-02306-y#ref-CR51 "Alexander, D. H. et al. Fast model-based estimation of ancestry in unrelated individuals. Genome Res. 19, 1655–1664 (2009).").

### Gene complementation and expression, RNA interference assay and genome editing

For complementation experiments with the _gsl5_ mutant, a full-length genomic DNA fragment of _GSL5_ including approximately 3.0 kb of the upstream sequence of the start codon and the open reading frame was amplified from _Arabidopsis_, _B._ _napus_, _B._ _rapa_, _B._ _oleracea_ and _R._ _sativus_ using gene-specific primers (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)). All fragments were respectively cloned into the pBI121 vector, generating the recombinant plasmids _proAtGSL5_Col-0_:AtGSL5_Col-0 (_GSL5_), _proBnaA09.GSL5_Westar_:BnaA09.GSL5_Westar(Hap\_1), _proBnaA09.GSL5_ZS11_:BnaA09.GSL5_ZS11(Hap\_2), _proBnaC09.GSL5_ZS11_:BnaC09.GSL5_ZS11 (_BnaC09.GSL5_), _proBraGSL5_Chiifu_:BraGSL5_Chiifu (_BraGSL5_), _proBolGSL5_ZG11_:BolGSL5_ZG11 (_BolGSL5_) and _proRsaGSL5_MTH_:RsaGSL5_MTH (_RsaGSL5_) for _Agrobacterium_\-mediated transfection of _Arabidopsis gsl5_ plants.

To overexpress _AtGSL5_ in _Arabidopsis_, the _GSL5_ genomic region was cloned into pBI121 driven by the cauliflower mosaic virus 35S promoter, generating the recombinant plasmid _35S:AtGSL5_ (_GSL5-OE_) for _Agrobacterium_\-mediated transfection of wild-type plants.

For the RNA interference assay, a 385 bp coding sequence of _PbPDIa_ was cloned into a pBI121-RNAi vector driven by the cauliflower mosaic virus 35S promoter, generating the recombinant plasmid for _Agrobacterium_\-mediated transfection of wild-type plants. The transgenic plants produced a hairpin containing a 385 bp double-stranded RNA and generated endogenous small RNA that was able to knock down the _PbPDIa_ expression of _P._ _brassicae_[52](https://www.nature.com/articles/s41588-025-02306-y#ref-CR52 "Zand Karimi, H. & Innes, R. W. Molecular mechanisms underlying host-induced gene silencing. Plant Cell. 34, 3183–3199 (2022).").

For genome editing, two single-guide RNA (sgRNA) sequences, sgRNA1 and sgRNA2, from the first exon of _GSL5_ were designed as the editing targets to knock out the _GSL5_ from _A._ _thaliana_, _B._ _napus_, _B._ _rapa_ and _B._ _oleracea_ (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02306-y#MOESM4)). All sgRNAs were designed using CRISPR-P (v.2.0) ([http://crispr.hzau.edu.cn/CRISPR2](http://crispr.hzau.edu.cn/CRISPR2))[53](https://www.nature.com/articles/s41588-025-02306-y#ref-CR53 "Liu, H. et al. CRISPR-P 2.0: an improved CRISPR–Cas9 tool for genome editing in plants. Mol. Plant 10, 530–532 (2017)."). A previously developed multiplex genome editing vector, pYLCRISPR-Cas9-DB[54](https://www.nature.com/articles/s41588-025-02306-y#ref-CR54 "Ma, X. et al. A robust CRISPR/Cas9 system for convenient, high-efficiency multiplex genome editing in monocot and dicot plants. Mol. Plant 8, 1274–1284 (2015)."), was used to knock out _GSL5_.

All fused plasmids were confirmed by sequencing and introduced into _Agrobacterium_ _tumefaciens_ strain GV3101 for the following transformation of _Arabidopsis_, _B._ _napus_, _B._ _rapa_ and _B._ _oleracea_, using previously described methods[55](https://www.nature.com/articles/s41588-025-02306-y#ref-CR55 "Clough, S. J. & Bent, A. F. Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana. Plant J. 16, 735–743 (2010)."). All transgenic plants were genotyped by PCR amplification or DNA sequencing with the specific primers listed in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02306-y#MOESM4). The homozygous _gsl5_ mutants of _A._ _thaliana_, _B._ _napus_, _B._ _rapa_ and _B._ _oleracea_ were used for further phenotyping.

### Pathogen maintenance and inoculation

_P._ _brassicae_ isolates were collected from diseased plants of different _Brassica_ crops across China and have been previously identified for the pathotypes[17](https://www.nature.com/articles/s41588-025-02306-y#ref-CR17 "Zeng, L. et al. Pathotype characterization of Plasmodiophora brassicae by European clubroot differential and Williams sets in China. Plant Dis. 108, 847–851 (2023)."). The clubbed roots were harvested, cleaned and stored in −20 °C freezers. To isolate the resting spores, the clubbed roots were cut into small pieces and smashed in distilled water with a blender. The suspensions were filtered with six layers of gauze and adjusted to 1.0 × 107 resting spores per ml. To evaluate clubroot resistance under controlled conditions (21–24 °C, 16 h light, 8 h darkness), 2-week-old seedlings of _Arabidopsis_ or 10-day-old seedlings of _B._ _napus_, _B._ _rapa_ and _B._ _oleracea_ were inoculated with 1 ml of resting spore suspension (1.0 × 107 spores per ml) per seedling. After 24–30 days, the inoculated plants were harvested for scoring based on disease scale and severity as described previously[7](https://www.nature.com/articles/s41588-025-02306-y#ref-CR7 "Liu, L. et al. Comparing the infection biology of Plasmodiophora brassicae in clubroot susceptible and resistant hosts and non-hosts. Front. Microbiol. 11, 507036 (2020).").

### Subcellular localization

For subcellular localization, the coding sequence of _PbPDIa_, after removing the signal peptide sequence, was cloned into pBI121-mCherry. Then, pBI121-mCherry-PbPDIa was transformed into _A._ _tumefaciens_ GV3101 and injected into the leaves of 4-week-old _N._ _benthamiana_ plants for transient expression. The empty vector pBI121-mCherry was used as a control. The mCherry signals were detected with a confocal microscope (Carl Zeiss) with an excitation wavelength of 561 nm and an emission wavelength of 560–620 nm.

### Microscopy analysis

Fluorescent probe-based confocal microscopy was used to visualize the zoosporangia of _P._ _brassicae_ during the primary infection, and the fluorescent probe HCS LipidTox Green neutral lipid stain (HLG; Thermo Fisher Scientific) was used to label the zoosporangia[6](https://www.nature.com/articles/s41588-025-02306-y#ref-CR6 "Liu, L. et al. Refining the life cycle of Plasmodiophora brassicae. Phytopathology 110, 1704–1712 (2020)."). To detect the fluorescence of HLG, an excitation wavelength of 488 nm and an emission wavelength of 500 to 540 nm were used.

Histological technique was used to detect _P._ _brassicae_ parasites during secondary infection. _P._ _brassicae_\-inoculated roots of _Arabidopsis_ were harvested at 12 dpi and 21 dpi, and the main roots adjacent to the hypocotyl were sampled for histological analysis as described previously[56](https://www.nature.com/articles/s41588-025-02306-y#ref-CR56 "Schuller, A. & Ludwig‐Müller, J. Histological methods to detect the clubroot pathogen Plasmodiophora brassicae during its complex life cycle. Plant Pathol. 65, 1223–1237 (2016)."). In brief, tissues were fixed overnight in a freshly prepared solution of 2% glutaraldehyde, dehydrated through a graded ethanol series, embedded in a mold in melted paraffin and trimmed to produce hemi-sections. The sections were then placed on a slide, stained with 0.05% toluidine blue O and subjected to microscopic analysis.

### Transcriptome profiling and quantitative real-time PCR analysis

The roots of _Arabidopsis_ plants with or without _P._ _brassicae_ inoculation were harvested at 12 dpi and 21 dpi with three replicates and immediately frozen in liquid nitrogen. High-quality RNA was extracted using the Fast Pure Plant Total RNA Isolation Kit (VAZYME). cDNA library construction and sequencing, data quality control and gene expression calculation were performed at Novogene. In brief, clean reads were obtained by filtering raw reads and then mapped to the _Arabidopsis_ Col-0 reference genome using HISAT2 software[57](https://www.nature.com/articles/s41588-025-02306-y#ref-CR57 "Kim, D., Langmead, B. & Salzberg, S. L. HISAT: a fast spliced aligner with low memory requirements. Nat. Methods 12, 357–360 (2015)."),[58](https://www.nature.com/articles/s41588-025-02306-y#ref-CR58 "Chalhoub, B. et al. Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome. Science 345, 950–953 (2014)."). The number of fragments per kilobase per million mapped fragments was calculated and used to estimate gene expression levels[59](https://www.nature.com/articles/s41588-025-02306-y#ref-CR59 "Trapnell, C. et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat. Biotechnol. 28, 511–515 (2010)."). Pearson’s correlation coefficient was calculated with the R script cor ([https://search.r-project.org/R/refmans/stats/html/cor.html](https://search.r-project.org/R/refmans/stats/html/cor.html)) to evaluate the correlation between biological replicates[60](https://www.nature.com/articles/s41588-025-02306-y#ref-CR60 "Leng, N. et al. EBSeq: an empirical Bayes hierarchical model for inference in RNA-seq experiments. Bioinformatics 29, 1035–1043 (2013)."). DESeq software was used to identify the differentially expressed genes with the thresholds |log2(fold change)| > 1 and adjusted _P_ < 0.05 (ref. [61](https://www.nature.com/articles/s41588-025-02306-y#ref-CR61 "Anders, S. & Huber, W. Differential expression analysis for sequence count data. Genome Biol. 11, R106 (2010).")). A Venn diagram of differentially expressed genes between groups was plotted on the website [http://jvenn.toulouse.inra.fr/app/example.html](http://jvenn.toulouse.inra.fr/app/example.html)[62](https://www.nature.com/articles/s41588-025-02306-y#ref-CR62 "Bardou, P., Mariette, J., Escudié, F., Djemiel, C. & Klopp, C. jvenn: an interactive Venn diagram viewer. BMC Bioinformatics 15, 293 (2014).").

For quantitative PCR with reverse transcription (RT–qPCR), total RNA was used for first-strand cDNA synthesis with the Hifair III 1st Strand cDNA Synthesis Kit (gDNA digester plus) following the manufacturer’s instructions (YEASEN). Quantitative PCR was performed with a CFX Connect Real-time PCR system (Bio-Rad), using Hieff UNICON Universal Blue qPCR SYBR Master Mix (YEASEN). _Ubiquitin 5_ (_UBQ5_) and _Actin 7_ were used as the internal control in _Arabidopsis_ and _B._ _napus_, respectively. The relative expression level of genes was achieved using the 2(-ΔΔCT) method[63](https://www.nature.com/articles/s41588-025-02306-y#ref-CR63 "Livak, K. J. & Schmittgen, T. D. Analysis of relative gene expression data using real-time quantitative PCR and the 2−ΔΔCT method. Methods 25, 402–408 (2001)."). The primers for RT–qPCR are listed in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02306-y#MOESM4).

### MeJA treatments

MeJA powder (Macklin) was dissolved in dimethylsulfoxide to prepare a 1 M stock solution. Different concentrations of working solution were prepared by diluting the stock solution of MeJA with distilled water and used to water the _Arabidopsis_ or _B._ _napus_ plants at 5 dpi and 9 dpi. The treated plants were collected at 30 dpi for scoring the disease severity.

### Measurement of plant hormones, ACC, callose, lignin and reactive oxygen species

The roots of _Arabidopsis_ plants with or without _P._ _brassicae_ inoculation were harvested at 12 dpi and immediately frozen in liquid nitrogen with three biological replicates. Extraction and quantification were performed at RUIYUAN BIOTECHNOLOGY. In brief, the frozen plant tissues were ground and extracted with specialized buffers, followed by successive incubation, centrifugation, purification and condensation for the preparations. Quantification analysis was performed by ultra-performance liquid chromatography–electrospray tandem mass spectrometry using a high-performance liquid chromatograph (Agilent 1290) and a mass spectrometer (AB Qtrap 6500).

### Expression and purification of GSL5 and PbPDIa

GSL5 and PbPDIa were cloned into the engineered pMlink vector with a Flag or His tag, respectively, and then transfected into Expi293F cells (A14528, ThermoFisher, cat. no. 100044202, cGMP bank) for 60 h incubation. After harvest and washing with PBS, the transfected cells were homogenized in lysis buffer (50 mM Tris-HCl pH 7.4, 150 mM NaCl), disrupted with high pressure and incubated in 1% _N_\-dodecyl-β-d\-maltoside (DDM) for 2 h at 4 °C. After centrifugation, the supernatant was incubated with anti-Flag M2 or Ni2+ affinity resin for 1 h. The resin was eluted for Flag-tagged GSL5 with lysis buffer containing 150 μg ml−1 Flag peptide and 0.02% DDM or for His-tagged PbPDIa, with the lysis buffer containing 250 mM imidazole and 0.02% DDM. The eluted samples were concentrated to 1 ml immediately before gel filtration chromatography (Superose 6 10/300, GE Healthcare).

### Pull-down assays and protein stability assay

The purified Flag-tagged GSL5, His-tagged PbPDIa and their mixture samples were incubated with anti-Flag M2 affinity resin for 1 h. The resin was washed with washing buffer containing 50 mM Tris-HCl pH 7.4, 150 mM NaCl and 0.02% DDM and then eluted with washing buffer containing 150 μg ml−1 Flag peptide. The proteins were detected through immunoblots with antibodies against His or Flag.

Two equal portions of Flag-tagged GSL5 proteins were treated with the lysis buffer without or with PbPDIa at 25 °C. Samples were collected every 4 h and detected for protein stability by immunoblot with Flag antibodies.

### In vitro PDI activity assay

An in vitro PDI activity assay using reduced ribonuclease A (rRNase A) as a substrate was carried out as previously described[64](https://www.nature.com/articles/s41588-025-02306-y#ref-CR64 "Tian, G. et al. The crystal structure of yeast protein disulfide isomerase suggests cooperativity between its active sites. Cell 124, 61–73 (2006)."). RNase A (50 μg) was unfolded in unfolding buffer (100 mM Tris-HCl pH 8.0, 0.3 M DTT, 6 M guanidine-HCl) for 1 h at 37 °C and desalted with a 0.1% acetic acid-equilibrated G25 column. The resultant rRNase A fractions were collected and quantified using a molar extinction coefficient at 280 nm. The PDI activity assay was carried out in refolding buffer (36 μM rRNase A, 100 mM Tris-HCl buffer pH 8.5, 1 mM glutathione, 0.2 mM oxidized glutathione, 16 μM PbPDIa). RNase activity was measured by the A260 absorption value resulting from the cleavage of total RNAs. The relative activity was calculated as (_k_treat − _k_RNA) / (_k_RNase A − _k_RNA) × 100%.

### Assessment of PbPDIa secretory activity

Assessment of PbPDIa secretory activity was performed with the Yeast Signal Trap Assay Kit (COOLABER). In brief, the DNA sequence of _PbPDIa_ encoding the signal peptide was amplified, cloned into the pSUC2 vector and transformed into the sucrase-deficient yeast strain YTK12. YPRAA medium (a medium containing 1% yeast extract, 2% peptone, 2% raffinose, 2 µg ml−1 antimycin A and 2% agar) was used to evaluate secretory activity, as only the strain containing the functional signal peptide can grow on the YPRAA medium. Furthermore, 2,3,5-triphenyltetrazolium chloride was also included as per the user guide and was reduced by secreted sucrase into insoluble, red-colored 1,3,5-triphenylformazan.

### Cell death assay

Cell death was assessed by Evans blue (Merck) staining and by measuring cytoplasmic ion leakage from plant tissues[28](https://www.nature.com/articles/s41588-025-02306-y#ref-CR28 "Mittler, R., Shulaev, V. & Lam, E. Coordinated activation of programmed cell death and defense mechanisms in transgenic tobacco plants expressing a bacterial proton pump. Plant Cell 7, 29–42 (1995)."),[29](https://www.nature.com/articles/s41588-025-02306-y#ref-CR29 "Mittler, R., Simon, L. & Lam, E. Pathogen-induced programmed cell death in tobacco. J. Cell Sci. 110, 1333–1344 (1997)."),[30](https://www.nature.com/articles/s41588-025-02306-y#ref-CR30 "Mittler, R. et al. Transgenic tobacco plants with reduced capability to detoxify reactive oxygen intermediates are hyperresponsive to pathogen infection. Proc. Natl Acad. Sci. USA 96, 14165–14170 (1999)."). The roots of _Arabidopsis_ plants with or without _P._ _brassicae_ inoculation were collected at 12 dpi and placed in a 2% Evans blue solution for 30 min of staining. The plants were then rinsed in distilled water until the roots of the solvent-treated plants could not be stained. The Evans blue-stained plants were imaged under a stereomicroscope SZX16 (Olympus).

To measure the cytoplasmic ion leakage, _P._ _brassicae_\-infected roots of _Arabidopsis_ were collected at 12 dpi for a 3 h inoculation in ultrapure water at room temperature and the conductivity of the bathing solution was measured with a conductivity meter (STARTER 300C; OHAUS), referred to as value _C_1; the conductivity of the ultrapure water is value _C_0. The bathing solution containing the root tissues was incubated at 95 °C for 30 min, and conductivity was determined after cooling to room temperature, referred to as value _C_2. The relative cytoplastic ion leakage was calculated with the following formula: (_C_1 − _C_0) / (_C_2 − _C_0) × 100%.

### Statistical analysis

All statistical analyses were performed using Graphpad Prism (v.7.0; [https://www.graphpad.com](https://www.graphpad.com/)) or R (v.4.1.2). Detailed information, including the testing model, sample size, replicates and _P_ values, is provided in the individual figures and figure legends.

### Inclusion and ethics

The data presented in this study were derived exclusively from _Arabidopsis_ and _Brassica_ crops, without involving any animal experiments. The transgenic planting and artificial inoculation of _P._ _brassicae_ are subject to strict regulation. All experimental data are included in the Data availability section.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02306-y#MOESM2) linked to this article.

Data availability
-----------------

The RNA sequencing data generated from this study are available under Bioproject accession number [PRJNA1142741](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1142741) from the National Center for Biotechnology Information. General information regarding RNA sequencing data is listed in Supplementary Table [6](https://www.nature.com/articles/s41588-025-02306-y#MOESM4). The genome resequencing data for the 244 _B._ _napus_ accessions generated in the previous study are publicly available under the Bioproject accession number [PRJNA1156901](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1156901) from the National Center for Biotechnology Information[20](https://www.nature.com/articles/s41588-025-02306-y#ref-CR20 "Zhang, Y. et al. Structural variation reshapes population gene expression and trait variation in 2,105 Brassica napus accessions. Nat. Genet. 56, 2538–2550 (2024)."). [Source data](https://www.nature.com/articles/s41588-025-02306-y#Sec32) are provided with this paper.

Code availability
-----------------

All software used in this study is publicly available on the Internet as described in the [Methods](https://www.nature.com/articles/s41588-025-02306-y#Sec10) and Reporting summary. Codes for GWAS and RNA sequencing analyses are available at Zenodo ([https://doi.org/10.5281/zenodo.13365025](https://doi.org/10.5281/zenodo.13365025)) (ref. [20](https://www.nature.com/articles/s41588-025-02306-y#ref-CR20 "Zhang, Y. et al. Structural variation reshapes population gene expression and trait variation in 2,105 Brassica napus accessions. Nat. Genet. 56, 2538–2550 (2024).")).

References
----------

1.  Javed, M. A. et al. The clubroot pathogen _Plasmodiophora brassicae_: a profile update. _Mol. Plant Pathol._ **24**, 89–106 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36448235)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20clubroot%20pathogen%20Plasmodiophora%20brassicae%3A%20a%20profile%20update&journal=Mol.%20Plant%20Pathol.&volume=24&pages=89-106&publication_year=2023&author=Javed%2CMA) 
    
2.  Strelkov, S. E. & Hwang, S. F. Clubroot in the Canadian canola crop: 10 years into the outbreak. _Can. J. Plant. Pathol._ **36**, 27–36 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVOns7zP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clubroot%20in%20the%20Canadian%20canola%20crop%3A%2010%E2%80%89years%20into%20the%20outbreak&journal=Can.%20J.%20Plant.%20Pathol.&volume=36&pages=27-36&publication_year=2014&author=Strelkov%2CSE&author=Hwang%2CSF) 
    
3.  Chai, A. L., Xie, X. W., Shi, Y. X. & Li, B. J. Research status of clubroot (_Plasmodiophora brassicae_) on cruciferous crops in China. _Can. J. Plant Pathol._ **36**, 142–153 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Research%20status%20of%20clubroot%20%28Plasmodiophora%20brassicae%29%20on%20cruciferous%20crops%20in%20China&journal=Can.%20J.%20Plant%20Pathol.&volume=36&pages=142-153&publication_year=2014&author=Chai%2CAL&author=Xie%2CXW&author=Shi%2CYX&author=Li%2CBJ) 
    
4.  Aist, J. R. & Williams, P. The cytology and kinetics of cabbage root hair penetration by _Plasmodiophora brassicae_. _Can. J. Bot._ **49**, 2023–2034 (1971).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20cytology%20and%20kinetics%20of%20cabbage%20root%20hair%20penetration%20by%20Plasmodiophora%20brassicae&journal=Can.%20J.%20Bot.&volume=49&pages=2023-2034&publication_year=1971&author=Aist%2CJR&author=Williams%2CP) 
    
5.  Ingram, D. & Tommerup, I. C. The life history of _Plasmodiophora brassicae_ Woron. _Proc. R. Soc. Lond. B_ **180**, 103–112 (1972).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20life%20history%20of%20Plasmodiophora%20brassicae%20Woron&journal=Proc.%20R.%20Soc.%20Lond.%20B&volume=180&pages=103-112&publication_year=1972&author=Ingram%2CD&author=Tommerup%2CIC) 
    
6.  Liu, L. et al. Refining the life cycle of _Plasmodiophora brassicae_. _Phytopathology_ **110**, 1704–1712 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32407251)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Refining%20the%20life%20cycle%20of%20Plasmodiophora%20brassicae&journal=Phytopathology&volume=110&pages=1704-1712&publication_year=2020&author=Liu%2CL) 
    
7.  Liu, L. et al. Comparing the infection biology of _Plasmodiophora brassicae_ in clubroot susceptible and resistant hosts and non-hosts. _Front. Microbiol._ **11**, 507036 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33178139)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7596292)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparing%20the%20infection%20biology%20of%20Plasmodiophora%20brassicae%20in%20clubroot%20susceptible%20and%20resistant%20hosts%20and%20non-hosts&journal=Front.%20Microbiol.&volume=11&publication_year=2020&author=Liu%2CL) 
    
8.  Hasan, J., Megha, S. & Rahman, H. Clubroot in _Brassica_: recent advances in genomics, breeding, and disease management. _Genome_ **64**, 735–760 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33651640)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clubroot%20in%20Brassica%3A%20recent%20advances%20in%20genomics%2C%20breeding%2C%20and%20disease%20management&journal=Genome&volume=64&pages=735-760&publication_year=2021&author=Hasan%2CJ&author=Megha%2CS&author=Rahman%2CH) 
    
9.  Mehraj, H. et al. Genetics of clubroot and _Fusarium_ wilt disease resistance in _Brassica_ vegetables: the application of marker assisted breeding for disease resistance. _Plants (Basel)_ **9**, 726 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFeksbrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32526827)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetics%20of%20clubroot%20and%20Fusarium%20wilt%20disease%20resistance%20in%20Brassica%20vegetables%3A%20the%20application%20of%20marker%20assisted%20breeding%20for%20disease%20resistance&journal=Plants%20%28Basel%29&volume=9&publication_year=2020&author=Mehraj%2CH) 
    
10.  Lai, S. et al. Clubroot resistant in cruciferous crops: recent advances in genes and QTLs identification and utilization. _Hortic. Res._ **12**, uhaf105 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhvVGqu7nJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=40406504)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC12096309)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clubroot%20resistant%20in%20cruciferous%20crops%3A%20recent%20advances%20in%20genes%20and%20QTLs%20identification%20and%20utilization&journal=Hortic.%20Res.&volume=12&publication_year=2025&author=Lai%2CS) 
    
11.  Ueno, H. et al. Molecular characterization of the _CRa_ gene conferring clubroot resistance in _Brassica rapa_. _Plant Mol. Biol._ **80**, 621–629 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhslansbrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23054353)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20characterization%20of%20the%20CRa%20gene%20conferring%20clubroot%20resistance%20in%20Brassica%20rapa&journal=Plant%20Mol.%20Biol.&volume=80&pages=621-629&publication_year=2012&author=Ueno%2CH) 
    
12.  Hatakeyama, K. et al. Identification and characterization of _Crr1a_, a gene for resistance to clubroot disease (_Plasmodiophora brassicae_ Woronin) in _Brassica rapa_ L. _PLoS ONE_ **8**, e54745 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXis1arsLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23382954)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3559844)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20and%20characterization%20of%20Crr1a%2C%20a%20gene%20for%20resistance%20to%20clubroot%20disease%20%28Plasmodiophora%20brassicae%20Woronin%29%20in%20Brassica%20rapa%20L&journal=PLoS%20ONE&volume=8&publication_year=2013&author=Hatakeyama%2CK) 
    
13.  Yang, Z. et al. R gene triplication confers European fodder turnip with improved clubroot resistance. _Plant Biotechnol. J._ **20**, 1502–1517 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVGnsrfP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35445530)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9342621)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=R%20gene%20triplication%20confers%20European%20fodder%20turnip%20with%20improved%20clubroot%20resistance&journal=Plant%20Biotechnol.%20J.&volume=20&pages=1502-1517&publication_year=2022&author=Yang%2CZ) 
    
14.  Williams, P. H. A system for the determination of races of _Plasmodiophora brassicae_ that infect cabbage and rutabaga. _Phytopathology_ **56**, 624–626 (1966).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20system%20for%20the%20determination%20of%20races%20of%20Plasmodiophora%20brassicae%20that%20infect%20cabbage%20and%20rutabaga&journal=Phytopathology&volume=56&pages=624-626&publication_year=1966&author=Williams%2CPH) 
    
15.  Buczacki, S. et al. Study of physiologic specialization in _Plasmodiophora brassicae_: proposals for attempted rationalization through an international approach. _Trans. Br. Mycological Soc._ **65**, 295–303 (1975).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Study%20of%20physiologic%20specialization%20in%20Plasmodiophora%20brassicae%3A%20proposals%20for%20attempted%20rationalization%20through%20an%20international%20approach&journal=Trans.%20Br.%20Mycological%20Soc.&volume=65&pages=295-303&publication_year=1975&author=Buczacki%2CS) 
    
16.  Zamani-Noor, N. et al. Pathotype characterization of _Plasmodiophora brassicae_, the cause of clubroot in central Europe and Sweden (2016–2020). _Pathogens_ **11**, 1440 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslWmtro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36558774)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9785528)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathotype%20characterization%20of%20Plasmodiophora%20brassicae%2C%20the%20cause%20of%20clubroot%20in%20central%20Europe%20and%20Sweden%20%282016%E2%80%932020%29&journal=Pathogens&volume=11&publication_year=2022&author=Zamani-Noor%2CN) 
    
17.  Zeng, L. et al. Pathotype characterization of _Plasmodiophora brassicae_ by European clubroot differential and Williams sets in China. _Plant Dis._ **108**, 847–851 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathotype%20characterization%20of%20Plasmodiophora%20brassicae%20by%20European%20clubroot%20differential%20and%20Williams%20sets%20in%20China&journal=Plant%20Dis.&volume=108&pages=847-851&publication_year=2023&author=Zeng%2CL) 
    
18.  Hollman, K. B., Hwang, S. F., Manolii, V. P. & Strelkov, S. E. Pathotypes of _Plasmodiophora brassicae_ collected from clubroot resistant canola (_Brassica napus_ L.) cultivars in western Canada in 2017–2018. _Can. J. Plant Pathol._ **43**, 622–630 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVGgtrk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathotypes%20of%20Plasmodiophora%20brassicae%20collected%20from%20clubroot%20resistant%20canola%20%28Brassica%20napus%20L.%29%20cultivars%20in%20western%20Canada%20in%202017%E2%80%932018&journal=Can.%20J.%20Plant%20Pathol.&volume=43&pages=622-630&publication_year=2021&author=Hollman%2CKB&author=Hwang%2CSF&author=Manolii%2CVP&author=Strelkov%2CSE) 
    
19.  Strelkov, S. E., Hwang, S. F., Manolii, V. P., Cao, T. & Feindel, D. Emergence of new virulence phenotypes of _Plasmodiophora brassicae_ on canola (_Brassica napus_) in Alberta, Canada. _Eur. J. Plant Pathol._ **145**, 517–529 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Emergence%20of%20new%20virulence%20phenotypes%20of%20Plasmodiophora%20brassicae%20on%20canola%20%28Brassica%20napus%29%20in%20Alberta%2C%20Canada&journal=Eur.%20J.%20Plant%20Pathol.&volume=145&pages=517-529&publication_year=2016&author=Strelkov%2CSE&author=Hwang%2CSF&author=Manolii%2CVP&author=Cao%2CT&author=Feindel%2CD) 
    
20.  Zhang, Y. et al. Structural variation reshapes population gene expression and trait variation in 2,105 _Brassica napus_ accessions. _Nat. Genet._ **56**, 2538–2550 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXisVaqsbzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39501128)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11549052)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Structural%20variation%20reshapes%20population%20gene%20expression%20and%20trait%20variation%20in%202%2C105%20Brassica%20napus%20accessions&journal=Nat.%20Genet.&volume=56&pages=2538-2550&publication_year=2024&author=Zhang%2CY) 
    
21.  Ren, D., An, L., Li, B., Qiao, L. & Liu, W. Efficient weighting methods for genomic best linear-unbiased prediction (BLUP) adapted to the genetic architectures of quantitative traits. _Heredity_ **126**, 320–334 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVSjurnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32980863)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20weighting%20methods%20for%20genomic%20best%20linear-unbiased%20prediction%20%28BLUP%29%20adapted%20to%20the%20genetic%20architectures%20of%20quantitative%20traits&journal=Heredity&volume=126&pages=320-334&publication_year=2021&author=Ren%2CD&author=An%2CL&author=Li%2CB&author=Qiao%2CL&author=Liu%2CW) 
    
22.  Wang, W. et al. WeiTsing, a pericycle-expressed ion channel, safeguards the stele to confer clubroot resistance. _Cell_ **186**, 2656–2671.e18 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1WlsrrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37295403)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WeiTsing%2C%20a%20pericycle-expressed%20ion%20channel%2C%20safeguards%20the%20stele%20to%20confer%20clubroot%20resistance&journal=Cell&volume=186&pages=2656-2671.e18&publication_year=2023&author=Wang%2CW) 
    
23.  Nishimura, M. T. et al. Loss of a callose synthase results in salicylic acid-dependent disease resistance. _Science_ **301**, 969–972 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXmt1elsLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12920300)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20a%20callose%20synthase%20results%20in%20salicylic%20acid-dependent%20disease%20resistance&journal=Science&volume=301&pages=969-972&publication_year=2003&author=Nishimura%2CMT) 
    
24.  Jacobs, A. K. et al. An _Arabidopsis_ callose synthase, GSL5, is required for wound and papillary callose formation. _Plant Cell_ **15**, 2503–2513 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXpt1Ortb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14555698)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC280557)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20Arabidopsis%20callose%20synthase%2C%20GSL5%2C%20is%20required%20for%20wound%20and%20papillary%20callose%20formation&journal=Plant%20Cell&volume=15&pages=2503-2513&publication_year=2003&author=Jacobs%2CAK) 
    
25.  Ellinger, D. et al. Elevated early callose deposition results in complete penetration resistance to powdery mildew in _Arabidopsis_. _Plant Physiol._ **161**, 1433–1444 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXmvFKrurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23335625)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3585607)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elevated%20early%20callose%20deposition%20results%20in%20complete%20penetration%20resistance%20to%20powdery%20mildew%20in%20Arabidopsis&journal=Plant%20Physiol.&volume=161&pages=1433-1444&publication_year=2013&author=Ellinger%2CD) 
    
26.  Liu, L. et al. The plant trans-Golgi network component ECHIDNA regulates defense, cell death, and endoplasmic reticulum stress. _Plant Physiol._ **191**, 558–574 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslWmu7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36018261)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20plant%20trans-Golgi%20network%20component%20ECHIDNA%20regulates%20defense%2C%20cell%20death%2C%20and%20endoplasmic%20reticulum%20stress&journal=Plant%20Physiol.&volume=191&pages=558-574&publication_year=2023&author=Liu%2CL) 
    
27.  Ellinger, D. & Voigt, C. A. Callose biosynthesis in _Arabidopsis_ with a focus on pathogen response: what we have learned within the last decade. _Ann. Bot._ **114**, 1349–1358 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitleltbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24984713)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4195556)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Callose%20biosynthesis%20in%20Arabidopsis%20with%20a%20focus%20on%20pathogen%20response%3A%20what%20we%20have%20learned%20within%20the%20last%20decade&journal=Ann.%20Bot.&volume=114&pages=1349-1358&publication_year=2014&author=Ellinger%2CD&author=Voigt%2CCA) 
    
28.  Mittler, R., Shulaev, V. & Lam, E. Coordinated activation of programmed cell death and defense mechanisms in transgenic tobacco plants expressing a bacterial proton pump. _Plant Cell_ **7**, 29–42 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXjsVyjsb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12242350)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC160762)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coordinated%20activation%20of%20programmed%20cell%20death%20and%20defense%20mechanisms%20in%20transgenic%20tobacco%20plants%20expressing%20a%20bacterial%20proton%20pump&journal=Plant%20Cell&volume=7&pages=29-42&publication_year=1995&author=Mittler%2CR&author=Shulaev%2CV&author=Lam%2CE) 
    
29.  Mittler, R., Simon, L. & Lam, E. Pathogen-induced programmed cell death in tobacco. _J. Cell Sci._ **110**, 1333–1344 (1997).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXktF2mur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9202394)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pathogen-induced%20programmed%20cell%20death%20in%20tobacco&journal=J.%20Cell%20Sci.&volume=110&pages=1333-1344&publication_year=1997&author=Mittler%2CR&author=Simon%2CL&author=Lam%2CE) 
    
30.  Mittler, R. et al. Transgenic tobacco plants with reduced capability to detoxify reactive oxygen intermediates are hyperresponsive to pathogen infection. _Proc. Natl Acad. Sci. USA_ **96**, 14165–14170 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXns1yjs7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10570216)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC24208)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transgenic%20tobacco%20plants%20with%20reduced%20capability%20to%20detoxify%20reactive%20oxygen%20intermediates%20are%20hyperresponsive%20to%20pathogen%20infection&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=96&pages=14165-14170&publication_year=1999&author=Mittler%2CR) 
    
31.  Matsusaki, M. et al. The protein disulfide isomerase family: from proteostasis to pathogenesis. _Biochim. Biophys. Acta Gen. Subj._ **1864**, 129338 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXns1yrs7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30986509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20protein%20disulfide%20isomerase%20family%3A%20from%20proteostasis%20to%20pathogenesis&journal=Biochim.%20Biophys.%20Acta%20Gen.%20Subj.&volume=1864&publication_year=2020&author=Matsusaki%2CM) 
    
32.  Zhou, J. M. & Zhang, Y. Plant immunity: danger perception and signaling. _Cell_ **181**, 978–989 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVSjsrnL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32442407)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plant%20immunity%3A%20danger%20perception%20and%20signaling&journal=Cell&volume=181&pages=978-989&publication_year=2020&author=Zhou%2CJM&author=Zhang%2CY) 
    
33.  Zhang, J., Coaker, G., Zhou, J. M. & Dong, X. Plant immune mechanisms: from reductionistic to holistic points of view. _Mol. Plant_ **13**, 1358–1378 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFaqurvN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32916334)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Plant%20immune%20mechanisms%3A%20from%20reductionistic%20to%20holistic%20points%20of%20view&journal=Mol.%20Plant&volume=13&pages=1358-1378&publication_year=2020&author=Zhang%2CJ&author=Coaker%2CG&author=Zhou%2CJM&author=Dong%2CX) 
    
34.  Ochoa, J. C. et al. Natural variation in _Arabidopsis_ responses to _Plasmodiophora brassicae_ reveals an essential role for _Resistance to_ _Plasmodiophora brassicae 1_ (_RPB1_). _Plant J._ **116**, 1421–1440 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVaqs7nP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37646674)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Natural%20variation%20in%20Arabidopsis%20responses%20to%20Plasmodiophora%20brassicae%20reveals%20an%20essential%20role%20for%20Resistance%20to%20Plasmodiophora%20brassicae%201%20%28RPB1%29&journal=Plant%20J.&volume=116&pages=1421-1440&publication_year=2023&author=Ochoa%2CJC) 
    
35.  Li, S. et al. Genome-edited powdery mildew resistance in wheat without growth penalties. _Nature_ **602**, 455–460 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtlSgurY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35140403)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-edited%20powdery%20mildew%20resistance%20in%20wheat%20without%20growth%20penalties&journal=Nature&volume=602&pages=455-460&publication_year=2022&author=Li%2CS) 
    
36.  Wang, N. et al. Inactivation of a wheat protein kinase gene confers broad-spectrum resistance to rust fungi. _Cell_ **185**, 2961–2974.e19 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVWmtrnN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35839760)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inactivation%20of%20a%20wheat%20protein%20kinase%20gene%20confers%20broad-spectrum%20resistance%20to%20rust%20fungi&journal=Cell&volume=185&pages=2961-2974.e19&publication_year=2022&author=Wang%2CN) 
    
37.  Sha, G. et al. Genome editing of a rice CDP-DAG synthase confers multipathogen resistance. _Nature_ **618**, 1017–1023 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXht1ejsLfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37316672)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11575942)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20editing%20of%20a%20rice%20CDP-DAG%20synthase%20confers%20multipathogen%20resistance&journal=Nature&volume=618&pages=1017-1023&publication_year=2023&author=Sha%2CG) 
    
38.  Zhou, X. et al. Loss of function of a rice TPR-domain RNA-binding protein confers broad-spectrum disease resistance. _Proc. Natl Acad. Sci. USA_ **115**, 3174–3179 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1Chs7bP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29432165)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5866533)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20function%20of%20a%20rice%20TPR-domain%20RNA-binding%20protein%20confers%20broad-spectrum%20disease%20resistance&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=115&pages=3174-3179&publication_year=2018&author=Zhou%2CX) 
    
39.  Nawrath, C. & Métraux, J.-P. Salicylic acid induction-deficient mutants of _Arabidopsis_ express PR-2 and PR-5 and accumulate high levels of camalexin after pathogen inoculation. _Plant Cell_ **11**, 1393–1404 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmtVWmu74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10449575)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC144293)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Salicylic%20acid%20induction-deficient%20mutants%20of%20Arabidopsis%20express%20PR-2%20and%20PR-5%20and%20accumulate%20high%20levels%20of%20camalexin%20after%20pathogen%20inoculation&journal=Plant%20Cell&volume=11&pages=1393-1404&publication_year=1999&author=Nawrath%2CC&author=M%C3%A9traux%2CJ-P) 
    
40.  Staswick, P. E., Tiryaki, I. & Rowe, M. L. Jasmonate response locus _JAR1_ and several related _Arabidopsis_ genes encode enzymes of the firefly luciferase superfamily that show activity on jasmonic, salicylic, and indole-3-acetic acids in an assay for adenylation. _Plant Cell_ **14**, 1405–1415 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xlt1alsbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12084835)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC150788)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Jasmonate%20response%20locus%20JAR1%20and%20several%20related%20Arabidopsis%20genes%20encode%20enzymes%20of%20the%20firefly%20luciferase%20superfamily%20that%20show%20activity%20on%20jasmonic%2C%20salicylic%2C%20and%20indole-3-acetic%20acids%20in%20an%20assay%20for%20adenylation&journal=Plant%20Cell&volume=14&pages=1405-1415&publication_year=2002&author=Staswick%2CPE&author=Tiryaki%2CI&author=Rowe%2CML) 
    
41.  Roman, G., Lubarsky, B., Kieber, J. J., Rothenberg, M. & Ecker, J. R. Genetic analysis of ethylene signal transduction in _Arabidopsis thaliana_: five novel mutant loci integrated into a stress response pathway. _Genetics_ **139**, 1393–1409 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XhsFSrurg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7768447)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1206465)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20analysis%20of%20ethylene%20signal%20transduction%20in%20Arabidopsis%20thaliana%3A%20five%20novel%20mutant%20loci%20integrated%20into%20a%20stress%20response%20pathway&journal=Genetics&volume=139&pages=1393-1409&publication_year=1995&author=Roman%2CG&author=Lubarsky%2CB&author=Kieber%2CJJ&author=Rothenberg%2CM&author=Ecker%2CJR) 
    
42.  Song, J. T., Lu, H. & Greenberg, J. T. Divergent roles in _Arabidopsis thaliana_ development and defense of two homologous genes, _ABERRANT GROWTH AND DEATH2_ and _AGD2-LIKE DEFENSE RESPONSE PROTEIN1_, encoding novel aminotransferases. _Plant Cell_ **16**, 353–366 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhsFKis78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14729919)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC341909)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Divergent%20roles%20in%20Arabidopsis%20thaliana%20development%20and%20defense%20of%20two%20homologous%20genes%2C%20ABERRANT%20GROWTH%20AND%20DEATH2%20and%20AGD2-LIKE%20DEFENSE%20RESPONSE%20PROTEIN1%2C%20encoding%20novel%20aminotransferases&journal=Plant%20Cell&volume=16&pages=353-366&publication_year=2004&author=Song%2CJT&author=Lu%2CH&author=Greenberg%2CJT) 
    
43.  Jirage, D. et al. _Arabidopsis thaliana PAD4_ encodes a lipase-like gene that is important for salicylic acid signaling. _Proc. Natl Acad. Sci. USA_ **96**, 13583–13588 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXns1Glsr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10557364)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC23991)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Arabidopsis%20thaliana%20PAD4%20encodes%20a%20lipase-like%20gene%20that%20is%20important%20for%20salicylic%20acid%20signaling&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=96&pages=13583-13588&publication_year=1999&author=Jirage%2CD) 
    
44.  Lawton, K. et al. Systemic acquired resistance in _Arabidopsis_ requires salicylic acid but not ethylene. _Mol. Plant Microbe Interact._ **8**, 863–870 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XitFaisA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8664495)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systemic%20acquired%20resistance%20in%20Arabidopsis%20requires%20salicylic%20acid%20but%20not%20ethylene&journal=Mol.%20Plant%20Microbe%20Interact.&volume=8&pages=863-870&publication_year=1995&author=Lawton%2CK) 
    
45.  Zhao, C. et al. Genome-wide association analysis combined with quantitative trait loci mapping and dynamic transcriptome unveil the genetic control of seed oil content in _Brassica napus_ L. _Front. Plant Sci._ **13**, 929197 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35845656)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9283957)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20analysis%20combined%20with%20quantitative%20trait%20loci%20mapping%20and%20dynamic%20transcriptome%20unveil%20the%20genetic%20control%20of%20seed%20oil%20content%20in%20Brassica%20napus%20L&journal=Front.%20Plant%20Sci.&volume=13&publication_year=2022&author=Zhao%2CC) 
    
46.  Gabriel, S. B. The structure of haplotype blocks in the human genome. _Science_ **296**, 2225–2229 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XkvFGhtrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12029063)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&journal=Science&volume=296&pages=2225-2229&publication_year=2002&author=Gabriel%2CSB) 
    
47.  Zhang, C., Dong, S. S., Xu, J. Y., He, W. M. & Yang, T. L. PopLDdecay: a fast and effective tool for linkage disequilibrium decay analysis based on variant call format files. _Bioinformatics_ **35**, 1786–1788 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PopLDdecay%3A%20a%20fast%20and%20effective%20tool%20for%20linkage%20disequilibrium%20decay%20analysis%20based%20on%20variant%20call%20format%20files&journal=Bioinformatics&volume=35&pages=1786-1788&publication_year=2018&author=Zhang%2CC&author=Dong%2CSS&author=Xu%2CJY&author=He%2CWM&author=Yang%2CTL) 
    
48.  Dong, S.-S. et al. LDBlockShow: a fast and convenient tool for visualizing linkage disequilibrium and haplotype blocks based on variant call format files. _Brief. Bioinform._ **22**, bbaa227 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LDBlockShow%3A%20a%20fast%20and%20convenient%20tool%20for%20visualizing%20linkage%20disequilibrium%20and%20haplotype%20blocks%20based%20on%20variant%20call%20format%20files&journal=Brief.%20Bioinform.&volume=22&publication_year=2020&author=Dong%2CS-S) 
    
49.  Retief, J. D. Phylogenetic analysis using PHYLIP. _Methods Mol. Biol._ **132**, 243–258 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmslKqsLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10547839)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phylogenetic%20analysis%20using%20PHYLIP&journal=Methods%20Mol.%20Biol.&volume=132&pages=243-258&publication_year=2000&author=Retief%2CJD) 
    
50.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
51.  Alexander, D. H. et al. Fast model-based estimation of ancestry in unrelated individuals. _Genome Res._ **19**, 1655–1664 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFCjsLvL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19648217)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752134)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&journal=Genome%20Res.&volume=19&pages=1655-1664&publication_year=2009&author=Alexander%2CDH) 
    
52.  Zand Karimi, H. & Innes, R. W. Molecular mechanisms underlying host-induced gene silencing. _Plant Cell._ **34**, 3183–3199 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35666177)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9421479)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20mechanisms%20underlying%20host-induced%20gene%20silencing&journal=Plant%20Cell.&volume=34&pages=3183-3199&publication_year=2022&author=Zand%20Karimi%2CH&author=Innes%2CRW) 
    
53.  Liu, H. et al. CRISPR-P 2.0: an improved CRISPR–Cas9 tool for genome editing in plants. _Mol. Plant_ **10**, 530–532 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslejurg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28089950)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CRISPR-P%202.0%3A%20an%20improved%20CRISPR%E2%80%93Cas9%20tool%20for%20genome%20editing%20in%20plants&journal=Mol.%20Plant&volume=10&pages=530-532&publication_year=2017&author=Liu%2CH) 
    
54.  Ma, X. et al. A robust CRISPR/Cas9 system for convenient, high-efficiency multiplex genome editing in monocot and dicot plants. _Mol. Plant_ **8**, 1274–1284 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFygsrbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25917172)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20robust%20CRISPR%2FCas9%20system%20for%20convenient%2C%20high-efficiency%20multiplex%20genome%20editing%20in%20monocot%20and%20dicot%20plants&journal=Mol.%20Plant&volume=8&pages=1274-1284&publication_year=2015&author=Ma%2CX) 
    
55.  Clough, S. J. & Bent, A. F. Floral dip: a simplified method for _Agrobacterium_\-mediated transformation of _Arabidopsis thaliana_. _Plant J._ **16**, 735–743 (2010).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Floral%20dip%3A%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana&journal=Plant%20J.&volume=16&pages=735-743&publication_year=2010&author=Clough%2CSJ&author=Bent%2CAF) 
    
56.  Schuller, A. & Ludwig‐Müller, J. Histological methods to detect the clubroot pathogen _Plasmodiophora brassicae_ during its complex life cycle. _Plant Pathol._ **65**, 1223–1237 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVCns7fF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Histological%20methods%20to%20detect%20the%20clubroot%20pathogen%20Plasmodiophora%20brassicae%20during%20its%20complex%20life%20cycle&journal=Plant%20Pathol.&volume=65&pages=1223-1237&publication_year=2016&author=Schuller%2CA&author=Ludwig%E2%80%90M%C3%BCller%2CJ) 
    
57.  Kim, D., Langmead, B. & Salzberg, S. L. HISAT: a fast spliced aligner with low memory requirements. _Nat. Methods_ **12**, 357–360 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvFOnsL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25751142)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655817)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HISAT%3A%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&journal=Nat.%20Methods&volume=12&pages=357-360&publication_year=2015&author=Kim%2CD&author=Langmead%2CB&author=Salzberg%2CSL) 
    
58.  Chalhoub, B. et al. Early allopolyploid evolution in the post-Neolithic _Brassica napus_ oilseed genome. _Science_ **345**, 950–953 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtlOmsr%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25146293)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Early%20allopolyploid%20evolution%20in%20the%20post-Neolithic%20Brassica%20napus%20oilseed%20genome&journal=Science&volume=345&pages=950-953&publication_year=2014&author=Chalhoub%2CB) 
    
59.  Trapnell, C. et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. _Nat. Biotechnol._ **28**, 511–515 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlsVyitLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20436464)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3146043)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&journal=Nat.%20Biotechnol.&volume=28&pages=511-515&publication_year=2010&author=Trapnell%2CC) 
    
60.  Leng, N. et al. EBSeq: an empirical Bayes hierarchical model for inference in RNA-seq experiments. _Bioinformatics_ **29**, 1035–1043 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXlvVKitLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23428641)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3624807)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=EBSeq%3A%20an%20empirical%20Bayes%20hierarchical%20model%20for%20inference%20in%20RNA-seq%20experiments&journal=Bioinformatics&volume=29&pages=1035-1043&publication_year=2013&author=Leng%2CN) 
    
61.  Anders, S. & Huber, W. Differential expression analysis for sequence count data. _Genome Biol._ **11**, R106 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVahs7bI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20979621)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218662)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Differential%20expression%20analysis%20for%20sequence%20count%20data&journal=Genome%20Biol.&volume=11&publication_year=2010&author=Anders%2CS&author=Huber%2CW) 
    
62.  Bardou, P., Mariette, J., Escudié, F., Djemiel, C. & Klopp, C. jvenn: an interactive Venn diagram viewer. _BMC Bioinformatics_ **15**, 293 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25176396)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4261873)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=jvenn%3A%20an%20interactive%20Venn%20diagram%20viewer&journal=BMC%20Bioinformatics&volume=15&publication_year=2014&author=Bardou%2CP&author=Mariette%2CJ&author=Escudi%C3%A9%2CF&author=Djemiel%2CC&author=Klopp%2CC) 
    
63.  Livak, K. J. & Schmittgen, T. D. Analysis of relative gene expression data using real-time quantitative PCR and the 2−ΔΔCT method. _Methods_ **25**, 402–408 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XhtFelt7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11846609)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202%E2%88%92%CE%94%CE%94CT%20method&journal=Methods&volume=25&pages=402-408&publication_year=2001&author=Livak%2CKJ&author=Schmittgen%2CTD) 
    
64.  Tian, G. et al. The crystal structure of yeast protein disulfide isomerase suggests cooperativity between its active sites. _Cell_ **124**, 61–73 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XntVGksg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16413482)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20crystal%20structure%20of%20yeast%20protein%20disulfide%20isomerase%20suggests%20cooperativity%20between%20its%20active%20sites&journal=Cell&volume=124&pages=61-73&publication_year=2006&author=Tian%2CG) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02306-y?format=refman&flavour=references)

Acknowledgements
----------------

We thank A. Wang from Wuhan Academy of Agricultural Sciences for providing _B._ _rapa_ and _B._ _oleracea_ seeds, and the Center for Protein Research, Huazhong Agricultural University, for service. This study was supported by the Innovation Program of the Chinese Academy of Agricultural Sciences (CAAS-CSNCB-202303), National Biological Breeding-Major Projects (2023ZD04042), National Natural Science Foundation of China (U20A2034, 31601628), China Agriculture Research System of MOF and MARA (CARS-12) and Wuhan Knowledge Innovation Program (2022020801010296).

Author information
------------------

Author notes

1.  These authors contributed equally: Yupo Wu, Chuanji Zhao, Yi Zhang, Cuicui Shen.
    

### Authors and Affiliations

1.  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences/The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs, Wuhan, China
    
    Yupo Wu, Chuanji Zhao, Yi Zhang, Cuicui Shen, Yuanyuan Zhang, Xiong Zhang, Lixia Gao, Lingyi Zeng, Qinglin Ke, Li Qin, Fan Liu, Junyan Huang, Li Ren, Yueying Liu, Hongtao Cheng, Chaobo Tong, Qiong Hu, Xiaohui Cheng, Shengyi Liu & Lijiang Liu
    
2.  State Key Laboratory of Agricultural Microbiology, Huazhong Agricultural University, Wuhan, China
    
    Lixia Gao
    
3.  Department of Biology, University of Saskatchewan, Saskatoon, Saskatchewan, Canada
    
    Yangdou Wei
    

Authors

1.  Yupo Wu
    
    [View author publications](https://www.nature.com/search?author=Yupo%20Wu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yupo%20Wu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yupo%20Wu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Chuanji Zhao
    
    [View author publications](https://www.nature.com/search?author=Chuanji%20Zhao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chuanji%20Zhao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chuanji%20Zhao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Yi Zhang
    
    [View author publications](https://www.nature.com/search?author=Yi%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yi%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yi%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Cuicui Shen
    
    [View author publications](https://www.nature.com/search?author=Cuicui%20Shen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cuicui%20Shen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cuicui%20Shen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Yuanyuan Zhang
    
    [View author publications](https://www.nature.com/search?author=Yuanyuan%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuanyuan%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuanyuan%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Xiong Zhang
    
    [View author publications](https://www.nature.com/search?author=Xiong%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiong%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiong%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Lixia Gao
    
    [View author publications](https://www.nature.com/search?author=Lixia%20Gao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lixia%20Gao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lixia%20Gao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Lingyi Zeng
    
    [View author publications](https://www.nature.com/search?author=Lingyi%20Zeng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lingyi%20Zeng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lingyi%20Zeng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Qinglin Ke
    
    [View author publications](https://www.nature.com/search?author=Qinglin%20Ke)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qinglin%20Ke) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qinglin%20Ke%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Li Qin
    
    [View author publications](https://www.nature.com/search?author=Li%20Qin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Li%20Qin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Li%20Qin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Fan Liu
    
    [View author publications](https://www.nature.com/search?author=Fan%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Fan%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Fan%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Junyan Huang
    
    [View author publications](https://www.nature.com/search?author=Junyan%20Huang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Junyan%20Huang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Junyan%20Huang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Li Ren
    
    [View author publications](https://www.nature.com/search?author=Li%20Ren)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Li%20Ren) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Li%20Ren%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Yueying Liu
    
    [View author publications](https://www.nature.com/search?author=Yueying%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yueying%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yueying%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Hongtao Cheng
    
    [View author publications](https://www.nature.com/search?author=Hongtao%20Cheng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hongtao%20Cheng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hongtao%20Cheng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Chaobo Tong
    
    [View author publications](https://www.nature.com/search?author=Chaobo%20Tong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chaobo%20Tong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chaobo%20Tong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Qiong Hu
    
    [View author publications](https://www.nature.com/search?author=Qiong%20Hu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qiong%20Hu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qiong%20Hu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Xiaohui Cheng
    
    [View author publications](https://www.nature.com/search?author=Xiaohui%20Cheng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiaohui%20Cheng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiaohui%20Cheng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Yangdou Wei
    
    [View author publications](https://www.nature.com/search?author=Yangdou%20Wei)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yangdou%20Wei) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yangdou%20Wei%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Shengyi Liu
    
    [View author publications](https://www.nature.com/search?author=Shengyi%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shengyi%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shengyi%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Lijiang Liu
    
    [View author publications](https://www.nature.com/search?author=Lijiang%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lijiang%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lijiang%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

L.L. and S.L. designed the study. L.L., Y.W. and C.S. performed molecular experiments. L.L., C.Z., Yi Zhang, L.G., L.Z., F.L., X.C. and L.R. performed the clubroot resistance test of the _B._ _napus_ population in the fields. Y.W., X.Z., Q.K., Yuanyuan Zhang and C.T. performed the bioinformatic analysis. S.L., J.H. and Y.L. developed and maintained the accession population. X.C., C.Z., Yi Zhang and X.Z. performed the analysis of agronomic traits of _B._ _napus_ plants in the fields. Yi Zhang, Y.W. and L.Q. performed inoculations of _P._ _brassicae_ and phenotypic scoring under controlled conditions. L.L., S.L., C.Z. and Y.W. prepared the figures and wrote the paper. Yangdou Wei, H.C. and Q.H. reviewed and edited the paper and contributed germplasm.

### Corresponding authors

Correspondence to [Shengyi Liu](mailto:liusy@oilcrops.cn) or [Lijiang Liu](mailto:liulijiang@caas.cn).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Graham Teakle and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02306-y#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Population structure of 244 _B. napus_ accessions.](https://www.nature.com/articles/s41588-025-02306-y/figures/7)

**a**, Phylogenetic tree of 244 _B. napus_ accessions. Line colors represent three _B. napus_ ecotypes. **b**, Principal component analysis plot of 244 _B. napus_ accessions. Dot colors represent three _B. napus_ ecotypes. The numbers in brackets represent the fraction of variance explained by each component. **c**, STRUCTURE analysis of 244 _B. napus_ accessions. Each color represents one ancestral population. Each line is represented by a vertical bar, and the length of each colored segment in each vertical bar represents the proportion contributed by ancestral populations. The K (the number of ancestral populations) was set from 2 to 8. No significant population structure was observed among all 244 _B. napus_ accessions.

### [Extended Data Fig. 2 GWAS of field clubroot phenotyping in _B. napus_ accessions with multiple models.](https://www.nature.com/articles/s41588-025-02306-y/figures/8)

**a**, Manhattan plot of the GWAS using general linear model (GLM, upper panel) and the model fixed and random model Circulating Probability Unification (FarmCPU, lower panel). The lead SNPs and associated gene repeatedly identified in both models were pointed with arrows. The red line indicates the Bonferroni-adjusted significance threshold (two-sided) for GWAS (_P_ = 1.8 × 10−8). **b**, Quantile-quantile plot for GWAS of field clubroot phenotyping with 3 different models. The red line indicates the Bonferroni adjusted significance threshold (two-sided) for GWAS (_P_ = 1.8 × 10−8). **c**, LD decay determined by squared correlations of allele frequencies (_r_2) against distance in the 244 _B. napus_ accessions.

### [Extended Data Fig. 3 Sequencing validation of Arabidopsis _gsl5-KO_ mutant generated by genome editing.](https://www.nature.com/articles/s41588-025-02306-y/figures/9)

**a**, Diagram illustrating the two deleted positions and sequences in _GSL5_ gene by genome editing. The interval of the two deleted sites is in 5 bp. **b** and **c**, Chromatogram plot showing the sequence and the quality of each sequenced base around the edited sites in wildtype and _gsl5-KO_ plants. Two deleted sites were indicated with arrows in _gsl5-KO_ in **c** and the deleted sequences were highlighted with gray colors in wildtype in **b**.

### [Extended Data Fig. 4 Resistance test of _GSL5_ inactivation against multiple _P. brassicae_ pathotypes in _B. rapa_ and _B. oleracea._](https://www.nature.com/articles/s41588-025-02306-y/figures/10)

Seven field _Pb_ pathotypes collected from 7 provinces were used for this test and the resistance evaluation was performed at 30 dpi. The recipient parental plants of _B. rapa_ and _B. oleracea_ were respectively used as the susceptible controls. Data below the x-axis represent the sample size. In **a** and **b**, statistical significance was determined by a two-sided Student’s _t_ test.

### [Extended Data Fig. 5 Evaluation of the impact of _GSL5_ inactivation on the key agronomic traits of _B. napus_ in clubroot-free fields.](https://www.nature.com/articles/s41588-025-02306-y/figures/11)

This evaluation was performed in Gansu province (spring ecotype region) and Hubei province (semi-winter ecotype region), respectively. Five agronomic traits were investigated at the mature period of _B. napus_ plants, respectively. Data were presented as mean ± s.d. Statistical significance was determined by a two-sided Student’s _t_ test, with three biological replicate, and each 10 plants.

### [Extended Data Fig. 6 Association of _gsl5_ resistance with ET biosynthesis.](https://www.nature.com/articles/s41588-025-02306-y/figures/12)

**a**, Measurement of ET precursor ACC content in the roots of _gsl5_ and wildtype plants before or 12 days after _Pb_ infection. **b**, Transcriptome profiling of the gene expressions involved in ET biosynthesis, signaling and responsive in the roots before or 9 days and 15 days after _P. brassicae_ inoculation. Data are presented as mean ± s.d. in **a**; mean value in **b**. Statistical significance in **a** was determined by a two-sided Student’s _t_ test. Relative content in **a** and transcriptomic data in **b** were determined with three biological replicate (_n_ = 3).

### [Extended Data Fig. 7 KEGG enrichment of the pathways affected by _GSL5_ inactivation during _Pb_ infection.](https://www.nature.com/articles/s41588-025-02306-y/figures/13)

**a**, Venn diagram to identify the exclusive differentially expressed genes (DEGs, Left) between wildtype and _gsl5_ plants at 9 dpi. Significantly enriched items in the KEGG database were plotted (Right). JA-associated KEGG items were highlighted with red color. **b**, Venn diagram to identify the exclusive differentially expressed genes (DEGs, Left) between wildtype and _gsl5_ plants at 15 dpi. Significantly enriched items in the KEGG database were plotted (Right). JA-associated KEGG items were highlighted with red color.

[Source data](https://www.nature.com/articles/s41588-025-02306-y#MOESM6)

### [Extended Data Fig. 8 Molecular characterization of _Pb_ effector PbPDIa.](https://www.nature.com/articles/s41588-025-02306-y/figures/14)

**a**, The protein diagram of PbPDIa, SP: signal peptide. **b**, The relative expressions of _PbPDIa_ during _Pb_ infection. The expression level of _PbPDIa_ at 8 dpi was used as the control. **c**, Validation of PbPDIa secretion in yeast. CMD-W: sucrose and glucose media lacking tryptophan. YPRAA: raffinose medium. This experiment was repeated twice with a similar result. **d**, Validation of PbPDIa secretion by TTC staining. TTC: 2,3,5-triphenyltetrazolium chloride, reduced by glucose to red 1,3,5-tribenzoic acid. This experiment was repeated twice with a similar result. In **b**, data were presented as mean ± s.d. and the statistical significance were determined by a two-sided Student’s _t_ test.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_MOESM1_ESM.pdf)

Supplementary Notes 1–3.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_MOESM4_ESM.xlsx)

Supplementary Tables 1–7.

Source data
-----------

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_MOESM5_ESM.xlsx)

The raw data of the yield and oil content in the fields for Fig. 3f,g.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_MOESM6_ESM.pdf)

Unprocessed original image of the blots for Fig. 6b,h.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02306-y/MediaObjects/41588_2025_2306_MOESM7_ESM.xlsx)

The library of GO enrichment terms for Extended Data Fig. 7.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Inactivation%20of%20%CE%B2-1%2C3-glucan%20synthase-like%205%20confers%20broad-spectrum%20resistance%20to%20Plasmodiophora%20brassicae%20pathotypes%20in%20cruciferous%20plants&author=Yupo%20Wu%20et%20al&contentID=10.1038%2Fs41588-025-02306-y&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-09-01&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY-NC-ND)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02306-y)

### Cite this article

Wu, Y., Zhao, C., Zhang, Y. _et al._ Inactivation of _β-1,3-glucan synthase-like 5_ confers broad-spectrum resistance to _Plasmodiophora brassicae_ pathotypes in cruciferous plants. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02306-y

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02306-y?format=refman&flavour=citation)

-   Received: 10 August 2024
    
-   Accepted: 21 July 2025
    
-   Published: 01 September 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02306-y
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02306-y.pdf)
