> id: https://www.nature.com/articles/s41588-025-02222-1

> link: https://www.nature.com/articles/s41588-025-02222-1

> title: The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability in uveal melanoma and across multiple tumor types

# The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability in uveal melanoma and across multiple tumor types
_Published on Fri, 04 Jul 2025 00:00:00 GMT_

Abstract
--------

Metastatic uveal melanoma is an aggressive disease with limited effective therapeutic options. To comprehensively map monogenic and digenic dependencies, we performed CRISPR–Cas9 screening in ten extensively profiled human uveal melanoma cell line models. Analysis involved genome-wide single-gene and combinatorial paired-gene CRISPR libraries. Among our 76 uveal melanoma-specific essential genes and 105 synthetic lethal gene pairs, we identified and validated the CDP-diacylglycerol synthase 2 gene (_CDS2_) as a genetic dependency in the context of low CDP-diacylglycerol synthase 1 gene (_CDS1)_ expression. We further demonstrate that _CDS1_/_CDS2_ forms a synthetic lethal interaction in vivo and reveal that _CDS2_ knockout results in the disruption of phosphoinositide synthesis and increased cellular apoptosis and that re-expression of _CDS1_ rescues this cell fitness defect. We extend our analysis using pan-cancer data, confirming increased _CDS2_ essentiality in diverse tumor types with low _CDS1_ expression. Thus, the CDS1/CDS2 axis is a therapeutic target across a range of cancers.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-024-59827-5/MediaObjects/41598_2024_59827_Fig1_HTML.png)

### [Identification and validation of a costimulatory molecule-related signature to predict the prognosis for uveal melanoma patients](https://www.nature.com/articles/s41598-024-59827-5?fromPaywallRec=false)

Article Open access 21 April 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-025-02221-2/MediaObjects/41588_2025_2221_Fig1_HTML.png)

### [Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers](https://www.nature.com/articles/s41588-025-02221-2?fromPaywallRec=false)

Article Open access 04 July 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41419-025-07598-w/MediaObjects/41419_2025_7598_Fig1_HTML.png)

### [CDC25A inhibition sensitizes melanoma cells to doxorubicin and NK cell therapy](https://www.nature.com/articles/s41419-025-07598-w?fromPaywallRec=false)

Article Open access 11 April 2025

Main
----

While the clinical benefit of immune checkpoint inhibition has revolutionized the treatment of cutaneous melanoma, survival outcomes for patients with metastatic uveal melanoma remain exceptionally poor[1](https://www.nature.com/articles/s41588-025-02222-1#ref-CR1 "Khoja, L. et al. Meta-analysis in metastatic uveal melanoma to determine progression free and overall survival benchmarks: an international rare cancers initiative (IRCI) ocular melanoma study. Ann. Oncol. 30, 1370–1380 (2019)."),[2](https://www.nature.com/articles/s41588-025-02222-1#ref-CR2 "Rantala, E. S., Hernberg, M. & Kivela, T. T. Overall survival after treatment for metastatic uveal melanoma: a systematic review and meta-analysis. Melanoma Res. 29, 561–568 (2019)."). Tebentafusp, the first approved systemic therapy, modestly improves survival but is limited to patients with uveal melanoma who carry the human leukocyte antigen (HLA)-A\*02:01 allele[3](https://www.nature.com/articles/s41588-025-02222-1#ref-CR3 "Nathan, P. et al. Overall survival benefit with tebentafusp in metastatic uveal melanoma. N. Engl. J. Med. 385, 1196–1206 (2021)."),[4](https://www.nature.com/articles/s41588-025-02222-1#ref-CR4 "Carvajal, R. D. et al. Clinical and molecular response to tebentafusp in previously treated patients with metastatic uveal melanoma: a phase 2 trial. Nat. Med. 28, 2364–2373 (2022)."). Although our molecular understanding of uveal melanoma has improved, targeted therapies have shown limited antitumor responses[5](https://www.nature.com/articles/s41588-025-02222-1#ref-CR5 "Shoushtari, A. N. et al. A phase Ib study of sotrastaurin, a PKC inhibitor, and alpelisib, a PI3Kα inhibitor, in patients with metastatic uveal melanoma. Cancers (Basel) 13, 5504 (2021)."),[6](https://www.nature.com/articles/s41588-025-02222-1#ref-CR6 "Shoushtari, A. N. et al. A phase 2 trial of everolimus and pasireotide long-acting release in patients with metastatic uveal melanoma. Melanoma Res. 26, 272–277 (2016)."),[7](https://www.nature.com/articles/s41588-025-02222-1#ref-CR7 "Shoushtari, A. N. et al. A randomized phase 2 study of trametinib with or without GSK2141795 in patients with advanced uveal melanoma. J. Clin. Oncol. 34, 9511 (2016)."),[8](https://www.nature.com/articles/s41588-025-02222-1#ref-CR8 "Sacco, J. J. et al. Sunitinib versus dacarbazine as first-line treatment in patients with metastatic uveal melanoma. J. Clin. Oncol. 31, 9031 (2013)."),[9](https://www.nature.com/articles/s41588-025-02222-1#ref-CR9 "Luke, J. J. et al. Randomized phase II trial and tumor mutational spectrum analysis from cabozantinib versus chemotherapy in metastatic uveal melanoma (alliance A091201). Clin. Cancer Res. 26, 804–811 (2020)."),[10](https://www.nature.com/articles/s41588-025-02222-1#ref-CR10 "Patel, S. P. et al. A phase II study of gefitinib in patients with metastatic melanoma. Melanoma Res. 21, 357–363 (2011)."). Thus, new therapeutic targets are urgently needed, and adopting a systematic approach is crucial to comprehensively identify new tumor-intrinsic vulnerabilities.

CRISPR–Cas9 screening of cancer cell lines using single-guide RNAs (sgRNAs) has enabled the high-throughput perturbation of individual genes on a genome-wide scale, leading to the discovery of cell-essential genes[11](https://www.nature.com/articles/s41588-025-02222-1#ref-CR11 "Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR–Cas9 essentiality screens in cancer cells. Nat. Genet. 49, 1779–1784 (2017)."),[12](https://www.nature.com/articles/s41588-025-02222-1#ref-CR12 "Koike-Yusa, H., Li, Y., Tan, E. P., Del Castillo Velasco-Herrera, M. & Yusa, K. Genome-wide recessive genetic screening in mammalian cells with a lentiviral CRISPR-guide RNA library. Nat. Biotechnol. 32, 267–273 (2014)."). Synthetic lethal interactions, where combined perturbation of two genes results in a significant growth/fitness defect, have traditionally been identified by performing large loss-of-function screens in cell lines that harbor defined genetic changes. A valuable source of synthetic lethal interactions are paralog gene pairs, which are derived from a common ancestral gene and often function in common or parallel pathways. Notably, shared functions between paralog gene pairs may enable cells to withstand the loss of one member of the pair[13](https://www.nature.com/articles/s41588-025-02222-1#ref-CR13 "Ryan, C. J., Mehta, I., Kebabci, N. & Adams, D. J. Targeting synthetic lethal paralogs in cancer. Trends Cancer 9, 397–409 (2023)."). Indeed, genes with identifiable paralogs are less likely to be essential compared with genes with no identifiable paralog[14](https://www.nature.com/articles/s41588-025-02222-1#ref-CR14 "Blomen, V. A. et al. Gene essentiality and synthetic lethality in haploid human cells. Science 350, 1092–1096 (2015)."),[15](https://www.nature.com/articles/s41588-025-02222-1#ref-CR15 "Dandage, R. & Landry, C. R. Paralog dependency indirectly affects the robustness of human cells. Mol. Syst. Biol. 15, e8871 (2019)."),[16](https://www.nature.com/articles/s41588-025-02222-1#ref-CR16 "De Kegel, B. & Ryan, C. J. Paralog buffering contributes to the variable essentiality of genes in cancer cell lines. PLoS Genet. 15, e1008466 (2019)."),[17](https://www.nature.com/articles/s41588-025-02222-1#ref-CR17 "Wang, T. et al. Identification and characterization of essential genes in the human genome. Science 350, 1096–1101 (2015)."),[18](https://www.nature.com/articles/s41588-025-02222-1#ref-CR18 "Gu, Z. et al. Role of duplicate genes in genetic robustness against null mutations. Nature 421, 63–66 (2003)."). Moreover, in situations where both genes are expressed, buffering between paralogs may prevent genes from being identified as essential in single-gene knockout studies[19](https://www.nature.com/articles/s41588-025-02222-1#ref-CR19 "Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. Genome Biol. 21, 262 (2020)."). Combinatorial CRISPR–Cas9 approaches have been developed to disrupt two genes simultaneously, and screens employing this technology have facilitated the identification of new synthetic lethal interactions between paralog pairs[19](https://www.nature.com/articles/s41588-025-02222-1#ref-CR19 "Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. Genome Biol. 21, 262 (2020)."),[20](https://www.nature.com/articles/s41588-025-02222-1#ref-CR20 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."),[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."),[22](https://www.nature.com/articles/s41588-025-02222-1#ref-CR22 "Gonatopoulos-Pournatzis, T. et al. Genetic interaction mapping and exon-resolution functional genomics with a hybrid Cas9-Cas12a platform. Nat. Biotechnol. 38, 638–648 (2020)."),[23](https://www.nature.com/articles/s41588-025-02222-1#ref-CR23 "Ito, T. et al. Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers. Nat. Genet. 53, 1664–1672 (2021)."), highlighting them as potential therapeutic targets.

In this study, we perform a comprehensive CRISPR–Cas9 knockout screen in uveal melanoma, analyzing ten cell line models that we also subject to extensive profiling, including whole-genome and transcriptome sequencing and proteomic profiling. Using combinatorial paired guide RNA (pgRNA) and genome-wide sgRNA libraries, we map gene-pair and single-gene dependencies, respectively. Our analysis reveals a previously uncharacterized interaction between the paralogous CDP-diacylglycerol synthase 1 and 2 (_CDS1_ and _CDS2_) genes, alongside previously described pairs. We show that low _CDS1_ expression identifies a subset of tumors that are selectively sensitive to _CDS2_ loss, both in uveal melanoma and other cancer types. We also show mechanistically that disruption of the CDS1/CDS2 axis causes precursor phosphatidic acid (PA) accumulation, lipid droplet accumulation, and decreased levels of the phosphoinositides phosphatidylinositol (PI) and phosphatidylinositol monophosphate (PIP), resulting from disruption of phosphoinositide synthesis. Thus, these findings reveal a potential therapeutic target for uveal melanoma and other malignancies.

Results
-------

### Genomic and proteomic analysis of uveal melanoma models

Few uveal melanoma models exist and none have been comprehensively characterized. We therefore collected and validated a panel of ten human uveal melanoma cell lines, which were subjected to whole-genome and deep-transcriptome sequencing. These models were also proteome profiled, with these data collectively generating rich catalogs of single-nucleotide variants (SNVs) and copy number/structural variant calls alongside RNA/protein expression profiles (Supplementary Figs. [1](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)–[3](https://www.nature.com/articles/s41588-025-02222-1#MOESM1) and Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)–[3](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Notably, our collection of models includes established driver events, and using both CELLector[24](https://www.nature.com/articles/s41588-025-02222-1#ref-CR24 "Najgebauer, H. et al. CELLector: genomics-guided selection of cancer in vitro models. Cell Syst. 10, 424–432.e6 (2020).") and Celligner[25](https://www.nature.com/articles/s41588-025-02222-1#ref-CR25 "Warren, A. et al. Global computational alignment of tumor and cell line transcriptional profiles. Nat. Commun. 12, 22 (2021).") and with the genome and transcriptome data of each cell line, respectively, we show these models align tightly with uveal melanoma data in The Cancer Genome Atlas (TCGA[26](https://www.nature.com/articles/s41588-025-02222-1#ref-CR26 "Liu, J. et al. An integrated TCGA pan-cencer clinical data resource to drive high-quality survival outcome analytics. Cell 173, 400–416 (2018)."); Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). Of note, in keeping with the melanocytic origin of the lines, reverse-phase protein array (RPPA) profiling revealed robust expression of the melanocyte marker gp-100 (Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)) and other key uveal melanoma factors including quinone oxidoreductase-1 (NQO1)[27](https://www.nature.com/articles/s41588-025-02222-1#ref-CR27 "Shen, L. et al. Pan-cancer and single-cell analysis reveal the prognostic value and immune response of NQO1. Front. Cell Dev. Biol. 11, 1174535 (2023).") and DUSP4 (refs. [23](https://www.nature.com/articles/s41588-025-02222-1#ref-CR23 "Ito, T. et al. Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers. Nat. Genet. 53, 1664–1672 (2021)."),[28](https://www.nature.com/articles/s41588-025-02222-1#ref-CR28 "Gutierrez-Prat, N. et al. DUSP4 protects BRAF- and NRAS-mutant melanoma from oncogene overdose through modulation of MITF. Life Sci. Alliance 5, e202101235 (2022)."); Supplementary Table [2](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Several of the lines showed characteristic features of uveal melanoma, including monosomy 3, and either full or partial amplification of chromosome 8, a late-occurring event associated with metastasis[29](https://www.nature.com/articles/s41588-025-02222-1#ref-CR29 "Shain, A. H. et al. The genetic evolution of metastatic uveal melanoma. Nat. Genet. 51, 1123–1130 (2019)."). All cell lines showed substantial copy number alterations (Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)), which in each line involved almost all chromosomes. These data provide a platform against which to interpret the analyses outlined below.

### Combinatorial CRISPR library design and screening

Our combinatorial CRISPR library comprised 25,499 constructs containing pgRNAs targeting 514 gene pairs (Fig. [1a](https://www.nature.com/articles/s41588-025-02222-1#Fig1) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). This library consisted primarily of paralogous gene pairs (_n_ = 262) and putative synthetic lethal gene pair targets (_n_ = 210). Paralog pairs were selected such that they had a minimum protein sequence identity of 45% (Ensembl v92) and a single essential ortholog in _Drosophila melanogaster_ or _Caenorhabditis elegans_, where disruption of this ortholog conferred a lethal phenotype in these organisms (FlyMine, FB2015\_15; WormBase, WS251; [Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12)). We reasoned that selecting paralogs in this way, as we have described previously[30](https://www.nature.com/articles/s41588-025-02222-1#ref-CR30 "De Kegel, B., Quinn, N., Thompson, N. A., Adams, D. J. & Ryan, C. J. Comprehensive prediction of robust synthetic lethality between paralog pairs in cancer cell lines. Cell Syst. 12, 1144–1159.e6 (2021)."), would enrich for lethal paralog pair interactions in human cells. Additional paralogous/nonparalogous putative synthetic lethal gene pairs were identified by linear modeling and data integration with MASH-up[31](https://www.nature.com/articles/s41588-025-02222-1#ref-CR31 "Cho, H., Berger, B. & Peng, J. Compact integration of multi-network topology for functional analysis of genes. Cell Syst. 3, 540–548 e5 (2016)."), using pan-cancer TCGA expression[32](https://www.nature.com/articles/s41588-025-02222-1#ref-CR32 "Cancer Genome Atlas Research Network et al. The Cancer Genome Atlas Pan-Cancer analysis project. Nat. Genet. 45, 1113–1120 (2013).") and copy number data, together with data from loss-of-function RNAi-based screens and copy number profiles from the Cancer Cell Line Encyclopedia cancer cell line dataset[33](https://www.nature.com/articles/s41588-025-02222-1#ref-CR33 "Barretina, J. et al. The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity. Nature 483, 603–607 (2012)."). We also used RNA-seq and CRISPR–Cas9 data from Project Score[34](https://www.nature.com/articles/s41588-025-02222-1#ref-CR34 "Dwane, L. et al. Project score database: a resource for investigating cancer cell dependencies and prioritizing therapeutic targets. Nucleic Acids Res. 49, D1365–D1372 (2021)."),[35](https://www.nature.com/articles/s41588-025-02222-1#ref-CR35 "Pacini, C. et al. Integrated cross-study datasets of genetic dependencies in cancer. Nat. Commun. 12, 1661 (2021).") to systematically identify associations between gene expression levels and essentiality, thereby prioritizing candidate pairs where low expression of one gene predicted essentiality for another (Methods). These two approaches identified 115 and 95 gene pairs of the abovementioned 210 putative synthetic lethal gene pairs, respectively. In addition to these collections, we also included a subset of gene pairs relevant to uveal melanoma (_n_ = 42), with one member of the pair known to be downregulated within a gene expression profile predictive of uveal melanoma outcome[36](https://www.nature.com/articles/s41588-025-02222-1#ref-CR36 "Onken, M. D., Worley, L. A., Ehlers, J. P. & Harbour, J. W. Gene expression profiling in uveal melanoma reveals two molecular classes and predicts metastatic death. Cancer Res. 64, 7205–7209 (2004)."),[37](https://www.nature.com/articles/s41588-025-02222-1#ref-CR37 "Onken, M. D., Worley, L. A., Tuscan, M. D. & Harbour, J. W. An accurate, clinically feasible multi-gene expression assay for predicting metastasis in uveal melanoma. J. Mol. Diagn. 12, 461–468 (2010)."), while the other gene in the pair was either a paralogous gene or a candidate synthetic lethal gene based upon the database of synthetic lethal interactions (SynLethDB)[38](https://www.nature.com/articles/s41588-025-02222-1#ref-CR38 "Guo, J., Liu, H. & Zheng, J. SynLethDB: synthetic lethality database toward discovery of selective and sensitive anticancer drug targets. Nucleic Acids Res. 44, D1011–D1017 (2016)."). Additional candidates came from mutual exclusivity analysis of TCGA uveal melanoma genome data[39](https://www.nature.com/articles/s41588-025-02222-1#ref-CR39 "Robertson, A. G. et al. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. Cancer Cell 32, 204–220 e15 (2017).") (Methods). Of note, this group of genes included drivers for uveal melanoma, such as _BAP1_ paired with other genes. Ten safe-targeting guides (STGs), which had previously been designed against regions of the genome with no annotated function[40](https://www.nature.com/articles/s41588-025-02222-1#ref-CR40 "Morgens, D. W. et al. Genome-scale measurement of off-target activity using Cas9 toxicity in high-throughput screens. Nat. Commun. 8, 15178 (2017)."), were also used in our library (Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). These STGs were paired with sgRNAs targeting control essential (_n_ = 53) and nonessential (_n_ = 112) single genes (Methods) to allow technical assessment of screen performance and with each sgRNA targeting a library gene (that is, each gene within the abovementioned 514 gene pairs) to allow the single-gene knockout effect to be computed in comparison to codisruption of both members of a gene pair.

**Fig. 1: Development of a combinatorial CRISPR library to screen for digenic dependencies in uveal melanoma.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02222-1/figures/1)

**a**, Composition of the combinatorial CRISPR library. In total, the library targeted 514 gene pairs (blue), with each sgRNA targeting a gene in these pairs also found in the library with an STG, a design allowing us to compute the single versus double guide effect on cell fitness. The individual selection of sgRNAs in each category is described in the [Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12). In addition to the abovementioned 514 pairs, we also included a collection of single essential (green) and single nonessential (yellow) genes paired with STGs to facilitate the calibration of our downstream analysis. **b**, Paired sgRNA construct. The first sgRNA in position 1 was placed under an hU6 promoter, and the second sgRNA in position 2 was placed under an mU6 promoter. Nonidentical tracrRNAs were used to minimize recombination (Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). To assess single-guide activity, genes were paired with STGs placed in either position. **c**, Combinatorial sgRNA pairing strategy. For gene pairs A–B, 48 pgRNA combinations were used in the library where possible. In position 1, 12 sgRNAs were selected: 4 each against gene A (A1–A4), gene B (B1–B4) and safe-targeting regions of the genome (S1–S4). In position 2, another 12 unique sgRNAs were selected—4 each against gene A (A5–A8), gene B (B5–B8) and safe-targeting regions of the genome (S5–S8). This balanced design allowed us to account for any differences in sgRNA efficiency at either position. **d**, Screen quality was computed by calculation of NNMD values[35](https://www.nature.com/articles/s41588-025-02222-1#ref-CR35 "Pacini, C. et al. Integrated cross-study datasets of genetic dependencies in cancer. Nat. Commun. 12, 1661 (2021).") for essential/nonessential genes in the library revealing a screen performance as good as DepMap/Project Score (further QC metrics are available in Supplementary Figs. [6](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)–[9](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). **e**, Boxplot of guide-level log2(FCs) for each category of pgRNA across all ten cell lines. The colors correspond to those in **a**. The box shows the IQR; the line marks the median; whiskers extend to data within 1.5× IQR from Q1 and Q3; and points beyond are outliers. _P_ values were computed using a two-tailed Wilcoxon rank-sum test (no correction was applied). IQR, interquartile range.

The pgRNA construct was designed with each sgRNA placed under an independent promoter (human U6 or mouse U6) and with different tracrRNA scaffolds to reduce recombination (Fig. [1b](https://www.nature.com/articles/s41588-025-02222-1#Fig1) and Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). To compensate for any possible promoter bias, we positioned an equal number of guides for each gene behind each promoter, selecting eight individual sgRNAs per gene. This resulted in 32 unique sgRNA pairings for every gene pair (Fig. [1c](https://www.nature.com/articles/s41588-025-02222-1#Fig1) and Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). Genes with fewer than six available unique sgRNAs were excluded from the library.

Each of the ten human uveal melanoma cell lines was engineered to stably express Cas9 and validated for high activity (>85%; [Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12); Supplementary Fig. [6](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). Screens were performed at 1,000-fold library representation in triplicate for up to 28 days post-transduction with null-normalized mean difference (NNMD), strictly standardized mean difference values and Pearson correlations between screen replicates (Fig. [1d](https://www.nature.com/articles/s41588-025-02222-1#Fig1) and Supplementary Figs. [7](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)–[9](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)) suggesting quality control (QC) concordant with Project Score/DepMap screens. We next analyzed the log2(fold change (FC)) of all double knockout pgRNAs targeting gene pairs and compared these values with the log2(FC) of single-gene knockouts where vectors contained a sgRNA targeting a gene paired with an STG sgRNA. This analysis revealed significantly lower log2(FCs) within the double knockout gene pair group (_P_ < 2.2 × 10−16, Wilcoxon rank-sum test), which indicates the significantly more deleterious effect of disrupting two genes in these cells and the potential that there exist gene pairs where there is a synergistic (synthetic lethal) effect on cell fitness (Figs. [1d–e](https://www.nature.com/articles/s41588-025-02222-1#Fig1)).

### Landscape of digenic dependencies in uveal melanoma

We used the Bliss independence model to identify synergistic gene pairs[41](https://www.nature.com/articles/s41588-025-02222-1#ref-CR41 "Geary, N. Understanding synergy. Am. J. Physiol. Endocrinol. Metab. 304, E237–E253 (2013)."), predicting the dual knockout log2(FC) to equal the sum of the individual knockout log2(FCs). This allowed us to quantify the difference between the predicted and observed log2(FC) as the genetic interaction (GI) score (Fig. [2a](https://www.nature.com/articles/s41588-025-02222-1#Fig2)). For each cell line, we called a gene pair a ‘hit’ if the mean normalized GI score was less than −0.5 and the false discovery rate (FDR) below 0.01 across all guide pairs and replicates (Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM4) and [6](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). At this threshold, we identified 105 unique gene pairs (20.4% of the library gene pairs) that were significantly depleted in at least one of the ten screened cell lines (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Most of our hits were reproduced in at least two cell lines (>60%), indicating that there were relatively few hits that were cell line-specific in our screen. We next focused our analysis on significant digenic interactions that were shared among at least six of our ten screened cell lines, which allowed us to consider the top 21.9% (23/105) of the total gene pair hits across all cell lines (Fig. [2b](https://www.nature.com/articles/s41588-025-02222-1#Fig2)).

**Fig. 2: Top digenic and monogenic dependencies identified from combinatorial pgRNA CRISPR library analysis.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02222-1/figures/2)

**a**, Calculation of the GI score to quantify synthetic lethality. The sum of the observed log2(FC) for sgRNAs targeting gene A and gene B was calculated to determine the predicted log2(FC) of that gene pair. The difference between the predicted and experimentally observed log2(FC) was also calculated. **b**, Number of significantly depleted gene pairs across the cell lines screened. **c**, Dot plot depicting top synthetic lethal gene pairs from the combinatorial CRISPR screen that were common to at least 6 of the 10 screened cell lines, ranked (top to bottom) by descending mean GI score. Each dot represents the GI score of a given gene pair in a single cell line. All gene pairs had previously been screened across five independent CRISPR combinatorial screens and are colored by whether they were reported to be synthetic lethal in those studies[19](https://www.nature.com/articles/s41588-025-02222-1#ref-CR19 "Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. Genome Biol. 21, 262 (2020)."),[20](https://www.nature.com/articles/s41588-025-02222-1#ref-CR20 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."),[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."),[22](https://www.nature.com/articles/s41588-025-02222-1#ref-CR22 "Gonatopoulos-Pournatzis, T. et al. Genetic interaction mapping and exon-resolution functional genomics with a hybrid Cas9-Cas12a platform. Nat. Biotechnol. 38, 638–648 (2020)."),[23](https://www.nature.com/articles/s41588-025-02222-1#ref-CR23 "Ito, T. et al. Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers. Nat. Genet. 53, 1664–1672 (2021)."). Any pair where one of the genes was defined as lethal on its own was removed. **d**, Dot plot depicting significantly depleted single genes (genes with an STG) that were defined by Ensembl to have a paralog and common to at least six of the ten screened cell lines, ranked (top to bottom) by descending mean normalized log2(FC). log2(FCs) are normalized so that the median of the STGs was 0 and the median of the reference essential guides was −1. Genes are labeled with their paralog in parentheses. Each dot represents a cell line. Color indicates the synthetic lethality across five previous combinatorial CRISPR screens[19](https://www.nature.com/articles/s41588-025-02222-1#ref-CR19 "Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. Genome Biol. 21, 262 (2020)."),[20](https://www.nature.com/articles/s41588-025-02222-1#ref-CR20 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."),[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."),[22](https://www.nature.com/articles/s41588-025-02222-1#ref-CR22 "Gonatopoulos-Pournatzis, T. et al. Genetic interaction mapping and exon-resolution functional genomics with a hybrid Cas9-Cas12a platform. Nat. Biotechnol. 38, 638–648 (2020)."),[23](https://www.nature.com/articles/s41588-025-02222-1#ref-CR23 "Ito, T. et al. Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers. Nat. Genet. 53, 1664–1672 (2021)."). A nonhit (gray) denotes a gene not previously reported to have a synthetic lethal interaction with its partner. Significance was defined using an FDR < 0.05 ([Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12)). **e**, Correlation between _INTS6L_ expression and _INTS6_ gene essentiality in 316 Project Score[47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019).") cell lines, and the same for _CDS1_/_CDS2_. Low expressers (log2(TPM + 1) < 1) versus high expressers (log2(TPM + 1) ≥ 1) were defined using expression profiles from Cell Model Passports. Significance was determined by a _t_ test (two-sided, equal variance). Box and whisker plots indicate median and 5th to 95th percentiles; points are outliers.

Of our 23 gene pair hits that were common to at least six of our screened cell lines, all had previously been screened in at least one independent combinatorial CRISPR study and all 23 gene pairs were paralog pairs, most coming from the approach of identifying conserved sequence-similar paralogs with essential orthologs in worms/flies, with some coming from the abovementioned MASH-up analysis approach. Notably, 22 of 23 pairs were identified as hits in at least one of these previous studies, with _SMARCC1_/_SMARCC2_ as the only pair previously screened but not identified as a hit in any screen (Fig. [2c](https://www.nature.com/articles/s41588-025-02222-1#Fig2))[19](https://www.nature.com/articles/s41588-025-02222-1#ref-CR19 "Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. Genome Biol. 21, 262 (2020)."),[20](https://www.nature.com/articles/s41588-025-02222-1#ref-CR20 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."),[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."),[22](https://www.nature.com/articles/s41588-025-02222-1#ref-CR22 "Gonatopoulos-Pournatzis, T. et al. Genetic interaction mapping and exon-resolution functional genomics with a hybrid Cas9-Cas12a platform. Nat. Biotechnol. 38, 638–648 (2020)."),[23](https://www.nature.com/articles/s41588-025-02222-1#ref-CR23 "Ito, T. et al. Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers. Nat. Genet. 53, 1664–1672 (2021)."). However, low-throughput experiments have previously validated this gene pair as a synthetic lethal dependency[42](https://www.nature.com/articles/s41588-025-02222-1#ref-CR42 "Schick, S. et al. Systematic characterization of BAF mutations provides insights into intracomplex synthetic lethalities in human cancers. Nat. Genet. 51, 1399–1410 (2019)."),[43](https://www.nature.com/articles/s41588-025-02222-1#ref-CR43 "Sasaki, M. & Ogiwara, H. Synthetic lethal therapy based on targeting the vulnerability of SWI/SNF chromatin remodeling complex-deficient cancers. Cancer Sci. 111, 774–782 (2020)."). This recovery of previously reported GIs suggests our screens are sensitive and robust.

While combinatorial screening can identify pairs of genes that represent synthetic lethal dependencies, we hypothesized that the complex genomic architecture of the uveal melanoma genome may obscure some synthetic lethal interactions. Specifically, loss-of-function mutations or transcriptional downregulation of one member of a target gene pair may hide/obscure an otherwise synthetic lethal interaction between genes, particularly if that genomic feature/expression profile is common to all/most of the uveal cell lines we screened. This phenomenon has been described previously for _VRK1_/_VRK2_, where _VRK1_ is a single-gene dependency in glioblastoma because _VRK2_ is frequently silenced[44](https://www.nature.com/articles/s41588-025-02222-1#ref-CR44 "Shields, J. A. et al. VRK1 is a synthetic-lethal target in VRK2-deficient glioblastoma. Cancer Res. 82, 4044–4057 (2022)."). Thus, we mined our combinatorial CRISPR data to identify single-gene essentialities that may have gene partners that are downregulated in uveal melanoma. To do this, we focused on paralogs and used data derived from all 514 screened pairs, as some paralog pairs were included in the library as part of the putative and uveal-specific gene sets. After removing reference essential genes[45](https://www.nature.com/articles/s41588-025-02222-1#ref-CR45 "Hart, T. et al. Evaluation and design of genome-wide CRISPR/SpCas9 knockout screens. G3 (Bethesda) 7, 2719–2727 (2017)."), 19 such genes were identified to be commonly depleted across at least six of the ten cell lines. This included several known synthetic lethal gene pair members, such as _SMARCA4_ and _NXT1_, which are known to have epistatic interactions with _SMARCA2_ and _NXT2_, respectively (Fig. [2d](https://www.nature.com/articles/s41588-025-02222-1#Fig2))[43](https://www.nature.com/articles/s41588-025-02222-1#ref-CR43 "Sasaki, M. & Ogiwara, H. Synthetic lethal therapy based on targeting the vulnerability of SWI/SNF chromatin remodeling complex-deficient cancers. Cancer Sci. 111, 774–782 (2020)."),[46](https://www.nature.com/articles/s41588-025-02222-1#ref-CR46 "Malone, C. F. et al. Selective modulation of a pan-essential protein as a therapeutic strategy in cancer. Cancer Discov. 11, 2282–2299 (2021)."). Using data obtained from Project Score[47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019)."), we observed negative correlations between several of our single-gene hits and the gene expression of their paralog partner, and this was particularly striking for _CDS2_ and _INTS6_ (Fig. [2e](https://www.nature.com/articles/s41588-025-02222-1#Fig2)). Interestingly, _INTS6_/_INTS6L_ had been screened in five cell lines across two independent published screens and was not reported to be synthetic lethal in either study[20](https://www.nature.com/articles/s41588-025-02222-1#ref-CR20 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."),[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."), potentially suggesting the synthetic lethal interaction between this gene pair was masked by low _INTS6L_ expression levels (Fig. [2e](https://www.nature.com/articles/s41588-025-02222-1#Fig2)).

### _CDS2_ is a dependency in uveal melanoma

We next conducted genome-wide sgRNA CRISPR knockout screens in our panel of uveal melanoma cell lines with three primary objectives as follows: (1) to identify additional uveal melanoma-specific dependencies of single genes that were not assessed in the combinatorial CRISPR library; (2) to further examine digenic paralog CRISPR dependencies by perturbing each member within a paralog family and integrating these data and (3) to provide orthogonal and complementary validation of the combinatorial CRISPR screen findings. For single gRNA library screening, each cell line was screened in technical triplicate after transduction with the Human Improved Genome-wide Knockout CRISPR Library v1.1 (ref. [47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019).")), cultured for 14 days and collected for sgRNA sequencing and quantification. A 14-day screen was performed so that the data we generated would align with data from Sanger’s Project Score[34](https://www.nature.com/articles/s41588-025-02222-1#ref-CR34 "Dwane, L. et al. Project score database: a resource for investigating cancer cell dependencies and prioritizing therapeutic targets. Nucleic Acids Res. 49, D1365–D1372 (2021)."),[35](https://www.nature.com/articles/s41588-025-02222-1#ref-CR35 "Pacini, C. et al. Integrated cross-study datasets of genetic dependencies in cancer. Nat. Commun. 12, 1661 (2021)."). We preprocessed the screening data and corrected possible gene-independent responses to CRISPR–Cas9 targeting[48](https://www.nature.com/articles/s41588-025-02222-1#ref-CR48 "Aguirre, A. J. et al. Genomic copy number dictates a gene-independent cell response to CRISPR/Cas9 targeting. Cancer Discov. 6, 914–929 (2016).") with CRISPRcleanR[49](https://www.nature.com/articles/s41588-025-02222-1#ref-CR49 "Iorio, F. et al. Unsupervised correction of gene-independent cell responses to CRISPR–Cas9 targeting. BMC Genomics 19, 604 (2018)."),[50](https://www.nature.com/articles/s41588-025-02222-1#ref-CR50 "Vinceti, A. et al. An interactive web application for processing, correcting, and visualizing genome-wide pooled CRISPR–Cas9 screens. Cell Rep. Methods 3, 100373 (2023).") and validated screen performance by calculating the abovementioned NNMD and standardized mean difference values (Fig. [3](https://www.nature.com/articles/s41588-025-02222-1#Fig3) and Supplementary Fig. [10](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). This revealed high screen quality with scores like those of Project Score/DepMap. After excluding BAGEL reference essential genes (CEGv2; [Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12)) and filtering for genes that had a MAGeCK MLE beta score of less than −0.5 and an FDR below 0.05, a total of 861 genes were identified to be depleted/essential in at least one of the ten screened cell lines ([Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12); Fig. [3a](https://www.nature.com/articles/s41588-025-02222-1#Fig3) and Supplementary Tables [8](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)–[10](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Notably, this gene list includes many essential genes, and although these genes could be therapeutically targeted[51](https://www.nature.com/articles/s41588-025-02222-1#ref-CR51 "Chang, L., Ruiz, P., Ito, T. & Sellers, W. R. Targeting pan-essential genes in cancer: challenges and opportunities. Cancer Cell 39, 466–479 (2021)."), we reasoned that genes with a widespread effect on cell fitness across multiple cell and tissue types would be more likely to have effects/toxicity outside the target cancer cells. We therefore excluded all pan-cancer CRISPR-inferred common essential genes (DepMap 24Q2). From this analysis, we identified 8 genes from two-member protein-coding paralog families (Fig. [3b](https://www.nature.com/articles/s41588-025-02222-1#Fig3)) and dependencies in two or more lines. The genes _CDS2(CDS1)_, _RIC8A(RIC8B)_ and _SPTSSA(SPTSSB)_ were a hit in ten, five and four cell lines, respectively. We next examined these pairs using TCGA uveal melanoma data[39](https://www.nature.com/articles/s41588-025-02222-1#ref-CR39 "Robertson, A. G. et al. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. Cancer Cell 32, 204–220 e15 (2017).") and the transcriptome analysis of our cell lines, observing expression profiles that align with our CRISPR results (Fig. [3c](https://www.nature.com/articles/s41588-025-02222-1#Fig3)), thus revealing possible context-dependent and potentially targetable digenic vulnerabilities.

**Fig. 3: Genome-wide single gRNA CRISPR screening identifies _CDS2_ as an essential gene in uveal melanoma.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02222-1/figures/3)

**a**, Plot showing the distribution of genes in each cell line following genome-wide CRISPR screening. Dotted line denotes a MAGeCK MLE beta of +0.5 or −0.5. For each cell line, ‘\_NE’ indicates scores filtered for common essential genes (DepMap 24Q2) and core essential genes (CEGv2), with genes that are significant in ≥6 lines shown as triangles (MLE beta +0.5 or −0.5 and an FDR < 0.01). The genes significant in most lines are labeled even if they are not significant in a specific line. **b**, Genes from two-gene paralog families that were a hit displayed with their beta values. The paralogous genes for these screen hits are shown in parentheses. Each dot represents a cell line. **c**, Top paralog gene pairs identified from single gRNA CRISPR data. The data shown is the relative gene expression levels of these pairs derived from our ten screened uveal cell lines (top) and the 80 tumors in the TCGA uveal melanoma dataset[38](https://www.nature.com/articles/s41588-025-02222-1#ref-CR38 "Guo, J., Liu, H. & Zheng, J. SynLethDB: synthetic lethality database toward discovery of selective and sensitive anticancer drug targets. Nucleic Acids Res. 44, D1011–D1017 (2016).") (bottom). These data illustrate that in patient samples, the genes _CDS2_, _RIC8A_ and _SPTSSA_ are robustly expressed at levels much higher than their paralog gene partner. Also shown is an analogous expression pattern in our uveal melanoma models (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). The box shows the IQR; the line marks the median; whiskers extend to data within 1.5× IQR from Q1 and Q3. **d**, NNMD values for the whole-genome screens for each of the ten cell lines analyzed in comparison to Project Score screens[47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019).").

To extend this analysis further and to orthogonally computationally validate _CDS2, RIC8A_ and _SPTSSA_, and to identify further candidates, including nonparalog genes, we adopted an additional approach to identify gene vulnerabilities that were specific to uveal melanoma. We again leveraged pan-cancer essentiality data derived from DepMap[52](https://www.nature.com/articles/s41588-025-02222-1#ref-CR52 "Tsherniak, A. et al. Defining a cancer dependency map. Cell 170, 564–576 e16 (2017).") (DepMap 22Q2) and the Mann–Whitney _U_ test to calculate a _P_ value using rank-normalized essentiality scores as input. Rank normalization allowed us to compare our screen results with those of DepMap and to define statistically significant uveal melanoma essential genes (Methods). Notably, before this analysis, we first removed any uveal or cutaneous melanoma lines screened as part of the DepMap resource. Following these steps, our analysis identified 76 genes as statistically significant (log2(FC) > 1.80 and _P_adj < 0.01; Fig. [4a](https://www.nature.com/articles/s41588-025-02222-1#Fig4) and Supplementary Table [11](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Gene set enrichment analysis using gene ontology (GO) pathways was performed and showed enrichment pertaining to diacylglycerol binding and CDP-alcohol phosphatidyltransferase activity (Fig. [4b](https://www.nature.com/articles/s41588-025-02222-1#Fig4)), in keeping with chronic activation of the phospholipase C (PLC) signal transduction pathway in uveal melanoma[53](https://www.nature.com/articles/s41588-025-02222-1#ref-CR53 "Gallegos, L. L. & Newton, A. C. Spatiotemporal dynamics of lipid signaling: protein kinase C as a paradigm. IUBMB Life 60, 782–789 (2008)."),[54](https://www.nature.com/articles/s41588-025-02222-1#ref-CR54 "Thillaiappan, N. B., Chakraborty, P., Hasan, G. & Taylor, C. W. IP(3) receptors and Ca2+ entry. Biochim. Biophys. Acta Mol. Cell. Res. 1866, 1092–1100 (2019)."),[55](https://www.nature.com/articles/s41588-025-02222-1#ref-CR55 "Shoushtari, A. N. & Carvajal, R. D. GNAQ and GNA11 mutations in uveal melanoma. Melanoma Res. 24, 525–534 (2014)."). Our top ten uveal-specific genes identified from this analysis (Fig. [4c](https://www.nature.com/articles/s41588-025-02222-1#Fig4)) had clear biological roles related to melanocyte differentiation and survival (_SOX10_, _MITF_), G-protein coupled receptor (GPCR) signaling (_GNAQ_, _RIC8A_) and downstream phosphoinositide signaling (_CDS2_ and _CDIPT_), and by themselves represent possible therapeutic targets. Of note, our findings of _CDS2_ essentiality are consistent with our combinatorial pgRNA screen results, providing additional support for _CDS2_ as a possible target in uveal melanoma, a result in keeping with several observations in the literature in other tumor types[20](https://www.nature.com/articles/s41588-025-02222-1#ref-CR20 "Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. Cell Rep. 36, 109597 (2021)."),[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."),[56](https://www.nature.com/articles/s41588-025-02222-1#ref-CR56 "Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. Cell Rep. 39, 110636 (2022)."),[57](https://www.nature.com/articles/s41588-025-02222-1#ref-CR57 "Lin, H. K. et al. Functional buffering via cell-specific gene expression promotes tissue homeostasis and cancer robustness. Sci. Rep. 12, 2974 (2022)."). Intersection of our genomic data with our list of 76 uveal melanoma-specific genes revealed further driver gene candidates (Supplementary Fig. [11](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)).

**Fig. 4: Uveal melanoma-specific essential genes are involved in GPCR signaling and phosphoinositide signaling pathways.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02222-1/figures/4)

**a**, Uveal melanoma-specific gene hits (log2(FC) > 1.80 and _P_adj < 0.01; two-tailed Mann–Whitney _U_ test with Benjamini–Hochberg correction). Significantly differential essential genes for uveal melanoma cell lines were computed by comparing gene essentiality scores from our uveal melanoma screens to genome-wide sgRNA CRISPR screens from DepMap 22Q2 ([Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12); Supplementary Table [11](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Blue dots denote the top uveal-specific essential genes; green dots are genes less essential in uveal cell lines versus pan-cancer. **b**, Pathway enrichment analysis of the 76 uveal melanoma-specific genes using GO molecular function pathways. **c**, Dependency rank of top ten genes in uveal melanoma compared with pan-cancer cell lines. As above, pan-cancer data obtained from DepMap[52](https://www.nature.com/articles/s41588-025-02222-1#ref-CR52 "Tsherniak, A. et al. Defining a cancer dependency map. Cell 170, 564–576 e16 (2017).") 22Q2 release. The analysis compared the 10 uveal lines to 982 pan-cancer cell lines. The box shows the IQR, the line marks the median, whiskers extend to data within 1.5× from Q1 and Q3, and points beyond are outliers. **d**, Validation of genome-wide sgRNA CRISPR screen hits with a competitive coculture assay ([Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12)). Cells were transduced with a lentivirus expressing an sgRNA and a BFP marker. BFP expression was measured by flow cytometry at baseline and on days 14 and 28. The proportion of surviving sgRNA-transduced cells compared with nontransduced cells is normalized to day 4. Two sgRNAs were tested against each gene, and an STG sgRNA was used as a control. Data represent three independent experiments performed in triplicate, with the mean and s.d. shown. Significance was calculated using a two-way ANOVA with Tukey’s multiple test correction. \*\*\*\*_P_ < 0.0001, \*\*_P_ < 0.01, \*_P_ < 0.05 (exact _P_ values are provided in the source data).

[Source data](https://www.nature.com/articles/s41588-025-02222-1#MOESM5)

### Orthogonal validation of the _CDS1_/_CDS_ _2_ interaction

_CDS1_ and _CDS2_, located on chromosome 4 (84,583,127–84,651,334) and chromosome 20 (5,126,879–5,197,887), respectively, are highly sequence-similar paralogs that encode key enzymes in the synthesis of phosphoinositides, which have key regulatory roles in the MAPK, AKT and other pathways[58](https://www.nature.com/articles/s41588-025-02222-1#ref-CR58 "Blunsom, N. J. & Cockcroft, S. CDP-diacylglycerol synthases (CDS): gateway to phosphatidylinositol and cardiolipin synthesis. Front. Cell Dev. Biol. 8, 63 (2020)."),[59](https://www.nature.com/articles/s41588-025-02222-1#ref-CR59 "Balla, T. Phosphoinositides: tiny lipids with giant impact on cell regulation. Physiol. Rev. 93, 1019–1137 (2013)."). _CDS1_/_CDS__2_ are each other’s only identifiable paralog in the human genome and are widely conserved across species, with orthologs identifiable in both _D. melanogaster_ and _C. elegans_, and in budding and fission yeasts. Of note, CDP-diacylglycerol synthases regulate the growth of lipid droplets and adipocyte development[60](https://www.nature.com/articles/s41588-025-02222-1#ref-CR60 "Qi, Y. et al. CDP-diacylglycerol synthases regulate the growth of lipid droplets and adipocyte development. J. Lipid Res. 57, 767–780 (2016)."). Our primary objective was to further genetically validate the _CDS1_/_CDS2_ interaction, focusing on _CDS2_ as the target. _CDS2_ sgRNAs were selected and cloned into a fluorescent protein-tagged vector, allowing us to quantify the dynamics of transduced cell populations over time (Methods). The results showed that by 14 days, _CDS2_ was clearly required for uveal melanoma cell line fitness (Fig. [4d](https://www.nature.com/articles/s41588-025-02222-1#Fig4)). Although the abovementioned sgRNAs were selected to have no off-targets, one concern of targeting genes such as _CDS2_ is unwanted sgRNA-mediated _CDS1_ disruption. We therefore transfected SW837 cells, a colon cancer line expressing both _CDS1_/_CDS2_, with our highest activity _CDS2_ sgRNA (GAGTAAAGGAAATGAACCGG) and after selection, whole-exome and transcriptome sequencing were performed. In keeping with off-target predictions, we observed only on-targeting “multi-hit” indel formation in _CDS2_. This sgRNA was used for all downstream experiments (Supplementary Fig. [12](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)).

### _CDS1_/_CDS2_ synthetic lethality across multiple tumor types

In both our screened uveal melanoma cell lines (_n_ = 10) and the TCGA uveal melanoma cohort[39](https://www.nature.com/articles/s41588-025-02222-1#ref-CR39 "Robertson, A. G. et al. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. Cancer Cell 32, 204–220 e15 (2017).") (_n_ = 80), _CDS1_ gene expression was consistently lower than that of _CDS2_. In the cell lines, _CDS1_ had a median expression of 2.2 (range = 0.6–3.2; log2(TPM + 1)), compared with 5.6 (range = 4.3–6.2) for _CDS2_. Similarly, in the TCGA cohort, _CDS1_ exhibited a median expression of 6.1 (2.9–9.1; log₂(RSEM + 1)), compared with 10.8 (10.0–12.4) for _CDS2_ (Fig. [3c](https://www.nature.com/articles/s41588-025-02222-1#Fig3)). Analysis of single-cell sequencing data from 26 uveal melanomas revealed high tumor cell expression of _CDS2_ and low expression of _CDS1_, further highlighting a unique opportunity to selectively target the _CDS1_/_CDS2_ axis in tumors by disruption or inhibition of _CDS2_ (Fig. [5a](https://www.nature.com/articles/s41588-025-02222-1#Fig5)). To determine whether this expression pattern could be observed in other cancers, we analyzed TCGA pan-cancer gene expression obtained from the UCSC Xena platform[61](https://www.nature.com/articles/s41588-025-02222-1#ref-CR61 "Goldman, M. J. et al. Visualizing and interpreting cancer genomics data via the Xena platform. Nat. Biotechnol. 38, 675–678 (2020)."). While _CDS2_ was widely expressed among all cancer subtypes analyzed, we found _CDS1_ expression to be low or absent in cases of cutaneous melanoma, glioblastoma, hematological malignancies, hepatocellular carcinoma and sarcoma (Fig. [5b](https://www.nature.com/articles/s41588-025-02222-1#Fig5)), suggesting that targeting _CDS2_ in these cancers may be a viable therapeutic strategy.

**Fig. 5: Identification of _CDS1_/_CDS2_ synthetic lethality across multiple tumor types.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02222-1/figures/5)

**a**, Analysis of single-cell sequencing data from the uveal melanoma microenvironment reveals strong expression of _CDS2_ in malignant cells. **b**, _CDS1_ and _CDS2_ gene expression in cancers based upon data generated by the TCGA Research Network and obtained from the UCSC Xena platform[61](https://www.nature.com/articles/s41588-025-02222-1#ref-CR61 "Goldman, M. J. et al. Visualizing and interpreting cancer genomics data via the Xena platform. Nat. Biotechnol. 38, 675–678 (2020)."). Shown is the data range (mean − s.e. to the mean + s.e.) with points equaling the mean. **c**, Waterfall plot depicting _CDS2_ Chronos gene effect across 1013 pan-cancer cell lines from Project Achilles, DepMap 23Q4. Each column represents a cell line colored by _CDS1_ expression. **d**, Comparison of _CDS2_ essentiality in _CDS1_ low expressers (log2(TPM + 1) < 1) versus high expressers (log2(TPM + 1) ≥ 1) across 937 nonuveal melanoma cell lines analyzed in Broad DepMap (22Q2). A _t_ test (two-sided, equal variance) was performed to compute significance. Box and whisker plots indicate the median and the 5th to 95th percentiles. Outliers as dots. **e**, Correlation between _CDS1_ methylation and expression. Methylation fraction represents a weighted average of the methylation ratios of all CpG sites within 1,000 bp from a gene’s transcriptional start site. The methylation ratio of each CpG site was determined by the number of reads where that CpG was methylated over the total number of reads covering that CpG. The weights of each CpG are calculated by the total number of reads covering that CpG overall and the total number of reads covering any CpG within 1,000 bp of the gene’s transcriptional start site. Data obtained from DepMap 23Q2 (ref. [69](https://www.nature.com/articles/s41588-025-02222-1#ref-CR69 "Ghandi, M. et al. Next-generation characterization of the Cancer Cell Line Encyclopedia. Nature 569, 503–508 (2019).")). Each dot represents a cell line. **f**, Genetic rescue of _CDS2_ lethality with a _CDS1_ cDNA. OMM2.5 cells were transduced with either an sgRNA targeting _CDS2_ or a control sgRNA (STG). As indicated, cells were also transduced with the _CDS1_ cDNA and cultured for 14 days before staining with crystal violet. The data shown is representative of three biological replicates performed in triplicate. Each circle is a well of a six-well plate.

[Source data](https://www.nature.com/articles/s41588-025-02222-1#MOESM6)

To determine whether the _CDS1_/_CDS2_ synthetic lethal interaction was observed in other tumor types, we extended our analysis of _CDS2_ gene essentiality to include 937 cell lines from Project Achilles (DepMap 22Q2)[34](https://www.nature.com/articles/s41588-025-02222-1#ref-CR34 "Dwane, L. et al. Project score database: a resource for investigating cancer cell dependencies and prioritizing therapeutic targets. Nucleic Acids Res. 49, D1365–D1372 (2021)."). Any uveal melanoma cell lines present within the data sets were excluded to ensure that any observed effect was not driven by uveal melanoma alone. Notably, _CDS2_ was significantly more essential in _CDS1_ low expressers (log2(transcripts per million (TPM) + 1) < 1; _P_ = 8.59 × 10−9; _t_ test (two-sided, equal variance)), and consistent with a synthetic lethal relationship between _CDS1_ and _CDS2_ (Figs. [5c–d](https://www.nature.com/articles/s41588-025-02222-1#Fig5)). Analysis of methylation data from these cell lines (DepMap 23Q2) revealed promoter hypermethylation in cell lines that had low _CDS1_ expression (Fig. [5e](https://www.nature.com/articles/s41588-025-02222-1#Fig5)), suggesting a probable mechanism of gene silencing. Finally, given that these studies revealed epistasis between _CDS1_/_CDS2_, we asked if the re-expression of _CDS1_ could rescue the cell fitness defect associated with _CDS2_ loss. In this way, we demonstrated that lentiviral transduction of a _CDS1_ cDNA restored colony formation in vitro upon _CDS2_ knockout (Fig. [5f](https://www.nature.com/articles/s41588-025-02222-1#Fig5)), albeit associated with a decrease in colony formation compared to _CDS1_ overexpression alone.

### Mechanistic insights into the effects of _CDS2_ loss

The highly active _CDS2_ sgRNA (described above) was placed under the control of a doxycycline (Dox)-inducible promoter in cell lines that constitutively expressed Cas9. We first confirmed CDS2 protein depletion and the generation of frameshift alleles upon Dox treatment (Supplementary Fig. [13](https://www.nature.com/articles/s41588-025-02222-1#MOESM1) and Extended Data Figs. [1](https://www.nature.com/articles/s41588-025-02222-1#Fig7)–[4](https://www.nature.com/articles/s41588-025-02222-1#Fig10)). In colony-forming assays, Dox treatment of _CDS2_ sgRNA-inducible MP41 and OMM2.5 cells resulted in fewer and smaller colonies compared with dimethyl sulfoxide (DMSO; vehicle)-treated cells or isogenic inducible lines carrying an STG sgRNA (Fig. [6a](https://www.nature.com/articles/s41588-025-02222-1#Fig6); [Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12)). Of note, in apoptosis assays we observed a significant increase in apoptotic (Annexin V+/DAPI−) and dead cells (Annexin V+/DAPI+) following the induction of _CDS2_ sgRNA expression, suggesting that _CDS1_/_CDS2_ loss results in reduced cell fitness via apoptosis rather than cytostasis (Fig. [6b](https://www.nature.com/articles/s41588-025-02222-1#Fig6)). Using these Dox-inducible OMM2.5 and MP41 cell lines, we next performed mass-spectrometry revealing significant loss of CDS2 protein in both lines, a result confirmed by Western blotting. We also collected a shared compendium of 50 upregulated and 27 downregulated proteins, with pathway analysis revealing downregulation of genes involved in DNA replication and upregulation of genes involved in cholesterol synthesis (Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02222-1#Fig8); Supplementary Table [12](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). We extended these analyses further using liquid chromatography–mass spectrometry (LC–MS) to quantify phosphoinositides and their metabolic precursor PA over 7 days. We observed that following disruption of CDS2, there was a progressive reduction in the phosphoinositides PI and PIP and in several phosphatidylinositol biphosphate (PIP2) species (Fig. [6c](https://www.nature.com/articles/s41588-025-02222-1#Fig6) and Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02222-1#Fig9)). Strikingly, we observed supersized lipid droplets that were formed because of the deletion of _CDS2_ (Fig. [6d](https://www.nature.com/articles/s41588-025-02222-1#Fig6)), likely related to the corresponding increase in PA, the substrate for CDS enzymes. This effect has been previously reported[58](https://www.nature.com/articles/s41588-025-02222-1#ref-CR58 "Blunsom, N. J. & Cockcroft, S. CDP-diacylglycerol synthases (CDS): gateway to phosphatidylinositol and cardiolipin synthesis. Front. Cell Dev. Biol. 8, 63 (2020)."),[62](https://www.nature.com/articles/s41588-025-02222-1#ref-CR62 "Xu, Y. et al. CDP-DAG synthase 1 and 2 regulate lipid droplet growth through distinct mechanisms. J. Biol. Chem. 294, 16740–16755 (2019)."), but in uveal melanoma cells seems particularly impressive, indicating the accumulation of triacylglycerol due to the diversion of precursor molecules not used for PI synthesis. Because GPCR signaling is intrinsically linked to uveal melanoma development and phosphoinositide synthesis, we next asked whether there was an association between the expression of GPCR pathways involved in receptor-ligand or signal transduction and the Chronos score for _CDS2_ across DepMap. This pan-cancer and unbiased analysis revealed a Kolmogorov–Smirnov _P_ value of 2.2 × 10−16, where higher _CDS2_ essentiality was associated with higher GPCR signal transduction activity and specific pathway enrichments, including those involved in Gq and PLC signaling (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02222-1#Fig10)).

**Fig. 6: Mechanistic insights into the effects of _CDS2_ loss.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02222-1/figures/6)

**a**, Clonogenic assay showing the effects of _CDS2_ disruption at day 14 following the addition of Dox to Cas9-expressing cells that contain a Dox-inducible _CDS2_ or an STG control sgRNA (STG2). Data were collected from three independent biological replicates for each cell line (triangles, OMM2.5; circles, MP41). Representative plates are shown for one replicate of the OMM2.5 cell line. _P_ values were calculated with Welsh’s two-tailed _t_ test (without correction). Circles are a well of a six-well plate. **b**, Flow cytometry results from apoptosis assays at day 14 following _CDS2_ sgRNA induction with Dox, compared with induction of an STG sgRNA control. Data collected from three biological replicates. Data shown are mean and s.d. _P_ values were calculated using a two-tailed _t_ test. Wild-type cells are shown for comparison. **c**, Disruption of _CDS2_ following Dox treatment leads to a progressive reduction in the phosphoinositides PI and PIP and an increase in the precursor PA, compared with the DMSO control. Data are represented as mean ± s.e.m. from three independent MP41 clones and one OMM2.5 clone across three independent experiments. _P_ values were calculated using a two-way ANOVA with Sidak’s correction with significant values shown. **d**, 4,4-Difluoro-1,3,5,7,8-pentamethyl-4-bora-3a,4a-diaza-_s_\-indacene (BODIPY 493/503) staining (green) of lipid droplets in cells treated with Dox or DMSO for 14 days. DAPI-stained nuclei (blue). This experiment was performed twice independently. **e**, Schematic of the in vivo experiments performed in 28 female mice at 8 weeks of age (NOD-_Prkdc__scid__\-IL2rg__Tm1_/Rj background). After implantation of tumor cells, mice were blindly randomized into two groups ([Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12)) with the switch to a Dox-containing diet occurring on day 24 (arrow). Data shown are mean ± s.d. Significance was determined using a mixed-effects model with the Geisser–Greenhouse correction. **f**, Schematic of the role of CDS1/CDS2 in phosphoinositide de novo synthesis and recycling.

[Source data](https://www.nature.com/articles/s41588-025-02222-1#MOESM7)

Finally, to gain additional mechanistic insights, we next attempted to identify genes whose disruption could rescue the lethality observed following _CDS2_ loss, reasoning that any such genes would represent potential pathways to drug resistance in patients treated with an agent that inhibited or depleted CDS2. To do this, we conducted a genetic suppressor screen using the MinLibCas9 library in an MP41-derived clonal cell line carrying a Dox-inducible _CDS2_ sgRNA allele (above; [Methods](https://www.nature.com/articles/s41588-025-02222-1#Sec12); Supplementary Figs. [13](https://www.nature.com/articles/s41588-025-02222-1#MOESM1) and [14](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). Notably, we were unable to identify any genes under significant positive selection at an FDR below 0.05, suggesting limited escape to loss of _CDS2_ in uveal melanoma (Supplementary Tables [13](https://www.nature.com/articles/s41588-025-02222-1#MOESM4) and [14](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). This is notable as it suggests that an inhibitor of CDS2 may provide effective and sustained antitumor activity with minimal bypass/resistance mechanisms.

### Synthetic lethal interaction of _CDS1_/_CDS2_ in vivo

Finally, given that our experiments assessing whether the effect of _CDS2_ loss on cell fitness were performed in vitro, we next determined if we could extend our observations to the in vivo context. This is relevant because in vivo tumor cells need to grow in three dimensions and interact with host cells, such as stroma, which can attenuate phenotypes previously observed in vitro. To perform these experiments, we used OMM2.5 cells containing a highly active and specific Dox-inducible _CDS2_ sgRNA construct (described above), selecting OMM2.5 from available lines because these cells readily and reproducibly formed xenografts. Notably, _CDS2_ disruption in grafted cells resulted in significantly reduced uveal melanoma cell expansion, which prolonged the time to the tumor growth limit, when xenografted mice were fed a Dox-containing diet (625 mg kg−1; ENVIGO). Of note, compared with control mice fed a normal diet, we saw no evidence of resistance/rebound tumor growth even up to 48 days after the experiment was initiated (Fig. [6e,f](https://www.nature.com/articles/s41588-025-02222-1#Fig6)), suggesting that resistance was not acquired within this timeframe. To explore this question further and define possible mechanisms of resistance associated with _CDS2_ disruption, we first performed PCR to detect whether residual cancer cells were still present at the graft site following prolonged Dox feeding. Subsequently, we sequenced RNA from these tissues using hybrid capture to enrich for human/cancer cell cDNA. It appeared that the survival of residual uveal melanoma cells was, in part, mediated by nondisruptive/in-frame events at the _CDS2_ locus in our model (Supplementary Fig. [14](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)).

Discussion
----------

Despite significant advances in our understanding of uveal melanoma biology and genetics, developing intuitive therapeutic strategies has been clinically challenging. In this study, we use an agnostic approach of both genome-wide sgRNA CRISPR screening and combinatorial pgRNA CRISPR screening to catalog monogenic and digenic dependencies. Combined with molecular profiling of all available uveal lines, we provide an invaluable resource for therapeutic target exploration in this disease. We observed a robust _CDS2_ dependency in all ten uveal melanoma cell lines screened and, notably, demonstrated a negative impact of _CDS2_ loss on phosphoinositide availability with a profound effect on cancer cell fitness, both in vitro and in vivo. Our analysis aligns with ref. [63](https://www.nature.com/articles/s41588-025-02222-1#ref-CR63 "Arnoldus, T. et al. Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers. Nat. Genet. 
https://doi.org/10.1038/s41588-025-02221-2
(2025)."), which independently identified the synthetic lethal interaction between _CDS1_ and _CDS2_ across a range of mesenchymal cancers. We extended our analysis of _CDS2_ essentiality across multiple tumor types and, using available CRISPR–Cas9 screen resources, show that low _CDS1_ expression predicts a cancer line’s sensitivity to _CDS2_ disruption. Thus, tumor _CDS1_ expression may be a predictive biomarker for therapeutic CDS2 inhibition. Analysis of normal tissues from the GTEx resource suggests that _CDS2_ is ubiquitously and strongly expressed. An important hurdle, however, will be to evaluate the effect of _CDS2_ loss/depletion in normal cells where _CDS1_ expression is low, including hepatic and heart tissues[64](https://www.nature.com/articles/s41588-025-02222-1#ref-CR64 "Uhlen, M. et al. Proteomics. Tissue-based map of the human proteome. Science 347, 1260419 (2015).") (Supplementary Fig. [15](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). In this regard, it is important to note that liver-specific knockout of _Cds2_ from early stages of development resulted in reversible liver steatosis with PA, PI and phosphatidylglycerol levels not significantly affected in primary hepatocytes[65](https://www.nature.com/articles/s41588-025-02222-1#ref-CR65 "Xu, J. et al. Hepatic CDP-diacylglycerol synthase 2 deficiency causes mitochondrial dysfunction and promotes rapid progression of NASH and fibrosis. Sci. Bull. (Beijing) 67, 299–314 (2022)."). Similarly, deletion of _Cds2_ in the vasculature results in reduced growth of grafted tumors with no adverse phenotypes[66](https://www.nature.com/articles/s41588-025-02222-1#ref-CR66 "Stratman, A. N. et al. Anti-angiogenic effects of VEGF stimulation on endothelium deficient in phosphoinositide recycling. Nat. Commun. 11, 1204 (2020)."). A therapeutic window may also exist in uveal melanoma because previous observations have shown loss of _CDS2_ results in a significant reduction of GPCR-stimulated resynthesis of PI[67](https://www.nature.com/articles/s41588-025-02222-1#ref-CR67 "Collins, D. M. et al. CDS2 expression regulates de novo phosphatidic acid synthesis. Biochem. J. 481, 1449–1473 (2024)."), with similar results observed here. Interestingly, almost all uveal melanomas harbor mutations that cause constitutive activation of GPCR signaling[39](https://www.nature.com/articles/s41588-025-02222-1#ref-CR39 "Robertson, A. G. et al. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. Cancer Cell 32, 204–220 e15 (2017)."), and we provide evidence to suggest this may result in sensitivity to _CDS2_ loss compared with other tissues with low _CDS1_ expression, including normal tissues. Our proposition is that while low _CDS1_ expression increases susceptibility to loss of cellular fitness from _CDS2_ disruption, the true extent of _CDS2_ dependency may be influenced by cell type and tumor-specific biological factors such as degree of chronic PLC activity, resulting in a viable therapeutic avenue for further investigation in cancers. We are intrigued by the observation that low _CDS1_ expression is found in tumors, such as uveal melanoma and mesenchymal cancers. Our findings suggest that overexpression of _CDS1_ impairs tumor cell fitness, suggesting that its expression is under tight physiological control. Finally, analysis of AlphaFold and AlphaFill data[68](https://www.nature.com/articles/s41588-025-02222-1#ref-CR68 "Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Nature 596, 583–589 (2021).") for CDS2 suggests multiple druggable cavities, including the region containing the CDS enzyme domain, suggesting this protein is a tractable therapeutic target.

Methods
-------

### Informed consent and ethics

All human datasets used in this paper were collected with the written informed consent of all patients and/or donors. The care and use of all mice in this study were in accordance with the UK Animals in Science Regulation Unit’s Code of Practice for the Housing and Care of Animals Bred, Supplied or Used for Scientific Purposes, the Animals (Scientific Procedures) Act 1986, and all procedures were performed under a UK Home Office Project license (PP8090463), which was reviewed and approved by the University of Cambridge Animal Welfare and Ethical Review Body. We operate an open and inclusive research environment that is welcoming to all regardless of their background, beliefs or identity.

### Cell line whole-genome sequencing and analysis

Genomic DNA was extracted from cell lines using the Gentra Puregene Core Kit A (Qiagen), and library construction was performed using the NEBNext NEB Ultra II custom kit (New England Biolabs) according to the manufacturer’s instructions. Genomes were sequenced such that, on average, 79% of the genome of each cell line was covered by >30× coverage, with the remaining 20% covered by more than 10× sequence depth. Sequencing was performed by Illumina HiSeq X Paired End Sequencing. SAMtools v1.10 was used to convert BAM files to FASTQ files. QC was performed using FastQC v0.11.8. Reads were aligned to GRCh38 using BWA mem v0.7.17. GATK v4.4.0.0 was used for germline short variant calling (SNVs, indels) with the HaplotypeCaller, as these cell lines are clonal, not tumors. Germline variants were removed with bcftools using gnomAD data, excluding all variants with an allele frequency >0.01. Identification of structural variants was performed using GRIDSS 2.0. Circos plots were generated in R v4.1.1 with StructuralVariantAnnotation v1.10.1, VariantAnnotation v1.40.0 and circlize v0.4.15 packages. Copy number calling was performed using CNVKit[70](https://www.nature.com/articles/s41588-025-02222-1#ref-CR70 "Talevich, E., Shain, A. H., Botton, T. & Bastian, B. C. CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing. PLoS Comput. Biol. 12, e1004873 (2016).") version 0.9.8. More details are available on GitHub/Figshare along with a complete multi-QC report.

### Transcriptome sequencing

For each of the 10 uveal melanoma cell lines in this study, RNA was extracted from five technical replicate cultures using Qiagen kits (50 samples total). Illumina transcriptome libraries were prepared and sequenced as a pool on the Illumina 6000 platform to reduce possible batch effects. Reads were mapped using STAR v2.5.0c against the GRCh38 human reference genome with the ERCC spike-in sequences and ENSEMBL v103 gene annotation. To assess gene expression, reads were counted using HTSeq-count v0.7.2 with the “union” counting mode and appropriate stranded parameters. Finally, read counts were transformed to TPM. On average, each replicate was sequenced to generate 280 million reads, and on average, 20,434 genes were detected after mapping. Samples clustered tightly with high Pearson correlation coefficients. A full QC report is available via GitHub/Figshare. Transcriptome sequencing of SW837 cells, as part of _CDS2_ sgRNA specificity assessment, was performed, generating at least 50 million reads per sample on the Illumina NovaSeq 6000 platform, with analysis essentially described as above.

### RPPA

RPPA analyses were performed by the RPPA core facility and MD Anderson Cancer Center[71](https://www.nature.com/articles/s41588-025-02222-1#ref-CR71 "Chen, M. M. et al. TCPA v3.0: an integrative platform to explore the pan-cancer analysis of functional proteomic data. Mol. Cell Proteom. 18, S15–S25 (2019)."). Cell pellets containing 2 × 106 cells were lysed, and lysates were serially diluted and arrayed on nitrocellulose-coated slides to produce sample spots. Sample spots were probed with antibodies by a tyramide-based signal amplification approach and visualized using a 3,3′-diaminobenzidine colorimetric reaction to quantify sample spot densities. For each cell line, 50 μg of protein was used for RPPA analysis. Antibody validation was performed by Western blotting. Relative protein levels for each sample were determined by interpolating each dilution curve produced from the densities of the dilution sample spots using a standard curve for each antibody. All relative protein level data points were normalized for protein loading and transformed to linear values. Normalized linear values were transformed to log2 values and median-centered. A total of 425 antibodies were processed across all 10 human uveal melanoma cell lines.

### Cell culture and Cas9 cell line generation

All cell lines were authenticated by short tandem repeat profiling and screened negative for mycoplasma contamination. Cell-line source, culture conditions and antibiotic selection doses are provided in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02222-1#MOESM4). Cell culture media was replaced every 2–3 days. Cell lines were incubated in a humidified atmosphere of 5% CO2 at 37 °C. To generate Cas9-expressing cell lines, all cell lines underwent lentiviral transduction with pKLV2-EF1aBsd2ACas9-W (Addgene; 68343) and blasticidin selection 72 h after transduction using a predetermined concentration according to a blasticidin dose–response assay (blasticidin range = 5–30 μg ml−1). Cas9 activity was confirmed to be >85% for all cell lines before screening using a BFP/GFP reporter assay as previously described[72](https://www.nature.com/articles/s41588-025-02222-1#ref-CR72 "Tzelepis, K. et al. A CRISPR dropout screen identifies genetic vulnerabilities and therapeutic targets in acute myeloid leukemia. Cell Rep. 17, 1193–1205 (2016).") (Supplementary Fig. [6](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). All Cas9-expressing cell lines were under blasticidin selection to maintain maximal Cas9 activity. For GFP/BFP assessment, we used uninfected cells to establish our control gates. First, we selected cells based on their forward scatter and side scatter properties to exclude dead cells and debris. Next, we gated for singlets and excluded doublets by analyzing the FSC-W versus FSC-A plot. The fluorescence of BFP and GFP cell populations was detected following excitation with the 405 nm and 488 nm lasers, respectively.

### Combinatorial CRISPR library design

Paralog gene pairs were included if they had >45% sequence homology (Ensembl v92) and single _D. melanogaster_ (Flymine; FB2015\_15) or _C. elegans_ (Wormbase; WS251) essential orthologs. To computationally derive putative synthetic lethal gene pairs, we made use of the observation that GIs often occur among functionally related genes[73](https://www.nature.com/articles/s41588-025-02222-1#ref-CR73 "Tong, A. H. et al. Global mapping of the yeast genetic interaction network. Science 303, 808–813 (2004)."). We first identified, for each gene, the top 50 functionally most similar genes using the Mashup algorithm[31](https://www.nature.com/articles/s41588-025-02222-1#ref-CR31 "Cho, H., Berger, B. & Peng, J. Compact integration of multi-network topology for functional analysis of genes. Cell Syst. 3, 540–548 e5 (2016).") by calculating the Pearson correlation among (Mashup-based) gene-specific vectors representing an integration of the human STRING network. As demonstrated in Mashup’s original publication[31](https://www.nature.com/articles/s41588-025-02222-1#ref-CR31 "Cho, H., Berger, B. & Peng, J. Compact integration of multi-network topology for functional analysis of genes. Cell Syst. 3, 540–548 e5 (2016)."), this strategy allowed the identification of gene pairs with highly similar biological functions, including—but not restricted to—gene paralogs. For these potentially interacting gene pairs, we next used ordinary least squares regression (as implemented by the Python package Statsmodels; [https://www.statsmodels.org](https://www.statsmodels.org/)) on data from project Achilles[52](https://www.nature.com/articles/s41588-025-02222-1#ref-CR52 "Tsherniak, A. et al. Defining a cancer dependency map. Cell 170, 564–576 e16 (2017).") (v2.20.2) to test if copy number loss (defined as a copy number score <−0.3, GISTIC’s default noise threshold for deletions) of one gene of a pair was associated with increased essentiality (which is, a lower DEMETER-based _Z_\-score) of the partner gene, after adjustment for tissue of origin (503 cancer cell line essentiality scores available in the dataset were used for this analysis). Here we excluded gene pairs that resided on the same chromosome, as these follow similar copy number dynamics. Hence, a significant ‘interaction’ might be driven by increased sensitivity to knockdown when the gene itself is (heterozygously) deleted. For the resulting 357 significant (Benjamini–Hochberg FDR < 0.1) gene pairs, we used ordinary least squares regression to test, in a pan-cancer manner using data of 7,537 tumors/patients in TCGA (Supplementary Table [15](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)), if the deletion of one gene (again defined as a GISTIC copy number score <−0.3) was associated with a significant upregulation of RNA expression of the partner gene. This resulted in 125 significant (Benjamini–Hochberg FDR < 0.1) gene pairs, 115 of which were taken forward and were used as our set of computationally derived putative synthetic lethal pairs for interaction testing in our combinatorial CRISPR screens. Additional putative synthetic lethal gene pairs were selected from a systematic association analysis, across 274 cancer cell lines (Supplementary Table [16](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)), between gene expression and essentiality (Project Score[47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019).") 2018 release). Associations were tested using a linear regression model between gene expression (RNA-seq limma voom transformed)[74](https://www.nature.com/articles/s41588-025-02222-1#ref-CR74 "Garcia-Alonso, L. et al. Transcription factor activities enhance markers of drug sensitivity in cancer. Cancer Res. 78, 769–780 (2018).") and gene essentiality (CRISPR–Cas9 CRISPRcleanR corrected log2(FCs))[47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019)."),[49](https://www.nature.com/articles/s41588-025-02222-1#ref-CR49 "Iorio, F. et al. Unsupervised correction of gene-independent cell responses to CRISPR–Cas9 targeting. BMC Genomics 19, 604 (2018)."). The linear models were built using cell line growth rates and cancer type as covariates[75](https://www.nature.com/articles/s41588-025-02222-1#ref-CR75 "Goncalves, E. et al. Drug mechanism-of-action discovery through the integration of pharmacological and CRISPR screens. Mol. Syst. Biol. 16, e9405 (2020)."). For each association, statistical significance was assessed using a log-ratio test, where the null hypothesis model does not consider the CRISPR–Cas9 gene essentiality; that is, only the covariates are used to fit the gene expression measurements. A _P_ value was then calculated using a Chi-square distribution with one degree of freedom. The top 95 gene pan-cancer associations at an FDR threshold <0.05 were selected. Uveal melanoma-specific genes were selected by identifying known tumor suppressor genes that undergo loss-of-function mutations in uveal melanoma and genes that are underexpressed based on gene expression profiling[36](https://www.nature.com/articles/s41588-025-02222-1#ref-CR36 "Onken, M. D., Worley, L. A., Ehlers, J. P. & Harbour, J. W. Gene expression profiling in uveal melanoma reveals two molecular classes and predicts metastatic death. Cancer Res. 64, 7205–7209 (2004)."),[37](https://www.nature.com/articles/s41588-025-02222-1#ref-CR37 "Onken, M. D., Worley, L. A., Tuscan, M. D. & Harbour, J. W. An accurate, clinically feasible multi-gene expression assay for predicting metastasis in uveal melanoma. J. Mol. Diagn. 12, 461–468 (2010)."). Corresponding gene partners were selected if they were either (1) paralogues within a two to three member paralogue family, (2) established mutually exclusive genes with loss-of-function mutations in uveal melanoma[39](https://www.nature.com/articles/s41588-025-02222-1#ref-CR39 "Robertson, A. G. et al. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. Cancer Cell 32, 204–220 e15 (2017).") or (3) synthetic lethal as defined by a database of synthetic lethal GIs (SynLethDB v1.0 (ref. [38](https://www.nature.com/articles/s41588-025-02222-1#ref-CR38 "Guo, J., Liu, H. & Zheng, J. SynLethDB: synthetic lethality database toward discovery of selective and sensitive anticancer drug targets. Nucleic Acids Res. 44, D1011–D1017 (2016)."))). There were 42 such pairs in the library.

To construct the library, sgRNAs from published genome-wide CRISPR knockout libraries were used to select eight sgRNAs per gene, comprising four sgRNAs from the Human CRISPR Knockout Pooled Library (Brunello)[76](https://www.nature.com/articles/s41588-025-02222-1#ref-CR76 "Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR–Cas9. Nat. Biotechnol. 34, 184–191 (2016).") and four sgRNAs from the Toronto KnockOut library (TKOv3)[77](https://www.nature.com/articles/s41588-025-02222-1#ref-CR77 "Hart, T. et al. High-resolution CRISPR screens reveal fitness genes and genotype-specific cancer liabilities. Cell 163, 1515–1526 (2015)."). If fewer than eight unique sgRNAs were available from these two libraries, the remaining sgRNAs were selected from the Sabatini/Lander library[17](https://www.nature.com/articles/s41588-025-02222-1#ref-CR17 "Wang, T. et al. Identification and characterization of essential genes in the human genome. Science 350, 1096–1101 (2015).") or designed using CRISPick (Genome Perturbation Platform, Broad Institute [https://portals.broadinstitute.org/gppx/crispick/public](https://portals.broadinstitute.org/gppx/crispick/public)). All pgRNAs included in the library are provided in Supplementary Table [4](https://www.nature.com/articles/s41588-025-02222-1#MOESM4), and all oligos/sequences in Supplementary Table [17](https://www.nature.com/articles/s41588-025-02222-1#MOESM4). Most gene pairs had a total of 48 unique pgRNA combinations, of which 16 comprised pairing of an sgRNA with an STG to assess single-gene knockout effects. When fewer than six unique sgRNAs were available for a gene, that gene was excluded from the final library. Ten previously published STGs designed against genomic regions with no previously defined function across 127 cell lines[40](https://www.nature.com/articles/s41588-025-02222-1#ref-CR40 "Morgens, D. W. et al. Genome-scale measurement of off-target activity using Cas9 toxicity in high-throughput screens. Nat. Commun. 8, 15178 (2017).") were selected and validated by performing a surveyor assay using the Surveyor Mutation Detection Kit (IDT) according to the manufacturer’s instructions (Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM1)). The paired gRNA construct contained a modified tracrRNA and spacer to prevent recombination (Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM1) and Supplementary Table [17](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). The sequence of the vector, including the tracrRNA sequences, is available on Figshare[78](https://www.nature.com/articles/s41588-025-02222-1#ref-CR78 "Park, J. S. jpark27/CDS1-2: v1.0 (v1.0). Zenodo 
https://doi.org/10.5281/zenodo.15375418
(2025)."). As shown in Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM1), the hU6 promoter in the construct is stronger than the mU6 promoter, and thus, we placed sgRNAs targeting each gene under the control of each promoter. We noted a small number of sgRNAs in this figure with strongly negative values. These are from later time points from the MEL285 and MP46 lines, where screen quality was poor. It does not reflect off-target or spurious STG sgRNAs.

### Combinatorial CRISPR library construction

The combinatorial CRISPR pgRNA library was constructed using a protocol based on the method established by Vidigal and Ventura[79](https://www.nature.com/articles/s41588-025-02222-1#ref-CR79 "Vidigal, J. A. & Ventura, A. Rapid and efficient one-step generation of paired gRNA CRISPR–Cas9 libraries. Nat. Commun. 6, 8083 (2015)."). The library oligonucleotide pool for the uveal component of the library and the remainder of the library were synthesized separately (Twist Bioscience) to allow use of the library without the uveal component in other models. Each oligonucleotide pool was PCR amplified using 25 μl of Q5 High-Fidelity 2× Master Mix (New England Biolabs), 0.05 ng of oligonucleotide pool, 5 μl of primer mix at a concentration of 10 μM and nuclease-free water to a final volume of 50 μl per reaction. The thermocycling conditions were 98 °C for 30 s, followed by 17 cycles of amplification at 98 °C for 10 s, 68 °C for 35 s and 72 °C for 30 s, and a final extension at 72 °C for 10 min. Annealed oligonucleotides for the main library and the uveal library were separately ligated into BbsI-digested pDonor\_mU6 vector (Invitrogen GeneArt) in a 60 μl reaction containing 263 ng linearized pDonor\_mU6 and 481 ng oligonucleotide amplicon with 30 μl Gibson Assembly Master Mix 2× (New England Biolabs) and incubated at 50 °C for 2 h. Ligation of each insert to BsmbI-digested lentiGuide-Puro was performed using the Quick Ligation Kit (New England Biolabs) according to the manufacturer’s instructions and electroporated into 10-beta electrocompetent _Escherichia coli_ (New England Biolabs) and inoculated into 100 μg ml−1 ampicillin-containing Luria–Bertani broth for 16 h, followed by plasmid DNA purification. Plasmid DNA from the uveal and main components of the library was combined in varying proportions to determine (via sequencing) the optimal combination with regard to pgRNA representation and correct proportion of pgRNAs from each library. Lentivirus production was carried out in 7.2 × 107 HEK293T cells transfected with 24 μg VSV-G, 40 μg pMDLg/pRRE, 20 μg pRSV-Rev, 67.2 μg pAdvantage and 69.6 μg of library plasmid with Lipofectamine 3000 (Thermo Fisher Scientific). Media was replaced 16 h after transduction, and viral supernatant was collected after 48 h, filtered through a 0.45 μM membrane (Merck; SLHP033RS), and stored at −80 °C.

### Combinatorial pgRNA CRISPR screening

For each cell line, 9 × 107 cells were transduced with the combinatorial CRISPR library at the required volume to achieve a transduction efficiency of 30% (1,000× library coverage), as determined by cell viability in puromycin (CellTiter 96 AQueous Non-Radioactive Cell Proliferation Assay (MTS), Promega). Puromycin concentrations for each cell line were determined using a dose–response assay (puromycin range = 2–4 μg ml−1). Puromycin selection was performed at day 3 post-transduction for a total of 7 days. Library infections were performed in triplicate, and a minimum library representation of 3,000× was maintained at every passage throughout the screen. Cells were first collected at 28 days post-transduction. Cas9-negative cell lines were transduced under the same conditions and collected at day 7 to allow calculation of pgRNA FC. Genomic DNA was extracted for each replicate from cell pellets using a Blood & Cell Culture DNA Maxi Kit (Qiagen).

### Analysis of the combinatorial pgRNA CRISPR screen

Plasmid and genomic DNA sequencing were performed as previously described[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."). All primer sequences are included in Supplementary Table [17](https://www.nature.com/articles/s41588-025-02222-1#MOESM4). Guide quantification was performed using pyCROQUET v1.5.1 ([https://github.com/cancerit/pycroquet](https://github.com/cancerit/pycroquet)). Raw counts are provided in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02222-1#MOESM4). Normalized counts from Cas9-expressing lines were compared with normalized counts from control Cas9-negative lines to calculate log(FCs). sgRNAs with less than 20 read counts in the control lines were removed. Single-gene essentiality was tested through C-SAR v1.3.6 ([https://github.com/cancerit/C-SAR](https://github.com/cancerit/C-SAR)) using BAGEL2 (ref. [80](https://www.nature.com/articles/s41588-025-02222-1#ref-CR80 "Kim, E. & Hart, T. Improved analysis of CRISPR fitness screens and reduced off-target effects with the BAGEL2 gene essentiality classifier. Genome Med. 13, 2 (2021).")) and MAGeCK[81](https://www.nature.com/articles/s41588-025-02222-1#ref-CR81 "Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Genome Biol. 15, 554 (2014)."). log(FCs) were scaled so the median log(FC) of nonessential genes was 0, and known essentials were −1 (log2(FCs) are scaled log2(FCs) unless otherwise specified). The reference essential gene set (CEGv2) and reference nonessential gene set (NEGv1) were obtained from [https://github.com/hart-lab/bagel](https://github.com/hart-lab/bagel). Synthetic lethal hits were called using an adapted version of the BASSIK method, as applied previously[21](https://www.nature.com/articles/s41588-025-02222-1#ref-CR21 "Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Nat. Commun. 12, 1302 (2021)."). Hits were called using the Bliss independence model, where the observed reduction in fitness on knockout of both genes in combination is significantly greater than predicted from the addition of the single knockout fitness effects. Predicted dual knockout log(FCs) were adjusted using Loess regression, modeling the local behavior of the data to account for a plateau in dual KO log(FCs) when one gene is already close to essential. GI scores were calculated per pair of guides from the difference between the predicted log(FCs) for both guides in combination and the observed dual log(FCs). This was normalized by the square root of the binned variance to account for heteroscedasticity. A gene pair was called a hit if (1) the mean normalized GI score across all guide pairs and replicates was <−0.5 and (2) a one-sided _t_ test for whether the median gene pair GI score is less than the median GI score across the dataset had an FDR < 0.01 (Benjamini–Hochberg multiple testing correction). Hit pairs where either gene in the pair was classified as singly essential by BAGEL and MAGeCK were filtered out to avoid calling hits identified because of one gene being essential.

### Assessment of _CDS1_ expression and _CDS2_ essentiality

To test the association between _CDS1_ expression and _CDS2_ essentiality across all cancer types, gene essentiality data were obtained from Project Score ([https://score.depmap.sanger.ac.uk/downloads](https://score.depmap.sanger.ac.uk/downloads)) and Project Achilles (DepMap 22Q2, [https://depmap.org/portal/download/all/](https://depmap.org/portal/download/all/)). For Project Score essentiality scores, we used quantile-normalized log(FC) values corrected using CRISPRcleanR[49](https://www.nature.com/articles/s41588-025-02222-1#ref-CR49 "Iorio, F. et al. Unsupervised correction of gene-independent cell responses to CRISPR–Cas9 targeting. BMC Genomics 19, 604 (2018)."). Quantitative essentiality data (copy number corrected Chronos scores[82](https://www.nature.com/articles/s41588-025-02222-1#ref-CR82 "Dempster, J. M. et al. Chronos: a cell population dynamics model of CRISPR experiments that improves inference of gene fitness effects. Genome Biol. 22, 343 (2021).")) for the Project Achilles data set were downloaded from the DepMap portal (22Q2 release). Gene expression data were obtained from Cell Model Passports ([https://cellmodelpassports.sanger.ac.uk](https://cellmodelpassports.sanger.ac.uk/)) for Project Score cell lines and the DepMap download portal (22Q4) for Project Achilles cell lines. Cell lines with _CDS1_ expression log2(TPM + 1) <1 were annotated as low expressers, and the rest were annotated as high expressers. _t_ tests (two-tailed, equal variance) were performed to test whether _CDS2_ is more essential in _CDS1_ low expressers.

### Whole-genome single gRNA CRISPR knockout screen and analysis

Genome-wide sgRNA CRISPR knockout screening was performed using the Human Improved Genome-wide Knockout CRISPR Library v1.1 (ref. [47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019).")). For each cell line, 3.3 × 107 cells were transduced with the required volume of Human CRISPR Library v1.1 to achieve a transduction efficiency of 30% (100× library coverage). Puromycin selection (2–4 μg ml−1) was performed at day 3 post-transduction for a total of 7 days. Screening was performed in technical triplicate, and cells were cultured throughout the screen at a minimum library representation of 500×. Cells were collected 14 days post-transduction, pelleted and stored at −80 °C. Genomic DNA was extracted from cell pellets using a Blood & Cell Culture DNA Maxi Kit (QIAGEN). PCR amplification and sequencing were performed as previously described[72](https://www.nature.com/articles/s41588-025-02222-1#ref-CR72 "Tzelepis, K. et al. A CRISPR dropout screen identifies genetic vulnerabilities and therapeutic targets in acute myeloid leukemia. Cell Rep. 17, 1193–1205 (2016).").

All samples were processed using CRISPRcleanR v3.0.1 (refs. [49](https://www.nature.com/articles/s41588-025-02222-1#ref-CR49 "Iorio, F. et al. Unsupervised correction of gene-independent cell responses to CRISPR–Cas9 targeting. BMC Genomics 19, 604 (2018)."),[50](https://www.nature.com/articles/s41588-025-02222-1#ref-CR50 "Vinceti, A. et al. An interactive web application for processing, correcting, and visualizing genome-wide pooled CRISPR–Cas9 screens. Cell Rep. Methods 3, 100373 (2023).")). Only sgRNAs in common with the Human CRISPR Library v1.0 (ref. [72](https://www.nature.com/articles/s41588-025-02222-1#ref-CR72 "Tzelepis, K. et al. A CRISPR dropout screen identifies genetic vulnerabilities and therapeutic targets in acute myeloid leukemia. Cell Rep. 17, 1193–1205 (2016).")) were analyzed to prevent unbalanced targeting of essential genes with additional guides in version 1.1 (ref. [47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019).")). Raw counts are provided in Supplementary Table [9](https://www.nature.com/articles/s41588-025-02222-1#MOESM4). sgRNAs with less than 30 read counts in the plasmid were removed. The remaining sgRNA raw counts were normalized by their total number within their replicate. Depletion and enrichment log-fold changes (logFCs) for individual sgRNAs were quantified at the individual replicate level between library plasmid read counts and postlibrary-transduction read counts. Gene-independent responses to CRISPR–Cas9 targeting were corrected using the default parameters. MAGeCK MLE v0.5.9.5 was run on the normalized corrected sgRNAs’ treatment counts after applying the inverse transformation on the corrected log(FCs) as previously described[49](https://www.nature.com/articles/s41588-025-02222-1#ref-CR49 "Iorio, F. et al. Unsupervised correction of gene-independent cell responses to CRISPR–Cas9 targeting. BMC Genomics 19, 604 (2018)."). Significantly depleted essential genes were identified as those with a MAGeCK MLE beta score < 0.5 and an FDR < 0.05. Previously established essential genes (CEGv2) were excluded from the list of essential genes.

### Identification of uveal melanoma-specific gene hits

To identify uveal melanoma-specific gene hits, the essentiality scores from our genome-wide sgRNA CRISPR screen and pan-cancer cell lines from DepMap 22Q2 were compared. Scores were rank normalized to allow for comparison between different screens. The Mann–Whitney _U_ test was used to identify significant differences, and log2 fold changes from the ratio of median ranks were used to determine the direction of essentiality. Pan-essential genes identified using ProdeTool ([https://github.com/cantorethomas/prodeTool](https://github.com/cantorethomas/prodeTool)) were removed from the results. UVM-specific vulnerabilities (_n_ = 76; log2(FC) > 1.80 and _P_adj < 0.01) were then used as an input for gene set enrichment analysis using the GO Molecular Function pathway gene set.

### Construction of the core UVmap reference

In developing the core UVmap (a uveal melanoma single-cell reference map), we used the GBmap pipeline approach as described in ref. [83](https://www.nature.com/articles/s41588-025-02222-1#ref-CR83 "Ruiz-Moreno, C. et al. Harmonized single-cell landscape, intercellular crosstalk and tumor architecture of glioblastoma. Preprint at bioRxiv 
https://doi.org/10.1101/2022.08.27.505439
(2022)."), collecting data from 264,624 cells from 26 tumors and 29 normal tissues. We included only those samples confirmed as healthy eye or primary uveal melanoma, and healthy liver or metastatic liver, with each sample containing no fewer than 1,000 cells. The datasets for the core UVmap, which include refs. [84](https://www.nature.com/articles/s41588-025-02222-1#ref-CR84 "Bakhoum, M. F. et al. Loss of polycomb repressive complex 1 activity and chromosomal instability drive uveal melanoma progression. Nat. Commun. 12, 5402 (2021)."),[85](https://www.nature.com/articles/s41588-025-02222-1#ref-CR85 "Lin, W. et al. Intra- and intertumoral heterogeneity of liver metastases in a patient with uveal melanoma revealed by single-cell RNA sequencing. Cold Spring Harb. Mol. Case Stud. 7, a006111 (2021)."),[86](https://www.nature.com/articles/s41588-025-02222-1#ref-CR86 "Ramachandran, P. et al. Resolving the fibrotic niche of human liver cirrhosis at single-cell level. Nature 575, 512–518 (2019)."),[87](https://www.nature.com/articles/s41588-025-02222-1#ref-CR87 "Gautam, P. et al. Multi-species single-cell transcriptomic analysis of ocular compartment regulons. Nat. Commun. 12, 5675 (2021).") were primarily in the form of raw count matrices. Where raw matrices were not available, BAM files were downloaded directly from the dbGaP cloud[88](https://www.nature.com/articles/s41588-025-02222-1#ref-CR88 "Durante, M. A. et al. Single-cell analysis reveals new evolutionary complexity in uveal melanoma. Nat. Commun. 11, 496 (2020).") ([phs001861.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001861.v1.p1); approved by dbGaP on 24 May 2022) or sourced directly from the authors[89](https://www.nature.com/articles/s41588-025-02222-1#ref-CR89 "Pandiani, C. et al. Single-cell RNA sequencing reveals intratumoral heterogeneity in primary uveal melanomas and identifies HES6 as a driver of the metastatic disease. Cell Death Differ. 28, 1990–2000 (2021)."), and were then transformed into FASTQ files and re-aligned using the STARsolo v2.7.10a pipeline ([https://github.com/cellgeni/STARsolo](https://github.com/cellgeni/STARsolo)). We updated all gene names to the most current HUGO nomenclature via HGNChelper and ensured all clinical and diagnostic metadata remained consistent. Before integrating the datasets, we applied stringent filtering parameters to select only high-quality cells, excluding those with fewer than 500 genes, fewer than 1,000 unique molecular identifier counts (where applicable), and over 30% mitochondrial reads. Doublets in each droplet-based dataset were identified and removed using DoubletFinder.

To mitigate batch effects across the datasets, we employed a semi-supervised neural network model called single-cell ANnotation using Variational Inference (scANVI)[90](https://www.nature.com/articles/s41588-025-02222-1#ref-CR90 "Xu, C. et al. Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models. Mol. Syst. Biol. 17, e9620 (2021)."), within the transfer-learning framework of the single-cell architectural surgery algorithm (scArches)[91](https://www.nature.com/articles/s41588-025-02222-1#ref-CR91 "Lotfollahi, M. et al. Mapping single-cell data to reference atlases by transfer learning. Nat. Biotechnol. 40, 121–130 (2022)."). scArches–SCANVI requires prior knowledge of cell types and labels to create a reference map. To standardize cell type labels from different sources, we annotated each dataset employing both automated and manual methods. For the automated process, we initially collected lists of melanoma and GEP markers from ref. [88](https://www.nature.com/articles/s41588-025-02222-1#ref-CR88 "Durante, M. A. et al. Single-cell analysis reveals new evolutionary complexity in uveal melanoma. Nat. Commun. 11, 496 (2020)."), 16 cancer cell states[92](https://www.nature.com/articles/s41588-025-02222-1#ref-CR92 "Barkley, D. et al. Cancer cell states recur across tumor types and form specific interactions with the tumor microenvironment. Nat. Genet. 54, 1192–1201 (2022)."), and a list of 174 adult eye and liver markers from a study published in ref. [93](https://www.nature.com/articles/s41588-025-02222-1#ref-CR93 "Quan, F. et al. Annotation of cell types (ACT): a convenient web server for cell type annotation. Genome Med. 15, 91 (2023)."). We then performed UCell signature scoring[94](https://www.nature.com/articles/s41588-025-02222-1#ref-CR94 "Andreatta, M. & Carmona, S. J. UCell: robust and scalable single-cell gene signature scoring. Comput. Struct. Biotechnol. J. 19, 3796–3798 (2021).") and applied a cutoff value of 0.2 to assign cells as state/marker positive.

Subsequently, manual cell identity was assigned based on results from the automated process, available original cell labels and specific gene expression patterns analyzed using the Wilcoxon rank-sum test. CNV analysis was conducted using the CopyKAT package[95](https://www.nature.com/articles/s41588-025-02222-1#ref-CR95 "Gao, R. et al. Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes. Nat. Biotechnol. 39, 599–608 (2021)."), categorizing cells as either diploid or aneuploid. This preliminary coarse cell type labeling facilitated the training and integration of the model through scANVI-scArches.

The analysis was conducted on the raw counts from the 5,000 most variable genes, using studies as the batch variable and adhering to the recommended tool parameters. The output from the pipeline was a latent representation of the integrated data, which then served as input for clustering and dimensional reduction visualizations. We applied Leiden clustering based on a k-nearest neighbor graph (k-NNG)[96](https://www.nature.com/articles/s41588-025-02222-1#ref-CR96 "Traag, V. A., Waltman, L. & van Eck, N. J. From Louvain to Leiden: guaranteeing well-connected communities. Sci. Rep. 9, 5233 (2019).") to identify distinct cell populations, and Uniform Manifold Approximation and Projection (UMAP)[97](https://www.nature.com/articles/s41588-025-02222-1#ref-CR97 "Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. Nat. Biotechnol. 37, 38–44 (2019).") for data embedding and two-dimensional reduction, using the plot1cell package[98](https://www.nature.com/articles/s41588-025-02222-1#ref-CR98 "Wu, H. et al. Mapping the single-cell transcriptomic response of murine diabetic kidney disease to therapies. Cell Metab. 34, 1064–1078.e6 (2022).") for UMAP visualization. Post-co-embedding, cell identities were refined manually for each cluster, using our unified preliminary annotations and evaluating specific marker gene expression to accurately define each broad cell type or state.

### Validation of on-target _CDS2_ sgRNA on-target activity

Because targeting closely related sequences, such as paralogs, could result in off-target cutting and paralog codisruption, we validated on-target sgRNA cutting for the sgRNA used in our validation experiments. We infected/transduced SW837\_C9 cells stably expressing Cas9 and having Cas9 activity >90% with either a _CDS2_ sgRNA (GAGTAAAGGAAATGAACCGG) or a safe-targeting control sgRNA (STG1; GTATCAACAGAGTGTCAGAT) at an MOI of 0.34–0.38. Cells were subsequently selected with puromycin for 7 days, and then RNA was extracted for whole transcriptome sequencing. The parental (uninfected) SW837\_C9 cell line was also sequenced as a control. Analysis was performed as described above.

### Validation by sgRNA competitive proliferation assay

sgRNA competitive proliferation assays were performed using the pKLV2-U6gRNA5(BbsI)-PGKpuro2ABFP-W (Addgene; 67974) vector. To validate individual target genes, one sgRNA from the Human Improved Genome-wide Knockout CRISPR Library v1.1 (ref. [47](https://www.nature.com/articles/s41588-025-02222-1#ref-CR47 "Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Nature 568, 511–516 (2019).")) was used, and an sgRNA derived from the MinLibCas9 library[99](https://www.nature.com/articles/s41588-025-02222-1#ref-CR99 "Goncalves, E. et al. Minimal genome-wide human CRISPR–Cas9 library. Genome Biol. 22, 40 (2021)."). Viral supernatants were collected 48 h after transfection into packaging lines, and transduction of _CDS1_\-null uveal cell lines (OMM2.5 and MP46) was performed three times independently at an MOI between 0.5 and 0.8. Flow cytometry was performed on six-well plates using a CytoFLEX flow cytometer (Beckman Coulter) and analyzed with FCS Express v7.22.0031 to determine the proportion of BFP-positive cells at days 4, 14 and 28.

### Construction of inducible _CDS2_ and STG2 knockout isogenic cell lines

A potent and specific sgRNA against _CDS2_ (validated above) or a safe-targeting sgRNA (STG2) was cloned into a tet-inducible vector pRSTGEBleo-U6Tet-sg-EF1-TetRep-2A-Bleo (Cellecta) according to the manufacturer’s instructions and sent for Sanger/capillary sequencing to confirm correct assembly. Vectors were transduced into MP41-Cas9 and OMM2.5-Cas9 cells. Where clones were used, each transduced cell line was sorted at single-cell density into 96-well plates containing media and then zeocin selected for 4 weeks; polyclonal lines were selected for 2 weeks. To confirm editing of the _CDS2_ gene, genomic DNA was extracted from colonies using the Gentra Puregene Blood Kit (Qiagen) and sent for Sanger sequencing. Western blot analysis was performed on _CDS2_ knockout clones using an anti-CDS2 antibody (Proteintech, 13175-1-AP; 1:1,000 dilution) and an anti-vinculin antibody (Thermo Fisher Scientific, MA5-11690; 1:10,000 dilution) as a loading control (Extended Data Figs. [1](https://www.nature.com/articles/s41588-025-02222-1#Fig7) and [2](https://www.nature.com/articles/s41588-025-02222-1#Fig8)). Thermo Novex Sharp prestained protein markers (Thermo Fisher Scientific) were used.

### Clonogenic assay

In each 6-well plate, up to 1,300 Dox-inducible cells were seeded per well. After 24 h, media were replaced with media containing 0.1 μg ml−1 Dox or the equivalent volume of dimethyl sulfoxide (DMSO). After 14 days, colonies were washed with ice-cold PBS, fixed with 4% paraformaldehyde (PFA) and stained with 0.1% crystal violet solution.

### Genetic rescue of _CDS2_ loss with a _CDS1_ cDNA

Constitutively Cas9-expressing OMM2.5 cells were transduced with a lentivirus containing a full-length _CDS1_ cDNA (Origene; RC210375L3) and selected in puromycin for 7 days. Transduction with an empty vector (Origene; PS100092) followed by puromycin selection for 7 days served as a negative control. Cells were then transduced with lentivirus containing a _CDS2_ sgRNA (GAGTAAAGGAAATGAACCGG) or a safe-targeting sgRNA (STG1) at a viral titer that achieved 100% transduction. Subsequently, 1,200 sgRNA-infected cells were seeded per well in a six-well plate in technical triplicate. Following culture for 18 days, colonies were washed with ice-cold PBS, fixed with 100% ethanol and stained with 1% crystal violet solution. The aforementioned experiments were performed in biological triplicate (that is, three independent trials).

### Apoptosis assessment

Apoptosis and cell death were assessed by flow cytometry using Annexin V staining. We infected the uveal melanoma cell lines with a _CDS2_ sgRNA (GAGTAAAGGAAATGAACCGG) or a safe-targeting control sgRNA (STG2; GTATCAACAGAGTGTCAGAT) lentivirus at an MOI of 0.8. Cells were selected with puromycin for 4 days, and then a total of 2 × 105 cells were plated in each well with 2 ml of media. At day 14, floating and adherent cells were collected from each well. Cells were washed with 500 μl of Annexin V Binding Buffer (BD Biosciences) and resuspended in 100 µl dilution containing 5 μl PE Annexin V (BioLegend) for 15 min at room temperature. Cells were centrifuged and resuspended in 100 µl dilution with 20 µl 7-AAD (BD Biosciences) for 10 min and then pelleted and resuspended in 500 µl Annexin V Binding Buffer for flow cytometry analysis using a CytoFLEX flow cytometer (Beckman Coulter). Annexin V-/7-AAD- cells were deemed to be viable, and Annexin V+/7-AAD- cells were early apoptotic. Finally, Annexin V+/7-AAD+ cells were considered late apoptotic, and 7-AAD+ necrotic cells. Analysis was performed with FCS Express v7.22.0031.

### Proteome-wide analysis following _CDS2_ disruption

These experiments were performed using clonal OMM2.5P and MP41 E cells and were performed in three biological replicate experiments with the sgRNA targeting _CDS2_ activated using Dox (as above) for 7 days before cells were pelleted and snap frozen for analysis. Analysis was performed essentially as previously described[100](https://www.nature.com/articles/s41588-025-02222-1#ref-CR100 "Sialana, F. J. et al. SimPLIT: simplified sample preparation for large-scale isobaric tagging proteomics. J. Proteome Res. 21, 1842–1856 (2022)."). Briefly, LC–MS analysis was carried out using a Dionex UltiMate 3000 UHPLC system coupled with an Orbitrap Ascend mass spectrometer (Thermo Fisher Scientific) and analyzed using a Real-Time Search-SPS-MS3 method. Approximately 3 μg of peptides from each fraction were injected onto a C18 trapping column (Acclaim PepMap 100, 100 μm × 2 cm, 5 μm, 100 Å) at a flow rate of 10 μl min−1. The peptides were then subjected to a 120-min low-pH gradient elution on a nanocapillary reversed-phase column (Acclaim PepMap C18, 75 μm × 50 cm, 2 μm, 100 Å) at 50 °C. MS1 scans were conducted over a mass range of _m_/_z_ 400–1600 using the Orbitrap at 120,000 resolution, with standard AGC settings and automatic injection time. Charge states between +2 and +6 were included. Dynamic exclusion was set to 45 s with a repeat count of 1, a mass tolerance of ±10 ppm and isotopes were excluded from further analysis.

MS2 spectra were acquired in the ion trap using a Turbo scan rate with a higher energy collision dissociation energy of 32% and a maximum injection time of 35 ms. Real-time database searching was conducted against _Homo sapiens_ (canonical and isoforms) using the Comet search engine with tryptic peptides, allowing a maximum of one missed cleavage. Static modifications included carbamidomethylation of C (+57.0215 Da) and TMTpro labeling on K and N termini (+304.207 Da). Variable modifications included deamidation of N/Q (+0.984 Da) and oxidation of M (+15.9949 Da), with a maximum of two variable modifications per peptide. Close-out was enabled, allowing a maximum of four peptides per protein.

Selected precursors were subjected to SPS10-MS3 scans using an Orbitrap detector at 45,000 resolution, with a higher energy collision dissociation energy of 55%, a normalized AGC target of 200% and a maximum injection time of 200 ms. Data were collected in centroid mode with a single microscan acquisition.

Proteome Discoverer 3.0 (Thermo Fisher Scientific) was used with SequestHT and Comet search engines for protein identification and quantification. Spectra were searched against Homo sapiens protein entries in UniProt, with a precursor mass tolerance of 20 ppm and a fragment mass tolerance of 0.02 Da. Fully tryptic peptides were considered, allowing up to two missed cleavages. Static modifications were TMT at N terminus/K and carbamidomethylation at C residues. Dynamic modifications included oxidation of methionine and deamidation of N/Q.

Peptide confidence was estimated using Percolator, maintaining an FDR of 0.01 with target-decoy database validation. Quantification was performed using the TMT quantifier node, with a 15 ppm integration window and the most confident centroid peak at the MS2 level. Only unique peptides were used for quantification, with a signal-to-noise ratio threshold of >3. Data were normalized to total protein loading, and relative abundances were calculated by dividing normalized values by the average abundance across all TMT channels per biological replicate. The MS proteomics data have been deposited in the ProteomeXchange Consortium

### Quantification of PA and PIPn

For PA and PI analysis, cells were cultured in 6-well plates at appropriate densities for each time point and treated with 0.1 μg ml−1 Dox or the equivalent volume of DMSO. At days 3, 5 and 7, culture medium was aspirated from each well and the cells were washed in ice-cold PBS before being quenched in 1 ml of 1 M ice-cold HCl. The cells from parallel wells were disaggregated in 1 ml of trypsin and counted to determine the cell number per well at each time point. The cells in the HCl wells were scraped and resuspended, and a volume of the cell suspension containing 7 × 105 cells was pelleted and frozen. A modified Folch method was used to extract their lipids and PA and PI were analyzed using LC–MS as previously described[67](https://www.nature.com/articles/s41588-025-02222-1#ref-CR67 "Collins, D. M. et al. CDS2 expression regulates de novo phosphatidic acid synthesis. Biochem. J. 481, 1449–1473 (2024)."). Technical duplicates were averaged for each clone. Data from three MP41 clones were pooled together. Data from one OMM2.5 clone was collected in three independent experiments and pooled. PA, PI, PIP and PIP2 internal standards (ISDs) were used for correction. No ISDs were used for PC. PC was analyzed in a one-tenth aliquot of the prederivatization organic phase (Folch extraction). To compensate for the heterogeneity between clones and experiments, normalization by the area of 34:1 PC and normalization by the average values for the DMSO-treated samples at days 3, 5 and 7 were performed.

### Genetic suppressor screen

To identify genes whose deletion rescues lethality following _CDS2_ loss, a genetic suppressor screen was performed in biological triplicate using the minimal genome-wide human CRISPR–Cas9 library (MinLibCas9) in a clonal population of MP41 Dox-inducible _CDS2_ knockout cells. Infections were carried out in biological triplicate at an MOI of 0.3 and a library representation of 1,000×. Puromycin selection was performed from days 3 to 7 post-transduction. Cells were split and treated with 0.1 μg ml−1 of Dox or the equivalent volume of DMSO, replenished every 3 days, and maintained at a minimum representation of 500× throughout the screen for 21 days. Genomic DNA was extracted using a Gentra Puregene Cell Kit (Qiagen), PCR amplified and sequenced.

sgRNAs with less than 30 read counts across all samples were removed. To account for between-sample sequencing depth biases, total normalization was performed on raw sample counts by adding a pseudocount of 5 and normalizing to 10 million reads. log2(FCs) were calculated from the normalized and filtered counts between DMSO-treated read counts and Dox-treated read counts. MAGeCK v0.5.9.3 was run to identify genes significantly under positive selection at an FDR < 0.05.

### In vivo assessment of _CDS2_ essentiality in OMM2.5 xenografts

A total of 28 female nonobese diabetic-severe combined immunodeficient (NOD-SCID) γ mice (NOD-_Prkdc__scid__\-IL2rg__Tm1_/Rj) at 8 weeks of age were subcutaneously administered 2 × 106 OMM2.5 cells (in 0.1 ml 50:50 Matrigel/phosphate-buffered saline mix) into the left flank. The mice were fed standard chow (Safe-Lab, Safe105) and 24 days after dosing, were randomly assigned into two cohorts, with one cohort being fed a Dox diet (625 mg kg−1; Envigo, TD.01306) and the other remaining on standard chow for the entirety of the study. The developing tumors were measured once per week initially. In week 3, after cohort assignment and Dox treatment, tumors were measured two to three times per week, and then four times per week for the last 8 weeks of the study. Individual mice reached an endpoint on the study if tumors reached 1.5 cm2 (calculated by the longest length measurement × the longest width measurement) or lost 15% of their maximum body weight or 10% of their body weight in combination with other signs of ill health as determined by body condition assessments. Mice reaching study endpoints were humanely killed and the mass was excised, weighed and either snap frozen and stored at −80 °C or fixed using 10% neutral-buffered formalin for 24 h before storing in 70% ethanol. During the study, mice were maintained in a specific pathogen-free unit on a 12-h light/12-h dark cycle. The ambient temperature was 21 ± 2 °C, and the humidity was 55 ± 10%. Mice were housed using a stocking density of three to five mice per cage in individually ventilated caging, receiving 60 air changes per hour. In addition to the bedding substrate, standard environmental enrichment was provided. Mice were given water and diet ad libitum.

### Sequence analysis of residue-grafted uveal melanoma cells

To sequence the transcriptome of tumors collected from the mice, we used exome capture sequencing with Agilent V5 baits to enrich for human reads and filtered the data to remove mouse reads using XenofilteR after mapping with the STAR aligner (v2.5.0c). Expression assessment used Kallisto (0.51.1) and Sleuth (0.30.1). For each sample, >50 million reads were generated.

### Statistics and reproducibility

Statistical methods are detailed in the figure legends. Unless specified, all tests were two-sided, and _P_ < 0.05 was considered statistically significant. Multiple testing correction was applied where appropriate using FDR or Bonferroni correction. No data were excluded from the analyses. Mouse experiments aligned with the Animal Research: Reporting of In Vivo Experiments (ARRIVE) guidelines. Allocation was randomized.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02222-1#MOESM2) linked to this article.

Data availability
-----------------

The sequencing data generated as part of this study are available using the following European Nucleotide Accession numbers:

[ERP151504](https://www.ebi.ac.uk/ena/data/view/ERP151504): Paired guide (combinatorial) uveal melanoma screen data.

[ERP151444](https://www.ebi.ac.uk/ena/data/view/ERP151444): Single-guide (CRISPRko) uveal melanoma screen data.

[ERP151445](https://www.ebi.ac.uk/ena/data/view/ERP151445): Uveal melanoma suppressor screen.

[ERP110320](https://www.ebi.ac.uk/ena/data/view/ERP110320): Uveal melanoma cell line WGS.

[ERP130186](https://www.ebi.ac.uk/ena/data/view/ERP130186): Whole transcriptome sequencing of uveal melanoma cell lines.

[ERP159012](https://www.ebi.ac.uk/ena/data/view/ERP159012): Sequencing of SW837 cells following _CDS2_ sgRNA transduction and selection. Transcriptome sequencing of mouse tumors.

[ERP159013](https://www.ebi.ac.uk/ena/data/view/ERP159013): Sequencing of SW837 cells following _CDS2_ sgRNA transduction and selection.

Proteome data generated as part of this study are available via the PRIDE repository:

[PXD053752](http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD053752): Targeting the CDS1/2 axis as a therapeutic strategy in uveal melanoma and pan-cancer.

TCGA data was downloaded from the Xena Browser: [https://xenabrowser.net/](https://xenabrowser.net/)

All other data is available in the Supplementary Information. [Source data](https://www.nature.com/articles/s41588-025-02222-1#Sec42) are provided with this paper.

Code availability
-----------------

All code is available via Zenodo at [https://doi.org/10.5281/zenodo.15124575](https://doi.org/10.5281/zenodo.15124575) (ref. [101](https://www.nature.com/articles/s41588-025-02222-1#ref-CR101 "Billington, J., Del Castillo Velasco-Herrera, M. & Offord, V. team113sanger/targeting-the-CDS1-2-axis-as-a-therapeutic-strategy-in-uveal-melanoma-and-pan-cancer: post review fixes (v1.0.1). Zenodo 
https://doi.org/10.5281/zenodo.15124575
(2025).")). The UVMap code is available via GitHub at [https://github.com/jpark27/CDS1-2](https://github.com/jpark27/CDS1-2) (ref. [78](https://www.nature.com/articles/s41588-025-02222-1#ref-CR78 "Park, J. S. jpark27/CDS1-2: v1.0 (v1.0). Zenodo 
https://doi.org/10.5281/zenodo.15375418
(2025).")).

References
----------

1.  Khoja, L. et al. Meta-analysis in metastatic uveal melanoma to determine progression free and overall survival benchmarks: an international rare cancers initiative (IRCI) ocular melanoma study. _Ann. Oncol._ **30**, 1370–1380 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB3M3hvVOgug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31150059)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Meta-analysis%20in%20metastatic%20uveal%20melanoma%20to%20determine%20progression%20free%20and%20overall%20survival%20benchmarks%3A%20an%20international%20rare%20cancers%20initiative%20%28IRCI%29%20ocular%20melanoma%20study&journal=Ann.%20Oncol.&volume=30&pages=1370-1380&publication_year=2019&author=Khoja%2CL) 
    
2.  Rantala, E. S., Hernberg, M. & Kivela, T. T. Overall survival after treatment for metastatic uveal melanoma: a systematic review and meta-analysis. _Melanoma Res._ **29**, 561–568 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30664106)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6887637)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overall%20survival%20after%20treatment%20for%20metastatic%20uveal%20melanoma%3A%20a%20systematic%20review%20and%20meta-analysis&journal=Melanoma%20Res.&volume=29&pages=561-568&publication_year=2019&author=Rantala%2CES&author=Hernberg%2CM&author=Kivela%2CTT) 
    
3.  Nathan, P. et al. Overall survival benefit with tebentafusp in metastatic uveal melanoma. _N. Engl. J. Med._ **385**, 1196–1206 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitV2ksbbE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34551229)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Overall%20survival%20benefit%20with%20tebentafusp%20in%20metastatic%20uveal%20melanoma&journal=N.%20Engl.%20J.%20Med.&volume=385&pages=1196-1206&publication_year=2021&author=Nathan%2CP) 
    
4.  Carvajal, R. D. et al. Clinical and molecular response to tebentafusp in previously treated patients with metastatic uveal melanoma: a phase 2 trial. _Nat. Med._ **28**, 2364–2373 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1eiur7I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36229663)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9671803)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20and%20molecular%20response%20to%20tebentafusp%20in%20previously%20treated%20patients%20with%20metastatic%20uveal%20melanoma%3A%20a%20phase%202%20trial&journal=Nat.%20Med.&volume=28&pages=2364-2373&publication_year=2022&author=Carvajal%2CRD) 
    
5.  Shoushtari, A. N. et al. A phase Ib study of sotrastaurin, a PKC inhibitor, and alpelisib, a PI3Kα inhibitor, in patients with metastatic uveal melanoma. _Cancers (Basel)_ **13**, 5504 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisVSisbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34771668)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20phase%20Ib%20study%20of%20sotrastaurin%2C%20a%20PKC%20inhibitor%2C%20and%20alpelisib%2C%20a%20PI3K%CE%B1%20inhibitor%2C%20in%20patients%20with%20metastatic%20uveal%20melanoma&journal=Cancers%20%28Basel%29&volume=13&publication_year=2021&author=Shoushtari%2CAN) 
    
6.  Shoushtari, A. N. et al. A phase 2 trial of everolimus and pasireotide long-acting release in patients with metastatic uveal melanoma. _Melanoma Res._ **26**, 272–277 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XmvFShs7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26795274)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5553199)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20phase%202%20trial%20of%20everolimus%20and%20pasireotide%20long-acting%20release%20in%20patients%20with%20metastatic%20uveal%20melanoma&journal=Melanoma%20Res.&volume=26&pages=272-277&publication_year=2016&author=Shoushtari%2CAN) 
    
7.  Shoushtari, A. N. et al. A randomized phase 2 study of trametinib with or without GSK2141795 in patients with advanced uveal melanoma. _J. Clin. Oncol._ **34**, 9511 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20randomized%20phase%202%20study%20of%20trametinib%20with%20or%20without%20GSK2141795%20in%20patients%20with%20advanced%20uveal%20melanoma&journal=J.%20Clin.%20Oncol.&volume=34&publication_year=2016&author=Shoushtari%2CAN) 
    
8.  Sacco, J. J. et al. Sunitinib versus dacarbazine as first-line treatment in patients with metastatic uveal melanoma. _J. Clin. Oncol._ **31**, 9031 (2013).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sunitinib%20versus%20dacarbazine%20as%20first-line%20treatment%20in%20patients%20with%20metastatic%20uveal%20melanoma&journal=J.%20Clin.%20Oncol.&volume=31&publication_year=2013&author=Sacco%2CJJ) 
    
9.  Luke, J. J. et al. Randomized phase II trial and tumor mutational spectrum analysis from cabozantinib versus chemotherapy in metastatic uveal melanoma (alliance A091201). _Clin. Cancer Res._ **26**, 804–811 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslyls7fN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31558480)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Randomized%20phase%20II%20trial%20and%20tumor%20mutational%20spectrum%20analysis%20from%20cabozantinib%20versus%20chemotherapy%20in%20metastatic%20uveal%20melanoma%20%28alliance%20A091201%29&journal=Clin.%20Cancer%20Res.&volume=26&pages=804-811&publication_year=2020&author=Luke%2CJJ) 
    
10.  Patel, S. P. et al. A phase II study of gefitinib in patients with metastatic melanoma. _Melanoma Res._ **21**, 357–363 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXos1amtrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21738104)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3132394)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20phase%20II%20study%20of%20gefitinib%20in%20patients%20with%20metastatic%20melanoma&journal=Melanoma%20Res.&volume=21&pages=357-363&publication_year=2011&author=Patel%2CSP) 
    
11.  Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR–Cas9 essentiality screens in cancer cells. _Nat. Genet._ **49**, 1779–1784 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslehtrjI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29083409)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computational%20correction%20of%20copy%20number%20effect%20improves%20specificity%20of%20CRISPR%E2%80%93Cas9%20essentiality%20screens%20in%20cancer%20cells&journal=Nat.%20Genet.&volume=49&pages=1779-1784&publication_year=2017&author=Meyers%2CRM) 
    
12.  Koike-Yusa, H., Li, Y., Tan, E. P., Del Castillo Velasco-Herrera, M. & Yusa, K. Genome-wide recessive genetic screening in mammalian cells with a lentiviral CRISPR-guide RNA library. _Nat. Biotechnol._ **32**, 267–273 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFylsL%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24535568)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20recessive%20genetic%20screening%20in%20mammalian%20cells%20with%20a%20lentiviral%20CRISPR-guide%20RNA%20library&journal=Nat.%20Biotechnol.&volume=32&pages=267-273&publication_year=2014&author=Koike-Yusa%2CH&author=Li%2CY&author=Tan%2CEP&author=Del%20Castillo%20Velasco-Herrera%2CM&author=Yusa%2CK) 
    
13.  Ryan, C. J., Mehta, I., Kebabci, N. & Adams, D. J. Targeting synthetic lethal paralogs in cancer. _Trends Cancer_ **9**, 397–409 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXkslyjsrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36890003)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20synthetic%20lethal%20paralogs%20in%20cancer&journal=Trends%20Cancer&volume=9&pages=397-409&publication_year=2023&author=Ryan%2CCJ&author=Mehta%2CI&author=Kebabci%2CN&author=Adams%2CDJ) 
    
14.  Blomen, V. A. et al. Gene essentiality and synthetic lethality in haploid human cells. _Science_ **350**, 1092–1096 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFamtL3E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26472760)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20essentiality%20and%20synthetic%20lethality%20in%20haploid%20human%20cells&journal=Science&volume=350&pages=1092-1096&publication_year=2015&author=Blomen%2CVA) 
    
15.  Dandage, R. & Landry, C. R. Paralog dependency indirectly affects the robustness of human cells. _Mol. Syst. Biol._ **15**, e8871 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVCksbfI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31556487)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6757259)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Paralog%20dependency%20indirectly%20affects%20the%20robustness%20of%20human%20cells&journal=Mol.%20Syst.%20Biol.&volume=15&publication_year=2019&author=Dandage%2CR&author=Landry%2CCR) 
    
16.  De Kegel, B. & Ryan, C. J. Paralog buffering contributes to the variable essentiality of genes in cancer cell lines. _PLoS Genet._ **15**, e1008466 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31652272)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6834290)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Paralog%20buffering%20contributes%20to%20the%20variable%20essentiality%20of%20genes%20in%20cancer%20cell%20lines&journal=PLoS%20Genet.&volume=15&publication_year=2019&author=Kegel%2CB&author=Ryan%2CCJ) 
    
17.  Wang, T. et al. Identification and characterization of essential genes in the human genome. _Science_ **350**, 1096–1101 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFamtL3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26472758)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4662922)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20and%20characterization%20of%20essential%20genes%20in%20the%20human%20genome&journal=Science&volume=350&pages=1096-1101&publication_year=2015&author=Wang%2CT) 
    
18.  Gu, Z. et al. Role of duplicate genes in genetic robustness against null mutations. _Nature_ **421**, 63–66 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXos1M%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12511954)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20duplicate%20genes%20in%20genetic%20robustness%20against%20null%20mutations&journal=Nature&volume=421&pages=63-66&publication_year=2003&author=Gu%2CZ) 
    
19.  Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. _Genome Biol._ **21**, 262 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFOis7%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33059726)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7558751)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiplex%20enCas12a%20screens%20detect%20functional%20buffering%20among%20paralogs%20otherwise%20masked%20in%20monogenic%20Cas9%20knockout%20screens&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Dede%2CM&author=McLaughlin%2CM&author=Kim%2CE&author=Hart%2CT) 
    
20.  Parrish, P. C. R. et al. Discovery of synthetic lethal and tumor suppressor paralog pairs in the human genome. _Cell Rep._ **36**, 109597 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFels7%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34469736)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8534300)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20of%20synthetic%20lethal%20and%20tumor%20suppressor%20paralog%20pairs%20in%20the%20human%20genome&journal=Cell%20Rep.&volume=36&publication_year=2021&author=Parrish%2CPCR) 
    
21.  Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. _Nat. Commun._ **12**, 1302 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXltlyksbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33637726)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7910459)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Combinatorial%20CRISPR%20screen%20identifies%20fitness%20effects%20of%20gene%20paralogues&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Thompson%2CNA) 
    
22.  Gonatopoulos-Pournatzis, T. et al. Genetic interaction mapping and exon-resolution functional genomics with a hybrid Cas9-Cas12a platform. _Nat. Biotechnol._ **38**, 638–648 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXltFShtb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32249828)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20interaction%20mapping%20and%20exon-resolution%20functional%20genomics%20with%20a%20hybrid%20Cas9-Cas12a%20platform&journal=Nat.%20Biotechnol.&volume=38&pages=638-648&publication_year=2020&author=Gonatopoulos-Pournatzis%2CT) 
    
23.  Ito, T. et al. Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers. _Nat. Genet._ **53**, 1664–1672 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1KhsLnK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34857952)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Paralog%20knockout%20profiling%20identifies%20DUSP4%20and%20DUSP6%20as%20a%20digenic%20dependence%20in%20MAPK%20pathway-driven%20cancers&journal=Nat.%20Genet.&volume=53&pages=1664-1672&publication_year=2021&author=Ito%2CT) 
    
24.  Najgebauer, H. et al. CELLector: genomics-guided selection of cancer in vitro models. _Cell Syst._ **10**, 424–432.e6 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVSjsrzN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32437684)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CELLector%3A%20genomics-guided%20selection%20of%20cancer%20in%20vitro%20models&journal=Cell%20Syst.&volume=10&pages=424-432.e6&publication_year=2020&author=Najgebauer%2CH) 
    
25.  Warren, A. et al. Global computational alignment of tumor and cell line transcriptional profiles. _Nat. Commun._ **12**, 22 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsVKktA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33397959)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7782593)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20computational%20alignment%20of%20tumor%20and%20cell%20line%20transcriptional%20profiles&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Warren%2CA) 
    
26.  Liu, J. et al. An integrated TCGA pan-cencer clinical data resource to drive high-quality survival outcome analytics. _Cell_ **173**, 400–416 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFaiu7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29625055)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6066282)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20TCGA%20pan-cencer%20clinical%20data%20resource%20to%20drive%20high-quality%20survival%20outcome%20analytics&journal=Cell&volume=173&pages=400-416&publication_year=2018&author=Liu%2CJ) 
    
27.  Shen, L. et al. Pan-cancer and single-cell analysis reveal the prognostic value and immune response of NQO1. _Front. Cell Dev. Biol._ **11**, 1174535 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37583897)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10424457)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-cancer%20and%20single-cell%20analysis%20reveal%20the%20prognostic%20value%20and%20immune%20response%20of%20NQO1&journal=Front.%20Cell%20Dev.%20Biol.&volume=11&publication_year=2023&author=Shen%2CL) 
    
28.  Gutierrez-Prat, N. et al. DUSP4 protects BRAF- and NRAS-mutant melanoma from oncogene overdose through modulation of MITF. _Life Sci. Alliance_ **5**, e202101235 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFSks7%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35580987)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9113946)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DUSP4%20protects%20BRAF-%20and%20NRAS-mutant%20melanoma%20from%20oncogene%20overdose%20through%20modulation%20of%20MITF&journal=Life%20Sci.%20Alliance&volume=5&publication_year=2022&author=Gutierrez-Prat%2CN) 
    
29.  Shain, A. H. et al. The genetic evolution of metastatic uveal melanoma. _Nat. Genet._ **51**, 1123–1130 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1yktL7F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31253977)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6632071)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genetic%20evolution%20of%20metastatic%20uveal%20melanoma&journal=Nat.%20Genet.&volume=51&pages=1123-1130&publication_year=2019&author=Shain%2CAH) 
    
30.  De Kegel, B., Quinn, N., Thompson, N. A., Adams, D. J. & Ryan, C. J. Comprehensive prediction of robust synthetic lethality between paralog pairs in cancer cell lines. _Cell Syst._ **12**, 1144–1159.e6 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34529928)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20prediction%20of%20robust%20synthetic%20lethality%20between%20paralog%20pairs%20in%20cancer%20cell%20lines&journal=Cell%20Syst.&volume=12&pages=1144-1159.e6&publication_year=2021&author=Kegel%2CB&author=Quinn%2CN&author=Thompson%2CNA&author=Adams%2CDJ&author=Ryan%2CCJ) 
    
31.  Cho, H., Berger, B. & Peng, J. Compact integration of multi-network topology for functional analysis of genes. _Cell Syst._ **3**, 540–548 e5 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFalsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27889536)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5225290)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Compact%20integration%20of%20multi-network%20topology%20for%20functional%20analysis%20of%20genes&journal=Cell%20Syst.&volume=3&pages=540-548%20e5&publication_year=2016&author=Cho%2CH&author=Berger%2CB&author=Peng%2CJ) 
    
32.  Cancer Genome Atlas Research Network et al. The Cancer Genome Atlas Pan-Cancer analysis project. _Nat. Genet._ **45**, 1113–1120 (2013).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3919969)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Cancer%20Genome%20Atlas%20Pan-Cancer%20analysis%20project&journal=Nat.%20Genet.&volume=45&pages=1113-1120&publication_year=2013) 
    
33.  Barretina, J. et al. The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity. _Nature_ **483**, 603–607 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmtVequ7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22460905)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320027)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Cancer%20Cell%20Line%20Encyclopedia%20enables%20predictive%20modelling%20of%20anticancer%20drug%20sensitivity&journal=Nature&volume=483&pages=603-607&publication_year=2012&author=Barretina%2CJ) 
    
34.  Dwane, L. et al. Project score database: a resource for investigating cancer cell dependencies and prioritizing therapeutic targets. _Nucleic Acids Res._ **49**, D1365–D1372 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFOnsLbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33068406)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Project%20score%20database%3A%20a%20resource%20for%20investigating%20cancer%20cell%20dependencies%20and%20prioritizing%20therapeutic%20targets&journal=Nucleic%20Acids%20Res.&volume=49&pages=D1365-D1372&publication_year=2021&author=Dwane%2CL) 
    
35.  Pacini, C. et al. Integrated cross-study datasets of genetic dependencies in cancer. _Nat. Commun._ **12**, 1661 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmslKis7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33712601)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7955067)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20cross-study%20datasets%20of%20genetic%20dependencies%20in%20cancer&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Pacini%2CC) 
    
36.  Onken, M. D., Worley, L. A., Ehlers, J. P. & Harbour, J. W. Gene expression profiling in uveal melanoma reveals two molecular classes and predicts metastatic death. _Cancer Res._ **64**, 7205–7209 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXosFyntrs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15492234)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5407684)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20expression%20profiling%20in%20uveal%20melanoma%20reveals%20two%20molecular%20classes%20and%20predicts%20metastatic%20death&journal=Cancer%20Res.&volume=64&pages=7205-7209&publication_year=2004&author=Onken%2CMD&author=Worley%2CLA&author=Ehlers%2CJP&author=Harbour%2CJW) 
    
37.  Onken, M. D., Worley, L. A., Tuscan, M. D. & Harbour, J. W. An accurate, clinically feasible multi-gene expression assay for predicting metastasis in uveal melanoma. _J. Mol. Diagn._ **12**, 461–468 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVehurrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20413675)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2893630)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20accurate%2C%20clinically%20feasible%20multi-gene%20expression%20assay%20for%20predicting%20metastasis%20in%20uveal%20melanoma&journal=J.%20Mol.%20Diagn.&volume=12&pages=461-468&publication_year=2010&author=Onken%2CMD&author=Worley%2CLA&author=Tuscan%2CMD&author=Harbour%2CJW) 
    
38.  Guo, J., Liu, H. & Zheng, J. SynLethDB: synthetic lethality database toward discovery of selective and sensitive anticancer drug targets. _Nucleic Acids Res._ **44**, D1011–D1017 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2nsr%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26516187)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SynLethDB%3A%20synthetic%20lethality%20database%20toward%20discovery%20of%20selective%20and%20sensitive%20anticancer%20drug%20targets&journal=Nucleic%20Acids%20Res.&volume=44&pages=D1011-D1017&publication_year=2016&author=Guo%2CJ&author=Liu%2CH&author=Zheng%2CJ) 
    
39.  Robertson, A. G. et al. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. _Cancer Cell_ **32**, 204–220 e15 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlClsbnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28810145)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5619925)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20identifies%20four%20molecular%20and%20clinical%20subsets%20in%20uveal%20melanoma&journal=Cancer%20Cell&volume=32&pages=204-220%20e15&publication_year=2017&author=Robertson%2CAG) 
    
40.  Morgens, D. W. et al. Genome-scale measurement of off-target activity using Cas9 toxicity in high-throughput screens. _Nat. Commun._ **8**, 15178 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXntlyhtb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28474669)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5424143)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-scale%20measurement%20of%20off-target%20activity%20using%20Cas9%20toxicity%20in%20high-throughput%20screens&journal=Nat.%20Commun.&volume=8&publication_year=2017&author=Morgens%2CDW) 
    
41.  Geary, N. Understanding synergy. _Am. J. Physiol. Endocrinol. Metab._ **304**, E237–E253 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXjsFKrs7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23211518)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%20synergy&journal=Am.%20J.%20Physiol.%20Endocrinol.%20Metab.&volume=304&pages=E237-E253&publication_year=2013&author=Geary%2CN) 
    
42.  Schick, S. et al. Systematic characterization of BAF mutations provides insights into intracomplex synthetic lethalities in human cancers. _Nat. Genet._ **51**, 1399–1410 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1WjurjJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31427792)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952272)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20characterization%20of%20BAF%20mutations%20provides%20insights%20into%20intracomplex%20synthetic%20lethalities%20in%20human%20cancers&journal=Nat.%20Genet.&volume=51&pages=1399-1410&publication_year=2019&author=Schick%2CS) 
    
43.  Sasaki, M. & Ogiwara, H. Synthetic lethal therapy based on targeting the vulnerability of SWI/SNF chromatin remodeling complex-deficient cancers. _Cancer Sci._ **111**, 774–782 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFSrtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31955490)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7060479)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthetic%20lethal%20therapy%20based%20on%20targeting%20the%20vulnerability%20of%20SWI%2FSNF%20chromatin%20remodeling%20complex-deficient%20cancers&journal=Cancer%20Sci.&volume=111&pages=774-782&publication_year=2020&author=Sasaki%2CM&author=Ogiwara%2CH) 
    
44.  Shields, J. A. et al. VRK1 is a synthetic-lethal target in VRK2-deficient glioblastoma. _Cancer Res._ **82**, 4044–4057 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFagt7bF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36069976)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9627132)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=VRK1%20is%20a%20synthetic-lethal%20target%20in%20VRK2-deficient%20glioblastoma&journal=Cancer%20Res.&volume=82&pages=4044-4057&publication_year=2022&author=Shields%2CJA) 
    
45.  Hart, T. et al. Evaluation and design of genome-wide CRISPR/SpCas9 knockout screens. _G3 (Bethesda)_ **7**, 2719–2727 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVKhtrrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28655737)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluation%20and%20design%20of%20genome-wide%20CRISPR%2FSpCas9%20knockout%20screens&journal=G3%20%28Bethesda%29&volume=7&pages=2719-2727&publication_year=2017&author=Hart%2CT) 
    
46.  Malone, C. F. et al. Selective modulation of a pan-essential protein as a therapeutic strategy in cancer. _Cancer Discov._ **11**, 2282–2299 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFCksLzO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33883167)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8419012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Selective%20modulation%20of%20a%20pan-essential%20protein%20as%20a%20therapeutic%20strategy%20in%20cancer&journal=Cancer%20Discov.&volume=11&pages=2282-2299&publication_year=2021&author=Malone%2CCF) 
    
47.  Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. _Nature_ **568**, 511–516 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVSgtrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30971826)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prioritization%20of%20cancer%20therapeutic%20targets%20using%20CRISPR%E2%80%93Cas9%20screens&journal=Nature&volume=568&pages=511-516&publication_year=2019&author=Behan%2CFM) 
    
48.  Aguirre, A. J. et al. Genomic copy number dictates a gene-independent cell response to CRISPR/Cas9 targeting. _Cancer Discov._ **6**, 914–929 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht12mur7O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27260156)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4972686)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20copy%20number%20dictates%20a%20gene-independent%20cell%20response%20to%20CRISPR%2FCas9%20targeting&journal=Cancer%20Discov.&volume=6&pages=914-929&publication_year=2016&author=Aguirre%2CAJ) 
    
49.  Iorio, F. et al. Unsupervised correction of gene-independent cell responses to CRISPR–Cas9 targeting. _BMC Genomics_ **19**, 604 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30103702)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6088408)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unsupervised%20correction%20of%20gene-independent%20cell%20responses%20to%20CRISPR%E2%80%93Cas9%20targeting&journal=BMC%20Genomics&volume=19&publication_year=2018&author=Iorio%2CF) 
    
50.  Vinceti, A. et al. An interactive web application for processing, correcting, and visualizing genome-wide pooled CRISPR–Cas9 screens. _Cell Rep. Methods_ **3**, 100373 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXkslKntw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36814834)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9939378)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20interactive%20web%20application%20for%20processing%2C%20correcting%2C%20and%20visualizing%20genome-wide%20pooled%20CRISPR%E2%80%93Cas9%20screens&journal=Cell%20Rep.%20Methods&volume=3&publication_year=2023&author=Vinceti%2CA) 
    
51.  Chang, L., Ruiz, P., Ito, T. & Sellers, W. R. Targeting pan-essential genes in cancer: challenges and opportunities. _Cancer Cell_ **39**, 466–479 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1Oms7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33450197)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8157671)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20pan-essential%20genes%20in%20cancer%3A%20challenges%20and%20opportunities&journal=Cancer%20Cell&volume=39&pages=466-479&publication_year=2021&author=Chang%2CL&author=Ruiz%2CP&author=Ito%2CT&author=Sellers%2CWR) 
    
52.  Tsherniak, A. et al. Defining a cancer dependency map. _Cell_ **170**, 564–576 e16 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1GisLjP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28753430)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5667678)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defining%20a%20cancer%20dependency%20map&journal=Cell&volume=170&pages=564-576%20e16&publication_year=2017&author=Tsherniak%2CA) 
    
53.  Gallegos, L. L. & Newton, A. C. Spatiotemporal dynamics of lipid signaling: protein kinase C as a paradigm. _IUBMB Life_ **60**, 782–789 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsFSnsbjF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18720411)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2718795)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatiotemporal%20dynamics%20of%20lipid%20signaling%3A%20protein%20kinase%20C%20as%20a%20paradigm&journal=IUBMB%20Life&volume=60&pages=782-789&publication_year=2008&author=Gallegos%2CLL&author=Newton%2CAC) 
    
54.  Thillaiappan, N. B., Chakraborty, P., Hasan, G. & Taylor, C. W. IP(3) receptors and Ca2+ entry. _Biochim. Biophys. Acta Mol. Cell. Res._ **1866**, 1092–1100 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1ymtL3I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30448464)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IP%283%29%20receptors%20and%20Ca2%2B%20entry&journal=Biochim.%20Biophys.%20Acta%20Mol.%20Cell.%20Res.&volume=1866&pages=1092-1100&publication_year=2019&author=Thillaiappan%2CNB&author=Chakraborty%2CP&author=Hasan%2CG&author=Taylor%2CCW) 
    
55.  Shoushtari, A. N. & Carvajal, R. D. GNAQ and GNA11 mutations in uveal melanoma. _Melanoma Res._ **24**, 525–534 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVelurvK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25304237)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GNAQ%20and%20GNA11%20mutations%20in%20uveal%20melanoma&journal=Melanoma%20Res.&volume=24&pages=525-534&publication_year=2014&author=Shoushtari%2CAN&author=Carvajal%2CRD) 
    
56.  Köferle, A. et al. Interrogation of cancer gene dependencies reveals paralog interactions of autosome and sex chromosome-encoded genes. _Cell Rep._ **39**, 110636 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35417719)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interrogation%20of%20cancer%20gene%20dependencies%20reveals%20paralog%20interactions%20of%20autosome%20and%20sex%20chromosome-encoded%20genes&journal=Cell%20Rep.&volume=39&publication_year=2022&author=K%C3%B6ferle%2CA) 
    
57.  Lin, H. K. et al. Functional buffering via cell-specific gene expression promotes tissue homeostasis and cancer robustness. _Sci. Rep._ **12**, 2974 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XkvFSmsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35194081)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8863889)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20buffering%20via%20cell-specific%20gene%20expression%20promotes%20tissue%20homeostasis%20and%20cancer%20robustness&journal=Sci.%20Rep.&volume=12&publication_year=2022&author=Lin%2CHK) 
    
58.  Blunsom, N. J. & Cockcroft, S. CDP-diacylglycerol synthases (CDS): gateway to phosphatidylinositol and cardiolipin synthesis. _Front. Cell Dev. Biol._ **8**, 63 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32117988)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7018664)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CDP-diacylglycerol%20synthases%20%28CDS%29%3A%20gateway%20to%20phosphatidylinositol%20and%20cardiolipin%20synthesis&journal=Front.%20Cell%20Dev.%20Biol.&volume=8&publication_year=2020&author=Blunsom%2CNJ&author=Cockcroft%2CS) 
    
59.  Balla, T. Phosphoinositides: tiny lipids with giant impact on cell regulation. _Physiol. Rev._ **93**, 1019–1137 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVekurfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23899561)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3962547)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phosphoinositides%3A%20tiny%20lipids%20with%20giant%20impact%20on%20cell%20regulation&journal=Physiol.%20Rev.&volume=93&pages=1019-1137&publication_year=2013&author=Balla%2CT) 
    
60.  Qi, Y. et al. CDP-diacylglycerol synthases regulate the growth of lipid droplets and adipocyte development. _J. Lipid Res._ **57**, 767–780 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XmvFOru7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26946540)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4847625)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CDP-diacylglycerol%20synthases%20regulate%20the%20growth%20of%20lipid%20droplets%20and%20adipocyte%20development&journal=J.%20Lipid%20Res.&volume=57&pages=767-780&publication_year=2016&author=Qi%2CY) 
    
61.  Goldman, M. J. et al. Visualizing and interpreting cancer genomics data via the Xena platform. _Nat. Biotechnol._ **38**, 675–678 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpvFGhtr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32444850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7386072)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Visualizing%20and%20interpreting%20cancer%20genomics%20data%20via%20the%20Xena%20platform&journal=Nat.%20Biotechnol.&volume=38&pages=675-678&publication_year=2020&author=Goldman%2CMJ) 
    
62.  Xu, Y. et al. CDP-DAG synthase 1 and 2 regulate lipid droplet growth through distinct mechanisms. _J. Biol. Chem._ **294**, 16740–16755 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFemt7fF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31548309)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6851299)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CDP-DAG%20synthase%201%20and%202%20regulate%20lipid%20droplet%20growth%20through%20distinct%20mechanisms&journal=J.%20Biol.%20Chem.&volume=294&pages=16740-16755&publication_year=2019&author=Xu%2CY) 
    
63.  Arnoldus, T. et al. Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers. _Nat. Genet._ [https://doi.org/10.1038/s41588-025-02221-2](https://doi.org/10.1038/s41588-025-02221-2) (2025).
    
64.  Uhlen, M. et al. Proteomics. Tissue-based map of the human proteome. _Science_ **347**, 1260419 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25613900)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteomics.%20Tissue-based%20map%20of%20the%20human%20proteome&journal=Science&volume=347&publication_year=2015&author=Uhlen%2CM) 
    
65.  Xu, J. et al. Hepatic CDP-diacylglycerol synthase 2 deficiency causes mitochondrial dysfunction and promotes rapid progression of NASH and fibrosis. _Sci. Bull. (Beijing)_ **67**, 299–314 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xmt1Wht74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36546079)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hepatic%20CDP-diacylglycerol%20synthase%202%20deficiency%20causes%20mitochondrial%20dysfunction%20and%20promotes%20rapid%20progression%20of%20NASH%20and%20fibrosis&journal=Sci.%20Bull.%20%28Beijing%29&volume=67&pages=299-314&publication_year=2022&author=Xu%2CJ) 
    
66.  Stratman, A. N. et al. Anti-angiogenic effects of VEGF stimulation on endothelium deficient in phosphoinositide recycling. _Nat. Commun._ **11**, 1204 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlt1SgsLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32139674)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7058007)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Anti-angiogenic%20effects%20of%20VEGF%20stimulation%20on%20endothelium%20deficient%20in%20phosphoinositide%20recycling&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Stratman%2CAN) 
    
67.  Collins, D. M. et al. CDS2 expression regulates de novo phosphatidic acid synthesis. _Biochem. J._ **481**, 1449–1473 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXisFejsLbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39312194)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CDS2%20expression%20regulates%20de%20novo%20phosphatidic%20acid%20synthesis&journal=Biochem.%20J.&volume=481&pages=1449-1473&publication_year=2024&author=Collins%2CDM) 
    
68.  Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. _Nature_ **596**, 583–589 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVaktrrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34265844)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&journal=Nature&volume=596&pages=583-589&publication_year=2021&author=Jumper%2CJ) 
    
69.  Ghandi, M. et al. Next-generation characterization of the Cancer Cell Line Encyclopedia. _Nature_ **569**, 503–508 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXptlOhtLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31068700)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6697103)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Next-generation%20characterization%20of%20the%20Cancer%20Cell%20Line%20Encyclopedia&journal=Nature&volume=569&pages=503-508&publication_year=2019&author=Ghandi%2CM) 
    
70.  Talevich, E., Shain, A. H., Botton, T. & Bastian, B. C. CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing. _PLoS Comput. Biol._ **12**, e1004873 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27100738)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4839673)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CNVkit%3A%20genome-wide%20copy%20number%20detection%20and%20visualization%20from%20targeted%20DNA%20sequencing&journal=PLoS%20Comput.%20Biol.&volume=12&publication_year=2016&author=Talevich%2CE&author=Shain%2CAH&author=Botton%2CT&author=Bastian%2CBC) 
    
71.  Chen, M. M. et al. TCPA v3.0: an integrative platform to explore the pan-cancer analysis of functional proteomic data. _Mol. Cell Proteom._ **18**, S15–S25 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVeiu7vP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TCPA%20v3.0%3A%20an%20integrative%20platform%20to%20explore%20the%20pan-cancer%20analysis%20of%20functional%20proteomic%20data&journal=Mol.%20Cell%20Proteom.&volume=18&pages=S15-S25&publication_year=2019&author=Chen%2CMM) 
    
72.  Tzelepis, K. et al. A CRISPR dropout screen identifies genetic vulnerabilities and therapeutic targets in acute myeloid leukemia. _Cell Rep._ **17**, 1193–1205 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhslSntLjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27760321)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5081405)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20CRISPR%20dropout%20screen%20identifies%20genetic%20vulnerabilities%20and%20therapeutic%20targets%20in%20acute%20myeloid%20leukemia&journal=Cell%20Rep.&volume=17&pages=1193-1205&publication_year=2016&author=Tzelepis%2CK) 
    
73.  Tong, A. H. et al. Global mapping of the yeast genetic interaction network. _Science_ **303**, 808–813 (2004).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXovVKntg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14764870)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20mapping%20of%20the%20yeast%20genetic%20interaction%20network&journal=Science&volume=303&pages=808-813&publication_year=2004&author=Tong%2CAH) 
    
74.  Garcia-Alonso, L. et al. Transcription factor activities enhance markers of drug sensitivity in cancer. _Cancer Res._ **78**, 769–780 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVWnsbY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29229604)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcription%20factor%20activities%20enhance%20markers%20of%20drug%20sensitivity%20in%20cancer&journal=Cancer%20Res.&volume=78&pages=769-780&publication_year=2018&author=Garcia-Alonso%2CL) 
    
75.  Goncalves, E. et al. Drug mechanism-of-action discovery through the integration of pharmacological and CRISPR screens. _Mol. Syst. Biol._ **16**, e9405 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWqurrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32627965)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7336273)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Drug%20mechanism-of-action%20discovery%20through%20the%20integration%20of%20pharmacological%20and%20CRISPR%20screens&journal=Mol.%20Syst.%20Biol.&volume=16&publication_year=2020&author=Goncalves%2CE) 
    
76.  Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR–Cas9. _Nat. Biotechnol._ **34**, 184–191 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xps1GisQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26780180)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4744125)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimized%20sgRNA%20design%20to%20maximize%20activity%20and%20minimize%20off-target%20effects%20of%20CRISPR%E2%80%93Cas9&journal=Nat.%20Biotechnol.&volume=34&pages=184-191&publication_year=2016&author=Doench%2CJG) 
    
77.  Hart, T. et al. High-resolution CRISPR screens reveal fitness genes and genotype-specific cancer liabilities. _Cell_ **163**, 1515–1526 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFagur7I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26627737)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-resolution%20CRISPR%20screens%20reveal%20fitness%20genes%20and%20genotype-specific%20cancer%20liabilities&journal=Cell&volume=163&pages=1515-1526&publication_year=2015&author=Hart%2CT) 
    
78.  Park, J. S. jpark27/CDS1-2: v1.0 (v1.0). _Zenodo_ [https://doi.org/10.5281/zenodo.15375418](https://doi.org/10.5281/zenodo.15375418) (2025).
    
79.  Vidigal, J. A. & Ventura, A. Rapid and efficient one-step generation of paired gRNA CRISPR–Cas9 libraries. _Nat. Commun._ **6**, 8083 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsVKhur%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26278926)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20and%20efficient%20one-step%20generation%20of%20paired%20gRNA%20CRISPR%E2%80%93Cas9%20libraries&journal=Nat.%20Commun.&volume=6&publication_year=2015&author=Vidigal%2CJA&author=Ventura%2CA) 
    
80.  Kim, E. & Hart, T. Improved analysis of CRISPR fitness screens and reduced off-target effects with the _BAGEL2_ gene essentiality classifier. _Genome Med._ **13**, 2 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjtVSgtbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33407829)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7789424)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improved%20analysis%20of%20CRISPR%20fitness%20screens%20and%20reduced%20off-target%20effects%20with%20the%20BAGEL2%20gene%20essentiality%20classifier&journal=Genome%20Med.&volume=13&publication_year=2021&author=Kim%2CE&author=Hart%2CT) 
    
81.  Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. _Genome Biol._ **15**, 554 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25476604)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAGeCK%20enables%20robust%20identification%20of%20essential%20genes%20from%20genome-scale%20CRISPR%2FCas9%20knockout%20screens&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Li%2CW) 
    
82.  Dempster, J. M. et al. Chronos: a cell population dynamics model of CRISPR experiments that improves inference of gene fitness effects. _Genome Biol._ **22**, 343 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34930405)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8686573)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chronos%3A%20a%20cell%20population%20dynamics%20model%20of%20CRISPR%20experiments%20that%20improves%20inference%20of%20gene%20fitness%20effects&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Dempster%2CJM) 
    
83.  Ruiz-Moreno, C. et al. Harmonized single-cell landscape, intercellular crosstalk and tumor architecture of glioblastoma. Preprint at _bioRxiv_ [https://doi.org/10.1101/2022.08.27.505439](https://doi.org/10.1101/2022.08.27.505439) (2022).
    
84.  Bakhoum, M. F. et al. Loss of polycomb repressive complex 1 activity and chromosomal instability drive uveal melanoma progression. _Nat. Commun._ **12**, 5402 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitVCnu77E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34518527)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8438051)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20polycomb%20repressive%20complex%201%20activity%20and%20chromosomal%20instability%20drive%20uveal%20melanoma%20progression&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Bakhoum%2CMF) 
    
85.  Lin, W. et al. Intra- and intertumoral heterogeneity of liver metastases in a patient with uveal melanoma revealed by single-cell RNA sequencing. _Cold Spring Harb. Mol. Case Stud._ **7**, a006111 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislCgtr%2FI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34470851)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8559622)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intra-%20and%20intertumoral%20heterogeneity%20of%20liver%20metastases%20in%20a%20patient%20with%20uveal%20melanoma%20revealed%20by%20single-cell%20RNA%20sequencing&journal=Cold%20Spring%20Harb.%20Mol.%20Case%20Stud.&volume=7&publication_year=2021&author=Lin%2CW) 
    
86.  Ramachandran, P. et al. Resolving the fibrotic niche of human liver cirrhosis at single-cell level. _Nature_ **575**, 512–518 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVags7bK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31597160)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6876711)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Resolving%20the%20fibrotic%20niche%20of%20human%20liver%20cirrhosis%20at%20single-cell%20level&journal=Nature&volume=575&pages=512-518&publication_year=2019&author=Ramachandran%2CP) 
    
87.  Gautam, P. et al. Multi-species single-cell transcriptomic analysis of ocular compartment regulons. _Nat. Commun._ **12**, 5675 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFamu7vM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34584087)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8478974)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-species%20single-cell%20transcriptomic%20analysis%20of%20ocular%20compartment%20regulons&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Gautam%2CP) 
    
88.  Durante, M. A. et al. Single-cell analysis reveals new evolutionary complexity in uveal melanoma. _Nat. Commun._ **11**, 496 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXksFantL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31980621)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6981133)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20analysis%20reveals%20new%20evolutionary%20complexity%20in%20uveal%20melanoma&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Durante%2CMA) 
    
89.  Pandiani, C. et al. Single-cell RNA sequencing reveals intratumoral heterogeneity in primary uveal melanomas and identifies HES6 as a driver of the metastatic disease. _Cell Death Differ._ **28**, 1990–2000 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmsVCitr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33462406)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8185008)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20RNA%20sequencing%20reveals%20intratumoral%20heterogeneity%20in%20primary%20uveal%20melanomas%20and%20identifies%20HES6%20as%20a%20driver%20of%20the%20metastatic%20disease&journal=Cell%20Death%20Differ.&volume=28&pages=1990-2000&publication_year=2021&author=Pandiani%2CC) 
    
90.  Xu, C. et al. Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models. _Mol. Syst. Biol._ **17**, e9620 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33491336)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7829634)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Probabilistic%20harmonization%20and%20annotation%20of%20single-cell%20transcriptomics%20data%20with%20deep%20generative%20models&journal=Mol.%20Syst.%20Biol.&volume=17&publication_year=2021&author=Xu%2CC) 
    
91.  Lotfollahi, M. et al. Mapping single-cell data to reference atlases by transfer learning. _Nat. Biotechnol._ **40**, 121–130 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFWqtb3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34462589)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20single-cell%20data%20to%20reference%20atlases%20by%20transfer%20learning&journal=Nat.%20Biotechnol.&volume=40&pages=121-130&publication_year=2022&author=Lotfollahi%2CM) 
    
92.  Barkley, D. et al. Cancer cell states recur across tumor types and form specific interactions with the tumor microenvironment. _Nat. Genet._ **54**, 1192–1201 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitV2lt7bP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35931863)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9886402)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cancer%20cell%20states%20recur%20across%20tumor%20types%20and%20form%20specific%20interactions%20with%20the%20tumor%20microenvironment&journal=Nat.%20Genet.&volume=54&pages=1192-1201&publication_year=2022&author=Barkley%2CD) 
    
93.  Quan, F. et al. Annotation of cell types (ACT): a convenient web server for cell type annotation. _Genome Med._ **15**, 91 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitlCit7rM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37924118)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10623726)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Annotation%20of%20cell%20types%20%28ACT%29%3A%20a%20convenient%20web%20server%20for%20cell%20type%20annotation&journal=Genome%20Med.&volume=15&publication_year=2023&author=Quan%2CF) 
    
94.  Andreatta, M. & Carmona, S. J. UCell: robust and scalable single-cell gene signature scoring. _Comput. Struct. Biotechnol. J._ **19**, 3796–3798 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisF2gsr%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34285779)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8271111)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UCell%3A%20robust%20and%20scalable%20single-cell%20gene%20signature%20scoring&journal=Comput.%20Struct.%20Biotechnol.%20J.&volume=19&pages=3796-3798&publication_year=2021&author=Andreatta%2CM&author=Carmona%2CSJ) 
    
95.  Gao, R. et al. Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes. _Nat. Biotechnol._ **39**, 599–608 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFOitb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33462507)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8122019)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Delineating%20copy%20number%20and%20clonal%20substructure%20in%20human%20tumors%20from%20single-cell%20transcriptomes&journal=Nat.%20Biotechnol.&volume=39&pages=599-608&publication_year=2021&author=Gao%2CR) 
    
96.  Traag, V. A., Waltman, L. & van Eck, N. J. From Louvain to Leiden: guaranteeing well-connected communities. _Sci. Rep._ **9**, 5233 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB3cbosVGisQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30914743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6435756)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=From%20Louvain%20to%20Leiden%3A%20guaranteeing%20well-connected%20communities&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Traag%2CVA&author=Waltman%2CL&author=Eck%2CNJ) 
    
97.  Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. _Nat. Biotechnol._ **37**, 38–44 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2isbrN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dimensionality%20reduction%20for%20visualizing%20single-cell%20data%20using%20UMAP&journal=Nat.%20Biotechnol.&volume=37&pages=38-44&publication_year=2019&author=Becht%2CE) 
    
98.  Wu, H. et al. Mapping the single-cell transcriptomic response of murine diabetic kidney disease to therapies. _Cell Metab._ **34**, 1064–1078.e6 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFansLnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35709763)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9262852)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20the%20single-cell%20transcriptomic%20response%20of%20murine%20diabetic%20kidney%20disease%20to%20therapies&journal=Cell%20Metab.&volume=34&pages=1064-1078.e6&publication_year=2022&author=Wu%2CH) 
    
99.  Goncalves, E. et al. Minimal genome-wide human CRISPR–Cas9 library. _Genome Biol._ **22**, 40 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXls1Kmsro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33478580)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7818936)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Minimal%20genome-wide%20human%20CRISPR%E2%80%93Cas9%20library&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Goncalves%2CE) 
    
100.  Sialana, F. J. et al. SimPLIT: simplified sample preparation for large-scale isobaric tagging proteomics. _J. Proteome Res._ **21**, 1842–1856 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVWjtrbL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35848491)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9361352)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SimPLIT%3A%20simplified%20sample%20preparation%20for%20large-scale%20isobaric%20tagging%20proteomics&journal=J.%20Proteome%20Res.&volume=21&pages=1842-1856&publication_year=2022&author=Sialana%2CFJ) 
    
101.  Billington, J., Del Castillo Velasco-Herrera, M. & Offord, V. team113sanger/targeting-the-CDS1-2-axis-as-a-therapeutic-strategy-in-uveal-melanoma-and-pan-cancer: post review fixes (v1.0.1). _Zenodo_ [https://doi.org/10.5281/zenodo.15124575](https://doi.org/10.5281/zenodo.15124575) (2025).
    
102.  Blunsom, N. J., Gomez-Espinosa, E., Ashlin, T. G. & Cockcroft, S. Mitochondrial CDP-diacylglycerol synthase activity is due to the peripheral protein, TAMM41 and not due to the integral membrane protein, CDP-diacylglycerol synthase 1. _Biochim. Biophys. Acta Mol. Cell. Biol. Lipids_ **1863**, 284–298 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVaku7vJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29253589)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mitochondrial%20CDP-diacylglycerol%20synthase%20activity%20is%20due%20to%20the%20peripheral%20protein%2C%20TAMM41%20and%20not%20due%20to%20the%20integral%20membrane%20protein%2C%20CDP-diacylglycerol%20synthase%201&journal=Biochim.%20Biophys.%20Acta%20Mol.%20Cell.%20Biol.%20Lipids&volume=1863&pages=284-298&publication_year=2018&author=Blunsom%2CNJ&author=Gomez-Espinosa%2CE&author=Ashlin%2CTG&author=Cockcroft%2CS) 
    
103.  Cox, J. & Mann, M. 1D and 2D annotation enrichment: a statistical method integrating quantitative proteomics with complementary high-throughput data. _BMC Bioinformatics_ **13**, S12 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVyis7bL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23176165)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3489530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=1D%20and%202D%20annotation%20enrichment%3A%20a%20statistical%20method%20integrating%20quantitative%20proteomics%20with%20complementary%20high-throughput%20data&journal=BMC%20Bioinformatics&volume=13&publication_year=2012&author=Cox%2CJ&author=Mann%2CM) 
    
104.  Arora, C. et al. The landscape of cancer-rewired GPCR signaling axes. _Cell Genom._ **4**, 100557 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtVagsr3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38723607)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11099383)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20landscape%20of%20cancer-rewired%20GPCR%20signaling%20axes&journal=Cell%20Genom.&volume=4&publication_year=2024&author=Arora%2CC) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02222-1?format=refman&flavour=references)

Acknowledgements
----------------

P.Y.C. was funded by a Wellcome Trust Clinician-Scientist PhD Fellowship. D.J.A. is funded by the Wellcome Trust (220540/Z/20/A), MRC (MR/V000292/1) and CR-UK (EDDPGM-Nov22/100004). We would like to thank the Sanger FACS facility, sequencing platform and cellular genetics team for making this project possible. E.G.’s work is partially supported by Portuguese national funds through FCT, Fundação para a Ciência e a Tecnologia, under projects UIDB/50021/2020 ([https://doi.org/10.54499/UIDB/50021/2020](https://doi.org/10.54499/UIDB/50021/2020)), SARC-RON-AI (2024.07252.IACDC) and SYNTHESIS (LISBOA2030-FEDER-00868200). P.T.H., L.S. and D.B. were funded by the BBSRC (BB/Y006925/1 and BB/T002530/1). A.V. and C.J.R. were funded by Research Ireland (grants 20/FFP-P/8641 and 18/CRT/6214).

Author information
------------------

Author notes

1.  These authors contributed equally: Pui Ying Chan, Diana Alexander, Ishan Mehta, Larissa Satiko Alcantara Sekimoto Matsuyama.
    

### Authors and Affiliations

1.  Wellcome Sanger Institute, Hinxton, UK
    
    Pui Ying Chan, Diana Alexander, Ishan Mehta, Larissa Satiko Alcantara Sekimoto Matsuyama, Victoria Harle, Rebeca Olvera-León, Jun Sung Park, Fernanda G. Arriaga-González, Louise van der Weyden, Saamin Cheema, Vivek Iyer, Victoria Offord, Kim Wong, Gabriel Balmus, Nicola A. Thompson, Martin Del Castillo Velasco-Herrera & David J. Adams
    
2.  Babraham Institute, Cambridge, UK
    
    David Barneda, Phillip T. Hawkins & Len Stephens
    
3.  The Institute of Cancer Research, London, UK
    
    Zuza Kozik & Jyoti Choudhary
    
4.  Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK
    
    Michael Woods & Gabriel Balmus
    
5.  UK Dementia Research Institute, Cambridge, UK
    
    Michael Woods & Gabriel Balmus
    
6.  Department of Molecular Neuroscience, Transylvanian Institute of Neuroscience, Cluj-Napoca, Romania
    
    Gabriel Balmus
    
7.  Human Technopole, Milan, Italy
    
    Alessandro Vinceti & Francesco Iorio
    
8.  Division of Molecular Oncology and Immunology, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Lodewyk Wessels & Joris van de Haar
    
9.  Division of Molecular Carcinogenesis, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Lodewyk Wessels & Joris van de Haar
    
10.  Oncode Institute, Utrecht, the Netherlands
    
    Joris van de Haar
    
11.  Instituto Superior Técnico (IST), Universidade de Lisboa, Lisbon, Portugal
    
    Emanuel Gonçalves
    
12.  INESC-ID, Lisbon, Portugal
    
    Emanuel Gonçalves
    
13.  Cancer Data Science Laboratory, Center for Cancer Research, National Cancer Institute, Bethesda, MD, USA
    
    Sanju Sinha & Sumeet Patiyal
    
14.  Laboratorio Internacional de Investigación sobre el Genoma Humano, Universidad Nacional Autónoma de México, Campus Juriquilla, Querétaro, Mexico
    
    Martha Estefania Vázquez-Cruz
    
15.  Laboratorio di Biologia Bio@SNS, Scuola Normale Superiore, Pisa, Italy
    
    Luisa Bisceglia
    
16.  Conway Institute and School of Computer Science, University College Dublin, Dublin, Ireland
    
    Francesco Raimondi, Anjan Venkatesh & Colm J. Ryan
    
17.  School of Medicine, University College Dublin, Dublin, Ireland
    
    Colm J. Ryan
    

Authors

1.  Pui Ying Chan
    
    [View author publications](https://www.nature.com/search?author=Pui%20Ying%20Chan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Pui%20Ying%20Chan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Pui%20Ying%20Chan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Diana Alexander
    
    [View author publications](https://www.nature.com/search?author=Diana%20Alexander)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Diana%20Alexander) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Diana%20Alexander%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Ishan Mehta
    
    [View author publications](https://www.nature.com/search?author=Ishan%20Mehta)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ishan%20Mehta) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ishan%20Mehta%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Larissa Satiko Alcantara Sekimoto Matsuyama
    
    [View author publications](https://www.nature.com/search?author=Larissa%20Satiko%20Alcantara%20Sekimoto%20Matsuyama)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Larissa%20Satiko%20Alcantara%20Sekimoto%20Matsuyama) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Larissa%20Satiko%20Alcantara%20Sekimoto%20Matsuyama%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Victoria Harle
    
    [View author publications](https://www.nature.com/search?author=Victoria%20Harle)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Victoria%20Harle) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Victoria%20Harle%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Rebeca Olvera-León
    
    [View author publications](https://www.nature.com/search?author=Rebeca%20Olvera-Le%C3%B3n)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rebeca%20Olvera-Le%C3%B3n) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rebeca%20Olvera-Le%C3%B3n%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Jun Sung Park
    
    [View author publications](https://www.nature.com/search?author=Jun%20Sung%20Park)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jun%20Sung%20Park) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jun%20Sung%20Park%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Fernanda G. Arriaga-González
    
    [View author publications](https://www.nature.com/search?author=Fernanda%20G.%20Arriaga-Gonz%C3%A1lez)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Fernanda%20G.%20Arriaga-Gonz%C3%A1lez) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Fernanda%20G.%20Arriaga-Gonz%C3%A1lez%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Louise van der Weyden
    
    [View author publications](https://www.nature.com/search?author=Louise%20van%20der%20Weyden)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Louise%20van%20der%20Weyden) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Louise%20van%20der%20Weyden%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Saamin Cheema
    
    [View author publications](https://www.nature.com/search?author=Saamin%20Cheema)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Saamin%20Cheema) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Saamin%20Cheema%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Vivek Iyer
    
    [View author publications](https://www.nature.com/search?author=Vivek%20Iyer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Vivek%20Iyer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Vivek%20Iyer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Victoria Offord
    
    [View author publications](https://www.nature.com/search?author=Victoria%20Offord)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Victoria%20Offord) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Victoria%20Offord%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  David Barneda
    
    [View author publications](https://www.nature.com/search?author=David%20Barneda)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=David%20Barneda) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22David%20Barneda%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Phillip T. Hawkins
    
    [View author publications](https://www.nature.com/search?author=Phillip%20T.%20Hawkins)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Phillip%20T.%20Hawkins) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Phillip%20T.%20Hawkins%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Len Stephens
    
    [View author publications](https://www.nature.com/search?author=Len%20Stephens)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Len%20Stephens) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Len%20Stephens%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Zuza Kozik
    
    [View author publications](https://www.nature.com/search?author=Zuza%20Kozik)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zuza%20Kozik) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zuza%20Kozik%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Michael Woods
    
    [View author publications](https://www.nature.com/search?author=Michael%20Woods)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Michael%20Woods) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Michael%20Woods%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Kim Wong
    
    [View author publications](https://www.nature.com/search?author=Kim%20Wong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kim%20Wong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kim%20Wong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Gabriel Balmus
    
    [View author publications](https://www.nature.com/search?author=Gabriel%20Balmus)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gabriel%20Balmus) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gabriel%20Balmus%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Alessandro Vinceti
    
    [View author publications](https://www.nature.com/search?author=Alessandro%20Vinceti)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alessandro%20Vinceti) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alessandro%20Vinceti%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Nicola A. Thompson
    
    [View author publications](https://www.nature.com/search?author=Nicola%20A.%20Thompson)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nicola%20A.%20Thompson) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nicola%20A.%20Thompson%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Martin Del Castillo Velasco-Herrera
    
    [View author publications](https://www.nature.com/search?author=Martin%20Del%20Castillo%20Velasco-Herrera)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Martin%20Del%20Castillo%20Velasco-Herrera) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Martin%20Del%20Castillo%20Velasco-Herrera%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Lodewyk Wessels
    
    [View author publications](https://www.nature.com/search?author=Lodewyk%20Wessels)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lodewyk%20Wessels) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lodewyk%20Wessels%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Joris van de Haar
    
    [View author publications](https://www.nature.com/search?author=Joris%20van%20de%20Haar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Joris%20van%20de%20Haar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Joris%20van%20de%20Haar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Emanuel Gonçalves
    
    [View author publications](https://www.nature.com/search?author=Emanuel%20Gon%C3%A7alves)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Emanuel%20Gon%C3%A7alves) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Emanuel%20Gon%C3%A7alves%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Sanju Sinha
    
    [View author publications](https://www.nature.com/search?author=Sanju%20Sinha)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sanju%20Sinha) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sanju%20Sinha%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  Martha Estefania Vázquez-Cruz
    
    [View author publications](https://www.nature.com/search?author=Martha%20Estefania%20V%C3%A1zquez-Cruz)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Martha%20Estefania%20V%C3%A1zquez-Cruz) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Martha%20Estefania%20V%C3%A1zquez-Cruz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Luisa Bisceglia
    
    [View author publications](https://www.nature.com/search?author=Luisa%20Bisceglia)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Luisa%20Bisceglia) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Luisa%20Bisceglia%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Francesco Raimondi
    
    [View author publications](https://www.nature.com/search?author=Francesco%20Raimondi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Francesco%20Raimondi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Francesco%20Raimondi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  Jyoti Choudhary
    
    [View author publications](https://www.nature.com/search?author=Jyoti%20Choudhary)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jyoti%20Choudhary) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jyoti%20Choudhary%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  Sumeet Patiyal
    
    [View author publications](https://www.nature.com/search?author=Sumeet%20Patiyal)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sumeet%20Patiyal) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sumeet%20Patiyal%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
32.  Anjan Venkatesh
    
    [View author publications](https://www.nature.com/search?author=Anjan%20Venkatesh)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anjan%20Venkatesh) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anjan%20Venkatesh%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
33.  Francesco Iorio
    
    [View author publications](https://www.nature.com/search?author=Francesco%20Iorio)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Francesco%20Iorio) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Francesco%20Iorio%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
34.  Colm J. Ryan
    
    [View author publications](https://www.nature.com/search?author=Colm%20J.%20Ryan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Colm%20J.%20Ryan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Colm%20J.%20Ryan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
35.  David J. Adams
    
    [View author publications](https://www.nature.com/search?author=David%20J.%20Adams)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=David%20J.%20Adams) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22David%20J.%20Adams%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

P.Y.C., D.A., I.M., J.S.P., S.C., V.I., V.O., K.W., A.V., N.A.T., M.D.C.V.-H., L.W., J.v.D.H., E.G., S.S., M.E.V.-C., L.B., F.R., S.P., A.V., F.I. and C.J.R. performed computational analysis of the CRISPR screening data, genomes, transcriptome data and/or proteome data. P.Y.C., L.S.A.S.M., V.H., R.O.-L., F.G.A.-G. and L.v.D.W. performed screens and validation experiments. Z.K. and J.C. performed mass-spec experiments and analysis. G.B. and M.W. performed mouse experiments. D.B., P.T.H. and L.S. performed phosphoinositide and PA analyses. P.Y.C. and D.J.A. wrote the paper with input from the other authors.

### Corresponding author

Correspondence to [David J. Adams](mailto:da1@sanger.ac.uk).

Ethics declarations
-------------------

### Competing interests

D.J.A. has received precompetitive funding from AstraZeneca and OpenTargets. F.I. provides consultancy services for the joint Cancer Research Horizons–AstraZeneca Functional Genomics Centre and for Mosaic Therapeutics. F.I. also receives funding from OpenTargets and Nerviano Medical Sciences Srl. The remaining authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02222-1#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral about jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Effect of _CDS2_ loss on colony formation and CDS2 protein expression.](https://www.nature.com/articles/s41588-025-02222-1/figures/7)

Top: clonogenic assay with clones produced from uveal melanoma cells containing a doxycycline-inducible _CDS2_ sgRNA construct. These data were collected from three independent experiments using different cell passages. Middle; replicate clonogenic assays using polyclonal cell lines. R refers to independent biological replicates/the experiment was performed three times independently. STG2 refers to safe-targeting sgRNA 2 (Supplementary Table [17](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Cells were seeded in six-well plates for these experiments. Colony formation after 14 days of treatment with 0.1 µg ml−1 doxycycline. Cells were fixed and stained with 0.1% crystal violet. Bottom: Western blots are shown which illustrate CDS2 protein depletion upon doxycycline treatment. Multiple clones of MP41 are shown (A, D, E) and one OMM2.5 clone. These blots were performed once. Days indicates the days after the addition of Dox. Vinculin was used as a loading control. Dox, doxycycline; DMSO, dimethyl sulfoxide. The higher band for CDS2 seen in MP41 has been reported previously[102](https://www.nature.com/articles/s41588-025-02222-1#ref-CR102 "Blunsom, N. J., Gomez-Espinosa, E., Ashlin, T. G. & Cockcroft, S. Mitochondrial CDP-diacylglycerol synthase activity is due to the peripheral protein, TAMM41 and not due to the integral membrane protein, CDP-diacylglycerol synthase 1. Biochim. Biophys. Acta Mol. Cell. Biol. Lipids 1863, 284–298 (2018).").

[Source data](https://www.nature.com/articles/s41588-025-02222-1#MOESM8)

### [Extended Data Fig. 2 Volcano plots for mass-spectrometry analysis following disruption of _CDS2._](https://www.nature.com/articles/s41588-025-02222-1/figures/8)

Top volcano plots showing depletion of CDS2 and other proteins in MP41 and OMM2.5 cells. These data represent analysis of three independent biological replicates (Methods). The Venn diagrams indicate the number of genes up- or downregulated in each model after Dox treatment (Supplementary Table [12](https://www.nature.com/articles/s41588-025-02222-1#MOESM4)). Bottom: significantly altered pathways. Proteins associated with selected enriched Gene Ontology (GO) terms are highlighted on the plots. Data visualization was performed using the Plotly package in Python. The differential expression was assessed using a paired two-sided t-test. Unadjusted _P_ values < 0.05 and log2(FC) > 0.5 were considered as denoting a significant difference. Bottom right shows 1D enrichment. 1D enrichment analysis was performed using the Perseus software, which applies a two-tailed t-test with Benjamini-Hochberg (BH) correction. This method has been outlined previously[103](https://www.nature.com/articles/s41588-025-02222-1#ref-CR103 "Cox, J. & Mann, M. 1D and 2D annotation enrichment: a statistical method integrating quantitative proteomics with complementary high-throughput data. BMC Bioinformatics 13, S12 (2012).").

### [Extended Data Fig. 3 Effect of _CDS2_ loss on PA and PIPn synthesis.](https://www.nature.com/articles/s41588-025-02222-1/figures/9)

The effect of CDS2 loss following treatment with 0.1 µg ml−1 of doxycycline for 7 days or the equivalent volume of DMSO (vehicle) on the synthesis of the major acyl chain species of PA and PI, PIP and PIP2 in MP41 and OMM2.5 cells. Values were calibrated to 34:1 PC and normalized to the mean DMSO values obtained at days 3, 5 and 7. Data are represented as mean ± SEM (n = 3) from three independent MP41 clones and one OMM2.5 clone across 3 independent experiments. Significance determined from analysis of viable and combined apoptotic/dead cell populations is shown. A t-test (two-sided) was used and exact _P_ values < 0.05 are shown. Dox, doxycycline; DMSO, dimethyl sulfoxide; PA, phosphatidic acid; PI, phosphatidylinositol; PIP, phosphatidylinositol monophosphate; PIP2, phosphatidylinositol bisphosphate; PIPn, phosphoinositide.

[Source data](https://www.nature.com/articles/s41588-025-02222-1#MOESM8)

### [Extended Data Fig. 4 _CDS2_ essentiality across DepMap correlates with GPCR signaling.](https://www.nature.com/articles/s41588-025-02222-1/figures/10)

**a**, GPCR pathways, involving receptor-ligand pairs in either signal transduction or metabolism processes (red bars; ref. [104](https://www.nature.com/articles/s41588-025-02222-1#ref-CR104 "Arora, C. et al. The landscape of cancer-rewired GPCR signaling axes. Cell Genom. 4, 100557 (2024).")). **b**, These GPCR pathways are characterized by a significantly different cumulative distribution function (CDF) of correlation values with respect to other pathways (Kolgomorov-Smirnoff (KS) _P_ value = 2.2 × 10−16), and specifically at negative correlations (KS _P_ value < 0.001). This suggests activated GPCR signaling correlates with an increase in _CDS2_ essentiality. **c**, _CDS2_ Chronos/GPCR pathway most negative correlations. The correlation shown is a Spearman coefficient. As noted above, we found that multiple pathways involved in GPCR signal transduction are more frequent at negative correlation values, suggesting that higher _CDS2_ essentiality (given by more negative Chronos score) is more common in those cancer cell lines displaying up-regulation of GPCR signal transduction pathways (given by higher normalized enrichment scores).

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM1_ESM.pdf)

Supplementary Figs. 1–15.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM4_ESM.xlsx)

Supplementary Tables 1–17.

Source data
-----------

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM5_ESM.xlsx)

Statistical source data for Fig. 4a,b,d.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM6_ESM.csv)

Statistical source data for Fig. 5b.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM7_ESM.xlsx)

Statistical source data for Fig. 6a–c,e.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM8_ESM.pdf)

Uncropped blots.

### [Source Data Extended Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02222-1/MediaObjects/41588_2025_2222_MOESM9_ESM.csv)

Statistical source data.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=The%20synthetic%20lethal%20interaction%20between%20CDS1%20and%20CDS2%20is%20a%20vulnerability%20in%20uveal%20melanoma%20and%20across%20multiple%20tumor%20types&author=Pui%20Ying%20Chan%20et%20al&contentID=10.1038%2Fs41588-025-02222-1&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-04&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02222-1)

### Cite this article

Chan, P.Y., Alexander, D., Mehta, I. _et al._ The synthetic lethal interaction between _CDS1_ and _CDS2_ is a vulnerability in uveal melanoma and across multiple tumor types. _Nat Genet_ **57**, 1672–1683 (2025). https://doi.org/10.1038/s41588-025-02222-1

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02222-1?format=refman&flavour=citation)

-   Received: 28 November 2023
    
-   Accepted: 09 May 2025
    
-   Published: 04 July 2025
    
-   Issue Date: July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02222-1
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02222-1.pdf)
