> id: https://www.nature.com/articles/s41588-025-02249-4

> link: https://www.nature.com/articles/s41588-025-02249-4

> title: Single-nucleus chromatin accessibility profiling identifies cell types and functional variants contributing to major depression

# Single-nucleus chromatin accessibility profiling identifies cell types and functional variants contributing to major depression
_Published on Tue, 05 Aug 2025 00:00:00 GMT_

Abstract
--------

Genetic variants associated with major depressive disorder (MDD) are enriched in the regulatory genome. Here, we investigate gene-regulatory mechanisms underlying MDD compared to neurotypical controls by combining single-cell chromatin accessibility with gene expression in over 200,000 cells from the dorsolateral prefrontal cortex of 84 individuals. MDD-associated alterations in chromatin accessibility were prominent in deep-layer excitatory neurons characterized by transcription factor (TF) motif accessibility and binding of NR4A2, an activity-dependent TF reactive to stress. The same neurons were enriched for MDD-associated genetic variants, disrupting TF binding sites linked to genes that likely affect synaptic communication. Furthermore, a gray matter microglia cluster exhibited decreased accessibility in individuals with MDD at binding sites bound by TFs known to regulate immune homeostasis. Finally, we identified gene-regulatory effects of MDD-risk variants using sequence-based accessibility predictions, donor-specific genotypes and cell-based assays. These findings shed light on the cell types and regulatory mechanisms through which genetic variation may increase the risk of MDD.

/\* style specs start \*/ /\* style specs end \*/

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

27,99 € / 30 days

cancel any time

[Learn more](https://shop.nature.com/products/plus/?region=ROW)

Subscribe to this journal

Receive 12 print issues and online access

195,33 € per year

only 16,28 € per issue

[Learn more](https://www.nature.com/ng/subscribe)

Buy this article

-   Purchase on SpringerLink
-   Instant access to full article PDF

[Buy now](https://link.springer.com/article/10.1038/s41588-025-02249-4?utm_source=nature&utm_medium=referral&utm_campaign=buyArticle)

Prices may be subject to local taxes which are calculated during checkout

/\* style specs start \*/ style { display: none !important; } .LiveAreaSection \* { align-content: stretch; align-items: stretch; align-self: auto; animation-delay: 0s; animation-direction: normal; animation-duration: 0s; animation-fill-mode: none; animation-iteration-count: 1; animation-name: none; animation-play-state: running; animation-timing-function: ease; azimuth: center; backface-visibility: visible; background-attachment: scroll; background-blend-mode: normal; background-clip: borderBox; background-color: transparent; background-image: none; background-origin: paddingBox; background-position: 0 0; background-repeat: repeat; background-size: auto auto; block-size: auto; border-block-end-color: currentcolor; border-block-end-style: none; border-block-end-width: medium; border-block-start-color: currentcolor; border-block-start-style: none; border-block-start-width: medium; border-bottom-color: currentcolor; border-bottom-left-radius: 0; border-bottom-right-radius: 0; border-bottom-style: none; border-bottom-width: medium; border-collapse: separate; border-image-outset: 0s; border-image-repeat: stretch; border-image-slice: 100%; border-image-source: none; border-image-width: 1; border-inline-end-color: currentcolor; border-inline-end-style: none; border-inline-end-width: medium; border-inline-start-color: currentcolor; border-inline-start-style: none; border-inline-start-width: medium; border-left-color: currentcolor; border-left-style: none; border-left-width: medium; border-right-color: currentcolor; border-right-style: none; border-right-width: medium; border-spacing: 0; border-top-color: currentcolor; border-top-left-radius: 0; border-top-right-radius: 0; border-top-style: none; border-top-width: medium; bottom: auto; box-decoration-break: slice; box-shadow: none; box-sizing: border-box; break-after: auto; break-before: auto; break-inside: auto; caption-side: top; caret-color: auto; clear: none; clip: auto; clip-path: none; color: initial; column-count: auto; column-fill: balance; column-gap: normal; column-rule-color: currentcolor; column-rule-style: none; column-rule-width: medium; column-span: none; column-width: auto; content: normal; counter-increment: none; counter-reset: none; cursor: auto; display: inline; empty-cells: show; filter: none; flex-basis: auto; flex-direction: row; flex-grow: 0; flex-shrink: 1; flex-wrap: nowrap; float: none; font-family: initial; font-feature-settings: normal; font-kerning: auto; font-language-override: normal; font-size: medium; font-size-adjust: none; font-stretch: normal; font-style: normal; font-synthesis: weight style; font-variant: normal; font-variant-alternates: normal; font-variant-caps: normal; font-variant-east-asian: normal; font-variant-ligatures: normal; font-variant-numeric: normal; font-variant-position: normal; font-weight: 400; grid-auto-columns: auto; grid-auto-flow: row; grid-auto-rows: auto; grid-column-end: auto; grid-column-gap: 0; grid-column-start: auto; grid-row-end: auto; grid-row-gap: 0; grid-row-start: auto; grid-template-areas: none; grid-template-columns: none; grid-template-rows: none; height: auto; hyphens: manual; image-orientation: 0deg; image-rendering: auto; image-resolution: 1dppx; ime-mode: auto; inline-size: auto; isolation: auto; justify-content: flexStart; left: auto; letter-spacing: normal; line-break: auto; line-height: normal; list-style-image: none; list-style-position: outside; list-style-type: disc; margin-block-end: 0; margin-block-start: 0; margin-bottom: 0; margin-inline-end: 0; margin-inline-start: 0; margin-left: 0; margin-right: 0; margin-top: 0; mask-clip: borderBox; mask-composite: add; mask-image: none; mask-mode: matchSource; mask-origin: borderBox; mask-position: 0 0; mask-repeat: repeat; mask-size: auto; mask-type: luminance; max-height: none; max-width: none; min-block-size: 0; min-height: 0; min-inline-size: 0; min-width: 0; mix-blend-mode: normal; object-fit: fill; object-position: 50% 50%; offset-block-end: auto; offset-block-start: auto; offset-inline-end: auto; offset-inline-start: auto; opacity: 1; order: 0; orphans: 2; outline-color: initial; outline-offset: 0; outline-style: none; outline-width: medium; overflow: visible; overflow-wrap: normal; overflow-x: visible; overflow-y: visible; padding-block-end: 0; padding-block-start: 0; padding-bottom: 0; padding-inline-end: 0; padding-inline-start: 0; padding-left: 0; padding-right: 0; padding-top: 0; page-break-after: auto; page-break-before: auto; page-break-inside: auto; perspective: none; perspective-origin: 50% 50%; pointer-events: auto; position: static; quotes: initial; resize: none; right: auto; ruby-align: spaceAround; ruby-merge: separate; ruby-position: over; scroll-behavior: auto; scroll-snap-coordinate: none; scroll-snap-destination: 0 0; scroll-snap-points-x: none; scroll-snap-points-y: none; scroll-snap-type: none; shape-image-threshold: 0; shape-margin: 0; shape-outside: none; tab-size: 8; table-layout: auto; text-align: initial; text-align-last: auto; text-combine-upright: none; text-decoration-color: currentcolor; text-decoration-line: none; text-decoration-style: solid; text-emphasis-color: currentcolor; text-emphasis-position: over right; text-emphasis-style: none; text-indent: 0; text-justify: auto; text-orientation: mixed; text-overflow: clip; text-rendering: auto; text-shadow: none; text-transform: none; text-underline-position: auto; top: auto; touch-action: auto; transform: none; transform-box: borderBox; transform-origin: 50% 50%0; transform-style: flat; transition-delay: 0s; transition-duration: 0s; transition-property: all; transition-timing-function: ease; vertical-align: baseline; visibility: visible; white-space: normal; widows: 2; width: auto; will-change: auto; word-break: normal; word-spacing: normal; word-wrap: normal; writing-mode: horizontalTb; z-index: auto; -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none; margin: 0; } .LiveAreaSection { width: 100%; } .LiveAreaSection .login-option-buybox { display: block; width: 100%; font-size: 17px; line-height: 30px; color: #222; padding-top: 30px; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-access-options { display: block; font-weight: 700; font-size: 17px; line-height: 30px; color: #222; font-family: Harding, Palatino, serif; } .LiveAreaSection .additional-login > li:not(:first-child)::before { transform: translateY(-50%); content: ""; height: 1rem; position: absolute; top: 50%; left: 0; border-left: 2px solid #999; } .LiveAreaSection .additional-login > li:not(:first-child) { padding-left: 10px; } .LiveAreaSection .additional-login > li { display: inline-block; position: relative; vertical-align: middle; padding-right: 10px; } .BuyBoxSection { display: flex; flex-wrap: wrap; flex: 1; flex-direction: row-reverse; margin: -30px -15px 0; } .BuyBoxSection .box-inner { width: 100%; height: 100%; padding: 30px 5px; display: flex; flex-direction: column; justify-content: space-between; } .BuyBoxSection p { margin: 0; } .BuyBoxSection .readcube-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 1; flex-basis: 255px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 300px; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .subscribe-buybox-nature-plus { background-color: #f3f3f3; flex-shrink: 1; flex-grow: 4; flex-basis: 100%; background-clip: content-box; padding: 0 15px; margin-top: 30px; } .BuyBoxSection .title-readcube, .BuyBoxSection .title-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .title-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 24px; line-height: 32px; color: #222; text-align: center; font-family: Harding, Palatino, serif; } .BuyBoxSection .asia-link, .Link-328123652, .Link-2926870917, .Link-2291679238, .Link-595459207 { color: #069; cursor: pointer; text-decoration: none; font-size: 1.05em; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 1.05em6; } .BuyBoxSection .access-readcube { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection ul { margin: 0; } .BuyBoxSection .link-usp { display: list-item; margin: 0; margin-left: 20px; padding-top: 6px; list-style-position: inside; } .BuyBoxSection .link-usp span { font-size: 14px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-asia-buybox { display: block; margin: 0; margin-right: 5%; margin-left: 5%; font-size: 14px; color: #222; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .access-buybox { display: block; margin: 0; margin-right: 10%; margin-left: 10%; font-size: 14px; color: #222; opacity: 0.8px; padding-top: 10px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .price-buybox { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; padding-top: 30px; text-align: center; } .BuyBoxSection .price-buybox-to { display: block; font-size: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; } .BuyBoxSection .price-info-text { font-size: 16px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-value { font-size: 30px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-per-period { font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .price-from { font-size: 14px; padding-right: 10px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 20px; } .BuyBoxSection .issue-buybox { display: block; font-size: 13px; text-align: center; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 19px; } .BuyBoxSection .no-price-buybox { display: block; font-size: 13px; line-height: 18px; text-align: center; padding-right: 10%; padding-left: 10%; padding-bottom: 20px; padding-top: 30px; color: #222; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; } .BuyBoxSection .vat-buybox { display: block; margin-top: 5px; margin-right: 20%; margin-left: 20%; font-size: 11px; color: #222; padding-top: 10px; padding-bottom: 15px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: 17px; } .BuyBoxSection .tax-buybox { display: block; width: 100%; color: #222; padding: 20px 16px; text-align: center; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; line-height: NaNpx; } .BuyBoxSection .button-container { display: flex; padding-right: 20px; padding-left: 20px; justify-content: center; } .BuyBoxSection .button-container > \* { flex: 1px; } .BuyBoxSection .button-container > a:hover, .Button-505204839:hover, .Button-1078489254:hover, .Button-2737859108:hover { text-decoration: none; } .BuyBoxSection .btn-secondary { background: #fff; } .BuyBoxSection .button-asia { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; margin-top: 75px; } .BuyBoxSection .button-label-asia, .ButtonLabel-3869432492, .ButtonLabel-3296148077, .ButtonLabel-1636778223 { display: block; color: #fff; font-size: 17px; line-height: 20px; font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; text-align: center; text-decoration: none; cursor: pointer; } .Button-505204839, .Button-1078489254, .Button-2737859108 { background: #069; border: 1px solid #069; border-radius: 0; cursor: pointer; display: block; padding: 9px; outline: 0; text-align: center; text-decoration: none; min-width: 80px; max-width: 320px; margin-top: 20px; } .Button-505204839 .btn-secondary-label, .Button-1078489254 .btn-secondary-label, .Button-2737859108 .btn-secondary-label { color: #069; } .uList-2102244549 { list-style: none; padding: 0; margin: 0; } /\* style specs end \*/

**Fig. 1: Chromatin architecture at the single-cell level.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_Fig1_HTML.png)

**Fig. 2: Chromatin profiling in individuals with MDD.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_Fig2_HTML.png)

**Fig. 3: The role of TFs in MDD.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_Fig3_HTML.png)

**Fig. 4: Genetic overlap with cell-specific and MDD-specific chromatin structure.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_Fig4_HTML.png)

**Fig. 5: Allelic effects of MDD-associated genetic variants.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_Fig5_HTML.png)

**Fig. 6: Allele-specific regulatory impact of MDD-risk variants.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_Fig6_HTML.png)

**Fig. 7: Proposed model for MDD-associated chromatin remodeling.**

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_Fig7_HTML.png)

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-025-59115-4/MediaObjects/41467_2025_59115_Fig1_HTML.png)

### [Large-scale transcriptomic analyses of major depressive disorder reveal convergent dysregulation of synaptic pathways in excitatory neurons](https://www.nature.com/articles/s41467-025-59115-4?fromPaywallRec=true)

Article Open access 28 April 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41398-024-02849-4/MediaObjects/41398_2024_2849_Fig1_HTML.png)

### [Integration of whole-exome sequencing and structural neuroimaging analysis in major depressive disorder: a joint study](https://www.nature.com/articles/s41398-024-02849-4?fromPaywallRec=true)

Article Open access 09 March 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41398-021-01493-6/MediaObjects/41398_2021_1493_Fig1_HTML.png)

### [Transcriptional-regulatory convergence across functional MDD risk variants identified by massively parallel reporter assays](https://www.nature.com/articles/s41398-021-01493-6?fromPaywallRec=true)

Article Open access 22 July 2021

Data availability
-----------------

Raw sequencing data (FASTQ files) generated in this study have been deposited in the GEO (accession no. [GSE246443](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE246443)) along with the processed peak-barcode matrices and metadata. The reference genome version used is available on the 10x Genomics website (refdata-cellranger-arc-GRCh38-2020-A-2.0.0). The raw sequencing data and gene-barcode matrix for corresponding snRNA-seq are also available on GEO (accession nos. [GSE213982](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE213982) and [GSE144136](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144136)). External PFC snATAC-seq data used for cell-type comparison are available from [GSE174367](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174367). Allen Brain Institute motor cortex data used for MetaNeighbor comparison are available at [https://portal.brain-map.org/atlases-and-data/rnaseq/human-m1-10x](https://portal.brain-map.org/atlases-and-data/rnaseq/human-m1-10x). The spatial transcriptomics data used for microglia label transfer are available using the spatialLIBD (v.1.6.0) R package and through the AWS download links provided in [https://github.com/LieberInstitute/spatialLIBD](https://github.com/LieberInstitute/spatialLIBD). MDD-associated GWAS genes are available to download from [https://www.nature.com/articles/s41467-018-03819-3](https://www.nature.com/articles/s41467-018-03819-3) (Supplementary Table [6](https://www.nature.com/articles/s41588-025-02249-4#Sec51)), [https://www.nature.com/articles/s41593-018-0326-7](https://www.nature.com/articles/s41593-018-0326-7) (Supplementary Table [9](https://www.nature.com/articles/s41588-025-02249-4#Sec51)), [https://www.nature.com/articles/s41588-018-0090-3](https://www.nature.com/articles/s41588-018-0090-3) (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02249-4#Sec51)) and [https://pmc.ncbi.nlm.nih.gov/articles/PMC10839245/](https://pmc.ncbi.nlm.nih.gov/articles/PMC10839245/); H-MAGMA-based MDD GWAS genes are available to download from [https://github.com/thewonlab/H-MAGMA/tree/master/Output\_Files](https://github.com/thewonlab/H-MAGMA/tree/master/Output_Files). MDD TWAS genes can be downloaded from [https://www.nature.com/articles/s41398-021-01411-w](https://www.nature.com/articles/s41398-021-01411-w), [https://pubmed.ncbi.nlm.nih.gov/33279206](https://pubmed.ncbi.nlm.nih.gov/33279206) and [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10839245](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10839245). MDD bulk DEGs are available at [https://pmc.ncbi.nlm.nih.gov/articles/PMC5898828](https://pmc.ncbi.nlm.nih.gov/articles/PMC5898828). PsychEncode DLPFC histone modification peaks are available from [https://www.synapse.org/#!Synapse:syn12245061/tables](https://www.synapse.org/#!Synapse:syn12245061/tables). PsychEncode DLPFC Hi-C 3D chromatin data are available from PMID: [34172755](https://pubmed.ncbi.nlm.nih.gov/34172755/) (their supplementary data 3). The PsyGeNET platform is available from [http://www.psygenet.org](http://www.psygenet.org/) and can be accessed using psygenet2r (v.1.28.0). Uniformly processed eQTL and sQTL summary statistics for DLPFC tissue, including BrainSeq (QTS000005), CommonMind (QTS000008), GTEx (QTS000015) and ROSMAP (QTS000025) cohorts, are available from the eQTL Catalog: [http://ftp.ebi.ac.uk/pub/databases/spot/eQTL/sumstats](http://ftp.ebi.ac.uk/pub/databases/spot/eQTL/sumstats). Single-cell DLPFC eQTLs were obtained from the PsychEncode2 brainSCOPE resource ([https://www.science.org/doi/10.1126/science.adi5199](https://www.science.org/doi/10.1126/science.adi5199)). We used STRING-DB (v.12.0) for protein–protein interaction analysis and MEME Suite (v.5.4.1) in the gkm-SVM workflow. Genotypes for 1000 Genomes European individuals are available for download from [https://console.cloud.google.com/storage/browser/\_details/broad-alkesgroup-public-requester-pays/LDSCORE/1000G\_Phase3\_plinkfiles.tgz](https://console.cloud.google.com/storage/browser/_details/broad-alkesgroup-public-requester-pays/LDSCORE/1000G_Phase3_plinkfiles.tgz).

Code availability
-----------------

Code used for data analysis is available at [https://github.com/MGSSdouglas/snATAC\_MDD](https://github.com/MGSSdouglas/snATAC_MDD) and is archived at Zenodo (version 0.2)[117](https://www.nature.com/articles/s41588-025-02249-4#ref-CR117 "Chawla, A., Cakmakci, D., Denniston, R. & MGSSdouglas. MGSSdouglas/snATAC_MDD: snATAC_MDD v0.2. Zenodo 
https://zenodo.org/records/15320132
(2025).").

References
----------

1.  James, S. L. et al. Global, regional, and national incidence, prevalence, and years lived with disability for 354 diseases and injuries for 195 countries and territories, 1990–2017: a systematic analysis for the Global Burden of Disease Study 2017. _The Lancet_ **392**, 1789–1858 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%2C%20regional%2C%20and%20national%20incidence%2C%20prevalence%2C%20and%20years%20lived%20with%20disability%20for%20354%20diseases%20and%20injuries%20for%20195%20countries%20and%20territories%2C%201990%E2%80%932017%3A%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202017&journal=The%20Lancet&volume=392&pages=1789-1858&publication_year=2018&author=James%2CSL) 
    
2.  Cui, L. et al. Major depressive disorder: hypothesis, mechanism, prevention and treatment. _Signal Transduct. Target. Ther._ **9**, 30 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38331979)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10853571)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXjtlKhtLc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Major%20depressive%20disorder%3A%20hypothesis%2C%20mechanism%2C%20prevention%20and%20treatment&journal=Signal%20Transduct.%20Target.%20Ther.&volume=9&publication_year=2024&author=Cui%2CL) 
    
3.  Nagy, C. et al. Astrocytic abnormalities and global DNA methylation patterns in depression and suicide. _Mol. Psychiatry_ **20**, 320–328 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24662927)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXkvVKjtrs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Astrocytic%20abnormalities%20and%20global%20DNA%20methylation%20patterns%20in%20depression%20and%20suicide&journal=Mol.%20Psychiatry&volume=20&pages=320-328&publication_year=2015&author=Nagy%2CC) 
    
4.  Edgar, N. & Sibille, E. A putative functional role for oligodendrocytes in mood regulation. _Transl. Psychiatry_ **2**, e109 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22832953)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3365253)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC38fjtVKmsg%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20putative%20functional%20role%20for%20oligodendrocytes%20in%20mood%20regulation&journal=Transl.%20Psychiatry&volume=2&publication_year=2012&author=Edgar%2CN&author=Sibille%2CE) 
    
5.  Yirmiya, R., Rimmerman, N. & Reshef, R. Depression as a microglial disease. _Trends Neurosci._ **38**, 637–658 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26442697)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1SqtbbF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Depression%20as%20a%20microglial%20disease&journal=Trends%20Neurosci.&volume=38&pages=637-658&publication_year=2015&author=Yirmiya%2CR&author=Rimmerman%2CN&author=Reshef%2CR) 
    
6.  Pantazatos, S. P. et al. Whole-transcriptome brain expression and exon-usage profiling in major depression and suicide: evidence for altered glial, endothelial and ATPase activity. _Mol. Psychiatry_ **22**, 760–773 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27528462)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlChs7zO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-transcriptome%20brain%20expression%20and%20exon-usage%20profiling%20in%20major%20depression%20and%20suicide%3A%20evidence%20for%20altered%20glial%2C%20endothelial%20and%20ATPase%20activity&journal=Mol.%20Psychiatry&volume=22&pages=760-773&publication_year=2017&author=Pantazatos%2CSP) 
    
7.  Howard, D. M. et al. Genome-wide meta-analysis of depression identifies 102 independent variants and highlights the importance of the prefrontal brain regions. _Nat. Neurosci._ **22**, 343–352 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30718901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6522363)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtF2ntrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20meta-analysis%20of%20depression%20identifies%20102%20independent%20variants%20and%20highlights%20the%20importance%20of%20the%20prefrontal%20brain%20regions&journal=Nat.%20Neurosci.&volume=22&pages=343-352&publication_year=2019&author=Howard%2CDM) 
    
8.  Als, T. D. et al. Depression pathophysiology, risk prediction of recurrence and comorbid psychiatric disorders using genome-wide analyses. _Nat. Med._ **29**, 1832–1844 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37464041)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10839245)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFSgtL3O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Depression%20pathophysiology%2C%20risk%20prediction%20of%20recurrence%20and%20comorbid%20psychiatric%20disorders%20using%20genome-wide%20analyses&journal=Nat.%20Med.&volume=29&pages=1832-1844&publication_year=2023&author=Als%2CTD) 
    
9.  Chen, Y. et al. The interaction of early life factors and depression-associated loci affecting the age at onset of the depression. _Transl. Psychiatry_ **12**, 294 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35879288)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9314326)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitF2jsLbM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20interaction%20of%20early%20life%20factors%20and%20depression-associated%20loci%20affecting%20the%20age%20at%20onset%20of%20the%20depression&journal=Transl.%20Psychiatry&volume=12&publication_year=2022&author=Chen%2CY) 
    
10.  Ochi, S. & Dwivedi, Y. Dissecting early life stress-induced adolescent depression through epigenomic approach. _Mol. Psychiatry_ **28**, 141–153 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36517640)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFSqur%2FI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dissecting%20early%20life%20stress-induced%20adolescent%20depression%20through%20epigenomic%20approach&journal=Mol.%20Psychiatry&volume=28&pages=141-153&publication_year=2023&author=Ochi%2CS&author=Dwivedi%2CY) 
    
11.  Maurano, M. T. et al. Systematic localization of common disease-associated variation in regulatory DNA. _Science_ **337**, 1190–1195 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22955828)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3771521)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1ylsLfL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20localization%20of%20common%20disease-associated%20variation%20in%20regulatory%20DNA&journal=Science&volume=337&pages=1190-1195&publication_year=2012&author=Maurano%2CMT) 
    
12.  Corces, M. R. et al. Single-cell epigenomic analyses implicate candidate causal variants at inherited risk loci for Alzheimer’s and Parkinson’s diseases. _Nat. Genet._ **52**, 1158–1168 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33106633)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7606627)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFKlu7rM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20epigenomic%20analyses%20implicate%20candidate%20causal%20variants%20at%20inherited%20risk%20loci%20for%20Alzheimer%E2%80%99s%20and%20Parkinson%E2%80%99s%20diseases&journal=Nat.%20Genet.&volume=52&pages=1158-1168&publication_year=2020&author=Corces%2CMR) 
    
13.  Nagy, C. et al. Single-nucleus transcriptomics of the prefrontal cortex in major depressive disorder implicates oligodendrocyte precursor cells and excitatory neurons. _Nat. Neurosci._ **23**, 771–781 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32341540)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVCisr8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-nucleus%20transcriptomics%20of%20the%20prefrontal%20cortex%20in%20major%20depressive%20disorder%20implicates%20oligodendrocyte%20precursor%20cells%20and%20excitatory%20neurons&journal=Nat.%20Neurosci.&volume=23&pages=771-781&publication_year=2020&author=Nagy%2CC) 
    
14.  Maitra, M. et al. Cell type specific transcriptomic differences in depression show similar patterns between males and females but implicate distinct cell types and genes. _Nat. Commun._ **14**, 2912 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37217515)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10203145)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtV2jtbrI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell%20type%20specific%20transcriptomic%20differences%20in%20depression%20show%20similar%20patterns%20between%20males%20and%20females%20but%20implicate%20distinct%20cell%20types%20and%20genes&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Maitra%2CM) 
    
15.  Buenrostro, J. D. et al. Single-cell chromatin accessibility reveals principles of regulatory variation. _Nature_ **523**, 486–490 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26083756)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4685948)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFyltLrN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20chromatin%20accessibility%20reveals%20principles%20of%20regulatory%20variation&journal=Nature&volume=523&pages=486-490&publication_year=2015&author=Buenrostro%2CJD) 
    
16.  Chawla, A., Nagy, C. & Turecki, G. Chromatin profiling techniques: exploring the chromatin environment and its contributions to complex traits. _Int. J. Mol. Sci._ **22**, 7612 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34299232)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8305586)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVWitbjI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20profiling%20techniques%3A%20exploring%20the%20chromatin%20environment%20and%20its%20contributions%20to%20complex%20traits&journal=Int.%20J.%20Mol.%20Sci.&volume=22&publication_year=2021&author=Chawla%2CA&author=Nagy%2CC&author=Turecki%2CG) 
    
17.  The 1000 Genomes Project Consortium. A global reference for human genetic variation. _Nature_ **526**, 68–74 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20global%20reference%20for%20human%20genetic%20variation&journal=Nature&volume=526&pages=68-74&publication_year=2015) 
    
18.  Granja, J. M. et al. ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis. _Nat. Genet._ **53**, 403–411 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33633365)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8012210)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXls1OgtLk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ArchR%20is%20a%20scalable%20software%20package%20for%20integrative%20single-cell%20chromatin%20accessibility%20analysis&journal=Nat.%20Genet.&volume=53&pages=403-411&publication_year=2021&author=Granja%2CJM) 
    
19.  Hauberg, M. E. et al. Common schizophrenia risk variants are enriched in open chromatin regions of human glutamatergic neurons. _Nat. Commun._ **11**, 5581 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33149216)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7643171)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1OqtL3I)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Common%20schizophrenia%20risk%20variants%20are%20enriched%20in%20open%20chromatin%20regions%20of%20human%20glutamatergic%20neurons&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Hauberg%2CME) 
    
20.  Morabito, S. et al. Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer’s disease. _Nat. Genet._ **53**, 1143–1155 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34239132)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8766217)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFSktLjK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-nucleus%20chromatin%20accessibility%20and%20transcriptomic%20characterization%20of%20Alzheimer%E2%80%99s%20disease&journal=Nat.%20Genet.&volume=53&pages=1143-1155&publication_year=2021&author=Morabito%2CS) 
    
21.  Girdhar, K. et al. Cell-specific histone modification maps in the human frontal lobe link schizophrenia risk to the neuronal epigenome. _Nat. Neurosci._ **21**, 1126–1136 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30038276)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6063773)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlOis7vM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell-specific%20histone%20modification%20maps%20in%20the%20human%20frontal%20lobe%20link%20schizophrenia%20risk%20to%20the%20neuronal%20epigenome&journal=Nat.%20Neurosci.&volume=21&pages=1126-1136&publication_year=2018&author=Girdhar%2CK) 
    
22.  Gutiérrez-Sacristán, A. et al. PsyGeNET: a knowledge platform on psychiatric disorders and their genes. _Bioinformatics_ **31**, 3075–3077 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25964630)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4565028)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PsyGeNET%3A%20a%20knowledge%20platform%20on%20psychiatric%20disorders%20and%20their%20genes&journal=Bioinformatics&volume=31&pages=3075-3077&publication_year=2015&author=Guti%C3%A9rrez-Sacrist%C3%A1n%2CA) 
    
23.  Malik, A. N. et al. Genome-wide identification and characterization of functional neuronal activity-dependent enhancers. _Nat. Neurosci._ **17**, 1330–1339 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25195102)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4297619)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFKms7jE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20identification%20and%20characterization%20of%20functional%20neuronal%20activity-dependent%20enhancers&journal=Nat.%20Neurosci.&volume=17&pages=1330-1339&publication_year=2014&author=Malik%2CAN) 
    
24.  Gosselin, D. et al. An environment-dependent transcriptional network specifies human microglia identity. _Science_ **356**, eaal3222 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28546318)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5858585)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20environment-dependent%20transcriptional%20network%20specifies%20human%20microglia%20identity&journal=Science&volume=356&publication_year=2017&author=Gosselin%2CD) 
    
25.  Lutz, P.-E. et al. Non-CG methylation and multiple histone profiles associate child abuse with immune and small GTPase dysregulation. _Nat. Commun._ **12**, 1132 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33602921)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7892573)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXkvFejs7s%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Non-CG%20methylation%20and%20multiple%20histone%20profiles%20associate%20child%20abuse%20with%20immune%20and%20small%20GTPase%20dysregulation&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Lutz%2CP-E) 
    
26.  Lutz, P.-E. et al. Association of a history of child abuse with impaired myelination in the anterior cingulate cortex: convergent epigenetic, transcriptional, and morphological evidence. _Am. J. Psychiatry_ **174**, 1185–1194 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28750583)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20of%20a%20history%20of%20child%20abuse%20with%20impaired%20myelination%20in%20the%20anterior%20cingulate%20cortex%3A%20convergent%20epigenetic%2C%20transcriptional%2C%20and%20morphological%20evidence&journal=Am.%20J.%20Psychiatry&volume=174&pages=1185-1194&publication_year=2017&author=Lutz%2CP-E) 
    
27.  Klemm, S. L., Shipony, Z. & Greenleaf, W. J. Chromatin accessibility and the regulatory epigenome. _Nat. Rev. Genet._ **20**, 207–220 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30675018)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtFGqsbk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20accessibility%20and%20the%20regulatory%20epigenome&journal=Nat.%20Rev.%20Genet.&volume=20&pages=207-220&publication_year=2019&author=Klemm%2CSL&author=Shipony%2CZ&author=Greenleaf%2CWJ) 
    
28.  Heinz, S. et al. Simple combinations of lineage-determining transcription factors prime _cis_\-regulatory elements required for macrophage and B cell identities. _Mol. Cell_ **38**, 576–589 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20513432)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898526)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXns1SlsLc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Simple%20combinations%20of%20lineage-determining%20transcription%20factors%20prime%20cis-regulatory%20elements%20required%20for%20macrophage%20and%20B%E2%80%89cell%20identities&journal=Mol.%20Cell&volume=38&pages=576-589&publication_year=2010&author=Heinz%2CS) 
    
29.  Schep, A. N., Wu, B., Buenrostro, J. D. & Greenleaf, W. J. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. _Nat. Methods_ **14**, 975–978 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28825706)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5623146)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlKjtbbE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=chromVAR%3A%20inferring%20transcription-factor-associated%20accessibility%20from%20single-cell%20epigenomic%20data&journal=Nat.%20Methods&volume=14&pages=975-978&publication_year=2017&author=Schep%2CAN&author=Wu%2CB&author=Buenrostro%2CJD&author=Greenleaf%2CWJ) 
    
30.  West, A. E. & Greenberg, M. E. Neuronal activity-regulated gene transcription in synapse development and cognitive function. _Cold Spring Harb. Perspect. Biol._ **3**, a005744 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21555405)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3098681)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neuronal%20activity-regulated%20gene%20transcription%20in%20synapse%20development%20and%20cognitive%20function&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&volume=3&publication_year=2011&author=West%2CAE&author=Greenberg%2CME) 
    
31.  Dennis, D. J., Han, S. & Schuurmans, C. bHLH transcription factors in neural development, disease, and reprogramming. _Brain Res._ **1705**, 48–65 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29544733)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXltFWru78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=bHLH%20transcription%20factors%20in%20neural%20development%2C%20disease%2C%20and%20reprogramming&journal=Brain%20Res.&volume=1705&pages=48-65&publication_year=2019&author=Dennis%2CDJ&author=Han%2CS&author=Schuurmans%2CC) 
    
32.  Harris, H. K. et al. Disruption of RFX family transcription factors causes autism, attention-deficit/hyperactivity disorder, intellectual disability, and dysregulated behavior. _Genet. Med._ **23**, 1028–1040 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33658631)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9472083)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXntFOisrc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disruption%20of%20RFX%20family%20transcription%20factors%20causes%20autism%2C%20attention-deficit%2Fhyperactivity%20disorder%2C%20intellectual%20disability%2C%20and%20dysregulated%20behavior&journal=Genet.%20Med.&volume=23&pages=1028-1040&publication_year=2021&author=Harris%2CHK) 
    
33.  Holtman, I. R., Skola, D. & Glass, C. K. Transcriptional control of microglia phenotypes in health and disease. _J. Clin. Invest._ **127**, 3220–3229 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28758903)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5669536)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptional%20control%20of%20microglia%20phenotypes%20in%20health%20and%20disease&journal=J.%20Clin.%20Invest.&volume=127&pages=3220-3229&publication_year=2017&author=Holtman%2CIR&author=Skola%2CD&author=Glass%2CCK) 
    
34.  Wehrspaun, C. C., Haerty, W. & Ponting, C. P. Microglia recapitulate a hematopoietic master regulator network in the aging human frontal cortex. _Neurobiol. Aging_ **36**, 2443.e9–2443.e20 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26002684)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXot1Wlsb8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Microglia%20recapitulate%20a%20hematopoietic%20master%20regulator%20network%20in%20the%20aging%20human%20frontal%20cortex&journal=Neurobiol.%20Aging&volume=36&pages=2443.e9-2443.e20&publication_year=2015&author=Wehrspaun%2CCC&author=Haerty%2CW&author=Ponting%2CCP) 
    
35.  Kierdorf, K. & Prinz, M. Factors regulating microglia activation. _Front. Cell. Neurosci._ **7**, 44 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23630462)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3632747)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtlOhtLvJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Factors%20regulating%20microglia%20activation&journal=Front.%20Cell.%20Neurosci.&volume=7&publication_year=2013&author=Kierdorf%2CK&author=Prinz%2CM) 
    
36.  BRAIN Initiative Cell Census Network (BICCN). A multimodal cell census and atlas of the mammalian primary motor cortex. _Nature_ **598**, 86–102 (2021).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8494634)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20multimodal%20cell%20census%20and%20atlas%20of%20the%20mammalian%20primary%20motor%20cortex&journal=Nature&volume=598&pages=86-102&publication_year=2021) 
    
37.  Peng, H. et al. Morphological diversity of single neurons in molecularly defined cell types. _Nature_ **598**, 174–181 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34616072)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8494643)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1WqsbjK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Morphological%20diversity%20of%20single%20neurons%20in%20molecularly%20defined%20cell%20types&journal=Nature&volume=598&pages=174-181&publication_year=2021&author=Peng%2CH) 
    
38.  Niu, M. et al. Claustrum mediates bidirectional and reversible control of stress-induced anxiety responses. _Sci. Adv._ **8**, eabi6375 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35302853)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8932664)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpslSls7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Claustrum%20mediates%20bidirectional%20and%20reversible%20control%20of%20stress-induced%20anxiety%20responses&journal=Sci.%20Adv.&volume=8&publication_year=2022&author=Niu%2CM) 
    
39.  Jankovic, J., Chen, S. & Le, W. D. The role of Nurr1 in the development of dopaminergic neurons and Parkinson’s disease. _Prog. Neurobiol._ **77**, 128–138 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16243425)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1GhtbvP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20Nurr1%20in%20the%20development%20of%20dopaminergic%20neurons%20and%20Parkinson%E2%80%99s%20disease&journal=Prog.%20Neurobiol.&volume=77&pages=128-138&publication_year=2005&author=Jankovic%2CJ&author=Chen%2CS&author=Le%2CWD) 
    
40.  Torretta, S. et al. NURR1 and ERR1 modulate the expression of genes of a _DRD2_ coexpression network enriched for schizophrenia risk. _J. Neurosci._ **40**, 932–941 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31811028)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6975285)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXksFyqsro%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NURR1%20and%20ERR1%20modulate%20the%20expression%20of%20genes%20of%20a%20DRD2%20coexpression%20network%20enriched%20for%20schizophrenia%20risk&journal=J.%20Neurosci.&volume=40&pages=932-941&publication_year=2020&author=Torretta%2CS) 
    
41.  Eells, J. B., Lipska, B. K., Yeung, S. K., Misler, J. A. & Nikodem, V. M. Nurr1-null heterozygous mice have reduced mesolimbic and mesocortical dopamine levels and increased stress-induced locomotor activity. _Behav. Brain Res._ **136**, 267–275 (2002).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12385813)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XnvVOltbw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nurr1-null%20heterozygous%20mice%20have%20reduced%20mesolimbic%20and%20mesocortical%20dopamine%20levels%20and%20increased%20stress-induced%20locomotor%20activity&journal=Behav.%20Brain%20Res.&volume=136&pages=267-275&publication_year=2002&author=Eells%2CJB&author=Lipska%2CBK&author=Yeung%2CSK&author=Misler%2CJA&author=Nikodem%2CVM) 
    
42.  Imura, T., Kobayashi, Y., Suzutani, K., Ichikawa‐Tomikawa, N. & Chiba, H. Differential expression of a stress‐regulated gene Nr4a2 characterizes early‐ and late‐born hippocampal granule cells. _Hippocampus_ **29**, 539–549 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30365199)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXptFamtL4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Differential%20expression%20of%20a%20stress%E2%80%90regulated%20gene%20Nr4a2%20characterizes%20early%E2%80%90%20and%20late%E2%80%90born%20hippocampal%20granule%20cells&journal=Hippocampus&volume=29&pages=539-549&publication_year=2019&author=Imura%2CT&author=Kobayashi%2CY&author=Suzutani%2CK&author=Ichikawa%E2%80%90Tomikawa%2CN&author=Chiba%2CH) 
    
43.  Campos-Melo, D., Galleguillos, D., Sánchez, N., Gysling, K. & Andrés, M. E. Nur transcription factors in stress and addiction. _Front. Mol. Neurosci._ **6**, 44 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24348325)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844937)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslKis73O)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nur%20transcription%20factors%20in%20stress%20and%20addiction&journal=Front.%20Mol.%20Neurosci.&volume=6&publication_year=2013&author=Campos-Melo%2CD&author=Galleguillos%2CD&author=S%C3%A1nchez%2CN&author=Gysling%2CK&author=Andr%C3%A9s%2CME) 
    
44.  Helbling, J.-C., Minni, A. M., Pallet, V. & Moisan, M.-P. Stress and glucocorticoid regulation of NR4A genes in mice. _J. Neurosci. Res._ **92**, 825–834 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24753204)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXislCgsLo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stress%20and%20glucocorticoid%20regulation%20of%20NR4A%20genes%20in%20mice&journal=J.%20Neurosci.%20Res.&volume=92&pages=825-834&publication_year=2014&author=Helbling%2CJ-C&author=Minni%2CAM&author=Pallet%2CV&author=Moisan%2CM-P) 
    
45.  Carpentier, R., Sacchetti, P., Ségard, P., Staels, B. & Lefebvre, P. The glucocorticoid receptor is a co-regulator of the orphan nuclear receptor Nurr1. _J. Neurochem._ **104**, 777–789 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17986226)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhvF2rsb4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20glucocorticoid%20receptor%20is%20a%20co-regulator%20of%20the%20orphan%20nuclear%20receptor%20Nurr1&journal=J.%20Neurochem.&volume=104&pages=777-789&publication_year=2008&author=Carpentier%2CR&author=Sacchetti%2CP&author=S%C3%A9gard%2CP&author=Staels%2CB&author=Lefebvre%2CP) 
    
46.  Rojas, P., Joodmardi, E., Perlmann, T. & Ögren, S. O. Rapid increase of Nurr1 mRNA expression in limbic and cortical brain structures related to coping with depression-like behavior in mice. _J. Neurosci. Res._ **88**, 2284–2293 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20175204)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXnsVWit7g%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20increase%20of%20Nurr1%20mRNA%20expression%20in%20limbic%20and%20cortical%20brain%20structures%20related%20to%20coping%20with%20depression-like%20behavior%20in%20mice&journal=J.%20Neurosci.%20Res.&volume=88&pages=2284-2293&publication_year=2010&author=Rojas%2CP&author=Joodmardi%2CE&author=Perlmann%2CT&author=%C3%96gren%2CSO) 
    
47.  He, Y. et al. Protective effect of Nr4a2 (Nurr1) against LPS-induced depressive-like behaviors via regulating activity of microglia and CamkII neurons in anterior cingulate cortex. _Pharmacol. Res._ **191**, 106717 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36948326)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmvVWnsr4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Protective%20effect%20of%20Nr4a2%20%28Nurr1%29%20against%20LPS-induced%20depressive-like%20behaviors%20via%20regulating%20activity%20of%20microglia%20and%20CamkII%20neurons%20in%20anterior%20cingulate%20cortex&journal=Pharmacol.%20Res.&volume=191&publication_year=2023&author=He%2CY) 
    
48.  Xing, G., Zhang, L., Russell, S. & Post, R. Reduction of dopamine-related transcription factors Nurr1 and NGFI-B in the prefrontal cortex in schizophrenia and bipolar disorders. _Schizophr. Res._ **84**, 36–56 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16631355)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reduction%20of%20dopamine-related%20transcription%20factors%20Nurr1%20and%20NGFI-B%20in%20the%20prefrontal%20cortex%20in%20schizophrenia%20and%20bipolar%20disorders&journal=Schizophr.%20Res.&volume=84&pages=36-56&publication_year=2006&author=Xing%2CG&author=Zhang%2CL&author=Russell%2CS&author=Post%2CR) 
    
49.  Gammie, S. C. Creation of a gene expression portrait of depression and its application for identifying potential treatments. _Sci. Rep._ **11**, 3829 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33589676)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7884719)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXksV2qur0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Creation%20of%20a%20gene%20expression%20portrait%20of%20depression%20and%20its%20application%20for%20identifying%20potential%20treatments&journal=Sci.%20Rep.&volume=11&publication_year=2021&author=Gammie%2CSC) 
    
50.  Loupe, J. M. et al. Multiomic profiling of transcription factor binding and function in human brain. _Nat. Neurosci_. [https://doi.org/10.1038/s41593-024-01658-8](https://doi.org/10.1038/s41593-024-01658-8) (2024).
    
51.  Van Der Poel, M. et al. Transcriptional profiling of human microglia reveals grey–white matter heterogeneity and multiple sclerosis-associated changes. _Nat. Commun._ **10**, 1139 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30867424)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6416318)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptional%20profiling%20of%20human%20microglia%20reveals%20grey%E2%80%93white%20matter%20heterogeneity%20and%20multiple%20sclerosis-associated%20changes&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Poel%2CM) 
    
52.  Maynard, K. R. et al. Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex. _Nat. Neurosci._ **24**, 425–436 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33558695)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8095368)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsFGqsrc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptome-scale%20spatial%20gene%20expression%20in%20the%20human%20dorsolateral%20prefrontal%20cortex&journal=Nat.%20Neurosci.&volume=24&pages=425-436&publication_year=2021&author=Maynard%2CKR) 
    
53.  Scheepstra, K. W. F. et al. Microglia transcriptional profiling in major depressive disorder shows inhibition of cortical gray matter microglia. _Biol. Psychiatry_ [https://doi.org/10.1016/j.biopsych.2023.04.020](https://doi.org/10.1016/j.biopsych.2023.04.020) (2023).
    
54.  ReproGen Consortium et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. _Nat. Genet._ **47**, 1228–1235 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Partitioning%20heritability%20by%20functional%20annotation%20using%20genome-wide%20association%20summary%20statistics&journal=Nat.%20Genet.&volume=47&pages=1228-1235&publication_year=2015&author=ReproGen%20Consortium%2C) 
    
55.  Nguyen, T.-D. et al. Genetic heterogeneity and subtypes of major depression. _Mol. Psychiatry_ **27**, 1667–1675 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34997191)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9106834)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtVeisLzL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20heterogeneity%20and%20subtypes%20of%20major%20depression&journal=Mol.%20Psychiatry&volume=27&pages=1667-1675&publication_year=2022&author=Nguyen%2CT-D) 
    
56.  Shi, H., Kichaev, G. & Pasaniuc, B. Contrasting the genetic architecture of 30 complex traits from summary association data. _Am. J. Hum. Genet._ **99**, 139–153 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27346688)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5005444)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVGitr3M)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Contrasting%20the%20genetic%20architecture%20of%2030%20complex%20traits%20from%20summary%20association%20data&journal=Am.%20J.%20Hum.%20Genet.&volume=99&pages=139-153&publication_year=2016&author=Shi%2CH&author=Kichaev%2CG&author=Pasaniuc%2CB) 
    
57.  Levey, D. F. et al. Bi-ancestral depression GWAS in the Million Veteran Program and meta-analysis in >1.2 million individuals highlight new therapeutic directions. _Nat. Neurosci._ **24**, 954–963 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34045744)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8404304)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtF2msLbE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bi-ancestral%20depression%20GWAS%20in%20the%20Million%20Veteran%20Program%20and%20meta-analysis%20in%20%3E1.2%20million%20individuals%20highlight%20new%20therapeutic%20directions&journal=Nat.%20Neurosci.&volume=24&pages=954-963&publication_year=2021&author=Levey%2CDF) 
    
58.  Lee, D. et al. A method to predict the impact of regulatory variants from DNA sequence. _Nat. Genet._ **47**, 955–961 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26075791)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4520745)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFeju7vM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20method%20to%20predict%20the%20impact%20of%20regulatory%20variants%20from%20DNA%20sequence&journal=Nat.%20Genet.&volume=47&pages=955-961&publication_year=2015&author=Lee%2CD) 
    
59.  Shrikumar, A., Prakash, E. & Kundaje, A. GkmExplain: fast and accurate interpretation of nonlinear gapped _k_\-mer SVMs. _Bioinformatics_ **35**, i173–i182 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31510661)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6612808)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjvF2lsbg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GkmExplain%3A%20fast%20and%20accurate%20interpretation%20of%20nonlinear%20gapped%20k-mer%20SVMs&journal=Bioinformatics&volume=35&pages=i173-i182&publication_year=2019&author=Shrikumar%2CA&author=Prakash%2CE&author=Kundaje%2CA) 
    
60.  Lee, D. LS-GKM: a new gkm-SVM for large-scale datasets. _Bioinformatics_ **32**, 2196–2198 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27153584)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4937189)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2lsLfI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LS-GKM%3A%20a%20new%20gkm-SVM%20for%20large-scale%20datasets&journal=Bioinformatics&volume=32&pages=2196-2198&publication_year=2016&author=Lee%2CD) 
    
61.  Hu, B. et al. Neuronal and glial 3D chromatin architecture informs the cellular etiology of brain disorders. _Nat. Commun._ **12**, 3968 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34172755)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8233376)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFOgtLnM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neuronal%20and%20glial%203D%20chromatin%20architecture%20informs%20the%20cellular%20etiology%20of%20brain%20disorders&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Hu%2CB) 
    
62.  Yao, X. et al. Integrative analysis of genome-wide association studies identifies novel loci associated with neuropsychiatric disorders. _Transl. Psychiatry_ **11**, 69 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33479212)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7820351)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXls1Kmsrg%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20analysis%20of%20genome-wide%20association%20studies%20identifies%20novel%20loci%20associated%20with%20neuropsychiatric%20disorders&journal=Transl.%20Psychiatry&volume=11&publication_year=2021&author=Yao%2CX) 
    
63.  Martins-de-Souza, D., Guest, P. C., Vanattou-Saifoudine, N., Rahmoune, H. & Bahn, S. Phosphoproteomic differences in major depressive disorder postmortem brains indicate effects on synaptic function. _Eur. Arch. Psychiatry Clin. Neurosci._ **262**, 657–666 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22350622)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3491199)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phosphoproteomic%20differences%20in%20major%20depressive%20disorder%20postmortem%20brains%20indicate%20effects%20on%20synaptic%20function&journal=Eur.%20Arch.%20Psychiatry%20Clin.%20Neurosci.&volume=262&pages=657-666&publication_year=2012&author=Martins-de-Souza%2CD&author=Guest%2CPC&author=Vanattou-Saifoudine%2CN&author=Rahmoune%2CH&author=Bahn%2CS) 
    
64.  Matt, L., Kim, K., Chowdhury, D. & Hell, J. W. Role of palmitoylation of postsynaptic proteins in promoting synaptic plasticity. _Front. Mol. Neurosci._ **12**, 8 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30766476)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6365469)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlWgsrzK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20palmitoylation%20of%20postsynaptic%20proteins%20in%20promoting%20synaptic%20plasticity&journal=Front.%20Mol.%20Neurosci.&volume=12&publication_year=2019&author=Matt%2CL&author=Kim%2CK&author=Chowdhury%2CD&author=Hell%2CJW) 
    
65.  Callejas-Marin, A. et al. _Gli2_\-mediated _Shh_ signaling is required for thalamocortical projection guidance. _Front. Neuroanat._ **16**, 830758 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35221935)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8866834)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gli2-mediated%20Shh%20signaling%20is%20required%20for%20thalamocortical%20projection%20guidance&journal=Front.%20Neuroanat.&volume=16&publication_year=2022&author=Callejas-Marin%2CA) 
    
66.  Pappas, A. L. et al. Deficiency of _Shank2_ causes mania-like behavior that responds to mood stabilizers. _JCI Insight_ **2**, e92052 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29046483)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5846902)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deficiency%20of%20Shank2%20causes%20mania-like%20behavior%20that%20responds%20to%20mood%20stabilizers&journal=JCI%20Insight&volume=2&publication_year=2017&author=Pappas%2CAL) 
    
67.  Hong, C. et al. Constitutive activation of LXR in macrophages regulates metabolic and inflammatory gene expression: identification of ARL7 as a direct target. _J. Lipid Res._ **52**, 531–539 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21187453)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3035689)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXisFKku7g%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Constitutive%20activation%20of%20LXR%20in%20macrophages%20regulates%20metabolic%20and%20inflammatory%20gene%20expression%3A%20identification%20of%20ARL7%20as%20a%20direct%20target&journal=J.%20Lipid%20Res.&volume=52&pages=531-539&publication_year=2011&author=Hong%2CC) 
    
68.  van de Geijn, B., McVicker, G., Gilad, Y. & Pritchard, J. K. WASP: allele-specific software for robust molecular quantitative trait locus discovery. _Nat. Methods_ **12**, 1061–1063 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26366987)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4626402)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WASP%3A%20allele-specific%20software%20for%20robust%20molecular%20quantitative%20trait%20locus%20discovery&journal=Nat.%20Methods&volume=12&pages=1061-1063&publication_year=2015&author=Geijn%2CB&author=McVicker%2CG&author=Gilad%2CY&author=Pritchard%2CJK) 
    
69.  Wilson, P. C. et al. Multimodal single cell sequencing implicates chromatin accessibility and genetic background in diabetic kidney disease progression. _Nat. Commun._ **13**, 5253 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36068241)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9448792)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlGmu7rN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multimodal%20single%20cell%20sequencing%20implicates%20chromatin%20accessibility%20and%20genetic%20background%20in%20diabetic%20kidney%20disease%20progression&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Wilson%2CPC) 
    
70.  Herring, C. A. et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. _Cell_ **185**, 4428–4447.e28 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36318921)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XislKgt7rI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20prefrontal%20cortex%20gene%20regulatory%20dynamics%20from%20gestation%20to%20adulthood%20at%20single-cell%20resolution&journal=Cell&volume=185&pages=4428-4447.e28&publication_year=2022&author=Herring%2CCA) 
    
71.  Eells, J. B., Misler, J. A. & Nikodem, V. M. Early postnatal isolation reduces dopamine levels, elevates dopamine turnover and specifically disrupts prepulse inhibition in Nurr1-null heterozygous mice. _Neuroscience_ **140**, 1117–1126 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16690213)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xmtlajt7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Early%20postnatal%20isolation%20reduces%20dopamine%20levels%2C%20elevates%20dopamine%20turnover%20and%20specifically%20disrupts%20prepulse%20inhibition%20in%20Nurr1-null%20heterozygous%20mice&journal=Neuroscience&volume=140&pages=1117-1126&publication_year=2006&author=Eells%2CJB&author=Misler%2CJA&author=Nikodem%2CVM) 
    
72.  Ouimet, C. C., Miller, P. E., Hemmings, H. C., Walaas, S. I. & Greengard, P. DARPP-32, a dopamine- and adenosine 3′:5′-monophosphate-regulated phosphoprotein enriched in dopamine-innervated brain regions. III. Immunocytochemical localization. _J. Neurosci._ **4**, 111–124 (1984).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=6319625)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6564746)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL2c7htlygtQ%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DARPP-32%2C%20a%20dopamine-%20and%20adenosine%203%E2%80%B2%3A5%E2%80%B2-monophosphate-regulated%20phosphoprotein%20enriched%20in%20dopamine-innervated%20brain%20regions.%20III.%20Immunocytochemical%20localization&journal=J.%20Neurosci.&volume=4&pages=111-124&publication_year=1984&author=Ouimet%2CCC&author=Miller%2CPE&author=Hemmings%2CHC&author=Walaas%2CSI&author=Greengard%2CP) 
    
73.  Santana, N., Mengod, G. & Artigas, F. Quantitative analysis of the expression of dopamine D1 and D2 receptors in pyramidal and GABAergic neurons of the rat prefrontal cortex. _Cereb. Cortex_ **19**, 849–860 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18689859)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantitative%20analysis%20of%20the%20expression%20of%20dopamine%20D1%20and%20D2%20receptors%20in%20pyramidal%20and%20GABAergic%20neurons%20of%20the%20rat%20prefrontal%20cortex&journal=Cereb.%20Cortex&volume=19&pages=849-860&publication_year=2009&author=Santana%2CN&author=Mengod%2CG&author=Artigas%2CF) 
    
74.  Baik, J.-H. Stress and the dopaminergic reward system. _Exp. Mol. Med._ **52**, 1879–1890 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33257725)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8080624)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVGksbnI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stress%20and%20the%20dopaminergic%20reward%20system&journal=Exp.%20Mol.%20Med.&volume=52&pages=1879-1890&publication_year=2020&author=Baik%2CJ-H) 
    
75.  Fang, C., Wang, H. & Naumann, R. K. Developmental patterning and neurogenetic gradients of Nurr1 positive neurons in the rat claustrum and lateral cortex. _Front. Neuroanat._ **15**, 786329 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34924965)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8675902)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtVygsbnL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Developmental%20patterning%20and%20neurogenetic%20gradients%20of%20Nurr1%20positive%20neurons%20in%20the%20rat%20claustrum%20and%20lateral%20cortex&journal=Front.%20Neuroanat.&volume=15&publication_year=2021&author=Fang%2CC&author=Wang%2CH&author=Naumann%2CRK) 
    
76.  Català-Solsona, J. et al. Activity-dependent Nr4a2 induction modulates synaptic expression of AMPA receptors and plasticity via a Ca2+/CRTC1/CREB pathway. _J. Neurosci._ **43**, 3028–3041 (2023).
    
77.  Bleakman, D., Alt, A. & Witkin, J. AMPA receptors in the therapeutic management of depression. _CNS Neurol. Disord. Drug Targets_ **6**, 117–126 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17430149)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXltValtbw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=AMPA%20receptors%20in%20the%20therapeutic%20management%20of%20depression&journal=CNS%20Neurol.%20Disord.%20Drug%20Targets&volume=6&pages=117-126&publication_year=2007&author=Bleakman%2CD&author=Alt%2CA&author=Witkin%2CJ) 
    
78.  Wang, Q., Roy, B., Turecki, G., Shelton, R. C. & Dwivedi, Y. Role of complex epigenetic switching in tumor necrosis factor-α upregulation in the prefrontal cortex of suicide subjects. _Am. J. Psychiatry_ **175**, 262–274 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29361849)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5832541)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20complex%20epigenetic%20switching%20in%20tumor%20necrosis%20factor-%CE%B1%20upregulation%20in%20the%20prefrontal%20cortex%20of%20suicide%20subjects&journal=Am.%20J.%20Psychiatry&volume=175&pages=262-274&publication_year=2018&author=Wang%2CQ&author=Roy%2CB&author=Turecki%2CG&author=Shelton%2CRC&author=Dwivedi%2CY) 
    
79.  O’Connor, J. C. et al. Lipopolysaccharide-induced depressive-like behavior is mediated by indoleamine 2,3-dioxygenase activation in mice. _Mol. Psychiatry_ **14**, 511–522 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18195714)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lipopolysaccharide-induced%20depressive-like%20behavior%20is%20mediated%20by%20indoleamine%202%2C3-dioxygenase%20activation%20in%20mice&journal=Mol.%20Psychiatry&volume=14&pages=511-522&publication_year=2009&author=O%E2%80%99Connor%2CJC) 
    
80.  Snijders, G. J. L. J. et al. Distinct non-inflammatory signature of microglia in post-mortem brain tissue of patients with major depressive disorder. _Mol. Psychiatry_ **26**, 3336–3349 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33028963)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitV2hsLvF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20non-inflammatory%20signature%20of%20microglia%20in%20post-mortem%20brain%20tissue%20of%20patients%20with%20major%20depressive%20disorder&journal=Mol.%20Psychiatry&volume=26&pages=3336-3349&publication_year=2021&author=Snijders%2CGJLJ) 
    
81.  Böttcher, C. et al. Single-cell mass cytometry of microglia in major depressive disorder reveals a non-inflammatory phenotype with increased homeostatic marker expression. _Transl. Psychiatry_ **10**, 310 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32917850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7486938)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20mass%20cytometry%20of%20microglia%20in%20major%20depressive%20disorder%20reveals%20a%20non-inflammatory%20phenotype%20with%20increased%20homeostatic%20marker%20expression&journal=Transl.%20Psychiatry&volume=10&publication_year=2020&author=B%C3%B6ttcher%2CC) 
    
82.  Zhang, J., Kaye, A. P., Wang, J. & Girgenti, M. J. Transcriptomics of the depressed and PTSD brain. _Neurobiol. Stress_ **15**, 100408 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34703849)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8524242)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVCitrjK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptomics%20of%20the%20depressed%20and%20PTSD%20brain&journal=Neurobiol.%20Stress&volume=15&publication_year=2021&author=Zhang%2CJ&author=Kaye%2CAP&author=Wang%2CJ&author=Girgenti%2CMJ) 
    
83.  Hannestad, J. et al. The neuroinflammation marker translocator protein is not elevated in individuals with mild-to-moderate depression: a \[11C\]PBR28 PET study. _Brain. Behav. Immun._ **33**, 131–138 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23850810)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3899398)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFKlt7bP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20neuroinflammation%20marker%20translocator%20protein%20is%20not%20elevated%20in%20individuals%20with%20mild-to-moderate%20depression%3A%20a%20%5B11C%5DPBR28%20PET%20study&journal=Brain.%20Behav.%20Immun.&volume=33&pages=131-138&publication_year=2013&author=Hannestad%2CJ) 
    
84.  Jaffe, A. E. et al. Decoding shared versus divergent transcriptomic signatures across cortico-amygdala circuitry in PTSD and depressive disorders. _Am. J. Psychiatry_ **179**, 673–686 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35791611)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10697016)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Decoding%20shared%20versus%20divergent%20transcriptomic%20signatures%20across%20cortico-amygdala%20circuitry%20in%20PTSD%20and%20depressive%20disorders&journal=Am.%20J.%20Psychiatry&volume=179&pages=673-686&publication_year=2022&author=Jaffe%2CAE) 
    
85.  Zhang, Y. et al. Microglia-specific transcriptional repression of interferon-regulated genes after prolonged stress in mice. _Neurobiol. Stress_ **21**, 100495 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36532375)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9755030)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1aisLrE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Microglia-specific%20transcriptional%20repression%20of%20interferon-regulated%20genes%20after%20prolonged%20stress%20in%20mice&journal=Neurobiol.%20Stress&volume=21&publication_year=2022&author=Zhang%2CY) 
    
86.  Bolton, J. L. et al. Early stress-induced impaired microglial pruning of excitatory synapses on immature CRH-expressing neurons provokes aberrant adult stress responses. _Cell Rep._ **38**, 110600 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35354026)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9014810)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XosFejtr8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Early%20stress-induced%20impaired%20microglial%20pruning%20of%20excitatory%20synapses%20on%20immature%20CRH-expressing%20neurons%20provokes%20aberrant%20adult%20stress%20responses&journal=Cell%20Rep.&volume=38&publication_year=2022&author=Bolton%2CJL) 
    
87.  Wray, N. R. et al. Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression. _Nat. Genet._ **50**, 668–681 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29700475)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5934326)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXosVCks74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20analyses%20identify%2044%20risk%20variants%20and%20refine%20the%20genetic%20architecture%20of%20major%20depression&journal=Nat.%20Genet.&volume=50&pages=668-681&publication_year=2018&author=Wray%2CNR) 
    
88.  Howard, D. M. et al. Genome-wide association study of depression phenotypes in UK Biobank identifies variants in excitatory synaptic pathways. _Nat. Commun._ **9**, 1470 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29662059)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5902628)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20of%20depression%20phenotypes%20in%20UK%20Biobank%20identifies%20variants%20in%20excitatory%20synaptic%20pathways&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Howard%2CDM) 
    
89.  Sey, N. Y. A. et al. A computational tool (H-MAGMA) for improved prediction of brain-disorder risk genes by incorporating brain chromatin interaction profiles. _Nat. Neurosci._ **23**, 583–593 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32152537)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7131892)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXks1Khu78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20computational%20tool%20%28H-MAGMA%29%20for%20improved%20prediction%20of%20brain-disorder%20risk%20genes%20by%20incorporating%20brain%20chromatin%20interaction%20profiles&journal=Nat.%20Neurosci.&volume=23&pages=583-593&publication_year=2020&author=Sey%2CNYA) 
    
90.  Li, X. et al. Transcriptome-wide association study identifies new susceptibility genes and pathways for depression. _Transl. Psychiatry_ **11**, 306 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34021117)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8140098)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1Olu7nN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcriptome-wide%20association%20study%20identifies%20new%20susceptibility%20genes%20and%20pathways%20for%20depression&journal=Transl.%20Psychiatry&volume=11&publication_year=2021&author=Li%2CX) 
    
91.  Dall’Aglio, L., Lewis, C. M. & Pain, O. Delineating the genetic component of gene expression in major depression. _Biol. Psychiatry_ **89**, 627–636 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33279206)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7886308)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Delineating%20the%20genetic%20component%20of%20gene%20expression%20in%20major%20depression&journal=Biol.%20Psychiatry&volume=89&pages=627-636&publication_year=2021&author=Dall%E2%80%99Aglio%2CL&author=Lewis%2CCM&author=Pain%2CO) 
    
92.  Gandal, M. J. et al. Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap. _Science_ **359**, 693–697 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29439242)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5898828)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1CjtLY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Shared%20molecular%20neuropathology%20across%20major%20psychiatric%20disorders%20parallels%20polygenic%20overlap&journal=Science&volume=359&pages=693-697&publication_year=2018&author=Gandal%2CMJ) 
    
93.  Dumais, A. et al. Risk factors for suicide completion in major depression: a case-control study of impulsive and aggressive behaviors in men. _Am. J. Psychiatry_ **162**, 2116–2124 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16263852)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD2MrosVegtQ%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Risk%20factors%20for%20suicide%20completion%20in%20major%20depression%3A%20a%20case-control%20study%20of%20impulsive%20and%20aggressive%20behaviors%20in%20men&journal=Am.%20J.%20Psychiatry&volume=162&pages=2116-2124&publication_year=2005&author=Dumais%2CA) 
    
94.  Maitra, M. et al. Extraction of nuclei from archived postmortem tissues for single-nucleus sequencing applications. _Nat. Protoc._ **16**, 2788–2801 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33972803)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVKlsb%2FM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extraction%20of%20nuclei%20from%20archived%20postmortem%20tissues%20for%20single-nucleus%20sequencing%20applications&journal=Nat.%20Protoc.&volume=16&pages=2788-2801&publication_year=2021&author=Maitra%2CM) 
    
95.  Lareau, C. A., Ma, S., Duarte, F. M. & Buenrostro, J. D. Inference and effects of barcode multiplets in droplet-based single-cell assays. _Nat. Commun._ **11**, 866 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32054859)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7018801)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXkvFajtbk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inference%20and%20effects%20of%20barcode%20multiplets%20in%20droplet-based%20single-cell%20assays&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Lareau%2CCA&author=Ma%2CS&author=Duarte%2CFM&author=Buenrostro%2CJD) 
    
96.  Huang, Y., McCarthy, D. J. & Stegle, O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. _Genome Biol._ **20**, 273 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31836005)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6909514)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Vireo%3A%20Bayesian%20demultiplexing%20of%20pooled%20single-cell%20RNA-seq%20data%20without%20genotype%20reference&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Huang%2CY&author=McCarthy%2CDJ&author=Stegle%2CO) 
    
97.  Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. _Nat. Methods_ **16**, 1289–1296 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31740819)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884693)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOqsr7N)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%2C%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&journal=Nat.%20Methods&volume=16&pages=1289-1296&publication_year=2019&author=Korsunsky%2CI) 
    
98.  Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). _Genome Biol._ **9**, R137 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18798982)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Model-based%20analysis%20of%20ChIP-Seq%20%28MACS%29&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Zhang%2CY) 
    
99.  Hagenauer, M. H. et al. Inference of cell type content from human brain transcriptomic datasets illuminates the effects of age, manner of death, dissection, and psychiatric diagnosis. _PLoS ONE_ **13**, e0200003 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30016334)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6049916)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inference%20of%20cell%20type%20content%20from%20human%20brain%20transcriptomic%20datasets%20illuminates%20the%20effects%20of%20age%2C%20manner%20of%20death%2C%20dissection%2C%20and%20psychiatric%20diagnosis&journal=PLoS%20ONE&volume=13&publication_year=2018&author=Hagenauer%2CMH) 
    
100.  Stuart, T., Srivastava, A., Madad, S., Lareau, C. A. & Satija, R. Single-cell chromatin state analysis with Signac. _Nat. Methods_ **18**, 1333–1341 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34725479)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9255697)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlyksL3L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20chromatin%20state%20analysis%20with%20Signac&journal=Nat.%20Methods&volume=18&pages=1333-1341&publication_year=2021&author=Stuart%2CT&author=Srivastava%2CA&author=Madad%2CS&author=Lareau%2CCA&author=Satija%2CR) 
    
101.  Stuart, T. et al. Comprehensive integration of single-cell data. _Cell_ **177**, 1888–1902.e21 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31178118)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6687398)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFens77L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20integration%20of%20single-cell%20data&journal=Cell&volume=177&pages=1888-1902.e21&publication_year=2019&author=Stuart%2CT) 
    
102.  Crow, M., Paul, A., Ballouz, S., Huang, Z. J. & Gillis, J. Characterizing the replicability of cell types defined by single cell RNA-sequencing data using MetaNeighbor. _Nat. Commun._ **9**, 884 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29491377)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5830442)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterizing%20the%20replicability%20of%20cell%20types%20defined%20by%20single%20cell%20RNA-sequencing%20data%20using%20MetaNeighbor&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Crow%2CM&author=Paul%2CA&author=Ballouz%2CS&author=Huang%2CZJ&author=Gillis%2CJ) 
    
103.  Law, C. W., Chen, Y., Shi, W. & Smyth, G. K. voom: precision weights unlock linear model analysis tools for RNA-seq read counts. _Genome Biol._ **15**, R29 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24485249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053721)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=voom%3A%20precision%20weights%20unlock%20linear%20model%20analysis%20tools%20for%20RNA-seq%20read%20counts&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Law%2CCW&author=Chen%2CY&author=Shi%2CW&author=Smyth%2CGK) 
    
104.  Crowell, H. L. et al. _muscat_ detects subpopulation-specific state transitions from multi-sample multi-condition single-cell transcriptomics data. _Nat. Commun._ **11**, 6077 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33257685)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7705760)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFWlu7nL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=muscat%20detects%20subpopulation-specific%20state%20transitions%20from%20multi-sample%20multi-condition%20single-cell%20transcriptomics%20data&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Crowell%2CHL) 
    
105.  Robinson, M. D., McCarthy, D. J. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. _Bioinformatics_ **26**, 139–140 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19910308)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1WlurvO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&journal=Bioinformatics&volume=26&pages=139-140&publication_year=2010&author=Robinson%2CMD&author=McCarthy%2CDJ&author=Smyth%2CGK) 
    
106.  Yu, G., Wang, L.-G., Han, Y. & He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. _OMICS_ **16**, 284–287 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22455463)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3339379)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmsFarsLw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=clusterProfiler%3A%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters&journal=OMICS&volume=16&pages=284-287&publication_year=2012&author=Yu%2CG&author=Wang%2CL-G&author=Han%2CY&author=He%2CQ-Y) 
    
107.  Nagel, M. et al. Meta-analysis of genome-wide association studies for neuroticism in 449,484 individuals identifies novel genetic loci and pathways. _Nat. Genet._ **50**, 920–927 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29942085)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqsLvN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Meta-analysis%20of%20genome-wide%20association%20studies%20for%20neuroticism%20in%20449%2C484%20individuals%20identifies%20novel%20genetic%20loci%20and%20pathways&journal=Nat.%20Genet.&volume=50&pages=920-927&publication_year=2018&author=Nagel%2CM) 
    
108.  Yengo, L. et al. Meta-analysis of genome-wide association studies for height and body mass index in ~700000 individuals of European ancestry. _Hum. Mol. Genet._ **27**, 3641–3649 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30124842)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6488973)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXotlGnsLs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Meta-analysis%20of%20genome-wide%20association%20studies%20for%20height%20and%20body%20mass%20index%20in%20~700000%20individuals%20of%20European%20ancestry&journal=Hum.%20Mol.%20Genet.&volume=27&pages=3641-3649&publication_year=2018&author=Yengo%2CL) 
    
109.  Trubetskoy, V. et al. Mapping genomic loci implicates genes and synaptic biology in schizophrenia. _Nature_ **604**, 502–508 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35396580)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9392466)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVWjurw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20genomic%20loci%20implicates%20genes%20and%20synaptic%20biology%20in%20schizophrenia&journal=Nature&volume=604&pages=502-508&publication_year=2022&author=Trubetskoy%2CV) 
    
110.  Stahl, E. A. et al. Genome-wide association study identifies 30 loci associated with bipolar disorder. _Nat. Genet._ **51**, 793–803 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31043756)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6956732)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslKqtr8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20identifies%2030%20loci%20associated%20with%20bipolar%20disorder&journal=Nat.%20Genet.&volume=51&pages=793-803&publication_year=2019&author=Stahl%2CEA) 
    
111.  Demontis, D. et al. Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder. _Nat. Genet._ **51**, 63–75 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30478444)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlWnu77E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20of%20the%20first%20genome-wide%20significant%20risk%20loci%20for%20attention%20deficit%2Fhyperactivity%20disorder&journal=Nat.%20Genet.&volume=51&pages=63-75&publication_year=2019&author=Demontis%2CD) 
    
112.  Watanabe, K. et al. Genome-wide meta-analysis of insomnia prioritizes genes associated with metabolic and psychiatric pathways. _Nat. Genet._ **54**, 1125–1132 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35835914)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVaqsrzK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20meta-analysis%20of%20insomnia%20prioritizes%20genes%20associated%20with%20metabolic%20and%20psychiatric%20pathways&journal=Nat.%20Genet.&volume=54&pages=1125-1132&publication_year=2022&author=Watanabe%2CK) 
    
113.  Mullins, N. et al. Dissecting the shared genetic architecture of suicide attempt, psychiatric disorders, and known risk factors. _Biol. Psychiatry_ **91**, 313–327 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34861974)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dissecting%20the%20shared%20genetic%20architecture%20of%20suicide%20attempt%2C%20psychiatric%20disorders%2C%20and%20known%20risk%20factors&journal=Biol.%20Psychiatry&volume=91&pages=313-327&publication_year=2022&author=Mullins%2CN) 
    
114.  Kerimov, N. et al. A compendium of uniformly processed human gene expression and splicing quantitative trait loci. _Nat. Genet._ **53**, 1290–1299 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34493866)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8423625)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFyjtbfN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20compendium%20of%20uniformly%20processed%20human%20gene%20expression%20and%20splicing%20quantitative%20trait%20loci&journal=Nat.%20Genet.&volume=53&pages=1290-1299&publication_year=2021&author=Kerimov%2CN) 
    
115.  Emani, P. S. et al. Single-cell genomics and regulatory networks for 388 human brains. _Science_ **384**, eadi5199 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38781369)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11365579)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1Snt7vL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20genomics%20and%20regulatory%20networks%20for%20388%20human%20brains&journal=Science&volume=384&publication_year=2024&author=Emani%2CPS) 
    
116.  Bates, D., Mächler, M., Bolker, B. & Walker, S. Fitting linear mixed-effects models using lme4. _J. Stat. Softw._ **67**, 1–48 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fitting%20linear%20mixed-effects%20models%20using%20lme4&journal=J.%20Stat.%20Softw.&volume=67&pages=1-48&publication_year=2015&author=Bates%2CD&author=M%C3%A4chler%2CM&author=Bolker%2CB&author=Walker%2CS) 
    
117.  Chawla, A., Cakmakci, D., Denniston, R. & MGSSdouglas. MGSSdouglas/snATAC\_MDD: snATAC\_MDD v0.2. _Zenodo_ [https://zenodo.org/records/15320132](https://zenodo.org/records/15320132) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02249-4?format=refman&flavour=references)

Acknowledgements
----------------

G.T. holds a Canada Research Chair (Tier 1) and is supported by grants from the Canadian Institute of Health Research (CIHR) (FDN148374, PJT183903, PJT189993) and National Institutes of Health (NIH; R01MH131818). C.N. is supported by the CIHR (PJT-183904) and Fonds de recherche du Québec–Santé (FRQS) (JC-339982). Y.L. is supported by an NSERC Discovery Grant DGECR-2019-00253 and CIHR Canada Research Chair (Tier 2) in Machine Learning for Genomics and Healthcare. M.S. works in a unit funded by the UK Medical Research Council (MC\_UU\_00011/5). D.C. is supported by a Doctoral Training Fellowship from the Fonds de recherche du Québec–Nature et technologies (FRQNT) (336340). A.C. is supported by a Doctoral Training Fellowship from the FRQS (319173). We acknowledge the expert help of the Douglas–Bell Canada Brain Bank staff. This project has been made possible with the financial support of Health Canada, through the Canada Brain Research Fund, an innovative partnership between the Government of Canada (through Health Canada) and Brain Canada, and in part by funding from the Canada First Research Excellence Fund, awarded to McGill University for the Healthy Brains for Healthy Lives Initiative, and from the FRQS through the Quebec Network on Suicide, Mood Disorders and Related Disorders. This research was enabled in part by support provided by Calcul Québec ([https://www.calculquebec.ca/en](https://www.calculquebec.ca/en)) and the Digital Research Alliance of Canada ([alliancecan.ca](https://www.alliancecan.ca/)). We thank Z. Aouabed for help with generating figures, V. Nathan for help with multi-modal SNP visualization and M. Zimmermann for help with primer validation.

Author information
------------------

Author notes

1.  These authors jointly supervised this work: Yue Li, Corina Nagy, Gustavo Turecki.
    

### Authors and Affiliations

1.  McGill Group for Suicide Studies, Douglas Institute, Department of Psychiatry, McGill University, Montreal, Quebec, Canada
    
    Anjali Chawla, Laura M. Fiori, Malosree Maitra, Jennie Yang, Dariusz Żurawek, Gabriella Frosi, Reza Rahimian, Haruka Mitsuhashi, Maria Antonietta Davoli, Ryan Denniston, Gary Gang Chen, Volodymyr Yerko, Naguib Mechawar, Corina Nagy & Gustavo Turecki
    
2.  Integrated Program in Neuroscience, McGill University, Montreal, Quebec, Canada
    
    Anjali Chawla, Malosree Maitra, Haruka Mitsuhashi, Corina Nagy & Gustavo Turecki
    
3.  School of Computer Science, McGill University, Montreal, Quebec, Canada
    
    Doruk Cakmakci & Yue Li
    
4.  Quantitative Life Sciences, McGill University, Montreal, Quebec, Canada
    
    Wenmin Zang & Yue Li
    
5.  Dr. Kiran C. Patel College of Allopathic Medicine, Nova Southeastern University, Fort Lauderdale, FL, USA
    
    Deborah Mash
    
6.  Department of Psychiatry, Department of Neuroscience and Department of Genetics and Genomic Sciences, Friedman Brain Institute Icahn School of Medicine at Mount Sinai, New York, NY, USA
    
    Kiran Girdhar & Schahram Akbarian
    
7.  Population Health Sciences, Bristol Medical School, University of Bristol, Bristol, UK
    
    Matthew Suderman
    
8.  MRC Integrative Epidemiology Unit, University of Bristol, Bristol, UK
    
    Matthew Suderman
    

Authors

1.  Anjali Chawla
    
    [View author publications](https://www.nature.com/search?author=Anjali%20Chawla)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anjali%20Chawla) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anjali%20Chawla%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Doruk Cakmakci
    
    [View author publications](https://www.nature.com/search?author=Doruk%20Cakmakci)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Doruk%20Cakmakci) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Doruk%20Cakmakci%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Laura M. Fiori
    
    [View author publications](https://www.nature.com/search?author=Laura%20M.%20Fiori)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Laura%20M.%20Fiori) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Laura%20M.%20Fiori%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Wenmin Zang
    
    [View author publications](https://www.nature.com/search?author=Wenmin%20Zang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenmin%20Zang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenmin%20Zang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Malosree Maitra
    
    [View author publications](https://www.nature.com/search?author=Malosree%20Maitra)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Malosree%20Maitra) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Malosree%20Maitra%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Jennie Yang
    
    [View author publications](https://www.nature.com/search?author=Jennie%20Yang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jennie%20Yang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jennie%20Yang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Dariusz Żurawek
    
    [View author publications](https://www.nature.com/search?author=Dariusz%20%C5%BBurawek)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Dariusz%20%C5%BBurawek) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Dariusz%20%C5%BBurawek%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Gabriella Frosi
    
    [View author publications](https://www.nature.com/search?author=Gabriella%20Frosi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gabriella%20Frosi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gabriella%20Frosi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Reza Rahimian
    
    [View author publications](https://www.nature.com/search?author=Reza%20Rahimian)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Reza%20Rahimian) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Reza%20Rahimian%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Haruka Mitsuhashi
    
    [View author publications](https://www.nature.com/search?author=Haruka%20Mitsuhashi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Haruka%20Mitsuhashi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Haruka%20Mitsuhashi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Maria Antonietta Davoli
    
    [View author publications](https://www.nature.com/search?author=Maria%20Antonietta%20Davoli)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maria%20Antonietta%20Davoli) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maria%20Antonietta%20Davoli%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Ryan Denniston
    
    [View author publications](https://www.nature.com/search?author=Ryan%20Denniston)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ryan%20Denniston) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ryan%20Denniston%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Gary Gang Chen
    
    [View author publications](https://www.nature.com/search?author=Gary%20Gang%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gary%20Gang%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gary%20Gang%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Volodymyr Yerko
    
    [View author publications](https://www.nature.com/search?author=Volodymyr%20Yerko)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Volodymyr%20Yerko) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Volodymyr%20Yerko%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Deborah Mash
    
    [View author publications](https://www.nature.com/search?author=Deborah%20Mash)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Deborah%20Mash) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Deborah%20Mash%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Kiran Girdhar
    
    [View author publications](https://www.nature.com/search?author=Kiran%20Girdhar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kiran%20Girdhar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kiran%20Girdhar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Schahram Akbarian
    
    [View author publications](https://www.nature.com/search?author=Schahram%20Akbarian)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Schahram%20Akbarian) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Schahram%20Akbarian%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Naguib Mechawar
    
    [View author publications](https://www.nature.com/search?author=Naguib%20Mechawar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Naguib%20Mechawar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Naguib%20Mechawar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Matthew Suderman
    
    [View author publications](https://www.nature.com/search?author=Matthew%20Suderman)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20Suderman) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20Suderman%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Yue Li
    
    [View author publications](https://www.nature.com/search?author=Yue%20Li)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yue%20Li) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yue%20Li%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Corina Nagy
    
    [View author publications](https://www.nature.com/search?author=Corina%20Nagy)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Corina%20Nagy) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Corina%20Nagy%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Gustavo Turecki
    
    [View author publications](https://www.nature.com/search?author=Gustavo%20Turecki)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gustavo%20Turecki) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gustavo%20Turecki%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

A.C. performed snATAC-seq experiments, conducted data analysis and interpretation, and wrote the paper. D.C. performed gkm-SVM modeling and QTL enrichment analysis. L.M.F. performed reporter assays. W.Z. performed GWAS fine mapping. W.Z. and A.C. performed GWAS heritability enrichment. M.M. provided support with data analysis and reviewed the paper. D.Z. performed western blotting and helped with data interpretation. G.F. and R.D. helped with processing genotypes and data management. R.R. and H.M. provided support with data interpretation. M.A.D., J.Y., G.G.C. and V.Y. provided wet lab support. D.M. provided samples. K.G. and S.A. provided data and reviewed the paper. N.M. contributed to sample procurement and reviewed the paper. M.S. provided bioinformatic and data analysis expertise and reviewed the paper. Y.L., C.N. and G.T. jointly supervised the study design, implementation and paper preparation. G.T. additionally obtained funding.

### Corresponding author

Correspondence to [Gustavo Turecki](mailto:gustavo.turecki@mcgill.ca).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Kristen Brennand and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 snATAC-seq study overview.](https://www.nature.com/articles/s41588-025-02249-4/figures/8)

Schematic of the snATAC-seq study design in 84 subjects with MDD (n = 44) and neurotypical controls (n = 40). Nuclei were combined from a male and female pair (either MDD or control) and pooled libraries were demultiplexed using chromatin accessibility at common variants (1000 Genomes[17](https://www.nature.com/articles/s41588-025-02249-4#ref-CR17 "The 1000 Genomes Project Consortium. A global reference for human genetic variation. Nature 526, 68–74 (2015).")) and validated using donor-specific genotypes obtained from the genotyping arrays. Cell-types were identified by unsupervised clustering of high-quality cells (>200,000) and integrated with snRNA-seq data from the same donors to identify genes regulated by MDD-associated differential chromatin regions, transcription factors (TFs), and genetic variants. Finally, gapped-kmer support vector machines (gkm-SVM) were used to predict cluster- and allele-specific accessibility alterations caused by MDD-associated genetic variants. Allele-specific gene-regulatory activity of MDD-risk variants were confirmed using donor-specific genotypes and in vitro reporter assays using cells that endogenously expressed TFs with disrupted binding sites. (Created in BioRender. Chawla, A. (2025) [https://BioRender.com/s86m244](https://biorender.com/s86m244)).

### [Extended Data Fig. 2 snATAC-seq cell-type and cluster annotation.](https://www.nature.com/articles/s41588-025-02249-4/figures/9)

**a**. UMAP plot of 201,456 cells clustered based on chromatin accessibility in 501 bp cCREs identified at cluster resolution. **b**. Dot plot shows gene-activity (ArchR GeneScore) module scores calculated based on snRNA-seq cell-type marker genes in BrainInABlender[99](https://www.nature.com/articles/s41588-025-02249-4#ref-CR99 "Hagenauer, M. H. et al. Inference of cell type content from human brain transcriptomic datasets illuminates the effects of age, manner of death, dissection, and psychiatric diagnosis. PLoS ONE 13, e0200003 (2018)."). Size of dots represents percentage of cells expressing the gene and color shows the average expression across all cells. **c**. Heatmap shows the overlap enrichment of the fluorescence-activated nuclei (FAN)-sorted bulk ATAC-seq[19](https://www.nature.com/articles/s41588-025-02249-4#ref-CR19 "Hauberg, M. E. et al. Common schizophrenia risk variants are enriched in open chromatin regions of human glutamatergic neurons. Nat. Commun. 11, 5581 (2020).") cell-type marker peaks in snATAC-seq cluster-specific marker cCREs (one-sided Hypergeometric test, FDR adjusted p < 0.05 & Log2FC > 0.5, MGAS: Microglia and Astrocytes; ExN: Excitatory neurons; InN: Inhibitory neurons; Oli: Oligodendrocyte-lineage cells). **d**. UMAP plot of 201,456 cells clustered based on chromatin accessibility in 501 bp cCREs and colored by cell-type labels predicted from the reference PFC snATAC-seq data[20](https://www.nature.com/articles/s41588-025-02249-4#ref-CR20 "Morabito, S. et al. Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer’s disease. Nat. Genet. 53, 1143–1155 (2021)."). Histogram shows label-transfer prediction scores across cells in our data (median prediction score: 0.99, in red). Violin plots show the distribution of label-transfer prediction scores across cells in each cluster.

### [Extended Data Fig. 3 Integration of snATAC-seq and snRNA-seq datasets.](https://www.nature.com/articles/s41588-025-02249-4/figures/10)

**a**. UMAP plot of 201,456 cells clustered based on chromatin accessibility in 501 bp cCREs and colored by cell-type labels predicted from reference MDD snRNA-seq data[14](https://www.nature.com/articles/s41588-025-02249-4#ref-CR14 "Maitra, M. et al. Cell type specific transcriptomic differences in depression show similar patterns between males and females but implicate distinct cell types and genes. Nat. Commun. 14, 2912 (2023)."). Histogram shows label-transfer prediction scores across cells in our data (median prediction score: 0.94; in red). Violin plots show the distribution of reference label-transfer prediction scores across cells in each cluster. **b**. MetaNeighbor AUROC plots shows correspondence of MDD snATAC-seq clusters with MDD snRNA-seq clusters[14](https://www.nature.com/articles/s41588-025-02249-4#ref-CR14 "Maitra, M. et al. Cell type specific transcriptomic differences in depression show similar patterns between males and females but implicate distinct cell types and genes. Nat. Commun. 14, 2912 (2023)."). **c**. K-nearest neighbors (KNN, k = 10) based on snATAC-seq cCRE accessibility z-scores (left) and snRNA-seq integrated gene-expression z-scores (right) computed for significant peak-to-gene linkages (r > 0.45) and grouped by snATAC-seq cell-types. **d**. Overlap of cluster-specific marker cCREs (Fig. [1g](https://www.nature.com/articles/s41588-025-02249-4#Fig1) ) and gene linked cCREs (peak-to-gene linkages, r > 0.45) with cortical ChIP-seq histone modification peaks (PsychEncode)[21](https://www.nature.com/articles/s41588-025-02249-4#ref-CR21 "Girdhar, K. et al. Cell-specific histone modification maps in the human frontal lobe link schizophrenia risk to the neuronal epigenome. Nat. Neurosci. 21, 1126–1136 (2018).") (H3K27ac marks enhancers, H3K27ac+H3K4me3 marks promoters). **e**. Overlap enrichment (one-sided Fisher’s exact test, unadjusted p < 0.05) between genes linked to cell-type marker cCREs in snATAC-seq (peak-to-gene linkages, r > 0.45) and cell-type marker genes from snRNA-seq data[14](https://www.nature.com/articles/s41588-025-02249-4#ref-CR14 "Maitra, M. et al. Cell type specific transcriptomic differences in depression show similar patterns between males and females but implicate distinct cell types and genes. Nat. Commun. 14, 2912 (2023)."). Heatmap is colored by Jaccard index and p values are mentioned in red. **f**. Bar plots show the distribution of gene-linked cCREs (peak-to-gene linkages, r > 0.45) split by the peak-type (promoter, intronic, exonic, distal). Median values for the number of cCREs linked to each gene are shown.

### [Extended Data Fig. 4 Differential accessibility analysis.](https://www.nature.com/articles/s41588-025-02249-4/figures/11)

**a-b**. Heatmap shows per-subject normalized log2 counts per million (E values from limma-voom, grouped by case (n = 44) versus control (n = 39) subjects) for significant A. ExN1 and B. Mic2 clusters DARs (FDR adjusted p < 0.05 & abs(logFC) > log(1.1) identified using limma-voom implemented in muscat[104](https://www.nature.com/articles/s41588-025-02249-4#ref-CR104 "Crowell, H. L. et al. muscat detects subpopulation-specific state transitions from multi-sample multi-condition single-cell transcriptomics data. Nat. Commun. 11, 6077 (2020)."). **c**. Dot plot shows distribution of DARs identified in each cluster and the number of subjects used for differential accessibility analysis. **d**. PsyGeNet analysis shows the total number of gene-disease associations (GDA) for genes linked to DARs identified across all clusters (peak-to-gene linkages, r > 0.45). For every GDA in PsyGeNET, an evidence index is provided, which keeps track of positive and negative findings. ‘100% association’ indicates all evidence is in support, ‘100% no association’ indicates the opposite, while ‘Both’ indicates mixed support. SI: substance-induced. **e**. Heatmap shows overlap enrichment (one-sided permutation test, nPerm=1000 times, unadjusted \*p < 0.05, \*\*p < 0.01) between cluster-specific MDD DARs and previously-identified MDD-associated histone modification peaks[25](https://www.nature.com/articles/s41588-025-02249-4#ref-CR25 "Lutz, P.-E. et al. Non-CG methylation and multiple histone profiles associate child abuse with immune and small GTPase dysregulation. Nat. Commun. 12, 1132 (2021)."), chromHMM based chromatin states[25](https://www.nature.com/articles/s41588-025-02249-4#ref-CR25 "Lutz, P.-E. et al. Non-CG methylation and multiple histone profiles associate child abuse with immune and small GTPase dysregulation. Nat. Commun. 12, 1132 (2021)."), and differentially methylated regions (DMRs)[25](https://www.nature.com/articles/s41588-025-02249-4#ref-CR25 "Lutz, P.-E. et al. Non-CG methylation and multiple histone profiles associate child abuse with immune and small GTPase dysregulation. Nat. Commun. 12, 1132 (2021)."),[26](https://www.nature.com/articles/s41588-025-02249-4#ref-CR26 "Lutz, P.-E. et al. Association of a history of child abuse with impaired myelination in the anterior cingulate cortex: convergent epigenetic, transcriptional, and morphological evidence. Am. J. Psychiatry 174, 1185–1194 (2017)."). In black are MDD-associated regions derived from the lateral amygdala and blue represents DMRs from the anterior cingulate cortex (ACC). **f**. Gene expression module scores comparing MDD versus control cells (two-sided Wilcoxon test, unadjusted p value is shown) in matched snRNA-seq clusters (ExN1: NR4A2 + ExN16\_L56 case n = 453, control n = 463; Mic2: Mic1 case n = 2465, control n = 1279) for genes linked (r > 0.45, peak-to-gene linkages) to down- or up-regulated DARs (LogFC > or < log2(1.1), FDR adjusted p < 0.05). Boxplot represents median (50th percentile, line within the box), interquartile range (IQR; 25th to 75th percentiles, bounds of the box), and the whiskers (extend to 5th and 95th percentile, data points within 1.5× IQR), Outliers beyond the whiskers are shown as dots.

### [Extended Data Fig. 5 MDD-associated TF prioritization.](https://www.nature.com/articles/s41588-025-02249-4/figures/12)

**a**. Flowchart for TF prioritization in MDD. **b**. Left: Tn5 bias subtracted TF binding footprints for the prioritized TFs in MDD versus control subjects; Right: TF target gene-expression module scores compared (two-sided Wilcoxon test, unadjusted p values are shown) between MDD vs control cells in matched snRNA-seq clusters (ExN1: NR4A2 + ExN16\_L56 case n = 453, control n = 463; Mic2: Mic1 case n = 2465, control n = 1279). Dots represent mean values in each group (red-case,green-control).

### [Extended Data Fig. 6 Functional characterization of MDD-associated clusters.](https://www.nature.com/articles/s41588-025-02249-4/figures/13)

**a**. MetaNeighbor AUROC plot shows the strongest correspondence of snATAC-seq ExN1 cluster with layer 6 IT Car3 neurons from the Allen brain institute (ABI) snRNA-seq data[36](https://www.nature.com/articles/s41588-025-02249-4#ref-CR36 "BRAIN Initiative Cell Census Network (BICCN). A multimodal cell census and atlas of the mammalian primary motor cortex. Nature 598, 86–102 (2021)."). **b**. Violin plot compares promoter-accessibility of _NR4A2_ gene in ExN clusters (two-sided Wilcoxon test, FDR adjusted p < 0.05). **c**. Tn5 bias subtracted binding footprints for NR4A2 TF in each ExN cluster. **d**. StringDB analysis depicts protein-protein interactions (PPI) between ExN1 marker TF, NR4A2 (circled in blue) and MDD-associated TFs enriched in ExN1 DARs (circled in red) (interaction evidence>0.7 (high); direct interactions: FOS, ASCL1, FOSB and indirect: EGR1). **e**. Module scores compared between snATAC-seq microglial clusters (Mic1 (n = 2622) versus Mic2 (n = 2275); two-sided Wilcoxon, unadjusted p value is shown) using snATAC-seq promoter-accessibility and snRNA-seq integrated gene-expression for the topmost (n = 25, sorted by fold change) grey matter (GM) versus white matter (WM) microglia marker genes[51](https://www.nature.com/articles/s41588-025-02249-4#ref-CR51 "Van Der Poel, M. et al. Transcriptional profiling of human microglia reveals grey–white matter heterogeneity and multiple sclerosis-associated changes. Nat. Commun. 10, 1139 (2019)."). **f**. 10x Visium spatial gene-expression in male and female DLPFC sections[52](https://www.nature.com/articles/s41588-025-02249-4#ref-CR52 "Maynard, K. R. et al. Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex. Nat. Neurosci. 24, 425–436 (2021).") colored by the label-transfer prediction scores of snATAC-seq microglial clusters, Mic1 (top) and Mic2 (bottom), using promoter-accessibility (left) and snRNA-seq integrated gene-expression (right). **g**. Module scores compared between snATAC-seq microglial clusters (Mic1 (n = 2622) versus Mic2 (n = 2275); two-sided Wilcoxon, unadjusted p values are shown) using snATAC-seq promoter-accessibility and snRNA-seq integrated gene-expression for significantly down- (n = 81) or up-regulated genes (n = 11) in a cortical GM microglia from MDD individuals[53](https://www.nature.com/articles/s41588-025-02249-4#ref-CR53 "Scheepstra, K. W. F. et al. Microglia transcriptional profiling in major depressive disorder shows inhibition of cortical gray matter microglia. Biol. Psychiatry 
https://doi.org/10.1016/j.biopsych.2023.04.020
(2023).") (called as depression disease-associated microglia, depDAM[53](https://www.nature.com/articles/s41588-025-02249-4#ref-CR53 "Scheepstra, K. W. F. et al. Microglia transcriptional profiling in major depressive disorder shows inhibition of cortical gray matter microglia. Biol. Psychiatry 
https://doi.org/10.1016/j.biopsych.2023.04.020
(2023).")). Boxplot represents median (50th percentile, line within the box), interquartile range (IQR; 25th to 75th percentiles, bounds of the box), and the whiskers (extend to 5th and 95th percentile, data points within 1.5× IQR), outliers beyond the whiskers are shown as dots.

### [Extended Data Fig. 7 MDD risk variants with cluster- and allele-specific effects.](https://www.nature.com/articles/s41588-025-02249-4/figures/14)

**a**. Stacked bar plots shows cdSNPs (top) and sSNPs (bottom) in each cluster (left to right: total number in each cluster, % SNPs that are significant eQTLs and sQTLs in DLPFC (FDR adjusted p < 0.05), % SNPs mapping to cluster-specific marker cCREs and candidate DARs). **b**. First row: Pie charts depicting percentage of cdSNPs in all MDD-associated SNPs, percentage of sSNPs in cdSNPs, and sSNPs partitioned into three confidence-sets (high, medium, low) based on <51 bp accessibility disruption (investigated for potential TF binding site disruption); Second row: Pie charts depicting percentage of significant DLPFC eQTLs in all MDD-associated SNPs, cdSNPs, and sSNPs. Arrow represents overlap enrichment p-values (one-sided hypergeometric test, unadjusted p values are shown); Third row: Pie charts depicting percentage of significant DLPFC sQTLs in all MDD-associated sSNPs, cdSNPs, and sSNPs. **c**. Genomic location based characterization of MDD-associated SNPs, cdSNPs and sSNPs showed a consistent overlap with intronic (ranging from 64.9% to 69.7%) and intergenic regions (ranging from 19.1% to 29.3%). A greater percentage of cdSNPs and sSNPs were in promoter regions (11.1% and 12.7%) compared to MDD-associated SNPs (3.8%). Additionally, the percentage of cdSNPs and sSNPs overlapping with 5’UTR were slightly greater than that of 3’UTR (0.5% and 1% increase) unlike MDD-associated SNPs (1.3% decrease). To analyze effects of non-coding sSNPs, those overlapping CDS (coding sequences) were removed. **d**. Gene ontology (GO) pathways (FDR adjusted p < 0.05) significantly enriched in the nearest and linked gene targets (peak-to-gene linkages, r > 0.45) of all MDD sSNPs across all clusters. **e**. GO pathway enrichment (FDR adjusted p < 0.05) for the genes (nearest, linked, and eQTLs) associated with MDD sSNPs in ExN1 cluster.

### [Extended Data Fig. 8 Allele-specific activity of MDD sSNPs in Mic2 cluster.](https://www.nature.com/articles/s41588-025-02249-4/figures/15)

**a**. MDD sSNPs with significant allelic-effects on chromatin accessibility in Mic2 cluster. Left to right: nearest genes to Mic2 sSNPs, linked genes with highest correlation (peak-to-gene, FDR adjusted p < 0.05) with accessibility of cCRE containing sSNPs (in grey are genes with highest correlation but FDR > 0.05), DLPFC based eQTLs and sQTLs genes associated with sSNPs, sSNPs categorized based on (<51 bp) accessibility disruption confidence-sets (high, medium, low), TF binding sites disrupted by sSNPs (TOMTOM, FDR adjusted p < 10%, listed in order of motif match significance). **b**. Multi-modal visualization for Mic2 sSNP ([rs10210757](https://www.ncbi.nlm.nih.gov/snp/?term=rs10210757)). Top to bottom: snATAC-seq-derived pseudobulk accessibility tracks for each cell-type and Mic2 cluster (normalized by reads in TSS), Manhattan plot showing MDD lead SNP (blue) and SNPs in LD (yellow, r > 0.8) with MDD sSNP (red, chr2: C > T) overlapping Mic marker cCRE (also a candidate DAR in Mic, FDR adjusted p = 0.1), Mic candidate DARs and marker cCREs within the plotted region, DLPFC neuronal and non-neuronal (NeuN+/−) H3K27ac and H3K4me3 ChIP-seq peaks, DLPFC NeuN+/− promoter-anchored Hi-C fragments and 3D loops, snATAC-seq peak-to-peak co-accessible loops (r > 0.45) present within 10kbps of sSNP associated cCRE, _ARL4C_ gene locus (chr2:234493041-234497081) showing Hi-C linkages to sSNP containing cCRE, gkmExplain importance scores for each base in 50-bp region surrounding sSNP for A1 and A2 alleles from Mic2-specific gkm-SVM model, TF binding site (FLI1) disrupted by sSNP. **c**. Example plot for Mic2 sSNP ([rs5995992](https://www.ncbi.nlm.nih.gov/snp/?term=rs5995992)). Top to bottom: snATAC-seq-derived pseudobulk accessibility tracks for each cell-type and Mic2 cluster, Manhattan plot showing MDD lead SNP (blue) and SNPs in LD (yellow, r > 0.8) with MDD sSNP (red, chr22: C > T) overlapping Mic2 candidate DAR (FDR adjusted p = 0.06) (r = 0.8, linked gene: _TOB2_), Mic2 candidate DARs and marker peaks, DLPFC NeuN+/− H3K27ac and H3K4me3 ChIP-seq peaks, DLPFC NeuN+/− promoter-anchored Hi-C fragments and 3D loops, snATAC-seq peak-to-peak co-accessibile loops (r > 0.45) present within 10kbps of sSNP containing cCRE, _TOB2_ gene locus (chr22:41433494-41446801), no known TF motif matched significantly to sSNP containing seqlet.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_MOESM1_ESM.pdf)

Supplementary results, methods, references, and Figs. 1–15 and source data (uncropped blots).

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_MOESM2_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02249-4/MediaObjects/41588_2025_2249_MOESM3_ESM.xlsx)

Supplementary Tables 1–27.

Rights and permissions
----------------------

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Single-nucleus%20chromatin%20accessibility%20profiling%20identifies%20cell%20types%20and%20functional%20variants%20contributing%20to%20major%20depression&author=Anjali%20Chawla%20et%20al&contentID=10.1038%2Fs41588-025-02249-4&copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&publication=1061-4036&publicationDate=2025-08-05&publisherName=SpringerNature&orderBeanReset=true)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02249-4)

### Cite this article

Chawla, A., Cakmakci, D., Fiori, L.M. _et al._ Single-nucleus chromatin accessibility profiling identifies cell types and functional variants contributing to major depression. _Nat Genet_ **57**, 1890–1904 (2025). https://doi.org/10.1038/s41588-025-02249-4

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02249-4?format=refman&flavour=citation)

-   Received: 05 October 2023
    
-   Accepted: 27 May 2025
    
-   Published: 05 August 2025
    
-   Issue Date: August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02249-4
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative
