> id: https://www.nature.com/articles/s41588-025-02260-9

> link: https://www.nature.com/articles/s41588-025-02260-9

> title: A contextual genomic perspective on physical activity and its relationship to health well being and illness

# A contextual genomic perspective on physical activity and its relationship to health well being and illness
_Published on Mon, 21 Jul 2025 00:00:00 GMT_

Abstract
--------

Physical activity (PA) is one of the most fundamental traits in the animal kingdom, has pervasive health benefits, and is genetically influenced. Using data from the Million Veteran Program, we conducted genetic analyses of leisure, work and home-time PA. For leisure, we included 189,812 individuals of European ancestry (SNP-based heritability (_h_2) = 0.083 ± 0.005), 27,044 of African ancestry (_h_2 = 0.034 ± 0.017) and 10,263 of Latin American ancestry (_h_2 = 0.083 ± 0.036) in a cross-ancestry meta-analysis with UK Biobank data, identifying 70 lead variants. Leisure-time PA was genetically distinct from PA at home or work, with the latter two showing less health benefit with respect to health outcomes and lifespan. Mendelian randomization analyses showed a protective role of leisure-time PA against COVID-19 hospitalization (_β_ = −0.067 ± 0.016; _P_ = 2.8 × 10−5), and with other traits including cardiovascular and respiratory system diseases, metabolic traits and aging. These findings provide new insights into the biology of PA, showing specific causal health benefits of leisure-time PA.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41514-025-00217-0/MediaObjects/41514_2025_217_Fig1_HTML.png)

### [Relationship between physical activity and DNA methylation-predicted epigenetic clocks](https://www.nature.com/articles/s41514-025-00217-0?fromPaywallRec=false)

Article Open access 12 April 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3Aplaceholder%2Fimages/placeholder-figure-nature.png)

### [Longitudinal associations between physical activity and other health behaviours during the COVID-19 pandemic: a fixed effects analysis](https://www.nature.com/articles/s41598-022-20196-6?fromPaywallRec=false)

Article Open access 24 September 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-024-77359-w/MediaObjects/41598_2024_77359_Fig1_HTML.png)

### [Dose response of leisure time physical activity and biological aging in type 2 diabetes: a cross sectional study](https://www.nature.com/articles/s41598-024-77359-w?fromPaywallRec=false)

Article Open access 01 November 2024

Main
----

Physical activity (PA) has pervasive health benefits, including beneficial influences on chronic conditions like cancer, hypertension and type 2 diabetes[1](https://www.nature.com/articles/s41588-025-02260-9#ref-CR1 "Bull, F. C. et al. World Health Organization 2020 guidelines on physical activity and sedentary behaviour. Br. J. Sports Med. 54, 1451–1462 (2020)."). The majority of these associations are supported by cross-sectional studies[2](https://www.nature.com/articles/s41588-025-02260-9#ref-CR2 "Courneya, K. S. & Friedenreich, C. M. Designing, analyzing, and interpreting observational studies of physical activity and cancer outcomes from a clinical oncology perspective. Front. Oncol. 13, 1098278 (2023)."),[3](https://www.nature.com/articles/s41588-025-02260-9#ref-CR3 "Watson, K. B. et al. Cross-sectional study of changes in physical activity behavior during the COVID-19 pandemic among US adults. Int. J. Behav. Nutr. Phys. Act. 18, 91 (2021)."),[4](https://www.nature.com/articles/s41588-025-02260-9#ref-CR4 "Oh, M. et al. Cross-sectional and longitudinal associations of lifestyle behaviors with pericardial adipose tissue: the MESA study. Med. Sci. Sports Exerc. 54, 984–993 (2022)."),[5](https://www.nature.com/articles/s41588-025-02260-9#ref-CR5 "Alzahrani, H., Mackey, M., Stamatakis, E., Zadro, J. R. & Shirley, D. The association between physical activity and low back pain: a systematic review and meta-analysis of observational studies. Sci. Rep. 9, 8244 (2019)."), which have great utility but cannot establish causality or determine the direction of causal relationships.

An individual’s level of PA is influenced by genetic factors, with estimated heritability (_h_2) between 48% and 71%[6](https://www.nature.com/articles/s41588-025-02260-9#ref-CR6 "Stubbe, J. H. et al. Genetic influences on exercise participation in 37,051 twin pairs from seven countries. PLoS ONE 1, e22 (2006)."). To identify genetic variants relevant to PA, genome-wide association studies (GWAS) have been conducted, initially focusing on PA during leisure time[7](https://www.nature.com/articles/s41588-025-02260-9#ref-CR7 "De Moor, M. H. et al. Genome-wide association study of exercise behavior in Dutch and American adults. Med Sci. Sports Exerc. 41, 1887–1895 (2009)."),[8](https://www.nature.com/articles/s41588-025-02260-9#ref-CR8 "Kim, J., Oh, S., Min, H., Kim, Y. & Park, T. Practical issues in genome-wide association studies for physical activity. Ann. N. Y. Acad. Sci. 1229, 38–44 (2011)."),[9](https://www.nature.com/articles/s41588-025-02260-9#ref-CR9 "Hara, M. et al. Genomewide association study of leisure-time exercise behavior in Japanese adults. Med. Sci. Sports Exerc. 50, 2433–2441 (2018)."),[10](https://www.nature.com/articles/s41588-025-02260-9#ref-CR10 "Lin, X. et al. Genetic determinants for leisure-time physical activity. Med Sci. Sports Exerc. 50, 1620–1628 (2018)."). Identifying genetic associations with PA affords us a unique opportunity; because we are born with genetic variations before exposures to psychological, social and environmental factors that may affect activity levels, we can begin to decompose some of the complex interactions that underlie and potentially confound the study of this important health trait. There is also a wide range of interindividual differences in requirements for PA in daily life. Some jobs require much more physical exertion than others; some individuals must exert more effort in their households than others. These kinds of PA tend to differ phenomenologically from exercise taken at leisure, occurring more in concentrated bursts in the first two cases, and with the potential to be more sustained in the latter case; yet the level of activity in different contexts is often treated as if equivalent. There are, however, prior reports that there may not be comparable health benefits for activity at leisure versus activity in other contexts[11](https://www.nature.com/articles/s41588-025-02260-9#ref-CR11 "Nabe-Nielsen, K. et al. The effect of occupational physical activity on dementia: results from the Copenhagen male study. Scand. J. Med. Sci. Sports 31, 446–455 (2021)."),[12](https://www.nature.com/articles/s41588-025-02260-9#ref-CR12 "Zotcheva, E. et al. Trajectories of occupational physical activity and risk of later-life mild cognitive impairment and dementia: the HUNT4 70+ study. Lancet Reg. Health Eur. 34, 100721 (2023)."), and genetic analyses may help shed light on such differences.

Two previous studies of PA used UK Biobank (UKB) data[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."),[14](https://www.nature.com/articles/s41588-025-02260-9#ref-CR14 "Doherty, A. et al. GWAS identifies 14 loci for device-measured physical activity and sleep duration. Nat. Commun. 9, 5257 (2018)."). The first study (_n_ = 91,105) identified three variants associated with PA, based on wrist-worn accelerometer data[14](https://www.nature.com/articles/s41588-025-02260-9#ref-CR14 "Doherty, A. et al. GWAS identifies 14 loci for device-measured physical activity and sleep duration. Nat. Commun. 9, 5257 (2018)."). The second study (_n_ ≤ 377,234) identified ten significant loci across five phenotypes, three based on self-report data (moderate-to-vigorous PA (MVPA), vigorous PA (VPA) and strenuous sport or other exercise (SSOE) during leisure time), and two on wrist-worn accelerometer data[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."). A multiancestry meta-analysis combining data from 703,901 individuals from 51 studies identified 11 associated loci for MVPA[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022)."). Combining this phenotype with two traits phenotypically contrary to PA (leisure screen time and sedentary behavior at work), 99 associated loci were found, summing results from PA (11 loci), leisure screen time (88 loci) and sedentary behavior at work (4 loci); that is, most loci were mapped for phenotypes correlated to a lack of PA[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022).").

We sought to maximize power to investigate PA genetics using a quantitative trait definition, combining different intensities of PA by incorporating their frequencies, using data from the Million Veteran Program (MVP). We focused on PA per se, as opposed to sedentary behavior, with our primary quantitative trait, All-PA–leisure, focused exclusively on PA during leisure time ([Methods](https://www.nature.com/articles/s41588-025-02260-9#Sec20)) with data from European (EUR; _n_ = 189,812), African (AFR; _n_ = 27,044) and Latin American (AMR; _n_ = 10,263) ancestries. We studied this trait in two meta-analyses (All-PA–leisure + SSOE), one of EUR ancestry (_n_ = 511,310) and one cross-ancestry (EUR, AFR and AMR ancestries; _n_ = 548,617), with SSOE data from UKB (_n_eff = 4/(1/_n_cases + 1/_n_controls) = 321,498), also based on information regarding PA during leisure time (Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02260-9#MOESM1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02260-9#MOESM3))[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."). In doing so, we substantially increased the number of discovered loci while also enhancing the diversity of the sample beyond EUR. This enabled us to investigate, from a genomic perspective, the relationship of PA to a range of health traits, and especially protective effects of PA against health outcomes including type 2 diabetes, gastroesophageal reflux disease, abdominal aortic aneurysm, heart failure, hospitalization caused by COVID-19, osteoarthritis, parental survival and an aging phenotype. We also investigated PA traits related to different activity contexts (home, work, and leisure), for which we found substantial genetic differences.

Results
-------

### GWAS of All-PA–leisure and All-PA–leisure + SSOE meta-analysis

#### All-PA–leisure and SSOE phenotypes

GWAS of All-PA–leisure in MVP EUR data identified 14 lead SNPs at 13 loci with a genome-wide significant (GWS) threshold _P_ value of 5 × 10−8 (or 6 lead SNPs in 6 loci if considering a threshold _P_ = 5 × 10−9; Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02260-9#MOESM1) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)), whereas there were no significant GWAS findings for MVP AFR and AMR data. GWAS of SSOE in UKB identified 15 lead SNPs in 14 loci with a threshold _P_ value of 5 × 10−8 (8 lead SNPs in 7 loci if considering a threshold _P_ _\=_ 5 × 10−9; Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02260-9#MOESM1) and Supplementary Table [3](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

The two most significant lead SNPs in the GWAS for All-PA–leisure (EUR, [rs761898](https://www.ncbi.nlm.nih.gov/snp/?term=rs761898), _P_ = 1.63 × 10−11; [rs7613360](https://www.ncbi.nlm.nih.gov/snp/?term=rs7613360), _P_ = 3.15 × 10−11) map to intergenic regions on chromosomes 14 and 3, respectively (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). Additional GWS loci included _SLC39A8_\*[rs13107325](https://www.ncbi.nlm.nih.gov/snp/?term=rs13107325) (_P_ = 1.57 × 10−10; solute carrier family 39 member 8). As previously described[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."), the most significant variant for GWAS of SSOE in UKB is _CADM2_\*[rs62253088](https://www.ncbi.nlm.nih.gov/snp/?term=rs62253088) (_P_ = 1.0 × 10−19 in this study; _Cell Adhesion Molecule 2_).

A sex-stratified analysis of the X chromosome in MVP (All-PA–leisure) showed no GWS variants (Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)).

#### All-PA–leisure + SSOE meta-analyses

We calculated SNP-based heritability (SNP-_h_2) in MVP data for All-PA–leisure (Table [1](https://www.nature.com/articles/s41588-025-02260-9#Tab1)), and the genetic correlation between All-PA–leisure and traits from the previous UKB report[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018).") ([Supplementary Notes](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)).

**Table 1 Estimated SNP-heritability (_h_2) of the cohorts used in the study**

The EUR All-PA–leisure + SSOE meta-analysis, including the UKB phenotype SSOE and the MVP phenotype All-PA–leisure in a sample-size-based meta-analysis, greatly increased discovery, yielding 67 lead SNPs in 55 loci (Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02260-9#MOESM1) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)), with intercept of 1.038 ± 0.011 and an attenuation ratio of the linkage disequilibrium score regression (LDSC) of 0.057 ± 0.016 (ref. [16](https://www.nature.com/articles/s41588-025-02260-9#ref-CR16 "Bulik-Sullivan, B. K. et al. LD score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat. Genet. 47, 291–295 (2015).")). The cross-ancestry meta-analysis provided 70 lead SNPs in 56 loci (Fig. [1](https://www.nature.com/articles/s41588-025-02260-9#Fig1) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). Differences in the number of lead SNPs and associated loci considering a more stringent threshold _P_ ≤ 5 × 10−9 are reported in Supplementary Table [6](https://www.nature.com/articles/s41588-025-02260-9#MOESM3). In both the meta-analyses, the strongest association was with _CADM2_\*[rs62253088](https://www.ncbi.nlm.nih.gov/snp/?term=rs62253088) (EUR meta-analysis _P_ = 2.0 × 10−21; cross-ancestry meta-analysis _P_ = 6.1 × 10−20). The second strongest associated lead SNP was also on chromosome 3: _MST1R_\*[rs3733134](https://www.ncbi.nlm.nih.gov/snp/?term=rs3733134) (EUR meta-analysis _P_ = 1.4 × 10−17; cross-ancestry meta-analysis _P_ = 6.9 × 10−18).

**Fig. 1: Cross-ancestry All-PA–leisure + SSOE meta-analysis.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02260-9/MediaObjects/41588_2025_2260_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02260-9/figures/1)

Annotated genes are the closest to each significant lead SNP. Different genes compared to EUR All-PA–leisure meta-analysis (Supplementary Fig. [6](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)) are highlighted in green. The red horizontal line indicates GWS (_P_ = 5 × 10−8) level, where _P_ values were calculated using sample-size-based approach, two-sided tests.

#### Gene-based and gene-set analyses of All-PA–leisure + SSOE

Focusing on the EUR meta-analysis, the gene-based test using MAGMA identified 119 significant genes from 19,030 protein-coding genes (GWS defined at _P_ = 2.6 × 10−6; Supplementary Table [7](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). _CADM2_ was the most significant gene (_P_ = 3.7 × 10−25) followed by _NPC1_ (_P_ = 4.8 × 10−16). The fastENLOC[17](https://www.nature.com/articles/s41588-025-02260-9#ref-CR17 "Wen, X., Pique-Regi, R. & Luca, F. Integrating molecular QTL data into genome-wide genetic association analysis: probabilistic assessment of enrichment and colocalization. PLoS Genet. 13, e1006646 (2017)."),[18](https://www.nature.com/articles/s41588-025-02260-9#ref-CR18 "Hukku, A., Sampson, M. G., Luca, F., Pique-Regi, R. & Wen, X. Analyzing and reconciling colocalization and transcriptome-wide association studies from the perspective of inferential reproducibility. Am. J. Hum. Genet. 109, 825–837 (2022).") analysis identified 41 genes with gene-level colocalization probability (GLCP) ≥ 0.9 (Supplementary Table [8](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

For the EUR All-PA–leisure + SSOE meta-analysis, we also conducted transcriptome-wide association study (TWAS) and fine-mapping ([Supplementary Results](https://www.nature.com/articles/s41588-025-02260-9#MOESM1); Supplementary Tables [9](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)–[11](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). These analyses, together with the previously mentioned gene-based approaches, highlighted two genes (_CADM2_ and _AKAP10_) shared by four approaches, and eight genes (_RGS12_, _NCP1_, _GATAD2A_, _DLAT_, _CYP46A1_, _ANK3, AFF3 and ACTN1_) shared by three approaches (discussion in ‘Supplementary Results—Genes detected by gene-based approaches’). We also conducted MTAG analysis with leisure screen time and liking of PA (‘Supplementary Results—MTAG analyses of EUR All-PA–leisure + SSOE’; Supplementary Tables [12](https://www.nature.com/articles/s41588-025-02260-9#MOESM3) and [13](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

The following four significant gene ontology (GO) terms resulted from the gene-set analysis: structural constituent of the presynapse, presynaptic active zone organization, maintenance of presynaptic active zone structure and structural constituent of the synapse (Supplementary Table [14](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

#### Tissue and cell-type specificity of All-PA–leisure + SSOE

Tissue Expression Analysis showed 11 significant enrichments in brain tissues, with the two most significant represented in the cerebellar hemisphere and in the cerebellum (Supplementary Figs. [6](https://www.nature.com/articles/s41588-025-02260-9#MOESM1) and [7](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). For gene-based association analyses and gene-set analyses, similar results were obtained for the cross-ancestry meta-analysis ([Supplementary Notes](https://www.nature.com/articles/s41588-025-02260-9#MOESM1); Supplementary Tables [15](https://www.nature.com/articles/s41588-025-02260-9#MOESM3) and [16](https://www.nature.com/articles/s41588-025-02260-9#MOESM3) and Supplementary Figs. [8](https://www.nature.com/articles/s41588-025-02260-9#MOESM1) and [9](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)).

Cell-type-enrichment analysis was performed with a collection of human cell types from brain, blood and pancreas (all available tissues). In the EUR meta-analysis of All-PA–leisure + SSOE, the only significant cell type was _GABA__+_ neurons from the [GSE76381](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76381) Linnarsson Human Midbrain dataset (adjusted _P_ value per dataset after multiple testing correction = 2.4 × 10−5). In the cross-ancestry meta-analysis, no cell types were significantly enriched. The results likely reflect the limited number of kinds of tissue included in this analysis.

#### Functional enrichment analysis

Functional enrichment analysis with g:Profiler using significant genes from the gene-based test as input, identified three significant GO terms shared by the EUR meta-analysis and the cross-ancestry meta-analysis (Supplementary Tables [17](https://www.nature.com/articles/s41588-025-02260-9#MOESM3) and [18](https://www.nature.com/articles/s41588-025-02260-9#MOESM3))—presynaptic active zone cytoplasmic component, cell cortex region and synapse. A human phenotype ontology term, polyclonal elevation of IgM, was also shared by the EUR meta-analysis and cross-ancestry meta-analysis.

#### Genetic correlations

We estimated the genetic correlation of the All-PA–leisure + SSOE phenotype (EUR meta-analysis) with traits of interest covering a broad range of diseases and physiological traits, using the Benjamini–Hochberg false discovery rate correction ([Methods](https://www.nature.com/articles/s41588-025-02260-9#Sec20)). Results showed significant negative genetic correlations between All-PA–leisure + SSOE with all disease traits studied, with the strongest values for stroke (_r__g_ = −0.62 ± 0.11; _P_ = 3.54 × 10−9) and gastroesophageal reflux disease (_r__g_ = −0.482 ± 0023; _P_ = 9.9 × 10−97; Fig. [2a](https://www.nature.com/articles/s41588-025-02260-9#Fig2), Supplementary Table [19](https://www.nature.com/articles/s41588-025-02260-9#MOESM3) and [Supplementary Notes](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). Conditional analysis adjusting for income as a measure of socioeconomic status (Supplementary Fig. [10](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)), highlighted a few changes compared to previous genetic correlations (Supplementary Table [20](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). Genetic correlations of All-PA–leisure + SSOE with height and executive functioning became nonsignificant, whereas multiple sclerosis became significant (_r__g_ = −0.184 ± 0.086; _P_ = 0.032). Consistent results were obtained from a phenome-wide association analysis (PheWAS; Fig. [2b](https://www.nature.com/articles/s41588-025-02260-9#Fig2); [Supplementary Notes](https://www.nature.com/articles/s41588-025-02260-9#MOESM1); Supplementary Table [21](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

**Fig. 2: Traits genetically correlated with All-PA–leisure + SSOE.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02260-9/MediaObjects/41588_2025_2260_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02260-9/figures/2)

**a**, Genetic correlations of EUR meta-analysis for All-PA–leisure + SSOE phenotype with traits of interest. We display here the traits of interest that were significant after the Benjamini–Hochberg false discovery procedure. Tests were two-sided. Black error bars represent the standard error. **b**, PheWAS of PA from the polygenic score derived from the EUR All-PA–leisure + SSOE meta-analysis. The red line represents the Bonferroni threshold _P_ = 3.99 × 10−5. Up-pointing and down-pointing triangles represent positive or negative associations, respectively, which were calculated using a logistic regression model.

#### Local genetic correlations

We next considered local correlations (LAVA[19](https://www.nature.com/articles/s41588-025-02260-9#ref-CR19 "Werme, J., van der Sluis, S., Posthuma, D. & de Leeuw, C. A. An integrated framework for local genetic correlation analysis. Nat. Genet. 54, 274–282 (2022).")) as global genetic correlations may mask important regional differences. There were 91 significant local bivariate genetic correlations at 61 loci between the EUR All-PA–leisure + SSOE meta-analysis and 18 traits in the genetic correlation analyses (Supplementary Table [21](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). Of these, oral cancer was not flagged by the global genetic correlation analysis. The trait with the highest number of significant local genetic correlations with All-PA–leisure + SSOE was educational attainment (31 loci), followed by body mass index (BMI; 15 loci) and income (9 loci). For two traits, height and well being, we observed local genetic correlations with EUR All-PA–leisure+SSOE in opposite directions (Supplementary Table [22](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

#### Mendelian randomization (MR) analyses of All-PA–leisure

We used MR analyses to investigate the causal relationship between All-PA–leisure and traits with which it had a significant genetic correlation. Since several statistics for target traits included UKB data, we used the GWAS of All-PA–leisure in EUR MVP (and not the meta-analysis) to avoid sample overlap. The following four methods were tested: MR-Egger, weighted median, inverse-variance weighted (IVW) and simple mode. Using a threshold _P_ < 1 × 10−5 for instrumental variables, we observed significant bidirectional negative (protective) effects of All-PA–leisure with respect to BMI, type 2 diabetes, leisure screen time, cigarettes per day and gastroesophageal reflux disease (Tables [2](https://www.nature.com/articles/s41588-025-02260-9#Tab2) and [3](https://www.nature.com/articles/s41588-025-02260-9#Tab3)). Unlike bidirectional effects, which show effects both caused by All-PA–leisure and on All-PA–leisure (that is, PA increases or decreases risk for the comparator trait, and that trait also increases or decreases PA), unidirectional effects show only one direction of causality. There were unidirectional negative (protective) effects of All-PA–leisure on abdominal aortic aneurysm, heart failure, COVID-19 hospitalization, triglycerides, phenotypic age acceleration (an age-adjusted biological age measure; [Supplementary Methods](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)) and osteoarthritis (Tables [2](https://www.nature.com/articles/s41588-025-02260-9#Tab2) and [3](https://www.nature.com/articles/s41588-025-02260-9#Tab3)). We found significant bidirectional positive effects between All-PA–leisure and HDL-cholesterol (HDL-C), parental survival, liking of PA, household income, well being and educational attainment. MR analyses with a stronger threshold _P_ = 5 × 10−8 for instrumental variables showed similar causal relationships, and also a possible protective effect of All-PA–leisure on asthma ([Supplementary Notes](https://www.nature.com/articles/s41588-025-02260-9#MOESM1); Supplementary Tables [23](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)–[25](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

**Table 2 MR analysis of causal effects of PA during leisure time (All-PA–leisure) on traits of interest**

**Table 3 MR analysis of causal effects of traits of interest on PA during leisure time (All-PA–leisure)**

#### MVMR analyses testing BMI as confounder

To understand the apparent protective effect of All-PA–leisure with respect to COVID-19 better, we considered a possible role of BMI as confounder, as high BMI leads to worse COVID-19 outcomes[20](https://www.nature.com/articles/s41588-025-02260-9#ref-CR20 "Gao, M. et al. Associations between body-mass index and COVID-19 severity in 6·9 million people in England: a prospective, community-based, cohort study. Lancet Diabetes Endocrinol. 9, 350–359 (2021)."). We performed a multivariable MR (MVMR) analysis, with the All-PA–leisure phenotype and BMI as exposures, and COVID-19 hospitalization as outcome. In this analysis, we still observed a significant protective effect of All-PA–leisure on the COVID-19 trait which considers hospitalized cases (effect = −0.067 ± 0.016, _P_ = 2.8 × 10−5; Table [4](https://www.nature.com/articles/s41588-025-02260-9#Tab4)). Considering the role that BMI has in many other health outcomes, we also conducted MVMR, including BMI for all other traits that had significant MR results. Despite the reduced power of the analysis due to a decrease in the number of instrumental variables suitable for all three traits, we found significant protective effects of All-PA–leisure on HDL-C, triglycerides, osteoarthritis and gastroesophageal reflux disease (Table [4](https://www.nature.com/articles/s41588-025-02260-9#Tab4) and Supplementary Table [26](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)).

**Table 4 MVMR analysis of All-PA–leisure setting BMI as exposures**

#### All-PA–leisure and home and All-PA–leisure, home and work phenotypes

To evaluate the potential gain in information from combining data from PA during leisure time with PA performed at home and work, we conducted two additional GWAS in the EUR population. First, we summed All-PA–leisure with All-PA from PA at home (All-PA–leisure and home; [Methods](https://www.nature.com/articles/s41588-025-02260-9#Sec20); Supplementary Fig. [11](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). We next added information from All-PA at work (All-PA–leisure, home and work; [Methods](https://www.nature.com/articles/s41588-025-02260-9#Sec20); Supplementary Fig. [12](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). In both these GWASs, we observed a substantial reduction of the number of association peaks compared to All-PA–leisure.

### GWAS of vigorous PA phenotypes (leisure, at home and at work)

We ran analyses on different contexts of vigorous PA to understand genetic differences and benefits related to the context in which PA was performed.

#### Kinds of vigorous PA and their genetic interrelationships

We ran independent MVP GWAS of all vigorous activity types depending on the context (Vig-PA–leisure, Vig-PA–home and Vig-PA–work) in EUR and investigated correlations between each set of two to study their genetic differences. For Vig-PA–leisure and Vig-PA–home, _r__g_ = 0.64 ± 0.04 (_P_ = 5.0 × 10−59); Vig-PA–leisure and Vig-PA–work, _r__g_ = 0.46 ± 0.07 (_P_ = 9.8 × 10−11). Vig-PA–home and Vig-PA–work were more strongly correlated with each other (_r__g_ = 0.97 ± 0.05, _P_ = 1.2 × 10−73). SNP-_h_2 of vigorous PA phenotypes are reported in Supplementary Table [27](https://www.nature.com/articles/s41588-025-02260-9#MOESM3).

For each vigorous PA trait, we ran GWAS, and for Vig-PA–leisure we also conducted X chromosome-wide association study (XWAS), gene-based testing, TWAS, fine-mapping, functional enrichment, mitochondria–SNV association and gene-based association analysis on the mitochondrial genome ([Supplementary Results](https://www.nature.com/articles/s41588-025-02260-9#MOESM1); Supplementary Tables [28](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)–[35](https://www.nature.com/articles/s41588-025-02260-9#MOESM3) and Supplementary Figs. [13](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)–[17](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)).

#### Genetic correlations of vigorous PA traits

To understand the differences between the vigorous activity traits better, we calculated the genetic correlations between Vig-PA–leisure, Vig-PA–work and Vig-PA–home individually with the previously analyzed traits of interest (Supplementary Tables [36](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)–[38](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). We observed more statistically significant differences between Vig-PA–leisure and Vig-PA–work (for 15 traits), than when considering Vig-PA–leisure and Vig-PA–home (for 12 traits) or Vig-PA–home and Vig-PA–work (for 3 traits; Fig. [3a](https://www.nature.com/articles/s41588-025-02260-9#Fig3)). Among these three pairs of vigorous activity traits, there were numerous significant differences regarding genetic correlations with income, leisure screen time and educational attainment, reporting better health benefits from Vig-PA–leisure compared to the other two vigorous traits. Analyses performed adjusting the vigorous PA traits with income using mtCOJO are described in [Supplementary Results](https://www.nature.com/articles/s41588-025-02260-9#MOESM1), Supplementary Table [39](https://www.nature.com/articles/s41588-025-02260-9#MOESM3) and Supplementary Fig. [18](https://www.nature.com/articles/s41588-025-02260-9#MOESM1).

**Fig. 3: Genetic differences among vigorous PA types.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02260-9/MediaObjects/41588_2025_2260_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02260-9/figures/3)

**a**, Genetic correlation between each pair of vigorous PA contexts and other traits of interest. Significantly different correlations were defined as having _P_ < 4.63 × 10−4 after multiple testing correction. An asterisk represents a significant difference between Vig-PA–leisure and Vig-PA–work, wedge represents a significant difference between Vig-PA–leisure and Vig-PA–home, and plus represents a significant difference between Vig-PA–home and Vig-PA–work. Tests were two-sided. Black error bars represent the s.e. The genetic correlation values of the vigorous PA with the other traits of interest are provided in Supplementary Tables [36](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)–[38](https://www.nature.com/articles/s41588-025-02260-9#MOESM3). **b**, Genomic-SEM of the two-factor model, showing the loadings for the inferred traits—their standardized estimates with the s.e. in parentheses. The CFA uses a chi-square test.

#### Genomic structural equation modeling (genomic-SEM) analysis

Considering the differences identified among the three vigorous PA traits, which had shown different benefits according to the context, we used genomic-SEM analysis[21](https://www.nature.com/articles/s41588-025-02260-9#ref-CR21 "Grotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nat. Hum. Behav. 3, 513–525 (2019).") to understand their relationship better, inferring the overall genetic architecture among them and two other PA traits in EUR, All-PA–leisure + SSOE and liking of PA. The exploratory factor analysis (EFA) indicated that a two-factor model fit the data best (Fig. [3b](https://www.nature.com/articles/s41588-025-02260-9#Fig3)), accounting for 91% of the cumulative variance explained (factor 1, 50%; factor 2, 41%). Both factors had strong strength of sum of squares (SS) loadings (factor 1 SS = 2.50; factor 2 SS = 2.04). Confirmatory factor analysis (CFA) showed that two correlated factors model fit the data well, with comparative fit index = 0.97, chi-square = 109.03, Akaike information criterion = 129.03 and standardized root mean square residual = 0.069. Vig-PA–leisure (loading = 1.00 ± 0.03), All-PA–leisure + SSOE (loading = 0.89 ± 0.03) and liking of PA (loading = 0.79 ± 0.04) loaded most strongly on factor 1 (leisure/liking PA), while Vig-PA–home (loading = 1.02 ± 0.07) and Vig-PA–work (loading = 0.82 ± 0.08) loaded most strongly on factor 2 (occupational/domestic PA; Fig. [3b](https://www.nature.com/articles/s41588-025-02260-9#Fig3)). The factor leisure/liking PA and factor occupational/domestic PA were correlated at 0.50 (0.04).

#### Survival analyses

Survival analyses showed a significant reduction of risk of all-cause death for Vig-PA–leisure (_β_ = −0.1173 ± 0.0037; _P_ ≤ 2 × 10−16) and for Vig-PA–work (_β_ = −0.0061 ± 0.0029; _P_ = 0.036). There was no statistically significant reduction of risk of death for Vig-PA–home, though the direction of effect and effect size were similar (_β_ = −0.0052 ± 0.0030; _P_ = 0.082).

Discussion
----------

PA has long been appreciated to ameliorate the risk of illness. Increasing PA on a population level could greatly reduce morbidity and mortality—this is a major public health undertaking and very difficult to accomplish. Understanding genetic factors that influence the level of activity for individuals could provide pathways for interventions to support efforts to increase PA. We present here comprehensive information on the genetics of PA and investigate the relationships between PA and health. This work provides tools to dissect causal relationships between a range of other traits and PA, and to uncover previously unknown biology responsible in part for interindividual differences in PA. Those relationships were remarkably different depending on the context of the PA—in particular, whether the activity was associated with PA during leisure time (protective) rather than at work or around the house (often not protective). Our main focus was accordingly on leisure-PA. All-PA–leisure considered different levels of PA intensity and frequency. We completed a GWAS of All-PA–leisure for EUR, AFR and AMR ancestry in MVP. We combined these data with the UKB SSOE trait in two meta-analyses, one with EUR individuals and the other cross-ancestry.

The polygenic architecture of leisure-time PA showed that genetic liability to PA was protective with respect to numerous diseases. It was previously reported that PA has negative genetic correlation with type 2 diabetes[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."),[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022)."),[22](https://www.nature.com/articles/s41588-025-02260-9#ref-CR22 "Carlsson, S., Ahlbom, A., Lichtenstein, P. & Andersson, T. Shared genetic influence of BMI, physical activity and type 2 diabetes: a twin study. Diabetologia 56, 1031–1035 (2013)."), triglyceride level[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."), rheumatoid arthritis[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022)."), coronary artery disease[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022).") and lung cancer[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022)."). There is prior genetic evidence of the role of PA in preventing colorectal cancer using MR methods[23](https://www.nature.com/articles/s41588-025-02260-9#ref-CR23 "Zhang, X. et al. Genetically predicted physical activity levels are associated with lower colorectal cancer risk: a Mendelian randomisation study. Br. J. Cancer 124, 1330–1338 (2021)."), but we did not find a significant genetic correlation with colorectal cancer. Our analyses confirmed the negative genetic correlations with type 2 diabetes, coronary artery disease and lung cancer. Moreover, we found significant negative genetic correlations between PA and other medically relevant traits—thoracic and abdominal aortic aneurysm, heart failure, stroke, asthma, type 1 diabetes, Alzheimer’s disease, inflammatory bowel disease, osteoarthritis and gastroesophageal reflux disease. PA was also protective with respect to COVID-19 and COVID-19-associated hospitalization, and positively correlated with parental survival.

Whereas genetic correlations are an average of the shared association across the genome, local genetic correlations can detect regions of shared genetic associations, including finding signals in opposing directions that a genome-wide analysis would not capture. Local genetic correlation, like global correlation, supported a protective role of PA for multiple health traits, showing negative local genetic correlations for diabetes types 1 and 2, gastroesophageal reflux disease and osteoarthritis. We also found negative local genetic correlations between All-PA–leisure + SSOE and BMI, triglycerides, phenotypic age acceleration, leisure screen time, neuroticism and cigarettes per day. Positive local genetic correlations were found between All-PA–leisure + SSOE and bone mineral density, income, liking of PA, educational attainment, and, unexpectedly, oral cancer—there was a single locus showing positive local genetic correlation between PA and oral cancer; however, genome-wide genetic correlation between these traits calculated with LDSC was not significant (_P_ = 0.418). We also found local genetic correlations in opposite directions—at different genomic locations—with All-PA–leisure + SSOE for height and well being.

Our MR results confirmed a protective effect of All-PA–leisure genetic liability against type 2 diabetes, abdominal aortic aneurysm, heart failure, COVID-19-associated hospitalization, triglyceride level, osteoarthritis and gastroesophageal reflux disease. Most of these causal effects were observed even when we tested for BMI as a confounder in the MR (MVMR; except for heart failure, which could not be tested due to sample overlap between the BMI and heart failure datasets). However, the low number of instrumental variables in our MVMR analyses could result in false negatives. We tested BMI as confounder since it is genetically correlated to numerous diseases and a previous study showed that it is likely a mediator of leisure time PA, as its causal effect on leisure time PA disappears when taking education into account[24](https://www.nature.com/articles/s41588-025-02260-9#ref-CR24 "Wang, Z., Davey Smith, G., Loos, R. J. F. & den Hoed, M. Distilling causality between physical activity traits and obesity via Mendelian randomization. Commun. Med. 3, 173 (2023)."). These findings nevertheless confirm the utility of All-PA–leisure to decrease risk for numerous diseases[25](https://www.nature.com/articles/s41588-025-02260-9#ref-CR25 "Guthold, R., Stevens, G. A., Riley, L. M. & Bull, F. C. Worldwide trends in insufficient physical activity from 2001 to 2016: a pooled analysis of 358 population-based surveys with 1.9 million participants. Lancet Glob. Health 6, e1077–e1086 (2018)."),[26](https://www.nature.com/articles/s41588-025-02260-9#ref-CR26 "Guthold, R., Stevens, G. A., Riley, L. M. & Bull, F. C. Global trends in insufficient physical activity among adolescents: a pooled analysis of 298 population-based surveys with 1·6 million participants. Lancet Child Adolesc. Health 4, 23–35 (2020)."), independent of the role of BMI as either a mediator or a confounder. MR results also suggested a protective effect of All-PA–leisure on the biological age measure phenotypic age acceleration, and a bidirectional effect between All-PA–leisure and parental survival, a protective effect of genetic liability for this PA phenotype versus aging and mortality.

PheWAS analysis in BioVU[27](https://www.nature.com/articles/s41588-025-02260-9#ref-CR27 "Roden, D. M. et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin. Pharmacol. Ther. 84, 362–369 (2008)."), a broad exploration of disease traits in an independent sample, showed a negative correlation of PA with several previously mentioned health traits, including diabetes, cardiovascular disease (for example, coronary artery disease and heart failure), gastroesophageal reflux disease, lung cancer and asthma. The strongest association was related to diabetes (_P_ = 5.89 × 10−36).

The EUR meta-analysis of All-PA–leisure + SSOE identified 67 lead SNPs, 59 of which were novel compared to the lead SNPs of the two GWASs, All-PA–leisure and SSOE, in the meta-analysis. Previous GWAS of self-report and device-measured PA identified 68 lead variants in total[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."),[14](https://www.nature.com/articles/s41588-025-02260-9#ref-CR14 "Doherty, A. et al. GWAS identifies 14 loci for device-measured physical activity and sleep duration. Nat. Commun. 9, 5257 (2018)."),[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022)."). We replicated associations for 17 previously identified lead variants. Two (_CADM2_\*[rs62253088](https://www.ncbi.nlm.nih.gov/snp/?term=rs62253088) and LOC642366\*[rs4865656](https://www.ncbi.nlm.nih.gov/snp/?term=rs4865656)) were still lead variants in both our meta-analyses of All-PA–leisure + SSOE (EUR ancestry and cross-ancestry); the other 15 were significantly associated (Supplementary Table [40](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)). _CADM2_\*[rs62253088](https://www.ncbi.nlm.nih.gov/snp/?term=rs62253088) (encoding _Cell Adhesion Molecule 2_), on chromosome 3, had the strongest association in both EUR and cross-ancestry All-PA–leisure + SSOE meta-analyses (EUR meta-analysis _P_ = 2.0 × 10−21; cross-ancestry (EUR, AFR and AMR) meta-analysis _P_ = 6.1 × 10−20), but was not significant in the GWAS of Vig-PA–leisure. However, considering Vig-PA–leisure, another variant mapping to _CADM2_ was a significant lead SNP in the GWAS ([rs2326266](https://www.ncbi.nlm.nih.gov/snp/?term=rs2326266); _P_ = 4.1 × 10−8) and _CADM2_ also had the most significant _P_ value in the gene-based test (2.5 × 10−8). _CADM2_ variants, including [rs62253088](https://www.ncbi.nlm.nih.gov/snp/?term=rs62253088), have previously been associated with PA, and also with decreased neuroticism and decreased self-reported nervous and anxious feelings[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."). _CADM2_ was also previously found to be associated with obesity and metabolic traits[28](https://www.nature.com/articles/s41588-025-02260-9#ref-CR28 "Morris, J. et al. Genetic variation in CADM2 as a link between psychological traits and obesity. Sci. Rep. 9, 7339 (2019)."), cannabis use[29](https://www.nature.com/articles/s41588-025-02260-9#ref-CR29 "Pasman, J. A. et al. GWAS of lifetime cannabis use reveals new risk loci, genetic overlap with psychiatric traits, and a causal effect of schizophrenia liability. Nat. Neurosci. 21, 1161–1170 (2018)."), smoking and alcohol related traits[30](https://www.nature.com/articles/s41588-025-02260-9#ref-CR30 "Liu, M. et al. Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use. Nat. Genet. 51, 237–244 (2019)."), risk-taking behavior[31](https://www.nature.com/articles/s41588-025-02260-9#ref-CR31 "Strawbridge, R. J. et al. Genome-wide analysis of self-reported risk-taking behaviour and cross-disorder genetic correlations in the UK Biobank cohort. Transl. Psychiatry 8, 39 (2018)."), educational attainment[32](https://www.nature.com/articles/s41588-025-02260-9#ref-CR32 "Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. Nat. Genet. 50, 1112–1121 (2018).") and autism spectrum disorder[33](https://www.nature.com/articles/s41588-025-02260-9#ref-CR33 "Casey, J. P. et al. A novel approach of homozygous haplotype sharing identifies candidate genes in autism spectrum disorder. Hum. Genet. 131, 565–579 (2012)."). _CADM2_ encodes a member of the synaptic CADM (SynCAM) family, which belongs to the immunoglobulin superfamily[34](https://www.nature.com/articles/s41588-025-02260-9#ref-CR34 "Šimunić-Briški, N. et al. Physical exercise induces significant changes in immunoglobulin G N-glycan composition in a previously inactive, overweight population. Biomolecules 13, 762 (2023)."). Through a previous summary-based MR[35](https://www.nature.com/articles/s41588-025-02260-9#ref-CR35 "Zhu, Z. et al. Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. Nat. Genet. 48, 481–487 (2016).") analysis of eQTL signals, _CADM2_ was suggested to influence PA through effects on skeletal muscle[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022).").

Significant variants also included _MST1R_\*[rs3733134](https://www.ncbi.nlm.nih.gov/snp/?term=rs3733134) (EUR meta-analysis _P_ = 1.4 × 10−17; cross-ancestry meta-analysis _P_ = 6.9 × 10−18), _EXD2_\*[rs4899292](https://www.ncbi.nlm.nih.gov/snp/?term=rs4899292) (EUR meta-analysis _P_ = 1.7 × 10−16; cross-ancestry meta-analysis _P_ = 3.8 × 10−16), _LONRF2_ and _SLC39A8_. _SLC39A8_ was previously associated with several psychiatric disorders, including problematic alcohol use[36](https://www.nature.com/articles/s41588-025-02260-9#ref-CR36 "Gaudino, G. et al. The proto-oncogene RON is involved in development of epithelial, bone and neuro-endocrine tissues. Oncogene 11, 2627–2637 (1995)."), schizophrenia[37](https://www.nature.com/articles/s41588-025-02260-9#ref-CR37 "Nieminuszczy, J. et al. EXD2 protects stressed replication forks and is required for cell viability in the absence of BRCA1/2. Mol. Cell 75, 605–619 (2019).") and opioid use disorder[38](https://www.nature.com/articles/s41588-025-02260-9#ref-CR38 "Zhou, H. et al. Genome-wide meta-analysis of problematic alcohol use in 435,563 individuals yields insights into biology and relationships with other traits. Nat. Neurosci. 23, 809–818 (2020).") ([Supplementary Discussion](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)).

Additional analyses demonstrated that PA is related to brain structure and cellular architecture. From MAGMA tissue expression analyses, the cerebellum and cerebellar hemisphere showed significant enrichments. Cerebellar function is critical for skills relevant to PA, such as coordination and movement. These results were similar to the results of the previous GWAS of SSOE[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."). Cell-type-specificity analyses showed a significant association of PA with GABA+ neurons[39](https://www.nature.com/articles/s41588-025-02260-9#ref-CR39 "Pardiñas, A. F. et al. Common schizophrenia alleles are enriched in mutation-intolerant genes and in regions under strong background selection. Nat. Genet. 50, 381–389 (2018)."). EUR All-PA–leisure + SSOE and cross-ancestry meta-analyses shared the following three significant gene sets in their MAGMA gene-set analysis results: structural constituent of presynapse, presynaptic active zone organization, and maintenance of presynaptic active zone structure. This finding highlights the role of presynaptic cells in PA and the importance of the cerebellum and GABAergic function in the biology of PA levels.

Vig-PA–leisure was, as noted, genetically different from Vig-PA–home and Vig-PA–work. We observed moderate genetic correlation between Vig-PA–leisure and Vig-PA–home (_r__g_ = 0.64 ± 0.04, _P_ = 5.0 × 10−59), but lower genetic correlation between Vig-PA–leisure and Vig-PA–work (_r__g_ = 0.46 ± 0.07, _P_ = 9.8 × 10−11). While there was high genetic correlation between Vig-PA–home and Vig-PA–work (_r__g_ = 0.97 ± 0.05, _P_ = 1.2 × 10−73), there were differences between these two traits in genetic correlation analyses with other traits of interest, in particular for income, educational attainment and leisure screen time. Previous work on dementia also showed genetic differences between PA during leisure versus work time[40](https://www.nature.com/articles/s41588-025-02260-9#ref-CR40 "Deak, J. D. et al. Genome-wide association study in individuals of European and African ancestry and multi-trait analysis of opioid use disorder identifies 19 independent genome-wide significant risk loci. Mol. Psychiatry 10, 3970–3979 (2022).").

The genomic-SEM results also supported that context-defined PA traits are genetically heterogeneous. Traits more similar to voluntary PA and healthy lifestyle habits loaded on the leisure/liking PA Factor, while the occupational/domestic PA Factor captured traits more consistent with strenuous work and labor performed as part of work duties and responsibilities at home.

While the leisure/liking PA and occupational/domestic PA factors were moderately correlated (0.50), the nature of the respective contexts of PA identified key differences. Perhaps the most notable distinction is that strenuous occupational and domestic PA is often not health-enhancing, and can be detrimental to health via injury and long-term stress on bodily systems[41](https://www.nature.com/articles/s41588-025-02260-9#ref-CR41 "Akiyama, M. et al. Genome-wide association study identifies 112 new loci for body mass index in the Japanese population. Nat. Genet. 49, 1458–1467 (2017)."),[42](https://www.nature.com/articles/s41588-025-02260-9#ref-CR42 "Pulit, S. L. et al. Meta-analysis of genome-wide association studies for body fat distribution in 694,649 individuals of European ancestry. Hum. Mol. Genet 28, 166–174 (2019)."). Individuals whose vocations are labor-intensive are also less likely to engage in health-promoting PA in leisure[43](https://www.nature.com/articles/s41588-025-02260-9#ref-CR43 "Vujkovic, M. et al. Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis. Nat. Genet. 52, 680–691 (2020)."), further compounding their risk for adverse health outcomes long-term.

When we adjusted for vigorous PA traits and income using mtCOJO, we still observed differences in genetic correlations with health traits, suggesting that these differences do not merely reflect differences in socioeconomic status. These findings together suggest that not all contexts of PA are equally beneficial, and some forms of vigorous activity may be more strongly linked than others to improved mental and physical health.

A further confirmation of differing benefits by PA context was provided by the survival analyses, which show a strong significant reduction of risk of death for Vig-PA–leisure (_β_ = −0.1173 ± 0.0037, _P_ ≤ 2 × 10−16) compared to a weak effect for Vig-PA–work (_β_ = −0.0061 ± 0.0029, _P_ = 0.036), and none for for Vig-PA–home (_β_ = −0.0052 ± 0.0030, _P_ = 0.082). The small protective effect for Vig-PA–work contrasts with the opposing directions for risk for several other important health traits. The survival analyses may be confounded by health status (healthier individuals are able to exercise more vigorously) and competing risks (death). A previous study on different kinds of PA during leisure time—for example, weightlifting and aerobic exercise—showed, consistent with the present report, that PA in the leisure context can lead to a longer and healthier life[44](https://www.nature.com/articles/s41588-025-02260-9#ref-CR44 "Maddock, R. J., Casazza, G. A., Fernandez, D. H. & Maddock, M. I. Acute modulation of cortical glutamate and GABA content by physical activity. J. Neurosci. 36, 2449–2457 (2016)."). We conclude that PA at leisure is the most protective with respect to deleterious medical outcomes.

This study has limitations, including a relative lack of females in the MVP dataset (approximately 9%, which nevertheless equates to >17,000 females for Vig-PA–leisure); this makes the interpretation of sex-specific associations infeasible. UKB, though, has excellent female representation. MVP is also exclusively based on military veterans, not fully representative of the general population. However, the high genetic correlation (_r__g_ = 0.76 ± 0.03, _P_ = 4.7 × 10−118) between MVP data (All-PA–leisure) and SSOE in UKB, a more representative cohort, may also be considered in this context. Moreover, while there are many non-EUR participants included, their numbers are still insufficient, so the cross-ancestry analysis mostly reflects findings in EUR. It would also be important to verify the tissue expression analyses, which highlight only significant associations with the brain, with more tissues and larger samples. Our results may reflect a lack of power for other tissues or our sample size—of the 53 tissue types contained in Genotype-Tissue Expression version 8 (GTEx v8), 13 are brain-specific.

In conclusion, we have made progress towards revealing the genetic and biological nature of PA, which can differ greatly depending on the context. In effect, the PA that one chooses to engage in in a leisure context has many positive effects with respect to health traits, while activity that one must do, in a work or household context, tends not to have these beneficial effects. This difference persisted even after we adjusted for income. Moreover, PA in the leisure context shows a much stronger decrease in the risk of death compared to the other contexts. This may reflect the differences in intensity and duration of leisure activities compared to work and home activities. We detected a protective role of PA genetic liability on several diseases, using global and local genetic correlation, PheWAS analyses and causal inference using MR. MR analyses showed significant protective effects of PA on several traits, including type 2 diabetes, abdominal aortic aneurysm, heart failure, triglyceride level, osteoarthritis, gastroesophageal reflux disease and COVID-19-associated hospitalization. These results provide genetic insights into the biological bases of PA and add to strong experimental, epidemiological and observational literature showing the value to health of PA.

Methods
-------

### MVP datasets

The U.S. Department of Veterans Affairs (VA) MVP is one of the largest and most diverse biobanks in the world, with genetic and electronic health record (EHR) data available[45](https://www.nature.com/articles/s41588-025-02260-9#ref-CR45 "Gaziano, J. M. et al. Million Veteran Program: a mega-biobank to study genetic influences on health and disease. J. Clin. Epidemiol. 70, 214–223 (2016)."),[46](https://www.nature.com/articles/s41588-025-02260-9#ref-CR46 "Harrington, K. M. et al. Gender differences in demographic and health characteristics of the Million Veteran Program cohort. Womens Health Issues 29, S56–S66 (2019)."). Ethical approval for the MVP study was obtained from the Central VA Institutional Review Board and the site-specific institutional review boards. All relevant ethical regulations for work with human participants were followed in the conduct of the study, and informed consent was obtained from all participants.

Participants provided a blood sample for genomic analyses, granted access to medical records, and a subset agreed to complete two questionnaires, the MVP Baseline and Lifestyle Surveys. PA information used in this study was collected from the MVP Lifestyle Survey, and data were subdivided according to context (leisure, work and home), intensity (vigorous, moderate and light) and frequency (Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). The three levels of activity were defined as follows:

1.  1.
    
    Vigorous—activities that cause your heart to beat rapidly and you work up a good sweat, and are breathing heavily; performed at least 10 min at a time.
    
2.  2.
    
    Moderate—activities that cause your heart rate to increase slightly and you typically work up a sweat, but are not physically exhausting; performed for at least 10 min at a time.
    
3.  3.
    
    Light—activities that require little physical effort.
    

The levels of activity were considered together with frequency information, which could be ‘daily’, ‘several times per week’, ‘once per week’, ‘several times per month’, ‘once per month or less’ or ‘never’, when participants answered three questions regarding the place where PA was effected ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). The three different levels of PA were summed considering All-PA phenotype frequency and weighting intensity to create a total score, reported in the [Supplementary Methods](https://www.nature.com/articles/s41588-025-02260-9#MOESM1).

From previous MVP analyses, after MVP genotyping performed using a customized Affymetrix Axiom Biobank Array and quality control[47](https://www.nature.com/articles/s41588-025-02260-9#ref-CR47 "Hunter-Zinck, H. et al. Genotyping array design and data quality control in the Million Veteran Program. Am. J. Hum. Genet. 106, 535–548 (2020)."), EAGLE2 (ref. [48](https://www.nature.com/articles/s41588-025-02260-9#ref-CR48 "Loh, P.-R. et al. Reference-based phasing using the Haplotype Reference Consortium panel. Nat. Genet. 48, 1443–1448 (2016).")) was used for phasing chromosomes and Minimac3 was used for imputation[49](https://www.nature.com/articles/s41588-025-02260-9#ref-CR49 "Das, S. et al. Next-generation genotype imputation service and methods. Nat. Genet. 48, 1284–1287 (2016)."), using the 1000 Genomes Project reference panel, Phase 3, version 5 (ref. [50](https://www.nature.com/articles/s41588-025-02260-9#ref-CR50 "Auton, A. et al. A global reference for human genetic variation. Nature 526, 68–74 (2015).")). Populations were defined using principal component analysis[45](https://www.nature.com/articles/s41588-025-02260-9#ref-CR45 "Gaziano, J. M. et al. Million Veteran Program: a mega-biobank to study genetic influences on health and disease. J. Clin. Epidemiol. 70, 214–223 (2016).").

For All-PA during leisure time (All-PA–leisure), we ran independent GWAS for each of the three ancestries (EUR, AFR and AMR; Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). For EUR, we also ran GWAS for vigorous PA during leisure time (Vig-PA–leisure), work time (Vig-PA–work) and home time (Vig-PA–home). We did not have enough power to run GWAS for vigorous PA for AFR and AMR ancestries.

In the quality control procedure for GWAS conducted using PLINK 2.0 (ref. [51](https://www.nature.com/articles/s41588-025-02260-9#ref-CR51 "Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaScience 4, 7 (2015).")), we removed variants with imputation quality scores < 0.6, Hardy–Weinberg equilibrium _P_ < 5 × 10−5, minor allele frequency <0.01, missing call rates for variants >0.1 and missing call rates for samples >0.1. Data were aligned to the GRCh37 reference genome. Considering EUR ancestry, after filtering, we had 6,660,206 variants for All-PA–leisure phenotype, and 6,548,052, 6,536,056, 6,537,158 variants for Vig-PA–leisure, Vig-PA–home and Vig-PA–work, respectively. For AFR and AMR ancestries, we kept 11,984,943 and 7,969,366 variants, respectively (All-PA–leisure phenotype).

To remove related individuals, we used a threshold of 0.0884 for the kinship coefficients calculated by KING[52](https://www.nature.com/articles/s41588-025-02260-9#ref-CR52 "Manichaikul, A. et al. Robust relationship inference in genome-wide association studies. Bioinformatics 26, 2867–2873 (2010)."), resulting in the removal of individuals with a minimum of a second-degree relationship. Then, we implemented an algorithm to optimize retaining the more informative individuals, keeping the maximum number of individuals with the highest score (for All-PA–leisure phenotype) or with the highest frequency (for the vigorous activity phenotypes). If two individuals had the same score (or frequency), we removed the one with the highest number of relationships. For EUR, the final sample sizes were as follows: 189,812 individuals for the All-PA–leisure phenotype, 201,050 for Vig-PA–leisure, 203,430 for Vig-PA–home and 171,278 for Vig-PA–work. For AFR and AMR ancestries, we kept 27,044 and 10,263 individuals, respectively (All-PA–leisure phenotype). We ran GWAS analysis using a linear regression model implemented in PLINK 2.0, using sex, age and the first ten principal components as covariates.

For All-PA–leisure&home phenotype and for All-PA–leisure&home&work, we ran the same QC as described in the [Methods](https://www.nature.com/articles/s41588-025-02260-9#Sec20), using EUR ancestry. After filtering, we had 6,678,873 variants for All-PA–leisure&home, and 6,761,070 variants for All-PA–leisure&home&work. For the final sample size, where we excluded individuals without full information on their PA, we retained 181,317 individuals for All-PA–leisure&home, and 146,929 for All-PA–leisure&home&work.

### UKB summary statistics

UKB is a cohort study of ~500,000 adults aged 40–69 years in the United Kingdom (UK) and was recruited from 22 centers across the country[53](https://www.nature.com/articles/s41588-025-02260-9#ref-CR53 "Sudlow, C. et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)."). All UKB participants provided written informed consent[53](https://www.nature.com/articles/s41588-025-02260-9#ref-CR53 "Sudlow, C. et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)."). Ethical approval of the UKB study was given by the North West Multicentre Research Ethics Committee, the National Information Governance Board for Health and Social Care and the Community Health Index Advisory Group[53](https://www.nature.com/articles/s41588-025-02260-9#ref-CR53 "Sudlow, C. et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015).").

From the UKB public databases, we downloaded GWAS summary statistics for the SSOE phenotype with the accession code GCST006100, for MVPA with the accession code GCST006097 and for VPA with the accession code GCST006098 (ref. [13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018).")). Self-reported information was reported via a touchscreen questionnaire. For SSOE, participants needed to answer the following question: ‘in the last 4 weeks, did you spend any time doing the following?’. Then, there were follow-up questions assessing the frequency and typical duration of ‘strenuous sports’ and of ‘other exercises’. Cases were defined as individuals spending 2–3 days per week or more doing strenuous sports or other exercises for a duration of 15–30 min or greater, and controls as those individuals who did not indicate spending any time in the last 4 weeks doing either SSOE, obtaining 124,842 cases (_n_case) and 225,650 controls (_n_control)[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."). We defined the population effective size as _n_eff = 4/((1/_n_case) + (1/_n_control)).

The downloaded summary statistics had SNPs filtered according to—Hardy–Weinberg equilibrium _P_ < 10−6, high missingness >1.5%, low minor allele frequency <0.1%[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."). We also filtered low imputation quality <0.6, yielding approximately 11.7 million available SNPs.

### Meta-analyses

METAL[54](https://www.nature.com/articles/s41588-025-02260-9#ref-CR54 "Willer, C. J., Li, Y. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics 26, 2190–2191 (2010).") was used for EUR ancestry meta-analysis, which included EUR MVP data for the All-PA–leisure phenotype and UKB summary statistics for the SSOE phenotype[13](https://www.nature.com/articles/s41588-025-02260-9#ref-CR13 "Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. Int. J. Obes. 42, 1161–1176 (2018)."), in an All-PA–leisure + SSOE meta-analysis. We also ran a cross-ancestry All-PA–leisure + SSOE meta-analysis, adding AFR and AMR summary statistics from MVP data to the EUR All-PA–leisure + SSOE meta-analysis (Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)). We conducted the meta-analyses using the sample-size method, which considers _P_ value and direction of effect, weighted according to sample size. We used this method to combine summary statistics from a logistic regression model (UKB summary statistics for SSOE phenotype) and a linear regression model (MVP summary statistics for All-PA–leisure phenotype).

### XWAS

Sex-stratified analysis of the X chromosome was conducted in the EUR MVP sample using XWAS 3.0 on hard-call genotypes[55](https://www.nature.com/articles/s41588-025-02260-9#ref-CR55 "Gao, F. et al. XWAS: a software toolset for genetic data analysis and association studies of the X chromosome. J. Hered. 106, 666–671 (2015)."). We included 174,375 males and 15,130 females for the All-PA–leisure phenotype and 229,331 males and 17,072 females for Vig-PA–leisure. Following sex-stratified analysis, test statistics from males and females were combined using Stouffer’s method[56](https://www.nature.com/articles/s41588-025-02260-9#ref-CR56 "Stouffer, S. A. A study of attitudes. Sci. Am. 180, 11–15 (1949)."). Variants with Hardy–Weinberg equilibrium _P_ < 5 × 10−6 were filtered; 191,114 variants were included for the All-PA–leisure phenotype and 191,148 for Vig-PA–leisure. We used age and the first ten principal components as covariates.

### Gene-based and gene-set analyses, functional enrichment

Using the tool MAGMA[57](https://www.nature.com/articles/s41588-025-02260-9#ref-CR57 "De Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. PLoS Comput. Biol. 11, e1004219 (2015)."), implemented in the FUMA web-based platform[58](https://www.nature.com/articles/s41588-025-02260-9#ref-CR58 "Watanabe, K., Taskesen, E., van Bochoven, A. & Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 8, 1826 (2017)."), we annotated variants and ran gene-based tests, gene-set analyses and functional enrichment for each of the two meta-analyses (EUR ancestry and cross-ancestry) for the All-PA–leisure phenotype and for the GWAS of Vig-PA–leisure. For annotation, we used 1000 Genome Phase 3 EUR for EUR ancestry analyses and 1000 Genome Phase 3 ALL for the cross-ancestry All-PA–leisure+SSOE meta-analysis. The other parameters for the identification of lead SNPs were a maximum cutoff _P_ < 0.05, _r_2 threshold to define independent significant SNPs >0.6, a second _r_2 threshold to define lead SNPs >0.1 and a maximum distance between linkage disequilibrium (LD) blocks to merge into a locus <250 kb.

The MAGMA gene-based test is based on a multiple regression approach to detect gene effects, considering SNP _P_ values and linkage disequilibrium. We also ran a gene-based approach on our EUR All-PA–leisure + SSOE meta-analysis using fastENLOC[17](https://www.nature.com/articles/s41588-025-02260-9#ref-CR17 "Wen, X., Pique-Regi, R. & Luca, F. Integrating molecular QTL data into genome-wide genetic association analysis: probabilistic assessment of enrichment and colocalization. PLoS Genet. 13, e1006646 (2017)."),[18](https://www.nature.com/articles/s41588-025-02260-9#ref-CR18 "Hukku, A., Sampson, M. G., Luca, F., Pique-Regi, R. & Wen, X. Analyzing and reconciling colocalization and transcriptome-wide association studies from the perspective of inferential reproducibility. Am. J. Hum. Genet. 109, 825–837 (2022).") (fast enrichment estimation aided colocalization analysis). This method was used to prioritize likely causal gene–trait associations with precomputed eQTL annotations from the 49 tissues in the GTEx v8 dataset. We kept as ‘noteworthy genes’ those with GLCP ≥ 0.9; if a gene was colocalizing among two or more tissues, we retained the tissue with the highest GLCP value.

Gene-set analyses are performed for curated gene sets and GO terms obtained from MsigDB, with terms having a Bonferroni-corrected _P_ < 0.05 considered significant[57](https://www.nature.com/articles/s41588-025-02260-9#ref-CR57 "De Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. PLoS Comput. Biol. 11, e1004219 (2015)."). Gene property analyses were performed for tissue-specific gene expression using GTEx v8 dataset[59](https://www.nature.com/articles/s41588-025-02260-9#ref-CR59 "Aguet, F. et al. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science 369, 1318–1330 (2020).").

### Single-cell expression

FUMA also includes a tool to test cell-type-specificity analysis[60](https://www.nature.com/articles/s41588-025-02260-9#ref-CR60 "Watanabe, K., Umićević Mirkov, M., de Leeuw, C. A., van den Heuvel, M. P. & Posthuma, D. Genetic mapping of cell type specificity for complex traits. Nat. Commun. 10, 3222 (2019)."), which was run using the following datasets of human samples: Allen Brain Atlas Cell Type[61](https://www.nature.com/articles/s41588-025-02260-9#ref-CR61 "Hodge, R. D. et al. Conserved cell types with divergent features in human versus mouse cortex. Nature 573, 61–68 (2019)."), DroNc[62](https://www.nature.com/articles/s41588-025-02260-9#ref-CR62 "Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat. Methods 14, 955–958 (2017)."), GSE104276 (ref. [63](https://www.nature.com/articles/s41588-025-02260-9#ref-CR63 "Zhong, S. et al. A single-cell RNA-seq survey of the developmental landscape of the human prefrontal cortex. Nature 555, 524–528 (2018).")), [GSE67835](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835) without fetal samples[64](https://www.nature.com/articles/s41588-025-02260-9#ref-CR64 "Darmanis, S. et al. A survey of human brain transcriptome diversity at the single cell level. Proc. Natl Acad. Sci. USA 112, 7285–7290 (2015)."), GSE81547 (ref. [65](https://www.nature.com/articles/s41588-025-02260-9#ref-CR65 "Enge, M. et al. Single-cell analysis of human pancreas reveals transcriptional signatures of aging and somatic mutation patterns. Cell 171, 321–330 (2017).")), GSE84133 (ref. [66](https://www.nature.com/articles/s41588-025-02260-9#ref-CR66 "Baron, M. et al. A single-cell transcriptomic map of the human and mouse pancreas reveals inter- and intra-cell population structure. Cell Syst. 3, 346–360 (2016).")), GSE89232 (ref. [67](https://www.nature.com/articles/s41588-025-02260-9#ref-CR67 "Breton, G. et al. Human dendritic cells (DCs) are derived from distinct circulating precursors that are precommitted to become CD1c+ or CD141+ DCs. J. Exp. Med. 213, 2861–2870 (2016).")), [GSE101601](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE101601) (Linnarsson’s lab)[68](https://www.nature.com/articles/s41588-025-02260-9#ref-CR68 "Hochgerner, H. et al. STRT-seq-2i: dual-index 5ʹ single cell and nucleus RNA-seq on an addressable microwell array. Sci. Rep. 7, 16327 (2017)."), [GSE76381](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76381) (Linnarsson’s lab)[69](https://www.nature.com/articles/s41588-025-02260-9#ref-CR69 "La Manno, G. et al. Molecular diversity of midbrain development in mouse, human, and stem. Cells Cell 167, 566–580.e19 (2016).") and PsychENCODE[70](https://www.nature.com/articles/s41588-025-02260-9#ref-CR70 "Wang, D. et al. Comprehensive functional genomic resource and integrative model for the human brain. Science 362, eaat8464 (2018).").

### SNP-_h_ 2 and genetic correlation

LDSC[16](https://www.nature.com/articles/s41588-025-02260-9#ref-CR16 "Bulik-Sullivan, B. K. et al. LD score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat. Genet. 47, 291–295 (2015).") was used to calculate SNP-_h_2 for EUR ancestry and to calculate genetic correlation[71](https://www.nature.com/articles/s41588-025-02260-9#ref-CR71 "Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Nat. Genet. 47, 1236–1241 (2015).") between the EUR meta-analysis of the All-PA–leisure phenotype and other traits, to quantify genetic similarity ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02260-9#MOESM1) ‘Traits studied for genetic correlation with PA’). To determine which traits were significant, we used the Benjamini–Hochberg procedure (Supplementary Table [15](https://www.nature.com/articles/s41588-025-02260-9#MOESM3); ‘Benjamini–Hochberg false discovery procedure’). To estimate SNP-_h_2 for AFR and AMR cohorts, we calculated LD scores with cov-LDSC[72](https://www.nature.com/articles/s41588-025-02260-9#ref-CR72 "Luo, Y. et al. Estimating heritability and its enrichment in tissue-specific gene sets in admixed populations. Hum. Mol. Genet 30, 1521–1534 (2021).") from 10,000 random independent individuals from MVP filtering the SNPs to keep only those that were previously identified in the HapMap Project[73](https://www.nature.com/articles/s41588-025-02260-9#ref-CR73 "Frazer, K. A. et al. A second generation human haplotype map of over 3.1 million SNPs. Nature 449, 851–861 (2007)."). We conducted multitrait conditional and joint analysis to evaluate the potential confounding of socioeconomic status with mtCOJO ([Supplementary Notes](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)).

### Benjamini–Hochberg false discovery procedure

To determine which traits were significantly genetically correlated with PA, we used the Benjamini–Hochberg procedure (Supplementary Table [19](https://www.nature.com/articles/s41588-025-02260-9#MOESM3))[74](https://www.nature.com/articles/s41588-025-02260-9#ref-CR74 "Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Series B Methodol. 57, 289–300 (1995)."). We ranked the traits according to their increasing _P_ value, then we calculated the _P_ value’s Benjamini–Hochberg critical value by multiplying each rank by 0.05 and dividing it by the total number of analyzed traits. Finally, the trait _P_ values were compared to the critical values, considering a trait significant until the largest _P_ value was smaller than its corresponding critical value.

### Local genetic correlation

We ran LAVA[19](https://www.nature.com/articles/s41588-025-02260-9#ref-CR19 "Werme, J., van der Sluis, S., Posthuma, D. & de Leeuw, C. A. An integrated framework for local genetic correlation analysis. Nat. Genet. 54, 274–282 (2022).") to calculate local genetic correlations between the EUR All-PA–leisure + SSOE meta-analysis and the 41 traits (Supplementary Table [41](https://www.nature.com/articles/s41588-025-02260-9#MOESM3)) also studied for genome-wide (global) genetic correlation, to identify shared genetic bases in specific genomic regions. LAVA allows estimation of local genetic heritability and local genetic correlations. We considered 2,495 genomic loci, defined by partitioning the genome into blocks of around 1 Mb and minimizing the LD between the blocks[19](https://www.nature.com/articles/s41588-025-02260-9#ref-CR19 "Werme, J., van der Sluis, S., Posthuma, D. & de Leeuw, C. A. An integrated framework for local genetic correlation analysis. Nat. Genet. 54, 274–282 (2022)."). Based on the number of genomic loci, we defined significant local genetic heritability as having a Bonferroni-corrected _P_ < 0.05/2,495. We evaluated local genetic correlation at loci with pairs of traits that have already been shown to have significant local genetic heritability for both phenotypes. Thus, we conducted 3,088 local genetic correlation tests, defining significance as those with a Bonferroni-corrected _P_ < 0.05/3,088.

### MR

MR analyses enable the inference of causality between traits with genetic similarity. Two-sample MR was performed using the following four methods: MR-Egger, weighted median, IVW and simple mode[75](https://www.nature.com/articles/s41588-025-02260-9#ref-CR75 "Hemani, G. et al. The MR-Base platform supports systematic causal inference across the human phenome. eLife 7, e34408 (2018)."). For the PA trait, we used EUR ancestry MVP data of the All-PA–leisure phenotype to account for problems of population stratification and of overlapping samples. We filtered SNPs with _P_ > 10−5 and clumping data using the default window of 10,000 kb and the _r_2 cutoff of 0.001. To designate significant results, we applied multiple testing correction for 36 outcomes used both as outcomes and as exposures (_P_ = 6.9 × 10−4). We also used TwoSampleMR to run MVMR analyses of the All-PA–leisure phenotype as exposure, to understand the influence that BMI could have as a confounder, setting it as exposure ([Supplementary Notes](https://www.nature.com/articles/s41588-025-02260-9#MOESM1)).

### TWAS

TWAS was used to identify genes associated to traits. We ran TWAS using FUSION[76](https://www.nature.com/articles/s41588-025-02260-9#ref-CR76 "Gusev, A. et al. Integrative approaches for large-scale transcriptome-wide association studies. Nat. Genet. 48, 245–252 (2016).") with the 1000 Genomes LD reference data ([https://data.broadinstitute.org/alkesgroup/FUSION/LDREF.tar.bz2](https://data.broadinstitute.org/alkesgroup/FUSION/LDREF.tar.bz2)) and the GTEx v8 multitissue expression weights from 49 tissues ([https://www.mancusolab.com](https://www.mancusolab.com/)). These tissues included adipose, adrenal gland, artery, brain, breast, skin, blood, colon, esophagus, heart, kidney, liver, lung, minor salivary gland, muscle, nerve, ovary, pancreas, pituitary, prostate, small intestine, spleen, stomach, testis, thyroid, uterus and vagina. We performed expression imputation for each autosome using these tissue weights; thus, we identified genes that were conditionally independent. Based on these 49 tissue weights and a total of 27,977 genes, we used a multiple testing correction of 3.65 × 10−8. If a gene was significant and expressed in at least two tissues, we chose the gene expression with the lowest _P_ value.

### Fine-mapping

Fine-mapping of causal gene sets was used to fine-map TWAS at genomic risk regions, identifying likely causal genes[77](https://www.nature.com/articles/s41588-025-02260-9#ref-CR77 "Mancuso, N. et al. Probabilistic fine-mapping of transcriptome-wide association studies. Nat. Genet. 51, 675–682 (2019)."). We used the All-PA–leisure + SSOE EUR meta-analysis and the Vig-PA–leisure as input GWAS, and the GTEx v7 weights from PrediXcan[78](https://www.nature.com/articles/s41588-025-02260-9#ref-CR78 "Gamazon, E. R. et al. A gene-based association method for mapping traits using reference transcriptome data. Nat. Genet. 47, 1091–1098 (2015).") combined with Metabolic Syndrome in Men Study[79](https://www.nature.com/articles/s41588-025-02260-9#ref-CR79 "Laakso, M. et al. The Metabolic Syndrome in Men study: a resource for studies of metabolic and cardiovascular diseases. J. Lipid Res. 58, 481–493 (2017)."), Netherlands Twins Registry[80](https://www.nature.com/articles/s41588-025-02260-9#ref-CR80 "Ligthart, L. et al. The Netherlands Twin Register: longitudinal research based on twin and twin-family designs. Twin Res Hum. Genet 22, 623–636 (2019)."), Young Finns Study[81](https://www.nature.com/articles/s41588-025-02260-9#ref-CR81 "Raitakari, O. T. et al. Cohort profile: the cardiovascular risk in Young Finns Study. Int. J. Epidemiol. 37, 1220–1226 (2008).") and CommonMind Consortium[82](https://www.nature.com/articles/s41588-025-02260-9#ref-CR82 "Fromer, M. et al. Gene expression elucidates functional impact of polygenic risk for schizophrenia. Nat. Neurosci. 19, 1442–1453 (2016).") weights. LD scores were obtained from the 1000 Genomes Phase 3.

We also conducted fine-mapping analyses at the SNP level using Polygenic functionally informed fine-mapping[83](https://www.nature.com/articles/s41588-025-02260-9#ref-CR83 "Weissbrod, O. et al. Functionally informed fine-mapping and polygenic localization of complex trait heritability. Nat. Genet. 52, 1355–1363 (2020)."), using precomputed prior causal probabilities based on a meta-analysis of 15 UKB traits, allowing for the extraction of per-SNP heritabilities (SNPVAR). These were necessary together with the summary statistics and EUR genotypes from 1000 Genomes Project Phase 3 to perform functionally informed fine-mapping with the sum of single effects[84](https://www.nature.com/articles/s41588-025-02260-9#ref-CR84 "Wang, G., Sarkar, A., Carbonetto, P. & Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. J. R. Stat. Soc. Series B Stat. Methodol. 82, 1273–1300 (2020)."),[85](https://www.nature.com/articles/s41588-025-02260-9#ref-CR85 "Zou, Y., Carbonetto, P., Wang, G. & Stephens, M. Fine-mapping from summary data with the ‘Sum of Single Effects’ model. PLoS Genet. 18, e1010299 (2022).").

### Multitrait analysis of GWAS (MTAG)

This method was employed to facilitate joint analysis of summary statistics from GWAS of different but related traits, such as the MTAG[86](https://www.nature.com/articles/s41588-025-02260-9#ref-CR86 "Turley, P. et al. Multi-trait analysis of genome-wide association summary statistics using MTAG. Nat. Genet. 50, 229–237 (2018)."), to increase power in GWAS analyses. Here we ran two MTAG analyses, both with the EUR All-PA–leisure + SSOE meta-analysis as a single trait. First, we joined this trait with leisure screen time[15](https://www.nature.com/articles/s41588-025-02260-9#ref-CR15 "Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. Nat. Genet. 54, 1332–1344 (2022)."). Because of their negative genetic correlation, we flipped the allele in the LSC statistics. Second, we joined the EUR All-PA–leisure + SSOE meta-analysis with a measure of PA-liking[87](https://www.nature.com/articles/s41588-025-02260-9#ref-CR87 "Klimentidis, Y. C. et al. Genome-wide association study of liking for several types of physical activity in the UK Biobank and two replication cohorts. Med. Sci. Sports Exerc. 54, 1252–1260 (2022).").

### Enrichment analysis

g:Profiler is a toolset that also includes finding biological categories enriched in a gene list[88](https://www.nature.com/articles/s41588-025-02260-9#ref-CR88 "Raudvere, U. et al. g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). Nucleic Acids Res. 47, W191–W198 (2019)."). We performed it on three gene lists resulting from the FUMA gene-based test—EUR All-PA–leisure+SSOE meta-analysis, cross-ancestry All-PA–leisure+SSOE meta-analysis and Vig-PA–leisure.

### Mitochondrial genome association analyses

We used mitochondria genomes of EUR ancestry individuals from MVP to run mitochondria–SNV association analyses and gene-based association analyses with All-PA–leisure and Vig-PA–leisure phenotypes. For the mitochondria–SNV association analyses, we filtered missing call rates for variants >0.1, missing call rates for samples >0.1, and we excluded monomorphic variants. We thus analyzed 141 SNVs from 189,782 individuals for All-PA–leisure and 142 SNVs from 201,018 individuals for Vig-PA–leisure. For the gene-based association analyses, we initially assigned the SNVs to mitochondrial genes using the reference sequence [NC\_012920](https://www.ncbi.nlm.nih.gov/nuccore/NC_012920).1 (ref. [89](https://www.nature.com/articles/s41588-025-02260-9#ref-CR89 "Andrews, R. M. et al. Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA. Nat. Genet. 23, 147–147 (1999).")); each SNV mapped to a single gene, and then we used the R software SKAT v2.2.4 to perform associations with All-PA–leisure and Vig-PA–leisure phenotypes, using age, sex and the first ten principal components as covariates[90](https://www.nature.com/articles/s41588-025-02260-9#ref-CR90 "Ionita-Laza, I., Lee, S., Makarov, V., Buxbaum, J. D. & Lin, X. Sequence kernel association tests for the combined effect of rare and common variants. Am. J. Hum. Genet. 92, 841–853 (2013)."). We considered the nominally significant _P_ value (≤0.05).

### PheWAS

We used Vanderbilt University Medical Center’s (VUMC) EHR database to conduct a PheWAS as an empirical exploration of PA and its related phenotypes, investigating a broader range of traits compared to those used in the genetic correlation analyses. The VUMC EHR database, also known as the Synthetic Derivative, houses de-identified, longitudinal medical histories of over 3.1 million patients, including their demographic information (age, sex and ethnicity), and clinical information like medications, lab values, procedural and surgical codes and diagnostic codes[27](https://www.nature.com/articles/s41588-025-02260-9#ref-CR27 "Roden, D. M. et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin. Pharmacol. Ther. 84, 362–369 (2008)."). Codes from both the International Classification of Diseases, ninth and tenth editions (ICD-9 and ICD-10), are used by healthcare professionals to code symptoms, procedures and diagnoses in the synthetic derivative. A subset of the patients from the Synthetic Derivative also has their biological data associated with their medical records. This subset is referred to as BioVU and was used for genomic and phenomic analyses[91](https://www.nature.com/articles/s41588-025-02260-9#ref-CR91 "Hellwege, J. N. et al. Predictive models for abdominal aortic aneurysms using polygenic scores and PheWAS-derived risk factors. Pac. Symp. Biocomput. 28, 425–436 (2023)."),[92](https://www.nature.com/articles/s41588-025-02260-9#ref-CR92 "Mounier, N. & Kutalik, Z. Bias correction for inverse variance weighting Mendelian randomization. Genet Epidemiol. 47, 314–331 (2023)."),[93](https://www.nature.com/articles/s41588-025-02260-9#ref-CR93 "Niarchou, M., Lin, G. T., Lense, M. D., Gordon, R. L. & Davis, L. K. Medical phenome of musicians: an investigation of health records collected on 9803 musically active individuals. Ann. N. Y. Acad. Sci. 1505, 156–168 (2021).").

For our analysis, we performed a PheWAS across BioVU records using a polygenic score for PA derived from the EUR All-PA–leisure + SSOE meta-analysis. First, we used PRS-CS[94](https://www.nature.com/articles/s41588-025-02260-9#ref-CR94 "Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat. Commun. 10, 1776 (2019).") to calculate a polygenic score for PA for 66,917 BioVU records of EUR participants with genotyped data. Next, using the PA polygenic score as the continuous predictor, we ran a PheWAS[95](https://www.nature.com/articles/s41588-025-02260-9#ref-CR95 "Denny, J. C. et al. PheWAS: demonstrating the feasibility of a phenome-wide scan to discover gene-disease associations. Bioinformatics 26, 1205–1210 (2010).") to explore the phenomic landscape of PA. The first step in PheWAS involves mapping related ICD codes to their phecode using the R PheWAS package[96](https://www.nature.com/articles/s41588-025-02260-9#ref-CR96 "Denny, J. C. et al. Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data. Nat. Biotechnol. 31, 1102–1111 (2013)."). The complete list of phecodes, along with the corresponding ICD-9 and ICD-10 codes mapped to them, can be found on the PheWAS catalog ([https://phewascatalog.org/](https://phewascatalog.org/)). Finally, using the EHR-driven case-control status for each phecode, we estimated the association between the PA polygenic score and the phecodes. We restricted our analysis to 1,254 phecodes that meet the following three criteria: (1) phecodes need over 100 cases or control, (2) phecodes should not be specific to a single sex and (3) phecodes should not be included in the broader mental health phenotypes. We also controlled for the first ten ancestry principal components, median age of the record and reported sex.

### Genomic structural equation modeling

Genomic structural equation modeling (genomic-SEM) was used to evaluate the overall genetic architecture of the PA traits included in the EUR GWAS analyses[21](https://www.nature.com/articles/s41588-025-02260-9#ref-CR21 "Grotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nat. Hum. Behav. 3, 513–525 (2019)."). Summary statistics for the All-PA–leisure + SSOE meta-analysis, and for GWAS of Vig-PA–leisure, Vig-PA–home, Vig-PA–work and liking of PA were used to perform EFA and CFA. EFA model fit was evaluated by the amount of cumulative variance explained, the strength of SS loadings (≥1), and balance in the proportion of variance explained by each of the individual factors. Traits with factor loadings ≥0.20 in the EFA were allowed to load on the respective factors and evaluated for CFA model fit as determined by conventional fit indices[21](https://www.nature.com/articles/s41588-025-02260-9#ref-CR21 "Grotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nat. Hum. Behav. 3, 513–525 (2019).").

### Survival analyses

We ran the Cox proportional-hazards model using the package survival 3.3-1 in _R_, to establish the impact of vigorous PA for the three contexts (leisure, home and work) on the risk of dying of any cause. Information was collected in the MVP dataset for age, censoring status, and vigorous PA. Age was calculated for censored individuals from the current year minus the year of birth; for dead participants, the year of death was provided. The vigorous PA phenotypes present in MVP were previously described in the [Methods](https://www.nature.com/articles/s41588-025-02260-9#Sec20) ‘MVP datasets’ as follows: we considered Vig-PA–leisure, Vig-PA–home and Vig-PA–work classification. We had complete information for 252,718 individuals (of whom 55,849 were deceased) for Vig-PA–leisure, 253,096 individuals (of whom 55,203 were deceased) for Vig-PA–home and 207,862 individuals (of whom 43,294 were deceased) for Vig-PA–work.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02260-9#MOESM2) linked to this article.

Data availability
-----------------

All MVP and meta-analysis summary statistics are made available through CIPHER [https://phenomics.va.ornl.gov/web/cipher/partner/mvp](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fphenomics.va.ornl.gov%2Fweb%2Fcipher%2Fpartner%2Fmvp&data=05%7C02%7Cmarco.galimberti%40yale.edu%7Cd7282c0eff2643c1f68708dd97c01dd4%7Cdd8cbebb21394df8b4114e3e87abeb5c%7C0%7C0%7C638833573579843816%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=qETDbzL0bfpsYiBGw69ONjwxV7THPu1skKj%2BXAAekwQ%3D&reserved=0): email CIPHER@va.gov with title ‘MVP and meta-analyses Summary Results–Galimberti et al.’.

Code availability
-----------------

Codes for software and packages used in this analysis are all publicly available through the citations for each method as introduced.

References
----------

1.  Bull, F. C. et al. World Health Organization 2020 guidelines on physical activity and sedentary behaviour. _Br. J. Sports Med._ **54**, 1451–1462 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33239350)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=World%20Health%20Organization%202020%20guidelines%20on%20physical%20activity%20and%20sedentary%20behaviour&journal=Br.%20J.%20Sports%20Med.&volume=54&pages=1451-1462&publication_year=2020&author=Bull%2CFC) 
    
2.  Courneya, K. S. & Friedenreich, C. M. Designing, analyzing, and interpreting observational studies of physical activity and cancer outcomes from a clinical oncology perspective. _Front. Oncol._ **13**, 1098278 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37124538)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10147404)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Designing%2C%20analyzing%2C%20and%20interpreting%20observational%20studies%20of%20physical%20activity%20and%20cancer%20outcomes%20from%20a%20clinical%20oncology%20perspective&journal=Front.%20Oncol.&volume=13&publication_year=2023&author=Courneya%2CKS&author=Friedenreich%2CCM) 
    
3.  Watson, K. B. et al. Cross-sectional study of changes in physical activity behavior during the COVID-19 pandemic among US adults. _Int. J. Behav. Nutr. Phys. Act._ **18**, 91 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34233691)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8261396)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-sectional%20study%20of%20changes%20in%20physical%20activity%20behavior%20during%20the%20COVID-19%20pandemic%20among%20US%20adults&journal=Int.%20J.%20Behav.%20Nutr.%20Phys.%20Act.&volume=18&publication_year=2021&author=Watson%2CKB) 
    
4.  Oh, M. et al. Cross-sectional and longitudinal associations of lifestyle behaviors with pericardial adipose tissue: the MESA study. _Med. Sci. Sports Exerc._ **54**, 984–993 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVKqt7rL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35576135)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9139422)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-sectional%20and%20longitudinal%20associations%20of%20lifestyle%20behaviors%20with%20pericardial%20adipose%20tissue%3A%20the%20MESA%20study&journal=Med.%20Sci.%20Sports%20Exerc.&volume=54&pages=984-993&publication_year=2022&author=Oh%2CM) 
    
5.  Alzahrani, H., Mackey, M., Stamatakis, E., Zadro, J. R. & Shirley, D. The association between physical activity and low back pain: a systematic review and meta-analysis of observational studies. _Sci. Rep._ **9**, 8244 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31160632)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547713)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20association%20between%20physical%20activity%20and%20low%20back%20pain%3A%20a%20systematic%20review%20and%20meta-analysis%20of%20observational%20studies&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Alzahrani%2CH&author=Mackey%2CM&author=Stamatakis%2CE&author=Zadro%2CJR&author=Shirley%2CD) 
    
6.  Stubbe, J. H. et al. Genetic influences on exercise participation in 37,051 twin pairs from seven countries. _PLoS ONE_ **1**, e22 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17183649)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1762341)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20influences%20on%20exercise%20participation%20in%2037%2C051%20twin%20pairs%20from%20seven%20countries&journal=PLoS%20ONE&volume=1&publication_year=2006&author=Stubbe%2CJH) 
    
7.  De Moor, M. H. et al. Genome-wide association study of exercise behavior in Dutch and American adults. _Med Sci. Sports Exerc._ **41**, 1887–1895 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19727025)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2895958)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20of%20exercise%20behavior%20in%20Dutch%20and%20American%20adults&journal=Med%20Sci.%20Sports%20Exerc.&volume=41&pages=1887-1895&publication_year=2009&author=Moor%2CMH) 
    
8.  Kim, J., Oh, S., Min, H., Kim, Y. & Park, T. Practical issues in genome-wide association studies for physical activity. _Ann. N. Y. Acad. Sci._ **1229**, 38–44 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21793837)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Practical%20issues%20in%20genome-wide%20association%20studies%20for%20physical%20activity&journal=Ann.%20N.%20Y.%20Acad.%20Sci.&volume=1229&pages=38-44&publication_year=2011&author=Kim%2CJ&author=Oh%2CS&author=Min%2CH&author=Kim%2CY&author=Park%2CT) 
    
9.  Hara, M. et al. Genomewide association study of leisure-time exercise behavior in Japanese adults. _Med. Sci. Sports Exerc._ **50**, 2433–2441 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30102679)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6282671)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomewide%20association%20study%20of%20leisure-time%20exercise%20behavior%20in%20Japanese%20adults&journal=Med.%20Sci.%20Sports%20Exerc.&volume=50&pages=2433-2441&publication_year=2018&author=Hara%2CM) 
    
10.  Lin, X. et al. Genetic determinants for leisure-time physical activity. _Med Sci. Sports Exerc._ **50**, 1620–1628 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWrtrnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29538177)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6087666)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20determinants%20for%20leisure-time%20physical%20activity&journal=Med%20Sci.%20Sports%20Exerc.&volume=50&pages=1620-1628&publication_year=2018&author=Lin%2CX) 
    
11.  Nabe-Nielsen, K. et al. The effect of occupational physical activity on dementia: results from the Copenhagen male study. _Scand. J. Med. Sci. Sports_ **31**, 446–455 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33038033)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20effect%20of%20occupational%20physical%20activity%20on%20dementia%3A%20results%20from%20the%20Copenhagen%20male%20study&journal=Scand.%20J.%20Med.%20Sci.%20Sports&volume=31&pages=446-455&publication_year=2021&author=Nabe-Nielsen%2CK) 
    
12.  Zotcheva, E. et al. Trajectories of occupational physical activity and risk of later-life mild cognitive impairment and dementia: the HUNT4 70+ study. _Lancet Reg. Health Eur._ **34**, 100721 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37927437)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10625024)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trajectories%20of%20occupational%20physical%20activity%20and%20risk%20of%20later-life%20mild%20cognitive%20impairment%20and%20dementia%3A%20the%20HUNT4%2070%2B%20study&journal=Lancet%20Reg.%20Health%20Eur.&volume=34&publication_year=2023&author=Zotcheva%2CE) 
    
13.  Klimentidis, Y. C. et al. Genome-wide association study of habitual physical activity in over 377,000 UK Biobank participants identifies multiple variants including CADM2 and APOE. _Int. J. Obes._ **42**, 1161–1176 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFyjsbnI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20of%20habitual%20physical%20activity%20in%20over%20377%2C000%20UK%20Biobank%20participants%20identifies%20multiple%20variants%20including%20CADM2%20and%20APOE&journal=Int.%20J.%20Obes.&volume=42&pages=1161-1176&publication_year=2018&author=Klimentidis%2CYC) 
    
14.  Doherty, A. et al. GWAS identifies 14 loci for device-measured physical activity and sleep duration. _Nat. Commun._ **9**, 5257 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30531941)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6288145)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GWAS%20identifies%2014%20loci%20for%20device-measured%20physical%20activity%20and%20sleep%20duration&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Doherty%2CA) 
    
15.  Wang, Z. et al. Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention. _Nat. Genet._ **54**, 1332–1344 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlCiu7rK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36071172)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9470530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20analyses%20of%20physical%20activity%20and%20sedentary%20behavior%20provide%20insights%20into%20underlying%20mechanisms%20and%20roles%20in%20disease%20prevention&journal=Nat.%20Genet.&volume=54&pages=1332-1344&publication_year=2022&author=Wang%2CZ) 
    
16.  Bulik-Sullivan, B. K. et al. LD score regression distinguishes confounding from polygenicity in genome-wide association studies. _Nat. Genet._ **47**, 291–295 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFCntb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25642630)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4495769)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=LD%20score%20regression%20distinguishes%20confounding%20from%20polygenicity%20in%20genome-wide%20association%20studies&journal=Nat.%20Genet.&volume=47&pages=291-295&publication_year=2015&author=Bulik-Sullivan%2CBK) 
    
17.  Wen, X., Pique-Regi, R. & Luca, F. Integrating molecular QTL data into genome-wide genetic association analysis: probabilistic assessment of enrichment and colocalization. _PLoS Genet._ **13**, e1006646 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28278150)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5363995)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrating%20molecular%20QTL%20data%20into%20genome-wide%20genetic%20association%20analysis%3A%20probabilistic%20assessment%20of%20enrichment%20and%20colocalization&journal=PLoS%20Genet.&volume=13&publication_year=2017&author=Wen%2CX&author=Pique-Regi%2CR&author=Luca%2CF) 
    
18.  Hukku, A., Sampson, M. G., Luca, F., Pique-Regi, R. & Wen, X. Analyzing and reconciling colocalization and transcriptome-wide association studies from the perspective of inferential reproducibility. _Am. J. Hum. Genet._ **109**, 825–837 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1WgtrrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35523146)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9118134)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Analyzing%20and%20reconciling%20colocalization%20and%20transcriptome-wide%20association%20studies%20from%20the%20perspective%20of%20inferential%20reproducibility&journal=Am.%20J.%20Hum.%20Genet.&volume=109&pages=825-837&publication_year=2022&author=Hukku%2CA&author=Sampson%2CMG&author=Luca%2CF&author=Pique-Regi%2CR&author=Wen%2CX) 
    
19.  Werme, J., van der Sluis, S., Posthuma, D. & de Leeuw, C. A. An integrated framework for local genetic correlation analysis. _Nat. Genet._ **54**, 274–282 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XntFCjur0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35288712)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20framework%20for%20local%20genetic%20correlation%20analysis&journal=Nat.%20Genet.&volume=54&pages=274-282&publication_year=2022&author=Werme%2CJ&author=Sluis%2CS&author=Posthuma%2CD&author=Leeuw%2CCA) 
    
20.  Gao, M. et al. Associations between body-mass index and COVID-19 severity in 6·9 million people in England: a prospective, community-based, cohort study. _Lancet Diabetes Endocrinol._ **9**, 350–359 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVartbfP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33932335)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8081400)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Associations%20between%20body-mass%20index%20and%20COVID-19%20severity%20in%206%C2%B79%20million%20people%20in%20England%3A%20a%20prospective%2C%20community-based%2C%20cohort%20study&journal=Lancet%20Diabetes%20Endocrinol.&volume=9&pages=350-359&publication_year=2021&author=Gao%2CM) 
    
21.  Grotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. _Nat. Hum. Behav._ **3**, 513–525 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30962613)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6520146)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20structural%20equation%20modelling%20provides%20insights%20into%20the%20multivariate%20genetic%20architecture%20of%20complex%20traits&journal=Nat.%20Hum.%20Behav.&volume=3&pages=513-525&publication_year=2019&author=Grotzinger%2CAD) 
    
22.  Carlsson, S., Ahlbom, A., Lichtenstein, P. & Andersson, T. Shared genetic influence of BMI, physical activity and type 2 diabetes: a twin study. _Diabetologia_ **56**, 1031–1035 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXmt1Ciu7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23404445)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Shared%20genetic%20influence%20of%20BMI%2C%20physical%20activity%20and%20type%202%20diabetes%3A%20a%20twin%20study&journal=Diabetologia&volume=56&pages=1031-1035&publication_year=2013&author=Carlsson%2CS&author=Ahlbom%2CA&author=Lichtenstein%2CP&author=Andersson%2CT) 
    
23.  Zhang, X. et al. Genetically predicted physical activity levels are associated with lower colorectal cancer risk: a Mendelian randomisation study. _Br. J. Cancer_ **124**, 1330–1338 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33510439)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8007642)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetically%20predicted%20physical%20activity%20levels%20are%20associated%20with%20lower%20colorectal%20cancer%20risk%3A%20a%20Mendelian%20randomisation%20study&journal=Br.%20J.%20Cancer&volume=124&pages=1330-1338&publication_year=2021&author=Zhang%2CX) 
    
24.  Wang, Z., Davey Smith, G., Loos, R. J. F. & den Hoed, M. Distilling causality between physical activity traits and obesity via Mendelian randomization. _Commun. Med._ **3**, 173 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38036650)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10689836)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distilling%20causality%20between%20physical%20activity%20traits%20and%20obesity%20via%20Mendelian%20randomization&journal=Commun.%20Med.&volume=3&publication_year=2023&author=Wang%2CZ&author=Davey%20Smith%2CG&author=Loos%2CRJF&author=Hoed%2CM) 
    
25.  Guthold, R., Stevens, G. A., Riley, L. M. & Bull, F. C. Worldwide trends in insufficient physical activity from 2001 to 2016: a pooled analysis of 358 population-based surveys with 1.9 million participants. _Lancet Glob. Health_ **6**, e1077–e1086 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30193830)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Worldwide%20trends%20in%20insufficient%20physical%20activity%20from%202001%20to%202016%3A%20a%20pooled%20analysis%20of%20358%20population-based%20surveys%20with%201.9%20million%20participants&journal=Lancet%20Glob.%20Health&volume=6&pages=e1077-e1086&publication_year=2018&author=Guthold%2CR&author=Stevens%2CGA&author=Riley%2CLM&author=Bull%2CFC) 
    
26.  Guthold, R., Stevens, G. A., Riley, L. M. & Bull, F. C. Global trends in insufficient physical activity among adolescents: a pooled analysis of 298 population-based surveys with 1·6 million participants. _Lancet Child Adolesc. Health_ **4**, 23–35 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31761562)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6919336)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20trends%20in%20insufficient%20physical%20activity%20among%20adolescents%3A%20a%20pooled%20analysis%20of%20298%20population-based%20surveys%20with%201%C2%B76%20million%20participants&journal=Lancet%20Child%20Adolesc.%20Health&volume=4&pages=23-35&publication_year=2020&author=Guthold%2CR&author=Stevens%2CGA&author=Riley%2CLM&author=Bull%2CFC) 
    
27.  Roden, D. M. et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. _Clin. Pharmacol. Ther._ **84**, 362–369 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtVShtr%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18500243)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&journal=Clin.%20Pharmacol.%20Ther.&volume=84&pages=362-369&publication_year=2008&author=Roden%2CDM) 
    
28.  Morris, J. et al. Genetic variation in CADM2 as a link between psychological traits and obesity. _Sci. Rep._ **9**, 7339 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31089183)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6517397)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20variation%20in%20CADM2%20as%20a%20link%20between%20psychological%20traits%20and%20obesity&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Morris%2CJ) 
    
29.  Pasman, J. A. et al. GWAS of lifetime cannabis use reveals new risk loci, genetic overlap with psychiatric traits, and a causal effect of schizophrenia liability. _Nat. Neurosci._ **21**, 1161–1170 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2nsrvF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30150663)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6386176)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GWAS%20of%20lifetime%20cannabis%20use%20reveals%20new%20risk%20loci%2C%20genetic%20overlap%20with%20psychiatric%20traits%2C%20and%20a%20causal%20effect%20of%20schizophrenia%20liability&journal=Nat.%20Neurosci.&volume=21&pages=1161-1170&publication_year=2018&author=Pasman%2CJA) 
    
30.  Liu, M. et al. Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use. _Nat. Genet._ **51**, 237–244 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFSns7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30643251)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6358542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20studies%20of%20up%20to%201.2%20million%20individuals%20yield%20new%20insights%20into%20the%20genetic%20etiology%20of%20tobacco%20and%20alcohol%20use&journal=Nat.%20Genet.&volume=51&pages=237-244&publication_year=2019&author=Liu%2CM) 
    
31.  Strawbridge, R. J. et al. Genome-wide analysis of self-reported risk-taking behaviour and cross-disorder genetic correlations in the UK Biobank cohort. _Transl. Psychiatry_ **8**, 39 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29391395)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5804026)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20analysis%20of%20self-reported%20risk-taking%20behaviour%20and%20cross-disorder%20genetic%20correlations%20in%20the%20UK%20Biobank%20cohort&journal=Transl.%20Psychiatry&volume=8&publication_year=2018&author=Strawbridge%2CRJ) 
    
32.  Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. _Nat. Genet._ **50**, 1112–1121 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlOis7zI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30038396)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6393768)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20discovery%20and%20polygenic%20prediction%20from%20a%20genome-wide%20association%20study%20of%20educational%20attainment%20in%201.1%20million%20individuals&journal=Nat.%20Genet.&volume=50&pages=1112-1121&publication_year=2018&author=Lee%2CJJ) 
    
33.  Casey, J. P. et al. A novel approach of homozygous haplotype sharing identifies candidate genes in autism spectrum disorder. _Hum. Genet._ **131**, 565–579 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21996756)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20novel%20approach%20of%20homozygous%20haplotype%20sharing%20identifies%20candidate%20genes%20in%20autism%20spectrum%20disorder&journal=Hum.%20Genet.&volume=131&pages=565-579&publication_year=2012&author=Casey%2CJP) 
    
34.  Šimunić-Briški, N. et al. Physical exercise induces significant changes in immunoglobulin G N-glycan composition in a previously inactive, overweight population. _Biomolecules_ **13**, 762 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37238633)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10216199)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Physical%20exercise%20induces%20significant%20changes%20in%20immunoglobulin%20G%20N-glycan%20composition%20in%20a%20previously%20inactive%2C%20overweight%20population&journal=Biomolecules&volume=13&publication_year=2023&author=%C5%A0imuni%C4%87-Bri%C5%A1ki%2CN) 
    
35.  Zhu, Z. et al. Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. _Nat. Genet._ **48**, 481–487 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XkvFGrs7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27019110)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integration%20of%20summary%20data%20from%20GWAS%20and%20eQTL%20studies%20predicts%20complex%20trait%20gene%20targets&journal=Nat.%20Genet.&volume=48&pages=481-487&publication_year=2016&author=Zhu%2CZ) 
    
36.  Gaudino, G. et al. The proto-oncogene RON is involved in development of epithelial, bone and neuro-endocrine tissues. _Oncogene_ **11**, 2627–2637 (1995).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XlvF2gsg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8545120)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20proto-oncogene%20RON%20is%20involved%20in%20development%20of%20epithelial%2C%20bone%20and%20neuro-endocrine%20tissues&journal=Oncogene&volume=11&pages=2627-2637&publication_year=1995&author=Gaudino%2CG) 
    
37.  Nieminuszczy, J. et al. EXD2 protects stressed replication forks and is required for cell viability in the absence of BRCA1/2. _Mol. Cell_ **75**, 605–619 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1KgtL%2FE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31255466)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6695479)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=EXD2%20protects%20stressed%20replication%20forks%20and%20is%20required%20for%20cell%20viability%20in%20the%20absence%20of%20BRCA1%2F2&journal=Mol.%20Cell&volume=75&pages=605-619&publication_year=2019&author=Nieminuszczy%2CJ) 
    
38.  Zhou, H. et al. Genome-wide meta-analysis of problematic alcohol use in 435,563 individuals yields insights into biology and relationships with other traits. _Nat. Neurosci._ **23**, 809–818 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVSis73O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32451486)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7485556)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20meta-analysis%20of%20problematic%20alcohol%20use%20in%20435%2C563%20individuals%20yields%20insights%20into%20biology%20and%20relationships%20with%20other%20traits&journal=Nat.%20Neurosci.&volume=23&pages=809-818&publication_year=2020&author=Zhou%2CH) 
    
39.  Pardiñas, A. F. et al. Common schizophrenia alleles are enriched in mutation-intolerant genes and in regions under strong background selection. _Nat. Genet._ **50**, 381–389 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29483656)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918692)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Common%20schizophrenia%20alleles%20are%20enriched%20in%20mutation-intolerant%20genes%20and%20in%20regions%20under%20strong%20background%20selection&journal=Nat.%20Genet.&volume=50&pages=381-389&publication_year=2018&author=Pardi%C3%B1as%2CAF) 
    
40.  Deak, J. D. et al. Genome-wide association study in individuals of European and African ancestry and multi-trait analysis of opioid use disorder identifies 19 independent genome-wide significant risk loci. _Mol. Psychiatry_ **10**, 3970–3979 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20in%20individuals%20of%20European%20and%20African%20ancestry%20and%20multi-trait%20analysis%20of%20opioid%20use%20disorder%20identifies%2019%20independent%20genome-wide%20significant%20risk%20loci&journal=Mol.%20Psychiatry&volume=10&pages=3970-3979&publication_year=2022&author=Deak%2CJD) 
    
41.  Akiyama, M. et al. Genome-wide association study identifies 112 new loci for body mass index in the Japanese population. _Nat. Genet._ **49**, 1458–1467 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVKltbjK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28892062)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20identifies%20112%20new%20loci%20for%20body%20mass%20index%20in%20the%20Japanese%20population&journal=Nat.%20Genet.&volume=49&pages=1458-1467&publication_year=2017&author=Akiyama%2CM) 
    
42.  Pulit, S. L. et al. Meta-analysis of genome-wide association studies for body fat distribution in 694,649 individuals of European ancestry. _Hum. Mol. Genet_ **28**, 166–174 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1CgsL3F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30239722)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Meta-analysis%20of%20genome-wide%20association%20studies%20for%20body%20fat%20distribution%20in%20694%2C649%20individuals%20of%20European%20ancestry&journal=Hum.%20Mol.%20Genet&volume=28&pages=166-174&publication_year=2019&author=Pulit%2CSL) 
    
43.  Vujkovic, M. et al. Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis. _Nat. Genet._ **52**, 680–691 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFKgtr%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32541925)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7343592)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20of%20318%20new%20risk%20loci%20for%20type%202%20diabetes%20and%20related%20vascular%20outcomes%20among%201.4%20million%20participants%20in%20a%20multi-ancestry%20meta-analysis&journal=Nat.%20Genet.&volume=52&pages=680-691&publication_year=2020&author=Vujkovic%2CM) 
    
44.  Maddock, R. J., Casazza, G. A., Fernandez, D. H. & Maddock, M. I. Acute modulation of cortical glutamate and GABA content by physical activity. _J. Neurosci._ **36**, 2449–2457 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xpt1OjsLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26911692)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6705493)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Acute%20modulation%20of%20cortical%20glutamate%20and%20GABA%20content%20by%20physical%20activity&journal=J.%20Neurosci.&volume=36&pages=2449-2457&publication_year=2016&author=Maddock%2CRJ&author=Casazza%2CGA&author=Fernandez%2CDH&author=Maddock%2CMI) 
    
45.  Gaziano, J. M. et al. Million Veteran Program: a mega-biobank to study genetic influences on health and disease. _J. Clin. Epidemiol._ **70**, 214–223 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26441289)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Million%20Veteran%20Program%3A%20a%20mega-biobank%20to%20study%20genetic%20influences%20on%20health%20and%20disease&journal=J.%20Clin.%20Epidemiol.&volume=70&pages=214-223&publication_year=2016&author=Gaziano%2CJM) 
    
46.  Harrington, K. M. et al. Gender differences in demographic and health characteristics of the Million Veteran Program cohort. _Womens Health Issues_ **29**, S56–S66 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31253243)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7061933)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gender%20differences%20in%20demographic%20and%20health%20characteristics%20of%20the%20Million%20Veteran%20Program%20cohort&journal=Womens%20Health%20Issues&volume=29&pages=S56-S66&publication_year=2019&author=Harrington%2CKM) 
    
47.  Hunter-Zinck, H. et al. Genotyping array design and data quality control in the Million Veteran Program. _Am. J. Hum. Genet._ **106**, 535–548 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXmsVSks7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32243820)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118558)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genotyping%20array%20design%20and%20data%20quality%20control%20in%20the%20Million%20Veteran%20Program&journal=Am.%20J.%20Hum.%20Genet.&volume=106&pages=535-548&publication_year=2020&author=Hunter-Zinck%2CH) 
    
48.  Loh, P.-R. et al. Reference-based phasing using the Haplotype Reference Consortium panel. _Nat. Genet._ **48**, 1443–1448 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1SltLzF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27694958)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5096458)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reference-based%20phasing%20using%20the%20Haplotype%20Reference%20Consortium%20panel&journal=Nat.%20Genet.&volume=48&pages=1443-1448&publication_year=2016&author=Loh%2CP-R) 
    
49.  Das, S. et al. Next-generation genotype imputation service and methods. _Nat. Genet._ **48**, 1284–1287 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVWksL%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27571263)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5157836)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Next-generation%20genotype%20imputation%20service%20and%20methods&journal=Nat.%20Genet.&volume=48&pages=1284-1287&publication_year=2016&author=Das%2CS) 
    
50.  Auton, A. et al. A global reference for human genetic variation. _Nature_ **526**, 68–74 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26432245)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20global%20reference%20for%20human%20genetic%20variation&journal=Nature&volume=526&pages=68-74&publication_year=2015&author=Auton%2CA) 
    
51.  Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. _GigaScience_ **4**, 7 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25722852)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&journal=GigaScience&volume=4&publication_year=2015&author=Chang%2CCC) 
    
52.  Manichaikul, A. et al. Robust relationship inference in genome-wide association studies. _Bioinformatics_ **26**, 2867–2873 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVSlt7bK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20926424)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3025716)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Robust%20relationship%20inference%20in%20genome-wide%20association%20studies&journal=Bioinformatics&volume=26&pages=2867-2873&publication_year=2010&author=Manichaikul%2CA) 
    
53.  Sudlow, C. et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. _PLoS Med._ **12**, e1001779 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25826379)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380465)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UK%20biobank%3A%20an%20open%20access%20resource%20for%20identifying%20the%20causes%20of%20a%20wide%20range%20of%20complex%20diseases%20of%20middle%20and%20old%20age&journal=PLoS%20Med.&volume=12&publication_year=2015&author=Sudlow%2CC) 
    
54.  Willer, C. J., Li, Y. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. _Bioinformatics_ **26**, 2190–2191 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVGru73F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20616382)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922887)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=METAL%3A%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&journal=Bioinformatics&volume=26&pages=2190-2191&publication_year=2010&author=Willer%2CCJ&author=Li%2CY&author=Abecasis%2CGR) 
    
55.  Gao, F. et al. XWAS: a software toolset for genetic data analysis and association studies of the X chromosome. _J. Hered._ **106**, 666–671 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkt1yksr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26268243)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4567842)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=XWAS%3A%20a%20software%20toolset%20for%20genetic%20data%20analysis%20and%20association%20studies%20of%20the%20X%20chromosome&journal=J.%20Hered.&volume=106&pages=666-671&publication_year=2015&author=Gao%2CF) 
    
56.  Stouffer, S. A. A study of attitudes. _Sci. Am._ **180**, 11–15 (1949).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaH1M%2FjvF2msA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18138375)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20study%20of%20attitudes&journal=Sci.%20Am.&volume=180&pages=11-15&publication_year=1949&author=Stouffer%2CSA) 
    
57.  De Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. _PLoS Comput. Biol._ **11**, e1004219 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25885710)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4401657)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAGMA%3A%20generalized%20gene-set%20analysis%20of%20GWAS%20data&journal=PLoS%20Comput.%20Biol.&volume=11&publication_year=2015&author=Leeuw%2CCA&author=Mooij%2CJM&author=Heskes%2CT&author=Posthuma%2CD) 
    
58.  Watanabe, K., Taskesen, E., van Bochoven, A. & Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. _Nat. Commun._ **8**, 1826 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29184056)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5705698)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&journal=Nat.%20Commun.&volume=8&publication_year=2017&author=Watanabe%2CK&author=Taskesen%2CE&author=Bochoven%2CA&author=Posthuma%2CD) 
    
59.  Aguet, F. et al. The GTEx Consortium atlas of genetic regulatory effects across human tissues. _Science_ **369**, 1318–1330 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsl2ksLfK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20GTEx%20Consortium%20atlas%20of%20genetic%20regulatory%20effects%20across%20human%20tissues&journal=Science&volume=369&pages=1318-1330&publication_year=2020&author=Aguet%2CF) 
    
60.  Watanabe, K., Umićević Mirkov, M., de Leeuw, C. A., van den Heuvel, M. P. & Posthuma, D. Genetic mapping of cell type specificity for complex traits. _Nat. Commun._ **10**, 3222 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVels73L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31324783)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6642112)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20mapping%20of%20cell%20type%20specificity%20for%20complex%20traits&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Watanabe%2CK&author=Umi%C4%87evi%C4%87%20Mirkov%2CM&author=Leeuw%2CCA&author=Heuvel%2CMP&author=Posthuma%2CD) 
    
61.  Hodge, R. D. et al. Conserved cell types with divergent features in human versus mouse cortex. _Nature_ **573**, 61–68 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1agtbvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31435019)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6919571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conserved%20cell%20types%20with%20divergent%20features%20in%20human%20versus%20mouse%20cortex&journal=Nature&volume=573&pages=61-68&publication_year=2019&author=Hodge%2CRD) 
    
62.  Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. _Nat. Methods_ **14**, 955–958 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtl2gtL7L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28846088)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5623139)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Massively%20parallel%20single-nucleus%20RNA-seq%20with%20DroNc-seq&journal=Nat.%20Methods&volume=14&pages=955-958&publication_year=2017&author=Habib%2CN) 
    
63.  Zhong, S. et al. A single-cell RNA-seq survey of the developmental landscape of the human prefrontal cortex. _Nature_ **555**, 524–528 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXksFegurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29539641)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single-cell%20RNA-seq%20survey%20of%20the%20developmental%20landscape%20of%20the%20human%20prefrontal%20cortex&journal=Nature&volume=555&pages=524-528&publication_year=2018&author=Zhong%2CS) 
    
64.  Darmanis, S. et al. A survey of human brain transcriptome diversity at the single cell level. _Proc. Natl Acad. Sci. USA_ **112**, 7285–7290 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXosVKrurg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26060301)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4466750)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20survey%20of%20human%20brain%20transcriptome%20diversity%20at%20the%20single%20cell%20level&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=112&pages=7285-7290&publication_year=2015&author=Darmanis%2CS) 
    
65.  Enge, M. et al. Single-cell analysis of human pancreas reveals transcriptional signatures of aging and somatic mutation patterns. _Cell_ **171**, 321–330 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFOrtrrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28965763)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6047899)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20analysis%20of%20human%20pancreas%20reveals%20transcriptional%20signatures%20of%20aging%20and%20somatic%20mutation%20patterns&journal=Cell&volume=171&pages=321-330&publication_year=2017&author=Enge%2CM) 
    
66.  Baron, M. et al. A single-cell transcriptomic map of the human and mouse pancreas reveals inter- and intra-cell population structure. _Cell Syst._ **3**, 346–360 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFalsrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27667365)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5228327)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single-cell%20transcriptomic%20map%20of%20the%20human%20and%20mouse%20pancreas%20reveals%20inter-%20and%20intra-cell%20population%20structure&journal=Cell%20Syst.&volume=3&pages=346-360&publication_year=2016&author=Baron%2CM) 
    
67.  Breton, G. et al. Human dendritic cells (DCs) are derived from distinct circulating precursors that are precommitted to become CD1c+ or CD141+ DCs. _J. Exp. Med._ **213**, 2861–2870 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitlWhu7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27864467)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5154947)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20dendritic%20cells%20%28DCs%29%20are%20derived%20from%20distinct%20circulating%20precursors%20that%20are%20precommitted%20to%20become%20CD1c%2B%20or%20CD141%2B%20DCs&journal=J.%20Exp.%20Med.&volume=213&pages=2861-2870&publication_year=2016&author=Breton%2CG) 
    
68.  Hochgerner, H. et al. STRT-seq-2i: dual-index 5ʹ single cell and nucleus RNA-seq on an addressable microwell array. _Sci. Rep._ **7**, 16327 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29180631)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5703850)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=STRT-seq-2i%3A%20dual-index%205%CA%B9%20single%20cell%20and%20nucleus%20RNA-seq%20on%20an%20addressable%20microwell%20array&journal=Sci.%20Rep.&volume=7&publication_year=2017&author=Hochgerner%2CH) 
    
69.  La Manno, G. et al. Molecular diversity of midbrain development in mouse, human, and stem. _Cells Cell_ **167**, 566–580.e19 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27716510)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20diversity%20of%20midbrain%20development%20in%20mouse%2C%20human%2C%20and%20stem&journal=Cells%20Cell&volume=167&pages=566-580.e19&publication_year=2016&author=Manno%2CG) 
    
70.  Wang, D. et al. Comprehensive functional genomic resource and integrative model for the human brain. _Science_ **362**, eaat8464 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhsLnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30545857)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6413328)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20functional%20genomic%20resource%20and%20integrative%20model%20for%20the%20human%20brain&journal=Science&volume=362&publication_year=2018&author=Wang%2CD) 
    
71.  Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. _Nat. Genet._ **47**, 1236–1241 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7vM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26414676)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4797329)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20atlas%20of%20genetic%20correlations%20across%20human%20diseases%20and%20traits&journal=Nat.%20Genet.&volume=47&pages=1236-1241&publication_year=2015&author=Bulik-Sullivan%2CB) 
    
72.  Luo, Y. et al. Estimating heritability and its enrichment in tissue-specific gene sets in admixed populations. _Hum. Mol. Genet_ **30**, 1521–1534 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVCktr%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33987664)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8330913)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estimating%20heritability%20and%20its%20enrichment%20in%20tissue-specific%20gene%20sets%20in%20admixed%20populations&journal=Hum.%20Mol.%20Genet&volume=30&pages=1521-1534&publication_year=2021&author=Luo%2CY) 
    
73.  Frazer, K. A. et al. A second generation human haplotype map of over 3.1 million SNPs. _Nature_ **449**, 851–861 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFOjt7jL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17943122)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20second%20generation%20human%20haplotype%20map%20of%20over%203.1%20million%20SNPs&journal=Nature&volume=449&pages=851-861&publication_year=2007&author=Frazer%2CKA) 
    
74.  Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. _J. R. Stat. Soc. Series B Methodol._ **57**, 289–300 (1995).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Controlling%20the%20false%20discovery%20rate%3A%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Methodol.&volume=57&pages=289-300&publication_year=1995&author=Benjamini%2CY&author=Hochberg%2CY) 
    
75.  Hemani, G. et al. The MR-Base platform supports systematic causal inference across the human phenome. _eLife_ **7**, e34408 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29846171)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5976434)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20MR-Base%20platform%20supports%20systematic%20causal%20inference%20across%20the%20human%20phenome&journal=eLife&volume=7&publication_year=2018&author=Hemani%2CG) 
    
76.  Gusev, A. et al. Integrative approaches for large-scale transcriptome-wide association studies. _Nat. Genet._ **48**, 245–252 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xit1ajtbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26854917)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4767558)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20approaches%20for%20large-scale%20transcriptome-wide%20association%20studies&journal=Nat.%20Genet.&volume=48&pages=245-252&publication_year=2016&author=Gusev%2CA) 
    
77.  Mancuso, N. et al. Probabilistic fine-mapping of transcriptome-wide association studies. _Nat. Genet._ **51**, 675–682 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGlt7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30926970)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6619422)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Probabilistic%20fine-mapping%20of%20transcriptome-wide%20association%20studies&journal=Nat.%20Genet.&volume=51&pages=675-682&publication_year=2019&author=Mancuso%2CN) 
    
78.  Gamazon, E. R. et al. A gene-based association method for mapping traits using reference transcriptome data. _Nat. Genet._ **47**, 1091–1098 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlSnurbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26258848)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552594)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20gene-based%20association%20method%20for%20mapping%20traits%20using%20reference%20transcriptome%20data&journal=Nat.%20Genet.&volume=47&pages=1091-1098&publication_year=2015&author=Gamazon%2CER) 
    
79.  Laakso, M. et al. The Metabolic Syndrome in Men study: a resource for studies of metabolic and cardiovascular diseases. _J. Lipid Res._ **58**, 481–493 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsFGqs7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28119442)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5335588)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Metabolic%20Syndrome%20in%20Men%20study%3A%20a%20resource%20for%20studies%20of%20metabolic%20and%20cardiovascular%20diseases&journal=J.%20Lipid%20Res.&volume=58&pages=481-493&publication_year=2017&author=Laakso%2CM) 
    
80.  Ligthart, L. et al. The Netherlands Twin Register: longitudinal research based on twin and twin-family designs. _Twin Res Hum. Genet_ **22**, 623–636 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31666148)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Netherlands%20Twin%20Register%3A%20longitudinal%20research%20based%20on%20twin%20and%20twin-family%20designs&journal=Twin%20Res%20Hum.%20Genet&volume=22&pages=623-636&publication_year=2019&author=Ligthart%2CL) 
    
81.  Raitakari, O. T. et al. Cohort profile: the cardiovascular risk in Young Finns Study. _Int. J. Epidemiol._ **37**, 1220–1226 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18263651)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cohort%20profile%3A%20the%20cardiovascular%20risk%20in%20Young%20Finns%20Study&journal=Int.%20J.%20Epidemiol.&volume=37&pages=1220-1226&publication_year=2008&author=Raitakari%2COT) 
    
82.  Fromer, M. et al. Gene expression elucidates functional impact of polygenic risk for schizophrenia. _Nat. Neurosci._ **19**, 1442–1453 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFOmsLnN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27668389)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5083142)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20expression%20elucidates%20functional%20impact%20of%20polygenic%20risk%20for%20schizophrenia&journal=Nat.%20Neurosci.&volume=19&pages=1442-1453&publication_year=2016&author=Fromer%2CM) 
    
83.  Weissbrod, O. et al. Functionally informed fine-mapping and polygenic localization of complex trait heritability. _Nat. Genet._ **52**, 1355–1363 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlGjsL%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33199916)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7710571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functionally%20informed%20fine-mapping%20and%20polygenic%20localization%20of%20complex%20trait%20heritability&journal=Nat.%20Genet.&volume=52&pages=1355-1363&publication_year=2020&author=Weissbrod%2CO) 
    
84.  Wang, G., Sarkar, A., Carbonetto, P. & Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. _J. R. Stat. Soc. Series B Stat. Methodol._ **82**, 1273–1300 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37220626)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10201948)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20simple%20new%20approach%20to%20variable%20selection%20in%20regression%2C%20with%20application%20to%20genetic%20fine%20mapping&journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol.&volume=82&pages=1273-1300&publication_year=2020&author=Wang%2CG&author=Sarkar%2CA&author=Carbonetto%2CP&author=Stephens%2CM) 
    
85.  Zou, Y., Carbonetto, P., Wang, G. & Stephens, M. Fine-mapping from summary data with the ‘Sum of Single Effects’ model. _PLoS Genet._ **18**, e1010299 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVeku7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35853082)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9337707)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fine-mapping%20from%20summary%20data%20with%20the%20%E2%80%98Sum%20of%20Single%20Effects%E2%80%99%20model&journal=PLoS%20Genet.&volume=18&publication_year=2022&author=Zou%2CY&author=Carbonetto%2CP&author=Wang%2CG&author=Stephens%2CM) 
    
86.  Turley, P. et al. Multi-trait analysis of genome-wide association summary statistics using MTAG. _Nat. Genet._ **50**, 229–237 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkslGitLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29292387)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5805593)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-trait%20analysis%20of%20genome-wide%20association%20summary%20statistics%20using%20MTAG&journal=Nat.%20Genet.&volume=50&pages=229-237&publication_year=2018&author=Turley%2CP) 
    
87.  Klimentidis, Y. C. et al. Genome-wide association study of liking for several types of physical activity in the UK Biobank and two replication cohorts. _Med. Sci. Sports Exerc._ **54**, 1252–1260 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35320144)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9288543)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20of%20liking%20for%20several%20types%20of%20physical%20activity%20in%20the%20UK%20Biobank%20and%20two%20replication%20cohorts&journal=Med.%20Sci.%20Sports%20Exerc.&volume=54&pages=1252-1260&publication_year=2022&author=Klimentidis%2CYC) 
    
88.  Raudvere, U. et al. g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). _Nucleic Acids Res._ **47**, W191–W198 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktVyit70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31066453)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6602461)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=g%3AProfiler%3A%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20conversions%20of%20gene%20lists%20%282019%20update%29&journal=Nucleic%20Acids%20Res.&volume=47&pages=W191-W198&publication_year=2019&author=Raudvere%2CU) 
    
89.  Andrews, R. M. et al. Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA. _Nat. Genet._ **23**, 147–147 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmtlOhtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10508508)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reanalysis%20and%20revision%20of%20the%20Cambridge%20reference%20sequence%20for%20human%20mitochondrial%20DNA&journal=Nat.%20Genet.&volume=23&pages=147-147&publication_year=1999&author=Andrews%2CRM) 
    
90.  Ionita-Laza, I., Lee, S., Makarov, V., Buxbaum, J. D. & Lin, X. Sequence kernel association tests for the combined effect of rare and common variants. _Am. J. Hum. Genet._ **92**, 841–853 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXnvVGltLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23684009)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3675243)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sequence%20kernel%20association%20tests%20for%20the%20combined%20effect%20of%20rare%20and%20common%20variants&journal=Am.%20J.%20Hum.%20Genet.&volume=92&pages=841-853&publication_year=2013&author=Ionita-Laza%2CI&author=Lee%2CS&author=Makarov%2CV&author=Buxbaum%2CJD&author=Lin%2CX) 
    
91.  Hellwege, J. N. et al. Predictive models for abdominal aortic aneurysms using polygenic scores and PheWAS-derived risk factors. _Pac. Symp. Biocomput._ **28**, 425–436 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36540997)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9782709)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Predictive%20models%20for%20abdominal%20aortic%20aneurysms%20using%20polygenic%20scores%20and%20PheWAS-derived%20risk%20factors&journal=Pac.%20Symp.%20Biocomput.&volume=28&pages=425-436&publication_year=2023&author=Hellwege%2CJN) 
    
92.  Mounier, N. & Kutalik, Z. Bias correction for inverse variance weighting Mendelian randomization. _Genet Epidemiol._ **47**, 314–331 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXns1Gltrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37036286)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bias%20correction%20for%20inverse%20variance%20weighting%20Mendelian%20randomization&journal=Genet%20Epidemiol.&volume=47&pages=314-331&publication_year=2023&author=Mounier%2CN&author=Kutalik%2CZ) 
    
93.  Niarchou, M., Lin, G. T., Lense, M. D., Gordon, R. L. & Davis, L. K. Medical phenome of musicians: an investigation of health records collected on 9803 musically active individuals. _Ann. N. Y. Acad. Sci._ **1505**, 156–168 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34414577)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8688228)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Medical%20phenome%20of%20musicians%3A%20an%20investigation%20of%20health%20records%20collected%20on%209803%20musically%20active%20individuals&journal=Ann.%20N.%20Y.%20Acad.%20Sci.&volume=1505&pages=156-168&publication_year=2021&author=Niarchou%2CM&author=Lin%2CGT&author=Lense%2CMD&author=Gordon%2CRL&author=Davis%2CLK) 
    
94.  Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. _Nat. Commun._ **10**, 1776 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30992449)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467998)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20prediction%20via%20Bayesian%20regression%20and%20continuous%20shrinkage%20priors&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Ge%2CT&author=Chen%2CC-Y&author=Ni%2CY&author=Feng%2CY-CA&author=Smoller%2CJW) 
    
95.  Denny, J. C. et al. PheWAS: demonstrating the feasibility of a phenome-wide scan to discover gene-disease associations. _Bioinformatics_ **26**, 1205–1210 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlt1equ7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20335276)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2859132)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PheWAS%3A%20demonstrating%20the%20feasibility%20of%20a%20phenome-wide%20scan%20to%20discover%20gene-disease%20associations&journal=Bioinformatics&volume=26&pages=1205-1210&publication_year=2010&author=Denny%2CJC) 
    
96.  Denny, J. C. et al. Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data. _Nat. Biotechnol._ **31**, 1102–1111 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVaiu7jE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24270849)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3969265)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Systematic%20comparison%20of%20phenome-wide%20association%20study%20of%20electronic%20medical%20record%20data%20and%20genome-wide%20association%20study%20data&journal=Nat.%20Biotechnol.&volume=31&pages=1102-1111&publication_year=2013&author=Denny%2CJC) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02260-9?format=refman&flavour=references)

Acknowledgements
----------------

This research is based in part on data from the Million Veteran Program, Office of Research and Development, Veterans Health Administration. More details regarding this consortium are provided in the [Supplementary Information](https://www.nature.com/articles/s41588-025-02260-9#MOESM1). This work was supported by funding from the Department of Veterans Affairs Office of Research and Development, USVA (grant I01CX001849 to J.G. and M.B.S.) and the VA Cooperative Studies Program study (575B to J.G. and M.B.S.). Additionally, it was supported by the VA National Center for PTSD Research and the West Haven VA Mental Illness Research, Education and Clinical Center. Further support was provided by NIH (grants R01DA054869 and R01 AA026364 to J.G.). D.F.L. is supported by a Career Development Award CDA-2 from the Veterans Affairs Office of Research and Development (1IK2BX005058-01A2) and is the Aimee Mann Fellow of Psychiatric Genetics. This publication does not represent the views of the Department of Veterans Affairs or the United States Government.

Author information
------------------

### Authors and Affiliations

1.  Veterans Affairs Connecticut Healthcare System, West Haven, CT, USA
    
    Marco Galimberti, Daniel F. Levey, Joseph D. Deak, Keyrun Adhikari, Cassie Overstreet, Priya Gupta, Hang Zhou & Joel Gelernter
    
2.  Department of Psychiatry, Yale School of Medicine, New Haven, CT, USA
    
    Marco Galimberti, Daniel F. Levey, Joseph D. Deak, Keyrun Adhikari, Cassie Overstreet, Priya Gupta, Hang Zhou & Joel Gelernter
    
3.  Department of Medicine, Vanderbilt University Medical Center, Nashville, TN, USA
    
    Rachana Nitin & Lea K. Davis
    
4.  Department of Genetics, Yale School of Medicine, New Haven, CT, USA
    
    Nicole J. Lake
    
5.  Department of Psychiatry, Boston University School of Medicine, Boston, MA, USA
    
    Kelly M. Harrington
    
6.  Million Veteran Program (MVP) Coordinating Center, VA Boston Healthcare System, Boston, MA, USA
    
    Kelly M. Harrington, Luc Djousse & J. Michael Gaziano
    
7.  Department of Medicine, Brigham and Women’s Hospital, Boston, MA, USA
    
    Luc Djousse & J. Michael Gaziano
    
8.  Department of Medicine, Harvard Medical School, Boston, MA, USA
    
    J. Michael Gaziano
    
9.  VA San Diego Healthcare System, San Diego, CA, USA
    
    Murray B. Stein
    
10.  Department of Psychiatry and School of Public Health, University of California, San Diego, La Jolla, CA, USA
    
    Murray B. Stein
    

Authors

1.  Marco Galimberti
    
    [View author publications](https://www.nature.com/search?author=Marco%20Galimberti)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marco%20Galimberti) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marco%20Galimberti%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Daniel F. Levey
    
    [View author publications](https://www.nature.com/search?author=Daniel%20F.%20Levey)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20F.%20Levey) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20F.%20Levey%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Joseph D. Deak
    
    [View author publications](https://www.nature.com/search?author=Joseph%20D.%20Deak)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Joseph%20D.%20Deak) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Joseph%20D.%20Deak%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Keyrun Adhikari
    
    [View author publications](https://www.nature.com/search?author=Keyrun%20Adhikari)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Keyrun%20Adhikari) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Keyrun%20Adhikari%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Cassie Overstreet
    
    [View author publications](https://www.nature.com/search?author=Cassie%20Overstreet)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cassie%20Overstreet) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cassie%20Overstreet%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Priya Gupta
    
    [View author publications](https://www.nature.com/search?author=Priya%20Gupta)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Priya%20Gupta) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Priya%20Gupta%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Rachana Nitin
    
    [View author publications](https://www.nature.com/search?author=Rachana%20Nitin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rachana%20Nitin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rachana%20Nitin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Hang Zhou
    
    [View author publications](https://www.nature.com/search?author=Hang%20Zhou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hang%20Zhou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hang%20Zhou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Nicole J. Lake
    
    [View author publications](https://www.nature.com/search?author=Nicole%20J.%20Lake)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nicole%20J.%20Lake) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nicole%20J.%20Lake%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Kelly M. Harrington
    
    [View author publications](https://www.nature.com/search?author=Kelly%20M.%20Harrington)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kelly%20M.%20Harrington) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kelly%20M.%20Harrington%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Luc Djousse
    
    [View author publications](https://www.nature.com/search?author=Luc%20Djousse)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Luc%20Djousse) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Luc%20Djousse%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Lea K. Davis
    
    [View author publications](https://www.nature.com/search?author=Lea%20K.%20Davis)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lea%20K.%20Davis) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lea%20K.%20Davis%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  J. Michael Gaziano
    
    [View author publications](https://www.nature.com/search?author=J.%20Michael%20Gaziano)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=J.%20Michael%20Gaziano) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22J.%20Michael%20Gaziano%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Murray B. Stein
    
    [View author publications](https://www.nature.com/search?author=Murray%20B.%20Stein)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Murray%20B.%20Stein) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Murray%20B.%20Stein%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Joel Gelernter
    
    [View author publications](https://www.nature.com/search?author=Joel%20Gelernter)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Joel%20Gelernter) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Joel%20Gelernter%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Consortia

### VA Million Veteran Program

-   Marco Galimberti
-   , Daniel F. Levey
-   , Joseph D. Deak
-   , Keyrun Adhikari
-   , Cassie Overstreet
-   , Priya Gupta
-   , Hang Zhou
-   , Kelly M. Harrington
-   , Luc Djousse
-   , J. Michael Gaziano
-   , Murray B. Stein
-    & Joel Gelernter

### Contributions

M.G. and J.G. designed the study and drafted the manuscript. D.F.L. provided ongoing feedback and refinement of the analytical plan. M.G. conducted GWAS on the included cohorts. M.G., D.F.L. and J.G. discussed, created and refined the phenotype in the MVP, and discussed and refined MVP analytic plans. M.G., D.F.L., J.D.D., K.A., C.O., P.G. and R.N. conducted original analyses. N.J.L. contributed to the mitochondrial analyses. H.Z. contributed to genetic correlations. M.G., D.F.L., J.D.D., L.K.D. and J.G. supervised original analyses. J.G. supervised the project. J.G. and M.B.S. obtained funding. M.G., D.F.L., J.D.D., K.A., C.O., P.G., R.N., H.Z., N.J.L, K.M.H., L.D., L.K.D., J.M.G., M.B.S. and J.G. critically evaluated and revised the manuscript.

### Corresponding author

Correspondence to [Joel Gelernter](mailto:joel.gelernter@yale.edu).

Ethics declarations
-------------------

### Competing interests

J.G. is paid for editorial work for the journal ‘Complex Psychiatry’. M.B.S. has, in the past 3 years, received consulting income from Acadia Pharmaceuticals, Aptinyx, atai Life Sciences, BigHealth, Bionomics, BioXcel Therapeutics, Boehringer Ingelheim, Clexio, Eisai, EmpowerPharm, Engrail Therapeutics, Janssen, Jazz Pharmaceuticals, NeuroTrauma Sciences, PureTech Health, Sumitomo Pharma and Roche/Genentech; has stock options in Oxeia Biopharmaceuticals and EpiVario; has been paid for his editorial work on Depression and Anxiety (Editor-in-Chief), Biological Psychiatry (Deputy Editor), and UpToDate (Co-Editor-in-Chief for Psychiatry); has also received research support from NIH, Department of Veterans Affairs, and the Department of Defense; and is on the scientific advisory board for the Brain and Behavior Research Foundation and the Anxiety and Depression Association of America. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Marcel den Hoed, Eske Derks and Yann Klimentidis for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02260-9/MediaObjects/41588_2025_2260_MOESM1_ESM.pdf)

Supplementary Notes (Supplementary Methods, Supplementary Results and Supplementary Discussion), Supplementary Figs. 1–20 and VA Million Veteran Program core acknowledgement.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02260-9/MediaObjects/41588_2025_2260_MOESM2_ESM.pdf)

### [Supplementary Tables 1–41](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02260-9/MediaObjects/41588_2025_2260_MOESM3_ESM.xlsx)

Supplementary Tables 1–41.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=A%20contextual%20genomic%20perspective%20on%20physical%20activity%20and%20its%20relationship%20to%20health%2C%20well%20being%20and%20illness&author=Marco%20Galimberti%20et%20al&contentID=10.1038%2Fs41588-025-02260-9&copyright=This%20is%20a%20U.S.%20Government%20work%20and%20not%20under%20copyright%20protection%20in%20the%20US%3B%20foreign%20copyright%20protection%20may%20apply&publication=1061-4036&publicationDate=2025-07-21&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02260-9)

### Cite this article

Galimberti, M., Levey, D.F., Deak, J.D. _et al._ A contextual genomic perspective on physical activity and its relationship to health, well being and illness. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02260-9

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02260-9?format=refman&flavour=citation)

-   Received: 18 January 2024
    
-   Accepted: 09 June 2025
    
-   Published: 21 July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02260-9
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02260-9.pdf)
