> id: https://www.nature.com/articles/s41588-025-02308-w

> link: https://www.nature.com/articles/s41588-025-02308-w

> title: Tamoxifen induces PI3K activation in uterine cancer

# Tamoxifen induces PI3K activation in uterine cancer
_Published on Fri, 22 Aug 2025 00:00:00 GMT_

Abstract
--------

Mutagenic processes and clonal selection contribute to the development of therapy-associated secondary neoplasms, a known complication of cancer treatment. The association between tamoxifen therapy and secondary uterine cancers is uncommon but well established; however, the genetic mechanisms underlying tamoxifen-driven tumorigenesis remain unclear. We find that oncogenic _PIK3CA_ mutations, common in spontaneously arising estrogen-associated de novo uterine cancer, are significantly less frequent in tamoxifen-associated tumors. In vivo, tamoxifen-induced estrogen receptor stimulation activates phosphoinositide 3-kinase (PI3K) signaling in normal mouse uterine tissue, potentially eliminating the selective benefit of PI3K-activating mutations in tamoxifen-associated uterine cancer. Together, we present a unique pathway of therapy-associated carcinogenesis in which tamoxifen-induced activation of the PI3K pathway acts as a non-genetic driver event, contributing to the multistep model of uterine carcinogenesis. While this PI3K mechanism is specific to tamoxifen-associated uterine cancer, the concept of treatment-induced signaling events may have broader applicability to other routes of tumorigenesis.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41419-021-03904-4/MediaObjects/41419_2021_3904_Fig1_HTML.png)

### [USP35, regulated by estrogen and AKT, promotes breast tumorigenesis by stabilizing and enhancing transcriptional activity of estrogen receptor α](https://www.nature.com/articles/s41419-021-03904-4?fromPaywallRec=false)

Article Open access 15 June 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41416-021-01293-1/MediaObjects/41416_2021_1293_Fig1_HTML.png)

### [PI3K activation promotes resistance to eribulin in HER2-negative breast cancer](https://www.nature.com/articles/s41416-021-01293-1?fromPaywallRec=false)

Article 15 March 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-022-09889-0/MediaObjects/41598_2022_9889_Fig1_HTML.png)

### [Pan-cancer analysis on the role of PIK3R1 and PIK3R2 in human tumors](https://www.nature.com/articles/s41598-022-09889-0?fromPaywallRec=false)

Article Open access 08 April 2022

Main
----

Therapy-related secondary malignancies associated with certain cytotoxic drugs or radiotherapy are relatively uncommon. Mechanistically, such secondary neoplasms are attributed to clonal selection of preexisting mutations or therapy-induced mutagenesis[1](https://www.nature.com/articles/s41588-025-02308-w#ref-CR1 "Kuijk, E., Kranenburg, O., Cuppen, E. & Van Hoeck, A. Common anti-cancer therapies induce somatic mutations in stem cells of healthy tissue. Nat. Commun. 13, 5915 (2022)."). Whether similar mechanisms also contribute to cancer evolution after hormonal therapy has remained controversial, particularly in the context of tamoxifen use[2](https://www.nature.com/articles/s41588-025-02308-w#ref-CR2 "Carthew, P. et al. DNA damage as assessed by 32P-postlabelling in three rat strains exposed to dietary tamoxifen: the relationship between cell proliferation and liver tumour formation. Carcinogenesis 16, 1299–1304 (1995)."),[3](https://www.nature.com/articles/s41588-025-02308-w#ref-CR3 "Carthew, P. et al. Cumulative exposure to tamoxifen: DNA adducts and liver cancer in the rat. Arch. Toxicol. 75, 375–380 (2001)."),[4](https://www.nature.com/articles/s41588-025-02308-w#ref-CR4 "Busch, H. Adducts and tamoxifen. Semin. Oncol. 24, S1-98–S1-104 (1997)."),[5](https://www.nature.com/articles/s41588-025-02308-w#ref-CR5 "Hernandez-Ramon, E. E. et al. Tamoxifen–DNA adduct formation in monkey and human reproductive organs. Carcinogenesis 35, 1172–1176 (2014)."),[6](https://www.nature.com/articles/s41588-025-02308-w#ref-CR6 "Andersson, H., Helmestam, M., Zebrowska, A., Olovsson, M. & Brittebo, E. Tamoxifen-induced adduct formation and cell stress in human endometrial glands. Drug Metab. Dispos. 38, 200–207 (2010)."),[7](https://www.nature.com/articles/s41588-025-02308-w#ref-CR7 "Kim, S. Y. et al. Formation of tamoxifen–DNA adducts in human endometrial explants exposed to α-hydroxytamoxifen. Chem. Res. Toxicol. 18, 889–895 (2005).").

Tamoxifen was the first endocrine drug approved for treating estrogen receptor (ER)-positive breast cancer[8](https://www.nature.com/articles/s41588-025-02308-w#ref-CR8 "Cole, M. P., Jones, C. T. & Todd, I. D. A new anti-oestrogenic agent in late breast cancer. An early clinical appraisal of ICI46474. Br. J. Cancer 25, 270–275 (1971)."),[9](https://www.nature.com/articles/s41588-025-02308-w#ref-CR9 "Fisher, B. et al. Adjuvant chemotherapy with and without tamoxifen in the treatment of primary breast cancer: 5-year results from the National Surgical Adjuvant Breast and Bowel Project Trial. J. Clin. Oncol. 4, 459–471 (1986).") and as a preventive drug in women with high risk of developing breast cancer[10](https://www.nature.com/articles/s41588-025-02308-w#ref-CR10 "Fisher, B. et al. Tamoxifen for prevention of breast cancer: report of the National Surgical Adjuvant Breast and Bowel Project P-1 Study. J. Natl Cancer Inst. 90, 1371–1388 (1998)."). Although estrogen-reducing aromatase inhibitors have superior outcome in the adjuvant setting[11](https://www.nature.com/articles/s41588-025-02308-w#ref-CR11 "Early Breast Cancer Trialists’ Collaborative Group. Aromatase inhibitors versus tamoxifen in early breast cancer: patient-level meta-analysis of the randomised trials. Lancet 386, 1341–1352 (2015)."), tamoxifen still has a clear benefit in reducing risk of recurrence and death from breast cancer and remains a standard endocrine treatment option in premenopausal and postmenopausal women with early-stage ER+ disease[12](https://www.nature.com/articles/s41588-025-02308-w#ref-CR12 "Sparano, J. A. et al. Adjuvant chemotherapy guided by a 21-gene expression assay in breast cancer. N. Engl. J. Med. 379, 111–121 (2018)."),[13](https://www.nature.com/articles/s41588-025-02308-w#ref-CR13 "Johnston, S. R. D. et al. Abemaciclib combined with endocrine therapy for the adjuvant treatment of HR+, HER2−, node-positive, high-risk, early breast cancer (monarchE). J. Clin. Oncol. 38, 3987–3998 (2020)."). One serious drawback of tamoxifen therapy is an association with increased risk of uterine cancer (UC): randomized clinical trials and large observational studies found a twofold to sevenfold increased risk 2–5 years after tamoxifen treatment either for breast cancer[14](https://www.nature.com/articles/s41588-025-02308-w#ref-CR14 "Fornander, T. et al. Adjuvant tamoxifen in early breast cancer: occurrence of new primary cancers. Lancet 1, 117–120 (1989)."),[15](https://www.nature.com/articles/s41588-025-02308-w#ref-CR15 "Bernstein, L. et al. Tamoxifen therapy for breast cancer and endometrial cancer risk. J. Natl Cancer Inst. 91, 1654–1662 (1999)."),[16](https://www.nature.com/articles/s41588-025-02308-w#ref-CR16 "Bergman, L. et al. Risk and prognosis of endometrial cancer after tamoxifen for breast cancer. Comprehensive Cancer Centres’ ALERT Group. Assessment of liver and endometrial cancer risk following tamoxifen. Lancet 356, 881–887 (2000)."),[17](https://www.nature.com/articles/s41588-025-02308-w#ref-CR17 "Fisher, B. et al. Endometrial cancer in tamoxifen-treated breast cancer patients: findings from the National Surgical Adjuvant Breast and Bowel Project (NSABP) B-14. J. Natl Cancer Inst. 86, 527–537 (1994)."),[18](https://www.nature.com/articles/s41588-025-02308-w#ref-CR18 "Swerdlow, A. J., Jones, M. E. & British Tamoxifen Second Cancer Study Group. Tamoxifen treatment for breast cancer and risk of endometrial cancer: a case–control study. J. Natl Cancer Inst. 97, 375–384 (2005).") or for prevention[19](https://www.nature.com/articles/s41588-025-02308-w#ref-CR19 "Cuzick, J. et al. Tamoxifen for prevention of breast cancer: extended long-term follow-up of the IBIS-I breast cancer prevention trial. Lancet Oncol. 16, 67–75 (2015)."),[20](https://www.nature.com/articles/s41588-025-02308-w#ref-CR20 "Fisher, B. et al. Tamoxifen for the prevention of breast cancer: current status of the National Surgical Adjuvant Breast and Bowel Project P-1 study. J. Natl Cancer Inst. 97, 1652–1662 (2005)."). Extended tamoxifen use of 10 versus 5 years correlated with an approximate twofold further increase in the risk of tamoxifen-associated UC (TA-UC)[21](https://www.nature.com/articles/s41588-025-02308-w#ref-CR21 "Davies, C. et al. Long-term effects of continuing adjuvant tamoxifen to 10 years versus stopping at 5 years after diagnosis of oestrogen receptor-positive breast cancer: ATLAS, a randomised trial. Lancet 381, 805–816 (2013)."), underscoring the link between tamoxifen and UC.

Tamoxifen is a selective ER modulator. In breast tissue, it functions as an ER antagonist; in the uterus, it has ER-agonistic activity stemming from the recruitment of ER coactivators rather than co-repressors[22](https://www.nature.com/articles/s41588-025-02308-w#ref-CR22 "Shang, Y. & Brown, M. Molecular determinants for the tissue specificity of SERMs. Science 295, 2465–2468 (2002)."). The pro-proliferative effect of tamoxifen in the uterus is well established to be ER dependent[23](https://www.nature.com/articles/s41588-025-02308-w#ref-CR23 "Korach, K. S. Insights from the study of animals lacking functional estrogen receptor. Science 266, 1524–1527 (1994)."),[24](https://www.nature.com/articles/s41588-025-02308-w#ref-CR24 "Couse, J. F. & Korach, K. S. Estrogen receptor null mice: what have we learned and where will they lead us? Endocr. Rev. 20, 358–417 (1999)."). However, whether this ER-agonistic effect is the key driver of oncogenesis in TA-UC remains unclear. Although tamoxifen has been reported to be mutagenic in the rat liver[25](https://www.nature.com/articles/s41588-025-02308-w#ref-CR25 "Davies, R. et al. Tamoxifen causes gene mutations in the livers of lambda/lacI transgenic rats. Cancer Res. 57, 1288–1293 (1997)."), whether similar mutagenic effects occur in human uterine tissue remains controversial[26](https://www.nature.com/articles/s41588-025-02308-w#ref-CR26 "Brown, K. Is tamoxifen a genotoxic carcinogen in women? Mutagenesis 24, 391–404 (2009)."). A previous study, limited in technological scope, did not find TA-UC-specific genomic changes[27](https://www.nature.com/articles/s41588-025-02308-w#ref-CR27 "Fles, R. et al. Genomic profile of endometrial tumors depends on morphological subtype, not on tamoxifen exposure. Genes Chromosomes Cancer 49, 699–710 (2010)."). Here, we extended the genomic profiling of TA-UCs to whole-exome sequencing (WES), allowing us to study a larger number and broader variety of genomic events. WES analysis and subsequent in vivo modeling in mice revealed a unique cancer development mechanism, an understanding that may have implications for counseling and risk-reducing interventions in tamoxifen-treated patients at high risk for UC as well as relevance to other therapy-related secondary cancers.

Results
-------

### No evidence of tamoxifen-induced mutagenesis

To determine whether TA-UC is molecularly distinct from spontaneously arising de novo UC (that is, not associated with tamoxifen), we performed WES on 21 TA-UCs from the ‘Tamoxifen Associated Malignancies: Aspects of Risk’ (TAMARISK) study[28](https://www.nature.com/articles/s41588-025-02308-w#ref-CR28 "Hoogendoorn, W. E. et al. Prognosis of uterine corpus cancer after tamoxifen treatment for breast cancer. Breast Cancer Res. Treat. 112, 99–108 (2008).") (discovery cohort; Fig. [1a](https://www.nature.com/articles/s41588-025-02308-w#Fig1), Supplementary Table [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02308-w#Fig6)) and compared their histological types to various de novo UC cohorts (Surveillance, Epidemiology, and End Results 9 (SEER9), TAMARISK[28](https://www.nature.com/articles/s41588-025-02308-w#ref-CR28 "Hoogendoorn, W. E. et al. Prognosis of uterine corpus cancer after tamoxifen treatment for breast cancer. Breast Cancer Res. Treat. 112, 99–108 (2008)."), TCGA[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."),[30](https://www.nature.com/articles/s41588-025-02308-w#ref-CR30 "Cherniack, A. D. et al. Integrated molecular characterization of uterine carcinosarcoma. Cancer Cell 31, 411–423 (2017)."),[31](https://www.nature.com/articles/s41588-025-02308-w#ref-CR31 "Cancer Genome Atlas Research Network. Comprehensive and integrated genomic characterization of adult soft tissue sarcomas. Cell 171, 950–965 (2017)."), Genomics Evidence Neoplasia Information Exchange (GENIE)[32](https://www.nature.com/articles/s41588-025-02308-w#ref-CR32 "AACR Project GENIE Consortium. AACR Project GENIE: powering precision medicine through an international consortium. Cancer Discov. 7, 818–831 (2017).")). Our analysis revealed no significant differences after correcting for multiple hypotheses (all _Q_ > 0.1, Benjamini–Hochberg (BH)-corrected Fisher’s exact test; Extended Data Fig. [1b](https://www.nature.com/articles/s41588-025-02308-w#Fig6) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). Similarly, the molecular subtypes in TA-UC closely matched those in de novo UC from TCGA[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013).") (all _Q_ > 0.5; Extended Data Fig. [1c,d](https://www.nature.com/articles/s41588-025-02308-w#Fig6), Supplementary Table [2](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)). These findings allow for downstream comparison of genomic alterations between TA-UC and de novo UC, independent of subtype.

**Fig. 1: Reduced frequency of PI3K pathway mutations in TA-UC.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02308-w/figures/1)

**a**, Time course for each patient shows duration of tamoxifen treatment (colored bars) and periods of UC diagnosis (diagn., gray bars); crossed dagger indicates treatment for at least 2 years, but exact duration is unknown. **b**, Plot of mutational features for TA-UCs from the discovery cohort, ordered by significantly mutated genes. From top to bottom, subpanels depict number (no.) of mutations per megabase (Mb), sample identifiers, significantly mutated genes (bold; red line, _Q_ < 0.1; top, unrestricted hypothesis testing; bottom, restricted hypothesis testing of known UC driver genes) and nonsignificantly mutated cancer genes (PI3K pathway genes are in violet and annotated with a dagger). **c**, Plot of SCNAs ordered as in **b**; top, significant SCNAs (red line, _Q_ < 0.25, from GISTIC); bottom, nonsignificant SCNAs in the PI3K pathway (violet and annotated with a dagger). **d**, Plot of molecular classifications and mutational processes (MSI, microsatellite instability; MSS, microsatellite stable; CIN, chromosomal instability; GS, genomically stable; POLE, polymerase ε), clinical annotations (mix., mixed; carcinosarc., carcinosarcoma; FIGO, International Federation of Gynecology and Obstetrics; NA, not available) and median length of tamoxifen use in years (yrs); samples ordered as in **b**. **e**, UC driver genes powered to detect differences (higher or lower) in mutation frequencies between TCGA de novo UC and TA-UC sample sets (_P_\-value threshold for statistical power analysis at <0.05 after Bonferroni correction for the 49 significant driver genes in de novo UC). Genes are colored by pathway; gray line indicates equal frequencies in both cohorts; data points represent number of mutated tumors; error bars reflect Poisson-based s.d. estimate. Significance analysis by two-sided BH-corrected Fisher’s exact test (_Q_ values added for all _Q_ < 0.1 and/or PI3K pathway genes; \* and sign denote significance). **f**, Bar plot of all (top, all mut) and hotspot (hs, bottom) _PIK3CA_ mutations; bars represent mutation frequencies; error bars reflect s.d. from the β-distribution; significance analysis by two-sided Fisher’s exact test; numbers in bars indicate mutated tumor count per group. **g**, Bar plot of PI3K pathway alterations including SNVs (mut) and SCNAs (gain or deletion (del)); only TCGA tumors with both data types were considered; genes altered by either type were counted once per tumor; bars represent mutation frequencies; error bars reflect s.d. from the β-distribution; significance analysis by two-sided Fisher’s exact test; numbers in bars indicate mutated tumor count per group.

We next analyzed frequencies of genomic alterations to test for tamoxifen-related mutagenesis. Tamoxifen did not increase the mutational burden (median number of mutations per Mb, 2.7 in TA-UC versus 2.3 in de novo UC; _P_ = 0.7, Wilcoxon test) or the genomic fraction affected by somatic copy number alterations (SCNAs; median of 0.05 versus 0.1, _P_ = 0.4; Extended Data Fig. [1e](https://www.nature.com/articles/s41588-025-02308-w#Fig6)), even after accounting for molecular subtypes (Extended Data Fig. [1f,g](https://www.nature.com/articles/s41588-025-02308-w#Fig6), Supplementary Table [2](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)). Similarly, the duration of tamoxifen treatment was unrelated to mutational (_r_ = 0.07, Pearson correlation coefficient, _P_ = 0.8) and SCNA burden (_r_ = 0.3, _P_ = 0.2). Mutational signatures can also reveal the mutagenic mechanisms of carcinogens[33](https://www.nature.com/articles/s41588-025-02308-w#ref-CR33 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)."). While de novo signature discovery did not identify a tamoxifen-specific mutational signature, previously described signatures were detected in de novo UC[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."),[30](https://www.nature.com/articles/s41588-025-02308-w#ref-CR30 "Cherniack, A. D. et al. Integrated molecular characterization of uterine carcinosarcoma. Cancer Cell 31, 411–423 (2017)."),[31](https://www.nature.com/articles/s41588-025-02308-w#ref-CR31 "Cancer Genome Atlas Research Network. Comprehensive and integrated genomic characterization of adult soft tissue sarcomas. Cell 171, 950–965 (2017).") (Extended Data Fig. [2a,b](https://www.nature.com/articles/s41588-025-02308-w#Fig7) and Supplementary Note [2](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)). In sum, tamoxifen does not show a direct mutagenic effect.

### TA-UC harbors fewer mutational events in _PIK3CA_ and _PIK3R1_

To discover mutation-based drivers of TA-UC, we used MutSig2CV (Fig. [1b–d](https://www.nature.com/articles/s41588-025-02308-w#Fig1); _Q_ < 0.1) and identified four significantly mutated genes, _PTEN_, _KRAS_, _TP53_ and _ARID1A_, all of which were also observed as drivers in de novo UC[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."),[30](https://www.nature.com/articles/s41588-025-02308-w#ref-CR30 "Cherniack, A. D. et al. Integrated molecular characterization of uterine carcinosarcoma. Cancer Cell 31, 411–423 (2017)."),[31](https://www.nature.com/articles/s41588-025-02308-w#ref-CR31 "Cancer Genome Atlas Research Network. Comprehensive and integrated genomic characterization of adult soft tissue sarcomas. Cell 171, 950–965 (2017).") (Extended Data Fig. [3a,b](https://www.nature.com/articles/s41588-025-02308-w#Fig8) and Supplementary Table [3](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). To increase statistical power for finding drivers using the smaller TA-UC cohort, we further restricted our analysis to 113 known UC drivers (Supplementary Table [4](https://www.nature.com/articles/s41588-025-02308-w#MOESM3))[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."),[30](https://www.nature.com/articles/s41588-025-02308-w#ref-CR30 "Cherniack, A. D. et al. Integrated molecular characterization of uterine carcinosarcoma. Cancer Cell 31, 411–423 (2017)."),[31](https://www.nature.com/articles/s41588-025-02308-w#ref-CR31 "Cancer Genome Atlas Research Network. Comprehensive and integrated genomic characterization of adult soft tissue sarcomas. Cell 171, 950–965 (2017)."),[34](https://www.nature.com/articles/s41588-025-02308-w#ref-CR34 "Gibson, W. J. et al. The genomic landscape and evolution of endometrial carcinoma progression and abdominopelvic metastasis. Nat. Genet. 48, 848–855 (2016).") to decrease the number of hypotheses tested and found that _RNF43_, _FGFR2_ and _CTNNB1_ were also significantly mutated (_Q_ < 0.1).

Next, to evaluate the relationship between driver gene mutation frequencies and tamoxifen exposure, we assessed the statistical power for finding differences (higher or lower) between TA-UC and de novo UC samples ([Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9)). Among the 49 genes identified as significantly mutated drivers in de novo UC (Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02308-w#Fig8)), we found five (_PTEN_, _PIK3CA_, _TP53_, _ARID1A_ and _PIK3R1_) that were powered ([Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9); Bonferroni-corrected optimal Fisher’s exact _P_ < 0.05; Extended Data Fig. [2c](https://www.nature.com/articles/s41588-025-02308-w#Fig7) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). We observed a significant difference in mutation frequencies for two of these genes (Fig. [1e](https://www.nature.com/articles/s41588-025-02308-w#Fig1)), both in the PI3K pathway: _PIK3CA_ (encoding the PI3K catalytic subunit p110α; 14% versus 48%; _P_ = 0.003, _Q_ = 0.007; two-sided BH-corrected Fisher’s exact test) and _PIK3R1_ (encoding the PI3K regulatory subunit p85α; 0% versus 31%; _P_ = 0.0009, _Q_ = 0.005). Surprisingly, both genes had lower mutation frequencies in TA-UC. Stratified Fisher’s exact tests confirmed that the lower mutation frequencies in TA-UC (_PIK3CA_, combined _P_ = 0.008; _PIK3R1_, combined _P_ = 0.001) were not driven by the different distributions of tumor grades in our TA-UC and de novo UC cohorts (Supplementary Note [3](https://www.nature.com/articles/s41588-025-02308-w#MOESM1) and Supplementary Fig. [1a](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)).

To search for additional genes among the 113 known UC drivers with reduced mutation frequency in TA-UC, we used a one-sided test and found 30 genes for which we had sufficient power to detect reduced mutation frequency ([Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9)). Again, only _PIK3CA_ (_P_ = 0.002, _Q_ = 0.03; one-sided BH-corrected Fisher’s exact test) and _PIK3R1_ (_P_ = 0.0004, _Q_ = 0.01) reached significance (Extended Data Fig. [2d](https://www.nature.com/articles/s41588-025-02308-w#Fig7) and Supplementary Table [6](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)).

Compared to de novo UC, TA-UC also had significantly fewer hotspot _PIK3CA_ mutations (10% versus 38%; _P_ = 0.009, Fisher’s exact test; Fig. [1f](https://www.nature.com/articles/s41588-025-02308-w#Fig1) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)), which confer stronger pathway activation[35](https://www.nature.com/articles/s41588-025-02308-w#ref-CR35 "Chang, M. T. et al. Identifying recurrent mutations in cancer reveals widespread lineage diversity and mutational specificity. Nat. Biotechnol. 34, 155–163 (2016)."). This observation held true even when controlling for gene coverage (Extended Data Fig. [2e](https://www.nature.com/articles/s41588-025-02308-w#Fig7) and Supplementary Note [4](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)) and was validated by droplet digital PCR (ddPCR; Extended Data Fig. [2f](https://www.nature.com/articles/s41588-025-02308-w#Fig7) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)). Of note, we identified two patients in the TCGA cohort exposed to tamoxifen before UC diagnosis ([Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9)) who did not harbor a _PIK3CA_ mutation (Fig. [1f](https://www.nature.com/articles/s41588-025-02308-w#Fig1)). Finally, genomic identification of significant targets in cancer (GISTIC) analysis[36](https://www.nature.com/articles/s41588-025-02308-w#ref-CR36 "Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 12, R41 (2011).") ([Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9)) did not detect significant enrichments of _PIK3CA_ amplifications and _PIK3R1_ deletions in TA-UC (Extended Data Fig. [4a,b](https://www.nature.com/articles/s41588-025-02308-w#Fig9)) compared to de novo UC (_Q_ < 0.25; Extended Data Fig. [3c](https://www.nature.com/articles/s41588-025-02308-w#Fig8)), ruling out the possibility that SCNAs account for the lack of _PIK3CA_ and _PIK3R1_ single-nucleotide variants (SNVs) in TA-UC. Together, even when considering SNVs and SCNAs, _PIK3CA_ (33% versus 67%; _P_ = 0.002; Fisher’s exact test) and, to a lesser statistical extent, _PIK3R1_ (19% versus 51%; _P_ = 0.006) remained significantly less altered in TA-UC than in de novo UC (Fig. [1g](https://www.nature.com/articles/s41588-025-02308-w#Fig1)), distinguishing these two genes, especially _PIK3CA_, from other PI3K pathway genes[37](https://www.nature.com/articles/s41588-025-02308-w#ref-CR37 "Zhang, Y. et al. A pan-cancer proteogenomic atlas of PI3K/AKT/mTOR pathway alterations. Cancer Cell 31, 820–832 (2017).") in TA-UC (Extended Data Fig. [4c](https://www.nature.com/articles/s41588-025-02308-w#Fig9)).

We further investigated whether obesity, a surrogate for higher estrogen[38](https://www.nature.com/articles/s41588-025-02308-w#ref-CR38 "Dashti, S. G. et al. Adiposity and breast, endometrial, and colorectal cancer risk in postmenopausal women: quantification of the mediating effects of leptin, C-reactive protein, fasting insulin, and estradiol. Cancer Med. 11, 1145–1159 (2022)."),[39](https://www.nature.com/articles/s41588-025-02308-w#ref-CR39 "Kaaks, R., Lukanova, A. & Kurzer, M. S. Obesity, endogenous hormones, and endometrial cancer risk: a synthetic review. Cancer Epidemiol. Biomarkers Prev. 11, 1531–1543 (2002)."),[40](https://www.nature.com/articles/s41588-025-02308-w#ref-CR40 "Schmandt, R. E., Iglesias, D. A., Co, N. N. & Lu, K. H. Understanding obesity and endometrial cancer risk: opportunities for prevention. Am. J. Obstet. Gynecol. 205, 518–525 (2011).") due to its association with elevated endogenous estrogen levels[41](https://www.nature.com/articles/s41588-025-02308-w#ref-CR41 "Freeman, E. W., Sammel, M. D., Lin, H. & Gracia, C. R. Obesity and reproductive hormone levels in the transition to menopause. Menopause 17, 718–726 (2010)."), a known UC risk factor[42](https://www.nature.com/articles/s41588-025-02308-w#ref-CR42 "Onstad, M. A., Schmandt, R. E. & Lu, K. H. Addressing the role of obesity in endometrial cancer risk, prevention, and treatment. J. Clin. Oncol. 34, 4225–4230 (2016)."), has effects similar to tamoxifen. Of note, obesity is not a surrogate for exogenous unopposed estrogen exposure as in hormone replacement treatment, which is associated with a higher UC risk[43](https://www.nature.com/articles/s41588-025-02308-w#ref-CR43 "Smith, D. C., Prentice, R., Thompson, D. J. & Herrmann, W. L. Association of exogenous estrogen and endometrial carcinoma. N. Engl. J. Med. 293, 1164–1167 (1975)."),[44](https://www.nature.com/articles/s41588-025-02308-w#ref-CR44 "Ziel, H. K. & Finkle, W. D. Increased risk of endometrial carcinoma among users of conjugated estrogens. N. Engl. J. Med. 293, 1167–1170 (1975)."). We found no significant differences in _PIK3CA_ mutation frequencies across obesity categories (all _P_ ≥ 0.1; Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02308-w#Fig10) and Supplementary Note [6](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)). To more directly assess the differential effects of estrogen and tamoxifen, we performed transcriptomic analysis of human endometrial cells, which showed upregulation of PI3K pathway genes after tamoxifen, but not estradiol (E2), treatment (Supplementary Fig. [2a,b](https://www.nature.com/articles/s41588-025-02308-w#MOESM1) and Supplementary Note [7](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)). These findings suggest that tamoxifen activates the PI3K pathway, which is commonly mutationally activated in de novo UC, and provide evidence that tamoxifen and E2 have different effects on the uterus.

### Cohorts validate low _PIK3CA_ mutation frequency in TA-UC

In our validation analysis, we prioritized _PIK3CA_ for two reasons: (1) in UC, _PIK3CA_ is more frequently mutated than _PIK3R1_ (Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02308-w#Fig8)), allowing for a more statistically powerful analysis and (2) unlike _PIK3R1_, which may require additional factors for PI3K pathway regulation, _PIK3CA_ directly activates this pathway, making results more interpretable. We confirmed our results from our discovery cohort in three validation cohorts. First, we analyzed an additional 39 TA-UCs from the TAMARISK study (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and Extended Data Fig. [6a](https://www.nature.com/articles/s41588-025-02308-w#Fig11)) for _PIK3CA_ hotspot mutations (E542K, E545K, H1047R) and detected three (8%) by ddPCR (Extended Data Fig. [6b](https://www.nature.com/articles/s41588-025-02308-w#Fig11)), which is lower but consistent with the 14% ddPCR-defined hotspots in our discovery cohort (Extended Data Fig. [2f](https://www.nature.com/articles/s41588-025-02308-w#Fig7)). Second, a clinical database cohort subjected to gene panel sequencing (Extended Data Fig. [6c–e](https://www.nature.com/articles/s41588-025-02308-w#Fig11) and Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM3), [8](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and [9](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)) confirmed the low _PIK3CA_ mutation frequency in TA-UC (19% versus 47%; _P_ = 0.01; Fig. [2a](https://www.nature.com/articles/s41588-025-02308-w#Fig2)). This was not attributable to differences in population descriptors between TA-UC and de novo UC (combined _P_ = 0.02; stratified Fisher’s exact test; Supplementary Note [3](https://www.nature.com/articles/s41588-025-02308-w#MOESM1) and Supplementary Fig. [1b](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)). Third, analysis of another clinicogenomic dataset (Extended Data Fig. [6f,g](https://www.nature.com/articles/s41588-025-02308-w#Fig11) and Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and [10](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)) corroborated a lower _PIK3CA_ mutation frequency in TA-UC (19%) compared to de novo UC (43%; _P_ = 0.001; Fig. [2b](https://www.nature.com/articles/s41588-025-02308-w#Fig2)). However, histological subtype frequencies in this dataset differed from the general patient population (based on SEER9 data) and further varied between TA-UC and de novo UC (Extended Data Fig. [6h](https://www.nature.com/articles/s41588-025-02308-w#Fig11) and Supplementary Table [9](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). To address this potential confounding factor, we performed a stratified Fisher’s exact test, which confirmed the lower _PIK3CA_ mutation frequency in TA-UC (combined _P_ = 0.01). Building on this, we next explored subtype-specific differences and extended our analysis to include both _PIK3CA_ and _PIK3R1_ mutation frequencies. Given the smaller sample sizes in some subtypes, we first calculated the statistical power to detect differences in mutation frequency between groups ([Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9)). Of the three powered subtypes (Bonferroni-corrected (_n_ = 8) optimal _P_ value < 0.05), endometrioid, mixed and other, and serous and clear cell endometrial UC showed significantly lower _PIK3CA_ mutation frequencies (20% versus 52%, _P_ = 0.04; 7% versus 37%, _P_ = 0.01; one-sided Fisher’s exact test; Supplementary Table [11](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). However, the dataset was underpowered to detect differences in _PIK3R1_ mutation frequencies between TA-UC and de novo UC. This is consistent with the generally lower frequency of _PIK3R1_ mutations than _PIK3CA_ mutations in de novo UC (26% versus 43%; _P_ < 2 × 10−16; two-sided Fisher’s exact test; Extended Data Fig. [6i](https://www.nature.com/articles/s41588-025-02308-w#Fig11)), suggesting that larger datasets are needed to test for differences in _PIK3R1_ mutations. However, to address this with the existing data, as _PIK3CA_ and _PIK3R1_ together encode the enzyme PI3K, we analyzed the combined mutation status and found that _PIK3CA_\- and/or _PIK3R1-_mutated tumors were less frequent in TA-UC (_P_ = 0.01; Fig. [2c](https://www.nature.com/articles/s41588-025-02308-w#Fig2)). Thus, this is consistent with our hypothesis that PI3K signaling represents a molecularly distinct feature of TA-UCs.

**Fig. 2: Independent clinical TA-UC cohorts confirm reduced _PIK3CA_ mutation frequency.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02308-w/figures/2)

**a**, Bar plot of clinical gene panel sequencing for TA-UC and de novo UC; bars represent _PIK3CA_ mutation frequencies; error bars reflect s.d. from the β-distribution; significance analysis by two-sided Fisher’s exact test; numbers in bars indicate mutated tumor count per group. **b**, Bar plot of clinical WES data for TA-UC and de novo UC; bars represent _PIK3CA_ mutation frequencies; error bars reflect s.d. from the β-distribution; numbers in bars indicate mutated tumor count per group. Significance analysis by two-sided Fisher’s exact test. **c**, Bar plot of WES data for TA-UC and de novo UC; bars represent _PIK3CA_ and/or _PIK3R1_ mutation frequencies; error bars reflect s.d. from the β-distribution; numbers in bars indicate mutated tumor count per group. Significance analysis by two-sided Fisher’s exact test.

We took a conservative approach by including only de novo UC from patients without a history of breast cancer as controls to confidently exclude patients with potential undocumented tamoxifen treatment. However, to further isolate the effect of tamoxifen on _PIK3CA_ mutation frequencies, we also compared clinicogenomic TA-UCs with a unique cohort of de novo UC from patients with breast cancer never treated with tamoxifen. Here, TA-UC also had a significantly lower _PIK3CA_ mutation frequency (_P_ = 0.005; two-sided Fisher’s exact test; Extended Data Fig. [6j](https://www.nature.com/articles/s41588-025-02308-w#Fig11)). Thus, a history of a breast cancer diagnosis before UC diagnosis cannot explain the lower frequency of _PIK3CA_ mutations observed in TA-UC compared to de novo UC. Collectively, the consistent finding of a lower frequency of _PIK3CA_ mutations in TA-UC across multiple cohorts, including real-world cohorts, supports a tamoxifen-specific effect and highlights the relevance of this discovery to clinical practice.

Most TA-UCs (12 of 21) and de novo UCs (472 of 554) in the discovery cohorts had at least one SNV event in a PI3K pathway gene[37](https://www.nature.com/articles/s41588-025-02308-w#ref-CR37 "Zhang, Y. et al. A pan-cancer proteogenomic atlas of PI3K/AKT/mTOR pathway alterations. Cancer Cell 31, 820–832 (2017).") (Extended Data Fig. [4c](https://www.nature.com/articles/s41588-025-02308-w#Fig9)). Consistent with previous reports[45](https://www.nature.com/articles/s41588-025-02308-w#ref-CR45 "Yuan, T. L. & Cantley, L. C. PI3K pathway alterations in cancer: variations on a theme. Oncogene 27, 5497–5510 (2008)."), multiple PI3K-related genes were often mutated within individual samples in both cohorts (Extended Data Fig. [2g](https://www.nature.com/articles/s41588-025-02308-w#Fig7)). However, TA-UC had a lower number of concurrent PI3K pathway mutations (median of one event per sample, range of 0–6) than de novo UC (median of two events per sample, range of 0–45; _P_ = 0.0002), suggesting fewer potential driver events that activate PI3K signaling in TA-UC. We explored the oncogenic role of _PIK3CA_ mutations in the context of other PI3K pathway events and observed a significant co-occurrence of _PTEN_ mutations with _PIK3CA_ mutations in de novo UC (odds ratio = 2, _P_ = 0.007; Fisher’s exact test), reflecting their known complementary but distinct functional roles[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."),[46](https://www.nature.com/articles/s41588-025-02308-w#ref-CR46 "Cheung, L. W. et al. High frequency of PIK3R1 and PIK3R2 mutations in endometrial cancer elucidates a novel mechanism for regulation of PTEN protein stability. Cancer Discov. 1, 170–185 (2011)."). By contrast, this co-occurrence was not observed in TA-UC (_P_ = 0.07), despite a similar frequency of _PTEN_ mutations (_Q_ = 0.2, BH-corrected Fisher’s exact test; Fig. [1e](https://www.nature.com/articles/s41588-025-02308-w#Fig1)). In addition, we observed almost complete mutual exclusivity between tamoxifen use (using our discovery cohorts and two TCGA patients with TA-UC) and _PIK3CA_ mutations (odds ratio = 0.2, _P_ = 0.001; Fisher’s exact test). In aggregate, these observations support the hypothesis that tamoxifen may act as an alternative mechanism for PI3K pathway activation in the absence of _PIK3CA_ mutations.

### In vivo studies support tamoxifen-induced PI3K signaling

To test the hypothesis that tamoxifen-mediated activation of ER affects PI3K signaling in the uterus, we performed in vivo studies in mice, initially analyzing the effects of E2 and tamoxifen on ER in the uterus. Because most UCs, including TA-UCs, develop in postmenopausal women[47](https://www.nature.com/articles/s41588-025-02308-w#ref-CR47 "Fleming, C. A. et al. Meta-analysis of the cumulative risk of endometrial malignancy and systematic review of endometrial surveillance in extended tamoxifen therapy. Br. J. Surg. 105, 1098–1106 (2018)."), we performed these experiments under postmenopausal conditions. To test the effects of E2, we used a relatively low dose to reflect the lower, clinically acceptable doses of exogenous estrogen currently permitted due to the risk of UC with unopposed estrogen[43](https://www.nature.com/articles/s41588-025-02308-w#ref-CR43 "Smith, D. C., Prentice, R., Thompson, D. J. & Herrmann, W. L. Association of exogenous estrogen and endometrial carcinoma. N. Engl. J. Med. 293, 1164–1167 (1975)."),[44](https://www.nature.com/articles/s41588-025-02308-w#ref-CR44 "Ziel, H. K. & Finkle, W. D. Increased risk of endometrial carcinoma among users of conjugated estrogens. N. Engl. J. Med. 293, 1167–1170 (1975)."). Female C57BL/6 mice were oophorectomized after sexual maturity and treated with (1) vehicle control (E2 deprived), (2) E2 or (3) tamoxifen, and uteri were collected 30 d after treatment. The uteri from the vehicle control showed an atrophic epithelial lining composed of a single layer of flattened cells devoid of glands (Fig. [3a,b](https://www.nature.com/articles/s41588-025-02308-w#Fig3)), confirming E2 dependency of endometrial epithelial cells. As expected, E2 supplementation promoted duct proliferation (mean number of ducts per mouse in E2 (16.8) versus vehicle (1.7), _P_ = 0.0048; one-way ANOVA with Tukey correction; Fig. [3c](https://www.nature.com/articles/s41588-025-02308-w#Fig3)) and enhanced cell growth (mean length of luminal epithelial cells per mouse in E2 (24.7 µm) versus vehicle (9.2 µm), _P_ = 0.004; Fig. [3d](https://www.nature.com/articles/s41588-025-02308-w#Fig3)). Tamoxifen enhanced the increase in the number of ducts and cell length compared to E2 (mean number of ducts per mouse in tamoxifen (28.1) versus E2 (16.8), _P_ = 0.007; mean length of luminal epithelial cells per mouse in tamoxifen (39.4 µm) versus E2 (24.7 µm), _P_ = 0.0015; Fig. [3c,d](https://www.nature.com/articles/s41588-025-02308-w#Fig3)), suggesting that the effects of tamoxifen on the endometrium are distinct from those of E2 at these doses.

**Fig. 3: Tamoxifen affects cell morphology and PI3K signaling in mouse endometrial epithelial cells.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02308-w/figures/3)

**a**,**b**, Representative hematoxylin and eosin (H&E)-stained endometrial sections from oophorectomized mice treated with vehicle, E2 or tamoxifen. Scale bars, 200 μm in **a**, 20 μm in **b**. **c**,**d**, Quantification of endometrial changes with number of ducts per mouse in **c** and mean length of luminal epithelial cells in **d**. Each symbol represents the mean of six sections per biologically independent mouse; sample sizes: vehicle (Veh), _n_ = 2 (small horn size and extensive fibrosis in the region surrounding the horns secondary to the oophorectomy made dissection difficult); E2, _n_ = 3; tamoxifen (Tam), _n_ = 5. Center line depicts median; error bars represent s.e.m.; significance analysis using one-way ANOVA. **e**, Volcano plot depicts differentially expressed genes identified using DESeq2 by comparing tamoxifen versus vehicle in endometrial epithelial cells (_Q_ < 0.01, BH-corrected two-sided Wald test). Red indicates upregulated (log2 (FC) > 1), blue indicates downregulated (log2 (FC) < −1), and genes not significantly changed are gray. **f**, Pathway enrichment analysis on the differently expressed genes from **e**. Bar plot depicts the odds ratio of pathway enrichment of MSigDB oncogenic signatures (gene set names shown on the _y_ axis) in tamoxifen-versus-vehicle upregulated genes (log2 (FC) > 2, _Q_ < 0.01, DESeq2); purple line indicates _Q_ values from BH-corrected two-sided Fisher’s exact tests. **g**, Differentially expressed genes comparing tamoxifen versus E2 treatment, analyzed as described in **e**. **h**, Pathway enrichment analysis of the differently expressed genes from **g**. Bar plot depicts the odds ratio of pathway enrichment in tamoxifen-upregulated genes when compared to E2 treatment, analyzed as described in **f**. **i**–**l**, Left: representative immunohistochemistry (IHC) images with H&E counterstaining showing expression (brown) of phospho-insulin receptor (pIR) or IGF1R (Tyr1162/Tyr1163) in **i**, phospho-AKT (pAKT) (Thr308) in **j**, pS6 (Ser240/Ser244) in **k** and Ki-67 in **l** in the endometrial epithelium from mice treated with vehicle, tamoxifen or tamoxifen plus alpelisib (Tam + Alp). Scale bars, 20 μm. Right: quantification of immunoreactivity shown as _H_ scores (product of percent positive cells × signal intensity in optical density). Each symbol represents the mean of five regions per biologically independent mouse, imaged at 20× magnification; sample sizes: vehicle, _n_ = 2 (small horn size and surrounding fibrosis secondary to oophorectomy made dissection difficult); tamoxifen (**i**,**j**,**l**, _n_ = 3; **k**, _n_ = 5); tamoxifen and alpelisib (**i**,**k**,**l**, _n_ = 5; **j**, _n_ = 3). Center line depicts median; error bars represent s.e.m. Significance analysis by one-way ANOVA.

To identify how tamoxifen increases epithelial cell proliferation through ER and, more specifically, to test the role of the PI3K pathway, we performed differential gene expression analysis of RNA sequencing (RNA-seq) from single-cell suspensions of endometrial epithelial cells isolated from mice treated with vehicle control, E2, tamoxifen or tamoxifen plus alpelisib, an α-selective PI3K inhibitor[48](https://www.nature.com/articles/s41588-025-02308-w#ref-CR48 "André, F. et al. Alpelisib for PIK3CA-mutated, hormone receptor-positive advanced breast cancer. N. Engl. J. Med. 380, 1929–1940 (2019).") (Extended Data Fig. [7a,b](https://www.nature.com/articles/s41588-025-02308-w#Fig12)). DESeq2 analysis identified 1,276 upregulated (log2 (fold change (FC)) > 1; _Q_ < 0.01, BH-corrected Wald test) and 1,103 downregulated (log2 (FC) < −1; _Q_ < 0.01) genes in the tamoxifen- versus vehicle-treated mice (Fig. [3e](https://www.nature.com/articles/s41588-025-02308-w#Fig3) and Supplementary Table [12](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). Pathway analysis of genes upregulated after tamoxifen treatment showed enrichment in genes involved in the receptor tyrosine kinase (RTK)–PI3K–AKT signaling pathway (Fig. [3f](https://www.nature.com/articles/s41588-025-02308-w#Fig3)). As most de novo UCs express ER and are associated with ER activation[49](https://www.nature.com/articles/s41588-025-02308-w#ref-CR49 "Rodriguez, A. C., Blanchard, Z., Maurer, K. A. & Gertz, J. Estrogen signaling in endometrial cancer: a key oncogenic pathway with several open questions. Horm. Cancer 10, 51–63 (2019)."), we assessed differences between tamoxifen and E2 treatment. We identified 1,373 upregulated and 1,338 downregulated genes in tamoxifen- versus E2-treated endometrial epithelial cells, respectively (|log2 (FC)| > 1; _Q_ < 0.01; Fig. [3g](https://www.nature.com/articles/s41588-025-02308-w#Fig3)). Genes upregulated after tamoxifen treatment were enriched in genes involved in the PI3K–AKT–mechanistic target of rapamycin (mTOR) and WNT signaling pathways (Fig. [3h](https://www.nature.com/articles/s41588-025-02308-w#Fig3)). By contrast, genes upregulated with E2 supplementation were enriched in gene sets associated with enhancer of zeste 2 polycomb repressive complex 2 subunit (EZH2) knockdown (PRC2 EZH2 UP.V1 UP) and proliferation (E2F3 UP.V1 UP; Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02308-w#Fig13)). Furthermore, when comparing tamoxifen or E2 to vehicle, 314 tamoxifen-upregulated genes (of the 1,276 in Fig. [3e](https://www.nature.com/articles/s41588-025-02308-w#Fig3)) overlapped with the E2-upregulated genes (_n_ = 686, log2 (FC) > 1; _Q_ < 0.01 versus vehicle; Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02308-w#Fig13)). Pathway analysis showed that genes uniquely upregulated by tamoxifen but not genes upregulated by E2 alone or by both tamoxifen and E2 were enriched in the AKT–mTOR pathway (Extended Data Fig. [8c–e](https://www.nature.com/articles/s41588-025-02308-w#Fig13)). Thus, the effects of tamoxifen over 30 d were distinct from those of E2 at this dose in terms of the AKT–mTOR pathway. Lastly, the addition of alpelisib to tamoxifen significantly downregulated tamoxifen-upregulated genes (Extended Data Fig. [8f](https://www.nature.com/articles/s41588-025-02308-w#Fig13) and Supplementary Table [13](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)), indicating that the effect of tamoxifen was at least partially through PI3K signaling.

We next deciphered key components of the tamoxifen–PI3K signaling axis. Crosstalk between ER and the PI3K–AKT pathway is well described[50](https://www.nature.com/articles/s41588-025-02308-w#ref-CR50 "Adesanya, O. O., Zhou, J., Samathanam, C., Powell-Braxton, L. & Bondy, C. A. Insulin-like growth factor 1 is required for G2 progression in the estradiol-induced mitotic cycle. Proc. Natl Acad. Sci. USA 96, 3287–3291 (1999)."),[51](https://www.nature.com/articles/s41588-025-02308-w#ref-CR51 "Klotz, D. M., Hewitt, S. C., Korach, K. S. & Diaugustine, R. P. Activation of a uterine insulin-like growth factor I signaling pathway by clinical and environmental estrogens: requirement of estrogen receptor-α. Endocrinology 141, 3430–3439 (2000)."). ER mediates insulin-like growth factor 1 (IGF1) synthesis, which activates the IGF1 receptor (IGF1R), followed by downstream PI3K–AKT pathway activation. IGF1-stimulated IGF1R can also activate ER, at least in part through PI3K–AKT-mediated phosphorylation of ER, creating a positive feedback loop[52](https://www.nature.com/articles/s41588-025-02308-w#ref-CR52 "Aronica, S. M. & Katzenellenbogen, B. S. Stimulation of estrogen receptor-mediated transcription and alteration in the phosphorylation state of the rat uterine estrogen receptor by estrogen, cyclic adenosine monophosphate, and insulin-like growth factor-I. Mol. Endocrinol. 7, 743–752 (1993)."),[53](https://www.nature.com/articles/s41588-025-02308-w#ref-CR53 "Martin, M. B. et al. A role for Akt in mediating the estrogenic functions of epidermal growth factor and insulin-like growth factor I. Endocrinology 141, 4503–4511 (2000)."). We therefore interrogated the impact of tamoxifen and alpelisib treatment on the IGF1R–PI3K–AKT axis in the uterus. Indeed, tamoxifen-activated IGF1R–PI3K–AKT signaling was evidenced by the significant increase in phospho-IGF1R (_P_ = 0.001; one-way ANOVA; Fig. [3i](https://www.nature.com/articles/s41588-025-02308-w#Fig3)), phospho-AKT (_P_ = 0.02; Fig. [3j](https://www.nature.com/articles/s41588-025-02308-w#Fig3)) and phospho-S6 (_P_ = 0.001; Fig. [3k](https://www.nature.com/articles/s41588-025-02308-w#Fig3)). Alpelisib abrogated the tamoxifen-induced increase in PI3K–AKT signaling, IGF1R activation (Fig. [3i–k](https://www.nature.com/articles/s41588-025-02308-w#Fig3)) and cell proliferation (Fig. [3l](https://www.nature.com/articles/s41588-025-02308-w#Fig3)), suggesting that tamoxifen-induced proliferation occurs via ER and IGF1R crosstalk-mediated activation of PI3K signaling.

Because ER is expressed in both endometrial epithelial and stromal cells independent of treatment conditions (Extended Data Fig. [8g](https://www.nature.com/articles/s41588-025-02308-w#Fig13)), and previous studies provided conflicting data for a paracrine versus autocrine effect[54](https://www.nature.com/articles/s41588-025-02308-w#ref-CR54 "Kashima, H. et al. Autocrine stimulation of IGF1 in estrogen-induced growth of endometrial carcinoma cells: involvement of the mitogen-activated protein kinase pathway followed by up-regulation of cyclin D1 and cyclin E. Endocr. Relat. Cancer 16, 113–122 (2009)."),[55](https://www.nature.com/articles/s41588-025-02308-w#ref-CR55 "Cooke, P. S. et al. Stromal estrogen receptors mediate mitogenic effects of estradiol on uterine epithelium. Proc. Natl Acad. Sci. USA 94, 6535–6540 (1997)."), we next asked how the tamoxifen-mediated effect on ER activates the IGF1R–PI3K–AKT pathway in the uterus. We examined the transcriptomic levels of _Igf1_ and _Igf2_ as well as their receptors (_Igf1r_, _Igf2r_) and IGF-binding proteins (_Igfbp1_–_Igfbp6_) in endometrial epithelial cells in uteri from mice treated with vehicle control, E2 and tamoxifen with or without alpelisib. Tamoxifen-treated mice showed a significant decrease in _Igfbp3_, _Igfbp4_ and _Igfbp6_ transcript levels compared to vehicle control (_Igfbp3_, log2 (FC) = −7, _Q_ = 6 × 10−37, DESeq2; _Igfbp4_, log2 (FC) = −1.7, _Q_ = 2 × 10−5; _Igfbp6_, log2 (FC) = −1.7, _Q_ = 3 × 10−5; Extended Data Fig. [8h](https://www.nature.com/articles/s41588-025-02308-w#Fig13)). As IGF-binding proteins, particularly IGFBP3, regulate the bioavailability of IGF in circulation and in the cell[56](https://www.nature.com/articles/s41588-025-02308-w#ref-CR56 "Baxter, R. C. Signalling pathways involved in antiproliferative effects of IGFBP-3: a review. Mol. Pathol. 54, 145–148 (2001)."), these decreased levels suggest a possible cell-intrinsic tamoxifen-mediated effect by which IGF1 has increased availability upstream of PI3K–AKT in endometrial epithelial cells. The addition of the PI3K inhibitor alpelisib to tamoxifen increased _Igfbp3_ (log2 (FC) = 4, _Q_ = 1.5 × 10−12) and _Igfbp6_ (log2 (FC) = 1.7, _Q_ = 6.2 × 10−13) levels (Extended Data Fig. [8h](https://www.nature.com/articles/s41588-025-02308-w#Fig13)). Given the low _Igf1_ messenger RNA (mRNA) levels observed in mouse epithelial endometrial cells in all four conditions in the RNA-seq data (Extended Data Fig. [8h](https://www.nature.com/articles/s41588-025-02308-w#Fig13)), we used RNAscope, an in situ hybridization assay, to detect mRNA within the intact tissue architecture. Consistent with the RNA-seq data, _Igf1_ levels were low in endometrial epithelial cells and predominantly detected in the stroma (_P_ = 0.025, paired two-sided _t_\-test; Fig. [4a,b](https://www.nature.com/articles/s41588-025-02308-w#Fig4)). These results suggest that tamoxifen-induced activation of the IGF1R–PI3K axis in endometrial epithelial cells is potentially mediated by paracrine (IGF1 secreted by stromal cells) and cell-intrinsic (decreased levels of IGFBP3 in epithelial cells) effects. Together, our in vivo and genomic findings suggest that tamoxifen activates PI3K signaling, contributing to increased cell proliferation and likely uterine carcinogenesis independent of oncogenic _PIK3CA_ mutations.

**Fig. 4: _Igf1_ expression in mouse endometrial stromal cells.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02308-w/figures/4)

**a**, Representative RNAscope images of _Igf1_ expression in mouse uteri (three mice, treated as indicated). Dashed white lines depict the border between the epithelium and the stroma. White foci represent _Igf1_ mRNA signal (top); merged images (bottom) show 4′,6-diamidino-2-phenylindole (DAPI) (teal) and _Igf1_ (red). Different contrast settings were used for top and bottom images of the vehicle control. Scale bars, 20 μm. **b**, Mean _Igf1_ staining intensity per nucleus across entire uterine tissue areas (epithelium and stroma) per biologically independent mouse (_n_ = 3). Significance analysis by paired two-sided _t_\-test. Center line depicts median; error bars represent s.e.m.

### TA-UCs have fewer clonal driver mutations

Our preclinical findings showed that PI3K pathway activation by tamoxifen occurs in a short period of time. We therefore sought to understand the timing of driver events in TA-UC and infer the early events in TA-UC compared to de novo UC and clonally expanded normal endometrial cells.

First, using discovery WES data and our PhylogicNDT suite of tools[57](https://www.nature.com/articles/s41588-025-02308-w#ref-CR57 "Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. Cell 184, 2239–2254 (2021)."),[58](https://www.nature.com/articles/s41588-025-02308-w#ref-CR58 "Leshchiner, I. et al. Inferring early genetic progression in cancers with unobtainable premalignant disease. Nat. Cancer 4, 550–563 (2023)."), we identified early clonal driver mutations in TA-UC and de novo UC (Supplementary Table [14](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). Comparing these events between cohorts, we found no difference in the timing of early driver events (Extended Data Fig. [2h](https://www.nature.com/articles/s41588-025-02308-w#Fig7)). However, TA-UC harbored significantly fewer early genomic events per sample (median, one event) than de novo UC (median, two events; _P_ = 0.02; Wilcoxon test; Fig. [5a](https://www.nature.com/articles/s41588-025-02308-w#Fig5)). The shift was not significantly larger than one event (TA-UC events + 1 versus de novo UCs, _P_ = 0.4), leading us to hypothesize that tamoxifen-associated perturbation of the PI3K signaling pathway acts as the missing driver event toward malignant transformation in the uterus.

**Fig. 5: Mutations in _PIK3CA_ are early events in tumorigenesis.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02308-w/figures/5)

**a**, Density histogram with bars representing fraction of tumors grouped by number of clonal mutations in commonly mutated early driver genes (Supplementary Table [14](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)) per sample; error bars reflect s.d. from the β-distribution; significance analysis by two-sided Wilcoxon test; numbers in or above bars indicate the mutated tumor count per group. **b**, Estimated phylogenetic trees (top), relative order and molecular timing of events (bottom) in _PIK3CA-_mutated TA-UC (discovery cohort). Circle plots indicate estimated clonal composition. **c**, Bar plot of WES and ddPCR data for TAMARISK TA-UC samples, normal endometrial tissue[62](https://www.nature.com/articles/s41588-025-02308-w#ref-CR62 "Lac, V. et al. Oncogenic mutations in histologically normal endometrium: the new normal? J. Pathol. 249, 173–181 (2019).") and benign endometrial disease endometriosis[63](https://www.nature.com/articles/s41588-025-02308-w#ref-CR63 "Anglesio, M. S. et al. Cancer-associated mutations in endometriosis without cancer. N. Engl. J. Med. 376, 1835–1848 (2017)."),[64](https://www.nature.com/articles/s41588-025-02308-w#ref-CR64 "Praetorius, T. H. et al. Molecular analysis suggests oligoclonality and metastasis of endometriosis lesions across anatomically defined subtypes. Fertil. Steril. 118, 524–534 (2022).") and atypical hyperplasia[65](https://www.nature.com/articles/s41588-025-02308-w#ref-CR65 "Li, L. et al. Genome-wide mutation analysis in precancerous lesions of endometrial carcinoma. J. Pathol. 253, 119–128 (2021)."),[66](https://www.nature.com/articles/s41588-025-02308-w#ref-CR66 "Hu, Z. et al. Proteogenomic insights into early-onset endometrioid endometrial carcinoma: predictors for fertility-sparing therapy response. Nat. Genet. 56, 637–651 (2024).") (AH); bars represent _PIK3CA_ mutation frequencies; error bars reflect s.d. from the β-distribution; numbers in bars indicate mutated tumor count per group; significance analysis by two-sided Fisher’s exact test. **d**, Schematic illustration depicting (1) _PIK3CA_ mutations in TA-UC and de novo UC (top two left subpanels; bars represent mutation frequencies; error bars reflect s.d. from the β-distribution), (2) the in vivo mouse model (top right) with cell morphological changes from normal atrophic (no tamoxifen (Tam)) and normal proliferative (+E2) to increased number of ducts and cell hypertrophy (with tamoxifen) and normalized number of ducts and cell length (with tamoxifen and PI3K inhibitor), (3) the model of PI3K signaling induced by tamoxifen (middle right) and (4) the model of UC evolution for de novo UC and TA-UC (bottom).

We next analyzed the timing of _PIK3CA_ mutations in TA-UC, focusing on the small subset of patients in whom _PIK3CA_ mutations were detected. Although the overall number of _PIK3CA_ mutations in TA-UC was lower than expected, we identified three patients with _PIK3CA_ mutation by WES and one additional patient with _PIK3CA_ mutation by ddPCR (Supplementary Note [5](https://www.nature.com/articles/s41588-025-02308-w#MOESM1)) in our discovery cohort. One possible explanation for this finding could be shorter tamoxifen exposure. However, no significant difference in intake time was observed between these four patients and the other ones with TA-UC (mean, 4.4 versus 3.6 years in mutant versus others; _P_ = 0.4). A second, alternative explanation is that these cases occurred by chance. Given previous calculations of a fivefold increase in absolute UC risk due to tamoxifen (from 0.5% in women not treated with tamoxifen to ~2.5% in women receiving tamoxifen over 10 years)[59](https://www.nature.com/articles/s41588-025-02308-w#ref-CR59 "van Leeuwen, F. E. et al. Risk of endometrial cancer after tamoxifen treatment of breast cancer. Lancet 343, 448–452 (1994)."), we expect approximately four women of our 21 patients with TA-UC to develop UC unrelated to tamoxifen treatment. This is consistent with the observed frequency of four _PIK3CA_ mutations. Of note, all three _PIK3CA_ mutations detected by WES, for which we could experimentally determine the cancer cell fraction (CCF), were clonal (CCF = 1; Extended Data Fig. [4a](https://www.nature.com/articles/s41588-025-02308-w#Fig9)). More specifically, these mutations were often early events, preceding whole-genome duplication (WGD; Fig. [5b](https://www.nature.com/articles/s41588-025-02308-w#Fig5)). Together, these findings are consistent with the presence of _PIK3CA_ mutations at early stages of cancer development and align with previous observations that the mutational activation of _PIK3CA_ is an early oncogenic event in UC[60](https://www.nature.com/articles/s41588-025-02308-w#ref-CR60 "Berg, A. et al. Molecular profiling of endometrial carcinoma precursor, primary and metastatic lesions suggests different targets for treatment in obese compared to non-obese patients. Oncotarget 6, 1327–1339 (2015)."). Given that clonally expanded normal endometrial cells can also harbor _PIK3CA_ mutations[61](https://www.nature.com/articles/s41588-025-02308-w#ref-CR61 "Moore, L. et al. The mutational landscape of normal human endometrial epithelium. Nature 580, 640–646 (2020)."), PI3K signaling activation might have occurred before UC initiation (and tamoxifen treatment). To test this, we compared _PIK3CA_ mutation frequencies between TA-UC and three noncancerous tissue types: untreated normal endometrium[62](https://www.nature.com/articles/s41588-025-02308-w#ref-CR62 "Lac, V. et al. Oncogenic mutations in histologically normal endometrium: the new normal? J. Pathol. 249, 173–181 (2019)."), benign disease endometriosis[63](https://www.nature.com/articles/s41588-025-02308-w#ref-CR63 "Anglesio, M. S. et al. Cancer-associated mutations in endometriosis without cancer. N. Engl. J. Med. 376, 1835–1848 (2017)."),[64](https://www.nature.com/articles/s41588-025-02308-w#ref-CR64 "Praetorius, T. H. et al. Molecular analysis suggests oligoclonality and metastasis of endometriosis lesions across anatomically defined subtypes. Fertil. Steril. 118, 524–534 (2022).") and atypical hyperplasia[65](https://www.nature.com/articles/s41588-025-02308-w#ref-CR65 "Li, L. et al. Genome-wide mutation analysis in precancerous lesions of endometrial carcinoma. J. Pathol. 253, 119–128 (2021)."),[66](https://www.nature.com/articles/s41588-025-02308-w#ref-CR66 "Hu, Z. et al. Proteogenomic insights into early-onset endometrioid endometrial carcinoma: predictors for fertility-sparing therapy response. Nat. Genet. 56, 637–651 (2024)."). TA-UC and noncancerous tissue had similar _PIK3CA_ mutation frequencies, a finding supported by both the TAMARISK discovery and validation cohorts (all _P_ > 0.2, Fisher’s exact test; Fig. [5c](https://www.nature.com/articles/s41588-025-02308-w#Fig5)). In aggregate, our observations that _PIK3CA_ mutations typically occur early in tumorigenesis or even before cancer onset highlight the importance of PI3K signaling as a driver event in UC in general. Their presence in TA-UC suggests that not all UCs in patients receiving tamoxifen are driven by tamoxifen-induced PI3K signaling. While tamoxifen likely mimics the role of _PIK3CA_ mutations, it does not prevent tumors from acquiring these mutations independently. However, tamoxifen decreases the selective advantage of these mutations, thereby reducing their frequency in TA-UC (Fig. [5d](https://www.nature.com/articles/s41588-025-02308-w#Fig5)).

Discussion
----------

In summary, we describe a previously uncharacterized mechanism of oncogenesis that promotes therapy-associated secondary cancer. In addition to the known mechanisms, including treatment-associated mutagenesis and clonal selection, we propose a nonmutagenic mechanism by which a drug activates an oncogenic pathway that is otherwise activated by driver mutations in de novo tumors.

While we found no evidence of tamoxifen being mutagenic in endometrial tissue, its effect on PI3K signaling through crosstalk with ER may eliminate the need for an additional oncogenic hit, accelerating the onset of UC and explaining the associated increased risk in tamoxifen-treated patients. The finding that tamoxifen likely confers a growth advantage to cells primed with preexisting UC driver mutations is supported by clinical observations of a higher TA-UC risk in postmenopausal women[47](https://www.nature.com/articles/s41588-025-02308-w#ref-CR47 "Fleming, C. A. et al. Meta-analysis of the cumulative risk of endometrial malignancy and systematic review of endometrial surveillance in extended tamoxifen therapy. Br. J. Surg. 105, 1098–1106 (2018).") older than 65 (ref. [20](https://www.nature.com/articles/s41588-025-02308-w#ref-CR20 "Fisher, B. et al. Tamoxifen for the prevention of breast cancer: current status of the National Surgical Adjuvant Breast and Bowel Project P-1 study. J. Natl Cancer Inst. 97, 1652–1662 (2005).")), as mutations accumulate in normal cells with age[67](https://www.nature.com/articles/s41588-025-02308-w#ref-CR67 "Yizhak, K. et al. RNA sequence analysis reveals macroscopic somatic clonal expansion across normal tissues. Science 364, eaaw0726 (2019)."). Furthermore, the role of tamoxifen as a potential driver of PI3K signaling activation is consistent with the observation that the excess risk of UC in tamoxifen-treated patients is mainly confined to the years of active treatment[19](https://www.nature.com/articles/s41588-025-02308-w#ref-CR19 "Cuzick, J. et al. Tamoxifen for prevention of breast cancer: extended long-term follow-up of the IBIS-I breast cancer prevention trial. Lancet Oncol. 16, 67–75 (2015).") and provides further reassurance to women who have completed tamoxifen treatment.

Although our discovery cohort was relatively small due to the rarity of this disease, our results of low _PIK3CA_ mutation frequencies in TA-UC were validated in three independent cohorts, including real-world clinicogenomic data, and supported by in vivo evidence that tamoxifen activates PI3K signaling in the uterus. We were unable to validate our _PIK3R1_ findings, which represents a limitation of the study. This is likely due to the lower overall _PIK3R1_ mutation frequency[37](https://www.nature.com/articles/s41588-025-02308-w#ref-CR37 "Zhang, Y. et al. A pan-cancer proteogenomic atlas of PI3K/AKT/mTOR pathway alterations. Cancer Cell 31, 820–832 (2017)."), indicating the need for larger datasets. Additionally, unlike our population-based discovery cohort, the validation datasets were derived from clinical databases, which may introduce bias from clinicians prioritizing sequencing of higher-risk disease, making direct validation of low-frequency mutations challenging. An alternative explanation is that _PIK3R1_, encoding the regulatory subunit p85α, may not directly drive tumorigenesis like _PIK3CA_, which encodes the catalytic subunit p110α. While _PIK3CA_ mutations result in constitutive PI3K pathway activation, _PIK3R1_ mutations may require additional genomic alterations to have an oncogenic effect, which we could not assess due to the lack of such data.

Consistent with previous reports demonstrating crosstalk between ER and the IGF1R–PI3K pathway[50](https://www.nature.com/articles/s41588-025-02308-w#ref-CR50 "Adesanya, O. O., Zhou, J., Samathanam, C., Powell-Braxton, L. & Bondy, C. A. Insulin-like growth factor 1 is required for G2 progression in the estradiol-induced mitotic cycle. Proc. Natl Acad. Sci. USA 96, 3287–3291 (1999)."),[51](https://www.nature.com/articles/s41588-025-02308-w#ref-CR51 "Klotz, D. M., Hewitt, S. C., Korach, K. S. & Diaugustine, R. P. Activation of a uterine insulin-like growth factor I signaling pathway by clinical and environmental estrogens: requirement of estrogen receptor-α. Endocrinology 141, 3430–3439 (2000)."),[52](https://www.nature.com/articles/s41588-025-02308-w#ref-CR52 "Aronica, S. M. & Katzenellenbogen, B. S. Stimulation of estrogen receptor-mediated transcription and alteration in the phosphorylation state of the rat uterine estrogen receptor by estrogen, cyclic adenosine monophosphate, and insulin-like growth factor-I. Mol. Endocrinol. 7, 743–752 (1993)."),[53](https://www.nature.com/articles/s41588-025-02308-w#ref-CR53 "Martin, M. B. et al. A role for Akt in mediating the estrogenic functions of epidermal growth factor and insulin-like growth factor I. Endocrinology 141, 4503–4511 (2000)."), we provide in vivo evidence that tamoxifen-induced ER activation stimulates PI3K signaling in the uterus, a response not seen with low-dose E2 supplementation. Our work also implies that this effect of tamoxifen involves an interaction between epithelial and stromal cells, ultimately instigating increased proliferation. Future studies will need to evaluate whether additional mechanisms, including those unrelated to genomic alterations, contribute to TA-UC development.

Our findings that alpelisib-mediated PI3K inhibition suppresses uterine cell proliferation suggest a strategy to prevent tamoxifen-induced UC while also supporting breast cancer treatment. In line with this, metformin, a drug known to reduce PI3K signaling[68](https://www.nature.com/articles/s41588-025-02308-w#ref-CR68 "Zhao, Y. et al. Metformin is associated with reduced cell proliferation in human endometrial cancer by inhibiting PI3K/AKT/mTOR signaling. Gynecol. Endocrinol. 34, 428–432 (2018)."), was shown to inhibit tamoxifen-induced endometrial proliferation in a randomized trial[69](https://www.nature.com/articles/s41588-025-02308-w#ref-CR69 "Davis, S. R. et al. The benefits of adding metformin to tamoxifen to protect the endometrium—a randomized placebo-controlled trial. Clin. Endocrinol. 89, 605–612 (2018)."). Furthermore, nonmutant-selective PI3K inhibitors[48](https://www.nature.com/articles/s41588-025-02308-w#ref-CR48 "André, F. et al. Alpelisib for PIK3CA-mutated, hormone receptor-positive advanced breast cancer. N. Engl. J. Med. 380, 1929–1940 (2019).") could potentially be exploited as a future therapeutic approach to prevent TA-UC development in patients who, in addition to tamoxifen, have multiple risk factors for UC development.

Methods
-------

### Ethics statement

This study complies with all relevant ethical regulations. TAMARISK specimens were obtained and sequenced with the approval of the institutional review boards (IRBs) of the Netherlands Cancer Institute (protocol CFMPB294) and the Dana-Farber Cancer Institute (DFCI) (protocol 12-049B). Approval to access clinical data from the DFCI was granted under protocols 17-000 and 11-104. All participants from both the TAMARISK and DFCI cohorts provided written informed consent, allowing their genomic and clinical data to be obtained and analyzed here. In accordance with the US Code of Federal Regulations, Title 45, Part 46, Section 104(d) (45 CFR §46.104(d)), the retrospective analysis of de-identified clinical data from Caris Life Sciences was deemed exempt by the IRB, which is the WIRB-Copernicus Group IRB (formerly known as WIRB). This exemption was granted because the data were fully de-identified and the research involved no intervention or interaction with human participants; therefore, informed patient consent was not required.

### Tamoxifen-associated uterine cancer from the TAMARISK study

We analyzed 60 primary TA-UCs from the TAMARISK study[28](https://www.nature.com/articles/s41588-025-02308-w#ref-CR28 "Hoogendoorn, W. E. et al. Prognosis of uterine corpus cancer after tamoxifen treatment for breast cancer. Breast Cancer Res. Treat. 112, 99–108 (2008)."), diagnosed between 1983 and 2002, for which sufficient residual tissue for DNA extraction was available (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02308-w#Fig6) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)). Of these, 21 samples and their matched normal counterparts underwent WES and constitute the discovery cohort. Another 39 TA-UC samples were subjected to ddPCR without matched normal counterparts and constitute the TAMARISK validation cohort. Formalin-fixed paraffin-embedded (FFPE) histopathology blocks were obtained, and H&E slides were reviewed by an expert pathologist to score tumor percentage and identify regions of high tumor content as well as regions of normal cells for isolation. Regions were macrodissected from five to ten 10-µm FFPE slides, and DNA was isolated from the excised tissue using the AllPrep DNA/RNA FFPE Isolation Kit (Qiagen, 80234) and the QIAcube according to the manufacturer’s protocols.

### Tamoxifen-associated uterine cancer from clinical databases

We identified a TA-UC clinical genomic data cohort by querying cancer registry data at the DFCI. We crossed the diagnosis of UC with the occurrence of breast cancer and tamoxifen treatment, searching for patients who had UC genotype data from the OncoPanel platform[70](https://www.nature.com/articles/s41588-025-02308-w#ref-CR70 "Sholl, L. M. et al. Institutional implementation of clinical tumor profiling on an unselected cancer population. JCI Insight 1, e87062 (2016)."). We identified an overall number of 120 patients, of whom 21 women had primary TA-UC (Extended Data Fig. [6c](https://www.nature.com/articles/s41588-025-02308-w#Fig11) and Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and [8](https://www.nature.com/articles/s41588-025-02308-w#MOESM3)), diagnosed between 2010 and 2022. A second TA-UC clinical genomic data cohort was obtained using the Caris Life Sciences internal cBioPortal, searching for patients treated with tamoxifen for breast cancer who were later diagnosed with UC. A total of 69 patients were identified, of whom 47 met the criteria for TA-UC, with diagnoses between 2015 and 2023 (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02308-w#MOESM3) and Extended Data Fig. [6g](https://www.nature.com/articles/s41588-025-02308-w#Fig11)). Two de novo UC control sets were also identified using the Caris Life Sciences cBioPortal instance: (1) 8,258 patients with primary UC and no prior breast cancer diagnosis and (2) 569 patients with a history of breast cancer but no tamoxifen treatment and primary UC negative for homologous recombination deficiency, identified by the absence of _BRCA1_ and _BRCA2_ driver mutations and/or a low genomic scar score[71](https://www.nature.com/articles/s41588-025-02308-w#ref-CR71 "Evans, E. et al. Whole exome sequencing provides loss of heterozygosity (LoH) data comparable to that of whole genome sequencing (171). Gynecol. Oncol. 166, S100 (2022)."). Genotype data were obtained as previously described[72](https://www.nature.com/articles/s41588-025-02308-w#ref-CR72 "Ogobuiro, I. et al. Multiomic characterization reveals a distinct molecular landscape in young-onset pancreatic cancer. JCO Precis. Oncol. 7, e2300152 (2023)."),[73](https://www.nature.com/articles/s41588-025-02308-w#ref-CR73 "Muquith, M. et al. Tissue-specific thresholds of mutation burden associated with anti-PD-1/L1 therapy benefit and prognosis in microsatellite-stable cancers. Nat. Cancer 5, 1121–1129 (2024)."). We assessed potential overlap between the two TA-UC clinicogenomic datasets by comparing de-identified clinical variables, including date of UC diagnosis, age at UC diagnosis, histological UC type and prior breast cancer diagnosis. No overlap was found between patients in the two datasets.

### Whole-exome sequencing

Whole-exome capture was performed from tumor and normal DNA at the Broad Institute. DNA was quantified in triplicate using a standardized PicoGreen dsDNA Quantitation Reagent (Invitrogen) assay. The quality control identification check was performed using fingerprint genotyping of 95 common SNVs by Fluidigm Genotyping (Fluidigm). Samples were plated at a concentration of 2 ng µl−1 and a volume of 50 µl into matrix tubes, which allowed for positive barcode tracking throughout processing. Samples were sheared using a Broad-developed protocol optimized for a size distribution of ~180 bp. Library construction was performed using the KAPA Library Prep kit with palindromic forked adaptors from Integrated DNA Technologies. Libraries were pooled before hybridization. Hybridization and capture were performed using the relevant components of Illumina’s Rapid Capture Enrichment Kit, with a 37-Mb target. All library construction, hybridization and capture steps were automated on the Agilent Bravo liquid-handling system. After post-capture enrichment, library pools were quantified using qPCR, normalized to 2 nM and denatured using 0.1 M NaOH on the Hamilton STARlet. Flow cell cluster amplification and sequencing were performed according to the manufacturer’s protocols (Illumina) on either the HiSeq 2000 version 3 or HiSeq 2500 runs and used sequencing-by-synthesis kits to produce 76-bp paired reads. The target coverage was 150× mean target coverage for each tumor sample and 60× mean target coverage for each normal sample.

### Genomic data alignment and quality control

Data derived from WES were processed using established analytical tools within the Firehose platform ([http://www.broadinstitute.org/cancer/cga/Firehose](http://www.broadinstitute.org/cancer/cga/Firehose)), which was later replaced with a cloud-based platform (FireCloud, Terra) operating on top of the Google Cloud Platform[74](https://www.nature.com/articles/s41588-025-02308-w#ref-CR74 "Auwera, Van der, G. A & O’Connor, B. D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra (O’Reilly Media, 2020)."). These platforms allow for coordinated and reproducible analysis of datasets using analytical pipelines. For each sample, the Picard data processing pipeline (version 2.9.2; [http://broadinstitute.github.io/picard/](http://broadinstitute.github.io/picard/)) combines data from multiple libraries and flow cell runs into a single BAM file. Sequencing reads were aligned to the hg19 human genome build using BWA ([http://bio-bwa.sourceforge.net](http://bio-bwa.sourceforge.net/)). All sample pairs of tumor and normal genotypes were subjected to testing the level of cross-contamination using ContEst version 4 (ref. [75](https://www.nature.com/articles/s41588-025-02308-w#ref-CR75 "Cibulskis, K. et al. ContEst: estimating cross-contamination of human samples in next-generation sequencing data. Bioinformatics 27, 2601–2602 (2011).")). We calculated the mean sequencing coverage for gene exonic regions using the DepthOfCoverage function from GATK version 4.1.6.0.

### Somatic mutation analysis

For each tumor–normal pair, somatic SNVs were called using MuTect (version 1)[76](https://www.nature.com/articles/s41588-025-02308-w#ref-CR76 "Cibulskis, K. et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. Nat. Biotechnol. 31, 213–219 (2013).") and small insertions and deletions (indels) with Strelka (version 2.9.0)[77](https://www.nature.com/articles/s41588-025-02308-w#ref-CR77 "Saunders, C. T. et al. Strelka: accurate somatic small-variant calling from sequenced tumor–normal sample pairs. Bioinformatics 28, 1811–1817 (2012)."). These SNVs and indels were annotated using Oncotator (version 1.9.9.0)[78](https://www.nature.com/articles/s41588-025-02308-w#ref-CR78 "Ramos, A. H. et al. Oncotator: cancer variant annotation tool. Hum. Mutat. 36, E2423–E2429 (2015)."). We excluded false-positive SNVs failing the following filters (version 25): (1) the OxoG filter[79](https://www.nature.com/articles/s41588-025-02308-w#ref-CR79 "Costello, M. et al. Discovery and characterization of artifactual mutations in deep coverage targeted capture sequencing data due to oxidative DNA damage during sample preparation. Nucleic Acids Res. 41, e67 (2013)."), which filters sequencing artifacts that are caused by oxidative damage to guanine during shearing in library preparation based on the read pair orientation bias, (2) the FFPE filter[80](https://www.nature.com/articles/s41588-025-02308-w#ref-CR80 "Giannakis, M. et al. Genomic correlates of immune-cell infiltrates in colorectal carcinoma. Cell Rep. 17, 1206 (2016)."), which filters sequencing artifacts caused by formaldehyde-induced deamination of cytosine based on the read pair orientation bias and (3) a mutational panel of normals[81](https://www.nature.com/articles/s41588-025-02308-w#ref-CR81 "Ellrott, K. et al. Scalable open science approach for mutation calling of tumor exomes using multiple genomic pipelines. Cell Syst. 6, 271–281 (2018).") built from FFPE samples sequenced using the same target regions, allowing us to filter the remaining potential sequencing artifacts as well as germline sites missed in the matched normal tissue. To recover SNVs lost to tumor-in-normal (TiN) contamination from adjacent tissue controls, we applied deTiN (version 3.0)[82](https://www.nature.com/articles/s41588-025-02308-w#ref-CR82 "Taylor-Weiner, A. et al. DeTiN: overcoming tumor-in-normal contamination. Nat. Methods 15, 531–534 (2018)."). In search for the presence of additional mutations (previously observed in TCGA de novo UCs) in the genes _ESR1_, _ESR2_, _PIK3CA_, _PIK3R1_ and _PTEN_, we applied a ‘force-calling’ method (version 2)[83](https://www.nature.com/articles/s41588-025-02308-w#ref-CR83 "Stachler, M. D. et al. Paired exome analysis of Barrett’s esophagus and adenocarcinoma. Nat. Genet. 47, 1047–1055 (2015)."), which calculates the number of reads supporting an alternate allele at predefined genomic coordinates. Manual review of mutations was performed using the Integrative Genomics Viewer[84](https://www.nature.com/articles/s41588-025-02308-w#ref-CR84 "Thorvaldsdottir, H., Robinson, J. T. & Mesirov, J. P. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief. Bioinform. 14, 178–192 (2013)."), and SNVs were filtered due to the following reasons: (1) low allelic fraction (AF) mutations, (2) mutations with orientation bias, (3) mutations called on reads that also contained indels and (4) mutations called in regions with poor mapping. Further downstream analysis was restricted to nonsynonymous mutations, ignoring mutations classified as 3′ UTR, 5′ UTR, IGR, intron, lincRNA, RNA or silent.

### Mutational significance analysis

Significance analysis of recurrently mutated genes was performed using MutSig2CV (version 3.11 with ‘gene\_min\_frac\_coverage\_required’ set to 0.02), which detects genes with a higher-than-expected SNV frequency or an unexpected pattern of SNVs[85](https://www.nature.com/articles/s41588-025-02308-w#ref-CR85 "Lawrence, M. S. et al. Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 505, 495–501 (2014)."). Significantly mutated genes were defined as genes with _Q_ < 0.1 using the method of Benjamini and Hochberg[86](https://www.nature.com/articles/s41588-025-02308-w#ref-CR86 "Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B. 57, 289–300 (1995).") to convert final _P_ values to false discovery rate _Q_ values. In addition, we used restricted hypothesis testing (as we have done previously[87](https://www.nature.com/articles/s41588-025-02308-w#ref-CR87 "Gopal, R. K. et al. Widespread chromosomal losses and mitochondrial DNA alterations as genetic drivers in Hürthle cell carcinoma. Cancer Cell 34, 242–255 (2018).")) using a panel of 113 previously published UC genes (Supplementary Table [4](https://www.nature.com/articles/s41588-025-02308-w#MOESM3))[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."),[30](https://www.nature.com/articles/s41588-025-02308-w#ref-CR30 "Cherniack, A. D. et al. Integrated molecular characterization of uterine carcinosarcoma. Cancer Cell 31, 411–423 (2017)."),[31](https://www.nature.com/articles/s41588-025-02308-w#ref-CR31 "Cancer Genome Atlas Research Network. Comprehensive and integrated genomic characterization of adult soft tissue sarcomas. Cell 171, 950–965 (2017)."),[34](https://www.nature.com/articles/s41588-025-02308-w#ref-CR34 "Gibson, W. J. et al. The genomic landscape and evolution of endometrial carcinoma progression and abdominopelvic metastasis. Nat. Genet. 48, 848–855 (2016).") to identify additional recurrently mutated genes. Because our aim was not to perform a de novo discovery of driver genes in the control cohort, we restricted the MutSig2CV analysis in the TCGA sample set of de novo UCs to the above panel of known UC drivers. We tested for mutual exclusivity and co-occurrence on a patient mutational level by applying Fisher’s exact test.

### Somatic copy number analysis

GATK4’s copy number variant discovery pipeline was used to analyze read coverage and detect copy number and allelic copy number alterations (release 4.1.6.0; variances of Gaussian kernel for copy ratio segmentation and allele fraction segmentation were set to 0.175 and 0.2, respectively). A copy number panel of normals used normal samples with low TiN to normalize the read depth at each capture probe. In addition, we tagged and removed copy number segments caused by potential germline events by comparing break points and reciprocal overlaps. Manual review of SCNAs was performed using the Integrative Genomics Viewer (version 2.16.2)[84](https://www.nature.com/articles/s41588-025-02308-w#ref-CR84 "Thorvaldsdottir, H., Robinson, J. T. & Mesirov, J. P. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief. Bioinform. 14, 178–192 (2013).").

### Copy number significance analysis

GISTIC2.0 (version 2.03.23)[36](https://www.nature.com/articles/s41588-025-02308-w#ref-CR36 "Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 12, R41 (2011).") was applied to detect significantly amplified or deleted SCNAs across a cohort using a threshold of _Q_ < 0.25. Peaks were annotated with genes from the Cancer Gene Census[88](https://www.nature.com/articles/s41588-025-02308-w#ref-CR88 "Sondka, Z. et al. The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Nat. Rev. Cancer 18, 696–705 (2018)."). _G_ scores were assigned to each peak considering the amplitude of the alteration and the frequency of its occurrence across specimens.

### ABSOLUTE, phylogeny and timing analyses

ABSOLUTE version 1.5 (ref. [89](https://www.nature.com/articles/s41588-025-02308-w#ref-CR89 "Carter, S. L. et al. Absolute quantification of somatic DNA alterations in human cancer. Nat. Biotechnol. 30, 413–421 (2012).")) was used to estimate purity (that is, the percentage of tumor cells in the cancer sample), ploidy (that is, the average copy number across the cancer genome), absolute copy numbers and WGD status for each tumor sample. ABSOLUTE solutions were manually curated. To determine whether mutations are clonal (that is, present in all tumor cells), we used the CCF of each mutation provided by ABSOLUTE (mutations with an estimated CCF ≥ 0.95 are considered clonal; mutations with lower CCFs are considered subclonal).

To analyze the phylogenetic relationship between tumor cell populations within a tumor, we used PhylogicNDT (version 35)[57](https://www.nature.com/articles/s41588-025-02308-w#ref-CR57 "Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. Cell 184, 2239–2254 (2021)."),[58](https://www.nature.com/articles/s41588-025-02308-w#ref-CR58 "Leshchiner, I. et al. Inferring early genetic progression in cancers with unobtainable premalignant disease. Nat. Cancer 4, 550–563 (2023)."), an N-dimensional Bayesian clustering framework based on mixtures of Dirichlet processes, in which the number of clusters is inferred over many Markov chain Monte Carlo iterations. Clusters of mutations with consistent CCF were used to determine the phylogenetic tree that best represents the clonal evolution. The tumor developmental trajectory was probabilistically determined, allowing us to order and estimate relative timing of clonal events and WGD (SinglePatientTiming and PhylogicNDT LeagueModel for ordering of events across a sample set).

### Prediction of microsatellite instability

MSI was predicted using MSIdetect (version 2) as described before[90](https://www.nature.com/articles/s41588-025-02308-w#ref-CR90 "Chung, J. et al. DNA polymerase and mismatch repair exert distinct microsatellite instability signatures in normal and malignant human cells. Cancer Discov. 11, 1176–1191 (2021)."). In short, MSIdetect assigns a probability for every read from a sequenced sample as coming from a tumor with MSI or an MSS tumor and aggregates it over all reads to generate an MSI score. Because the MSI score varies between sequencing platforms, we used normal samples to set the threshold between MSI and MSS patients.

### Mutational signature analysis

SignatureAnalyzer (version 0.0.8)[91](https://www.nature.com/articles/s41588-025-02308-w#ref-CR91 "Kim, J. et al. Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors. Nat. Genet. 48, 600–606 (2016)."), a Bayesian nonnegative matrix factorization method, was used to extract mutational signatures from SNVs by considering the 96 single-base substitutions within the trinucleotide sequence context. Signatures were then compared with previously described signatures in COSMIC version 3 ([https://cancer.sanger.ac.uk/cosmic/signatures](https://cancer.sanger.ac.uk/cosmic/signatures)). We also applied supervised Bayesian nonnegative matrix factorization implemented for GPUs[92](https://www.nature.com/articles/s41588-025-02308-w#ref-CR92 "Taylor-Weiner, A. et al. Scaling computational genomics to millions of individuals with GPUs. Genome Biol. 20, 228 (2019).") specifying a set of 13 expected COSMIC version 3 signatures (aging: SBS1, SBS5; MSI: SBS6, SBS14, SBS15, SBS20, SBS21, SBS26, SBS44; POLE: SBS10a, SBS10b, SBS14) to infer their contributions.

### Analysis of molecular subtypes

To replicate the molecular subtype analysis from TCGA[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."), we used the following approach. First, samples were assigned to the POLE subtype if they had POLE exonuclease domain mutations and associated mutational signatures (COSMIC signatures SBS10a, SBS10b and SBS14). Next, samples with MSI (MSI subtype) were classified using MSIdetect and then validated by the presence of mutational signatures associated with it[93](https://www.nature.com/articles/s41588-025-02308-w#ref-CR93 "Haradhvala, N. J. et al. Distinct mutational signatures characterize concurrent loss of polymerase proofreading and mismatch repair. Nat. Commun. 9, 1746 (2018).") (COSMIC signatures SBS6, SBS14, SBS15, SBS20, SBS21, SBS26 and SBS44). The remaining samples were categorized into two groups (CIN and genomically stable) based on their copy number pattern. As described previously[94](https://www.nature.com/articles/s41588-025-02308-w#ref-CR94 "Dou, Y. et al. Proteogenomic characterization of endometrial carcinoma. Cell 180, 729–748 (2020)."), the CIN subtype is characterized by a high rate of deletions. We calculated the fraction of the genome that was deleted by including copy number events of all lengths with a copy number change larger than a given threshold (_R_1 = 0.36). Because impure samples have a smaller change in copy number than samples with high purity, the threshold was normalized by the inferred purity. Samples were categorized as CIN when the fraction of the deleted genome was larger than a given threshold (_R_2 = 0.034). Molecular subtyping was applied to TA-UC and de novo TCGA UC where we did not have previous annotations for molecular subtypes; published molecular subtypes were used for endometrial carcinomas[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."). Above thresholds were determined by analyzing TCGA Uterine Corpus Endometrial Carcinoma data. ABSOLUTE purity data for TCGA samples were used from Taylor et al.[95](https://www.nature.com/articles/s41588-025-02308-w#ref-CR95 "Taylor, A. M. et al. Genomic and functional approaches to understanding cancer aneuploidy. Cancer Cell 33, 676–689 (2018).").

### Droplet digital PCR

ddPCR was used to detect hotspot mutations in the _PIK3CA_ and _ESR1_ genes using FFPE-derived DNA from (1) 19 TA-UCs that had undergone WES and had residual DNA and (2) an independent cohort of 39 TA-UC tumors. TaqMan PCR reaction mixtures were assembled from a 2× ddPCR master mix (Bio-Rad) and custom 40× TaqMan probes or primers made specific for each assay (Thermo Fisher Scientific). Assembled ddPCR reaction mixture (25 μl), which included either 5 μl DNA sample or water as a no-template control, was loaded into wells of a 96-well PCR plate. The heat-sealed PCR plate was subsequently loaded onto the Automated Droplet Generator (Bio-Rad). After droplet generation, the new 96-well PCR plate was heat sealed, placed on a conventional thermal cycler and amplified to the end point. After PCR, the 96-well PCR plate was read on the QX100 Droplet Reader (Bio-Rad). The primers applied in this analysis have been validated and described previously[96](https://www.nature.com/articles/s41588-025-02308-w#ref-CR96 "Kuang, Y. et al. Unraveling the clinicopathological features driving the emergence of ESR1 mutations in metastatic breast cancer. NPJ Breast Cancer 4, 22 (2018)."),[97](https://www.nature.com/articles/s41588-025-02308-w#ref-CR97 "Janiszewska, M. et al. In situ single-cell analysis identifies heterogeneity for PIK3CA mutation and HER2 amplification in HER2-positive breast cancer. Nat. Genet. 47, 1212–1219 (2015)."). Analysis of the ddPCR data was performed with QuantaSoft analysis software (Bio-Rad) that accompanied the droplet reader. We calculated the AF (in percent) as AF = (count mutant droplets)(count wild-type droplets + count mutant droplets)−1 × 100 and applied a cutoff of >2% AF to reduce FFPE-associated false positives.

### Published human datasets

For comparison of histologic subtypes, research data from 40,587 unique UC tumors diagnosed between 1973 and 2015 were obtained from the SEER9 registries (data released April 2018, based on the November 2017 submission). Tumors were distributed among the nine SEER registries as follows: 17% from San Francisco–Oakland, 13% from Connecticut, 16% from Metropolitan Detroit, 4% from Hawaii, 16% from Iowa, 5% from New Mexico, 16% from Seattle, 6% from Utah and 7% from Metropolitan Atlanta. To match the time frame of our cohorts, only tumors diagnosed between 1983 and 2002 were included. Primary site UCs (ICD-0-2 codes C54.0–C54.3, C54.8–C54.9, C55.9) classified as malignant (ICD-0-3 code 3) were used. To conservatively restrict the dataset to de novo UCs, women with breast cancer history (ICD-0-2 codes C50.0–C50.6, C50.8–C50.9) were excluded, as some may have developed TA-UC following prior tamoxifen treatment. Histologic subtypes were categorized as follows: endometrioid endometrial adenocarcinoma (8050, 8140, 8143, 8210, 8211, 8260, 8261, 8262, 8263, 8380, 8381, 8382, 8383, 8384, 8560, 8570); clear cell (8310) and serous adenocarcinoma (8441, 8460, 8461); mixed (8255, 8323); malignant Mullerian mixed tumors or carcinosarcoma (8950, 8951, 8980, 8981); and sarcoma (8890, 8891, 8896, 8930, 8931, 8935, 8933, 8800, 8801, 8802, 8803, 8804, 8805).

Additionally, we used 554 whole-exome sequenced primary de novo UC samples from TCGA for which data on absolute copy number, SNVs, survival, histological subtype and other clinical variables were available from the MC3 TCGA project[81](https://www.nature.com/articles/s41588-025-02308-w#ref-CR81 "Ellrott, K. et al. Scalable open science approach for mutation calling of tumor exomes using multiple genomic pipelines. Cell Syst. 6, 271–281 (2018).") (Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02308-w#Fig8)). CCFs were identified from the ABSOLUTE-annotated MAF file of the Pan-Cancer TCGA project and Haradhvala et al.[93](https://www.nature.com/articles/s41588-025-02308-w#ref-CR93 "Haradhvala, N. J. et al. Distinct mutational signatures characterize concurrent loss of polymerase proofreading and mismatch repair. Nat. Commun. 9, 1746 (2018).") for 536 of 554 TCGA UC samples. Copy number data were retrieved for a whitelisted set of 544 of 554 tumors. We applied the following criteria to identify de novo TCGA UC samples and exclude prior tamoxifen use: (1) 54 patients were annotated as having no prior tamoxifen use, (2) 482 patients had no prior diagnosis of a malignancy, (3) 16 patients had a prior diagnosis of cancer other than a breast malignancy and (4) two patients were diagnosed with breast cancer, but detailed treatment information excluded prior tamoxifen use. This set of 554 TCGA samples was composed of the following histological types: (1) a sample set containing 371 endometrioid endometrial adenocarcinomas, 96 serous endometrial adenocarcinomas and 19 mixed serous and endometrioid tumors from TCGA Uterine Corpus Endometrial Carcinoma[29](https://www.nature.com/articles/s41588-025-02308-w#ref-CR29 "Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. Nature 497, 67–73 (2013)."), (2) 52 uterine carcinosarcomas from TCGA-UCS[30](https://www.nature.com/articles/s41588-025-02308-w#ref-CR30 "Cherniack, A. D. et al. Integrated molecular characterization of uterine carcinosarcoma. Cancer Cell 31, 411–423 (2017).") and (3) 16 uterine sarcomas from TCGA-SARC[31](https://www.nature.com/articles/s41588-025-02308-w#ref-CR31 "Cancer Genome Atlas Research Network. Comprehensive and integrated genomic characterization of adult soft tissue sarcomas. Cell 171, 950–965 (2017)."). For 508 of these patients, height and weight data were available, and BMI was calculated by dividing body weight in kilograms by height in meters squared (kg m−2).

In addition, we searched TCGA annotation files and pathology reports to identify patients with UC and a previous history of tamoxifen use and identified two such patients with TA-UC in the TCGA cohort (TCGA TA-UCs TCGA-BG-A0MS and TCGA-IW-A3M6), who were analyzed separately.

Another set of 130 de novo UC specimens (111 endometrioid endometrial adenocarcinomas, 13 serous endometrial adenocarcinomas, three clear cell carcinomas, three not further defined) with available data on BMI as determined above were used from the Clinical Proteomic Tumor Analysis Consortium[94](https://www.nature.com/articles/s41588-025-02308-w#ref-CR94 "Dou, Y. et al. Proteogenomic characterization of endometrial carcinoma. Cell 180, 729–748 (2020).").

We also included 834 primary de novo UC specimens with consistent histology and available mutation data from unique patients from the AACR GENIE Project (version 13.0)[32](https://www.nature.com/articles/s41588-025-02308-w#ref-CR32 "AACR Project GENIE Consortium. AACR Project GENIE: powering precision medicine through an international consortium. Cancer Discov. 7, 818–831 (2017).") that originated from the DFCI. Patients with TA-UC (as identified at the DFCI and described above) were excluded. The final set included 527 endometrioid and mixed endometrial adenocarcinomas; 165 serous and clear cell tumors; 93 carcinosarcomas; and 49 leiomyosarcomas.

Although overlap between the US de novo UC cohorts (TCGA, GENIE, CARIS) is highly unlikely due to differences in sample origin, diagnosis data, histology and age at diagnosis, the use of de-identified data means that we cannot completely exclude this possibility, which is a limitation of the study.

In addition, somatic mutation sets from the following noncancerous FFPE tissue types were used: (1) normal endometrial tissue[62](https://www.nature.com/articles/s41588-025-02308-w#ref-CR62 "Lac, V. et al. Oncogenic mutations in histologically normal endometrium: the new normal? J. Pathol. 249, 173–181 (2019)."), (2) endometriosis[63](https://www.nature.com/articles/s41588-025-02308-w#ref-CR63 "Anglesio, M. S. et al. Cancer-associated mutations in endometriosis without cancer. N. Engl. J. Med. 376, 1835–1848 (2017)."),[64](https://www.nature.com/articles/s41588-025-02308-w#ref-CR64 "Praetorius, T. H. et al. Molecular analysis suggests oligoclonality and metastasis of endometriosis lesions across anatomically defined subtypes. Fertil. Steril. 118, 524–534 (2022).") and (3) atypical hyperplasia[65](https://www.nature.com/articles/s41588-025-02308-w#ref-CR65 "Li, L. et al. Genome-wide mutation analysis in precancerous lesions of endometrial carcinoma. J. Pathol. 253, 119–128 (2021)."),[66](https://www.nature.com/articles/s41588-025-02308-w#ref-CR66 "Hu, Z. et al. Proteogenomic insights into early-onset endometrioid endometrial carcinoma: predictors for fertility-sparing therapy response. Nat. Genet. 56, 637–651 (2024).").

Finally, we also included histological subtype data from a set of 161 TAMARISK patients with de novo UC[28](https://www.nature.com/articles/s41588-025-02308-w#ref-CR28 "Hoogendoorn, W. E. et al. Prognosis of uterine corpus cancer after tamoxifen treatment for breast cancer. Breast Cancer Res. Treat. 112, 99–108 (2008).") diagnosed after breast cancer but without prior use of tamoxifen.

### Statistics and reproducibility

Statistical analysis and visualization were performed using R (version 4.1.1) in an RStudio environment and Julia (version 1.7.3) in a Jupyter environment. To determine significance, we used Fisher’s exact test (with Monte Carlo simulation for tables larger than 2 × 2, using 106 iterations), the _t_\-test and the Wilcoxon rank-sum test, all two sided unless otherwise indicated. Multiple-hypothesis testing was performed using the method of Benjamini and Hochberg[86](https://www.nature.com/articles/s41588-025-02308-w#ref-CR86 "Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B. 57, 289–300 (1995)."), which converted the final _P_ values to false discovery rate _Q_ values; _Q_ < 0.1 was considered significant. The strength of associations between variables was analyzed using Pearson’s correlation. Two-sided stratified Fisher’s exact test was used to control for potential confounding variables when analyzing mutation frequency data across multiple subgroups (or strata), providing a combined _P_ value calculated across the strata, with zero-marginal tables excluded from the calculation[98](https://www.nature.com/articles/s41588-025-02308-w#ref-CR98 "Jung, S. H. Stratified Fisher’s exact test and its sample size calculation. Biom. J. 56, 129–140 (2014)."),[99](https://www.nature.com/articles/s41588-025-02308-w#ref-CR99 "Martín-Andrés, A. & Herranz-Tejedor, I. Regarding Paper ‘Stratified Fisher’s exact test and its sample size calculation’. Biom. J. 57, 930 (2015)."). No statistical method was used to predetermine sample size. No data were excluded from the analyses. Randomization and blinding were not applicable, as this study involved retrospective analysis of genomic and clinical data.

### Power calculations

We assessed the statistical power to detect differences in driver gene mutation frequencies (either higher or lower) between the TA-UC and de novo UC sample sets given the observed sample sizes in both the WES discovery cohort and the WES validation subtypes. We identified powered genes by computing Bonferroni-corrected two-sided optimal Fisher’s exact test _P_ values across all possible 2 × 2 contingency tables, maintaining the same marginal totals but allowing zero counts. For each configuration, we calculated _P_ values, focusing on the smallest _P_ value as an indication of the extreme case in which the effect size is close to or equal to zero. A Bonferroni-corrected optimal _P_ value of <0.05 was considered a powered test. We also calculated the power to identify driver genes that are significantly less mutated in the TA-UC discovery cohort by computing _P_ values from one-sided Fisher’s exact tests for the different frequencies. Genes at a threshold of _P_ < 0.05 can potentially be considered significantly less mutated in the TA-UC discovery cohort, as they are mutated in at least 76 de novo TCGA UC samples.

### Analysis of human expression data

We used previously published[100](https://www.nature.com/articles/s41588-025-02308-w#ref-CR100 "Wu, H. et al. Hypomethylation-linked activation of PAX2 mediates tamoxifen-stimulated endometrial carcinogenesis. Nature 438, 981–987 (2005).") gene expression levels from Affymetrix U95A Human Genome arrays of enriched human-derived endometrial cells that were short-term cultured with either E2 (100 nM) or tamoxifen (5 µM) for 3 h. After removal of one outlier sample ([GSM65291](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM65291)), we performed quantile normalization followed by differential gene expression using limmaVoom[101](https://www.nature.com/articles/s41588-025-02308-w#ref-CR101 "Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47 (2015).") (version 3.50.0), focusing on genes in the KEGG PATHWAY Database, estrogen response genes from the hallmark gene sets and genes in the AKT–mTOR oncogenic signature gene sets (all from GSEA). Pathway analysis was carried out using Enrichr ([https://maayanlab.cloud/Enrichr/](https://maayanlab.cloud/Enrichr/))[102](https://www.nature.com/articles/s41588-025-02308-w#ref-CR102 "Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res. 44, W90–W97 (2016)."), the NCI–Nature Pathway Interaction Database[103](https://www.nature.com/articles/s41588-025-02308-w#ref-CR103 "Schaefer, C. F. et al. PID: the Pathway Interaction Database. Nucleic Acids Res. 37, D674–D679 (2009).") and differentially expressed genes with a cutoff of |log2 (FC)| > log2 (1.5) and _Q_ value < 0.01.

### In vivo mouse study

All mice were maintained in accordance with local guidelines, and therapeutic interventions were approved by the Animal Care and Use Committee of the DFCI (protocol 08-023). To mimic the postmenopausal condition that is typically observed in patients with TA-UC, 20 C57BL/6 female mice (Jackson Laboratory) were oophorectomized after sexual maturity (6–7 weeks) to allow for proper uterine development. Oophorectomy also circumvented the ER-dependent endometrial changes that occur during the estrous cycle, which could confound the interpretation of results. As the hormone E2, a major female sex hormone produced during the estrous cycle, binds to ER and increases cell proliferation, we used exogenous E2 as a positive control. Mice were randomized (_n_ = 5 per arm) to E2 (0.01 mg per pellet, 60-d release), vehicle control (E2 deprived), tamoxifen (Sigma, in 20% ethanol in corn oil, 0.5 mg per mouse per day, subcutaneous injection, comparable to the concentration seen in humans[104](https://www.nature.com/articles/s41588-025-02308-w#ref-CR104 "Reid, J. M. et al. Pharmacokinetics of endoxifen and tamoxifen in female mice: implications for comparative in vivo activity studies. Cancer Chemother. Pharmacol. 74, 1271–1278 (2014).")) or tamoxifen plus alpelisib (Selleckchem, in 30% PEG 400 + 0.5% Tween-80 + 5% propylene glycol, 30 mg per kg per day, oral gavage) for 30 d. At the end of the study, mice were euthanized, and uterine horns were collected.

### Mouse tissue collection and processing

Mouse uterine horns were collected from five mice per cohort, as reported by De Clercq et al.[105](https://www.nature.com/articles/s41588-025-02308-w#ref-CR105 "De Clercq, K., Hennes, A. & Vriens, J. Isolation of mouse endometrial epithelial and stromal cells for in vitro decidualization. J. Vis. Exp. 
https://doi.org/10.3791/55168
(2017)."). Samples were allocated for downstream applications as follows: (1) single-cell suspensions were prepared and used to isolate epithelial and stromal cell populations. For the E2, tamoxifen and tamoxifen-plus-alpelisib groups, three mice per condition were used; in the vehicle control group, five mice were processed to obtain sufficient material despite the minuscule size of the uteri in this condition. (2) FFPE samples for IHC were prepared from three mice (E2), five mice (tamoxifen, tamoxifen plus alpelisib) and two mice (vehicle control, in which sample collection was limited by the miniscule size of the uterine horns, a consequence of oophorectomy without hormonal supplementation, and by fibrosis secondary to the surgical procedure).

### Immunohistochemistry

For immunohistochemical detection, samples were stained with primary antibodies and incubated with anti-mouse (G21040, Invitrogen) or anti-rabbit (G21234, Invitrogen) antibodies (both at a 1:2,000 dilution) for 50 min at room temperature. Samples were stained with the DAB (3,3′-diaminobenzidine) colorimetric substrate and counterstained with hematoxylin. The following primary antibodies were used: anti-ER-α (06-938, 1:1,000, Millipore), anti-phospho-IR/IGF1R Tyr1162/Tyr1163 (44-804, 1:500, Invitrogen), anti-Ki-67 (ab15580, 1:1,000, Abcam), anti-phospho-AKT Thr308 (ab81283, 1:50, Abcam) and anti-phospho-S6 Ser240/Ser244 (2215, 1:500, Cell Signaling).

Numbers of ducts per mouse were counted in six distinct sections using a 20× high-power field. The length (in µm) of endometrial epithelial cells per mouse was measured in six sections using five distinct regions of the internal lumen. IHC images were analyzed with QuPath version 0.2.0 software ([https://qupath.github.io/](https://qupath.github.io/)). IHC staining was quantified as the product of percent positive cells per section × staining intensity in optical density (_H_ score). Statistical analyses for immunohistochemical studies were performed in GraphPad Prism version 9.0 (GraphPad Software) using one-way ANOVA.

### Messenger RNA in situ hybridization

In situ hybridization was performed with the RNAscope Intro Pack for Multiplex Fluorescent Reagent Kit v2-Mm from Advanced Cell Diagnostics according to the manufacturer’s protocol. Briefly, FFPE sections were deparaffinized with xylene and rehydrated with alcohol. The sections were hybridized at 40 °C for 2 h with the RNAscope Probe-Mm-Igf1 that is specific for mouse _Igf1_ mRNA (Advanced Cell Diagnostics), and the signal was visualized with RNAscope fluorescent reagents. Sections were counterstained with ProLong Gold Antifade Reagent (Life Technologies) before dehydrating, and coverslips were affixed with Permount (Thermo Fisher Scientific). Images were acquired with a Leica SP8X STED/confocal microscope using Leica Application Suite X (version 3.7) acquisition software. Images were acquired as _Z_ stacks (1 µm) using the Piezo Z stage.

### RNA extraction and quantitative PCR with reverse transcription

Total RNA was isolated using TRIzol (Life Technologies) and the RNeasy Mini Kit (Qiagen) according to the manufacturer’s instructions. To test the purity of epithelial cells, we used quantitative PCR with reverse transcription and primers summarized in Supplementary Table [15](https://www.nature.com/articles/s41588-025-02308-w#MOESM3). mRNA was retrotranscribed using the High-Capacity cDNA Reverse Transcription Kit (Applied Biosystem), and detection was accomplished using the Roche LightCycler 480 Real-time PCR system in combination with the Power SYBR Green PCR Master Mix (Life Technologies).

### RNA sequencing

RNA-seq libraries were made after enrichment with oligo(dT) beads. First, mRNA was randomly fragmented by adding fragmentation buffer. Next, cDNA was synthesized using mRNA template and random hexamer primers, after which a custom second-strand synthesis buffer (Illumina), dNTPs, RNase H and DNA polymerase I were added to initiate second-strand synthesis. After a series of terminal repair, A ligation and sequencing adaptor ligation, the double-stranded cDNA library was completed through size selection and PCR enrichment. Samples were sequenced on an Illumina NextSeq 500 instrument (libraries generated and sequencing performed at Novogene).

### RNA sequencing analysis

RNA-seq analysis was performed using the VIPER analysis pipeline (version 1.41.0)[106](https://www.nature.com/articles/s41588-025-02308-w#ref-CR106 "Cornwell, M. et al. VIPER: Visualization Pipeline for RNA-seq, a Snakemake workflow for efficient and complete RNA-seq analysis. BMC Bioinformatics 19, 135 (2018)."). Alignment to the hg19 human genome was accomplished using STAR version 2.7.0f followed by transcript assembly using cufflinks version 2.2.1 (ref. [107](https://www.nature.com/articles/s41588-025-02308-w#ref-CR107 "Trapnell, C. et al. Transcript assembly and quantification by RNA-seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat. Biotechnol. 28, 511–515 (2010).")) and RSeQC version 2.6.2 (ref. [108](https://www.nature.com/articles/s41588-025-02308-w#ref-CR108 "Wang, L., Wang, S. & Li, W. RSeQC: quality control of RNA-seq experiments. Bioinformatics 28, 2184–2185 (2012).")). Differential expression analysis was carried out using DESeq2 version 1.18.1 (ref. [109](https://www.nature.com/articles/s41588-025-02308-w#ref-CR109 "Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014).")). Pathway analysis was carried out using Enrichr ([https://maayanlab.cloud/Enrichr/](https://maayanlab.cloud/Enrichr/)) and applying MsigDB oncogenic signatures[102](https://www.nature.com/articles/s41588-025-02308-w#ref-CR102 "Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res. 44, W90–W97 (2016).").

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02308-w#MOESM2) linked to this article.

Data availability
-----------------

TCGA pan-cancer data are available through a data portal: [https://gdc.cancer.gov/node/905/](https://gdc.cancer.gov/node/905/); [https://gdc.cancer.gov/about-data/publications/pancanatlas](https://gdc.cancer.gov/about-data/publications/pancanatlas). In compliance with the data access policy, most data are in an open tier that does not require access approval. Some data files with potentially identifying information and underlying sequencing data are controlled-access data and may be hosted at dbGaP. Researchers will need to apply to the TCGA Data Access Committee via dbGaP ([https://dbgap.ncbi.nlm.nih.gov/aa/wga.cgi?page=login](https://dbgap.ncbi.nlm.nih.gov/aa/wga.cgi?page=login)) to request access. Clinical Proteomic Tumor Analysis Consortium endometrial cancer mutation data are available from the Genomic Data Commons ([https://gdc.cancer.gov/](https://gdc.cancer.gov/)) or upon request from dbGaP ([https://www.ncbi.nlm.nih.gov/gap/](https://www.ncbi.nlm.nih.gov/gap/), [phs001287](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001287)). SEER data are available through a data portal ([https://seer.cancer.gov/data/](https://seer.cancer.gov/data/)) after data use agreement forms have been signed. The Affymetrix U95A Human Genome arrays of enriched human-derived endometrial cells can be accessed at the Gene Expression Omnibus via [GSE3013](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE3013). Data from the GENIE database can be found on the Sage Bionetworks portal ([https://www.synapse.org/#!Synapse:syn7222066/wiki/405659](https://www.synapse.org/#!Synapse:syn7222066/wiki/405659)). To request access to protected GENIE data, researchers need to apply to dbGaP for access (study accession [phs001337](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001337)). Analyses in this paper also used published datasets that are available from the corresponding studies, which are referenced where relevant. WES data of TA-UCs are available through the EGA; the accession number is [EGAS00001006453](https://ega-archive.org/studies/EGAS00001006453) ([https://www.ega-archive.org/studies/EGAS00001006453](https://www.ega-archive.org/studies/EGAS00001006453)). Mouse endometrial epithelial RNA-seq data are available at the Gene Expression Omnibus through [GSE179647](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179647) ([https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179647](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179647)). The Caris datasets generated and/or analyzed during the current study are available upon reasonable request. De-identified sequencing data are owned by Caris Life Sciences and cannot be publicly shared without a data usage agreement. Qualified researchers can apply for access to these summarized data by contacting J. Xiu (jxiu@carisls.com) and signing a data usage agreement. MsigDB oncogenic signatures, KEGG PATHWAY database, estrogen response genes from the hallmark gene sets and genes in the AKT–mTOR oncogenic signature gene sets are from [https://www.gsea-msigdb.org/gsea](https://www.gsea-msigdb.org/gsea); the NCI–Nature Pathway Interaction Database can be found at [https://www.ndexbio.org/](https://www.ndexbio.org/).

Code availability
-----------------

No custom code was used. All software packages and analysis code are publicly available, and the relevant sources are cited in [Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9).

References
----------

1.  Kuijk, E., Kranenburg, O., Cuppen, E. & Van Hoeck, A. Common anti-cancer therapies induce somatic mutations in stem cells of healthy tissue. _Nat. Commun._ **13**, 5915 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36207433)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9546852)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Common%20anti-cancer%20therapies%20induce%20somatic%20mutations%20in%20stem%20cells%20of%20healthy%20tissue&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Kuijk%2CE&author=Kranenburg%2CO&author=Cuppen%2CE&author=Hoeck%2CA) 
    
2.  Carthew, P. et al. DNA damage as assessed by 32P-postlabelling in three rat strains exposed to dietary tamoxifen: the relationship between cell proliferation and liver tumour formation. _Carcinogenesis_ **16**, 1299–1304 (1995).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7788846)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20damage%20as%20assessed%20by%2032P-postlabelling%20in%20three%20rat%20strains%20exposed%20to%20dietary%20tamoxifen%3A%20the%20relationship%20between%20cell%20proliferation%20and%20liver%20tumour%20formation&journal=Carcinogenesis&volume=16&pages=1299-1304&publication_year=1995&author=Carthew%2CP) 
    
3.  Carthew, P. et al. Cumulative exposure to tamoxifen: DNA adducts and liver cancer in the rat. _Arch. Toxicol._ **75**, 375–380 (2001).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11570696)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cumulative%20exposure%20to%20tamoxifen%3A%20DNA%20adducts%20and%20liver%20cancer%20in%20the%20rat&journal=Arch.%20Toxicol.&volume=75&pages=375-380&publication_year=2001&author=Carthew%2CP) 
    
4.  Busch, H. Adducts and tamoxifen. _Semin. Oncol._ **24**, S1-98–S1-104 (1997).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adducts%20and%20tamoxifen&journal=Semin.%20Oncol.&volume=24&pages=S1-98-S1-104&publication_year=1997&author=Busch%2CH) 
    
5.  Hernandez-Ramon, E. E. et al. Tamoxifen–DNA adduct formation in monkey and human reproductive organs. _Carcinogenesis_ **35**, 1172–1176 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24501327)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4004208)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen%E2%80%93DNA%20adduct%20formation%20in%20monkey%20and%20human%20reproductive%20organs&journal=Carcinogenesis&volume=35&pages=1172-1176&publication_year=2014&author=Hernandez-Ramon%2CEE) 
    
6.  Andersson, H., Helmestam, M., Zebrowska, A., Olovsson, M. & Brittebo, E. Tamoxifen-induced adduct formation and cell stress in human endometrial glands. _Drug Metab. Dispos._ **38**, 200–207 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19812351)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen-induced%20adduct%20formation%20and%20cell%20stress%20in%20human%20endometrial%20glands&journal=Drug%20Metab.%20Dispos.&volume=38&pages=200-207&publication_year=2010&author=Andersson%2CH&author=Helmestam%2CM&author=Zebrowska%2CA&author=Olovsson%2CM&author=Brittebo%2CE) 
    
7.  Kim, S. Y. et al. Formation of tamoxifen–DNA adducts in human endometrial explants exposed to α-hydroxytamoxifen. _Chem. Res. Toxicol._ **18**, 889–895 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15892583)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Formation%20of%20tamoxifen%E2%80%93DNA%20adducts%20in%20human%20endometrial%20explants%20exposed%20to%20%CE%B1-hydroxytamoxifen&journal=Chem.%20Res.%20Toxicol.&volume=18&pages=889-895&publication_year=2005&author=Kim%2CSY) 
    
8.  Cole, M. P., Jones, C. T. & Todd, I. D. A new anti-oestrogenic agent in late breast cancer. An early clinical appraisal of ICI46474. _Br. J. Cancer_ **25**, 270–275 (1971).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=5115829)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2008453)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20new%20anti-oestrogenic%20agent%20in%20late%20breast%20cancer.%20An%20early%20clinical%20appraisal%20of%20ICI46474&journal=Br.%20J.%20Cancer&volume=25&pages=270-275&publication_year=1971&author=Cole%2CMP&author=Jones%2CCT&author=Todd%2CID) 
    
9.  Fisher, B. et al. Adjuvant chemotherapy with and without tamoxifen in the treatment of primary breast cancer: 5-year results from the National Surgical Adjuvant Breast and Bowel Project Trial. _J. Clin. Oncol._ **4**, 459–471 (1986).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2856857)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adjuvant%20chemotherapy%20with%20and%20without%20tamoxifen%20in%20the%20treatment%20of%20primary%20breast%20cancer%3A%205-year%20results%20from%20the%20National%20Surgical%20Adjuvant%20Breast%20and%20Bowel%20Project%20Trial&journal=J.%20Clin.%20Oncol.&volume=4&pages=459-471&publication_year=1986&author=Fisher%2CB) 
    
10.  Fisher, B. et al. Tamoxifen for prevention of breast cancer: report of the National Surgical Adjuvant Breast and Bowel Project P-1 Study. _J. Natl Cancer Inst._ **90**, 1371–1388 (1998).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9747868)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen%20for%20prevention%20of%20breast%20cancer%3A%20report%20of%20the%20National%20Surgical%20Adjuvant%20Breast%20and%20Bowel%20Project%20P-1%20Study&journal=J.%20Natl%20Cancer%20Inst.&volume=90&pages=1371-1388&publication_year=1998&author=Fisher%2CB) 
    
11.  Early Breast Cancer Trialists’ Collaborative Group. Aromatase inhibitors versus tamoxifen in early breast cancer: patient-level meta-analysis of the randomised trials. _Lancet_ **386**, 1341–1352 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aromatase%20inhibitors%20versus%20tamoxifen%20in%20early%20breast%20cancer%3A%20patient-level%20meta-analysis%20of%20the%20randomised%20trials&journal=Lancet&volume=386&pages=1341-1352&publication_year=2015) 
    
12.  Sparano, J. A. et al. Adjuvant chemotherapy guided by a 21-gene expression assay in breast cancer. _N. Engl. J. Med._ **379**, 111–121 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860917)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6172658)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adjuvant%20chemotherapy%20guided%20by%20a%2021-gene%20expression%20assay%20in%20breast%20cancer&journal=N.%20Engl.%20J.%20Med.&volume=379&pages=111-121&publication_year=2018&author=Sparano%2CJA) 
    
13.  Johnston, S. R. D. et al. Abemaciclib combined with endocrine therapy for the adjuvant treatment of HR+, HER2−, node-positive, high-risk, early breast cancer (monarchE). _J. Clin. Oncol._ **38**, 3987–3998 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32954927)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7768339)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Abemaciclib%20combined%20with%20endocrine%20therapy%20for%20the%20adjuvant%20treatment%20of%20HR%2B%2C%20HER2%E2%88%92%2C%20node-positive%2C%20high-risk%2C%20early%20breast%20cancer%20%28monarchE%29&journal=J.%20Clin.%20Oncol.&volume=38&pages=3987-3998&publication_year=2020&author=Johnston%2CSRD) 
    
14.  Fornander, T. et al. Adjuvant tamoxifen in early breast cancer: occurrence of new primary cancers. _Lancet_ **1**, 117–120 (1989).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2563046)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adjuvant%20tamoxifen%20in%20early%20breast%20cancer%3A%20occurrence%20of%20new%20primary%20cancers&journal=Lancet&volume=1&pages=117-120&publication_year=1989&author=Fornander%2CT) 
    
15.  Bernstein, L. et al. Tamoxifen therapy for breast cancer and endometrial cancer risk. _J. Natl Cancer Inst._ **91**, 1654–1662 (1999).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10511593)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen%20therapy%20for%20breast%20cancer%20and%20endometrial%20cancer%20risk&journal=J.%20Natl%20Cancer%20Inst.&volume=91&pages=1654-1662&publication_year=1999&author=Bernstein%2CL) 
    
16.  Bergman, L. et al. Risk and prognosis of endometrial cancer after tamoxifen for breast cancer. Comprehensive Cancer Centres’ ALERT Group. Assessment of liver and endometrial cancer risk following tamoxifen. _Lancet_ **356**, 881–887 (2000).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11036892)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Risk%20and%20prognosis%20of%20endometrial%20cancer%20after%20tamoxifen%20for%20breast%20cancer.%20Comprehensive%20Cancer%20Centres%E2%80%99%20ALERT%20Group.%20Assessment%20of%20liver%20and%20endometrial%20cancer%20risk%20following%20tamoxifen&journal=Lancet&volume=356&pages=881-887&publication_year=2000&author=Bergman%2CL) 
    
17.  Fisher, B. et al. Endometrial cancer in tamoxifen-treated breast cancer patients: findings from the National Surgical Adjuvant Breast and Bowel Project (NSABP) B-14. _J. Natl Cancer Inst._ **86**, 527–537 (1994).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8133536)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Endometrial%20cancer%20in%20tamoxifen-treated%20breast%20cancer%20patients%3A%20findings%20from%20the%20National%20Surgical%20Adjuvant%20Breast%20and%20Bowel%20Project%20%28NSABP%29%20B-14&journal=J.%20Natl%20Cancer%20Inst.&volume=86&pages=527-537&publication_year=1994&author=Fisher%2CB) 
    
18.  Swerdlow, A. J., Jones, M. E. & British Tamoxifen Second Cancer Study Group. Tamoxifen treatment for breast cancer and risk of endometrial cancer: a case–control study. _J. Natl Cancer Inst._ **97**, 375–384 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15741574)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen%20treatment%20for%20breast%20cancer%20and%20risk%20of%20endometrial%20cancer%3A%20a%20case%E2%80%93control%20study&journal=J.%20Natl%20Cancer%20Inst.&volume=97&pages=375-384&publication_year=2005&author=Swerdlow%2CAJ&author=Jones%2CME) 
    
19.  Cuzick, J. et al. Tamoxifen for prevention of breast cancer: extended long-term follow-up of the IBIS-I breast cancer prevention trial. _Lancet Oncol._ **16**, 67–75 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25497694)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772450)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen%20for%20prevention%20of%20breast%20cancer%3A%20extended%20long-term%20follow-up%20of%20the%20IBIS-I%20breast%20cancer%20prevention%20trial&journal=Lancet%20Oncol.&volume=16&pages=67-75&publication_year=2015&author=Cuzick%2CJ) 
    
20.  Fisher, B. et al. Tamoxifen for the prevention of breast cancer: current status of the National Surgical Adjuvant Breast and Bowel Project P-1 study. _J. Natl Cancer Inst._ **97**, 1652–1662 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16288118)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen%20for%20the%20prevention%20of%20breast%20cancer%3A%20current%20status%20of%20the%20National%20Surgical%20Adjuvant%20Breast%20and%20Bowel%20Project%20P-1%20study&journal=J.%20Natl%20Cancer%20Inst.&volume=97&pages=1652-1662&publication_year=2005&author=Fisher%2CB) 
    
21.  Davies, C. et al. Long-term effects of continuing adjuvant tamoxifen to 10 years versus stopping at 5 years after diagnosis of oestrogen receptor-positive breast cancer: ATLAS, a randomised trial. _Lancet_ **381**, 805–816 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23219286)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596060)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Long-term%20effects%20of%20continuing%20adjuvant%20tamoxifen%20to%2010%20years%20versus%20stopping%20at%205%20years%20after%20diagnosis%20of%20oestrogen%20receptor-positive%20breast%20cancer%3A%20ATLAS%2C%20a%20randomised%20trial&journal=Lancet&volume=381&pages=805-816&publication_year=2013&author=Davies%2CC) 
    
22.  Shang, Y. & Brown, M. Molecular determinants for the tissue specificity of SERMs. _Science_ **295**, 2465–2468 (2002).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11923541)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20determinants%20for%20the%20tissue%20specificity%20of%20SERMs&journal=Science&volume=295&pages=2465-2468&publication_year=2002&author=Shang%2CY&author=Brown%2CM) 
    
23.  Korach, K. S. Insights from the study of animals lacking functional estrogen receptor. _Science_ **266**, 1524–1527 (1994).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7985022)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20from%20the%20study%20of%20animals%20lacking%20functional%20estrogen%20receptor&journal=Science&volume=266&pages=1524-1527&publication_year=1994&author=Korach%2CKS) 
    
24.  Couse, J. F. & Korach, K. S. Estrogen receptor null mice: what have we learned and where will they lead us? _Endocr. Rev._ **20**, 358–417 (1999).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10368776)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estrogen%20receptor%20null%20mice%3A%20what%20have%20we%20learned%20and%20where%20will%20they%20lead%20us%3F&journal=Endocr.%20Rev.&volume=20&pages=358-417&publication_year=1999&author=Couse%2CJF&author=Korach%2CKS) 
    
25.  Davies, R. et al. Tamoxifen causes gene mutations in the livers of lambda/_lacI_ transgenic rats. _Cancer Res._ **57**, 1288–1293 (1997).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9102215)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tamoxifen%20causes%20gene%20mutations%20in%20the%20livers%20of%20lambda%2FlacI%20transgenic%20rats&journal=Cancer%20Res.&volume=57&pages=1288-1293&publication_year=1997&author=Davies%2CR) 
    
26.  Brown, K. Is tamoxifen a genotoxic carcinogen in women? _Mutagenesis_ **24**, 391–404 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19505894)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Is%20tamoxifen%20a%20genotoxic%20carcinogen%20in%20women%3F&journal=Mutagenesis&volume=24&pages=391-404&publication_year=2009&author=Brown%2CK) 
    
27.  Fles, R. et al. Genomic profile of endometrial tumors depends on morphological subtype, not on tamoxifen exposure. _Genes Chromosomes Cancer_ **49**, 699–710 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20544844)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20profile%20of%20endometrial%20tumors%20depends%20on%20morphological%20subtype%2C%20not%20on%20tamoxifen%20exposure&journal=Genes%20Chromosomes%20Cancer&volume=49&pages=699-710&publication_year=2010&author=Fles%2CR) 
    
28.  Hoogendoorn, W. E. et al. Prognosis of uterine corpus cancer after tamoxifen treatment for breast cancer. _Breast Cancer Res. Treat._ **112**, 99–108 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18064567)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prognosis%20of%20uterine%20corpus%20cancer%20after%20tamoxifen%20treatment%20for%20breast%20cancer&journal=Breast%20Cancer%20Res.%20Treat.&volume=112&pages=99-108&publication_year=2008&author=Hoogendoorn%2CWE) 
    
29.  Levine, D. A. et al. Integrated genomic characterization of endometrial carcinoma. _Nature_ **497**, 67–73 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23636398)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3704730)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20genomic%20characterization%20of%20endometrial%20carcinoma&journal=Nature&volume=497&pages=67-73&publication_year=2013&author=Levine%2CDA) 
    
30.  Cherniack, A. D. et al. Integrated molecular characterization of uterine carcinosarcoma. _Cancer Cell_ **31**, 411–423 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28292439)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5599133)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20molecular%20characterization%20of%20uterine%20carcinosarcoma&journal=Cancer%20Cell&volume=31&pages=411-423&publication_year=2017&author=Cherniack%2CAD) 
    
31.  Cancer Genome Atlas Research Network. Comprehensive and integrated genomic characterization of adult soft tissue sarcomas. _Cell_ **171**, 950–965 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20and%20integrated%20genomic%20characterization%20of%20adult%20soft%20tissue%20sarcomas&journal=Cell&volume=171&pages=950-965&publication_year=2017) 
    
32.  AACR Project GENIE Consortium. AACR Project GENIE: powering precision medicine through an international consortium. _Cancer Discov._ **7**, 818–831 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=AACR%20Project%20GENIE%3A%20powering%20precision%20medicine%20through%20an%20international%20consortium&journal=Cancer%20Discov.&volume=7&pages=818-831&publication_year=2017) 
    
33.  Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. _Nature_ **578**, 94–101 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32025018)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054213)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&journal=Nature&volume=578&pages=94-101&publication_year=2020&author=Alexandrov%2CLB) 
    
34.  Gibson, W. J. et al. The genomic landscape and evolution of endometrial carcinoma progression and abdominopelvic metastasis. _Nat. Genet._ **48**, 848–855 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27348297)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4963271)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genomic%20landscape%20and%20evolution%20of%20endometrial%20carcinoma%20progression%20and%20abdominopelvic%20metastasis&journal=Nat.%20Genet.&volume=48&pages=848-855&publication_year=2016&author=Gibson%2CWJ) 
    
35.  Chang, M. T. et al. Identifying recurrent mutations in cancer reveals widespread lineage diversity and mutational specificity. _Nat. Biotechnol._ **34**, 155–163 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26619011)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identifying%20recurrent%20mutations%20in%20cancer%20reveals%20widespread%20lineage%20diversity%20and%20mutational%20specificity&journal=Nat.%20Biotechnol.&volume=34&pages=155-163&publication_year=2016&author=Chang%2CMT) 
    
36.  Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. _Genome Biol._ **12**, R41 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21527027)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218867)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GISTIC2.0%20facilitates%20sensitive%20and%20confident%20localization%20of%20the%20targets%20of%20focal%20somatic%20copy-number%20alteration%20in%20human%20cancers&journal=Genome%20Biol.&volume=12&publication_year=2011&author=Mermel%2CCH) 
    
37.  Zhang, Y. et al. A pan-cancer proteogenomic atlas of PI3K/AKT/mTOR pathway alterations. _Cancer Cell_ **31**, 820–832 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28528867)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5502825)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20pan-cancer%20proteogenomic%20atlas%20of%20PI3K%2FAKT%2FmTOR%20pathway%20alterations&journal=Cancer%20Cell&volume=31&pages=820-832&publication_year=2017&author=Zhang%2CY) 
    
38.  Dashti, S. G. et al. Adiposity and breast, endometrial, and colorectal cancer risk in postmenopausal women: quantification of the mediating effects of leptin, C-reactive protein, fasting insulin, and estradiol. _Cancer Med._ **11**, 1145–1159 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35048536)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8855919)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adiposity%20and%20breast%2C%20endometrial%2C%20and%20colorectal%20cancer%20risk%20in%20postmenopausal%20women%3A%20quantification%20of%20the%20mediating%20effects%20of%20leptin%2C%20C-reactive%20protein%2C%20fasting%20insulin%2C%20and%20estradiol&journal=Cancer%20Med.&volume=11&pages=1145-1159&publication_year=2022&author=Dashti%2CSG) 
    
39.  Kaaks, R., Lukanova, A. & Kurzer, M. S. Obesity, endogenous hormones, and endometrial cancer risk: a synthetic review. _Cancer Epidemiol. Biomarkers Prev._ **11**, 1531–1543 (2002).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12496040)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity%2C%20endogenous%20hormones%2C%20and%20endometrial%20cancer%20risk%3A%20a%20synthetic%20review&journal=Cancer%20Epidemiol.%20Biomarkers%20Prev.&volume=11&pages=1531-1543&publication_year=2002&author=Kaaks%2CR&author=Lukanova%2CA&author=Kurzer%2CMS) 
    
40.  Schmandt, R. E., Iglesias, D. A., Co, N. N. & Lu, K. H. Understanding obesity and endometrial cancer risk: opportunities for prevention. _Am. J. Obstet. Gynecol._ **205**, 518–525 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21802066)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%20obesity%20and%20endometrial%20cancer%20risk%3A%20opportunities%20for%20prevention&journal=Am.%20J.%20Obstet.%20Gynecol.&volume=205&pages=518-525&publication_year=2011&author=Schmandt%2CRE&author=Iglesias%2CDA&author=Co%2CNN&author=Lu%2CKH) 
    
41.  Freeman, E. W., Sammel, M. D., Lin, H. & Gracia, C. R. Obesity and reproductive hormone levels in the transition to menopause. _Menopause_ **17**, 718–726 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20216473)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2888623)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity%20and%20reproductive%20hormone%20levels%20in%20the%20transition%20to%20menopause&journal=Menopause&volume=17&pages=718-726&publication_year=2010&author=Freeman%2CEW&author=Sammel%2CMD&author=Lin%2CH&author=Gracia%2CCR) 
    
42.  Onstad, M. A., Schmandt, R. E. & Lu, K. H. Addressing the role of obesity in endometrial cancer risk, prevention, and treatment. _J. Clin. Oncol._ **34**, 4225–4230 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27903150)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5455320)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Addressing%20the%20role%20of%20obesity%20in%20endometrial%20cancer%20risk%2C%20prevention%2C%20and%20treatment&journal=J.%20Clin.%20Oncol.&volume=34&pages=4225-4230&publication_year=2016&author=Onstad%2CMA&author=Schmandt%2CRE&author=Lu%2CKH) 
    
43.  Smith, D. C., Prentice, R., Thompson, D. J. & Herrmann, W. L. Association of exogenous estrogen and endometrial carcinoma. _N. Engl. J. Med._ **293**, 1164–1167 (1975).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1186789)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20of%20exogenous%20estrogen%20and%20endometrial%20carcinoma&journal=N.%20Engl.%20J.%20Med.&volume=293&pages=1164-1167&publication_year=1975&author=Smith%2CDC&author=Prentice%2CR&author=Thompson%2CDJ&author=Herrmann%2CWL) 
    
44.  Ziel, H. K. & Finkle, W. D. Increased risk of endometrial carcinoma among users of conjugated estrogens. _N. Engl. J. Med._ **293**, 1167–1170 (1975).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=171569)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Increased%20risk%20of%20endometrial%20carcinoma%20among%20users%20of%20conjugated%20estrogens&journal=N.%20Engl.%20J.%20Med.&volume=293&pages=1167-1170&publication_year=1975&author=Ziel%2CHK&author=Finkle%2CWD) 
    
45.  Yuan, T. L. & Cantley, L. C. PI3K pathway alterations in cancer: variations on a theme. _Oncogene_ **27**, 5497–5510 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18794884)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3398461)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PI3K%20pathway%20alterations%20in%20cancer%3A%20variations%20on%20a%20theme&journal=Oncogene&volume=27&pages=5497-5510&publication_year=2008&author=Yuan%2CTL&author=Cantley%2CLC) 
    
46.  Cheung, L. W. et al. High frequency of _PIK3R1_ and _PIK3R2_ mutations in endometrial cancer elucidates a novel mechanism for regulation of PTEN protein stability. _Cancer Discov._ **1**, 170–185 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21984976)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3187555)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High%20frequency%20of%20PIK3R1%20and%20PIK3R2%20mutations%20in%20endometrial%20cancer%20elucidates%20a%20novel%20mechanism%20for%20regulation%20of%20PTEN%20protein%20stability&journal=Cancer%20Discov.&volume=1&pages=170-185&publication_year=2011&author=Cheung%2CLW) 
    
47.  Fleming, C. A. et al. Meta-analysis of the cumulative risk of endometrial malignancy and systematic review of endometrial surveillance in extended tamoxifen therapy. _Br. J. Surg._ **105**, 1098–1106 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29974455)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Meta-analysis%20of%20the%20cumulative%20risk%20of%20endometrial%20malignancy%20and%20systematic%20review%20of%20endometrial%20surveillance%20in%20extended%20tamoxifen%20therapy&journal=Br.%20J.%20Surg.&volume=105&pages=1098-1106&publication_year=2018&author=Fleming%2CCA) 
    
48.  André, F. et al. Alpelisib for _PIK3CA_\-mutated, hormone receptor-positive advanced breast cancer. _N. Engl. J. Med._ **380**, 1929–1940 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31091374)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Alpelisib%20for%20PIK3CA-mutated%2C%20hormone%20receptor-positive%20advanced%20breast%20cancer&journal=N.%20Engl.%20J.%20Med.&volume=380&pages=1929-1940&publication_year=2019&author=Andr%C3%A9%2CF) 
    
49.  Rodriguez, A. C., Blanchard, Z., Maurer, K. A. & Gertz, J. Estrogen signaling in endometrial cancer: a key oncogenic pathway with several open questions. _Horm. Cancer_ **10**, 51–63 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30712080)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6542701)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estrogen%20signaling%20in%20endometrial%20cancer%3A%20a%20key%20oncogenic%20pathway%20with%20several%20open%20questions&journal=Horm.%20Cancer&volume=10&pages=51-63&publication_year=2019&author=Rodriguez%2CAC&author=Blanchard%2CZ&author=Maurer%2CKA&author=Gertz%2CJ) 
    
50.  Adesanya, O. O., Zhou, J., Samathanam, C., Powell-Braxton, L. & Bondy, C. A. Insulin-like growth factor 1 is required for G2 progression in the estradiol-induced mitotic cycle. _Proc. Natl Acad. Sci. USA_ **96**, 3287–3291 (1999).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10077676)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC15934)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insulin-like%20growth%20factor%201%20is%20required%20for%20G2%20progression%20in%20the%20estradiol-induced%20mitotic%20cycle&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=96&pages=3287-3291&publication_year=1999&author=Adesanya%2COO&author=Zhou%2CJ&author=Samathanam%2CC&author=Powell-Braxton%2CL&author=Bondy%2CCA) 
    
51.  Klotz, D. M., Hewitt, S. C., Korach, K. S. & Diaugustine, R. P. Activation of a uterine insulin-like growth factor I signaling pathway by clinical and environmental estrogens: requirement of estrogen receptor-α. _Endocrinology_ **141**, 3430–3439 (2000).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10965916)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Activation%20of%20a%20uterine%20insulin-like%20growth%20factor%20I%20signaling%20pathway%20by%20clinical%20and%20environmental%20estrogens%3A%20requirement%20of%20estrogen%20receptor-&journal=Endocrinology&volume=141&pages=3430-3439&publication_year=2000&author=Klotz%2CDM&author=Hewitt%2CSC&author=Korach%2CKS&author=Diaugustine%2CRP) 
    
52.  Aronica, S. M. & Katzenellenbogen, B. S. Stimulation of estrogen receptor-mediated transcription and alteration in the phosphorylation state of the rat uterine estrogen receptor by estrogen, cyclic adenosine monophosphate, and insulin-like growth factor-I. _Mol. Endocrinol._ **7**, 743–752 (1993).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7689695)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stimulation%20of%20estrogen%20receptor-mediated%20transcription%20and%20alteration%20in%20the%20phosphorylation%20state%20of%20the%20rat%20uterine%20estrogen%20receptor%20by%20estrogen%2C%20cyclic%20adenosine%20monophosphate%2C%20and%20insulin-like%20growth%20factor-I&journal=Mol.%20Endocrinol.&volume=7&pages=743-752&publication_year=1993&author=Aronica%2CSM&author=Katzenellenbogen%2CBS) 
    
53.  Martin, M. B. et al. A role for Akt in mediating the estrogenic functions of epidermal growth factor and insulin-like growth factor I. _Endocrinology_ **141**, 4503–4511 (2000).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11108261)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20role%20for%20Akt%20in%20mediating%20the%20estrogenic%20functions%20of%20epidermal%20growth%20factor%20and%20insulin-like%20growth%20factor%20I&journal=Endocrinology&volume=141&pages=4503-4511&publication_year=2000&author=Martin%2CMB) 
    
54.  Kashima, H. et al. Autocrine stimulation of IGF1 in estrogen-induced growth of endometrial carcinoma cells: involvement of the mitogen-activated protein kinase pathway followed by up-regulation of cyclin D1 and cyclin E. _Endocr. Relat. Cancer_ **16**, 113–122 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18852162)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autocrine%20stimulation%20of%20IGF1%20in%20estrogen-induced%20growth%20of%20endometrial%20carcinoma%20cells%3A%20involvement%20of%20the%20mitogen-activated%20protein%20kinase%20pathway%20followed%20by%20up-regulation%20of%20cyclin%20D1%20and%20cyclin%20E&journal=Endocr.%20Relat.%20Cancer&volume=16&pages=113-122&publication_year=2009&author=Kashima%2CH) 
    
55.  Cooke, P. S. et al. Stromal estrogen receptors mediate mitogenic effects of estradiol on uterine epithelium. _Proc. Natl Acad. Sci. USA_ **94**, 6535–6540 (1997).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9177253)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC21085)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stromal%20estrogen%20receptors%20mediate%20mitogenic%20effects%20of%20estradiol%20on%20uterine%20epithelium&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=94&pages=6535-6540&publication_year=1997&author=Cooke%2CPS) 
    
56.  Baxter, R. C. Signalling pathways involved in antiproliferative effects of IGFBP-3: a review. _Mol. Pathol._ **54**, 145–148 (2001).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11376125)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1187052)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Signalling%20pathways%20involved%20in%20antiproliferative%20effects%20of%20IGFBP-3%3A%20a%20review&journal=Mol.%20Pathol.&volume=54&pages=145-148&publication_year=2001&author=Baxter%2CRC) 
    
57.  Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. _Cell_ **184**, 2239–2254 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33831375)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8054914)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterizing%20genetic%20intra-tumor%20heterogeneity%20across%202%2C658%20human%20cancer%20genomes&journal=Cell&volume=184&pages=2239-2254&publication_year=2021&author=Dentro%2CSC) 
    
58.  Leshchiner, I. et al. Inferring early genetic progression in cancers with unobtainable premalignant disease. _Nat. Cancer_ **4**, 550–563 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37081260)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10132986)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inferring%20early%20genetic%20progression%20in%20cancers%20with%20unobtainable%20premalignant%20disease&journal=Nat.%20Cancer&volume=4&pages=550-563&publication_year=2023&author=Leshchiner%2CI) 
    
59.  van Leeuwen, F. E. et al. Risk of endometrial cancer after tamoxifen treatment of breast cancer. _Lancet_ **343**, 448–452 (1994).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7905955)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Risk%20of%20endometrial%20cancer%20after%20tamoxifen%20treatment%20of%20breast%20cancer&journal=Lancet&volume=343&pages=448-452&publication_year=1994&author=Leeuwen%2CFE) 
    
60.  Berg, A. et al. Molecular profiling of endometrial carcinoma precursor, primary and metastatic lesions suggests different targets for treatment in obese compared to non-obese patients. _Oncotarget_ **6**, 1327–1339 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25415225)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20profiling%20of%20endometrial%20carcinoma%20precursor%2C%20primary%20and%20metastatic%20lesions%20suggests%20different%20targets%20for%20treatment%20in%20obese%20compared%20to%20non-obese%20patients&journal=Oncotarget&volume=6&pages=1327-1339&publication_year=2015&author=Berg%2CA) 
    
61.  Moore, L. et al. The mutational landscape of normal human endometrial epithelium. _Nature_ **580**, 640–646 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32350471)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20landscape%20of%20normal%20human%20endometrial%20epithelium&journal=Nature&volume=580&pages=640-646&publication_year=2020&author=Moore%2CL) 
    
62.  Lac, V. et al. Oncogenic mutations in histologically normal endometrium: the new normal? _J. Pathol._ **249**, 173–181 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31187483)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Oncogenic%20mutations%20in%20histologically%20normal%20endometrium%3A%20the%20new%20normal%3F&journal=J.%20Pathol.&volume=249&pages=173-181&publication_year=2019&author=Lac%2CV) 
    
63.  Anglesio, M. S. et al. Cancer-associated mutations in endometriosis without cancer. _N. Engl. J. Med._ **376**, 1835–1848 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28489996)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5555376)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cancer-associated%20mutations%20in%20endometriosis%20without%20cancer&journal=N.%20Engl.%20J.%20Med.&volume=376&pages=1835-1848&publication_year=2017&author=Anglesio%2CMS) 
    
64.  Praetorius, T. H. et al. Molecular analysis suggests oligoclonality and metastasis of endometriosis lesions across anatomically defined subtypes. _Fertil. Steril._ **118**, 524–534 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35715244)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20analysis%20suggests%20oligoclonality%20and%20metastasis%20of%20endometriosis%20lesions%20across%20anatomically%20defined%20subtypes&journal=Fertil.%20Steril.&volume=118&pages=524-534&publication_year=2022&author=Praetorius%2CTH) 
    
65.  Li, L. et al. Genome-wide mutation analysis in precancerous lesions of endometrial carcinoma. _J. Pathol._ **253**, 119–128 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33016334)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20mutation%20analysis%20in%20precancerous%20lesions%20of%20endometrial%20carcinoma&journal=J.%20Pathol.&volume=253&pages=119-128&publication_year=2021&author=Li%2CL) 
    
66.  Hu, Z. et al. Proteogenomic insights into early-onset endometrioid endometrial carcinoma: predictors for fertility-sparing therapy response. _Nat. Genet._ **56**, 637–651 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38565644)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteogenomic%20insights%20into%20early-onset%20endometrioid%20endometrial%20carcinoma%3A%20predictors%20for%20fertility-sparing%20therapy%20response&journal=Nat.%20Genet.&volume=56&pages=637-651&publication_year=2024&author=Hu%2CZ) 
    
67.  Yizhak, K. et al. RNA sequence analysis reveals macroscopic somatic clonal expansion across normal tissues. _Science_ **364**, eaaw0726 (2019).
    
68.  Zhao, Y. et al. Metformin is associated with reduced cell proliferation in human endometrial cancer by inhibiting PI3K/AKT/mTOR signaling. _Gynecol. Endocrinol._ **34**, 428–432 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29182407)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Metformin%20is%20associated%20with%20reduced%20cell%20proliferation%20in%20human%20endometrial%20cancer%20by%20inhibiting%20PI3K%2FAKT%2FmTOR%20signaling&journal=Gynecol.%20Endocrinol.&volume=34&pages=428-432&publication_year=2018&author=Zhao%2CY) 
    
69.  Davis, S. R. et al. The benefits of adding metformin to tamoxifen to protect the endometrium—a randomized placebo-controlled trial. _Clin. Endocrinol._ **89**, 605–612 (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20benefits%20of%20adding%20metformin%20to%20tamoxifen%20to%20protect%20the%20endometrium%E2%80%94a%20randomized%20placebo-controlled%20trial&journal=Clin.%20Endocrinol.&volume=89&pages=605-612&publication_year=2018&author=Davis%2CSR) 
    
70.  Sholl, L. M. et al. Institutional implementation of clinical tumor profiling on an unselected cancer population. _JCI Insight_ **1**, e87062 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27882345)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5111542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Institutional%20implementation%20of%20clinical%20tumor%20profiling%20on%20an%20unselected%20cancer%20population&journal=JCI%20Insight&volume=1&publication_year=2016&author=Sholl%2CLM) 
    
71.  Evans, E. et al. Whole exome sequencing provides loss of heterozygosity (LoH) data comparable to that of whole genome sequencing (171). _Gynecol. Oncol._ **166**, S100 (2022).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole%20exome%20sequencing%20provides%20loss%20of%20heterozygosity%20%28LoH%29%20data%20comparable%20to%20that%20of%20whole%20genome%20sequencing%20%28171%29&journal=Gynecol.%20Oncol.&volume=166&publication_year=2022&author=Evans%2CE) 
    
72.  Ogobuiro, I. et al. Multiomic characterization reveals a distinct molecular landscape in young-onset pancreatic cancer. _JCO Precis. Oncol._ **7**, e2300152 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37944072)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10645414)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiomic%20characterization%20reveals%20a%20distinct%20molecular%20landscape%20in%20young-onset%20pancreatic%20cancer&journal=JCO%20Precis.%20Oncol.&volume=7&publication_year=2023&author=Ogobuiro%2CI) 
    
73.  Muquith, M. et al. Tissue-specific thresholds of mutation burden associated with anti-PD-1/L1 therapy benefit and prognosis in microsatellite-stable cancers. _Nat. Cancer_ **5**, 1121–1129 (2024).
    
74.  Auwera, Van der, G. A & O’Connor, B. D. _Genomics in the Cloud: Using Docker, GATK, and WDL in Terra_ (O’Reilly Media, 2020).
    
75.  Cibulskis, K. et al. ContEst: estimating cross-contamination of human samples in next-generation sequencing data. _Bioinformatics_ **27**, 2601–2602 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21803805)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3167057)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ContEst%3A%20estimating%20cross-contamination%20of%20human%20samples%20in%20next-generation%20sequencing%20data&journal=Bioinformatics&volume=27&pages=2601-2602&publication_year=2011&author=Cibulskis%2CK) 
    
76.  Cibulskis, K. et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. _Nat. Biotechnol._ **31**, 213–219 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23396013)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3833702)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sensitive%20detection%20of%20somatic%20point%20mutations%20in%20impure%20and%20heterogeneous%20cancer%20samples&journal=Nat.%20Biotechnol.&volume=31&pages=213-219&publication_year=2013&author=Cibulskis%2CK) 
    
77.  Saunders, C. T. et al. Strelka: accurate somatic small-variant calling from sequenced tumor–normal sample pairs. _Bioinformatics_ **28**, 1811–1817 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22581179)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Strelka%3A%20accurate%20somatic%20small-variant%20calling%20from%20sequenced%20tumor%E2%80%93normal%20sample%20pairs&journal=Bioinformatics&volume=28&pages=1811-1817&publication_year=2012&author=Saunders%2CCT) 
    
78.  Ramos, A. H. et al. Oncotator: cancer variant annotation tool. _Hum. Mutat._ **36**, E2423–E2429 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25703262)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Oncotator%3A%20cancer%20variant%20annotation%20tool&journal=Hum.%20Mutat.&volume=36&pages=E2423-E2429&publication_year=2015&author=Ramos%2CAH) 
    
79.  Costello, M. et al. Discovery and characterization of artifactual mutations in deep coverage targeted capture sequencing data due to oxidative DNA damage during sample preparation. _Nucleic Acids Res._ **41**, e67 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23303777)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3616734)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20and%20characterization%20of%20artifactual%20mutations%20in%20deep%20coverage%20targeted%20capture%20sequencing%20data%20due%20to%20oxidative%20DNA%20damage%20during%20sample%20preparation&journal=Nucleic%20Acids%20Res.&volume=41&publication_year=2013&author=Costello%2CM) 
    
80.  Giannakis, M. et al. Genomic correlates of immune-cell infiltrates in colorectal carcinoma. _Cell Rep._ **17**, 1206 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27760322)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5638785)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20correlates%20of%20immune-cell%20infiltrates%20in%20colorectal%20carcinoma&journal=Cell%20Rep.&volume=17&publication_year=2016&author=Giannakis%2CM) 
    
81.  Ellrott, K. et al. Scalable open science approach for mutation calling of tumor exomes using multiple genomic pipelines. _Cell Syst._ **6**, 271–281 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29596782)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6075717)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scalable%20open%20science%20approach%20for%20mutation%20calling%20of%20tumor%20exomes%20using%20multiple%20genomic%20pipelines&journal=Cell%20Syst.&volume=6&pages=271-281&publication_year=2018&author=Ellrott%2CK) 
    
82.  Taylor-Weiner, A. et al. DeTiN: overcoming tumor-in-normal contamination. _Nat. Methods_ **15**, 531–534 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29941871)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6528031)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DeTiN%3A%20overcoming%20tumor-in-normal%20contamination&journal=Nat.%20Methods&volume=15&pages=531-534&publication_year=2018&author=Taylor-Weiner%2CA) 
    
83.  Stachler, M. D. et al. Paired exome analysis of Barrett’s esophagus and adenocarcinoma. _Nat. Genet._ **47**, 1047–1055 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26192918)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Paired%20exome%20analysis%20of%20Barrett%E2%80%99s%20esophagus%20and%20adenocarcinoma&journal=Nat.%20Genet.&volume=47&pages=1047-1055&publication_year=2015&author=Stachler%2CMD) 
    
84.  Thorvaldsdottir, H., Robinson, J. T. & Mesirov, J. P. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. _Brief. Bioinform._ **14**, 178–192 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22517427)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20Genomics%20Viewer%20%28IGV%29%3A%20high-performance%20genomics%20data%20visualization%20and%20exploration&journal=Brief.%20Bioinform.&volume=14&pages=178-192&publication_year=2013&author=Thorvaldsdottir%2CH&author=Robinson%2CJT&author=Mesirov%2CJP) 
    
85.  Lawrence, M. S. et al. Discovery and saturation analysis of cancer genes across 21 tumour types. _Nature_ **505**, 495–501 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24390350)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4048962)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20and%20saturation%20analysis%20of%20cancer%20genes%20across%2021%20tumour%20types&journal=Nature&volume=505&pages=495-501&publication_year=2014&author=Lawrence%2CMS) 
    
86.  Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. _J. R. Stat. Soc. Ser. B._ **57**, 289–300 (1995).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Controlling%20the%20false%20discovery%20rate%3A%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B.&volume=57&pages=289-300&publication_year=1995&author=Benjamini%2CY&author=Hochberg%2CY) 
    
87.  Gopal, R. K. et al. Widespread chromosomal losses and mitochondrial DNA alterations as genetic drivers in Hürthle cell carcinoma. _Cancer Cell_ **34**, 242–255 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30107175)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6121811)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Widespread%20chromosomal%20losses%20and%20mitochondrial%20DNA%20alterations%20as%20genetic%20drivers%20in%20H%C3%BCrthle%20cell%20carcinoma&journal=Cancer%20Cell&volume=34&pages=242-255&publication_year=2018&author=Gopal%2CRK) 
    
88.  Sondka, Z. et al. The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. _Nat. Rev. Cancer_ **18**, 696–705 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30293088)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6450507)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20COSMIC%20Cancer%20Gene%20Census%3A%20describing%20genetic%20dysfunction%20across%20all%20human%20cancers&journal=Nat.%20Rev.%20Cancer&volume=18&pages=696-705&publication_year=2018&author=Sondka%2CZ) 
    
89.  Carter, S. L. et al. Absolute quantification of somatic DNA alterations in human cancer. _Nat. Biotechnol._ **30**, 413–421 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22544022)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383288)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Absolute%20quantification%20of%20somatic%20DNA%20alterations%20in%20human%20cancer&journal=Nat.%20Biotechnol.&volume=30&pages=413-421&publication_year=2012&author=Carter%2CSL) 
    
90.  Chung, J. et al. DNA polymerase and mismatch repair exert distinct microsatellite instability signatures in normal and malignant human cells. _Cancer Discov._ **11**, 1176–1191 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33355208)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20polymerase%20and%20mismatch%20repair%20exert%20distinct%20microsatellite%20instability%20signatures%20in%20normal%20and%20malignant%20human%20cells&journal=Cancer%20Discov.&volume=11&pages=1176-1191&publication_year=2021&author=Chung%2CJ) 
    
91.  Kim, J. et al. Somatic _ERCC2_ mutations are associated with a distinct genomic signature in urothelial tumors. _Nat. Genet._ **48**, 600–606 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27111033)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4936490)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20ERCC2%20mutations%20are%20associated%20with%20a%20distinct%20genomic%20signature%20in%20urothelial%20tumors&journal=Nat.%20Genet.&volume=48&pages=600-606&publication_year=2016&author=Kim%2CJ) 
    
92.  Taylor-Weiner, A. et al. Scaling computational genomics to millions of individuals with GPUs. _Genome Biol._ **20**, 228 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31675989)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823959)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Scaling%20computational%20genomics%20to%20millions%20of%20individuals%20with%20GPUs&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Taylor-Weiner%2CA) 
    
93.  Haradhvala, N. J. et al. Distinct mutational signatures characterize concurrent loss of polymerase proofreading and mismatch repair. _Nat. Commun._ **9**, 1746 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29717118)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5931517)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20mutational%20signatures%20characterize%20concurrent%20loss%20of%20polymerase%20proofreading%20and%20mismatch%20repair&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Haradhvala%2CNJ) 
    
94.  Dou, Y. et al. Proteogenomic characterization of endometrial carcinoma. _Cell_ **180**, 729–748 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32059776)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233456)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Proteogenomic%20characterization%20of%20endometrial%20carcinoma&journal=Cell&volume=180&pages=729-748&publication_year=2020&author=Dou%2CY) 
    
95.  Taylor, A. M. et al. Genomic and functional approaches to understanding cancer aneuploidy. _Cancer Cell_ **33**, 676–689 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29622463)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6028190)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20and%20functional%20approaches%20to%20understanding%20cancer%20aneuploidy&journal=Cancer%20Cell&volume=33&pages=676-689&publication_year=2018&author=Taylor%2CAM) 
    
96.  Kuang, Y. et al. Unraveling the clinicopathological features driving the emergence of _ESR1_ mutations in metastatic breast cancer. _NPJ Breast Cancer_ **4**, 22 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30083595)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6072793)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unraveling%20the%20clinicopathological%20features%20driving%20the%20emergence%20of%20ESR1%20mutations%20in%20metastatic%20breast%20cancer&journal=NPJ%20Breast%20Cancer&volume=4&publication_year=2018&author=Kuang%2CY) 
    
97.  Janiszewska, M. et al. In situ single-cell analysis identifies heterogeneity for _PIK3CA_ mutation and _HER2_ amplification in HER2-positive breast cancer. _Nat. Genet._ **47**, 1212–1219 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26301495)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4589505)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In%20situ%20single-cell%20analysis%20identifies%20heterogeneity%20for%20PIK3CA%20mutation%20and%20HER2%20amplification%20in%20HER2-positive%20breast%20cancer&journal=Nat.%20Genet.&volume=47&pages=1212-1219&publication_year=2015&author=Janiszewska%2CM) 
    
98.  Jung, S. H. Stratified Fisher’s exact test and its sample size calculation. _Biom. J._ **56**, 129–140 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24395208)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stratified%20Fisher%E2%80%99s%20exact%20test%20and%20its%20sample%20size%20calculation&journal=Biom.%20J.&volume=56&pages=129-140&publication_year=2014&author=Jung%2CSH) 
    
99.  Martín-Andrés, A. & Herranz-Tejedor, I. Regarding Paper ‘Stratified Fisher’s exact test and its sample size calculation’. _Biom. J._ **57**, 930 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26059610)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regarding%20Paper%20%E2%80%98Stratified%20Fisher%E2%80%99s%20exact%20test%20and%20its%20sample%20size%20calculation%E2%80%99&journal=Biom.%20J.&volume=57&publication_year=2015&author=Mart%C3%ADn-Andr%C3%A9s%2CA&author=Herranz-Tejedor%2CI) 
    
100.  Wu, H. et al. Hypomethylation-linked activation of _PAX2_ mediates tamoxifen-stimulated endometrial carcinogenesis. _Nature_ **438**, 981–987 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16355216)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hypomethylation-linked%20activation%20of%20PAX2%20mediates%20tamoxifen-stimulated%20endometrial%20carcinogenesis&journal=Nature&volume=438&pages=981-987&publication_year=2005&author=Wu%2CH) 
    
101.  Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. _Nucleic Acids Res._ **43**, e47 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25605792)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4402510)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies&journal=Nucleic%20Acids%20Res.&volume=43&publication_year=2015&author=Ritchie%2CME) 
    
102.  Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. _Nucleic Acids Res._ **44**, W90–W97 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27141961)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987924)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enrichr%3A%20a%20comprehensive%20gene%20set%20enrichment%20analysis%20web%20server%202016%20update&journal=Nucleic%20Acids%20Res.&volume=44&pages=W90-W97&publication_year=2016&author=Kuleshov%2CMV) 
    
103.  Schaefer, C. F. et al. PID: the Pathway Interaction Database. _Nucleic Acids Res._ **37**, D674–D679 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18832364)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PID%3A%20the%20Pathway%20Interaction%20Database&journal=Nucleic%20Acids%20Res.&volume=37&pages=D674-D679&publication_year=2009&author=Schaefer%2CCF) 
    
104.  Reid, J. M. et al. Pharmacokinetics of endoxifen and tamoxifen in female mice: implications for comparative in vivo activity studies. _Cancer Chemother. Pharmacol._ **74**, 1271–1278 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25318936)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4343319)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pharmacokinetics%20of%20endoxifen%20and%20tamoxifen%20in%20female%20mice%3A%20implications%20for%20comparative%20in%20vivo%20activity%20studies&journal=Cancer%20Chemother.%20Pharmacol.&volume=74&pages=1271-1278&publication_year=2014&author=Reid%2CJM) 
    
105.  De Clercq, K., Hennes, A. & Vriens, J. Isolation of mouse endometrial epithelial and stromal cells for in vitro decidualization. _J. Vis. Exp._ [https://doi.org/10.3791/55168](https://doi.org/10.3791/55168) (2017).
    
106.  Cornwell, M. et al. VIPER: Visualization Pipeline for RNA-seq, a Snakemake workflow for efficient and complete RNA-seq analysis. _BMC Bioinformatics_ **19**, 135 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29649993)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5897949)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=VIPER%3A%20Visualization%20Pipeline%20for%20RNA-seq%2C%20a%20Snakemake%20workflow%20for%20efficient%20and%20complete%20RNA-seq%20analysis&journal=BMC%20Bioinformatics&volume=19&publication_year=2018&author=Cornwell%2CM) 
    
107.  Trapnell, C. et al. Transcript assembly and quantification by RNA-seq reveals unannotated transcripts and isoform switching during cell differentiation. _Nat. Biotechnol._ **28**, 511–515 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20436464)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3146043)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transcript%20assembly%20and%20quantification%20by%20RNA-seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&journal=Nat.%20Biotechnol.&volume=28&pages=511-515&publication_year=2010&author=Trapnell%2CC) 
    
108.  Wang, L., Wang, S. & Li, W. RSeQC: quality control of RNA-seq experiments. _Bioinformatics_ **28**, 2184–2185 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22743226)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RSeQC%3A%20quality%20control%20of%20RNA-seq%20experiments&journal=Bioinformatics&volume=28&pages=2184-2185&publication_year=2012&author=Wang%2CL&author=Wang%2CS&author=Li%2CW) 
    
109.  Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._ **15**, 550 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25516281)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Love%2CMI&author=Huber%2CW&author=Anders%2CS) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02308-w?format=refman&flavour=references)

Acknowledgements
----------------

We acknowledge the American Association for Cancer Research and its financial and material support in the development of the AACR Project GENIE registry as well as members of the consortium for their commitment to data sharing. We acknowledge the efforts of the National Cancer Institute; the Office of Research, Development and Information, CMS; Information Management Services; and SEER Program tumor registries in the creation of the SEER-Medicare database. Interpretations are the responsibility of study authors. We thank L. C. Cantley, M. Goncalves, M. Brown, I. Lee and L. Ellisen for helpful discussion. We acknowledge C. Birger for help in setting up the Terra workspace and H. Hollema for his contributions to the TAMARISK study. We thank K. Slowik for assistance in sequencing data management. We are greatly indebted to SciStories for assistance with scientific cartoons. We thank M. Capelletti and S. Ressler of Caris Life Sciences for their assistance with the letter of intent, their valuable advice and input and their support in identifying the cohort. We acknowledge the DFCI Oncology Data Retrieval System for the aggregation, management and delivery of the clinical and operational research data used in this project and NKI-AVL Core Facility Molecular Pathology & Biobanking for supplying NKI-AVL Biobank material and /or laboratory support. We acknowledge funding from the Susan F. Smith Center for Women’s Cancers at the DFCI to R.J., a R01 from the NCI (5R01CA237414-05) to R.J. and the Claudia Adams Barr Program to R.J. Additional support was provided by Pink Ribbon and a KWF Dutch Cancer Society grant to W.Z. K.K. and Y.E.M. were partly supported by startup funds from G.G. at Massachusetts General Hospital. K.K. and G.G. were supported by a CDMRP award (W81XWH-17-1-0084). K.K. also received support from the Private Excellence Initiative Johanna Quandt of the Stiftung Charité. G.G. is partly supported by the Paul C. Zamecnik Chair in Oncology at the Massachusetts General Hospital Cancer Center. U.A.M. receives grant funding from the Dana-Farber–Harvard Cancer Center Ovarian Cancer SPORE grant (P50CA240243) and the Breast Cancer Research Foundation.

Author information
------------------

Author notes

1.  Tara Akhshi
    
    Present address: Harvard Medical School, Boston, MA, USA
    
2.  These authors contributed equally: Kirsten Kübler, Agostina Nardone.
    
3.  These authors jointly supervised this work: Wilbert Zwart, Yosef E. Maruvka, Gad Getz, Rinath Jeselsohn.
    

### Authors and Affiliations

1.  Broad Institute of MIT and Harvard, Cambridge, MA, USA
    
    Kirsten Kübler, Shankara Anand, Justin Cha, Mendy Miller, William J. Gibson, Eliezer M. Van Allen, Ignaty Leshchiner, Chip Stewart, Gad Getz & Rinath Jeselsohn
    
2.  Krantz Family Center for Cancer Research, Massachusetts General Hospital, Charlestown, MA, USA
    
    Kirsten Kübler & Gad Getz
    
3.  Harvard Medical School, Boston, MA, USA
    
    Kirsten Kübler, Ursula A. Matulonis, Gad Getz & Rinath Jeselsohn
    
4.  Berlin Institute of Health at Charité–Universitätsmedizin Berlin, Berlin, Germany
    
    Kirsten Kübler
    
5.  Department of Hematology, Oncology and Cancer Immunology, Charité–Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Berlin, Germany
    
    Kirsten Kübler
    
6.  German Cancer Consortium (DKTK), Partner Site Berlin and German Cancer Research Center (DKFZ), Heidelberg, Germany
    
    Kirsten Kübler
    
7.  Center for Functional Cancer Epigenetics, Dana-Farber Cancer Institute, Boston, MA, USA
    
    Agostina Nardone, Francisco Hermida-Prado, Tara Akhshi, Avery S. Feit, Gabriella Cohen Feit, Matthew Pun & Rinath Jeselsohn
    
8.  Biotechnology and Food Engineering, Technion, Haifa, Israel
    
    Daniel Gurevich & Yosef E. Maruvka
    
9.  Lokey Center for Life Science and Engineering, Technion, Haifa, Israel
    
    Daniel Gurevich & Yosef E. Maruvka
    
10.  Caris Life Sciences, Phoenix, AZ, USA
    
    Jianjiong Gao & Milan Radovich
    
11.  Division of Oncogenomics, Oncode Institute, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Marjolein Droog, Sebastian Gregoricchio & Wilbert Zwart
    
12.  Department of Biomedical Informatics, Harvard Medical School, Boston, MA, USA
    
    Ariel Feiglin
    
13.  Division of Molecular Pathology, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Gwen Dackus
    
14.  Department of Pathology, Radboud University Medical Center, Nijmegen, the Netherlands
    
    Gwen Dackus
    
15.  Belfer Center for Applied Cancer Science, Dana-Farber Cancer Institute, Boston, MA, USA
    
    Yanan Kuang & Cloud P. Paweletz
    
16.  Department of Laboratory Medicine, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Mirthe Lanfermeijer
    
17.  Core Facility Molecular Pathology & Biobanking, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Sten Cornelissen
    
18.  Center for Cancer Precision Medicine, Dana-Farber Cancer Institute, Boston, MA, USA
    
    Eliezer M. Van Allen
    
19.  Department of Epidemiology, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Flora E. van Leeuwen
    
20.  Department of Pathology, Netherlands Cancer Institute, Amsterdam, the Netherlands
    
    Petra M. Nederlof
    
21.  Lurie Family Imaging Center, Center for Biomedical Imaging in Oncology, Dana-Farber Cancer Institute, Boston, MA, USA
    
    Quang-Dé Nguyen
    
22.  Department of Gynecological Oncology, University Medical Center Groningen, University of Groningen, Groningen, the Netherlands
    
    Marian J. E. Mourits
    
23.  Department of Medicine, Boston University School of Medicine, Boston, MA, USA
    
    Ignaty Leshchiner
    
24.  The Susan F. Smith Center for Women’s Cancers, Dana-Farber Cancer Institute, Boston, MA, USA
    
    Ursula A. Matulonis & Rinath Jeselsohn
    
25.  Division of Gynecologic Oncology, Dana-Farber Cancer Institute, Boston, MA, USA
    
    Ursula A. Matulonis
    
26.  Laboratory of Chemical Biology and Institute for Complex Molecular Systems, Department of Biomedical Engineering, Eindhoven University of Technology, Eindhoven, the Netherlands
    
    Wilbert Zwart
    
27.  Department of Pathology, Massachusetts General Hospital, Boston, MA, USA
    
    Gad Getz
    

Authors

1.  Kirsten Kübler
    
    [View author publications](https://www.nature.com/search?author=Kirsten%20K%C3%BCbler)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kirsten%20K%C3%BCbler) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kirsten%20K%C3%BCbler%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Agostina Nardone
    
    [View author publications](https://www.nature.com/search?author=Agostina%20Nardone)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Agostina%20Nardone) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Agostina%20Nardone%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Shankara Anand
    
    [View author publications](https://www.nature.com/search?author=Shankara%20Anand)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shankara%20Anand) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shankara%20Anand%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Daniel Gurevich
    
    [View author publications](https://www.nature.com/search?author=Daniel%20Gurevich)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20Gurevich) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20Gurevich%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Jianjiong Gao
    
    [View author publications](https://www.nature.com/search?author=Jianjiong%20Gao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jianjiong%20Gao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jianjiong%20Gao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Marjolein Droog
    
    [View author publications](https://www.nature.com/search?author=Marjolein%20Droog)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marjolein%20Droog) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marjolein%20Droog%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Francisco Hermida-Prado
    
    [View author publications](https://www.nature.com/search?author=Francisco%20Hermida-Prado)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Francisco%20Hermida-Prado) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Francisco%20Hermida-Prado%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Tara Akhshi
    
    [View author publications](https://www.nature.com/search?author=Tara%20Akhshi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tara%20Akhshi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tara%20Akhshi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Ariel Feiglin
    
    [View author publications](https://www.nature.com/search?author=Ariel%20Feiglin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ariel%20Feiglin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ariel%20Feiglin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Avery S. Feit
    
    [View author publications](https://www.nature.com/search?author=Avery%20S.%20Feit)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Avery%20S.%20Feit) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Avery%20S.%20Feit%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Gabriella Cohen Feit
    
    [View author publications](https://www.nature.com/search?author=Gabriella%20Cohen%20Feit)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gabriella%20Cohen%20Feit) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gabriella%20Cohen%20Feit%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Gwen Dackus
    
    [View author publications](https://www.nature.com/search?author=Gwen%20Dackus)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gwen%20Dackus) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gwen%20Dackus%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Matthew Pun
    
    [View author publications](https://www.nature.com/search?author=Matthew%20Pun)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20Pun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20Pun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Yanan Kuang
    
    [View author publications](https://www.nature.com/search?author=Yanan%20Kuang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yanan%20Kuang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yanan%20Kuang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Justin Cha
    
    [View author publications](https://www.nature.com/search?author=Justin%20Cha)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Justin%20Cha) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Justin%20Cha%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Mendy Miller
    
    [View author publications](https://www.nature.com/search?author=Mendy%20Miller)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mendy%20Miller) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mendy%20Miller%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Sebastian Gregoricchio
    
    [View author publications](https://www.nature.com/search?author=Sebastian%20Gregoricchio)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sebastian%20Gregoricchio) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sebastian%20Gregoricchio%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Mirthe Lanfermeijer
    
    [View author publications](https://www.nature.com/search?author=Mirthe%20Lanfermeijer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mirthe%20Lanfermeijer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mirthe%20Lanfermeijer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Sten Cornelissen
    
    [View author publications](https://www.nature.com/search?author=Sten%20Cornelissen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sten%20Cornelissen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sten%20Cornelissen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  William J. Gibson
    
    [View author publications](https://www.nature.com/search?author=William%20J.%20Gibson)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=William%20J.%20Gibson) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22William%20J.%20Gibson%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Cloud P. Paweletz
    
    [View author publications](https://www.nature.com/search?author=Cloud%20P.%20Paweletz)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Cloud%20P.%20Paweletz) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Cloud%20P.%20Paweletz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Eliezer M. Van Allen
    
    [View author publications](https://www.nature.com/search?author=Eliezer%20M.%20Van%20Allen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eliezer%20M.%20Van%20Allen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eliezer%20M.%20Van%20Allen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Flora E. van Leeuwen
    
    [View author publications](https://www.nature.com/search?author=Flora%20E.%20van%20Leeuwen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Flora%20E.%20van%20Leeuwen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Flora%20E.%20van%20Leeuwen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Petra M. Nederlof
    
    [View author publications](https://www.nature.com/search?author=Petra%20M.%20Nederlof)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Petra%20M.%20Nederlof) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Petra%20M.%20Nederlof%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Quang-Dé Nguyen
    
    [View author publications](https://www.nature.com/search?author=Quang-D%C3%A9%20Nguyen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Quang-D%C3%A9%20Nguyen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Quang-D%C3%A9%20Nguyen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Marian J. E. Mourits
    
    [View author publications](https://www.nature.com/search?author=Marian%20J.%20E.%20Mourits)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marian%20J.%20E.%20Mourits) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marian%20J.%20E.%20Mourits%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  Milan Radovich
    
    [View author publications](https://www.nature.com/search?author=Milan%20Radovich)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Milan%20Radovich) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Milan%20Radovich%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Ignaty Leshchiner
    
    [View author publications](https://www.nature.com/search?author=Ignaty%20Leshchiner)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ignaty%20Leshchiner) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ignaty%20Leshchiner%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Chip Stewart
    
    [View author publications](https://www.nature.com/search?author=Chip%20Stewart)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chip%20Stewart) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chip%20Stewart%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  Ursula A. Matulonis
    
    [View author publications](https://www.nature.com/search?author=Ursula%20A.%20Matulonis)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ursula%20A.%20Matulonis) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ursula%20A.%20Matulonis%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  Wilbert Zwart
    
    [View author publications](https://www.nature.com/search?author=Wilbert%20Zwart)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wilbert%20Zwart) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wilbert%20Zwart%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
32.  Yosef E. Maruvka
    
    [View author publications](https://www.nature.com/search?author=Yosef%20E.%20Maruvka)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yosef%20E.%20Maruvka) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yosef%20E.%20Maruvka%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
33.  Gad Getz
    
    [View author publications](https://www.nature.com/search?author=Gad%20Getz)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gad%20Getz) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gad%20Getz%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
34.  Rinath Jeselsohn
    
    [View author publications](https://www.nature.com/search?author=Rinath%20Jeselsohn)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rinath%20Jeselsohn) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rinath%20Jeselsohn%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

K.K., A.N., U.A.M., W.Z., Y.E.M., G.G. and R.J. designed the study. A.N., G.C.F., M.D., G.D., F.H.-P., T.A., M.P., M.L., S.C. and Y.K. contributed to the wet laboratory experiments; C.P.P. supervised wet laboratory experiments. P.M.N., J.G. and Q.-D.N. provided data or metadata. K.K., A.N., A.S.F., S.A., A.F., D.G., S.G. and Y.E.M. performed analyses; J.C., I.L., W.J.G., E.M.V.A. and C.S. contributed advice on data analyses. F.E.v.L., U.A.M., M.R. and M.J.E.M. provided scientific insight and/or contributed to the interpretation of parts of the data. K.K., A.N., Y.E.M., G.G. and R.J. wrote the paper with support from M.M.; W.Z., Y.E.M., G.G. and R.J. supervised the study; all authors edited and reviewed the paper.

### Corresponding authors

Correspondence to [Kirsten Kübler](mailto:kirsten.kuebler@bih-charite.de), [Gad Getz](mailto:gadgetz@broadinstitute.org) or [Rinath Jeselsohn](mailto:rinath_jeselsohn@dfci.harvard.edu).

Ethics declarations
-------------------

### Competing interests

G.G. receives research funds from IBM, Pharmacyclics–AbbVie, Bayer, Genentech, Calico, Ultima Genomics, Inocras, Google, Kite and Novartis and is also an inventor on patent applications filed by the Broad Institute related to MSMuTect, MSMutSig, POLYSOLVER, SignatureAnalyzer-GPU, MSEye and MinimuMM-seq, and DLBclass. He is a founder of and a consultant to and holds privately held equity in Scorpion Therapeutics; he is also a founder of and holds privately held equity in PreDICTA Biosciences; and he holds privately held equity in Antares Therapeutics. R.J. received research funding from Lilly, Pfizer and Novartis and serves on an advisory board for GE Healthcare and Carrick Therapeutics. Y.E.M. is a consultant in Foresee Genomics. C.P.P. holds stock and other ownership interests in Xsphera Biosciences and receives honoraria from Bio-Rad and consults or advises for DropWorks and Xsphera Biosciences. C.P.P. also has sponsored research agreements with Daiichi Sankyo, Bicycle Therapeutics, Transcenta, Bicara Therapeutics, AstraZeneca, Intellia Therapeutics, Janssen Pharmaceuticals and Array BioPharma. W.J.G. is a cofounder of and holds equity in Ampressa Therapeutics, is a consultant for and holds equity in inference and has received consulting fees from Boston Clinical Research Institute, Belharra Therapeutics, Faze Medicine and ImmPACT Bio. E.M.V.A. reports an advisory role and/or consulting with Tango Therapeutics, Genome Medical, Invitae, Enara Bio, Janssen, Manifold Bio and Monte Rosa; research support from Novartis and BMS; equity in Tango Therapeutics, Genome Medical, Syapse, Enara Bio, Manifold Bio, Microsoft and Monte Rosa; travel reimbursement from Roche–Genentech and institutional patents filed on chromatin mutations and immunotherapy response and methods for clinical interpretation. W.Z. receives research funding and advises for Astellas Pharma. U.A.M. reports receiving consulting fees received from Merck, Novartis, Blueprint Medicines, AstraZeneca and NextCure as well as participating on a data safety monitoring board or an advisory board for Symphogen and Advaxis. M.J.E.M. receives research funding from W.J. Thijn Stichting. I.L. is a consultant for PACT Pharma and is a board member, scientific advisor and consultant to Ennov1. A.N. is currently employed by AstraZeneca. J.G. and M.R. disclose a financial association with Caris Life Sciences, including full-time employment, travel and/or speaking expenses and stock and/or stock options. The other authors declare no competing interests.

### Peer review information

_Nature Genetics_ thanks the anonymous reviewers for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Absence of tamoxifen-induced mutagenesis in TA-UC.](https://www.nature.com/articles/s41588-025-02308-w/figures/6)

(**a**) CONSORT flow diagram depicts allocation of Netherlands Cancer Institute (NKI) patients from the TAMARISK study for our analysis. (**b**) Bar plot of uterine cancer (UC) cohorts with and without history of tamoxifen (TA); bars represent histological type frequencies (endom., endometrial); error bars reflect standard deviation from the β-distribution; significance analysis by two-sided Fisher’s exact test with Benjamini-Hochberg procedure; numbers in/above bars indicate tumor count per group. (**c**) Bar plot of TA-UC and de novo UC cases (excluding 7 endometroid and 4 serous endometrial TCGA tumors due to lack of annotation); bars represent molecular subtype frequencies; error bars reflect standard deviation from the β-distribution; numbers in bars indicate tumor count per group. Significance analysis by two-sided Fisher’s exact test with Benjamini-Hochberg procedure (CIN, chromosomal instable; GS, genomically stable; MSI, microsatellite instability; POLE, polymerase ε). (**d**) MSI scores for each TA-UC sample (dots), generated by MSIDetect (see [Methods](https://www.nature.com/articles/s41588-025-02308-w#Sec9)); corresponding normal samples served as controls; tumors with a higher score than in the normal were classified as MSI cases. (**e**) Number of non-synonymous mutations per exome (mutations/Megabase, left) and fraction of chromosomal regions affected by ABSOLUTE somatic copy number alterations (SCNAs) out of all measured regions (right); dots represent single samples; horizontal lines indicate group medians. Significance analysis by two-sided Wilcoxon test. (**f**) Number of non-synonymous mutations grouped by molecular subtype as in **c**. Individual data points (black) overlay summary statistic boxplots; horizontal center lines indicate median; boxes span the interquartile range (IQR, 25th to 75th percentile); whiskers extend to the most extreme values within 1.5×IQR. Significance analysis by two-sided Wilcoxon test with Benjamini-Hochberg procedure. (**g**) Number of chromosomal regions affected by somatic copy number alterations (that is, amplifications and deletions; top) or deletions only (bottom), grouped by molecular subtypes as in **c**. Individual data points (black) overlay summary statistic boxplots; horizontal center lines indicate median; boxes span the interquartile range (IQR, 25th to 75th percentile); whiskers extend to the most extreme values within 1.5×IQR. Significance analysis by two-sided Wilcoxon test with Benjamini-Hochberg procedure.

### [Extended Data Fig. 2 Mutational alterations in TA-UC.](https://www.nature.com/articles/s41588-025-02308-w/figures/7)

(**a**) Mutational matrix of TA-UC (discovery cohort) decomposed into four signatures; colors represent the six base substitution types (top y-axis), further stratified by 5’ and 3’ flanking bases (bottom y-axis). Patterns were matched to COSMIC reference signatures; known etiologies are shown on the right. (**b**) Mutational signature activity per sample, shown as count (left) and fraction (right) of mutations attributed to each signature (color-coded; identified as in **a**). (**c**) Rank order of UC driver genes powered to detect differences (higher or lower) in mutation frequencies (mut freq) between TCGA de novo UC and TA-UC (discovery cohort); lines connect gene ranks between cohorts. (**d**) UC driver genes powered to detect lower mutation frequencies in TA-UC compared to de novo UC (_P_\-value threshold for statistical power analysis at <0.05, genes mutated in at least 76 de novo UC samples can potentially be considered significantly less mutated in TA-UC). Genes are colored by pathway. Gray line indicates equal frequencies; data points represent number (no) of mutated tumors; error bars reflect Poisson-based standard deviation estimate. Significance analysis by one-sided Fisher’s exact test with Benjamini-Hochberg procedure (_Q_\-values added for all _Q_ < 0.1 and/or PI3K pathway genes; \* and _sign_ denote significance). (**e**) Bar plot of mean gene coverage across samples, ordered high to low; gray line indicates the low-coverage threshold; white crosses indicate presence of a mutation. (**f**) Integrated plot of _PIK3CA_ mutations in TA-UC samples detected by whole-exome sequencing (WES; blue; cancer cell fraction \[CCF\] shown) and droplet digital PCR (ddPCR; red; variant allele frequency \[VAF; %\] shown), ordered top to bottom by protein change (NA, not available). (**g**) Density histogram showing fraction of tumors grouped by number (no.) of mutations in key PI3K pathway genes per sample; error bars reflect standard deviation from the β-distribution; significance analysis by two-sided Wilcoxon test; numbers in bars indicate mutated tumor counts per group. (**h**) Violin plots showing timing differences of early clonal driver mutations between TA-UC and TCGA de novo UC; significance values from permutation tests with Benjamini-Hochberg procedure.

### [Extended Data Fig. 3 Genomic alterations in de novo UC samples from TCGA.](https://www.nature.com/articles/s41588-025-02308-w/figures/8)

(**a**) CONSORT flow diagram of de novo UC allocation from the TCGA PanCanAtlas. (**b**) Plot of genomic features, top panel depicts mutation frequency per megabase (Mb); bottom panel depicts significantly mutated genes detected by MutSig2CV (red line, _Q_ < 0.1), ordered by significance; genes significantly mutated in the TA-UC cohort are shown in bold. (**c**) Amplifications (left, red) and deletions (right, blue) detected by GISTIC. Chromosomal positions from top to bottom; _Q-_values from left to right (green line, _Q_ < 0.25). Significant peaks are annotated with chromosomal position and candidate cancer genes, where applicable.

### [Extended Data Fig. 4 Copy number changes in TA-UC.](https://www.nature.com/articles/s41588-025-02308-w/figures/9)

(**a**) TA-UC samples are grouped according to mutated genes in Fig. [1b](https://www.nature.com/articles/s41588-025-02308-w#Fig1), with each column representing one tumor. _Top_: Sample identifiers of the TA-UC discovery cohort; molecular (MSI, microsatellite instability; CIN, chromosomally instability; GS, genomically stable; POLE, polymerase ε) and histological classifications; ABSOLUTE-generated ploidy values; presence of whole-genome doubling (WGD). _Middle_: ABSOLUTE total copy numbers of individual segments delineated by their genomic position along the 22 chromosomes (top to bottom); colors indicate loss, copy neutral loss-of-heterozygosity (LOH) or gain at genomic loci. _Bottom_: colored boxes indicate presence of a mutation; bars on the right of the boxes depict cancer cell fraction (CCF); significantly mutated genes are shown in bold as in Fig. [1b](https://www.nature.com/articles/s41588-025-02308-w#Fig1); genes of the PI3K pathway are in violet. (**b**) Amplifications (left, red) and deletions (right; blue) detected by GISTIC in the TA-UC discovery cohort. Chromosomal positions from top to bottom; _Q-_values from left to right (green line, _Q_ < 0.25). Significant peaks are annotated with chromosomal position and candidate cancer genes, where applicable (black); positions of non-significant genes of the PI3K pathway are also annotated (gray). (**c**) Bar plot of tumors with genomic alterations in key PI3K pathway genes including single-nucleotide variants (mut) and somatic copy number alterations (gain/deletion); only TCGA tumors with both data types considered; genes altered by either type counted once per tumor; bars represent mutation frequencies, with genes ordered by _P_\-value (top to bottom); error bars reflect standard deviation from the β-distribution; significance analysis by two-sided Fisher’s exact test with and without Benjamini-Hochberg procedure; numbers in bars indicate mutated tumor count per group.

### [Extended Data Fig. 5 _PIK3CA_ mutation frequencies in de novo UC by body mass.](https://www.nature.com/articles/s41588-025-02308-w/figures/10)

Bar plots of TCGA (left) and CPTAC (right) de novo UC cohorts; bars represent _PIK3CA_ mutation frequencies across three body mass index (BMI) groups: normal weight (NW, BMI < 25), overweight (OW, BMI 25 – 29.9), and obese (OB, BMI ≥ 30). Error bars reflect standard deviation from the β-distribution; significance analysis by two-sided Fisher’s exact test; numbers in bars indicate mutated tumor count per group.

### [Extended Data Fig. 6 TA-UC validation cohorts.](https://www.nature.com/articles/s41588-025-02308-w/figures/11)

(**a**, **d**, **f**) Time course plots for patients in the TAMARISK validation cohort (**a**), the clinical gene panel sequencing cohort (**d**), and the clinical whole-exome sequencing (WES) validation cohort (**f**), showing duration of tamoxifen (TA) treatment (colored bars) and periods of uterine cancer (UC) diagnosis (diagn., gray bars). (**b**) Integrated plot of _PIK3CA_ and _ESR1_ hotspot mutations in TA-UC (TAMARISK validation cohort), detected by droplet digital PCR (ddPCR; red; variant allelic fraction \[VAF, %\] shown); each column represents one tumor (NA, not available). (**c**, **g**) CONSORT flow diagrams showing patient allocation (BC, breast cancer; CxCa, cervical cancer; HRD, homologous recombination deficiency; met, metastatic; OvCa, ovarian cancer; synchr, synchronous; UC, uterine cancer; yrs, years) for clinical gene panel sequencing at Dana-Farber Cancer Institute (DFCI; **c**) and clinical whole-exome sequencing (WES; **g**). (**e**, **h**) Bar plots showing frequencies of histological uterine cancer (UC) types (endom, endometrial) in patients with and without history of tamoxifen (TA) from clinical gene panel sequencing (**e**) and clinical whole-exome sequencing (WES) compared to SEER9 data (**h**); error bars reflect the standard deviation from the β-distribution; numbers in/above bars indicate tumor count per group; significance analysis by two-sided Fisher’s exact test with Monte Carlo Benjamini-Hochberg procedure. (**i**) Bar plot of clinical (clin) whole-exome sequencing (WES) data from patients with de novo UC (endom, endometrial); bars show _PIK3CA_ and _PIK3R1_ mutation frequencies, grouped by histological subtype; numbers in/above bars indicate number of mutated samples (before the slash) and total number of samples in that subtype (after the slash); error bars reflect the standard deviation from the β-distributions. (**j**) Bar plot of clinical WES data from breast cancer patients with and without history of tamoxifen (TA); bars show mutation frequencies; error bars reflect the standard deviation from the β-distribution; numbers in bars indicate mutated tumor count per group. Significance analysis by two-sided Fisher’s exact test.

### [Extended Data Fig. 7 Mouse endometrial epithelial cell purification.](https://www.nature.com/articles/s41588-025-02308-w/figures/12)

(**a**) Schematic illustrating the experimental design for the collection of uterine horns, isolation of endometrial epithelial cells and downstream analyses. Uterine horns were used for formalin fixation and paraffin embedding followed by immunohistochemistry (left) and for RNA sequencing (RNA-seq, right) from mice as indicated. (**b**) Relative quantitative reverse transcription PCR (qRT-PCR) of _Epcam_ and _Vim_ mRNA expression in mouse endometrial cell populations isolated from a tamoxifen-treated mouse (n = 1) to confirm the purity of the isolated cell fractions. _Gapdh_ was used as reference. Each symbol represents a technical replicate; center line depicts mean; error bars represent SEM.

### [Extended Data Fig. 8 Effects of tamoxifen and E2 in mouse endometrial epithelial cells.](https://www.nature.com/articles/s41588-025-02308-w/figures/13)

(**a**) Pathway enrichment analysis on the differently expressed genes identified by DEseq2 from comparing estradiol (E2) versus tamoxifen (Tam) in endometrial epithelial cells (_Q_ < 0.01, Benjamini-Hochberg-corrected two-sided Wald test). Bar plot depicts the odds ratio of pathway enrichment (MSigDB oncogenic signatures); purple line indicates the _Q_\-values from the Benjamini-Hochberg-corrected two-sided Fisher’s exact tests. (**b**) Venn diagram showing the genes upregulated after tamoxifen treatment compared to vehicle (veh) control, and genes upregulated after E2 compared to veh control (DEseq2, log2FC > 1, _Q_ < 0.01, Benjamini-Hochberg-corrected two-sided Wald test). (**c**–**e**) Pathway enrichment analysis (MSigDB oncogenic signatures) on: (**c**) genes upregulated with tamoxifen treatment versus vehicle that are not shared with the E2-upregulated genes (n = 962, as shown in the Venn diagram in **b**); (**d**) shared genes upregulated by tamoxifen treatment versus vehicle and E2 treatment versus vehicle; and (**e**) genes upregulated by E2 treatment versus vehicle but not upregulated by tamoxifen versus vehicle. Bar plots depict the odds ratio of pathway enrichment, _Q_\-values from the Benjamini-Hochberg-corrected two-sided Fisher’s exact tests. (**f**) Comparison of differentially expressed genes (log2 fold change \[FC\]) between tamoxifen (tam) over vehicle control (x-axis) and tamoxifen plus alpelisib (y-axis). Genes with FDR < 0.05 (DEseq2) are categorized and color-coded. (**g**) Representative immunohistochemistry images of ER expression (nuclear brown staining) in epithelial (black arrow) and stromal cells (red arrow) in the uteri from mice treated with vehicle control, estrogen (E2), and tamoxifen. Scale bars: 10μm, H&E counterstaining. Each treatment group represents independently repeated experiments with similar results: vehicle (n = 2 mice), E2 (n = 3 mice), and tamoxifen (n = 5 mice). (**h**) Heatmap of log2 FPKM expression levels (scale 0-5) of genes related to _Igf1r_ in mouse endometrial epithelial cells from vehicle (Veh), estradiol (E2), tamoxifen (Tam) and tamoxifen plus alpelisib (Tam+Alp) treatment cohorts.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_MOESM1_ESM.pdf)

Supplementary Figs. 1 and 2 and Notes 1–7.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_MOESM2_ESM.pdf)

### [Supplementary Tables 1–15](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02308-w/MediaObjects/41588_2025_2308_MOESM3_ESM.xlsx)

See specific legends in the first tab ‘Table Explanations’.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Tamoxifen%20induces%20PI3K%20activation%20in%20uterine%20cancer&author=Kirsten%20K%C3%BCbler%20et%20al&contentID=10.1038%2Fs41588-025-02308-w&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-22&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02308-w)

### Cite this article

Kübler, K., Nardone, A., Anand, S. _et al._ Tamoxifen induces PI3K activation in uterine cancer. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02308-w

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02308-w?format=refman&flavour=citation)

-   Received: 17 March 2022
    
-   Accepted: 22 July 2025
    
-   Published: 22 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02308-w
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02308-w.pdf)
