> id: https://www.nature.com/articles/s41588-025-02296-x

> link: https://www.nature.com/articles/s41588-025-02296-x

> title: Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis

# Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis
_Published on Thu, 28 Aug 2025 00:00:00 GMT_

Abstract
--------

The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in _PPM1D_ and the _TERT_ gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41592-024-02495-0/MediaObjects/41592_2024_2495_Fig1_HTML.png)

### [The dynamics of hematopoiesis over the human lifespan](https://www.nature.com/articles/s41592-024-02495-0?fromPaywallRec=false)

Article 05 December 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-024-07214-1/MediaObjects/42003_2024_7214_Fig1_HTML.png)

### [Targeting heterochromatin eliminates chronic myelomonocytic leukemia malignant stem cells through reactivation of retroelements and immune pathways](https://www.nature.com/articles/s42003-024-07214-1?fromPaywallRec=false)

Article Open access 22 November 2024

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01755-1/MediaObjects/41588_2024_1755_Fig1_HTML.png)

### [Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis](https://www.nature.com/articles/s41588-024-01755-1?fromPaywallRec=false)

Article Open access 14 May 2024

Main
----

Advancing age is associated with the development of somatic-mutation-driven clonal expansions in most tissues[1](https://www.nature.com/articles/s41588-025-02296-x#ref-CR1 "Martincorena, I. & Campbell, P. J. Somatic mutation in cancer and normal cells. Science 349, 1483–1489 (2015)."). In blood, somatic mutations that augment cellular fitness of individual hematopoietic stem cells (HSCs) give rise to CH[2](https://www.nature.com/articles/s41588-025-02296-x#ref-CR2 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[3](https://www.nature.com/articles/s41588-025-02296-x#ref-CR3 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[4](https://www.nature.com/articles/s41588-025-02296-x#ref-CR4 "McKerrell, T. et al. Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis. Cell Rep. 10, 1239–1245 (2015).")—a common phenomenon associated with an increased risk of hematological cancers and some nonhematological conditions[2](https://www.nature.com/articles/s41588-025-02296-x#ref-CR2 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[3](https://www.nature.com/articles/s41588-025-02296-x#ref-CR3 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[6](https://www.nature.com/articles/s41588-025-02296-x#ref-CR6 "Gu, M. et al. Multiparameter prediction of myeloid neoplasia risk. Nat. Genet. 55, 1523–1530 (2023)."). Mutations in the epigenetic regulator genes _DNMT3A_, _TET2_ and _ASXL1_ are responsible for ~70% of CH cases, with most of the remaining cases driven by mutations in genes involved in RNA splicing (_SF3B1_, _SRSF2_ and _U2AF1_), DNA damage response (DDR) (_TP53_ and _PPM1D_), cytokine signaling (_JAK2_) and G-protein signaling (_GNB1_ and _GNAS_)[2](https://www.nature.com/articles/s41588-025-02296-x#ref-CR2 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[3](https://www.nature.com/articles/s41588-025-02296-x#ref-CR3 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022).").

Despite substantial progress in understanding the natural history of CH[7](https://www.nature.com/articles/s41588-025-02296-x#ref-CR7 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[8](https://www.nature.com/articles/s41588-025-02296-x#ref-CR8 "Robertson, N. A. et al. Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects. Nat. Med. 28, 1439–1446 (2022)."), and identifying many of its causes and consequences[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[9](https://www.nature.com/articles/s41588-025-02296-x#ref-CR9 "Bick, A. G. et al. Inherited causes of clonal haematopoiesis in 97,691 whole genomes. Nature 586, 763–768 (2020)."),[10](https://www.nature.com/articles/s41588-025-02296-x#ref-CR10 "Kessler, M. D. et al. Common and rare variant associations with clonal haematopoiesis phenotypes. Nature 612, 301–309 (2022)."), our understanding of its pathogenesis remains limited. An important insight into CH pathogenesis came from the identification of a significant association between polymorphisms at the telomerase reverse transcriptase (_TERT_) locus and CH risk[9](https://www.nature.com/articles/s41588-025-02296-x#ref-CR9 "Bick, A. G. et al. Inherited causes of clonal haematopoiesis in 97,691 whole genomes. Nature 586, 763–768 (2020)."),[11](https://www.nature.com/articles/s41588-025-02296-x#ref-CR11 "Zink, F. et al. Clonal hematopoiesis, with and without candidate driver mutations, is common in the elderly. Blood 130, 742–752 (2017)."), with subsequent Mendelian randomization (MR) analyses supporting a causal relationship[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[12](https://www.nature.com/articles/s41588-025-02296-x#ref-CR12 "Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. Sci. Adv. 8, eabl6579 (2022)."). Similarly, people with a monogenic long telomere syndrome associated with _POT1_ mutations displayed a very high prevalence of CH[13](https://www.nature.com/articles/s41588-025-02296-x#ref-CR13 "DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. N. Engl. J. Med. 388, 2422–2433 (2023)."). These findings indicate that normal replication-associated telomere shortening can act to curtail HSC clonal expansion, such that the inheritance of longer telomeres or an enhanced ability to maintain telomere length in HSCs favors CH development[14](https://www.nature.com/articles/s41588-025-02296-x#ref-CR14 "Vassiliou, G. Telomere length and clonal hematopoiesis. N. Engl. J. Med. 388, 2481–2484 (2023).").

Other insights came from the study of mutations driving CH in particular contexts; such as in autoimmune aplastic anemia, where CH is commonly driven by mutations in _PIGA_ or the _HLA_ locus, ostensibly because these mutations enable HSCs to evade the immunological attack against them[15](https://www.nature.com/articles/s41588-025-02296-x#ref-CR15 "Yoshizato, T. et al. Somatic mutations and clonal hematopoiesis in aplastic anemia. N. Engl. J. Med. 373, 35–47 (2015)."). Similarly, _TP53_\-mutant and _PPM1D-_mutant CH arises commonly after cytotoxic chemotherapy, to which these mutations confer HSC chemoresistance by attenuating the DDR[16](https://www.nature.com/articles/s41588-025-02296-x#ref-CR16 "Gibson, C. J. et al. Clonal hematopoiesis associated with adverse outcomes after autologous stem-cell transplantation for lymphoma. J. Clin. Oncol. 35, 1598–1605 (2017)."),[17](https://www.nature.com/articles/s41588-025-02296-x#ref-CR17 "Hsu, J. I. et al. PPM1D mutations drive clonal hematopoiesis in response to cytotoxic chemotherapy. Cell Stem Cell 23, 700–713 e706 (2018).").

We reported previously that _SF3B1_ and _SRSF2_ mutations cause CH specifically in the elderly[4](https://www.nature.com/articles/s41588-025-02296-x#ref-CR4 "McKerrell, T. et al. Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis. Cell Rep. 10, 1239–1245 (2015)."), suggesting that advanced age provides a specific context within which these mutations confer a clonal advantage[18](https://www.nature.com/articles/s41588-025-02296-x#ref-CR18 "McKerrell, T. & Vassiliou, G. S. Aging as a driver of leukemogenesis. Sci. Transl. Med. 7, 306fs338 (2015)."). Here we discover that telomere attrition provides such a ‘context’ and becomes an instrument for clonal selection specifically in advanced age. Our findings demonstrate that mutations in splicing factor genes promote clonal expansion by preventing critical telomere shortening in aged HSCs and that _PPM1D_ mutations may do so by reducing DDR signaling from short telomeres. This study exposes age-associated telomere attrition as a conduit between aging and leukemogenesis. Although future studies are still needed to delineate the molecular basis of these observations, this work may propose new therapeutic approaches against CH and hematological or other malignancies.

Results
-------

### Associations between telomere length and subtypes of CH

We noted previously that the age-related prevalence of splicing-factor-mutant CH (SF-CH) differs from that of other CH subtypes[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[18](https://www.nature.com/articles/s41588-025-02296-x#ref-CR18 "McKerrell, T. & Vassiliou, G. S. Aging as a driver of leukemogenesis. Sci. Transl. Med. 7, 306fs338 (2015)."). To corroborate this in a large cohort, we investigated CH prevalence amongst 454,098 UK Biobank (UKB) participants with CH identified as described previously[19](https://www.nature.com/articles/s41588-025-02296-x#ref-CR19 "Cheloor Kovilakam, S. et al. Prevalence and significance of DDX41 gene variants in the general population. Blood 142, 1185–1192 (2023)."). This confirmed that, unlike other types of CH whose prevalence rose steadily with advancing age, SF-CH remained rare until the age of ~55 years and increased sharply in prevalence beyond that (Fig. [1a](https://www.nature.com/articles/s41588-025-02296-x#Fig1)).

**Fig. 1: Genetically determined variation in telomere length and CH subtype.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02296-x/figures/1)

**a**, Age-related prevalence of CH subtype by driver gene (prevalence shown on log scale). Colored lines depict the smoothed model fitted to a generalized additive model with 95% CIs represented by colored shadows. Dagger (†), prevalence is shown in male and female participants respectively for LOY and LOX mCA. **b**, Top, violin plots of genetically determined LTL-PRS among people with different CH (gray) and mCA (purple) subtypes. White dots and black boxes mark the LTL-PRS median and interquartile range (IQR), respectively. Whiskers extend to the lowest and highest datapoints within Q1 − 1.5 × IQR and Q3 + 1.5 × IQR where Q1 and Q3 represent the first and third quartiles, respectively. The control group (pink) includes all participants without any CH mutation or mCA. Bottom, odds ratio (OR) of developing different CH/mCA subtypes per 1 s.d. increase in LTL-PRS. Dots represent the estimated OR and error bars represent the 95% CI, both derived from a logistic regression model. ORs for genes with an FDR-adjusted _P_ < 0.05 are indicated with an asterisk (\*). FDR adjustment was performed using the Benjamini–Hochberg procedure. Number of mutation carriers is indicated in brackets along with the gene names. The _y_ axis is limited to the range \[−4,4\] to allow for better visualization of PRS differences. **c**, Results of MR showing the OR of developing the depicted subtype of CH per 1 s.d. increase in LTL. Dots represent the estimated OR and error bars represent the 95% CI for each OR. ‘Long Telomere CH’ refers to CH mutations that are more common among people with longer genetically predicted telomere length (PRS) in **b**, namely _DNMT3A_, _JAK2_, _TET2_, _GNAS_, _GNB1_ and _CBL_ and ‘Short Telomere CH’ refers to CH mutations that are more common among people with shorter genetically predicted telomere length (PRS), namely _PPM1D_, _SF3B1_ and _SRSF2_.

Noting recent observations that CH driven by splicing factor mutations can arise in young patients with telomere biology disorders (TBD)[20](https://www.nature.com/articles/s41588-025-02296-x#ref-CR20 "Ferrer, A. et al. Patients with telomere biology disorders show context specific somatic mosaic states with high frequency of U2AF1 variants. Am. J. Hematol. 98, E357–E359 (2023)."),[21](https://www.nature.com/articles/s41588-025-02296-x#ref-CR21 "Groarke, E. M. et al. U2AF1 and other splicing factor gene mutations in telomere biology disorders are associated with hematologic neoplasia and worse overall survival. Blood 138, 862 (2021)."),[22](https://www.nature.com/articles/s41588-025-02296-x#ref-CR22 "Gutierrez-Rodrigues, F. et al. Clonal hematopoiesis in telomere biology disorders associates with the underlying germline defect and somatic mutations in POT1, PPM1D, and TERT promoter. Blood 138, 1111 (2021)."),[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."), we posited that telomere maintenance may underlie the unusual age distribution of sporadic SF-CH. To investigate this, we examined the association between previously derived polygenic risk scores (PRS) for leukocyte telomere length (LTL)[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[24](https://www.nature.com/articles/s41588-025-02296-x#ref-CR24 "Codd, V. et al. Polygenic basis and biomedical consequences of telomere length variation. Nat. Genet. 53, 1425–1433 (2021).") and different CH subtypes. This confirmed the reported association of higher LTL-PRS (predictive of longer LTL) with increased prevalence of several common CH subtypes such as those driven by _DNMT3A_, _TET2_ and _JAK2_ mutations, and with CH associated with autosomal mosaic chromosome alterations (mCAs) (Fig. [1b](https://www.nature.com/articles/s41588-025-02296-x#Fig1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). However, CH driven by mutations in _SF3B1_, _SRSF2_ and _PPM1D_ was associated with lower LTL-PRS—an association that was not observed in previous studies grouping these genes with other, more common, CH subtypes. A similar trend was seen with _U2AF1_\-CH, but did not reach statistical significance (false discovery rate (FDR)-adjusted _P_ = 0.15). Consistent with previous observations, CH associated with loss-of-Y (LOY) but not loss-of-X (LOX) chromosome was associated with lower LTL-PRS[25](https://www.nature.com/articles/s41588-025-02296-x#ref-CR25 "Brown, D. W. et al. Shared and distinct genetic etiologies for different types of clonal hematopoiesis. Nat. Commun. 14, 5536 (2023).") (Fig. [1b](https://www.nature.com/articles/s41588-025-02296-x#Fig1)). We also found that high PRS was associated with an increased risk of myeloproliferative neoplasm (MPN), but not myelodysplastic syndrome (MDS) where splicing gene mutations are very common (Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02296-x#Fig6) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)).

MR analyses confirmed the reported associations of longer genetically determined telomere length with CH driven by mutant _DNMT3A_, _TET2_ or _JAK2_ and with autosomal mCAs[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[12](https://www.nature.com/articles/s41588-025-02296-x#ref-CR12 "Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. Sci. Adv. 8, eabl6579 (2022)."). MR analyses also identified causal associations between shorter genetically determined telomere length and CH due to mutations in _PPM1D_, splicing factor genes (_SRSF2_, _SF3B1_ and _U2AF1_) and mosaic LOY (Fig. [1c](https://www.nature.com/articles/s41588-025-02296-x#Fig1) and Supplementary Table [3](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). In search of additional evidence for a causative association, we investigated the impact of another heritable influence on telomere length—paternal age[26](https://www.nature.com/articles/s41588-025-02296-x#ref-CR26 "Codd, V. et al. Measurement and initial characterization of leukocyte telomere length in 474,074 participants in UK Biobank. Nat. Aging 2, 170–179 (2022).")—on CH prevalence amongst 101,340 UKB participants for whom paternal age data are available. Logistic regression analysis was performed after grouping CH genes associated with longer telomere length in one group (‘Long Telomere CH’) and those associated with shorter telomeres in another (‘Short Telomere CH’). ‘Long Telomere CH’ was associated significantly with paternal age, whereas ‘Short Telomere CH’ showed an opposite trend, although this did not reach statistical significance, reflecting the markedly smaller numbers in this group (Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02296-x#Fig7) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)).

To validate our observations regarding LTL-PRS and CH in an external cohort, we applied the UKB-derived LTL-PRS to participants in the All of Us cohort (_n_ = 133,656) and examined its association with each CH subtype (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02296-x#Fig8) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). This revealed significantly higher LTL-PRS values in _DNMT3A_\-CH and _CBL_\-CH, as well as lower LTL-PRS values in _PPM1D_\-CH and _U2AF1_\-CH. However, we did not observe differences in LTL-PRS in _SF3B1_\-CH and _SRSF2_\-CH in the All of Us dataset, potentially owing to the substantially lower numbers of splicing factor-CH (SF-CH) cases in this smaller cohort. Furthermore, we speculated that an LTL-PRS derived using the predominantly European ancestry of the UKB may reflect LTL less accurately in a more genetically diverse cohort, and vice versa. To test this, we re-analyzed the UKB cohort using an alternative LTL-PRS derived from TOPMed[27](https://www.nature.com/articles/s41588-025-02296-x#ref-CR27 "Taub, M. A. et al. Genetic determinants of telomere length from 109,122 ancestrally diverse whole-genome sequences in TOPMed. Cell Genom. 2, 100084 (2022).")—a cohort with a much higher proportion of people with non-European genetic ancestry (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02296-x#Fig9) and Supplementary Table [6](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). This retained the significant associations between LTL-PRS and _DNMT3A_\-CH, _TET2_\-CH, _JAK2_\-CH, _PPM1D_\-CH and _SF3B1_\-CH. Taken together, analyses using the All of Us and TOPMed datasets replicate many of our core observations, taking into account the limitations imposed by the substantially smaller number of SF-CH cases and more diverse genetic ancestries in these cohorts.

### Variation in LTL by CH subtype

Next, we investigated how the associations identified by LTL-PRS and MR analyses relate to measured LTL (derived previously by quantitative PCR (qPCR[26](https://www.nature.com/articles/s41588-025-02296-x#ref-CR26 "Codd, V. et al. Measurement and initial characterization of leukocyte telomere length in 474,074 participants in UK Biobank. Nat. Aging 2, 170–179 (2022)."))) in the UKB. After accounting for relevant covariates, we found marked differences in LTL between participants with CH and controls (Fig. [2a](https://www.nature.com/articles/s41588-025-02296-x#Fig2) and Supplementary Table [7](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). First, as reported before[12](https://www.nature.com/articles/s41588-025-02296-x#ref-CR12 "Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. Sci. Adv. 8, eabl6579 (2022)."), we found that most CH subtypes associated with higher LTL-PRS were not associated with longer measured LTLs and could even display significantly shorter LTL than controls (for example, _JAK2_\-CH). In contrast, despite its association with lower LTL-PRS, _SRSF2_\-CH was not associated with shorter measured LTL (Fig. [2a](https://www.nature.com/articles/s41588-025-02296-x#Fig2)).

**Fig. 2: Variation in measured telomere length by CH subtype and clonal size.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02296-x/figures/2)

**a**, Variation in measured LTL by CH (gray) or mCA (purple) subtype. White dots and black boxes mark the LTL median and IQR, respectively. For more details on boxplot representation, see Fig. [1](https://www.nature.com/articles/s41588-025-02296-x#Fig1). Genes with significantly different LTL (FDR-adjusted _P_ < 0.05, Benjamini–Hochberg procedure) compared to controls (pink) are marked by an asterisk in the label, with _P_ values derived from a linear regression model. The _y_ axis is limited to the range \[−5,5\] for better visualization of LTL differences. **b**, Top, impact of clonal size on the measured LTL across CH/mCA subtypes. Participants in each CH/mCA subgroup were divided into those with small clones (VAF < 0.1 for CH or cell fraction <0.2 for mCA, dark gray/violet) and those with large clones (VAF ≥ 0.1 for CH or cell fraction ≥0.2 for mCA, light gray/violet); data presented as in **a**. Bottom, estimated effects of clone size on LTL. The bars represent the coefficients of CH subtype-specific VAF predictors from a linear regression model of LTL and error bars represent 95% CI. Coefficients that are significantly different from zero (FDR-adjusted _P_ < 0.05, Benjamini–Hochberg procedure) are marked with an asterisk and the corresponding genes are highlighted using colored arrows in the top panel. In **a** and **b**, the number of mutation carriers is shown in brackets below gene names (with separate counts for low and high VAF groups in **b**). For details of boxplot representation, see Fig. [1](https://www.nature.com/articles/s41588-025-02296-x#Fig1). **c**, Trends in the association between LTL-PRS and measured LTL among people with _DNMT3A_\-other-CH, _TET2_\-CH, _SRSF2_\-CH and _PPM1D_\-CH. Both LTL-PRS and measured LTLs were divided into high/low groups using the median value as cutoff, such that participants were categorized into four quadrants based on high/low status of each. The ORs of finding the specific driver gene mutation in each quadrant are depicted using a color scale. Separate ORs for measured LTL and LTL-PRS groups are shown below and to the left of each square, respectively. The analysis was done by categorizing mutations as low (VAF < 0.1) and high VAF (VAF ≥ 0.1). Significant ORs are indicated by asterisks (_P_ value < 0.05, two-sided Fisherʼs exact test). Notable differences in ORs between low and high VAF are indicated by connecting the relevant quadrants.

To better understand these observations, we separated carriers of each CH subtype based on variant allele fraction (VAF) into those with small (VAF < 0.1, ‘low-VAF’) and those with large (VAF ≥ 0.1, ‘high-VAF’) CH clones. This was done to capture the LTL of samples composed primarily of non-CH cells (VAF < 0.1) and contrast this with the LTL of samples with higher proportions of CH cells (VAF ≥ 0.1). This revealed that, for most CH subtypes, including _DNMT3A_\-CH, _TET2_\-CH, _ASXL1_\-CH, _JAK2_\-CH and _CALR_\-CH, low-VAF samples had LTLs close to controls without CH whereas high-VAF samples had shorter LTLs, presumably reflecting telomere attrition during clonal expansion (Fig. [2b](https://www.nature.com/articles/s41588-025-02296-x#Fig2)). By contrast, low-VAF samples from _PPM1D_\-CH, _SF3B1_\-CH and _SRSF2_\-CH, as well as LOY, (that is, subtypes associated with lower LTL-PRS) had significantly shorter LTLs than controls (Fig. [2b](https://www.nature.com/articles/s41588-025-02296-x#Fig2)). High-VAF _PPM1D_\-CH and LOY samples had shorter LTLs than low-VAF ones, although this was not the case for _SF3B1_ and _SRSF2_\-CH. In fact, _SRSF2_\-CH high-VAF samples had significantly longer LTLs than low-VAF samples (Fig. [2b](https://www.nature.com/articles/s41588-025-02296-x#Fig2), top and Supplementary Table [8](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). In some CH subtypes (_CALR_, _DNMT3A_\_R882, _DNMT3A_\_other, _JAK2_ and _SRSF2_) these effects were driven largely by large clones, whereas other CH subtypes (_ASXL1_, _PPM1D_ and _TET2_) showed a more dose-dependent association between VAF and measured LTL (Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02296-x#MOESM1) and Supplementary Table [9](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)).

We next used a linear regression model to examine the relationship between clone size (VAF) and LTL after accounting for factors known to affect telomere length, including age, sex, smoking and genetic ancestry ([Methods](https://www.nature.com/articles/s41588-025-02296-x#Sec9)). This revealed that model coefficients for VAF were negative for several common CH driver genes (including _DNMT3A_, _TET2_, _ASXL1_, _JAK2_ and _PPM1D_), but positive for _SRSF2_\-CH (Fig. [2b](https://www.nature.com/articles/s41588-025-02296-x#Fig2) and Supplementary Table [8](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)).

To examine the interaction between rising VAF, LTL and LTL-PRS more closely, we divided UKB participants with _DNMT3A_\-CH, _TET2_\-CH, _SRSF2_\-CH and _PPM1D_\-CH into upper and lower halves with respect to measured LTL versus LTL-PRS (Fig. [2c](https://www.nature.com/articles/s41588-025-02296-x#Fig2)). This confirmed that _DNMT3A-_CH and _TET2-_CH are more common among those in the upper LTL-PRS group, whereas _SRSF2_\-CH and _PPM1D-_CH are more common amongst those in the lower LTL-PRS group. Also, by contrasting low-VAF versus high-VAF CH, it becomes evident that a rise in VAF is associated with a shift towards lower LTL for _DNMT3A_\-CH, _TET2_\-CH and _PPM1D-_CH, but a higher LTL for _SRSF2_\-CH. We also examined the interaction between LTL-PRS and VAF (LTL ~ LTL-PRS + VAF + LTL-PRS × VAF) directly and found that the interaction terms did not reach statistical significance in any of our gene-specific models (Supplementary Table [10](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). Moreover, inclusion of an interaction term did not noticeably improve model performance. This suggests that LTL-PRS and VAF largely affect measured LTL independently.

### Splicing-factor-mutant cells have longer telomeres

Our above findings reveal that, whereas clonal expansion of most common CH subtypes is associated with LTL shortening, expansion of _SRSF2_\-CH clones may not be, and may even be associated with telomere elongation. This raises the hypothesis that _SRSF2_ mutations, and potentially mutations in other splicing factor genes (higher _SF3B1_ and _U2AF1_ VAFs were not associated with shorter LTL, unlike other common CH drivers), enable cells to maintain or lengthen their telomeres. This would provide mutant HSCs with a relative clonal advantage only when progressive telomere shortening begins to restrain the replicative potential of normal HSCs, hence the predilection for SF-CH to develop in older people with low LTL-PRS.

To examine this hypothesis directly, we constructed the hematopoietic phylogenies of two older people with SF-CH from whole-genome sequencing (WGS) of 186 single hematopoietic stem and progenitor cell (HSPC)-derived colonies using somatic mutations as barcodes[7](https://www.nature.com/articles/s41588-025-02296-x#ref-CR7 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022).") (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02296-x#MOESM1) and Supplementary Table [11](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). In parallel, we used the same WGS data to estimate telomere length for each colony and overlaid these estimates over the phylogenetic trees. This revealed that, in both of these individuals, _SF3B1_\-mutant colonies had significantly longer telomeres than _SF3B1_\-wild type and _U2AF1_\-wild type colonies (mean telomere lengths: 3,671 bp versus 3,129 bp and 3,844 bp versus 3,335 bp; _P_ = 6.82 × 10−5 and _P_ = 4.69 × 10−4 for PD34493 and PD41082, respectively; Fig. [3a–d](https://www.nature.com/articles/s41588-025-02296-x#Fig3)). Furthermore, analysis of hematopoietic phylogenies revealed a nonsignificant trend towards longer telomeres in the sole _U2AF1_\-Q157R mutant clade in our cohort (_P_ = 0.078; Fig. [3b](https://www.nature.com/articles/s41588-025-02296-x#Fig3)), with the difference in telomere lengths closely approaching statistical significance (_P_ = 0.053) when this clade was compared to ancestrally related colonies, despite rapid recent clonal expansion after _U2AF1_\-Q157R acquisition (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02296-x#MOESM1) and Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02296-x#Fig10)). To confirm these findings using a different method, we adapted a protocol for telomere length estimation by qPCR[28](https://www.nature.com/articles/s41588-025-02296-x#ref-CR28 "Cawthon, R. M. Telomere measurement by quantitative PCR. Nucleic Acids Res. 30, e47 (2002).") (Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). For this, we generated new colonies from the same individual (PD34493; [Methods](https://www.nature.com/articles/s41588-025-02296-x#Sec9)), genotyped these for _SF3B1_\-K666N and _U2AF1_\-Q157R, and quantified their telomere length. This confirmed the findings from WGS-derived telomere lengths, with both _SF3B1-_mutant and _U2AF1_\-mutant colonies displaying significantly longer telomeres than wild type colonies (Extended Data Fig. [6](https://www.nature.com/articles/s41588-025-02296-x#Fig11)).

**Fig. 3: Splicing factor mutations and telomere length of single-HSPC-derived colonies.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02296-x/figures/3)

**a**, Ultrametric hematopoietic phylogeny from an 83.8 year old man with splicing-factor-mutant CH. Colonies belonging to _SF3B1_\-mutant clades are colored in green, whilst colonies belonging to the _U2AF1_\-mutant clade are colored in blue. Corresponding WGS-estimated telomere length for each single-HSPC-derived colony is shown on the right panel. **b**, Pairwise comparisons of WGS-estimated telomere length performed between clades using two-sided Wilcoxon rank sum test without adjustment for multiple comparisons. Independent colonies were used as biological replicates within each individual participant and the number of colonies within each group is shown in brackets below their respective label. Center line represents the median telomere length, upper and lower hinges represent the upper and lower quartiles, respectively, and whiskers represent 1.5× the IQR. Individual datapoints corresponding to telomere lengths of single-HSPC-derived colonies have been overlaid on each plot. **c**, Ultrametric hematopoietic phylogeny from an 73.9 year old woman with _SF3B1_\-mutant CH. Colonies belonging to _SF3B1_\-mutant clades are colored in green. Corresponding WGS-estimated telomere length for each single-HSPC-derived colony is shown on the right panel. **d**, Pairwise comparisons of WGS-estimated telomere length performed between clades using two-sided Wilcoxon rank sum test without adjustment for multiple comparisons. Independent colonies were used as biological replicates within each individual participant and the number of colonies within each group is shown in brackets below their respective label. For details of box plot representation, see **b**. **e**, Ultrametric hematopoietic phylogeny from man with _SF3B1_\-mutant CCUS/MDS derived from heterochronous samples obtained at 50.2 (CCUS) and 53.8 years of age (progression to MDS). Colonies belonging to _SF3B1_\-mutant clades are colored in green. Corresponding WGS-estimated telomere length for each single-HSPC-derived colony is shown on the right panel. **f**, Pairwise comparisons of WGS-estimated telomere length performed within the _SF3B1_\-mutant (green) and wild type (red) clades using two-sided Wilcoxon rank sum test without adjustment for multiple comparisons. Independent colonies were used as biological replicates within each individual participant and the number of colonies within each group is shown in brackets below their respective label. For details of box plot representation, see **b**.

[Source data](https://www.nature.com/articles/s41588-025-02296-x#MOESM5)

To understand what happens to the telomere length of splicing-factor-mutant HSPCs over time, we derived hematopoietic phylogenies from a third person (PD48499) using WGS of 62 HSPC-derived colonies generated across two timepoints 3.6 years apart. This did not reveal a difference in telomere lengths between the expanded _SF3B1_\-mutant clade and polyclonal wild-type colonies, but did reveal a trend toward an increase in telomere length in the _SF3B1_\-mutant clade (_P_ = 0.27) but not in _SF3B1_\-wild type cells, during the 3.6-year interval between samples (Fig. [3e,f](https://www.nature.com/articles/s41588-025-02296-x#Fig3)), over which time the clinical phenotype progressed from clonal cytopenia of undetermined significance (CCUS) to MDS. Notably, this person had very short telomere lengths for their age (<1st centile) with respect to LTL reference ranges derived using flow-fluorescence in situ hybridization (FISH) (flow-FISH) or Southern blot, and were similar in length to the two older people in our study (PD34493 and PD41082)[29](https://www.nature.com/articles/s41588-025-02296-x#ref-CR29 "Alter, B. P. et al. Very short telomere length by flow fluorescence in situ hybridization identifies patients with dyskeratosis congenita. Blood 110, 1439–1447 (2007)."),[30](https://www.nature.com/articles/s41588-025-02296-x#ref-CR30 "Steenstrup, T. et al. Telomeres and the natural lifespan limit in humans. Aging (Albany NY) 9, 1130–1142 (2017)."),[31](https://www.nature.com/articles/s41588-025-02296-x#ref-CR31 "Aubert, G., Baerlocher, G. M., Vulto, I., Poon, S. S. & Lansdorp, P. M. Collapse of telomere homeostasis in hematopoietic cells caused by heterozygous mutations in telomerase genes. PLoS Genet. 8, e1002696 (2012).").

Pairwise telomere length comparisons of splicing-factor-mutant colonies with other colonies from the same person are confounded by the fact that many colonies are derived from HSPCs with shared ancestry and are therefore not independent. To account for this, we fitted a linear mixed effects model to test the hypothesis that genotype (splicing driver versus nonsplicing driver versus no driver) is associated with telomere length in hematopoietic colonies, adjusting for confounders of next-generation sequencing (NGS)-estimated telomere length (Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02296-x#MOESM1) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02296-x#MOESM1); [Methods](https://www.nature.com/articles/s41588-025-02296-x#Sec9)). Taking all HSPC colonies into account (_n_ = 248 colonies from three individuals), we found a significant association between colony genotype and telomere length (_P_ = 9.21 × 10−9), with splicing factor mutations showing an increased telomere length (+298 base pairs (bp)), whereas nonsplicing drivers were associated with decreased telomere length (−174 bp) compared to ‘no driver’ colonies (Table [1](https://www.nature.com/articles/s41588-025-02296-x#Tab1)), mirroring our UKB findings (Fig. [2](https://www.nature.com/articles/s41588-025-02296-x#Fig2)).

**Table 1 Association between colony telomere length, driver mutations and age**

### Late-acquired splicing factor mutations and telomere length

A potential limitation of colony-derived telomere length measurements is the requirement for HSPCs to both divide and differentiate to give rise to colonies. The impact of these in vitro processes on telomere length may be altered in the presence of splicing factor mutations, leading to potentially erroneous conclusions about their roles in vivo. To circumvent this, we next adapted and validated a flow-FISH protocol[32](https://www.nature.com/articles/s41588-025-02296-x#ref-CR32 "Baerlocher, G. M., Vulto, I., de Jong, G. & Lansdorp, P. M. Flow cytometry and FISH to measure the average length of telomeres (flow FISH). Nat. Protoc. 1, 2365–2376 (2006)."),[33](https://www.nature.com/articles/s41588-025-02296-x#ref-CR33 "Gutierrez-Rodrigues, F., Santana-Lemos, B. A., Scheucher, P. S., Alves-Paiva, R. M. & Calado, R. T. Direct comparison of flow-FISH and qPCR as diagnostic tests for telomere length measurement in humans. PLoS ONE 9, e113747 (2014).") to estimate the telomere length of individual leukocytes in patients with splicing factor mutations without the need to derive colonies (Extended Data Fig. [7](https://www.nature.com/articles/s41588-025-02296-x#Fig12)). Since the telomere length of the original HSC acquiring a somatic driver mutation is a principal determinant of the telomere length of its clonal progeny, we focused on samples with a subclonal splicing factor mutation acquired by an ancestral clone driven by other driver mutations, to capture the impact of such late-acquired splicing factor mutations on telomere length.

As there is no current methodology to simultaneously genotype and measure telomere length of individual cells, we instead used flow-FISH to sort cells by telomere length (for example, high versus low) and quantified the proportion of mutant cells within each fraction (Fig. [4a](https://www.nature.com/articles/s41588-025-02296-x#Fig4)). We first studied a patient with chronic lymphocytic leukemia (CLL) with a late-acquired _SF3B1_\-K700E mutation (Supplementary Table [12](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). After isolating CLL B cells (CD5+CD19+CD3−), we sorted telomerelow and telomerehigh fractions (Extended Data Fig. [8](https://www.nature.com/articles/s41588-025-02296-x#Fig13)) by percentile telomere length and confirmed telomere length differences by qPCR (Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). Both Sanger sequencing (Fig. [4b](https://www.nature.com/articles/s41588-025-02296-x#Fig4)) and targeted amplicon sequencing (Fig. [4c](https://www.nature.com/articles/s41588-025-02296-x#Fig4)) showed that _SF3B1_\-K700E cells were enriched within the telomerehigh fraction compared to the telomerelow fraction.

**Fig. 4: Impact of splicing factor mutations on telomere length in chronic leukemias.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02296-x/figures/4)

**a**, Peripheral blood samples were collected from patients with clonal blood disorders and sorted by percentile telomere length using flow-FISH. Sorted populations were prepared for sequencing to quantify the enrichment of mutant cells within each fraction. **b**,**c**, Sanger sequencing (**b**) and targeted amplicon sequencing (**c**) of unsorted (left), telomerelow (middle, ≤33rd percentile telomere length) and telomerehigh (right, ≥66th percentile telomere length) populations of MNCs from a patient with CLL with a subclonal _SF3B1_\-K700E mutation. Black arrows indicate mutated base. **d**, Ratio of _SRSF2_\-mutant and _TET2_\-mutant cells in MNCs sorted by percentile telomere length from an individual with AML, with VAF quantified by Sanger sequencing. **e**, Ratio of _SRSF2_\-mutant and _TET2_\-mutant cells in MNCs sorted by percentile telomere length from an individual with CMML, with VAF quantified by Sanger sequencing. **f**, Ratio of _SRSF2_\-mutant and _TET2_\-mutant cells in MNCs sorted by percentile telomere length from an individual with CMML, with VAF quantified by Sanger sequencing. For panels **d**–**f**, ‘unsorted’ populations were derived from the same population as sorted fractions but were gated on live single cells only (that is, not sorted by telomere length). Diagnosis and clinically reported VAF (assessed by clinical NGS performed on bone marrow DNA) is shown above each plot and may differ from the VAF of ‘Unsorted’ cells (MNCs) due to differences in the cellular composition of these fractions. Absence of bars in ‘Telomere<10%’ (**d**–**f**) and ‘Telomere10–33%’ (**f**) indicates that no _SRSF2_\-mutant cells were detected within those fractions. **g**, Summary of data from the three participants shown in **d**–**f**. Points represent the ratio of _SRSF2_\-mutant to _TET2_\-mutant cells in a single person, bar heights represent the mean _SRSF2_/_TET2_ ratio within each telomere length fraction and error bars represent the mean values ± s.e.m. _P_ values were derived by one-way ANOVA followed by Tukey’s multiple comparison test with individual participants treated as biological replicates. Only adjusted _P_ values below the significance threshold (_P_ < 0.05) are shown. **h**, Ratio of _U2AF1_\-mutant and _TET2_\-mutant cells in MNCs sorted by percentile telomere length from a patient with MDS-single lineage dysplasia (SLD) with VAF quantified by Sanger sequencing. Panel **a** created using [BioRender.com](http://biorender.com/).

[Source data](https://www.nature.com/articles/s41588-025-02296-x#MOESM6)

We then analyzed patients with chronic myelomonocytic leukemia (CMML) or acute myeloid leukemia (AML) harboring _TET2_ and _SRSF2_ comutations. Patients were chosen such that _TET2_ mutations were clonal (that is, acquired first) and _SRSF2_ mutations subclonal (that is, acquired second). Unlike CLL, we could not isolate CMML cells by cell surface markers so instead sorted bulk mononuclear cells (MNCs) by telomere length and quantified enrichment for the _TET2_ versus the _SRSF2_ mutations in each fraction. We found that, relative to _TET2_\-mutant, _SRSF2_\-mutant VAFs were lower in the telomerelow and enriched in the telomerehigh fractions by both Sanger sequencing (Fig. [4d–g](https://www.nature.com/articles/s41588-025-02296-x#Fig4)) and NGS (Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). Interestingly, this was also true for a CMML harboring the rare pathogenic mutation _SRSF2_\-V18L[34](https://www.nature.com/articles/s41588-025-02296-x#ref-CR34 "Schwaab, J. et al. Mutational complexity is related to disease severity in systemic mastocytosis. Blood 122, 4094 (2013)."),[35](https://www.nature.com/articles/s41588-025-02296-x#ref-CR35 "Christopeit, M. et al. Clonal evolution after allogeneic hematopoietic stem cell transplantation: the case of myelofibrosis. Biol. Blood Marrow Transpl. 26, e167–e170 (2020).") (Fig. [4f](https://www.nature.com/articles/s41588-025-02296-x#Fig4)). A similar pattern was seen in a single participant with MDS containing _TET2_ and _U2AF1_ mutations (Fig. [4h](https://www.nature.com/articles/s41588-025-02296-x#Fig4)). These findings provide further support to the premise that, in contrast to other CH-associated mutations, splicing factor mutations facilitate clonal expansion while maintaining telomere length. Interestingly, we did not see this difference in a single individual whose CMML had been treated with 31 cycles of Azacitidine treatment by the time of sampling (Extended Data Fig. [9](https://www.nature.com/articles/s41588-025-02296-x#Fig14)), potentially because of selective elimination of faster growing double-mutant cells.

### Somatic TERT promoter mutations drive late-onset CH in UKB

In light of the fact that _PPM1D_\-CH and splicing factor-CH are prevalent amongst both TBD patients and older people with lower LTL-PRS, we hypothesized that somatic mutations in the _TERT_ gene promoter (_TERT_p), which drive the most common form of CH in TBD patients[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."), may also be enriched in people with polygenic short telomeres. To identify putative _TERT_p-CH in the UKB at each of the three known _TERT_p mutational hotspots[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."), we performed pileup analysis of WGS data across the _TERT_ promoter and implemented several filters to reduce erroneous calls (Supplementary Note [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). This identified 148 people with _TERT_p-CH, as well as three people with both _TERTp_ mutations and a prevalent diagnosis of hematological malignancy who were excluded from downstream analyses. Notably, _TERT_p-CH exhibited an age-and sex-related prevalence resembling that of SF-CH (median age, 65 years; 76.4% male), as well as a similar overall prevalence, when SF-CH was identified from WGS using the same pileup analysis pipeline (Supplementary Note [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM1) and Extended Data Fig. [10](https://www.nature.com/articles/s41588-025-02296-x#Fig15)). Notably, people with _TERT_p-CH had significantly lower LTL-PRS values relative to UKB participants without _TERT_p-CH (Supplementary Note [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM1); _P_ = 0.001), mirroring what we observed with _PPM1D_\-CH and SF-CH. These observations further endorse the main premise of our study, that telomere attrition becomes an instrument for clonal selection in aging hematopoiesis/leukemogenesis.

Discussion
----------

The dependence of hematopoiesis on normal telomere function is strikingly evident in TBDs, Mendelian disorders involving telomere maintenance genes, in which abnormal telomere shortening is commonly associated with hematopoietic failure/aplastic anemia[36](https://www.nature.com/articles/s41588-025-02296-x#ref-CR36 "Armanios, M. & Blackburn, E. H. The telomere syndromes. Nat. Rev. Genet. 13, 693–704 (2012)."),[37](https://www.nature.com/articles/s41588-025-02296-x#ref-CR37 "Yamaguchi, H. et al. Mutations in TERT, the gene for telomerase reverse transcriptase, in aplastic anemia. N. Engl. J. Med. 352, 1413–1424 (2005)."). At the other end of the spectrum lies the increased risk of CH in people with polygenic[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[12](https://www.nature.com/articles/s41588-025-02296-x#ref-CR12 "Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. Sci. Adv. 8, eabl6579 (2022).") or monogenic[13](https://www.nature.com/articles/s41588-025-02296-x#ref-CR13 "DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. N. Engl. J. Med. 388, 2422–2433 (2023).") inheritance of longer telomeres. Here we uncover another facet of the close interaction between telomeres and hematopoiesis, by discovering that age-related telomere attrition becomes an instrument for clonal selection, particularly amongst people with lower genetically predicted telomere length. Specifically, we find that mutations in splicing factor genes, _PPM1D_ and _TERT_p seem to rescue HSCs from critical telomere shortening or its consequences, conferring a fitness advantage over their unmutated peers in a manner reminiscent of what has been observed in much younger people with TBD[20](https://www.nature.com/articles/s41588-025-02296-x#ref-CR20 "Ferrer, A. et al. Patients with telomere biology disorders show context specific somatic mosaic states with high frequency of U2AF1 variants. Am. J. Hematol. 98, E357–E359 (2023)."),[21](https://www.nature.com/articles/s41588-025-02296-x#ref-CR21 "Groarke, E. M. et al. U2AF1 and other splicing factor gene mutations in telomere biology disorders are associated with hematologic neoplasia and worse overall survival. Blood 138, 862 (2021)."),[22](https://www.nature.com/articles/s41588-025-02296-x#ref-CR22 "Gutierrez-Rodrigues, F. et al. Clonal hematopoiesis in telomere biology disorders associates with the underlying germline defect and somatic mutations in POT1, PPM1D, and TERT promoter. Blood 138, 1111 (2021)."),[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024).")(Fig. [5a–e](https://www.nature.com/articles/s41588-025-02296-x#Fig5)).

**Fig. 5: Proposed model for the impact of telomere length on the development of different types of CH.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02296-x/figures/5)

Hypothesis explaining how telomere length shapes the global landscape of clonal hematopoiesis throughout life. **a**, HSCs of people inheriting average/near-average length telomeres (through polygenic inheritance of variants associated with long telomeres) that shorten with age. These people have an average risk of developing CH. **b**, People inheriting longer telomeres than average have a greater propensity to develop CH driven by mutations in genes like _DNMT3A_, _TET2_ and _JAK2_, which are associated with significant telomere shortening during clonal expansion. Initiation in an HSC with longer telomeres allows clones to expand for longer before their telomeres become critically short[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."),[12](https://www.nature.com/articles/s41588-025-02296-x#ref-CR12 "Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. Sci. Adv. 8, eabl6579 (2022)."). **c**, People inheriting shorter telomeres than average are more likely to develop critical telomere shortening, triggering DDR signaling and replicative senescence in a large proportion of their HSCs. In this context, there is selection for mutations that enable HSCs to avoid this fate, either by promoting telomere maintenance (for example, splicing factor mutations) or by attenuating DDR signaling (_PPM1D_). **d**, People with monogenic inheritance of variants in genes such as _POT1_, which are associated with sustained long HSC telomeres with advancing age, are at highly increased risk of early-onset CH driven by mutations in genes such as _JAK2_ that normally drive marked telomere shortening (for example, Fig. [2b](https://www.nature.com/articles/s41588-025-02296-x#Fig2))[13](https://www.nature.com/articles/s41588-025-02296-x#ref-CR13 "DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. N. Engl. J. Med. 388, 2422–2433 (2023)."),[14](https://www.nature.com/articles/s41588-025-02296-x#ref-CR14 "Vassiliou, G. Telomere length and clonal hematopoiesis. N. Engl. J. Med. 388, 2481–2484 (2023)."). **e**, People with TBD develop marked telomere shortening in their HSCs (and other tissue stem cells) at a young age. In this setting, the global shortening of telomeres presents a strong selection pressure that favors CH driven by adaptive mutations that ‘reverse’ the monogenic defect (_TERTp_, _POT1_) or maladaptive mutations mirroring those seen in people with polygenic short telomeres (_U2AF1_, _PPM1D_)[20](https://www.nature.com/articles/s41588-025-02296-x#ref-CR20 "Ferrer, A. et al. Patients with telomere biology disorders show context specific somatic mosaic states with high frequency of U2AF1 variants. Am. J. Hematol. 98, E357–E359 (2023)."),[21](https://www.nature.com/articles/s41588-025-02296-x#ref-CR21 "Groarke, E. M. et al. U2AF1 and other splicing factor gene mutations in telomere biology disorders are associated with hematologic neoplasia and worse overall survival. Blood 138, 862 (2021)."),[22](https://www.nature.com/articles/s41588-025-02296-x#ref-CR22 "Gutierrez-Rodrigues, F. et al. Clonal hematopoiesis in telomere biology disorders associates with the underlying germline defect and somatic mutations in POT1, PPM1D, and TERT promoter. Blood 138, 1111 (2021)."),[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."). **f**, Mirroring what is observed in the context of inherited short telomeres leading to a ‘field’ of HSCs with critical telomere shortening (**c** and **e**), clonal expansion-mediated telomere attrition can also become an instrument for clonal selection within previously expanded HSC clones (red border). In this context, mutations that prevent telomere attrition (_SRSF2 or SF3B1_) can restore clonal fitness and facilitate further expansion or leukemic progression of clones previously expanded by driver mutations associated with telomere shortening. Figure created using [BioRender.com](http://biorender.com/).

We reported previously that mutations in splicing factor genes are rare until the seventh decade of life and then rise rapidly in prevalence[4](https://www.nature.com/articles/s41588-025-02296-x#ref-CR4 "McKerrell, T. et al. Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis. Cell Rep. 10, 1239–1245 (2015)."). This mirrors the age-related prevalence of MDS—a disease group where splicing factor mutations are very common[38](https://www.nature.com/articles/s41588-025-02296-x#ref-CR38 "Papaemmanuil, E. et al. Clinical and biological implications of driver mutations in myelodysplastic syndromes. Blood 122, 3616–3627 (2013)."),[39](https://www.nature.com/articles/s41588-025-02296-x#ref-CR39 "Haferlach, T. et al. Landscape of genetic lesions in 944 patients with myelodysplastic syndromes. Leukemia 28, 241–247 (2014)."). Progress in understanding the molecular consequences of these mutations has identified specific target mRNAs responsible for known MDS-related cellular phenotypes such as ring sideroblasts[40](https://www.nature.com/articles/s41588-025-02296-x#ref-CR40 "Dolatshad, H. et al. Cryptic splicing events in the iron transporter ABCB7 and other key target genes in SF3B1-mutant myelodysplastic syndromes. Leukemia 30, 2322–2331 (2016)."), anemia[41](https://www.nature.com/articles/s41588-025-02296-x#ref-CR41 "Mian, S. A. et al. Vitamin B5 and succinyl-CoA improve ineffective erythropoiesis in SF3B1-mutated myelodysplasia. Sci. Transl. Med. 15, eabn5135 (2023).") or aberrant hematopoietic differentiation[42](https://www.nature.com/articles/s41588-025-02296-x#ref-CR42 "Kim, E. et al. SRSF2 mutations contribute to myelodysplasia by mutant-specific effects on exon recognition. Cancer Cell 27, 617–630 (2015)."). However, the mechanisms through which _SF3B1_, _SRSF2_ and _U2AF1_ mutations drive clonal expansion remain unknown[43](https://www.nature.com/articles/s41588-025-02296-x#ref-CR43 "Florez, M. A. et al. Clonal hematopoiesis: mutation-specific adaptation to environmental change. Cell Stem Cell 29, 882–904 (2022)."). The fact that these mutations are mutually exclusive[38](https://www.nature.com/articles/s41588-025-02296-x#ref-CR38 "Papaemmanuil, E. et al. Clinical and biological implications of driver mutations in myelodysplastic syndromes. Blood 122, 3616–3627 (2013)."),[39](https://www.nature.com/articles/s41588-025-02296-x#ref-CR39 "Haferlach, T. et al. Landscape of genetic lesions in 944 patients with myelodysplastic syndromes. Leukemia 28, 241–247 (2014)."),[44](https://www.nature.com/articles/s41588-025-02296-x#ref-CR44 "Shiozawa, Y. et al. Aberrant splicing and defective mRNA production induced by somatic spliceosome mutations in myelodysplasia. Nat. Commun. 9, 3649 (2018)."),[45](https://www.nature.com/articles/s41588-025-02296-x#ref-CR45 "Bamopoulos, S. A. et al. Clinical presentation and differential splicing of SRSF2, U2AF1 and SF3B1 mutations in patients with acute myeloid leukemia. Leukemia 34, 2621–2634 (2020).") suggests that they drive clonal selection through shared mechanisms. However, although mutations in any of these genes lead to mRNA mis-splicing, there is little overlap between mis-spliced[44](https://www.nature.com/articles/s41588-025-02296-x#ref-CR44 "Shiozawa, Y. et al. Aberrant splicing and defective mRNA production induced by somatic spliceosome mutations in myelodysplasia. Nat. Commun. 9, 3649 (2018)."),[46](https://www.nature.com/articles/s41588-025-02296-x#ref-CR46 "Hershberger, C. E. et al. Complex landscape of alternative splicing in myeloid neoplasms. Leukemia 35, 1108–1120 (2021).") or differentially expressed[45](https://www.nature.com/articles/s41588-025-02296-x#ref-CR45 "Bamopoulos, S. A. et al. Clinical presentation and differential splicing of SRSF2, U2AF1 and SF3B1 mutations in patients with acute myeloid leukemia. Leukemia 34, 2621–2634 (2020).") mRNAs associated with each. Our findings propose that prevention of telomere attrition is such a shared mechanism, a premise that could explain: (1) the rapid rise in the prevalence of splicing-mutant CH and MDS in old age and as telomeres become critically short in at-risk people (that is, those with the shortest telomeres) and (2) the higher prevalence of splicing factor gene mutations in HSC-derived myeloid malignancies compared to any other cancer, with blood being the tissue that incurs the most marked age-related telomere shortening[47](https://www.nature.com/articles/s41588-025-02296-x#ref-CR47 "Demanelis, K. et al. Determinants of telomere length across human tissues. Science 369, eaaz6876 (2020).").

_PPM1D_ mutations are known to drive context-dependent CH in patients receiving genotoxic therapies, imparting a relative fitness advantage on mutant HSCs by attenuating DDR[17](https://www.nature.com/articles/s41588-025-02296-x#ref-CR17 "Hsu, J. I. et al. PPM1D mutations drive clonal hematopoiesis in response to cytotoxic chemotherapy. Cell Stem Cell 23, 700–713 e706 (2018)."). However, most cases of _PPM1D_\-CH arise in people without such history, and this was also the case in the UKB. In particular, of 647 cases of _PPM1D_\-CH, only 32 had a documented history of previous chemotherapy, five had radiotherapy and only 98 had a documented history of cancer (excluding nonmelanoma skin cancer). The total number of cases after amalgamating all three categories (chemotherapy + radiotherapy + malignant neoplasm) was 100. This finding indicates that a substantial proportion of cases of _PPM1D_\-CH are not related to genotoxic therapy, but instead arise in the context of replicative senescence linked to telomere attrition, as happens with TBDs[20](https://www.nature.com/articles/s41588-025-02296-x#ref-CR20 "Ferrer, A. et al. Patients with telomere biology disorders show context specific somatic mosaic states with high frequency of U2AF1 variants. Am. J. Hematol. 98, E357–E359 (2023)."),[21](https://www.nature.com/articles/s41588-025-02296-x#ref-CR21 "Groarke, E. M. et al. U2AF1 and other splicing factor gene mutations in telomere biology disorders are associated with hematologic neoplasia and worse overall survival. Blood 138, 862 (2021)."). It is also noteworthy that cisplatin, the chemotherapeutic agent linked most strongly to _PPM1D_\-CH, can directly bind and shorten telomeres[48](https://www.nature.com/articles/s41588-025-02296-x#ref-CR48 "Ishibashi, T. & Lippard, S. J. Telomere loss in cells treated with cisplatin. Proc. Natl Acad. Sci. USA 95, 4219–4223 (1998)."),[49](https://www.nature.com/articles/s41588-025-02296-x#ref-CR49 "Saker, L. et al. Platinum complexes can bind to telomeres by coordination. Int. J. Mol. Sci. 19, 1951 (2018)."). Similarly, it is notable that we did not see an increase in _TP53_\-CH amongst people with lower LTL-PRS, proposing a specific link between _PPM1D_ and replicative senescence that is endorsed by findings of direct physical and functional interactions between _PPM1D_ and the shelterin complex[50](https://www.nature.com/articles/s41588-025-02296-x#ref-CR50 "Storchova, R. et al. Phosphorylation of TRF2 promotes its interaction with TIN2 and regulates DNA damage response at telomeres. Nucleic Acids Res. 51, 1154–1172 (2023).").

Collectively, the above findings reveal that telomere attrition constrains normal hematopoiesis in old age and becomes an instrument for clonal selection by mutations that either prevent telomere attrition (splicing factor genes) or attenuate replicative senescence-related DDR signaling (_PPM1D_). These observations mirror what is seen in TBD[20](https://www.nature.com/articles/s41588-025-02296-x#ref-CR20 "Ferrer, A. et al. Patients with telomere biology disorders show context specific somatic mosaic states with high frequency of U2AF1 variants. Am. J. Hematol. 98, E357–E359 (2023)."),[21](https://www.nature.com/articles/s41588-025-02296-x#ref-CR21 "Groarke, E. M. et al. U2AF1 and other splicing factor gene mutations in telomere biology disorders are associated with hematologic neoplasia and worse overall survival. Blood 138, 862 (2021)."),[22](https://www.nature.com/articles/s41588-025-02296-x#ref-CR22 "Gutierrez-Rodrigues, F. et al. Clonal hematopoiesis in telomere biology disorders associates with the underlying germline defect and somatic mutations in POT1, PPM1D, and TERT promoter. Blood 138, 1111 (2021)."),[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."), but with significant differences. For example, _U2AF1_ is the splicing factor gene most commonly mutated in TBD[20](https://www.nature.com/articles/s41588-025-02296-x#ref-CR20 "Ferrer, A. et al. Patients with telomere biology disorders show context specific somatic mosaic states with high frequency of U2AF1 variants. Am. J. Hematol. 98, E357–E359 (2023)."),[21](https://www.nature.com/articles/s41588-025-02296-x#ref-CR21 "Groarke, E. M. et al. U2AF1 and other splicing factor gene mutations in telomere biology disorders are associated with hematologic neoplasia and worse overall survival. Blood 138, 862 (2021)."),[22](https://www.nature.com/articles/s41588-025-02296-x#ref-CR22 "Gutierrez-Rodrigues, F. et al. Clonal hematopoiesis in telomere biology disorders associates with the underlying germline defect and somatic mutations in POT1, PPM1D, and TERT promoter. Blood 138, 1111 (2021)."),[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."), whereas _SF3B1_ and _SRSF2_ mutations are more common in aging-related telomere attrition[2](https://www.nature.com/articles/s41588-025-02296-x#ref-CR2 "Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)."),[3](https://www.nature.com/articles/s41588-025-02296-x#ref-CR3 "Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)."),[5](https://www.nature.com/articles/s41588-025-02296-x#ref-CR5 "Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. Nat. Genet. 54, 1155–1166 (2022)."). The basis of these differences is unknown, but may relate to the fact that single gene defects underlie the abnormal telomere attrition in TBDs, whilst age-related attrition is multifactorial and polygenic in origin.

Finally, we examined the role of splicing gene mutations as later (noninitiating) events in blood cancer development. Our findings show that in this context too, splicing gene mutations seem to prevent telomere attrition, ostensibly restoring the fitness of a clone whose previous expansion led to substantial telomere shortening (Fig. [5f](https://www.nature.com/articles/s41588-025-02296-x#Fig5)). It is also plausible that splicing gene mutations may act in this same way to restore fitness of apparently ‘driverless’ clones (for example _U2AF1_\-mutant clone in Fig. [3a](https://www.nature.com/articles/s41588-025-02296-x#Fig3)), which become very common after the seventh decade of life[51](https://www.nature.com/articles/s41588-025-02296-x#ref-CR51 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). In line with this is the observation that splicing factor gene mutations (predominantly _SF3B1_) are late events in CLL[52](https://www.nature.com/articles/s41588-025-02296-x#ref-CR52 "Wan, Y. & Wu, C. J. SF3B1 mutations in chronic lymphocytic leukemia. Blood 121, 4627–4634 (2013).") and solid cancers[53](https://www.nature.com/articles/s41588-025-02296-x#ref-CR53 "Luo, J., Chen, C., Liu, Z. & Wang, X. The mutation in splicing factor genes correlates with unfavorable prognosis, genomic instability, anti-tumor immunosuppression and increased immunotherapy response in pan-cancer. Front Cell Dev. Biol. 10, 1045130 (2022)."), which may reflect the fact that telomere attrition is slower in the cells of origin of these malignancies compared to HSCs[47](https://www.nature.com/articles/s41588-025-02296-x#ref-CR47 "Demanelis, K. et al. Determinants of telomere length across human tissues. Science 369, eaaz6876 (2020)."), such that telomere shortening only becomes critical after substantial clonal expansion. Collectively, our findings propose that urgent studies are required to decipher how splicing gene mutations act to prevent telomere attrition as this may lead to new therapeutic approaches targeting splicing factor-mutant myeloid and other cancers.

Methods
-------

### Ethical regulations

This study was conducted under approved UKB application no. 56844. Clinical samples were obtained with informed written consent from the Cambridge Blood and Stem Cell Biobank with approval by the Cambridge East Research Ethics Committee (REC) (REC 18/EE/0199 and 24/EE/0116), from the SardiNIA longitudinal study of immune senescence (REC 15/EE/0327) with approval by the East of England (Essex) REC, or from the Manchester Cancer Research Centre Biobank with approval by the South Manchester REC (REC 07/H1003/161+5; HTA license 30004).

### Statistics and reproducibility

In this project, we included 454,340 UKB participants with somatic variant call data from our previous study[19](https://www.nature.com/articles/s41588-025-02296-x#ref-CR19 "Cheloor Kovilakam, S. et al. Prevalence and significance of DDX41 gene variants in the general population. Blood 142, 1185–1192 (2023)."). From this group, participants who had withdrawn consent, had a mismatch between genetic and self-reported sex or had differences in the dates of attending the assessment center and the blood sample collection, were excluded from the study resulting in _n_ = 454,098 participants. Power calculations were conducted to determine the minimum number of cases for inclusion (‘Mutation Calling’). These analyses were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment.

### Mutation calling

Mutations in 41 CH driver genes (Supplementary Table [13](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)) were called using Mutect2 GATK v.4.1.3.0 from whole-exome sequencing (WES) data of peripheral blood DNA from 454,340 UKB participants and filtered as described previously[19](https://www.nature.com/articles/s41588-025-02296-x#ref-CR19 "Cheloor Kovilakam, S. et al. Prevalence and significance of DDX41 gene variants in the general population. Blood 142, 1185–1192 (2023).") ([Methods](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). A specific VAF cutoff was not used to define participants with CH. Mutations in _DNMT3A_ at the hotspot R882 were grouped as ‘DNMT3A\_R882’ and the rest as ‘DNMT3A\_other.’ _U2AF1_ mutations were identified using Samtools mpileup (v.1.15.1) and the variants with at least three alternate allele reads and a VAF ≥ 0.05 were included in the analysis. Participants who were diagnosed with hematological malignancy before recruitment were removed from all analyses involving LTL. Participants harboring mutations in several genes, or mutations in less frequently mutated genes (<100 cases), were excluded from LTL and LTL-PRS analyses. The threshold of 100 cases was chosen following power calculations performed using the ‘samplesizelogisticcasecontrol’ package (v.2.0.2) in R ([Methods](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). An exception to this threshold was made for mutations in the splicing factor gene _U2AF1_ (_n_ = 82) in light of its recently reported association with CH in TBD[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."). We also excluded _ATM_, _BRCC3_ and _STAT3_ from downstream analysis as these are not widely recognized as drivers of myeloid CH[54](https://www.nature.com/articles/s41588-025-02296-x#ref-CR54 "Niroula, A. et al. Distinction of lymphoid and myeloid clonal hematopoiesis. Nat. Med. 27, 1921–1927 (2021)."). Somatic mutations in the ‘All of Us’ cohort[55](https://www.nature.com/articles/s41588-025-02296-x#ref-CR55 "The All of Us Research Program InvestigatorsThe ‘All of Us’ research program. New Engl. J. Med. 381, 668–676 (2019).") were identified as described previously[56](https://www.nature.com/articles/s41588-025-02296-x#ref-CR56 "Vlasschaert, C. et al. A practical approach to curate clonal hematopoiesis of indeterminate potential in human genetic data sets. Blood 141, 2214–2223 (2023).").

### Mosaic chromosomal alterations

mCA calls were obtained from Loh et al.[57](https://www.nature.com/articles/s41588-025-02296-x#ref-CR57 "Loh, P. R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. Nature 559, 350–355 (2018)."). Before analyses, participants carrying several mCAs or any CH driver gene mutations or mCAs of unknown copy number change/cell fraction were filtered out. Based on the chromosome and the type of copy number change, mCAs were grouped into autosomal mCAs (any type of copy number change), LOX and LOY.

### PRS calculation

We used PRSice-2 (v.2.3.5) to compute PRS associated with telomere length based on the 131 SNPs identified in the GWAS by Codd et al.[24](https://www.nature.com/articles/s41588-025-02296-x#ref-CR24 "Codd, V. et al. Polygenic basis and biomedical consequences of telomere length variation. Nat. Genet. 53, 1425–1433 (2021).") with beta coefficients from the same study serving as weights in the PRS computation. Imputed genotypes available in the UKB were used for this analysis. Calculated PRS were _Z_\-normalized. Participants with a prevalent hematological diagnosis were not excluded for LTL-PRS analyses, as those people would have developed CH at some stage before development of malignancy.

### Myeloid malignancy phenotypes

UKB participants with a prevalent diagnosis of hematological malignancy were defined using ICD codes (Supplementary Table [14](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). If a participant had several myeloid neoplasms, only the first diagnosed disease was considered for analysis. People who had chemotherapy before diagnosing myeloid malignancies were excluded from the association analysis with LTL and LTL-PRS.

### Regression analyses

All linear and logistic regression analyses were performed using the Python (v.3.9.7) module statmodels (v.0.12.2). First, the association between the presence of a CH mutation and LTL was investigated using a linear regression model on LTL with binary predictor variables representing presence/absence (1/0) of mutations in each of the CH driver genes and covariates. For quantifying the variation in telomere length with respect to VAF, a linear regression model for predicting telomere length was built with the variables (Gene + Gene VAF)for all genes where (Gene + Gene VAF)for all genes = DNMT3A + DNMT3A VAF + ASXL1 + ASXL1 VAF + TET2 + TET2 VAF and so on for all genes and covariates. DNMT3A, ASXL1 and so on are variables that represent whether a mutation is present (1) or not (0) in the specific gene. The covariates used were sex, age, smoking status, genetic principal components from one to ten, white blood cell counts and percentages of types of white blood cell. Blood-count-related parameters were winsorized to 99% before regression. Similar analysis was performed for mCAs using cell fraction instead of VAF. Correction for multiple testing was performed using the Benjamini–Hochberg procedure and applying a threshold of FDR < 0.05.

Logistic regression analyses were performed to quantify the association between polygenic risk scores and CH/mCA. Age, sex, smoking status and first ten genetic principal components were used as the covariates in the regression. Correction for multiple testing was performed using the Benjamini–Hochberg procedure and applying a threshold of FDR < 0.05.

### Mendelian randomization

The same set of variants as used in PRS calculation were employed as genetic instruments in the MR analyses to identify causal associations between telomere length and various types of CH. Coefficients quantifying the association between each of the genetic instruments and each of CH types were obtained by Firth’s logistic regression analysis performed using the logistf function in R (v.4.2.1). MR analyses were performed using the TwoSampleMR package (v.0.5.7) in R (v.4.3.0) using these coefficients along with the coefficient estimates for association between genetic instruments and telomere length from Codd et al.[24](https://www.nature.com/articles/s41588-025-02296-x#ref-CR24 "Codd, V. et al. Polygenic basis and biomedical consequences of telomere length variation. Nat. Genet. 53, 1425–1433 (2021).") and the results were reported for the inverse-variance-weighted method. Correction for multiple testing was performed using the Benjamini–Hochberg procedure and applying a threshold of FDR < 0.05.

### Analysis of _TERT_p mutations in the UKB

_TERT_ promoter mutations we identified from WGS of blood DNA from 488,364 UKB participants as this region is not captured adequately by the UKB WES panel. We used samtools mpileup (v.1.15.1) to identify single nucleotide variants (SNVs) across the entire _TERT_ promoter (chr5:129489–1295157) with high sensitivity and then applied several manual filters (depth ≥ 15 bp, at least three supporting reads, VAF ≥ 30%). This approach was used in place of somatic variant calling pipelines due to the low depth of WGS across the promoter (median 34×). We then focused our subsequent analysis on three mutational hotspots identified previously as somatic rescue mutations in TBD (chr5:1295046:T:G, chr5:1295113:G:A and chr5:1295135:G:A)[22](https://www.nature.com/articles/s41588-025-02296-x#ref-CR22 "Gutierrez-Rodrigues, F. et al. Clonal hematopoiesis in telomere biology disorders associates with the underlying germline defect and somatic mutations in POT1, PPM1D, and TERT promoter. Blood 138, 1111 (2021)."),[23](https://www.nature.com/articles/s41588-025-02296-x#ref-CR23 "Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. Blood 144, 2402–2416 (2024)."). To benchmark our approach for calling _TERT_p hotspot mutations from WGS data, we used the same approach to call hotspot mutations in _SF3B1_ (R625, K666 and K700) and _SRSF2_ (P95) from WGS, filtered them as described above, and compared their age-related prevalence to _TERT_p-CH, as well as _SF3B1_/_SRSF2_\-CH identified from WES. A detailed outline of the approach used to call _TERT_p mutations and subsequent benchmarking is contained in Supplementary Note [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM1).

### Construction of phylogenetic trees from WGS of hematopoietic cell colonies

We analyzed data from a man aged 83.8 years with SF-CH detected in blood DNA (PD34493: _U2AF1_\-Q157R 10.3%, _SF3B1_\-K666N 8.7%, _NOTCH1_\-L441L 0.3%), studied previously by phylogenetic analysis using WGS of single-HSPC-derived colonies[7](https://www.nature.com/articles/s41588-025-02296-x#ref-CR7 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."). Specifically, for this study, we also performed colony WGS and phylogenetic analyses on samples from a woman aged 73.9 years with SF-CH (PD41082: _TET2_\-Q1825X 33.8%, _SF3B1_\-K666N 7.1%, _TET2_\-S315fs 3.2%, _GNB1_\-K57E 1.5%, _TET2_\-L1322Q 1.3%, _TET2_\-H435fs 1.2%, _TET2_\-Q1274R 1.1%, _TET2_\-Q1542X 0.8%). Both were participants in the SardiNIA study[58](https://www.nature.com/articles/s41588-025-02296-x#ref-CR58 "Orrù, V. et al. Genetic variants regulating immune cell levels in health and disease. Cell 155, 242–256 (2013).") and were studied because they harbored SF-CH with sizeable clones[7](https://www.nature.com/articles/s41588-025-02296-x#ref-CR7 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."). Ninety-six colonies per person were picked from methylcellulose-based medium previously plated with peripheral blood mononuclear cells (PBMCs) and used for WGS as described previously[7](https://www.nature.com/articles/s41588-025-02296-x#ref-CR7 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[51](https://www.nature.com/articles/s41588-025-02296-x#ref-CR51 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). To investigate trends in clonal expansion and telomere length over time, heterochronous peripheral blood samples were taken from a man with _SF3B1_\-CCUS (PD48499) aged 50.2 years (_n_ = 24 colonies, _SF3B1_\-K700E 42.4% on clinical NGS of bone marrow DNA) and _SF3B1_\-MDS at age of 53.8 years (_n_ = 72 colonies, _SF3B1_\-K700E 42.8% on clinical NGS of bone marrow DNA). This man was selected because of the presence of SF-CH and availability of longitudinal blood samples.

Phylogenetic relationships were derived from colony WGS data as described previously[7](https://www.nature.com/articles/s41588-025-02296-x#ref-CR7 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[59](https://www.nature.com/articles/s41588-025-02296-x#ref-CR59 "Williams, N. et al. Life histories of myeloproliferative neoplasms inferred from phylogenies. Nature 602, 162–168 (2022)."),[60](https://www.nature.com/articles/s41588-025-02296-x#ref-CR60 "Spencer Chapman, M. et al. Lineage tracing of human development through somatic mutations. Nature 595, 85–90 (2021)."). Briefly, reads were aligned to the human reference genome (GRCh38) using BWA-MEM ([https://github.com/lh3/bwa](https://github.com/lh3/bwa)). Variant calling was performed using CaVEMAN[61](https://www.nature.com/articles/s41588-025-02296-x#ref-CR61 "Jones, D. et al. cgpCaVEManWrapper: simple execution of CaVEMan in order to detect somatic single nucleotide variants in NGS data. Curr. Protoc. Bioinformatics 56, 15.10.11–15.10.18 (2016).") (SNV) and Pindel[62](https://www.nature.com/articles/s41588-025-02296-x#ref-CR62 "Ye, K., Schulz, M. H., Long, Q., Apweiler, R. & Ning, Z. Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Bioinformatics 25, 2865–2871 (2009).") (indels) against an in silico generated unmatched normal. Colonies with low sequencing depth (<6×) or low clonality (median VAF < 0.4) were removed from downstream analyses. Filtering was performed to remove germline variants and artefacts arising from low DNA input, using pooled information across per-person colonies as outlined previously[59](https://www.nature.com/articles/s41588-025-02296-x#ref-CR59 "Williams, N. et al. Life histories of myeloproliferative neoplasms inferred from phylogenies. Nature 602, 162–168 (2022)."),[60](https://www.nature.com/articles/s41588-025-02296-x#ref-CR60 "Spencer Chapman, M. et al. Lineage tracing of human development through somatic mutations. Nature 595, 85–90 (2021)."). For all mutations passing quality filters in at least one colony, matrices were generated of mutant and normal reads at each site for every colony from the same person, using vafCorrect ([https://github.com/cancerit/vafCorrect](https://github.com/cancerit/vafCorrect)) to correct for reference bias arising during alignment of reads containing indels. Genotype matrices of SNVs were used as input to MPBoot[63](https://www.nature.com/articles/s41588-025-02296-x#ref-CR63 "Hoang, D. T. et al. MPBoot: fast phylogenetic maximum parsimony tree inference and bootstrap approximation. BMC Evol. Biol. 18, 11 (2018).") to infer the phylogenetic relationships between colonies using a maximum parsimony approach with bootstrap approximation. The treeMut package ([https://github.com/nangalialab/treemut](https://github.com/nangalialab/treemut)) was then used to assign mutations (SNVs and indels) to branches and estimate branch lengths. To convert the _x_ axis of each phylogenetic tree from number of mutations to chronological age, where the tips of the tree are the age of the person at sampling, we used the package Rtreefit[59](https://www.nature.com/articles/s41588-025-02296-x#ref-CR59 "Williams, N. et al. Life histories of myeloproliferative neoplasms inferred from phylogenies. Nature 602, 162–168 (2022).") ([https://github.com/nangalialab/rtreefit](https://github.com/nangalialab/rtreefit)) to scale branch lengths, accounting for differences in mutation rate across the human lifespan and intersample variation in the sensitivity of detecting somatic variants.

### Telomere length estimation from WGS data

Telomere length estimates were estimated from the NovaSeq-sequenced colony WGS data described above using Telomerecat[64](https://www.nature.com/articles/s41588-025-02296-x#ref-CR64 "Farmery, J. H. R., Smith, M. L., Diseases, N. B.-R. & Lynch, A. G. Telomerecat: a ploidy-agnostic method for estimating telomere length from whole genome sequencing data. Sci. Rep. 8, 1300 (2018)."). Novaseq’s two-dye technology interprets the absence of signal from a failed cluster as a run of ‘G’ base calls that can confound Telomerecat due to its resemblance to the telomere sequence (TTAGGG). The likelihood of cluster failure increases with read length; hence, we ran Telomerecat with the ‘-trim 75’ argument to estimate telomere lengths from the first 75 bp of each read and avoid the higher error regions towards the end of the read. Phylogenetic trees were then annotated with telomere length estimates using the ggtree (v.3.8.2) package in R[65](https://www.nature.com/articles/s41588-025-02296-x#ref-CR65 "Yu, G., Smith, D. K., Zhu, H., Guan, Y. & Lam, T. T.-Y. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods Ecol. Evol. 8, 28–36 (2017).").

Pairwise comparison of telomere length estimates were performed using the Wilcoxon rank sum test. Alongside this, we also fitted a linear mixed effects model using the lme4 package (v.1.1) in R[66](https://www.nature.com/articles/s41588-025-02296-x#ref-CR66 "Bates, D., Mächler, M., Bolker, B. & Walker, S. Fitting linear mixed-effects models using lme4. J. Stat. Softw. 67, 1–48 (2015).") to model colony telomere length with sequencing batch as a random effect and genotype and age as fixed effects (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)):

${\\rm{Colony}\\; telomere\\; length} \\sim {\\rm{Age}}+{\\rm{Genotype}}+(1{|\\rm{Batch}})$

This model was fitted on all colonies passing filters and included in the final phylogenetic trees (_n_ = 248). To test the hypothesis that genotype (splicing factor driver mutation/other driver mutation/driverless) is associated with colony telomere length at a cohort level, we compared linear mixed effects models with and without genotype as a fixed effect and compared both models using one-way analysis of variance (ANOVA). Confidence intervals (CIs) for fixed effect coefficients were estimated using bootstrap resampling with 10,000 resamples and calculating the 95% CI for each coefficient based on the first 5,000 converged models.

### Cell-line culture

K562 were cultured in IMDM (Gibco, cat no. 12440053) supplemented with 10% FBS (Gibco, catalogue number SH30071.03), 2 mM l\-glutamine and 1% penicillin/streptomycin. OCI-AML2 were cultured in α-MEM (Gibco, catalogue number 12571063) supplemented with 20% FBS, 2 mM l\-glutamine and 1% penicillin/streptomycin. HEK293FT were cultured in DMEM (Gibco, catalogue number 11960085) supplemented with 10% FBS, 2 mM l\-glutamine and 1% penicillin/streptomycin and passaged using trypsin. Cells were maintained at 37 °C and 5% CO2 in a humidified incubator and passaged every 2–3 days. Cas9-expressing cell lines were generated using lentivirus generated from pKLV2-EF1aBsd2ACas9-W plasmid (Addgene, catalogue number 67978) as described below.

### Lentivirus generation and transduction

Tissue culture plates (15 cm2) were coated in 0.1% gelatin for 37 °C for 30 min. Plates were washed with PBS (Sigma, catalogue number D8537-500) and seeded with 8 × 106 HEK293FT cells. Vector plasmid (7.5 μg) was mixed with 18.5 μg of pPAX2 (Addgene, catalogue number 12260), 4 μg of pMD2.G (Addgene, catalogue number 12259), 30 μl of PLUS reagent and 7.5 ml of Opti-MEM (Gibco, catalogue number 51985026) and incubated at room temperature for 5 min. Lipofectamine LTX (180 μl; Invitrogen, cat no. 15338030) was added, and the mixture was incubated at room temperature for an additional 30 min. After this, the transfection mixture was added dropwise to cells followed by 20 ml of HEK293FT medium (prepared as above) and placed in a humidified incubator overnight. Medium was changed the following morning. On day 2, viral supernatant was filtered with 0.45 μM low-protein binding filter (Nalgene, catalogue number 190-2545), mixed with Lenti-X (Takara Bio, catalogue number 631232) and kept at 4 °C overnight. Viral supernatant was then spun at 1,500_g_ for 45 mins at 4 °C and the pellet was resuspended in 300 μl of ice-cold PBS.

Concentrated virus (15 μl) was added to 1 × 105 cells in 1 ml of medium supplemented with 6.7 μg ml−1 polybrene. Cells were centrifuged at 870_g_ and 37 °C for 1 h and returned to the incubator. Following 2 days in culture, transduced cells were selected by supplementing medium with 10 μg ml−1 blasticidin or 1 μg ml−1 puromycin for 5 days.

### _TERT_ knockout and validation

Two gRNAs targeting _TERT_ exon 2 (Supplementary Table [15](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)) were cloned into the pKLV2-U6gRNA5(BbsI)-PGKpuro2ABFP-W vector (Addgene, catalogue number 67974) and lentivirus was generated and transduced as described above. Transduced cells were selected using 1 μg ml−1 puromycin and maintained in culture for a total of 14 days. Cells (1 × 106) cells were transferred to a 1.5 ml tube and centrifuged at 300_g_ for 5 min and supernatant was discarded. Genomic DNA was extracted from the pellet using the DNeasy Blood and Tissue Kit (Qiagen, catalogue number 69504). DNA was quantified and diluted in UltraPure DNase/RNase-Free Distilled Water (Invitrogen, catalogue number 11538646). _TERT_ gRNA activity was validated using PCR with primers spanning the region of interest (Supplementary Table [15](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)) followed by Sanger sequencing.

PCR was performed on 1 ng of diluted gDNA using HiFi HotStart ReadyMix (Kapa, catalogue number 07958927001) and primers spanning the _TERT_ region of interest using the following reaction conditions: 95 °C for 3 min, 35 cycles of (98 °C for 20 s, 60 °C for 15 s, 72 °C for 30 s) and 72 °C for 5 min. PCR product was purified using QIA quick PCR Purification Kit (Qiagen, catalogue number 28104) and submitted for Sanger sequencing with the forward primer using GeneWiz. Sequencing traces were analyzed in SnapGene to confirm _TERT_ gRNA activity.

### Clinical samples

Peripheral blood was collected into lithium heparin tubes (Sarstedt, catalogue number 02.1065.001) and bone marrow aspirate was collected in RPMI (Gibco, catalogue number 21875034) supplemented with 1% penicillin/streptomycin and 10 IU ml−1 sodium heparin (Merck, catalogue number H3149-10KU). Samples were processed using Ficoll (Merck, catalogue number GE17-1440-02) and/or PharmLyse (catalogue number BD 555899) to isolate MNCs, leukocytes or granulocytes. Cells were used immediately in experiments or cryopreserved in FBS supplemented with 50% human serum albumin and 10% dimethylsulfoxide and stored for future use.

### Colony-derived WGS

Samples were plated to form colonies and prepared for WGS as described previously[7](https://www.nature.com/articles/s41588-025-02296-x#ref-CR7 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022)."),[51](https://www.nature.com/articles/s41588-025-02296-x#ref-CR51 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). Briefly, peripheral blood or bone marrow MNCs were plated at 3 × 106cells ml−1 in MethoCult H4034 (Stemcell Technologies, catalogue number 04034) and cultured in a humidified incubator at 37 °C and 5% CO2 for 14 days. Colonies were picked and resuspended in RLT (Qiagen, catalogue number 79216). Libraries were prepared using a low-input pipeline and 150 bp paired-end sequencing was performed on a NovaSeq 6000 at 15× coverage.

### DNA extraction and quantification

For cell lines, genomic DNA was isolated using DNeasy Blood and Tissue Kit and quantified using the Qubit dsDNA HS Kit (Invitrogen, catalogue number Q32851). Telomere qPCR ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)) on colonies lysed in RLT was attempted but yielded poor and inconsistent results, particularly at higher RLT concentrations and low DNA input (Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). Instead, cells were plated as described above and picked after 14 days into 17 μl of PicoPure (Applied Biosystems, catalogue number KIT0103) buffer supplemented with Proteinase K according to the manufacturer’s instructions, lysing the cells. Lysate was placed in a thermocycler under the following conditions: 65 °C for 6 h, 75 °C for 30 min, 4 °C hold. Volume was made up to 50 μl with UltraPure H2O and DNA was quantified using the Quant-iT PicoGreen dsDNA Assay Kit (Invitrogen, catalogue number P7589).

### Flow-FISH

Cryopreserved cells were thawed and washed twice in warmed RPMI supplemented with 10% FBS. Cells were centrifuged at 300_g_ for 5 min and resuspended in FACS buffer (PBS supplemented with 0.1% BSA (Fisher, catalogue number BP9702-100)). Cells were counted and 1–3 × 106 cells were aliquoted into 1.5 ml tubes. Cells were centrifuged at 300_g_ for 5 min and resuspended in 1 ml PBS containing 1:1,000 Fixable Viability Dye eFluor 780 (eBioscience, catalogue number 65-0865-14) and incubated at 4 °C in the dark for 20 min. Following this, cells were washed twice in FACS buffer. For the CLL sample only, cells were then centrifuged at 300_g_ for 5 min and resuspended in FACS buffer supplemented with the following antibodies: 1:100 CD3-BUV395, 1:160 CD19-BV421, 1:160 CD11b-PE, 1:100 CD33-BV510, 1:50 CD5-FITC (Supplementary Table [16](https://www.nature.com/articles/s41588-025-02296-x#MOESM4)). Cells were incubated at 4 °C in the dark for 20 min and washed twice with FACS buffer and sorted as described below.

Following sorting (CLL sample) or viability staining (remaining samples), cells were centrifuged at 300_g_ for 5 min at resuspended in 250 μl of hybridization buffer (70% formamide (Thermo Scientific, catalogue number 17899), 20 mM Tris (Thermo Scientific, catalogue number AM9850G) and 0.1% BSA in water) containing 0.3 μg ml−1 TelC-Alexa647 (PNA Bio F1013) and 0.3 μg ml−1 CENPB-Alexa488 (PNA Bio, catalogue number F3004) PNA probes which had been heated briefly at 55 °C for 5 min and vortexed before addition. Cells were heated at 80 °C for 10 min and incubated overnight at room temperature in the dark.

The following morning, cells were centrifuged at 300_g_ for 7 min at 16 °C and resuspended gently in 1 ml of formamide wash buffer (70% formamide, 10 mM Tris, 0.1% Tween 20 (Sigma, catalogue number P1379) and 0.1% BSA in water). This step was repeated once more. After this, cells were centrifuged at 300_g_ for 7 min at 16 °C and resuspended gently in 1 ml of PBS wash buffer (PBS supplemented with 0.1% Tween 20 and 0.1% BSA). Finally, cells were centrifuged at 300_g_ for 5 min at 16 °C, resuspended in 500 ml of FACS buffer supplemented with 10 mg ml−1 RNase A (Invitrogen, catalogue number 12091021) and transferred to FACS tubes through a 40-μm cell strainer (Fisher, catalogue number 22363547). Cells were sorted using a BD FACSAria Fusion flow cytometer. For each sample, a small proportion of cells were analyzed to give the distribution of telomere lengths within that sample and then sorting gates were set by the specified percentile telomere length ranges.

DNA was extracted from sorted populations using PicoPure and quantified as described above. Purified DNA was prepared for Sanger sequencing (as described above, PCR annealing temperature optimized for each primer pair) alongside targeted amplicon sequencing (Supplementary Table [17](https://www.nature.com/articles/s41588-025-02296-x#MOESM4); [Supplementary Methods](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)).

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02296-x#MOESM2) linked to this article.

Data availability
-----------------

UKB data is publicly available, and access can be requested at [https://www.ukbiobank.ac.uk](https://www.ukbiobank.ac.uk/). All of Us data can be accessed at [https://researchallofus.org/](https://researchallofus.org/). WGS data used to generate hematopoietic phylogenies is available at the European Genome–phenome Archive (EGA) with accession [EGAS00001004280](https://ega-archive.org/studies/EGAS00001004280) ([https://ega-archive.org/studies/EGAS00001004280](https://ega-archive.org/studies/EGAS00001004280)). Targeted amplicon sequencing data has been uploaded to SRA ([PRJNA1121075](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1121075)). Additional data is included as Supplementary Tables. [Source data](https://www.nature.com/articles/s41588-025-02296-x#Sec32) are provided with this paper.

Code availability
-----------------

No custom software, tools or packages were used in the UKB analysis. The code used to generate phylogenies from single-HSPC-derived colonies is available on GitHub: SNVs were called using the cancer variants through expectation maximization (CaVEMan) algorithm ([https://github.com/cancerit/CaVEMan](https://github.com/cancerit/CaVEMan)), indels were called using Pindel ([https://github.com/cancerit/cgpPindel](https://github.com/cancerit/cgpPindel)) and allele counts at SNV/indel sites were calculated using vafCorrect ([https://github.com/cancerit/vafCorrect](https://github.com/cancerit/vafCorrect)). Rtreemut and rtreefit, used to assign mutations to branches and infer temporal branch lengths/mutation rates, respectively, are available from [https://github.com/nangalialab/treemut](https://github.com/nangalialab/treemut) and [https://github.com/nangalialab/rtreefit](https://github.com/nangalialab/rtreefit). Telomerecat, used to estimate colony telomere lengths from WGS, can be accessed from: ([https://github.com/cancerit/telomerecat](https://github.com/cancerit/telomerecat)). The pairwise comparisons of splicing mutant clades, construction of linear mixed effects models and analysis of TERTp mutations from UKB WGS data were carried out using custom R scripts available at: [https://github.com/billydunn/telomeres-ch](https://github.com/billydunn/telomeres-ch) (ref. [67](https://www.nature.com/articles/s41588-025-02296-x#ref-CR67 "Dunn, W. billydunn/telomeres-ch: v.1.0.0. Zenodo 
https://doi.org/10.5281/zenodo.15472283
(2025).")). Python code used to perform the primary LTL-CH and PRS-CH association analyses in the UKB and the R script for MR analysis are available at [https://github.com/cksruthi/telomeres-CH](https://github.com/cksruthi/telomeres-CH) (ref. [68](https://www.nature.com/articles/s41588-025-02296-x#ref-CR68 "cksruthi. cksruthi/telomeres-CH: v.1.0. Zenodo 
https://doi.org/10.5281/zenodo.15499165
(2025).")). Scripts used for the analysis of targeted sequencing data can also be found at [https://github.com/cksruthi/telomeres-CH](https://github.com/cksruthi/telomeres-CH).

References
----------

1.  Martincorena, I. & Campbell, P. J. Somatic mutation in cancer and normal cells. _Science_ **349**, 1483–1489 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFCrtbnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26404825)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutation%20in%20cancer%20and%20normal%20cells&journal=Science&volume=349&pages=1483-1489&publication_year=2015&author=Martincorena%2CI&author=Campbell%2CPJ) 
    
2.  Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. _N. Engl. J. Med._ **371**, 2477–2487 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25426838)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290021)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20and%20blood-cancer%20risk%20inferred%20from%20blood%20DNA%20sequence&journal=N.%20Engl.%20J.%20Med.&volume=371&pages=2477-2487&publication_year=2014&author=Genovese%2CG) 
    
3.  Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. _N. Engl. J. Med._ **371**, 2488–2498 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25426837)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4306669)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Age-related%20clonal%20hematopoiesis%20associated%20with%20adverse%20outcomes&journal=N.%20Engl.%20J.%20Med.&volume=371&pages=2488-2498&publication_year=2014&author=Jaiswal%2CS) 
    
4.  McKerrell, T. et al. Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis. _Cell Rep._ **10**, 1239–1245 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjs1Omt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25732814)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542313)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Leukemia-associated%20somatic%20mutations%20drive%20distinct%20patterns%20of%20age-related%20clonal%20hemopoiesis&journal=Cell%20Rep.&volume=10&pages=1239-1245&publication_year=2015&author=McKerrell%2CT) 
    
5.  Kar, S. P. et al. Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis. _Nat. Genet._ **54**, 1155–1166 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVaqsrzP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35835912)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9355874)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20analyses%20of%20200%2C453%20individuals%20yield%20new%20insights%20into%20the%20causes%20and%20consequences%20of%20clonal%20hematopoiesis&journal=Nat.%20Genet.&volume=54&pages=1155-1166&publication_year=2022&author=Kar%2CSP) 
    
6.  Gu, M. et al. Multiparameter prediction of myeloid neoplasia risk. _Nat. Genet._ **55**, 1523–1530 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslKnt7%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37620601)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10484784)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiparameter%20prediction%20of%20myeloid%20neoplasia%20risk&journal=Nat.%20Genet.&volume=55&pages=1523-1530&publication_year=2023&author=Gu%2CM) 
    
7.  Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. _Nature_ **606**, 335–342 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVant7jN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35650444)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9177423)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20longitudinal%20dynamics%20and%20natural%20history%20of%20clonal%20haematopoiesis&journal=Nature&volume=606&pages=335-342&publication_year=2022&author=Fabre%2CMA) 
    
8.  Robertson, N. A. et al. Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects. _Nat. Med._ **28**, 1439–1446 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhslSlu7fK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35788175)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9307482)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Longitudinal%20dynamics%20of%20clonal%20hematopoiesis%20identifies%20gene-specific%20fitness%20effects&journal=Nat.%20Med.&volume=28&pages=1439-1446&publication_year=2022&author=Robertson%2CNA) 
    
9.  Bick, A. G. et al. Inherited causes of clonal haematopoiesis in 97,691 whole genomes. _Nature_ **586**, 763–768 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVyiu7nK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33057201)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7944936)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inherited%20causes%20of%20clonal%20haematopoiesis%20in%2097%2C691%20whole%20genomes&journal=Nature&volume=586&pages=763-768&publication_year=2020&author=Bick%2CAG) 
    
10.  Kessler, M. D. et al. Common and rare variant associations with clonal haematopoiesis phenotypes. _Nature_ **612**, 301–309 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVWhtLzN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36450978)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9713173)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Common%20and%20rare%20variant%20associations%20with%20clonal%20haematopoiesis%20phenotypes&journal=Nature&volume=612&pages=301-309&publication_year=2022&author=Kessler%2CMD) 
    
11.  Zink, F. et al. Clonal hematopoiesis, with and without candidate driver mutations, is common in the elderly. _Blood_ **130**, 742–752 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVSku73I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28483762)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5553576)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%2C%20with%20and%20without%20candidate%20driver%20mutations%2C%20is%20common%20in%20the%20elderly&journal=Blood&volume=130&pages=742-752&publication_year=2017&author=Zink%2CF) 
    
12.  Nakao, T. et al. Mendelian randomization supports bidirectional causality between telomere length and clonal hematopoiesis of indeterminate potential. _Sci. Adv._ **8**, eabl6579 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1GhtL3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35385311)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8986098)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mendelian%20randomization%20supports%20bidirectional%20causality%20between%20telomere%20length%20and%20clonal%20hematopoiesis%20of%20indeterminate%20potential&journal=Sci.%20Adv.&volume=8&publication_year=2022&author=Nakao%2CT) 
    
13.  DeBoy, E. A. et al. Familial clonal hematopoiesis in a long telomere syndrome. _N. Engl. J. Med._ **388**, 2422–2433 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1Kgtr7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37140166)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10501156)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Familial%20clonal%20hematopoiesis%20in%20a%20long%20telomere%20syndrome&journal=N.%20Engl.%20J.%20Med.&volume=388&pages=2422-2433&publication_year=2023&author=DeBoy%2CEA) 
    
14.  Vassiliou, G. Telomere length and clonal hematopoiesis. _N. Engl. J. Med._ **388**, 2481–2484 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37140164)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomere%20length%20and%20clonal%20hematopoiesis&journal=N.%20Engl.%20J.%20Med.&volume=388&pages=2481-2484&publication_year=2023&author=Vassiliou%2CG) 
    
15.  Yoshizato, T. et al. Somatic mutations and clonal hematopoiesis in aplastic anemia. _N. Engl. J. Med._ **373**, 35–47 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht12hs7bP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26132940)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7478337)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutations%20and%20clonal%20hematopoiesis%20in%20aplastic%20anemia&journal=N.%20Engl.%20J.%20Med.&volume=373&pages=35-47&publication_year=2015&author=Yoshizato%2CT) 
    
16.  Gibson, C. J. et al. Clonal hematopoiesis associated with adverse outcomes after autologous stem-cell transplantation for lymphoma. _J. Clin. Oncol._ **35**, 1598–1605 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVWlsL7F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28068180)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5455707)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20associated%20with%20adverse%20outcomes%20after%20autologous%20stem-cell%20transplantation%20for%20lymphoma&journal=J.%20Clin.%20Oncol.&volume=35&pages=1598-1605&publication_year=2017&author=Gibson%2CCJ) 
    
17.  Hsu, J. I. et al. PPM1D mutations drive clonal hematopoiesis in response to cytotoxic chemotherapy. _Cell Stem Cell_ **23**, 700–713 e706 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVyhur%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30388424)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6224657)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PPM1D%20mutations%20drive%20clonal%20hematopoiesis%20in%20response%20to%20cytotoxic%20chemotherapy&journal=Cell%20Stem%20Cell&volume=23&pages=700-713%20e706&publication_year=2018&author=Hsu%2CJI) 
    
18.  McKerrell, T. & Vassiliou, G. S. Aging as a driver of leukemogenesis. _Sci. Transl. Med._ **7**, 306fs338 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aging%20as%20a%20driver%20of%20leukemogenesis&journal=Sci.%20Transl.%20Med.&volume=7&publication_year=2015&author=McKerrell%2CT&author=Vassiliou%2CGS) 
    
19.  Cheloor Kovilakam, S. et al. Prevalence and significance of DDX41 gene variants in the general population. _Blood_ **142**, 1185–1192 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVOrt73J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37506341)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prevalence%20and%20significance%20of%20DDX41%20gene%20variants%20in%20the%20general%20population&journal=Blood&volume=142&pages=1185-1192&publication_year=2023&author=Cheloor%20Kovilakam%2CS) 
    
20.  Ferrer, A. et al. Patients with telomere biology disorders show context specific somatic mosaic states with high frequency of U2AF1 variants. _Am. J. Hematol._ **98**, E357–E359 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37665761)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Patients%20with%20telomere%20biology%20disorders%20show%20context%20specific%20somatic%20mosaic%20states%20with%20high%20frequency%20of%20U2AF1%20variants&journal=Am.%20J.%20Hematol.&volume=98&pages=E357-E359&publication_year=2023&author=Ferrer%2CA) 
    
21.  Groarke, E. M. et al. U2AF1 and other splicing factor gene mutations in telomere biology disorders are associated with hematologic neoplasia and worse overall survival. _Blood_ **138**, 862 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=U2AF1%20and%20other%20splicing%20factor%20gene%20mutations%20in%20telomere%20biology%20disorders%20are%20associated%20with%20hematologic%20neoplasia%20and%20worse%20overall%20survival&journal=Blood&volume=138&publication_year=2021&author=Groarke%2CEM) 
    
22.  Gutierrez-Rodrigues, F. et al. Clonal hematopoiesis in telomere biology disorders associates with the underlying germline defect and somatic mutations in POT1, PPM1D, and TERT promoter. _Blood_ **138**, 1111 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20in%20telomere%20biology%20disorders%20associates%20with%20the%20underlying%20germline%20defect%20and%20somatic%20mutations%20in%20POT1%2C%20PPM1D%2C%20and%20TERT%20promoter&journal=Blood&volume=138&publication_year=2021&author=Gutierrez-Rodrigues%2CF) 
    
23.  Gutierrez-Rodrigues, F. et al. Clonal landscape and clinical outcomes of telomere biology disorders: somatic rescue and cancer mutations. _Blood_ **144**, 2402–2416 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXis1ynsrjJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39316766)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20landscape%20and%20clinical%20outcomes%20of%20telomere%20biology%20disorders%3A%20somatic%20rescue%20and%20cancer%20mutations&journal=Blood&volume=144&pages=2402-2416&publication_year=2024&author=Gutierrez-Rodrigues%2CF) 
    
24.  Codd, V. et al. Polygenic basis and biomedical consequences of telomere length variation. _Nat. Genet._ **53**, 1425–1433 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Wqs7rK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34611362)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8492471)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20basis%20and%20biomedical%20consequences%20of%20telomere%20length%20variation&journal=Nat.%20Genet.&volume=53&pages=1425-1433&publication_year=2021&author=Codd%2CV) 
    
25.  Brown, D. W. et al. Shared and distinct genetic etiologies for different types of clonal hematopoiesis. _Nat. Commun._ **14**, 5536 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvV2itrrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37684235)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10491829)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Shared%20and%20distinct%20genetic%20etiologies%20for%20different%20types%20of%20clonal%20hematopoiesis&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Brown%2CDW) 
    
26.  Codd, V. et al. Measurement and initial characterization of leukocyte telomere length in 474,074 participants in UK Biobank. _Nat. Aging_ **2**, 170–179 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmtFSkt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37117760)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Measurement%20and%20initial%20characterization%20of%20leukocyte%20telomere%20length%20in%20474%2C074%20participants%20in%20UK%20Biobank&journal=Nat.%20Aging&volume=2&pages=170-179&publication_year=2022&author=Codd%2CV) 
    
27.  Taub, M. A. et al. Genetic determinants of telomere length from 109,122 ancestrally diverse whole-genome sequences in TOPMed. _Cell Genom._ **2**, 100084 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVWksbbL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35530816)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9075703)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20determinants%20of%20telomere%20length%20from%20109%2C122%20ancestrally%20diverse%20whole-genome%20sequences%20in%20TOPMed&journal=Cell%20Genom.&volume=2&publication_year=2022&author=Taub%2CMA) 
    
28.  Cawthon, R. M. Telomere measurement by quantitative PCR. _Nucleic Acids Res._ **30**, e47 (2002).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12000852)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC115301)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomere%20measurement%20by%20quantitative%20PCR&journal=Nucleic%20Acids%20Res.&volume=30&publication_year=2002&author=Cawthon%2CRM) 
    
29.  Alter, B. P. et al. Very short telomere length by flow fluorescence in situ hybridization identifies patients with dyskeratosis congenita. _Blood_ **110**, 1439–1447 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSgsb%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17468339)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1975834)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Very%20short%20telomere%20length%20by%20flow%20fluorescence%20in%20situ%20hybridization%20identifies%20patients%20with%20dyskeratosis%20congenita&journal=Blood&volume=110&pages=1439-1447&publication_year=2007&author=Alter%2CBP) 
    
30.  Steenstrup, T. et al. Telomeres and the natural lifespan limit in humans. _Aging (Albany NY)_ **9**, 1130–1142 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28394764)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomeres%20and%20the%20natural%20lifespan%20limit%20in%20humans&journal=Aging%20%28Albany%20NY%29&volume=9&pages=1130-1142&publication_year=2017&author=Steenstrup%2CT) 
    
31.  Aubert, G., Baerlocher, G. M., Vulto, I., Poon, S. S. & Lansdorp, P. M. Collapse of telomere homeostasis in hematopoietic cells caused by heterozygous mutations in telomerase genes. _PLoS Genet._ **8**, e1002696 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XnvFSht70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22661914)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355073)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Collapse%20of%20telomere%20homeostasis%20in%20hematopoietic%20cells%20caused%20by%20heterozygous%20mutations%20in%20telomerase%20genes&journal=PLoS%20Genet.&volume=8&publication_year=2012&author=Aubert%2CG&author=Baerlocher%2CGM&author=Vulto%2CI&author=Poon%2CSS&author=Lansdorp%2CPM) 
    
32.  Baerlocher, G. M., Vulto, I., de Jong, G. & Lansdorp, P. M. Flow cytometry and FISH to measure the average length of telomeres (flow FISH). _Nat. Protoc._ **1**, 2365–2376 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFGjtLbL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17406480)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flow%20cytometry%20and%20FISH%20to%20measure%20the%20average%20length%20of%20telomeres%20%28flow%20FISH%29&journal=Nat.%20Protoc.&volume=1&pages=2365-2376&publication_year=2006&author=Baerlocher%2CGM&author=Vulto%2CI&author=Jong%2CG&author=Lansdorp%2CPM) 
    
33.  Gutierrez-Rodrigues, F., Santana-Lemos, B. A., Scheucher, P. S., Alves-Paiva, R. M. & Calado, R. T. Direct comparison of flow-FISH and qPCR as diagnostic tests for telomere length measurement in humans. _PLoS ONE_ **9**, e113747 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25409313)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237503)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Direct%20comparison%20of%20flow-FISH%20and%20qPCR%20as%20diagnostic%20tests%20for%20telomere%20length%20measurement%20in%20humans&journal=PLoS%20ONE&volume=9&publication_year=2014&author=Gutierrez-Rodrigues%2CF&author=Santana-Lemos%2CBA&author=Scheucher%2CPS&author=Alves-Paiva%2CRM&author=Calado%2CRT) 
    
34.  Schwaab, J. et al. Mutational complexity is related to disease severity in systemic mastocytosis. _Blood_ **122**, 4094 (2013).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20complexity%20is%20related%20to%20disease%20severity%20in%20systemic%20mastocytosis&journal=Blood&volume=122&publication_year=2013&author=Schwaab%2CJ) 
    
35.  Christopeit, M. et al. Clonal evolution after allogeneic hematopoietic stem cell transplantation: the case of myelofibrosis. _Biol. Blood Marrow Transpl._ **26**, e167–e170 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsV2lsb8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20evolution%20after%20allogeneic%20hematopoietic%20stem%20cell%20transplantation%3A%20the%20case%20of%20myelofibrosis&journal=Biol.%20Blood%20Marrow%20Transpl.&volume=26&pages=e167-e170&publication_year=2020&author=Christopeit%2CM) 
    
36.  Armanios, M. & Blackburn, E. H. The telomere syndromes. _Nat. Rev. Genet._ **13**, 693–704 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlamtbrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22965356)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3548426)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20telomere%20syndromes&journal=Nat.%20Rev.%20Genet.&volume=13&pages=693-704&publication_year=2012&author=Armanios%2CM&author=Blackburn%2CEH) 
    
37.  Yamaguchi, H. et al. Mutations in TERT, the gene for telomerase reverse transcriptase, in aplastic anemia. _N. Engl. J. Med._ **352**, 1413–1424 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXjtVWltr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15814878)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutations%20in%20TERT%2C%20the%20gene%20for%20telomerase%20reverse%20transcriptase%2C%20in%20aplastic%20anemia&journal=N.%20Engl.%20J.%20Med.&volume=352&pages=1413-1424&publication_year=2005&author=Yamaguchi%2CH) 
    
38.  Papaemmanuil, E. et al. Clinical and biological implications of driver mutations in myelodysplastic syndromes. _Blood_ **122**, 3616–3627 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVKjsL%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24030381)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3837510)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20and%20biological%20implications%20of%20driver%20mutations%20in%20myelodysplastic%20syndromes.&journal=Blood&volume=122&pages=3616-3627&publication_year=2013&author=Papaemmanuil%2CE) 
    
39.  Haferlach, T. et al. Landscape of genetic lesions in 944 patients with myelodysplastic syndromes. _Leukemia_ **28**, 241–247 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVyktL7N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24220272)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Landscape%20of%20genetic%20lesions%20in%20944%20patients%20with%20myelodysplastic%20syndromes&journal=Leukemia&volume=28&pages=241-247&publication_year=2014&author=Haferlach%2CT) 
    
40.  Dolatshad, H. et al. Cryptic splicing events in the iron transporter ABCB7 and other key target genes in SF3B1-mutant myelodysplastic syndromes. _Leukemia_ **30**, 2322–2331 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVWlsbvL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27211273)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5029572)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cryptic%20splicing%20events%20in%20the%20iron%20transporter%20ABCB7%20and%20other%20key%20target%20genes%20in%20SF3B1-mutant%20myelodysplastic%20syndromes&journal=Leukemia&volume=30&pages=2322-2331&publication_year=2016&author=Dolatshad%2CH) 
    
41.  Mian, S. A. et al. Vitamin B5 and succinyl-CoA improve ineffective erythropoiesis in SF3B1-mutated myelodysplasia. _Sci. Transl. Med._ **15**, eabn5135 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXltFyjt70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36857430)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7614516)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Vitamin%20B5%20and%20succinyl-CoA%20improve%20ineffective%20erythropoiesis%20in%20SF3B1-mutated%20myelodysplasia&journal=Sci.%20Transl.%20Med.&volume=15&publication_year=2023&author=Mian%2CSA) 
    
42.  Kim, E. et al. SRSF2 mutations contribute to myelodysplasia by mutant-specific effects on exon recognition. _Cancer Cell_ **27**, 617–630 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXosVamur8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25965569)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4429920)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SRSF2%20mutations%20contribute%20to%20myelodysplasia%20by%20mutant-specific%20effects%20on%20exon%20recognition&journal=Cancer%20Cell&volume=27&pages=617-630&publication_year=2015&author=Kim%2CE) 
    
43.  Florez, M. A. et al. Clonal hematopoiesis: mutation-specific adaptation to environmental change. _Cell Stem Cell_ **29**, 882–904 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVCnsbnL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35659875)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9202417)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%3A%20mutation-specific%20adaptation%20to%20environmental%20change&journal=Cell%20Stem%20Cell&volume=29&pages=882-904&publication_year=2022&author=Florez%2CMA) 
    
44.  Shiozawa, Y. et al. Aberrant splicing and defective mRNA production induced by somatic spliceosome mutations in myelodysplasia. _Nat. Commun._ **9**, 3649 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30194306)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6128865)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aberrant%20splicing%20and%20defective%20mRNA%20production%20induced%20by%20somatic%20spliceosome%20mutations%20in%20myelodysplasia&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Shiozawa%2CY) 
    
45.  Bamopoulos, S. A. et al. Clinical presentation and differential splicing of SRSF2, U2AF1 and SF3B1 mutations in patients with acute myeloid leukemia. _Leukemia_ **34**, 2621–2634 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXosVyltbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32358566)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20presentation%20and%20differential%20splicing%20of%20SRSF2%2C%20U2AF1%20and%20SF3B1%20mutations%20in%20patients%20with%20acute%20myeloid%20leukemia&journal=Leukemia&volume=34&pages=2621-2634&publication_year=2020&author=Bamopoulos%2CSA) 
    
46.  Hershberger, C. E. et al. Complex landscape of alternative splicing in myeloid neoplasms. _Leukemia_ **35**, 1108–1120 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFGitr7F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32753690)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complex%20landscape%20of%20alternative%20splicing%20in%20myeloid%20neoplasms&journal=Leukemia&volume=35&pages=1108-1120&publication_year=2021&author=Hershberger%2CCE) 
    
47.  Demanelis, K. et al. Determinants of telomere length across human tissues. _Science_ **369**, eaaz6876 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32913074)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8108546)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Determinants%20of%20telomere%20length%20across%20human%20tissues&journal=Science&volume=369&publication_year=2020&author=Demanelis%2CK) 
    
48.  Ishibashi, T. & Lippard, S. J. Telomere loss in cells treated with cisplatin. _Proc. Natl Acad. Sci. USA_ **95**, 4219–4223 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXis1OhtL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9539717)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC22469)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomere%20loss%20in%20cells%20treated%20with%20cisplatin&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=95&pages=4219-4223&publication_year=1998&author=Ishibashi%2CT&author=Lippard%2CSJ) 
    
49.  Saker, L. et al. Platinum complexes can bind to telomeres by coordination. _Int. J. Mol. Sci._ **19**, 1951 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29970863)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6073198)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Platinum%20complexes%20can%20bind%20to%20telomeres%20by%20coordination&journal=Int.%20J.%20Mol.%20Sci.&volume=19&publication_year=2018&author=Saker%2CL) 
    
50.  Storchova, R. et al. Phosphorylation of TRF2 promotes its interaction with TIN2 and regulates DNA damage response at telomeres. _Nucleic Acids Res._ **51**, 1154–1172 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVagtb%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36651296)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9943673)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phosphorylation%20of%20TRF2%20promotes%20its%20interaction%20with%20TIN2%20and%20regulates%20DNA%20damage%20response%20at%20telomeres&journal=Nucleic%20Acids%20Res.&volume=51&pages=1154-1172&publication_year=2023&author=Storchova%2CR) 
    
51.  Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. _Nature_ **606**, 343–350 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVant7vO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35650442)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9177428)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20dynamics%20of%20haematopoiesis%20across%20the%20human%20lifespan&journal=Nature&volume=606&pages=343-350&publication_year=2022&author=Mitchell%2CE) 
    
52.  Wan, Y. & Wu, C. J. SF3B1 mutations in chronic lymphocytic leukemia. _Blood_ **121**, 4627–4634 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpsFWktb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23568491)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3674664)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SF3B1%20mutations%20in%20chronic%20lymphocytic%20leukemia&journal=Blood&volume=121&pages=4627-4634&publication_year=2013&author=Wan%2CY&author=Wu%2CCJ) 
    
53.  Luo, J., Chen, C., Liu, Z. & Wang, X. The mutation in splicing factor genes correlates with unfavorable prognosis, genomic instability, anti-tumor immunosuppression and increased immunotherapy response in pan-cancer. _Front Cell Dev. Biol._ **10**, 1045130 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36684432)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutation%20in%20splicing%20factor%20genes%20correlates%20with%20unfavorable%20prognosis%2C%20genomic%20instability%2C%20anti-tumor%20immunosuppression%20and%20increased%20immunotherapy%20response%20in%20pan-cancer&journal=Front%20Cell%20Dev.%20Biol.&volume=10&publication_year=2022&author=Luo%2CJ&author=Chen%2CC&author=Liu%2CZ&author=Wang%2CX) 
    
54.  Niroula, A. et al. Distinction of lymphoid and myeloid clonal hematopoiesis. _Nat. Med._ **27**, 1921–1927 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Ohs77F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34663986)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8621497)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinction%20of%20lymphoid%20and%20myeloid%20clonal%20hematopoiesis&journal=Nat.%20Med.&volume=27&pages=1921-1927&publication_year=2021&author=Niroula%2CA) 
    
55.  The All of Us Research Program InvestigatorsThe ‘All of Us’ research program. _New Engl. J. Med._ **381**, 668–676 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20%E2%80%98All%20of%20Us%E2%80%99%20research%20program.&journal=New%20Engl.%20J.%20Med.&volume=381&pages=668-676&publication_year=2019&author=The%20All%20of%20Us%20Research%20Program%20Investigators%2C) 
    
56.  Vlasschaert, C. et al. A practical approach to curate clonal hematopoiesis of indeterminate potential in human genetic data sets. _Blood_ **141**, 2214–2223 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXpt1emtr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36652671)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10273159)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20practical%20approach%20to%20curate%20clonal%20hematopoiesis%20of%20indeterminate%20potential%20in%20human%20genetic%20data%20sets&journal=Blood&volume=141&pages=2214-2223&publication_year=2023&author=Vlasschaert%2CC) 
    
57.  Loh, P. R. et al. Insights into clonal haematopoiesis from 8,342 mosaic chromosomal alterations. _Nature_ **559**, 350–355 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjsbbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29995854)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6054542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Insights%20into%20clonal%20haematopoiesis%20from%208%2C342%20mosaic%20chromosomal%20alterations&journal=Nature&volume=559&pages=350-355&publication_year=2018&author=Loh%2CPR) 
    
58.  Orrù, V. et al. Genetic variants regulating immune cell levels in health and disease. _Cell_ **155**, 242–256 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24074872)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5541764)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20variants%20regulating%20immune%20cell%20levels%20in%20health%20and%20disease&journal=Cell&volume=155&pages=242-256&publication_year=2013&author=Orr%C3%B9%2CV) 
    
59.  Williams, N. et al. Life histories of myeloproliferative neoplasms inferred from phylogenies. _Nature_ **602**, 162–168 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhslCns7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35058638)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Life%20histories%20of%20myeloproliferative%20neoplasms%20inferred%20from%20phylogenies&journal=Nature&volume=602&pages=162-168&publication_year=2022&author=Williams%2CN) 
    
60.  Spencer Chapman, M. et al. Lineage tracing of human development through somatic mutations. _Nature_ **595**, 85–90 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVOksLjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33981037)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lineage%20tracing%20of%20human%20development%20through%20somatic%20mutations&journal=Nature&volume=595&pages=85-90&publication_year=2021&author=Spencer%20Chapman%2CM) 
    
61.  Jones, D. et al. cgpCaVEManWrapper: simple execution of CaVEMan in order to detect somatic single nucleotide variants in NGS data. _Curr. Protoc. Bioinformatics_ **56**, 15.10.11–15.10.18 (2016).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=cgpCaVEManWrapper%3A%20simple%20execution%20of%20CaVEMan%20in%20order%20to%20detect%20somatic%20single%20nucleotide%20variants%20in%20NGS%20data&journal=Curr.%20Protoc.%20Bioinformatics&volume=56&pages=15.10.11-15.10.18&publication_year=2016&author=Jones%2CD) 
    
62.  Ye, K., Schulz, M. H., Long, Q., Apweiler, R. & Ning, Z. Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. _Bioinformatics_ **25**, 2865–2871 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtlWls7nP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19561018)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2781750)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pindel%3A%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads&journal=Bioinformatics&volume=25&pages=2865-2871&publication_year=2009&author=Ye%2CK&author=Schulz%2CMH&author=Long%2CQ&author=Apweiler%2CR&author=Ning%2CZ) 
    
63.  Hoang, D. T. et al. MPBoot: fast phylogenetic maximum parsimony tree inference and bootstrap approximation. _BMC Evol. Biol._ **18**, 11 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29390973)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5796505)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MPBoot%3A%20fast%20phylogenetic%20maximum%20parsimony%20tree%20inference%20and%20bootstrap%20approximation&journal=BMC%20Evol.%20Biol.&volume=18&publication_year=2018&author=Hoang%2CDT) 
    
64.  Farmery, J. H. R., Smith, M. L., Diseases, N. B.-R. & Lynch, A. G. Telomerecat: a ploidy-agnostic method for estimating telomere length from whole genome sequencing data. _Sci. Rep._ **8**, 1300 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29358629)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5778012)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Telomerecat%3A%20a%20ploidy-agnostic%20method%20for%20estimating%20telomere%20length%20from%20whole%20genome%20sequencing%20data&journal=Sci.%20Rep.&volume=8&publication_year=2018&author=Farmery%2CJHR&author=Smith%2CML&author=Diseases%2CNB-R&author=Lynch%2CAG) 
    
65.  Yu, G., Smith, D. K., Zhu, H., Guan, Y. & Lam, T. T.-Y. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. _Methods Ecol. Evol._ **8**, 28–36 (2017).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ggtree%3A%20an%20R%20package%20for%20visualization%20and%20annotation%20of%20phylogenetic%20trees%20with%20their%20covariates%20and%20other%20associated%20data&journal=Methods%20Ecol.%20Evol.&volume=8&pages=28-36&publication_year=2017&author=Yu%2CG&author=Smith%2CDK&author=Zhu%2CH&author=Guan%2CY&author=Lam%2CTT-Y) 
    
66.  Bates, D., Mächler, M., Bolker, B. & Walker, S. Fitting linear mixed-effects models using lme4. _J. Stat. Softw._ **67**, 1–48 (2015).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fitting%20linear%20mixed-effects%20models%20using%20lme4&journal=J.%20Stat.%20Softw.&volume=67&pages=1-48&publication_year=2015&author=Bates%2CD&author=M%C3%A4chler%2CM&author=Bolker%2CB&author=Walker%2CS) 
    
67.  Dunn, W. billydunn/telomeres-ch: v.1.0.0. _Zenodo_ [https://doi.org/10.5281/zenodo.15472283](https://doi.org/10.5281/zenodo.15472283) (2025).
    
68.  cksruthi. cksruthi/telomeres-CH: v.1.0. _Zenodo_ [https://doi.org/10.5281/zenodo.15499165](https://doi.org/10.5281/zenodo.15499165) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02296-x?format=refman&flavour=references)

Acknowledgements
----------------

We thank the participants and investigators involved in the UKB resource and in the other genome-wide association studies cited in this work who collectively made this research possible. This work was funded by a Consolidator Award from the European Research Council (819956), an Early Detection Project Grant from Cancer Research UK (EDDCPJT\\100010) and a joint Leukemia and Lymphoma Society—Blood Cancer UK Specialized Centre of Research Grant (7035-24) awarded to G.S.V. The Cambridge Stem Cell Institute is supported by the Wellcome Trust (203151/Z/16/Z, 203151/A/16/Z) and the UKRI Medical Research Council (MC\_PC\_17230). M.A.M. is funded by a Wellcome Trust PhD Studentship (102160/B/13/Z). W.G.D. is funded by a Clinical Research Fellowship from the Cancer Research UK Cambridge Centre (CTRQQR-2021\\100012). S.K. is supported by a UK Research and Innovation (UKRI) Future Leaders Fellowship (MR/T043202/2). P.M.Q. is supported by the Ramon y Cajal Program (RYC2022-036793-I) funded by MICIU/AEI/10.13039/501100011033 and cofunded by FSE+, and his work is funded by ISCIII (PI22/00218) cofunded by the EU. C.B. is supported by the Ramon y Cajal program (RYC2021-031291-I) and funded by MICU/AEI/10.13039/50100011033 and cofunded by European Union NextGenerationEU/PRTR. A.J.W. is supported by Cancer Research UK (DRCNPG-Jun24/100002), the UK Medical Research Council (UKRI1443) and the Rosetrees Trust. Y.P. is supported by the National Institutes of Health (T32 GM007347). A.G.B. is supported by a Burroughs Wellcome Fund Career Award for Medical Scientists, a Pew Charitable Trusts and Alexander and Margaret Stewart Trust Pew-Stewart Scholar for Cancer Research award and a Hevolution/AFAR New Investigator Award in Aging Biology and Geroscience Research. G.S.V. is supported by a Cancer Research UK Senior Cancer Fellowship (C22324/A23015) and work in his laboratory is also funded by the Kay Kendall Leukemia Fund, AstraZeneca, Blood Cancer UK and the Wellcome Trust. This research was supported by the Cambridge NIHR BRC Cell Phenotyping Hub, Cambridge Blood and Stem Cell Biobank and the Manchester Cancer Research Centre Biobank. This research was conducted using the UKB resource under approved application 56844.

Author information
------------------

Author notes

1.  These authors contributed equally: Matthew A. McLoughlin, Sruthi Cheloor Kovilakam, William G. Dunn.
    

### Authors and Affiliations

1.  Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK
    
    Matthew A. McLoughlin, Sruthi Cheloor Kovilakam, William G. Dunn, Muxin Gu, Jake Tobin, Laura Bond, Ludovica Marando, Rachael Wilson, Giampiero Valenzano, Vasiliki Symeonidou, Justyna Rak, Aristi Damaskou, Malgorzata Gozdecka, Xiaoxuan Liu, Irina Mohorianu, Alan J. Warren, E. Joanna Baxter, Jyoti Nangalia & George S. Vassiliou
    
2.  Department of Haematology, University of Cambridge, Cambridge, UK
    
    Matthew A. McLoughlin, Sruthi Cheloor Kovilakam, William G. Dunn, Muxin Gu, Jake Tobin, Laura Bond, Ludovica Marando, Sean Wen, Rachael Wilson, Giampiero Valenzano, Vasiliki Symeonidou, Justyna Rak, Aristi Damaskou, Malgorzata Gozdecka, Xiaoxuan Liu, Matthew Cullen, Theodora Foukaneli, Alan J. Warren, Chi Wong, E. Joanna Baxter, Margarete Fabre, Jyoti Nangalia & George S. Vassiliou
    
3.  Department of Haematology, Cambridge University Hospitals NHS Trust, Cambridge, UK
    
    Matthew A. McLoughlin, William G. Dunn, Muxin Gu, Jake Tobin, Theodora Foukaneli, Alan J. Warren, Chi Wong, George Follows, Anna L. Godfrey, Emma Gudgin, Margarete Fabre, Jyoti Nangalia & George S. Vassiliou
    
4.  Department of Medicine, Division of Genetic Medicine, Vanderbilt University Medical Center, Nashville, TN, USA
    
    Yash Pershad & Alexander G. Bick
    
5.  Wellcome Sanger Institute, Hinxton, Cambridge, UK
    
    Nicholas Williams, Daniel Leongamornlert, Kevin Dawson & Jyoti Nangalia
    
6.  Department of Hematology, Mayo Clinic, Rochester, MI, USA
    
    Ludovica Marando
    
7.  Centre for Genomics Research, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca, Cambridge, UK
    
    Sean Wen, Jonathan Mitchell & Margarete Fabre
    
8.  Department of Biochemistry and Molecular Biology, Instituto Universitario de Oncología (IUOPA),Universidad de Oviedo, Oviedo, Spain
    
    Clea Barcena & Pedro M. Quiros
    
9.  Hospital Sant Pau, Barcelona, Spain
    
    Josep Nomdedeu
    
10.  Centre for the Study of Haematological Malignancies, Nicosia, Cyprus
    
    Paul Costeas & George S. Vassiliou
    
11.  Cyprus Cancer Research Institute, Nicosia, Cyprus
    
    Ioannis D. Dimitriou & George S. Vassiliou
    
12.  Institute for Genetic and Biomedical Research, National Research Council, Lanusei, Italy
    
    Edoardo Fiorillo, Valeria Orrù & Francesco Cucca
    
13.  Computational Clinical Imaging Group, Champalimaud Foundation, Lisbon, Portugal
    
    Jose Guilherme de Almeida
    
14.  Department of Haematology, University Hospital Geelong, Geelong, Victoria, Australia
    
    Thomas McKerrell
    
15.  School of Medicine, Deakin University Geelong, Geelong, Victoria, Australia
    
    Thomas McKerrell
    
16.  Cambridge Institute for Medical Research, University of Cambridge, Cambridge, UK
    
    Alan J. Warren
    
17.  Department of Biomedical Science, University of Sassari, Sassari, Italy
    
    Francesco Cucca
    
18.  Cambridge Institute of Therapeutic Immunology and Infectious Disease, University of Cambridge, Cambridge, UK
    
    Eoin McKinney
    
19.  Division of AI in Oncology, German Cancer Research Centre DKFZ, Heidelberg, Germany
    
    Moritz Gerstung
    
20.  Cancer Research UK, Manchester Institute, Manchester, UK
    
    Daniel Wiseman
    
21.  Department of Oncology, Early Cancer Institute, University of Cambridge, Cambridge, UK
    
    Siddhartha Kar
    

Authors

1.  Matthew A. McLoughlin
    
    [View author publications](https://www.nature.com/search?author=Matthew%20A.%20McLoughlin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20A.%20McLoughlin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20A.%20McLoughlin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Sruthi Cheloor Kovilakam
    
    [View author publications](https://www.nature.com/search?author=Sruthi%20Cheloor%20Kovilakam)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sruthi%20Cheloor%20Kovilakam) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sruthi%20Cheloor%20Kovilakam%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  William G. Dunn
    
    [View author publications](https://www.nature.com/search?author=William%20G.%20Dunn)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=William%20G.%20Dunn) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22William%20G.%20Dunn%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Muxin Gu
    
    [View author publications](https://www.nature.com/search?author=Muxin%20Gu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Muxin%20Gu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Muxin%20Gu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Jake Tobin
    
    [View author publications](https://www.nature.com/search?author=Jake%20Tobin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jake%20Tobin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jake%20Tobin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Yash Pershad
    
    [View author publications](https://www.nature.com/search?author=Yash%20Pershad)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yash%20Pershad) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yash%20Pershad%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Nicholas Williams
    
    [View author publications](https://www.nature.com/search?author=Nicholas%20Williams)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nicholas%20Williams) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nicholas%20Williams%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Daniel Leongamornlert
    
    [View author publications](https://www.nature.com/search?author=Daniel%20Leongamornlert)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20Leongamornlert) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20Leongamornlert%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Kevin Dawson
    
    [View author publications](https://www.nature.com/search?author=Kevin%20Dawson)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kevin%20Dawson) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kevin%20Dawson%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Laura Bond
    
    [View author publications](https://www.nature.com/search?author=Laura%20Bond)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Laura%20Bond) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Laura%20Bond%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Ludovica Marando
    
    [View author publications](https://www.nature.com/search?author=Ludovica%20Marando)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ludovica%20Marando) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ludovica%20Marando%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Sean Wen
    
    [View author publications](https://www.nature.com/search?author=Sean%20Wen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sean%20Wen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sean%20Wen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Rachael Wilson
    
    [View author publications](https://www.nature.com/search?author=Rachael%20Wilson)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rachael%20Wilson) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rachael%20Wilson%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Giampiero Valenzano
    
    [View author publications](https://www.nature.com/search?author=Giampiero%20Valenzano)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Giampiero%20Valenzano) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Giampiero%20Valenzano%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Vasiliki Symeonidou
    
    [View author publications](https://www.nature.com/search?author=Vasiliki%20Symeonidou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Vasiliki%20Symeonidou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Vasiliki%20Symeonidou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Justyna Rak
    
    [View author publications](https://www.nature.com/search?author=Justyna%20Rak)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Justyna%20Rak) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Justyna%20Rak%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Aristi Damaskou
    
    [View author publications](https://www.nature.com/search?author=Aristi%20Damaskou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Aristi%20Damaskou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Aristi%20Damaskou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Malgorzata Gozdecka
    
    [View author publications](https://www.nature.com/search?author=Malgorzata%20Gozdecka)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Malgorzata%20Gozdecka) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Malgorzata%20Gozdecka%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Xiaoxuan Liu
    
    [View author publications](https://www.nature.com/search?author=Xiaoxuan%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Xiaoxuan%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Xiaoxuan%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Clea Barcena
    
    [View author publications](https://www.nature.com/search?author=Clea%20Barcena)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Clea%20Barcena) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Clea%20Barcena%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Josep Nomdedeu
    
    [View author publications](https://www.nature.com/search?author=Josep%20Nomdedeu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Josep%20Nomdedeu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Josep%20Nomdedeu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Paul Costeas
    
    [View author publications](https://www.nature.com/search?author=Paul%20Costeas)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Paul%20Costeas) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Paul%20Costeas%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Ioannis D. Dimitriou
    
    [View author publications](https://www.nature.com/search?author=Ioannis%20D.%20Dimitriou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ioannis%20D.%20Dimitriou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ioannis%20D.%20Dimitriou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Edoardo Fiorillo
    
    [View author publications](https://www.nature.com/search?author=Edoardo%20Fiorillo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Edoardo%20Fiorillo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Edoardo%20Fiorillo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Valeria Orrù
    
    [View author publications](https://www.nature.com/search?author=Valeria%20Orr%C3%B9)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Valeria%20Orr%C3%B9) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Valeria%20Orr%C3%B9%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Jose Guilherme de Almeida
    
    [View author publications](https://www.nature.com/search?author=Jose%20Guilherme%20de%20Almeida)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jose%20Guilherme%20de%20Almeida) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jose%20Guilherme%20de%20Almeida%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  Thomas McKerrell
    
    [View author publications](https://www.nature.com/search?author=Thomas%20McKerrell)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Thomas%20McKerrell) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Thomas%20McKerrell%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Matthew Cullen
    
    [View author publications](https://www.nature.com/search?author=Matthew%20Cullen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20Cullen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20Cullen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Irina Mohorianu
    
    [View author publications](https://www.nature.com/search?author=Irina%20Mohorianu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Irina%20Mohorianu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Irina%20Mohorianu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  Theodora Foukaneli
    
    [View author publications](https://www.nature.com/search?author=Theodora%20Foukaneli)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Theodora%20Foukaneli) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Theodora%20Foukaneli%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  Alan J. Warren
    
    [View author publications](https://www.nature.com/search?author=Alan%20J.%20Warren)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alan%20J.%20Warren) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alan%20J.%20Warren%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
32.  Chi Wong
    
    [View author publications](https://www.nature.com/search?author=Chi%20Wong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chi%20Wong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chi%20Wong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
33.  George Follows
    
    [View author publications](https://www.nature.com/search?author=George%20Follows)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=George%20Follows) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22George%20Follows%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
34.  Anna L. Godfrey
    
    [View author publications](https://www.nature.com/search?author=Anna%20L.%20Godfrey)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anna%20L.%20Godfrey) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anna%20L.%20Godfrey%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
35.  Emma Gudgin
    
    [View author publications](https://www.nature.com/search?author=Emma%20Gudgin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Emma%20Gudgin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Emma%20Gudgin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
36.  Francesco Cucca
    
    [View author publications](https://www.nature.com/search?author=Francesco%20Cucca)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Francesco%20Cucca) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Francesco%20Cucca%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
37.  Eoin McKinney
    
    [View author publications](https://www.nature.com/search?author=Eoin%20McKinney)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eoin%20McKinney) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eoin%20McKinney%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
38.  E. Joanna Baxter
    
    [View author publications](https://www.nature.com/search?author=E.%20Joanna%20Baxter)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=E.%20Joanna%20Baxter) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22E.%20Joanna%20Baxter%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
39.  Moritz Gerstung
    
    [View author publications](https://www.nature.com/search?author=Moritz%20Gerstung)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Moritz%20Gerstung) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Moritz%20Gerstung%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
40.  Jonathan Mitchell
    
    [View author publications](https://www.nature.com/search?author=Jonathan%20Mitchell)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jonathan%20Mitchell) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jonathan%20Mitchell%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
41.  Daniel Wiseman
    
    [View author publications](https://www.nature.com/search?author=Daniel%20Wiseman)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20Wiseman) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20Wiseman%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
42.  Alexander G. Bick
    
    [View author publications](https://www.nature.com/search?author=Alexander%20G.%20Bick)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alexander%20G.%20Bick) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alexander%20G.%20Bick%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
43.  Margarete Fabre
    
    [View author publications](https://www.nature.com/search?author=Margarete%20Fabre)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Margarete%20Fabre) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Margarete%20Fabre%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
44.  Pedro M. Quiros
    
    [View author publications](https://www.nature.com/search?author=Pedro%20M.%20Quiros)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Pedro%20M.%20Quiros) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Pedro%20M.%20Quiros%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
45.  Jyoti Nangalia
    
    [View author publications](https://www.nature.com/search?author=Jyoti%20Nangalia)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jyoti%20Nangalia) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jyoti%20Nangalia%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
46.  Siddhartha Kar
    
    [View author publications](https://www.nature.com/search?author=Siddhartha%20Kar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Siddhartha%20Kar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Siddhartha%20Kar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
47.  George S. Vassiliou
    
    [View author publications](https://www.nature.com/search?author=George%20S.%20Vassiliou)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=George%20S.%20Vassiliou) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22George%20S.%20Vassiliou%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

G.S.V. conceived, designed and supervised the study. M.A.M. designed and performed the majority of experiments. J.T., L.B., R.W. and G.V. performed additional experimental work. S.C.K. and M.G. performed UKB analysis. Y.P. and A.G.B. performed All of Us analysis. W.G.D, N.W., D.L. and K.D. performed phylogenetic analysis. S.C.K, W.G.D., M.G., and S.W. performed additional bioinformatics analysis. N.W., D.L., K.D., V.S., L.M., S.W., J.R., A.D., M. Gozdecka, X.L., C.B., J. Nomdedeu, P.C., I.D.D., J.G.dA., I.M., M.F., P.M.Q., M. Gerstung, J.M., J. Nangalia and S.K. provided technical advice and expertise. E.F., V.O., T.M., T.F., A.J.W., C.W., G.F., A.L.G., E.G., M.C., F.C., E.M., E.J.B. and D.W. provided patient material used in the study. G.S.V., M.A.M., S.C.K and W.G.D. prepared the paper and all authors reviewed the final paper.

### Corresponding author

Correspondence to [George S. Vassiliou](mailto:gsv20@cam.ac.uk).

Ethics declarations
-------------------

### Competing interests

G.S.V. is a consultant to STRM.BIO and holds a research grant from AstraZeneca for research unrelated to that presented here. M.A.F., S.W. and J.M. are employees and stockholders of AstraZeneca. A.J.W. is a consultant for SDS Therapeutics. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Alex Bick, Kelly Bolton and Mitchell Machiela for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02296-x#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Association of telomere length with myeloid malignancy risk.](https://www.nature.com/articles/s41588-025-02296-x/figures/6)

Odds ratio (OR) of developing different myeloid malignancies with every 1 standard deviation increase in leukocyte telomere length polygenic risk score (LTL-PRS) and measured telomere length (LTL). OR was calculated using logistic regression in which the outcome variable is the development of myeloid malignancy, and the input variable is LTL or LTL-PRS along with the covariates sex, age, smoking status and the first ten genetic principal components. Dots represent the estimated OR and error bars represent the 95% CI for each OR. Color indicates ORs with a P < 0.05 (pink for LTL, dark green for LTL-PRS) or P≥0.05 (light green). ‘MM’ represents myeloid malignancies, namely AML, MDS, MPN and CMML. The numbers of UK Biobank participants with myeloid malignancy included in the LTL-PRS and LTL association analyses are indicated in brackets, respectively.

[Source data](https://www.nature.com/articles/s41588-025-02296-x#MOESM7)

### [Extended Data Fig. 2 Association of paternal age at birth with clonal hematopoiesis risk.](https://www.nature.com/articles/s41588-025-02296-x/figures/7)

Odds ratio (OR) of developing CH per one year increase in paternal age at birth. ‘Long Telomere CH’ refers to CH mutations that are more common amongst individuals with longer genetically predicted telomere length (PRS), namely _DNMT3A_, _JAK2_, _TET2_, _GNAS_, _GNB1_ and _CBL_ (n = 2204) and ‘Short Telomere CH’ refers to CH mutations that are more common amongst individuals with shorter genetically predicted telomere length (PRS), namely _PPM1D_, _SF3B1_ and _SRSF2_ (n = 98). Dots represent the estimated OR and error bars represent the 95% CI for each OR. OR was calculated using logistic regression in which the outcome variable is the development of CH, and the input variable is paternal age at birth along with the covariates sex, age, smoking status and the first ten genetic principal components. Color indicates ORs with a P < 0.05 (purple) or P≥0.05 (pink). Number of UKB participants in each of the ‘Long Telomere CH’ and ‘Short Telomere CH’ groups is shown in brackets.

### [Extended Data Fig. 3 Distribution of UKB-derived LTL-PRS scores amongst All of Us participants.](https://www.nature.com/articles/s41588-025-02296-x/figures/8)

Upper panel: Violin plots of genetically determined LTL-PRS derived from the UKB[24](https://www.nature.com/articles/s41588-025-02296-x#ref-CR24 "Codd, V. et al. Polygenic basis and biomedical consequences of telomere length variation. Nat. Genet. 53, 1425–1433 (2021).") and applied to All of Us participants (top panel). White dots and black boxes mark the LTL-PRS median and interquartile range (IQR), respectively. Whiskers extend to the lowest and highest data points within Q1-1.5×IQR and Q3 + 1.5×IQR where Q1 and Q3 represent the first and third quartiles, respectively. The control group includes all participants without any CH mutation. Y-axis is limited to the range \[−4,4\] to allow for better visualization of PRS differences. Lower panel: Odds ratio (OR) of developing different CH subtypes per 1 standard deviation increase in LTL-PRS obtained using linear regression models. Dots represent the estimated OR and bars represent the 95% CI for each OR. Color indicates ORs with an unadjusted P < 0.05 (dark grey) or unadjusted P≥0.05 (light grey). ORs for genes with an FDR-adjusted P < 0.05 are indicated with an asterisk (\*). FDR correction was performed using the Benjamini-Hochberg procedure. The number of individuals in each group is indicated in brackets next to the gene names.

### [Extended Data Fig. 4 Distribution of TOPMed-derived LTL-PRS scores amongst UKB participants.](https://www.nature.com/articles/s41588-025-02296-x/figures/9)

Upper panel: Distribution of TOPMed-derived LTL-PRS scores[27](https://www.nature.com/articles/s41588-025-02296-x#ref-CR27 "Taub, M. A. et al. Genetic determinants of telomere length from 109,122 ancestrally diverse whole-genome sequences in TOPMed. Cell Genom. 2, 100084 (2022).") amongst UKB participants with or without different types of CH and mCA. White dots and black boxes mark the LTL-PRS median and interquartile range (IQR), respectively. Whiskers extend to the lowest and highest data points within Q1-1.5×IQR and Q3 + 1.5×IQR where Q1 and Q3 represent the first and third quartiles, respectively. The control group includes all participants without any CH mutation or mCA. Lower panel: Odds ratio (OR) of developing different CH/mCA subtypes per 1 standard deviation increase in LTL-PRS with values derived from a linear regression model. Dots represent the estimated OR and bars represent the 95% CI for each OR. Color indicates ORs with an unadjusted P < 0.05 (dark grey, dark purple) or unadjusted P≥0.05 (light grey, light purple). ORs for genes with an FDR-adjusted P < 0.05 are indicated with an asterisk (\*). FDR correction was performed using the Benjamini-Hochberg procedure. The number of individuals in each group is indicated in brackets next to the gene names.

### [Extended Data Fig. 5 Telomere lengths in U2AF1-mutant and closely related non-mutant clades.](https://www.nature.com/articles/s41588-025-02296-x/figures/10)

Comparison of telomere lengths of HSPC-derived colonies from the _U2AF1_\-mutant clade (blue) in individual PD34493 with iteratively more closely related colonies of non-mutant clades (pink), showing that the observed trend for longer telomeres is maintained when comparing with more closely ancestrally related colonies, despite the _U2AF1_\-mutant clade having undergone recent clonal expansion. Panels **a, c** and **e** highlight (pink) the clade being compared with the _U2AF1_ clade (blue). Panels **b, d** and **f** show boxplots and p values of pairwise comparisons of NGS-estimated telomere lengths of the clades highlighted in Panels **a, c** and **e** respectively (two-sided Wilcoxon Rank Sum Test). Center line represents the median telomere length, upper and lower hinges the upper and lower quartiles respectively, and whiskers represent 1.5 \* the inter-quartile range. Individual data points corresponding to telomere lengths of single-HSPC derived colonies have been overlaid on each plot.

[Source data](https://www.nature.com/articles/s41588-025-02296-x#MOESM8)

### [Extended Data Fig. 6 Orthogonal validation of NGS-estimated telomere lengths by qPCR.](https://www.nature.com/articles/s41588-025-02296-x/figures/11)

Telomere lengths for colonies cultured from PD34493 were estimated using quantitative real-time PCR (qPCR) as T/S ratios (left panel). These recapitulate the pattern seen when telomere length is estimated from NGS data (right panel). By capturing more _U2AF1_\-mutant colonies, the qPCR results demonstrate a significant difference in telomere length between _U2AF1_\-mutant and wild-type colonies. P values were calculated using a two-sided Wilcoxon rank sum test comparing _SF3B1_\- or _U2AF1_\-mutant colonies to WT colonies. Center line represents the median T/S ratio or telomere length, upper and lower hinges the upper and lower quartiles respectively, and whiskers represent 1.5 X the IQR. Individual data points corresponding to T/S ratios/telomere lengths of single-HSPC derived colonies have been overlaid on each plot.

[Source data](https://www.nature.com/articles/s41588-025-02296-x#MOESM9)

### [Extended Data Fig. 7 Validation of the sensitivity of telomere qPCR and flow-FISH in _TERT_ KO cells.](https://www.nature.com/articles/s41588-025-02296-x/figures/12)

**a**, Timeline for _TERT_ knockdown experiment (d = day). **b-c**, Sanger sequencing of _TERT_ exon 2 from K562-Cas9 cells at day 14 following transduction with empty gRNA (top) or gRNAs targeting _TERT_ exon 2 (bottom, _TERT_ gRNA 1 (**b**) or _TERT_ gRNA 2 (**c**)). **d**, Telomere qPCR of three Cas9-expressing cell lines (K562, OCI-AML2 and HEK293FT) 14 days after _TERT_ knockout (gRNA). **e-h**, Telomere flow-FISH results at day 17 following _TERT_ knockdown: **e,f**, Mean fluorescence intensity of telomere and centromere probes by flow-FISH for K562 (**e**) and OCI-AML2 (**f**). **g-h**, Example histograms for centromere (**g**) and telomere (**h**) probes in K562. Panel **a** created using [BioRender.com](http://biorender.com/).

[Source data](https://www.nature.com/articles/s41588-025-02296-x#MOESM10)

### [Extended Data Fig. 8 Isolation of CLL cells from peripheral blood.](https://www.nature.com/articles/s41588-025-02296-x/figures/13)

Example gating strategy for isolating CD5+ CLL B cells with different telomere lengths from peripheral blood. Starting with mononuclear cells isolated from peripheral blood, live CD5+ CD19+ CD3− single cells were sorted, fixed and hybridized overnight with flow-FISH probes. The following day, cells in G1 (assessed by centromere staining) were sorted by telomere length into telomerelow (≤33rd percentile telomere length) and telomerehigh (≥66th percentile telomere length) fractions as depicted. Sorted cells were prepared for DNA extraction and sequencing and results are shown in Fig. [4b](https://www.nature.com/articles/s41588-025-02296-x#Fig4)/c. For samples in Fig. [4d–g](https://www.nature.com/articles/s41588-025-02296-x#Fig4), isolated mononuclear cells underwent fixation and overnight probe hybridization and then G1 cells were sorted by percentile telomere length into 4 fractions (telomere<10%, telomere10–33%, telomere66–90%, telomere\>90%). An increased number of fractions was used due to the availability of higher cell numbers from these samples.

### [Extended Data Fig. 9 Patient treated with Azacitidine does not show enrichment of splicing mutant cells in telomerehigh cells.](https://www.nature.com/articles/s41588-025-02296-x/figures/14)

Ratio of _SRSF2_\-mutant to _TET2_\-mutant cells in an individual with CMML following 31 cycles of Azacitidine treatment.

[Source data](https://www.nature.com/articles/s41588-025-02296-x#MOESM11)

### [Extended Data Fig. 10 Age-related prevalence of _SF3B1_, _SRSF2_ and _TERT_p-CH amongst UK Biobank participants.](https://www.nature.com/articles/s41588-025-02296-x/figures/15)

The prevalence of _SF3B1_ and _SRSF2_ hotspot mutations increases sharply after the sixth decade of life. The sensitivity to detect CH clones is a function of sequencing depth and modality. Since _TERT_p is not adequately captured by WES, we identified _TERT_p-CH (red line) from blood DNA WGS data of (methodology described in Supplementary Note [2](https://www.nature.com/articles/s41588-025-02296-x#MOESM1)). When we call _SF3B1_ and _SRSF2_ hotspot mutations from WGS using the same filtering method, the prevalence of _TERT_p-CH is comparable to the combined prevalence of _SF3B1_\-CH (light green line) and _SRSF2_\-CH (light blue line) and exhibits a similar association with advancing age.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM1_ESM.pdf)

Supplementary Figs. 1–5, Methods and Notes 1–2.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM4_ESM.xlsx)

Supplementary Tables 1–17.

Source data
-----------

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM5_ESM.xlsx)

Source data for Fig. 3.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM6_ESM.xlsx)

Source data for Fig. 4.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM7_ESM.xlsx)

Source data for Extended Data Fig. 1.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM8_ESM.xlsx)

Source data for Extended Data Fig. 5.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM9_ESM.xlsx)

Source data for Extended Data Fig. 6.

### [Source Data Extended Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM10_ESM.xlsx)

Source data for Extended Data Fig. 7.

### [Source Data Extended Data Fig. 9](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02296-x/MediaObjects/41588_2025_2296_MOESM11_ESM.xlsx)

Source data for Extended Data Fig. 9.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Telomere%20attrition%20becomes%20an%20instrument%20for%20clonal%20selection%20in%20aging%20hematopoiesis%20and%20leukemogenesis&author=Matthew%20A.%20McLoughlin%20et%20al&contentID=10.1038%2Fs41588-025-02296-x&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-28&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02296-x)

### Cite this article

McLoughlin, M.A., Cheloor Kovilakam, S., Dunn, W.G. _et al._ Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02296-x

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02296-x?format=refman&flavour=citation)

-   Received: 09 June 2024
    
-   Accepted: 14 July 2025
    
-   Published: 28 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02296-x
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02296-x.pdf)
