> id: https://www.nature.com/articles/s41588-025-02289-w

> link: https://www.nature.com/articles/s41588-025-02289-w

> title: ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L

# ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L
_Published on Tue, 26 Aug 2025 00:00:00 GMT_

Abstract
--------

Despite the high prevalence of ERG transcription factor translocations in prostate cancer, the mechanism of tumorigenicity remains poorly understood. Using lineage tracing, we find the tumor-initiating activity of ERG resides in a subpopulation of murine basal cells that coexpress luminal genes (BasalLum) and not in the larger population of ERG+ luminal cells. Upon ERG activation, BasalLum cells give rise to highly proliferative intermediate (IM) cells with stem-like features that coexpress basal, luminal, hillock and club marker genes, before transitioning to Krt8+ luminal cells. Transcriptomic analysis of ERG+ human prostate cancers confirms the presence of rare ERG+ BasalLum cells, as well as IM cells whose presence is associated with a worse prognosis. Single-cell analysis revealed a chromatin state in ERG+ IM cells enriched for STAT3 transcription factor binding sites and elevated expression of the KMT2A/MLL1 and DOT1L, all three of which are essential for ERG-driven tumorigenicity in vivo. In addition to providing translational opportunities, this work illustrates how single-cell approaches combined with lineage tracing can identify cancer vulnerabilities not evident from bulk analysis.

Main
----

Multiple lines of evidence implicate translocations in ERG, which are present in ~40–50% of prostate cancers (PCa) in Western cohorts, as an initiating event in PCa. For example, ERG expression is seen in prostatic intraepithelial neoplasia (PIN) and proliferative inflammatory atrophy, lesions that can precede the development of cancer, and is uniform across all cells. Experimental models, notably through transgenic expression of ERG in the mouse prostate, establish that ERG is sufficient to induce PCa, particularly in the context of PI3K pathway activation[1](https://www.nature.com/articles/s41588-025-02289-w#ref-CR1 "Carver, B. S. et al. Aberrant ERG expression cooperates with loss of PTEN to promote cancer progression in the prostate. Nat. Genet. 41, 619–624 (2009)."),[2](https://www.nature.com/articles/s41588-025-02289-w#ref-CR2 "Cerveira, N. et al. TMPRSS2-ERG gene fusion causing ERG overexpression precedes chromosome copy number changes in prostate carcinomas and paired HGPIN lesions. Neoplasia 8, 826–832 (2006)."),[3](https://www.nature.com/articles/s41588-025-02289-w#ref-CR3 "Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. Nat. Med. 19, 1023–1029 (2013)."),[4](https://www.nature.com/articles/s41588-025-02289-w#ref-CR4 "Gerhauser, C. et al. Molecular evolution of early-onset prostate cancer identifies molecular risk markers and clinical trajectories. Cancer Cell 34, 996–1011 e8 (2018)."),[5](https://www.nature.com/articles/s41588-025-02289-w#ref-CR5 "King, J. C. et al. Cooperativity of TMPRSS2-ERG with PI3-kinase pathway activation in prostate oncogenesis. Nat. Genet. 41, 524–526 (2009)."),[6](https://www.nature.com/articles/s41588-025-02289-w#ref-CR6 "Mosquera, J. M. et al. Characterization of TMPRSS2-ERG fusion high-grade prostatic intraepithelial neoplasia and potential clinical implications. Clin. Cancer Res. 14, 3380–3385 (2008)."),[7](https://www.nature.com/articles/s41588-025-02289-w#ref-CR7 "Perner, S. et al. TMPRSS2-ERG fusion prostate cancer: an early molecular event associated with invasion. Am. J. Surg. Pathol. 31, 882–888 (2007)."),[8](https://www.nature.com/articles/s41588-025-02289-w#ref-CR8 "Shrestha, E. et al. Oncogenic gene fusions in nonneoplastic precursors as evidence that bacterial infection can initiate prostate cancer. Proc. Natl Acad. Sci. USA 118, e2018976118 (2021)."),[9](https://www.nature.com/articles/s41588-025-02289-w#ref-CR9 "Taylor, B. S. et al. Integrative genomic profiling of human prostate cancer. Cancer Cell 18, 11–22 (2010).").

Despite the availability of several models of ERG-driven disease initiation[3](https://www.nature.com/articles/s41588-025-02289-w#ref-CR3 "Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. Nat. Med. 19, 1023–1029 (2013)."),[10](https://www.nature.com/articles/s41588-025-02289-w#ref-CR10 "Bose, R. et al. ERF mutations reveal a balance of ETS factors controlling prostate oncogenesis. Nature 546, 671–675 (2017)."),[11](https://www.nature.com/articles/s41588-025-02289-w#ref-CR11 "Brenner, J. C. et al. Mechanistic rationale for inhibition of poly(ADP-ribose) polymerase in ETS gene fusion-positive prostate cancer. Cancer Cell 19, 664–678 (2011)."),[12](https://www.nature.com/articles/s41588-025-02289-w#ref-CR12 "Cai, C. et al. ERG induces androgen receptor-mediated regulation of SOX9 in prostate cancer. J. Clin. Invest. 123, 1109–1122 (2013)."),[13](https://www.nature.com/articles/s41588-025-02289-w#ref-CR13 "Gupta, S. et al. FZD4 as a mediator of ERG oncogene-induced WNT signaling and epithelial-to-mesenchymal transition in human prostate cancer cells. Cancer Res. 70, 6735–6745 (2010)."),[14](https://www.nature.com/articles/s41588-025-02289-w#ref-CR14 "Hollenhorst, P. C. et al. Oncogenic ETS proteins mimic activated RAS/MAPK signaling in prostate cells. Genes Dev. 25, 2147–2157 (2011)."),[15](https://www.nature.com/articles/s41588-025-02289-w#ref-CR15 "Kim, T. D., Shin, S. & Janknecht, R. ETS transcription factor ERG cooperates with histone demethylase KDM4A. Oncol. Rep. 35, 3679–3688 (2016)."),[16](https://www.nature.com/articles/s41588-025-02289-w#ref-CR16 "Kron, K. J. et al. TMPRSS2-ERG fusion co-opts master transcription factors and activates NOTCH signaling in primary prostate cancer. Nat. Genet. 49, 1336–1345 (2017)."),[17](https://www.nature.com/articles/s41588-025-02289-w#ref-CR17 "Li, F. et al. ERG orchestrates chromatin interactions to drive prostate cell fate reprogramming. J. Clin. Invest. 130, 5924–5941 (2020)."),[18](https://www.nature.com/articles/s41588-025-02289-w#ref-CR18 "Liao, S. Y. et al. SND1 binds to ERG and promotes tumor growth in genetic mouse models of prostate cancer. Nat. Commun. 14, 7435 (2023)."),[19](https://www.nature.com/articles/s41588-025-02289-w#ref-CR19 "Linn, D. E., Penney, K. L., Bronson, R. T., Mucci, L. A. & Li, Z. Deletion of interstitial genes between TMPRSS2 and ERG promotes prostate cancer progression. Cancer Res. 76, 1869–1881 (2016)."),[20](https://www.nature.com/articles/s41588-025-02289-w#ref-CR20 "Liu, C. Y., Yu, T., Huang, Y., Cui, L. & Hong, W. ETS (E26 transformation-specific) up-regulation of the transcriptional co-activator TAZ promotes cell migration and metastasis in prostate cancer. J. Biol. Chem. 292, 9420–9430 (2017)."),[21](https://www.nature.com/articles/s41588-025-02289-w#ref-CR21 "Mao, N. et al. Aberrant expression of ERG promotes resistance to combined PI3K and AR pathway inhibition through maintenance of AR target genes. Mol. Cancer Ther. 18, 1577–1586 (2019)."),[22](https://www.nature.com/articles/s41588-025-02289-w#ref-CR22 "Mounir, Z. et al. TMPRSS2:ERG blocks neuroendocrine and luminal cell differentiation to maintain prostate cancer proliferation. Oncogene 34, 3815–3825 (2015)."),[23](https://www.nature.com/articles/s41588-025-02289-w#ref-CR23 "Nguyen, L. T. et al. ERG activates the YAP1 transcriptional program and induces the development of age-related prostate tumors. Cancer Cell 27, 797–808 (2015)."),[24](https://www.nature.com/articles/s41588-025-02289-w#ref-CR24 "Sandoval, G. J. et al. Binding of TMPRSS2-ERG to BAF chromatin remodeling complexes mediates prostate oncogenesis. Mol. Cell 71, 554–566 (2018)."),[25](https://www.nature.com/articles/s41588-025-02289-w#ref-CR25 "Tomlins, S. A. et al. Role of the TMPRSS2-ERG gene fusion in prostate cancer. Neoplasia 10, 177–188 (2008)."),[26](https://www.nature.com/articles/s41588-025-02289-w#ref-CR26 "Wang, J. et al. Activation of NF-κB by TMPRSS2/ERG fusion isoforms through Toll-like receptor-4. Cancer Res. 71, 1325–1333 (2011)."),[27](https://www.nature.com/articles/s41588-025-02289-w#ref-CR27 "Wasmuth, E. V. et al. Modulation of androgen receptor DNA binding activity through direct interaction with the ETS transcription factor ERG. Proc. Natl Acad. Sci. USA 117, 8584–8592 (2020)."),[28](https://www.nature.com/articles/s41588-025-02289-w#ref-CR28 "Wu, L. et al. ERG is a critical regulator of Wnt/LEF1 signaling in prostate cancer. Cancer Res. 73, 6068–6079 (2013)."),[29](https://www.nature.com/articles/s41588-025-02289-w#ref-CR29 "Yu, J. et al. An integrated network of androgen receptor, polycomb, and TMPRSS2-ERG gene fusions in prostate cancer progression. Cancer Cell 17, 443–454 (2010)."), the mechanism by which ERG initiates PCa remains unclear. ERG binding sites in chromatin, as well as ERG-induced transcriptional changes, have been extensively annotated across a range of mouse and human models, but these datasets fail to reveal a clear sequence of downstream events responsible for oncogenicity. RNA sequencing (RNA-seq) analysis has shown that ERG activates a pro-luminal epithelial differentiation program, with consequent loss of basal epithelial cells[3](https://www.nature.com/articles/s41588-025-02289-w#ref-CR3 "Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. Nat. Med. 19, 1023–1029 (2013)."),[10](https://www.nature.com/articles/s41588-025-02289-w#ref-CR10 "Bose, R. et al. ERF mutations reveal a balance of ETS factors controlling prostate oncogenesis. Nature 546, 671–675 (2017)."),[17](https://www.nature.com/articles/s41588-025-02289-w#ref-CR17 "Li, F. et al. ERG orchestrates chromatin interactions to drive prostate cell fate reprogramming. J. Clin. Invest. 130, 5924–5941 (2020)."),[27](https://www.nature.com/articles/s41588-025-02289-w#ref-CR27 "Wasmuth, E. V. et al. Modulation of androgen receptor DNA binding activity through direct interaction with the ETS transcription factor ERG. Proc. Natl Acad. Sci. USA 117, 8584–8592 (2020)."). But the question of how activation of this pro-luminal program is coupled with oncogenesis remains unclear. Cooperativity of ERG with androgen receptor (AR) has been documented at the level of chromatin binding but the consequences on AR target gene activation can be activating, re-directing or inhibitory depending on the cellular context[3](https://www.nature.com/articles/s41588-025-02289-w#ref-CR3 "Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. Nat. Med. 19, 1023–1029 (2013)."),[10](https://www.nature.com/articles/s41588-025-02289-w#ref-CR10 "Bose, R. et al. ERF mutations reveal a balance of ETS factors controlling prostate oncogenesis. Nature 546, 671–675 (2017)."),[27](https://www.nature.com/articles/s41588-025-02289-w#ref-CR27 "Wasmuth, E. V. et al. Modulation of androgen receptor DNA binding activity through direct interaction with the ETS transcription factor ERG. Proc. Natl Acad. Sci. USA 117, 8584–8592 (2020)."),[29](https://www.nature.com/articles/s41588-025-02289-w#ref-CR29 "Yu, J. et al. An integrated network of androgen receptor, polycomb, and TMPRSS2-ERG gene fusions in prostate cancer progression. Cancer Cell 17, 443–454 (2010)."),[30](https://www.nature.com/articles/s41588-025-02289-w#ref-CR30 "The Cancer Genome Atlas Research Network. The molecular taxonomy of primary prostate cancer. Cell 163, 1011–1025 (2015)."). Of note, datasets in these earlier studies were generated using bulk tissue methodologies, raising the possibility that critical changes relevant to tumor initiation may be overlooked if they occur only in subpopulations of cells. Lineage tracing and tissue recombination studies have shown that tumors can initiate in basal cells or luminal cells[31](https://www.nature.com/articles/s41588-025-02289-w#ref-CR31 "Goldstein, A. S., Huang, J., Guo, C., Garraway, I. P. & Witte, O. N. Identification of a cell of origin for human prostate cancer. Science 329, 568–571 (2010)."),[32](https://www.nature.com/articles/s41588-025-02289-w#ref-CR32 "Lawson, D. A. et al. Basal epithelial stem cells are efficient targets for prostate cancer initiation. Proc. Natl Acad. Sci. USA 107, 2610–2615 (2010)."),[33](https://www.nature.com/articles/s41588-025-02289-w#ref-CR33 "Lu, T. L. et al. Conditionally ablated Pten in prostate basal cells promotes basal-to-luminal differentiation and causes invasive prostate cancer in mice. Am. J. Pathol. 182, 975–991 (2013)."),[34](https://www.nature.com/articles/s41588-025-02289-w#ref-CR34 "Park, J. W. et al. Prostate epithelial cell of origin determines cancer differentiation state in an organoid transformation assay. Proc. Natl Acad. Sci. USA 113, 4482–4487 (2016)."),[35](https://www.nature.com/articles/s41588-025-02289-w#ref-CR35 "Stoyanova, T. et al. Prostate cancer originating in basal cells progresses to adenocarcinoma propagated by luminal-like cells. Proc. Natl Acad. Sci. USA 110, 20111–20116 (2013)."),[36](https://www.nature.com/articles/s41588-025-02289-w#ref-CR36 "Choi, N., Zhang, B., Zhang, L., Ittmann, M. & Xin, L. Adult murine prostate basal and luminal cells are self-sustained lineages that can both serve as targets for prostate cancer initiation. Cancer Cell 21, 253–265 (2012)."),[37](https://www.nature.com/articles/s41588-025-02289-w#ref-CR37 "Wang, X. et al. A luminal epithelial stem cell that is a cell of origin for prostate cancer. Nature 461, 495–500 (2009)."),[38](https://www.nature.com/articles/s41588-025-02289-w#ref-CR38 "Wang, Z. A. et al. Lineage analysis of basal epithelial cells reveals their unexpected plasticity and supports a cell-of-origin model for prostate cancer heterogeneity. Nat. Cell Biol. 15, 274–283 (2013)."),[39](https://www.nature.com/articles/s41588-025-02289-w#ref-CR39 "Wang, Z. A., Toivanen, R., Bergren, S. K., Chambon, P. & Shen, M. M. Luminal cells are favored as the cell of origin for prostate cancer. Cell Rep. 8, 1339–1346 (2014)."),[40](https://www.nature.com/articles/s41588-025-02289-w#ref-CR40 "Yoo, Y. A. et al. Bmi1 marks distinct castration-resistant luminal progenitor cells competent for prostate regeneration and tumour initiation. Nat. Commun. 7, 12943 (2016)."). In the context of ERG translocations, these cell of origin distinctions could be relevant in defining the critical cellular context required for ERG function and uncover dependencies. Recent advances in multiomic technology provide an opportunity to explore these questions at a single-cell level.

As a first step, we used a combination of mouse organoid transplantation and lineage tracing to address whether ERG-mediated prostate tumorigenesis initiates in basal cells or luminal cells. Both approaches implicate basal cells as the preferred cell of origin, with the important nuance that initiation occurs in a rare subpopulation of basal cells, which we call BasalLum, based on coexpression of a subset of luminal marker genes including _Tmprss2_ and _Nkx3.1_. Rare ERG-expressing BasalLum cells are also detectable in ERG+ human PCa. Within weeks following ERG activation, BasalLum cells give rise to a highly proliferative population of IM cells expressing basal and luminal genes, which subsequently give rise to a larger population of mature luminal cells that account for the bulk of invasive PCa seen in mice. Through single-cell transcriptomic and chromatin accessibility profiling, we find these ERG+ IM cells have a unique, ERG-specific chromatin landscape with increased chromatin accessibility of binding sites for the STAT3 transcription factor (TF) and elevated expression of two histone methyltransferase genes, _Kmt2/Mll1_ and _Dot1l_. In orthotopic transplantation assays, we find that STAT3, KMT2/MLL1 and DOT1L are all required for ERG-dependent prostate tumorigenicity in vivo, raising the possibility of pharmacologic intervention using existing clinical-grade inhibitors[41](https://www.nature.com/articles/s41588-025-02289-w#ref-CR41 "Stein, E. M. et al. The DOT1L inhibitor pinometostat reduces H3K79 methylation and has modest clinical activity in adult acute leukemia. Blood 131, 2661–2669 (2018)."),[42](https://www.nature.com/articles/s41588-025-02289-w#ref-CR42 "Tsimberidou, A. M. et al. Phase 1 trial of TTI-101, a first-in-class oral inhibitor of STAT3 in patients with advanced solid tumors. Clin. Cancer Res. 31, 965–974 (2025)."). Importantly, expression signatures derived from IM cells in ERG+ genetically engineered mouse models (GEMMs) are enriched in ERG+ human PCa and correlate with shorter disease-free survival, suggesting that cell of origin of the ERG translocation may impact outcome.

Results
-------

### ERG-driven prostate cancer preferentially initiates in basal cells

We previously reported a robust mouse model of ERG-driven PCa (_Rosa26-ERG__LSL/LSL__; Pten__flox/flox__;Pb-Cre4_, hereafter called _EPC_)[3](https://www.nature.com/articles/s41588-025-02289-w#ref-CR3 "Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. Nat. Med. 19, 1023–1029 (2013).") in which ERG is expressed in basal and luminal prostate epithelial cells. To explore whether these tumors preferentially arise from basal or luminal cells (or both), we used a previously reported method to interrogate cell of origin by collecting primary mouse prostate tissue, sorting into luminal and basal subpopulations, genetically manipulating each subpopulation ex vivo, then immediately (same day) reintroducing these cells orthotopically to score subsequent tumor formation[43](https://www.nature.com/articles/s41588-025-02289-w#ref-CR43 "Feng, W. et al. Rapid interrogation of cancer cell of origin through CRISPR editing. Proc. Natl Acad. Sci. USA 118, e2110344118 (2021)."). To address the cell of origin using ERG, we isolated luminal and basal cells from multiple lobes of the prostates of _Rosa26-ERG__LSL/LSL__; Pten__flox/flox_ (_EP_) mice, activated the _EP_ genotype by infection with Ad-Cre virus and performed orthotopic implantations (Fig. [1a](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). Post-sort analysis confirmed >85% purity of the sorted populations preimplantation and >90% Cre recombination efficiency (Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02289-w#Fig9)). Although luminal cells are intrinsically fragile, our approach enabled successful engraftment of luminal-derived orthografts as indicated by the detection of ERG+ cells after 5 months. However, ERG+ cells were present at a much higher rate in basal-derived grafts (~45% versus ~11%) (Fig. [1b](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). Furthermore, only the basal-derived orthografts developed invasive adenocarcinomas (6 of 6) (Fig. [1c](https://www.nature.com/articles/s41588-025-02289-w#Fig1) and Extended Data Fig. [1b,c](https://www.nature.com/articles/s41588-025-02289-w#Fig9)). Histologically, the basal-derived orthografts displayed luminal morphology, with glandular architecture and evidence of nuclear and nucleolar enlargement. By contrast, luminal-derived orthografts were smaller, less penetrant (4 of 6) and displayed benign histology (Fig. [1c](https://www.nature.com/articles/s41588-025-02289-w#Fig1) and Extended Data Fig. [1b,c](https://www.nature.com/articles/s41588-025-02289-w#Fig9)). These findings are consistent with earlier work using human prostate epithelial cells, which showed superior engraftment of basal cells in tissue recombination experiments following infection with lentivirus expressing ERG + activated AKT + AR (ref. [31](https://www.nature.com/articles/s41588-025-02289-w#ref-CR31 "Goldstein, A. S., Huang, J., Guo, C., Garraway, I. P. & Witte, O. N. Identification of a cell of origin for human prostate cancer. Science 329, 568–571 (2010).")).

**Fig. 1: ERG-driven prostate cancer preferentially initiates from an NKX3.1+ subset of basal cells but not luminal cells.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/1)

**a**, Orthotopic transplantation comparing _EP_ orthografts freshly derived from basal versus luminal origin. **b**, Flow cytometry quantifying ERG+ cells in grafts collected at the 5-month endpoint (_n_ = 6 mice per group). **c**, Histological analysis of orthografts at 5-month endpoint. Inset: a high-power view. Scale bars, 100 µm. **d**, An intraprostatic adenoviral (Ad) injection approach to compare _EP_ activation by Ad-K8-Cre versus Ad-K5-Cre. **e**,**f**, Histology (**e**) and characterization (**f**) of _EP_ prostates collected at 40 weeks postinfection (_n_ = 13 mice per group). Inset: a high-power view. PCa is defined as invasive adenocarcinoma. Scale bars, 100 µm. **g**, UMAP of epithelial cells from normal human prostates[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022).") (cell types annotated in circles). Cells from normal samples are colored based on gene expression, with cells from tumor samples in gray. **h**, Intracellular flow cytometry quantifying YFP+ basal cells (_n_ = 4 mice). **i**, Prostate IF highlighting YFP+K5+ cells. Inset: a high-power view. Scale bars, 25 µm. **j**, Diagram highlighting distinct morphology and cytokeratin expression of normal prostate epithelial cell types. **k**, UMAP of epithelial cells from patients[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022).") highlighting the presence of ERG+ cells, as colored by ERG expression (left) and patient ID (right). ERG+ PCa samples are shown ([Methods](https://www.nature.com/articles/s41588-025-02289-w#Sec11)). Basal and PCa cells are annotated in black and pink circles/arrows, respectively. **l**, Lineage tracing activating _EP_ from indicated population. **m**, Flow cytometry quantifying ERG+ cells from indicated mice (left to right: _n_ = 3, 2, 4, 8, 8, 6 mice). **n**, Prostate IF. Dashed line encircles clusters of K5+ERG+ cells. Arrows highlight the K5-single-positive (K8/18-negative) cells at the gland periphery. Inset: a high-power view. Scale bars, 50 µm. **o**, Prostate histology. High-power view (insets) highlights cell morphology. Scale bars, 500 µm. **p**, PCa-free survival analysis (top to bottom, _n_ = 20, 12, 19 mice). PCa is defined as invasive adenocarcinoma originating from any region of the prostate. Data represent mean ± s.d.; unpaired two-tailed _t_\-test (**b**); chi-squared test (**f**); two-way ANOVA with Tukey posttest (**m**); log-rank test (**p**). ANOVA, analysis of variance; Bas, basal; BasLum, BasalLum; DN, double-negative; DP, double-positive; H&E, hematoxylin and eosin; IF, immunofluorescence; Lum, luminal; mo, month(s); NS, not significant; SPbas, single-positive for basal markers; SPlum, single-positive for luminal markers; Tam, tamoxifen; wk, week(s).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM5)

Having implicated basal cells as cell of origin in this transplantation assay, we turned to conventional lineage tracing approaches to validate our findings in an autochthonous model. Due to a severe, early-onset skin phenotype in _EP_ mice crossed with _K5-CreER__T2_ or _K14-CreER_ mice (Extended Data Fig. [1d–j](https://www.nature.com/articles/s41588-025-02289-w#Fig9) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02289-w#MOESM1)), we delivered Cre locally by intraprostatic injection of adenoviruses expressing Cre downstream of K5 (Ad-K5-Cre), together with a parallel set of experiments targeting luminal cells using Ad-K8-Cre virus (Fig. [1d](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). To evaluate the fidelity and robustness of Cre delivery, we used _Rosa26-YFP__LSL/LSL_ reporter mice as a control. At 1 week after injection of Ad-K5-Cre, we found all YFP+ cells were K5+ with a flat-shaped basal morphology. By contrast, nearly all YFP+ cells in Ad-K8-Cre-injected mice were K8/18+ with a columnar luminal morphology (Extended Data Fig. [1k](https://www.nature.com/articles/s41588-025-02289-w#Fig9)). Having demonstrated the lineage specificity of the Ad-K5 and Ad-K8-Cre viruses, we performed similar injections in _EP_ mice (two injections per mouse into either anterior or dorsal lobes), then followed each cohort for disease onset. In Ad-K5-Cre-injected mice, expansion of ERG+ cells occurred at 12 of 13 injection sites, with histologic evidence of invasive adenocarcinoma in half (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02289-w#Fig1) and Extended Data Fig. [1l](https://www.nature.com/articles/s41588-025-02289-w#Fig9)). Despite targeting ERG activation in K5+ basal cells, the ERG+ invasive cells had luminal morphology with glandular architecture (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). By contrast, we did not observe consistent expansion or invasion of ERG+ cells in mice injected with Ad-K8-Cre virus (small foci were seen in 2 of 13 mice) (Fig. [1e,f](https://www.nature.com/articles/s41588-025-02289-w#Fig1) and Extended Data Fig. [1l](https://www.nature.com/articles/s41588-025-02289-w#Fig9)). Thus, both the transplantation and in situ models implicate basal cells as the preferred cell of origin for ERG-driven PCa, whereas luminal cells support limited, noninvasive cell expansion.

### A rare ERG+ basal subset exists in ERG+ human prostate cancer

ERG expression in ERG+ human PCa is under the control of the TMPRSS2 regulatory locus, a gene highly expressed in normal luminal epithelial cells. Having shown that ERG-driven mouse PCa initiates in basal cells (using a R26 lox-STOP-lox cassette), we explored the human relevance of this result by asking whether TMPRSS2 is expressed in basal cells by examining single-cell RNA sequencing (scRNA-seq) datasets of normal mouse and human PCa. As expected[44](https://www.nature.com/articles/s41588-025-02289-w#ref-CR44 "Norris, J. D. et al. The homeodomain protein HOXB13 regulates the cellular response to androgens. Mol. Cell 36, 405–416 (2009)."),[45](https://www.nature.com/articles/s41588-025-02289-w#ref-CR45 "Wang, Q. et al. A hierarchical network of transcription factors governs androgen receptor-dependent prostate cancer growth. Mol. Cell 27, 380–392 (2007)."),[46](https://www.nature.com/articles/s41588-025-02289-w#ref-CR46 "Chen, Y. W. et al. TMPRSS2, a serine protease expressed in the prostate on the apical surface of luminal epithelial cells and released into semen in prostasomes, is misregulated in prostate cancer cells. Am. J. Pathol. 176, 2986–2996 (2010)."),[47](https://www.nature.com/articles/s41588-025-02289-w#ref-CR47 "Gao, D. et al. A Tmprss2-CreERT2 knock-in mouse model for cancer genetic studies on prostate and colon. PLoS ONE 11, e0161084 (2016)."), TMPRSS2 was uniformly expressed in nearly all luminal subpopulations. However, we also noted small numbers of TMPRSS2+ cells residing in basal clusters in both human and mouse normal prostates. Furthermore, these basal cells express other canonical luminal genes such as _NKX3-1_, _NDRG1_, _FKBP5_, _KLK3_ and _Pbsn_ (Fig. [1g](https://www.nature.com/articles/s41588-025-02289-w#Fig1) and Extended Data Fig. [2a–c](https://www.nature.com/articles/s41588-025-02289-w#Fig10)), indicative of a mixed identity. Of note, expression of these genes is regulated by androgen in basal cells, just as seen in luminal cells[48](https://www.nature.com/articles/s41588-025-02289-w#ref-CR48 "Karthaus, W. R. et al. Regenerative potential of prostate luminal cells revealed by single-cell analysis. Science 368, 497–505 (2020).") (Extended Data Fig. [2d](https://www.nature.com/articles/s41588-025-02289-w#Fig10)).

To provide additional evidence for luminal gene expression in basal cells, we crossed _Nkx3-1-_CreERT2 mice into the _Rosa26-YFP__LSL/LSL_ background. The largest population of YFP+ cells were luminal (K8/18+), but a subset of basal cells (K5+) clearly express YFP (Fig. [1h](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). Because these K5+, P63+, YFP+ cells have a flat morphology typical of basal cells yet are adjacent to luminal cells at the basement membrane (Fig. [1i](https://www.nature.com/articles/s41588-025-02289-w#Fig1) and Extended Data Fig. [2e](https://www.nature.com/articles/s41588-025-02289-w#Fig10)), we refer to them henceforth as BasalLum cells (Fig. [1j](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). Basal cells with a similar profile of luminal gene expression (called Basal-B or LumI) were recently reported by others[49](https://www.nature.com/articles/s41588-025-02289-w#ref-CR49 "Guo, W. et al. JAK/STAT signaling maintains an intermediate cell population during prostate basal cell fate determination. Nat. Genet. 56, 2776–2789 (2024)."),[50](https://www.nature.com/articles/s41588-025-02289-w#ref-CR50 "Luo, F. et al. A luminal intermediate cell state maintains long-term prostate homeostasis and contributes to tumorigenesis. Preprint at bioRxiv 
https://doi.org/10.1101/2023.02.24.529762
(2023).").

We next asked whether ERG+ human PCa, which typically display uniform ERG expression in luminal cells, also have evidence of ERG expression in rare basal cells by annotating ERG and TMPRSS2 expression in two independent published scRNA-seq datasets of localized PCa (total PCa _n_ = 29; ERG-positive PCa _n_ = 9)[51](https://www.nature.com/articles/s41588-025-02289-w#ref-CR51 "Hirz, T. et al. Dissecting the immune suppressive human prostate tumor microenvironment via integrated single-cell and spatial transcriptomic analyses. Nat. Commun. 14, 663 (2023)."),[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022)."). As expected, large clusters of ERG+ luminal cancer cells are present in the ERG+ patients in both datasets, but we also observed rare ERG+ cells in the corresponding basal clusters of these same patients (Fig. [1k](https://www.nature.com/articles/s41588-025-02289-w#Fig1) and Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02289-w#Fig11)). Basal cell expression of TMPRSS2 has been previously documented in normal human prostate tissues, in primary cultures derived from ERG-positive human tumors and in mice engineered to express ERG from human _TMPRSS2_ BAC construct[53](https://www.nature.com/articles/s41588-025-02289-w#ref-CR53 "Casey, O. M. et al. TMPRSS2-driven ERG expression in vivo increases self-renewal and maintains expression in a castration resistant subpopulation. PLoS ONE 7, e41668 (2012)."),[54](https://www.nature.com/articles/s41588-025-02289-w#ref-CR54 "Polson, E. S. et al. Monoallelic expression of TMPRSS2/ERG in prostate cancer stem cells. Nat. Commun. 4, 1623 (2013)."),[55](https://www.nature.com/articles/s41588-025-02289-w#ref-CR55 "Lin, B. et al. Prostate-localized and androgen-regulated expression of the membrane-bound serine protease TMPRSS2. Cancer Res. 59, 4180–4184 (1999).").

### ERG drives expansion of Nkx3.1+ basal cells

Having implicated basal cells as a preferred cell of origin for ERG-positive PCa and demonstrated _Tmprss2_ expression in a subset of these basal cells (BasalLum), we next asked whether these tumors originate from BasalLum cells using the _Nkx3-1-_CreERT2 driver allele[38](https://www.nature.com/articles/s41588-025-02289-w#ref-CR38 "Wang, Z. A. et al. Lineage analysis of basal epithelial cells reveals their unexpected plasticity and supports a cell-of-origin model for prostate cancer heterogeneity. Nat. Cell Biol. 15, 274–283 (2013)."),[56](https://www.nature.com/articles/s41588-025-02289-w#ref-CR56 "Aytes, A. et al. ETV4 promotes metastasis in response to activation of PI3-kinase and Ras signaling in a mouse model of advanced prostate cancer. Proc. Natl Acad. Sci. USA 110, E3506–E3515 (2013)."),[57](https://www.nature.com/articles/s41588-025-02289-w#ref-CR57 "Zou, M. et al. Transdifferentiation as a mechanism of treatment resistance in a mouse model of castration-resistant prostate cancer. Cancer Discov. 7, 736–749 (2017)."). Because _Nkx3.1_ is expressed in luminal cells as well as a subpopulation of basal cells[37](https://www.nature.com/articles/s41588-025-02289-w#ref-CR37 "Wang, X. et al. A luminal epithelial stem cell that is a cell of origin for prostate cancer. Nature 461, 495–500 (2009)."), we compared the phenotype of ERG expression in _Nkx3-1-_CreERT2 mice with that seen using the luminal-restricted _K8-CreER__T2_ driver to provide insight into consequences of ERG expression in Nkx3.1+ BasalLum cells (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02289-w#Fig9))[58](https://www.nature.com/articles/s41588-025-02289-w#ref-CR58 "Van Keymeulen, A. et al. Distinct stem cells contribute to mammary gland development and maintenance. Nature 479, 189–193 (2011)."),[59](https://www.nature.com/articles/s41588-025-02289-w#ref-CR59 "Zhang, L. et al. Targeting CreERT2 expression to keratin 8-expressing murine simple epithelia using bacterial artificial chromosome transgenesis. Transgenic Res. 21, 1117–1123 (2012)."). Within 1 month of tamoxifen induction, ERG+ cells expanded more than threefold (~5% to ~18% at 1 month) in _EP;Nkx3-1-CreER__T2_ mice but not in _EP; K8-CreER__T2_ mice (Fig. [1l,m](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). Initially, ERG+ basal cells in _EP;Nkx3-1-CreER__T2_ mice are individual, flat-shaped, K5+ cells adjacent to the basement membrane, but, by 1 month, progress to clusters of K5+ single-positive and K5+, K8/18+ double-positive cells (Fig. [1n](https://www.nature.com/articles/s41588-025-02289-w#Fig1)), which we call IM cells based on previous use of this term to describe rare (<1%) K5+/K8+ double-positive cells in normal prostate tissue[60](https://www.nature.com/articles/s41588-025-02289-w#ref-CR60 "Hudson, D. L. et al. Epithelial cell differentiation pathways in the human prostate: identification of intermediate phenotypes by keratin expression. J. Histochem. Cytochem. 49, 271–278 (2001)."),[61](https://www.nature.com/articles/s41588-025-02289-w#ref-CR61 "Peehl, D. M., Sellers, R. G. & McNeal, J. E. Keratin 19 in the adult human prostate: tissue and cell culture studies. Cell Tissue Res. 285, 171–176 (1996)."),[62](https://www.nature.com/articles/s41588-025-02289-w#ref-CR62 "Robinson, E. J., Neal, D. E. & Collins, A. T. Basal cells are progenitors of luminal cells in primary cultures of differentiating human prostatic epithelium. Prostate 37, 149–160 (1998)."),[63](https://www.nature.com/articles/s41588-025-02289-w#ref-CR63 "Taylor, R. A., Toivanen, R. & Risbridger, G. P. Stem cells in prostate cancer: treating the root of the problem. Endocr. Relat. Cancer 17, R273–R285 (2010)."),[64](https://www.nature.com/articles/s41588-025-02289-w#ref-CR64 "van Leenders, G., Dijkman, H., Hulsbergen-van de Kaa, C., Ruiter, D. & Schalken, J. Demonstration of intermediate cells during human prostate epithelial differentiation in situ and in vitro using triple-staining confocal scanning microscopy. Lab. Invest. 80, 1251–1258 (2000)."),[65](https://www.nature.com/articles/s41588-025-02289-w#ref-CR65 "Verhagen, A. P. et al. Colocalization of basal and luminal cell-type cytokeratins in human prostate cancer. Cancer Res. 52, 6182–6187 (1992)."),[66](https://www.nature.com/articles/s41588-025-02289-w#ref-CR66 "Wang, Y., Hayward, S., Cao, M., Thayer, K. & Cunha, G. Cell differentiation lineage in the prostate. Differentiation 68, 270–279 (2001)."),[67](https://www.nature.com/articles/s41588-025-02289-w#ref-CR67 "Xue, Y., Smedts, F., Debruyne, F. M., de la Rosette, J. J. & Schalken, J. A. Identification of intermediate cell types by keratin expression in the developing human prostate. Prostate 34, 292–301 (1998)."). After 9 months, all _Nkx3-1-CreER__T2_;_EP_ mice developed invasive adenocarcinomas featuring luminal histology (Fig. [1o,p](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). By contrast, ERG+ cells did not expand in mice with luminal-restricted ERG expression (_EP_; _K8-CreER__T2_) or generate invasive cancer after 9–12 months despite early and sustained luminal-specific ERG induction throughout the entire observation period (Fig. [1m,o,p](https://www.nature.com/articles/s41588-025-02289-w#Fig1)). The only histologic changes in _EP_; _K8-CreER__T2_ mice were hyperplasia and PIN in nearly all mice (Fig. [1o](https://www.nature.com/articles/s41588-025-02289-w#Fig1)), a phenotype seen after _Pten_ loss alone[33](https://www.nature.com/articles/s41588-025-02289-w#ref-CR33 "Lu, T. L. et al. Conditionally ablated Pten in prostate basal cells promotes basal-to-luminal differentiation and causes invasive prostate cancer in mice. Am. J. Pathol. 182, 975–991 (2013)."),[36](https://www.nature.com/articles/s41588-025-02289-w#ref-CR36 "Choi, N., Zhang, B., Zhang, L., Ittmann, M. & Xin, L. Adult murine prostate basal and luminal cells are self-sustained lineages that can both serve as targets for prostate cancer initiation. Cancer Cell 21, 253–265 (2012)."),[38](https://www.nature.com/articles/s41588-025-02289-w#ref-CR38 "Wang, Z. A. et al. Lineage analysis of basal epithelial cells reveals their unexpected plasticity and supports a cell-of-origin model for prostate cancer heterogeneity. Nat. Cell Biol. 15, 274–283 (2013)."). These results, as well as additional experiments comparing the efficiency of different Cre drivers (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02289-w#Fig12) and Supplementary Note [2](https://www.nature.com/articles/s41588-025-02289-w#MOESM1)), lead us to conclude that ERG+ cancers preferentially initiate in Nkx3.1+ BasalLum cells.

### ERG drives basal–luminal transition via an IM state

We next sought to understand why the histologic phenotype of these cancers is nearly exclusively luminal by examining the relative proportions of ERG+ basal (K5+) and luminal (K8+) cells at different timepoints in the mouse models discussed above (Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02289-w#MOESM1)). In the fresh cell transplantation model, ERG+ cells in basal cell-derived tumors contained a mix of K8+ single-positive and K5+/K8+ double-positive IM cells (Fig. [2a](https://www.nature.com/articles/s41588-025-02289-w#Fig2)). The same IM population was evident 1 month after ERG activation in the Nkx3.1-CreERT2 lineage tracing model (where ERG is induced in luminal and BasalLum cells) (Fig. [2b](https://www.nature.com/articles/s41588-025-02289-w#Fig2)), but not in the K8-CreERT2 model where ERG expression is limited to luminal cells (Fig. [2c](https://www.nature.com/articles/s41588-025-02289-w#Fig2) and Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02289-w#Fig13)). Furthermore, IM cells in the transplantation model are highly proliferative, with a larger fraction of Ki67+ cells (40–50%) compared with K8 single-positive cells (SPLum) (25–30%) (Fig. [2d,e](https://www.nature.com/articles/s41588-025-02289-w#Fig2)). Coexpression of luminal (K8, K18, AR) and basal genes (K5) in IM cells was confirmed by immunofluorescence and displayed luminal morphology (Fig. [2f–h](https://www.nature.com/articles/s41588-025-02289-w#Fig2) and Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02289-w#Fig13)). IM cells were also detected in invasive adenocarcinoma lesions, but only following ERG activation in basal cells (Fig. [2f–h](https://www.nature.com/articles/s41588-025-02289-w#Fig2) and Extended Data Fig. [5c](https://www.nature.com/articles/s41588-025-02289-w#Fig13)). IM cells in the _EPC_ mice and those with _Pten_ deletion alone (_Pten__flox/flox__;Pb-Cre4_, hereafter called _PC_) showed similar features but only those generated in the _EPC_ model result in invasion (Fig. [2i–k](https://www.nature.com/articles/s41588-025-02289-w#Fig2), Extended Data Fig. [5d–l](https://www.nature.com/articles/s41588-025-02289-w#Fig13) and Supplementary Note [3](https://www.nature.com/articles/s41588-025-02289-w#MOESM1)). We conclude that basal cells generate highly proliferative IM cells in response to ERG activation (in the setting of PTEN loss).

**Fig. 2: Basal cells expand into highly proliferative basal–luminal IM cells in cancer.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/2)

**a**, Flow cytometry quantification of indicated populations from orthografts from Fig. [1a](https://www.nature.com/articles/s41588-025-02289-w#Fig1). The DP and SPbas quadrants in ERG+ cells form one continuous population and are hereafter grouped together as IM cells. **b**, Flow cytometry (top) and quantification (bottom) of populations from ERG+ cells from whole prostates of indicated mice (_n_ = 8 mice per group). **c**, Flow cytometry quantifying luminal populations of ERG+ cells from whole prostates of indicated mice (left to right: _n_ = 3, 2, 4, 6, 2 mice). **d**, Flow cytometry quantifying Ki67+ cells from ERG+ populations of basal-derived _EP_ orthografts from **a** at the 5-month endpoint (_n_ = 6 mice). **e**, Flow cytometry quantifying Ki67+ cells from the ERG+ population of indicated mice (_n_ = 6 mice per group). **f**, IF images of indicated _EP_ orthografts from **a** at the 5-month endpoint. Inset: a high-power view. Scale bars, 100 µm. **g**, IF images of indicated mice. Inset: a high-power view. Whole prostates were collected at 9 months after tamoxifen. Scale bars, 100 µm. **h**, IF of _EP_ prostates collected at 40 weeks after infection. Inset: a high-power view. PCa from **f**–**h** is defined as invasive adenocarcinoma. Scale bars, 20 µm. **i**, Flow cytometry (left) and quantification (right) highlighting the expansion of IM populations in _Pten__flox/flox__;Rosa26-YFP__LSL/LSL__;Pb-_Cre4 (_PYC_) and _EPC_ mice (left to right: _n_ = 3, 5, 11 mice). Whole prostate cells were collected at 3 months of age. Recombined cells labeled by YFP (_YC/PYC_) or ERG (_EPC_) were analyzed. **j**, Flow cytometry quantification of Ki67+ cells from indicated population (_n_ = 7 mice). Whole prostate cells were collected at 3 months of age. **k**, IF staining showing expression of ERG (left) and AR (right) in both K5− luminal and K5+ IM cells from invasive adenocarcinomas. Whole prostate tissues were collected from 3-month _EPC_ mice. Inset: a high-power view. Scale bars, 50 µm. Data represent mean ± s.d.; unpaired two-tailed _t_\-test (**b**,**d**); one-way ANOVA (**c**); two-way ANOVA with Sidak posttest (**e**); one-way ANOVA with Tukey posttest (**i**). _YC_, _Rosa26-YFP__LSL/LSL__;Pb-Cre4_.

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM6)

To assess the lineage relationships between IM and luminal cells, we used in vivo 5-ethynyl-2′-deoxyuridine (EdU) labeling of _EPC_ mice with early cancers (age 2 months) to trace the proliferative cells. As expected from their elevated proliferation rate, ERG+ IM cells were preferentially labeled over luminal cells after a brief (2-h) EdU pulse. However, within 1 week of chase, the label shifted to K8+, K5-negative, ERG+ luminal cells, indicating that ERG+ IM cells give rise to ERG+ luminal cells (Fig. [3a](https://www.nature.com/articles/s41588-025-02289-w#Fig3) and Extended Data Fig. [6a–c](https://www.nature.com/articles/s41588-025-02289-w#Fig14)). To provide direct visual evidence for this transition, we identified ~300 EdU-labeled cell doublets (dividing cells), of which ~26% showed asymmetric divisions, always with one DP daughter cell paired with an SPlum or SPbas daughter cell, suggestive of a fate transition within one cell division. Approximately 80% of these asymmetric divisions consisted of DP–SPlum pairs, providing further evidence that DP cells give rise to SPlum cells (Fig. [3b–c](https://www.nature.com/articles/s41588-025-02289-w#Fig3)). Additional experiments using prostate organoid cultures confirm luminal lineage specification by ERG induction in basal cells (Fig. [3d–i](https://www.nature.com/articles/s41588-025-02289-w#Fig3), Extended Data Fig. [6d–l](https://www.nature.com/articles/s41588-025-02289-w#Fig14), Supplementary Table [1](https://www.nature.com/articles/s41588-025-02289-w#MOESM4) and Supplementary Note [4](https://www.nature.com/articles/s41588-025-02289-w#MOESM1)). Collectively, ERG-driven cancers initiate in basal cells, which expand initially as highly proliferative IM cells then transition to luminal cells that typify human PCa.

**Fig. 3: IM and basal cells transit to a luminal fate in the presence of ERG.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/3)

**a**, EdU pulse–chase assay on _EPC_ mice at 2 months of age (schematic on the left). Indicated lineage representation was analyzed from whole prostate cells on the right (_n_ = 5 mice for pulse, _n_ = 4 mice for chase). The EdU+ population of ERG+ cells (normalized to all ERG+ cells) shows an increase of SPlum cells and decrease of IM after the chase. **b**, Schematic (top) and representative IF images (bottom) exemplifying symmetric and asymmetric divisions of EdU-labeled cells using _EPC_ mice from **a**. Scale bars, 5 µm. **c**, The cell type composition of EdU+ doublets highlights that DP cells are present in all asymmetric divisions and preferentially associate with an SPlum daughter cell (_n_ = 7 mice). **d**, Schematic showing in vitro generation of an isogenic pair of _EP_ and _P_ organoids. **e**,**f**, EdU pulse–chase assay in organoids followed by flow cytometry assessment highlights an ERG-dependent shift of the EdU-labeled cells towards a luminal (SPlum) fate (_n_ = 3 independent assays for pulse, _n_ = 2 independent assays for chase). Representative data (**e**) and quantification of the luminal fraction in EdU-labeled populations (**f**) are shown. **g**, Left: schematic showing _EP_ organoids with a dual lineage reporter knocked in to form C-terminal fusions with endogenous K5 and K8. Middle: live cell imaging highlighting the expected spatial expression pattern of the reporter signals, which enables live sort of basal and luminal cells (schematic on the right). Scale bar, 20 µm. **h**, The basal- and luminal-derived _EP_ reporter organoids from **g** were analyzed by flow cytometry based on the reporter signals along the time course postsort (_n_ = 3 independent assays). An ERG-dependent luminal fate transition from basal cells from luminal cells was observed. **i**, GSEA showing ERG-regulated pathways in basal cells. Data represent mean ± s.d.; one-way ANOVA with Tukey posttest (**a**); two-sided Fisher’s exact test (**c**); two-way ANOVA (**h**). mNG, mNeonGreen; NES, normalized enrichment score; RFP, TagRFP.

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM7)

### IM cells represent a multi-lineage, stem-like state

Having implicated BasalLum cells as the cell of origin for ERG-driven PCa, we next turned to single-cell analysis to gain insight into the gene expression and chromatin landscape changes in these cells following ERG induction. We selected _EPC_ mice for these studies given the potent ERG-driven phenotype and ERG expression in basal and luminal cells, which ensures representation of both cell types in the analysis. At age 3 months, EPC mice have small foci of invasive cancer which progress within 6 months to highly penetrant glandular invasion (Extended Data Fig. [7a](https://www.nature.com/articles/s41588-025-02289-w#Fig15)). By contrast, _PC_ mice develop intraductal hyperplasia and/or PIN over this 3–6-month time frame[3](https://www.nature.com/articles/s41588-025-02289-w#ref-CR3 "Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. Nat. Med. 19, 1023–1029 (2013)."),[68](https://www.nature.com/articles/s41588-025-02289-w#ref-CR68 "Chen, Z. et al. Crucial role of p53-dependent cellular senescence in suppression of Pten-deficient tumorigenesis. Nature 436, 725–730 (2005).").

To capture early neoplastic changes, we performed scRNA-seq of whole prostate tissue from _EPC_, _PC_ and _WT_ mice at age 3 months, which revealed genotype-specific differences in epithelial and nonepithelial populations (Fig. [4a–c](https://www.nature.com/articles/s41588-025-02289-w#Fig4) and Extended Data Fig. [7b–f](https://www.nature.com/articles/s41588-025-02289-w#Fig15)). For example, _EPC_ mice have much more pronounced myeloid infiltration than _PC_ and _WT_ mice[69](https://www.nature.com/articles/s41588-025-02289-w#ref-CR69 "Brina, D. et al. The Akt/mTOR and MNK/eIF4E pathways rewire the prostate cancer translatome to secrete HGF, SPP1 and BGN and recruit suppressive myeloid cells. Nat. Cancer 4, 1102–1121 (2023).") (Extended Data Fig. [7d–f](https://www.nature.com/articles/s41588-025-02289-w#Fig15)). Focusing on the epithelial cells (Epcam+), we noted that large clusters of secretory luminal cells, which account for >90% of luminal cells in _WT_ mice (called Lum\_L1)[48](https://www.nature.com/articles/s41588-025-02289-w#ref-CR48 "Karthaus, W. R. et al. Regenerative potential of prostate luminal cells revealed by single-cell analysis. Science 368, 497–505 (2020)."),[70](https://www.nature.com/articles/s41588-025-02289-w#ref-CR70 "Crowley, L. et al. A single-cell atlas of the mouse and human prostate reveals heterogeneity and conservation of epithelial progenitors. eLife 9, e59465 (2020)."),[71](https://www.nature.com/articles/s41588-025-02289-w#ref-CR71 "Guo, W. et al. Single-cell transcriptomics identifies a distinct luminal progenitor cell type in distal prostate invagination tips. Nat. Genet. 52, 908–918 (2020)."), are absent in _EPC_ and _PC_ mice. By contrast, a distinct subset of luminal cells (also known as Lum\_L2 or LumP) that express stem-like markers (Sca1, Trop2) and have increased regenerative potential[48](https://www.nature.com/articles/s41588-025-02289-w#ref-CR48 "Karthaus, W. R. et al. Regenerative potential of prostate luminal cells revealed by single-cell analysis. Science 368, 497–505 (2020)."),[70](https://www.nature.com/articles/s41588-025-02289-w#ref-CR70 "Crowley, L. et al. A single-cell atlas of the mouse and human prostate reveals heterogeneity and conservation of epithelial progenitors. eLife 9, e59465 (2020)."),[71](https://www.nature.com/articles/s41588-025-02289-w#ref-CR71 "Guo, W. et al. Single-cell transcriptomics identifies a distinct luminal progenitor cell type in distal prostate invagination tips. Nat. Genet. 52, 908–918 (2020).") are more abundant in _EPC_ and _PC_ mice (Fig. [4a–c](https://www.nature.com/articles/s41588-025-02289-w#Fig4) and Extended Data Fig. [7g–j](https://www.nature.com/articles/s41588-025-02289-w#Fig15)) (together designated Lum\_Mut)[72](https://www.nature.com/articles/s41588-025-02289-w#ref-CR72 "Germanos, A. A. et al. Defining cellular population dynamics at single-cell resolution during prostate cancer progression. eLife 11, e79076 (2022)."). Most striking, however, are two cell clusters (1, 10) that coexpress basal (_Krt5/14/15/17_, _Trp63_) and luminal (_Krt8/18/19_, _Cd24a_, _Foxa1_) lineage genes, which we collectively refer to as IM, based on earlier use of this terminology (Fig. [4d–f](https://www.nature.com/articles/s41588-025-02289-w#Fig4) and Extended Data Fig. [7k](https://www.nature.com/articles/s41588-025-02289-w#Fig15)).

**Fig. 4: IM cells represent a multi-lineage, stem-like state.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/4)

**a**, Experimental design including _WT_ (1 mouse), _PC_ (2 mice) and _EPC_ (2 mice). Whole prostates from indicated mice at 3 months of age were collected for scRNA-seq analysis. **b**,**c**, UMAP of epithelial cell clusters with all genotypes together (**b**) or assigned to each genotype (**c**). Luminal and IM clusters specific to mutant samples (_PC_ and _EPC_) are highlighted in circles and defined as Lum\_Mut and IM cells, respectively. Cluster IDs are shown. A cluster with low expression of epithelial lineage markers is defined as lineage-negative. **d**, UMAP plotting the _ERG_ transgene expression. **e**, Cell cycle scores (top) and GSEA (bottom) across all epithelial clusters. The clusters are numbered according to the UMAP in **b**. Pathway enrichment is calculated via GSEA using clusterProfiler and shown for those with an FDR-adjusted _P_ < 0.05. **f**, Violin plots comparing canonical prostate lineage marker expression across all epithelial clusters. The clusters are numbered according to the UMAP in **b**. **g**,**h**, Transcriptional programs in indicated cell types from single-cell transcriptomes in mice (**g**) and human (**h**). LN, lineage-negative; C2, MSigDB Collection 2.

Differential gene expression analysis of IM cells revealed elevated stemness, MYC, NF-κB, Stat signatures, cellular processes related to RNA processing and translation, as well as growth factor signaling specifically in IM cells in _EPC_ mice (Fig. [4e,g](https://www.nature.com/articles/s41588-025-02289-w#Fig4), Extended Data Fig. [7l](https://www.nature.com/articles/s41588-025-02289-w#Fig15) and Supplementary Tables [2](https://www.nature.com/articles/s41588-025-02289-w#MOESM4) and [3](https://www.nature.com/articles/s41588-025-02289-w#MOESM4)). Cluster 10 is noteworthy for increased S and G2/M cycle scores and elevated levels of Ki67, Top2a and Pcna, indicative of high proliferation. IM cells also express marker genes seen in Hillock and club cells (Fig. [4e,f](https://www.nature.com/articles/s41588-025-02289-w#Fig4)), two stem-like epithelial subtypes of lung and prostate involved in tissue regeneration[73](https://www.nature.com/articles/s41588-025-02289-w#ref-CR73 "Lin, B. et al. Airway hillocks are injury-resistant reservoirs of unique plastic stem cells. Nature 629, 869–877 (2024)."),[74](https://www.nature.com/articles/s41588-025-02289-w#ref-CR74 "Henry, G. H. et al. A cellular anatomy of the normal adult human prostate and prostatic urethra. Cell Rep. 25, 3530–3542 (2018)."),[75](https://www.nature.com/articles/s41588-025-02289-w#ref-CR75 "Chen, Y. et al. Club cells employ regeneration mechanisms during lung tumorigenesis. Nat. Commun. 13, 4557 (2022)."),[76](https://www.nature.com/articles/s41588-025-02289-w#ref-CR76 "Huang, F. W. et al. Club-like cells in proliferative inflammatory atrophy of the prostate. J. Pathol. 261, 85–95 (2023).").

### Relevance of IM state in ERG+ human prostate cancer

To determine whether the expanded IM population seen in GEMMs is relevant in human PCa, we first verified that gene sets identified through single-cell analysis of our _EPC_ mice are present in single-cell data from ERG+ human PCa (Fig. [4g,h](https://www.nature.com/articles/s41588-025-02289-w#Fig4), Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02289-w#Fig16), Supplementary Table [3](https://www.nature.com/articles/s41588-025-02289-w#MOESM4) and Supplementary Note [5](https://www.nature.com/articles/s41588-025-02289-w#MOESM1)). To explore whether the IM cell state in ERG+ GEMMs is present in human PCa, we first noted that IM cells in EPC mice have reduced AR target gene signature expression compared with luminal cells, consistent with the fact that they are in transition between basal and luminal cell states (Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02289-w#Fig16)). Similarly, ERG+ human PCa (particularly those with concurrent PTEN loss) display reduced AR pathway activation, reduced luminal signatures and elevated GEMM-derived IM signatures, as measured transcriptionally in single-cell and bulk RNA-seq datasets (Fig. [5a,b](https://www.nature.com/articles/s41588-025-02289-w#Fig5)). We next asked whether luminal or IM cell state signatures derived from ERG+ GEMMs might distinguish between different clinical outcomes in ERG+ human patients with PCa by interrogating baseline tumor transcriptomes (bulk RNA-seq) in two widely studied cohorts. Because luminal tumor cells should be highly represented in these samples, we first performed the analysis using Lum\_Mut signatures (derived independently from GEMM clusters 11, 3 and 4), or using either of two widely used human AR pathway signatures. ERG+ patients whose tumors scored in the top luminal signature quartile had longer disease-free survival than those with ERG+ tumors that scored in the bottom luminal quartile (Fig. [5c](https://www.nature.com/articles/s41588-025-02289-w#Fig5) and Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02289-w#Fig16)), an association not seen in ERG-negative tumors. We performed the same analysis using the GEMM-derived IM signature, as well as a previously reported human basal stem cell signature, to ask whether the reciprocal association (IM signature and worse outcome) was observed. ERG+ patients in the top quartile had shorter disease-free survival than those in the bottom quartile (Extended Data Fig. [8c](https://www.nature.com/articles/s41588-025-02289-w#Fig16)). Although the strength of the IM signature association was less pronounced than for the Lum\_Mut signature, we find it remarkable considering the IM signature is likely present in only a subpopulation of cells but is sufficient to generate a signal detected through bulk RNA-seq analysis. With the caveat that these associations are based on retrospective analysis, the prognostic signal seen here using IM and luminal cell state signatures raises the provocative hypothesis that the cell of origin in which the ERG translocation occurs (basal or luminal) could impact outcome.

**Fig. 5: IM signatures are associated with a worse outcome in ERG+ patients.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/5)

**a**, Expression of basal/IM and luminal signatures across indicated scRNA-seq cell types from human primary PCa[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022)."). **b**, Expression of basal/IM and luminal signatures in bulk RNA-seq data of primary human PCa, separated based on ERG and PTEN statuses. E, ERG-fusion-positive (_n_ = 141); P, PTEN deep deletion (_n_ = 23); EP, ERG-fusion-positive and PTEN deep deletion (_n_ = 62); DN, ERG-fusion-negative, PTEN deep deletion-negative (_n_ = 268). The center line represents the median, the box limits represent the upper and lower quartiles, and the minimum and maximum whiskers represent the 10th and 90th percentiles, respectively. **c**, Progression-free survival outcome using indicated signatures from two independent cohorts of patients with primary PCa, stratified based on ERG status. Data represent mean ± s.d.; one-way ANOVA with Tukey posttest to correct for multiple comparisons (**b**); log-rank (Mantel–Cox) test (**c**).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM8)

### The ERG-unique IM chromatin state reveals epigenetic dependencies

The lineage tracing, single-cell profiling and cell state signature analyses reported here all point to the importance of the IM population. To gain further mechanistic insight into this cell state, we profiled chromatin accessibility changes within the _EPC_ model using single-cell assay for transposase-accessible chromatin with high-throughput sequencing (scATAC-seq). Cell identity of the resulting assay for transposase-accessible chromatin (ATAC) clusters was inferred by aligning scRNA-seq and scATAC-seq datasets using anchor-based integration ([Methods](https://www.nature.com/articles/s41588-025-02289-w#Sec11)). Notably, a prominent cluster unique to _EPC_ mice was detected and was transcriptionally defined as IM cells (Fig. [6a](https://www.nature.com/articles/s41588-025-02289-w#Fig6) and Extended Data Fig. [9a–h](https://www.nature.com/articles/s41588-025-02289-w#Fig17)). This cluster (hereafter called EPC-IM) is highly enriched for ETS (putatively ERG) binding motifs (Fig. [6b](https://www.nature.com/articles/s41588-025-02289-w#Fig6)), implicating ERG as the primary contributor to the shift in chromatin landscape. The ETS binding motif is also enriched in _EPC_ Lum\_Mut cells but, in contrast, luminal cells of _EPC_ and _PC_ mice (EPC-Lum and PC-Lum) belong to the same scATAC-seq cluster (Fig. [6a,b](https://www.nature.com/articles/s41588-025-02289-w#Fig6) and Extended Data Fig. [9b](https://www.nature.com/articles/s41588-025-02289-w#Fig17)). To examine the relationship between cell types, we performed trajectory analysis with Palantir[77](https://www.nature.com/articles/s41588-025-02289-w#ref-CR77 "Setty, M. et al. Characterization of cell fate probabilities in single-cell data with Palantir. Nat. Biotechnol. 37, 451–460 (2019)."), starting with a cell in the basal cluster based on the lineage tracing data ([Methods](https://www.nature.com/articles/s41588-025-02289-w#Sec11)). The calculated trajectories suggest EPC-IM cells precede and may ultimately give rise to Lum\_Mut cells (Fig. [6c](https://www.nature.com/articles/s41588-025-02289-w#Fig6) and Extended Data Fig. [9i](https://www.nature.com/articles/s41588-025-02289-w#Fig17)), consistent with the lineage tracing studies described earlier (Fig. [3](https://www.nature.com/articles/s41588-025-02289-w#Fig3)).

**Fig. 6: ERG drives a unique chromatin state in IM cells featuring STAT3 and other TFs.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/6)

**a**, UMAP of scATAC-seq showing epithelial cell clusters assigned to each genotype (_n_ = 16,006 total number of epithelial cells). Dorsal–lateral lobes from indicated mice at different ages (4 weeks, 3 months and 6 months) were pooled together and analyzed (Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02289-w#Fig17)). **b**, Single-cell ChromVar enrichment of ETS family (ERG, ETS, ETV, FLI) motifs. **c**, Violin plots showing cells grouped by cell type on the _x_ axis and arranged by pseudotime on the _y_ axis. **d**, Heatmap of TF motifs enriched per cell type. Hypergeometric tests were performed for each motif’s accessibility within each cell type compared with all other cell types. Motifs with an FDR-adjusted _P_ < 10−20 are shown. Major cell-type-specific motif families are highlighted. **e**, Trajectories of three EPC-IM-enriched motifs. **f**, Co-accessible ChromVar _Z_ scores of indicated motifs in EPC-IM cells with or without ETS factors. Boxplots are centered at the median value and the lower and upper hinges correspond to the interquartile range (IQR) at the 25th and 75th percentiles, respectively. The lower and upper whiskers extend from the hinge to the smallest or largest value at most 1.5 × IQR of the hinge, respectively. FDR-adjusted _P_ values above the violin were calculated with one-sided Wilcoxon rank sum test using ChromVar _Z_ scores of peaks with co-occurring motifs versus exclusively one motif. **g**, Left: schematic showing individual genes perturbed in fresh tumor-derived organoids (_EPC_) before orthotopic transplantation. Whole prostates of _EPC_ mice (3–9 months) were pooled, edited and shortly expanded as organoids before orthotopic transplantation into host dorsal lobes. Right: normalized weight of injected lobes from indicated orthografts (left to right: _n_ = 26, 4, 4, 6, 6, 12, 6, 14 grafts). TFs are categorized based on motif accessibility enrichment in indicated clusters. **h**, Histological analysis on orthografts collected at the 1-month time point (8 grafts per group). Tumor areas were quantified based on ERG/Stat3/pAkt status as highlighted in yellow dashed lines. Arrows in the _EPC;sgStat3_ group highlight ERG+ cells that retain Stat3 expression. Scale bars, 250 µm. Data represent mean ± s.d.; one-way ANOVA with Dunnett posttest (**g**); one-way ANOVA with Tukey posttest (**h**).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM9)

The scATAC-seq findings, as well as the scRNA-seq and lineage tracing studies described earlier, collectively point to the EPC-IM cluster as the cell population where ERG activates a tumor initiation program. To gain insight into how this happens, we searched for additional binding motifs enriched within this cluster to identify a TF that might cooperate with ERG. As a quality control for the robustness of this approach, clusters from _WT_ mice with inferred basal and luminal identity showed enrichment of known TF motifs including P63 and FOX/GATA/HOXB13, respectively[44](https://www.nature.com/articles/s41588-025-02289-w#ref-CR44 "Norris, J. D. et al. The homeodomain protein HOXB13 regulates the cellular response to androgens. Mol. Cell 36, 405–416 (2009)."),[45](https://www.nature.com/articles/s41588-025-02289-w#ref-CR45 "Wang, Q. et al. A hierarchical network of transcription factors governs androgen receptor-dependent prostate cancer growth. Mol. Cell 27, 380–392 (2007)."),[78](https://www.nature.com/articles/s41588-025-02289-w#ref-CR78 "Signoretti, S. et al. p63 is a prostate basal cell marker and is required for prostate development. Am. J. Pathol. 157, 1769–1775 (2000)."),[79](https://www.nature.com/articles/s41588-025-02289-w#ref-CR79 "Adams, E. J. et al. FOXA1 mutations alter pioneering activity, differentiation and prostate cancer phenotypes. Nature 571, 408–412 (2019)."),[80](https://www.nature.com/articles/s41588-025-02289-w#ref-CR80 "Asselin-Labat, M. L. et al. Gata-3 is an essential regulator of mammary-gland morphogenesis and luminal-cell differentiation. Nat. Cell Biol. 9, 201–209 (2007)."),[81](https://www.nature.com/articles/s41588-025-02289-w#ref-CR81 "Kouros-Mehr, H., Slorach, E. M., Sternlicht, M. D. & Werb, Z. GATA-3 maintains the differentiation of the luminal cell fate in the mammary gland. Cell 127, 1041–1055 (2006)."),[82](https://www.nature.com/articles/s41588-025-02289-w#ref-CR82 "Parolia, A. et al. Distinct structural classes of activating FOXA1 alterations in advanced prostate cancer. Nature 571, 413–418 (2019)."),[83](https://www.nature.com/articles/s41588-025-02289-w#ref-CR83 "Pomerantz, M. M. et al. The androgen receptor cistrome is extensively reprogrammed in human prostate tumorigenesis. Nat. Genet. 47, 1346–1351 (2015).") (Fig. [6d](https://www.nature.com/articles/s41588-025-02289-w#Fig6)). Two patterns emerged in IM clusters: (1) NF-κB and AP-1 motifs enriched in PC-IM and EPC-IM clusters, and (2) STAT and NFAT motifs enriched only in _EPC_ mice (Fig. [6d–f](https://www.nature.com/articles/s41588-025-02289-w#Fig6), Extended Data Fig. [9j,k](https://www.nature.com/articles/s41588-025-02289-w#Fig17) and Supplementary Note [6](https://www.nature.com/articles/s41588-025-02289-w#MOESM1)).

The analysis of enriched motifs raises the possibility of cooperativity between ERG and one or more of the above TFs. To test this hypothesis at a functional level, we performed orthotopic transplantation experiments using fresh tumor-derived organoids from _EPC_ mice in which we deleted either a candidate TF using single guide RNAs (sgRNAs), or a nontargeting sgRNA as a control, using weight of the injected lobe as a quick readout of disease burden (Fig. [6g](https://www.nature.com/articles/s41588-025-02289-w#Fig6)). As expected, ERG ablation impaired graft weight, establishing the robustness and ERG dependence of the assay. Of six candidate TFs tested (representing AP-1, NF-κB, NFAT and STAT), only _Stat3_ deletion led to a reduction in graft weight similar to that seen with ERG deletion (Fig. [6g](https://www.nature.com/articles/s41588-025-02289-w#Fig6)). We therefore focused on STAT3 for further analysis, recognizing that AP-1 and NF-κB cannot be fully eliminated as candidates due to redundancy of other proteins within the same family that might compensate for single gene deletion. Prostate adenocarcinomas developed in all mice transplanted with nontargeting (NT) sgRNA organoids but not in those in which either ERG or Stat3 was deleted (Fig. [6h](https://www.nature.com/articles/s41588-025-02289-w#Fig6)). The few residual areas of ERG+ adenocarcinoma that retained Stat3 expression likely represent cells that escaped Stat3 knockout (Fig. [6h](https://www.nature.com/articles/s41588-025-02289-w#Fig6)).

Having defined an ERG-specific chromatin context associated with tumor initiation, we postulated that the transcriptional programs activated by ERG likely involve one or more chromatin modifying enzymes. Using the single-cell data we identified two candidates (Kmt2a, Dot1l) highly and preferentially expressed in IM cells from _PC_ and _EPC_ mice (Fig. [7a](https://www.nature.com/articles/s41588-025-02289-w#Fig7)). Of these, KMT2A messenger RNA expression also correlated with ERG-positive human PCa (Fig. [7b](https://www.nature.com/articles/s41588-025-02289-w#Fig7)). Furthermore, the only two ETS-positive PCa cell lines examined in DepMap both show selective dependency on both KMT2A and DOT1L relative to seven ETS-negative PCa cell lines (Fig. [7c](https://www.nature.com/articles/s41588-025-02289-w#Fig7)). Finally, sgRNA knockdown of either _Kmt2a_ or _Dot1l_ impaired ERG-induced luminal lineage specification in the organoid assay described earlier, providing evidence both enzymes play a role in ERG function (Fig. [7d](https://www.nature.com/articles/s41588-025-02289-w#Fig7)) (see also Fig. [3h](https://www.nature.com/articles/s41588-025-02289-w#Fig3)).

**Fig. 7: ERG oncogenicity depends on KMT2A/MLL1 and DOT1L but not MENIN.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/7)

**a**, Violin plots comparing inferred expression of a panel of histone-modifying enzymes across all epithelial clusters from scATAC-seq. **b**, Box plot showing expression of indicated genes in human PCa from two public datasets, separated by ERG-fusion status. _n_ = 181 (ERG−), 152 (ERG+) for TCGA; _n_ = 108 (ERG−), 100 (ERG+) for SU2C. The center line represents the median, the box limits represent the upper and lower quartiles, and the minimum and maximum whiskers represent the 10th and 90th percentiles, respectively. **c**, DepMap dependency scores of human prostate cell lines. A negative CRISPR score indicates dependency (_n_ = 8 for ETS−, _n_ = 2 for ETS+ lines). ETS TF positive cell lines, VCaP (ERG+) and LNCaP (ETV1+), are highlighted. The center line represents the median, the box limits represent the upper and lower quartiles, and the minimum and maximum whiskers represent the minimum and maximum, respectively. **d**, Luminal output quantified by flow cytometry (left to right: _n_ = 7, 2, 2, 2, 3, 3 independent assays). _EP_ reporter organoids were treated with indicated guide RNAs and analyzed 2 weeks later. **e**, Schematic showing orthotopic transplantation assay using engineered tumor-derived organoids. Whole prostates of _EPC_ mice (3–9 months) were pooled, edited and shortly expanded in organoid culture before transplantation into dorsal lobes of host mice (2 months old). Prostates were collected 5~7 weeks after grafting. **f**–**h**, Histology and tumor area quantification of orthografts with Kmt2a (**f**), Dot1l (**g**) and Men1 (**h**) perturbation. Left to right: _n_ = 16, 8, 6 grafts (**f**); _n_ = 14, 12, 6, 14 grafts (**g**); _n_ = 6, 2, 6, 6 grafts (**g**). Tumor areas were quantified as highlighted, based on ERG (**f**), ERG and loss of H3K79me2 (**g**), ERG and loss of Menin (**h**). H3K79me2 immunohistochemistry is shown in Extended Data Fig. [10c](https://www.nature.com/articles/s41588-025-02289-w#Fig18). Scale bars, 250 µm. Data represent mean ± s.d.; two-tailed unpaired _t_\-test (**b**,**c**); one-way ANOVA with Dunnett posttest (**d**); one-way ANOVA with Tukey posttest (**f**–**h**). Neg, negative; Pos, positive.

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM10)

In light of the development of pharmacological inhibitors for KMT2A (Menin inhibitors)[84](https://www.nature.com/articles/s41588-025-02289-w#ref-CR84 "Krivtsov, A. V. et al. A Menin-MLL inhibitor induces specific chromatin changes and eradicates disease in models of MLL-rearranged leukemia. Cancer Cell 36, 660–673 (2019)."),[85](https://www.nature.com/articles/s41588-025-02289-w#ref-CR85 "Issa, G. C. et al. The menin inhibitor revumenib in KMT2A-rearranged or NPM1-mutant leukaemia. Nature 615, 920–924 (2023)."),[86](https://www.nature.com/articles/s41588-025-02289-w#ref-CR86 "Issa, G. C. et al. Menin inhibition with revumenib for KMT2A-rearranged relapsed or refractory acute leukemia (AUGMENT-101). J. Clin. Oncol. 43, 75–84 (2025).") and DOT1L[41](https://www.nature.com/articles/s41588-025-02289-w#ref-CR41 "Stein, E. M. et al. The DOT1L inhibitor pinometostat reduces H3K79 methylation and has modest clinical activity in adult acute leukemia. Blood 131, 2661–2669 (2018)."),[87](https://www.nature.com/articles/s41588-025-02289-w#ref-CR87 "Sarno, F., Nebbioso, A. & Altucci, L. DOT1L: a key target in normal chromatin remodelling and in mixed-lineage leukaemia treatment. Epigenetics 15, 439–453 (2020)."),[88](https://www.nature.com/articles/s41588-025-02289-w#ref-CR88 "Vatapalli, R. et al. Histone methyltransferase DOT1L coordinates AR and MYC stability in prostate cancer. Nat. Commun. 11, 4153 (2020)."), we evaluated both _Kmt2a_ and _Dot1l_ in the same ERG-dependent in vivo tumorigenicity model used earlier for _Stat3_. Remarkably, both _Kmt2a_ and _Dot1l_ deletion impaired the growth of _EPC_\-derived tumors to a comparable magnitude as that seen with _ERG_ deletion (Fig. [7e–g](https://www.nature.com/articles/s41588-025-02289-w#Fig7) and Extended Data Fig. [10a](https://www.nature.com/articles/s41588-025-02289-w#Fig18)). As with the Stat3 experiments, the small foci of residual adenocarcinoma remaining in _Kmt2a_ and _Dot1l_ sgRNA orthografts retained target protein expression (Kmt2a/Mll1) or activity (H3K79me2 for Dot1l) (Extended Data Fig. [10b,c](https://www.nature.com/articles/s41588-025-02289-w#Fig18)), further supporting the critical dependency of ERG-driven cancer on _Kmt2a_/_Mll1_ and _Dot1l_. Interestingly, _Men1_ (encoding Menin) sgRNA did not impair ERG-dependent tumor growth despite efficient reduction of Menin protein levels in tumors (Fig. [7h](https://www.nature.com/articles/s41588-025-02289-w#Fig7) and Extended Data Fig. [10d](https://www.nature.com/articles/s41588-025-02289-w#Fig18)), raising the possibility of ERG/KMT2A cooperativity through a mechanism independent of Menin.

Discussion
----------

ERG translocations are the presumed driver event in nearly half of PCa in Western cohorts (~150,000 new cases per year), yet we have limited insight into precisely how ERG initiates PCa. This lack of understanding is, in part, due to limited availability of human PCa cell lines, patient-derived xenografts or organoid models that retain ERG expression. Through a combination of lineage tracing experiments coupled with cell-type-specific activation of ERG in basal versus luminal cells, we find that ERG-driven PCa initiate in a rare subset of basal cells present in healthy mouse and human prostates (which we call BasalLum cells) that coexpress various canonical luminal lineage genes including, importantly, TMPRSS2. Upon ERG activation, BasalLum cells give rise to the highly proliferative IM cells, which subsequently differentiate into invasive luminal adenocarcinomas (Fig. [8](https://www.nature.com/articles/s41588-025-02289-w#Fig8)). These findings help refine previous conclusions using various mouse and human models that implicate both basal and luminal cells as cells of origin for PCa[31](https://www.nature.com/articles/s41588-025-02289-w#ref-CR31 "Goldstein, A. S., Huang, J., Guo, C., Garraway, I. P. & Witte, O. N. Identification of a cell of origin for human prostate cancer. Science 329, 568–571 (2010)."),[32](https://www.nature.com/articles/s41588-025-02289-w#ref-CR32 "Lawson, D. A. et al. Basal epithelial stem cells are efficient targets for prostate cancer initiation. Proc. Natl Acad. Sci. USA 107, 2610–2615 (2010)."),[33](https://www.nature.com/articles/s41588-025-02289-w#ref-CR33 "Lu, T. L. et al. Conditionally ablated Pten in prostate basal cells promotes basal-to-luminal differentiation and causes invasive prostate cancer in mice. Am. J. Pathol. 182, 975–991 (2013)."),[34](https://www.nature.com/articles/s41588-025-02289-w#ref-CR34 "Park, J. W. et al. Prostate epithelial cell of origin determines cancer differentiation state in an organoid transformation assay. Proc. Natl Acad. Sci. USA 113, 4482–4487 (2016)."),[35](https://www.nature.com/articles/s41588-025-02289-w#ref-CR35 "Stoyanova, T. et al. Prostate cancer originating in basal cells progresses to adenocarcinoma propagated by luminal-like cells. Proc. Natl Acad. Sci. USA 110, 20111–20116 (2013)."),[36](https://www.nature.com/articles/s41588-025-02289-w#ref-CR36 "Choi, N., Zhang, B., Zhang, L., Ittmann, M. & Xin, L. Adult murine prostate basal and luminal cells are self-sustained lineages that can both serve as targets for prostate cancer initiation. Cancer Cell 21, 253–265 (2012)."),[37](https://www.nature.com/articles/s41588-025-02289-w#ref-CR37 "Wang, X. et al. A luminal epithelial stem cell that is a cell of origin for prostate cancer. Nature 461, 495–500 (2009)."),[38](https://www.nature.com/articles/s41588-025-02289-w#ref-CR38 "Wang, Z. A. et al. Lineage analysis of basal epithelial cells reveals their unexpected plasticity and supports a cell-of-origin model for prostate cancer heterogeneity. Nat. Cell Biol. 15, 274–283 (2013)."),[39](https://www.nature.com/articles/s41588-025-02289-w#ref-CR39 "Wang, Z. A., Toivanen, R., Bergren, S. K., Chambon, P. & Shen, M. M. Luminal cells are favored as the cell of origin for prostate cancer. Cell Rep. 8, 1339–1346 (2014)."),[40](https://www.nature.com/articles/s41588-025-02289-w#ref-CR40 "Yoo, Y. A. et al. Bmi1 marks distinct castration-resistant luminal progenitor cells competent for prostate regeneration and tumour initiation. Nat. Commun. 7, 12943 (2016).").

**Fig. 8: Model on cell context dependency of ERG oncogenicity.**

[![figure 8](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_Fig8_HTML.png)](https://www.nature.com/articles/s41588-025-02289-w/figures/8)

ERG translocations (in the context of PTEN loss) that occur in a subset of basal cells with luminal transcriptomic features (BasalLum cells) enter a highly proliferative multi-lineage IM state that, in turn, drives invasive cancer with luminal differentiation. By contrast, ERG translocations that originate in luminal cells may develop intraductal hyperplasia but fail to progress to invasive cancer. The basal-derived IM cells provide a chromatin context to support maximal ERG-driven chromatin changes, featuring TFs (NF-κB, AP-1, NFAT, STAT) in addition to ERG and epigenetic regulators (MLL1, DOT1L).

Taking advantage of recent advances that make it possible to profile transcriptomic and chromatin landscape changes at single-cell resolution, we then interrogated disease initiation and progression over time in an ERG GEMM model that accurately models ERG-driven disease initiation and progression. Through this analysis we identified a subfraction of highly proliferative epithelial cells with multi-lineage features (basal, luminal, hillock, club) (IM cells) that appear within weeks of ERG activation and give rise to the invasive luminal adenocarcinomas that represent the clinical manifestation of the disease. Postulating that this IM cell subpopulation reflects a cellular context optimally primed to respond to the oncogenic potential of ERG, we searched for transcriptomic and chromatin landscape features uniquely present in ERG+ IM cells versus ERG+ luminal cells. This analysis revealed an ERG-specific chromatin state exclusively in IM cells, with selective enrichment in regions of newly open chromatin of binding motifs for ERG, NF-κB, AP-1, NFATC1 and STAT3, as well as elevated expression of the lysine methyltransferases KMT2A/MLL1 and DOT1L (Fig. [8](https://www.nature.com/articles/s41588-025-02289-w#Fig8)).

To test whether these IM-specific differences in chromatin modifying enzyme expression and TF motif accessibility are linked with the oncogenic potential of ERG, we developed an ERG-dependent in vivo tumorigenicity assay through which we could efficiently score the consequences of perturbing each candidate (via CRISPR deletion). STAT3, KMT2A/MLL1 and DOT1L emerged as clear hits from these in vivo studies, with all three blocking tumorigenicity at levels comparable to that seen with ERG deletion. The fact that ERG remains robustly expressed in regions where _Stat3_\-, _Kmt2a_\- or _Dot1l_\-deleted cells persist (as small noninvasive clusters) suggests that ERG may partner with all three proteins to activate tumorigenicity programs. Furthermore, prostate data from DepMap, albeit limited due to the small number of prostate cell lines evaluated, suggest that ERG/ETS+ human PCa cell lines may be selectively dependent on KMT2A and DOT1L (Fig. [7c](https://www.nature.com/articles/s41588-025-02289-w#Fig7)).

The fact that pharmacological inhibitors of KMT2A/MLL1, DOT1L and STAT3 are in various stages of clinical investigation begs the question of whether these compounds may have activity in PCa[41](https://www.nature.com/articles/s41588-025-02289-w#ref-CR41 "Stein, E. M. et al. The DOT1L inhibitor pinometostat reduces H3K79 methylation and has modest clinical activity in adult acute leukemia. Blood 131, 2661–2669 (2018)."),[84](https://www.nature.com/articles/s41588-025-02289-w#ref-CR84 "Krivtsov, A. V. et al. A Menin-MLL inhibitor induces specific chromatin changes and eradicates disease in models of MLL-rearranged leukemia. Cancer Cell 36, 660–673 (2019)."),[85](https://www.nature.com/articles/s41588-025-02289-w#ref-CR85 "Issa, G. C. et al. The menin inhibitor revumenib in KMT2A-rearranged or NPM1-mutant leukaemia. Nature 615, 920–924 (2023)."),[88](https://www.nature.com/articles/s41588-025-02289-w#ref-CR88 "Vatapalli, R. et al. Histone methyltransferase DOT1L coordinates AR and MYC stability in prostate cancer. Nat. Commun. 11, 4153 (2020)."),[89](https://www.nature.com/articles/s41588-025-02289-w#ref-CR89 "Deshpande, A. J. et al. Leukemic transformation by the MLL-AF6 fusion oncogene requires the H3K79 methyltransferase Dot1l. Blood 121, 2533–2541 (2013)."). KMT2A/MLL1 has been previously implicated as a dependency in castration resistance PCa due to its role as a co-activator of AR signaling[90](https://www.nature.com/articles/s41588-025-02289-w#ref-CR90 "Malik, R. et al. Targeting the MLL complex in castration-resistant prostate cancer. Nat. Med. 21, 344–352 (2015)."). The in vivo sensitivity to Menin/MLL1 inhibition reported in this earlier work is based on ERG+ (VCaP) and ETV1+ (LNCaP/AR) PCa xenograft models, raising the possibility that these phenotypes may in fact be a consequence of ETS-specific dependencies. It is also worth noting that KMT2/MLL1 dependency has been reported for gastrointestinal stromal tumors[91](https://www.nature.com/articles/s41588-025-02289-w#ref-CR91 "Hemming, M. L. et al. MOZ and Menin-MLL complexes are complementary regulators of chromatin association and transcriptional output in gastrointestinal stromal tumor. Cancer Discov. 12, 1804–1823 (2022)."), a KIT-driven malignancy that is also ETS dependent based on the lineage requirement for ETV1 in interstitial cells of Cajal, the cell of origin for gastrointestinal stromal tumors[92](https://www.nature.com/articles/s41588-025-02289-w#ref-CR92 "Chi, P. et al. ETV1 is a lineage survival factor that cooperates with KIT in gastrointestinal stromal tumours. Nature 467, 849–853 (2010)."). Thus, several lines of evidence from different tumor types and model systems suggest a dependency of ETS-driven malignancies on KMT2A/MLL1 function. One critical caveat regarding the allograft data reported here is the ablation of ERG-driven tumorigenicity by Kmt2a/Mll1 deletion but not by Menin deletion (Fig. [7f–h](https://www.nature.com/articles/s41588-025-02289-w#Fig7)). Further work is required to determine whether the _Kmt2a/Mll1_\-dependent, _Menin_\-independent phenotype seen here is specific to the model used here. In any case, further studies of KMT2A/MLL1, DOT1L and STAT3 dependency across a range of ETS-driven mouse and human tumor models are warranted, particularly with the potential for near-term translational impact using existing clinical-grade inhibitors.

In closing, our work expands the notion that cancers can initiate in a cell with a different lineage identity from the resultant tumor, a concept first established in chronic myeloid leukemia, where the BCR–ABL translocation initiates in a hematopoietic stem cell but the clinical manifestation of the disease is elevated levels of terminally differentiated myeloid cells (neutrophils)[93](https://www.nature.com/articles/s41588-025-02289-w#ref-CR93 "Deininger, M. W., Goldman, J. M. & Melo, J. V. The molecular biology of chronic myeloid leukemia. Blood 96, 3343–3356 (2000)."),[94](https://www.nature.com/articles/s41588-025-02289-w#ref-CR94 "Vetrie, D., Helgason, G. V. & Copland, M. The leukaemia stem cell: similarities, differences and clinical prospects in CML and AML. Nat. Rev. Cancer 20, 158–173 (2020)."). Furthermore, our identification of a highly proliferative, stem-like population with a unique chromatin state may provide an opportunity to reveal ERG-specific dependencies that could be exploited therapeutically (see Supplementary Notes [7](https://www.nature.com/articles/s41588-025-02289-w#MOESM1) and [8](https://www.nature.com/articles/s41588-025-02289-w#MOESM1) for further discussion of cell context and study limitations).

Methods
-------

### Ethical statement

Mouse experiments were conducted under protocol 0607012 approved by the Institutional Animal Care and Use Committee of Memorial Sloan Kettering Cancer Center (MSKCC), New York.

### Mouse strains and dosing

_Rosa26-ERG__LSL/LSL_, _Pten__flox/flox_, _Pb-Cre4_ and _Rosa26-YFP__LSL/LSL_ (_Gt(ROSA)26Sor__tm3(CAG-EYFP)Hze__/J_) mice have been previously described[3](https://www.nature.com/articles/s41588-025-02289-w#ref-CR3 "Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. Nat. Med. 19, 1023–1029 (2013)."),[95](https://www.nature.com/articles/s41588-025-02289-w#ref-CR95 "Wu, X. et al. Generation of a prostate epithelial cell-specific Cre transgenic mouse model for tissue-specific gene ablation. Mech. Dev. 101, 61–69 (2001)."),[96](https://www.nature.com/articles/s41588-025-02289-w#ref-CR96 "Trotman, L. C. et al. Pten dose dictates cancer progression in the prostate. PLoS Biol. 1, E59 (2003)."),[97](https://www.nature.com/articles/s41588-025-02289-w#ref-CR97 "Madisen, L. et al. A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. Nat. Neurosci. 13, 133–140 (2010).").

For tamoxifen-inducible Cre recombination, _K8-CreER__T2_ (_Tg(Krt8-cre/ERT2)17Blpn/J_)[58](https://www.nature.com/articles/s41588-025-02289-w#ref-CR58 "Van Keymeulen, A. et al. Distinct stem cells contribute to mammary gland development and maintenance. Nature 479, 189–193 (2011)."), _K5-CreER__T2_ (_Krt5__tm1.1(cre/ERT2)Blh__/J_)[58](https://www.nature.com/articles/s41588-025-02289-w#ref-CR58 "Van Keymeulen, A. et al. Distinct stem cells contribute to mammary gland development and maintenance. Nature 479, 189–193 (2011)."), _K14-CreER_ (_Tg(KRT14-cre/ERT)20Efu/J_)[98](https://www.nature.com/articles/s41588-025-02289-w#ref-CR98 "Vasioukhin, V., Degenstein, L., Wise, B. & Fuchs, E. The magical touch: genome targeting in epidermal stem cells induced by tamoxifen application to mouse skin. Proc. Natl Acad. Sci. USA 96, 8551–8556 (1999).") or _Nkx3-1-CreER__T2_ (_Nkx3-1__tm4(cre/ERT2)Mms__/AbshnJ_)[37](https://www.nature.com/articles/s41588-025-02289-w#ref-CR37 "Wang, X. et al. A luminal epithelial stem cell that is a cell of origin for prostate cancer. Nature 461, 495–500 (2009).") mice were crossed to the desired floxed strains and introduced as hemizygous/heterozygous alleles. To induce CreER or CreERT2\-mediated recombination, tamoxifen (Sigma) was suspended in corn oil and injected intraperitoneally to mice at 7–9 weeks of age. For _K14-CreER, K8-__CreER__T2_ and _Nkx3-1-CreER__T2_, mice were treated with two doses of 3 mg of tamoxifen per 25 g of body weight on day 1 and day 3, respectively. For _K5-CreER__T2_, mice were administered one dose of 0.5 mg of tamoxifen per 25 g of body weight. For EdU pulse–chase experiments, 2-month-old mice were treated with EdU (40 mg kg−1; Thermo Fisher) by intraperitoneal injection.

The maximal burden for prostate tumors allowed in our protocol is when animals show signs of abdomen distension or other signs of disease/distress, none of which has been observed throughout the course of our study.

### Histology and immunostaining

For GEMM experiments, whole mouse prostates containing all lobes were collected at autopsy. Histological data from all lobes were pooled for quantification. For intraprostatic transplantation and adenoviral infection experiments, injected prostate lobes were collected at autopsy. Prostate tissues were fixed using 4% paraformaldehyde, dehydrated with 70% ethanol, paraffin-embedded and sectioned. H&E staining was performed following standard protocols by the MSKCC Molecular Cytology Core or IDEXX. Immunohistochemistry and immunofluorescence were performed on a Leica Bond RX automatic stainer using antibodies listed in Supplementary Table [4](https://www.nature.com/articles/s41588-025-02289-w#MOESM4). All formalin-fixed paraffin-embedded stained tissue was scanned using a MIRAX scanner. Histological assessments were performed blinded by an MSKCC pathologist.

### Tissue dissociation, organoid culture and fresh isolation of prostate epithelial cells

Whole mouse prostates containing all lobes were isolated as described previously[99](https://www.nature.com/articles/s41588-025-02289-w#ref-CR99 "Drost, J. et al. Organoid culture systems for prostate epithelial and cancer tissue. Nat. Protoc. 11, 347–358 (2016)."),[100](https://www.nature.com/articles/s41588-025-02289-w#ref-CR100 "Karthaus, W. R. et al. Identification of multipotent luminal progenitor cells in human prostate organoid cultures. Cell 159, 163–175 (2014)."). Briefly, prostates were collected and subsequently digested with collagenase type II (Gibco) for 2 h at 37 °C, followed by TrypLE (Gibco) digestion at 37 °C until a single-cell suspension was obtained. Digestions were supplemented with Y-27632 (10 μM) to inhibit anoikis and filtered to obtain a single-cell suspension.

Murine prostate organoid culture was established and maintained under standard conditions as described previously[99](https://www.nature.com/articles/s41588-025-02289-w#ref-CR99 "Drost, J. et al. Organoid culture systems for prostate epithelial and cancer tissue. Nat. Protoc. 11, 347–358 (2016)."),[100](https://www.nature.com/articles/s41588-025-02289-w#ref-CR100 "Karthaus, W. R. et al. Identification of multipotent luminal progenitor cells in human prostate organoid cultures. Cell 159, 163–175 (2014)."). Briefly, dissociated prostatic epithelial cells were embedded in 20-µl drops of Matrigel (Corning) and overlaid with mouse prostate organoid medium. Organoids were passaged by mechanical splitting by repetitively passing through a 200-µl pipette tip or by trypsinization using TryplE. For lineage marker analysis, organoids were trypsinized into single cells and seeded at 20,000–50,000 cells per well of a 12-well plate in the absence of EGF (counted as day 0). For EdU pulse–chase experiments, the organoid culture was incubated with EdU (10 µM, Thermo Fisher) for 2 h before collecting (pulse) or wash-off and re-seeding (chase). To compare cell fate from reporter cells, basal and luminal cells were sorted based on single-positive reporter signals and seeded at equal cell number between sgNT- and sgERG-expressing organoids in the absence of EGF (counted as day 0). Basal cells were seeded at 20,000–50,000 cells per well of a 12-well plate; fewer luminal cells were seeded (5,000–10,000 per well) due to a low recovery from the sgERG group.

To isolate basal and luminal cells from donor prostates[101](https://www.nature.com/articles/s41588-025-02289-w#ref-CR101 "Lawson, D. A., Xin, L., Lukacs, R. U., Cheng, D. & Witte, O. N. Isolation and functional characterization of murine prostate stem cells. Proc. Natl Acad. Sci. USA 104, 181–186 (2007)."), freshly dissociated prostate cells were stained for 1 h on ice with CD49f-PE (1:200; BD, cat no. 555736) and CD24-Alexa Fluor 647 (1:200; BioLegend, cat. no. 101818). Sorted basal and luminal cells were transduced with Cre-expressing adenovirus (Ad-CMV-iCre, Vector Biolabs) and directly used for downstream experiments.

To induce Cre recombinase expression, adenoviral transduction was performed as described previously[100](https://www.nature.com/articles/s41588-025-02289-w#ref-CR100 "Karthaus, W. R. et al. Identification of multipotent luminal progenitor cells in human prostate organoid cultures. Cell 159, 163–175 (2014)."). Briefly, 50,000 dissociated organoid cells or sorted fresh tissue cells were mixed with 1 µl of Ad-CMV-iCre (1.0 × 107 plaque-forming units per µl, Vector Biolabs) in 500 µl of growth medium. The suspension was spin-infected at 600_g_ for 1 h before a PBS wash and collection of the infected cells for downstream experiments. The amount of virus used was scaled according to the number of cells available.

### Orthotopic transplantation and intraprostatic adenoviral infection

For orthotopic transplantation, 250,000–500,000 freshly sorted prostate epithelial cells or organoid cells were resuspended in 20 μl of 50% Matrigel (Corning) and 50% organoid culture medium before injection into prostate anterior or dorsal lobes of immunodeficient NSG mice (The Jackson Laboratory) at 2 months of age.

For intraprostatic adenoviral administration, the approach was adapted from previous reports[102](https://www.nature.com/articles/s41588-025-02289-w#ref-CR102 "Leow, C. C., Wang, X. D. & Gao, W. Q. Novel method of generating prostate-specific Cre-LoxP gene switching via intraductal delivery of adenovirus. Prostate 65, 1–9 (2005)."),[103](https://www.nature.com/articles/s41588-025-02289-w#ref-CR103 "DuPage, M., Dooley, A. L. & Jacks, T. Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase. Nat. Protoc. 4, 1064–1072 (2009)."),[104](https://www.nature.com/articles/s41588-025-02289-w#ref-CR104 "Park, S. et al. Novel mouse models of bladder cancer identify a prognostic signature associated with risk of disease progression. Cancer Res. 81, 5161–5175 (2021).") with modifications. Ad-K5-Cre (Ad5-bK5-Cre) (A. Berns, the Netherlands Cancer Institute) and Ad-K8-Cre (Ad5-mK8-nlsCre)[105](https://www.nature.com/articles/s41588-025-02289-w#ref-CR105 "Tao, L., van Bragt, M. P., Laudadio, E. & Li, Z. Lineage tracing of mammary epithelial cells using cell-type-specific cre-expressing adenoviruses. Stem Cell Rep. 2, 770–779 (2014).") were obtained at high titer (1.0 × 108 plaque-forming units per µl) from the Viral Vector Core at the University of Iowa. Adenoviruses were prepared in two ways, mixed with 1/9th volume of either 10 mM CaCl2 or 16 µg ml−1 Polybrene (Millipore Sigma). Then, 10 µl of adenoviral solution was injected into either dorsal or anterior lobes of floxed mice at 2–3 months of age. Similar results were obtained which were pooled together in downstream analysis.

### Organoid engineering

Genetic perturbation in organoids was carried out by CRISPR-RNP as previously described[43](https://www.nature.com/articles/s41588-025-02289-w#ref-CR43 "Feng, W. et al. Rapid interrogation of cancer cell of origin through CRISPR editing. Proc. Natl Acad. Sci. USA 118, e2110344118 (2021)."). Briefly, Cas9 protein (IDT) was first mixed with sgRNA (IDT) to form the RNP complex before nucleofection. A total of 1.2 µM cRNP was used per individual sgRNA. In total, 500,000–1,000,000 dissociated organoid cells were resuspended with nucleofection buffer, RNP complexes and electroporation enhancer (IDT, 1:1 molar ratio to RNP) in a total volume of 100 µl. The cell suspension was transferred to a nucleofection cuvette and nucleofected using Lonza Amaxa Nucleofector II (program T-030). Cells were centrifuged and seeded for culture.

To generate _EP_ organoids, the parental floxed organoids (_Rosa26-ERG__LSL/LSL__; Pten__flox/flox_) were transduced with Cre-expressing adenovirus (Ad-CMV-iCre, Vector Biolabs), sorted based on GFP signal (linked with ERG expression as ires-GFP) and maintained under a non-EGF condition. To generate isogenic _P_ organoids, the ERG transgene was preemptively knocked out by CRISPR-RNP from the same parental floxed organoids before Cre transduction. The GFP-negative population were sorted out and maintained under a non-EGF condition. The expected ERG and Pten statuses were confirmed by intracellular flow cytometry at the protein level.

The knock-in lineage reporters were engineered by CRISPaint as previously described[106](https://www.nature.com/articles/s41588-025-02289-w#ref-CR106 "Schmid-Burgk, J. L., Honing, K., Ebert, T. S. & Hornung, V. CRISPaint allows modular base-specific gene tagging using a ligase-4-dependent mechanism. Nat. Commun. 7, 12338 (2016)."),[107](https://www.nature.com/articles/s41588-025-02289-w#ref-CR107 "Artegiani, B. et al. Fast and efficient generation of knock-in human organoids using homology-independent CRISPR-Cas9 precision genome editing. Nat. Cell Biol. 22, 321–331 (2020)."). An _EP_ organoid line with a stable transduction of lentiCas9-blast (Addgene plasmid no. 52962)[108](https://www.nature.com/articles/s41588-025-02289-w#ref-CR108 "Sanjana, N. E., Shalem, O. & Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. Nat. Methods 11, 783–784 (2014).") was used as the starting material. To knock in TagRFP at the _Krt8_ locus, a targeting vector, pCRISPaint-TagRFP (Addgene plasmid no. 67167), was electroporated into the organoids together with CRISPR-RNPs directed to the targeting vector (sgFrame) and _Krt8_ (targeting exon 9). The TagRFP+ population was enriched by sorting and prepared for the next steps of engineering. To knock in mNeonGreen at the _Krt5_ locus, a targeting vector, pCRISPaint-mNeonGreen, was electroporated into the organoids together with CRISPR-RNPs directed to the targeting vector (sgFrame) and _Krt5_ (targeting exon 9). The TagRFP-mNeonGreen+ population was first enriched by sorting before a secondary enrichment for the TagRFP+mNeonGreen+ population. The resulting population was used for further study without picking single clones. The knock-in products were first verified by PCR across the break junction (oligos 1 and 2 for Krt8-TagRFP; 3 and 4 for Krt5-mNeonGreen) and Sanger sequencing using the PCR primers. A further verification was performed by intracellular flow cytometry comparing the expression pattern of K8 versus the TagRFP reporter (not possible with K5 due to the epitope ablation by the targeting event). As a further validation of the Krt5-mNeonGreen engineering, Krt5 depletion by short interfering RNA (siGENOME, Horizon Discovery) led to a reduction of the mNeonGreen reporter signal. The expected spatial expression patterns of the reporters were validated by live cell confocal imaging using Leica SP5 or SP8 confocal microscopes.

sgRNA target and oligo sequences are listed in Supplementary Table [5](https://www.nature.com/articles/s41588-025-02289-w#MOESM4).

### Intracellular flow cytometry

Intracellular flow cytometry was performed using a Fixation/Permeabilization Solution Kit (BD cat. no. 554714). Dissociated prostate or organoid cells were fixed and permeabilized with the fixation/permeabilization buffer for 30 min on ice, followed by incubation with primary antibodies for at least 1 h at room temperature and secondary antibodies (for nonconjugated primary antibodies) for 30 min at room temperature with washes between the steps before analysis by flow cytometry (BD Fortessa or MACSQuant 16). Dead cells were excluded using either live/dead fixable dyes (Thermo Fisher) on ice for 15 min before fixation and gated by excluding the positive population, or co-stained with DAPI during antibody incubation and gated by excluding the sub-G1 population. Where applicable, EdU+ cells were detected using Click-iT Plus EdU Flow Cytometry Assay Kit (Thermo Fisher) following the manufacturer’s instructions before antibody incubation. Antibody concentrations and catalog numbers are listed in Supplementary Table [6](https://www.nature.com/articles/s41588-025-02289-w#MOESM4).

### Western blot

Boiled whole cell lysates were run on a precast Tris-acetate gel (Thermo Fisher) or a Mini-PROTEAN TGX protein gel (Bio-Rad) and then transferred to a nitrocellulose membrane (Bio-Rad). The membrane was incubated overnight at 4 °C with primary antibodies against Kmt2a (1:1,000; Cell Signaling Technology, cat. no. 14689), Menin (1:1,000; Cell Signaling Technology, cat. no. 6891S), Hsp90 (1:1,000; Cell Signaling Technology, cat. no. 4877S) and Actin-HRP (horseradish peroxidase) (1:10,000; Abcam, cat. no. ab49900). Signal was visualized with secondary HRP-conjugated antibodies and chemiluminescence detection.

### Plasmid construction

To construct pCRISPaint-mNeonGreen, mNeonGreen coding sequence was synthesized and cloned into pCRISPaint-TagRFP (Addgene plasmid no. 67167)[106](https://www.nature.com/articles/s41588-025-02289-w#ref-CR106 "Schmid-Burgk, J. L., Honing, K., Ebert, T. S. & Hornung, V. CRISPaint allows modular base-specific gene tagging using a ligase-4-dependent mechanism. Nat. Commun. 7, 12338 (2016).") between BamHI/NotI sites to replace the TagRFP cassette. The correct cloning was confirmed by Sanger sequencing.

### Bulk RNA-seq analysis of sorted organoid cells

Basal and luminal organoid cells were sorted based on reporter signals. Three replicates were performed. RNA was extracted from bulk cells and sequenced at the Integrated Genomics Operation Core (MSKCC). Briefly, the SMARTer (Switching Mechanism At the 5′ end of RNA Template) method was performed for minimal cells RNA extraction. After amplification, complementary DNA was subjected to automated Illumina paired-end library construction. Libraries were sequenced on Illumina HiSeq2000 instruments with paired reads of 100 base pairs (bp) in length per sample. The samples were sequenced at 40–50 million reads per sample. Sequence data were analyzed by Basepair ([https://www.basepairtech.com/](https://www.basepairtech.com/)). Briefly, FASTQ data were aligned using the tophat software to the mouse (mm10) reference genome. Fragments per kilobase of transcript per million mapped reads were calculated using the cufflink software.

Gene set enrichment analysis (GSEA) was performed using the GSEA software ([https://www.gsea-msigdb.org/gsea/index.jsp](https://www.gsea-msigdb.org/gsea/index.jsp)). Briefly, differentially expressed gene rank lists were generated based on gene expression profiles. Annotated gene sets were obtained from MSigDB ([https://www.gsea-msigdb.org/gsea/msigdb/index.jsp](https://www.gsea-msigdb.org/gsea/msigdb/index.jsp)) or manually curated. Enrichr pathway analyses were performed using the Enrichr website ([https://maayanlab.cloud/Enrichr/](https://maayanlab.cloud/Enrichr/))[109](https://www.nature.com/articles/s41588-025-02289-w#ref-CR109 "Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC Bioinformatics 14, 128 (2013)."),[110](https://www.nature.com/articles/s41588-025-02289-w#ref-CR110 "Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res. 44, W90–W97 (2016)."),[111](https://www.nature.com/articles/s41588-025-02289-w#ref-CR111 "Xie, Z. et al. Gene set knowledge discovery with Enrichr. Curr. Protoc. 1, e90 (2021)."). Gene sets generated (cutoff padj < 0.05, FC > 2) were used. The combined score was computed by taking the log of the _P_ value from the Fisher exact test and multiplying that by the _Z_ score of the deviation from the expected rank. _c_ = log(_p_) × _z_, where _c_ is the combined score, _p_ is the Fisher exact test _P_ value and _z_ is the _Z_ score for deviation from expected rank.

### Sample preparation for single-cell analysis

For scRNA-seq, whole mouse prostates from the indicated age were freshly dissociated (described above) and flow sorted for live single cells using DAPI, before processing for scRNA-seq using 10X Genomics Chromium following the manufacture’s instructions. For scATAC-seq, dorsal–lateral lobes of mouse prostates from the indicated age were freshly dissociated (described above) and flow sorted for live single cells using DAPI. Fresh nuclei were isolated before processing for 10X Genomics Chromium Single Cell Multiome ATAC + Gene Expression sequencing following the manufacturer’s instructions. ATAC data were retrieved for downstream analysis.

### scRNA-seq processing and filtering

scRNA-seq was preprocessed using 10X Cell Ranger software. This included genome alignment and minimal cell filtering for RNA-seq. We used a modified genome reference to align sequencing reads to the _Mus musculus_ genome (mm10) plus an additional sequence for the _ERG_ transgene. After alignment, the 10X Cell Ranger software performs unique molecular identifier and read filtering to create a filtered cell by gene read count matrix. We examined this filtered count matrix to visualize cell and gene coverage distributions and to gather cell/gene statistics. Based on these metrics, several plots were visualized and used to remove poor quality cells (those with fewer than 500 transcripts) or putative multiplets (defined as cells containing greater than 100 k transcripts). Cells detected with 40% or more mitochondrial reads were removed without further consideration.

### scRNA-seq matrix normalization and variance stabilization

To account for technical variation and normalize expression data, the R (v.4.4.0) package Seurat (v.5.2.1)[112](https://www.nature.com/articles/s41588-025-02289-w#ref-CR112 "Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587 (2021).") was used on the filtered raw count matrix. SCT transform was run to perform normalization, variance stabilization and feature selection. We regressed out cell cycle scores and mitochondrial percentage in addition to Seurat defaults.

### Dimensional reduction and visualization of single-cell RNA

Dimensionality reduction and uniform manifold approximation and projection (UMAP) were applied to the count-corrected single-cell read matrix to enable a two-dimensional cell embedding. A nearest neighbor graph of cells was learned and used as input to the Leiden algorithm for cell clustering. Ten different cell clustering resolutions were evaluated and scored by silhouette score, a clustering metric. UMAPs were generated and used to hypothesize cell similarities and identify clusters for differential expression testing.

### Differential genes and pathways

Differential expression testing was performed using the R package presto (v.1.0.0) ([https://github.com/immunogenomics/presto](https://github.com/immunogenomics/presto)). Genes were ranked using the average log fold change and a false discovery rate (FDR)-adjusted _P_ value. A GSEA analysis with fsGSEA (v.1.30.0)[113](https://www.nature.com/articles/s41588-025-02289-w#ref-CR113 "Korotkevich, G. et al. Fast gene set enrichment analysis. Preprint at bioRxiv 
https://doi.org/10.1101/060012
(2021).") utilized a list of ranked genes per comparison and the collection of molecular signatures database (MSigDB).

### Epithelial cell identification

We investigated all single cells for epithelial cell populations by summarizing clusters of Epcam+ cells. Cell clusters containing 50% or more of Epcam+ cells were flagged as epithelial related. Next, individual cells were scored using an epithelial cell signature set of differentially expressed genes defined in wild-type mouse in ref. [48](https://www.nature.com/articles/s41588-025-02289-w#ref-CR48 "Karthaus, W. R. et al. Regenerative potential of prostate luminal cells revealed by single-cell analysis. Science 368, 497–505 (2020)."). Those cells found in flagged epithelial clusters and defined as basal, luminal 1 (L1), luminal 2 (L2) or luminal 3 (L3) were retained as confident epithelial cells and used in downstream analysis and clustering.

### RNA-seq data from human prostate cancer

The source of bulk RNA-seq data for RNA signature analysis was primary PCa samples initially reported by The Cancer Genome Atlas (TCGA) and in refs. [9](https://www.nature.com/articles/s41588-025-02289-w#ref-CR9 "Taylor, B. S. et al. Integrative genomic profiling of human prostate cancer. Cancer Cell 18, 11–22 (2010)."),[30](https://www.nature.com/articles/s41588-025-02289-w#ref-CR30 "The Cancer Genome Atlas Research Network. The molecular taxonomy of primary prostate cancer. Cell 163, 1011–1025 (2015)."). The sources of scRNA-seq data were primary PCa samples reported in refs. [51](https://www.nature.com/articles/s41588-025-02289-w#ref-CR51 "Hirz, T. et al. Dissecting the immune suppressive human prostate tumor microenvironment via integrated single-cell and spatial transcriptomic analyses. Nat. Commun. 14, 663 (2023)."),[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022)."). Data were reproduced from these studies using the same cell type classification with the following exceptions. For the ref. [52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022).") study, the two subtypes of PCa cells which were originally annotated as ERG-positive and ERG-negative tumor cells are now together named as PCa in this study because ERG+ cells also exist in the ‘ERG-negative’ clusters. The ERG-positive and mixed (denoting those carrying both ERG-positive and ERG-negative tumor cells by scRNA-seq) patients defined by the original study are now grouped together as ERG+ patients in this study considering the typically uniform ERG expression pattern in PCa by histology[2](https://www.nature.com/articles/s41588-025-02289-w#ref-CR2 "Cerveira, N. et al. TMPRSS2-ERG gene fusion causing ERG overexpression precedes chromosome copy number changes in prostate carcinomas and paired HGPIN lesions. Neoplasia 8, 826–832 (2006)."),[4](https://www.nature.com/articles/s41588-025-02289-w#ref-CR4 "Gerhauser, C. et al. Molecular evolution of early-onset prostate cancer identifies molecular risk markers and clinical trajectories. Cancer Cell 34, 996–1011 e8 (2018)."),[7](https://www.nature.com/articles/s41588-025-02289-w#ref-CR7 "Perner, S. et al. TMPRSS2-ERG fusion prostate cancer: an early molecular event associated with invasion. Am. J. Surg. Pathol. 31, 882–888 (2007)."). Similarly for the ref. [51](https://www.nature.com/articles/s41588-025-02289-w#ref-CR51 "Hirz, T. et al. Dissecting the immune suppressive human prostate tumor microenvironment via integrated single-cell and spatial transcriptomic analyses. Nat. Commun. 14, 663 (2023).") study, ERG+ patients were defined as those with >50% PCa cells found to be ERG+.

### scATAC-seq filtering features matrix

scATAC-seq data were evaluated from the Cell Ranger ARC (10x Genomics) filtered count matrix. Cell filtering, feature selection and analyses were performed using ArchR software (v.1.0.2)[114](https://www.nature.com/articles/s41588-025-02289-w#ref-CR114 "Granja, J. M. et al. Single-cell multiomic analysis identifies regulatory programs in mixed-phenotype acute leukemia. Nat. Biotechnol. 37, 1458–1465 (2019)."). We generated output plots to visualize distributions of cell counts within promoters, gene bodies and transcription start sites. Quality metrics were used to remove poor quality cells, including low transcription start site enrichment (<4) or low number of unique nuclear fragment (<1,000) or putative multiplets (defined as cells containing greater than 1 million reads).

### Dimensional reduction

Dimensionality reduction was calculated via ArchR with Iterative Latent Semantic Indexing (LSI) using 500-bp genomic tiles. A nearest neighbor graph of the LSI reduced data was performed by Seurat’s shared nearest neighbor implementation. We excluded the first LSI component that was correlated >75% to read depth. Clusters were then identified with the Leiden algorithm.

### Peak and motif identification

Peaks with scATAC-seq data were called using MACS3 (v.3.0.0b1)[115](https://www.nature.com/articles/s41588-025-02289-w#ref-CR115 "Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol. 9, R137 (2008).") through the ArchR interface. Differentially accessible regions were defined as those having significant (FDR < 0.05) difference with log2 fold change ≥ 0.5. For motif detection we utilized two databases, (1) CisBP and (2) the Non-redundant TF motif matches genome-wide[116](https://www.nature.com/articles/s41588-025-02289-w#ref-CR116 "Vierstra, J. et al. Global reference mapping of human transcription factor footprints. Nature 583, 729–736 (2020)."). Motif matching within peaks was performed using MotifMatchr (v.1.26.0) with defaults. The gene identity of each position weight matrix match was recorded so that we retained only those motifs with corresponding normalized average gene expression per cell type >1. To identify motifs statistically significant between comparisons, we calculated motif ChromVar[117](https://www.nature.com/articles/s41588-025-02289-w#ref-CR117 "Schep, A. N., Wu, B., Buenrostro, J. D. & Greenleaf, W. J. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. Nat. Methods 14, 975–978 (2017).") scores per cell and performed Spearman correlation of those scores to the motif’s gene score values. Those motifs having positive correlation and significant ChromVar scores per cell type were retained.

### Motif co-accessibility

The ETS family of factors includes several highly similar TF motif position weight matrices. To simplify the identification of ETS family factors within scATAC-seq, we visualized the distribution of similar ETS family motif deviation _Z_ scores before reducing the following motifs from oncogenic ETS family TFs in PCa[30](https://www.nature.com/articles/s41588-025-02289-w#ref-CR30 "The Cancer Genome Atlas Research Network. The molecular taxonomy of primary prostate cancer. Cell 163, 1011–1025 (2015)."): ERG, ETS, ETV, FLI, into one identifier, hereafter referred to as ETS motifs. To evaluate co-accessibility of ETS motifs with other significantly accessible motifs within cell types, called candidate motifs (described above), we identified peaks containing each candidate motif and the ETS motifs. We then calculated the per cell deviation and deviation _Z_ score with ChromVar. Similarly, we calculated ChromVar _Z_ scores for each candidate motif whose peaks did not overlap ETS motifs and then the complement, that is, ETS motifs peaks, that do not harbor the candidate motif. For each candidate motif a two-sample Wilcoxon rank sum test was used to establish whether co-occurences of the candidate and ETS motifs were significantly different (one-sided alternative ‘greater’) from: (1) ETS motifs that do not co-occur with the candidate motif, or (2) candidate motifs that do not co-occur with ETS motifs. For NFKB2, NFAC1 and STAT3 the co-occurrences with ETS were significantly greater than without ETS within EPC-IM cells (Fig. [6f](https://www.nature.com/articles/s41588-025-02289-w#Fig6)).

### Statistics and reproducibility

Statistical analysis was performed as specified in the figure legends. No statistical method was used to predetermine sample size. No data were excluded from the analyses. Mice were randomized for in vivo experiments. Histological assessments were performed blinded by an MSKCC pathologist. Experiments were repeated with a minimum of two independent experiments as noted in figure legends. Data distribution was assumed to be normal but this was not formally tested.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02289-w#MOESM2) linked to this article.

Data availability
-----------------

Raw sequencing data are publicly available from the Gene Expression Omnibus: [GSE257543](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE257543) (scRNA-seq), [GSE258962](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE258962) (scATAC-seq) and [GSE294013](https://www.ncbi.xyz/geo/query/acc.cgi?acc=GSE294013) (bulk RNA-seq). Existing scRNA-seq data reanalyzed in our paper are available as [GSE176031](https://www.ncbi.xyz/geo/query/acc.cgi?acc=GSE176031) and [GSE181294](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE181294). [Source data](https://www.nature.com/articles/s41588-025-02289-w#Sec38) are provided with this paper.

Code availability
-----------------

This study uses standard preprocessing and published tools for analysis, which have been cited in the [Methods](https://www.nature.com/articles/s41588-025-02289-w#Sec11) and are available via Zenodo at [https://doi.org/10.5281/zenodo.15802144](https://doi.org/10.5281/zenodo.15802144) (ref. [118](https://www.nature.com/articles/s41588-025-02289-w#ref-CR118 "Feng, W. et al. ERG-driven prostate cancer initiation is cell context dependent and requires KMT2A and DOT1L. Zenodo 
https://doi.org/10.5281/zenodo.15802144
(2025).")).

References
----------

1.  Carver, B. S. et al. Aberrant ERG expression cooperates with loss of PTEN to promote cancer progression in the prostate. _Nat. Genet._ **41**, 619–624 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXltValtrw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19396168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835150)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aberrant%20ERG%20expression%20cooperates%20with%20loss%20of%20PTEN%20to%20promote%20cancer%20progression%20in%20the%20prostate&journal=Nat.%20Genet.&volume=41&pages=619-624&publication_year=2009&author=Carver%2CBS) 
    
2.  Cerveira, N. et al. TMPRSS2-ERG gene fusion causing ERG overexpression precedes chromosome copy number changes in prostate carcinomas and paired HGPIN lesions. _Neoplasia_ **8**, 826–832 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1WgtrzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17032499)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1715930)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TMPRSS2-ERG%20gene%20fusion%20causing%20ERG%20overexpression%20precedes%20chromosome%20copy%20number%20changes%20in%20prostate%20carcinomas%20and%20paired%20HGPIN%20lesions&journal=Neoplasia&volume=8&pages=826-832&publication_year=2006&author=Cerveira%2CN) 
    
3.  Chen, Y. et al. ETS factors reprogram the androgen receptor cistrome and prime prostate tumorigenesis in response to PTEN loss. _Nat. Med._ **19**, 1023–1029 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVaku7fM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23817021)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3737318)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETS%20factors%20reprogram%20the%20androgen%20receptor%20cistrome%20and%20prime%20prostate%20tumorigenesis%20in%20response%20to%20PTEN%20loss&journal=Nat.%20Med.&volume=19&pages=1023-1029&publication_year=2013&author=Chen%2CY) 
    
4.  Gerhauser, C. et al. Molecular evolution of early-onset prostate cancer identifies molecular risk markers and clinical trajectories. _Cancer Cell_ **34**, 996–1011 e8 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVygsLnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30537516)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7444093)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20evolution%20of%20early-onset%20prostate%20cancer%20identifies%20molecular%20risk%20markers%20and%20clinical%20trajectories&journal=Cancer%20Cell&volume=34&pages=996-1011%20e8&publication_year=2018&author=Gerhauser%2CC) 
    
5.  King, J. C. et al. Cooperativity of TMPRSS2-ERG with PI3-kinase pathway activation in prostate oncogenesis. _Nat. Genet._ **41**, 524–526 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXltValtL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19396167)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898503)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cooperativity%20of%20TMPRSS2-ERG%20with%20PI3-kinase%20pathway%20activation%20in%20prostate%20oncogenesis&journal=Nat.%20Genet.&volume=41&pages=524-526&publication_year=2009&author=King%2CJC) 
    
6.  Mosquera, J. M. et al. Characterization of TMPRSS2-ERG fusion high-grade prostatic intraepithelial neoplasia and potential clinical implications. _Clin. Cancer Res._ **14**, 3380–3385 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmslygsb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18519767)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3717517)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterization%20of%20TMPRSS2-ERG%20fusion%20high-grade%20prostatic%20intraepithelial%20neoplasia%20and%20potential%20clinical%20implications&journal=Clin.%20Cancer%20Res.&volume=14&pages=3380-3385&publication_year=2008&author=Mosquera%2CJM) 
    
7.  Perner, S. et al. TMPRSS2-ERG fusion prostate cancer: an early molecular event associated with invasion. _Am. J. Surg. Pathol._ **31**, 882–888 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17527075)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TMPRSS2-ERG%20fusion%20prostate%20cancer%3A%20an%20early%20molecular%20event%20associated%20with%20invasion&journal=Am.%20J.%20Surg.%20Pathol.&volume=31&pages=882-888&publication_year=2007&author=Perner%2CS) 
    
8.  Shrestha, E. et al. Oncogenic gene fusions in nonneoplastic precursors as evidence that bacterial infection can initiate prostate cancer. _Proc. Natl Acad. Sci. USA_ **118**, e2018976118 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslOgu7jJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34341114)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8364155)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Oncogenic%20gene%20fusions%20in%20nonneoplastic%20precursors%20as%20evidence%20that%20bacterial%20infection%20can%20initiate%20prostate%20cancer&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=118&publication_year=2021&author=Shrestha%2CE) 
    
9.  Taylor, B. S. et al. Integrative genomic profiling of human prostate cancer. _Cancer Cell_ **18**, 11–22 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXpslSjt7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20579941)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3198787)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrative%20genomic%20profiling%20of%20human%20prostate%20cancer&journal=Cancer%20Cell&volume=18&pages=11-22&publication_year=2010&author=Taylor%2CBS) 
    
10.  Bose, R. et al. ERF mutations reveal a balance of ETS factors controlling prostate oncogenesis. _Nature_ **546**, 671–675 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXpvFGmurk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28614298)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5576182)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ERF%20mutations%20reveal%20a%20balance%20of%20ETS%20factors%20controlling%20prostate%20oncogenesis&journal=Nature&volume=546&pages=671-675&publication_year=2017&author=Bose%2CR) 
    
11.  Brenner, J. C. et al. Mechanistic rationale for inhibition of poly(ADP-ribose) polymerase in ETS gene fusion-positive prostate cancer. _Cancer Cell_ **19**, 664–678 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmtFGlsL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21575865)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3113473)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mechanistic%20rationale%20for%20inhibition%20of%20poly%28ADP-ribose%29%20polymerase%20in%20ETS%20gene%20fusion-positive%20prostate%20cancer&journal=Cancer%20Cell&volume=19&pages=664-678&publication_year=2011&author=Brenner%2CJC) 
    
12.  Cai, C. et al. ERG induces androgen receptor-mediated regulation of SOX9 in prostate cancer. _J. Clin. Invest._ **123**, 1109–1122 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXjvVyhtr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23426182)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3582143)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ERG%20induces%20androgen%20receptor-mediated%20regulation%20of%20SOX9%20in%20prostate%20cancer&journal=J.%20Clin.%20Invest.&volume=123&pages=1109-1122&publication_year=2013&author=Cai%2CC) 
    
13.  Gupta, S. et al. FZD4 as a mediator of ERG oncogene-induced WNT signaling and epithelial-to-mesenchymal transition in human prostate cancer cells. _Cancer Res._ **70**, 6735–6745 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtV2rsLjF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20713528)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FZD4%20as%20a%20mediator%20of%20ERG%20oncogene-induced%20WNT%20signaling%20and%20epithelial-to-mesenchymal%20transition%20in%20human%20prostate%20cancer%20cells&journal=Cancer%20Res.&volume=70&pages=6735-6745&publication_year=2010&author=Gupta%2CS) 
    
14.  Hollenhorst, P. C. et al. Oncogenic ETS proteins mimic activated RAS/MAPK signaling in prostate cells. _Genes Dev._ **25**, 2147–2157 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsVant73K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22012618)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205585)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Oncogenic%20ETS%20proteins%20mimic%20activated%20RAS%2FMAPK%20signaling%20in%20prostate%20cells&journal=Genes%20Dev.&volume=25&pages=2147-2157&publication_year=2011&author=Hollenhorst%2CPC) 
    
15.  Kim, T. D., Shin, S. & Janknecht, R. ETS transcription factor ERG cooperates with histone demethylase KDM4A. _Oncol. Rep._ **35**, 3679–3688 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjtVOqtLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27109047)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4869937)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETS%20transcription%20factor%20ERG%20cooperates%20with%20histone%20demethylase%20KDM4A&journal=Oncol.%20Rep.&volume=35&pages=3679-3688&publication_year=2016&author=Kim%2CTD&author=Shin%2CS&author=Janknecht%2CR) 
    
16.  Kron, K. J. et al. TMPRSS2-ERG fusion co-opts master transcription factors and activates NOTCH signaling in primary prostate cancer. _Nat. Genet._ **49**, 1336–1345 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1yhs77F)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28783165)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TMPRSS2-ERG%20fusion%20co-opts%20master%20transcription%20factors%20and%20activates%20NOTCH%20signaling%20in%20primary%20prostate%20cancer&journal=Nat.%20Genet.&volume=49&pages=1336-1345&publication_year=2017&author=Kron%2CKJ) 
    
17.  Li, F. et al. ERG orchestrates chromatin interactions to drive prostate cell fate reprogramming. _J. Clin. Invest._ **130**, 5924–5941 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlSnt77E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32701507)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7598085)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ERG%20orchestrates%20chromatin%20interactions%20to%20drive%20prostate%20cell%20fate%20reprogramming&journal=J.%20Clin.%20Invest.&volume=130&pages=5924-5941&publication_year=2020&author=Li%2CF) 
    
18.  Liao, S. Y. et al. SND1 binds to ERG and promotes tumor growth in genetic mouse models of prostate cancer. _Nat. Commun._ **14**, 7435 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37973913)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10654515)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SND1%20binds%20to%20ERG%20and%20promotes%20tumor%20growth%20in%20genetic%20mouse%20models%20of%20prostate%20cancer&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Liao%2CSY) 
    
19.  Linn, D. E., Penney, K. L., Bronson, R. T., Mucci, L. A. & Li, Z. Deletion of interstitial genes between TMPRSS2 and ERG promotes prostate cancer progression. _Cancer Res._ **76**, 1869–1881 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XltlSktLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26880803)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4873435)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deletion%20of%20interstitial%20genes%20between%20TMPRSS2%20and%20ERG%20promotes%20prostate%20cancer%20progression&journal=Cancer%20Res.&volume=76&pages=1869-1881&publication_year=2016&author=Linn%2CDE&author=Penney%2CKL&author=Bronson%2CRT&author=Mucci%2CLA&author=Li%2CZ) 
    
20.  Liu, C. Y., Yu, T., Huang, Y., Cui, L. & Hong, W. ETS (E26 transformation-specific) up-regulation of the transcriptional co-activator TAZ promotes cell migration and metastasis in prostate cancer. _J. Biol. Chem._ **292**, 9420–9430 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXptFCns74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28408625)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5454120)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETS%20%28E26%20transformation-specific%29%20up-regulation%20of%20the%20transcriptional%20co-activator%20TAZ%20promotes%20cell%20migration%20and%20metastasis%20in%20prostate%20cancer&journal=J.%20Biol.%20Chem.&volume=292&pages=9420-9430&publication_year=2017&author=Liu%2CCY&author=Yu%2CT&author=Huang%2CY&author=Cui%2CL&author=Hong%2CW) 
    
21.  Mao, N. et al. Aberrant expression of ERG promotes resistance to combined PI3K and AR pathway inhibition through maintenance of AR target genes. _Mol. Cancer Ther._ **18**, 1577–1586 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVOntbnK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31296553)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6726496)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aberrant%20expression%20of%20ERG%20promotes%20resistance%20to%20combined%20PI3K%20and%20AR%20pathway%20inhibition%20through%20maintenance%20of%20AR%20target%20genes&journal=Mol.%20Cancer%20Ther.&volume=18&pages=1577-1586&publication_year=2019&author=Mao%2CN) 
    
22.  Mounir, Z. et al. TMPRSS2:ERG blocks neuroendocrine and luminal cell differentiation to maintain prostate cancer proliferation. _Oncogene_ **34**, 3815–3825 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhs1GmsbjE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25263440)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TMPRSS2%3AERG%20blocks%20neuroendocrine%20and%20luminal%20cell%20differentiation%20to%20maintain%20prostate%20cancer%20proliferation&journal=Oncogene&volume=34&pages=3815-3825&publication_year=2015&author=Mounir%2CZ) 
    
23.  Nguyen, L. T. et al. ERG activates the YAP1 transcriptional program and induces the development of age-related prostate tumors. _Cancer Cell_ **27**, 797–808 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtVejurnL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26058078)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4461839)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ERG%20activates%20the%20YAP1%20transcriptional%20program%20and%20induces%20the%20development%20of%20age-related%20prostate%20tumors&journal=Cancer%20Cell&volume=27&pages=797-808&publication_year=2015&author=Nguyen%2CLT) 
    
24.  Sandoval, G. J. et al. Binding of TMPRSS2-ERG to BAF chromatin remodeling complexes mediates prostate oncogenesis. _Mol. Cell_ **71**, 554–566 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVersL7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30078722)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6140332)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Binding%20of%20TMPRSS2-ERG%20to%20BAF%20chromatin%20remodeling%20complexes%20mediates%20prostate%20oncogenesis&journal=Mol.%20Cell&volume=71&pages=554-566&publication_year=2018&author=Sandoval%2CGJ) 
    
25.  Tomlins, S. A. et al. Role of the TMPRSS2-ERG gene fusion in prostate cancer. _Neoplasia_ **10**, 177–188 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXltleitr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18283340)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244693)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Role%20of%20the%20TMPRSS2-ERG%20gene%20fusion%20in%20prostate%20cancer&journal=Neoplasia&volume=10&pages=177-188&publication_year=2008&author=Tomlins%2CSA) 
    
26.  Wang, J. et al. Activation of NF-κB by TMPRSS2/ERG fusion isoforms through Toll-like receptor-4. _Cancer Res._ **71**, 1325–1333 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhvFSjsLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21169414)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Activation%20of%20NF-%CE%BAB%20by%20TMPRSS2%2FERG%20fusion%20isoforms%20through%20Toll-like%20receptor-4&journal=Cancer%20Res.&volume=71&pages=1325-1333&publication_year=2011&author=Wang%2CJ) 
    
27.  Wasmuth, E. V. et al. Modulation of androgen receptor DNA binding activity through direct interaction with the ETS transcription factor ERG. _Proc. Natl Acad. Sci. USA_ **117**, 8584–8592 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXntFOit7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32220959)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7165421)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Modulation%20of%20androgen%20receptor%20DNA%20binding%20activity%20through%20direct%20interaction%20with%20the%20ETS%20transcription%20factor%20ERG&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=8584-8592&publication_year=2020&author=Wasmuth%2CEV) 
    
28.  Wu, L. et al. ERG is a critical regulator of Wnt/LEF1 signaling in prostate cancer. _Cancer Res._ **73**, 6068–6079 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFGrsrbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23913826)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3790861)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ERG%20is%20a%20critical%20regulator%20of%20Wnt%2FLEF1%20signaling%20in%20prostate%20cancer&journal=Cancer%20Res.&volume=73&pages=6068-6079&publication_year=2013&author=Wu%2CL) 
    
29.  Yu, J. et al. An integrated network of androgen receptor, polycomb, and TMPRSS2-ERG gene fusions in prostate cancer progression. _Cancer Cell_ **17**, 443–454 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXmslKru7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20478527)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2874722)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20network%20of%20androgen%20receptor%2C%20polycomb%2C%20and%20TMPRSS2-ERG%20gene%20fusions%20in%20prostate%20cancer%20progression&journal=Cancer%20Cell&volume=17&pages=443-454&publication_year=2010&author=Yu%2CJ) 
    
30.  The Cancer Genome Atlas Research Network. The molecular taxonomy of primary prostate cancer. _Cell_ **163**, 1011–1025 (2015).
    
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4695400)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20taxonomy%20of%20primary%20prostate%20cancer&journal=Cell&volume=163&pages=1011-1025&publication_year=2015) 
    
31.  Goldstein, A. S., Huang, J., Guo, C., Garraway, I. P. & Witte, O. N. Identification of a cell of origin for human prostate cancer. _Science_ **329**, 568–571 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXptlCltbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20671189)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2917982)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20a%20cell%20of%20origin%20for%20human%20prostate%20cancer&journal=Science&volume=329&pages=568-571&publication_year=2010&author=Goldstein%2CAS&author=Huang%2CJ&author=Guo%2CC&author=Garraway%2CIP&author=Witte%2CON) 
    
32.  Lawson, D. A. et al. Basal epithelial stem cells are efficient targets for prostate cancer initiation. _Proc. Natl Acad. Sci. USA_ **107**, 2610–2615 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXit1aksb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20133806)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2823887)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Basal%20epithelial%20stem%20cells%20are%20efficient%20targets%20for%20prostate%20cancer%20initiation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=107&pages=2610-2615&publication_year=2010&author=Lawson%2CDA) 
    
33.  Lu, T. L. et al. Conditionally ablated Pten in prostate basal cells promotes basal-to-luminal differentiation and causes invasive prostate cancer in mice. _Am. J. Pathol._ **182**, 975–991 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXivFyrtr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23313138)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conditionally%20ablated%20Pten%20in%20prostate%20basal%20cells%20promotes%20basal-to-luminal%20differentiation%20and%20causes%20invasive%20prostate%20cancer%20in%20mice&journal=Am.%20J.%20Pathol.&volume=182&pages=975-991&publication_year=2013&author=Lu%2CTL) 
    
34.  Park, J. W. et al. Prostate epithelial cell of origin determines cancer differentiation state in an organoid transformation assay. _Proc. Natl Acad. Sci. USA_ **113**, 4482–4487 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XltlarurY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27044116)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4843433)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prostate%20epithelial%20cell%20of%20origin%20determines%20cancer%20differentiation%20state%20in%20an%20organoid%20transformation%20assay&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=113&pages=4482-4487&publication_year=2016&author=Park%2CJW) 
    
35.  Stoyanova, T. et al. Prostate cancer originating in basal cells progresses to adenocarcinoma propagated by luminal-like cells. _Proc. Natl Acad. Sci. USA_ **110**, 20111–20116 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFKms7fE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24282295)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864278)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prostate%20cancer%20originating%20in%20basal%20cells%20progresses%20to%20adenocarcinoma%20propagated%20by%20luminal-like%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=110&pages=20111-20116&publication_year=2013&author=Stoyanova%2CT) 
    
36.  Choi, N., Zhang, B., Zhang, L., Ittmann, M. & Xin, L. Adult murine prostate basal and luminal cells are self-sustained lineages that can both serve as targets for prostate cancer initiation. _Cancer Cell_ **21**, 253–265 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xit12gs7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22340597)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3285423)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adult%20murine%20prostate%20basal%20and%20luminal%20cells%20are%20self-sustained%20lineages%20that%20can%20both%20serve%20as%20targets%20for%20prostate%20cancer%20initiation&journal=Cancer%20Cell&volume=21&pages=253-265&publication_year=2012&author=Choi%2CN&author=Zhang%2CB&author=Zhang%2CL&author=Ittmann%2CM&author=Xin%2CL) 
    
37.  Wang, X. et al. A luminal epithelial stem cell that is a cell of origin for prostate cancer. _Nature_ **461**, 495–500 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFSntL7J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19741607)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2800362)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20luminal%20epithelial%20stem%20cell%20that%20is%20a%20cell%20of%20origin%20for%20prostate%20cancer&journal=Nature&volume=461&pages=495-500&publication_year=2009&author=Wang%2CX) 
    
38.  Wang, Z. A. et al. Lineage analysis of basal epithelial cells reveals their unexpected plasticity and supports a cell-of-origin model for prostate cancer heterogeneity. _Nat. Cell Biol._ **15**, 274–283 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXivFyiu7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23434823)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3743266)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lineage%20analysis%20of%20basal%20epithelial%20cells%20reveals%20their%20unexpected%20plasticity%20and%20supports%20a%20cell-of-origin%20model%20for%20prostate%20cancer%20heterogeneity&journal=Nat.%20Cell%20Biol.&volume=15&pages=274-283&publication_year=2013&author=Wang%2CZA) 
    
39.  Wang, Z. A., Toivanen, R., Bergren, S. K., Chambon, P. & Shen, M. M. Luminal cells are favored as the cell of origin for prostate cancer. _Cell Rep._ **8**, 1339–1346 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVOhtrfM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25176651)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4163115)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Luminal%20cells%20are%20favored%20as%20the%20cell%20of%20origin%20for%20prostate%20cancer&journal=Cell%20Rep.&volume=8&pages=1339-1346&publication_year=2014&author=Wang%2CZA&author=Toivanen%2CR&author=Bergren%2CSK&author=Chambon%2CP&author=Shen%2CMM) 
    
40.  Yoo, Y. A. et al. Bmi1 marks distinct castration-resistant luminal progenitor cells competent for prostate regeneration and tumour initiation. _Nat. Commun._ **7**, 12943 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1CgtrbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27703144)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059479)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bmi1%20marks%20distinct%20castration-resistant%20luminal%20progenitor%20cells%20competent%20for%20prostate%20regeneration%20and%20tumour%20initiation&journal=Nat.%20Commun.&volume=7&publication_year=2016&author=Yoo%2CYA) 
    
41.  Stein, E. M. et al. The DOT1L inhibitor pinometostat reduces H3K79 methylation and has modest clinical activity in adult acute leukemia. _Blood_ **131**, 2661–2669 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFOiurnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29724899)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6265654)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20DOT1L%20inhibitor%20pinometostat%20reduces%20H3K79%20methylation%20and%20has%20modest%20clinical%20activity%20in%20adult%20acute%20leukemia&journal=Blood&volume=131&pages=2661-2669&publication_year=2018&author=Stein%2CEM) 
    
42.  Tsimberidou, A. M. et al. Phase 1 trial of TTI-101, a first-in-class oral inhibitor of STAT3 in patients with advanced solid tumors. _Clin. Cancer Res._ **31**, 965–974 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXptFOqtLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39792482)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11911802)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phase%201%20trial%20of%20TTI-101%2C%20a%20first-in-class%20oral%20inhibitor%20of%20STAT3%20in%20patients%20with%20advanced%20solid%20tumors&journal=Clin.%20Cancer%20Res.&volume=31&pages=965-974&publication_year=2025&author=Tsimberidou%2CAM) 
    
43.  Feng, W. et al. Rapid interrogation of cancer cell of origin through CRISPR editing. _Proc. Natl Acad. Sci. USA_ **118**, e2110344118 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslOgu7nJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34353917)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8364185)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20interrogation%20of%20cancer%20cell%20of%20origin%20through%20CRISPR%20editing&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=118&publication_year=2021&author=Feng%2CW) 
    
44.  Norris, J. D. et al. The homeodomain protein HOXB13 regulates the cellular response to androgens. _Mol. Cell_ **36**, 405–416 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFOlt7jJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19917249)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2788777)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20homeodomain%20protein%20HOXB13%20regulates%20the%20cellular%20response%20to%20androgens&journal=Mol.%20Cell&volume=36&pages=405-416&publication_year=2009&author=Norris%2CJD) 
    
45.  Wang, Q. et al. A hierarchical network of transcription factors governs androgen receptor-dependent prostate cancer growth. _Mol. Cell_ **27**, 380–392 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17679089)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3947890)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20hierarchical%20network%20of%20transcription%20factors%20governs%20androgen%20receptor-dependent%20prostate%20cancer%20growth&journal=Mol.%20Cell&volume=27&pages=380-392&publication_year=2007&author=Wang%2CQ) 
    
46.  Chen, Y. W. et al. TMPRSS2, a serine protease expressed in the prostate on the apical surface of luminal epithelial cells and released into semen in prostasomes, is misregulated in prostate cancer cells. _Am. J. Pathol._ **176**, 2986–2996 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXptFKrtr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20382709)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2877858)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TMPRSS2%2C%20a%20serine%20protease%20expressed%20in%20the%20prostate%20on%20the%20apical%20surface%20of%20luminal%20epithelial%20cells%20and%20released%20into%20semen%20in%20prostasomes%2C%20is%20misregulated%20in%20prostate%20cancer%20cells&journal=Am.%20J.%20Pathol.&volume=176&pages=2986-2996&publication_year=2010&author=Chen%2CYW) 
    
47.  Gao, D. et al. A Tmprss2-CreERT2 knock-in mouse model for cancer genetic studies on prostate and colon. _PLoS ONE_ **11**, e0161084 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27536883)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990297)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20Tmprss2-CreERT2%20knock-in%20mouse%20model%20for%20cancer%20genetic%20studies%20on%20prostate%20and%20colon&journal=PLoS%20ONE&volume=11&publication_year=2016&author=Gao%2CD) 
    
48.  Karthaus, W. R. et al. Regenerative potential of prostate luminal cells revealed by single-cell analysis. _Science_ **368**, 497–505 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXosVSjsb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32355025)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7313621)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regenerative%20potential%20of%20prostate%20luminal%20cells%20revealed%20by%20single-cell%20analysis&journal=Science&volume=368&pages=497-505&publication_year=2020&author=Karthaus%2CWR) 
    
49.  Guo, W. et al. JAK/STAT signaling maintains an intermediate cell population during prostate basal cell fate determination. _Nat. Genet._ **56**, 2776–2789 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXisV2nu7%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39537874)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=JAK%2FSTAT%20signaling%20maintains%20an%20intermediate%20cell%20population%20during%20prostate%20basal%20cell%20fate%20determination&journal=Nat.%20Genet.&volume=56&pages=2776-2789&publication_year=2024&author=Guo%2CW) 
    
50.  Luo, F. et al. A luminal intermediate cell state maintains long-term prostate homeostasis and contributes to tumorigenesis. Preprint at _bioRxiv_ [https://doi.org/10.1101/2023.02.24.529762](https://doi.org/10.1101/2023.02.24.529762) (2023).
    
51.  Hirz, T. et al. Dissecting the immune suppressive human prostate tumor microenvironment via integrated single-cell and spatial transcriptomic analyses. _Nat. Commun._ **14**, 663 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtVagu7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36750562)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9905093)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dissecting%20the%20immune%20suppressive%20human%20prostate%20tumor%20microenvironment%20via%20integrated%20single-cell%20and%20spatial%20transcriptomic%20analyses&journal=Nat.%20Commun.&volume=14&publication_year=2023&author=Hirz%2CT) 
    
52.  Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. _Nat. Commun._ **13**, 141 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xns1ClsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35013146)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8748675)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20analysis%20of%20human%20primary%20prostate%20cancer%20reveals%20the%20heterogeneity%20of%20tumor-associated%20epithelial%20cell%20states&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Song%2CH) 
    
53.  Casey, O. M. et al. TMPRSS2-driven ERG expression in vivo increases self-renewal and maintains expression in a castration resistant subpopulation. _PLoS ONE_ **7**, e41668 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFCnsbrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22860005)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408501)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TMPRSS2-driven%20ERG%20expression%20in%20vivo%20increases%20self-renewal%20and%20maintains%20expression%20in%20a%20castration%20resistant%20subpopulation&journal=PLoS%20ONE&volume=7&publication_year=2012&author=Casey%2COM) 
    
54.  Polson, E. S. et al. Monoallelic expression of TMPRSS2/ERG in prostate cancer stem cells. _Nat. Commun._ **4**, 1623 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23535644)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Monoallelic%20expression%20of%20TMPRSS2%2FERG%20in%20prostate%20cancer%20stem%20cells&journal=Nat.%20Commun.&volume=4&publication_year=2013&author=Polson%2CES) 
    
55.  Lin, B. et al. Prostate-localized and androgen-regulated expression of the membrane-bound serine protease TMPRSS2. _Cancer Res._ **59**, 4180–4184 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXlvVSnurc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10485450)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prostate-localized%20and%20androgen-regulated%20expression%20of%20the%20membrane-bound%20serine%20protease%20TMPRSS2&journal=Cancer%20Res.&volume=59&pages=4180-4184&publication_year=1999&author=Lin%2CB) 
    
56.  Aytes, A. et al. ETV4 promotes metastasis in response to activation of PI3-kinase and Ras signaling in a mouse model of advanced prostate cancer. _Proc. Natl Acad. Sci. USA_ **110**, E3506–E3515 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFWrs7vI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23918374)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773788)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETV4%20promotes%20metastasis%20in%20response%20to%20activation%20of%20PI3-kinase%20and%20Ras%20signaling%20in%20a%20mouse%20model%20of%20advanced%20prostate%20cancer&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=110&pages=E3506-E3515&publication_year=2013&author=Aytes%2CA) 
    
57.  Zou, M. et al. Transdifferentiation as a mechanism of treatment resistance in a mouse model of castration-resistant prostate cancer. _Cancer Discov._ **7**, 736–749 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFCisrbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28411207)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501744)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transdifferentiation%20as%20a%20mechanism%20of%20treatment%20resistance%20in%20a%20mouse%20model%20of%20castration-resistant%20prostate%20cancer&journal=Cancer%20Discov.&volume=7&pages=736-749&publication_year=2017&author=Zou%2CM) 
    
58.  Van Keymeulen, A. et al. Distinct stem cells contribute to mammary gland development and maintenance. _Nature_ **479**, 189–193 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21983963)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20stem%20cells%20contribute%20to%20mammary%20gland%20development%20and%20maintenance&journal=Nature&volume=479&pages=189-193&publication_year=2011&author=Keymeulen%2CA) 
    
59.  Zhang, L. et al. Targeting CreERT2 expression to keratin 8-expressing murine simple epithelia using bacterial artificial chromosome transgenesis. _Transgenic Res._ **21**, 1117–1123 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1yrt7rP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22350718)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3433578)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20CreERT2%20expression%20to%20keratin%208-expressing%20murine%20simple%20epithelia%20using%20bacterial%20artificial%20chromosome%20transgenesis&journal=Transgenic%20Res.&volume=21&pages=1117-1123&publication_year=2012&author=Zhang%2CL) 
    
60.  Hudson, D. L. et al. Epithelial cell differentiation pathways in the human prostate: identification of intermediate phenotypes by keratin expression. _J. Histochem. Cytochem._ **49**, 271–278 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXpsFGgtA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11156695)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epithelial%20cell%20differentiation%20pathways%20in%20the%20human%20prostate%3A%20identification%20of%20intermediate%20phenotypes%20by%20keratin%20expression&journal=J.%20Histochem.%20Cytochem.&volume=49&pages=271-278&publication_year=2001&author=Hudson%2CDL) 
    
61.  Peehl, D. M., Sellers, R. G. & McNeal, J. E. Keratin 19 in the adult human prostate: tissue and cell culture studies. _Cell Tissue Res._ **285**, 171–176 (1996).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28Xlt1Oisbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8766871)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Keratin%2019%20in%20the%20adult%20human%20prostate%3A%20tissue%20and%20cell%20culture%20studies&journal=Cell%20Tissue%20Res.&volume=285&pages=171-176&publication_year=1996&author=Peehl%2CDM&author=Sellers%2CRG&author=McNeal%2CJE) 
    
62.  Robinson, E. J., Neal, D. E. & Collins, A. T. Basal cells are progenitors of luminal cells in primary cultures of differentiating human prostatic epithelium. _Prostate_ **37**, 149–160 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK1M%2Fgs1Kiug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9792132)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Basal%20cells%20are%20progenitors%20of%20luminal%20cells%20in%20primary%20cultures%20of%20differentiating%20human%20prostatic%20epithelium&journal=Prostate&volume=37&pages=149-160&publication_year=1998&author=Robinson%2CEJ&author=Neal%2CDE&author=Collins%2CAT) 
    
63.  Taylor, R. A., Toivanen, R. & Risbridger, G. P. Stem cells in prostate cancer: treating the root of the problem. _Endocr. Relat. Cancer_ **17**, R273–R285 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXislyms7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20660571)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stem%20cells%20in%20prostate%20cancer%3A%20treating%20the%20root%20of%20the%20problem&journal=Endocr.%20Relat.%20Cancer&volume=17&pages=R273-R285&publication_year=2010&author=Taylor%2CRA&author=Toivanen%2CR&author=Risbridger%2CGP) 
    
64.  van Leenders, G., Dijkman, H., Hulsbergen-van de Kaa, C., Ruiter, D. & Schalken, J. Demonstration of intermediate cells during human prostate epithelial differentiation in situ and in vitro using triple-staining confocal scanning microscopy. _Lab. Invest._ **80**, 1251–1258 (2000).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10950116)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Demonstration%20of%20intermediate%20cells%20during%20human%20prostate%20epithelial%20differentiation%20in%20situ%20and%20in%20vitro%20using%20triple-staining%20confocal%20scanning%20microscopy&journal=Lab.%20Invest.&volume=80&pages=1251-1258&publication_year=2000&author=Leenders%2CG&author=Dijkman%2CH&author=Hulsbergen-van%20de%20Kaa%2CC&author=Ruiter%2CD&author=Schalken%2CJ) 
    
65.  Verhagen, A. P. et al. Colocalization of basal and luminal cell-type cytokeratins in human prostate cancer. _Cancer Res._ **52**, 6182–6187 (1992).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK3s%2Fks1WitA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1384957)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Colocalization%20of%20basal%20and%20luminal%20cell-type%20cytokeratins%20in%20human%20prostate%20cancer&journal=Cancer%20Res.&volume=52&pages=6182-6187&publication_year=1992&author=Verhagen%2CAP) 
    
66.  Wang, Y., Hayward, S., Cao, M., Thayer, K. & Cunha, G. Cell differentiation lineage in the prostate. _Differentiation_ **68**, 270–279 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD38%2Fks1KqsQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11776479)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell%20differentiation%20lineage%20in%20the%20prostate&journal=Differentiation&volume=68&pages=270-279&publication_year=2001&author=Wang%2CY&author=Hayward%2CS&author=Cao%2CM&author=Thayer%2CK&author=Cunha%2CG) 
    
67.  Xue, Y., Smedts, F., Debruyne, F. M., de la Rosette, J. J. & Schalken, J. A. Identification of intermediate cell types by keratin expression in the developing human prostate. _Prostate_ **34**, 292–301 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXhvVynsL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9496904)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20intermediate%20cell%20types%20by%20keratin%20expression%20in%20the%20developing%20human%20prostate&journal=Prostate&volume=34&pages=292-301&publication_year=1998&author=Xue%2CY&author=Smedts%2CF&author=Debruyne%2CFM&author=Rosette%2CJJ&author=Schalken%2CJA) 
    
68.  Chen, Z. et al. Crucial role of p53-dependent cellular senescence in suppression of Pten-deficient tumorigenesis. _Nature_ **436**, 725–730 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXmvFenu78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16079851)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1939938)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Crucial%20role%20of%20p53-dependent%20cellular%20senescence%20in%20suppression%20of%20Pten-deficient%20tumorigenesis&journal=Nature&volume=436&pages=725-730&publication_year=2005&author=Chen%2CZ) 
    
69.  Brina, D. et al. The Akt/mTOR and MNK/eIF4E pathways rewire the prostate cancer translatome to secrete HGF, SPP1 and BGN and recruit suppressive myeloid cells. _Nat. Cancer_ **4**, 1102–1121 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFaitrfN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37460872)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11331482)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Akt%2FmTOR%20and%20MNK%2FeIF4E%20pathways%20rewire%20the%20prostate%20cancer%20translatome%20to%20secrete%20HGF%2C%20SPP1%20and%20BGN%20and%20recruit%20suppressive%20myeloid%20cells&journal=Nat.%20Cancer&volume=4&pages=1102-1121&publication_year=2023&author=Brina%2CD) 
    
70.  Crowley, L. et al. A single-cell atlas of the mouse and human prostate reveals heterogeneity and conservation of epithelial progenitors. _eLife_ **9**, e59465 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVegtLjL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32915138)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7529463)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single-cell%20atlas%20of%20the%20mouse%20and%20human%20prostate%20reveals%20heterogeneity%20and%20conservation%20of%20epithelial%20progenitors&journal=eLife&volume=9&publication_year=2020&author=Crowley%2CL) 
    
71.  Guo, W. et al. Single-cell transcriptomics identifies a distinct luminal progenitor cell type in distal prostate invagination tips. _Nat. Genet._ **52**, 908–918 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1aiurjN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32807988)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8383310)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20transcriptomics%20identifies%20a%20distinct%20luminal%20progenitor%20cell%20type%20in%20distal%20prostate%20invagination%20tips&journal=Nat.%20Genet.&volume=52&pages=908-918&publication_year=2020&author=Guo%2CW) 
    
72.  Germanos, A. A. et al. Defining cellular population dynamics at single-cell resolution during prostate cancer progression. _eLife_ **11**, e79076 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisVOltr7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36511483)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9747158)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Defining%20cellular%20population%20dynamics%20at%20single-cell%20resolution%20during%20prostate%20cancer%20progression&journal=eLife&volume=11&publication_year=2022&author=Germanos%2CAA) 
    
73.  Lin, B. et al. Airway hillocks are injury-resistant reservoirs of unique plastic stem cells. _Nature_ **629**, 869–877 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXpsl2ku7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38693267)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11890216)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Airway%20hillocks%20are%20injury-resistant%20reservoirs%20of%20unique%20plastic%20stem%20cells&journal=Nature&volume=629&pages=869-877&publication_year=2024&author=Lin%2CB) 
    
74.  Henry, G. H. et al. A cellular anatomy of the normal adult human prostate and prostatic urethra. _Cell Rep._ **25**, 3530–3542 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisFKhsbjM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30566875)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6411034)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20cellular%20anatomy%20of%20the%20normal%20adult%20human%20prostate%20and%20prostatic%20urethra&journal=Cell%20Rep.&volume=25&pages=3530-3542&publication_year=2018&author=Henry%2CGH) 
    
75.  Chen, Y. et al. Club cells employ regeneration mechanisms during lung tumorigenesis. _Nat. Commun._ **13**, 4557 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFWlsrbP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35931677)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9356049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Club%20cells%20employ%20regeneration%20mechanisms%20during%20lung%20tumorigenesis&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Chen%2CY) 
    
76.  Huang, F. W. et al. Club-like cells in proliferative inflammatory atrophy of the prostate. _J. Pathol._ **261**, 85–95 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslWgtLbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37550827)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10527202)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Club-like%20cells%20in%20proliferative%20inflammatory%20atrophy%20of%20the%20prostate&journal=J.%20Pathol.&volume=261&pages=85-95&publication_year=2023&author=Huang%2CFW) 
    
77.  Setty, M. et al. Characterization of cell fate probabilities in single-cell data with Palantir. _Nat. Biotechnol._ **37**, 451–460 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosV2qs74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30899105)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7549125)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterization%20of%20cell%20fate%20probabilities%20in%20single-cell%20data%20with%20Palantir&journal=Nat.%20Biotechnol.&volume=37&pages=451-460&publication_year=2019&author=Setty%2CM) 
    
78.  Signoretti, S. et al. p63 is a prostate basal cell marker and is required for prostate development. _Am. J. Pathol._ **157**, 1769–1775 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhsVKktw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11106548)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1885786)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=p63%20is%20a%20prostate%20basal%20cell%20marker%20and%20is%20required%20for%20prostate%20development&journal=Am.%20J.%20Pathol.&volume=157&pages=1769-1775&publication_year=2000&author=Signoretti%2CS) 
    
79.  Adams, E. J. et al. FOXA1 mutations alter pioneering activity, differentiation and prostate cancer phenotypes. _Nature_ **571**, 408–412 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1ykt7fL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31243370)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6661172)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FOXA1%20mutations%20alter%20pioneering%20activity%2C%20differentiation%20and%20prostate%20cancer%20phenotypes&journal=Nature&volume=571&pages=408-412&publication_year=2019&author=Adams%2CEJ) 
    
80.  Asselin-Labat, M. L. et al. Gata-3 is an essential regulator of mammary-gland morphogenesis and luminal-cell differentiation. _Nat. Cell Biol._ **9**, 201–209 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFGlsbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17187062)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gata-3%20is%20an%20essential%20regulator%20of%20mammary-gland%20morphogenesis%20and%20luminal-cell%20differentiation&journal=Nat.%20Cell%20Biol.&volume=9&pages=201-209&publication_year=2007&author=Asselin-Labat%2CML) 
    
81.  Kouros-Mehr, H., Slorach, E. M., Sternlicht, M. D. & Werb, Z. GATA-3 maintains the differentiation of the luminal cell fate in the mammary gland. _Cell_ **127**, 1041–1055 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtlWktrzP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17129787)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2646406)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GATA-3%20maintains%20the%20differentiation%20of%20the%20luminal%20cell%20fate%20in%20the%20mammary%20gland&journal=Cell&volume=127&pages=1041-1055&publication_year=2006&author=Kouros-Mehr%2CH&author=Slorach%2CEM&author=Sternlicht%2CMD&author=Werb%2CZ) 
    
82.  Parolia, A. et al. Distinct structural classes of activating FOXA1 alterations in advanced prostate cancer. _Nature_ **571**, 413–418 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1yktL%2FO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31243372)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6661908)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20structural%20classes%20of%20activating%20FOXA1%20alterations%20in%20advanced%20prostate%20cancer&journal=Nature&volume=571&pages=413-418&publication_year=2019&author=Parolia%2CA) 
    
83.  Pomerantz, M. M. et al. The androgen receptor cistrome is extensively reprogrammed in human prostate tumorigenesis. _Nat. Genet._ **47**, 1346–1351 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslant7bN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26457646)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707683)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20androgen%20receptor%20cistrome%20is%20extensively%20reprogrammed%20in%20human%20prostate%20tumorigenesis&journal=Nat.%20Genet.&volume=47&pages=1346-1351&publication_year=2015&author=Pomerantz%2CMM) 
    
84.  Krivtsov, A. V. et al. A Menin-MLL inhibitor induces specific chromatin changes and eradicates disease in models of MLL-rearranged leukemia. _Cancer Cell_ **36**, 660–673 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlOmtLrN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31821784)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7227117)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20Menin-MLL%20inhibitor%20induces%20specific%20chromatin%20changes%20and%20eradicates%20disease%20in%20models%20of%20MLL-rearranged%20leukemia&journal=Cancer%20Cell&volume=36&pages=660-673&publication_year=2019&author=Krivtsov%2CAV) 
    
85.  Issa, G. C. et al. The menin inhibitor revumenib in KMT2A-rearranged or NPM1-mutant leukaemia. _Nature_ **615**, 920–924 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXltlKitbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36922593)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10060155)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20menin%20inhibitor%20revumenib%20in%20KMT2A-rearranged%20or%20NPM1-mutant%20leukaemia&journal=Nature&volume=615&pages=920-924&publication_year=2023&author=Issa%2CGC) 
    
86.  Issa, G. C. et al. Menin inhibition with revumenib for KMT2A-rearranged relapsed or refractory acute leukemia (AUGMENT-101). _J. Clin. Oncol._ **43**, 75–84 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXivVKhu7zO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39121437)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Menin%20inhibition%20with%20revumenib%20for%20KMT2A-rearranged%20relapsed%20or%20refractory%20acute%20leukemia%20%28AUGMENT-101%29&journal=J.%20Clin.%20Oncol.&volume=43&pages=75-84&publication_year=2025&author=Issa%2CGC) 
    
87.  Sarno, F., Nebbioso, A. & Altucci, L. DOT1L: a key target in normal chromatin remodelling and in mixed-lineage leukaemia treatment. _Epigenetics_ **15**, 439–453 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31790636)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DOT1L%3A%20a%20key%20target%20in%20normal%20chromatin%20remodelling%20and%20in%20mixed-lineage%20leukaemia%20treatment&journal=Epigenetics&volume=15&pages=439-453&publication_year=2020&author=Sarno%2CF&author=Nebbioso%2CA&author=Altucci%2CL) 
    
88.  Vatapalli, R. et al. Histone methyltransferase DOT1L coordinates AR and MYC stability in prostate cancer. _Nat. Commun._ **11**, 4153 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Kltb3O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32814769)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7438336)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Histone%20methyltransferase%20DOT1L%20coordinates%20AR%20and%20MYC%20stability%20in%20prostate%20cancer&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Vatapalli%2CR) 
    
89.  Deshpande, A. J. et al. Leukemic transformation by the MLL-AF6 fusion oncogene requires the H3K79 methyltransferase Dot1l. _Blood_ **121**, 2533–2541 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXlsV2jsL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23361907)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3612861)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Leukemic%20transformation%20by%20the%20MLL-AF6%20fusion%20oncogene%20requires%20the%20H3K79%20methyltransferase%20Dot1l&journal=Blood&volume=121&pages=2533-2541&publication_year=2013&author=Deshpande%2CAJ) 
    
90.  Malik, R. et al. Targeting the MLL complex in castration-resistant prostate cancer. _Nat. Med._ **21**, 344–352 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsVOnsrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25822367)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4390530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Targeting%20the%20MLL%20complex%20in%20castration-resistant%20prostate%20cancer&journal=Nat.%20Med.&volume=21&pages=344-352&publication_year=2015&author=Malik%2CR) 
    
91.  Hemming, M. L. et al. MOZ and Menin-MLL complexes are complementary regulators of chromatin association and transcriptional output in gastrointestinal stromal tumor. _Cancer Discov._ **12**, 1804–1823 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1KlsrrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35499757)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9453853)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MOZ%20and%20Menin-MLL%20complexes%20are%20complementary%20regulators%20of%20chromatin%20association%20and%20transcriptional%20output%20in%20gastrointestinal%20stromal%20tumor&journal=Cancer%20Discov.&volume=12&pages=1804-1823&publication_year=2022&author=Hemming%2CML) 
    
92.  Chi, P. et al. ETV1 is a lineage survival factor that cooperates with KIT in gastrointestinal stromal tumours. _Nature_ **467**, 849–853 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1elu73N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20927104)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2955195)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETV1%20is%20a%20lineage%20survival%20factor%20that%20cooperates%20with%20KIT%20in%20gastrointestinal%20stromal%20tumours&journal=Nature&volume=467&pages=849-853&publication_year=2010&author=Chi%2CP) 
    
93.  Deininger, M. W., Goldman, J. M. & Melo, J. V. The molecular biology of chronic myeloid leukemia. _Blood_ **96**, 3343–3356 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXotFKktb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11071626)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20molecular%20biology%20of%20chronic%20myeloid%20leukemia&journal=Blood&volume=96&pages=3343-3356&publication_year=2000&author=Deininger%2CMW&author=Goldman%2CJM&author=Melo%2CJV) 
    
94.  Vetrie, D., Helgason, G. V. & Copland, M. The leukaemia stem cell: similarities, differences and clinical prospects in CML and AML. _Nat. Rev. Cancer_ **20**, 158–173 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjvF2msg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31907378)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20leukaemia%20stem%20cell%3A%20similarities%2C%20differences%20and%20clinical%20prospects%20in%20CML%20and%20AML&journal=Nat.%20Rev.%20Cancer&volume=20&pages=158-173&publication_year=2020&author=Vetrie%2CD&author=Helgason%2CGV&author=Copland%2CM) 
    
95.  Wu, X. et al. Generation of a prostate epithelial cell-specific Cre transgenic mouse model for tissue-specific gene ablation. _Mech. Dev._ **101**, 61–69 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhsFSht7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11231059)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Generation%20of%20a%20prostate%20epithelial%20cell-specific%20Cre%20transgenic%20mouse%20model%20for%20tissue-specific%20gene%20ablation&journal=Mech.%20Dev.&volume=101&pages=61-69&publication_year=2001&author=Wu%2CX) 
    
96.  Trotman, L. C. et al. Pten dose dictates cancer progression in the prostate. _PLoS Biol._ **1**, E59 (2003).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14691534)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC270016)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pten%20dose%20dictates%20cancer%20progression%20in%20the%20prostate&journal=PLoS%20Biol.&volume=1&publication_year=2003&author=Trotman%2CLC) 
    
97.  Madisen, L. et al. A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. _Nat. Neurosci._ **13**, 133–140 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFyhsLrO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20023653)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20robust%20and%20high-throughput%20Cre%20reporting%20and%20characterization%20system%20for%20the%20whole%20mouse%20brain&journal=Nat.%20Neurosci.&volume=13&pages=133-140&publication_year=2010&author=Madisen%2CL) 
    
98.  Vasioukhin, V., Degenstein, L., Wise, B. & Fuchs, E. The magical touch: genome targeting in epidermal stem cells induced by tamoxifen application to mouse skin. _Proc. Natl Acad. Sci. USA_ **96**, 8551–8556 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXkslOktro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10411913)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC17554)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20magical%20touch%3A%20genome%20targeting%20in%20epidermal%20stem%20cells%20induced%20by%20tamoxifen%20application%20to%20mouse%20skin&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=96&pages=8551-8556&publication_year=1999&author=Vasioukhin%2CV&author=Degenstein%2CL&author=Wise%2CB&author=Fuchs%2CE) 
    
99.  Drost, J. et al. Organoid culture systems for prostate epithelial and cancer tissue. _Nat. Protoc._ **11**, 347–358 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xos1Ortg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26797458)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4793718)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Organoid%20culture%20systems%20for%20prostate%20epithelial%20and%20cancer%20tissue&journal=Nat.%20Protoc.&volume=11&pages=347-358&publication_year=2016&author=Drost%2CJ) 
    
100.  Karthaus, W. R. et al. Identification of multipotent luminal progenitor cells in human prostate organoid cultures. _Cell_ **159**, 163–175 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFagsLfM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25201529)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772677)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20multipotent%20luminal%20progenitor%20cells%20in%20human%20prostate%20organoid%20cultures&journal=Cell&volume=159&pages=163-175&publication_year=2014&author=Karthaus%2CWR) 
    
101.  Lawson, D. A., Xin, L., Lukacs, R. U., Cheng, D. & Witte, O. N. Isolation and functional characterization of murine prostate stem cells. _Proc. Natl Acad. Sci. USA_ **104**, 181–186 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjt1Ojug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17185413)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Isolation%20and%20functional%20characterization%20of%20murine%20prostate%20stem%20cells&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=104&pages=181-186&publication_year=2007&author=Lawson%2CDA&author=Xin%2CL&author=Lukacs%2CRU&author=Cheng%2CD&author=Witte%2CON) 
    
102.  Leow, C. C., Wang, X. D. & Gao, W. Q. Novel method of generating prostate-specific Cre-LoxP gene switching via intraductal delivery of adenovirus. _Prostate_ **65**, 1–9 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtFSns73N)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15791629)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Novel%20method%20of%20generating%20prostate-specific%20Cre-LoxP%20gene%20switching%20via%20intraductal%20delivery%20of%20adenovirus&journal=Prostate&volume=65&pages=1-9&publication_year=2005&author=Leow%2CCC&author=Wang%2CXD&author=Gao%2CWQ) 
    
103.  DuPage, M., Dooley, A. L. & Jacks, T. Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase. _Nat. Protoc._ **4**, 1064–1072 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXovFSrtLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19561589)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2757265)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conditional%20mouse%20lung%20cancer%20models%20using%20adenoviral%20or%20lentiviral%20delivery%20of%20Cre%20recombinase&journal=Nat.%20Protoc.&volume=4&pages=1064-1072&publication_year=2009&author=DuPage%2CM&author=Dooley%2CAL&author=Jacks%2CT) 
    
104.  Park, S. et al. Novel mouse models of bladder cancer identify a prognostic signature associated with risk of disease progression. _Cancer Res._ **81**, 5161–5175 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1Cjsr7K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34470779)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8609963)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Novel%20mouse%20models%20of%20bladder%20cancer%20identify%20a%20prognostic%20signature%20associated%20with%20risk%20of%20disease%20progression&journal=Cancer%20Res.&volume=81&pages=5161-5175&publication_year=2021&author=Park%2CS) 
    
105.  Tao, L., van Bragt, M. P., Laudadio, E. & Li, Z. Lineage tracing of mammary epithelial cells using cell-type-specific cre-expressing adenoviruses. _Stem Cell Rep._ **2**, 770–779 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtVWmt7vM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lineage%20tracing%20of%20mammary%20epithelial%20cells%20using%20cell-type-specific%20cre-expressing%20adenoviruses&journal=Stem%20Cell%20Rep.&volume=2&pages=770-779&publication_year=2014&author=Tao%2CL&author=Bragt%2CMP&author=Laudadio%2CE&author=Li%2CZ) 
    
106.  Schmid-Burgk, J. L., Honing, K., Ebert, T. S. & Hornung, V. CRISPaint allows modular base-specific gene tagging using a ligase-4-dependent mechanism. _Nat. Commun._ **7**, 12338 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhtlaju7%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27465542)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4974478)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CRISPaint%20allows%20modular%20base-specific%20gene%20tagging%20using%20a%20ligase-4-dependent%20mechanism&journal=Nat.%20Commun.&volume=7&publication_year=2016&author=Schmid-Burgk%2CJL&author=Honing%2CK&author=Ebert%2CTS&author=Hornung%2CV) 
    
107.  Artegiani, B. et al. Fast and efficient generation of knock-in human organoids using homology-independent CRISPR-Cas9 precision genome editing. _Nat. Cell Biol._ **22**, 321–331 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktFeksbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32123335)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20efficient%20generation%20of%20knock-in%20human%20organoids%20using%20homology-independent%20CRISPR-Cas9%20precision%20genome%20editing&journal=Nat.%20Cell%20Biol.&volume=22&pages=321-331&publication_year=2020&author=Artegiani%2CB) 
    
108.  Sanjana, N. E., Shalem, O. & Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. _Nat. Methods_ **11**, 783–784 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslelsLjK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25075903)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4486245)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improved%20vectors%20and%20genome-wide%20libraries%20for%20CRISPR%20screening&journal=Nat.%20Methods&volume=11&pages=783-784&publication_year=2014&author=Sanjana%2CNE&author=Shalem%2CO&author=Zhang%2CF) 
    
109.  Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. _BMC Bioinformatics_ **14**, 128 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23586463)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3637064)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enrichr%3A%20interactive%20and%20collaborative%20HTML5%20gene%20list%20enrichment%20analysis%20tool&journal=BMC%20Bioinformatics&volume=14&publication_year=2013&author=Chen%2CEY) 
    
110.  Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. _Nucleic Acids Res._ **44**, W90–W97 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27141961)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987924)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Enrichr%3A%20a%20comprehensive%20gene%20set%20enrichment%20analysis%20web%20server%202016%20update&journal=Nucleic%20Acids%20Res.&volume=44&pages=W90-W97&publication_year=2016&author=Kuleshov%2CMV) 
    
111.  Xie, Z. et al. Gene set knowledge discovery with Enrichr. _Curr. Protoc._ **1**, e90 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXntl2gtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33780170)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8152575)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20set%20knowledge%20discovery%20with%20Enrichr&journal=Curr.%20Protoc.&volume=1&publication_year=2021&author=Xie%2CZ) 
    
112.  Hao, Y. et al. Integrated analysis of multimodal single-cell data. _Cell_ **184**, 3573–3587 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtrvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34062119)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238499)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&journal=Cell&volume=184&pages=3573-3587&publication_year=2021&author=Hao%2CY) 
    
113.  Korotkevich, G. et al. Fast gene set enrichment analysis. Preprint at _bioRxiv_ [https://doi.org/10.1101/060012](https://doi.org/10.1101/060012) (2021).
    
114.  Granja, J. M. et al. Single-cell multiomic analysis identifies regulatory programs in mixed-phenotype acute leukemia. _Nat. Biotechnol._ **37**, 1458–1465 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1yjsrvF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31792411)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7258684)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20multiomic%20analysis%20identifies%20regulatory%20programs%20in%20mixed-phenotype%20acute%20leukemia&journal=Nat.%20Biotechnol.&volume=37&pages=1458-1465&publication_year=2019&author=Granja%2CJM) 
    
115.  Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). _Genome Biol._ **9**, R137 (2008).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18798982)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Model-based%20analysis%20of%20ChIP-Seq%20%28MACS%29&journal=Genome%20Biol.&volume=9&publication_year=2008&author=Zhang%2CY) 
    
116.  Vierstra, J. et al. Global reference mapping of human transcription factor footprints. _Nature_ **583**, 729–736 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVygtLbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32728250)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7410829)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Global%20reference%20mapping%20of%20human%20transcription%20factor%20footprints&journal=Nature&volume=583&pages=729-736&publication_year=2020&author=Vierstra%2CJ) 
    
117.  Schep, A. N., Wu, B., Buenrostro, J. D. & Greenleaf, W. J. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. _Nat. Methods_ **14**, 975–978 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlKjtbbE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28825706)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5623146)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=chromVAR%3A%20inferring%20transcription-factor-associated%20accessibility%20from%20single-cell%20epigenomic%20data&journal=Nat.%20Methods&volume=14&pages=975-978&publication_year=2017&author=Schep%2CAN&author=Wu%2CB&author=Buenrostro%2CJD&author=Greenleaf%2CWJ) 
    
118.  Feng, W. et al. ERG-driven prostate cancer initiation is cell context dependent and requires KMT2A and DOT1L. _Zenodo_ [https://doi.org/10.5281/zenodo.15802144](https://doi.org/10.5281/zenodo.15802144) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02289-w?format=refman&flavour=references)

Acknowledgements
----------------

We thank the Sawyers lab for valuable critiques and discussions. We appreciate the feedback from M. M. Shen and C. Abate-Shen, and advice from the C. Abate-Shen lab for setting up the intraprostatic adenoviral injection assay. We are grateful to R. Lester for help in mouse colony management, the Molecular Cytology Core Facility from MSKCC for help with microscopy, the Single Cell Analytics Innovation Lab Core Facility for help with single-cell library preparation, the Integrated Genomics Operation Core Facility for next-generation sequencing and the Flow Cytometry Core Facility from MSKCC for help with FACS experiments. This work is supported by the following funding sources: the Howard Hughes Medical Institute (HHMI) (C.L.S.); National Institute of Health grants no. CA193837 (Y.C., C.L.S.), no. CA092629 (Y.C., C.L.S.), no. CA224079 (B.S.C., Y.C., C.L.S.), no. CA224044 (Y.C.), no. CA208100 (Y.C.), no. CA155169 (C.L.S.), no. CA276888-01 (W.F.), no. CA257811 (B.S.C.), no. CA008748 (H.Z., W.K., N.F., E.R., E.D.S., Y.C.), no. DK128602-03 (H.L.) and no. CA274492 (E.L., C.S.L.); Department of Defense grants no. W81XWH-19-1-0323 (W.F.) and no. W81XWH-21-1-0431 (B.S.C.); the Prostate Cancer Foundation (W.F., Y.C.); and the STARR Cancer Consortium (Y.C.).

Author information
------------------

Author notes

1.  Weiran Feng & Hanzhi Luo
    
    Present address: Nuclear Dynamics and Cancer Program, Fox Chase Cancer Center, Philadelphia, PA, USA
    
2.  Weiran Feng & Hanzhi Luo
    
    Present address: Cancer Epigenetics Institute, Fox Chase Cancer Center, Philadelphia, PA, USA
    
3.  These authors contributed equally: Weiran Feng, Erik Ladewig, Matthew Lange, Nazifa Salsabeel, Huiyong Zhao.
    

### Authors and Affiliations

1.  Human Oncology and Pathogenesis Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Weiran Feng, Matthew Lange, Nazifa Salsabeel, Young Sun Lee, Yu Chen, Brett S. Carver & Charles L. Sawyers
    
2.  Computational and Systems Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Erik Ladewig & Christina S. Leslie
    
3.  Antitumor Assessment Core Facility, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Huiyong Zhao & Elisa de Stanchina
    
4.  Department of Pathology, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Anuradha Gopalan
    
5.  Molecular Pharmacology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Hanzhi Luo
    
6.  Molecular Cytology Core Facility, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Wenfei Kang, Ning Fan & Eric Rosiek
    
7.  Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Brett S. Carver
    
8.  Division of Urology, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Brett S. Carver
    
9.  Howard Hughes Medical Institute, Memorial Sloan Kettering Cancer Center, New York, NY, USA
    
    Charles L. Sawyers
    

Authors

1.  Weiran Feng
    
    [View author publications](https://www.nature.com/search?author=Weiran%20Feng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Weiran%20Feng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Weiran%20Feng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Erik Ladewig
    
    [View author publications](https://www.nature.com/search?author=Erik%20Ladewig)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Erik%20Ladewig) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Erik%20Ladewig%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Matthew Lange
    
    [View author publications](https://www.nature.com/search?author=Matthew%20Lange)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20Lange) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20Lange%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Nazifa Salsabeel
    
    [View author publications](https://www.nature.com/search?author=Nazifa%20Salsabeel)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nazifa%20Salsabeel) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nazifa%20Salsabeel%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Huiyong Zhao
    
    [View author publications](https://www.nature.com/search?author=Huiyong%20Zhao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Huiyong%20Zhao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Huiyong%20Zhao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Young Sun Lee
    
    [View author publications](https://www.nature.com/search?author=Young%20Sun%20Lee)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Young%20Sun%20Lee) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Young%20Sun%20Lee%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Anuradha Gopalan
    
    [View author publications](https://www.nature.com/search?author=Anuradha%20Gopalan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anuradha%20Gopalan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anuradha%20Gopalan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Hanzhi Luo
    
    [View author publications](https://www.nature.com/search?author=Hanzhi%20Luo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hanzhi%20Luo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hanzhi%20Luo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Wenfei Kang
    
    [View author publications](https://www.nature.com/search?author=Wenfei%20Kang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenfei%20Kang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenfei%20Kang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Ning Fan
    
    [View author publications](https://www.nature.com/search?author=Ning%20Fan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ning%20Fan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ning%20Fan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Eric Rosiek
    
    [View author publications](https://www.nature.com/search?author=Eric%20Rosiek)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eric%20Rosiek) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eric%20Rosiek%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Elisa de Stanchina
    
    [View author publications](https://www.nature.com/search?author=Elisa%20de%20Stanchina)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Elisa%20de%20Stanchina) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Elisa%20de%20Stanchina%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Yu Chen
    
    [View author publications](https://www.nature.com/search?author=Yu%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yu%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yu%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Brett S. Carver
    
    [View author publications](https://www.nature.com/search?author=Brett%20S.%20Carver)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Brett%20S.%20Carver) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Brett%20S.%20Carver%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Christina S. Leslie
    
    [View author publications](https://www.nature.com/search?author=Christina%20S.%20Leslie)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Christina%20S.%20Leslie) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Christina%20S.%20Leslie%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Charles L. Sawyers
    
    [View author publications](https://www.nature.com/search?author=Charles%20L.%20Sawyers)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Charles%20L.%20Sawyers) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Charles%20L.%20Sawyers%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Conceptualization: W.F. and C.L.S. Methodology: W.F., E.L., H.Z., H.L. and C.S.L. Investigation: W.F., E.L., N.S., H.Z., Y.S.L., A.G., M.L., W.K., N.F. and E.R. Formal analysis: W.F. and E.L. Resources: Y.C. and B.S.C. Project administration: E.D.S. Funding acquisition: W.F. and C.L.S. Software: E.L. and C.S.L. Supervision: C.S.L. and C.L.S. Writing—original draft: W.F., E.L., C.S.L. and C.L.S. Writing—review and editing: W.F., E.L., C.S.L. and C.L.S.

### Corresponding authors

Correspondence to [Weiran Feng](mailto:weiran.feng@fccc.edu) or [Charles L. Sawyers](mailto:sawyersc@mskcc.org).

Ethics declarations
-------------------

### Competing interests

C.L.S. served on the Board of Directors of Novartis (2013–2025), is a co-founder of ORIC Pharmaceuticals and is co-inventor of enzalutamide and apalutamide. He is a science advisor to Beigene, Blueprint, CellCarta, Column Group, Foghorn, Housey Pharma, Nextech and PMV. Y.C. received research funding from Foghorn therapeutics, consulted for Fog Pharma and Belharra Therapeutics and has stock ownership of ORIC Pharmaceuticals. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Arul Chinnaiyan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02289-w#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Fresh cell transplantation and lineage tracing for gene activation from basal and luminal cells.](https://www.nature.com/articles/s41588-025-02289-w/figures/9)

(**a**) Quality control for freshly isolated basal and luminal-derived cells generated in Fig. [1a](https://www.nature.com/articles/s41588-025-02289-w#Fig1). (Top) Post-sort analysis of sorted basal and luminal populations. (Bottom) Assessment of Cre recombination efficiency at 5 days post Cre. GFP was used as a surrogate for ERG expression. (**b**) Images showing prostates harvested at 5 months endpoint. Visible grafts are highlighted in yellow circles. (**c**) Weights of indicated orthografts harvested at the 5 months endpoint (n = 6 mice per group). (**d**) Flow cytometry assessing the cell type specificity of _K5-_ and _K8-CreER__T2_. Whole prostates were harvested for analysis. (**e**) Schematic summarizing that lineage tracing using _K5-CreER__T2_ and _K14-CreER_ to activate _EP_ leads to a severe skin disease and premature death. (**f-g**) Survival curve (**f**) and skin histology (**g**) from indicated mice. Scale bar, 100 µm. (**h-j**) Mice at harvest (**h**), survival curve (n = 6 mice per group) (**i**) and skin histology (**j**) of indicated mice. Scale bar, 100 µm. (**k**) IF imaging assessing the cell type specificity and infection efficiency of the indicated Ad-Cre using the _Rosa26-YFP__LSL/LSL_ allele (_Y_) at the 1-week time point (n = 41 sections from 4 mice for K5-Cre group; n = 27 sections from 4 mice for K8-Cre group). Scale bars, 10 µm. (**l**) Sagittal view of whole prostates with ERG IHC. Two injection sites are shown. Scale bar, 2 mm. Data represent mean ± s.d.; ns, not significant. unpaired two-tailed t-test (**c**); Log-rank test (**f, i**).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM11)

### [Extended Data Fig. 2 A subset of basal cells express androgen-regulated luminal genes.](https://www.nature.com/articles/s41588-025-02289-w/figures/10)

((**a**) UMAP of all epithelial cells from human prostates. Data were reproduced from Song et al.[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022).") with the same cell type classification, except that the two subtypes of the previously defined prostate cancer cells are together named as PCa here (see [Methods](https://www.nature.com/articles/s41588-025-02289-w#Sec11)). The basal and PCa clusters are highlighted in black and pink circles, respectively. (**b**) UMAP of epithelial cells from normal human prostates highlighting the expression of canonical luminal genes in a subset of basal cells. An expanded list of genes beyond Fig. [1g](https://www.nature.com/articles/s41588-025-02289-w#Fig1) are shown. Normal basal clusters are highlighted in circles. Cells from normal samples are colored from white to green based on gene expression, with cells from tumor samples in grey in the background. The complete UMAPs and cell type annotations are shown in **A**. (**c**) UMAP of epithelial cells from normal mouse prostates highlighting the expression of canonical luminal genes (_Tmprss2_, _Nkx3-1_) in a subset of basal cells. Cell types from normal prostates are annotated based on Fig. [4b](https://www.nature.com/articles/s41588-025-02289-w#Fig4) and highlighted in circles. Cells from normal samples are colored from white to green based on gene expression, with cells from tumor samples in grey in the background. (**d**) Changes in expression of luminal genes expressed in a subset of basal cells during a castration-regeneration cycle, indicative of androgen dependent expression, as seen in canonical luminal cells (n = 13,398 cells). Data were mined from Karthaus et al.[49](https://www.nature.com/articles/s41588-025-02289-w#ref-CR49 "Guo, W. et al. JAK/STAT signaling maintains an intermediate cell population during prostate basal cell fate determination. Nat. Genet. 56, 2776–2789 (2024)."). The center line represents the median, the box limits represent the upper and lower quartiles and the minimum and maximum whiskers represent the minimum and maximum, respectively. (**e**) YFP+ cells that co-express P63 from _Y;Nkx3-1-CreERT2_ mice; inset shows high-power view. Mice were treated with tamoxifen at 2 months before whole prostates were harvested 1 week post treatment. Scale bars, 25 μm.

### [Extended Data Fig. 3 Evidence of ERG+ basal cells in human prostate cancer.](https://www.nature.com/articles/s41588-025-02289-w/figures/11)

(**a**) UMAP of epithelial cells from primary PCa patients (Song et al.[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022).")) showing RNA count to assess doublet potential. The low to intermediate level of RNA count in basal cluster suggest the ERG+ basal cells in Fig. [1k](https://www.nature.com/articles/s41588-025-02289-w#Fig1) are unlikely an artificial outcome of doublet formation. (**b**) UMAP of epithelial cells showing ERG expression in individual patients. ERG+ PCa samples are shown (see [Methods](https://www.nature.com/articles/s41588-025-02289-w#Sec11)). Cells from each patient are colored from white to red based on gene expression, with cells from the rest of patients in grey in the background. Arrows highlight the presence of ERG+ cells in the basal cluster. Cell types in this figure are annotated based on Extended Data Fig. [2a](https://www.nature.com/articles/s41588-025-02289-w#Fig10), with basal and PCa clusters highlighted in black and pink circles, respectively. (**c**) UMAP of epithelial cells from a different primary PCa patient cohort (Hirz et al.[52](https://www.nature.com/articles/s41588-025-02289-w#ref-CR52 "Song, H. et al. Single-cell analysis of human primary prostate cancer reveals the heterogeneity of tumor-associated epithelial cell states. Nat. Commun. 13, 141 (2022).")) showing all epithelial cells. (**d**) UMAP from **c** showing ERG+ cancer cells colored by ERG expression (top) and patient ID (bottom) from all epithelial cells (left) and PCa cells only (right).

### [Extended Data Fig. 4 Comparison of Cre drivers across different genetic backgrounds.](https://www.nature.com/articles/s41588-025-02289-w/figures/12)

(**a**) Cre activation as assessed by YFP reporter expression using flow cytometry (n = 2 mice for _K8_ group; n = 6 mice for _Nkx3-1_ group). Mice were treated with tamoxifen at 2 months before whole prostates were harvested at 1 week post treatment. (**b**) IHC documenting the presence of recombined cells resulting from _K8-CreER__T2_ vs _Nkx3-1-CreER__T2_ activity at different histological locations. Mice were treated with tamoxifen at 2 months before whole prostates were harvested 1 week post treatment. YFP was used as a surrogate for Cre recombination. Scale bars, 100 µm. (**c**) ERG expression activated by different Cre drivers, as quantified by flow cytometry from indicated mice (n = 5 mice for _K8_ group; n = 12 mice for _Nkx3-1_ group). Mice were treated with tamoxifen at 2 months before whole prostates were harvested at 1 week post treatment. ERG+ cells are shown, with ERG intensity normalized to background signal from the corresponding ERG-negative population. MFI, mean florescence intensity. (**d**) IF and IHC highlighting Nkx3.1 loss in _EP;K8-CreER__T2_ mice. Mice were harvested at 1 month post tamoxifen. High-power view in insets highlights the absence of Nkx3.1 signal in ERG+ cells (arrow) and the neighboring Nkx3.1+ normal luminal cells. Scale bars, 100 µm. Data represent mean ± s.d.; ns, not significant; unpaired two-tailed t-test.

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM12)

### [Extended Data Fig. 5 Origin, presence and characterization of intermediate state in prostate cancer.](https://www.nature.com/articles/s41588-025-02289-w/figures/13)

(**a**) IF image showing rare foci of K5 + IM cells from one mouse at the 9 months endpoint; inset shows high-power view. Scale bar, 100 µm. (**b**) IF showing Ar expression in invasive adenocarcinomas of indicated mice; inset shows high-power view. Scale bars, 50 µm (left), 20 µm (right). (**c**) immunohistochemistry IHC (left) and quantification (right) of K5 + IM cells in PCas (invasive adenocarcinomas) from indicated mice; insets show high-power view. Scale bars, 100 µm. (**d**) Flow cytometry quantification of DP/IM across of recombined cells (YFP+ for _YC/PYC_, ERG+ for _EPC_) from indicated mice (left to right: n = 3, 5, 11 mice). Whole prostate cells were harvested at 3-month age. (**e**) Flow cytometry quantifying p63 expression in indicated population (_WT_, n = 5 mice; _PYC_, n = 4 mice; _EPC_, n = 8 mice). Recombined cells labeled by YFP or ERG were analyzed for _YC/PYC_ and _EPC_ mice, respectively. (**f**) Prostate histology of 3-month _EPC_ mice highlighting a luminal morphology of both K5+ and K5- cells from invasive adenocarcinomas, in contrast to the K5+ basal cells that encapsulate the precursor PIN lesions (arrow); inset shows high-power view. PIN, prostatic intraepithelial neoplasia; inv, invasive adenocarcinoma; PCa, prostate cancer. Scale bars, 100 µm. (**g**) Quantification of K5 IHC signal in PIN and invasive adenocarcinomas of _EPC_ prostates at 2 ~ 3-month age. (**h**) Summary of cell morphology and protein marker expression across different cell types based on flow cytometry and immunostaining. Asterisk, basal cells are absent in PCa, which displays IM cells instead. (**i**) Representative flow plot showing lineage identities of recombined cells (YFP + ) from indicated mice. Whole prostates were harvested at indicated time point. (**j**) Flow cytometry quantifying indicated cell states = from indicated mice. Left to right: n = 4,3,5,3,2 mice (_K8-CreERT2;PY_); n = 3,3,6,2 mice (_K5-CreERT2;Y_); n = 4,4,3,3 mice (_K5-CreERT2;PY_) (**k**) Flow cytometry quantifying Ki67 status (n = 3 mice per group). (**l**) Prostate IF of indicated mice. Scale bars, 100 µm. Data represent mean ± s.d.; one-way ANOVA with Tukey posttest (**d**, **e**, **k right**); one-way ANOVA with Dunnett posttest (**j**); unpaired two-tailed t-test (**k**, left).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM13)

### [Extended Data Fig. 6 ERG+ basal and intermediate cells proliferate towards a luminal fate _in vitro_ and _in vivo._](https://www.nature.com/articles/s41588-025-02289-w/figures/14)

(**a**) _In vivo_ EdU pulse chase assay in ERG + _EPC_ cells (n = 5 mice for pulse, 4 mice for chase). (**b**) EdU quantification in SPlum and IM populations of ERG + _EPC_ cells in pulse samples from **A** (n = 5 mice). (**c**) Flow cytometry analysis on ERG+ cells from indicated mice. (**d**) Flow cytometry validating the ERG and Pten expression status in _EP_ organoids and the isogenic controls. (**e**) Flow cytometry quantifying luminal cells (SPlum) in organoids (left to right: n = 10, 12, 9 independent assays). (**f**) Flow cytometry quantification in _EP_ cells after a EdU pulse (n = 7 independent assays). (**g**) Junction PCR across the target loci and the engineered reporters in the bulk organoid population. (**h**) Live cell flow cytometry using the engineered reporter signals. (**i**) Sanger sequencing of the junction PCR clones from **G**. (**j**) Intracellular flow cytometry comparing the expression pattern between the endogenous K8 and the TagRFP reporter signal in targeted organoid population. (**k**) Functional validation of the _Krt5-mNeonGreen_ engineering by Krt5 depletion. (**l**) Intracellular flow cytometry quantifying ERG in _EP_ organoids with indicated CRISPR engineering. Cells were harvested 2 days after introducing the indicated CRISPR-RNP. Data represent mean ± s.d.; ns, not significant; unpaired two-tailed t-test (**a, b**); unpaired two-tailed t-test (**e**); multiple paired t-test (two-sided) with FDR correction by Benjamini, Krieger and Yekutieli (**f**).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM14)

### [Extended Data Fig. 7 Time point characterization and annotation of scRNA-seq clusters of genetically engineered mouse models (GEMMs).](https://www.nature.com/articles/s41588-025-02289-w/figures/15)

(**a**) H&E prostate histology. Dashed lines encircle invasive adenocarcinoma (n = 4 mice per group). Scale bars, 250 µm. (**b**) UMAP of all cells (N = 36,961 cells) from the prostates of 5 mice (2 _EPC_, 2 _PC_, 1 _WT_) annotated based on leiden clustering. (**c**) UMAP of all cells colored by individual replicates per genotype. (**d**) UMAP of all cells showing cells assigned to each genotype and colored by log10(UMI). **(e)** UMAP of all cells previously defined normal mouse prostate signatures. Epithelial cell clusters defined based on _Epcam_ expression are highlighted. Myeloid cells specific to the _EPC_ samples were defined as Myeloid-EPC. (**f**) Barplot of all cells colored by individual replicates (left) or each genotype (right). Numbers of cells in each cluster were included (n = ) and cluster number corresponding to **B** is shown. (**g**) UMAP (left) and barplot (right) of epithelial cells colored by cell types based on normal prostate signatures from Karthaus et al.[49](https://www.nature.com/articles/s41588-025-02289-w#ref-CR49 "Guo, W. et al. JAK/STAT signaling maintains an intermediate cell population during prostate basal cell fate determination. Nat. Genet. 56, 2776–2789 (2024)."). L2 cells (purple circles) and basal-like cells (red circles) specific to _PC_ and _EPC_ prostates are highlighted (termed Lum\_Mut and IM respectively). (**h**) UMAP showing epithelial clusters assigned to each genotype. The IDs of the two IM clusters are shown. (**i**) Barplot of epithelial clusters showing the cell type composition of each cluster. Numbers of cells in each cluster were included. (**j**) IHC of Trop2 (L2 marker) on indicated mice at 3 months age. Scale bars, 1 mm. (**k**) Violin plots comparing proliferative marker expression across all epithelial clusters. The two Tumor-Bas (IM) clusters are numbered according to the UMAP in **G**. (**l**) Gene set enrichment analysis showing EPC-specific pathways (ERG UP) within the Tumor-Bas/IM population.

### [Extended Data Fig. 8 Intermediate signatures are associated with a worse clinical outcome in prostate cancer.](https://www.nature.com/articles/s41588-025-02289-w/figures/16)

(**a**) IM marker gene expression in publicly available human normal and prostate cancer datasets (TCGA 2015; Bolis et al 2021). (Top) n = 173 (normal), 708 (primary PCa). (Bottom) E, ERG-fusion-positive (n = 89); P, PTEN deep deletion (n = 10); EP, ERG-fusion-positive and PTEN deep deletion (n = 40); DN, double negative (ERG fusion-negative, PTEN deep deletion-negative) (n = 151). The center line represents the median, the box limits represent the upper and lower quartiles and the minimum and maximum whiskers represent the 10th and 90th percentiles, respectively. (**b**) Expression of _Ar_ and Ar pathway genes across scRNA-seq clusters in mice. (**c**) Progression-free survival outcome using indicated signatures from two independent cohorts of patients with primary PCa, stratified based on ERG status (see also in Fig. [5c](https://www.nature.com/articles/s41588-025-02289-w#Fig5)). Data represent mean ± s.d.; ns, not significant; unpaired two-tailed t-test (**a**, top); one-way ANOVA with Tukey posttest (**a**, bottom); Log-rank (Mantel-Cox) test (**c**).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM15)

### [Extended Data Fig. 9 ERG drives a unique chromatin state in intermediate cells.](https://www.nature.com/articles/s41588-025-02289-w/figures/17)

**(a)** scATAC-seq UMAPS by indicated genotypes and time points. Cell density is overlaid and colored by increasing density, from dark to light. **(b)** Clustering generated by leiden algorithm is shown labeled and also colored on scATAC-seq UMAP. **(c)** Barplot showing number of peaks annotated as distal, exonic, intronic, or promoter by cluster. (**d**) Heatmap showing differential peaks between cell types. **(e)** Tornado plot showing significantly enriched peaks (FDR < = 0.05 and Log2FC >= 1) for each annotated cell type (Bas, PC-IM, EPC-IM, Lum\_Mut, Lum\_L2, Lum\_L1) vs all other cell types. Signal plotted is normalized scATAC-seq aggregated by cell type, centered at peak and extended by 1000 nt up and down. **(f)** Grid showing amount of RNA shared with scATAC-seq by cluster. Labels from scRNA were transferred to scATAC using Seurat’s CCA. **(g)** scATAC-seq reads piled up per cell type around the mm10 _KRT5_ locus, a classical basal cell marker (top panel). As expected, signal in Lum\_L1 is low compared to Basal. Second panel from top shows tick marks to indicate detected peaks. Third panel from top show inferred Co-Accessibility using Cicero, an algorithm used to predict DNA interactions from scATAC-seq data. Fourth panel from top shows _Krt5_ gene locus and neighboring _Krt6a_ locus in mouse. **(h)** Barplot percentage of cell type per genotype is shown. (**i**) UMAP of cells colored by pseudotime indicating later trajectory branches map to Lum\_L1, Lum\_L2, and EPC-IM. **(j)** scATAC-seq heatmap of most enriched transcription factor candidates when comparing one cell type to all others. Cells are grouped by cell type as columns and rows indicate motifs from database. The heatmap elements are colored by chromvar inferred z-score. Motif significance was calculated (see [Methods](https://www.nature.com/articles/s41588-025-02289-w#Sec11)) and those having an p-value < 1e-20 after BH-false discovery rate correction are shown. **(k)** Representative UMAPs colored by motif deviation z-score (left) and inferred gene expression (gene score, right) of EPC-IM enriched TFs (NFKB2, NFATC1, STAT3).

### [Extended Data Fig. 10 Characterization of Kmt2a, Dot1l, Men1 perturbation in pre- and post-grafts.](https://www.nature.com/articles/s41588-025-02289-w/figures/18)

**(a)** Western blot confirming the knockout status of Kmt2a/Mll1 in _EPC;sgKmt2a_ pre-graft organoids (n = 2 independent cohorts). (**b**) IF showing that the residual cells from _EPC;sgKmt2a_ orthografts largely maintained Mll1 expression, suggestive of knockout escapers (n = 6 grafts per group). High-power view from insets highlight the rare existence of Mll1-null cells. Scale bars, 50 µm. (**c**) ERG and H3K79me2 IHC (n = 6 grafts per group). Tumor areas were quantified based on ERG+H3K79me2– signals as highlighted in green dashed lines. Arrows in the sgDot1l groups highlight ERG+ cells that retain H3K79me2 signals. Inset shows high-power view. Scale bars, 250 µm. (**d**) Western blot confirming the knockout status of Menin in _EPC;sgMen1_ pre-graft organoids (n = 2 independent sgRNAs).

[Source data](https://www.nature.com/articles/s41588-025-02289-w#MOESM16)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM1_ESM.pdf)

Supplementary Notes 1–8, Fig. 1 and References.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM3_ESM.pdf)

### [Supplementary Table 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM4_ESM.xlsx)

Supplementary Table 1. ERG DEG in basal and luminal cells sorted from EP organoids. Supplementary Table 2. Differentially expressed genes of each epithelial cluster (versus the others) in GEMMs. Supplementary Table 3. Gene set enrichment analysis for single-cell RNA clusters. Supplementary Table 4. Antibodies for tissue IHC and IF. Supplementary Table 5. sgRNA target and oligo sequences. Supplementary Table 6. Antibodies for flow cytometry.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM5_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM7_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM8_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM9_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM12_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM13_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM14_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 8](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM15_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 10](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02289-w/MediaObjects/41588_2025_2289_MOESM16_ESM.pdf)

Unprocessed western blots.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=ERG-driven%20prostate%20cancer%20initiation%20is%20cell-context%20dependent%20and%20requires%20KMT2A%20and%20DOT1L&author=Weiran%20Feng%20et%20al&contentID=10.1038%2Fs41588-025-02289-w&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-26&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02289-w)

### Cite this article

Feng, W., Ladewig, E., Lange, M. _et al._ ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02289-w

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02289-w?format=refman&flavour=citation)

-   Received: 14 June 2024
    
-   Accepted: 08 July 2025
    
-   Published: 26 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02289-w
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02289-w.pdf)
