> id: https://www.nature.com/articles/s41588-025-02322-y

> link: https://www.nature.com/articles/s41588-025-02322-y

> title: Mutational landscape of triple-negative breast cancer in African American women

# Mutational landscape of triple-negative breast cancer in African American women
_Published on Tue, 26 Aug 2025 00:00:00 GMT_

Abstract
--------

African American (AA) women have the highest incidence of triple-negative breast cancer (TNBC) among all racial groups, but are underrepresented in cancer genomic studies. In 462 AA women with TNBC, we characterized the tumor mutational landscape by whole-exome sequencing and RNA sequencing. We unveiled a high-resolution mutational portrait of TNBC in AA women reminiscent of that in Asian and non-Hispanic white women, with no evidence of associations of mutational features with African ancestry. We also made some distinctive discoveries, including an almost complete dominance of _TP53_ mutations, low frequency of _PIK3CA_ mutations and mutational signature-based subtypes with etiologic and prognostic significance. These findings do not support major racial differences in TNBC biology at the level of somatic mutations. Our study contributes considerably to diversifying the knowledge base of breast cancer genomics and provides insights into the disease etiology, disparities and therapeutic vulnerability of TNBC in AA women.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41523-025-00809-9/MediaObjects/41523_2025_809_Fig1_HTML.png)

### [Genomic landscapes of breast cancer in African populations: a systematic review](https://www.nature.com/articles/s41523-025-00809-9?fromPaywallRec=false)

Article Open access 14 August 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-021-88613-w/MediaObjects/41598_2021_88613_Fig1_HTML.png)

### [Investigation of triple-negative breast cancer risk alleles in an International African-enriched cohort](https://www.nature.com/articles/s41598-021-88613-w?fromPaywallRec=false)

Article Open access 29 April 2021

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41523-021-00215-x/MediaObjects/41523_2021_215_Fig1_HTML.png)

### [Ancestry-associated transcriptomic profiles of breast cancer in patients of African, Arab, and European ancestry](https://www.nature.com/articles/s41523-021-00215-x?fromPaywallRec=false)

Article Open access 08 February 2021

Main
----

Triple-negative breast cancer (TNBC) remains an enigma in oncology and epidemiology. This subtype is defined by the lack of expression of estrogen receptor (ER), progesterone receptor (PR) and ErbB2 on immunostaining and represents 10–15% of all breast cancer diagnoses[1](https://www.nature.com/articles/s41588-025-02322-y#ref-CR1 "Dietze, E. C., Sistrunk, C., Miranda-Carboni, G., O’Regan, R. & Seewaldt, V. L. Triple-negative breast cancer in African-American women: disparities versus biology. Nat. Rev. Cancer 15, 248–254 (2015)."). Although new treatments such as PARP inhibitors and immune checkpoint inhibitors improve outcomes for some patients[2](https://www.nature.com/articles/s41588-025-02322-y#ref-CR2 "Bianchini, G., De Angelis, C., Licata, L. & Gianni, L. Treatment landscape of triple-negative breast cancer—expanded options, evolving needs. Nat. Rev. Clin. Oncol. 19, 91–113 (2022)."), TNBC is still the most deadly subtype of breast cancer. A prominent epidemiological feature of TNBC in the United States is the disproportionately high incidence in African American (AA) women, estimated at 25.2 per 100,000 in comparison to 12.9 per 100,000 in non-Hispanic white (NHW) women[3](https://www.nature.com/articles/s41588-025-02322-y#ref-CR3 "Sung, H., Wiese, D., Jatoi, I. & Jemal, A. State variation in racial and ethnic disparities in incidence of triple-negative breast cancer among US women. JAMA Oncol. 9, 700–704 (2023).").

One important question regarding the distinctive TNBC epidemiology in AA women is whether it is a result of differences in cancer biology or socio-environmental exposures[1](https://www.nature.com/articles/s41588-025-02322-y#ref-CR1 "Dietze, E. C., Sistrunk, C., Miranda-Carboni, G., O’Regan, R. & Seewaldt, V. L. Triple-negative breast cancer in African-American women: disparities versus biology. Nat. Rev. Cancer 15, 248–254 (2015)."). In recognition of TNBC as a group of heterogeneous diseases, multi-omic breast tumor characterization is warranted. However, despite a high disease burden, AA women have long been underrepresented in cancer genomic studies[4](https://www.nature.com/articles/s41588-025-02322-y#ref-CR4 "Spratt, D. E. et al. Racial/ethnic disparities in genomic sequencing. JAMA Oncol. 2, 1070–1074 (2016)."). Of the few breast tumor sequencing studies in AA women[5](https://www.nature.com/articles/s41588-025-02322-y#ref-CR5 "Ansari-Pour, N. et al. Whole-genome analysis of Nigerian patients with breast cancer reveals ethnic-driven somatic evolution and distinct genomic subtypes. Nat. Commun. 12, 6946 (2021)."),[6](https://www.nature.com/articles/s41588-025-02322-y#ref-CR6 "Pitt, J. J. et al. Characterization of Nigerian breast cancer reveals prevalent homologous recombination deficiency and aggressive molecular features. Nat. Commun. 9, 4181 (2018)."),[7](https://www.nature.com/articles/s41588-025-02322-y#ref-CR7 "Keenan, T. et al. Comparison of the genomic landscape between primary breast cancer in African American versus white women and the association of racial differences with tumor recurrence. J. Clin. Oncol. 33, 3621–3627 (2015)."),[8](https://www.nature.com/articles/s41588-025-02322-y#ref-CR8 "Huo, D. et al. Comparison of breast cancer molecular features and survival by African and European ancestry in The Cancer Genome Atlas. JAMA Oncol. 3, 1654–1662 (2017)."), none focused on TNBC, except for one with 51 patients[9](https://www.nature.com/articles/s41588-025-02322-y#ref-CR9 "Chang, C. S., Kitamura, E., Johnson, J., Bollag, R. & Hawthorn, L. Genomic analysis of racial differences in triple negative breast cancer. Genomics 111, 1529–1542 (2019)."), in contrast to much larger such studies in Asian and NHW women[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."),[11](https://www.nature.com/articles/s41588-025-02322-y#ref-CR11 "Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. Nat. Med. 25, 1526–1533 (2019)."),[12](https://www.nature.com/articles/s41588-025-02322-y#ref-CR12 "Shah, S. P. et al. The clonal and mutational evolution spectrum of primary triple-negative breast cancers. Nature 486, 395–399 (2012).").

Herein, we assembled matched tumor and normal samples from self-identified AA women with TNBC in five population-based studies and carried out whole-exome sequencing (WES) and RNA sequencing analysis, with goals to chart the mutational landscape of TNBC in AA women and to investigate the implications for cancer etiology and therapy.

Results
-------

### Patient population

Paired tumor and normal samples from 513 self-identified AA women with TNBC were interrogated by WES. After data processing and quality control steps, 462 (90%) cases were included in the final analysis. Patient descriptive characteristics are shown in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02322-y#MOESM4). The average (±s.d.) age at diagnosis was 53 (±11) years, with 38% before the age of 50 years.

### Mutational landscape of TNBC in AA women

From the 462 tumors, we identified 39,103 mutations in the coding regions, including 36,059 (92%) single-nucleotide variants (SNVs) and 2,690 (7%) insertion/deletions (indels; Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)). The median mutation burden was 1.29 (range = 0.07–22.2) SNVs per Mb, with five tumors (1%) considered hypermutated (>10 SNVs per Mb)[13](https://www.nature.com/articles/s41588-025-02322-y#ref-CR13 "Campbell, B. B. et al. Comprehensive analysis of hypermutation in human cancer. Cell 171, 1042–1056 (2017).") (Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)), three of which carrying a mutation in mismatch repair genes (_MLH1_, _MSH3_ and _LIG1_). At the gene level, we identified nonsilent mutations in 11,273 genes (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02322-y#MOESM4)), with a median of 47 (range = 1–664) mutated genes per tumor.

Figure [1a](https://www.nature.com/articles/s41588-025-02322-y#Fig1) illustrates the compendium of somatic mutations in TNBC from AA women. The mutational landscape is predominated by alterations in _TP53_, with a total of 463 mutations found in 437 (95%) tumors, including 18 with two or more mutations. A majority (59%) of the mutations were recurrent and all but six were nonsilent. We classified 294 (66%) of the _TP53_ coding mutations as loss of function, 113 (26%) as gain of function, 2 (0.5%) as benign, and 33 (7%) as function unknown (most in-frame indels). Most tumors (_n_ = 431 or 93%) had at least one nonsilent mutation. One tumor harbored E224E, a known cancer-driving synonymous mutation[14](https://www.nature.com/articles/s41588-025-02322-y#ref-CR14 "Supek, F., Minana, B., Valcarcel, J., Gabaldon, T. & Lehner, B. Synonymous mutations frequently act as driver mutations in human cancers. Cell 156, 1324–1335 (2014)."). In addition, five tumors harbored intronic mutations only. Using transcriptomic data available from four of these tumors, we found evidence of aberrant RNA splicing in three (Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02322-y#Fig7)).

**Fig. 1: Mutational landscape of TNBC from AA women.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02322-y/figures/1)

**a**, CoMut plot of somatic and germline mutations in TNBC from AA women. Mutation rate is presented as the number of SNVs per Mb. The proportion of African ancestry was estimated based on germline variant data from matched normal DNA samples and presented as a numeric value between 0 and 1. TNBC subtype was classified based on tumor transcriptomic data available from 260 cases using the method discussed in ref. [20](https://www.nature.com/articles/s41588-025-02322-y#ref-CR20 "Lehmann, B. D. et al. Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies. J. Clin. Invest. 121, 2750–2767 (2011)."). HRD was estimated based on WES data using scarHRD R package[57](https://www.nature.com/articles/s41588-025-02322-y#ref-CR57 "Sztupinszki, Z. et al. Migrating the SNP array-based homologous recombination deficiency measures to next generation sequencing data of breast cancer. npj Breast Cancer 4, 16 (2018)."). Somatic mutations are sorted by mutation frequency and shown in the upper section of the CoMut plot. Germline variants are shown in the lower section of the plot. Gene symbols are labeled in colors to indicate known TNBC genes (red), breast cancer genes (yellow) and pan-cancer genes (green). NA, not available. **b**, Frequency of CNAs (_y_ axis) across chromosomes (_x_ axis), with red color for copy number gain, blue color for copy number loss and dark red and dark blue for regions tested substantially in GISTIC2 analysis. CNV, copy number variation. **c**, Heatmap of known cancer genes in substantial peaks identified by GISTIC2. BL1, basal-like 1; BL2, basal-like 2; LAR, luminal androgen receptor; M, mesenchymal; UNS, unassigned.

### Confirmation of _TP53_ mutations

In validation analysis of _TP53_ mutations using transcriptomic data available from 260 patients, 215 (83%) of the 259 mutations identified by WES were detected at the RNA level (Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02322-y#Fig8)). Given the uneven coverage of RNA sequencing, we restricted the analysis to tumors with ≥10× and were able to confirm 183 of 187 (98%) mutations in these tumors. The concordance reached 100% (109/109 mutations) in tumors with ≥30× coverage.

Furthermore, we resequenced the _TP53_ region in 338 tumors with DNA available (317 with and 21 without mutations) using targeted amplicon sequencing (TAS). These 317 tumors harbored 326 mutations identified by WES. We confirmed 324 (99%) mutations, with only two going undetected by TAS (Extended Data Fig. [2](https://www.nature.com/articles/s41588-025-02322-y#Fig8)). Of the 21 tumors that had no _TP53_ mutations in WES data, TAS analysis identified two new mutations at low variant allele frequency (<5%).

### Known cancer genes and significantly mutated genes

Of the 11,273 genes harboring nonsilent mutations, 218 had a frequency ≥2% (≥10 tumors; Supplementary Table [2](https://www.nature.com/articles/s41588-025-02322-y#MOESM4)). Aside from _TP53_, all other genes were mutated at a much lower frequency (Fig. [1a](https://www.nature.com/articles/s41588-025-02322-y#Fig1)). These included 16 known TNBC genes, namely, _NOTCH1_ (7%), _RB1_ (7%), _KMT2D_ (6%), _PIK3CA_ (5%), _PTEN_ (5%), _KMT2C_ (5%), _BRCA1_ (5%), _NF1_ (5%), _SPEN_ (4%), _FAT3_ (4%), _CREBBP_ (4%), _PIK3R1_ (3%), _NOTCH2_ (3%), _BRCA2_ (2%), _ERBB2_ (2%) and _KDM6A_ (2%), and 7 known breast cancer genes, albeit not specific to TNBC, _ARID1A_ (4%), _CIC_ (4%), _GNAS_ (4%), _AXIN1_ (4%), _RYR2_ (3%), _USH2A_ (3%) and _GATA3_ (2%). Additionally, 21 driver genes identified in previous pan-cancer analysis[15](https://www.nature.com/articles/s41588-025-02322-y#ref-CR15 "Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. Cell 173, 371–385 (2018)."),[16](https://www.nature.com/articles/s41588-025-02322-y#ref-CR16 "ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium Pan-cancer analysis of whole genomes. Nature 578, 82–93 (2020).") mutated in ≥2% in our cohort. These included several well-known cancer genes or their family members with previously less recognized role in breast cancer—_FGFR3_, _FGFR4_, _NOTCH3_, _KMT2B_, _EP300_, _FLT4_ and _FAT1_.

Significantly mutated genes (SMG) analysis identified 13 genes with _q_ ≤ 0.20 by two or more programs used (MutSigCV[17](https://www.nature.com/articles/s41588-025-02322-y#ref-CR17 "Lawrence, M. S. et al. Mutational heterogeneity in cancer and the search for new cancer-associated genes. Nature 499, 214–218 (2013)."), MutSig2CV[18](https://www.nature.com/articles/s41588-025-02322-y#ref-CR18 "Lawrence, M. S. et al. Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 505, 495–501 (2014).") and MuSiC[19](https://www.nature.com/articles/s41588-025-02322-y#ref-CR19 "Dees, N. D. et al. MuSiC: identifying mutational significance in cancer genomes. Genome Res. 22, 1589–1598 (2012).")), which were all known cancer genes in TNBC (Supplementary Tables [3](https://www.nature.com/articles/s41588-025-02322-y#MOESM4)–[6](https://www.nature.com/articles/s41588-025-02322-y#MOESM4)).

In comparisons across TNBC transcriptional subtypes[20](https://www.nature.com/articles/s41588-025-02322-y#ref-CR20 "Lehmann, B. D. et al. Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies. J. Clin. Invest. 121, 2750–2767 (2011)."),[21](https://www.nature.com/articles/s41588-025-02322-y#ref-CR21 "Lehmann, B. D. et al. Multi-omics analysis identifies therapeutic vulnerabilities in triple-negative breast cancer subtypes. Nat. Commun. 12, 6276 (2021)."), the luminal androgen receptor subtype had enrichment of somatic mutations in _PTEN_ (_P_ = 0.003) and _PIK3R1_ (_P_ = 0.04) and slight depletion of _TP53_ (_P_ = 0.009) mutation, consistent with previous studies in Asian and NHW patients (Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02322-y#MOESM1))[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."),[22](https://www.nature.com/articles/s41588-025-02322-y#ref-CR22 "Li, R. Q. et al. Genomic characterization reveals distinct mutational landscapes and therapeutic implications between different molecular subtypes of triple-negative breast cancer. Sci. Rep. 14, 12386 (2024)."),[23](https://www.nature.com/articles/s41588-025-02322-y#ref-CR23 "Lehmann, B. D. et al. PIK3CA mutations in androgen receptor-positive triple negative breast cancer confer sensitivity to the combination of PI3K and androgen receptor inhibitors. Breast Cancer Res. 16, 406 (2014).").

### Pathogenic germline mutations

Using sequencing data from matched normal samples, we identified 124 germline mutations in nine known TNBC predisposition genes from 241 patients (Fig. [1a](https://www.nature.com/articles/s41588-025-02322-y#Fig1)). Of these variants, 115 were found in gnomAD and other reference datasets[24](https://www.nature.com/articles/s41588-025-02322-y#ref-CR24 "All of Us Research Program Genomics Investigators. Genomic data in the All of Us Research Program. Nature 627, 340–346 (2024)."),[25](https://www.nature.com/articles/s41588-025-02322-y#ref-CR25 "Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. Nature 625, 92–100 (2024)."),[26](https://www.nature.com/articles/s41588-025-02322-y#ref-CR26 "Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Nature 590, 290–299 (2021)."),[27](https://www.nature.com/articles/s41588-025-02322-y#ref-CR27 "Sun, K. Y. et al. A deep catalogue of protein-coding variation in 983,578 individuals. Nature 631, 583–592 (2024)."), 22 being exclusive to populations of African ancestry (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02322-y#MOESM4)). When minor allele frequency was compared with reference populations, 60 variants had a higher frequency in TNBC patients (_P_ < 0.05; Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02322-y#Fig9)). These results confirmed benignity for 28 of 30 variants classified as ‘benign/likely benign’ and pathogenicity for 23 of 25 variants classified as ‘pathogenic/likely pathogenic’ by ClinVar[28](https://www.nature.com/articles/s41588-025-02322-y#ref-CR28 "Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 42, D980–D985 (2014)."), while yielding new evidence of pathogenicity for 14 of 35 variants annotated as ‘conflicting classification of pathogenicity’ and 15 of 18 variants annotated as ‘uncertain significance’. Moreover, we identified six variants, including two in _PTEN_ and four in _BRCA1_, with no pathogenicity annotation in ClinVar[28](https://www.nature.com/articles/s41588-025-02322-y#ref-CR28 "Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 42, D980–D985 (2014)."), all of which had a higher frequency in TNBC cases than in the reference datasets, including R119C mutation in _PTEN_ (_P_ = 9 × 10−7). Lastly, we discovered nine new germline mutations not previously reported in any reference databases, including three in _BRCA1_, three in _BRCA2_, two in _PALB2_ and one in _NF1_. Two of the _BRCA1_ variants were deemed damaging in saturation genome editing[29](https://www.nature.com/articles/s41588-025-02322-y#ref-CR29 "Findlay, G. M. et al. Accurate classification of BRCA1 variants with saturation genome editing. Nature 562, 217–222 (2018).").

### Copy number aberrations

Figure [1b](https://www.nature.com/articles/s41588-025-02322-y#Fig1) illustrates the copy number abberation (CNA) landscape based on WES data captured with additional baits representing human array comparative genomic hybridization (aCGH) probes. We identified multiple substantial copy number gains or losses, including 18 regions containing known cancer genes (Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)). As expected, high-level _MYC_ amplification was one of the most common copy number changes found in 36% of the tumors (Fig. [1c](https://www.nature.com/articles/s41588-025-02322-y#Fig1)). Other known cancer genes residing within substantial aberrant regions defined by Genomic Identification of Significant Targets in Cancer (GISTIC2) included high-level amplifications of _MCL1_ (44%), _AKT3_ (21%), _GATA3_ (19%), _E2F3_ (18%), _NFIB_ (13%), _CCNE1_ (12%), _IRS2_ (12%), _PIK3CA_ (12%), _MYB_ (8%), _NOTCH2_ (8%), _EGFR_ (5%), _FGFR2_ (5%) and _TERT_ (4%) and homozygous deletion of _RB1_ (8%), _PTEN_ (6%), _CDKN2A_/_CDKN2B_ (5%) and _ESR1_ (3%).

### Commonly altered signaling pathways

In gene set enrichment analysis (GSEA) based on 218 cancer driver genes harboring recurrent (≥1%) somatic mutations in the cohort, we identified four commonly altered signaling pathways (_q_ < 0.05). The p53 signaling pathway was altered in most (95%), if not all, TNBC in our cohort (Fig. [2a](https://www.nature.com/articles/s41588-025-02322-y#Fig2)). Mutations in other genes in the core p53 signaling pathway were much rarer (1% in _ATM_ and 5% in _CHEK2_) and all co-occurred with _TP53_ mutations. When CNAs were also considered, the p53 signaling pathway was implicated in 97% (_n_ = 450) of the tumors.

**Fig. 2: Common signaling pathways altered in TNBC from AA women.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02322-y/figures/2)

Commonly altered pathways in TNBC from AA women based on GSEA results using point mutations and indels. The numbers in the plots indicate the percentages of tumors harboring the alterations that are color coded. **a**, p53 signaling pathway. **b**, Notch signaling pathway. **c**, Cell cycle checkpoints. **d**, PI3K–Akt signaling pathway.

_NOTCH1_ was the second commonly mutated gene (7%) in our cohort. Nonsynonymous mutations were found at a lower frequency in three other _NOTCH_ family genes, with an aggregated mutation frequency of 14%, which was further increased to 22% when other genes in the core Notch signaling pathway were considered (Fig. [2b](https://www.nature.com/articles/s41588-025-02322-y#Fig2)). Moreover, high-level amplifications were also identified in _NOTCH_ family genes and two co-activators _EP300_ and _CREBBP_.

_RB1_ is the third most commonly mutated gene (7%) in our cohort, and homozygous deletion was observed in 8% of the tumors (Fig. [2c](https://www.nature.com/articles/s41588-025-02322-y#Fig2)). In addition to germline mutations, _BRCA1_ and _BRCA2_ somatic mutations were observed in another 5% and 2% of tumors, respectively. Further, 5% of the tumors demonstrated homozygous deletion of _CDKN2A_ and _CDKN2B_. On the contrary, high-level amplification occurred at a high frequency in several key genes driving cell cycle progression, including _CCND1_ (8%), _CCNE1_ (12%), _CDK4_/_CDK6_ (10%), _E2F3_ (18%) and _MYC_ (36%).

Several genes in the PI3K–Akt signaling pathway that encode growth factor receptors demonstrated mutation and/or high-level amplification, including the four _FGFR_ family members, _ERBB2_, _IGFR1_ and _EGFR_ (Fig. [2d](https://www.nature.com/articles/s41588-025-02322-y#Fig2)). In addition, several core PI3K pathway members were among the top mutated genes, including _PIK3CA_, _PTEN_ and _TSC2_ each at 5%, plus _PIK3R1_ at 3% and _MTOR_ at 2%.

### Mutational signatures

Three de novo single base substitution (SBS) mutational signatures were extracted from 457 tumors after excluding five hypermutated samples, which were then decomposed to five of the COSMIC SBS96 signatures (Fig. [3a](https://www.nature.com/articles/s41588-025-02322-y#Fig3) and Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02322-y#MOESM1))[30](https://www.nature.com/articles/s41588-025-02322-y#ref-CR30 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)."). These include two clock-like signatures, SBS1 and SBS5, homologous recombination deficiency (HRD)-related signature, SBS3, and two APOBEC-related signatures, SBS2 and SBS13, all of which have previously been found in TNBC in Asian and NHW women[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."),[11](https://www.nature.com/articles/s41588-025-02322-y#ref-CR11 "Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. Nat. Med. 25, 1526–1533 (2019)."),[31](https://www.nature.com/articles/s41588-025-02322-y#ref-CR31 "Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. Nature 534, 47–54 (2016)."),[32](https://www.nature.com/articles/s41588-025-02322-y#ref-CR32 "Nik-Zainal, S. & Morganella, S. Mutational signatures in breast cancer: the problem at the DNA level. Clin. Cancer Res. 23, 2617–2629 (2017)."). The clock-like SBS1 and SBS5 were moderately correlated with each other (_r_ = 0.45, _P_ < 0.001) and together showed a moderate correlation with patient age (_r_ = 0.21, _P_ < 0.001; Supplementary Fig. [6a](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)).

**Fig. 3: Mutational signatures in TNBC from AA women.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02322-y/figures/3)

**a**, From top to bottom: the first row shows HRD estimated based on WES data using scarHRD R package[57](https://www.nature.com/articles/s41588-025-02322-y#ref-CR57 "Sztupinszki, Z. et al. Migrating the SNP array-based homologous recombination deficiency measures to next generation sequencing data of breast cancer. npj Breast Cancer 4, 16 (2018)."). The second row shows the proportion of African ancestry was estimated based on germline variant data from matched normal DNA samples and presented as a numeric value between 0 and 1. The third row shows TNBC subtype classified based on tumor transcriptomic data available from 260 cases using the method discussed in ref. [20](https://www.nature.com/articles/s41588-025-02322-y#ref-CR20 "Lehmann, B. D. et al. Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies. J. Clin. Invest. 121, 2750–2767 (2011)."). The fourth and fifth rows show SBS and ID (indel) mutational signatures, respectively. The sixth and seventh rows show genes with differences in frequency of somatic mutations and CNAs between SBS subtype 1 (low aging and high HRD) and subtype 3 (high aging and low HRD), respectively. **b**, Demographic and mutational features that show substantial differences between SBS subtype 1 (_n_ = 103) and subtype 3 (_n_ = 131) by the Wilcoxon test and the _P_ values were two-sided without adjustment for multiple comparisons. Subtypes 2 and 4 have characteristics that fall somewhere between subtypes 1 and 3 and are not shown. The bar in the middle of a box indicates the subgroup median, and the lower and upper edges indicate the first and third quartiles, respectively. The whiskers indicate the range in each subgroup. _P_ values were derived from two-sided Wilcoxon test between Black and white patients. TILs, tumour-infiltrating lymphocytes. **c**, Tumor microenvironment immune signatures that show substantial differences between SBS subtype 1 and subtype 3 by the Wilcoxon test and the _P_ values were two-sided without adjustment for multiple comparisons. **d**, Kaplan–Meier curves of all-cause mortality (death due to any cause) by SBS signatures, with _P_ values derived from the log-rank test.

While SBS1 and SBS5 were found in virtually all tumors, representing the dominant mutagenic processes in almost half of the tumors, the HRD-related SBS3 dominated the other half (53%; Fig. [3a](https://www.nature.com/articles/s41588-025-02322-y#Fig3)). As expected, SBS3 was correlated with HRD score (_r_ = 0.62, _P_ < 0.001; Supplementary Fig. [6b](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)), and was more active among patients carrying _BRCA1_ and _BRCA2_ germline variants (_P_ = 0.005; Supplementary Fig. [6c](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)). Notably, there was also a moderate negative correlation between SBS3 and SBS5 (_r_ = −0.29, _P_ < 0.001). The two APOBEC-related signatures, SBS2 and SBS13, were strongly correlated with each other (_r_ = 0.85, _P_ < 0.001) and manifested in 35% TNBC tumors at lower activity relative to the other three signatures.

For indels, three de novo signatures were extracted from 439 tumors. Decomposition analyses yielded six indel signatures in reference to COSMIC ID83 (ref. [30](https://www.nature.com/articles/s41588-025-02322-y#ref-CR30 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020).")), including the following five known ones: ID2 related to slippage during DNA replication, ID4 with no known etiology, ID6 related to HRD, ID7 related to defective DNA mismatch damage repair and ID8 related to double strand break repair (Fig. [3a](https://www.nature.com/articles/s41588-025-02322-y#Fig3) and Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02322-y#MOESM1))[30](https://www.nature.com/articles/s41588-025-02322-y#ref-CR30 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)."). ID6 and ID8 were the most active mutagenic processes found in 38% and 42%, respectively, of the tumors and both showed a moderate correlation with estimated HRD score (_r_ = 0.48 and _r_ = 0.42, respectively, _P_ < 0.001; Supplementary Fig. [6d,e](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)). The sixth indel signature, characterized by longer indels ≥5 bp was new and presented in one-third of the TNBC tumors, which displayed a weak negative correlation with HRD (_r_ = −0.21, _P_ < 0.001; Supplementary Fig. [6f](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)).

When examined across TNBC transcriptional subtypes[20](https://www.nature.com/articles/s41588-025-02322-y#ref-CR20 "Lehmann, B. D. et al. Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies. J. Clin. Invest. 121, 2750–2767 (2011)."),[21](https://www.nature.com/articles/s41588-025-02322-y#ref-CR21 "Lehmann, B. D. et al. Multi-omics analysis identifies therapeutic vulnerabilities in triple-negative breast cancer subtypes. Nat. Commun. 12, 6276 (2021)."), the M subtype had relatively lower APOBEC-related SBS2 (_P_ = 0.005) and SBS13 (_P_ = 0.09) signatures, and the luminal androgen receptor subtype had lower HRD-related SBS3 (_P_ = 0.006) but higher ID4 signature (_P_ = 0.006; Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)).

### Genomic and immune differences by SBS subtype

We defined SBS signature-based TNBC subtypes by combining SBS1 + SBS5 (aging) and SBS3 (HRD). The differences between subtype 1 (low aging and high HRD) and subtype 3 (high aging and low HRD) were the most apparent, whereas subtype 2 (low aging and low HRD) and subtype 4 (high aging and high HRD) were somewhere in between. Tumor classified as subtype 1 had higher mutation rate, HRD score and pathological tumor infiltrating lymphocyte score, lower _BRCA1_ expression and were less likely from older patients or those with higher body mass index (BMI; _P_ ≤ 0.05; Fig. [3b](https://www.nature.com/articles/s41588-025-02322-y#Fig3)). For every 10-year increment of age and every 5 kg m−2 increment of BMI, the odds of having subtype 1 versus subtype 3 TNBC decreased by 68% (_P_ = 0.0001) and 31% (_P_ = 0.008), respectively.

Moreover, these two SBS subtypes differed in somatic mutations in several cancer driver genes, including higher mutation frequency of _ERBB2_, _GATA3_ and _FGFR4_, and lower frequency of _DMD_, _INHBA_, _OGDHL_, _PLEKHG5_ and _RYR2_ in subtype 3 than in subtype 1 (_P_ < 0.05; Fig. [3a](https://www.nature.com/articles/s41588-025-02322-y#Fig3)). In addition, subtype 1 tumors were also more likely to have high-level amplification of _MYC_, _MCL1_, _AKT3_, _E2F3_ and _GATA3_ (_P_ < 0.05). In analysis of gene expression-based immune signatures, subtype 1 manifested with stronger immune cytolytic activity signature[33](https://www.nature.com/articles/s41588-025-02322-y#ref-CR33 "Rooney, M. S., Shukla, S. A., Wu, C. J., Getz, G. & Hacohen, N. Molecular and genetic properties of tumors associated with local immune cytolytic activity. Cell 160, 48–61 (2015).") (_P_ = 0.002) and two B cell signatures[34](https://www.nature.com/articles/s41588-025-02322-y#ref-CR34 "Conte, B. et al. A 14-gene B-cell immune signature in early-stage triple-negative breast cancer (TNBC): a pooled analysis of seven studies. EBioMedicine 102, 105043 (2024)."),[35](https://www.nature.com/articles/s41588-025-02322-y#ref-CR35 "Petitprez, F. et al. B cells are associated with survival and immunotherapy response in sarcoma. Nature 577, 556–560 (2020).") (_P_ = 0.005 and _P_ = 0.04) than subtype 3 (Fig. [3c](https://www.nature.com/articles/s41588-025-02322-y#Fig3)). Consistent with this, GSEA showed substantial enrichment of many immune response gene sets in subtype 1 relative to subtype 3 tumors (Supplementary Fig. [7](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)).

### Mutational signatures and patient survival

As shown in Fig. [3d](https://www.nature.com/articles/s41588-025-02322-y#Fig3), higher SBS1 + SBS5 (aging) was associated with higher all-cause mortality (high versus low—hazard ratio (HR) = 1.97, 95% confidence interval (CI) = 1.24–3.13, _P_ = 0.004), whereas higher SBS3 (HRD) was associated with lower mortality (HR = 0.55, 95% CI = 0.33–0.92, _P_ = 0.02). The associations became only borderline substantial after adjusting for age, study and stage (Supplementary Table [8](https://www.nature.com/articles/s41588-025-02322-y#MOESM4)). No substantial association of patient survival was found with APOPEC signatures SBS2 or SBS13. In analyses of SBS signature-based TNBC subtype, patients with subtype 3 had the higher all-cause mortality, in comparison to those with SBS subtype 1 (HR = 2.63, 95% CI = 1.47–4.69, _P_ = 0.001; Fig. [3d](https://www.nature.com/articles/s41588-025-02322-y#Fig3)), which remained substantial after controlling for age and cancer stage (HR = 1.96, 95% CI = 1.05–3.64, _P_ = 0.03). Meta-analyses across the three studies show similar results (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02322-y#Fig10)). No substantial association was observed with subtype 2 or subtype 4.

### Comparisons of somatic mutations across racial groups

Figure [4](https://www.nature.com/articles/s41588-025-02322-y#Fig4) shows the three-way comparisons of mutation frequency of known breast cancer genes across AA patients from ‘Breast Cancer in African Americans: Understanding Somatic Mutations and Etiology’ (B-CAUSE) study (_n_ = 462), Asian patients from Fudan University Shanghai Cancer Center (FUSCC; _n_ = 279)[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019).") and NHW patients (_n_ = 626) pooled from The Cancer Genome Atlas (TCGA)[36](https://www.nature.com/articles/s41588-025-02322-y#ref-CR36 "Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. Nature 490, 61–70 (2012)."), Sweden Cancerome Analysis Network—Breast (SCAN-B)[11](https://www.nature.com/articles/s41588-025-02322-y#ref-CR11 "Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. Nat. Med. 25, 1526–1533 (2019).") and Molecular Taxonomy of Breast Cancer International Consortium (METABRIC)[37](https://www.nature.com/articles/s41588-025-02322-y#ref-CR37 "Pereira, B. et al. The somatic mutation profiles of 2,433 breast cancers refines their genomic and transcriptomic landscapes. Nat. Commun. 7, 11479 (2016)."), with the full results provided in Supplementary Table [9](https://www.nature.com/articles/s41588-025-02322-y#MOESM4). The mutation frequencies were largely similar between Asian and NHW patients, yet several genes had notably different mutation frequency from these in AA patients, including higher frequencies of _TP53_ (95%, 78% and 75% in AA, Asian and NHW patients, respectively; _P_ < 1 × 10−9) and _NOTCH1_ (7%, 2% and 4%, respectively; _P_ < 0.01) mutations. On the contrary, a number of genes were found to mutate at a lower frequency in AA patients than in Asian or NHW patients, most notably _PIK3CA_ (5%, 19% and 15%, respectively; _P_ < 2 × 10−6), _RYR2_ (3%, 7% and 8%, respectively; _P_ < 0.03) and _USH2A_ (3%, 1% and 8%, respectively; _P_ < 9 × 10−4), while the mutation frequency of _AKT1_, _ATR_, ATRX, _MAP3K1_, _PREX2_ and _SETD2_ were very low in AA patients (<1%) but relatively common (≥3%) in Asian and/or NHW patients.

**Fig. 4: Comparison of somatic mutations in TNBC between AA, Asian and NHW women.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02322-y/figures/4)

**a**–**c**, The frequency of nonsilent mutations in known breast cancer genes in TNBC from AA women (_y_ axis) and NHW women (_x_ axis) (**a**), AA women (_y_ axis) and Asian women (_x_ axis) (**b**) and Asian women (_y_ axis) versus NHW women (_x_ axis) (**c**). AA women were from B-CAUSE study; Asian women were from FUSCC and NHW women were pooled from TCGA, SCAN-B and METABRIC. Each dot represents one gene with the dot size corresponding to the negated log10\-transformed two-sided _P_ value from comparison test. Genes that were substantial at _q_ < 0.10 after FDR correction are shown in red.

For _TP53_ and _PIK3CA_, the two genes showing the largest mutation frequency discrepancy across the three patient populations, the gene mutation spectrums were, nevertheless, largely similar, with some minor yet notable differences. Most of the mutations in _TP53_ were found in the DNA-binding domain (DBD), featured prominently with four hotspot mutations (R175, R213, R248 and R273), plus another hotspot mutation R342 in the tetramerization domain (Fig. [5a](https://www.nature.com/articles/s41588-025-02322-y#Fig5)). Tumors from AA women had two other hotspot mutations, H179 and E286 in the DBD, which were absent in Asian patients and at only low frequency in NHW patients. On the contrary, nonsense mutation R196\* was rare in AA patients but more common in Asian and NHW patients. The spectrum of _PIK3CA_ mutations was dominated by one hyperactivating mutation, H1047R/L, in all three populations; however, the other three hyperactivating hotspot mutations, N345K, E542K and E545K, were found only in tumors from Asian and NWH women but not from AA women (Fig. [5](https://www.nature.com/articles/s41588-025-02322-y#Fig5)b).

**Fig. 5: Mutation spectrum of _TP53_ and _PIK3CA_ in TNBC from AA, Asian and NHW women.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02322-y/figures/5)

**a**,**b**, Lollipop plot of _TP53_ (**a**) and _PIK3CA_ (**b**) somatic mutations in TNBC from AA (B-CAUSE, _n_ = 462), Asian (FUSCC, _n_ = 279) and NHW women (TCGA, SCAN-B and METABRIC, _n_ = 626). ABD, adaptor-binding domain; RBD, RAS-binding domain. The numbers in the circles indicate the number of tumors harboring the mutation in the cohort.

### African ancestry and somatic mutational features

The median proportion of African ancestry was 0.82 (interquartile range = 0.74–0.89; Supplementary Fig. [8](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)). There was no correlation of African ancestry with mutation rate, HRD score or any of the mutational signatures (Supplementary Fig. [9](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)). There were also no differences in percent African ancestry by TNBC subtype or _TP53_ hotspot mutations.

### Neoantigen analysis

Supplementary Fig. [10](https://www.nature.com/articles/s41588-025-02322-y#MOESM1) shows the number of predicted neoantigens in each tumor (median = 2, range = 0–35) with the number of nonsynonymous missense mutations, where moderate correlation was found between the two (_r_ = 0.58, _P_ < 0.001).

### RNA fusion events

We characterized fusion events in 260 TNBC patients with transcriptomic data. Using stringent filtering criteria, we identified 471 fusion mutations in 148 (56%) of the tumors, including seven recurrent fusions and 96 fusions involving a known cancer gene (Supplementary Table [10](https://www.nature.com/articles/s41588-025-02322-y#MOESM4)). The most common recurrent fusions were characterized by adjacent rearrangements involving _PTK2_ or _ETV6_, the latter of which is a tumor suppressor that turns to an oncogene in its fusion forms[38](https://www.nature.com/articles/s41588-025-02322-y#ref-CR38 "Bhojwani, D. et al. ETV6–RUNX1-positive childhood acute lymphoblastic leukemia: improved outcome with contemporary therapy. Leukemia 26, 265–270 (2012)."). We identified one tumor with _BCL2L14–ETV6_ associated with mesenchymal TNBC[39](https://www.nature.com/articles/s41588-025-02322-y#ref-CR39 "Lee, S. et al. Landscape analysis of adjacent gene rearrangements reveals BCL2L14–ETV6 gene fusions in more aggressive triple-negative breast cancer. Proc. Natl Acad. Sci. USA 117, 9912–9921 (2020).") and another with _ETV6–NTRK3_ that was a marker of secretory breast carcinoma, a rare basal-like breast cancer[40](https://www.nature.com/articles/s41588-025-02322-y#ref-CR40 "Bishop, J. A. et al. Utility of mammaglobin immunohistochemistry as a proxy marker for the ETV6–NTRK3 translocation in the diagnosis of salivary mammary analogue secretory carcinoma. Hum. Pathol. 44, 1982–1988 (2013)."),[41](https://www.nature.com/articles/s41588-025-02322-y#ref-CR41 "Tognon, C. et al. Expression of the ETV6–NTRK3 gene fusion as a primary event in human secretory breast carcinoma. Cancer Cell 2, 367–376 (2002)."). Six tumors had fusions involving _PTK2_ with multiple partners, and none retained the kinase domain (Supplementary Fig. [11](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)). Moreover, three tumors contained _PARG–BMS1_ fusion associated with metaplastic TNBC[42](https://www.nature.com/articles/s41588-025-02322-y#ref-CR42 "Piscuoglio, S. et al. Genomic and transcriptomic heterogeneity in metaplastic carcinomas of the breast. npj Breast Cancer 3, 48 (2017)."). In addition, four tumors had fusion mutations involving _NOTCH2_ or _NOTCH2NL_[43](https://www.nature.com/articles/s41588-025-02322-y#ref-CR43 "Robinson, D. R. et al. Functionally recurrent rearrangements of the MAST kinase and Notch gene families in breast cancer. Nat. Med. 17, 1646–1651 (2011).").

### Potential therapeutic targets in TNBC

Based on deleterious mutations in _BRCA1_ and _BRCA2_ and an HRD score ≥42 (ref. [44](https://www.nature.com/articles/s41588-025-02322-y#ref-CR44 "Telli, M. L. et al. Homologous recombination deficiency (HRD) score predicts response to platinum-containing neoadjuvant chemotherapy in patients with triple-negative breast cancer. Clin. Cancer Res. 22, 3764–3773 (2016).")), 332 (70%) tumors were predicted responsive to neoadjuvant chemotherapy (Fig. [6a](https://www.nature.com/articles/s41588-025-02322-y#Fig6)). We also annotated somatic mutations, CNAs and gene fusions using OncoKB[45](https://www.nature.com/articles/s41588-025-02322-y#ref-CR45 "Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. JCO Precis. Oncol. 2017, PO.17.00011 (2017)."), and identified 53% of the tumors (_n_ = 246) harboring genetic alterations with known target therapeutic agents at various confidence levels (Supplementary Table [11](https://www.nature.com/articles/s41588-025-02322-y#MOESM1) and Fig. [6b](https://www.nature.com/articles/s41588-025-02322-y#Fig6)). It should be noted that none of these molecular targets nor the associated therapies have been approved for TNBC treatment. Finally, 163 (35%) and 62 (13%) tumors had copy number gain and high-level amplification of _CD274_ (_PD-L1_), respectively, associated with higher mRNA expression (Supplementary Fig. [12](https://www.nature.com/articles/s41588-025-02322-y#MOESM1)) and predictive of response to pembrolizumab[46](https://www.nature.com/articles/s41588-025-02322-y#ref-CR46 "Huang, R. S. P. et al. Pan-cancer landscape of CD274 (PD-L1) copy number changes in 244,584 patient samples and the correlation with PD-L1 protein expression. J. Immunother. Cancer 9, e002680 (2021)."), an immune checkpoint inhibitor approved for TNBC treatment.

**Fig. 6: Clinically actionable genomic changes in TNBC from AA women.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02322-y/figures/6)

Donut plots of actionable alterations in TNBC from AA women. Numbers in the plots are percentage of cases classified to each category. **a**, HRD defined on the basis of germline and somatic mutations in _BRCA1_ and _BRCA2_, and an HRD score ≥42, which predicts response to neoadjuvant chemotherapy. **b**, Classification of actionable somatic mutations, CNAs or fusion events based on OncoKB database[45](https://www.nature.com/articles/s41588-025-02322-y#ref-CR45 "Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. JCO Precis. Oncol. 2017, PO.17.00011 (2017)."). Level 1, Food and Drug Administration (FDA)-recognized biomarker predictive of response to an FDA-approved drug; level 2, standard care biomarkers predictive of response to an FDA-approved drug; level 3a, compelling clinical evidence for the biomarker predictive of response to a drug; level 3b, standard care of investigational biomarker predictive of response to an FDA-approved or investigational drug; level 4, compelling biological evidence for the biomarker predictive of response to a drug.

Discussion
----------

This large TNBC genomic study based on 462 AA women makes a considerable contribution to the diversification of the cancer genomic knowledge base. The findings reveal that the mutational landscape of TNBC in AA women is largely similar to that in Asian and NHW women and there was no evidence of associations of mutational features with African ancestry. Therefore, our results do not support major racial differences in TNBC biology at the level of somatic mutations. Nevertheless, there are several distinctive genomic characteristics noteworthy in TNBC from AA women.

In comparison to the approximately 80% of mutation frequency in _TP53_ in previous studies[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."),[11](https://www.nature.com/articles/s41588-025-02322-y#ref-CR11 "Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. Nat. Med. 25, 1526–1533 (2019)."),[36](https://www.nature.com/articles/s41588-025-02322-y#ref-CR36 "Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. Nature 490, 61–70 (2012)."), we identified 95% TNBC from AA women harboring _TP53_ mutations. The dominance is more complete than previously recognized in Asian and NHW women[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."),[11](https://www.nature.com/articles/s41588-025-02322-y#ref-CR11 "Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. Nat. Med. 25, 1526–1533 (2019)."),[36](https://www.nature.com/articles/s41588-025-02322-y#ref-CR36 "Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. Nature 490, 61–70 (2012)."). The difference could be due to TNBC risk factors more common in AA women, or alternatively, to technical differences in sequencing or variant identification. Compared with TCGA and other WES-based studies, we used a larger exome library design with sizable custom contents, longer 150-bp reads and deeper sequencing depth. Variant calling algorithms could have also contributed to the difference in mutation frequency, where we integrated four callers followed by a manual review step for quality control purposes. The fact that we validated essentially all _TP53_ mutations in two orthogonal replication analyses supported the internal validity of our variant calling. However, caveats should be taken when comparing our data with those from external studies due to these technical differences.

We found a partial mutually exclusive pattern between the aging and HRD mutational signatures in AA women that separated the tumors into subtypes with distinct demographic, genomic and prognostic features. HRD signature is known to dominate in TNBC[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."),[11](https://www.nature.com/articles/s41588-025-02322-y#ref-CR11 "Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. Nat. Med. 25, 1526–1533 (2019)."), and occurred in 53% of the tumors in our cohort that were characterized by younger age at diagnosis, _BRCA1_ and _BRCA2_ mutations, higher mutation burden, more immunogenic and better survival. In contrast, the aging-related signatures dominated the other half of the tumors, particularly in older women and those with higher BMI, and were associated with low mutation burden, less immunogenic and poor survival. This revelation is rather surprising since TNBC in AA women is often associated with younger age, and epidemiological evidence for the association of obesity with TNBC risk has been mixed[47](https://www.nature.com/articles/s41588-025-02322-y#ref-CR47 "Bandera, E. V. et al. Obesity, body fat distribution, and risk of breast cancer subtypes in African American women participating in the AMBER Consortium. Breast Cancer Res. Treat. 150, 655–666 (2015)."),[48](https://www.nature.com/articles/s41588-025-02322-y#ref-CR48 "Dietze, E. C., Chavez, T. A. & Seewaldt, V. L. Obesity and triple-negative breast cancer: disparities, controversies, and biology. Am. J. Pathol. 188, 280–290 (2018)."). Our results suggest that there exist at least two etiological pathways of TNBC in AA women, one occurring more often in younger patients driven by genetic predisposition and deficient DNA damage repair, and the other more often in older patients driven by aging and obesity-related dysregulation. Notably, there was a recent trend of increasing TNBC incidence in AA women aged ≥55 years in the United States[49](https://www.nature.com/articles/s41588-025-02322-y#ref-CR49 "Du, X. L. & Li, Z. Incidence trends in triple-negative breast cancer among women in the United States from 2010 to 2019 by race/ethnicity, age and tumor stage. Am. J. Cancer Res. 13, 678–691 (2023)."). It is possible that the higher incidence of TNBC in AA women could be, in part, attributed to obesity, which is more prevalent in AA women than in white women[50](https://www.nature.com/articles/s41588-025-02322-y#ref-CR50 "Siddharth, S. & Sharma, D. Racial disparity and triple-negative breast cancer in African-American women: a multifaceted affair between obesity, biology, and socioeconomic determinants. Cancers 10, 514 (2018)."). It is known that SBS1 is similar to formalin-fixed paraffin-embedded (FFPE) artifacts after chemical repairment in DNA library preparation[51](https://www.nature.com/articles/s41588-025-02322-y#ref-CR51 "Guo, Q. et al. The mutational signatures of formalin fixation on the human genome. Nat. Commun. 13, 4487 (2022)."). Although we did not perform the repairment step, caution is advised when interpreting the results related to the SBS1 signature.

The comprehensive genomic characterization of TNBC from AA women provides an opportunity to assess actionable targets for potential clinical benefits. Our analysis suggested that 72% of the TNBC from AA women would be responsive to neoadjuvant chemotherapy. This estimate is higher than the approximately 60% in white patients[11](https://www.nature.com/articles/s41588-025-02322-y#ref-CR11 "Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. Nat. Med. 25, 1526–1533 (2019)."),[52](https://www.nature.com/articles/s41588-025-02322-y#ref-CR52 "Stecklein, S. R. et al. Dual prognostic classification of triple-negative breast cancer by DNA damage immune response and homologous recombination deficiency. JCO Precis. Oncol. 7, e2300197 (2023)."), which suggests that AA patients should have similar, if not higher, PCR rate to neoadjuvant chemotherapy than white patients. However, several earlier studies found the opposite[53](https://www.nature.com/articles/s41588-025-02322-y#ref-CR53 "Killelea, B. K. et al. Racial differences in the use and outcome of neoadjuvant chemotherapy for breast cancer: results from the National Cancer Database. J. Clin. Oncol. 33, 4267–4276 (2015)."),[54](https://www.nature.com/articles/s41588-025-02322-y#ref-CR54 "Roy, A. M. et al. Racial and ethnic disparity in preoperative chemosensitivity and survival in patients with early-stage breast cancer. JAMA Netw. Open 6, e2344517 (2023)."). This could be due to social or structural healthcare barriers faced by AA women[55](https://www.nature.com/articles/s41588-025-02322-y#ref-CR55 "Hossain, F. et al. Neighborhood social determinants of triple negative breast cancer. Front. Public Health 7, 18 (2019)."). Beyond chemotherapy, our data also uncovered that 53% of the TNBC tumors in AA women harbored somatic changes linked with targeted therapeutic agents. Notably, 12 TNBC patients had _ERBB2_ amplification despite being HER2 negative in clinical assays, which was also found in other TNBC studies[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."),[36](https://www.nature.com/articles/s41588-025-02322-y#ref-CR36 "Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. Nature 490, 61–70 (2012)."),[37](https://www.nature.com/articles/s41588-025-02322-y#ref-CR37 "Pereira, B. et al. The somatic mutation profiles of 2,433 breast cancers refines their genomic and transcriptomic landscapes. Nat. Commun. 7, 11479 (2016)."). This discordance could be due to false-negative clinical test, false-positive CNV results or tumor heterogeneity between samples used for clinical assays and those used later for research analysis. It might also reflect the biological heterogeneity of TNBC, as it has been shown that some TNBC tumors had activating ERBB2 signaling[10](https://www.nature.com/articles/s41588-025-02322-y#ref-CR10 "Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. Cancer Cell 35, 428–440 (2019)."). Further, 48% of the TNBC tumors in AA women had copy number gain of _CD274_, which predicts better response to pembrolizumab as a part of the chemoimmunotherapy regimen approved for TNBC. These data suggest that a majority of the AA patients with TNBC may benefit from immunotherapy or targeted therapy. A limitation of our analysis was that all patients were diagnosed and treated before the era of targeted therapy and immunotherapy for TNBC, and, thus, the therapeutic prediction was based solely on molecular data. Future clinical trials on new treatments for TNBC should strive to increase the participation of AA women[56](https://www.nature.com/articles/s41588-025-02322-y#ref-CR56 "Loree, J. M. et al. Disparity of race reporting and representation in clinical trials leading to cancer drug approvals from 2008 to 2018. JAMA Oncol. 5, e191870 (2019).").

In conclusion, in this large study of TNBC from AA women, we unveiled a high-resolution mutational portrait reminiscent of that in Asian and NHW women yet with distinctive features, including an almost complete dominance of _TP53_ mutations, fewer _PIK3CA_ mutations and mutational signature-based subtypes with etiologic and prognostic implications. These findings provide new insights into the disease epidemiology, etiology and therapeutic vulnerability of TNBC from AA women, which also deepens our understanding of this aggressive disease in other populations. Our research highlights the importance of continuous, inclusive scientific efforts to ensure that the rapidly growing cancer knowledge benefits all human populations.

Methods
-------

### Patient populations and ethical regulations

B-CAUSE Study pooled data and biospecimens from the following five population-based breast cancer studies with large number of AA women in the United States to investigate epidemiological and clinical significance of tumor somatic mutations. B-CAUSE Study was supported in part by the National Institutes of Health (grant R01 CA228156). For the protection of human participants, all participants provided informed written consent. The study was reviewed and approved by the Institutional Review Board (IRB) at all participating institutions where the patient data and samples came from, specifically Roswell Park Comprehensive Cancer Center (STUDY00000692/BDR 102718), Boston University (H-38636) and Vanderbilt University Medical Center (110190). Detailed information on the participating studies in B-CAUSE is provided in [Supplementary Note](https://www.nature.com/articles/s41588-025-02322-y#MOESM1).

### FFPE tumor tissue processing, DNA/RNA extraction and sample QC

The tumor tissue workflow has been described in detail previously[58](https://www.nature.com/articles/s41588-025-02322-y#ref-CR58 "Ondracek, R. P. et al. Results and lessons from dual extraction of DNA and RNA from formalin-fixed paraffin-embedded breast tumor tissues for a large cancer epidemiologic study. BMC Genomics 23, 614 (2022)."). Dedicated study pathologists reviewed haematoxylin and eosin slides to identify regions enriched with tumors for coring, and the resulting punches were used for DNA and RNA extraction. When tumor blocks were not available, whole sections were macrodissected to trim nontumor tissues and excess wax where possible before being used for DNA and RNA extraction. For the Women’s Circle of Health Study and the Black Women’s Health Study, pathological tumor infiltrating lymphocyte score was also recorded by a pathologist based on established guidelines[59](https://www.nature.com/articles/s41588-025-02322-y#ref-CR59 "Salgado, R. et al. The evaluation of tumor-infiltrating lymphocytes (TILs) in breast cancer: recommendations by an International TILs Working Group 2014. Ann. Oncol. 26, 259–271 (2015)."). An optimized DNA/RNA co-extraction protocol on the basis of the Qiagen AllPrep DNA/RNA FFPE kit was used to derive tumor DNA and RNA samples. After extraction, pre-analytic quality control (QC) will be performed using Agilent Bioanalyzer for size distribution check. Matched germline DNA came from those extracted from blood, saliva or mouthwash samples in the parent studies.

### WES

WES were performed by the University of Southern California (USC) Translational Genomics Laboratory using the Keck Genomics Platform and a customized Agilent SureSelect capture probeset to (1) interrogate exons from >20,000 genes based upon the SureSelect Human Whole Exome V6 backbone, (2) to analyze copy number across the genome at 60 kb resolution using additional baits representing Agilent 44k human aCGH probes and (3) to completely tile across known oncogenic breakpoints and tumor suppressor loci to uncover large indels or inversions that can be missed by standard exome sequencing[60](https://www.nature.com/articles/s41588-025-02322-y#ref-CR60 "Zhrebker, L. et al. Case report: whole exome sequencing of primary cardiac angiosarcoma highlights potential for targeted therapies. BMC Cancer 17, 17 (2017)."). The total size of the library was 73.499 Mb. Next-generation sequencing (NGS) was conducted using 150-bp paired-end reads on the Illumina NovaSeq 6000 system, with a target sequencing depth of at least 70% bases at 20× for normal DNA and 80% bases at 20× for tumor DNA. A total of 513 pairs of matched tumor and normal DNA samples were sequenced, which generated 0.3 trillion paired-end reads. The median sequencing depth was 154× (range = 14× to 712×), with 89% (range = 16–100%) bases above 20×.

### Bioinformatic processing and variant calling

Bioinformatic processing work was carried out using integrative workflows implemented in the RcwlPipelines package[61](https://www.nature.com/articles/s41588-025-02322-y#ref-CR61 "Hu, Q., Hutson, A., Liu, S., Morgan, M. & Liu, Q. Bioconductor toolchain for reproducible bioinformatics pipelines using Rcwl and RcwlPipelines. Bioinformatics 37, 3351–3352 (2021)."). The bioinformatics analysis began by aligning high-quality paired-end reads passing Illumina RTA filter to the NCBI human reference genome (GRCh37) using BWA-MEM aligner[62](https://www.nature.com/articles/s41588-025-02322-y#ref-CR62 "Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25, 1754–1760 (2009)."). PCR duplicated reads are marked and removed by using Picard ([http://picard.sourceforge.net/](http://picard.sourceforge.net/)). The median map rate was 0.997, median duplication rate was 0.203 and median sequencing depth was 127×. There were 20 tumor samples and 1 normal sample that did not reach the targeted sequencing depth, 10 samples with unmatched tumor-normal sample identity and 4 samples with cryptic relatedness were removed from analysis, leaving 478 tumor-normal pairs. Tumor purity was estimated using FACETS[63](https://www.nature.com/articles/s41588-025-02322-y#ref-CR63 "Shen, R. & Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. Nucleic Acids Res. 44, e131 (2016).") and 16 samples with low tumor purity (<0.10) were also removed. As a result, 462 tumor-normal pairs were retained in the final analysis. After QC, the median sequencing coverage was 91% (range = 80–100%) for tumor DNA and 87% (70–100%) for normal DNA, respectively, of the bases within the targeted regions with at least 20× depth.

Somatically acquired mutations in tumors were identified from WES data by comparing tumor sequences with individual-matched germline sequences as previously described[64](https://www.nature.com/articles/s41588-025-02322-y#ref-CR64 "Wei, L. et al. Intratumoral and intertumoral genomic heterogeneity of multifocal localized prostate cancer impacts molecular classifications and genomic prognosticators. Eur. Urol. 71, 183–192 (2016)."). Putative SNVs and small indels were initially identified by running four different variant callers, Strelka[65](https://www.nature.com/articles/s41588-025-02322-y#ref-CR65 "Saunders, C. T. et al. Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs. Bioinformatics 28, 1811–1817 (2012)."), MuSE[66](https://www.nature.com/articles/s41588-025-02322-y#ref-CR66 "Fan, Y. et al. MuSE: accounting for tumor heterogeneity using a sample-specific error model improves sensitivity and specificity in mutation calling from sequencing data. Genome Biol. 17, 178 (2016)."), VarDict[67](https://www.nature.com/articles/s41588-025-02322-y#ref-CR67 "Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. Nucleic Acids Res. 44, e108 (2016).") and Mutect[68](https://www.nature.com/articles/s41588-025-02322-y#ref-CR68 "Cibulskis, K. et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. Nat. Biotechnol. 31, 213–219 (2013)."), through established pipelines in the RcwlPipelines package. All putative SNVs were further filtered based on the following standard set of criteria to remove the common types of false calls: (1) the alternative allele was present in the matching normal sample and the contingency between the tumor and normal samples was not statistically significant, (2) the mutant alleles were only present in one stand and the strand bias was statistically significant, (3) the putative mutation occurred at a site with systematically dropped base quality scores and (4) the reads harboring the mutant allele were associated with poor mapping quality. Ambiguous cases were manually inspected to ensure accuracy using Bambino viewer[69](https://www.nature.com/articles/s41588-025-02322-y#ref-CR69 "Edmonson, M. N. et al. Bambino: a variant detector and alignment viewer for next-generation sequencing data in the SAM/BAM format. Bioinformatics 27, 865–866 (2011)."). Putative indels were evaluated by a re-alignment process to filter out potential false calls introduced by unapparent germline events, mapping artifacts and homopolymer. The identified somatic mutations were compared with the public human germline databases, including dbSNP[70](https://www.nature.com/articles/s41588-025-02322-y#ref-CR70 "Sherry, S. T. et al. dbSNP: the NCBI database of genetic variation. Nucleic Acids Res. 29, 308–311 (2001)."), 1000 Genomes Project[71](https://www.nature.com/articles/s41588-025-02322-y#ref-CR71 "Genomes Project, C. et al. An integrated map of genetic variation from 1,092 human genomes. Nature 491, 56–65 (2012)."), National Heart, Lung and Blood Institute’s Exome Sequencing Project[26](https://www.nature.com/articles/s41588-025-02322-y#ref-CR26 "Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Nature 590, 290–299 (2021).") and the Genome Aggregation Database (gnomAD)[72](https://www.nature.com/articles/s41588-025-02322-y#ref-CR72 "Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)."). The identified mutations were annotated by ANNOVAR[73](https://www.nature.com/articles/s41588-025-02322-y#ref-CR73 "Wang, K., Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 38, e164 (2010).") using the Ensembl database[74](https://www.nature.com/articles/s41588-025-02322-y#ref-CR74 "McLaren, W. et al. The Ensembl variant effect predictor. Genome Biol. 17, 122 (2016)."). Multiple nucleotide variants were identified and annotated using MAC[75](https://www.nature.com/articles/s41588-025-02322-y#ref-CR75 "Wei, L. et al. MAC: identifying and correcting annotation for multi-nucleotide variations. BMC Genomics 16, 569 (2015)."). Tumor mutation burden was calculated as the number of SNVs per Mb of genomic region sequenced. Hypermutated samples were identified as those with a mutation rate ≥10 SNVs per Mb[13](https://www.nature.com/articles/s41588-025-02322-y#ref-CR13 "Campbell, B. B. et al. Comprehensive analysis of hypermutation in human cancer. Cell 171, 1042–1056 (2017).").

### Confirmation of _TP53_ mutations

Two orthogonal approaches were used to confirm _TP53_ mutations identified by WES data. First, for the 260 tumor samples with available matched transcriptome data, the mutant and wild-type (WT) reads were extracted from the transcriptome data to determine the expression level of the mutant allele. Second, an independent resequencing assay by TAS that covered the entire coding region of _TP53_ was carried out using 338 tumor samples and eight normal samples with adequate amount of DNA samples left after WES. Each putative _TP53_ mutation identified from WES was revisited in the TAS data to extract the mutant and WT read counts from the BAM file as previously described[76](https://www.nature.com/articles/s41588-025-02322-y#ref-CR76 "Wei, L. et al. Accurate quantification of residual cancer cells in pelvic washing reveals association with cancer recurrence following robot-assisted radical cystectomy. J. Urol. 201, 1105–1114 (2019)."). A mutation was considered as confirmed if its variant allele fractions in the tumor TAS were substantially higher than in the control sample using a highly stringent criterion (_P_ < 10 × 10−10, Fisher’s exact test). For the samples with no _TP53_ mutation detected by the original WES, de novo mutation detection was performed in the TAS data by comparing each tumor with a pool of all normal samples as previously described[77](https://www.nature.com/articles/s41588-025-02322-y#ref-CR77 "Wei, L. et al. Ultradeep sequencing differentiates patterns of skin clonal mutations associated with sun-exposure status and skin cancer burden. Sci. Adv. 7, eabd7703 (2021)."), which identified two new _TP53_ mutations, including one SNV and one indel. These two newly identified _TP53_ mutations were manually inspected using Bambino viewer[69](https://www.nature.com/articles/s41588-025-02322-y#ref-CR69 "Edmonson, M. N. et al. Bambino: a variant detector and alignment viewer for next-generation sequencing data in the SAM/BAM format. Bioinformatics 27, 865–866 (2011).") and the indel was further confirmed using IndelPost[78](https://www.nature.com/articles/s41588-025-02322-y#ref-CR78 "Hagiwara, K., Edmonson, M. N., Wheeler, D. A. & Zhang, J. IndelPost: harmonizing ambiguities in simple and complex indel alignments. Bioinformatics 38, 549–551 (2022).").

### Somatic copy number alteration analysis

Tumor segmentation and somatic copy number alteration (SCNA) calling were carried out using FACETS[63](https://www.nature.com/articles/s41588-025-02322-y#ref-CR63 "Shen, R. & Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. Nucleic Acids Res. 44, e131 (2016).") based on the WES data from a customized sequencing library originally designed with additional baits representing Agilent 44k human aCGH array probes. GISTIC2 (ref. [79](https://www.nature.com/articles/s41588-025-02322-y#ref-CR79 "Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 12, R41 (2011).")) was then used to identify substantially amplified or deleted SCNAs with a residual false discovery rate (FDR) _q_ < 0.25. Genes residing within the wide peak regions were identified and a GISTIC threshold CN value (−2, −1, 0, 1, 2) indicating the amplitude (high-level deletion, low-level deletion, diploid, low-level amplification and high-level amplification) of the CNA was assigned to each tumor.

### SMG analysis

MutSigCV[17](https://www.nature.com/articles/s41588-025-02322-y#ref-CR17 "Lawrence, M. S. et al. Mutational heterogeneity in cancer and the search for new cancer-associated genes. Nature 499, 214–218 (2013)."), MutSig2CV[18](https://www.nature.com/articles/s41588-025-02322-y#ref-CR18 "Lawrence, M. S. et al. Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 505, 495–501 (2014).") and MuSiC[19](https://www.nature.com/articles/s41588-025-02322-y#ref-CR19 "Dees, N. D. et al. MuSiC: identifying mutational significance in cancer genomes. Genome Res. 22, 1589–1598 (2012).") were used to identify genes mutated at a substantially higher rate than the background mutation rate, with significance set at FDR _q_ < 0.20. The final list of SMGs were determined as those that were substantial in two or more of the three programs used. Known TNBC genes are defined based on SMGs identified in prior studies focusing specifically on TNBC; known breast cancer genes are defined based on SMGs identified in prior studies focusing on breast cancer but not specifically on TNBC; and known pan-cancer genes are defined based on SMGs in prior pan-cancer studies.

### Pathogenic germline mutation analysis

Germline WES data from AA TNBC patients in B-CAUSE were used for analysis of germline mutations in nine known breast cancer genes, including _ATM_, _BRCA1_, _BRCA2_, _CDH1_, _CHEK2_, _NF1_, _PALB2_, _PTEN_ and _TP53_. Pathogenicity data were extracted from ClinVar[28](https://www.nature.com/articles/s41588-025-02322-y#ref-CR28 "Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 42, D980–D985 (2014)."). All germline variants identified were queried in public population reference databases, including gnomAD[25](https://www.nature.com/articles/s41588-025-02322-y#ref-CR25 "Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. Nature 625, 92–100 (2024)."), TOPMed[26](https://www.nature.com/articles/s41588-025-02322-y#ref-CR26 "Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Nature 590, 290–299 (2021)."), National Institutes of Health All of US[24](https://www.nature.com/articles/s41588-025-02322-y#ref-CR24 "All of Us Research Program Genomics Investigators. Genomic data in the All of Us Research Program. Nature 627, 340–346 (2024).") and the Regeneron Genetics Center. Whenever possible, reference data from populations of African ancestry were used for comparison. To formally test the allele frequency of the germline variants in TNBC patients in B-CAUSE against the reference datasets, chi-square or Fisher’s exact test was used, whichever was appropriate. Finally, the germline variants were cross-referenced with those characterized in functional studies, including _BRCA1_ (ref. [29](https://www.nature.com/articles/s41588-025-02322-y#ref-CR29 "Findlay, G. M. et al. Accurate classification of BRCA1 variants with saturation genome editing. Nature 562, 217–222 (2018).")), _BRCA2_ (ref. [80](https://www.nature.com/articles/s41588-025-02322-y#ref-CR80 "Huang, H. et al. Functional evaluation and clinical classification of BRCA2 variants. Nature 638, 528–537 (2025).")), _CHEK2_ (ref. [81](https://www.nature.com/articles/s41588-025-02322-y#ref-CR81 "McCarthy-Leo, C. E. et al. Comprehensive analysis of the functional impact of single nucleotide variants of human CHEK2. PLoS Genet. 20, e1011375 (2024).")) and _TP53_ (ref. [82](https://www.nature.com/articles/s41588-025-02322-y#ref-CR82 "Funk, J. S. et al. Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations. Nat. Genet. 57, 140–153 (2025).")).

### GSEA

To identify common pathways somatically altered in TNBC, GSEA was carried out based on 218 driver genes harboring recurrent mutations ≥1% in the cohort using Hallmark gene sets from the MSigDB[83](https://www.nature.com/articles/s41588-025-02322-y#ref-CR83 "Liberzon, A. et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst. 1, 417–425 (2015)."). An FDR _q_ ≤ 0.05 was considered substantial.

### Mutational signature analysis

Mutational signature analysis was performed with sigProfilerExtractor[84](https://www.nature.com/articles/s41588-025-02322-y#ref-CR84 "Islam, S. M. A. et al. Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor. Cell Genom. 2, 100179 (2022)."), which allows de novo extraction of operative mutational signatures and subsequently decomposition to known COSMIC mutational signatures[30](https://www.nature.com/articles/s41588-025-02322-y#ref-CR30 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)."). SBS and indel signatures were extracted, whereas double base substation mutations were sparse in the WES data and not analyzed. The five tumor samples with hypermutation phenotype were excluded from mutational signature analysis. To define SBS-based TNBC subtypes, the combined activity of the aging-related signatures SBS1 and SBS5, along with the HRD-related signature SBS3, was categorized into tertiles and assessed for association with patient survival outcomes. Based on the survival differences, the aging signature and the HRD signature were further collapsed into two groups (T1 and T2 versus T3). The resultant binary classifications of the two signatures were used to create a combined SBS-based TNBC subtype between aging and HRD signatures. SBS subtype 1 had low aging and high HRD signatures, subtype 2 had low aging and low HRD signatures, subtype 3 had high aging and low HRD signatures and subtype 4 had high aging and high HRD signatures.

### Genetic ancestry

Global genetic ancestry was estimated based on germline genotype data derived from WES of the matched normal samples using ALStructure[85](https://www.nature.com/articles/s41588-025-02322-y#ref-CR85 "Cabreros, I. & Storey, J. D. A likelihood-free estimator of population structure bridging admixture models and principal components analysis. Genetics 212, 1009–1029 (2019)."). Data of reference ancestral populations were obtained from the 1000 Genomes Project[71](https://www.nature.com/articles/s41588-025-02322-y#ref-CR71 "Genomes Project, C. et al. An integrated map of genetic variation from 1,092 human genomes. Nature 491, 56–65 (2012).") and used as benchmarks in the estimation.

### RNA sequencing and bioinformatic processing

RNA sequencing for tumors from the Women’s Circle of Health Study and the Black Women’s Health Study was performed using the Agilent SureSelectXT Exome RNA kit, which is optimized for library preparation from degraded RNA samples in FFPE tissues. NGS was then conducted on an Illumina NovaSeq 6000 sequencer, yielding an average of 50 million 150-bp paired-end reads per sample. Sequencing work for tumors from Vanderbilt studies was performed using the BGI-Seq platform with 100-bp read length. Raw reads were demultiplexed and preprocessed by using FastQC[86](https://www.nature.com/articles/s41588-025-02322-y#ref-CR86 "Andrew, S. FastQC: a quality control tool for high throughput sequence data. Babraham Bioinform. 
www.bioinformatics.babraham.ac.uk/projects/fastqc
(2010).") and Cutadapt[87](https://www.nature.com/articles/s41588-025-02322-y#ref-CR87 "Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J. 17, 10–12 (2011)."). The remaining reads were mapped to the GRCh38 and reference transcriptome GENCODE using STAR-Aligner[88](https://www.nature.com/articles/s41588-025-02322-y#ref-CR88 "Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013)."). Additional QC steps were performed to identify library preparation problems using RSeQC[89](https://www.nature.com/articles/s41588-025-02322-y#ref-CR89 "Wang, L., Wang, S. & Li, W. RSeQC: quality control of RNA-seq experiments. Bioinformatics 28, 2184–2185 (2012).") and/or RNA degradation issues using mRIN[90](https://www.nature.com/articles/s41588-025-02322-y#ref-CR90 "Feng, H., Zhang, X. & Zhang, C. mRIN for direct assessment of genome-wide and gene-specific mRNA integrity from large-scale RNA-sequencing data. Nat. Commun. 6, 7816 (2015)."). RSEM[91](https://www.nature.com/articles/s41588-025-02322-y#ref-CR91 "Li, B. & Dewey, C. N. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics 12, 323 (2011).") was used to quantify gene expression as transcript per million reads and fragments per kilobase per million mapped reads. Batch correction was conducted using ComBat-seq[92](https://www.nature.com/articles/s41588-025-02322-y#ref-CR92 "Zhang, Y., Parmigiani, G. & Johnson, W. E. ComBat-seq: batch effect adjustment for RNA-seq count data. NAR Genom. Bioinform 2, lqaa078 (2020).") and wherever appropriate, study site was adjusted as a covariate in multivariable models. For sample-level and gene-level filtering, we excluded genes with low or no expression; those with either an interquartile range equal to zero or a sum across samples ≤1; and outliers based on the principal component analysis and the RNA degradation threshold.

### RNA fusion events

RNA fusions were characterized in 260 TNBC samples with transcriptomic data available. Arriba[93](https://www.nature.com/articles/s41588-025-02322-y#ref-CR93 "Uhrig, S. et al. Accurate and efficient detection of gene fusions from RNA sequencing data. Genome Res. 31, 448–460 (2021).") and STAR-Fusion[94](https://www.nature.com/articles/s41588-025-02322-y#ref-CR94 "Haas, B. J. et al. STAR-Fusion: fast and accurate fusion transcript detection from RNA-seq. Preprint at bioRxiv 
https://doi.org/10.1101/120295
(2017).") programs were both used. For stringent filtering, high-confidence chimeric transcripts identified from each program were intersected and only those overlapping between the two programs were reported. Genomic plots of the fusion mutations and resultant protein domains are generated using script provided in Arriba.

### Neoantigen analysis

Neoantigens are aberrant peptides introduced by nonsynonymous somatic mutations in tumors, some of which can be recognized by the immune system and illicit antigen-specific immune response in the tumor microenvironment and thus are predictive of cancer immunotherapy. To characterize neoantigen profiles in TNBC tumors, we cross-referenced the predicted binding affinity (IC50 < 500 nM) of 8-mer to 15-mer amino acid sequences generated by each nonsynonymous missense mutation with mRNA expression (read count ≥ 1) to estimate the number of aberrant peptides as potential neoantigens.

### Potential molecular alterations

Three approaches were applied to identify actionable alterations in TNBC. First, to predict response to neoadjuvant chemotherapy, an HRD score was calculated as the sum of the three factors, including loss of heterozygosity, telomeric allelic imbalance and large-scale state transition scores, using BAM files of tumor samples by the scarHRD R package with default parameters[57](https://www.nature.com/articles/s41588-025-02322-y#ref-CR57 "Sztupinszki, Z. et al. Migrating the SNP array-based homologous recombination deficiency measures to next generation sequencing data of breast cancer. npj Breast Cancer 4, 16 (2018)."). An established cutoff value of HRD score ≥42 (ref. [44](https://www.nature.com/articles/s41588-025-02322-y#ref-CR44 "Telli, M. L. et al. Homologous recombination deficiency (HRD) score predicts response to platinum-containing neoadjuvant chemotherapy in patients with triple-negative breast cancer. Clin. Cancer Res. 22, 3764–3773 (2016).")) was used to classify tumor with HRD-high status. Further, patients carrying either pathogenic germline variants or somatic mutations in _BRCA1_ and _BRCA2_ were automatically classified as HRD-high status. Second, all somatic mutations, SCNAs and RNA fusion events were annotated for targeted therapeutic agents using the OncoKB[45](https://www.nature.com/articles/s41588-025-02322-y#ref-CR45 "Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. JCO Precis. Oncol. 2017, PO.17.00011 (2017)."),[95](https://www.nature.com/articles/s41588-025-02322-y#ref-CR95 "Suehnholz, S. P. et al. Quantifying the expanding landscape of clinical actionability for patients with cancer. Cancer Discov. 14, 49–65 (2024)."), which classified the molecular alterations into four levels based on the strength of the evidence. Third, copy number gain of _CD274_ encoding for PD-L1 was analyzed as predictive marker for response to immune checkpoint inhibitors.

### Publicly available TNBC mutation datasets

For comparison to somatic mutations in TNBC patients from AA women in B-CAUSE, publicly available data were aggregated from Asian women (_n_ = 279) in FUSCC and NHW women in TCGA (_n_ = 69), the SCAN-B (_n_ = 254) and the METABRIC (_n_ = 320). More details of the public datasets are provided in [Supplementary Note](https://www.nature.com/articles/s41588-025-02322-y#MOESM1).

### Statistical analysis

This study was designed as an observational analysis to characterize the tumor somatic mutational landscape of TNBC in AA women, and thus is not considered a randomized study. To maximize statistical power, all AA women with TNBC from the participating studies with available tumor and matched normal DNA samples were included, and thus no statistical method was used to predetermine sample size. Except for samples removed due to poor data quality in the quality control steps, no other samples were excluded. Because all samples were from TNBC cases and tumor and normal samples were sequenced separately to reach different targeted sequencing depth, the laboratory performing the sequencing was not blinded to tumor versus normal status of the samples.

Descriptive statistics were computed as mean and s.d. for numeric variables and count and percentage for categorical variables. Correlation analyses were performed using the Pearson method. For comparisons of means or medians between groups, _t_ tests, analysis of variance or equivalent nonparametric version tests were applied. For survival analysis, Kaplan–Meier curves were plotted with _P_ values derived from log-rank tests, followed by Cox proportional hazard models with adjustment for covariates (age at diagnosis, study and cancer stage). All analyses were performed in R (v.4.4.0) and a two-side _P_ ≤ 0.05 was considered statistically significant, unless otherwise specified.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02322-y#MOESM2) linked to this article.

Data availability
-----------------

TCGA data: [https://xena.ucsc.edu/public/](https://xena.ucsc.edu/public/). The SCAN-B: [https://data.mendeley.com/datasets/2mn4ctdpxp/3](https://data.mendeley.com/datasets/2mn4ctdpxp/3). The METABRIC: [https://github.com/cBioPortal/datahub/tree/master/reference\_data/gene\_panels](https://github.com/cBioPortal/datahub/tree/master/reference_data/gene_panels). The FUSCC: [https://figshare.com/articles/dataset/A\_comprehensive\_genomic\_and\_transcriptomic\_dataset\_of\_triple-negative\_breast\_cancers/19783498/5](https://figshare.com/articles/dataset/A_comprehensive_genomic_and_transcriptomic_dataset_of_triple-negative_breast_cancers/19783498/5). Human Reference Genome (GRCh37): [https://www.ncbi.nlm.nih.gov/datasets/genome/GCF\_000001405.13/](https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.13/)

The WES data of matched tumor and normal TNBC samples in B-CAUSE have been deposited to the database of Genotypes and Phenotypes (dbGaP), with access number phs003962.v1.p1.

Code availability
-----------------

No original or custom code or special software or program was used that was central to the data analyses. The following standard open-source software and programs were used in the data analysis: RcwlPipelines R package (v.1.20.0), BWA-MEM aligner (v.0.7.18), Strelka2 (v.2.9.10), MuSE (v.2.0), VarDict (v.1.1), Mutect2 (v.4.1.0.0), FACETS (v.0.6.2), GISTIC2 (v.2.0.23), MutSigCV (v.1.3.01), MutSig2CV, MuSiC (v.0.4), sigProfilerExtractor (v.1.1.24), ALStructure R package, FastQC (v.0.12.1), Cutadapt (v.2.0), STAR-Aligner (v.2.7.11b), RSeQC (v.5.0.1), mRIN (v.1.2.0), RSEM (v.1.3.3), sva-ComBat-seq R package (v.3.52.0), Arriba (v.2.4.0), STAR-Fusion (v.1.13.0), scarHRD R package (v.0.1.1), GSEA (v.4.3.3), MSigDB (v.2023.2), OncoKB API (v.3.4.1) and R programming (v.4.4.0).

References
----------

1.  Dietze, E. C., Sistrunk, C., Miranda-Carboni, G., O’Regan, R. & Seewaldt, V. L. Triple-negative breast cancer in African-American women: disparities versus biology. _Nat. Rev. Cancer_ **15**, 248–254 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXisFCgu7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25673085)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5470637)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Triple-negative%20breast%20cancer%20in%20African-American%20women%3A%20disparities%20versus%20biology&journal=Nat.%20Rev.%20Cancer&volume=15&pages=248-254&publication_year=2015&author=Dietze%2CEC&author=Sistrunk%2CC&author=Miranda-Carboni%2CG&author=O%E2%80%99Regan%2CR&author=Seewaldt%2CVL) 
    
2.  Bianchini, G., De Angelis, C., Licata, L. & Gianni, L. Treatment landscape of triple-negative breast cancer—expanded options, evolving needs. _Nat. Rev. Clin. Oncol._ **19**, 91–113 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XkvV2qurs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34754128)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Treatment%20landscape%20of%20triple-negative%20breast%20cancer%E2%80%94expanded%20options%2C%20evolving%20needs&journal=Nat.%20Rev.%20Clin.%20Oncol.&volume=19&pages=91-113&publication_year=2022&author=Bianchini%2CG&author=Angelis%2CC&author=Licata%2CL&author=Gianni%2CL) 
    
3.  Sung, H., Wiese, D., Jatoi, I. & Jemal, A. State variation in racial and ethnic disparities in incidence of triple-negative breast cancer among US women. _JAMA Oncol._ **9**, 700–704 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36862439)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9982739)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=State%20variation%20in%20racial%20and%20ethnic%20disparities%20in%20incidence%20of%20triple-negative%20breast%20cancer%20among%20US%20women&journal=JAMA%20Oncol.&volume=9&pages=700-704&publication_year=2023&author=Sung%2CH&author=Wiese%2CD&author=Jatoi%2CI&author=Jemal%2CA) 
    
4.  Spratt, D. E. et al. Racial/ethnic disparities in genomic sequencing. _JAMA Oncol._ **2**, 1070–1074 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27366979)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5123755)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Racial%2Fethnic%20disparities%20in%20genomic%20sequencing&journal=JAMA%20Oncol.&volume=2&pages=1070-1074&publication_year=2016&author=Spratt%2CDE) 
    
5.  Ansari-Pour, N. et al. Whole-genome analysis of Nigerian patients with breast cancer reveals ethnic-driven somatic evolution and distinct genomic subtypes. _Nat. Commun._ **12**, 6946 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFyrtLbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34836952)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8626467)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20analysis%20of%20Nigerian%20patients%20with%20breast%20cancer%20reveals%20ethnic-driven%20somatic%20evolution%20and%20distinct%20genomic%20subtypes&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Ansari-Pour%2CN) 
    
6.  Pitt, J. J. et al. Characterization of Nigerian breast cancer reveals prevalent homologous recombination deficiency and aggressive molecular features. _Nat. Commun._ **9**, 4181 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30327465)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6191428)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Characterization%20of%20Nigerian%20breast%20cancer%20reveals%20prevalent%20homologous%20recombination%20deficiency%20and%20aggressive%20molecular%20features&journal=Nat.%20Commun.&volume=9&publication_year=2018&author=Pitt%2CJJ) 
    
7.  Keenan, T. et al. Comparison of the genomic landscape between primary breast cancer in African American versus white women and the association of racial differences with tumor recurrence. _J. Clin. Oncol._ **33**, 3621–3627 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVakt73I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26371147)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4979243)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparison%20of%20the%20genomic%20landscape%20between%20primary%20breast%20cancer%20in%20African%20American%20versus%20white%20women%20and%20the%20association%20of%20racial%20differences%20with%20tumor%20recurrence&journal=J.%20Clin.%20Oncol.&volume=33&pages=3621-3627&publication_year=2015&author=Keenan%2CT) 
    
8.  Huo, D. et al. Comparison of breast cancer molecular features and survival by African and European ancestry in The Cancer Genome Atlas. _JAMA Oncol._ **3**, 1654–1662 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28472234)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5671371)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparison%20of%20breast%20cancer%20molecular%20features%20and%20survival%20by%20African%20and%20European%20ancestry%20in%20The%20Cancer%20Genome%20Atlas&journal=JAMA%20Oncol.&volume=3&pages=1654-1662&publication_year=2017&author=Huo%2CD) 
    
9.  Chang, C. S., Kitamura, E., Johnson, J., Bollag, R. & Hawthorn, L. Genomic analysis of racial differences in triple negative breast cancer. _Genomics_ **111**, 1529–1542 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlClsbzF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30366040)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20analysis%20of%20racial%20differences%20in%20triple%20negative%20breast%20cancer&journal=Genomics&volume=111&pages=1529-1542&publication_year=2019&author=Chang%2CCS&author=Kitamura%2CE&author=Johnson%2CJ&author=Bollag%2CR&author=Hawthorn%2CL) 
    
10.  Jiang, Y. Z. et al. Genomic and transcriptomic landscape of triple-negative breast cancers: subtypes and treatment strategies. _Cancer Cell_ **35**, 428–440 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXks1Sit7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30853353)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20and%20transcriptomic%20landscape%20of%20triple-negative%20breast%20cancers%3A%20subtypes%20and%20treatment%20strategies&journal=Cancer%20Cell&volume=35&pages=428-440&publication_year=2019&author=Jiang%2CYZ) 
    
11.  Staaf, J. et al. Whole-genome sequencing of triple-negative breast cancers in a population-based clinical study. _Nat. Med._ **25**, 1526–1533 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVOlur7J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31570822)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6859071)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Whole-genome%20sequencing%20of%20triple-negative%20breast%20cancers%20in%20a%20population-based%20clinical%20study&journal=Nat.%20Med.&volume=25&pages=1526-1533&publication_year=2019&author=Staaf%2CJ) 
    
12.  Shah, S. P. et al. The clonal and mutational evolution spectrum of primary triple-negative breast cancers. _Nature_ **486**, 395–399 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmtVersL4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22495314)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20clonal%20and%20mutational%20evolution%20spectrum%20of%20primary%20triple-negative%20breast%20cancers&journal=Nature&volume=486&pages=395-399&publication_year=2012&author=Shah%2CSP) 
    
13.  Campbell, B. B. et al. Comprehensive analysis of hypermutation in human cancer. _Cell_ **171**, 1042–1056 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs12lsL3K)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29056344)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5849393)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20analysis%20of%20hypermutation%20in%20human%20cancer&journal=Cell&volume=171&pages=1042-1056&publication_year=2017&author=Campbell%2CBB) 
    
14.  Supek, F., Minana, B., Valcarcel, J., Gabaldon, T. & Lehner, B. Synonymous mutations frequently act as driver mutations in human cancers. _Cell_ **156**, 1324–1335 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXksVWrtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24630730)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synonymous%20mutations%20frequently%20act%20as%20driver%20mutations%20in%20human%20cancers&journal=Cell&volume=156&pages=1324-1335&publication_year=2014&author=Supek%2CF&author=Minana%2CB&author=Valcarcel%2CJ&author=Gabaldon%2CT&author=Lehner%2CB) 
    
15.  Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. _Cell_ **173**, 371–385 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFaiu7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29625053)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6029450)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20characterization%20of%20cancer%20driver%20genes%20and%20mutations&journal=Cell&volume=173&pages=371-385&publication_year=2018&author=Bailey%2CMH) 
    
16.  ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium Pan-cancer analysis of whole genomes. _Nature_ **578**, 82–93 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-cancer%20analysis%20of%20whole%20genomes&journal=Nature&volume=578&pages=82-93&publication_year=2020) 
    
17.  Lawrence, M. S. et al. Mutational heterogeneity in cancer and the search for new cancer-associated genes. _Nature_ **499**, 214–218 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpsFSgsbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23770567)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3919509)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20heterogeneity%20in%20cancer%20and%20the%20search%20for%20new%20cancer-associated%20genes&journal=Nature&volume=499&pages=214-218&publication_year=2013&author=Lawrence%2CMS) 
    
18.  Lawrence, M. S. et al. Discovery and saturation analysis of cancer genes across 21 tumour types. _Nature_ **505**, 495–501 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFOis7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24390350)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4048962)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovery%20and%20saturation%20analysis%20of%20cancer%20genes%20across%2021%20tumour%20types&journal=Nature&volume=505&pages=495-501&publication_year=2014&author=Lawrence%2CMS) 
    
19.  Dees, N. D. et al. MuSiC: identifying mutational significance in cancer genomes. _Genome Res._ **22**, 1589–1598 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFOnsLrE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22759861)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3409272)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MuSiC%3A%20identifying%20mutational%20significance%20in%20cancer%20genomes&journal=Genome%20Res.&volume=22&pages=1589-1598&publication_year=2012&author=Dees%2CND) 
    
20.  Lehmann, B. D. et al. Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies. _J. Clin. Invest._ **121**, 2750–2767 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXovVKgu78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21633166)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3127435)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20human%20triple-negative%20breast%20cancer%20subtypes%20and%20preclinical%20models%20for%20selection%20of%20targeted%20therapies&journal=J.%20Clin.%20Invest.&volume=121&pages=2750-2767&publication_year=2011&author=Lehmann%2CBD) 
    
21.  Lehmann, B. D. et al. Multi-omics analysis identifies therapeutic vulnerabilities in triple-negative breast cancer subtypes. _Nat. Commun._ **12**, 6276 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlyktrvP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34725325)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8560912)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-omics%20analysis%20identifies%20therapeutic%20vulnerabilities%20in%20triple-negative%20breast%20cancer%20subtypes&journal=Nat.%20Commun.&volume=12&publication_year=2021&author=Lehmann%2CBD) 
    
22.  Li, R. Q. et al. Genomic characterization reveals distinct mutational landscapes and therapeutic implications between different molecular subtypes of triple-negative breast cancer. _Sci. Rep._ **14**, 12386 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtl2nsbzO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38811720)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11137060)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20characterization%20reveals%20distinct%20mutational%20landscapes%20and%20therapeutic%20implications%20between%20different%20molecular%20subtypes%20of%20triple-negative%20breast%20cancer&journal=Sci.%20Rep.&volume=14&publication_year=2024&author=Li%2CRQ) 
    
23.  Lehmann, B. D. et al. PIK3CA mutations in androgen receptor-positive triple negative breast cancer confer sensitivity to the combination of PI3K and androgen receptor inhibitors. _Breast Cancer Res._ **16**, 406 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25103565)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4187324)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PIK3CA%20mutations%20in%20androgen%20receptor-positive%20triple%20negative%20breast%20cancer%20confer%20sensitivity%20to%20the%20combination%20of%20PI3K%20and%20androgen%20receptor%20inhibitors&journal=Breast%20Cancer%20Res.&volume=16&publication_year=2014&author=Lehmann%2CBD) 
    
24.  All of Us Research Program Genomics Investigators. Genomic data in the All of Us Research Program. _Nature_ **627**, 340–346 (2024).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20data%20in%20the%20All%20of%20Us%20Research%20Program&journal=Nature&volume=627&pages=340-346&publication_year=2024) 
    
25.  Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. _Nature_ **625**, 92–100 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFGgsrjF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38057664)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20genomic%20mutational%20constraint%20map%20using%20variation%20in%2076%2C156%20human%20genomes&journal=Nature&volume=625&pages=92-100&publication_year=2024&author=Chen%2CS) 
    
26.  Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. _Nature_ **590**, 290–299 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjslKnsrY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33568819)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875770)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sequencing%20of%2053%2C831%20diverse%20genomes%20from%20the%20NHLBI%20TOPMed%20Program&journal=Nature&volume=590&pages=290-299&publication_year=2021&author=Taliun%2CD) 
    
27.  Sun, K. Y. et al. A deep catalogue of protein-coding variation in 983,578 individuals. _Nature_ **631**, 583–592 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFCqsbvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38768635)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11254753)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20deep%20catalogue%20of%20protein-coding%20variation%20in%20983%2C578%20individuals&journal=Nature&volume=631&pages=583-592&publication_year=2024&author=Sun%2CKY) 
    
28.  Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. _Nucleic Acids Res._ **42**, D980–D985 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXoslGh)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24234437)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ClinVar%3A%20public%20archive%20of%20relationships%20among%20sequence%20variation%20and%20human%20phenotype&journal=Nucleic%20Acids%20Res.&volume=42&pages=D980-D985&publication_year=2014&author=Landrum%2CMJ) 
    
29.  Findlay, G. M. et al. Accurate classification of BRCA1 variants with saturation genome editing. _Nature_ **562**, 217–222 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslSnu7bI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30209399)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6181777)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20classification%20of%20BRCA1%20variants%20with%20saturation%20genome%20editing&journal=Nature&volume=562&pages=217-222&publication_year=2018&author=Findlay%2CGM) 
    
30.  Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. _Nature_ **578**, 94–101 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32025018)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054213)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&journal=Nature&volume=578&pages=94-101&publication_year=2020&author=Alexandrov%2CLB) 
    
31.  Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. _Nature_ **534**, 47–54 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XntVCksrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27135926)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4910866)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Landscape%20of%20somatic%20mutations%20in%20560%20breast%20cancer%20whole-genome%20sequences&journal=Nature&volume=534&pages=47-54&publication_year=2016&author=Nik-Zainal%2CS) 
    
32.  Nik-Zainal, S. & Morganella, S. Mutational signatures in breast cancer: the problem at the DNA level. _Clin. Cancer Res._ **23**, 2617–2629 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28572256)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5458139)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mutational%20signatures%20in%20breast%20cancer%3A%20the%20problem%20at%20the%20DNA%20level&journal=Clin.%20Cancer%20Res.&volume=23&pages=2617-2629&publication_year=2017&author=Nik-Zainal%2CS&author=Morganella%2CS) 
    
33.  Rooney, M. S., Shukla, S. A., Wu, C. J., Getz, G. & Hacohen, N. Molecular and genetic properties of tumors associated with local immune cytolytic activity. _Cell_ **160**, 48–61 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2lsbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25594174)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4856474)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20and%20genetic%20properties%20of%20tumors%20associated%20with%20local%20immune%20cytolytic%20activity&journal=Cell&volume=160&pages=48-61&publication_year=2015&author=Rooney%2CMS&author=Shukla%2CSA&author=Wu%2CCJ&author=Getz%2CG&author=Hacohen%2CN) 
    
34.  Conte, B. et al. A 14-gene B-cell immune signature in early-stage triple-negative breast cancer (TNBC): a pooled analysis of seven studies. _EBioMedicine_ **102**, 105043 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXltlSgu7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38447275)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10924177)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%2014-gene%20B-cell%20immune%20signature%20in%20early-stage%20triple-negative%20breast%20cancer%20%28TNBC%29%3A%20a%20pooled%20analysis%20of%20seven%20studies&journal=EBioMedicine&volume=102&publication_year=2024&author=Conte%2CB) 
    
35.  Petitprez, F. et al. B cells are associated with survival and immunotherapy response in sarcoma. _Nature_ **577**, 556–560 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1Sit74%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31942077)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=B%20cells%20are%20associated%20with%20survival%20and%20immunotherapy%20response%20in%20sarcoma&journal=Nature&volume=577&pages=556-560&publication_year=2020&author=Petitprez%2CF) 
    
36.  Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. _Nature_ **490**, 61–70 (2012).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20molecular%20portraits%20of%20human%20breast%20tumours&journal=Nature&volume=490&pages=61-70&publication_year=2012) 
    
37.  Pereira, B. et al. The somatic mutation profiles of 2,433 breast cancers refines their genomic and transcriptomic landscapes. _Nat. Commun._ **7**, 11479 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xnslamtrc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27161491)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4866047)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20somatic%20mutation%20profiles%20of%202%2C433%20breast%20cancers%20refines%20their%20genomic%20and%20transcriptomic%20landscapes&journal=Nat.%20Commun.&volume=7&publication_year=2016&author=Pereira%2CB) 
    
38.  Bhojwani, D. et al. ETV6–RUNX1-positive childhood acute lymphoblastic leukemia: improved outcome with contemporary therapy. _Leukemia_ **26**, 265–270 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XitFGgtLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21869842)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ETV6%E2%80%93RUNX1-positive%20childhood%20acute%20lymphoblastic%20leukemia%3A%20improved%20outcome%20with%20contemporary%20therapy&journal=Leukemia&volume=26&pages=265-270&publication_year=2012&author=Bhojwani%2CD) 
    
39.  Lee, S. et al. Landscape analysis of adjacent gene rearrangements reveals _BCL2L14_–_ETV6_ gene fusions in more aggressive triple-negative breast cancer. _Proc. Natl Acad. Sci. USA_ **117**, 9912–9921 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtV2ltbrP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32321829)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7211963)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Landscape%20analysis%20of%20adjacent%20gene%20rearrangements%20reveals%20BCL2L14%E2%80%93ETV6%20gene%20fusions%20in%20more%20aggressive%20triple-negative%20breast%20cancer&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=117&pages=9912-9921&publication_year=2020&author=Lee%2CS) 
    
40.  Bishop, J. A. et al. Utility of mammaglobin immunohistochemistry as a proxy marker for the ETV6–NTRK3 translocation in the diagnosis of salivary mammary analogue secretory carcinoma. _Hum. Pathol._ **44**, 1982–1988 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpslGjt7Y%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23773480)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3779501)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Utility%20of%20mammaglobin%20immunohistochemistry%20as%20a%20proxy%20marker%20for%20the%20ETV6%E2%80%93NTRK3%20translocation%20in%20the%20diagnosis%20of%20salivary%20mammary%20analogue%20secretory%20carcinoma&journal=Hum.%20Pathol.&volume=44&pages=1982-1988&publication_year=2013&author=Bishop%2CJA) 
    
41.  Tognon, C. et al. Expression of the _ETV6_–_NTRK3_ gene fusion as a primary event in human secretory breast carcinoma. _Cancer Cell_ **2**, 367–376 (2002).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xptlalurw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12450792)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Expression%20of%20the%20ETV6%E2%80%93NTRK3%20gene%20fusion%20as%20a%20primary%20event%20in%20human%20secretory%20breast%20carcinoma&journal=Cancer%20Cell&volume=2&pages=367-376&publication_year=2002&author=Tognon%2CC) 
    
42.  Piscuoglio, S. et al. Genomic and transcriptomic heterogeneity in metaplastic carcinomas of the breast. _npj Breast Cancer_ **3**, 48 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29214215)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5711926)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20and%20transcriptomic%20heterogeneity%20in%20metaplastic%20carcinomas%20of%20the%20breast&journal=npj%20Breast%20Cancer&volume=3&publication_year=2017&author=Piscuoglio%2CS) 
    
43.  Robinson, D. R. et al. Functionally recurrent rearrangements of the MAST kinase and Notch gene families in breast cancer. _Nat. Med._ **17**, 1646–1651 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsV2gt7fE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22101766)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3233654)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functionally%20recurrent%20rearrangements%20of%20the%20MAST%20kinase%20and%20Notch%20gene%20families%20in%20breast%20cancer&journal=Nat.%20Med.&volume=17&pages=1646-1651&publication_year=2011&author=Robinson%2CDR) 
    
44.  Telli, M. L. et al. Homologous recombination deficiency (HRD) score predicts response to platinum-containing neoadjuvant chemotherapy in patients with triple-negative breast cancer. _Clin. Cancer Res._ **22**, 3764–3773 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1yhtbrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26957554)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6773427)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Homologous%20recombination%20deficiency%20%28HRD%29%20score%20predicts%20response%20to%20platinum-containing%20neoadjuvant%20chemotherapy%20in%20patients%20with%20triple-negative%20breast%20cancer&journal=Clin.%20Cancer%20Res.&volume=22&pages=3764-3773&publication_year=2016&author=Telli%2CML) 
    
45.  Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. _JCO Precis. Oncol._ **2017**, PO.17.00011 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28890946)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OncoKB%3A%20a%20precision%20oncology%20knowledge%20base&journal=JCO%20Precis.%20Oncol.&volume=2017&publication_year=2017&author=Chakravarty%2CD) 
    
46.  Huang, R. S. P. et al. Pan-cancer landscape of CD274 (PD-L1) copy number changes in 244,584 patient samples and the correlation with PD-L1 protein expression. _J. Immunother. Cancer_ **9**, e002680 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33972391)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8112409)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Pan-cancer%20landscape%20of%20CD274%20%28PD-L1%29%20copy%20number%20changes%20in%20244%2C584%20patient%20samples%20and%20the%20correlation%20with%20PD-L1%20protein%20expression&journal=J.%20Immunother.%20Cancer&volume=9&publication_year=2021&author=Huang%2CRSP) 
    
47.  Bandera, E. V. et al. Obesity, body fat distribution, and risk of breast cancer subtypes in African American women participating in the AMBER Consortium. _Breast Cancer Res. Treat._ **150**, 655–666 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsV2iuro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25809092)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4440799)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity%2C%20body%20fat%20distribution%2C%20and%20risk%20of%20breast%20cancer%20subtypes%20in%20African%20American%20women%20participating%20in%20the%20AMBER%20Consortium&journal=Breast%20Cancer%20Res.%20Treat.&volume=150&pages=655-666&publication_year=2015&author=Bandera%2CEV) 
    
48.  Dietze, E. C., Chavez, T. A. & Seewaldt, V. L. Obesity and triple-negative breast cancer: disparities, controversies, and biology. _Am. J. Pathol._ **188**, 280–290 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29128565)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5785535)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity%20and%20triple-negative%20breast%20cancer%3A%20disparities%2C%20controversies%2C%20and%20biology&journal=Am.%20J.%20Pathol.&volume=188&pages=280-290&publication_year=2018&author=Dietze%2CEC&author=Chavez%2CTA&author=Seewaldt%2CVL) 
    
49.  Du, X. L. & Li, Z. Incidence trends in triple-negative breast cancer among women in the United States from 2010 to 2019 by race/ethnicity, age and tumor stage. _Am. J. Cancer Res._ **13**, 678–691 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36895969)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9989620)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Incidence%20trends%20in%20triple-negative%20breast%20cancer%20among%20women%20in%20the%20United%20States%20from%202010%20to%202019%20by%20race%2Fethnicity%2C%20age%20and%20tumor%20stage&journal=Am.%20J.%20Cancer%20Res.&volume=13&pages=678-691&publication_year=2023&author=Du%2CXL&author=Li%2CZ) 
    
50.  Siddharth, S. & Sharma, D. Racial disparity and triple-negative breast cancer in African-American women: a multifaceted affair between obesity, biology, and socioeconomic determinants. _Cancers_ **10**, 514 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFKgs7jO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30558195)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6316530)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Racial%20disparity%20and%20triple-negative%20breast%20cancer%20in%20African-American%20women%3A%20a%20multifaceted%20affair%20between%20obesity%2C%20biology%2C%20and%20socioeconomic%20determinants&journal=Cancers&volume=10&publication_year=2018&author=Siddharth%2CS&author=Sharma%2CD) 
    
51.  Guo, Q. et al. The mutational signatures of formalin fixation on the human genome. _Nat. Commun._ **13**, 4487 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlGmu7%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36068219)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9448750)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20signatures%20of%20formalin%20fixation%20on%20the%20human%20genome&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Guo%2CQ) 
    
52.  Stecklein, S. R. et al. Dual prognostic classification of triple-negative breast cancer by DNA damage immune response and homologous recombination deficiency. _JCO Precis. Oncol._ **7**, e2300197 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37972336)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10681491)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dual%20prognostic%20classification%20of%20triple-negative%20breast%20cancer%20by%20DNA%20damage%20immune%20response%20and%20homologous%20recombination%20deficiency&journal=JCO%20Precis.%20Oncol.&volume=7&publication_year=2023&author=Stecklein%2CSR) 
    
53.  Killelea, B. K. et al. Racial differences in the use and outcome of neoadjuvant chemotherapy for breast cancer: results from the National Cancer Database. _J. Clin. Oncol._ **33**, 4267–4276 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26598753)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Racial%20differences%20in%20the%20use%20and%20outcome%20of%20neoadjuvant%20chemotherapy%20for%20breast%20cancer%3A%20results%20from%20the%20National%20Cancer%20Database&journal=J.%20Clin.%20Oncol.&volume=33&pages=4267-4276&publication_year=2015&author=Killelea%2CBK) 
    
54.  Roy, A. M. et al. Racial and ethnic disparity in preoperative chemosensitivity and survival in patients with early-stage breast cancer. _JAMA Netw. Open_ **6**, e2344517 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37991763)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10665980)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Racial%20and%20ethnic%20disparity%20in%20preoperative%20chemosensitivity%20and%20survival%20in%20patients%20with%20early-stage%20breast%20cancer&journal=JAMA%20Netw.%20Open&volume=6&publication_year=2023&author=Roy%2CAM) 
    
55.  Hossain, F. et al. Neighborhood social determinants of triple negative breast cancer. _Front. Public Health_ **7**, 18 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30834239)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6387917)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Neighborhood%20social%20determinants%20of%20triple%20negative%20breast%20cancer&journal=Front.%20Public%20Health&volume=7&publication_year=2019&author=Hossain%2CF) 
    
56.  Loree, J. M. et al. Disparity of race reporting and representation in clinical trials leading to cancer drug approvals from 2008 to 2018. _JAMA Oncol._ **5**, e191870 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31415071)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6696743)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Disparity%20of%20race%20reporting%20and%20representation%20in%20clinical%20trials%20leading%20to%20cancer%20drug%20approvals%20from%202008%20to%202018&journal=JAMA%20Oncol.&volume=5&publication_year=2019&author=Loree%2CJM) 
    
57.  Sztupinszki, Z. et al. Migrating the SNP array-based homologous recombination deficiency measures to next generation sequencing data of breast cancer. _npj Breast Cancer_ **4**, 16 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29978035)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6028448)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Migrating%20the%20SNP%20array-based%20homologous%20recombination%20deficiency%20measures%20to%20next%20generation%20sequencing%20data%20of%20breast%20cancer&journal=npj%20Breast%20Cancer&volume=4&publication_year=2018&author=Sztupinszki%2CZ) 
    
58.  Ondracek, R. P. et al. Results and lessons from dual extraction of DNA and RNA from formalin-fixed paraffin-embedded breast tumor tissues for a large cancer epidemiologic study. _BMC Genomics_ **23**, 614 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisVGmurrM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36008758)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9404650)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Results%20and%20lessons%20from%20dual%20extraction%20of%20DNA%20and%20RNA%20from%20formalin-fixed%20paraffin-embedded%20breast%20tumor%20tissues%20for%20a%20large%20cancer%20epidemiologic%20study&journal=BMC%20Genomics&volume=23&publication_year=2022&author=Ondracek%2CRP) 
    
59.  Salgado, R. et al. The evaluation of tumor-infiltrating lymphocytes (TILs) in breast cancer: recommendations by an International TILs Working Group 2014. _Ann. Oncol._ **26**, 259–271 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC2M7gsFOnug%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25214542)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20evaluation%20of%20tumor-infiltrating%20lymphocytes%20%28TILs%29%20in%20breast%20cancer%3A%20recommendations%20by%20an%20International%20TILs%20Working%20Group%202014&journal=Ann.%20Oncol.&volume=26&pages=259-271&publication_year=2015&author=Salgado%2CR) 
    
60.  Zhrebker, L. et al. Case report: whole exome sequencing of primary cardiac angiosarcoma highlights potential for targeted therapies. _BMC Cancer_ **17**, 17 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28056866)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5217318)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Case%20report%3A%20whole%20exome%20sequencing%20of%20primary%20cardiac%20angiosarcoma%20highlights%20potential%20for%20targeted%20therapies&journal=BMC%20Cancer&volume=17&publication_year=2017&author=Zhrebker%2CL) 
    
61.  Hu, Q., Hutson, A., Liu, S., Morgan, M. & Liu, Q. Bioconductor toolchain for reproducible bioinformatics pipelines using Rcwl and RcwlPipelines. _Bioinformatics_ **37**, 3351–3352 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1Cqtrk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33772584)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8504628)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bioconductor%20toolchain%20for%20reproducible%20bioinformatics%20pipelines%20using%20Rcwl%20and%20RcwlPipelines&journal=Bioinformatics&volume=37&pages=3351-3352&publication_year=2021&author=Hu%2CQ&author=Hutson%2CA&author=Liu%2CS&author=Morgan%2CM&author=Liu%2CQ) 
    
62.  Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. _Bioinformatics_ **25**, 1754–1760 (2009).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19451168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&journal=Bioinformatics&volume=25&pages=1754-1760&publication_year=2009&author=Li%2CH&author=Durbin%2CR) 
    
63.  Shen, R. & Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. _Nucleic Acids Res._ **44**, e131 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27270079)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5027494)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FACETS%3A%20allele-specific%20copy%20number%20and%20clonal%20heterogeneity%20analysis%20tool%20for%20high-throughput%20DNA%20sequencing&journal=Nucleic%20Acids%20Res.&volume=44&publication_year=2016&author=Shen%2CR&author=Seshan%2CVE) 
    
64.  Wei, L. et al. Intratumoral and intertumoral genomic heterogeneity of multifocal localized prostate cancer impacts molecular classifications and genomic prognosticators. _Eur. Urol._ **71**, 183–192 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27451135)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5906059)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intratumoral%20and%20intertumoral%20genomic%20heterogeneity%20of%20multifocal%20localized%20prostate%20cancer%20impacts%20molecular%20classifications%20and%20genomic%20prognosticators&journal=Eur.%20Urol.&volume=71&pages=183-192&publication_year=2016&author=Wei%2CL) 
    
65.  Saunders, C. T. et al. Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs. _Bioinformatics_ **28**, 1811–1817 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVSgtr%2FM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22581179)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Strelka%3A%20accurate%20somatic%20small-variant%20calling%20from%20sequenced%20tumor-normal%20sample%20pairs&journal=Bioinformatics&volume=28&pages=1811-1817&publication_year=2012&author=Saunders%2CCT) 
    
66.  Fan, Y. et al. MuSE: accounting for tumor heterogeneity using a sample-specific error model improves sensitivity and specificity in mutation calling from sequencing data. _Genome Biol._ **17**, 178 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27557938)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995747)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MuSE%3A%20accounting%20for%20tumor%20heterogeneity%20using%20a%20sample-specific%20error%20model%20improves%20sensitivity%20and%20specificity%20in%20mutation%20calling%20from%20sequencing%20data&journal=Genome%20Biol.&volume=17&publication_year=2016&author=Fan%2CY) 
    
67.  Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. _Nucleic Acids Res._ **44**, e108 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27060149)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914105)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=VarDict%3A%20a%20novel%20and%20versatile%20variant%20caller%20for%20next-generation%20sequencing%20in%20cancer%20research&journal=Nucleic%20Acids%20Res.&volume=44&publication_year=2016&author=Lai%2CZ) 
    
68.  Cibulskis, K. et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. _Nat. Biotechnol._ **31**, 213–219 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXjsFCntbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23396013)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3833702)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sensitive%20detection%20of%20somatic%20point%20mutations%20in%20impure%20and%20heterogeneous%20cancer%20samples&journal=Nat.%20Biotechnol.&volume=31&pages=213-219&publication_year=2013&author=Cibulskis%2CK) 
    
69.  Edmonson, M. N. et al. Bambino: a variant detector and alignment viewer for next-generation sequencing data in the SAM/BAM format. _Bioinformatics_ **27**, 865–866 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXjt1Sktbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21278191)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3051333)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bambino%3A%20a%20variant%20detector%20and%20alignment%20viewer%20for%20next-generation%20sequencing%20data%20in%20the%20SAM%2FBAM%20format&journal=Bioinformatics&volume=27&pages=865-866&publication_year=2011&author=Edmonson%2CMN) 
    
70.  Sherry, S. T. et al. dbSNP: the NCBI database of genetic variation. _Nucleic Acids Res._ **29**, 308–311 (2001).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXjtlWmtb0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11125122)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC29783)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=dbSNP%3A%20the%20NCBI%20database%20of%20genetic%20variation&journal=Nucleic%20Acids%20Res.&volume=29&pages=308-311&publication_year=2001&author=Sherry%2CST) 
    
71.  Genomes Project, C. et al. An integrated map of genetic variation from 1,092 human genomes. _Nature_ **491**, 56–65 (2012).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20map%20of%20genetic%20variation%20from%201%2C092%20human%20genomes&journal=Nature&volume=491&pages=56-65&publication_year=2012&author=Genomes%20Project%2CC) 
    
72.  Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. _Nature_ **581**, 434–443 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32461654)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&journal=Nature&volume=581&pages=434-443&publication_year=2020&author=Karczewski%2CKJ) 
    
73.  Wang, K., Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. _Nucleic Acids Res._ **38**, e164 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20601685)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2938201)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ANNOVAR%3A%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&journal=Nucleic%20Acids%20Res.&volume=38&publication_year=2010&author=Wang%2CK&author=Li%2CM&author=Hakonarson%2CH) 
    
74.  McLaren, W. et al. The Ensembl variant effect predictor. _Genome Biol._ **17**, 122 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27268795)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4893825)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Ensembl%20variant%20effect%20predictor&journal=Genome%20Biol.&volume=17&publication_year=2016&author=McLaren%2CW) 
    
75.  Wei, L. et al. MAC: identifying and correcting annotation for multi-nucleotide variations. _BMC Genomics_ **16**, 569 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26231518)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4521406)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MAC%3A%20identifying%20and%20correcting%20annotation%20for%20multi-nucleotide%20variations&journal=BMC%20Genomics&volume=16&publication_year=2015&author=Wei%2CL) 
    
76.  Wei, L. et al. Accurate quantification of residual cancer cells in pelvic washing reveals association with cancer recurrence following robot-assisted radical cystectomy. _J. Urol._ **201**, 1105–1114 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30730413)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6784327)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20quantification%20of%20residual%20cancer%20cells%20in%20pelvic%20washing%20reveals%20association%20with%20cancer%20recurrence%20following%20robot-assisted%20radical%20cystectomy&journal=J.%20Urol.&volume=201&pages=1105-1114&publication_year=2019&author=Wei%2CL) 
    
77.  Wei, L. et al. Ultradeep sequencing differentiates patterns of skin clonal mutations associated with sun-exposure status and skin cancer burden. _Sci. Adv._ **7**, eabd7703 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33523857)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7775785)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ultradeep%20sequencing%20differentiates%20patterns%20of%20skin%20clonal%20mutations%20associated%20with%20sun-exposure%20status%20and%20skin%20cancer%20burden&journal=Sci.%20Adv.&volume=7&publication_year=2021&author=Wei%2CL) 
    
78.  Hagiwara, K., Edmonson, M. N., Wheeler, D. A. & Zhang, J. IndelPost: harmonizing ambiguities in simple and complex indel alignments. _Bioinformatics_ **38**, 549–551 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XmtlOqu7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34431982)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=IndelPost%3A%20harmonizing%20ambiguities%20in%20simple%20and%20complex%20indel%20alignments&journal=Bioinformatics&volume=38&pages=549-551&publication_year=2022&author=Hagiwara%2CK&author=Edmonson%2CMN&author=Wheeler%2CDA&author=Zhang%2CJ) 
    
79.  Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. _Genome Biol._ **12**, R41 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21527027)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218867)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GISTIC2.0%20facilitates%20sensitive%20and%20confident%20localization%20of%20the%20targets%20of%20focal%20somatic%20copy-number%20alteration%20in%20human%20cancers&journal=Genome%20Biol.&volume=12&publication_year=2011&author=Mermel%2CCH) 
    
80.  Huang, H. et al. Functional evaluation and clinical classification of BRCA2 variants. _Nature_ **638**, 528–537 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXhtVWrtL8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39779857)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11821525)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20evaluation%20and%20clinical%20classification%20of%20BRCA2%20variants&journal=Nature&volume=638&pages=528-537&publication_year=2025&author=Huang%2CH) 
    
81.  McCarthy-Leo, C. E. et al. Comprehensive analysis of the functional impact of single nucleotide variants of human CHEK2. _PLoS Genet._ **20**, e1011375 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitVenur3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39146382)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11349238)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20analysis%20of%20the%20functional%20impact%20of%20single%20nucleotide%20variants%20of%20human%20CHEK2&journal=PLoS%20Genet.&volume=20&publication_year=2024&author=McCarthy-Leo%2CCE) 
    
82.  Funk, J. S. et al. Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations. _Nat. Genet._ **57**, 140–153 (2025).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXotVCmsg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39774325)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11735402)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20CRISPR%20mutagenesis%20characterizes%20the%20functional%20diversity%20of%20TP53%20mutations&journal=Nat.%20Genet.&volume=57&pages=140-153&publication_year=2025&author=Funk%2CJS) 
    
83.  Liberzon, A. et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. _Cell Syst._ **1**, 417–425 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFaltLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26771021)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Molecular%20Signatures%20Database%20%28MSigDB%29%20hallmark%20gene%20set%20collection&journal=Cell%20Syst.&volume=1&pages=417-425&publication_year=2015&author=Liberzon%2CA) 
    
84.  Islam, S. M. A. et al. Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor. _Cell Genom._ **2**, 100179 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisVOhsb%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36388765)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9646490)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Uncovering%20novel%20mutational%20signatures%20by%20de%20novo%20extraction%20with%20SigProfilerExtractor&journal=Cell%20Genom.&volume=2&publication_year=2022&author=Islam%2CSMA) 
    
85.  Cabreros, I. & Storey, J. D. A likelihood-free estimator of population structure bridging admixture models and principal components analysis. _Genetics_ **212**, 1009–1029 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31028112)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707457)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20likelihood-free%20estimator%20of%20population%20structure%20bridging%20admixture%20models%20and%20principal%20components%20analysis&journal=Genetics&volume=212&pages=1009-1029&publication_year=2019&author=Cabreros%2CI&author=Storey%2CJD) 
    
86.  Andrew, S. FastQC: a quality control tool for high throughput sequence data. _Babraham Bioinform._ [www.bioinformatics.babraham.ac.uk/projects/fastqc](http://www.bioinformatics.babraham.ac.uk/projects/fastqc) (2010).
    
87.  Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. _EMBnet J._ **17**, 10–12 (2011).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&journal=EMBnet%20J.&volume=17&pages=10-12&publication_year=2011&author=Martin%2CM) 
    
88.  Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. _Bioinformatics_ **29**, 15–21 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23104886)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&journal=Bioinformatics&volume=29&pages=15-21&publication_year=2013&author=Dobin%2CA) 
    
89.  Wang, L., Wang, S. & Li, W. RSeQC: quality control of RNA-seq experiments. _Bioinformatics_ **28**, 2184–2185 (2012).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtF2rsrnJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22743226)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RSeQC%3A%20quality%20control%20of%20RNA-seq%20experiments&journal=Bioinformatics&volume=28&pages=2184-2185&publication_year=2012&author=Wang%2CL&author=Wang%2CS&author=Li%2CW) 
    
90.  Feng, H., Zhang, X. & Zhang, C. mRIN for direct assessment of genome-wide and gene-specific mRNA integrity from large-scale RNA-sequencing data. _Nat. Commun._ **6**, 7816 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsVCrtrnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26234653)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=mRIN%20for%20direct%20assessment%20of%20genome-wide%20and%20gene-specific%20mRNA%20integrity%20from%20large-scale%20RNA-sequencing%20data&journal=Nat.%20Commun.&volume=6&publication_year=2015&author=Feng%2CH&author=Zhang%2CX&author=Zhang%2CC) 
    
91.  Li, B. & Dewey, C. N. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. _BMC Bioinformatics_ **12**, 323 (2011).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtFajtLvM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21816040)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163565)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&journal=BMC%20Bioinformatics&volume=12&publication_year=2011&author=Li%2CB&author=Dewey%2CCN) 
    
92.  Zhang, Y., Parmigiani, G. & Johnson, W. E. ComBat-seq: batch effect adjustment for RNA-seq count data. _NAR Genom. Bioinform_ **2**, lqaa078 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33015620)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7518324)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ComBat-seq%3A%20batch%20effect%20adjustment%20for%20RNA-seq%20count%20data&journal=NAR%20Genom.%20Bioinform&volume=2&publication_year=2020&author=Zhang%2CY&author=Parmigiani%2CG&author=Johnson%2CWE) 
    
93.  Uhrig, S. et al. Accurate and efficient detection of gene fusions from RNA sequencing data. _Genome Res._ **31**, 448–460 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXit1ajsb7E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33441414)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7919457)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20and%20efficient%20detection%20of%20gene%20fusions%20from%20RNA%20sequencing%20data&journal=Genome%20Res.&volume=31&pages=448-460&publication_year=2021&author=Uhrig%2CS) 
    
94.  Haas, B. J. et al. STAR-Fusion: fast and accurate fusion transcript detection from RNA-seq. Preprint at _bioRxiv_ [https://doi.org/10.1101/120295](https://doi.org/10.1101/120295) (2017).
    
95.  Suehnholz, S. P. et al. Quantifying the expanding landscape of clinical actionability for patients with cancer. _Cancer Discov._ **14**, 49–65 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsVOlsLvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37849038)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantifying%20the%20expanding%20landscape%20of%20clinical%20actionability%20for%20patients%20with%20cancer&journal=Cancer%20Discov.&volume=14&pages=49-65&publication_year=2024&author=Suehnholz%2CSP) 
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02322-y?format=refman&flavour=references)

Acknowledgements
----------------

This research was supported in part by the National Institutes of Health (grants R01 CA228156 to S.Y., J.R.P., W.Z. and J.D.C.; R01 CA100598 to C.B.A.; R01 CA185623 to C.C.H. and E.V.B.; R01 CA164974 to J.R.P.; UM1 CA164974 to J.R.P.; R01 CA092447 to W.Z.; U01 CA202979 to W.Z.; R01 CA100374 to W.Z. and U54 CA91405 to W.Z.) and Komen Foundation (SAC220228 to J.R.P.). Sample preparation, RNA sequencing, targeted sequencing assays and bioinformatic analysis at Roswell Park Comprehensive Cancer Center were conducted at the Biobanking and Laboratory Service, Genomics Shared Resource and Bioinformatics Shared Resource, which are supported in part by the Roswell Park Cancer Center Support Grant (P30 CA016056 to C.S. Johnson). Data collection and sample preparation work for Vanderbilt samples were performed by the Survey and Biospecimen Shared Resource, supported in part by the Vanderbilt-Ingram Cancer Center (P30 CA068485 to B.H. Park). High-performance computing data analyses were conducted using the Center for Computational Research at the University at Buffalo. C.B.A. received support from the Breast Cancer Research Foundation. J.R.P. and M.E.B. received support from the Karin Grunebaum Cancer Research Foundation. The content is solely the responsibility of the authors and does not necessarily represent the official views of the funding agents. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.

Author information
------------------

Author notes

1.  These authors contributed equally: Song Yao, Lei Wei, Qiang Hu, Song Liu.
    

### Authors and Affiliations

1.  Department of Cancer Prevention and Control, Roswell Park Comprehensive Cancer Center, Buffalo, NY, USA
    
    Song Yao, Peter N. Fiorica, Warren Davis, Jianhong Chen, Rochelle P. Ondracek, Chi-Chen Hong & Christine B. Ambrosone
    
2.  Department of Biostatistics and Bioinformatics, Roswell Park Comprehensive Cancer Center, Buffalo, NY, USA
    
    Lei Wei, Qiang Hu, Song Liu, Mark Long, Mitsuko Murakami, Jianmin Wang & Qianqian Zhu
    
3.  Department of Translational Genomics, Keck School of Medicine of USC, Los Angeles, CA, USA
    
    Zarko Manojlovic
    
4.  Slone Epidemiology Center, Boston University, Boston, MA, USA
    
    Gary R. Zirpoli, Mollie E. Barnard & Julie R. Palmer
    
5.  Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University Medical Center, Nashville, TN, USA
    
    Qiuyin Cai, Jirong Long, Jie Ping & Wei Zheng
    
6.  Comprehensive Cancer Center, City of Hope, Duarte, CA, USA
    
    Yuxin Jin, David W. Craig & John D. Carpten
    
7.  Department of Pathology, Roswell Park Comprehensive Cancer Center, Buffalo, NY, USA
    
    Thaer Khoury
    
8.  Department of Medicine, Roswell Park Comprehensive Cancer Center, Buffalo, NY, USA
    
    Shipra Gandhi
    
9.  Department of Surgical Oncology, Roswell Park Comprehensive Cancer Center, Buffalo, NY, USA
    
    Kazuaki Takabe
    
10.  Department of Medical Oncology, Boston Medical Center, Boston, MA, USA
    
    Naomi Ko
    
11.  Department of Family and Community Medicine, Meharry Medical College, Nashville, TN, USA
    
    Maureen Sanderson
    
12.  Section of Cancer Epidemiology and Health Outcomes, Rutgers Cancer Institute of New Jersey, New Brunswick, NJ, USA
    
    Elisa V. Bandera
    

Authors

1.  Song Yao
    
    [View author publications](https://www.nature.com/search?author=Song%20Yao)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Song%20Yao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Song%20Yao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Lei Wei
    
    [View author publications](https://www.nature.com/search?author=Lei%20Wei)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Lei%20Wei) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Lei%20Wei%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Qiang Hu
    
    [View author publications](https://www.nature.com/search?author=Qiang%20Hu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qiang%20Hu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qiang%20Hu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Song Liu
    
    [View author publications](https://www.nature.com/search?author=Song%20Liu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Song%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Song%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Zarko Manojlovic
    
    [View author publications](https://www.nature.com/search?author=Zarko%20Manojlovic)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zarko%20Manojlovic) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zarko%20Manojlovic%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Peter N. Fiorica
    
    [View author publications](https://www.nature.com/search?author=Peter%20N.%20Fiorica)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peter%20N.%20Fiorica) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peter%20N.%20Fiorica%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Mark Long
    
    [View author publications](https://www.nature.com/search?author=Mark%20Long)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mark%20Long) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mark%20Long%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Gary R. Zirpoli
    
    [View author publications](https://www.nature.com/search?author=Gary%20R.%20Zirpoli)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gary%20R.%20Zirpoli) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gary%20R.%20Zirpoli%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Qiuyin Cai
    
    [View author publications](https://www.nature.com/search?author=Qiuyin%20Cai)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qiuyin%20Cai) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qiuyin%20Cai%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Jirong Long
    
    [View author publications](https://www.nature.com/search?author=Jirong%20Long)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jirong%20Long) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jirong%20Long%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Jie Ping
    
    [View author publications](https://www.nature.com/search?author=Jie%20Ping)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jie%20Ping) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jie%20Ping%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Mollie E. Barnard
    
    [View author publications](https://www.nature.com/search?author=Mollie%20E.%20Barnard)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mollie%20E.%20Barnard) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mollie%20E.%20Barnard%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Yuxin Jin
    
    [View author publications](https://www.nature.com/search?author=Yuxin%20Jin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuxin%20Jin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuxin%20Jin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Mitsuko Murakami
    
    [View author publications](https://www.nature.com/search?author=Mitsuko%20Murakami)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mitsuko%20Murakami) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mitsuko%20Murakami%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Jianmin Wang
    
    [View author publications](https://www.nature.com/search?author=Jianmin%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jianmin%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jianmin%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Qianqian Zhu
    
    [View author publications](https://www.nature.com/search?author=Qianqian%20Zhu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Qianqian%20Zhu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Qianqian%20Zhu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Warren Davis
    
    [View author publications](https://www.nature.com/search?author=Warren%20Davis)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Warren%20Davis) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Warren%20Davis%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Jianhong Chen
    
    [View author publications](https://www.nature.com/search?author=Jianhong%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jianhong%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jianhong%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Rochelle P. Ondracek
    
    [View author publications](https://www.nature.com/search?author=Rochelle%20P.%20Ondracek)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rochelle%20P.%20Ondracek) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rochelle%20P.%20Ondracek%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Thaer Khoury
    
    [View author publications](https://www.nature.com/search?author=Thaer%20Khoury)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Thaer%20Khoury) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Thaer%20Khoury%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Shipra Gandhi
    
    [View author publications](https://www.nature.com/search?author=Shipra%20Gandhi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shipra%20Gandhi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shipra%20Gandhi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Kazuaki Takabe
    
    [View author publications](https://www.nature.com/search?author=Kazuaki%20Takabe)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kazuaki%20Takabe) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kazuaki%20Takabe%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Naomi Ko
    
    [View author publications](https://www.nature.com/search?author=Naomi%20Ko)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Naomi%20Ko) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Naomi%20Ko%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Maureen Sanderson
    
    [View author publications](https://www.nature.com/search?author=Maureen%20Sanderson)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maureen%20Sanderson) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maureen%20Sanderson%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Chi-Chen Hong
    
    [View author publications](https://www.nature.com/search?author=Chi-Chen%20Hong)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chi-Chen%20Hong) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chi-Chen%20Hong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Elisa V. Bandera
    
    [View author publications](https://www.nature.com/search?author=Elisa%20V.%20Bandera)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Elisa%20V.%20Bandera) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Elisa%20V.%20Bandera%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  David W. Craig
    
    [View author publications](https://www.nature.com/search?author=David%20W.%20Craig)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=David%20W.%20Craig) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22David%20W.%20Craig%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Christine B. Ambrosone
    
    [View author publications](https://www.nature.com/search?author=Christine%20B.%20Ambrosone)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Christine%20B.%20Ambrosone) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Christine%20B.%20Ambrosone%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Julie R. Palmer
    
    [View author publications](https://www.nature.com/search?author=Julie%20R.%20Palmer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Julie%20R.%20Palmer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Julie%20R.%20Palmer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  Wei Zheng
    
    [View author publications](https://www.nature.com/search?author=Wei%20Zheng)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wei%20Zheng) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wei%20Zheng%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  John D. Carpten
    
    [View author publications](https://www.nature.com/search?author=John%20D.%20Carpten)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=John%20D.%20Carpten) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22John%20D.%20Carpten%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

S.Y., S.L., C.B.A., J.R.P., W.Z. and J.D.C. conceived of and designed the study. M.S., C.C.H., E.V.B., C.B.A., J.R.P. and W.Z. recruited the study participants and collected the data and specimens. S.Y., L.W., Q.H., D.W.C., Z.M., G.R.Z., J.P., Q.C., J.L., M.E.B., Y.J., W.D., M.M., J.C., R.P.O. and T.K. managed sample and data preparation, and carried out quality control. S.Y., L.W., Q.H., Z.M., P.N.F., M.L., M.M., J.W. and Q.Z. analyzed the data. S.Y., L.W., Q.H., Z.M., T.K., S.G., K.T., N.K., S.L., C.B.A., J.R.P., W.Z. and J.D.C. interpreted the findings. S.Y., L.W., P.N.F., S.L., C.B.A., J.R.P., W.Z. and J.D.C. drafted or substantively revised the paper. S.Y., J.R.P., W.Z. and J.D.C. supervised the project.

### Corresponding author

Correspondence to [Song Yao](mailto:song.yao@roswellpark.org).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks the anonymous reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02322-y#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Impact of _TP53_ intronic mutations on RNA transcript splicing.](https://www.nature.com/articles/s41588-025-02322-y/figures/7)

Four triple-negative breast cancer (TNBC) samples carried only an intronic mutation in _TP53_ with matched tumor transcriptomic data available. To explore the functional impact of the mutations, the splicing patterns of the transcripts of these samples are examined, with one sample harboring no _TP53_ mutation as the wild-type (WT) reference. DNA mutations are indicated by a red ‘X’ mark on DNA sequence at the top, and wild-type (WT) splicing patterns indicated by blue color and aberrant splicing patterns indicated by red color on the RNA sequence at the bottom.

### [Extended Data Fig. 2 Confirmation of TP53 mutations in transcriptomic data and ultra-deep sequencing.](https://www.nature.com/articles/s41588-025-02322-y/figures/8)

Two orthogonal approaches were used to validate _TP53_ mutations identified from whole-exome sequencing analysis—first, using tumor transcriptomic data available from 260 matched RNA samples, and second, using targeted amplicon sequencing (TAS) performed on 338 tumors with DNA available. The numbers shown in the bar indicate the percentage of samples with mutations replicated (green) or not replicated (orange). For transcriptomic data, the results are shown for all samples and by the sequencing depth of TP53 region. For TAS data, the results are shown at both mutation level and tumor level.

### [Extended Data Fig. 3 Minor allele frequency of germline mutations in known breast cancer genes in B-CAUSE TNBC patients and reference populations.](https://www.nature.com/articles/s41588-025-02322-y/figures/9)

The minor allele frequency in B-CAUSE data (x-axis) and AFR reference data (y_\-_axis) are log-transformed and negated and germline variants are color coded according to ClinVar classification. P-values from two-sided Fisher’s exact test are indicated by the dot diameter.

### [Extended Data Fig. 4 Forest plots of overall survival with mutational signatures and SBS-based subtypes.](https://www.nature.com/articles/s41588-025-02322-y/figures/10)

The low, medium and high levels of SBS1&5 (aging) and SBS3 (HRD) were determined by tertiles of the signatures. The low levels were used as the reference group (not shown in the figures). In the analysis of the combined SBS1&5 and SBS3 group, the subtype 1 (low aging and high HRD) was used as the reference group (not shown in the figure). For subtype 4, the model did not converge in BWHS and the results are not shown. The dots indicate the point estimates of hazards ratio (HR) with the bars indicate the lower and upper 95% confidence intervals (CI). The squares are pooled HR estimates from meta-analyses across studies. Sample size for each study was: WCHS: n = 146; VUMC: n = 84; BWHS: n = 72.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_MOESM1_ESM.pdf)

Supplementary Note and Supplementary Figs. 1–12.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_MOESM3_ESM.pdf)

### [Supplementary Tables 1–11](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02322-y/MediaObjects/41588_2025_2322_MOESM4_ESM.xlsx)

Supplementary Tables 1–11.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Mutational%20landscape%20of%20triple-negative%20breast%20cancer%20in%20African%20American%20women&author=Song%20Yao%20et%20al&contentID=10.1038%2Fs41588-025-02322-y&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-26&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02322-y)

### Cite this article

Yao, S., Wei, L., Hu, Q. _et al._ Mutational landscape of triple-negative breast cancer in African American women. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02322-y

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02322-y?format=refman&flavour=citation)

-   Received: 31 January 2025
    
-   Accepted: 30 July 2025
    
-   Published: 26 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02322-y
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02322-y.pdf)
