> id: https://www.nature.com/articles/s41588-025-02246-7

> link: https://www.nature.com/articles/s41588-025-02246-7

> title: Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize

# Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize
_Published on Mon, 11 Aug 2025 00:00:00 GMT_

Abstract
--------

Comprehensive maps of functional variation at transcription factor (TF) binding sites (_cis_\-elements) are crucial for elucidating how genotype shapes phenotype. Here, we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped over 200,000 variants, genetic, epigenetic, or both (termed binding quantitative trait loci (bQTL)), linked to _cis_\-element occupancy. Three lines of evidence support the functional significance of bQTL: (1) coincidence with causative loci that regulate traits, including _vgt1_, _ZmTRE1_ and the MITE transposon near _ZmNAC111_ under drought; (2) bQTL allelic bias is shared between inbred parents and matches chromatin immunoprecipitation sequencing results; and (3) partitioning genetic variation across genomic regions demonstrates that bQTL capture the majority of heritable trait variation across ~72% of 143 phenotypes. Our study provides an auspicious approach to make functional _cis_\-variation accessible at scale for genetic studies and targeted engineering of complex traits.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41477-025-02007-8/MediaObjects/41477_2025_2007_Fig1_HTML.png)

### [Transcription factor binding divergence drives transcriptional and phenotypic variation in maize](https://www.nature.com/articles/s41477-025-02007-8?fromPaywallRec=false)

Article 12 June 2025

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41437-022-00541-1/MediaObjects/41437_2022_541_Fig1_HTML.png)

### [Assessment of two statistical approaches for variance genome-wide association studies in plants](https://www.nature.com/articles/s41437-022-00541-1?fromPaywallRec=false)

Article 10 May 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41437-022-00503-7/MediaObjects/41437_2022_503_Fig1_HTML.png)

### [Trade-offs in the genetic control of functional and nutritional quality traits in UK winter wheat](https://www.nature.com/articles/s41437-022-00503-7?fromPaywallRec=false)

Article Open access 07 April 2022

Main
----

Over the past two decades, genome-wide association studies (GWAS) have transformed our understanding of the inheritance of many complex traits in important crops such as maize. Several studies have estimated that non-coding variation accounts for about 50% of the additive genetic variance underlying phenotypic diversity in plants[1](https://www.nature.com/articles/s41588-025-02246-7#ref-CR1 "Chia, J.-M. et al. Maize HapMap2 identifies extant variation from a genome in flux. Nat. Genet. 44, 803–807 (2012)."),[2](https://www.nature.com/articles/s41588-025-02246-7#ref-CR2 "Rodgers-Melnick, E., Vera, D. L., Bass, H. W. & Buckler, E. S. Open chromatin reveals the functional maize genome. Proc. Natl Acad. Sci. USA 113, E3177–E3184 (2016)."),[3](https://www.nature.com/articles/s41588-025-02246-7#ref-CR3 "Lorant, A., Ross-Ibarra, J. & Tenaillon, M. Genomics of long- and short-term adaptation in maize and teosintes. in Statistical Population Genomics (ed. Dutheil, J. Y.) 289–311 (Springer, 2020). 
https://doi.org/10.1007/978-1-0716-0199-0_12
"),[4](https://www.nature.com/articles/s41588-025-02246-7#ref-CR4 "Song, B. et al. Conserved noncoding sequences provide insights into regulatory sequence and loss of gene expression in maize. Genome Res. 7, 1245–1257 (2021)."). Although identification of functional non-coding variants is advancing with the development of new genomics technologies[5](https://www.nature.com/articles/s41588-025-02246-7#ref-CR5 "Marand, A. P., Eveland, A. L., Kaufmann, K. & Springer, N. M. Cis-regulatory elements in plant development, adaptation, and evolution. Annu. Rev. Plant Biol. 74, 111–137 (2023)."), it remains challenging to discern functional variants that impact _cis_\-elements efficiently and at cistrome (defined as the genome-wide set of _cis_\-acting regulatory loci) scale. Knowing which loci to target has become one of the obstacles for trait improvement by targeted genome editing[5](https://www.nature.com/articles/s41588-025-02246-7#ref-CR5 "Marand, A. P., Eveland, A. L., Kaufmann, K. & Springer, N. M. Cis-regulatory elements in plant development, adaptation, and evolution. Annu. Rev. Plant Biol. 74, 111–137 (2023)."),[6](https://www.nature.com/articles/s41588-025-02246-7#ref-CR6 "Wolter, F. & Puchta, H. Application of CRISPR/Cas to understand cis- and trans-regulatory elements in plants. in Plant Transcription Factors: Methods and Protocols (ed. Yamaguchi, N.) 23–40 (Springer, 2018). 
https://doi.org/10.1007/978-1-4939-8657-6_2
"),[7](https://www.nature.com/articles/s41588-025-02246-7#ref-CR7 "Sharon, E. et al. Functional genetic variants revealed by massively parallel precise genome editing. Cell 175, 544–557.e16 (2018)."). Scalable methods to construct comprehensive _cis_\-element maps are essential to understand complex transcriptional networks that underlie development, growth and disease. The potential of _cis_\-element maps has been demonstrated by the ENCODE projects that exist for many eukaryotes, including humans. However, genome-wide, high-resolution maps of functional variants are currently lacking in plants[8](https://www.nature.com/articles/s41588-025-02246-7#ref-CR8 "Lane, A. K., Niederhuth, C. E., Ji, L. & Schmitz, R. J. pENCODE: a plant encyclopedia of DNA elements. Annu. Rev. Genet. 48, 49–70 (2014)."). Despite many successes, GWAS generally suffer from insufficient resolution, which limits the identification of individual causal single-nucleotide polymorphisms (SNPs) or insertions or deletions (INDELs) and cannot provide independent molecular information on the potential function of variants, requiring laborious follow-up analyses of numerous individual loci[7](https://www.nature.com/articles/s41588-025-02246-7#ref-CR7 "Sharon, E. et al. Functional genetic variants revealed by massively parallel precise genome editing. Cell 175, 544–557.e16 (2018).").

An alternative approach to identify functional polymorphisms would be to annotate non-coding variants within a GWAS region based on their association with TF binding. This approach has considerable potential, as TF activity has an important role in the regulation of genes, and thereby traits, and the affinity of TF binding is mostly determined by specific local sequences (_cis_\-elements)[9](https://www.nature.com/articles/s41588-025-02246-7#ref-CR9 "White, M. A., Myers, C. A., Corbo, J. C. & Cohen, B. A. Massively parallel in vivo enhancer assay reveals that highly local features determine the cis-regulatory function of ChIP-seq peaks. Proc. Natl Acad. Sci. USA 110, 11952–11957 (2013)."),[10](https://www.nature.com/articles/s41588-025-02246-7#ref-CR10 "Levo, M. & Segal, E. In pursuit of design principles of regulatory sequences. Nat. Rev. Genet. 15, 453–468 (2014)."). Identifying _cis_\-elements for individual TFs through approaches such as chromatin immunoprecipitation with sequencing (ChIP–seq) is time-consuming, not strictly quantitative, limited in scope and often provides relatively low resolution of functional regions. By contrast, MNase-defined cistrome occupancy analysis (MOA-seq) identifies putative TF binding sites globally, in a single experiment with relatively high resolution and yields footprint regions typically of <100 bp (ref. [11](https://www.nature.com/articles/s41588-025-02246-7#ref-CR11 "Savadel, S. D. et al. The native cistrome and sequence motif families of the maize ear. PLoS Genet. 17, e1009689 (2021).")). In maize, MOA-seq identified ∼100,000 TF-occupied loci, including about 70% of the sequences (bp overlap) identified in more than 100 ChIP–seq experiments[11](https://www.nature.com/articles/s41588-025-02246-7#ref-CR11 "Savadel, S. D. et al. The native cistrome and sequence motif families of the maize ear. PLoS Genet. 17, e1009689 (2021)."),[12](https://www.nature.com/articles/s41588-025-02246-7#ref-CR12 "Tu, X. et al. Reconstructing the maize leaf regulatory network using ChIP-seq data of 104 transcription factors. Nat. Commun. 11, 5089 (2020)."). Notably, many of the MOA footprint regions were previously uncharacterized, with only 35% identified in previous assay for transposase-accessible chromatin sequencing (ATAC-seq) data; by contrast, MOA-seq identified 76% of previous ATAC-seq peaks[11](https://www.nature.com/articles/s41588-025-02246-7#ref-CR11 "Savadel, S. D. et al. The native cistrome and sequence motif families of the maize ear. PLoS Genet. 17, e1009689 (2021)."). Similarly, an analysis of small MNase-defined fragments from _Arabidopsis_ seedlings revealed more than 15,000 accessible chromatin regions missed by ATAC-seq or DNase-seq[13](https://www.nature.com/articles/s41588-025-02246-7#ref-CR13 "Zhao, H. et al. Genome-wide MNase hypersensitivity assay unveils distinct classes of open chromatin associated with H3K27me3 and DNA methylation in Arabidopsis thaliana. Genome Biol. 21, 24 (2020).").

Here, we quantified haplotype-specific TF footprints across the maize pan-genome with MOA-seq, using F1 hybrids that share a common reference to minimize biological, technical and _trans_\-effect variation between the haplotypes. We defined a maize leaf pan-cistrome and identified ~210,000 variants that were genetic, epigenetic, or both linked to haplotype-specific variation in MOA coverage at candidate _cis_\-element loci, which we term bQTL. The bQTL explained the majority of heritable trait variation in >70% of the tested traits in the nested association mapping (NAM) panel. Haplotype-specific TF footprints coincided with causative loci known to affect leaf angle, branching and flowering time traits, and identified _ZmTINY_ (Zm00001eb120590) and more than 3,500 drought-response putative _cis_\-regulatory regions as candidate loci for future smart breeding.

Results
-------

### Quantification of functional _cis_\-variation

To focus on genetic differences affecting TF binding in _cis_, we quantified TF footprints (defined as the area significantly covered by MOA-seq reads) specific to each haplotype in F1 hybrids with a shared reference parent (B73) (Fig. [1a](https://www.nature.com/articles/s41588-025-02246-7#Fig1)). We applied MOA-seq to nuclei of the inbred lines B73 (ref. [14](https://www.nature.com/articles/s41588-025-02246-7#ref-CR14 "Manosalva Pérez, N. et al. MINI-AC: inference of plant gene regulatory networks using bulk or single-cell accessible chromatin profiles. Plant J. 117, 280–301 (2024).")) and Mo17—founders of key maize breeding populations whose hybrid has been extensively studied[15](https://www.nature.com/articles/s41588-025-02246-7#ref-CR15 "Springer, N. M. & Stupar, R. M. Allele-specific expression patterns reveal biases and embryo-specific parent-of-origin effects in hybrid maize. Plant Cell 19, 2391–2402 (2007)."),[16](https://www.nature.com/articles/s41588-025-02246-7#ref-CR16 "Pressoir, G. et al. Natural variation in maize architecture is mediated by allelic differences at the PINOID co-ortholog barren inflorescence2. Plant J. 58, 618–628 (2009)."),[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).")—and their F1 hybrids. MOA footprints were determined by mapping sequencing reads to a concatenated hybrid genome and retaining reads that mapped uniquely (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02246-7#MOESM5); for some analyses we used reads mapping equally well to two locations; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)). We detected 327,029 MOA footprints or peaks (false discovery rate (FDR) of 5%) with strong correlation across biological replicates (Pearson’s correlation coefficient > 0.95; Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). A total of 53,220 genes in the F1, representing 67.9% of B73 and Mo17 annotated genes (5 kb upstream and 1 kb downstream; Supplementary Table [2](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)) were flanked by at least one MOA footprint. Furthermore, the MOA footprints harbored 325,933 SNPs, which we termed MOA polymorphisms (MPs). Among all MPs, we identified 48,505 with an allelic ratio that significantly deviated from the expected 1:1 in F1s, which we termed allele-specific MPs (AMPs; binomial test with 1% FDR, validated with whole-genome sequencing (WGS) controls; Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)).

**Fig. 1: Quantitative _cis_\-element occupancy analysis in F1 hybrids.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02246-7/figures/1)

**a**, Haplotype-specific MOA flowchart: 1) Nuclei purified from diverse nested (B73 common mother) F1s are analyzed by MOA-seq, producing small, non-nucleosomal, protein–DNA interaction footprints. 2) SNPs in MOA peaks (MPs) allow the identification, quantification and, in a population, association of variants coupled to occupancy of putative _cis_\-elements. Allele-specific MOA footprints can be compared between treatments; for example, well-watered versus drought. 3) Allele-specific mRNA-seq allows further characterization of functional variants associated with gene regulation. Created with [Biorender.com](https://biorender.com/)**b**, Correlation of haplotype-specific MOA-seq data at all MPs in nuclei from B73 versus Mo17 inbreds (_x_ axis) versus those from the F1 (_y_ axis) (Pearson correlation, 0.78). MPs with significant (red, _P_ < 0.05, expected _trans_) and without significant (black, expected _cis_) differences between F1 and parental alleles are marked. **c**, Genome-wide comparison of allelic bias (50–60% to one allele considered no bias, >60% considered biased) at B73 × Mo17 F1 AMP sites to inbred B73 versus Mo17 data. Only sites that displayed binding in both inbreds and hybrids were considered. **d**, Genome-wide directionality analysis, comparing AMPs detected by MOA-seq to ChIP–seq data of a single TF BZR1 (ref. [17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).")) in the B73 × Mo17 hybrid. Only sites that displayed binding in both ChIP and hybrid MOA were considered. In **c** and **d**, MOA occupancy was largely consistent (red circle) between either F1 and parents or compared to ChIP–seq, respectively, in showing bias towards B73 (green) or Mo17 (blue) in both cases, with a smaller fraction of allele-specific F1 MOA sites showing no bias (gray) in inbreds or ChIP–seq, or bias to the opposite parent or allele (B73 in F1 and Mo17 in inbred or ChIP, purple or Mo17 in F1 and B73 in inbred or ChIP, yellow).

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM6)

The vast majority (88% or 194,594 out of 221,187) of all MPs showed no significant difference in their allelic bias comparing F1 (B73 and Mo17 haplotypes) to B73 versus Mo17 inbred alleles (black dots in Fig. [1b](https://www.nature.com/articles/s41588-025-02246-7#Fig1)), and about 90% (31,949 out of 35,638) of AMP sites in the B73 × Mo17 F1 showed bias towards the same allele as when comparing the inbred parents (red line in Fig. [1c](https://www.nature.com/articles/s41588-025-02246-7#Fig1)). Fewer than 0.6% (199 out of 35,638) of AMP sites showed bias in the opposite direction. In the F1, allele-specific bias at AMPs should not be affected by _trans_\-factors, biological or technical variation, as the relative haplotype differences originate from the same F1 cells. The high concordance between haplotype-specific bias in the F1 and inbreds at AMP loci is consistent with this expectation, further establishes the reproducibility of the assay and indicates that the majority of AMPs are coupled to genotypic differences in _cis_ at the binding site, rather than resulting from _trans_\-acting or _cis_\-by-_trans_ interaction effects. The fact that some differences are observed, however, underpins the importance of using F1 hybrids rather than inbred lines, in which _trans_\-acting and _cis_\-acting effects cannot be easily disentangled.

To independently validate haplotype-specific, MOA-defined, putative TF footprints in B73 × Mo17, we compared AMPs to recently published allele-specific ChIP–seq data of the major brassinosteroid TF ZmBZR1 in the same F1 (ref. [17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).")). More than 70% of AMPs overlapping with ZmBZR1 binding sites showed allelic bias in the same direction in both studies (red line in Fig. [1d](https://www.nature.com/articles/s41588-025-02246-7#Fig1)). About 22% of AMPs showed no bias in the ChIP–seq data, probably because of the lower resolution of haplotype-specific ChIP–seq (~500 bp fragments compared to ~65 bp for MOA-seq). Only 7% of AMPs showed bias for different alleles than in ChIP–seq, potentially reflecting biological differences in the tissues analyzed (meristem and leaf versus leaf) or ectopic BZR1 activity owing to exogenous brassinosteroid treatment[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023)."). Detailed comparison of MOA-seq occupancy to the ZmBZR1 ChIP–seq data demonstrates the accuracy and resolution of our approach, accurately predicting expression of downstream genes and enabling the identification of likely causal polymorphisms within the TF binding site (Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02246-7#Fig8) and Supplementary Note [1](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). Together, these examples illustrate the potential of MOA-seq to annotate candidate _cis_\-regulatory elements with quantitative chromatin footprint data that connects _cis_\-variation to biases in _cis_\-element occupancy.

### Defining functional sites in a maize pan-cistrome

To define a leaf pan-cistrome of maize, we analyzed a population of 25 F1 hybrids using haplotype-specific MOA-seq (Fig. [1a](https://www.nature.com/articles/s41588-025-02246-7#Fig1)). The hybrid population, created by crossing 25 inbred lines with high-quality genome assemblies[18](https://www.nature.com/articles/s41588-025-02246-7#ref-CR18 "Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science 373, 655–662 (2021)."),[19](https://www.nature.com/articles/s41588-025-02246-7#ref-CR19 "Lin, G. et al. Chromosome-level genome assembly of a regenerable maize inbred line A188. Genome Biol. 22, 175 (2021)."),[20](https://www.nature.com/articles/s41588-025-02246-7#ref-CR20 "Springer, N. M. et al. The maize W22 genome provides a foundation for functional genomics and transposon biology. Nat. Genet. 50, 1282–1288 (2018)."),[21](https://www.nature.com/articles/s41588-025-02246-7#ref-CR21 "Sun, S. et al. Extensive intraspecific gene order and gene structural variations between Mo17 and other maize genomes. Nat. Genet. 50, 1289–1295 (2018).") to the reference genome line B73, represents a diverse set of maize including many of the parents of an important mapping population and several important genetic stocks (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). We analyzed allele-specific TF occupancy and mRNA abundance in leaf blades of each F1 cross (Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02246-7#MOESM5) and [4](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). By aligning MOA-seq and RNA-seq reads to concatenated dual-reference genomes rather than a single reference, our approach resolves issues of reference bias that confound most allele-specific analyses[22](https://www.nature.com/articles/s41588-025-02246-7#ref-CR22 "Castel, S. E., Levy-Moonshine, A., Mohammadi, P., Banks, E. & Lappalainen, T. Tools and best practices for data processing in allelic expression analysis. Genome Biol. 16, 195 (2015).") (Fig. [2a](https://www.nature.com/articles/s41588-025-02246-7#Fig2); [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)). We identified an average of 237,000 MOA peaks (FDR, 5%) per F1, covering approximately 2% (around 80 Mbp) of each hybrid genome (Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). On average, 19.9% (14–30%) of MPs showed a significant allelic bias (binomial test, FDR, 1%; Supplementary Table [5](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)) with an overall even split between the parental alleles (50.2% B73 and 49.8% diverse parents; Supplementary Fig. [5](https://www.nature.com/articles/s41588-025-02246-7#MOESM1) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). It is noteworthy that the average rate of AMPs (19.9%) closely matches allele-specific TF binding sites detected by the gold standard of ChIP–seq for an individual TF (18.3%)[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023)."). In total, AMPs overlapped with 35.6% of all MOA footprint peaks in B73 (Fig. [2b](https://www.nature.com/articles/s41588-025-02246-7#Fig2)), and plots of the identified MOA peaks and cumulative base pairs indicate that our diverse population is near saturation and has identified the majority of the B73 leaf cistrome (Supplementary Fig. [6](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)).

**Fig. 2: Construction of a maize leaf pan-cistrome.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02246-7/figures/2)

**a**, Mapping strategies comparison showing the density of AMPs (allele-specific occupied sites) over the percentage of binding to B73. B73 × HP301 F1 MOA data were analyzed using either only B73 as reference genome (single ref.), a pseudo-genome with B73/HP301 SNPs replaced by Ns (SNP-replaced) or our dual-parent mapping strategy using a concatenated B73 × HP301 genome (dual ref.). Without mapping bias, a symmetric distribution is expected (as observed for dual ref.), while a higher density at higher B73 allelic bias indicates biased mapping to the reference genome B73 (single ref. and SNP-replaced). For the A619 F1 (no assembled genome available), our ‘reference-guided’ strategy (see [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11) for details) showed similar AMP-balanced haplotypes without reference bias (A619 dual ref.). **b**, Additive percent of B73 MOA peak covered by MPs (brown) and AMPs (red) relative to the number of F1s analyzed. **c**, Density of mean MOA binding frequencies over all F1s carrying a SNP at positions where at least one, two, three or four F1s had AMPs, compared to a control with randomized binding frequencies. **d**, Overview of bQTL (red arrows), MOA coverage (blue) and Hi-C interaction sites (black lines, Hi-C from a previous publication[32](https://www.nature.com/articles/s41588-025-02246-7#ref-CR32 "Ricci, W. A. et al. Widespread long-range cis-regulatory elements in the maize genome. Nat. Plants 5, 1237–1249 (2019).")) near the classical flowering repressor _RAP2.7_. bQTL overlap with both known enhancers, _vgt1_ and _vgt1_\-DMR (green), associated with _RAP2.7_ expression. An additional bQTL, termed _vgt1_\-MOA (magenta), also interacts with _vgt1_ and the RAP2-7 promoter.

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM7)

We next sought to identify variants, genetic, epigenetic, or both, associated with differences in MOA-detected TF occupancy between haplotypes, or bQTL, in our population. We first verified that F1s that shared haplotypes at AMP loci also share similar patterns of allelic bias (Fig. [2c](https://www.nature.com/articles/s41588-025-02246-7#Fig2)), indicating that our SNPs were in sufficient linkage disequilibrium with causal differences to perform association analysis. Differences in DNA methylation between parental alleles can affect TF binding affinity[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023)."),[23](https://www.nature.com/articles/s41588-025-02246-7#ref-CR23 "O’Malley, R. C. et al. Cistrome and epicistrome features shape the regulatory DNA landscape. Cell 165, 1280–1292 (2016)."). After validating that DNA methylation differences at AMPs detected between F1 haplotypes were consistent between the parental lines (Extended Data Fig. [2a,b](https://www.nature.com/articles/s41588-025-02246-7#Fig9)), we added previously published methylation data for 24 of our parental lines[18](https://www.nature.com/articles/s41588-025-02246-7#ref-CR18 "Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science 373, 655–662 (2021)."),[19](https://www.nature.com/articles/s41588-025-02246-7#ref-CR19 "Lin, G. et al. Chromosome-level genome assembly of a regenerable maize inbred line A188. Genome Biol. 22, 175 (2021)."),[20](https://www.nature.com/articles/s41588-025-02246-7#ref-CR20 "Springer, N. M. et al. The maize W22 genome provides a foundation for functional genomics and transposon biology. Nat. Genet. 50, 1282–1288 (2018)."). We performed linear modeling to test all positions that are MPs (SNPs or INDELs in MOA peaks; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)) in at least two lines for association of MOA signal variation with either the genotype information, DNA methylation level or both. We identified a total of 176,613 (147,942 SNP and 28,671 INDEL loci, FDR < 0.05; Supplementary Table [6](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)) significant associations, termed bQTL, of which 93,682, 51,192 and 31,739 bQTL were a result of genotype variation alone, DNA methylation variation alone or both features, respectively (Extended Data Fig. [2c,d](https://www.nature.com/articles/s41588-025-02246-7#Fig9)). As expected, the genome-wide distribution of bQTL was distinct from all SNPs and more closely matched those of previously published allele-specific TF binding sites determined by ChIP–seq[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).") (Supplementary Fig. [7](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). A notable bQTL includes an 8 bp (T/TTAGCGTGT) INDEL in the hypervariable region of the _ZmBIF2_ (Zm00001eb031760) promoter (Supplementary Note [1](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)) at a site bound by multiple TF families[12](https://www.nature.com/articles/s41588-025-02246-7#ref-CR12 "Tu, X. et al. Reconstructing the maize leaf regulatory network using ChIP-seq data of 104 transcription factors. Nat. Commun. 11, 5089 (2020).") (bZIP, EREB, bHLH, MYB and WRKY; Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02246-7#Fig10)). Overall, we found that INDEL variants show patterns very similar to SNPs, with, for example, 63% of INDEL bQTL overlapping with a SNP bQTL within 65 bp. We thus focused our further analysis on the SNP bQTL while providing the INDEL bQTL to increase the resolution of the pan-cistrome map.

### bQTL coincide with known, causative regulatory loci

Detailed analyses of regulatory variation for a number of maize genes provide an opportunity to compare bQTL to previously identified causal variation. One bQTL was directly adjacent to the YABBY TF binding site underlying the leaf architecture QTL upright plant architecture2 (ref. [24](https://www.nature.com/articles/s41588-025-02246-7#ref-CR24 "Tian, J. et al. Teosinte ligule allele narrows plant architecture and enhances high-density maize yields. Science 365, 658–664 (2019).")) (Zm00001eb073010) (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02246-7#Fig10)). bQTL also identified haplotype-specific footprints at flowering time loci, including the causative transposon insertions at _ZmCCT9_ (Zm00001eb391230) that was targeted by selection during maize adaptation to higher latitudes[25](https://www.nature.com/articles/s41588-025-02246-7#ref-CR25 "Yang, Q. et al. CACTA-like transposable element in ZmCCT attenuated photoperiod sensitivity and accelerated the postdomestication spread of maize. Proc. Natl Acad. Sci. USA 110, 16969 (2013)."),[26](https://www.nature.com/articles/s41588-025-02246-7#ref-CR26 "Huang, C. et al. ZmCCT9 enhances maize adaptation to higher latitudes. Proc. Natl Acad. Sci. USA 115, E334–E341 (2018)."), INDEL-2339 in the promoter of the FT-like _ZmZCN8_ (ref. [27](https://www.nature.com/articles/s41588-025-02246-7#ref-CR27 "Meng, X., Muszynski, M. G. & Danilevskaya, O. N. The FT-like ZCN8 gene functions as a floral activator and is involved in photoperiod sensitivity in maize. Plant Cell 23, 942–960 (2011).")) (Zm00001eb353250), a 850 bp structural variant in the promoter of _ZmPHYB2_ (ref. [28](https://www.nature.com/articles/s41588-025-02246-7#ref-CR28 "Li, C. et al. Gene expression and expression quantitative trait loci analyses uncover natural variations underlying the improvement of important agronomic traits during modern maize breeding. Plant J. 115, 772–787 (2023).")) (Zm00001eb396030) as well as multiple GWAS hits for flowering time (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02246-7#Fig10)). In addition to identifying bQTL in both of the known distal regulatory regions, vegetative to generative transition 1 (_vgt1_) and _vgt1_\-DMR, of the key flowering time locus _ZmRAP2.7_ (refs. [29](https://www.nature.com/articles/s41588-025-02246-7#ref-CR29 "Salvi, S. et al. Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize. Proc. Natl Acad. Sci. USA 104, 11376–11381 (2007)."),[30](https://www.nature.com/articles/s41588-025-02246-7#ref-CR30 "Castelletti, S., Tuberosa, R., Pindo, M. & Salvi, S. A MITE transposon insertion is associated with differential methylation at the maize flowering time QTL Vgt1. G3 (Bethesda) 4, 805–812 (2014)."),[31](https://www.nature.com/articles/s41588-025-02246-7#ref-CR31 "Xu, G. et al. Evolutionary and functional genomics of DNA methylation in maize domestication and improvement. Nat. Commun. 11, 5539 (2020).")) (Zm00001eb355240), our bQTL analysis identified an undescribed, third putative enhancer more than 100 kb upstream, which we termed _vgt1_\-MOA (Fig. [2d](https://www.nature.com/articles/s41588-025-02246-7#Fig2)). Hi-C long-range interaction data[32](https://www.nature.com/articles/s41588-025-02246-7#ref-CR32 "Ricci, W. A. et al. Widespread long-range cis-regulatory elements in the maize genome. Nat. Plants 5, 1237–1249 (2019).") confirmed that _vgt1_\-MOA physically interacts with both _vgt1_ and the proximal _ZmRAP2.7_ promoter (Fig. [2d](https://www.nature.com/articles/s41588-025-02246-7#Fig2)). However, future functional tests are needed to establish whether _vgt1_\-MOA effects _ZmRAP2.7_ expression alone or in combination with _vgt1_ and/or _vgt1_\-DMR.

We further observed that bQTL colocalized with the regulatory variation upstream of _ZmGT1_ (Zm00001eb007950), which is targeted by ZmTB1 (Zm00001eb054440), with the two forming a regulatory module involved in bud dormancy and growth repression[33](https://www.nature.com/articles/s41588-025-02246-7#ref-CR33 "Dong, Z. et al. The regulatory landscape of a core maize domestication module controlling bud dormancy and growth repression. Nat. Commun. 10, 3810 (2019)."). bQTL coincide with the transposon-associated causal regulatory region for prolificacy (_prol1.1_) upstream of _ZmGT1_, including one bQTL directly adjacent to the TB1 binding site[34](https://www.nature.com/articles/s41588-025-02246-7#ref-CR34 "Wills, D. M. et al. From many, one: genetic control of prolificacy during maize domestication. PLoS Genet. 9, e1003604 (2013).") (Supplementary Fig. [8](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)).

Our MOA-seq pan-cistrome also provides an opportunity to evaluate how variation at these sites compares to changes in _cis_\-element occupancy. For example, an INDEL in the TREHALASE1 (_ZmTRE1_, Zm00001eb021270) promoter has been associated with both trehalose amounts and _ZmTRE1_ transcript levels in maize[35](https://www.nature.com/articles/s41588-025-02246-7#ref-CR35 "Wen, W. et al. An integrated multi-layered analysis of the metabolic networks of different tissues uncovers key genetic components of primary metabolism in maize. Plant J. 93, 1116–1128 (2018)."). We observed haplotype-specific footprints, both at a previously reported 8 bp insertion[35](https://www.nature.com/articles/s41588-025-02246-7#ref-CR35 "Wen, W. et al. An integrated multi-layered analysis of the metabolic networks of different tissues uncovers key genetic components of primary metabolism in maize. Plant J. 93, 1116–1128 (2018).") and an additional SNP 29 bp upstream, which coincided with a bQTL (Supplementary Fig. [9](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). Notably, although the 8 bp insertion creates a potential ABI motif (TGCCACAC), the _ZmTRE1_\-bQTL overlaps with a DOF binding motif (AAAAGGTG). Previously published ChIP–seq results confirm that the _ZmTRE1_\-bQTL site is targeted by ZmDOF17 (ref. [12](https://www.nature.com/articles/s41588-025-02246-7#ref-CR12 "Tu, X. et al. Reconstructing the maize leaf regulatory network using ChIP-seq data of 104 transcription factors. Nat. Commun. 11, 5089 (2020).")) (Supplementary Fig. [9](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). Furthermore, all alleles (6 out of 6) in our F1 population without the 8 bp insertion and with the non-canonical DOF motif (C instead of G) at the bQTL site showed concomitant low MOA signal (strong bias towards B73’s G allele) and _ZmTRE1_ mRNA levels (higher B73 mRNA level) (Fig. [3a,b](https://www.nature.com/articles/s41588-025-02246-7#Fig3)). In another example, ZmSUBTILISIN11 (_ZmSUB11_, Zm00001eb152020) has been associated with cell wall compositions, peduncle vascular traits and abscisic acid (ABA) levels[36](https://www.nature.com/articles/s41588-025-02246-7#ref-CR36 "Cui, H. et al. Genome-wide identification and analysis of the maize serine peptidase S8 family genes in response to drought at seedling stage. Plants 12, 369 (2023)."),[37](https://www.nature.com/articles/s41588-025-02246-7#ref-CR37 "Sun, G. et al. Dissection of the genetic architecture of peduncle vascular bundle-related traits in maize by a genome-wide association study. Plant Biotechnol. J. 20, 1042–1053 (2022)."). A previously identified _cis_\-expression QTL lead SNP for _ZmSUB11_ transcript levels[38](https://www.nature.com/articles/s41588-025-02246-7#ref-CR38 "Sun, G. et al. A role for heritable transcriptomic variation in maize adaptation to temperate environments. Genome Biol. 24, 55 (2023).") coincided with a bQTL in its proximal promoter, and we observed a strong correlation of haplotype-specific MOA footprints at the bQTL and _ZmSUB11_ transcript levels (Fig. [3a,b](https://www.nature.com/articles/s41588-025-02246-7#Fig3)).

**Fig. 3: bQTL correlate with haplotype-specific transcript variation.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02246-7/figures/3)

**a**, Allelic distribution (%B73) of MOA reads at bQTL in either the _TRE1_ (Zm00001eb021270, bQTL: B73-chr1:80,826,022) or _SUB11_ (Zm00001eb152020, bQTL B73-chr3:198,733,446) proximal promoters (_TRE1_: 19 F1s with G/G and six F1s with G/C alleles; _SUB11_: eight F1s with C/C and 16 F1s with C/T alleles). **b**, Haplotype-specific mRNA counts for _TRE1_ and _SUB11_ grouped by their respective bQTL alleles in **a** (_TRE1_: nine F1s with G/G and six F1s carrying G/C alleles; _SUB11_: six F1s with C/C and 15 F1s with C/T alleles). Only lines with polymorphic alleles were considered. FC, fold change. **c**, Genes with ASE and non-ASE mRNA abundance are significantly more and less enriched for AMPs in their 3 kb upstream promoter, respectively (_n_ = 24 F1s, hypergeometric test; Supplementary Table [7a–c](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). **d**, bQTL are more often in linkage disequilibrium with _cis-_expression QTL, identified in roots of 340 recombinant inbred lines than matched bgSNPs. **e**, Average, normalized MOA coverage for B73 and NAM alleles of B73 × Mo18W and B73 × CML69 upstream of _PGM1_, Zm00001eb196320. The Mo18W allele showed significantly higher MOA occupancy (green panel), while the CML69 allele showed similar MOA coverage to B73, yet peaks were shifted ~300 bp because of a MITE transposon (red triangle) insertion (purple panel). **f**, Allele-specific mRNA counts (_n_ = 3 biological replicates) of _PGM1_ in the different F1 hybrids. Colors indicate MOA ratios at bQTL: NAM > B73 (green, >60% bias to NAM, non-B73 allele for at least one bQTL), B73 = NAM (yellow, %B73 occupancy between 40% and 60%, sharing B73 genotype) and transposon insertion haplotype (purple). All F1s within the NAM > B73 and transposon category displayed significantly higher or lower mRNA levels in the NAM allele compared to B73, respectively (detected by DESeq2; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)), while none of the B73 = NAM category were significantly different. Boxplots in **a**, **b**, **c** and **f** denote the range from the first to the third quartile, lines within boxes indicate the median and whiskers represent 1.5-fold of the interquartile range.

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM8)

### MOA bQTL correlate with transcript levels

If variation in MOA coverage accurately captures TF binding affinity, we would expect to see associations between haplotype-specific MOA coverage and transcript abundance in our F1s. Indeed, we find that the promoters (within 3 kb upstream of the transcription start site (TSS)) of genes with significant allele-specific expression (ASE, _P_ < 0.05; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)) were ~34% and ~74% enriched for the presence of AMPs compared to all expressed and non-haplotype-specific (non-ASE, _P_ > 0.95) expressed genes, respectively (Fig. [3c](https://www.nature.com/articles/s41588-025-02246-7#Fig3), Supplementary Table [7a–c](https://www.nature.com/articles/s41588-025-02246-7#MOESM5) and Supplementary Fig. [10](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). Genotype-associated bQTL were also substantially more likely to be in high linkage disequilibrium (>0.6) with nearby _cis_\-expression QTL in a panel of 340 maize genotypes[38](https://www.nature.com/articles/s41588-025-02246-7#ref-CR38 "Sun, G. et al. A role for heritable transcriptomic variation in maize adaptation to temperate environments. Genome Biol. 24, 55 (2023).") than matched background SNPs (bgSNPs) (49.1% more intergenic bQTL relative to bgSNPs (456/306) and 28.2% more total bQTL versus bgSNPs (1,775/1,384), respectively) (Fig. [3d](https://www.nature.com/articles/s41588-025-02246-7#Fig3)). These broad patterns are reflected at the level of individual genes as well. For example, all of the NAM parents showing greater MOA occupancy at the bQTL upstream of PHOSPHOGLYCERATE MUTASE1 (_ZmPGM1_, Zm00001eb196320) showed significantly increased abundance of the NAM transcript, whereas F1s with no polymorphism between B73 and NAM in their promoter or 5′ untranslated region showed no significant difference in haplotype-specific transcript levels (Fig. [3e,f](https://www.nature.com/articles/s41588-025-02246-7#Fig3)). Two NAM parents, Ki3 and CML69, showed much lower _PGM1_ transcript levels (Fig. [3f](https://www.nature.com/articles/s41588-025-02246-7#Fig3)), while no significant variation in MOA footprint was detected. Instead, Ki3 and CML69 harbored a PIF/Harbinger transposon insertion accompanied by hypermethylation between the MOA peak and _PGM1_ TSS, not found in any of the haplotypes (including B73) with higher _PGM1_ transcript levels (Fig. [3e,f](https://www.nature.com/articles/s41588-025-02246-7#Fig3) and Supplementary Fig. [11](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)).

### Variation in DNA methylation can predict MOA occupancy

The vast majority of TFs in _Arabidopsis_ have been shown, in vitro, to have higher binding affinity to hypomethylated DNA[23](https://www.nature.com/articles/s41588-025-02246-7#ref-CR23 "O’Malley, R. C. et al. Cistrome and epicistrome features shape the regulatory DNA landscape. Cell 165, 1280–1292 (2016)."). We explored this association in our data, focusing on variation in CG and CHG methylation (mCG/mCHG), as they accounted for >99.8% of methylation differences at MOA sites. DNA methylation differences (following a previous publication[39](https://www.nature.com/articles/s41588-025-02246-7#ref-CR39 "Regulski, M. et al. The maize methylome influences mRNA splice sites and reveals widespread paramutation-like switches guided by small RNA. Genome Res. 23, 1651–1662 (2013)."), one allele <10% methylated and the other >70%) overlapped with 14.8% of MPs in the F1s. At AMPs, haplotype-specific mCG/mCHG overlap increased by 2.6-fold (38.1%) and reached more than half (51.5%) for AMPs with a strong haplotype-bias (≥85% to one allele) (Fig. [4a](https://www.nature.com/articles/s41588-025-02246-7#Fig4)). We observed a very strong correlation between a higher footprint occupancy and the hypomethylated allele (Fig. [4b](https://www.nature.com/articles/s41588-025-02246-7#Fig4)), with 98.2% of AMPs showing higher MOA coverage at the hypomethylated allele. Furthermore, nearly half of the remaining 1.8% AMPs biased towards hypermethylation alleles did not display methylation difference immediately surrounding the AMP (11 bp window), intimating that there may be no actual methylation difference at the occupied site despite hypermethylation of the surrounding region (41 bp window) (Supplementary Fig. [12](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). On average, the vast majority (71.2%) of F1s that shared differentially methylated alleles at a given locus also shared haplotype-specific MOA footprints at that site, compared to only 42.9% of the F1s with shared equally methylated alleles at that same site (Fig. [4c](https://www.nature.com/articles/s41588-025-02246-7#Fig4)). The observed strong correlations between differential CG and CHG methylation and haplotype-specific MOA occupancy confirm an important role for DNA methylation in determining TF binding in maize.

**Fig. 4: bQTL are linked to variation in DNA methylation.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02246-7/figures/4)

**a**, Genome-wide overlap of differentially methylated (CG and/or CHG) DNA regions with MPs, AMPs and AMPs with strong (≥85%) allelic bias, across the 24 F1s. **b**, Correlation of differentially CG-methylated DNA with allelic bias for MPs in the 24 F1 hybrids. MP methylation categories: equally methylated (eqM), B73 hypermethylated versus NAM hypomethylated (B-hyper N-hypo) or B73 hypomethylated versus NAM hypermethylated (B-hypo N-hyper). **c,** Correlation between MOA footprint bias and differential methylation at loci that varied both in allele-specific footprint occupancy (≥1 F1s with AMP) and CG methylation (≥2 F1s with and without allele-specific methylation difference) between the 24 F1s. At each position, F1s were partitioned into those with either differential allelic CG methylation (red) or equal CG methylation (blue). Box and violin plots were drawn for the two categories, showing the distribution of percentages of F1s with haplotype-specific binding (AMPs). _n_ = 24 F1s in **a**, **b** and **c**; boxplots were generated as in Fig. [3](https://www.nature.com/articles/s41588-025-02246-7#Fig3).

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM9)

### MOA bQTL explain a large portion of heritable variation

Regulatory variation is thought to underlie a significant proportion of phenotypic variation in maize[40](https://www.nature.com/articles/s41588-025-02246-7#ref-CR40 "Wallace, J. G. et al. Association mapping across numerous traits reveals patterns of functional variation in maize. PLoS Genet. 10, e1004845 (2014)."). To assess the relationship between bQTL and complex trait variation, we first quantified the enrichment of genotype-associated bQTL surrounding GWAS hits (lead SNP ± 100 bp) across two curated datasets of 41 and 279 traits[40](https://www.nature.com/articles/s41588-025-02246-7#ref-CR40 "Wallace, J. G. et al. Association mapping across numerous traits reveals patterns of functional variation in maize. PLoS Genet. 10, e1004845 (2014)."),[41](https://www.nature.com/articles/s41588-025-02246-7#ref-CR41 "Tian, D. et al. GWAS Atlas: a curated resource of genome-wide variant–trait associations in plants and animals. Nucleic Acids Res. 48, D927–D932 (2020)."). Given that bQTLs showed a genome-wide distribution distinct from all SNPs (Supplementary Fig. [7](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)), with bQTL located closer to genes, we generated a background dataset to match this distribution (similar allele frequency and distance to the nearest gene, 100 permutations; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)) to avoid any bias caused by location in the genome. For both GWAS datasets tested, bQTL were approximately twofold (1.75-fold and 2.17-fold, respectively) enriched for co-localization with GWAS hits compared to the matched bgSNPs (Supplementary Fig. [13](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). This enrichment remained stable as a function of distance to the nearest gene, indicating comparable efficacy of bQTL to mark functionally significant loci genome-wide (Fig. [5a](https://www.nature.com/articles/s41588-025-02246-7#Fig5)). To explore the degree to which bQTL can more broadly capture the genetic variation underlying phenotypic diversity, we partitioned heritable trait variance for 143 traits in the NAM population ([Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11) and previous publications[2](https://www.nature.com/articles/s41588-025-02246-7#ref-CR2 "Rodgers-Melnick, E., Vera, D. L., Bass, H. W. & Buckler, E. S. Open chromatin reveals the functional maize genome. Proc. Natl Acad. Sci. USA 113, E3177–E3184 (2016)."),[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).")). We modeled additive genetic variation for traits as a function of three genomic relatedness matrices. Variances estimated this way for several trait datasets simulated from matrices highly similar to our observed matrices accurately reflected the proportional contributions of each SNP set (Supplementary Fig. [14](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). Across a large majority of phenotypes in the NAM panel (103 of 143 or ~72%), bQTL associated with genotype alone (that is, excluding methylation, 78,398 bQTL) explained the majority of the total additive genetic variance captured by SNPs (Fig. [5b](https://www.nature.com/articles/s41588-025-02246-7#Fig5), Supplementary Fig. [15](https://www.nature.com/articles/s41588-025-02246-7#MOESM1) and Supplementary Table [8](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). Consistent with previous findings that open chromatin and TF binding, found at a higher frequency close to genes, have a key role in trait variation[2](https://www.nature.com/articles/s41588-025-02246-7#ref-CR2 "Rodgers-Melnick, E., Vera, D. L., Bass, H. W. & Buckler, E. S. Open chromatin reveals the functional maize genome. Proc. Natl Acad. Sci. USA 113, E3177–E3184 (2016)."),[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023)."), our matched bgSNPs (matched allele frequency and distance to the nearest gene compared to bQTL) often accounted for more additive genetic variation than SNPs from the rest of the genome (that is non-bQTL, non-matched bgSNPs; 121 out of 143 traits), but bQTL also outperformed bgSNPs for most traits (81.1%, 116 traits; Fig. [5b](https://www.nature.com/articles/s41588-025-02246-7#Fig5)). The inclusion of bQTL with additional significantly associated differential methylation (105,398 bQTL) slightly decreased the variation explained (Supplementary Fig. [16](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). This is consistent with theoretical arguments that epigenetic variation, which is highly labile on an evolutionary timescale, cannot explain much heritability for phenotypes[42](https://www.nature.com/articles/s41588-025-02246-7#ref-CR42 "Slatkin, M. Epigenetic inheritance and the missing heritability problem. Genetics 182, 845–850 (2009)."). Traits for which bQTL explained the largest portion of genetic variance included plant height, leaf size or shape and disease resistance, whereas almost all traits related to, for example, vitamin E production were best explained by the bQTL-matched bgSNPs or the remaining SNPs from the rest of the genome (Fig. [5c](https://www.nature.com/articles/s41588-025-02246-7#Fig5)), probably because of the oligogenic nature of the vitamin E traits and that bQTL identified in leaf tissue may not be representative of regulatory patterns in genes specifically expressed in kernels[43](https://www.nature.com/articles/s41588-025-02246-7#ref-CR43 "Diepenbrock, C. H. et al. Novel loci underlie natural variation in vitamin E levels in maize grain. Plant Cell 29, 2374–2392 (2017).").

**Fig. 5: A large fraction of heritability is explained by bQTL.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02246-7/figures/5)

**a**, Association of ~42,000 curated GWAS hits[41](https://www.nature.com/articles/s41588-025-02246-7#ref-CR41 "Tian, D. et al. GWAS Atlas: a curated resource of genome-wide variant–trait associations in plants and animals. Nucleic Acids Res. 48, D927–D932 (2020).") (±100 bp) with bQTL (only those associated with genotype alone), or _n_ = 100 bootstraps of matched bgSNPs (same bgSNP sets used as for VCAP; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)) at distances ranging from intragenic to >20 kb to the nearest gene. **b**, Estimated additive genetic variance organized by 143 traits. Colored ridges show the estimated additive genetic variance across 100 permutations for either bQTL, bgSNPs or remaining genome SNPs. Black symbols represent the mean estimated value across permutations. Traits are arranged by bQTL mean variance estimates and color-coded according to general trait groupings: vitamin E metabolites, navy blue; metabolites, purple; stalk strength, light blue; flowering time, gold; plant architecture, red; disease, green; tassel architecture, pink; ear architecture, orange; miscellaneous, gray. **c**, A subset of traits (_y_ axis) and their estimated percent additive genetic variance (_x_ axis) shown as colored box plots instead of ridges. PH, plant height[51](https://www.nature.com/articles/s41588-025-02246-7#ref-CR51 "Hung, H.-Y. et al. The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population. Heredity 108, 490–499 (2012)."),[52](https://www.nature.com/articles/s41588-025-02246-7#ref-CR52 "Peiffer, J. A. et al. The genetic architecture of maize height. Genetics 196, 1337–1356 (2014)."); LeafL, leaf length[51](https://www.nature.com/articles/s41588-025-02246-7#ref-CR51 "Hung, H.-Y. et al. The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population. Heredity 108, 490–499 (2012)."); DTA, days to anthesis[51](https://www.nature.com/articles/s41588-025-02246-7#ref-CR51 "Hung, H.-Y. et al. The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population. Heredity 108, 490–499 (2012)."),[52](https://www.nature.com/articles/s41588-025-02246-7#ref-CR52 "Peiffer, J. A. et al. The genetic architecture of maize height. Genetics 196, 1337–1356 (2014)."); DTS, days to silking[51](https://www.nature.com/articles/s41588-025-02246-7#ref-CR51 "Hung, H.-Y. et al. The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population. Heredity 108, 490–499 (2012)."); SLB, southern leaf blight[53](https://www.nature.com/articles/s41588-025-02246-7#ref-CR53 "Kump, K. L. et al. Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population. Nat. Genet. 43, 163–168 (2011)."),[54](https://www.nature.com/articles/s41588-025-02246-7#ref-CR54 "Bian, Y., Yang, Q., Balint-Kurti, P. J., Wisser, R. J. & Holland, J. B. Limits on the reproducibility of marker associations with southern leaf blight resistance in the maize nested association mapping population. BMC Genomics 15, 1068 (2014)."); and delta-tocopherol concentration, vitamin E biosynthesis[43](https://www.nature.com/articles/s41588-025-02246-7#ref-CR43 "Diepenbrock, C. H. et al. Novel loci underlie natural variation in vitamin E levels in maize grain. Plant Cell 29, 2374–2392 (2017)."); _n_ = 100 permutations. Boxplots in **a** and **c** were generated as described in Fig. [3](https://www.nature.com/articles/s41588-025-02246-7#Fig3); data outside the whisker range are considered outliers.

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM10)

### Characterization of a drought-responsive cistrome

To evaluate differences in _cis_\-element regulation induced by changes in environmental conditions, we compared the morphological and molecular response of our F1 population under well-watered (WW) and drought-stress (DS) conditions. We observed diverse drought responses, with reductions of relative leaf water content of 3–30% and remaining soil water contents of 6.3–25.6%, depending on the F1 (Fig. [6a,b](https://www.nature.com/articles/s41588-025-02246-7#Fig6), Supplementary Fig. [17](https://www.nature.com/articles/s41588-025-02246-7#MOESM1) and Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02246-7#Fig11)). Haplotype-specific MOA-seq and RNA-seq of WW and DS samples for all 25 F1s revealed on average 287,844 MPs and 56,863 AMPs under DS, slightly less than for WW conditions (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)), and a similar correlation with allele-specific transcript abundance (Supplementary Fig. [10](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). MOA peaks showing significant (_P_ < 0.05) drought-induced increases or decreases in occupancy varied substantially among F1s, ranging from ~9,000 to 40,000 and 16,000 to 90,000, respectively (Supplementary Table [9](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). Local association mapping identified 124,504 DS-bQTL for SNPs and 23,554 for small INDELs under drought conditions (Supplementary Table [10](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)), for a combined total of 206,368 unique SNP bQTL in DS and/or WW. To identify candidate drought-response loci, we selected bQTL with drought-responsive occupancy near genes (5 kb upstream or 1 kb downstream) that displayed both haplotype-specific and drought-responsive transcript accumulation, resulting in 1,025 (655 genes) and 2,604 (1,548 genes) bQTL with increased and decreased occupancy, respectively. Further integration with drought-response GWAS and _cis_\-expression QTL hits[44](https://www.nature.com/articles/s41588-025-02246-7#ref-CR44 "Li, C. et al. Numerous genetic loci identified for drought tolerance in the maize nested association mapping populations. BMC Genomics 17, 894 (2016)."),[45](https://www.nature.com/articles/s41588-025-02246-7#ref-CR45 "Shikha, M. et al. Genomic selection for drought tolerance using genome-wide SNPs in maize. Front. Plant Sci. 8, 550 (2017)."),[46](https://www.nature.com/articles/s41588-025-02246-7#ref-CR46 "Wu, X. et al. Using high-throughput multiple optical phenotyping to decipher the genetic architecture of maize drought tolerance. Genome Biol. 22, 185 (2021).") resulted in high-confidence candidates (Supplementary Table [11](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). Notably, the candidate list included known drought-tolerance-related genes, such as _ZmNAC111_ (Zm00001eb405590). Haplotype-specific MOA footprinting identified multiple DS-bQTL upstream and downstream of _ZmNAC111_, including adjacent to the causative 84 bp MITE transposon insertion site, which reduces both _ZmNAC111_ expression and drought tolerance in maize seedlings, probably through RNA-directed DNA methylation[47](https://www.nature.com/articles/s41588-025-02246-7#ref-CR47 "Mao, H. et al. A transposable element in a NAC gene is associated with drought tolerance in maize seedlings. Nat. Commun. 6, 8326 (2015).") (Supplementary Fig. [18](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). Another interesting DS-bQTL was located within the previously discovered 119 bp proximal promoter fragment required for the drought-response of SULFITE OXIDASE1 (_ZmSO_, Zm00001eb036560), a gene linked to the ABA response and drought tolerance of maize seedlings[48](https://www.nature.com/articles/s41588-025-02246-7#ref-CR48 "Xu, Z., Wang, M., Guo, Z., Zhu, X. & Xia, Z. Identification of a 119-bp promoter of the maize Sulfite Oxidase Gene (ZmSO) that confers high-level gene expression and ABA or drought inducibility in transgenic plants. Int. J. Mol. Sci. 20, 3326 (2019)."). Although none of our haplotypes contained the putative Myb-binding site (CAGTTG) previously linked to drought-response in the 119 bp _ZmSO_ promoter[48](https://www.nature.com/articles/s41588-025-02246-7#ref-CR48 "Xu, Z., Wang, M., Guo, Z., Zhu, X. & Xia, Z. Identification of a 119-bp promoter of the maize Sulfite Oxidase Gene (ZmSO) that confers high-level gene expression and ABA or drought inducibility in transgenic plants. Int. J. Mol. Sci. 20, 3326 (2019)."), we nonetheless found a strong correlation between increased MOA occupancy for the C allele at the bQTL and elevated _ZmSO_B73 transcript levels, both under WW and DS conditions (Fig. [6c,d](https://www.nature.com/articles/s41588-025-02246-7#Fig6)). We also observed a strong correlation between MOA occupancy and drought-induced transcript levels at DS-bQTL in the proximal promoter of the maize homolog of aquaporin BETA-TONOPLAST INTRINSIC PROTEIN 3 (_ZmTIP3d_, Zm00001eb076690; Fig. [6e,f](https://www.nature.com/articles/s41588-025-02246-7#Fig6)), which has been linked to drought-response in various plants[49](https://www.nature.com/articles/s41588-025-02246-7#ref-CR49 "Johansson, I., Karlsson, M., Johanson, U., Larsson, C. & Kjellbom, P. The role of aquaporins in cellular and whole plant water balance. Biochim. Biophys. Acta Biomembr. 1465, 324–342 (2000).").

**Fig. 6: Characterization of a drought-responsive cistrome.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02246-7/figures/6)

**a**,**b**, Morphological phenotypes of the more tolerant B73 × Oh43 (**a**) and susceptible B73 × Mo18W (**b**) F1s grown under WW and DS conditions. Scale bars, 10 cm. **c**, Allelic distribution of MOA read coverage at bQTL (B73-chr1:198,205,029) in the _ZmSO_ promoter. The six F1s sharing the B73 allele (C/C) were compared to 17 F1s carrying C/A alleles. **d**, Haplotype-specific mRNA counts, normalized to the B73 WW allele, grouped by the _ZmSO_ bQTL alleles shown in **c**. Haplotype-specific read counts (reads per million (RPM), also adjusted for differences in SNP counts between F1s) relative to the B73 allele WW level to allow count comparisons between F1s (_n_ = 21 F1s with gene polymorphism to permit haplotype-specific analysis, 5 C/C and 16 C/A alleles). **e**, Allelic distribution of MOA reads at a bQTL (B73-chr2:28,118,442) in the _ZmTIP3d_ promoter. The 12 F1s sharing the B73 allele (C/C) were compared to 13 F1s carrying C/G alleles. **f**, Haplotype-specific mRNA abundance grouped by the _ZmTIP3d_ bQTL alleles shown in **e**. RPM values per haplotype are normalized to the B73 DS mRNA level (_n_ = 23 F1s that permitted haplotype-specific MOA/RNA analysis: DS, 10 C/C, 13 C/G; WW: 5 C/C, 5 C/G). n.d., not detected. Boxplots in **c**, **d**, **e** and **f** were generated as described in Fig. [3](https://www.nature.com/articles/s41588-025-02246-7#Fig3).

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM11)

To further test the correlation of DS-bQTL and drought-responsive promoter activity, we analyzed the maize homolog of the _Arabidopsis_ drought-inducible AP2/ERF TF AtTINY independently in a transient expression assay. Over-expression of _AtTINY_ increases drought tolerance at the cost of severely stunted growth, a limitation often observed with drought-related TFs[50](https://www.nature.com/articles/s41588-025-02246-7#ref-CR50 "Xie, Z. et al. The AP2/ERF transcription factor TINY modulates brassinosteroid-regulated plant growth and drought responses in Arabidopsis. Plant Cell 31, 1788–1806 (2019)."). The maize homolog of _ZmTINY_ (Zm00001eb120590) is a candidate gene for drought response and leaf size variation[45](https://www.nature.com/articles/s41588-025-02246-7#ref-CR45 "Shikha, M. et al. Genomic selection for drought tolerance using genome-wide SNPs in maize. Front. Plant Sci. 8, 550 (2017)."). We found DS-bQTL in multiple MOA footprints surrounding _ZmTINY_, which showed significantly higher occupancy in, for example, CML333 and Oh43 compared to B73 under drought (ranging from 1.4-fold to 5.5-fold higher; Fig. [7a](https://www.nature.com/articles/s41588-025-02246-7#Fig7) and Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02246-7#Fig12)). Similarly, higher MOA occupancy under DS for CML333 and Oh43 compared to B73 was also observed downstream of _ZmTINY_ (Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02246-7#Fig12)). These variations in MOA footprints were correlated with allele-specific transcript levels of _ZmTINY_. In F1s under DS conditions, mRNA transcripts of the CML333 and Oh43 alleles were 84-fold and 18-fold more abundant than B73 transcripts, respectively (Fig. [7b](https://www.nature.com/articles/s41588-025-02246-7#Fig7) and Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02246-7#Fig12)). MOA signals in the B73 and CML333 upstream promoter showed the highest correlation to _ZmTINY_ mRNA levels (Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02246-7#Fig12)). We tested these sequences in a dual-luciferase expression assay with and without ABA treatment to simulate DS. Both promoter fragments exhibited significantly higher LUC/REN ratios than the vector control. Consistent with trends observed for MOA and mRNA levels (Fig. [7b,c](https://www.nature.com/articles/s41588-025-02246-7#Fig7)), prom::_ZmTINY_CML333 showed a higher LUC/REN ratio than prom::_ZmTINY_B73 under WW conditions, and exogenous application of 1 µM and 10 µM ABA further increased the LUC/REN ratio significantly for protoplasts harboring the prom::_ZmTINY_CML333 but not prom::_ZmTINY_B73 fragment by 41.4% and 60.3%, respectively. Together, the results support previous findings of _ZmTINY_ as a drought candidate gene and indicate that bQTL can identify _cis_\-regulatory regions that act condition-dependently. That said, the drought-responsive regulation of _ZmTINY_ may include additional regulatory sequences, such as the drought-responsive loci downstream.

**Fig. 7: MOA-seq detects drought-responsive _cis_\-regulatory loci.**

[![figure 7](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_Fig7_HTML.png)](https://www.nature.com/articles/s41588-025-02246-7/figures/7)

**a**, Average (three biological replicates), normalized MOA coverage for B73 × CML333 near _ZmTINY_ under WW and DS conditions. Roman numerals highlight respective DS-peak regions identified (_P_ < 0.05, MACS3) in each line. **b**, Haplotype-specific mRNA counts of _ZmTINY_ for B73 × CML333 (C333) under WW and DS conditions (three biological replicates). The zoomed-in square highlights (left to right) WW-B73, WW-CML333 and DS-B73. **c**, Relative luciferase levels in maize leaf protoplasts harboring either ~0.5 kb of the B73 or CML333 proximal promoter allele upstream of the _ZmTINY_ TSS, with and without 1–100 µM ABA treatment. Letters represent significant differences (_P_ < 0.05, ANOVA with Tukey test; [Source data](https://www.nature.com/articles/s41588-025-02246-7#Sec31)). **d**, Heatmap of MOA allelic bias at AMP loci under WW and DS conditions. AMPs (under WW, DS or both) in drought-responsive MOA peaks (_P_ < 0.05; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)) are displayed for B73 × Oh43. Color scale ranging from green (100% bias towards B73) to blue (100% bias towards Oh43); gray represents MOA signal below the detection limit (MACS3; [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)). Clusters represent (I) allele-specific occupancy in one condition and below detection limit in the other, (II) allele-specific occupancy with consistent bias under both conditions, (III) allele-specific occupancy with bias in the opposite direction under the two conditions and (IV) occupancy with a significant allele-specific bias under only one condition. Only sites with significant allele-specific bias (binomial testing, FDR, 1%; for details see [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)) in at least one condition were considered. To avoid additional statistical cut-off effects for the second condition, 60% or more occupancy bias towards one allele was considered allele-specific. Boxplots in **b** and **d** were generated as described in Fig. [3](https://www.nature.com/articles/s41588-025-02246-7#Fig3).

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM12)

Differences in MOA-seq coverage between WW and DS conditions at allele-specific sites could be caused by changes in occupancy level, the direction of allelic bias or a combination thereof. To better understand which scenario is more common, we clustered drought-responsive AMPs in the B73 × Oh43 F1 (11,970 AMPs located in drought-responsive footprints). The results showed that 83% of drought-responsive AMPs changed MOA occupancy between WW and DS conditions, either from no detectable MOA signal to haplotype-specific binding (~45%, group I), or in the amount of MOA coverage between WW and DS conditions while maintaining their allelic bias (~38%, group II) (Fig. [7d](https://www.nature.com/articles/s41588-025-02246-7#Fig7)). By contrast, only about 17% of AMPs showed bias changes, either from no significant bias in one condition to a significant bias in the other (~15%, group IV) or changing the direction of the allelic bias (~2%, group III) (Fig. [7d](https://www.nature.com/articles/s41588-025-02246-7#Fig7)). Although groups I and IV are somewhat dependent on statistical cut-offs (peak calling and thus AMP definition and/or calling allelic bias), groups II and III show an allele-specific bias under both conditions. Focusing on groups II and III, it becomes evident that changes in allelic bias are ~20-fold less frequent compared to the constant binding bias accompanied by overall changes in MOA signal. Similar clusters between WW and DS conditions were observed for AMPs in all 25 F1s (Supplementary Fig. [19](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). We therefore propose that the majority of DS-induced TF occupancy dynamics at sites of functional genetic variation results from condition-specific TF abundance changes rather than changes in allelic bias between WW and DS conditions.

Discussion
----------

The gene regulatory landscape involves primary sequence, chromatin accessibility and DNA and protein modifications[10](https://www.nature.com/articles/s41588-025-02246-7#ref-CR10 "Levo, M. & Segal, E. In pursuit of design principles of regulatory sequences. Nat. Rev. Genet. 15, 453–468 (2014)."). Although our ability to assemble complex genomes has made great progress, decoding gene regulation, population-wide, high-resolution maps of the regulatory loci and efficient pinpointing of functional variation remain elusive in plants[5](https://www.nature.com/articles/s41588-025-02246-7#ref-CR5 "Marand, A. P., Eveland, A. L., Kaufmann, K. & Springer, N. M. Cis-regulatory elements in plant development, adaptation, and evolution. Annu. Rev. Plant Biol. 74, 111–137 (2023).").

We present a robust, high-throughput method for identifying functional variants, genetic, epigenetic, or both, linked to trait variation in plants. By integrating haplotype-specific TF footprints and transcript abundance, F1 hybrids and local association mapping at putative _cis_\-element loci, we defined a pan-cistrome of the maize leaf under WW and DS conditions. Use of concatenated dual-reference genomes and F1 hybrid analysis resolved issues of reference bias, _trans_\-effects and technical variation that commonly compromise haplotype-specific quantitation. Although MOA-seq footprints with their high-resolution (~100 bp) and comprehensive cistrome-wide analysis are well suited for this method, similar results may be obtained by, for example, allele-specific ATAC-seq (with putatively a lower fraction of the genome covered owing to the large size of the Tn5 dimer[11](https://www.nature.com/articles/s41588-025-02246-7#ref-CR11 "Savadel, S. D. et al. The native cistrome and sequence motif families of the maize ear. PLoS Genet. 17, e1009689 (2021)."),[13](https://www.nature.com/articles/s41588-025-02246-7#ref-CR13 "Zhao, H. et al. Genome-wide MNase hypersensitivity assay unveils distinct classes of open chromatin associated with H3K27me3 and DNA methylation in Arabidopsis thaliana. Genome Biol. 21, 24 (2020).")) or for single TFs using allele-specific ChIP–seq, as we demonstrated previously in one F1 hybrid[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).").

Our analysis demonstrates a high level of variation in _cis_\-regulatory networks among 25 diverse maize genotypes and provides a high-resolution map of regulatory elements underpinning the function of over 200,000 putative _cis_\-element loci in the maize leaf. We note that the high genetic diversity between maize inbred lines allowed us to detect variants in 25 F1 lines. For species with lower diversity, more F1s or the inclusion of more distant interspecies hybrids might be necessary. Finally, we highlight the relevance of genotype-associated bQTL for understanding phenotypic diversity in maize, demonstrating that haplotype-specific MOA-seq in leaves allowed us to capture the majority of additive genetic variation for most tested phenotypes.

Methods
-------

### Statistics and reproducibility

Experiments were conducted with three biological replicates unless otherwise stated. Pan-cistrome analysis was performed on 26 genomes (25 paternal and one maternal genome). Data collection and analysis were not performed blind to the conditions of the experiments. For randomization of plant positions of drought-treated plants, see the plant materials section. No data were excluded from the analyses. Data distribution was assumed to be normal, but this was not formally tested.

### Plant materials

The GRIN National Agricultural Library supplied B73, Mo17, A619, W22, A188 and US-NAM seeds. Seeds were pre-germinated for 48 h at 28–30 °C. Each pot contained soil equalized by volume and four seedlings (Einheitserde VMV800/D373 soil). Plants were grown in a greenhouse using a randomized block design, under long-day conditions (16 h day, 8 h night, 28–30 °C) for approximately 26 days until 75% of the plants per genotype showed the formation of the leaf four auricle. Plants were then randomized, and 12 plants (three pots) per treatment and replicate were grown with or without periodic watering through a bottom drench system for 86 h. Plants were then harvested, and the leaf blades of the oldest leaf without a yet-formed auricle were immediately frozen in liquid nitrogen. Additionally, the relative water content and soil water content were measured (Supplementary Note [2](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). No statistical methods were used to pre-determine sample sizes, but our sample sizes are similar to those reported in previous publications[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).").

### MOA-seq and RNA-seq sample preparation and sequencing

MOA-seq and RNA-seq sample and library preparation were performed as previously described[11](https://www.nature.com/articles/s41588-025-02246-7#ref-CR11 "Savadel, S. D. et al. The native cistrome and sequence motif families of the maize ear. PLoS Genet. 17, e1009689 (2021)."),[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023)."); for details, see Supplementary Note [3](https://www.nature.com/articles/s41588-025-02246-7#MOESM1).

### MOA-seq data analysis

Reads were filtered using SeqPurge[55](https://www.nature.com/articles/s41588-025-02246-7#ref-CR55 "Sturm, M., Schroeder, C. & Bauer, P. SeqPurge: highly-sensitive adapter trimming for paired-end NGS data. BMC Bioinformatics 17, 208 (2016).") (v.2022-07-15) with parameters ‘-min\_len 20 -qcut 0’. Owing to the short fragment length in MOA, read pairs almost completely overlapped. MOA-seq paired-end reads were merged into single-end reads, including base quality score correction, using NGmerge[56](https://www.nature.com/articles/s41588-025-02246-7#ref-CR56 "Gaspar, J. M. NGmerge: merging paired-end reads via novel empirically-derived models of sequencing errors. BMC Bioinformatics 19, 536 (2018).") (v.0.3) with parameters ‘-p 0.2 -m 15 -d -e 30 -z -v’. Diploid genomes were created by concatenating the B73 v5 genome with the respective paternal genome (NAM v1/2 genomes[18](https://www.nature.com/articles/s41588-025-02246-7#ref-CR18 "Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science 373, 655–662 (2021)."), Mo17 CAU v1, W22 v2 (ref. [20](https://www.nature.com/articles/s41588-025-02246-7#ref-CR20 "Springer, N. M. et al. The maize W22 genome provides a foundation for functional genomics and transposon biology. Nat. Genet. 50, 1282–1288 (2018).")), A188 v1 (ref. [19](https://www.nature.com/articles/s41588-025-02246-7#ref-CR19 "Lin, G. et al. Chromosome-level genome assembly of a regenerable maize inbred line A188. Genome Biol. 22, 175 (2021).")) and A619, with accession ID added as a prefix to the chromosome name; Supplementary Note [4](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). Reads were mapped to the diploid genome (or the separate genomes for inbred data) using STAR[57](https://www.nature.com/articles/s41588-025-02246-7#ref-CR57 "Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013).") (v.2.7.7a). STAR was designed to map RNA; therefore, we used the flag–alignIntronMax 1 for DNA (no introns allowed) as well as parameters ‘–outSAMmultNmax 2,–winAnchorMultimapNmax 100’ and ‘-outBAMsortingBinsN 5’. We generated two datasets: one in which reads were only allowed to map once in the diploid genome (mapping quality 255, used to generate MPs and AMPs data) and one in which reads mapped exactly twice, with double mapping reads being randomly assigned to one of the two positions (used for visualization and overall peak coverage data). Format conversion and calculation of the average mapped fragment length (AMFL) were done using SAMtools[58](https://www.nature.com/articles/s41588-025-02246-7#ref-CR58 "Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079 (2009).") (v.1.9). The effective genome size was calculated using unique-kmers.py ([https://github.com/dib-lab/khmer](https://github.com/dib-lab/khmer), commit fb65d21), with AFML and respective genome fasta as inputs. The deeptools[59](https://www.nature.com/articles/s41588-025-02246-7#ref-CR59 "Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 44, W160–W165 (2016).") (v.3.5.0) function bamCoverage was used to generate normalized (reads per genome coverage (RPGC)) bedgraph files of full-length read data.

Fragment-center tracks were generated as previously described[60](https://www.nature.com/articles/s41588-025-02246-7#ref-CR60 "Liang, Z. et al. Mapping responsive genomic elements to heat stress in a maize diversity panel. Genome Biol. 23, 234 (2022)."): bam files were converted to bed format using bamToBed of Bedtools (v.2.29.0)[61](https://www.nature.com/articles/s41588-025-02246-7#ref-CR61 "Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010)."), and each mapped read was shortened to 20 bp centered around the middle of the read using awk; reads with an uneven number of bases were extended 10 bp to each site from the middle of the read. One of the two middle bases was chosen at random for reads with even number of bases, and reads were extended 10 bp to each site. The function genomeCoverageBed of Bedtools was then used to convert the bed files to bedgraph, scaled by the quotient of the effective genome size and the number of uniquely mapped reads (similar to RPGC of deeptools bamCoverage). BigWig files for visualization were generated using bedGraphToBigWig (v.4)[62](https://www.nature.com/articles/s41588-025-02246-7#ref-CR62 "Kent, W. J., Zweig, A. S., Barber, G., Hinrichs, A. S. & Karolchik, D. BigWig and BigBed: enabling browsing of large distributed datasets. Bioinformatics 26, 2204–2207 (2010).").

### MP and AMP identification

To enable translation between coordinates of the B73 genome and the paternal genomes, hal files were generated using the cactus function of progressive cactus[63](https://www.nature.com/articles/s41588-025-02246-7#ref-CR63 "Armstrong, J. et al. Progressive Cactus is a multiple-genome aligner for the thousand-genome era. Nature 587, 246–251 (2020).") (v.1.0.0, 2020-04-19) with standard parameters. SNPs between B73 and the paternal lines were determined with the halSnps function of cactus, using parameters ‘unique’ and ‘noDupes’. From the resulting SNP lists, we selected all SNPs that carried either the B73 or one other base in all analyzed lines (biallelic SNPs, 117,898,189). Of the remaining SNPs, only those occurring in at least two of the 25 parental lines were retained (60,432,443; minor allele frequency, 0.08). For allele-specific analysis, B73 coordinates of the filtered, biallelic SNPs were translated back to paternal coordinates using halLiftover (hal-release-V2.1), and all SNPs with ambiguous corresponding positions in one of the two parental genomes were removed (de-duplicated biallelic SNPs in at least two lines, 58,823,746). At each of these SNP positions, we counted RPGC values for both alleles using bedtools map (bedtools v.2.29.0) and calculated the read numbers corresponding to the RPGC numbers for further calculation (for example, binomial testing was performed on read numbers, not normalized values). Binding frequencies at SNP positions were determined as RPGC-B73 / (RPGC-B73 + RPGC-Pat). We defined MPs as SNPs that were located within MOA peaks and had more than seven RPGC (approximately >25 reads) for at least one allele and at least one read in the corresponding allele. Allele-specific binding at MPs (significant deviation from the expected 0.5 binding frequency) was determined by binomial testing in R (v.4.1.1). SNP positions with an FDR-corrected _P_ value of <0.01 were considered AMPs. Additionally, we determined the allelic ratio of WGS control reads (Supplementary Note [5](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)) in a 65 bp window at all MPs, and excluded all AMPs with a WGS ratio above or below the upper and lower fifth percentile value of all MPs, respectively.

### Peak calling

For peak calling, MOA bam files were used with MACS3 (v.3.0.1, [https://github.com/macs3-project/MACS](https://github.com/macs3-project/MACS)) using the following parameters: -s and–min-length ‘AMFL’,–max-gap ‘2x AMFL’, –nomodel, –extsize ‘AMFL’,–keep-dup all, -g ‘effective genome size’, where AMFL represents average MOA fragment length, calculated with SAMtools stats using default parameters.

### Treatment-specific peak calling

MOA-alignment bam files were converted to bed format using bedtools bamToBed (v.2.29.0). The genomeCoverage function of bedtools was used to convert pooled replicated bed files to bedgraph with the reads per million scaling factor. The reads-per-million-normalized coverage difference between treatments was calculated using the intersect and subtract functions of bedtools. The resulting differences in coverage counts for WW and DS treatments were used to create an unbinned (1 bp bin) bed file to produce a bigwig coverage track, which was used as input for MACS3 (v.3.0.1) peak calling, using parameters: –min-length 30,–max-gap 60, –nomodel, –extsize 1,–keep-dup all, -g ‘effective genome size’, -q 0.01. Significant differences between WW and DS peaks were determined by a two-sided Welch _t_\-test using the individual bio replicate coverages, and peaks with _P_ < 0.05 were retained.

### Transient luciferase assay

Protoplasts were isolated and transformed by electroporation as previously described[64](https://www.nature.com/articles/s41588-025-02246-7#ref-CR64 "Gomez-Cano, L., Yang, F. & Grotewold, E. Isolation and efficient maize protoplast transformation. Bio Protoc. 
https://doi.org/10.21769/BioProtoc.3346
(2019).") (Supplementary Note [6](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)) using 10 µg of a plasmid encoding firefly luciferase downstream of the respective B73 or CML333 prom::_TINY_ alleles (B73 fragment +570 bp upstream of ATG or CML333 + 451 bp upstream of ATP; primers in Supplementary Table [12](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)), along with 5 µg of a plasmid containing 35S-renilla luciferase. For ABA treatments, a 20 mM stock solution of (+)-Cis, Trans-Abscisic Acid (Duchefa Biochemie, cat. no. A0941) in ethanol was prepared. Round-bottom 2 ml microcentrifuge tubes were pre-loaded with 50 µl of ABA solution in protoplast buffer[64](https://www.nature.com/articles/s41588-025-02246-7#ref-CR64 "Gomez-Cano, L., Yang, F. & Grotewold, E. Isolation and efficient maize protoplast transformation. Bio Protoc. 
https://doi.org/10.21769/BioProtoc.3346
(2019).") (Supplementary Note [6](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)), achieving the required ABA concentration upon the addition of 950 µl of electroporated protoplasts. After transformation, the protoplasts were incubated for 18–22 h in the dark for recovery. The cells were sedimented for 2 min at 260_g_ at room temperature and resuspended in 80 µl of 1× Passive Lysis Buffer (Promega, cat. no. E1941). Cells were disrupted by vortexing for 10 min, and cell lysates were cleared by centrifugation for 10 min at 12,000_g_ at 4 °C.

The dual-luciferase assay was performed as previously described[65](https://www.nature.com/articles/s41588-025-02246-7#ref-CR65 "Hampf, M. & Gossen, M. A protocol for combined Photinus and Renilla luciferase quantification compatible with protein assays. Anal. Biochem. 356, 94–99 (2006).") (Supplementary Note [7](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). All experiments were conducted with three biological and three technical replications. Values were calculated by dividing the activity of firefly by that of renilla luciferase.

### DNA methylation analysis

Parental DNA methylation data of the NAM lines[18](https://www.nature.com/articles/s41588-025-02246-7#ref-CR18 "Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science 373, 655–662 (2021).") were obtained from iPlant ([/iplant/home/maizegdb/maizegdb/NAM\_PROJECT\_JBROWSE\_AND\_ANALYSES](https://datacommons.cyverse.org/browse/iplant/home/maizegdb/maizegdb/NAM_PROJECT_JBROWSE_AND_ANALYSES)). Methylation data for non-NAM lines[19](https://www.nature.com/articles/s41588-025-02246-7#ref-CR19 "Lin, G. et al. Chromosome-level genome assembly of a regenerable maize inbred line A188. Genome Biol. 22, 175 (2021)."),[66](https://www.nature.com/articles/s41588-025-02246-7#ref-CR66 "Liang, Z. et al. Genetic and epigenetic variation in transposable element expression responses to abiotic stress in maize. Plant Physiol. 186, 420 (2021).") were obtained as SRA archives (Bioprojects [PRJNA657677](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA657677) and [PRJNA635654](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA635654)) and processed as previously described[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).") (Supplementary Note [8](https://www.nature.com/articles/s41588-025-02246-7#MOESM1)). B73 × Mo17 hybrid methylation data were previously published[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023).") and showed strong correlation with parental methylation at B73 × Mo17 AMPs (Extended Data Fig. [2a–d](https://www.nature.com/articles/s41588-025-02246-7#Fig9)). Context-specific methylation around AMPs and MPs was determined separately for the B73 and paternal alleles in the three sequence contexts (CG, CHG or CH) as the averaged methylation levels within a window of ±20 bp around the position as previously described[17](https://www.nature.com/articles/s41588-025-02246-7#ref-CR17 "Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. Genome Biol. 24, 108 (2023)."). Significant differences in DNA methylation were determined following a previous publication[39](https://www.nature.com/articles/s41588-025-02246-7#ref-CR39 "Regulski, M. et al. The maize methylome influences mRNA splice sites and reveals widespread paramutation-like switches guided by small RNA. Genome Res. 23, 1651–1662 (2013).") (one allele <10% methylated and the other >70%). Sites for testing the consistency of DNA methylation or haplotype-specific binding relations among the F1 hybrids were selected based on having at least two F1 lines differentially methylated, at least two F1 lines equally methylated and at least one F1 line AMP at the given site. In this analysis (Fig. [4c](https://www.nature.com/articles/s41588-025-02246-7#Fig4)), a more stringent definition of equal methylation (as opposed to not being differentially methylated) was used: equal methylation was defined as both alleles <10% or both >70% methylated.

### Local association mapping to map bQTL

The binding ratio of the MOA peaks, as well as methylation ratio information for mCG, mCHG and mCHH, were collected separately for all hybrids for the WW and DS conditions. The binding frequency at loci with no reads was set to ‘NA’. Genotyping information (GTi) at sequence variants and the methylation ratio information were used to conduct local association studies using five different linear models for each MP. All MPs with the respective haplotype-specific MOA coverage (binding frequency) and average surrounding (±20 bp) methylation ratios were considered:

${\\rm{MP}}={\\rm{mCHH}}$

${\\rm{MP}}={\\rm{mCG}}$

${\\rm{MP}}={\\rm{mCHG}}$

${\\rm{MP}}={{\\rm{GT}}}\_{{\\rm{i}}}$

${\\rm{MP}}={{\\rm{GT}}}\_{{\\rm{i}}}+{\\rm{mCG}}+{\\rm{mCHG}}+{\\rm{mCHH}}$

The analyses were performed in Julia (v.1.8.1) and R (v.4.4.1). Associated MPs at a FDR of 5% were selected (R v.4.1.2). bQTL located within 65 bases were combined into linkage groups (the lowest _P_ value determined the lead bQTL).

### Analysis of MPs at INDELs

A list of candidate INDELs was generated through pairwise whole-genome alignment of each of the 25 inbred parental genomes to the common mother’s, B73, genome. Alignments were created using Anchorwave’s (v.1.2.2) proali function[67](https://www.nature.com/articles/s41588-025-02246-7#ref-CR67 "Song, B. et al. AnchorWave: sensitive alignment of genomes with high sequence diversity, extensive structural polymorphism, and whole-genome duplication. Proc. Natl Acad. Sci. USA 119, e2113075119 (2022)."), with anchor regions determined with minimap2 (ref. [68](https://www.nature.com/articles/s41588-025-02246-7#ref-CR68 "Li, H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics 34, 3094–3100 (2018).")) (v.2.27-r1193). Variants were called from the alignment using wgatools’ (v.0.1.0[69](https://www.nature.com/articles/s41588-025-02246-7#ref-CR69 "Wei, W. et al. wgatools: an ultrafast toolkit for manipulating whole-genome alignments. Bioinformatics 41, btaf132 (2025).")) call function using the -s and -l 1 parameters to call variants of any size. For later ease of handling, variants were extracted from the created variant call format file and written into BED (browser extensible data) file format. Liftover of coordinates from B73 to each of the other 25 parental genomes was facilitated with CrossMap[70](https://www.nature.com/articles/s41588-025-02246-7#ref-CR70 "Zhao, H. et al. CrossMap: a versatile tool for coordinate conversion between genome assemblies. Bioinformatics 30, 1006–1007 (2014).") (v.0.7.0), using chain files created with wgatools maf2chain[69](https://www.nature.com/articles/s41588-025-02246-7#ref-CR69 "Wei, W. et al. wgatools: an ultrafast toolkit for manipulating whole-genome alignments. Bioinformatics 41, btaf132 (2025)."). The following steps were performed using custom code as well as the bedtools suite[61](https://www.nature.com/articles/s41588-025-02246-7#ref-CR61 "Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010).") (v.2.30). From the list of variants, a set of INDELs between 2 and 50 bp in length and biallelic in the population was extracted. In addition, this set was filtered so that the non-B73 allele occurs at least twice in the population. Synteny analysis of INDELs by whole-genome alignment across genomes is challenging; hence, we tried to minimize ambiguously mapped INDELs with additional filters[71](https://www.nature.com/articles/s41588-025-02246-7#ref-CR71 "Kok, A., Schneider, M., Hartwig, T. & Engelhorn, J. FIND-CIS-analysis. Zenodo 
https://doi.org/10.5281/zenodo.15212007
(2025).").

RPGC values for B73 and NAM INDEL regions were determined using bedtools intersect (-wao parameter; bedtools v.2.29.0) on bedgraphs containing the normalized read counts determined as described above and the bed file containing the INDEL positions. The -wao function returns, for each bed entry, the overlapping bedgraph entries, including the length of the overlap, even if the entry count is zero. From this result, custom awk commands were used to calculate the average RPGC (sum over all bases/length of region, including bases with zero count) for the B73 and parental allele. For each deletion allele, average RPGC values were calculated over a window of 3 bp before and 3 bp after the deletion. For each insertion allele, average RPGC values were calculated over the whole insertion itself. After this step, counts were treated in the same way as SNP counts. The same methylation data and analysis steps were used for INDELs as for SNPs (see above), with methylation being calculated in a window from 20 bp upstream of the insertion or deletion start to 20 bp downstream of the insertion or deletion end coordinate. Methylation and count data were then used to perform bQTL analysis in the same way as described for SNPs.

### RNA-seq analysis

RNA-seq data were mapped to the concatenated diploid genomes using STAR (v.2.7.7a), with options –outSAMmultNmax 1, –outFilterMultimapNmax 1, –winAnchorMultimapNmax 100, –twopassMode Basic, –outFilterIntronMotifs RemoveNoncanonical, –outFilterType BySJout, –quantMode GeneCounts, using a concatenated gff3 file containing gene models from both parents. To determine allele-specific transcript abundance, for each line, B73 and corresponding paternal positions for all SNPs determined by halSnps were generated by halLiftover. Of the resulting position pairs, ambiguous ones (mapping to more than one position in one of the genomes) were removed. Each SNP was then assigned the B73 genes it overlaps. The respective NAM gene info was added using a Pan-gene file (downloaded from MaizGDB), retaining strand information in both cases. Mapped read information was converted into read bed files using bamToBed, and each SNP was assigned all reads overlapping with it in B73 and at the parental genome coordinates (strand-specific, separately for the three replicates). Only SNPs carrying reads in both alleles were retained to ensure that the SNP was truly located within the gene in both alleles. Afterwards, reads for each gene were counted per replicate (reads that had two or more SNPs were counted only once) and allele (Supplementary Table [13a,b](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). For A188, for which no Pan-gene entries were available, SNPs were also mapped onto the A188 gff3, and gene pairs were generated based on shared SNP positions. In this way, B73 reads and paternal reads could be compared for differential transcript abundance analysis in DEseq2 (ref. [72](https://www.nature.com/articles/s41588-025-02246-7#ref-CR72 "Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014).")) in R (v.4.1.1). Genes with an FDR-corrected _P_ value of <0.05 were considered to have ASE in their transcript abundance.

### Variance component analysis pipeline

To run the variance component analysis pipeline (VCAP), we required three datasets: genome-wide markers across the NAM population recombinant inbred lines (RILs); trait values across NAM RILs for each trait analyzed; and coordinates for MOA peaks or bQTL SNPs across founder lines to partition each component. For the genome-wide markers, we used publicly available resequencing SNPs from the NAM founders[18](https://www.nature.com/articles/s41588-025-02246-7#ref-CR18 "Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science 373, 655–662 (2021).") that had been projected onto the NAM RILs ([/iplant/home/shared/NAM/Misc/NAM-SV-projected-V8](https://datacommons.cyverse.org/browse/iplant/home/shared/NAM/Misc/NAM-SV-projected-V8)). Trait data collected from the NAM RILs (_n_ = 143) were curated from previous publications[73](https://www.nature.com/articles/s41588-025-02246-7#ref-CR73 "Khaipho-Burch, M. et al. Elucidating the patterns of pleiotropy and its biological relevance in maize. PLoS Genet. 19, e1010664 (2023).") (Supplementary Table [14](https://www.nature.com/articles/s41588-025-02246-7#MOESM5)). We used two sampling schemes to create our MOA partitions. First, only the bQTL SNPs with significant association to the genotype, not methylation, were used to represent MOA. Second, bQTL associated with genotype and methylation at the same time were included. Any SNP outside of the bQTL SNPs created the non-MOA pool from which the bgSNPs were drawn. Given the non-random distribution of bQTLs throughout the genome, we also included a matched background component: each bQTL SNP was matched to a random non-bQTL SNP by allele frequency (number of lines containing the alt allele / total lines without missing data at that position, 0.1 bin size) and distance from the nearest gene (TSS or transcription termination site as calculated by bedtools[61](https://www.nature.com/articles/s41588-025-02246-7#ref-CR61 "Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010).") closest -d).

This matched set of MOA and bgSNPs, equal in number to SNPs with similar genomic contexts, was used for a single VCAP run. Kinship matrices were created for the bQTL SNPs, bgSNPs and the rest of the genome (remaining non-bQTL and non-bgSNPs) using Tassel (v.5)[74](https://www.nature.com/articles/s41588-025-02246-7#ref-CR74 "Bradbury, P. J. et al. TASSEL: software for association mapping of complex traits in diverse samples. Bioinformatics 23, 2633–2635 (2007)."). To calculate the heritabilities of all 143 traits, the set of three kinship matrices and traits was run through a REML model using LDAK (v.5.2)[75](https://www.nature.com/articles/s41588-025-02246-7#ref-CR75 "Speed, D., Hemani, G., Johnson, M. R. & Balding, D. J. Improved heritability estimation from genome-wide SNPs. Am. J. Hum. Genet. 91, 1011–1021 (2012)."). We sampled 100 times, creating 100 permutations of kinship matrix sets. Thus, the permutations gave us a range of heritability estimates that could result from these particular components, traits and the population (Fig. [5b](https://www.nature.com/articles/s41588-025-02246-7#Fig5)). The same bQTL SNPs were used in every permutation, whereas the bgSNPs differed across permutations.

To evaluate the reliability of our heritability estimation method, we simulated traits with defined contributions from specific sets of kinship matrices and compared estimates of the heritabilities generated by the above VCAP protocol. We used one of our previously generated kinship matrix sets (one SNP per peak sampling) to simulate traits assigned certain heritabilities for each component (four sets of heritabilities, ten traits per set). We simulated traits as the sum of four normally distributed random vectors, each with zero mean and covariance equal to one of the three kinship matrices or the identity matrix (for residual variation) multiplied by a specific heritability value. The simulated traits and kinship matrices were used in the REML modeling step to estimate heritabilities. Estimated heritabilities were then compared to known heritabilities. All scripts written for the analyses in the study were deposited at [https://github.com/Snodgras/MOA\_Analysis](https://github.com/Snodgras/MOA_Analysis).

### MOA bQTL and eQTL linkage analysis

Linkage disequilibrium was calculated between the binding QTL reported in this study and a set of 10,618 _cis_\-eQTL identified based on expression data of the roots of 340 maize genotypes[38](https://www.nature.com/articles/s41588-025-02246-7#ref-CR38 "Sun, G. et al. A role for heritable transcriptomic variation in maize adaptation to temperate environments. Genome Biol. 24, 55 (2023)."). Genomic coordinates of the 78,398 binding QTL on the B73\_RefGen\_V5 maize genome were converted to B73\_RefGen\_V4 positions using CrossMap (v.0.6.4) as implemented in EnsemblPlants[70](https://www.nature.com/articles/s41588-025-02246-7#ref-CR70 "Zhao, H. et al. CrossMap: a versatile tool for coordinate conversion between genome assemblies. Bioinformatics 30, 1006–1007 (2014)."),[76](https://www.nature.com/articles/s41588-025-02246-7#ref-CR76 "Howe, K. L. et al. Ensembl Genomes 2020—enabling non-vertebrate genomic research. Nucleic Acids Res. 48, D689–D695 (2020)."). A total of 99.4% of bQTL positions could be successfully converted to B73\_RefGen\_V4 positions, and of these, 38,291 were present in a set of 12,191,984 genetic markers segregating in the population of 340 maize lines used to conduct eQTL analysis with a minor allele frequency of ≥0.05 and less than 2% of genotypes exhibiting heterozygous genotype calls. Linkage disequilibrium was calculated between bQTL markers and _cis_\-eQTL markers in all cases in which a _cis_\-eQTL and a bQTL were located within 10 kbp of each other, using genotype calls from the 340 maize varieties[38](https://www.nature.com/articles/s41588-025-02246-7#ref-CR38 "Sun, G. et al. A role for heritable transcriptomic variation in maize adaptation to temperate environments. Genome Biol. 24, 55 (2023)."),[77](https://www.nature.com/articles/s41588-025-02246-7#ref-CR77 "Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007)."). To assemble a control set of genetic markers with the same properties as the bQTL, bQTL that were successfully converted to B73\_RefGen\_v4 and matched to Sun et al.[38](https://www.nature.com/articles/s41588-025-02246-7#ref-CR38 "Sun, G. et al. A role for heritable transcriptomic variation in maize adaptation to temperate environments. Genome Biol. 24, 55 (2023).") markers were divided into ten bins based on their distance from the closest annotated TSS (0–1 kbp, 1–2 kbp and so on), plus two additional categories for intragenic SNPs and SNPs > 10 kbp from the nearest annotated gene. A random subset of two million B73\_RefGen\_v5 SNPs used to detect bQTL were also converted to B73\_RefGen\_v4 and matched to segregating markers from Sun et al.[38](https://www.nature.com/articles/s41588-025-02246-7#ref-CR38 "Sun, G. et al. A role for heritable transcriptomic variation in maize adaptation to temperate environments. Genome Biol. 24, 55 (2023)."), as described above. These markers were subsampled to create a second set of 38,291 control markers with representation in each of the 12 bins equal to the levels observed for the real bQTL.

### Further data processing

To obtain the high-confidence list of drought-responsive MOA regions, all MOA bQTL (unclumped WW or DS) were filtered for overlap with AMPs located in regions with significantly (_P_ < 0.05) increased or reduced MOA occupancy between DS and WW conditions in at least two F1s (overlap within 65 bp). A total of 3,198 and 11,060 drought-induced and repressed loci, respectively, were retained.

Comparisons and calculations of lists were either performed in bedtools intersect or with custom awk and bash scripts. Hypergeometric tests for over-representation or under-representation, ANOVA and data visualization were performed in R. Pearson correlation coefficients of bigwig file format MOA-seq data were calculated and visualized using the multiBigwigSummary and plotCorrelation functions of deepTools[59](https://www.nature.com/articles/s41588-025-02246-7#ref-CR59 "Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 44, W160–W165 (2016).") with a window size of 1,000 bases.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02246-7#MOESM2) linked to this article.

Data availability
-----------------

All raw MOA-seq and RNA-seq data discussed in this publication have been deposited at NCBI SRA under accession number [PRJNA1101486](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1101486). MOA coverage tracks and peak files have been deposited to the Gene Expression Omnibus under accession number [GSE294039](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE294039). Coordinates in processed data files are based on the concatenated genomes (chromosome names: LineID-chr), which, for convenience, were deposited at Zenodo ([https://doi.org/10.5281/zenodo.15177272](https://doi.org/10.5281/zenodo.15177272) (ref. [78](https://www.nature.com/articles/s41588-025-02246-7#ref-CR78 "Engelhorn, J., Hartwig, T., Kok, A., Stich, B. & Schneider, M. Binding QTL data and concatenated versions of maize genomes. Zenodo 
https://doi.org/10.5281/zenodo.15177272
(2025)."))). Coverage and binding frequency data for all bQTL is accessible at maizegdb ([https://jbrowse.maizegdb.org](https://jbrowse.maizegdb.org/)), at a custom browser ([https://www.plabipd.de/ceplas/?config=maize\_hartwig\_config.json](https://www.plabipd.de/ceplas/?config=maize_hartwig_config.json&session=local-R7uRvOZKNy9uyYx-ruAIE)) and at Zenodo ([https://doi.org/10.5281/zenodo.15177272](https://doi.org/10.5281/zenodo.15177272) (ref. [78](https://www.nature.com/articles/s41588-025-02246-7#ref-CR78 "Engelhorn, J., Hartwig, T., Kok, A., Stich, B. & Schneider, M. Binding QTL data and concatenated versions of maize genomes. Zenodo 
https://doi.org/10.5281/zenodo.15177272
(2025)."))). Previously published datasets used in this study include [/iplant/home/maizegdb/maizegdb/NAM\_PROJECT\_JBROWSE\_AND\_ANALYSES](https://datacommons.cyverse.org/browse/iplant/home/maizegdb/maizegdb/NAM_PROJECT_JBROWSE_AND_ANALYSES) ref. [18](https://www.nature.com/articles/s41588-025-02246-7#ref-CR18 "Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science 373, 655–662 (2021)."), SRA accession numbers [PRJNA961163](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA961163) (ref. [14](https://www.nature.com/articles/s41588-025-02246-7#ref-CR14 "Manosalva Pérez, N. et al. MINI-AC: inference of plant gene regulatory networks using bulk or single-cell accessible chromatin profiles. Plant J. 117, 280–301 (2024).")), [PRJNA657677](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA657677) (ref. [66](https://www.nature.com/articles/s41588-025-02246-7#ref-CR66 "Liang, Z. et al. Genetic and epigenetic variation in transposable element expression responses to abiotic stress in maize. Plant Physiol. 186, 420 (2021).")), [PRJNA635654](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA635654) (ref. [19](https://www.nature.com/articles/s41588-025-02246-7#ref-CR19 "Lin, G. et al. Chromosome-level genome assembly of a regenerable maize inbred line A188. Genome Biol. 22, 175 (2021).")), [PRJNA311133](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA311133) (ref. [20](https://www.nature.com/articles/s41588-025-02246-7#ref-CR20 "Springer, N. M. et al. The maize W22 genome provides a foundation for functional genomics and transposon biology. Nat. Genet. 50, 1282–1288 (2018).")), [PRJEB31061](https://www.ncbi.nlm.nih.gov/bioproject/600447) (ref. [18](https://www.nature.com/articles/s41588-025-02246-7#ref-CR18 "Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science 373, 655–662 (2021).")), [PRJNA10769](https://www.ncbi.nlm.nih.gov/bioproject/10769) (ref. [79](https://www.nature.com/articles/s41588-025-02246-7#ref-CR79 "Jiao, Y. et al. Improved maize reference genome with single-molecule technologies. Nature 546, 524–527 (2017).")), [PRJNA540700](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA540700), [PRJNA565870](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA565870), [PRJNA531553](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA531553), [PRJNA399729](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA399729), [PRJNA389800](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA389800) (ref. [80](https://www.nature.com/articles/s41588-025-02246-7#ref-CR80 "Bukowski, R. et al. Construction of the third-generation Zea mays haplotype map. GigaScience 7, gix134 (2018).")) and [SRP011907](https://www.ncbi.nlm.nih.gov/sra/?term=SRP011907) (ref. [1](https://www.nature.com/articles/s41588-025-02246-7#ref-CR1 "Chia, J.-M. et al. Maize HapMap2 identifies extant variation from a genome in flux. Nat. Genet. 44, 803–807 (2012).")). [Source data](https://www.nature.com/articles/s41588-025-02246-7#Sec31) are provided with this paper.

Code availability
-----------------

Custom scripts have been deposited to Github repositories ([https://github.com/Snodgras/MOA\_Analysis](https://github.com/Snodgras/MOA_Analysis), [https://github.com/corn2code/bQTL](https://github.com/corn2code/bQTL), [https://github.com/jengelhorn/AS-MOA](https://github.com/jengelhorn/AS-MOA), [https://github.com/jengelhorn/AS-RNAseq](https://github.com/jengelhorn/AS-RNAseq), [https://github.com/Ako31415/FIND-CIS-analysis](https://github.com/Ako31415/FIND-CIS-analysis)) and Zenodo ([https://doi.org/10.5281/zenodo.15098013](https://doi.org/10.5281/zenodo.15098013) (ref. [81](https://www.nature.com/articles/s41588-025-02246-7#ref-CR81 "Snodgrass, S. S. MOA_Analysis. Zenodo 
https://doi.org/10.5281/zenodo.15098013
(2025).")), [https://doi.org/10.5281/zenodo.15225769](https://doi.org/10.5281/zenodo.15225769) (ref. [82](https://www.nature.com/articles/s41588-025-02246-7#ref-CR82 "Torres-Rodriguez, J. V. bQTL. Zenodo 
https://doi.org/10.5281/zenodo.15225769
(2025).")), [https://doi.org/10.5281/zenodo.15097609](https://doi.org/10.5281/zenodo.15097609) (ref. [83](https://www.nature.com/articles/s41588-025-02246-7#ref-CR83 "Engelhorn, J. AS-MOA. Zenodo 
https://doi.org/10.5281/zenodo.15097609
(2025).")), [https://doi.org/10.5281/zenodo.15097644](https://doi.org/10.5281/zenodo.15097644) (ref. [84](https://www.nature.com/articles/s41588-025-02246-7#ref-CR84 "Engelhorn, J. AS-RNAseq. Zenodo 
https://doi.org/10.5281/zenodo.15097644
(2025).")), [https://doi.org/10.5281/zenodo.15212007](https://doi.org/10.5281/zenodo.15212007) (ref. [71](https://www.nature.com/articles/s41588-025-02246-7#ref-CR71 "Kok, A., Schneider, M., Hartwig, T. & Engelhorn, J. FIND-CIS-analysis. Zenodo 
https://doi.org/10.5281/zenodo.15212007
(2025)."))) under the GNU General Public License v3.0. Other software used in this study are included in the [Methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11).

References
----------

1.  Chia, J.-M. et al. Maize HapMap2 identifies extant variation from a genome in flux. _Nat. Genet._ **44**, 803–807 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22660545)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XnvVyls78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Maize%20HapMap2%20identifies%20extant%20variation%20from%20a%20genome%20in%20flux&journal=Nat.%20Genet.&volume=44&pages=803-807&publication_year=2012&author=Chia%2CJ-M) 
    
2.  Rodgers-Melnick, E., Vera, D. L., Bass, H. W. & Buckler, E. S. Open chromatin reveals the functional maize genome. _Proc. Natl Acad. Sci. USA_ **113**, E3177–E3184 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27185945)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896728)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XnvFeis7Y%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Open%20chromatin%20reveals%20the%20functional%20maize%20genome&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=113&pages=E3177-E3184&publication_year=2016&author=Rodgers-Melnick%2CE&author=Vera%2CDL&author=Bass%2CHW&author=Buckler%2CES) 
    
3.  Lorant, A., Ross-Ibarra, J. & Tenaillon, M. Genomics of long- and short-term adaptation in maize and teosintes. in _Statistical Population Genomics_ (ed. Dutheil, J. Y.) 289–311 (Springer, 2020). [https://doi.org/10.1007/978-1-0716-0199-0\_12](https://doi.org/10.1007/978-1-0716-0199-0_12)
    
4.  Song, B. et al. Conserved noncoding sequences provide insights into regulatory sequence and loss of gene expression in maize. _Genome Res._ **7**, 1245–1257 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conserved%20noncoding%20sequences%20provide%20insights%20into%20regulatory%20sequence%20and%20loss%20of%20gene%20expression%20in%20maize&journal=Genome%20Res.&volume=7&pages=1245-1257&publication_year=2021&author=Song%2CB) 
    
5.  Marand, A. P., Eveland, A. L., Kaufmann, K. & Springer, N. M. _Cis_\-regulatory elements in plant development, adaptation, and evolution. _Annu. Rev. Plant Biol._ **74**, 111–137 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36608347)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9881396)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXlvFektw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cis-regulatory%20elements%20in%20plant%20development%2C%20adaptation%2C%20and%20evolution&journal=Annu.%20Rev.%20Plant%20Biol.&volume=74&pages=111-137&publication_year=2023&author=Marand%2CAP&author=Eveland%2CAL&author=Kaufmann%2CK&author=Springer%2CNM) 
    
6.  Wolter, F. & Puchta, H. Application of CRISPR/Cas to understand _cis_\- and _trans_\-regulatory elements in plants. in _Plant Transcription Factors: Methods and Protocols_ (ed. Yamaguchi, N.) 23–40 (Springer, 2018). [https://doi.org/10.1007/978-1-4939-8657-6\_2](https://doi.org/10.1007/978-1-4939-8657-6_2)
    
7.  Sharon, E. et al. Functional genetic variants revealed by massively parallel precise genome editing. _Cell_ **175**, 544–557.e16 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30245013)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563827)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslOjsbvP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20genetic%20variants%20revealed%20by%20massively%20parallel%20precise%20genome%20editing&journal=Cell&volume=175&pages=544-557.e16&publication_year=2018&author=Sharon%2CE) 
    
8.  Lane, A. K., Niederhuth, C. E., Ji, L. & Schmitz, R. J. pENCODE: a plant encyclopedia of DNA elements. _Annu. Rev. Genet._ **48**, 49–70 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25149370)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4463858)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCkurjP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=pENCODE%3A%20a%20plant%20encyclopedia%20of%20DNA%20elements&journal=Annu.%20Rev.%20Genet.&volume=48&pages=49-70&publication_year=2014&author=Lane%2CAK&author=Niederhuth%2CCE&author=Ji%2CL&author=Schmitz%2CRJ) 
    
9.  White, M. A., Myers, C. A., Corbo, J. C. & Cohen, B. A. Massively parallel in vivo enhancer assay reveals that highly local features determine the _cis_\-regulatory function of ChIP-seq peaks. _Proc. Natl Acad. Sci. USA_ **110**, 11952–11957 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23818646)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3718143)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXht1emurnF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Massively%20parallel%20in%20vivo%20enhancer%20assay%20reveals%20that%20highly%20local%20features%20determine%20the%20cis-regulatory%20function%20of%20ChIP-seq%20peaks&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=110&pages=11952-11957&publication_year=2013&author=White%2CMA&author=Myers%2CCA&author=Corbo%2CJC&author=Cohen%2CBA) 
    
10.  Levo, M. & Segal, E. In pursuit of design principles of regulatory sequences. _Nat. Rev. Genet._ **15**, 453–468 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24913666)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXpsVKjsLo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=In%20pursuit%20of%20design%20principles%20of%20regulatory%20sequences&journal=Nat.%20Rev.%20Genet.&volume=15&pages=453-468&publication_year=2014&author=Levo%2CM&author=Segal%2CE) 
    
11.  Savadel, S. D. et al. The native cistrome and sequence motif families of the maize ear. _PLoS Genet._ **17**, e1009689 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34383745)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8360572)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVWnsrfO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20native%20cistrome%20and%20sequence%20motif%20families%20of%20the%20maize%20ear&journal=PLoS%20Genet.&volume=17&publication_year=2021&author=Savadel%2CSD) 
    
12.  Tu, X. et al. Reconstructing the maize leaf regulatory network using ChIP-seq data of 104 transcription factors. _Nat. Commun._ **11**, 5089 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33037196)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7547689)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVKhu7fM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reconstructing%20the%20maize%20leaf%20regulatory%20network%20using%20ChIP-seq%20data%20of%20104%20transcription%20factors&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Tu%2CX) 
    
13.  Zhao, H. et al. Genome-wide MNase hypersensitivity assay unveils distinct classes of open chromatin associated with H3K27me3 and DNA methylation in _Arabidopsis thaliana_. _Genome Biol._ **21**, 24 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32014062)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6996174)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20MNase%20hypersensitivity%20assay%20unveils%20distinct%20classes%20of%20open%20chromatin%20associated%20with%20H3K27me3%20and%20DNA%20methylation%20in%20Arabidopsis%20thaliana&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Zhao%2CH) 
    
14.  Manosalva Pérez, N. et al. MINI-AC: inference of plant gene regulatory networks using bulk or single-cell accessible chromatin profiles. _Plant J._ **117**, 280–301 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37788349)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MINI-AC%3A%20inference%20of%20plant%20gene%20regulatory%20networks%20using%20bulk%20or%20single-cell%20accessible%20chromatin%20profiles&journal=Plant%20J.&volume=117&pages=280-301&publication_year=2024&author=Manosalva%20P%C3%A9rez%2CN) 
    
15.  Springer, N. M. & Stupar, R. M. Allele-specific expression patterns reveal biases and embryo-specific parent-of-origin effects in hybrid maize. _Plant Cell_ **19**, 2391–2402 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17693532)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2002603)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFyisbjP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Allele-specific%20expression%20patterns%20reveal%20biases%20and%20embryo-specific%20parent-of-origin%20effects%20in%20hybrid%20maize&journal=Plant%20Cell&volume=19&pages=2391-2402&publication_year=2007&author=Springer%2CNM&author=Stupar%2CRM) 
    
16.  Pressoir, G. et al. Natural variation in maize architecture is mediated by allelic differences at the PINOID co-ortholog _barren inflorescence2_. _Plant J._ **58**, 618–628 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19154226)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXmvVahurk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Natural%20variation%20in%20maize%20architecture%20is%20mediated%20by%20allelic%20differences%20at%20the%20PINOID%20co-ortholog%20barren%20inflorescence2&journal=Plant%20J.&volume=58&pages=618-628&publication_year=2009&author=Pressoir%2CG) 
    
17.  Hartwig, T. et al. Hybrid allele-specific ChIP-seq analysis identifies variation in brassinosteroid-responsive transcription factor binding linked to traits in maize. _Genome Biol._ **24**, 108 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37158941)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10165856)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXps1Klt7o%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hybrid%20allele-specific%20ChIP-seq%20analysis%20identifies%20variation%20in%20brassinosteroid-responsive%20transcription%20factor%20binding%20linked%20to%20traits%20in%20maize&journal=Genome%20Biol.&volume=24&publication_year=2023&author=Hartwig%2CT) 
    
18.  Hufford, M. B. et al. De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. _Science_ **373**, 655–662 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34353948)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8733867)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslOisLnM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20assembly%2C%20annotation%2C%20and%20comparative%20analysis%20of%2026%20diverse%20maize%20genomes&journal=Science&volume=373&pages=655-662&publication_year=2021&author=Hufford%2CMB) 
    
19.  Lin, G. et al. Chromosome-level genome assembly of a regenerable maize inbred line A188. _Genome Biol._ **22**, 175 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34108023)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8188678)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFKmtL%2FM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosome-level%20genome%20assembly%20of%20a%20regenerable%20maize%20inbred%20line%20A188&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Lin%2CG) 
    
20.  Springer, N. M. et al. The maize W22 genome provides a foundation for functional genomics and transposon biology. _Nat. Genet._ **50**, 1282–1288 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30061736)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVWru7rE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20maize%20W22%20genome%20provides%20a%20foundation%20for%20functional%20genomics%20and%20transposon%20biology&journal=Nat.%20Genet.&volume=50&pages=1282-1288&publication_year=2018&author=Springer%2CNM) 
    
21.  Sun, S. et al. Extensive intraspecific gene order and gene structural variations between Mo17 and other maize genomes. _Nat. Genet._ **50**, 1289–1295 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30061735)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVWru7vO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Extensive%20intraspecific%20gene%20order%20and%20gene%20structural%20variations%20between%20Mo17%20and%20other%20maize%20genomes&journal=Nat.%20Genet.&volume=50&pages=1289-1295&publication_year=2018&author=Sun%2CS) 
    
22.  Castel, S. E., Levy-Moonshine, A., Mohammadi, P., Banks, E. & Lappalainen, T. Tools and best practices for data processing in allelic expression analysis. _Genome Biol._ **16**, 195 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26381377)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4574606)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tools%20and%20best%20practices%20for%20data%20processing%20in%20allelic%20expression%20analysis&journal=Genome%20Biol.&volume=16&publication_year=2015&author=Castel%2CSE&author=Levy-Moonshine%2CA&author=Mohammadi%2CP&author=Banks%2CE&author=Lappalainen%2CT) 
    
23.  O’Malley, R. C. et al. Cistrome and epicistrome features shape the regulatory DNA landscape. _Cell_ **165**, 1280–1292 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27203113)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4907330)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cistrome%20and%20epicistrome%20features%20shape%20the%20regulatory%20DNA%20landscape&journal=Cell&volume=165&pages=1280-1292&publication_year=2016&author=O%E2%80%99Malley%2CRC) 
    
24.  Tian, J. et al. Teosinte ligule allele narrows plant architecture and enhances high-density maize yields. _Science_ **365**, 658–664 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31416957)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Sis7rF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Teosinte%20ligule%20allele%20narrows%20plant%20architecture%20and%20enhances%20high-density%20maize%20yields&journal=Science&volume=365&pages=658-664&publication_year=2019&author=Tian%2CJ) 
    
25.  Yang, Q. et al. CACTA-like transposable element in _ZmCCT_ attenuated photoperiod sensitivity and accelerated the postdomestication spread of maize. _Proc. Natl Acad. Sci. USA_ **110**, 16969 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24089449)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3801022)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslejsbjP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CACTA-like%20transposable%20element%20in%20ZmCCT%20attenuated%20photoperiod%20sensitivity%20and%20accelerated%20the%20postdomestication%20spread%20of%20maize&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=110&publication_year=2013&author=Yang%2CQ) 
    
26.  Huang, C. et al. _ZmCCT9_ enhances maize adaptation to higher latitudes. _Proc. Natl Acad. Sci. USA_ **115**, E334–E341 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29279404)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVeqtbfO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ZmCCT9%20enhances%20maize%20adaptation%20to%20higher%20latitudes&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=115&pages=E334-E341&publication_year=2018&author=Huang%2CC) 
    
27.  Meng, X., Muszynski, M. G. & Danilevskaya, O. N. The FT-like _ZCN8_ gene functions as a floral activator and is involved in photoperiod sensitivity in maize. _Plant Cell_ **23**, 942–960 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21441432)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3082274)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmt1yjs7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20FT-like%20ZCN8%20gene%20functions%20as%20a%20floral%20activator%20and%20is%20involved%20in%20photoperiod%20sensitivity%20in%20maize&journal=Plant%20Cell&volume=23&pages=942-960&publication_year=2011&author=Meng%2CX&author=Muszynski%2CMG&author=Danilevskaya%2CON) 
    
28.  Li, C. et al. Gene expression and expression quantitative trait loci analyses uncover natural variations underlying the improvement of important agronomic traits during modern maize breeding. _Plant J._ **115**, 772–787 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37186341)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtVOhtLzE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Gene%20expression%20and%20expression%20quantitative%20trait%20loci%20analyses%20uncover%20natural%20variations%20underlying%20the%20improvement%20of%20important%20agronomic%20traits%20during%20modern%20maize%20breeding&journal=Plant%20J.&volume=115&pages=772-787&publication_year=2023&author=Li%2CC) 
    
29.  Salvi, S. et al. Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize. _Proc. Natl Acad. Sci. USA_ **104**, 11376–11381 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17595297)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2040906)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXnvFOmsrc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Conserved%20noncoding%20genomic%20sequences%20associated%20with%20a%20flowering-time%20quantitative%20trait%20locus%20in%20maize&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=104&pages=11376-11381&publication_year=2007&author=Salvi%2CS) 
    
30.  Castelletti, S., Tuberosa, R., Pindo, M. & Salvi, S. A MITE transposon insertion is associated with differential methylation at the maize flowering time QTL _Vgt1_. _G3 (Bethesda)_ **4**, 805–812 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24607887)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhs1SitrvI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20MITE%20transposon%20insertion%20is%20associated%20with%20differential%20methylation%20at%20the%20maize%20flowering%20time%20QTL%20Vgt1&journal=G3%20%28Bethesda%29&volume=4&pages=805-812&publication_year=2014&author=Castelletti%2CS&author=Tuberosa%2CR&author=Pindo%2CM&author=Salvi%2CS) 
    
31.  Xu, G. et al. Evolutionary and functional genomics of DNA methylation in maize domestication and improvement. _Nat. Commun._ **11**, 5539 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33139747)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7606521)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1OjtbrF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolutionary%20and%20functional%20genomics%20of%20DNA%20methylation%20in%20maize%20domestication%20and%20improvement&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Xu%2CG) 
    
32.  Ricci, W. A. et al. Widespread long-range _cis_\-regulatory elements in the maize genome. _Nat. Plants_ **5**, 1237–1249 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31740773)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6904520)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFyrtrrE)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Widespread%20long-range%20cis-regulatory%20elements%20in%20the%20maize%20genome&journal=Nat.%20Plants&volume=5&pages=1237-1249&publication_year=2019&author=Ricci%2CWA) 
    
33.  Dong, Z. et al. The regulatory landscape of a core maize domestication module controlling bud dormancy and growth repression. _Nat. Commun._ **10**, 3810 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31444327)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707278)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20regulatory%20landscape%20of%20a%20core%20maize%20domestication%20module%20controlling%20bud%20dormancy%20and%20growth%20repression&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Dong%2CZ) 
    
34.  Wills, D. M. et al. From many, one: genetic control of prolificacy during maize domestication. _PLoS Genet._ **9**, e1003604 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23825971)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3694832)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFGjsLrM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=From%20many%2C%20one%3A%20genetic%20control%20of%20prolificacy%20during%20maize%20domestication&journal=PLoS%20Genet.&volume=9&publication_year=2013&author=Wills%2CDM) 
    
35.  Wen, W. et al. An integrated multi-layered analysis of the metabolic networks of different tissues uncovers key genetic components of primary metabolism in maize. _Plant J._ **93**, 1116–1128 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29381266)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjt12nsLo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20multi-layered%20analysis%20of%20the%20metabolic%20networks%20of%20different%20tissues%20uncovers%20key%20genetic%20components%20of%20primary%20metabolism%20in%20maize&journal=Plant%20J.&volume=93&pages=1116-1128&publication_year=2018&author=Wen%2CW) 
    
36.  Cui, H. et al. Genome-wide identification and analysis of the maize serine peptidase S8 family genes in response to drought at seedling stage. _Plants_ **12**, 369 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36679082)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9865268)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXivFSms78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20identification%20and%20analysis%20of%20the%20maize%20serine%20peptidase%20S8%20family%20genes%20in%20response%20to%20drought%20at%20seedling%20stage&journal=Plants&volume=12&publication_year=2023&author=Cui%2CH) 
    
37.  Sun, G. et al. Dissection of the genetic architecture of peduncle vascular bundle-related traits in maize by a genome-wide association study. _Plant Biotechnol. J._ **20**, 1042–1053 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35080335)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9129077)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVSnu7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dissection%20of%20the%20genetic%20architecture%20of%20peduncle%20vascular%20bundle-related%20traits%20in%20maize%20by%20a%20genome-wide%20association%20study&journal=Plant%20Biotechnol.%20J.&volume=20&pages=1042-1053&publication_year=2022&author=Sun%2CG) 
    
38.  Sun, G. et al. A role for heritable transcriptomic variation in maize adaptation to temperate environments. _Genome Biol._ **24**, 55 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36964601)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10037803)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmt1Sgsrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20role%20for%20heritable%20transcriptomic%20variation%20in%20maize%20adaptation%20to%20temperate%20environments&journal=Genome%20Biol.&volume=24&publication_year=2023&author=Sun%2CG) 
    
39.  Regulski, M. et al. The maize methylome influences mRNA splice sites and reveals widespread paramutation-like switches guided by small RNA. _Genome Res._ **23**, 1651–1662 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23739895)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3787262)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1eltrzI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20maize%20methylome%20influences%20mRNA%20splice%20sites%20and%20reveals%20widespread%20paramutation-like%20switches%20guided%20by%20small%20RNA&journal=Genome%20Res.&volume=23&pages=1651-1662&publication_year=2013&author=Regulski%2CM) 
    
40.  Wallace, J. G. et al. Association mapping across numerous traits reveals patterns of functional variation in maize. _PLoS Genet._ **10**, e1004845 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25474422)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4256217)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20mapping%20across%20numerous%20traits%20reveals%20patterns%20of%20functional%20variation%20in%20maize&journal=PLoS%20Genet.&volume=10&publication_year=2014&author=Wallace%2CJG) 
    
41.  Tian, D. et al. GWAS Atlas: a curated resource of genome-wide variant–trait associations in plants and animals. _Nucleic Acids Res._ **48**, D927–D932 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31566222)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslWltrrP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GWAS%20Atlas%3A%20a%20curated%20resource%20of%20genome-wide%20variant%E2%80%93trait%20associations%20in%20plants%20and%20animals&journal=Nucleic%20Acids%20Res.&volume=48&pages=D927-D932&publication_year=2020&author=Tian%2CD) 
    
42.  Slatkin, M. Epigenetic inheritance and the missing heritability problem. _Genetics_ **182**, 845–850 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19416939)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2710163)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Epigenetic%20inheritance%20and%20the%20missing%20heritability%20problem&journal=Genetics&volume=182&pages=845-850&publication_year=2009&author=Slatkin%2CM) 
    
43.  Diepenbrock, C. H. et al. Novel loci underlie natural variation in vitamin E levels in maize grain. _Plant Cell_ **29**, 2374–2392 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28970338)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5774569)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFamtbrP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Novel%20loci%20underlie%20natural%20variation%20in%20vitamin%20E%20levels%20in%20maize%20grain&journal=Plant%20Cell&volume=29&pages=2374-2392&publication_year=2017&author=Diepenbrock%2CCH) 
    
44.  Li, C. et al. Numerous genetic loci identified for drought tolerance in the maize nested association mapping populations. _BMC Genomics_ **17**, 894 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27825295)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5101730)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Numerous%20genetic%20loci%20identified%20for%20drought%20tolerance%20in%20the%20maize%20nested%20association%20mapping%20populations&journal=BMC%20Genomics&volume=17&publication_year=2016&author=Li%2CC) 
    
45.  Shikha, M. et al. Genomic selection for drought tolerance using genome-wide SNPs in maize. _Front. Plant Sci._ **8**, 550 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28484471)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5399777)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genomic%20selection%20for%20drought%20tolerance%20using%20genome-wide%20SNPs%20in%20maize&journal=Front.%20Plant%20Sci.&volume=8&publication_year=2017&author=Shikha%2CM) 
    
46.  Wu, X. et al. Using high-throughput multiple optical phenotyping to decipher the genetic architecture of maize drought tolerance. _Genome Biol._ **22**, 185 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34162419)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8223302)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFKmtLfL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Using%20high-throughput%20multiple%20optical%20phenotyping%20to%20decipher%20the%20genetic%20architecture%20of%20maize%20drought%20tolerance&journal=Genome%20Biol.&volume=22&publication_year=2021&author=Wu%2CX) 
    
47.  Mao, H. et al. A transposable element in a NAC gene is associated with drought tolerance in maize seedlings. _Nat. Commun._ **6**, 8326 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26387805)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFeitr7F)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20transposable%20element%20in%20a%20NAC%20gene%20is%20associated%20with%20drought%20tolerance%20in%20maize%20seedlings&journal=Nat.%20Commun.&volume=6&publication_year=2015&author=Mao%2CH) 
    
48.  Xu, Z., Wang, M., Guo, Z., Zhu, X. & Xia, Z. Identification of a 119-bp promoter of the maize Sulfite Oxidase Gene (_ZmSO_) that confers high-level gene expression and ABA or drought inducibility in transgenic plants. _Int. J. Mol. Sci._ **20**, 3326 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31284569)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6651508)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVGmtrk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20a%20119-bp%20promoter%20of%20the%20maize%20Sulfite%20Oxidase%20Gene%20%28ZmSO%29%20that%20confers%20high-level%20gene%20expression%20and%20ABA%20or%20drought%20inducibility%20in%20transgenic%20plants&journal=Int.%20J.%20Mol.%20Sci.&volume=20&publication_year=2019&author=Xu%2CZ&author=Wang%2CM&author=Guo%2CZ&author=Zhu%2CX&author=Xia%2CZ) 
    
49.  Johansson, I., Karlsson, M., Johanson, U., Larsson, C. & Kjellbom, P. The role of aquaporins in cellular and whole plant water balance. _Biochim. Biophys. Acta Biomembr._ **1465**, 324–342 (2000).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXit1Wgt7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20aquaporins%20in%20cellular%20and%20whole%20plant%20water%20balance&journal=Biochim.%20Biophys.%20Acta%20Biomembr.&volume=1465&pages=324-342&publication_year=2000&author=Johansson%2CI&author=Karlsson%2CM&author=Johanson%2CU&author=Larsson%2CC&author=Kjellbom%2CP) 
    
50.  Xie, Z. et al. The AP2/ERF transcription factor TINY modulates brassinosteroid-regulated plant growth and drought responses in _Arabidopsis_. _Plant Cell_ **31**, 1788–1806 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31126980)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6713308)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVeisbnF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20AP2%2FERF%20transcription%20factor%20TINY%20modulates%20brassinosteroid-regulated%20plant%20growth%20and%20drought%20responses%20in%20Arabidopsis&journal=Plant%20Cell&volume=31&pages=1788-1806&publication_year=2019&author=Xie%2CZ) 
    
51.  Hung, H.-Y. et al. The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population. _Heredity_ **108**, 490–499 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22027895)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XlvVGmsr4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20relationship%20between%20parental%20genetic%20or%20phenotypic%20divergence%20and%20progeny%20variation%20in%20the%20maize%20nested%20association%20mapping%20population&journal=Heredity&volume=108&pages=490-499&publication_year=2012&author=Hung%2CH-Y) 
    
52.  Peiffer, J. A. et al. The genetic architecture of maize height. _Genetics_ **196**, 1337–1356 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24514905)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3982682)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtl2rs7rL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genetic%20architecture%20of%20maize%20height&journal=Genetics&volume=196&pages=1337-1356&publication_year=2014&author=Peiffer%2CJA) 
    
53.  Kump, K. L. et al. Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population. _Nat. Genet._ **43**, 163–168 (2011).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21217757)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXjtVOitQ%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome-wide%20association%20study%20of%20quantitative%20resistance%20to%20southern%20leaf%20blight%20in%20the%20maize%20nested%20association%20mapping%20population&journal=Nat.%20Genet.&volume=43&pages=163-168&publication_year=2011&author=Kump%2CKL) 
    
54.  Bian, Y., Yang, Q., Balint-Kurti, P. J., Wisser, R. J. & Holland, J. B. Limits on the reproducibility of marker associations with southern leaf blight resistance in the maize nested association mapping population. _BMC Genomics_ **15**, 1068 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25475173)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4300987)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Limits%20on%20the%20reproducibility%20of%20marker%20associations%20with%20southern%20leaf%20blight%20resistance%20in%20the%20maize%20nested%20association%20mapping%20population&journal=BMC%20Genomics&volume=15&publication_year=2014&author=Bian%2CY&author=Yang%2CQ&author=Balint-Kurti%2CPJ&author=Wisser%2CRJ&author=Holland%2CJB) 
    
55.  Sturm, M., Schroeder, C. & Bauer, P. SeqPurge: highly-sensitive adapter trimming for paired-end NGS data. _BMC Bioinformatics_ **17**, 208 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27161244)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4862148)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SeqPurge%3A%20highly-sensitive%20adapter%20trimming%20for%20paired-end%20NGS%20data&journal=BMC%20Bioinformatics&volume=17&publication_year=2016&author=Sturm%2CM&author=Schroeder%2CC&author=Bauer%2CP) 
    
56.  Gaspar, J. M. NGmerge: merging paired-end reads via novel empirically-derived models of sequencing errors. _BMC Bioinformatics_ **19**, 536 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30572828)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6302405)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1SksrrM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NGmerge%3A%20merging%20paired-end%20reads%20via%20novel%20empirically-derived%20models%20of%20sequencing%20errors&journal=BMC%20Bioinformatics&volume=19&publication_year=2018&author=Gaspar%2CJM) 
    
57.  Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. _Bioinformatics_ **29**, 15–21 (2013).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23104886)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&journal=Bioinformatics&volume=29&pages=15-21&publication_year=2013&author=Dobin%2CA) 
    
58.  Li, H. et al. The Sequence Alignment/Map format and SAMtools. _Bioinformatics_ **25**, 2078–2079 (2009).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19505943)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&journal=Bioinformatics&volume=25&pages=2078-2079&publication_year=2009&author=Li%2CH) 
    
59.  Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. _Nucleic Acids Res._ **44**, W160–W165 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27079975)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987876)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=deepTools2%3A%20a%20next%20generation%20web%20server%20for%20deep-sequencing%20data%20analysis&journal=Nucleic%20Acids%20Res.&volume=44&pages=W160-W165&publication_year=2016&author=Ram%C3%ADrez%2CF) 
    
60.  Liang, Z. et al. Mapping responsive genomic elements to heat stress in a maize diversity panel. _Genome Biol._ **23**, 234 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36345007)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9639295)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivVCit7jJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20responsive%20genomic%20elements%20to%20heat%20stress%20in%20a%20maize%20diversity%20panel&journal=Genome%20Biol.&volume=23&publication_year=2022&author=Liang%2CZ) 
    
61.  Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. _Bioinformatics_ **26**, 841–842 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20110278)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXivFGkurc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&journal=Bioinformatics&volume=26&pages=841-842&publication_year=2010&author=Quinlan%2CAR&author=Hall%2CIM) 
    
62.  Kent, W. J., Zweig, A. S., Barber, G., Hinrichs, A. S. & Karolchik, D. BigWig and BigBed: enabling browsing of large distributed datasets. _Bioinformatics_ **26**, 2204–2207 (2010).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20639541)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922891)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVGru7rP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BigWig%20and%20BigBed%3A%20enabling%20browsing%20of%20large%20distributed%20datasets&journal=Bioinformatics&volume=26&pages=2204-2207&publication_year=2010&author=Kent%2CWJ&author=Zweig%2CAS&author=Barber%2CG&author=Hinrichs%2CAS&author=Karolchik%2CD) 
    
63.  Armstrong, J. et al. Progressive Cactus is a multiple-genome aligner for the thousand-genome era. _Nature_ **587**, 246–251 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33177663)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7673649)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlWktLnF)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Progressive%20Cactus%20is%20a%20multiple-genome%20aligner%20for%20the%20thousand-genome%20era&journal=Nature&volume=587&pages=246-251&publication_year=2020&author=Armstrong%2CJ) 
    
64.  Gomez-Cano, L., Yang, F. & Grotewold, E. Isolation and efficient maize protoplast transformation. _Bio Protoc._ [https://doi.org/10.21769/BioProtoc.3346](https://doi.org/10.21769/BioProtoc.3346) (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Isolation%20and%20efficient%20maize%20protoplast%20transformation&journal=Bio%20Protoc.&doi=10.21769%2FBioProtoc.3346&publication_year=2019&author=Gomez-Cano%2CL&author=Yang%2CF&author=Grotewold%2CE) 
    
65.  Hampf, M. & Gossen, M. A protocol for combined _Photinus_ and _Renilla_ luciferase quantification compatible with protein assays. _Anal. Biochem._ **356**, 94–99 (2006).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16750160)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XotlKntrk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20protocol%20for%20combined%20Photinus%20and%20Renilla%20luciferase%20quantification%20compatible%20with%20protein%20assays&journal=Anal.%20Biochem.&volume=356&pages=94-99&publication_year=2006&author=Hampf%2CM&author=Gossen%2CM) 
    
66.  Liang, Z. et al. Genetic and epigenetic variation in transposable element expression responses to abiotic stress in maize. _Plant Physiol._ **186**, 420 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33591319)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8154091)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVOhsLrK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20and%20epigenetic%20variation%20in%20transposable%20element%20expression%20responses%20to%20abiotic%20stress%20in%20maize&journal=Plant%20Physiol.&volume=186&publication_year=2021&author=Liang%2CZ) 
    
67.  Song, B. et al. AnchorWave: sensitive alignment of genomes with high sequence diversity, extensive structural polymorphism, and whole-genome duplication. _Proc. Natl Acad. Sci. USA_ **119**, e2113075119 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34934012)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvV2nu78%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=AnchorWave%3A%20sensitive%20alignment%20of%20genomes%20with%20high%20sequence%20diversity%2C%20extensive%20structural%20polymorphism%2C%20and%20whole-genome%20duplication&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=119&publication_year=2022&author=Song%2CB) 
    
68.  Li, H. Minimap2: pairwise alignment for nucleotide sequences. _Bioinformatics_ **34**, 3094–3100 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29750242)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6137996)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVamu73J)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Minimap2%3A%20pairwise%20alignment%20for%20nucleotide%20sequences&journal=Bioinformatics&volume=34&pages=3094-3100&publication_year=2018&author=Li%2CH) 
    
69.  Wei, W. et al. wgatools: an ultrafast toolkit for manipulating whole-genome alignments. _Bioinformatics_ **41**, btaf132 (2025).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=40152239)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11978383)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2MXpvVGgt70%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=wgatools%3A%20an%20ultrafast%20toolkit%20for%20manipulating%20whole-genome%20alignments&journal=Bioinformatics&volume=41&publication_year=2025&author=Wei%2CW) 
    
70.  Zhao, H. et al. CrossMap: a versatile tool for coordinate conversion between genome assemblies. _Bioinformatics_ **30**, 1006–1007 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24351709)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CrossMap%3A%20a%20versatile%20tool%20for%20coordinate%20conversion%20between%20genome%20assemblies&journal=Bioinformatics&volume=30&pages=1006-1007&publication_year=2014&author=Zhao%2CH) 
    
71.  Kok, A., Schneider, M., Hartwig, T. & Engelhorn, J. FIND-CIS-analysis. _Zenodo_ [https://doi.org/10.5281/zenodo.15212007](https://doi.org/10.5281/zenodo.15212007) (2025).
    
72.  Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._ **15**, 550 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25516281)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Love%2CMI&author=Huber%2CW&author=Anders%2CS) 
    
73.  Khaipho-Burch, M. et al. Elucidating the patterns of pleiotropy and its biological relevance in maize. _PLoS Genet._ **19**, e1010664 (2023).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36943844)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10030035)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmsVyksL8%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elucidating%20the%20patterns%20of%20pleiotropy%20and%20its%20biological%20relevance%20in%20maize&journal=PLoS%20Genet.&volume=19&publication_year=2023&author=Khaipho-Burch%2CM) 
    
74.  Bradbury, P. J. et al. TASSEL: software for association mapping of complex traits in diverse samples. _Bioinformatics_ **23**, 2633–2635 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17586829)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXht1ejtr7F)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&journal=Bioinformatics&volume=23&pages=2633-2635&publication_year=2007&author=Bradbury%2CPJ) 
    
75.  Speed, D., Hemani, G., Johnson, M. R. & Balding, D. J. Improved heritability estimation from genome-wide SNPs. _Am. J. Hum. Genet._ **91**, 1011–1021 (2012).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23217325)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3516604)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVWis7bP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improved%20heritability%20estimation%20from%20genome-wide%20SNPs&journal=Am.%20J.%20Hum.%20Genet.&volume=91&pages=1011-1021&publication_year=2012&author=Speed%2CD&author=Hemani%2CG&author=Johnson%2CMR&author=Balding%2CDJ) 
    
76.  Howe, K. L. et al. Ensembl Genomes 2020—enabling non-vertebrate genomic research. _Nucleic Acids Res._ **48**, D689–D695 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31598706)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslWltrvP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ensembl%20Genomes%202020%E2%80%94enabling%20non-vertebrate%20genomic%20research&journal=Nucleic%20Acids%20Res.&volume=48&pages=D689-D695&publication_year=2020&author=Howe%2CKL) 
    
77.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
78.  Engelhorn, J., Hartwig, T., Kok, A., Stich, B. & Schneider, M. Binding QTL data and concatenated versions of maize genomes. _Zenodo_ [https://doi.org/10.5281/zenodo.15177272](https://doi.org/10.5281/zenodo.15177272) (2025).
    
79.  Jiao, Y. et al. Improved maize reference genome with single-molecule technologies. _Nature_ **546**, 524–527 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28605751)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7052699)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXpvFCmu7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improved%20maize%20reference%20genome%20with%20single-molecule%20technologies&journal=Nature&volume=546&pages=524-527&publication_year=2017&author=Jiao%2CY) 
    
80.  Bukowski, R. et al. Construction of the third-generation _Zea mays_ haplotype map. _GigaScience_ **7**, gix134 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29300887)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Construction%20of%20the%20third-generation%20Zea%20mays%20haplotype%20map&journal=GigaScience&volume=7&publication_year=2018&author=Bukowski%2CR) 
    
81.  Snodgrass, S. S. MOA\_Analysis. _Zenodo_ [https://doi.org/10.5281/zenodo.15098013](https://doi.org/10.5281/zenodo.15098013) (2025).
    
82.  Torres-Rodriguez, J. V. bQTL. _Zenodo_ [https://doi.org/10.5281/zenodo.15225769](https://doi.org/10.5281/zenodo.15225769) (2025).
    
83.  Engelhorn, J. AS-MOA. _Zenodo_ [https://doi.org/10.5281/zenodo.15097609](https://doi.org/10.5281/zenodo.15097609) (2025).
    
84.  Engelhorn, J. AS-RNAseq. _Zenodo_ [https://doi.org/10.5281/zenodo.15097644](https://doi.org/10.5281/zenodo.15097644) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02246-7?format=refman&flavour=references)

Acknowledgements
----------------

We are grateful to M. Sachs for seed advice. We thank M. Krist, F. Eikelmann and F. Esser for field support. We thank S.-D. Pophaly, J. Reza and M. H. Thoben for IT support. We thank C. Becker, K. Becker and B. Huettel for sequencing support. We are grateful to S. Weizel, M. Blank, M. D. Mueller, R. S. Kumar, G. Koentjes, I. Tol, S. Mukherjee, N. Sargheini, L. Watschke, V. Rajan, E. van Moore and M. van Boesschoten for greenhouse and experimental support. We would like to thank F. Andrews for not irrigating our drought experiment. We also thank N. Springer for his critical discussion of the paper. This study was supported by funding from the ‘Deutsche Forschungsgemeinschaft’ (DFG, ID:458854361 to T.H.) as part of DFG Sequencing call 3 and the European Union’s Horizon Europe program BOOSTER (ID: 101081770 to T.H.). B.S., W.B.F. and T.H. received funding from the DFG under Germany’s Excellence Strategy (CEPLAS Cluster, EXC 2048/1, Project ID: 390686111). W.B.F. was supported by an Alexander von Humboldt professorship. J.R.I. was supported by funding from the United States National Science Foundation (ID: 1822330) and the United States Department of Agriculture (Hatch project CA-D-PLS-2066-H 548). J.C.S. and J.V.T.R. were supported by the United States Department of Energy (ID: DE-SC0020355). S.J.S. was supported by funding from the United States National Science Foundation through the Graduate Research Fellowship Program (DGE: 1744592). A.S. and H.W.B. were supported by the National Science Foundation Plant Genome Research Program (IOS 1444532). M. Stam was supported by European Commission Seventh Framework-People-2012-ITN Project EpiTRAITS, GA-316965 (Epigenetic Regulation of Economically Important Plant Traits). F.F. received support from the Program Oriented Funding of the Helmholtz Association. Most of the next-generation sequencing was performed and supported by the DFG Research Infrastructure West German Genome Center (407493903) as part of the Next Generation Sequencing Competence Network (project 423957469). Some of the results reported in this paper were partially supported by the HPC@ISU equipment at Iowa State University, some of which has been purchased through funding provided by NSF under MRI grant numbers 1726447 and MRI2018594. Some of the work done by J.E. was supported by the French National Research Agency (ANR, ID: ANR-22-CPJ2-0110-01). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the paper.

Funding
-------

Open access funding provided by Heinrich-Heine-Universität Düsseldorf.

Author information
------------------

### Authors and Affiliations

1.  Heinrich Heine University Düsseldorf, Faculty of Mathematics and Natural Sciences, Institute for Molecular Physiology, Düsseldorf, Germany
    
    Julia Engelhorn, Amelie Kok, Tatjana Kiwit, Duong Thi Hai Doan, Wolf B. Frommer & Thomas Hartwig
    
2.  Independent Research Groups, Max Planck Institute for Plant Breeding Research, Cologne, Germany
    
    Julia Engelhorn, Amelie Kok, Tatjana Kiwit, Duong Thi Hai Doan, Wolf B. Frommer & Thomas Hartwig
    
3.  DIADE, University of Montpellier, CIRAD, IRD, Montpellier, France
    
    Julia Engelhorn
    
4.  Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, IA, USA
    
    Samantha J. Snodgrass, Arun S. Seetharam & Matthew B. Hufford
    
5.  Cluster of Excellence on Plant Sciences (CEPLAS), Heinrich Heine University, Düsseldorf, Germany
    
    Amelie Kok, Benjamin Stich, Wolf B. Frommer & Thomas Hartwig
    
6.  Heinrich Heine University Düsseldorf, Faculty of Mathematics and Natural Sciences, Institute for Quantitative Genetics and Genomics of Plants, Düsseldorf, Germany
    
    Amelie Kok, Michael Schneider & Benjamin Stich
    
7.  Department of Biological Science, Florida State University, Tallahassee, FL, USA
    
    Ayush Singh & Hank W. Bass
    
8.  Perelyn, Munich, Germany
    
    Michael Banf
    
9.  School of Integrative Plant Sciences, Plant Breeding and Genetics Section, Cornell University, Ithaca, NY, USA
    
    Merritt Khaipho-Burch
    
10.  Department of Plant Sciences, University of California, Davis, CA, USA
    
    Daniel E. Runcie
    
11.  Swammerdam Institute for Life Sciences, University of Amsterdam, Amsterdam, Netherlands
    
    Victor A. Sánchez-Camargo, Rechien Bader & Maike Stam
    
12.  Department of Agronomy and Horticulture, University of Nebraska–Lincoln, Lincoln, NE, USA
    
    J. Vladimir Torres-Rodriguez & James C. Schnable
    
13.  Maize Research Institute, Sichuan Agricultural University, Wenjiang, China
    
    Guangchao Sun
    
14.  Institute of Bio- and Geosciences (IBG-2: Plant Sciences), Forschungszentrum Jülich, Jülich, Germany
    
    Fabio Fiorani
    
15.  Institute of Bio- and Geosciences (IBG-4: Bioinformatics), Bioeconomy Science Center (BioSC), Forschungszentrum Jülich, Jülich, Germany
    
    Sebastian Beier
    
16.  Julius Kühn Institute, Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, Groß Lüsewitz, Germany
    
    Benjamin Stich
    
17.  Institute for Transformative Biomolecules, ITbM, Nagoya University, Nagoya, Japan
    
    Wolf B. Frommer
    
18.  Department of Evolution and Ecology, Center for Population Biology and Genome Center, University of California, Davis, CA, USA
    
    Jeffrey Ross-Ibarra
    

Authors

1.  Julia Engelhorn
    
    [View author publications](https://www.nature.com/search?author=Julia%20Engelhorn)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Julia%20Engelhorn) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Julia%20Engelhorn%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Samantha J. Snodgrass
    
    [View author publications](https://www.nature.com/search?author=Samantha%20J.%20Snodgrass)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Samantha%20J.%20Snodgrass) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Samantha%20J.%20Snodgrass%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Amelie Kok
    
    [View author publications](https://www.nature.com/search?author=Amelie%20Kok)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Amelie%20Kok) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Amelie%20Kok%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Arun S. Seetharam
    
    [View author publications](https://www.nature.com/search?author=Arun%20S.%20Seetharam)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Arun%20S.%20Seetharam) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Arun%20S.%20Seetharam%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Michael Schneider
    
    [View author publications](https://www.nature.com/search?author=Michael%20Schneider)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Michael%20Schneider) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Michael%20Schneider%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Tatjana Kiwit
    
    [View author publications](https://www.nature.com/search?author=Tatjana%20Kiwit)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tatjana%20Kiwit) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tatjana%20Kiwit%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Ayush Singh
    
    [View author publications](https://www.nature.com/search?author=Ayush%20Singh)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ayush%20Singh) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ayush%20Singh%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Michael Banf
    
    [View author publications](https://www.nature.com/search?author=Michael%20Banf)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Michael%20Banf) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Michael%20Banf%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Duong Thi Hai Doan
    
    [View author publications](https://www.nature.com/search?author=Duong%20Thi%20Hai%20Doan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Duong%20Thi%20Hai%20Doan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Duong%20Thi%20Hai%20Doan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Merritt Khaipho-Burch
    
    [View author publications](https://www.nature.com/search?author=Merritt%20Khaipho-Burch)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Merritt%20Khaipho-Burch) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Merritt%20Khaipho-Burch%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Daniel E. Runcie
    
    [View author publications](https://www.nature.com/search?author=Daniel%20E.%20Runcie)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20E.%20Runcie) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20E.%20Runcie%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Victor A. Sánchez-Camargo
    
    [View author publications](https://www.nature.com/search?author=Victor%20A.%20S%C3%A1nchez-Camargo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Victor%20A.%20S%C3%A1nchez-Camargo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Victor%20A.%20S%C3%A1nchez-Camargo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Rechien Bader
    
    [View author publications](https://www.nature.com/search?author=Rechien%20Bader)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rechien%20Bader) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rechien%20Bader%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  J. Vladimir Torres-Rodriguez
    
    [View author publications](https://www.nature.com/search?author=J.%20Vladimir%20Torres-Rodriguez)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=J.%20Vladimir%20Torres-Rodriguez) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22J.%20Vladimir%20Torres-Rodriguez%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Guangchao Sun
    
    [View author publications](https://www.nature.com/search?author=Guangchao%20Sun)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Guangchao%20Sun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Guangchao%20Sun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Maike Stam
    
    [View author publications](https://www.nature.com/search?author=Maike%20Stam)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Maike%20Stam) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Maike%20Stam%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Fabio Fiorani
    
    [View author publications](https://www.nature.com/search?author=Fabio%20Fiorani)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Fabio%20Fiorani) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Fabio%20Fiorani%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Sebastian Beier
    
    [View author publications](https://www.nature.com/search?author=Sebastian%20Beier)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sebastian%20Beier) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sebastian%20Beier%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  James C. Schnable
    
    [View author publications](https://www.nature.com/search?author=James%20C.%20Schnable)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=James%20C.%20Schnable) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22James%20C.%20Schnable%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Hank W. Bass
    
    [View author publications](https://www.nature.com/search?author=Hank%20W.%20Bass)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hank%20W.%20Bass) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hank%20W.%20Bass%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Matthew B. Hufford
    
    [View author publications](https://www.nature.com/search?author=Matthew%20B.%20Hufford)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20B.%20Hufford) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20B.%20Hufford%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Benjamin Stich
    
    [View author publications](https://www.nature.com/search?author=Benjamin%20Stich)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Benjamin%20Stich) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Benjamin%20Stich%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Wolf B. Frommer
    
    [View author publications](https://www.nature.com/search?author=Wolf%20B.%20Frommer)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wolf%20B.%20Frommer) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wolf%20B.%20Frommer%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Jeffrey Ross-Ibarra
    
    [View author publications](https://www.nature.com/search?author=Jeffrey%20Ross-Ibarra)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jeffrey%20Ross-Ibarra) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jeffrey%20Ross-Ibarra%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Thomas Hartwig
    
    [View author publications](https://www.nature.com/search?author=Thomas%20Hartwig)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Thomas%20Hartwig) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Thomas%20Hartwig%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

T.H. conceived the research project. J.E., S.J.S., J.R.I., D.E.R., M.B.H., B.S., M. Stam, W.B.F. and T.H. designed experiments. J.E., S.J.S., T.K., V.A.S.-C., R.B., A.S., J.V.T.R., A.K., D.T.H.D. and T.H. performed experiments. J.E., S.J.S., A.S., A.K., M-K.B., M. Schneider, J.R.I., D.E.R., G.S., V.A.S.-C., F.F., A.S.S., J.V.T.R., J.C.S., M.B. and T.H. analyzed the data. J.E., S.J.S., A.S.S. and T.H. developed the hybrid mapping pipeline and J.E., S.J.S., J.R.I., D.E.R., J.V.T.R., S.J.S., M. Schneider, A.K., M.B. and T.H. developed the analysis scripts. S.B. developed the custom genome browser. J.E., J.R.I. and T.H. wrote the paper and W.B.F., H.W.B., F.F., S.J.S. and B.S. revised it.

### Corresponding author

Correspondence to [Thomas Hartwig](mailto:hartwit@hhu.de).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Mingliang Xu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02246-7#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Examples of allele-specific B73xMo17 MOA-seq and comparison to allele-specific ChIP-seq.](https://www.nature.com/articles/s41588-025-02246-7/figures/8)

**a**) Average, normalized, allele-specific ChIP-seq of the ZmBZR1 TF (top two rows, black) and MOA-seq (bottom two rows, B73 green, Mo17 blue; fragment center data, see [methods](https://www.nature.com/articles/s41588-025-02246-7#Sec11)) shown upstream of _ZmPGIP2_. **c**) Average, normalized, allele-specific differences in MOA coverage upstream of _ZmBIF2_ overlap with a known, ‘hypervariable’ (Hv, purple box) _cis_\-regulatory region. **b**, **d**) Normalized, average MOA coverage of three biological replicas at orange dashed boxes in **a** (P = 0.015, two-sided t-test) and **c** (P = 0.003, two-sided t-test). RPGC = number of reads per 1 bp / scaling factor (total number of mapped reads multiplied by fragment length / effective genome size). Boxes in plots denote the range from the first to the third quartile, and lines within boxes indicate the median. Whiskers represent 1.5-fold of the interquartile range.

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM13)

### [Extended Data Fig. 2 Correlation of methylation differences between the alleles in the B73xMo17 hybrid and the respective inbred parents and additional MOA-seq variation explained by methylation.](https://www.nature.com/articles/s41588-025-02246-7/figures/9)

**a** and **b**) Methylation differences at AMP loci are expressed as Mo17 methylation values subtracted from B73 methylation values in inbred and hybrid comparisons (**a**: CG, **b**: CHG). Blue line indicates x = y for orientation. PCC: Pearson Correlation Coefficient. **c** and **d**) Numbers of clumbed bQTL (SNP approach **c**, INDEL approach **d**) in WW conditions that display either a significant correlation of MOA-seq signal and the genotype (Genotype only), of MOA-seq signal and the methylation status within +/− 20 bp of the position (Meth. only) or both.

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM14)

### [Extended Data Fig. 3 Distributions of bQTL, GWAS hits, and selective sweep genomic features across ten maize chromosomes.](https://www.nature.com/articles/s41588-025-02246-7/figures/10)

From inner to outer circle, the tracks are: 1. chromosome names; 2. XP-CLR scores of selective sweeps[31](https://www.nature.com/articles/s41588-025-02246-7#ref-CR31 "Xu, G. et al. Evolutionary and functional genomics of DNA methylation in maize domestication and improvement. Nat. Commun. 11, 5539 (2020).") detected between modern maize and teosinte, 3. -log10 p-values of the maize GWAS Atlas lead SNPs[41](https://www.nature.com/articles/s41588-025-02246-7#ref-CR41 "Tian, D. et al. GWAS Atlas: a curated resource of genome-wide variant–trait associations in plants and animals. Nucleic Acids Res. 48, D927–D932 (2020)."); and 4. -log10 p-values of bQTL (SNP and INDEL) associated with variation in MOA coverage in the pan-cistrome. Red markers denote selected examples of bQTL that coincide with natural variation of classical domestication and flowering time genes.

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM15)

### [Extended Data Fig. 4 Assessment of relative water content of the 25 F1 hybrids under well-watered and drought conditions.](https://www.nature.com/articles/s41588-025-02246-7/figures/11)

\* indicate significant difference between WW and DS based on ANOVA followed by Tukey HSD test, p < 0.01 (exact p-values provided in source data). n = 9 pots with 4 plants each.

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM16)

### [Extended Data Fig. 5 Overview of the _ZmTINY_ locus with allele-specific mRNA abundance and TF-binding.](https://www.nature.com/articles/s41588-025-02246-7/figures/12)

**a**) Genome browser view of the upstream and downstream region of _ZmTINY_ in the hybrid of B73xCML333. Green arrows mark DS-bQTL positions. Yellow blocks mark regions analyzed in (**c**). The black transposable elements (TE) block marks a TE that is present in B73, A188, Ki3, and Mo18W. The scale bar applies to both tracks. **b**) Fold change (FC) in mRNA abundance between the NAM allele and the B73 allele under DS conditions. **c**) FC in MOA occupancy between the NAM allele and the B73 allele under DS conditions. Average MOA occupancy per base was calculated in the three yellow regions marked in (**a**). To ensure comparison of homologous regions, regions between two homologous SNPs determined by whole-genome alignment were chosen. Note that the W22 promoter contains a TE in the (A) region, adding ~5 kb compared to the syntenic W22 region. Two additional regions in W22 were compared to the B73 (A) region: the region up to the TE (FC 0.36) and a region of similar length to the B73 (A) region (FC 0.21). The downstream regions of A188 and Mo18W do not differ from B73, and thus, no FC could be detected. **d**: CG methylation in 40 bp surrounding SNP B73-chr3:4729798 (just after the TE sequence downstream of _ZmTINY_).

[Source data](https://www.nature.com/articles/s41588-025-02246-7#MOESM17)

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM1_ESM.pdf)

Supplementary Notes 1–8, Figs. 1–19, and References.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM3_ESM.pdf)

### [Supplementary Data](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM4_ESM.xlsx)

Source data for Supplementary Figs. 2–7, 10, 12–17 and 19.

### [Supplementary Table](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM5_ESM.xlsx)

Supplementary Tables 1–14.

Source data
-----------

### [Source Data Fig. 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM6_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM7_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM8_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM9_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM10_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 6](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM11_ESM.xlsx)

Statistical source data.

### [Source Data Fig. 7](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM12_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 1/Table 1](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM13_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 2/Table 2](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM14_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 3/Table 3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM15_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 4/Table 4](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM16_ESM.xlsx)

Statistical source data.

### [Source Data Extended Data Fig. 5/Table 5](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02246-7/MediaObjects/41588_2025_2246_MOESM17_ESM.xlsx)

Statistical source data.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Genetic%20variation%20at%20transcription%20factor%20binding%20sites%20largely%20explains%20phenotypic%20heritability%20in%20maize&author=Julia%20Engelhorn%20et%20al&contentID=10.1038%2Fs41588-025-02246-7&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-11&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02246-7)

### Cite this article

Engelhorn, J., Snodgrass, S.J., Kok, A. _et al._ Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02246-7

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02246-7?format=refman&flavour=citation)

-   Received: 09 May 2024
    
-   Accepted: 28 May 2025
    
-   Published: 11 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02246-7
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02246-7.pdf)
