> id: https://www.nature.com/articles/s41588-025-02300-4

> link: https://www.nature.com/articles/s41588-025-02300-4

> title: Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue

# Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue
_Published on Wed, 20 Aug 2025 00:00:00 GMT_

Abstract
--------

The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, _TET1_ and _DNMT3A_ are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome.

Main
----

The global prevalence of abdominal obesity, defined as an excessive accumulation of adipose tissue in the abdominal region, has been increasing at an alarming rate over the past few decades[1](https://www.nature.com/articles/s41588-025-02300-4#ref-CR1 "González-Muniesa, P. et al. Obesity. Nat. Rev. Dis. Prim. 3, 17034 (2017)."),[2](https://www.nature.com/articles/s41588-025-02300-4#ref-CR2 "Sun, J.-Y. et al. Trends in general and abdominal obesity in US adults: evidence from the National Health and Nutrition Examination Survey (2001–2018). Front. Public Health 10, 925293 (2022)."). Abdominal obesity is a known predictor of all-cause mortality, probably because of its increased risk of cardiometabolic disease (CMD) and other adverse pathological conditions[3](https://www.nature.com/articles/s41588-025-02300-4#ref-CR3 "Després, J.-P. Abdominal obesity: the most prevalent cause of the metabolic syndrome and related cardiometabolic risk. Eur. Heart J. Suppl. 8, B4–B12 (2006)."). The risks related to abdominal obesity are thought to be partly mediated through obesity-induced low-grade inflammation at the tissue and cell-type level[4](https://www.nature.com/articles/s41588-025-02300-4#ref-CR4 "Fuster, J. J., Ouchi, N., Gokce, N. & Walsh, K. Obesity-induced changes in adipose tissue microenvironment and their impact on cardiovascular disease. Circ. Res. 118, 1786–1807 (2016)."). This has stimulated research interest in investigating the molecular origin of abdominal obesity and related co-morbidities by focusing on SAT, the key fat depot in expanding and buffering against obesity.

SAT is highly heterogeneous and comprises an array of cell types[5](https://www.nature.com/articles/s41588-025-02300-4#ref-CR5 "Glastonbury, C. A., Couto Alves, A., El-Sayed Moustafa, J. S. & Small, K. S. Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs. Am. J. Hum. Genet. 104, 1013–1024 (2019)."). Single-nucleus RNA sequencing (snRNA-seq) enables the discovery of cell-type-level gene expression patterns in SAT[6](https://www.nature.com/articles/s41588-025-02300-4#ref-CR6 "Das, S. S. et al. Cross-tissue single-nucleus RNA sequencing discovers tissue-resident adipocytes involved in propanoate metabolism in the human heart. Arterioscler. Thromb. Vasc. Biol. 43, 1788–1804 (2023)."),[7](https://www.nature.com/articles/s41588-025-02300-4#ref-CR7 "Kar, A. et al. Age-dependent genes in adipose stem and precursor cells affect regulation of fat cell differentiation and link aging to obesity via cellular and genetic interactions. Genome Med. 16, 19 (2024)."). However, this modality is limited to gene expression, even though SAT function is also influenced by epigenomic processes, such as cytosine DNA methylation at CpG sites (mCG)[8](https://www.nature.com/articles/s41588-025-02300-4#ref-CR8 "Samblas, M., Milagro, F. I. & Martínez, A. DNA methylation markers in obesity, metabolic syndrome, and weight loss. Epigenetics 14, 421–444 (2019)."), and chromatin conformation[9](https://www.nature.com/articles/s41588-025-02300-4#ref-CR9 "Margueron, R. & Reinberg, D. Chromatin structure and the inheritance of epigenetic information. Nat. Rev. Genet. 11, 285–296 (2010)."). Previous studies in other tissues have shown that cell-type-level dynamic mCG in gene regulatory regions and gene bodies affect the expression of genes[10](https://www.nature.com/articles/s41588-025-02300-4#ref-CR10 "Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. Science 382, eadf5357 (2023)."). Furthermore, gene regulatory mechanisms need proper chromatin conformation, which is organized into compartments, domains and loops[11](https://www.nature.com/articles/s41588-025-02300-4#ref-CR11 "Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. Science 353, 598–602 (2016)."). However, the cell-type-level epigenomic landscape underlying the extensive heterogeneity in SAT is poorly understood in humans, which also hinders genetic risk assessment of abdominal obesity, the functional basis of which probably includes specific cell-type-level epigenomic sites.

Single-nucleus methyl-3C sequencing (snm3C-seq) has emerged as a powerful and innovative platform to study DNA methylation and chromatin conformation at cell-type resolution[12](https://www.nature.com/articles/s41588-025-02300-4#ref-CR12 "Lee, D.-S. et al. Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. Nat. Methods 16, 999–1006 (2019)."). Recent studies identified cell-type-level epigenomic signatures in various complex tissues in humans, such as oocytes[13](https://www.nature.com/articles/s41588-025-02300-4#ref-CR13 "Yan, R. et al. Decoding dynamic epigenetic landscapes in human oocytes using single-cell multi-omics sequencing. Cell Stem Cell 28, 1641–1656.e7 (2021)."), prefrontal[12](https://www.nature.com/articles/s41588-025-02300-4#ref-CR12 "Lee, D.-S. et al. Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. Nat. Methods 16, 999–1006 (2019).") and frontal cortex[14](https://www.nature.com/articles/s41588-025-02300-4#ref-CR14 "Luo, C. et al. Single-cell methylomes identify neuronal subtypes and regulatory elements in mammalian cortex. Science 357, 600–604 (2017)."),[15](https://www.nature.com/articles/s41588-025-02300-4#ref-CR15 "Luo, C. et al. Single nucleus multi-omics identifies human cortical cell regulatory genome diversity. Cell Genomics 2, 100107 (2022)."), and other diverse brain regions[10](https://www.nature.com/articles/s41588-025-02300-4#ref-CR10 "Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. Science 382, eadf5357 (2023)."). Using a similar approach, previous studies have also comprehensively assessed the epigenomes of mouse brain cell types[16](https://www.nature.com/articles/s41588-025-02300-4#ref-CR16 "Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nature 598, 120–128 (2021)."),[17](https://www.nature.com/articles/s41588-025-02300-4#ref-CR17 "Liu, H. et al. Single-cell DNA methylome and 3D multi-omic atlas of the adult mouse brain. Nature 624, 366–377 (2023)."),[18](https://www.nature.com/articles/s41588-025-02300-4#ref-CR18 "Flint, J. et al. Single-cell methylation analysis of brain tissue prioritizes mutations that alter transcription. Cell Genomics 3, 100454 (2023)."). However, cell-type-level epigenomic signatures in the human key fat depot SAT have been poorly studied. To address this important biomedical knowledge gap, we determined cell-type-level DNA methylation, chromatin conformation and gene expression signatures in SAT, assessed the involvement of methylation pathway genes in SAT cell-type-level dynamic methylation patterns, identified cell-type-level hypomethylated region-associated transcription factor (TF) binding motifs and investigated the contribution of variants in SAT cell-type-level epigenomic sites to polygenic risk of abdominal obesity and a related inflammatory marker, blood C-reactive protein (CRP).

Results
-------

### Multimodal profiling of human SAT cell types

To investigate the cell-type-level epigenomic landscape of human SAT, we profiled DNA methylation, chromatin conformation and gene expression on nuclei isolated from SAT biopsies from Finnish females (Fig. [1](https://www.nature.com/articles/s41588-025-02300-4#Fig1) and Supplementary Note [1.1](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). We used snm3C-seq to simultaneously profile single-cell-level DNA methylation and chromatin conformation of nuclei isolated from five SAT biopsies (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). A total of 6,652 nuclei passed our quality control (Supplementary Fig. [1a,b](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and Supplementary Note [1.2](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). We independently identified seven main cell types, including adipocytes, adipose stem and progenitor cells (ASPCs), perivascular, endothelial, myeloid, lymphoid and mast cells, using the global mCG of non-overlapping 5 kb bins and the intrachromosomal contacts among non-overlapping 100 kb bins (Fig. [2a](https://www.nature.com/articles/s41588-025-02300-4#Fig2) and Supplementary Figs. [2](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and [3](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Notably, when analyzing the two modalities jointly to derive the de novo snm3C-seq annotation, we discovered a group of nuclei (_n_ = 63 nuclei), present in all five samples (constituting the remaining 0.8 ± 0.6%), that demonstrated inconsistent cell-type annotations between the two modalities (that is, categorized as perivascular cells by mCG and as adipocytes by chromatin conformation) (Fig. [2b](https://www.nature.com/articles/s41588-025-02300-4#Fig2)). We labeled them as a transitional cell-type cluster to highlight their potential developmental stage, observed using the two different omic profiles (Fig. [2a,b](https://www.nature.com/articles/s41588-025-02300-4#Fig2)).

**Fig. 1: Schematic overview of the study design using snm3C-seq and snRNA-seq to profile cell-type-level DNA methylation, chromatin conformation and gene expression in human SAT and partition the genetic risk of abdominal obesity.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02300-4/figures/1)

**a**, Illustration of snm3C-seq and snRNA-seq on nuclei isolated from SAT biopsies from Finnish females. **b**–**g**, Comprehensive analyses of DNA methylation, chromatin conformation and gene expression profiles across SAT cell types to identify cell-type-level differences in DNA methylation patterns (**b**) and chromatin conformation dynamics (**c**). Subsequently, we used the cell-type-level SAT expression data (**d**) to determine whether methylation pathway genes contribute to the observed differences in methylation patterns in SAT cell types and longitudinally cluster with adipogenesis pathway genes (**e**), identify cell-type-level TF binding motifs associated with hypomethylated regions in SAT cell types (**f**) and test the contribution of variants in cell-type-level DMRs and A and B compartments to the genetic risk of abdominal obesity (**g**).

**Fig. 2: Single-nucleus-level multiomic profiles of SAT by jointly profiling methylation and chromatin conformation with snm3C-seq, followed by an integrative analysis with transcriptomic profiles, generated using SAT snRNA-seq.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02300-4/figures/2)

**a**, Dimension reduction of cells using 5 kb bin mCG (top left), 100 kb bin chromatin conformation (top right) and jointly integrating mCG and chromatin conformation (bottom), profiled by snm3C-seq and visualized with uniform manifold approximation and projection (UMAP). Cells are colored by SAT cell type. **b**, Sankey diagram showcases the high consistency among the SAT cell-type annotations derived from the 5 kb bin mCG (left), 100 kb bin chromatin conformation (right) and joint profiling of mCG and chromatin conformation (middle), with the exception of the transition cell-type cluster that was annotated as perivascular cells by mCG and adipocytes by chromatin conformation. **c**–**f**, Integrative analysis with snRNA-seq, evaluating the concordance of cell-type cluster annotations and cell-type marker genes across modalities. **c**, Comparison of gene body mCG and gene expression profiles of cell-type marker genes across the matching SAT cell types, independently identified within the respective modalities, excluding the expression profiles of the transition cell-type cluster that was not identified in the SAT snRNA-seq data. Dashed lines stratify the marker genes by cell type. Dot colors represent the average gene body mCG ratio normalized per cell (left), and the average log-transformed counts per million normalized gene expression (right). **d**, Co-embedding of cells profiled by snm3C-seq and snRNA-seq. Cells are colored by annotations as in **c** (top) and modalities (bottom). **e**, Concordance matrix comparing the snm3C-seq and snRNA-seq derived annotations, colored by the overlapping scores between the pairs of the SAT cell types evaluated in the co-embedding space. **f**, UMAP visualization of the gene body mCG ratio (left) and gene expression (right) for one adipocyte marker gene, _GPAM_, colored per cell similarly as in **c**. The dashed line highlights the group of cells annotated as adipocytes.

We next investigated whether SAT snm3C-seq data can be integrated with SAT snRNA-seq data. First, we applied snRNA-seq on nuclei isolated from the same five SAT samples and three additional SAT samples from the Tilkka cohort to obtain 29,423 SAT nuclei (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)) and annotated them with matching cell-type resolution (Supplementary Note [1.3](https://www.nature.com/articles/s41588-025-02300-4#MOESM1), Extended Data Fig. [1a](https://www.nature.com/articles/s41588-025-02300-4#Fig7) and Supplementary Fig. [1c](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Similar to previous brain results[15](https://www.nature.com/articles/s41588-025-02300-4#ref-CR15 "Luo, C. et al. Single nucleus multi-omics identifies human cortical cell regulatory genome diversity. Cell Genomics 2, 100107 (2022)."), we observed strong and consistent correlations between gene body mCG hypomethylation and RNA expression across the identified cell types (Fig. [2c](https://www.nature.com/articles/s41588-025-02300-4#Fig2)). This correlation enabled us to integrate and co-embed the snm3C-seq and snRNA-seq cells in the shared canonical component space[19](https://www.nature.com/articles/s41588-025-02300-4#ref-CR19 "Stuart, T. et al. Comprehensive integration of single-cell data. Cell 177, 1888–1902.e21 (2019).") (Fig. [2d](https://www.nature.com/articles/s41588-025-02300-4#Fig2), Extended Data Fig. [1b,c](https://www.nature.com/articles/s41588-025-02300-4#Fig7) and Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Overall, these independently performed modality-specific annotations achieved a ≥0.94 overlap score across all cell-type pairs, in which a higher score indicates better integrated cells in the co-embedding space (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)) (Fig. [2e](https://www.nature.com/articles/s41588-025-02300-4#Fig2)). Comparison between the snm3C-seq de novo annotation and its RNA-derived counterpart resulted in an adjusted Rand index of 0.975 and ≥0.95 confusion fraction (Extended Data Fig. [1d](https://www.nature.com/articles/s41588-025-02300-4#Fig7)). Unique cell-type marker genes by these two modalities are shown in Supplementary Tables [1](https://www.nature.com/articles/s41588-025-02300-4#MOESM3) and [2](https://www.nature.com/articles/s41588-025-02300-4#MOESM3). Taken together, the observed cell-type epigenome profiles, identified using snm3C-seq, exhibit strong concordance with those derived from the snRNA-seq transcriptome; however, at the same time, they carry distinct modality-specific information. For example, the expression of a key adipocyte marker gene, _GPAM_, coincides with demethylation of the gene in the co-embedding space, which may allow for the recruitment of relevant proteins; for example, TFs (Fig. [2f](https://www.nature.com/articles/s41588-025-02300-4#Fig2) and Extended Data Fig. [2a–f](https://www.nature.com/articles/s41588-025-02300-4#Fig8)). In summary, mCG and chromatin conformation profiles generated by snm3C-seq robustly recapitulated epigenomic profiles of known major SAT cell types while also uncovering a subtle transition cluster, supporting the differentiation of human adipocytes through a previously less well-characterized route from the perivascular progenitors to adipocytes (Supplementary Note [1.4](https://www.nature.com/articles/s41588-025-02300-4#MOESM1), Fig. [2](https://www.nature.com/articles/s41588-025-02300-4#Fig2), Extended Data Figs. [1](https://www.nature.com/articles/s41588-025-02300-4#Fig7) and [2](https://www.nature.com/articles/s41588-025-02300-4#Fig8) and Supplementary Figs. [5](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and [6](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

### Modality-specific and modality-shared molecular mechanisms

Highly expressed SAT cell-type marker genes, revealed by each modality, may elucidate how epigenomic regulation is involved in biological processes and pathways. For both gene body mCG and gene expression, we identified modality-specific and modality-shared marker genes across all SAT cell types (Supplementary Note [1.5](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and Extended Data Fig. [3a](https://www.nature.com/articles/s41588-025-02300-4#Fig9)) and subsequently in shared and non-shared biological processes (Extended Data Fig. [3b](https://www.nature.com/articles/s41588-025-02300-4#Fig9)) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways enriched among adipocyte marker genes (Extended Data Fig. [3c](https://www.nature.com/articles/s41588-025-02300-4#Fig9)). In adipocytes, for instance, although only 77 marker genes (21% of mCG and 9% of gene expression markers) are present in both modalities, the majority of biological processes (63% of the pathways identified from mCG and 65% from gene expression) and KEGG pathways (65% of the pathways from both mCG and gene expression) are shared (Supplementary Note [1.6](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Focusing on genes involved in the well-characterized PPAR signaling pathway, we observed that six genes, _ACSL1_, _ADIPOQ_, _LPL_, _PCK1_, _PLIN1_ and _PLIN4_, are hypomethylated in adipocytes and the transition cell type, while being hypermethylated in the rest of the cell types. Correspondingly, their transcriptomic profiles reveal that they are predominantly expressed only in adipocytes, with minimal expression in other SAT cell types (Fig. [3a](https://www.nature.com/articles/s41588-025-02300-4#Fig3)). Similarly, fat cell differentiation genes _ADIPOQ_, _LPL_, _LEP_, _TCF7L2_, _AKT2_ and _SREBF1_ are hypomethylated and predominantly expressed in adipocytes compared to the other SAT cell types (Extended Data Fig. [3d](https://www.nature.com/articles/s41588-025-02300-4#Fig9)). Additionally, modality-shared adipocyte marker genes encode TFs, encompassing both known and less-known regulators critical to SAT (Supplementary Note [1.7](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and Extended Data Fig. [3e,f](https://www.nature.com/articles/s41588-025-02300-4#Fig9)). Collectively, these findings suggest a coordinated regulation of key pathways, biological processes and TFs through both transcriptional and epigenomic mechanisms.

**Fig. 3: Functional pathways and gene regulatory potential of cell-type-level gene body mCG markers and DMRs.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02300-4/figures/3)

**a**, Dot plots of PPAR signaling pathway genes (_ACSL1_, _ADIPOQ_, _LPL_, _PCK1_, _PLIN1_ and _PLIN4_) that are shared adipocyte marker genes between the gene body mCG and gene expression modalities, showing their gene body mCG (left) and gene expression profiles (right) across SAT cell types. The color of the dot represents the mean percentage of mCG (left) and average expression of genes (right), while the size of the dot represents the percentage of cells in which the gene is expressed (right). **b**, Horizontal stacked bar plot (left) showing the marginal proportions of assigned methylation states across DMRs for each SAT cell type (n.s., non-significant) and upset plot (right) displaying the top 20 most prevalent methylation state combinations across all detected DMRs, ordered by decreasing frequency, along with their respective percentages. **c**, Circular plot summarizing the cell-type-level TF binding motifs associated with hypomethylated regions in SAT cell types. Statistical significance was determined using HOMER (binomial test) on each cell type separately. The false-positive rate was calibrated by a stringent cutoff on the unadjusted _P_ values (that is, _P_ < 1 × 10−12). Track 1 shows the negative log _P_ values (green lollipops) and track 2 shows the enrichment scores (yellow lollipops). **d**, Violin plots reflecting the empirical null distribution (_n_ = 1,000) of the percentages of DMRs overlapping ChIP-seq peaks for three TFs highlighted in **c** (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). Diamonds mark the observed overlapping proportion of the hypomethylated regions in myeloid cells in the corresponding ChIP-seq experiment. Asterisks (\*) indicate statistical significance, evaluated based on a one-tailed hypergeometric test for overrepresentation of peaks in the corresponding cell-type-level hypomethylated regions (−log10_P_ = 646, 1,860 and 1,618 for IRF4, MEF2B and CEBPG, respectively). GM12878, lymphoblastoid cell line; K562, chronic myelogenous leukemia cell line.

### Striking differences between adipocytes and myeloid DMRs

To delineate the patterns of cell-type-level DNA methylation in SAT, we identified genome-wide DMRs in eight SAT cell types (adipocytes, ASPCs, transition, perivascular, endothelial, myeloid, lymphoid and mast cells) using methylpy[20](https://www.nature.com/articles/s41588-025-02300-4#ref-CR20 "Schultz, M. D. et al. Human body epigenome maps reveal noncanonical DNA methylation variation. Nature 523, 212–216 (2015)."),[21](https://www.nature.com/articles/s41588-025-02300-4#ref-CR21 "He, Y. et al. Spatiotemporal DNA methylome dynamics of the developing mouse fetus. Nature 583, 752–759 (2020).") (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). Overall, 15.4% of the CG sites are differentially methylated across the SAT cell types, with a total of 705,063 CG DMRs covering 5.39% of the genome. These DMRs have a mean (±s.d.) length of 220 ± 152 bp and consist of an average of 4.5 ± 5.5 differentially methylated sites (DMSs). The large numbers of DMRs we identified in SAT cell types support distinct cell-type-level methylation patterns.

We observed striking genome-wide differences in the number and abundance of hypomethylated and hypermethylated regions among the SAT cell types (Fig. [3b](https://www.nature.com/articles/s41588-025-02300-4#Fig3) and Supplementary Table [3](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). In particular, of the total DMRs, 56.3% (_n_ = 396,758; −log10_P_ = 129 using one-tailed _t_\-test; see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)) and 50.6% (_n_ = 356,844; −log10_P_ = 104, one-tailed _t_\-test) are hypomethylated in adipocytes and ASPCs, contrasting with only 14.6% (_n_ = 102,756) in myeloid cells. Conversely, we observed that up to 73.0% of the DMRs demonstrate a hypermethylation pattern in the myeloid cells (_n_ = 514,434; −log10_P_ = 163, one-tailed _t_\-test) versus merely 14.2% in adipocytes and 21.4% in ASPCs. Jointly investigating the differential methylation states across all cell types revealed that 47.3% of the DMRs exhibit opposing profiles between adipocytes, ASPCs and those of the myeloid cells. Taken together, our finding suggests that the widespread repression of regulatory activity in myeloid cells is typically associated with heightened regulatory activity in adipocytes and ASPCs.

### Cell-type-level TF binding motifs

To investigate the relevance of cell-type-level hypomethylated regions in gene regulation, we performed TF binding motif enrichment analysis using cell-type-level hypomethylated regions. We identified several cell-type-level TFs (Supplementary Note [1.8](https://www.nature.com/articles/s41588-025-02300-4#MOESM1), Fig. [3c](https://www.nature.com/articles/s41588-025-02300-4#Fig3), Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02300-4#Fig10) and Supplementary Table [4](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)), suggesting that these TFs might either bind to the DNA in a cell-type-specific manner or regulate cell-type-level differential methylation patterns. We then sought to validate the cell-type-level TFs that we identified using HOMER[22](https://www.nature.com/articles/s41588-025-02300-4#ref-CR22 "Heinz, S. et al. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Mol. Cell 38, 576–589 (2010).") (Fig. [3c](https://www.nature.com/articles/s41588-025-02300-4#Fig3)) by analyzing external chromatin immunoprecipitation sequencing (ChIP-seq) data from ENCODE[23](https://www.nature.com/articles/s41588-025-02300-4#ref-CR23 "ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature 489, 57–74 (2012).") (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). As only immune cell ChIP-seq data are available among the SAT cell types, we focused on validating myeloid-specific TFs (Fig. [3c](https://www.nature.com/articles/s41588-025-02300-4#Fig3)). As expected, binding peaks for the TFs IRF4 and MEF2B in GM12878 demonstrate 3.5-fold and 2.8-fold enrichments compared to the matching genomic background (empirical _P_ < 0.001 for both TFs, −log10_P_ = 1,618 and 1,860 using one-tailed hypergeometric test, respectively) (Fig. [3d](https://www.nature.com/articles/s41588-025-02300-4#Fig3), Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02300-4#Fig11) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). For CEBPG binding peaks in K562, we observed a 2.2-fold enrichment (empirical _P_ < 0.001, −log10_P_ = 647 using one-tailed hypergeometric test) (Fig. [3d](https://www.nature.com/articles/s41588-025-02300-4#Fig3), Extended Data Fig. [5](https://www.nature.com/articles/s41588-025-02300-4#Fig11) and Supplementary Table [5](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). Together, these TF enrichments using ENCODE ChIP-seq data in a relevant cell line independently validated our de novo TF results obtained using HOMER (Fig. [3c](https://www.nature.com/articles/s41588-025-02300-4#Fig3)).

### Enrichment of short-range interactions in SAT adipocytes

Tight packaging of DNA inside the nucleus leads to physical contacts between genomic regions, which affects the gene expression machinery[24](https://www.nature.com/articles/s41588-025-02300-4#ref-CR24 "Dekker, J. & Misteli, T. Long-range chromatin interactions. Cold Spring Harb. Perspect. Biol. 7, a019356 (2015)."). We observed substantial differences in the distribution of closely and distantly located interaction contacts at various genomic distances across the cells profiled by snm3C-seq (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). In pairwise comparisons with other cell types, adipocytes harbor significantly higher proportions of short-range interactions (100 kb to 2 Mb) compared to long-range interactions (10–100 Mb), with the exception of the transition cell type (−log10_P_ > 91; one-tailed Wilcoxon rank-sum test) (Fig. [4a,b](https://www.nature.com/articles/s41588-025-02300-4#Fig4) and Supplementary Note [1.9](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). This observation is in line with all cell-level clustering analyses using chromatin conformation information at various resolutions, which indicate that the transition cell type shares more similarity with adipocytes (Figs. [2b](https://www.nature.com/articles/s41588-025-02300-4#Fig2) and [4c,d](https://www.nature.com/articles/s41588-025-02300-4#Fig4) and Extended Data Fig. [6a,b](https://www.nature.com/articles/s41588-025-02300-4#Fig12)).

**Fig. 4: Analysis of chromatin conformation profiles in SAT reveals cell-type-level diversity in compartments, domains and loops.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02300-4/figures/4)

**a**,**b**, Frequency of contacts per cell against genomic distance (_y_ axis in log scale), grouped by cell types (**a**) and ordered by the median short-range to long-range interaction ratios (**b**). Dashed lines in **a** mark short-range and long-range contacts. The center of the box in **b** represents the median; the bounds of the box indicate the 25th and 75th percentiles and the whiskers show the minimum and maximum values within 1.5 times the interquartile range. Statistical significance was evaluated based on pairwise one-tailed Wilcoxon rank-sum tests for a higher short-range interaction ratio in adipocytes. Asterisks (\*\*\*) indicate unadjusted −log10_P_ > 50. Specifically, unadjusted −log10_P_ = 197, 116, 221, 258, 128 and 91 for ASPCs, perivascular, endothelial, myeloid, lymphoid and mast cells; _n_ = 63, 1,212, 1,814, 482, 1,206, 1,387, 316 and 172 for transition, adipocytes, ASPCs, perivascular, endothelial, myeloid, lymphoid and mast cells, respectively. **c**,**d**, UMAP visualization of low-dimensional embeddings of cells using domains (**c**) and loops (**d**) as features, colored by the snm3C-seq annotation; adjusted Rand index (ARI) evaluates the clustering concordance against snm3C-seq annotation. **e**, Heatmap visualization of the normalized interaction contact map on chromosome 12 and its corresponding compartment scores. **f**, Upset plot (left) visualizing a subset of the differential 100 kb bins and their corresponding percentages; horizontal stacked bar plot (right) showing the marginal A compartment enrichment of differential 100 kb bins. The vertical dotted line separates cell-type-level A compartment flips, B compartment flips and differential bins detected within homogeneous A or B compartments. **g**, Dendrogram of the five most abundant SAT cell types constructed with compartment scores on differential 100 kb bins. **h**, Similar to **g**, except on all annotated SAT cell types, constructed with mCG fractions across DMRs.

In addition, the observed differences in the ratio of short-range to long-range interactions between ASPCs and adipocytes (one-tailed Wilcoxon rank-sum −log10_P_ _\>_ 197) could suggest a link between the contact distances and functionally important genomic regions in adipogenesis. Given that ASPCs develop into adipocytes, we speculate that this change may reflect the unilocular lipid droplet formation in adipocytes that makes them larger than ASPCs.

### Chromosomal conformation dynamics reflect cell-type lineage

Chromatin compartments, which connect stretches of the genome that are tens of megabases apart, reflect how cells arrange their chromosomal structures in three-dimensional space at the highest level[25](https://www.nature.com/articles/s41588-025-02300-4#ref-CR25 "Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014)."). We started our investigation of aggregated SAT cell-type-level genome spatial topology by calculating the compartment scores on the pseudobulk contact matrices for the five most abundant cell types (adipocytes, ASPCs, endothelial, perivascular and myeloid cells) at 100 kb resolution. Based on the sign of the compartment scores, we partitioned the genome into either the active A compartment regions or the more repressive B compartment regions. The correlation matrices derived from the normalized interaction contact maps revealed visually distinct cell-type-level plaid patterns. For example, on chromosome 6 (Extended Data Fig. [6c](https://www.nature.com/articles/s41588-025-02300-4#Fig12)) and chromosome 12 (Fig. [4e](https://www.nature.com/articles/s41588-025-02300-4#Fig4)), endothelial and myeloid cells harbor more intricate structures, indicated by the frequent compartment switches, whereas adipocytes, ASPCs and perivascular cells tend to have longer stretches of region being annotated as the same compartment (Supplementary Note [1.10](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and Extended Data Fig. [6d](https://www.nature.com/articles/s41588-025-02300-4#Fig12)).

Upon a closer inspection, a total of 11,571 100 kb bins, spanning 44.3% of the genome, are statistically differentially conformed among the five cell types at a false discovery rate (FDR) < 0.1 cutoff. The empirical FDR is estimated to be ≤0.02 (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). Although each cell type has its own distinct pattern of chromosomal compartments, differential 100 kb bins consistently demonstrate at least a 1.36-fold enrichment landing in the active A compartments compared to the genome-wide background (Fig. [4f](https://www.nature.com/articles/s41588-025-02300-4#Fig4), Extended Data Fig. [6e](https://www.nature.com/articles/s41588-025-02300-4#Fig12) and Supplementary Table [6](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). Interestingly, across all investigated SAT cell types, the two leading predominant compartment combinations are the homogeneous A and B. These combinations account for 18.8% and 9.7% of the total differentially conformed regions (8.3% and 4.3% of the genome), suggesting significant heterogeneity within each compartment stratification (that is, A and B). These are followed by combinations driven by myeloid and endothelial cells, either through cell-type-level compartment flips (A→B or B→A) or coordinated flips involving both cell types (Supplementary Note [1.10](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and Extended Data Fig. [6e–g](https://www.nature.com/articles/s41588-025-02300-4#Fig12)). Given that we observed similar distinct mCG patterns for endothelial and myeloid cells, we aimed to confirm whether the lineage dendrogram constructed from differential 100 kb bins would mirror the developmental trajectory inferred from DMRs. Indeed, hierarchical clustering consistently grouped endothelial and myeloid cells, characterized by pronounced hypomethylation and frequent compartmental switches, into a distinct branch in both modalities (Fig. [4g,h](https://www.nature.com/articles/s41588-025-02300-4#Fig4)), in line with a previous report showing that myeloid progenitors also give rise to vascular endothelial cells[26](https://www.nature.com/articles/s41588-025-02300-4#ref-CR26 "Bailey, A. S. et al. Myeloid lineage progenitors give rise to vascular endothelium. Proc. Natl Acad. Sci. USA 103, 13156–13161 (2006).").

### Cell-type specificity in regional 3D genome structures

In addition to compartmentalization, the genome maintains its finer spatial structure by forming interaction domains and cohesion-mediated chromatin loops. Analyzing snapshots of the 3D genome at 25 kb and 10 kb resolution allowed us to delineate these regional features at both cell and aggregated cell-type resolution. In addition to the transition cell type, adipocytes showcase an accumulation of significantly denser interaction domains (an average of 4,120 per cell, compared to 3,574 in others; −log10_P_ > 45, pairwise one-tailed Wilcoxon rank-sum test), while spanning a much shorter distance (median of 679,121 bp per cell, compared to 783,213 bp in others; Extended Data Fig. [7a–c](https://www.nature.com/articles/s41588-025-02300-4#Fig13)). Interestingly, the number of detected domains is highly correlated with the ratio of short-range to long-range interactions (Pearson’s correlation coefficient, 0.76; Extended Data Fig. [7d](https://www.nature.com/articles/s41588-025-02300-4#Fig13)). This observation reinforces the idea that regional contacts are necessary to support the more intricate local 3D structures. Both features correlate with the general transcriptomic activity in the matching snRNA-seq data, in which adipocytes show a 1.5-fold increase in the total number of unique molecular identifiers (Extended Data Fig. [7e,f](https://www.nature.com/articles/s41588-025-02300-4#Fig13)). Overlapping cell-type pseudobulk insulation scores with boundary probability, calculated as the fraction of cells with a boundary detected in a given cell type, identified a total of 1,791 differential boundaries (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). Regarding chromatin loops, we detected a median of 47,837 and 5,797 cell-type-level loop pixels and merged loop summits, respectively. Adipocytes demonstrate a similar trend of having more loop summits (_n_ = 8,852) and a relatively shorter median loop length (230,000 bp, compared to others 290,000 bp; Extended Data Fig. [7g,h](https://www.nature.com/articles/s41588-025-02300-4#Fig13)). Along with the clustering results derived from regional interaction features (for example, insulation scores, domains and loops), which show highly concordant annotations (Fig. [4c,d](https://www.nature.com/articles/s41588-025-02300-4#Fig4) and Extended Data Fig. [6a,b](https://www.nature.com/articles/s41588-025-02300-4#Fig12)), we conclude that granular 3D genomic features also exhibit significant heterogeneity across SAT cell types.

### Influence of 3D topology on gene regulatory landscapes

The 3D topology of a cell also influences its transcriptomic dynamics with cell-type specificity. As expected, genes expressed in a cell tend to localize in its active A compartment, exhibiting ≥2.24-fold enrichment relative to the B compartments across the five most abundant cell types. These ratios increase when restricted to the set of cell-type-level unique marker genes. Perivascular cells, in particular, exhibit a staggering 13-fold A/B ratio, leading to 92.8% of the unique marker genes landing in the A compartments (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). We next focused on ASPCs, a cell type that undergoes active differentiation into adipocytes, and systematically evaluated how compartment flipping affects the downstream expression. On a global scale, 23.4% of the differentially conformed regions change from the ASPC A compartments to adipocyte B, while 27.5% convert from the B compartments to A. Residing within these topologically interesting regions are some key cell-type marker genes crucial for adipogenesis (Supplementary Note [1.11](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). For example, _ADIPOQ_ is located in an interaction domain unique to adipocytes with a pronounced demethylation pattern around its gene body, probably facilitating additional TFs to bind and activate it functionally. The adjacent differential boundary marks a stretch of the genome, encapsulating 1 Mb upstream and downstream of _ADIPOQ_, that transitions from the inactive B compartments in ASPCs to the active A compartments in adipocytes (Extended Data Fig. [8a](https://www.nature.com/articles/s41588-025-02300-4#Fig14)). Other strong gene expression de novo adipocyte marker genes, including _TENM3_, _CSMD1_ and _PCDH9_, also land within differential domains specific to adipocytes; additionally, _CSMD1_ and _PCDH9_ have cell-type-level loop domains near the transcription start sites (Extended Data Fig. [8b](https://www.nature.com/articles/s41588-025-02300-4#Fig14) and Supplementary Fig. [7a,b](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Together, our findings suggest that chromosome conformation, ranging from mega-base compartmentalization to kilo-base loop formation, reflects a higher level of carefully balanced coordination among the SAT cell types, influencing both their epigenetic regulation profiles and trickling down to their transcriptional activities.

### Contribution of _DNMT3A_ and _TET1_ to DMR patterns

DNA methylation involves the covalent addition of a methyl group to DNA, a process facilitated by DNA methyltransferase enzymes, such as DNA methyltransferase 3 alpha (DNMT3A) and DNA methyltransferase 3 beta (DNMT3B)[27](https://www.nature.com/articles/s41588-025-02300-4#ref-CR27 "Moore, L. D., Le, T. & Fan, G. DNA methylation and its basic function. Neuropsychopharmacology 38, 23–38 (2013)."). Conversely, DNA demethylation comprises the removal of this methyl group from the DNA by the ten-eleven translocation (TET) family proteins, specifically, TET1, TET2 and TET3 (Fig. [5a](https://www.nature.com/articles/s41588-025-02300-4#Fig5)). Among the demethylase genes, _TET1_ is preferentially expressed in adipocytes (−log10_P_ > 300; two-sided Wilcoxon rank-sum test) (Fig. [5b](https://www.nature.com/articles/s41588-025-02300-4#Fig5)), in line with our observation that adipocytes have significantly more hypomethylated (56.3%) than hypermethylated regions (14.2%) (Fig. [5c,d](https://www.nature.com/articles/s41588-025-02300-4#Fig5)). Among the DNA methyltransferases, our cell-type-level SAT snRNA-seq data show that _DNMT3A_ is predominantly expressed in myeloid cells, with minimal or no expression in adipocytes (−log10_P_ = 136; two-sided Wilcoxon rank-sum test) (Fig. [5b](https://www.nature.com/articles/s41588-025-02300-4#Fig5)). Consistent with our _DNMT3A_ expression results, 73.0% of DMRs are hypermethylated in the myeloid cells while only 14.2% are hypermethylated in adipocytes (Fig. [5c](https://www.nature.com/articles/s41588-025-02300-4#Fig5)). In addition, the expression of a methylation maintenance gene, _DNMT1_, is significantly lower in adipocytes than other SAT cell types (−log10_P_ = 170; two-sided Wilcoxon rank-sum test) (Extended Data Fig. [9a](https://www.nature.com/articles/s41588-025-02300-4#Fig15)). Expression of other methylation (_DNMT3B_ and _UHRF1_) and demethylation genes (_TET2, TET3_ and _TDG_), shown in Extended Data Fig. [9a,b](https://www.nature.com/articles/s41588-025-02300-4#Fig15), suggest that _TET1_ and _DNMT3A_ are the most important genes that contribute to the observed adipocyte hypomethylation and myeloid hypermethylation patterns, indicating their potential mechanistic role in cell-type-level DNA methylation signatures in SAT. The SAT bulk RNA-seq dataset further revealed that bulk expression of _DNMT3A_ is negatively associated with insulin sensitivity using the Matsuda Index, whereas bulk expression of _TET1_ is positively correlated with the Matsuda Index (Supplementary Note [1.12](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and Supplementary Table [8](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). In addition, we observed that _TET1_ is temporally co-expressed with known adipogenesis genes across human primary preadipocyte differentiation (Supplementary Note [1.13](https://www.nature.com/articles/s41588-025-02300-4#MOESM1), Fig. [5e,f](https://www.nature.com/articles/s41588-025-02300-4#Fig5) and Supplementary Table [9](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). Taken together, the cell-type-level expression patterns of _DNMT3A_ in myeloid cells and _TET1_ in adipocytes suggest that their association signals with CMD traits we observed in the SAT bulk expression may largely originate from their high expression in these two specific cell types.

**Fig. 5: Analysis of mean gene expression and DMRs across SAT cell types reveals the potential involvement of DNA methylation pathway genes in regulating cell-type-level hypermethylation and hypomethylation in SAT.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02300-4/figures/5)

**a**, A schematic representation of basic mechanisms and key players in DNA methylation and demethylation. **b**, Dot plot of _TET1_ and _DNMT3A_ showing their expression profiles across the SAT cell types. The size of the dot represents the percentage of cells in which a gene is expressed within a cell type, and the color represents the average expression of each gene across all cells within a cell type (blue indicates higher expression). **c**, Proportions of assigned hypomethylated (left) and hypermethylated states (right) across DMRs. **d**, UMAP visualization of the average global mCG ratio in a cell. The dashed line highlights those annotated as myeloid cells. **e**, Bar plot reflecting the distribution of normalized mCG fractions across genes that co-clustered with _TET1_ in **f** for ASPCs and adipocytes. Asterisk (\*) indicates statistical significance from a paired one-tailed Wilcoxon rank-sum test, comparing the median cluster expression across _n_ = 5 samples, showing higher expression in ASPCs than in adipocytes (unadjusted _P_ = 0.031). The center of the bar represents the median, and the error bars represent the highest and lowest expression across _n_ = 5 samples. **f**, Longitudinal expression of _TET1_ is plotted across the 14-day SAT preadipocyte differentiation. The shaded ribbon behind the trajectory of _TET1_ reflects the mean and standard deviation of the genes that clustered into similar trajectory patterns as _TET1_ using DPGP.

### Variants in DMRs and compartments are enriched for CMD risk

We next explored the genetic contributions to key CMDs relevant to SAT from variants residing in the cell-type-level compartments and DMRs. Specifically, we assessed abdominal obesity using waist-hip-ratio adjusted for body mass index (WHRadjBMI) as its well-established proxy[28](https://www.nature.com/articles/s41588-025-02300-4#ref-CR28 "Emdin, C. A. et al. Genetic association of waist-to-hip ratio with cardiometabolic traits, type 2 diabetes, and coronary heart disease. JAMA 317, 626–634 (2017)."),[29](https://www.nature.com/articles/s41588-025-02300-4#ref-CR29 "Yusuf, S. et al. Obesity and the risk of myocardial infarction in 27,000 participants from 52 countries: a case–control study. Lancet 366, 1640–1649 (2005)."), body mass index (BMI), CRP and metabolic dysfunction-associated steatotic liver disease (MASLD) in the UK Biobank (UKB)[30](https://www.nature.com/articles/s41588-025-02300-4#ref-CR30 "Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)."),[31](https://www.nature.com/articles/s41588-025-02300-4#ref-CR31 "Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018).").

We first evaluated the partitioned polygenic risk of variants landing in cell-type-level compartments and DMRs (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). Among the four traits, we highlight WHRadjBMI. All WHRadjBMI polygenic risk scores (PRSs) built from variants in the cell-type-level A compartments are significant (_P_R2 < 0.05) and enriched predictors (_P_perm10,000 < 0.05), explaining ≥80% of variance in WHRadjBMI captured by the full genome (Fig. [6a](https://www.nature.com/articles/s41588-025-02300-4#Fig6), Extended Data Fig. [10a,b](https://www.nature.com/articles/s41588-025-02300-4#Fig16) and Supplementary Table [10](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). Conversely, we observed no PRS enrichment from the B compartments. We also observed enrichment of the WHRadjBMI polygenic risk from variants in cell-type-level DMRs and strong genetic contributions to CMDs from the target eGenes of the SAT _cis_\-expression quantitative trait locus variants, residing in these cell-type-level DMRs using HuGE scoring[32](https://www.nature.com/articles/s41588-025-02300-4#ref-CR32 "Dornbos, P. et al. Evaluating human genetic support for hypothesized metabolic disease genes. Cell Metab. 34, 661–666 (2022).") (Supplementary Note [1.14](https://www.nature.com/articles/s41588-025-02300-4#MOESM1), Fig. [6b](https://www.nature.com/articles/s41588-025-02300-4#Fig6) and Supplementary Tables [11](https://www.nature.com/articles/s41588-025-02300-4#MOESM3) and [12](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). Additionally, we found that PRSs constructed from the A compartments and the hypomethylated regions of myeloid cells are enriched predictors of CRP, suggesting a low-grade inflammatory role for this immune cell type (Supplementary Note [1.14](https://www.nature.com/articles/s41588-025-02300-4#MOESM1), Fig. [6c](https://www.nature.com/articles/s41588-025-02300-4#Fig6) and Supplementary Tables [10](https://www.nature.com/articles/s41588-025-02300-4#Sec38) and [11](https://www.nature.com/articles/s41588-025-02300-4#Sec38)).

**Fig. 6: Partitioned PRS and GWAS enrichment results for four key cardiometabolic traits, stratified by cell-type-level DMRs and compartments.**

[![figure 6](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_Fig6_HTML.png)](https://www.nature.com/articles/s41588-025-02300-4/figures/6)

**a**,**b**, Lollipop plots depict the incremental variance explained of each cell-type-level PRS for abdominal obesity (using WHRadjBMI as a proxy) from the (**a**) A and B compartments and (**b**) hypomethylated and hypermethylated regions. Each lollipop represents a WHRadjBMI PRS, in which dot size corresponds to the incremental variance explained of the PRS. Significant enrichment of incremental variance explained was determined empirically by comparing to _n_ = 10,000 permutated PRSs (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). The gray vertical dashed line indicates the significance cutoff based on unadjusted one-tailed permutation _P_ values (that is, _P_perm10,000 < 0.05). Bars and lollipops corresponding to cell types with significantly enriched PRSs are colored by cell type, whereas those without are outlined in gray without a filling. **c**, Summary table visualizing the overall PRS and GWAS enrichment results across four key cardiometabolic traits (that is, WHRadjBMI, MASLD, BMI and CRP), stratified by cell-type-level hypomethylated regions and A compartments. Orange indicates nominal significance (_P_ < 0.05) on the one-tailed permutation _P_ values for PRSs and the one-tailed hypergeometric _P_ values for GWAS enrichment. The bolded black border highlights the following consistently significant cell-type-cardiometabolic trait pair: adipocyte–WHRadjBMI.

Next, we examined the cell-type-level compartments and DMRs for overrepresentation of independent GWAS variants for the four outcomes. We observed that all five cell-type-level A compartments are enriched for both WHRadjBMI and CRP GWAS variants (Fig. [6c](https://www.nature.com/articles/s41588-025-02300-4#Fig6), Extended Data Fig. [10c](https://www.nature.com/articles/s41588-025-02300-4#Fig16) and Supplementary Table [13](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)). Adipocyte A compartments are additionally enriched for BMI and MASLD GWAS variants. For DMRs, we observed enrichment of WHRadjBMI GWAS variants in adipocyte and myeloid hypomethylated and in ASPC and perivascular hypermethylated regions, as well as strong genetic contributions to obesity-related outcomes from their adjacent genes using HuGE scoring (Supplementary Note [1.14](https://www.nature.com/articles/s41588-025-02300-4#MOESM1), Fig. [6c](https://www.nature.com/articles/s41588-025-02300-4#Fig6) and Supplementary Table [14](https://www.nature.com/articles/s41588-025-02300-4#MOESM3)).

Overall, our results highlight the SAT cell-type-level methylation and spatial conformation profiles as important contexts underlying the genetic risk of CMDs, with the hypomethylated regions and the active A compartments of adipocytes giving the strongest signal for abdominal obesity risk and the A compartments of myeloid cells more indicative of inflammation risk.

Discussion
----------

Delineating the cell-type-level epigenomic landscape in human SAT is crucial for understanding its regulatory mechanisms and impact on obesity risk. Our data revealed a highly dynamic reciprocal interplay between the SAT cell-type-level epigenomes, particularly between adipocytes and myeloid cells. We further integrated the differential epigenomic sites with variant-level data in the UKB, thus uncovering their significant contributions to the genetic risks of CMDs, most notably abdominal obesity. Finally, we elucidated the potential role of specific methylation and demethylation pathway genes in the cell-type-level differential methylation of human SAT.

Focusing on bulk methylome profiles, previous studies have reported differential DNA methylation patterns at the tissue level in SAT and their association with obesity[33](https://www.nature.com/articles/s41588-025-02300-4#ref-CR33 "Benton, M. C. et al. An analysis of DNA methylation in human adipose tissue reveals differential modification of obesity genes before and after gastric bypass and weight loss. Genome Biol. 16, 8 (2015)."),[34](https://www.nature.com/articles/s41588-025-02300-4#ref-CR34 "Mirzaeicheshmeh, E. et al. Alterations of DNA methylation during adipogenesis differentiation of mesenchymal stem cells isolated from adipose tissue of patients with obesity is associated with type 2 diabetes. Adipocyte 10, 493–504 (2021)."). However, underlying non-captured cell-type-level methylation patterns and composition often confound tissue-level analyses[35](https://www.nature.com/articles/s41588-025-02300-4#ref-CR35 "Jaffe, A. E. & Irizarry, R. A. Accounting for cellular heterogeneity is critical in epigenome-wide association studies. Genome Biol. 15, R31 (2014)."). Our SAT cell-type-level methylation profiles and DMRs could serve as reference panels and provide informative features for computationally decomposing the heterogeneous SAT mixtures[36](https://www.nature.com/articles/s41588-025-02300-4#ref-CR36 "Zheng, S. C., Breeze, C. E., Beck, S. & Teschendorff, A. E. Identification of differentially methylated cell types in epigenome-wide association studies. Nat. Methods 15, 1059–1066 (2018)."),[37](https://www.nature.com/articles/s41588-025-02300-4#ref-CR37 "Rahmani, E. et al. Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies. Nat. Methods 13, 443–445 (2016)."),[38](https://www.nature.com/articles/s41588-025-02300-4#ref-CR38 "Thompson, M., Chen, Z. J., Rahmani, E. & Halperin, E. CONFINED: distinguishing biological from technical sources of variation by leveraging multiple methylation datasets. Genome Biol. 20, 138 (2019)."), a critical step for reducing false discoveries in tissue-level studies and facilitating cell-type-specific biomarker identification[36](https://www.nature.com/articles/s41588-025-02300-4#ref-CR36 "Zheng, S. C., Breeze, C. E., Beck, S. & Teschendorff, A. E. Identification of differentially methylated cell types in epigenome-wide association studies. Nat. Methods 15, 1059–1066 (2018)."),[39](https://www.nature.com/articles/s41588-025-02300-4#ref-CR39 "Rahmani, E. et al. Cell-type-specific resolution epigenetics without the need for cell sorting or single-cell biology. Nat. Commun. 10, 3417 (2019)."),[40](https://www.nature.com/articles/s41588-025-02300-4#ref-CR40 "Chen, Z. J., Rahmani, E. & Halperin, E. A unified model for cell-type resolution genomics from heterogeneous omics data. Preprint at bioRxiv 
https://doi.org/10.1101/2024.01.27.577588
(2024)."),[41](https://www.nature.com/articles/s41588-025-02300-4#ref-CR41 "Gorla, A. et al. Phenotypic subtyping via contrastive learning. Preprint at bioRxiv 
https://doi.org/10.1101/2023.01.05.522921
(2023)."),[42](https://www.nature.com/articles/s41588-025-02300-4#ref-CR42 "Gorla, A. et al. Epigenetic patient stratification via contrastive machine learning refines hallmark biomarkers in minoritized children with asthma. Preprint at Res. Sq. 
https://doi.org/10.21203/rs.3.rs-5066762/v1
(2024)."). The cell-type composition itself could also hold substantial clinical implications. For example, previous research using transcriptome profiling of perigonadal adipose tissue in mice and immunohistochemistry of human SAT has suggested that the accumulation of myeloid cells, particularly macrophages, correlates with increased adiposity[43](https://www.nature.com/articles/s41588-025-02300-4#ref-CR43 "Weisberg, S. P. et al. Obesity is associated with macrophage accumulation in adipose tissue. J. Clin. Invest. 112, 1796–1808 (2003).").

Among the identified cell-type-level 3D genome structures in SAT, adipocytes showcase a distinct regional topology, with a 1.51-fold enrichment in relative short-range interactions, a 1.15-fold increase in the number of domains and a 1.72-fold increase in overall transcriptomic activity. Similar patterns have been observed in other human solid tissues, and notably, these types of differences in the non-neuronal cells in the brain have been linked to larger nuclear size[10](https://www.nature.com/articles/s41588-025-02300-4#ref-CR10 "Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. Science 382, eadf5357 (2023)."),[44](https://www.nature.com/articles/s41588-025-02300-4#ref-CR44 "Tan, L. et al. Lifelong restructuring of 3D genome architecture in cerebellar granule cells. Science 381, 1112–1119 (2023)."),[45](https://www.nature.com/articles/s41588-025-02300-4#ref-CR45 "Heffel, M. G. et al. Temporally distinct 3D multi-omic dynamics in the developing human brain. Nature 635, 481–489 (2024)."). The presence of the key adipocyte marker gene and adipokine _ADIPOQ_ in a genomic region differentially conformed between ASPCs and adipocytes while heavily demethylated in adipocytes further supports the idea that epigenomic structures reorganize during cell differentiation[46](https://www.nature.com/articles/s41588-025-02300-4#ref-CR46 "Chakraborty, A., Wang, J. G. & Ay, F. dcHiC detects differential compartments across multiple Hi-C datasets. Nat. Commun. 13, 6827 (2022)."), ultimately regulating downstream, regional and cell-type-level gene expression.

In our cell-type-level investigations of methylation pathway genes, we found notably high expression of _TET1_ in adipocytes and _DNMT3A_ in myeloid cells, supporting a tissue and cell-type-level reciprocal coordination and cross-talk between _TET1_ expression and hypomethylation in adipocytes and _DNMT3A_ expression and hypermethylation in myeloid cells. A previous study showed that _TET1_ is an important DNA demethylase in adipose bulk tissue[47](https://www.nature.com/articles/s41588-025-02300-4#ref-CR47 "Qian, H. et al. TET1 promotes RXRα expression and adipogenesis through DNA demethylation. Biochim. Biophys. Acta Mol. Cell. Biol. Lipids 1866, 158919 (2021)."). An earlier study reported the involvement of _TET1_ in adipocytokine promoter hypomethylation in adipocytes[48](https://www.nature.com/articles/s41588-025-02300-4#ref-CR48 "Ali, M. M., Phillips, S. A. & Mahmoud, A. M. HIF1α/TET1 pathway mediates hypoxia-induced adipocytokine promoter hypomethylation in human adipocytes. Cells 9, 134 (2020)."). Furthermore, prior studies have also demonstrated that both _TET1_ and _DNMT3A_ compete to regulate epigenetic mechanisms[49](https://www.nature.com/articles/s41588-025-02300-4#ref-CR49 "Gu, T. et al. DNMT3A and TET1 cooperate to regulate promoter epigenetic landscapes in mouse embryonic stem cells. Genome Biol. 19, 88 (2018)."). Although previous studies have investigated _TET1_ (refs. [47](https://www.nature.com/articles/s41588-025-02300-4#ref-CR47 "Qian, H. et al. TET1 promotes RXRα expression and adipogenesis through DNA demethylation. Biochim. Biophys. Acta Mol. Cell. Biol. Lipids 1866, 158919 (2021)."),[50](https://www.nature.com/articles/s41588-025-02300-4#ref-CR50 "Damal et al. TET1 is a beige adipocyte-selective epigenetic suppressor of thermogenesis. Nat. Commun. 11, 4313 (2020)."),[51](https://www.nature.com/articles/s41588-025-02300-4#ref-CR51 "Byun, S. et al. Loss of adipose TET proteins enhances β-adrenergic responses and protects against obesity by epigenetic regulation of β3-AR expression. Proc. Natl Acad. Sci. USA 119, e2205626119 (2022).")) and _DNMT3A_[52](https://www.nature.com/articles/s41588-025-02300-4#ref-CR52 "Tovy, A. et al. Constitutive loss of DNMT3A causes morbid obesity through misregulation of adipogenesis. Elife 11, e72359 (2022)."),[53](https://www.nature.com/articles/s41588-025-02300-4#ref-CR53 "You, D. et al. Dnmt3a is an epigenetic mediator of adipose insulin resistance. Elife 6, e30766 (2017).") in adipose bulk tissue level or some of the SAT cell types, we investigated their expression at the single-cell level in all main SAT cell types from the human SAT biopsies, thus providing more granular information about these key methylation genes.

Abdominal obesity is highly polygenic[54](https://www.nature.com/articles/s41588-025-02300-4#ref-CR54 "Christiansen, M. R., Kilpeläinen, T. O. & McCaffery, J. M. Abdominal obesity genetic variants predict waist circumference regain after weight loss. Diabetes 72, 1424–1432 (2023)."). Previous studies have successfully built predictive genome-wide PRSs for abdominal obesity[55](https://www.nature.com/articles/s41588-025-02300-4#ref-CR55 "Pan, D. Z. et al. Identification of TBX15 as an adipose master trans regulator of abdominal obesity genes. Genome Med. 13, 123 (2021)."),[56](https://www.nature.com/articles/s41588-025-02300-4#ref-CR56 "Khera, A. V. et al. Polygenic prediction of weight and obesity trajectories from birth to adulthood. Cell 177, 587–596.e9 (2019).") and shown that a high genetic predisposition to abdominal obesity predicts the regain of abdominal obesity following weight loss[54](https://www.nature.com/articles/s41588-025-02300-4#ref-CR54 "Christiansen, M. R., Kilpeläinen, T. O. & McCaffery, J. M. Abdominal obesity genetic variants predict waist circumference regain after weight loss. Diabetes 72, 1424–1432 (2023)."). However, less is known about the characteristics of specific genomic regions that contribute most to the polygenic risk of abdominal obesity, which could ultimately improve individual disease risk assessment. By constructing the partitioned PRSs and examining the presence of the independent strongest GWAS variants for abdominal obesity based on the two epigenomic modalities, we discovered that the GWAS variants landing in both the hypomethylated regions and those in the active A compartments of adipocytes show robust associations for abdominal obesity.

Our study has some limitations. First, even though snm3C-seq reveals both intrachromosomal and inter-chromosomal interactions, we have only presented extensive analyses of the former. Although we recognize the potential interest in inter-chromosomal interactions, single-cell 3D genome data currently remains relatively sparse and does not yet support robust and reliable identification of inter-chromosomal interactions[10](https://www.nature.com/articles/s41588-025-02300-4#ref-CR10 "Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. Science 382, eadf5357 (2023)."),[12](https://www.nature.com/articles/s41588-025-02300-4#ref-CR12 "Lee, D.-S. et al. Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. Nat. Methods 16, 999–1006 (2019)."),[16](https://www.nature.com/articles/s41588-025-02300-4#ref-CR16 "Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nature 598, 120–128 (2021)."),[17](https://www.nature.com/articles/s41588-025-02300-4#ref-CR17 "Liu, H. et al. Single-cell DNA methylome and 3D multi-omic atlas of the adult mouse brain. Nature 624, 366–377 (2023)."). Second, generating snm3C-seq data in frozen human solid tissues remains time-consuming and expensive, requiring relatively large biopsies compared to other single-cell omics platforms. As a result, this study only includes Finnish females without obesity. Inclusion of males and individuals with obesity would help elucidate potential sex-specific epigenomic landscapes and differences across the various BMI categories. Similarly, larger sets of samples from additional populations are also warranted. Third, inclusion of visceral adipose tissue data into future studies would uncover cell-type-level methylation and chromatin conformation patterns in this other metabolically important adipose depot as well as their differences compared to SAT. Nevertheless, taken together, our study provides a valuable insight into the cell-type-level epigenomes in human SAT to be followed up in future studies.

Methods
-------

### Ethics

The Tilkka study was approved by the Helsinki University Hospital Ethics Committee (approval no. HUS/1039/2019). The UKB study was approved by the North West Multi-centre Research Ethics Committee (approval no. 21/NW/0157). The Finnish METabolic Syndrome In Men (METSIM) study was approved by the Ethics Committee of the Northern Savo Hospital District (approval no. 171/2004). All participants provided written informed consent. All research conformed to the principles of the Declaration of Helsinki. No compensation was provided to the participants.

### Tilkka cohort

Eight Finnish females without obesity underwent abdominal SAT liposuction at Tilkka Hospital, Helsinki, Finland. We performed snRNA-seq on all eight SAT biopsies and snm3C-seq on five SAT biopsies. The mean age (±s.d.) for the snRNA-seq cohort was 44.5 ± 6.03 years, and for the snm3C-seq cohort, the mean age was 44.4 ± 4.78 years.

### UKB cohort

For our GWAS enrichment and PRS analyses, we used genotype and phenotype data from 391,701 unrelated individuals (54.0% female; mean age, 56.9 ± 8.0 years) of European origin from the UKB[30](https://www.nature.com/articles/s41588-025-02300-4#ref-CR30 "Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)."),[31](https://www.nature.com/articles/s41588-025-02300-4#ref-CR31 "Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)."). Data for UKB were collected across 22 assessment centers[30](https://www.nature.com/articles/s41588-025-02300-4#ref-CR30 "Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)."),[31](https://www.nature.com/articles/s41588-025-02300-4#ref-CR31 "Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)."). Genotype data were obtained using either the Applied Biosystems UK BiLEVE Axiom Array or Applied Biosystems UKB Axiom Array and imputed with the Haplotype Reference Consortium and the merged UK10K and 1000 Genomes phase 3 reference panels[30](https://www.nature.com/articles/s41588-025-02300-4#ref-CR30 "Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)."),[31](https://www.nature.com/articles/s41588-025-02300-4#ref-CR31 "Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)."). Data from UKB were accessed under application 33934.

### METSIM cohort

Unrelated men (_n_ = 335; mean age, 54.1 ± 4.9 years) from the METSIM cohort[57](https://www.nature.com/articles/s41588-025-02300-4#ref-CR57 "Laakso, M. et al. The Metabolic Syndrome in Men study: a resource for studies of metabolic and cardiovascular diseases. J. Lipid Res. 58, 481–493 (2017).") underwent SAT biopsies for bulk RNA-seq. These METSIM participants were recruited from the University of Eastern Finland and Kuopio University Hospital, Kuopio, Finland.

### In situ chromatin conformation capture and fluorescence-activated nuclei sorting

We performed in situ chromatin conformation capture using an Arima Genomics Arima-HiC Kit[45](https://www.nature.com/articles/s41588-025-02300-4#ref-CR45 "Heffel, M. G. et al. Temporally distinct 3D multi-omic dynamics in the developing human brain. Nature 635, 481–489 (2024).") with the following modification: the amount of Triton-X 100 in the NIBT buffer was increased to 1% to account for the large amount of fat in adipose tissue. Fluorescence-activated nuclei sorting and library preparation were performed using the snmC-seq3 workflow ([https://www.protocols.io/view/snm3c-seq3-kqdg3x6ezg25](https://www.protocols.io/view/snm3c-seq3-kqdg3x6ezg25)). The snmC-seq3 libraries of human SAT were sequenced using an Illumina NovaSeq 6000 instrument with S4 flow cells, generating 150 bp paired-end reads. The sequencing reads of snm3C-seq were mapped using Taurus-MH[12](https://www.nature.com/articles/s41588-025-02300-4#ref-CR12 "Lee, D.-S. et al. Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. Nat. Methods 16, 999–1006 (2019).") ([https://github.com/luogenomics/Taurus-MH](https://github.com/luogenomics/Taurus-MH)).

### Snm3C-seq quality control and preprocessing

We filtered the cells profiled by snm3C-seq based on the following metrics[10](https://www.nature.com/articles/s41588-025-02300-4#ref-CR10 "Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. Science 382, eadf5357 (2023)."),[12](https://www.nature.com/articles/s41588-025-02300-4#ref-CR12 "Lee, D.-S. et al. Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. Nat. Methods 16, 999–1006 (2019)."): (1) estimated non-conversion rate of mCCC% < 0.015; (2) global mCG% > 0.5; (3) global mCH% < 0.15; (4) total number of interaction contacts >100,000 and < 500,000; and (5) at least one intra-chromosome contact present in each autosome after filtering out reads with either end mapped to the ENCODE blacklist regions[58](https://www.nature.com/articles/s41588-025-02300-4#ref-CR58 "Amemiya, H. M., Kundaje, A. & Boyle, A. P. The ENCODE Blacklist: identification of problematic regions of the genome. Sci. Rep. 9, 9354 (2019).") (Supplementary Fig. [1a](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

### Genotype quality control and imputation in the Tilkka cohort

We genotyped the DNAs from the Tilkka participants using the Infinium Global Screening Array-24 (v.1) (Illumina). In our quality control, we used PLINK (v.1.9)[59](https://www.nature.com/articles/s41588-025-02300-4#ref-CR59 "Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015).") to remove individuals with missingness of >2%; unmapped, strand-ambiguous and monomorphic single nucleotide polymorphisms; and variants with missingness of >2% and Hardy–Weinberg equilibrium _P_ < 10−6. In addition, we imputed biological sex using the ‘–sex-check’ function in PLINK (v.1.9)[59](https://www.nature.com/articles/s41588-025-02300-4#ref-CR59 "Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015).") and confirmed that they matched the reported sex for all individuals.

We used the HRC reference panel (v.r1.1 2016)[60](https://www.nature.com/articles/s41588-025-02300-4#ref-CR60 "McCarthy, S. et al. A reference panel of 64,976 haplotypes for genotype imputation. Nat. Genet. 48, 1279–1283 (2016).") to perform genotype imputation on the Michigan imputation server. Before imputation, we removed duplicate variants as well as variants with allele mismatch with the HRC reference panel, and we matched strand flips and allele switches to the panel before haplotype phasing using Eagle (v.2.4)[61](https://www.nature.com/articles/s41588-025-02300-4#ref-CR61 "Loh, P.-R. et al. Reference-based phasing using the Haplotype Reference Consortium panel. Nat. Genet. 48, 1443–1448 (2016)."). To perform the genotype imputation, we used minimac4 (ref. [62](https://www.nature.com/articles/s41588-025-02300-4#ref-CR62 "Das, S. et al. Next-generation genotype imputation service and methods. Nat. Genet. 48, 1284–1287 (2016).")) and performed quality control on the data by removing single nucleotide polymorphisms with an imputation score R2 < 0.3 and Hardy–Weinberg equilibrium _P_ < 10−6.

### SnRNA-seq of human SAT in the Tilkka cohort

We performed SAT snRNA-seq experiments (Supplementary Note [2.1](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)) on the snap-frozen SAT biopsies from the Tilkka participants. We measured the concentration and quality of nuclei, separately for each sample, using a Countess II FL Automated Cell Counter after staining with trypan blue and Hoechst dyes. To construct the libraries, we used the Single Cell 3′ Reagent Kit (v.3.1) (10× Genomics) and analyzed the quality of cDNA and gene expression using an Agilent Bioanalyzer. We sequenced the libraries from each participant together on an Illumina NovaSeq S4 with a target sequencing depth of 600 million read pairs. Processing of the SAT snRNA-seq data from the Tilkka cohort is described in Supplementary Note [2.2](https://www.nature.com/articles/s41588-025-02300-4#MOESM1).

### Snm3C-seq data integration, clustering and annotation

We aligned the raw snm3C-seq reads against the GRCh38 human genome reference and represented the mCG profiles of each cell by 5 kb bins across autosomal chromosomes. In brief, per cell and for each 5 kb bin, we calculated a hypomethylation score. Bins that overlapped with the ENCODE blacklist regions were excluded from the clustering analysis. Next, we performed latent semantic indexing to obtain the mCG profile embedding (Supplementary Note [2.3](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). For the chromosome conformation modality, we imputed the contact matrix of each cell at 100 kb resolution using scHicluster[63](https://www.nature.com/articles/s41588-025-02300-4#ref-CR63 "Zhou, J. et al. Robust single-cell Hi-C clustering by convolution- and random-walk-based imputation. Proc. Natl Acad. Sci. USA 116, 14011–14018 (2019).") (v.1.3.5) with pad = 1 and used singular value decomposition to project all intra-chromosome contacts between 100 kb and 10 Mb that land in autosomal chromosomes to a low-dimensional space (Supplementary Note [2.3](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

We annotated the clusters de novo by leveraging the negative correlation between mCG and transcriptional activity[10](https://www.nature.com/articles/s41588-025-02300-4#ref-CR10 "Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. Science 382, eadf5357 (2023)."),[15](https://www.nature.com/articles/s41588-025-02300-4#ref-CR15 "Luo, C. et al. Single nucleus multi-omics identifies human cortical cell regulatory genome diversity. Cell Genomics 2, 100107 (2022)."),[16](https://www.nature.com/articles/s41588-025-02300-4#ref-CR16 "Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nature 598, 120–128 (2021).") and the known SAT marker genes, reported previously[64](https://www.nature.com/articles/s41588-025-02300-4#ref-CR64 "Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)."). Specifically, we aligned the snm3C-seq data to GENCODE (v.33) annotations and calculated the average mCG fractions of the gene body (Supplementary Note [2.4](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

We then tested for de novo gene body hypomethylation patterns across all protein-coding genes for each snm3C-seq cluster, defined using both the global mCG and chromatin conformation embeddings. To identify top cluster markers, we adapted criteria similar to those used for CH methylation in the brain[16](https://www.nature.com/articles/s41588-025-02300-4#ref-CR16 "Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nature 598, 120–128 (2021)."), applying them to mCG in SAT. In more detail, we required a hypomethylation difference of ≥0.15 in the average normalized fraction compared to other clusters, a Benjamini–Hochberg-adjusted Wilcoxon rank-sum test _P_ value of ≤0.01 and an area under the receiver operating characteristic curve of ≥0.75 for distinguishing cells within the cluster from those outside it. Next, we leveraged the existing snRNA atlas of SAT cells for annotation[64](https://www.nature.com/articles/s41588-025-02300-4#ref-CR64 "Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022).") (Supplementary Note [2.4](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

For modality-specific analysis of snm3C-seq cells (that is, those independently identified within the global mCG or chromatin conformation profiles), we performed separate clustering, visualization and annotation similarly. The final annotations for each modality-specific cluster were determined by identifying the most common snm3C-seq cell-type labels among its constituent cells (Fig. [2b](https://www.nature.com/articles/s41588-025-02300-4#Fig2)). We categorized the group of cells that clustered with adipocytes if using only the chromosome conformation information and with perivascular cells if using only the mCG profiles as the ‘transition’ cluster.

### SnRNA-seq data integration, clustering and annotation

We integrated all remaining high-quality droplets from all snRNA-seq data with reciprocal principal components analysis (rPCA) implemented in Seurat (v.4.3.0)[65](https://www.nature.com/articles/s41588-025-02300-4#ref-CR65 "Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587.e29 (2021).") and clustered integrated data with a standard Louvain algorithm, using parameters of the first 30 principal components and a resolution of 0.5. We annotated each cluster with its cell type using SingleR (v.1.8.1)[66](https://www.nature.com/articles/s41588-025-02300-4#ref-CR66 "Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Nat. Immunol. 20, 163–172 (2019)."), using a previously published single-cell atlas of human SAT as a ref. [64](https://www.nature.com/articles/s41588-025-02300-4#ref-CR64 "Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022).").

### Co-embedding of snm3C-seq and snRNA-seq data

We aligned the snm3C-seq cells as the query and snRNA-seq cells as the refs. [18](https://www.nature.com/articles/s41588-025-02300-4#ref-CR18 "Flint, J. et al. Single-cell methylation analysis of brain tissue prioritizes mutations that alter transcription. Cell Genomics 3, 100454 (2023)."),[67](https://www.nature.com/articles/s41588-025-02300-4#ref-CR67 "Chen, P. B. et al. Complementation testing identifies genes mediating effects at quantitative trait loci underlying fear-related behavior. Cell Genomics 4, 100545 (2024).") under the canonical correlation analysis framework in Seurat (v.4.1.0)[19](https://www.nature.com/articles/s41588-025-02300-4#ref-CR19 "Stuart, T. et al. Comprehensive integration of single-cell data. Cell 177, 1888–1902.e21 (2019)."). To capture the shared variance between modalities, we started by reversing the sign of the normalized gene body mCG fractions. In more detail, given that a decrease in gene body methylation is typically associated with increased transcriptional activity, we began by reversing the sign of the normalized gene body mCG fractions for all snm3C-seq nuclei to correct for the mismatch in directionality between the two modalities. As a result, in the adjusted mCG fraction matrix, cell-type marker genes show higher gene body methylation values in their respective cell types than other cell types, aligning with the patterns observed in the snRNA-seq data. These adjusted gene body methylation values serve as a proxy for the transcriptional activity of the snm3C-seq nuclei, which we then co-embedded with those from the snRNA-seq nuclei (Supplementary Note [2.5](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Cells profiled by both technologies were merged on their imputed expression profiles, projected to low-dimensional space with PCA and visualized by uniform manifold approximation and projection (constructed on the top ten principal components). We evaluated the concordance score and the confusion matrix between each de novo snm3C-seq and the snRNA-seq cell-type cluster pair in the shared canonical correlation analysis co-embedding space (Supplementary Note [2.6](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). We also performed adipogenesis trajectory analysis using SAT snRNA-seq data (Supplementary Note [2.7](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

### Cell-type-level SAT marker gene identification

We excluded the following genes from differential testing of the cell-type marker genes: genes that overlap with the ENCODE blacklisted regions; smaller genes (≤200 bp) mostly covered by other genes (overlap region of ≥90% of the gene length)[16](https://www.nature.com/articles/s41588-025-02300-4#ref-CR16 "Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nature 598, 120–128 (2021)."); and genes with a shallow coverage, constantly methylated or un-methylated, defined as those without ≥10 methylated or un-methylated counts in ≥10 cells belonging to the cell type under investigation. Cell-type-level differentially methylated genes were determined de novo by performing a Wilcoxon rank-sum test on the normalized gene body mCG fractions of the snm3C-seq cells in a one-vs-rest way. We retained genes that had a Benjamini–Hochberg adjusted _P_ value of <0.05 and at the same time exhibited a hypomethylation difference of ≥0.1 in terms of the average normalized fraction compared to the other cell types[16](https://www.nature.com/articles/s41588-025-02300-4#ref-CR16 "Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nature 598, 120–128 (2021)."). For transcriptomics, we first filtered for the set of expressed genes in SAT, defined as those with ≥3 counts in ≥3 cells[68](https://www.nature.com/articles/s41588-025-02300-4#ref-CR68 "Yang, S. et al. Decontamination of ambient RNA in single-cell RNA-seq with DecontX. Genome Biol. 21, 57 (2020)."). We used the ‘FindAllMarkers’ function in Seurat under default parameters except restricting to positive marker genes with ≥25% non-zero expression in either the tested cell type or the other ones[69](https://www.nature.com/articles/s41588-025-02300-4#ref-CR69 "Melms, J. C. et al. A molecular single-cell lung atlas of lethal COVID-19. Nature 595, 114–119 (2021)."),[70](https://www.nature.com/articles/s41588-025-02300-4#ref-CR70 "Mathys, H. et al. Single-cell atlas reveals correlates of high cognitive function, dementia, and resilience to Alzheimer’s disease pathology. Cell 186, 4365–4385.e27 (2023)."),[71](https://www.nature.com/articles/s41588-025-02300-4#ref-CR71 "Lake, B. B. et al. An atlas of healthy and injured cell states and niches in the human kidney. Nature 619, 585–594 (2023)."). Subsequently, we filtered out genes with Bonferroni-adjusted _P_ ≥ 0.05. To obtain unique marker genes from both snm3C-seq and snRNA-seq, we removed genes identified as marker genes for more than one cell type.

### Pathway enrichment analyses for SAT cell-type marker genes

To identify cell-type-level biological processes and functional pathways enriched among the cell-type marker genes in mCG and gene expression modalities, we used the web-based tool WebGestalt 2024 (ref. [72](https://www.nature.com/articles/s41588-025-02300-4#ref-CR72 "Elizarraras, J. M. et al. WebGestalt 2024: faster gene set analysis and new support for metabolomics and multi-omics. Nucleic Acids Res. 52, W415–W421 (2024).")), which identifies the overrepresentation of gene sets in Gene Ontology biological processes and KEGG pathways. For each SAT cell type, we used the unique cell-type marker genes as the input, with only the genes expressed within that cell type as the reference for the enrichment analysis. Biological processes and KEGG pathways with FDR < 0.05 were considered statistically significant.

### Cell-type-level methylation profile analysis

To obtain the cell-type-level mCG profiles, we aggregated the single-cell-level number of CG methylated counts and total coverage based on the snm3C-seq joint annotation and further merged reads mapped to adjacent CpGs in +/− strands. We then used MethylPy[20](https://www.nature.com/articles/s41588-025-02300-4#ref-CR20 "Schultz, M. D. et al. Human body epigenome maps reveal noncanonical DNA methylation variation. Nature 523, 212–216 (2015)."),[21](https://www.nature.com/articles/s41588-025-02300-4#ref-CR21 "He, Y. et al. Spatiotemporal DNA methylome dynamics of the developing mouse fetus. Nature 583, 752–759 (2020)."), implemented in the ALLCools package, to detect genomic regions that display distinct mCG patterns across various cell types. DMSs on autosomes were tested across all eight annotated cell types using default parameters. For all DMSs, we assigned one of the three states per cell type—hypomethylated, neutral or hypermethylated—based on whether the fitted residual (that is, the normalized deviation away from the mean methylation level) fell below the 0.4, between the 0.4 and 0.6 or above the 0.6 quantile of its chromosome-wide background, respectively[16](https://www.nature.com/articles/s41588-025-02300-4#ref-CR16 "Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nature 598, 120–128 (2021)."). Nearby DMSs (within 250 bp) with Pearson correlations of >0.8 for the methylation fractions across the cell types were merged into DMRs. Differential methylation states were assigned to each DMR based on the average of those of the DMSs it encompasses. DMRs containing only one DMS or without any hypermethylation or hypomethylation state assignment, along with DMRs or DMSs overlapping ENCODE blacklist regions, were excluded from downstream analyses. We further evaluated cell-type preferential DMR patterns, adjusting for variability in statistical power owing to differences in coverage across cell types (Supplementary Note [2.8](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

### Prediction of cell-type-level TF binding motifs

We performed TF binding motif enrichment analysis using the motif discovery tool HOMER (Hypergeometric Optimization of Motif EnRichment) (v.4.11.1)[22](https://www.nature.com/articles/s41588-025-02300-4#ref-CR22 "Heinz, S. et al. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Mol. Cell 38, 576–589 (2010)."). For each SAT main cell type, we used the hypomethylated regions as input data for motif enrichment analysis with the HOMER function ‘findMotifsGenome.pl’. TF binding motifs with _P_ < 1 × 10−12 were considered statistically significant. Circular visualization of cell-type-level TF binding motif enrichment results was prepared using the circlize package (v.0.4.16)[73](https://www.nature.com/articles/s41588-025-02300-4#ref-CR73 "Gu, Z., Gu, L., Eils, R., Schlesner, M. & Brors, B. circlize implements and enhances circular visualization in R. Bioinformatics 30, 2811–2812 (2014).") in R. We further validated cell-type-level TFs by confirming the overrepresentation of ChIP-seq peaks overlapping DMRs (Supplementary Note [2.9](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)), using external data from ENCODE[23](https://www.nature.com/articles/s41588-025-02300-4#ref-CR23 "ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature 489, 57–74 (2012).").

### Cell-type-level compartment analysis

Based on the snm3C-seq joint annotation, we merged scHicluster imputed single-cell-level contact matrices at 100 kb resolution per chromosome to form the cell-type-level pseudobulk conformation profiles for the five most abundant cell types (adipocytes, ASPCs, endothelial, perivascular and myeloid cells) as well as a cell-type aggregated version (Supplementary Note [2.10](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Cell-type-level pseudobulk conformation profiles were then normalized by the distance between the contacts and converted to correlation matrices by dcHiC (v.2.1)[46](https://www.nature.com/articles/s41588-025-02300-4#ref-CR46 "Chakraborty, A., Wang, J. G. & Ay, F. dcHiC detects differential compartments across multiple Hi-C datasets. Nat. Commun. 13, 6827 (2022)."). For all five cell types, we fitted PCA on the resulting matrices per chromosome and selected the appropriate leading principal component as the compartment scores (Supplementary Note [2.11](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Finally, we tested for genomic bins that demonstrated large deviations from the cell-type average under a multivariate normal distribution measured by the Mahalanobis distance using the covariance matrix learned with outlier bins removed. Bins with FDR-corrected _P_ < 0.1 were labeled as differentially conformed regions[46](https://www.nature.com/articles/s41588-025-02300-4#ref-CR46 "Chakraborty, A., Wang, J. G. & Ay, F. dcHiC detects differential compartments across multiple Hi-C datasets. Nat. Commun. 13, 6827 (2022)."). Empirically, we observed FDR < 0.02 when repeating the same analysis but only on a null set of cell-type-level contact maps, obtained by arbitrarily shuffling the annotation of the cells before merging to the pseudobulk level (that is, a scenario in which any differential compartment detected is false-positive by construction), indicating a conservative calibration of the testing result by dcHiC.

### Characterizing domains and chromatin loops in SAT

For interaction domains, we used scHiCluster (v.1.3.5)[63](https://www.nature.com/articles/s41588-025-02300-4#ref-CR63 "Zhou, J. et al. Robust single-cell Hi-C clustering by convolution- and random-walk-based imputation. Proc. Natl Acad. Sci. USA 116, 14011–14018 (2019).") with pad =2 to impute the contact matrix of each cell per autosomal chromosome at a 25 kb resolution, restricted to contacts within 10 Mb. We detected domains for each cell using TopDom (v.0.0.2)[74](https://www.nature.com/articles/s41588-025-02300-4#ref-CR74 "Shin, H. et al. TopDom: an efficient and deterministic method for identifying topological domains in genomes. Nucleic Acids Res. 44, e70 (2016)."), calculated the insulation scores across all 25 kb genomic bins with a window size of ten bins using the imputed contact profiles and visualized with uniform manifold approximation and projection (Supplementary Note [2.12](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Cell-type-level domain boundary probabilities were calculated as the fraction of cells with a detected domain boundary in a given 25 kb bin across all cells belonging to the specified cell type. Differential domain boundaries were evaluated per bin based on the following criteria[45](https://www.nature.com/articles/s41588-025-02300-4#ref-CR45 "Heffel, M. G. et al. Temporally distinct 3D multi-omic dynamics in the developing human brain. Nature 635, 481–489 (2024)."): a _z_\-score-transformed chi-squared statistic of >1.960 (97.5th percentile of standard normal distribution); differences between the maximum and minimum cell-type boundary probabilities of >0.05; detection as a local boundary peak (maximum); simultaneous detection as a local insulation score valley (minimum); and FDR < 0.001.

To analyze chromosomal looping, we used scHiCluster (v.1.3.5)[63](https://www.nature.com/articles/s41588-025-02300-4#ref-CR63 "Zhou, J. et al. Robust single-cell Hi-C clustering by convolution- and random-walk-based imputation. Proc. Natl Acad. Sci. USA 116, 14011–14018 (2019).") with pad = 2, window\_size = 30,000,000 and step\_size = 10,000,000 to impute a contact matrix of each cell per autosomal chromosome at a 10 kb resolution, restricted to contacts within 10 Mb. Loop pixels were detected from cell-type pseudobulk imputed contact profiles based on enrichment relative to both its global and local backgrounds. We aggregated nearby loop pixels passing an empirical FDR of 0.1 to loop summits (Supplementary Note [2.12](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

### Further analyses of methylation pathway genes

We assessed associations between _DNMT3A_ and _TET1_ SAT bulk expression and key CMD traits, including WHRadjBMI, BMI, serum triglyceride levels, Matsuda Index and fat mass (Supplementary Note [2.13](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)). Using six-time-point preadipocyte differentiation data (Supplementary Note [2.14](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)), we also analyzed longitudinal co-expression patterns of adipogenesis genes and their methylation regulators (Supplementary Notes [2.15](https://www.nature.com/articles/s41588-025-02300-4#MOESM1) and [2.16](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

### Partitioned cardiometabolic PRSs for DMRs and compartments

To assess the contributions of the SAT cell-type-level DMRs and cell-type-level compartments on the genetic risk for cardiometabolic traits, we constructed partitioned PRSs from variants landing in each set of cell-type-level DMRs and compartments for BMI, WHRadjBMI, CRP and MASLD in the UKB[30](https://www.nature.com/articles/s41588-025-02300-4#ref-CR30 "Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)."),[31](https://www.nature.com/articles/s41588-025-02300-4#ref-CR31 "Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)."), using a previously published imputed MASLD status[75](https://www.nature.com/articles/s41588-025-02300-4#ref-CR75 "Miao, Z. et al. Identification of 90 NAFLD GWAS loci and establishment of NAFLD PRS and causal role of NAFLD in coronary artery disease. HGG Adv. 3, 100056 (2022).").

We first generated GWAS summary statistics for each trait with a 50% base group (_n_ = 195,863) by applying a rank-based inverse-normal transform to each trait and used the linear-mixed model approach of BOLT-LMM (v.2.3.6)[76](https://www.nature.com/articles/s41588-025-02300-4#ref-CR76 "Loh, P.-R. et al. Efficient Bayesian mixed-model analysis increases association power in large cohorts. Nat. Genet. 47, 284–290 (2015)."), including age, age2, sex, the top 20 genetic principal components, testing center and genotyping array as covariates. Variants with a minor allele frequency of <1% and INFO < 0.8 were removed from the summary statistics. We then partitioned the remaining 50% into a 30% target and 20% validation groups for developing and applying the PRS model, respectively. Variants with a minor allele frequency of <1% and INFO < 0.8 were removed from the GWAS summary statistics, and the variants missing in >1% of subjects, with a minor allele frequency of <1% or violating Hardy–Weinberg equilibrium, as well as individuals with >1% missing genotypes or extreme heterozygosity, were removed from the target and validation genotype data[77](https://www.nature.com/articles/s41588-025-02300-4#ref-CR77 "Choi, S. W., Mak, T. S.-H. & O’Reilly, P. F. Tutorial: a guide to performing polygenic risk score analyses. Nat. Protoc. 15, 2759–2772 (2020).").

To compute the PRS for each outcome, we first generated independent marker sets by performing linkage disequilibrium clumping on all variants passing quality control in the genome using PLINK (v.1.9)[78](https://www.nature.com/articles/s41588-025-02300-4#ref-CR78 "Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007)."), with a linkage disequilibrium R2 threshold of 0.2 and a window size of 250 kb. We then used the 30% test set (_n_ = 115,120) to empirically identify the optimal _P_ value cut point at the genome-wide level. In brief, we applied the PLINK[78](https://www.nature.com/articles/s41588-025-02300-4#ref-CR78 "Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007).") ‘–score’ functionality to separately compute aggregated scores from subsets of the genome-wide clumped variants passing a _P_ value threshold from 5 × 10−8 to 0.5, using effect sizes and _P_ values from the GWAS summary statistics. After identifying the best thresholding cutoff in the 30% test set (0.05 for WHRadjBMI and CRP, 0.3 for BMI and 0.2 for MASLD), we computed regional PRSs in the 20% validation set (_n_ = 76,758), consisting of the clumped and thresholded variants landing within the DMRs or compartments. Variance explained (R2) by each PRS was calculated by adjusting each trait for age, age2, the top 20 genetic principal components, testing center, genotyping array and sex, applying a rank-based inverse-normal transform and then regressing the PRS on the adjusted trait. We assessed the statistical significance of the variance explained by each PRS by comparing it to those of 10,000 permuted PRSs constructed from randomly selected variant sets with matching size and genomic background (that is, genome-wide clumped and thresholded). We also assessed overrepresentation of GWAS variants (Supplementary Note [2.17](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)), their _cis_\-expression quantitative trait locus effects and relevance to CMDs (Supplementary Note [2.18](https://www.nature.com/articles/s41588-025-02300-4#MOESM1)).

### Statistics and reproducibility

No statistical method was used to predetermine the sample size of the snm3C-seq cohort; however, our sample sizes and number of cells profiled are similar to those reported in previous snm3C-seq publications[12](https://www.nature.com/articles/s41588-025-02300-4#ref-CR12 "Lee, D.-S. et al. Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. Nat. Methods 16, 999–1006 (2019)."). No samples were excluded from the Tilkka cohort. In the UKB and METSIM cohorts, related individuals were excluded. As this is an observational study, no randomization was performed. Data collection and analysis were performed blind to the conditions of the experiments. Results obtained from SAT snm3C-seq and snRNA-seq were not replicated owing to the unique nature of these datasets. Results reported from the UKB GWAS analyses were not replicated because of the unprecedentedly large sample size of the cohort. Correlations observed with the METSIM bulk RNA-seq data were not replicated because of the unique refined phenotype profiles of the METSIM cohort. Statistical significance was assessed using empirical permutation or non-parametric tests when possible. Assumed data distributions for other analyses (binomial, hypergeometric, multivariate normal) were not formally tested. Owing to the large sample size (for example, genomic bins, ChIP-seq peaks), individual data points were not visualized.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02300-4#MOESM2) linked to this article.

Data availability
-----------------

The data that support the findings in this manuscript are available from the UKB. However, restrictions apply to the availability of these data, which were used in this study under UKB application number 33934. UKB data are available for bona fide researchers through the application process at [https://www.ukbiobank.ac.uk/learn-more-about-uk-biobank/contact-us](https://www.ukbiobank.ac.uk/learn-more-about-uk-biobank/contact-us). The snm3C-seq and snRNA-seq data from the Tilkka cohort are available in the Gene Expression Omnibus (GEO) under accession number [GSE297267](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE297267), along with the epigenomic annotations of SAT DMRs, compartments, domains and loops characterized in this study. The GRCh38 reference genome is available through the UCSC genome browser ([https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips](https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips))[79](https://www.nature.com/articles/s41588-025-02300-4#ref-CR79 "Navarro Gonzalez, J. et al. The UCSC Genome Browser database: 2021 update. Nucleic Acids Res. 49, D1046–D1057 (2021)."). The bulk RNA-seq data from the primary human preadipocyte differentiation experiment were previously made available in GEO under accession number [GSE249195](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249195). The METSIM SAT bulk RNA-seq data[55](https://www.nature.com/articles/s41588-025-02300-4#ref-CR55 "Pan, D. Z. et al. Identification of TBX15 as an adipose master trans regulator of abdominal obesity genes. Genome Med. 13, 123 (2021).") are available in GEO under accession number [GSE135134](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135134). Bulk SAT _cis_\-expression quantitative trait locus variants are available from the GTEx v10 cohort[80](https://www.nature.com/articles/s41588-025-02300-4#ref-CR80 "GTEx Consortium. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science 369, 1318–1330 (2020)."). ChIP-seq datasets are publicly available on the ENCODE portal with the following accession numbers: ENCSR000BGY ([https://www.encodeproject.org/experiments/ENCSR000BGY](https://www.encodeproject.org/experiments/ENCSR000BGY)), ENCSR177VFS ([https://www.encodeproject.org/experiments/ENCSR177VFS](https://www.encodeproject.org/experiments/ENCSR177VFS)) and ENCSR490LWA ([https://www.encodeproject.org/experiments/ENCSR490LWA](https://www.encodeproject.org/experiments/ENCSR490LWA)). Source data files and GWAS summary statistics from this study are available on Zenodo ([https://zenodo.org/records/15318595](https://zenodo.org/records/15318595))[81](https://www.nature.com/articles/s41588-025-02300-4#ref-CR81 "Chen, Z. J. et al. Data files for the manuscript entitled, ‘Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue’. Zenodo 
https://doi.org/10.5281/zenodo.15318595
(2025).").

Code availability
-----------------

All packages and software used in this study were from their publicly available sources, as outlined in the [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14). No custom code was generated.

References
----------

1.  González-Muniesa, P. et al. Obesity. _Nat. Rev. Dis. Prim._ **3**, 17034 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28617414)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity&journal=Nat.%20Rev.%20Dis.%20Prim.&volume=3&publication_year=2017&author=Gonz%C3%A1lez-Muniesa%2CP) 
    
2.  Sun, J.-Y. et al. Trends in general and abdominal obesity in US adults: evidence from the National Health and Nutrition Examination Survey (2001–2018). _Front. Public Health_ **10**, 925293 (2022).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36276394)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9582849)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Trends%20in%20general%20and%20abdominal%20obesity%20in%20US%20adults%3A%20evidence%20from%20the%20National%20Health%20and%20Nutrition%20Examination%20Survey%20%282001%E2%80%932018%29&journal=Front.%20Public%20Health&volume=10&publication_year=2022&author=Sun%2CJ-Y) 
    
3.  Després, J.-P. Abdominal obesity: the most prevalent cause of the metabolic syndrome and related cardiometabolic risk. _Eur. Heart J. Suppl._ **8**, B4–B12 (2006).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Abdominal%20obesity%3A%20the%20most%20prevalent%20cause%20of%20the%20metabolic%20syndrome%20and%20related%20cardiometabolic%20risk&journal=Eur.%20Heart%20J.%20Suppl.&volume=8&pages=B4-B12&publication_year=2006&author=Despr%C3%A9s%2CJ-P) 
    
4.  Fuster, J. J., Ouchi, N., Gokce, N. & Walsh, K. Obesity-induced changes in adipose tissue microenvironment and their impact on cardiovascular disease. _Circ. Res._ **118**, 1786–1807 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XosFyrs7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27230642)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4887147)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity-induced%20changes%20in%20adipose%20tissue%20microenvironment%20and%20their%20impact%20on%20cardiovascular%20disease&journal=Circ.%20Res.&volume=118&pages=1786-1807&publication_year=2016&author=Fuster%2CJJ&author=Ouchi%2CN&author=Gokce%2CN&author=Walsh%2CK) 
    
5.  Glastonbury, C. A., Couto Alves, A., El-Sayed Moustafa, J. S. & Small, K. S. Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs. _Am. J. Hum. Genet._ **104**, 1013–1024 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVegt7vO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31130283)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6556877)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell-type%20heterogeneity%20in%20adipose%20tissue%20is%20associated%20with%20complex%20traits%20and%20reveals%20disease-relevant%20cell-specific%20eQTLs&journal=Am.%20J.%20Hum.%20Genet.&volume=104&pages=1013-1024&publication_year=2019&author=Glastonbury%2CCA&author=Couto%20Alves%2CA&author=El-Sayed%20Moustafa%2CJS&author=Small%2CKS) 
    
6.  Das, S. S. et al. Cross-tissue single-nucleus RNA sequencing discovers tissue-resident adipocytes involved in propanoate metabolism in the human heart. _Arterioscler. Thromb. Vasc. Biol._ **43**, 1788–1804 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvFyrt7%2FL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37409528)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10538422)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-tissue%20single-nucleus%20RNA%20sequencing%20discovers%20tissue-resident%20adipocytes%20involved%20in%20propanoate%20metabolism%20in%20the%20human%20heart&journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&volume=43&pages=1788-1804&publication_year=2023&author=Das%2CSS) 
    
7.  Kar, A. et al. Age-dependent genes in adipose stem and precursor cells affect regulation of fat cell differentiation and link aging to obesity via cellular and genetic interactions. _Genome Med._ **16**, 19 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXisFaisbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38297378)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10829214)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Age-dependent%20genes%20in%20adipose%20stem%20and%20precursor%20cells%20affect%20regulation%20of%20fat%20cell%20differentiation%20and%20link%20aging%20to%20obesity%20via%20cellular%20and%20genetic%20interactions&journal=Genome%20Med.&volume=16&publication_year=2024&author=Kar%2CA) 
    
8.  Samblas, M., Milagro, F. I. & Martínez, A. DNA methylation markers in obesity, metabolic syndrome, and weight loss. _Epigenetics_ **14**, 421–444 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30915894)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6557553)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20methylation%20markers%20in%20obesity%2C%20metabolic%20syndrome%2C%20and%20weight%20loss&journal=Epigenetics&volume=14&pages=421-444&publication_year=2019&author=Samblas%2CM&author=Milagro%2CFI&author=Mart%C3%ADnez%2CA) 
    
9.  Margueron, R. & Reinberg, D. Chromatin structure and the inheritance of epigenetic information. _Nat. Rev. Genet._ **11**, 285–296 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXjsFSrtb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20300089)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3760772)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromatin%20structure%20and%20the%20inheritance%20of%20epigenetic%20information&journal=Nat.%20Rev.%20Genet.&volume=11&pages=285-296&publication_year=2010&author=Margueron%2CR&author=Reinberg%2CD) 
    
10.  Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. _Science_ **382**, eadf5357 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFygur7O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37824674)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10572106)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20DNA%20methylation%20and%203D%20genome%20architecture%20in%20the%20human%20brain&journal=Science&volume=382&publication_year=2023&author=Tian%2CW) 
    
11.  Wang, S. et al. Spatial organization of chromatin domains and compartments in single chromosomes. _Science_ **353**, 598–602 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Oks7vF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27445307)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4991974)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatial%20organization%20of%20chromatin%20domains%20and%20compartments%20in%20single%20chromosomes&journal=Science&volume=353&pages=598-602&publication_year=2016&author=Wang%2CS) 
    
12.  Lee, D.-S. et al. Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. _Nat. Methods_ **16**, 999–1006 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsleltLjE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31501549)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6765423)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Simultaneous%20profiling%20of%203D%20genome%20structure%20and%20DNA%20methylation%20in%20single%20human%20cells&journal=Nat.%20Methods&volume=16&pages=999-1006&publication_year=2019&author=Lee%2CD-S) 
    
13.  Yan, R. et al. Decoding dynamic epigenetic landscapes in human oocytes using single-cell multi-omics sequencing. _Cell Stem Cell_ **28**, 1641–1656.e7 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVCrt7nI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33957080)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Decoding%20dynamic%20epigenetic%20landscapes%20in%20human%20oocytes%20using%20single-cell%20multi-omics%20sequencing&journal=Cell%20Stem%20Cell&volume=28&pages=1641-1656.e7&publication_year=2021&author=Yan%2CR) 
    
14.  Luo, C. et al. Single-cell methylomes identify neuronal subtypes and regulatory elements in mammalian cortex. _Science_ **357**, 600–604 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlehur%2FE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28798132)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5570439)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20methylomes%20identify%20neuronal%20subtypes%20and%20regulatory%20elements%20in%20mammalian%20cortex&journal=Science&volume=357&pages=600-604&publication_year=2017&author=Luo%2CC) 
    
15.  Luo, C. et al. Single nucleus multi-omics identifies human cortical cell regulatory genome diversity. _Cell Genomics_ **2**, 100107 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvFCktrfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35419551)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9004682)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single%20nucleus%20multi-omics%20identifies%20human%20cortical%20cell%20regulatory%20genome%20diversity&journal=Cell%20Genomics&volume=2&publication_year=2022&author=Luo%2CC) 
    
16.  Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. _Nature_ **598**, 120–128 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1WqsbjJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34616061)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8494641)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20methylation%20atlas%20of%20the%20mouse%20brain%20at%20single-cell%20resolution&journal=Nature&volume=598&pages=120-128&publication_year=2021&author=Liu%2CH) 
    
17.  Liu, H. et al. Single-cell DNA methylome and 3D multi-omic atlas of the adult mouse brain. _Nature_ **624**, 366–377 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXis1SksLbF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38092913)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10719113)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20DNA%20methylome%20and%203D%20multi-omic%20atlas%20of%20the%20adult%20mouse%20brain&journal=Nature&volume=624&pages=366-377&publication_year=2023&author=Liu%2CH) 
    
18.  Flint, J. et al. Single-cell methylation analysis of brain tissue prioritizes mutations that alter transcription. _Cell Genomics_ **3**, 100454 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFWktbrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38116123)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10726494)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20methylation%20analysis%20of%20brain%20tissue%20prioritizes%20mutations%20that%20alter%20transcription&journal=Cell%20Genomics&volume=3&publication_year=2023&author=Flint%2CJ) 
    
19.  Stuart, T. et al. Comprehensive integration of single-cell data. _Cell_ **177**, 1888–1902.e21 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFens77L)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31178118)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6687398)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comprehensive%20integration%20of%20single-cell%20data&journal=Cell&volume=177&pages=1888-1902.e21&publication_year=2019&author=Stuart%2CT) 
    
20.  Schultz, M. D. et al. Human body epigenome maps reveal noncanonical DNA methylation variation. _Nature_ **523**, 212–216 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFaitrvL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26030523)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4499021)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Human%20body%20epigenome%20maps%20reveal%20noncanonical%20DNA%20methylation%20variation&journal=Nature&volume=523&pages=212-216&publication_year=2015&author=Schultz%2CMD) 
    
21.  He, Y. et al. Spatiotemporal DNA methylome dynamics of the developing mouse fetus. _Nature_ **583**, 752–759 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVygtLnM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32728242)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7398276)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatiotemporal%20DNA%20methylome%20dynamics%20of%20the%20developing%20mouse%20fetus&journal=Nature&volume=583&pages=752-759&publication_year=2020&author=He%2CY) 
    
22.  Heinz, S. et al. Simple combinations of lineage-determining transcription factors prime _cis_\-regulatory elements required for macrophage and B cell identities. _Mol. Cell_ **38**, 576–589 (2010).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXns1SlsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20513432)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898526)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Simple%20combinations%20of%20lineage-determining%20transcription%20factors%20prime%20cis-regulatory%20elements%20required%20for%20macrophage%20and%20B%E2%80%89cell%20identities&journal=Mol.%20Cell&volume=38&pages=576-589&publication_year=2010&author=Heinz%2CS) 
    
23.  ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. _Nature_ **489**, 57–74 (2012).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&journal=Nature&volume=489&pages=57-74&publication_year=2012) 
    
24.  Dekker, J. & Misteli, T. Long-range chromatin interactions. _Cold Spring Harb. Perspect. Biol._ **7**, a019356 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26430217)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4588061)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Long-range%20chromatin%20interactions&journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&volume=7&publication_year=2015&author=Dekker%2CJ&author=Misteli%2CT) 
    
25.  Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. _Cell_ **159**, 1665–1680 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtrrK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25497547)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635824)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%203D%20map%20of%20the%20human%20genome%20at%20kilobase%20resolution%20reveals%20principles%20of%20chromatin%20looping&journal=Cell&volume=159&pages=1665-1680&publication_year=2014&author=Rao%2CSSP) 
    
26.  Bailey, A. S. et al. Myeloid lineage progenitors give rise to vascular endothelium. _Proc. Natl Acad. Sci. USA_ **103**, 13156–13161 (2006).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xptlelsrg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16920790)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1559769)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Myeloid%20lineage%20progenitors%20give%20rise%20to%20vascular%20endothelium&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=103&pages=13156-13161&publication_year=2006&author=Bailey%2CAS) 
    
27.  Moore, L. D., Le, T. & Fan, G. DNA methylation and its basic function. _Neuropsychopharmacology_ **38**, 23–38 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1ynurbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22781841)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNA%20methylation%20and%20its%20basic%20function&journal=Neuropsychopharmacology&volume=38&pages=23-38&publication_year=2013&author=Moore%2CLD&author=Le%2CT&author=Fan%2CG) 
    
28.  Emdin, C. A. et al. Genetic association of waist-to-hip ratio with cardiometabolic traits, type 2 diabetes, and coronary heart disease. _JAMA_ **317**, 626–634 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28196256)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5571980)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20association%20of%20waist-to-hip%20ratio%20with%20cardiometabolic%20traits%2C%20type%202%20diabetes%2C%20and%20coronary%20heart%20disease&journal=JAMA&volume=317&pages=626-634&publication_year=2017&author=Emdin%2CCA) 
    
29.  Yusuf, S. et al. Obesity and the risk of myocardial infarction in 27,000 participants from 52 countries: a case–control study. _Lancet_ **366**, 1640–1649 (2005).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16271645)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity%20and%20the%20risk%20of%20myocardial%20infarction%20in%2027%2C000%20participants%20from%2052%20countries%3A%20a%20case%E2%80%93control%20study&journal=Lancet&volume=366&pages=1640-1649&publication_year=2005&author=Yusuf%2CS) 
    
30.  Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. _PLoS Med._ **12**, e1001779 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25826379)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380465)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=UK%20Biobank%3A%20an%20open%20access%20resource%20for%20identifying%20the%20causes%20of%20a%20wide%20range%20of%20complex%20diseases%20of%20middle%20and%20old%20age&journal=PLoS%20Med.&volume=12&publication_year=2015&author=Sudlow%2CC) 
    
31.  Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. _Nature_ **562**, 203–209 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2qsbvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30305743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&journal=Nature&volume=562&pages=203-209&publication_year=2018&author=Bycroft%2CC) 
    
32.  Dornbos, P. et al. Evaluating human genetic support for hypothesized metabolic disease genes. _Cell Metab._ **34**, 661–666 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVant7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35421386)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9166611)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluating%20human%20genetic%20support%20for%20hypothesized%20metabolic%20disease%20genes&journal=Cell%20Metab.&volume=34&pages=661-666&publication_year=2022&author=Dornbos%2CP) 
    
33.  Benton, M. C. et al. An analysis of DNA methylation in human adipose tissue reveals differential modification of obesity genes before and after gastric bypass and weight loss. _Genome Biol._ **16**, 8 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjtlKns78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25651499)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4301800)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20analysis%20of%20DNA%20methylation%20in%20human%20adipose%20tissue%20reveals%20differential%20modification%20of%20obesity%20genes%20before%20and%20after%20gastric%20bypass%20and%20weight%20loss&journal=Genome%20Biol.&volume=16&publication_year=2015&author=Benton%2CMC) 
    
34.  Mirzaeicheshmeh, E. et al. Alterations of DNA methylation during adipogenesis differentiation of mesenchymal stem cells isolated from adipose tissue of patients with obesity is associated with type 2 diabetes. _Adipocyte_ **10**, 493–504 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlOmtrjI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34699309)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8555535)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Alterations%20of%20DNA%20methylation%20during%20adipogenesis%20differentiation%20of%20mesenchymal%20stem%20cells%20isolated%20from%20adipose%20tissue%20of%20patients%20with%20obesity%20is%20associated%20with%20type%202%20diabetes&journal=Adipocyte&volume=10&pages=493-504&publication_year=2021&author=Mirzaeicheshmeh%2CE) 
    
35.  Jaffe, A. E. & Irizarry, R. A. Accounting for cellular heterogeneity is critical in epigenome-wide association studies. _Genome Biol._ **15**, R31 (2014).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24495553)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053810)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accounting%20for%20cellular%20heterogeneity%20is%20critical%20in%20epigenome-wide%20association%20studies&journal=Genome%20Biol.&volume=15&publication_year=2014&author=Jaffe%2CAE&author=Irizarry%2CRA) 
    
36.  Zheng, S. C., Breeze, C. E., Beck, S. & Teschendorff, A. E. Identification of differentially methylated cell types in epigenome-wide association studies. _Nat. Methods_ **15**, 1059–1066 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2hsrfK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30504870)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6277016)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20differentially%20methylated%20cell%20types%20in%20epigenome-wide%20association%20studies&journal=Nat.%20Methods&volume=15&pages=1059-1066&publication_year=2018&author=Zheng%2CSC&author=Breeze%2CCE&author=Beck%2CS&author=Teschendorff%2CAE) 
    
37.  Rahmani, E. et al. Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies. _Nat. Methods_ **13**, 443–445 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XkvV2jtLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27018579)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5548182)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sparse%20PCA%20corrects%20for%20cell%20type%20heterogeneity%20in%20epigenome-wide%20association%20studies&journal=Nat.%20Methods&volume=13&pages=443-445&publication_year=2016&author=Rahmani%2CE) 
    
38.  Thompson, M., Chen, Z. J., Rahmani, E. & Halperin, E. CONFINED: distinguishing biological from technical sources of variation by leveraging multiple methylation datasets. _Genome Biol._ **20**, 138 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31300005)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6624895)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=CONFINED%3A%20distinguishing%20biological%20from%20technical%20sources%20of%20variation%20by%20leveraging%20multiple%20methylation%20datasets&journal=Genome%20Biol.&volume=20&publication_year=2019&author=Thompson%2CM&author=Chen%2CZJ&author=Rahmani%2CE&author=Halperin%2CE) 
    
39.  Rahmani, E. et al. Cell-type-specific resolution epigenetics without the need for cell sorting or single-cell biology. _Nat. Commun._ **10**, 3417 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31366909)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6668473)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cell-type-specific%20resolution%20epigenetics%20without%20the%20need%20for%20cell%20sorting%20or%20single-cell%20biology&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Rahmani%2CE) 
    
40.  Chen, Z. J., Rahmani, E. & Halperin, E. A unified model for cell-type resolution genomics from heterogeneous omics data. Preprint at _bioRxiv_ [https://doi.org/10.1101/2024.01.27.577588](https://doi.org/10.1101/2024.01.27.577588) (2024).
    
41.  Gorla, A. et al. Phenotypic subtyping via contrastive learning. Preprint at _bioRxiv_ [https://doi.org/10.1101/2023.01.05.522921](https://doi.org/10.1101/2023.01.05.522921) (2023).
    
42.  Gorla, A. et al. Epigenetic patient stratification via contrastive machine learning refines hallmark biomarkers in minoritized children with asthma. Preprint at _Res. Sq_. [https://doi.org/10.21203/rs.3.rs-5066762/v1](https://doi.org/10.21203/rs.3.rs-5066762/v1) (2024).
    
43.  Weisberg, S. P. et al. Obesity is associated with macrophage accumulation in adipose tissue. _J. Clin. Invest._ **112**, 1796–1808 (2003).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXhtVSmtLvO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=14679176)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC296995)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Obesity%20is%20associated%20with%20macrophage%20accumulation%20in%20adipose%20tissue&journal=J.%20Clin.%20Invest.&volume=112&pages=1796-1808&publication_year=2003&author=Weisberg%2CSP) 
    
44.  Tan, L. et al. Lifelong restructuring of 3D genome architecture in cerebellar granule cells. _Science_ **381**, 1112–1119 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVOrtrfJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37676945)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11059189)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lifelong%20restructuring%20of%203D%20genome%20architecture%20in%20cerebellar%20granule%20cells&journal=Science&volume=381&pages=1112-1119&publication_year=2023&author=Tan%2CL) 
    
45.  Heffel, M. G. et al. Temporally distinct 3D multi-omic dynamics in the developing human brain. _Nature_ **635**, 481–489 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitFymu7zM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39385032)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11560841)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Temporally%20distinct%203D%20multi-omic%20dynamics%20in%20the%20developing%20human%20brain&journal=Nature&volume=635&pages=481-489&publication_year=2024&author=Heffel%2CMG) 
    
46.  Chakraborty, A., Wang, J. G. & Ay, F. dcHiC detects differential compartments across multiple Hi-C datasets. _Nat. Commun._ **13**, 6827 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivFSisrzK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36369226)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9652325)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=dcHiC%20detects%20differential%20compartments%20across%20multiple%20Hi-C%20datasets&journal=Nat.%20Commun.&volume=13&publication_year=2022&author=Chakraborty%2CA&author=Wang%2CJG&author=Ay%2CF) 
    
47.  Qian, H. et al. TET1 promotes RXRα expression and adipogenesis through DNA demethylation. _Biochim. Biophys. Acta Mol. Cell. Biol. Lipids_ **1866**, 158919 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXms1Kjur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33684567)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TET1%20promotes%20RXR%CE%B1%20expression%20and%20adipogenesis%20through%20DNA%20demethylation&journal=Biochim.%20Biophys.%20Acta%20Mol.%20Cell.%20Biol.%20Lipids&volume=1866&publication_year=2021&author=Qian%2CH) 
    
48.  Ali, M. M., Phillips, S. A. & Mahmoud, A. M. HIF1α/TET1 pathway mediates hypoxia-induced adipocytokine promoter hypomethylation in human adipocytes. _Cells_ **9**, 134 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVygu7bL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31935962)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7016890)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=HIF1%CE%B1%2FTET1%20pathway%20mediates%20hypoxia-induced%20adipocytokine%20promoter%20hypomethylation%20in%20human%20adipocytes&journal=Cells&volume=9&publication_year=2020&author=Ali%2CMM&author=Phillips%2CSA&author=Mahmoud%2CAM) 
    
49.  Gu, T. et al. DNMT3A and TET1 cooperate to regulate promoter epigenetic landscapes in mouse embryonic stem cells. _Genome Biol._ **19**, 88 (2018).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30001199)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6042404)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=DNMT3A%20and%20TET1%20cooperate%20to%20regulate%20promoter%20epigenetic%20landscapes%20in%20mouse%20embryonic%20stem%20cells&journal=Genome%20Biol.&volume=19&publication_year=2018&author=Gu%2CT) 
    
50.  Damal et al. TET1 is a beige adipocyte-selective epigenetic suppressor of thermogenesis. _Nat. Commun._ **11**, 4313 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TET1%20is%20a%20beige%20adipocyte-selective%20epigenetic%20suppressor%20of%20thermogenesis&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Damal%2C) 
    
51.  Byun, S. et al. Loss of adipose TET proteins enhances β-adrenergic responses and protects against obesity by epigenetic regulation of β3-AR expression. _Proc. Natl Acad. Sci. USA_ **119**, e2205626119 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVGqsbjE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35737830)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9245707)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Loss%20of%20adipose%20TET%20proteins%20enhances%20%CE%B2-adrenergic%20responses%20and%20protects%20against%20obesity%20by%20epigenetic%20regulation%20of%20%CE%B23-AR%20expression&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=119&publication_year=2022&author=Byun%2CS) 
    
52.  Tovy, A. et al. Constitutive loss of DNMT3A causes morbid obesity through misregulation of adipogenesis. _Elife_ **11**, e72359 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlCmsbvI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35635747)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9150890)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Constitutive%20loss%20of%20DNMT3A%20causes%20morbid%20obesity%20through%20misregulation%20of%20adipogenesis&journal=Elife&volume=11&publication_year=2022&author=Tovy%2CA) 
    
53.  You, D. et al. Dnmt3a is an epigenetic mediator of adipose insulin resistance. _Elife_ **6**, e30766 (2017).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29091029)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5730374)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dnmt3a%20is%20an%20epigenetic%20mediator%20of%20adipose%20insulin%20resistance&journal=Elife&volume=6&publication_year=2017&author=You%2CD) 
    
54.  Christiansen, M. R., Kilpeläinen, T. O. & McCaffery, J. M. Abdominal obesity genetic variants predict waist circumference regain after weight loss. _Diabetes_ **72**, 1424–1432 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitFersLbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37494631)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Abdominal%20obesity%20genetic%20variants%20predict%20waist%20circumference%20regain%20after%20weight%20loss&journal=Diabetes&volume=72&pages=1424-1432&publication_year=2023&author=Christiansen%2CMR&author=Kilpel%C3%A4inen%2CTO&author=McCaffery%2CJM) 
    
55.  Pan, D. Z. et al. Identification of TBX15 as an adipose master trans regulator of abdominal obesity genes. _Genome Med._ **13**, 123 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1yitrfN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34340684)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8327600)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%20TBX15%20as%20an%20adipose%20master%20trans%20regulator%20of%20abdominal%20obesity%20genes&journal=Genome%20Med.&volume=13&publication_year=2021&author=Pan%2CDZ) 
    
56.  Khera, A. V. et al. Polygenic prediction of weight and obesity trajectories from birth to adulthood. _Cell_ **177**, 587–596.e9 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXnslyhtLg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31002795)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6661115)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Polygenic%20prediction%20of%20weight%20and%20obesity%20trajectories%20from%20birth%20to%20adulthood&journal=Cell&volume=177&pages=587-596.e9&publication_year=2019&author=Khera%2CAV) 
    
57.  Laakso, M. et al. The Metabolic Syndrome in Men study: a resource for studies of metabolic and cardiovascular diseases. _J. Lipid Res._ **58**, 481–493 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsFGqs7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28119442)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5335588)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20Metabolic%20Syndrome%20in%20Men%20study%3A%20a%20resource%20for%20studies%20of%20metabolic%20and%20cardiovascular%20diseases&journal=J.%20Lipid%20Res.&volume=58&pages=481-493&publication_year=2017&author=Laakso%2CM) 
    
58.  Amemiya, H. M., Kundaje, A. & Boyle, A. P. The ENCODE Blacklist: identification of problematic regions of the genome. _Sci. Rep._ **9**, 9354 (2019).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31249361)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6597582)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20ENCODE%20Blacklist%3A%20identification%20of%20problematic%20regions%20of%20the%20genome&journal=Sci.%20Rep.&volume=9&publication_year=2019&author=Amemiya%2CHM&author=Kundaje%2CA&author=Boyle%2CAP) 
    
59.  Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. _Gigascience_ **4**, 7 (2015).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25722852)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&journal=Gigascience&volume=4&publication_year=2015&author=Chang%2CCC) 
    
60.  McCarthy, S. et al. A reference panel of 64,976 haplotypes for genotype imputation. _Nat. Genet._ **48**, 1279–1283 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhtlykt7zO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27548312)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388176)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20reference%20panel%20of%2064%2C976%20haplotypes%20for%20genotype%20imputation&journal=Nat.%20Genet.&volume=48&pages=1279-1283&publication_year=2016&author=McCarthy%2CS) 
    
61.  Loh, P.-R. et al. Reference-based phasing using the Haplotype Reference Consortium panel. _Nat. Genet._ **48**, 1443–1448 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1SltLzF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27694958)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5096458)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reference-based%20phasing%20using%20the%20Haplotype%20Reference%20Consortium%20panel&journal=Nat.%20Genet.&volume=48&pages=1443-1448&publication_year=2016&author=Loh%2CP-R) 
    
62.  Das, S. et al. Next-generation genotype imputation service and methods. _Nat. Genet._ **48**, 1284–1287 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVWksL%2FK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27571263)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5157836)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Next-generation%20genotype%20imputation%20service%20and%20methods&journal=Nat.%20Genet.&volume=48&pages=1284-1287&publication_year=2016&author=Das%2CS) 
    
63.  Zhou, J. et al. Robust single-cell Hi-C clustering by convolution- and random-walk-based imputation. _Proc. Natl Acad. Sci. USA_ **116**, 14011–14018 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlCgsr3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31235599)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6628819)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Robust%20single-cell%20Hi-C%20clustering%20by%20convolution-%20and%20random-walk-based%20imputation&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&volume=116&pages=14011-14018&publication_year=2019&author=Zhou%2CJ) 
    
64.  Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. _Nature_ **603**, 926–933 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XntlGgtbw%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35296864)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9504827)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20single-cell%20atlas%20of%20human%20and%20mouse%20white%20adipose%20tissue&journal=Nature&volume=603&pages=926-933&publication_year=2022&author=Emont%2CMP) 
    
65.  Hao, Y. et al. Integrated analysis of multimodal single-cell data. _Cell_ **184**, 3573–3587.e29 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtrvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34062119)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238499)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&journal=Cell&volume=184&pages=3573-3587.e29&publication_year=2021&author=Hao%2CY) 
    
66.  Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. _Nat. Immunol._ **20**, 163–172 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFSgu7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30643263)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6340744)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reference-based%20analysis%20of%20lung%20single-cell%20sequencing%20reveals%20a%20transitional%20profibrotic%20macrophage&journal=Nat.%20Immunol.&volume=20&pages=163-172&publication_year=2019&author=Aran%2CD) 
    
67.  Chen, P. B. et al. Complementation testing identifies genes mediating effects at quantitative trait loci underlying fear-related behavior. _Cell Genomics_ **4**, 100545 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXpsFKns7k%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38697120)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11099346)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Complementation%20testing%20identifies%20genes%20mediating%20effects%20at%20quantitative%20trait%20loci%20underlying%20fear-related%20behavior&journal=Cell%20Genomics&volume=4&publication_year=2024&author=Chen%2CPB) 
    
68.  Yang, S. et al. Decontamination of ambient RNA in single-cell RNA-seq with DecontX. _Genome Biol._ **21**, 57 (2020).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32138770)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7059395)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Decontamination%20of%20ambient%20RNA%20in%20single-cell%20RNA-seq%20with%20DecontX&journal=Genome%20Biol.&volume=21&publication_year=2020&author=Yang%2CS) 
    
69.  Melms, J. C. et al. A molecular single-cell lung atlas of lethal COVID-19. _Nature_ **595**, 114–119 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFOltbbE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33915568)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8814825)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20molecular%20single-cell%20lung%20atlas%20of%20lethal%20COVID-19&journal=Nature&volume=595&pages=114-119&publication_year=2021&author=Melms%2CJC) 
    
70.  Mathys, H. et al. Single-cell atlas reveals correlates of high cognitive function, dementia, and resilience to Alzheimer’s disease pathology. _Cell_ **186**, 4365–4385.e27 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVaqtrbM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37774677)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10601493)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20atlas%20reveals%20correlates%20of%20high%20cognitive%20function%2C%20dementia%2C%20and%20resilience%20to%20Alzheimer%E2%80%99s%20disease%20pathology&journal=Cell&volume=186&pages=4365-4385.e27&publication_year=2023&author=Mathys%2CH) 
    
71.  Lake, B. B. et al. An atlas of healthy and injured cell states and niches in the human kidney. _Nature_ **619**, 585–594 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsFSmtLnI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37468583)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10356613)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20atlas%20of%20healthy%20and%20injured%20cell%20states%20and%20niches%20in%20the%20human%20kidney&journal=Nature&volume=619&pages=585-594&publication_year=2023&author=Lake%2CBB) 
    
72.  Elizarraras, J. M. et al. WebGestalt 2024: faster gene set analysis and new support for metabolomics and multi-omics. _Nucleic Acids Res._ **52**, W415–W421 (2024).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38808672)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11223849)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=WebGestalt%202024%3A%20faster%20gene%20set%20analysis%20and%20new%20support%20for%20metabolomics%20and%20multi-omics&journal=Nucleic%20Acids%20Res.&volume=52&pages=W415-W421&publication_year=2024&author=Elizarraras%2CJM) 
    
73.  Gu, Z., Gu, L., Eils, R., Schlesner, M. & Brors, B. _circlize_ implements and enhances circular visualization in R. _Bioinformatics_ **30**, 2811–2812 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFOrtL7I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24930139)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=circlize%20implements%20and%20enhances%20circular%20visualization%20in%20R&journal=Bioinformatics&volume=30&pages=2811-2812&publication_year=2014&author=Gu%2CZ&author=Gu%2CL&author=Eils%2CR&author=Schlesner%2CM&author=Brors%2CB) 
    
74.  Shin, H. et al. TopDom: an efficient and deterministic method for identifying topological domains in genomes. _Nucleic Acids Res._ **44**, e70 (2016).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26704975)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TopDom%3A%20an%20efficient%20and%20deterministic%20method%20for%20identifying%20topological%20domains%20in%20genomes&journal=Nucleic%20Acids%20Res.&volume=44&publication_year=2016&author=Shin%2CH) 
    
75.  Miao, Z. et al. Identification of 90 NAFLD GWAS loci and establishment of NAFLD PRS and causal role of NAFLD in coronary artery disease. _HGG Adv._ **3**, 100056 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFyhu73O)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35047847)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Identification%20of%2090%20NAFLD%20GWAS%20loci%20and%20establishment%20of%20NAFLD%20PRS%20and%20causal%20role%20of%20NAFLD%20in%20coronary%20artery%20disease&journal=HGG%20Adv.&volume=3&publication_year=2022&author=Miao%2CZ) 
    
76.  Loh, P.-R. et al. Efficient Bayesian mixed-model analysis increases association power in large cohorts. _Nat. Genet._ **47**, 284–290 (2015).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFCntb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25642633)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342297)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20Bayesian%20mixed-model%20analysis%20increases%20association%20power%20in%20large%20cohorts&journal=Nat.%20Genet.&volume=47&pages=284-290&publication_year=2015&author=Loh%2CP-R) 
    
77.  Choi, S. W., Mak, T. S.-H. & O’Reilly, P. F. Tutorial: a guide to performing polygenic risk score analyses. _Nat. Protoc._ **15**, 2759–2772 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVGltLbI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32709988)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612115)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tutorial%3A%20a%20guide%20to%20performing%20polygenic%20risk%20score%20analyses&journal=Nat.%20Protoc.&volume=15&pages=2759-2772&publication_year=2020&author=Choi%2CSW&author=Mak%2CTS-H&author=O%E2%80%99Reilly%2CPF) 
    
78.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. _Am. J. Hum. Genet._ **81**, 559–575 (2007).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 
    
79.  Navarro Gonzalez, J. et al. The UCSC Genome Browser database: 2021 update. _Nucleic Acids Res._ **49**, D1046–D1057 (2021).
    
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33221922)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20UCSC%20Genome%20Browser%20database%3A%202021%20update&journal=Nucleic%20Acids%20Res.&volume=49&pages=D1046-D1057&publication_year=2021&author=Navarro%20Gonzalez%2CJ) 
    
80.  GTEx Consortium. The GTEx Consortium atlas of genetic regulatory effects across human tissues. _Science_ **369**, 1318–1330 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20GTEx%20Consortium%20atlas%20of%20genetic%20regulatory%20effects%20across%20human%20tissues&journal=Science&volume=369&pages=1318-1330&publication_year=2020) 
    
81.  Chen, Z. J. et al. Data files for the manuscript entitled, ‘Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue’. _Zenodo_ [https://doi.org/10.5281/zenodo.15318595](https://doi.org/10.5281/zenodo.15318595) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02300-4?format=refman&flavour=references)

Acknowledgements
----------------

We thank the participants of the Tilkka and METSIM cohorts and the UKB. This study was supported by the National Institutes of Health (grant nos. R01HL170604 to P.P.; R01DK132775 to P.P.; R01HG010505 to E.H. and P.P.); Research Council of Finland (grant nos. 266286, 272376, 314383 and 335443 to K.H.P.); Finnish Medical Foundation; Gyllenberg Foundation; Novo Nordisk Foundation (grant nos. NNF20OC0060547, NNF17OC0027232 and NNF10OC1013354 to K.H.P.); Finnish Diabetes Research Foundation; Paulo Foundation; Sigrid Jusélius Foundation; University of Helsinki and Helsinki University Hospital; and Government Research Funds. This research was conducted using the UKB Resource under application number 33934. This work uses data provided by patients and collected by the NHS as part of their care and support.

Author information
------------------

Author notes

1.  These authors contributed equally: Zeyuan Johnson Chen, Sankha Subhra Das.
    

### Authors and Affiliations

1.  Department of Computer Science, University of California, Los Angeles, CA, USA
    
    Zeyuan Johnson Chen, Oren Avram & Sriram Sankararaman
    
2.  Department of Computational Medicine, University of California, Los Angeles, CA, USA
    
    Zeyuan Johnson Chen, Oren Avram, Elior Rahmani, Sriram Sankararaman & Eran Halperin
    
3.  Department of Human Genetics, David Geffen School of Medicine at UCLA, University of California, Los Angeles, CA, USA
    
    Sankha Subhra Das, Asha Kar, Seung Hyuk T. Lee, Kevin D. Abuhanna, Marcus Alvarez, Mihir G. Sukhatme, Zitian Wang, Kyla Z. Gelev, Matthew G. Heffel, Yi Zhang, Sriram Sankararaman, Chongyuan Luo & Päivi Pajukanta
    
4.  Bioinformatics Interdepartmental Program, University of California, Los Angeles, CA, USA
    
    Asha Kar, Zitian Wang, Matthew G. Heffel, Chongyuan Luo & Päivi Pajukanta
    
5.  Department of Anesthesiology and Perioperative Medicine, University of California, Los Angeles, CA, USA
    
    Oren Avram
    
6.  Department of Medicine, University of Eastern Finland and Kuopio University Hospital, Kuopio, Finland
    
    Markku Laakso
    
7.  Obesity Research Unit, Research Program for Clinical and Molecular Metabolism, Faculty of Medicine, University of Helsinki, Helsinki, Finland
    
    Sini Heinonen & Kirsi H. Pietiläinen
    
8.  Eira Hospital, Helsinki, Finland
    
    Hilkka Peltoniemi
    
9.  HealthyWeightHub, Endocrinology, Abdominal Center, Helsinki University Central Hospital and University of Helsinki, Helsinki, Finland
    
    Kirsi H. Pietiläinen
    
10.  Institute for Precision Health, David Geffen School of Medicine at UCLA, University of California, Los Angeles, CA, USA
    
    Päivi Pajukanta
    

Authors

1.  Zeyuan Johnson Chen
    
    [View author publications](https://www.nature.com/search?author=Zeyuan%20Johnson%20Chen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zeyuan%20Johnson%20Chen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zeyuan%20Johnson%20Chen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  Sankha Subhra Das
    
    [View author publications](https://www.nature.com/search?author=Sankha%20Subhra%20Das)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sankha%20Subhra%20Das) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sankha%20Subhra%20Das%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Asha Kar
    
    [View author publications](https://www.nature.com/search?author=Asha%20Kar)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Asha%20Kar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Asha%20Kar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Seung Hyuk T. Lee
    
    [View author publications](https://www.nature.com/search?author=Seung%20Hyuk%20T.%20Lee)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Seung%20Hyuk%20T.%20Lee) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Seung%20Hyuk%20T.%20Lee%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Kevin D. Abuhanna
    
    [View author publications](https://www.nature.com/search?author=Kevin%20D.%20Abuhanna)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kevin%20D.%20Abuhanna) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kevin%20D.%20Abuhanna%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Marcus Alvarez
    
    [View author publications](https://www.nature.com/search?author=Marcus%20Alvarez)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marcus%20Alvarez) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marcus%20Alvarez%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Mihir G. Sukhatme
    
    [View author publications](https://www.nature.com/search?author=Mihir%20G.%20Sukhatme)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Mihir%20G.%20Sukhatme) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Mihir%20G.%20Sukhatme%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Zitian Wang
    
    [View author publications](https://www.nature.com/search?author=Zitian%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Zitian%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Zitian%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Kyla Z. Gelev
    
    [View author publications](https://www.nature.com/search?author=Kyla%20Z.%20Gelev)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kyla%20Z.%20Gelev) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kyla%20Z.%20Gelev%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Matthew G. Heffel
    
    [View author publications](https://www.nature.com/search?author=Matthew%20G.%20Heffel)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20G.%20Heffel) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20G.%20Heffel%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Yi Zhang
    
    [View author publications](https://www.nature.com/search?author=Yi%20Zhang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yi%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yi%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Oren Avram
    
    [View author publications](https://www.nature.com/search?author=Oren%20Avram)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Oren%20Avram) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Oren%20Avram%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Elior Rahmani
    
    [View author publications](https://www.nature.com/search?author=Elior%20Rahmani)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Elior%20Rahmani) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Elior%20Rahmani%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Sriram Sankararaman
    
    [View author publications](https://www.nature.com/search?author=Sriram%20Sankararaman)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sriram%20Sankararaman) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sriram%20Sankararaman%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Markku Laakso
    
    [View author publications](https://www.nature.com/search?author=Markku%20Laakso)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Markku%20Laakso) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Markku%20Laakso%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Sini Heinonen
    
    [View author publications](https://www.nature.com/search?author=Sini%20Heinonen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sini%20Heinonen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sini%20Heinonen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Hilkka Peltoniemi
    
    [View author publications](https://www.nature.com/search?author=Hilkka%20Peltoniemi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hilkka%20Peltoniemi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hilkka%20Peltoniemi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Eran Halperin
    
    [View author publications](https://www.nature.com/search?author=Eran%20Halperin)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Eran%20Halperin) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Eran%20Halperin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  Kirsi H. Pietiläinen
    
    [View author publications](https://www.nature.com/search?author=Kirsi%20H.%20Pietil%C3%A4inen)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kirsi%20H.%20Pietil%C3%A4inen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kirsi%20H.%20Pietil%C3%A4inen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  Chongyuan Luo
    
    [View author publications](https://www.nature.com/search?author=Chongyuan%20Luo)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Chongyuan%20Luo) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Chongyuan%20Luo%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Päivi Pajukanta
    
    [View author publications](https://www.nature.com/search?author=P%C3%A4ivi%20Pajukanta)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=P%C3%A4ivi%20Pajukanta) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22P%C3%A4ivi%20Pajukanta%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

Z.J.C., S.S.D. and P.P. conceptualized and designed the project. Z.J.C., S.S.D., A.K., S.H.T.L., K.D.A., M.A., M.G.S., Z.W. and K.Z.G. carried out the computational analyses. M.G.H., O.A., E.R., S.S., E.H., C.L. and P.P. suggested the analyses, provided support to perform them and participated in the discussion of results. M.A., Y.Z., S.H.T.L., C.L. and P.P. generated the snm3C-seq data, snRNA-seq and the bulk RNA-seq data. S.H., H.P., M.L. and K.H.P. collected the cohorts and samples. P.P. and E.H. funded the omics data generation and computational resources. Z.J.C., S.S.D., A.K., S.H.T.L., M.G.S., Z.W., K.Z.G. and P.P. wrote the manuscript. All authors read, reviewed and/or edited the manuscript.

### Corresponding author

Correspondence to [Päivi Pajukanta](mailto:ppajukanta@mednet.ucla.edu).

Ethics declarations
-------------------

### Competing interests

The authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Dong-Sung Lee and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Integrative analysis between subcutaneous adipose tissue (SAT) cells profiled by single-nucleus methyl-3C sequencing (snm3C-seq) and single-nucleus RNA sequencing (snRNA-seq).](https://www.nature.com/articles/s41588-025-02300-4/figures/7)

**a**, Dimension reduction of cells (n = 29,423) profiled by snRNA-seq and visualized with uniform manifold approximation and projection (UMAP). **b**, The total number of cells profiled by snm3C-seq and snRNA-seq stratified by the SAT cell-types. **c**, Co-embedding of snm3C-seq gene body mCG and snRNA-seq gene expression, visualized with UMAP, highlighting the transition cell-type in red and other SAT cell-types in grey. **d**, Confusion matrix comparing the concordance between the _de novo_ snm3C-seq annotations (row) and the snRNA-seq-derived annotations (column). The confusion fraction is calculated as the multi-class confusion matrix normalized by the cell counts per row. ASPC, adipose stem and progenitor cell.

### [Extended Data Fig. 2 Gene body mCG and RNA expression profiles across SAT cell-type marker genes and clustering analysis of the transition cell-type.](https://www.nature.com/articles/s41588-025-02300-4/figures/8)

**a-f**, Uniform manifold approximation and projection (UMAP) visualization of the gene body mCG ratio, normalized per cell (left) and log-transformed counts per million normalized gene expression (right) for perivascular marker gene _NOTCH3_ (**a**), ASPC marker gene _COL5A1_ (**b**), endothelial cell marker gene _EGFL7_ (**c**), lymphoid cell marker gene _CD2_ (**d**), mast cell marker gene _SLC18A2_ (**e**), and myeloid cell marker gene _CSF1R_ (**f**). The dashed line highlights the group of cells annotated as the corresponding cell-type. **g**, Gene body hypo-methylation of adipocyte marker genes (left 10 columns) and perivascular cell marker genes (right 10 columns) across adipocytes, perivascular cells, and the transition cell-type. Dot colors represent the average gene body mCG ratio normalized per cell. **h**, Dimension reduction of cells profiled by snm3C-seq and restricted to adipocytes, perivascular cells, and the transition cell-type, using exclusively the 5-kb bin mCG profiles and visualized with UMAP. ASPC, adipose stem and progenitor cell.

### [Extended Data Fig. 3 Comparisons of unique cell-type marker genes in SAT cell-types, and biological processes and functional pathways enriched among the adipocyte marker genes between the gene body mCG and gene expression modalities, as well as both known and less-known transcription factors (TFs) in SAT.](https://www.nature.com/articles/s41588-025-02300-4/figures/9)

**a**, Venn diagrams showing the number of shared and modality-specific unique SAT cell-type marker genes between the gene body mCG and gene expression modalities. **b-c**, Dot plots showing significantly enriched biological processes (**b**) and KEGG functional pathways (**c**) using unique adipocyte marker genes in gene body mCG and gene expression modalities, evaluated using WebGestalt (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). _P_ values were adjusted for multiple hypothesis testing by the Benjamini-Hochberg procedure. Statistical significance was determined as FDR < 0.05. The size of the dot represents the enrichment ratio for biological processes (**b**) and KEGG functional pathways (**c**), while the color of the dot indicates FDR. **d**, Dot plots of fat cell differentiation biological process genes that are shared adipocyte marker genes between the mCG and gene expression modalities, showing their average gene body mCG (left) and average gene expression profiles (right) across the SAT cell-types. Dot size (right) represents the percentage of cells where the gene is expressed. **e-f**, Dot plots of (e) adipocyte marker genes encoding both known and less known TFs that are shared between the gene body mCG and gene expression modalities and (f) adipocyte marker genes encoding both known and less known SAT TFs that are unique to gene body mCG, showing their gene body mCG (left) and gene expression profiles (right). The color of the dot represents the mean percentage of mCG (red is high) and the average expression of a gene. ASPC, adipose stem and progenitor cell; and FDR, false discovery rate.

### [Extended Data Fig. 4 Cell-type level hypo-methylated regions are enriched for specific transcription factor (TF) binding motifs.](https://www.nature.com/articles/s41588-025-02300-4/figures/10)

We show the top five cell-type level TF binding motifs (sorted by _P_) that are enriched among the hypo-methylated regions of the SAT cell-types. Statistical significance was determined using HOMER (binomial test) on each cell-type separately (see [Methods](https://www.nature.com/articles/s41588-025-02300-4#Sec14)). False positive rate was calibrated by a stringent cutoff on the unadjusted _P_ values (that is, _P_ < 1×10−12). ASPC, adipose stem and progenitor cell.

### [Extended Data Fig. 5 Validation of myeloid-specific transcription factors (TFs), IRF4, MEF2B, and CEBPG, using external ENCODE ChIP-seq data.](https://www.nature.com/articles/s41588-025-02300-4/figures/11)

**a-c**, Violin plots reflecting the empirical null distribution (n = 1,000) of the percentages of DMRs overlapping ChIP-seq peaks in each experiment, stratified by the SAT cell-types. Diamonds mark the observed overlapping proportions of the corresponding cell-type level hypo-methylated regions. \*Indicates the statistical significance after multiple hypothesis correction (Bonferroni) on the empirical one-tailed _P_ values for the over-representation of TF peaks. IRF4 in GM12878 cell line (empirical unadjusted _P_\=0.001 for myeloid, mast, and lymphoid cells) (**a**); MEF2B in GM12878 cell line (empirical unadjusted _P_\=0.001 for perivascular, myeloid, mast, and lymphoid cells) (**b**); and CEBPG in K562 cell line (empirical unadjusted _P_\=0.001 for adipocytes, myeloid, and mast cells) (**c**). ChIP-seq, Chromatin immunoprecipitation sequencing; ASPC, adipose stem and progenitor cell; GM12878, lymphoblastoid cell line; and K562, chronic myelogenous leukemia cell line.

### [Extended Data Fig. 6 Cell-type level differences in chromatin conformation of subcutaneous adipose tissue (SAT).](https://www.nature.com/articles/s41588-025-02300-4/figures/12)

**a-b**, Uniform manifold approximation and projection (UMAP) visualization of low dimensional embeddings of cells using compartments (**a**) and insulation scores (**d**) as features, colored by the snm3C-seq annotation. Adjusted rand index (ARI) evaluates the clustering concordance against the snm3C-seq annotation. **c**, Heatmap visualization of the normalized interaction contact map on chromosome 6 and its corresponding compartment scores across the SAT cell-types. **d**, Box plots visualizing the distributions of the number of consecutive 100-kb bins within segments that exhibit consistent A or B compartment annotations, stratified by the SAT cell-types. Lone compartments (that is, singular bins, the annotation of which differs from the ones of both neighbors) are excluded. The center of the box represents the median; the bounds of the box indicate the 25% and 75% percentiles, while the whiskers show the minimum and maximum values within 1.5 times the interquartile range. \*Indicates unadjusted –log10_P_\=25, one-tailed Wilcoxon rank-sum test between the two groups marked by the horizontal line: consistent compartment segments in adipocytes, ASPCs, and perivascular cells (n = 2,890) versus those in myeloid and endothelial cells (n = 2,891). **e**, Horizontal stacked bar plot (left) showing the marginal proportions of differential 100-kb bins stratified by their annotated A and B compartments in the 5 most abundant SAT cell-types and upset plot (right) showing all compartment combinations across differential 100-kb bins in decreasing order with their corresponding percentages. The vertical dotted line separates the following three major categories: ‘Homogeneous’, ‘Cell-type Enriched’, and ‘Heterogeneous’, which correspond to unique A or B compartments in 0, 1, or more than 1 cell-type, respectively. **f**, Sankey diagram breaking down the numbers of differential 100-kb bins annotated as A (red) and B (blue) compartments belonging to ASPCs (left), adipocytes (middle), and myeloid cells (right). **g**, Similar to (f), except on perivascular cells (left), adipocytes (middle), and endothelial cells (right). ASPC, adipose stem and progenitor cell.

### [Extended Data Fig. 7 Cell-type specificity in interaction domains and loops.](https://www.nature.com/articles/s41588-025-02300-4/figures/13)

**a-c**, Box plots visualizing the distribution of the number of interaction domains (**a**), the total number (**b**), and the average span (**c**) of interaction domains detected in each cell, stratified by cell-types. The center of the box represents the median; the bounds of the box indicate the 25% and 75% percentiles, while the whiskers show the minimum and maximum values within 1.5 times the interquartile range. Statistical significance was evaluated using pairwise one-tailed Wilcoxon rank-sum tests against adipocytes. \*\*\* Indicates Bonferroni multiple hypothesis adjusted _P_ < 0.05 and n.s. denotes non-significant. Specifically, the unadjusted -log10_P_\=133, 45, 92, 132, 70, and 50 for ASPCs, perivascular, endothelial, myeloid, lymphoid, and mast cells, respectively in (**a**); the unadjusted -log10_P_\=5, 7, and 4 for ASPCs, myeloid, and lymphoid cells, respectively in (**b**), and the unadjusted -log10_P_\=132, 45, 90, 132, 71, 50 for ASPCs, perivascular, endothelial, myeloid, lymphoid, and mast cells, respectively in (**c**). Each cell is treated as an independent replicate; thus n = 1,212, 1,814, 63, 1206, 482, 1,387, 172, and 316 for adipocytes, ASPCs, transition, endothelial, perivascular, myeloid, mast, and lymphoid cells, respectively. **d**, Scatter plot showing the short to long-range interaction ratio per cell against the number of interaction domains detected. Cells are colored by their snm3C-seq annotation. **e-f**, Scatter plots showing the aggregated cell-type level median number of unique molecular identifiers (UMIs) from snRNA-seq against the median number of interaction domains (**e**) and the ratio of short to long-range interaction contacts (**f**) from snm3C-seq, colored similarly as in **d**. **g-h**, Bar plots showing the median distance (**g**) and the total number (**h**) of loop summits detected across the SAT cell-types (x-axis is ordered by the abundance of each cell-type in snm3C-seq). ASPC, adipose stem and progenitor cell.

### [Extended Data Fig. 8 Browser-based stacked views of multimodal data for adipocyte marker genes in five major SAT cell-types.](https://www.nature.com/articles/s41588-025-02300-4/figures/14)

**a-b**, From top to bottom: chromatin conformation contact heatmap, imputed at a 10-kb resolution and colored at log-normalized scale; boundary probabilities calculated at a 25-kb resolution; and chromosome compartment scores evaluated at a 100-kb resolution, where the A compartments are colored red and the B compartments blue. The shaded trapezoid connects the top panels to a zoomed-in view of the CG methylation patterns surrounding the gene body, extending 5-kb up- and downstream. Vertical dashed lines mark the gene start and end positions. Black dots represent the detected chromatin loop summits. The black dashed triangles in the contact heatmap highlight differential domains that are present only in adipocytes: one encapsulates the _ADIPOQ_ gene (**a**), and two reside within the _TENM3_ gene body (**b**). SAT, subcutaneous adipose tissue; and ASPC, adipose stem and progenitor cell.

### [Extended Data Fig. 9 Mean gene expression of DNA methylation- and demethylation-related genes across the cell-types in subcutaneous adipose tissue (SAT).](https://www.nature.com/articles/s41588-025-02300-4/figures/15)

**a-b**, Dot plot showing expression of (**a**) DNA methylation genes (_DNMT1, DNMT3B_, and _UHRF1_) and (**b**) DNA demethylation genes (_TET2_, _TET3_, and _TDG_) across SAT cell-types. The size of the dot represents the percentage of cells in which a gene is expressed within a cell-type, while the color represents the average expression of each gene across all cells within a cell-type (blue indicates a higher expression). ASPC, adipose stem and progenitor cell.

### [Extended Data Fig. 10 Abdominal obesity -associated variants are enriched for the adipocyte A compartment.](https://www.nature.com/articles/s41588-025-02300-4/figures/16)

**a**, The clumped and thresholded variants (R2 < 0.2, unadjusted _P_ < 0.05) used for constructing the adipocyte compartment PRSs for abdominal obesity (employing WHRadjBMI as a proxy) are plotted by genomic position against the unadjusted -log10_P_ from the UK Biobank WHRadjBMI GWAS (n = 195,863 unrelated Europeans). SNPs landing in the adipocyte A compartments are colored red, while SNPs landing in the adipocyte B compartments are colored blue. The horizontal dashed line indicates genome-wide significance (unadjusted _P_ < 5 × 10−8). **b**, Bar plot showing the number of WHRadjBMI-associated variants in (**a**), stratified by the adipocyte compartment assignment. **c**, Similar to (**b**) except showing the number of independent (R2 < 0.2) WHRadjBMI GWAS variants, passing genome-wide significance (unadjusted _P_ < 5 × 10−8). PRS, polygenic risk score; WHRadjBMI, waist-hip-ratio adjusted for body mass index; and SNP, single nucleotide polymorphism.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_MOESM1_ESM.pdf)

Supplementary Notes, Supplementary Figs. 1–7 and References.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_MOESM2_ESM.pdf)

### [Supplementary Tables 1–14](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02300-4/MediaObjects/41588_2025_2300_MOESM3_ESM.xlsx)

Supplementary Tables 1–14.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Single-cell%20DNA%20methylome%20and%203D%20genome%20atlas%20of%20human%20subcutaneous%20adipose%20tissue&author=Zeyuan%20Johnson%20Chen%20et%20al&contentID=10.1038%2Fs41588-025-02300-4&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-08-20&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02300-4)

### Cite this article

Chen, Z.J., Das, S.S., Kar, A. _et al._ Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue. _Nat Genet_ (2025). https://doi.org/10.1038/s41588-025-02300-4

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02300-4?format=refman&flavour=citation)

-   Received: 03 October 2024
    
-   Accepted: 17 July 2025
    
-   Published: 20 August 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02300-4
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02300-4.pdf)
