> id: https://www.nature.com/articles/s41588-025-02234-x

> link: https://www.nature.com/articles/s41588-025-02234-x

> title: The long-term effects of chemotherapy on normal blood cells

# The long-term effects of chemotherapy on normal blood cells
_Published on Tue, 01 Jul 2025 00:00:00 GMT_

Abstract
--------

Several chemotherapeutic agents act by increasing DNA damage in cancer cells, triggering cell death. However, there is limited understanding of the extent and long-term consequences of collateral DNA damage in normal tissues. To investigate the impact of chemotherapy on mutation burdens and the cell population structure of normal tissue, we sequenced blood cell genomes from 23 individuals aged 3–80 years who were treated with a range of chemotherapy regimens. Substantial additional somatic mutation loads with characteristic mutational signatures were imposed by some chemotherapeutic agents, but the effects were dependent on the drug and blood cell types. Chemotherapy induced premature changes in the cell population structure of normal blood, similar to those caused by normal aging. The results show the long-term biological consequences of cytotoxic agents to which a substantial fraction of the population is exposed as part of disease management, raising mechanistic questions and highlighting opportunities for the mitigation of adverse effects.

### Similar content being viewed by others

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs43018-020-00124-1/MediaObjects/43018_2020_124_Fig1_HTML.png)

### [Mutational and functional genetics mapping of chemotherapy resistance mechanisms in relapsed acute lymphoblastic leukemia](https://www.nature.com/articles/s43018-020-00124-1?fromPaywallRec=false)

Article 19 October 2020

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41408-022-00637-1/MediaObjects/41408_2022_637_Fig1_HTML.png)

### [Time trends in primary therapy and relative survival of diffuse large B-cell lymphoma by stage: a nationwide, population-based study in the Netherlands, 1989–2018](https://www.nature.com/articles/s41408-022-00637-1?fromPaywallRec=false)

Article Open access 09 March 2022

![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41698-023-00373-0/MediaObjects/41698_2023_373_Fig1_HTML.png)

### [Prognostic Mutational Signatures of NSCLC Patients treated with chemotherapy, immunotherapy and chemoimmunotherapy](https://www.nature.com/articles/s41698-023-00373-0?fromPaywallRec=false)

Article Open access 27 March 2023

Main
----

Over the course of a lifetime, one in two people develops cancer. A long-standing approach to cancer treatment is systemic administration of a diverse group of cytotoxic chemicals, often termed ‘chemotherapy’, which includes alkylating agents, platinum compounds, antimetabolites, topoisomerase inhibitors, vinca alkaloids and cytotoxic antibiotics[1](https://www.nature.com/articles/s41588-025-02234-x#ref-CR1 "DeVita, V. T. Jr. & Chu, E. A history of cancer chemotherapy. Cancer Res. 68, 8643–8653 (2008)."). Some of these agents exert their therapeutic effects by causing damage to DNA that, in turn, triggers the death of malignant cells[2](https://www.nature.com/articles/s41588-025-02234-x#ref-CR2 "Tilsed, C. M., Fisher, S. A., Nowak, A. K., Lake, R. A. & Lesterhuis, W. J. Cancer chemotherapy: insights into cellular and tumor microenvironmental mechanisms of action. Front. Oncol. 12, 960317 (2022)."). Approximately 30% of individuals with cancer, and thus approximately 10% of the whole population in developed countries, are exposed to chemotherapy at some point in their lifetime ([www.cancerresearchuk.org/health-professional/cancer-statistics-for-the-uk](http://www.cancerresearchuk.org/health-professional/cancer-statistics-for-the-uk)), providing considerable exposure of normal tissues to the actions of these drugs.

Chemotherapy can have long-term side effects on normal tissues. It confers an increased risk of cancers of the blood[3](https://www.nature.com/articles/s41588-025-02234-x#ref-CR3 "Kollmannsberger, C., Hartmann, J. T., Kanz, L. & Bokemeyer, C. Risk of secondary myeloid leukemia and myelodysplastic syndrome following standard-dose chemotherapy or high-dose chemotherapy with stem cell support in patients with potentially curable malignancies. J. Cancer Res. Clin. Oncol. 124, 207–214 (1998)."),[4](https://www.nature.com/articles/s41588-025-02234-x#ref-CR4 "Bhatia, S. Therapy-related myelodysplasia and acute myeloid leukemia. Semin. Oncol. 40, 666–675 (2013)."),[5](https://www.nature.com/articles/s41588-025-02234-x#ref-CR5 "Morton, L. M. et al. Association of chemotherapy for solid tumors with development of therapy-related myelodysplastic syndrome or acute myeloid leukemia in the modern era. JAMA Oncol. 5, 318–325 (2019)."),[6](https://www.nature.com/articles/s41588-025-02234-x#ref-CR6 "Bhatia, S. & Sklar, C. Second cancers in survivors of childhood cancer. Nat. Rev. Cancer 2, 124–132 (2002)."), lung, bladder and colon[7](https://www.nature.com/articles/s41588-025-02234-x#ref-CR7 "Travis, L. B. The epidemiology of second primary cancers. Cancer Epidemiol. Biomarkers Prev. 15, 2020–2026 (2006)."),[8](https://www.nature.com/articles/s41588-025-02234-x#ref-CR8 "Allan, J. M. & Travis, L. B. Mechanisms of therapy-related carcinogenesis. Nat. Rev. Cancer 5, 943–955 (2005).") and is sometimes toxic to the kidney, blood, heart, brain, gastrointestinal tract, peripheral nervous system and gonads, engendering long-term deterioration in organ function[9](https://www.nature.com/articles/s41588-025-02234-x#ref-CR9 "Nicolaysen, A. Nephrotoxic chemotherapy agents: old and new. Adv. Chronic Kidney Dis. 27, 38–49 (2020)."),[10](https://www.nature.com/articles/s41588-025-02234-x#ref-CR10 "Groopman, J. E. & Itri, L. M. Chemotherapy-induced anemia in adults: incidence and treatment. J. Natl Cancer Inst. 91, 1616–1634 (1999)."),[11](https://www.nature.com/articles/s41588-025-02234-x#ref-CR11 "Crawford, J., Dale, D. C. & Lyman, G. H. Chemotherapy-induced neutropenia: risks, consequences, and new directions for its management. Cancer 100, 228–237 (2004)."),[12](https://www.nature.com/articles/s41588-025-02234-x#ref-CR12 "Szász, R., Telek, B. & Illés, Á. Fludarabine–cyclophosphamide–rituximab treatment in chronic lymphocytic leukemia, focusing on long term cytopenias before and after the era of targeted therapies. Pathol. Oncol. Res. 27, 1609742 (2021)."),[13](https://www.nature.com/articles/s41588-025-02234-x#ref-CR13 "Robison, L. L. & Hudson, M. M. Survivors of childhood and adolescent cancer: life-long risks and responsibilities. Nat. Rev. Cancer 14, 61–70 (2014)."),[14](https://www.nature.com/articles/s41588-025-02234-x#ref-CR14 "Erdmann, F. et al. Childhood cancer: survival, treatment modalities, late effects and improvements over time. Cancer Epidemiol. 71, 101733 (2021)."). There is limited understanding of the biological mechanisms underlying these sequelae. It is plausible that some are related to the consequences of DNA damage and thus could be elucidated through the genome sequences of normal tissues, which may reveal changes in somatic mutation burdens or clonal composition immediately or decades following chemotherapy.

Sequencing of cancers arising after chemotherapy treatment has revealed variably elevated somatic mutation loads, in some instances characterized by distinct mutational signatures[15](https://www.nature.com/articles/s41588-025-02234-x#ref-CR15 "Alexandrov, L. B. et al. Signatures of mutational processes in human cancer. Nature 500, 415–421 (2013)."),[16](https://www.nature.com/articles/s41588-025-02234-x#ref-CR16 "Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)."),[17](https://www.nature.com/articles/s41588-025-02234-x#ref-CR17 "Pich, O. et al. The mutational footprints of cancer therapies. Nat. Genet. 51, 1732–1740 (2019)."),[18](https://www.nature.com/articles/s41588-025-02234-x#ref-CR18 "de Kanter, J. K. et al. Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients. Cell Stem Cell 28, 1726–1739 (2021)."). However, there is little direct information concerning the mutagenic effects of chemotherapy on normal tissues in vivo. Studies of a small number of individuals show that normal colorectal epithelium, blood and sperm can exhibit additional somatic mutations after chemotherapy[19](https://www.nature.com/articles/s41588-025-02234-x#ref-CR19 "Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019)."),[20](https://www.nature.com/articles/s41588-025-02234-x#ref-CR20 "Kaplanis, J. et al. Genetic and chemotherapeutic influences on germline hypermutation. Nature 605, 503–508 (2022)."). Furthermore, chemotherapy can alter the clonal structure of normal cell populations, as illustrated in the blood, where treatment increases the incidence of clonal hematopoiesis, favoring clones with driver mutations in the DNA damage response genes _PPM1D_, _TP53_ and _CHEK2_ (refs. [21](https://www.nature.com/articles/s41588-025-02234-x#ref-CR21 "Bolton, K. L. et al. Cancer therapy shapes the fitness landscape of clonal hematopoiesis. Nat. Genet. 52, 1219–1226 (2020)."),[22](https://www.nature.com/articles/s41588-025-02234-x#ref-CR22 "Hsu, J. I. et al. PPM1D mutations drive clonal hematopoiesis in response to cytotoxic chemotherapy. Cell Stem Cell 23, 700–713 (2018)."),[23](https://www.nature.com/articles/s41588-025-02234-x#ref-CR23 "Coombs, C. C. et al. Therapy-related clonal hematopoiesis in patients with non-hematologic cancers is common and associated with adverse clinical outcomes. Cell Stem Cell 21, 374–382 (2017).")).

To survey the long-term impacts of chemotherapeutic agents on normal body tissues, we here investigate their effects on normal blood by whole-genome sequencing (WGS) of cells from chemotherapy-exposed individuals. The blood offers several desirable features in this regard, including ease of randomly sampling cells from the whole tissue, predictable mutation accumulation in unexposed individuals[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."), opportunities to interrogate different cell subtypes and maturation states, and feasibility of surveying changes in cell population clonal structure.

Results
-------

### Genome sequencing of chemotherapy-exposed blood

To conduct a primary survey of chemotherapy effects on normal blood cell genomes, we analyzed 23 individuals with hematological or solid malignancies who had collectively been exposed to multiple chemotherapy classes and multiple members of each class, with variable time intervals since exposure. These individuals were aged 3–80 years and had been treated with commonly used chemotherapy regimens for hematological malignancies (Hodgkin lymphoma, _n_ = 2; follicular lymphoma, _n_ = 5; diffuse large B cell lymphoma, _n_ = 2; lymphoplasmacytic lymphoma, _n_ = 1; marginal zone lymphoma, _n_ = 1; multiple myeloma, _n_ = 1; acute myeloid leukemia (AML), _n_ = 1) and solid cancers (colorectal carcinoma, _n_ = 9; neuroblastoma, _n_ = 1; lung cancer, _n_ = 1). One individual had been treated with chemotherapy for both multiple myeloma and colorectal carcinoma. The individual with AML had also been treated with chemotherapy for Behcet disease, a noncancer condition (Fig. [1a](https://www.nature.com/articles/s41588-025-02234-x#Fig1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). Most had received a combination of agents and, collectively, had been exposed to 21 drugs from all of the main chemotherapy classes, including alkylating agents (cyclophosphamide, _n_ = 8; chlorambucil, _n_ = 2; bendamustine, _n_ = 5; procarbazine, _n_ = 2; melphalan, _n_ = 1), platinum agents (oxaliplatin, _n_ = 7; carboplatin, _n_ = 2; cisplatin, _n_ = 1), antimetabolites (capecitabine, _n_ = 7; 5-fluorouracil, _n_ = 6; gemcitabine, _n_ = 1; cytarabine, _n_ = 1), topoisomerase I inhibitors (irinotecan, _n_ = 5), topoisomerase II inhibitors (etoposide, _n_ = 4; doxorubicin, _n_ = 4; daunorubicin, _n_ = 1; mitoxantrone, _n_ = 1), vinca alkaloids (vincristine, _n_ = 7; vinblastine, _n_ = 1; vinorelbine, _n_ = 1) and cytotoxic antibiotics (bleomycin, _n_ = 1). The time intervals from chemotherapy exposure to tissue sampling ranged from less than 1 month to 6 years for most cases. However, one individual sampled at age 48 years had been treated for Hodgkin lymphoma at ages 10 and 47 years. Additionally, the individual sampled at age 43 years following induction chemotherapy for AML had also received long-term chlorambucil for Behcet disease diagnosed at age 13 years. Seven patients had also received localized radiotherapy (Supplementary Table [1](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). We endeavored to exclude a chemotherapy agent being administered in the context of only a single cancer type to avoid any confounding effects, but this was not always possible (Fig. [1a](https://www.nature.com/articles/s41588-025-02234-x#Fig1)). Results were compared to those from nine healthy, non-chemotherapy-exposed individuals (Fig. [1a](https://www.nature.com/articles/s41588-025-02234-x#Fig1) and Supplementary Table [1](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)).

**Fig. 1: Donor information and experimental approach.**

[![figure 1](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_Fig1_HTML.png)](https://www.nature.com/articles/s41588-025-02234-x/figures/1)

**a**, Donor demographic details, chemotherapy exposure and sample information. CC, colorectal carcinoma; LC, lung cancer; NB, neuroblastoma; FL, follicular lymphoma; DLBL, diffuse large B cell lymphoma; MZL, marginal zone lymphoma; LL, lymphoplasmacytic lymphoma; M, multiple myeloma; HL, Hodgkin lymphoma; 5FU, 5-fluorouracil; Topo, topoisomerase. **b**, Experimental approach.

Three experimental designs for detecting and analyzing somatic mutations were used. First, 189 single-cell-derived hematopoietic stem and progenitor cell (HSPC) colonies from the 23 chemotherapy-exposed individuals and 90 colonies from the 9 controls were expanded and individually subjected to WGS at 23-fold average coverage to compare mutation burdens and mutational signatures (Extended Data Fig. [1a,b](https://www.nature.com/articles/s41588-025-02234-x#Fig6)). Second, from six individuals exposed to a range of chemotherapeutic agents, a further 589 single-cell colonies underwent WGS (41–259 colonies per individual; mean sequencing depth 15-fold). These phylogenies were compared to similar-sized phylogenies (608 colonies) from five normal individuals across a similar age range to survey the effect of chemotherapy on the clonal structure of the HSPC population. Third, flow-sorted subpopulations of B cells, T memory cells, T naive cells and monocytes from whole-blood samples from 18 chemotherapy-exposed individuals and 3 unexposed normal individuals (Fig. [1b](https://www.nature.com/articles/s41588-025-02234-x#Fig1)) underwent WGS using duplex sequencing, which allows reliable identification of somatic mutations in polyclonal cell populations[25](https://www.nature.com/articles/s41588-025-02234-x#ref-CR25 "Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. Nature 593, 405–410 (2021).").

### Chemotherapy-induced somatic mutations in the blood

Somatic single-base substitution (SBS) mutations in HSPCs from normal adults accrue constantly at a rate of ~18 per year, leading to a burden of ~1,500 SBSs in 80-year-old individuals[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). HSPCs from 17 of the 23 chemotherapy-exposed individuals showed elevated mutation burdens compared to those expected for their ages (_P_ < 2.2 × 10−16, mixed-effects model) (Fig. [2](https://www.nature.com/articles/s41588-025-02234-x#Fig2)). Four showed large increases of >1,000 SBSs (Fig. [2a](https://www.nature.com/articles/s41588-025-02234-x#Fig2)), thirteen showed more modest increases of 200–600 SBSs (Fig. [2b](https://www.nature.com/articles/s41588-025-02234-x#Fig2)), and six showed no increases (Extended Data Fig. [1c–f](https://www.nature.com/articles/s41588-025-02234-x#Fig6)). The burdens of small indels in HSPCs were also increased in the four individuals with the greatest elevations in SBS burdens (Extended Data Fig. [2a,b](https://www.nature.com/articles/s41588-025-02234-x#Fig7)). Increases in structural variant and copy number changes were not observed, including in those individuals exposed to topoisomerase II inhibitors, which have been implicated in the development of secondary malignancies driven by specific oncogenic rearrangements[26](https://www.nature.com/articles/s41588-025-02234-x#ref-CR26 "Zhang, W., Gou, P., Dupret, J.-M., Chomienne, C. & Rodrigues-Lima, F. Etoposide, an anticancer drug involved in therapy-related secondary leukemia: enzymes at play. Transl. Oncol. 14, 101169 (2021).") (Extended Data Fig. [2c,d](https://www.nature.com/articles/s41588-025-02234-x#Fig7)). However, the small number of individuals may have limited the statistical power to identify minor differences.

**Fig. 2: Mutation burden and mutational signatures in normal and chemotherapy-exposed blood cells.**

[![figure 2](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_Fig2_HTML.png)](https://www.nature.com/articles/s41588-025-02234-x/figures/2)

**a**, Burden of SBS across normal individuals and the four chemotherapy-exposed individuals with the highest SBS burdens. The points represent individual HSPC colonies. The boxes indicate the median and interquartile range; the whiskers denote the minimum and maximum values. The black line represents a regression of age on mutation burden across the unexposed individuals, with the 95% confidence interval shaded. Annotations indicate the corresponding individuals from Fig. [1](https://www.nature.com/articles/s41588-025-02234-x#Fig1), providing details on the type of malignancy (as previously defined) and chemotherapy treatment (1, platinum agents; 2, alkylating agents; 3, antimetabolites; 4, topoisomerase I inhibitors; 5, topoisomerase II inhibitors; 6, vinca alkaloids; 7, cytotoxic antibiotics). **b**, Depiction of data as in **a**, but the _y_ axis is cut off at 2,000 SBSs for better visualization of the majority of the chemotherapy-exposed cohort data. The points represent individual HSPC colonies. The boxes indicate the median and interquartile range; the whiskers denote the minimum and maximum values. The gray shading in **b** represents the 95% CI of the regression of age on mutation burden across the unexposed individuals. The black line represents a regression of age on mutation burden, with the 95% confidence interval shaded. **c**, Mutational signatures extracted using the hierarchical Dirichlet process (HDP) from the full dataset of normal and chemotherapy-exposed HSPC colonies and duplex sequencing of bulk mature blood cell subsets.

[Source data](https://www.nature.com/articles/s41588-025-02234-x#MOESM5)

Nineteen of the twenty-three chemotherapy-treated individuals received multiple agents. Therefore, in many cases, it was uncertain which agents were responsible for the elevated mutation loads. To address this, we extracted mutational signatures from the SBS and indel mutation catalogs of chemotherapy-exposed individuals and controls and estimated the contribution of each signature to the somatic mutations in the blood cells of each individual (Fig. [2c](https://www.nature.com/articles/s41588-025-02234-x#Fig2) and Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig8)). We used prior knowledge of previously described mutational signatures attributed to normal endogenous mutational processes and to some mutagenic exposures[27](https://www.nature.com/articles/s41588-025-02234-x#ref-CR27 "Tate, J. G. et al. COSMIC: the catalogue of somatic mutations in cancer. Nucleic Acids Res. 47, D941–D947 (2019)."), as well as the specific chemotherapy regimens received by each individual, to associate each signature with its putative causative agent.

Twelve SBS mutational signatures were extracted (Fig. [2c](https://www.nature.com/articles/s41588-025-02234-x#Fig2) and Supplementary Table [2](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). Four were composed of known signatures of normal HSPCs and mature lymphocytes (Supplementary Table [3](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). The first was predominantly constituted by SBS1, characterized by C>T mutations at CG dinucleotides, together with a contribution from SBS5, which is relatively flat and featureless. SBS1 and SBS5 are found in most normal cell types thus far studied. The second was SBSBlood, a blood-specific signature predominant in HSPCs[28](https://www.nature.com/articles/s41588-025-02234-x#ref-CR28 "Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Nature 561, 473–478 (2018)."),[29](https://www.nature.com/articles/s41588-025-02234-x#ref-CR29 "Osorio, F. G. et al. Somatic mutations reveal lineage relationships and age-related mutagenesis in human hematopoiesis. Cell Rep. 25, 2308–2316 (2018)."). The third was SBS7a, an ultraviolet light-caused signature found in memory T cells that have presumably resided in the skin during life[30](https://www.nature.com/articles/s41588-025-02234-x#ref-CR30 "Machado, H. E. et al. Diverse mutational landscapes in human lymphocytes. Nature 608, 724–732 (2022)."). The fourth was SBS9, a signature of somatic hypermutation found in B cells (Fig. [2c](https://www.nature.com/articles/s41588-025-02234-x#Fig2)). Three indel mutational signatures were extracted (Supplementary Table [4](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). Two were similar to known indel signatures and were present in both normal and chemotherapy-exposed individuals: the first comprised ID1, and the second was a composite of ID3, ID5 and ID9 (ID3/5/9; Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig8)).

Eight SBS mutational signatures were interpreted as being present exclusively in chemotherapy-treated individuals (Fig. [2c](https://www.nature.com/articles/s41588-025-02234-x#Fig2)), based on the observation that they accounted for <1% of mutations in HSPCs from adult controls (Extended Data Fig. [4](https://www.nature.com/articles/s41588-025-02234-x#Fig9)). Four of these have not been reported previously and are not represented in the COSMIC SBS mutational signature database. SBSA is likely due to the triazene alkylating agent procarbazine. There were three similar but distinct signatures relating predominantly to specific nitrogen mustard alkylating agents: SBSC to chlorambucil, SBSD to bendamustine and SBSE to melphalan. SBSF is associated with the platinum agents cisplatin and carboplatin, and SBSG is associated with the antimetabolite 5-fluorouracil or its prodrug capecitabine. The etiologies of SBSB and SBSH are less clear-cut and are discussed further below. Excess SBSs and specific SBS mutational signatures were not obviously associated with topoisomerase inhibitors (which cause DNA strand breaks), vinca alkaloids (which inhibit microtubule formation during cell division) and the cytotoxic antibiotic bleomycin (which is thought to bind and cleave DNA). Only one high-confidence indel mutational signature was found exclusively in chemotherapy-treated individuals: IDA, associated with procarbazine exposure.

SBSA contributed substantial additional mutation loads to blood cells from two individuals treated for Hodgkin lymphoma (PD50308 and PD47703) (Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig3)). The only chemotherapy common to their treatment regimens was the alkylating agent procarbazine; no other individuals had been treated with procarbazine, and HSPC phylogenies indicated that SBSA mutations occurred early during PD47703’s life, consistent with procarbazine treatment at age 10 years (Extended Data Fig. [5a](https://www.nature.com/articles/s41588-025-02234-x#Fig10)). The observed mutational signatures were compared to those previously reported, using a combination of visual inspection, data review and the cosine similarity between trinucleotide mutation profiles. Cosine similarities >0.90 between two mutational signatures are highly unlikely to occur by chance, and a cosine similarity of >0.95 generally suggests the same underlying process. SBSA exhibits similarity to COSMIC signature SBS25 (cosine similarity 0.84), which has previously been associated with procarbazine[19](https://www.nature.com/articles/s41588-025-02234-x#ref-CR19 "Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019)."),[31](https://www.nature.com/articles/s41588-025-02234-x#ref-CR31 "Maura, F. et al. Molecular evolution of classic Hodgkin lymphoma revealed through whole-genome sequencing of Hodgkin and Reed Sternberg cells. Blood Cancer Discov. 4, 208–227 (2023)."). An indel signature (IDA) was also identified as being most likely attributable to procarbazine, being found only in the two individuals treated with procarbazine (Extended Data Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig8)). Alkylating agents cause alkyl DNA adducts, resulting in base mispairing and DNA breaks. Procarbazine is a triazene/hydrazine monofunctional alkylating agent.

**Fig. 3: Phylogenetic trees and mutational signatures across a range of normal and chemotherapy-exposed individuals.**

[![figure 3](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_Fig3_HTML.png)](https://www.nature.com/articles/s41588-025-02234-x/figures/3)

Phylogenies were constructed using shared mutation data and the algorithm MPBoot ([Methods](https://www.nature.com/articles/s41588-025-02234-x#Sec7)). Branch lengths correspond to SBS burdens (_x_ axes). A stacked bar plot represents the signatures contributing to each branch, with the color code below the trees. SBSUnassigned indicates mutations that could not confidently be assigned to any reported signature. Drugs in parentheses are those received by the individual at the same time but not believed to be the mutagenic agents.

SBSB was found predominantly in the individual exposed to chlorambucil, procarbazine and bendamustine (PD47703). SBSB, like SBSA (procarbazine), is predominantly composed of T>A substitutions, with a cosine similarity to SBSA of 0.82 and cosine similarities to SBSD (bendamustine; 0.82) and SBSC (chlorambucil; 0.74) suggesting that it is unlikely to be due to any of these in isolation. It is also present at low levels in the T memory cells of the other procarbazine-exposed individual who was also exposed to cyclophosphamide (PD50308). It seems plausible that SBSB may result from an interaction between two classes of alkylating agents.

Of the nitrogen mustard-associated signatures, SBSC contributed all mutations to the individual who received chlorambucil from childhood (PD37580); SBSD contributed all excess mutations to one of the individuals exposed to only bendamustine (PD60010) and was also present at a much lower burden in a subset of cyclophosphamide-exposed individuals; SBSE was found only in the single individual exposed to low-dose melphalan (PD47699). Nitrogen mustard alkylating agents have two reactive sites and are, in consequence, bifunctional, forming intrastrand and interstrand DNA cross-links in addition to simple adducts. The SBSC and SBSD signatures identified here are similar to a recently published mutational signature found in the germlines of two individuals whose fathers had been treated with two different nitrogen mustard agents (chlorambucil and ifosfamide)[20](https://www.nature.com/articles/s41588-025-02234-x#ref-CR20 "Kaplanis, J. et al. Genetic and chemotherapeutic influences on germline hypermutation. Nature 605, 503–508 (2022)."),[32](https://www.nature.com/articles/s41588-025-02234-x#ref-CR32 "Bertrums, E. J. M. et al. Elevated mutational age in blood of children treated for cancer contributes to therapy-related myeloid neoplasms. Cancer Discov. 12, 1860–1872 (2022)."), and SBSE is similar (cosine similarity 0.84) to the previously described signature in multiple myeloma genomes with prior melphalan exposure[33](https://www.nature.com/articles/s41588-025-02234-x#ref-CR33 "Bolli, N. et al. Heterogeneity of genomic evolution and mutational profiles in multiple myeloma. Nat. Commun. 5, 2997 (2014)."),[34](https://www.nature.com/articles/s41588-025-02234-x#ref-CR34 "Rustad, E. H. et al. Timing the initiation of multiple myeloma. Nat. Commun. 11, 1917 (2020)."),[35](https://www.nature.com/articles/s41588-025-02234-x#ref-CR35 "Maura, F. et al. The mutagenic impact of melphalan in multiple myeloma. Leukemia 35, 2145–2150 (2021).").

SBSF was found in individuals treated with carboplatin or cisplatin and in a subset of oxaliplatin-treated individuals in whom it was present at much lower burdens. It is highly similar to COSMIC SBS31 (cosine similarity 0.95), which has previously been associated with prior platinum exposure in cancer genomes[32](https://www.nature.com/articles/s41588-025-02234-x#ref-CR32 "Bertrums, E. J. M. et al. Elevated mutational age in blood of children treated for cancer contributes to therapy-related myeloid neoplasms. Cancer Discov. 12, 1860–1872 (2022)."),[36](https://www.nature.com/articles/s41588-025-02234-x#ref-CR36 "Coorens, T. H. H. et al. Clonal hematopoiesis and therapy-related myeloid neoplasms following neuroblastoma treatment. Blood 137, 2992–2997 (2021).") (Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig3)). Platinum compounds act by binding DNA and forming intrastrand and interstrand DNA cross-links, in a similar manner to bifunctional alkylating agents. However, SBSF/SBS31 is different from the bifunctional nitrogen mustard signatures, indicating that the patterns of DNA damage and/or DNA repair induced by platinum agents and nitrogen mustards differ.

SBSG is highly similar to COSMIC SBS17 (cosine similarity 0.93), which has previously been found in the genomes of cancers exposed to 5-fluorouracil[37](https://www.nature.com/articles/s41588-025-02234-x#ref-CR37 "Christensen, S. et al. 5-Fluorouracil treatment induces characteristic T>G mutations in human cancer. Nat. Commun. 10, 4571 (2019).") and in the normal intestine of one 5-fluorouracil-exposed individual[19](https://www.nature.com/articles/s41588-025-02234-x#ref-CR19 "Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019)."). It was undetectable in HSPCs and found at the highest burdens in lymphoid cells from individuals treated with 5-fluorouracil or its prodrug capecitabine (Fig. [4](https://www.nature.com/articles/s41588-025-02234-x#Fig4)). 5-Fluorouracil is a pyrimidine analog misincorporated into DNA in place of thymine, consistent with causing a mutational signature characterized predominantly by thymine mutations.

**Fig. 4: Mutation burden and SBS mutational signatures across different blood cell types.**

[![figure 4](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_Fig4_HTML.png)](https://www.nature.com/articles/s41588-025-02234-x/figures/4)

Stacked bar plots represent the absolute contributions of each SBS mutational signature to the SBS mutation burden across cell types (left), compared to the proportionate contribution of each signature (right). HSPC data were generated by pooling HSPC WGS colony data from each individual. Mature blood cell data were generated using duplex sequencing of ~40,000 cells of each type. For the normal unexposed individuals, the T cell subset data are from CD4+ T cells; for the chemotherapy-exposed individuals, the T cell subsets contain both CD4+ and CD8+ T cells. SBSUnassigned indicates mutations that could not confidently be assigned to any reported signature. SBSNA indicates that duplex sequencing data are unavailable for this subset. In seven individuals, granulocyte mutation profiles were available, which were not discernibly different from the mutational spectra observed in HSPCs and monocytes from those individuals. Due to the lack of availability of this cell type for most patients, the data are not shown.

[Source data](https://www.nature.com/articles/s41588-025-02234-x#MOESM5)

SBSH was detectable only in the T cells of a single individual who was also the only person to have received gemcitabine, a cytosine analog. However, the origin of SBSH remains uncertain.

The isolation of multiple HSPC colonies from each individual allowed for assessing the variation in mutagenic exposures across each of their HSPC populations. Although there was some variability in the mutation burdens attributable to cisplatin/carboplatin, procarbazine, chlorambucil and bendamustine (the most highly mutagenic agents) across HSPCs from each individual (Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig3)), the evidence suggests that there were no HSPCs completely protected from DNA damage. The multiple HSPCs from each individual also allowed the formation of phylogenetic trees, permitting the timing of mutagenic impacts. The phylogenetic timings were in keeping with the known periods of exposure: PD47703 with both early-life exposure to procarbazine and chlorambucil and later-life exposure to bendamustine, PD37580 with both early- and late-life exposure to chlorambucil, and PD47699 with late-life exposure to melphalan (Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig3)).

Although limited numbers of individuals, different drug combinations and different dose regimens preclude definitive evaluation, the inclusion of individuals treated with different members of the same chemotherapy class enabled a preliminary comparison of their effects. Among the nitrogen mustard alkylating agents, chlorambucil, bendamustine and melphalan caused substantially greater alkylating agent-associated mutation burdens in normal blood cells than cyclophosphamide, which engendered only minimal (<5% and hence not shown in the figures) increases in mutation load (Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig3), Extended Data Fig. [5b](https://www.nature.com/articles/s41588-025-02234-x#Fig10) and Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02234-x#MOESM4) and [6](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). Similarly, carboplatin and cisplatin caused much higher SBSF mutation burdens than oxaliplatin, which conferred SBSF mutation burdens of <5% in all cases, despite prolonged oxaliplatin treatment (up to 22 cycles) in some individuals (Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#Fig3), Extended Data Fig. [6](https://www.nature.com/articles/s41588-025-02234-x#Fig11) and Supplementary Tables [5](https://www.nature.com/articles/s41588-025-02234-x#MOESM4) and [6](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). Therefore, chemotherapeutic agents of the same class, some used interchangeably in cancer treatment, may confer substantially different mutation burdens in normal blood cells.

Flow sorting of monocytes, B cells, T memory cells and T naive cells enabled us to investigate the responses of different cell types to identical chemotherapy exposures. Overall, the patterns of SBS signature burdens in monocytes were similar to those in HSPCs, whereas the patterns in B and T lymphocytes showed differences for some agents (Supplementary Table [7](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). For example, SBSG, caused by 5-fluorouracil/capecitabine, contributed additional mutation burdens in B lymphocytes (_P_ = 0.0), T naive cells (_P_ = 0.0097) and T memory lymphocytes (_P_ = 0.0014), but was undetectable in HSPCs and monocytes (Fig. [4](https://www.nature.com/articles/s41588-025-02234-x#Fig4)). In contrast, SBSF, caused by the platinum agents, contributed larger mutation burdens in HSPCs, monocytes and B cells than in T naive and T memory cells, although we only have T cell data for one carboplatin-exposed individual (Fig. [4](https://www.nature.com/articles/s41588-025-02234-x#Fig4)). The mutation loads contributed by SBSA, caused by procarbazine, were similar across cell types. Therefore, some chemotherapeutic agents engender different mutation burdens in different cell types.

### Hematopoietic clonal architecture after chemotherapy

To investigate the effect of chemotherapies on the architecture of cell populations, we generated extensive phylogenies of HSPCs from six chemotherapy-exposed individuals and compared them to the HSPC phylogenies of nonexposed individuals of similar ages. An exemplar HSPC phylogeny of a normal, non-chemotherapy-exposed 48-year-old individual showed only one barely detectable clonal expansion and no ‘driver’ mutations in cancer genes (Fig. [5a](https://www.nature.com/articles/s41588-025-02234-x#Fig5)). Such trees are typical of healthy middle-aged adults[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).").

**Fig. 5: HSPC phylogenies for two normal unexposed and two chemotherapy-exposed adult individuals.**

[![figure 5](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_Fig5_HTML.png)](https://www.nature.com/articles/s41588-025-02234-x/figures/5)

**a**,**b**, Phylogenies for two normal unexposed donors: one young adult (**a**) and one older adult (**b**). **c**,**d**, Phylogenies for two young adult chemotherapy-treated individuals, both with more than one chemotherapy exposure. Phylogenies were constructed using shared mutation data and the algorithm MPBoot ([Methods](https://www.nature.com/articles/s41588-025-02234-x#Sec7)). Branch lengths reflect the number of mutations assigned to the branch, with the terminal branches adjusted for sequence coverage; the overall root-to-tip branch lengths have been normalized to the same total length (because all colonies were collected from a single time point). The _y_ axis represents the number of SBSs accumulating over time. Each tip on a phylogeny represents a single colony, with the respective numbers of colonies of each cell and tissue type recorded at the top. Onto these trees, we have layered clone- and colony-specific phenotypic information. We have highlighted branches on which we have identified known oncogenic drivers in 1 of 18 clonal hematopoiesis genes (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)) color-coded by gene. A heat map at the bottom of each phylogeny highlights colonies from known driver clades colored by gene and the expanded clades (defined as those with a clonal fraction of >1%) in blue. In the individual in **d**, the AML was derived from the biallelic _TP53_\-mutated clade carrying _TP53_ p.I195F and _TP53_ p.C176Y. Drugs not highlighted in bold text are those received by the individual at the same time but not believed to be the mutagenic agents.

Given that changes in the clonal composition of the HSPC population due to chemotherapy-induced bottlenecks and positive selection may take many years to become apparent, we focused on two individuals sampled 30 and 39 years after their earliest exposure to chemotherapy (PD37580 and PD47703; Fig. [5c,d](https://www.nature.com/articles/s41588-025-02234-x#Fig5)). In contrast to normal middle-aged individuals, a 48-year-old woman (PD47703) treated for Hodgkin lymphoma with chlorambucil and procarbazine at age 10 years and bendamustine at age 47 years showed multiple independent clonal expansions carrying ‘driver’ mutations in the DNA damage response gene _PPM1D_ (Fig. [5c](https://www.nature.com/articles/s41588-025-02234-x#Fig5)). A similar pattern, with expanded _PPM1D_ and _TP53_ mutant clones, was observed in a 43-year-old woman (PD37580) after long-term chlorambucil treatment (Fig. [5d](https://www.nature.com/articles/s41588-025-02234-x#Fig5)). This pattern of multiple, large clonal expansions is characteristic of normal individuals aged >70 years[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). However, in healthy older adults, clonal expansions exhibit predominantly _DNMT3A_ and _TET2_ driver mutations or no apparent driver (Fig. [5b](https://www.nature.com/articles/s41588-025-02234-x#Fig5)). While _TP53_ and _PPM1D_ mutations can also be observed in the context of normal aging, the pattern of dozens of parallel clonal expansions harboring these mutations is likely to reflect the unique selective landscape induced by some chemotherapies.

Chemotherapy could induce this prematurely aged HSPC cell population profile by increasing mutation loads and/or by altering microenvironmental selection. Chemotherapy favors the survival of clonal hematopoiesis of indeterminate potential (CHIP) clones with driver mutations in _PPM1D_, _TP53_ and _CHEK2_ (ref. [21](https://www.nature.com/articles/s41588-025-02234-x#ref-CR21 "Bolton, K. L. et al. Cancer therapy shapes the fitness landscape of clonal hematopoiesis. Nat. Genet. 52, 1219–1226 (2020).")), which usually predate the chemotherapy. Similarly, the HSPC phylogenetic tree of PD37580 indicates that at least two _PPM1D_ driver mutations arose before the chemotherapy given during childhood (Extended Data Fig. [7a](https://www.nature.com/articles/s41588-025-02234-x#Fig12)). Furthermore, in PD47703, a comparison of two samples taken 1 year apart, during which additional chemotherapy (cyclophosphamide, doxorubicin and vincristine) had been administered, revealed an approximately 50% increase in the size of preexisting _PPM1D_ mutated clones and no new mutant clones (Extended Data Fig. [7b](https://www.nature.com/articles/s41588-025-02234-x#Fig12)). Thus, chemotherapy-induced changes in selection appear more influential than chemotherapy-induced creation of new driver mutations in generating the prematurely aged HSPC profile.

The prematurely aged architecture of the HSPC population was not observed in two young adults (PD50308 aged 29 years and PD50307 aged 40 years) who received chemotherapy that caused substantial increases in mutation loads and was administered 2 years or less before sampling (Extended Data Fig. [8](https://www.nature.com/articles/s41588-025-02234-x#Fig13)). It was also not observed in two further individuals who were treated with cyclophosphamide and oxaliplatin (PD44579 aged 63 years and PD47537 aged 61 years) and exhibited minimally increased mutation loads (Extended Data Fig. [9](https://www.nature.com/articles/s41588-025-02234-x#Fig14)). Therefore, it is conceivable that multiple and/or prolonged chemotherapeutic exposures are required to generate the prematurely aged architecture. However, it is also possible that chemotherapy-engendered clonal expansions require decades to become detectable, as already demonstrated for clones under positive selection during normal aging[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."),[38](https://www.nature.com/articles/s41588-025-02234-x#ref-CR38 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022).").

Changes in clonal architecture resulting from chemotherapy exposure are relevant for two reasons: first, _PPM1D_ mutant clones may themselves reduce the regenerative ability of the bone marrow[21](https://www.nature.com/articles/s41588-025-02234-x#ref-CR21 "Bolton, K. L. et al. Cancer therapy shapes the fitness landscape of clonal hematopoiesis. Nat. Genet. 52, 1219–1226 (2020)."),[39](https://www.nature.com/articles/s41588-025-02234-x#ref-CR39 "Stelmach, P. et al. Clonal hematopoiesis with DNMT3A and PPM1D mutations impairs regeneration in autologous stem cell transplant recipients. Haematologica 108, 3308–3320 (2023)."), or the presence of _PPM1D_ clones may simply be a marker of a more general state of reduced hematopoietic stem cell (HSC) function after chemotherapy. One may speculate that the presence of many such clones had a role in the development of cytopenias and infections in PD47703 following autograft treatment. Second, the selection of _TP53_ mutant clones confers a high risk of developing secondary myeloid malignancies, including AML as seen in PD37580, whose disease was treatment-refractory and carried biallelic _TP53_ mutations and a complex karyotype.

Discussion
----------

This initial survey demonstrates that some commonly used chemotherapies, at dose regimens used in clinical practice, increase somatic mutation burdens and alter the population structure of normal blood cells. Individuals with elevated mutation burdens have likely experienced very high mutation rates over short periods. For example, an additional 1,000 SBSs acquired in an HSPC due to chemotherapy administered during the course of 1 year is equivalent to an approximately 50-fold increase in the average mutation rate over the year, and it is plausible that mutation rates within hours or days of chemotherapy are even higher. The additional long-term mutation loads are also sometimes considerable. A 3-year-old boy treated for neuroblastoma had more than tenfold the number of somatic SBSs expected for his age, exceeding the burden in normal 80-year-old individuals.

The additional mutation burdens differed substantially both between chemotherapy classes and between agents of the same class. As an important mechanism underlying the therapeutic effect of many chemotherapies is thought to be DNA damage induction, it is notable that different agents of the same class, at their therapeutic doses, have such different impacts on mutation generation in normal cells. The reasons for this are unclear but may reflect subtle differences between agents in the nature of the DNA damage caused, the repairability of the damage, the extent of induction of normal cell death and the levels of normal cell exposure. For example, cyclophosphamide is thought to relatively spare HSPCs due to their higher levels of aldehyde dehydrogenase, an enzyme that inactivates a cyclophosphamide intermediary[40](https://www.nature.com/articles/s41588-025-02234-x#ref-CR40 "Brodsky, R. A. High dose cyclophosphamide treatment for autoimmune disorders. Sci. World J. 2, 1808–1815 (2002)."). However, it may also be the case that the extent of DNA damage does not directly correlate with the level of cytotoxicity of some chemotherapies.

The additional mutation burdens caused by chemotherapies are characterized by distinct mutational signatures, often shared by agents of the same class. The signatures are similar to those induced by the same agents in cancer cells, suggesting that the patterns of induced DNA damage, and its processing into mutations through DNA repair and replication, are similar in normal and cancer cells, even if the tolerance of DNA damage by normal and cancer cells differs.

Increases in mutation loads imposed by chemotherapies differed between blood cell types, and the profile of differences between cell types differed between chemotherapeutic agents. The mechanisms underlying these complex landscapes are uncertain but may reflect intrinsic differences in the metabolic capabilities, DNA repair capacities and cell division rates of the different cell types.

Changes in hematopoietic clonal architecture characterized by increasing dominance of large clones, often with driver mutations in cancer genes, are a feature of normal aging. Chemotherapy caused a similar pattern of change prematurely, although with a different repertoire of mutated genes. However, these changes were not observed in all individuals. Whether these chemotherapy-induced changes in population architecture are contingent on the long duration and/or multiplicity of treatment, or simply occur with the passage of decades after treatment (which may allow clones with limited growth advantage under normal conditions to become detectable), requires further investigation.

This study has limitations. In addition to the small number of individuals, solid tumors were predominantly colorectal carcinoma, hematological cancers were predominantly of lymphoid origin, and not all mutagenic chemotherapies may have been included. We were unable to address the role of multiple other plausible factors influencing mutation burdens and selective effects, such as the pharmacokinetics of drug administration. We acknowledge that the in vitro culture of HSPCs may have introduced an unintended sampling bias, and future studies should address how the observed changes in the clonal composition of HSPCs following chemotherapy translate to differences in circulating mature blood cells.

Together with other work[17](https://www.nature.com/articles/s41588-025-02234-x#ref-CR17 "Pich, O. et al. The mutational footprints of cancer therapies. Nat. Genet. 51, 1732–1740 (2019)."),[32](https://www.nature.com/articles/s41588-025-02234-x#ref-CR32 "Bertrums, E. J. M. et al. Elevated mutational age in blood of children treated for cancer contributes to therapy-related myeloid neoplasms. Cancer Discov. 12, 1860–1872 (2022)."),[41](https://www.nature.com/articles/s41588-025-02234-x#ref-CR41 "Koh, G. C. C. et al. The chemotherapeutic drug CX-5461 is a potent mutagen in cultured human cells. Nat. Genet. 56, 23–26 (2024)."), our study points to a future agenda for systematic genomic analysis of normal tissues after chemotherapy. These studies could incorporate a wider range of at-risk normal tissues, sampled before and after treatment, after short and long periods, and across the full range of chemotherapies. Within the single-disease setting of clinical trials, this approach could aim to improve combination chemotherapeutic regimens and address dose–mutation relationships. Comprehensive prospective surveys of this nature would improve the understanding of the consequences of widespread, self-administered mutagenic exposure in human populations and provide a scientific basis for optimizing long-term patient health.

In conclusion, some chemotherapies impose additional mutational loads and change the cell population structure of normal blood. Both impacts plausibly contribute to long-term consequences, including second malignancies, infertility and loss of normal tissue resilience. Clinical data support this view, with the most mutagenic agents in this study having measurably greater long-term treatment toxicities. For example, of the bifunctional alkylating agents, melphalan and chlorambucil are associated with higher risks of secondary malignancies than cyclophosphamide[3](https://www.nature.com/articles/s41588-025-02234-x#ref-CR3 "Kollmannsberger, C., Hartmann, J. T., Kanz, L. & Bokemeyer, C. Risk of secondary myeloid leukemia and myelodysplastic syndrome following standard-dose chemotherapy or high-dose chemotherapy with stem cell support in patients with potentially curable malignancies. J. Cancer Res. Clin. Oncol. 124, 207–214 (1998)."),[42](https://www.nature.com/articles/s41588-025-02234-x#ref-CR42 "Curtis, R. E. et al. Risk of leukemia after chemotherapy and radiation treatment for breast cancer. N. Engl. J. Med. 326, 1745–1751 (1992)."),[43](https://www.nature.com/articles/s41588-025-02234-x#ref-CR43 "Kaldor, J. M. et al. Leukemia following Hodgkin’s disease. N. Engl. J. Med. 322, 7–13 (1990)."). In addition, procarbazine has been associated with a particularly high risk of second cancer and infertility and is, for this reason, no longer used in the treatment of pediatric Hodgkin lymphoma[44](https://www.nature.com/articles/s41588-025-02234-x#ref-CR44 "Mauz-Körholz, C. et al. Procarbazine-free OEPA–COPDAC chemotherapy in boys and standard OPPA–COPP in girls have comparable effectiveness in pediatric Hodgkin’s lymphoma: the GPOH-HD-2002 study. J. Clin. Oncol. 28, 3680–3686 (2010)."). Given that, in many cancer types, chemotherapeutic agents within a single class can be used interchangeably to achieve similar clinical outcomes[45](https://www.nature.com/articles/s41588-025-02234-x#ref-CR45 "Eichhorst, B. et al. First-line chemoimmunotherapy with bendamustine and rituximab versus fludarabine, cyclophosphamide, and rituximab in patients with advanced chronic lymphocytic leukaemia (CLL10): an international, open-label, randomised, phase 3, non-inferiority trial. Lancet Oncol. 17, 928–942 (2016)."),[46](https://www.nature.com/articles/s41588-025-02234-x#ref-CR46 "Lacout, C. et al. R–DHA–oxaliplatin (R-DHAOx) versus R–DHA–cisplatin (R-DHAP) regimen in B-cell lymphoma treatment: a eight-year trajectory study. Eur. J. Haematol. 105, 223–230 (2020)."),[47](https://www.nature.com/articles/s41588-025-02234-x#ref-CR47 "Mehmood, R. K. Review of cisplatin and oxaliplatin in current immunogenic and monoclonal antibody treatments. Oncol. Rev. 8, 256 (2014)."), it may be possible to prospectively use these types of data when improving existing regimens or developing new treatment protocols. In patients previously exposed to chemotherapy, knowledge of their altered mutational and clonal landscape could also prompt discussions as to suitability for standard treatment protocols, particularly in the autologous transplant setting, and allow exploring alternative options where appropriate.

Methods
-------

### Ethical regulation compliance

Our research complies with all relevant ethical regulations as approved by the National Health Service (NHS) Cambridgeshire 4 Research Ethics Committee, Cambridge East Ethics Committee and the Research Ethics Committee of the University of Texas MD Anderson Cancer Center Institutional Review Board (see additional information below).

### Statistics and reproducibility

No statistical methods were used to predetermine the sample size. We removed a total of 96 colonies from the dataset of 931 previously unpublished colonies: 32 for being technical duplicates, 29 for showing evidence of nonclonality or contamination and 23 due to low coverage. The experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment.

### Samples

Blood or bone marrow samples from individuals unexposed to chemotherapy were obtained from three sources: (1) STEMCELL Technologies provided frozen mononuclear cells (MNCs) for the cord blood sample that had been collected with written informed consent, including for WGS (catalogue number 70007); all data were previously published. (2) Cambridge Blood and Stem Cell Biobank provided fresh peripheral blood samples taken with written informed consent from two patients at Addenbrooke’s Hospital (NHS Cambridgeshire 4 Research Ethics Committee reference 07/MRE05/44 for samples collected before November 2019 and Cambridge East Ethics Committee reference 18/EE/0199 for samples collected from November 2019 onward); all data were previously published. (3) Cambridge Biorepository for Translational Medicine provided frozen bone marrow samples with or without peripheral blood MNCs taken with informed consent from seven deceased organ donors. Samples were obtained at the time of abdominal organ collection (Cambridgeshire 4 Research Ethics Committee reference 15/EE/0152); data were previously published from four individuals, with new data generated from an additional two individuals (PD49236 and PD49327).

Blood samples from individuals previously exposed to chemotherapy were obtained from two sources: (1) Cambridge Blood and Stem Cell Biobank provided fresh peripheral blood samples taken with written informed consent from 22 patients at Addenbrooke’s Hospital (NHS Cambridgeshire 4 Research Ethics Committee reference 07/MRE05/44 for samples collected before November 2019 and Cambridge East Ethics Committee reference 18/EE/0199 for samples collected from November 2019 onward); all data were unpublished. One chemotherapy-exposed individual, PD47703, had two samples taken at time points a year apart. All others were sampled at a single time point. (2) Baylor College of Medicine provided single-cell-derived hematopoietic colonies from the bone marrow taken following written informed consent from one patient from MD Anderson Cancer Center (Research Ethics Committee of the University of Texas MD Anderson Cancer Center Institutional Review Board reference PA12-0305 (genomic analysis protocol) and LAB01-473 (laboratory protocol)).

Details of the individuals studied and the samples they provided are listed in Fig. [1a](https://www.nature.com/articles/s41588-025-02234-x#Fig1), with additional information in Supplementary Table [1](https://www.nature.com/articles/s41588-025-02234-x#MOESM4). All participants provided written informed consent to have their anonymized details published. Participants did not receive compensation for taking part in the study.

### Isolation of MNCs from fresh peripheral blood samples

Whole blood was diluted 1:1 with PBS, after which MNCs were isolated using Lymphoprep density gradient centrifugation (STEMCELL Technologies). Red cell lysis was performed on the MNC fraction using one incubation at 4 °C for 15 min with red blood cell lysis buffer (BioLegend).

### Single-cell colony expansion in vitro—liquid culture (unexposed samples)

For all the unexposed normal samples and the PD47703 second-time-point sample, single-cell colony expansion in vitro was undertaken in liquid culture, exactly as previously described[1](https://www.nature.com/articles/s41588-025-02234-x#ref-CR1 "DeVita, V. T. Jr. & Chu, E. A history of cancer chemotherapy. Cancer Res. 68, 8643–8653 (2008).").

Peripheral blood and cord blood MNC samples underwent CD34+ cell selection using the EasySep human whole blood CD34-positive selection kit (STEMCELL Technologies), with only a single round of magnetic selection. Bone marrow MNCs were not selected for CD34+ cells before sorting.

MNC or CD34-enriched samples were stained (30 min at 4 °C) in PBS/3% FBS containing the following antibodies: CD3 FITC (1 in 500), CD90 PE (1 in 50), CD49f PE-Cy5 (1 in 100), CD38 PE-Cy7 (1 in 100), CD19 A700 (1 in 300), CD34 APC-Cy7 (1 in 100), CD45RA BV421 (1 in 100) and Zombie Aqua (1 in 2,000) (Supplementary Table [8](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). Cells were then washed and resuspended in PBS/3% FBS for cell sorting. Either a BD Aria III or BD Aria Fusion cell sorter (BD Biosciences) was used to sort ‘HSC/MPP’ pool cells (Lin−, CD34+, CD38−, CD45RA−) at the National Institute for Health and Care Research (NIHR) Cambridge Biomedical Research Centre (BRC) Cell Phenotyping Hub. Supplementary Fig. [1](https://www.nature.com/articles/s41588-025-02234-x#MOESM1) illustrates the gating strategy used.

Single phenotypic HSPCs were index-sorted into single wells of 96-well plates containing StemPro medium (STEMCELL Technologies), StemPro Nutrients (0.035%, STEMCELL Technologies), l\-glutamine (1%, ThermoFisher), penicillin–streptomycin (1%, ThermoFisher) and cytokines (stem cell factor, 100 ng ml−1; FLT3, 20 ng ml−1; thrombopoietin, 100 ng ml−1; erythropoietin, 3 ng ml−1; interleukin-6 (IL-6), 50 ng ml−1; IL-3, 10 ng ml−1; IL-11, 50 ng ml−1; granulocyte–macrophage colony-stimulating factor, 20 ng ml−1; IL-2, 10 ng ml−1; IL-7, 20 ng ml−1; lipids, 50 ng ml−1) and expanded into colonies. Cells were incubated at 37 °C, and the colonies that formed were topped up with 50 μl StemPro medium plus supplements at 14 ± 2 days as necessary. At 21 ± 2 days, colonies were collected. DNA extraction was performed using either the DNEasy 96 blood and tissue plate kit (Qiagen) or the Arcturus PicoPure DNA extraction kit (ThermoFisher) per the manufacturer’s instructions.

### Single-cell colony expansion in vitro—MethoCult (chemotherapy-exposed samples)

For the chemotherapy-exposed peripheral blood samples, single-cell colonies were expanded in MethoCult (H4435 or H4034, STEMCELL Technologies). MNCs were plated at a density of 7.5–45 × 104 cells per ml in MethoCult and incubated at 37 °C for 14 days. The cell suspensions were made up in StemSpan II (STEMCELL Technologies) before being mixed thoroughly with MethoCult and plated into a SmartDish (STEMCELL Technologies). Individual BFU-E (burst-forming unit erythroid) or CFU-GM (colony-forming unit granulocyte–macrophage) colonies were picked, added to 17 μl proteinase K (PicoPure DNA extraction kit, Fisher Scientific; with each vial of lyophilized proteinase K resuspended in 130 μl reconstitution buffer), and incubated at 65 °C for 6 h and at 75 °C for 30 min to extract DNA in preparation for sequencing.

Previous studies have shown that there is no difference in mutation burden between HSCs and hematopoietic progenitor cells[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).") and that there is a mutation burden difference of only approximately 30 SBS mutations between HSCs and mature granulocytes[25](https://www.nature.com/articles/s41588-025-02234-x#ref-CR25 "Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. Nature 593, 405–410 (2021).").

### WGS of colonies

WGS libraries were prepared from 1–5 ng of extracted DNA from each colony using a low-input enzymatic fragmentation-based library preparation method[48](https://www.nature.com/articles/s41588-025-02234-x#ref-CR48 "Ellis, P. et al. Reliable detection of somatic mutations in solid tissues by laser-capture microdissection and low-input DNA sequencing. Nat. Protoc. 16, 841–871 (2021)."),[49](https://www.nature.com/articles/s41588-025-02234-x#ref-CR49 "Moore, L. et al. The mutational landscape of normal human endometrial epithelium. Nature 580, 640–646 (2020)."). WGS was performed on the NovaSeq platform (Illumina). Paired-end reads of 150 bp were aligned to the human reference genome (National Center for Biotechnology Information build 37) using BWA-MEM.

### SBS, indel, structural variant and copy number variant calling

The method for substitution calling involved three main steps: mutation discovery, filtering and genotyping, as described previously[19](https://www.nature.com/articles/s41588-025-02234-x#ref-CR19 "Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019).") ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02234-x#MOESM1)).

### Additional variant filtering steps

#### Larger dataset containing samples sequenced at a lower sequencing depth

For the creation of the larger phylogenies (a subset of six chemotherapy-exposed and five unexposed individuals with >40 sequenced colonies), a binomial filtering strategy could be applied as previously described[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).") ([https://github.com/emily-mitchell/normal\_haematopoiesis/2\_variant\_filtering\_tree\_building/scripts/](https://github.com/emily-mitchell/normal_haematopoiesis/2_variant_filtering_tree_building/scripts/d)).

#### Small dataset containing samples sequenced at a relatively high sequencing depth

For the subset of colonies sequenced at the highest depth (four to ten colonies per individual), we were unable to use binomial filtering approaches due to the low sample number. Instead, the following filters were applied to the data using a custom R script: (1) variants present in more than half the samples from an individual were removed as being most likely germline. (2) Variants were only called as present in a given sample if they had two or more supporting reads and were present at a variant allele fraction (VAF) of ≥0.2 in autosomes or ≥0.4 for sex chromosomes. (3) High- and low-depth sites with a mean depth of >50 or <8 across all samples from an individual were removed.

This variant filtering approach was validated using samples from the normal individuals, in whom both the binomial and nonbinomial filtering strategies were applied to the same samples, giving comparable results (Supplementary Fig. [2](https://www.nature.com/articles/s41588-025-02234-x#MOESM1)). This dataset, comparable across all the individuals in the study, was used for the analysis of SBS and indel mutation burdens.

### Filtering at the colony level

We removed a total of 96 colonies from the dataset of 931 previously unpublished colonies: 32 for being technical duplicates, 29 for showing evidence of nonclonality or contamination and 23 due to low coverage. Visual inspection of the VAF distribution plots was performed, and a peak VAF threshold of <0.4 was used (after the removal of in vitro variants) to identify colonies with evidence of nonclonality.

### Mutation burden analysis

Due to the difficulty in correcting for sequencing depth when only a small number of samples are sequenced per individual, SBS and indel burden analysis was performed on raw data from the subset of chemotherapy-exposed and unexposed samples sequenced at a relatively high depth (four to ten samples per individual; mean coverage 23×, range 13–33×). We have previously shown that sequencing depth has little impact on the SBS mutation burden over this higher range[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022)."). There were minor differences in sequencing depth when comparing the chemotherapy and normal cohorts or when comparing sequencing depth by chemotherapy exposure, but these would not be expected to affect the interpretation of the results presented (Extended Data Fig. [1](https://www.nature.com/articles/s41588-025-02234-x#Fig6)).

Given the known influence of age on mutation burden, we built a linear mixed-effect model to quantify how age and chemotherapy treatment influence mutation burdens. Patient ID (PDID) was added as a random effect to the model, which was as follows:

$\\begin{array}{lll}{\\rm{glmer}}\\\_{\\rm{chemo}} < -\\, {\\rm{glmer}}\\left(\\right.{\\rm{round}}({\\rm{Number}}\\\_{\\rm{mutations}}) \\sim {\\rm{Age}}+{\\rm{Exposure}}\\\\+\\,(1|{\\rm{PDID}}),\\,{\\rm{data}}={\\rm{Summary}}\\\_{\\rm{All}},\\,{\\rm{family}}={\\rm{poisson}}({\\rm{link}}={\\text{\`\`}} {\\rm{identity}}{\\text{''}})\\end{array}$

### Construction of phylogenetic trees

MPBoot, a maximum parsimony tree approximation method[50](https://www.nature.com/articles/s41588-025-02234-x#ref-CR50 "Hoang, D. T. et al. MPBoot: fast phylogenetic maximum parsimony tree inference and bootstrap approximation. BMC Evol. Biol. 
https://doi.org/10.1186/s12862-018-1131-3
(2018)."), was used to build and annotate phylogenetic trees of the relationships between the sampled HSPCs, as previously described[24](https://www.nature.com/articles/s41588-025-02234-x#ref-CR24 "Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Nature 606, 343–350 (2022).") ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02234-x#MOESM1)).

The key steps to generate the phylogenies shown in Fig. [5](https://www.nature.com/articles/s41588-025-02234-x#Fig5) and Extended Data Figs. [8](https://www.nature.com/articles/s41588-025-02234-x#Fig13) and [9](https://www.nature.com/articles/s41588-025-02234-x#Fig14) are as follows:

1.  1.
    
    Generate a ‘genotype matrix’ of mutation calls for every colony within a donor. Our protocol, based on WGS of single-cell-derived colonies, generates consistent and even coverage across the genome, leading to very few missing values within this matrix (ranging from 0.005 to 0.034 of mutated sites in a given colony across different donors within our cohort). This generates a high degree of accuracy in the constructed trees.
    
2.  2.
    
    Reconstruct phylogenetic trees from the genotype matrix. This is a standard and well-studied problem in phylogenetics. The low fraction of the genome that is mutated in a given colony (<1 per million bases), coupled with the highly complete genotype matrix, means that different phylogenetic methods produce reassuringly concordant trees. We used the MPBoot algorithm for the tree reconstruction, as it proved both accurate and computationally efficient for our dataset.
    
3.  3.
    
    Correct terminal branch lengths for sensitivity to detect mutations in each colony. The trees generated in the previous step have branch lengths proportional to the number of mutations assigned to each branch. For the terminal branches, which contain mutations unique to that colony, variable sequencing depth can underestimate the true numbers of unique mutations, so we correct these branch lengths for the estimated sensitivity to detect mutations based on genome coverage.
    
4.  4.
    
    Make phylogenetic trees ultrametric. After step 3, there is little more than Poisson variation in the corrected mutation burden among colonies from a given donor. As these colonies were all derived from the same time point, we can normalize the branch lengths to have the same overall distance from root to tip (known as an ultrametric tree). We used an ‘iteratively reweighted means’ algorithm for this purpose.
    
5.  5.
    
    Scale trees to chronological age. As the mutation rate is constant across the human lifespan, we can use it as a ‘molecular clock’ to linearly scale the ultrametric tree to chronological age.
    
6.  6.
    
    Overlay phenotypic and genotypic information on the tree.
    

Overlay phenotypic and genotypic information on the tree. The tip of each branch in the resulting phylogenetic tree represents a specific colony in the dataset, meaning that we can depict phenotypic information about each colony underneath its terminal branch (the colored stripes along the bottom of Fig. [5](https://www.nature.com/articles/s41588-025-02234-x#Fig5) and Extended Data Figs. [8](https://www.nature.com/articles/s41588-025-02234-x#Fig13) and [9](https://www.nature.com/articles/s41588-025-02234-x#Fig14)). Furthermore, every mutation in the dataset is confidently assigned to a specific branch in the phylogenetic tree. This means that we can highlight branches on which specific genetic events occurred (such as _DNMT3A_ or other driver mutations).

To estimate the number of somatic mutations that may have already been acquired by PD37580 by age 13 years (before commencing chlorambucil), we used the linear mixed model defined by Mitchell et al. This model estimates an intercept of 54.57 (that is, the mean number of somatic mutations present at birth), with a slope of 16.832 representing the mean number of somatic mutations acquired each year of life. This results in an expected mean somatic mutation burden of 273 at age 13 years. Assuming that this mutation burden is Poisson distributed provides a 95% prediction interval of 241–306.

### Analysis of driver variants

The variants identified were annotated with VAGrENT (Variation Annotation GENeraTor) ([https://github.com/cancerit/VAGrENT](https://github.com/cancerit/VAGrENT)) to identify protein-coding mutations and putative driver mutations in each dataset. Supplementary Table [9](https://www.nature.com/articles/s41588-025-02234-x#MOESM4) lists the 18 genes we have used as our top clonal hematopoiesis genes (those identified in ref. [38](https://www.nature.com/articles/s41588-025-02234-x#ref-CR38 "Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. Nature 606, 335–342 (2022).") as being under positive selection in a targeted sequencing dataset of 385 older individuals, with _CHEK2_ added as being an additional gene commonly under positive selection in chemotherapy-exposed individuals). ‘Oncogenic’ mutations (as assessed by E.M.) are shown in Fig. [5](https://www.nature.com/articles/s41588-025-02234-x#Fig5) and Extended Data Figs. [8](https://www.nature.com/articles/s41588-025-02234-x#Fig13) and [9](https://www.nature.com/articles/s41588-025-02234-x#Fig14).

### Bulk cell sorts for NanoSeq sequencing

#### Chemotherapy-exposed samples

MNCs were stained for 30 min at 4 °C in PBS/3% FCS containing the following antibodies: Zombie Aqua (1 in 400), CD3 APC (1 in 80), CD19 AF700 (1 in 80), CD45RA PerCP-Cy5.5 (1 in 80), CCR7 BV711 (1 in 80) and CD14 BV605 (1 in 80). Cells were then washed and resuspended in PBS/3% FBS for sorting. Either a BD Aria III or BD Aria Fusion cell sorter (BD Biosciences) was used to sort various mature cell compartments (B cells, T naive cells, T memory cells and monocytes) at the NIHR Cambridge BRC Cell Phenotyping Hub. For each cell type, approximately 40,000 cells were sorted into Eppendorf tubes containing 50 μl PBS. Further details are provided in Supplementary Table [10](https://www.nature.com/articles/s41588-025-02234-x#MOESM4) and Supplementary Fig. [3](https://www.nature.com/articles/s41588-025-02234-x#MOESM1).

#### Unexposed normal samples

MNCs were stained for 30 min at 4 °C in PBS/3% FCS containing the following antibodies: CD3 APC (1 in 80), CD4 BV785 (1 in 80), CD8 BV785 (1 in 40), CD14 BV605 (1 in 80), CD19 AF700 (1 in 80), CD20 PE-Dazzle (1 in 80), CD27 BV421 (1 in 80), CD34 APC-Cy7 (1 in 27), CD38 FITC (1 in 80), CD45RA PerCP-Cy5.5 (1 in 80), CD56 PE (1 in 80), CCR7 BV711 (1 in 80), IgD PE-Cy7 (1 in 100) and Zombie Aqua (1 in 400). Cells were then washed and resuspended in PBS/3% FBS for sorting. Either a BD Aria III or BD Aria Fusion cell sorter (BD Biosciences) was used to sort various mature cell compartments (B cells, CD4+ T naive cells, CD4+ T memory cells, CD8+ T naive cells, CD8+ T memory cells and monocytes) at the NIHR Cambridge BRC Cell Phenotyping Hub. For each cell type, approximately 40,000 cells were sorted into Eppendorf tubes containing 50 μl PBS. Further details are provided in Supplementary Table [11](https://www.nature.com/articles/s41588-025-02234-x#MOESM4) and Supplementary Fig. [4](https://www.nature.com/articles/s41588-025-02234-x#MOESM1).

### DNA extraction from bulk mature cell sorts

Approximately 40,000 cells of each mature cell type from the above sorts (suspended in 200 μl PBS) were added to single wells of a 96-well PCR plate and centrifuged. Pellets were resuspended in 17 μl proteinase K (PicoPure DNA extraction kit, Fisher Scientific; with each vial of lyophilized proteinase K resuspended in 130 μl reconstitution buffer) and incubated at 65 °C for 6 h and at 75 °C for 30 min to extract DNA in preparation for sequencing.

### NanoSeq (duplex) sequencing

Extracted DNA (1–5 ng) from bulk cell sorts was submitted to the NanoSeq pipeline for library preparation and sequencing, as has been described previously[25](https://www.nature.com/articles/s41588-025-02234-x#ref-CR25 "Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. Nature 593, 405–410 (2021).") ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02234-x#MOESM1)).

### Mutational signature analysis

The HDP ([https://github.com/nicolaroberts/hdp](https://github.com/nicolaroberts/hdp)), based on the Bayesian HDP, was used to extract mutational signatures. The HDP was run without priors on SBSs derived from phylogenetic trees obtained from HSPCs and mutations from NanoSeq samples. The NanoSeq mutations were corrected for the trinucleotide context abundance for each sample ([Supplementary Methods](https://www.nature.com/articles/s41588-025-02234-x#MOESM1)).

The COSMIC (v3.4)[27](https://www.nature.com/articles/s41588-025-02234-x#ref-CR27 "Tate, J. G. et al. COSMIC: the catalogue of somatic mutations in cancer. Nucleic Acids Res. 47, D941–D947 (2019).") signatures identified were SBS1, SBS5, SBS7a, SBS9 and SBS17. One of the HDP components corresponded to the SBSBlood signature previously reported[30](https://www.nature.com/articles/s41588-025-02234-x#ref-CR30 "Machado, H. E. et al. Diverse mutational landscapes in human lymphocytes. Nature 608, 724–732 (2022)."). Eight components were de novo signatures called predominantly in individuals with chemotherapy exposure. Only signatures with a contribution of >5% of the mutations of the sample’s burden were considered.

Eight chemotherapy-related signatures derived from the HDP (SBSA–SBSH) were examined for their occurrence in exposed versus nonexposed individuals. The proportions of the signatures contributing to the samples in the exposed versus nonexposed group were compared using the _t_ test for independent samples, with an equal variance assumption. The test was performed in two ways: (1) per cell type (Supplementary Table [5](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)) and (2) by combining the samples across cell types for each individual (Supplementary Table [6](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)). The _P_ values were adjusted for multiple testing corrections using the false discovery rate and Bonferroni methods.

Indel signatures were extracted from small indels called from the HSPC dataset using two different methods. First, mSigHdp was run, identifying three distinct indel signatures ([https://github.com/steverozen/mSigHdp](https://github.com/steverozen/mSigHdp))[51](https://www.nature.com/articles/s41588-025-02234-x#ref-CR51 "Liu, M., Wu, Y., Jiang, N., Boot, A. & Rozen, S. G. mSigHdp: hierarchical Dirichlet process mixture modeling for mutational signature discovery. NAR Genom. Bioinform. 5, lqad005 (2023)."). To investigate whether the extracted de novo signatures were composed of reference COSMIC signatures, we used the SigProfilerAssignment decompose tool ([https://github.com/AlexandrovLab/SigProfilerAssignment](https://github.com/AlexandrovLab/SigProfilerAssignment))[52](https://www.nature.com/articles/s41588-025-02234-x#ref-CR52 "Díaz-Gay, M. et al. Assigning mutational signatures to individual samples and individual somatic mutations with SigProfilerAssignment. Bioinformatics 39, btad756 (2023)."). Two signatures were successfully decomposed, the first into ID1 and ID2 with a reconstructed cosine similarity of 0.99 (compared to the de novo signature). The second signature was decomposed into ID3, ID5 and ID9 with a reconstructed cosine similarity of 0.93. The final signature, IDA, was initially decomposed into ID2 and ID18 with a reconstructed cosine similarity of 0.88. ID18 is a signature associated with colibactin exposure[19](https://www.nature.com/articles/s41588-025-02234-x#ref-CR19 "Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019)."), which is unlikely in this context. This, in combination with the lower cosine similarity and strong support from the mutation spectra of individuals treated with procarbazine, led to this decomposition being rejected.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://www.nature.com/articles/s41588-025-02234-x#MOESM2) linked to this article.

Data availability
-----------------

Smaller derived datasets needed to perform mutation burden and mutational signature analysis are available on GitHub ([https://github.com/emily-mitchell/chemotherapy/](https://github.com/emily-mitchell/chemotherapy/)) and Zenodo ([https://doi.org/10.5281/zenodo.15235476](https://doi.org/10.5281/zenodo.15235476))[53](https://www.nature.com/articles/s41588-025-02234-x#ref-CR53 "Mitchell, E. Emily-mitchell/chemotherapy: the long-term effects of chemotherapy on normal blood cells. Zenodo 
https://doi.org/10.5281/zenodo.15235476
(2025)."). Raw sequencing data are available on the European Genome–phenome Archive (accession numbers [EGAD00001015339](https://ega-archive.org/datasets/EGAD00001015339) (WGS dataset) and [EGAD00001015340](https://ega-archive.org/datasets/EGAD00001015340) (NanoSeq dataset)). The main data needed to reanalyze and reproduce the results presented are available on Mendeley Data ([https://doi.org/10.17632/2fczcd49yj.1](https://doi.org/10.17632/2fczcd49yj.1)). [Source data](https://www.nature.com/articles/s41588-025-02234-x#Sec33) are provided with this paper.

Code availability
-----------------

Code and smaller derived datasets are available on GitHub ([https://github.com/emily-mitchell/chemotherapy/](https://github.com/emily-mitchell/chemotherapy/)) and Zenodo ([https://doi.org/10.5281/zenodo.15235476](https://doi.org/10.5281/zenodo.15235476))[53](https://www.nature.com/articles/s41588-025-02234-x#ref-CR53 "Mitchell, E. Emily-mitchell/chemotherapy: the long-term effects of chemotherapy on normal blood cells. Zenodo 
https://doi.org/10.5281/zenodo.15235476
(2025).").

Change history
--------------

-   ### 31 July 2025
    
    A Correction to this paper has been published: [https://doi.org/10.1038/s41588-025-02315-x](https://doi.org/10.1038/s41588-025-02315-x)
    

References
----------

1.  DeVita, V. T. Jr. & Chu, E. A history of cancer chemotherapy. _Cancer Res._ **68**, 8643–8653 (2008).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlSktbbJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20history%20of%20cancer%20chemotherapy&journal=Cancer%20Res.&volume=68&pages=8643-8653&publication_year=2008&author=DeVita%2CVT&author=Chu%2CE) 
    
2.  Tilsed, C. M., Fisher, S. A., Nowak, A. K., Lake, R. A. & Lesterhuis, W. J. Cancer chemotherapy: insights into cellular and tumor microenvironmental mechanisms of action. _Front. Oncol._ **12**, 960317 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVWmsr%2FK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cancer%20chemotherapy%3A%20insights%20into%20cellular%20and%20tumor%20microenvironmental%20mechanisms%20of%20action&journal=Front.%20Oncol.&volume=12&publication_year=2022&author=Tilsed%2CCM&author=Fisher%2CSA&author=Nowak%2CAK&author=Lake%2CRA&author=Lesterhuis%2CWJ) 
    
3.  Kollmannsberger, C., Hartmann, J. T., Kanz, L. & Bokemeyer, C. Risk of secondary myeloid leukemia and myelodysplastic syndrome following standard-dose chemotherapy or high-dose chemotherapy with stem cell support in patients with potentially curable malignancies. _J. Cancer Res. Clin. Oncol._ **124**, 207–214 (1998).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXlslOisrw%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Risk%20of%20secondary%20myeloid%20leukemia%20and%20myelodysplastic%20syndrome%20following%20standard-dose%20chemotherapy%20or%20high-dose%20chemotherapy%20with%20stem%20cell%20support%20in%20patients%20with%20potentially%20curable%20malignancies&journal=J.%20Cancer%20Res.%20Clin.%20Oncol.&volume=124&pages=207-214&publication_year=1998&author=Kollmannsberger%2CC&author=Hartmann%2CJT&author=Kanz%2CL&author=Bokemeyer%2CC) 
    
4.  Bhatia, S. Therapy-related myelodysplasia and acute myeloid leukemia. _Semin. Oncol._ **40**, 666–675 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmtFan)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Therapy-related%20myelodysplasia%20and%20acute%20myeloid%20leukemia&journal=Semin.%20Oncol.&volume=40&pages=666-675&publication_year=2013&author=Bhatia%2CS) 
    
5.  Morton, L. M. et al. Association of chemotherapy for solid tumors with development of therapy-related myelodysplastic syndrome or acute myeloid leukemia in the modern era. _JAMA Oncol._ **5**, 318–325 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20of%20chemotherapy%20for%20solid%20tumors%20with%20development%20of%20therapy-related%20myelodysplastic%20syndrome%20or%20acute%20myeloid%20leukemia%20in%20the%20modern%20era&journal=JAMA%20Oncol.&volume=5&pages=318-325&publication_year=2019&author=Morton%2CLM) 
    
6.  Bhatia, S. & Sklar, C. Second cancers in survivors of childhood cancer. _Nat. Rev. Cancer_ **2**, 124–132 (2002).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Second%20cancers%20in%20survivors%20of%20childhood%20cancer&journal=Nat.%20Rev.%20Cancer&volume=2&pages=124-132&publication_year=2002&author=Bhatia%2CS&author=Sklar%2CC) 
    
7.  Travis, L. B. The epidemiology of second primary cancers. _Cancer Epidemiol. Biomarkers Prev._ **15**, 2020–2026 (2006).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20epidemiology%20of%20second%20primary%20cancers&journal=Cancer%20Epidemiol.%20Biomarkers%20Prev.&volume=15&pages=2020-2026&publication_year=2006&author=Travis%2CLB) 
    
8.  Allan, J. M. & Travis, L. B. Mechanisms of therapy-related carcinogenesis. _Nat. Rev. Cancer_ **5**, 943–955 (2005).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ykurbJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mechanisms%20of%20therapy-related%20carcinogenesis&journal=Nat.%20Rev.%20Cancer&volume=5&pages=943-955&publication_year=2005&author=Allan%2CJM&author=Travis%2CLB) 
    
9.  Nicolaysen, A. Nephrotoxic chemotherapy agents: old and new. _Adv. Chronic Kidney Dis._ **27**, 38–49 (2020).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nephrotoxic%20chemotherapy%20agents%3A%20old%20and%20new&journal=Adv.%20Chronic%20Kidney%20Dis.&volume=27&pages=38-49&publication_year=2020&author=Nicolaysen%2CA) 
    
10.  Groopman, J. E. & Itri, L. M. Chemotherapy-induced anemia in adults: incidence and treatment. _J. Natl Cancer Inst._ **91**, 1616–1634 (1999).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmvVyntbc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chemotherapy-induced%20anemia%20in%20adults%3A%20incidence%20and%20treatment&journal=J.%20Natl%20Cancer%20Inst.&volume=91&pages=1616-1634&publication_year=1999&author=Groopman%2CJE&author=Itri%2CLM) 
    
11.  Crawford, J., Dale, D. C. & Lyman, G. H. Chemotherapy-induced neutropenia: risks, consequences, and new directions for its management. _Cancer_ **100**, 228–237 (2004).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chemotherapy-induced%20neutropenia%3A%20risks%2C%20consequences%2C%20and%20new%20directions%20for%20its%20management&journal=Cancer&volume=100&pages=228-237&publication_year=2004&author=Crawford%2CJ&author=Dale%2CDC&author=Lyman%2CGH) 
    
12.  Szász, R., Telek, B. & Illés, Á. Fludarabine–cyclophosphamide–rituximab treatment in chronic lymphocytic leukemia, focusing on long term cytopenias before and after the era of targeted therapies. _Pathol. Oncol. Res._ **27**, 1609742 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fludarabine%E2%80%93cyclophosphamide%E2%80%93rituximab%20treatment%20in%20chronic%20lymphocytic%20leukemia%2C%20focusing%20on%20long%20term%20cytopenias%20before%20and%20after%20the%20era%20of%20targeted%20therapies&journal=Pathol.%20Oncol.%20Res.&volume=27&publication_year=2021&author=Sz%C3%A1sz%2CR&author=Telek%2CB&author=Ill%C3%A9s%2C%C3%81) 
    
13.  Robison, L. L. & Hudson, M. M. Survivors of childhood and adolescent cancer: life-long risks and responsibilities. _Nat. Rev. Cancer_ **14**, 61–70 (2014).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVOiu7rJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Survivors%20of%20childhood%20and%20adolescent%20cancer%3A%20life-long%20risks%20and%20responsibilities&journal=Nat.%20Rev.%20Cancer&volume=14&pages=61-70&publication_year=2014&author=Robison%2CLL&author=Hudson%2CMM) 
    
14.  Erdmann, F. et al. Childhood cancer: survival, treatment modalities, late effects and improvements over time. _Cancer Epidemiol._ **71**, 101733 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Childhood%20cancer%3A%20survival%2C%20treatment%20modalities%2C%20late%20effects%20and%20improvements%20over%20time&journal=Cancer%20Epidemiol.&volume=71&publication_year=2021&author=Erdmann%2CF) 
    
15.  Alexandrov, L. B. et al. Signatures of mutational processes in human cancer. _Nature_ **500**, 415–421 (2013).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtlWjur7M)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Signatures%20of%20mutational%20processes%20in%20human%20cancer&journal=Nature&volume=500&pages=415-421&publication_year=2013&author=Alexandrov%2CLB) 
    
16.  Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. _Nature_ **578**, 94–101 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL0%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&journal=Nature&volume=578&pages=94-101&publication_year=2020&author=Alexandrov%2CLB) 
    
17.  Pich, O. et al. The mutational footprints of cancer therapies. _Nat. Genet._ **51**, 1732–1740 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOru7nJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20footprints%20of%20cancer%20therapies&journal=Nat.%20Genet.&volume=51&pages=1732-1740&publication_year=2019&author=Pich%2CO) 
    
18.  de Kanter, J. K. et al. Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients. _Cell Stem Cell_ **28**, 1726–1739 (2021).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Antiviral%20treatment%20causes%20a%20unique%20mutational%20signature%20in%20cancers%20of%20transplantation%20recipients&journal=Cell%20Stem%20Cell&volume=28&pages=1726-1739&publication_year=2021&author=Kanter%2CJK) 
    
19.  Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. _Nature_ **574**, 532–537 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3J)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20landscape%20of%20somatic%20mutation%20in%20normal%20colorectal%20epithelial%20cells&journal=Nature&volume=574&pages=532-537&publication_year=2019&author=Lee-Six%2CH) 
    
20.  Kaplanis, J. et al. Genetic and chemotherapeutic influences on germline hypermutation. _Nature_ **605**, 503–508 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1Krsb7L)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20and%20chemotherapeutic%20influences%20on%20germline%20hypermutation&journal=Nature&volume=605&pages=503-508&publication_year=2022&author=Kaplanis%2CJ) 
    
21.  Bolton, K. L. et al. Cancer therapy shapes the fitness landscape of clonal hematopoiesis. _Nat. Genet._ **52**, 1219–1226 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFKlu73E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cancer%20therapy%20shapes%20the%20fitness%20landscape%20of%20clonal%20hematopoiesis&journal=Nat.%20Genet.&volume=52&pages=1219-1226&publication_year=2020&author=Bolton%2CKL) 
    
22.  Hsu, J. I. et al. _PPM1D_ mutations drive clonal hematopoiesis in response to cytotoxic chemotherapy. _Cell Stem Cell_ **23**, 700–713 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVyhur%2FP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PPM1D%20mutations%20drive%20clonal%20hematopoiesis%20in%20response%20to%20cytotoxic%20chemotherapy&journal=Cell%20Stem%20Cell&volume=23&pages=700-713&publication_year=2018&author=Hsu%2CJI) 
    
23.  Coombs, C. C. et al. Therapy-related clonal hematopoiesis in patients with non-hematologic cancers is common and associated with adverse clinical outcomes. _Cell Stem Cell_ **21**, 374–382 (2017).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlWksb7E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Therapy-related%20clonal%20hematopoiesis%20in%20patients%20with%20non-hematologic%20cancers%20is%20common%20and%20associated%20with%20adverse%20clinical%20outcomes&journal=Cell%20Stem%20Cell&volume=21&pages=374-382&publication_year=2017&author=Coombs%2CCC) 
    
24.  Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. _Nature_ **606**, 343–350 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVant7vO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20dynamics%20of%20haematopoiesis%20across%20the%20human%20lifespan&journal=Nature&volume=606&pages=343-350&publication_year=2022&author=Mitchell%2CE) 
    
25.  Abascal, F. et al. Somatic mutation landscapes at single-molecule resolution. _Nature_ **593**, 405–410 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVSrur7F)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutation%20landscapes%20at%20single-molecule%20resolution&journal=Nature&volume=593&pages=405-410&publication_year=2021&author=Abascal%2CF) 
    
26.  Zhang, W., Gou, P., Dupret, J.-M., Chomienne, C. & Rodrigues-Lima, F. Etoposide, an anticancer drug involved in therapy-related secondary leukemia: enzymes at play. _Transl. Oncol._ **14**, 101169 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVGhtLvO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Etoposide%2C%20an%20anticancer%20drug%20involved%20in%20therapy-related%20secondary%20leukemia%3A%20enzymes%20at%20play&journal=Transl.%20Oncol.&volume=14&publication_year=2021&author=Zhang%2CW&author=Gou%2CP&author=Dupret%2CJ-M&author=Chomienne%2CC&author=Rodrigues-Lima%2CF) 
    
27.  Tate, J. G. et al. COSMIC: the catalogue of somatic mutations in cancer. _Nucleic Acids Res._ **47**, D941–D947 (2019).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Cgt7bM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=COSMIC%3A%20the%20catalogue%20of%20somatic%20mutations%20in%20cancer&journal=Nucleic%20Acids%20Res.&volume=47&pages=D941-D947&publication_year=2019&author=Tate%2CJG) 
    
28.  Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. _Nature_ **561**, 473–478 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1OmsrrJ)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Population%20dynamics%20of%20normal%20human%20blood%20inferred%20from%20somatic%20mutations&journal=Nature&volume=561&pages=473-478&publication_year=2018&author=Lee-Six%2CH) 
    
29.  Osorio, F. G. et al. Somatic mutations reveal lineage relationships and age-related mutagenesis in human hematopoiesis. _Cell Rep._ **25**, 2308–2316 (2018).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlCls7vO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Somatic%20mutations%20reveal%20lineage%20relationships%20and%20age-related%20mutagenesis%20in%20human%20hematopoiesis&journal=Cell%20Rep.&volume=25&pages=2308-2316&publication_year=2018&author=Osorio%2CFG) 
    
30.  Machado, H. E. et al. Diverse mutational landscapes in human lymphocytes. _Nature_ **608**, 724–732 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFSjtbvI)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Diverse%20mutational%20landscapes%20in%20human%20lymphocytes&journal=Nature&volume=608&pages=724-732&publication_year=2022&author=Machado%2CHE) 
    
31.  Maura, F. et al. Molecular evolution of classic Hodgkin lymphoma revealed through whole-genome sequencing of Hodgkin and Reed Sternberg cells. _Blood Cancer Discov._ **4**, 208–227 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlCrsb%2FK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20evolution%20of%20classic%20Hodgkin%20lymphoma%20revealed%20through%20whole-genome%20sequencing%20of%20Hodgkin%20and%20Reed%20Sternberg%20cells&journal=Blood%20Cancer%20Discov.&volume=4&pages=208-227&publication_year=2023&author=Maura%2CF) 
    
32.  Bertrums, E. J. M. et al. Elevated mutational age in blood of children treated for cancer contributes to therapy-related myeloid neoplasms. _Cancer Discov._ **12**, 1860–1872 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1ynur3P)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Elevated%20mutational%20age%20in%20blood%20of%20children%20treated%20for%20cancer%20contributes%20to%20therapy-related%20myeloid%20neoplasms&journal=Cancer%20Discov.&volume=12&pages=1860-1872&publication_year=2022&author=Bertrums%2CEJM) 
    
33.  Bolli, N. et al. Heterogeneity of genomic evolution and mutational profiles in multiple myeloma. _Nat. Commun._ **5**, 2997 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Heterogeneity%20of%20genomic%20evolution%20and%20mutational%20profiles%20in%20multiple%20myeloma&journal=Nat.%20Commun.&volume=5&publication_year=2014&author=Bolli%2CN) 
    
34.  Rustad, E. H. et al. Timing the initiation of multiple myeloma. _Nat. Commun._ **11**, 1917 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnslSktbo%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Timing%20the%20initiation%20of%20multiple%20myeloma&journal=Nat.%20Commun.&volume=11&publication_year=2020&author=Rustad%2CEH) 
    
35.  Maura, F. et al. The mutagenic impact of melphalan in multiple myeloma. _Leukemia_ **35**, 2145–2150 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlKnu73E)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutagenic%20impact%20of%20melphalan%20in%20multiple%20myeloma&journal=Leukemia&volume=35&pages=2145-2150&publication_year=2021&author=Maura%2CF) 
    
36.  Coorens, T. H. H. et al. Clonal hematopoiesis and therapy-related myeloid neoplasms following neuroblastoma treatment. _Blood_ **137**, 2992–2997 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFaiu7vM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20and%20therapy-related%20myeloid%20neoplasms%20following%20neuroblastoma%20treatment&journal=Blood&volume=137&pages=2992-2997&publication_year=2021&author=Coorens%2CTHH) 
    
37.  Christensen, S. et al. 5-Fluorouracil treatment induces characteristic T>G mutations in human cancer. _Nat. Commun._ **10**, 4571 (2019).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=5-Fluorouracil%20treatment%20induces%20characteristic%20T%3EG%20mutations%20in%20human%20cancer&journal=Nat.%20Commun.&volume=10&publication_year=2019&author=Christensen%2CS) 
    
38.  Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. _Nature_ **606**, 335–342 (2022).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVant7jN)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20longitudinal%20dynamics%20and%20natural%20history%20of%20clonal%20haematopoiesis&journal=Nature&volume=606&pages=335-342&publication_year=2022&author=Fabre%2CMA) 
    
39.  Stelmach, P. et al. Clonal hematopoiesis with _DNMT3A_ and _PPM1D_ mutations impairs regeneration in autologous stem cell transplant recipients. _Haematologica_ **108**, 3308–3320 (2023).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitFSis74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clonal%20hematopoiesis%20with%20DNMT3A%20and%20PPM1D%20mutations%20impairs%20regeneration%20in%20autologous%20stem%20cell%20transplant%20recipients&journal=Haematologica&volume=108&pages=3308-3320&publication_year=2023&author=Stelmach%2CP) 
    
40.  Brodsky, R. A. High dose cyclophosphamide treatment for autoimmune disorders. _Sci. World J._ **2**, 1808–1815 (2002).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High%20dose%20cyclophosphamide%20treatment%20for%20autoimmune%20disorders&journal=Sci.%20World%20J.&volume=2&pages=1808-1815&publication_year=2002&author=Brodsky%2CRA) 
    
41.  Koh, G. C. C. et al. The chemotherapeutic drug CX-5461 is a potent mutagen in cultured human cells. _Nat. Genet._ **56**, 23–26 (2024).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXisFSmu7rK)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20chemotherapeutic%20drug%20CX-5461%20is%20a%20potent%20mutagen%20in%20cultured%20human%20cells&journal=Nat.%20Genet.&volume=56&pages=23-26&publication_year=2024&author=Koh%2CGCC) 
    
42.  Curtis, R. E. et al. Risk of leukemia after chemotherapy and radiation treatment for breast cancer. _N. Engl. J. Med._ **326**, 1745–1751 (1992).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK383nvVeqsA%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Risk%20of%20leukemia%20after%20chemotherapy%20and%20radiation%20treatment%20for%20breast%20cancer&journal=N.%20Engl.%20J.%20Med.&volume=326&pages=1745-1751&publication_year=1992&author=Curtis%2CRE) 
    
43.  Kaldor, J. M. et al. Leukemia following Hodgkin’s disease. _N. Engl. J. Med._ **322**, 7–13 (1990).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK3c%2FpvVSjtw%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Leukemia%20following%20Hodgkin%E2%80%99s%20disease&journal=N.%20Engl.%20J.%20Med.&volume=322&pages=7-13&publication_year=1990&author=Kaldor%2CJM) 
    
44.  Mauz-Körholz, C. et al. Procarbazine-free OEPA–COPDAC chemotherapy in boys and standard OPPA–COPP in girls have comparable effectiveness in pediatric Hodgkin’s lymphoma: the GPOH-HD-2002 study. _J. Clin. Oncol._ **28**, 3680–3686 (2010).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Procarbazine-free%20OEPA%E2%80%93COPDAC%20chemotherapy%20in%20boys%20and%20standard%20OPPA%E2%80%93COPP%20in%20girls%20have%20comparable%20effectiveness%20in%20pediatric%20Hodgkin%E2%80%99s%20lymphoma%3A%20the%20GPOH-HD-2002%20study&journal=J.%20Clin.%20Oncol.&volume=28&pages=3680-3686&publication_year=2010&author=Mauz-K%C3%B6rholz%2CC) 
    
45.  Eichhorst, B. et al. First-line chemoimmunotherapy with bendamustine and rituximab versus fludarabine, cyclophosphamide, and rituximab in patients with advanced chronic lymphocytic leukaemia (CLL10): an international, open-label, randomised, phase 3, non-inferiority trial. _Lancet Oncol._ **17**, 928–942 (2016).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XosVSmu7c%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=First-line%20chemoimmunotherapy%20with%20bendamustine%20and%20rituximab%20versus%20fludarabine%2C%20cyclophosphamide%2C%20and%20rituximab%20in%20patients%20with%20advanced%20chronic%20lymphocytic%20leukaemia%20%28CLL10%29%3A%20an%20international%2C%20open-label%2C%20randomised%2C%20phase%203%2C%20non-inferiority%20trial&journal=Lancet%20Oncol.&volume=17&pages=928-942&publication_year=2016&author=Eichhorst%2CB) 
    
46.  Lacout, C. et al. R–DHA–oxaliplatin (R-DHAOx) versus R–DHA–cisplatin (R-DHAP) regimen in B-cell lymphoma treatment: a eight-year trajectory study. _Eur. J. Haematol._ **105**, 223–230 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVGnsb3I)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=R%E2%80%93DHA%E2%80%93oxaliplatin%20%28R-DHAOx%29%20versus%20R%E2%80%93DHA%E2%80%93cisplatin%20%28R-DHAP%29%20regimen%20in%20B-cell%20lymphoma%20treatment%3A%20a%20eight-year%20trajectory%20study&journal=Eur.%20J.%20Haematol.&volume=105&pages=223-230&publication_year=2020&author=Lacout%2CC) 
    
47.  Mehmood, R. K. Review of cisplatin and oxaliplatin in current immunogenic and monoclonal antibody treatments. _Oncol. Rev._ **8**, 256 (2014).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Review%20of%20cisplatin%20and%20oxaliplatin%20in%20current%20immunogenic%20and%20monoclonal%20antibody%20treatments&journal=Oncol.%20Rev.&volume=8&publication_year=2014&author=Mehmood%2CRK) 
    
48.  Ellis, P. et al. Reliable detection of somatic mutations in solid tissues by laser-capture microdissection and low-input DNA sequencing. _Nat. Protoc._ **16**, 841–871 (2021).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1Sgs7zP)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reliable%20detection%20of%20somatic%20mutations%20in%20solid%20tissues%20by%20laser-capture%20microdissection%20and%20low-input%20DNA%20sequencing&journal=Nat.%20Protoc.&volume=16&pages=841-871&publication_year=2021&author=Ellis%2CP) 
    
49.  Moore, L. et al. The mutational landscape of normal human endometrial epithelium. _Nature_ **580**, 640–646 (2020).
    
    [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVCjsb4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20mutational%20landscape%20of%20normal%20human%20endometrial%20epithelium&journal=Nature&volume=580&pages=640-646&publication_year=2020&author=Moore%2CL) 
    
50.  Hoang, D. T. et al. MPBoot: fast phylogenetic maximum parsimony tree inference and bootstrap approximation. _BMC Evol. Biol._ [https://doi.org/10.1186/s12862-018-1131-3](https://doi.org/10.1186/s12862-018-1131-3) (2018).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MPBoot%3A%20fast%20phylogenetic%20maximum%20parsimony%20tree%20inference%20and%20bootstrap%20approximation&journal=BMC%20Evol.%20Biol.&doi=10.1186%2Fs12862-018-1131-3&publication_year=2018&author=Hoang%2CDT) 
    
51.  Liu, M., Wu, Y., Jiang, N., Boot, A. & Rozen, S. G. mSigHdp: hierarchical Dirichlet process mixture modeling for mutational signature discovery. _NAR Genom. Bioinform._ **5**, lqad005 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=mSigHdp%3A%20hierarchical%20Dirichlet%20process%20mixture%20modeling%20for%20mutational%20signature%20discovery&journal=NAR%20Genom.%20Bioinform.&volume=5&publication_year=2023&author=Liu%2CM&author=Wu%2CY&author=Jiang%2CN&author=Boot%2CA&author=Rozen%2CSG) 
    
52.  Díaz-Gay, M. et al. Assigning mutational signatures to individual samples and individual somatic mutations with SigProfilerAssignment. _Bioinformatics_ **39**, btad756 (2023).
    
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Assigning%20mutational%20signatures%20to%20individual%20samples%20and%20individual%20somatic%20mutations%20with%20SigProfilerAssignment&journal=Bioinformatics&volume=39&publication_year=2023&author=D%C3%ADaz-Gay%2CM) 
    
53.  Mitchell, E. Emily-mitchell/chemotherapy: the long-term effects of chemotherapy on normal blood cells. _Zenodo_ [https://doi.org/10.5281/zenodo.15235476](https://doi.org/10.5281/zenodo.15235476) (2025).
    

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02234-x?format=refman&flavour=references)

Acknowledgements
----------------

This work was delivered as part of the Mutographs team supported by the Cancer Grand Challenges partnership funded by Cancer Research UK (CRUK) (C98/A24032). This work was supported by Wellcome grants 206194 and 220540/Z/20/A and by the NIHR Cambridge Biomedical Research Centre (BRC-1215-555 20014). J.N. was supported by CRUK, Alborda Trust, Rosetrees Trust, Blood Cancer UK and Wellcome. D.J.H. was supported by a fellowship from CRUK (RCCFEL\\100072) and received core funding from the CRUK Cambridge Centre (A25117). E.L. was supported by a Wellcome–Royal Society Sir Henry Dale Fellowship (107630/Z/15/Z). E.L. and D.J.H. were supported by core support grants from Wellcome and the Medical Research Council (MRC) to the Wellcome–MRC Cambridge Stem Cell Institute (203151/Z/16/Z). Samples were provided by the Cambridge Blood and Stem Cell Biobank, which is supported by the NIHR Cambridge BRC, Wellcome Trust–MRC Stem Cell Institute and the Cambridge Experimental Cancer Medicine Centre, UK. This research was supported by the NIHR Cambridge BRC Cell Phenotyping Hub. We are grateful to the donors, donor families and the Cambridge Biorepository for Translational Medicine for the gift of their tissue. We would like to thank L. O’Neill, K. Roberts, K. Smith, S. Austin-Guest and the staff of DNA Pipelines at the Wellcome Sanger Institute for their contributions. For the purpose of open access, we have applied a CC BY public copyright license to any author-accepted paper version arising from this submission. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.

Author information
------------------

Author notes

1.  These authors contributed equally: Emily Mitchell, My H. Pham, Anna Clay, Jyoti Nangalia, Michael R. Stratton.
    

### Authors and Affiliations

1.  Wellcome Sanger Institute, Hinxton, UK
    
    Emily Mitchell, My H. Pham, Rashesh Sanghvi, Nicholas Williams, Hyunchul Jung, Sarah Moody, Jingwei Wang, Daniel Leonganmornlert, Michael Spencer Chapman, Ellie Dunstone, Alex Cagan, Heather E. Machado, Ultan McDermott, Inigo Martincorena, Laura Humphreys, Peter J. Campbell, Raheleh Rahbari, Jyoti Nangalia & Michael R. Stratton
    
2.  Wellcome–MRC Cambridge Stem Cell Institute, Cambridge Biomedical Campus, Cambridge, UK
    
    Emily Mitchell, Anna Clay, Sandra Pietsch, Nina Friesgaard Øbro, Aditi Vedi, Anna Santarsieri, Daniel J. Hodson, Elisa Laurenti & Jyoti Nangalia
    
3.  Department of Haematology, University of Cambridge, Cambridge, UK
    
    Emily Mitchell, Sandra Pietsch, Anna Santarsieri, E. Joanna Baxter, George Follows, Daniel J. Hodson, Elisa Laurenti & Jyoti Nangalia
    
4.  Signalling Programme, The Babraham Institute, Babraham Research Campus, Cambridge, UK
    
    Anna Clay
    
5.  Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX, USA
    
    Joanne I. Hsu & Margaret A. Goodell
    
6.  Department of Clinical Immunology, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark
    
    Nina Friesgaard Øbro
    
7.  Department of Paediatrics, Cambridge University Hospitals Foundation Trust, Cambridge, UK
    
    Aditi Vedi
    
8.  Department of Oncology, Cambridge University Hospitals Foundation Trust, Cambridge, UK
    
    Ultan McDermott & Gary J. Doherty
    
9.  Oncology R&D, AstraZeneca, Cambridge, UK
    
    Ultan McDermott
    
10.  Department of Surgery, University of Cambridge, Cambridge, UK
    
    Krishnaa Mahbubani & Kourosh Saeb Parsy
    
11.  Cambridge Biorepository for Translational Medicine, NIHR Cambridge Biomedical Research Centre, University of Cambridge, Cambridge, UK
    
    Krishnaa Mahbubani & Kourosh Saeb Parsy
    
12.  Department of Leukemia, Division of Cancer Medicine, MD Anderson Cancer Center, University of Texas, Houston, TX, USA
    
    Koichi Takahashi
    
13.  York Biomedical Research Institute, Department of Biology, University of York, York, UK
    
    David Kent
    

Authors

1.  Emily Mitchell
    
    [View author publications](https://www.nature.com/search?author=Emily%20Mitchell)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Emily%20Mitchell) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Emily%20Mitchell%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
2.  My H. Pham
    
    [View author publications](https://www.nature.com/search?author=My%20H.%20Pham)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=My%20H.%20Pham) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22My%20H.%20Pham%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
3.  Anna Clay
    
    [View author publications](https://www.nature.com/search?author=Anna%20Clay)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anna%20Clay) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anna%20Clay%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
4.  Rashesh Sanghvi
    
    [View author publications](https://www.nature.com/search?author=Rashesh%20Sanghvi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rashesh%20Sanghvi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rashesh%20Sanghvi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
5.  Nicholas Williams
    
    [View author publications](https://www.nature.com/search?author=Nicholas%20Williams)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nicholas%20Williams) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nicholas%20Williams%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
6.  Sandra Pietsch
    
    [View author publications](https://www.nature.com/search?author=Sandra%20Pietsch)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sandra%20Pietsch) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sandra%20Pietsch%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
7.  Joanne I. Hsu
    
    [View author publications](https://www.nature.com/search?author=Joanne%20I.%20Hsu)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Joanne%20I.%20Hsu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Joanne%20I.%20Hsu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
8.  Nina Friesgaard Øbro
    
    [View author publications](https://www.nature.com/search?author=Nina%20Friesgaard%20%C3%98bro)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Nina%20Friesgaard%20%C3%98bro) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Nina%20Friesgaard%20%C3%98bro%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
9.  Hyunchul Jung
    
    [View author publications](https://www.nature.com/search?author=Hyunchul%20Jung)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hyunchul%20Jung) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hyunchul%20Jung%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
10.  Aditi Vedi
    
    [View author publications](https://www.nature.com/search?author=Aditi%20Vedi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Aditi%20Vedi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Aditi%20Vedi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
11.  Sarah Moody
    
    [View author publications](https://www.nature.com/search?author=Sarah%20Moody)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sarah%20Moody) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sarah%20Moody%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
12.  Jingwei Wang
    
    [View author publications](https://www.nature.com/search?author=Jingwei%20Wang)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jingwei%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jingwei%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
13.  Daniel Leonganmornlert
    
    [View author publications](https://www.nature.com/search?author=Daniel%20Leonganmornlert)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20Leonganmornlert) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20Leonganmornlert%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
14.  Michael Spencer Chapman
    
    [View author publications](https://www.nature.com/search?author=Michael%20Spencer%20Chapman)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Michael%20Spencer%20Chapman) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Michael%20Spencer%20Chapman%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
15.  Ellie Dunstone
    
    [View author publications](https://www.nature.com/search?author=Ellie%20Dunstone)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ellie%20Dunstone) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ellie%20Dunstone%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
16.  Anna Santarsieri
    
    [View author publications](https://www.nature.com/search?author=Anna%20Santarsieri)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anna%20Santarsieri) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anna%20Santarsieri%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
17.  Alex Cagan
    
    [View author publications](https://www.nature.com/search?author=Alex%20Cagan)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Alex%20Cagan) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Alex%20Cagan%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
18.  Heather E. Machado
    
    [View author publications](https://www.nature.com/search?author=Heather%20E.%20Machado)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Heather%20E.%20Machado) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Heather%20E.%20Machado%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
19.  E. Joanna Baxter
    
    [View author publications](https://www.nature.com/search?author=E.%20Joanna%20Baxter)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=E.%20Joanna%20Baxter) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22E.%20Joanna%20Baxter%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
20.  George Follows
    
    [View author publications](https://www.nature.com/search?author=George%20Follows)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=George%20Follows) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22George%20Follows%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
21.  Daniel J. Hodson
    
    [View author publications](https://www.nature.com/search?author=Daniel%20J.%20Hodson)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Daniel%20J.%20Hodson) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Daniel%20J.%20Hodson%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
22.  Ultan McDermott
    
    [View author publications](https://www.nature.com/search?author=Ultan%20McDermott)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ultan%20McDermott) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ultan%20McDermott%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
23.  Gary J. Doherty
    
    [View author publications](https://www.nature.com/search?author=Gary%20J.%20Doherty)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gary%20J.%20Doherty) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gary%20J.%20Doherty%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
24.  Inigo Martincorena
    
    [View author publications](https://www.nature.com/search?author=Inigo%20Martincorena)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Inigo%20Martincorena) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Inigo%20Martincorena%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
25.  Laura Humphreys
    
    [View author publications](https://www.nature.com/search?author=Laura%20Humphreys)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Laura%20Humphreys) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Laura%20Humphreys%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
26.  Krishnaa Mahbubani
    
    [View author publications](https://www.nature.com/search?author=Krishnaa%20Mahbubani)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Krishnaa%20Mahbubani) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Krishnaa%20Mahbubani%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
27.  Kourosh Saeb Parsy
    
    [View author publications](https://www.nature.com/search?author=Kourosh%20Saeb%20Parsy)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kourosh%20Saeb%20Parsy) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kourosh%20Saeb%20Parsy%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
28.  Koichi Takahashi
    
    [View author publications](https://www.nature.com/search?author=Koichi%20Takahashi)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Koichi%20Takahashi) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Koichi%20Takahashi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
29.  Margaret A. Goodell
    
    [View author publications](https://www.nature.com/search?author=Margaret%20A.%20Goodell)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Margaret%20A.%20Goodell) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Margaret%20A.%20Goodell%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
30.  David Kent
    
    [View author publications](https://www.nature.com/search?author=David%20Kent)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=David%20Kent) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22David%20Kent%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
31.  Elisa Laurenti
    
    [View author publications](https://www.nature.com/search?author=Elisa%20Laurenti)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Elisa%20Laurenti) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Elisa%20Laurenti%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
32.  Peter J. Campbell
    
    [View author publications](https://www.nature.com/search?author=Peter%20J.%20Campbell)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peter%20J.%20Campbell) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peter%20J.%20Campbell%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
33.  Raheleh Rahbari
    
    [View author publications](https://www.nature.com/search?author=Raheleh%20Rahbari)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Raheleh%20Rahbari) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Raheleh%20Rahbari%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
34.  Jyoti Nangalia
    
    [View author publications](https://www.nature.com/search?author=Jyoti%20Nangalia)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jyoti%20Nangalia) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jyoti%20Nangalia%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    
35.  Michael R. Stratton
    
    [View author publications](https://www.nature.com/search?author=Michael%20R.%20Stratton)
    
    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Michael%20R.%20Stratton) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Michael%20R.%20Stratton%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
    

### Contributions

A. Clay and E.M. performed cell sorting and culture. E.M. performed mutation calling and phylogeny construction. M.H.P. performed mutational signature analysis and prepared figures. R.S. and R.R. supported mutational signature analysis. N.W., M.S.C. and D.L. supported phylogenetic and signature analysis. S.P., H.E.M., D.K. and E.L. supported cell isolation and culture. H.J., S.M., E.D. and J.W. supported mutation analysis. A. Cagan helped with figure design. U.M., G.J.D., K.M., A.S., A.V., G.F., D.J.H., K.S.P., K.T. and E.J.B. provided clinical advice and patient samples. P.J.C. and I.M. provided scientific advice. J.I.H. and M.A.G. provided clinical samples and supported cell culture and mutation identification. L.H. provided project coordination. N.F.Ø. provided development of laboratory methods. E.M., J.N. and M.R.S. wrote the paper. J.N. and M.R.S. directed the research.

### Corresponding authors

Correspondence to [Jyoti Nangalia](mailto:jn5@sanger.ac.uk) or [Michael R. Stratton](mailto:mrs@sanger.ac.uk).

Ethics declarations
-------------------

### Competing interests

G.J.D. and U.M. are employees of and shareholders in AstraZeneca. M.R.S. and P.J.C. are cofounders of and shareholders in Quotient Therapeutics. The other authors declare no competing interests.

Peer review
-----------

### Peer review information

_Nature Genetics_ thanks Terrence Wong and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [Peer reviewer reports](https://www.nature.com/articles/s41588-025-02234-x#MOESM3) are available.

Additional information
----------------------

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Extended data
-------------

### [Extended Data Fig. 1 Mean sequencing depth in the normal and chemotherapy exposed HSPC colonies used for mutation burden analysis, and patterns of mutation accumulation.](https://www.nature.com/articles/s41588-025-02234-x/figures/6)

**a**, Box plot representing the quartile distribution of mean sequencing depth in 90 colonies from normal (blue) and 189 colonies from chemotherapy exposed (red) individuals. The boxes indicate the median and interquartile range, the whiskers denote the minimum and maximum, with outlying values represented as points. **b**, Boxplot comparing the mean sequencing depth between Alkylating/ Platinum agent exposed and non-exposed colonies. The number of colonies in each agent group are shown at the top of the plot. The boxes indicate the median and interquartile range, the whiskers denote the minimum and maximum, with outlying values represented as points. **c-f**, HSC single base substitutions associated with chemotherapy (**c,d**) and age (**e,f**). CC, colorectal carcinoma; LC, lung cancer; NB, neuroblastoma; FL, follicular lymphoma; DLBL, diffuse large B cell lymphoma; MZL, marginal zone lymphoma; LL, lymphoplasmacytic lymphoma; M, multiple myeloma; HL, Hodgkin lymphoma; AML, acute myeloid leukaemia; 1, Platinum agents; 2,Alkylating agents; 3,Antimetabolites; 4,Topo I inhibitors; 5,Topo II inhibitors; 6,Vinca alkaloids; 7,Cytotoxic antibiotics.

[Source data](https://www.nature.com/articles/s41588-025-02234-x#MOESM5)

### [Extended Data Fig. 2 Indel mutational burden in normal and chemotherapy exposed HSPCs.](https://www.nature.com/articles/s41588-025-02234-x/figures/7)

Burden of small indels in single HSPC colonies with age (years) across normal (blue) and the four chemotherapy exposed (red) individuals with the highest indel burdens. The points represent individual HSPC colonies. The boxes indicate the median and interquartile range, the whiskers denote the minimum and maximum. The blue line represents a regression of age on mutation burden, with 95% CI shaded. **b**, Depiction of data as in a, but the y-axis is cut off at 120 indels for better visualisation of the majority of the chemotherapy-exposed data. The points represent individual HSPC colonies. The boxes indicate the median and interquartile range, the whiskers denote the minimum and maximum. The blue line represents a regression of age on mutation burden, with 95% CI shaded **c,d**, Bar plots showing the number of structure variant types (**c**) and the number of the number of independently acquired autosomal copy number aberrations (CNAs) (**d**) in each individual from chemotherapy and normal groups. The absolute number of events found in each individual is shown at the top of each bar. Individuals and the total number of isolated colonies are sorted by age within each group.

[Source data](https://www.nature.com/articles/s41588-025-02234-x#MOESM5)

### [Extended Data Fig. 3 Indel signatures that are present in normal and chemotherapy exposed blood.](https://www.nature.com/articles/s41588-025-02234-x/figures/8)

**a**, Three indel signatures (ID1/2, ID3/5/9, IDA) were extracted by sigHDP. The context on the x-axis show the contributions of different types of indels, grouped by whether variants are deletions or insertions, the size of the event, the presence within repeat units and the sequence content of the indel. **b**, The proportion of indels and indels burden per mutational signatures across 22 chemotherapy exposed and 9 normal individuals, extracting using msigHDP ([Methods](https://www.nature.com/articles/s41588-025-02234-x#Sec7)). Each column represents samples from one individual. Signatures with the contribution <5% are considered as ‘unassigned’.

[Source data](https://www.nature.com/articles/s41588-025-02234-x#MOESM5)

### [Extended Data Fig. 4 Phylogenetic trees and mutational signatures in normal individuals.](https://www.nature.com/articles/s41588-025-02234-x/figures/9)

Branch lengths correspond to SBS burdens. A stacked bar plot represents the SBS mutational signatures contributing to each branch with color code below the trees. SBSUnassigned indicates mutations that could not confidently be assigned to any reported signature.

### [Extended Data Fig. 5 Phylogenetic trees and mutational signatures in individuals treated with alkylating agents.](https://www.nature.com/articles/s41588-025-02234-x/figures/10)

**a**, Phylogenetic tree of 48-year-old chemotherapy exposed female (PD47703). Branch lengths correspond to SBS burdens. A stacked bar plot represents the SBS mutational signatures contributing to each branch with colour code below the trees. SBSUnassigned indicates mutations that could not confidently be assigned to any reported signature. She had been treated with chlorambucil and procarbazine at age 10 (early), and bendamustine at age 47 (late). **b**, Phylogenetic trees and SBS mutational signatures in individuals treated with cyclophosphamide.

### [Extended Data Fig. 6 Phylogenetic trees and mutational signatures in individuals treated with oxaliplatin.](https://www.nature.com/articles/s41588-025-02234-x/figures/11)

Branch lengths correspond to SBS burdens. A stacked bar plot represents the SBS mutational signatures contributing to each branch with colour code below the trees. SBSUnassigned indicates mutations that could not confidently be assigned to any reported signature.

### [Extended Data Fig. 7 Annotated HSPC phylogenies for two chemotherapy treated individuals.](https://www.nature.com/articles/s41588-025-02234-x/figures/12)

Phylogenies were constructed for PD37580 (**a**) and PD47703 (**b**) individuals using shared mutation data and the algorithm MPBoot ([Methods](https://www.nature.com/articles/s41588-025-02234-x#Sec7)). In all phylogenies, branch lengths reflect the number of SBS mutations assigned to the branch. The y-axis represents the number of SBSs accumulating over time. Each tip on a phylogeny represents a single colony. Chemotherapy agents and the age of exposure to them are shown on top of the trees. **a**, PD37580 phylogeny of early life, truncated at 400 SBS mutations to allow better visualisation of the timing of acquisition of two early _PPM1D_ mutations (pink). The number of mutations at age 13 was estimated using the linear mixed model described in Mitchell et al with 95% CI based on mutation burden being Poisson distributed as described in methods (241-306 single base subsitutions). **b**, Comparison of phylogenies created from peripheral blood samples taken from PD44703 one year apart. Pathogenic mutations in _PPM1D_ have been highlighted (pink) to facilitate comparison of clone sizes at each timepoint. In addition a loss of function mutation in _CSF3R_ has been highlighted (blue), which could also be contributing to loss of haematopoietic reserve and cytopenias. Red bars show the size of clonal fractions at each timepoint. Terminal branches have been adjusted for sequence coverage, and overall root-to-tip branch lengths have been normalized to the same total length (because all colonies were collected from a single time point).

### [Extended Data Fig. 8 HSPC phylogenies for three young adult individuals.](https://www.nature.com/articles/s41588-025-02234-x/figures/13)

Phylogenies for one normal young adult individual (top) and two young adult chemotherapy-treated individuals (bottom) were constructed using shared mutation data and the algorithm MPBoot ([Methods](https://www.nature.com/articles/s41588-025-02234-x#Sec7)). Branch lengths reflect the number of mutations assigned to the branch with terminal branches adjusted for sequence coverage, and overall root-to-tip branch lengths have been normalised to the same total length (because all colonies were collected from a single time point). The y-axis represents the number of SBSs accumulating over time. Each tip on a phylogeny represents a single colony, with the respective numbers of colonies of each cell and tissue type recorded at the top. Onto these trees, we have layered clone and colony-specific phenotypic information. We have highlighted branches on which we have identified known oncogenic drivers in one of 18 clonal haematopoiesis genes (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)) colour-coded by gene. A heat map at the bottom of each phylogeny highlights colonies from known driver clades coloured by gene, and the expanded clades (defined as those with a clonal fraction above 1%) in blue. Regarding the treatment of PD50307 donor, carboplatin was administered via intravenous infusion on Day 1, followed by Etoposide on the same day. Subsequently, the patient received oral doses of Etoposide on Days 2 and 3.

### [Extended Data Fig. 9 HSPC phylogenies for four older adult individuals.](https://www.nature.com/articles/s41588-025-02234-x/figures/14)

Phylogenies for two normal individuals (top) and two chemotherapy-treated individuals (bottom) were constructed using shared mutation data and the algorithm MPBoot ([Methods](https://www.nature.com/articles/s41588-025-02234-x#Sec7)). Branch lengths reflect the number of mutations assigned to the branch with terminal branches adjusted for sequence coverage, and overall root-to-tip branch lengths have been normalised to the same total length (because all colonies were collected from a single time point). The y-axis represents the number of SBSs accumulating over time. Each tip on a phylogeny represents a single colony, with the respective numbers of colonies of each cell and tissue type recorded at the top. Onto these trees, we have layered clone and colony-specific phenotypic information. We have highlighted branches on which we have identified known oncogenic drivers in one of 18 clonal haematopoiesis genes (Supplementary Table [2](https://www.nature.com/articles/s41588-025-02234-x#MOESM4)) colour-coded by gene. A heat map at the bottom of each phylogeny highlights colonies from known driver clades coloured by gene, and the expanded clades (defined as those with a clonal fraction above 1%) in blue.

Supplementary information
-------------------------

### [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_MOESM1_ESM.pdf)

Supplementary Methods, Figs. 1–4 and refs. 1–11.

### [Reporting Summary](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_MOESM2_ESM.pdf)

### [Peer Review File](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_MOESM3_ESM.pdf)

### [Supplementary Tables](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_MOESM4_ESM.xlsx)

Supplementary Tables 1–11.

Source data
-----------

### [Source Data Figs. 2 and 4 and Extended Data Figs. 1–3](https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-025-02234-x/MediaObjects/41588_2025_2234_MOESM5_ESM.xlsx)

Source data provided to support Figs. 2a,b, 2c and 4 and Extended Data Figs. 1a,b, 2a,b, 2c, 2d, 3a and 3b. Source data are not applicable for the remaining figures as these contain phylogenies only.

Rights and permissions
----------------------

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=The%20long-term%20effects%20of%20chemotherapy%20on%20normal%20blood%20cells&author=Emily%20Mitchell%20et%20al&contentID=10.1038%2Fs41588-025-02234-x&copyright=The%20Author%28s%29&publication=1061-4036&publicationDate=2025-07-01&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY)

About this article
------------------

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-025-02234-x)

### Cite this article

Mitchell, E., Pham, M.H., Clay, A. _et al._ The long-term effects of chemotherapy on normal blood cells. _Nat Genet_ **57**, 1684–1694 (2025). https://doi.org/10.1038/s41588-025-02234-x

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41588-025-02234-x?format=refman&flavour=citation)

-   Received: 11 September 2024
    
-   Accepted: 16 May 2025
    
-   Published: 01 July 2025
    
-   Issue Date: July 2025
    
-   DOI: https://doi.org/10.1038/s41588-025-02234-x
    

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

[Download PDF](https://www.nature.com/articles/s41588-025-02234-x.pdf)
